<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/7/1403" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/7/1403.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/7/1403.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/7/1403.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/7/1403.atom" />
<meta name="HW.pisa" content="ggg;5/7/1403" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection" />
<meta name="DC.Identifier" content="10.1534/g3.115.018796" />
<meta name="DC.Date" content="2015-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Leal-Bertioli et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Rust is a major pathogen of the peanut crop. Development and adoption of rust-resistant cultivars is the most cost efficient and effective way to control the spread of the disease and reduce yield losses. Some cultivated peanut germplasm accessions have a degree of resistance, but the secondary gene pool is a source of much stronger resistance alleles. Wild species, however, have undesirable agronomic traits that are a disincentive to their use in breeding. The identification of genomic regions that harbor disease resistance in wild species is the first step in the implementation of marker-assisted selection that can speed the introgression of wild disease resistances and the elimination of linkage drag. In this work, we identify genome regions that control different components of rust resistance in a recombinant inbred line population developed from a cross between two Arachis species, the susceptible most probable B genome ancestor of cultivated peanut, Arachis ipaënsis , and an accession of its closest relative, Arachis magna , which is resistant to rust. Quantitative trait loci for several components of resistance were placed in the same position on linkage group B08. Single-nucleotide polymorphism Kompetitive allele-specific polymerase chain reaction markers for rust resistance region were designed and validated for marker function in both diploid and tetraploid contexts." />
<meta name="DC.Contributor" content="Soraya C. M. Leal-Bertioli" />
<meta name="DC.Contributor" content="Uiara Cavalcante" />
<meta name="DC.Contributor" content="Ediene G. Gouvea" />
<meta name="DC.Contributor" content="Carolina Ballén-Taborda" />
<meta name="DC.Contributor" content="Kenta Shirasawa" />
<meta name="DC.Contributor" content="Patrícia M. Guimarães" />
<meta name="DC.Contributor" content="Scott A. Jackson" />
<meta name="DC.Contributor" content="David J. Bertioli" />
<meta name="DC.Contributor" content="Márcio C. Moretzsohn" />
<meta name="article:published_time" content="2015-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Rust is a major pathogen of the peanut crop. Development and adoption of rust-resistant cultivars is the most cost efficient and effective way to control the spread of the disease and reduce yield losses. Some cultivated peanut germplasm accessions have a degree of resistance, but the secondary gene pool is a source of much stronger resistance alleles. Wild species, however, have undesirable agronomic traits that are a disincentive to their use in breeding. The identification of genomic regions that harbor disease resistance in wild species is the first step in the implementation of marker-assisted selection that can speed the introgression of wild disease resistances and the elimination of linkage drag. In this work, we identify genome regions that control different components of rust resistance in a recombinant inbred line population developed from a cross between two &lt;i&gt;Arachis&lt;/i&gt; species, the susceptible most probable B genome ancestor of cultivated peanut, &lt;i&gt;Arachis ipaënsis&lt;/i&gt;, and an accession of its closest relative, &lt;i&gt;Arachis magna&lt;/i&gt;, which is resistant to rust. Quantitative trait loci for several components of resistance were placed in the same position on linkage group B08. Single-nucleotide polymorphism Kompetitive allele-specific polymerase chain reaction markers for rust resistance region were designed and validated for marker function in both diploid and tetraploid contexts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/07/01" />
<meta name="citation_mjid" content="ggg;5/7/1403" />
<meta name="citation_id" content="5/7/1403" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/7/1403" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/7/1403.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/7/1403.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/7/1403.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.018796" />
<meta name="citation_pmid" content="25943521" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1403" />
<meta name="citation_lastpage" content="1413" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Soraya C. M. Leal-Bertioli" />
<meta name="citation_author_institution" content="Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil," />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810" />
<meta name="citation_author_email" content="soraya.bertioli@embrapa.br" />
<meta name="citation_author" content="Uiara Cavalcante" />
<meta name="citation_author_institution" content="Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil," />
<meta name="citation_author_institution" content="University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil" />
<meta name="citation_author" content="Ediene G. Gouvea" />
<meta name="citation_author_institution" content="Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil," />
<meta name="citation_author_institution" content="University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil" />
<meta name="citation_author" content="Carolina Ballén-Taborda" />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810" />
<meta name="citation_author" content="Kenta Shirasawa" />
<meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, 292-0818, Japan" />
<meta name="citation_author" content="Patrícia M. Guimarães" />
<meta name="citation_author_institution" content="Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil," />
<meta name="citation_author" content="Scott A. Jackson" />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810" />
<meta name="citation_author" content="David J. Bertioli" />
<meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810" />
<meta name="citation_author_institution" content="University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil" />
<meta name="citation_author" content="Márcio C. Moretzsohn" />
<meta name="citation_author_institution" content="Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil," />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=M. Bechara;citation_author=M. C. Moretzsohn;citation_author=D. Palmieri;citation_author=J. Monteiro;citation_author=M. Bacci;citation_title=Phylogenetic relationships in genus Arachis based on ITS and 5.8S rDNA sequences.;citation_pages=255-255;citation_volume=10;citation_year=2010;citation_pmid=21092103" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=D.J. Bertioli;citation_author=P. Ozias-Akins;citation_author=Y. Chu;citation_author=K.M. Dantas;citation_author=S.P. Santos;citation_title=The use of SNP markers for linkage mapping in diploid and tetraploid peanuts.;citation_volume=4;citation_year=2014;citation_pmid=24212082;citation_doi=10.1534/g3.113.007617" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. D. Burow;citation_author=C. E. Simpson;citation_author=W. Faries;citation_author=J. L. Starr;citation_author=A. H. Paterson;citation_title=Molecular biogeographic study of recently described B- and A-genome Arachis species, also providing new insights into the origins of cultivated peanut.;citation_pages=107-119;citation_volume=52;citation_year=2009;citation_issue=2;citation_pmid=19234559;citation_doi=10.1139/G08-094" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. D. Burow;citation_author=J. L. Starr;citation_author=C.-H. Park;citation_author=C. E. Simpson;citation_author=A. H. Paterson;citation_title=Introgression of homeologous quantitative trait loci (QTLs) for resistance to the root-knot nematode [Meloidogyne arenaria (Neal) Chitwood] in an advanced backcross-QTL population of peanut (Arachis hypogaea L.).;citation_pages=393-406;citation_volume=34;citation_year=2014" />
<meta name="citation_reference" content="citation_title=The Plant Genome;citation_author=Y. Chu;citation_author=C. Wu;citation_author=C.C. Holbrook;citation_author=B. Tillman;citation_author=G. Person;citation_title=Marker-assisted selection to pyramid nematode resistance and the high oleic trait in peanut.;citation_pages=110-117;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. W. Doerge;citation_author=G. A. Churchill;citation_title=Permutation Tests for Multiple Loci Affecting a Quantitative Character;citation_pages=285-294;citation_volume=142;citation_year=1996;citation_issue=1;citation_pmid=8770605" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Duitama;citation_author=J. C. Quintero;citation_author=D. F. Cruz;citation_author=C. Quintero;citation_author=G. Hubmann;citation_title=An integrated framework for discovery and genotyping of genomic variants from high-throughput sequencing experiments.;citation_volume=42;citation_year=2014;citation_pmid=24413664;citation_doi=10.1093/nar/gkt1381" />
<meta name="citation_reference" content="citation_journal_title=Scientia Agricolae;citation_author=A. P. Fávero;citation_author=S. A. Moraes;citation_author=A. A. F. Garcia;citation_author=J. F. M. Valls;citation_author=N. A. Vello;citation_title=Characterization of rust, early and late leaf spot resistance in wild and cultivated peanut germplasm.;citation_pages=110-117;citation_volume=66;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Foncéka;citation_author=H. Tossim;citation_author=R. Rivallan;citation_author=H. Vignes;citation_author=E. Lacut;citation_title=Construction of Chromosome Segment Substitution Lines in Peanut (Arachis hypogaea L.) Using a Wild Synthetic and QTL Mapping for Plant Morphology.;citation_pages=e48642-e48642;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185268;citation_doi=10.1371/journal.pone.0048642" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Foncéka;citation_author=H.-A. Tossim;citation_author=R. Rivallan;citation_author=H. Vignes;citation_author=I. Faye;citation_title=Fostered and left behind alleles in peanut: interspecific QTL mapping reveals footprints of domestication and useful natural variation for breeding.;citation_pages=26-26;citation_volume=12;citation_year=2012;citation_pmid=22340522;citation_doi=10.1186/1471-2229-12-26" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Gautami;citation_author=D. Foncéka;citation_author=M. K. Pandey;citation_author=M. C. Moretzsohn;citation_author=V. Sujay;citation_title=An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (Arachis hypogaea L.).;citation_pages=e41213-e41213;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22815973;citation_doi=10.1371/journal.pone.0041213" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Grattapaglia;citation_author=R. Sederoff;citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers.;citation_pages=1121-1137;citation_volume=137;citation_year=1994;citation_issue=4;citation_pmid=7982566" />
<meta name="citation_reference" content="citation_journal_title=Nat. Prot;citation_author=B. J. Haas;citation_author=P. A. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis;citation_pages=1494-1512;citation_volume=8;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Peanuts: Culture and Uses;citation_title=Early history and origin of the peanut;citation_author=R. O. Hammons;citation_pages=17-46;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Reg.;citation_author=C. C. Holbrook;citation_author=P. Timper;citation_author=A. K. Culbreath;citation_author=C. K. Kvien;citation_title=Registration of ‘Tifguard’ Peanut.;citation_pages=92-94;citation_volume=2;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Khedikar;citation_author=M. Gowda;citation_author=C. Sarvamangala;citation_author=K. Patgar;citation_author=H. Upadhyaya;citation_title=A QTL study on late leaf spot and rust revealed one major QTL for molecular breeding for rust resistance in groundnut (Arachis hypogaea L.).;citation_pages=971-984;citation_volume=121;citation_year=2010;citation_issue=5;citation_pmid=20526757;citation_doi=10.1007/s00122-010-1366-x" />
<meta name="citation_reference" content="citation_journal_title=Bonplandia;citation_author=A. Krapovickas;citation_author=W. C. Gregory;citation_title=Taxonomy of the genus Arachis (Leguminosae);citation_pages=1-205;citation_volume=16;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. Lander;citation_author=P. Green;citation_author=J. Abrahamson;citation_author=A. Barlow;citation_author=M. Daly;citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations.;citation_pages=174-181;citation_volume=1;citation_year=1987;citation_issue=2;citation_pmid=3692487;citation_doi=10.1016/0888-7543(87)90010-3" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=S. C. M. Leal-Bertioli;citation_author=A. C. V. F. Jose;citation_author=D. M. T. Alves-Freitas;citation_author=M. C. Moretzsohn;citation_author=P. M. Guimaraes;citation_title=Identification of candidate genome regions controlling disease resistance in Arachis.;citation_pages=112-112;citation_volume=9;citation_year=2009;citation_pmid=19698131;citation_doi=10.1186/1471-2229-9-112" />
<meta name="citation_reference" content="citation_journal_title=J. Phytopathol.;citation_author=S. C. M. Leal-Bertioli;citation_author=M. P. Farias;citation_author=P. T. Silva;citation_author=P. M. Guimarães;citation_author=A. C. M. Brasileiro;citation_title=Ultrastructure of the initial interaction of Puccinia arachidis and Cercosporidium personatum with leaves of Arachis hypogaea and Arachis stenosperma.;citation_pages=792-796;citation_volume=158;citation_year=2010;citation_doi=10.1111/j.1439-0434.2010.01704.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=S. C. M. Leal-Bertioli;citation_author=S. P. Santos;citation_author=K. M. Dantas;citation_author=P. W. Inglis;citation_author=S. Nielen;citation_title=Arachis batizocoi: a study of its relationship to cultivated peanut (A. hypogaea) and its potential for introgression of wild genes into the peanut crop using induced allotetraploids.;citation_pages=237-249;citation_volume=115;citation_year=2014;citation_pmid=25538110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. M. Leal-Bertioli;citation_author=K. Shirasawa;citation_author=B. Abernathy;citation_author=M. C. Moretzsohn;citation_author=C. Chavarro;citation_title=Tetrasomic recombination is surprisingly frequent in allotetraploid Arachis;citation_volume=199;citation_year=2015;citation_pmid=25701284;citation_doi=10.1534/genetics.115.174607" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=S. R. Milla;citation_author=T. G. Isleib;citation_author=H. T. Stalker;citation_title=Taxonomic relationships among Arachis sect. Arachis species as revealed by AFLP markers.;citation_pages=1-11;citation_volume=48;citation_year=2005;citation_issue=1;citation_pmid=15729391" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Milne;citation_author=P. Shaw;citation_author=G. Stephen;citation_author=M. Bayer;citation_author=L. Cardle;citation_title=Flapjack—graphical genotype visualization.;citation_volume=26;citation_year=2010;citation_pmid=20956241;citation_doi=10.1093/bioinformatics/btq580" />
<meta name="citation_reference" content="citation_journal_title=Ell. Ev. Summa Phytopathologica;citation_author=S. A. Moraes;citation_author=C. L. Salgado;citation_title=Utilização da técnica de folhas destacadas de amendoim (Arachis hypogaea L.) para inoculações com Cercospora arachidicola Hori e Cercospora personata (Bert. &amp; Curt.);citation_pages=39-55;citation_volume=8;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=M. Moretzsohn;citation_author=A. Barbosa;citation_author=D. Alves-Freitas;citation_author=C. Teixeira;citation_author=S. Leal-Bertioli;citation_title=A linkage map for the B-genome of Arachis (Fabaceae) and its synteny to the A-genome.;citation_pages=40-40;citation_volume=9;citation_year=2009;citation_pmid=19351409;citation_doi=10.1186/1471-2229-9-40" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=M. C. Moretzsohn;citation_author=E. G. Gouvea;citation_author=P. W. Inglis;citation_author=S. C. M. Leal-Bertioli;citation_author=J. F. M. Valls;citation_title=A study of the relationships of cultivated peanut (Arachis hypogaea) and its most closely related wild species using intron sequences and microsatellite markers.;citation_volume=111;citation_year=2013;citation_pmid=23131301;citation_doi=10.1093/aob/mcs237" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=S. Pande;citation_author=J. N. Rao;citation_title=Resistance of wild Arachis species to late leaf spot and rust in greenhouse trials.;citation_pages=851-855;citation_volume=85;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Robledo;citation_author=G. Lavia;citation_author=G. Seijo;citation_title=Species relations among wild Arachis species with the A genome as revealed by FISH mapping of rDNA loci and heterochromatin detection.;citation_pages=1295-1307;citation_volume=118;citation_year=2009;citation_issue=7;citation_pmid=19234686;citation_doi=10.1007/s00122-009-0981-x" />
<meta name="citation_reference" content="citation_journal_title=J. Phytopathol;citation_author=S. Savary;citation_author=P. V. Subba Rao;citation_author=J. C. Zadoksa;citation_title=Scale of reaction types of groundnut to Puccinia arachidis Speg;citation_pages=259-266;citation_volume=124;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=G. Seijo;citation_author=G. I. Lavia;citation_author=A. Fernandez;citation_author=A. Krapovickas;citation_author=D. A. Ducasse;citation_title=Genomic relationships between the cultivated peanut (Arachis hypogaea, Leguminosae) and its close relatives revealed by double GISH;citation_pages=1963-1971;citation_volume=94;citation_year=2007;citation_issue=12;citation_pmid=21636391;citation_doi=10.3732/ajb.94.12.1963" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=K. Shirasawa;citation_author=D. J. Bertioli;citation_author=M. C. Moretzsohn;citation_author=S. C. M. Leal-Bertioli;citation_author=R. K. Varshney;citation_title=An integrated consensus map of cultivated peanut and wild relatives revealsed structures of the A and B genomes of Arachis and divergences with other of the legume genomes.;citation_volume=20;citation_year=2013;citation_pmid=23315685;citation_doi=10.1093/dnares/dss042" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. E. Simpson;citation_author=J. L. Starr;citation_title=Registration of 'COAN' Peanut;citation_pages=918-918;citation_volume=41;citation_year=2001;citation_issue=3;citation_doi=10.2135/cropsci2001.413918x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. E. Simpson;citation_author=J. L. Starr;citation_author=G. T. Church;citation_author=M. D. Burow;citation_author=A. H. Paterson;citation_title=Registration of 'NemaTAM' Peanut;citation_pages=1561-1561;citation_volume=43;citation_year=2003;citation_issue=4;citation_doi=10.2135/cropsci2003.1561" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. T. Stalker;citation_author=R. E. Lynch;citation_title=Registration of Four Insect-Resistant Peanut Germplasm Lines.;citation_pages=313-314;citation_volume=42;citation_year=2002;citation_issue=1;citation_pmid=11756307" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=H. T. Stalker;citation_author=M. K. Beute;citation_author=B. B. Shew;citation_author=K. R. Barker;citation_title=Registration of Two Root-Knot Nematode-Resistant Peanut Germplasm Lines.;citation_pages=312-313;citation_volume=42;citation_year=2002;citation_issue=1;citation_pmid=11756306;citation_doi=10.2135/cropsci2002.0312" />
<meta name="citation_reference" content="citation_journal_title=Peanut Sci;citation_author=P. Subrahmanyam;citation_author=D. McDonald;citation_author=R. W. Gibbons;citation_author=S. N. Nigam;citation_author=D. J. Nevill;citation_title=Resistance to rust and late leafspot diseases in some genotypes of Arachis hypogaea.;citation_pages=6-10;citation_volume=9;citation_year=1982" />
<meta name="citation_reference" content="citation_journal_title=Econ. Bot.;citation_author=P. Subrahmanyam;citation_author=V. R. Rao;citation_author=D. McDonald;citation_author=J. P. Moss;citation_author=R. W. Gibbons;citation_title=Origins of resistance to rust and late leaf spot in peanut (Arachis hypogaea) Fabaceae.;citation_pages=444-455;citation_volume=43;citation_year=1989" />
<meta name="citation_reference" content="citation_title=Durability of Disease Resistance;citation_title=Origin and utilization of rust resistance in groundnut;citation_author=P. Subrahmanyam;citation_author=D. McDonald;citation_author=U. Reddy;citation_author=S. N. Nigam;citation_author=D. H. Smith;citation_pages=147-158;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Breed;citation_author=V. Sujay;citation_author=M. V. C. Gowda;citation_author=M. K. Pandey;citation_author=R. S. Bhat;citation_author=Y. P. Khedikar;citation_title=Quantitative trait locus analysis and construction of consensus genetic map for foliar disease resistance based on two recombinant inbred line populations in cultivated groundnut (Arachis hypogaea L.).;citation_pages=773-788;citation_volume=30;citation_year=2012;citation_issue=2;citation_pmid=22924018;citation_doi=10.1007/s11032-011-9661-z" />
<meta name="citation_reference" content="citation_author=The FAOSTAT System;citation_year=2012" />
<meta name="citation_reference" content="citation_title=Plant Diseases: Epidemics and Control;citation_author=J. E. Van der Plank;citation_year=1963" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. K. Varshney;citation_author=M. K. Pandey;citation_author=P. Janila;citation_author=S. N. Nigam;citation_author=H. Sudini;citation_title=Marker-assisted introgression of a QTL region to improve rust resistance in three elite and popular varieties of peanut (Arachis hypogaea L.).;citation_pages=1771-1781;citation_volume=127;citation_year=2014;citation_pmid=24927821" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_title=Windows QTL Cartographer 2.5.;citation_author=S. Wang;citation_author=C. J. Basten;citation_author=Z. B. Zeng;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=Z. B. Zeng;citation_title=Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.;citation_pages=10972-10976;citation_volume=90;citation_year=1993;citation_issue=23;citation_pmid=8248199;citation_doi=10.1073/pnas.90.23.10972" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. B. Zeng;citation_title=Precision mapping of quantitative trait loci.;citation_pages=1457-1468;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013918" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
<meta name="twitter:description" content="Rust is a major pathogen of the peanut crop. Development and adoption of rust-resistant cultivars is the most cost efficient and effective way to control the spread of the disease and reduce yield losses. Some cultivated peanut germplasm accessions have a degree of resistance, but the secondary gene pool is a source of much stronger resistance alleles. Wild species, however, have undesirable agronomic traits that are a disincentive to their use in breeding. The identification of genomic regions that harbor disease resistance in wild species is the first step in the implementation of marker-assisted selection that can speed the introgression of wild disease resistances and the elimination of linkage drag. In this work, we identify genome regions that control different components of rust resistance in a recombinant inbred line population developed from a cross between two Arachis species, the susceptible most probable B genome ancestor of cultivated peanut, Arachis ipaënsis , and an accession of its closest relative, Arachis magna , which is resistant to rust. Quantitative trait loci for several components of resistance were placed in the same position on linkage group B08. Single-nucleotide polymorphism Kompetitive allele-specific polymerase chain reaction markers for rust resistance region were designed and validated for marker function in both diploid and tetraploid contexts." />
<meta name="og-title" property="og:title" content="Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/7/1403" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Rust is a major pathogen of the peanut crop. Development and adoption of rust-resistant cultivars is the most cost efficient and effective way to control the spread of the disease and reduce yield losses. Some cultivated peanut germplasm accessions have a degree of resistance, but the secondary gene pool is a source of much stronger resistance alleles. Wild species, however, have undesirable agronomic traits that are a disincentive to their use in breeding. The identification of genomic regions that harbor disease resistance in wild species is the first step in the implementation of marker-assisted selection that can speed the introgression of wild disease resistances and the elimination of linkage drag. In this work, we identify genome regions that control different components of rust resistance in a recombinant inbred line population developed from a cross between two Arachis species, the susceptible most probable B genome ancestor of cultivated peanut, Arachis ipaënsis , and an accession of its closest relative, Arachis magna , which is resistant to rust. Quantitative trait loci for several components of resistance were placed in the same position on linkage group B08. Single-nucleotide polymorphism Kompetitive allele-specific polymerase chain reaction markers for rust resistance region were designed and validated for marker function in both diploid and tetraploid contexts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/7.cover-source.jpg" />
    <title>Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Hqykcn2XQLUcYZBvbYdTv9Lyi5TPLAhTMgyown77jzQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459990","apath":"\/ggg\/5\/7\/1403.atom","pisa":"ggg;5\/7\/1403","ac":{"\/ggg\/5\/7\/1403.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/7\/1403.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1403"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/7\/1403"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/7\/1403":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+gsdlqqfHe7TIGxv2luN0edwLwCWLZAkJwKYc3v0t1rZKxHoV5JFRV42YxxMfUXwEo\\\\\\\/5NOYTnVhuseEodSOXA3J+hc87M99ffDYEm0wtVtNgDp0cW4aHogmd0rhzg60EGxIsAtomzpahdWmKTo97gf\\\\\\\/KniqpzaTIWL+cnyyQCeai70p+uSmkTQEt6XI2Rlk+5oNpzVGCYOm8I+bO95CSnJT\\\\\\\/JruPQVg9j1+m6xsezE4x2VGbi\\\\\\\/49NfTMnCBCrIX4NMZCZWQwQbffYyix+lG+V9kaX0nOWWG6Nmg6U35OFPIHjjjJ1PwbtM6ha66RWNLT4fWxPh1IyDgLDvzqVbAQzyoNrNUXnodITxD3s1UiIk8NTPzCRMd0m8A88DejpOV0ZdVYf0XJ697iwpXwJeJf2a5upLulISipuEnoojSA1TfMTNeZWG+uuO+dTPzybqKoGnU3RP94Mozq5UGTaaZ1Wp++0\\\\\\\/cOBy1zzvKLPrzEVE5I6VuHP36TK4rLkG53ITLp+BkXgJRufVsLo85wvxdVhZ0wbo1YRRYRvuPoOIT4Ksn5fnRhLEU5qvM1HVVIpgKlwAYKWYNTvlxmI6eJLqd4A+YxYYfH3STaEocUoUmqE5xuH3fUhtZvHJ\\\\\\\/2NPUsFjXb2aj2DnAdI+1QeQW7m9fJMenKA5J8z+MKrauGZOSwIPNC1X\\\\\\\/OHaYUcF9yh5gruOMla9HAI6uWiJj1BhuRy\\\\\\\/o7RoufhC3k9mVJN7CXEfh0pdA2Szt\\\\\\\/kyaj2N7fjzTMZnTBBfqW7V3+Jxim1gNYzXuwaIo0FOnQ9AoovkUCxiiKgoYc6MagyTBclaoxSgY3ATmKQg1xr38WehNs2tRnhbLfzhh2eLWV\\\\\\\/a2zm0mppNlW8Qi0CS5kQnzQJwje1JyfGOgHwzqzRnGMCCxD08RukgSHH63W9PtCt\\\\\\\/LYKmhvDHRgMenVqevLD2yLyIq\\\\\\\/GcCB65q2\\\\\\\/rvHQuAa5j5t\\\\\\\/+Mz\\\\\\\/FHgM+hGPBILMGwDgzx9JeQ2RFEe8rBYRlQhLMQ5KGTbUS1kY5aETjskONFGmsO++V28fuUExxfMqTcrhpwoJxacfU98NT4ghOJOLLnXi4YMEGTHNQOhFfV4FAoPrkxQUAayO5ftWdvMQB2JXYdjefNj+NQW5Gzw6erBSfHsDkNESeSvV1dI8hCXqn7vqGeOI7swe8jSgGi6H9gmgrZgsVEzdILtt8kBQOhcKIVAcRgAwVALqR8tBsi7ydtW7zE\\\\\\\/a937C2OkteasXtU+tfi4a67iRmvg5+WIrB4f\\\\\\\/CtB4h+eQHuo\\\u0022,\\\u0022iv\\\u0022:\\\u00222fyvJCL\\\\\\\/4gbwwShUawqr7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228c36d37986a8c70abc7046bae6d732e4d39fc34150de2d01d3b1035ed76268b9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223DKafnadWwThtrF+SkyePXaSpYytkVyaKwFznz+ugwEtciY0je3FDlIxW0M+zLN5BgQDcXE9PWxAGVROjjcuhi8A\\\\\\\/1atP8LG2byBAB2cAEz+UmgImEXM3KeZubBUIke1+PrKMWNNV9QlcI7PuqrAsEFFEKz2LX+oy\\\\\\\/H3ASBJMCfVDnV1t4fO7q4v\\\\\\\/JLT8oVbVqj1NBd\\\\\\\/tF\\\\\\\/t2JvoFGHTtdJxAtmACPA2fTwdcFrr2dLi0cvWXUlqKsQnJx0UwCTX+xT2K5H7l+GXMqhCZSUD0XZo7uFij1KZ3T55dXWzzPfUr6yi3E\\\\\\\/nQfjcFTurK8YJgxbo\\\\\\\/IGPVau2PN8WKHrf3iW43yIo2Xdhs9YGZPNNdc0i5sfz8+ULaaO\\\\\\\/k\\\\\\\/T3n0daCh59yov0OndMyA+edp4RMUPyGiNyIxEnxFq\\\\\\\/mv5ifS3EtQkEL11evKVfRj2VWIbBpUAmvYsPy8534JUMbRygSiG\\\\\\\/VK5o0413z5w\\\\\\\/FZjJt6q55SJ4yNRFiKBImoQazh7vlai\\\\\\\/xIyb72FO3gZLsS93736QCXXO8+ewtKjFBp91FuLJjYZzb3\\\\\\\/js2c2FnPReueHcvDfeaoi1aLp+ocVZlJTuMpq1T8MqVUSB2Ik2DFJ6t9+ElaNfLmoHNzetDL+gwlQxklaKup3LgElX3cUdMhujnaOCpri3FcsOVKoFVAFCad4w0MclGJXIhAst9XZ9xJ1nyEko68Um1kapadzJcnSBZd6YLp7iMZ0Sq+PsJsqsf3OVylxM8OXeKdOFd4rjzopPbkrAkWXDYTJvpf5OIh\\\\\\\/aD9DmJUnLnAX4ZP\\\\\\\/XO1hPO3Ch+oDgYvybRjWFegOG3SjSzhMj39aLMTXSqP\\\\\\\/Aosfy9b6JROymBXBlPrb\\\\\\\/j+j45LQ8xTFYdiKCcPNqSlvb3SoxrejDaW4taNJkynrm7en+eEkt+a8qixyPkk1kVwQe13PhEKT14XMVV\\\\\\\/REj9tc9P5RylAIsatqw8J8RoOi5mCTFF+v\\\\\\\/TDbWYtDf+wmcoq0XEHVprb3SQzy7AamrZrzcDVQ7MoHi7zh5zI1mUXJUUIy\\\\\\\/SgxiLq+27JRK5qfTmMK4n1wfTobkXhkHrXzZ1kZ9ISzfIMlGHWYkTZ9DGt1ANsfqAd8QGLZJgo75tjE1fcDWyyxIA6fvKfpQeR277Tfzl9NMix+Phr0teLoMjlx4UMLC3pfu6y2qeeG0IJ5USh0ase1NjHNB6lNbRNoP7e5SWH46V46Fvgvweg1XOt64TKZtovUbChmaKcGY1KDf1w0yL8xjeUgIbYs49x9Q\\\\\\\/du9HzzgNKfx2PuEvHClqnDyyObIGMZIDOMS+3TBUo8kZGtH0G7JWXdFJshAIItvn5QoNbfllUuwmsRZOQhopJ63dpEl2Gq01j4GrgNslbbIZ3zpRNkx63SI5FlEnkuYLEgvF0tOcIs63gEykjJXocmMchfbWMkV8dTf8IBbRUW7ajOe2r6x\\\\\\\/ZxW+CFTq3EoOp3mx05wZSBVFfCUCpJgaWh1RRiWXwUPTwjWCPzh9ThQXzAMbZY3aYA+SvowcvfvH8w0OAPB8BkEmkiFTj2YNHIPToT7YPPf\\\\\\\/Lm+UZxS6S0+fwa5T9EAW15qL3qajkCoUWOeFTzUA7BqraS\\\\\\\/HTe4dnP22dNeq3zXGPWG4SII4gcYm\\\\\\\/JNbk1\\\\\\\/bTsHiR2pjN7nTTeBg8BZ39nPfu1CHJYFFQaAxJ+wog++hJ1XzWReCPq5lqklz5qKs2ZfM4RUVuv4Sc5Sd3gDO62MyuMYB9dNJJ2+nHBsv\\\\\\\/D8nioS9e9NtPFQDQn3+nwucFOEfjqKA3N90kcd5ZdHVBlhUl3cf6dCgtM7JEQHv16cU8vqS8tWyn\\\\\\\/uwW9ramyzbM7cQmqgMW1gX36eisqhnN\\\\\\\/Fqpn6dlseY1XSXeEE6hRXySR5EcGJAIUsQ+b6jEV21U4pUX0aFbqacgHrWmH9\\\\\\\/Z1oyIBXpIOkMqWdXVKNoSAEFXMENOc9YQ7ToMYBYV20o+sl8OKKyz3KK9CX64GdE3YuN3zLVmISbxI5rpcaMiG0+I\\\\\\\/sHn0\\\\\\\/T\\\\\\\/CK1BVQkqLuQhijlYViqqKRTP72xgA8\\\\\\\/+z1xSCS3X6AJzUuAIBKyJpa4aAROIRzMqYMlZ0O0TUl+iEd3kW\\\\\\\/I2zMBUV0Eol8Oqg8RD8nKHR8njFOFmk3Rjd2L\\\\\\\/9s7SazfkmZ2YQ5RrgcAu5UBQYQRAK13xcBrnHiREUUScIYr4VjUYWW6YckTIrJvAOIhHJ9S5LOeKpN\\\\\\\/Z7T\\\\\\\/YKJWHSD1gna3ztctb1jkj69OkMDrrO35b7aA10Hv07WM82Tr\\\\\\\/8tCSx7I99v\\\\\\\/FQn875CGFXRYUeeZ7VyNtVBtuyXgJLSFQc33ZN26pNlJad+PkEW5C01vv+x\\\\\\\/6Xzm+v\\\\\\\/K0iE7+412CNL0EMPQvHGH1exj8fTgvtm4lGobSe9+lfROfvqnyF4VNd1k2\\\\\\\/3H5eID2K1ylm8Dg7vJco9FA3gKvA87gKcJ98yc+\\\\\\\/16lLx7gFxNhk4q\\\\\\\/j8E5ei1lZDpoeydV0zWyBCB7qqvWMn5+b0S5QaUPVoTktYp49eGwJX5BOlwkFbRVSIfhicBznNcHS51lAeWLXKmazX5ZGjTe8lCL+bmETfqTK0in15fqMtWz\\\\\\\/hPSmJKb3+ZWxWcWAzweP0VlYHnEfqHVZDuGNlCodkov2OhNNC\\\\\\\/bGRNr\\\\\\\/DGD0b7GjWYjG8Dq5PejnyvVDs6tumiRvE9tm\\\\\\\/y\\\\\\\/yMFaPiqcrsIS4ThIp\\\\\\\/4epIt914IwwXxigOSHToHxgMJlecNI3cGO+lm+8O+ro7fVTYS94xFQoTaFuOY24KNtJTGKMT8TMp6rDWmc4ikfay4hY7yqQPUtIzarla+vI4c7hP5liW85Jz\\\\\\\/Wai+Hyi35CzG3x\\\\\\\/\\\\\\\/sjvye\\\\\\\/v0z\\\\\\\/quaVCwUkQlCkMQLDXQEZ8Fq68xCPF2E\\\\\\\/g4PoMQMSNmFb3J9anm34W7u0hHuR\\\\\\\/K5JxgO7e9x2sx+H0XOHELLYGaCvylJ\\\\\\\/ATsESblagPT6QWTsvCmHpAiVpFT1smr+gyPUuzwgrlnB4b3Qt5TjnefxTqbR8S\\\\\\\/5zV+NR1dkON42ALaveYo3\\\\\\\/A46vqqaDLx6Qp2V601gMxSFsRjHtWhMZSkyCuKMfxlLCUpkZSCMAj32ExObjqXRdwfj7c+Uji0hQwgRHrc6pXeO\\\\\\\/EZVTTTGpY26OUzTb61ezBnjwQrRmM6lMqHaVBrlE8wtbeFeUqz1syqAgKcN1hDvG5BUSCrN9QTTDAYnlYWNmVI55bdsM3J1jPuz4tNv7BIn3rHIuoosCwRl5eEuKHBzOvnTvvzAbHtZy4nuQTLfFUPtD+BQbYDnx2YFPvmxT1pf\\\\\\\/CJC9bGOM\\\\\\\/6EkoH2BRryUKyqtlhVmzb6ceCa+ixg4pLTeEmv5ZS\\\\\\\/1NII6XUeKnsc83TnJV7uKjw1NoIkDhdE3ZfRNwMwnHzYKxUB2i6PL1wGuQBXkfUS2giEk5jmfYh1WiUdb0u1f6HHE00+fIviTWTCp0zZjG6B5tfAXs0sAlW88hUdaDumLSbnzqpNtJd4qstG2OgKlJJZcGddG0jXkcudOIJ488xWfdIH7oPnZ4\\\\\\\/ItVgT4pciImBqYdRU3QDWXFCY7Tn9laqeD35HXpRZboChITlAysyuH7P6eGqUYWanfkUu2PPPqZVWc666Q98rCNQteMyfjwu\\\\\\\/yvwY2DBglkG4grO0BlPiQAv9g+JALlsGbfSKiC57H3Tq0dPzEngtVis5xOG\\\\\\\/lfwIteSajyiAnvziumI+vpRD47+b\\\\\\\/Ir49lY67X3rKMdENeYpRBLoIPj0Cgea0QMT+MRUpd\\\\\\\/UtjF8LRzKDJPWJi5UHhm0vkaZJfbpi+UlRhFWJllxQRkUT9GeZvce7G7aUCQsdRkYLD0H05wLX\\\\\\\/is0Ai+0I8zP\\\\\\\/d+1hd09FRIokocHtjL5iVSMWxgHYicr5XxSVxmQ\\\\\\\/G4nTQYE33KcpE0JXkAa9aQuvU4ItydRI9GshFEtonCE3jZfsajcWu\\\\\\\/AdPY+hqaSfWP\\\\\\\/JmZTefwx4dCzCugUBAkLUorm30jrebUPhqe91XMkj0tMIxetv0uZSDU1IUlCYef44yw0zgBhpO+vVuSC7Bx2F4WP+Z6E6qjaS+9PkA4IMtjMP6H0LDBE4dXN8MDC5DakQ6qLHp4vMZ9LiZUSbIR\\\\\\\/KWyvegs8TESeRRa\\\\\\\/xe3VGwdgEh\\\\\\\/3CyiNL1XwB9UJWYyz9lO\\\\\\\/oLbsuCckK5bnUOWiplHjU6wJPljXwEFJmKirqmfrYal4cGk6DDXDe6qTdrmHgmatSgIRzvTuZX1ISI\\\\\\\/NC0ENjRD3ahASgzf8UTI0P3TtdxtOeBHtcEtBshvD\\\\\\\/jy\\\\\\\/yBI6AggWMiYaUjaRe9h5l+rvnl8kJmf56b9yybYT5O2IfwlGLTvDZ8Glpi6vdmRTRmne4mvDQLpQ9oKASRaVC4aM+T06ty91KFCuI0aH+6MYSao9OrBGP8apgl0\\\\\\\/inMbD0ozSL+jEq6a\\\\\\\/7G9\\\\\\\/m1IXAUJ4RV0i3edXd1T69f0rQ3L6RwXtPc42y8z0sQkKfKR6Zn9Q\\\\\\\/iu1VW2KQImYxryMBEDC5TeISPnYFxzsv8ZWw5KTsoq3q2srer4WaLRUz0Ml9Y1b8vhOTrZDdpun7LDK1nuM7\\\\\\\/WKvT4\\\\\\\/Q5R15n1YqAWyFrYNUaWvCHW67E82JpKO9CBKg++WwcTFqRqpoDu80u7pGfpb73SeX1TEvkh712T+ktPKTtrjs5OATF+EkJcWlA\\\\\\\/Ly27daZkGQU3HynPznxx0yoaJNFueYOnwxXhGFMp96gu\\\\\\\/8L7oQUCi3x7NDDpN9PgpZiqYCE6cH9PdwyER607whRbLC3uAXZociF5yDjKq8qA\\\\\\\/mTWvOZmFtQlhwLLRewqoVKRgoLtQq\\\\\\\/22G2kVQAbi3B1XwxzUNE\\\\\\\/kMt0ErvyeYVMnDq5htL0WklpxfkS0qAZPYarRbu3FTR6pi6nXVgP4\\\\\\\/GX8KhozZt9B\\\\\\\/UAAtrW4G1dbRGCAJawa6GzE16+lZBOOfjfoYZE+j64miC79frgwYu96\\\\\\\/M+jFz5dvxJjQ10lWY9ek6iBKXahZiPF3kUbK5GcbtY8XYaotp7PKOC7rHsymKeFqeEI53LELRfJM+MiZgopdVZqiYWShZTcshc2zW2lUdPQMXYLb4qLOhtRRELZvTCgKSMAWcYmhMrsfUunEF4RzdymPM+K4BLL7F+URGSCIFkX6ouv3InJkP5737wmreMMj8IakWK1+KrqPfNyIRYhWXyzSpNmLI2kxuiTgn+zrYPsg5OhOF77udCI9z1nQbT0nj4CMUJHK9SoYaRpku8x9TUTD\\\\\\\/mIQ+O2u3SHuwl8gdNGgOocbn0u0KeOQRVfuRfnRx3dBoXfSLPLdb6Vesd56fsGxgam\\\\\\\/uy6xtAErh3WB9IQw9yqwBK08NE432GmmireL5utSvgoP+JBTPZTJucfJLE0vk8+4CjZjGsnJ5pQ9f8v4uaLYE+QRSDd8Ya5kI5saUi17wRRABfraW2R0c7k8r9yL\\\\\\\/Kq5uPUYGCx1pa2j1yMAPBhiselpnHhawHg6m5J4S+wTdHeov1lJco9CnKyodIfmSZ8RFZvBC0PvF3JB2kfqNOgaa6NeWsrP6IzTZo56ttXBcZFNMZfRWSTFgXHEE\\\\\\\/vMvP26KZee0Wp9j2r8EgRvEK2NJ1XvA3k3tXKS\\\\\\\/yClLo0O+3K8+b64KCgfB\\\\\\\/NwZ9Op8HfS\\\\\\\/s8b8balysltHzMCAIDSmwcSXrUqvPH5mE6IVB\\\\\\\/t+hQHvaUuCdht+dPQ\\\\\\\/9rNWxCsBpBnynG0D1uQUwLDdCgZExBPmtcaX8dBTfVKmt9IvvYMh6nvpcWS0fjND9IAFxMkv31pYT6VEQwCKE50ccM62WdKMbWWbE+qo6XyqWiX4bd59E4krkTiSfoSt3s3DKsrmWD2fjGMv\\\\\\\/HU3f8ZNPLjEfE4\\\\\\\/wm10t+g1zefby0ZOXkCZkrxEhcLoOvDDvvaeWvPzsALQXIOmRseqlB0hh\\\\\\\/kF+HWpw4TVsKMzC4DJt9OkljGri0hT\\\\\\\/yPVBaEa24p66\\\\\\\/GecVlXCqOTIDnUuAwKICeHZc7mSnuZMS0q7RFSEUh47Yh\\\\\\\/UHBlyBFhEXsLOPlgZPwP76G0oe6N2xacr\\\\\\\/HDARAuwmwwfciDs3Nn99V5Ryoxh2FvEDvNg5psKvaeKhvHKXnyFZPTUSoy6qcHwzl5DsK0zUhRDwwI\\\\\\\/UkbBr+W+6GQBK3l3Rw4kf6H3uAYKSdzVQtURfEoPDm\\\\\\\/qCtew9DYk66SRzRdFlgRobHgg7\\\\\\\/Im8oaiP3MeuSSaxQeTX8clpyZRa9imUQFpiHJwLbaBeq5ItKaHaUwXwpGkkEWvM3FH1i24kwXM0NpP0uK8wWReAr27tmEuPdr+4TWJYOItQLQ8aNr8qUnr6Ffp02olO\\\\\\\/6aRWaFA02WsO+dZfAXnrfcK9FE4xWWyoo2NcsvQ\\\\\\\/DOgJ8ZtRxwi6wU4Tew4RHsHwedYzqxrigIWCwapW5rlmuLmFLoNYhpZ4FKMcETVBboNRm2Gn3ol\\\\\\\/XsZxYLj5P6z1woDnAzjLOD6MmZ+dOaH9x\\\\\\\/5Vq3wHWz1gW8XjFGFR2R\\\\\\\/ecLxLIBF5PT\\\\\\\/SAtX02qUX\\\\\\\/LzkoxkX1teU4GOpt8Ra5PxHfgINyL8OxbARpqKQJfw3BVbsW9b0M6TCrB3le7oDNuS4bSEo\\\\\\\/FJffDDtTHapKnSC2\\\\\\\/nxuoWH\\\\\\\/H+aes+ppi01u\\\\\\\/PqLyzobW88mq4EGbelKPlnREzUITTK7daWiBWuYRXFzHow3fz4QNVsu\\\\\\\/uHLKUo28t0AKQuFDpNEM5piVaGTQfBWox7elsJDKe\\\\\\\/+T9W5hs1XbnVgsbpfWDzq7QbFnk7S55I2PAX6ZBm0JSfcsd1y0NyhBZe74q3GgiNdewrAmKLqqoLLol3hwtGvzBuyaucIKtJNEOVMOnR\\\\\\\/lelgAfsWrnkNHW5GZKFSFaUx+gx58dP\\\\\\\/gsv5oT607P0IpNy5hwuflACf4Y4nky5XcM58mnors7x5kuUNXIKnB6R+\\\\\\\/QbcpkzdL\\\\\\\/\\\\\\\/88sRtai\\\\\\\/gD0JSqxuBhKr6xlv9tT2qRcqRlBwSH1bVIk3eZtwRsMpjEKOwGx37ZUENawJpU3OsbLOlqYNmOtj0urKofppnqzIGamrGQXuO8RdDtLSo\\\\\\\/ptQt\\\\\\\/CimOrtyRliG6Oty5M8ZryIffCqLIZAkzjiBGIzbu070WvEzE4sfqIf438CtS3bp2tbCVeivfpqtorEfJuMZxZJ8GBjnihiWkWQVmJgsAMQxb0tfVP\\\\\\\/8hTkj+nJWs9LWzxDXX0+Bu9b5h+84S9tLItnYu+RcIWORfs1CXiEwJZBsSbdKy85pH3ZNBlGjYiUAgHLTP+rM5vLCD2Av\\\\\\\/nHHooG+9icUcEySocSy97aYbd\\\\\\\/5k09BeAJvEPQnMZCHB2bTuRYcWIbGV+nqoRFk8N0gHqjx8ATWsf8\\\\\\\/rEQghsULoueX\\\\\\\/C95cpL\\\\\\\/d4pGUUNVKNyXJz8aZ8LG\\\\\\\/1aPCiTRTjnu0HOOHPtBlsL3fEhitKg3kUGNrrwY7XTCXowPBvd1028O84DnPpFEXsFJjnQ5w12rqChOXsl5IBG5xiWtw4sWL2n8JoGKRkEbb2DImt9VUMjSofpqfwXIRsnZqk\\\\\\\/d3DSVp0F0Tz+XCAYuMvnHEko39U5NYiPgQc0k00pKNGYFR+wwK9UHMcAOsbS6TWrdG9dRFQ9rFaRXQdjYV7h72aZZd\\\\\\\/yITG3iCGTrdzs9GFyJ9W7Gh8AjCJLmUzJekVvGa53XF0\\\\\\\/FboXsPPqjWoboWn1dIdh9kD+PIW8yP42caVpht+WfQdpsFvjgcQfh0FLy73mDEm+Y26FSm6bgWPzk\\\\\\\/XmoxDocsivnywy8cZymU\\\\\\\/snfjPQGC18nqRCFj\\\\\\\/mAMKNKmJO7AoOqYYF++9GSUuvQ5wmdnoWKFx33wRsMR8v05+65EYdrM+pMSEE4+zq7rFDcWP\\\\\\\/zjpVeil9ALxUu0dBZUXTXC33AfOn7FO1ZqqaVyggUWFMObCx\\\\\\\/MO9H4qFIbDkgZId0GP2SPmRmnvGXaHUiK5Sr4PYT4MOKwRpRMRBl1Kkto\\\\\\\/2CgvU+18twstOjrHpO3tSdO\\\\\\\/BqmephFBeitLOQb\\\\\\\/+Kl7+VW2a0tXCfWKTKgSv4cLa4YZP07ce9k1Rm5PyRiC2TKt8iYXDKU\\\\\\\/xgIdsRE1PeOnmkhNcIAN88s\\\\\\\/CITIegVdIDFJ25pXOGMO5CKmbvns3u4Vs5Kx85R9PpfqwqM3YTq+YaISORJHjiLUMubXn0uYQV1TUlxmrpItQGOYoXH70QSvqW7\\\\\\\/YqAZD7YGsmM\\\\\\\/WiA5THqLDXJzoTtoNw3xy1u6juu2ELnnF5pI8yJXWTpZfRPSgukw40qzr3+gzjfqDOFEOFb+9Qmtu+tQuSWuxSR6TvISDK8dlrP6zbTrXKUzFwFFZYZR7uk8ZJM3b45+Hi7RBexqVa3EvxjpElt2IA4p1hqGybQS7NRMKLNJA4VZB81KlFomP80PZ3gJ8VOII6jqFFIi66RYn0IS41Gdwoi9JCT4ICvAVNSjie9KNlepuK1szwWvpGraH3muq0mkT+zdCIjkQlXxPIb7MnKnRk2oRp4BuMycI8Wv6wVIHJMKZriNs\\\\\\\/AnI2KUcT7XvMn8AhdGTCoSC7jgizispmdtTm7zj7orvMVNjENVLpEFwDYwc5WW+8knSv+mPUSeKIE4G0EoL81v8ROPi9kyWuJzKCsjLThkpaNL8v6ty50sps50S+aJqlQpkZsxEjjIfjqFwisr0YguKHufX5soH+rfhD930164QSMAql\\\\\\\/kEwIaPtJwH+b79+nx9FOycDhlhP74Las9vx5fXBm\\\\\\\/Il7oDBmStqp9h73SmnhSPqr7DI4gEbXmTLyKaJFyuhaSTvfHHpzROcQ8QHCE89ikkxjaX3Nu+WyoD9ARgvOPamOzyP6At0tRxa+89+71xsZX\\\\\\\/G3vsjD\\\\\\\/M8wIWUL6AJ4LVoCaV4\\\\\\\/dKpebuYTuItlmzzpeWiU7nQceKbXwdtXE9RtY1XHWvkij3v+7wUhNbF4p7fAyHvripU2MuH+zJcVzBsdYtHkIk36fHPInsnvKTurccaiGDTS\\\\\\\/HG24m7NBXHglESmATng2JObA7BdWTSisR1byCcsH0q1Lq38hz4kjKQx4GPbQOvxL6xLYI6AsT94UWmFgpTGgEiiCoa+p2+MeLcOuC7uSEdPDjawZ0IozjDNhpHK07d53xmuU\\\\\\\/d5UE0wFP6SbGA7EyCPIyEX1jH07o5vEU6PTv9fPPS0+IXlI4Eero1B3b8oMd6HPpZPK\\\\\\\/RbuqEHaWlETCJCXdsXAGUloQ5Yjxl5V6445BDRogmD7COiho+KsQDlV1PT36UeQwtDA9zAQSPosYQmTtth\\\\\\\/DSguGbtR0KaAqdXit6wpozN6+l1nqOnyDNqSpDxSLNx9VasQ5crulCCHvBLc5xcJLRAu3LoWMC8XqSoTvJnKAN0cc2aaeddecHtcme5yQYMMeV9r3cOvxGfLkH6YEUdwOTc3Y0XUxtcesycYjxSSGZCF2EhTwf3tccwkRtn+mUM\\\\\\\/IrGZZMaNoFkWBwqyqG1FOeolz6t\\\\\\\/iJFRmwfq7wE7p1VDJcRXU3df1QhRlEIN0sfyTosz3bba41k7FHjuYO5dVb4K1jWWfyj6w0LwK8xtzh5\\\\\\\/sct9rfOZmlUqF8+y\\\\\\\/igziCwoKveZWszczod\\\\\\\/yZjuZ4hbz7w15FvTlUyGI7VejeiQN6seFNAY+cWbGu1E5p8oSu5Ymmc9tbFk7Qmi+UY596kWb9VJ3qsmOCWeKjLRf8kmG3PEKnYq5FtcTOVBLnpOa0dbtYBegp3U+AoHTdmUp6sVbXAotR2qxg6ztC9wKCBC9ehApw9v\\\\\\\/Az+4QWDbHKrtAoPz3nO6rSYKfdOCWSZPHeOw1dF4JbggIbE\\\\\\\/P9PtosKvJgszBSIjrrSyTzdakSvTCeCCforshppCtqBwzQqUTtiTmNdBjZqu+BofjjVGjhhoRBGGS+Eh\\\\\\\/MMzUkFrh42yFwLWTU7\\\\\\\/fVmeHXwd4OduGJSky4Lo+\\\\\\\/A12dDdLbpsJGMe1VheSsYrUDhn\\\\\\\/UhvbxLee2I\\\\\\\/3iJ1Yt4WrAcqt1KVCfBi+3HeWcYZFiEO9Ul3Nyu9fzzk+bo60jE+8A64dCq2UWMPvBMcKqm6ERHEsb3mHMumFRcME6\\\\\\\/qBA4V3D6u4Le1ReGK9Mj6DMmECzc4dhNH80T4jjoq3vpghNUQKjn4\\\\\\\/HSMT5YnJjg5GCAnhMaPKhYHunJHfhRleBCmmKM\\\\\\\/mljBhAvb9cVscaEFfWfFkNQCeHeHymrWJe+GAoInjQqEtE1Azsy1\\\\\\\/9SXZZ8UGv5oRUGQoslG5ee9uTJyCjcOyKbNasetXyCeZexQm5bOmlsh6DnLXOWch2KzNGFbE8m0Fkw2xiTxJvrERiU7aqIjg7iwK3LssN3sy1Sf3UfgG50zLdfwP3JiwURNjumT6XYfoERrAiYCT0feyH7NK4R2kzyUwrgCt3OkFgCqlhXoP8bksoFCianuncfouWSI0CSync5eh5NuVPzbV0pI6ir37nkujN+qj5PF5DqvnYYS9ull0o7t+mH\\\\\\\/A5IajSYrhDLNCC56c2Eo6JMzP3DfvcrttWeGQLFAKov6VmLlbD\\\\\\\/RhBeuiZJJFx+jLuu5+Ipigf6kIo5QkJoQw0YmoPUJyeKbyZLPVdX5G0eYUEta1xOdfoqjdfoK6SSr3RMe+maOVw0L\\\\\\\/eFBKL9hlQnWIKYoBZuBrrgVwgyQn7rTwJzIip3VBg6HOkPzooSSqXA7EmY9mZ7dQWKijoFKud896KrGbT3vM+KR+RKhm29vLjrg2nL7VfwjXJrEQTokgQlQ3atd\\\\\\\/y4w\\\\\\\/mwD\\\\\\\/Oe+cRXZxoaahSPasPwKXALjqRZ9nwR0nYgGo5WNWrkR+qxuXzcP0Tyl10O0w+G3otuQJkpKCE7AbGSZMXKuk06a2Ql3LEPUswkqOtXoeGC7b5P3nf3I4FhiGTtBC1+dCMz\\\\\\\/f5QiijRC4nTBdRFw5rC2BfWzgSoIidioymldR4lTKXg1y2qh413ryloKmg+XVx6LHNAvaIPpRpSlmAWJvt1L7AdJ5UFvjyVg8V5bxf54rcU8ZXhHNPS6vUDTmemdmzXnn0Fq3oS0TEoopyvfBSSohUU3FhSjiJ8t4vaxzNkGTTjoxGY7kIGK8wi9jya6yuu8t5QT\\\\\\\/gaddPgfs1Dx6oXWJeaDoGy5YpL1eILKAYPZkatkeIcnEGmLVVbSexLim3E6HkQXHRqg9iL7Yg3+YjVAPfc+\\\\\\\/nk6DsKhyNtpR2Zwe1lZ2Pj\\\\\\\/my3SERDi46P+jKRcTgbjrPkOSfXQWLXj2GcqI1CdzxZipTE1iVal11KLn6rjhzPH\\\\\\\/si9zfzSoAb8hrXEqBy9VG0Oqte4neAShuO42x1OmelsGFL1fx7lcV1ODseGlMjfk99vnL\\\\\\\/6gVHGk8RiEmuwLYRiFKIFCbqzmkdc30ap2C+1Qc0DBp7obpOJU5mQxVeUhxeoc3Vl\\\\\\\/oq8FzKCz1IKOuUzkGTy1o1Nso0+VDMOmOVGNTD2ajbMJTR80hYgwxIOy8CU5Axd33PZuaM4PjmKnzNdoOHv28VojJJUNR5C\\\\\\\/IM4mTjgcZB2yYuINmNArfqa8F0ClqKmb0eWip2Y7\\\\\\\/Z4wHlJtt+4XQXYaJ3A+JmO0WIEZwLoY8cwJ2P1oqizNBVAtjE6x4IN1vK\\\\\\\/vwUGhb7JOCxd+ZrFk+pvHrMwFBB1dDqv8ZZvB\\\\\\\/xbaPdIppE4AmdsgfFQAlL9gyFnuKLY3xD6FKta0KnHozuuaXPmB8bMZNF8tJOSMB2nmQb07M9WJjblMmAaCJ\\\\\\\/lu8NEOysW26M+ZwylWdkJ1BLkBBNQS3bM2TErF2wabr0DJmAB3tM3YOV9m0Vv2v9rJqMMBuld\\\\\\\/jDjKw165g021NjgW3lGTeP05XFjxeMCrPGe1udC\\\\\\\/U5RHGU3c2UgNZ+3AOB6\\\\\\\/3lNmTLdO88NfSeb\\\\\\\/UahIEmcAYPEjDPdZ5HbWpE7AOB+RqrpLS9gjbTm9J2zLav+oiYqmQ89kUUB5e5H2b8VTec0u6d+O89ketOtVPuSg2SBRd26hqCkC6XqRvQAP+2hRII5u3BKw04cCRICfZNwtslah7uUdcyz6Gp7QPX0pVQA2Md0LBcNfRAHAwfJhZrJJ8wR7fc4fg88JiHu1nZ6M9p+adX0tjtPOcWOkyd452sGRj0VORJyHK1XiJV1gtbZZb95cUTe32QA1g18gBwmB+kv0Mh1oWBYYibTTl24bGHdGM0zzPMJgVwNEvNf4JpOaY+cV3jnalU7XWDDnXJ6ypDMMlRPbwLh2kO0vRN8HmPBtuntZ3QIiZ75oliVCwdY9m\\\\\\\/yOG5LM0sWToorMM3SqVPwJkbUW6y4MuIU1eiv1245fPnIIr9WMmu6ZkswqNAA4JmzlnZggUGqpx5WnVK9436t+lbMWjA44J1HosDLA46+6cggkkliyxLSFnCvzH+\\\\\\\/RazwzrUyuqxRNah7pwZPreU2A7SpujcrjIgQWdeuRd\\\\\\\/wEaGf58slIM73Uq1j+2QS1\\\\\\\/s6PZvLRntsmZfuqHvV91RQ4TAsvwOm8Ku2WQTgubBV4ICzI\\\\\\\/X\\\\\\\/WS5flrVBW21s6+mKU6G0ETwb54WdMdx41DLlOpM1Ni4aSLLA3Q7tQ6PqUFGpT\\\\\\\/QZfgbTlkkYQJAjyWNFUN3qkihVnoT4HPN4sErfkjmoLoPnlEnJ+X\\\\\\\/qM4lFgqXwIg22xMKPzme7MvI+YjdyT2V7UEiyt4rUWCQkb6bTMSTGC0gamNFxz3Ng7SZOqoDoeAXNlbFS4E5dei33J5cmESUy3vvWDQvOchsXHCEdeplnE4SYMEdZaflI7XDc8CZRmwRrPazfU7Fo72\\\\\\\/XJjUPPZqFngqsPIFIK6o2aiu0OIjP7mmP57GhiVVuxfX0HfD405XOJ8x1FLMLzvxvkvJtXLiKTMJBLQFpf0UZVpP\\\\\\\/yElzGpMC0OyWp0JCrEgHJDjunaRVSkHoHpzmR2PRucXD2SKXwAyM9vymDNjBTSGaOjyPmP+wT2wXHm4oXyi7PMlFFZUsucdAZs4HA5NGNsge2sOoIcjjzT4jJ1nYSBm8C321wrUsu\\\\\\\/ZkQuCyCgL3fXgkfJRnBsfUlHLUAb+JgKudmPoXkDXMfpGXgkG9FGcT7AQ6bczy9Ye9aspdQRpQUZs9nTEunPbJ7rYMjNgRUnbaTGMXwH0Eqh2lKwRXI1Al5XGua0iVJT8SvB0MnqOgKlH9rwg8LAhd\\\\\\\/\\\\\\\/Kt61IkWKchzChpedYXwdL05t5zP6cPrsNgv0WihKKI1Dn5hlpr8Ad2GS35myBBEwsGy3Gfikcx1aLFBh7p9jtlVtiWO731oZo\\\\\\\/4vlwaCGfZvTnY3mg7hRZZLBXG6CqaqHKurnPZyaealaBOos2bsadkiKJDKDibA1tjrH1CHZTAW6esd2k+t\\\\\\\/ZN0FH\\\\\\\/XzuNwzi2w8tYAyr4ilooRFe0ePmVT5kNbIYh+sHvP3rdAaumIbEASaQRcDZGdk1mF07Y1w+1hs+gW0Ns5fzRQOQbjepjn7qQCe5RLTmesZtdZY9hURKW0eYtKESpPuxqJlwOkXPPTn1a\\\\\\\/5NAX9mcECS1YiIbVxcK0OUJLzLIpvMS5ESkq0UTg5pe79\\\\\\\/AcjRXW64mTV8BNvQ5+Xp1YGHv31DG1TZYnY4ZWAWyHFPlz7asADVSefZ\\\\\\\/mkR5NH91PfpxHnpbfl5\\\\\\\/vkNKZGjUpn+KqywNPR\\\\\\\/rS5hHqSarnuyubqcV00q6uk\\\\\\\/jfZBUojkAIszM9K9pMgiEcxv\\\\\\\/mnqKo+LkVFdHzN65WPQZlwtoUUr5yZbSMtosIXuk4Sb9hIqlvcXgcmlYzRAEAZFpDWyd+yZwlNlqOZY8drPOXnQKzF6FSqLslBL7y8va8kzXN6k7ZjHCHU\\\\\\\/cebY8ugOmd148iTNPo3+Hxpyvks1IFeSb4JDT589EnK5uHe+DlylZXhOjhF0+a\\\\\\\/Sku2MdvNrYgTo3\\\\\\\/geMON7aDzCz1O\\\\\\\/YznASFa\\\\\\\/1GJ3kqaQc7lqAt02Xt1pwXOc+ADPUsIf7EyVG\\\\\\\/36Mv+4o7foWlHjC4jzuFRrvtBSZroANPE5gnpspsKE9m2SjocHE3ueB1fE0rJ1lulnbqCIYqPCQZeNouyFoB09VA4iHSMl6QaKKLozLtfsnej5oxlMmEn1ivjxnzW6fHGs3vY9kKlN5lLDrvpuEKNvlOyNVNhfnpUxqXVE2eQ0CfNfiP+4XrNB\\\\\\\/pQvqmRFx2KlzvfZ2gGcKciNP9Jlc9yVbShYSwZTnzyCB6sZ0L\\\\\\\/2YpcslxTOu5kKGCDRMp23eHUmp+M6iv41WKdjU4Y\\\\\\\/S+HiNOqLDQEXUEuLxpfqGAC\\\\\\\/H\\\\\\\/9MT+En7j1XDuNaiPlpFLrMd3ajiwmlvQXMYHmuG5bmIKR+58HP+tmop4Fn7SMQ3RetMC9rfv0tnYqhsssM0pMV04eF29GdiV4omciwIwA16Hxvrg0Gj1Mb1KMOIo7llCst8D0sebKOkpzFATIp8oCA26IDF5YESSCd+7cy5EYpjJ871k5379oQGxljQ9hkdgNbuuO61ZxW61TmjE067GT4Bhg+CkxaFBnU5+Z12lz+vFxbcyroLMAwV\\\\\\\/wYlliOqCFqXye1HA\\\\\\\/NTd6Nl7fxl4ReXRFlXUhXTg9HzhV4C7hNxJO6pRNjUFzgNRatJ+rxDDBFj2uAUliCc+ZTY79NwchSdOYiON1pv\\\\\\\/SYcUgcoko3\\\\\\\/2wzN8h3LoLUMqMD8+oDkDrHtxXVpoLMSFC2I+V\\\\\\\/4rV4BIYKbaoQl1F1uwU1Ic+lTfw8ZeJmZRcTFSE5271wEsn2\\\\\\\/sH7yHc7vNYzyjOQWYAafjxgInFCsLuPbBOnUmYOauMPvs06IE16LWExQObCsFcldg2L\\\\\\\/+6VlYcEBib4J8DZMlx2XgndSXNHwu084+lJy+JxK0cBf5EMEnr3+sOKCfkpY6chdCCmg0mQEA\\\\\\\/1iLJm3Ss2jxyHoO9F3Ol3K\\\\\\\/2ZpszzLHZzMJqz4fSwATENubtc1VEc2TOvVrtZCcg5gdcSpyJ+rV6TQCvY8qCJVa8ChvMNz1tlbNnxJeyuGlRMidkpolHGcwLdYNrQP3AeyO3JIBL4gUz4dutVEE1q2DH2I2Jv6vZd98kbpRsBNvmCAQ4rC3sEsJspJQYxCQgN+LK7Et9ymCUUh7FCitHLuUpjtByDnW45JF2vTZ+bK12GYNegZk2UmS33WRPVf6Tb66VrtCS7ImDQsr+XzL2xWgUw3xJGsg23CftAewKzVfslNqmk7u8ZCQsXd3N8AanQO7\\\\\\\/C5f49SrGga1ssbThrG+cJMGSnXciPWfyzYDNObLTiAlHRy03Aq45VGFzuX0fVtNxB+93ijZqKTJYb57eH6OzP1RX4XfJABnU97K5ERaH44kNM6mOTfHWmC7Ojbe1vSadPbwcSmCxP331k+JNQK2QM2gXZQYz9Oa2SLinG3sO109g3YqwjDNHWJecpGfXbSzjz88PMnyfsSJlH1QbMmDpxAm41IFra31kZu3kCABM4bfWWe4aeRLP0n8nsXJ3xDB0PDVy53fp1JEj6If3sgPU0j4xmgOVorTxmZhb5AG4q+6CDXxwrFGgnVBwMB\\\\\\\/TP\\\\\\\/FU4Kn4V3OlKoAPXGVwW1ypg+vRFAnLCF9fTcCXwXne91xp5AP8UBDhQMQgDBDdIG+KC2R3fvUIkrZXQftZ04KjF2rDtXbipgOW\\\\\\\/9lGaIFEuismFdFR8XIL4wPVAJeSmiTSTDIxR8HWMAD1dxFOZcMEfwECgLqm0SJygCpcYwVTcraikqTxgooh5anBJIcctMGtqZ9tWhyN0gsVMBImYaV20jCyrg3OoYkOSxIYgs4iAB9mzUYSNW41o5DqFfTuz8TgLDIR8MUZz87ljBykHCknQK9ncDRsrt+7EaBSkbhWDvHFTqn2h5akyA1Sv+1igq+rguQ\\\\\\\/eNlzxi77tlpDULjrXeKLhCK4AYbCKvKhosE5ODudrveNG0lNIrsljJjU\\\\\\\/hprBWBiwMJWPFe1d1VlD5q\\\\\\\/iFV0hBBLHd2A4cGJSqfjINpTZYio+Is1n30nHGTm9Ygry4K5K7fYho7laTRLP62jsTXvT0ETVZjw5M\\\\\\\/Z8Llem3BcFsJCSx34wh4ijbhIr5WqoWjVKCewli0jONCBluEWtp3xQseI1lOO3bgV1bNXRUGl\\\\\\\/Dkg\\\\\\\/YnY8uYZAqq4eIujZGvwf7PXW\\\\\\\/3pCjL0FBYKV7cOxKMawiRZ1FxOJHAU7Oz\\\\\\\/P+FmSIyrj0aPUFw40EfgsRUQDAJsLelTwEcBRmrw98d6xOuEDRq+nGsvhvYjTRWc9gdiE1FH+1Q2V1PfHiCuqnO52l0HvCLoy2UKpyY92iunsBReNfY2OWmN+69LyBazSlp7zUBDnmwprTcWp1bSU31As0pGCvFqOdxbtt7k3IIvR2enRrYgwGEQZzRdi5EhWJq7xbGBxkuzZIQY\\\\\\\/2hskDv1lGeFpfH5bqYOxX82NNUBgvAIdD8tdFC8Ey8KnbfWJXDGbbxhC74At6y\\\\\\\/atiP8y8OuKJHIipdsyS5JBccsXKj2T\\\\\\\/0lCpA49Il+f8wBMzqNbsMmGhBUDOSz\\\\\\\/s\\\\\\\/TGdMUfvRppa\\\\\\\/FNehMgGC0NXgUAEXkvM22C5HdGvooR63rMePHZfkabQdES9QpJyvt67YHGzYJFhwwzkmdNMzy8q4q4ExVzPL1pKsL\\\\\\\/a7T5xW\\\\\\\/PUcVN3WtvUiU3jOXljyTNPD0bXzACbedjeXzQxdSfW4K9pv7evfFs3e0eOCb2ORAymidogfLZRURhbfZ8srHyCgYASgXs4no9z2aKvJmvcQKIwRBt4YRiVMGSRNIEJABcN5ia5Kp\\\\\\\/av7mR5yYkETPj6UHCqbOa2W0GkS2CyNxQQRJB7d9yrMuDzTMytM5BMzJ0ckW3UyZubVQ66NE55iapTXN2ViZL\\\\\\\/DMEc4Mr57HDFlGTJNRg2oV5REnyAQGRCiWY6r9tC2VgByuvUHkyfDxcORtV5mByry0IwPnoq0O1FIWKg1SAXwiExp9AQ4cxOA4JHGF0F3XXjVzvZQo8k3vONua25gVgUxC1UgNBv+5zABCYVfHzl\\\\\\\/0HJdcQuQMejeoTc1KE\\\\\\\/NioiQ1w8DZyGsrXUTOPs7mEz8kEZ1K1mixRxweEPCj4lYOtCKMA2AOC6Z2Fg1noB\\\\\\\/pdEfhwchKgLxP\\\\\\\/38yeXvKJSKcUEWjyBP3D892Ue92Fu\\\\\\\/op++XIagshtuygt2bzpM32B85cTUaHJKpfuIGVqoT4PjAoDJ5oDcWm184ZPlnNmxKR8jpu3FURUmnfFGDrQrdvsc1OFmhl\\\\\\\/btTG3gtfZGusoLwCCePP3in2JErLIKo+96mCDxXP9\\\\\\\/a1xj6zo40k5aV9ESsbYyUEwct4GnIvI9T+UDzJfjhNQtjSZ0MDSF+qanbuInQyflTNlF+22weMwSdIeLjgG8VccLRQFwaKplNuXUcfn\\\\\\\/pOhcRtTsW9hNiHY+VJz7xnAQ9LL8uQuk4YPO8tRuUTTHnxSE2P2A6uAlhdkOsdkcPuPu\\\\\\\/9NkyqQxBXgABjfSZvErIYQwdO\\\\\\\/Y3M5tTdHDnDz4wwzjbc11k7dSkFUEfPq1qEntNlWgQzkqUPiDn0S4DKt8FAORoI9QY8j8o\\\\\\\/C4tlcRajIf0eievh9rr4Qwo\\\\\\\/J0gf8wbKkoq7wGI1tZAup+V9rxJE\\\\\\\/wcY0HshBBD3zPHz6W3PKnbB6aNAtMQPEtEkbEqlbEAO7SWOY9+ljoe7AfsqVFb3Y3\\\\\\\/XDyNcOMb6dPtBngAFG3YIlkaoqdaIxIS3H733FF34HYMGfOgtn4qnb9XY8xLgx7Grnic+91inn7ko1UfUJ2xSgFR8UcVOJwS4Sn3yt+v9PXGjjzbAg0mccyCxRNBtPavsnVzOZ6OxUtYpIDkxYg9NSrSWi7zIxh\\\\\\\/hiwuMxpbO1Z0q2Kimo9mlU4FT9PlKvhSr5bmYuFF1nJe02hGH4srSNDRLp2HiA68\\\\\\\/ajpKZnJd2K9vLqS5RCbMapgzC5Yg1zOsq\\\\\\\/qNfBaZnxL+opZdL6Fr1Uj5EmjXsQFBsrCKwbgZr+2zZAd4VECKypgL3MX17ma9zUM98uKpYEaamhMl+R6fVz9cWiJcVzYuLDBGOw387B3YSz0XpPRMVT7zRzRERuqAPISHFtyvoo5b1PzZDf79fcWcKMAXik2QRaVAq0Y8cdBDeBFGGIoKPwHp16cy9WaMmmCawQXd9bU2FvrpXt+KV7TIvMVGkm+nM5U75I5zG0q38tFXv07Nik74rvNPFGid3NraX0kPpDBZMfRLva+RNZwVUjTKrGA8hKopzXoHlnsFSKVVf6NchEmV3o1eQU97xbsYpBTFtp82ZaKxDR77xhzfOJMyTpFcv5myRHX55Jx\\\\\\\/cCjCVrbfuM9+O+x2MhRlrYa5pMjbMIPdjvOvgWYISmh11TBmIXCEVuUNGaNAgOMu6uZMoDc6dAGKPZXOrZRefCwJPMYC\\\\\\\/WjgcWOQutt3qcihkoPKLudFeVmMnadVFl9IzQ80kbQMGILp96IPl3tlnUio5j9KnFcBCpdnDkVV8TaYxAQ+De8jTBZqIUuF0XCi\\\\\\\/rxHZEb\\\\\\\/OJwmeMD0uW\\\\\\\/snTR0TvWlK7dtPOglBqLbRiuFV7xiKL7FQpwjU+Fe76XdYYMjbXF05ucgiARYKjh3LSjG0tYBSQj6jKo2FK4k+QfLvpX08mxi3\\\\\\\/J6ZzOfzHl908JjW1zM+1sV\\\\\\\/lFLDkY2EILgz7WACQYAjgvB7jLr2NvS8FTtp8OyeB+IYuECizIE3oSy0cxytBSAGKRYRzTu7akdKE8glft7bDEeb1QlJhfTMcgBwmBjuGLlGHcPPNhohfxLF2P7BLpgZfYEuuOkndsBkd3hE1hDNHfmi3qxMRXiv+PGlUkhOG96pU8AP9PZuEuQAk0byfTsN+aBHZN\\\\\\\/D1W5eWKr5N+NQnMg0FcQq\\\\\\\/mfkYuDnkFMbvgLkBhJ\\\\\\\/CY5jyeA+Snrb0wWwMjB9qR3gAHW4L4ZnmyQtW0Wae67FuR1YoEltz9+PSk\\\\\\\/1VgDuYzWbn1aOp4+BkuFzkv7GeCED8GbFI1O8JgiMDmyoCz6kUyq7FozMos3h+fVzuJ33GHUVSdbIACeaTegQhpsPv6M8ouFoORRaDnvC86KhBKoDK++0qqIrNQ2Wt6ExQnvlgNrngJZrZAaR5IdoIrdXUrYMcj3yq\\\\\\\/Dja4hyicVDJ51C9HMVSAo8kFXzU7jG3i+pm7KLuSUueblP0An5cFoglJytsWJb2f9AQtLfrQyPE1drmvoHmfq69AUjaw2xaRa+zAuKGxa6kzJ\\\\\\\/hd+bsqTwaItClhR2Zkd2gJfLwgZLNbRVFXIEyARAVLG6ClBm0p3MmaiIkrx6pJQwG2MZTMtSr5fRd9JwwrlcWRA0L7plJxPR3FhwDDrowZ0rpud0TXLgmHMJcIHEmy6qgoCkfj8xxUwx38PgseHi0hO7Rh+qujDFNHsOsyoHLZJvoQRYvZ0TVKKTQtwjvyBM4xBxRxcM8E8kkPtBnP\\\\\\\/fw\\\\\\\/DB2kV+I62UwzOAmHpaW5\\\\\\\/91hPHWpuk50zbcQQdn6VqJQaXqV3uhSy0pwSqFFtvDIwgMi660SwLoGqKxXT359HqmmK6wjsXW9LaLXUDq+CtF2BrnS53eTkZzUQSKuRijI16jK0DbiFttFPXbnUsCoYh64CWs3Ynn6KmK9QHR+vyOtmhIB5aC0zkko+iKH2LUC61YGgIZmXtpqxjkCLXVpwDFe2sQ4fiq\\\\\\\/TfaBQ6iinTrwc4GKfI7C8BkYlpMVd7+Gm\\\\\\\/4SnYMgm6HSXaPq3Ha9RDnKZCtzLXFu3a9qiFjS\\\\\\\/XUtYwtlpZsWoiLRqVJqfRhoG1ZbvoAE9rMboQdP3VmPYKyrRA1h12SHPqpvEPK+ftsVTHMubXP5wdFLrUfzZ9UisBtW2mcvzTSqKJmg+9iKimW9ytpRgGLBtLgdSMUmFptX5oVyNfwd\\\\\\\/SWltPy6GRY2Hw6wpkJzq5rMdfqjUyanV7kvpgbkG13stR5i0k1uuZiVZH2\\\\\\\/x3isfjFrXV7\\\\\\\/El7zEybIEYi9VC\\\\\\\/SVfCMsErlW4Jxy4o6ozuSPgxRLiGLzJfgf\\\\\\\/xBKzV29yvigAVOB8bsXW83vWHMAgkmOPGypGjr3NqE9Va+A7psJHfaJVYKjMSM0KDEectk0asMViqnPv9xSeIVT04wCHcmVe1IU4BjeoIWEhYHbYsHV90LwPgjXG7DPEgM2nsKKewlx2uVpMNNk7RWXUrlq\\\\\\\/hS1MLnXFt9A9TdXePyEO5hK\\\\\\\/udVn1shmcWSAQ7xXylVFP6kApxRgzwEq7ca58YBBvAo0LT91VA2KVEDYgPzgMQC11HKwoKqckTKKqUObQ9f\\\\\\\/aFVZVaziWEJtTtJMpti9gBXRUfCeO1nEJoi93LZZGpf+ilaOjNZ2UNdEcTdagsMr6C07\\\\\\\/pnSUOOPJaWchBjEuciQZ1g4WKXOERyb5+WRVpqbInwc8vn7Q8LpV28Nu7tgCifG3l5illhsseRWroc2IMHe+ap\\\\\\\/oQKAR5MQoZfgFejzRWqIJygd2Sz8LHj9dvGM+P1DxWJfNHEdmClkCv7BYbA5h5\\\\\\\/naMqT5+Eq1gb48vE44nNPUtrdRkH1b4M1QlWzaDFb0K5MGmRDqgmGb5bYYKXSZgJerqGGV\\\\\\\/0v3ER4l8GADNqatN2i7DjGFJikjxez9wGwmeTvnnbI8gjwGT1m00agPzRttvp2bddbWwoHXtolxdRwE2up5HyjlP\\\\\\\/lRHalLy065wpi0R5ZazRmVDUmxXgZ7agmDxFIa9PSrg\\\\\\\/rXxJldx7CeYV+WtWj1Fsk9lBjyQvpuggDjjkcPAZuHnNKx1nbyuOir\\\\\\\/z+w35DlEiCjnBSURbTT6GXu1cBoyIH7Zkbbe3W\\\\\\\/hd1Hmbkzx9jU4UL1nvFztV0gaN98Iql+6VGZ8oPm+Sng8+Z8fdQqN7+FLU8cluUpL3S2YJLabD+oZqDqtsL0jOMl0cIjtDfq8dMN7J6LeQH6\\\\\\\/frAeLRa4ipfzydx5BaBNe5B0VTHMFO7UU0hPeK0Ew+ys+phjCY5WSCTpZrfjPicZvN6uAXCDOzC9yjeyhYwKifn8WY8WpMKDYNF8ajj4wAGG8QBk8dOgZHeuVHxsY1DFfgAzJ5ag4qA2TzOLJXgbrSW\\\\\\\/\\\\\\\/hmAyT4KlZ0pqUbnkZoSXBvBG\\\\\\\/p+ZpkbOSczXU4E\\\\\\\/jm6EqI369KnMliA+kYjTQHio5kYw9JJUvnI+m0tZJfhHlV3mfHY7GynsFVjqocmQKST5IWOgUmN3z\\\\\\\/24H4IazWmRYNFXuJyCe2In2qaqA5N8\\\\\\\/ZcLlw44HWbUI1CZ8IUxUvxXbGN3nISLz+x2ZeTov0v1gAQdPnkfkJtVBMeX49jKrX0PV5UjS05UAvAIioonOIHngJXSqhIcPk7lWU5Z+BildpDwngZ09NAOPtQTy8ViLpEX2kFTfGvW\\\\\\\/7uPgMl6+QNCz\\\\\\\/UZXxLwQKvtEKvHzIBRD3bu6EjDZv0N8Zq1hSV6Vb\\\\\\\/h6XKN0nXZtPgCL1YDAp80uBL2ZJq31UGHWF+nanQ7Z+CH3JgM6ocTQpfSAJ9z75AITh6TZO\\\\\\\/uGB\\\\\\\/3uLrcHjK2qI6wVTVAm146BTV6N9ZwlTvz8KJeVli2Sn18inEB43zMFhIJcDuoJ+hdIo5L1ziRKpkViiw8Z9mdlyY3uf6fbG9cs3aNZ9FapI0UWolOghyvsq4UQgmXoPgZMu3nBukIswEGKT5PE8okB+9OHVM7la\\\\\\\/3iXj\\\\\\\/egxW1dZu2F4\\\\\\\/e2kXdy3HT8ThtiZnEAV0LvMr5sHyXeE+cpsrsK7RBCltjm\\\\\\\/c3IlbfnYZGJeDyxPIMTThFPBn+YiD5BjlslZyWNTmye1Y6wpcpFcRrHk+XmYJflQbPGPxJSozMK0Q1j5dFbNMjlteLdK9OKYol\\\\\\\/8WlX7DtXJ0waQBA+soolh0o4ZikdGuBQVziKk+97EdKtgtKlPdyjepxmPI6CL2itQO9aGADkCttvViyASL2TRN7Mp4mNUBc2AtcMPFnMXfXKC75z516+LyyFqfUb5buFviXrBH8kWV9CzNFgZ8I9Wf25UL56ZTAVbWFIbVKcQXyKnlvA4\\\\\\\/mhucc0PJfdgul6J+khMebaUc8oCg9V\\\\\\\/afEjJNBjnHnzXSyh3GF0i4uH9vgjtThyb1XnL8FDuAfz9A3zbyfzo1BOzETfx4LRltd1DTh6ND9OrFGIcCiVDW5NEW78WiQlqANSE77zrz5f9EbKB4ebI45HK7nJ42uD9D5vJvffTTH8Jgmm1aCNrsPQyn0FGePKj\\\\\\\/Z+TH0ijtPy8QT24MSfCReCETe8bJB3PFd05Ckxd1mxYzoZpOPPuH6e332kgjK8EsM9oyjbWzFQ9KpL5FIRepmW4axjjc0tofhBrFcQ3o2na0+3XvsFK8RQkgX2iktQFikx4jCYWxlSu2ctsd87fGuWBjXVu7hRbV+czUePXGeRJ6NbFbCwh9x31k4T+YyoixORewxuwNJt45JJV5\\\\\\\/KgLGDd0qXPELKMRdr8ueov0702XXeinlAbfcIzyNudpCgfJHi+x++Sb1E5F7a9w3OmIN6206QjVjv0ejC4+1BPF0WsnN1eeHCuYUjv+QroNRf58g1sUZ82yawwMY\\\\\\\/vbEewJlPLo\\\\\\\/09gczE8uofqA9heBIA7c0uspWE9MqZbuMVqlGOuhdA\\\\\\\/4SHqITLvL9KzAZ6CMrmJMWbgx9SEaA1SrJM535LOHlamOzD8x2OtH+gNKfYLH4ZAiceLtgWlHYjwLgF4AcDk3MWGN8zC9Uc5FmuKwXHtSCI\\\\\\\/ngjqjjIx0Dh3cD5Atapd0Gmrv4gUWlTvygWqAHCNNNbAx2ArHyYHNiI9XKCrrq43YiNe9ZtnhFIeEOwx45f45BoueTq1qpGSGTzmqvJxLva1ajeYTvcabnLkxJ7MofmnEvMR6rMlUk7h8hwNkc7xBRPT4rYJw57fgqYBC9H2H0CHSuau+LHRVm9TEKGjA5eFqc6vrgbFiz2hVLduydaKs3SwkCIIrjOVSc5UIqKAKUKDMpRW2GV7\\\\\\\/D5Te0F03kzEY3sac3Ms8JGGjDXp2yrFTMXJLiIUDwSesCLWNMeL8VSMlj3921KDdyeXMGOi0uXsQTwok9NC2gentNL2F\\\\\\\/Mr4ZDhiNDHYAKWWbX+agDiTJD7SRzddeuM7C5973293GU4mjjb0prIRQWgba9mbPZoltWbQQcQOi4woJfSh0vQkWqNX5ZCOVHmYXgyKyFPVza3\\\\\\\/j5Fh8Hpg\\\\\\\/1lhXuCE71KtkgEYj53zfkPvj14PeeG\\\\\\\/05q4aQuW\\\\\\\/4Xrg6Dcp0evE5\\\\\\\/i8HnLwVhJy7nP1fZIZy0W2qyL3jZYnal8P4\\\\\\\/woz\\\\\\\/99VyDamk4safUmbUk+Uu7a0rcL3qgXejKdICUUzOJfQXxybKiiCUX52iKuxdwOdFC4neDi9YiOvHeS\\\\\\\/M92yePTNrGovKtOa5xd+QH2s0a8emmaEzLIShAvoiZT5duconhDWmahst+XzN9ps3dnN21UnOOePiOimvwteWhmp6RBHbh7e7QYp9dJ0EIXM5QuhjZ5wQXRCPGX8Hik4ah1BW2ntrsKH3rA9lPxjKJB7FsbAp9T\\\\\\\/P1pnCKCpUfhJXBY7a+F1\\\\\\\/WU3vh6AYdyJx2+47WA8hkxSqWERVMdsPbPt7+pt5tp9zOjOKeHRZ9dK5AHicJxbPqs5M55H3pJGErKOPNpf789SfBP3jtEYztPJz\\\\\\\/vk852VWO7Wru0sWjdwtPKBx\\\\\\\/DBR+Z6JqxdLeSrFAlpyqzYkwCYnixvAajMcVpEiBw5aSj5bDcBWQK2J1kyYqBQ3Ni+5E6GVIBJCCNH7fJeZDG0o7Ta\\\\\\\/tsLw8uprxPAjK\\\\\\\/tBuk2NEmIiuYfC7MR\\\\\\\/8rklpVzsGeyt43aSQRFMTGaRcQSuiRyS+8oayH5uUBGLGQpHdjYNFrLllHnCJ0Eolin+6BpkElzMsoEcguWfoSMDPTqjxfthE04RbdGHPEwceFS\\\\\\\/PYOwJ4BfHYvhGA4WBpfcUBor\\\\\\\/YGQNu7rKowtJjuPIWQyOSQemagAjr4qN4vz+UHtc\\\\\\\/qpyOYDuT1rA+i2grC1Un0fuUaCNSX9fqJkpHOb1+QM4+0Z7KTxYIqlHpKDl6S6oy9w4WqZDprAP4z0lzuZk02VfVJIY3ISroRfGT+3AwpsWvgQahqGArguRC70+lNGQKwTUSs0sVBkIyLgaD0bKEKNIMmZENjo7Omdj3QxCjz2yexFqwF74IJcZSthh0l3NFlz\\\\\\\/oM3QCkPZfRBKuTSwq4rqgCM6\\\\\\\/dGzb10M4EGFlSn8gX1exIf3vlC\\\\\\\/7xv2214N8bSGS2rdwnH05oKU21y81n4gfaF3k8lYDbLpDcIoKynqgYwPu+VXv7GqJChW1W31O7uCcComtlkf4VRag9z9RKa2y6BqHPmlZ2sKsm+7qTcWUkd0Lnhgemr1Pq\\\\\\\/5I2foxlSoc6NUxCrfusHj7XOLvUaf6YIdkx6NYaENNtdcKTatoxdGnNZ\\\\\\\/uDr3PDxnUcgbHxPngInYIddJKB+HZgIWYofC1cOvmUI9Jt3cYYEUW3lA5L9klzu9ndqk1J72K\\\\\\\/4mXf+ZTUkp4UwmGq0kDCHdyxk1qXuZLTIihnW6s9O4CYv3cLQTxgqG8bvCsnR0XMsdW36bbk0NaJSsDKIIpwV3iaYAg3qp4XOg23ijP7w4JYsAIcr0yWS7Toze90XSHizZdMCfQW7CXZu\\\\\\\/5nAhJ0h2A90p07bExrBtklin72lvL91l8U1VfDEAX+vFJ2hONV8iQT4d3zMjY5UE23B+KRifddwEdCvJKBx2mkjXMfiufQRchwnYDLPmNQNOBpDCEVnnqO+vB\\\\\\\/1Nsi+S1RJvvUOlmFH1ugabqKlC+X6U4SWxfpcG+VFhPZDf+\\\\\\\/tkkvnVg\\\\\\\/exgWVJtAxzrT7fuLmPmwhOZIoycqrK3dQz9fuJhTku+13WsD7WmHBki9utQyKe5YxUJ3iKlM9uEnQHhmSqbFhPJVC9hW8mssda88PloSuGl2cdHDXyzuwmKNU9+Rdht+JJoyShcVPh8a1rm02v3IB627eHRut+28ORkMEhhb14e8ABpIRzIhESl85i6eEsSaRVn610euXYyVjR9uPAiLSLtGRXa936mMlgU7l8ZF8PpzaOEeSZU\\\\\\\/ZTdd3msZqUN+g8OabB+UF8cSy7GF9\\\\\\\/+\\\\\\\/sgMk5kU3J8Fe0Pd6OL4GjMqK3LEYqdsbo09irOEPvKS8EPGDdMw3s+pK2bzUCSgioJPuA4p79nEMQsZfCipnIhTHbH1tIdqJ+ykAP0oEA8v0cvN9z004ivu2tNI8Bl7SxzsjfsiotPkDc3M+18YAqcO5TBz+ia03D3XBQ8yE4nZJxdpDKwhrNMwpKBOS2DU7Zo2Sj18z44s8Qu2tvwtQWWShLC5dfBl9BVG0JxacXWyD1XlRFTB+LEHSic2hapzke7VWa3t7Ixy91tEFlGNBmwgcF896EJhwRrIqJyweQuxCqWjIIe5VhaPxC5gFbOkUlPNQlTi7pepfKmSit6YJg4r1SD8eJ+YpJnr0uoBwyl7PhNH+ul6zo5dvl8x8a2zC5bYEKxA7ak0eX1BqvqaYUsHhS524en5m5VUlp7ldiK5F0peoJ2yZ9kavjWf6VcI1Rp6SYTdNy8eqaeXH2qs4lAOueIr54rh9e3DjPKtcxDf4pHXTlpwX4qMGIyRWSqclOBnnI5wcBUuwS8LzCt5qXrNg7viXTQvPhSxDI6lX74g1\\\\\\\/1U4vkVbJpNRpOuZGDQu\\\\\\\/qQ36z1NlxS\\\\\\\/yFyaEOFB6Wn4G3qgyGFywm\\\\\\\/OOcbY1nib2R7GoSshIM5tjv35yOUmvoRW5wkk+1LmV7mof\\\\\\\/OHifTLsmuTggvlcM0nqs\\\\\\\/\\\\\\\/ngBGYJ3Xi3vgrMWFc0npMwLZPeukthmXErekCmZDJvjtfeYNLwURAl0iQlWBCbduI72rUFkGylw6mi9OyjFa+3+SHCYo4rgh\\\\\\\/jAg82YpPnrrBfs0LDdrCnre7O1LdgietDo0Ukkewqh0+TJO3\\\\\\\/dyDCGndvcESXAtKp5OinKJxeCZLyZ26T8kIhmtHmzYD5xTNGiaGL7HrWtKpV5fONxnspL5tIT2s6ee2Vbd6OnwyD2yOkzFSDd\\\\\\\/b4syKHSq4xUZ0nOxWB1NgE9I6vgP2b+HmCZslA\\\\\\\/aXtZmWlBazCLPA\\\\\\\/05MKIHgukumQ5rxV\\\\\\\/9XimSAiwidKKoNZJrjD4HI3R3aFoM85ldHyMe+gBGQ2Jcd7KkZHDjBlMgEJCC6\\\\\\\/FALib1uovMUqMPkrQxZd3cBs3G6WvPwgaytc+1Q76c+hSLWHd1WCDHZV7daVY18JR\\\\\\\/rTfNVTsCLQm10am9njFKghpb8usqNRGjPksj6S9GhB\\\\\\\/Bf7Aq0MJhKNEqdhF5RRQ6k2IeBGWuKpI1rDmXgn\\\\\\\/K14EzFgYc5iD\\\\\\\/hsLgOkPvGogM80HWx4mAA1fCVsVwNVp3PVYbfDRJG9rwBklEwyHakZWRE1UHBaPH9UUwT9\\\\\\\/6uJ\\\\\\\/mzffcCEUk1IBklzo2tCmeSMBNnpLccqVaD9wJzcpGHozOgjbDUYwCAaLH0XlX6Smmdswm8jYhUrUUMB4t6VLa0XfT\\\\\\\/1w3YqTgsjn9+eGLwcXCVU4\\\\\\\/A17dJeD4l4mlZpcV02IemVGx+koinaPG9c3p2mzt9piBlvw\\\\\\\/YrocAsTB+b9ZfAwmv4SRn9gFNnqnaAIb1Oy4LYDfszlcY3\\\\\\\/bKtGJk5OSvGkgJaGAiKZ3jwfhGeferymOhewxgpH7Hott0zW98qxROjpaNFZYDWe61j3tOi5Q9tz7oB2h2R0NwV9CHRFCNgJSOulUQL8y\\\\\\\/RoQok6xrQyF1nw4m6mZK3uaIrKFvxIpyLHzGrCV4VGKcvq9fPHpQHjhFjv3awceD+RcCqpq8laJ01mrCk1Tg\\\\\\\/F+L0dPqs37QFHWk+FoPKYcjUPgIV3ZOCH4nvQ\\\\\\\/kBPKSOaTlTpEBe6V04cxd5WU1UZ0pIXie8j9Ks2DPVQzoPsFp9y4+sArwrAvoBMb\\\\\\\/j2fzZY5eQR75chlcDhUy92yFtchHyhaUYylAVzG7al1fGoGSXPDtKtiHfeLtn2pumMVgg1qAYZMukA5FaLKb8KTgEtg2tbwP2WxkHsVa11EXp8UrYS8OzlAZ1Ri0nyZAkN9R40iOm0GE4MQnbZhVRNv1g4MMSMQ5QYIP\\\\\\\/TGjjWVc3C3Uhhhhdkbgg5ElR832UY326Fx+Zb8acbU5NeqsrCuMbIqsivQAQNdrsAK0Umgd8bXZwPGgO3h3Ql4rh7CdkVTuBTYv1iFtQMuFz22+07Ht44rg+RFx9pQawr\\\\\\\/lL9NdQc3EaADum9ieg0sMAtO\\\\\\\/diXVSVYqCPi+YPPPSxcnKo+KnUA1bJX9KOPWSr82lWwJTd1ZspHoh+2CKMuueZRAx3yQTnfUsgxy1ErJY5LibvSCpJwKhQ4DxdAV10wSQECX4yDdpBJvGEndXBkha+8G\\\\\\\/Wo0MYDM0FQ6famtPToK1KNFXS1xlv0fcs\\\\\\\/bFQm0MnkWIn3+wotohBB3MHuUyEO6MCNRkmwUr43SOd2fBhvUEYptU51Ibj5vtw9dM+cpgPVHj90Iln\\\\\\\/rPQffS\\\\\\\/maeXKJ4YOiwMvRQvkiBOO+Wei5vxm+O8hvZgz+icQvH76wVu\\\\\\\/F6RY2Q93BVC5qssz4RDdGwNsgQy7nU6CQm0+pykD0FiHx7i5pyh+ojfEXcadtWA3tuKOh7rHPb\\\\\\\/1K56AxL7w+6+35uu4kw5+xDm5PE2w8eWJ3p9VokewQ+2OIqAT0B1djk1BzoeZrnKQNUKEvaZ3uKFf8DsPDle2wEtnz7hsES++e178XME7BckFlu5y+SoOD5y0SCQysViU\\\\\\\/yAA3D7o\\\\\\\/DDi+XnY2K4590F\\\\\\\/ecrFvcMMAhiWcsFOpkBmJ8vjumQcyex5TUMAAEfsPigCGpraYgqUi8K5bCRdrfuAyQ3kiLqVpX25XJ6K1zpfqrNFDcQ1FZp7tA6ngxWBJrh6F6tF8+5i1YggG3uPJL2jyU0w2p6BND+spZCCtXXgbrc\\\\\\\/\\\\\\\/RQuW7EZ+FQTDvGEy7lHCTX1aaITlEUMkQ5buoDjn7yze9UfGlH6X7MKSBi16IBJs1yLVwsEu44EdpbZICayZCU75LizeTaN0RBhJeMg9RWPMpeAd+yOkF8ofvP6Sxum5EF6l+0Qjb6\\\\\\\/dS0nuhwy9ghsdUSm8yq\\\\\\\/c7gFxO4w4lNVsgyP3u51mFxI2Ne\\\\\\\/VpkX+ZsqYNZUj5jCIDDQfuwEm7eNUvk4Hq4mtgup6t9dCvsV9EUFso1TZlxqExsRR5Ho7gBGBRHFD\\\\\\\/Gb5x0GZIE\\\\\\\/xlpnKP1i80j\\\\\\\/KJMxYBSqQfjob2l2nIM5K8sYDxmXdaf2TQW5E4v1B2ZeuZH5GMghzz7tUBHu47E\\\\\\\/94mjVhAgR362j+JD5wZ7Lj7zZXAOQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/rRah\\\\\\\/6Few77MWVY4T\\\\\\\/hyQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002232c1d0a10c9ccaf55f7d5fc2cecc046cc68021a5de90abd3c238f34eac9cde93\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fqFM9\\\\\\\/1CJsXQ6G+3zj6EbBXIPAfwvSLMfslBEizvt5l8p\\\\\\\/JPhySdcproj4Uy1\\\\\\\/ilVja0Dwb7Vpz6b+EtQM\\\\\\\/X35KHO2Bh0vwh4gpXPbUk5tZ7SkMomN4sfAW70gKnlf4AhuRU\\\\\\\/MT3\\\\\\\/EbO8w6Qz5bY4iEYgOGfJkptuKg6ja+gZ9Z8upHvuVl\\\\\\\/bx2RH3ND+ITJxMZn\\\\\\\/AL\\\\\\\/A8ZBZOagB6lrdL1HZCA4s4PkNV+eUVEbIdtD2lvfHdY5PjVuVctbGTp2SVjvz728Ne5Mw5QnJlqdBWBuYmNLxvWAQbzfes\\\\\\\/YkGWIoEiNEeSqqy+XKilyPcghwet4pL1kxVm5iJEY+2jRiocUlB5q\\\\\\\/7IWmCEoLmSUu+3PlPL2lLZ1OMk4Sz95qFHZ+4GpnAXVcIkF4iKdkw20O+mZ2M9OcXuaeeJmckeP\\\\\\\/YgK3lJBl4S91sdMS8nDUC0shoSPIzTti5Xh\\\\\\\/qaoaV+K6sejJH8Et5WMHEYPnuXAQAG1Mvroz1igcd9W9tjtTGXXVvc7urvQI2HNDmKG3\\\\\\\/nGeP+olQNTheFwzZGr7zeci4\\\\\\\/EJfLUqklj1K3XsGY+J4kJdupT5B2IszEWbzhYRFV4swg\\\\\\\/WAlHQV4S5I\\\\\\\/lazoCdr+IINlpeeDTk0rgAXLPHHjmQH8p+6t546iE6jsJ\\\\\\\/VA4TX+rPCKI\\\\\\\/Mxku1GFo3XPxGJEAseRJ94Q+i0X6ZebRppZkOioBVMxz75VRmnzZxDQdadHosA7dRv5JL2IEuCrBheMbjnrx8ojIGsRpAx9Xpy8WDm5YSDZ72WxKNWrTrsSF8YFqfmEVk0ZcWCPW8cMZL\\\\\\\/3baAxHBIS9usEZHdyL5dGkrMtzncfmexlfFPImG9bRNSqy2YX1oqPFX7yiuloT1J9tExGNsPMiSwfprLn7jAaiALVx4\\\\\\\/JYZewMmTTAQRE0AlsGaGbTphsAGqUWNrd1jEjNJk+\\\\\\\/HWsemgU4S3SDdB3HIoU2MCgCkKwpXHBssfZpJMjuauIH1a0POzrSPhpcfb8MOXlhODS7nWq2o1nyBl9SqnXp3PoT6\\\\\\\/wriQEc4coaAsU41qWfGBv1lm8\\\\\\\/A1RHR1HmYenHRszYfg64T57Hm+yoZXuvws9APe5capKDTGW1NvPmE978f9aYmHgqfui\\\\\\\/9N2FyMKOpfzQ8irLOKser58gU7qxHsVwZoFxIl2flZ4Td+m6k+otcO49pjh8Djrc6k+11QiubT4MppMp9VZYb+79TlVceMF4kRuVhtHc72\\\\\\\/i\\\\\\\/X16b2mr+ocjTCGDxxWEjkPJGLerkszzZ8J\\\\\\\/Y34MbkmvO61+c2N+nfSMX+jTiNeJPdk2cbTIl6h4pypF5rzKNdmBOL8kpBfhLcBZBkv9YycfpyW6y+V9wWRuzxw4wObyiZRd4Ah\\\\\\\/9m\\\\\\\/Sb3rWVw5NdN9lAwk\\\\\\\/EvQRz21N3j5A9ujBd7uoHjubrxOLRpgoXq52ZN7stCo6Vqrdki5uPvVVQbld6oFEsHwzFWRjF4n\\\\\\\/0RpFFF8jS490u\\\\\\\/0Q+l6fEw2apoCmPbQEAIVdbVJjHbSCQj\\\\\\\/E07m5pVBod322iHJFIKnNMxSAJfMN8C+IdjR5Lamw+6OXRvliZrQ6L6EazKAkRDb8vkW3oz1TbGm0nRd3M6V9SL0ulPqkM6bZ\\\\\\\/f0dqytSAr9PF2fcY\\\\\\\/0AaWDMGYyaN+8R4N8qqaBFPXyQM\\\\\\\/N3Rt5KfNiCQsRV0yXQbYp24nl0jL8pbSNAoCoVLa7gq7LIIqNBjNbwXHrDyVjIV87SdFR2rYzyHmGWZ7Q3AeEaAq\\\\\\\/F1EeRNaTZCDAgFSHBUBU9gF\\\\\\\/XiiPbJbrRCBGf8WR75PnAqnFiZYvwO\\\\\\\/UNmyP\\\\\\\/Zyii3HmwSaHyodtX21WI3vPD7sA3G45U0x+geSVP8TtgSUjw3vh0RZKXarZ2iTnVvEC68CAb2QBV2lKnLXovuGMpVp7LINvubQFWhkHToIBxiU7vs7DFyqpJMiwkCBxuMUdow09yrkiieXw7a9jGMC3edHvSKbYoWfQRwthjQmhxKEWQWzANeQinbusHZB7be+sfw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HKZ0KuKv1gJBbW58jNsWuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229c783177269c7432d874ae1ff4ee4386a38a5f145f415790be40e5e85bbdba19\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459990 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1403" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PkJZuS8RLe9rhqJkVZlNb634CW0RStmycBTnWx4Algw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1403" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3tdNGBOUyRO-6xqeEleewOey4ha7wCqneA43YR19-tg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1391" title="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1415" title="XSim: Simulation of Descendants from Ancestors with Sequence Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459990" id="node459990" data-pisa="ggg;5/7/1403" data-pisa-master="ggg;g3.115.018796" data-apath="/ggg/5/7/1403.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of QTLs for Rust Resistance in the Peanut Wild Species <em>Arachis magna</em> and the Development of KASP Markers for Marker-Assisted Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Soraya C. M. Leal-Bertioli</span>, <span class="highwire-citation-author" data-delta="1">Uiara Cavalcante</span>, <span class="highwire-citation-author" data-delta="2">Ediene G. Gouvea</span>, <span class="highwire-citation-author" data-delta="3">Carolina Ballén-Taborda</span>, <span class="highwire-citation-author" data-delta="4">Kenta Shirasawa</span>, <span class="highwire-citation-author" data-delta="5">Patrícia M. Guimarães</span>, <span class="highwire-citation-author" data-delta="6">Scott A. Jackson</span>, <span class="highwire-citation-author" data-delta="7">David J. Bertioli</span> and <span class="highwire-citation-author" data-delta="8">Márcio C. Moretzsohn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1403-1413;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018796 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459990" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Soraya C. M. Leal-Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Soraya%2BC.%2BM.%2BLeal-Bertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leal-Bertioli%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASoraya%2BC.%2BM.%2BLeal-Bertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:soraya.bertioli@embrapa.br" class="" data-icon-position="" data-hide-link-title="0">soraya.bertioli@embrapa.br</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Uiara Cavalcante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Uiara%2BCavalcante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavalcante%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUiara%2BCavalcante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ediene G. Gouvea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ediene%2BG.%2BGouvea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouvea%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdiene%2BG.%2BGouvea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Carolina Ballén-Taborda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBall%C3%A9n-Taborda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ball%C3%A9n-Taborda%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BBall%25C3%25A9n-Taborda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenta Shirasawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Kazusa DNA Research Institute, Kisarazu, Chiba, 292-0818, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenta%2BShirasawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirasawa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenta%2BShirasawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrícia M. Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patr%C3%ADcia%2BM.%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatr%25C3%25ADcia%2BM.%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Scott A. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BA.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BA.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David J. Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BBertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertioli%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BBertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Márcio C. Moretzsohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A1rcio%2BC.%2BMoretzsohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moretzsohn%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM%25C3%25A1rcio%2BC.%2BMoretzsohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/7/1403" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459990" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1403.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459990" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1403.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459990" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/7/1403.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459990" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/7/1403.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459990" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459990/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Rust is a major pathogen of the peanut crop. Development and adoption of rust-resistant cultivars is the most cost efficient and effective way to control the spread of the disease and reduce yield losses. Some cultivated peanut germplasm accessions have a degree of resistance, but the secondary gene pool is a source of much stronger resistance alleles. Wild species, however, have undesirable agronomic traits that are a disincentive to their use in breeding. The identification of genomic regions that harbor disease resistance in wild species is the first step in the implementation of marker-assisted selection that can speed the introgression of wild disease resistances and the elimination of linkage drag. In this work, we identify genome regions that control different components of rust resistance in a recombinant inbred line population developed from a cross between two <em>Arachis</em> species, the susceptible most probable B genome ancestor of cultivated peanut, <em>Arachis ipaënsis</em>, and an accession of its closest relative, <em>Arachis magna</em>, which is resistant to rust. Quantitative trait loci for several components of resistance were placed in the same position on linkage group B08. Single-nucleotide polymorphism Kompetitive allele-specific polymerase chain reaction markers for rust resistance region were designed and validated for marker function in both diploid and tetraploid contexts.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/arachis" class="hw-term hw-article-keyword hw-article-keyword-arachis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Arachis</em></a></li><li class="kwd"><a href="/keyword/peanut" class="hw-term hw-article-keyword hw-article-keyword-peanut" rel="nofollow">peanut</a></li><li class="kwd"><a href="/keyword/marker-assisted-selection" class="hw-term hw-article-keyword hw-article-keyword-marker-assisted-selection" rel="nofollow">marker-assisted selection</a></li><li class="kwd"><a href="/keyword/rust" class="hw-term hw-article-keyword hw-article-keyword-rust" rel="nofollow">rust</a></li><li class="kwd"><a href="/keyword/molecular-breeding" class="hw-term hw-article-keyword hw-article-keyword-molecular-breeding" rel="nofollow">molecular breeding</a></li></ul><p id="p-5">Peanut or groundnut (<em>Arachis hypogaea</em> L.) is a very important oilseed and food crop, cultivated on more than 24 million hectares in tropical and subtropical regions and with a global annual production of approximately 38 million tons (FAOSTAT 2012). It originated in a region encompassing the south of Bolivia and the northwest of Argentina by hybridization of two wild species and spontaneous chromosome duplication to form an allotetraploid.</p><p id="p-6">Peanut is affected by a number of pests and diseases that reduce yield and increase production costs. Rust, a foliar disease caused by <em>Puccinia arachidis</em> Speg. is one of the most important and widespread (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Subrahmanyam <em>et al.</em> 1993</a>). It is believed that the pathosystem rust-peanut co-evolved in Peru, where the host has been cultivated since prehistoric times. The peanut crop spread widely around the world after the Spanish and Portuguese colonization of South America, but the pathogen was largely confined to South America until late 1960s. However, it has since spread to all peanut-growing areas (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hammons 1973</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Subrahmanyam <em>et al.</em> 1993</a>). The pathogen is host-specific. Losses are particularly severe if the crop is also attacked by leaf spots [<em>Cercospora arachidicola</em> Hori and <em>Cercosporidium personatum</em> (Berk. and Curt.)]. Unlike leaf spots, rust-infected leaves tend to remain attached to the plant, and this, combined with a short life cycle, favor fast and prolific pathogen multiplication and rapid spread of the disease. Modern locally preferred cultivars generally are susceptible to rust and other foliar diseases. Chemical control of the fungus can be effective but is costly; timing of pesticide application also can be a challenge.</p><p id="p-7">The development and adoption of high-yielding, rust-resistant cultivars is seen as the best strategy to control the spread of the disease (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gibbons 1980</a>). However, no complete resistance has been found in cultivated germplasm (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Subrahmanyam <em>et al.</em> 1982</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pande and Rao 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Fávero <em>et al.</em> 2009</a>). This finding is consistent with the recent allotetraploid origin of cultivated peanut and the resulting limited number of disease resistance alleles available in its evolutionary arms race against rust. Additionally, since until late 1960s rust essentially was confined to South America and peanut cultivars elsewhere were not subject to selection pressure. In a wide screen of germplasm, more resistant genotypes were found to be mostly from Peru (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Subrahmanyam <em>et al.</em> 1989</a>, <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">1993</a>). Although partially resistant, cultivated peanut genotypes do exist, high levels of resistance or even immunity to rust only are found in the wild species (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Pande and Rao 2001</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Fávero <em>et al.</em> 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Leal-Bertioli <em>et al.</em> 2010</a>).</p><p id="p-8">Introgression of resistances from diploid wild relatives into allotetraploid peanut involves special crossing schemes to overcome the ploidy differences, followed by multiple rounds of backcrossing (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Simpson and Starr 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stalker and Lynch 2002</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Simpson <em>et al.</em> 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Holbrook <em>et al.</em> 2008</a>). The use of markers linked to rust resistance are likely to aid in the selection of backcrossed progeny, the breaking of linkage drag, the pyramiding of multiple resitance loci, and in increasing the speed of selection. Previously, a linkage map has been published using a population derived from a cross between the single available accession of the most probable B genome ancestor of peanut, <em>Arachis ipaënsis</em> K 30076, and a highly rust-resistant accession of the closely related <em>Arachis magna</em>, K 30097 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Shirasawa <em>et al.</em> 2013</a>). Here we report the identification of strong quantitative trait loci (QTL) linked to rust resistance and the development of single-nucleotide polymorphism (SNP) Kompetitive allele-specific polymerase chain reaction (KASP) markers suitable for foreground selection in breeding programs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material</h3><p id="p-9">The F<sub>6</sub> population composed of 94 individuals was obtained by single seed descent from the F<sub>2</sub> population described in <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> (2009)</a>. Progenies are derived from a cross between <em>A. ipaënsis</em> (accession GKBSPSc 30076, hereafter referred to in the abbreviated form K 30076), and the closely related <em>A. magna</em> (GKSSc 30097 hereafter referred to in the abbreviated form K 30097), used as the female and male parents, respectively. Seeds were obtained from the Brazilian <em>Arachis</em> germplasm collection, maintained at Embrapa Genetic Resources and Biotechnology (Brasília-DF, Brazil).</p></div><div id="sec-3" class="subsection"><h3>Phenotyping</h3><div id="sec-4" class="subsection"><h4>Rust phenotyping:</h4><p id="p-10">The recombinant inbred lines (RILs) population and the parents were phenotyped for resistance to <em>P. arachidis</em>. <em>Arachis hypogaea</em> cv. Runner IAC 886 was included as susceptible control. The population was evaluated on F<sub>6</sub> and F<sub>7</sub> generations. Phenotyping was performed using the detached leaf technique (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Moraes and Salgado 1982</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Leal-Bertioli <em>et al.</em> 2009</a>). Field assays would not be suitable because of the architecture of the wild-derived diploid plants. Rust spores were collected from infested peanut plants in Pindorama, São Paulo State, Brazil (coordinates 21.1858° S, 48.9072° W). Two bioassays were done, one in 2012 and the other in 2013. In the bioassay of 2012, leaves were inoculated with ca. 4 × 10<sup>5</sup> urediniospores/mL in 0.05% Tween 20 fungal spores and maintained at 26–28° and photoperiod of 10-hr light and 14-hr dark. In the bioassay of 2013, ca. 2 × 10<sup>5</sup> urediniospores/mL were used. Four replicates of each individual were analyzed 25 d after inoculation. Susceptibility was measured using the following parameters: total number of lesions/leaf area (cm) (TL/LA), number of sporulated lesions/leaf area (cm) (SL/LA), Incubation period (time for appearance of first lesion in number of days after inoculation) (IncPer), and susceptibility index (SI). SI was calculated with the scale of <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Savary <em>et al.</em> (1989)</a>, with the following modifications: index was the number of lesions times a number that reflected lesion size/reaction. I = ∫(s * n)/LA, where s = lesion size (1 = necrotic aborted lesion, 2−6 = ruptured, sporulating pustules, varying between 0.5 and 3 mm in diameter), n = number of lesions of a particular size, LA = leaf area (mm<sup>2</sup>). Sporulation was evaluated with the aid of a stereoscope microscope. LA was calculated with the software Quant (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Vale <em>et al.</em> 2001</a>). In genotypes that did not present symptoms and therefore did not have incubation period, for QTL analyses, this trait was artificially tabulated as 200.</p></div><div id="sec-5" class="subsection"><h4>Other agronomic/domestication traits:</h4><p id="p-11">Plants were grown in long trays (1 m × 30 cm × 30 cm), with enough space for lateral branch trailing and seed set. Branches were regularly trailed back to the pots to ensure that pegs would penetrate the soil. At between 40 and 60 d after planting, height of main stem of up to 10 plants of each RIL was measured (main stem height; MSH). At harvest (approximately 120 d after planting), peg length (PL) and pod constriction (PC) was measured with six replications. Harvested seeds were counted (seed number), dried at 20° at 15% relative humidity (RH) for 15 d, and 10 seeds, randomly selected, were weighted (10-SW). Evaluations were performed in three years, 2011, 2012 and 2013, except for PL and PC, which were evaluated only in 2012.</p></div></div><div id="sec-6" class="subsection"><h3>DNA extraction</h3><p id="p-12">Total genomic DNA was extracted from young leaflets essentially as described by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grattapaglia and Sederoff (1994)</a>. The quality and quantity of the DNA were evaluated on 1% agarose gel electrophoresis and spectrophotometer NanoDrop 1000 (Thermo Fisher Scientific).</p></div><div id="sec-7" class="subsection"><h3>Genetic mapping and QTL analyses</h3><p id="p-13">A linkage map for this population has been published in <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Shirasawa <em>et al.</em> (2013)</a>. This map contained 773 microsatellite and 25 miniature inverted-repeat transposable element loci. We used these 798 markers plus 26 newly genotyped microsatellite markers to construct an updated linkage map, by using Mapmaker Macintosh 2.0 and Mapmaker/EXP 3.0 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lander <em>et al.</em> 1987</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lincoln <em>et al.</em> 1992</a>). A χ<sup>2</sup> test was performed to test the null hypothesis of 1:1 segregation on all scored markers. A minimum logarithm of the odds (LOD) score of 6.0 and maximum recombination fraction (θ) of 0.35 were set as thresholds for linkage groups (LGs) determination with the “group” command. The most likely marker order within each LG was estimated by the matrix correlation method using the “first order” command. Marker orders were confirmed by comparing the log-likelihood of the possible orders by permuting all adjacent triple orders (“ripple” command). After establishment of the group orders, the LOD score was set to 3.0 to include additional markers in the groups. The “try” command was then used to determine the exact position of the new markers within each group. The new marker orders were again confirmed with the “ripple” command. Recombination fractions were converted into map distances in centimorgans (cM) using the Kosambi’s mapping function and the “error detection” command available in Mapmaker/EXP 3.0 (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lander <em>et al.</em> 1987</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Lincoln <em>et al.</em> 1992</a>). Based on this map, genomic regions with no recombination or identical markers were identified and all loci but one were removed from these regions (pairs or groups of loci with 0 cM distance).</p><p id="p-14">This newly developed framework map was used for QTL analysis. Phenotyping data included the components of resistance to <em>P. arachidis</em> and agronomic traits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018796/-/DC1/018796SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a>). Traits phenotyped in different trials or bioassays were analyzed separately. The normality of data distribution was evaluated by skewness and kurtosis values using WinQTL Cartographer, version 2.5 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2006</a>). QTL were mapped by using the composite interval mapping method proposed by <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zeng (1993</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">1994</a>) and also the WinQTL Cartographer. This software assumes that the quantitative data under analysis are normally distributed. Some of the data sets did not fit this assumption and were log(x) transformed. We performed composite interval mapping analysis with the standard model (Model 6), scanning the genetic map and estimating the likelihood of a QTL and its corresponding effects at every 1 cM while using eight significant marker cofactors to adjust the phenotypic effects associated with other positions in the genetic map. A window size of 10 cM was used, and therefore cofactors within 10 cM on either side of the QTL test site were not included in the QTL model. Thresholds were determined for each trait by permutation tests (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Churchill and Doerge 1994</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Doerge and Churchill 1996</a>), by the use of 1000 permutations and a significance level of 0.05. Graphic presentation of the LGs and the significant QTL was drawn with MapChart, version 2.1 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Voorrips 2002</a>).</p></div><div id="sec-8" class="subsection"><h3>KASP marker development and validation in tetraploid genetics</h3><div id="sec-9" class="subsection"><h4>Rationale:</h4><p id="p-15">The aim of the methods in this section was to develop reliable and easy-to-use DNA markers for the genomic region in <em>A. magna</em> K 30097 that confers rust resistance. Although <em>A. magna</em> K 30097 was of primary interest for this study, <em>A. batizocoi</em> K9484 also is being used in our research and for introgression in breeding programs (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Leal-Bertioli <em>et al.</em> 2014b</a>). Therefore, we aimed to develop markers that would function for both these species (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Leal-Bertioli <em>et al.</em> 2014a</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">b</a>).</p><p id="p-16">Because introgression will be in allotetraploid cultivated peanut, the markers must function in this genetic context, but for SNP discovery, we used a strategy of SNP calling in the diploid context. This strategy relies on the very close relationship of <em>A. ipaënsis</em> and the B genome of <em>A. hypogaea</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Moretzsohn <em>et al.</em> 2013</a>). Because of this close relationship, a polymorphism identified between <em>A. magna</em> and <em>A. ipaënsis</em> is very likely to be conserved between <em>A. magna</em> and the B genome of <em>A. hypogaea</em>. After marker design, this conservation was confirmed by marker assays.</p></div><div id="sec-10" class="subsection"><h4>Production and assembly of transcript sequences:</h4><p id="p-17">Total RNA from <em>A. magna</em> K 30097 and <em>A. batizocoi</em> K9484 was extracted from the first expanded leaf of the main axis using the QIAGEN Plant RNeasy kit (QIAGEN) with on-column DNAse treatment. cDNA libraries were constructed using equal amounts of RNA from five individuals of each genotype using the TruSeq v2 library construction kit (Illumina), as described in <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Leal-Bertioli <em>et al.</em> (2015)</a>. To obtain long reads to improve transcriptome assemblies, size-selected libraries were sequenced using MiSEQ v3.0. Adapter and quality trimming was performed using Trim_galore! v0.3.5. (<a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>). Adapters were trimmed with Cutadapt (<a href="http://code.google.com/p/cutadapt/">http://code.google.com/p/cutadapt/</a>). FastQC (<a href="http://galaxy.csdb.cn:8000/tool_runner?tool_id=fastqc">http://galaxy.csdb.cn:8000/tool_runner?tool_id=fastqc</a>) was used to display quality information for cleaned reads. Transcripts were assembled using Trinity (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haas <em>et al.</em> 2013</a>). Assembled transcripts were filtered to include only the longest isoform from each read cluster. The longest isoforms were then aligned to each other by the use of NCBI blastn v2.2.29. Alignments with 100% sequence identity and ≥ 90% sequence length were considered redundant and removed from the final assembly.</p></div><div id="sec-11" class="subsection"><h4>SNP discovery:</h4><p id="p-18"><em>Arachis ipaënsis</em> reference genome sequence (<a href="http://www.Peanutbase.org">www.Peanutbase.org</a>) was used as proxy of the B genome of <em>A. hypogaea</em> to discover SNPs between the rust-resistant accessions and peanut (susceptible). This was done by aligning <em>A. magna</em> and <em>A. batizocoi</em> transcripts (resistant) with the reference genome of <em>A. ipaënsis</em> (GenBank assembly accession GCA_000816755.1) using the NGSEP pipeline (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Duitama <em>et al.</em> 2014</a>) tagging the region where the main QTL for rust resistance was identified (pseudomolecule Araip.B08, peanutbase.org), in the vicinity of the microsatellite marker Ah-280 (region between 117048352 and 129519037 bp) and also for another QTL linked marker on Araip.B08, AHGS1350 (region between 346729 and 848328 bp) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Default parameters were used, except the minimum and maximum fragment length for valid paired-end alignments, which we estimated separately for each genotype aligning their first 250,000 fragments and then plotting the distribution of estimated insert lengths (Script available at the NGSEP Web site <a href="http://sourceforge.net/projects/ngsep/files/Library/scripts/">http://sourceforge.net/projects/ngsep/files/Library/scripts/</a>). We used the recommended parameters of NGSEP for analysis of WGS data: (1) minimum genotype quality 40; (2) minimum value allowed for a base quality score 30; and (3) Maximum number of alignments allowed to start at the same reference site 2. We also used NGSEP for filtering (the most relevant was a maximum minor allele frequency of 0.01) and conversion from VCF to other formats for primer design and visualization of SNPs with Flapjack software (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Milne <em>et al.</em> 2010</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/466454/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/466454/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Quantitative trait loci identified for resistance to <em>Puccinia arachidis</em> and agronomic traits on an <em>A. ipaënsis</em> × <em>A. magna</em> F<sub>6</sub> population</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h4>Primer design and test:</h4><p id="p-24">Allele-specific forward primers and a common reverse primer were designed for use in KASP assays (LGC Genomics Ltd. Hoddesdon, UK; <a href="http://www.lgcgenomics.com/kasp-genotyping-reagents">http://www.lgcgenomics.com/kasp-genotyping-reagents</a>) using BatchPrimer3 (<a href="http://probes.pw.usda.gov/batchprimer3/">http://probes.pw.usda.gov/batchprimer3/</a>) with the “Allele specific primers and allele flanking primers” option. The parameters used were 60−120 bp in size, Tm between 58 and 60°, and GC content between 30 and 80%. The alternative alleles were marked with 6-FAM and reference alleles with VIC. For each SNP, two allele-specific forward primers and one common reverse primer were designed. A schematic diagram of SNP discovery and primer design is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A and B</a>. Primer information is listed in <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic diagram of single-nucleotide polymorphism (SNP) discovery and Kompetitive allele-specific polymerase chain reaction (KASP) primer design. A. ipa&#xEB;nsis K 30076 is used as proxy for the B-genome of A. hypogaea. (A) Alignment of A. ipa&#xEB;nsis K 30076, A. magna K 30097, and A. batizocoi K 9484 paired-end cDNA reads onto A. ipa&#xEB;nsis K 30076 genomic sequence, Pseudomolecule Araip.B08 (where rust resistance QTL reside) and the identification of SNPs. (B) Example of design of allele-specific and site specific (common) primers for the SNPs identified." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628833549" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A schematic diagram of single-nucleotide polymorphism (SNP) discovery and Kompetitive allele-specific polymerase chain reaction (KASP) primer design. &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; K 30076 is used as proxy for the B-genome of &lt;em&gt;A. hypogaea&lt;/em&gt;. (A) Alignment of &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; K 30076, &lt;em&gt;A. magna&lt;/em&gt; K 30097, and &lt;em&gt;A. batizocoi&lt;/em&gt; K 9484 paired-end cDNA reads onto &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; K 30076 genomic sequence, Pseudomolecule Araip.B08 (where rust resistance QTL reside) and the identification of SNPs. (B) Example of design of allele-specific and site specific (common) primers for the SNPs identified.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1403/F1.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/7/1403/F1.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466404" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">A schematic diagram of single-nucleotide polymorphism (SNP) discovery and Kompetitive allele-specific polymerase chain reaction (KASP) primer design. <em>A. ipaënsis</em> K 30076 is used as proxy for the B-genome of <em>A. hypogaea</em>. (A) Alignment of <em>A. ipaënsis</em> K 30076, <em>A. magna</em> K 30097, and <em>A. batizocoi</em> K 9484 paired-end cDNA reads onto <em>A. ipaënsis</em> K 30076 genomic sequence, Pseudomolecule Araip.B08 (where rust resistance QTL reside) and the identification of SNPs. (B) Example of design of allele-specific and site specific (common) primers for the SNPs identified.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">KASP assays were performed with the following genotypes: the diploids <em>A. ipaënsis</em> K 30076, <em>A. batizocoi</em> K9484 and <em>A. magna</em> K 30097, the induced allotetraploids (<em>A. magna</em> K 30097 x <em>A. stenosperma</em> V15076)<sup>4x</sup> (here called MagSten) and (<em>A. batizocoi</em> K9484 × <em>A. stenosperma</em> V10309)<sup>4x</sup> (here called BatSten1) and six <em>A. hypogaea</em> cultivars (Tifrunner, Tifguard, GA-06G, NC3033, ICVG 88145, and SPTG_06). Reactions consisted of 2 μL of KASP 2X reaction mix, 0.055 μL of assay primer mix (12 mM of each allele-specific primer and 30 mM of common primer) and 20 ng of genomic DNA, in a 4-µL volume. A C1000 Thermal Cycler (Bio-Rad) was used with the following cycling conditions: 94° for 15 min, nine cycles of 94° for 20 sec, touchdown starting at 65° for 60 sec (decreasing 0.8° per cycle), 29 cycles of 94° for 20 sec, and 57° for 60 sec (<a href="http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/PDFs/KASP_SNP_Genotyping_Manual.pdf">http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/PDFs/KASP_SNP_Genotyping_Manual.pdf</a>). To improve the results, a second KASP program was run as following: 9 cycles of 94° for 20 sec and 57° for 60 sec. Fluorescence was read by a The LightCycler 480 Instrument II (Roche Life Science) and analyzed using the LightCycler 480 Software (V.1.5.1).</p></div></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Phenotyping</h3><p id="p-27">The F<sub>6</sub> RIL population used here was produced by single seed descent from a cross <em>A. ipaënsis</em> × <em>A. magna</em>. The F<sub>1</sub> showed high fertility, reflecting that the parents are very closely related. The population shows large variability for main stem height, length of lateral branches, seed size and number, and resistance to rust. Most individuals show mid-parent values, but the population showed large transgressive segregation. High skewness and kurtosis values showed that four of the 15 traits evaluated were not normally distributed. These were incubation period for 2012 and 2013 (IncPer_2012, IncPer_2013); seed number, average of 3 years; and main stem height for 2011 (MSH_2011). To achieve approximately normal distributions, these four traits were log(x) transformed. All phenotyping information is presented on <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a>.</p><div id="sec-15" class="subsection"><h4>Rust:</h4><p id="p-28">The frequency distribution based on the pooled data for TL/LA, number of SL/LA, and incubation period (IncPer) showed strong biased toward resistance, because 41 individuals and the resistant parent presented no lesions in either experiments (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>). The susceptible parent <em>A. ipaënsis</em> K 30076 showed greater susceptibility than the control, <em>A. hypogaea</em> cv. Runner IAC 886, in all components tested: larger SI, TL/LA and SL/LA, and shorter IncPer than the susceptible control (<a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a>). Seventeen individuals had greater SI, SL/LA, and TL/LA than the susceptible control. In addition, five had shorter IncPer (<a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a>). As expected, the number and size of lesions were negatively correlated with Incubation period (Pearson r = – 0.6, <em>P</em> &lt; 0.005). No chlorosis was observed on any accession of the F<sub>6</sub>/F<sub>7</sub> population. On leaves of some of the less-susceptible genotypes, necrotic areas corresponding to colonies aborted at a late developmental stage were observed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distribution of rust resistance, peg, and seed characteristics (A&#x2212;F) in recombinant inbred lines (F6/F7 RILs) derived from a cross of A. ipa&#xEB;nsis K 30076 (Ai) with A. magna K 30097 (Am). For rust tests, A. hypogaea cv. Runner IAC 886 (R) was the susceptible control. With the exception of pod constriction, for all traits the means of the parents are significantly different (P &lt; 0.05). In (A) genotypes without symptoms (therefore without incubation period), this trait was artificially tabulated as 200." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628833549" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency distribution of rust resistance, peg, and seed characteristics (A&#x2212;F) in recombinant inbred lines (F&lt;sub&gt;6&lt;/sub&gt;/F&lt;sub&gt;7&lt;/sub&gt; RILs) derived from a cross of &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; K 30076 (Ai) with &lt;em&gt;A. magna&lt;/em&gt; K 30097 (Am). For rust tests, &lt;em&gt;A. hypogaea&lt;/em&gt; cv. Runner IAC 886 (R) was the susceptible control. With the exception of pod constriction, for all traits the means of the parents are significantly different (&lt;em&gt;P&lt;/em&gt; &lt; 0.05). In (A) genotypes without symptoms (therefore without incubation period), this trait was artificially tabulated as 200.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1403/F2.medium.gif" width="440" height="384"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/7/1403/F2.medium.gif" width="440" height="384"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466417" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Frequency distribution of rust resistance, peg, and seed characteristics (A−F) in recombinant inbred lines (F<sub>6</sub>/F<sub>7</sub> RILs) derived from a cross of <em>A. ipaënsis</em> K 30076 (Ai) with <em>A. magna</em> K 30097 (Am). For rust tests, <em>A. hypogaea</em> cv. Runner IAC 886 (R) was the susceptible control. With the exception of pod constriction, for all traits the means of the parents are significantly different (<em>P</em> &lt; 0.05). In (A) genotypes without symptoms (therefore without incubation period), this trait was artificially tabulated as 200.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Other agronomic/domestication traits:</h4><p id="p-30">Phenotypic evaluations were performed at different generations. Values were normally distributed for most traits for most years. With the exception of PC and PL, the means of the parents are significantly different (<em>P</em> &lt; 0.05). Comparison of the means of the parents and the segregating genotypes reveals that for all traits there is transgressive segregation in the progenies (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, C−F). On an average for 3 years, <em>A. magna</em> produced more seed than <em>A. ipaënsis</em>, but 10-seed weight was lower (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2, E and F</a>).</p></div></div><div id="sec-17" class="subsection"><h3>Map construction</h3><p id="p-31">All markers evaluated in this study were amplified by the use of heterologous primers. With the use of minimum LOD score of 6.0 and a maximum recombination fraction (θ) of 0.35, and after the exclusion of co-segregating markers, we found that 399 markers mapped onto 10 LGs, spanning a total map distance of 678.2 cM. These markers included 378 microsatellites and 21 transposon (miniature inverted-repeat transposable element) markers. LGs were numbered and oriented essentially according to <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> (2009)</a>, but with B05 and B08 reoriented (“flipped”) to ensure forward compatibility with pseudomolecules produced by the Peanut Genome Sequencing Consortium (<a href="http://www.peanutbase.org">www.peanutbase.org</a>; Genbank GCA_000816755.1). LGs ranged from 41.5 cM (with 35 markers) to 139.2 cM (55 markers), with an average distance of 1.7 cM between adjacent markers. A total of 91 (22.8%) of the 399 mapped markers deviated from the expected 1:1 ratio at <em>P</em> &lt; 0.05 level. Of these, 79 markers were skewed toward <em>A. ipaënsis</em> and only 12 markers toward <em>A. magna</em>. Most LGs have few distorted markers, and LG B05 and B08 had no distorted markers. In contrast, LG B02 and the upper portion of LG B04 were almost entirely composed of distorted markers. These two LGs and LG B06 were distorted toward <em>A. ipaënsis</em> alleles. Distorted markers at <em>P</em> &lt; 0.05 were identified by # (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A genetic linkage map of the B-genome of Arachis obtained through the analysis of 94 F6 plants, generated from a cross between A. ipa&#xEB;nsis K 30076 and A. magna K 30097. Numbers on the left of each group are Kosambi map distances (cM). Markers that amplified more than one loci have numbers _1 and _2 after the marker name. Quantitative trait loci (QTL) are indicated as colored bars running alongside linkage groups. Colors/textures are according to categories: black, rust resistance [total number of lesions/leaf area (TL/LA), number of sporulated lesions/leaf area (SL/LA), susceptibility index (SI), and Incubation Period (IncPer)]; white, seed characteristics [seed number (SN) and 10-seed weight (10_SW)]; textured, plant architecture [main stem height (MSH)]; and domestication traits [peg length (PL) and pod constriction (PC)]. Distorted alleles (P &lt; 0.05) are indicated by #." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628833549" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A genetic linkage map of the B-genome of &lt;em&gt;Arachis&lt;/em&gt; obtained through the analysis of 94 F&lt;sub&gt;6&lt;/sub&gt; plants, generated from a cross between &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; K 30076 and &lt;em&gt;A. magna&lt;/em&gt; K 30097. Numbers on the left of each group are Kosambi map distances (cM). Markers that amplified more than one loci have numbers _1 and _2 after the marker name. Quantitative trait loci (QTL) are indicated as colored bars running alongside linkage groups. Colors/textures are according to categories: black, rust resistance [total number of lesions/leaf area (TL/LA), number of sporulated lesions/leaf area (SL/LA), susceptibility index (SI), and Incubation Period (IncPer)]; white, seed characteristics [seed number (SN) and 10-seed weight (10_SW)]; textured, plant architecture [main stem height (MSH)]; and domestication traits [peg length (PL) and pod constriction (PC)]. Distorted alleles (&lt;em&gt;P&lt;/em&gt; &lt; 0.05) are indicated by #.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1403/F3.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/7/1403/F3.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466398" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">A genetic linkage map of the B-genome of <em>Arachis</em> obtained through the analysis of 94 F<sub>6</sub> plants, generated from a cross between <em>A. ipaënsis</em> K 30076 and <em>A. magna</em> K 30097. Numbers on the left of each group are Kosambi map distances (cM). Markers that amplified more than one loci have numbers _1 and _2 after the marker name. Quantitative trait loci (QTL) are indicated as colored bars running alongside linkage groups. Colors/textures are according to categories: black, rust resistance [total number of lesions/leaf area (TL/LA), number of sporulated lesions/leaf area (SL/LA), susceptibility index (SI), and Incubation Period (IncPer)]; white, seed characteristics [seed number (SN) and 10-seed weight (10_SW)]; textured, plant architecture [main stem height (MSH)]; and domestication traits [peg length (PL) and pod constriction (PC)]. Distorted alleles (<em>P</em> &lt; 0.05) are indicated by #.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-18" class="subsection"><h4>QTL identification:</h4><p id="p-33">The framework map, containing 399 markers, was used for QTL analysis. LOD significance threshold estimated for each trait ranged from 2.5 to 3.2, and only QTL with LOD values exceeding these values were included. At least one QTL was detected for each of the 15 traits analyzed, with a total of 38 QTL mapped. A summary of QTL is provided in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Detailed information about markers and QTL are presented in <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a>.</p></div><div id="sec-19" class="subsection"><h4>QTL for rust resistance:</h4><p id="p-34">For rust resistance, 13 QTLs were identified on the two bioassays. A major QTL for the four components of rust resistance (SI, TL/LA, SL/LA, and IncPer) was consistently identified in both bioassays and mapped at the same marker interval on map position 35.1−42.9 cM on LG B08, with LOD scores between 2.9 and 8.2. Its closest marker is Ah-280. This QTL explained 5.8–59.3% of the phenotypic variance of the four different traits (SI, TL/LA, SL/LA, and IncPer). Another QTL for SI and TL/LA evaluated in 2012 and for IncPer_2013 was found at 25.4−33.1 cM on the same LG B08 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), explaining 13.2–34.8% of the total variance in 2012 and 2013, respectively. The closest markers are AHGS1350 and AHGS2541. In addition to these two QTL, two minor QTL were identified in LGs B04 and B07 for SI_2012. For all QTL, resistance was derived from <em>A. magna</em>. The contribution of <em>A. magna</em> alleles for the rust resistance-related traits was evaluated by calculating the average phenotypes of the homozygote plants for the <em>A. magna</em> allele <em>vs.</em> the average phenotype of those homozygote for the <em>A. ipaënsis</em> allele (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). <em>Arachis magna</em> alleles contributed significantly for the reduction of SI, TL/LA, and SL/LA. The effect of <em>A. magna</em> alleles was less pronounced for IncPer (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bar graph of contribution of A. ipa&#xEB;nsis K 30076 (blue bars) vs. A. magna K 30097 (orange bars) alleles to the rust resistance-related traits: susceptibility index (SI), number of sporulated lesions/leaf area (SL/LA), total number of lesions/leaf area (TL/LA). and incubation period (days/20) (IncPer/20)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628833549" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bar graph of contribution of &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; K 30076 (blue bars) &lt;em&gt;vs. A. magna&lt;/em&gt; K 30097 (orange bars) alleles to the rust resistance-related traits: susceptibility index (SI), number of sporulated lesions/leaf area (SL/LA), total number of lesions/leaf area (TL/LA). and incubation period (days/20) (IncPer/20).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1403/F4.medium.gif" width="349" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/7/1403/F4.medium.gif" width="349" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Bar graph of contribution of <em>A. ipaënsis</em> K 30076 (blue bars) <em>vs. A. magna</em> K 30097 (orange bars) alleles to the rust resistance-related traits: susceptibility index (SI), number of sporulated lesions/leaf area (SL/LA), total number of lesions/leaf area (TL/LA). and incubation period (days/20) (IncPer/20).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h4>QTL for other agronomic/domestication traits:</h4><p id="p-36">Agronomic and domestication traits were evaluated in three different years. QTL were consistent between years; therefore, for final QTL analyses, data from different years were pooled and considered different replicates of the same experiment. The exception was MSH, which was not pooled, because measures were performed in different stages of plant growth in each year. In total, 25 QTL were identified. Interestingly, the QTL for two domestication traits (PC and PL) were placed in the same map positions: 37.1−40.8 cM in LG B01 and 64.3−64.9 cM in LG B04 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). LG B04 also harbors a cluster of QTL for MSH at 64.3−64.9 cM, explaining 10.7–30.4% of the total phenotypic variance. For all these traits, the main markers associated were AHGS1917 and AHGS2155. Alleles increasing values of domestication-related traits originated from both parents. However, alleles increasing seed number are all derived from <em>A. ipaënsis</em>. No QTL for domestication/productivity traits co-localized with the rust resistance QTL.</p></div></div><div id="sec-21" class="subsection"><h3>KASP primer design and validation on a tetraploid background</h3><p id="p-37">Of a total of 24 assays designed, 22 worked well with the samples tested. Nineteen of them successfully distinguished <em>A. ipaënsis</em> and <em>A. hypogaea</em> from <em>A. magna</em> and <em>A. batizocoi</em>. Four cluster configurations were observed: (1) with nine assays, two clusters were present: one with <em>A. ipaënsis</em> plus the six cultivars of <em>A. hypogaea</em> and another with <em>A. magna</em>, <em>A. batizocoi</em>, and their derived induced allotetraploids MagSten and BatSten1 (noted in <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a> as Ah = Ai≠(Ab = Am = MagSten = BatSten)). An example is shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>; (2) with eight assays, one extra cluster was observed: <em>A. hypogaea</em> formed a different cluster intermediate in position (noted in <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a> as Ah≠Ai≠(Ab = Am = MagSten = BatSten))(<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>); (3) with two assays, a different extra cluster was observed: MagSten was distinguished from all other genotypes (noted as (Ah = Ai)≠(Ab = Am = BatSten)≠MagSten)); (4) and finally, in three assays, <em>A. ipaënsis</em> formed an isolated cluster, being the other cluster formed by all the other genotypes (noted as Ai≠(Ah = Ab = Am = MagSten = BatSten)).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screenshots of the two most common examples of Arachis B-genome single-nucleotide polymorphism genotyping using Kompetitive allele-specific polymerase chain reaction assays. Both patterns show differentiation between A. ipa&#xEB;nsis K 30076 and the B-genome of A. hypogaea from the wild species A. magna K 30097 and A. batizocoi K 9484 and the induced allotetraploids MagSten and BatSten1. In (A), two clusters are present: one with A. ipa&#xEB;nsis and all A. hypogaea cultivars, and another with the wild species and induced allotetraploids (noted in File S1 as Ah = Ai&#x2260;(Ab = Am = MagSten = BatSten)). In (B), three clusters are present. In these cases, A. hypogaea forms a different cluster, intermediate in position [noted in File S1 as Ah&#x2260;Ai&#x2260;(Ab = Am = MagSten = BatSten)]. All genotypes with A. batizocoi derived rust-resistant cluster in different groups to the susceptible genotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1628833549" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Screenshots of the two most common examples of &lt;em&gt;Arachis&lt;/em&gt; B-genome single-nucleotide polymorphism genotyping using Kompetitive allele-specific polymerase chain reaction assays. Both patterns show differentiation between &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; K 30076 and the B-genome of &lt;em&gt;A. hypogaea&lt;/em&gt; from the wild species &lt;em&gt;A. magna&lt;/em&gt; K 30097 and &lt;em&gt;A. batizocoi&lt;/em&gt; K 9484 and the induced allotetraploids MagSten and BatSten1. In (A), two clusters are present: one with &lt;em&gt;A. ipa&#xEB;nsis&lt;/em&gt; and all &lt;em&gt;A. hypogaea&lt;/em&gt; cultivars, and another with the wild species and induced allotetraploids (noted in File S1 as Ah = Ai&#x2260;(Ab = Am = MagSten = BatSten)). In (B), three clusters are present. In these cases, &lt;em&gt;A. hypogaea&lt;/em&gt; forms a different cluster, intermediate in position [noted in File S1 as Ah&#x2260;Ai&#x2260;(Ab = Am = MagSten = BatSten)]. All genotypes with &lt;em&gt;A. batizocoi&lt;/em&gt; derived rust-resistant cluster in different groups to the susceptible genotypes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/7/1403/F5.medium.gif" width="440" height="182"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/7/1403/F5.medium.gif" width="440" height="182"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/7/1403/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/466399" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-38" class="first-child">Screenshots of the two most common examples of <em>Arachis</em> B-genome single-nucleotide polymorphism genotyping using Kompetitive allele-specific polymerase chain reaction assays. Both patterns show differentiation between <em>A. ipaënsis</em> K 30076 and the B-genome of <em>A. hypogaea</em> from the wild species <em>A. magna</em> K 30097 and <em>A. batizocoi</em> K 9484 and the induced allotetraploids MagSten and BatSten1. In (A), two clusters are present: one with <em>A. ipaënsis</em> and all <em>A. hypogaea</em> cultivars, and another with the wild species and induced allotetraploids (noted in <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a> as Ah = Ai≠(Ab = Am = MagSten = BatSten)). In (B), three clusters are present. In these cases, <em>A. hypogaea</em> forms a different cluster, intermediate in position [noted in <a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a> as Ah≠Ai≠(Ab = Am = MagSten = BatSten)]. All genotypes with <em>A. batizocoi</em> derived rust-resistant cluster in different groups to the susceptible genotypes.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-39">In this study, we aimed to generate the information and tools for the introgression of genomic regions of the diploid <em>A. magna</em> K 30097 that confer rust resistance into allotetraploid cultivated peanut, the information being the definition of the QTL that control disease resistance, and the tools being genetic markers to track the genomic regions of interest in a tetraploid genetic context (the context of introgression).</p><p id="p-40">For QTL mapping, we used a progeny derived from a cross between the most likely B genome ancestor of cultivated peanut <em>A. ipaënsis</em> K 30076 (susceptible to rust), and the wild species <em>A. magna</em> K 30097 (resistant to rust). <em>Arachis ipaënsis</em> is very closely related to the B genome of cultivated peanut and has only ever been identified and collected from a single site in the wild. <em>Arachis magna</em> is very closely related to it (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Krapovickas and Gregory 2007</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bechara <em>et al.</em> 2010</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Moretzsohn <em>et al.</em> 2013</a>) and may even be considered the same biological species (J. Valls, personal communication). We have used the progeny from this cross in two previously published studies. In the first, an F<sub>2</sub> progeny was used to produce a map with 149 markers in 10 LGs, spanning a total of 1294 cM (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Moretzsohn <em>et al.</em> 2009</a>). In the second, a higher density genetic map of 798 loci, but covering only 461 cM, was constructed using F<sub>6</sub> RILs generated by single seed descent. This map was integrated into a consensus framework of the separate diploid A and B genome components and tetraploid peanut (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Shirasawa <em>et al.</em> 2013</a>). Here, we used the same F<sub>6</sub> lines and genotyping information from some additional markers to develop a map consisting of 399 high quality markers, none of which cosegregated. This map covered 678 cM. The differences in sizes of these maps, generated from the same parents, seem to be best explained by the different type of software used and parameters that treat genotyping errors in different ways (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bertioli <em>et al.</em> 2014</a>). The largest size (1294 cM) was obtained with Mapmaker without error detection, the intermediate size (678 cM) was obtained by Mapmaker with error detection (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lincoln <em>et al.</em> 1992</a>), and the smallest (461 cM) was obtained with Joinmap, a program that contains its own algorithms that detect error and correct genetic distances. Although it seems clear that the F<sub>2</sub> genetic map distance of 1294 cM must be substantially inflated, it is unclear which of the other two estimates is better. Nevertheless, the map presented here has markers positioned with very high confidence, and, because it contains no cosegregating markers, it is better suited to QTL detection.</p><p id="p-41">The main focus of this study was the identification of disease resistance QTL. However, we also investigated QTL for domestication traits. Because <em>A. ipaënsis</em> and <em>A. magna</em> are both wild species, they differ little for these traits, both having long pegs and PCs and small seeds. However, many segregants present somewhat more cultivated-like characteristics: 38 segregants have shorter pegs, 32 shorter PC, and eight larger seeds than both parents. New allelic combinations conferred traits that were not within the range of the parents. The positioning of these QTL may be of use in localizing cryptic wild alleles that could be used to improve cultivated peanut. However, we can expect that these traits are complex, quantitative, and highly dependent on the environment. Despite this, QTL were surprisingly consistent between years. We can also expect that the effects of these QTL vary with genetic background. Unfortunately, comparison of QTL for similar phenotypes identified by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Foncéka <em>et al.</em> (2012a</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">b</a>) in studies of wild alleles incorporated into a cultivated genetic background is difficult because of difficulty of comparing maps. Furthermore, interpretation of the significance of possible overlaps is also problematic because of the numerous QTL identified in both studies.</p><p id="p-42">For the identification of disease resistance QTL, we evaluated the RILs for four measures of rust susceptibility in two independent bioassays carried out in different years. These evaluations were based on the detached leaf technique that offers precise phenotyping in controlled conditions. We did not use field assays for three reasons: (1) rust occurrence in the field is very patchy, it varies from year to year and it is fragmented within the same field; (2) the usual predominance of either Late Leaf Spot (LLS) or Early Leaf Spot (ELS) complicates evaluation (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Subrahmanyam <em>et al.</em> 1982</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sujay <em>et al.</em> 2012</a>); and, finally, (3) the extreme difference of architecture between wild genotypes and cultivated peanut would have made the field score system meaningless for this study. We anticipate the use of field evaluations later in our research within a backcrossing scheme, when advanced lines with wild segments have similar architecture to the cultivated recurrent parental, then, the occurrence of several diseases will be evaluated concomitantly.</p><p id="p-43">Here, by the use of detached leaf assays, different phenotypes of disease resistance were observed. Some progenies developed no lesions. In others, lesions appeared and then necrotic tissue developed around them in a way that resembled a hypersensitive reaction. These lesions did not sporulate. Furthermore, delayed incubation period also was observed. This finding suggests multiple resistance mechanisms and genes involved in resistance. In total, we detected 13 QTL with LOD scores above the limits established by the permutation tests (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Churchill and Doerge 1994</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Doerge and Churchill 1996</a>), which ranged from 2.5 to 3.2. All alleles that confer resistance were derived from the resistant parent <em>A. magna</em>. They confer less and smaller lesions, lesions with less sporulation, and longer disease incubation period. Although multiple QTL were detected, one particular marker locus (Ah-280) located on LGB08 is linked to QTL associated to all the four components of rust resistance evaluated in both years. The differences between the average phenotypes of RILs that harbor the <em>A. magna</em> allele of this marker compared with those that harbor the <em>A. ipaënsis</em> one are striking (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The effect of <em>A. magna</em> alleles on incubation period appear less pronounced. However, this is attributable to the fact that this can only be measured on a subset of genotypes: the minority that shows symptoms. Because resistant genotypes do not have an incubation period, it is not possible to measure with precision the allelic effect on this trait of the whole range of genotypes. This fact could also explains the high estimates of the proportion of phenotypic variance explained by the QTL detected, which tend to be inflated by the small number of progenies with that particular trait.</p><p id="p-44">Because this QTL confers such strong disease resistance, and it is located at the very end of the chromosome of a species with very close affinity to the B genome of cultivated peanut, it is unlikely to suffer strong linkage drag. Overall it seems like an ideal candidate for the introgression of rust resistance into cultivated peanut. To facilitate this, we developed multiple markers surrounding the locus Ah-280. Also, markers were developed around the locus AHGS1350, which is linked to two strong QTL. To do this, we identified SNPs between <em>A. magna</em> and <em>A. ipaënsis</em> around these loci. Because of the very high similarity of <em>A. ipaënsis</em> to the B genome of <em>A. hypogaea</em>, 19 of 22 KASP markers designed successfully distinguished the rust-resistant wild genotypes and their allotetraploid derivatives from <em>A. ipaënsis</em> and <em>A. hypogaea</em>. For nine of the assays, the <em>A. ipaënsis</em> and <em>A. hypogaea</em> genotypes clustered together; for the other eight, the clusters of the <em>A. hypogaea</em> were shifted to an intermediate position because of the interfering signal generated from the DNA bases on the homeologous A-genome (as described in <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bertioli <em>et al.</em> 2014</a>). This distortion of clustering is easy to account for and does not affect the function of the markers.</p><p id="p-45">Previous studies of rust resistance in pure <em>A. hypogaea</em> have yielded limited information. There are indications that rust-resistance is controlled by a few major genes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Van der Plank 1963</a>). However, a quantitative genetic analysis of parents and progenies indicated that rust resistance is more complex and does not fit a typical race specific pattern (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Subrahmanyam <em>et al.</em> 1993</a>). More recent studies in which authors used germplasm where one of the parents has a small genomic contribution from wild species have been more informative. These have identified a major QTL for rust resistance that explains up to 83% of the phenotypic variance (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Khedikar <em>et al.</em> 2010</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Sujay <em>et al.</em> 2012</a>). The origin of this resistance is almost certainly a chromosome segment derived from the A-genome species <em>A. cardenasii</em>. This segment was likely inherited from the common parent GPBD4, which in turn has a parent ICGV 86855 that is derived from an <em>A. hypogaea</em> × <em>A. cardenasii</em> cross (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gowda <em>et al.</em> 2002</a>). Microsatellite markers on this QTL region currently are being used for marker-assisted backcrossing, with very promising results (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Varshney <em>et al.</em> 2014</a>). The QTL was located in LG AhXV according to a consensus map published for <em>A. hypogaea</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gautami <em>et al.</em> 2012</a>). Sequence similarity searches of the five markers linked to this QTL against the recently available pseudomolecule sequences of <em>A. duranensis</em>, the most probable A-genome ancestral species of cultivated peanut (<a href="http://www.peanutbase.org">www.peanutbase.org</a>; Genbank accession PRJNA258023), indicate that best matches for four of them are at the end of chromosome 3 (Aradu.A03) between 131.3 and 133.7 Mbp (<a href="http://www.g3journal.org/content/suppl/2015/05/05/g3.115.018796.DC1/FileS1.xlsx">File S1</a>, <em>A. cardenasii</em>-rust-QTL-markers). Therefore, this QTL and the one identified in this study would appear to be distinct and could, in principle, both be incorporated into the same peanut cultivar to provide stronger and more durable resistance.</p><p id="p-46">For the introgression of chromosomal segments from <em>A. magna</em> K 30097, an allotetraploid with <em>A. stenosperma</em> ([<em>A. magna</em> × <em>A. stenosperma</em>]<sup>4x</sup>; MagSten) has been created (A. Favero, personal communication) and it is being used in the peanut breeding program in the Instituto Agronômico de Campinas, São Paulo, Brazil, in a collaborative effort with Embrapa Genetic Resources and Biotechnology, Brasília, Brazil (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Leal-Bertioli <em>et al.</em> 2014a</a>). The SSR (Ah-280) and KASP markers described here are now being used for the selection of backcrossed lines that harbor the rust QTL in this breeding program. We anticipate that this will greatly facilitate the testing of function of this chromosomal segment in the genetic background of cultivated peanut.</p><p id="p-47">During the past two decades, the introgression of wild alleles for resistance to pests and diseases has proven very valuable for the peanut crop. Wild alleles confer the strongest known resistance phenotypes not only against rust (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Varshney <em>et al.</em> 2014</a>) but also late leaf spot (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Khedikar <em>et al.</em> 2010</a>) and root-knot nematode (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Stalker <em>et al.</em> 2002</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Simpson <em>et al.</em> 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chu <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burow <em>et al.</em> 2014</a>). These alleles have proven to be stable over different environments and in different genetic backgrounds. The use of alleles harbored on wild chromosome segments also has facilitated the use of molecular markers for backcross selection because the segments have a high rate of DNA polymorphism relative to cultivated peanut. The improved understanding of wild and cultivated species relationships (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Milla <em>et al.</em> 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Seijo <em>et al.</em> 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burow <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Robledo <em>et al.</em> 2009</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Moretzsohn <em>et al.</em> 2013</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Leal-Bertioli <em>et al.</em> 2014b</a>), increased ease of marker development, and better understanding of genome structure that is being gained from genome sequencing of peanut’s diploid ancestors (<a href="http://www.peanutbase.org">www.peanutbase.org</a>) are likely to facilitate the greater use of wild alleles and enable further gains for the peanut crop.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We acknowledge the greenhouse assistance of Leandro F. Mesquita; undergraduate students Aline Santana Pereira, Karinne Dantas, Iugo Galhardo, and Silvio Santos for assistance with phenotyping; Drs. Marcos Gimenes and Andrea Barbosa for providing F<sub>2</sub> plants of the mapping population; and Drs. Alessandra P. Fávero and Ignácio Godoy for providing MagSten DNAs for marker analysis. This work was supported by the Fundação de Apoio a Pesquisa do DF (FAP-DF); the Generation Challenge Program Tropical Legumes 1; National Council for Scientific and Technological Development of Brazil (CNPq); MARS Inc.; The Peanut Foundation; and host institutions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-8"><p id="p-49">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018796/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.018796/-/DC1</a></p></li><li class="fn" id="fn-9"><p id="p-50"><em>Communicating editor: D. Zamir</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-03-17"><span class="received-label">Received </span>March 17, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-04-29"><span class="accepted-label">Accepted </span>April 29, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Leal-Bertioli <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bechara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmieri</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monteiro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bacci</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phylogenetic relationships in genus <em>Arachis</em> based on ITS and 5.8S rDNA sequences.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DBechara%26rft.auinit1%253DM.%2BD.%26rft.volume%253D10%26rft.spage%253D255%26rft.epage%253D255%26rft.atitle%253DPhylogenetic%2Brelationships%2Bin%2Bgenus%2BArachis%2Bbased%2Bon%2BITS%2Band%2B5.8S%2BrDNA%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F21092103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21092103&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-other" id="cit-5.7.1403.2" data-doi="10.1534/g3.113.007617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D.J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozias-Akins</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dantas</span>  <span class="cit-name-given-names">K.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">S.P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The use of SNP markers for linkage mapping in diploid and tetraploid peanuts.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">4</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007617%26rft_id%253Dinfo%253Apmid%252F24212082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjQvMS84OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xNDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.3" data-doi="10.1139/G08-094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faries</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular biogeography study of recently described B- and A-genome <em>Arachis</em> species, also providing new insights into the origins of cultivated peanut.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBurow%26rft.auinit1%253DM.%2BD.%26rft.volume%253D52%26rft.issue%253D2%26rft.spage%253D107%26rft.epage%253D119%26rft.atitle%253DMolecular%2Bbiogeographic%2Bstudy%2Bof%2Brecently%2Bdescribed%2BB-%2Band%2BA-genome%2BArachis%2Bspecies%252C%2Balso%2Bproviding%2Bnew%2Binsights%2Binto%2Bthe%2Borigins%2Bof%2Bcultivated%2Bpeanut.%26rft_id%253Dinfo%253Adoi%252F10.1139%252FG08-094%26rft_id%253Dinfo%253Apmid%252F19234559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/G08-094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19234559&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Introgression of homeologous quantitative trait loci (QTLs) for resistance to the root-knot nematode [<em>Meloidogyne arenaria</em> (Neal) Chitwood] in an advanced backcross-QTL population of peanut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-5.7.1403.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">C.C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tillman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Person</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Marker-assisted selection to pyramid nematode resistance and the high oleic trait in peanut.</span> <span class="cit-source">The Plant Genome</span> <span class="cit-vol">4</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Permutation tests for multiple loci affecting a quantitative character.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDoerge%26rft.auinit1%253DR.%2BW.%26rft.volume%253D142%26rft.issue%253D1%26rft.spage%253D285%26rft.epage%253D294%26rft.atitle%253DPermutation%2BTests%2Bfor%2BMultiple%2BLoci%2BAffecting%2Ba%2BQuantitative%2BCharacter%26rft_id%253Dinfo%253Apmid%252F8770605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQyLzEvMjg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.8" data-doi="10.1093/nar/gkt1381"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duitama</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintero</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubmann</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An integrated framework for discovery and genotyping of genomic variants from high-throughput sequencing experiments.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">e44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1381%26rft_id%253Dinfo%253Apmid%252F24413664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQyLzYvZTQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fávero</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moraes</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J. F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vello</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Characterization of rust, early and late leaf spot resistance in wild and cultivated peanut germplasm.</span> <abbr class="cit-jnl-abbrev">Scientia Agricolae</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientia%2BAgricolae%26rft.volume%253D66%26rft.spage%253D110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-other" id="cit-5.7.1403.10" data-doi="10.1371/journal.pone.0048642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foncéka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tossim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivallan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vignes</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacut</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Construction of chromosome segment substitution lines in peanut (Arachis hypogaea L.) using a wild synthetic and QTL mapping for plant morphology.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">7</span>: <span class="cit-fpage">e48642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFonceka%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De48642%26rft.epage%253De48642%26rft.atitle%253DConstruction%2Bof%2BChromosome%2BSegment%2BSubstitution%2BLines%2Bin%2BPeanut%2B%2528Arachis%2Bhypogaea%2BL.%2529%2BUsing%2Ba%2BWild%2BSynthetic%2Band%2BQTL%2BMapping%2Bfor%2BPlant%2BMorphology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048642%26rft_id%253Dinfo%253Apmid%252F23185268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185268&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.11" data-doi="10.1186/1471-2229-12-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foncéka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tossim</span>  <span class="cit-name-given-names">H.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivallan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vignes</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faye</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Fostered and left behind alleles in peanut: interspecific QTL mapping reveals footprints of domestication and useful natural variation for breeding.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DFonceka%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DFostered%2Band%2Bleft%2Bbehind%2Balleles%2Bin%2Bpeanut%253A%2Binterspecific%2BQTL%2Bmapping%2Breveals%2Bfootprints%2Bof%2Bdomestication%2Band%2Buseful%2Bnatural%2Bvariation%2Bfor%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-26%26rft_id%253Dinfo%253Apmid%252F22340522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22340522&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.12" data-doi="10.1371/journal.pone.0041213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautami</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foncéka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sujay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e41213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGautami%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41213%26rft.epage%253De41213%26rft.atitle%253DAn%2Binternational%2Breference%2Bconsensus%2Bgenetic%2Bmap%2Bwith%2B897%2Bmarker%2Bloci%2Bbased%2Bon%2B11%2Bmapping%2Bpopulations%2Bfor%2Btetraploid%2Bgroundnut%2B%2528Arachis%2Bhypogaea%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041213%26rft_id%253Dinfo%253Apmid%252F22815973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22815973&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.7.1403.13"><div class="cit-metadata unstructured">Gibbons, R. W., 1980 Groundnut improvement research technology for the semi-arid tropics, pp. 27–37 in <em>Proceedings of the International Symposium on Development and Transfer of Technology for Rain-fed Agriculture and the SAT Farmer</em>, ICRISAT, India.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1403.14"><div class="cit-metadata unstructured">Gowda, M. V. C., B. N. Motagi, G. K. Naidu, S. B. Diddimani, and R. Sheshagiri, 2002 GPBD 4: a Spanish bunch groundnut genotype resistant to rust and late leaf spot, pp. 29–32 in International Arachis Newsletter.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sederoff</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrattapaglia%26rft.auinit1%253DD.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D1121%26rft.epage%253D1137%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bof%2BEucalyptus%2Bgrandis%2Band%2BEucalyptus%2Burophylla%2Busing%2Ba%2Bpseudo-testcross%253A%2Bmapping%2Bstrategy%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7982566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNy80LzExMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</span>. <abbr class="cit-jnl-abbrev">Nat. Prot</abbr>. <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProt%26rft.volume%253D8%26rft.spage%253D1494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1403.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hammons</span>  <span class="cit-name-given-names">R. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Early history and origin of the peanut</span>, pp. <span class="cit-fpage">17</span>–<span class="cit-lpage">46</span> in <span class="cit-source">Peanuts: <em>Culture</em> and <em>Uses</em></span>. <span class="cit-publ-name">American Peanut Research and Education Society</span>, <span class="cit-publ-loc">Stillwater, OK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timper</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culbreath</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvien</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Registration of ‘Tifguard’ Peanut.</span> <abbr class="cit-jnl-abbrev">J. Plant Reg.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BReg.%26rft.volume%253D2%26rft.spage%253D92%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.19" data-doi="10.1007/s00122-010-1366-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khedikar</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarvamangala</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patgar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A QTL study on late leaf spot and rust revealed one major QTL for molecular breeding for rust resistance in groundnut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">971</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKhedikar%26rft.auinit1%253DY.%2BP.%26rft.volume%253D121%26rft.issue%253D5%26rft.spage%253D971%26rft.epage%253D984%26rft.atitle%253DA%2BQTL%2Bstudy%2Bon%2Blate%2Bleaf%2Bspot%2Band%2Brust%2Brevealed%2Bone%2Bmajor%2BQTL%2Bfor%2Bmolecular%2Bbreeding%2Bfor%2Brust%2Bresistance%2Bin%2Bgroundnut%2B%2528Arachis%2Bhypogaea%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1366-x%26rft_id%253Dinfo%253Apmid%252F20526757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1366-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20526757&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Taxonomy of the genus <em>Arachis</em> (Leguminosae)</span>. <abbr class="cit-jnl-abbrev">Bonplandia</abbr> <span class="cit-vol">16</span>(<span class="cit-supplement">Suppl</span>)<strong>:</strong> <span class="cit-fpage">1</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBonplandia%26rft.volume%253D16%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.21" data-doi="10.1016/0888-7543(87)90010-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrahamson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">MAPMAKER: an interactive computer package for constructing primary genetic maps of experimental and natural populations.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D181%26rft.atitle%253DMAPMAKER%253A%2Ban%2Binteractive%2Bcomputer%2Bpackage%2Bfor%2Bconstructing%2Bprimary%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bexperimental%2Band%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252887%252990010-3%26rft_id%253Dinfo%253Apmid%252F3692487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(87)90010-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3692487&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.22" data-doi="10.1186/1471-2229-9-112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jose</span>  <span class="cit-name-given-names">A. C. V. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves-Freitas</span>  <span class="cit-name-given-names">D. M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimaraes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of candidate genome regions controlling disease resistance in <em>Arachis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DLeal-Bertioli%26rft.auinit1%253DS.%2BC.%26rft.volume%253D9%26rft.spage%253D112%26rft.epage%253D112%26rft.atitle%253DIdentification%2Bof%2Bcandidate%2Bgenome%2Bregions%2Bcontrolling%2Bdisease%2Bresistance%2Bin%2BArachis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-112%26rft_id%253Dinfo%253Apmid%252F19698131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19698131&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.23" data-doi="10.1111/j.1439-0434.2010.01704.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farias</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guimarães</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brasileiro</span>  <span class="cit-name-given-names">A. C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ultrastructure of the initial interaction of <em>Puccinia arachidis</em> and <em>Cercosporidium personatum</em> with leaves of <em>Arachis hypogaea</em> and <em>Arachis stenosperma</em>.</span> <abbr class="cit-jnl-abbrev">J. Phytopathol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">792</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhytopathol.%26rft.volume%253D158%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0434.2010.01704.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0434.2010.01704.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.7.1403.24"><div class="cit-metadata unstructured">Leal-Bertioli, S. C. M., M. C. Moretzsohn, P. Ozias-Akins, Y. Chu, C. Holbrook <em>et al.</em>, 2014a The use of the diploid Arachis genomes to aid introgression of wild segments into Peanut in <em>7th International Conference on Advances in Arachis through Genomics &amp; Biotechnology - AAGB</em>., Savannah, GA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dantas</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inglis</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title"><em>Arachis batizocoi:</em> a study of its relationship to cultivated peanut (<em>A. hypogaea</em>) and its potential for introgression of wild genes into the peanut crop using induced allotetraploids.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft.volume%253D115%26rft.spage%253D237%26rft_id%253Dinfo%253Apmid%252F25538110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25538110&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.26" data-doi="10.1534/genetics.115.174607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abernathy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavarro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tetrasomic recombination is surprisingly frequent in allotetraploid <em>Arachis</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174607%26rft_id%253Dinfo%253Apmid%252F25701284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OS80LzEwOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1403.27"><div class="cit-metadata unstructured">Lincoln, S. E., M. J. Daly, and E. S. Lander, 1992 Mapping genes controlling quantitative traits with MAPMAKER/EXP3.0, pp. XXX-XXX in Whitehead Institute Technical Report. Ed. 3. Cambridge, MA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milla</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isleib</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Taxonomic relationships among <em>Arachis</em> sect. <em>Arachis</em> species as revealed by AFLP markers.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DMilla%26rft.auinit1%253DS.%2BR.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DTaxonomic%2Brelationships%2Bamong%2BArachis%2Bsect.%2BArachis%2Bspecies%2Bas%2Brevealed%2Bby%2BAFLP%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F15729391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15729391&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.29" data-doi="10.1093/bioinformatics/btq580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardle</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Flapjack—graphical genotype visualization.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3133</span>–<span class="cit-lpage">3134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq580%26rft_id%253Dinfo%253Apmid%252F20956241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMjQvMzEzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xNDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moraes</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salgado</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Utilização da técnica de folhas destacadas de amendoim (<em>Arachis hypogaea</em> L.) para inoculações com <em>Cercospora arachidicola</em> Hori e <em>Cercospora personata</em> (Bert. &amp; Curt.)</span>. <abbr class="cit-jnl-abbrev">Ell. Ev. Summa Phytopathologica</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEll.%2BEv.%2BSumma%2BPhytopathologica%26rft.volume%253D8%26rft.spage%253D39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.31" data-doi="10.1186/1471-2229-9-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves-Freitas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A linkage map for the B-genome of <em>Arachis</em> (Fabaceae) and its synteny to the A-genome.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DMoretzsohn%26rft.auinit1%253DM.%2BC.%26rft.volume%253D9%26rft.spage%253D40%26rft.epage%253D40%26rft.atitle%253DA%2Blinkage%2Bmap%2Bfor%2Bthe%2BB-genome%2Bof%2BArachis%2B%2528Fabaceae%2529%2Band%2Bits%2Bsynteny%2Bto%2Bthe%2BA-genome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-40%26rft_id%253Dinfo%253Apmid%252F19351409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19351409&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.32" data-doi="10.1093/aob/mcs237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouvea</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inglis</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valls</span>  <span class="cit-name-given-names">J. F. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A study of the relationships of cultivated peanut (<em>Arachis hypogaea</em>) and its most closely related wild species using intron sequences and microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcs237%26rft_id%253Dinfo%253Apmid%252F23131301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjExMS8xLzExMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzUvNy8xNDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Resistance of wild Arachis species to late leaf spot and rust in greenhouse trials.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D85%26rft.spage%253D851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.34" data-doi="10.1007/s00122-009-0981-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Species relations among wild <em>Arachis</em> species with the A genome as revealed by FISH mapping of rDNA loci and heterochromatin detection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRobledo%26rft.auinit1%253DG.%26rft.volume%253D118%26rft.issue%253D7%26rft.spage%253D1295%26rft.epage%253D1307%26rft.atitle%253DSpecies%2Brelations%2Bamong%2Bwild%2BArachis%2Bspecies%2Bwith%2Bthe%2BA%2Bgenome%2Bas%2Brevealed%2Bby%2BFISH%2Bmapping%2Bof%2BrDNA%2Bloci%2Band%2Bheterochromatin%2Bdetection.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-0981-x%26rft_id%253Dinfo%253Apmid%252F19234686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-0981-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19234686&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265443000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savary</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subba Rao</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zadoksa</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Scale of reaction types of groundnut to <em>Puccinia arachidis</em> Speg</span>. <abbr class="cit-jnl-abbrev">J. Phytopathol</abbr>. <span class="cit-vol">124</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhytopathol%26rft.volume%253D124%26rft.spage%253D259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.36" data-doi="10.3732/ajb.94.12.1963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seijo</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavia</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krapovickas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducasse</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic relationships between the cultivated peanut (<em>Arachis hypogaea</em>, Leguminosae) and its close relatives revealed by double GISH.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">1963</span>–<span class="cit-lpage">1971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DSeijo%26rft.auinit1%253DG.%26rft.volume%253D94%26rft.issue%253D12%26rft.spage%253D1963%26rft.epage%253D1971%26rft.atitle%253DGenomic%2Brelationships%2Bbetween%2Bthe%2Bcultivated%2Bpeanut%2B%2528Arachis%2Bhypogaea%252C%2BLeguminosae%2529%2Band%2Bits%2Bclose%2Brelatives%2Brevealed%2Bby%2Bdouble%2BGISH%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.94.12.1963%26rft_id%253Dinfo%253Apmid%252F21636391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiI5NC8xMi8xOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.37" data-doi="10.1093/dnares/dss042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertioli</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moretzsohn</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leal-Bertioli</span>  <span class="cit-name-given-names">S. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An integrated consensus map of cultivated peanut and wild relatives revealsed structures of the A and B genomes of <em>Arachis</em> and divergences with other of the legume genomes.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdss042%26rft_id%253Dinfo%253Apmid%252F23315685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNS83LzE0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.38" data-doi="10.2135/cropsci2001.413918x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Registration of ‘COAN’ peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSimpson%26rft.auinit1%253DC.E.%26rft.volume%253D41%26rft.issue%253D3%26rft.spage%253D918%26rft.epage%253D918%26rft.atitle%253DRegistration%2Bof%2B%2527COAN%2527%2BPeanut%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2001.413918x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2001.413918x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170696200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.39" data-doi="10.2135/cropsci2003.1561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starr</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burow</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Registration of “NemaTAM” Peanut.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DSimpson%26rft.auinit1%253DC.E.%26rft.volume%253D43%26rft.issue%253D4%26rft.spage%253D1561%26rft.epage%253D1561%26rft.atitle%253DRegistration%2Bof%2B%2527NemaTAM%2527%2BPeanut%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1561&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000183762800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Registration of four insect-resistant peanut germplasm lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DStalker%26rft.auinit1%253DH.T.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D313%26rft.epage%253D314%26rft.atitle%253DRegistration%2Bof%2BFour%2BInsect-Resistant%2BPeanut%2BGermplasm%2BLines.%26rft_id%253Dinfo%253Apmid%252F11756307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11756307&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.41" data-doi="10.2135/cropsci2002.0312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stalker</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beute</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shew</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Registration of two root-knot nematode-resistant peanut germplasm lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DStalker%26rft.auinit1%253DH.T.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D312%26rft.epage%253D313%26rft.atitle%253DRegistration%2Bof%2BTwo%2BRoot-Knot%2BNematode-Resistant%2BPeanut%2BGermplasm%2BLines.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2002.0312%26rft_id%253Dinfo%253Apmid%252F11756306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2002.0312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11756306&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173296000060&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subrahmanyam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigam</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevill</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Resistance to rust and late leafspot diseases in some genotypes of <em>Arachis hypogaea</em>.</span> <abbr class="cit-jnl-abbrev">Peanut Sci</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeanut%2BSci%26rft.volume%253D9%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subrahmanyam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moss</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbons</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Origins of resistance to rust and late leaf spot in peanut (<em>Arachis hypogaea</em>) Fabaceae.</span> <abbr class="cit-jnl-abbrev">Econ. Bot.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcon.%2BBot.%26rft.volume%253D43%26rft.spage%253D444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1403.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Subrahmanyam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigam</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Origin and utilization of rust resistance in groundnut</span>, pp. <span class="cit-fpage">147</span>–<span class="cit-lpage">158</span> in <span class="cit-source">Durability of Disease Resistance</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Parlevliet</span>  <span class="cit-name-given-names">J. E.</span></span>. <span class="cit-publ-name">Kluwer Academic Publishers</span>, <span class="cit-publ-loc">Dordrecht, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.45" data-doi="10.1007/s11032-011-9661-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sujay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M. V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khedikar</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative trait locus analysis and construction of consensus genetic map for foliar disease resistance based on two recombinant inbred line populations in cultivated groundnut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBreed%26rft.aulast%253DSujay%26rft.auinit1%253DV.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D773%26rft.epage%253D788%26rft.atitle%253DQuantitative%2Btrait%2Blocus%2Banalysis%2Band%2Bconstruction%2Bof%2Bconsensus%2Bgenetic%2Bmap%2Bfor%2Bfoliar%2Bdisease%2Bresistance%2Bbased%2Bon%2Btwo%2Brecombinant%2Binbred%2Bline%2Bpopulations%2Bin%2Bcultivated%2Bgroundnut%2B%2528Arachis%2Bhypogaea%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-011-9661-z%26rft_id%253Dinfo%253Apmid%252F22924018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-011-9661-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22924018&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-5.7.1403.46"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The FAOSTAT System</span>, <span class="cit-pub-date">2012</span> <a href="http://faostat3.fao.org/browse/FB/BC/E">http://faostat3.fao.org/browse/FB/BC/E</a>. Accessed May 15, 2015.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1403.47"><div class="cit-metadata unstructured">Vale, F. X. R., E. Fernandes Filho, and J. R. Liberato, 2001 <em>Quant - Quantificação de doenças</em>.: versão 1.0.1., pp. 1.0.1. UFV, Viçosa.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1403.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Plank</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-source">Plant Diseases: Epidemics and Control</span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshney</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janila</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigam</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudini</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Marker-assisted introgression of a QTL region to improve rust resistance in three elite and popular varieties of peanut (<em>Arachis hypogaea</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1771</span>–<span class="cit-lpage">1781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1771%26rft_id%253Dinfo%253Apmid%252F24927821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24927821&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F7%2F1403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.50" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-5.7.1403.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Windows QTL Cartographer 2.5.</span> <span class="cit-publ-name">Department of Statistics, North Carolina State University</span>, <span class="cit-publ-loc">Raleigh, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.52" data-doi="10.1073/pnas.90.23.10972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">10972</span>–<span class="cit-lpage">10976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D90%26rft.issue%253D23%26rft.spage%253D10972%26rft.epage%253D10976%26rft.atitle%253DTheoretical%2Bbasis%2Bfor%2Bseparation%2Bof%2Bmultiple%2Blinked%2Bgene%2Beffects%2Bin%2Bmapping%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.90.23.10972%26rft_id%253Dinfo%253Apmid%252F8248199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTAvMjMvMTA5NzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.7.1403.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.%2BB.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1457%26rft.epage%253D1468%26rft.atitle%253DPrecision%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8013918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0NTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy81LzcvMTQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/7/1403.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/7/1391" title="Composite Selection Signals for Complex Traits Exemplified Through Bovine Stature Using Multibreed Cohorts of European and African Bos taurus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/7/1415" title="XSim: Simulation of Descendants from Ancestors with Sequence Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/7"><span>Volume 5 Issue 7, July 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/7.cover-source.jpg?itok=IZbYpE05" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/7/1403" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-a0cFAJFAOb5HkpXW3r2kloOXcfE-u9wO4LmYVgpE83g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/7/1403" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/7/1403" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/7/1403" class="active" data-icon-position="" data-hide-link-title="0">Identification of QTLs for Rust Resistance in the Peanut Wild Species Arachis magna and the Development of KASP Markers for Marker-Assisted Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459990" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-U1BK2xSEzbsQoGLYFxz1CAF3bW7Fby4bF4pEmXuc2Po" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1403.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/7/1403" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-fPlt3igzelMf8rwdjORrNXX9ruxl_3xz5883v6xK72E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/7/1403" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/7/1403.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459990" id="node459990--2" data-pisa="ggg;5/7/1403" data-pisa-master="ggg;g3.115.018796" data-apath="/ggg/5/7/1403.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of QTLs for Rust Resistance in the Peanut Wild Species <em>Arachis magna</em> and the Development of KASP Markers for Marker-Assisted Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Soraya C. M. Leal-Bertioli</span>, <span class="highwire-citation-author" data-delta="1">Uiara Cavalcante</span>, <span class="highwire-citation-author" data-delta="2">Ediene G. Gouvea</span>, <span class="highwire-citation-author" data-delta="3">Carolina Ballén-Taborda</span>, <span class="highwire-citation-author" data-delta="4">Kenta Shirasawa</span>, <span class="highwire-citation-author" data-delta="5">Patrícia M. Guimarães</span>, <span class="highwire-citation-author" data-delta="6">Scott A. Jackson</span>, <span class="highwire-citation-author" data-delta="7">David J. Bertioli</span> and <span class="highwire-citation-author" data-delta="8">Márcio C. Moretzsohn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1403-1413;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018796 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459990--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Soraya C. M. Leal-Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Soraya%2BC.%2BM.%2BLeal-Bertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leal-Bertioli%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASoraya%2BC.%2BM.%2BLeal-Bertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:soraya.bertioli@embrapa.br" class="" data-icon-position="" data-hide-link-title="0">soraya.bertioli@embrapa.br</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Uiara Cavalcante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Uiara%2BCavalcante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavalcante%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUiara%2BCavalcante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ediene G. Gouvea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ediene%2BG.%2BGouvea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouvea%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdiene%2BG.%2BGouvea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Carolina Ballén-Taborda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBall%C3%A9n-Taborda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ball%C3%A9n-Taborda%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BBall%25C3%25A9n-Taborda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenta Shirasawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Kazusa DNA Research Institute, Kisarazu, Chiba, 292-0818, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenta%2BShirasawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirasawa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenta%2BShirasawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrícia M. Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patr%C3%ADcia%2BM.%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatr%25C3%25ADcia%2BM.%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Scott A. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BA.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BA.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David J. Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BBertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertioli%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BBertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Márcio C. Moretzsohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A1rcio%2BC.%2BMoretzsohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moretzsohn%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM%25C3%25A1rcio%2BC.%2BMoretzsohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/7/1403" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/7/1403&amp;title=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/7/1403&amp;title=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/7/1403&amp;title=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/7/1403&amp;text=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/7/1403&amp;title=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/7/1403&amp;t=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/7/1403&amp;title=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/7/1403&amp;title=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459990/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459990" id="node459990--3" data-pisa="ggg;5/7/1403" data-pisa-master="ggg;g3.115.018796" data-apath="/ggg/5/7/1403.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of QTLs for Rust Resistance in the Peanut Wild Species <em>Arachis magna</em> and the Development of KASP Markers for Marker-Assisted Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Soraya C. M. Leal-Bertioli</span>, <span class="highwire-citation-author" data-delta="1">Uiara Cavalcante</span>, <span class="highwire-citation-author" data-delta="2">Ediene G. Gouvea</span>, <span class="highwire-citation-author" data-delta="3">Carolina Ballén-Taborda</span>, <span class="highwire-citation-author" data-delta="4">Kenta Shirasawa</span>, <span class="highwire-citation-author" data-delta="5">Patrícia M. Guimarães</span>, <span class="highwire-citation-author" data-delta="6">Scott A. Jackson</span>, <span class="highwire-citation-author" data-delta="7">David J. Bertioli</span> and <span class="highwire-citation-author" data-delta="8">Márcio C. Moretzsohn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1403-1413;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.018796 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459990--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Soraya C. M. Leal-Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Soraya%2BC.%2BM.%2BLeal-Bertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leal-Bertioli%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASoraya%2BC.%2BM.%2BLeal-Bertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:soraya.bertioli@embrapa.br" class="" data-icon-position="" data-hide-link-title="0">soraya.bertioli@embrapa.br</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Uiara Cavalcante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Uiara%2BCavalcante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cavalcante%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUiara%2BCavalcante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ediene G. Gouvea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ediene%2BG.%2BGouvea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouvea%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdiene%2BG.%2BGouvea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Carolina Ballén-Taborda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BBall%C3%A9n-Taborda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ball%C3%A9n-Taborda%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BBall%25C3%25A9n-Taborda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kenta Shirasawa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Kazusa DNA Research Institute, Kisarazu, Chiba, 292-0818, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenta%2BShirasawa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shirasawa%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenta%2BShirasawa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Patrícia M. Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patr%C3%ADcia%2BM.%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20PM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatr%25C3%25ADcia%2BM.%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Scott A. Jackson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Scott%2BA.%2BJackson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jackson%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AScott%2BA.%2BJackson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">David J. Bertioli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Applied Genetic Technologies, University of Georgia, Athens, Georgia 30602-6810</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>University of Brasília, Institute of Biological Sciences, Campus Darcy Ribeiro, 70910-900 Brasília, DF, Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BBertioli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertioli%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BBertioli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Márcio C. Moretzsohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Embrapa Genetic Resources and Biotechnology, 70770-917 Brasília, DF, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A1rcio%2BC.%2BMoretzsohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moretzsohn%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM%25C3%25A1rcio%2BC.%2BMoretzsohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1403&amp;rft.epage=1413&amp;rft.atitle=Identification%20of%20QTLs%20for%20Rust%20Resistance%20in%20the%20Peanut%20Wild%20Species%20Arachis%20magna%20and%20the%20Development%20of%20KASP%20Markers%20for%20Marker-Assisted%20Selection&amp;rft.volume=5&amp;rft.issue=7&amp;rft.date=2015-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Leal-Bertioli%2C+Soraya+C.+M.&amp;rft.au=Cavalcante%2C+Uiara&amp;rft.au=Gouvea%2C+Ediene+G.&amp;rft.au=Ball%C3%A9n-Taborda%2C+Carolina&amp;rft.au=Shirasawa%2C+Kenta&amp;rft.au=Guimar%C3%A3es%2C+Patr%C3%ADcia+M.&amp;rft.au=Jackson%2C+Scott+A.&amp;rft.au=Bertioli%2C+David+J.&amp;rft.au=Moretzsohn%2C+M%C3%A1rcio+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459990/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459990/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459990/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459990/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459990/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459990/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459990/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459990/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459990/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459990/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459990/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459990/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/7/1403" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/7/1403.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/7/1403.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/7/1403.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/7/1403.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
