<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/4/665" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/4/665.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/4/665.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/4/665.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/4/665/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/4/665.atom" />
<meta name="HW.pisa" content="ggg;3/4/665" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="ZRT1 Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.112.005082" />
<meta name="DC.Date" content="2013-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Engle, Fay. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Estimates of the fraction of nucleotide substitutions driven by positive selection vary widely across different species. Accounting for different estimates of positive selection has been difficult, in part because selection on polymorphism within a species is known to obscure a signal of positive selection among species. While methods have been developed to control for the confounding effects of negative selection against deleterious polymorphism, the impact of balancing selection on estimates of positive selection has not been assessed. In Saccharomyces cerevisiae , there is no signal of positive selection within protein coding sequences as the ratio of nonsynonymous to synonymous polymorphism is higher than that of divergence. To investigate the impact of balancing selection on estimates of positive selection, we examined five genes with high rates of nonsynonymous polymorphism in S. cerevisiae relative to divergence from S. paradoxus . One of the genes, the high-affinity zinc transporter [ZRT1][1] showed an elevated rate of synonymous polymorphism indicative of balancing selection. The high rate of synonymous polymorphism coincided with nonsynonymous divergence among three haplotype groups, among which we found no detectable differences in [ZRT1][1] function. Our results implicate balancing selection in one of five genes exhibiting a large excess of nonsynonymous polymorphism in yeast. We conclude that balancing selection is a potentially important factor in estimating the frequency of positive selection across the yeast genome.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224" />
<meta name="DC.Contributor" content="Elizabeth K. Engle" />
<meta name="DC.Contributor" content="Justin C. Fay" />
<meta name="article:published_time" content="2013-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="ZRT1 Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Estimates of the fraction of nucleotide substitutions driven by positive selection vary widely across different species. Accounting for different estimates of positive selection has been difficult, in part because selection on polymorphism within a species is known to obscure a signal of positive selection among species. While methods have been developed to control for the confounding effects of negative selection against deleterious polymorphism, the impact of balancing selection on estimates of positive selection has not been assessed. In &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, there is no signal of positive selection within protein coding sequences as the ratio of nonsynonymous to synonymous polymorphism is higher than that of divergence. To investigate the impact of balancing selection on estimates of positive selection, we examined five genes with high rates of nonsynonymous polymorphism in &lt;i&gt;S. cerevisiae&lt;/i&gt; relative to divergence from &lt;i&gt;S. paradoxus&lt;/i&gt;. One of the genes, the high-affinity zinc transporter &lt;i&gt;ZRT1&lt;/i&gt; showed an elevated rate of synonymous polymorphism indicative of balancing selection. The high rate of synonymous polymorphism coincided with nonsynonymous divergence among three haplotype groups, among which we found no detectable differences in &lt;i&gt;ZRT1&lt;/i&gt; function. Our results implicate balancing selection in one of five genes exhibiting a large excess of nonsynonymous polymorphism in yeast. We conclude that balancing selection is a potentially important factor in estimating the frequency of positive selection across the yeast genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/04/01" />
<meta name="citation_mjid" content="ggg;3/4/665" />
<meta name="citation_id" content="3/4/665" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/4/665" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/4/665.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/4/665.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/4/665.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005082" />
<meta name="citation_pmid" content="23550117" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="665" />
<meta name="citation_lastpage" content="673" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Elizabeth K. Engle" />
<meta name="citation_author_institution" content="Molecular Genetics and Genomics Program, Washington University, St. Louis, Missouri 63108" />
<meta name="citation_author" content="Justin C. Fay" />
<meta name="citation_author_institution" content="Department of Genetics and Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108" />
<meta name="citation_author_email" content="jfay@genetics.wustl.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Andolfatto;citation_author=M. Nordborg;citation_title=The Effect of Gene Conversion on Intralocus Associations;citation_pages=1397-1399;citation_volume=148;citation_year=1998;citation_issue=3;citation_pmid=9539452" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=D. Bachtrog;citation_title=Similar rates of protein adaptation in Drosophila miranda and D. melanogaster, two species with different current effective population sizes.;citation_pages=334-334;citation_volume=8;citation_year=2008;citation_pmid=19091130;citation_doi=10.1186/1471-2148-8-334" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Bierne;citation_author=A. Eyre-Walker;citation_title=The genomic rate of adaptive amino acid substitution in Drosophila;citation_volume=21;citation_year=2004;citation_pmid=15044594;citation_doi=10.1093/molbev/msh134" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=A. J. Bird;citation_author=M. Gordon;citation_author=D. J. Eide;citation_author=D. R. Winge;citation_title=Repression of ADH1 and ADH3 during zinc deficiency by Zap1-induced intergenic RNA transcripts.;citation_pages=5726-5734;citation_volume=25;citation_year=2006;citation_issue=24;citation_pmid=17139254;citation_doi=10.1038/sj.emboj.7601453" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=C. A. Brown;citation_author=A. W. Murray;citation_author=K. J. Verstrepen;citation_title=Rapid expansion and functional divergence of subtelomeric gene families in yeasts.;citation_pages=895-903;citation_volume=20;citation_year=2010;citation_issue=10;citation_pmid=20471265;citation_doi=10.1016/j.cub.2010.04.027" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=M. Nordborg;citation_author=D. Charlesworth;citation_title=The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.;citation_pages=155-174;citation_volume=70;citation_year=1997;citation_issue=2;citation_pmid=9449192;citation_doi=10.1017/S0016672397002954" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Charlesworth;citation_author=A. Eyre-Walker;citation_title=The McDonald-Kreitman test and slightly deleterious mutations;citation_volume=25;citation_year=2008;citation_pmid=18195052;citation_doi=10.1093/molbev/msn005" />
<meta name="citation_reference" content="citation_journal_title=International journal of food microbiology;citation_journal_abbrev=Int J Food Microbiol;citation_author=M. Combina;citation_author=A. Elía;citation_author=L. Mercado;citation_author=C. Catania;citation_author=A. Ganga;citation_title=Dynamics of indigenous yeast populations during spontaneous fermentation of wines from Mendoza, Argentina.;citation_pages=237-243;citation_volume=99;citation_year=2005;citation_issue=3;citation_pmid=15808358;citation_doi=10.1016/j.ijfoodmicro.2004.08.017" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. D. Cutter;citation_author=A. M. Moses;citation_title=Polymorphism, divergence, and the role of recombination in Saccharomyces cerevisiae genome evolution;citation_volume=28;citation_year=2011;citation_pmid=21199893;citation_doi=10.1093/molbev/msq356" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Brew. Chem.;citation_author=R. De Nicola;citation_author=G. Walker;citation_title=Zinc interactions with brewing yeast: impact on fermentation performance;citation_pages=214-219;citation_volume=69;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=R. De Nicola;citation_author=L. A. Hazelwood;citation_author=E. A. F. De Hulster;citation_author=M. C. Walsh;citation_author=T. A. Knijnenburg;citation_title=Physiological and transcriptional responses of Saccharomyces cerevisiae to zinc limitation in chemostat cultures;citation_volume=73;citation_year=2007;citation_pmid=17933919;citation_doi=10.1128/AEM.01445-07" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. W. Doniger;citation_author=H. S. Kim;citation_author=D. Swain;citation_author=D. Corcuera;citation_author=M. Williams;citation_title=A catalog of neutral and deleterious polymorphism in yeast.;citation_pages=e1000183-e1000183;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18769710;citation_doi=10.1371/journal.pgen.1000183" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Elyashiv;citation_author=K. Bullaughey;citation_author=S. Sattath;citation_author=Y. Rinott;citation_author=M. Przeworski;citation_title=Shifts in the intensity of purifying selection: an analysis of genome-wide polymorphism data from two closely related yeast species;citation_volume=20;citation_year=2010;citation_pmid=20817943;citation_doi=10.1101/gr.108993.110" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. K. Engle;citation_author=J. C. Fay;citation_title=Divergence of the yeast transcription factor FZF1 affects sulfite resistance.;citation_pages=e1002763-e1002763;citation_volume=8;citation_year=2012;citation_issue=6;citation_pmid=22719269;citation_doi=10.1371/journal.pgen.1002763" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Eyre-Walker;citation_title=Changing Effective Population Size and the McDonald-Kreitman Test;citation_pages=2017-2024;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524367" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Eyre-Walker;citation_author=P. D. Keightley;citation_title=Estimating the rate of adaptive molecular evolution in the presence of slightly deleterious mutations and population size change;citation_volume=26;citation_year=2009;citation_pmid=19535738;citation_doi=10.1093/molbev/msp119" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. C. Fay;citation_title=Weighing the evidence for adaptation at the molecular level.;citation_pages=343-349;citation_volume=27;citation_year=2011;citation_issue=9;citation_pmid=21775012;citation_doi=10.1016/j.tig.2011.06.003" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=J. C. Fay;citation_author=C. I. Wu;citation_title=The neutral theory in the genomic era.;citation_pages=642-646;citation_volume=11;citation_year=2001;citation_issue=6;citation_pmid=11682307;citation_doi=10.1016/S0959-437X(00)00247-1" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=J. C. Fay;citation_author=C. I. Wu;citation_title=Sequence divergence, functional constraint, and selection in protein evolution.;citation_pages=213-235;citation_volume=4;citation_year=2003;citation_pmid=14527302;citation_doi=10.1146/annurev.genom.4.020303.162528" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. C. Fay;citation_author=J. A. Benavides;citation_title=Evidence for domesticated and wild populations of Saccharomyces cerevisiae.;citation_pages=66-71;citation_volume=1;citation_year=2005;citation_issue=1;citation_pmid=16103919;citation_doi=10.1371/journal.pgen.0010066" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. C. Fay;citation_author=G. J. Wyckoff;citation_author=C. Wu;citation_title=Testing the neutral theory of molecular evolution with genomic data from Drosophila.;citation_pages=1024-1026;citation_volume=415;citation_year=2002;citation_issue=6875;citation_pmid=11875569;citation_doi=10.1038/4151024a" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. P. Foxe;citation_author=V. Dar;citation_author=H. Zheng;citation_author=M. Nordborg;citation_author=B. S. Gaut;citation_title=Selection on amino acid substitutions in Arabidopsis;citation_volume=25;citation_year=2008;citation_pmid=18390851;citation_doi=10.1093/molbev/msn079" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=R. S. Gitan;citation_author=H. Luo;citation_author=J. Rodgers;citation_author=M. Broderius;citation_author=D. Eide;citation_title=Zinc-induced Inactivation of the Yeast ZRT1 Zinc Transporter Occurs through Endocytosis and Vacuolar Degradation;citation_pages=28617-28624;citation_volume=273;citation_year=1998;citation_issue=44;citation_pmid=9786854;citation_doi=10.1074/jbc.273.44.28617" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. S. Gitan;citation_author=M. Shababi;citation_author=M. Kramer;citation_author=D. J. Eide;citation_title=A cytosolic domain of the yeast Zrt1 zinc transporter is required for its post-translational inactivation in response to zinc and cadmium;citation_volume=278;citation_year=2003;citation_pmid=12893829;citation_doi=10.1074/jbc.M302760200" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=D. S. Gomes;citation_author=L. C. Fragoso;citation_author=C. J. Riger;citation_author=A. D. Panek;citation_author=E. C. A. Eleutherio;citation_title=Regulation of cadmium uptake by Saccharomyces cerevisiae.;citation_pages=21-25;citation_volume=1573;citation_year=2002;citation_issue=1;citation_pmid=12383937;citation_doi=10.1016/S0304-4165(02)00324-0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. I. Gossmann;citation_author=B. Song;citation_author=A. J. Windsor;citation_author=T. Mitchell-Olds;citation_author=C. J. Dixon;citation_title=Genome wide analyses reveal little evidence for adaptive evolution in many plant species;citation_volume=27;citation_year=2010;citation_pmid=20299543;citation_doi=10.1093/molbev/msq079" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=T. I. Gossmann;citation_author=P. D. Keightley;citation_author=A. Eyre-Walker;citation_title=The effect of variation in the effective population size on the rate of adaptive molecular evolution in eukaryotes;citation_volume=4;citation_year=2012;citation_pmid=22436998;citation_doi=10.1093/gbe/evs027" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. W. Hahn;citation_title=Toward a selection theory of molecular evolution.;citation_pages=255-265;citation_volume=62;citation_year=2008;citation_issue=2;citation_pmid=18302709;citation_doi=10.1111/j.1558-5646.2007.00308.x" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=C. Hall;citation_author=S. Brachat;citation_author=F. S. Dietrich;citation_title=Contribution of Horizontal Gene Transfer to the Evolution of Saccharomyces cerevisiae;citation_pages=1102-1115;citation_volume=4;citation_year=2005;citation_issue=6;citation_pmid=15947202;citation_doi=10.1128/EC.4.6.1102-1115.2005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. L. Halligan;citation_author=F. Oliver;citation_author=A. Eyre-Walker;citation_author=B. Harr;citation_author=P. D. Keightley;citation_title=Evidence for pervasive adaptive protein evolution in wild mice.;citation_pages=e1000825-e1000825;citation_volume=6;citation_year=2010;citation_issue=1;citation_pmid=20107605;citation_doi=10.1371/journal.pgen.1000825" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Hudson;citation_author=N. Kaplan;citation_title=STATISTICAL PROPERTIES OF THE NUMBER OF RECOMBINATION EVENTS IN THE HISTORY OF A SAMPLE OF DNA SEQUENCES;citation_pages=147-164;citation_volume=111;citation_year=1985;citation_issue=1;citation_pmid=4029609" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Hudson;citation_author=M. Kreitman;citation_author=M. Aguade;citation_title=A Test of Neutral Molecular Evolution Based on Nucleotide Data;citation_pages=153-159;citation_volume=116;citation_year=1987;citation_issue=1;citation_pmid=3110004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. L. Hughes;citation_author=R. Friedman;citation_author=P. Rivailler;citation_author=J. O. French;citation_title=Synonymous and nonsynonymous polymorphisms vs. divergences in bacterial genomes;citation_volume=25;citation_year=2008;citation_pmid=18667439;citation_doi=10.1093/molbev/msn166" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal W and Clustal X version 2.0;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. J. Lyons;citation_author=A. P. Gasch;citation_author=L. A. Gaither;citation_author=D. Botstein;citation_author=P. O. Brown;citation_title=Genome-wide characterization of the Zap1p zinc-responsive regulon in yeast;citation_pages=7957-7962;citation_volume=97;citation_year=2000;citation_issue=14;citation_pmid=10884426;citation_doi=10.1073/pnas.97.14.7957" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. McDonald;citation_author=M. Kreitman;citation_title=Adaptive protein evolution at the Adh locus in Drosophila.;citation_pages=652-654;citation_volume=351;citation_year=1991;citation_issue=6328;citation_pmid=1904993;citation_doi=10.1038/351652a0" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=L. A. H. Muller;citation_author=J. H. McCusker;citation_title=Nature and distribution of large sequence polymorphisms in Saccharomyces cerevisiae;citation_volume=11;citation_year=2011;citation_pmid=22093685;citation_doi=10.1111/j.1567-1364.2011.00748.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Navarro;citation_author=A. Barbadilla;citation_author=A. Ruiz;citation_title=Effect of Inversion Polymorphism on the Neutral Nucleotide Variability of Linked Chromosomal Regions in Drosophila;citation_pages=685-698;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835391" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=M. Nei;citation_author=Y. Suzuki;citation_author=M. Nozawa;citation_title=The neutral theory of molecular evolution in the genomic era.;citation_pages=265-289;citation_volume=11;citation_year=2010;citation_pmid=20565254;citation_doi=10.1146/annurev-genom-082908-150129" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Parts;citation_author=F. A. Cubillos;citation_author=J. Warringer;citation_author=K. Jain;citation_author=F. Salinas;citation_title=Revealing the genetic structure of a trait by sequencing a population under selection;citation_volume=21;citation_year=2011;citation_pmid=21422276;citation_doi=10.1101/gr.116731.110" />
<meta name="citation_reference" content="citation_journal_title=Journal of Applied Microbiology (Print);citation_journal_abbrev=Journal of Applied Microbiology (Print);citation_author=D. P. Romancino;citation_author=S. Di Maio;citation_author=R. Muriella;citation_author=D. Oliva;citation_title=Analysis of non-Saccharomyces yeast populations isolated from grape musts from Sicily (Italy).;citation_pages=2248-2254;citation_volume=105;citation_year=2008;citation_issue=6;citation_pmid=19120668;citation_doi=10.1111/j.1365-2672.2008.03894.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Ruderfer;citation_author=S. Pratt;citation_author=H. Seidel;citation_author=L. Kruglyak;citation_title=Population genomic analysis of outcrossing and recombination in yeast.;citation_pages=1077-1081;citation_volume=38;citation_year=2006;citation_issue=9;citation_pmid=16892060;citation_doi=10.1038/ng1859" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=S. L. Sawyer;citation_author=L. I. Wu;citation_author=M. Emerman;citation_author=H. S. Malik;citation_title=Positive selection of primate TRIM5alpha identifies a critical species-specific retroviral restriction domain;citation_volume=102;citation_year=2005;citation_pmid=15689398;citation_doi=10.1073/pnas.0409853102" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Sella;citation_author=D. A. Petrov;citation_author=M. Przeworski;citation_author=P. Andolfatto;citation_title=Pervasive natural selection in the Drosophila genome?;citation_pages=e1000495-e1000495;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19503600;citation_doi=10.1371/journal.pgen.1000495" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Sikorski;citation_author=P. Hieter;citation_title=A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.;citation_pages=19-27;citation_volume=122;citation_year=1989;citation_issue=1;citation_pmid=2659436" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. Siol;citation_author=S. I. Wright;citation_author=S. C. H. Barrett;citation_title=The population genomics of plant adaptation.;citation_pages=313-332;citation_volume=188;citation_year=2010;citation_issue=2;citation_pmid=20696011;citation_doi=10.1111/j.1469-8137.2010.03401.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Slotte;citation_author=J. P. Foxe;citation_author=K. M. Hazzouri;citation_author=S. I. Wright;citation_title=Genome-wide evidence for efficient positive and purifying selection in Capsella grandiflora, a plant species with a large effective population size;citation_volume=27;citation_year=2010;citation_pmid=20194429;citation_doi=10.1093/molbev/msq062" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=E. N. Smith;citation_author=L. Kruglyak;citation_title=Gene-environment interaction in yeast gene expression.;citation_pages=e83-e83;citation_volume=6;citation_year=2008;citation_issue=4;citation_pmid=18416601;citation_doi=10.1371/journal.pbio.0060083" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Estimation of the neutrality index;citation_volume=28;citation_year=2011;citation_pmid=20837603;citation_doi=10.1093/molbev/msq249" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. U S A;citation_author=I. J. Tsai;citation_author=D. Bensasson;citation_author=A. Burt;citation_author=V. Koufopanou;citation_title=Population genomics of the wild yeast Saccharomyces paradoxus: Quantifying the life cycle;citation_volume=105;citation_year=2008;citation_pmid=18344325;citation_doi=10.1073/pnas.0707314105" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. Wach;citation_author=A. Brachat;citation_author=R. Pöhlmann;citation_author=P. Philippsen;citation_title=New heterologous modules for classical or PCR-based gene disruptions in Saccharomyces cerevisiae.;citation_pages=1793-1808;citation_volume=10;citation_year=1994;citation_issue=13;citation_pmid=7747518;citation_doi=10.1002/yea.320101310" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Syst.;citation_author=S. Wright;citation_author=P. Andolfatto;citation_title=The impact of natural selection on the genome: Emerging patterns in Drosophila and Arabidopsis;citation_pages=193-213;citation_volume=39;citation_year=2008;citation_doi=10.1146/annurev.ecolsys.39.110707.173342" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. I. Wright;citation_author=B. Charlesworth;citation_title=The HKA Test Revisited: A Maximum-Likelihood-Ratio Test of the Standard Neutral Model;citation_pages=1071-1076;citation_volume=168;citation_year=2004;citation_issue=2;citation_pmid=15514076;citation_doi=10.1534/genetics.104.026500" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Zhao;citation_author=D. Eide;citation_title=The yeast ZRT1 gene encodes the zinc transporter protein of a high-affinity uptake system induced by zinc limitation.;citation_pages=2454-2458;citation_volume=93;citation_year=1996;citation_issue=6;citation_pmid=8637895;citation_doi=10.1073/pnas.93.6.2454" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biotechnology;citation_journal_abbrev=Journal of Biotechnology;citation_author=X. Zhao;citation_author=F. Bai;citation_title=Zinc and yeast stress tolerance: micronutrient plays a big role.;citation_pages=176-183;citation_volume=158;citation_year=2012;citation_issue=4;citation_pmid=21763361;citation_doi=10.1016/j.jbiotec.2011.06.038" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="ZRT1 Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="twitter:description" content="Estimates of the fraction of nucleotide substitutions driven by positive selection vary widely across different species. Accounting for different estimates of positive selection has been difficult, in part because selection on polymorphism within a species is known to obscure a signal of positive selection among species. While methods have been developed to control for the confounding effects of negative selection against deleterious polymorphism, the impact of balancing selection on estimates of positive selection has not been assessed. In Saccharomyces cerevisiae , there is no signal of positive selection within protein coding sequences as the ratio of nonsynonymous to synonymous polymorphism is higher than that of divergence. To investigate the impact of balancing selection on estimates of positive selection, we examined five genes with high rates of nonsynonymous polymorphism in S. cerevisiae relative to divergence from S. paradoxus . One of the genes, the high-affinity zinc transporter [ZRT1][1] showed an elevated rate of synonymous polymorphism indicative of balancing selection. The high rate of synonymous polymorphism coincided with nonsynonymous divergence among three haplotype groups, among which we found no detectable differences in [ZRT1][1] function. Our results implicate balancing selection in one of five genes exhibiting a large excess of nonsynonymous polymorphism in yeast. We conclude that balancing selection is a potentially important factor in estimating the frequency of positive selection across the yeast genome.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224" />
<meta name="og-title" property="og:title" content="ZRT1 Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/4/665" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Estimates of the fraction of nucleotide substitutions driven by positive selection vary widely across different species. Accounting for different estimates of positive selection has been difficult, in part because selection on polymorphism within a species is known to obscure a signal of positive selection among species. While methods have been developed to control for the confounding effects of negative selection against deleterious polymorphism, the impact of balancing selection on estimates of positive selection has not been assessed. In Saccharomyces cerevisiae , there is no signal of positive selection within protein coding sequences as the ratio of nonsynonymous to synonymous polymorphism is higher than that of divergence. To investigate the impact of balancing selection on estimates of positive selection, we examined five genes with high rates of nonsynonymous polymorphism in S. cerevisiae relative to divergence from S. paradoxus . One of the genes, the high-affinity zinc transporter [ZRT1][1] showed an elevated rate of synonymous polymorphism indicative of balancing selection. The high rate of synonymous polymorphism coincided with nonsynonymous divergence among three haplotype groups, among which we found no detectable differences in [ZRT1][1] function. Our results implicate balancing selection in one of five genes exhibiting a large excess of nonsynonymous polymorphism in yeast. We conclude that balancing selection is a potentially important factor in estimating the frequency of positive selection across the yeast genome.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
    <title>ZRT1 Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RniO6QMZLd_4-UL8IaCKROq9Q7MBs6xHNYCTtSGMbcw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459453","apath":"\/ggg\/3\/4\/665.atom","pisa":"ggg;3\/4\/665","ac":{"\/ggg\/3\/4\/665.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/4\/665.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/4\/665"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/4\/665"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/4\/665":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bu1KsKD9boUvVpg4TEsua5WkE0hdkXGLEpN6v7W1acpjrxh6aeCxVYflg94+45wiUPDgtMe4WE04m34\\\\\\\/T0NcTfNM+lTv7xuwVkPFZO9Z65e+p3FKIn90oYGT2uSzCcfc7j803Xa65xk1rVqKluteZtsVD3urdoj76Sdjq4rEfrsLQHpaexFwgr\\\\\\\/YCU6DATQ1sK5ApxOIGeC1dDzEyNnAl\\\\\\\/cY65uocmjNASF172efBavnY4r9O+XGYUYBu\\\\\\\/vJcybyjt6CmBECGfvGX0luKiL8hnbwfh9lhdiY3Y19QgMXXlBUzDDADenwYb4EyrxGsIPIVhh1tMxXl14BZLQSFyT3IDFSu4qzaWUF6ormYxn5ww\\\\\\\/NVDotuAvZndiWY5IC5SQFL\\\\\\\/AaYbk1kpRGopiiWRdckvVbnhUIuHGTtJAIilzfzsxcIsmRqsBtKFY43OxED09xcCBCTBjiX53JwzKx+TrpOk+hfBYZu1AIhhrhMCcnVnRrGziOChCc6TYNWdCgFwrQ0EsX8QaXn2Fj431bh8A2gf2eaKR0ysFENleGxbBQTaiObtmFu2fBYc4j4OTvXmq8riAAGYEvbhHDWcIaFhpx730gKeu5xrBx\\\\\\\/S1oeEk\\\\\\\/QaeyWcmvhJDA+RSeY+XGb5wPjW\\\\\\\/8hpLgNXHrb91xLrlVaJjYSQpnyEJ8Qcj9BLMW6sI0CUBF8LqKbWV7IWGe7X1EasjD+zser6JBkoT9STmohccoeBwK0WLKfu9XT5ue9kA99ItY1IpIYNPEhjra7LFoS6621Lu\\\\\\\/JUB8j4MWBZynHmcWry+EisPHRrj1dMOT5lDgwSE98L4UmHavTCWVshW0jWXX4XvBPZclHXqph9TKJB051ltByPwj2orZGn8FxImBSS0982Xa4\\\\\\\/yXiNkQWLoeBpliU+IzmiIDNJ1AabNQpJHXWLaZHdYoz\\\\\\\/Nm5Obcb\\\\\\\/CFlKyGbYo+R9RkRIorIK6uLvRriwYaYJmu3yKGRLcgoLmJdBMQbnlxD\\\\\\\/MRd2J1GQ92b8Maf0qqXjnsybXo7hrweO+iSdJUrdGZRRXAWoh88bYAXcHtjMR69105FgIguegV7kJ4UihDF2hJqKXMW7EqXbl\\\\\\\/up9RJQNdrP8yB9wDhIcrpw\\\\\\\/9vF+E5r2I1iHfbzdaHHroUy4yA9eeuFcr+q8DZqep9QukATLR8bzfle556e9gTegvgjFKHyDCFVW3WV8U9peoI5pV9poFoDFoRqpsb8fWBmPu2QuQW9Aq4N84h73+GRi9w8SIcxPT18Xfng8uYetYNYkIfra7UYXPSSGM\\\u0022,\\\u0022iv\\\u0022:\\\u0022148yUNBdcwRqLRVdMWAlbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229f14e02eb1ad615efcd16a8fd49492c17a895ae88237f8a1087507366c9713ab\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UpPZNPur95PZYJh4NxrTmWEqtgFkJENEUzqn9UjdEfD0aMfDRUFFtjBIIhG8T2O0DQwbElKz8JS53Bi2chK383e25V1cQhmEfwTcRfKN3WdJtUQCIvhZOWS8w8vEdOZt6PWpvwjkGyvKNv+1eAB2GVM+M6MRf25tdnhHM10bBKU3bGbcKcrpZLa7GZ0lkKZxuf1uP0XPYzacmwoWU9E9KsWaEBIBPhTmwfb0kzNh\\\\\\\/RaMlHYEdJfAqXJbG1YXA4CYyIgImDadgAWxxVuTXqFPyri\\\\\\\/wb9E6FogIEbaNwtEFEpFeMSjksb1f8uM2d4eKLv7zoZZUYJweaAK6umWHex6XbuP6Kkwwi40HDc9KxgaK49NpTxAYKEGjLUTp7TPmVOyRX9oTvi0Wnow0+ODPMtoktGIiTi7oQchuIAhcK+FFS0s3R3DNmd9Y105N1dIX6jbi6E8m+oJ6fyFkwG7NV2jz7B33emh3bK0VVfDOs+TGO8r1Thmt+B0xAu1oeLtdtZ\\\\\\\/Ligk6UmICdy75sjptFK1UDm7rT1r6zBmSCZ+wsg2PFAtAoTkEN82kqROV6I3IFIIzKNmAyHFLVJgVFlXD7v0\\\\\\\/N+DNdrXAZamXOcW2WgHaFFwicUn7w57srI5JfBi85jYpLUHbA\\\\\\\/JhrNmyzQULPvPWUslCWsHlQ2GO\\\\\\\/2HLCJDoeyiD8QXjT\\\\\\\/hUNhKB\\\\\\\/yzbK6zcBpGyF+Ptdy9KjGajobG1kbsk9BQiGn1MiqqhoaquM26GPHwOWleMZv\\\\\\\/2iID\\\\\\\/puqIHdEVtNzO16gEOkWL\\\\\\\/dYhvlBtqNyrLApSjnFBESPmGIZs++lBezoqXmKFTbQOa+MkbenjbxR6+MMaNGxFq42U2I75Tuczctnz3PUBzVUUcodzNl5n3emIs7islMONzf+lch4hRaoiCrPjOWurDpzByvu4TXyajpXw72cv3iomrKmnttEnrofe+9bhQmbhWKwpnJenxudpu5N61Ecax9mOjicqNCBXGku+NOSfddLs7AKgxdlvUSUabCyvCTUBJPtnDqpKd1re79WXIV5GL\\\\\\\/XD1w3kK7+rJf++mVVt8WAZaRkOh\\\\\\\/yArOuzusgIBW+iKyzaR1fn3sX722qdPJRgzNWDYny9T2RXnYG3dRfpYckQ6qFsnV0yjxxyfAasXUmAZUGgprJhdnUpkPk9i2+VQVYzufOX2QGyCQoxd1Go5jhaufJLS0nd2NLp35MTLG0gH57USm552WxIo9XdCXRHSLdzQPi8MbtxvfJyhrkoOVr96LFMrh\\\\\\\/KosU99uocezucDr3afMKyzlvRq\\\\\\\/CN6WV0CZJ6dIZhvFt16BbBEW63cb89BsUMRoICzLbvd5nd0SD4ZBci0Z3woGri\\\\\\\/E2tbw\\\\\\\/M8jBQQOhKSegxLcixRNUq6FwnjGQpRod3yB5fImec5b8BbFd\\\\\\\/NqaeGFPiWyLyZ8jVvjQvkta8oR59t9h8Z0yI4GnYzPfIkoy12MMH7JbLSFECr28SHhMAAfzYDgsSpe7gKy6B6kNwpeWKK53ddXp5svu5b9k5MZeGMtbS7D3xVZ0dTrkTAry\\\\\\\/XkMC8hqnHoROnQbnNrqhHl0kftL+iAwEzT+iQLUo8TWihnz9ZsaGvSfJ4CvAYUfeoz8XNpdaadKTs\\\\\\\/AVU6hps9Desu1jkGLCD17yHP\\\\\\\/eHAjHojOCvFvyJ6PSolWP7C0EO88xXLkon\\\\\\\/AjdnLDrDovdrulAgBzD3h49F9coiOyRMRTobeUeJgttJpmpwYIsVvd\\\\\\\/1T6\\\\\\\/kQCiTLnhWrWKmWbSwlgo\\\\\\\/sEuWhcg9Clbk3ytpK0xeFYoqPKFMObDCiDwh7xgrgAc\\\\\\\/2FCiiwET\\\\\\\/8vLfIxehEHnLHRF4Y+z7vOO8fuT8+qzV6Sa3gEGsBmOGT9KqnuCHkuEIV9X0gsGsnL4X4WWgi4DlRfhmTvNnzRSZFVoK3Izbtt2z3DKvZ1\\\\\\\/ptttKcZeYZ7OsakmJPswNTntcebOLybQ1W4Iy9qlZZUliDUlXuh\\\\\\\/trHaMOrdefjzpHaU1b51Bu\\\\\\\/tJpVzPcxRXx1DDckZXNZA8PqXR8vFV7ugz4QnY4x1jfieuGJYAg9qJl1qJ1KFMrILPbTb\\\\\\\/705F9telNYwLs0+MTFhrxZR728TRoYSSBnQa69HzSLjpckkHWjAdK3hzgiwSPOnKlSvkO1C6EjcaPiGyo0fwgZrIhZgGMW2WRIfnSNBQy\\\\\\\/PDNrJBYnyC8CiCfT6rpX7x13at+sNmmHscjtrHbezMOiWvPaCp\\\\\\\/ZogL27BtH7Kz9o98nOl3GQI755YpiieC0pw9F24qq4nEOZFkRvxG51v5qgUDlF9Mhp13w2\\\\\\\/xbI+\\\\\\\/LDh\\\\\\\/rhEv6N0V9Qh+BX+QFCRxtek0RSsz9bwrl3U426xqPKW5I7yeWXjWkm7qk8j3813lUg\\\\\\\/gv5\\\\\\\/k08OCYmy1Rh6LWbzG9gDdPVvT7sKE6g7NWWfO64Ku3PwYDDNjCwN7CsK1edUnngGEnT1QRi9I9Y52QWEGDTWThFEvL4SmwqwdDDEw0gbjnSBb6MMZ08W9L741YxvlpuX\\\\\\\/gin12D0ots7h4eqD60S+b4IhWrbF5HfWsa+bGPAEFyPc6M7zGzdlwoRdDYzOLrW4hROWjpE39mk7Jtd19xKwTJKroBv+evzfOgjh9h9Cw\\\\\\\/XYyLpHwyvjcIAkNGNyiiuhSX7s4dU\\\\\\\/gdMagfQkoCpT4k7Z4rOSnOT9\\\\\\\/kAh3NHTfX41cKZ3hgw53OMDytgDpzCrbOGPIBaKHs+lRStAOJ50lMfIRVyTv0FaiJS+wBxhL+FFEDBI8q8upI64cKoWWcJDP3Csl0rGMdXD75D2f1vdOH79+fqcYj03NL7FQrEWrdz34V00Hl61k5E+H6URgr27SScV+VcO55oTNEeBwn0QIS\\\\\\\/fKTkKMWKaoccsy0aOIMyZgnjRbtLv5cNfxnrnyowsm8TFsrxr\\\\\\\/MjqiKr9Km0Nyk9v2qduHekQ7gpCtAzpI1UB9yvqBaGwyGeMCgkWMYTkIahev1aQSIF4le4IQlbrzvDURf3NuHOyciRJ7TqDhom00DeFhENjiKKHobICpzbeWnf1wFtwSs8eZtIUnGFkxhd6UK9hNynHuz4lbIh5Z1o5yXQeCAGURwFWWkTYA1QItelQ26Tn9ksh2Q4J1eU7zVoNEjOxvO4JY9Fw8BrkOGECPk0JdpbRTTynpBldWz\\\\\\\/6hJLVpdkYpmAxne4LeA9hUGgFc4KkskQ5iiZmiICKIkTSYKtaaql36TlX\\\\\\\/cqg\\\\\\\/Coe28\\\\\\\/PxCshVA8YhbawU7bFCDXJ2V+sVHoQm\\\\\\\/eCHIKELNxzrk717O5QsCbW0ETjEcR1ax49s21j7H3qB0RMrFeZVUZzSh1Up0u8N3W+lX6STI49BzSmxgCrXEMOzG7Ftbdxt7th5qNCDbrDukbVAf5dev+X9cG6maVW7Q+HVBa3nZlAlKdP64V\\\\\\\/HwoIfmn1+89Hl7cbbPRHweiikrgwcBZ6x9nTw2NOct83Ky\\\\\\\/SlKlG2+5Mhsg7LiJc73o3ntAg0SAruufZnVgcLHFjauGray+CJkTKTjLMFRLk7ROmOjNMBCub1OWmqjFl\\\\\\\/FxlRyGoLHKETI+MXnyz5a7ViDjxDRPsZlqBc2Wqh8f+40jOwJztHy8QxmNyvH72b0mCTFzf7vA5WJJQuAl1i\\\\\\\/SJfBdLM9jR6vyCBWrOiwuNmvOptNZveN2E4dmrYZql8t6SNAp9AUzqYR2kVBiWp\\\\\\\/EzRb2Q1GthGZks15EcWO4DOpcRRos6DqHtIQVjCDeHBb3nmFbH1JQrCOtmwbEnsMhIAo29jiIv5BsInO6FgNyXRUbNxhtaz41WhOIW+e0trv6RzL1YlDVuqv6euLuP+nXx9U9KTIOtWuBDPH+VCND+gLZPE9rDG6f4NkM9\\\\\\\/e9bCTt+tuPog3YaMlF4cAaALg0l7ZY65aCgnL9pvBSJOGRKgVc73PtWsM38SmyXFv+gH62xY5pG21nhT2qy30hNnSIDdZpeMIyv1OaxRDiKTbAqjkK+1CrIq8zvYLJYKAaQviogq2ucRyioNh3xhLp3wsFUjzzbYjTSVjv6lN8LDc9HJUYePtOMrKBE5iXyL7iWLzJp\\\\\\\/wkBi2gwZmZa5IZ4rxu3lP0noDIjxwMce9DI63z4hb\\\\\\\/w0Uf83rc7Bs5IAtcVHt0ojys\\\\\\\/wx0iKcMtZrmBCCBP7kgvYJAXuyA0GTkmKKEEm4+r\\\\\\\/Y8eh0UOnkzAugACi\\\\\\\/XP3sIwiKCVQMUXGB54WaeyVnPr5ODWs8\\\\\\\/qmljo3fW4njT\\\\\\\/k9yDjuLZiTLMLYhOQGD6LsF+wfszqrkiFTV9pNH+lfTPwizdZG4K1UJLbIB1fscpyLbCBPLETWjc+kskke0n8sS34T5LJ5cY+REUGjy20Lzllat3wO8HvZ5qJ19f1Q+y2KcB5\\\\\\\/NkyOJdETxRyb3fE4CpsWuL9TgaQJ\\\\\\\/IEO4Nn+juWZBx9AgnVoOeHg+tAUk4ir3yRVyOu8V7T19OEwoyrq17ZbGqvlVG1T2boKNtpLpFloNjVBIgZnU2v2VONyyWTp+fRNHZu2yM7mEt0gcxMNb5KtHc0TF113eb+uHoLgmY3Vyr1Yl0a8hzsnLdRCp24P1LWmLQfHiMHxzSFk57y46qr2IB6vEZ3Dr3BvltS6USwmnAXCMxNUQAqUqlD3syIAHb+8CW3qjSS+Kxa69zB8667FMrSazd+yu1SE2ovsS0aFGGLu80Bx+jNUVDhEj2tcSgNRV91vioolPrBnw3Yq0kknGAzJ2VdLFuKC48UjBAXTtYrVhEh1eXVsWN2loB23jtGBQWCFHKJBgM9fcEE\\\\\\\/ryDtJbypzErzKPjg32buF9wlu4H78wfakcuEJIUt2AO764\\\\\\\/F4LLv3oostQbCkGOx1rswVgCi0EgD\\\\\\\/76njH15XhUXP35MsXvqVep+z2Jw5neukaTyJZAZ6lHFu3NL6yF4HGBKJHVbOqBEarNSyNt3eRq8aldJ7qX8Lvq+pClkoSH9vKF7AGXeFUB8C\\\\\\\/XM2rc+uFdBgRE0bgEjgPB\\\\\\\/ZDzv6SQM\\\\\\\/jJpFH2uBPn3\\\\\\\/+1gOA3Kpwvfs2IDaqGDiPbrZTDlKYaHcWIxRBOSKzmHYitDK6LcVGI1uFg08BtwlCn3LpB3ShffHSLg2lvPTmfFtel+QnBreWGcnr9aZDXWN3nfbkFUdFyoN\\\\\\\/dHAiHshbICcX9QKDRJVA9xOuKBCan3Xb7xD\\\\\\\/f5jp7CeT9VD0bhghYFwkuSUCu8JkTMKVwox27nQffpiUEuS5oKwEf3GLCG4Ue2HbS8SHJtSIfc\\\\\\\/UQOXXqzxRbs\\\\\\\/EjQKzC6FrDOJ8simXsYWMmlVLBjczbyzLedRqqJDc7ehP84UfMqT7Q32frR0lRKBXufuQGfQgogU0w1BWRZpwPh4\\\\\\\/Fs8omgElg68xS51IlobhFzS\\\\\\\/4DGp+Q10IZe+KnH9tOBNiyzL36Qc0y3TKFop35vii5s3HzYcc5ZzMjcDjwJiDihewfqei+wf5zLQ2x8zPVYSOM08QkQ4X\\\\\\\/\\\\\\\/aDCTN2pmvEwaB+pookg5LgwyhwKmu48DRvE0qoircSLVVmJF5aeICVcjATWPS61eZKOjmrl46a89pYIwdYChV1TAMTRaO6eqEmrMQd1V9bHFT+sMVLaU2GHy8ZQXno0CZo0sbJN3i4H4ozPj38sdlLjJ90N1y6fkfkV+qyMhFkp5cIROXmd78CnKVidQAt8tDra7pf32FJjjSkmrk+vCivSuLCGbPAsQ0KDEUmA5QMyz\\\\\\\/qTB+1MiOeHfJRwXhp6KFr0AUECq8SPWO4TpDtWJBsv6dO9l0yVngVTorgXhduBjy2GilpeutRNB\\\\\\\/Zb8aM7Xf1+NAeAeUglPg4WkPfO40U6GIw0HATeuPqGPXHyiYEFfQ+SHoUbsdkuFEbdUIUfadzPZEO6mNcECf0Q1Fll6uszBz6MveVj6hwbOL2\\\\\\\/+ycJf\\\\\\\/DqwJ1GjjZALGAbJQeHvSu7dWdLveqTErrxgdbQUlzciKPr56R9P7t+MijcyQZ71lE15jshYTqqgwY9OImosxaLWGEQ\\\\\\\/2AXh+I5l3zwwjdAyxybeaeB9Fc5JGUekeFXnGvMgjvZw5IhB1ku9+NKQShZiZdZVJxEPmTH97OI9dJnvPfXEY6SyG\\\\\\\/0rUQA7bweP9CthfKBqj2QMgodHiSpIwwRW7GFEt6WRNCqldX07oyViw5Jv+s2cjmDEa11ECY51Jjrag2324zpwJGgi7iwRuZm\\\\\\\/zQPpM34nqcWrT2Y6r\\\\\\\/GbX43p2QKzq8yWEEjCQ\\\\\\\/3inE5lT5AuB8RDSeAB0utour9oAZcfd8ddBjXcbIOsahtPoqrPleuZEQOtQue60G\\\\\\\/Xmd8DUs8AL3xmrNtHXumoSIm\\\\\\\/tFPlWYut7VAmnY71YYdn9WEfNlf40icWlmDdKiAVe+ofG\\\\\\\/lOdtCgSK8N3jN\\\\\\\/Lb6HkSCY0RJxr6+\\\\\\\/wbSBBzcvoeeAaun0AJNL1pt3uLg8t06lP1ZdHL00GPEzQSbJ54LpI+cQrllUbdCqGl+usZ07MhwsTDpZFRT2uELr311PDQm+uTqe7cznNPuUZaNZCb1LsYV+uFQq\\\\\\\/\\\\\\\/qTKFelm\\\\\\\/muOMDhLuoORX\\\\\\\/aQfs2jZNc1FIA7P0J\\\\\\\/MXo\\\\\\\/3VdWInhfj8144C+H5UUMq4BIBTWeqOFsW2TbgQZ0xwKoUizI2K\\\\\\\/433s6owUB9puy+v1klq62IpqF4E0uIDcI0v6kwgC4o\\\\\\\/HYp\\\\\\\/teQlAlu2HgJUZ9h\\\\\\\/kJ\\\\\\\/jKT1ByDUqSIgFRzQkfJ1Ge9t8ix3T7FVOKID+EScrbegngVoakM2BUY2s11TOrIHlcg9K5+tLqvPOuDzpPV70q+NUAYfsjuz3XXgYJacs46CYKz5GRduBd\\\\\\\/0itKjLUyEQA7pE0EtKBxhYQyAFc3OYnuI2xfByNt2eg4h1z7dUm68pEV0re0pcdJuSDIQLCMRQjHNU1R3GQ4jhutXRZ3Dz\\\\\\\/3rzMEj8kvG1lesJpFDl5paQOPi8l4scP4bDsc4+HfevSFw+lUSPaavBb+ZEHD7FLPTyvEAu5okNAbr3M2btYYkYAkAcfLc4D4cANFU7uSp98K+8DJywqM3UHUuyQSVgbxe\\\\\\\/Q9L8zW3Gh2gyYtAGTN0eR7SI4nGdEWZa2mgHddH6NthMKKqxzX42TLGhHFfxrl\\\\\\\/wbr6Ad1qjTHwbfcaVSh51JwcaoxN5B\\\\\\\/Rg8Bcs74YBcQk97ZqLo7ltANgU0Gg3w4iJjZI23XGSX1q84e3jUo6As8+bzYqnNfNvXPkKXcCSOgT9D5HeVT2TsXcHlaelmLU1p6CFFjNbBQvGrLVV8nPb46zPya7Lp3hK1Jg\\\\\\\/M3HO6ooQ7uKVJConTb9RXg+1bV4iV5vTBszSiADNpS5ZoHxZNrBeciNgVDWuTLyviT40v7yYNvSK6EfoBY1Uc5cqNhUSHhZiMfKHSoPwisiFQ+V0zpzqPlMHuSrkWjjg0ytMZhGt0s6Wlppa7+dM2qnldeoNFp5NFVQXnEqQTTsMnB8VMZusFXyZ40wwjtT8QtDyQCvJb5ecIZXuvC618064G4iB2uMcDy7jdO+0TB3LfvY+AABzKVP9+QQOd8a2aBvxLEimuG10HTz4oW7PJYe89su20ZKtfT\\\\\\\/ovvXXduMzZTdnNsGURZtRsMcFiIdjtT2VgyyD+ag6u4cl+iGILgQdKTTNKUokUpxCwxj\\\\\\\/ttDv1gbpQM5gSh6ZmI5WDm2I0Niw\\\\\\\/sS+BPc62UXBV1TP9lO+T5PVlzbF8V0yewwpXkUnXz9dCZvhaFD3wUBOiSA7JBaawS9qO4u6vIsMaa78j0\\\\\\\/Q5zkqTFWyWEUBGGjNaJAtdOl3RQ97AxVr0ovgicDfXwMk4zfOIWdqGfIgkAPze7FOZHE7Agk+2o6Rb9GBS+tFgWYrfpgrjV+iPNpFJ0\\\\\\\/sCSdwidhmuvvThrLWsK3UuBq7rKZNvB+QO1+ZvhILBm08Lb+bbpqUcXMCKH53+LsOp8IYk\\\\\\\/2VQyKUWqEhYWBjBLsSzZN1SrTbvnEJ6HCoJgq0lXZZRtu+O3yb2zI4ft2fnZjNZuC21PmBlnntmsaOAymx36xTpJwd4vS\\\\\\\/IliEkG9nnhi3iyX97S0HGurEEU9T40EQjmhndeIe120Grff89qQRyp4Sb2ViBwBn7Usw5Y6EAbJbNKhjkVbIoniCK5ryrcTDd4EBDPaGTEhFyTdHvrgPWU\\\\\\\/0FDVeJggC7ebHohkKwxO29oGO4+IAmqxVxY\\\\\\\/5t3sEG+XB7UcGxCTEeAPGoYBxoAoAGmVcrUr1Sroy8r2nWJwhqK0BG7oYuSMWUTx\\\\\\\/NzyOv7CMnjzd60GyIC74x62fMgYW9z6XWPdn+py+oZ09gnBs5LCJ8Fv9OsV7wkXgF403ZbnYyhGqJ3AkTPuYE06ZOWmfoAbHVlnOAMu4dZJPi0jY+7Ti2A8Kgt1B7WimTKzqjQ6xJmf4fVOs0a2QnZiqlP9RxDE2+Tj7VLIzYzZykZE8GiReMJKBHnMO0wfoKcXm84toNd4tVSo5uoOZOr6ZpumwrsazHbS+\\\\\\\/ADvEUesKB+6AJlz9i2yOnmsFrHCoJIdw\\\\\\\/3FsAffxxPS5p52\\\\\\\/DITHBmwO778ohn5d5ktOxQWFEyzxQNNSQwm2uZaqixL\\\\\\\/PaDBDRRs2NOZkyZH6TgyORNxFgwJ2a7IvPd4B2XCjewtsnmqyPyrNejdnS86JXXWbh\\\\\\\/IHThjSY9FRzW9wS7xy9zp98QUQixRmaYbMftGqOMPt+Mey5LvAc8cMgM2NZ4fg2vPBPxXCE78B7dUkNeqt7xzNXonkkG67oBffRrfsRbrx77mlrXUnLGFo27Lt\\\\\\\/sg8yrpRGEDQefXBH7XKpVYWJXJPNuN8gTa6k3P+oGmg1UjD5Nm5jFj7ygChmsKAqtqx2zGz7FooRIWowLLhnnMs89I8i1XEsKf+EOi1yS0+pRsGr+b3\\\\\\\/PvT+h4YyATWebFn8Ak1aH5m6VjvXH5ImPnGNLfBhFyB8uBc1LdP3wS7kx7YrJll53TQmXsGWw509tLCOmHCdbwQx5NfLuClVgZtFxbkEoFJTqlF7srlbhJw8FdE5B1bi\\\\\\\/vHU3IMnYenXdprGAYbJNU2AGjwr51Fb\\\\\\\/e7g+MfajnbPWXFBdjvwokoefzOMveKSpJOFurNVHDwMcCHJIULb3szLIoXqvoLczLD9SHN5iLl61FWQeCZn3v13olB9iNNaMWdQSyFKTIFsA\\\\\\\/m0bOhG8qz59AyS5qdkDLSML2rkcKwWJzm5L+Xr65PY0u2dN0Lr4HEpBuMsYZARfOEwpCNOuB4ZvQE+vAiWG158ppcatI3Cg2T3SRuoc\\\\\\\/Cv3ESYmWzd6XqpqfEKZrH46IoJ4vj9FpWGWxIuiJT6CAjJHjtlASatCAkl9SkrLnw5P0m0f9XD8USB51om029bSZzzMGzfjnO0gtjWkvN8kgo8DcDnIETXRs2UWO8zZDxku6TyNvClf870Bvvz4\\\\\\\/r2kzP0VGKZ7tmeULeng1tEmNPb78yerMV+eGotXkDICHmMpm\\\\\\\/jvxXTJQsrORA33o4W3aNuAdWWgjC+1uXlHkby2zdEHSNeUnRTgrYWtLExvpC7+ojLAlTu597p9gcW6i3EgjQJQ\\\\\\\/2YaWO5y2djsDdKULXPTmpIG+lo1PXBbOtsL16ABLS0z7vDq\\\\\\\/VxP9xX0mdvUK6hrc3+Xl85Fio0ulTmEvdoiWfXLlqznImDLFwJrvJXxY3I9tRqN3AWiAY9U+tjU6ZUokX93p1TMfhL+hPShrxGdaiShEH4pNSe7haGQjQ6ElI7MoMqmu98HWF8KmtCXfZrkxoIOQps\\\\\\\/3A0BwLXCqW3Bb9vGGLUoKWwkAPar5zmkeA5qM0k6SyhUe8vMYY4xWxLlmtrsmr7moTsa2ZT2ddG7LmZ2B0Klny35weoFFWWQZXwq7+HJ6XF3TWF26K6UUULxv2eeCABrqBHVXou5jJLN7gaevh1rUhgxCEC\\\\\\\/sNQ0KuF11RhDZitQB0RvMUQrfYiunCabwL0AbMFIw6EM2klfdDEyJQnonZSF6sh5DYQL4LZFjQ2si3NojbHjOP1M1NiqtNRNyUWUMI7J0kD6FI87PwnQ1G+ZVLgUIYCEAQVGeA8wIMXLuEikgVAAoY3PoBLsK4Me\\\\\\\/\\\\\\\/22lShF06ZEo7iUQ9kBM5+Dirl\\\\\\\/M4RV3QJ0ZlVM1q12knf2QEAQR5nkydD2CRZSXasPLX1dzjeRvhR97VtbnOw2t35L40vuqx1+M6T+aCeb1UKkNMjMijrXfPm9dow8SQF91yDyXU962rgzTMX6ahC0d\\\\\\\/4rj99Aa6QJS+XalqKUVpRd\\\\\\\/i+2ZCVa2yL2H5U+8Fnx\\\\\\\/xMti\\\\\\\/EgDUVS9pekS0X7tb9tP4xsmBhznHKU2t4+ndKtrDLjQZt3N8awZUAT7I1\\\\\\\/p0wF\\\\\\\/d\\\\\\\/VUJXh6\\\\\\\/AuPyOW76FYhtaS3ARiXioyJKO62A0xy3bUNttyiYUU5M0vEYXQKdD+ku6oFYZlWJS0+FqHf8xo1reAHpLT8XH+6uzrr4XCwKo+UauKvX2MNnA\\\\\\\/bM+dbz5dIcrZ20D9HoDbIq\\\\\\\/WlbnycbR+B0Z7BVFYnQeBDC2pGbqqnzD3tGfxz6q0N32z\\\\\\\/Cw3+l6qW6EMMLhd99v93p2CQt8IemdJ3PK+zXj5BjHQkfy6T0QZ1o9OzwrO+slnW9QPL5kj+ZJeXxbedX87I3AJClRsFwKKgZhxGOxOpri4FlxF\\\\\\\/SrIxiGPo5uCuZfghgkQgGqqVokqHD8aS4mexfXXbrIkQMGy\\\\\\\/q2fGskj2y54aqDaugMmZPslbgB0LZMlWDCfyueAoPUI4ytd861Fr+Xs4aCQTo2Wu+nWgGRyGLSvud8JFvawpr64AtnRCZxPb4NJLAO6sW9yiqFSbh7m\\\\\\\/g84rOtwe8I3SpkbInT6Cd2\\\\\\\/s6cRnf1mnEB3c9mS1jOgVEeznwqSU0uH2VWHtbsqBhOW\\\\\\\/3OflN+1bG5zWAM3lAZH9\\\\\\\/Z68uCow7C6AEslyXxZ4J8st1xhueAckK8LsuKUu3mvr2JfKFsDeMKHKJvlnfUwPY2omDxEZx3SAOgVuY4LAWeEUGT6FIClYcwodrv7GPvSHddLUkUv0kLeYjW8jo74POVv3SQ9oMP1QR\\\\\\\/2QZ1zaJLr7tpM5kUn9jzr88EBY0LUsKajif29RxtMvmtEsmwJXmdQu6PpOH7n2nOKZxAxA4NK1siMAZEaUrHBcO926aYaoIdaw1KxPhMuZL\\\\\\\/M4FeJENE2ENibeWrt0\\\\\\\/RZXjFpksI2fweNdwAuqikLBteRu1eRisa+HLEe0yFebaeKMi2JYKxu94bkQdahEb9HBiTR\\\\\\\/QI\\\\\\\/g4EotM+IFtNBxuDqNkkg0mDlz3nmlqLZkSAt12kZSnmFuITcxATGpobX+pf1qN39aoOztHBWDH8pKMHSSamtJBkqOoVS802ZhU6RsV1yhD\\\\\\\/2tYSzwEopzybqrTG8GMyIBuoR1VmKV88TGRs+2kAyrpwoZ\\\\\\\/q24AiA9SbEY01kZU5RdOn8uuDuN7uE2vdvP1qmRIXHYLWKjXrIEFU8Maxri6IchJ7Sh7830XfLhcF7L3zRSkhZGO\\\\\\\/QnTo382KwrbbJUG2od\\\\\\\/MsvEqAy8ajvN2R3tW\\\\\\\/D+MTnQTBBg+CPz61VStgUmocmUJ9wGhTh5qRbhrMV63a6Xsz\\\\\\\/1NPAXU0bbzKPbNyGZ4HnA99No\\\\\\\/40uyx1dpfWkZ\\\\\\\/ydzGLLQVoHtn0XpeutJmociWwRFt8+xz6KTF14es\\\\\\\/ZWWCxJsSCk67WZGSoQVhrrjQFf1AkfYLpJ+bXFb4gWUJIxeV1RtssdlpSxO8\\\\\\\/pz9uMItxxttjahq7z44pfNFpi\\\\\\\/o7QxDS8rCQyVpY44y0+qETcgbMaOA6dXZyXELFSzPwU4ejiu1fkjI0YY\\\\\\\/3bFTQ51f4UpduHXCJ0Jqa71seb\\\\\\\/ov8zVIT9dp2Im6e5uG9VEzldkAxFOgiIGd\\\\\\\/SOQTefFXOLF3Vu0AzDZ3lZQhRrotJa0O\\\\\\\/xjYX4eyGN8Qpjk24VTKwYqkryC3QgDlw1zFSouK8NOJ25mh2oo3v581jB+sGNM41xaOQMgnZzKERiGdb9VK9wcRPBye2uYXbtYiqrnDDhxAhZ+IewkJ1gmB6mDeOtBbpxPWvSGG4GR8qyhK7E1NLZGRwKHJQOIsq0L4Jn0LdXIriKPL8xvQMBzIQhD8F8aBZ8b5X60RMno8FiPX9lYg3+JpEKDN8DFQkfG0bgR85CG67IpTwxKJONGolFR6IUva1P64nC7ng08y4wykyW3G3X62gDj\\\\\\\/49z1ia3Wcgb+iiUxKGbq9HfsyKmZQsibniAN3sWQekSdPmWLXWh9ayoD\\\\\\\/frLsx6magTsSJWkQr4COrGJg7KC\\\\\\\/7uHqMnnvXXPEIdq\\\\\\\/SX7BpylsfTAFTnZWBMUmwxM1MhJc8uQZ2B77juP05hHecShiUdAZCvVmdbzBB+1Ch\\\\\\\/VpnAX33wcfC4vRaxSEMsaSF8dGTSjpaE08Eqbez7gNJMzLqHHjwHOh28iM3ovH5NQRqb6q2KTfrDVUZXmPoTPr\\\\\\\/yIthAVLl5sv5HI3BRjPhM9w87GKDe0mryl7O\\\\\\\/s5DhO18j38YNZ79zc5BPKKXaj0zquDn1DDDMk2SSJIncFD4hdAnGcX9P6kq4NVDHzBfPnkPUuLgqfoSCsWgcOeZjHBPKcSTC01ZP3s2bmnXyLUoQx3odV39LkLhRz3EnwQYKiHX\\\\\\\/PzgJn\\\\\\\/5t\\\\\\\/A1j3i8Wt\\\\\\\/7UDz1XU+Tp9GuFXRmclh8cl5F\\\\\\\/76ZmU0qTd5kKFNKDwhxRPev640AeuCLHA5tEbBJHzMeMr+sviqwRN+8gRCgIH8pU7TevLWRK3yRDcBbb1q9j345B6UEmsMNqhqrpjUaqca5p51PdxXJBsYPB\\\\\\\/8ZIc8CpPcpTsVci2Da\\\\\\\/H8+Yt9UxbZGwbCR1Gl9tMVfc9kQYCLmzPyz1DN7ow0JUoAjVV\\\\\\\/+v8yZ69KgNQnTo8IMKRbCVwmsOvcBywDmLW9Fdl6rHtrUmK\\\\\\\/rWafvBzv60ABP2rq9lEEdrPPcwdkSe4IB\\\\\\\/Heh0VtklXGeqO+cIK0almCfRKQjcfIQdd7RR3P9tnh2LzpSzCTR2KMGd\\\\\\\/aV9CnGR5t95nHTjQbDhwrVbEFgH+1uRilnwW\\\\\\\/yNBIxiYUFgHBamndp5rQcyhP\\\\\\\/sE5a32GoP5CnMoTE9A2SzjtV2TxWxHm5DNVdc9vol3Sui3oNBxhtjf6DpOzVlLnUVx515Zh1\\\\\\\/uLSVvaBuYsnTGSN9mzb14c9lr8vdWLEb6\\\\\\\/pQaWT7vA5TUjxX2m40Ym4xFWWj3O0nOu68wfz+MmSAO3Wvvm7FLzeL+vm3C4qq2+wLF0Sf5ohNHwdnuhO6VXQxk4sAytTYhG1LjijuRW2fYxcwwdmoJ9\\\\\\\/Op2qGBuKcy8jW58ZZObNLvxVnWlZYUTWHNf93jV6jxDfSGIQM20\\\\\\\/HXayoCdYtVu63S0n8IxKfc8FWxEgAZ90x5Q\\\\\\\/Co81ZNlWy74Cn1oqOPp1HjT2cCCNJmCV6HkUiiNEiTPUhLS1J0QSCD5zCUuQlF\\\\\\\/760Y7BdgmbksJQ8Qvy+j09sYXLCuh+PpL8hyQkojgEi\\\\\\\/48ax8\\\\\\\/kZtQpaPICmlBBDk6d0kcl9DAdhw6dY9s8wWX\\\\\\\/khVzcfmFLF5RKLzJN\\\\\\\/0XOD4MOrgjTXANlceSH05HgXbztSTcJkzuNDAn9tXUYXpdB4mRWxtn8dcv4TvohZdvA9PXcqcsNZiiauYPfSwbvO9v5X39\\\\\\\/Xibc8gUDfqqJn\\\\\\\/oKcD8L0HBAZzVXta5UZ0ScPu3DDyw\\\\\\\/XVS4hRnHrRJwU7WwUH6qIbkZc7J4ibWxI4i18OLBIqdfMDA+EcwQ8k263vpRg0xk3Pv9nJ6AEShtSXJvyLf+OjKBsNiqfs0VxGCwiEjj\\\\\\\/9TDYX\\\\\\\/9+NKzvEAnBRL3PIsAWgGXeB6Qu2\\\\\\\/jU5nvSFY8h7e9v6YjNOOPCSRfpykpxkqltkQbZDJL8nqoZWOqbGMh05Gb3K03tklZnDbcAbNtFOmTSj4J4LvyGbLnRO7GXzFLfTvU+dLtYDZ4oRf04QEANI6+xWPmLDZlDxAo4YdzrwWFMdNeEK+5S4IKqM45d4hLKPiUd2Qm4UrfVOFAyvWR8Hrwh6sqHbIQoG7ZKzubM8s+ibkQXyyZYyfk5WNy\\\\\\\/J0DTg0k6xFf\\\\\\\/wzo5mP5XudGOpCQ5y1ZoAbpZqoLk5ZuwCHHw26jkj36+EJ0GBkL9H95gCSu\\\\\\\/VZd9lSOma7wc2WpdReo1OjvVEx0Sd9ssVSk\\\\\\\/YvQoGk1+ZH2suWxpc4TB6iNhSkuJHRYHiTxS4B4Xe6ywGVcsQpiFhPWyqxn1rw1ue7Juvi0DuZ3p5wrSvJmHVmAEhenefS7CRuuxkh7nrHJvSeHkME3IKKQRwe\\\\\\\/wDqDNWhn1zEpftar9QdoZjaVEZL0plSr\\\\\\\/8VwAZW+pL6cn6iKll5ENMs8T7VOjPDuYmZrlcI\\\\\\\/uhdoSB2mZuWJQ97oCJ4erC4DZISslkXg\\\\\\\/9LxwTnNq6q7hUylhvSrk3cBx2gPCMSU8Yvgnp09cawpvJr5jPext9i4KUcsadNFevP1ZP7vd5ISK8bNKHhISrtX7yxG0mldCnNqh3f5VWzHVvdcFLuHFsrQS\\\\\\\/+9+1y+UxLorwBAjpNF8TPb7RSTjRFwksyOKzKk\\\\\\\/RTw1dzMhtbhMsJUETTXdThOHYC2mJfwqA5JtvwPg\\\\\\\/hKxxVZTBuILT5qWDIdm6xF6KqnlB06JJF4AEoo8F0\\\\\\\/DAVUacMiUSDa1r5tA9iJFVsEUivop9rxKtZJFHuQr4izzYph+OrE6rNSb+7mCTMXSI\\\\\\\/tlp8rFR4wKK+vGl03x03H+T0QH6LeRSVCkpPFelLkEDN9Nps6Ygt5SVzyMEs+hnvb8wDly9FV99X\\\\\\\/\\\\\\\/ExYME\\\\\\\/3jAElTgNd4keBOBku8BiYIW0e15DN1l3YbS5eoCYZ32DCK9+HAUaMA6qu+hPSZ7DoG\\\\\\\/ZjqjXTuxKpq682+Sfz9hkIPXa0zkhWPUevHmuTqF0REcTO9Q4co82xTuUuGR2VUVJvzK4qgd6hSw51aJngR6tFA28AMw9o+In+2Fl9Ph+lT+FCjzNWFd7uTvacwYMB+kGBGLJ62zxBr26ENA1OmktytfgfE2NEKfHYMKik2Lv7Dwvgs4KlGgMmosAx8QtOW\\\\\\\/+t4j0TAyxiiLyeQwhvh5bgO8FEvTk+ZQw+H8sPehCWcv3MwRmu5Az9B5lRgw\\\\\\\/lJKMTyFGe6pkrqgiJZBJKDE7s6cAC8WGSwA0gmW9Pkiw9S1xakYAhkEG\\\\\\\/c3kmJ78SMYIKRaqqT3DcG7bhnb6ZahAplbm1llznNIXokz706duubhxw72B7Sv6IMmqefG9U6JHmIRWf9JkwUVVulu+SrcSf\\\\\\\/1\\\\\\\/m8Yrql9wPrC1FHVfIFGPPcpPEJLtUpAaCf9ZjslvDasBcM+9UgMcMf0cyJzygUQ52oW+EeWAUzdgGAhz8r2eUMLS266EgfAlDw+pFqfXEj1lr2m8NZrvMzjSJ4Fpj66vJFi8RLGKPQGcBa6kKmfE+6sgFHgS3FaiiMghZyJkdFmAJ0TB4CePxYC1EdyojEnthHELvXfAmSyUWiOrdKkfX20jB\\\\\\\/vVVfR2WyuN2mmg6Tdb9upWjFPDSv6YQRc+m+dJGZuVHej\\\\\\\/AakPDLZ0UKZkpCAbbncaOIYjjc3eFhExPHyk4Iis+hE5GrGNh7fXlit\\\\\\\/uHT9EHt88iwqUG2jacL\\\\\\\/GviHv\\\\\\\/Xt5a29YwjpIHCIvMB9O6ZDWsmca\\\\\\\/XpmQ8lkp9cwKKHMC3pZxzcKGGp42aaHCGh5Jn3A3uR3f2tzc4iyErR2avvFCBt7fLrsvgjVfZmCoBgeK0C5webi0IB5Yd1oACp3Uc+0w9ZyCFXzNjpkyFa0qsSS8heZXdTdd4vdK76XkKKD7Ld+bnIiv0UF1pDTpjCnt6JzqTPxC7++YyD4xe+nEudDDz+hslQwKgiSHhrPxHjZmYNI0\\\\\\\/Wm7ax2hbIQF1UCWSbqYIaLSFyZn87O3rsuBOrt5zdVgcnDBKXG0raa68CDEgmvr1WXrnwcBc+QrhfD8gx0hEEVa19etg79\\\\\\\/wTQTgygvSis346\\\\\\\/rR4b\\\\\\\/bChBOSAxZKDhUfbwv71aqCXQOlkt1Pk7FsHa4PNuiNuYfroA2Z9suxSecHjQ56bNUD9FQQ4RL\\\\\\\/wcaGOgXXGOnrwsOXwuZdVesD1xtdDdtkyoaKKUS1hf5W9PRqSa2\\\\\\\/uB6PpyeSE56DqEVvq+usQv7dmIyR+8BkSMkPHujLh0ryy2koR1JvqpyWyqjYBFJ4enaHf+LAP6rRhFilbgxtHJlGOLDmlRD+3HtRAkKKRO8iBR142GK0MLGL4h9BU4BCYfTDsvd948zjZGJyaCY8ZEDlHUBRMSC8rflEV8GISG\\\\\\\/Rl2V6V6Q5V\\\\\\\/arhycv7WuNay82KFdZ7LMUyhcvwbvfWFW2bJ79gjuFSm8eO8MsUobRiejIYwTjuYEiL3eAIa89gtwtxGvmb+WcXDCxGCr+gaj0aj03IxGTfHyfWXgqYeSQK15TzCOPAuTpHUOLHq4nxS6guFcViAJN+Ohv4kwuOJtGVRHvpcX6RfQcEs5TaxUjjySekeBFsDkzE3XTPfC+Sf7bWZ0WTx\\\\\\\/WKz\\\\\\\/oJvbz2Mb7jQP68LWT2SBacfMRWT4HOhTja6u1Ri\\\\\\\/d6xJt74LOQrMIRuIQIPbgLTeBcz7fNjw09sUGqwKj7lxCsQwyzkdQhaTqxYXNN1BNm\\\\\\\/FykFSQmyc76+N+KBtwkcHtw5UVfRMevXYikuOZ8q95UfbLqCWwRGvfu1RwbImaPKi0rabLouP7QAwUpJHazXhJ5+P4yCNw1fxRVdg8dkIWOVGlp8YQfRoP4S908i6fasBzyJbUrzuuLZdTpHL6UbluZEaKV1VakBVQufRGJj60exAbFyoufpLKlXcwXq8fn7E3tXnEWiuytNXs6e4p4t5mWOSr5uUftqAayhdUKL6Zsy3RL4jZM5xj6ZB\\\\\\\/wTwXq2i99WRBMXvExb5wIdMZ88fCpYMt8101ecyu51sYyfm7QQYWjdSfIJ4iDMIF9zQCwRxjuO+LR8Lt9\\\\\\\/ZcxlyX\\\\\\\/UXCyGKCn+UYh9JmtpVKDiDew0Oj1beKp\\\\\\\/OAJvCaDUYpO5Vxy3GDnUXmkC8LB\\\\\\\/0dk\\\\\\\/TlUZFpGanTDdHHLDSdt37pn\\\\\\\/dMuu5rl9R9DaORHzs75ieKOui4QMxypnbGkT202+TJWRBxTdz7I362FvtE\\\\\\\/d+AqwSyAICVASmvzAa3OGxk\\\\\\\/PjRcLm+HqAcWnSItc7dtoHc\\\\\\\/\\\\\\\/xXkpl6fXqLh3W+Nk\\\\\\\/k3f1XjQyOQpgdk964V\\\\\\\/0ntdiSp5CwnXV+DPCTrguHOTuZTdo+SZek96SJEEPTPwaJotF\\\\\\\/7HYB4LqWbLgD2y+WJH66PP5sfQjvv\\\\\\\/mxJKLkPkUAwwQ77HZqXmwffEQ8rdAdBQVbtma5V44JM9u\\\\\\\/+NvPStP1a7O8p0snFxZbKQ41bTDO9ZoLx\\\\\\\/P26UC0rKZpOOBrLtXfVBCTFPmzIdvP3xh7ZnjTTrxRw7qHJrGpsm76iCcxsPsHr3oIYF8b\\\\\\\/\\\\\\\/7M5d6BJClO0\\\\\\\/uuwdKLV+6lx7BbtiGToxv+8ODVcdohwLNLhVf9jj\\\\\\\/1gXcuXYllBr2nOiixu5MhiDTTyomWBjTivNGaeUbKHgfl7FYozW9L7zIbwcR9tlL\\\\\\\/iWmN0FPzx834LRm19\\\\\\\/rqPCh8WYKNXag8Dk\\\\\\\/Tvv32+c7vkfayZfJgZF1iRinzKZZa6o+CsIxizhNA7QrS7RJ2jhSKDxXNWN+afHkQLxhj6nSjkloOTm1M1f\\\\\\\/paeiBXCjzt4T\\\\\\\/qntsJwGK68FI4rogTNX+ob2qOmNXrajIAPAU93WvHR\\\\\\\/sGuqJJMW5zr5QJrmYexxNhvSYSi57pgvio9JNfq7M98zGxTxawaEKpdXLv\\\\\\\/Z8jam\\\\\\\/fWch\\\\\\\/g5DiAQY39Qsc46fuHMfEYd\\\\\\\/VSBhXYCfqmI7\\\\\\\/KIitzGG7ebl9J7IS0PRKNVz+ZoZY7C+RqPq7IXjyvcC+C8vnu8gk38yaThQZauwzNM5uYC+y8miBDCnueeEvTJ8SPrOesPdhYLFCxVbq9HyjnLhf5WNHYpVyKO8FG9\\\\\\\/qjbvPzw0I+7\\\\\\\/Y1NEt3SEcZaQPd5smzAe2udyq0dqPahfNNtrRAgdXpWLIcTUL3X5Zs9J+FJxILsh77H67UUBLvR+trwUc5zf9pn8bTSLSLT\\\\\\\/Wd1ToDPUfQMwGw2vINN3M2WcAT\\\\\\\/VIveJFW52\\\\\\\/GFQMeUd7sHGkDUpgUI\\\\\\\/XVLxBShDq3IcMS\\\\\\\/cq0l7dhzPsUh3GbyRbpUoD9LL3hT9zOUU+V7pCtzNOKrhSpIOaGbOeSCMyvgOBhBMyN78mCFaVL3HZKhekEiGRPc4xeDAg+FV10TcI1UpjnTH4gjQgUp5PN6jqyOzs6lb9p7dPn8Kyuy0ZeCY+XncAYD5kbyYX9bcs548R\\\\\\\/HFWlh\\\\\\\/cZXkTu6Sjz2kxGX90fw4lTYJVd2\\\\\\\/\\\\\\\/d+1T9uJLJdzjUZP+JZOEsgQiJgmJsZCz2Xw6+0POgR06qulgo\\\\\\\/MJT0cmGcQWpNWAfVfMpKJDZADZdX++vO8UNd51zbTTNwsX0pb0Ffy+fnqSDQwcHg3HzA+hnb7aUqCHujZI8KPCIRjpyohXI87DnaOfGAzg2H0SGUGhCEm0e1fowKiW8Kz8CUnv7jrFtTtm7kEgYVMYjustQ3QCllHtCiiJ7LaolqHxs78Wb9qDmWUz\\\\\\\/RFxYT83PpZdEZ\\\\\\\/2aGkCrNNl2\\\\\\\/LtLu50eemVVJHwdvAOyDKs12gHJh+gIswIKQSPMZMffBubh2y4fNHV+3IpLj4YF5Qnxeg8qojg+vyg93ryd3k6CM\\\\\\\/HqlZ7U\\\\\\\/b0QaH\\\\\\\/mKbPoJ5Ew0yBa85hwoaf7qrlEvE2\\\\\\\/DBLlIQgiyACQh62tNtVspTPhZPBVGv2UQT44xNb6q1dUoX1I8ZPmM4ziVwhTUMePeAA9qS3rVhlcnMJWi4re2DST4qh\\\\\\\/ZlgumjUYKC5xGEtmjt4M6Y2wYWoPG+HdeHRjcWsf8Uwd3ruXcKZiAm4fYvTFE\\\\\\\/pRaRdv9ZXxVP70DWw25LUL8LBaJ24Px8ZUmzksSotDqwfLKeWHDPEaPIQsjNvuMZw1Qy5Nm3TduykIeXzQqS4bhCoIJFxiioZajlLD9cG23ZSWXX3jKSdWWKawPYv7uYlPdVJ6LiOBQzTKR50HWRaWrfULTiY1kNxJMbQW5qw5tGcpVx3cZHceiqky1N2e0XF2onzS0RA2QGXs2s22wy4RY9gUxnZ1kcaXnpCKfPuUEfy0tHfdOp7XSafPb5wQ3O7VMQE27vkk1bc1ivcVAwSll4b6VL3P5Sa1C3YIlZ2QXaKV+g9OYAeTk1hoyvLmjDRNLnDTfmA4Qi+6oC0wNfQczKJOqgIHj9kxzn3rXSE2v08IhxQEYBGkKjZ6+bbJ3lqoiofWrqlr4g0QuM4Wrh2XXyZPMr9mbgOdgTa9wEMjqo3SJmQHXcQrDeI8tKSkt+JQAfR863Cp+GWuguayy8Uph\\\\\\\/C\\\\\\\/8huj+rf\\\\\\\/B6U\\\\\\\/VAGFWNUKTgwTo9IpkO\\\\\\\/tKNVfc8gN0\\\\\\\/bX5gWlzZ13krsswCJeIf3AahUZU4SOqKRpZoQKHD8SSXlxuYynlw3NR6WIOZAIAXfnNtdg7OUVcYrMXU+mov2autvitJO7RKC4BihE2r3uqLYbMjnZu75plKk7R8DNGnIhW41BatSwucQDOinZhiliNPAQ+EU2BUMQ6B8KTDQutVag42edSrDHzLM0ZfOAWIUVPKzTilCPR0EvHRBtczMtjPTcyuBEajR+uxuoUdnOyEhzsRmjkXLYenQ7d\\\\\\\/bXifyNkjknZ9mnBZDOx8Tg5eUVgc2dVCgCxxkao7QvTmPCfzzognDLosp4xkLjuNggQssXE3rxCOeLMf9DokAKKeetDcHd69TZqInoHmUpNrvn\\\\\\\/SgVyFE0wj8L1AsYLqt9XXwAj5vTFkPpxr1OmpueEJFQz3B\\\\\\\/GlcdWRjWWtEg8ktIiZAnqWuL0wfjzN9tBmEqnfkTrhsvVysGC0Ny+umDyhGRM8X4O2o28IpMseQ6xbJtgNHANK4IibuFvyjjea0P84hzsaHxuOYkqw9eS++i770MZxYvz2EAigsDCZm\\\\\\\/IO7bFAP2oDRwccLR3gXsaFbVQMqKGfB7X6nyOulaYCreypfuoelSa4tctRUHhMrmbTnIiqx2L7ZPMdwk0L3tVDvqWFj+ShiJC6\\\\\\\/qDTCmjMV0YBumZFxggDs2F4VbDuMYyEzEj0ahMUGfJxVyMJzKxtQNX3CHjUs+4LnQsPj0wwE4eEs65iTutlBHsJxWfpRh\\\\\\\/v\\\\\\\/M9nz9LdAiyIIkY7PrSaYomTAhIOn0+DWMjWhtxHBev8XOJvebcfshdCzX8OlomKNQ9BpVhSIhk1y0TpiIGTDKdnQfdj3Y5N8zRsPAa7VK2rZtvN9rfHFzG0OKYBsGXjNjsl3LOFMN+WegyOq1b0t4sivLkB4\\\\\\\/jR83kC45KTu7qXCMMMQPIoZIj1KCbJuWhR2XbVlDS1voYq0Go2Nm6tKJ7MIhjCbgBaubarZq0dHrBmw+dJPubh9L7CAvWP9faKi9DVAqx+sYyHZs8YKc\\\\\\\/63ql8DUyE+gyms4yVzz+ftzoTR5AUGHy+eBe0nl483iuOvcBryo9F8Sy6Keu2LOWIzrHGtzKajVX6bisLoGLUcEjyLCh9El4YSG1Ksjc9A+t\\\\\\\/PfARBZzjWEqjLRY7PJcI6LZA4FDLRY7tIJoKmusO1I8G6L2QQz+N7Vfvy5Pjaay6MySbNiKa8GzXmJhTqn2TvHb0aHZJXX8XG1ty1um6ygrSQtH571vB0rao7GtGFgvS79\\\\\\\/\\\\\\\/TF\\\\\\\/TT25C249nDijj1d6CCbaZ+HuIaprHFOS7NmlVcVGKQ4QnryfLRLo6W0Kt8KJkLEceSYVXKWUCmfKtg5ac0k5SYQE4oPXnecvlFb1UnIbT+tt+dDH2cjW694yYcikc1C2kuYA2GljioWpCN4\\\\\\\/06f3rePAEpvLXMUrMcMstxt0LW\\\\\\\/IewY\\\\\\\/nn97x8SOMEx\\\\\\\/DDvE19mqioeC2DtQr4ygTdqPuKaOpUeE7Ru4L33xKYuhJAHWWKESvJ5uUcX6t1uI4QvGRN1xlqn9kITz6MoIRHer00zU2HbgTDHQ3J9gQXo1dZxv9OiJ0zeaU8ASTfAJzBae+JRZMK4QDElGKg2hL\\\\\\\/6YmkQgn0l5FlgqwOL\\\\\\\/PUHd37lVycrwXRc6fsnPyYH3ziKBKU4U0tYvOyWQPmjzJYttxlGBhBInmVkknWZWgg56L2zRHivnzx9KTrc5XfZF5kq+o0jbXXz4P9eN+4F8KiLfC2msX8Fgxcp5JQTW6vEwCHDZUMBGnADBpoV9PS+9oiL13atRMrgkX1MLZQHeSDZh9tuEYWsFShE3ngEoMdFFWjVEUFUF1GPNZ6lf89JiyzZTagMANCWUhGvmuS+OObxFa1oldALbch\\\\\\\/t+wKwZRrIJTEYGD09GXy\\\\\\\/rG7BUWuad03MY7lXKJLhH8TqgVGetN8ZBpzUQeAOMxRcpbnklEBt\\\\\\\/+u09VNtgwvTX0ocOWlIJ8vdrubpUbf5a+6s0b9FQ2XqdaUDYUfISZIj+xa7NuAQn0oA+XqLbGHEcY0fLbj6B8xJHhQCJkPnpaPY5sjHyQtWhlAfVLhG2a90QByggegEFiVsJYSZSPdTfLqcRsk9N5ayOhOdDaKIF+vNyabahXSqUJ5yfsotOp+e7qxijvOJ7hhUjV7ELNU385SMcUldZ1Sk2m1OVbJ\\\\\\\/DsxnpWDxC0tjVrZKNeIW2yjrCnjZAVVQDnQ131eCEKg94ZsCGFnC\\\\\\\/R5A8u6EJr\\\\\\\/PSymnBjwVSBlAPGL95hmv4IEv1B36IpdFhdD8MD8Rb1l3lcNyldfRrprM7WHLp1bzoL93JrFF4I7eDpcalRXaytw0CG20QpI1cSr+Cm2T9QFhHgQvprQzQFfYjSzI2kcHPYno1PTOdcX71vUNuZzHIc\\\\\\\/lwA6mJK0aylKeEGVUMdW\\\\\\\/MjLkLoXY1ZAWQH9WpMPYtrRseXbmr7nAjOqZil45t0mxzUxglyioDUIQPAAn+V5fTSeHIXr78ExexaX2AX7W8nZsgi1FqctdgTgtZjfNAHNvY5ngJruvFS1ZwFzHW5AYlH8nLqRleXDy3euoR8S0XzhnLdJ9Ag8WfmTh0\\\\\\\/jfiUhwH5ZPHPWNUoIQQ\\\\\\\/bhz9h2nIdspzJZkDwuCwZC+kTYtiAm9hhv2UHAwoHGj+gUfgCJlxV4RfX4IyJK7yzIfbNFogD0dvcxwLqi8uDxpNYIvapyX57N3j5X7OJgoLO+OIoZ5jnsutj7I6LkSE7KwPWhZZn6CwHIPuVP3ywwneEWOtOr2\\\\\\\/0Zc9Q8a8ZlsY2fNDhrW3X0N2Nf\\\\\\\/11Wyuc8P2dYmPdF7tWDyx4w8LXeyIfukFDK\\\\\\\/4xV3Yjg45p4twbiF6OePfn86y5\\\\\\\/BGYfjqLkaTb8oIGqITCHq3MMQYAWgGOYOT\\\\\\\/e9H5Fc4e+rEEr3HVLNX72otwBR6uZrbLjzpN9aS5pfzpIQ+U\\\\\\\/s8nrBOX+Ba5WW5WNGtOHZEf5LUITu+1DOQ4xmILyyRn1r3FOUvhpw5f\\\\\\\/C83SH6GLYo4My\\\\\\\/FNgv4cE1Gyx8AhR\\\\\\\/jOh3PRcmLBJeURPYczqkocOKSmxAEU\\\\\\\/fombfErQxRLar6npN9WJadUCW5UFDuQ8zHPeYsujS48XGEkRJ\\\\\\\/kaMLQsylpwQuxgk0h8+1O5I35ffOI3GMJFqt7BF9TrICB9CtfpCP3u6vkYkIoR7Ndnn\\\\\\\/Pv8vS8ZggDodW8neG717PqAldTAAdyO1l\\\\\\\/c7g9pPE3eJl48hPC3CSw7W9vcF+8zuwZiojf5F48EzoOAFjpMvEdHu\\\\\\\/aIEQy0hHB\\\\\\\/TjnC8TAnaoilv1a\\\\\\\/nvsVFsGourCbAOALkrVy\\\\\\\/KRUbVZF01HAz4tBlpOMrKizRmhEtMwYnEYvoP2SwgTgEUC0gr\\\\\\\/7csNqREkPb1DQbWD3ZVXmETdQ05svJu18CUZWe8Pwh3G2GLuq3MjRb+VFUWLdeMko+QKCp2gkQplT7dQUAeepMtJwSjRfWxw13PqG8JsT41HF1dCn63V2oeY2i0xjTUfCxWknGVUrWlGmDrUvxbwv3vjjWBzqa3\\\\\\\/uLcTa1kJi7bgNIQ+gzPy58j+YuW+jyEAh\\\\\\\/WAwdaw\\\\\\\/ydUzY4BFOf6gl20U70qvCnswoZwJhMZiLTLyNvAiUpO8GRiF9pXnEuXkTdQeEAOgQCD6Rj5rRJVsEHXZ7kQ3yu0i9uPAsiSbGzwqGgpaomzf9F35U8iZ7Fg+EEdc+E7GftaXh7LRB1jM7QyKdG2anbJbqd1XWyCW4EDs\\\\\\\/SYitrCDNqY\\\\\\\/ad+pb7ICBRvdAQR0aNXQ1KL9ynfSdRamOr84vfNrad8\\\\\\\/jli8xsP+OY2EeR+Y4diiwvF3FoOxKZN1xTcFqtRglhmtkvXpIsy2\\\\\\\/yTgwf3CsCt6xll4TRdXS8KJeTe9cEwX5IHCFsA5gkcJsru0M11r0Hiz2zO6yDsFB6BSEgeWvlDNPm4sEY6ImHw86djbcdT4QO8bgYgEKAtg8fPfb\\\\\\\/jVWMQg3zM7LB1RDXnfKCLSmYbELD1xZUlLQdmpdV\\\\\\\/UCAqHc+EvJl8njmmsa0fyZ6Qa8zm1Um99NzNb2wzbIbisHwjgzJIMPGVZ0yvcZYiLCaEqVTZn0ShyqrwxXfNA9Toedf4ePDqW5lz5YdINCETV026cdQiGNwS2JHwAg6ZvGeThBCiPZD0XcYnlOEdXgX0FCDZd3tdttaDUyt+7pCRy8KxAk8gKrzqD0duQIs6dVaspeyF7Um31g8xS9PNTzoK8DMbcD13XmmMQqTblmapsOSW6kNnCts4PiqQnqAih+hhgVBrFPZtcFCcLzvwXCJabiU1jRo6KTmLL5oJPqNO0HOD697PL8tR3Ub9EEkqWx7RzwwBFdbOy9SpBZYQSSA1NAIXoTMYtdzdSxUnNBmXgeuRum0sqkttu7ZVEeZLuj1f6VdoIWTtPe3HjBts6LHSSkzQcUFpsVCb7CuAwaFso3Oo\\\\\\\/zhB8zS2g7iXM2csW+ogaofcLsQReg9uQzPQzqo11seCpuw\\\\\\\/wLYemDUrl\\\\\\\/\\\\\\\/y6xuAambA9AZzKov0nKmnT1R\\\\\\\/peSIaZe3bVyfswfuauxsvG+L+QcBHHk9SZGkqN6RUIrQuPhxMb++rldSaMp5deduS8rPxsuna105sSfx0DMnUVbQ4HXcBLFZrbRsoy7RdnVmQBX+Btxe34IcxWJsXBvykeYJQHbBCMAlx5PgO6Bl+Z8l3xD7PD8ocuvJmNmcPmXlrnPnt6f2kpLKkVpugRdD0LOpNoGYUd2ilm83+FPDRpJsXduRqoQ36TRPIf+uonBRFSxcQmisf8JZznjZ62w6T+hhfuPL4G2YnsPlA91CafohvlKjKi39KsallY7Mq3RAJHf6MySZtdF4LmkRKwUdKDA6tlkfjhEfMmMxNHmkYEW4XXHkJzisvLFvExZH41B1DxHJz9ShOV+sySGShwsFjsjjnImknBOf6p2IkeIGN9fU\\\\\\\/iMuVrZz+tK7MQZfmB4VoB5ha45MiWb5pGd+VX52pMY2n+MOqJFZPoGBVtZwqgNs9fBFgLYEqz4b97Pt+sK6IfwOuvVVhfIxtcRZqDCsAED1E2Atz9AjmIMU+izSrqR0t0WIpPPSlMdERmkJeC20LiF30KL\\\\\\\/KdjtWaWJ1\\\\\\\/eD7hgIcpUN6I09AdxzZdCyh01UGGCrf0GAUIjtq1z4zc5kkMXGWMtBQ6RyWT7OFZ4TGVunpFmLB70jwLzEq9TKCZVz3XIqSKRjPk4b4M2LYYrhDd0dDiN6vdqW7oZ8QtyBwIIdVVDQWVWArq+YeQPD5mr5R0rBRuX2jciTYkq780nC2ezW3v\\\\\\\/V2c99HDtmsEK9J\\\\\\\/lUMWdVudzAbR1oChW7uPNyEWAE1KM8n\\\\\\\/bN0y4si+Ce\\\\\\\/RKwIy2x46txHF76RYTo2+OCqgwmMNx0e39UnzjkiTQzZjFx8ott1ouykQAkveKgnqHpG9x1pLcSILGGu3goDZmRMFvy1qhiYZs\\\\\\\/v8vr2lWq\\\\\\\/sslvEp2DNn7UrnxAFr3YH72FgWg8UfdJPo6X6FQxsKZTcMuLfPAIJdrexZu0LAFk2TvBQDyV0RtCIbWEng\\\\\\\/13xwBrxWBmwQidQZgAOS8zIiguFahhA\\\\\\\/rXKjp6cWzCH9Wqsjmzeed3BLT2a\\\\\\\/EfuYSmVLFsDUlyf3lOEQsvC8I+F1zRMEToeTsXo3sr3VMXM8PfuVmiJhE2RJdtfGA4NtUjw2o9canZv3LNz19YTMHAPrZc+6HRdP5CJ2kMCtJCeNa79o\\\\\\\/Tu07XsXHt\\\\\\\/6krhMnUXq4rFT2\\\\\\\/4lgueOe2GTh1jer+dORQWPl6N2Mb9hLFYKCjD\\\\\\\/Xr2PvpJKqaWDYPXGm8482c9RfXu6mhlyXbzbTA\\\\\\\/FdJqkJYhO9TGhA6IDJZI3dRzJeAGqt+BjN5dPtZ6NS9MKgsyB+RXpy3EGNJfkgu6I5KTAEZXrrb44DabICGjFKM6pXzBT28KFHXgnpsgxkxuqUQKBVZ9CrF+r6NX+0rDKGA2VDWQ2vRIqd7i5W7Gmwm3Vss9Y8h8xzJvk\\\\\\\/Gc\\\\\\\/Ev58yONM1o8IqjCRWcNHS7VzxsESochzu\\\\\\\/npaYpgpQBb39gMf+3eoZc1efV4SWmM+MMegXaIROCnGIsr\\\\\\\/eYMbgJYjRb\\\\\\\/E4nvKui7Sxnod19gJuF2YgUpEj92NIC8AJMEaIUumebg9VvXS14S\\\\\\\/r9A+dTgeUCU27vOFFYhAE1HUDyWNqoEyTCb+OU2C6QCoSfiT3ucGtgzoEwanClYMgi1Pf3eZOPZBej+PU6lZW+1ybFYi4H2w7ctjmW3Xm1HkVaMrzpETaLl8qBXxgiGLVQVh\\\\\\\/emxlVsBYXU88uzhB\\\\\\\/EXyKusS7K4pTNdbwM9GfztXK6BFl0793f02Kj9B19ohzj0v83ZjgD5uZqtf0eBTxaW5vRuJTwPvUIi7aXl8VOAkXGpWpTiRVI2Vd4V8nb\\\\\\\/0DgJagMd3gT+svUSzwXx9+2Pj8Io+XKNXpWT22+ZAWGLkaeSmDfCzwxT859BpT7dXx3\\\\\\\/rpnsaUZmKbyKPOnzqdbpmscR+J6kBXcXd3nwl9NQuKpfUG4HOakItktBKgb7r8zIi+HtRUl\\\\\\\/C0RHjbpRJtCL\\\\\\\/Zye7B+ynQiaFHhLA2Zm8xCxJKhU6b5ycZ8uQjknVoaBeQ8DnIdj3gE1BGzFh13jzvSXIL9RvARcJoqaCwPj57t65CfLIA94o3IaEGFDCHAxpzXWMWWu2DVi8O56M\\\\\\\/aprysdTjudv11OTbjDGQMXHQRrwqsMillrBdf6yuHENhef3kikpyfFTMfXBJ1xNS7BaAAwvRL1yydG5hZbFUYcU5TmkeeS+68MIFeet3\\\\\\\/Te0O9Vg1pUo8V94Li1fFLLIbgdWPQC0LYj7D40g0FaSC2Qt7dk\\\\\\\/BHdFFLdZRjxV2bgRtCnTAcS4NAl7iIRDZTe0ShKbVW8axRuD+tQigdCIGBga5e334Xzza\\\\\\\/KL+0E287NsaxTebMYtkplkPPvUbk6TXeWfQfpI2ENQPU0RLkBN+Ni3EGNU17Q4XUnZIfHK+MYKfmA6AROIsRN26WCF3YiH7FiOmJ36t8tRUnrRV7X0EYBtpEGbHBaMa7F1EKG672PHwDLs\\\\\\\/7CwsA7lArPYR\\\\\\\/ZxssSjfrwI53zXsQ0OJr\\\\\\\/V7qbMY\\\\\\\/VRdzdK9k+sMYmeVtPOvFIRmi5bumzUgf09NFKBn1gvTTzkORTe26T90RzpZZCqOSrtFX2uhpUwIRZOf5qxda95z7vcJEKRJwHT2apvFBGPgx3i8hAXFf4xUz0KF8ZxktPj\\\\\\\/LnQwQeR3uDsrfswo9L2V896tI26RnHCeAw4xeBfaSDR6euXfJiGivCoAofhffETUvIwgVEvzv8bR7Dondu59AoUuEGbRIih3cXQvXmg8yJ68Fke7LtItrnnOqKobaCkNC9nKwQrBdriiqrfUPeyUJG3Agof\\\\\\\/PrmswyCyTma\\\\\\\/xCaOr4cny+3q\\\\\\\/8vA7pytS+IDxixEDUPUYDOWB\\\\\\\/hguaceCxvO8VikOOv\\\\\\\/Es2HNY0lZL0snJS1W68MaPQ2xrYStwsKp5C36HarXFb2ihGpLv8I8KQeX8ctI9IsY3EnMKPNDFSPjUr+vdD3L1StS9HzZfGJDBuPD3BmsVPVdiK+eAX3YS8KwXaE3hHnYDW+oEC1WzFrmfpy\\\\\\\/GtJuhpPzX804nOZ\\\\\\\/5C+nE09VhBmdIAILgQwex5LUcojiG9KJm9fU7hkauV3k+kwnv9hW9Tz2AmgqqI3t9190eq9WdwSjt1gkTBDKQ9k5WJ2aP35sV3TUKMFTPnTUN8qxusDzQ4wmga0kidzUZdKOfkwR07N7DuvGQHAczp9e00FbLjMW6tXAZjY8PM8zniOKf\\\\\\\/IM4J2LC0rloII0lYRcI0iDX1A\\\\\\\/HHxcCXU+JecjOPfsQgABZCNomFXCynznkyPi1ZrhHnrotiVPCLPn10AcUbr7xgEJkr1cHw930UzroBTY4un+GScUtYlxJo+q635hsZO8uLxWx7a3shaOau1ILj4OXS27RNe\\\\\\\/F9ic2yiFkbspO\\\\\\\/OE7reeblz5uLiWhct0E2ov9pXhjxKe\\\\\\\/M5k1XUAZEUZkReG8CdFeER1ZTMBH7gfK0sc\\\\\\\/vJzFUSnMxyuQXeEpwQRoy7PtvJ8\\\\\\\/kx42THycHIaFUPbR845cwG5nCGd+UW9K46YvCRqWoyT0l31yUiL9Mr+ky4dnURzpRtghRdxmG0q4Wq\\\\\\\/kI96LMvVdBjU8EPEp0XOfbRvridlYg01FMNuTzM3V5E0gxF49stUK8MsraUEmuHlm6nOalimlEDzkfxaoerc7Ko0esVbMi9kAz83flYGHbituocFvuKlnFOVjOL6UfByP43u3x4YeAklt2vLbEhb9rz5evFdl58iIv+mjiaSvxD3OE0skS2wtiomJ7wazRqAwayt1mgnrvt+THUex+anB3yQ9YenBrkXOEQsha\\\\\\\/juDwNEqrXEruGIUMgju4G4a8HoGvwqUfIPe0FVNrBHoqqJUxFNWUWxznvmB8kSPbb8H\\\\\\\/4sdegHufYq6fam5JMFHlzyfCiDU+UT80jDjJUKYKycMMHgYkHJEvr8y6wpmLEkFZ9Ayp0m\\\\\\\/tnw+\\\\\\\/oCG5M4BR9khNUALCv7MUBchggvpxStegyobf9vbpFCXGpsWxHwdaLbz+eYfajLhgIJOdWV\\\\\\\/HcLk01SgU+ylOhs0+WRHIEx8PR4NDgzUosDWpOaZxMVauKusogqmWrt9cpOUZTKrHUXDRh+ybCKO+c7OdSIJiYuCEQKKvxRcrpANyBlyUwbNuufrex8\\\\\\\/AM6Bky6vPgO6KA5dlJKUCIcLsSG5zJulnjtTQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nLIuQJbS8xAoqPqSc+QJMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f7fad06d29cff779e1c417ad1b2faa1a29adcd6055696d253eebb7ee2a808c1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022d5xXbn6GxNQBpxA2+lWcoyH5LEqNTlzEKEvHAk0fIHwvEj9CnuKj36wFeBBb9glespwo34TSyN262VBshQKvD4wPMX3tZfkAmTufu+J6uQN7m\\\\\\\/m3GNMSO8xF1F5Gk2Oy\\\\\\\/wz+qLK8BIask3\\\\\\\/+C1NSwj7pXK7zQbdWVOcJg9vEBjTJk2k0NeUSCfJxJrAeXqeMb11MJAtVy2CHsVKr1OmX2geYVp7yrmHL7j\\\\\\\/FIKWmwhTkzYJqMgmOV2o4ye\\\\\\\/\\\\\\\/EjiK2kOl+noGOXxDIBRZRE\\\\\\\/mG+UNnJ1weewmWkgBGX\\\\\\\/Is4aRW4CzpnaMfA8m\\\\\\\/DS+ZonH+6CZ0pZ9ls+Nl6b9XbB5Qk6pM\\\\\\\/zHKzZ2lLsMaffSBRb0FiJdhDABwbdsrx72wExYHMuNkSRqyz3DKyr0pO92hkFWUGBo79cSNbRECW5UQDMWaEJt0SS0FAy3lWDiSi2qaO7OQeGxMumz31Sd7I2cS0YTjDd1FjfSj78jAh65QfNlpNVKc6Opbq94UdaX83FEvSTWTEpOCKIcmzLqBDNeqAFAW+779jqneC4YxPIfEpxVwMoKbA\\\\\\\/50OfsStNC9lVAMWe7+I7qB8OIKJ+a+znOgmAO0wZooDtczEKzYWxYmrPX9lDTII5ZqgTjm2XiM+fayxr5FMplYdnWziMTP52UCAvl5X0LpJONdJx0hUYtO1wQVnRZrRRmgt2oFGsDKrOel5af8qnl2qE6PsePlXi\\\\\\\/8RMhz5pWOCl0xHqNZyWMdf3blwqCnedBNvJBfyCwat7mmXVD9EelcNgDThqia7V\\\\\\\/+VdyLKGWuK3xt26TqzE54+BkTg61vwDrLlAtsILvV5iORWr0486t2HBZsd+RnfXYjW1bTXz1OBd3RKkLLSaRRFQsmcr6k17QJ2HZprfVFq9ZRxUV9weuheAiNc6aarDH44j5VdOcguC9Yb8HNMp3JBNsO\\\\\\\/PEJXln4Z8nBD4I5FWSTyeD3eNf+PpYdPu9xOWrfrPFHnT\\\\\\\/WZM\\\\\\\/KID7SQ+iwy3931oZzL7Y+ED4Ydk89mHHhKc1MFkAfCiXAVshgn2H6I4yNV0UlaP7j6AO3MsL3s8VcGSse0pagR42PxE7HeFhFgbM6V2YFEEkgwdUPecIA\\\\\\\/uctm+M+satP2jt2ucDYoET\\\\\\\/xmd2X+ss30xY5+We\\\\\\\/YGdmkh\\\\\\\/oqhonBy0ks713heouqhsmOskt1at+bK6vlxRtHiuvtWHzKRA0b2Qab3q2EJmDEuHd+zNDTQYY3TnTwHDd\\\\\\\/hl9gjZzd6rCOBQC3nGZuTtWPJKUZA86L35otshWVIQhMT7iZ3ABotRbGaxnjoM9nPltahvSMo6cxUkBelmykOeXJaSavdkbW28PWlWKc4Jb4L8YmrMo0qjXWxcE3tKrEATijGoOXeyxsRuiCc5zr6AuxiNXLxT7\\\\\\\/kv0t4R1ndU86EUPjCvBJC6Qs4yNMZQKWyjWsEiTTWXl9yg+FTP7P56b+nORtO4GkkhJRQrNxUmHE0cfTC7V04ZKqRYdWQWRYVfUtjp7OAOvKFm4p9EsD6LKTbU6Jffk6quO4vINr+M0Wp2Ja9wwIvltq3g0Rez2EcASq9DQ3TtzALvalfkRG6xqKIqiHHrKMCLVb1kjLVqsKY0OljzPHEy8GD6ugJHHeuRjWaKtp4uBgvWB4hmeTuH8DlH+zhpFT8XAnOiZXRRx0I6SXOpxEenbxxaDYw7AEIKwidDRn5jCDOt6NGZ74JtNKmXuMNryyYsvfeAR6qDQqBFgJfnFI6dBI7co409XgL\\\\\\\/VzbGtNoEkweb12QD47lVBkHtYNQnUkYTk2si7zjJHoaOCEcRYIRh3K9CutEe3rOd0pXAZ4LHfAlrOvwBfPpCj99dVuiKCMSPX\\\\\\\/keQeBvjJK692HiLC\\\\\\\/PUO2eZVxUtV+7ZxjD5x\\\\\\\/SlORu4ydmwKKwVg5SI6BkXICuH+39vDJYQL+SvOZRh9ztc8+x8jXwIpNQ8TstAjirLHa3zVjUZLlYZC41\\\\\\\/dYtW\\\\\\\/u6Ii0alhzG9gE0Qoob4IFGdccWkgZ1Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022TDcOHKLsUspbq56LEcy4AA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229f2c0201449fa7cb4cf8715e9f3a0b025cd52d5a867acf2e18d1c9c676c4660a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459453 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/665" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yTHR9Y5MzHw89PcEARpzMM2laqMtgS1xgaBXZiXqS4E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/665" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fDiPsnJQiCkHGbgg9jy5PLKF2xnjIPArX0yibgVY7-k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/657" title="Donor DNA Utilization During Gene Targeting with Zinc-Finger Nucleases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/675" title="Comparative Genomics of Serial Isolates of Cryptococcus neoformans Reveals Gene Associated With Carbon Utilization and Virulence" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459453" id="node459453" data-pisa="ggg;3/4/665" data-pisa-master="ggg;g3.112.005082" data-apath="/ggg/3/4/665.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>ZRT1</em> Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elizabeth K. Engle</span> and <span class="highwire-citation-author" data-delta="1">Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">665-673;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459453" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth K. Engle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Genetics and Genomics Program, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BK.%2BEngle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Engle%20EK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BK.%2BEngle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/4/665" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459453" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/665.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459453" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/665.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459453" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/665.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459453" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/4/665.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459453" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459453/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Estimates of the fraction of nucleotide substitutions driven by positive selection vary widely across different species. Accounting for different estimates of positive selection has been difficult, in part because selection on polymorphism within a species is known to obscure a signal of positive selection among species. While methods have been developed to control for the confounding effects of negative selection against deleterious polymorphism, the impact of balancing selection on estimates of positive selection has not been assessed. In <em>Saccharomyces cerevisiae</em>, there is no signal of positive selection within protein coding sequences as the ratio of nonsynonymous to synonymous polymorphism is higher than that of divergence. To investigate the impact of balancing selection on estimates of positive selection, we examined five genes with high rates of nonsynonymous polymorphism in <em>S. cerevisiae</em> relative to divergence from <em>S. paradoxus</em>. One of the genes, the high-affinity zinc transporter <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> showed an elevated rate of synonymous polymorphism indicative of balancing selection. The high rate of synonymous polymorphism coincided with nonsynonymous divergence among three haplotype groups, among which we found no detectable differences in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> function. Our results implicate balancing selection in one of five genes exhibiting a large excess of nonsynonymous polymorphism in yeast. We conclude that balancing selection is a potentially important factor in estimating the frequency of positive selection across the yeast genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/balancing-selection" class="hw-term hw-article-keyword hw-article-keyword-balancing-selection" rel="nofollow">balancing selection</a></li><li class="kwd"><a href="/keyword/zinc" class="hw-term hw-article-keyword hw-article-keyword-zinc" rel="nofollow">zinc</a></li><li class="kwd"><a href="/keyword/mcdonald-kreitman-test" class="hw-term hw-article-keyword hw-article-keyword-mcdonald-kreitman-test" rel="nofollow">McDonald-Kreitman test</a></li></ul><p id="p-4">The frequency of adaptive substitutions driven by positive selection is central to our understanding of molecular evolution and divergence among species. The neutral theory assumes that most substitutions are effectively neutral and generates predictions that can be tested based on patterns of molecular evolution (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fay and Wu 2003</a>). While many individual genes have been found to deviate from neutral patterns of evolution, the overall impact of positive selection across the genome remains a contentious issue (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hahn 2008</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sella <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nei <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fay 2011</a>).</p><p id="p-5">Genome-wide comparisons of polymorphism <em>vs.</em> divergence have been the primary means of estimating the frequency of positive selection among species. The McDonald-Kreitman (MK) test (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mcdonald and Kreitman 1991</a>) has been used to estimate the frequency of positive selection within protein coding sequences based on an elevated ratio of nonsynonymous to synonymous divergence relative to that of polymorphism. However, applications of the MK test to plant, animal, and microbial genomes have revealed substantial differences in estimates of positive selection among species, ranging from zero to over half of all amino acid substitutions (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fay 2011</a>). While the frequency of positive selection may differ due to a species’ effective population size and species-specific selective pressures (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bachtrog 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gossmann <em>et al.</em> 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Siol <em>et al.</em> 2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Slotte <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gossmann <em>et al.</em> 2012</a>), estimating the frequency of positive selection during divergence among species depends on controlling for the effects of selection on polymorphism within species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fay and Wu 2001</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bierne and Eyre-Walker 2004</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hughes <em>et al.</em> 2008</a>).</p><p id="p-6">Estimates of the frequency of positive selection can be influenced by a number of factors that can make it difficult to detect adaptation when it is present. Slightly deleterious polymorphisms segregate at low frequencies due to weak negative selection and can increase the nonsynonymous-to-synonymous polymorphism ratio to a greater extent than that of divergence. As a consequence, deleterious polymorphism can obscure evidence of positive selection (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fay <em>et al.</em> 2002</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bierne and Eyre-Walker 2004</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Charlesworth and Eyre-Walker 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eyre-Walker and Keightley 2009</a>). Methods have been developed to account for the effects of low-frequency deleterious polymorphism, but, even so, there are still some species with little or no evidence of positive selection (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Fay 2011</a>).</p><p id="p-7">A number of other factors can influence the detection of positive selection through their effects on slightly deleterious polymorphism. Such factors include mating system as well as population size and structure. For example, a decrease in population size can increase the abundance of slightly deleterious polymorphism within a species and obscure evidence of positive selection among species (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eyre-Walker 2002</a>). In humans, there is little evidence for an excess of nonsynonymous divergence, yet it has been estimated that up to 40% of amino acid substitutions could have been driven by positive selection without being detected (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Eyre-Walker and Keightley 2009</a>). Controlling for these additional factors is often difficult as it requires specific knowledge of the species being examined and its population history.</p><p id="p-8">Another factor that has received less attention but can also influence estimates of positive selection is balancing selection (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wright and Andolfatto 2008</a>). The maintenance of multiple nonsynonymous polymorphisms within a species by balancing selection could increase the genome-wide ratio of nonsynonymous to synonymous polymorphism within a species above the ratio of nonsynonymous-to-synonymous divergence among species. Elevated rates of nonsynonymous polymorphism may also occur because of local adaptation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth <em>et al.</em> 1997</a>). If an appreciable number of genes are involved in adaptive divergence between different populations of the same species, genome-wide estimates of the frequency of positive selection among species could be substantially underestimated.</p><p id="p-9">The yeast <em>Saccharomyces cerevisiae</em> is one species with little or no evidence of positive selection based on the MK test (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elyashiv <em>et al.</em> 2010</a>). In contrast to other species that lack evidence of positive selection (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Foxe <em>et al.</em> 2008</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gossmann <em>et al.</em> 2010</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gossmann <em>et al.</em> 2012</a>), its large effective population size ensures the efficient removal of weakly deleterious mutations and the ability to fix weakly advantageous mutations. However, <em>S. cerevisiae</em> also exhibits strong population structure, potentially facilitated by its low rate of outcrossing, <em>i.e.</em>, mating between unrelated parents (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ruderfer <em>et al.</em> 2006</a>), low rate of migration, or local adaptation to the diverse array of environments from which it has been isolated (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fay and Benavides 2005</a>). Genome-wide patterns of population structure have revealed a number of genetically differentiated groups, including strains originating from sake in Japan, vineyards in Europe, and oak trees in North America (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schacherer <em>et al.</em> 2009</a>). While these groups may have arisen as a result of geographic barriers, they might also have arisen as a consequence of domestication or adaptation to human-modified environments (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Fay and Benavides 2005</a>). However, even when these groups are taken into consideration and examined separately, the ratios of nonsynonymous to synonymous polymorphism within or between groups are higher than the ratio of nonsynonymous to synonymous divergence among species (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Elyashiv <em>et al.</em> 2010</a>).</p><p id="p-10">In this study, we tested the hypothesis that genes with a large excess of nonsynonymous polymorphism are underbalancing selection. We reasoned that such genes have a disproportionate effect on estimates of positive selection and should be considered separately if underbalancing selection. We examined five genes that were previously shown to contain a large excess of nonsynonymous to synonymous polymorphism (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>). To distinguish between purifying selection and balancing selection on nonsynonymous polymorphism, we examined rates of synonymous polymorphism as negative selection is expected to decrease linked neutral variation, whereas balancing selection is expected to increase linked neutral variation (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Charlesworth <em>et al.</em> 1997</a>). We found one of the genes, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, showed a significantly elevated rate of synonymous polymorphism based on the Hudson-Kreitman-Aguade (HKA) test (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hudson <em>et al.</em> 1987</a>), consistent with balancing selection. Our results show that a large number of amino acid polymorphisms can occur at certain loci, underbalancing selection.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Polymorphism and divergence data</h3><p id="p-11">Data were collected for five genes that were previously found to exhibit an excess of nonsynonymous polymorphism in two studies (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>) and 30 randomly selected control genes, using 36 <em>S. cerevisiae</em> strains with genome sequence data (<a href="http://www.g3journal.org/content/suppl/2013/03/07/g3.112.005082.DC1/005082SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS1.xlsx">Table S1</a>). Twenty-seven of the genome sequences were accessed through a BLAST server (<a href="http://www.moseslab.csb.utoronto.ca/sgrp/">www.moseslab.csb.utoronto.ca/sgrp/</a>), and the other 9 were accessed through the <em>Saccharomyces</em> Genome Database (<a href="http://www.yeastgenome.org">www.yeastgenome.org</a>). For each gene, sequences homologous to the coding region of the reference genome (S288C) were aligned using Clustal X version 2.0 software (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Larkin <em>et al.</em> 2007</a>)<span class="underline">.</span> Strains with sequences that were &lt;99% and &lt;90% of the S288C sequence length for the neutral and selected genes, respectively, were removed. The cutoff of &lt;90% was used to accommodate <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005441">IRA2</a></em>, for which few strains had BLAST hits covering the entire 9.2 kb of coding sequence present in the reference genome. Strains were removed from a gene analysis if a polymorphism led to an internal stop codon (4 cases), while unique single-base insertions were considered sequencing errors and the base was removed from the sequence (10 cases). A small number of heterozygous sites were present within the strains Vin13, VL3, and LalvinAQ23. At these sites, we randomly selected one of the two observed nucleotides to represent the position. Divergence was measured by comparison to the CBS432 strain of <em>Saccharomyces paradoxus</em>.</p><p id="p-12">The final dataset included an average of 29.9 strain alleles per gene, ranging from 23 to 36, and the five-gene set included an average of 21.4 strain alleles per gene, ranging from 18 to 24. Eight of the control genes were removed from analysis because they had few strains with sufficient sequence coverage or multiple strains with frameshifts. One gene, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004254">RPS28B</a>, was removed because it showed evidence of introgression between species (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Doniger <em>et al.</em> 2008</a>). The final dataset is available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Population genetic analysis</h3><p id="p-13">MK tests were conducted using the number of nonsynonymous and synonymous polymorphic sites and fixed differences calculated using DnaSP version 5.10.01 software (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Librado and Rozas 2009</a>). The weighted neutrality index (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Stoletzki and Eyre-Walker 2011</a>) was estimated by the equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/665/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/665/embed/mml-math-1.gif"/></noscript></span></span>where <em>P</em> and <em>D</em> are the number of polymorphic sites and fixed differences, respectively; subscripts <em>s</em> and <em>n</em> indicate synonymous and nonsynonymous changes, respectively; and <em>i</em> indicates the <em>i</em>th gene.</p><p id="p-14">HKA tests were conducted using maximum likelihood HKA test (MLHKA version 2.0) software (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wright and Charlesworth 2004</a>) with rates of synonymous polymorphism and divergence obtained from DnaSP (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS2.xlsx">Table S2</a>). Each of the five genes found to be significant by the MK test were compared to 21 control genes, covering 22,974 sites, using the MLHKA test. The program was run with a chain length of 100,000 interactions for all analyses.</p><p id="p-15">For analysis of the region surrounding <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a> t</em>hrough <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em> (∼11 kb), we downloaded <em>S. cerevisiae</em> and <em>S. paradoxus</em> strain sequences from the <em>Saccharomyces</em> Genome Resequencing Project (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>). A sliding window analysis of polymorphism and divergence was calculated with DnaSP using 36 <em>S. cerevisiae</em> strains and 1 <em>S. paradoxus</em> strain, CBS432, with gaps in the alignment excluded. Because of difficulty in aligning the 4.67-kb noncoding region between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, we used only ∼200 bases downstream of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and 800 bases upstream of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, where we were confident of alignment.</p><p id="p-16">Bootstrapped neighbor-joining trees for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> and the concatenated control gene set were constructed using MEGA5 and pairwise gap removal (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tamura <em>et al.</em> 2011</a>).</p></div><div id="sec-4" class="subsection"><h3>Strain construction and phenotype analysis</h3><p id="p-17"><em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> was deleted in YJF186 (YPS163 background, <em>Mat a</em>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002386">HO</a>::<em>dsdAMX4</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-140</em>) by using the kanMX deletion cassette (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wach <em>et al.</em> 1994</a>). Three <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles were integrated into this strain at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em> locus by amplifying the entire <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> gene region, including 878 bases of the 5′ noncoding region and the entire 195 bases of the 3′ noncoding region, as well as 186 bases of the 3′ gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em>, using primers with homology to pRS306, and transforming the product along with the yeast integrative plasmid pRS306 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sikorski and Hieter 1989</a>). Integration of these constructs at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a></em> locus was achieved by selection on plates lacking uracil, and each transformant was confirmed by PCR. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles were found to have between 1 and 3 mutations. However, most alleles had only single synonymous changes or changes within the 5′ or 3′ regions, and no mutations were shared among the alleles, including the replicated transformants. These mutations were considered not functional because of the lack of any phenotypic effects. Wild-type (YJF186) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> deletion strains were integrated with the empty plasmid pRS306 as a control.</p><p id="p-18">Experiments comparing growth under low zinc conditions were conducted using low zinc media (LZM) composed of 0.17% yeast nitrogen base without amino acids, (NH)<sub>2</sub>SO, or zinc (MP Biomedicals); 0.5% (NH4)<sub>2</sub>SO<sub>4</sub>; 20 mM trisodium citrate, pH 4.2; 2% glucose; 1 mM Na<sub>2</sub>EDTA; 25 µM MnCl<sub>2</sub>; and 10 µM FeCl<sub>3</sub>, as previously described (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gitan <em>et al.</em> 1998</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gitan <em>et al.</em> 2003</a>). Strains were grown overnight in LZM, washed, diluted to a starting optical density (OD) of 0.05 (absorbance at 600 nm) in fresh LZM with 0.1 mM of ZnCl<sub>2</sub>, and then grown for 20 hr in a plate reader at 30°C with shaking at 1200 rpm (iEMS model 1400; Thermo Lab Systems, Helsinki, Finland). For each strain, the maximum OD was determined after normalization to the initial cell concentration. For each <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> construct and controls, 3–9 independent transformants were phenotyped.</p><p id="p-19">Rates of fermentation were measured using grape juice. Strains were grown overnight in Reserve Chardonnay grape juice (Winexpert, Port Coquitiam, BC, Canada), washed, and diluted to a starting OD of 0.1 in fresh grape juice or grape juice with metal chelators (20 mM trisodium citrate, pH 4.2, and 1 mM Na<sub>2</sub>EDTA). Fermentation was conducted in 250-ml flasks sealed with airlocks and incubated at room temperature, out of direct sunlight without shaking. Flasks were weighed daily to determine CO<sub>2</sub> loss and shaken once daily, immediately following measurement. Four independent transformants were examined for each construct.</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>Identification of genes exhibiting an excess of amino acid polymorphism</h3><p id="p-20">From two previous independent genome-wide screens based on the McDonald-Kreitman (MK) test, we identified five genes that were significant (<em>P</em> &lt; 0.001) in both studies (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>). The five genes were <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005441">IRA2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006398">OPT2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000113">PEP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002312">SAS10</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>. All of the genes showed a ratio of nonsynonymous to synonymous polymorphism (<em>P<sub>n</sub></em>/<em>P<sub>s</sub></em>) that was more than twofold greater than that of divergence (<em>D<sub>n</sub></em>/<em>D<sub>s</sub></em>). We repeated the MK test using a strain set composed of 36 strains for which genome sequences are publicly available (see <em>Materials and Methods</em> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS1.xlsx">Table S1</a>), of which 29 were not included in previous screens. All five genes retained significance according to the MK test results (<em>P</em> &lt; 0.05, Bonferroni corrected) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). As a control, we randomly selected 21 genes from those not significant in either previous study. Only two of the genes were significant according to the MK test (<em>P</em> &lt; 0.05, Bonferroni corrected) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS2.xlsx">Table S2</a>), both were characterized as having a <em>P<sub>n</sub></em>/<em>P<sub>s</sub></em> ratio greater than that of <em>D<sub>n</sub></em>/<em>D<sub>s</sub></em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462708/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462708/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">McDonald-Kreitman (MK) test results</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">The two gene sets exhibited marked differences in their neutrality indexes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The weighted neutrality index (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Stoletzki and Eyre-Walker 2011</a>) is 3.80 for the five selected genes and 1.33 for the 21 control genes. The high neutrality index of the five genes, indicating an excess of nonsynonymous polymorphism, is unlikely a consequence of selective pressure on synonymous sites as average codon bias is similar between the two groups, and the five genes have nearly equal numbers of changes to preferred and unpreferred codons for both polymorphism, 63 and 58 respectively, and divergence, 414 and 397, respectively. Thus, the five-gene set is highly enriched for genes with an excess of nonsynonymous polymorphism relative to that of divergence.</p></div><div id="sec-7" class="subsection"><h3>Balancing selection in <em>ZRT1</em></h3><p id="p-23">A high ratio of nonsynonymous to synonymous polymorphism can result from slightly deleterious nonsynonymous mutations that contribute to polymorphism but not divergence or from a recent loss of functional constraint. In either scenario, the rate of synonymous polymorphism should not be affected. Alternatively, a high rate of nonsynonymous polymorphism can result from balancing selection on multiple nonsynonymous alleles. If balancing selection is responsible for the elevated rate of nonsynonymous polymorphism, rates of linked synonymous polymorphism should also be elevated (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Charlesworth <em>et al.</em> 1997</a>).</p><p id="p-24">To test whether the rate of synonymous polymorphism was elevated in any of the five genes, we used the HKA test (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hudson <em>et al.</em> 1987</a>). Using a maximum likelihood MLHKA test, we found only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> showed a significantly elevated rate of synonymous polymorphism in comparison to that of the control gene set (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS3.xlsx">Table S3</a>). <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows that of all the genes we tested, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> was characterized by an exceptionally high rate of synonymous polymorphism. One gene in the control gene set, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003441">TRX2</a></em>, appeared to be an outlier characterized by both a high rate of synonymous polymorphism and a low rate of synonymous divergence. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003441">TRX2</a></em> is nominally significant for an excess synonymous polymorphism relative to the remaining neutral gene set, by MLHKA test results (<em>P</em> = 0.0132). However, removal of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003441">TRX2</a></em> from the neutral gene set only increased the significance of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>. Thus, of the five genes, only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> exhibits evidence of balancing selection.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/665/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synonymous polymorphism vs. divergence. Synonymous nucleotide diversity (&#x3C0;syn) vs. synonymous nucleotide divergence (Ksyn) is shown for the five selected genes (red), the 21 control genes (black), and the three genes neighboring ZRT1 (blue). Nucleotide diversity was measured by the average number of pairwise differences among strains of S. cerevisiae, and nucleotide divergence was measured by differences between S. cerevisiae and S. paradoxus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1599637268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Synonymous polymorphism &lt;em&gt;vs.&lt;/em&gt; divergence. Synonymous nucleotide diversity (&#x3C0;&lt;sub&gt;syn&lt;/sub&gt;) &lt;em&gt;vs.&lt;/em&gt; synonymous nucleotide divergence (&lt;em&gt;K&lt;/em&gt;&lt;sub&gt;syn&lt;/sub&gt;) is shown for the five selected genes (red), the 21 control genes (black), and the three genes neighboring &lt;em&gt;ZRT1&lt;/em&gt; (blue). Nucleotide diversity was measured by the average number of pairwise differences among strains of &lt;em&gt;S. cerevisiae&lt;/em&gt;, and nucleotide divergence was measured by differences between &lt;em&gt;S. cerevisiae&lt;/em&gt; and &lt;em&gt;S. paradoxus&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/665/F1.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/665/F1.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/665/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/665/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462650" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-25" class="first-child">Synonymous polymorphism <em>vs.</em> divergence. Synonymous nucleotide diversity (π<sub>syn</sub>) <em>vs.</em> synonymous nucleotide divergence (<em>K</em><sub>syn</sub>) is shown for the five selected genes (red), the 21 control genes (black), and the three genes neighboring <em>ZRT1</em> (blue). Nucleotide diversity was measured by the average number of pairwise differences among strains of <em>S. cerevisiae</em>, and nucleotide divergence was measured by differences between <em>S. cerevisiae</em> and <em>S. paradoxus</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Balancing selection of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> is also supported by patterns of nonsynonymous and synonymous divergence among strains. A neighbor-joining tree of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> shows that all of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles, except for the EC1118 allele, cluster into three groups distinguished by multiple nonsynonymous and synonymous differences (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). With the exception of the wine strain group, the groups showed no close correspondence to the source from which each strain was obtained or to a neighbor-joining tree generated from the concatenated control gene set (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/FigureS1.pdf">Figure S1</a>). Of the 111 polymorphic sites used to generate the tree, 86 can be placed on a single branch without homoplasious traits, of which 24 nonsynonymous and 17 synonymous changes occurred on one of the four main internal branches (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). In comparison, 62 synonymous but only 12 nonsynonymous differences separate <em>S. cerevisiae</em> from <em>S. paradoxus</em>. Thus, the high ratio of nonsynonymous to synonymous polymorphism is not limited to external branches, as would be expected to occur if most nonsynonymous polymorphisms were deleterious.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/665/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining tree of ZRT1 is shown along with bootstrap values greater than 90% (gray). S. cerevisiae st rains are color coded by class (see color key). The position of the branch leading to S. paradoxus (dashed line) is not drawn to scale. The number of nonsynonymous/synonymous/complex (two changes within a codon) changes unique to each of the four main lineages are listed along the respective branches. Inset (at right) shows the unrooted neighbor-joining tree of the concatenated 21-control gene set drawn to the same scale and for the same strains as the ZRT1 tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1599637268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Neighbor-joining tree of &lt;em&gt;ZRT1&lt;/em&gt; is shown along with bootstrap values greater than 90% (gray). &lt;em&gt;S. cerevisiae&lt;/em&gt; st rains are color coded by class (see color key). The position of the branch leading to &lt;em&gt;S. paradoxus&lt;/em&gt; (dashed line) is not drawn to scale. The number of nonsynonymous/synonymous/complex (two changes within a codon) changes unique to each of the four main lineages are listed along the respective branches. Inset (at right) shows the unrooted neighbor-joining tree of the concatenated 21-control gene set drawn to the same scale and for the same strains as the &lt;em&gt;ZRT1&lt;/em&gt; tree.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/665/F2.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/665/F2.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/665/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/665/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462824" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-27" class="first-child">Neighbor-joining tree of <em>ZRT1</em> is shown along with bootstrap values greater than 90% (gray). <em>S. cerevisiae</em> st rains are color coded by class (see color key). The position of the branch leading to <em>S. paradoxus</em> (dashed line) is not drawn to scale. The number of nonsynonymous/synonymous/complex (two changes within a codon) changes unique to each of the four main lineages are listed along the respective branches. Inset (at right) shows the unrooted neighbor-joining tree of the concatenated 21-control gene set drawn to the same scale and for the same strains as the <em>ZRT1</em> tree.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The presence of intermediate frequency alleles, many of which contribute to the unique grouping of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles, also supports balancing selection. For synonymous sites, results of Tajima’s D test were positive across all strains (D = 0.718, <em>P</em> &gt; 0.10) but negative within each of the three <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> strain groups (M22 group D = −1.265; YPS163 group D = −0.59894; and S288C group D = −0.61182; all <em>P</em> &gt; 0.10). In comparison, the average D result of the control gene set was −0.294, and only four of the genes had positive D values greater than 0.3. These results further highlight the unique pattern of variation present in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>.</p></div><div id="sec-8" class="subsection"><h3>Regional variation around <em>ZRT1</em></h3><p id="p-29">The elevated rate of synonymous polymorphism in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> could be a consequence of balancing selection on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> amino acid polymorphism but also could be caused by selection on its promoter or on adjacent genes. To determine whether the signal of balancing selection extends into adjacent genes and gene regions, we applied the MLHKA test to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em>, the two genes adjacent to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>. Only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> was significant in comparison to the control gene set (MLHKA test, <em>P</em> = 0.0007) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS3.xlsx">Table S3</a>) and was characterized by both high rates of polymorphism and also low rates of divergence at synonymous sites (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Hence, we also tested <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a></em>, the next gene adjacent to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em>, and found no significant departure from neutrality as measured by the MLHKA test. Additionally, none of the three adjacent genes that were examined showed a significant excess of amino acid polymorphism as measured by the MK test (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS2.xlsx">Table S2</a>).</p><p id="p-30">To more precisely track the signal of balancing selection within and around <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, we used a sliding window analysis of polymorphism to divergence, including both coding and noncoding regions. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows that the highest rate of polymorphism occurred within the coding region of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> and extended into its 5′ noncoding region. The overall rate of polymorphism is much lower in the two adjacent genes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em>. In <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em>, the rate of divergence is also quite low and likely contributes to the significance of the MLHKA test. Interestingly, a portion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a></em> had a very low rate of polymorphism, whereas its more distal portion had another peak of polymorphism. Based on the sliding window analysis and the MLHKA test results, the signature of balancing selection appears to be concentrated at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> locus.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/665/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sliding window analysis of polymorphism and divergence within and around ZRT1. The sliding window plot includes ZRT1 and three neighboring genes, with their positions and orientations indicated below the graph. Polymorphism (solid line) and divergence from S.paradoxus (dashed line) are shown for a window size of 200 bp and step size of 50 bp. A break is shown between ADH4 and ZRT1 where ~3600 intergenic bases were excluded because of uncertainty in the alignment with S. paradoxus. The average nucleotide diversity of synonymoussites for the control gene set is indicated by the gray horizontal line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1599637268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sliding window analysis of polymorphism and divergence within and around &lt;em&gt;ZRT1&lt;/em&gt;. The sliding window plot includes &lt;em&gt;ZRT1&lt;/em&gt; and three neighboring genes, with their positions and orientations indicated below the graph. Polymorphism (solid line) and divergence from &lt;em&gt;S.paradoxus&lt;/em&gt; (dashed line) are shown for a window size of 200 bp and step size of 50 bp. A break is shown between &lt;em&gt;ADH4&lt;/em&gt; and &lt;em&gt;ZRT1&lt;/em&gt; where ~3600 intergenic bases were excluded because of uncertainty in the alignment with &lt;em&gt;S. paradoxus&lt;/em&gt;. The average nucleotide diversity of synonymoussites for the control gene set is indicated by the gray horizontal line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/665/F3.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/665/F3.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/665/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/665/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462686" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-31" class="first-child">Sliding window analysis of polymorphism and divergence within and around <em>ZRT1</em>. The sliding window plot includes <em>ZRT1</em> and three neighboring genes, with their positions and orientations indicated below the graph. Polymorphism (solid line) and divergence from <em>S.paradoxus</em> (dashed line) are shown for a window size of 200 bp and step size of 50 bp. A break is shown between <em>ADH4</em> and <em>ZRT1</em> where ~3600 intergenic bases were excluded because of uncertainty in the alignment with <em>S. paradoxus</em>. The average nucleotide diversity of synonymoussites for the control gene set is indicated by the gray horizontal line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We next examined the degree to which polymorphism within <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> is independent of polymorphism within adjacent genes. There is ample evidence for recombination within and around <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>. Across the entire region, from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a></em> through <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em>, there have been a minimum of 26 recombination events based on the four-gamete test (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hudson and Kaplan 1985</a>). As expected in the presence of recombination, the genealogies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em> differed from that of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/FigureS2.pdf">Figure S2</a>), although all three genes showed a similar grouping of wine strains. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> was the most similar to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> but had less divergence. As measured by the HKA test, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> showed significantly elevated rates of polymorphism compared to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em> (<em>P</em> = 0.0087) but not <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a></em> (<em>P</em> &gt; 0.05).</p></div><div id="sec-9" class="subsection"><h3><em>ZRT1</em> alleles confer no detectable phenotype differences</h3><p id="p-33">If selection has acted on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, then different alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> should confer different phenotypes. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> is a high-affinity zinc transporter that is activated only when zinc levels are very low and facilitates growth under limiting zinc conditions (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhao and Eide 1996</a>). We compared the effects of three <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles integrated into a strain in which the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> gene was deleted. The three alleles were from the S288C (laboratory), M22 (wine), and YPS163 (nature) strains and were selected as representatives from the three major groups of strains (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). While deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> resulted in a significant growth defect in zinc-limiting conditions and each of the three alleles rescued the growth deficiency, we found no significant difference among the three <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles for maximum growth (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>) or growth rate (not shown).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/665/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of strain-specific ZRT1 alleles on growth under low-zinc conditions. The maximum cell density under low-zinc conditions is shown for YPS163 with an unmodified ZRT1 allele (WildType), the same strain with a deletion of ZRT1 (ZRT1 deletion), and three ZRT1 alleles integrated into the ZRT1 deletion strain (YPS163, S288C, and M22). The three integrated alleles represent alleles from the three major strain groupings based on ZRT1. Error bars show the 95% confidence interval of the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1599637268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of strain-specific &lt;em&gt;ZRT1&lt;/em&gt; alleles on growth under low-zinc conditions. The maximum cell density under low-zinc conditions is shown for YPS163 with an unmodified &lt;em&gt;ZRT1&lt;/em&gt; allele (WildType), the same strain with a deletion of &lt;em&gt;ZRT1&lt;/em&gt; (&lt;em&gt;ZRT1&lt;/em&gt; deletion), and three &lt;em&gt;ZRT1&lt;/em&gt; alleles integrated into the &lt;em&gt;ZRT1&lt;/em&gt; deletion strain (YPS163, S288C, and M22). The three integrated alleles represent alleles from the three major strain groupings based on &lt;em&gt;ZRT1&lt;/em&gt;. Error bars show the 95% confidence interval of the mean.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/665/F4.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/665/F4.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/665/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/665/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462684" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-34" class="first-child">Effects of strain-specific <em>ZRT1</em> alleles on growth under low-zinc conditions. The maximum cell density under low-zinc conditions is shown for YPS163 with an unmodified <em>ZRT1</em> allele (WildType), the same strain with a deletion of <em>ZRT1</em> (<em>ZRT1</em> deletion), and three <em>ZRT1</em> alleles integrated into the <em>ZRT1</em> deletion strain (YPS163, S288C, and M22). The three integrated alleles represent alleles from the three major strain groupings based on <em>ZRT1</em>. Error bars show the 95% confidence interval of the mean.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">In addition to its requirements for growth, zinc is an essential cofactor for many enzymes, including alcohol dehydrogenase, and has been shown to influence rates of fermentation (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">De Nicola and Walker 2011</a>). To test whether the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles affect rates of fermentation, we measured CO<sub>2</sub> release during fermentation of grape juice into wine. In the presence of metal chelators, deletion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> had a dramatic effect on the rate of fermentation, but no differences were found among the three <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles tested (analysis of variance [ANOVA], <em>P</em> &gt; 0.05) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). No differences in rates of fermentation were found among any of the four strains in grape juice without chelators at any of the time points (ANOVA, <em>P</em> &gt; 0.05).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/665/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effects of strain-specific ZRT1 alleles on fermentation rate. Fermentation rate, measured by CO2 release (grams per hour), for strains grown in grape juice containing metal chelators. All strains have ZRT1 deleted, and three have either an S288C (orange), YPS163 (yellow), or M22 (green) allele of ZRT1 inserted at the URA3 locus. Lines show the average of four replicates, each from an independent transformation. Standard deviations are not shown for clarity and average between 0.0028 and 0.0056 for the four strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1599637268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Effects of strain-specific &lt;em&gt;ZRT1&lt;/em&gt; alleles on fermentation rate. Fermentation rate, measured by CO&lt;sub&gt;2&lt;/sub&gt; release (grams per hour), for strains grown in grape juice containing metal chelators. All strains have &lt;em&gt;ZRT1&lt;/em&gt; deleted, and three have either an S288C (orange), YPS163 (yellow), or M22 (green) allele of &lt;em&gt;ZRT1&lt;/em&gt; inserted at the &lt;em&gt;URA3&lt;/em&gt; locus. Lines show the average of four replicates, each from an independent transformation. Standard deviations are not shown for clarity and average between 0.0028 and 0.0056 for the four strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/665/F5.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/665/F5.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/665/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/665/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462710" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-36" class="first-child">Effects of strain-specific <em>ZRT1</em> alleles on fermentation rate. Fermentation rate, measured by CO<sub>2</sub> release (grams per hour), for strains grown in grape juice containing metal chelators. All strains have <em>ZRT1</em> deleted, and three have either an S288C (orange), YPS163 (yellow), or M22 (green) allele of <em>ZRT1</em> inserted at the <em>URA3</em> locus. Lines show the average of four replicates, each from an independent transformation. Standard deviations are not shown for clarity and average between 0.0028 and 0.0056 for the four strains.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The ability of each <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> allele to rescue the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> deletion phenotypes indicates that none of the 38 amino acid polymorphisms that distinguish these three <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles caused a substantial loss of function as measured by growth or fermentation rate under the conditions assayed.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-38">Application of the MK test to a variety of species has revealed substantial differences in the estimated frequency of positive selection on protein coding sequences (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Fay 2011</a>). While differences in effective population size are capable of explaining some of the differences among species (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Eyre-Walker and Keightley 2009</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Gossmann <em>et al.</em> 2010</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Halligan <em>et al.</em> 2010</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Siol <em>et al.</em> 2010</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Slotte <em>et al.</em> 2010</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Gossmann <em>et al.</em> 2012</a>), a small effective population cannot explain the absence of evidence for positive selection in yeast. In this study, we examined whether balancing selection can explain the high rate of nonsynonymous polymorphism observed in a small set of genes exhibiting a disproportionately large excess of nonsynonymous polymorphism, as this could obscure evidence of positive selection in <em>S. cerevisiae</em>. We showed that one of the five genes tested exhibited a significantly elevated rate of synonymous polymorphism, indicative of balancing selection. While patterns of polymorphism and divergence around <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> suggest that nonsynonymous polymorphism within <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> itself is the most likely target of balancing selection, we found no functional differences among three alleles, using two different phenotype assays. Our results illustrate how balancing selection might obscure a signal of positive selection.</p><div id="sec-11" class="subsection"><h3>Balancing selection in <em>ZRT1</em></h3><p id="p-39">Evidence for balancing selection on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> is based on an elevated rate of synonymous polymorphism as measured by the HKA test (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1/TableS3.xlsx">Table S3</a>), a high ratio of polymorphism to divergence that is centered on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), an increased frequency of intermediate frequency alleles, and the coincidence of multiple synonymous and nonsynonymous changes that distinguish three groups of strains (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). However, it is worth noting that balancing selection in the general sense (<em>i.e.</em>, selective maintenance of distinct alleles) can result from temporal or spatial variation in selection coefficients as well as heterozygote advantage. Local adaptation, which can result from spatial variation in selection coefficients, also provides an explanation for the presence of multiple nonsynonymous differences among alleles. Yet, our results are not able to distinguish between these different forms of selection, but rather distinguish them from patterns that can be explained by population structure, loss of selective constraint, and selection on adjacent genes.</p><p id="p-40">In <em>S. cerevisiae</em>, there is extensive population structure related to both geographic origin and the ecological source from which each strain was isolated (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Fay and Benavides 2005</a>; <a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Schacherer <em>et al.</em> 2009</a>), which are frequently correlated with one another. Such groups include sake strains from Japan, oak tree strains from North America, and strains isolated from Europe or vineyards. The neighbor-joining tree of the 21 control genes generally recapitulates these previously defined groups. While the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> tree bears some resemblance to that of the control gene set, particularly the vineyard group, the three main groups of strains that differ at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> are not obviously related by either their geographic origin or the ecological source from which they were isolated. More importantly, population structure by itself does not explain the elevated rates of polymorphism at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> relative to the 21-control-gene set. In support of selection acting at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, we observed negative Tajima’s D values for most of the control gene set but a positive Tajima’s D value at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, consistent with balancing selection.</p><p id="p-41">Loss of functional constraint or weak negative selection is another explanation for an excess of nonsynonymous polymorphism as measured by the MK test. Genome-wide estimates in yeast suggest that much of the nonsynonymous polymorphism may be weakly deleterious (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Elyashiv <em>et al.</em> 2010</a>). In the case of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, we cannot exclude the possibility that some of the nonsynonymous polymorphisms are neutral or slightly deleterious. However, two lines of evidence indicate that at least some of the nonsynonymous changes within <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> have been underbalancing selection. First, many neutral and most deleterious polymorphisms are expected to be rare and present only in a small number of strains. While 78% of nonsynonymous alleles are at less than 10% frequency in the 21-control-genee set, only 38% of nonsynonymous polymorphism are at less than 10% frequency in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>. In addition, of the nonsynonymous changes that are specific to one or more lineages, 55% are positioned along the four internal branches that distinguish the three major groups of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles. Second, neither loss of constraint or negative selection on nonsynonymous polymorphism should increase variation at linked synonymous sites.</p><p id="p-42">While patterns of variation within and around <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> indicated that it is the most likely target of balancing selection, selection on linked sites could have influenced observed patterns of variation at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>. Patterns of polymorphism within <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em>, the adjacent gene, indicated no excess of synonymous or nonsynonymous polymorphism. However, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em> may have experienced a recent selective sweep as there is evidence of positive selection during <em>S. cerevisiae</em> and <em>S. paradoxus</em> divergence, both within its coding region and within the intergenic region between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003223">FZF1</a></em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sawyer <em>et al.</em> 2005</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Engle and Fay 2012</a>). Patterns of polymorphism within the adjacent genes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a></em> are more complex. Both genes show rates of synonymous polymorphism that are higher than those of the 21-control-genee set, except for the outlier gene <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003441">TRX2</a></em>. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a></em> shows regions with high and low polymorphism levels, but the region closest to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> has the lower rate of polymorphism (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> shows a significantly elevated rate of synonymous polymorphism relative to divergence by the HKA test. Yet, in comparison to other regions (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>) the significance of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> appears to be a partial consequence of the low rate of synonymous divergence. These observations combined with the ample evidence for recombination within the region indicate that while sites within <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003226">MNT2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> may have also been under selection, selection on linked sites in adjacent regions are unlikely to be solely responsible for the high rate of synonymous polymorphism at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>.</p><p id="p-43">Relevant to the possibility of selection on adjacent genes, there are functional links between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> are both activated by <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003592">ZAP1</a></em> in zinc limiting conditions (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lyons <em>et al.</em> 2000</a>), and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003225">ADH4</a></em> is an alcohol dehydrogenase that may help conserve zinc or work more efficiently under zinc limiting conditions (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bird <em>et al.</em> 2006</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">De Nicola <em>et al.</em> 2007</a>), or during fermentation of sugars to ethanol (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Zhao and Bai 2012</a>). Interestingly, the closest homologs of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> outside of those present within the <em>sensu stricto Saccharomyces</em> species are from two distantly related species commonly found in wine fermentations, <em>Lachancea thermotolerans</em> and <em>Zygosaccharomyces rouxii</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Combina <em>et al.</em> 2005</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Romancino <em>et al.</em> 2008</a>), rather than other more closely related species, suggesting that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> may have been introgressed into the ancestral lineage of the <em>Saccharomyces</em> species. The subtelomeric physical location of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> in <em>S. cerevisiae</em> is consistent with other genes acquired by horizontal gene transfer (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hall <em>et al.</em> 2005</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Muller and Mccusker 2011</a>) and genes likely to be involved with adaptations to specific environments (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brown <em>et al.</em> 2010</a>).</p></div><div id="sec-12" class="subsection"><h3>Phenotypic effects of <em>ZRT1</em> alleles</h3><p id="p-44">We found that the three distinct <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles conferred no detectable phenotypic differences from one another. Although the alleles were not integrated at the endogenous <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> locus, each was able to fully rescue the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> deletion phenotype (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This result indicates that under the conditions tested, none of the nonsynonymous differences among the three alleles caused a substantial loss of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> function. The lack of phenotypic differences among the different <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> alleles implies that either the alleles are functionally equivalent to one another and so are not involved in balancing selection or that the lack of a discernible phenotype is a consequence of the conditions tested or an effect too small to be detected. For example, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em>, as a metal transporter, could also influence fitness due to transport of other metals, such as cadmium (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gitan <em>et al.</em> 1998</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gomes <em>et al.</em> 2002</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gitan <em>et al.</em> 2003</a>).</p></div><div id="sec-13" class="subsection"><h3>Prevalence of balancing selection</h3><p id="p-45">Of the five genes that exhibited an excess of nonsynonymous polymorphism according to MK test results, only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003224">ZRT1</a></em> showed evidence of balancing selection. The excess of nonsynonymous polymorphism in the other four genes is most likely a consequence of loss of functional constraint or slightly deleterious polymorphism. Interestingly, alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005441">IRA2</a></em>, a GTPase that negatively regulates RAS signaling, are responsible for numerous environment-specific differences in gene expression across the genome (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Smith and Kruglyak 2008</a>), and alleles of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005441">IRA2</a></em> also have been shown to affect high temperature growth (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Parts <em>et al.</em> 2011</a>). However, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005441">IRA2</a></em> does not show an excess of synonymous polymorphism as measured by the HKA test.</p><p id="p-46">The prevalence of balancing selection across the entire yeast genome is more difficult to assess. The observation that rates of nonsynonymous and synonymous polymorphism are correlated with one another provides some evidence for the possibility of weak balancing selection throughout the yeast genome (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cutter and Moses 2011</a>). However, genes with high rates of synonymous polymorphism do not show a tendency toward an excess of nonsynonymous polymorphism (Kendall’s tau = −0.15) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) as predicted by the MK test using the data of <a id="xref-ref-36-8" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> (2009)</a>. The challenge to interpreting genome-wide evidence for balancing selection is that many cases of balancing selection may be difficult to detect. First, the effect of balancing selection on linked variation decreases as a function of the rate of recombination; nucleotide diversity is 1 + 1/4 <em>Nr</em>(1-<em>F</em>) relative to a neutral locus, where <em>N</em> is the effective population size and <em>r</em> is the rate of recombination and <em>F</em> is the inbreeding coefficient (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Charlesworth <em>et al.</em> 1997</a>). Using a rate of recombination of 3.5 × 10<sup>−6</sup>/bp, a rate of outcrossing of 2 × 10<sup>−5</sup>/generation, and an effective population size of 1.6 × 10<sup>7</sup> cells (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Ruderfer <em>et al.</em> 2006</a>), we expect diversity to be increased by a factor of 10 and 2, 13 bp and 113 bp from a site underbalancing selection, respectively. Gene conversion is expected to narrow this window even further (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andolfatto and Nordborg 1998</a>). Second, balancing selection must act over many generations, on the order of the effective population size (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Navarro <em>et al.</em> 2000</a>), to noticeably influence linked neutral variation. However, the ability to detect balancing selection may be increased if there are multiple selected sites at a single locus, which might be the case for genes identified by a high rate of nonsynonymous polymorphism. Thus, it is hard to rule out the possibility that balancing selection has inflated the rate of nonsynonymous polymorphism across many genes without generating a strong effect on linked synonymous sites.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/665/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="No abundance of genes exhibiting high rates of synonymous polymorphism and an excess of nonsynonymous polymorphism. The rate of synonymous polymorphism, measured by the number of synonymous single nucleotide polymorphisms (SNPs) per codon compared to the observed (obs.) minus the expected (exp.) number of nonsynonymous SNPs (data are from Liti et al. 2009). The expected number of nonsynonymous SNPs was derived from Pn &#x2212; Ps &#xD7; (Dn/Ds), where Pn and Ps are the number of nonsynonymous and synonymous SNPs, respectively, and Dn and Ds are the number of nonsynonymous and synonymous fixed differences, respectively. Genes with nonsynonymous polymorphism below &#x2212;10 or above 10 are shown as points at the values of &#x2212;10 or 10, respectively. The red point is ZRT1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1599637268" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;No abundance of genes exhibiting high rates of synonymous polymorphism and an excess of nonsynonymous polymorphism. The rate of synonymous polymorphism, measured by the number of synonymous single nucleotide polymorphisms (SNPs) per codon compared to the observed (obs.) minus the expected (exp.) number of nonsynonymous SNPs (data are from Liti &lt;em&gt;et al.&lt;/em&gt; 2009). The expected number of nonsynonymous SNPs was derived from &lt;em&gt;P&lt;sub&gt;n&lt;/sub&gt;&lt;/em&gt; &#x2212; &lt;em&gt;P&lt;sub&gt;s&lt;/sub&gt;&lt;/em&gt; &#xD7; (&lt;em&gt;D&lt;sub&gt;n&lt;/sub&gt;&lt;/em&gt;/&lt;em&gt;D&lt;sub&gt;s&lt;/sub&gt;&lt;/em&gt;), where &lt;em&gt;P&lt;sub&gt;n&lt;/sub&gt;&lt;/em&gt; and &lt;em&gt;P&lt;sub&gt;s&lt;/sub&gt;&lt;/em&gt; are the number of nonsynonymous and synonymous SNPs, respectively, and &lt;em&gt;D&lt;sub&gt;n&lt;/sub&gt;&lt;/em&gt; and &lt;em&gt;D&lt;sub&gt;s&lt;/sub&gt;&lt;/em&gt; are the number of nonsynonymous and synonymous fixed differences, respectively. Genes with nonsynonymous polymorphism below &#x2212;10 or above 10 are shown as points at the values of &#x2212;10 or 10, respectively. The red point is &lt;em&gt;ZRT1&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/665/F6.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/665/F6.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/665/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/665/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-47" class="first-child">No abundance of genes exhibiting high rates of synonymous polymorphism and an excess of nonsynonymous polymorphism. The rate of synonymous polymorphism, measured by the number of synonymous single nucleotide polymorphisms (SNPs) per codon compared to the observed (obs.) minus the expected (exp.) number of nonsynonymous SNPs (data are from <a id="xref-ref-36-9" class="xref-bibr" href="#ref-36">Liti <em>et al.</em> 2009</a>). The expected number of nonsynonymous SNPs was derived from <em>P<sub>n</sub></em> − <em>P<sub>s</sub></em> × (<em>D<sub>n</sub></em>/<em>D<sub>s</sub></em>), where <em>P<sub>n</sub></em> and <em>P<sub>s</sub></em> are the number of nonsynonymous and synonymous SNPs, respectively, and <em>D<sub>n</sub></em> and <em>D<sub>s</sub></em> are the number of nonsynonymous and synonymous fixed differences, respectively. Genes with nonsynonymous polymorphism below −10 or above 10 are shown as points at the values of −10 or 10, respectively. The red point is <em>ZRT1</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Why is there little evidence of adaptive evolution within the yeast genome?</h3><p id="p-48">An important and persistent question in genome-wide estimates of adaptive evolution based on the MK test is why some species show high rates of adaptive evolution whereas others, such as yeast, do not. A small effective population size is one explanation as adaptive substitutions are expected to be more infrequent and deleterious polymorphism more common. This provides a reasonable explanation for the absence of signal in humans and many plant species (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Eyre-Walker and Keightley 2009</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Gossmann <em>et al.</em> 2010</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Halligan <em>et al.</em> 2010</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Siol <em>et al.</em> 2010</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Slotte <em>et al.</em> 2010</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Gossmann <em>et al.</em> 2012</a>). However, it does not explain the lack of signal in yeast, which has a large effective population size, on the order of 10<sup>7</sup> for <em>S. paradoxus</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Tsai <em>et al.</em> 2008</a>) and <em>S. cerevisiae</em> (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Ruderfer <em>et al.</em> 2006</a>). The rate of outcrossing may also be relevant to detecting selection in yeast. Selfing helps purge recessive deleterious alleles but also limits recombination between different haplotypes. Despite the presence of selfing in yeast, <em>S. cerevisiae</em> exhibits an excess of rare nonsynonymous polymorphism indicative of deleterious alleles and a rapid decay in levels of linkage disequilibrium, an observation that can be attributed to its exceptionally high rate of recombination even if outcrossing is rare. Thus, there is no obvious aspect of <em>S. cerevisiae</em> diversity that distinguishes it from outcrossing species. Furthermore, both a selfing and an outcrossing species of <em>Arabidopsis</em> show no signal of adaptive evolution (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Foxe <em>et al.</em> 2008</a>). As it stands, neither population size nor selfing provide a particularly compelling explanation for why yeast show little adaptive evolution based on the MK test.</p><p id="p-49">In the present study, we considered the possibility that balancing selection obscured patterns of positive selection in yeast. While we focused only on a small number of genes exhibiting a large excess of nonsynonymous polymorphism, we found one that exhibited evidence of balancing selection. Thus, while our results highlight the need to consider balancing selection in estimating the frequency of positive selection in yeast, it remains difficult to assess the impact of this consideration. We conclude that balancing selection is a potentially important factor in estimating the frequency of positive selection in yeast. While not emphasized here, it is also important to consider whether adaptive evolution is rare and estimates of positive selection are inflated in other species (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Fay 2011</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgment</h2><p id="p-50">This work was supported by National Institutes of Health grant GM080669.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005082/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-51"><em>Communicating editor: D. J. Gresham</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-29"><span class="received-label">Received </span>November 29, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-02-16"><span class="accepted-label">Accepted </span>February 16, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Engle, Fay</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The effect of gene conversion on intralocus associations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D148%26rft.issue%253D3%26rft.spage%253D1397%26rft.epage%253D1399%26rft.atitle%253DThe%2BEffect%2Bof%2BGene%2BConversion%2Bon%2BIntralocus%2BAssociations%26rft_id%253Dinfo%253Apmid%252F9539452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OC8zLzEzOTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.2" data-doi="10.1186/1471-2148-8-334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachtrog</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Similar rates of protein adaptation in <em>Drosophila miranda</em> and <em>D. melanogaster</em>, two species with different current effective population sizes</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DBachtrog%26rft.auinit1%253DD.%26rft.volume%253D8%26rft.spage%253D334%26rft.epage%253D334%26rft.atitle%253DSimilar%2Brates%2Bof%2Bprotein%2Badaptation%2Bin%2BDrosophila%2Bmiranda%2Band%2BD.%2Bmelanogaster%252C%2Btwo%2Bspecies%2Bwith%2Bdifferent%2Bcurrent%2Beffective%2Bpopulation%2Bsizes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-334%26rft_id%253Dinfo%253Apmid%252F19091130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19091130&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.3" data-doi="10.1093/molbev/msh134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The genomic rate of adaptive amino acid substitution in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1350</span>–<span class="cit-lpage">1360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh134%26rft_id%253Dinfo%253Apmid%252F15044594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTM1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.4" data-doi="10.1038/sj.emboj.7601453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eide</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winge</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Repression of <em>ADH1</em> and <em>ADH3</em> during zinc deficiency by Zap1-induced intergenic RNA transcripts</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">5726</span>–<span class="cit-lpage">5734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DBird%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D25%26rft.issue%253D24%26rft.spage%253D5726%26rft.epage%253D5734%26rft.atitle%253DRepression%2Bof%2BADH1%2Band%2BADH3%2Bduring%2Bzinc%2Bdeficiency%2Bby%2BZap1-induced%2Bintergenic%2BRNA%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601453%26rft_id%253Dinfo%253Apmid%252F17139254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7601453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17139254&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.5" data-doi="10.1016/j.cub.2010.04.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid expansion and functional divergence of subtelomeric gene families in yeasts</span>. <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBrown%26rft.auinit1%253DC.%2BA.%26rft.volume%253D20%26rft.issue%253D10%26rft.spage%253D895%26rft.epage%253D903%26rft.atitle%253DRapid%2Bexpansion%2Band%2Bfunctional%2Bdivergence%2Bof%2Bsubtelomeric%2Bgene%2Bfamilies%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.04.027%26rft_id%253Dinfo%253Apmid%252F20471265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.04.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20471265&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278113300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.6" data-doi="10.1017/S0016672397002954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D70%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D174%26rft.atitle%253DThe%2Beffects%2Bof%2Blocal%2Bselection%252C%2Bbalanced%2Bpolymorphism%2Band%2Bbackground%2Bselection%2Bon%2Bequilibrium%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bsubdivided%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397002954%26rft_id%253Dinfo%253Apmid%252F9449192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397002954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9449192&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071240500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.7" data-doi="10.1093/molbev/msn005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The McDonald-Kreitman test and slightly deleterious mutations</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1007</span>–<span class="cit-lpage">1015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn005%26rft_id%253Dinfo%253Apmid%252F18195052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI1LzYvMTAwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.8" data-doi="10.1016/j.ijfoodmicro.2004.08.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combina</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elía</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercado</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catania</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganga</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Dynamics of indigenous yeast populations during spontaneous fermentation of wines from Mendoza, Argentina</span>. <abbr class="cit-jnl-abbrev">Int. J. Food Microbiol.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bfood%2Bmicrobiology%26rft.stitle%253DInt%2BJ%2BFood%2BMicrobiol%26rft.aulast%253DCombina%26rft.auinit1%253DM.%26rft.volume%253D99%26rft.issue%253D3%26rft.spage%253D237%26rft.epage%253D243%26rft.atitle%253DDynamics%2Bof%2Bindigenous%2Byeast%2Bpopulations%2Bduring%2Bspontaneous%2Bfermentation%2Bof%2Bwines%2Bfrom%2BMendoza%252C%2BArgentina.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ijfoodmicro.2004.08.017%26rft_id%253Dinfo%253Apmid%252F15808358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ijfoodmicro.2004.08.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15808358&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.9" data-doi="10.1093/molbev/msq356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutter</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Polymorphism, divergence, and the role of recombination in <em>Saccharomyces cerevisiae</em> genome evolution</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq356%26rft_id%253Dinfo%253Apmid%252F21199893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzUvMTc0NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Nicola</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Zinc interactions with brewing yeast: impact on fermentation performance</span>. <abbr class="cit-jnl-abbrev">J. Am. Soc. Brew. Chem.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">214</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BBrew.%2BChem.%26rft.volume%253D69%26rft.spage%253D214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.11" data-doi="10.1128/AEM.01445-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Nicola</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazelwood</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Hulster</span>  <span class="cit-name-given-names">E. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knijnenburg</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Physiological and transcriptional responses of <em>Saccharomyces cerevisiae</em> to zinc limitation in chemostat cultures</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">7680</span>–<span class="cit-lpage">7692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01445-07%26rft_id%253Dinfo%253Apmid%252F17933919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3My8yMy83NjgwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.12" data-doi="10.1371/journal.pgen.1000183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcuera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A catalog of neutral and deleterious polymorphism in yeast</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000183%26rft.epage%253De1000183%26rft.atitle%253DA%2Bcatalog%2Bof%2Bneutral%2Band%2Bdeleterious%2Bpolymorphism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000183%26rft_id%253Dinfo%253Apmid%252F18769710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18769710&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.13" data-doi="10.1101/gr.108993.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bullaughey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sattath</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinott</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Shifts in the intensity of purifying selection: an analysis of genome-wide polymorphism data from two closely related yeast species</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.108993.110%26rft_id%253Dinfo%253Apmid%252F20817943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMS8xNTU4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.14" data-doi="10.1371/journal.pgen.1002763"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engle</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Divergence of the yeast transcription factor <em>FZF1</em> affects sulfite resistance</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DEngle%26rft.auinit1%253DE.%2BK.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002763%26rft.epage%253De1002763%26rft.atitle%253DDivergence%2Bof%2Bthe%2Byeast%2Btranscription%2Bfactor%2BFZF1%2Baffects%2Bsulfite%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002763%26rft_id%253Dinfo%253Apmid%252F22719269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22719269&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Changing effective population size and the McDonald-Kreitman test</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">2017</span>–<span class="cit-lpage">2024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D2017%26rft.epage%253D2024%26rft.atitle%253DChanging%2BEffective%2BPopulation%2BSize%2Band%2Bthe%2BMcDonald-Kreitman%2BTest%26rft_id%253Dinfo%253Apmid%252F12524367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzIwMTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.16" data-doi="10.1093/molbev/msp119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimating the rate of adaptive molecular evolution in the presence of slightly deleterious mutations and population size change</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2097</span>–<span class="cit-lpage">2108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp119%26rft_id%253Dinfo%253Apmid%252F19535738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI2LzkvMjA5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.17" data-doi="10.1016/j.tig.2011.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Weighing the evidence for adaptation at the molecular level</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D27%26rft.issue%253D9%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DWeighing%2Bthe%2Bevidence%2Bfor%2Badaptation%2Bat%2Bthe%2Bmolecular%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.06.003%26rft_id%253Dinfo%253Apmid%252F21775012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21775012&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294743200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.18" data-doi="10.1016/S0959-437X(00)00247-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The neutral theory in the genomic era</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D642%26rft.epage%253D646%26rft.atitle%253DThe%2Bneutral%2Btheory%2Bin%2Bthe%2Bgenomic%2Bera.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252800%252900247-1%26rft_id%253Dinfo%253Apmid%252F11682307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(00)00247-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11682307&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171989300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.19" data-doi="10.1146/annurev.genom.4.020303.162528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sequence divergence, functional constraint, and selection in protein evolution</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D4%26rft.spage%253D213%26rft.epage%253D235%26rft.atitle%253DSequence%2Bdivergence%252C%2Bfunctional%2Bconstraint%252C%2Band%2Bselection%2Bin%2Bprotein%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.4.020303.162528%26rft_id%253Dinfo%253Apmid%252F14527302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.4.020303.162528&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14527302&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222580700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.20" data-doi="10.1371/journal.pgen.0010066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benavides</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for domesticated and wild populations of <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D71%26rft.atitle%253DEvidence%2Bfor%2Bdomesticated%2Band%2Bwild%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010066%26rft_id%253Dinfo%253Apmid%252F16103919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16103919&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234295900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.21" data-doi="10.1038/4151024a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyckoff</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Testing the neutral theory of molecular evolution with genomic data from <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">1024</span>–<span class="cit-lpage">1026</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D415%26rft.issue%253D6875%26rft.spage%253D1024%26rft.epage%253D1026%26rft.atitle%253DTesting%2Bthe%2Bneutral%2Btheory%2Bof%2Bmolecular%2Bevolution%2Bwith%2Bgenomic%2Bdata%2Bfrom%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F4151024a%26rft_id%253Dinfo%253Apmid%252F11875569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/4151024a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11875569&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174075000045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.22" data-doi="10.1093/molbev/msn079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foxe</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on amino acid substitutions in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1375</span>–<span class="cit-lpage">1383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn079%26rft_id%253Dinfo%253Apmid%252F18390851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMTM3NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.23" data-doi="10.1074/jbc.273.44.28617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gitan</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodgers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broderius</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eide</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Zinc-induced inactivation of the yeast <em>ZRT1</em> zinc transporter occurs through endocytosis and vacuolar degradation</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">28617</span>–<span class="cit-lpage">28624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DGitan%26rft.auinit1%253DR.%2BS.%26rft.volume%253D273%26rft.issue%253D44%26rft.spage%253D28617%26rft.epage%253D28624%26rft.atitle%253DZinc-induced%2BInactivation%2Bof%2Bthe%2BYeast%2BZRT1%2BZinc%2BTransporter%2BOccurs%2Bthrough%2BEndocytosis%2Band%2BVacuolar%2BDegradation%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.44.28617%26rft_id%253Dinfo%253Apmid%252F9786854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzMvNDQvMjg2MTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.24" data-doi="10.1074/jbc.M302760200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gitan</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shababi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eide</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A cytosolic domain of the yeast Zrt1 zinc transporter is required for its post-translational inactivation in response to zinc and cadmium</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">39558</span>–<span class="cit-lpage">39564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M302760200%26rft_id%253Dinfo%253Apmid%252F12893829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvNDEvMzk1NTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.25" data-doi="10.1016/S0304-4165(02)00324-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomes</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riger</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panek</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eleutherio</span>  <span class="cit-name-given-names">E. C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Regulation of cadmium uptake by <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1573</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DGomes%26rft.auinit1%253DD.%2BS.%26rft.volume%253D1573%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D25%26rft.atitle%253DRegulation%2Bof%2Bcadmium%2Buptake%2Bby%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0304-4165%252802%252900324-0%26rft_id%253Dinfo%253Apmid%252F12383937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0304-4165(02)00324-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12383937&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.26" data-doi="10.1093/molbev/msq079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windsor</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome wide analyses reveal little evidence for adaptive evolution in many plant species</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1822</span>–<span class="cit-lpage">1832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq079%26rft_id%253Dinfo%253Apmid%252F20299543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTgyMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.27" data-doi="10.1093/gbe/evs027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gossmann</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The effect of variation in the effective population size on the rate of adaptive molecular evolution in eukaryotes</span>. <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">658</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs027%26rft_id%253Dinfo%253Apmid%252F22436998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjQvNS82NTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.28" data-doi="10.1111/j.1558-5646.2007.00308.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Toward a selection theory of molecular evolution</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHahn%26rft.auinit1%253DM.%2BW.%26rft.volume%253D62%26rft.issue%253D2%26rft.spage%253D255%26rft.epage%253D265%26rft.atitle%253DToward%2Ba%2Bselection%2Btheory%2Bof%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00308.x%26rft_id%253Dinfo%253Apmid%252F18302709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00308.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18302709&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253373900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.29" data-doi="10.1128/EC.4.6.1102-1115.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Contribution of horizontal gene transfer to the evolution of Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1102</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DHall%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D1102%26rft.epage%253D1115%26rft.atitle%253DContribution%2Bof%2BHorizontal%2BGene%2BTransfer%2Bto%2Bthe%2BEvolution%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.6.1102-1115.2005%26rft_id%253Dinfo%253Apmid%252F15947202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI0LzYvMTEwMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.30" data-doi="10.1371/journal.pgen.1000825"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harr</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evidence for pervasive adaptive protein evolution in wild mice</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHalligan%26rft.auinit1%253DD.%2BL.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1000825%26rft.epage%253De1000825%26rft.atitle%253DEvidence%2Bfor%2Bpervasive%2Badaptive%2Bprotein%2Bevolution%2Bin%2Bwild%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000825%26rft_id%253Dinfo%253Apmid%252F20107605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000825&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20107605&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Statistical properties of the number of recombination events in the history of a sample of DNA sequences</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D111%26rft.issue%253D1%26rft.spage%253D147%26rft.epage%253D164%26rft.atitle%253DSTATISTICAL%2BPROPERTIES%2BOF%2BTHE%2BNUMBER%2BOF%2BRECOMBINATION%2BEVENTS%2BIN%2BTHE%2BHISTORY%2BOF%2BA%2BSAMPLE%2BOF%2BDNA%2BSEQUENCES%26rft_id%253Dinfo%253Apmid%252F4029609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTExLzEvMTQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguade</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A test of neutral molecular evolution based on nucleotide data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D116%26rft.issue%253D1%26rft.spage%253D153%26rft.epage%253D159%26rft.atitle%253DA%2BTest%2Bof%2BNeutral%2BMolecular%2BEvolution%2BBased%2Bon%2BNucleotide%2BData%26rft_id%253Dinfo%253Apmid%252F3110004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE2LzEvMTUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.33" data-doi="10.1093/molbev/msn166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivailler</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Synonymous and nonsynonymous polymorphisms <em>vs.</em> divergences in bacterial genomes</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2199</span>–<span class="cit-lpage">2209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn166%26rft_id%253Dinfo%253Apmid%252F18667439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMC8yMTk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.34" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjEvMjk0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.35" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTEvMTQ1MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.36" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.37" data-doi="10.1073/pnas.97.14.7957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaither</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome-wide characterization of the Zap1p zinc-responsive regulon in yeast</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7957</span>–<span class="cit-lpage">7962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLyons%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D97%26rft.issue%253D14%26rft.spage%253D7957%26rft.epage%253D7962%26rft.atitle%253DGenome-wide%2Bcharacterization%2Bof%2Bthe%2BZap1p%2Bzinc-responsive%2Bregulon%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.14.7957%26rft_id%253Dinfo%253Apmid%252F10884426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTQvNzk1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.38" data-doi="10.1038/351652a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Adaptive protein evolution at the <em>Adh</em> locus in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcDonald%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D351%26rft.issue%253D6328%26rft.spage%253D652%26rft.epage%253D654%26rft.atitle%253DAdaptive%2Bprotein%2Bevolution%2Bat%2Bthe%2BAdh%2Blocus%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F351652a0%26rft_id%253Dinfo%253Apmid%252F1904993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/351652a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904993&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT11200066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.39" data-doi="10.1111/j.1567-1364.2011.00748.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">L. A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Nature and distribution of large sequence polymorphisms in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2011.00748.x%26rft_id%253Dinfo%253Apmid%252F22093685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjExLzcvNTg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Effect of inversion polymorphism on the neutral nucleotide variability of linked chromosomal regions in <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNavarro%26rft.auinit1%253DA.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D685%26rft.epage%253D698%26rft.atitle%253DEffect%2Bof%2BInversion%2BPolymorphism%2Bon%2Bthe%2BNeutral%2BNucleotide%2BVariability%2Bof%2BLinked%2BChromosomal%2BRegions%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F10835391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvNjg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.41" data-doi="10.1146/annurev-genom-082908-150129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nozawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The neutral theory of molecular evolution in the genomic era</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D11%26rft.spage%253D265%26rft.epage%253D289%26rft.atitle%253DThe%2Bneutral%2Btheory%2Bof%2Bmolecular%2Bevolution%2Bin%2Bthe%2Bgenomic%2Bera.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082908-150129%26rft_id%253Dinfo%253Apmid%252F20565254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082908-150129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20565254&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282662200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.42" data-doi="10.1101/gr.116731.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revealing the genetic structure of a trait by sequencing a population under selection</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1131</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.116731.110%26rft_id%253Dinfo%253Apmid%252F21422276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTEzMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.43" data-doi="10.1111/j.1365-2672.2008.03894.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romancino</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Maio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muriella</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliva</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Analysis of non-<em>Saccharomyces</em> yeast populations isolated from grape musts from Sicily (Italy)</span>. <abbr class="cit-jnl-abbrev">J. Appl. Microbiol.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">2248</span>–<span class="cit-lpage">2254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DRomancino%26rft.auinit1%253DD.%2BP.%26rft.volume%253D105%26rft.issue%253D6%26rft.spage%253D2248%26rft.epage%253D2254%26rft.atitle%253DAnalysis%2Bof%2Bnon-Saccharomyces%2Byeast%2Bpopulations%2Bisolated%2Bfrom%2Bgrape%2Bmusts%2Bfrom%2BSicily%2B%2528Italy%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2008.03894.x%26rft_id%253Dinfo%253Apmid%252F19120668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2008.03894.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19120668&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.44" data-doi="10.1038/ng1859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population genomic analysis of outcrossing and recombination in yeast</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRuderfer%26rft.auinit1%253DD.%2BM.%26rft.volume%253D38%26rft.issue%253D9%26rft.spage%253D1077%26rft.epage%253D1081%26rft.atitle%253DPopulation%2Bgenomic%2Banalysis%2Bof%2Boutcrossing%2Band%2Brecombination%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1859%26rft_id%253Dinfo%253Apmid%252F16892060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16892060&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240112100028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.45" data-doi="10.1073/pnas.0409853102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emerman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malik</span>  <span class="cit-name-given-names">H. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Positive selection of primate <em>TRIM5alpha</em> identifies a critical species-specific retroviral restriction domain</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">2832</span>–<span class="cit-lpage">2837</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409853102%26rft_id%253Dinfo%253Apmid%252F15689398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzgvMjgzMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.46" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.47" data-doi="10.1371/journal.pgen.1000495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sella</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pervasive natural selection in the <em>Drosophila</em> genome?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSella%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000495%26rft.epage%253De1000495%26rft.atitle%253DPervasive%2Bnatural%2Bselection%2Bin%2Bthe%2BDrosophila%2Bgenome%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000495%26rft_id%253Dinfo%253Apmid%252F19503600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19503600&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sikorski</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSikorski%26rft.auinit1%253DR.%2BS.%26rft.volume%253D122%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D27%26rft.atitle%253DA%2Bsystem%2Bof%2Bshuttle%2Bvectors%2Band%2Byeast%2Bhost%2Bstrains%2Bdesigned%2Bfor%2Befficient%2Bmanipulation%2Bof%2BDNA%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2659436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTIyLzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.49" data-doi="10.1111/j.1469-8137.2010.03401.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">S. C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The population genomics of plant adaptation</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSiol%26rft.auinit1%253DM.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D332%26rft.atitle%253DThe%2Bpopulation%2Bgenomics%2Bof%2Bplant%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03401.x%26rft_id%253Dinfo%253Apmid%252F20696011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03401.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20696011&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.50" data-doi="10.1093/molbev/msq062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foxe</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide evidence for efficient positive and purifying selection in <em>Capsella grandiflora</em>, a plant species with a large effective population size</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1813</span>–<span class="cit-lpage">1821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq062%26rft_id%253Dinfo%253Apmid%252F20194429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMTgxMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.51" data-doi="10.1371/journal.pbio.0060083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene-environment interaction in yeast gene expression</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSmith%26rft.auinit1%253DE.%2BN.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De83%26rft.epage%253De83%26rft.atitle%253DGene-environment%2Binteraction%2Bin%2Byeast%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060083%26rft_id%253Dinfo%253Apmid%252F18416601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18416601&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.52" data-doi="10.1093/molbev/msq249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimation of the neutrality index</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq249%26rft_id%253Dinfo%253Apmid%252F20837603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjI4LzEvNjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.53" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>–<span class="cit-lpage">2739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMC8yNzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzY2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.54" data-doi="10.1073/pnas.0707314105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koufopanou</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Population genomics of the wild yeast <em>Saccharomyces paradoxus</em>: Quantifying the life cycle</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">4957</span>–<span class="cit-lpage">4962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0707314105%26rft_id%253Dinfo%253Apmid%252F18344325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzEyLzQ5NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.55" data-doi="10.1002/yea.320101310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pöhlmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippsen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">New heterologous modules for classical or PCR-based gene disruptions in <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1793</span>–<span class="cit-lpage">1808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWach%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D13%26rft.spage%253D1793%26rft.epage%253D1808%26rft.atitle%253DNew%2Bheterologous%2Bmodules%2Bfor%2Bclassical%2Bor%2BPCR-based%2Bgene%2Bdisruptions%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320101310%26rft_id%253Dinfo%253Apmid%252F7747518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320101310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7747518&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QJ42800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.56" data-doi="10.1146/annurev.ecolsys.39.110707.173342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The impact of natural selection on the genome: Emerging patterns in <em>Drosophila</em> and <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.volume%253D39%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.57" data-doi="10.1534/genetics.104.026500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The HKA test revisited: a maximum-likelihood-ratio test of the standard neutral model</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1071</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%2BI.%26rft.volume%253D168%26rft.issue%253D2%26rft.spage%253D1071%26rft.epage%253D1076%26rft.atitle%253DThe%2BHKA%2BTest%2BRevisited%253A%2BA%2BMaximum-Likelihood-Ratio%2BTest%2Bof%2Bthe%2BStandard%2BNeutral%2BModel%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.026500%26rft_id%253Dinfo%253Apmid%252F15514076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC8yLzEwNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.58" data-doi="10.1073/pnas.93.6.2454"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eide</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The yeast <em>ZRT1</em> gene encodes the zinc transporter protein of a high-affinity uptake system induced by zinc limitation</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">2454</span>–<span class="cit-lpage">2458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZhao%26rft.auinit1%253DH.%26rft.volume%253D93%26rft.issue%253D6%26rft.spage%253D2454%26rft.epage%253D2458%26rft.atitle%253DThe%2Byeast%2BZRT1%2Bgene%2Bencodes%2Bthe%2Bzinc%2Btransporter%2Bprotein%2Bof%2Ba%2Bhigh-affinity%2Buptake%2Bsystem%2Binduced%2Bby%2Bzinc%2Blimitation.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.6.2454%26rft_id%253Dinfo%253Apmid%252F8637895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5My82LzI0NTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.665.59" data-doi="10.1016/j.jbiotec.2011.06.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Zinc and yeast stress tolerance: micronutrient plays a big role</span>. <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiotechnology%26rft.stitle%253DJournal%2Bof%2BBiotechnology%26rft.aulast%253DZhao%26rft.auinit1%253DX.%2BQ.%26rft.volume%253D158%26rft.issue%253D4%26rft.spage%253D176%26rft.epage%253D183%26rft.atitle%253DZinc%2Band%2Byeast%2Bstress%2Btolerance%253A%2Bmicronutrient%2Bplays%2Ba%2Bbig%2Brole.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiotec.2011.06.038%26rft_id%253Dinfo%253Apmid%252F21763361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiotec.2011.06.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21763361&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F665.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/4/665.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/657" title="Donor DNA Utilization During Gene Targeting with Zinc-Finger Nucleases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/675" title="Comparative Genomics of Serial Isolates of Cryptococcus neoformans Reveals Gene Associated With Carbon Utilization and Virulence" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/4"><span>Volume 3 Issue 4, April 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/4.cover-source.jpg?itok=tXw4AXt4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/665" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-h-z5DqG7oOxwu6fQayOV6JcDFyJXuxTzEUD8m0Yyyog" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/4/665" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/4/665" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/4/665" class="active" data-icon-position="" data-hide-link-title="0">ZRT1 Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459453" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-6QvP44CtYv386zk99OTYk_Vpan0OLhuo1YsxLDq3l3c" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/665.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/4/665" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-uq47IG0FsN2_KVsk_SXVA1rplDAkWHFMsp47BmoBEx0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/4/665" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/665.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459453" id="node459453--2" data-pisa="ggg;3/4/665" data-pisa-master="ggg;g3.112.005082" data-apath="/ggg/3/4/665.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>ZRT1</em> Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elizabeth K. Engle</span> and <span class="highwire-citation-author" data-delta="1">Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">665-673;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459453--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth K. Engle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Genetics and Genomics Program, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BK.%2BEngle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Engle%20EK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BK.%2BEngle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/4/665" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/4/665&amp;title=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/4/665&amp;title=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/4/665&amp;title=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/4/665&amp;text=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/4/665&amp;title=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/4/665&amp;t=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/4/665&amp;title=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/4/665&amp;title=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459453/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459453" id="node459453--3" data-pisa="ggg;3/4/665" data-pisa-master="ggg;g3.112.005082" data-apath="/ggg/3/4/665.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title"><em>ZRT1</em> Harbors an Excess of Nonsynonymous Polymorphism and Shows Evidence of Balancing Selection in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Elizabeth K. Engle</span> and <span class="highwire-citation-author" data-delta="1">Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">665-673;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459453--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth K. Engle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Genetics and Genomics Program, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BK.%2BEngle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Engle%20EK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElizabeth%2BK.%2BEngle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics and Center for Genome Sciences and Systems Biology, Washington University, St. Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jfay@genetics.wustl.edu" class="" data-icon-position="" data-hide-link-title="0">jfay@genetics.wustl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=665&amp;rft.epage=673&amp;rft.atitle=ZRT1%20Harbors%20an%20Excess%20of%20Nonsynonymous%20Polymorphism%20and%20Shows%20Evidence%20of%20Balancing%20Selection%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=3&amp;rft.issue=4&amp;rft.date=2013-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Engle%2C+Elizabeth+K.&amp;rft.au=Fay%2C+Justin+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459453/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459453/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459453/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459453/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459453/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459453/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459453/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459453/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459453/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459453/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459453/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459453/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/4/665" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgment</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/4/665.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/4/665.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/4/665.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/4/665.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
