<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/4/763" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/4/763.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/4/763.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/4/763.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/4/763/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/4/763.atom" />
<meta name="HW.pisa" content="ggg;3/4/763" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome" />
<meta name="DC.Identifier" content="10.1534/g3.113.005637" />
<meta name="DC.Date" content="2013-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Shanku et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In this paper we present a de novo assembly of the transcriptome of the damselfly ( Enallagma hageni ) through the use of 454 pyrosequencing. E. hageni is a member of the suborder Zygoptera, in the order Odonata, and Odonata organisms form the basal lineage of the winged insects (Pterygota). To date, sequence data used in phylogenetic analysis of Enallagma species have been derived from either mitochondrial DNA or ribosomal nuclear DNA. This Enallagma transcriptome contained 31,661 contigs that were assembled and translated into 14,813 individual open reading frames. Using these data, we constructed an extensive dataset of 634 orthologous nuclear protein-encoding genes across 11 species of Arthropoda and used Bayesian techniques to elucidate the position of Enallagma in the arthropod phylogenetic tree. Additionally, we demonstrated that the Enallagma transcriptome contains 169 genes that are evolving at rates that differ relative to those of the rest of the transcriptome (29 accelerated and 140 decreased), and, through multiple Gene Ontology searches and clustering methods, we present the first functional annotation of any palaeopteran’s transcriptome in the literature." />
<meta name="DC.Contributor" content="Alexander G. Shanku" />
<meta name="DC.Contributor" content="Mark A. McPeek" />
<meta name="DC.Contributor" content="Andrew D. Kern" />
<meta name="article:published_time" content="2013-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In this paper we present a &lt;i&gt;de novo&lt;/i&gt; assembly of the transcriptome of the damselfly (&lt;i&gt;Enallagma hageni&lt;/i&gt;) through the use of 454 pyrosequencing. &lt;i&gt;E. hageni&lt;/i&gt; is a member of the suborder Zygoptera, in the order Odonata, and Odonata organisms form the basal lineage of the winged insects (Pterygota). To date, sequence data used in phylogenetic analysis of &lt;i&gt;Enallagma&lt;/i&gt; species have been derived from either mitochondrial DNA or ribosomal nuclear DNA. This &lt;i&gt;Enallagma&lt;/i&gt; transcriptome contained 31,661 contigs that were assembled and translated into 14,813 individual open reading frames. Using these data, we constructed an extensive dataset of 634 orthologous nuclear protein-encoding genes across 11 species of Arthropoda and used Bayesian techniques to elucidate the position of &lt;i&gt;Enallagma&lt;/i&gt; in the arthropod phylogenetic tree. Additionally, we demonstrated that the &lt;i&gt;Enallagma&lt;/i&gt; transcriptome contains 169 genes that are evolving at rates that differ relative to those of the rest of the transcriptome (29 accelerated and 140 decreased), and, through multiple Gene Ontology searches and clustering methods, we present the first functional annotation of any palaeopteran’s transcriptome in the literature.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/04/01" />
<meta name="citation_mjid" content="ggg;3/4/763" />
<meta name="citation_id" content="3/4/763" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/4/763" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/4/763.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/4/763.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/4/763.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005637" />
<meta name="citation_pmid" content="23550132" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="763" />
<meta name="citation_lastpage" content="770" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Alexander G. Shanku" />
<meta name="citation_author_institution" content="Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082" />
<meta name="citation_author_email" content="alexander.shanku@rutgers.edu" />
<meta name="citation_author" content="Mark A. McPeek" />
<meta name="citation_author_institution" content="Department of Biological Science, Dartmouth College, New Hampshire 03755" />
<meta name="citation_author" content="Andrew D. Kern" />
<meta name="citation_author_institution" content="Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. Abascal;citation_author=R. Zardoya;citation_author=D. Posada;citation_title=ProtTest: selection of best-fit models of protein evolution;citation_volume=21;citation_year=2005;citation_pmid=15647292;citation_doi=10.1093/bioinformatics/bti263" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. Abascal;citation_author=D. Posada;citation_author=R. Zardoya;citation_title=MtArt: A new model of amino acid replacement for arthropoda;citation_volume=24;citation_year=2007;citation_pmid=17043087;citation_doi=10.1093/molbev/msl136" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. K. Abbott;citation_author=S. Bensch;citation_author=T. P. Gosden;citation_author=E. I. Svesson;citation_title=Patterns of differentiation in a colour polymorphism and in neutral markers reveal rapid genetic changes in natural damselfly populations.;citation_pages=1597-1604;citation_volume=17;citation_year=2008;citation_issue=6;citation_pmid=18284565;citation_doi=10.1111/j.1365-294X.2007.03641.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. Adachi;citation_author=M. Hasegawa;citation_title=Model of amino acid substitution in proteins encoded by mitochondrial DNA.;citation_pages=459-468;citation_volume=42;citation_year=1996;citation_issue=4;citation_pmid=8642615;citation_doi=10.1007/BF02498640" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. Adachi;citation_author=P. Waddell;citation_author=W. Martin;citation_author=M. Hasegawa;citation_title=Plastid genome phylogeny and a model of amino acid substitution for proteins encoded by chloroplast DNA.;citation_pages=348-358;citation_volume=50;citation_year=2000;citation_issue=4;citation_pmid=10795826" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Altekar;citation_author=S. Dwarkadas;citation_author=J. Huelsenbeck;citation_author=F. Ronquist;citation_title=Parallel metropolis coupled Markov chain Monte Carlo for Bayesian phylogenetic inference;citation_volume=20;citation_year=2004;citation_pmid=14960467;citation_doi=10.1093/bioinformatics/btg427" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. Bellin;citation_author=A. Ferrarini;citation_author=A. Chimento;citation_author=O. Kaiser;citation_author=N. Levenkova;citation_title=Combining next-generation pyrosequencing with microarray for large scale expression analysis in non-model species.;citation_pages=555-555;citation_volume=10;citation_year=2009;citation_pmid=19930683;citation_doi=10.1186/1471-2164-10-555" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=A. Bourret;citation_author=M. A. McPeek;citation_author=J. Turgeon;citation_title=Regional divergence and mosaic spatial distribution of two closely related damselfly species (Enallagma hageni and Enallagma ebrium);citation_pages=196-209;citation_volume=25;citation_year=2011;citation_pmid=22122075" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=Y. Cao;citation_author=J. Adachi;citation_author=A. Janke;citation_author=S. Paabo;citation_author=M. Hasegawa;citation_title=Phylogenetic relationships among eutherian orders estimated from inferred sequences of mitochondrial proteins: instability of a tree based on a single gene.;citation_pages=519-527;citation_volume=39;citation_year=1994;citation_issue=5;citation_pmid=7807540" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Conesa;citation_author=S. Götz;citation_author=J. M. García-Gómez;citation_author=J. Terol;citation_author=M. Talón;citation_title=Blast2GO: a Universal tool for annotation, visualization and analysis in functional genomics research;citation_volume=21;citation_year=2005;citation_pmid=16081474;citation_doi=10.1093/bioinformatics/bti610" />
<meta name="citation_reference" content="citation_title=The design, implementation, and evaluation of mpiBLAST. 4th International Conference on Linux Clusters: The HPC Revolution 2003 in conjunction with ClusterWorld Conference &amp; Expo, San Jose, CA.;citation_author=A. Darling;citation_author=L. Carey;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Darriba;citation_author=G. L. Taboada;citation_author=R. Doallo;citation_author=D. Posada;citation_title=ProtTest 3: fast selection of best-fit models of protein evolution;citation_volume=27;citation_year=2011;citation_pmid=21335321;citation_doi=10.1093/bioinformatics/btr088" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=M. Dimmic;citation_author=J. Rest;citation_author=D. Mindell;citation_author=R. Goldstein;citation_title=rtREV: an amino acid substitution matrix for inference of retrovirus and reverse transcriptase phylogeny.;citation_pages=65-73;citation_volume=55;citation_year=2002;citation_issue=1;citation_pmid=12165843;citation_doi=10.1007/s00239-001-2304-y" />
<meta name="citation_reference" content="citation_journal_title=Syst. Entomol.;citation_author=H. J. Dumont;citation_author=A. Vierstraete;citation_author=J. R. Vanfleteren;citation_title=A molecular phylogeny of the Odonata (Insecta);citation_pages=6-18;citation_volume=35;citation_year=2010;citation_doi=10.1111/j.1365-3113.2009.00489.x" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=T. Gabaldón;citation_title=Large-scale assignment of orthology: back to phylogenetics?;citation_pages=235-235;citation_volume=9;citation_year=2008;citation_issue=10;citation_pmid=18983710;citation_doi=10.1186/gb-2008-9-10-235" />
<meta name="citation_reference" content="citation_journal_title=J Exp. Zool. B. Mol. Dev. Evol.;citation_author=S. R. Gadagkar;citation_author=M. S. Rosenberg;citation_author=S. Kumar;citation_title=Inferring species phylogenies from multiple genes: concatenated sequence tree vs. consensus gene tree;citation_pages=64-74;citation_volume=304;citation_year=2005;citation_pmid=15593277" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=O. Harismendy;citation_author=P. C. Ng;citation_author=R. L. Strausberg;citation_author=X. Wang;citation_author=T. B. Stockwell;citation_title=Evaluation of next generation sequencing platforms for population targeted sequencing studies.;citation_pages=R32-R32;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19327155;citation_doi=10.1186/gb-2009-10-3-r32" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. G. Henikoff;citation_title=Amino acid substitution matrices from protein blocks.;citation_pages=10915-10919;citation_volume=89;citation_year=1992;citation_issue=22;citation_pmid=1438297;citation_doi=10.1073/pnas.89.22.10915" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. P. Huelsenbeck;citation_author=F. Ronquist;citation_author=R. Nielsen;citation_author=J. P. Bollback;citation_title=Bayesian Inference of Phylogeny and Its Impact on Evolutionary Biology;citation_pages=2310-2314;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743192;citation_doi=10.1126/science.1065889;citation_georef=2002024703" />
<meta name="citation_reference" content="citation_journal_title=J. Zool. (Lond.);citation_author=A. Iserbyt;citation_author=J. Bots;citation_author=H. Van Gossum;citation_author=K. Jordaens;citation_title=Did historical events shape current geographic variation in morph frequencies of a polymorphic damselfly?;citation_pages=256-265;citation_volume=282;citation_year=2010;citation_doi=10.1111/j.1469-7998.2010.00735.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phyl. Evol.;citation_author=K. Ishiwata;citation_author=G. Sasaki;citation_author=J. Ogawa;citation_author=T. Miyata;citation_author=Z.-H. Su;citation_title=Phylogenetic relationships among insect orders based on three nuclear protein-coding gene sequences;citation_pages=169-180;citation_volume=58;citation_year=2011;citation_pmid=21075208;citation_doi=10.1016/j.ympev.2010.11.001" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. T. Jones;citation_author=W. R. Taylor;citation_author=J. M. Thornton;citation_title=The rapid generation of mutation data matrices from protein sequences;citation_pages=275-282;citation_volume=8;citation_year=1992;citation_issue=3;citation_pmid=1633570;citation_doi=10.1093/bioinformatics/8.3.275" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal W and Clustal X version 2.0;citation_volume=23;citation_year=2007;citation_issue=21;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Q. Le;citation_author=O. Gascuel;citation_title=An improved general amino acid replacement matrix;citation_volume=25;citation_year=2008;citation_pmid=18367465;citation_doi=10.1093/molbev/msn067" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=L. Li;citation_author=C. J. Stoeckert;citation_author=D. S. Roos;citation_title=OrthoMCL: Identification of Ortholog Groups for Eukaryotic Genomes;citation_pages=2178-2189;citation_volume=13;citation_year=2003;citation_issue=9;citation_pmid=12952885;citation_doi=10.1101/gr.1224503" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=E. R. Mardis;citation_title=The impact of next-generation sequencing technology on genetics.;citation_pages=133-141;citation_volume=24;citation_year=2008;citation_issue=3;citation_pmid=18262675;citation_doi=10.1016/j.tig.2007.12.007" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. A. McPeek;citation_title=Biochemical evolution associated with antipredator adaptation in damselflies;citation_pages=1835-1845;citation_volume=53;citation_year=1999;citation_doi=10.2307/2640444" />
<meta name="citation_reference" content="citation_journal_title=Int J. Org. Evol.;citation_author=M. A. McPeek;citation_title=Predisposed to adapt? Clade-level differences in characters affecting swimming performance in damselflies;citation_pages=2072-2080;citation_volume=54;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Meusemann;citation_author=B. M. von Reumont;citation_author=S. Simon;citation_author=F. Roeding;citation_author=S. Strauss;citation_title=A phylogenomic approach to resolve the arthropod tree of life;citation_volume=27;citation_year=2010;citation_pmid=20534705;citation_doi=10.1093/molbev/msq130" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Meyer;citation_author=G. Aglyamova;citation_author=S. Wang;citation_author=J. Buchanan-Carter;citation_author=D. Abrego;citation_title=Sequencing and de novo analysis of a coral larval transcriptome using 454 GSFlx.;citation_pages=219-219;citation_volume=10;citation_year=2009;citation_pmid=19435504;citation_doi=10.1186/1471-2164-10-219" />
<meta name="citation_reference" content="citation_journal_abbrev=Ecol Lett;citation_author=G. G. Mittelbach;citation_author=D. W. Schemske;citation_author=H. V. Cornell;citation_author=A. P. Allen;citation_author=J. M. Brown;citation_title=Evolution and the latitudinal diversity gradient: speciation, extinction and biogeography.;citation_pages=315-331;citation_volume=10;citation_year=2007;citation_issue=4;citation_pmid=17355570;citation_doi=10.1111/j.1461-0248.2007.01020.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Moreno-Hagelsieb;citation_author=K. Latimer;citation_title=Choosing BLAST options for better detection of orthologs as reciprocal best hits;citation_volume=24;citation_year=2008;citation_pmid=18042555;citation_doi=10.1093/bioinformatics/btm585" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=T. Müller;citation_author=M. Vingron;citation_title=Modeling amino acid replacement.;citation_pages=761-776;citation_volume=7;citation_year=2000;citation_issue=6;citation_pmid=11382360;citation_doi=10.1089/10665270050514918" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=T. Nawy;citation_title=Non–model organisms;citation_pages=37;citation_volume=9;citation_year=2011;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=David C. Nickle;citation_author=L. Heath;citation_author=M. A. Jensen;citation_author=P. B. Gilbert;citation_author=J. I. Mullins;citation_title=HIV-specific probabilistic models of protein evolution.;citation_pages=e503-e503;citation_volume=2;citation_year=2007;citation_issue=6;citation_pmid=17551583;citation_doi=10.1371/journal.pone.0000503" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Novaes;citation_author=D. R. Drost;citation_author=W. G. Farmerie;citation_author=G. J. Pappas;citation_author=D. Grattapaglia;citation_title=High-throughput gene and SNP discovery in Eucalyptus grandis, an uncharacterized genome.;citation_pages=312-312;citation_volume=9;citation_year=2008;citation_pmid=18590545;citation_doi=10.1186/1471-2164-9-312" />
<meta name="citation_reference" content="citation_journal_title=Evol. Biol.;citation_author=D. Outomuro;citation_author=F. Bokma;citation_author=F. Johansson;citation_title=Hind wing shape evolves faster than front wing shape in Calopteryx damselflies;citation_pages=116-125;citation_volume=39;citation_year=2011;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=D. Posada;citation_author=K. Crandall;citation_title=Selecting the best-fit model of nucleotide substitution;citation_volume=50;citation_year=2001;citation_pmid=12116655;citation_doi=10.1080/10635150118469" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=F. Ronquist;citation_author=J. P. Huelsenbeck;citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models;citation_pages=1572-1574;citation_volume=19;citation_year=2003;citation_issue=12;citation_pmid=12912839;citation_doi=10.1093/bioinformatics/btg180" />
<meta name="citation_reference" content="citation_journal_title=Ann. Entomol. Soc. Am.;citation_author=C. Saux;citation_author=C. Simon;citation_author=G. S. Spicer;citation_title=Phylogeny of the dragonfly and damselfly order Odonata as inferred by mitochondrial 12S ribosomal RNA sequences;citation_volume=96;citation_year=2003;citation_doi=10.1603/0013-8746(2003)096[0693:POTDAD]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Simon;citation_author=S. Strauss;citation_author=A. von Haeseler;citation_author=H. Hadrys;citation_title=A phylogenomic approach to resolve the basal pterygote divergence;citation_volume=26;citation_year=2009;citation_pmid=19713325;citation_doi=10.1093/molbev/msp191" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Entomol.;citation_author=S. Slos;citation_author=L. D. Meester;citation_author=R. Stoks;citation_title=Behavioural activity levels and expression of stress proteins under predation risk in two damselfly species;citation_pages=297-303;citation_volume=34;citation_year=2009;citation_doi=10.1111/j.1365-2311.2008.01077.x" />
<meta name="citation_reference" content="citation_journal_title=Behav. Ecol. Sociobiol.;citation_author=F. Strobbe;citation_author=M. A. McPeek;citation_author=M. Block;citation_author=R. Stoks;citation_title=Fish predation selects for reduced foraging activity;citation_pages=241-247;citation_volume=65;citation_year=2010;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Surget-Groba;citation_author=J. I. Montoya-Burgos;citation_title=Optimization of de novo transcriptome assembly from next-generation sequencing data;citation_volume=20;citation_year=2010;citation_pmid=20693479;citation_doi=10.1101/gr.103846.109" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=G. Talavera;citation_author=J. Castresana;citation_title=Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments;citation_volume=56;citation_year=2007;citation_pmid=17654362;citation_doi=10.1080/10635150701472164" />
<meta name="citation_reference" content="citation_journal_title=Annual review of entomology;citation_journal_abbrev=Annu Rev Entomol;citation_author=M. D. Trautwein;citation_author=B. M. Wiegmann;citation_author=R. Beutel;citation_author=K. M. Kjer;citation_author=D. K. Yeates;citation_title=Advances in insect phylogeny at the dawn of the postgenomic era.;citation_pages=449-468;citation_volume=57;citation_year=2012;citation_pmid=22149269;citation_doi=10.1146/annurev-ento-120710-100538" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. Turgeon;citation_author=M. A. McPeek;citation_title=Phylogeographic analysis of a recent radiation of Enallagma damselflies (Odonata: Coenagrionidae).;citation_pages=1989-2001;citation_volume=11;citation_year=2002;citation_issue=10;citation_pmid=12296943;citation_doi=10.1046/j.1365-294X.2002.01580.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Am Nat;citation_author=J. Turgeon;citation_author=R. Stoks;citation_author=R. A. Thum;citation_author=J. M. Brown;citation_author=M. A. McPeek;citation_title=Simultaneous Quaternary radiations of three damselfly clades across the Holarctic.;citation_pages=E78-107;citation_volume=165;citation_year=2005;citation_issue=4;citation_pmid=15791532;citation_doi=10.1086/428682" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids. Res.;citation_author=R. Wernersson;citation_title=VIRTUAL RIBOSOME---A COMPREHENSIVE DNA TRANSLATION TOOL WITH SUPPORT FOR INTEGRATION OF SEQUENCE FEATURE ANNOTATION;citation_pages= W385-W388;citation_volume=34;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=S. Whelan;citation_author=N. Goldman;citation_title=A General Empirical Model of Protein Evolution Derived from Multiple Protein Families Using a Maximum-Likelihood Approach;citation_pages=691-699;citation_volume=18;citation_year=2001;citation_issue=5;citation_pmid=11319253;citation_doi=10.1093/oxfordjournals.molbev.a003851" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Ye;citation_author=L. Fang;citation_author=H. Zheng;citation_author=Y. Zhang;citation_author=J. Chen;citation_title=WEGO: a web tool for plotting GO annotations.;citation_pages=W293-W297;citation_volume=34;citation_year=2006;citation_issue=suppl 2;citation_pmid=16845012;citation_doi=10.1093/nar/gkl031" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. Yon Rhee;citation_author=V. Wood;citation_author=K. Dolinski;citation_author=S. Draghici;citation_title=Use and misuse of the gene ontology annotations.;citation_pages=509-515;citation_volume=9;citation_year=2008;citation_issue=7;citation_pmid=18475267;citation_doi=10.1038/nrg2363" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="twitter:description" content="In this paper we present a de novo assembly of the transcriptome of the damselfly ( Enallagma hageni ) through the use of 454 pyrosequencing. E. hageni is a member of the suborder Zygoptera, in the order Odonata, and Odonata organisms form the basal lineage of the winged insects (Pterygota). To date, sequence data used in phylogenetic analysis of Enallagma species have been derived from either mitochondrial DNA or ribosomal nuclear DNA. This Enallagma transcriptome contained 31,661 contigs that were assembled and translated into 14,813 individual open reading frames. Using these data, we constructed an extensive dataset of 634 orthologous nuclear protein-encoding genes across 11 species of Arthropoda and used Bayesian techniques to elucidate the position of Enallagma in the arthropod phylogenetic tree. Additionally, we demonstrated that the Enallagma transcriptome contains 169 genes that are evolving at rates that differ relative to those of the rest of the transcriptome (29 accelerated and 140 decreased), and, through multiple Gene Ontology searches and clustering methods, we present the first functional annotation of any palaeopteran’s transcriptome in the literature." />
<meta name="og-title" property="og:title" content="Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/4/763" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In this paper we present a de novo assembly of the transcriptome of the damselfly ( Enallagma hageni ) through the use of 454 pyrosequencing. E. hageni is a member of the suborder Zygoptera, in the order Odonata, and Odonata organisms form the basal lineage of the winged insects (Pterygota). To date, sequence data used in phylogenetic analysis of Enallagma species have been derived from either mitochondrial DNA or ribosomal nuclear DNA. This Enallagma transcriptome contained 31,661 contigs that were assembled and translated into 14,813 individual open reading frames. Using these data, we constructed an extensive dataset of 634 orthologous nuclear protein-encoding genes across 11 species of Arthropoda and used Bayesian techniques to elucidate the position of Enallagma in the arthropod phylogenetic tree. Additionally, we demonstrated that the Enallagma transcriptome contains 169 genes that are evolving at rates that differ relative to those of the rest of the transcriptome (29 accelerated and 140 decreased), and, through multiple Gene Ontology searches and clustering methods, we present the first functional annotation of any palaeopteran’s transcriptome in the literature." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
    <title>Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"cdA0YtXQe3hXnH5WjX7KjA9u1nre4syhR15Dsai1g8Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459445","apath":"\/ggg\/3\/4\/763.atom","pisa":"ggg;3\/4\/763","ac":{"\/ggg\/3\/4\/763.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/4\/763.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/4\/763"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/4\/763"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/4\/763":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t0HpjR6h9IpDSq9r4nX3GJmLBSAtfQdXplEvQdiYfnscnE\\\\\\\/qks19vKvPxZVLd6KHtwZgKbgK9HQcsxgVzxGe1k9jUuTjwHpDGgtI9Od+\\\\\\\/8jkNY5xfTxvdVbF\\\\\\\/l7Uk+RTMptoVlUplU38xGdDpXzFMW7b3n+UcpyU8+dQW5wPDkLOgBG+fTyErEfGsiWHCtA3zGn7dv+Gt5tBvC5w4mpUjEWCvtON8O9+Vk4yu+NdGM18wwo3Md2hhiBK1DhBN1uEyyEV1mulermZwRZShkuYx5Zmw4Io5BNcmoLT0A47yXeRB\\\\\\\/+1n6mxP42inmLzjfjnwuIBmZqhrecwdk02MulN1hoX2m7iHlmS+LkBocL+zG8LAMY4EdaDd0xDghX08IJvJdKurvoWl1hGYjB+nEvwwHB+eskZYvNVERkFJaaYMnM6931\\\\\\\/w9VsXgL9NP1siwtoTvVmwKcjdRBWSUbE3yQHaySIjWYcK70\\\\\\\/Y\\\\\\\/4pZVWYtoi5nDUoMokxtrJSH5BZN7Kiqcd8ZZ\\\\\\\/T3+WpgOpxKJ+98D\\\\\\\/1Pev24VK5mpxfipm35Pj46MuHoBQHXbdGlFoOsPsBPe\\\\\\\/dCiWE5wl\\\\\\\/UcWUZpzBura\\\\\\\/7jv3ZF16XwHVVEbsLxrobXND6TRhBOonAgUogpoLQhGbt2If\\\\\\\/3uV470qHOZZ+TDGQG21yAzcM0uaMVhwaevLYKTrj9LK4+fO9+KZ3ookwzc1D+lKLPSE1IFc7XVlKN2YRbRA3lt47ASmB26zy1TGCYwEIZuWmZnIiMZXAEzhjOszfm+kzl6UAT3GhnmopA1Y0LutR8IRbZ6Pu\\\\\\\/Ugxcx6ROGpcDOOSqAhIsPbIkCqeQv5fV2UxYfvSgMZ893QlGUixA8r7lrJDn9O6CbSGTLaw9eplE0y7jiTKXSDPLDYXYV1v2zHQoaf1EPA\\\\\\\/SarjyS652\\\\\\\/94KQ6\\\\\\\/JulsgKqHGOOZfbMWOhI45z31sEZZ0uxb4z\\\\\\\/smGrLU9guwZo94LSGRFuzfZgbtyck7yaqy8PsmJeVVHxnI0h2XLD5MFnoUjqSnD6V0TiINlDXroki8j7PsQiPpOhtdPRqqgcezD9Etkii3HJOxmZvpU9Rfo4Pe3VR4bJXK70JVTxWDrUBsbF68bgQaCb2o+ExkaUMQdHyzC83WYRZY0Km8p235OxEpCriXthbTBqai97vYrSKK8jXpVf\\\\\\\/Nu4Khq1eZDRHHHpVWWT6NNoJLTMsixFs7SfAGtoMoEtAyjexfoZ3OnEXl9CO4B\\\\\\\/nKVNgQlFIgLcBUNzk0j96Dz7wWOUjZb51Y+TM3mu\\\u0022,\\\u0022iv\\\u0022:\\\u0022FL7sphckEU3sba1VfhbpLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226aca47e19dd3c425ad2b9a24d57537dd5101c6a43ee986e2cde2498bbc3a268c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kcPOp\\\\\\\/JQr80JVZWKTY1yvrgceFr3XbProCamFsE2b8QAmCYhiTVjpeUq24jfNioq5DuANtE1ZVTRBOMx1zk7DNq3WcKJeoXhOg2XkNDqVF\\\\\\\/1FNCWzHawWevgfKoU39qVoScpD5YTsyDH7I0WWfmO4dg31WpwYLRqdbQRL2oehhfqv+05FUMLoz4llcJYD1vxcXRAH+e+yv7CxcmayCgwiKVl\\\\\\\/MEqgf3ttJDzk4y9yIwdOiU+pm0RGNo25UtYAvF53T767HIGbi5JnRod8hdaELAswLEqVyZpylHqAjteevevHo7QFrZcJCXSvdt7DInMUwF97t2zJlhCWSC+QA8pnCD+EavcJQUr8weSONYaT\\\\\\\/mtEQxwtJqCuJ4xrHuzOAdRl4S7xSufLSqiDb5KRW3IvP76ZXmDcykuR7gnf+j8MtvV\\\\\\\/u2TC9b0cxMhg5fpoavGQT0xvGR4OqrI7LQPu1lcVI5iE+Kpns7zVIe6Tcrb7Al5sS4X3YKIX\\\\\\\/WTJLCPu4fdBy53b2PoVR4kdKN6dcE3kDPiu0+t0VIr7Mt0vPobfdUoWQpMizZLJEtjpSfe8QgsVm0GSuC90oc\\\\\\\/HNx2nSXSWlEEIQ2feuD1d6avzuspjI3pgaDO9QpYnHxflfENSakuznQZ1YzDNg4UfCoVuFTd93jRubozL1QY8z4irrfd5wnmu7Y\\\\\\\/nkvo1NwX4qaHoWUFw6kR2nrSl29zdvqHHq+Af5nFqXN4ru0bpqmvl7XKVzeKOlNooaqwxirksALDvR3nSkqCMqoLQaRbVv3+42T4gacFTsk7E4alTZWuXtnrRtwLq4ZeEEsNVRYvRAhiVu71von\\\\\\\/Y4B2771lQVvmVDp72xP7JT6kAT4OYtiwIKx2gRxwGZ2WqJcGUruivX\\\\\\\/0PlYWWvCKmXwh\\\\\\\/TwIH1uRUtiEMRGv9z3GbC2aX3TDKdlwCD7yjWVOazwqvEEP4oWc7nUMZ0a9R\\\\\\\/8DoF7eGaCBZ+15DL8ztKzulRwS+ypQaZvTkWIsQYA01c+mBXL9cvucRw6wJt\\\\\\\/EfAotRY+5HKsJYk86WmJjcEva76svgeZ\\\\\\\/F\\\\\\\/4BRzNF8dEhfqscZhBiqASys1Zzuydfd4KtIr5+Y4qJ3lzwY+Kb2EhG2ymCnSaRMnej8PcBh\\\\\\\/TkjJ6v+n8OlyB8gnDi9ocOLyMkq0zVJMVs2ojXu\\\\\\\/xKfu+EPWcvCp59YoVfDoC0BHdZBxi+HESGsBI8bYOUYfz\\\\\\\/Ejk51Hib\\\\\\\/YG3kmSuBHRVP7vnyFXUqK4D9DBQQ6LMnO2Si8AhF+QGqV4rXz0IpdjRA9CtvLRs7BeljVd3ASjovO3JI62F7oFWymOc3WuEd9pXv\\\\\\\/4RlrHKJhTBQkqgQ1eACXoE0NqNBvS8ho91OO3uFuK5PzBL+i9F\\\\\\\/CoNVl92CZR1TZ3nxYbknNKp9kF+5qjS\\\\\\\/i1y0eQRPOfIKQfNnzDRwD2rm0kmfiHQggZUT9judoER27CEZY2H8IolOqiZhQ63EEmPW1sX9peTRfgk1RZliR7iRsW2WYyNf1gImfrr1MNb7XuP6cbGwNwG37pmRii98Z2+vXwu9SbagKXIs+0hQRpbuISu7ndjU7c+7KwFg1x5LRDp1PfWe7tl5ATmP6+CyUb66P7nUTd2AsXZcCRrpiBZE4tpa80ao8qL3qEdtW9WyR33okuSi1W+O2ArU329GFiEYI4lxsLYlPCn2g+z523s8AyXgwR0okjn1K5jdVxO5EYDyvx9qwfu+hpLxJrTnHSncdH22SRtpBqdhZnOXUuPHOHVezFqD473My4EoaAKwHg3jjAyCmpR7x9DKZ8scPMQ65mLC+Ue3+tyY8RctegWo9RLbKSv8i+9A4DiJGlJCJAAnOi+mP8uFR7VepYKok\\\\\\\/x49gvISeFDvhKTgMFXnGV\\\\\\\/GsocTvfT4uWAMee8Mxlxz\\\\\\\/49G4dW9g8Zqvibp2VIiGAwXcaN5P5qyEKdXgwysnSVs9wJDCcXulRfCl\\\\\\\/gr9+YSQiKslwoju4M00vwQQw8fjM9PZf3sZWBccYeMRMAGEBrnXlp+CMimZ2zhK3Upzz5uji2d\\\\\\\/PJW30SB9hPbKRiXPUWeeXdcjo+JHhKaG1YlvMiOMzV6+ox5plYSLNUXFv6onLkEKQan1o8aqAlMm\\\\\\\/DhpBnD+DBnLjAw\\\\\\\/IrLy7sepcOvMG\\\\\\\/cG75KiEaFw+akjfnBm5iMOl09BMBAo4rIpXM0\\\\\\\/BP1SReyIsQxWB+yWN6VysAF8Q4umfsJ5FTU6MHXnMoY6fzSyscBKD6yONGmGuS023lu3nKFJ0VYCFe2kYIhab8rDuC2Yt4FKoWhLIsOhFqhqSaFf5iwXGvJCg6LdHVfK1\\\\\\\/gf2vo78VXETk17f64V0qv4SqCkC+lR8\\\\\\\/kvfofILiAAKPBq4aJ1d8syCzOTA5L8qSnQVOhD2tK+D3qsmX7ymG86ot3c1I5dGGENVu4fH+zZCRgYmQRFaMUbRU+uoFxS72W4eO3OpZ4r7nM87pTmwyKtHIKLHM1kw8nO0NALUtM\\\\\\\/2ihoGB4hqCyHFTLlmGO8gGIkXoDioHJ1KxyKQ5lypwPSYyp5f2kvinZyKQs4NhiS7T59xYgOcYUjFkgIwEkwxbvUyrvk3Wr0rPeii4b3tkkKLxlWs6r2uHgF4QFaY10NOGf9hU0TvX0+YBMDCcOSBp5uqPUFXB2NCJP6JUrdb0WUvJP8D5lBsWRtZx+i0yewwGFQtpNylrkUxqwNTQYsLvnEukCRNY6P9jG5SE8cSrkoJ1QgUUalCOTMYsvbdvvqyMJ0mkBqwKffxVhA9jMJskTmnzMWnEeiA1NmFFv2i\\\\\\\/D4FFlBugaltcvR5jnZcDwsM+yvOlctOvLPa83UqcCwTJs3JbZZ0ckjVjRAqk8nbEQxmoHC5iXWRhfu\\\\\\\/Fu7s4mHbMQ3o4I0sztJwRt6gPOcPcq3VUURiVRyFeNzhp8w7NaAFQYGJ2hwp1DoYUfoZ8Eiou\\\\\\\/GneRQxRWVIJZzlJgsIBsJjO40N7g9AWBy6W4AB6seMZF1dka4LaIBkRaTWvASuZP2r4knxpWiIWx6iGcoQP1QXbJJB33yaW3JGQloZXW1QqH6qPCAw5QuGXrGmCq7yniD0ILcoHr7FaD2c8J0nhTs0lTk95eYXEUh\\\\\\\/U9MpMmDNHi3BKXMvneHQy\\\\\\\/zeTj2wZECaCVYGcH4n4Af\\\\\\\/kY4piiXH\\\\\\\/iN45vejCM0IvLycTxxcTGngYWYTfkMIVvVY79r8KZ\\\\\\\/7SCBrfdV0BJfllQ+tc2uEdyIB3AnCg9VaS1akVaOCQEAtoo3TNDPvtSoGpOe0c+xUfPE5+mbg1siLW1CuonHX95SSezFuwFmDHc+zIWCeEp4CLNQvAScMtDDIxgvg5jWdk5EX6BwjXydPlD5lx2jPbfiCkvwtA2GDyD9Zo170HGCIEuWBOSonu2JgqQo5DwXwf842NQFLaV34T6umA5aCmtTCDni023IpaL8zTyjb2X5apMuxbAvXF8SbMPpq5Oe4BMsv+0VE08w\\\\\\\/s8ERA4hsycNJrww90kKbtpCd+YCibyVgr6lrslRP\\\\\\\/Tp+6M\\\\\\\/3KE7Yy7c3c6Ah9fE7MLUH8p+EAa72E+Try5ZZn+5NA\\\\\\\/0Bu7flmGH5D+56qFzLU8D2f4YyuqmebavtTyXq\\\\\\\/eKj6hQi7XMuuQBbB9GeQCelkGbRGCfgEv+b7Y4FuvTbm3t1\\\\\\\/qDp2Ejz2MSVnqGk6HaLaEkF3+FFxw0LTcr++CPO1nxyieM08JNL3hx3fb8q1xlvcQkEedpj7wdcae2LMGY0k5zfupCu7HLDSUxalN1WJLn2\\\\\\\/5UkwIOzc9sNfzjU4tt5gEuu2dpLgFSSHXt2e6mXe4\\\\\\\/EKjNJxFoYclxc6iMZmU4xHGTL3\\\\\\\/qk+xH4Pxz\\\\\\\/H0H2E\\\\\\\/aad69v81LNYAeaN9Csapcw9ATwbqyCMTrAjNEq7JCEouyvA+Kj9by2ygVO1L5H7RhcTxnguynlYSG5FTxdcCt5T90VKwU00vRhmjnwm8xTNc8GU5tZfSFB7CjETx4jCvtytyv9ruh9MXx0wtQy3btfGccbmCUZ2GSZesG1Ei3BDuvUPLJUhpYwm1Qf2oCD1D3128Y8huningldmw9LgVEangnrRhEhp3+TMfzPg+mVhkmkjRxDjBczqlVTc\\\\\\\/Cih46lidyfTjppTd\\\\\\\/tUb6uTwLisKwPh9s6dGPOHpgCGp7wnFkaa3BOJOGaeXncnDthEF5OOhg63RlapkcYH1UICgkfrlzZvl8Co338PnpJsl\\\\\\\/MwAHoQWijC+LwTlyM1sACUojcL\\\\\\\/+aOMawe9vcgo3F+k5iLD9gd1NqYVAibB62OaRrgDlcETtfMua+EqLWzH81DLRhOGmSnEcd2AxXynOlgUyj9fFwL1cJ0QxE57pq6fSz2NbufcvJwjSP\\\\\\\/fC\\\\\\\/SCTonlWkLMCVcE9M1bJD4\\\\\\\/gotiTuosh1YG3oniodZiJqesSY\\\\\\\/fpAzJo8t2wH9wsA0XRSN+YYwQjZCyHF+xLIVhFdvDYNF2NBGEFFpCDNI\\\\\\\/\\\\\\\/yBZXU8bYpM3Iqrj9UHP1YzN1+reLSS2cSX5uMippf4Cw5xTr3q078D73N4Cs\\\\\\\/aF6o9dMGacawWTbGEkb6u3DRaWpjK2peUC78XJechLj8deFZ9lEsOeRWZvjorubYdwfd8K1vxfM8Ec7UeJz67vZCJf6zmR8s3D\\\\\\\/v0yw1NUakPW1++pXL+Bhgnv0cA0MWxNWMUQyDP3jDr03VywRasAtOLckswiWZ3XV7xcVhMPGRhLrgl19tgCUDCE5wyKTU4E06g2xegNuPemBF2WXSaCGjDku4gOzyH1BqSe4KOZqxgO5zw8PxyaeeFTEwkB4QFVdmxFIZybHVRDNfq4dfTOFk5adtddFtd5nZynWuh2NsdpgMgGUNnQTuW+wEG9EpfilsiZk0m3FvSuW5Ilf2UoBDto+KmlKCxQ4vUCnE9yrWHmFC+kO+gFy1ycx5\\\\\\\/jbkILbL2pL5uannSgg50LabkqKX0Szf0k\\\\\\\/ZPn+MCbd0gaJUuLEKnjcf8e5Z8LFGVeC5ITQ45LQFCABkbW2XvxyGDUR\\\\\\\/7A8tdTxmzrlocyabUBm3PZRHzvU5X5mbYBHunDNdFZnM486LGh61+llU1OrJji3ZAf+Fbm20jzcXTRpGUzTFrkxIDIuMFE9odSDEJ4C34F2mhRaNKX3QNeCVJF2b227vZz3SUob4k8Ol7\\\\\\\/sMqrNnotAhZHCPJMSrLRbAXEzkdTHNZqqqC9iOrabX+YLpBTvrIwRWaQcWO0uGi3Q0Bkk4FopdnCKEtRuZpue0kE1sTDEIBSf3LWNmmRXz9fIypSOKh4RSY5P+Cs5PzFXBXQ9Jh69KBg\\\\\\\/PfBDyjVjG\\\\\\\/RpybqmYqDsgrkZKB7jlMJjcyKAKP9mPmPfOpNoggctJGHKI25WPTD0TGxI6fShWkf1mp6LdzdfDxWBezvsBVqV0oii5pRrO3jSyP54OEpRrPPNBasCnEBP8ZHfDPn0E+UP9pvjnmEgSOUnnjw1bYK2Z5y2Rt\\\\\\\/I76ab9Hi9IKACP8sjreioqZ9RQsqzShorv5cLa1KRymmffAkMC7o53IHhnb67vRhSREWhpF\\\\\\\/ITE6VUPlk3XcsiSeChk4xEEtUl0dTHARCVdBsscfFiNrPMBfRQufJcidJlD4tCMtkwDJ3APBSQrFCr7G0g0dWsPOgBPN\\\\\\\/d578KJBTCh3xgccQjzB2Ptsrgn4f+h2Qy0qvrryzKEl7d5Pd9hhWj\\\\\\\/DMi5wKhRRCF9LRtEgM1efo1OVaSKfgp7WhLJd2lB\\\\\\\/sxTJhTMaY9QERYC9bhrYX12BwSl2tDugKP5V3pKTCIUw+SyEblLfsoQ7Y1TEFQ+7oyOh\\\\\\\/le6tX0k\\\\\\\/tAD3Jgpt0ZyOOg7Kqd8hn33bPAPFGqgvu1zj9MOO\\\\\\\/Ddc5uQBXVDHDMQvFGdrMoDByta70JgPD3rgxIL5U1Op3EWrLUYeThZW1qRPYeKKdg\\\\\\\/\\\\\\\/fJQXYlJGtZwrOmvWefNSRDsfdsaB+Il4SEqpTGqvIGl66Li\\\\\\\/jYWzuraDZDljiZbJd1J\\\\\\\/kIctvEcYYYet5EYEOkUyGJEbf5mOT\\\\\\\/8t0F7KxlWg\\\\\\\/RUuVbUcoNRVQfRBWaJAiI\\\\\\\/jIGFW+MK1bR6ypNF04hQbc0kN3hQEw15WAKnxVA5yqFhwHAbNjoBFS5o9ysy7dq\\\\\\\/3Gen0ypxls0GN9Mv2RO2C5DO6VCd9S05oOH135HD8EHFHsQ+MN3GZcxI3615UAjPQXkuW0PgWVM9UsPRFyreIKgdh8EqF++NKvNV6n04wJdb\\\\\\\/JyPDN0uUpvLHFtJiD8U7CYfJBfhMNpd5m2Mc3a5V8G8ylEjJ9JbOr8Qo8O0yUORSSB5rVIpEzDpZN5d6hCtMcM24Tq1M7q4Yl3aSkMJVlPHPS5zxfDFyTO6mF8a9GoD2wb6zJRR0YabJz+gQAmqS0vITZD2XToWGNnC5aZXDL\\\\\\\/O2UnH0UBw4UhlPdk+J3mi2iJvDdPkzcgdFKsYxW6lAy8uGohbCuKcYEZ7xIi\\\\\\\/7cMlWUJTFbnmjzG+bnn7zL36tHKajcO5M7+\\\\\\\/jGx2NwEjuE7wkG96mNG7C1CleYZWq4kAFWojqohBIVuEKBnvKKK5srHyPr3zFlpQZvitT8kIjUYFtkoWLvi6\\\\\\\/I\\\\\\\/TeKDiDPeotrnTaC887wQ6uGM+aiQ2mGICjVeLFRKJOAjJks4\\\\\\\/Cd40\\\\\\\/DsUMFfcvDwGyFQa2D1qNcWjkQMH5b7KbfhWOie1+LsYBq6tJyPBeC4uA2I9QvPRc075Hq8Jvf6T3gAAon9gAC5PbEMNOfPd80YTrfnIHtHjtsxv+0WcwUU9H\\\\\\\/MM0mtgG02se9ulrzCtvCdLzpSq5tTJ87VG\\\\\\\/SqucgIbPopBOTvFgXl3a5C52QVrX+GPxPPkeyN8rjK6Mu\\\\\\\/FlSYHQP2ehe401Z7nTLGkuwDdL6rWtb0TrZIj0iBCzGZSilTZOQeXa5Mlparj8En5VXMR19ztIHzESB6Q\\\\\\\/JOGEtPqo4X4clOI1jyEKdolfEYUmTQfFGAO8BcZq5kQUrk94EGBWcJgiYbTn8AptmSKed\\\\\\\/eiMla2BQDmhKPp7F3iqKcEVhlbRxLmOAt2gKV4FrYHsX6zSG+lMMRE46izbfiAZfpnnyYbJnQltRm9q9kEyCgceA1\\\\\\\/Sq5YvklA5f+oOGVp\\\\\\\/tfKPxy8YKKb84IXBMihEyFErAljjcnQFf07wUp3guc1JiQ9769zQmvUQ3IxwZEBJSndZHznKPCgn3rzpgoWqMgEgeKlK8BMwGbNL\\\\\\\/zY1HyEGtmw5y2HxWc\\\\\\\/sP3yksScOuau4zvF0gIK2DDdzJNVFINPtBvglbgRrZy8zB9\\\\\\\/9wDXDW0PzD+6zIEcf0IaUzpihNI4pov5zSgJhKorS2xoyzcBfbc5vbJQrpS4PoSIYU\\\\\\\/FYnW7dLR4ADgjx\\\\\\\/8AqahhjRZzLYGXzxrW4DEtkvmwpq44iL8L727nGIpWA7tPjJj6JRTV10gnXvFE9T++Ldr8N8LTGueexmfXUJfTbQq2wz\\\\\\\/N8AMt4ZCk+YUgnCh2Uph2I20d60rGdoMyg6jdGefqQbG3ag4h17keq+9GKO8RkXa0LtBreO63On3KQY47xGfrIfvdE3Eq49mRCq3HgzxPUr7eUpHPkSSnkdxlhRt6kHXP+gEs6rlhKSwEc360xPKIzJUVz+gho4MUEDm7WHB28aE4O\\\\\\\/J6EchiQpR5btOekCRP4Ih84PaI8jlTyXXR+X0ovxypeBMB8UQ9rYMdjgMvy2BTGEPhznqvOE1xonuTbQYXs+BZtmSu9Osdg1PAsi1tILT6\\\\\\\/6+boQfaYhWy08EQJ63Jv826lLVZtt0wd0UNf\\\\\\\/d3HTzXgF7GN2zv5yK+wgMMQoDjcVY1\\\\\\\/gYfbJ9Y\\\\\\\/5pGOD1UJ8pvHkKvYjg7iBjNm5EGKtogH4oTEFo3AdbRlwca2VwUl1iWWPdlbsU\\\\\\\/+tGTaO0xzaKAFyRtgsh9CRfhb8Yf2MpxVURURxyxomw7GYxHJFVd34+qz1Vcp+alR7RAnAmZTGzko1jwwSyMODUzjArVZ0S+Mu6Iq0jOyezRdqHojo8yUFczHrXxKouKxsEZMjjqjTogOYwZEn8Zwpqili3MQOSX2ySdEnVx6+jTlz4ZV8x4UMc\\\\\\\/DqHK7xpxzFjXZ+Jn2x2h4Qp\\\\\\\/g0px25Sb1h2WOlBXL1wHRl0iyJERiVkSmsez4CVN8Bfx35ohQfhDvCMB1VGvt4Yt\\\\\\\/aOdTUeGuMcXKm\\\\\\\/YkfbjsK4Pfw62FUFY2xZLUpNviy\\\\\\\/TlviNeODpl3BkR\\\\\\\/Fkt1m5l8cFrqQlJ2hs552J95\\\\\\\/l63ON2CNxri5Yhz1eSpmJT9xQA6sVsbA8+tWeAma2xdELVS3rHcem+hO0zjsUTWCwvm8QqGdMOBwAxeDtaSdaQqaada0v8ZWg3Tomt\\\\\\\/BPKMvbLT+Sh1PlnqB37hD1+MXWFDTm0M1LVd4ZlpE4lY5Zs03JvlGTCekJINdtTv2soLr+iaIraC+WKPp1F1aR8B5CYWcQT3teRCRLxk4ydvu0Npqbbhnke44udnbUlMALa8XHs\\\\\\\/ar2QBqwJnMcFQ1as6S6CnWZV3dcQjA8r3o4LNuCPEXp7l400bYfXjx55BwMB59tag9peICmqp6sJBPiUYDK4TeGhDjt0GGql\\\\\\\/BbJvd8NKD3JcQNF7qcZElB8X6W1PcMG9qYqkB0Zv87kfuxBfNxV6iL6qWZO1Rpeabfqyc8r2IP9UmY\\\\\\\/9y1e\\\\\\\/VQ\\\\\\\/PNhnkMDosvWMcIfUcaRRz2M6OceY1OWaidic58ajLMxml06RuJCJBzHm\\\\\\\/HngKHgmmnXfEbPHI2sc52bom3460gBc4NHOXDwb5GEBgv\\\\\\\/AcapT1xfhlh9CUcUKK2UZ7\\\\\\\/1BUBjvMMCyktls\\\\\\\/i75i+Mlzlm8VuUOkyv8adHm49Jd8Ll6qm0N2cMkKjKglh6hJ\\\\\\\/Uyu96BA0HIXagIrWfzuCf4jiTFL9WathptvK21mGYORdcjH1W0Re\\\\\\\/JDwuofsjVU5lqwAg\\\\\\\/cycXfUbGQbnSTEMdYfwrbXTyePfmRCepvZbOYCSoE6tUt3sSD222SHqro9j8ohfawTZAJOwu12XB3fyGFuim9SafQBA7FC9IW9KbkBPNG\\\\\\\/wz0h1TN48CkO3SzT5uoj7uoEqBgCcFmysDD53wJPHMBjonBrS\\\\\\\/i8xQ+wJgNa8kTo1LVvEhUzLTwNaaOQQLCPP9X14ri2E6rzQO395Jt7QECoRFSTJwzsHvxhHV2NYe8LyTSNY08e7xSWqbq8kuRNbClVWT3hZ0mqyDhu1k6lz30CQ\\\\\\\/zUnwY4eQ+oMmL9TqKy6R1ODoZUYO+6DJ5ZuaZN\\\\\\\/ATMSwM4nHMI\\\\\\\/5Rd4Rt0o8gQd0ErVsIuXBn7Chetv1WeIIWxYGHHqQtTFA1m4ATJPqGLdLzyaFmm1yCT9hSXInoQtoWUypQs1EjyqcigIfObn+OqKAYM95ntEFgXkrcqNo10bo+k+07whqh0zmfhRhc4gpC1maqtWyoTSjWtm1MasAOv3b1hfIob48OpKasuWR2T9dEk1FH8ouk3bMDQSZDtqJeTas0cBtgTeQNMkctvktlrV1i9snqyrhNyVOlIdxHOgQNdYxulvS3djub3AczqrWIHKDGE4qj4RGVuh0kb3\\\\\\\/E9vCEJmuApT4v+cuRslhdDqpFLMw7myjFnWhxFDlMIqRGfu3QppBLa0pg8q6EQir2CnhQiegMULmA6Tkh4zdxjD4P33jDGZoKN\\\\\\\/\\\\\\\/4\\\\\\\/Z3swydXDcZl81RKGuXrmNMJooBCOquHM5uQFSjyMiBexgZE5fOjCUEd89tX8qovEE+fYLPwG83f44QsC8+XqfFP+fO0P8Kh5Lyg72Bhwjikh+\\\\\\\/U2Jtsi6DCKZ4Xrz9xbRR9nEqsh+xvIczjrmu6xb8cXvyWoosB7D4Oi592eleTmgUNbbSZFpAZnGQC+fMB5rsT0Gw3ZfKvDuHZF4qB6KapTtD5MWSCZ\\\\\\\/E8WMfO6Tcsqqx9WKcM7poXGvFPSc1d6klycfJPWuC1N01euX\\\\\\\/ZRClN0so1ZtqnoJ6Qb6AhOO1dISRA78RM40hfVAfF0HdPyKvgEn0TYYhprDscrPq2IiDOpOf5ofVDmG2tmfjZUCs2egqfXO1aSPrT\\\\\\\/i0qGM6dqCpGQzMw8qkhC3uLWTM0aMMzZJu0uS8mCBeQhJEckEHBnKx511J6ECoHFToAStZzTC\\\\\\\/p0L\\\\\\\/Y7eL6jtRyHdveUf491yt4VGGlrf0hGjrG6IUh2GaUxufidW\\\\\\\/9bG308A5YniioZXdTd69zxB\\\\\\\/nTxOGojc9c4E4ZcFCBukCFVNK8j6mmkiealxwIVSfHOWwsA7q4y24NXbElUL\\\\\\\/0PioSLoCrfl67NYP139ynPmQbHwMLYi2Rc5UNAkNw8V0uU3IS3\\\\\\\/\\\\\\\/GTE2yoDKyHfzWguN7qGSQBZ6FCHi5c7R9HrOuVFLnbtJHd7H2QZG508hh3Wdj\\\\\\\/lORaNZruNO1zLzOJmnUJwhtjg4rkLgYZIzeKcM3++WLFZVF\\\\\\\/xdgvBdxlRusaHmvRT6bfDXLOBtGr1Q1qrZkYv8rIdd1q9oqHsAmCerMaZRSGzYSpddocyhKFBWXG88OH1KrGFT321pWKbl+h+L\\\\\\\/pBeb4TieOZzG9X2gmb7T5b4zWolGL3ClW826CJig9ss9cUSFUasTqbiBVctKTM3oo9EqkYU4cYLw4lVcdlyZLmBax968IsTozAa9EjaXAxAzskHy3KKD4vIHxt+XpTcsO+eEgxrRhG6utrFMkDg9uczOdP3mXXMY92iD6b0dEjdOPwGVaxe8zMlhA0ICtGbdKWBZBZrDb20MdJthB+0cgByRfPwE96gXZ+XJLeo2EnH77NxOy6ifwaMSVAzihm7\\\\\\\/tEEuJ1yJOWXkMKDENkoTdhNvdcC47N7uRMjztlLyBKnZC2NFlDtNZCdSSZHaIdIoOst\\\\\\\/sivDEKyRY7yzCelj1y6Oh1vQyuEWLZTm1JBvZQV+ZHUwknSbe7g6S59WSfnxacFqT59nQxIBODh3CMgJrYVRI9rJOwl2ZHP1nWpT+3x9Ws5q2mgMutQys6G89DzqRzcwJdI7w\\\\\\\/lGU8YEdJlO4LX0qVK1JNzkhVfo6jdqux+Usuvh3brfRP\\\\\\\/vgYBw156IKy4vMK8bn2W9QZW3guDqPThPQbk9nuWJI17BZmd+Qcys+rkNiMUSa9ElhduQNqs8FGM7ke5tVEYQCTCJiISekm6IVLkPlmJyKIL5VG7bMKnpgYYDVWHsAqRfKcim8G0hpc\\\\\\\/w9\\\\\\\/v79nJj2c3Slk7AFYyLeMs6n5+zotF0waZerpEvhRugtp9acVWBgpa7AOba5eI5GzeDyD+lEer3hT5UqO9FmtIhAGQgK4qELxDqAtMfweEU3d3ef9kHOTDRT5YKQFvlfvPtGRYzxPOJXAvC8wVtceOl4ykFSxcgb4vhGG0zHH20v7HTslIBZLVe8Pv3eYfTdcpcZNn49JeDNmaAfiimhxsdiDLz3tQDltNQZY6PgLcrUdlPlPbjwfIRcEXmHmJSIZEXRRIYz9Nd5iKSAENjEYHMFXkFHgeXzUhYyLAmp7ggokSUSMsVKW8LoTy5BdYpU5J6Xobu+JfOh94XlfEVA\\\\\\\/QPRlHYfX+wS45UrkgiKlWmLHdKm4M+Ht+sUk6kipoh4XYJ4LaZSTAwcO+VB\\\\\\\/+Dp5PecLCZDWpl8cPhB\\\\\\\/IEDXkWPt0sHypHqtdYilLzqpcsmnW9qLvk4LrpwdQSBFirq0rszdtzQPvvzNFY\\\\\\\/uXSplYdr379HTr7b5aXyTtnSdEr7LdIIvW+RavBAs8f\\\\\\\/fnPqEg1x71RFx7QC41F4StNGYERUTQLyR1sentTxEWDKb9hs5BFLkvT0thI+j1W5aPL10e\\\\\\\/d+jUM6xI6Eiy1rBNU0pXYLrrA3P2vDA6nZisAXSOcrxqQ+cxhd+yMo+bG6sBsj0UzgZWgh0WpSuXBoZBkGjygmY5qND1h7tjmSmumibrgdGs3go1Rd+xh3VJA9fclwF\\\\\\\/P9gAgtqXaTJlKeTH+2Sgu90jqkQPZMZ+hrcoU5dE6ReyR6ix\\\\\\\/ejXMIb\\\\\\\/z05hY9xcVeo0k34LQFPAe+FJYDVUxT+AakHixY7mHmgmcTRa9hu9LYDp9uVDTMtI69xKY27pynpwrfHKpDTVJHHNVY52AUq4Oiqd3Bc\\\\\\\/cAgAUyASbroXY55TGbNGYCYnijAzSNq61jBO1tvhGp9251zvpGKpvltu3HSvHR3HJMgsj0REvHBwv9A8lPNLnj2864OK3uKfmgTh6vugtGpwAUBzx23VYX2T3rqc9Gjj8Kthzyq2BnD+KG5mRp2wvZwyOG0J+c9bn5oLvaN0d4yfsmaWc9VMJBUPz2bIwdqe1yYkpMaWpIDS2uftCiRVBSjB7kBw+RVyqalwOjX24m3aDYXyWZMe+FbjI0gfMv3buhOq0IC5dOitc+SQpoMBjQJJNw0n6XeKX9KnJZy\\\\\\\/iXLqb\\\\\\\/DqtEC5ruaPO\\\\\\\/l6PiEesD4g0DHlW6BloX\\\\\\\/BdtP\\\\\\\/ojfJbHesBkLs\\\\\\\/gpVHm2VbUYYbhR80LV9PAdosvifzY9nEjwwN2Rs5z0RKDASms8LWKlRzDXokvGsYgESk8v8QdHwIh+9z0+EIc3gf6CEs8GetFt9mLtqpXR7L+g24cfviXbook3fQZwmufZ6CA4medXvV2HhX\\\\\\\/Tjlg1Dqoeito5UiMPoS+CkNTy1xJeY6VslOaZWeGco8tzuRlwkMC7ewEoMyTHFyiUU\\\\\\\/G89AZM0pixSxfhbNTUQAhUT2IQb5cCq9F4pHTfXS1httHpSq8jflZledrisEr1Os8\\\\\\\/8ZfuatHBBQIcI\\\\\\\/ZwHhs70MV1Z601NO7WxrSdWbWgZ9CJxF3fVT5gtWU9gZ6e9VXh+vUU29\\\\\\\/\\\\\\\/u\\\\\\\/7gE0gSNftT0LDROF6yUANLSzryuk7yeuRpyCNyKHucF+c8vBfsTG7oLHUPRYwJlIxXnRmmQ4T\\\\\\\/VO\\\\\\\/EVHZlNum65GXrzRtX1MVOG6\\\\\\\/IhGrgIfNm5UdA1S85Z6FpqFMP3rF5tRtzosAA16nj3byloe3cpPTO9ZIUssOL4zF3u5P70a1cPCbcruX6J6UXhNINHGORySK3wxm2oTE\\\\\\\/E5sDDmEwzsHpPz7fG4wyKU\\\\\\\/sDjx7Rx+7dPYLlCLa3raMNJId6Y+75LG2ZvN4OtFf0uXDESZUtcwb3LybwcM6b6b8sHESZhPCZO59rcKucW+M9YvqFLL9Oo30Ky+nnZ2BEiBbCGJMciU7I8GIE36IY6aNfV65JmqD6jbUhEOppLi5V1f5peYJ3kDmfOEZvbdQKvngkAcvAy2i+oDD3eiDO1Df7X1QoYNPJOlbADtX4JrmJ1K\\\\\\\/EY5pg7BZqoWgLplG4CLm4W7Du18aqOGoJ5R8pPA\\\\\\\/4SKcyCaCLLmbM7D4SoNPNecDegbPOF7UhtnzyuTEoGg+ywpfMip5sDUYut4KvwLhVX6RsUVo6TeP4+V+QGLOcw133lqu2AdHZ0F7\\\\\\\/EYet1YWavi8Mt\\\\\\\/\\\\\\\/IzdGWXdo26saeY90TSVwjsX+RqPEwPMl63QMPDPYojxEQBAttPZTjVzld7TaATbeghIY71WSfa7\\\\\\\/S+90h4ykMgHotkSK8BjWD5xR8LfZEaB7ymMve\\\\\\\/fT09kl7r3AmPThR38lWMlgz6mJ0WyJPWlPe3M+eOQrzVNcRe2ixCUBWqBsUKE9C5qnwN6jX6fHZjzn1X3ULq6yb4nWR2koxIws9FH7tDfpSb0ShubVZb17MTF3TN3zh8eKR5mJ\\\\\\\/5uOkO3acc1R\\\\\\\/lUSRPawutxM2Px1PedpPKoEvQ77MQiTtB\\\\\\\/9Ir6b0c5VziJcXuLzJ4Ct8ErprU\\\\\\\/7HA3PwXTHU73vxsSpAWcC59wxiHPjcECfrevz8adE6HiXuNdXAjyGvCQv9lqwXicHj2CXvG3W8OCXoUxyct6rM9HJD2GMa3BN63LhVNvFeA9iTfCpA36rBYgG3yEEqz6zBxeIt4b6jwmXhUwYVB6hbqTOgaSLZkjwnVqoVTvH5TAzzszSCMv05v9fDzEuwaqWcx1NZyo5L7YLoqQwb0z5QyvTtggIQM+6hm+VMHeVu9Hwoi6bnDApeCypuibvOo5ZYHeC+pkK\\\\\\\/mJmWazKGu+3K+5EX5qP1kFgCequiJgbJ9se8nkLfSZ\\\\\\\/bgbeYPQbKypZU5\\\\\\\/g7y8Y+NOyslSI+ZdI5Dwv+gwxJkZAGbWGZYE5G7kWcSme4rPP27afxPeh0N3aXBaeTm\\\\\\\/T4c6xvIUaWwYgoAViyrfAZ4ycrAOKICxL96tbWXUr04sW5cCcDbghVGbwWnavUV9KxsPax9iKHR0UX4N6Qi0YFz7oyGusoCC6ThQO0lyhzwZMAAlvSly06FHmXE7Efj6y0GPn5QTTlOIjbJV4sCTZfZkrCqMX\\\\\\\/TQyStv7gNFUF6UrqPXhRlqrnn9cGMDX2qWVHiGiDb+xT+XicG\\\\\\\/UzDmG7qZh29OHDGkZgK3aDmj9iRyvNyL3eUSxzRQsWEryTQbtZqCvxWTPnbTOoDsiFeuisBkS8xbO2sJwq7DkbuivrJ\\\\\\\/g1265MPHjXYRZCQ7cauCVLN5Vj5hLDReBjkj2pNvH22SOTbLOzKHSaxp2OX7F8\\\\\\\/iW6aBg7kgfNhum6eY9cvu+vjB5PiMV71C2biuDXPTESt95lP9sl65+sNaI40cqxU2ZX9fGM4NvNxgQ17nemHwsMCOR1pO53eR3Ksx2vBYRQi6INhfJ2eE4wy7rDxe2cOID7R0ZRWVxBqMT5F99Ylt8f\\\\\\\/4NEik8g2dAUpMVKmpcUmcirzkjcLl0pGEHbA2R4THjgfBrW7GaiBhdqj+T+Cm5XGZoo\\\\\\\/eerVTtN3IiDKNIIMNF+KoGlZRSbXDS2za2v85xW24VqtYyc7m37ZgiHpz0n\\\\\\\/tqrWtvydxtzZfHem\\\\\\\/k4+Oo2XLJyl77iNuLrvk7QRgtojDl6ndzLzvPF5yJUNPWlNh7TO0jLRRDjmi2pB0FSpFvjdcIpyKvLsgmhOmaXw3LCdrJhGwGBiMD+ly3yW1XvuYsMXKs8dONDL6HbeL+MYsjKU+Vg3weKtoQ37zAlmTMlzrv8O09467Pvx8Gq9zu6se9u762SI\\\\\\\/VVfQ1nEIWGcgzbKVZS3ufBSgitFgi5ygWoRMFUjBQZeMZ71orrSEc7RkYI7EIcTEZLgLgK\\\\\\\/EZduC2O9j5eV58XTm72Nl9ceyHDIWFxLvdOeAIkGQtbm0BAtuY1rbQIeQl65sU\\\\\\\/oaerY0jc1Dl+AigIjpy95wTtGy\\\\\\\/\\\\\\\/BTpfSEwpP3Sjo5yT6mhDtDhTwawE6hFDLjxNN9NOr5hcgYabZP1mQ2BynuFwi8gJkCHG32Aeh7j\\\\\\\/2FwrdoR2QL0iF9P5Oo0GAv7GbPTaiP1fm739QYS072ScztfTQACllcRYGReO1uA8g820hnUrjIjFyfFNAjPzciV02A0H62JVudr2dcqjfcM1suq2BCoyFDzqw0FT8CHZhFmfirgX1cTg7rnxdfqLlXazYSzQas9JOeTVJo3k+ENNGNZ23T2UAuW8VWJnxDBvEk1Qu31D+rHGB+UyXpPAYRKZy5m5DBt43QD0Mkb3chV32NMAYGjfP+JadSwz+5w30qOYVBrlS5K2sYoaJyb\\\\\\\/hesOi1F0Ctn4YbwtGW2l+2Xf8nJs0Bba0JtWzr2A\\\\\\\/47XBEtDTYPpnJvAvaB4JpSJezC\\\\\\\/SKTWINeV7lmJRwWMhMIjbVQ6WBFiwSMvH5\\\\\\\/Wgai5BOPtmU6zcvuGew9K9HT577B+1FkQfmjnJMk511jTjFPwEm2Ibk7CbImYP7NB\\\\\\\/js3wQBmVNwfoYis1OUv20IyFLHwdmLJqcHn+97OqE9wwJUJpgd6jUP7kx041ILTgGxu3Y8iKGPtDsh9RAEnhqrTkufBecbpai9ZTgc3mzpaCkCzGbWKIwz+e4b6KiJFNUVtyjch4R+HURTmczY4sp4fA0yUsdTy0Qzp\\\\\\\/U3opX70OT+dm9veUe825DcfLWPeA40aGBaGfp6jV5LxLLMi2u+y3lQ12wcYOVDKbUfaPOb5aAWWAVcxE0IJuSawpyZfVEux3W\\\\\\\/IonXuhu9jFtdZ97HxtwvN+rUPOT0+g0QoJ+A663EW2\\\\\\\/Zqq\\\\\\\/jpRdh9L98NNwoR36v00nAM1Ir\\\\\\\/XA9fkK+4SxHpbymRg+Gg3nWuWUuuBwu09LMXIJH2aOKekvGZvTnL8Nc4ipwYBx3CZLz+VpvR7xvz3Bglr\\\\\\\/mtEvNNJiwriovrZIegXwox3rwXXXHXOW6omYIc9JqwgKfY3Mb0R74CL0sQwU0pKe6OVnEFIKIPqcuwYUwFY7RbuYqrR7n7g+gV0wd9v2yqM2caRjsa7HJWVRkKO8CKlS37sVycGcL9vK3R+U99tHiFxzivpWYsmAMlEkKf0nz6rifmhT+mkvo4jQZMamorFXT0\\\\\\\/adbbGK5Ox5Qx4bw\\\\\\\/Bda7BV\\\\\\\/QtXe2W34QNnv4nLEgvDz77Pl6Ob2m8Y07mXp+rRh\\\\\\\/5uAp4gsIG88L5Kx2IJq1Lye+ljgLhlA60jkHH8742+xU1cc6XSFCtBly8x1VRdp4yg+J6gL+tt87F3qkOQKCwdqQqx+L+abUzwUx91cP1keKCho2O8fPc\\\\\\\/glbKsuApYlX4B4\\\\\\\/efIR1mmpkqbz1sYUvgAlZGzCKgoNDDNekBPyqLIHbTztSQZKyYxVV4XQjupOMkE7905b7GQFYcTff5BwqyaPOsk37Xt8JlYuIIb9akZ\\\\\\\/jIPTwY5ceBFryHCW3e8tOJBgGVokuWE5n8\\\\\\\/l\\\\\\\/uLldZNAwJZnIbAgfet47xDTzDKQp1xe3JN682tkf\\\\\\\/vev0iy7J8zpWS8n35MltZVGLzkfZbs7j3\\\\\\\/QLRW7hpbrZ1bvNzht467sdnpmU68zMCe8H\\\\\\\/bTSabuWVRCrDxnhce3lRb9gbNXFUKwbpPI\\\\\\\/qWpmqJzMOo+u0k\\\\\\\/3wmSdT7OjDAwsyYzGPrXUZDmhQ6Na8Km\\\\\\\/xRgoP\\\\\\\/O25515l+Zbv8PhuLr1hmm6VRQO0Z\\\\\\\/gR1pMhsSJWSkHNCjKjopYo23NijIAtiT\\\\\\\/gyEFXPaW3HmrYBD4Vv3Yp9DDc1LBmFaOMPS8DhuCdzByIWm7Ns1\\\\\\\/sD2+uL8JQ0tBNhRH2EJG4kZ6WWGdFzXdgCftiL7Zp2csUuJwviRZEBdNXp7dsh\\\\\\\/O3ubpw+tVGYoSM74mFW1XXXfoVZkIM8xXVskpkLorgKx+3hJrpkX8O6MBEN44r\\\\\\\/dg7dfD4iOK9wVz2F58VZIexts\\\\\\\/aPW5gRvxL11OCEkEo+zMgRi8MZCXl\\\\\\\/KB4Y4CIgBNc9FDer8Iwhm2ACnGSr+We9s7p+wFXusNaf4v3K56oMCC\\\\\\\/3YaGIcIOu4ALQ8haQpAQcQuLZpv8i03gowxjN0sjTThAvfDwRBh5t\\\\\\\/4Hz70hQ93vR6nOUYq\\\\\\\/VgYVCe7m9iqJKdu8tDfSHJ7ta+Hrc6++dL4zmjZH8BhNA+p3q8xtbn5qBR4fXSxR9xjxRrTs+Mx0kPR6dThHm2hKSLxoiw9GOTemAXKCOWclCmv8Nz9Ek+CA3IvuDAJLDKtCmzmkOOnu5J\\\\\\\/2sUbgumQ1rgsRJgErIjeQ86Do69MmHwoyqv05x5JtUbaMQ1ziC4UAWQxmAE6y7qJY3b4H9\\\\\\\/qWoCzdFWljX8CzVXMhzsKsasnfaVU3jE8xBH+X3Ja6YhJqc08EGTurr\\\\\\\/lO0Y4D25CnAzwaDh4WMrjbbqSKipacvsB6y2KzKf1Qh0O9x1B2osaqntQrdxf6iFsbALkEc+kVvTqQqesDFf6r\\\\\\\/N1HlWMg58hh1SzrwUd9t6NtXBKHfwe+oUYPcKGGH2F1g0TWQJCFpgMhFIQnr5sRFESNTUlldApymOw4jKU8kHFCvipZxudUsVkB++ujLJQpwVKRC7yjqP36nCOhy0keoLAoLXvQx+csj5TMlcjdhIu8EWAI4oOPLSw0qnIyfjO1VeFW\\\\\\\/I9xFKqPid8rihiL6veFf2YXscjy78b+ZNZsgrLr2Eoom0DvSwnc6PQ6OWLWefgH2UZIcuU10q8dkWXARcgIhYsOM\\\\\\\/qydEhi3wKP7XXtyMj7k8mQQjUVRBAI4f7I3PQuJpGvcG5yF0ypC5\\\\\\\/fTVQRevSHQM9\\\\\\\/IiumylwENHjH\\\\\\\/qcB++SKoFd7K2DFlHArCWM8GlI8gWVU5gUINPjijTPQ5xpNpUoDhwjZHwQQIMRCC0Zf+dxP4zEiZI0yMBUumWvQJJlNY1gcAT\\\\\\\/0e2vswywTLAHS3YxwdLq7lAqPVQSYFzY7ND1prGFwl9SwMJCJezhJI\\\\\\\/mMTBtR\\\\\\\/kc7\\\\\\\/ZVL3Yqzv+Tl1DRlmIsfERkMM63UgjxZlvzIVS2SkMR4r+kHCywre6zwg924OymEUkKqfa8teENEmsYZQoHfPKECKsRd8jXY\\\\\\\/MFsTI8perzB9U\\\\\\\/drjyRVEkvDz\\\\\\\/DQlnorJ1Jspz9ElnsnsnaJgCXbO+RNZJNJPbDDmdI1FQiPSUikuwmAPTKAMy6JNplx5\\\\\\\/P1q8ZICy67tSftf5qaZETa\\\\\\\/ZrBLYTRHuRT2KuFy73laAisHNmoGpJgwTwSd8+kjNO1L\\\\\\\/cjfX97FbZdzLc3zY3Y\\\\\\\/QNHFCgdOMJsjxotvp8DFbKXdz+r+H7XvGDKBQ7ufbZQJhUbVxHrAJRmJ7BDZGfjP092lxcH\\\\\\\/PzbUBsF7OQFglBgKC6oqy6aEkzirdodvJGJP0xzKRJg9e6i5oBm4TfIEX9jKJYHPB8wkh0N5+8L9Twre1yrk5SbdKANaZYziVoz0M2OxADPmllhRM0vgy86mtrE5d5QWm9Eimph9EMgR6iJfMIHmMozZjpbOUTU3aFt5hxxalSIP4GL8JsD+kO3\\\\\\\/ZAlYxZVp2hWSFqayhTFwaL+Nvpx1Zos1LABdwgU9\\\\\\\/lRv+E2hJHQZ\\\\\\\/rmmtKoYo9iPTxAIZpktJWLsSQ9h9JUZIYX01yy+HsMm+\\\\\\\/krwDnyPed8dOcrBqD+YhzrGjq6WGuWCeagfvODe0W3Gpppja5sPOD0UdXdy4DBWj1+vIfML8Dqsel7gputIjbIP0gr60rMjhUXeDMXQtcOV3BAj07qVuWmXRBag6EU57rExL5rwMdpONr0iM8Rp5GtGZL0GADjhL0XJtO0byRQcdo+D3Gs0GdVoQWpi1vFOys1AewgDoiHG8o4k++yz4lYdk0btngV4jgwwe8M4iZYdtFC1EX26QGxLPhA2jR33jN6jFIG4EqdEO2+BpmKdndURliHeQbdP0FvxtIBsSyUs8ZwbrJ1wpxc5GZ9W0TizlRGTiYjh4gWPdMzE3wS4JcET6ggMDZzps\\\\\\\/mwo05NenMKRmVgYAcxxyy0Byd11I3BGR7nvm2ySuEGtAqp\\\\\\\/CMBdWxvDD7xfuniV4ifA3eOfI0AM1Zz9zPZJXS2cYIG\\\\\\\/P\\\\\\\/bV5RrQ9OwDJQwfZtC00uU5n80CkIl7gf75Rj5WFcSSzgwVHxU5FMOSWhdPqT9K1mJyP5iMozSwEGwnumtEvI0MpMmDKisBU45Oeb7r3QKeAfZ2f1aiaL6ii26gh62NBg8UwLg7TdV53FNTuFAdqEpHaPpIyR0ZpwmIgspweh6a48zc9pKy1RpPmX8eWJwzXEmKiLzimxuvKIyNrL+nwJ5UDq2dNVRoTrCearPCyOOda41th0t4443\\\\\\\/qKTmAUYzm6e6CmCwE1VhsRU1tfIRBZMe9vkw8Fqq9csPvYq7Uwa5eVx7fw\\\\\\\/dabrOAS+Y40pXwN7txWiXua0DWFBZe1gOirvsTggDYPQ8itwpHLLxTCcRwfpy4IafxG0yiHtbM6ZUpVW54lPzX5CoKQeYHTWSuf3W008uQB0LxfuVUgqLGPUsuIKPJWDrFxfW7GmfHMPH\\\\\\\/TMTFwZuNAq3dE7mjeehb26AiYsDO6qngGgAcJUeox1sVI2gIh9cO8OcKidm3uQ7UKb3NAeRVKMegw0+LcITBcVLMQbPTSbVI5mywJCz9msJsOp8\\\\\\\/fZjTqWa5W4wsxXu+6xHFVC5UNrq10lH8YzILIq40GQdID1KzfROyt7zXwR12Tg74s4EdUWMRYRPZR\\\\\\\/sb2flB7Lyt2xyTori93TztCyGuUiH7B+H+AAoNIKEg8egyqJu436GqE6Gk\\\\\\\/Prvbc8vVplET91eYzdwzCWD0nIHikS5QDUE7vUx7v6b6b1gQOKmqLe9SEoqgqIydSCpfPj86AHhhLqNy2kyvGOi67FyDrIhVssPdxibEi9sXCXn1GqSx7VEBPc9VvfFRYv3D\\\\\\\/8LanYvmd95ORWyB0jF9ickjXuZdMJrtfromfHrZUqoOo8fJZ4ZXZIWcBa\\\\\\\/ixgqxK0kky7VrxkrCtLrKbCvLgxBLnx0iaXx4OzTNIQm27+NKL3B1rEl5dQ0Z31T0wH80v44cDC3pbCvVKVw2+LfafAehFjYO17WFaP5mraWSxbEPavEWE1Raoz1GIsQcaL34hUrni63onTxbcpxHqTzPVVLoXJceQwl4x9RGAzFwFPx+UkZuqtzC+7B5gI3BfqvXNSoXuzvWSZrzhOipOa9BMGYs69TbVlqwzINO1wgtbPF6fCIu6VhjIsHInWIjSJ6v2tq4lh25FW5dgervIEY0TROsEamcIsxKFj2NRe6LYgTQmYGy5g4hBzc5NJ+a7TaXsW4Z\\\\\\\/tgXKlIaJQHZRz7dvI4j9hZHgVjHQItQVezpejF6KsZ1nPa6\\\\\\\/9BROjM4y5\\\\\\\/WYTx3G7AXomOPB3nvC46Y9NH5aBWHNbc1xLUhBY\\\\\\\/micwihfb1DN4jS5y0Lm7vwvzdzBi0yfnzKV+q30rArkJ5f3u4af3cN3UknI6vJadv0QZYF3J8aaEAPXxxNFfF2zznQq0pQMUmsmHShUpMF+PviFcgafJZNynRfnNin\\\\\\\/Aa9CCAwxvleAgVZKWVeW+vBLTLEyw6SMAUOC5FyBzR8HaLiXcGhHDwe5bMC8ERZ0B1jiCfipzevrxUMZ6qsJYJUZk3e8FC7WA7xW1f1pP4Hdhycz8d9AGo7UwvOFBvPsBhftXeibGCbhmDJdly76XNmu46434l28fQjRmr2E9WB7ezmX9KjszeV6D6UKgaLvXtQFM2lUGCYFSiat9dB+ickS7PZDRoOPlRfMEQi6c8Wntd4Swh7ZtwEzu6ltrTteU9nxerQZ3QHtf50MjKEEgngWMof9iPW8yjcMVaWnZHF0X6cHu32rqCaALq5OXyfYk2tLhzC\\\\\\\/JqxV7m\\\\\\\/zb\\\\\\\/DbXDmdpeMLvqQ6gKDDFLy05Q9eu5vNZ7QJPUN1xsZ1nycJj2MZbDrf9gMowfJ4hiLLT6sNTF1NbPKDpYkkVCWNFfNnfueJqyrapHbeE2Ys55wkn44pPotztVNNJ9uIQGq6A+B0ftqLUPeBC98VcZrq+i38GRP29s5FjyZJYFgFznsawxGq3xTWkqugQWDLYaK0IwupoTivuEY2LY7zIFJvI2aHXBs1eb4stgGdUkuHJ6S2Dqf3uMO\\\\\\\/ZfwPVoM+eppxdGnN07YoQ4mhhJFqFcTPjlqGdMiiryl\\\\\\\/VI5S0vPjX2tOIBzI+gA+OmDMwuz31gnTfKSV3JgvigRpCERDjv2B9ak164zljwDjZ8QvLaStTc2lP+xHOM3PoF3DJEQMFkFbxuSGpM+UUuwIplV\\\\\\\/GGVNZcp77cVbiarETKUFQtgL2ULYFrGUCGTOgrqspyAHj1d8G\\\\\\\/+LZHmoQ7q3pvJoct9ss5OaPsReXrQK2vnxTO55vWXjoFc+dkAHjE19C0KY2QDduqKiIw\\\\\\\/nzxgeHWsndDaKzNkIW487ubQ9QCvN1\\\\\\\/9Jv+KUoYc09LDrVrfqReV8nzB\\\\\\\/OjW7Jf+kuNnEpi97VH40qHsYmwv8Tpf8Hta40jz836VPjF1NFXbmibFUl2OheygfVdYEgl3\\\\\\\/YCWj2P0gBPq3L8A5wORx6U2k3Ox332OKtCL\\\\\\\/wUJ7r3gq\\\\\\\/AFQxuE75jXNHpN4ey3Ddr\\\\\\\/SN\\\\\\\/wgRPkdtHxP0w6iY+mRGoAX97Jrfgiy4b4\\\\\\\/Tde1gKZKYkuu8xgqYasmvfxxkklJdTAF5bWsPGjomDRNs12zL0BLK1N\\\\\\\/4cD8L3BJuv1x3AzKm6E3tn1C8AuQWYe7sMjMyVFxem1TXMtANmAiuB8tiRRIivV9Vl1SgrttDGihkHXCjYhP0OfbA3kQ+sn7Bsy4K6BD0jgAEpZqzVAHVVJn9OQYqUWKDMjz+iBvjJIAVEL0pogdhY2zeWAWm32UCecO6N6vG2haTRxCGdUhuA9MJSeYzd8E30xWvhC1xdyulq+yU9wV0A75ejzj9sZ0bHdy4rxjUsSgWWm2bd2b7tWj2nTMvwcrq2Y82deygXtc2yX0R1o7GLJz+JDdHLjJQouFWYvyA3eV\\\\\\\/aHBgQgEZGPba5E8OCG2UyjWl8OwOtwaMGwpfThRMNsf7HF3dWkXhBcCyw8T8hoQ681Dwe+ZgHZnlIUM1kBAtKBmLZQn8cZlu63kF9LadDM78dtQA92yRmw6v9izXcvFXghoyqCWFhba5btfr8vXYZCSsm15Plznh0ttdBKqMQCeuvrdhnzN\\\\\\\/YpiMbee\\\\\\\/C8mqrfadcGhuBYAsIt3QUQY4iOsntz6wgflqOJCmDqsWl+sGiG8JZTJHn9v9JatYUT43nLDgYkjPCdYKNrGFPs60E5HnpTmTx2Tpllvox1etY0wnz5UERrZEzgayfJiNWKZulMm+rwqoRTmpfeMOk8ToMDxmXRI2fLPadd8suqKvaBN\\\\\\\/wNvsso9NCr7PMrho3Rxky5lBgByc0h\\\\\\\/TMccvmcl4SxN+Btd3aLObBugxxyL7zrrz1AoYzwnIVyPGyfjhE7ml2nW95pGGJPNlzvBw9RsFBkfxHcYFZAHEkQMAzqPDpXBEzwHhU6o1dOCU3Mm3LOi3lIgFTzg5AUiSepZyN+FlKAUcgKAQmAUixNBSLqjX+X\\\\\\\/byezf43Hk+ngGvqWQmJF6jkOyxWIZ1PRv7ChuW8qjeHkgR7QFMd3rrw0qT9L4LEroCs4qv6NGDx2yN0UDucDbOz+8ojZtMB2ivNRYed6aZ72\\\\\\\/p4Z8+MXytR3Pc1BrykFV7590KVrgmxoj5JuizoMExnoKyTIfKkB8EQuxG73eFulefIwzjCewgZ5lzxOdgFy1ITDGhvX2vUUQsYaE8glLPJBn\\\\\\\/JL+KcbU786Wh\\\\\\\/HW1hZ+NvBR9EFq0XVOW3pFK77S1rm7HDY5AOEr9AA\\\\\\\/oTKDFRGzdq0Yj6sR2HhQ9QYCqW+B26bNmzt4whcjBl1x8ehffgYS+eyoY7zttLDi4ig70JndrF28Qoskn6WayzgFa8qJXYckRNQ7BAWksxpnJL12FHyzJ4JYkZ0qyohaY6IARvQwyE+3UDsDyNKPTou7UbAeTxQHatuhok8jkQvm6OcJBfExb+VpX4zJMjaAeB\\\\\\\/dbzvMD\\\\\\\/VBftj5OT66nniWVISlWIoWtfZFTSirpHh1n5IOlgvtJseLCKDSpn2q5rRUjTX\\\\\\\/wp9vPjGHWDZbOZdQVANytvHJrqZrBZUNCY2JGomktd73mCFLiBD8V1TfH3slu67S4GzOO4ByF155TMJD9VjY+lruTfBXmt\\\\\\\/azJ6PbKkM2jR5SW4KwAzjYJ9Dw6XJKM3mP+TrNFLRRXmZ9QWL810PniXWb7REjWR\\\\\\\/BtzC5qsNj\\\\\\\/WRwtgEhl3TNSmCsxQRdnC80zpvlLKW6skGzRZ8K8xh7C4hIjiqcARr+PNn7Dl3iEdmaY5hKnKptdv\\\\\\\/u60ekLnsMVOqe8554uQvucNSPH\\\\\\\/M7NH8aCHWrQyhb8tNEGpJ8YlPqXmE4bWfBdweXnwHgqptyQlR9lHzuzZCiZYqYPiB4aXLxKp\\\\\\\/i9qhtQOAbf4ODeql8aa17QOORZKC0tyBOxvCzKRgt6F4rJFdeY\\\\\\\/imhgxmU70idTyyQrbv+wscHm6xDw0nt52VjDwtU3hn1ZxAfnhAnHJX+zkOSS5ZaJaZOQSG9SRqHtRrNn5eg9de0u7K4hGU30XZwt6YpLUmwa2\\\\\\\/pwfBfyYs+vIpHDdxOMhkb7szc9JROwt3yUywQ7jioynJ7e2dPHOO4svL1byJHgH2bOuNLoxnH7JpmiEGwJyOA6QXKr8McJbRa6o9NUOOSAgeCcukPJZWdQnNvppfRMgS2JIYu7YKlr38cywpHdUw9XxbDBBpsDTvsq9XZagf26sPp6woLZUvsZ4Ckf6rE\\\\\\\/0V3mqlh3F2+jnan+uKbGLl1IhMDmiqlG7pxdU+NiI88iJ3DL8sMwBHD6WgJlPmEHyK3YuQGHRmFHr8xGrZRxhT07ZceNUYhwI7ZjQUwxrJf15m1R1YvLdTWv+5VCUlqgURY4pf+N6pFDM7bCsEBEFuaVuAJI5Os3J79A+RCCIcGVYSCiZMsZMEP8lggBquPeGPv7Pi4IOpaznAZMy1u9mzSbwyfVU6vZ5Uq6aV5hyuU4BkRbMzEZhPCANI\\\\\\\/JjPvmGTFlni876tESe7J3b8XiICVJFBREpxwXklMs+ndEP3gL4Lwjd20FCAoGVKw+\\\\\\\/srlOKTxSBVhL3LbpBxhNFuVAFUjDSrsbcQ\\\\\\\/utzrq7ngaFseqmdLimlfMkbfA8HW67fjkvpUOAOFhOqZfflygQHdyUrENddzIFNnysDgkqqQRny3Gb2TIAJcX7CvuX9HQEwwPhoCkzfuXVsMgyYw9njz7qT2UC0gjhaW2F6+JCuCOmYS2UDEI3rlGSYp9mvcSGN+jPYU\\\\\\\/spPQN\\\\\\\/3sKiG4X3pIN4Ed1lSQ\\\\\\\/uIfxAFgHe420yYHxubAOT3pzYNVd6FnAvrBPnvitbld9Gjdynh6YpK0Pz4XCLQ+Neuu7cnmxYBFyO6JZhkOojW6QnkdasHXIC5A7KjF3RqN9pm\\\\\\\/4Gpgy4h4bUbMBo1uBFzOslQ572UyWWGndpZK5swo7r+B4cmYnuzKbPJ1UMf82an9pPjt2eewiu2tAjtsQjUnHaYZHbEeVkXSz4WCmtbx7uWeSyq7lgrhvNDAsYQ5Zc7tOnZBpSDQdshdTtztAKrzUGG4R+ZNGg8DKfC4ycWP+D9\\\\\\\/Ng12o5by5EyPVEFW1vCIwSZYREsUwLAhPgtCLFoMiqHgLaWoSHp85UyQwg195duxLRr0Xc1OV7Nh8G56ZOE+v7ok0mxoRj58LTf\\\\\\\/97ccj0CL1VQeOg8YWyQ07hM+dU+EWLch1zapXfQ0J7ZNiNk+gV76NnHUdPCrpzV5arYM+sbv4UhQUUBJ3eW8j3yArz38RrXv19VtcCpPDsTWdCjruGc42BbSA8uigsTrAjQ58SqcGChLqqQoX5zQzZnirak1M\\\\\\\/xS7Jf8FYxgtWmgsSnmcuZW853UPjm+KZSm9LSlykE6rdEfggQ\\\\\\\/rBdu8YXUT2i1MpjOmyURkBhPppjuwQjIaD1iZRegGAmXFVFmfgOgTKZB+8A26LYKYAZjoQuxCjhS8k6ML5Rp8\\\\\\\/zva56DMxWxSx44TX2oY1VQkH22VMQcvnxennH6A0LJfeNUOu1B5urW6Oi37Um\\\\\\\/oePrFUSsgABxoE5jWn3TkwffhBpm0Dss6tK2a2JW0kKApoZ9ruV+a4IhoVdN7JYa7xNfi6Zf0UcRJ8wMxBzWSshKOS4hlBU8eHxKSWTyXXVGVtNZ1shhUi6nPkcziqkyKeJ22bBGrEeclnN448lParKaExGYZR2S6k2zPp\\\\\\\/o5\\\\\\\/6K6ma2nxuGvJy5qMCJPLaEqKpo94xQrNHnu5afmRlKjiX8Vvav5LK5woGOS1bnZ4wZa9t0RA1YqLfIG7bU5GKwCJn9IzVs22sXm1CyLpWhjTIJlDjo\\\\\\\/mGfOythTArw0KncQhTBxPGDxrMagfz+sBTo24VCp+wGyTC\\\\\\\/xCAjvMi1AFv\\\\\\\/GozXR9zV\\\\\\\/Cy8OacyMg4LHBhvE4\\\\\\\/KK9aXXxiJyrTvkEOnWx\\\\\\\/HeU6G2LKxVEm\\\\\\\/Bs1BKMFIOSC61fJIg+Gn8v8q324\\\\\\\/3KrgkY7zTn8UixHb5CkF3J2UpxuJgaFTPcLaKQSbmOpqB2VLrx2zdhZZKseKqtM8uW37S8YHcrnCDazznSGWZhC+iy56u10RtFE8H\\\\\\\/FlvMrs1huHOw99i\\\\\\\/Ha9IMO9kf8ewkO3dZkKYR6ufFPsmPDenFsgRlif2NVgZx1MYmw5pmVWT49KYMBdDGj3E4gxa9jVlehvw6MSOO+\\\\\\\/ap8qUGhK2jcNEjHv8NRWd0\\\\\\\/NeH6QNjLTYMHVgRfJZc9TvGXslQESxCz4DB06PhJizNQArxPIHPumy0RNjGyfbqxhJnpVpeuSBR0DDgV1jM\\\\\\\/Djt+jNtjgmiu7aX0ukyXaE2cOuaDH0vtT7vmaudcUJsl1cd8ndvbjST2vgRtIa9i3jQxUilJdZDDmNnmSAbnPEsK8MExRbT6odHvxYXSUm9SD9cPF2pJeWEB4bPQCcFEsRLIZa9at3Ma5YuQqL6RWVbfLvnZbqVTuAg77Wkh9Bdfj4denpj9w3gtzh51pPPhpcs3Nt1axpfsNCf\\\\\\\/l8fGw5nKyyZ8ulBeYHPGxWNnN181OvsY3qhBysgzn7r3HBKXvDj8PedIw3Cgk5Ptss3l6i7a3EnZNV903zS3K0N57VfYHAJltjFoR7uxLfeMNidWnIld9Va6O9nW+4UIGd5Tal6bLkmbu7R4dZ+tLbvc9rEeynIC93HEYq0s+OXcG+MuwbpRzaT5FTrnsRU9O1vVVlL++JfB5x\\\\\\\/guD\\\\\\\/nUhuLPv1Lxa2CocEetxZq7MzWNEHybUd9fw5SbziPIF2vGK1cr8eK+Lb5KHTDA517pr7kF0AeW3WjlI0DhQ0CNoChv97PQrbSff5lI26fEeyzwdqFKw\\\\\\\/7z6RtT6wG5s25rdSQ5PJSHoDG6Dqj2fXX0skaIvNtU4mV3Yr2Z\\\\\\\/FUssOkePlPLNCp6uOqBYGACTdJlvUfAEMT4lwGnq35w2aJADCbmBdM7L3VOtFILKzaJvrJHy4JjcBvbQm\\\\\\\/8guBvCnUY\\\\\\\/UZ1CrmJMLCcLrkyI2coTFTjZftYzonzDRv30k0LfHRfuZhn+jOWZeuS6gRE4dTN4t+3O1QmVk7X4CCm2QySHN6oTuLWWbIbtOTs8IILkbDa1Ms06kyFZH3Ig6QZAZq+X0rlLrVdt8LAC5RpP17S+Oex2tVnlnySKx7TM+AeHz9Ur+\\\\\\\/C4fnLf9hk+IWWbTBsG1SqmOfctYlg3oE+ALCez4VfdvJa+HLnv7GQmeHFj37C3T\\\\\\\/55L+gv+SbO6bMI\\\\\\\/NEDzZWUTHHkim2B\\\\\\\/L94Oep445GjCrArAkoUrQIk\\\\\\\/vvV8z84O\\\\\\\/JQzvuBmyop1sc68GAjYsVRwNIYfMll3vWtAe1Gi7B8iMBJLAhXtToj5x\\\\\\\/ycqw8epfOB3AMQey5J2xKdqJkVXY6gVS9Klm9tfZ7L1U0GQNwcW9GlGhJuU4\\\\\\\/TYPvlEZq9ZdSed0PosT0DU6fWum4v+IJ10bV3uLvUIGTeJEDI3CjxMcgcKaRcqg\\\\\\\/vK94vMS1\\\\\\\/Rvc43XZ1429Vq0psyhaT3OiBIb7rbovfYFGfQLW1PBBru9YCxLEBWZf4JaE\\\\\\\/p3FCsOUkpGVLcGvFni9HJqkBzbIZECo648avOYtI0SyLUyyT5+Fgx+eGcjabzUDXkSuxpdVvSCdWKwV9yjhJqBvOVOuftBSALAjO8FMUcpgR\\\\\\\/yLctmcvsFEjGkYNSp0dJLoKUMk9ws4bXMyeP\\\\\\\/iTX819d4uCshIWJnMravjfx7NxNN4tj8\\\\\\\/QOmUheU23cSBofUBU9G6SayhA0lS+WuBTICTWzbmP+symYoXcI\\\\\\\/c+VWu0XxfAp+dn7RtQLJiDgoJ+5R0P5nNbV5VbGC5xV0tZ\\\\\\\/JF+SKzslLmP5qUIr4sB\\\\\\\/dqAlNzis04OagClwbIrS02b36xt7d0dhu9f3mrPcCBAK+N+S\\\\\\\/jci7+sP8R+8bNItVkX8cUu9e85g4Rsk+oY+50ZuYDaL3pZ7BY+EB5W0ryA\\\\\\\/l+IMtRn21n45LTuGZ3HgzHl2xFfjZ4Ik8pA5t+58\\\\\\\/vLQvrwixP03AoRUJpCiR4C2IYN3e9mhXpQX3WWUQ3Rqv9hh9tlt5C\\\\\\\/FknhMWTAsdIRd7OtqBKJy5UopFwmZbdhtHSku32Oq+zizIPkUkz2F8rBRbHI1Kx8SCPY9D7cBllNPDxtMPB8+a7CYeVB7L+5WWZQTMfBel8Ji64kK7JMofJdzrkxcN8XAcKLy0mJKKo7FA1nSeKp0EzjqHTMkLbJAddW2hTQv94KLMwBfyBV29usrmW9TQUDfVZlWoqT80k+djncciPPb7zOA\\\\\\\/GEJxGOTAi8fI+H\\\\\\\/s2p2po8\\\\\\\/+o1\\\\\\\/oA8PhyBjFB\\\\\\\/EmI8w59oeQTxamYsG6lTFanKB36OY2iD67jrh8CGmZxFBVrfFTb1DBv6KX8WksHRHBBE7sbEKC43FpYxJfzg\\\\\\\/XExrKAtPUySjE28z3c77WkOy4DUSJkkxhvCpQHRXLkxDVgIBASJ40ZYD9rddT2wbrlTveLcqle8Uvmnq1N\\\\\\\/3GCFpuctB4CQaqaPD4PnQ5miUdeaGaml\\\u0022,\\\u0022iv\\\u0022:\\\u0022FLgHxOI9r\\\\\\\/yuPk2bk69+Lg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f5c4623b4560a2432211bc9a0ee3aa86a08f69a0253230c2bc3c3e3778920a71\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sB9npLRjtNAAXrwi86j3l2OS\\\\\\\/5FlJlUTfGU8fdCZvbMev9ciORidXh4bIow5QfdxPnZZ6dWQnRYLNAHPO+YL\\\\\\\/35AJBuFrXLGemfvtlDJSFrLJ71RxDTTTEjjFgwNrABLiZFk\\\\\\\/kDZaXoFkltPrT1MpMJUNGbsngvwqmgSFbkzt0I\\\\\\\/3l1+uEKDjKZYxlGabMuM14sMAw9vp7jGDd\\\\\\\/dFg64Nlem1sKYmCfOjBXK8Z7LGz73GYxIf+WeqKMCqeSjYZIm+CMCwfEXjnwjPf\\\\\\\/5Op84VXGb+UDkQUmJbyyzCuwJNYgkBZMcQ\\\\\\\/20n4zyA89fnpnEqo81rK3Ag5bDUzZk+Rja8+vITkQm02VsTZi7QV1Edl38EXVhpb9HMNgmouJvpjt6R2gcsY9tC2YRBTdtIL4hAiZlsjrovJvkD0lQYh6W+UrD+yhAICVjHrZITyVyVtrLzqgi2JVJ6sVDOs2fgG+vMDqg5NKF0hMFYlFNmt4j4KsqjTAnGGjdymDnpO\\\\\\\/7ADsXzRS\\\\\\\/SORmVW+4oEbEVOOMuDDr1nr3\\\\\\\/4s1Mtx5joRHM19GutPIiSjDdAhsdR9qKDDUmPmtu+dv62brzf88X6zvxIqnO5MQsbAW\\\\\\\/z0VCchmzzmzocGlPG\\\\\\\/j98ijfzQl8u4SRO4U7z0\\\\\\\/pKKwiqyKHUuzEQ03IorXv63saVM6i6oCQJmOoLvuKZ33arO3xrH1vfziXokLhgj50b4JJUrs2w1pgDhxTSvk+utbBhbfoWrCVqlryj6hH58\\\\\\\/UHsNQMrXWvwck\\\\\\\/4ng6ykCrVqL06z9MOUJrTYsrE2fJkdoarPvAGZbrmAo+xWg+mxHTz83y6umqILOmoi03JySrqw3+NuQCTTjncA2E5gVAGbR6MPdM+eLWDVen\\\\\\\/x21cr816UgydAX5c7Q7wBg1JBhxIw7kh1ggzGR8IfVTc+1V85EZYxm2prSNIgbAasbqxlbcRXFoKCIB0ivZEHsIE04KjXFPRJP5JAeq9t8yd2SatKK9vnooRUmqW8VuGN3MhWtao9TdV3Ua8ErnYoFIyMOxrS5CmmvFsKbUPeqjLm9u1WyhrKP2l\\\\\\\/+IZaGZMQL0VtdqkdHtEPXbYra97ID7SpDrgqycqNlXZExna76SivOjYO6HB3ZAdiZVmw1z3vv1GbwS4C5svpA7ujZ5HNpjQyjq4fRILpPmGla7Rn5nNFGhH+AWfWvI\\\\\\\/PBcU2Gnu5B\\\\\\\/CIbz2NkZuGC\\\\\\\/rMvYu5hyMzgn+pU+JSdNEYt1lSjvzlzRlhXbQJA\\\\\\\/2yvJQ0gYseNy9LoVQFeu7YkG8QTT4voXNS5J09aB4Q0qw5WF56ydHTThWP07G0cesNEwrU\\\\\\\/Nd6wLGRhOqK6evt19gzCrutdu9Fx8zWM3FCODlUX8+5zBSjsPjWk7mjo5XwlyDHUgWdwbKPrzrl83eEHIveOKL9IpbzTeq2Ivn4VwtJYMHDWNMn8jLLKoYBk7hbHPmxxFUgak2UkAM3ei+KVunBq9SrMzby5BM0IWoJFoKB9I4s86Auc5CObVcYujYjYwPPNSGNWZyY1uJadhwdHJP7lnlS5TC6daMbwc9jEqvxyYDvHyTwewd9uFpoq4YbkN8maeEPaRjN9ulCGsz2RalOq+rFKOHTghwNShB7OX0NrrfdGU0mONblS3Ol6nJi8mUQJyv5d2WDN3akYtmPWYvLvIgsFYFf99tGVdx9b50k4Tr1xaltU6VscDyrkkk9nDZkVGVfMMYb05kMEcFhf4zrU\\\\\\\/Ufw31lKddADfB6C04vm8YkP6yGXzvcIvZxKehbXK6btzRkuOtChAQP78pm4UHwh3ZePCvGabK++V0\\\\\\\/uSRMoWB861GCIGD10bP+V8SzFB+xRZ5VmQadTRKGnDjTqBwcolW+VFExXxq+V6WRtVUPfdFRO60YNpM1IMijRdokyMAzGSNxHN2y7IPkst9ndL\\\\\\\/FDIY6HTk3gkF5rukYWjwv\\\\\\\/pHxG+4Lz9WtOPN42Q6rvGz6ZF4FX+k1EEYadbVy4nzRe+\\\\\\\/8UCA3jyE2DSxlNyulKf7ZVyvY8z6zlbqfhmnlPtU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022XYWvO0MyW\\\\\\\/tY5aZbYHRvjA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e8926f11ad1835ad37c8abf82f334d5fb8016d84ff06a6185271b7466860efc0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459445 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/763" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-40JXZJejOt5shhOdF4dXIQqhsE_PEHmsHSZXi63w8Fw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/763" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uZIUBDMuE2Qce-19tkQZQpuqRMaPcXopMMlyxSOv1Bo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/757" title="Parents Without Partners: Drosophila as a Model for Understanding the Mechanisms and Evolution of Parthenogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/771" title="Physical and Genetic Interactions Between Uls1 and the Slx5–Slx8 SUMO-Targeted Ubiquitin Ligase" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459445" id="node459445" data-pisa="ggg;3/4/763" data-pisa-master="ggg;g3.113.005637" data-apath="/ggg/3/4/763.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexander G. Shanku</span>, <span class="highwire-citation-author" data-delta="1">Mark A. McPeek</span> and <span class="highwire-citation-author" data-delta="2">Andrew D. Kern</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">763-770;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459445" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexander G. Shanku </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BG.%2BShanku%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shanku%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BG.%2BShanku%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alexander.shanku@rutgers.edu" class="" data-icon-position="" data-hide-link-title="0">alexander.shanku@rutgers.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark A. McPeek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Dartmouth College, New Hampshire 03755</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BMcPeek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McPeek%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BA.%2BMcPeek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew D. Kern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BKern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kern%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BD.%2BKern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/4/763" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459445" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459445/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/763.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459445" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459445/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/763.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459445" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459445/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/763.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459445" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459445/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/4/763.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459445" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459445/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">In this paper we present a <em>de novo</em> assembly of the transcriptome of the damselfly (<em>Enallagma hageni</em>) through the use of 454 pyrosequencing. <em>E. hageni</em> is a member of the suborder Zygoptera, in the order Odonata, and Odonata organisms form the basal lineage of the winged insects (Pterygota). To date, sequence data used in phylogenetic analysis of <em>Enallagma</em> species have been derived from either mitochondrial DNA or ribosomal nuclear DNA. This <em>Enallagma</em> transcriptome contained 31,661 contigs that were assembled and translated into 14,813 individual open reading frames. Using these data, we constructed an extensive dataset of 634 orthologous nuclear protein-encoding genes across 11 species of Arthropoda and used Bayesian techniques to elucidate the position of <em>Enallagma</em> in the arthropod phylogenetic tree. Additionally, we demonstrated that the <em>Enallagma</em> transcriptome contains 169 genes that are evolving at rates that differ relative to those of the rest of the transcriptome (29 accelerated and 140 decreased), and, through multiple Gene Ontology searches and clustering methods, we present the first functional annotation of any palaeopteran’s transcriptome in the literature.</p></div><p id="p-5"><em>Enallagma</em> damselflies are aquatic invertebrates belonging to the order Odonata. Included in this group are dragonflies (suborder Anisoptera) and other damselflies (suborder Zygoptera), which together represent one of the most ancient branches of the winged insects (Pterygota) and furthermore represent a basal group within the division Palaeoptera (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Simon <em>et al.</em> 2009</a>). The damselfly has a rich history as an organism used in evolutionary and ecological studies, spanning research in speciation (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bourret <em>et al.</em> 2011</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Turgeon <em>et al.</em> 2005</a>), species distribution (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bourret <em>et al.</em> 2011</a>), selection (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Outomuro <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Abbott <em>et al.</em> 2008</a>), population diversity (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Iserbyt <em>et al.</em> 2010</a>), and predator-prey interactions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mittelbach <em>et al.</em> 2007</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Slos <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Strobbe <em>et al.</em> 2010</a>).</p><p id="p-6">Despite the fact that this organism is an ideal candidate for many types of biological studies, there has been relatively little examination of the genetic makeup of damselflies on a large scale (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bellin <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Surget-Groba and Montoya-Burgos 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nawy 2011</a>). For example, most of the sequence data used to determine phylogenetic relationships among <em>Enallagma</em> species, as well as to infer <em>Enallagma</em> phylogenetic relationships within Odonata, has been in the form of mtDNA (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Turgeon and McPeek 2002</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Saux <em>et al.</em> 2003</a>) or ribosomal nuclear DNA (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dumont and Vierstraete 2010</a>). Therefore, in this study, we attempted to investigate the nuclear, protein-encoding gene profile of the damselfly <em>Enallagma hageni</em> by using next-generation sequencing technology and, by doing so, (1) give further resolution and support to this organism’s phylogenetic position within Arthropoda, (2) determine the evolutionary rates of the protein-encoding genes in the <em>Enallagma</em> transcriptome, and (3) give functional annotation to the proteins expressed in our dataset.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Insect capture and RNA preparation</h3><p id="p-7">Individuals across the entire life cycle were included in the sample from which RNA was extracted. Some <em>Enallagma</em> larvae are difficult to identify as to species, with <em>E. hageni</em> being one of these. <em>E. hageni</em> larvae are largely indistinguishable from those of four other species that are all derived from a very recent radiation (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Turgeon <em>et al.</em> 2005</a>). To ensure that we were unambiguously collecting <em>E. hageni</em> larvae, we collected larvae from Martin’s Pond, Green Bay, VT, a lake where we have found only <em>E. hageni</em> and none of the other species as adults in previous years (M. A. McPeek, personal communication). Embryos were obtained by allowing females to oviposit in the laboratory and then allowing 2 weeks for development prior to RNA extraction. Aquatic larvae from across the entire range of the larval period and adults were collected and immediately placed in RNAlater (Ambion Inc.) until RNA isolation. Total RNA was isolated from the pooled material of roughly 50 embryos, 150 larvae, and 25 adults by first flash freezing the insects in liquid nitrogen and then processing the frozen material using RNeasy protocols (Qiagen). From our isolations, we collected roughly 100 mg of total RNA.</p></div><div id="sec-3" class="subsection"><h3>Transcriptome sequencing and assembly</h3><p id="p-8">mRNA isolation, library construction, and 454 sequencing were contracted out to Beckman Coulter Genomics, using 1 mg of total RNA as starting material. All sequencing was of un-normalized cDNA libraries, using standard 454 protocols with the 454GS instrument. This produced 976,767 reads (see <em>Results</em> for details of the sequencing output).</p><p id="p-9">To perform <em>de novo</em> transcriptome assembly with our reads, we used the Newbler assembler (version 2.3) using parameter settings specifically for mRNA assembly (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/005637SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2013/03/06/g3.113.005637.DC1/TableS5.txt">Table S5</a>).</p></div><div id="sec-4" class="subsection"><h3>Protein translation</h3><p id="p-10">To compile a dataset of proteins which would form the basis of our analysis, assembled contigs were translated using Virtual Ribosome (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wernersson 2006</a>). Each of 6 open reading frames (ORFs) was translated (where –readingframe = all), and the longest resulting translated read was kept, provided it was initiated with a start codon (where –orf = any). To account for contigs that might have had their upstream start codon truncated during assembly, we again translated more than 6 ORFs, all contigs that did not posses a start codon but terminated with a stop codon (where –orf = none). Of these two sets of putative proteins, the longest read that possessed both a start and a stop codon was determined to be the translated protein for a given contig, unless it was a fragment not initiated by a start codon but terminated with a stop codon, was greater in length. Contigs composed of fewer than 10 nucleotides were excluded from translation and removed from further analysis.</p></div><div id="sec-5" class="subsection"><h3>Arthropod proteins</h3><p id="p-11">Comparative analysis of phylogenetic relationships necessitates alignment of homologous sequences among individuals being compared. To compile the data for such an analysis, we began by conducting a search aimed at identifying orthology across expressed proteins in a group of selected arthropods. To build this set of putative orthologous proteins, we obtained transcriptome data from ten arthropod species housed in public databases (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462620/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462620/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Arthropod Species Used in Phylogenetic Analysis</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Ortholog Detection</h3><p id="p-13">To construct a working set of orthologous proteins, we used the method of one-to-one reciprocal best BLAST hits (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Moreno-Hagelsieb and Latimer 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gabaldón 2008</a>), rather than attempt to use ortholog clustering methods (<em>e.g.</em>, OrthoMCL) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2003</a>). We used BLAST to search for protein-encoding genes between each species’ transcriptome and those in <em>D. melanogaster</em>, and conversely, the <em>D. melanogaster</em> transcriptome was also searched using BLAST for all protein-encoding genes present in each of the species’ transcriptomes in the dataset. The best hit was determined using the “-K 1” and “-b 1” BLAST parameters, which limit output, in this case the “-m 8” tabulated output format, to the best scoring hit of each BLAST query. Following this methodology and using mpiBLAST, an open-source, parallelized version of BLAST (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Darling and Carey 2003</a>), we constructed a set of reciprocal-best, one-to-one orthologs. To expedite computational processing time, each species’ database file was partitioned into 94 fragments (where <em>n</em>frags = 94), and the parameter setting “–output-search-stats–use-parallel-write–use-virtual-frags–removedb” was used for each job. Using customized scripts, individual orthologs that were present across all 11 arthropod species were grouped together into individual .fasta files. Following this search and grouping method, the protein sequences within each file were aligned using ClustalW2 software using the flags “-OUTPUT=FASTA” and “-OUTORDER=INPUT,” the latter being necessary to later allow for concatenation of all aligned orthologs when conducting phylogenetic analysis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Larkin <em>et al.</em> 2007</a>).</p></div><div id="sec-7" class="subsection"><h3>Phylogenetics</h3><p id="p-14">Each orthologous gene alignment was concatenated into a “super-gene” (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gadagkar <em>et al.</em> 2005</a>), that is, we took individual .fasta files and joined them into one singular, interleaved .nexus file by using a customized Ruby script. If an amino acid position in the concatenated alignment contained a gap at a position in any of the species, or in multiple species, that position was removed prior to analysis by using Gblocks 0.91b (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Talavera and Castresana 2007</a>), as we did not use a model of sequence evolution that allowed for insertions or deletions.</p></div><div id="sec-8" class="subsection"><h3>Model selection</h3><p id="p-15">To determine the optimal model of protein evolution for phylogenetic analysis of our dataset, ProtTest version 2.4 software was used for model selection (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Darriba <em>et al.</em> 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abascal <em>et al.</em> 2005</a>). All amino acid evolutionary rate models available in ProtTest were examined, as were the parameters “+I,” “+G,” and “+F”. (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dayhoff <em>et al.</em> 1978</a>), JTT (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jones <em>et al.</em> 1992</a>), WAG (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Whelan and Goldman 2001</a>), mtREV (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Adachi and Hasegawa 1996</a>), MtMam (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cao <em>et al.</em> 1994</a>), VT (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Müller and Vingron 2000</a>), CpREV (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Adachi <em>et al.</em> 2000</a>), RtREV (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dimmic <em>et al.</em> 2002</a>), MtArt (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Abascal <em>et al.</em> 2007</a>), HIVb/HIVw (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nickle <em>et al.</em> 2007</a>), LG (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Le and Gascuel 2008</a>), and Blosum62 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Henikoff 1992</a>).</p><p id="p-16">Ideally, we would optimize tree topology, branch lengths, and parameters of the model for each model investigated. This was inefficient in our case, as the dataset is too large to realistically attempt topology optimization for each model and each additional model parameter associated with that model. Instead, we allowed a neighbor-joining tree to be constructed with our data, and fix the topology and from that topology, optimize branch lengths and select model parameters (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Posada and Crandall 2001</a>).</p></div><div id="sec-9" class="subsection"><h3>Bayesian phylogenetic inference</h3><p id="p-17">Once the optimum model was selected, we searched topology space of the 11 arthropod species in our dataset with a Bayesian Markov chain Monte Carlo (MCMC) approach using MrBayes version 3.1.2 software (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ronquist and Huelsenbeck 2003</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huelsenbeck <em>et al.</em> 2001</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Altekar <em>et al.</em> 2004</a>).</p><p id="p-18">The following settings were used in our MCMC analysis: two runs, 750,000 generations; number of chains = 240; sample frequency = 250; 240 processors were used in parallel. The evolutionary model used was the WAG model that allows for 20 states. Rates were set to “Invgamma,” with the gamma shape parameter uniformly distributed on the interval (0.00, 200.00). The proportion of invariable sites was also uniformly distributed on the interval (0.00, 1.00). All topologies were equally probable, and branch lengths were unconstrained.</p></div><div id="sec-10" class="subsection"><h3>Rate testing</h3><p id="p-19">To address the question of whether certain orthologous protein-encoding genes present in <em>Enallagma</em> were evolving at different rates relative to those of other arthropods, branch length rate tests were conducted with each <em>Enallagma</em> gene in our dataset. Using PAML (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yang 2007</a>), we generated two models for each protein, one that assumed a global clock across all species and the other that fixed the rate of evolution of each <em>Enallagma</em> protein to a local clock while keeping the rest of the species evolutionary rates confined to a global clock. In this manner, we generate two likelihood estimates (one for each model) for these proposed modes of evolution of a particular protein. To that extent, a likelihood ratio test was performed between the null model (global clock) and alternative model (local clock).<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/763/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/763/embed/mml-math-1.gif"/></noscript></span></span>Where <em>D</em> is the test statistic, <em>lnL</em><em><sub>G</sub></em> is the log likelihood of the global clock model, and <em>lnL</em><em><sub>L</sub></em> is the log likelihood of the local clock model. The probability distribution of the test statistic, <em>D</em>, can be approximated by the chi-squared distribution, where the degree of freedom of the distribution is equal to the number of free parameters of the global model minus the number of free parameters of the local model, which for our purposes, will be 1. (Parameters of the local model = 11 parameters of global model = 10.) Once a raw probability for each likelihood ratio was calculated, we performed Bonferroni corrections to determine significance.</p></div><div id="sec-11" class="subsection"><h3>GO annotation</h3><p id="p-20">The complete set of all <em>Enallagma hageni</em> proteins was queried against a local NCBI “nonredundant” (nr) protein database (obtained October 14, 2011) using mpiBLAST. The output was saved in .xls format (-m 7–output-search-stats), which was then analyzed using Blast2GO without graphical interface (B2G4PIPE) and a local B2G database (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Conesa <em>et al.</em> 2005</a>).</p><p id="p-21">We examined GO term distributions for three partitions of our dataset. First, we derived the distributions of 3<sup>rd</sup>- and 4<sup>th</sup>-level GO term hierarchies for the complete dataset of <em>Enallagma</em> proteins. The hierarchical system of gene ontology is represented as a directed acyclic graph in which parent-child relationships describe specific GO terms. That is, parent terms are less specific in their description of a biological function than are their respective child terms. This leads to “levels” within the Gene Ontology structure, with the 1<sup>st</sup> level containing the broadest categories: biological processes, cellular components, and molecular function. An individual gene may then have many parents and many levels of categorization before reaching the 1<sup>st</sup> level (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yon Rhee <em>et al.</em> 2008</a>). Second, using <em>Drosophila melanogaster</em> as a background dataset, we determined those <em>Enallagma</em> genes that were enriched by a hypergeometric distribution test and corrected for multiple tests with false discovery rate (FDR) under dependency (Groppe 2012; Benjamini and Yekutieli 2001). Finally, we evaluated those <em>Enallagma</em> genes that were shown to have undergone either accelerated or reduced rates of evolution, per the branch length rate tests. These genes were examined for their overall GO 3<sup>rd</sup>- and 4<sup>th</sup>-level profiles and analyzed to determine if any gene was enriched. Enrichment was determined by setting all <em>Enallagma</em> genes as a background and using the hypergeometric test with FDR correction mentioned above.</p><p id="p-22">We constructed a hash for each of the 3 partitions, using the annotations from the Blast2GO pipeline. Each gene and that gene’s associated GO accession terms made up the key:value relationship, which was then imported into the WeGO web-based program in order to sort the data by GO term hierarchy (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ye <em>et al.</em> 2006</a>).</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Transcriptome assembly</h3><p id="p-23">After assembly, we obtained 31,662 contigs made up of 13,191,394 nucleotides. Of these contigs, 1656 were singletons (5.23%). Median coverage was 25 reads/contig (mean = 179.71 reads/contig; SD = 746.27), and median contig length was 355 bases (mean = 416.6; SD = 429.7). With singletons excluded, the dataset was reduced to 29,996 contigs. Of these, median coverage was 26 reads/contig (mean coverage = 173.73 reads/contig; SD = 677.99), and median contig length was 406 bases/contig (mean contig length = 439.7; SD = 429.9). The largest contig in the dataset was composed of 3036 nucleotides. The assembled transcriptome contained an AT bias at 59.86% and GC at 40.13%, and 0.01% was labeled “N.” CpG sites occurred at 2.69% of the transcriptome. (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/content/suppl/2013/03/06/g3.113.005637.DC1/TableS5.txt">Table S5</a> for assembly details).</p></div><div id="sec-14" class="subsection"><h3>Translated proteins</h3><p id="p-24">Translation of the <em>Enallagma</em> contigs yielded 14,813 individual open reading frames consisting of 1,621,208 amino acids (singletons not included). Mean length was 109 amino acids. Shortest and longest protein sequences were composed of 19 amino acids and 735 amino acids, respectively (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-15" class="subsection"><h3>Orthologs</h3><p id="p-25">The one-to-one, reciprocal best method of elucidating orthologous proteins generated 634 orthologs across the 11 species in the study. The <em>Enallagma</em> orthologs themselves contained 108,866 amino acids with a mean length of 171 amino acids, and shortest and longest sequence length of 46 amino acids and 413 amino acids, respectively (see <a href="http://www.g3journal.org/content/suppl/2013/03/06/g3.113.005637.DC1/TableS3.csv">Table S3</a> for ortholog groups.)</p></div><div id="sec-16" class="subsection"><h3>GO annotation</h3><p id="p-26">Our annotation methodology mapped 3998 <em>Enallagma</em> genes to at least one GO term, using BLAST2GO and the NCBI “nr” database. There were 24,439 total GO terms mapped to those 3998 genes, with 3812 of the GO terms being unique. The mean mapping was 6.1 GO terms/gene with a minimum and maximum mapping of 1 and 78 GO terms per gene, respectively. Using 3<sup>rd</sup>- and 4<sup>th</sup>-level GO term distributions, we mapped our dataset to 404 GO terms across 3 ontologies for 3<sup>rd</sup>-level terms (cellular component, biological process and molecular function) and 1463 terms across 3 ontologies for 4<sup>th</sup>-level terms. (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>). At the 3<sup>rd</sup> level of the hierarchy, the top GO terms represented were (1) biological processes: 58.7% of the genes were mapped to “primary metabolic processes” (GO:0044238), 53.5% of genes to “cellular metabolic processes” (GO:0044237), and 41.9% to “macromolecule metabolic processes” (GO:0043170); (2) cellular components: 43.4% to “intracellular organelles” (GO:0043229), 33.3% to “membrane-bound organelles” (GO:0043227), and 27.3% to “organelle parts” (GO:0044422); and (3) molecular function: 25.3% to “hydrolase activity” (GO:0016787), 19.3% to “ion binding” (GO:0043167), and 17.2% to “nucleotide binding” (GO:0000166). See <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> for 3<sup>rd</sup>-level distribution. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS4.pdf">Figure S4</a> for 4<sup>th</sup>-level distributions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/763/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="3rd-level GO term distributions for all annotated Enallagma genes. GO term distributions were plotted for each of the three 1st-level categories. The full dataset mapped to 404 unique GO terms at the 3rd level. Shown are the top 25 terms in each of the broadest, 1st-level categories: (A) Biological Process, (B) Cellular Component, and (C) Molecular Function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124249825" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;3&lt;sup&gt;rd&lt;/sup&gt;-level GO term distributions for all annotated &lt;em&gt;Enallagma&lt;/em&gt; genes. GO term distributions were plotted for each of the three 1&lt;sup&gt;st&lt;/sup&gt;-level categories. The full dataset mapped to 404 unique GO terms at the 3&lt;sup&gt;rd&lt;/sup&gt; level. Shown are the top 25 terms in each of the broadest, 1&lt;sup&gt;st&lt;/sup&gt;-level categories: (A) Biological Process, (B) Cellular Component, and (C) Molecular Function.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/763/F1.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/763/F1.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/763/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/763/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-27" class="first-child">3<sup>rd</sup>-level GO term distributions for all annotated <em>Enallagma</em> genes. GO term distributions were plotted for each of the three 1<sup>st</sup>-level categories. The full dataset mapped to 404 unique GO terms at the 3<sup>rd</sup> level. Shown are the top 25 terms in each of the broadest, 1<sup>st</sup>-level categories: (A) Biological Process, (B) Cellular Component, and (C) Molecular Function.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">To look for enriched or diminished GO terms, we then compared the <em>Enallagma</em> GO annotations to <em>Drosophila melanogaster</em> GO annotations. We queried 3986 annotated <em>Enallagma</em> genes against 13,127 annotated <em>Drosophila</em> genes and found that 1080 unique (1089 total) <em>Enallagma</em> GO terms were enriched or diminished. Described in terms of the GO hierarchy, we discovered 33 2<sup>nd</sup>-level GO terms and 161 3<sup>rd</sup>-level GO terms.</p><p id="p-29">Some of these enriched 3<sup>rd</sup>-level GO annotations included hydrolase activity (GO:16787), ion and nucleotide binding (GO:43167 and GO:0000166), and primary metabolic processes (GO:44238). Examples of diminished GO terms included anatomical structural development (GO:48856) and protein-DNA complex (GO:32993).</p><p id="p-30">Additionally, we mapped 488 genes within the orthologous protein-encoding set to 1669 GO IDs, 691 of these GO IDs being unique (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>); for the gene ID, GO ID, and gene product/function see <a href="http://www.g3journal.org/content/suppl/2013/03/06/g3.113.005637.DC1/TableS4.txt">Table S4</a>.)</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/763/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Enrichment or reduction of Enallagma GO terms relative to annotated Drosophila melanogaster genes. Using D. melanogaster as a background set, hypergeometric distribution tests were performed to identify Enallagma genes that were enriched or diminished. The background set consisted of 13,127 D. melanogaster annotated genes and was queried by 3986 Enallagma genes. We discovered 1080 unique enriched or diminished terms. (A) Biological Process, (B) Cellular Component, and (C) Molecular Function are the top 25 most significant results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124249825" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Enrichment or reduction of &lt;em&gt;Enallagma&lt;/em&gt; GO terms relative to annotated &lt;em&gt;Drosophila melanogaster&lt;/em&gt; genes. Using &lt;em&gt;D. melanogaster&lt;/em&gt; as a background set, hypergeometric distribution tests were performed to identify &lt;em&gt;Enallagma&lt;/em&gt; genes that were enriched or diminished. The background set consisted of 13,127 &lt;em&gt;D. melanogaster&lt;/em&gt; annotated genes and was queried by 3986 &lt;em&gt;Enallagma&lt;/em&gt; genes. We discovered 1080 unique enriched or diminished terms. (A) Biological Process, (B) Cellular Component, and (C) Molecular Function are the top 25 most significant results.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/763/F2.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/763/F2.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/763/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/763/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-31" class="first-child">Enrichment or reduction of <em>Enallagma</em> GO terms relative to annotated <em>Drosophila melanogaster</em> genes. Using <em>D. melanogaster</em> as a background set, hypergeometric distribution tests were performed to identify <em>Enallagma</em> genes that were enriched or diminished. The background set consisted of 13,127 <em>D. melanogaster</em> annotated genes and was queried by 3986 <em>Enallagma</em> genes. We discovered 1080 unique enriched or diminished terms. (A) Biological Process, (B) Cellular Component, and (C) Molecular Function are the top 25 most significant results.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Phylogenetics</h3><p id="p-32">After concatenating the 634 orthologous genes, the resulting multi-way alignment contained 182,478 amino acid positions. This alignment was then filtered with Gblocks, using the default parameters that do not allow for gaps at any position in the matrix, resulting in an ungapped alignment of 27,594 amino acid positions (15.1% of the original data). This ungapped matrix was then analyzed using MrBayes software with settings described in <em>Material and Methods</em>.</p><p id="p-33">We removed 50 samples of burn-in after each MCMC run, therefore sampling from the posterior 2952 times for each of the two runs. Each of the two MCMC analyses took 224,340 seconds (62.3 hours) and 227,756 seconds (63.3 hours) to complete, respectively. The plotted phylogram, based on the consensus tree data of the MCMC runs, is shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/763/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Arthropod phylogram. Eleven taxa and 27,594 amino acid positions were used in the analysis. Branch lengths are labeled, and posterior probabilities at each branching node are 1.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-124249825" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Arthropod phylogram. Eleven taxa and 27,594 amino acid positions were used in the analysis. Branch lengths are labeled, and posterior probabilities at each branching node are 1.0.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/763/F3.medium.gif" width="440" height="360"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/763/F3.medium.gif" width="440" height="360"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/763/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/763/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462655" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-34" class="first-child">Arthropod phylogram. Eleven taxa and 27,594 amino acid positions were used in the analysis. Branch lengths are labeled, and posterior probabilities at each branching node are 1.0.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35"><em>Ixodes scapularis</em> (class Arachnida) was chosen as the out group, and the tree was rooted upon it. The posterior probability for each node in the tree was 1.0.</p><p id="p-36">Trace plots of the MCMC analysis and Gelman convergence plots are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS5.pdf">Figure S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS6.pdf">Figure S6</a>.</p></div><div id="sec-18" class="subsection"><h3>Rate testing</h3><p id="p-37">The branch length test indicated that 439 of the 634 (69.2%) orthologs fit a local clock model better and were therefore deduced to be evolving at a rate that varied relative to that gene’s orthologs (raw <em>P</em> &lt; 0.05). However, a Bonferroni correction for multiple tests, (<em>P</em> &lt; 0.05/634 = 0.0000788) reduced that set and yielded 169 genes which were shown to be evolving at significantly different rates in <em>Enallagma</em>. Of these 169 genes, 29 genes were shown to be evolving at an accelerated rate, while the remaining 140 genes were determined to be evolving at a reduced rate. We successfully mapped 37 of these genes to at least one GO term. In the accelerated case, 4 of the 29 genes were mapped to 17 GO terms, while in the decreased case, 33 of the 140 genes mapped to 105 GO terms. Of those 37 genes that we were able to annotate, no significant enrichment was noted by using the hypergeometric test (<em>P</em> &lt; 0.05), relative to the background set of all <em>Enallagma</em> GO annotations. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/TableS1.pdf">Table S1</a> shows the 4 accelerated genes and their gene products. These include <em>Nol10</em> (nucleolar protein), <em>Art7</em> (protein arginine N-methyltransferase), <em>Rrp45</em> (RNA processing), and <em>Uba3</em> (ubiquitin-like protein). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS7.pdf">Figure S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/FigureS8.pdf">Figure S8</a> show 3<sup>rd</sup>- and 4<sup>th</sup>-level distributions of the decreased rate genes (see also <a href="http://www.g3journal.org/content/suppl/2013/03/06/g3.113.005637.DC1/TableS2.txt">Table S2</a>.)</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-38">At the level of resolution we used to examine (other species within Arthropoda which had assembled transcriptomes), our phylogenetic analysis of <em>Enallagma</em> and the compared arthropods appears congruent to that of other current studies and reviews (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meusemann <em>et al.</em> 2010</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ishiwata <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Trautwein <em>et al.</em> 2012</a>).</p><p id="p-39">Our hypergeometric tests of the accelerated and decreased rates of proteins’ GO annotations, relative to the background set of all genes we were able to annotate, indicated no significant enrichments (<em>P</em> &lt; 0.05 raw, FDR corrections). Nevertheless, the GO term distributions of the altered rate genes were shown to similarly represent the distributions of the overall dataset. For example, the top three GO terms represented by both the biological processes and cellular component 3<sup>rd</sup>-level domains were the same. In the case of biological processes, we saw the terms “primary metabolic process,” “cellular metabolic process,” and “macromolecule metabolic process” encompassing the top three positions, while the top three terms in the domain of cellular component were “intracellular organelle,” “membrane-bounded organelle,” and “intracellular organelle part.” However, there were some deviations from that, especially in the molecular function domain. For example, the top two GO terms represented in the decelerated genes category, in the “Molecular Function” domain, were shown to be “nucleotide binding” and “nucleic acid binding,” whereas in the full set, the top two expressed GO terms for that same domain were “hydrolase activity” and “ion binding”.</p><p id="p-40">One of the interesting ecological and evolutionary scenarios involving <em>Enallagma</em> is that various <em>Enallagma</em> lineages have adapted to living with predators by increasing their burst swimming speeds to increase their probability of escape during predator attacks (McPeek <em>et al.</em> 1996; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McPeek 1999</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McPeek 2000</a>). In agreement with this, we annotated genes involved in muscle mass increase and differentiation (GO:0003012) and genes with roles in arginine kinase (GO:0004054) and arginine methylation [accelerated (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1/TableS1.pdf">Table S1</a>); GO:0019918], which has been shown to partially responsible for the observed rapid movements of the damselflies (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">McPeek 1999</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">McPeek 2000</a>).</p><p id="p-41">Another issue worth noting is that analysis by short read sequencing in transcriptome assembly relies on the use of reads typically 35–250 bp in length (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mardis 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Harismendy <em>et al.</em> 2009</a>). Our annotation methodology mapped 3998 <em>Enallagma</em> genes to at least one associated GO term. While this number represents less than 30% of the genes in our dataset associating with a GO term, it should be noted that small contigs, like those generated in 454 sequencing, can be difficult to successfully map to GO terms and that mapping success increases successively with read size. (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Novaes <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Meyer <em>et al.</em> 2009</a>).</p><p id="p-42">In summary, we have generated a draft functional annotation of nearly 4000 genes in the transcriptome of <em>Enallagma hageni</em>, which to our knowledge is the first examined and annotated transcriptome of any palaeopteran in the literature. We examined the rate at which <em>E. hageni</em> proteins are evolving and found 169 genes which better fit the hypothesis of having an altered evolutionary history, relative to other genes in its transcriptome. We examined the distributions of GO terms for each of three classes of our data: the whole annotated transcriptome, the transcriptome with <em>D. melanogaster</em> as a background, and the set of altered genes with all <em>Enallagma</em> genes as a background. Of those, we additionally deduced which annotations are enriched or diminished through the use of hypergeometric distribution testing. Finally, we have produced a strongly supported phylogenetic analysis that in turn further strengthens support for the position of Odonata in the Arthropoda tree.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">The authors thank the editor and two anonymous reviewers for their valuable comments and advice. This study was supported by National Science Foundation (NSF) grant MCB 1161367 (A.D.K.) and NSF grant DEB-0714782 (M.A.M.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005637/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence and transcriptome data from this article have been deposited in the National Center for Biotechnology Information (NCBI) database under <em>Enallagma hageni</em> BioProject no. PRJNA185185 ID:185185, which contains links and access to insect sampling data under BioSample SAMN01881995 and raw sequencing under Sequence Read Archive (SRA) SRR649536 and transcriptome SUB156504 data.</p></li><li class="fn" id="fn-4"><p id="p-44"><em>Communicating editor: I. M. Hall</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-01-14"><span class="received-label">Received </span>January 14, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-02-25"><span class="accepted-label">Accepted </span>February 25, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Shanku <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.1" data-doi="10.1093/bioinformatics/bti263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abascal</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zardoya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">ProtTest: selection of best-fit models of protein evolution</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2104</span>–<span class="cit-lpage">2105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti263%26rft_id%253Dinfo%253Apmid%252F15647292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzIxMDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.2" data-doi="10.1093/molbev/msl136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abascal</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zardoya</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MtArt: A new model of amino acid replacement for arthropoda</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl136%26rft_id%253Dinfo%253Apmid%252F17043087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjY6IjI0LzEvMSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.3" data-doi="10.1111/j.1365-294X.2007.03641.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gosden</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svesson</span>  <span class="cit-name-given-names">E. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Patterns of differentiation in a colour polymorphism and in neutral markers reveal rapid genetic changes in natural damselfly populations</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1597</span>–<span class="cit-lpage">1604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DAbbott%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D1597%26rft.epage%253D1604%26rft.atitle%253DPatterns%2Bof%2Bdifferentiation%2Bin%2Ba%2Bcolour%2Bpolymorphism%2Band%2Bin%2Bneutral%2Bmarkers%2Breveal%2Brapid%2Bgenetic%2Bchanges%2Bin%2Bnatural%2Bdamselfly%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03641.x%26rft_id%253Dinfo%253Apmid%252F18284565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03641.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18284565&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254703000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.4" data-doi="10.1007/BF02498640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adachi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Model of amino acid substitution in proteins encoded by mitochondrial DNA</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">459</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DAdachi%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D459%26rft.epage%253D468%26rft.atitle%253DModel%2Bof%2Bamino%2Bacid%2Bsubstitution%2Bin%2Bproteins%2Bencoded%2Bby%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02498640%26rft_id%253Dinfo%253Apmid%252F8642615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02498640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8642615&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UK38000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adachi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waddell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Plastid genome phylogeny and a model of amino acid substitution for proteins encoded by chloroplast DNA</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DAdachi%26rft.auinit1%253DJ.%26rft.volume%253D50%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D358%26rft.atitle%253DPlastid%2Bgenome%2Bphylogeny%2Band%2Ba%2Bmodel%2Bof%2Bamino%2Bacid%2Bsubstitution%2Bfor%2Bproteins%2Bencoded%2Bby%2Bchloroplast%2BDNA.%26rft_id%253Dinfo%253Apmid%252F10795826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10795826&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086438700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.6" data-doi="10.1093/bioinformatics/btg427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altekar</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dwarkadas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Parallel metropolis coupled Markov chain Monte Carlo for Bayesian phylogenetic inference</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg427%26rft_id%253Dinfo%253Apmid%252F14960467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMC8zLzQwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.7" data-doi="10.1186/1471-2164-10-555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrarini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chimento</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levenkova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Combining next-generation pyrosequencing with microarray for large scale expression analysis in non-model species</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBellin%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.spage%253D555%26rft.epage%253D555%26rft.atitle%253DCombining%2Bnext-generation%2Bpyrosequencing%2Bwith%2Bmicroarray%2Bfor%2Blarge%2Bscale%2Bexpression%2Banalysis%2Bin%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-555%26rft_id%253Dinfo%253Apmid%252F19930683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930683&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourret</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turgeon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Regional divergence and mosaic spatial distribution of two closely related damselfly species (<em>Enallagma hageni</em> and <em>Enallagma ebrium</em>)</span>. <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">196</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D25%26rft.spage%253D196%26rft_id%253Dinfo%253Apmid%252F22122075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22122075&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adachi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paabo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Phylogenetic-relationships among Eutherian orders estimated from inferred sequences of mitochondrial proteins---instability of a tree-based on a single-gene</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCao%26rft.auinit1%253DY.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D519%26rft.epage%253D527%26rft.atitle%253DPhylogenetic%2Brelationships%2Bamong%2Beutherian%2Borders%2Bestimated%2Bfrom%2Binferred%2Bsequences%2Bof%2Bmitochondrial%2Bproteins%253A%2Binstability%2Bof%2Ba%2Btree%2Bbased%2Bon%2Ba%2Bsingle%2Bgene.%26rft_id%253Dinfo%253Apmid%252F7807540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7807540&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PP70700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.10" data-doi="10.1093/bioinformatics/bti610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Götz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talón</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Blast2GO: a Universal tool for annotation, visualization and analysis in functional genomics research</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3674</span>–<span class="cit-lpage">3676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti610%26rft_id%253Dinfo%253Apmid%252F16081474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTgvMzY3NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-3.4.763.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darling</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carey</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The design, implementation, and evaluation of mpiBLAST. 4th International Conference on Linux Clusters: The HPC Revolution 2003 in conjunction with ClusterWorld Conference &amp; Expo, San Jose, CA.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.12" data-doi="10.1093/bioinformatics/btr088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ProtTest 3: fast selection of best-fit models of protein evolution</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1164</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr088%26rft_id%253Dinfo%253Apmid%252F21335321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNy84LzExNjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-3.4.763.13"><div class="cit-metadata unstructured">Dayhoff, M. O., R. M. Schwartz, and B. C. Orcutt, 1978 A model of evolutionary change in proteins. In <em>Atlas of Protein Sequence and Structure</em>, vol. 5, suppl. 3, edited by M.O. Dayhoff. National Biomedical Research Foundation, Washington, DC.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.14" data-doi="10.1007/s00239-001-2304-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dimmic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rest</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mindell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">rtREV: An amino acid substitution matrix for inference of retrovirus and reverse transcriptase phylogeny</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">65</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DDimmic%26rft.auinit1%253DM.%2BW.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D73%26rft.atitle%253DrtREV%253A%2Ban%2Bamino%2Bacid%2Bsubstitution%2Bmatrix%2Bfor%2Binference%2Bof%2Bretrovirus%2Band%2Breverse%2Btranscriptase%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-001-2304-y%26rft_id%253Dinfo%253Apmid%252F12165843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-001-2304-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12165843&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176243200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.15" data-doi="10.1111/j.1365-3113.2009.00489.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vierstraete</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanfleteren</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A molecular phylogeny of the <em>Odonata</em> (Insecta)</span>. <abbr class="cit-jnl-abbrev">Syst. Entomol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BEntomol.%26rft.volume%253D35%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3113.2009.00489.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3113.2009.00489.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.16" data-doi="10.1186/gb-2008-9-10-235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gabaldón</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Large-scale assignment of orthology: back to phylogenetics?</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGabaldon%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.issue%253D10%26rft.spage%253D235%26rft.epage%253D235%26rft.atitle%253DLarge-scale%2Bassignment%2Bof%2Borthology%253A%2Bback%2Bto%2Bphylogenetics%253F%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-10-235%26rft_id%253Dinfo%253Apmid%252F18983710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-10-235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983710&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gadagkar</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Inferring species phylogenies from multiple genes: concatenated sequence tree <em>vs.</em> consensus gene tree</span>. <abbr class="cit-jnl-abbrev">J Exp. Zool. B. Mol. Dev. Evol.</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp.%2BZool.%2BB.%2BMol.%2BDev.%2BEvol.%26rft.volume%253D304%26rft.spage%253D64%26rft_id%253Dinfo%253Apmid%252F15593277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15593277&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.18" data-doi="10.1186/gb-2009-10-3-r32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harismendy</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strausberg</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stockwell</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evaluation of next generation sequencing platforms for population targeted sequencing studies</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHarismendy%26rft.auinit1%253DO.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR32%26rft.epage%253DR32%26rft.atitle%253DEvaluation%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%2Bfor%2Bpopulation%2Btargeted%2Bsequencing%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r32%26rft_id%253Dinfo%253Apmid%252F19327155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19327155&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.19" data-doi="10.1073/pnas.89.22.10915"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Amino acid substitution matrices from protein blocks</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U S A</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">10915</span>–<span class="cit-lpage">10919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHenikoff%26rft.auinit1%253DS.%26rft.volume%253D89%26rft.issue%253D22%26rft.spage%253D10915%26rft.epage%253D10919%26rft.atitle%253DAmino%2Bacid%2Bsubstitution%2Bmatrices%2Bfrom%2Bprotein%2Bblocks.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.22.10915%26rft_id%253Dinfo%253Apmid%252F1438297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODkvMjIvMTA5MTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.20" data-doi="10.1126/science.1065889"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Bayesian Inference of Phylogeny and Its Impact on Evolutionary Biology</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>(<span class="cit-issue">5550</span>): <span class="cit-fpage">2310</span>–<span class="cit-lpage">2314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHuelsenbeck%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2310%26rft.epage%253D2314%26rft.atitle%253DBayesian%2BInference%2Bof%2BPhylogeny%2Band%2BIts%2BImpact%2Bon%2BEvolutionary%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065889%26rft_id%253Dinfo%253Apmid%252F11743192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.21" data-doi="10.1111/j.1469-7998.2010.00735.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iserbyt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bots</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Gossum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordaens</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Did historical events shape current geographic variation in morph frequencies of a polymorphic damselfly?</span> <abbr class="cit-jnl-abbrev">J. Zool. (Lond.)</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BZool.%2B%2528Lond.%2529%26rft.volume%253D282%26rft.spage%253D256%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-7998.2010.00735.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-7998.2010.00735.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.22" data-doi="10.1016/j.ympev.2010.11.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishiwata</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Z.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phylogenetic relationships among insect orders based on three nuclear protein-coding gene sequences</span>. <abbr class="cit-jnl-abbrev">Mol. Phyl. Evol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhyl.%2BEvol.%26rft.volume%253D58%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2010.11.001%26rft_id%253Dinfo%253Apmid%252F21075208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2010.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21075208&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.23" data-doi="10.1093/bioinformatics/8.3.275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The rapid generation of mutation data matrices from protein sequences</span>. <abbr class="cit-jnl-abbrev">Comput. Appl. Biosci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DJones%26rft.auinit1%253DD.%2BT.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D275%26rft.epage%253D282%26rft.atitle%253DThe%2Brapid%2Bgeneration%2Bof%2Bmutation%2Bdata%2Bmatrices%2Bfrom%2Bprotein%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F8.3.275%26rft_id%253Dinfo%253Apmid%252F1633570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo3OiI4LzMvMjc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.24" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMjEvMjk0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.25" data-doi="10.1093/molbev/msn067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An improved general amino acid replacement matrix</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn067%26rft_id%253Dinfo%253Apmid%252F18367465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMTMwNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.26" data-doi="10.1101/gr.1224503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoeckert</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D2178%26rft.epage%253D2189%26rft.atitle%253DOrthoMCL%253A%2BIdentification%2Bof%2BOrtholog%2BGroups%2Bfor%2BEukaryotic%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1224503%26rft_id%253Dinfo%253Apmid%252F12952885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMjE3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.27" data-doi="10.1016/j.tig.2007.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The impact of next-generation sequencing technology on genetics</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMardis%26rft.auinit1%253DE.%2BR.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D133%26rft.epage%253D141%26rft.atitle%253DThe%2Bimpact%2Bof%2Bnext-generation%2Bsequencing%2Btechnology%2Bon%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.12.007%26rft_id%253Dinfo%253Apmid%252F18262675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18262675&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254058200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.28" data-doi="10.2307/2640444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Biochemical evolution associated with antipredator adaptation in damselflies</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1835</span>–<span class="cit-lpage">1845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D53%26rft.spage%253D1835%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2640444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2640444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Predisposed to adapt? Clade-level differences in characters affecting swimming performance in damselflies</span>. <abbr class="cit-jnl-abbrev">Int J. Org. Evol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">2072</span>–<span class="cit-lpage">2080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ.%2BOrg.%2BEvol.%26rft.volume%253D54%26rft.spage%253D2072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.30" data-doi="10.1093/molbev/msq130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meusemann</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Reumont</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeding</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strauss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A phylogenomic approach to resolve the arthropod tree of life</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2451</span>–<span class="cit-lpage">2464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq130%26rft_id%253Dinfo%253Apmid%252F20534705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMS8yNDUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.31" data-doi="10.1186/1471-2164-10-219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aglyamova</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan-Carter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrego</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sequencing and de novo analysis of a coral larval transcriptome using 454 GSFlx</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMeyer%26rft.auinit1%253DE.%26rft.volume%253D10%26rft.spage%253D219%26rft.epage%253D219%26rft.atitle%253DSequencing%2Band%2Bde%2Bnovo%2Banalysis%2Bof%2Ba%2Bcoral%2Blarval%2Btranscriptome%2Busing%2B454%2BGSFlx.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-219%26rft_id%253Dinfo%253Apmid%252F19435504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19435504&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.32" data-doi="10.1111/j.1461-0248.2007.01020.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mittelbach</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornell</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolution and the latitudinal diversity gradient: speciation, extinction and biogeography</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DMittelbach%26rft.auinit1%253DG.%2BG.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D331%26rft.atitle%253DEvolution%2Band%2Bthe%2Blatitudinal%2Bdiversity%2Bgradient%253A%2Bspeciation%252C%2Bextinction%2Band%2Bbiogeography.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2007.01020.x%26rft_id%253Dinfo%253Apmid%252F17355570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2007.01020.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17355570&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244883700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.33" data-doi="10.1093/bioinformatics/btm585"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Hagelsieb</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latimer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Choosing BLAST options for better detection of orthologs as reciprocal best hits</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm585%26rft_id%253Dinfo%253Apmid%252F18042555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNC8zLzMxOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.34" data-doi="10.1089/10665270050514918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vingron</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Modeling amino acid replacement</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DM%25C3%25BCller%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D761%26rft.epage%253D776%26rft.atitle%253DModeling%2Bamino%2Bacid%2Breplacement.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F10665270050514918%26rft_id%253Dinfo%253Apmid%252F11382360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/10665270050514918&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11382360&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168049700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nawy</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Non–model organisms</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D9%26rft.spage%253D37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.36" data-doi="10.1371/journal.pone.0000503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nickle</span>  <span class="cit-name-given-names">David C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heath</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullins</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">HIV-specific probabilistic models of protein evolution</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNickle%26rft.auinit1%253DD.%2BC.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253De503%26rft.epage%253De503%26rft.atitle%253DHIV-specific%2Bprobabilistic%2Bmodels%2Bof%2Bprotein%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000503%26rft_id%253Dinfo%253Apmid%252F17551583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17551583&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.37" data-doi="10.1186/1471-2164-9-312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novaes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drost</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmerie</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pappas</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-throughput gene and SNP discovery in Eucalyptus grandis, an uncharacterized genome</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNovaes%26rft.auinit1%253DE.%26rft.volume%253D9%26rft.spage%253D312%26rft.epage%253D312%26rft.atitle%253DHigh-throughput%2Bgene%2Band%2BSNP%2Bdiscovery%2Bin%2BEucalyptus%2Bgrandis%252C%2Ban%2Buncharacterized%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-312%26rft_id%253Dinfo%253Apmid%252F18590545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18590545&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Outomuro</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bokma</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johansson</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hind wing shape evolves faster than front wing shape in <em>Calopteryx</em> damselflies</span>. <abbr class="cit-jnl-abbrev">Evol. Biol.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">116</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BBiol.%26rft.volume%253D39%26rft.spage%253D116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.39" data-doi="10.1080/10635150118469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Selecting the best-fit model of nucleotide substitution</span>. <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150118469%26rft_id%253Dinfo%253Apmid%252F12116655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUwLzQvNTgwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.40" data-doi="10.1093/bioinformatics/btg180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1572</span>–<span class="cit-lpage">1574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DRonquist%26rft.auinit1%253DF.%26rft.volume%253D19%26rft.issue%253D12%26rft.spage%253D1572%26rft.epage%253D1574%26rft.atitle%253DMrBayes%2B3%253A%2BBayesian%2Bphylogenetic%2Binference%2Bunder%2Bmixed%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg180%26rft_id%253Dinfo%253Apmid%252F12912839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTIvMTU3MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.41" data-doi="10.1603/0013-8746(2003)096[0693:POTDAD]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spicer</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Phylogeny of the dragonfly and damselfly order Odonata as inferred by mitochondrial 12S ribosomal RNA sequences</span>. <abbr class="cit-jnl-abbrev">Ann. Entomol. Soc. Am.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEntomol.%2BSoc.%2BAm.%26rft_id%253Dinfo%253Adoi%252F10.1603%252F0013-8746%25282003%2529096%255B0693%253APOTDAD%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYWVzYSI7czo1OiJyZXNpZCI7czo4OiI5Ni82LzY5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.42" data-doi="10.1093/molbev/msp191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strauss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hadrys</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A phylogenomic approach to resolve the basal pterygote divergence</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2719</span>–<span class="cit-lpage">2730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp191%26rft_id%253Dinfo%253Apmid%252F19713325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi8yNzE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.43" data-doi="10.1111/j.1365-2311.2008.01077.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meester</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoks</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Behavioural activity levels and expression of stress proteins under predation risk in two damselfly species</span>. <abbr class="cit-jnl-abbrev">Ecol. Entomol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEntomol.%26rft.volume%253D34%26rft.spage%253D297%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2311.2008.01077.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2311.2008.01077.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strobbe</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Block</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoks</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fish predation selects for reduced foraging activity</span>. <abbr class="cit-jnl-abbrev">Behav. Ecol. Sociobiol.</abbr> <span class="cit-vol">65</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">241</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BEcol.%2BSociobiol.%26rft.volume%253D65%26rft.spage%253D241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.45" data-doi="10.1101/gr.103846.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Surget-Groba</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montoya-Burgos</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimization of de novo transcriptome assembly from next-generation sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1432</span>–<span class="cit-lpage">1440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.103846.109%26rft_id%253Dinfo%253Apmid%252F20693479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMC8xNDMyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.46" data-doi="10.1080/10635150701472164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talavera</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castresana</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments</span>. <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701472164%26rft_id%253Dinfo%253Apmid%252F17654362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjU2LzQvNTY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.47" data-doi="10.1146/annurev-ento-120710-100538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trautwein</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiegmann</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beutel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjer</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeates</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Advances in insect phylogeny at the dawn of the postgenomic era</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.aulast%253DTrautwein%26rft.auinit1%253DM.%2BD.%26rft.volume%253D57%26rft.spage%253D449%26rft.epage%253D468%26rft.atitle%253DAdvances%2Bin%2Binsect%2Bphylogeny%2Bat%2Bthe%2Bdawn%2Bof%2Bthe%2Bpostgenomic%2Bera.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ento-120710-100538%26rft_id%253Dinfo%253Apmid%252F22149269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ento-120710-100538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22149269&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299834000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.48" data-doi="10.1046/j.1365-294X.2002.01580.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turgeon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Phylogeographic analysis of a recent radiation of <em>Enallagma</em> damselflies (Odonata: Coenagrionidae)</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1989</span>–<span class="cit-lpage">2001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DTurgeon%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D1989%26rft.epage%253D2001%26rft.atitle%253DPhylogeographic%2Banalysis%2Bof%2Ba%2Brecent%2Bradiation%2Bof%2BEnallagma%2Bdamselflies%2B%2528Odonata%253A%2BCoenagrionidae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2002.01580.x%26rft_id%253Dinfo%253Apmid%252F12296943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2002.01580.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12296943&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.49" data-doi="10.1086/428682"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turgeon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoks</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thum</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPeek</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Simultaneous quaternary radiations of three damselfly clades across the holarctic</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">E78</span>–<span class="cit-lpage">E107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DTurgeon%26rft.auinit1%253DJ.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253DE78%26rft.epage%253D107%26rft.atitle%253DSimultaneous%2BQuaternary%2Bradiations%2Bof%2Bthree%2Bdamselfly%2Bclades%2Bacross%2Bthe%2BHolarctic.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F428682%26rft_id%253Dinfo%253Apmid%252F15791532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/428682&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15791532&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227517000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wernersson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Virtual ribosome---a comprehensive DNA translation tool with support for integration of sequence feature annotation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids. Res.</abbr> <span class="cit-vol">34</span>::<span class="cit-fpage"> W385</span>–<span class="cit-lpage">W388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids.%2BRes.%26rft.volume%253D34%26rft.spage%253D%2BW385%26rft.atitle%253DVIRTUAL%2BRIBOSOME---A%2BCOMPREHENSIVE%2BDNA%2BTRANSLATION%2BTOOL%2BWITH%2BSUPPORT%2BFOR%2BINTEGRATION%2BOF%2BSEQUENCE%2BFEATURE%2BANNOTATION%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.51" data-doi="10.1093/oxfordjournals.molbev.a003851"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whelan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">691</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DWhelan%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D691%26rft.epage%253D699%26rft.atitle%253DA%2BGeneral%2BEmpirical%2BModel%2Bof%2BProtein%2BEvolution%2BDerived%2Bfrom%2BMultiple%2BProtein%2BFamilies%2BUsing%2Ba%2BMaximum-Likelihood%2BApproach%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003851%26rft_id%253Dinfo%253Apmid%252F11319253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvNjkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80Lzc2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.52" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC83NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.53" data-doi="10.1093/nar/gkl031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">WEGO: a web tool for plotting GO annotations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">suppl 2</span>): <span class="cit-fpage">W293</span>–<span class="cit-lpage">W297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DYe%26rft.auinit1%253DJ.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW293%26rft.epage%253DW297%26rft.atitle%253DWEGO%253A%2Ba%2Bweb%2Btool%2Bfor%2Bplotting%2BGO%2Bannotations.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl031%26rft_id%253Dinfo%253Apmid%252F16845012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8yL1cyOTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.763.54" data-doi="10.1038/nrg2363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yon Rhee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Draghici</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Use and misuse of the gene ontology annotations</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DYon%2BRhee%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D509%26rft.epage%253D515%26rft.atitle%253DUse%2Band%2Bmisuse%2Bof%2Bthe%2Bgene%2Bontology%2Bannotations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2363%26rft_id%253Dinfo%253Apmid%252F18475267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18475267&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F763.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256832900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/4/763.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/757" title="Parents Without Partners: Drosophila as a Model for Understanding the Mechanisms and Evolution of Parthenogenesis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/771" title="Physical and Genetic Interactions Between Uls1 and the Slx5–Slx8 SUMO-Targeted Ubiquitin Ligase" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/4"><span>Volume 3 Issue 4, April 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/4.cover-source.jpg?itok=tXw4AXt4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/763" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pcQ8PU2n6dd5nDSDV3oljNK8EsEsgV64EJSjFLVnyAY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/4/763" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/4/763" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/4/763" class="active" data-icon-position="" data-hide-link-title="0">Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459445" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-P8gqHOyp6iNQ8BgetsKBCzevfcnp2LWdS-aO_fW-2Ao" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/763.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/4/763" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-c9dSP84oURamakLPeZ9xI6D95Y2MiroSF_ZYkhzdAVE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/4/763" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/763.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459445" id="node459445--2" data-pisa="ggg;3/4/763" data-pisa-master="ggg;g3.113.005637" data-apath="/ggg/3/4/763.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexander G. Shanku</span>, <span class="highwire-citation-author" data-delta="1">Mark A. McPeek</span> and <span class="highwire-citation-author" data-delta="2">Andrew D. Kern</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">763-770;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459445--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexander G. Shanku </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BG.%2BShanku%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shanku%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BG.%2BShanku%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alexander.shanku@rutgers.edu" class="" data-icon-position="" data-hide-link-title="0">alexander.shanku@rutgers.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark A. McPeek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Dartmouth College, New Hampshire 03755</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BMcPeek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McPeek%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BA.%2BMcPeek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew D. Kern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BKern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kern%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BD.%2BKern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/4/763" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/4/763&amp;title=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/4/763&amp;title=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/4/763&amp;title=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/4/763&amp;text=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/4/763&amp;title=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/4/763&amp;t=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/4/763&amp;title=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/4/763&amp;title=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459445/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459445" id="node459445--3" data-pisa="ggg;3/4/763" data-pisa-master="ggg;g3.113.005637" data-apath="/ggg/3/4/763.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Functional Annotation and Comparative Analysis of a Zygopteran Transcriptome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Alexander G. Shanku</span>, <span class="highwire-citation-author" data-delta="1">Mark A. McPeek</span> and <span class="highwire-citation-author" data-delta="2">Andrew D. Kern</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">763-770;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459445--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Alexander G. Shanku </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BG.%2BShanku%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shanku%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BG.%2BShanku%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alexander.shanku@rutgers.edu" class="" data-icon-position="" data-hide-link-title="0">alexander.shanku@rutgers.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark A. McPeek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Dartmouth College, New Hampshire 03755</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BMcPeek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McPeek%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BA.%2BMcPeek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew D. Kern </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Rutgers, The State University of New Jersey, Department of Genetics, Piscataway, New Jersey 08854-8082</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BD.%2BKern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kern%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BD.%2BKern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=763&amp;rft.epage=770&amp;rft.atitle=Functional%20Annotation%20and%20Comparative%20Analysis%20of%20a%20Zygopteran%20Transcriptome&amp;rft.volume=3&amp;rft.issue=4&amp;rft.date=2013-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shanku%2C+Alexander+G.&amp;rft.au=McPeek%2C+Mark+A.&amp;rft.au=Kern%2C+Andrew+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459445/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459445/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459445/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459445/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459445/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459445/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459445/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459445/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459445/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459445/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459445/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459445/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/4/763" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/4/763.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/4/763.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/4/763.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/4/763.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
