<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/4/597" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/4/597.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/4/597.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/4/597.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/4/597.atom" />
<meta name="HW.pisa" content="ggg;3/4/597" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains" />
<meta name="DC.Identifier" content="10.1534/g3.113.005546" />
<meta name="DC.Date" content="2013-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Scott-Boyer, Deschepper. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Although gene coexpression domains have been reported in most eukaryotic organisms, data available to date suggest that coexpression rarely concerns more than doublets or triplets of adjacent genes in mammals. Using expression data from hearts of mice from the panel of AxB/BxA recombinant inbred mice, we detected (according to window sizes) 42−53 loci linked to the expression levels of clusters of three or more neighboring genes. These loci thus formed “ cis -expression quantitative trait loci (eQTL) clusters” because their position matched that of the genes whose expression was linked to the loci. Compared with matching control regions, genes contained within cis -eQTL clusters showed much greater levels of coexpression. Corresponding regions showed: (1) a greater abundance of polymorphic elements (mostly short interspersed element retrotransposons), and (2) significant enrichment for the motifs of binding sites for various transcription factors, with binding sites for the chromatin-organizing CCCTC-binding factor showing the greatest levels of enrichment in polymorphic short interspersed elements. Similar cis -eQTL clusters also were detected when we used data obtained with several tissues from BxD recombinant inbred mice. In addition to strengthening the evidence for gene expression domains in mammalian genomes, our data suggest a possible mechanism whereby noncoding polymorphisms could affect the coordinate expression of several neighboring genes." />
<meta name="DC.Contributor" content="Marie-Pier Scott-Boyer" />
<meta name="DC.Contributor" content="Christian F. Deschepper" />
<meta name="article:published_time" content="2013-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Although gene coexpression domains have been reported in most eukaryotic organisms, data available to date suggest that coexpression rarely concerns more than doublets or triplets of adjacent genes in mammals. Using expression data from hearts of mice from the panel of AxB/BxA recombinant inbred mice, we detected (according to window sizes) 42−53 loci linked to the expression levels of clusters of three or more neighboring genes. These loci thus formed “&lt;i&gt;cis&lt;/i&gt;-expression quantitative trait loci (eQTL) clusters” because their position matched that of the genes whose expression was linked to the loci. Compared with matching control regions, genes contained within &lt;i&gt;cis&lt;/i&gt;-eQTL clusters showed much greater levels of coexpression. Corresponding regions showed: (1) a greater abundance of polymorphic elements (mostly short interspersed element retrotransposons), and (2) significant enrichment for the motifs of binding sites for various transcription factors, with binding sites for the chromatin-organizing CCCTC-binding factor showing the greatest levels of enrichment in polymorphic short interspersed elements. Similar &lt;i&gt;cis&lt;/i&gt;-eQTL clusters also were detected when we used data obtained with several tissues from BxD recombinant inbred mice. In addition to strengthening the evidence for gene expression domains in mammalian genomes, our data suggest a possible mechanism whereby noncoding polymorphisms could affect the coordinate expression of several neighboring genes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/04/01" />
<meta name="citation_mjid" content="ggg;3/4/597" />
<meta name="citation_id" content="3/4/597" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/4/597" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/4/597.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/4/597.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/4/597.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.005546" />
<meta name="citation_pmid" content="23550129" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="597" />
<meta name="citation_lastpage" content="605" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marie-Pier Scott-Boyer" />
<meta name="citation_author_institution" content="Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada" />
<meta name="citation_author" content="Christian F. Deschepper" />
<meta name="citation_author_institution" content="Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada" />
<meta name="citation_author_email" content="deschec@ircm.qc.ca" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Akagi;citation_author=J. Li;citation_author=R. M. Stephens;citation_author=N. Volfovsky;citation_author=D. E. Symer;citation_title=Extensive variation between inbred mouse strains due to endogenous L1 retrotransposition;citation_volume=18;citation_year=2008;citation_pmid=18381897;citation_doi=10.1101/gr.075770.107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Akagi;citation_author=R. M. Stephens;citation_author=J. Li;citation_author=E. Evdokimov;citation_author=M. R. Kuehn;citation_title=MouseIndelDB: a database integrating genomic indel polymorphisms that distinguish mouse strains;citation_volume=38;citation_year=2010;citation_pmid=19933259;citation_doi=10.1093/nar/gkp1046" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. J. Blow;citation_author=D. J. McCulley;citation_author=Z. Li;citation_author=T. Zhang;citation_author=J. A. Akiyama;citation_title=ChIP-Seq identification of weakly conserved heart enhancers.;citation_pages=806-810;citation_volume=42;citation_year=2010;citation_issue=9;citation_pmid=20729851;citation_doi=10.1038/ng.650" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Bourque;citation_author=B. Leong;citation_author=V. B. Vega;citation_author=X. Chen;citation_author=Y. L. Lee;citation_title=Evolution of the mammalian transcription factor binding repertoire via transposable elements;citation_volume=18;citation_year=2008;citation_pmid=18682548;citation_doi=10.1101/gr.080663.108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Brunschwig;citation_author=L. Levi;citation_author=E. Ben-David;citation_author=R. W. Williams;citation_author=B. Yakir;citation_title=Fine-scale maps of recombination rates and hotspots in the mouse genome;citation_volume=191;citation_year=2012;citation_pmid=22562932;citation_doi=10.1534/genetics.112.141036" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=T. Chernova;citation_author=F. M. Higginson;citation_author=R. Davies;citation_author=A. G. Smith;citation_title=B2 SINE retrotransposon causes polymorphic expression of mouse 5-aminolevulinic acid synthase 1 gene.;citation_pages=515-520;citation_volume=377;citation_year=2008;citation_issue=2;citation_pmid=18929534;citation_doi=10.1016/j.bbrc.2008.10.020" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. J. Chesler;citation_author=L. Lu;citation_author=S. Shou;citation_author=Y. Qu;citation_author=J. Gu;citation_title=Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function.;citation_pages=233-242;citation_volume=37;citation_year=2005;citation_issue=3;citation_pmid=15711545;citation_doi=10.1038/ng1518" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Didion;citation_author=H. Yang;citation_author=K. Sheppard;citation_author=C.-P. Fu;citation_author=L. McMillan;citation_title=Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias.;citation_pages=34-34;citation_volume=13;citation_year=2012;citation_pmid=22260749;citation_doi=10.1186/1471-2164-13-34" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. Dixon;citation_author=S. Selvaraj;citation_author=F. Yue;citation_author=A. Kim;citation_author=Y. Li;citation_title=Topological domains in mammalian genomes identified by analysis of chromatin interactions.;citation_pages=376-380;citation_volume=485;citation_year=2012;citation_issue=7398;citation_pmid=22495300;citation_doi=10.1038/nature11082" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=S. Durinck;citation_author=P. T. Spellman;citation_author=E. Birney;citation_author=W. Huber;citation_title=Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt.;citation_pages=1184-1191;citation_volume=4;citation_year=2009;citation_issue=8;citation_pmid=19617889;citation_doi=10.1038/nprot.2009.97" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=L. I. Elizondo;citation_author=P. Jafar-Nejad;citation_author=J. M. Clewing;citation_author=C. F. Boerkoel;citation_title=Gene clusters, molecular evolution and disease: a speculation.;citation_pages=64-75;citation_volume=10;citation_year=2009;citation_issue=1;citation_pmid=19721813;citation_doi=10.2174/138920209787581271" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=An integrated encyclopedia of DNA elements in the human genome.;citation_pages=57-74;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955616;citation_doi=10.1038/nature11247" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=E. Gogvadze;citation_author=A. Buzdin;citation_title=Retroelements and their impact on genome evolution and functioning.;citation_pages=3727-3742;citation_volume=66;citation_year=2009;citation_issue=23;citation_pmid=19649766;citation_doi=10.1007/s00018-009-0107-2" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. H. H. Göring;citation_author=J. E. Curran;citation_author=M. P. Johnson;citation_author=T. D. Dyer;citation_author=J. Charlesworth;citation_title=Discovery of expression QTLs using large-scale transcriptional profiling in human lymphocytes.;citation_pages=1208-1216;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17873875;citation_doi=10.1038/ng2119" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=I. C. Grieve;citation_author=N. J. Dickens;citation_author=M. Pravenec;citation_author=V. Kren;citation_author=N. Hubner;citation_title=Genome-wide co-expression analysis in multiple tissues.;citation_pages=e4033-e4033;citation_volume=3;citation_year=2008;citation_issue=12;citation_pmid=19112506;citation_doi=10.1371/journal.pone.0004033" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. Jurka;citation_author=O. Kohany;citation_author=A. Pavlicek;citation_author=V. V. Kapitonov;citation_author=M. V. Jurka;citation_title=Clustering, duplication and chromosomal distribution of mouse SINE retrotransposons.;citation_pages=117-123;citation_volume=110;citation_year=2005;citation_issue=1-4;citation_pmid=16093663;citation_doi=10.1159/000084943" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=D. H. Kass;citation_author=J. Kim;citation_author=A. Rao;citation_author=P. L. Deininger;citation_title=Evolution of B2 repeats: the muroid explosion.;citation_pages=1-13;citation_volume=99;citation_year=1997;citation_issue=1;citation_pmid=9226433" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Lander;citation_author=L. Kruglyak;citation_title=Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results.;citation_pages=241-247;citation_volume=11;citation_year=1995;citation_issue=3;citation_pmid=7581446;citation_doi=10.1038/ng1195-241" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. J. Lercher;citation_author=A. O. Urrutia;citation_author=L. D. Hurst;citation_title=Clustering of housekeeping genes provides a unified model of gene order in the human genome.;citation_pages=180-183;citation_volume=31;citation_year=2002;citation_issue=2;citation_pmid=11992122;citation_doi=10.1038/ng887" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Li;citation_author=K. Akagi;citation_author=Y. Hu;citation_author=A. L. Trivett;citation_author=C. J. W. Hlynialuk;citation_title=Mouse endogenous retroviruses can trigger premature transcriptional termination at a distance;citation_volume=22;citation_year=2012;citation_pmid=22367191;citation_doi=10.1101/gr.130740.111" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=B. Llamas;citation_author=S. Bélanger;citation_author=S. Picard;citation_author=C. F. Deschepper;citation_title=Cardiac mass and cardiomyocyte size are governed by different genetic loci on either autosomes or chromosome Y in recombinant inbred mice;citation_volume=31;citation_year=2007;citation_pmid=17566079;citation_doi=10.1152/physiolgenomics.00072.2007" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. Llamas;citation_author=R. A. Verdugo;citation_author=G. A. Churchill;citation_author=C. F. Deschepper;citation_title=Chromosome Y variants from different inbred mouse strains are linked to differences in the morphologic and molecular responses of cardiac cells to postpubertal testosterone.;citation_pages=150-150;citation_volume=10;citation_year=2009;citation_pmid=19351403;citation_doi=10.1186/1471-2164-10-150" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. A. Manolio;citation_author=F. S. Collins;citation_author=N. J. Cox;citation_author=D. B. Goldstein;citation_author=L. A. Hindorff;citation_title=Finding the missing heritability of complex diseases.;citation_pages=747-753;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19812666;citation_doi=10.1038/nature08494" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=J. D. Marshall;citation_author=J. L. Mu;citation_author=Y. C. Cheah;citation_author=M. N. Nesbitt;citation_author=W. N. Frankel;citation_title=The AXB and BXA set of recombinant inbred mouse strains.;citation_pages=669-680;citation_volume=3;citation_year=1992;citation_issue=12;citation_pmid=1477475;citation_doi=10.1007/BF00444361" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=P. Michalak;citation_title=Coexpression, coregulation, and cofunctionality of neighboring genes in eukaryotic genomes.;citation_pages=243-248;citation_volume=91;citation_year=2008;citation_issue=3;citation_pmid=18082363;citation_doi=10.1016/j.ygeno.2007.11.002" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=C. Nellåker;citation_author=T. M. Keane;citation_author=B. Yalcin;citation_author=K. Wong;citation_author=A. Agam;citation_title=The genomic landscape shaped by selection on transposable elements across 18 mouse strains.;citation_pages=R45-R45;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22703977;citation_doi=10.1186/gb-2012-13-6-r45" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Neurosci;citation_author=A. A. Palmer;citation_author=S. C. Dulawa;citation_title=Murine Warriors or Worriers: The Saga of Comt1, B2 SINE Elements, and the Future of Translational Genetics.;citation_pages=177-177;citation_volume=4;citation_year=2010;citation_pmid=20976038" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. E. Phillips;citation_author=V. G. Corces;citation_title=CTCF: master weaver of the genome.;citation_pages=1194-1211;citation_volume=137;citation_year=2009;citation_issue=7;citation_pmid=19563753;citation_doi=10.1016/j.cell.2009.06.001" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=E. M. Pugacheva;citation_author=T. Suzuki;citation_author=S. D. Pack;citation_author=N. Kosaka-Suzuki;citation_author=J. Yoon;citation_title=The structural complexity of the human BORIS gene in gametogenesis and cancer.;citation_pages=e13872-e13872;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21079786;citation_doi=10.1371/journal.pone.0013872" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=A. Purmann;citation_author=J. Toedling;citation_author=M. Schueler;citation_author=P. Carninci;citation_author=H. Lehrach;citation_title=Genomic organization of transcriptomes in mammals: Coregulation and cofunctionality.;citation_pages=580-587;citation_volume=89;citation_year=2007;citation_issue=5;citation_pmid=17369017;citation_doi=10.1016/j.ygeno.2007.01.010" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. E. Schadt;citation_author=S. A. Monks;citation_author=T. A. Drake;citation_author=A. J. Lusis;citation_author=N. Che;citation_title=Genetics of gene expression surveyed in maize, mouse and man.;citation_pages=297-302;citation_volume=422;citation_year=2003;citation_issue=6929;citation_pmid=12646919;citation_doi=10.1038/nature01434" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Schlesinger;citation_author=M. Schueler;citation_author=M. Grunert;citation_author=J. J. Fischer;citation_author=Q. Zhang;citation_title=The cardiac transcription network modulated by Gata4, Mef2a, Nkx2.5, Srf, histone modifications, and microRNAs.;citation_pages=e1001313-e1001313;citation_volume=7;citation_year=2011;citation_issue=2;citation_pmid=21379568;citation_doi=10.1371/journal.pgen.1001313" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Schmidt;citation_author=P. C. Schwalie;citation_author=M. D. Wilson;citation_author=B. Ballester;citation_author=Â. Gonçalves;citation_title=Waves of retrotransposon expansion remodel genome organization and CTCF binding in multiple mammalian lineages.;citation_pages=335-348;citation_volume=148;citation_year=2012;citation_issue=1-2;citation_pmid=22244452;citation_doi=10.1016/j.cell.2011.11.058" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Biosyst;citation_author=M. Schueler;citation_author=Q. Zhang;citation_author=J. Schlesinger;citation_author=M. Tönjes;citation_author=S. R. Sperling;citation_title=Dynamics of Srf, p300 and histone modifications during cardiac maturation in mouse.;citation_pages=495-503;citation_volume=8;citation_year=2012;citation_issue=2;citation_pmid=22101280;citation_doi=10.1039/c1mb05363a" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=C. Schurmann;citation_author=K. Heim;citation_author=A. Schillert;citation_author=S. Blankenberg;citation_author=M. Carstensen;citation_title=Analyzing illumina gene expression microarray data from different tissues: methodological aspects of data analysis in the MetaXpress consortium;citation_pages=e50938;citation_volume=7;citation_year=2012;citation_pmid=23236413;citation_doi=10.1371/journal.pone.0050938" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Sémon;citation_author=L. Duret;citation_title=Evolutionary origin and maintenance of coexpressed gene clusters in mammals;citation_volume=23;citation_year=2006;citation_pmid=16757654;citation_doi=10.1093/molbev/msl034" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_author=G. K. Smyth;citation_title=LINEAR MODELS AND EMPIRICAL BAYES METHODS FOR ASSESSING DIFFERENTIAL EXPRESSION IN MICROARRAY EXPERIMENTS;citation_pages=Article 3;citation_volume=3;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Journal of biology (Online);citation_journal_abbrev=J Biol;citation_author=P. T. Spellman;citation_author=G. M. Rubin;citation_title=Evidence for large domains of similarly expressed genes in the Drosophila genome.;citation_pages=5-5;citation_volume=1;citation_year=2002;citation_issue=1;citation_pmid=12144710;citation_doi=10.1186/1475-4924-1-5" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. Teng;citation_author=H. A. Firpi;citation_author=K. Tan;citation_title=Enhancers in embryonic stem cells are enriched for transposable elements and genetic variations associated with cancers;citation_volume=39;citation_year=2011;citation_pmid=21685454;citation_doi=10.1093/nar/gkr476" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=L. N. van de Lagemaat;citation_author=J.-R. Landry;citation_author=D. L. Mager;citation_author=P. Medstrand;citation_title=Transposable elements in mammals promote regulatory variation and diversification of genes with specialized functions.;citation_pages=530-536;citation_volume=19;citation_year=2003;citation_issue=10;citation_pmid=14550626;citation_doi=10.1016/j.tig.2003.08.004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Visel;citation_author=E. M. Rubin;citation_author=L. A. Pennacchio;citation_title=Genomic views of distant-acting enhancers.;citation_pages=199-205;citation_volume=461;citation_year=2009;citation_issue=7261;citation_pmid=19741700;citation_doi=10.1038/nature08451" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. H. Woo;citation_author=M. Walker;citation_author=G. A. Churchill;citation_title=Coordinated expression domains in mammalian genomes.;citation_pages=e12158-e12158;citation_volume=5;citation_year=2010;citation_issue=8;citation_pmid=20805879;citation_doi=10.1371/journal.pone.0012158" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Wu;citation_author=D. L. Delano;citation_author=N. Mitro;citation_author=S. V. Su;citation_author=J. Janes;citation_title=Gene set enrichment in eQTL data identifies novel annotations and pathway regulators.;citation_pages=e1000070-e1000070;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18464898;citation_doi=10.1371/journal.pgen.1000070" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Yalcin;citation_author=K. Wong;citation_author=A. Agam;citation_author=M. Goodson;citation_author=T. M. Keane;citation_title=Sequence-based characterization of structural variation in the mouse genome.;citation_pages=326-329;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921916;citation_doi=10.1038/nature10432" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=H. Yang;citation_author=Y. Ding;citation_author=L. N. Hutchins;citation_author=J. Szatkiewicz;citation_author=T. A. Bell;citation_title=A customized and versatile high-density genotyping array for the mouse.;citation_pages=663-666;citation_volume=6;citation_year=2009;citation_issue=9;citation_pmid=19668205;citation_doi=10.1038/nmeth.1359" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Zhang;citation_author=K.-H. Pan;citation_author=S. N. Cohen;citation_title=Senescence-specific gene expression fingerprints reveal cell-type-dependent physical clustering of up-regulated chromosomal loci;citation_volume=100;citation_year=2003;citation_pmid=12626749;citation_doi=10.1073/pnas.2627983100" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="twitter:description" content="Although gene coexpression domains have been reported in most eukaryotic organisms, data available to date suggest that coexpression rarely concerns more than doublets or triplets of adjacent genes in mammals. Using expression data from hearts of mice from the panel of AxB/BxA recombinant inbred mice, we detected (according to window sizes) 42−53 loci linked to the expression levels of clusters of three or more neighboring genes. These loci thus formed “ cis -expression quantitative trait loci (eQTL) clusters” because their position matched that of the genes whose expression was linked to the loci. Compared with matching control regions, genes contained within cis -eQTL clusters showed much greater levels of coexpression. Corresponding regions showed: (1) a greater abundance of polymorphic elements (mostly short interspersed element retrotransposons), and (2) significant enrichment for the motifs of binding sites for various transcription factors, with binding sites for the chromatin-organizing CCCTC-binding factor showing the greatest levels of enrichment in polymorphic short interspersed elements. Similar cis -eQTL clusters also were detected when we used data obtained with several tissues from BxD recombinant inbred mice. In addition to strengthening the evidence for gene expression domains in mammalian genomes, our data suggest a possible mechanism whereby noncoding polymorphisms could affect the coordinate expression of several neighboring genes." />
<meta name="og-title" property="og:title" content="Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/4/597" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Although gene coexpression domains have been reported in most eukaryotic organisms, data available to date suggest that coexpression rarely concerns more than doublets or triplets of adjacent genes in mammals. Using expression data from hearts of mice from the panel of AxB/BxA recombinant inbred mice, we detected (according to window sizes) 42−53 loci linked to the expression levels of clusters of three or more neighboring genes. These loci thus formed “ cis -expression quantitative trait loci (eQTL) clusters” because their position matched that of the genes whose expression was linked to the loci. Compared with matching control regions, genes contained within cis -eQTL clusters showed much greater levels of coexpression. Corresponding regions showed: (1) a greater abundance of polymorphic elements (mostly short interspersed element retrotransposons), and (2) significant enrichment for the motifs of binding sites for various transcription factors, with binding sites for the chromatin-organizing CCCTC-binding factor showing the greatest levels of enrichment in polymorphic short interspersed elements. Similar cis -eQTL clusters also were detected when we used data obtained with several tissues from BxD recombinant inbred mice. In addition to strengthening the evidence for gene expression domains in mammalian genomes, our data suggest a possible mechanism whereby noncoding polymorphisms could affect the coordinate expression of several neighboring genes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
    <title>Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FYK8YKaBYsMvf9mU6TeCOI65lOQELyp_0Pi7iujAcXQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459440","apath":"\/ggg\/3\/4\/597.atom","pisa":"ggg;3\/4\/597","ac":{"\/ggg\/3\/4\/597.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/4\/597.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/4\/597"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/4\/597"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/4\/597.full":true,"\/content\/3\/4\/597":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227XqhKyXdt232JpAde\\\\\\\/su5l0P7CkRoYH8j1gEZYZf\\\\\\\/zmUtHJGUn\\\\\\\/uyraVzJwE+enNhELkjfMgdMdylDG5VzTwXWD8K2gz9Cdq\\\\\\\/1LQTPKLtWzCeFyVaGO7WtSDv3vvslGkk5rbNDSuLbZtWTM2\\\\\\\/HIhkXNzDWuHAlJPlD+lbqjwUC5jWRcq\\\\\\\/NBLg3XVLd7LM1eShkSPkZ5HKsP7ugFm8KFXc3L1YdpPrpOv2iYJYyAs2SkahjBbn5fZKnKPZPK++VUsLfNNmiuKdcgDT1UzUOP7e10UWo8IAln8jhBwxA+W7KBgwiMNccAET6zd83KLaq9uLxx1qEzGNBBlUOyXTbTV+TVOZE1oM7wlG8x7ETls0sTg7o5JHMcr\\\\\\\/vXYrbgWLOOOLoZjz1TpRaocb+Tw0HZviKLdYKh4+RTKpTV7otedo9ae7cfKfBzpP+NUal73qzTgEANFhcdpVVLooS3Bu\\\\\\\/Fnl6onGcQzK3Y65Wz\\\\\\\/2ZGDlJt1TKrOJpSt3LnAde1wNPyZp6mmOoGgKqKhvECPPv8Q09cJbC5sBjaKT4c3QaE5DMme37HUJje7UpfNvH5iMD9jIX3dYunC\\\\\\\/4sK0stq0mGfmK9K+j8\\\\\\\/x2mz+hM7MaEipKYpNNOBbiHwAUN8F4bZeIs+Yfk6qpwWFoBiD9xOqm\\\\\\\/gQrdH4eUYrOO7G2DJKAN6yA2JlV\\\\\\\/fCbvNWAekazmzbQEkYL1gh6bjhuh31xjhvP6E+121qGcfFZGdbLWHsg1F2S2Wc51GmxexIOEzNs1vLu5RYju\\\\\\\/WaAHseT\\\\\\\/Q+etnKhMJDcsYaDVTI7nfH3C34olNtdacFr7O\\\\\\\/AdLy91lJ4pljRxOiEfwOv6UUQf2lLnxD9wEnroNrEVbiYM4K1Ji\\\\\\\/nEmejfLqEPtAG8Ff0SpCIXaeOBvM5WxbLPae43Fe\\\\\\\/bTkiWJZu8KlJPgFeSPEeQa5XAMt7kj0QqpxMelyVosSyqA+pEPsAtpn4xbj+W0LxG+Kd76Vp0AzIa6Y9IYZwU\\\\\\\/1VB2qZOT0wH3O8Un8QJYpGkX1oIZ0IWw+Ekp5n7KvA06CqeaxkU9mNHe9r1mszmCjg\\\\\\\/61GDZocbtkb8gdm81jrU97XhhRj+hvDkpTDTstSyk8kC16jbejfXilKUx0o3Y2GTE85yEmJ3gr6wKxL2qqSXb3dxc09abZDAk7d2SY5LJ1L23rPxhZLPa1a1rsXV+WCpyXQ9UB2jt1RsTIvVCW2CSpx6OzP\\\\\\\/xg\\\\\\\/6JfnYODF\\\\\\\/WU6YahfF7nfhmIk5BTxerJOK\\\\\\\/tEqEPGisSSFn5p6c5oCrSLV\\\u0022,\\\u0022iv\\\u0022:\\\u0022XPcyN8MiV5LYrT9ca654CA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d92a9206fc59d96c82afd5dfa6ab46e4910072bf8986d54b4c52c539731877c2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wippMmlGFwLQqkbTagt2BzoE+Nr0KkBZFIYJdHe8DfG44VK+S43b3qZLJd6QaZysl8+AWlQnfAxgvMMBLjLH5LbVjJgBDY08RhBEqg5ZxqrhHeLqTh8tnH8R3c21cNaRq\\\\\\\/SKbI3v+5kF3HWTJ8yJbOFlIh5lND1PV+0Hk6O9zv3kpViJYH53+DtMsDAGVJOm4T10HF2ZgxNyIY87EWuHzT63mAb6fApv\\\\\\\/NFdkOfeYqtoPTPmhvtPu3EAFUi13ANaIpy1qiudd+q64Or78OXmfUXgzR+L+6F8nDFTy8KPx1wEnqZ3wpUhhsHAnXU8Gn3GsdqYpDcxe3Jnqdp2ygBcIExWoQRkPz+YLqkYc+GOCRmUUYgY3c8y9bIHQwTfbOdyJTbT+VRripv+5dbtUoObvcaogXMHeGBrrAILxHdG\\\\\\\/7HOs+fF+HtPsaP9UgTzxaTbe+MMALAYCHHc6tWLHM\\\\\\\/ENvZlJ79mvwss65LETEiQLgcPUIjdDEIuL4css2B7l0h4luzLDz1z\\\\\\\/iaDrJIEmLGOJ2TZ2Ri3p4jr1LQokTum\\\\\\\/6yCPDYJJeGx3uz7ILaPnqeZ4sd8RObo0TN6wgKdsNkFxdICx5IZ8gXIc23wS79Z4PnvdgojNw810DAnaLyOMMd7U+TqZt47XLJgt7tI3n+6r+8sOPUNmaF\\\\\\\/jvozJq2BjHHy6hdH49KZDqGk5Ly5PZvPd8K0CEURUgIc3CcfH+qvKLJiC6ahiq+\\\\\\\/3bBTdHFjRLxdUzMmtzlFXlXNKqsWlM3IJausoBsarZyP9BsF6tG2poNNzyGaM3RqsFFS+5jAzdcLtcri3uUbhC8R9JDDZJiyYdqhr3CzEJXanscAouzDWCIptlrVaKa9Vun3Z2v008eFjMLyL4Sqi+0xrKHEuESNC9Y\\\\\\\/0x6Ro7lACBZGfpzQqnnQR5Ln8uz\\\\\\\/Td257rBirodGjsR\\\\\\\/Kzxz\\\\\\\/1ZEAi4OmoPAqKmNGbibGtIQmRXejAk4b0kj9S9nJ2UIwf3ZJ8OzsoN6YA8tbmph6RfN9jvxLyIZJA69GOeoMF1pEDSengdiBlrgBUmuX0qgjIcPLkk8F+ldutBzKL8F9XqeNyL5aH+pVIu70lUL0UPAKMpQ7v5koC9P\\\\\\\/1nga99VPAtkwlKV2lW0a7kucLfFbd162usr3PltdJhf8FTAqJIf6ZDqCOsgEWh+zVUhtVc6S+dbsxpRxtykECiTyCgxsgwc5FtXteGFjoS8fR4ldJJy1SOQej6C0y\\\\\\\/QKxnUZiy\\\\\\\/MQ27bl8h8oxDdbTY+\\\\\\\/3Ix2ZIIIaZ5TBbjYymrmUlvqIH8dCXeJLC9zZ2JDoot1irt01ywG1Oo\\\\\\\/FwTIjM4PLi\\\\\\\/DLkUpUxHhvNtGukTAf+b6T6TiMmL5aB18vMfj1\\\\\\\/qyD62YG9cyArwEX8dhnN6IvSsf9cyM0gPdWWL13+tQArbSCyV6NexISCwymZo0GWe6kiMv34txNXSHDyV1KFyIGaC42pt+EV\\\\\\\/DA61azu8I8zw+KkASd\\\\\\\/qcyjgWjngS64qDpPJ7wOx387H9mla64zakI47e06OdRenISfLoZE1IzzqOo7uXShaVRlCdLg6xrB1iv2Smv6\\\\\\\/HdcqPSKbNJsb+e2hVLioP8yo7vjaAfdPS5pJBF4m3SMir4WP4qJBrNJutJZsBIg5xsCT2\\\\\\\/3us\\\\\\\/HstPuCy1hkdYjEP9gBZlgWqFkjuvHwz7feQlrglpmpW093Ih0AV\\\\\\\/smqf4A2mvv4mfVuHfUrN\\\\\\\/tzjQ3XDhsDDSk+fyK\\\\\\\/VKI00+YxWKQizEtr+Rf0fsylV8tcKReUN8r1l3d\\\\\\\/HhB0knXPo5x9fQkfHj3Mp2YPJubM6LHb1DzeCz7qcJ+O52jeZe\\\\\\\/EYmKxW\\\\\\\/drUAja+8tOq6P4749Cm6tk+zXvuiHD78UBDnhwpdEDmb7eFibrUBESbgg3MYuY9Tn\\\\\\\/yLgrH9eWEKisPyCQIFWu4HLtvA\\\\\\\/xxZiSP8iKhgZpH3eiFrDD0lDm21G2rsdcmsGDIubW09guZhfUOzc6R2sf90RYo5fxj7xG7HFvzY36CfN55pv9dfTzWSNeeq1cOpr5u2EOPpVNaNNz3h32GBfC1N\\\\\\\/tRjCVA+bxyLIHyN1lt4tpLOWJxIfUZYwj7CRYiI6a+v4NvhByg7KVIWYO\\\\\\\/SMbyQAfB+Uw2GxpbLSH9rt9XeJAhnUFJ1uAJ4ikzMRn2rnB1a+00W\\\\\\\/RqcGQI5Rbncr+PFrYRmxCUpJQxKbrEVlFBCMnGpcKv+FsWOa1AMlODsYTtUjiQIVDhsbPC51rdmVidsVHFCImuCYJ1k6Udrolk7c3reGqwOy1JDC0zhQKQcDT9UKL8DXnXm0ReI3f04BxM0q9eeD\\\\\\\/XuoAwJ4KI5azXcVLVVJXKNYg6EPosGT0XuTmJSoKSvuz8PlC1f55riDkLBZELGDT\\\\\\\/m5MUGqBrAo1D1PKEXsb0CjLCi4hcW4NMp\\\\\\\/aM6V5\\\\\\\/WurReCHXnoogwyK1jPEByteIWH1mSJPJInXYgXF8s8owIazAu16pJaPWkrtjCYJJ6w\\\\\\\/bIIZnKcsGsyMLLRC4ZR\\\\\\\/CkalYg0JmS+jWaOMZ\\\\\\\/RVH+LdJnVAXWixfJbTDjp+L6Tj2vYjPSWHFDdRRgmhzvOJPbtvQZCtmIEMM2Ia8dwh6IYDJrsFdhJYPMZD8+ztK6UWxaHzflDtImM2NHVmowogNb6SNg3hN4FX4H8hPt3DPt892yglVbqXYZx1ki0mblCOgicxfzj2\\\\\\\/9uLnNUhxWR3Mk23\\\\\\\/r6rj5WEzM+LXFB1doqGVlMvpvUKJO0UvyzuyXIkvWlArCFADtze\\\\\\\/Wwmjs9S+k7eNMV88vfA1IWCzce0mKwQYLDYSMJni\\\\\\\/s3Utv4tVAmL4YaBCaZBV4tftRb4GmuYMo9zMld\\\\\\\/bWynpfxClOPdcIOvZaOeMjmHxUu6vU6oCtEyNZyrT68VqTWDAT2I3tmsZYNOZjGHUos0yr+H8WfyXseBIBHYDRO6p2b20Mxt0ZaPe30Cz4Qir7PCwy\\\\\\\/Tpoxt1Cjbpdow7lNp8KyMB75hPM6dPgx+SISu0LcDiESr+QhfKunvsdrNBOX0Wcf7NU28xHkG5XBKNDJ5WeCaGk4xRFLRjSCSnOTPh1LTScg172\\\\\\\/XRupWEDKQgo0KHCGcQBfr0C6oNQjNzzXOfqu2L116CF7TpYvBUmQBj1MQq3Niv62+qdI1A\\\\\\\/g9MNFyg305fp\\\\\\\/JPwmHAIJPOhwgTQstykVhqV\\\\\\\/lzfZALvEHwTBkK8y+4JlPh24pwhzPNPVUY3mpdSUVypdWjUsv0r8oQau0VzJSKT5NGnSXGSSaI1R3JWxEf3hIiqSV+5cBcSFFYw71Hzg4RMSq8NWJB+kIfmVOgUu1FL7mbV0LA19s5dLPE0UoDrsk3asuycAKjRO59mPCRi2WX6+n47DmYQhPLCVBPVbMLXa5L\\\\\\\/0YtTVG77I\\\\\\\/0fGFNJHpwRSHsw4XOgFAXjuscNBVIAMsGEVua1SgdVVlk84W8RkGtWZVJDNq2j\\\\\\\/bh5yOKu02NYR2d6FQk0tNi2nxi0YeASzBaGwmcHYb8G71htvhSBEruZ3KLqjY3FGTPKVsxD\\\\\\\/frDyNhxSUuvNsD4S1faYLgT5bdU8cwK1ZemItQOPMf\\\\\\\/oK7GhF9D2nDPUIvyS8pajt9\\\\\\\/v2nWRRav4IBER\\\\\\\/M1ZXmCVYu6FpDp\\\\\\\/ub2yFmmlN4ILwgHl+aPWzIvokpbe4AV+UnQPLVN2auycULoH4lvdkXineqW7kdTZD7PcVJC13Jur2D4gCnyCv1dxvkmPRfpLqZXb5b6YVBlGmKoz0Zu7FmqaeGw\\\\\\\/9+NVpM4TVRExu6y1ivPlPKIpkZuygSIH40035FbOMgpCGqEawXVtFEUde1xCWqqHrS2fS2j+FV4stzumDqFYiqooJUx\\\\\\\/hc7DMdTG1emSsVUYHkeyuvTWGo57IbO485e6wF\\\\\\\/7su3w3z6AHqCs5PYReGSJ7gtrtxxmubiMF\\\\\\\/eELqn688j7m8iQswIA8XzI0BFuETWrfVYdt\\\\\\\/6sbuJCe4Zf1hKoDiZX\\\\\\\/12CpdVhtoeUvezEqbWUCBytZ6yVzaV+YdiiqwQoZNJDqqO50+ZnQZgmEyvlQrxrmc2IptZIcGKJuZZJgZTOm4reP8kbjtHDvusVNdv8c4Uxcz05wbkS1RxIaejnYAf9JVup11Isiq\\\\\\\/E0cjuPAvrTCfU\\\\\\\/TPO+mWqeGdp06knows65G26T8D18QbT\\\\\\\/N1nAGcQ89mL5ALPYpybru8aULoE1f9GT6qcv2dKas\\\\\\\/t0jOhFTpUrurHviD+FpLeuarA8BMco+kKpbFx+KLKAi5HMv6nrt0Zq5IUESXRdmnuiNWCe8SD5AXDST\\\\\\\/J1tK\\\\\\\/8jTOMAomv5JoGz9ubb3AaxeunXI9i6jNc695XCzhFsCF2Us\\\\\\\/sk4AFSvd4nIDK1sVQVovcq9IabF31y\\\\\\\/X\\\\\\\/bjQGKtWCS3zPR5ZHivBtZa4Z\\\\\\\/PAR5\\\\\\\/DxYONNxqEq7Iws0NeGTVhIyZ4jgkiJKTrHJ+g2RJ4qQQnHHY2RgHxx6SLgYRYPSOJywLrdSlN4VeWaBZoJh40j\\\\\\\/jkPSIO7g5neJfrvhnKoy4APYx+ryUIsHkcbI3ZskozYtH2\\\\\\\/GfXlyElQeINj3EAnKxkVtO6SFEE8DEmEhluRtMrI7f9eNthiHfl33fYdZ\\\\\\\/p9jt7RqskaNOdxENbjwD\\\\\\\/AtWxnrHybnapuP4JtN58Pgi5QvnFpb3ktd\\\\\\\/LIqhFhYelDYJQxaZ5zPq7SSb3f5dEH\\\\\\\/iyUn8dJOpvyMH7Syl0wmSDwF3jEKcoaZwI12rCGGysu+KARBWAFWMRgwPa+gjqGuwfBrXJg\\\\\\\/AS+TYkZXI5dtcYV6SYSntAXNMPfSxcofRVasnyU3g9FKiwHzCsJVkQnegsnhXX\\\\\\\/r26VncJ9UN43Z1\\\\\\\/r8Y9CwWmvKZDRazMWM0gaEXX7U2Ccj3VfRUXgTpM9B\\\\\\\/LLtnGNWxW+UomE0U6tw\\\\\\\/n3uJM5fj3N0ZPVIzESf30UKeimc0wPdYU8DPcocdrA6TUodo1maA\\\\\\\/l94o\\\\\\\/FsdC8A1uwRucX01uJb5V56FKPE13AaltR9QX9Uv2iHZ0ZmBatnMJ99IvN0Xa02pjb\\\\\\\/2HukBBgzNV4SsKxAhjsA7qmMxbHhpQkxSS1imZqYG11F3bjp0IMvi9MW+LN+AalmkZ6vrqiE3AyL4lXNfngiBdxYAAxJlgMuIn5aDYlFIWSv4Nywmv\\\\\\\/DMplC1H4cK4IaVV9i0\\\\\\\/Li3A1LqX92kgUn+DZ0dQivZVGRzAd6YduFqGfkbu+FJWlBY8BAIpe34qBAXFyTVjuCzPs7iCW1ukRbqwy\\\\\\\/85MfQZoj7eUFkt8EWiGQ3NPcRFtFzTljEob6Grq8OC2jPySFGrcnyyz+9frjcAAvPU8w1isbPo1hJEklJoEAEQVl+XmoZR4YRCsPb3TKBLHhrhDSrxaAfcFfVdt6gtKMB1pgQ4CeDJEsAdduiHo7OmkZrGJNcrjU2Xvuv5Mw7NAQ0vBicz27oUa3VqZ6cWFWo4JHJWyw8tdwcCuVH3UqIQ7CsUl6UQIjOwYd2se310SPxnIOYvSMSyDN7LaEm6p54J5euoTeyDJfeHGwkrXkh+GKGFuqEtF8YkkdLSIWaSHzpGW0bFa9TajbM3u9AgEpih3Jqv+\\\\\\\/CR6T2tCBr9H1nsUiv\\\\\\\/aQk0eDdf363XFlZ\\\\\\\/c7xBCHxnRMwt5V9bDSvXvTB3ct1QhLqQcoAjNzZOvyLSUe+th2Wavlz1e1gzr74IMRLRjjpPf4Ty5LwdcPVoN\\\\\\\/bvFGp1VkICu7kjf3TLMEqqxIION1iRy9FE1keBaJW5+tdJ2RkT5uC761XlszTWpPgLhIAbV3ifH8\\\\\\\/nzCQFr9L35bvx4dUSG24J1ijuYyVWreAho7lAiwEeaqp+ibdv6iOcyA0CaBstScnujk78atH29k8MamdK6xs6kdsjA9K2EScaTJC83ZBXYj0TPvBH7S83XgGrMJQ21emN+6wmYNB6mx\\\\\\\/7naYMEG3ml\\\\\\\/oX8AWrBTWyoDLCAfTY3h1f1KekzOSPnUg\\\\\\\/YFm0jZGIft7q7cyG2LE2wP8C5XUfxGHAbG0CarfHwh1PMMgscP\\\\\\\/tHKsKqNrz5Vq9wuzAudGGkU26OfsLFzu6HkWbMa072MxkKMe\\\\\\\/VWiUuFd685u6UZAJBMWQM3qU+FuKkFzyH3xs52rW48L+jnJjXQf7l+GW8w3M9RiZ9pAODfh\\\\\\\/AOxDvDyEgMTDl9bBFaxJoS9yXWyseXW\\\\\\\/as7fhDOEUt0aGHXqS44qUSir+F915wG38tTTjzwfVC0kzmu+co4UMnWmnBr6BDXvBF7SVc6f2r2ywjnVbqUl4ZLCRb9TJyCKawXKa6LFRKfyinuox735TGhdqZxhM58NOZExy+TalBlIWLJqlOjF7wNrFhlL1VVig7AoBAmHN\\\\\\\/VRh3+7Up9dN6h5Guhhk3UnpKrK4aIoyuT66CIp\\\\\\\/id6w2TLMYj4Pa1NeZRZNNTF++v\\\\\\\/8QzhyuTXaC8oQvbelnm8QauyhqPzSkV3GXZ51kvR6JKm8Q0eKt0unJWw+i8DXT8sYRCrCQYyg0b0NS+IKwtIof0UeFmOUOn8ZVkkhEjRcPEmQuYlHK8GyQKhk1lICo5MMIbnJVtnKHjWSKEh5RwP0dy9\\\\\\\/q\\\\\\\/+6BQ3fhLr72Q9+CMSsLhlKZIh5\\\\\\\/xFzSIovUDobUWrdrr67KmcyAFpyLe6UOGwZMLAnYdFmRYZNnadv\\\\\\\/KbMcVsZWiEbgtrBe3uKH6v8mg2AMFhDaV2ReKfiexoaMjLzmNXRouloRls7zsP3vgGS8ppfEeYeuTy1mZ6qNyzBHtnSvhlQFF3NQwCN5Zu\\\\\\\/OEpzo5gPJkb3s7A\\\\\\\/DCiUEIWWwnW\\\\\\\/eDlFdZ1wgnwrSkDf7WS5g+9BcqjsORm9KndHwmubPqYYFPHag1ad6hcW4Y7AIQlvRxvkd\\\\\\\/eGYvWC9jQDHXgsfOj67j2rmGwZoNSSQgrtlhOHAFB2dASKYq\\\\\\\/UmYNVI3uB1qlmMln0spP8pf2TPt8KZqVVeMFLRWcusq7zeELgCS\\\\\\\/z6jq4xUmr\\\\\\\/3A00N2mDl8Rh0VOVxtKcTsUxBhkzX1Yzl3KhulVKkStnvfl2dQMOd+iEc7aRM8SEg5MqHpj35RfYTz+uxIcZmK9jkE9r9XZTUNhZJgW3InX01lHJl0kuvpVqbeystdDDqcLKz1RjcapV7jQot96XUUWokbdON7GCTZAYbDDh3VsBK8kLsxpX5l\\\\\\\/n\\\\\\\/K9by8RzYOXq2KZslRoXbk16tQ7VxSstcAQ7YmOT1MeD9k0snFK84JTV3Dt3mUpNkoMnpYTxc2fBqeeSqKqU2sb5bu6i\\\\\\\/WpLFFp\\\\\\\/+ixt92F1wh4fVUUaTWZhvkiOPto5QOTDOxks4AZHCBcbifkduvV4bzPKFXkZjAszLeeyxXw\\\\\\\/fldnRVCp5nkP9Unids4Ahb+Oud1UFJ2BWIW7Gkz2kUi8Vx+uAlBgknw6Wz\\\\\\\/liS7AsMIk3Dp4mnAM0PbcvKx8pmY7IOYTabwfubFbmZlH3+ZOj8Qx9Oidt3h+gxD0CxouQQKCoqodpEznNI6U\\\\\\\/tE0N3YX7AQ+qgBgBN9zGcpQk59CA440exbpfm6igY7NW6OhVSZqeVAGfChryin3yBNHJTt4mAvJCT9Q+eM8YGsXkL6mBcMs89VQSl1arvceCaCLC1InzGRG+P6LWSGfKnf90xPHKSUEun524kqyDtkP6w5KumKC9Y7uEgDISGAyIiOXn0OL5DUKDC7TCvaojL9OYnB0XBAHrFyZk4UbRHmPk2aHZCavN\\\\\\\/z01LE6gKBwROXqlX2wURmVLze+hnRxF3z6yu1a7WKCRwu2L9pQ\\\\\\\/9TSr4IaY204JCS8q9Inxm1l5UpoU06ikavc34QCHjX78fCRfwEdrW1V1dm7Qe+SfJh\\\\\\\/Qk48o4YqfKOfJVFacqGq+swynTF7lA2qZADGFyGoxo1xeV+L\\\\\\\/8SPSL1WIG8+f+2JscKJg84A49FpKUhMzYo+A9xeisKIuIaG3va4mz9Yet\\\\\\\/WaSOxFVDga2c6ErJlo9cA63rg8GDrs13pw1WX4IcpE+JJp+nNFIAelIyqdKwS9QZ6mEePPr3wIpFky9qj6TpWF0XwA6TtzUFeWm+BQlnNFoWAgWPjUUFjDEkxHFwygPa6VMy561Od6ECbk+sR\\\\\\\/owemGO9Lqng4mVYARZ5EjC8fo5KmpqAnp6HVVCKdk1d0waOA+\\\\\\\/NTr72bDL6RPPLG0J3qDSUYhEdHQj\\\\\\\/w0W89TkzZYCq8XE+FEwY9GttmevUGc5nUbB24SrgN+Z85RFzqgnVFCngMQXyh1vwAJ3thnQDOTmj\\\\\\\/blHCdtx2xQ21LHvzZfuGOY8x8aumrXfLHRjaZ30nWRMHIn7jiOH8tywe+U3V1RfXTVnbHNgiWEPTrAgKwD7aMiiSBe+g6TFmBV\\\\\\\/E8GiMpMBM8X8Rzkb\\\\\\\/L7EGszRUQL76DmIaaAyLIiyXZhxrAjPD6SesjRnF83TYYgrarCjm2cAL4wbkbOb1dBWBUz+7Smk1sd4nv1gap2ed12XSRX2JC89i3GTjzHuzQfjIJczzydotWdV4BG3s9W3A5B3cK4woErz59w2a7qwWl7nK9ZLRM382Bi825QKhyTM\\\\\\\/O\\\\\\\/1\\\\\\\/zH1HjiEmxVse463EsrhvykAP\\\\\\\/g0aC5DaqL8Kvr5R4aMlFtKdZocFpjjN0HqTGnN6NFv1ABpT5957vCxjv2ew1D97BMr3oe3ygs5X6nQuQqivih+Ion2RNrYB5hvKOyauNk9j2WKVT2B4Ir6zhhXczf0fEs+WpqzmW3ptE4MVxuZPj+7a6ryZQdFYEBqyyEWOVm\\\\\\\/5QJvjbk3HTDcBzsEY7cBmQTXyL+Gml7dB3juSrBB5PJ\\\\\\\/Pa61aqJv9tG8Lsj4BGEbNAmFPQ\\\\\\\/XryjREWWx6yLW4HknwzkN6prHlUIVOtvAFBleMPvCMaOWW\\\\\\\/hQnvrjfXGeAwS6GycuiFZsOoTHiCSOjcEFd8GKOBZ0GaPuyS6E0Pfh6sy4PpnZwc3PB45ulsqzf2Pqk78n68ae2aGoImJD3es6K4XIjQA4CfbEzuZDxBWDUb5Z7TtuR3rPWSGIhoglKcwWFBzPdzFrWykt3ievWdULvXn\\\\\\\/CB8vnt\\\\\\\/YA1WD1lQGOOi7oapHVFqVHpFQug8miziF62YtFo58DL6qhdCE5Oqc43Mv7k5eGVx03J2OhfYZ5n90\\\\\\\/AJtttq0PwyjufCiXY2tYQOIljgrHahXNrzj16mXcU4E+aFJMBUhLLmb4ijKVLnphIF4O2GHbOt55ZsTMI0zx+nbZjpqJi2Av7lmmsgqdtWKwtCU3HwEPkmm\\\\\\\/ExTM1bf8QKMdO6w1\\\\\\\/irBPcoEZKQGAz83\\\\\\\/aYd4SX804zhlxAkx9ilsHa9D5rYY4pGlr4XuP4iOLnNX\\\\\\\/lt7qjVyEryIuCIZY5wJFiWw+5B+qt90\\\\\\\/EpuaWY2e4XHyHTRO\\\\\\\/8m8qVoyfWDhV9zxG5YJn7LZOwgYi5WKpEFiSbV+9BOnFqeaxGJD4ivU2KFPYq5dEPFbVDBAxAo71wApjwe18wYU0inzRpIvn5fEKKkND9ShoMl63Zl1LlZiCtp5Kw3n88YqG3MG\\\\\\\/GwdU4IqC0dXAKQvMqxdwPHi3Y9A9+U1R0XCrUJ6MuN\\\\\\\/d6\\\\\\\/5TGkLOuzbHED8WuWlX3O6b4c2MD0gFUT0MM\\\\\\\/yFzvDyUJwhm+PwfnMm2xl8CK3pL0dxOYOVrUSW+NW\\\\\\\/UeLRRH3QUSixsH30T8bFYPKcue72vNtPiHYKhtCZAw+fx6WBWUThNOEC7irgtWTzLw+TarKW8wj\\\\\\\/tX8tTs59itjqazIFwWckZaocBCPZC8\\\\\\\/dDrtcwVQ55CHWhnBZsGpGcVfLstIHBVgHuHWtDaitT8Bej3jPXbhXYYsYyKM55qn7iynQEg4D9d27xvov8yF5eNJfZ5XMBeFvZX1W4YOKAMvHW0eyfGOJ0QlzWVptt8bU+l73qQLVNO5yK+tQ8Idj65t2mmqVgHbhcEq3dOl3UKF3nD9MwJ1w4ASXtpkNL1jN+I4+JxONLS4rDNlKFY9YYvzS73GAjCiuIyDMt+PiJ5OF9gtsl9j+oUBrtru0H33HR6mfOvT4XCmykrGHor0KfSL0G9su1vfLe9p8eF2UOOGttokr+4rjnjBlaXg6fsw2pfFgHw62L4MlbGjnqOIvPKMPN7OPSqtSluVK80jlLJGRrbS91RA7690T\\\\\\\/UmUR9LpRQ8FhxZxK8ewPYNBcOIF9mSC\\\\\\\/U\\\\\\\/80DeOKPjl6VVb5X3dOrE4ksndpL8HwgH9IPIqcBTaq+D6hhGPs6mDxY\\\\\\\/nmjmbWpnXD8iWT7daO7tDs+6KTr8d4GRR7eL0OylyN9UXczJ+W4Wn7UXXj3rdl2xpHpPj2huII952+mQbCduGoE4EoZXol1ICAbRcwGKAXWcJYV5JltJxFhe2kJ6gSI5P7i2wfKG2KYoF24l3yldqMpwDQN6uWcsMgE7\\\\\\\/FpLocBEn2Z0QW1iMAOsczIPhfSdlshYt8S02kjSvwZhknMXHGY9BFmdxu99asTugrxVSBd\\\\\\\/GtRQVYZXM441ICYLpmUC9Fk4N\\\\\\\/VZTayWuDXLTuPsn6n+c89YUlBW+m\\\\\\\/5gJxSe2YLpLHjyrafUh+cr7QvmXCS33xX3c2SZoJbvZh3b7+HhaX2Ieu4V1IIUJz3nq7PvRZrm1cojFRg4Q2oAW07fKlPAciUtTD8N4k4YGKDyyk56EMKN+E432n3jNAXT0oF1\\\\\\\/h9ZaBMbeeKk8N7sTiNTFtNLPGCdmXcHmGOC5HAGAc27pKGqP1nD16+qw6b8gtyVoTt8oS3Njy8OSISfE3IlJNzqs5ayUM3skyFZqoCARz3Co7UqJJ+GT\\\\\\\/PSk3xlFEAV7HVsltwmnqTijkm960+\\\\\\\/vFt24UcXv2SbpZqWItDtA3yK5emUj3aG8UiiaHApxG3aIq4gni\\\\\\\/xdvTKxoxIszvyKw\\\\\\\/uSpJFxgTzu9Ly72mvfcAeyr66R6i22Fk5J867j0yBl7g\\\\\\\/BEkUHQo6un\\\\\\\/J8XH9fmdcDYHdqUijTT9c2xsnPCyDCpsL\\\\\\\/XJI8T7zt56i+iNhLoxVjHY8EWlKFINsbwPrRZXgs0lzqvoSJUbfkl4oWRkrgN4arslJyORwNQpRZgFNL9kqgXBecwD8aW1hPOPMwNlAuWdNuXgBgQ1d9pyBhr64LR9yG3XdHZGP6fOiY8MFwzkSRQCeGKzD3PKYcplYvC4dinyvyupUsUGV\\\\\\\/X+u9i5Ep0tXvdlEo5StJlzLP9aF6f+6LAM1yW8g26\\\\\\\/3B4lgxHaqPmRQOr+MV6VvMuJIeY6+11B\\\\\\\/qkcexA4iU3HcYMNr2V7YdH+mYbTbSwcFPt4y7INLGq542yJpGk36wiXPKpHb3p7iimypIMDuyCuHgYG6Pksk32DQSqiAO\\\\\\\/ngDAZkTNNa2YDS0mggnAHZiYYKk2MiL4fSK5VK160X9C7xJHt2thRgy83ma5Y\\\\\\\/dLHqlyvDyACVvP6Q2HJtHUG+232Nk2qM0fJ7NHzLLb1LR75j7FZrkEU8IgPFqjd+QYijHJZ41n37erluJ9VWCSC79wQsBXonPIcAc7t9pNttrUGk0QUFus0Fp+RBC9b0HXOuOChxVnXCDxuv5TnoPwRrEXrl+40mgMkVchNqoIATLkhGfOq8mno\\\\\\\/He4YYUMyrcdewvf92qzzsVVMBky4+CApG3YQYzyCOsTiCZyKMSX0\\\\\\\/1PuSufy9bFBelfFwZcsa3U3XbTaGJ\\\\\\\/qAByLAHXhDS5wf71cCY3SPexk7nIh\\\\\\\/hDDJN6BsNeE9BP6vj\\\\\\\/qxOwvLd4cQeehPQBGbsJsoItun8wx6lEjB7SCgyPPHTEBnnGIzjkchSfhPVGyeScqholi2dggPoB3U1k1il4dcUeXs1a3HnmMuSuMt35ENYqaqGj85KT3RXVwM9Cx0DGPqLtQH93SIQPhKsSN3fJkgpCwa+l3A3aYI5F5X2IddSk0uiNK3HS1F7peJLGkq2YYqgHruCmt6x540UIbAr9a0kFjkW4p1rRszos54MpkVzeo4jlhC1UVLjCJYjK3Fa71BFEE27Mzjfg9\\\\\\\/g\\\\\\\/kmX+Bb6rQiilhlac9eX8LW2CFmbQUbXFJ2Rc9u0SpOpFlyxaZ1\\\\\\\/Nxy0Il94T4zfe24OYoRn2JOscCKKn2RVXdvK\\\\\\\/+o24Q4h0r4eRT03LNe784k0vToTCjtGdXVtMEoyw0CdutT27dFOOYb8TxgR4pf1Zf3EsuiFdyLmsIb1ODQ5ZRKQqoQ1KIxhzkFPCFptIcRp+5hwrJXe0Iaqv4xesZOOnnXCRKRW6AEdvvQTognX9zlltSmmioNjx8cTMnAE\\\\\\\/i2MNMKceGDhgUuKr1FXUm7iVQGijEbYAq+O6DMtafur\\\\\\\/Fpc1anJhFoRYPTO9ofvs0QQiBn9VYVl7ZJZs4Gj0eU5IgpyECIy7A\\\\\\\/ZBMqyYML9PplqBK8RcJanCD1S7rVfMykpoqXSOaPonC1FL3nbPZ8D5\\\\\\\/ZDUxmlLugKo1g0YrWnoSgjIa+mrT3bDtoD5OyB2\\\\\\\/JdlBA2NAFArK7BHD3TeM3BBxdwj3Qsjl\\\\\\\/MWohGwy5nJLmtzgbZt9Mo6eTUy2uoyw8rgJqElvUzBpNqhulpRPoJdKBhM+oioTCWerPOvmtbUcy8Ej9ZBMfyY4CEKjTZpqKLSmYl\\\\\\\/8LIh\\\\\\\/Z+EuJPFUzRR2hwTx1AEXfnSMjdYi1wSjRzDUnvfiYPDLIkz0kgJH5oFITMn4Q2ABzIoU4NfTTrTXHpcEhx9fBRlhCXgWMnFg4tD+uBgVT4xkOvPcwISm4WQ4ZvmKqcCA\\\\\\\/jJSxZ3iMFVLLJD\\\\\\\/2FGzBGkYAF80kGKCqjNrb9vUkcxwV5U2Gb67erLKdfZlt1kKKOaZytzxZPf5wcC23SCR2IVGEKyeI1k3wQb7o6aItcuTSNfyXcCOGRxgsc8IUpIFsHuM34TnuLpiyU39r73mfnMhsf8CnWTj6dP7LryY5jPVjKKIGz+H5GFlNbJAFPvC7Um55XbTdaS4enq5PlLnexXD\\\\\\\/3Pq9tEcXvEbFjk\\\\\\\/mWIHgp4fBJ6pKKcqw\\\\\\\/Wnoz6IlX+\\\\\\\/YDSMVAZRKXN14hKeundZkjJCRLK9qGg6dXEvAT1t3qmeTofPOq9Jy0Fy8UXWlJW0arx7sNnyFTvzk6yIqTevQixT24QdhDXC0t\\\\\\\/5VVWTeKj\\\\\\\/eRpdbBB7zPmaiHc5H2Yf5h6FG4gL81Et\\\\\\\/sYCtHEhGYcmngzvAbYzCp0bVSQvCsk2bRKpJItNvXv2+TWcHdgjoMvBwRffC0HW2YqXO2hfgzXvb9B+Yocq52Bn5KhwgBm0XVLVvUqkw+6OCIW3icpNohuotC971oerWUfz8kViXwbGNK1QDpM4YPbtqHPONSWmmEOp05xEfCrApN\\\\\\\/nfcNcfnjMUzbqPaGemdVuvLC\\\\\\\/64ZUj\\\\\\\/OoABdBvXGXC9pOgjmcdVz8iKPBENTy2\\\\\\\/Kg1m3+y+9eLNiD5\\\\\\\/JL3mU1z3GQsfXGG5H12ktGt4yWFINXuVAKVEY+6cZiNnlVkMEOCZ7t1rVc4\\\\\\\/ntjo\\\\\\\/xl2qgC688JIa9HLgO6QvoRNFLazgZQ1MIQjFRzGSsp6YL\\\\\\\/FqMrRmsZwEGHuUk+K685t7PMDGvinz2HYmA5xYfUTiV0gf6RofWm4qb2HdvAt3ikBAA2B3vIUtyFq+xwlO7gO1K41S+ym7uSJkTDKfToXhT5OOB3cbhAGM0p7p14928JA+zP\\\\\\\/M7FjyANqF0wFksb4JxXVkQXUUA78On1hjyY4OOP7GwuIOmrK0cVh2594bco+4au8tsl9ffmpzwRbNCpHCAxr15L04UZF+wHrNoBaaZ9s8yvULwVNskIGV695Ur5+Ok0N0IfwYSP0I6iJqF\\\\\\\/Hke+K3u1+N+fMUUV2oxd4D\\\\\\\/\\\\\\\/GUBauBIs\\\\\\\/TluAxqi9gO41pENmpS+\\\\\\\/OqFc7hslvjo5+pwsKdyor0AAAL5On8\\\\\\\/ohniwpXXhNqy8vfeXJixpyXdAR7QXIQyfagwANHiLBIwCndgBtyCC2mGJqq+A5YK+S2r3t2BLI22hacN2SoQ0W7d\\\\\\\/SZmmP6iWqGvFnJfeHQCnBSFaiMNVQwTBaBT71FWamyQ2moNY25kRz8Zyj9ex+s\\\\\\\/5cqzlSvCOqwmwlvQ0kf3ECwFh62lZC53nLgqAYClGmL0rJg2w9YEB2mO9qvtpSw50aEHg0xads2UgdtYdx79H4GklzzA9ahc3tbkUvxfJcf420rwUPCnhnfNcIxEtTEFWtZDgrpBGUtjitaYewAchJR69Fqc0OPPoQ2u1vnVAl0x42lFcdrB2PTqpb1sRguazeTnVH\\\\\\\/J7oQvqd5R0ruMV3gxfymmSLNiPfNgwHFO3e6NwN1hJj5xZ5TWfv6+gRCjRxbkir5Ct\\\\\\\/8r7hhu4H3\\\\\\\/L+mD6rcm7Wo4EedS8b9gdzffay4eDwPhA2abWlaQAbK1tTeLzLK1\\\\\\\/vGOGc27rQpl4WJEcd8YziGeibdTcdgHF9dThTQ4wj3KCWtUN5wAPrOckwMxAThjWp7Ge4RfK1zaGwpl++AIlL49c\\\\\\\/erxwdShall7lil4wE3yo9cnETaT2xmS8VYvS\\\\\\\/Kw7LHD\\\\\\\/eUq0jzfZ16mOF5JdqbP\\\\\\\/R\\\\\\\/BG\\\\\\\/WMXj7zCCK2QxxWBtF1eqJceV5U1VSEhL9SaWcESjW0aa2jb65Uq2OaqnTrHLazhEWqbBXbgRogZ\\\\\\\/0XIS0Fw1cKIw0dF7qghmjeYsxZD3KNcZ8rjZMc9FEuq5SaO+QgNyUBHTjm8E9zyHI2rQawZtQe\\\\\\\/9A4UTrUieTK4t+UOLkWdLn5Ffm99WwTqQaEtcTZo4A9e1DVU+otRfXiKYposnSm7eC\\\\\\\/CM9beYy6hFpVFjUiU1VYaioy2cTmI5BSxBzpcAB2WtaXqTBZXolOmyVas+doacUtGsXsoWmKGc+mfJJslyksXV8ALuUzHPCezS+\\\\\\\/m9H9kb6P6PQy\\\\\\\/jwKLIBpb0y8qjaP39JSoS6cgNXG8L9qwV4iMJM1ND6S1j0p2yGSWNC5wzZE8PVysyr47yFYHW1ueDQN4KKdgM+gW+Jh7yBwXy58GpS6gTYuA8jgeWhTv76oS99aWAU+pvRLtm39GpZRhSDiCv31YmfbcPkWhkL0RcFPLVqoVnZcjeqdpvJWsrpWOXAH3xUcoR6xpTJCfTpr\\\\\\\/eLkWmBT8gf8DbodAYzSIQHFO6j3cXDlRXvR9KoDOzasUv4lKUs9gpot6Jc9q\\\\\\\/N1AkYdjPybTjE5Lyo79Pz2EwyNAKsSigsJxX0\\\\\\\/A44fExIhttCO6DLJvf6Z\\\\\\\/+uuJMYiyTUBKSEn\\\\\\\/n+\\\\\\\/xldYLl8rvqM9t6yPyNr4PRWcKiGmhUe1gbeGjjoZPembcaqW18TIIxUQHbJPmxpNfY0m+nnSVl7VaZbmt+439KuqZPX54bBvGdAf7HdqkNyiOg7ChfvBcO44\\\\\\\/FXL2hkFaL0mruvJppNQnBUQnNyhXcLDhC5zMJ\\\\\\\/9LuvtHzkZDvppEwBEdNwBY\\\\\\\/qoVHReI4\\\\\\\/22gPag+WHrrUrzbDwBMsTBk5c\\\\\\\/1flfHkuf4hYaEvDxUngRJLk7rhB2\\\\\\\/WXp0W5o+Q1who94ipx0AxfbM3yDkgj\\\\\\\/NpojPZtVqcocPAY44hk5DMe1ZiMwqqOj4x1L\\\\\\\/L9BmFv8JfVbGdk3URyioWww86sj2Bzda\\\\\\\/V+nD3EbGeXGLtTy67+uiRY2Sd4g13UdOsELURjzj7H4luhXURbBSVtcZ+vCInWzVkzaXR9XY6Ii4TTZkkcUrxXb2cExlqsQlaXheuPlVq+p1die50WHY6nniFxiU9BhlNdlkz4BMAbKyWlj4eE65AhT2IvvDUzrPKm4GyjCoQlMuYTHATpaap358SfQ9ltXq8Ra9hVmB0X78Da\\\\\\\/xj6xBnBcQ2vWk1HsKeddZKlmUiWdLLwlkhhF0\\\\\\\/jHjL4I6GES6jNE8q7umjyl3j2iev0wG99AO1Kin\\\\\\\/pHwyM2q7dr1aBVmpyWM0dizczn7VnxKzHAqJgtT7bnjpe0ODQ1Bna\\\\\\\/6CengTMP0oWPbjoI3T4kbav0w8V\\\\\\\/KrGO9iVGpXgh+tAXcW9WjAavV4kG8upBU8iUEcv0w5um73qsbUeDI\\\\\\\/+3UJovG1AP46MMH7XBGPH95IyavtqESS\\\\\\\/m1qhgH7LASyHITt+Z6RX0t5XZh0yTNKAzVR4DDmfys6ogSD\\\\\\\/wQSPZby6Dm3PamoW4r6AqjIVqWhNPLDdX6CgI9VqjDV1pAJM+cWLyBAbnsF16BMTPZV8yq7hBp8pKU\\\\\\\/Grsiy2QPwyKMWX9AG\\\\\\\/q2NgNaIANtkbBAOjrDS4iHuVePS6C8YGPdNNnkEqXEwjQhm8osfwJ+30DcmPR0k1F5GupOaGlk3PcMOy56ekizkY9XdMX+9OZiDkpoUyYHn2F352dKCoERT9BZtRDZfwqZSifEVmSFbO79MKawFoPwS5kKEPjUUsGwZMkoOIJ8V9vZ7ahtMXSk1gbdyDBVXcBUouzsYtpnEPqrFlIl2CRHHGKp+dgs7pODKr7U1m8hi0C8xBs7tgyROrXWUYHV1+u5wEZH1TLOMFWa9phGXMth9dLVgGuxhEuL19ZZIREGRcAmMx6kPpZDRiUn71CqAQ6qS6XrfbjfgORUBQw9igVMzF4Rr6EC1xzZfCy4oIv8vV8BjM0GkNFhKQp0mzmREMNEtYCXM3eGR+TRY8sc3RXv8eEnqE+EfP3OUE\\\\\\\/ZUI4ZLJwMDMjHky2tBdsV+tkLjERWUvkGxGHnZFVmjP4Sz4dWFkLaKcQSkNqDOExJCW+6uzoYMbGhBpv\\\\\\\/JjE8ebROe9H6Y7uprmZM7fFLR+XTcPWXb8Z7C7S3LnBYg5l33r65Vr9eSte48fMVuDJtUJmD2YGRFeHTW\\\\\\\/T9Gv\\\\\\\/8oX0j3MzEazuYZoC3WfTqyx3eGuMxEI1EagJilw6EG1ZnQljU0o6dawDU4eN\\\\\\\/Nk1R0Xzt1Ia6xq03wNkW3cjJoLnEpCXo5iq7B6pWXx+wUpVXWASTZ+7yNbZrOuv67S2jrnDFFPyLab2WnQdonsRDYo\\\\\\\/Ymm+K\\\\\\\/Hoqxr9LXRx4Osr6drmjqGbKjjvdjMVJn4j31sHIpXM1EWQ3D47yhiCXBuDbv2NJZuNRgXTADOQixIHYX8PGLcEDkbDeyyGJdEmW4FWrVCHTiNN1g4YQ3vbzcwy5AGbLZbu33icCdKLhKZ3rk8b+uDGvnNwzwr\\\\\\\/cRXHxggh6ZYmvAhIgN+6uI75baYuq4SRTsxexg6MAEr7a7ulBWAYWV3pec7drMCRxGQVOKOQQlWB9mOpsfDU41SzTbsYo5XNjs2CInN\\\\\\\/yv4+7nWXElr2SS5jUlzOIZmWSeXaqgu+fqp+EIL6wov58i9Ii3mYzFWKm3yK6ntF8LiLXrFB3GZAcVVlvDL\\\\\\\/imkGLWo4Sldui3FT5lpf3g8L3QtILYsk9JkTx\\\\\\\/WnpM0w\\\\\\\/tKvsfIu7CBxtpcfqLGR\\\\\\\/poJX0lTIpzojiEsIExkXIB7NoRz\\\\\\\/rFEGIumY5Fzl8XvxmjClaVztIvINpixfKbG6Lmv\\\\\\\/C7uh5c\\\\\\\/KHsgYyaUUF0inC\\\\\\\/L8P8ZKodQXw77gkd\\\\\\\/5dOjZ7DHMRbJQTLyVih\\\\\\\/LzsUdt1KNC1XEnmvLOmLPdQ\\\\\\\/LbjcQGzkZpGfG66YzLYCquyiUj09R9xrPa86XVVVsyIT8oqCubvbLbLdhLVEdDl65gX2iMJNy45xB+dJZWgb59CwUKL2\\\\\\\/xUAIiaL6HDNxfYw5Wo50lZ5gBQsymFaVIQxPQWSdzVN6V15i4iK2Nmk9d3c3CnDh6McsrQJAVdEdp0vyJT7DFULgiVGcltzb8HNqVTA\\\\\\\/wvCNSW7gBVnriscoyF\\\\\\\/JtFXPPJD8Ma5TtVsGSsaMor7URL1KYmXIQLz5mslJjBzXa5xD5yRFcCLIH4k+xtvUVQIKFkW9tTVHjQnZHkTVvL8kKpMDIRkZSE1PWI7i43pJig6ddwWgbQAp1otTJXCO0\\\\\\\/KMR652wOHKGQD\\\\\\\/Kp6zOFAg57reja\\\\\\\/NFJYnppv1UgR\\\\\\\/8k\\\\\\\/7oikMbhEzE6GWH3K8EoqMtV4deFSzR\\\\\\\/k7PHY50Uq7DIxyuXh4gh\\\\\\\/rz1P++nccgRoUwvsglHI4hhg7tj2XtCWfQvhdTPsYWQdulHUTfrmtXUpR19wazuTkU9EBZuBtvjRdpA2bttbIytqfrEO3Ps2\\\\\\\/jW+dlTXbDf9pRBaAy4QfqHPDD3s4k5u+iHB9Wwk5LF\\\\\\\/X7wNDgptI5pkum31\\\\\\\/7dXKuQO07bs4SyjV8LADsW1jR1+S6ebHkPu1cTeuaNITmidgzYxcG2eFAPAhCsY5qcpKP9vAiqmg6oH1m5Osx5iv0vfTYcrfr8Rsv5EgMAwgwoN2Jqtol8IjkGn+LTE2qA7JFD+5uSz0UUNSabUPrs1zKLoAjjk8WDgLRXBK+NJ0DMu7sbu3hTbsJBVhobIrrBxtEj7aebEzq9US5gsye0DmfxCpnEfoPNXq+dPZPrVmBanPzcfbPx5eCc6FkWCr4HUFZb3peeyi3vWEuXtdOyJabc1cB2pm1QIEhvI3xTtcASFBWQUze+Oz3tQ8H+Lsa8YwCUCKQRUw8knr5bhUGUARVBGVEnPG0dS1sqCiAg7wXFEsMORAGrT4PiHJAt0fOu8C2mfGRqLzmfRQXQReCbiy7seEJnTY5ESWSQPuz4b1y\\\\\\\/TQ+GywjfzXyHTlMVhUawaKMgDpx\\\\\\\/0NZ15D2mosqI5E\\\\\\\/DwYtwQMqxIxPnpoZSh0u\\\\\\\/a9EDCRvAfzBVfnESaWDNw7g\\\\\\\/6iUJ4GtA\\\\\\\/Tc2xAshSXgToLrk6v4eynA9jEGFSMwhaVNGLLsUuqjV7cY8Ev5roe6vrFKIPSgsIygmSROJ9+CZw3Bc8JmiIQBn4SSac5N5Sqm6nQ0AcMnhujHkPiQ2CN7SVbVfS74bs49vd1+PZiyZzwO0kOgQXsNvcxD1hmDu2ppup+jR1+GjReHJNibPojp\\\\\\\/BsnD6uAwzHJV3NN7PYjiGoTWr16zVrctg\\\\\\\/KdhWYbtjPp+E8VNBA+nRYqNCeLqLLvBwaAqi5EbLJvQp6ehUVGjhBY2DMUJvksLIfSsIVPo9VeiRCHb8Kkq7Wzmn1dYGPzfUFXhb3xHJjdcY4lNbj+35NkSGJAPZAWQspd15z6gdLHah0doHDY3112xcFz3ecjVCDFKngL6MuVWLaaFYl8cU9ajjpUzQUznWSda8XGxQeHFEAMaE4ExzQ1OxMEa73xstZFSX3BMdUIOoyQitaKHwzbIXUr4NB\\\\\\\/RCODN8UgjGvmYuEn85LbWiAOqiFsGksxMICKg9V948\\\\\\\/xvLiq2WSzry9\\\\\\\/fY5DaSM4UAllLE7vZccIeIarb8yX1aHcRsMXeaqBDFkvkv07BPqdsmSM0dncCmeatyUgfWm7qHUdJlAHi4MFN6BHXSbLmwBHWVUzQMoR78iYa4VO8R51hnBpDhruZ9HN1D9NPb\\\\\\\/cOneMGyvBzrAJoxSptrrm7J+\\\\\\\/31HixO1on8Yfyee307OIK59KhX1EZNIn2tDr5nVCfxUSXjRCRKMeKn6PJfzheYHxrv1JlVUYPQe0MY++DMNaHy9sDGarlU7Y2jjgDR6S4+80F4bAatt8aqI1kiTalwmAhK7pXR4fHUA36NrVF6eravx1TH43F9ODHeu0FbDyEPobXXyhoUYaZyt78GomQMPQmGwSDJ5S7AKQ+QkMf1qUdOsfZ6r9CmSJ2QvLqIm+pfC4tEpf9zKQbJ+31VFFcM3eBMqLjHisTZM19P4\\\\\\\/SzpXqaKl+Qg6Rg9kuVqrmp+nihzXT5ZY+GQ3e+eVyUcb0JaO2gPcpElx2B0oQ0k9veN\\\\\\\/nrrz9PBinePyqt23hXhCjVZbs8HX62Y07We4QU3GjJWIfMHloN1PdqoDhbGWDRY\\\\\\\/slefyGIzPjtqTH17D9pNjOK9s\\\\\\\/QyI\\\\\\\/fgIvnxINQ7F\\\\\\\/vgBJK13gXGVCFjGUZb5wT2DNiWJuLf+bqXLyrgof7iOSFYKmevYb8\\\\\\\/hwAfvlFltfTwVl5AvfbWP6yj24NfTMnhk7dwxPkANUBldJAYRbar8PgdSSMZ3RpfmZWzBzg+Q9ZJCK5k0TIYxb\\\\\\\/Ozo6+9PlJRCi4X9YadO+Az+yDFcy2p7WWRxfLBFzp4UEaVbNC6ytpmOAAFZx2j4mX0zHjXljQxNPVESPqurDe1hWtq9ABnhenvQZmrx+DLrgeZa6WnUDiivsQhI6D\\\\\\\/twmqAv21TneDd1X9Fyroc3+afWTvKCHYlvg2HlH4n7sh1xTEmfyQrPbcB1gFh+Dmz7R2XGmdOCdtV5Ca5\\\\\\\/2U\\\\\\\/BTqgGdSQJ15gpm4UhiWZl++WRNwkdYTpi5DkFDyDLtm+AHsrgdGYFokKUTSIoLGCPP7JPE61UHO28sES4pIyNKn1jv9864yKr0mL3CCXWQBqxjjpGl+Ht7Q5bQm9x\\\\\\\/1wrY4LVCe5Wm55FdsQayF6fd9qgsCYiQCX+uhorhJNrBSZnMlK2WuZfsq4mx+T2z\\\\\\\/9hkjP18lQox4LoFKXqLVtNHFND6GYOkLPL6yatSpzmdXQ6QeZhgC2LBsD3RUaXdL+cmQToTbqYbO59hgYYzeBzYX0mejz3HEbA3DDXeL6ydCCsUFnOuuCfsXXSc9dFhF3SHMMAxYmqTy6+eJy\\\\\\\/1M07R1ANv7Mt0N0MnKN4joWNgGkuL+zrcYCgGCCArSpdTm3W8+jUm5xNv\\\\\\\/9BeNRrkUN6T3EtCAX+prrNnSTQhijDh0hPLkFAGnHs9jYFNpobwZcwsg7Wm\\\\\\\/fao1oUVMU253XxNzoX8INdslCJamqr\\\\\\\/eiQrTP1VAhUYcul5LFLXTRISIh7Bw87XyI0xH3jtWb7ouF74brUrRJCrKtyrVGKVJAgf1MMg2tF+1IsntLearNHQfu45umVzCDPQJomFPH+pSruSG4vkRfKI2+xQhOuvFugTR27cByN4sdwy8\\\\\\\/Pmd0XFMml41tVxRvsrAR1tFZpbHnVMwj6cNtEjvecif2v+i72mCfXIPUoSgQXJ4K21izJ9zCPzppT9bIjhheHV1aQ+y7IXO66zrjKeRCS+yhfxEgHcu7Hv09ThmyP12kQFmxNidYfGqTqPsmePu4FtaBv7TTYKU8ce5bgDkR1ce64fKLR5q6Ggel7oSz8uJVQllvwKZyicKiZs0x29R64Of4NK\\\\\\\/3IAkRMe5RpDFvX2bKlFZReDcPm5Ldmfc2H3Pt3e0n\\\\\\\/hR3Sef5uoN9aXlAudhac2Y8de5giRrRn8wivgCa7aH8whbf4Xg8usTkAmkMIxL6CFDnRNjtrvqGOBGPznyEl75Kp456nMZPoFPTm4TuYoxoOTydczDkBnhqwXy0hoOevofJU8bjYZdsn+GzBIWLU2vmHeXC04b70ke\\\\\\\/sz\\\\\\\/uUcOLBaL29P9J7mCfrnmDAEnOoLIyPtY3DM8VvDnYXcHXE2v7n5fw+bniuhhsF\\\\\\\/UtvCF5VXAS501RuitS7Wghso8raXHrP4qXOJ8CMTSv2GF7hLv44u0a0Z6yIeTRcXwfSWjWTQUML\\\\\\\/TGjlVve3gJZAMVq7kR078SgHMTARbWjdjWzzwxVJeTnU3E5BbVrRe\\\\\\\/DpIQytHGP3AMMS4FPk3vZJU\\\\\\\/Cj2lgLDDqEzC9rowvc5dAVFNLPDIiaztmlF2VkieD6Vwz7MpXXxTg8XjD0ze7KwWEd5xTUVw\\\\\\\/l1jpG8QKReAGvgXbNDi4DNW0IzGZIiWYbAEN1d5PM9I2bOMRgWnRPw1Euz6\\\\\\\/VuPvBGkq+uT\\\\\\\/eDKQMSPCtOOMkEja2oxdxjL5XJdpvohVhBgqtxDHidQrmF2U2ba0QOYvjaSl0IIsX5xqSiPGi\\\\\\\/Wd7AX8IqkmCY9SLwX7WWOHmGA4NSFF6V9AhW2ZEp0RaiyhKTZeQK3yaolvfs22Kw\\\\\\\/YKI5zYAdK4Vpqzsm2l\\\\\\\/nkhldg9y+pkT2MT2wkPNNs55YkQPeGlxAAHIDhuAkIWBnIE9BYt\\\\\\\/slgMuaEi83J0OKJNWObsD1EvdYXFy54L9LMHU7hg6FtqM2KQLmDN81XRfA2V0Ijx5tm7TWIH3c797rx9A+71cfufwv9SWBlrztzi7JtI4hk\\\\\\\/eRiAhdZ\\\\\\\/l8ElfouqXTO3\\\\\\\/jTOZPz5kSEluzRKfaftp5Tmeg4CW1AV3oGCTzkYNI\\\\\\\/kTfTD5qSpbf0\\\\\\\/ZX96Jeiy3KoY4O7pdRjSER3r9\\\\\\\/z73HA2f4rdMcz7Zeg7KCy7EGcWnHGcAv51seutcC22vuX0wQPX39NwIYtT3AgITnOSNSwQKXhSk2bTfVFnxGiHgewbRkVW+q677DHkeDmZfZQP5g+VC3R0Izk4QDvIRTATYCzmKCKktjyPlVGBqQAMjYKcL1K3k7Vc6lC+bVKkqRA+jMFZdsBledVWyfJHnc585lusxvAxGew6wjj3tJbcHks28p3jZG5NbyHV6pQwzQKaAj2W\\\\\\\/ZkvubBdN7\\\\\\\/zA7idsS5JrwsTHI3BAyQhWZnGcl\\\\\\\/Xhnatz1VCCUQtvfEwIfI\\\\\\\/+t9a0m9mcChpjnN+OlyPVOm2Ez3stGvkc0dPPbvUQwK4MGqZmfyRy4ew3CQYp6us4o\\\\\\\/G80ug36KndbZ9zG9YLQ0ZBxF9\\\\\\\/O+aea\\\\\\\/DNEEpC5XSFVKNZZzXtijKMXU\\\\\\\/+oRzVuOnzoeGm3CNRoeBHcEpydjDVsz2eqk6OSQDzS9\\\\\\\/dWuJfCskpmXshDR1DUtNAaNGijjl1vkv+nIW7ZKvFYsjCfDqmWu4yCeqrASjUxJPVP4t+3bjcZUVUmE1h0BZKQIhvDaGckvA7qo0KzP01qvrOaUKNWC90tvNVh96h\\\\\\\/Bsm62I\\\\\\\/8hR6DXtwiKDk9xYQJTscFwO9JaIvd44OOYLC4lKoHzXJppB01uEKs+GI4fqVdvEiNv6llTW7SUSY1dc3TNz7MBf5czhHOQfinGhV44V8PWIM6B0K0lkVyhLgY0hD3FV91fVw15xk4Ew321AnAhUy+qGXV3HY\\\\\\\/tsiXy39qPbagmTpU0bt04ePKgPB+bQyNqCD2Klq1ykPJWdqz13mf4X026QFqBEVgZklo7CIhf9qyCIVcFVGWG\\\\\\\/RFsDM8N6alb0w31l7nYiDQxNXjcLQY4JTNxgAayRzidRve0+481gVn4LlaN3Da0XNw3po52lzAYiKRFGowfM5o4UasAp1BKDm8zcFDvkRf9KXon+vZwuf0\\\\\\\/E3rnwbyq75l03XQUMh9c0uLlSW85NjDpLayjyamMBFyzEUGauQflqzPfWlfuA0ez9DvJf1anMlgHPaAusspsG+WLkSDIC3WJmGkCn+kJNiLT8HFZIvWI5JO7Jqmf+Yc7IGGCzaGPnoOp\\\\\\\/eS92qHE43CwaSzNK9TzeKlnYbeHSbjfNa\\\\\\\/MI+MYW+y4w2Y8V7b7Ao4sFXFX1haljEuKhOC5y8vxFscTK4WcDNgaoHDRVgfh1FeVbLaOnc4npWsjju6hgxs7xrFoye6vdMOpR3KJBTky8chZqVykIt2huLzJD3G6MkstY7Q8MJhZuPBFd1+8ecpGehKFf9X9qqLwA4CPa0xDo0kY8pv+1ZSQ0NdwFo8PaFl9sIRwrJceJkmy9Em5LsFxzQvSNSBvicvd2AiyELNzn5YuYqVlDHx6frylBnJI+B0YcWZ9LeeElTxBSU1wn3Npe+\\\\\\\/h4H9dTaxZpjYStOYDCB1Un8n4EO45XuA4At6G\\\\\\\/9RROrwxFPV+jr90RSxBpy56KonJkHx0AzjZANw7IFh83KSYYKq5zgT90g3YNjqNi\\\\\\\/QyVnqE82msoHJdTcuehGRJcCdi6dgtPENA1FLn24ckG17isg3zJmqwmIKHsofuatzknWKhmY+fvhK13+83GHf7kVzgHEjM+YiT1hgdaQsaEADPiTRg4GP\\\\\\\/WUPco1vjao4eOh36R0T+cGquTKH9by3FaFWXT8NjG8Qlagr46mMVF91HwWKhagAAuQ1auNkktCH9gSjfIMBWJZ2zS2ORSniBJJnGZffsJVz+W2FeUZyQQkytK\\\\\\\/W6nD9jrFNT7RNl+T7No4KqpAkZcicznR4PRGUbNjTW1gcqgCMf8N\\\\\\\/PMnUDIvMaZ2wppweLFPdpjywcx8HgRNve6Dnu4aGSQBKfQshqLkLJpUo7DwPg8tFg8JwnAOO4DvunmY67wZurdvNsABjn7xbA2UK9N2ZldNt1G3+2v3SfTpyqEYAOv5F0g4YC5ch4BPlu\\\\\\\/pTVLIF809XDv7\\\\\\\/xtpAi7rBbpwaM\\\\\\\/U31OGEGf97dhmQ8mI\\\\\\\/TExAdzU3piySAQ\\\\\\\/mqpt2GOsQcpyMx1c4SoQKk6\\\\\\\/qPDQ\\\\\\\/HIKKjYlNQpOcDdNp08d2ey9zgnjzPApWbe\\\\\\\/M4Bde0SrCJnFmwBEKGN+nGBy7LUaM3xLfDp++VE1sFSpu+OiHDy9ouXsJm9h2J4\\\\\\\/ZFTegbc2zkEzokHZxf\\\\\\\/WZZVojcko0xHsChW9nKhekJFlkQLH9Ct4No2TLTxNHnmaaDXc541K83bfnjYpkdAh3mTlJBcukhu8S\\\\\\\/MGhXeeWKJXI8RBu32AWkQiMrT6pCAqC7kPOapTycXFsXm5Udvu2\\\\\\\/00HGnB7GiS6YdAcq6N3iqT6QjWa5o86iCQ\\\\\\\/DnKOB\\\\\\\/i+cSrC4\\\\\\\/1EbsMbK7Uu\\\\\\\/1rqeHOckXs6bhsq71fwBhZBkldefwHuUjod3p2PcLyb8uDz+0X7+J7SniQD3eTn8CTe53R+yPIB+083mMduogFpGIU6yF2kcNsI4yL1qXMau4ALNa7PWMNq02sqcRKih7K7JOXfjB1HodsSeh3zPpMXIaUarSR2mFqcUpUL4niLRf9DSyt9NqhqXmuAsNmG3hOw5jWMcrSFl9DG7TdMRHw\\\\\\\/WreYzFoK0+PIfEsLvXIzeeacjR+jtdP9RgDeW7BJAsIMCr0ij05EnA7x26Ipe31TZPy0W94UhtYhHk90+OlTTgLmA\\\\\\\/+hTC41ZhIgUzVvQZlEfmUoW91CXScAQCWQ47YC1G35cZxerll6eKdXuFxAkpGxcBoZgmtivFXMrlT2PIjWpcizRGvqbT5J1C8aFp0zjftb1\\\\\\\/ys+Vrw8Uwl4NjKOxSdXKHYqp0c5heb5Pt8oiULbdkFxaH1rDH1MG1Ntx\\\\\\\/YiBF61ywICGHzpdDKIWKaGFA4P3+Sc0IN13DaueraH8igZSsVDqtB2CDHkqm4Unt4HUVL51+L4M954B4m7puihUHJIko4aoDkAG69fEI3wDjNkcTpyACRYsv9HXLzy8oB06XlLvw4QV488nBpY81H0xUAWzhGjqDqCxbaTRVJIw8y0UVlIuSo28peNkjz5GADqXhNUM5p9\\\\\\\/U2sbB475KepcgILZdmmZyQebnskTOzTRfH8BDLyCaxiI12Cu1bX0l\\\\\\\/WCyhERzZ1cMekNinuGfWDgocN7JY0cC9o57dLl5T0u7OFHGTCTIWfNc5QgNWxQRNoGhXKRTJTnhsFTAC9JmlgZpMCI3wAAzP9bxfp6yeLmTyv7gGGSMBDodv9Syfu3VltxCFPQXxqcHnGidkc\\\\\\\/3CRtyUazfvZesJauUriAaYPLlM5ZidrnE2MS\\\\\\\/WzB+JRAr+nvIBZMWTZmsYGm0DO6SGPmVFRU6rgVsV+yZow5eyt160NTyNmnpMNik6FaKc0r7Vq0HNa3RNOXEjracIs0JszgFVW7fM3ie43BzHvtF+eGoOeklU4rjll434bfl1DS30CpOProzaLHFNKdL8gwfvMrccu9Fe1rvq5TwfYKk\\\\\\\/++3uerE\\\\\\\/9pw5G\\\\\\\/JW7b3JeIcRpqX48hmxYcSd+XmfaoE2F2Yy\\\\\\\/u7V3+\\\\\\\/viuto9SQHO\\\\\\\/ELd+D3JrLT8zJyFrQhCaKMEjpTeOe1ZEV+2hHOobE31OYdDqtHGP7e3hCDK4bo6NDauwQ9DpL84U2rZIpsDcEOb7iepz0TGT+vh+RUer63kW6ytD0kOd02hp1b2UGbCY+1doQrKBZ5lw3dtbm4O4ZF5c48K1nZ0psJ7zdYg9+3WJXM06xX1a11iNsck+yk+V7+AUT6kQ9TpSp\\\\\\\/N\\\\\\\/cayyafwMO1QC7bMejlrsI77ryJXJzwHnSaIV1D6ixxZkpEPHcwJ\\\\\\\/hvv0VaFtDd12PuGL8RrzKKYq\\\\\\\/\\\\\\\/WeWMOIpz4sbsZ1S\\\\\\\/jGNYHgXbr0lTST0VDlVev2jikw75egCOW+iB3LBoFfrVGTDk8sCsBlj9d2hWhGXg6ZzTRcs2u5hN8zMsAXYiw6iIVkCLneYwYsWCTA3OHaQnfx2QDL1xg\\\\\\\/ixTDM11pLecq8aqLpQ89aPv8VEd9UZrlAtA0ZtBub\\\\\\\/pEHJJ8RGK5H0wzJedGcVfF0W+zFpzU7\\\\\\\/opKLOMceOoZeToE1nzL3KL2ArBfgTDJnraHGS7ZK+p+5kOSBoJt9\\\\\\\/Yi3+ZYldPt+V9Y9pAiU4CPgGUn8cGQCoNB0oWZidpIvkAjHPZWN0w3l3+B7SgVHUBCe4umS9u6lGRGqStrFvu8lFHhgShywMNVFiEbFzGWtrpA5cF9\\\\\\\/UHL85ki7usThx\\\\\\\/1tfkDA6HuEg8Igu+ShvwBhZ7+V+TXzLAF+EcnD89Pr\\\\\\\/g7zIjZ5noKvk3BO0by+kEBIqBUlek1eMOqr1c5vhqIA7YsARKipiz4iUCNR1TqTUf6rSoZY1VPs5tL5hdJUpDV1rBmjT9Etr64KtfVLBmx\\\\\\\/kBKtBKclItuw01kHp+lJxiFEBcEUJlR10rVfvrsB1a9vgX9vw9qfKnBA3tEdqFmdKcri9rOMusZQ65ZuecG9Ts48V5QrI3d4UQ\\\\\\\/LZBrpfr1r0XvojXF5mUUwJ5pqcj6jlerHIgKeMsvKJ65X\\\\\\\/PHbIl84atEojydGUhEcoOmAwmzgiuleVUe1RWDGLPswggw0vVeCmUNT4uewTf8Au2EG\\\\\\\/8SgDA5VG23+q4qSujakEH6SmtCw\\\\\\\/G1LHIS82v6gauyAqKPIFD3ePxQB25IwQ7U6\\\\\\\/VTLxl43gjNqysPoiHhAcDTkI0aio6Vx1xMGfZrk\\\\\\\/HL4hJdWS8akW+HtCPYO9ZBbGqHEe4DhjS1aTHEr0QsrmrAK3bL8ib0CrlTzrPqhrhv7La0rPk7z3ip8z6ZFPUWQW2QqS21ooax6axun3XUJQtKp+ipdnpjk4pHdyQfKNCifE0NDcFEUghJgxBhZmFru3Z4UWFJhwZ3W66xH3jkO1jAo0poIc2qF0W2hP7rKvP5HlmGiQGe1a62RrX2rt8CDuIKw4rKlfq+vEuZMupZ3U1VAFT2zD7tDGsvt3VxGAH2hAi1BeNXm8MOQg6ZwRTjC75CRwR8zVMYb08ancP7Efk6GpJsBalh4djPvWTKGsEcG8HHxqlM\\\\\\\/dbgf6X1r0AtR\\\\\\\/VHSTuXmd6q+nJMy6iBzOqJLsi4glNuzBNSMi\\\\\\\/4\\\\\\\/RHvnFZWGNRdC3qgYt98L8C7BDhQuD7zxXhMxjweD8trvvEIw5ljpvjBNYf5F8w6Q239CRJPxTnkv64ngJUKvoD3diP6swKFE7AlZDp2L4jBN2uO+ikWQuIQizLrgJDgO2b5hczs9yBgN8nFB6HRRXHakUqRjgskaXz0Au6GVEn0idGHyRfn2a3sfE6rskPf4ojmCcwxTbrS8i\\\\\\\/snTsQBtpGoEtJLuIdyKqI\\\\\\\/iK1haVP43XMfAWZpfCgZ+A0mgZWJvckp9sY8BZRc9mvI\\\\\\\/nQ9qFVHimD1rcWWoxlEw2mD3ByZI8LZ52hs3GTdguEMUgaGJwoKKQvmEhhbzw0DhbGvrNuQwZxQGPwILzUdj7ykEXG56D7ZiZdTrt2UipaW2o+lKvNzgl6PxXm+Axx7ocCDWItSEcCgn7ukGUoSWY1rE48EmxzcyEDSeduBtMuhmZ71pUZLCTLCNhui84laJLlCBTOD+qbrZK2P6gJPvVvwZzzdNXSYMJDieCMJXU\\\\\\\/95sYH5YAihpE2R+xUaM92l9CSY7+f7qpEV+wLHYwGhrWQlIvxZP7c4C1NQtFzxqqUuICUO\\\\\\\/ngVDbv\\\\\\\/lT1b9dUKbBC5FbtTii4qRq0IDDxQWVLU+3AgBf2ZzRwmUYNIVXD1NVvsfxnKJUujNRG0E8+E2bw9ot6LtG0uiWcA1WZYgugK40PNrR0zy+QPb6RYx26YBAreyLZyp5r4Tle\\\\\\\/Zfsf+Y6CMVf+eTG+ucq6ds1ZJI2j85NUxUJYHaH4X\\\\\\\/Rm1G8mqAHQJ9e1XMhqKaP3D2pmoq3Q56VyGeru3L8h902yE+3LIBtDocQI65+OK\\\\\\\/P8xwOVjzhzna5AwVrBPfJwVN77AoNjfYJTdgmZNmr0sAop0WfbnAQG2+Aw6BE9EMnDNfB9imb7dhvr01VSieQEtUDnB2J3eUVDRLZ0QT2avmkmdelTEk6usB4W0Xzgf661KAnIR0QexZsGcx0Xgs87E9e\\\\\\\/1Ny4nrdh2zPjcpTGylilMnQ11R8a13wC8xy9ysxvLkuUyE9elymC5MUhSPSz+2YVhLgUTmd6apM9\\\\\\\/sytoX4oRPGGxteVOIwOxfivrcgk\\\\\\\/lxw4IZFfNQit0VDNjdKrIDtDT\\\\\\\/is69MQBYe7hDhWeYrZIJRHFClSeRJ\\\\\\\/BNXj99BJ54HvjW8Ku1RtUmJ\\\\\\\/XDGhN+NRLCE4NO\\\u0022,\\\u0022iv\\\u0022:\\\u0022GBBzFD8ksYqoh2JSQb4pNA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022472db9214b09e2d4fc0f295cdc7af8176e334b2e1ad1e0e75f7d40b78dbd08f5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ndRJxf\\\\\\\/vbdHmRYTEK5642GdMxR\\\\\\\/9OpX2LivLFv4JcEhI+M3+0Lm415nc\\\\\\\/a6iNN5ZQkuawv409to4jG+9i7WS8nWnfxeYBkg7D\\\\\\\/6PyPrBm1iGF1YVL67sPlgTKlyEwD9oHg1K\\\\\\\/VKrqWSOIkT++hKeWsfdo2HYXH6xsTVMgVk+aZi5b+mh5TJiWyzm9QVFOZZ75fRJZ+KEN4qAwFse+JbnxXgeZSxrrZXplDWI+3xyeFShr03SPSUpKdoZR1o7nAibRiR\\\\\\\/p5sYqR9xKABOttv23ICFJEezQ+ZuYjw9PlMR4iXuNSQJo2TLtnC9YJYDEdiBVuJfGaNlHYApXX3ZSItIbRyjqkQgSoC0sfNkA9g06aUEwOkplxgKNkL9IrhKpABdPItPCGNSWH6omJ6LjqWM7YdjJcVea66qPBuok07IPUpl7uGZW99957712w51E4GLtEbSKt8CnI1eCciRipE6urJqwCqW6+93UpoH4J5+0IwbjIdAkgNxXP0dFo+eb5kfNPhar9tVyONDeORz8mBPkuv6ESkO+8qjmcXB9NpwWhgpBe51VnSO13vG1GF8Z6hYqR2\\\\\\\/pAAbXJ9cpeKnQzketeuXIzs0B3E\\\\\\\/\\\\\\\/2sgSfTCIuPoXtJP9Uc5ny5CKP7S9sldaAxYuWBQlrCs8zAaiMdru7tIHQBJ4XSspfLzxUzsGP6qfx+dgUAYrAOdypNrpD8qgRq9hA4D0T6\\\\\\\/KYMHcV0jpI3xOMDa3CFMkXFnkpYFu0GXqhJQgq5SmWtX18QazN4k2H7licW4qfG26SDjCry7CIUuGbMPPiN\\\\\\\/cpe30bkN1sXluBjlW1m\\\\\\\/PZov\\\\\\\/n3Gmdl3e2gECH3uhjPYhG2VZrRGbeP0rc\\\\\\\/78mt4kcydGRbyyJFbgZPPGpSQwIukIU90F4t1Luycf43ZyhJqb9dZSLeLK53aZqvU4BDPLTKLWvZajN7DDcRazP48EriWdlqH3YjNTStuheLzpAesFkWFufuy8fQobkeAKGJCMt3yUBf0vrivSc\\\\\\\/K8PhEcUtkf9sLkW72KmKubRJCpezcsyddQuI6v\\\\\\\/YBnrehvedsMQwg8D4D0w1eV0tSMk6DEVs0zWqDl2FyGCFQi+fmBWI3sj29yMZsyXKR7uWleEtfRFjPcENNX6ZWqkfgNfl0\\\\\\\/8+wX96E84edhhYpwuRX3S5LJpYIfomooYw16qsLQJEmmSOM4+enfBoB9t8bBXWUwkuaYUL8TgEQZV1Gn56ceMdpul6rYI+hsU\\\\\\\/m2NRpylWJg3i+Go4GUNJJB4\\\\\\\/gQRKFTVt7gKPXzN70igRaVpbl1Mqi0qOKwW3lo7HTxTJkKHORKJzdU9u4tt1T6dmzuZecTGzqBH9wBgcBZ\\\\\\\/zDDzrW1NJkfmmE3HjLI3Ij2cMocz7CDWjTGe0Blk9wGlRV6CI4caHEhMKaZT\\\\\\\/SXJ4\\\\\\\/xufNyRJiNpCsGeMQI6PcXK5GHhUxo5\\\\\\\/6Qf5pCangbzEh+yNLpPuPZG5HJD8eYgr57qGhg+fZ+UGttfA\\\\\\\/BVyGKHhcUrFUewp6ccXPmPr6t86kpt60Wo38vOsaV7cbqHcP0hwleYCQv\\\\\\\/KAOBizsDkljJIUpJ+dO\\\\\\\/NGUIGnEk+OyXN4WOpP6gbkii2Gb1qVsnIhzB20deR4xYXWwLFZOQ2rWcpb2+bsi\\\\\\\/mF8ashaFxQ3KtNBEzLzE6ccFX3rrg5+0L2yODFp9PszfQirQD+aVsyoaHX3\\\\\\\/EGjoaiwj9icwZXk+jWokzdzOw8gTR5x822K\\\\\\\/ftGOz8VJA7Dz3XE3\\\\\\\/Uo\\\\\\\/xtsrbGWFH802i6p9agvIrxhSb1sSQDymYiG5luj9FWk2zrmQmfPjCExbvPACliFYsE+43zaoJ8KLYY44h9t8Gi5xi5vmhq1Z5QbtqpEqnNeSsS3QPAid685KFr4cqhFz\\\\\\\/bHK9MaTkOjOsfDSu0JfgPq3+mJaibYP3wbpAT48Kq6i513FgBr+O0hxY7nClbJxmq2Tk\\\\\\\/QNVscmqxo4dnqc93Vzl55QkcFqZmIjy4zd2BfNdi4OKXuBSqnRFzAvnPtPmdg9rHJiUmDsJyLxkvhx0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022aHRxVw\\\\\\\/SQ7mNCJ\\\\\\\/iMJUI8g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221b5c412f56050145beb4cb4b8509f88b00fff5e715dd98b24ea5e2a15be504ff\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459440 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/597" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j1x2GEuIEwAKl1Yu8uP3T64_cA0qd2u2KM0dMxMxtOM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/597" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-evsd0-IzFqhTioblym_Wikl3Lw7nfAgjPuKnV461WrM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/585" title="Assembly of the Synaptonemal Complex Is a Highly Temperature-Sensitive Process That Is Supported by PGL-1 During Caenorhabditis elegans Meiosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/607" title="Plastome Sequence Determination and Comparative Analysis for Members of the Lolium-Festuca Grass Species Complex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459440" id="node459440" data-pisa="ggg;3/4/597" data-pisa-master="ggg;g3.113.005546" data-apath="/ggg/3/4/597.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marie-Pier Scott-Boyer</span> and <span class="highwire-citation-author" data-delta="1">Christian F. Deschepper</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">597-605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005546 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459440" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marie-Pier Scott-Boyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Pier%2BScott-Boyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott-Boyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Pier%2BScott-Boyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian F. Deschepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BF.%2BDeschepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deschepper%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BF.%2BDeschepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:deschec@ircm.qc.ca" class="" data-icon-position="" data-hide-link-title="0">deschec@ircm.qc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/4/597" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459440" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/597.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459440" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/597.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459440" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/597.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459440" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/4/597.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459440" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Although gene coexpression domains have been reported in most eukaryotic organisms, data available to date suggest that coexpression rarely concerns more than doublets or triplets of adjacent genes in mammals. Using expression data from hearts of mice from the panel of AxB/BxA recombinant inbred mice, we detected (according to window sizes) 42−53 loci linked to the expression levels of clusters of three or more neighboring genes. These loci thus formed “<em>cis</em>-expression quantitative trait loci (eQTL) clusters” because their position matched that of the genes whose expression was linked to the loci. Compared with matching control regions, genes contained within <em>cis</em>-eQTL clusters showed much greater levels of coexpression. Corresponding regions showed: (1) a greater abundance of polymorphic elements (mostly short interspersed element retrotransposons), and (2) significant enrichment for the motifs of binding sites for various transcription factors, with binding sites for the chromatin-organizing CCCTC-binding factor showing the greatest levels of enrichment in polymorphic short interspersed elements. Similar <em>cis</em>-eQTL clusters also were detected when we used data obtained with several tissues from BxD recombinant inbred mice. In addition to strengthening the evidence for gene expression domains in mammalian genomes, our data suggest a possible mechanism whereby noncoding polymorphisms could affect the coordinate expression of several neighboring genes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetics-gene-expression" class="hw-term hw-article-keyword hw-article-keyword-genetics-of-gene-expression" rel="nofollow">genetics of gene expression</a></li><li class="kwd"><a href="/keyword/quantitative-trait-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci" rel="nofollow">quantitative trait loci</a></li><li class="kwd"><a href="/keyword/clustering-coexpressed-genes" class="hw-term hw-article-keyword hw-article-keyword-clustering-of-coexpressed-genes" rel="nofollow">clustering of coexpressed genes</a></li><li class="kwd"><a href="/keyword/transposable-elements" class="hw-term hw-article-keyword hw-article-keyword-transposable-elements" rel="nofollow">transposable elements</a></li><li class="kwd"><a href="/keyword/structural-variants" class="hw-term hw-article-keyword hw-article-keyword-structural-variants" rel="nofollow">structural variants</a></li></ul><p id="p-5">A significant component of gene expression variability in populations is due to variations in the DNA sequence (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Schadt <em>et al.</em> 2003</a>). Accordingly, genetic mapping studies have led to the identification of quantitative trait loci (QTL) linked to the expression levels of particular genes within cells and/or tissues (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Göring <em>et al.</em> 2007</a>). When the expression of a given gene associates with a genetic polymorphism that maps close to that gene’s locus, the corresponding “expression QTL” (eQTL) is identified as a proximal eQTL (also called “<em>cis</em>-eQTL”). In other cases, when the expression level of a gene associates with a locus clearly distinct from that of the gene itself, it is defined as a distal eQTL (also called “<em>trans</em>-eQTL”). In the case of <em>trans</em>-eQTLs, it has been observed that a single genetic locus can show linkage to the abundance of the mRNA transcript of several genes across the genome, forming so-called <em>trans</em>-eQTL “hotspots” or “bands” (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grieve <em>et al.</em> 2008</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wu <em>et al.</em> 2008</a>). One example would, for instance, be that of a <em>cis</em>-eQTL regulating the expression of a transcription factor, which in turns regulates the expression of many other genes belonging to a hotspot of <em>trans</em>-eQTLs.</p><p id="p-6">Contrary to <em>trans</em>-eQTLs, investigators typically associate <em>cis</em>-eQTLs with the expression level of just one gene. The premise is that the <em>cis</em>-acting polymorphism that is located in close proximity to the gene is likely to affect the regulatory machinery of that same gene and that machinery is unlikely to be shared by genes other than the ones that are immediately adjacent. Nonetheless, if the genome contained features that could influence the expression of several neighboring (and not necessary immediately adjacent) genes, one consequence from a genetic standpoint would be the clustering of several <em>cis</em>-eQTLs within a narrow genetic interval.</p><p id="p-7">Although clustering of <em>cis</em>-eQTLs has not been reported yet, there is evidence that gene coexpression domains exist in several eukaryotic organisms (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Michalak 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elizondo <em>et al.</em> 2009</a>). For instance, approximately 20% of the genes in Drosophila are arranged into clusters of similarly expressed genes, with the clusters spanning intervals from 20 to 200 kb and containing 10 to 30 genes each (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Spellman and Rubin 2002</a>). In mammals, coexpressed genes have been reported to cluster both at either short-range (1 Mb) or long-range (&gt;10 Mb) levels (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Woo <em>et al.</em> 2010</a>). One particular case of short-range coexpression clusters concerns that of conserved clusters of paralogous genes arising from tandem duplication (such as, for instance, Hox, globin, and major histocompatibility complex genes). Beyond these paralogous clusters, it was reported in humans that the overall level of coexpression of genes was greater than expected by chance when the genes are located within distances smaller than 1 Mb, although the level of expression did not exceed that of more distant genes by a very large margin (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lercher <em>et al.</em> 2002</a>). Likewise, in other studies reporting on the clustering of co-expressed genes in mammals, it was found that coexpression rarely concerned more than doublets or triplets of immediately adjacent genes (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sémon and Duret 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Purmann <em>et al.</em> 2007</a>). In such cases, coexpression is generally believed to derive from the sharing of one regulatory element by adjacent genes. Nonetheless, clusters containing on average two to six coordinately regulated genes within 1-Mb intervals have been observed under special circumstances, such as in fibroblasts during replicative senescence (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhang <em>et al.</em> 2003</a>).</p><p id="p-8">One limitation of many studies to date is that they have been not been performed within the framework of panels of individuals (or strains) with well-characterized genetic backgrounds. If “short-range” clustering of coexpressed genes could derive from physical elements in the genome, the impact of the latter would be easier to detect in situations in which they are polymorphic, such as in animals from genetic crosses. Moreover, because clusters of <em>cis</em>-eQTLs would all map to precise genomic regions, further analysis of these regions might reveal the nature of the polymorphisms associated with coordinate changes in gene expression.</p><p id="p-9">To complement previous genetic studies reporting on QTL linked to cardiac morphologic characteristics, we used Illumina microarrays to obtain the profiles of cardiac gene expression in a panel of 24 mouse recombinant inbred strains (RIS). When performing linkage analysis to detect eQTLs for all detected genes, we observed several instances in which three or more <em>cis</em>-eQTLs clustered within small genomic intervals. Because such clustering of <em>cis</em>-eQTLs had not been reported previously, we used our dataset to analyze the characteristics of corresponding regions in a more systematic fashion. Likewise, to test to which extent this observation could be generalized to other tissues and/or crosses, we compared our findings with those obtained in other tissues from either the same or other mouse RIS panels. We found that clusters of <em>cis</em>-eQTLs could be detected in all the tissues from all genetic mouse crosses we tested and that coexpression of <em>cis</em>-eQTLs within these clusters reached very high levels. Further analysis of these regions revealed that they showed enrichment for particular types of structural polymorphisms.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Detection of eQTLs in hearts from AxB/BxA mouse RIS</h3><p id="p-10">The AxB/BxA mouse RIS originate from reciprocal crosses between the two parental C57BL/6 and A/J inbred strains and were derived from 20 generations of inbreeding of the F2 progeny of these two strains (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Marshall <em>et al.</em> 1992</a>). We had previously used a set of 24 RIS from that panel to detect QTLs linked to cardiac left ventricular mass (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Llamas <em>et al.</em> 2007</a>). Using the same 24 RIS, we extracted total RNA from cardiac left ventricles from four male mice for each strain and used Illumina MouseRef-8 v2.0 BeadChips to obtain the profile of gene expression in the tissues, as described previously (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Llamas <em>et al.</em> 2009</a>). The raw data were obtained by using the BeadStudio software (Illumina) and imported into the R programming environment. The data were processed and normalized using the Limma software (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Smyth 2004</a>). After filtering out genes not detected across the chips (by retaining only genes detected in more than 50% of the biologic replicates for at least one strain), we selected a set of 8725 genes for further analysis. Processed data have been submitted for public access to GeneNetwork (<a href="http://www.genenetwork.org">www.genenetwork.org</a>; accession number GN421).</p><p id="p-11">For genomic mapping, genomic DNA was extracted from spleens of all corresponding 24 RIS using the DNeasy tissue kit (QIAGEN, Mississauga, ON). All samples were hybridized at The Jackson Laboratory on the Affymetrix Mouse Diversity Array, which contains 623,124 single-nucleotide polymorphic (SNP) and 916,269 invariant genomic probes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yang <em>et al.</em> 2009</a>). Signal intensities were extracted from CEL files using the MouseDivGeno package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Didion <em>et al.</em> 2012</a>), and genotyping was performed by comparing the intensity and contrast of signals in a given line with that in the parental strains (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Simecek <em>et al.</em> 2011</a>). In total, we detected 977 informative SNPs (meaning that they were polymorphic for at least one strain among all 24 strains from the panel) defining intervals averaging 2.59 ± 2.95 Mb. The average value of the r<sup>2</sup> coefficient (calculated as a descriptor of linkage disequilibrium for all pairs of adjacent informative SNPs) was r<sup>2</sup> = 0.8 (where 0 is the value for perfect equilibrium and 1 is the value obtained when two markers have identical information).</p><p id="p-12">To detect and map eQTLs, the data were analyzed with the “R-QTL” tool, and all other statistical analyzes were performed with the statistical language R. We used a detection threshold corresponding to a “logarithm-of-the-odds” score of 3.3, as suggested previously (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lander and Kruglyak 1995</a>). For each eQTL, we then determined whether the transcription was regulated <em>in cis</em> or <em>in trans</em> by defining <em>cis</em>-eQTLs as those whose peak eQTL was within 1 Mbp of the physical location of the corresponding gene start. Of note, artifactual detection of eQTLs might theoretically occur when polymorphic SNPs occur within sequences corresponding to the probes used by the microarray. Although the vast majority of SNPs within probes have been shown to have no significant effect on hybridization efficiency for Illumina microarrays (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schurmann <em>et al.</em> 2012</a>), we nonetheless used data from the Sanger website to detect all high-quality (score &gt; 100) polymorphic SNPs, compared their positions with those of all probes in the microarray (as annotated in GeneNetwork), and verified that the polymorphisms had no impact the eQTL analysis.</p></div><div id="sec-3" class="subsection"><h3>Origin of datasets</h3><p id="p-13">Lists of transposable elements (TEs) that are polymorphic between the C57BL/6J and the A/J mouse strains were obtained from two different sources. The first one corresponded to a supplementary file from the recent publication of <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nellåker <em>et al.</em> (2012)</a>, in which the authors developed a comprehensive catalog of TE variants across 18 mouse strains (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Nellåker <em>et al.</em> 2012</a>). The second source corresponded to the MouseIndelDB database (<a href="http://variation.osu.edu/mouse_indel/index.html">http://variation.osu.edu/mouse_indel/index.html</a>), which reports on structural variants that show polymorphisms between four inbred strains (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akagi <em>et al.</em> 2008</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2010</a>). From both databases, we extracted the locations of elements that showed either “insertion” (<em>i.e.</em>, present in C57BL/6J but absent in A/J) or “deletion” (<em>i.e.</em>, present in A/J but absent in C57BL/6J) <em>vs.</em> the mm9 reference sequence of the whole genome from C57BL/6J. For simplicity, the aforementioned publications used the convention of referring to these two types of structural variants as polymorphic “insertion-deletions” (indels). Although the number of polymorphic TEs reported by MouseIndelDB is lower than that reported by <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Nellåker <em>et al.</em> 2012</a>. (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/005546SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS1.pdf">Table S1</a>), sequences in MouseIndelDB have been characterized in greater detail and contain useful annotations. Of note, the vast majority of TE sequences present in a given species are “fixed,” so that in mice, they will be present in both C57BL/6J and A/J and thus nonpolymorphic between the two strains. To obtain data on the abundance of “fixed” TEs in mice, lists of TEs located within all protein-coding genes were downloaded from the TranspoGene database (<a href="http://transpogene.tau.ac.il">http://transpogene.tau.ac.il</a>). Recombination rates across the mouse genomes corresponded to the values calculated in a recent report (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brunschwig <em>et al.</em> 2012</a>).</p><p id="p-14">Data on genomic binding sites for several factors were obtained from several sources. A list of 33,172 regions associated with CCCTC-binding factor (CTCF) in chromatin from the hearts of adult C57BL/6J mice (as assessed by chromatin immunoprecipitation and massively parallel sequencing) was obtained from the ENCODE/LICR database of transcription factor binding sites using a custom track in the UCSC Genome browser. These data (Release 2, April 2012) correspond to the results of experiments performed by the laboratory of Bing Ren at the Ludwig Institute for Cancer Research. Regions corresponding to binding sites of transcription factors Gata4, Mef2A, Nkx2.5, Srf, and Tbx5 in cardiac chromatin (amounting to either 16,753, 1337, 20,573, 23,806, or 55,582 regions, respectively) corresponded to those published by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Schlesinger <em>et al.</em> (2011)</a>. The 10,486 regions corresponding to the abundance of acetylated histone 3 sites and the 3596 binding sites for the p300 histone acetylase in cardiac chromatin corresponded to those published by <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blow <em>et al.</em> (2010)</a> and <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schueler <em>et al.</em> (2012)</a>, respectively.</p><p id="p-15">Lists of other eQTLs obtained in RIS were downloaded from the <a href="http://www.genenetwork.org">www.genenetwork.org</a> web site using the Genograph tool. In short, the tool uses WebQTL to detect all eQTLs associated to the expression levels of genes within a given dataset (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chesler <em>et al.</em> 2005</a>). Datasets used are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS2.pdf">Table S2</a>. In addition to data for whole eyes from AxB/BxA mice, we used data obtained with five tissues (eye, kidney, hippocampus, hypothalamus, and cerebrum) from BxD mouse RIS. The latter originate from crosses between the parental C57BL/6J (B6) and DBA/2J strains. After analysis, eQTLs were selected using a false-discovery rate threshold of 0.2. As for our own data, we defined <em>cis</em>-eQTLs as those whose peak eQTL was within 1 MB of the physical location of the corresponding gene start. Using the same parameters as for detection of <em>cis</em>-eQTL clusters (<em>i.e.</em>, boxes containing at least three <em>cis</em>-eQTL separated by maximum interval of 500 kb), we calculated for each pair of analyzed tissues which proportion of <em>cis</em>-eQTL−containing regions overlapped between the two datasets.</p></div><div id="sec-4" class="subsection"><h3>Selection and comparative analysis of genomic regions</h3><p id="p-16">Clusters of <em>cis</em>-eQTLs were detected by defining regions in which <em>cis</em>-eQTLs were separated by maximum distances of either 250, 500, or 750 kb. Control clusters were defined using the same maximum intervals between genes detected by the Illumina array in mouse hearts and imposing a maximal limit on the overall size of control clusters to obtained clusters whose size was not significantly different from that of matching <em>cis</em>-eQTL clusters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS3.pdf">Table S3</a>). To further verify that both types of clusters had similar properties, we calculated the number of “Entrez” genes in each cluster using the biomaRt R package (version 2.10.0) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Durinck <em>et al.</em> 2009</a>) interfaced to BioMart databases. Coexpression levels were quantified by calculating the absolute value of the Pearson correlation coefficient among expression levels of detected genes in the <em>cis</em>-eQTL clusters and compared with the coexpression levels observed in two other kinds of boxes: (1) control clusters (whose characteristics were similar to cis eQTL clusters in terms of size, number of genes detected in the heart by the Illumina microarray, total number of genes, and overall level of expression of detected genes); and (2) random regions (corresponding to boxes of similar size chosen randomly within the genome).</p><p id="p-17">For comparisons of the abundance of structural variants and/or binding sites of regulatory factors, the regions analyzed were slightly larger than the clusters themselves and were selected by adding flanking regions of either 250, 500 or 1000 kb to four types of boxes: (1) the same <em>cis</em>-eQTL and control clusters defined above (using maximum intervals between <em>cis</em>-eQTLS or detected genes of either 250 or 500 kb; and (2) regions with the same size as the previous ones but either centered around single <em>cis</em>-eQTLs or selected randomly throughout the genome. Data calculated represented the number of features per Mb in each different region (<em>cis</em>-eQTL cluster, control cluster, and random region). For easy comparison across different types of features, all data were normalized by dividing them by the mean number of features (<em>i.e.</em>, structural variants or binding sites) found in the random group. Accordingly, the mean normalized number of features in random groups was 1 (± SD), and the values in other regions corresponded to “fold difference” compared to random regions.</p><p id="p-18">Motif searches were performed in the sequences of polymorphic short interspersed elements (SINE) and long-terminal repeat (LTR)-TEs, using the HOMER bioinformatic package (<a href="http://biowhat.ucsd.edu/homer/motif">http://biowhat.ucsd.edu/homer/motif</a>). Sequences analyzed corresponded to those that were present in C57BL/6 but absent in A/J (referred to as C57(+)/A/J(−) in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), as these were the only ones where full sequence information was available. To test whether the regions containing polymorphic SINEs had specific characteristics, we use the “annotate_peaks” function provided by the HOMER package.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462715/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462715/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Properties of regions containing polymorphic TEs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Statistics</h3><p id="p-20">Comparisons between groups were performed by Student’s <em>t</em>-test (in the case of 2 groups) or by one-way analysis of variance followed be Tukey’s honestly significant difference post-hoc multiple comparison tests (in the cases of comparisons between more than two groups). Differences in the relative abundance of total and polymorphic TEs in different genomic regions were tested by two-way analysis of variance followed by Tukey’s honestly significant difference post-hoc tests.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Detection of <em>cis</em>-eQTL clusters</h3><p id="p-21">Using the Illumina MouseRef-8 microarray, we detected a total of 8725 genes expressed in hearts from the AXB/BXA mice. Further genomic mapping revealed that 777 of these genes were linked to <em>cis</em>-eQTLs, and several of them formed clusters of three or more <em>cis</em>-eQTLs within genomic intervals of a few hundred kilobases. We thus tested several window sizes to best define <em>cis</em>-eQTL clusters and control clusters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS3.pdf">Table S3</a>). By using maximum intervals of 250 kb, we detected a total of 42 <em>cis</em>-eQTL clusters (containing in average 4.23 ± 1.9 detected genes, ranging from 3 to 11, within intervals averaging 221.9 ± 130 kb), and 188 control clusters (containing in average 4.75 ± 1.85 detected genes, ranging from 3 to 13, within intervals averaging 248 ± 77.6 kb). By using maximum intervals of 500 kb, we detected a total of 53 <em>cis</em>-eQTL clusters (containing in average 4.9 ± 3.5 <em>cis</em>-eQTL genes, ranging from 3 to 19, within intervals averaging 467 ± 486 kb), and 59 control clusters (containing in average 5.2 ± 2.5 detected genes, ranging from 3 to 17, within intervals averaging 456 ± 119 kb). There were no significant differences between <em>cis</em>-eQTL and control clusters for any of the aforementioned values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS3.pdf">Table S3</a>). Because genes detected by the Illumina array in heart extracts do not correspond to all genes present in the genome, we also verified the density of all Entrez-annotated genes in the same regions to estimate total gene density, and found no significant difference between <em>cis</em>-eQTL and control clusters. When using maximum intervals of 750 kb, we detected a total of 61 <em>cis</em>-eQTL clusters but detected only 21 matching control clusters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS3.pdf">Table S3</a>). Further analyses were thus restricted to the “250 kb” and “500 kb” clusters. None of the <em>cis</em>-eQTL clusters corresponded to clusters of paralogous genes known to arise from tandem duplication. The coordinates of all 42 “250 kb” <em>cis</em>-eQTL clusters are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS9.pdf">Table S9</a>, along with the symbols of corresponding <em>cis</em>-eQTL genes. All control clusters and random regions are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS10.xlsx">Table S10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS11.xlsx">Table S11</a>.</p><p id="p-22">Although SNPs within probes are not likely to affect the hybridization efficiency of Illumina microarray probes (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Schurmann <em>et al.</em> 2012</a>), we nonetheless used data from the Sanger website to detect all high-quality (score &gt; 100) SPs that are polymorphic between A/J and C57Bl/6J and verified that no <em>cis</em>-eQTL within the clusters could represent an artifact resulting from a SNP polymorphism. Accordingly, we found a total of 91 SNPs falling within probe sequences. Among them, eight corresponded to <em>cis</em>-eQTLs, but none of them corresponded to those found in the <em>cis</em>-eQTL clusters.</p><p id="p-23">Coexpression levels were quantified by calculating the absolute values of the Pearson correlation coefficients between each pair of genes within clusters. Within each <em>cis</em>-eQTL cluster, overall coexpression levels were calculated by the averaging all pairwise coexpression values and then compared with values obtained in either corresponding control clusters or in 500 “random groups.” The latter comprised either 3 to 11 genes (for the “250 kb” clusters) or 3 to 19 genes (for the “500 kb” clusters), all genes being chosen randomly throughout the genome. For <em>cis</em>-eQTL genes within “250 kb” clusters, coexpression level was 0.755 ± 0.07 (mean ± SD), this value being significantly greater (<em>P</em> &lt; 10e-16) than that obtained for detected genes in control clusters (0.23 ± 0.09). Although mean coexpression level in control clusters was about 17% greater than in random groups of genes (0.196 ± 0.04, <em>P</em> = 7.4e-07), it was also more than three times lower than that found <em>in cis</em>-eQTL clusters (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Very similar results were obtained in terms of coexpression levels and intergroup differences when using the “500” kb clusters (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The increased coexpression of genes within the <em>cis</em>-eQTL clusters was not due to an overall greater level of expression of genes in the cluster: the mean log2 value of expression level of genes within the <em>cis</em>-eQTL clusters was 9.023, this value being not significantly different (<em>P</em> = 0.9) from the level of expression of genes in control clusters (<em>i.e.</em>, 9.02).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/597/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution plots of coexpression values (calculated as Pearson&#x2019;s coefficients) of genes in cis-eQTL clusters (dotted line), control clusters (dashed line), and random boxes (solid line). For each cluster, the coexpression values represent the mean of all pairwise coexpression values between genes in the cluster. The absolute coexpression values (mean &#xB1; SD) was 0.76 &#xB1; 0.07 in cis-eQTL clusters; this value was significantly greater (P &lt; 0.001) than that obtained for genes in either control clusters (0.26 &#xB1; 0.12) or random groups of genes (0.19 &#xB1; 0.19). (A) Coexpression levels in &#x201C;250 kb&#x201D; clusters. (B) Coexpression levels in &#x201C;500 kb&#x201D; clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1588152566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution plots of coexpression values (calculated as Pearson&#x2019;s coefficients) of genes in &lt;em&gt;cis&lt;/em&gt;-eQTL clusters (dotted line), control clusters (dashed line), and random boxes (solid line). For each cluster, the coexpression values represent the mean of all pairwise coexpression values between genes in the cluster. The absolute coexpression values (mean &#xB1; SD) was 0.76 &#xB1; 0.07 in &lt;em&gt;cis&lt;/em&gt;-eQTL clusters; this value was significantly greater (&lt;em&gt;P&lt;/em&gt; &lt; 0.001) than that obtained for genes in either control clusters (0.26 &#xB1; 0.12) or random groups of genes (0.19 &#xB1; 0.19). (A) Coexpression levels in &#x201C;250 kb&#x201D; clusters. (B) Coexpression levels in &#x201C;500 kb&#x201D; clusters.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/597/F1.medium.gif" width="440" height="191"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/597/F1.medium.gif" width="440" height="191"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/597/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/597/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462607" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-24" class="first-child">Distribution plots of coexpression values (calculated as Pearson’s coefficients) of genes in <em>cis</em>-eQTL clusters (dotted line), control clusters (dashed line), and random boxes (solid line). For each cluster, the coexpression values represent the mean of all pairwise coexpression values between genes in the cluster. The absolute coexpression values (mean ± SD) was 0.76 ± 0.07 in <em>cis</em>-eQTL clusters; this value was significantly greater (<em>P</em> &lt; 0.001) than that obtained for genes in either control clusters (0.26 ± 0.12) or random groups of genes (0.19 ± 0.19). (A) Coexpression levels in “250 kb” clusters. (B) Coexpression levels in “500 kb” clusters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Given the average size of <em>cis</em>-eQTL clusters (<em>i.e.</em>, 248 to 456 kb), the average size of intervals between polymorphic SNPs in the RIS panel (2.59 ± 2.95 Mb) and the high level of linkage disequilibrium between adjacent informative SNPs (r<sup>2</sup> = 0.8), the vast majority of neighboring and coexpressed <em>cis</em>-eQTLs within <em>cis</em>-eQTL clusters are likely to have the same allelic origin. In contrast, <em>cis</em>-eQTL genes within <em>cis</em>-eQTL clusters did not show homogeneity either in terms of regulation (because consistent up- or down-regulation of <em>cis</em>-eQTL genes was found in only 26% of the <em>cis</em>-eQTL clusters) or in terms of genomic strand origin (because both strands of genomic DNA contributed to the sequences of neighboring and co-expressed <em>cis</em>-eQTL in 77% of the <em>cis</em>-eQTL clusters). There was little evidence to suggest that the <em>cis</em>-eQTL clusters could correspond to either recombinant blocks or to regions with different recombination rates. When defining minimal haplotype blocks as regions flanked by polymorphic markers, we found a total of 930 blocks whose average size (2.59 ± 2.95 Mb) was considerably larger than that of <em>cis</em>-eQTL clusters (248 to 456 kb). Moreover, the average coexpression value of detected genes within these minimal haplotype blocks was 0.25 ± 0.11 and thus much lower than that of <em>cis</em>-eQTLs within <em>cis</em>-eQTL clusters (0.755 ± 0.07). Thus, high coexpression levels were found only for <em>cis</em>-eQTL genes within <em>cis</em>-eQTL clusters, and not for all detected genes throughout the haplotype blocks. Finally, the distribution of recombinant rate values <em>in cis</em>-eQTL clusters did not appear to be different from that of detected genes in control clusters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-8" class="subsection"><h3>Structural characteristics of regions containing <em>cis</em>-eQTL clusters</h3><p id="p-26">The detection of clusters of <em>cis</em>-eQTLs suggested that genetic polymorphisms in some regions could associate with changes in the expression levels of several genes in the same region. Considering that it would be unlikely that such coordinate changes in the regulation of several neighboring genes could result from SNPs each affecting the expression of corresponding <em>cis</em>-eQTL genes in a proportionate manner, we mined databases to question whether <em>cis</em>-eQTL cluster regions could show enrichment in structural variants (with potential of affecting expression of all <em>cis</em>-eQTLs in the region). Because the majority (<em>i.e.</em>, 98%) of mouse structural variants have been reported to correspond to TE variants (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akagi <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yalcin <em>et al.</em> 2011</a>), we used data from the most recent report that established a catalog of TE variants across mouse strains (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Nellåker <em>et al.</em> 2012</a>) to test whether <em>cis</em>-eQTL clusters and their surrounding regions would contain more TE variants than either control clusters or regions of similar size centered around single <em>cis</em>-eQTLs. The respective abundances of TEs that were reported as polymorphic between A/J and C57BL/6J are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS1.pdf">Table S1</a>. Considering that regulatory regions can be located either upstream or downstream of the genes under consideration, we defined the regions to be analyzed by adding flanking sequences with lengths of either 250, 500, or 1000 kb to the regions corresponding to both types of clusters, thus corresponding to regions of six different sizes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS5.pdf">Table S5</a>).</p><p id="p-27">For each of the six sizes of regions, comparisons were made between the three types of “defined” regions (<em>cis</em>-eQTL clusters, control clusters, and regions centered on single <em>cis</em>-eQTLs) and the fourth type of region, consisting of random regions of matching size. Overall, the same interregion differences were found regardless of how the regions were defined. For simplicity, the regions corresponding to the “250 kb” clusters augmented by flanking regions of 250 kb were chosen as representative data for presentation (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Long interspersed elements (LINEs) and LINE fragments showed some minor (usually nonsignificant) differences in abundance between the four types of regions. Both polymorphic and fixed SINEs and LTRs were more abundant in all three defined regions than in random regions. For SINEs, this finding is in keeping with the fact that these elements are generally more abundant in gene-rich than in gene-poor regions (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jurka <em>et al.</em> 2005</a>). However, for polymorphic TEs, only SINEs were significantly greater (<em>P</em> &lt; e-5) <em>in cis</em>-eQTL clusters compared with both control clusters and regions centered on single eQTLs. Moreover, the fold-enrichment and the significance of these differences were of greater magnitude for polymorphic SINEs than for fixed SINEs, which indicated that the greater abundance of polymorphic SINEs <em>in cis</em>-eQTL clusters was not a mere consequence of total abundance of TEs but rather a consequence of the genetic differences between C57BL/6J and A/J mice. These differences were not due to differences in total gene density because the abundance of total genes <em>in cis</em>-eQTL regions (10.1 ± 17) was not significantly different than that in control regions (10.18 ± 3.5). Representative examples that compare two <em>cis</em>-eQTL clusters and control clusters of matching sizes are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Altogether, polymorphic SINEs appeared to be a signature characteristic of <em>cis</em>-eQTL regions. We also found that the density of polymorphic SINEs <em>in cis</em>-eQTL cluster regions (taken along with their 250 kb flanking regions) was calculated to be 10.3 ± 10.7 polymorphic SINEs/MB; this value was significantly greater (<em>P</em> &lt; 0.002) than that found in other regions of corresponding haplotype blocks outside of the <em>cis</em>-eQTL clusters (5.1 ± 8.1). This finding provided additional evidence that the <em>cis</em>-eQTL clusters had features that differentiated them from the haplotype blocks that contained them.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/597/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative abundance of polymorphic and fixed TEs (SINEs and LTR-TEs) in cis-eQTL clusters, control clusters, single cis-eQTL regions, and random boxes. Each bar represents mean &#xB1; SEM of the relative abundance of TEs in each genomic region box (absolute number of TEs in each box divided by the mean value of the number of TEs in the corresponding random boxes). P values are as indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1588152566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative abundance of polymorphic and fixed TEs (SINEs and LTR-TEs) in &lt;em&gt;cis&lt;/em&gt;-eQTL clusters, control clusters, single &lt;em&gt;cis&lt;/em&gt;-eQTL regions, and random boxes. Each bar represents mean &#xB1; SEM of the relative abundance of TEs in each genomic region box (absolute number of TEs in each box divided by the mean value of the number of TEs in the corresponding random boxes). &lt;em&gt;P&lt;/em&gt; values are as indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/597/F2.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/597/F2.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/597/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/597/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462666" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-28" class="first-child">Relative abundance of polymorphic and fixed TEs (SINEs and LTR-TEs) in <em>cis</em>-eQTL clusters, control clusters, single <em>cis</em>-eQTL regions, and random boxes. Each bar represents mean ± SEM of the relative abundance of TEs in each genomic region box (absolute number of TEs in each box divided by the mean value of the number of TEs in the corresponding random boxes). <em>P</em> values are as indicated.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/597/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative examples illustrating two cis-eQTL clusters and control clusters of matching size. For each cluster, the first line (on top) represents respective genomic scales (in Mb); the second line represents the genomic positions of either cis-eQTLs or detected genes; and the third and fourth lines represent the genomic positions of polymorphic LTR-TE and SINEs. On the second line, cis-eQTL genes are represented at their respective position by arrows; the arrow for the first gene on the left points upwards; other genes are represented by either upwards or downward arrows, according to whether they correlated positively or negatively, respectively, with the change in expression of the first gene. On the third and fourth lines, plain vertical lines correspond to TEs that are C57(+)/A/J(&#x2212;); dashed lines correspond to TEs that are C57(&#x2212;)/A/J(+)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1588152566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Representative examples illustrating two &lt;em&gt;cis&lt;/em&gt;-eQTL clusters and control clusters of matching size. For each cluster, the first line (on top) represents respective genomic scales (in Mb); the second line represents the genomic positions of either &lt;em&gt;cis&lt;/em&gt;-eQTLs or detected genes; and the third and fourth lines represent the genomic positions of polymorphic LTR-TE and SINEs. On the second line, &lt;em&gt;cis&lt;/em&gt;-eQTL genes are represented at their respective position by arrows; the arrow for the first gene on the left points upwards; other genes are represented by either upwards or downward arrows, according to whether they correlated positively or negatively, respectively, with the change in expression of the first gene. On the third and fourth lines, plain vertical lines correspond to TEs that are C57(+)/A/J(&#x2212;); dashed lines correspond to TEs that are C57(&#x2212;)/A/J(+).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/597/F3.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/597/F3.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/597/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/597/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462820" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-29" class="first-child">Representative examples illustrating two <em>cis</em>-eQTL clusters and control clusters of matching size. For each cluster, the first line (on top) represents respective genomic scales (in Mb); the second line represents the genomic positions of either <em>cis</em>-eQTLs or detected genes; and the third and fourth lines represent the genomic positions of polymorphic LTR-TE and SINEs. On the second line, <em>cis</em>-eQTL genes are represented at their respective position by arrows; the arrow for the first gene on the left points upwards; other genes are represented by either upwards or downward arrows, according to whether they correlated positively or negatively, respectively, with the change in expression of the first gene. On the third and fourth lines, plain vertical lines correspond to TEs that are C57(+)/A/J(−); dashed lines correspond to TEs that are C57(−)/A/J(+).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">To test the possible functional impact of polymorphic SINEs, motif enrichment analyses were performed for TE sequences that are present in C57BL/6 and deleted in A/J mice (because the full sequences of TEs deleted in C57BL/6 and present in A/J are not available yet). The list of most significantly enriched binding sites is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS6.pdf">Table S6</a>. For polymorphic SINEs, the binding site that was most statistically enriched (<em>P</em> = 1e-1283) was that previously reported to be bound by BORIS, a CTCF paralogue that binds a CTCF-like binding site (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pugacheva <em>et al.</em> 2010</a>). This site matched the composition of the M1 moiety of the full CTCF binding site recently described by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schmidt <em>et al.</em> (2012)</a>. Most other significantly enriched sites corresponded to binding sites for transcription factors from several families, in addition to another CTCF binding site that in fact matches the composition of the M2 moiety of the full CTCF binding site (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Schmidt <em>et al.</em> 2012</a>).</p><p id="p-31">According to MouseIndelDB annotation, 19% of SINEs that are polymorphic between C56BL/6J and A/J correspond to B1 SINEs, with the remainder corresponding to B2 SINEs. We also investigated the nature of regions harboring polymorphic TEs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Notwithstanding a few exceptions, the great majority of polymorphic TEs fell into either intronic or intergenic regions. Not surprisingly, C57(+)/A/J(−) LTR-TEs and SINEs fell into C57 regions previously annotated as having these characteristics. Interestingly, C57(−)/A/J(+) LTR-TEs sometimes fell into regions annotated as containing LINEs or SINEs in C57BL/6J, whereas C57(−)/A/J(+) SINEs sometimes fell into regions already containing LTR-TEs or LINEs. Beyond polymorphic TEs, we obtained from previous publications (reporting the results of Chip-Seq experiments performed on mouse heart chromatin) lists of all regions corresponding to either binding sites for transcription factors or chromatin modifications. We then tested whether all corresponding genomic features showed differential abundance between <em>cis</em>-eQTL, control, and random regions by using the six different types of window sizes described previously (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS7.pdf">Table S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS8.pdf">Table S8</a>). Overall, the same interregion differences were found regardless of how the regions were defined, but differences tended to be more pronounced for regions surrounding and comprising the “250 kb” clusters. Accordingly, the regions corresponding to the “250 kb” clusters augmented by flanking regions of 250 kb were chosen as representative examples for presentation (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Overall, the abundance of all binding sites was lower in random regions than in all types of defined regions. For some regulatory factors (CTCF, H3Ac, SRF, and Tbx5), their abundance was much greater <em>in cis</em>-eQTL regions than in the other two types of defined regions control regions and more abundant in the latter than in random regions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/597/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative abundance of binding sites for CTCF, SRF, and Tbx5 and of the H3Ac chromatin marks previously reported in chromatin from mouse hearts cis-eQTL clusters, control clusters, single cis-eQTL regions, and random boxes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1588152566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative abundance of binding sites for CTCF, SRF, and Tbx5 and of the H3Ac chromatin marks previously reported in chromatin from mouse hearts &lt;em&gt;cis&lt;/em&gt;-eQTL clusters, control clusters, single &lt;em&gt;cis&lt;/em&gt;-eQTL regions, and random boxes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/597/F4.medium.gif" width="440" height="109"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/597/F4.medium.gif" width="440" height="109"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/597/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/597/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462765" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-32" class="first-child">Relative abundance of binding sites for CTCF, SRF, and Tbx5 and of the H3Ac chromatin marks previously reported in chromatin from mouse hearts <em>cis</em>-eQTL clusters, control clusters, single <em>cis</em>-eQTL regions, and random boxes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Comparisons with other panels of RIS</h3><p id="p-33">To test to which extent <em>cis</em>-eQTL clusters would be conserved across tissues, we questioned whether regions containing <em>cis</em>-eQTL clusters for cardiac genes would overlap with regions containing <em>cis</em>-eQTL clusters for genes expressed in another tissues (with clusters of <em>cis</em>-eQTLs being defined on the basis of maximum intervals of 250 Mb between each <em>cis</em>-eQTL). We first analyzed gene expression data obtained in AxB/BxA eyes with Illumina microarrays (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS2.pdf">Table S2</a>), which allowed us to detect a total of 35 <em>cis</em>-eQTL clusters (while verifying, as explained previously, that none of the <em>cis</em>-eQTL genes could represent an artifact due to the presence of a SNP polymorphism within the sequence of the probes). Despite the fact that the latter data were obtained by other investigators, 12 of the 42 regions containing <em>cis</em>-eQTLs clusters for genes expressed in AxB/BxA hearts also contained <em>cis</em>-eQTL clusters for genes expressed in eyes from the same RIS.We also analyzed gene expression data for other tissues from the BxD RIS mouse panel, where gene expression was analyzed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1/TableS2.pdf">Table S2</a>) with either the Affymetrix MoGene 1.0 ST microarray (for hypothalamus) or with the Affymetrix Mouse Genome 430 microarray (for eye, kidney, hippocampus, and cerebellum). Of note, consultation of the list of SNP polymorphisms between C57BL/6J and DBA/2 mice revealed a total of 0.005% and 0.0006% of probes used by the Affymetrix MoGene 1.0 ST and Affymetrix Mouse Genome 430 microarrays, respectively, are affected by such polymorphisms. Corresponding genes were excluded from the <em>cis</em>-eQTL analysis. Analysis of the gene expression data from BxD RIS tissues allowed us to detect 52, 279, 260, 77, and 36 <em>cis</em>-eQTL clusters for kidneys, eyes, hippocampus, hypothalamus, and cerebrum, respectively. Again, high proportions of <em>cis</em>-eQTL−containing regions were shared across tissues. For instance, of the 52 regions detected for genes expressed in kidneys, the number of regions also detected for genes from other tissues amounted to 44 for eyes, 44 for hippocampus, 15 for hypothalamus, and 15 for cerebrum. Finally, to test whether regions containing <em>cis</em>-eQTL clusters in one tissue would overlap with regions containing <em>cis</em>-eQTL clusters in one same tissue from two RIS panels, we compared data obtained for eye genes from AxB/BxA and BxD RIS. Of the 35 regions containing <em>cis</em>-eQTL clusters for genes from AxB/BxA eyes, 16 corresponded to regions containing <em>cis</em>-eQTL clusters for genes from BxD eyes.</p></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-34">Despite evidence suggesting the existence of gene coexpression domains in several eukaryotic organisms (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Michalak 2008</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Elizondo <em>et al.</em> 2009</a>), data available to date suggested that coexpression in mammals rarely concerns more than doublets or triplets of immediately adjacent genes (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sémon and Duret 2006</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Purmann <em>et al.</em> 2007</a>). Our data show that within panels of mouse RIS, where gene expression variability is due in part to genetic polymorphisms, it is indeed possible to detect short-range clustering of more than three neighboring (but not necessarily adjacent) coexpressed genes. Moreover, all coexpressed genes within these domains showed linkage to loci having the same position as that of the domains, thus showing that genetic polymorphisms can associate with the expression levels of several neighboring genes within these domains. Depending on the sizes of windows used to define them, <em>cis</em>-eQTL clusters detected for AxB/BxA hearts contained in average 4.2 to 4.9 <em>cis</em>-eQTL genes within intervals averaging 248-456 kb. The number of <em>cis</em>-eQTL clusters detected for hearts from AxB/BxA panel (<em>i.e.</em>, 53) was within the same range as the numbers of <em>cis</em>-eQTL clusters detected with three other tissues from the distinct BxD RIS panel (<em>i.e.</em>, 60 for cerebrum, 74 for kidneys, and 96 for hypothalamus). Moreover, a great proportion of the regions containing <em>cis</em>-eQTL for AxB/BxA hearts were identical to those containing <em>cis</em>-eQTL clusters for eyes from either AxB/BxA or BxD RIS, and many of the regions containing <em>cis</em>-eQTL clusters for BxD kidneys overlapped with those detected for four other tissues from the same panel. Altogether, the data indicate that <em>cis</em>-eQTL clusters occur in a fairly robust and consistent manner across different tissues and/or genetic backgrounds in mouse RIS. Of note, all tissues do not necessarily express the same genes, which may explain in part why some <em>cis</em>-eQTL clusters are distinct between tissues. With some tissues (eyes and hippocampus from BxD RIS), up to 300 <em>cis</em>-eQTL clusters were detected. A comprehensive list of all genomic regions having potential to contain <em>cis</em>-eQTLs and a full understanding of the extent to which they overlap would require data from a more exhaustive list of tissues.</p><p id="p-35">One distinct feature of <em>cis</em>-eQTL clusters was that <em>cis</em>-eQTL genes in these regions showed a level of coexpression that greatly exceeded that found for detected genes in control regions with similar gene density. For instance, in regions selected as controls for the “500 kb” <em>cis</em>-eQTL clusters, only 2 of 59 regions displayed coexpression levels greater than 0.56 (<em>i.e.</em>, the value corresponding to the lowest value for gene coexpression <em>in cis</em>-eQTL clusters). The <em>cis</em>-eQTL clusters thus corresponded to “gene coexpression domain” QTLs. The fact that all coexpressed genes within domains showed linkage to a common locus suggested that the genome contains polymorphisms that can alter coexpression levels. Reasoning that identification of the nature of such polymorphisms could reveal insights as to what drives coexpression of genes within coexpression domains, we mined databases to test whether the <em>cis</em>-eQTL regions harbored any particular types of polymorphic structural variants. Accordingly, we found that polymorphic SINEs [either C57(+)/A/J(−) or C57(−)/A/J(+)] were significantly more abundant <em>in cis</em>-eQTL clusters than in any other type of control regions. Given the possibility that polymorphic SINEs could in fact drive these high levels of coexpression in corresponding domains, we tested whether they showed enrichment for particular motifs. In addition to binding sites for various transcription factors belonging to different families, polymorphic SINEs showed enrichment for two sites corresponding to CTCF-binding regions. This finding is an agreement with the previous report in which the authors showed that CTCF-binding regions in the mouse genome were preferentially embedded in B2 SINE elements (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bourque <em>et al.</em> 2008</a>). B2 SINEs constitute types of SINEs that are specific to rodent genomes, where they have undergone waves of amplification (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kass <em>et al.</em> 1997</a>). Accordingly, 81% of the SINEs that are polymorphic between C57BL/6J and A/J were in fact B2 SINEs. Moreover, we compared the relative abundance of either chromatin marks or binding sites for either CTCF or cardiac transcription factors <em>in cis</em>-eQTL clusters <em>vs.</em> all three other types of regions. Accordingly, we found that binding sites for the transcription factors SRF and TBX5, the chromatin-organizing factors CTCF and p300, and the H3Ac chromatin were all significantly enriched <em>in cis</em>-eQTL clusters <em>vs.</em> all three other regions.</p><p id="p-36">SINEs and CTCF binding sites are of particular interest. The structural and regulatory organization of the mammalian genome is fundamentally dependent on CTCF, which has been dubbed the “master weaver of the genome” (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Phillips and Corces 2009</a>). CTCF generally acts as an insulator preventing the spread of inactive heterochromatin, and is often associated with open chromatin (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">The ENCODE Project Consortium 2012</a>). This may be particularly pertinent in the context of our current data further documenting the existence of genetically controlled gene coexpression domains. As a matter of fact, recent genome-wide studies on chromatin structure have revealed that mammalian genomes are organized into topological domains, the boundaries of which show enrichment for SINEs and CTCF binding, and where the spread of heterochromatin is constrained (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dixon <em>et al.</em> 2012</a>). Likewise, it was recently shown that in several mammalian species, CTCF-binding events are associated with waves of retrotransposon expansion, thus revealing the mechanism by which there are born (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Schmidt <em>et al.</em> 2012</a>). Our data extend these previous reports in several ways: (1) in addition to interspecies differences, polymorphic SINEs can cause differential abundance of CTCF binding sites between strains of one same species; (2) the CTCF-dependent organization of genomic mammalian domains may not be static because polymorphic SINEs could reshape that organization; and (3) polymorphic SINEs and CTCF-binding sites may constitute a mechanism defining gene coexpression domains, for which there was so far little evidence in mammals beyond doublets of triplets of genes.</p><p id="p-37">Of note, the allelic origin of the <em>cis</em>-eQTL cluster regions did not affect the expression of all corresponding <em>cis</em>-eQTL in a consistent manner (as illustrated in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). However, because <em>cis</em>-eQTL clusters contained an average of five polymorphic TEs, changes in corresponding chromatin domains may be more complex than simply corresponding to a chromatin structure that is entirely open or closed for the whole region. Moreover, TEs can affect single gene expression by a variety of different possible mechanisms, as for instance by providing alternative promoters or enhancers, serving as insulators or transcriptional silencers, disrupting the exon-intron structure and/or causing premature transcriptional termination (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gogvadze and Buzdin 2009</a>). TE might also affect gene regulation by mechanisms other than just providing regulatory elements. For instance, we found that some C57(−)/A/J(+) polymorphisms fell themselves within regions containing other TEs, and might thus disrupt in A/J the organization of some TE elements that have regulatory effects within the C57BL/6J strain. In combination with complex changes in chromatin structure, all the above mechanisms might account for the nonuniform effects of the allelic origin of <em>cis</em>-eQTL clusters on gene expression.</p><p id="p-38">Close to half of mammalian genomes is derived from ancient TEs (primarily retroelements) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">van de Lagemaat <em>et al.</em> 2003</a>). Given their great abundance in the genome and the emerging recognition of their role in gene regulation, TEs that show polymorphism between inbred strains are increasingly being recognized as potential players in the genetics of quantitative traits (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Nellåker <em>et al.</em> 2012</a>). Accordingly, some classes of polymorphic TEs have been reported to show small but significant enrichment in refined genomic intervals selected on the basis of previous detection of quantitative trait loci for mouse quantitative traits (<a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Nellåker <em>et al.</em> 2012</a>). In terms of gene regulation, recent reports on the effects of TEs have so far concerned mostly single genes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chernova <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Palmer and Dulawa 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li <em>et al.</em> 2012</a>). However, complex quantitative traits are usually considered to result from the combined regulatory effects of several genes rather than from the highly penetrant effect of single mutations (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Manolio <em>et al.</em> 2009</a>). Gene coexpression domains might thus be of particular interest, as coordinate dysregulation of the expression levels of several genes within <em>cis</em>-eQTL clusters could possibly have greater effects on the phenotypic expression of complex traits than dysregulation of single genes.</p><p id="p-39">Regulatory sequences located outside of coding regions are also interesting in the light of evidence that: (1) up to one third of noncoding sequence variation contributes causally to the traits under investigation in genome-wide association studies (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Visel <em>et al.</em> 2009</a>); (2) a great proportion of regulatory variants of gene expression are found at a fairly great distance from transcription start sites (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Teng <em>et al.</em> 2011</a>); and (3) chromatin structure plays important roles in the organization and regulation of our genes (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">The ENCODE Project Consortium 2012</a>). Genome-wide approaches allowing the discovery and functional characterization of such elements might improve our understanding of their role in human biology and disease susceptibility (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Teng <em>et al.</em> 2011</a>). However, if polymorphic TEs turn out to have important consequences on gene regulation, appropriate technologies to detect them genome-wide will need to be developed because technologies based on detection of SNP polymorphisms are not sufficient in this regard.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This work was supported by grant MOP-93583 from the Canadian Institutes for Health Research (CIHR).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.005546/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-2">Processed data from this article have been submitted for public access to GeneNetwork (<a href="http://www.genenetwork.org">www.genenetwork.org</a>; accession number GN421).</p></li><li class="fn" id="fn-4"><p id="p-41"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-01"><span class="received-label">Received </span>November 1, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-01-26"><span class="accepted-label">Accepted </span>January 26, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Scott-Boyer, Deschepper</li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.1" data-doi="10.1101/gr.075770.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akagi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volfovsky</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symer</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Extensive variation between inbred mouse strains due to endogenous L1 retrotransposition</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.075770.107%26rft_id%253Dinfo%253Apmid%252F18381897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzYvODY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.2" data-doi="10.1093/nar/gkp1046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akagi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evdokimov</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MouseIndelDB: a database integrating genomic indel polymorphisms that distinguish mouse strains</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D600</span>–<span class="cit-lpage">D606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp1046%26rft_id%253Dinfo%253Apmid%252F19933259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOC9zdXBwbF8xL0Q2MDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNTk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.3" data-doi="10.1038/ng.650"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blow</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCulley</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akiyama</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ChIP-Seq identification of weakly conserved heart enhancers</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">806</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBlow%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D42%26rft.issue%253D9%26rft.spage%253D806%26rft.epage%253D810%26rft.atitle%253DChIP-Seq%2Bidentification%2Bof%2Bweakly%2Bconserved%2Bheart%2Benhancers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.650%26rft_id%253Dinfo%253Apmid%252F20729851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20729851&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281388400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.4" data-doi="10.1101/gr.080663.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourque</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution of the mammalian transcription factor binding repertoire via transposable elements</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1752</span>–<span class="cit-lpage">1762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.080663.108%26rft_id%253Dinfo%253Apmid%252F18682548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8xNzUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.5" data-doi="10.1534/genetics.112.141036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brunschwig</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben-David</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakir</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fine-scale maps of recombination rates and hotspots in the mouse genome</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">757</span>–<span class="cit-lpage">764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141036%26rft_id%253Dinfo%253Apmid%252F22562932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvNzU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.6" data-doi="10.1016/j.bbrc.2008.10.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chernova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higginson</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">B2 SINE retrotransposon causes polymorphic expression of mouse 5-aminolevulinic acid synthase 1 gene</span>. <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">377</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DChernova%26rft.auinit1%253DT.%26rft.volume%253D377%26rft.issue%253D2%26rft.spage%253D515%26rft.epage%253D520%26rft.atitle%253DB2%2BSINE%2Bretrotransposon%2Bcauses%2Bpolymorphic%2Bexpression%2Bof%2Bmouse%2B5-aminolevulinic%2Bacid%2Bsynthase%2B1%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2008.10.020%26rft_id%253Dinfo%253Apmid%252F18929534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2008.10.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18929534&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.7" data-doi="10.1038/ng1518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChesler%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D37%26rft.issue%253D3%26rft.spage%253D233%26rft.epage%253D242%26rft.atitle%253DComplex%2Btrait%2Banalysis%2Bof%2Bgene%2Bexpression%2Buncovers%2Bpolygenic%2Band%2Bpleiotropic%2Bnetworks%2Bthat%2Bmodulate%2Bnervous%2Bsystem%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1518%26rft_id%253Dinfo%253Apmid%252F15711545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15711545&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227297800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.8" data-doi="10.1186/1471-2164-13-34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheppard</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Discovery of novel variants in genotyping arrays improves genotype retention and reduces ascertainment bias</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDidion%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D13%26rft.spage%253D34%26rft.epage%253D34%26rft.atitle%253DDiscovery%2Bof%2Bnovel%2Bvariants%2Bin%2Bgenotyping%2Barrays%2Bimproves%2Bgenotype%2Bretention%2Band%2Breduces%2Bascertainment%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-34%26rft_id%253Dinfo%253Apmid%252F22260749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-34&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22260749&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.9" data-doi="10.1038/nature11082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selvaraj</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Topological domains in mammalian genomes identified by analysis of chromatin interactions</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDixon%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D485%26rft.issue%253D7398%26rft.spage%253D376%26rft.epage%253D380%26rft.atitle%253DTopological%2Bdomains%2Bin%2Bmammalian%2Bgenomes%2Bidentified%2Bby%2Banalysis%2Bof%2Bchromatin%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11082%26rft_id%253Dinfo%253Apmid%252F22495300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495300&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304099100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.10" data-doi="10.1038/nprot.2009.97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durinck</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1184</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DDurinck%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D1184%26rft.epage%253D1191%26rft.atitle%253DMapping%2Bidentifiers%2Bfor%2Bthe%2Bintegration%2Bof%2Bgenomic%2Bdatasets%2Bwith%2Bthe%2BR%252FBioconductor%2Bpackage%2BbiomaRt.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.97%26rft_id%253Dinfo%253Apmid%252F19617889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.97&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19617889&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.11" data-doi="10.2174/138920209787581271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elizondo</span>  <span class="cit-name-given-names">L. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jafar-Nejad</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clewing</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerkoel</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene clusters, molecular evolution and disease: a speculation</span>. <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DElizondo%26rft.auinit1%253DL.%2BI.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D75%26rft.atitle%253DGene%2Bclusters%252C%2Bmolecular%2Bevolution%2Band%2Bdisease%253A%2Ba%2Bspeculation.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920209787581271%26rft_id%253Dinfo%253Apmid%252F19721813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920209787581271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19721813&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265681700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.12" data-doi="10.1038/nature11247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The ENCODE Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated encyclopedia of DNA elements in the human genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955616&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.13" data-doi="10.1007/s00018-009-0107-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gogvadze</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buzdin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Retroelements and their impact on genome evolution and functioning</span>. <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">3727</span>–<span class="cit-lpage">3742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DGogvadze%26rft.auinit1%253DE.%26rft.volume%253D66%26rft.issue%253D23%26rft.spage%253D3727%26rft.epage%253D3742%26rft.atitle%253DRetroelements%2Band%2Btheir%2Bimpact%2Bon%2Bgenome%2Bevolution%2Band%2Bfunctioning.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-009-0107-2%26rft_id%253Dinfo%253Apmid%252F19649766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-009-0107-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19649766&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.14" data-doi="10.1038/ng2119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Göring</span>  <span class="cit-name-given-names">H. H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curran</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Discovery of expression QTLs using large-scale transcriptional profiling in human lymphocytes</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1208</span>–<span class="cit-lpage">1216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGoring%26rft.auinit1%253DH.%2BH.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1208%26rft.epage%253D1216%26rft.atitle%253DDiscovery%2Bof%2Bexpression%2BQTLs%2Busing%2Blarge-scale%2Btranscriptional%2Bprofiling%2Bin%2Bhuman%2Blymphocytes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2119%26rft_id%253Dinfo%253Apmid%252F17873875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17873875&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.15" data-doi="10.1371/journal.pone.0004033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grieve</span>  <span class="cit-name-given-names">I. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickens</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pravenec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kren</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome-wide co-expression analysis in multiple tissues</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e4033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGrieve%26rft.auinit1%253DI.%2BC.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De4033%26rft.epage%253De4033%26rft.atitle%253DGenome-wide%2Bco-expression%2Banalysis%2Bin%2Bmultiple%2Btissues.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004033%26rft_id%253Dinfo%253Apmid%252F19112506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19112506&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.16" data-doi="10.1159/000084943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohany</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavlicek</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kapitonov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Clustering, duplication and chromosomal distribution of mouse SINE retrotransposons</span>. <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D110%26rft.issue%253D1-4%26rft.spage%253D117%26rft.epage%253D123%26rft.atitle%253DClustering%252C%2Bduplication%2Band%2Bchromosomal%2Bdistribution%2Bof%2Bmouse%2BSINE%2Bretrotransposons.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000084943%26rft_id%253Dinfo%253Apmid%252F16093663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000084943&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093663&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231064600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deininger</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Evolution of B2 repeats: the muroid explosion</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DKass%26rft.auinit1%253DD.%2BH.%26rft.volume%253D99%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D13%26rft.atitle%253DEvolution%2Bof%2BB2%2Brepeats%253A%2Bthe%2Bmuroid%2Bexplosion.%26rft_id%253Dinfo%253Apmid%252F9226433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9226433&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XH66600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.18" data-doi="10.1038/ng1195-241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLander%26rft.auinit1%253DE.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D247%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bcomplex%2Btraits%253A%2Bguidelines%2Bfor%2Binterpreting%2Band%2Breporting%2Blinkage%2Bresults.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1195-241%26rft_id%253Dinfo%253Apmid%252F7581446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1195-241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7581446&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TC63900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.19" data-doi="10.1038/ng887"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urrutia</span>  <span class="cit-name-given-names">A. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Clustering of housekeeping genes provides a unified model of gene order in the human genome</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLercher%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D180%26rft.epage%253D183%26rft.atitle%253DClustering%2Bof%2Bhousekeeping%2Bgenes%2Bprovides%2Ba%2Bunified%2Bmodel%2Bof%2Bgene%2Border%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng887%26rft_id%253Dinfo%253Apmid%252F11992122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng887&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11992122&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175903500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.20" data-doi="10.1101/gr.130740.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akagi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trivett</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hlynialuk</span>  <span class="cit-name-given-names">C. J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mouse endogenous retroviruses can trigger premature transcriptional termination at a distance</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">870</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.130740.111%26rft_id%253Dinfo%253Apmid%252F22367191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzUvODcwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.21" data-doi="10.1152/physiolgenomics.00072.2007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Llamas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bélanger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deschepper</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Cardiac mass and cardiomyocyte size are governed by different genetic loci on either autosomes or chromosome Y in recombinant inbred mice</span>. <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00072.2007%26rft_id%253Dinfo%253Apmid%252F17566079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTU6InBoeXNpb2xnZW5vbWljcyI7czo1OiJyZXNpZCI7czo4OiIzMS8yLzE3NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC81OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.22" data-doi="10.1186/1471-2164-10-150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Llamas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deschepper</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Chromosome Y variants from different inbred mouse strains are linked to differences in the morphologic and molecular responses of cardiac cells to postpubertal testosterone</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLlamas%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.spage%253D150%26rft.epage%253D150%26rft.atitle%253DChromosome%2BY%2Bvariants%2Bfrom%2Bdifferent%2Binbred%2Bmouse%2Bstrains%2Bare%2Blinked%2Bto%2Bdifferences%2Bin%2Bthe%2Bmorphologic%2Band%2Bmolecular%2Bresponses%2Bof%2Bcardiac%2Bcells%2Bto%2Bpostpubertal%2Btestosterone.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-150%26rft_id%253Dinfo%253Apmid%252F19351403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19351403&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.23" data-doi="10.1038/nature08494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the missing heritability of complex diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D747%26rft.epage%253D753%26rft.atitle%253DFinding%2Bthe%2Bmissing%2Bheritability%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08494%26rft_id%253Dinfo%253Apmid%252F19812666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812666&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.24" data-doi="10.1007/BF00444361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheah</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesbitt</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankel</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The AXB and BXA set of recombinant inbred mouse strains</span>. <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DMarshall%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253D669%26rft.epage%253D680%26rft.atitle%253DThe%2BAXB%2Band%2BBXA%2Bset%2Bof%2Brecombinant%2Binbred%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00444361%26rft_id%253Dinfo%253Apmid%252F1477475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00444361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1477475&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KA53100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.25" data-doi="10.1016/j.ygeno.2007.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michalak</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Coexpression, coregulation, and cofunctionality of neighboring genes in eukaryotic genomes</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DMichalak%26rft.auinit1%253DP.%26rft.volume%253D91%26rft.issue%253D3%26rft.spage%253D243%26rft.epage%253D248%26rft.atitle%253DCoexpression%252C%2Bcoregulation%252C%2Band%2Bcofunctionality%2Bof%2Bneighboring%2Bgenes%2Bin%2Beukaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2007.11.002%26rft_id%253Dinfo%253Apmid%252F18082363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2007.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18082363&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253475900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.26" data-doi="10.1186/gb-2012-13-6-r45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nellåker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomic landscape shaped by selection on transposable elements across 18 mouse strains</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DNellaker%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253DR45%26rft.epage%253DR45%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Bshaped%2Bby%2Bselection%2Bon%2Btransposable%2Belements%2Bacross%2B18%2Bmouse%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-6-r45%26rft_id%253Dinfo%253Apmid%252F22703977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-6-r45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22703977&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dulawa</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Murine warriors or worriers: the saga of <em>Comt1</em>, B2 SINE elements, and the future of translational genetics</span>. <abbr class="cit-jnl-abbrev">Front. Neurosci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BNeurosci%26rft.aulast%253DPalmer%26rft.auinit1%253DA.%2BA.%26rft.volume%253D4%26rft.spage%253D177%26rft.epage%253D177%26rft.atitle%253DMurine%2BWarriors%2Bor%2BWorriers%253A%2BThe%2BSaga%2Bof%2BComt1%252C%2BB2%2BSINE%2BElements%252C%2Band%2Bthe%2BFuture%2Bof%2BTranslational%2BGenetics.%26rft_id%253Dinfo%253Apmid%252F20976038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20976038&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.28" data-doi="10.1016/j.cell.2009.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corces</span>  <span class="cit-name-given-names">V. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">CTCF: master weaver of the genome</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1194</span>–<span class="cit-lpage">1211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPhillips%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D137%26rft.issue%253D7%26rft.spage%253D1194%26rft.epage%253D1211%26rft.atitle%253DCTCF%253A%2Bmaster%2Bweaver%2Bof%2Bthe%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.06.001%26rft_id%253Dinfo%253Apmid%252F19563753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19563753&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267373400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.29" data-doi="10.1371/journal.pone.0013872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pugacheva</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pack</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosaka-Suzuki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The structural complexity of the human BORIS gene in gametogenesis and cancer</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPugacheva%26rft.auinit1%253DE.%2BM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De13872%26rft.epage%253De13872%26rft.atitle%253DThe%2Bstructural%2Bcomplexity%2Bof%2Bthe%2Bhuman%2BBORIS%2Bgene%2Bin%2Bgametogenesis%2Band%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013872%26rft_id%253Dinfo%253Apmid%252F21079786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079786&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.30" data-doi="10.1016/j.ygeno.2007.01.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toedling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schueler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carninci</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehrach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic organization of transcriptomes in mammals: coregulation and cofunctionality</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DPurmann%26rft.auinit1%253DA.%26rft.volume%253D89%26rft.issue%253D5%26rft.spage%253D580%26rft.epage%253D587%26rft.atitle%253DGenomic%2Borganization%2Bof%2Btranscriptomes%2Bin%2Bmammals%253A%2BCoregulation%2Band%2Bcofunctionality.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2007.01.010%26rft_id%253Dinfo%253Apmid%252F17369017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2007.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17369017&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246164900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.31" data-doi="10.1038/nature01434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schadt</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monks</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lusis</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetics of gene expression surveyed in maize, mouse and man</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchadt%26rft.auinit1%253DE.%2BE.%26rft.volume%253D422%26rft.issue%253D6929%26rft.spage%253D297%26rft.epage%253D302%26rft.atitle%253DGenetics%2Bof%2Bgene%2Bexpression%2Bsurveyed%2Bin%2Bmaize%252C%2Bmouse%2Band%2Bman.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01434%26rft_id%253Dinfo%253Apmid%252F12646919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12646919&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181637300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.32" data-doi="10.1371/journal.pgen.1001313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schueler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The cardiac transcription network modulated by Gata4, Mef2a, Nkx2.5, Srf, histone modifications, and MicroRNAs</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSchlesinger%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De1001313%26rft.epage%253De1001313%26rft.atitle%253DThe%2Bcardiac%2Btranscription%2Bnetwork%2Bmodulated%2Bby%2BGata4%252C%2BMef2a%252C%2BNkx2.5%252C%2BSrf%252C%2Bhistone%2Bmodifications%252C%2Band%2BmicroRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001313%26rft_id%253Dinfo%253Apmid%252F21379568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21379568&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.33" data-doi="10.1016/j.cell.2011.11.058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwalie</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballester</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">Â.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Waves of retrotransposon expansion remodel genome organization and CTCF binding in multiple mammalian lineages</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchmidt%26rft.auinit1%253DD.%26rft.volume%253D148%26rft.issue%253D1-2%26rft.spage%253D335%26rft.epage%253D348%26rft.atitle%253DWaves%2Bof%2Bretrotransposon%2Bexpansion%2Bremodel%2Bgenome%2Borganization%2Band%2BCTCF%2Bbinding%2Bin%2Bmultiple%2Bmammalian%2Blineages.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.11.058%26rft_id%253Dinfo%253Apmid%252F22244452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.11.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22244452&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299540700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.34" data-doi="10.1039/c1mb05363a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schueler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tönjes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sperling</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Dynamics of Srf, p300 and histone modifications during cardiac maturation in mouse</span>. <abbr class="cit-jnl-abbrev">Mol. Biosyst.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DSchueler%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D495%26rft.epage%253D503%26rft.atitle%253DDynamics%2Bof%2BSrf%252C%2Bp300%2Band%2Bhistone%2Bmodifications%2Bduring%2Bcardiac%2Bmaturation%2Bin%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc1mb05363a%26rft_id%253Dinfo%253Apmid%252F22101280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1039/c1mb05363a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22101280&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.35" data-doi="10.1371/journal.pone.0050938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schurmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schillert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carstensen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Analyzing illumina gene expression microarray data from different tissues: methodological aspects of data analysis in the MetaXpress consortium</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e50938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De50938%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0050938%26rft_id%253Dinfo%253Apmid%252F23236413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0050938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23236413&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.36" data-doi="10.1093/molbev/msl034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sémon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evolutionary origin and maintenance of coexpressed gene clusters in mammals</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1715</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl034%26rft_id%253Dinfo%253Apmid%252F16757654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTcxNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC81OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.4.597.37"><div class="cit-metadata unstructured">Simecek, P., J. Forejt, R. W. Williams, L. Lu, T. E. Johnson <em>et al.</em>, 2011 High resolution genomic architecture of genetic reference populations: chromosome substitution panels and recombinant inbred strains. Abstracts of the Meeting of the Genetics Society of America, June 22-25, 2011, Washington, DC.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linear models and empirical Bayes methods for assessing differential expression in microarray experiments</span>. <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">Article 3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BAppl.%2BGenet.%2BMol.%2BBiol.%26rft.volume%253D3%26rft.spage%253DArticle%2B3%26rft.atitle%253DLINEAR%2BMODELS%2BAND%2BEMPIRICAL%2BBAYES%2BMETHODS%2BFOR%2BASSESSING%2BDIFFERENTIAL%2BEXPRESSION%2BIN%2BMICROARRAY%2BEXPERIMENTS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.39" data-doi="10.1186/1475-4924-1-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evidence for large domains of similarly expressed genes in the Drosophila genome</span>. <abbr class="cit-jnl-abbrev">J. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiology%2B%2528Online%2529%26rft.stitle%253DJ%2BBiol%26rft.aulast%253DSpellman%26rft.auinit1%253DP.%2BT.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D5%26rft.atitle%253DEvidence%2Bfor%2Blarge%2Bdomains%2Bof%2Bsimilarly%2Bexpressed%2Bgenes%2Bin%2Bthe%2BDrosophila%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1475-4924-1-5%26rft_id%253Dinfo%253Apmid%252F12144710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1475-4924-1-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12144710&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.40" data-doi="10.1093/nar/gkr476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Firpi</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Enhancers in embryonic stem cells are enriched for transposable elements and genetic variations associated with cancers</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">7371</span>–<span class="cit-lpage">7379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr476%26rft_id%253Dinfo%253Apmid%252F21685454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xNy83MzcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzU5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.41" data-doi="10.1016/j.tig.2003.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van de Lagemaat</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">J.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mager</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medstrand</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Transposable elements in mammals promote regulatory variation and diversification of genes with specialized functions</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">530</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253Dvan%2Bde%2BLagemaat%26rft.auinit1%253DL.%2BN.%26rft.volume%253D19%26rft.issue%253D10%26rft.spage%253D530%26rft.epage%253D536%26rft.atitle%253DTransposable%2Belements%2Bin%2Bmammals%2Bpromote%2Bregulatory%2Bvariation%2Band%2Bdiversification%2Bof%2Bgenes%2Bwith%2Bspecialized%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2003.08.004%26rft_id%253Dinfo%253Apmid%252F14550626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2003.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14550626&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186058200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.42" data-doi="10.1038/nature08451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pennacchio</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic views of distant-acting enhancers</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DVisel%26rft.auinit1%253DA.%26rft.volume%253D461%26rft.issue%253D7261%26rft.spage%253D199%26rft.epage%253D205%26rft.atitle%253DGenomic%2Bviews%2Bof%2Bdistant-acting%2Benhancers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08451%26rft_id%253Dinfo%253Apmid%252F19741700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08451&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19741700&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269654600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.43" data-doi="10.1371/journal.pone.0012158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Coordinated expression domains in mammalian genomes</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DWoo%26rft.auinit1%253DY.%2BH.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De12158%26rft.epage%253De12158%26rft.atitle%253DCoordinated%2Bexpression%2Bdomains%2Bin%2Bmammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012158%26rft_id%253Dinfo%253Apmid%252F20805879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20805879&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.44" data-doi="10.1371/journal.pgen.1000070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delano</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene set enrichment in eQTL data identifies novel annotations and pathway regulators</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000070%26rft.epage%253De1000070%26rft.atitle%253DGene%2Bset%2Benrichment%2Bin%2BeQTL%2Bdata%2Bidentifies%2Bnovel%2Bannotations%2Band%2Bpathway%2Bregulators.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000070%26rft_id%253Dinfo%253Apmid%252F18464898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18464898&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.45" data-doi="10.1038/nature10432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sequence-based characterization of structural variation in the mouse genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYalcin%26rft.auinit1%253DB.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D326%26rft.epage%253D329%26rft.atitle%253DSequence-based%2Bcharacterization%2Bof%2Bstructural%2Bvariation%2Bin%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10432%26rft_id%253Dinfo%253Apmid%252F21921916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921916&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.46" data-doi="10.1038/nmeth.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchins</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatkiewicz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A customized and versatile high-density genotyping array for the mouse</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D663%26rft.epage%253D666%26rft.atitle%253DA%2Bcustomized%2Band%2Bversatile%2Bhigh-density%2Bgenotyping%2Barray%2Bfor%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1359%26rft_id%253Dinfo%253Apmid%252F19668205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19668205&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F597.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269382900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.597.47" data-doi="10.1073/pnas.2627983100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">K.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Senescence-specific gene expression fingerprints reveal cell-type-dependent physical clustering of up-regulated chromosomal loci</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">3251</span>–<span class="cit-lpage">3256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2627983100%26rft_id%253Dinfo%253Apmid%252F12626749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzYvMzI1MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC81OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/4/597.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/585" title="Assembly of the Synaptonemal Complex Is a Highly Temperature-Sensitive Process That Is Supported by PGL-1 During Caenorhabditis elegans Meiosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/607" title="Plastome Sequence Determination and Comparative Analysis for Members of the Lolium-Festuca Grass Species Complex" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/4"><span>Volume 3 Issue 4, April 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/4.cover-source.jpg?itok=tXw4AXt4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/597.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nuVJw9dI6hYntgCJlWmF1RPF7SRHU5Q2xFpc8NVpGL4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/4/597" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/4/597" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/4/597" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459440" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LhknfYrwh0F4W41kfHMXwtu6mW6gMh0qzsvZWQlk-U4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/597.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/4/597" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hvN464c9tIUHMywEtXx6mQPDCX-SV0hm4dTP0LewzeM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/4/597" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/597.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459440" id="node459440--2" data-pisa="ggg;3/4/597" data-pisa-master="ggg;g3.113.005546" data-apath="/ggg/3/4/597.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marie-Pier Scott-Boyer</span> and <span class="highwire-citation-author" data-delta="1">Christian F. Deschepper</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">597-605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005546 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459440--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marie-Pier Scott-Boyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Pier%2BScott-Boyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott-Boyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Pier%2BScott-Boyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian F. Deschepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BF.%2BDeschepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deschepper%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BF.%2BDeschepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:deschec@ircm.qc.ca" class="" data-icon-position="" data-hide-link-title="0">deschec@ircm.qc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/4/597" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/4/597&amp;title=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/4/597&amp;title=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/4/597&amp;title=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/4/597&amp;text=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/4/597&amp;title=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/4/597&amp;t=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/4/597&amp;title=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/4/597&amp;title=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459440/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459440" id="node459440--3" data-pisa="ggg;3/4/597" data-pisa-master="ggg;g3.113.005546" data-apath="/ggg/3/4/597.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Detection of Gene Coexpression Domains Showing Linkage to Regions Enriched with Polymorphic Retrotransposons in Recombinant Inbred Mouse Strains</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marie-Pier Scott-Boyer</span> and <span class="highwire-citation-author" data-delta="1">Christian F. Deschepper</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">597-605;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.005546 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459440--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marie-Pier Scott-Boyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Pier%2BScott-Boyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott-Boyer%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Pier%2BScott-Boyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian F. Deschepper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Cardiovascular Biology Research Unit, Institut de recherches cliniques de Montréal (IRCM) and Université de Montréal, Montréal, Québec, H2W 1R7, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BF.%2BDeschepper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deschepper%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BF.%2BDeschepper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:deschec@ircm.qc.ca" class="" data-icon-position="" data-hide-link-title="0">deschec@ircm.qc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=597&amp;rft.epage=605&amp;rft.atitle=Genome-Wide%20Detection%20of%20Gene%20Coexpression%20Domains%20Showing%20Linkage%20to%20Regions%20Enriched%20with%20Polymorphic%20Retrotransposons%20in%20Recombinant%20Inbred%20Mouse%20Strains&amp;rft.volume=3&amp;rft.issue=4&amp;rft.date=2013-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Scott-Boyer%2C+Marie-Pier&amp;rft.au=Deschepper%2C+Christian+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459440/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459440/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459440/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459440/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459440/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459440/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459440/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459440/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459440/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459440/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459440/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459440/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/4/597" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/4/597.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/4/597.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/4/597.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/4/597.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
