<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/4/709" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/4/709.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/4/709.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/4/709.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/4/709/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/4/709.atom" />
<meta name="HW.pisa" content="ggg;3/4/709" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (Scomberomorus commerson)" />
<meta name="DC.Identifier" content="10.1534/g3.112.005124" />
<meta name="DC.Date" content="2013-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 The State of Queensland, Department of Agriculture, Fisheries and Forestry. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Estimates of genetic effective population size ( Ne ) using molecular markers are a potentially useful tool for the management of endangered through to commercial species. However, pitfalls are predicted when the effective size is large because estimates require large numbers of samples from wild populations for statistical validity. Our simulations showed that linkage disequilibrium estimates of Ne up to 10,000 with finite confidence limits can be achieved with sample sizes of approximately 5000. This number was deduced from empirical allele frequencies of seven polymorphic microsatellite loci in a commercially harvested fisheries species, the narrow-barred Spanish mackerel ( Scomberomorus commerson ). As expected, the smallest SD of Ne estimates occurred when low-frequency alleles were excluded. Additional simulations indicated that the linkage disequilibrium method was sensitive to small numbers of genotypes from cryptic species or conspecific immigrants. A correspondence analysis algorithm was developed to detect and remove outlier genotypes that could possibly be inadvertently sampled from cryptic species or nonbreeding immigrants from genetically separate populations. Simulations demonstrated the value of this approach in Spanish mackerel data. When putative immigrants were removed from the empirical data, 95% of the Ne estimates from jacknife resampling were greater than 24,000." />
<meta name="DC.Contributor" content="Gilbert Michael Macbeth" />
<meta name="DC.Contributor" content="Damien Broderick" />
<meta name="DC.Contributor" content="Rik C. Buckworth" />
<meta name="DC.Contributor" content="Jennifer R. Ovenden" />
<meta name="article:published_time" content="2013-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (Scomberomorus commerson)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Estimates of genetic effective population size (&lt;i&gt;Ne&lt;/i&gt;) using molecular markers are a potentially useful tool for the management of endangered through to commercial species. However, pitfalls are predicted when the effective size is large because estimates require large numbers of samples from wild populations for statistical validity. Our simulations showed that linkage disequilibrium estimates of &lt;i&gt;Ne&lt;/i&gt; up to 10,000 with finite confidence limits can be achieved with sample sizes of approximately 5000. This number was deduced from empirical allele frequencies of seven polymorphic microsatellite loci in a commercially harvested fisheries species, the narrow-barred Spanish mackerel (&lt;i&gt;Scomberomorus commerson&lt;/i&gt;). As expected, the smallest SD of &lt;i&gt;Ne&lt;/i&gt; estimates occurred when low-frequency alleles were excluded. Additional simulations indicated that the linkage disequilibrium method was sensitive to small numbers of genotypes from cryptic species or conspecific immigrants. A correspondence analysis algorithm was developed to detect and remove outlier genotypes that could possibly be inadvertently sampled from cryptic species or nonbreeding immigrants from genetically separate populations. Simulations demonstrated the value of this approach in Spanish mackerel data. When putative immigrants were removed from the empirical data, 95% of the &lt;i&gt;Ne&lt;/i&gt; estimates from jacknife resampling were greater than 24,000.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/04/01" />
<meta name="citation_mjid" content="ggg;3/4/709" />
<meta name="citation_id" content="3/4/709" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/4/709" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/4/709.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/4/709.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/4/709.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.005124" />
<meta name="citation_pmid" content="23550119" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="709" />
<meta name="citation_lastpage" content="717" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Gilbert Michael Macbeth" />
<meta name="citation_author_institution" content="Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia" />
<meta name="citation_author_email" content="gilbert.macbeth1@uq.net.au" />
<meta name="citation_author" content="Damien Broderick" />
<meta name="citation_author_institution" content="Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia" />
<meta name="citation_author" content="Rik C. Buckworth" />
<meta name="citation_author_institution" content="CSIRO Marine &amp; Atmospheric Research, Wealth from Oceans National Research Flagship, Brisbane, Queensland, 4001, Australia" />
<meta name="citation_author" content="Jennifer R. Ovenden" />
<meta name="citation_author_institution" content="Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. S. F. Barker;citation_title=Effective population size of natural populations of Drosophila buzzatii, with a comparative evaluation of nine methods of estimation.;citation_pages=4452-4471;citation_volume=20;citation_year=2011;citation_issue=21;citation_pmid=21951766;citation_doi=10.1111/j.1365-294X.2011.05324.x" />
<meta name="citation_reference" content="citation_journal_title=Trans. Am. Fish. Soc.;citation_author=T. D. Beacham;citation_author=J. R. Candy;citation_author=B. McIntosh;citation_author=C. MacConnachie;citation_author=A. Tabat;citation_title=Estimation of stock composition and individual identification of Sockeye salmon on a Pacific rim basis using microsatellite and major histocompatibility complex variation;citation_pages=1124-1146;citation_volume=134;citation_year=2005;citation_doi=10.1577/T05-005.1" />
<meta name="citation_reference" content="citation_journal_title=Mar. Ecol. Prog. Ser.;citation_author=D. Bekkevold;citation_author=L. A. W. Clausen;citation_author=S. Mariani;citation_author=C. Andre;citation_author=T. B. Christensen;citation_title=Divergent origins of sympatric herring population components determined using genetic mixture analysis;citation_pages=187-196;citation_volume=337;citation_year=2007;citation_doi=10.3354/meps337187" />
<meta name="citation_reference" content="citation_journal_title=Mar. Ecol. Prog. Ser.;citation_author=D. C. Blower;citation_author=J. M. Pandolfi;citation_author=M. C. Gomez-Cabrera;citation_author=B. D. Bruce;citation_author=J. R. Ovenden;citation_title=Population genetics of Australian white sharks reveals fine-scale spatial structure, transoceanic dispersal events and low effective population sizes;citation_pages=229-244;citation_volume=455;citation_year=2012;citation_doi=10.3354/meps09659" />
<meta name="citation_reference" content="citation_title=The stock structure of northern and western Australian Spanish mackerel;citation_author=R. C. Buckworth;citation_author=S. J. Newman;citation_author=J. R. Ovenden;citation_author=R. J. G. Lester;citation_author=G. R. McPherson;citation_year=2007" />
<meta name="citation_reference" content="citation_title=GENETAG: Genetic mark-recapture for real-time harvest rate monitoring: Pilot studies in Northern Australia Spanish Mackerel fisheries;citation_author=R. C. Buckworth;citation_author=J. R. Ovenden;citation_author=D. Broderick;citation_author=G. M. Macbeth;citation_author=G. R. McPherson;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=FAO Species Catalogue. Vol. 2. Scombrids of the World. An Annotated and Illustrated Catalogue of Tunas, Mackerels, Bonitos and Related Species Known to Date.;citation_author=B. B. Collette;citation_author=C. E. Nauen;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. Der;citation_author=C. L. Epstein;citation_author=J. B. Plotkin;citation_title=Generalized population models and the nature of genetic drift.;citation_pages=80-99;citation_volume=80;citation_year=2011;citation_issue=2;citation_pmid=21718713;citation_doi=10.1016/j.tpb.2011.06.004" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=C. Fauvelot;citation_author=P. Borsa;citation_title=Patterns of genetic isolation in a widely distributed pelagic fish, the narrow-barred Spanish mackerel (Scomberomous commerson);citation_pages=886-902;citation_volume=104;citation_year=2011;citation_doi=10.1111/j.1095-8312.2011.01754.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Conserv Biol;citation_author=M. P. Hare;citation_author=L. Nunney;citation_author=M. K. Schwartz;citation_author=D. E. Ruzzante;citation_author=M. Burford;citation_title=Understanding and estimating effective population size for practical application in marine species management.;citation_pages=438-449;citation_volume=25;citation_year=2011;citation_issue=3;citation_pmid=21284731;citation_doi=10.1111/j.1523-1739.2010.01637.x" />
<meta name="citation_reference" content="citation_journal_title=Mar. Biol.;citation_author=D. Hedgecock;citation_author=S. Launey;citation_author=A. I. Pudovkin;citation_author=Y. Naciri;citation_author=S. Lapegue;citation_title=Small effective number of parents (N-b) inferred for a naturally spawned cohort of juvenile European flat oysters Ostrea edulis;citation_pages=1173-1182;citation_volume=150;citation_year=2007;citation_doi=10.1007/s00227-006-0441-y" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci. B.;citation_author=S. J. Jorgensen;citation_author=C. A. Reeb;citation_author=T. K. Chapple;citation_author=S. Anderson;citation_author=C. Perle;citation_title=Philopatry and migration of Pacific white sharks;citation_volume=277;citation_year=2010;citation_pmid=19889703;citation_doi=10.1098/rspb.2009.1155" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=G. Luikart;citation_author=N. Ryman;citation_author=D. A. Tallmon;citation_author=M. K. Schwartz;citation_author=F. W. Allendorf;citation_title=Estimation of census and effective population sizes: the increasing usefulness of DNA-based approaches;citation_pages=355-373;citation_volume=11;citation_year=2010;citation_doi=10.1007/s10592-010-0050-7" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. M. Macbeth;citation_author=D. Broderick;citation_author=J. R. Ovenden;citation_author=R. C. Buckworth;citation_title=Likelihood-based genetic mark-recapture estimates when genotype samples are incomplete and contain typing errors.;citation_pages=185-196;citation_volume=80;citation_year=2011;citation_issue=3;citation_pmid=21763337;citation_doi=10.1016/j.tpb.2011.06.006" />
<meta name="citation_reference" content="citation_title=A review of large coastal pelagic fishes in the South Pacific Region, with special reference to Scomberomorus commerson in North-East Australian waters [WP 15];citation_author=G. R. McPherson;citation_year=1988" />
<meta name="citation_reference" content="citation_title=Multiple Correspondence Analysis and Related Methods;citation_title=Computation of multiple correspondence analysis, with code in R;citation_author=O. Nenadic;citation_author=M. Greenacre;citation_pages=523-551;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Glob. Ecol. Biogeogr.;citation_author=S. J. Newman;citation_author=R. C. Buckworth;citation_author=M. Mackie;citation_author=P. Lewis;citation_author=I. Wright;citation_title=Spatial subdivision of adult assemblages of Spanish mackerel, Scomberomorus commerson (Pisces: Scombridae) across northern Australia: implications for fisheries management;citation_pages=711-723;citation_volume=18;citation_year=2009;citation_doi=10.1111/j.1466-8238.2009.00475.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. Ovenden;citation_author=D. Peel;citation_author=R. Street;citation_author=A. J. Courtney;citation_author=S. D. Hoyle;citation_title=The genetic effective and adult census size of an Australian population of tiger prawns (Penaeus esculentus).;citation_pages=127-138;citation_volume=16;citation_year=2007;citation_issue=1;citation_pmid=17181726" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. P. Palstra;citation_author=D. E. Ruzzante;citation_title=Genetic estimates of contemporary effective population size: what can they tell us about the importance of genetic stochasticity for wild population persistence?;citation_pages=3428-3447;citation_volume=17;citation_year=2008;citation_issue=15;citation_pmid=19160474;citation_doi=10.1111/j.1365-294X.2008.03842.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=F. P. Palstra;citation_author=D. E. Ruzzante;citation_title=Demographic and genetic factors shaping contemporary metapopulation effective size and its empirical estimation in salmonid fish.;citation_pages=444-455;citation_volume=107;citation_year=2011;citation_issue=5;citation_pmid=21522167;citation_doi=10.1038/hdy.2011.31" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. I. Pudovkin;citation_author=D. V. Zaykin;citation_author=D. Hedgecock;citation_title=On the Potential for Estimating the Effective Number of Breeders From Heterozygote-Excess in Progeny;citation_pages=383-387;citation_volume=144;citation_year=1996;citation_issue=1;citation_pmid=8878701" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. J. Schaid;citation_title=Linkage Disequilibrium Testing When Linkage Phase Is Unknown;citation_pages=505-512;citation_volume=166;citation_year=2004;citation_issue=1;citation_pmid=15020439;citation_doi=10.1534/genetics.166.1.505" />
<meta name="citation_reference" content="citation_journal_title=Coll. Vol. Sci. Pap. ICCAT;citation_author=B. L. Smith;citation_author=J. R. Alvarado-Bremer;citation_title=Inferring population admixture with multiple nuclear genetic markers and Bayesian genetic clustering in Atlantic swordfish (Xiphlas gladius);citation_pages=185-190;citation_volume=65;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Biodivers. Conserv.;citation_author=Z. H. Sulaiman;citation_author=J. R. Ovenden;citation_title=Population genetic evidence for the east–west division of the narrow-barred Spanish mackerel (Scomberomorus commerson, Perciformes: Teleostei) along Wallace’s Line;citation_pages=563-574;citation_volume=19;citation_year=2010;citation_doi=10.1007/s10531-009-9699-y" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. A. Sved;citation_title=Linkage disequilibrium and homozygosity of chromosome segments in finite populations.;citation_pages=125-141;citation_volume=2;citation_year=1971;citation_issue=2;citation_pmid=5170716;citation_doi=10.1016/0040-5809(71)90011-6" />
<meta name="citation_reference" content="citation_journal_title=Mar. Ecol. Prog. Ser.;citation_author=H. Svedang;citation_author=D. Righton;citation_author=P. Jonsson;citation_title=Migratory behaviour of Atlantic cod Gadus morhua: natal homing is the prime stock-separating mechanism;citation_pages=1-12;citation_volume=345;citation_year=2007;citation_doi=10.3354/meps07140" />
<meta name="citation_reference" content="citation_journal_title=Journal of Fish Biology;citation_journal_abbrev=Journal of Fish Biology;citation_author=B. J. Tillett;citation_author=M. G. Meekan;citation_author=I. C. Field;citation_author=D. C. Thorburn;citation_author=J. R. Ovenden;citation_title=Evidence for reproductive philopatry in the bull shark Carcharhinus leucas.;citation_pages=2140-2158;citation_volume=80;citation_year=2012;citation_issue=6;citation_pmid=22551174;citation_doi=10.1111/j.1095-8649.2012.03228.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. S. Waples;citation_title=A generalized approach for estimating effective population size from temporal changes in allele frequency.;citation_pages=379-391;citation_volume=121;citation_year=1989;citation_issue=2;citation_pmid=2731727" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=R. S. Waples;citation_title=A bias correction for estimates of effective population size based on linkage disequilibrium at unlinked gene loci;citation_pages=167-184;citation_volume=8;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resources.;citation_author=R. S. Waples;citation_author=C. Do;citation_title=LDNE: a program for estimating effective population size from data on linkage disequilibrium;citation_pages=753-756;citation_volume=8;citation_year=2008;citation_doi=10.1111/j.1755-0998.2007.02061.x" />
<meta name="citation_reference" content="citation_journal_title=Evol. Applications;citation_author=R. S. Waples;citation_author=C. Do;citation_title=Linkage disequilibrium estimates of contemporary Ne using highly variable genetic markers: a largely untapped resource for applied conservation and evolution;citation_pages=244-262;citation_volume=3;citation_year=2010;citation_doi=10.1111/j.1752-4571.2009.00104.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. S. Waples;citation_author=P. R. England;citation_title=Estimating contemporary effective population size on the basis of linkage disequilibrium in the face of migration;citation_volume=189;citation_year=2011;citation_pmid=21840864;citation_doi=10.1534/genetics.111.132233" />
<meta name="citation_reference" content="citation_title=Genetic Data Analysis II;citation_author=B. Weir;citation_pages=137;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Wright;citation_title=Evolution in mendelian populations;citation_pages=97-159;citation_volume=16;citation_year=1930" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=O. Zhdanova;citation_author=A. I. Pudovkin;citation_title=Nb_HetEx: a program to estimate the effective number of breeders;citation_volume=99;citation_year=2008;citation_pmid=18703539;citation_doi=10.1093/jhered/esn061" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (Scomberomorus commerson)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="twitter:description" content="Estimates of genetic effective population size ( Ne ) using molecular markers are a potentially useful tool for the management of endangered through to commercial species. However, pitfalls are predicted when the effective size is large because estimates require large numbers of samples from wild populations for statistical validity. Our simulations showed that linkage disequilibrium estimates of Ne up to 10,000 with finite confidence limits can be achieved with sample sizes of approximately 5000. This number was deduced from empirical allele frequencies of seven polymorphic microsatellite loci in a commercially harvested fisheries species, the narrow-barred Spanish mackerel ( Scomberomorus commerson ). As expected, the smallest SD of Ne estimates occurred when low-frequency alleles were excluded. Additional simulations indicated that the linkage disequilibrium method was sensitive to small numbers of genotypes from cryptic species or conspecific immigrants. A correspondence analysis algorithm was developed to detect and remove outlier genotypes that could possibly be inadvertently sampled from cryptic species or nonbreeding immigrants from genetically separate populations. Simulations demonstrated the value of this approach in Spanish mackerel data. When putative immigrants were removed from the empirical data, 95% of the Ne estimates from jacknife resampling were greater than 24,000." />
<meta name="og-title" property="og:title" content="Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (Scomberomorus commerson)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/4/709" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Estimates of genetic effective population size ( Ne ) using molecular markers are a potentially useful tool for the management of endangered through to commercial species. However, pitfalls are predicted when the effective size is large because estimates require large numbers of samples from wild populations for statistical validity. Our simulations showed that linkage disequilibrium estimates of Ne up to 10,000 with finite confidence limits can be achieved with sample sizes of approximately 5000. This number was deduced from empirical allele frequencies of seven polymorphic microsatellite loci in a commercially harvested fisheries species, the narrow-barred Spanish mackerel ( Scomberomorus commerson ). As expected, the smallest SD of Ne estimates occurred when low-frequency alleles were excluded. Additional simulations indicated that the linkage disequilibrium method was sensitive to small numbers of genotypes from cryptic species or conspecific immigrants. A correspondence analysis algorithm was developed to detect and remove outlier genotypes that could possibly be inadvertently sampled from cryptic species or nonbreeding immigrants from genetically separate populations. Simulations demonstrated the value of this approach in Spanish mackerel data. When putative immigrants were removed from the empirical data, 95% of the Ne estimates from jacknife resampling were greater than 24,000." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
    <title>Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (Scomberomorus commerson) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"96SecPSRy_1O_yS9TMiWX50fgVRlkuJ1-T3Ayfec4AU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459447","apath":"\/ggg\/3\/4\/709.atom","pisa":"ggg;3\/4\/709","ac":{"\/ggg\/3\/4\/709.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/4\/709.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/4\/709"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/4\/709"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/4\/709.figures-only":true,"\/content\/3\/4\/709":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GwiR9n4eWr3q3mNDWBrat4QwLOwz88teQgytFq+EkBOjY8zpdXSknLqZ3B5GgJNv+YxusmliTna30o6wp29xPEJK5IKIe4qyoSKGKwfcfLD6ugAqiX+MN64Mh2JPuNRJH5v\\\\\\\/3uR+4p3rT472RWVGTlTKpNmKw5MPXmQBy6MVdw6RFzj+H6IrTRi9pOWPENIQHABK9SlJEQK++calH4br3JsN5AMJg3Zx+H7AX3z0NP71BJbp56GrLUTp8WBfBVi9KTDX96diueIunFb40uxhzi\\\\\\\/EKVSkk74NEVlhU4i6NKQ480s2qVQRc+MFZX1sX1gxr6FWlToIibfE9a8TB4EfOPjjjNaZtqOmIOIrnS4DpnnO+jP7Ny2ynfgiFO08KMd2cOQ\\\\\\\/OVO8N+heM4TkNU\\\\\\\/xEgztBseKL2OVuxqxDirFOn+dZfF8VaZ0LKY8OgH2DDuKtlCjhs1LMHZpxkOeuztrxbx5aJ+mCepw1gj53xUfoSPlNnL+9PACf2dX9gBgUV0T3JmO7kzZWtV5zOMjaApLWci8zMr2Kn1SfvqCSYmMU4L8uGfVfUXzJ0u5pUrHE+PR7ULKSbBFCIAudSYiFkGIZMElffj5zkps16vJPZOWBvlgX5feGJ+KEmDyUwT+BnDu+0xlmxCqGrIOdxU+DIgYmd7jMLVHvMzD6EqlzJSl4Jguip6o99wekQ9nye+4\\\\\\\/4q4f15AUf30Nt6zbPTG1RlEW9zvoSxmSZ506YTVKm7t+jKZ6JClFTMffCOVNKz5gOx7sJ837NS78JKilqAYygrzDb7Hhd3wnJ7yR1GIeEQvyROOO4oUDAR8qAY5VPBOmIqhuSDX4KVoVJMhGQj4IZelT50kWTrSSAECeJPx+Zacr84v5guVjmY2pGepdWU828oE5IcqC0uwY5xlQd6MTZG3MkC4IbYzE144Xt24At7GD0WMPalpJd6BayBJhCXq0kIXfcTacgC62gorLM3v\\\\\\\/GfwpI1suhjjNN2xVKrfsCMg83CCOIpfJHQDm3t7m2X7mpqlsgTUPykL9c\\\\\\\/B1x6D+l8FyYUDChblBA0CckY3zE\\\\\\\/xL9Gb7B893KcSeaehekRF3lt8F31PEu11BasRz9OuCH2esxcndmsxq6prjiTlHNkg9MyD8h5DVVo3TTLicUbHI3zuEFteee\\\\\\\/tnWPJ1c7k4LcE656faCvHcgY3hnq2pujBK\\\\\\\/wa85Wrj6V5PZNV1CaBzIwo+3JlkIe3lq07UYbqbESnR\\\\\\\/DsMzb1j7zQmf7+st5qAXAExvIVXnaHPyqGNaoiQy5l\\\u0022,\\\u0022iv\\\u0022:\\\u0022GbbBHtamec3tYqgNqHF6yg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fb27bc41cdfbe09489a316f0020a02abf9d57de4c046ed1a28885007280a0846\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223P08zzFA9ccgJtQEYUCi5z5StjbbjgRzLmgNCyjZoGVLp+0NB4JtH4EMXzI5ZRNuqtUTS10PZI4WXOANu4Yx1jZ2xBf0jEGWuXPpx0k1Q09PKf8Q2akSpTyhQZJKopPLZTyeBQvX+pOD5d2ISiBiYDZT5OvQBb2tUMFSylpM8LPPYYKs8dtQ4bKupvN7OjvNwWm\\\\\\\/+aHlpymSswQc1TTShpLu3tGHnl9l7BY\\\\\\\/0m1v+duJIJJ\\\\\\\/NCOQ1kDSE428bUoT1MKgMR1+Vfei0eu1PSeV1sG1fBOGi1yroTPVkLjHZM\\\\\\\/8e8o2hHxEcrQ5VMIpEiaaDWdXmR5jx3yWcNwMbZLAxrw1\\\\\\\/1wnJFPuREroDbNA9w5sprK2rVacU+TGWaUl7rRHHK5kEA3Yp6\\\\\\\/RoSDm8d3rnYv\\\\\\\/dfSTz+kpS+Tym1tGvOKOWB9w8WlF31bS6Bg9oz1zjyR5bOYk2xcReRaya1LpjvjXQfWx3O0ki+PCAiaKfLy+R5ZiGPDRIzrjHz4T7UmKTqQh5Gkk3R6sKTjRecA2FJ8rwRH7z7Udel+WuPo0OukHKl7PQvWo2ZN4RM6mnGbxQLI3Uy0+MYpwUOTQTLncnJrwpjYpp5fMUxFcyo3XA5juPJcdpz4Rasr7VWDXWwIxtht4WaP3TqP+zA99f0tz5wsgS5AKvM2c4snQ0hQF0sSeySv4r27O6mDbGV2Pm5KowPzPuY0YYava3dbt8ve3Hn1rLVYx1QZRhjpa94p3YdCeZFwFQSkH0SW\\\\\\\/u4CzOCcHkwSTuHTwTtGgsPKDz6iOXvqR1DPYhG8rRbdKESlStQywf30cRgdEniFaK39dzHzUfGm6UA6J7Dy3bXflq6oWTwmLzxngS0lRxw6J2PgZc30XaKvDInsJ5eFzLyDE5UYtZxBQZo0eWmvijPUWNb8QBh3RMP9MjsaGlH3quNTvnzgGNdt04CB19aevRroi+e7g8QWi05V3c32+YJfnUp9OEtJEpvepYTznSICKvAY0ipwv2UwB0tc2LuppSl40i+NpcKtwb+yLgUpow6rj2vDNurlwpo58CqTHLIiXwtHVrSs0Y7Wu1Sn9KjZFI3DXMLa0oQWFZ7YuNUwDYajcdFQL5Hp9LJfKlA2fFQsWVfRppeHuOOaGg3El+ccgbYQMAuC5F83XrnwCuB20DrJXWlpbk9SwwW6VrUKOtppFUXx+Z\\\\\\\/pfhDqCnYWP1Z5qi0Y0ffYkFQl2jdma9dQTIGdg0CNB4jRhMRf\\\\\\\/XzBs0E54WrYhkdpM1A57cTiit69\\\\\\\/D3MzLSFYsbDObl+3BBaZztVa5i5l8ETCOrW91gW1lurDApzYn9uzrcM46ULSSX6I4UAh07zEZ9x5n\\\\\\\/KSlndMKRG\\\\\\\/+yRkl0+LV7DBkSC8jmoDCxmYFHBFrj1yFiVC+Vy5Y1Sd6zWV97qozvXM86tMNKKtQzeTnq53psh10u9VWgtGV7aWTnDHTLXjc3I2D\\\\\\\/HFFk9yUNwRgbdnhCnLqNqGPybLyPwO\\\\\\\/BMZxWjvnuyicsBp3G5Iy2MNiJUSgCShIZ2NWEyRFm0orj0xh9\\\\\\\/C2Yd4VEDH8+HWc0HYOgGctwFmtWsp35OfTWuP9mLQcM0OvtjzfhrC2T4KRZ+DUNsCUpcmmxn7f3AQS7N4sxsN5Vc2H9H9OCYnGQYE6z7HkJWNGUiPbJRD5UXv5RPNJxh1Bhe6H92rvvks6lpy3pXRqk2v3bLZw4hU3PB2GuM73MUTx81YU3X0sAf9SmgA5+SL+XXrM9eF9p2ABvDMswdlIfyFPYorwp73a1Mz6OKouORw7wVmKkSUL+gH7vft+mrkiot4Rt2HKinauwKrLNql6mkZiNufOg2aDJ3BtmQmpzroASsL4lQLAe7QbgnjnC1zpJKq1C57vhcyXOdWdSqjdCPO0lTxgfqSIwM+gaKQ99\\\\\\\/aFiHRtw4hSSYWJYvX32M31PX5LjErz2\\\\\\\/FMb50RbezmW7dETESclcteWQGyY6mtHSb+NTFsx72g5ij\\\\\\\/ycI5nytoGqSEbYQZQXzwRYkK7eicfjxcLl\\\\\\\/5ud9vG99lSLrSWRWCqHaji6BAoXlY4OnGxiowxpWwkyGrorYa9jY3RLsJtLRtWOspH+cDa78UTkdiCVqOSD+BTLeVygK9s\\\\\\\/FRXfNTldmoxBxtJStlonsvzqIuraMHyoDrKNfA2yqYk9Ff2OC95BWiZ5D9FlVBfsshcR1W4PLEPKdeqSOOycUmDIID3m\\\\\\\/RCHLoUAg1Rw1OApR2nzSBw4XQbmHe+TWvn7GuasXEAFJo3U8TLKcYn+HjK\\\\\\\/uahKK6PlAzpkZdORRqm\\\\\\\/9+MrksRDpnnJC+WwLNFSM4YllKe2BqbaqPWqPFPepTtKPdsLG4YkgVrLPvhLfpfqobcSHdiDaBSTKsMG41ZxQb2\\\\\\\/lY54PugjZrPGDVzkxZf2A9xopznlleq6tHYZ6suJVGaz0Nh+rCm6LWY9HdTsJAodNhAShLNsAZItlU+Pn5b8RGc7RBV4tVIsHIyOMMo5Jc8+0jcOkY2GS5cN4tAxZf+Y\\\\\\\/imijImi+WeHtRr6dmHUxMlMWNXIOPkvjVJm02B6Uyti\\\\\\\/zmXRYLWWw6oZBaIU2ISq4Q76XAdF3pXMiqzi4jJLxGBPCCOGwMSAW+yBDkys14KOYxrlXxbYrKfDCxgBD6N4gpdcPIfFWKtaKb0U\\\\\\\/cjV2JknYOLN6J2z7iqah5U4mL06LwdQS1cRNnh1QlCK6skc3CEAs1p4JUiOoHwXGVs2E1yl0l27crexp2aVNygmTkzit1Z5Upum22QkwTWwbQwHqbua02SXoPLR1SQMUCMgJ1Jok1+KendhDErMRyzg2mhZ4tIWzT0SXGWOAXbkMMFamKHE1zynqyySZ3\\\\\\\/hgrsoPUozeg8UNXxoWPlIB0n7YMTAaq5LKge475MW11huPEbbO5QLiepkWnL5mpl7LoPurKKAm5YhsdkmgFMOLswmFVFE2uYxm9Z4oPsrcIUtZcxop78kFG7f9KSW1tgY\\\\\\\/7thiDtG+YGWWm2JVqhaVyCkVWXEaQGw31\\\\\\\/L\\\\\\\/jJDxZNGy5F4OLQZ3\\\\\\\/IeIRZktBKnwFcSJk3qtfpoDJmHFePW1UBAtRPVMPXmx1nP9VPiPFjtb5y7ttmafDtpSqGfcdteSMTqnGgc7a2M6SSa\\\\\\\/QCkcdRByuo0JI\\\\\\\/US3rayxmbXlF7pJfGoHA1K9mR3mLUcpMYq5Q6D25oSwvi+5Qk3Xpe7oU\\\\\\\/M+6GPTKIYDh2h9pV+hcB3Qk6CIupK3l3JyhP1LUZv5AXA+fcpJzqzsjm4pL\\\\\\\/pRhJhBWUsr6BKfx0zVYN+AGAHLf8PrZCr2ZE8V8Rk34PmMtTN6pnaKXO0vobUv66GJ2smFcxdgG6j7BOBw+eZBpFuDK38KjGrV7wfnuvKO45zBsXUlxeQWt5hsZeQC5zPz6i2eT8gE\\\\\\\/xSYxirUmd1HAQuLv1pIhjibF9rwulhTDPpF7S+K6npDpjjZrqkJhCgz7chfSOL0oEMuu6Tz5rOInjkr5\\\\\\\/lLSKITVyEifleajrU38Mo4D992Dqcl71iPaAesFhnXD0Sj8Z457NaxmFnaQrVrVbu80I1MNAdceSkso8elSxr790dSkLREJRANhVzeZWEBcUGlAbO8JtOw4eeNMkM1wSOA5UALRTZKjvpiqhDL2xhl9RJd9hjifAgB9Zd602x9LCUku99toPw4Q2K0VLN0yC1Oa2zVCoQjE7uwM9zcqDa0Xnb5DBGQ4iJYp++KKJrlAw7U\\\\\\\/P5ukbbb8oRuWu2GKk\\\\\\\/8iajj8M0x5MArQju7dJzu\\\\\\\/WiveEY\\\\\\\/pXrUynAy2gKIqf\\\\\\\/El1lpHHLcuQ0Y5uqGEEV9yQFtxZfIkVDSCvCbDCdDGDfQuW22Df8sX8boRliqvivf1llYskSpQ4eLVpro\\\\\\\/BrjyVq4bTMUUBmYa6c7uZ7EVr6bmqHOLjZG49eNaGsoI6vPtZ8+KB0oDufY2kr2mv1dW\\\\\\\/C6jOftityGv8YqRcGS4XeCYB3sQLQjlZ+wIVodnLbOAa\\\\\\\/uRvytDDxc+h7HLXem06Wp8KF0xhp5zMG40evhMEg2YgYAUNpfE834KIc9+p\\\\\\\/\\\\\\\/+bhcCcVEdz3dO7cdtw5pA9hLs1yfhRSNcBaqrxqR1x84SLMaUwq0gxj05Ez2fGrFCZ8IavtX7MP6PheZ4yksTzpy7\\\\\\\/ryTJS9j7BRGJqmSpDovWUpwYpa2u9IB5EzBkp4EgdIVTkJ5wYJYjtDjfEmSy\\\\\\\/ycRexWSiJ0wH7Z9HHe7+MpfFGJXG+JE4fgbOlErOf0sXv02MhZzGaUMfX8Ho5+SoJ44Rgp2ZeINkB8wqxREqvYZgqHVh\\\\\\\/DniJTXRjeHm\\\\\\\/HSbMe8VeIVVwKQTWIaRQ5JtLG5Ov4b+vPyO83zag2+hn99cpN\\\\\\\/gPvzi0WadUbyr0tDrXaQAElsLfVfdcWN7PeXvJmenhaACfXB4\\\\\\\/40Bay5eloUA7XoInF64cyIuc0APOmu3Ky1644zKgd+KrIRgrIZuOXh6MI4FuLZyTYo6OoEsL2QQKAQM93C2QpATQeb0dytcefHWg8xMO5o0fyQt+P4cRL2BSDTPITAJIVnTYtMSXB8e0v5doWpKzDIDBAP5vCswn0COokIlcL2e3xrmcL7Ge28I+IRgfv0rXAiETWalDxkn6l37ZYzbMtk7WMsZntAmLbf2ePS5tYbk+ljBBNWIp0Ecoh04uZSIBbo4w8iSoRbWaFZRjHM1+u1AJAWdwDWf\\\\\\\/kGfXwjjjrfHTgXAIJAGXZzycbdow93CtaNkVO0R+C7iqBDTo1MZHhyclfyYK4M3APq8FIoDgPyaJUpJ6DTHGSVj79MLSYoDJIp0l2SFT\\\\\\\/TkzX6Z8IrkfrF6mNQGO9PlRBNFZX0pjwg9DrPDaH3mUrzdX7Fvi8ih609Qt3x3aOPzsSUQzuB7SWTsysdrYDmCmcUvFRgViRI4MsNcqKScBnP4U76OsICkSOrKSHboOyaUQeBC1h+LVszypo6vMdmkxwG65IHiuC9WuqFlY9\\\\\\\/akT6zzCkLqpvMTrdVmiojLNsZbO4N4NQyHbBi0DPiJB\\\\\\\/hSSe2oUXFmH4JWNp+UwBna1wONKl\\\\\\\/qjexjs32nJgiPxTss\\\\\\\/617b8KEaNYd2tVGJAMXASLoSrPvEB6Htt+dE3aV\\\\\\\/RUcb7jCOjEsBDEMadvdmFsoWXJHqwBwZAl4ieXW2K0doY6vs07j0vccw4\\\\\\\/qjrboTIf6W7M9bIpxBpjS6Bb0le7fGAc5sAPNR\\\\\\\/ue7N79pSEsTBNmUoshHacQsNtHBXQUfDVtHsQb01MMfK3iMqoOGfkQBortfb0EKdCEGOFppZlQGG7dZjHnGwEdfZJMVdmSVjKVD+i\\\\\\\/VnEJvS+7awhsVR3d3QYWIPmUjIWQMFecRMeNWgTgQKorqXX0XBE1+021EqqHGqmeyIas3ke7je6kz7RkTkkjzhmnX7mKsQ\\\\\\\/6tSAO4Jq3U9QSN\\\\\\\/tJXSFydmriyCD6VO\\\\\\\/cC\\\\\\\/MW8v86PQNEP8vPI89YaNL2FQVZPASRuZy+mT056Zk9tZFZiIA3bfJzk6KXryButuMjgVvGMqiYUkEwwW\\\\\\\/yVf9WOWP+sdxI2XPhG9I\\\\\\\/QN26rkLQH3ErZviAjSzjcTzij8ATGGwZJmPF3wMGrJLWoBE6uUTqvscY5Nepj\\\\\\\/wVC09fr2DU6p+v0SK7d3nCI2zheW704SF1UNHUcQFtqbVIJy8Sstl7J5LeH6GX4hfX8S092ZjtHtxnZnlDF5RH1VSIIN\\\\\\\/1a2pETsAwn8vtPNfSpZk4BOYhOvT1R36nGtatqcn009mYItzJazhru8SfVQLTnesyGDtn7s3cndeTI+q+uYCkSeKpbTFkXQbS2Yv8t\\\\\\\/hx7VCjz0ZgO4boSa1OPNlhANrDMXkDff1GxwwY0YsSKfX4vv2FX8woRNUKneYEXOl9sdagAuY8ZD\\\\\\\/fgXdr9QqbOkIUhXaZ3K3eqmBvBYnL12GJnujKLA17IgnnnzoQDmNWre8oxU8pRh\\\\\\\/kEqy9xNoTT7NaA5v+gA981MhFN6UZ85dHUNj\\\\\\\/Y6LeRzEz+GCetZ1f\\\\\\\/Xsg01w5YTngylY\\\\\\\/brKTOgJgFHFOwi1MXpnFC0zuD8fDE5bpuU8zMdxuWP9I7rlIQxayoq4mwSWPMtGXQB+J1\\\\\\\/5nRkFATeS9DGWB72rKK6fcuAYacusJUhWqbqe4Rucj5\\\\\\\/Oc+YVTWYZOCB93BVZURTUgPtay+40BuD3iHDHsElEUOgwy0HX2oM15hHRZ8ekEzBPdaPvzXUscrYCWVrm209B1x27eP9ut2bTe9YjMJMQYKi4w+RCk\\\\\\\/2K0+KU03IS3jM3xR5TOvC3NKh2w2dfmSI6Ns5uRDwb8Tk+nagfA56V7her7IkR7cNfpRV1GRUUBa5RxVeOiL\\\\\\\/sou\\\\\\\/sBI\\\\\\\/07lniUub++Cta\\\\\\\/eqtC1AClB7MCo5mpFlivigehOnqwxS9XZaBVXePyMJkla3Tnc+5g3URInNaJmjltGCshfZrQfF\\\\\\\/8cta\\\\\\\/zsE3Uy9EaWssbhzQ5c8TbIKOoz18vTOVQd5XD4hW0NjozFGHCZZ7tEmHlL8vpH4p\\\\\\\/uzbWQFzP8Wziev0FO8C7KeT8s103j\\\\\\\/od77jSIO6ZhU8So7CEd7BYRrhIWeqXe4kevQdWfj0vo+kg5Qq+uvZViNIZiwY+QLym5PNQ8l1a6+Y8qOU0nSMvNyROfPxohOfqcfJZ5MhpyUvqR9GclB5SdqsJUw\\\\\\\/Fbk5kuTRzkznkPOL78bL+fwuCF5cE5cKh4Jpc35rB2Fj\\\\\\\/ZGkWny2VTlBZNnkAYcDurfH8s8pea5vAEKfRHdSgEHksAy5r9r0YfK+1kC5a8CehqM\\\\\\\/8lXEa9DYjQMAL\\\\\\\/OxeIFQ9oDN\\\\\\\/bD\\\\\\\/r761XzIXJiNYjqB7x0UX2cuiXFvccuiem8ejfHyGMFKtZUWbXhMB4pYCS2xR8Ce6BiCOYfWFxC3oxtPQXynWo1XwBnO0RHeOzY5qN3pHCRWgQT7rampTRvQoSGMHDhPeS3eZ0gl6s4sjp5cZ8ViDny6nzPJfY\\\\\\\/\\\\\\\/SkWZTfBnWmbsQFdeCZJeP6THszKx\\\\\\\/ximyDBSAuHovIu6X1edu+H91RE7h+Vw3EY\\\\\\\/CoHFzX2VsZk6azWG2dAXG5AmvWbt4ZLvwZgxw\\\\\\\/c2acTo4ugNSZy4a5g58fEPNj+3vm843ZlDqUPmVRv0EFB7pE6Fu5BCmV3t5HfecKaueeyDc4ZMdo9IzEZY+ES8PA7YIw2a0i7HgF850LOz9nWPce3N\\\\\\\/Li0Pr\\\\\\\/RTfRdC7eDsrjiJM58L04OtlODb5GOGHSa9ktJnAiGAaKnIkRH3b+45MGVPP6AZaA6sCRn7x6DWc\\\\\\\/rFjUI2ygXOfAwI\\\\\\\/66XquOX021rETVgflO36q+56c7ijuPf0nBbGdizRa4hdnJpnSXP1aLiPJ6VAG8+JUpxnZnRvUAjYNXlASPIsZhs30FNHcf0SQMUm1OYrOdTbXzUoTXwTt+ulOzoFWPf3p74PyAliin0BNltrBhNMiY7\\\\\\\/SqWnrRnKlHPQIe9IJ\\\\\\\/EMagnJ3ulOlonFDTVskOzgJ2Y5Z0ncHTvgcKOO2EAr8Oo4jWLRETC7WQTj\\\\\\\/kOsdpwktN\\\\\\\/sVtCHNbqbwGgBIbcgKNilPr0GcLYIZgn8ul3GoyZql\\\\\\\/BBLenVivvteghvzT82nRzY\\\\\\\/fb8rnxvN+Z\\\\\\\/2BkE9kJCV5dWJeW3dtiShQdQt0YXeIEFk8cL9kO3x1ZiRPqWYbQ3Nbl6KDFHxv1GwlC5YKA\\\\\\\/hTMFLkfsdcgYc1ph1NttGfi8LxE912V+LNklNT\\\\\\\/aw35NbGdPR5XcPTk8EJVikKuOG0ZzZJe9mAnKUjWJYcn\\\\\\\/3d9ug+O4Q3yrXEmN52xJ\\\\\\\/QYu4XxcsSCRfY1WjRcF6lN4IWgM\\\\\\\/OX2Lq5cYoQK136vv1lW+BMQN5pu5oTKIpJS8E22Dnx\\\\\\\/myPsf+8e6H\\\\\\\/y8r\\\\\\\/1SShmHH\\\\\\\/oaXyD4TZf\\\\\\\/eW99PoMZ5T4cySdq40PMKPN8zLDSnIGkGfcfOfc899T4UBKUffVBa4u8L+2zCUbo7kH\\\\\\\/1\\\\\\\/r8oNruCwdwFMnWMgTFPHknKtzx\\\\\\\/Tn6piXteVKjOUe+kZz2ZuX5ens5rebqKKsxBmoqdv7htq8ZQ43MwmRmeyVb1YSOhNXKbpJb1\\\\\\\/Ye4SbrC9bJf3SoHE22SvssjTAc2aiv2M8vhaW4tldP2iNEoEsqG35f0jA7Eb+XNXWFzGhMK1OUOyw\\\\\\\/OhKIOrpg4wlQivfTd4wsCH1PxZI3FDvheWmtnzPiDwpXZtlmpIEpUamw5pj9gY9VhWjNH+EqUVTfpGR7U4A\\\\\\\/CHQIZxmI9rCpmx+Odeat8Uq3FfIQbxqDqrgVBzjy45TNfhV2ttgb7FpbMnIfLiqdhM796kiOdoXEh4\\\\\\\/qwnIhQAgljQGOARv51UvmXYr93Gt0LcCKkMdMe8G7754cvCHF38Hl2O53Hkk\\\\\\\/p4t7F0KTdtNYCQoeZHtHj\\\\\\\/YsVPg4FRLOHx6noabvseCJ4CgfRzT+nbc8SUJuwhQjJgCw5MMjo5HvaNc+hI9tMhjyDPzbAXeG47brqgiqgJNLFy3ncpIFFAw+cvwoqlaS6fg7Sju091oqHJWVL5\\\\\\\/bhDvqSyc0cw\\\\\\\/9nqH900ysR64G09Tpbgyh6EG5ALO\\\\\\\/7R6jbJ6kbec9TSRlwGJZuhqX6FoWxrXcQ29IamQ0gSrARvHHdrTC8o0Z5U5Qqt8rio6Fb4H809HqlVaNhaYA\\\\\\\/hbG1FMdcUvqX6Ph\\\\\\\/X45SLSARTrVx1LVPghjHerrli41ChAaIyibTPVGoPmGXt0bU9ebHvwfvskZmEzi47+FYBQejPAME60P6rwugbaSnvxrhtu5SkBiqQu699LyqWZy0jrA\\\\\\\/PyEj5ONGZBdqMXUcz6YYSgNPh7hyzv8qgYZUDpdsWocQDZosZ40m79xLp1IHByNvKg9V4cfzcHGXAP+5mHc7qISP0KlOKRDcxFAEf9FNJZ3DzmCfLN3U5TQ5VpM1xpJc9a1MWuxPB8heR8LWwXUkG6p2o\\\\\\\/Bkq\\\\\\\/LLn23AtnXvGM8qQ6hMFDE6TUe+DlNzNXp74ep2QmrpqVla7ej53yYI0Xdnbnq2wwHVqPW+w+t0GFrmv\\\\\\\/BCCnDr15wn2p0dKBnFCND5TF3DncqEs5HWkCyM9qBznM5LYbkgQJL8F2dTz79yMAJjP6bccn3rcyfuDa07qPLs7s9X3C2ka2zaugXkbkvxkeqcVwFzK7GKySbsEWTDxed2KZclChx7tVzPgV4fzHlU1ND0GGfYj2BXBpKJFENBMo1Uh\\\\\\\/yiCaZ7DSDL9HBY+ICvsoGrn5qYUESezVaq2dBBNT4yFza9goskNmEWjz+Mjf0pnLF9mkXYM1uj92gbDY9KsTbv667LsTkA4FLICHkrZm2SaVuKVFgQti7D7IzxMWW+vqC5vS3A1aEhSd2Gfj1\\\\\\\/1LEliRBJDmKqAUvW6ivryolwsLOswhsn9IRtOxgcpt5zp\\\\\\\/XKMYrLMS3BRUDAESB3QIX6wwMTJ7dAcHmVH6RkWVL\\\\\\\/JwXv5vncmQiyi9AygVx1N2FcPPH2c4y0EXNKA9nwnAT3w8i4qov3MJNxWn5F\\\\\\\/bXVgWY0vna\\\\\\\/TcIC0r\\\\\\\/6h6nOnDhCA8E0G3J+8odNigYIFIis5cEDy2tTfnQEnSX0XhFWE4oEhFYfXLl0viVIphf88L75vbuhb9RmmAxZ10lBJ52Aiymopql42rVdtJ8VnwU75uDrmR7oI7morfHOECA\\\\\\\/dqqg6nejvn48PLE8zlt1kbnM\\\\\\\/xSyh6brE7mVikJ54muWvY+gaJVAZoNHm2\\\\\\\/WrPpwcTbuZumSMJThCc7kjW2bWcrwRdiyrEz8fmwVZynCgKjCL1lcr+iUzlxwm2z4asyISXI5r5E0sV4l7jqjQDjp6RrZtInQW8LVFn2ipI\\\\\\\/Z2mN+dbte6DKKyoxFsE+nrKoexKkZ8HAW7b3HhCMpRePc+UE++SQ0WjsxKk6U40polUz23GUxF8ImO50TRIfWEaYBL7OoCeaw0YJ8ABo6sn89eW6HOnNA8VR+ghtcAbgxyLvGINRub1p+R4HwvXEvy9BfQ+cMlEkHF6OAPIZuAqPAAG+D0mud+XmLZCB15YorJEH1l7OE2j87MgogwoOHgL69eM0TD0d8GDpDIGolO6mxy63Q+Gk+tN\\\\\\\/nmpVFGv9tmVwEEbx3+tw\\\\\\\/1VbWqusAjROYI0NJLA6r1YgPafKMYDgz7cqF7sIz5WfFW0eguUOwWflk+TdeFKn8lP8VZfwttsEwylDyoTlKT3ih\\\\\\\/kKaxxA8rRf+RuX12hJGYT83x67qCVYi8ogHmcWK6LooJzDSr7mTAms0FBlVUgBol\\\\\\\/3L6ai+9Gple9Yyztyd2RdS+Xe4nbOC4ZCgZzudvMXd4JBU2ddzvRo8F\\\\\\\/o+cujTGfSi5EV5t23sTduu8Fqn\\\\\\\/hw28ogPUbTO4Q4szoI1Xsg0f86zADm7vTYj6RZGhaS4ZgdxA3WdqxXOMyBJ6nhcLM1\\\\\\\/fSHMG2PRRoL2n1vjR3cEE7AKyeMxtjAYzFJyVZWo+pq+dzkrMO8u0\\\\\\\/qoYXom++\\\\\\\/Aapev5hQU5VZlA4NMGJapSpKqyMbwSg7grD\\\\\\\/VmlGiBze5JRBcsbzFhJeYff4nNUYq9rHMAXrEFDgm8iyanngMh93kbRB7gUTW2BGMo3Uo64+9GLdQC1GmjvbStuyaNabSMqc3R1LqheKgESqvczVLcGYdAwoctyDeTKXhcQuVqZ2guZrY\\\\\\\/bbAjrIMCKoug+HvxmnRCafV8MnvSOoVdm1MGnk8tlGLU2lA3jarnMgV3IDDU05cdaRHuJJsA\\\\\\\/u5wSfZFKXD3GwzlmFpYGDuMbVDAIp4tzR2sdrjSKSn7RTLNa+REJXfEHZjhXMPK+Onn72NAdAhc55d8P4pSlymHklf7CmUlBvJCY06qQcCFGOvp4PdQGNI\\\\\\\/iXoioxYorNeIgzC+IEW0wwzOGQ\\\\\\\/0KLT4bZWGFzMZBh0pJ\\\\\\\/oaV4kWA5qGhIC33JLd6k6ZYxz8tpmwNQaBt4+uysxDZEO6EcveSLM41ptsRp5d3BAto706Vm3FWZ4yBpNXPBr6GzgT3fRHK74jwmA2Upmqi\\\\\\\/KsoRVZzb1r1zM78+oyP1pWKjKQjfZpQfJSwMXri6uv6eB3tSkoqAS\\\\\\\/zjeq2Ym8ElFjlkRfVefIuOHWpdKC5EnjQlFSrUzYlvxLW8KmMDNzDGtIRrlYITlB06FiNWPEhOFH9cwqv4qlcp4gWjhj8mTVzaFm2gT2BGXZXOQ2zxartep6pnj6yYoKl3cLtvT9kkjRQJ90WLsw9RN2cZyna\\\\\\\/JOK8+aNBg4jy7iicR6oBHDjNXlNUdO2jCxaca+j60xb3GNIEuxJU9OwyUrpGbnD9KkuLy3P5sZaa0D8Ly\\\\\\\/j1jw8dWb8yMk\\\\\\\/PbzQObDS754VL0MHbUPvQszEBUqaQ0\\\\\\\/nN521a0j0v+E7Grhguqo9lfYxA6nfrfgH8T+j1dcOkKX\\\\\\\/LjmFsZhwqoFOlSGYZq9hy9+5maXnLlqZ49bYXGRZGh3hwgXM7PJap\\\\\\\/0k8veG+lJC3OHm4cDy3nbzI7V6x4YTAST62R\\\\\\\/FfsWViS91iYfGkRn8aQHZU3pdrunbhK9dO1f9Ki+SX2+pvIXhnGjnC8P6OniZxCWzfK8OEPziGwdRCwMDIJicQoTuVIPH3UpznibDC\\\\\\\/E8s0WJ6dH5TTd6Dot5uB0uD6U983uMDnVGrp9v5SGyg52xc9xNP6oGAlacaI0xi0QAYqs58BloNz4kDFgTpGOIXCv0Nfq4OEVye106sFbBo0FsgfVgT4GgmG0pc\\\\\\\/4lZVMBvtQMSJtqJ\\\\\\\/+cPq4ssb5t8AXdzBTChe1IdN3YWjWq38ZHWOYwS2YEikgs5FKziaVZwDUuVdbaGHM6Eby4M7PtRNMHH5lVX\\\\\\\/RddBI4XwYQ\\\\\\\/hX\\\\\\\/IZfTNon9wxKx6bUI2ide7ktrpWonGr\\\\\\\/V9n7uwX3tgF7x3pOYwC3SI\\\\\\\/lZGPYkdzoXJpR6xFPmPlzIcEwMg9NTq6wPHYf2djuE06ZLDHMfyCTJakK8R7tvfFmVM5JRjTt0Cgia4IzrDm3RQh1nAx4DS9EmpQrW61pM8KPxhrUVmqNgl\\\\\\\/7RjSequkpGQrW7Ut5AvpT04c3iOaYO5d1BimlqWTedf1NJD0Go8M9QpQVb0micKAATp5mGzP2VPs6IPxWlsKWNuxc8RjqrQYDTNWq3H6KMpFnlhieErtoij5SvTXbin24Tv\\\\\\\/m5LB5X1AOY+qlYPfXcif1Twoz2gRQeH+9hwx86ooxJciNzyGlV5xkZv1XV7hUIQQcaxVo5W3E1\\\\\\\/+BtcMdb550Qsacle7UMwIcJXtuDrdZPAKFao4kd+ZDNOlrn+fbqbS6OdlxieA2e1aYSGujpZaaYyS9mfrwKl54Szja9625KSPHndRoZAFek9C3UM6WfNJTrN59t2\\\\\\\/VNxxgAjLNy8GeV96XTXKyOu5Z6IA4V8vAGfP08pmBp1gyq1Oecza96p+rQt1mjdZHEl0WuNCzEhyHWLoiRXymIst55KHULivuWXIyYqPn2NYJDKD5ArMGpzYkFZ6w\\\\\\\/oZn+sraw1kbWkZYBqHqfO\\\\\\\/coTCZWhpR+yygYMEH5QU+D35RpBdUi835B+UZdB8GZffhDu73SpSrsHo53OkcE+YNqZm5RjyYuUMOTlWPfFQERjEDHcpTMWOhIyfZZtPSnM\\\\\\\/yO3BDD0xNg4i62boxhApRHOPlPYbi380W5sJdi8WEsDTUWmWbRplxXQRMM5Mi2fUSjVjc5WaMSdPZKQQ6iYPerZP3RzOx6mKzwPiRk9ouLF9DDpdURFDJ+B0pQu8fI2KRutEQpb256fysTx\\\\\\\/EZTA9ZYR+elUf3fKpcMyDsqo7Sp3pBdojFyQ3gXos\\\\\\\/SUa6Nnih+t34lzmP+wRdTZ\\\\\\\/\\\\\\\/\\\\\\\/WDdqCUH9kMxBo6MEXXwcCSQblWq7Z1KHbLIxIj1D1FtCLuGX9sWRIoXODFbG0G2HTBv0IT8hFcm4pC5LCxVIKU2S8GIWV6ub5mBVYmyHGPhp7zHxFYjQZRDO4CLeyGZP1LRoHwMBg2SeK3TiD\\\\\\\/MKFvgVUKR54pJQyVekLIirea9phmRdot9FqSPPtPiaAOc6HdxWmKFRdT8fwCYgf2Ww0Bn\\\\\\\/oTZg5sq0AU021BMaQtqYJc2DDz4i1Yy3RyBvNrVP+ElKYoOnwBJa4qZlwP07iNLcsYcuRHCxKH6DnsWU4Aet\\\\\\\/T9zOav0BblMgSEN19Sgpp9SpbH+9dzkBRojw+XwoUXk76WHXPbI+O1ripilu5W5Yov665gdg\\\\\\\/TZpTrpZ0OeSI6xzVz5YGmLFTfDwhF5ejhSeO7\\\\\\\/u9XrKBmDVgPhAPUCMWTw7j0wKVa988PSjKbtZlACWPactj8amcsv0k8zPLwsNOiSgkK5qDsuSQoKAoE+UIBYcy1jxVpvOhCnrpcZ6Ftb4GBBqOzWsrCp2iKAtIzx41mAJlLU8Wv6BIsIubHjS5keCdkXfrlh6tvCoK01FGUE4orJgYdb+Ult7EqS56+S43lHCiiEkr1qVWcEzjWSMLBuRZEhJxWfGJMX2eUICHb8vO\\\\\\\/FSVC3q0lvULwH90TVzbkhkLwG71o40cV0xPjhq+DE25onZJgXw3HPM0qkwiMM0uI426KNvOUE6bYWJX\\\\\\\/2t3v7SsP5Gu6YPRbkENFW4b90IifhCozdKENswFGZxPBRHeRgPvMYN7wZbqATlvSLVK8GvVD+bpJ\\\\\\\/tDQKXM\\\\\\\/RWQ7waEBtfklexireBOSL19\\\\\\\/RAruOz3orwqyCAp5E7wVlUFvNZlp8oPV\\\\\\\/6ahjqWq3Y029MULHEznn2jHU0QFnRZ9rZkHCYGnl69WwG0b0yBjUG7fW0U8W\\\\\\\/MsQnRYjZmE6LB\\\\\\\/Qm0UToGt4uabDu74\\\\\\\/lI1lKpe3ioCVVNy0L++c2kp9w+YETh9V3AT2X57hHgpsQXyJ46PeHwW\\\\\\\/2qGtTbnrA8R6cPnjFMoaQrca7FeP68zagzYUaZmoktInJ4jEx6jStQ4DMibKZCXpQYmGLBKqXko\\\\\\\/VyQpvo1TJO7ZFcATe0ulNethxhJ6iG5jFPljgPzEMMZngsYYXJwFUqj4QdUbscGEnQrSUa02dOWdI0wn7k\\\\\\\/TE\\\\\\\/BSFvy5d86DtXzlBC724JTz+BIfI7dxAQKIovxfupdXBhQMjGZgCpTSCt9olY2CNqEYIWXPM0Zt\\\\\\\/5rasYHjAbW8mNV40bs6tbZPrnWF90rTab8aFsGbiBvpskQ8yS0yXzy1RGBXl8bW+niy9sY+H3OcEtZwRA5QHyFzKelNZQp0Ibx5oGqY86T\\\\\\\/Y9TAmfHdByhHje\\\\\\\/3scXx4+l\\\\\\\/OuDFsWv9k8yOY9AGdFvr+hxjRmpBl5gCNgIzP0r2XPJ9k+yFFU79\\\\\\\/3Q2zTrWlWyZGmwg3vJ5C\\\\\\\/9URB89waokRmeNbbV35QxV1LNhnvZrwzSSHgt5fZVx0+ns0fhwyl2JUyr7kkIo\\\\\\\/1rSralC7EFK6UKTb\\\\\\\/uIXzLAf5+Vf89VCW+W0EaA9hvw3oXbrA3cbAvW\\\\\\\/zSL1bD5ryFGNTbV8APa6ndVtRfC+8wE9cyg1sAYeJpFoSPLBdjh16bV+8\\\\\\\/U5P1RjyAZ45WeE187B75U0rNLV3+WB0iVw788kFejuVwzlPVeZy\\\\\\\/krRBxZMBvtws32IjMu3FhuQzqrRyY5LLv\\\\\\\/mxPnf1HtQHjrQLyJgSvtwbiPElxHx0litOiJ9Vs0geD8Qp\\\\\\\/aUt17mEY+Qx6gab3qD89FCy0HrMvEqJ4XYwM4S7vT3YdzhZ2CM5M98f6GUuRZcmwhTyjYrISl43fN2sbtBnWV4kesFZhs5jeBMCrxoaFpfQ5YNGY+4BQaNAYKHTJL36VVbDcjEq29t1VvQJMS7uUqo3FQMv55P8lcLZIYSCkCtH+VyLYrechuAm3BJvMEyxKTkp5ZeWU0uE2gSe1eb8BXZj4zyJHAVh5PFwKqh3TUM88gf05RAFzIHnx\\\\\\\/I8OyFfhJX70mIiajOHWkycXFpzLWE84lB7YgUfvW\\\\\\\/G0e4J4VWWlOin7clD+DkCRqiK+U3X18ew9IXO9fPwSYwpJlxzJIHV6C+vd8BXdq4YfutfAg7x0uycoPH7gZx1zZz29ITpxCACRKovVgw9I5VNYExI4pFyRoJj9QiQ2g4anjJiS+jMjGPYcdRk6sks8Vp3IyMmos7xRUJceay3w4cwCidvBhBjxxLsOMXX78du+0HXLJwE3BbdxSQ8xJCkyWq6Pf0Hd66Ctz3cPxunVXAx0kV1wrMZAENv9uPrWtDZJNGWpWlSNZLPmY62JqN\\\\\\\/eJJJ7uiO1KVXeSc6DSQ3F2IH2wwzq4NmPdjimiQhCXrUSl2ZdzVzHi4uoYQwaLyTDJZlQFQHv1m5mRDcdYWyUqi9+MaqF4J5RDaHTL5krePyTsE3g5xW\\\\\\\/dU2lu2hWBMoHP\\\\\\\/a05B2aF4Vt76zzPIPCbnDkwqh+UgXqChbsqx55KhnFsCA8c+eOS5NbVK6yilaAbfI9WhmeEPjOR5DRGn07m+8U8MfqAlEUj0Bhy+gICx8MiYHojAIBKw4AfIxpyedj5K0vVqf14uDHocDr29K\\\\\\\/u3iUUzZRFcYmUdqbhTpPEQKqrURlfgs+VY9tosi\\\\\\\/E5pmKfCj7O1gFdWn8U2sSV+jbG\\\\\\\/IgPXNIoqSHHpg305DPTxopFA3qmbudDDiNd3d4NK8Kaplx2Z\\\\\\\/trek7s2Eglj7xJBnGTI4DyFDm+VXuo8Pkgy\\\\\\\/JJ6yIkDmpSr9t46\\\\\\\/mmIrDKQmXqjcEP\\\\\\\/T7OQqMwmoG7Ft+MFLpMrm1Uj5T8d0N7aMUUmdySKuLd2sDeZNjcOr8\\\\\\\/\\\\\\\/WaxxCwCC1pkfOJDBQ2C39aKHhZ+Z43gDcVFKQbrwZf372gtoCCPgnOUBignk0OkqHHQm1mTF+JKPBd2rmtPyqM1thKt0pduHy1VdOAmQd1ALr7KmVSCnQgyWjPkmt5c1GeAwy\\\\\\\/mSYuCxyrD8fTLtWSidocTvOLD421BkwEMHy8X3MyUJGdVXMks6x0rkQT2CE7h4OR5CtBy6Pg\\\\\\\/zCzg1pbXqcTpvq4wSknGNb3mJed7EE3oCIB1ghJcSK3lvf4SKNqwaMLdHB4WfYntOtaspS+6bzQgeW6wErqFebDAY0uITTTtlNiY19yzWak9IUktE3LKl4tgTXookdfKbfTMx\\\\\\\/IEL5PXmOvd38pwtm5aTuQt3TxWzE9xf+cfOJeK3SfVP3DchCue6twJ4XWa1BLHhDq3K0RDRTKjrJpAsDyfzhs6Pjeb4QFiUSnUmsZp1LXV6b7j+8D+TEk4rGubhkPJkN\\\\\\\/bMcrzfoF09hkpFuJPF5U+I4O+uVjv\\\\\\\/hOMS8QNKVXFwj+bOrtYFPKmzAH\\\\\\\/EUvp3iOKjb8esCIRo6+xjACvkKe9PMTF6Ho1iMvGIbfQxNBmCNHZ\\\\\\\/X2CN8ea6YTW3VqEy1cYmwXMF0pd2yo2Iz4kPEpwU6TV71CNRjh1+Xxl66exhC0ybwSQG\\\\\\\/2vKslYFIuy5eK4qDmzWWd3ZM+\\\\\\\/GN9AD813O+3tJf5w5bI4btaf6c\\\\\\\/gsQGI\\\\\\\/C5JFG+1xFIXYQ+miW9RFN06rJPKtMbk8v3kLUQQYsN\\\\\\\/x2RzFJIgNXM\\\\\\\/jQZp8sxDNUPdPfQI+kvT0tQhByvHPy51xqUn6kayB5t6dWj2HI\\\\\\\/TZiSZapWgNEjwDdK6oceUWPSHKdPMorCaLV7RBKRVHhUv8dxCJTVNeNTKhU7ZsPbQIgBgNZOGKFtNyD+\\\\\\\/SqKfdzVLH9XZ8AsQIiWypXsUA2eyb+8oFFE5VMxycdZDTMHeb08gWXMTERw3FCHU4TLy5OsN\\\\\\\/9fJWkYMzSCbLwsdm\\\\\\\/erkM49YyR2StALqY\\\\\\\/oyA85yToej5Jm2dJpF4QhIGghF3FX5tk0UcLqdwvhGKoB9ep3ERBQOzIPM3czczmJ+CngRtJUglVVW7NlP0AsSPQOFnwTh2m2ZOJBibYa03gvyKIFicF+BSULH0hbi\\\\\\\/TwuD+nPr3m0YuYUYN\\\\\\\/ph4gdKImBeprAYeM8hXDBizM\\\\\\\/\\\\\\\/xbMlHFBoEMljqUb+eq4Dvl81KRqANbEu22QemdjGIE3pwQmgMzXaIgNukKZF1QqRz4nX4x1KmKokeodygHUMTfJQrxnHyudwTkuciIUOkMljFygF0jdJxJW\\\\\\\/AQSAHPd6e0rVEDb8Pqk6l65hHaWSl7ovQIggedNZ3UFnNQaX72wDMg2UjpBQeiU4dw4U5IslHxyCGPxcwIp2egdUuFyYKG7Xdr8pkO\\\\\\\/UfNwAofqbNqL2YQlRh7VUmkP1MdAVlFuFLrH4eTaj+ONLT5pqP\\\\\\\/qpn1LM8NhnNLfBnh1KiCJCYE9wwDlMWQPCJMkur8PIhtSZzrHgPfWphU1LV0wnExZKumseYDZVr6j06I2Y3ZkHK7ZQmJnzHWmEs0ry\\\\\\\/h2VFJkoSSEyUxVOrv\\\\\\\/Q6ySsSUFtz82+X2zyeXkVbporEXPnNFioCwqGFVezVtJp\\\\\\\/qf46eo1uo4YbZWfR28uurETwmT6DfaVDx4K4p9BYlI8PsCVzguG\\\\\\\/A2nmUGn46a7nS\\\\\\\/Z8JPd81VoBOEet8YEUDhGcIt8U8GjFeQmG4lozarMmr4Mpam6tpx5BWTiOGeUpD3kfDHr3Ry8fcRoY2nyZfEQYSosYu6ELkOmxd7jZXzGF9TiIkNNxgCkIqg27Z8Ps6toWlrrwXE6eNMJrPVYTuv4iQjLODgYh+XBk8wNU16OR7W0Bu0oB6AwuXbeOP+qfIVX3BqIrL9TX7mBrEFd5dapeDcbZq\\\\\\\/KtUHFffjfriZX3hYOv0ObCrx4q06GoPxPtqVs4sRqJ3sZKeqFh76cvgxrKDoa7pZU\\\\\\\/qQf4yQQFdVpdSeTAHpoCS4yrFGuAk1kqeDUW+vhChdEMcpX0Nkd9AZ1a1FvzQeLF+py96ceXxpVF4\\\\\\\/BQFwBDw6EkAtSQOxjQ8Ja7M1175YWE7jA6MDVJAaI5i+8hKLKXz+hFtIEU2AX\\\\\\\/a9U\\\\\\\/eqJIzosSeyer47+Jb8v+JjN+QX4BBPmwoxyrcRjeWq0jZh7hx4RhbCvDSay3YUmhRLXQ5A7eibDLbmw\\\\\\\/9YHarIcB5GUnVceBJ\\\\\\\/z4AfmtiluwXieFEZW0552MZJFhKDgI1XNJ9vBveNddn\\\\\\\/VNGRiB7yeCrwLNATr\\\\\\\/6vPZNOT64JS5shEIoWqROe2i+JYSaELRbUGWsW8Kovogk0PXmvmGdkE3HOV49Jx\\\\\\\/o7sLh\\\\\\\/PvziRribhEhQzTUXJaiQCAtCrtc5AHfB\\\\\\\/ksoFIVEX0j7Pm+RQXfCiOzZI7mVxL+0cPtPa9NLqeGxVkVf3MiFG6pFyZC8250hP+3lm0NDGSkDTQ7txJ9Dji47QDtRsJYabkp2R5hBOhF9UlKizD6GbQVp61ZqZBqzQUzYVeAwwKr1nf6IJCkCP4rkvBLuXqitPjo7RaP3jYxXiEFe7W4DTTzGjtLC7xMhK357+CLrOA4liu45+\\\\\\\/6rvLPUpZJmITLCfjnAUUIC\\\\\\\/A+JGt2YZ7A5Se+BpbofVATaws1QQc9l5r0VCFAGgzJDcVTxMzHEo68iWwxvvj8ObqWOgCtSZ2kOOlkeD6XNw\\\\\\\/ZkxuDGW7CMOAL66x3WR5BleTMrICPmJvOSbpSpsEff3kN2dS7xhVetDRBcObskw3A\\\\\\\/Wo77f\\\\\\\/AQwGe7bgbinjnfpaKQlV1+EpsjJ4ReCaOcCPKPv2Nvy6mTrjNFkUlbRjecIlHBIwbzOQ8x3TVJdOCUu6neblUfMPh8fLJAlNu5AfXp2uhDuOqfpNsi2fglWZlFfSLQ4oP8PggAvH33kc0BOVk1BifzmB\\\\\\\/UgmPR5f73BP+H6Ij5pVCUlx9we\\\\\\\/VO42vzupg0uUPBMMxETWBmy4BwlrnizJalLAOsbpaZcvDekxjjS8YUdsXzrIlETIWqwcrOpjJzM85SEJKW3CWn0hLBcCr2AQu7ebWaiPe7Gw5eWuzHQakObFbp9v7cs3PI8GCtvsm0DGrCVovBnoGzADOsrn9cXmCzL3MG9QpGtrnihIF\\\\\\\/wFEk28UkeyekdY3E502aUP9bNrz\\\\\\\/EZEVrdfZHKNPNdg8Vk2iLysrAgzaIK3m5momkV5hzyW6AP6+sx9fqLKVmZCVoXMyl5rd2scsA75qY9BkM+TISkN1kTnx559EMgsMi6Eqyewe5mwmdhoybjI6xqecZcz28DLK\\\\\\\/rm86LyytOUK6v1X52HYrr8NqJjeQ1iNN4aWq96YIuMf+yi\\\\\\\/L3dV5tebsHjAeLns6M1ay\\\\\\\/nkBzc2rppP1ypLxd3bRFeieAGDWAwaxlPU2jnYEylWQfhRtRwSUFgh+EZbRCLKRPeAa1vf4Ef0nbSrY2tv6f+MuykJx8ICCug19SXR+6lz+EXcxzkIdSkxnelsJkcQA2oLE+1y5wrPbk20prDfufVfF1uRsv58Yeok9ddNUXbwBstRWTbFcvo4D15avuWaL\\\\\\\/2QlvH7DnMuYL\\\\\\\/tR+GLRzbBwZw3otagHn\\\\\\\/Jp992rlUMLbcXYYT6bZohxkzdqXrstKSmxoprUwUliJis0H1WfM0sNdc6ij+J9NrxDgFobbHTB3OhXqePkDlhaUCnnFqXRRIhVS4G20P7SVknqOG6fZe0TCketb0DVkAfBycb03CooBpeGDhg86vIdMNriStDYOPgalvj+\\\\\\\/zHZDi\\\\\\\/NSI4OQdhaijhW4vRCEj\\\\\\\/XyL9Ztmmb3ElltvnmNjBUio\\\\\\\/xWmxUysuFw0pki893d8Hq7LIaRYjpRdjZyK4bfY3ULqpSV78Ce6FafYkPuq8UgZJ0n5KV7wGwdZRLphau8Jc4cggxzIlv5yK8PUgPK4w68uM7kcGakRfwMf9m8S4OggbL9\\\\\\\/kEbMTloQXF5qOJ1KNKr+yH2xiDn+XqHPvFxiuPp4oBuvYmZAE35esz3s9Hq9vc5bqGxvmMUxrAzqsFM3DaV4TNyPhO9D9W+mSdkEdyovTaKnBbwvp208Fu5nlFhZCeGyXIM1azs5bCNOL3sKk50\\\\\\\/++6XNh5r2Xp9fNoSdXQedRY1YW5d45ysU6+2PK+NAGgmplgNjOc9aF6n4je8Ork4E78oat2bIBFT0atwDLGN0DwlvDFsMz1\\\\\\\/LLb2bfWHyB41pn\\\\\\\/ILOLoqaIy9fNmA7CYLPlMx26lSilv5BFmuwJEOKECl7JRkbp3xbN37b6LoF5zQQDmQ0K4QlHuaox+rbSgB8XPCbhS5ZBFkhbGXKzwHdNMSQxPT5VcJDoxQm2OcpbWTuxXyBa61VVd4zGhAga3wXmsy\\\\\\\/iDgcAGwC48ZA\\\\\\\/ZS6Yct\\\\\\\/KkpMGStM6jr4UH4Pv\\\\\\\/zKs2PKneNPNdJx\\\\\\\/HTbiO7P8RQmLcXmXO9MfncaCGxhcq22clUettXEvyR9lB6ThWk02fRvzTHsbx1W\\\\\\\/zPXQgZnwe4GFbn7Uz9I1wzRT5q15\\\\\\\/lUZ3If2Bqp+udRv0\\\\\\\/vx0GTwr9iBqgGzn8dlGMDwg2ctEdeGqGX0iv1cASNdMvNdwkN7JMXCpHKq8imtYEr81qD52\\\\\\\/ibAzB6B4xoY7X+sUX3Hm1vcSJ9r+OP+GJ6afbg1\\\\\\\/3WwBVkFk+F\\\\\\\/hWW\\\\\\\/\\\\\\\/RXtd1eZm+XPwEG90NH97d65tg\\\\\\\/VjsaPpgTFCoI+L7icPFqSizMb3lFni4D4z\\\\\\\/GC47fnKJGGKavyOOeN53hbGmQ+nwwUREbUlVGoCo7T4aXAEpjJOS\\\\\\\/pUbAb\\\\\\\/oiLgIzmb38AHemXqYRmLEDsjtrkKBJ7reVKoa7VD6Pno1i+GH++Gqf13EwXeCqz9Qg9Egl6NJxtIqhIE9YnlTnMhVXYpXx4DDdNyk8K4NkiXfs8jtTb31pW\\\\\\\/55jAeA3KnvSFsqJJW0MFiBZZJRhN8pF6o8xuxidUdRdKbG80l+gfn9H0AZxZh+bQTMm3xb0bMDnnFlFjj1M3DUngadBJbspgKbNwaaNGL+TC8eYIznHhrreWCbsrY\\\\\\\/iYh1NjLcQT4O03m6GKDYj6HRZRF0NrcmjxP9o+T5vNCfbcMACVafvGLnSJyVclfROr7Ftq+DY87rwlfLEiCRGBYlPwvQPsDWRsqQirCZQTKYYUMFZAQPKK+Oc+uJjjpL5msjAA7iwxFeAxof5A+WZdnnZZEkjF4o\\\\\\\/dIf4gsKnXRggh\\\\\\\/D81G1JReDbCmGsYC9VxqT1h09w9sDXbfq9Cg5VxoBTC4n6vsTKusA9iFnahu5YKeuAJ3jRnVrlArfnDZb3XG2Q6i4tDy8XVBanYMYHB8VHyovVgvq5E1hoII0vYhOGuSDyJzKQXgTrgz1gMn+V19963gTFw3zLdni4f0nGFPiQH7Bm2vR9EBANIWp0yn9n3Sna9\\\\\\\/aWixF1XZzVso8Iwl7jjzYMvt\\\\\\\/iOqzH0rqDUMQRzfsLqm590FIm9oQI+t7hBJ6qSPJBQWgIWEgA2OUOX6MDS4zQvO6XNuiVvPbrcLs+mUOm1RpIvIuFeKitfNSbSdLQGEWY\\\\\\\/OlR19ReTYY295JPa20nFB4+F86am2\\\\\\\/FjNyl5UJJqL4ZaNyf7KSUx447ZpH7SZS69YzUBER+ZpawFwXA6+NlzXUNrBzKS0bGsI0hCrUWIix35LTBedTjKD9chy\\\\\\\/GUWpNvhSV12QXtbQO4w0lWKbROyMgdAD1a9lxMrGLYpA+Uhf2r1WpqgAJSSOEVYfnNpXeLx2EnrdvmpseuT+1+aj7o5y5iisXQ2CyV4YExcxcEhOE8MVrRVtz7tW16LqMK2Vd+aO0uvTzHzzvDDAR+opetyw8FBVTdaDSx2O55uAuKenhVkB64sW6tV5GqTzfGhutCdt\\\\\\\/wYvH5XD2P9O2rG74bMGvJCzAqWl6FYo+UvFQGjnaj\\\\\\\/5UsTPdsiic1XfufXWNANNi06xVd7C8XSRXPnwYi1\\\\\\\/qUU6EpUMdV+TENf6DzubjlwQo44AZpvrA7nsnfar0mLTbk8m9cDTi8bYu1ZZNmqgB6qrhq+Bi0T4oakw5KckoAdx7E3pOqGhXFFtTVV2Sml4YStt71PmDgcWvxQYBe9dlQ8Guxorgol7wAvsfxwY+XwLw7DQSXScDO5tpk7DT8eBNOXXzIssOUJHzOE35N566KvHVwZQA8FxxZiIvHCYuHxnyxF9\\\\\\\/tJVgNjTdZgD99E8XJOVVm8kNURpFHHJKTctQXhsMKdGLaGlYYBkO\\\\\\\/Eijn8b4TXVMqqQzAYDV1isHQ4DV+NNj2oh\\\\\\\/dgKSKNbQtIoIFQFtJcJgAMMaAVAGP\\\\\\\/XRn099+m0J86+rF3r1DR0HB5AaDeftk3nSkR+LxRp8DI8w\\\\\\\/SJIOR9aVN9OUYqODGVPvALClCA3DXPz9sM3zcbiTdxeTVkgPQac6zHCmHRxSxT\\\\\\\/BZBHKMS+ihWRRM2EEukKsoZk0gVgqE4Lfh1d9+xxt+mZXxrXCegwWXkukBVXcqDDFym9twPbyXT6DsoQQ4F3DtSsMVFkGSTsvm1fuJM4UnTBbu0PQyNw2UuhnQa6bQ1cO6nHlLD9ez+yxKre2AkC6enZImiMSffztu86a\\\\\\\/ZDoPyjU+eZUxVyij+5nqDX9WhhYDpDwOx\\\\\\\/3QNzta4maj6zZBkmY06RuZBNlIFXzLh2n1GQp3l8WidVjiaYFe\\\\\\\/MN\\\\\\\/qGomaUCVxflNOyUyIphw2OJjTJ+mSQrjPjBe7Sn5jTYYfo3kXHEkbDtUxcQ+6+1nFvp6oZymOrmOwiFYhytWnMbFOlAKBeZyds\\\\\\\/IKpgK7pNMFilwQ\\\\\\\/EOUeRBO2e4eLe4YYFI69biL6tZYmVQ4XLpFVft3dgwsr37BFmSsWU8Fs\\\\\\\/zCfKwHlLDORefESbUjPY51gXzZVWcNX11hA\\\\\\\/z0v5Em1FWgKaZJdFobsV+vqztRHeYFc5Cyn0GT77CwilNPzYZQrIX+jcFbvZCldjMHxMgybfmOOz6JT5NULkzOwREMb6kGNqKfeEx0MsOWUTGWXnlciIak0m5fXKyt+xSXJSahzRRN8teUUT72cHLLgFZyffoxFvmO0yLVMsXJsYExKWcxThqJmZV9mst02nb783xgZo944u+wiNU819\\\\\\\/uBGWViKKhnl7e9+JWZQ3m1jbGx3I7BndSrPxeaBeO9klhUMEU6dsu73xmEs60x8YtXLAZXmnXHx0X8yaZ8D\\\\\\\/kih8\\\\\\\/vBBikxxePQ88bMgF6D4W8iiioGsKlJJlTNt51ffOVdhKiEfbZx\\\\\\\/kThjUah3YYFIzU7rt7Sl\\\\\\\/rbvEmcv5rC\\\\\\\/Xc5GOR6yOy4TxSQ\\\\\\\/nurF2SjtYdxm5iQsH0kvu9Rwjf9lKf0h5io3KaFNS+o3xrzDM6b6SSdklJw\\\\\\\/\\\\\\\/K650X70rErbDRvhxQV8MvoNngE1ECqb8a\\\\\\\/lpuuZvyjEsgxfnCN427jmLaabECrjIBkGJkk5xxIJ7947iT20+rig4wgmVK2DlgCVK3tjOZyunDe5w+UInhn6X1hTWgKK9k30OUeO5DAg6jqr5Jq7gZxQgZOSJ89YU7+p1p27t8N0ec1LxSxO5uB5z6IBxS59XhIpy5bWIk6g2JmHhvfgyC06Ndd9MgKy8x9laEHUQ1GbwmABOQdf5tayAMX8gWAJeSoC19Mwn9zAIEsl2exqR\\\\\\\/AX77U+EaQHojpSeo+dGfN1M+ksu\\\\\\\/kxuQU7RvXU5piZ5XqSA122Eu0opm29lJMxJei36MKBf3HJIKo2jOQlsngyDrE1OAEDpiOFACDHhh+u6HDs5s0AO6bczAms6aY4rBShefyVB4+5PjIBpl\\\\\\\/Hge\\\\\\\/W9BEVlYIoGLTS8EHYTEu6+W\\\\\\\/dlLSj1o5f8EzI\\\\\\\/JMwdjOoPc1y760kSGvNE0to+Ldglx5qnWC3wx30tXd7ZfyUd6dFIPjLVAqdhGC5uWeL+fQ3K5c6Mi1jmNtpsXD3vQnKGJ4ZRw91N3ttOnhfAFhu1DLcUMDFiFLbZ9aPP\\\\\\\/S3OPX1wHEnX8vaz5pxFUdmNxlUEdcL8HGcytMdyr9TsY6OXBpdTm9hsZRo+Z\\\\\\\/Fr03IVyBhioEE2b4IUTq4a\\\\\\\/I25I3\\\\\\\/VR5V81buCWuscOfD9Wm28gAaZqDNwYLXuBiN4zCao9r+Lq61aRmodq09zOfa3X+iqOBjCcjGOTGF24HVOcFJr7uciYaJgoAtb4Fu29b1Uj7nFpXuuoAG\\\\\\\/95PDltaH7neJeUinPhVsnX9ZMkkSenvbwmE6hbxIuySmOH82Md8dyqf9bcRWIMRQg2r6c\\\\\\\/S5Vs2v+RSebh3RjUp7IAfq8GS+6i\\\\\\\/z89Ke8ZgXPt6qNs529HRQg5pLZstMXRYSqHQbwnAqNxr6wyZNPBjPJszO26g3mPlD7\\\\\\\/XPozkgQZ3kaHf6RlNEdZFGZ7eEL5MXTIIoHs9CNv7fdspbesXNgFPRPmHDsIYJz36xoonc4FI5aO9rBuZq\\\\\\\/gAlWTSwifk9cDNAhysHAxMECRIjbe1ex+l8cWx0ebwZFLqzL5lW+AR\\\\\\\/fnTBgoqwllfFgyVxanb2o8V0hdN2uH9ncgTMmvq16N34nI+NfEqB1lB7oJCi0Co03CNYdUtJfU+tjleKsiRPCVHhUWCgo\\\\\\\/bsyaRqZCMOanwVMpOhTOBWese3tSqfyclDGa28DHh1fqUSgypSmFm94wBe0VrPcbs6EUwsy\\\\\\\/0vE6g6DivEeO44bYsY7XUzMLffsTeQDTAc0FGoWHvr6RM\\\\\\\/Q6L6A58kX3Ags0pnd24dPSkqbh8NBLkCqk+Dm1K2fO96dwhG0RjaP2EXCaI3tbwnjStw3C5fRwTWAa+mSGMwjld0ayDdSy09bKlFCBoq9\\\\\\\/VQzIvCjCJ080W6j5wNwxyo1l6kFK+Bg50LSrE4eaMs1mskPcpyA3KxGYKsQ1Uf5KXhIKanzF10RYJZEKgmlNMWVz8DM2RBeZw+MMPrK64P\\\\\\\/L35W+OEVcyawDE1aN1dJJwMmjY3n6lbbsUTo\\\\\\\/RAXhyJtkeEPNrNZ\\\\\\\/wkEhd2DeCSV\\\\\\\/1DvK+6kP8muKiSy17BTWVc1rnFfPDZa0R+TdXhNsmM18y\\\\\\\/N4kaWdxH5dekRQutWXcitfsvMux\\\\\\\/2uPoCQvLfj4d1SwZC9DIXiFZVymPOPNrv2PCCh\\\\\\\/gg5aqSerLM3Jnp8+5GdgmTS6Q3MNOWr06xxLL\\\\\\\/d4HZvWHkAxHr5ZV4kVCjXOZqZw35+\\\\\\\/WajrbTxLbG8HX\\\\\\\/q6SyaHqgQcrU0S3X13s359h12dM5XSeu85eCayy2Gie+Sv\\\\\\\/XMg\\\\\\\/+c1ZU2LWzvAb8x0rvNwShysrQexUt4XlY74pnyWcuflE135xaHL3NYWflth9GipDbKMMknakJpdLULYRhnS93Wuy35w5tNoHryH0PzcRsQ4eBD5PPceJiZy8z0H\\\\\\\/rBAX6NH87+Bb8huPTimAHNOeUe7i1tRsdwElLDSYrZWFdWfDZGtjNWTLQT9IYQXMfxJLs3VZexJGofJSET0fLc\\\\\\\/tpdsdQT9cN9uV4LLOuxg4SiZk0GWWRjBp2qkLOKfHh+Vwk1kiGN3kLrGcpbopNT6FxXjKppEJrJ0bpSqECPHQ5CYvxHFDDxjVqFWVINl2zue6RuU0f0LBU3NlzAi3tax7lbxbWstPxcHu0l57mjt4iPmf1h7+vYOSCcse65XdEpDx+1qJL5jXKPoJLHvRjlA2AaIUaTrndDHubD0s1AqArN139O00LtLRnhNYBouQ\\\\\\\/f4gBd1ny35eqzymbk0\\\\\\\/\\\\\\\/mqESFfhcNGHwP3TgGOq3xwun86LaxgOgyhni\\\\\\\/\\\\\\\/rdEIZZm3B9MPJtM4UhMW0jl9wYjb0cXfYoxHWYtkVgSnzrL3O+igBS5JMFjmt93xec5fYUaDEOzmK36CIQakXirkyuPS9yPX18uWB88ZUQQJ9zNwJcWwxWLxnxVMQgCMJBQmY9YIaFVDxbP74P3pgfD9oWv7sBHvIFcO4a0NGXrZJWe4jmiC81mCPCA\\\\\\\/zWzi5A0dj5KYNE\\\\\\\/3qQX7JZAIClMReE3UbX2HWs6J8VqPC+IKTULyMcIVy7E393ngTpRkpqZPoCvDtId\\\\\\\/9Q7CQCKf3dEgiNJhKMiwlFuUnbubJAJkjzBL7JvGQ8eHkDUXqlCWQHAuj6OYj0dTmn+EWr1Nia2V9ukJcdYL8tnASyQKk9cdUjxqWxhN6ycvXEbYFBM1jQ1GH+olQj87JAuc6XkPpU+m92jpBzAL9QgBFeklBbCR4OS5VlOERxIO5cJ9XoQD4FzzUkKupubgXR5BuCL50W58s8jFukt10yhLFwu56PjQHLEeudhQYah4cDcOoleiT1hyQuvlpRC9totDtjjwTd7APTLGHhScNi3iz+h25ijcTyjZOT3eBKV3buKUBZKQ7vIfIyRTlAK5+eNUMUubnVex\\\\\\\/H7CDFCIHWnMhhHRmCVD2Q582dzZVwTQ2rlDDsYxYh0ZVKGEwWXa5ULPvX+orcQhxwIOYbh4DJfjlHK1rMVqTA\\\\\\\/do23M0Sqxk\\\\\\\/UeNbXlok3k3e6Bp6lDivcjiQvrwseTxFDEcoCQ7BADZ3ly+5\\\\\\\/JxFrSFGwypeKGN9drOk9Y\\\\\\\/DbOiRVh\\\\\\\/BOMDxZgN5wIqgMkmDwoERwDQIXUfV27mR0p6jIzRnzTMtyAR7dMlVH5nbpgHLeUwFLyP6Afqf8s+VrpfDVpby4OXZE67tJuiRa\\\\\\\/a1niArxPkkOx1LOg7xLCES363ygs0FGmlqcNKOXwn4WnaxInl6gl7M8Zp9xJjWpImpczUTt8bgpm+az7CYaBGPYbySNYPNBsOICyZMuF+9XStZ83LHfS1PL9sCAn1hFFPv9AaYgr4JRFnj6RGk9yHyiBLQeyTrsneq4DSJSr6WMOaJOGxnrd0da+HuMgOzNH1nUQbWdJTcz6Dpn8vL3c8SojMGNwYWnBBOAEuKra2qke8X6ZVDZ4RBoNHjDa5xUHCmcwV7pgeoY8cbz5FMMipAB1QGNvxPRHn3GpeVM\\\\\\\/X1SHgdJOyenvoRnwnAm46ZWd51yRgtq0r6w1LfahCyks8PDGuRGxdvMo86O\\\\\\\/93iE+lPGyC77rAD5MYPtefvnGXzF61F\\\\\\\/IsDBIg5m2Esak8ZQaUW0nx4+Zbr1B2IrEioB6ugh32xSEE4Yq6q\\\\\\\/KjoDfa+AhvWlQVNy2IwRdI7nYFgYjXfyjBcDvbmWwel25Ly\\\\\\\/cBjtRgbdcdktsHf6V50IT7LrbpsWZgI7O\\\\\\\/fTSyo74jdvkzgOuP2AG0X5D\\\\\\\/6B6t0OwqtVNuEji\\\\\\\/YX0gfjso4jq1ixpC+AN2olR\\\\\\\/fZmcVgNv7soB3G7bbRSvVe\\\\\\\/yADtwP9byqZlNBhJWBynfEIKTVuR9p93ugC55Wig3APjAgkSs8BWnt95tlgQuIxG3C5y7f0yemoo5LF+P07LEksLQl20zDWSn+dfl4Plim9dHBp0hxX47byYVydhx68PT0HQ1XwwvCowiddm5R4y28pitUEicARQGkfMtuk6X\\\\\\\/3Femo5L9USehLLIp6NyTUTEDzo+E18kBruw9g6J536eS7rxJxuGLm6gBpNlU0E2+sr017kv+l9Pph7Mk9tZ9MjWKtCQeeCVf39gsyviRVRYdMZzKVbKT5AlXK8xVxvNE3EwM02Y6Lqm68TbGLphXq8gyOUo\\\\\\\/D00VvqS6cZXanKWYyw6VVnJOavVtXXmz3FLSkfKplF1HMx\\\\\\\/R+AO+8HYSrfnnon0fsRQagJN3uociFk7AZir0CFO8LJLuuX+ZDmO5u9Zo9wJIyR3JB9gmCvETHTBaKc6fxf0XKAI+lONgj5gVhy67LsNc5obyutumEDc\\\\\\\/gvGkxLvtlWd0PH3t64aC2X6TRP58LlLdgBRMFQFvB3ePr5hGiL2ZSa8EOPt0\\\\\\\/Essi\\\\\\\/vf7H7BJebcfhYfJnDXg\\\\\\\/drI0RBl2ADFm+9cTKPsmDnY\\\\\\\/p+lyV1vSblp8g6YZHm0yZocpGrR9Y+h9UeNq\\\\\\\/AWEJGsyRv7d4Az8FvvZaWV34osYXvUP7amJ53a7IwBoKI8HT7uoeHxkieFNI\\\\\\\/EQWbk4ZQFtOL2sx2Y03mIsEuxEMdYz075KJHaAlHyXNdjK67GTqlN1uLYajgLaJGrHfAlMDSuVd0t\\\\\\\/QO0o0trxzKkp5ArjTOxmvphUPNbFY77pagUhuN2tatwc+MGqHtdvAaYbO5wv+Zh3+dDWgEzGSHCe5QfauzXKWZHG70WPvnDdsTShkXHnyK0vHi5nN517SNF40M0jlSfdRSk4VvNVcQk4cs5d7o7qN+mfjvQyp8d7vlk8j+MBCHbhPfcUBYjNcoGAhTTjoI4T4QFwzU6BWk1HRR6Xdx+th39VjHi645LyAEQ7I1mld7KF+K\\\\\\\/NELmRhtyKcEcMbXdxV0hu7nApIAqbI2uUqHfQS0BMZYhYLe0QFy16utqqcTZZpfs8q1WmU6jL\\\\\\\/1BOVUo1WJ5iNn\\\\\\\/6CbIhdyf6jXtoTmNQcr74naa0mjpalkiB+9Fy\\\\\\\/VmC8w7h06riLZXhsSSMSJzuDRxf8iG0GRpM95CgZA2n+zC+gLd7PaXQe7Z\\\\\\\/vr74CQ9m5JVbLM+I0uMeLuPKyBPZQwtZkOFVOhTDcEmgP6ImiyXxb1sYOvew2jovwFc20CiwoxXgQzM7oCf55gH\\\u0022,\\\u0022iv\\\u0022:\\\u00223dyO3IzCWbpwPSwZgWFpLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225633b1c57db67bba220b039b0e340801fbdaebba19a4777a55eaf8e59131c0ba\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aD\\\\\\\/ai80DoDtipt0razyRDZ+Kk4v3jHeC3MOY1mXcOZZrFSpWRJVoGUreimDjH7MlC5t\\\\\\\/RibS7V7OpWtpw9IuYWDAwvAvfEwSca8mJEj4z6OfCuTNH4b4BDUF9pxYKj0lzLCNnOHAoql4gzaNFChy9hQFLLYIb1ltSrcnLX\\\\\\\/UvYZpqpOjemHje0xdg4AOrasvJzJ6WsVT\\\\\\\/KYzvDyFuuXYUfKJbvNKB9PbqC5DDd7xo\\\\\\\/Qzf1d1DxTjaW\\\\\\\/1C2IBAghBFngWt+v9IlEhulf9ACzFTHDICp0CiJQCwVdBNNzmhGLsjlMcqjCMl9ow8MV\\\\\\\/0\\\\\\\/jK38RWaX6a5XoGWeEfmqlPiKaU3WAgTEoT2hwaZVvRUdQmlJ9uLjZR1MD6FiNeJSlvsKbQYIql3\\\\\\\/5QPjBQxWvaXDwPTNMwQnIO21OHdLKAgfXoOR9xZJyO8FPjXUwm3d1beKAmjNaJJkxZGa0bid0POCrsh6ugZFi68FiT1wfi+SLNK4AHMoPai1bjvCaj9GQsf5yYve6SRKut2vSSkwLdGNXNnKSSZ1gKj0BhG+N87SYcSXLolSnTrZCp6w7uQ4oDDPNbrVPWhOZiUU68lXnM40lGYlge9o83AI4PpEPq5Wun7KlK+mt7D7f4auUdPKdOkVvrB+xpE33rCX4mEWyIDfEUJgtxhzSThnVmVO0z5hPAQnY2bhV3L5bxSPBD5ciKpRVrOsv9E80KfiWXpwfU8ytQFN8K3Wki33P\\\\\\\/wFNai8sefX9zcH3tab9cR3CDKPyvCa8jTeCwIhqCSQxJcX4Ngz5fCYo2BduhUU4z5O6Oo6CElsJ7+oafkgDBMC0UYm+Sj2Df1xivAGq1qyt23kgiiRJncV8RLbxmioHPVBAWE9sH4PN8hTQGj4ybqZRgqggqfP0MQ1Z4c+eCHM9oBGGgSWVSDCv\\\\\\\/j+4yltd\\\\\\\/X6siTPhBb4AqwUe0K\\\\\\\/uaX\\\\\\\/v0dlF4AeMrMRRF9X9R2AuiOEHV4VVELAu2lCRLZWQTfKlqPZehAjNfB+cBwMiLbk4o5VSfZLH1qTkNcBWcwHD0cwo6yoRWVYY3l4LoDkFscbGoMeH1ZrGtzfuLOo5s6mmbXxLcY2kBcZWCHNE5UtIB01Gz1O+PcK8Miz6hrwGYvTdaFPEmrVKGZA60Z\\\\\\\/DNO2iAcLe+A5yliXoAxe8pNco4EIgy5MTQEw0mcrMjMeFF5mpuGKTrzpTr5naNZkUc7KsqPpRp89pelG0nKFctykXUZ40ZFLWRDdEF3QZJEXzKgGZFfdFKbSmWWzQvxcppY2L9m\\\\\\\/Quea5BCT2\\\\\\\/UDg8l\\\\\\\/fJF8snP0wbnEYWPYN1rDMM0W4UUwi1T44Lqm5nurZdwvorLvHpyKWiM6QxHuAiDZv5fZDG8DAw7kWVDaxaXHOf8meiSjHDcHM6Ny+kkReZkIa260Kqo4uE5YQ6KJxGYg\\\\\\\/BWBqmRki0qY6SozK2hIidnNZwNPl3TkqqI2BrPOqs3v9\\\\\\\/0vuhvSlKo1RFxDQZ2aWFW11gtdFsyjFzYCfvVTHk\\\\\\\/DIKZY5oeO6JHM+sZ3W6NOoW7VevLgRRx6oHHKxBP1l2zdwcL2dsNkipREqp10\\\\\\\/t4lBWLEkaecrr+LUXLZDs628VjsQdWdRM4lzeawEXCFjaYImqQjZBGw+jbO9Dzz6LVYZgIptLDxIRZsNfqzVZFXTTUPNSSUe1Yz9gRtPnQdeBZZi6JWXSt64XLVC94fwbaAuURl97GcjZI7jcbup+GsonoJU8gyopYYPeDw1nVLE3xElHOz\\\\\\\/sZdpI7xHZTJaz8ioHZ2zAs\\\\\\\/dj8kr05im+EeXiOQJujylpzAylHxg8Yzf5rzPC\\\\\\\/GCMJhKLXedVLp1AhdfkySNH95DHmfHIOkZ7HQ0Qi6qjnEnZktezKGLob3RpixJorRm24iPmrECe0lFq0mP2HZEn1\\\\\\\/BMp8OKjc2zbC5OdkRqwqySX3L3HGaHdrlnXLgJNJ02ynKRULACdtW90ZgcGYjJF+6ACke3PmpBRY3jkY5K3fBpEmCWpNg6hqMeIOKRkqdcVsUIfzg+qJ9QfG7F2BwF6fhKCa4O4NrlcKeLWlrm\\\u0022,\\\u0022iv\\\u0022:\\\u0022xOh02u6CTfERDNwdT7U9Wg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b2ba919e41ca487b156f633d06fcae0c77f2d3c98baa4423b8809d450f05f5e1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459447 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/709" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0sEzsDVzLt0bSLyOGtbAUd1n50t7DcgrjfVWbpk0X24" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/709" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gW4ZAqDukhwH3DyxWrZv1HYk3Jh3DmK0WIPYZczUF30" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/695" title="Transcriptome Profiling Following Neuronal and Glial Expression of ALS-Linked SOD1 in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/719" title="Involvement of MLPK Pathway in Intraspecies Unilateral Incompatibility Regulated by a Single Locus With Stigma and Pollen Factors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459447" id="node459447" data-pisa="ggg;3/4/709" data-pisa-master="ggg;g3.112.005124" data-apath="/ggg/3/4/709.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (<em>Scomberomorus commerson</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gilbert Michael Macbeth</span>, <span class="highwire-citation-author" data-delta="1">Damien Broderick</span>, <span class="highwire-citation-author" data-delta="2">Rik C. Buckworth</span> and <span class="highwire-citation-author" data-delta="3">Jennifer R. Ovenden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">709-717;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005124 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459447" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gilbert Michael Macbeth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilbert%2BMichael%2BMacbeth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macbeth%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGilbert%2BMichael%2BMacbeth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gilbert.macbeth1@uq.net.au" class="" data-icon-position="" data-hide-link-title="0">gilbert.macbeth1@uq.net.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Damien Broderick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damien%2BBroderick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broderick%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADamien%2BBroderick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rik C. Buckworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Marine & Atmospheric Research, Wealth from Oceans National Research Flagship, Brisbane, Queensland, 4001, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rik%2BC.%2BBuckworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckworth%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARik%2BC.%2BBuckworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer R. Ovenden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BR.%2BOvenden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ovenden%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BR.%2BOvenden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/4/709" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459447" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/709.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459447" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/709.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459447" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/709.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459447" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/4/709.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459447" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Estimates of genetic effective population size (<em>Ne</em>) using molecular markers are a potentially useful tool for the management of endangered through to commercial species. However, pitfalls are predicted when the effective size is large because estimates require large numbers of samples from wild populations for statistical validity. Our simulations showed that linkage disequilibrium estimates of <em>Ne</em> up to 10,000 with finite confidence limits can be achieved with sample sizes of approximately 5000. This number was deduced from empirical allele frequencies of seven polymorphic microsatellite loci in a commercially harvested fisheries species, the narrow-barred Spanish mackerel (<em>Scomberomorus commerson</em>). As expected, the smallest SD of <em>Ne</em> estimates occurred when low-frequency alleles were excluded. Additional simulations indicated that the linkage disequilibrium method was sensitive to small numbers of genotypes from cryptic species or conspecific immigrants. A correspondence analysis algorithm was developed to detect and remove outlier genotypes that could possibly be inadvertently sampled from cryptic species or nonbreeding immigrants from genetically separate populations. Simulations demonstrated the value of this approach in Spanish mackerel data. When putative immigrants were removed from the empirical data, 95% of the <em>Ne</em> estimates from jacknife resampling were greater than 24,000.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/effective-population-size" class="hw-term hw-article-keyword hw-article-keyword-effective-population-size" rel="nofollow">effective population size</a></li><li class="kwd"><a href="/keyword/bias" class="hw-term hw-article-keyword hw-article-keyword-bias" rel="nofollow">bias</a></li><li class="kwd"><a href="/keyword/nontarget-populations" class="hw-term hw-article-keyword hw-article-keyword-nontarget-populations" rel="nofollow">nontarget populations</a></li><li class="kwd"><a href="/keyword/correspondence-analysis" class="hw-term hw-article-keyword hw-article-keyword-correspondence-analysis" rel="nofollow">correspondence analysis</a></li><li class="kwd"><a href="/keyword/outliers" class="hw-term hw-article-keyword hw-article-keyword-outliers" rel="nofollow">outliers</a></li></ul><p id="p-4">The effective number in a breeding stock was defined by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wright (1930)</a> as an idealized number of parents in a population that cause a given level of inbreeding or given change in allele frequencies. This effective number “is much smaller as a rule than the actual number of adult individuals” (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Wright 1930</a>) but is an important parameter in ecological studies because any change over time indicates underlying changes in population structure. The mean squared correlation of alleles at different loci is a measure of linkage disequilibrium, which can be used to estimate genetic effective population size <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-1.gif"/></noscript></span></span> of diploid individuals. In small populations there is a greater correlation of alleles between loci compared with larger populations (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pudovkin <em>et al.</em> 1996</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hedgecock <em>et al.</em> 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhdanova and Pudovkin 2008</a>) and hence there is a relationship with genetic effective population size (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Waples 2006</a>). It was suggested by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Waples and Do (2010)</a> that strong assortative mating would lead to biases in <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-2.gif"/></noscript></span></span>. Later, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Waples and England (2011)</a> investigated migration between populations and concluded that the linkage disequilibrium method was robust to equilibrium migration with <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-3.gif"/></noscript></span></span>, reflecting that of the local subpopulation. <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Waples and England (2011)</a> also showed that pulse migration of strongly divergent individuals was found to depress estimates of local <em>Ne</em>.</p><p id="p-5">The effect of pulse migration is an important finding because related factors could also lead to depressed <em>Ne</em> estimates. These factors could include inadvertent sampling of nontarget species and sampling of the same species but from populations that have become genetically divergent over many generations. Some fish species are known to exhibit natal philopatry, in which individuals have home spawning grounds but later disperse. Examples include herring, cod, sharks, swordfish, and anadromous salmonids (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beacham <em>et al.</em> 2005</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bekkevold <em>et al.</em> 2007</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Svedang <em>et al.</em> 2007</a>; Jorgensen <em>et al.</em> 2009; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Smith and Alvarado-Bremer 2010</a>). Under this model, samples from a single location taken when the species was in the dispersed phase could represent several genetically distinct (<em>i.e.</em>, mixed) stocks. The samples would not represent a panmictic population, causing deviations from the expected linkage disequilibria and a bias in the linkage disequilibrium estimation of <em>Ne</em>. For example, a downward bias in <em>Ne</em> estimates was simulated by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Palstra and Ruzzante (2011)</a> when divergent populations were pooled.</p><p id="p-6">The frequency of natal philopatry is poorly known across marine species and virtually unknown in Australian fisheries species (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blower <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Tillett <em>et al.</em> 2012</a>). A species of considerable fisheries interest in Australia, and much of the Indo-West Pacific, is the narrow-barred Spanish mackerel, <em>Scomberomorus commerson</em>. It is a large, fast-swimming pelagic predator found throughout tropical and subtropical neritic waters of the Indo-West Pacific (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Collette and Nauen 1983</a>). If <em>S. commerson</em> exhibit natal philopatry, the mixing of genetically distinct populations within the sample collection area could depress <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-4.gif"/></noscript></span></span> in a similar manner suggested by pulse migration (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Waples and England 2011</a>). Seasonal aggregation for spawning followed by dispersal is supported by several lines of evidence: (1) seasonal variations in the availability of <em>S. commerson</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buckworth <em>et al.</em> 2007</a>), (2) a tag release study in northern Australia showing dispersal of recaptured fish with 12% more than 600 nautical miles away (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Buckworth <em>et al.</em> 2007</a>), (3) movement of fish on the eastern Australian coast southwards in summer presumably for feeding (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">McPherson 1988</a>), and (4) multiple genetically distinct stocks in Southeast Asia (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fauvelot and Borsa 2011</a>). The species is under active management throughout its range in Australia, and accurate estimates of effective population size have the potential to assist (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hare <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Luikart <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ovenden <em>et al.</em> 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Palstra and Ruzzante 2008</a>).</p><p id="p-7">In this work we document a case study of the pitfalls associated with the estimation of <em>Ne</em> in <em>S. commerson</em> when large samples of genotypes (<em>S</em> &gt; 5000) were sampled from a single location in northern Australia. We compare the estimated <em>Ne</em> from simulated populations with those from the empirical data. We critically review the estimates of <em>Ne</em> by testing hypotheses that the sampled population is a mixed stock. We also develop a method of screening and removing individuals likely to be from nontarget populations or species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Linkage disequilibrium estimation of effective population size (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-5.gif"/></noscript></span></span>)</h3><p id="p-8">Linkage disequilibrium estimation of effective population size is derived from the correlation of alleles between loci. The correlation is determined from allele frequencies and has the general form of the phi correlation coefficient<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-6.gif"/></noscript></span></span>(<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Weir 1996</a>, p137) where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-7.gif"/></noscript></span></span> is the estimated correlation between the <em>j</em><sup>th</sup> allele in locus <em>A</em> and <em>k</em><sup>th</sup> allele in locus <em>B</em> given <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-8.gif"/></noscript></span></span> is the empirical frequency estimation of allele <em>j</em> in locus <em>A</em>, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-9.gif"/></noscript></span></span> is the empirical frequency estimation of allele <em>k</em> in locus <em>B</em>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-11.gif"/></noscript></span></span> represent the additional variance in allele frequencies due to deviations in Hardy Weinberg equilibrium, where <em>f</em>() in the aforementioned formulae denote the observed homozygote frequencies. When diploid genotypes are sampled, the gametic phase is unknown with linkage disequilibrium determined by the Burrows estimate <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-12.gif"/></noscript></span></span> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Schaid 2004</a>). In this equation, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-13.gif"/></noscript></span></span> is the deviation from the estimated probability of <em>A<sub>j</sub>B<sub>k</sub></em> gametes, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-14.gif"/></noscript></span></span>, from their expected probability <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-15.gif"/></noscript></span></span>. The value <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-16.gif"/></noscript></span></span> had to be determined indirectly from the count of <em>A<sub>j</sub>B<sub>k</sub></em> combinations within biallelic genotypes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) because the gamete frequencies <em>A<sub>j</sub>B<sub>k</sub></em> were unknown. In <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, # indicates that there were no <em>A<sub>j</sub>B<sub>k</sub></em> gametes present within the genotype; thus, the expected number of <em>A<sub>j</sub>B<sub>k</sub></em> gametes given the genotype <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-17.gif"/></noscript></span></span> is equal to <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-18.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-19.gif"/></noscript></span></span> is the number of observed <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-20.gif"/></noscript></span></span> genotypes. The estimated observed frequency of <em>A<sub>j</sub>B<sub>k</sub></em> gametes summed from both intra- and intergametic loci is as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-21.gif"/></noscript></span></span>with <em>X</em> being the count of each genotype and <em>G</em> is the total number of gametes (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Schaid 2004</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462649/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462649/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Count of <em>A<sub>j</sub>B<sub>k</sub></em> pairs within genotypes created from parental gametes at locus <em>A</em> and <em>B</em>, where <em>j*</em> (or <em>k*</em>) is not allele <em>j</em> (or <em>k</em>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-10">Under the assumption of unlinked and neutral loci, effective population size was estimated using linkage disequilibrium by correcting second-order terms for sampling error:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-23.gif"/></noscript></span></span>, given <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-24.gif"/></noscript></span></span> is the observed <em>r</em>-squared component calculated as the mean <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-25.gif"/></noscript></span></span> between all alleles over <em>L</em>(<em>L</em>-1)/2 pairwise comparisons of <em>L</em> loci, and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-26.gif"/></noscript></span></span> is the term correcting upward bias due to sampling <em>S</em> individuals (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Waples 2006</a>). The derivation of these equations was the subject of an entire article (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Waples 2006</a>); in summary <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-27.gif"/></noscript></span></span> is a quadratic solution (equation 1) for <em>Ne</em> formed by equating <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-28.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-29.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-30.gif"/></noscript></span></span> is the drift term assuming loci are unlinked in a random mating population and<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-31.gif"/></noscript></span></span> is a second-order correction determined by <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Waples (2006)</a> using simulations.</p><p id="p-11">Large undefined <em>Ne</em> estimates occur when the correction due to finite sample size <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-32.gif"/></noscript></span></span> is greater than <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-33.gif"/></noscript></span></span>, resulting in a negative <em>Ne</em> estimate. Negative estimates are plausible and indicate that the sample size <em>S</em> is too small, with the correction for sample size being larger than the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-34.gif"/></noscript></span></span> value determined from the data. <em>Ne</em> estimates were determined using program LDNE, where the lower 95% confidence intervals of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-35.gif"/></noscript></span></span> were determined by the jackknife method (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Waples and Do 2008</a>).</p><p id="p-12">Built into the program of <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Waples and Do (2008)</a> is a threshold called <em>P</em><sub>crit</sub>, which is used to exclude <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-36.gif"/></noscript></span></span> from the average <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-37.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-38.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-39.gif"/></noscript></span></span> are below the <em>P</em><sub>crit</sub> threshold. Allele frequencies close to zero can bias <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-40.gif"/></noscript></span></span> (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Waples 2006</a>). We investigate <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-41.gif"/></noscript></span></span> across a range of <em>P</em><sub>crit</sub> values because low-frequency alleles are more common in large datasets. Although the theory of <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Waples (2006)</a> was tested using diallelic loci, it applies equally well in polymorophic data sets (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Waples and Do 2010</a>).</p></div><div id="sec-3" class="subsection"><h3>Collection of empirical data</h3><p id="p-13">Effective population size was estimated from genotypes of <em>S. commerson</em> individuals collected from a defined area, largely within 500 km northwest of Darwin, Northern Territory. Detailed genotyping methods are provided (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/005124SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-4" class="subsection"><h3>Simulations with different effective population sizes</h3><p id="p-14">Ten-thousand replicate linkage disequilibrium <em>Ne</em> estimates were determined each for a range of population sizes <em>N</em> from 3000 to 60,000. The genotypes in each simulated population were generated using program SHAZA (<a href="http://molecularfisherieslaboratory.com.au/download-software/">http://molecularfisherieslaboratory.com.au/download-software/</a>) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Macbeth <em>et al.</em> 2011</a>). This program simulated <em>N</em> first-generation diploid genotypes by random sampling alleles within loci from the empirical allele frequencies of <em>S. commerson</em> from the Darwin population. The first <em>N</em>/2 genotypes were defined as females and the remainder males. Each individual in the next generation was simulated by random selection of a male and female with replacement. For each parental genotype and for all seven loci a single allele was randomly selected to create an individual diploid genotype. After this process, a total number of <em>N</em> individuals was created in four discrete generations.</p><p id="p-15">In this design <em>N</em> is approximately equal to <em>Ne</em> (<a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Waples 2006</a>). In each replicate, <em>Ne</em> was estimated from 5413 generation four genotypes using a plan 2 sampling procedure (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Waples 1989</a>). Generation four was used to estimate <em>Ne</em> because this was sufficient for <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-42.gif"/></noscript></span></span> to approach an asymptotic value (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sved 1971</a>, <a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">Waples 2006</a>). For example, the expectation of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-43.gif"/></noscript></span></span> in the first generation of simulated genotypes will be zero, resulting in upwardly biased estimates of <em>Ne</em>. Simulated genotypes have no missing loci; therefore, before estimating <em>Ne</em>, we introduced missing loci to emulate the empirical data structure that had missing loci. The missing loci were introduced for each and every genotype in the simulated data by randomly drawing with replacement a genotype in the empirical data and deleting all loci in the simulated genotype that were found to be missing in the empirical genotype sampled.</p></div><div id="sec-5" class="subsection"><h3>Ne estimates from empirical data with outlier genotypes removed</h3><p id="p-16">Putative “outlier” genotypes, defined as genotypes not originating from the focal population under investigation, were identified and removed from the empirical data using a correspondence analysis (CA). The CA algorithm used here was developed in a pilot study by visual assessment of simulated outliers from plots of the first two principal components of a singular value decomposition. Up to 10 CA iterations were performed with iterations continuing until no further outliers are found. In each iteration, outlier genotypes were defined when principal components <em>PC1</em> and <em>PC2</em> (Appendix A) satisfied a threshold <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-44.gif"/></noscript></span></span>, which removed outliers furthest from the central cluster.</p></div><div id="sec-6" class="subsection"><h3>Ne estimates from empirical data with outlier genotypes removed and genotypes from nontarget species added</h3><p id="p-17">To test the sensitivity of <em>Ne</em> estimates in genotype samples containing nontarget species, a test was conducted by adding 100 genotypes of a nontarget species (gray mackerel, <em>Scomberomorus semifasciatus</em>) to the “cleaned” <em>S. commerson</em> data. We would anticipate that adding foreign genotypes will increase <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-45.gif"/></noscript></span></span> bias and indirectly show that cleaning the data could reduce bias in empirical data estimates. <em>Scomberomorus semifasciatus</em> genotypes amplified at five of the seven <em>S. commerson</em> loci with alleles at loci SCA47 and SCA49 marked as missing.</p></div><div id="sec-7" class="subsection"><h3>Simulation of genetically divergent populations</h3><p id="p-18">To further test the efficiency of the CA algorithm for detecting outlier genotypes, we considered 10 simulated populations that diverged from a founding population across numerous generations. The allele frequencies of the founding population matched those from empirical <em>S. commerson</em> samples. Population size was set at <em>N</em> = 10,000 and after 100, 200, 500, 1000, or 2000 generations, the population was sampled (sample size of 5413 genotypes). As described previously, the program SHAZA was used to generate <em>N</em> genotypes of the founding population. This was followed by creating <em>N</em> genotypes each successive generation from random sampling of parental alleles as described previously using an equal sex ratio. Pairwise <em>F<sub>ST</sub></em> values were determined between divergent, simulated populations using Genetix 4.05 software (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Belkhir <em>et al.</em> 1996−2004</a>). For each of the 10 populations, 100 samples were randomly removed and replaced by 100 random genotypes selected from one of the other nine populations. Following this procedure, we had <em>n</em> = 90 populations with 100 immigrant genotypes from nontarget populations and <em>n</em> = 10 populations with no immigrants. <em>Ne</em> was estimated before and after the data were cleaned using CA.</p><p id="p-19">The ability of the CA algorithm to identify immigrants was compared with the Bayesian clustering approach of STRUCTURE, version 2.3.3 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Pritchard <em>et al.</em> 2000</a>). STRUCTURE analysis was applied to the 90 populations that contained 100 immigrants after diverging 2000 generations. Runs were performed by specifying: <em>k</em> = 2 clusters, an admixture ancestry model with allele frequencies correlated and a burn in length of 100,000 iterations followed by 100,000 MCMC iterations. One sample location was assumed with no location prior possible.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Empirical data</h3><p id="p-20">The majority of the 5413 <em>S. commerson</em> samples were genotyped with all seven loci (71%), but some samples were genotyped with either six (12%), five (10%), and four (7%) polymorphic microsatellite loci. The numbers of alleles per microsatellite locus varied from 24 (90RTE) to 38 (SCA8), with 65% of alleles across all loci having frequencies less than or equal to 0.01 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). These low-frequency alleles were selectively removed from data used to estimate <em>Ne</em> by the LDNe software depending on the chosen <em>P</em><sub>crit</sub> thresholds (for more details see: Supplementary genotype results).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462725/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462725/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Locus and allele frequency summary</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Against expectations, LDNE estimates <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-46.gif"/></noscript></span></span> from empirical data varied systematically across <em>P</em><sub>crit</sub> values (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). As the <em>P</em><sub>crit</sub> threshold decreased in magnitude, so too did the magnitude of non-negative estimates of <em>Ne</em>. This covariance raised doubts about setting <em>P</em><sub>crit</sub> to 1/(2<em>S</em>) = 1/(2 × 5413)~0.0001, where all singleton alleles would be removed, and the general effectiveness of removing low-frequency alleles for the estimation of <em>Ne</em>. The lower confidence interval of <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-47.gif"/></noscript></span></span> was more stable than the mean estimates but still varied widely from 406 to 24,728 and as such provided no informative value of the lower bound of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-48.gif"/></noscript></span></span>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462639/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462639/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimates of LDNE effective population size <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/T3/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/T3/embed/mml-math-49.gif"/></noscript></span></span> in <em>S. commerson</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Simulations with different effective population sizes</h3><p id="p-25">Simulations indicated that 5413 genotype samples should be sufficient to estimate effective population size if the true size was 3000 and 10,000 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Simulations with <em>N</em> = 3000 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) had no extreme estimates of <em>Ne</em>, whereas simulations with <em>N</em> = 10,000 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) had a small number of outlier estimates that were greater than 40,000 or less than minus 20,000. In <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">2</a>, <em>P</em><sub>crit</sub> values between 0.01 and 0.001 gave the smallest SD of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-55.gif"/></noscript></span></span>, illustrating the importance of removing the majority of low frequency alleles.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/709/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency of 10,000 Ne estimates when simulating a population size of N = 3000 at different Pcrit values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1453907481" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency of 10,000 &lt;em&gt;Ne&lt;/em&gt; estimates when simulating a population size of &lt;em&gt;N&lt;/em&gt; = 3000 at different &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;crit&lt;/sub&gt; values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/709/F1.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/709/F1.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/709/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/709/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462670" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-26" class="first-child">Frequency of 10,000 <em>Ne</em> estimates when simulating a population size of <em>N</em> = 3000 at different <em>P</em><sub>crit</sub> values.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/709/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency of 10,000 Ne estimates when simulating a population size of N = 10,000 at different Pcrit values. The frequency of all Ne estimates less than 20,000 and greater than 40,000 were pooled and are indicated on the x-axis limits of each graph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1453907481" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Frequency of 10,000 &lt;em&gt;Ne&lt;/em&gt; estimates when simulating a population size of &lt;em&gt;N&lt;/em&gt; = 10,000 at different &lt;em&gt;P&lt;/em&gt;&lt;sub&gt;crit&lt;/sub&gt; values. The frequency of all &lt;em&gt;Ne&lt;/em&gt; estimates less than 20,000 and greater than 40,000 were pooled and are indicated on the &lt;em&gt;x&lt;/em&gt;-axis limits of each graph.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/709/F2.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/709/F2.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/709/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/709/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462626" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-27" class="first-child">Frequency of 10,000 <em>Ne</em> estimates when simulating a population size of <em>N</em> = 10,000 at different <em>P</em><sub>crit</sub> values. The frequency of all <em>Ne</em> estimates less than 20,000 and greater than 40,000 were pooled and are indicated on the <em>x</em>-axis limits of each graph.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">As expected, simulations with <em>N</em> = 100 and <em>N</em> = 1000 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS2.pdf">Figure S2</a>) gave more precise estimates of <em>Ne</em> than with <em>N</em> = 3000 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Increasing <em>N</em> from 10,000 to 30,000 and 60,000 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS4.pdf">Figure S4</a>) resulted in a lower precision of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-56.gif"/></noscript></span></span> with a greater number of negative and extremely large estimates of <em>Ne</em>. An interesting finding was that, at large <em>N</em> values such as 60,000, the lower 95% confidence interval (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS5.pdf">Figure S5</a>) was more precise than the expected mean value (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS4.pdf">Figure S4</a>), particularly at <em>P</em><sub>crit</sub> values around 0.01. The results indicate that we had sufficiency in the data to detect the lower 95% confidence interval if <em>N</em> was equal to 60000 with the mean lower confidence interval being 22,188 (<em>P</em><sub>crit</sub> = 0.01).</p><p id="p-29">It is important to note that the smallest 1% of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-57.gif"/></noscript></span></span> using <em>P</em><sub>crit</sub> = 0.0000 determined from the 100th-ranked positive value was 4134, 5308, and 5846 when <em>N</em> was 10,000, 30,000, and 60,000, respectively, which revealed an anomaly between the simulation results and empirical data estimates of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-58.gif"/></noscript></span></span>. If the true <em>Ne</em> was larger than 10,000, then the smallest <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-59.gif"/></noscript></span></span> estimate expected at <em>P</em><sub>crit</sub> = 0.0000 would be greater than 4134 (<em>P</em> &lt; 0.01), which differs from the empirical estimate of 418. Conversely, if the true <em>Ne</em> was smaller than or equal to 10,000, then simulations indicated that no negative estimates of <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-60.gif"/></noscript></span></span> would be expected at <em>P</em><sub>crit</sub> = 0.02 (<em>P</em> &lt; 0.0001), which was contrary to that observed from empirical data with <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-61.gif"/></noscript></span></span> (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). This finding highlighted that there was a significant difference between the empirical and simulated data, which was subsequently investigated by examining outlier genotypes.</p></div><div id="sec-11" class="subsection"><h3><em>Ne</em> estimates from empirical data with outlier genotypes removed</h3><p id="p-30">The removal of putative outlier genotypes from empirical <em>S. commerson</em> data took nine <em>CA</em> iterations before there were no genotypes exceeding the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-62.gif"/></noscript></span></span> threshold (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS6.pdf">Figure S6</a>). An order of magnitude increase in <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-63.gif"/></noscript></span></span> (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) was observed after the first iteration, which removed just 33 outliers (0.6% of total number of genotypes). This finding indicated that putative outlier genotypes can significantly bias <em>Ne</em> estimates in empirical data.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462673/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462673/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Estimates of <em>Ne</em> in <em>S. commerson</em> after CA iterations</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">After the nine <em>CA</em> iterations, 3.2% of samples were removed. Subsequent <em>Ne</em> estimates on the cleaned data were negative at all <em>P</em><sub>crit</sub> thresholds, except when <em>P</em><sub>crit</sub> was 0.01 <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-65.gif"/></noscript></span></span>. This finding indicated that a <em>P</em><sub>crit</sub> of 0.01 provided the greatest accuracy, as it had the smallest confidence interval assessed by the fact that it was the only <em>P</em><sub>crit</sub> value in which the correlation of alleles between loci was greater than that expected from sampling error. At this <em>P</em><sub>crit</sub> value <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-66.gif"/></noscript></span></span> was relatively stable at around 80,000 to 100,000 until the last two iterations with <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-67.gif"/></noscript></span></span> increasing to 550,582. When <em>P</em><sub>crit</sub> was 0.01, the harmonic mean of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-68.gif"/></noscript></span></span> across all nine iterations was 110,000.</p><p id="p-34">The lower 95% confidence interval of the <em>Ne</em> estimates <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-69.gif"/></noscript></span></span> from <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> is reported in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. The lower confidence intervals appeared to be more stable than the estimates provided in <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> when the <em>P</em><sub>crit</sub> values were equal to or greater than 0.001. The range of <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-70.gif"/></noscript></span></span> estimates when <em>P</em><sub>crit</sub> = 0.01 were within 21% of each other with a harmonic mean of 24000.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462679/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462679/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Lower 95% confidence interval of <em>Ne</em> from <em>S. commerson</em> genotypes</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3><em>Ne</em> estimates from empirical data with outlier genotypes removed and genotypes from nontarget species added</h3><p id="p-37">Adding nontarget species (gray mackerel, <em>S. semifasciatus</em>) to the “cleaned” <em>S. commerson</em> data significantly reduced <em>Ne</em> estimates (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). Considering the total sample size was 5413, the results clearly show that only a small proportion of nontarget species can have a large impact on linkage disequilibrium estimates of <em>Ne</em>. For example, adding as few as eight (0.15%) <em>S. semifasciatus</em> genotypes resulted in a 5.7-fold reduction in <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-74.gif"/></noscript></span></span> when <em>P</em><sub>crit</sub> = 0.01. All of the 200 nontarget gray mackerel genotypes were identified and removed by the first iteration of CA analysis compared with the nine iterations that were required with the empirical data (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). This finding suggests that the putative outliers in the empirical data were more similar to <em>S. commerson</em> than <em>S. semifasciatus</em>.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462682/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462682/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Effect of <em>S. commerson Ne</em> estimates when adding nontarget species</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Our <em>S. semifasciatus</em> samples did not amplify at loci SCA47 and SCA49. Removing all genotypes in the empirical data that did not amplify at these two loci produced a similar <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-77.gif"/></noscript></span></span> profile to <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, indicating that <em>S. semifasciatus</em> cannot be solely implicated in the anomaly between the simulated and empirical data.</p></div><div id="sec-13" class="subsection"><h3>Simulation of genetically divergent populations</h3><p id="p-41">Ten populations simulated after divergence from a common founder population had average pairwise <em>F<sub>ST</sub></em> values of 0.004, 0.010, 0.027, 0.048, and 0.091 after 100, 200, 500, 1000, and 2000 generations, respectively. With no mixing of the populations during genotype sampling, <em>Ne</em> estimates approximated the simulated population size (<em>N</em> = 10,000, <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462731/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462731/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Harmonic mean of <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/T7/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/T7/embed/mml-math-78.gif"/></noscript></span></span> before and after outlier genotypes removed</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">Ninety populations with 100 immigrants were created from pairs of the 10 divergent populations. Across these 90 populations CA analysis found an average (SD) of 7 (4), 18 (8), 44 (12), 74 (12), and 93 (6) immigrants after 100, 200, 500, 1000, and 2000 generations, respectively. The average number of CA iterations required before no more immigrants could be detected were 3.4, 3.6, 3.6, 3.1, 3.0 after 100, 200, 500, 1000, and 2000 generations, respectively. As a comparison, the program STRUCTURE was not able to distinguish the immigrants, even after 2000 generations of divergence. When two populations were specified in STRUCTURE 97 of the 100 immigrants and 47.3% of the remaining 5313 samples were partitioned into the same population. This finding indicated that there was not sufficient genetic divergence between the populations to cluster the small proportion of immigrants into a separate population.</p><p id="p-44">In the presence of 100 immigrants, there was a downward bias in <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-81.gif"/></noscript></span></span> of the focal population for <em>P</em><sub>crit</sub> values of 0.00 and 0.01 (<a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>) as the number of generations of divergence increased. After outlier genotypes were removed <em>Ne</em> estimates were more consistent with an expected value of <em>N</em> = 10,000. After outlier (<em>i.e.</em>, immigrant) genotypes were removed by CA, the smallest bias and highest accuracy of <em>Ne</em> occurred when <em>P</em><sub>crit</sub> = 0.01.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-45"><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Palstra and Ruzzante (2011)</a> urged further theoretical developments to avoid a downward bias in estimating linkage disequilibrium <em>Ne</em> in naturally occurring metapopulations. Our results have demonstrated that under certain circumstances even estimates for focal populations can be downwardly biased. We believe this bias could be due to the presence of 1) nontarget species and 2) immigrant genotypes from diverged populations among the samples taken for estimation. Importantly, only a few ‘contaminant’ genotypes can severely bias <em>Ne</em> estimates.</p><p id="p-46">The nature of how the contaminant genotypes differ is at the crux of what causes the downward bias in effective population size. We assumed that contaminant genotypes were from transient individuals that did not interbreed among members of the focal population. Our results are therefore not in disagreement with a study in which the authors showed that linkage disequilibrium estimates of effective population size are robust to populations displaying equilibrium migration and mating over many generations (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Waples and England 2011</a>). We propose the bias expectations are different for contamination by unrelated species or reproductively isolated subpopulations <em>vs.</em> subpopulations from the same metapopulation.</p><p id="p-47">The CA algorithm performed well in identifying and removing nontarget genotypes that were added to simulated population samples. In our hands, standard methods of population clustering such as STRUCTURE (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Pritchard <em>et al.</em> 2000</a>) were incapable of identifying the simulated immigrants. The threshold value of 2 used in the CA algorithm was developed by trial and error as a reasonable threshold to exclude outlier genotypes without removing too many target population genotypes. A series of scatter plots on principal coordinates is shown after each iteration of removing outliers on the threshold (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS6.pdf">Figure S6</a>). The pattern in this series was typical for many of the simulations runs in which a final cluster of points becomes clearly visible. As expected, as the <em>F<sub>ST</sub></em> between nontarget and the target populations decreased, it was more difficult to detect the nontarget genotypes using the CA algorithm. Although the simulated results seem sensible, the theoretical basis of this algorithm and its generalizability to removing nontarget genotypes in other data sets would provide additional support for this method. Our findings suggest that it is worthwhile to detect and remove putative nontarget genotypes prior to LDNE analysis.</p><p id="p-48">Our simulated divergent populations were implemented using a simple Wright-Fisher model with mating modified such that gametes were chosen from populations having equal numbers in each sex. This model was used by <a id="xref-ref-32-9" class="xref-bibr" href="#ref-32">Waples (2006)</a>; however, many other models could have been used, including those with mutation and selection (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Der <em>et al.</em> 2011</a>). These additional processes would cause a larger divergence at the same number of generations compared with the simple genetic drift model used in our study.</p><p id="p-49">Our investigation suggests that mackerel genotypes collected around Darwin contained a small proportion from genetically divergent <em>S. commerson</em> population(s) or from congeneric species. It is possible that tissue samples of closely related species were taken inadvertently, thus mimicking an admixed <em>S. commerson</em> population. Our 100 gray mackerel (<em>S. semifasciatus</em>) samples amplified at five of the seven loci used in our study, whereas another closely related endemic species (<em>Scomberomorus queenslandicus</em>) amplifies at all the seven loci (unpublished data). The fact that all gray mackerel genotypes were successfully removed by our CA method does indicate that our methods work well when nontarget species are implicated. We would expect intermediate results when populations are at varying levels of population divergence as indicated by our simulations.</p><p id="p-50">We assumed no genotyping errors when estimating linkage disequilibrium, although prescreening of the data resulted in one locus being removed due to a deviation from Hardy Weinberg equilibrium. Although this deviation might indicate the presence of a null allele error, there could be other errors, such as allelic dropout errors. Random dropout errors are not expected to change the expectation of linkage disequilibrium estimates nor the outcome of the expected <em>Ne</em> estimate.</p><p id="p-51">Assuming that all samples represented <em>S. commerson</em>, it is likely that the population adjacent to Darwin is an admixed population containing small numbers of individuals from genetically distinct populations. These individuals could also have been transient vagrants of genetically distinct populations of <em>S. commerson</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sulaiman and Ovenden 2010</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Fauvelot and Borsa 2011</a>) that were sampled in the same geographical region. The hypothesis that a small (rather than large) number of immigrant genotypes were present in the empirical genotypes is supported by the observations that (1) most adults in a mark-recapture study were found to move less than 100km per year parallel to the shore and (2) isotope signatures in the sagittal otolith carbonate of <em>S. commerson</em> indicated spatial separation across northern Australia (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Newman <em>et al.</em> 2009</a>).</p><p id="p-52">In our <em>S. commerson</em> data, it was very difficult to get a precise estimate of <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-82.gif"/></noscript></span></span>. Before “cleaning” the data with CA, <em>Ne</em> estimates varied at different <em>P</em><sub>crit</sub> levels, including some negative estimates of <em>Ne</em>. Using a <em>P</em><sub>crit</sub> value of 0.01 the likely <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-83.gif"/></noscript></span></span> seems very large with an estimate of 110,000 from empirical data. We believe this estimate to be unreliable as inferred from the lack of sufficiency of the data when estimating the mean <em>Ne</em> with <em>N</em> = 600,000 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-53">Negative estimates of <em>Ne</em> are counterintuitive and indicate that the true <em>Ne</em> is large and undefined. <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Waples and Do (2010)</a> point out that even if the <em>Ne</em> estimate is negative, if adequate data are available, the lower bound of the confidence interval may be finite and can provide useful information. This finding was also supported by our simulations with large <em>N</em> values in which the lower 95% confidence interval for <em>S. commerson</em> appears to be much more stable than the estimate and upper limits. Using a <em>P</em><sub>crit</sub> value of 0.01, we found that the lower 95% confidence interval gave a harmonic mean of <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-84.gif"/></noscript></span></span> from empirical data. More loci could be used to achieve more precise estimates of <em>Ne</em>. However, we had sufficiency in the data to detect <em>Ne</em> when <em>N</em> = 30,000 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS3.pdf">Figure S3</a>, <em>P</em><sub>crit</sub> = 0.01). We also had sufficiency in the data when estimating the lower 95% confidence of <em>Ne</em> with <em>N</em> = 60,000 giving <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/709/embed/mml-math-85.gif"/></noscript></span></span> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1/FigureS5.pdf">Figure S5</a>, <em>P</em><sub>crit</sub> = 0.01). We conclude from these simulations that the empirical <em>Ne<sub>lower</sub></em> estimate of 24,000 is reasonably reliable. In ecological terms 24,000 represents a large and stable genetic population size, and we would expect to reach a similar conclusion with the addition of more loci.</p><p id="p-54">This study was primarily focused on the bias in the linkage disequilibrium estimation of <em>Ne</em> when a population may include genetically divergent conspecifics. There are many other approaches used to estimate <em>Ne</em> that have different underlying assumptions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Barker 2011</a>) and that should be evaluated as being suitable for the estimation of <em>Ne</em> in large, naturally occurring populations. A natural progression in this area of research is to develop inferences of census population sizes based on effective size estimates (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Palstra and Ruzzante 2008</a>) and how these could be used to assist management of natural resource species.</p><p id="p-55">Realistic simulations have showed that it is possible to make effective population size estimates using the linkage disequilibrium method with finite confidence limits up to several thousand depending on the number of loci and genotypes assayed. Estimates of effective size made from samples taken from naturally occurring populations must be treated with caution. We recommend pretreatment of the sampled genotypes to identify outliers, particularly if the population being studied is sympatric with closely related species, or is possibly receiving immigrants from adjacent populations.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">We thank the authors of an earlier Fisheries Research Development Corporation project (No 2002/011) for making the <em>S. commerson</em> genotypes available and Raewyn Street and Marcus McHale for their technical support. Helpful comments were provided by Robin Waples, Phillip England, Friso Palstra, and two anonymous referees.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-15"><h3>CA R script</h3><p id="p-56">The genotype file is presented as an incidence matrix <em>Z</em> having columns for each allele within every locus. For each and every genotype a single row marking the presence ‘1’ or absence ‘0’ of each allele present within each column is appended to fill the content of the <em>Z</em> matrix. The R code modified from <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Nenadic and Greenacre (2006)</a> converts the incidence matrix to a format that can be read and manipulated by R (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">R Development Core Team 2011</a>) with the first two principal components <em>PC1</em> and <em>PC2</em> determined as follows:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-57">Z &lt;- data.matrix(Z)  # convert to matrix</p></li><li id="list-item-2"><p id="p-58">P &lt;- Z / sum(Z)  # proportional contribution</p></li><li id="list-item-3"><p id="p-59">rm &lt;- apply(P, 1, sum)  # sum rows</p></li><li id="list-item-4"><p id="p-60">cm &lt;- apply(P, 2, sum)  # sum columns</p></li><li id="list-item-5"><p id="p-61">eP &lt;- rm %*% t(cm)  # multiply by transpose</p></li><li id="list-item-6"><p id="p-62">dec &lt;- svd((P - eP) / sqrt(eP))  # singular value decomposition</p></li><li id="list-item-7"><p id="p-63">PC1 &lt;- dec$u[,1] * dec$d[1] / sqrt(rm)  # Principal component 1</p></li><li id="list-item-8"><p id="p-64">PC2 &lt;- dec$u[,2] * dec$d[2] / sqrt(rm)  # Principal component 2</p></li></ul></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.005124/-/DC1</a></p></li><li class="fn" id="fn-13"><p id="p-66"><em>Communicating editor: P. Pfaffelhuber</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-27"><span class="received-label">Received </span>November 27, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-02-18"><span class="accepted-label">Accepted </span>February 18, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 The State of Queensland, Department of Agriculture, Fisheries and Forestry</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.1" data-doi="10.1111/j.1365-294X.2011.05324.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">J. S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effective population size of natural populations of <em>Drosophila buzzatii</em>, with a comparative evaluation of nine methods of estimation</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4452</span>–<span class="cit-lpage">4471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBarker%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D20%26rft.issue%253D21%26rft.spage%253D4452%26rft.epage%253D4471%26rft.atitle%253DEffective%2Bpopulation%2Bsize%2Bof%2Bnatural%2Bpopulations%2Bof%2BDrosophila%2Bbuzzatii%252C%2Bwith%2Ba%2Bcomparative%2Bevaluation%2Bof%2Bnine%2Bmethods%2Bof%2Bestimation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05324.x%26rft_id%253Dinfo%253Apmid%252F21951766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05324.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21951766&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.2" data-doi="10.1577/T05-005.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beacham</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Candy</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntosh</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacConnachie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Estimation of stock composition and individual identification of Sockeye salmon on a Pacific rim basis using microsatellite and major histocompatibility complex variation</span>. <abbr class="cit-jnl-abbrev">Trans. Am. Fish. Soc.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BAm.%2BFish.%2BSoc.%26rft.volume%253D134%26rft.spage%253D1124%26rft_id%253Dinfo%253Adoi%252F10.1577%252FT05-005.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1577/T05-005.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.3" data-doi="10.3354/meps337187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bekkevold</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clausen</span>  <span class="cit-name-given-names">L. A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andre</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Divergent origins of sympatric herring population components determined using genetic mixture analysis</span>. <abbr class="cit-jnl-abbrev">Mar. Ecol. Prog. Ser.</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEcol.%2BProg.%2BSer.%26rft.volume%253D337%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.3354%252Fmeps337187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3354/meps337187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-3.4.709.4"><div class="cit-metadata unstructured">Belkhir, K., P. Borsa, L. Chikhi, N. Raufaste, and F. Bonhomme, 1996–2004 GENETIX 4.05, logiciel sous Windows TM pour la genetique des populations. Laboratorie Genome, Populations, Interactions, CNRS UMR 5171, Universite de Montpellier II, Montpellier (France). Available at: <a href="http://kimura.univ-montp2.fr/genetix/">http://kimura.univ-montp2.fr/genetix/</a>. Accessed: March 5, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.5" data-doi="10.3354/meps09659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blower</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandolfi</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Cabrera</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruce</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovenden</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genetics of Australian white sharks reveals fine-scale spatial structure, transoceanic dispersal events and low effective population sizes</span>. <abbr class="cit-jnl-abbrev">Mar. Ecol. Prog. Ser.</abbr> <span class="cit-vol">455</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEcol.%2BProg.%2BSer.%26rft.volume%253D455%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.3354%252Fmeps09659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3354/meps09659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000304607100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-other" id="cit-3.4.709.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckworth</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovenden</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lester</span>  <span class="cit-name-given-names">R. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPherson</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The stock structure of northern and western Australian Spanish mackerel</span>. <span class="cit-publ-name">Department of Primary Industry, Fisheries and Mines</span>, <span class="cit-publ-loc">Darwin, Australia</span>. <span class="cit-comment">Fishery Report No. 88</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-3.4.709.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckworth</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovenden</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broderick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macbeth</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPherson</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GENETAG: Genetic mark-recapture for real-time harvest rate monitoring: Pilot studies in Northern Australia Spanish Mackerel fisheries</span>. <span class="cit-publ-name">Northern Territory Government,</span> <span class="cit-publ-loc">Queensland, Australia</span>. <span class="cit-comment">Fishery Report No. 107</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collette</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nauen</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <abbr class="cit-jnl-abbrev">FAO Species Catalogue. Vol. 2. Scombrids of the World. An Annotated and Illustrated Catalogue of Tunas, Mackerels, Bonitos and Related Species Known to Date.</abbr> <span class="cit-publ-name">Food and Agriculture Organization Fish Synopsis No. 125,</span> <span class="cit-publ-loc">Rome</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.9" data-doi="10.1016/j.tpb.2011.06.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Der</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Generalised population models and the nature of genetic drift</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">80</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DDer%26rft.auinit1%253DR.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D80%26rft.epage%253D99%26rft.atitle%253DGeneralized%2Bpopulation%2Bmodels%2Band%2Bthe%2Bnature%2Bof%2Bgenetic%2Bdrift.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2011.06.004%26rft_id%253Dinfo%253Apmid%252F21718713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2011.06.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21718713&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.10" data-doi="10.1111/j.1095-8312.2011.01754.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fauvelot</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borsa</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Patterns of genetic isolation in a widely distributed pelagic fish, the narrow-barred Spanish mackerel (<em>Scomberomous commerson</em>)</span>. <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">886</span>–<span class="cit-lpage">902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D104%26rft.spage%253D886%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2011.01754.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2011.01754.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.11" data-doi="10.1111/j.1523-1739.2010.01637.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hare</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunney</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruzzante</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burford</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Understanding and estimating effective population size for practical application in marine species management</span>. <abbr class="cit-jnl-abbrev">Conserv. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DConserv%2BBiol%26rft.aulast%253DHare%26rft.auinit1%253DM.%2BP.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D438%26rft.epage%253D449%26rft.atitle%253DUnderstanding%2Band%2Bestimating%2Beffective%2Bpopulation%2Bsize%2Bfor%2Bpractical%2Bapplication%2Bin%2Bmarine%2Bspecies%2Bmanagement.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1523-1739.2010.01637.x%26rft_id%253Dinfo%253Apmid%252F21284731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1523-1739.2010.01637.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21284731&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.12" data-doi="10.1007/s00227-006-0441-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Launey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pudovkin</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naciri</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapegue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Small effective number of parents (N-b) inferred for a naturally spawned cohort of juvenile European flat oysters <em>Ostrea edulis</em></span>. <abbr class="cit-jnl-abbrev">Mar. Biol.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BBiol.%26rft.volume%253D150%26rft.spage%253D1173%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00227-006-0441-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00227-006-0441-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244454900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.4.709.13" data-doi="10.1098/rspb.2009.1155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeb</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapple</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Philopatry and migration of Pacific white sharks</span>. <abbr class="cit-jnl-abbrev">Proc. Biol. Sci. B.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%2BB.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2009.1155%26rft_id%253Dinfo%253Apmid%252F19889703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjc3LzE2ODIvNjc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.14" data-doi="10.1007/s10592-010-0050-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tallmon</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Estimation of census and effective population sizes: the increasing usefulness of DNA-based approaches</span>. <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D11%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10592-010-0050-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10592-010-0050-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.15" data-doi="10.1016/j.tpb.2011.06.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macbeth</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broderick</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovenden</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckworth</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Likelihood-based genetic mark-recapture estimates when genotype samples are incomplete and contain typing errors</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DMacbeth%26rft.auinit1%253DG.%2BM.%26rft.volume%253D80%26rft.issue%253D3%26rft.spage%253D185%26rft.epage%253D196%26rft.atitle%253DLikelihood-based%2Bgenetic%2Bmark-recapture%2Bestimates%2Bwhen%2Bgenotype%2Bsamples%2Bare%2Bincomplete%2Band%2Bcontain%2Btyping%2Berrors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2011.06.006%26rft_id%253Dinfo%253Apmid%252F21763337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2011.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21763337&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.4.709.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McPherson</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">A review of large coastal pelagic fishes in the South Pacific Region, with special reference to <em>Scomberomorus commerson</em> in North-East Australian waters [WP 15]</span>. <span class="cit-comment">Noumea: SPC. Workshop on Pacific Inshore Fishery Resources, Noumea, New Caledonia, March 14-25, 1988</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-3.4.709.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nenadic</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenacre</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Computation of multiple correspondence analysis, with code in R</span>, pp. <span class="cit-fpage">523</span>–<span class="cit-lpage">551</span> in <span class="cit-source">Multiple Correspondence Analysis and Related Methods</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Greenacre</span>  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Blasius</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Chapman &amp; Hall/CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.18" data-doi="10.1111/j.1466-8238.2009.00475.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckworth</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Spatial subdivision of adult assemblages of Spanish mackerel, <em>Scomberomorus commerson</em> (Pisces: Scombridae) across northern Australia: implications for fisheries management</span>. <abbr class="cit-jnl-abbrev">Glob. Ecol. Biogeogr.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob.%2BEcol.%2BBiogeogr.%26rft.volume%253D18%26rft.spage%253D711%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1466-8238.2009.00475.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1466-8238.2009.00475.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ovenden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtney</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyle</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The genetic effective and adult census size of an Australian population of tiger prawns (<em>Penaeus esculentus</em>)</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DOvenden%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D16%26rft.issue%253D1%26rft.spage%253D127%26rft.epage%253D138%26rft.atitle%253DThe%2Bgenetic%2Beffective%2Band%2Badult%2Bcensus%2Bsize%2Bof%2Ban%2BAustralian%2Bpopulation%2Bof%2Btiger%2Bprawns%2B%2528Penaeus%2Besculentus%2529.%26rft_id%253Dinfo%253Apmid%252F17181726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17181726&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242964100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.20" data-doi="10.1111/j.1365-294X.2008.03842.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palstra</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruzzante</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic estimates of contemporary effective population size: what can they tell us about the importance of genetic stochasticity for wild population persistence?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">3428</span>–<span class="cit-lpage">3447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPalstra%26rft.auinit1%253DF.%2BP.%26rft.volume%253D17%26rft.issue%253D15%26rft.spage%253D3428%26rft.epage%253D3447%26rft.atitle%253DGenetic%2Bestimates%2Bof%2Bcontemporary%2Beffective%2Bpopulation%2Bsize%253A%2Bwhat%2Bcan%2Bthey%2Btell%2Bus%2Babout%2Bthe%2Bimportance%2Bof%2Bgenetic%2Bstochasticity%2Bfor%2Bwild%2Bpopulation%2Bpersistence%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03842.x%26rft_id%253Dinfo%253Apmid%252F19160474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03842.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19160474&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257781300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.21" data-doi="10.1038/hdy.2011.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palstra</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruzzante</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Demographic and genetic factors shaping contemporary metapopulation effective size and its empirical estimation in salmonid fish</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DPalstra%26rft.auinit1%253DF.%2BP.%26rft.volume%253D107%26rft.issue%253D5%26rft.spage%253D444%26rft.epage%253D455%26rft.atitle%253DDemographic%2Band%2Bgenetic%2Bfactors%2Bshaping%2Bcontemporary%2Bmetapopulation%2Beffective%2Bsize%2Band%2Bits%2Bempirical%2Bestimation%2Bin%2Bsalmonid%2Bfish.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.31%26rft_id%253Dinfo%253Apmid%252F21522167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21522167&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296560300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.22" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pudovkin</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaykin</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">On the potential for estimating the effective number of breeders from heterozygote excess in progeny</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPudovkin%26rft.auinit1%253DA.%2BI.%26rft.volume%253D144%26rft.issue%253D1%26rft.spage%253D383%26rft.epage%253D387%26rft.atitle%253DOn%2Bthe%2BPotential%2Bfor%2BEstimating%2Bthe%2BEffective%2BNumber%2Bof%2BBreeders%2BFrom%2BHeterozygote-Excess%2Bin%2BProgeny%26rft_id%253Dinfo%253Apmid%252F8878701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzEvMzgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-web" id="cit-3.4.709.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">R: A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.25" data-doi="10.1534/genetics.166.1.505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaid</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage disequilibrium testing when linkage phase is unknown</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchaid%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D505%26rft.epage%253D512%26rft.atitle%253DLinkage%2BDisequilibrium%2BTesting%2BWhen%2BLinkage%2BPhase%2BIs%2BUnknown%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.1.505%26rft_id%253Dinfo%253Apmid%252F15020439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY2LzEvNTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarado-Bremer</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Inferring population admixture with multiple nuclear genetic markers and Bayesian genetic clustering in Atlantic swordfish (<em>Xiphlas gladius</em>)</span>. <abbr class="cit-jnl-abbrev">Coll. Vol. Sci. Pap. ICCAT</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DColl.%2BVol.%2BSci.%2BPap.%2BICCAT%26rft.volume%253D65%26rft.spage%253D185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.27" data-doi="10.1007/s10531-009-9699-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sulaiman</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovenden</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population genetic evidence for the east–west division of the narrow-barred Spanish mackerel (<em>Scomberomorus commerson</em>, <em>Perciformes: Teleostei</em>) along Wallace’s Line</span>. <abbr class="cit-jnl-abbrev">Biodivers. Conserv.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiodivers.%2BConserv.%26rft.volume%253D19%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10531-009-9699-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10531-009-9699-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.28" data-doi="10.1016/0040-5809(71)90011-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sved</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Linkage disequilibrium and homozygosity of chromosome segments in finite populations</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DSved%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D141%26rft.atitle%253DLinkage%2Bdisequilibrium%2Band%2Bhomozygosity%2Bof%2Bchromosome%2Bsegments%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252871%252990011-6%26rft_id%253Dinfo%253Apmid%252F5170716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(71)90011-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5170716&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.29" data-doi="10.3354/meps07140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svedang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Righton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonsson</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Migratory behaviour of Atlantic cod <em>Gadus morhua</em>: natal homing is the prime stock-separating mechanism</span>. <abbr class="cit-jnl-abbrev">Mar. Ecol. Prog. Ser.</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEcol.%2BProg.%2BSer.%26rft.volume%253D345%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3354%252Fmeps07140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3354/meps07140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.30" data-doi="10.1111/j.1095-8649.2012.03228.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tillett</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meekan</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">I. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorburn</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovenden</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evidence for reproductive philopatry in the bull shark, <em>Carcharhinus leucas</em> in northern Australia</span>. <abbr class="cit-jnl-abbrev">J. Fish Biol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">2140</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BFish%2BBiology%26rft.stitle%253DJournal%2Bof%2BFish%2BBiology%26rft.aulast%253DTillett%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D80%26rft.issue%253D6%26rft.spage%253D2140%26rft.epage%253D2158%26rft.atitle%253DEvidence%2Bfor%2Breproductive%2Bphilopatry%2Bin%2Bthe%2Bbull%2Bshark%2BCarcharhinus%2Bleucas.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8649.2012.03228.x%26rft_id%253Dinfo%253Apmid%252F22551174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8649.2012.03228.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22551174&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F709.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A generalized approach for estimating effective population size from temporal changes in allele frequency</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWaples%26rft.auinit1%253DR.%2BS.%26rft.volume%253D121%26rft.issue%253D2%26rft.spage%253D379%26rft.epage%253D391%26rft.atitle%253DA%2Bgeneralized%2Bapproach%2Bfor%2Bestimating%2Beffective%2Bpopulation%2Bsize%2Bfrom%2Btemporal%2Bchanges%2Bin%2Ballele%2Bfrequency.%26rft_id%253Dinfo%253Apmid%252F2731727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzIvMzc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A bias correction for estimates of effective population size based on linkage disequilibrium at unlinked gene loci</span>. <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D8%26rft.spage%253D167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.33" data-doi="10.1111/j.1755-0998.2007.02061.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">LDNE: a program for estimating effective population size from data on linkage disequilibrium</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Resources.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResources.%26rft.volume%253D8%26rft.spage%253D753%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2007.02061.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2007.02061.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257511600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.34" data-doi="10.1111/j.1752-4571.2009.00104.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Linkage disequilibrium estimates of contemporary Ne using highly variable genetic markers: a largely untapped resource for applied conservation and evolution</span>. <abbr class="cit-jnl-abbrev">Evol. Applications</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BApplications%26rft.volume%253D3%26rft.spage%253D244%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2009.00104.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2009.00104.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.35" data-doi="10.1534/genetics.111.132233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">England</span>  <span class="cit-name-given-names">P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Estimating contemporary effective population size on the basis of linkage disequilibrium in the face of migration</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132233%26rft_id%253Dinfo%253Apmid%252F21840864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNjMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-3.4.709.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>, p. <span class="cit-fpage">137</span> in <span class="cit-source">Genetic Data Analysis II</span>. <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">Evolution in mendelian populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D16%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.709.38" data-doi="10.1093/jhered/esn061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhdanova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pudovkin</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Nb_HetEx: a program to estimate the effective number of breeders</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">694</span>–<span class="cit-lpage">695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesn061%26rft_id%253Dinfo%253Apmid%252F18703539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk5LzYvNjk0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzcwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/4/709.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/695" title="Transcriptome Profiling Following Neuronal and Glial Expression of ALS-Linked SOD1 in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/719" title="Involvement of MLPK Pathway in Intraspecies Unilateral Incompatibility Regulated by a Single Locus With Stigma and Pollen Factors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/4"><span>Volume 3 Issue 4, April 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/4.cover-source.jpg?itok=tXw4AXt4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/709.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jniZCY-pUgEh6Agimr333Jwv3ltC9xRfqsVytPtdbSA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/4/709" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/4/709" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/4/709" class="active" data-icon-position="" data-hide-link-title="0">Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (Scomberomorus commerson)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459447" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rl_mVIF6S_exTf26OPnwzbdTXjmnqrE-2i1JfCbWyYg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/709.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/4/709" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nJW-awiEP2WITqKb44Ty9HVOXMIjA4MIbPnowTB0mz8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/4/709" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/709.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459447" id="node459447--2" data-pisa="ggg;3/4/709" data-pisa-master="ggg;g3.112.005124" data-apath="/ggg/3/4/709.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (<em>Scomberomorus commerson</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gilbert Michael Macbeth</span>, <span class="highwire-citation-author" data-delta="1">Damien Broderick</span>, <span class="highwire-citation-author" data-delta="2">Rik C. Buckworth</span> and <span class="highwire-citation-author" data-delta="3">Jennifer R. Ovenden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">709-717;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005124 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459447--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gilbert Michael Macbeth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilbert%2BMichael%2BMacbeth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macbeth%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGilbert%2BMichael%2BMacbeth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gilbert.macbeth1@uq.net.au" class="" data-icon-position="" data-hide-link-title="0">gilbert.macbeth1@uq.net.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Damien Broderick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damien%2BBroderick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broderick%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADamien%2BBroderick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rik C. Buckworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Marine & Atmospheric Research, Wealth from Oceans National Research Flagship, Brisbane, Queensland, 4001, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rik%2BC.%2BBuckworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckworth%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARik%2BC.%2BBuckworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer R. Ovenden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BR.%2BOvenden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ovenden%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BR.%2BOvenden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/4/709" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/4/709&amp;title=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/4/709&amp;title=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/4/709&amp;title=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/4/709&amp;text=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/4/709&amp;title=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/4/709&amp;t=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/4/709&amp;title=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/4/709&amp;title=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459447/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459447" id="node459447--3" data-pisa="ggg;3/4/709" data-pisa-master="ggg;g3.112.005124" data-apath="/ggg/3/4/709.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium Estimation of Effective Population Size with Immigrants from Divergent Populations: A Case Study on Spanish Mackerel (<em>Scomberomorus commerson</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Gilbert Michael Macbeth</span>, <span class="highwire-citation-author" data-delta="1">Damien Broderick</span>, <span class="highwire-citation-author" data-delta="2">Rik C. Buckworth</span> and <span class="highwire-citation-author" data-delta="3">Jennifer R. Ovenden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">709-717;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.005124 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459447--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gilbert Michael Macbeth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilbert%2BMichael%2BMacbeth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macbeth%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGilbert%2BMichael%2BMacbeth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gilbert.macbeth1@uq.net.au" class="" data-icon-position="" data-hide-link-title="0">gilbert.macbeth1@uq.net.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Damien Broderick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damien%2BBroderick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Broderick%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADamien%2BBroderick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rik C. Buckworth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Marine & Atmospheric Research, Wealth from Oceans National Research Flagship, Brisbane, Queensland, 4001, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rik%2BC.%2BBuckworth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckworth%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARik%2BC.%2BBuckworth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer R. Ovenden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular Fisheries Laboratory, Queensland Government, St Lucia, Queensland, 4072, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BR.%2BOvenden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ovenden%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BR.%2BOvenden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=709&amp;rft.epage=717&amp;rft.atitle=Linkage%20Disequilibrium%20Estimation%20of%20Effective%20Population%20Size%20with%20Immigrants%20from%20Divergent%20Populations%3A%20A%20Case%20Study%20on%20Spanish%20Mackerel%20%28Scomberomorus%20commerson%29&amp;rft.volume=3&amp;rft.issue=4&amp;rft.date=2013-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Macbeth%2C+Gilbert+Michael&amp;rft.au=Broderick%2C+Damien&amp;rft.au=Buckworth%2C+Rik+C.&amp;rft.au=Ovenden%2C+Jennifer+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459447/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459447/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459447/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459447/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459447/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459447/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459447/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459447/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459447/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459447/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459447/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459447/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/4/709" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/4/709.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/4/709.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/4/709.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/4/709.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
