<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/4/617" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/4/617.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/4/617.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/4/617.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/4/617/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/4/617.atom" />
<meta name="HW.pisa" content="ggg;3/4/617" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce" />
<meta name="DC.Identifier" content="10.1534/g3.112.004929" />
<meta name="DC.Date" content="2013-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Truco et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We have generated an ultra-high-density genetic map for lettuce, an economically important member of the Compositae, consisting of 12,842 unigenes (13,943 markers) mapped in 3696 genetic bins distributed over nine chromosomal linkage groups. Genomic DNA was hybridized to a custom Affymetrix oligonucleotide array containing 6.4 million features representing 35,628 unigenes of Lactuca spp. Segregation of single-position polymorphisms was analyzed using 213 F7:8 recombinant inbred lines that had been generated by crossing cultivated Lactuca sativa cv. Salinas and L. serriola acc. US96UC23, the wild progenitor species of L. sativa . The high level of replication of each allele in the recombinant inbred lines was exploited to identify single-position polymorphisms that were assigned to parental haplotypes. Marker information has been made available using GBrowse to facilitate access to the map. This map has been anchored to the previously published integrated map of lettuce providing candidate genes for multiple phenotypes. The high density of markers achieved in this ultradense map allowed syntenic studies between lettuce and Vitis vinifera as well as other plant species." />
<meta name="DC.Contributor" content="Maria José Truco" />
<meta name="DC.Contributor" content="Hamid Ashrafi" />
<meta name="DC.Contributor" content="Alexander Kozik" />
<meta name="DC.Contributor" content="Hans van Leeuwen" />
<meta name="DC.Contributor" content="John Bowers" />
<meta name="DC.Contributor" content="Sebastian Reyes Chin Wo" />
<meta name="DC.Contributor" content="Kevin Stoffel" />
<meta name="DC.Contributor" content="Huaqin Xu" />
<meta name="DC.Contributor" content="Theresa Hill" />
<meta name="DC.Contributor" content="Allen Van Deynze" />
<meta name="DC.Contributor" content="Richard W. Michelmore" />
<meta name="article:published_time" content="2013-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We have generated an ultra-high-density genetic map for lettuce, an economically important member of the Compositae, consisting of 12,842 unigenes (13,943 markers) mapped in 3696 genetic bins distributed over nine chromosomal linkage groups. Genomic DNA was hybridized to a custom Affymetrix oligonucleotide array containing 6.4 million features representing 35,628 unigenes of &lt;i&gt;Lactuca&lt;/i&gt; spp. Segregation of single-position polymorphisms was analyzed using 213 F&lt;sub&gt;7:8&lt;/sub&gt; recombinant inbred lines that had been generated by crossing cultivated &lt;i&gt;Lactuca sativa&lt;/i&gt; cv. Salinas and &lt;i&gt;L. serriola&lt;/i&gt; acc. US96UC23, the wild progenitor species of &lt;i&gt;L. sativa&lt;/i&gt;. The high level of replication of each allele in the recombinant inbred lines was exploited to identify single-position polymorphisms that were assigned to parental haplotypes. Marker information has been made available using GBrowse to facilitate access to the map. This map has been anchored to the previously published integrated map of lettuce providing candidate genes for multiple phenotypes. The high density of markers achieved in this ultradense map allowed syntenic studies between lettuce and &lt;i&gt;Vitis vinifera&lt;/i&gt; as well as other plant species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/04/01" />
<meta name="citation_mjid" content="ggg;3/4/617" />
<meta name="citation_id" content="3/4/617" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/4/617" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/4/617.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/4/617.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/4/617.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.004929" />
<meta name="citation_pmid" content="23550116" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="617" />
<meta name="citation_lastpage" content="631" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Maria José Truco" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Hamid Ashrafi" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Alexander Kozik" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Hans van Leeuwen" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="John Bowers" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Sebastian Reyes Chin Wo" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Kevin Stoffel" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Huaqin Xu" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Theresa Hill" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author" content="Allen Van Deynze" />
<meta name="citation_author_institution" content="Seed Biotechnology Center, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Richard W. Michelmore" />
<meta name="citation_author_institution" content="The Genome Center, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="rwmichelmore@ucdavis.edu" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Argyris;citation_author=M. J. Truco;citation_author=O. Ochoa;citation_author=S. J. Knapp;citation_author=D. W. Still;citation_title=Quantitative trait loci associated with seed and seedling traits in Lactuca.;citation_pages=1365-1376;citation_volume=111;citation_year=2005;citation_issue=7;citation_pmid=16177902;citation_doi=10.1007/s00122-005-0066-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. M. Argyris;citation_author=P. Dahal;citation_author=E. Hayashi;citation_author=D. W. Still;citation_author=K. J. Bradford;citation_title=Genetic variation for lettuce seed thermoinhibition is associated with temperature-sensitive expression of abscisic acid, gibberellins, and ethylene biosynthesis, metabolism, and response genes;citation_volume=148;citation_year=2008;citation_pmid=18753282;citation_doi=10.1104/pp.108.125807" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Argyris;citation_author=M. J. Truco;citation_author=O. Ochoa;citation_author=L. McHale;citation_author=P. Dahal;citation_title=A gene encoding an abscisic acid biosynthetic enzyme (LsNCED4) colocates with the high temperature germination locus Htg6.1 in lettuce (Lactuca sp.);citation_pages=95-108;citation_volume=122;citation_year=2010;citation_pmid=20703871" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=T. W. Banks;citation_author=M. C. Jordan;citation_author=D. J. Somers;citation_title=Single-feature polymorphism mapping in bread wheat;citation_pages=167-178;citation_volume=2;citation_year=2009;citation_doi=10.3835/plantgenome2009.02.0009" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. S. Barker;citation_author=N. C. Kane;citation_author=M. Matvienko;citation_author=A. Kozik;citation_author=R. W. Michelmore;citation_title=Multiple paleopolyploidizations during the evolution of the Compositae reveal parallel patterns of duplicate gene retention after millions of years;citation_volume=25;citation_year=2008;citation_pmid=18728074;citation_doi=10.1093/molbev/msn187" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. N. Bernardo;citation_author=P. J. Bradbury;citation_author=H. Ma;citation_author=S. Hu;citation_author=R. L. Bowden;citation_title=Discovery and mapping of single feature polymorphisms in wheat using Affymetrix arrays.;citation_pages=251-251;citation_volume=10;citation_year=2009;citation_pmid=19480702;citation_doi=10.1186/1471-2164-10-251" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. O. Borevitz;citation_author=D. Liang;citation_author=D. Plouffe;citation_author=H. S. Chang;citation_author=T. Zhu;citation_title=Large-scale identification of single-feature polymorphisms in complex genomes;citation_volume=13;citation_year=2003;citation_pmid=12618383;citation_doi=10.1101/gr.541303" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. O. Borevitz;citation_author=S. P. Hazen;citation_author=T. P. Michael;citation_author=G. P. Morris;citation_author=I. R. Baxter;citation_title=Genome-wide patterns of single-feature polymorphism in Arabidopsis thaliana;citation_volume=104;citation_year=2007;citation_pmid=17626786;citation_doi=10.1073/pnas.0705323104" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. E. Bowers;citation_author=E. Bachlava;citation_author=R. L. Brunick;citation_author=L. H. Rieseberg;citation_author=S. J. Knapp;citation_title=Development of a 10,000 Locus Genetic Map of the Sunflower Genome Based on Multiple Crosses;citation_pages=721-729;citation_volume=2;citation_year=2012;citation_issue=7;citation_pmid=22870395;citation_doi=10.1534/g3.112.002659" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. A. Cartwright;citation_author=M. Troggio;citation_author=R. Velasco;citation_author=A. Gutin;citation_title=Genetic mapping in the presence of genotyping errors;citation_volume=176;citation_year=2007;citation_pmid=17277374;citation_doi=10.1534/genetics.106.063982" />
<meta name="citation_reference" content="citation_journal_title=Asp. Appl. Biol.;citation_author=I. R. Crute;citation_title=The impact of breeding on pest and disease control in lettuce;citation_pages=305-312;citation_volume=17;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=I. M. de Vries;citation_title=Origin and domestication of Lactuca sativa L;citation_pages=165-174;citation_volume=44;citation_year=1997;citation_doi=10.1023/A:1008611200727" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. R. Drost;citation_author=E. Novaes;citation_author=C. Boaventura-Novaes;citation_author=C. I. Benedict;citation_author=R. S. Brown;citation_title=A microarray-based genotyping and genetic mapping approach for highly heterozygous outcrossing species enables localization of a large fraction of the unassembled Populus trichocarpa genome sequence.;citation_pages=1054-1067;citation_volume=58;citation_year=2009;citation_issue=6;citation_pmid=19220791;citation_doi=10.1111/j.1365-313X.2009.03828.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Methods;citation_author=J. D. Edwards;citation_author=J. Janda;citation_author=M. T. Sweeney;citation_author=A. B. Gaikwad;citation_author=B. Liu;citation_title=Development and evaluation of a high-throughput, low-cost genotyping platform based on oligonucleotide microarrays in rice.;citation_pages=13-13;citation_volume=4;citation_year=2008;citation_pmid=18510771;citation_doi=10.1186/1746-4811-4-13" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_author=E. S. Buckler;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. H. Fu;citation_author=Z. W. Zheng;citation_author=H. K. Dooner;citation_title=Recombination rates between adjacent genic and retrotransposon regions in maize vary by 2 orders of magnitude;citation_volume=99;citation_year=2002;citation_pmid=11792865;citation_doi=10.1073/pnas.022635499" />
<meta name="citation_reference" content="citation_journal_abbrev=Evol Appl;citation_author=Y. Hartman;citation_author=D. A. P. Hooftman;citation_author=B. Uwimana;citation_author=C. C. M. van de Wiel;citation_author=M. J. M. Smulders;citation_title=Genomic regions in crop-wild hybrids of lettuce are affected differently in different environments: implications for crop breeding.;citation_pages=629-640;citation_volume=5;citation_year=2012;citation_issue=6;citation_pmid=23028403;citation_doi=10.1111/j.1752-4571.2012.00240.x" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=E. Hayashi;citation_author=N. Aoyama;citation_author=D. W. Still;citation_title=Quantitative trait loci associated with lettuce seed germination under different temperature and light environments.;citation_pages=928-947;citation_volume=51;citation_year=2008;citation_issue=11;citation_pmid=18956026;citation_doi=10.1139/G08-077" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=T. Hill;citation_author=H. Ashrafi;citation_author=S. Reyes-Chin-Wo;citation_author=J. Yao;citation_author=K. Stoffel;citation_title=Genome-wide analysis of Capsicum annuum germplasm reveals marked changes in diversity from pungent to non-pungent pepper;citation_pages=e56200;citation_volume=8;citation_year=2013;citation_pmid=23409153;citation_doi=10.1371/journal.pone.0056200" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. T. Hu;citation_author=P. Pattyn;citation_author=E. G. Bakker;citation_author=J. Cao;citation_author=J. F. Cheng;citation_title=The Arabidopsis lyrata genome sequence and the basis of rapid genome size change.;citation_pages=476-481;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478890;citation_doi=10.1038/ng.807" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_journal_abbrev=Biostatistics;citation_author=R. A. Irizarry;citation_author=B. Hobbs;citation_author=F. Collin;citation_author=Y. D. Beazer-Barclay;citation_author=K. J. Antonellis;citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data;citation_pages=249-264;citation_volume=4;citation_year=2003;citation_issue=2;citation_pmid=12925520;citation_doi=10.1093/biostatistics/4.2.249" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=S. N. Isobe;citation_author=H. Hisano;citation_author=S. Sato;citation_author=H. Hirakawa;citation_author=K. Okumura;citation_title=Comparative Genetic Mapping and Discovery of Linkage Disequilibrium Across Linkage Groups in White Clover (Trifolium repens L.);citation_pages=607-617;citation_volume=2;citation_year=2012;citation_issue=5;citation_pmid=22670230;citation_doi=10.1534/g3.112.002600" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=O. Jaillon;citation_author=J.-M. Aury;citation_author=B. Noel;citation_author=A. Policriti;citation_author=C. Clepet;citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla.;citation_pages=463-467;citation_volume=449;citation_year=2007;citation_issue=7161;citation_pmid=17721507;citation_doi=10.1038/nature06148" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Jeuken;citation_author=R. van Wijk;citation_author=J. Peleman;citation_author=P. Linhout;citation_title=An integrated interspecific AFLP map of lettuce (Lactuca) based on two L. sativa x L. saligna F2 populations;citation_pages=638-647;citation_volume=103;citation_year=2001;citation_doi=10.1007/s001220100657" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. C. Johnson;citation_author=L. E. Jackson;citation_author=O. Ochoa;citation_author=R. van Wijk;citation_author=J. Peleman;citation_title=Lettuce, a shallow-rooted crop, and Lactuca serriola, its wild progenitor, differ at QTL determining root architecture and deep soil water exploitation;citation_pages=1066-1073;citation_volume=101;citation_year=2000;citation_doi=10.1007/s001220051581" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=R. V. Kesseli;citation_author=O. Ochoa;citation_author=R. W. Michelmore;citation_title=Variation at RFLP loci in Lactuca spp. and the origin of cultivated lettuce (L. sativa);citation_pages=430-436;citation_volume=34;citation_year=1991;citation_doi=10.1139/g91-065" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. V. Kesseli;citation_author=I. Paran;citation_author=R. W. Michelmore;citation_title=Analysis of a detailed genetic linkage map of Lactuca sativa (lettuce) constructed from RFLP and RAPD markers.;citation_pages=1435-1446;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=7912217" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. S. Landry;citation_author=R. V. Kesseli;citation_author=B. Farrara;citation_author=R. W. Michelmore;citation_title=A Genetic Map of Lettuce (Lactuca sativa L.) With Restriction Fragment Length Polymorphism, Isozyme, Disease Resistance and Morphological Markers;citation_pages=331-337;citation_volume=116;citation_year=1987;citation_issue=2;citation_pmid=17246387" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=H. Li;citation_author=A. Kilian;citation_author=M. Zhou;citation_author=P. Wenzl;citation_author=E. Huttner;citation_title=Construction of a high-density composite map and comparative mapping of segregation distortion regions in barley.;citation_pages=319-331;citation_volume=284;citation_year=2010;citation_issue=5;citation_pmid=20803217;citation_doi=10.1007/s00438-010-0570-3" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_author=K. Lindqvist;citation_title=Inheritance studies in lettuce;citation_pages=387-470;citation_volume=46;citation_year=1960" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Lysak;citation_author=A. Berr;citation_author=A. Pecinka;citation_author=R. Schmidt;citation_author=K. McBreen;citation_title=Mechanisms of chromosome number reduction in Arabdiopsis thaliana and related Brassicaceae species;citation_volume=103;citation_year=2006;citation_pmid=16549785;citation_doi=10.1073/pnas.0510791103" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. F. MacDaniel;citation_author=W. J. H. Shaw;citation_title=A linkage map reveals a complex basis for segregation distortion in an interpopulation cross in the moss Ceratodon pupureus;citation_volume=176;citation_year=2007;citation_pmid=17603096;citation_doi=10.1534/genetics.107.075424" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=L. K. McHale;citation_author=M. J. Truco;citation_author=A. Kozik;citation_author=T. Wroblewski;citation_author=O. E. Ochoa;citation_title=The genomic architecture of disease resistance in lettuce.;citation_pages=565-580;citation_volume=118;citation_year=2009;citation_issue=3;citation_pmid=19005638;citation_doi=10.1007/s00122-008-0921-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. W. Michelmore;citation_author=B. C. Myers;citation_title=Clusters of Resistance Genes in Plants Evolve by Divergent Selection and a Birth-and-Death Process;citation_pages=1113-1130;citation_volume=8;citation_year=1998;citation_issue=11;citation_pmid=9847076" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. W. Michelmore;citation_author=I. Paran;citation_author=R. V. Kesseli;citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.;citation_pages=9828-9832;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1682921;citation_doi=10.1073/pnas.88.21.9828" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L. C. Moyle;citation_author=E. B. Graham;citation_title=Genome-wide associations between hybrid sterility QTL and marker transmission ratio distortion;citation_volume=23;citation_year=2006;citation_pmid=16495346;citation_doi=10.1093/molbev/msj112" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. G. Neves;citation_author=E. M. C. Mamani;citation_author=A. C. Alfenas;citation_author=M. Kirst;citation_author=D. Grattapaglia;citation_title=A high-density transcript linkage map with 1,845 expressed genes positioned by microarray-based Single Feature Polymorphisms (SFP) in Eucalyptus.;citation_pages=189-189;citation_volume=12;citation_year=2011;citation_pmid=21492453;citation_doi=10.1186/1471-2164-12-189" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Rebollo;citation_author=M. T. Romanish;citation_author=D. L. Mager;citation_title=Transposable elements: an abundant and natural source of regulatory sequences for host genes.;citation_pages=21-42;citation_volume=46;citation_year=2012;citation_pmid=22905872;citation_doi=10.1146/annurev-genet-110711-155621" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=P. A. Salomé;citation_author=K. Bomblies;citation_author=J. Fitz;citation_author=R. A. E. Laitinen;citation_author=N. Warthmann;citation_title=The recombination landscape in Arabidopsis thaliana F2 populations;citation_pages=447-455;citation_volume=108;citation_year=2011;citation_pmid=22072068" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. Singer;citation_author=Y. Fan;citation_author=H. S. Chang;citation_author=T. Zhu;citation_author=S. P. Hazen;citation_title=A high-resolution map of Arabidopsis recombinant inbred lines by whole-genome exon array hybridization.;citation_pages=e144-e144;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=17044735;citation_doi=10.1371/journal.pgen.0020144" />
<meta name="citation_reference" content="citation_journal_title=Functional &amp; integrative genomics;citation_journal_abbrev=Funct Integr Genomics;citation_author=P. Sourdille;citation_author=S. Singh;citation_author=T. Cadalen;citation_author=G. L. Brown-Guedira;citation_author=G. Gay;citation_title=Microsatellite-based deletion bin system for the establishment of genetic-physical map relationships in wheat (Triticum aestivum L.).;citation_pages=12-25;citation_volume=4;citation_year=2004;citation_issue=1;citation_pmid=15004738;citation_doi=10.1007/s10142-004-0106-1" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=K. Stoffel;citation_author=H. van Leeuwen;citation_author=A. Kozik;citation_author=D. Caldwell;citation_author=H. Ashrafi;citation_title=Development and application of a 6.5 million feature Affymetrix GenechipÂ® for massively parallel discovery of single position polymorphisms in lettuce (Lactuca spp.).;citation_pages=185-185;citation_volume=13;citation_year=2012;citation_pmid=22583801;citation_doi=10.1186/1471-2164-13-185" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Z. Sun;citation_author=Z. Wang;citation_author=J. Tu;citation_author=J. Zhang;citation_author=F. Yu;citation_title=An ultradense genetic recombination map for Brassica napus, consisting of 13551 SRAP markers.;citation_pages=1305-1317;citation_volume=114;citation_year=2007;citation_issue=8;citation_pmid=17426959;citation_doi=10.1007/s00122-006-0483-z" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. H. Syed;citation_author=A. P. Sorensen;citation_author=R. Antonise;citation_author=C. van de Wiel;citation_author=C. G. van der Linden;citation_title=A detailed linkage map of lettuce based on SSAP, AFLP and NBS markers.;citation_pages=517-527;citation_volume=112;citation_year=2006;citation_issue=3;citation_pmid=16341837;citation_doi=10.1007/s00122-005-0155-4" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. D. Tanksley;citation_author=M. W. Ganal;citation_author=G. B. Martin;citation_title=Chromosome landing: a paradigm for map-based gene cloning in plants with large genomes.;citation_pages=63-68;citation_volume=11;citation_year=1995;citation_issue=2;citation_pmid=7716809;citation_doi=10.1016/S0168-9525(00)88999-4" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution;citation_pages=635-641;citation_volume=31;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Troggio;citation_author=G. Malacarne;citation_author=G. Coppola;citation_author=C. Segala;citation_author=D. A. Cartwright;citation_title=A dense single-nucleotide polymorphism-based genetic linkage map of grapevine (Vitis vinifera L.) anchoring Pinot Noir bacterial artificial chromosome contigs;citation_volume=176;citation_year=2007;citation_pmid=17603124;citation_doi=10.1534/genetics.106.067462" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. J. Truco;citation_author=R. Antonise;citation_author=D. Lavelle;citation_author=O. Ochoa;citation_author=A. Kozik;citation_title=A high-density, integrated genetic linkage map of lettuce (Lactuca spp.).;citation_pages=735-746;citation_volume=115;citation_year=2007;citation_issue=6;citation_pmid=17828385;citation_doi=10.1007/s00122-007-0599-9" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. T. Truong;citation_author=A. M. Ramos;citation_author=F. Yalcin;citation_author=M. de Ruiter;citation_author=H. J. A. van der Poel;citation_title=Sequence-based genotyping for marker discovery and co-dominant scoring in germplasm and populations.;citation_pages=e37565-e37565;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22662172;citation_doi=10.1371/journal.pone.0037565" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. Uwimana;citation_author=M. J. M. Smulders;citation_author=D. A. P. Hooftman;citation_author=Y. Hartman;citation_author=P. H. van Tienderen;citation_title=Hybridization between crops and wild relatives: the contribution of cultivated lettuce to the vigour of crop-wild hybrids under drought, salinity and nutrient deficiency conditions.;citation_pages=1097-1111;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22660630;citation_doi=10.1007/s00122-012-1897-4" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=R. Van Berloo;citation_title=Computer note: GGT 2.0: versatile software for visualization and analysis of genetic data;citation_volume=99;citation_year=2008;citation_pmid=18222930;citation_doi=10.1093/jhered/esm109" />
<meta name="citation_reference" content="citation_title=Software for the Calculation of Genetic Linkage Maps;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. van Os;citation_author=P. Stam;citation_author=R. G. F. Visser;citation_author=H. J. van Eck;citation_title=RECORD: a novel method for ordering loci on a genetic linkage map.;citation_pages=30-40;citation_volume=112;citation_year=2005;citation_issue=1;citation_pmid=16228189;citation_doi=10.1007/s00122-005-0097-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. van Os;citation_author=S. Andrzejewski;citation_author=E. Bakker;citation_author=I. Barrena;citation_author=G. J. Bryan;citation_title=Construction of a 10,000-marker ultradense genetic recombination map of potato: Providing a framework for accelerated gene isolation and a genomewide physical map;citation_volume=173;citation_year=2006;citation_pmid=16582432;citation_doi=10.1534/genetics.106.055871" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=M. J. Wade;citation_author=R. G. Winther;citation_author=A. F. Agrawal;citation_author=C. J. Goodnight;citation_title=Alternative definitions of epistasis: dependence and interaction;citation_pages=498-504;citation_volume=16;citation_year=2001;citation_doi=10.1016/S0169-5347(01)02213-3" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=G. Wang;citation_author=J. P. Xu;citation_author=Y. P. Tang;citation_author=L. L. Zhou;citation_author=F. Wang;citation_title=Molecular characterization of a genomic interval with highly uneven recombination distribution on maize chromosome 10. L;citation_pages=1109-1118;citation_volume=139;citation_year=2011;citation_pmid=22057628;citation_doi=10.1007/s10709-011-9613-x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=W. Waycott;citation_author=S. B. Fort;citation_author=E. J. Ryder;citation_author=R. W. Michelmore;citation_title=Mapping morphological genes relative to molecular markers in lettuce (Lactuca sativa L.);citation_pages=245-251;citation_volume=82;citation_year=1999;citation_pmid=10336698;citation_doi=10.1038/sj.hdy.6884730" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. D. Werner;citation_author=J. O. Borevitz;citation_author=N. H. Uhlenhaut;citation_author=J. R. Ecker;citation_author=J. Chory;citation_title=FRIGIDA-independent variation in flowering time of natural Arabidiopsis thaliana accessions;citation_volume=170;citation_year=2005;citation_pmid=15911588;citation_doi=10.1534/genetics.104.036533" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. L. West;citation_author=H. van Leeuwen;citation_author=A. Kozik;citation_author=D. J. Kliebenstein;citation_author=R. W. Doerge;citation_title=High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis;citation_volume=16;citation_year=2006;citation_pmid=16687733;citation_doi=10.1101/gr.5105206" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Wu;citation_author=J. N. Jenkins;citation_author=J. C. McCarty;citation_author=X.-Y. Lou;citation_title=Comparisons of four approximation algorithms for large-scale linkage map construction.;citation_pages=649-655;citation_volume=123;citation_year=2011;citation_issue=4;citation_pmid=21611760;citation_doi=10.1007/s00122-011-1614-8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Xie;citation_author=Q. Feng;citation_author=H. Yua;citation_author=X. Huang;citation_author=Q. Zhao;citation_title=Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing;citation_volume=107;citation_year=2010;citation_pmid=20498060;citation_doi=10.1073/pnas.1005931107" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=K. Yogeeswaran;citation_author=A. Frary;citation_author=T. L. York;citation_author=A. Amenta;citation_author=A. H. Lesser;citation_title=Comparative genome analyses of Arabidopsis spp.: Inferring chromosomal rearrangement events in the evolutionary history of A. thaliana;citation_pages=505-515;citation_volume=15;citation_year=2005;citation_issue=4;citation_pmid=15805492;citation_doi=10.1101/gr.3436305" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=F. Z. Zhang;citation_author=C. Wagstaff;citation_author=A. M. Rae;citation_author=A. K. Sihota;citation_author=C. W. Keevil;citation_title=QTLs for shelf life in lettuce co-locate with those for leaf biophysical properties but not with those for leaf developmental traits;citation_volume=58;citation_year=2007;citation_pmid=17347132;citation_doi=10.1093/jxb/erm006" />
<meta name="citation_reference" content="citation_journal_title=Nat. Comm.;citation_author=K. Zhao;citation_author=C. W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa;citation_pages=467;citation_volume=2;citation_year=2011;citation_doi=10.1038/ncomms1467" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
<meta name="twitter:description" content="We have generated an ultra-high-density genetic map for lettuce, an economically important member of the Compositae, consisting of 12,842 unigenes (13,943 markers) mapped in 3696 genetic bins distributed over nine chromosomal linkage groups. Genomic DNA was hybridized to a custom Affymetrix oligonucleotide array containing 6.4 million features representing 35,628 unigenes of Lactuca spp. Segregation of single-position polymorphisms was analyzed using 213 F7:8 recombinant inbred lines that had been generated by crossing cultivated Lactuca sativa cv. Salinas and L. serriola acc. US96UC23, the wild progenitor species of L. sativa . The high level of replication of each allele in the recombinant inbred lines was exploited to identify single-position polymorphisms that were assigned to parental haplotypes. Marker information has been made available using GBrowse to facilitate access to the map. This map has been anchored to the previously published integrated map of lettuce providing candidate genes for multiple phenotypes. The high density of markers achieved in this ultradense map allowed syntenic studies between lettuce and Vitis vinifera as well as other plant species." />
<meta name="og-title" property="og:title" content="An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/4/617" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We have generated an ultra-high-density genetic map for lettuce, an economically important member of the Compositae, consisting of 12,842 unigenes (13,943 markers) mapped in 3696 genetic bins distributed over nine chromosomal linkage groups. Genomic DNA was hybridized to a custom Affymetrix oligonucleotide array containing 6.4 million features representing 35,628 unigenes of Lactuca spp. Segregation of single-position polymorphisms was analyzed using 213 F7:8 recombinant inbred lines that had been generated by crossing cultivated Lactuca sativa cv. Salinas and L. serriola acc. US96UC23, the wild progenitor species of L. sativa . The high level of replication of each allele in the recombinant inbred lines was exploited to identify single-position polymorphisms that were assigned to parental haplotypes. Marker information has been made available using GBrowse to facilitate access to the map. This map has been anchored to the previously published integrated map of lettuce providing candidate genes for multiple phenotypes. The high density of markers achieved in this ultradense map allowed syntenic studies between lettuce and Vitis vinifera as well as other plant species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4.cover-source.jpg" />
    <title>An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eA4pyEH1Lyo8i7Gp4OTFO6cQ8Hs67cBKfwoB_T01RKg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459455","apath":"\/ggg\/3\/4\/617.atom","pisa":"ggg;3\/4\/617","ac":{"\/ggg\/3\/4\/617.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/4\/617.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/4\/617"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/4\/617"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/4\/617":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AldpgVl2gEvBqUwyg8FnFYxySfipGxQ8pJRBGYCIN7Pa2gP+v3os\\\\\\\/lunxVEByPYe3hxYkBbNVnuPv9cQQYPTRcmFdBylEwBVn7rxUrA\\\\\\\/\\\\\\\/oMQcRPfjQ5zJgKD8F4aTEChB2HTu5eo+wFqY7CL14wy4BB7+NNMSXbtHhOfBLpnWPqsA\\\\\\\/5CWcyeCFv5\\\\\\\/K2LVWwLr+CnE1ez9CTYMAI7JMR5xjXC\\\\\\\/Vh9g2LuX70+G\\\\\\\/h8+gIDdXOF6qzkDwmxtr\\\\\\\/J9JaUFfOfxQ7wcykyWWJLbkd9TNVaYb+oATtgvoTbaagM9d1f+VkXDpX7dbfd\\\\\\\/ktcv6p+klKYad+RtrkprCJK2xGI7ja71HkXHOa2ZA\\\\\\\/9OKaM8Mz8N+fVq0KgjNUWGtVRbNDkrAoE3Bqy1QfjWfuR6KesWiVHiJo7RvPMw6wHDH0RhES5l87OgfwAwqHD94dCP05DvmhH6mpt9NKbQQppm+EAZijq8v2NJltZ1ilWkFTpSJ2Is2BKhs5DtXRDUGsCo3U1CEyah8m+SUvmotOf8YewejYhuyDH60inmnItpTnVfuc+d9QXsieaoizUcuQR0r6y8r1+wQuS46bShp\\\\\\\/I+dt0TRsG144EY+Tngh6om6ag8wGtEXET3TgagLK3HgYdisVFC5LBbM0BzZS34AU1GIHQgKrA4KbgKXkL39I8t847kl2P6A1PnOhba8hFNMDPkBpBEn8ojXIiJzK6bpKc\\\\\\\/TiNxDq+bPwFGs+NPQ0hz6gGrtbE5YBbLeHHzvhZ\\\\\\\/+ccvDByka0QbM141KjxhhUR4jVcGtXcD2vMlQX7jPwcX\\\\\\\/lSzh3+mnLP3B91R4UebdKcrrf906CeLScd9jix2IbB78Itquhxcv8Myi+ZSj0rQjq5Ge3byzs0ySPGqtHSs9OzFc7Bizm898vnJ8FkIVYMuCluGtdDwnbaki981hWat9AKdm7ScVA+plMt0GRNF+vcMbSCGkH0qMRYokby4zyifHI51Z9tkMDZw0OSSExbD1yzEDJpUCK7u4hW0pL9BMn373xQnHF1TqRWghEOeeAk0lGel8PDcn4vqWBNzJssxfKnkbvZLrogz9dFz\\\\\\\/ihxEzfoXs\\\\\\\/f4JrEtajaY4mSZ1WP8E7Gi4PcY8MgT+gsQI5cd65p9Z6EPEWeHJs3WiheYdQF6PkseBBatsZbTtcga8t0mlxS7SZZ53fnxzJ6ON5yAGd7DDaav\\\\\\\/0rHuc\\\\\\\/ex0c9UpgJTvu+Fqh5aKrqvvdhZvItiW7n6yYLu7LqG13StBDaupxJuwdyAzTsAJc9OR8oQg+8Vb\\\u0022,\\\u0022iv\\\u0022:\\\u0022zuj2IJjy\\\\\\\/D\\\\\\\/BBoUK1LJjsQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f0a03779cbfe2a7836dc3a88935605f9b94c03e566002260fa495133ac9fa5c1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+m\\\\\\\/wftsyys3EgJVPFC9FfDqV5myd4NSQJDFeADDyBndvk+61G6LkuuQJ1dyZoXwCSqAp4MI6kd9WEmUNhNzQmabDZCYWHNS8wSQdALOyaGVZm0QRazX5iSP0Etubkn2NMYV5Ycf6O5yIDcyg8IB0ex6xGr5TjVRmmAwP204wE4sHkmMU1aHOIV7m2lHSBhs\\\\\\\/JhttNl4+cFx+4C3boiRYBW2V9fBB65lUfGPeW8+FWv3H\\\\\\\/eJcmmZVMzxGIIF7ha7iEgoAlD4EskWXC3mKhtfdcTkgHzaFRWDudL4XyRi9JUs4febb2yapI2Szch9UXqhynw3NxIz+jsM7wc9hzOYGfyYNHqIlKy2PFr+4MUJFCHyIN0IhhE0MDf0lNUtrnzI8alAVPti0gRSK9KdhxKUrasrmUXUxBExTPPILlMuXuiP9abhfVSzy8DrW0xpDvhmxXRsxf6s0fbQwYocUlDm7vkPH+s6BaWlFCp44UEwSXufAWOme+4PP4mQHZIJemdcVwnqwBDxIyXPUogd4P7deV2pGpO273ysGDgGJ8OHmBKUXyZtRV1FHR3VByrNgfOtdqj2X2B6Hit09sTxklT9uu3RncCKaCMhuLOalbJlNQcVKRqcUbR9cnqPKKg2oVa+T9IHhpOy4VXdycnE3iW5FOsd05CsuRalcwUlxezmp9MIOzpbq0UYlrmZfCgXBpBrc8Nadro0amI917nlVAhSh4+Ld6SNyXrClBX8XKlFJshVt9J4bBLFw3ReB1xj+GCNYbuCnUHP7bM19yYqP4+CfSN+LPPzcr0HNHOEHYCK1OtW+assSEBl+zVkzJdMtyz1jbZuweR3MdtEYzFjM0PedJgVTSbiDyy3jx+Xl0RDnn7cUMLVnbahxlJMzwjB\\\\\\\/gNUudsxPSxMrfBD3gk4mzxnDocVl4cfCXPwYdKAKsy3fRkXb89mDdSbT1Ax\\\\\\\/czDob6Mu9lEL+X+dUG+CUlJjcFBlvv3MvZAA5A04J5L7QIPMJY8b1LFLcickypX\\\\\\\/J\\\\\\\/V1YYYCjcfEkoHy4ulD2uZppxD2GyTF0lybdmttpaXiMt4zTWgU\\\\\\\/hGmFL3tX8Njq6bcqYDYHzcMAKacLIWZauBlxnhNfdNNgyXUmv8sSFmErGotxUthScn0h4vFy0O45qSM3Kg58LniTIRsY3GTenD\\\\\\\/Owd0xscSMEQCBFMq85kz6GCWRTfy\\\\\\\/J6Ru3Ad\\\\\\\/eF1bUDGhPWi1+bjU3HyVOnLS\\\\\\\/QsbJxiguJ9qlxRjLEfFiCngzAwudj8eXN5sdEs7c\\\\\\\/uPY1vSf8WnZxR5X\\\\\\\/xtiP+rkWdGHrfNrwLgafM\\\\\\\/6QXWAZJfGddBlv\\\\\\\/geHDBxgVZEU6aDcRxUgiJ3sy\\\\\\\/kp5SHqgclRFB4+hZnJ57+5i\\\\\\\/ML1f3DDF0SkhUqax8bvRL9ijq1qsSssTa0Q51rabTHhDCk1HKl2Sql9IOYRu\\\\\\\/5QUW2HVmws8wftxnd+ts3NCdW5bDtqhnmTnKVCZ\\\\\\\/XHr9IZVXXOfqA4QWEIxIrgRwhOJz\\\\\\\/KyLr33Wiq5CMQaG5PZ1R7dteENQyJNMBVCTSdcNO9D2OuUYlHELrqGxv+9vNm5GbrY4\\\\\\\/R45sq1GODUyeDUnQU0d4brD3z5ACxEbx6PVgO9S3UystBXeejtmFw9oPUfX\\\\\\\/bziKeG2UvC132B8IW77Fq3\\\\\\\/hgZAV93r1t915nu5Hu1LkVQvBxK4bTr+9Ru9GrpJDQF75jkR6CZ8tni+qmV6f42F3ky31tH0UR4k5MvJNeVMXqmfxdwZ9gmzDwNj+Kdn8gAQyhszbxp7+phrDBx79cSK+EZ04ulTSyBKa9ftKTkelO1HwNx2XOVl6VyaTH4NgSQccsshHNdGAaDMGOJaccjl\\\\\\\/9cuR6+BOur1YGRsK56ESUZtj9YnRMBJecNFGFbp41LTsRL9ZlszABQGic1pQSFyWMIa0udd75FmIe6tX7EQpqyQwxEDj4WMAOPgdERsk4ov1JaA3N8Zj7Wd1HbtEyGGj8+h31glDh3nlvMF\\\\\\\/axaqv97pu0Ho8yoitWnl758DmenBg7lYilWbkJOUAIfYoGHQkypnFWejOjMpuRNHUXCvwajs4Nk75VY3dKocudlhiUhBN9k3tht4yqmu4baQEK8zCYGqTBfjGx\\\\\\\/xg3PvB5BjBqTwJu+LqZhC1cEKqXykQGnrfNUoZCY0db5GNtgAgqW9\\\\\\\/k7OtgC4MNSFOZbwARZS88PHoyyM2OYLkOkZ8aTArQKhnPSCALbJ96NJ2l1Ql0nwsPv4IwOmZtBsU3hhz8MqX2WStvxEzT6pvxxuDp8IHEgI5\\\\\\\/5yrVcaw3Eg0bM8PQtbSPl6mYgcW4yq\\\\\\\/iFNkNvBKsDaHEBSGXLMehlhaxXb0h2tjGd4lXvgHzAIdJvc0c3iufrin112Y7RpNL7ncNvcEpfR9zGfqzdsO0jfQBUudEUTjNq69cVapHa9oO3E\\\\\\\/KOL2zeYlY2QIA+Ue9rIki8lJvUp3TGA8RssRgM\\\\\\\/N8rzr\\\\\\\/cYqXrirqYxdO\\\\\\\/+x7Odk7C+v65ifBd922eeqfKvFbQzaEP+YdM7AoxwcI9MXdeIctKM1TXFgpDXmWRUc2vMnk\\\\\\\/yemdwLWYvaki7xvcW+7uYecv5JUZoUqOlfktiFaDfloZALEE8f9sZuD4zsnk7JdpG91Y9YEeBpD4QVODeg2V6HUAy8zZFfdg4ZMskqXlGrefzFPnCQtOK8VlY4jXI668qYHbsxASHGiOT6bE83tF3vGoKDpNsBdF3ETBWlIb4vb\\\\\\\/ZQfGDCsL0Ylj1irHe4LckGg6610J9OOoeiqxvyuqXxoEFORW6b8Zx0bCRER150HODsbHv+90\\\\\\\/QZ0gsSlo6zzNz47nM6bp29UZrG\\\\\\\/y9LhiK+5Fri071foKg9bGsgUc2XM+0jIjar1UnJ+z\\\\\\\/krCBLVovWRGISq7FmpclWIqJ8nTvYjmlviIvg+NUp61zr0T5zg8X9hU3h3BlVQNXCCik6\\\\\\\/NtscYjwdzIph5+j4es1Im5bXsmzBMxGltuZuXl+9vLGbb7IXTipBXfwlHgsmjRwk6mhu5miY\\\\\\\/n33Y0DECWIMhwRu1sFEAgsr4oqdoJ8xtA90piGNsqU7dIm1+NkIDIx+SNKMT4eJ+t6QW5ky0yEKj+L+a8vN1IrO1IUNsOiR1ysx1UMXOn+bVXYA18xgvI7R6aCYkDs9yR2y47hWCzK+FoAGVRKoWlybViaJYB\\\\\\\/4KAIlE8a\\\\\\\/AQ4dWLWtZ6ExBe5qAb7BpV+eVYSfXpkSOURpmQ7TDuoFDv\\\\\\\/ETv4pUzvgWdI+F8epXS+LAmdUe+0BttiBVnNTq0NxZIzMEDaJJwoUpNY7bJzwITUA+YTDfJUWg7LD5LwC1mUcIX3AMgKfNTOSmE0qQAkxQZt5AQQyeJSijfOmKDdIbqZ0Lgq0eQhTmdKDSDFdW5smVDgFTuxKLGRWAb8lyOR1UGVzecvm7+VEQsVC3ZLCncY9E1nwk4JZ8ntXfIiLngDVVVS9Z9Exo\\\\\\\/o3tw44kMY3AjAQtf1aQG0O0xdhqOqh+ZfZ2IS0QnR0bSGCYR8sozei\\\\\\\/KxIgoY4omi0\\\\\\\/C3OS9391kxQbe4s4NEBy1VAWvGPNHpMosq4rddsDG14bsrTlKr6jNoggHOeOPZOezBcTEpy+ZdPkYb7bYFsv+8AEXJfWd2H9fY2BmrWM28HW0z\\\\\\\/+KIfgzOrRbjjEZqDw580Ld9oDjo\\\\\\\/QGgDEPW1GSbOOO42x7rQ\\\\\\\/bpTBLx1+HAcEWZosyaLoU5Hx4qZKc4G547+16dGTOTh+t\\\\\\\/gR9uU\\\\\\\/P1WdVkKw6Hu\\\\\\\/\\\\\\\/qVJzJ\\\\\\\/mY0R0WG7cij3CqLuhWYMcP4UwxkvVcFvqAIOSQbGtqMtOPhcPoNEGGzUhI3DBi+WFPDkqiNZfw6XcAALZepAiq0mywuQ+AS4rIOZHQnhnQps2G34h7rPZqfMHNu\\\\\\\/sWeZv8p9o9B3+lb0o4kgqTLLvmb77Qj+zCeV9C+o4D2BPSG6VJNXQ8HYfxP4c1Ba6LEoVfLFuhfcGS84dvG46RJR+uq4g8essjcfBjqxnXR6Wrgo5ZyOVot\\\\\\\/mNgsmibPKExv6NaHWWGZfJ4Td5\\\\\\\/96iFEeqCnUOgUjp9EsQDS+72wydio2J1ijQWAkGOjMs2UAvQ3jdO1mqqWZBTpgyVr3FhWlJz4xslwMZk0BV3nKn\\\\\\\/A8rXbXIlbnj7ZIWemuLp+U46uJbV5UQMfYKBRxsCbYNHFgn2IAizH\\\\\\\/Xg+lQStLmVuE+HEPv0hojUFTDwwUXTLL89UZId416aCm2muS8Pa+XNUB+kKOBYunC\\\\\\\/+Dfenb5IC+Ru8ogDlOyKs7usABOL5iFb8YkTjM\\\\\\\/9TTanSvZ210mFI0anV06rcYyclTBWcqYQk4HdhAeshk9e9HhcyZDUvRO+s6Jq5HnyOki7OizAf7iNtMAyzt\\\\\\\/RGKGoCGf90AYyZ3iZ\\\\\\\/8nxkcQTm0jUdj\\\\\\\/M1AOKmHFCjH5eVXAW0XbxVDUL\\\\\\\/2P33eK+utXJaG8JHOHV6OCJ74UkSNU25GwlK\\\\\\\/Q43GBSl8Obx22HkSbgzP8aYhKMfpgrq+lba1VaeMnWxt0G5o+svUS48KZ\\\\\\\/uWubk\\\\\\\/XtJCbOot6tH3o5fNQWTUWCmHg\\\\\\\/vBYzsVTZqToq8\\\\\\\/cQCJnOitcIzNUcBbs4MrpbmAgJ6A7LiBAxlC5CGOED8dByY97VYqSDQ2t9MLOl3Vq82SLjmJhn+pm\\\\\\\/3jozn7ark5YJIJuhhv8eASAP0ThX+8r+amRFQQz6uTglC9W4ayMr8s3b3+XBXtMBsf27u3tJbrL\\\\\\\/oHO1OebyawzhGiQjgaZc7u2tgvTOoJ5azfhNhUEcTBiwafEjo1TB73lIqF2ql2aJzRB5wd12+D5GXxpZL2R5QZcdWP40k6yNtSPrZwSzl4zyLv6sJyead2j\\\\\\\/B0UAQHo1V9LUu+A5comv\\\\\\\/SL13EUOUXkksFi+fK9JMMkk40a3iE7xg6ybvwYUuC\\\\\\\/kfW4+7tyWLaoXpkS9hc87CxjQyv\\\\\\\/DMRhiey9tKuLMtXcPRHbj2M0he4HK7oOLDMmIvNFdaQH4dgfQ8MLL0GueZS5h+uDwE1N69280t9bxOofiazWEmfmOoMQNp79gLZ69+9W83uSX0jo6DRDYSUU2fiGiCP\\\\\\\/T+6mU5s1agXsqa9ULFni61b4aWgn5t2wuR58RAVZ7X+HV9lwsPYse7\\\\\\\/mdNprCXDbhJOlhMmSPNT3JNf27aVoqKCNnB0XvB7us48HadAOpHjQg+XRVYXGuhvNMZ+eMfOthh2ERi8ehDsTuozOfRaCGEjfdVzPmSHgIAA4Cv9wPRdqShfsu+6E6eSGZnim9Qcc4vySPS8OeUpBMsHbBFy9E3RpPUPqsCXZjZmcUIkWvCrMGYTrxdwIyi6M7yFRcJ74rK0iFXTrcZ5n6xJtVv\\\\\\\/a8BC2LmDC63kDYwbT1a+97UKygWat3nttYRijsatYqcjtjmrb9PSyFaBrOxkvvn\\\\\\\/br05e8qsyNqdOK+6JpjTaPq33S5lABC0ZdzRzl8bLi5r3hDo12NyM2Oeg+eB79JejrmsFtEaJJncVwI\\\\\\\/lWtNo4w\\\\\\\/2ukKPUciHHp5BX+bFncdkkZOU3iBSX\\\\\\\/SyXmZ56gGthIwYpaGSgP+TnPEekraaIHf\\\\\\\/2krqk\\\\\\\/c0uDeLjS\\\\\\\/MJu62mDBWf4XYciRW1EQjf9Iwe8zZgz6GkbJS0Enogp+qg1oYxIq+1anuryvr86ODTiIx3t6\\\\\\\/Dws4KBf0kwxyRyvbHh0rN6qMY0q\\\\\\\/2qbWxj8vlgjYfhek+gC2JV0t0LWtUn4Va83UEg+0x1hQfPX5XI17EX0G6\\\\\\\/pj3pPS4Ev+BIdLCAiVuyRUAssh0P9ZWlB\\\\\\\/C1mrugPlSgjNyevUnoNgT2t4NDYzHSpGQVQQZ43RmURa+Eo1ETrqzCLq9d7adpO9GUvzPrhlisLbUUZJp\\\\\\\/xuGtkELs9XFZU6S0DXlKTjgmdAJO2pgcivcZnfRINbiWt8quR0Hltidf8uk0+EWXjXXGG7HEUDJD7pdUbyqrYMXiRhiF2YN52zUDxWqFZ52XRLMwYY7O94otJucQogFjLegaR5ldr+1h\\\\\\\/1fzV8UJqEqH1JMcTrVuFILOqrIeNlnhiN80yOEFvaQIXSnEtdPMQtMEtCnDb9urJi25KnKVr+bALKRVLk2YdhH9KV7RKSu6hO9hJMgp3zmyRjj17YmFzVowCrLm+5QR6J48\\\\\\\/gE\\\\\\\/RyCGUg9x15L2gxn6ptUCh2A34osrpn41FLusXyjAKD78F\\\\\\\/YM+75zH4wKkWqMSO6UnEOPvnzfP5mWZ1gAAoPWnwJmpMgYkb80Gq4z6TrUUyFZd8KOKZ7QT1e4iVjAgWQa2DzSZdtOQK32m0DOtTquQa1lfwCnoHxeUXCxOSvVaHetx7b4\\\\\\\/zlh8F0DUuTSeLXyyN1cb4w\\\\\\\/FavGmAQa5v+mg2WuPZCa5sGuITdaS5+1sm+D58LXecCbsHGTue4EvF5Bw\\\\\\\/WHVUJp98eoEaNV7lYcJIRHdJCOzt2IOD\\\\\\\/+EH2H6S5WbIXafnF1g6WYvTFXIWphf\\\\\\\/o8VLOpzrWSjohyOF8Nwp7Fn\\\\\\\/XPR\\\\\\\/znzy86PhKPdlRgZpJPtDB4HQ6vHkgd1GX0em22nFXstXXyNpFgKNtzshSrRYh0XWAYcCvDhkqyUl7FEn9DosyrwXQ59YsLwyiBVUcwLJg6+BhK8NRi\\\\\\\/FhHILdfFwQiijMHlKS112Op9UYj3jPzA+MHCtQTOq+p\\\\\\\/iRLQs7Fx6UP\\\\\\\/xU6rMvU0npbaHvXpMGEtCiSnZPpbYSOgSlwJtcG+2SrEUMUn3CXnH2SAJjiphHVfsIdFmLr\\\\\\\/QNuhOOX\\\\\\\/vkC6T0hw4t1DpM7nqGVH0\\\\\\\/KwUWZOLL4ANkguRjYnEYRzRa92yYCaeZQNb7pyrLaHtTM7uZJ+CB7rWsrTd+6hY\\\\\\\/YEPRIDMk117Dqb27KPTUVYoNnWmNfp8B4FE5eY1mRDAtJlxkR3z0\\\\\\\/OpC5ERehepytcb5ynlx8enknuV4gTcDNCKilDzWuflQfQxyc+7Ymogt+sZEROeU9eCcd57AbtBxpVDlyBuGrgYo3oYMbtSPquDPmMPkKbxnRWx967p3NHdnh6hp1xilrJuGni13vT3bo5DiKjrVsQTGyrPaJvq0\\\\\\\/10TBp6C2u3zXXhKlL\\\\\\\/1nr\\\\\\\/dWnegYJZOo1K97GFBpVcpa1qCs8LVOslrI0Fk0N\\\\\\\/UHycty0bsNdf2NSbqVD5VS6298YZhGJB2L3nIiv+L7cO4MEZbIFUvkRAUB1Yqsa4d0M1oHN\\\\\\\/k8jP+nEc76aHhEj8Kl7j3gbXGH37d\\\\\\\/kPs1k7ZsjhH9upusg7m4YGYh4R7oMTH3rpkTKFAuyRDDOxx4Bpyk5bsYia7wSDvEwVEUTVwe8esLClDHJWj35RkEAltbAdyo1+dP71kk+QS89jzmsuiPZOSLpZuQ022CXmr3wFobmf9H9nuq+OjNH\\\\\\\/A8wJmbXB5smwBteiqZv2xhKblOlx+4plDT6oj9WmfI0\\\\\\\/hMzs63qxw5OVhwhPF3\\\\\\\/7b9E1JIAn7ah8vW9KCCai1ncchipoInom2JCJnqqwJfWlarmbylV\\\\\\\/r4YLUiY7QNmHFsE1KmBcbLU\\\\\\\/+OHjR4MfXi2aiLNdMjUjcl3zZDVJ4JRBT7LTz3vNJC2\\\\\\\/RfCYSD2ydO1ZRfzCX96MQLXj8ofXU0aoR\\\\\\\/DEgv+w+4sx9FfkbDFCpsIZYNhF6JG8mRLZysGKjwHdE5SVCHOBUJ72obgKnI+00jbrRb3ZqaT0QsYhJXOrqIVefF7hFOxoVCfyfWH4\\\\\\\/OVheO\\\\\\\/dGoNUTvZVPdl2vvQJ9lunmA1Wpx8\\\\\\\/CrA9fcAooiQ29FhOjJtgATg7i8dvAZQbOr+6w7bBL1Fe7Yo58iX\\\\\\\/NuAu8+ezFtz6zRttRdDHCKrKyFwOtkBFU33WbDrFnnmEJN2YTpj5y2dB\\\\\\\/zPR5+rI+HbOmhVtViOospMojTMNo05NSo0Q+MYLQuDRW8oWj2MPJNRXeIf7tCYveZ+wEpYRcEye2W45rqF1N0GxW\\\\\\\/S5yNC2t1zbQq57xfMw2oUrgqXo9AQDRJdhOE4SyMkHCgYsDTpdAoPcqz2ZNUdumThFfHkKzrfNBB7pFcs\\\\\\\/KouZ\\\\\\\/4ewD80gboCnMW6z48xgG5E4fiUa3MRW7MU6T3TG4tGxVrkj4I6KnQy6rJrdQ2+tY\\\\\\\/+Sj0gbtGe7NhiFwuH921Ikzw0QZeAq6wp41lqcuyiazHLaqU993AX667iwi0IzUyrxqShyI3o5M18QpsajoUKdnaJnavVnCD41brY9fXQkCAxb4AuI6Mrq5rx8yfAvzvoU6JgxLy\\\\\\\/xWi0D3floyg5w6zzkzWd+N6dH6d7EKIJXrDiN7fkxd0dSAR793Z+yhV1UzwANrKxmc6StmxSQrJvj2xsAZSBnU5OIoTfYbuwj9SIL6FOm0rev1YwnP9m0WTnQFbLtnmUXly5VX9ijRGKpBs4lfgCMZnm5nHUkC8lAUL582oo78V5H4ww2w+mKQ7bbYku96kKxDE2DG\\\\\\\/JmUjiSkCqiXFULLl581mwayyHZnm8gxxaLH8nHa49KJU\\\\\\\/5PQoDgtwuKuOPRtMVeynCvuNlz80Uu3MXE+Y4cb00DL40vccwS\\\\\\\/hsCg4LfkT9hJCePvFKHZ5HW7OAQgNZmGinz4Qz+pBlXsii+1Cubdhhx7vHdRX8p3LI7TCIcbo\\\\\\\/9yqxS1uASrmH9x3euXu\\\\\\\/7SBJolANTfWdUz2qDj5CHmjGzCeyHF5Hwyd7p5iHNTXBSjXWPWD2COokZxGInbxWX7l3zy9eiu4rHFbtEgdKjGmj5GBrfXGneQnGiHUDHhI\\\\\\\/vOdHqEo7nu4UPm+G11duvGtY6LDzFBWVGrHBcO6ktVIVYHgyooV54A+GyQobjroEVLlRQmYGkhdrTs0\\\\\\\/JNj4EnYk6D4i6bmcahv4mU1tnVvrk4ymHLIwkJIOrA7ubY+dYOcjOEZQkgpj3zbl0ncvOZgqBYZ\\\\\\\/LjgSa70Sx0eKrVK4890j5zdoxkmdbb1h2YJnN5bcPgUYGapuozwF6L8XlVEPF6FiSj0zWcXDS+Ghfd9JEK7vSr0awNtWIckMhRvZRHTEePjAFTI3eNl2kOdarvLzQ4SQYC8tAZlKyi8B6sF65G1QT5evQv8rKuQa8VlzPThxDyY6iZkVaFu05n4oKmushjxjb7Sm9Cho9IznINV7a5hHzKnx1UK0NNYTgkDRlx47uw3XLGL+gZUzu8L\\\\\\\/hrvBMhdfZ0fN\\\\\\\/zSKwa6WZqGGgG2iRRKzRnMUrHYCAuk2B\\\\\\\/\\\\\\\/p6bEGJX2ly9XwDYdW3u\\\\\\\/NBIGxtgvC3OFgK9oHMnBcrWCjU7jcR2J5lX97KcM4ICXeB05W+2ngyZQEEfF1aW7c7HCu7RClJXSxi7aaP8yFNzeJh8NCakIi4\\\\\\\/QncHW58rR7S12VJf9TFUmWRy5LWq3DH0p0ecmLs40izhW+jzZvZ9OGrectkCSRPK1WdyKTbPs8Y+2JC9ypH0QbfNu\\\\\\\/MKLdzEDgHs\\\\\\\/3kV+bRgxf4RoWABwjmk0JJjEvkfvRVRPc3d6uQ1JtRKuonozQiDQqttENI3bCISWBixGqUPCJM39tJALAJR0jwplVmRXZdKdAgb0iCdwHJA\\\\\\\/\\\\\\\/PVAOapop\\\\\\\/NSAjDYMYWteZfs2aus0I38nC6FEpA8tHXrE75L7yaMwSBu+QJlFZSCu6WMsLMmQq3cb5Jnlvs2n95ZNKPtqQQoUBBy359AcjsH\\\\\\\/+C6FgKCrre5zuZa\\\\\\\/oIA2\\\\\\\/yTjaOma9lO\\\\\\\/iSi+bXDTNruz9HIiLScG+Py17+947f48wJH1L+4MKWCzqUArhUByhu6Xj1OGgAtTFNHpA\\\\\\\/NvEA5cgkAAVuQsXDZCHSkuzwbJtuhV1MO9fGudfSXzfDkMM0EqlgntTuTjLjCAHNKP9rZhrSg5g1D5WJQc+oO3sfMzyplKrgx2iq\\\\\\\/fyAHkaM1w3T1NGkjeBKRyKzV\\\\\\\/moXl6AaHC1m3IozptWpoSVM3J2E3te7E1RTXBZ+OsZRWue6rPYWCj+fOAbE1IJhf4ihYBYYsrVgoG2ea0gwIQttkKf3OR6d6vYKdpwoHhsT0vDnKKTvOziwmdA3E2zrZSic6SKc8sIyddRkkgKysvS0jql9gAEnQxLmreySNAoqiHAxQG+I++lJJ20lJii9DYTOFphYHUeKSBvax3x8VlwTl0qO\\\\\\\/ATnNoCayRq2taYYK10RYkoqRMwdloLcBLWcAjgibvioJP1Ian79UrFu4Y\\\\\\\/YzeKGQAnGBkCnxW\\\\\\\/fgCjAf5DtSlWhfZOyZV0y5ih4kns0yx8SzIYw2S9KwYF7W3Rv857O1Z0HcXb5++4QJad04NmCwEeOqfelcQqQ5NSAH3KmMI+eS0WfAsQhBjXejeYz5bnjJIHPXotudm9HxYc3Vu\\\\\\\/9GNMGrW9ObZUNr\\\\\\\/jFCS3R830oMbwsoUpttDLzKThAUVkaZq0l72TRd5eA\\\\\\\/PMDJ2R0Niu8bInjeH6\\\\\\\/v5phCdPVAhXYoEvN9nvBXkilDZri7exBT8i3YNE+5t7ScqoOCWue9pKnOy3RBsrvFtZdZNChT1WfozOzEbKQeVJrdQjmqonC3RXPyLdUCMxrlu7GsvzYovfCfdCFBjcmIMQCgjVtZVWQFdqHgN++oJGrnIWdG8ZuEC+VUl+Q8z96PlYJvgWEVRAVfPJeUTCTI1fi0CDHh9q936PE7V1xyqB1AdIs4G1JHsP03r+wrmqFDEHnWivHbRA\\\\\\\/Q3PwPP1jV1cSiyG6RwjCvdMWtu8Go0CdPTJvJHOqz4PQNLu9Xx+7LgxqcanavKVXwj+iVtpnWov9KhhttwOauB94yQkat+jl8r+MTy9cfYDa3+IE9ytZc41tfiWkmJVJTIAapo6l17PuDFaLVqg9ZeIAShQG+hfb8HLpGcbIAaK4OBy856rF7RHsygtTEXL7NC\\\\\\\/2GgBvzHuCAzBaPDukrNqAZMn\\\\\\\/wwudos8vwiu2pz6nE7V8UWtpHPOzpPDpMNhYLEovR8vRfVovVNJ0VHDOFRWZTOtmorgDQlnplbNVqZyqyIvvV8UEg+8z\\\\\\\/owgTANiWe0tkY7M7Q90UPvyVsR2fc2FdeE9pXDNA1U7SVVovKUU18BKdGJgkQGbzcnZVjkatMxCadCLqpGA+7AxKtZWHnwZ82UM3Y4HF1lMacy31prIlYFBGPrICjZfag2exw3JFmrM3xmZJg+BKdiMYhsYnZ5+Ay+zzPzWbO+05jA\\\\\\\/35bd071+LX3gimbvZC+WnZNgvPQft2PsVwQy3DhZeuigamniS+qZYp1Wr8LwPhSSoXYPO9GOVTB+2jWgM1SsVec8gh2SUk4xEX3K4SMFpiS7r5e1N6p6R4Oq3B1uImCxAS1za03v2WFhT+23rCkA\\\\\\\/Zaovp2+eTStk9w9dcdQIusd14m\\\\\\\/F4mLt\\\\\\\/TaR7EZ1+gND5\\\\\\\/opxUkaHaz1PW+E3f9iAPPBzky0Cn1llzrFaY1GtELz4\\\\\\\/XQ2bGS8QRboo01XQSWcKDWs0Xgi6PZnr8uZ1MRAPkpuUDABbYu8yCNlzwE+LMFzs43kfAU3hfHg9i\\\\\\\/yvlwUQuI3CZxz5j2TRuh9QmG+bxEKeH4sO200gvOUMXqboHU0Pmb7MNoZfa4IWl9zEC9DYWPSDYnH17QAVKw8T\\\\\\\/+aqLgV+IhzqhzgePjxKWp4teuF6hXR5Ag3ZxH6As7F\\\\\\\/EIb46D94AAA9k4Y3LW0UnK4U78w19gnIXqfR+iKYkILtXlPhkhDRQC6Ta4\\\\\\\/kkJYI9ao0n\\\\\\\/n04GJwv8p8VfucDNn1fbZ74dKUKhiCxivbj0Q4RoO29HkPl\\\\\\\/gztbfwNJk4SS7+8AEdQpvFsRpjgCQ3MF3iI\\\\\\\/FThttC4\\\\\\\/rEH4+i+3XreC8dZ0WlqVQtpB2kHaIoCa77A6Kgw7SlnV5K3pRtBXgbCg\\\\\\\/UOd8tLm6L8ZdS1IUFElVbjVfbbNKeQxLPf6g8+XVm3+zMoT8goi7fLBdHeIqMyW\\\\\\\/VwhG\\\\\\\/OC0+Z\\\\\\\/cyOHYbYgba7oviG9vxjBUpBkLS+EctkY15E62EZCU3GJZ+HHt0VecLkcquM3z4c2uGuv\\\\\\\/l0eezpMIY2CSiZJOYP36fnceBpP\\\\\\\/pBtTS+BgcXvwJk\\\\\\\/p7liqNtwIx5yALi09U34YkGMNz9UmjW0BrKYCpRfnCIl26bF4\\\\\\\/CVshjCyY9bGjdvTgEWMfGBcCrauCxCDuADHUrynQLUwwOEL\\\\\\\/iYarAmrlRHwvvL37IvZl7DwNsmGwn\\\\\\\/SX0aTijSXkB16RClGclfChnBOz61TZxgvKRA2YiU+zUQu8VH1lj0b0jRQX45p6J+rEgqFonylHdYH+MlQ\\\\\\\/voQqE2a2DeZHatikj53rP9xkDqYfQXqStqA7oP7NB6cSPIwCZ9zYRJWEKQyif\\\\\\\/1RJVv9bLJTUJX7FU9lP5mxK15+gjLEUqUBND3DTUUI9AOMfKL4UzzNYtwWoBZ\\\\\\\/GcvYNJEW+I0jMIqNEtTVLXsULg6HjoBmcXUH8TWiSWq6TflEdBx70eGstAdz7hCqvU+zPFNysbXmF4NIaM6wXq8i8LtMp68X2U+\\\\\\\/pYWTVW56Zjq+cNzhPF2jU+Lw0ia1CE23TFhCyfR\\\\\\\/wDplawXbQ5dr0SY44AB+lG7P2cMQyvdajmkTDMvsknvHa06Jrd8kjsDD+Zrke8HjvdKarEfmKyjDNCDAcxL8qFDK+PHqKqiqX8rR10IzbzNKlm2LuZk0r4ymRilypbgiTajIsGZR9FGvx\\\\\\\/sH4OQ0SjxhK+XUGrWbwrtuBylBZiEnapvjTFgFWJvmrTRpcZd4u0VE6\\\\\\\/AsWzwYutgNv+KXD4NZbF7A32+OOLk7ff+qXg5jyOuCrgWibHw29aJocl9tQRHjcsFyJRbczPwJ\\\\\\\/E8lI8E1YQlhYHN+6ebw75Vwah693xaDVtVVUbj5VzuH1qKXFLUZrl7Wc7ppLw98+PdP+Iyr7HB7EXi49FzwpCeFyRczOLBaxEtYc8EXsw6hE8CgerOQIKA3bocLsUT+qNL3YIoItD3+kcIYkCuV8Vr2G6V7Q7aPA\\\\\\\/bU86RAneb3HnS0QgNOnxx7Ku2cy9TV0qP94l\\\\\\\/3Z9DaTLIv+rb73PzhCV6LGidto1Ol8tCPoAD5fKRwWXV7QJmNfKui21KcG5Wc9e8+TNpQukRoNgah56sSQuD72utOhJZdclda07vSM9GkVpZp+EUhR7ZhPPRq8RCVNwrXQf7Y3KTOS+s0veSvb\\\\\\\/JBDyYxN9+c02yoltxfhNDePtCI2ci8KSNPRHGFBTrL0CW8tkSuBfcajlerAM+JCf1QfzkMQMTC8A7SJ8GRQKEzHRk1ukkwe1Ihqt0rkMdOu0zLXRSVTQ4qKq3HQ6XeZXn0YcQYisgJXx9DyaeLqWn30EI2y\\\\\\\/tC1g18OBBXKj\\\\\\\/BU+U9roc2qnm04VpZAiedXoAVVpaYjrxZue4SiCnahvfreqoh5QFEff77OfrTPr6eDsVK7QozN4CL1csKmR8WaG\\\\\\\/BCNzlY7Q3f9k6UsCftREKGIn0ryngdHtcb9fmypGVJgk4YIclershiWxoMNesJiQKy4lnI+d\\\\\\\/Pkom+BXZhpn5Ib3TZwOgHJKsyZGDsT7KbUdU48AYf+UxHD3RRxRn1G\\\\\\\/fFEQ20Ce5ELj4E\\\\\\\/C\\\\\\\/zKdSct7ZfkIoLiYcUVzZYmN9yiVaad\\\\\\\/0kAy7rmop6nhzGgX\\\\\\\/6zMUWE6h38Y4js2meGSWm32A06ZCiYMz0ZE2YfrPoFSMj5WqJKfhGRgfCEeFTXqn\\\\\\\/9FsgOgJPhWICVFwS5VMOj1iS0B+icNb01SQtIIceWzT0Bdv6WuiCHUP2BC+mwmxJXG7mVmJ87nm8JnGPkAihCc3GZ\\\\\\\/qfvg0Ip7Zt+3OQBpuel5COulljtM79witCTxIGBMHkZdCAOClAabMDqYeYMjJ5WkLfEvgbMyx4yxE9+rGSF50z1y+UZrrzzXUGC67S\\\\\\\/ADYb5PRYQ5dxfU7b3efQA1WKsP3M\\\\\\\/Uaa\\\\\\\/X0+JQNmWhpX3UVCWXAgdH6P1wgvvnmDjrbAFKaP+7hTUwZ3oJ5wFJI9y204Jn4COvcTvPy4fGAIiiU2winH8oTtZF7lkLwgjKEDf5WePu59LlPOs93hNwwBOeLV9oOldLz3IAUMMdKojna4vC\\\\\\\/PqVmVAOTXlFpM+vSni7fi4uvjUXGl\\\\\\\/7p69jIC0vxbjBSdLI8RUp3mir+quUOOSWPLnUpaElkBh3oVshduGjHjcLl6FTw+Wo13KNnW7dLXXgFxNCA4WUZ3DWit415hFRIzhFHXj28fKutu294VtW6O5w4rPfPp9\\\\\\\/yUAH81rKgV21Vp9f8G5BVWRFqdxVQUwWVCCwcXAXs2iGki7YrxyRWcCk433VW0kRYazBdbTJD0h3Nld0bUTgxVMXdoQHvP0G7ujlWzrNwderOTUbW0x2NF3NPxTZ5I7kSLP\\\\\\\/hIY6P4ZfRfZPxRPWNj1xl\\\\\\\/QnkdAF9p66I+SFUKvj3LkJ9jOwV+FnVBwv347oVi7JIbLgjsjGj\\\\\\\/Dx1kl5N1Y5F\\\\\\\/i8qv+nlqknvw8cnrlXJEKlBS\\\\\\\/9giG1HuXzsGQ0qpYnvgqvG75QSPfWzpVGLYu7CnAv0RlGF43l9fx4nFi+NS3J0kaZhm18WBa8MC0tmCTxm9h26NB6a7Psz6lOy9OLCHJnpn6WyghDDDUMsGGrbwtiE3DGRIQZB5xZ9iboBqBDMnmnebR+N0nIwJ0Hs7HpvOTR5fgUTv+MZ++ARfxeRj8GbJ1ND4u06PbC\\\\\\\/+2IaLljykC3LmTaqFdlYo2uO\\\\\\\/2l\\\\\\\/k\\\\\\\/lMpIkKMtjHAwh87EhtbjzIJ1xmRS+Pw6wLX\\\\\\\/JZuhCE2uXmvHyv90CHCvhU3NgJjvTVvzNJdcAL\\\\\\\/kW7vnGnwAgPNX8heNXo1Q3KPJRRsMTRJZixehDnpPDfMg+wvlqBlZcAqBdRxub+PV11wi7Os9SQCyYFX7plgvK0WnIP0MeGk+ybpUx3oqPjzSHA4IRcbEkROj1QrXnnNUZ\\\\\\\/xX2wi1z+P7FdRjqblwJo1vX0YHuMMrBYw8lspRe8UBauabngJlYIsF9T2sNU6HdLfSA3XgkCR8lqxkYxYV2BJdHy84MGtNR6ze0u\\\\\\\/JBjWKhYp170id0Qs14XozN9yR9rERqTaJTf5pLgbpm5nszLJswmlCYSyxWBhzQog0+TKduxCfCnNUvgXVb\\\\\\\/siFXhXURVGIVU2JqTX6Vh47tOOncI7+Mh0K02tL2BbGSalr7ovx1045okCk0iQAB2ZXrYjhpa0IEUr5dUXc9UiK0FdBLvcCdIoPqo62\\\\\\\/d+\\\\\\\/OzPNV0rMfachplcdDsp0Rkm\\\\\\\/WJuqr\\\\\\\/YULAJugsFvsqwdcpIiDJ9oxqPu+EjBoyebDoSMrTXEtkOrORk5LsKSWPVqTEHKWcGJtrzUWe2No6ZQXoLRWKXU8oYzlyVan3S5q\\\\\\\/+i3mM9NierQUyOxBGa1FhFG+Fj7a63+VvZAPLRegkdXHMavKFtsEcGVEjK6fqkGUaOVRZ58V6PyQz7omG8HMm8J7Q7UKrnGrw71Zlh6I3ZRo3I9uxqTkFEd7uXp4KD3b\\\\\\\/oAEvf+Clr7yGsHOuae1gGdSxFKpxN0Tvw3Wkk+cw8fstZtpoVmdBRyr\\\\\\\/u0Njj5BO\\\\\\\/U9A72rR+WACtH1c47A6lxncCMSD5oTOmZqhLz0i9t0gq4mA+8omVekmocC16vMMFHLF4Dcu6v9sTR4uYbFbkGY1O1i56Y8QyczbCC7I4+8Junc\\\\\\\/J1HMMUx94eJa\\\\\\\/VC98VjTQUYC+KfPPWxaIsjwL\\\\\\\/CpSfd9Wp5TZ9C2Lv50Q5ApnlFg\\\\\\\/RV23R0gTtRH1rckGqpQMm6TR9J6Ayr4rkFKtUD4lyiM6MCCC298RUMvQhUoqL+o3Mm8y\\\\\\\/8D90WA6T1nGjCQG4y\\\\\\\/3uCtZMM+uQhJi6TAcvuamagdgUfE2UaMtQi8SHx5O3UQ+zfmlFGaDwDwRLVyDqEuEVTeG0+J4D8YrkDa9HoEW3tG9Qussf4m7Snh2sO1OxYevb9a0CFpXQAQA9+\\\\\\\/QtVBiK5Vynm2WsDJFl68Jli1n3yIR0ajFAKcrG75a5GG6\\\\\\\/i9COfv+4pa5ad1ig9BKXoHVI16S5B1Z21UQEZ1sweDc9yuOKMiXQNlgKAIYEyxD2KtAlgrYhl91k9n2dH\\\\\\\/QhWcRK1uF6JgY7kC8iZrwodrCoC02SJVLcpdPuNDrpO+ScvbUtaSEcuVFxTNYuad+N+zpEOIiBZMPMR1r0lA2uU12rj9glxfRTpBO+RBhPd9p5hVjel8KQ2YE6\\\\\\\/KzMrx054sT5AblbPclkQUoSe93MjuVkRmJCtk90ZmS2o8ONGwy3u6egZ0zAb\\\\\\\/1+ZUZjdqsAhCzE2NYQBICGmaaAJKyaKyy\\\\\\\/kWsSdSxucqqp1L1h9pRA2m4Xb9iYvSGcE1epz2Z49EZzUtaVefRxF1lKUjCL4IiVQn+0754YKAKeIqgu1pcFyNavK1g8DZ\\\\\\\/j2MGhSpRL01DnFTX6Az\\\\\\\/R98nvMTKcczP9hmkGFtMpM1562+0bxsDlvtTCmr121e+NqdlNEnzhgrppiJS6w\\\\\\\/9YzGXlsxial\\\\\\\/ahkEiq3+qR2p3UbjkAc8tG9JStTGHtlSjSboMZDwPPljwU96QKgUtpim\\\\\\\/xGxIbV3o1q9hwJxL1z7t3MDqiGJ4\\\\\\\/KfjRIWdh4QxnwBd45egAAZJuBSJkl\\\\\\\/9C2kxHGGYS7IcqRk2jmns0QnbqwcExztGs6\\\\\\\/WeAstfnph6E8dGZWTp+PXln6QR+8L0jETOMXRFV3DkMHjXl3b+qqUTQ9PkVgdEsLrzT89D9fCPFS99nRS+2C0X73Q4BJDBn71uw\\\\\\\/O\\\\\\\/1SoNC48NwNDTUIRmF3a5oR8VcS+alsnh47TAsfZ\\\\\\\/UB\\\\\\\/qADhNMaf31e6BPZpSwJDgz0CbNkoFl69ixswefcJSjSAQYRX1YqqK\\\\\\\/YhOg9miOna9R04OXZNqmupYy5o5idhXz77LG0XpcumvhmR6cwownHQjjntyOBTXnXzouU5hHaCV5oGe1Zn6ARDw5y75ymj3kdaLkEt61Y7jM6XsUJXdcn49qHadt9J9F1oyJkh2nFk0Osp4NLRhGAjEr3WUP04L3fLtOMP10XSKFMbedjT5X39bRwJainuj5ZS0nBciSHjPrCANyM328O5G6ij0bkr6mJPdB55STdzQlyp56aGYd6+OWDbjY8pH5FSj4WzxofnP0RO\\\\\\\/mdn76Kv66bGzwCdu6tXXuvuMhjl6vtFeVIay2gl1d5qcKjo5gkYXoO4CrVt7pPxmnaNS\\\\\\\/vV6EiW2cLopPynMjpbHG5cD12DlqgrQyH25XCKzLtILTcO7SPx7iC+AqHgcLu6V\\\\\\\/H\\\\\\\/tzk3L2f95Gh8o1WmxdgvqLwTbnGaaZ7eI43f2q3A09xwytloIyjJMK9zR5QO9dg3gPnrDPuFjYmA2rELIZfXQiBgh34ZT6nfnp2JTMpwYfWWipapxA9I9wHWnCjs5qXav9sWkPcu75\\\\\\\/xbRuSxyNJeO\\\\\\\/YIdR5gTk5LKs33pDkkYCTbOBySgERI78KdJsMA2hABJdGZrRlX\\\\\\\/ZiTtvJKx1dbPbXzdOytd8YVAIprxvkxF0o4gKipH+dMXofrpzW0v4TSyz+dRYUB4igIM9GSbX9Ww1Qt5DlalvQUaPsnn0S8qM7xqW281nV00+D6vjRmf+7T+BpL3dUKvHCbGXewkchW2MuFGAEuioMP8xN+6YAmbKJ2hMPfn2gDupgFPDc2k2PjEo8A\\\\\\\/PHMnQlmA3HRUkzyj0iRq4W0HzwZmlpol1WeznSYG4cQkexlXxnauXOgKinUx0mcQsAZZCAfOHDu2XstI8gnWfvhUOnpjVqkRTdezOVwQLAlCMCoBfulTEkm25gP2+92bnvYKDOKtqeDNmT+0MUJDm0maie8XfYs4Mwu477DDWJVHPS2DlYVanu15zlceYcH9+N3eERc5fttb8fEtbX3\\\\\\\/whoTsw8dvt3NoUY\\\\\\\/rYKoNWi5vQuhek+xyqCkGsdRwltKFzNiaRiubH+3pwXuJdo\\\\\\\/fhwvWQluz1D75SLjk4Ii8WXUROFbmtZNoZmRBHL0ptlUfFKON7cTZLvS1fnXOnKI01N\\\\\\\/fI3bWfgQLP4UZ\\\\\\\/bC4C1oBgCxh7FprB76wCoO\\\\\\\/Dq6Gn3gOfbBICX\\\\\\\/239RuMhaIdVhw91fMYc\\\\\\\/F+2DEfWsX9Pq31jeUnbqypXF4lXFuBtOItmGxECw6aSZu74eahoTpoueepMNVECoVLAFx8INkKmo37OL2nDK6ha9ygqhCXs01j1YdWCiQ5Wha7m6TKZNDKW400UeMVlbcjwDNDxERHlOdeEkgV55tUDHS9tfJeUujIf8v5PNouraHoPpErkNZGiKavJug1rblKpuZPiKBCPc9ETQ2W7exvbdzNeCIkMyJ2t6gyU\\\\\\\/9SvtNTWfpxCYj\\\\\\\/Msi2LUdq5idZkqDq\\\\\\\/06YnbqWayQNMIWwHPj8Olm+uJCH6GWLMwf\\\\\\\/knYYP8sJkQw8PBrC0qWsSi3XqciYfNT7XCdyui9QqTVZMJDtCoftYahDqKdmqStwDTLIgFvqtdCaMZVgkkmKFCu+YbMzsNMMwDsCVm8wxrMiRWVOyfxeXWYAOBl0u2LRMp85ye54MXifub0kw77c86j4n7pWqEFRtvUKVfuai9muQ47cQ3UbDiAOJls2aq6EtJboV2gOrR5tMp+Z5EjgfM7ZCxj2QfWGfqSwttAjym+qCelPN2pPn7PWcWA1yrSpy2E9P+iJhAIByc4lHCC0pCjpxyFj0Ol4CUWp8ygb1droRuSpS\\\\\\\/BKuAyp2HE\\\\\\\/7e3Ft3wWVtfhgVNxejhPwDp4sGis3VsH4hU9NzEfFxd9npEvho6emMp\\\\\\\/p8zi1vILEEUj5hOpGdXt2lriGMuImF+ybUBamRpRKXtf8SJrKsSSwRm5tSrshnk3QjXUmh0flh+hmTDPaN\\\\\\\/4\\\\\\\/wdbJwEeY7pBco11czKieAKBj+hcWpEBU7xa9\\\\\\\/MJTqv\\\\\\\/m2KxblEkA2dpQcJdyrpeDPq6u6PLEc984XQVrA\\\\\\\/GBtv9vFYLBlCVOivyghl9tOisU7easIHOFCbmXagKX\\\\\\\/LohPhV5a+Bt4olRFmWmVkMS\\\\\\\/n\\\\\\\/mHKNwiOmuSRhoepqOZpKb7B\\\\\\\/gWZoTZlbUXE8REE+aDGvov32HnbnqzRiD1+riTDOrlHeHrwCja6s2tVfTflwPaMSKV+0tCX23y0D\\\\\\\/VDu3w3\\\\\\\/C01ve6QuFQ0UKwLRG+Dx8P5BPA\\\\\\\/+WZ912lJVsH0hxE6oHwb0qu1bP1OeeQBholLj\\\\\\\/kKvUGM35t9HucLA1fdjSD2Xki7scyXIkgQFFK09Or6U6Mea0tzQ0NL5\\\\\\\/8oh8yqpz1qOjXOBi8wtAmHbqxY4WaEwK9qNgAloqFkV4e9nlMmgfriL0u30fhAahJDR4ymoEKr2pnVD3eCQljGQ7LmhZQ1JxWEtjCDZOsb6k2H7vzFuI3zJQAdzpSAKP2iSyZL5yhXKtoyU9OgKuHmrVrkJ5\\\\\\\/u1eUWE+UiE5stj1NZWGISHyoE\\\\\\\/FuJTa5BPYktZiCeZTQifTjPXIPwR\\\\\\\/MwMTXsOaAf+ixBqSlNCKgYWkPi9PnueG3iJBuQKRlc\\\\\\\/lNMYXud0LnvuoAItvFR6XTOTfmpHoV0QtkgTQtNoQGxd1epdtKvTEKdD42bK8\\\\\\\/ARy9nHP0Rn3MUdbbxUn3F1acFXuXJsc9d+UMgz1tGR7asXKnl3pBlPNTlxHFe1QaEI7P\\\\\\\/iqxMZyTYImupozMR8u0qoEFtV6e+Z3mYEJc7Eb6h6sARxx1196ZmGgXpNGHm3OSSi0s4lMBHEd6k\\\\\\\/qXdUhA3\\\\\\\/RVJko07Q4dDnrECun+zrxjmnqQOWekcL+RzLOqL6+ZSPYl3qFobUwqnrJd2+0Cop+Snvh4mpKVUfMiSOdNZURsSJvFa71\\\\\\\/BN8ExnXyNCF\\\\\\\/tPV9wwbNksdArHqU61iTJBhxNIGg12IX3616spCY851Fb\\\\\\\/pOwK8eatiFOIicu8\\\\\\\/ewMNzkTDQ9rYI+IzmSz8SFxP+2Aa+vWatXttXb82bJ3xIHfcIQJn7Sr5N6lbepxjTEgMdGlF7bdA5uM5fTTAf3EuujS1e+xTcPgP\\\\\\\/bXaQbBuSylj\\\\\\\/tPPwOC9pBfhLJg0vwmB2s8\\\\\\\/4CGUVjuMcTBxZ9hJVyZiCubVqbYQRR4kceOwTZijpqcaU7659Y8HlhchLxr6jNJ6WIYD1i4j0RDKn5Faxu2HUWGvlTFbSV24q0NjQhyjbQgtUm1oxNuKEepneehQnxSxUzt8wuLX+8vBueMWQwXyXbSXihWRzzCH5LyUFCE2BUpWb+v25PL0BFADftj3YGDwS9GPuPuSoQkQa8rpJdz7udkOQIiV4dEvQ9Kl+MJZuQQBqhgy45SoCp6bZQGoaK52mlIIIxEchxApomEXsjsW0eOmknCG3dzhiAT0zx6JxRD325hML7AWIqgtZ+CT8l4AXO4IjAZndfQ16jMW+MP7vUt002kj5Yq5tOU6gtL0wcNLTI6ArfWJqiUEwxzXp445HnPsUVlbyLyZoTOHFG4NHdXx2SQNHVUiALWvxp1rMmKpC82aK0geFxEtqXCJjq6jC1DPaoHWti\\\\\\\/6yLq1jaqrVRuTv2Z\\\\\\\/S2qv1nIZhm24mggF5c8ocdd6mxqw5PC1nAEejtzLW6B7YERz71o0ZxKOgvWJ8v9FBbsHrwner8g4053+eKEhz97OV1dMt92Cj2SK6Ti3tqQvk+RIP9fmwx7W8MjdrILyKsNrK9yckRaPhDKBDuU\\\\\\\/jAubdRbN45PaoaffcwwR91A7YE6ZtAaMx\\\\\\\/tYP6Sojk6kFojJmmbc5K5Lhh\\\\\\\/W37IxuSPJ1y5dWonSCYBglZNSLUNQ25eE0MUCmOgE7nchN7lCFDioNMlykj22CxZtZEb6H1WxZc0Eo8RNtSKKg3wPf7ufxwwkrgO2j6pKRZsJmCRSCmXSrLtP+HBtfyPG\\\\\\\/pyb65oVb3fj1bOPnP275fQs7tY1dC499Iom+uGQJCYo\\\\\\\/qdriYY+TVCqxng0OMOT0BamjxhDsRpcITZRgKpKb+DlB0NNylJmeO+Lj7EzfTrmFYYwZjuk7MU4yKetsZXq\\\\\\\/NztuPL3bAkOn1WMz5LzzMrsMym55JEdALBiMBmenct\\\\\\\/OSHInKDvdflfdDAd+6XugF1FjJ0K8rmhesPfhrcuiZQ43PeHrt2hCO88l7wGHUyHqUjxLJwYx0CedJDQHrG3nwFMpAmOxC4f3L7vSjlHdFNEOH6kQLUQyR6+\\\\\\\/isMPwsoPiHDXyUdltuQw4VbGqOoGqgGNL8L5MOCLvLdsEO+KToh1q2SA+TgVEf3\\\\\\\/2ApIcOnoF\\\\\\\/2sZmD28oOntkezdrGknBRnVPkypY8alDaidUGwlsbDI5TXgAHEZAOryX3h9+3eoCinXFc8SRs88Hj7JZBe2J3jqr5ihH3zbuL6WMUJosOSZm4EYE6Gl6DfMAWvjAyT\\\\\\\/6gYgyhTDNBWjlrsoVKX0jZJmfQN89zUFoxLtrb3E9G+VIaylNt5no4AxaHRXIKLMgdAnCELiS\\\\\\\/tK7nI52bMMjAUQrQpjcV9eDD33P0JbLkKYcK78ieNLTapnltoXtmL4DWpUJyg4XJQo1t\\\\\\\/r01ZT\\\\\\\/MFA+e5nUF8qHcEN9kFrjV2j6C3Ylb4ZO7zf1n84VsefuqgvrlCdCjM9xUlvMJsaGWxKiL7f4qZnElT3Vk3lmam7a2yPo7VeeiNh2TvGn\\\\\\\/UxOhof3hTbCa2M6\\\\\\\/h5Qx3QFoeySOBKtWWd9JQk+tufiMYXbhrhqkYgCXuOPBVK2F5ijM+A+ErywlMTr9fH\\\\\\\/GzYBZK1t\\\\\\\/7rpe7F8Ql3saS57l1UG62edOx00uWZorez\\\\\\\/MJdnX9AvWGNAkHwH8uzU3HxvGxRw9ApfYMbs44kFPup+FRbxa+emzc5MXqjM6m1MyXF3dwyxjaCwPISfGBnfjYDgH8f4I\\\\\\\/3s3TgggApl3I4uai1UYN0WHAC0kJgjElc6dSzCq\\\\\\\/9+6knvLJz0HiyyFqVCUjiwvY3laryNESzI7wIgzliMNdr97MjaF67TR9EoFMw4uHUNbGz8PqBzxwBYCpRDyERVL\\\\\\\/GlQK2yEv6e1NF06hr+oiBk7PcLXIgXt1BR3LPYuKWpYhHgZfWfdcaBGJjkDVKErD7H1HpZ8TYrZRMuN3rT8DzcfmHKkO6+0PoAKOrOs6QPpKy9jLSCoqXP\\\\\\\/iRCWmgdjYRPYQxpC6++gWG7PJAkEuhmPJb33RKjOf2GYdEGvsOSDHh127tshGTG9m3\\\\\\\/1M3yvQ4K0cfTp\\\\\\\/HjtGiCX2eYkTuX\\\\\\\/T+XenXCtdtGYDNGoT48hrMJLx4zql\\\\\\\/9\\\\\\\/W+1adBoaiSOnIwYbdOhYJQaUh+JsT8gGdT9IMX02EOl3dQFhF2tNZNtBEYsR2gTfbDN0ktmYbLWENRv15h+edwa2HFvxuZIPvNFiQEfwu03L+UjIIZ7xCeApByOilCnGzHyYio4+8a5bVjLuV2TkHbBwIXR+CxuAEXvCZoUeghcDLBhEXInvvUuX4idwUbzLYh7ROkINgH7yVQwjieicPu8frM1rRtbGUF7pE\\\\\\\/dubsnjNYu56ZQQi7WYijXvf+Yl2LdxBrRlBgpHMbEnY+FKC3h3MSQFtC3CRopUjXZthDQzISJRyrSOKZu3d8U7FEEUMB52fG3pfhVKtSuyMz8hzwAnApG0jbk+oIJt\\\\\\\/B\\\\\\\/eW7blQmWAiH07tEg1BIZdCRcXxh6FZnHCw+bu66Wfdqwk5G1viSo\\\\\\\/JTinpFExZhLR1WrouipdezXIpOueZdmdKyUDoqU298Vg3IorRO51CDIRAMqmtbOvLIeZkYFbvtXOBoBfVXgnmy5YUJWmtfIBppk63p83fH6hNfR3rfGavb93yIJ3B03mqTrp+n\\\\\\\/DkPCNSjvQXIw+j1n0mxhDAqoSdsKdlZwmydKqI4rVobViCHc364CSJ511KwogCThQYRra6cXVtEitW68hEmV+jkLPtVfrdoUUGIffoYrZkGuOg3jZTQY65c9y5tBjDz9mN99YyYQPgmBmdJdi6ALSMmQntlb8mq22AtWDOI17YBDitBbdIW9t\\\\\\\/qtSzapcJcPFbPIKz0CMkesQjybCDcZqZWl6JA5xgT\\\\\\\/qJoFwl9+b362o2m4QE9TPug9QOqT276g+NkyALH0xwowLY0qTtvy8AEPglCtcy1F9DwIATMGZSSVe3+wt0sARas7fYKq0M2ELneIt6m5C5kFIoRRLxyziUDASdgSfRgJf5K0qjPtBWAPDFbOjHY06rJemiPpLMwDCclxgqIl7ZBTDaLZSnCWTDrtyyyqP9VxYwqyaNrycOME\\\\\\\/Qjl8W+ZoHIt2iknKPODfHOfdVHlJVhvRefOLxtsbUbh7qm5vd4BNPIrXeZqw8ua\\\\\\\/at83+zL60fVj9lDgKS6N2q6ISS0d9u11jcg+sc9ymRug766HkWfZ\\\\\\\/lLCxEj3355Rvu38gyreYtiR6PsJ3GxyDIJyCuMXzwp2S7G5vfuWYUy3PuvEDsKb7OG117io38vDtsx5O6\\\\\\\/\\\\\\\/rLio+u3qWUINYT90GWQuKh0YweqrY1KSavihq387uf5E\\\\\\\/wfP1aFgsfIp67qTH6hVActTRns+UwlcX8njP+QEl6\\\\\\\/8wbdIzfjCiVP5PCuBhD37dEgi85ervbAun77TlLXYvEExpPKUEdqtdkNt79HZBD5GmXguj800m5HvhzLSsn25cIvxUW97D+R\\\\\\\/x3\\\\\\\/9phJGFdjnU9BhdnLSVpMRTTeThDKGFyqyTzS9hFMTnLQbDJ2S4JhvXGKfqfEiz\\\\\\\/usgIv8R3J2s1p9m1I9GbKEjvhsF8DbGuHFn\\\\\\\/yHlv9\\\\\\\/k3RtbxCdFOeuv+AGSLzqM9uo\\\\\\\/9JYKdaa5\\\\\\\/v06vsf02VVKzjSKXOAdYGi3IfU8HzAlww++uEn5SX1Yc7s37ayId5vaN8NwSHaBscDCIz846Ifzg42HYsOPceBr5ehrCn3sEKRlbcOfPaY3x\\\\\\\/xh5GweK2zHLlCRfImYOL4anZw5U4ve9Ufd720PXh65a1z7Pwupt6yDOEfgXerBBJA5RYZAMF265HcSSdrLOurRE4Q7MzbUCynW7D9o3Mjrpif\\\\\\\/6cQh3TlFcSvzUgKZKoC+NSBWJjGzO0jpCBoPF\\\\\\\/793yVQCBbCNT30KX251oFn3kvatBZGf11IwJaF3m9ELbBo4qQmRTurIEnwQxBBi8MoL5a9dSEUv3c+UdhEgZ5AGwhBkusXTHbB3Wzg1K5DZtBtRQMFOlDXoAvpEnv7\\\\\\\/ewhgRpWPXuKiPvKShRXrglXm3\\\\\\\/kHNrH31f5zOfm8JtpECfnVI9Atztc6MkaJ0feRj8AXX5RzA2EE9tHPIYTCyjed2n2f\\\\\\\/dmW1MZfHbXNzQlALIlfExgy\\\\\\\/L0Ae4fKA1MlmZC4Jyz+T4HdjLHejtz8PXmw+7x5JW5Hx\\\\\\\/FG2AA\\\\\\\/CrstIIOaCLjV2bVR0haOOT7XKYrynRNicsOjsLpAoDY2VvJ8ZdYLB7I95P8EoJD0wgV7X5FWQCC8e7paVbLe8VMwYi8DRHEzwxgdo2cweRIx0s\\\\\\\/BD\\\\\\\/s0g12lOAAjbfONqvZoBUi9A1k1gHO17obKZ8uhe6nFD976IYPd1s5f68qh8HtnxmhuEYTQbnn6yzBYmW6gcjcpsJu2ilCPned2GTdbp4JYiEvwJFz+Gv7gplcr5rGwU5\\\\\\\/JFoGyuSZmqbUk5oXGyVRepKLXVr91mUv8\\\\\\\/QfJE4021nHYt2ocPYDYGDPR10mSuA55SkkTdEOukU68ku0DX6X3wzFLUG2FhVaymfqD+4RsXadgMQM0EBFb8cYLaRWNbMtPthVuY7PAgs0LLWYz9ckNhYhRiSkOfB20CYSJchsSyYqc+5igikVh6o2dUQVZnsLDLZn7oNuggxL4b3bPGfPg\\\\\\\/7tb2w3Jo94jxkj2Gmc7I+f4gnOmP0k5WjM0soR20w9IdbgjI5T7hY8ZbztapNAPRW9Qd16bQjGU56UdXFTZeFbQwLXKPY7kYh54WLqwQoCLcNNty7lGx1Eq+B97PgkOfg+CI0yo3PBzAM27k3F4G+iJw7CoSFYqJX\\\\\\\/izft78IMtMeoYyCsSqSZTZsBhB4IY6Y5C\\\\\\\/AwR+\\\\\\\/vdpsOLHfGEaKJYhGJ7BgrcUI32alHjSwHpGzWMxPigO5tKwNF30mGvFXmsUkG\\\\\\\/rJJoLcEEB+QKzy7EYj+AnL8BAKoqzSQEpyq2utbFjFYT52JPl3vxUPYmMK7UGVvylUpEcn197fLKDcm\\\\\\\/pb2mN80u6X5LFhILvN8jk+41ms3nRYtZsOctQZCKZttqoi7NCkdXBOh7ma9m0B\\\\\\\/96XmCfR+olcKe+gDe5p1zc1sxse7\\\\\\\/8YpKIy0J6nRBgRWIXw26mlkjIjpdEsas0yh+arSdn7kVIx4F7wuHSyWrxj69x52Oq6hX8vULPfod+EAOy\\\\\\\/gAaQGc1okHLN9PDNziqXLfevBB2KQC91sgJvBYYkwgpBsk0KG06RliSqhwJoCj2hUl4y\\\\\\\/0O7Sb\\\\\\\/2i1wQ3w7gkpxka0AcSAEJ1z2xfOadtTUG18OAl0jhW\\\\\\\/XhAIzHvlTGvA53IF0i\\\\\\\/GwiE193L+sMZAbx008FsdzpXESy1wMBCrGQR0x\\\\\\\/pV9yDJleCdHEoVuZiaWI1uYblFi8gevQkJ52UVomo+Ysc8133SLtWO+RcGCpjMmpwflIxEPLmpwcWhxzmpj3jhkmfdcB0fmdFSZef1IVR4t07mafG9RKtaPdwHsPBVF2KRTq0\\\\\\\/OwWbjeobpOOQU3FR2zj2TTxhUyRT7mT3HH6nfvjD5Bj+YlTejKvgEn+v6XJETcMhsmxZI4wWNYgfUw7iqrEKFVzAmPfMrwtGU9m77CNt3LCYbTWqIsCywiZ4j3ROeaoEpGLNj4lq\\\\\\\/+SfYtKe0OMl3RnBAMRSIkfqIjBHV\\\\\\\/HiGhkrPU0boPZBJneXIAqUKp8d7Lgt9w1a2jrvtiCbxm+Jfq7MRdxqBc6iChT4V41DoUtPW8yL5ml55Uk5Uhv1aCJSufbrZN3hzWoj0xxTnIrxMy1PAkdXRgmK5bZgmSQ0FNyMssYI2jznnyA+S0Wz84bgiOwr\\\\\\\/PltYq1GRa6Yrw67XmHjTrC4fGXeJdKej0jOabTKarxFBEjatOv6BdjR9cElcKuA9MDsFrZUF4ev+uKPsKnH1r8ptXqyxYt1oBuAA7bcXC4dGTysEJahIB3u9UFXWF8dlFKcCOmYvMhn7IzuMzUuq6yHeZNNgtvs7tcl3BxcYx5rwmQ+wRqTdBem8AqgqKD4PoLg3HCFCWKbcDVwW2MDeci+ixXPnnm1m7CXZ\\\\\\\/j9V4JAXtOEy1KNKkzB3efeeED0suWfpZkYALhOU3CTOT699FE35Ze0nr3tspAAdN7fxiISKgtv5W\\\\\\\/g9mCykSL7THsyKGy0ryI+x5wpaZ0PpY57bej4kTZAYVHDMJ+e1AQ1ODOvrm9Ipcy0rtsgCx5OjssZDyODclVgrIopEzRHf0bAkRIz0xADKzZJc5y8BTID5lNlCQJSAIDjN4PdsiCo6FDhcQhfOJz0oCFvwD0v2j87RQgomsKL40H4skQ+OdAGURPrZBWrhObBzdZI0mdg\\\\\\\/rbHrDtddmawJbjDWuLutSFh2NIfpuZivUrGkbW0KDxKU1MHaE1FfuzoGQrsghkNnGSru8fWha0jKqumgaQ0JQ7MI6jzGOAvTtuiGXtqRNduNcXtuz48sBrxMEFNRlLszBeoahs8kzD5U4phFVz0VDGJQAWS925P2TuU1d4Ko16TqOePfosRxNkWbe84IphN+1U2I4nNPbM4Sr2ro4Fgunn2vn5eAKlS5OW7PWYM8T5F9gE6IpFwVUYd+\\\\\\\/JfDrcjT1VfWS9RS3jI5+5XQdS3tIx0N5HkU1vcNzfX\\\\\\\/MXNTAjpQ215byeVNV0FbvcR\\\\\\\/V\\\\\\\/QZbbb0m4x57CkMj+tj3skVK35ydhYZQIg+prXflJeou1zmEwlpnyIrmmMlEOd4oqGmvXA+24nEuPnagT8KoZwoE4uGB66nkuu56bYDysln\\\\\\\/p7D+N12LKjNpdqgOmmE6ALdb0\\\\\\\/tHwfkRiZDx8p6uPiVFgzkD5AcIfjplhfHyxwCOHIpAFwy63AzX3mFuWs07O41ahrQxQpJ4D3dGw4c+0nCnDkm0Sk6Hi0EphBEN7w2fWbkFVpqpneFaSE0ddE0uQevgw\\\\\\\/YQ7WoQjVFufz651DoZ8hgyXET4t\\\\\\\/TZpQ2PgUxbz3peUtTOcAKAmA3pXeKRptOg4AahcAIWHA05SvP\\\\\\\/XzvMJGVRxD4VtYN6kiE3ENMjwVV+ftGf+xT+cEC1ld8jn4Pp0nFJrJud1kKatWsYEtaIarMHr8P3QrKc5FDYuNWkMuf+Q5Wcrq\\\\\\\/9+Ml971GHu3U31zTZwlyB76aW0ES9+kW6QVSFsM8Nw3o1m4NQbR77DGJYKo4wqYw15KU0imwvmJ5hJhihmU6ZFDIUn1X7OdCwbSWumT8+aiUbp7Od\\\\\\\/AU+Giib4tc04wf\\\\\\\/5TY2KZHqyxfpjSK0X2\\\\\\\/LfyD2eRuc7OfFl2NAQcgA1D94UKa3LPuYll353nDtkTviqFvKTtkkQsp13AwDDV4vu+ZlOE8Q0fxzxa0c4\\\\\\\/Ngujzk7kOn0XBcV\\\\\\\/IryEWtVj7z3ciOstDoOr0ovvksCUG+toHdTbZ0DjcPIwjTP0P7urmxd3rM+\\\\\\\/B3Q1y2EodlbuEAhF0tDUZk\\\\\\\/UPFseZW4+2lqpu2QW01gGccoNirqnmBqv4XmUV8kkRk5wpfemeDLmZK\\\\\\\/eQXsnAiBlUbq3\\\\\\\/iRJ4402U5fFwJapv4A4EpjKdY6j6Xb3XilbfALPsqlk3LsL+4nQbxW8d8GR0cmiAlNnvCww97M6kcGXyR5mHFmvYjqdst2afp3NsytkszAAPvt4dsUyX8HjEodnC03iuzCHZAc6JlarDK150M9MHGBQmyjnpKgv4MvQQwpDAG6WnZqAuD4MpSFl5qsVdkPqb8TlAJ4Ec3d4ODjaVx\\\\\\\/flPwB66SASz+4KEs9LjnepPd0OS7SoYSOZeQEklowg\\\\\\\/aO1XSrRs3Rwx\\\\\\\/BCfbdPBP2fgwKKjm158wO+GyH9qKOO39p3g\\\\\\\/TP8Xdu8RhtxHKA5ZwQUj5qLSVd234VAIsBN2hhn3fxzZXzxPqbVUvjSGxQbVcim0PaXd8OIMPtsRWQyZCNpMht\\\\\\\/OUak1rNjVINBb178u34Il3TuWRt0ALBsyEaQ8NJK7BaxJVSrV5kV2KfNUrggeJSGcpNXsOh9RbulOL2gMWbxLZPcjIPiQ4z4N1M7\\\\\\\/\\\\\\\/jCxgGFxb7WrpXMFiqOHN1+ZK3fyYjCD7dKYnzkz3r+f3w4\\\\\\\/2h4iy43iUVeAlRQoIYCLeCSZxfALA2uUB1Xr447sW+nlNizVe0z3EIH8vDY4Y1Sw3S\\\\\\\/y+OrprzNWvEGU+bQZghqv6rW60et3S1H2WWkgJRAZC2kdmfJ0pLkfgPO8lk7bW5l6JYfZeb7AtJqACeLhpg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022M+BouRatYTyzaKzdLySlaA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022757f42844dc497ceaf2eba7b62c94215576139b034aae5bf5aeabf47557a52db\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220f7vDlM17Vobvo1+oTF6Iw3H+OY7i9D8aEZErTcAHJYipfAuKqMav\\\\\\\/OyxYo3XNRdAnpyNKc6YfVnXd21QG+RS0FpzLkxmRzYLOiRt7oEPVWFUeudEjgjKe4XpgRPw8g1TYPkmowo96cEL6Ect4a2vFoYph7q7urUVnvNIPmGJ4LHDUfVtgmGZQE8j7dZPEUwAe8Aqwd7CRVLHJj7FKDa0EFsm6qTzGzWh1zFd9q86YRa73fj+JvTRJqk3sx684UNRcSDuiUJfrFfTicKezENq1iJ833VSGRPLEiNGE8KE7JQ62lC2r6Zwx9xdv9fjHlKpbAyv1et5tq2x87+yUQF+e2aTzVYsmfZYkE+nQUOIrX\\\\\\\/XjjjBjXzSDSfxEN4ch+Ehox+9X9PW5tiJPCc1rOLusjObJf5UPMCIyDSnrDXPOU96aI8RlQJ04d5DZemSjZgpffXKtmZyRHTkf9rW0AeCiEJ5lm\\\\\\\/yvhz64\\\\\\\/9n3irUG8H5IpliTMaaiCwBF0UeFpn9EuiyOjfYylIFTaDsvOWJJr9KwRS7Fs3cfKVjBoXK+tb1hSOmlKQkxzi035N5DlDIOXwcXx3Kgh52I2aV3nzxqL2Tp+L9guzxxlM9dY9CtZ64QB5UHbPWSRtzO+wDO41SCvtVtEbdzSkx6RqmLXUNP3z+AFJOaTqRMdtlz+ikIbucXwiYA3WdmT1jYB4e6zKATjSrtbkl2Jw8JOeort+n\\\\\\\/ODBpTlOUMbk\\\\\\\/8ZV6XA+jnYr7YkUV6L7xS3K51xqBaSTn+yDfWRIudg8rdOvj9aU+RosX+aU7uGNTU7tI1KXSi7qrtZYfj+o+dRV9dFjjMESX6wmfHRsZXJcP\\\\\\\/KawkMO+g3OFlH2PbBlJTt9QWbKAEmTpwuucDX75tfdibvj6iSnEO+p6YPt1YCWy+4K4WguiyzzcC9+VD\\\\\\\/jtWQqAW5b8dF4M2\\\\\\\/pixfwnaWCvsKthymDrwosG7i2ALZc55E1rkY+BZjFOM45JEyHrfUi4RpV0tFT6fLGfkC6FkUVDYOH0jPw+XaiioWS49XA+3uSA2kC8lt\\\\\\\/otXd2z0o0g7mSD2E+2kkn5vGPyet\\\\\\\/+G2o0BGYyfKoNcEiauivCDdfKuuW337FAGWoIsadcOcWLsSxwmMqtGL0oZnlCJHTachMlOpFWH\\\\\\\/EjikwcfI\\\\\\\/3AnM1RclVA4IVRybUi\\\\\\\/bB\\\\\\\/DKfSOgQu8hrHa87rwZU2XnvHd\\\\\\\/\\\\\\\/s1KwM3Xy4AT6W0tuuzrzvh+rCbxYKet3XdSvhgmpmZzNzl8yLOPF+WPsRVzDaOhG9IJbGu54JMeiywR232vr9ATz8U4w9mDQi0g8RYG2+ABJcd2lSaBFWJfgJvoIjrZyp8KD621klZR\\\\\\\/SEmh\\\\\\\/cUu+f4jpUcoo03XiwqmdI4b2oxSTwbRVyZuy7rw6iL6ehgt+CR3kxYlNHQJIiy+98Fr5aTZnFOjPFY1BV6RM7hHbSoSUvEQMobQnqj+spjAjEaYDFqUL61v4g6mV9zFST1gZwL657k4RK0FnvXE4I2VMvc5\\\\\\\/QFy08wsQU4U3jhmzfrzz8E3THoi9cUnndrmUWqMC+ZYQ53TVPm93KOYgNItxuNxzvZq3v16+n48PCDdZtwMeU8AoRGT6pKsoQVtuqmIUXHbkDc0ATkvs+x3mCDnnVQB8PzsXLm\\\\\\\/sNZnU3ZRoj0llhXPZ8kuDnU1sQrJCyVoCM\\\\\\\/UiJNo9V21PRBR5BM3\\\\\\\/QSMtnDv8xBkrzga7PffCE0hFSsyA7zQ6nRlR3Vash3T4A3RyBQO6hvfbOj9CMoytD4DmJlKmbvP9BAyzDcUsfxJ5mMrLI3rQ16rIs30oGSM2vKd8NAzf7imzDAK3X3I+Fxr+s2YC\\\\\\\/InA3b+WE7bQFWU7sMRjpzLZjmDAtQA\\\\\\\/kTGGtXhSa5Kd5XUE5FCTRfLUBNpLWy60DNjBcKcfPi2CMykBe6vwXXJpSgp41L8RR9pHqSC4KuM24bWadGAj7DuLkitrqKAUlk0QwBlj+Llr\\\\\\\/sdLlDQTtMN\\\\\\\/fd8druUWK2rMMCIeRw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q6yXyQytzzOSmPf3QWcF9A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022968fb7b44b5c62ea0c22695b184ad4df38587f2dcd1a38067b0ccca682a046e0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459455 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/617" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8vk1XAMR4oFkeSFR1GqcmyKWqnqeP0ahmrNlO6m5ULg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/617" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ksNbRCz9XWDGnLjp-u_oXt9UhkJz_XCW1uZEU_aFq4E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/607" title="Plastome Sequence Determination and Comparative Analysis for Members of the Lolium-Festuca Grass Species Complex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/633" title="Trans-Lesion DNA Polymerases May Be Involved in Yeast Meiosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459455" id="node459455" data-pisa="ggg;3/4/617" data-pisa-master="ggg;g3.112.004929" data-apath="/ggg/3/4/617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Maria José Truco</span>, <span class="highwire-citation-author" data-delta="1">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="2">Alexander Kozik</span>, <span class="highwire-citation-author" data-delta="3">Hans van Leeuwen</span>, <span class="highwire-citation-author" data-delta="4">John Bowers</span>, <span class="highwire-citation-author" data-delta="5">Sebastian Reyes Chin Wo</span>, <span class="highwire-citation-author" data-delta="6">Kevin Stoffel</span>, <span class="highwire-citation-author" data-delta="7">Huaqin Xu</span>, <span class="highwire-citation-author" data-delta="8">Theresa Hill</span>, <span class="highwire-citation-author" data-delta="9">Allen Van Deynze</span> and <span class="highwire-citation-author" data-delta="10">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">617-631;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004929 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459455" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maria José Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hans van Leeuwen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bvan%2BLeeuwen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Leeuwen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bvan%2BLeeuwen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Bowers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BBowers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowers%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BBowers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastian Reyes Chin Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes%2BChin%2BWo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes%2BChin%2BWo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Huaqin Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huaqin%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuaqin%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Theresa Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theresa%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheresa%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/4/617" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459455" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459455/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/617.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459455" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459455/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/617.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459455" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459455/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/4/617.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459455" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459455/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/4/617.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459455" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459455/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">We have generated an ultra-high-density genetic map for lettuce, an economically important member of the Compositae, consisting of 12,842 unigenes (13,943 markers) mapped in 3696 genetic bins distributed over nine chromosomal linkage groups. Genomic DNA was hybridized to a custom Affymetrix oligonucleotide array containing 6.4 million features representing 35,628 unigenes of <em>Lactuca</em> spp. Segregation of single-position polymorphisms was analyzed using 213 F<sub>7:8</sub> recombinant inbred lines that had been generated by crossing cultivated <em>Lactuca sativa</em> cv. Salinas and <em>L. serriola</em> acc. US96UC23, the wild progenitor species of <em>L. sativa</em>. The high level of replication of each allele in the recombinant inbred lines was exploited to identify single-position polymorphisms that were assigned to parental haplotypes. Marker information has been made available using GBrowse to facilitate access to the map. This map has been anchored to the previously published integrated map of lettuce providing candidate genes for multiple phenotypes. The high density of markers achieved in this ultradense map allowed syntenic studies between lettuce and <em>Vitis vinifera</em> as well as other plant species.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/lactuca-sativa" class="hw-term hw-article-keyword hw-article-keyword-lactuca-sativa" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Lactuca sativa</em></a></li><li class="kwd"><a href="/keyword/microarray" class="hw-term hw-article-keyword hw-article-keyword-microarray" rel="nofollow">microarray</a></li><li class="kwd"><a href="/keyword/linkage-analysis" class="hw-term hw-article-keyword hw-article-keyword-linkage-analysis" rel="nofollow">linkage analysis</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li></ul><p id="p-6">Cultivated lettuce, <em>Lactuca sativa</em>, is extensively grown worldwide and is one of the most valuable crops in the US [$2.4 billion in 2011 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">National Agricultural Statistics Service 2012</a>)], as well as an important part of the American diet (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">United States Department of Agriculture 2011</a>). It is a diploid (2n = 18) member of the Compositae with a genome size of 2.7 Gb (The Lettuce Genome Sequencing Consortium, unpublished data). <em>Lactuca serriola</em> is the likely wild progenitor of cultivated lettuce and a highly successful weedy species (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lindqvist 1960</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kesseli <em>et al.</em> 1991</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de Vries 1997</a>). <em>L. sativa</em> and <em>L. serriola</em> can be readily crossed, and there are no major barriers to gene flow between these species (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lindqvist 1960</a>). <em>L. serriola</em> has served as a source of genetic variation for crop improvement, particularly disease-resistance genes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crute 1988</a>). A variety of genetic and genomic resources have been developed for lettuce as part of the Compositae Genome Project, including a consensus genetic map that comprises more than 2700 loci based on a variety of mainly anonymous molecular markers from several segregating populations (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>) and extensive expressed sequence tag (EST) resources [<a href="http://compgenomics.ucdavis.edu">http://compgenomics.ucdavis.edu</a> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>)].</p><p id="p-7">During the past 30 years, genetic maps for a broad variety of species have been generated with greater resolution and saturation as the numbers of molecular markers increased due to improvements in marker technologies. However, high-density maps have primarily relied on anonymous polymerase chain reaction−based markers such as random amplified polymorphic DNA, sequence-related amplified polymorphisms, and amplified fragment-length polymorphisms (<em>e.g.</em>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">van Os <em>et al.</em> 2006</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Troggio <em>et al.</em> 2007</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sun <em>et al.</em> 2007</a>). Affymetrix microarrays provide a massively parallel technology for identification of polymorphism in genes. Thousands of loci can be genotyped in parallel by hybridizing DNA from each individual to an array comprised of millions of short oligonucleotide probes representing thousands of genes. Single-nucleotide polymorphisms (SNPs) or small insertion/deletions between the parents result in differences in the intensity of hybridization to the probes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Borevitz <em>et al.</em> 2003</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">West <em>et al.</em> 2006</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>). Identification of segregating polymorphisms can be enhanced over parental comparisons by exploiting the high level of replication in recombinant inbred line (RIL) populations in which each individual is a mosaic of the two parental haplotypes (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">West <em>et al.</em> 2006</a>).</p><p id="p-8">Microarray data have been used to evaluate genetic diversity, construct genetic maps, and analyze the genetic architecture of traits. <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Borevitz <em>et al.</em> (2003)</a> first demonstrated the detection of DNA sequence polymorphisms termed single-feature polymorphisms (SFPs) in a greater eukaryote that were subsequently used for studies of diversity in Arabidopsis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Borevitz <em>et al.</em> 2007</a>). <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">West <em>et al.</em> (2006)</a> developed a genetic map with 599 SFPs to dissect expression of quantitative trait loci (QTL) and <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Singer <em>et al.</em> (2006)</a> used an exon array for the generation of a high-resolution map comprising 676 genetic bins in Arabidopsis. In wheat, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bernardo <em>et al.</em> (2009)</a> used an Affymetrix array to identify and map 877 SFPs by using 96 RILs and <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Banks <em>et al.</em> (2009)</a> mapped 1035 SFPs by using a segregating population of doubled haploid lines. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Edwards <em>et al.</em> (2008)</a> used microarrays to analyze 880 insertion/deletion polymorphisms in rice and more recently <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zhao <em>et al.</em> (2011)</a> analyzed genetic variation in rice by using a 44,100 SNP array. <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Drost <em>et al.</em> (2009)</a> constructed a genetic map of more than 600 markers by using an interspecific cross in <em>Populus</em> spp. and <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Neves <em>et al.</em> (2011)</a> constructed a linkage map of 1845 genes in <em>Eucalyptus</em>. SFP-based maps also have led to identification of candidate genes and QTL (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Werner <em>et al.</em> 2005</a>; <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">West <em>et al.</em> 2006</a>).</p><p id="p-9">Genetic maps have been constructed for lettuce using several intraspecific (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Landry <em>et al.</em> 1987</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kesseli <em>et al.</em> 1994</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Waycott <em>et al.</em> 1999</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hayashi <em>et al.</em> 2008</a>) and interspecific populations (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Johnson <em>et al.</em> 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jeuken <em>et al.</em> 2001</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Syed <em>et al.</em> 2006</a>;) using a variety of molecular markers to provide increasingly dense maps as different technologies became available. Particular markers often only segregated in one or a few of these populations, which hampered the transfer of genetic and phenotypic information among the different maps. A single integrated map with 2744 loci was therefore constructed that combined seven separate maps into nine chromosomal linkage groups (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>). Recently, genotyping by sequencing was used to generate a map of 1113 SNP markers (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Truong <em>et al.</em> 2012</a>). However, the majority of these loci in all of these maps are anonymous random amplified polymorphic DNA− and amplified fragment-length polymorphism−derived markers and therefore do not provide gene-based functional information. Only the map constructed by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">McHale <em>et al.</em> (2009)</a> contains 772 EST sequence-based markers.</p><p id="p-10">One of the populations used to produce the integrated map of lettuce was a RIL population resulting from an interspecific cross between <em>L. sativa</em> cv. Salinas × <em>L. serriola</em> acc. US96UC23. This population has emerged as a core reference mapping one and now comprises 356 F<sub>7:8</sub> RILs. Sets of these RILs have been distributed to researchers worldwide for diverse studies. This population has also been genotyped for genes involved in disease resistance (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">McHale <em>et al.</em> 2009</a>) and candidate genes for horticultural and domestication traits (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lavelle 2009</a>) as well as phenotyped for multiple traits (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Argyris <em>et al.</em> 2005</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2008</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhang <em>et al.</em> 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hartman <em>et al.</em> 2012</a>). A large amount of genotypic and phenotypic information for this RIL population is available in the Compositae Genome Project database (<a href="http://compgenomics.ucdavis.edu">http://compgenomics.ucdavis.edu</a>).</p><p id="p-11">In this article, we report the construction and characterization of an ultradense, transcript-based, genetic map of lettuce based on this intensively studied <em>L. sativa</em> cv. Salinas × <em>L. serriola</em> acc. US96UC23 RIL population. Genomic DNA samples from 213 RILs were hybridized to a custom Affymetrix GeneChip (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>) to assay for polymorphisms in 35,628 unigenes derived from EST sequences of multiple <em>Lactuca</em> spp. Custom algorithms were used to identify single-position polymorphisms (SPPs) and to construct a genetic map composed of 12,841 unigenes in nine chromosomal linkage groups. This map provided a sufficient gene density for the identification of syntenic regions between lettuce and other plant species, such as <em>Vitis vinifera</em>, for which genome sequences were available.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-12">A population of 213 F<sub>7</sub> RILs from a cross between <em>L. sativa</em> cv. Salinas and <em>L. serriola</em> acc. US96UC23 (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Johnson <em>et al.</em> 2000</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>) was used to map SPPs. The cultivated parent, cv. Salinas (syn. Saladin), is a commercial crisphead cultivar that is widely grown in California and Europe and in the pedigree of many modern cultivars. Acc. US96UC23 is a wild accession of <em>L. serriola</em> that was collected in Davis, CA. <em>L. sativa</em> and <em>L. serriola</em> are closely related, have no barriers to hybridization, and may be con-specific (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Lindqvist 1960</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kesseli <em>et al.</em> 1991</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">de Vries 1997</a>). Care was taken during the generation of the RILs to minimize inadvertent selection and retain as much of the diversity in the population as possible; of the 375 F<sub>2</sub> plants that were initially self-pollinated, 356 resulted in RILs, 213 of which were randomly selected for use in this study.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation, labeling, and microarray hybridization</h3><p id="p-13">Total genomic DNA was extracted from bulked leaf tissue from over 40 F<sub>8</sub> plants from each of the 213 RILs and the parental lines and hybridized to the lettuce Affymetrix GeneChip following procedures described in <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> (2012)</a>. To summarize, 30 µg of DNA from each RIL<sub>7:8</sub> family and the two parental lines was sheared with DNase I to produce fragments between 50 and 250 bp in length, end-labeled, and hybridized to the lettuce GeneChip. Hybridizations to the 213 RIL DNA samples were carried out in duplicate, whereas the parental lines were each hybridized four times.</p></div><div id="sec-4" class="subsection"><h3>Array design and data analysis</h3><p id="p-14">Details of the design of the lettuce GeneChip and data analysis are described in <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> (2012)</a>. The lettuce GeneChip contains 6.4-million, 25 bp-long probes designed from 26,809 unigenes from cultivated lettuce (<em>L. sativa</em>) and an additional 8819 unique sequences from four other related <em>Lactuca</em> species (<em>L. serriola</em>, <em>L. saligna</em>, <em>L. virosa</em>, and <em>L. perennis</em>). This provided a total of 35,628 unigenes on the chip, of which 1184 were duplicated to assess the reproducibility of detecting polymorphisms (<a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>); for the purpose of the initial analysis (algorithm training), duplicated unigenes were treated as separate markers. The lettuce GeneChip has an average of 187 probes per unigene that are tiled on alternating strands every 2 bp. A total of 21,920,928 transcribed bases are covered by the probes. Algorithms previously developed by <a id="xref-ref-66-5" class="xref-bibr" href="#ref-66">West <em>et al.</em> (2006)</a> were modified and a new software suite was developed to take advantage of the tiled design of the lettuce GeneChip that provides up to 13 overlapping probes at each position in a unigene covered by probes. We also took advantage of the redundancy that was provided by the genetic replication in the RIL population: hybridization intensity of each probe was measured 426 times corresponding to the 213 RILs replicated twice. This software suite for analyzing the lettuce GeneChip is available at <a href="http://chiplett.ucdavis.edu">http://chiplett.ucdavis.edu</a>.</p><p id="p-15">The raw data (CEL files) for hybridizations of 434 chips (2 × 213 RILs plus 4 replications of each parent) were obtained from Affymetrix GCOS (Affymetrix GeneChip Operating Software) v 1.3 software and subjected to robust multiarray analysis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Irizarry <em>et al.</em> 2003</a>), background correction, and normalization using the Aroma Package of Bioconductor and R (<a href="http://www.braju.com/R/">http://www.braju.com/R/</a>; <a href="http://www.r-project.org/">http://www.r-project.org/</a>). We subjected normalized data to quality control analysis by performing cluster analysis by using R on a subset of more than 20,000 probes that were considered informative based on a preliminary study. The two replications of each RIL clustered together for all but four RILs (~1% of all RILs); this lack of clustering between replicate hybridizations for these four RILs could have been the result of a variety of technical events and therefore another hybridization was performed for each of these RILs. Background correction, normalization, and QC analysis were repeated for chips of only those RILs that did not cluster together; in all cases the repeated chips clustered with one of the replicates and allowed the identification of the discordant chips, which were then removed from the original set of 434 chips and data from the replacement chips substituted. This provided a high confidence dataset of 434 chips on which a final background correction, normalization, and cluster analysis for all the data were performed.</p></div><div id="sec-5" class="subsection"><h3>Detection of SPPs</h3><p id="p-16">The normalized data for ~6.4 million lettuce probes were analyzed using a custom Perl package (<a href="http://chiplett.ucdavis.edu/public/Manuals/manuals.php">http://chiplett.ucdavis.edu/public/Manuals/manuals.php</a>) to identify differences in hybridization in a sliding 2-bp window in regions of all unigenes covered by probes. Probes were assigned to GC bins based on their relative GC content and, within each GC bin, weighted mean hybridization intensity was calculated for each chip, hereafter called the reference set value. The hybridization intensity of each probe was weighted to take into consideration the distance of the central nucleotide of the probe to the position of the 2-bp window being interrogated using an empirically derived weighting curve that resulted in greater weight being given to central positions (<a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>). The reference set value was calculated for all GC bins and all chips so that the data from each chip were comparable with a chip’s own reference set. A background signal for each chip was also calculated based on anti-genomic probes within each GC bin (<a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>). A probe was excluded from further analysis if its hybridization intensity was less than the 90 percentile of the hybridization intensity of all anti-genomic probes (see <a id="xref-ref-48-9" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>) in its specific GC bin. The weighted hybridization intensities were used to calculate the SPPdev at each 2-bp position covered by probes for all the chips using the equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/617/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/4/617/embed/mml-math-1.gif"/></noscript></span></span>For each 2-bp position, the SPPdev values for all the chips were then sorted from low to high. Our algorithm searched for a bimodal distribution using a user-defined difference (<em>e.g.</em>, 0.2). If the gap between two consecutive values was larger than the specified difference, an SPP was declared at that 2-bp position. If no bimodal distribution was found, the position was considered non-polymorphic and the next 2-bp position was analyzed. This was carried out iteratively for all positions on all contigs. A genotype was assigned to that 2-bp position based on the SPPdev values relative to the parental chips; the “A” genotype was assigned to alleles from cv. Salinas and the “B” genotype to alleles from acc. US96UC23. Both replicates of an individual RIL had to have the same genotype for an assignment to be made. If the replicates were inconsistent, the haplotype was treated as missing data. When contiguous 2-bp positions had bimodal distributions, they were summarized as one SPP range.</p><p id="p-17">Summarized SPPs were filtered based on several criteria including minimum average SPPdev value, maximum percentage of missing data of a haplotype, minimum number of valid probes (those hybridizing to more than 90% of the anti-genomic probes), and minimum number of bases spanning an SPP. Consensus haplotypes were called from concatenated SPPs using a Python script (<a href="http://code.google.com/p/xuhu-rwm-map/source/browse/#svn/trunk">http://code.google.com/p/xuhu-rwm-map/source/browse/#svn/trunk</a>). If the SPP calls within a unigene varied within a RIL, consistent with recombination within the unigene, the two most likely consensus haplotypes were calculated for these loci (<a href="http://code.google.com/p/xuhu-rwm-map/wiki/SplitSPPs">http://code.google.com/p/xuhu-rwm-map/wiki/SplitSPPs</a>).</p></div><div id="sec-6" class="subsection"><h3>Genetic linkage analysis</h3><p id="p-18">Marker names were converted to four letter codes for simplicity and compatibility with mapping software; a conversion table for each four letter code and unigene ID is available at <a href="http://chiplett.ucdavis.edu/map_2012">http://chiplett.ucdavis.edu/map_2012</a>. SPPs were clustered into linkage groups using MadMapper (<a href="http://cgpdb.ucdavis.edu/XLinkage/MadMapper/">http://cgpdb.ucdavis.edu/XLinkage/MadMapper/</a>) after filtering for missing data (genotype calls missing in no more than 25 RILs, 12%) and allelic distortion (the ratio between minor allele and major allele frequency had to be &gt; 0.2). The linkage groups were assigned using previously mapped loci to ensure consistency with the published integrated map of lettuce (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>). Pairwise recombination values among all markers were then calculated for each linkage group using Joinmap v.4 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Van Ooijen 2006</a>). Markers with zero recombination among them were considered to belong to the same genetic bin (<em>e.g.</em>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/004929SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/FigureS1.pdf">Figure S1</a>). However, the assignment of markers into genetic bins was not always unequivocal due to missing data. Adjacent markers had to be assigned to a super-bin when there was no recombination for some combinations of markers but there was recombination between other markers within the same super-bin (<em>e.g.</em>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/FigureS2.pdf">Figure S2</a>). The marker with the least missing data points was chosen to represent each bin or super-bin for genetic mapping purposes.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462630/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462630/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Pairwise recombination values among markers in super-bin 36 in linkage group 4</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">Markers representing each bin and super-bin, plus singletons (markers exhibiting recombination with all other markers in the linkage group) were mapped using RECORD_WIN (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">van Os <em>et al.</em> 2005</a>), MSTMap (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Wu <em>et al.</em> 2008</a>), and Joinmap v4 (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Van Ooijen 2006</a>). RECORD_WIN was run with the settings: 30-cM gap size for all the linkage groups except LG4 and LG8 for which the gap size was 50 cM, Kosambi mapping function and 0.1 as the fraction of recombination allowed. MSTMap was run with the objective function COUNT to estimate map order and Kosambi mapping function to assign genetic distances. Due to the large number of markers segregating in the population, in Joinmap we used maximum likelihood mapping with the map order optimization parameters: chain length of 5000, initial acceptance probability 0.250, cooling control parameter of 0.001 stopping after 30,000 chains without improvement. For the estimation of recombination frequencies, we used 1000 for the length of burning-in chain, 4 for the number of EM cycles and 1000 for the chain length per EM cycle. Joinmap calculates genetic distances using Haldane’s mapping function which assumes no cross-over interference and therefore results in larger genetic maps than those generated using the Kosambi mapping function (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Salomé <em>et al.</em> 2011</a>). The marker orders generated with the three programs were examined using CheckMatrix (<a href="http://www.atgc.org/XLinkage/MadMapper/">http://www.atgc.org/XLinkage/MadMapper/</a>; <a href="http://code.google.com/p/atgc-map/">http://code.google.com/p/atgc-map/</a>) for inconsistencies.</p><p id="p-20">Once the order of the unigenes was determined, regions of heterozygosity in individual RILs were identified by looking for contiguous regions with abnormally high rates of missing data compared with the same region in other RILs or high frequencies of apparent double cross-overs. Homozygous haplotypes were distinguished from heterozygous haplotypes using a sliding window of three markers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/FigureS3.pdf">Figure S3</a>). Heterozygous regions were defined as those where heterozygous haplotype windows were clustered.</p><p id="p-21">Allelic segregation ratios were calculated for all unigenes. Goodness-of-fit χ<sup>2</sup> tests were computed to the expected 1:1 Mendelian ratio. In each generation of selfing, the proportion of heterozygotes is reduced by half; therefore, the residual heterozygosity of an F<sub>7</sub> RIL population is expected to be 1.56%. Because the RIL algorithm does not account for heterozygous alleles, this allelic class was not considered in the computations of goodness-of-fit. Markers were considered distorted at a level of significance of <em>P</em> &lt; 0.05.</p><p id="p-22">We estimated genome-wide linkage disequilibrium (LD) using the graphical genotyping program GTT 2.0 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Van Berloo 2008</a>). To decrease the computational load, we analyzed 609 unigenes distributed at 5 cM intervals throughout the nine linkage groups. We used r<sup>2</sup> as a measure of LD (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hill and Robertson 1968</a>).</p></div><div id="sec-7" class="subsection"><h3>Data curation</h3><p id="p-23">To facilitate data visualization, we generated a quasi-genomic sequence based on genetic distance and displayed the resultant chromosomal groups using GBrowse (<a href="http://www.Gbrowse.org">http://www.Gbrowse.org</a>). Each unigene was assigned coordinates based on the cumulative genetic distance from the beginning of the linkage group. Unigenes were separated by ‘N’s; the number of ‘N’s between markers was scaled to reflect the total number of cM (1585) and the physical size of the genome (2.7 Gb). Files were generated in GFF3 format and uploaded into GBrowse. For unigenes with two consensus haplotypes, both haplotypes were displayed if they mapped to different linkage groups (seven cases) or they were more than 5 cM apart in the same linkage group (four cases); otherwise, the position of only one of the haplotypes was displayed.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Identification of SPPs</h3><p id="p-24">Analysis of hybridization values for the 213 RILs at all 2-bp windows covered by probes identified a total of 303,359 SPPs corresponding to 14,693 polymorphic loci of the 35,628 unigenes on the chip. This represents a frequency of 1 SPP per 72 bp assayed over a total of 21,920,928 bp of coding sequence addressed by the probes on the GeneChip. After consolidating contiguous SPPs into single consensus calls, we found that only 1137 polymorphic loci (7.7%) had a single SPP. Most polymorphic loci (13,124; 89.3%) had complete consensus of allele calls across the unigene within each RIL, the only differences among the RILs being due to missing data; single consensus haplotypes were designated for these loci. Therefore, the vast majority of the unigenes had a single haplotype in each RIL. However, a few loci (3%) varied in the allele calls for different SPPs within one or more RILs, which is consistent with recombination within the unigene; the two most likely consensus haplotypes were calculated for these unigenes accounting for 1569 additional loci. The average number of SPPs per polymorphic unigene was 20.6 with a median value of 12 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Loci with more than 25 (~8%) missing data points or with a minor allele frequency divided by major allele frequency of less than 0.2 (which equals to a minor allele frequency of 0.167) were not used for the initial genetic map construction. This resulted in a total of 13,788 polymorphic loci that were utilized for the initial mapping.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distribution of number of SPP calls per polymorphic unigene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequency distribution of number of SPP calls per polymorphic unigene.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F1.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F1.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462791" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-25" class="first-child">Frequency distribution of number of SPP calls per polymorphic unigene.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Genetic map construction</h3><p id="p-26">Approximately half of the loci (53.2%) grouped into super-bins, 27% grouped into bins, and 19.8% were singletons (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The average number of loci per super-bin was 9.18 with a median value of 7. The average number of loci per bin was 2.82 with a median value of 2. The genetic maps generated using three different mapping software programs (RECORD_Win, MSTMap, and Joinmap) were very similar. Discrepancies in order only occurred among very closely linked unigenes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The sizes of the linkage groups generated with RECORD_Win (2974 cM total) and MSTMap (2948 cM total) were very similar; Joinmap generated longer linkage groups (3483 cM total). None of the three maps was obviously superior to the others. We selected the genetic map generated using RECORD_Win as the reference map for the remainder of this study.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/462744/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/462744/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distribution of bins, super-bins and singletons by linkage group</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dot plots of marker orders comparing the maps of LG1 generated by Joinmap, Record_Win, and MSTMap." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dot plots of marker orders comparing the maps of LG1 generated by Joinmap, Record_Win, and MSTMap.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F2.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F2.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462668" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-27" class="first-child">Dot plots of marker orders comparing the maps of LG1 generated by Joinmap, Record_Win, and MSTMap.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">All 13,788 markers were initially grouped into 10 linkage groups using MadMapper. Because some unigenes had been previously mapped using Illumina GoldenGate SNP assays (<a href="http://cgpdb.ucdavis.edu/GeneticMapViewer/display/">http://cgpdb.ucdavis.edu/GeneticMapViewer/display/</a>) (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">McHale <em>et al.</em> 2009</a>), we were able to assign each of the linkage groups to the nine chromosomal linkage groups and orient them to be consistent with the published consensus integrated map (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) (<a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>). Markers were distributed throughout the linkage groups without any breaks (<em>e.g.</em>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) except for linkage group 3. Linkage group A was provisionally assigned to linkage group 3 on the basis of markers that had been previously mapped in an interspecific <em>L. saligna</em> × <em>L. sativa</em> cross (<a id="xref-ref-54-7" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>). Subsequent analysis using Record_Win joined LGA with LG3 separated by a gap of 21.3 cM.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map showing linkage along linkage group 1 displayed using Checkmatrix. Each chromosome is plotted in the linear order of the loci against itself and the degree of linkage between each locus indicated by a color. Red indicates 0.51 cM. The first column at the right lists the locus name. The second column indicates the genetic distance. The red and blue bars at the bottom indicate the allelic composition at each locus. Red indicates Salinas allele; blue, L. serriola allele; yellow , distortion toward the Salinas allele; light blue, distortion toward L. serriola allele. Figures for each linkage group can be viewed at high magnification at http://chiplett.ucdavis.edu/map_2012. The insert in the bottom right corner illustrates a high magnification view of a subset of the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heat map showing linkage along linkage group 1 displayed using Checkmatrix. Each chromosome is plotted in the linear order of the loci against itself and the degree of linkage between each locus indicated by a color. Red indicates &lt;0.1 cM; yellow, 0.1&#x2212;0.4 cM; green, 0.4&#x2212;0.49 cM; blue, &gt;0.51 cM. The first column at the right lists the locus name. The second column indicates the genetic distance. The red and blue bars at the bottom indicate the allelic composition at each locus. Red indicates Salinas allele; blue, &lt;em&gt;L. serriola&lt;/em&gt; allele; yellow , distortion toward the Salinas allele; light blue, distortion toward &lt;em&gt;L. serriola&lt;/em&gt; allele. Figures for each linkage group can be viewed at high magnification at http://chiplett.ucdavis.edu/map_2012. The insert in the bottom right corner illustrates a high magnification view of a subset of the data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F3.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F3.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462644" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-29" class="first-child">Heat map showing linkage along linkage group 1 displayed using Checkmatrix. Each chromosome is plotted in the linear order of the loci against itself and the degree of linkage between each locus indicated by a color. Red indicates &lt;0.1 cM; yellow, 0.1−0.4 cM; green, 0.4−0.49 cM; blue, &gt;0.51 cM. The first column at the right lists the locus name. The second column indicates the genetic distance. The red and blue bars at the bottom indicate the allelic composition at each locus. Red indicates Salinas allele; blue, <em>L. serriola</em> allele; yellow , distortion toward the Salinas allele; light blue, distortion toward <em>L. serriola</em> allele. Figures for each linkage group can be viewed at high magnification at <a href="http://chiplett.ucdavis.edu/map_2012">http://chiplett.ucdavis.edu/map_2012</a>. The insert in the bottom right corner illustrates a high magnification view of a subset of the data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Residual heterozygosity in an F<sub>7</sub> RIL population is expected to be 1.56% (assuming a reduction of half in each generation of selfing). This would result in more than 46,000 heterozygous genotype calls for the 13,788 markers in the 213 RILs. Using a sliding window of three mapped markers (see <em>Materials and Methods</em>), we detected 1,031,346 windows indicative of heterozygosity indicating <em>ca</em>. 3% heterozygous positions. These regions of heterozygosity detected with the GeneChip were validated using SNPs previously mapped using Illumina GoldenGate Assays (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">McHale <em>et al.</em> 2009</a>). The heterozygous regions detected using the two different technologies were concordant (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/FigureS4.pdf">Figure S4</a>). More than 94% of the genome was heterozygous in at least one of the 213 RILs; 89% of the genome was heterozygous in two or more RILs.</p><p id="p-31">The total map length generated by all three mapping programs was approximately twice or more the size of previous genetic maps constructed using the same population (RECORD_Win = 2,995 <em>vs.</em> 1,505 cM; <a id="xref-ref-54-8" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">McHale <em>et al.</em> 2009</a>). Heterozygous regions resulted in abnormally high rates of missing data and/or high frequencies of apparent double cross-overs (single SPP in a RIL that differed from flanking loci; see <em>Materials and Methods</em>). Erroneously called double cross-overs would have artificially inflated genetic distances. To avoid this, we masked heterozygous regions by designating them as missing data and recalculated the marker order and genetic distances. This reduced the number of apparent double recombinants by 46% and the map size to 2247.5 cM. We then masked the remaining double recombinants because true double recombinants will be extremely rare with this marker density and again recalculated the gene order and genetic distances. These two revisions reduced the total map length to 1579 cM (48% reduction) with 13,788 markers and only <em>ca</em>. 4% missing data. This revised map is close to the total of 1505 cM observed in the previous studies (<a id="xref-ref-54-9" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>; <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">McHale <em>et al.</em> 2009</a>). Map reduction occurred proportionally in all linkage groups. These recalculations occasionally changed the order of closely linked markers but marker order was generally maintained.</p><p id="p-32">Significant segregation distortion (<em>P</em> &lt; 0.05) from the expected segregation ratio of 1:1 indicative of unintended selection during the generation of the RILs (<em>e.g.</em>, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Salomé <em>et al.</em> 2011</a>) was observed in all linkage groups (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Regions of linkage groups 3, 4, 5, 7, and 8 were significantly distorted (<em>P</em> &lt; 0.01). The distortions on linkage groups 3, 7, and bottom part of 8 were toward an excess of the <em>L. sativa</em> allele. On linkage groups 4, 5, top 8, and 9 the distortions were toward an excess of the <em>L. serriola</em> allele (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Overall, segregation distortion slightly favored alleles from the domesticated parent over those from the wild parent (11 <em>vs.</em> 6% respectively).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of 831 loci exhibiting distorted segregation on the nine chromosomes of lettuce. Orange and green lines indicate distortion at a significance level of P &lt; 0.05. Red and blue lines indicate distortion at a significance level of P &lt; 0.01. Distortions toward the L. sativa cv. Salinas and L. serriola alleles are in red/orange and blue/green, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of 831 loci exhibiting distorted segregation on the nine chromosomes of lettuce. Orange and green lines indicate distortion at a significance level of &lt;em&gt;P&lt;/em&gt; &lt; 0.05. Red and blue lines indicate distortion at a significance level of &lt;em&gt;P&lt;/em&gt; &lt; 0.01. Distortions toward the &lt;em&gt;L. sativa&lt;/em&gt; cv. Salinas and &lt;em&gt;L. serriola&lt;/em&gt; alleles are in red/orange and blue/green, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F4.medium.gif" width="210" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F4.medium.gif" width="210" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462645" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-33" class="first-child">Distribution of 831 loci exhibiting distorted segregation on the nine chromosomes of lettuce. Orange and green lines indicate distortion at a significance level of <em>P</em> &lt; 0.05. Red and blue lines indicate distortion at a significance level of <em>P</em> &lt; 0.01. Distortions toward the <em>L. sativa</em> cv. Salinas and <em>L. serriola</em> alleles are in red/orange and blue/green, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">A region of distortion (<em>P</em> &lt; 0.01) toward the Salinas allele in LG3 (see lower region of LG3 in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) included the gap of 21.3 cM in the initial map (see above). Three hundred and forty-three markers had been excluded from the initial mapping analysis because the frequency of the minor allele was below the threshold of ~17% (see <em>Materials and Methods</em>). Of these skewed loci, 261 were distorted toward the Salinas allele. We conducted a further round of grouping with MadMapper using all loci that mapped in LG3 plus all those loci that had been previously excluded with distortion toward the Salinas allele. One hundred fifty-five skewed loci grouped with LG3 and after a new round of mapping using Record_Win were linked inside and around the 21.3 cM gap, filling it completely. The final corrected map has 1585 cM and 13,943 loci corresponding to 12,842 unigenes distributed in nine linkage groups corresponding to the nine chromosomes of lettuce (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Raw data for the map construction and maps of all the linkage groups can be viewed at <a href="http://chiplett.ucdavis.edu/map_2012">http://chiplett.ucdavis.edu/map_2012</a>.</p><p id="p-35">To search for potential epistatic interactions resulting in unexpected combinations of unlinked loci (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wade <em>et al.</em> 2001</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Isobe <em>et al.</em> 2012</a>), we estimated genome-wide LD for 609 unigenes distributed approximately every 5 cM throughout the genome. No r<sup>2</sup> values greater than 0.5 were observed among pairs of loci that mapped to different linkage groups. Greater r<sup>2</sup> values were only observed among loci mapping together in the same linkage group. Therefore, there was no evidence for particular combinations of alleles at loci in different chromosomes being selected for or against during the generation of the RILs.</p><p id="p-36">The distribution of polymorphic loci with respect to genetic distance was analyzed for each linkage group. We determined the number of polymorphic loci within each 5 cM interval along the total length of each linkage group. The distribution of number of loci in each interval was compared to the Poisson distribution (P(x) = e<sup>-µ</sup>µ<sup>x</sup>/x!, where µ is the average number of loci per interval over the entire map and x in the actual number of loci) expected if loci were distributed randomly. Observed and expected frequencies were compared using a χ<sup>2</sup> test (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Troggio <em>et al.</em> 2007</a>). Polymorphic loci were not randomly distributed along the linkage groups (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The distribution of loci was different for each linkage group (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/FigureS5.pdf">Figure S5</a>). Linkage groups 1 and 2 had a greater average number of loci and the greatest deviation from their respective means.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average distribution of loci in intervals of 5 cM along LG1. Deviation from the mean value for LG1 (x = 51) is shown by bars. Similar analyses for the other LGs are shown in Figure S5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average distribution of loci in intervals of 5 cM along LG1. Deviation from the mean value for LG1 (x = 51) is shown by bars. Similar analyses for the other LGs are shown in Figure S5.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F5.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F5.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462803" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-37" class="first-child">Average distribution of loci in intervals of 5 cM along LG1. Deviation from the mean value for LG1 (x = 51) is shown by bars. Similar analyses for the other LGs are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/FigureS5.pdf">Figure S5</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Validation of SPPs using Illumina GoldenGate SNP assays</h3><p id="p-38">To validate the SPP allele calls we used data for unigenes present on the lettuce Affymetrix GeneChip that had also been mapped on 188 RILs using Illumina GoldenGate Assays (<a id="xref-ref-37-7" class="xref-bibr" href="#ref-37">McHale <em>et al.</em> 2009</a>). Allele calls were compared individually for 96 unigenes distributed throughout the nine chromosomal linkage groups. The allelic calls for 95 of the 96 unigenes were 99.9% identical; of 17,860 comparisons between the two marker technologies (188 RILs × 95 unigenes), there were only 22 (0.1%) discordant allelic calls. The remaining unigene, <em>LSRGCAE05-1</em>, encodes a NBS-LRR candidate disease resistance protein (Genbank number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY153843&amp;atom=%2Fggg%2F3%2F4%2F617.atom">AY153843</a>) and nearly 50% of its allele calls were discordant between the two technologies. Resistance genes are frequently members of multi-gene families (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Michelmore and Myers 1998</a>). Detection of gene assignment is likely to be problematic for such highly polymorphic multigene families with both of these technologies (Hill <em>et al.</em> 2012); it is possible that each technology detected polymorphism among paralogs located at different chromosomal positions.</p></div><div id="sec-12" class="subsection"><h3>Distribution of recombination events</h3><p id="p-39">The high density of markers mapped in this study allowed the identification of nearly all recombination events present in each RIL (<em>e.g.</em>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). We calculated the number of cross-overs per RIL using markers spaced every 5 cM; this varied from 10 to 56 with a mean of 28.8. This is close to the value of 35 that is expected assuming one cross-over per chromosome arm per meiosis and a reduction each generation of selfing in the ability to detect cross-overs by half due to homozygosity. The number of cross-overs detected per chromosome ranged from 0 to more than 8; the number of cross-overs was correlated with genetic size (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Chromosomes with no apparent cross-overs were found for all the linkage groups but were more frequent for the smaller linkage groups. We performed a QTL analysis of cross-over frequency to determine whether QTL influencing recombination rate were segregating in this population; however, we did not detect any such QTL.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of haplotypes along linkage group nine displayed vertically for 213 RILs using Checkmatrix. The RIL families are arranged along the x-axis and the loci displayed in linear order along the y-axis. Red indicates L. sativa haplotype; blue, L. serriola haplotype. Gray, no allele called; missing data. The first column at the right lists the locus name. The second column indicates the genetic distance. The number of double cross-overs and the proportions of each haplotype are shown below each RIL. Displays for each linkage group can be viewed at high magnification at http://chiplett.ucdavis.edu/map_2012. The insert in the bottom right corner illustrates a high magnification view of a subset of the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of haplotypes along linkage group nine displayed vertically for 213 RILs using Checkmatrix. The RIL families are arranged along the x-axis and the loci displayed in linear order along the y-axis. Red indicates &lt;em&gt;L. sativa&lt;/em&gt; haplotype; blue, &lt;em&gt;L. serriola&lt;/em&gt; haplotype. Gray, no allele called; missing data. The first column at the right lists the locus name. The second column indicates the genetic distance. The number of double cross-overs and the proportions of each haplotype are shown below each RIL. Displays for each linkage group can be viewed at high magnification at http://chiplett.ucdavis.edu/map_2012. The insert in the bottom right corner illustrates a high magnification view of a subset of the data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F6.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F6.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462638" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-40" class="first-child">Distribution of haplotypes along linkage group nine displayed vertically for 213 RILs using Checkmatrix. The RIL families are arranged along the x-axis and the loci displayed in linear order along the y-axis. Red indicates <em>L. sativa</em> haplotype; blue, <em>L. serriola</em> haplotype. Gray, no allele called; missing data. The first column at the right lists the locus name. The second column indicates the genetic distance. The number of double cross-overs and the proportions of each haplotype are shown below each RIL. Displays for each linkage group can be viewed at high magnification at <a href="http://chiplett.ucdavis.edu/map_2012">http://chiplett.ucdavis.edu/map_2012</a>. The insert in the bottom right corner illustrates a high magnification view of a subset of the data.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency distribution of the number of cross-overs. Numbers of cross-overs detected per RIL (0 to &gt;8) are displayed chromatically for each linkage group, which are organized by genetic size (longest at top, shortest at bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequency distribution of the number of cross-overs. Numbers of cross-overs detected per RIL (0 to &gt;8) are displayed chromatically for each linkage group, which are organized by genetic size (longest at top, shortest at bottom).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F7.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F7.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462795" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-41" class="first-child">Frequency distribution of the number of cross-overs. Numbers of cross-overs detected per RIL (0 to &gt;8) are displayed chromatically for each linkage group, which are organized by genetic size (longest at top, shortest at bottom).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Chromosomal distribution of diversity</h3><p id="p-42">We assessed the distribution of genetic diversity along each linkage group by using a panel of 52 accessions that was composed mostly of <em>L. sativa</em> cultivars encompassing diversity in each of the cultivated types of lettuce (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1/TableS1.pdf">Table S1</a>) (<a id="xref-ref-48-10" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>) and 10,243 unigenes distributed throughout the nine linkage groups. The average number of haplotypes detected per unigene across all the linkage groups was 3.4 with a median of 3.0 and a range from 2 to 22. We estimated diversity by summing the number of haplotypes detected in the diversity panel over each cM and averaging the values in sliding windows of 5 cM. Regions of high or low diversity were detected in all LGs (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Across most of the genome the pattern of diversity was remarkably parallel for all lettuce types with <em>L. serriola</em> being the most diverse as expected. In a few regions, <em>e.g.</em>, on LG7, the level of diversity in the cultivated types was much lower than in the <em>L. serriola</em>, indicating a loss of diversity during domestication (turquoise ellipse; <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). Some regions of low diversity were specific to a particular type, <em>e.g.</em>, butterhead types had two regions of low diversity in LG1 (orange ellipses; <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). Five regions with high levels of diversity in all types of lettuce similar to the level of polymorphism observed in <em>L. serriola</em> corresponded to five clusters of NBS-LRR-encoding genes responsible for pathogen recognition that had been mapped previously in LG1, 2, 3b, 4, and 8; only the cluster of NBS-LRR-encoding genes that did not show high levels of polymorphism mapped to LG9 (purple ellipses, <a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8</a>) (<a id="xref-ref-37-8" class="xref-bibr" href="#ref-37">McHale <em>et al.</em>, 2009</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of haplotype diversity across the nine linkage groups for butterhead (yellow line), romaine (blue line), crisphead (green line), and leafy (red line) lettuce types and between L. sativa and L. serriola (gray line). Each y-axis indicates the average number of polymorphic loci in a sliding 5-cM window (from 0 to 16). Turquoise ellipse: example of region with low level of diversity in cultivated types relative to L. serriola. Orange ellipses: examples of regions with low diversity specific to one type, in this case butterhead. Purple ellipses: regions associated with clusters of NBS-LRR-encoding genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of haplotype diversity across the nine linkage groups for butterhead (yellow line), romaine (blue line), crisphead (green line), and leafy (red line) lettuce types and between &lt;em&gt;L. sativa&lt;/em&gt; and &lt;em&gt;L. serriola&lt;/em&gt; (gray line). Each y-axis indicates the average number of polymorphic loci in a sliding 5-cM window (from 0 to 16). Turquoise ellipse: example of region with low level of diversity in cultivated types relative to &lt;em&gt;L. serriola&lt;/em&gt;. Orange ellipses: examples of regions with low diversity specific to one type, in this case butterhead. Purple ellipses: regions associated with clusters of NBS-LRR-encoding genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F8.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F8.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462716" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8 </span> <p id="p-43" class="first-child">Distribution of haplotype diversity across the nine linkage groups for butterhead (yellow line), romaine (blue line), crisphead (green line), and leafy (red line) lettuce types and between <em>L. sativa</em> and <em>L. serriola</em> (gray line). Each y-axis indicates the average number of polymorphic loci in a sliding 5-cM window (from 0 to 16). Turquoise ellipse: example of region with low level of diversity in cultivated types relative to <em>L. serriola</em>. Orange ellipses: examples of regions with low diversity specific to one type, in this case butterhead. Purple ellipses: regions associated with clusters of NBS-LRR-encoding genes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Map curation</h3><p id="p-44">To provide convenient access to the unigenes on the ultradense map of lettuce, a quasi-genomic sequence of lettuce was generated based on genetic distances and visualized using GBrowse (<a href="http://lgr.genomecenter.ucdavis.edu">http://lgr.genomecenter.ucdavis.edu</a>). The 13,943 mapped markers were concatenated in their genetic order for each of the nine linkage groups. Markers within a bin were ordered randomly. Bins were separated by ‘N’s scaled to reflect genetic distances between them with 1 cM represented by 1730 kb for a total of 1585 cM spanning 2.7 Gb. Unigenes were assigned coordinates and displayed relative to information for each unigene such as SNPs, SPPs and the sequences of the GeneChip probes.</p></div><div id="sec-15" class="subsection"><h3>Syntenic analyses</h3><p id="p-45">We analyzed synteny between lettuce and sequenced eudicots including grape (<em>Vitis vinifera</em>), poplar (<em>Populus trichocarpa</em>), Arabidopsis (<em>A. thaliana</em>), and potato (<em>Solanum tuberosum</em>). The 13,788 loci from the lettuce map were compared to genes predicted in the 12x assembly of grape (<a href="http://www.genoscope.cns.fr/externe/Download/Projets/Projet_ML/data/12X/assembly/">http://www.genoscope.cns.fr/externe/Download/Projets/Projet_ML/data/12X/assembly/</a>), a revision of the 8x published assembly by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jaillon <em>et al.</em> (2007)</a>. Similar genes were identified as the best TBLASTX hit with e &lt; 1 × e<sup>-6</sup>. Of the 12,658 mapped lettuce unigenes, 10,552 matched chromosomally located genes of grape and 564 matched genes that were on non-chromosomally assigned assemblies of the grape genome. Many small regions of synteny were identified (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). Each of 38 chromosomal arms of grape is present three times in lettuce. This has resulted in <em>ca</em>. 114 major segments of synteny between the two species, approximately 11 per lettuce chromosome. Lettuce is, therefore, an ancient hexaploid compared with grape. In contrast, each region of the lettuce genome matched only one region in the grape genome indicating that grape has not undergone additional polyploidy events since its last common ancestor with lettuce. The gene order for some complete chromosomes of grape has been maintained intact in lettuce, whereas for other chromosomes only chromosomal arms have been maintained intact. Comparisons with poplar, Arabidopsis, and potato (data not shown) revealed more complex patterns of synteny because these species have undergone one or more additional polyploidy events in their lineages since their last common ancestor with lettuce. Arabidopsis has undergone two additional cycles of polyploidy followed by differential gene loss, along with a relatively large number of chromosomal fusions, translocations and inversions (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Yogeeswaran <em>et al.</em> 2005</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lysak <em>et al.</em> 2006</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hu <em>et al.</em> 2011</a>), with the result that remaining segments of synteny are far shorter and more complex than those observed in the comparison to grape.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/4/617/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dot plot of the syntenic positions of genes in grape and lettuce. The nine chromosomes of lettuce are ordered along the x-axis and the 17 chromosomes of grape along the y-axis. The positions of 10,552 loci mapped in lettuce on the grape genome are displayed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2141852782" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dot plot of the syntenic positions of genes in grape and lettuce. The nine chromosomes of lettuce are ordered along the x-axis and the 17 chromosomes of grape along the y-axis. The positions of 10,552 loci mapped in lettuce on the grape genome are displayed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/4/617/F9.medium.gif" width="314" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9&#x2002;" src="https://www.g3journal.org/content/ggg/3/4/617/F9.medium.gif" width="314" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/4/617/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/4/617/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/462616" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9 </span> <p id="p-46" class="first-child">Dot plot of the syntenic positions of genes in grape and lettuce. The nine chromosomes of lettuce are ordered along the x-axis and the 17 chromosomes of grape along the y-axis. The positions of 10,552 loci mapped in lettuce on the grape genome are displayed.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-47">The genetic map described in this article represents the most dense map developed for lettuce to date and will be the foundation for fine mapping of quantitative trait loci, identifying candidate genes, map-based cloning and genome assembly efforts, as well as marker-assisted selection in breeding programs. The initial map that we constructed was based on 13,788 loci that spanned 2,974 cM. This was nearly double the size of the previously published consensus map that was developed from seven intra- and interspecific mapping populations using a total of 560 markers that segregated in two or more populations (<a id="xref-ref-54-10" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>). Genotyping errors can cause increases in genetic map sizes; every 1% error rate in a marker adds <em>ca</em>. 2 cM to the map (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cartwright <em>et al.</em> 2007</a>). By masking heterozygous regions and removing double-crossover events, and adding a few heavily skewed loci the total map distance was reduced to 1585 cM, close to the previously observed value of 1505 cM for this population (<a id="xref-ref-54-11" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>). A small increase in size was also observed in a SFP-based linkage map for Arabidopsis as compared to maps generated previously with fewer markers [from 397.1 to 422.5 cM (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Singer <em>et al.</em> 2006</a>) or to 625 cM (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Truong <em>et al.</em> 2012</a>)].</p><p id="p-48">The analysis of 213 RILS with 13,943 SPP markers provided a highly accurate map in which most of the recombination events in each of the RILs were detected. SPP marker calls were validated by their consistency within unigenes and genetic bins as well as concordance using a different SNP marker technology. The similarity of the maps generated with three different mapping programs provides further support for the quality of the map. Recently the IN algorithm that is a part of RECORD was reported to be the most accurate for estimating gene orders in dense maps (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wu <em>et al.</em> 2011</a>). This finding is consistent with high levels of polymorphism segregating in this wild by domesticate population and no regions having identity by descent.</p><p id="p-49">This map is one of the densest meiotic maps for any eukaryotic species. With more than 10,000 markers, it is the first ultradense map for a member of the Compositae and complements the recent 10,000 marker map for sunflower (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bowers <em>et al.</em> 2012</a>). Although genetic maps with several thousand markers are available for a broad range of plant and animal species, few maps have more than 5000 markers, and most of the dense maps have been based on anonymous, polymerase chain reaction−based markers (reviewed in <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">van Os <em>et al.</em> 2006</a>), although the situation is changing rapidly with the advent of genotyping by sequencing (reviewed in <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Davey <em>et al.</em> 2011</a>). In contrast to these maps, our map of lettuce is based on 12,842 transcribed unigenes and therefore has much greater information value.</p><p id="p-50">Assuming lettuce has <em>ca</em>. 45,000 genes (Lettuce Genome Sequencing Consortium, unpublished data), this map provides the genetic locations of approximately one quarter to one third of the protein-encoding genes expressed in lettuce. Given that the GeneChip detects polymorphisms in ESTs, this map captures a large fraction of the genes determining the phenotypic differences segregating in this population. Details of the mapped genes are provided through a GBrowse-based web browser (<a href="http://lgr.genomecenter.ucdavis.edu">http://lgr.genomecenter.ucdavis.edu</a>). Ultradense maps avoid the necessity for separate bulked segregant analyses to saturate individual target regions (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Michelmore <em>et al.</em> 1991</a>). High densities of markers also allow chromosome landing (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tanksley <em>et al.</em> 1995</a>) rather than walking using bacterial artificial chromosomes or genome assemblies for map-based cloning. Given a lettuce genome size of 2.7 Gb (The Lettuce Genome Sequencing Consortium, unpublished data), there is a marker every <em>ca</em>. 200 kb on average, although the relationship between physical and genetic distances varies greatly (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fu <em>et al.</em> 2002</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sourdille <em>et al.</em> 2004</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wang <em>et al.</em> 2011</a>). The average genetic distance between markers (bins) was 0.4 cM. The analysis of 213 RILs (representing &gt;2100 meioses) provided a genetic resolution of less than three mapped genes on average per genetic bin facilitating the identification of candidate genes. The analysis of such a large number of RILs also had the advantage in that more than 94% of the genome was heterozygous in at least one RIL. This provides suitable starting material for fine mapping genes by saturating target regions with recombinants using flanking heterozygous markers with minimal genetic noise from other regions of the genome.</p><p id="p-51">The genetic structure of this RIL population is similar to that observed in RIL populations of other species. Although this map was generated from a wild by domesticate cross, there was only limited segregation distortion and no major blocks of repressed recombination indicative of structural rearrangements. This provides further support for the conspecific relationship of <em>L. sativa</em> and <em>L. serriola</em> and the domestication of cultivated lettuce from the weedy <em>L. serriola</em> (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Lindqvist 1960</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Kesseli <em>et al.</em> 1991</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">de Vries 1997</a>). The clustering of loci exhibiting segregation distortion suggests that the distortion has a genetic basis rather than being an experimental artifact. Seventeen percent of the genome exhibited segregation distortion with 11% and 6% elevated for the domesticated and wild haplotypes respectively. However, segregation distortion was never absolute; all haplotypes were recovered, albeit sometimes infrequently. The regions exhibiting segregation distortion detected here are similar to those detected in other lettuce populations (<a id="xref-ref-54-12" class="xref-bibr" href="#ref-54">Truco <em>et al.</em> 2007</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Uwimana <em>et al.</em> 2012</a>). Segregation distortion can be due to several reasons, including preferential gametic transmission (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Moyle and Graham 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2010</a>), genetic elements that enhance their own transmission (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rebollo <em>et al.</em> 2012</a>), inadvertent selection due to differences in viability and fitness over multiple generations during RIL construction due to one or the other allele of a specific gene conferring a selective advantage (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Salomé <em>et al.</em> 2011</a>), and the effects of deleterious allelic combinations generated during inbreeding (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">MacDaniel and Shaw 2007</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Salomé <em>et al.</em> 2011</a>). Considerable care was taken to minimize selection during the generation of RILs used in this study. Also, there was no evidence for deleterious allelic combinations at unlinked loci.</p><p id="p-52">Syntenic comparisons of sequenced plant genomes provide insights into the evolutionary events that shaped the lettuce genome as well as allow inferences as to the positions of genes that have yet to be mapped. Our data on the triplication of the lettuce genome relative to <em>Vitis vinifera</em> is indicative of two genome duplication events in the Compositae lineage since its divergence from <em>V. vinifera</em> and confirms the inference based on EST analyses (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barker <em>et al.</em> 2008</a>). The same result was observed in tomato, which is in the Asterid I group <em>vs.</em> the Asterid II group, which contains lettuce (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">The Tomato Genome Consortium 2012</a>); this finding suggests that the triplication event occurred before the radiation of the Asterids but after their divergence from the lineage that gave rise to <em>V. vinifera</em>. Extensive rearrangements and only short regions of synteny were expected because <em>Lactuca</em> spp. are rapid cycling, highly successful weeds. This is similar to Arabidopsis and the <em>Brassica</em> species relative to species such as <em>V. vinifera</em> and poplar that reproduce clonally and have longer sexual generation times (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Yogeeswaran <em>et al.</em> 2005</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lysak <em>et al.</em> 2006</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Hu <em>et al.</em> 2011</a>).</p><div id="sec-17" class="subsection"><h3>Future directions</h3><p id="p-53">This map provides many polymorphic candidate genes for map-based cloning. Several studies are in progress to clone and characterize genes involved in domestication traits such as branching, spininess, leaf shape, seed physiology, and flowering time. In addition, the lettuce genome has recently been sequenced and annotation is underway (Lettuce Genome Consortium, unpublished data); this map provides the opportunity to validate and refine the genome assemblies and place them in chromosomal linkage groups.</p><p id="p-54">The same lettuce GeneChip used to produce this ultradense map has been used for studies of diversity in 52 genotypes of cultivated lettuce (<em>L. sativa</em>) and several wild relatives of lettuce (<a id="xref-ref-48-11" class="xref-bibr" href="#ref-48">Stoffel <em>et al.</em> 2012</a>). Coupling the ultradense genetic map with the diversity data will allow the deployment of the information generated in a single population to other populations for which the parental lines were included in the diversity panel, which will facilitate the identification of polymorphic markers in regions of interest and further assist in the identification of candidate genes for traits of scientific interest and/or agronomic importance (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Argyris <em>et al.</em> 2010</a>).</p><p id="p-55">The ultradense map reported here was based on SPPs detected using a custom microarray. However, microarray technology is being superseded by genotyping by sequencing the whole or reduced components of the genome (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2009</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Xie <em>et al.</em> 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Poland <em>et al.</em> 2012</a>). The gene space of 96 RILs from the same <em>L. sativa</em> x <em>L. serriola</em> population is being sequenced and the data from the lettuce GeneChip will be used to validate the mapping of the SNPs detected by sequencing.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We thank Smitha Mathrakott for technical and Belinda Martineau for editorial assistance. This project was supported by the University of California UC Discovery Program Bio04-10470 with matching funds from Rijk Zwaan BV, Enza Zaden BV, and Vilmorin Co as well as funding from the National Science Foundation Plant Genome Program grant # DBI0421630 in support of the Compositae Genome Project.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.004929/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Lettuce sequences used for array construction are available at: <a href="http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&amp;id=4235">http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&amp;id=4235</a> and <a href="http://www.ncbi.nlm.nih.gov/nucest/?term=txid4235[Organism:exp]">http://www.ncbi.nlm.nih.gov/nucest/?term=txid4235[Organism:exp]</a>.</p></li><li class="fn" id="fn-4"><p id="p-57"><em>Communicating editor: K. Dawe</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-11-09"><span class="received-label">Received </span>November 9, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-02-07"><span class="accepted-label">Accepted </span>February 7, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Truco <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.1" data-doi="10.1007/s00122-005-0066-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argyris</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Still</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Quantitative trait loci associated with seed and seedling traits in <em>Lactuca</em></span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1365</span>–<span class="cit-lpage">1376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DArgyris%26rft.auinit1%253DJ.%26rft.volume%253D111%26rft.issue%253D7%26rft.spage%253D1365%26rft.epage%253D1376%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bassociated%2Bwith%2Bseed%2Band%2Bseedling%2Btraits%2Bin%2BLactuca.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0066-4%26rft_id%253Dinfo%253Apmid%252F16177902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0066-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16177902&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233323900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.2" data-doi="10.1104/pp.108.125807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argyris</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Still</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic variation for lettuce seed thermoinhibition is associated with temperature-sensitive expression of abscisic acid, gibberellins, and ethylene biosynthesis, metabolism, and response genes</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">926</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.108.125807%26rft_id%253Dinfo%253Apmid%252F18753282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDgvMi85MjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Argyris</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McHale</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A gene encoding an abscisic acid biosynthetic enzyme (<em>LsNCED4</em>) colocates with the high temperature germination locus Htg6.1 in lettuce (<em>Lactuca</em> sp.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D122%26rft.spage%253D95%26rft_id%253Dinfo%253Apmid%252F20703871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20703871&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.4" data-doi="10.3835/plantgenome2009.02.0009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single-feature polymorphism mapping in bread wheat</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D2%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2009.02.0009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2009.02.0009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.5" data-doi="10.1093/molbev/msn187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matvienko</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple paleopolyploidizations during the evolution of the Compositae reveal parallel patterns of duplicate gene retention after millions of years</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2445</span>–<span class="cit-lpage">2455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn187%26rft_id%253Dinfo%253Apmid%252F18728074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8yNDQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.6" data-doi="10.1186/1471-2164-10-251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowden</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Discovery and mapping of single feature polymorphisms in wheat using Affymetrix arrays</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBernardo%26rft.auinit1%253DA.%2BN.%26rft.volume%253D10%26rft.spage%253D251%26rft.epage%253D251%26rft.atitle%253DDiscovery%2Band%2Bmapping%2Bof%2Bsingle%2Bfeature%2Bpolymorphisms%2Bin%2Bwheat%2Busing%2BAffymetrix%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-251%26rft_id%253Dinfo%253Apmid%252F19480702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19480702&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.7" data-doi="10.1101/gr.541303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plouffe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Large-scale identification of single-feature polymorphisms in complex genomes</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.541303%26rft_id%253Dinfo%253Apmid%252F12618383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvNTEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.8" data-doi="10.1073/pnas.0705323104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazen</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide patterns of single-feature polymorphism in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">12057</span>–<span class="cit-lpage">12062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0705323104%26rft_id%253Dinfo%253Apmid%252F17626786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI5LzEyMDU3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.9" data-doi="10.1534/g3.112.002659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachlava</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunick</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of a 10,000 locus genetic map of the sunflower genome based on multiple crosses</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DBowers%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D721%26rft.epage%253D729%26rft.atitle%253DDevelopment%2Bof%2Ba%2B10%252C000%2BLocus%2BGenetic%2BMap%2Bof%2Bthe%2BSunflower%2BGenome%2BBased%2Bon%2BMultiple%2BCrosses%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.002659%26rft_id%253Dinfo%253Apmid%252F22870395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNy83MjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.10" data-doi="10.1534/genetics.106.063982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cartwright</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutin</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic mapping in the presence of genotyping errors</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2521</span>–<span class="cit-lpage">2527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.063982%26rft_id%253Dinfo%253Apmid%252F17277374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzI1MjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crute</span>  <span class="cit-name-given-names">I. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The impact of breeding on pest and disease control in lettuce</span>. <abbr class="cit-jnl-abbrev">Asp. Appl. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsp.%2BAppl.%2BBiol.%26rft.volume%253D17%26rft.spage%253D305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.12" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.13" data-doi="10.1023/A:1008611200727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Vries</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Origin and domestication of <em>Lactuca sativa</em> L</span>. <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D44%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1008611200727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1008611200727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WQ45100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.14" data-doi="10.1111/j.1365-313X.2009.03828.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drost</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novaes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boaventura-Novaes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benedict</span>  <span class="cit-name-given-names">C. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A microarray-based genotyping and genetic mapping approach for highly heterozygous outcrossing species enables localization of a large fraction of the unassembled <em>Populus trichocarpa</em> genome sequence</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DDrost%26rft.auinit1%253DD.%2BR.%26rft.volume%253D58%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1067%26rft.atitle%253DA%2Bmicroarray-based%2Bgenotyping%2Band%2Bgenetic%2Bmapping%2Bapproach%2Bfor%2Bhighly%2Bheterozygous%2Boutcrossing%2Bspecies%2Benables%2Blocalization%2Bof%2Ba%2Blarge%2Bfraction%2Bof%2Bthe%2Bunassembled%2BPopulus%2Btrichocarpa%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03828.x%26rft_id%253Dinfo%253Apmid%252F19220791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03828.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220791&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266925800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.15" data-doi="10.1186/1746-4811-4-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janda</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaikwad</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Development and evaluation of a high-throughput, low-cost genotyping platform based on oligonucleotide microarrays in rice</span>. <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BMethods%26rft.aulast%253DEdwards%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D4%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DDevelopment%2Band%2Bevaluation%2Bof%2Ba%2Bhigh-throughput%252C%2Blow-cost%2Bgenotyping%2Bplatform%2Bbased%2Bon%2Boligonucleotide%2Bmicroarrays%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-4-13%26rft_id%253Dinfo%253Apmid%252F18510771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-4-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18510771&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.16" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.17" data-doi="10.1073/pnas.022635499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dooner</span>  <span class="cit-name-given-names">H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recombination rates between adjacent genic and retrotransposon regions in maize vary by 2 orders of magnitude</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.022635499%26rft_id%253Dinfo%253Apmid%252F11792865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OS8yLzEwODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.18" data-doi="10.1111/j.1752-4571.2012.00240.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooftman</span>  <span class="cit-name-given-names">D. A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uwimana</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Wiel</span>  <span class="cit-name-given-names">C. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smulders</span>  <span class="cit-name-given-names">M. J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic regions in crop–wild hybrids of lettuce are affected differently in different environments: implications for crop breeding</span>. <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">629</span>–<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEvol%2BAppl%26rft.aulast%253DHartman%26rft.auinit1%253DY.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253D629%26rft.epage%253D640%26rft.atitle%253DGenomic%2Bregions%2Bin%2Bcrop-wild%2Bhybrids%2Bof%2Blettuce%2Bare%2Baffected%2Bdifferently%2Bin%2Bdifferent%2Benvironments%253A%2Bimplications%2Bfor%2Bcrop%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2012.00240.x%26rft_id%253Dinfo%253Apmid%252F23028403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2012.00240.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028403&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.19" data-doi="10.1139/G08-077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aoyama</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Still</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quantitative trait loci associated with lettuce seed germination under different temperature and light environments</span>. <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">928</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DHayashi%26rft.auinit1%253DE.%26rft.volume%253D51%26rft.issue%253D11%26rft.spage%253D928%26rft.epage%253D947%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bassociated%2Bwith%2Blettuce%2Bseed%2Bgermination%2Bunder%2Bdifferent%2Btemperature%2Band%2Blight%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1139%252FG08-077%26rft_id%253Dinfo%253Apmid%252F18956026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/G08-077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18956026&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.4.617.20" data-doi="10.1371/journal.pone.0056200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Chin-Wo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide analysis of <em>Capsicum annuum</em> germplasm reveals marked changes in diversity from pungent to non-pungent pepper</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e56200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De56200%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0056200%26rft_id%253Dinfo%253Apmid%252F23409153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0056200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23409153&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.21" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.22" data-doi="10.1038/ng.807"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pattyn</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The <em>Arabidopsis lyrata</em> genome sequence and the basis of rapid genome size change</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHu%26rft.auinit1%253DT.%2BT.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D476%26rft.epage%253D481%26rft.atitle%253DThe%2BArabidopsis%2Blyrata%2Bgenome%2Bsequence%2Band%2Bthe%2Bbasis%2Bof%2Brapid%2Bgenome%2Bsize%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.807%26rft_id%253Dinfo%253Apmid%252F21478890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478890&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.23" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.24" data-doi="10.1093/biostatistics/4.2.249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobbs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beazer-Barclay</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonellis</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Exploration, normalization, and summaries of high density oligonucleotide array probe level data</span>. <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.stitle%253DBiostatistics%26rft.aulast%253DIrizarry%26rft.auinit1%253DR.%2BA.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D264%26rft.atitle%253DExploration%252C%2Bnormalization%252C%2Band%2Bsummaries%2Bof%2Bhigh%2Bdensity%2Boligonucleotide%2Barray%2Bprobe%2Blevel%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252F4.2.249%26rft_id%253Dinfo%253Apmid%252F12925520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjc6IjQvMi8yNDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.25" data-doi="10.1534/g3.112.002600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hisano</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative genetic mapping and discovery of linkage disequilibrium across linkage groups in white clover (<em>Trifolium repens</em> L.)</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DIsobe%26rft.auinit1%253DS.%2BN.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D607%26rft.epage%253D617%26rft.atitle%253DComparative%2BGenetic%2BMapping%2Band%2BDiscovery%2Bof%2BLinkage%2BDisequilibrium%2BAcross%2BLinkage%2BGroups%2Bin%2BWhite%2BClover%2B%2528Trifolium%2Brepens%2BL.%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.002600%26rft_id%253Dinfo%253Apmid%252F22670230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNS82MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.26" data-doi="10.1038/nature06148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaillon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Policriti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clepet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJaillon%26rft.auinit1%253DO.%26rft.volume%253D449%26rft.issue%253D7161%26rft.spage%253D463%26rft.epage%253D467%26rft.atitle%253DThe%2Bgrapevine%2Bgenome%2Bsequence%2Bsuggests%2Bancestral%2Bhexaploidization%2Bin%2Bmajor%2Bangiosperm%2Bphyla.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06148%26rft_id%253Dinfo%253Apmid%252F17721507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17721507&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249724800041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.27" data-doi="10.1007/s001220100657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeuken</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Wijk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peleman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linhout</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">An integrated interspecific AFLP map of lettuce (<em>Lactuca</em>) based on two <em>L. sativa</em> x <em>L. saligna</em> F<sub>2</sub> populations</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">638</span>–<span class="cit-lpage">647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D103%26rft.spage%253D638%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220100657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220100657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000171240000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.28" data-doi="10.1007/s001220051581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Wijk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peleman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lettuce, a shallow-rooted crop, and <em>Lactuca serriola</em>, its wild progenitor, differ at QTL determining root architecture and deep soil water exploitation</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1066</span>–<span class="cit-lpage">1073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D101%26rft.spage%253D1066%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000165482700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.29" data-doi="10.1139/g91-065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Variation at RFLP loci in <em>Lactuca</em> spp. and the origin of cultivated lettuce (<em>L. sativa</em>)</span>. <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D34%26rft.spage%253D430%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg91-065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g91-065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Analysis of a detailed genetic linkage map of <em>Lactuca sativa</em> (lettuce) constructed from RFLP and RAPD markers</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1435</span>–<span class="cit-lpage">1446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKesseli%26rft.auinit1%253DR.%2BV.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1435%26rft.epage%253D1446%26rft.atitle%253DAnalysis%2Bof%2Ba%2Bdetailed%2Bgenetic%2Blinkage%2Bmap%2Bof%2BLactuca%2Bsativa%2B%2528lettuce%2529%2Bconstructed%2Bfrom%2BRFLP%2Band%2BRAPD%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F7912217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzE0MzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrara</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A genetic map of lettuce (<em>Lactuca sativa</em> L.) with restriction fragment length polymorphism, isozyme, disease resistance and morphological markers</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLandry%26rft.auinit1%253DB.%2BS.%26rft.volume%253D116%26rft.issue%253D2%26rft.spage%253D331%26rft.epage%253D337%26rft.atitle%253DA%2BGenetic%2BMap%2Bof%2BLettuce%2B%2528Lactuca%2Bsativa%2BL.%2529%2BWith%2BRestriction%2BFragment%2BLength%2BPolymorphism%252C%2BIsozyme%252C%2BDisease%2BResistance%2Band%2BMorphological%2BMarkers%26rft_id%253Dinfo%253Apmid%252F17246387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE2LzIvMzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-thesis" id="cit-3.4.617.32"><div class="cit-metadata unstructured">Lavelle, D. O., 2009  Genetics of candidate genes for developmental and domestication-related traits in lettuce. Ph.D. dissertation in Genetics, U. C. Davis.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.33" data-doi="10.1007/s00438-010-0570-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenzl</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huttner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Construction of a high-density composite map and comparative mapping of segregation distortion regions in barley</span>. <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D284%26rft.issue%253D5%26rft.spage%253D319%26rft.epage%253D331%26rft.atitle%253DConstruction%2Bof%2Ba%2Bhigh-density%2Bcomposite%2Bmap%2Band%2Bcomparative%2Bmapping%2Bof%2Bsegregation%2Bdistortion%2Bregions%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-010-0570-3%26rft_id%253Dinfo%253Apmid%252F20803217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-010-0570-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20803217&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindqvist</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">Inheritance studies in lettuce</span>. <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.volume%253D46%26rft.spage%253D387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.35" data-doi="10.1073/pnas.0510791103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lysak</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pecinka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McBreen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mechanisms of chromosome number reduction in <em>Arabdiopsis thaliana</em> and related Brassicaceae species</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">5224</span>–<span class="cit-lpage">5229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0510791103%26rft_id%253Dinfo%253Apmid%252F16549785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzEzLzUyMjQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.36" data-doi="10.1534/genetics.107.075424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacDaniel</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">W. J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A linkage map reveals a complex basis for segregation distortion in an interpopulation cross in the moss <em>Ceratodon pupureus</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2489</span>–<span class="cit-lpage">2500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075424%26rft_id%253Dinfo%253Apmid%252F17603096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzI0ODkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.37" data-doi="10.1007/s00122-008-0921-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McHale</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genomic architecture of disease resistance in lettuce</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMcHale%26rft.auinit1%253DL.%2BK.%26rft.volume%253D118%26rft.issue%253D3%26rft.spage%253D565%26rft.epage%253D580%26rft.atitle%253DThe%2Bgenomic%2Barchitecture%2Bof%2Bdisease%2Bresistance%2Bin%2Blettuce.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0921-1%26rft_id%253Dinfo%253Apmid%252F19005638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0921-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19005638&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262577300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">B. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Clusters of resistance genes in plants evolve by divergent selection and a birth-and-death process</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1113</span>–<span class="cit-lpage">1130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D1113%26rft.epage%253D1130%26rft.atitle%253DClusters%2Bof%2BResistance%2BGenes%2Bin%2BPlants%2BEvolve%2Bby%2BDivergent%2BSelection%2Band%2Ba%2BBirth-and-Death%2BProcess%26rft_id%253Dinfo%253Apmid%252F9847076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjgvMTEvMTExMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.39" data-doi="10.1073/pnas.88.21.9828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of markers linked to disease-resistance genes by bulked segregant analysis—a rapid method to detect markers in specific genomic regions by using segregating populations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9828</span>–<span class="cit-lpage">9832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9828%26rft.epage%253D9832%26rft.atitle%253DIdentification%2Bof%2Bmarkers%2Blinked%2Bto%2Bdisease-resistance%2Bgenes%2Bby%2Bbulked%2Bsegregant%2Banalysis%253A%2Ba%2Brapid%2Bmethod%2Bto%2Bdetect%2Bmarkers%2Bin%2Bspecific%2Bgenomic%2Bregions%2Bby%2Busing%2Bsegregating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9828%26rft_id%253Dinfo%253Apmid%252F1682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTgyOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.40" data-doi="10.1093/molbev/msj112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">E. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide associations between hybrid sterility QTL and marker transmission ratio distortion</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj112%26rft_id%253Dinfo%253Apmid%252F16495346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIzLzUvOTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-web" id="cit-3.4.617.41"><div class="cit-metadata unstructured">National Agricultural Statistics Service, 2012 National Statistics for Lettuce. Available at: <a href="http://www.nass.usda.gov/Statistics_by_Subject/index.php?sector=CROPS">http://www.nass.usda.gov/Statistics_by_Subject/index.php?sector=CROPS</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.42" data-doi="10.1186/1471-2164-12-189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neves</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamani</span>  <span class="cit-name-given-names">E. M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alfenas</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirst</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A high-density transcript linkage map with 1,845 expressed genes positioned by microarray-based single feature polymorphisms (SFP) in <em>Eucalyptus</em></span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DNeves%26rft.auinit1%253DL.%2BG.%26rft.volume%253D12%26rft.spage%253D189%26rft.epage%253D189%26rft.atitle%253DA%2Bhigh-density%2Btranscript%2Blinkage%2Bmap%2Bwith%2B1%252C845%2Bexpressed%2Bgenes%2Bpositioned%2Bby%2Bmicroarray-based%2BSingle%2BFeature%2BPolymorphisms%2B%2528SFP%2529%2Bin%2BEucalyptus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-189%26rft_id%253Dinfo%253Apmid%252F21492453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21492453&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.43" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.44" data-doi="10.1146/annurev-genet-110711-155621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebollo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romanish</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mager</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transposable elements: an abundant and natural source of regulatory sequences for host genes</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DRebollo%26rft.auinit1%253DR.%26rft.volume%253D46%26rft.spage%253D21%26rft.epage%253D42%26rft.atitle%253DTransposable%2Belements%253A%2Ban%2Babundant%2Band%2Bnatural%2Bsource%2Bof%2Bregulatory%2Bsequences%2Bfor%2Bhost%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-110711-155621%26rft_id%253Dinfo%253Apmid%252F22905872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-110711-155621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22905872&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311568300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salomé</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laitinen</span>  <span class="cit-name-given-names">R. A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The recombination landscape in <em>Arabidopsis thaliana</em> F<sub>2</sub> populations</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D108%26rft.spage%253D447%26rft_id%253Dinfo%253Apmid%252F22072068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22072068&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.46" data-doi="10.1371/journal.pgen.0020144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazen</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution map of Arabidopsis recombinant inbred lines by whole-genome exon array hybridization</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSinger%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De144%26rft.epage%253De144%26rft.atitle%253DA%2Bhigh-resolution%2Bmap%2Bof%2BArabidopsis%2Brecombinant%2Binbred%2Blines%2Bby%2Bwhole-genome%2Bexon%2Barray%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020144%26rft_id%253Dinfo%253Apmid%252F17044735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17044735&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.47" data-doi="10.1007/s10142-004-0106-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sourdille</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cadalen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown-Guedira</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microsatellite-based deletion bin system for the establishment of genetic-physical map relationships in wheat (<em>Triticum aestivum</em> L.)</span>. <abbr class="cit-jnl-abbrev">Funct. Integr. Genomics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2B%2526%2Bintegrative%2Bgenomics%26rft.stitle%253DFunct%2BIntegr%2BGenomics%26rft.aulast%253DSourdille%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D25%26rft.atitle%253DMicrosatellite-based%2Bdeletion%2Bbin%2Bsystem%2Bfor%2Bthe%2Bestablishment%2Bof%2Bgenetic-physical%2Bmap%2Brelationships%2Bin%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10142-004-0106-1%26rft_id%253Dinfo%253Apmid%252F15004738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10142-004-0106-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15004738&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.48" data-doi="10.1186/1471-2164-13-185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldwell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development and application of a 6.5 million feature Affymetrix GeneChip for massively parallel discovery of single-position polymorphisms in lettuce (<em>Lactuca spp</em>.)</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DStoffel%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.spage%253D185%26rft.epage%253D185%26rft.atitle%253DDevelopment%2Band%2Bapplication%2Bof%2Ba%2B6.5%2Bmillion%2Bfeature%2BAffymetrix%2BGenechip%25C2%25AE%2Bfor%2Bmassively%2Bparallel%2Bdiscovery%2Bof%2Bsingle%2Bposition%2Bpolymorphisms%2Bin%2Blettuce%2B%2528Lactuca%2Bspp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-185%26rft_id%253Dinfo%253Apmid%252F22583801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22583801&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.49" data-doi="10.1007/s00122-006-0483-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An ultradense genetic recombination map for <em>Brassica napus</em>, consisting of 13551 SRAP markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1305</span>–<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSun%26rft.auinit1%253DZ.%26rft.volume%253D114%26rft.issue%253D8%26rft.spage%253D1305%26rft.epage%253D1317%26rft.atitle%253DAn%2Bultradense%2Bgenetic%2Brecombination%2Bmap%2Bfor%2BBrassica%2Bnapus%252C%2Bconsisting%2Bof%2B13551%2BSRAP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0483-z%26rft_id%253Dinfo%253Apmid%252F17426959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0483-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17426959&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.50" data-doi="10.1007/s00122-005-0155-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Syed</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonise</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van de Wiel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Linden</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A detailed linkage map of lettuce based on SSAP, AFLP and NBS markers</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSyed%26rft.auinit1%253DN.%2BH.%26rft.volume%253D112%26rft.issue%253D3%26rft.spage%253D517%26rft.epage%253D527%26rft.atitle%253DA%2Bdetailed%2Blinkage%2Bmap%2Bof%2Blettuce%2Bbased%2Bon%2BSSAP%252C%2BAFLP%2Band%2BNBS%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0155-4%26rft_id%253Dinfo%253Apmid%252F16341837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0155-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16341837&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.51" data-doi="10.1016/S0168-9525(00)88999-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Chromosome landing: a paradigm for map-based gene cloning in plants with large genomes</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D63%26rft.epage%253D68%26rft.atitle%253DChromosome%2Blanding%253A%2Ba%2Bparadigm%2Bfor%2Bmap-based%2Bgene%2Bcloning%2Bin%2Bplants%2Bwith%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252988999-4%26rft_id%253Dinfo%253Apmid%252F7716809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)88999-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7716809&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QG42400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Tomato Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D31%26rft.spage%253D635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.53" data-doi="10.1534/genetics.106.067462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Troggio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malacarne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coppola</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segala</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartwright</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A dense single-nucleotide polymorphism-based genetic linkage map of grapevine (<em>Vitis vinifera</em> L.) anchoring Pinot Noir bacterial artificial chromosome contigs</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2637</span>–<span class="cit-lpage">2650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067462%26rft_id%253Dinfo%253Apmid%252F17603124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzI2MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.54" data-doi="10.1007/s00122-007-0599-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonise</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavelle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A high-density, integrated genetic linkage map of lettuce (<em>Lactuca</em> spp.)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTruco%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D115%26rft.issue%253D6%26rft.spage%253D735%26rft.epage%253D746%26rft.atitle%253DA%2Bhigh-density%252C%2Bintegrated%2Bgenetic%2Blinkage%2Bmap%2Bof%2Blettuce%2B%2528Lactuca%2Bspp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0599-9%26rft_id%253Dinfo%253Apmid%252F17828385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0599-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828385&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249635000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.55" data-doi="10.1371/journal.pone.0037565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Ruiter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Poel</span>  <span class="cit-name-given-names">H. J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequence-based genotyping for marker discovery and co-dominant scoring in germplasm and populations</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTruong%26rft.auinit1%253DH.%2BT.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37565%26rft.epage%253De37565%26rft.atitle%253DSequence-based%2Bgenotyping%2Bfor%2Bmarker%2Bdiscovery%2Band%2Bco-dominant%2Bscoring%2Bin%2Bgermplasm%2Band%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037565%26rft_id%253Dinfo%253Apmid%252F22662172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22662172&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-web" id="cit-3.4.617.56"><div class="cit-metadata unstructured">United States Department of Agriculture, 2011 U.S. lettuce statistics. Available at: <a href="http://usda.mannlib.cornell.edu/MannUsda/viewDocumentInfo.do?documentID=1576">http://usda.mannlib.cornell.edu/MannUsda/viewDocumentInfo.do?documentID=1576</a>. Accessed: February 18, 2013.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.57" data-doi="10.1007/s00122-012-1897-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uwimana</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smulders</span>  <span class="cit-name-given-names">M. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooftman</span>  <span class="cit-name-given-names">D. A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Tienderen</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybridization between crops and wild relatives: the contribution of cultivated lettuce to the vigour of crop-wild hybrids under drought, salinity and nutrient deficiency conditions</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1097</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DUwimana%26rft.auinit1%253DB.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1097%26rft.epage%253D1111%26rft.atitle%253DHybridization%2Bbetween%2Bcrops%2Band%2Bwild%2Brelatives%253A%2Bthe%2Bcontribution%2Bof%2Bcultivated%2Blettuce%2Bto%2Bthe%2Bvigour%2Bof%2Bcrop-wild%2Bhybrids%2Bunder%2Bdrought%252C%2Bsalinity%2Band%2Bnutrient%2Bdeficiency%2Bconditions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1897-4%26rft_id%253Dinfo%253Apmid%252F22660630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1897-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660630&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.58" data-doi="10.1093/jhered/esm109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Berloo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Computer note: GGT 2.0: versatile software for visualization and analysis of genetic data</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesm109%26rft_id%253Dinfo%253Apmid%252F18222930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMjMyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-3.4.617.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Software for the Calculation of Genetic Linkage Maps</span>. <span class="cit-publ-name">Kyazma B. V.</span>, <span class="cit-publ-loc">Wageningen, the Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.60" data-doi="10.1007/s00122-005-0097-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Os</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eck</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">RECORD: a novel method for ordering loci on a genetic linkage map</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVan%2BOs%26rft.auinit1%253DH.%26rft.volume%253D112%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D40%26rft.atitle%253DRECORD%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Bordering%2Bloci%2Bon%2Ba%2Bgenetic%2Blinkage%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0097-x%26rft_id%253Dinfo%253Apmid%252F16228189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0097-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16228189&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.61" data-doi="10.1534/genetics.106.055871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Os</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrzejewski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrena</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Construction of a 10,000-marker ultradense genetic recombination map of potato: Providing a framework for accelerated gene isolation and a genomewide physical map</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.055871%26rft_id%253Dinfo%253Apmid%252F16582432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8yLzEwNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.62" data-doi="10.1016/S0169-5347(01)02213-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winther</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agrawal</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodnight</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Alternative definitions of epistasis: dependence and interaction</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D16%26rft.spage%253D498%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252801%252902213-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(01)02213-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170534800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.63" data-doi="10.1007/s10709-011-9613-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular characterization of a genomic interval with highly uneven recombination distribution on maize chromosome 10. L</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1109</span>–<span class="cit-lpage">1118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D139%26rft.spage%253D1109%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-011-9613-x%26rft_id%253Dinfo%253Apmid%252F22057628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-011-9613-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22057628&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.64" data-doi="10.1038/sj.hdy.6884730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waycott</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fort</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryder</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Mapping morphological genes relative to molecular markers in lettuce (<em>Lactuca sativa</em> L.)</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D82%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6884730%26rft_id%253Dinfo%253Apmid%252F10336698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6884730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10336698&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.65" data-doi="10.1534/genetics.104.036533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Werner</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhlenhaut</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chory</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">FRIGIDA-independent variation in flowering time of natural <em>Arabidiopsis thaliana</em> accessions</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1197</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036533%26rft_id%253Dinfo%253Apmid%252F15911588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzExOTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8zLzQvNjE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.66" data-doi="10.1101/gr.5105206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">702</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5105206%26rft_id%253Dinfo%253Apmid%252F16687733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzAyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.67" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.68" data-doi="10.1007/s00122-011-1614-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">X.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparisons of four approximation algorithms for large-scale linkage map construction</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DJ.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D649%26rft.epage%253D655%26rft.atitle%253DComparisons%2Bof%2Bfour%2Bapproximation%2Balgorithms%2Bfor%2Blarge-scale%2Blinkage%2Bmap%2Bconstruction.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1614-8%26rft_id%253Dinfo%253Apmid%252F21611760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1614-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21611760&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F4%2F617.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.69" data-doi="10.1073/pnas.1005931107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yua</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10578</span>–<span class="cit-lpage">10583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1005931107%26rft_id%253Dinfo%253Apmid%252F20498060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIzLzEwNTc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.70" data-doi="10.1101/gr.3436305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yogeeswaran</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">York</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amenta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesser</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comparative genome analyses of Arabidopsis spp.: Inferring chromosomal rearrangement events in the evolutionary history of <em>A. thaliana</em></span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DYogeeswaran%26rft.auinit1%253DK.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D505%26rft.epage%253D515%26rft.atitle%253DComparative%2Bgenome%2Banalyses%2Bof%2BArabidopsis%2Bspp.%253A%2BInferring%2Bchromosomal%2Brearrangement%2Bevents%2Bin%2Bthe%2Bevolutionary%2Bhistory%2Bof%2BA.%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3436305%26rft_id%253Dinfo%253Apmid%252F15805492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzQvNTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMy80LzYxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.71" data-doi="10.1093/jxb/erm006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagstaff</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rae</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sihota</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keevil</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">QTLs for shelf life in lettuce co-locate with those for leaf biophysical properties but not with those for leaf developmental traits</span>. <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1433</span>–<span class="cit-lpage">1449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferm006%26rft_id%253Dinfo%253Apmid%252F17347132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjU4LzYvMTQzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzMvNC82MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.4.617.72" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em></span>. <abbr class="cit-jnl-abbrev">Nat. Comm.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BComm.%26rft.volume%253D2%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/4/617.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/4/607" title="Plastome Sequence Determination and Comparative Analysis for Members of the Lolium-Festuca Grass Species Complex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/4/633" title="Trans-Lesion DNA Polymerases May Be Involved in Yeast Meiosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/4"><span>Volume 3 Issue 4, April 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/4.cover-source.jpg?itok=tXw4AXt4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/4/617" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Jm55WMJ6tNXjbO6sJclHMA3orB14l632wRXb5gY-buI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/4/617" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/4/617" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/4/617" class="active" data-icon-position="" data-hide-link-title="0">An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459455" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kUgMNQoVkaVjX66BjNmWL0AHpMEQFzAFd_MPAR3nZ2M" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/617.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/4/617" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_bLKbNycswsDmsdFjzY_v3kcazLZmH5fWHumkbD-5-k" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/4/617" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/4/617.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459455" id="node459455--2" data-pisa="ggg;3/4/617" data-pisa-master="ggg;g3.112.004929" data-apath="/ggg/3/4/617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Maria José Truco</span>, <span class="highwire-citation-author" data-delta="1">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="2">Alexander Kozik</span>, <span class="highwire-citation-author" data-delta="3">Hans van Leeuwen</span>, <span class="highwire-citation-author" data-delta="4">John Bowers</span>, <span class="highwire-citation-author" data-delta="5">Sebastian Reyes Chin Wo</span>, <span class="highwire-citation-author" data-delta="6">Kevin Stoffel</span>, <span class="highwire-citation-author" data-delta="7">Huaqin Xu</span>, <span class="highwire-citation-author" data-delta="8">Theresa Hill</span>, <span class="highwire-citation-author" data-delta="9">Allen Van Deynze</span> and <span class="highwire-citation-author" data-delta="10">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">617-631;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004929 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459455--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maria José Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hans van Leeuwen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bvan%2BLeeuwen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Leeuwen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bvan%2BLeeuwen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Bowers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BBowers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowers%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BBowers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastian Reyes Chin Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes%2BChin%2BWo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes%2BChin%2BWo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Huaqin Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huaqin%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuaqin%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Theresa Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theresa%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheresa%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/4/617" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/4/617&amp;title=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/4/617&amp;title=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/4/617&amp;title=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/4/617&amp;text=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/4/617&amp;title=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/4/617&amp;t=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/4/617&amp;title=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/4/617&amp;title=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459455/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459455" id="node459455--3" data-pisa="ggg;3/4/617" data-pisa-master="ggg;g3.112.004929" data-apath="/ggg/3/4/617.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Maria José Truco</span>, <span class="highwire-citation-author" data-delta="1">Hamid Ashrafi</span>, <span class="highwire-citation-author" data-delta="2">Alexander Kozik</span>, <span class="highwire-citation-author" data-delta="3">Hans van Leeuwen</span>, <span class="highwire-citation-author" data-delta="4">John Bowers</span>, <span class="highwire-citation-author" data-delta="5">Sebastian Reyes Chin Wo</span>, <span class="highwire-citation-author" data-delta="6">Kevin Stoffel</span>, <span class="highwire-citation-author" data-delta="7">Huaqin Xu</span>, <span class="highwire-citation-author" data-delta="8">Theresa Hill</span>, <span class="highwire-citation-author" data-delta="9">Allen Van Deynze</span> and <span class="highwire-citation-author" data-delta="10">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">617-631;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.004929 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459455--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Maria José Truco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BJos%C3%A9%2BTruco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Truco%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BJos%25C3%25A9%2BTruco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hamid Ashrafi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hamid%2BAshrafi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashrafi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHamid%2BAshrafi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alexander Kozik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BKozik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kozik%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BKozik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hans van Leeuwen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bvan%2BLeeuwen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Leeuwen%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bvan%2BLeeuwen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John Bowers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BBowers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowers%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BBowers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastian Reyes Chin Wo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BReyes%2BChin%2BWo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wo%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BReyes%2BChin%2BWo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Kevin Stoffel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BStoffel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stoffel%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BStoffel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Huaqin Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huaqin%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuaqin%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Theresa Hill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Theresa%2BHill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hill%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATheresa%2BHill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Allen Van Deynze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Seed Biotechnology Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BVan%2BDeynze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Deynze%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BVan%2BDeynze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Genome Center, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=617&amp;rft.epage=631&amp;rft.atitle=An%20Ultra-High-Density%2C%20Transcript-Based%2C%20Genetic%20Map%20of%20Lettuce&amp;rft.volume=3&amp;rft.issue=4&amp;rft.date=2013-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Truco%2C+Maria+Jos%C3%A9&amp;rft.au=Ashrafi%2C+Hamid&amp;rft.au=Kozik%2C+Alexander&amp;rft.au=van+Leeuwen%2C+Hans&amp;rft.au=Bowers%2C+John&amp;rft.au=Wo%2C+Sebastian+Reyes+Chin&amp;rft.au=Stoffel%2C+Kevin&amp;rft.au=Xu%2C+Huaqin&amp;rft.au=Hill%2C+Theresa&amp;rft.au=Van+Deynze%2C+Allen&amp;rft.au=Michelmore%2C+Richard+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459455/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459455/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459455/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459455/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459455/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459455/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459455/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459455/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459455/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459455/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459455/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459455/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/4/617" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/4/617.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/4/617.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/4/617.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/4/617.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
