<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/5/1721" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/5/1721.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/5/1721.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/5/1721.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/5/1721/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/5/1721.atom" />
<meta name="HW.pisa" content="ggg;8/5/1721" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments" />
<meta name="DC.Identifier" content="10.1534/g3.118.200140" />
<meta name="DC.Date" content="2018-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Gapare et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) has successfully been used in plant breeding to improve selection efficiency and reduce breeding time and cost. However, there has not been a study to evaluate GS prediction models that may be used for predicting cotton breeding lines across multiple environments. In this study, we evaluated the performance of Bayes Ridge Regression, BayesA, BayesB, BayesC and Reproducing Kernel Hilbert Spaces regression models. We then extended the single-site GS model to accommodate genotype × environment interaction (G×E) in order to assess the merits of multi- over single-environment models in a practical breeding and selection context in cotton, a crop for which this has not previously been evaluated. Our study was based on a population of 215 upland cotton ( Gossypium hirsutum ) breeding lines which were evaluated for fiber length and strength at multiple locations in Australia and genotyped with 13,330 single nucleotide polymorphic (SNP) markers. BayesB, which assumes unique variance for each marker and a proportion of markers to have large effects, while most other markers have zero effect, was the preferred model. GS accuracy for fiber length based on a single-site model varied across sites, ranging from 0.27 to 0.77 (mean = 0.38), while that of fiber strength ranged from 0.19 to 0.58 (mean = 0.35) using randomly selected sub-populations as the training population. Prediction accuracies from the M×E model were higher than those for single-site and across-site models, with an average accuracy of 0.71 and 0.59 for fiber length and strength, respectively. The use of the M×E model could therefore identify which breeding lines have effects that are stable across environments and which ones are responsible for G×E and so reduce the amount of phenotypic screening required in cotton breeding programs to identify adaptable genotypes." />
<meta name="DC.Contributor" content="Washington Gapare" />
<meta name="DC.Contributor" content="Shiming Liu" />
<meta name="DC.Contributor" content="Warren Conaty" />
<meta name="DC.Contributor" content="Qian-Hao Zhu" />
<meta name="DC.Contributor" content="Vanessa Gillespie" />
<meta name="DC.Contributor" content="Danny Llewellyn" />
<meta name="DC.Contributor" content="Warwick Stiller" />
<meta name="DC.Contributor" content="Iain Wilson" />
<meta name="article:published_time" content="2018-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) has successfully been used in plant breeding to improve selection efficiency and reduce breeding time and cost. However, there has not been a study to evaluate GS prediction models that may be used for predicting cotton breeding lines across multiple environments. In this study, we evaluated the performance of Bayes Ridge Regression, BayesA, BayesB, BayesC and Reproducing Kernel Hilbert Spaces regression models. We then extended the single-site GS model to accommodate genotype × environment interaction (G×E) in order to assess the merits of multi- over single-environment models in a practical breeding and selection context in cotton, a crop for which this has not previously been evaluated. Our study was based on a population of 215 upland cotton (&lt;i&gt;Gossypium hirsutum&lt;/i&gt;) breeding lines which were evaluated for fiber length and strength at multiple locations in Australia and genotyped with 13,330 single nucleotide polymorphic (SNP) markers. BayesB, which assumes unique variance for each marker and a proportion of markers to have large effects, while most other markers have zero effect, was the preferred model. GS accuracy for fiber length based on a single-site model varied across sites, ranging from 0.27 to 0.77 (mean = 0.38), while that of fiber strength ranged from 0.19 to 0.58 (mean = 0.35) using randomly selected sub-populations as the training population. Prediction accuracies from the M×E model were higher than those for single-site and across-site models, with an average accuracy of 0.71 and 0.59 for fiber length and strength, respectively. The use of the M×E model could therefore identify which breeding lines have effects that are stable across environments and which ones are responsible for G×E and so reduce the amount of phenotypic screening required in cotton breeding programs to identify adaptable genotypes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/05/01" />
<meta name="citation_mjid" content="ggg;8/5/1721" />
<meta name="citation_id" content="8/5/1721" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/5/1721" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/5/1721.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/5/1721.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/5/1721.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200140" />
<meta name="citation_pmid" content="29559536" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1721" />
<meta name="citation_lastpage" content="1732" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Washington Gapare" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, GPO Box 1600, Canberra, ACT 2601, Australia" />
<meta name="citation_author_email" content="Washington.Gapare@csiro.au" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9787-8543" />
<meta name="citation_author" content="Shiming Liu" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, Locked Bag 59, Narrabri, NSW 2390, Australia" />
<meta name="citation_author" content="Warren Conaty" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, Locked Bag 59, Narrabri, NSW 2390, Australia" />
<meta name="citation_author" content="Qian-Hao Zhu" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, GPO Box 1600, Canberra, ACT 2601, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6505-7417" />
<meta name="citation_author" content="Vanessa Gillespie" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, GPO Box 1600, Canberra, ACT 2601, Australia" />
<meta name="citation_author" content="Danny Llewellyn" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, GPO Box 1600, Canberra, ACT 2601, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9535-8707" />
<meta name="citation_author" content="Warwick Stiller" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, Locked Bag 59, Narrabri, NSW 2390, Australia" />
<meta name="citation_author" content="Iain Wilson" />
<meta name="citation_author_institution" content="CSIRO Agriculture &amp; Food, GPO Box 1600, Canberra, ACT 2601, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4626-8059" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. M. Bernal-Vasquez;citation_author=A. Gordillo;citation_author=M. Schmidt;citation_author=H. P. Piepho;citation_title=Genomic prediction in early selection stages using multi-year data in a hybrid rye breeding program.;citation_pages=51;citation_volume=18;citation_year=2017;citation_issue=1;citation_doi=10.1186/s12863-017-0512-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. M. Bernal-Vasquez;citation_author=J. Möhring;citation_author=M. Schmidt;citation_author=M. Schönleben;citation_author=C. C. Schön;citation_title=The importance of phenotypic data analysis for genomic prediction - a case study comparing different spatial models in rye.;citation_pages=646;citation_volume=15;citation_year=2014;citation_issue=1;citation_doi=10.1186/1471-2164-15-646" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype x environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_issue=2;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_title=ASReml-R reference manual, release 3. Technical report;citation_author=D. G. Butler;citation_author=B. R. Cullis;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_title=Cotton, America Society of Agronomy, Agronomy Monograph;citation_author=B. T. Campbell;citation_author=G. O. Myers;citation_title=Quantitative Genetics;citation_year=2015;citation_doi=10.2134/agronmonogr57.2013.0024" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=S. A. Clark;citation_author=J. M. Hickey;citation_author=H. D. Daetwyler;citation_author=J. H. J. van der Werf;citation_title=The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes.;citation_pages=4-4;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=22321529;citation_doi=10.1186/1297-9686-44-4" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. D. Clement;citation_author=G. A. Constable;citation_author=W. N. Stiller;citation_author=S. M. Liu;citation_title=Negative associations still exist between yield and fiber quality in cotton breeding programs in Australia and USA.;citation_pages=1-7;citation_volume=128;citation_year=2012;citation_doi=10.1016/j.fcr.2011.12.002" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. N. Cobb;citation_author=G. DeClerck;citation_author=A. Greenberg;citation_author=R. Clark;citation_author=S. McCouch;citation_title=Next-generation phenotyping: requirements and strategies for enhancing our understanding of genotype–phenotype relationships and its relevance to crop improvement.;citation_pages=867-887;citation_volume=126;citation_year=2013;citation_issue=4;citation_pmid=23471459;citation_doi=10.1007/s00122-013-2066-0" />
<meta name="citation_reference" content="citation_title=Genetic Improvement of Cotton: Emerging Technologies;citation_title=Approaches utilized in breeding and development of cotton cultivars in Australia;citation_author=G. A. Constable;citation_author=N. J. Thomson;citation_author=P. E. Reid;citation_pages=1-15;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=M. Maccaferri;citation_author=R. Tuberosa;citation_author=J. Burgueño;citation_title=Extending the marker × environment interaction model for genomic-enabled prediction and genome-wise association analyses in durum wheat.;citation_volume=56;citation_year=2016;citation_issue=5;citation_doi=10.2135/cropsci2015.04.0260" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_issue=1;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_issue=11;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=O. A. Montesinos-Lopez;citation_author=J. Burgueño;citation_title=Bayesian genomic prediction with genotype x environment interaction kernel models.;citation_volume=7;citation_year=2017;citation_pmid=27793970;citation_doi=10.1534/g3.116.035584" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. R. Cullis;citation_author=F. M. Thomson;citation_author=J. A. Fisher;citation_author=A. R. Gilmour;citation_author=R. Thompson;citation_title=The analysis of the NSW wheat variety database. 1. Modelling trial error variance.;citation_pages=21-27;citation_volume=92;citation_year=1996;citation_issue=1;citation_pmid=24166112;citation_doi=10.1007/BF00222947" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic Prediction in Animals and Plants: Simulation of Data, Validation, Reporting, and Benchmarking.;citation_volume=193;citation_year=2013;citation_issue=2;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=04;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_issue=2;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigrees.;citation_volume=182;citation_year=2009;citation_issue=1;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_year=2015a" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=Gianola. D;citation_title=Genomic heritability: what is it?;citation_pages=e1005048;citation_volume=11;citation_year=2015b;citation_issue=5;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. A. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: Genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_issue=9;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=V. Edriss;citation_author=Y. Gao;citation_author=X. Zhang;citation_author=M. Jumbo;citation_author=D. Makumbi;citation_title=Genomic prediction in a large African maize population.;citation_pages=2361-2371;citation_volume=57;citation_year=2017;citation_issue=5;citation_doi=10.2135/cropsci2016.08.0715" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_issue=3;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=W. J. Gapare;citation_author=W. Conaty;citation_author=Q. H. Zhu;citation_author=S. Liu;citation_author=W. Stiller;citation_title=Genome-wide association study of yield components and fiber quality traits in a cotton germplasm diversity panel.;citation_pages=66;citation_volume=213;citation_year=2017;citation_issue=3;citation_doi=10.1007/s10681-017-1855-y" />
<meta name="citation_reference" content="citation_journal_title=Anal. Chim. Acta;citation_author=P. Geladi;citation_author=B. R. Kowalski;citation_title=Partial least-squares regression: a tutorial.;citation_pages=1-17;citation_volume=185;citation_year=1986;citation_doi=10.1016/0003-2670(86)80028-9" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=M. E. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=D. Grattapaglia;citation_author=M. D. Resende;citation_title=Genomic selection in forest tree breeding.;citation_pages=241-255;citation_volume=7;citation_year=2011;citation_issue=2;citation_doi=10.1007/s11295-010-0328-4" />
<meta name="citation_reference" content="citation_title=ASReml User Guide Release 3.0;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_pages=372;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. L. Garrick;citation_title=Genomic BLUP decoded: A look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_issue=3;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=J. Hamblin;citation_author=M. J. O. Zimmermann;citation_title=Breeding common bean for yield in mixtures.;citation_pages=245-272;citation_volume=4;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. J. Chamberlain;citation_author=M. E. Goddard;citation_title=Invited review: Genomic selection in dairy cattle: Progress and challenges;citation_pages=433-443;citation_volume=92;citation_year=2009;citation_issue=2;citation_pmid=19164653;citation_doi=10.3168/jds.2008-1646" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_issue=1;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Using genomic prediction to characterise environments and optimize prediction accuracy in applied breeding data.;citation_pages=921-933;citation_volume=53;citation_year=2013;citation_issue=3;citation_doi=10.2135/cropsci2012.07.0420" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-55;citation_volume=55;citation_year=2015;citation_issue=1;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Heslot;citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Impact of marker ascertainment bias on genomic selection accuracy and estimates of genetic diversity.;citation_pages=e74612;citation_volume=8;citation_year=2013;citation_issue=9;citation_doi=10.1371/journal.pone.0074612" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_issue=1;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Hinze;citation_author=D. Fang;citation_author=M. Gore;citation_author=B. Scheffler;citation_author=J. Yu;citation_title=Molecular characterization of the Gossypium diversity reference set of the US national cotton germplasm collection.;citation_pages=313-327;citation_volume=128;citation_year=2015;citation_issue=2;citation_pmid=25431191;citation_doi=10.1007/s00122&amp;ndash;014&amp;ndash;2431&amp;ndash;7" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. Howard;citation_author=A. L. Carriquiry;citation_author=W. D. Beavis;citation_title=Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures.;citation_volume=4;citation_year=2014;citation_issue=6;citation_pmid=24727289;citation_doi=10.1534/g3.114.010298" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Huang;citation_author=A. Cabrera;citation_author=A. Hoffstetter;citation_author=C. Griffey;citation_author=D. V. Sanford;citation_title=Genomic selection for wheat traits and trait stability.;citation_pages=1697-1710;citation_volume=129;citation_year=2016;citation_issue=9;citation_doi=10.1007/s00122-016-2733-z" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. M. Hulse-Kemp;citation_author=J. Lemm;citation_author=J. Plieske;citation_author=H. Ashrafi;citation_author=R. Buyyarapu;citation_title=Development of a 63K SNP array for cotton and high-density mapping of intraspecific and interspecific populations of Gossypium spp.;citation_volume=5;citation_year=2015;citation_issue=6;citation_pmid=25908569;citation_doi=10.1534/g3.115.018416" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J. L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_issue=1;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_issue=3;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. M. Lacape;citation_author=T. B. Nguyen;citation_author=B. Courtois;citation_author=J. L. Belot;citation_author=M. Giband;citation_title=QTL Analysis of Cotton Fiber Quality Using Multiple Gossypium hirsutum x Gossypium barbadense Backcross Generations;citation_pages=123-140;citation_volume=45;citation_year=2005;citation_issue=1;citation_doi=10.2135/cropsci2005.0123a" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. Lado;citation_author=P. G. Barrios;citation_author=M. Quincke;citation_author=P. Silva;citation_author=L. Gutiérrez;citation_title=Modeling genotype× environment interaction for genomic selection with unbalanced data from a wheat breeding program.;citation_pages=2165-2179;citation_volume=56;citation_year=2016;citation_issue=5;citation_doi=10.2135/cropsci2015.04.0207" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=Z. Lin;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Genomic selection in crops, trees and forages: A review.;citation_pages=1177-1191;citation_volume=65;citation_year=2014;citation_issue=11;citation_doi=10.1071/CP13363" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Z. Lin;citation_author=J. Wang;citation_author=N. O. I. Cogan;citation_author=L. W. Pembleton;citation_author=P. Badenhorst;citation_title=Genetic gain and inbreeding from genomic selection in a simulated commercial breeding program for Perennial Ryegrass.;citation_pages=243-252;citation_volume=57;citation_year=2016;citation_doi=10.2135/cropsci2016.07.0577" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=S. M. Liu;citation_author=G. A. Constable;citation_author=P. E. Reid;citation_author=W. N. Stiller;citation_author=B. R. Cullis;citation_title=The interaction between breeding and crop management in improved cotton yield.;citation_pages=49-60;citation_volume=148;citation_year=2013;citation_doi=10.1016/j.fcr.2013.04.006" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=S. M. Liu;citation_author=D. L. Llewellyn;citation_author=W. N. Stiller;citation_author=J. Jacobs;citation_author=J. M. Lacape;citation_title=Heritability and predicted selection response of yield components and fiber properties in an inter-specific derived RIL population of cotton.;citation_pages=309-320;citation_volume=178;citation_year=2011;citation_issue=3;citation_doi=10.1007/s10681-010-0289-6" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. A. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnet;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using a markers x environment interaction genomic selection model.;citation_volume=5;citation_year=2015;citation_issue=4;citation_pmid=25660166;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. J. Lorenz;citation_title=Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment.;citation_pages=481-491;citation_volume=3;citation_year=2013;citation_issue=3;citation_doi=10.1534/g3.112.004911" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=Cot. Grow. Rev.;citation_author=J. McMahon;citation_author=A. Low;citation_title=Growing degree days as measure of temperature effects on cotton.;citation_pages=39-49;citation_volume=49;citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=H. Oakey;citation_author=B. Cullis;citation_author=R. Thompson;citation_author=J. Comadran;citation_author=C. Halpin;citation_title=Genomic selection in multi-environment crop trials.;citation_volume=6;citation_year=2016;citation_issue=5;citation_pmid=26976443;citation_doi=10.1534/g3.116.027524" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=e190-e190;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194218;citation_doi=10.1371/journal.pgen.0020190" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci;citation_author=P. Perez-Rodriguez;citation_author=J. Crossa;citation_author=K. Bondalapati;citation_author=G. Meyer;citation_author=F. Pita;citation_author=G. de los Campos;citation_title=A pedigree-based reacton norm model for prediction of cotton yield in multi-environment trials;citation_pages=1143-1151;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.08.0577" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_title=Genome-wide regression &amp; prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_issue=2;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="R Development Core Team, 2014 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria (http://www.gbif.org/resource/81287)." />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Riedelsheimer;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines.;citation_pages=452-452;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22947126;citation_doi=10.1186/1471-2164-13-452" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=W. D. Rosenthal;citation_author=T. J. Gerik;citation_title=Radiation Use Efficiency among Cotton Cultivars;citation_pages=655-658;citation_volume=83;citation_year=1991;citation_issue=4;citation_doi=10.2134/agronj1991.00021962008300040001x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_title=Genomic selection for quantitative adult plant stem rust resistance in wheat.;citation_volume=7;citation_year=2014;citation_issue=3;citation_doi=10.3835/plantgenome2014.02.0006" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_author=J. Poland;citation_title=Efficient use of historical data for genomic selection: a case study of stem rust resistance in wheat.;citation_volume=8;citation_year=2015;citation_issue=1;citation_doi=10.3835/plantgenome2014.09.0046" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. Sallam;citation_author=J. Endelman;citation_author=J. L. Jannink;citation_author=K. Smith;citation_title=Assessing genomic selection prediction accuracy in a dynamic barley breeding population.;citation_volume=8;citation_year=2015;citation_issue=1;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic selection and association mapping in rice (Oryza sativa): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical Rice breeding lines.;citation_pages=e1005350;citation_volume=11;citation_year=2015;citation_issue=6;citation_doi=10.1371/journal.pgen.1005350" />
<meta name="citation_reference" content="citation_title=Australian Cotton Germplasm Resources, World Cotton Germplasm Resources;citation_author=W. N. Stiller;citation_author=I. W. Wilson;citation_year=2014;citation_doi=10.5772/58414" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. Sukumaran;citation_author=J. Crossa;citation_author=D. Jarquin;citation_author=M. Lopes;citation_author=M. P. Reynolds;citation_title=Genomic prediction with pedigree and genotype x environment interaction in spring wheat grown in South and West Asia, North Africa, and Mexico.;citation_volume=7;citation_year=2017;citation_issue=2;citation_pmid=27903632;citation_doi=10.1534/g3.116.036251" />
<meta name="citation_reference" content="citation_journal_title=Journal of Chemical Information and Computer Sciences;citation_journal_abbrev=Journal of Chemical Information and Computer Sciences;citation_author=V. Svetnik;citation_author=A. Liaw;citation_author=C. Tong;citation_author=J. C. Culberson;citation_author=R. P. Sheridan;citation_title=Random forest: a classification and regression tool for compound classification and QSAR modeling.;citation_pages=1947-1958;citation_volume=43;citation_year=2003;citation_issue=6;citation_pmid=14632445;citation_doi=10.1021/ci034160g" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the Lasso.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Velu;citation_author=J. Crossa;citation_author=R. P. Singh;citation_author=Y. Hao;citation_author=S. Dreisigacker;citation_title=Genomic prediction for grain zinc and iron concentrations in spring wheat.;citation_pages=1595-1605;citation_volume=129;citation_year=2016;citation_issue=8;citation_doi=10.1007/s00122&amp;ndash;016&amp;ndash;2726-y" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=N. R. Wray;citation_author=J. Yang;citation_author=B. J. Hayes;citation_author=A. L. Price;citation_author=M. E. Goddard;citation_title=Pitfalls of predicting complex traits from SNPs.;citation_pages=507-515;citation_volume=14;citation_year=2013;citation_issue=7;citation_pmid=23774735;citation_doi=10.1038/nrg3457" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian Antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_issue=4;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Yi;citation_author=S. Xu;citation_title=Bayesian LASSO for quantitative trait loci mapping.;citation_volume=179;citation_year=2008;citation_issue=2;citation_pmid=18505874;citation_doi=10.1534/genetics.107.085589" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=W. Liu;citation_author=T. Würschum;citation_author=H. P. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations.;citation_pages=769-776;citation_volume=124;citation_year=2012;citation_issue=4;citation_pmid=22075809;citation_doi=10.1007/s00122-011-1745-y" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=Q. H. Zhu;citation_author=J. Zhang;citation_author=D. X. Liu;citation_author=W. N. Stiller;citation_author=D. J. Liu;citation_title=Integrated mapping and characterization of the gene underlying the okra leaf trait in Gossypium hirsutum L.;citation_volume=67;citation_year=2016;citation_issue=3;citation_pmid=26567355;citation_doi=10.1093/jxb/erv494" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) has successfully been used in plant breeding to improve selection efficiency and reduce breeding time and cost. However, there has not been a study to evaluate GS prediction models that may be used for predicting cotton breeding lines across multiple environments. In this study, we evaluated the performance of Bayes Ridge Regression, BayesA, BayesB, BayesC and Reproducing Kernel Hilbert Spaces regression models. We then extended the single-site GS model to accommodate genotype × environment interaction (G×E) in order to assess the merits of multi- over single-environment models in a practical breeding and selection context in cotton, a crop for which this has not previously been evaluated. Our study was based on a population of 215 upland cotton ( Gossypium hirsutum ) breeding lines which were evaluated for fiber length and strength at multiple locations in Australia and genotyped with 13,330 single nucleotide polymorphic (SNP) markers. BayesB, which assumes unique variance for each marker and a proportion of markers to have large effects, while most other markers have zero effect, was the preferred model. GS accuracy for fiber length based on a single-site model varied across sites, ranging from 0.27 to 0.77 (mean = 0.38), while that of fiber strength ranged from 0.19 to 0.58 (mean = 0.35) using randomly selected sub-populations as the training population. Prediction accuracies from the M×E model were higher than those for single-site and across-site models, with an average accuracy of 0.71 and 0.59 for fiber length and strength, respectively. The use of the M×E model could therefore identify which breeding lines have effects that are stable across environments and which ones are responsible for G×E and so reduce the amount of phenotypic screening required in cotton breeding programs to identify adaptable genotypes." />
<meta name="og-title" property="og:title" content="Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/5/1721" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) has successfully been used in plant breeding to improve selection efficiency and reduce breeding time and cost. However, there has not been a study to evaluate GS prediction models that may be used for predicting cotton breeding lines across multiple environments. In this study, we evaluated the performance of Bayes Ridge Regression, BayesA, BayesB, BayesC and Reproducing Kernel Hilbert Spaces regression models. We then extended the single-site GS model to accommodate genotype × environment interaction (G×E) in order to assess the merits of multi- over single-environment models in a practical breeding and selection context in cotton, a crop for which this has not previously been evaluated. Our study was based on a population of 215 upland cotton ( Gossypium hirsutum ) breeding lines which were evaluated for fiber length and strength at multiple locations in Australia and genotyped with 13,330 single nucleotide polymorphic (SNP) markers. BayesB, which assumes unique variance for each marker and a proportion of markers to have large effects, while most other markers have zero effect, was the preferred model. GS accuracy for fiber length based on a single-site model varied across sites, ranging from 0.27 to 0.77 (mean = 0.38), while that of fiber strength ranged from 0.19 to 0.58 (mean = 0.35) using randomly selected sub-populations as the training population. Prediction accuracies from the M×E model were higher than those for single-site and across-site models, with an average accuracy of 0.71 and 0.59 for fiber length and strength, respectively. The use of the M×E model could therefore identify which breeding lines have effects that are stable across environments and which ones are responsible for G×E and so reduce the amount of phenotypic screening required in cotton breeding programs to identify adaptable genotypes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
    <title>Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6lUd5_HJE0SsWO5MmUUK9h4qE47Z1Iew2s6BNtZmXEg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489451","apath":"\/ggg\/8\/5\/1721.atom","pisa":"ggg;8\/5\/1721","ac":{"\/ggg\/8\/5\/1721.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/5\/1721.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1721"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1721"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/5\/1721":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221S3TCT1uGlPgUaWCzXQ9phD6SNFiZx+rQiZVwkCQDnOYLXkQUzeaIYMKR2ft8Ss\\\\\\\/ztx+Ve1AED9z\\\\\\\/xS+f68B9IaWExNWiCNz2fN+WKR1b\\\\\\\/x9UgnhrDJSpHVC2H2yrM4NQsSw69yiBcHLaw+TJdLxXZRsj7IZ+m1SdQ59P4dU8cj7a9PmzbMS3GHB1IVOllfFHsKjBrr7auIDY2m3NtCEpAcfLfFM+0VJ4Tr8tB+eCc+SgC9gSC56SPJE8e20\\\\\\\/dPnC\\\\\\\/OIi3zs5KAn9Gegk1rR38MAFYu1ONPjIpsBOIA0OZ3HnUFgYXGM0829oityerb244N1E3vZZx3VJbw99k0ynYEnd7u\\\\\\\/8\\\\\\\/Qxmt1mkewlH4k8pmyNM0uM\\\\\\\/mVYb8X8MM9lbm7KRQaUj2WWYmLt+ZSBmgw\\\\\\\/XHd0q\\\\\\\/1k6SjM8WxA+7qmJWDhAaNbP\\\\\\\/M0N80v\\\\\\\/P7ZmgojaeMfmorCnQK8JDkCv4bFn8JzvxLeq+3smWJsD5I6irdXKChSvGPdsunZHW19N5dNEEsn\\\\\\\/t23Z63Pp6KEWmVJgZPvbF3PLJb38Zqn7dpRj22w6b00DgZ\\\\\\\/rMGUzJgZUkJz\\\\\\\/MsK+mgC6tjidYBPIBoOvLbsHBUXLNHXYjBaMLIb9d0Xv0fI7mmmzMmwdzZEaS7MNyjiT1FZ4xN1R+bMBW3xBmJaR2U6ECH+8\\\\\\\/Euhji9EY9mqGvWfPOEZZeTHRaZzUiT4WLom2jktr53vN4CNz09A57gvInEfDDcNu8Rf6uIX6dyZyjROrM6GBtgGZR2A2bOzuSFTfn8CWkESsNbRw8VCIDWKZ7N12S2iXDKkbo9H6vd2PAgrmsuLHnPx465ePDgkTDxc\\\\\\\/xvzzjVjV10j482dmRY+Td8hMqoYsBd\\\\\\\/aIbM1EE77H0JECQbDfzxuOa3lySqICVUgLVJuogH+bm5NlDDrYSnHo0Xpjpoy1x+UH31Kqp65vXYkLvsCa+vGbaOiU6klk4gvplMq7clK5EzZU0BaDXF17\\\\\\\/Smf3sDx2HagKSUjIOMySWnUJFWS42cNey2UeWPicEgYbIRnKmdN0CdVTJ1aOc9LfsRenCm8TdgEt5ArGCXikb4Dx7fgjsPeHCLQhpc6ffNj0IgJfLo+WuriRgojx2erwmCjmGCoMX6mupObCwRVu47lliZhZZLYx2i47CGNWXS3OFSFr1G2fvksCjaTpiMhOAcT4V5GLfqWZGzksoLvPG2kG5UP7xCrRQKz0u3zYQ3mZ\\\\\\\/1mFZLK+A30fC0mBX+EKVaBdbMmV81DT8xTeVMktc12Aaiei\\\\\\\/Fpe\\\u0022,\\\u0022iv\\\u0022:\\\u00224Emuie17zZ+fs5myQIJzeQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a9391c96517137f312d8ac1b97b05d4de6d4f76b0a92420714155cf8a6013a09\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ojxqi6WbJ7I6mv7LG4SgGJ3phhZoNz5aPw3hD3EHJbL48DYoL95V2EcSKPtfyGgqjn6rhYcXwmJI6s7lx5PbuFUYupClEuG9QeXXqQm0YHeKo3WBiWVtdGtHrCFykbZgKOwYRu7Tm8CtDskeAIsDpF0DTA9awJDrHpUs\\\\\\\/azai1bWNsbYhrTcwbf8fPMG\\\\\\\/ft52rt32xZ8GcOi+O1EY9lH90SBFzjqOGZBLClZfPOz38bQG3oZkzqQgdeUwrLblZFaAodIyxv3s4\\\\\\\/3n7rpRNSlHn66umtnMNGIfmaJPHlerlStDUSROwIoSGGn+Oouz0navWlNs+w8FcFRy2kc7morbm6DLydqo47U1oVgMD5hYm5a5T1fOF4fnXxxZSfGu+YyVLJ+VRqr5KKLSYJRM1jbAOy5WXws2EUFKVBbY3dQhvDYycGWVlHWI9K2tdyAxMEzcl\\\\\\\/Iqlf1w2Icv9Q4JoWDFWewG8bQDTY4judUG39qd+mQ+1Y1HL1\\\\\\\/RFK5ScCuUqySiqA\\\\\\\/iDH1cOI3A8seVlUPDFrOYmbLhzJK2MFw2VjeQQx0hsyZD3faNztCsPM3r7a6JeQyyQHjGAjw2MDUzp2fC8bu7\\\\\\\/\\\\\\\/QxVL+DlKGk1HhSsNym\\\\\\\/+EZc3IhEHOglyLF7nHphGDyCDBycao6ZtINnZ9hL5+9PUsXdV5ImwwYXnmBZy7jN\\\\\\\/GgKlfDTTdsq+YVpLuy1RQykfDLzk\\\\\\\/DVXT3MRr6yUDPJDghQLZmRLjZ3MJk06xDN7hoQDayzq6k7CJcG5MZpajqOAV2HhK2CVUfusrrXqR1Uh8ok8zmQe2u8E\\\\\\\/PQg9rBBOnFbSxg1zNRHF+6uepEHS78wigXfLZXKsnlMeaFD0mAYQUE0lKllvbopjhUahUBcRksmVw4S6u\\\\\\\/yDQldVl0nxBMHyFlzGu7DfXfB1BlZRMHSqKMef\\\\\\\/vqjH1M00IMG+He3jSCk0C3JkvFO\\\\\\\/RTyLnKQsk1pD8z5ICuc5AF\\\\\\\/RvMPpi+E6BX88DGdm0BOZTYXnZMAnA1wW+\\\\\\\/58yri8zDXQZ1o4cEa5x2jwUwB+4OCiKA05ejjaQOAzkTLcDmRrFyaJqCzSku8rHmyroi69pnyp5gF2HdwBxemAXUdMSl7wN+lxqWy5zpIbtfWicOr4RwkYtswAd8l129FpDT3K7Y3wSyZ3VPy6LRVJoWJ\\\\\\\/OfkgqLFOgcnqZyIQLQe5wQ4aaQyuV6FdKeqxTgR+01ReouSI3smejvX\\\\\\\/l1AjaTGkVOsXaqvc1h0XWcDXxtzlIRlmcg0+cIixLFbH1Mw5fZCRlZWwsDLJ+MD4Pl8o2m8cmaEn6\\\\\\\/\\\\\\\/6FWWSRcUAuGH0jNAeb1WW1EIS4WaTPVSpAQ89I7pmG7iEFv1TlupNJhUQYb0wsqyvgiMSgkVDr4JCzDIGkkl791PGGN0L\\\\\\\/EsPkUKElmfd2h8o4mgXL3C0OC+AjoG7uSAYSKOltklrWNyqvwxc3z0ndI7zgI0KVXGRWNsOBgfSiBUNwv7zfhXSLnF+L0pZFv5toTwy3UCojYvsKGMnbxOek9oUxHi6fsYvuPmlpOkRxGSOOCKEb9K6LdaO\\\\\\\/W22RpChQYPG2IqJBYc50okRgfmt+PKjvztiarXf696PZiEgmfWKiR4mDQchZaKJeQ+wCJR5yuCiuBCD6xgSb1meGjVEqv1xmnWA0TEFnEO9AlndPFz75uV5LnIMI6Fn7MmR4lEnCwf8jbOE5vs2tM0X4Vj9i3\\\\\\\/Nvfle+Nm0L+bUHSLvj6ntpvchIELJasuMHodG19o7gN\\\\\\\/ooHjbt\\\\\\\/MHBlDmlDQyqQT4CBpCIWm5YW\\\\\\\/e7i61UyjGb+6+r1W3KPV\\\\\\\/GgETAu\\\\\\\/vrn2s6SFxcqjeC6mmB3o\\\\\\\/vBd47wAiAMcvncRB+SGjPCemza6Iq7PibchMyuJiaeQatNFdnp3\\\\\\\/6T+Fhf\\\\\\\/Gne3Gvjdznz6kvfUtl3155ZeKYJGcOyJqmf85xtc3yR+H8QNo31SLT7F2tluiosxS35lOVq6ph+Vsmr8ZZ7gE0J\\\\\\\/M5aHuyR5JJQtnNXtAh\\\\\\\/IYZgNl\\\\\\\/9dfgdnaNqjtoBJ22Blv5KrYf8ikcq448zacsRP8JQ0+Cw0BoRy+TMXGasCMnD51V8pssd+HLdRFuxFfk\\\\\\\/mcIdhRnXzgv9tYAC0mtKhj2iS8\\\\\\\/r3z+BzRxTFBW03I\\\\\\\/nqrMXdWbGWXX0QxvOpTtMs2rAX4UXcjSqSehIN6g52fRt7EYhqBi0o6p+F2Kdexc8hbbHAkzja1XGtw8Z33Vrjluncr2LfOql79chlyAhR58+VHkbFT+il8VrjiLhrZpDZ6cFhkvJrIJj6UsbyeltQL0WqJdgdTix0OS8v6v2WwgBJYcK\\\\\\\/Cl7B5wbxmROu7XaGQGHDXdMfnIK+zeBjXAOI9Uv1wwXND6cK2G3n15nq65bYeWeAdFMBYL7RCa8l7DWx6mNR6TAuaeyPfNWrjAmIuF+\\\\\\\/46ZVZUAOcYd6eCA\\\\\\\/EYKwhYxsRZjiXkfXxjC064G2NltqloO+ZxqBFR2qw+P1edgtcmh+KWk2WX3V\\\\\\\/ZRTdCV5p5ZDT6IPY\\\\\\\/ywXTFS\\\\\\\/leLUuEuWlWYXdSdIePHDQ3UxLbauMOi5r8Po1mAx2HXKNd3NY3ejzfVkg7SnYR8jxIgsrsqq+mVgcYUJJtJxIdz4\\\\\\\/kYPjLW0sWnEOMj7qrJLpGQaMzrpiTtKqufziLKaJ415UNRxdX3y5NG3aJciVxQ9yZlFf9nNa8bqhjk01Q+KLMLaWKt\\\\\\\/3KzCfVKu4CQ3Z4hgzPiFZWoohmi8hFtNnZlglg5CyqXvrnxNq5pLkesBY79k4sNRoTiLZHEqW9L2OFSZp\\\\\\\/9lJmmRzZap6zTereoH+di8vKcAdcRVopD3C6YyQ\\\\\\\/JNMWe9i7Z8GNZXObMsT3tfxfL8WK9O4SnL+dDW7x8DwS9mgwcKI\\\\\\\/0LkE86o4BGYbKqN2VOuyyjxU8\\\\\\\/Jvq75omw0yUM0sl\\\\\\\/3wOaj8\\\\\\\/ZCZWVWX6F\\\\\\\/MDQ7DA\\\\\\\/dw4hxKAoCBgLa9r8qpbVoRyYZ2D7Ij74iUyn21PP1NseLXlV5lZqWQOVFVZ9bHQz0qwBXwCX3TQiG4eAvWr+kMJZa3eh5C5p8QUwi7lzyp39y3QkGV48xtU6XhOdS6NqcRFG5Ri6RjK3224i0AFTVpC+OEaIVk4z5rsDavf0ETZ2yuicqs1NAMllh6SUvK46uHug9xeh\\\\\\\/swphoPyMaV0eJ0ZcvWw0NiSBz+P6VTkSI9rnSSMiW5SD7mtN05RVyGmzhzv5Cm26yuMjihhY8p7S9sM6yaT3Se9iy+WNEHqiggD9V1ggRQfb9Wx\\\\\\\/cak0Sz54Y5EzsYPF032Be8YkJdywCvFy6jO3eGOxBLYt09QtIHM\\\\\\\/uqQFXsY2dHM4bJLLLQNFkcLE44G\\\\\\\/aW9dDeJeo0FYijCU\\\\\\\/kWmsC7nl6v77UwT\\\\\\\/Rrt4Ca1KKcdQ1dadhJq6E5MmszNioERiCAyErnHip1fldljmPVK+WvUFbD\\\\\\\/mIM0zZa0UulAWOzJjnGzMiIbcoaCQMtLK1gXx4hFCp82UAW5Boo8OCe7Hm604+MXevN+QMtS076CkuoZZw0f8k7L9FCx0LIbLCVSNWCgG\\\\\\\/IBb9GrtpiNuNBZz7IbvfR9\\\\\\\/NS0lSYKUd7TzkIyud5DNgtALMpQhrld4NXS0Zj5LC8jdlNOCzcHS1cOdocaMRlnsvGsxdfc36hKvsBkhOy5va3m42ra+6uQHT1nf2L\\\\\\\/IR7JDgbJ+mcLz2HfUCpb72kcNRY9uNFpP0fFJfhHucwB6WHBYpyU+0NA4SVluly+dfL5Cs56Zf95bxwFvCm+aZTCguCCqUpK3mMpydGtcjRp31NIPxWWEkPqY8\\\\\\\/\\\\\\\/zoDhgAV8UHsD2cpqRcTKl\\\\\\\/qc8zsuwG4BOSV2xqFITENlhJC+SW989ZS0I5u7pWBBUwRFVtK2aBlXJlcWOl64P69EsPLspUeLQ3agifGR7d8chA2BdoZN7xne3bEZT3d4b3QyRsgkgwX7pKZ3\\\\\\\/XSE0Kyjri8i3sxqJcn3mahY7rZLdn3xZG5wbxJzzHzQSZlsA0eyf+a6tEcqvbpOKuyzq9qGH5lNhT8PUB8WzE73StE\\\\\\\/RQCqcRT5HT2gWdoYAkzJ1T+Ay8kQLRM\\\\\\\/Pe7lmquL5tg\\\\\\\/NVTm4zu3\\\\\\\/32pP3WapJkpUjAMGNuloPD9wnPE3m8bUOS5FfbMT1DwLQHHoAmjtestPoYQJWBnsGzxsaBcOEsdK4JbP9L3AN6xCY2Yfz4UpKTnmQ6FyWYl0jbN024xN7S8WkOEVagIeOaVnbqYP1qYAB4By7by6MAz4alo0XJ+G8Vswf27P+P0lADR0EhsJgHz0Dcvular7F+lxpBxK8IO9VyVrRrzSvoZ1KMjguAoFFXRmDlZ5xG83zKfHcft7PQ\\\\\\\/vIyRvs08gWZ0nu+NhsyfBw37r5wSl\\\\\\\/Iha3T9EeS+r3UI4ykUILns0amqgsOMRnqaKNMr+letQr9baK+z5ZLLtLlq4tp223thwE9q3f\\\\\\\/k4p7vLAG4u7+ki3PZFZZ8oZrK79a+LQbE+kEsxJHXzvP7y0CDVvrzeEsaPh7YmNkjk7og\\\\\\\/q1linumHa5bks5fkQ5leeeem20qS7F2DUoiLtyt4b4SpZq6UEG3UhVXQYR1xFVxBjWrAHvfW+BRfrhqes97LiDSkAnsN3pbuHcS\\\\\\\/4vIx3R8NqYrLJGDQKDBXIs41BveuD\\\\\\\/T+HArPsIMr1EG1CtnHe8RX8o\\\\\\\/zIqXum9d6qahrPb0uFBGxFFctqjosjPPyGR1H0mzHvR5cd18u8mLor1NWqCc2lzFqIEWavKwjjQ\\\\\\\/m9DumaAawfE2fkf9gfj+VyqT3N+5zHg53FEO4cNi11iWfLxGVuOTTlXLpZP\\\\\\\/2zGG9b0IjHQ5Zw\\\\\\\/oz3wplf134ovKx11TD7atqeHYBPlQOUKjTqhelh3kYvHnUf9sRGaHmpui0YmAiRVzu6apWtmBCKS0ezb+HYefhvnGSTLlN+54594ig6\\\\\\\/iGQ2xF8C5hTKuewJkTtc1Nv0fd611e+tR38L4eUZuHe\\\\\\\/owSVS5NaVfVqlDUZGJuoK67OUC1Zkg3pq0KccM+x7fW+PLxxLs6p5Q1P7HpOwBt4EWiYYaDZgT8Duh954Em4HtjIAMQNal18msy86YQwZZSqQbZ1uQH1fwm7FC\\\\\\\/ex7+RH52wPfAWeKYeaoz9KqejMxpbb6jarmK4mldm9o9NjoxdkBP23mb+oRLmJqks7ArHjl3NfVAaZLxN1pmjTM1p1Zm99EREJ2C9rjuE\\\\\\\/XcY\\\\\\\/m8uOTI\\\\\\\/WaC0rxl9jU5eMe58IK9Zn8baN8wZE9yy0CHIKxDmCMhwlDfkaEjMHyeUYyw9vER4BYXZdjnKyULyIfDY2cdT5blIUQ5xNnjvHpVl\\\\\\\/ge050JtM75XxrNjtNHVbZx0yfi6k422KFuKyUqSqEZBXsWCTpskywAr7wdCvD7TFrt8kRYqFtqMlz2BU1RnVsFyWU89drRWBoaqE34KTLRHTJwdGllOi5cqLY9uwNm6YrCVbrtacdoqQ0\\\\\\\/tTuh4laPNWLIpEiMziKDaxe3yxu+Y1Gj9vakCqnyD5wHjc7VzI4iKIyDMptgWL30dls+3ZvCdZ8e\\\\\\\/rQ\\\\\\\/2oICb10xBytSCSNk+jbUKcC7QNIL5gkhMA2N7DYaD4vY184B5MugN5T\\\\\\\/qfhq9JBq1JY2D6nTGufZwdZYOH+TNFrY+tjfvLVN00OiEib\\\\\\\/XKW9hLFH2VHLwAHgQeWN49wFQsGzmo3qw75Em4WC4dXN2Q6FYYSINCyiOmGRZxCpvRtT2d3qLfBRsaXU4YGoMAuxdokpOfiegtMNF0D4PWVEiZjkYCL6e6CzfZHEkP\\\\\\\/ZLXJWU2bYE2MROOSytsGGIfNh5VcbjyEM\\\\\\\/jYjssFSU+pi4G9zBFeDCvax6yV+I62SbmrQknBE\\\\\\\/CG1ex\\\\\\\/o4w6RmhkuxooxVIlenrcMCWmC5qk0DXSPh9GqnwImK\\\\\\\/Pm2omm4Bhujnb1eWRp8DS6xZPM\\\\\\\/E0dl39qM2bvdvOApPfHF0P8Gg5+NULYPwrxYUax5oB+Pipb+ZsExNli3UX4xNNYfRsPZyMQL07\\\\\\\/g+eECMXEdqyQ7TUwHEFo30doKs6Rhb\\\\\\\/oQchkyS2JWWNIOQUNOI5ToggyVcZCTuc1QhUvY4UFlHqxEUe4FKjHKP0AF2l1sHe5m0BrFy48D4pqdW\\\\\\\/hPv0H8Mn6OOwIJCxiqbxdyzZHFchFqZNRwh86C9JDL3gifbLnxrBvwwvKv8BExPGe3hPO9PRUXdeAfEDBuw4pNlnfYdWS94XYT0PTWponElQflmfa89vsYxR62bnYUXJjQivSO\\\\\\\/jfSD6iovCBluOCb70PFu1HF74aiaKpEKfzJvj27Uv+j9bCxOqm4EFrb8ytcLgEmR3LhwNhJ5HOESP+7VXzQt1INQuyMOpbqzEdR1itBI2e0934Hk01OKLtlEUWBrx+d\\\\\\\/pcJ0CpNMCiI+GirIIYE+o6yOBB5duP6VfCF8HTgQZwYaiwlOcko\\\\\\\/wXfvMav9gJh+64LHN64+ZtELkgXrLmjayhlPtaD9TK6\\\\\\\/7RCgwtmvu6Dv8iefnNVAJm4MAqazvfv8ZUwZNSNtphLef6pOfHHvXNXbhhPj\\\\\\\/Bu3UkVSooSnltVg25fLxdLla7TTLIL3\\\\\\\/Qthb61CPFBU6LpC5grkl3L2rrF6LgHVx+iSkqqPm4lGqw4C3W39y89U\\\\\\\/9KXh7\\\\\\\/\\\\\\\/BUjrCRzPSFVlcQqVHEnJ\\\\\\\/mqnd2N\\\\\\\/BDRTcBkfm\\\\\\\/jug5XezlGle6b8ckoq7Qgwv1lBL+WJT+Oq0vlxlDmQCn+GFt+ofGbRNH6wLg62tMYmHTDtcTJZUn4tKUwPDkGWu\\\\\\\/MuwvDV2Af8K0cL0yLi5pLKAcCXInGQxkgnpMy2geiLa3aPQUj5CA7FOMLEEuGoBCSA0uXJipTtqMuNpp7HEOIkRTjOMXHRc0pa+hc6FiEDEl+4gjrJ759u7s4vxqgm86iQsAqBcXKyvhmwyqvLsW1unrTFtvkPoVoAVlxniva2vumTF+e00xMg5sdA9YefrDuTEFeEGnf3CQ16p7nFNmauTALUEfFTnhLlPe5QJ6jKJTQom1kliWEsXY6cl1l7kScFqAydaDut2BOAsTEs5sv27L1z\\\\\\\/zSF0n9OE\\\\\\\/fq0HBZwUU920R+b4ZQirl\\\\\\\/EwCOo2fl9AVjf8hO+BxtDCGKdGfAbbcDDAtpXTGzqDfoJ\\\\\\\/ScJVi8XY6N+z2kj\\\\\\\/9jd9SDd3ENjwICPvpNRnlPq2W1sFQHkvkxavupeQ6oW4a+Loc2FGCaJnhDHu7u5EQNxXUpPjqqS0syZq1gLuO7CmMfu\\\\\\\/2KkFWzhqTcVt\\\\\\\/ZZ+fgrtVYWuMn42NwSGUFIq1n21u1XTTkMOuLQOc3YiB\\\\\\\/HWFH8PhimRVQhlXp5Q\\\\\\\/LPewZwuWvPp7UJBwsT7HD0BrKrrGnH5UWYk3INH\\\\\\\/UMnUqydo15Uwtea3+XPTsWPR3WivnECBENcTRzVMJirkm9WAOB8hLeLMUCFehGUOmIQfCdAqij2DSq1wI4ZYOwZAm3FmSQT52n957MY5L5wd\\\\\\\/4j1WrXOmEwzhMm74iNJSQzDvzVBEjUQYc5cYrDQaOdAp9KCImbbmgwbVLmG64u3Gv317HBuvNgytyaduvoukoE4gdjZ2QgPXHQtlSw8rAtMzpQoNKu+4SEiy8b23sSaZT7BmdFlg19J1cov4+lSeUFLCc9AHAeL1rGmkEMZ6uyUw4xVmgsjefHRLZZ4eqtX+cpq41NWYW9zes2OWwiBu7gbVncMaOYGSq3U\\\\\\\/BcOPXTn78ySpAy4RMBekzWcrmeN3+eIKq50XZwolzBXBysO+fJXHxL5HyA2Zi06JQCS80ccX8a76IajzPFphhHEOhmu5PVpGMq20y21u5TRrivNS1\\\\\\\/TAT69M8VjHcN\\\\\\\/PKKZdS2Bxala3dlcuhAchsaYzPEVGOFii9LPGnf2tF24x6ruAgB2uTRa97OM2KP8SiDO3BZw15JyqpLGiFtxevzwAwCTSqvdLQB7d7zVRhduYHQha+qggynYw3mLjQbHOEdVgT\\\\\\\/Jh1bpb28vtUslfn5ec6sbn3KbyWt1kopVTyKbEyuu\\\\\\\/AttJu7hUGUMcZf3hIvzZkj5wC5QeRB814gNH4PUN4G6p96eQcfbYZPVS6FiDN+48jdXo58XjdhVX0E5yDs7BU+WfMC49sCA5cvHDi0qzoD7IQMIWqukdf2HLOyIn6GZ8B\\\\\\\/6G4Y21Q\\\\\\\/j4ru6\\\\\\\/9rJKmG+Q\\\\\\\/oyTZlFZUIE33l0u9o\\\\\\\/xAayY7eO2LV8lKUfD2zgFR1dpYv+oJyG9B4D8J0QoYz+1dzIJgU48aGf8P5hWYRjlf9PvAOv5GKeR8HIYu6sG2p9g6fNXNj5IDGZyYzghnt5y5Ba0sarRok36We2HKZhLBPPRrob24Rdj0XYyXSvaPwHsrmL14jDQAp9XVYe7cU\\\\\\\/VLLPrXR2Vf7i2YfO4SkTf3VJbuGM6p49sG045gf1zk21nCisI\\\\\\\/Pl94J6ie8IgpDKPEDhhO5ef+xw\\\\\\\/iz\\\\\\\/Qxz8INDDJ2r7Xx30MlRT\\\\\\\/SPIP3jF6kZorPOk9d+OcAF3bv9nA+ePaH2hhmevd3MYYFslAg8GgTo77OSNQVVUFis1PodfXH\\\\\\\/TuZ86VGNt+WjC+Sug\\\\\\\/fpyPNbTQHeDskaY1Kc7kEIlpVAP5KathxQQUHAOyr09FBm6IEFtR2wleDj6X8dYBeJ\\\\\\\/vKLHxx0vLMrVN2rFwld5B+A7qB0Wok7W9m+aLo72xPQbPIghbutt+a+BOVTb7dwp+KCOuml2pcHxCGIxXtQwVPfUu2SXTU4Dbm1TGUtjOnpOQ9NevDDkQtWJyYuwHudq8\\\\\\\/A3fpZgSmKpqsv9vYL7UgMXmRMEcOkt3XSCt0VUw7gDZnXS2xrxvLM81\\\\\\\/VSe129Rl8CdcCRho6eEHUKYDDvJb6O+BObT9V9V7JgovFRqGIFJ4Fp3vsYf1rribauMlWz3TMaLgRPvaJWUCH+rjKIm4GW\\\\\\\/KcQxQjezdPLn+3Re9cOpOiWc3Z0OHakA6En8VYwymyf\\\\\\\/VIot8OHwdAIDnUSymc\\\\\\\/Vn4a1BeS8Y+wpLFQC+KQ3e5aFiOJmW21ZPVeGcry9yH4qV0W6PJNmsk\\\\\\\/jPXMJWFFiDvxRfGTbzsMw8IXaV3Pih6fYzvxswWhWe1aWy4TfgEYaYjhZnWLO221\\\\\\\/tWo3IczKPocxqvto2IGc5c31nYLmxu1kgZ8+ZfjQi4XhisCEuJ3nFiLWYxgo\\\\\\\/CN57Vv5Y6IGhgtYAOMuAYJuNf3AQVhbXbAnoOqzQBO+VHDRtjH6ze8Q2xq8SFlB1tzDB7PCcZyTF6FYORQbMsMjHL8XNzuafBOVLc4ylSP5Ir2v7fkU3oSp96q55UulHcVfu+A1pGlPC5rWEtqZhwkhIDeaWb1QpmmGM2RmMl570m62Jd8pUeiIip8+MzQ53TrmZIHG9J6RXiaJGNpbbXpEexpUF6mY2V9L0WTBLPF+Ls1y3R3JZY0jAuprwZarLbAaxokpU6tcXQFORsnN6F7ETXerM5R5UfGleo3p0w\\\\\\\/2WyZaJZURCFPy\\\\\\\/Fdg+azZyFx9e8XKT4Gh9MJTb3V5bwboVWpHMVIdrDGUXKksYYUWwpswhWlmJh5xH7nMwFhUSf4oaRlbQZgXXcRdfZZueZXKhLuYeZ0q7auATdXGm0laUoO2fg40AUkQ3KYQwEKHya9KcUuKOBHE9CUIhetEMLMeHgok8j4Vkph61CJGP8gM+C1\\\\\\\/TU4xazTCizNa7IanddikIXICJugm0Fjd06RHVkemotcatkekW2s9yLPbxNE9\\\\\\\/qIwGazOF1PPi2MH06QmlefINvKQ98V31YSL9EaH4IlxxOlesbmdJ4VSBv4DIjC4YzLRzmBqeZgslFe0JWOqX8U1NAX\\\\\\\/Zvt\\\\\\\/hwkqRZGXkuioNyOqLqPoYnr25wFi5F1id0WLFbEcVGnGPSn\\\\\\\/IhbgCruAXwRexlrYuI1+\\\\\\\/mEF8d6nKRNy\\\\\\\/I+Wl3uRdw6EeucoUn7CQ6lGU77\\\\\\\/dyCSgTQBrhoM5k5w+Y7C\\\\\\\/s9meGfG1V0x7zZHwsSdLNm37INOHhne6kwe9KUbwCu6gqy+ThGYgRNm\\\\\\\/H59nVboIkHlLA3LRC4rIymWbWo3DGooIuZaz2939uGrg5I\\\\\\\/JDw9+TdWx9qV3MAItE8D6IFNvpVLbpTM8BPE\\\\\\\/636ZHGF4Ne9RDYPSYB1slWzkI7LwSR4tuwgjLlxpP1h93SuyXbkYKxwTGRCxm6ZbCnBLfchsFOsYyDuGTzDSI9xDOTcZK49fuLhCsoT84sE3hEbRCdrjT5E25sbiAuScnl1Lw8hTTv0IIm7ZcarLiCrRlP4hEi01F38y14MtsrZ6CW+YDfx+JpK2fnCb7eEP3QQBO6Bhxwv2uaUQSmJoTlOieKbOuWN7lZBSzibA3xaHZGvgIyJd+WasIA+ozfObEk\\\\\\\/5FDAT0aZOd0kHrOoWsqAPyA0qjFzpQoBeklQardQaVRiFGXuHE88nF8\\\\\\\/G9Srl5MmPUc8CgVUktYa7CquJdmVvGJ3OSs8+ZdGRPM9fTQR4I9hJGvnDmzy294QEf6kdHuIjCz7Hw0rIZpzyKF5XA4T5D6mHmwoPBaGOYOrBJ1vNT1M39lBcbf3EYi5Edw747cL8cB5cZUWBqRBoQTIX\\\\\\\/UfAlhSwtfFyW27239UIOIMaovBjYm3OSTS8+3UuoAuNqpZJ09OK4tebQwWX\\\\\\\/96o8VOLnbz4syRQ1iGZ7Zf6Z0YENKFXF+IBUUnWEZytfXpLJYx2GT2NHYn14a4kJ5l5\\\\\\\/Gw414vdql15f0\\\\\\\/HXt+oLDt\\\\\\\/DWqHQojBeIHrY4\\\\\\\/67Lf6pMlr6nHevCKhF5M29ulOuU3d58\\\\\\\/HVM3TOV0s5vvtnleOtVpNWMs+\\\\\\\/F6V05HQNXOw6eG+n8vCp5L3jbVcnsrxEe1boxaH6p5miOTpyYHG45O\\\\\\\/MPFj1w0bye0FwQx7FhwvEinBF4pua5zRqhCAyftE7Nzs4oMBekbJmrgkN8AZMhhlOwKYvfO8j7icLLF\\\\\\\/yer3tzITGikRpKIeTt12WQG+d8IlKFf\\\\\\\/LJVd3vEb8rakd6ksR6g1XTL6bBdRBVjxyNG4CePAYoIj2Mkzlvm1970xYqLOd4vqxxJSd8L3KDGtJpjSHs99eac4ORLEUv4wc9AoJabuczHBd35nQ2TyjB8IPOJpKWOctVkMpk0YQnTWXSj72rG2mroeVYSbKki2foTzlshA2DmwOciBWd++RzI1M8PMYJ5kEIzv1emqzrcNudNX8886kUDtFgJYUss5Fa2nGzdtWpuL4\\\\\\\/3F27jM\\\\\\\/R4N170VOnU3LFJKOOZ8hBiUfy5u\\\\\\\/WMilBUxyf3WN1ho6tO8wAwh4iesAkeunYW5EQMxyNJ3YnZcXNEkW6bAFSNfk+iq+P2NjSUdptivARZVZoV1rGTyE2rtRMnLnhEBsNWYvQcLFwIc7ccQxEse9aOMKSMGP051rYu7OJnriocUQBIvFjxL+aukQvaRuZbzOEcZHA3TzYBuI3dwpHwelrz1ZDecOkSnjdWETYEaXgOWfANIrGOPEox7iCPEVvRXV\\\\\\\/dwIL4ZsaJJSdrdRQJywvCcVcirKhD2HPkepVFgj1zOxCtBoBSrzzk4eZz8oE\\\\\\\/oDwqSvd6od9YNvaoAul2MKhhdqLPn7uYR1za1d5V3i+kV7wVm4ZZn3UjwJTUMngv84OP2JL\\\\\\\/S770pD1RswBfVCybbbgtJO2X\\\\\\\/KSzJdkYUqEUBlZBI0\\\\\\\/fj9o8lhkjhYpgY6ik2GLYr0bq8u6pNu\\\\\\\/pvejhkCTh1ILSHRVsIyBfaEiMz+h\\\\\\\/WRWgyWBx0gv1e5a8\\\\\\\/ad1rqOqJmUyh1kDdTMYHSizMly0UsjjS99rgx28kCZHlS9sC1tQlNDYLa2rvZa7ZfyPEzUk2PVHspcgKhYGZ1bzwxReLQEEYZSSA1jEpv9ZnRBTETccOoXbaogDfrt5N\\\\\\\/+tbd97uRHAIc1U9B1rso6ER01qHeVgrhIbY46ehXF2LQm1ERf02C\\\\\\\/cb7p+WccIZFkAAPn0Zy2Fmrskn25WW0Y9QfiQAIK9Am9ANINM9N+m2CyneSHYm9s8+8qOlck4Ftq5XsisGHZHUXVdJ0MtVtCFsSrKHpw4UDYbxSH6BTrWSxeeCocLkY+o9vEiD0bQs7d96Hup\\\\\\\/NMdo6gG8PgcI5QmpfqcJ4HURy0o8Cpdt27SxD10VvQyOedxcNeWkQrU9G3gAyo2S8VEAWEAosbIBFYIJJvoPK2sZ2ZhbpObPYw1BxmLISJ5NOJ4pj9arBeV5\\\\\\\/x3kcHhTTl8++YSPp\\\\\\\/tUeY2k6kuY9pXd6lgB1lGnAgiMug4d41hXin2iqep8hMbugr89TJ29Mj34EY15kEkarpm8T8Gzz9wqPNsLlHQoMTh1z9IdKhES5ekEPhiJOhWRX7zzzLhJaCvHjZokUx9Pp6TZ+b+rgo0\\\\\\\/Hx79RrRtFIEYZIVsPKPVEQkZa0D7NPpv2CVFQgnpHhQzIGu2P\\\\\\\/XWLrMlyPv3qtxbtxymnoL\\\\\\\/wqbKn0ag\\\\\\\/iMPpfZ9KCGKz0Ar2cY+G9XyQpdkrOFn0ysTz\\\\\\\/3HS4oh5XaprZYlFWxyUkFC3ZrwLiQGr4S4trIDIdxFaDznZQXOLgAnX2oZwfFHOm+wq2kbvVZlIGkmMu\\\\\\\/n1WPDzO4F3lp\\\\\\\/ABQXgLaiAa9zWph32gJkoO+5wtR7fWqpZASB3QmpppXqUGsdIau7X8Gb6CLUjUWWNuALsLTxKA1hqHoVZ\\\\\\\/oVjWpsayvZURnqpWG8Tk5Z5LvkEXmtWoHjget9YRzs810lo\\\\\\\/dvDO79U6S4VTGmDqzz5GfVgwkeDQHeaZbKT2YCl7AuoO\\\\\\\/DjQ3hJdIbxtJ7tnvkH3IlffH+0iw6XxBoVIzFbBkQov5XuRZky1wTIKow\\\\\\\/ZJIjlOhKePl3syvs9\\\\\\\/QTPZzhhXa3RAIYEHJe5gkbEX\\\\\\\/J5J0hBGvzgsVcawX4x151i2128zN\\\\\\\/cWSFu\\\\\\\/VCt9LYAOPCOxhzAjlto2fSQKdRrMPQLIDhThJ8vuIGkuO2Qo2NNNGED2n4KJnNJsMCCYcTo7TfHfZ4mDExYrqHHn4pJ+cMEfjFhw4OOiK\\\\\\\/+PdM0XTVDoERB4TmrWCy7KtwThYUPNnapoc4owmmcwGEPvmprIEEV2qPh6T3D8z9AqWLN4pUdcS1Y37b7FhGf5CyLdBP\\\\\\\/sUO0w1crezmrkNN+Uo3obV87Qj7cfXu4Y0+FsrR1mtaHjxbNhwBw1ZfrIl1CW1efnOwcRVHUsrqgFz1kh4\\\\\\\/7i0Z5FgHt7g9TU0PJNVSGOgefb9zEIsTT7Gc41rXCAXM3piaa4urQQGsnlIa\\\\\\\/WcbrGnxsT21cXxqC4+6GgAhFZzQr+pCHmyeekZsE\\\\\\\/fz3b2lDgWpA6A7lacCtbCh+IwUJ0TAzd7+cYJaf9zej6dCBiWeYIqQBQZwqpC34uKW66cL\\\\\\\/X7OzsTWe0pJ+2drn+5n6h9cvAUtQrd5OoJLQpQPCgY7dCCHyBpzDUpV2FPbMIzNkpSFRP0RH5ZvnIbEdhrz7maMQhwgaIWcz+0GVwXS41qqY4juoLzzaV+ayedKx5njZeYM9tAagztJ9Bwul6YZS44B4yok3FZJN2FkJjlyo5JuzBwMpMHA1flZRVj80MUrPwQHct\\\\\\\/OuXsBznZR2JmHyxKtGTDdeMJlFbG5n4hG+GcXmrdlgjE9mM\\\\\\\/CMkdQu1uGCskBr1zycewSrzqbL74Hft3PLip3deZ7RcHmaf31+xsMOBePrxgbVXgvneRxH5aPfOMqEYOF\\\\\\\/of5gXal5NFCNVPW4lwz86nk1nnjf7TpMwqMaHzwG\\\\\\\/nvQw0ivv7VFPeLkjahOlziLGHBU8+N\\\\\\\/w5F1LKriZjM+t7it8lEdpAmSYMEt9HHD59XB2iKDKHSCyO\\\\\\\/7M4SrkF4DrXbnk8WD4+NhwJHwps0zBlaOTqca0sOSwjeTf3W\\\\\\\/2i0IWA2uMZ+T5plvCEYDFO18A27MYkNS6X+fNCZ1du7LejQHDmyo1JiMKYCKM3qOWhrXTTQ2C1yPMUPHsnw0HnvSjSdWg9nNYBBJEscfUKMgay9A\\\\\\\/5lNjjDE0qVSKvcBooq+1Ah5m9LSjnoXR8sAesCW5AO4M4X6jF4K\\\\\\\/stnvfbc74FnvoBCAuaX9Ql7SsPlxvPoORkjdOiPNjjByG0\\\\\\\/6uAb9FUqIfpbjEkigKWzW1jmp9bo\\\\\\\/em+Z7qFE0jIPtGGdMcWs4v\\\\\\\/raNO5jgOFQWt8WmHMF7qPp2uu9iXp\\\\\\\/IT30XwRnyzsudJ1jijBoeYNButXLRVESQ1GHizw1FsukCWPO5yh+FUJrJQJW7XTiU6ydMX1EBMQHIyue0vhvst4vbtD4JALwTPtbQmgf0z\\\\\\\/8lrtpneOsn5YtS6n\\\\\\\/S0VsqAtHEbfDdms9\\\\\\\/Rdi+MG96UU1q0FP2aLzuPt77RhJAS5XtObn6UsDtwU3P9eX8tYWYCe43qtcuolCOiPsa0ANa6pgrTEJ\\\\\\\/ZiWqumimTc4VfrvTLY6seg31C5qJetd3YRhcDOhVTSfHWuCPQ6zB8Vzk4Hjh8a6Pay1fzS\\\\\\\/f10TG1bFe+K1JhOGCU8TdNB1wHfuEwa19uCldYk7+tmpgu0b408DMtz0OSVESxGU3Gk1nkas6LXl\\\\\\\/pg8+jRHvSwmXw2AOx2z6Ki5bZVleb1F3dUTxofiFd7s7uX\\\\\\\/B74DDQJtWXk6lv0vdXaiHKa\\\\\\\/lax6yOg7CR9Pz58\\\\\\\/7ocigmDwF1VJVi3X3qgJGRof7auVbM4EUfOztkk1uDWWrh9lc6KT4SeikPsjuSEwILRqHN5h5HhB8IPUzMqTXPWrFIV97FShXhHlnm78DmNKE1LEyKFnpLjc\\\\\\\/AhfRafo7vNlhodgyIOXGdZC+oXCk3rqY7JWH+D89qeuSRTT89nfYmufioiojLD\\\\\\\/wVOsR0NQKuXzVrvg8FHLL53128Os1ZnkHT6n3q5sr7BXLOwBA8ruC\\\\\\\/9To3wi7EnUHpnIDqVme4XOLj7SNkfEKH1LZEw9G8BrctCRyNKi4g+zLH5SdNNPBdU2LmAQqtnowXQXdQ1D\\\\\\\/AKkFaLJDPRCfPEFw\\\\\\\/c37NAgBq9d4Fz\\\\\\\/hYsyn2GE4+S2eTxU5JbYvxVxLxpA5Z0g2mtqnTZOnW+TKBGHlfWf5lOy8QIlbiJmD5xzmobmstZdRoRxbxHjB2mqfUm1WzxlirQm91NQH9eDOOkmbXN\\\\\\\/UwvwaQOmlkdYUwBeFQqdHDmZcerX\\\\\\\/xk\\\\\\\/h3tcHLTvJGKJuG\\\\\\\/EoekM9sWk5BaKVRIXPixv9vVRak1LZYJbBBMflmAPhXUalMmquB3pt7FuKcX8KzUWs+dO0dtTvk7bmGIGIabBLd5rjnQjw14bOUmvyGam7cbgc3hQbVw8zqrl9Uny+qH0tkmVwbf43eWEKcsbn7voqIvUm0lfKWq2N+3vbxXSqM0Mlbz\\\\\\\/oBxeY6VwTp5asAw6kZ+Y258bj25bmx+oR5c+xvAbWWBdzSeXTuOiaEV2+5PiswVPgDaJmPmdMVeoyPs8TM1aSyMJtK2EdR2RR9Ucth11nyz0Nxu9hgdvBw27lro3YxVuo4iT22WxMRI6d1EumbHIJ\\\\\\\/S7aJQjaHH52\\\\\\\/vIISmk8rJWbeKsplywpKTekXwY2iNWE4Xs7x4i70CfpjZAzvn+8e0nFFX\\\\\\\/mggq5Ox\\\\\\\/YfWqJU59zXOj0o5ZzO2k1b\\\\\\\/FL4nFaD1YpBlLIvK7QpIgHq+ZpGVQyIATJE+3NPY0mblJw97uNSjRQq201ninfrLu1zAY+CCjQU\\\\\\\/cgJAmbLa3Sjns3H5SChy8Pdnuz1vjuRFVF\\\\\\\/9lixo6y+LNYuBETvYP2DoXuMfKsUsAmTq56U85EBItUXZpeGYPBtw3SlmKCNNUBI4LC6sX2G3dPSB8+56dNuooCTGufeowLj449C8mgQBCPCGG6ZKnf1saeFlxgbvPfsXON+E3gB7NZFBtjZNZLo3pjx6+GPbe5eaGhFGEYW7gHF+qKhW8lVxceW3RQkjuiuQIefMQKiRxStpu3X5uKfnWz042reYW07erBkFeCYztYD+LOw42yFptkTm6xRvV47LL8Kem+uf+0WcEfIxRnKlNtn2hv2lYmQtPejl6Q0pPj+o1ek2UwqnQ0eQ+qlO2mHD6gaVjn3+mctv6sF2mVHUAZqGzURvLo0QZTRaGXtOB7eGmEfZEeoya7M3dJpPOtqQYJcEHaStxj35l7cuv3aVx7nxQQOYjGbZ+hCR6cpEFEbLUN\\\\\\\/\\\\\\\/QiHOYjR7CrBUogUEQleQT6V48TJd+\\\\\\\/HKdQimPP1xdD7tmgxcGLoXSsrQoOAhzB2N39A10C19LsApBYtC0mF40ZWqho\\\\\\\/79B9lXRS9dgqgWko5TpVsybZksrlvhj+LnvRi9r7W97aNKN1GLWeCCoduc\\\\\\\/CCUFUMR81Y6WZ8p0hUFOdyyUmvDnVEYpBLjy5dpg1wUI2t5aYHR9fzdB8Qadc0bMXl78B99oOeN44Fm3Kd3KYIqGf2pzsvuO5qNEDwto99UB0pNEMvfZmqtZM4H9LPhfKNhz7kUPtf+j\\\\\\\/mzKIr8nuTv122yvWxxOJyTEN\\\\\\\/T7y6BdIQlAaLGGF60M7yc66bpKyumuNilK1rnu2rfa5Wwjc\\\\\\\/6VXZeQFVOl3H3u+VuYMCjibF1JGfbKijgqxCsw7U4+JVy6OFUiQg4UtogfBZLBw3dNjlGwTOggKMEIL6ZIBwTyQEGl3bDlvCT22j+zDYPgp1PvprrESnZ5nyeOurLEHMnmboVmEWF2H8QvXDwFUMBlu+b2gX10lX7pwRFMxsyoMBtcgk5n3N7CoXd78IiiSegscbdrStURbmW7je3Ts0YuG\\\\\\\/DL34Jux8meMLvyJMEKUL6VEHi7zp6W\\\\\\\/g940K4xQYjnFHfx9i9lXL3RpCqowwSFKWSRIHVPljUqdgIQDkndSIDbaSuzSRmZlWOUKu3jfVd0bLjHDFKDux\\\\\\\/DReXfyd9m1o5HCbIN7rzM0bRd0AWc2guJIZYt4WwXx6HkavzU47HNBpfjmehfXHKJ\\\\\\\/vFsiNJSJieTaQa8yAnSM9yjFI36+5RNr7+B3PJwCNCCS0pE+rZ8BWseVESQWXVTWaSMRGfPCB2e\\\\\\\/6lTHODfRyaw5gCfndZAiMBPTym4f3ucimI2xWITOAgXTITY42YQd9Tq5nZ5AyRcUEVwE3Uu3sohqgt12GCew0ayW1lz30shsMZUUM+WW6aybBCPcaGK8hVR1ikfGJUMd2ZuLA7EFBF+1IWeAOmQQtD7fz8zcUpJxPSF3YA27rE36hqF62eje9hferpO76B+5bvSYHqgtGeF6dZfxIdR4qK7Gb0ryXoT0d2Zy7CwguGC8xiHUzBLsYy0EAKug0k6yy60B0Yu1HT8jghXarEGSiXYJiHLq6QPvww\\\\\\\/fALy4OyVRvqvHwW7ipq82QpAgY6LeFHL1XKGz+omQoDCbozSvaEOkAENbGE4WFma8ay7\\\\\\\/SwcbaURRPi5wDLFYKPYBbVLv4MjTlzyKJ3cXvUX99l\\\\\\\/UOa3BnFJcPV2aNsNCR0\\\\\\\/BSndfOsSJ8zD7uux2+Kj6Ke8Nbix6v1BG85yrzhPZOvFGc0KZ3vkAL3DZ+ZsRg3xPerY7muNbjISpM0vRTmczyInHabktEecKqePNQvmGnlh76WQIMHUcTnHt127dgjgQvkUx1cMUGQmiwq0T4VG5T7BlTUraLBrtg4\\\\\\\/GIhl\\\\\\\/xaxu0AnHXLRZaW86R6DiwnvFsXPxY\\\\\\\/3cNPk0C4Gw8audvfl+cUlz2\\\\\\\/xzp3hOGraOxohsNZZNIv\\\\\\\/6499Kzd7qL\\\\\\\/pwvqZuuE6i+FCy\\\\\\\/HuWyBN6L09Tsv7nvbLwPLKX\\\\\\\/UNo+BkSGJiwTlArZC2BoDmbbVxBk1VCpIwAk2f3yvDHGUOLd0qMJgHdAR\\\\\\\/3OXu+uVYGMDt074XNF0i9GciWl9sfi1h8tSExyFJOycEsYxNdYa3JzFxRIyNfN9rLzoGnoNWI82keiCB8G68hnqnH3zaSqloWHiAvX5+Z7O2bafuGFVqhAfNsHviky2KzaX5PcuUjFTbD37+SI4FlO4z66OwkTXaXlNJ3eaWFHnAKmohDOm8VymgVpdDtgUOCU0Vce0h1MKj7aEXQFngi3deQE7orPnB+5vM4Qs\\\\\\\/2W2\\\\\\\/VeVrJRKmpwUJU91gyueDZuJfz\\\\\\\/Clmt7wq\\\\\\\/m+6tR0hHBvHUwZByaJBC99bn4EzuEvGXhBZ5xR64ENiy8DnJ329Qqoetn5tf1IdRmR4lbRtpDBIW8FrBEBahQ2M54N+iBiV0nk4HNyF8G4WLFiR6HfltnrkUyZWoercy8hZ\\\\\\\/1GWK\\\\\\\/RBFBPN0Gyb\\\\\\\/WgEKcC+9rypLcV242XmwWaTvmzSphi+nTkReZ5Y8NeUTv4Os+RVLO9vIu9KIbkEERS7LyjPNP\\\\\\\/9U2ALL\\\\\\\/Op5dgjRa2AZ5vAY6C4mKHBP\\\\\\\/Ymi+JZcSeTVwqX+KpzUaI0lZeOxGau9sVQG\\\\\\\/pEpl6luPSORO001a7EULvN61suYGPb6YRg4mc4h6pqexw+dCiFJjUVU\\\\\\\/EdnW11r65ghhiOVyqKggPCgID2XINpRFYBvvXVPlAc76ZzvJdPUA6TGOt6j2tGWDIlCTeoSqJS6tzrRKk1sL796PstYEOFmXyLjJcY0Q0to5YVQOfJneeVA4boKXnt3XGX6BnlNeh4Ee1GTkSt7Xq2NqpChy25lQ6Ip\\\\\\\/pBhryqEgEoJQctpS0hIij4ASieiZOlua4kfPHT3Kx7DLcjt2F9CB790HRIVxaE0Bv\\\\\\\/BW0A8oUmrsq3SUp2InnMGXNhRQ9bIcpbp9OgNElrdMwnz9EZN1iQyNBvF2JfVL3YRUF+x+U3rdm6V4yNCMORb8w1L1ZXabZ2eXnsiHCPKdAHdqqbKEERolCHVsPiEcBUrVcWfHVGPxgCASQlSSRJ3eJRn+17Ryw3oeHTGLeixrnW6IeSIneEyIvnBVXMAK7h1u74npwZyTd2ks+VePT5AQ3I05kmK55OKjJfU9EI6uV6qzcCFfZICL7UhXVK3qR9wzefS4a+PwHUq6ZGK0naDu+ff3wHB0G82HXsb1oAMUPRCHH1X4NDHE7XqNrGVgKLURJXIAqN7hFXMbkFcPqmUtWF1TNaBKelhuQUCxuNfTbkh2Fg6gDEWy8m46IkAbenHob7Zh1+dOZsLJZpvc42sIq6DBMY2B+\\\\\\\/l2L8QvhjE9tV3WA4s4dMHRkxRwddYaNLIS0pBGllRLgVb+WgseusdwT6+xUyi14b70az+b90Muh3NkDCJ2sLcu0NzJIjFhAE9eCh2y5XC3hefcG9cj9xq3efHPe6dvdny13KDx7s9ZI0Woh02X1qdBAh6QhIOZTJWmLqPXAUMHzW1prtRaU7OcpbYmEEjTrka3tf8N7+Daq7+bNZZt\\\\\\\/2dFYX3GS9wDHBDdcS0THAgvpbgQF95T1PULh0cP+TD3E9swxx8BYfYkGWxslPkD0GztiXRvC3tfbVpmOvCP72KS90G6c0wITJXLug3ekeeOyernB1rAOQdrWPcpT7i\\\\\\\/WR0Qa0SVYBohoJT79FhR5Dn7tIcXRS7A0aqHwwHJfjmNvfXF2fbmHqAbsJ8mEasud4cFRGbwom1qpy2G8VDpx\\\\\\\/Kf3rtp08KlN2\\\\\\\/CgjZua5UDpHuUP9ElQE\\\\\\\/63f+JZcqc3ciKMdW9MPJ7rkipKeV6k9KvAAmqztVXlqFMxTjousBh\\\\\\\/f8pcDfqhIo534lFURVDdkRztJZw1BYtcho0nxlsLFUZ+iHufEMte7NHoyjnvmdszoeSOj\\\\\\\/3hH7PSL30PIttUqIu53\\\\\\\/f\\\\\\\/4MK\\\\\\\/\\\\\\\/dHnssalSH1w\\\\\\\/EBLZsuuwGNpYk7O89rhFs0mC0PIXyH7RURllY4EDOJjZZRmyES6wPXQSdeWawM3EYrnEFFxVQDkKPUIbpjoSEKEU4hec9TBKf1UErP9tbOzSeUCD\\\\\\\/JP66+7oJFCTy6KBy8Kzt\\\\\\\/QbFHsG\\\\\\\/TQrcH3WNkXgv2eckAKu\\\\\\\/tXlWFiTbeY7YwovlFEbkJ+6QE9DhCj0bOorGO9ERkkI\\\\\\\/YohrWOgaWQUFuFmYLsOE7Jku36pjfA3ocnxdBR\\\\\\\/jlL\\\\\\\/7D1GC3aujPf6UMvDc7Xt5ZSKHEdMMYkNvj3MJeA41\\\\\\\/fo7LcIiS5yihnHIvupwLSlQu2QDU4+tkLnS4DXAccXIG3\\\\\\\/9IKK5RuiRXrVIOmIXj6hdyQkujN5x0RO7DfmWZLRnqz\\\\\\\/3x9PzZdfl+fzgP1wOkrKd8EBBxGQiDuTn2O9zHeytliFikiWRjt4Y9e5GgC86A0bGjrAH7sRmfC4YaCcko+COcCXxDrZm60FsdkJUzxH31FXOKb2xfDyuFLpe\\\\\\\/7b3B3FP4F0Yg2l3WUmj1BFQSsC1t1abS38MmTgnJ6YgGEWMmE4BzNWJEpT66YKNftRjqSVrsOag2Do1z6AXph4o7xHxLI2XvJf2raC\\\\\\\/fvf44B9UnTczOYPE5gSkDDwrb1O6sHaMNAd3g9RLMIcKWX2hbpqBE07XEOmFQSC3Owr9J93ra\\\\\\\/GTqDH2KVE6kkoV415Ic6fSOKlSaxpDpqMTtrQmZX4S9pZAx7ObRLDt8LWjbPI4EkCjP7grezHNQKKfH4sbo0IZVRbteNkDQERE6H6Bx7dbE\\\\\\\/ASwdZmEUwvEvQ37\\\\\\\/Xfn4QwWq1msVXCWNZ7LCt2wGoJ58\\\\\\\/lkRzsNDPAd981NOTCYGRAy39EOKUJdgbAznZIZ2w7veHeap+wdykRWWLKkviN1Q4eRGucyPIlNKkGhv3jYKpOxzpyJDzlLcm2E7+hHqRMdughySOMEAwHgIX7LNo4znuNGzVSu1zy8h8mxp+aRYz8PJn6RvFFJ0NULfp+aIkqojGYCSu6bkcpqMTVWqs6DlLL3RgUAJFpSSCdaHmV7rHcVv3zJvIUfVQ06rfDew1P7a+gzEDMD\\\\\\\/MPp3a3Yw7HrmR9nd0MQL9ynvYS+EmXJGqtcOFETbFgmtc2qDNHh8vL1Yd8XgvUoNRm+AxCcINb388SJ3s0ZqcHLu5w0zDihdzxYTEwyk0KG2rrfyyYGkEe2um7jGKTiwJF7rPp9h12SQljewofC6ImGzIdtmuw1e0yYBn66ouaqhltEsJ5myHylm\\\\\\\/D9YlcRC+XP2dsSGvS2mrYbxwP1NDRu5cxJn3EhkXvTjYIaJaMwo7+ErOu\\\\\\\/GNp6re\\\\\\\/ZdZtmlj3XMyKZ7xiOQgngVWvFhJ0EbMRAuM4gaAE2p20CheMeLKDRgWLHk8R8pP4NdLbJA50x9levs\\\\\\\/bYawdytSn9IchCVU94DVGQnh4ChocdmHwI+wBVKd5uwB7ZN3Bx85S7jMmyitaU+a\\\\\\\/mKUj7kkgKQtNWXnqWH0tuYU7x6kkqtcncOs5CxMn0MFu1OrTJ\\\\\\\/pvDxOGNAMyensdm46a6kFpKkfmhwRk7ZvEMqlA7fRC7kqyst77h8KnDjJ9SZ5BInGdwtUi86pPgHu65DbAZi+a3YoqzdFM9qipJZriGUBpyrXXDF4jRFYKrOUZFt7XgfpE7+pdpnRbBepVYpoO3GfsfHrqwgNXyvp3Zp9OSXIilxYjOtqS+xEdl40W\\\\\\\/IK5NCS1cJ4Hfe5eTbocOn3pMJXP3haDH\\\\\\\/Op1ZGudrLU7SBU5ajrZwKpwygtz7hkThhpgTsDOvbzi\\\\\\\/js0vmlcnuJPvfmZiiO6NhwFByuXiGsxchfRuaxgvSKwShIZPUusQ9\\\\\\\/fiuT5LSnp0E+haJIXNIRJbRQrXi4u+51a+TYx7+Xbn+ZZi4fG2+gbKb6rnTpc\\\\\\\/CNFbW9M22BY3uyA\\\\\\\/\\\\\\\/LxCJLex3VBmOyfXmhT4ti+O5g1HloV+VWu++uSgYAuEMNv\\\\\\\/Yj95S0KS6tR5vcvEiacUmwQVTm92a+hGLzrAPYx1p2A05nh7IMRH0pWMwUQz0aeHeJNGygq\\\\\\\/+fAi\\\\\\\/bK1rctsSDrMIj+49E+VDJfUF33HJDEgZ6c55VCDzm3\\\\\\\/bRLH+V+pkneZ2jKJagJ9xfAzmYyI94EB\\\\\\\/xjHIwmeONqzXRjVxleQ5o\\\\\\\/kQlln7llU8McDQ65emF6gG86lwangxpxZLxw0R8OFSEL+vWHbArfK4fPVRBnpJER+IQuL0zrUETqPDOjMCgTUjZgPgR80eDEzkPf7\\\\\\\/2IYr\\\\\\\/o60LRoMW\\\\\\\/Dk8wbbDRY8v6aPU\\\\\\\/QtKgSOkl5L2V1eI1\\\\\\\/xVhSnmO36dFud2YjNp0pMeAP3Lkqq\\\\\\\/0BpULwhkparTmxbnyB1FWW\\\\\\\/+GmMQHJBRK+5\\\\\\\/AP7TFWXXIacAPUw1t5mTcHYnn606xZ4Eq11S8OoVHxaTtYOlQ5lRHkdrHHXr7Odhtufksu+OAiWzM6yUzaXJMp6ssPUFhuMthSYZ0L8MnGr6CoZGeJlnErv9EG72L08fye1PUyuzMM9ghBET5WTfy6cWQS5yBk8tWQyH3VIOeeKklM+KF5VDavPuSa5RymPvC5ACHjLt33zURseMpDi3PFSquuQdF6Wths6t9qpgjZPaEqua\\\\\\\/HiUhWhuCEVDLmiWhbDIwSdZ+NiZhlz+COmWhs5z6tGgnlKzbcddgyybrdt8rI\\\\\\\/crQnm0ZYT+LjqH0H3RhhvhOyzAOB32HwQU57uqqKo31yHF74zSpuaK1sow\\\\\\\/6M+gAA2BO\\\\\\\/fils1twW\\\\\\\/mQbJq0bVIvYgN4R0gaoAJ3+Jl2ggrVjne6ts0\\\\\\\/dIuZMamzDDOZ0Mhf2GzceTPXyt7+BoFJroVZAyE5YWV+UxNyW8FnwxqpcPgbZXgjUtEtb9TnOORSESskKD9wLnb5sUaAbhxSuW33r6W5h4c7mnc4hAKqHBm5L1VnXTXlmgKHrh0tNsgWi9PXLCxiwkKKBRmzuM\\\\\\\/Bvqm6ntmen9mFy\\\\\\\/C2UoN\\\\\\\/zPkSEQy92Kq7v\\\\\\\/F\\\\\\\/KYAOKv1VI3RGJefpnfcAyxMh+x8U9sX\\\\\\\/ngTWSVXjmnvKz7SyuHENcH+CiTIjHqAYb0j+a2jCGptwmCNCrXUeEpuVv1FxOoz\\\\\\\/4QgJidE5JYY0uwppcX\\\\\\\/RtiSkWLXO\\\\\\\/ps5B+7XWAlXTpDHXr3LEbufAWyx7tJLtQ9wYjRJQKhd4F3NLejjUtsh4JpQMUBgFUJ2mOd3Fq4WF+rX0Om3Hmxga0jkuEEwtM7TQ\\\\\\\/BKna5NfIXlkKIwHusedB6Ol1qvlBgsXA5ffaXh0VXPh9iD4bjSvwJeg0xFJGpfHvojvaSvjlQx8whaXGWVsjUW056vE9prk7yK+EmEE+avl1eheDp3BUYaWjX0pwI4GQFw5ALz90lLDP4qzhgcfYq5+m2XiV+Ypsg2FbKsD4\\\\\\\/4lVYEgSdw8FxlQ8YTe8ztiGyNcpO5cXP+x637G7PCI1nzXRowpRK\\\\\\\/OhE6SHkmEsFVJAXDwjuDJsADSMUlYOvtnm3AJclf5glwnQqpcOAdoRsIAky4uGIgdmReLUmfEETAjD3Lznra8R2sufaij27bO2urFz7bIR2HaDBOZFgCyeLHPwr3eOLhlmnRRphjQoIScbeVnOzF+7KrU8uoZK3X6OZ82WGpvBkd+eZBBvc32+sH89c99FENrfgF1wNqu3S\\\\\\\/b+XO4nIElRZKQXDHfJZWai5hgV7GJ+pjf6Qa3p6nh5DKgC0R7UpZvTVj0fz70Kg\\\\\\\/lHek9N\\\\\\\/1KyxxWq30MN\\\\\\\/5oFfCcoJZhWjyaNrZDMewoW4tttk5Hcv6x6UHArpogJ6xY4G6zU2cueghP7OYGiluaGYFuDYg1p02kZYAfG4epjNGIsaILC9L+WrMwIoV3mk4uEoI09rwvnNIK4HYTxBth9EOrrDlRSUCXjk7fIBoB0cwHnpBSvdyS+JjQIGvV++zhXV+51uqYYpooGaODNZfzDkuGMggFewfScMFGNbXoT4Np+76ixyJK\\\\\\\/SbwUC9staMFr+DtOpstQ5G5V85UbNuBXKRo9dC+EKEIfqiy\\\\\\\/5AhZFkms5ZYYVmD1QBWzybqEeC0ohZp4V61gS7rCnABeyM\\\\\\\/GdOYdCAUBXhh8T\\\\\\\/IlEiVO9Jg1vlb1C63fOIxjUWlajx5ub+Rf1GEtLliGv5qpRzLSJ1f25WRd9XvUcV036ngD4pX1Vm24NW8YOd+xYhwp+bfdQCqx54iAsVCEpk48j7w7NRaPNxPf4Ra7ClJ0dbZaoNeRSv\\\\\\\/jTtnfmtnkIHqcXzF6YSGM8ju5OhL250K3w14DsJCDgR1y3KmunWIof4Tcjq8\\\\\\\/Jesmi99VTs\\\\\\\/aBRoPIX65WpMlmTFvR90SVZKfo6ekf1UICY16mRX6K+50Yj5\\\\\\\/wSgi5\\\\\\\/Kr+KMtFQquPfTA\\\\\\\/KKY+IAGimsbOimU8nPvpJFb+h0bB9ztj+GbjJ1XPr2w97hKxh7t96kqX59Bm62s2cDoEZo8\\\\\\\/59XdSolgDkLuVbzhTjxy2zBx2bCSXgKE3R8djCe5rgs0VUUDT\\\\\\\/7I\\\\\\\/eETWOhKpUvvgp1MJwfbf63aC4LDHufnrAcarQ0Bj9BCVcY25avLPVYmJohT\\\\\\\/z3aml1X63P7tg3vNYdBz33aMX0zu6BR5W1kLaZzXjh4AwOEtuZ5jFyTD7SUJuVaqSAPkSNWAlpEBItTl8PusC7SFaVTe5n7kOUA\\\\\\\/HYjlwtSE\\\\\\\/MUyX29\\\\\\\/hVNB2Cmi7kTUcvcg19AkP1RY73PcRbaNgydnbmZ5Je+REo+HACEEdKaXfrs7mL07oUwG1BY8XZ0SlAy8gmauhs0XpktimVe01y9Q4xq\\\\\\\/LT80efS\\\\\\\/2MgdSZ1LSsKvOgK31WCveYAfdoAPMQGCTKOI+dDV2h3sivG2LSSKqAu3MjuaAvs8FnncwM63VOirbo2hy6++9\\\\\\\/8eV\\\\\\\/vm+GpMb12aYEhQGbwPPxbnMbImVEpwKTLNQQIb4sXT3kau+dn9NezzEko8KC1aqpafp90UB\\\\\\\/YuUa+rSm9CpF0YdGV8t8DTzqq5dPdj1BMXfQGR\\\\\\\/7d4pvdIUjR352o0XWjeowtXcLYSkLFpj7qV1j6ameUFKJLQqrwq5eFcLObCCjDxA9uGepsmwVmWY1bgzXyUF8p17XRXeTH8J5Peb9titCqhwbWdnPcMoPtCpyW2qNl\\\\\\\/4Mem4vBA7tz\\\\\\\/tWqXPOGiW6R+wGDuqGVLal4kfWCOS+0RJ71Xon3Ym2xOckJnYmo2Y10ukQrFPrAhjDKcvtf4kw0S\\\\\\\/cy1FI5ETZlldfQ2Ry\\\\\\\/OiotqHfQ\\\\\\\/CpwdPf\\\\\\\/yRqxYVwMHhfprK0gpK1E2v7p9aOaZzeAv7xK8trL45VY2Fhq+vTCv44dK1MKm3zLLeM0SFefHaKAl2xqhFX2wzTZQ6FAwBRq7IR3PMGMg30I15STTd5a9F7qdqw632JPF0F3GR+o0431pq9O3YS+JIINZ7SYcoZnIdJbmtPliasmefc45RXevsy3x9Q6CtkHaP2ZcvoHMGJIO6aoutO8ugFm1vtpDymsfXY1qsiZSWaa1IjKBfxgAeULzs1OXzaWzCT57jgqtzXvIKGrGuN7L97CZHZ\\\\\\\/u4ApDMeUMzA4e4v5Gqjec7eK8IwbsE5QBZaYzJf0D65XMg1uKk+fDi9cAYMhfhm8Sv7Wg4E3zKxSHrCvs1yLgRlJ5pRid1sFq4RjArm3sUjYVKkqFDRnV+e914e68aTnYBAo0Jor246Y2Xs+RcYoPK4y3sPicqCCKDTzI119JS7ssRl7tWIb5oK4+Q40Ho0p2B7Tgu\\\\\\\/qaHrKH271Re1dwSfMgjZrEaDkRmTulF8K\\\\\\\/9uRfQZAQPYzokuhBPnFT6IwUHihJaABgoeB0K61ZgpqbgjurL72bWhcGne6bU7KH4DF5vFeD\\\\\\\/g47IJoB51NALiIVZEVvEyRySJGPXB1mcizzXp\\\\\\\/qnWLZSNCUEP5cX7xOymQLlX9i55Urhwwnc\\\\\\\/2WEP4Q0o72pEKqFwXEgZQpckhOTL0VOMl\\\\\\\/Dqv5mYN+k4YdsjYNPQzuH7lupDyGduqO75edcYkMZGOazuTI1tYvBi4jb1hbEqlAM5Wb\\\\\\\/IAnXm5F8PglKCGNuHxyv\\\\\\\/SmCxDbr0tRnAyMZAfE8tJvCn3523GRsYyQC5vokYBeVyPOsC0xKe8On9+uPHVOWKnOcpRHnoFYCYF\\\\\\\/XUKNrn4vQrNjgReagw4\\\\\\\/vx1MMHGH04FteHdfA2rRteC7ryve\\\\\\\/Ncw\\\\\\\/CK066ur2W8AseS7Ao1KD838DVLSey24tNurV4CgBLjOS0eJoMKMwgXzDX29xJeuuDkzM0Y3j2XJcTnpE\\\\\\\/SusCnGETT7W7GtonUNXpsk+iDY5Loz7xe5326a3dUfZ3IGUMASSLC77zDI2g9MXYWUY3ZXVddLPJ6f7IP9WFylI3WmWK9zaSnbvww5TC6yGsTKlNE\\\\\\\/QhQQDONd0nXtAzkm\\\\\\\/IdXlnHWkcyHmtrUWMW3nF3HNdggV79pHssK+lsNmOv+HHngQyWmiHzulM1mT7QQiKLRVPJcyvi8J15FrZTHIE1qSMo48kh2tNKkWQ4w6cRiWJF3M1luE7BCBsGit1iVYa97x6dvsJbROkdftpzgaUwJ53UfbS26yjIZg22teG7+1wUHzb2QrgV3BcEUWXeV7qOZppTxWrIHAQVsVLVyFZGOOEOgYDiUHRpEH9uT5ic4XBlF3n2y9ubfSfsoLSX7wB+eag33kgKoaRzyK7gn6I1+Yl3igdhKADIeLjdzIaOeD\\\\\\\/ofagWWdzcnKIySPhbN5Pf2ARZx8R6qOx8xlVfwKbmXBGaCZ4oC9acg8GfV4zuFs4QGxit0G7D8693tYtgw1I8QHAtih586Lz44PAID8jj3diEVSA0UtTzJaFKEDErU859GYfGvhYFfxJjAP7fNkYnhFKpxEHcAkOILc6KmJiCeLsQdraVMAcpD8EnQhEmXsLMGUKJqI7MguenbbHw8J\\\\\\\/CNNTNp296C8vmrJotOLoWhqnKjqvqqxzgn\\\\\\\/AeN91\\\\\\\/kcXeQIoIM6kcl8mMWx4\\\\\\\/jE+I+uodl37JH\\\\\\\/TtdXZJ1e6AcCCNgCN4XIaOjFiTfZ\\\\\\\/XtZ9r7OxlmUDPy4Gry0ej0cp6U5tzDyReawqcJsPSRUYRsdQz8GNc4QEIk4hjspsP5+MKc\\\\\\\/Dba6MDLdPJdbDQ5hXtDult+G0FO46bHm1BhKXPYt\\\\\\\/XtWv5VApuDVmn8R3DaGgSSqkjy8Kevqf99REKH6kgVObmMOJhTHGnKJlaf6h2mIao8Y9PFv3W8CddKVeZqL\\\\\\\/09tTasHwhZtoVi+6Be5Eq9RGs+rXcjNkmnk+LQaBhcO2CKm4FcMCSSmuuAfU+iGFUrm3Llmwhh0hyzFwMkjOUruafXehsFx7aMJRUbrGyo2KEVWtidgbzfSj\\\\\\\/E\\\\\\\/2JU5efzcOos5mwEQD3rSsPQYHl5YlDWZifQgHkEXxLot0ZLREWZyJGIjL\\\\\\\/lODaFjCG2zTlu\\\\\\\/tVz9bMzuzQwJjUehoK9OX5B\\\\\\\/tzLj4FDbxDyjIcf\\\\\\\/nxfDlFTtVytkUELE7R0rDnXopIs32iayybCe+GWu8T9rx77eYjLNzO84ua9HPtUfo3vh4PReC2nu95CsRWxaJmrMJHntC+YW1Jk+ufwCLLQjLGVOGvLQ8lMrXf2M+8UB9D\\\\\\\/M1s9ssmE1DlblmzyT\\\\\\\/EjdOLqA1PEGf3UqBqSB8UYYdzhPOIIXVUNaAIgYrHr3FpFxraT2okkQAUqH5i+i7atocThq6bk3nL7t91tsrVNS5f7aznMRz8HJ1qZOSjFr+7kV15yzhihAnv9OHhRKoPL1df\\\\\\\/46zHk1mUg0KbItNPPfpakfuUO0mln1+r\\\\\\\/cRZoTEtvxc4MmQCEE7M5sI2YLBC6eTtnH77CAq5IUxTx2ZVHG8kVzx+\\\\\\\/ePoiE8e5J\\\\\\\/MYHWiiGW69\\\\\\\/MrinBHDK\\\\\\\/RHfXGc1X\\\\\\\/0sXRs7g\\\\\\\/tOcnHllAtvK4nuk01dkPSbMEz561754urFulDKCdtsMKZijpzD+BoIg2nVkdJQ1oE0UxDPfGwh9DBui+rQsySKmMdL1u4kOWJouMPqCYaFKisUM+3gziKMSWN1d60rBOFysPS2bsMQoy6dbrairColil7wwJPzOgwZdhXesjjDSq8N3ShiT29nBzoK8SHTzOM54cbvl\\\\\\\/XIgKROs+CSmgFNwgEBdRaSC39oO\\\\\\\/mMWzkX45OfPcnj5nMKHGRUlMHChgiyKcDbrEsO7tAGE79E8fU9NIhrQ0zEIUE6X2Fq4ewjPfRinY1GzAOQC424z7\\\\\\\/Q\\\\\\\/dQjfEcVERdjuSONFxpVIhe\\\\\\\/WvV+rKrJPqiIVpmDJtJEPaWEs+I9uYY3irzO6QQg2cvWrY\\\\\\\/sBi4cjDciIsKh4lsGYVoKyiuXdkkVqEpFpJTLKVyJGnjk44DDxAch+tgiwjeNNZUM24fT\\\\\\\/EIjNQ+bif7icHb9NdpKoqhMtjZ5SmC28AwrIg8UhDWmj3+iHuwOjBwWEWVS4RIJF4iafcRhxumwLArUbtehfLjoksPECgcV469i+9qMp684ctMjgkbRSUr2SmUwpK\\\\\\\/2kF3\\\\\\\/FHT62qpiYzgIZ4icFhzMdoTnpaANzaraoRaddeXMlctjRu0Sg0pDPdxETedtQ0UYAIawhhg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+biZXi833nUKwHigb+i7kg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b34a8a9c93f7d477fafa1dedf757ad06b7ec9365598931abc00953ad64e53e6e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hKGsDkediaAdyoIFTiYoyWFEI8VlNqY7wMDOLbvuyX7wguhpvkjtBhnQyHr56LLO9MUJDxA3qAyCrvCjUraSIGv7WppQX20GlZTpyLv1Kdkk7HkYnmEhlDI5FsMzMCvmcklF6N7dNPG2CE9DiQbFktsthYo9jatLkv2k5+ho\\\\\\\/Xb2rrLqWIZ0L9iqhxZ+XanIeYwlaVYjVmp7MuDsRSZ0R7VjWu6Hh7hcRiizhd\\\\\\\/hsiZiLNAComKFZd69qgs6lY+jPxITLB4yRTqC63TugjZMehuJJz1wu+DVYLQ6YjfmRuvNjLIemKyQ4l86VgO9hyP8vkr6BA7iPXNlpdzQwSvU+kridifx\\\\\\\/ISPyaN9Pu8kgXT+P5\\\\\\\/jipsXDKXeEL75Qy8sBVoPoJgtbsGjGqpDkJui2V1XSMOut6NguCSqzVyMlTPIxxU9L+AKq14UKZdiCSWbv5UhFFywIhOyIJx66dFx3P\\\\\\\/mySCUx+CYG78RjoSbz9uK0pU0rMqHzvgziibNn82F\\\\\\\/VQpZ9nBVmCgYT7IBg7oG\\\\\\\/XaocBkoSiTuLOLpLAKaNj0E6cmlQzbofk3NZz7kVY5s83b95Ez2HL9tSNqt6smpsIQWu522y4O+R+r\\\\\\\/NDfxSYOkf2ogObKBLbkZws1ROSzdElOowA66pwchU4aKc0N5jxdrZMHIkqlwrw5xXUuaGXhj68lZiel4kHmCInaFhuqAdtO1cMkPc1agzOczO1nIMtNPFabHp+FUR+SofnWo6agCkgBqz0LcN61bf0uJb8Odw9RyyUHE9Zq\\\\\\\/cTANtd9czgx4cIUb3hzIWeyRiwtZhYa\\\\\\\/ksx+mfjIs21XAUWvbxIJT5tYaY\\\\\\\/b5okBrJ32WsdZVWl2Y\\\\\\\/KKVVjo4D\\\\\\\/5\\\\\\\/VE41ti+6swMWK2x\\\\\\\/wkafmia9D4dpvWYwjPJ7X9HhCqZAw72XIQQQuMMjDJK9XdZq4RkxJl4ZAoS1GDn9u3vAFA+zuvoGARF9kV4opw3wAabURGdgnoj9qmgFJ35dB1jMHQ1qAJ4uNUoPuCqDABU3fB8h2+48jofEnZjh6iQZ74RuHwAYqC4g6h5QlSW34kqc1iWkbt6wNq7G5YN8OxyZ0ggxu0oJKuz6WFT6Dfxe8CkTS3kGOzVuxQ8EuaWKI0LZxFaWMpSP1ju8Bp\\\\\\\/aPjiVCZULSB9Vdwv+7f\\\\\\\/PKzCerG4zzcG1cmDd2ZHhZKNAiq6xmR9LRxYsFr1Yt8DOZNlFv7n\\\\\\\/bem5sbXdCh4\\\\\\\/uhCQ6nKXPxXcD4n9vnlk4J+D\\\\\\\/vnPdE\\\\\\\/So9yxUICqWfMDjnNC9hFglydGCx1fAW3zP1Cs7+Qu+RwgImPt+v3JBUrQ2TTNHLlAWFocnwxAslZ+kmtYztfgeSk24FuSZycKuuFQVtKKaALzT4nAmuB8hJ3lTlQ4EZd488e1+lirWZPyYYoWhNIMjyXVJq5LImNEnNh8Tj6VUFDThB0b7kATPDKhbIzvNNeg\\\\\\\/B+dt+k+UKwCetYuvO01SJAMIc+OVh7\\\\\\\/+V8p9vUSmaspU4EovSRytQd4zowGmXHpkUvDtxaLq3\\\\\\\/NpTY0PwnAvhk9WCcPJeY4X2BGQvMWBTSvQq4blGJH5XrqiN0FNZpK4cKDZpxLm+9fP5ZTwhamV7COEThiZ6g8hpuM4DMPwq\\\\\\\/0cB3MNNiWMEkbMq3fBwrzTiyhr8A5xdKdjjhYBXqcoWzJLL9Dk6NUhVHPpt09ZRpZC7wOwgAmpWNYNp+RVDqHxvsXcF8NozN3e22mPdsuvpCb7LUEgYw\\\\\\\/Brdqbs3PATAtXblk+RTcs42R58XwacerPNTDWelROgbIkLIp+XMzT9lBt2+I3YvjCCQ+8FMmUhRuTLdca9mkNBzVbLVTI+7fXnR2jHLBLgzo8odrho6W4GEBSLDtlELyVMF9hGampXG1tfROqsLI4Md9KF8r2CqzNxelFG0mz+N7O+XQDCK++lO2b+CBVP\\\\\\\/zrRaqZpPFel11SrOy3PgMFl\\\\\\\/SWlKlWceh3fsLm5+TlgvFvzo5I9dcsfGojPUzHybJ\\\\\\\/M9k76PWkJSg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zevrasuf5yVObUkMA5HtCw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002211da09d4db482b746989dfd214747a5194c3a4c943855c698ecf03615f26a3ec\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489451 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1721" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y6h5wP4MyfeyVDvtcJq2YSdT3w6jjLdxIekMeMWzA4g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1721" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H5ibQaCV3k9f94iEcjFuaXbMa38TefBOprJiHpuB2SM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1711" title="Inter-replicon Gene Flow Contributes to Transcriptional Integration in the Sinorhizobium meliloti Multipartite Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1733" title="Comparison of the Relative Potential for Epigenetic and Genetic Variation To Contribute to Trait Stability" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489451" id="node489451" data-pisa="ggg;8/5/1721" data-pisa-master="ggg;g3.118.200140" data-apath="/ggg/8/5/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9787-8543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Washington Gapare</span>, <span class="highwire-citation-author" data-delta="1">Shiming Liu</span>, <span class="highwire-citation-author" data-delta="2">Warren Conaty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qian-Hao Zhu</span>, <span class="highwire-citation-author" data-delta="4">Vanessa Gillespie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9535-8707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny Llewellyn</span>, <span class="highwire-citation-author" data-delta="6">Warwick Stiller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-4626-8059" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Iain Wilson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1732;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200140 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489451" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Washington Gapare </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Washington%2BGapare%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gapare%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWashington%2BGapare%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9787-8543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Washington Gapare</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Washington.Gapare@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Washington.Gapare@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shiming Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shiming%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShiming%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Warren Conaty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Warren%2BConaty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conaty%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWarren%2BConaty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qian-Hao Zhu</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vanessa Gillespie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BGillespie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gillespie%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BGillespie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Danny Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9535-8707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny Llewellyn</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Warwick Stiller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Warwick%2BStiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stiller%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWarwick%2BStiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Iain Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIain%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4626-8059" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Iain Wilson</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/5/1721" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489451" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1721.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489451" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1721.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489451" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1721.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489451" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/5/1721.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489451" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489451/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection (GS) has successfully been used in plant breeding to improve selection efficiency and reduce breeding time and cost. However, there has not been a study to evaluate GS prediction models that may be used for predicting cotton breeding lines across multiple environments. In this study, we evaluated the performance of Bayes Ridge Regression, BayesA, BayesB, BayesC and Reproducing Kernel Hilbert Spaces regression models. We then extended the single-site GS model to accommodate genotype × environment interaction (G×E) in order to assess the merits of multi- over single-environment models in a practical breeding and selection context in cotton, a crop for which this has not previously been evaluated. Our study was based on a population of 215 upland cotton (<em>Gossypium hirsutum</em>) breeding lines which were evaluated for fiber length and strength at multiple locations in Australia and genotyped with 13,330 single nucleotide polymorphic (SNP) markers. BayesB, which assumes unique variance for each marker and a proportion of markers to have large effects, while most other markers have zero effect, was the preferred model. GS accuracy for fiber length based on a single-site model varied across sites, ranging from 0.27 to 0.77 (mean = 0.38), while that of fiber strength ranged from 0.19 to 0.58 (mean = 0.35) using randomly selected sub-populations as the training population. Prediction accuracies from the M×E model were higher than those for single-site and across-site models, with an average accuracy of 0.71 and 0.59 for fiber length and strength, respectively. The use of the M×E model could therefore identify which breeding lines have effects that are stable across environments and which ones are responsible for G×E and so reduce the amount of phenotypic screening required in cotton breeding programs to identify adaptable genotypes.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic prediction</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li><li class="kwd"><a href="/keyword/bayesian-models" class="hw-term hw-article-keyword hw-article-keyword-bayesian-models" rel="nofollow">Bayesian models</a></li><li class="kwd"><a href="/keyword/marker-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-marker-&#xD7;-environment-interaction" rel="nofollow">marker × environment interaction</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/gossypium-hirsutum" class="hw-term hw-article-keyword hw-article-keyword-gossypium-hirsutum" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Gossypium hirsutum</em></a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">The main goal of commercial crop breeding programs is to identify the best performing individuals (or genotypes) as potential cultivars, with the secondary goal to identify genotypes that can be used as parents in future crosses to advance specific breeding objectives. However, the principal limitation in most crop breeding programs is the long timeframes required for the completion of one cycle of breeding, testing and selection, often in the order of 10 years. Genomic selection (GS) emerged from the need to improve prediction of complex traits and to reduce selection cycles and phenotyping (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Meuwissen <em>et al.</em> 2001</a>). In GS, a training population is phenotyped and genotyped to train a model, which can be used to predict genomic estimated breeding values (GEBV) of another set of individuals that has only been genotyped but not phenotyped (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Meuwissen <em>et al.</em> 2001</a>). Under a GS model, precision-phenotyping is most important when evaluating a training population because that dataset provides the basis for developing the statistical model that is then used to predict phenotypic performance in related members of a breeding population (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cobb <em>et al.</em> 2013</a>). Likewise, a proper phenotypic analysis and appropriate modeling are crucial prerequisites for accurate calibration of genomic prediction procedures (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernal-Vasquez <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">2017</a>).</p><p id="p-4">The potential utility of GS in crop improvement is currently being intensively studied in different types of plant populations and traits (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Heffner <em>et al.</em> 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Heslot <em>et al.</em> 2015</a>), but has been most successfully used in dairy cattle breeding (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Goddard and Hayes 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hayes <em>et al.</em> 2009</a>). Initial studies in plants have included data on several important traits for major crops such as grain amylase activity in barley (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Lorenzana and Bernardo 2009</a>), grain moisture and grain yield in maize (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zhao <em>et al.</em> 2012</a>), grain yield and plant height in rice (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Spindel <em>et al.</em> 2015</a>), grain yield, resistance to <em>Fusarium</em> head blight and stem rust resistance in wheat (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2014</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rutkoski <em>et al.</em> 2014</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Huang <em>et al.</em> 2016</a>) and stem diameter, pulp yield, and rust resistance in eucalypts (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Grattapaglia and Resende 2011</a>). The body of plant GS research has grown substantially since the first early descriptions of using genomic selection to predict unmeasured phenotypes. However, we still lack sufficient understanding of the potential value of GS in an operational breeding program, particularly for polygenic traits of agronomic significance, and the factors that might determine its success in specific crop species.</p><p id="p-5">The accuracy of GS is crucial for its successful application in a practical breeding scheme. An increasing number of studies have provided an empirical estimation of genomic prediction accuracies for different traits and crops such as, barley, maize, and wheat and trees like eucalypts (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lin <em>et al.</em> 2014</a>), and shown that they can vary greatly depending on whether the crops is inbred or outbred. In fact, determining in advance the prediction accuracy for a specific population and a specific trait is difficult, because accuracy is influenced by many factors (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Desta and Ortiz 2014</a>) such as the marker density relative to the effective population size, the linkage disequilibrium between markers and quantitative trait loci (QTL), the suitability of the prediction model with regards to the population genetics (relationship between individuals in the population, genetic structure), and the architecture of the trait of interest (<em>e.g.</em>, heritability, underlying gene actions) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Goddard 2009</a>; Heslot <em>et al.</em> 2013b; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Isidro <em>et al.</em> 2015</a>).</p><p id="p-6">Since <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Meuwissen <em>et al.</em> (2001)</a> first proposed this concept of GS along with several models, numerous statistical methods, including parametric and nonparametric methods, have been used to predict quantitative traits. Parametric methods include best linear unbiased prediction (BLUP; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Henderson 1975</a>), least absolute shrinkage and selection operator (LASSO; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tibshirani 1996</a>), partial least squares (PLS; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Geladi and Kowalski 1986</a>) and Bayesian based methods such as Bayes Ridge Regression (BRR), BayesA, BayesB and Bayesian LASSO (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Yi and Xu 2008</a>); nonparametric methods include random forests (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Svetnik <em>et al.</em> 2003</a>) and Reproducing Kernel Hilbert Spaces regression (RKHS) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">de los Campos <em>et al.</em> 2010</a>). Recently, many investigators have evaluated the performance of various statistical methods used in GS. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de los Campos <em>et al.</em> (2013)</a> gave an overview of the parametric methods and concluded that BLUP performs well for most traits and BayesB yields slightly higher predictive accuracy for traits with large-effect quantitative trait loci (QTL). <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Riedelsheimer <em>et al.</em> (2012a</a>,b) compared the predictive performance of five different GS methods for traits measured in maize inbred lines, and found that these methods differ slightly in their predictive abilities. <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Howard <em>et al.</em> (2014)</a> compared the predictive abilities of parametric methods with nonparametric models using simulation data, and observed that parametric methods performed slightly better than nonparametric methods for predicting traits with more additive genetic components in their genetic architectures. However, nonparametric methods perform better than parametric methods when epistatic effects exists in a population (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Howard <em>et al.</em> 2014</a>).</p><p id="p-7">Recently, several GS studies have paid closer attention to the fact that crop breeding lines are often assessed in multi environment trials (METs), <em>i.e.</em>, in different geographic locations, seasons, or years, in order to determine performance stability across environments (<em>i.e.</em>, G×E effects) (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pérez and de los Campos 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2016</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Velu <em>et al.</em> 2016</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sukumaran <em>et al.</em> 2017</a>). METs in a GS context are therefore an important extension as they allow the examination of marker by environment (M×E) interactions, and, in particular, the identification of markers whose effects are stable across environments, as well as those that are environment-specific (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2016</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Oakey <em>et al.</em> 2016</a>). For example, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> (2015)</a> extended the single-trait, single-environment genomic estimated best linear unbiased prediction (GBLUP) model to a multi-environment context, and reported important gains in prediction accuracy with the multi-environment model relative to single-environment analysis in wheat. <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cuevas <em>et al.</em> (2017)</a> considered modeling G×E using both genetic markers and environmental covariates. These studies showed that modeling M×E interactions can give substantial gains in the prediction accuracy of GS.</p><p id="p-8">The development of a new training population for GS requires considerable monetary and time inputs. Therefore, within a breeding program, well phenotyped historical populations, such as advanced breeding lines tested under numerous METs, may be suitable for implementation of GS in a commercial breeding setting. For example, <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Rutkoski <em>et al.</em> (2015)</a> empirically demonstrated the utility of using historical wheat lines from several MET as a training population, although a training population of closer relatives increased predictive ability. <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Edriss <em>et al.</em> (2017)</a> investigated GS accuracy using historical maize yield data from 2022 breeding lines tested across 156 trials in 18 different locations in Kenya, Tanzania, Ethiopia and Uganda.</p><p id="p-9">The Commonwealth Scientific and Industrial Research Organization (CSIRO) cotton (<em>Gossypium hirsutum</em>) breeding program in Australia has been developing accessions targeting lint yield and fiber quality traits for the past 50 years. Most of these traits are complex and behave as quantitatively inherited traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Campbell and Myers 2015</a>). The polygenic architecture of the majority of these economically important traits make them extremely difficult to manipulate and improve. GS Bayesian models have proven to be advantageous for complex traits in other crops, such as grain yield where many loci of small effects control the trait (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Crossa <em>et al.</em> 2017</a>). These traits are also expensive to measure and require relatively large amounts of seeds. For example, to obtain an accurate measurement of the most important trait, yield, requires significant replication both within a field, across geographical regions and over multiple seasons (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stiller and Wilson 2014</a>). As a result, a complete breeding cycle, from initial crossing to commercial release, takes a minimum of eight to ten years.</p><p id="p-10">In this context, we have compared the accuracies of different GS prediction models on two important end-use quality traits (fiber length and strength) that are regularly assessed by CSIRO’s cotton breeding program. The specific objectives of this study were (1) to evaluate the accuracy of GS for fiber length and strength, and (2) assess the merits of multi- over single-environment GS models in a practical breeding and selection context in cotton, a crop for which this has not previously been evaluated. GS models for cotton lint yield and lint percent, both much more genetically complex traits than fiber quality traits, are the subject of a future study. The long-term goal of this research is to optimize GS approaches and use it in cotton breeding to fast track the release of improved cotton varieties.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and methods</h2><div id="sec-2" class="subsection"><h3>Traits measured and structure of phenotypic data</h3><p id="p-11">In order to assemble a training population, we mined historical breeding lines that had both phenotype data (trait means) and remnant seeds for DNA extraction. These lines represented a mix of released cultivars, breeding lines and other external breeding lines developed and collected in an Australian cotton germplasm repository for the CSIRO cotton breeding program. The phenotype data used in this study were from breeding trials conducted from 1993 to 2010 at 7 sites (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and were not completely orthogonal (see Supplemental material <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200140/-/DC1/FileS2.xlsx">File S2</a>). The number of breeding lines that had seed available per site ranged from 80 to 215 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Of the 7 sites, only four had an average of 116 breeding lines overlapping between sites and were used to test for marker stability across sites. The traits analyzed for the study were fiber length and strength. These traits were phenotyped using the Uster High Volume Instrument (HVI 900) as described in <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Liu <em>et al.</em> (2011)</a> and <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Clement <em>et al.</em> (2012)</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489574/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489574/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Details of test sites for fiber quality traits over several years and estimates of genomic heritability (h<sup>2</sup><sub>g</sub>) ± SEs</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">The trials were established in the Australian cotton belt which follows inland rivers of New South Wales and Queensland (<a href="http://cottonaustralia.com.au/uploads/publications/POCKET_GUIDE_MAP.pdf">http://cottonaustralia.com.au/uploads/publications/POCKET_GUIDE_MAP.pdf</a>). The cotton belt is divided into three regions, <em>i.e.</em>, hot, central and cool based on day-degrees as described by <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">McMahon and Low (1972)</a>. Day-degrees decline from North to South in the cotton cropping season (September to March) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Constable <em>et al.</em> 2001</a>). Rainfall tends to be summer-dominant in the North and Central but winter-dominant in the South. On average, there are 248, 206 and 186 mm of in-crop rainfall for the central, hot and cool region, respectively (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Liu <em>et al.</em> 2013</a>) and more than 4700 MJ/m<sup>2</sup> in radiation during the crop season (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rosenthal and Gerik 1991</a>).</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-15">Phenotypic data were analyzed in a two-stage model. First, a within trial single site analysis was carried out using ASReml (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gilmour <em>et al.</em> 2009</a>). The final adjusted means over site, trials, years and row-column designs were estimated with a mixed model applied to the alpha-lattice design of each trial using ASReml R (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Butler <em>et al.</em> 2009</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gilmour <em>et al.</em> 2009</a>). In order to minimize information loss when adjusted means are passed on from the first to the second stage analysis, the adjusted means were weighted by the inverse of their squared standard errors (<em>e.g.</em>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cullis <em>et al.</em> 1996</a>). The adjusted means for the breeding lines were then used in the second stage to predict GEBVs based on markers and corresponding marker effects.</p></div><div id="sec-4" class="subsection"><h3>DNA extraction, SNP genotyping and calling</h3><p id="p-16">SNP genotyping was done using DNA isolated from cotyledons of 10-12 young seedlings with two true leaves of each accession. DNA extraction was performed using the DNeasy Plant Mini Kit (Qiagen) according to the manufacturer’s instructions. All DNA samples were quantified using a NanoDrop 1000 (Thermo Scientific) and normalized to the same concentration (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2016</a>).</p><p id="p-17">DNA at 50 ng/µL for each of the breeding lines was processed according to Illumina protocols and hybridized to the CottonSNP63K array at CSIRO Agriculture and Food (Brisbane, Australia) according to the manufacturer’s instructions. Single-base extension was performed and the chips were scanned using the Illumina iScan. Image files were saved and analyzed using the GenomeStudio Genotyping Module (v 1.9.4, Illumina). Genotype calls for each SNP were performed based on the cluster file generated specifically for the CottonSNP63K array (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hulse-Kemp <em>et al.</em> 2015</a>). The SNP calling module was developed for diploids, so for each locus there are three possible genotypes - <em>AA</em>, <em>AB</em>, and <em>BB</em>. Filtering was performed to return polymorphic SNPs with call rate above 80% and minor allele frequency &gt; 5%. Missing data (4.8% of data points) were replaced by the mean value of the non-missing data for the loci, using the “mean” option implemented in the ridge regression best linear unbiased predictions (rrBLUP) package in R (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Endelman 2011</a>). A set of 13,330 polymorphic SNPs were used for model training and genomic predictions (see Supplemental material <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200140/-/DC1/FileS1.xlsx;">File S1</a>). These SNPs were distributed across all the 26 chromosomes of cotton with a density of ∼5.3 SNPs/Mbp.</p></div><div id="sec-5" class="subsection"><h3>Genomic relationship and population structure assessment</h3><p id="p-18">A random set of 5,000 SNPs were used to estimate the genomic relationship matrix (<strong>G</strong>) of the breeding lines, following <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">VanRaden (2008)</a>. To explore genetic population structure in the breeding lines, principal component analysis (PCA) was performed on <strong>G</strong> using the R function ‘prcomp’ (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">R Development Core Team 2014</a>) with 5000 SNPs chosen at random. This resulted in a matrix of eigenvectors (with dimensions equal to the number of breeding lines) ordered by descending eigenvalues with the largest eigenvalue for PC1. We refer to these eigenvectors of <strong>G</strong> as principal component (PC), where the PC1 had the largest eigenvalue. The PCs were plotted and annotated with population composition of the breeding lines to investigate where PCA was able to cluster the various accession composition together. We tested the significance of population structure following <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Patterson <em>et al.</em> (2006)</a>.</p></div><div id="sec-6" class="subsection"><h3>Genomic prediction models at single-site level</h3><p id="p-19">We compared five Bayesian regression models (BRR, BayesA, BayesB, BayesC and RKHS) in order to identify the best model for genomic selection for implementation in cotton breeding. All prediction models were tested using the R (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">R Development Core Team 2014</a>) “BGLR” package (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Pérez-Rodríguez and de los Campos 2014</a>).</p><p id="p-20">All Bayesian models used in this study can be written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">[1]</span></span>where <em>γ</em> is the trait value, <em>µ</em> is the population mean, <strong><em>X</em></strong> matrix of marker-centered and standardized genotypes (<em>i.e.</em>, each marker was centered by subtracting the mean and standardized by dividing by the sample standard deviation), <strong><em>b</em></strong> is the vector of marker effects, and <em>ε</em> is the vector of model residuals. Marker effects and model residuals were assumed to be independent of each other and both normally distributed. The sum of all allele effects is the GEBV of the breeding line. Centering implies that variances and covariances between genetic values are measured as deviations with respect to a center defined by the average genotype.</p></div><div id="sec-7" class="subsection"><h3>Estimating genomic heritability of the traits</h3><p id="p-21">The genomic heritability (h<sup>2</sup><sub>g</sub>) (the proportion of variance of a trait that can be explained (in the population) by a linear regression on a set of markers (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">de los Campos <em>et al.</em> 2015b</a>)) of each trait at each site was estimated from the genomic data. The genomic relationship matrix showed that our population is highly related and presumably comes from a very small effective population size as is typical of most cotton breeding lines (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hinze <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gapare <em>et al.</em> 2017</a>). Under these conditions a likelihood, constructed based on proportions of allele sharing at markers, is unlikely to be misspecified and consistency may hold, leading to very small bias in genomic heritability estimate. h<sup>2</sup><sub>g</sub> was estimated as the ratio of the genomic over the phenotypic variance, where the genomic variance is obtained with a REML analysis using the genomic relationship matrix (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">de los Campos <em>et al.</em> 2015b</a>).</p></div><div id="sec-8" class="subsection"><h3>Statistical models in genomic prediction at multi-site level</h3><p id="p-22">To assess the merits of multi- over single-environment GS models in cotton, we used phenotype data (adjusted means) from four sites that had 116 breeding lines in common (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) and 13,316 SNPs. For comparison purposes, we performed a combined analysis based on an M×E model from which within-site analysis (single-site model) and across-site analysis can be derived and computed. All models were fitted using BGLR software – release 1.0.4; (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">de los Campos and Pérez-Rodríguez, 2015a</a>) which support heterogeneous variances in the model residuals. We also applied BayesB that allows for incorporating priors that can induce variable selection and shrinkage simultaneously (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Meuwissen <em>et al.</em> 2001</a>) and also allows for environment-specific error variances (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2016</a>). The model is an extension of the <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> (2015)</a> model that assumes homogeneity of within-site error variance.</p><p id="p-23">A single-site regression model in matrix notation (Equation [2]) is similar to Equation [1] but can be obtained by removing the main effects of the markers, <strong><em>b</em></strong><sub>0</sub> = 0, such that<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">[2]</span></span>All assumptions as specified in Equation [1].</p><p id="p-24">The across-site model (Equation [3]) based on a GS model where marker effects are assumed to be constant across sites (<em>i.e.</em>, ignoring M×E). The model assumes heterogeneous error variances but assumes that marker effects are the same across sites such that <strong><em>b</em></strong><sub>1</sub> <em>=</em> <strong><em>b</em></strong><sub>2 =</sub> <strong><em>b</em></strong><sub>3 =</sub> <strong><em>b</em></strong><sub>4;</sub> and because all 116 breeding lines are tested at each site, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-3.gif"/></noscript></span></span> = <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-4.gif"/></noscript></span></span> = <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-5.gif"/></noscript></span></span> = <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-6.gif"/></noscript></span></span> The regression equation becomes a multi-variate model with the following notation:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">[3]</span></span>A marker×environment model (M×E) (Equation [4]) allows analyzing data from all sites jointly and accounts for G×E. In M×E model, the effect of the <em>k</em>th marker in the <em>j</em>th environment is modeled as the sum of a main effect (<strong><em>b</em></strong><em><sub>0k</sub></em>) plus an interaction term <strong><em>b</em></strong><em><sub>jk</sub></em> representing deviations from the main effect resulting from M×E. The marker effect model for the <em>k</em>th marker in the <em>j</em>th environment is <strong><em>b</em></strong><em><sub>jk</sub></em> = <strong><em>b</em></strong><em><sub>0k</sub></em> + <strong><em>b</em></strong><em><sub>jk</sub></em>. The model can be expressed in matrix notation and for four environments as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">[4]</span></span>where the vectors of the main and interaction effects and model residuals were all assumed to be normally distributed, specifically <strong><em>b</em></strong><em><sub>0</sub></em> ∼ <em>N</em>(0,<em>Iσ<sup>2</sup><sub>b0</sub></em>), <strong><em>b</em></strong><em><sub>j</sub></em> ∼ <em>N</em>(0,<em>Iσ<sup>2</sup><sub>bj</sub></em>) and <strong><em>ε</em></strong> <em>∼N</em>(0,<strong>D</strong> ⊗ <strong>I</strong><sub>n</sub>), where <strong>D</strong> = diag (<em>σ<sup>2</sup><sub>ε1</sub></em><sub>,</sub> <em>σ<sup>2</sup><sub>ε2</sub></em>, <em>σ<sup>2</sup><sub>ε3</sub></em><sub>,</sub> <em>σ<sup>2</sup><sub>ε4</sub></em>.) denoting the residual variance for each site, and <strong>I</strong><sub>n</sub> is an <em>n</em>-dimensional identity matrix.</p><p id="p-25">All models were adjusted via Bayesian approach, using Gibbs Sampler MCMC (Markov chain Monte Carlo) method. Thus, we obtained marginal posterior distributions equal to the restricted maximum likelihood (REML) procedure, to obtain variance and covariance components, and to compute the GBLUP values. In the implementation of the Gibbs Sampler, we considered 210,000 samples collected from the posterior distribution after discarding 10,000 for burn-in, and considering 20 as thinning.</p></div><div id="sec-9" class="subsection"><h3>Estimating accuracy of genomic predictions</h3><p id="p-26">To assess GS prediction accuracy at each site, we generated multiple TRN–TST partitions with random assignment of breeding lines to either TRN (training) or TST (testing) subsets of the whole dataset. This approach was favored because with a replicated TRN-TST design, one can obtain as many partitions as one needs and this allows estimating standard errors of prediction accuracy more precisely than with a cross-validation approach (<em>e.g.</em>, <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> 2015</a>). For assessment of prediction accuracy across-sites, we adapted cross-validation (CV) schemes of <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> (2012)</a>: first CV1 that mimics the prediction problem faced by breeders when breeding lines have not been evaluated in any field trials (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> 2015</a>); second CV2 that evaluates the prediction ability of models when some breeding lines have been evaluated in some sites, but not in others (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> 2015</a>). We used an R code provided in <a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> (2015)</a> supporting information to generate TRN-TST partitions in CV2. This validation mimics the prediction challenge faced by breeders where breeding lines are evaluated in some, but not all target environments across an industry (<a id="xref-ref-52-7" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> 2015</a>). In such an evaluation, information from related breeding lines and the correlated sites is used, and prediction assessment benefits from borrowing information between breeding lines within a site, between breeding lines across sites, and among correlated sites (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>).</p><p id="p-27">For both single-site and across-sites predictions, the data were divided randomly into 50 partitions, with 70% of the breeding lines assigned to the TRN set used to derive the models and 30% assigned to the TST set to test those models. The adjusted means of breeding lines at each site were used as observed phenotypic records in the GS model. Each partition yielded a point estimate of prediction accuracy (<em>e.g.</em>, correlation coefficient between predictions and adjusted phenotypes). The variability of the point estimate across partitions (replicates) reflects uncertainty due to sampling of TRN and TST sets, and a precise estimate of prediction accuracy was obtained by averaging the estimates of accuracy obtained in each partition. In selection theory, the accuracy is defined as the correlation between the selection criterion and the true breeding value (TBV). As calculated here, this correlation is reduced by deviations of the phenotype from the TBV. The average correlation for each trait was divided by the square root of the heritability of the respective trait (<em>e.g.</em>, <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Daetwyler <em>et al.</em> 2013</a>). For each model-trait combination, we also evaluated the bias associated with the prediction. The slope coefficient (<strong><em>b</em></strong>) for the linear regression of GEBVs of the validation set on their EBVs was defined as a measurement of the degree of bias of a model. In this case, <strong><em>b</em></strong> ≥ 1 indicates no bias, whereas <strong><em>b</em></strong> &lt; 1 indicates a biased overestimation of GEBVs.</p></div></div><div class="section other1" id="sec-10"><h2 class="">Data Availability</h2><p id="p-28">Genotype data and Phenotype data are included in Supplemental materials <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200140/-/DC1/FileS1.xlsx;">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200140/-/DC1/FileS2.xlsx">File S2</a>, respectively. The authors confirm that all data necessary for confirming the conclusions are presented fully in this article.</p></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Phenotype data</h3><p id="p-29">Genotype by year interaction accounted for only 1.14% and &lt; 0.5% of the total phenotypic variation for fiber length and strength, respectively. Boxplots of standardized cotton fiber length and strength showed that the overall empirical distribution of each trait was approximately normal (data not shown), allowing the use of Gaussian distribution models. Genomic heritability estimates at each site for each traits are presented in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> (see materials and methods section). Individual site estimates were varied for both traits, ranging from 0.21 to 0.62. The estimates across-sites averaged 0.44 and 0.32 for fiber length and strength, respectively.</p></div><div id="sec-13" class="subsection"><h3>Genomic relationship and population structure</h3><p id="p-30"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> depicts the heatmap of the genomic (<strong>G</strong>) matrix for 215 breeding lines. The 215 breeding lines comprise three groups, one small group (top left) with two subgroups unrelated to one either of the two and two other large groups, each with a number of subgroups closely related to each other. Principal component analysis was performed to gain insight into population structure that may affect prediction accuracy (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The first axis (PC1), which explained ∼24.1% of the variability present in the genomic data, showed no obvious clustering but evidence of strong admixture of breeding lines from pre-2000 crosses and post-2000 crosses. There was also evidence of wide dispersion of a limited number of breeding lines especially defined by PC1. We accepted the hypothesis of absence of significant population structure using <em>P</em> = 0.01. Thus, we did not detect distinct genetic groups in this population but instead evidence of shared common alleles as seen by the mixture of colors representing different breeding lines in the biplot.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heat map of the G matrix of 215 cotton historical breeding lines genotyped with 13,330 SNP markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865172322" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heat map of the G matrix of 215 cotton historical breeding lines genotyped with 13,330 SNP markers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1721/F1.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/5/1721/F1.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489682" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Heat map of the G matrix of 215 cotton historical breeding lines genotyped with 13,330 SNP markers.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of principle component (PC) 1 vs. PC 2 scores for each historical breeding line (N = 215). Principal component analysis performed on genomic relationship matrix (G) estimated from single nucleotide polymorphism data for each breeding line. Green, red, black and blue squares represent a mix of elite varieties and overseas introduced lines, lines derived from pre-2000, post-2000 crosses and elite varieties, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865172322" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of principle component (PC) 1 &lt;em&gt;vs.&lt;/em&gt; PC 2 scores for each historical breeding line (&lt;em&gt;N&lt;/em&gt; = 215). Principal component analysis performed on genomic relationship matrix (G) estimated from single nucleotide polymorphism data for each breeding line. Green, red, black and blue squares represent a mix of elite varieties and overseas introduced lines, lines derived from pre-2000, post-2000 crosses and elite varieties, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1721/F2.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/5/1721/F2.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489507" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Plot of principle component (PC) 1 <em>vs.</em> PC 2 scores for each historical breeding line (<em>N</em> = 215). Principal component analysis performed on genomic relationship matrix (G) estimated from single nucleotide polymorphism data for each breeding line. Green, red, black and blue squares represent a mix of elite varieties and overseas introduced lines, lines derived from pre-2000, post-2000 crosses and elite varieties, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Model selection using data From Myall Vale</h3><p id="p-33">We tested all the five models using data from each site and they gave similar results, so we only present results from one site (Myall Vale) that had the most number of breeding lines. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> provides five different commonly used models for GS and their respective residual variance estimates and the deviance information criterion (<em>DIC</em>) (measures of goodness of fit and model complexity) for each of the traits tested at Myall Vale site using 215 breeding lines. The five models gave very similar estimates and predictions, as the correlation between phenotypes and GEBVs derived from each of the models were relatively high (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). However, a smaller <em>DIC</em> is preferable suggesting that BayesB was the preferred model for both traits. The model with the smallest <em>DIC</em> also had the minimum residual variance. The phenotypes were standardized to a unit sample variance and the estimated residual variances for fiber length was 0.233, thus the model explained ∼77% of its phenotypic variance. The preferred model for fiber strength only explained 29% of the phenotypic variance. Based on residual variance estimates, all models fitted the data better for fiber length than fiber strength (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489731/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489731/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Measures of goodness of fit for different models for two fiber quality traits using data at Myall Vale site</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Estimates of variance components From Across-site and M×E models</h3><p id="p-35"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> provides phenotypic correlations of fiber length and strength across four sites that had common breeding lines. Correlations among sites were all positively correlated. For fiber length, correlations ranged from 0.51 to 0.68 and were higher for strength, ranging from 0.71 to 0.81.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489703/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489703/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Sample phenotypic correlation estimates ± SE for fiber length and strength evaluated at four sites</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Estimates of residual and genomic variance components for the single-site, across-site and M×E models are presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. For all traits and sites, the estimated residual variances for the M×E model were always smaller than those derived from the single-site and across-site models. These results indicate the M×E model fits the data better than models that (a) force the marker effects to be constant across sites and (b) single-site models.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489663/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489663/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Estimated posterior residual variance components (and their posterior standard deviations, SD) and the estimated posterior probability of markers with nonnull effects from the single-site, across-site and the marker × environment interaction models for fiber length (LEN) and strength measured at four sites</span><div class="sb-div caption-clear"></div></div></div><p id="p-37">For the M×E interaction model (BayesB), marker effects are given in terms of the probability or proportion of markers with effects different from zero (nonnull) that are estimated for each of the components of the marker effects (<em>i.e.</em>, main effect and site marker specific effect (see <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Overall, across-site model had a lower proportion of markers with nunnull effect than the M×E model (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). For example, across-site model gave proportions of 29.8 and 23.8% of markers with nunnull effects for fiber length and strength, respectively, whereas the M×E model gave, for the marker main effect, proportions of 38.2 and 37.6% of the markers with nunnull effects for fiber length and strength, respectively. On average, the environment-specific proportion of markers with nunnull effect were greater than main effect, with the exception of Bourke (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>).</p></div><div id="sec-16" class="subsection"><h3>Prediction accuracy based on multiple training-testing partitions at each site</h3><p id="p-38">Prediction accuracy based on training-testing partitions (TRN70-TST30) for fiber length and fiber strength at each of the seven sites individually are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Prediction accuracy for fiber length ranged from 0.27 at Collarenebri to 0.77 at Myall Vale (with the highest number of breeding lines). However, at five of seven sites, excluding Collarenebri, prediction accuracy averaged 0.46. For fiber strength, accuracies ranged from 0.19 at Bourke to 0.58 at Darling Downs, with an average of 0.50, excluding Bourke.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated prediction accuracy (prediction accuracy between phenotypes and predictions averaged over 50 TRN-TST partitions) for cotton fiber length and fiber strength at seven test sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865172322" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated prediction accuracy (prediction accuracy between phenotypes and predictions averaged over 50 TRN-TST partitions) for cotton fiber length and fiber strength at seven test sites.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1721/F3.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/5/1721/F3.medium.gif" width="440" height="204"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1721/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489511" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">Estimated prediction accuracy (prediction accuracy between phenotypes and predictions averaged over 50 TRN-TST partitions) for cotton fiber length and fiber strength at seven test sites.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Assessment of prediction accuracy in a multi-site context for fiber length and strength</h3><p id="p-40">Prediction accuracies obtained under different models for fiber length and strength are presented in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> (CV1) and 6 (CV2). Single-site analysis by either validation approaches yielded moderately low prediction correlations for fiber length, ranging from as low as 0.19 for to 0.34 (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Tables 5</a> and <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">6</a>). Similarly, low prediction correlations were observed for fiber strength, ranging from 0.12 to 0.35 (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Tables 5</a> and <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">6</a>). Under CV1, across-site model and the M×E model performed similarly to single-site model. For an across-site model that analyzed data jointly under CV2, the model yielded higher prediction accuracies than single-site models for length and strength, with an average prediction correlation of 0.41 and 0.39, respectively (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>). Prediction accuracies from the M×E model were higher than those for the across-site model, with an average prediction accuracy of 0.71 and 0.59 for fiber length and strength, respectively (<a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>). For all models and trait combination, the slope coefficients were not significantly different than 1, <em>i.e.</em>, indicating no significant bias in the GEBVs prediction.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489755/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489755/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Estimated prediction accuracy (correlation coefficient between predicted and observed phenotypes, averaged over 50 TRN-TST partitions) ± SE for fiber length and strength in cotton by CV1</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489602/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489602/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Estimated prediction accuracy (correlation coefficient between predicted and observed phenotypes, averaged over 50 TRN-TST partitions) ± SE for studied traits in cotton, CV2</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">To compare changes in ranking of the breeding lines based on GEBVs from each of the models, we used the coincidence index, which estimates the selection efficiency of multi-site relative to single-site models (<em>e.g.</em>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hamblin and Zimmermann, 1986</a> – page 253). Briefly, when the number of breeding lines identified by the multi-site and M×E interaction models is no more than the number of breeding lines expected by chance, the selection efficiency is zero (<em>e.g.</em>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Hamblin and Zimmermann 1986</a>). Likewise, when the number of breeding lines identified in the multi-site and M×E interaction models is the same as the number selected by the single-site model, the selection efficiency is 100%. Each site had 116 breeding lines and therefore a selection intensity of 20% identified 23 breeding lines. Assuming a random relationship of the breeding lines in different models, approximately 5 breeding lines (23 × 20%) would be identified by chance. For example, under CV1 at Myall Vale, 5 of the breeding lines identified using the single-site model were also in the top 20% identified by the multi-site model. The selection efficiency in this case would be estimated as follows:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1721/embed/mml-math-9.gif"/></noscript></span></span>Under CV2, mean selection efficiency of the across-site and M×E interaction models over single-site model was 40% and 48% for fiber length and strength, respectively (<a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a>).</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-44">Genomic selection is now taking advantage of the large amount of phenotypic data collected by breeding programs across years, provided there is seed available DNA extraction upon which we get genotypic data. However, this also raises new challenges to optimally exploit those data. By nature, historical data can be extremely unbalanced. Historical data will include trials of varying quality without readily accessible meta-information on issues affecting trial quality (<em>e.g.</em>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Heslot <em>et al.</em> 2013</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Rutkoski <em>et al.</em> 2015</a>). This study demonstrated that historical data on fiber length and strength when combined with high density SNP data, can be used to develop predictive models for complex agronomic traits of cotton. Varying training population sizes ranging from 200 to 500 individuals were used in most initial GS studies in other crops (<em>e.g.</em>, <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2014</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Spindel <em>et al.</em> 2015</a>). Some studies have reported genetic gain to double even when using merely 500 individuals, compared to phenotypic selection. For example, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lorenz (2013)</a> and <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lin <em>et al.</em> (2016)</a> have found that GS with optimized breeding designs can enhance genetic gain, while consuming less cost per unit time as compared to traditional breeding.</p><p id="p-45">This study focused on two economically important fiber quality traits – fiber length and strength that have similar and reasonably moderate genomic heritabilities. We were not able to compare our genomic heritability estimates from other studies as there are no estimates in the literature, except for those estimated using the additive relationship matrix. However, our estimates fell within the range reported by <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Campbell and Myers (2015)</a> that showed that the estimates range from 0.23 to 0.57 for both traits, depending on the population sampled and the environment for which they were evaluated. Because of a high level of relatedness in the breeding lines, they likely share long chromosome segments and, under these circumstances, the patterns of allele sharing at markers and at QTL are very similar. This leads to very small bias in genomic heritability estimates (<em>e.g.</em>, <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">de los Campos <em>et al.</em> 2015b</a>).</p><p id="p-46">Heritability of a trait corresponds to the upper limit of the phenotypic variance explained by a linear genetic prediction model (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Clark <em>et al.</em> 2012</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wray <em>et al.</em> 2013</a>). A linear relationship was expected, since traits with moderate heritability, normally, present phenotypes with expression of relatively high genetic variance, and are expected to be more predictable by a genomic approach. This was the case for fiber length at most sites but uniquely not fiber strength at Myall Vale, where the residual variance was almost 70%, suggesting that other factors not accounted for by the model were contributing to phenotypic variance (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Incorporating other factors such as environmental covariates would reduce the amount of residual variance for fiber strength (<em>e.g.</em>, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Jarquín <em>et al.</em> 2014</a>).</p><div id="sec-19" class="subsection"><h3>Model selection</h3><p id="p-47">Complex traits are possibly affected by large numbers of small-effect QTL and the analysis of such traits requires fitting a large number of variants concurrently using a GS approach such as the one proposed by <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Meuwissen <em>et al.</em> (2001)</a>. Since the introduction of genomic selection models, empirical evidence has demonstrated that no single model performs best across species and traits and grouping traits based on their architecture is not always straightforward (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">de los Campos <em>et al.</em> 2013</a>). All five models performed similarly although, the deviance information criterion favored BayesB which is based on specifying all SNP-associated effects to be independent of each other, allowing a large proportion of SNP markers to be associated with null effects (<a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Yang and Tempelman 2012</a>). When markers and QTL co-segregate, variable selection does not seem to be needed (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">de los Campos <em>et al.</em> 2015b</a>). Given that fiber length and strength are moderately heritable, we would expect both traits to be associated with many loci, each explaining only a small portion of the genetic variance. We would expect BayesB to have increased prediction accuracy over BRR or RR-BLUP because they avoid over-shrinking those QTL with moderate effects (<em>e.g.</em>, <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">de los Campos <em>et al.</em> 2013</a>). We would also expect prediction accuracies from BayesB to persist in new populations because they focus more on analyzing QTL-marker associations rather than on relationships, whereas RR-BLUP rely on kinship more strongly. The genetic correlation between the two traits is known to be positive and high ∼0.80 (<em>e.g.</em>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lacape <em>et al.</em> 2005</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gapare <em>et al.</em> 2017</a>). Likewise, we would expect the two traits to be controlled by similar sets of loci, although the influence of other factors on fiber strength cannot be excluded, given that the residual variance for fiber strength was much higher than that for fiber length.</p><p id="p-48">Marker density plays an important role in genomic prediction. For GS, it is desirable to obtain adequate genome coverage so that all contributing QTL are in LD with at least one marker. Studies have shown this optimum marker size to be trait and population specific (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Heffner <em>et al.</em> 2011</a>). The SNPs used in our study were distributed across all the 26 chromosomes of cotton with a density of ∼5.3 SNPs/Mbp. Implementing GS into a breeding program would likely involve increasing training panel size over time as more phenotypic data are generated. Thus, as more individuals are added to the model, there may be a need to reassess the optimum marker number. However, if the accuracy obtained in this study is due to LD, then it is more likely to persist across generations and breeding lines than if the accuracy due to relationships (<em>e.g.</em>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Habier <em>et al.</em> 2007</a>; Habier <em>et al</em>. 2013).</p></div><div id="sec-20" class="subsection"><h3>Relationships and population structure</h3><p id="p-49">Naturally and artificially selected/breeding populations usually exhibit some degree of relatedness and stratification. Pronounced population structure has to be considered when evaluating the potential of genomic selection (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Isidro <em>et al.</em> 2015</a>). In general, when population structure is not taken into account, genomic prediction accuracy decreases (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sallam <em>et al.</em> 2015</a>). As might be expected for breeding lines that have gone through repeated selection and breeding, leading to relatedness. The heat map based on <strong>G</strong> matrix suggested that our population was highly related. In our study, we did not detect any evidence of significant subpopulation structure although, there was evidence of strong admixture, with 24.1% of the total variation being explained by PC1 and 12.7% by PC2.</p></div><div id="sec-21" class="subsection"><h3>Prediction accuracy based on multiple training-testing partitions at each site</h3><p id="p-50">Prediction accuracy for fiber strength at Myall Vale was almost half that observed for fiber length. However, based on their known heritabilities, we had expected that prediction accuracy for fiber strength would be in the same range as fiber length. Possible reasons for such low prediction accuracy for strength include (i) model inadequacy as evidenced by the high residual variance (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>); (ii) the genomic region affecting the trait might not have been effectively covered in the current genotyping data using a SNP chip that does have some gaps in coverage on some chromosomes (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Hulse-Kemp <em>et al.</em> 2015</a>), and (iii) small size of the training population. There are ongoing efforts to increase the size of the training population. Accuracy may improve when we can move to more extensive genotype-by-sequencing approaches to genotyping. One possibility would be to expand any of the previously presented models by including an interaction term between environmental covariates and the random effect of the markers. For example, LD between markers and genes at causal loci or because of model misspecification (<em>e.g.</em>, interactions between alleles that are unaccounted for), the regression on markers may not fully describe genetic differences among lines (<em>e.g.</em>, <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Jarquín <em>et al.</em> 2014</a>).</p><p id="p-51">With the exception of Myall Vale, prediction accuracies for fiber length at each site were almost the same, averaging 0.43. One possible reason for differences in prediction accuracy between Myall Vale and other sites may be the high genomic heritability estimate and also the size of the training population that was almost double that of any other site (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Some of the variation observed in our results could be due to other unmeasured features (<em>e.g.</em>, environmental variables), because accuracies from prediction models depend on a complex network of different, interrelated factors. The unknown factors may explain some proportion of the variation in the datasets and increase the power of the model for prediction.</p></div><div id="sec-22" class="subsection"><h3>Variance components From Across-site and M×E models</h3><p id="p-52">Our results agreed with those found by <a id="xref-ref-52-8" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> (2015)</a> and <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> (2016)</a>, where the M×E model fitted the data better than the single-site and across-site models. In this study, fiber length and strength had positive and high sample phenotypic correlations among sites. The across-site model had relatively higher residual variance than the M×E model, indicating that forcing constant marker effects across sites, <strong><em>b</em></strong><sub>1</sub> <em>=</em> <strong><em>b</em></strong><sub>2</sub> = <strong><em>b</em></strong><sub>3</sub> = <strong><em>b</em></strong><sub>4</sub>, does not produce a better fit of this model (<em>e.g.</em>, <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2016</a>). The M×E model is based on variance component estimation of the marker main effects and site-specific marker effects and, in terms of prediction accuracy, it performed well at all sites since they were positively correlated (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). As might be expected for traits that have positive correlations between sites, the M×E model will tend to have higher prediction accuracy for traits that have positive correlations than traits with zero or negative correlations between sites (<em>e.g.</em>, <a id="xref-ref-52-9" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> 2015</a>).</p></div><div id="sec-23" class="subsection"><h3>Genomic selection accuracy Across sites</h3><p id="p-53">Genomic prediction models have been proposed that take into account the random effects of markers and their interaction with environments based on genetic and environmental similarities among individuals (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Perez-Rodriguez <em>et al.</em> 2015</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cuevas <em>et al.</em> 2017</a>). As might be expected for traits that are moderately influenced by environment, M×E model outperformed both single- and across-site models. Overall, the M×E model performed best under CV2 (<a id="xref-table-wrap-6-6" class="xref-table" href="#T6">Table 6</a>). The low prediction accuracies under CV1 and higher accuracies under CV2 are consistent with those of previous studies in cotton (<em>e.g.</em>, <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Perez-Rodriguez <em>et al.</em> 2015</a>) and in wheat (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lado <em>et al.</em> 2016</a>) that have used similar cross-validation designs. These studies also found that the inclusion of G×E produced a considerable increase in prediction accuracy. Predictions in TST data sets are derived using information from the genotypes included in the TRN data; therefore, prediction accuracy depends on how much information can be borrowed for genotypes in the TRN set that is relevant to those in the TST set (<a id="xref-ref-52-10" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Velu <em>et al.</em> 2016</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Sukumaran <em>et al.</em> 2017</a>).</p><p id="p-54">Our results support evidence of increased accuracies by incorporating G×E terms in prediction models (<a id="xref-table-wrap-6-7" class="xref-table" href="#T6">Table 6</a>). For example <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Perez-Rodriguez <em>et al.</em> (2015)</a> demonstrated that prediction accuracies in cotton yield could increase from 0.45 to 0.51 by using models that incorporate G×E terms. One advantage of the M×E model is that it can be used with priors that induce differential shrinkage of estimates as well as priors that produce variable selection (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">de los Campos <em>et al.</em> 2013</a>). Such treatment would potentially aid in identifying sets of markers with effects that are stable across environments and others that are responsible for G×E (<a id="xref-ref-52-11" class="xref-bibr" href="#ref-52">López-Cruz <em>et al.</em> 2015</a>). For example, <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> (2016)</a> used the M×E GS model to identify genomic regions in which the effects are stable across environments and other regions that are specific to certain environments and therefore responsible for G×E in grain yield in durum wheat. We envisage that this model would be useful for fiber yield components traits that exhibit considerable amount of G×E (<em>e.g.</em>, <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Perez-Rodriguez <em>et al.</em> 2015</a>) and have low heritability (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Liu <em>et al.</em> 2011</a>).</p><p id="p-55">Using data from one site to predict the performance in independent sites is crucial for plant breeding. The GS prediction accuracy between sites for fiber length and strength were moderately high (<a id="xref-table-wrap-6-8" class="xref-table" href="#T6">Table 6</a>). The M×E model under CV2 is based on variance component estimation of the marker main effects and environment-specific marker effects and, in terms of prediction accuracy, it performed well at all sites. This is an important result for cotton breeding given that all initial selection and breeding are carried out at the breeding program base location (Myall Vale site) with the follow-up of elite lines being tested across regions in breeding target environments. Thus, as for fiber length and strength, the ability of the data from one environment to predict the other, was similar, irrespective of which site’s data were used to build the prediction model.</p></div><div id="sec-24" class="subsection"><h3>Conclusions</h3><p id="p-56">GS is still in its infancy in most plant breeding programs, and one of the biggest obstacles for implementing GS in practical breeding is the high start-up costs required. The investment in starting GS is substantial in both technology and human resources required with regards to the costs of phenotyping, maintaining a large training population, costs of genotyping entire breeding populations and model optimization analysis. However, genotyping costs are continually decreasing and genotyping of large plant populations is much more manageable today than it was just a few years ago. We also envisage that further cost reductions could be made by utilizing historical phenotype data, as it would reduce the costs of establishing, phenotyping, and maintaining initial training populations significantly. If historical data can be correctly adjusted for annual variation of environmental factors, they represent a substantial resource. Historical data could be used to initiate a GS breeding program, allowing breeders to realize the potential and benefits of GS, before incorporating contemporary data and recalibrating the model.</p><p id="p-57">Further work is required to increase the size of the training population and test the models on cotton fiber yield and other fiber quality traits before implementation of GS in cotton breeding. In the near future, a model that integrates the analysis of multi-traits and multi-environments and takes into account trait × genotype × environment interaction (T×G×E) in a unified whole genome prediction would be desirable. GS could have its greatest potential use at points in the breeding program where selection using traditional methods (for example, through the generation of phenotypes via replicated trials) is too expensive, time consuming, or not biologically or logistically possible because of the wide geographical distribution of the industry in Australia (∼1,500 km from hot to cool climates). The most important questions relating to the applicability of GS in the CSIRO cotton breeding program are whether it will better help breeders predict: (1) breeding values of individuals for more rapid selection cycling or (2) genotypic values of advanced lines that are in the last stages of testing and we believe that for the traits examined in this study that it will provide value if adopted as a routine part of Australian cotton breeding. This study can contribute to breeding programs for other crops, like wheat or barley, where a conventional strategy of selection based on the phenotype is used and much historic phenotype data are available that has been collected over multiple years by breeders.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">This study was financially supported by Cotton Breeding Australia, a joint venture between CSIRO and Cotton Seed Distributors. The authors would like to thank technical staff of CSIRO cotton breeding group as well as past cotton breeders, for their invaluable contribution to this work. We also thank Rowan Bunch and Dr. Bill Barendse for processing and help with the analysis of the Illumina CottonSNP63K assays. The authors thank Dr. Klara Verbyla for constructive discussions on GS and reviews of the initial drafts of the manuscript by Drs Sonya Dominik and Yutao Li which greatly improved the manuscript. We also appreciate the comments and suggestions from the journal reviewers that improved the readability of the manuscript.</p><p id="p-59">Competing interests</p><p id="p-60">The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-61"><em>Communicating editor: P. Brown</em></p></li><li class="fn" id="fn-7"><p id="p-62">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200140/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200140/-/DC1</a>.</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-07"><span class="received-label">Received </span>February 7, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-03-17"><span class="accepted-label">Accepted </span>March 17, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Gapare <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.1" data-doi="10.1186/s12863-017-0512-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernal-Vasquez</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction in early selection stages using multi-year data in a hybrid rye breeding program.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-017-0512-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-017-0512-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-017-0512-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.2" data-doi="10.1186/1471-2164-15-646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernal-Vasquez</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Möhring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönleben</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The importance of phenotypic data analysis for genomic prediction - a case study comparing different spatial models in rye.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.3" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype x environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1721.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>••• <span class="cit-source"><em>ASReml-R reference manual</em>, <em>release 3. Technical report</em></span>, <span class="cit-publ-name">NSW Department of Primary Industries</span>, <span class="cit-publ-loc">Orange, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1721.5" data-doi="10.2134/agronmonogr57.2013.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">G. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative Genetics</span>, pp. <span class="cit-fpage">187</span>–<span class="cit-lpage">204</span> in <span class="cit-ed"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D.D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Percy</span>  <span class="cit-name-given-names">R.G.</span></span>, editors, <span class="cit-source"><em>Cotton</em>, <em>America Society of Agronomy</em>, <em>Agronomy Monograph</em></span>, <span class="cit-comment">Vol. 57, Madison, WI</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronmonogr57.2013.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCotton%252C%2BAmerica%2BSociety%2Bof%2BAgronomy%252C%2BAgronomy%2BMonograph%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronmonogr57.2013.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronmonogr57.2013.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.6" data-doi="10.1186/1297-9686-44-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-44-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DClark%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DThe%2Bimportance%2Bof%2Binformation%2Bon%2Brelatives%2Bfor%2Bthe%2Bprediction%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Band%2Bthe%2Bimplications%2Bfor%2Bthe%2Bmakeup%2Bof%2Breference%2Bdata%2Bsets%2Bin%2Blivestock%2Bbreeding%2Bschemes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-4%26rft_id%253Dinfo%253Apmid%252F22321529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22321529&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.7" data-doi="10.1016/j.fcr.2011.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Constable</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Negative associations still exist between yield and fiber quality in cotton breeding programs in Australia and USA.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2011.12.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D128%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2011.12.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2011.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.8" data-doi="10.1007/s00122-013-2066-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeClerck</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Next-generation phenotyping: requirements and strategies for enhancing our understanding of genotype–phenotype relationships and its relevance to crop improvement.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">867</span>–<span class="cit-lpage">887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2066-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D867%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2066-0%26rft_id%253Dinfo%253Apmid%252F23471459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2066-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23471459&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1721.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Constable</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Approaches utilized in breeding and development of cotton cultivars in Australia</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span> in <span class="cit-source">Genetic Improvement of Cotton: Emerging Technologies</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">J. N.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>. <span class="cit-publ-name">Science Publishers</span>, <span class="cit-publ-loc">Enfield, NH</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.10" data-doi="10.2135/cropsci2015.04.0260"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuberosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Extending the marker × environment interaction model for genomic-enabled prediction and genome-wise association analyses in durum wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">2193</span>–<span class="cit-lpage">2209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.04.0260</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.04.0260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.04.0260&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.11" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.12" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic Selection in Plant Breeding: Methods, Models, and Perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.13" data-doi="10.1534/g3.116.035584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bayesian genomic prediction with genotype x environment interaction kernel models.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.035584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.035584%26rft_id%253Dinfo%253Apmid%252F27793970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjcvMS80MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNS8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.14" data-doi="10.1007/BF00222947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The analysis of the NSW wheat variety database. 1. Modelling trial error variance.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">21</span>–<span class="cit-lpage">27</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00222947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D92%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00222947%26rft_id%253Dinfo%253Apmid%252F24166112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00222947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166112&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TV93500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.15" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic Prediction in Animals and Plants: Simulation of Data, Validation, Reporting, and Benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.16" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">04</span>): <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.17" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.18" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigrees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-web" id="cit-8.5.1721.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015a</span> <span class="cit-comment">Bayesian generalized linear regression. R package version 1.0.4. CRAN</span>. <a href="https://cran.r-project.org/web/packages/BGLR/index.html">https://cran.r-project.org/web/packages/BGLR/index.html</a> (accessed 15 Jan. 2018).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.20" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D</span>  <span class="cit-name-given-names">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2015b</span> <span class="cit-article-title">Genomic heritability: what is it?</span> <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">e1005048</span>. <span class="cit-comment">doi:10.1371/journal. pgen.1005048</span>.<span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.21" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: Genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2014.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.22" data-doi="10.2135/cropsci2016.08.0715"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edriss</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jumbo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makumbi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction in a large African maize population.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">2361</span>–<span class="cit-lpage">2371</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.08.0715</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D2361%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.08.0715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.08.0715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.23" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.24" data-doi="10.1007/s10681-017-1855-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gapare</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conaty</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association study of yield components and fiber quality traits in a cotton germplasm diversity panel.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">213</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-017-1855-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D213%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-017-1855-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-017-1855-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.25" data-doi="10.1016/0003-2670(86)80028-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geladi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalski</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Partial least-squares regression: a tutorial.</span> <abbr class="cit-jnl-abbrev">Anal. Chim. Acta</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0003-2670(86)80028-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BChim.%2BActa%26rft.volume%253D185%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-2670%252886%252980028-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-2670(86)80028-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986E477500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.26" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: Prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9308-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.27" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.28" data-doi="10.1007/s11295-010-0328-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in forest tree breeding.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">241</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-010-0328-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0328-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0328-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000288656800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1721.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml User Guide Release 3.0</span>. <span class="cit-publ-name">VSN International Ltd.</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>. pp. <span class="cit-fpage">372</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.30" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.5.1721.31" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: A look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>  <span class="cit-name-given-names">M. J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Breeding common bean for yield in mixtures.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D4%26rft.spage%253D245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.33" data-doi="10.3168/jds.2008-1646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited review: Genomic selection in dairy cattle: Progress and challenges.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">433</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1646</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D433%26rft.epage%253D443%26rft.atitle%253DInvited%2Breview%253A%2BGenomic%2Bselection%2Bin%2Bdairy%2Bcattle%253A%2BProgress%2Band%2Bchallenges%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1646%26rft_id%253Dinfo%253Apmid%252F19164653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19164653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262654900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.34" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2010.12.0029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.35" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">423</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2529430</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.36" data-doi="10.2135/cropsci2012.07.0420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Using genomic prediction to characterise environments and optimize prediction accuracy in applied breeding data.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">921</span>–<span class="cit-lpage">933</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.07.0420</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D921%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.07.0420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.07.0420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.37" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.03.0249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.5.1721.38" data-doi="10.1371/journal.pone.0074612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Impact of marker ascertainment bias on genomic selection accuracy and estimates of genetic diversity.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">e74612</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0074612</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De74612%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0074612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0074612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.39" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.40" data-doi="10.1007/s00122&amp;ndash;014&amp;ndash;2431&amp;ndash;7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinze</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Molecular characterization of the Gossypium diversity reference set of the US national cotton germplasm collection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">313</span>–<span class="cit-lpage">327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2431-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B014%2526ndash%253B2431%2526ndash%253B7%26rft_id%253Dinfo%253Apmid%252F25431191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;014&amp;ndash;2431&amp;ndash;7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25431191&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.41" data-doi="10.1534/g3.114.010298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carriquiry</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1027</span>–<span class="cit-lpage">1046</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.010298</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010298%26rft_id%253Dinfo%253Apmid%252F24727289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvNi8xMDI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.42" data-doi="10.1007/s00122-016-2733-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabrera</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffstetter</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanford</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection for wheat traits and trait stability.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1697</span>–<span class="cit-lpage">1710</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2733-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1697%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2733-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2733-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.43" data-doi="10.1534/g3.115.018416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hulse-Kemp</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemm</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plieske</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buyyarapu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development of a 63K SNP array for cotton and high-density mapping of intraspecific and interspecific populations of <em>Gossypium</em> spp.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1187</span>–<span class="cit-lpage">1209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.018416</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.018416%26rft_id%253Dinfo%253Apmid%252F25908569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvNi8xMTg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.44" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2418-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.45" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.46" data-doi="10.2135/cropsci2005.0123a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belot</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giband</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">QTL analysis of cotton fiber quality using multiple × backcross generations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">123</span>–<span class="cit-lpage">140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2005.0123a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DLacape%26rft.auinit1%253DJ.-M.%26rft.volume%253D45%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D140%26rft.atitle%253DQTL%2BAnalysis%2Bof%2BCotton%2BFiber%2BQuality%2BUsing%2BMultiple%2BGossypium%2Bhirsutum%2Bx%2BGossypium%2Bbarbadense%2BBackcross%2BGenerations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0123a%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0123a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000226435300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.47" data-doi="10.2135/cropsci2015.04.0207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrios</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quincke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, and <span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">L.</span></span>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Modeling genotype× environment interaction for genomic selection with unbalanced data from a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">2165</span>–<span class="cit-lpage">2179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.04.0207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2165%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.04.0207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.04.0207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.48" data-doi="10.1071/CP13363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection in crops, trees and forages: A review.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">65</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1177</span>–<span class="cit-lpage">1191</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/CP13363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D65%26rft.spage%253D1177%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP13363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP13363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.49" data-doi="10.2135/cropsci2016.07.0577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pembleton</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badenhorst</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic gain and inbreeding from genomic selection in a simulated commercial breeding program for <em>Perennial Ryegrass</em>.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2016.07.0577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2016.07.0577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2016.07.0577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.50" data-doi="10.1016/j.fcr.2013.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Constable</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The interaction between breeding and crop management in improved cotton yield.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2013.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D148%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2013.04.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2013.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.51" data-doi="10.1007/s10681-010-0289-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Llewellyn</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacape</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Heritability and predicted selection response of yield components and fiber properties in an inter-specific derived RIL population of cotton.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">178</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">309</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-010-0289-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D178%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-010-0289-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-010-0289-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.52" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using a markers x environment interaction genomic selection model.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft_id%253Dinfo%253Apmid%252F25660166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNC81NjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.53" data-doi="10.1534/g3.112.004911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Resource allocation for maximizing prediction accuracy and genetic gain of genomic selection in plant breeding: a simulation experiment.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.54" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1166-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-2" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMahon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Low</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Growing degree days as measure of temperature effects on cotton.</span> <abbr class="cit-jnl-abbrev">Cot. Grow. Rev.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCot.%2BGrow.%2BRev.%26rft.volume%253D49%26rft.spage%253D39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.56" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.57" data-doi="10.1534/g3.116.027524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oakey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halpin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection in multi-environment crop trials.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1313</span>–<span class="cit-lpage">1326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.027524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.027524%26rft_id%253Dinfo%253Apmid%252F26976443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNS8xMzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.58" data-doi="10.1371/journal.pgen.0020190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and eigenanalysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De190%26rft.epage%253De190%26rft.atitle%253DPopulation%2Bstructure%2Band%2Beigenanalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020190%26rft_id%253Dinfo%253Apmid%252F17194218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.59" data-doi="10.2135/cropsci2014.08.0577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez-Rodriguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bondalapati</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pita</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A pedigree-based reacton norm model for prediction of cotton yield in multi-environment trials</span>. <abbr class="cit-jnl-abbrev">Crop Sci</abbr>. <span class="cit-vol">55</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>. <span class="cit-comment">doi:10.2135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci%26rft.volume%253D55%26rft.spage%253D1143%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.08.0577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.08.0577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.60" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-web" id="cit-8.5.1721.61"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>, <span class="cit-pub-date">2014</span> <span class="cit-comment">R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria (<a href="http://www.gbif.org/resource/81287">http://www.gbif.org/resource/81287</a>)</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.62" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.1033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.5.1721.63" data-doi="10.1186/1471-2164-13-452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">452</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D452%26rft.epage%253D452%26rft.atitle%253DComparison%2Bof%2Bwhole-genome%2Bprediction%2Bmodels%2Bfor%2Btraits%2Bwith%2Bcontrasting%2Bgenetic%2Barchitecture%2Bin%2Ba%2Bdiversity%2Bpanel%2Bof%2Bmaize%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-452%26rft_id%253Dinfo%253Apmid%252F22947126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22947126&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.64" data-doi="10.2134/agronj1991.00021962008300040001x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenthal</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerik</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Radiation use efficiency among cotton cultivars.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">83</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">655</span>–<span class="cit-lpage">658</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj1991.00021962008300040001x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DRosenthal%26rft.auinit1%253DW.%2BD.%26rft.volume%253D83%26rft.issue%253D4%26rft.spage%253D655%26rft.epage%253D658%26rft.atitle%253DRadiation%2BUse%2BEfficiency%2Bamong%2BCotton%2BCultivars%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1991.00021962008300040001x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1991.00021962008300040001x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.65" data-doi="10.3835/plantgenome2014.02.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection for quantitative adult plant stem rust resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.02.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.02.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.02.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.66" data-doi="10.3835/plantgenome2014.09.0046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Efficient use of historical data for genomic selection: a case study of stem rust resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.09.0046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.09.0046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.09.0046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.67" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing genomic selection prediction accuracy in a dynamic barley breeding population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.68" data-doi="10.1371/journal.pgen.1005350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection and association mapping in rice (<em>Oryza sativa</em>): effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical Rice breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e1005350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005350%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1721.69" data-doi="10.5772/58414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">I. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source"><em>Australian Cotton Germplasm Resources</em>, <em>World Cotton Germplasm Resources</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Abdurakhmonov</span>  <span class="cit-name-given-names">I.</span></span>. <span class="cit-publ-name">InTech</span>, <span class="cit-publ-loc">Rijeka, Croatia</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.5772/58414</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAustralian%2BCotton%2BGermplasm%2BResources%252C%2BWorld%2BCotton%2BGermplasm%2BResources%26rft_id%253Dinfo%253Adoi%252F10.5772%252F58414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5772/58414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.70" data-doi="10.1534/g3.116.036251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction with pedigree and genotype x environment interaction in spring wheat grown in South and West Asia, North Africa, and Mexico.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">481</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.036251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036251%26rft_id%253Dinfo%253Apmid%252F27903632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi80ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.71" data-doi="10.1021/ci034160g"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svetnik</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Culberson</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheridan</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Random forest: a classification and regression tool for compound classification and QSAR modelling</span>. <abbr class="cit-jnl-abbrev">J. Chem. Inf. Comput. Sci.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1947</span>–<span class="cit-lpage">1958</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/ci034160g</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BChemical%2BInformation%2Band%2BComputer%2BSciences%26rft.stitle%253DJournal%2Bof%2BChemical%2BInformation%2Band%2BComputer%2BSciences%26rft.aulast%253DSvetnik%26rft.auinit1%253DV.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1947%26rft.epage%253D1958%26rft.atitle%253DRandom%2Bforest%253A%2Ba%2Bclassification%2Band%2Bregression%2Btool%2Bfor%2Bcompound%2Bclassification%2Band%2BQSAR%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci034160g%26rft_id%253Dinfo%253Apmid%252F14632445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ci034160g&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14632445&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186848700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the Lasso.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.73" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.74" data-doi="10.1007/s00122&amp;ndash;016&amp;ndash;2726-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction for grain zinc and iron concentrations in spring wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2726-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122%2526ndash%253B016%2526ndash%253B2726-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122&amp;ndash;016&amp;ndash;2726-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.75" data-doi="10.1038/nrg3457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pitfalls of predicting complex traits from SNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">507</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3457</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D507%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3457%26rft_id%253Dinfo%253Apmid%252F23774735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23774735&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.76" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian Antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.131540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.77" data-doi="10.1534/genetics.107.085589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian LASSO for quantitative trait loci mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">1045</span>–<span class="cit-lpage">1055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.085589</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085589%26rft_id%253Dinfo%253Apmid%252F18505874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzEwNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.78" data-doi="10.1007/s00122-011-1745-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">769</span>–<span class="cit-lpage">776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1745-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D769%26rft.epage%253D776%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2BEuropean%2Bmaize%2Belite%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1745-y%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1745-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1721.79" data-doi="10.1093/jxb/erv494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiller</span>  <span class="cit-name-given-names">W. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Integrated mapping and characterization of the gene underlying the okra leaf trait in <em>Gossypium hirsutum</em> L.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">67</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">763</span>–<span class="cit-lpage">774</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erv494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferv494%26rft_id%253Dinfo%253Apmid%252F26567355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erv494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26567355&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/5/1721.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1711" title="Inter-replicon Gene Flow Contributes to Transcriptional Integration in the Sinorhizobium meliloti Multipartite Genome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1733" title="Comparison of the Relative Potential for Epigenetic and Genetic Variation To Contribute to Trait Stability" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/5"><span>Volume 8 Issue 5, May 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/5.cover-source.jpg?itok=Bl7MXdvV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1721" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hmyAoifYpiZwkaQs2PtYlllGvcBQR5lN70q78S3Aj9E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/5/1721" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/5/1721" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/5/1721" class="active" data-icon-position="" data-hide-link-title="0">Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489451" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8Jj4eql97mZOXdYhlDaGGnklp0kUnMFa7J4DcJbcBzQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1721.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/5/1721" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OkDAc0fSnqKV-d8Qq3NIeL6RsVUViqfoyqTW-MtMnXU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/5/1721" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1721.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489451" id="node489451--2" data-pisa="ggg;8/5/1721" data-pisa-master="ggg;g3.118.200140" data-apath="/ggg/8/5/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9787-8543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Washington Gapare</span>, <span class="highwire-citation-author" data-delta="1">Shiming Liu</span>, <span class="highwire-citation-author" data-delta="2">Warren Conaty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qian-Hao Zhu</span>, <span class="highwire-citation-author" data-delta="4">Vanessa Gillespie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9535-8707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny Llewellyn</span>, <span class="highwire-citation-author" data-delta="6">Warwick Stiller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-4626-8059" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Iain Wilson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1732;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200140 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489451--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Washington Gapare </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Washington%2BGapare%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gapare%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWashington%2BGapare%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9787-8543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Washington Gapare</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Washington.Gapare@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Washington.Gapare@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shiming Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shiming%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShiming%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Warren Conaty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Warren%2BConaty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conaty%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWarren%2BConaty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qian-Hao Zhu</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vanessa Gillespie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BGillespie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gillespie%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BGillespie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Danny Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9535-8707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny Llewellyn</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Warwick Stiller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Warwick%2BStiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stiller%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWarwick%2BStiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Iain Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIain%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4626-8059" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Iain Wilson</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/5/1721" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/5/1721&amp;title=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/5/1721&amp;title=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/5/1721&amp;title=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/5/1721&amp;text=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/5/1721&amp;title=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/5/1721&amp;t=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/5/1721&amp;title=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/5/1721&amp;title=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489451/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489451" id="node489451--3" data-pisa="ggg;8/5/1721" data-pisa-master="ggg;g3.118.200140" data-apath="/ggg/8/5/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Historical Datasets Support Genomic Selection Models for the Prediction of Cotton Fiber Quality Phenotypes Across Multiple Environments</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9787-8543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Washington Gapare</span>, <span class="highwire-citation-author" data-delta="1">Shiming Liu</span>, <span class="highwire-citation-author" data-delta="2">Warren Conaty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Qian-Hao Zhu</span>, <span class="highwire-citation-author" data-delta="4">Vanessa Gillespie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-9535-8707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny Llewellyn</span>, <span class="highwire-citation-author" data-delta="6">Warwick Stiller</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-4626-8059" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Iain Wilson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1732;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200140 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489451--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Washington Gapare </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Washington%2BGapare%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gapare%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWashington%2BGapare%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9787-8543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Washington Gapare</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Washington.Gapare@csiro.au" class="" data-icon-position="" data-hide-link-title="0">Washington.Gapare@csiro.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Shiming Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shiming%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShiming%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Warren Conaty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Warren%2BConaty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Conaty%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWarren%2BConaty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qian-Hao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian-Hao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AQian-Hao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6505-7417" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Qian-Hao Zhu</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vanessa Gillespie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BGillespie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gillespie%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BGillespie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Danny Llewellyn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BLlewellyn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Llewellyn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BLlewellyn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9535-8707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny Llewellyn</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Warwick Stiller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>CSIRO Agriculture & Food, Locked Bag 59, Narrabri, NSW 2390, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Warwick%2BStiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stiller%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWarwick%2BStiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Iain Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>CSIRO Agriculture & Food, GPO Box 1600, Canberra, ACT 2601, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iain%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIain%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4626-8059" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Iain Wilson</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1721&amp;rft.epage=1732&amp;rft.atitle=Historical%20Datasets%20Support%20Genomic%20Selection%20Models%20for%20the%20Prediction%20of%20Cotton%20Fiber%20Quality%20Phenotypes%20Across%20Multiple%20Environments&amp;rft.volume=8&amp;rft.issue=5&amp;rft.date=2018-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gapare%2C+Washington&amp;rft.au=Liu%2C+Shiming&amp;rft.au=Conaty%2C+Warren&amp;rft.au=Zhu%2C+Qian-Hao&amp;rft.au=Gillespie%2C+Vanessa&amp;rft.au=Llewellyn%2C+Danny&amp;rft.au=Stiller%2C+Warwick&amp;rft.au=Wilson%2C+Iain"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489451/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489451/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489451/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489451/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489451/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489451/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489451/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489451/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489451/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489451/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489451/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489451/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/5/1721" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Data Availability</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/5/1721.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/5/1721.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/5/1721.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/5/1721.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
