<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/5/1455" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/5/1455.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/5/1455.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/5/1455.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/5/1455/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/5/1455.atom" />
<meta name="HW.pisa" content="ggg;8/5/1455" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fast Ordered Sampling of DNA Sequence Variants" />
<meta name="DC.Identifier" content="10.1534/g3.117.300465" />
<meta name="DC.Date" content="2018-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2018 Greenberg. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/genetics.5968987" />
<meta name="DC.Description" content="Explosive growth in the amount of genomic data is matched by increasing power of consumer-grade computers. Even applications that require powerful servers can be quickly tested on desktop or laptop machines if we can generate representative samples from large data sets. I describe a fast and memory-efficient implementation of an on-line sampling method developed for tape drives 30 years ago. Focusing on genotype files, I test the performance of this technique on modern solid-state and spinning hard drives, and show that it performs well compared to a simple sampling scheme. I illustrate its utility by developing a method to quickly estimate genome-wide patterns of linkage disequilibrium (LD) decay with distance. I provide open-source software that samples loci from several variant format files, a separate program that performs LD decay estimates, and a C++ library that lets developers incorporate these methods into their own projects." />
<meta name="DC.Contributor" content="Anthony J. Greenberg" />
<meta name="article:published_time" content="2018-05-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="Fast Ordered Sampling of DNA Sequence Variants" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Explosive growth in the amount of genomic data is matched by increasing power of consumer-grade computers. Even applications that require powerful servers can be quickly tested on desktop or laptop machines if we can generate representative samples from large data sets. I describe a fast and memory-efficient implementation of an on-line sampling method developed for tape drives 30 years ago. Focusing on genotype files, I test the performance of this technique on modern solid-state and spinning hard drives, and show that it performs well compared to a simple sampling scheme. I illustrate its utility by developing a method to quickly estimate genome-wide patterns of linkage disequilibrium (LD) decay with distance. I provide open-source software that samples loci from several variant format files, a separate program that performs LD decay estimates, and a C++ library that lets developers incorporate these methods into their own projects.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/05/01" />
<meta name="citation_mjid" content="ggg;8/5/1455" />
<meta name="citation_id" content="8/5/1455" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/5/1455" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/5/1455.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/5/1455.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/5/1455.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300465" />
<meta name="citation_pmid" content="29531124" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="1455" />
<meta name="citation_lastpage" content="1460" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Anthony J. Greenberg" />
<meta name="citation_author_institution" content="Bayesic Research, Ithaca, NY 14850" />
<meta name="citation_author_email" content="tonyg@bayesicresearch.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3296-5811" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_issue=15;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=B. Efron;citation_title=Bootstrap methods: Another look at the jacknife.;citation_pages=1-26;citation_volume=7;citation_year=1979;citation_issue=1;citation_doi=10.1214/aos/1176344552" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=C. T. Fan;citation_author=M. E. Muller;citation_author=I. Rezucha;citation_title=Development of sampling plans by using sequential (item by item) selection techniques and digital computers.;citation_pages=387-402;citation_volume=57;citation_year=1962;citation_issue=298;citation_doi=10.1080/01621459.1962.10480667" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. B. Lack;citation_author=J. D. Lange;citation_author=A. D. Tang;citation_author=R. B. Corbett-Detig;citation_author=J. E. Pool;citation_title=A thousand fly genomes: An expanded Drosophila genome nexus.;citation_volume=33;citation_year=2016;citation_issue=12;citation_pmid=27687565;citation_doi=10.1093/molbev/msw195" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_title=THE INTERACTION OF SELECTION AND LINKAGE. I. GENERAL CONSIDERATIONS; HETEROTIC MODELS;citation_pages=49-67;citation_volume=49;citation_year=1964;citation_issue=1;citation_pmid=17248194" />
<meta name="citation_reference" content="citation_journal_title=ACM TOMACS;citation_author=M. Matsumoto;citation_author=T. Nishimura;citation_title=Mersenne twister: a 623-dimensionally equidistributed uniform pseudo-random number generator.;citation_pages=3-30;citation_volume=8;citation_year=1998;citation_issue=1;citation_doi=10.1145/272991.272995" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. R. McCouch;citation_author=M. H. Wright;citation_author=C.-W. Tung;citation_author=L. G. Maron;citation_author=K. L. McNally;citation_title=Open access resources for genome-wide association mapping in rice.;citation_pages=10532;citation_volume=7;citation_year=2016;citation_pmid=26842267;citation_doi=10.1038/ncomms10532" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. M. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic selection and association mapping in rice (Oryza sativa): Effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.;citation_pages=e1004982;citation_volume=11;citation_year=2015;citation_issue=2;citation_pmid=25689273;citation_doi=10.1371/journal.pgen.1004982" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The International HapMap Project.;citation_pages=789-796;citation_volume=426;citation_year=2003;citation_issue=6968;citation_pmid=14685227;citation_doi=10.1038/nature02168" />
<meta name="citation_reference" content="citation_journal_title=Commun. ACM;citation_author=J. S. Vitter;citation_title=Faster methods for random sampling.;citation_pages=703-718;citation_volume=27;citation_year=1984;citation_issue=7;citation_doi=10.1145/358105.893" />
<meta name="citation_reference" content="citation_journal_title=ACM Trans. Math. Softw.;citation_author=J. S. Vitter;citation_title=An efficient algorithm for sequential random sampling.;citation_pages=58-67;citation_volume=13;citation_year=1987;citation_issue=1;citation_doi=10.1145/23002.23003" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis;citation_author=H. Wickham;citation_year=2009" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fast Ordered Sampling of DNA Sequence Variants" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="twitter:description" content="Explosive growth in the amount of genomic data is matched by increasing power of consumer-grade computers. Even applications that require powerful servers can be quickly tested on desktop or laptop machines if we can generate representative samples from large data sets. I describe a fast and memory-efficient implementation of an on-line sampling method developed for tape drives 30 years ago. Focusing on genotype files, I test the performance of this technique on modern solid-state and spinning hard drives, and show that it performs well compared to a simple sampling scheme. I illustrate its utility by developing a method to quickly estimate genome-wide patterns of linkage disequilibrium (LD) decay with distance. I provide open-source software that samples loci from several variant format files, a separate program that performs LD decay estimates, and a C++ library that lets developers incorporate these methods into their own projects." />
<meta name="og-title" property="og:title" content="Fast Ordered Sampling of DNA Sequence Variants" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/5/1455" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Explosive growth in the amount of genomic data is matched by increasing power of consumer-grade computers. Even applications that require powerful servers can be quickly tested on desktop or laptop machines if we can generate representative samples from large data sets. I describe a fast and memory-efficient implementation of an on-line sampling method developed for tape drives 30 years ago. Focusing on genotype files, I test the performance of this technique on modern solid-state and spinning hard drives, and show that it performs well compared to a simple sampling scheme. I illustrate its utility by developing a method to quickly estimate genome-wide patterns of linkage disequilibrium (LD) decay with distance. I provide open-source software that samples loci from several variant format files, a separate program that performs LD decay estimates, and a C++ library that lets developers incorporate these methods into their own projects." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
    <title>Fast Ordered Sampling of DNA Sequence Variants | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Zf0zuUXjvo6xVsyter6BA9ALKXBPAWfLEZyIqygjux8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489473","apath":"\/ggg\/8\/5\/1455.atom","pisa":"ggg;8\/5\/1455","ac":{"\/ggg\/8\/5\/1455.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/5\/1455.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1455"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1455"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/5\/1455":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sf\\\\\\\/sGMZPrK\\\\\\\/S5XoPZv6brtjFWvvTlrOCk\\\\\\\/qq4IcBXSpEHqa5SRkIP5ANvsbljuYdF\\\\\\\/cCI79B0HNmT9XNQ1hMgAOrM7w2U48mW0XafJ6vWxmnYcfcjAs3A\\\\\\\/e\\\\\\\/rqrv1t84FTqD2L227QG1Zv41j8Vri1kc4RU45qVBCqr2NMCDqRLAWtcphzIqecezshBXb5sEFa0ycPDWW5S8FxpXW7ydKkJIAE8+NIQ1LCCA+DLG63pAN8g50em0FYvKTx3zcAuEkA864fhDRwv5ZdZEuDkWo6KlD8MpKY\\\\\\\/x+dxZpST17MNTih2dsXhIiPHVlyhSOk1+vEV5RxEfXRUJgHyspbeTRDvDHrWeTp1bxQ0VAX5N17FrOIvpoDOUxbPf+RB4L6zr5T8W0MQoyhB3pxKo+ooL2\\\\\\\/NoPgphyHRcxeKKp8R6IMJbOUcIBKWAy29vovcwE9YqSX2tSNMRKzJbXnpM8YUY03x9G7xKFKhCCHmTSMD7buOaiJ\\\\\\\/hKa8q6mY+hnUIUf6qDJz0B4Ym9YUH37k\\\\\\\/G3KfjG1uvs6Dysh4TPRJCVxUbevR\\\\\\\/y89j6q72vjn1VfKaUamZeyo8mz7X4zfPH4q2OJ0A87i1B6dzo\\\\\\\/a8yBmd5Ff2jvfYHHTmjI1yp0O7Sl51CSGe0vDe8NWASEABmhNchngMd9mzCYRgWlYG2e8Fo0QBLn0avhkSc7+Z5zoUIsSuMOa+nZo3RbqaOZ19W\\\\\\\/y5ODXv4SytFOW\\\\\\\/FZYKCJrHBq+VjKoDrAgCTXbhTvqNUokW0nCwAknAcxrmU+BS7varDxUWXh3VxjxNACl54oA\\\\\\\/VcW+BiuTzAa4LSFjmKcsCunUa3T5GkVB1Ci+1ZQTWpwJMFXAM41rB4jtoQPAHL\\\\\\\/xH5eaZE1wx4T+Kh5l\\\\\\\/cB17Dx0jFqOJSCNk8WqIrpNFQ4od3dGR7\\\\\\\/3Gy\\\\\\\/gwTZCeCdBecwdTUpro7bxirgJhCQiS5O27\\\\\\\/jfNrc4nLYwOhcvIgLPDOaTzTtxdhfabXrhLpHQID0rNkXqDdPcoK2vY5tUOV8c07uNX8O1NkHCkXcETEcL7ZFI\\\\\\\/3KrITkzKzlornasGPk2jdgcwYHy2DbGmDmw7UefakB3754zBxgZ+wZmkM\\\\\\\/UwX4PO57yHXeV1x4bI\\\\\\\/4KRGz53b+ZUGEDNOyh5aTefV9+kCdM2Xln6VoHT4w6LC1lVsZ7BxrTwpjDuoHvfhgek0KlHs\\\\\\\/BoNf0RnDrOefvkzhDykl2dK617EVrKQ9Ck75AXGQZaJbjO8FjFMF70IN58ZCG2jQRmRSR6O6N\\\\\\\/sWG6NsWeUT\\\u0022,\\\u0022iv\\\u0022:\\\u0022N+k\\\\\\\/Nmpcc2fdUNfgoD7SiQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022950b0cf58d3479513359cf62f4e850be1b67fde8793e17efbd749555e9f52789\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022N+KnP81Q+ImlxbijW0Szev7RuP7KtjC5NKgcLKTnU1iyGzk0heck3Zllg7mG2B2UXpBAjoTMVGSdxeizS7VH+9uuC7uctANk1eG04GOyzbxEEIEh061mCosmNFzswT2cyCg8jwWGOEKXv4Gf1gyuFxKYPOSu+W0wHi9MC6R8zwrQg2AE7C\\\\\\\/G3kQ226tCW8hon8W\\\\\\\/TCchSMftoGLN\\\\\\\/flMlvkA9M+DU6qaqmdUb\\\\\\\/gBFGDuLuGmn6v38FcaoHjoRucvGoMFBBWyc0CSc5AzuBfY7Lqicdbb6TRBziqVmmy+ZlsK33b9rWWOHYz6EtVV6Ex1N\\\\\\\/8bFy4Y\\\\\\\/3lXZxfv7Vz6HFNHpHSot5+g1CPzNLcSGXxVa9nuL3qi12tG3VSEGS9kM\\\\\\\/CWpI4dvsPBfUC9W2Bllj1ufRmJmf4\\\\\\\/qiqO0slHU5NLPJSluzk++ydYmOv75QAvCRujlEBD9ZpyClLNJg4OjTUJp4LRBysIaI1Ge5trOOKhYnwtB6uk5cV3ohTu9Ppxz\\\\\\\/lfNwlZEM+1x4UWT92mlB22ivZOrMWtKfd+2aDCPyroLARq8NCr8En1K4R49I4+Lz6oZcoQaC+eSOC++OKc\\\\\\\/FEV2IwAxFN2Q3v43KfE0HIy7cIY9y6RIiU44vwc1POQRt8tudXwfS4lVM1NgDjTKvBwDjXurb4nZK7eLksnXPc8yrIJujrCgYUPTyCA2JiqE2gFlsvmKOJJPODbrEk3Bb0W5SQPahNZE4dl4PIyYQVWtDZ58DdROYW9tfiVNSIwdPQmv9X0OHriJ52CGzISyLdMgSbaxrmOlz8n1enRaItDE2DrVf0XBo2neCgB5vGqNs5hVtWBUpiz2ZN1uWM\\\\\\\/CxCPXbNvVmuzqSF\\\\\\\/rKYD5oByVF2mal8XcQuiiYiOdgfgmEKp2Frcv411JAgLWBsWs64RKdRkPDsXXtsaFEZgTGCtUj1FMpgEbUMY5yXUn3JdbfxWXROuxxx5oIv1TiEZJRyTgaraPD5yOHckGHS1gnjENj9sYnUf5u8mZusX9j5lvhMlDOW1V2wlRGDNqnzWu\\\\\\\/i+BKOpsh+KGbavhtU1fWgssGdmNXJhoVnrVU5fg5u9PI8Y47WkbXWMNiIcJOf0v01NGocaM1qxaxeJqY84IlOUPb4tF2Zj+4OqdxVFttJsy5ywh+JZzA1ILUe4livEFwvjcdMSgWZLua11cOW6uWPraAOT22K8Ehm8D5tGVs6NaafwUrT+EYLIwuUryL0BMng1B2ZIHInxcSuhy2bMDgWaNngtIHk\\\\\\\/rSrDgTuKTOV5Vc5Cvi\\\\\\\/LaZSxr2Xylj6bEF2RLJaNJVZwe+9LI65p918Brf2HSnrMu2doMNEXluYImPQZiLAY4glwbbi\\\\\\\/AlnC0BOmDjYF+1jvYbSQ02x2O7ezeeX\\\\\\\/IfzGB8zN5NFdtDf3aK6nAZPQEwM\\\\\\\/ysKIf31n0dIEDx4XmY1vkcz2u\\\\\\\/o0LACgqhh0ajqnFxTSblGnob6B5tdQLLS\\\\\\\/OMidLBJjC6zrqeTg2WPrgIkgMXPdLx8YEAyUJMiB2yMs0FEWW\\\\\\\/0MVQ2t39eXYlzp3yaslCzS5Lqcu9voB0FTo3EhgnunVN4G5\\\\\\\/eeQXjpQ7P6vyk4Mf3RmQvyvGu7uoBe8jRFJs1rM8NtWxEHdJ+VJGWDfdqcM2viY4FHYRlgltn3a7XSNUfH8pCi0rbUZzRXXKyBXTePK3OzP7o49HGwo9VlKf7JY1lQqzkIYapB25sfD\\\\\\\/9+OD5xd8dsL6HIqmRth3PHbshs5fe8YKudSkxZCQy9Oqn\\\\\\\/Pk+66akNySIhRL20LjGvwM88iBbzIJ3\\\\\\\/GM3rCR3ZjZ99z1GPUKhv9XgHRGM99PyJF64sxAdssHWggTtF0UBqDf9zox00zYu+EqtL71LY5vYVkyb34MBhvb2fFeHGrIoP7Q9+fuY\\\\\\\/tQ09YHVHUXlemajBrIQUnXUqNBHdNglk4R2L16fW3F\\\\\\\/zzjxxi4j+zjVtrWqmVnVsgxk0wl0iFwNNnIrFMhJeDXzk\\\\\\\/a7Cj4USzbRZFDc9GcmyGBoDdFac52UcslBxl6fSDniTvnXZ1T\\\\\\\/qbFviJb7Tam+kscJYSCanARyuHZJG7\\\\\\\/NK11i2wbIQs3hP7\\\\\\\/bJb21JbOFDJBNZ3IePo4neiuwoMYceCot\\\\\\\/QiXdY9K9fh0mo45vCNz8vqzWLMPqzeHesimVzn4HyXMBd1rO3toNNCdLJW05+jAW7IZ1XTf23p8WdVZimjs4uvD2dFqCI1TfBpitMObd3Gs2ffzCtq7fmVwzm2WrQdDPL8ljFG2Bao+n8dApikyC3mO2rkBvo+OLa44CnpM3\\\\\\\/8UY0nxc0pf660rCKPzqVH7uF6meqIoRnQajvEeRqE7hopi9beCCRJ1PdGmD8wL0JZdxiCOXO2cQ3Htw2fgAP0zVmzrKeQqDidvLhzFPp5RpH18W7cDr0tCUDP6GjXrSsbstJIiDRvoSBf9S2cDYoQSON2vjlIJo7U8zmVR1To++GePBDuiaDAJRf7UuCTqKnxM07R3mBhV31RCiwKFzoE1ldIx3CPr0v8MN9qStEw7y05CWx40N7w5jF\\\\\\\/nSX2ygoUF3CH1ga+B3Jl6rT7F\\\\\\\/V\\\\\\\/hLrTAhfzAvMeehngBPw1u\\\\\\\/SZKjt84v0Rf7jLJdYcnHcTY5TuACpXQJnP1mTQVmvTWUorgK8IcvvQ6IoSpPnoci6Eu24FgKAY3mD7mjKFqLmelkjDuIjeq5fAm0pF5BoHLZw0ZuLZYpz5PLcpJbqCg2eR4roRae6U1wLEJ9dPuwwBAElr7DbQ+gJloHJLPf4n65hxsvMjBKy4O\\\\\\\/eMQNEtEUMns3ZbTt32EaalUyYMeGLxeV40bhZcQ7kq8zxbfWOfjowEZbkc1t4yvstZ\\\\\\\/5pwVkjc3ri3Li059GH788pf6DKWjWQE\\\\\\\/+KLXZLX3cp+3xCh7z2n+b84leiBBdsOmy1ly\\\\\\\/4w19xghgz09U+4THHYtvE3SlxXwIaGRr6+QbS07TROoc+Tym96TKGLzJZZxAysufk0QSwTeGPVjEGOQgEf7Pz2BC4YD7Kgars5wYjcil8lJnJl\\\\\\\/K1MNlxvmZeVIm+ztvA681ywx7im58oHcvKLqAPkxHueDZbSPavttUokmRp\\\\\\\/bp0P7954Yg65ANihOJPcBcwy33A5Hll7xXeirhtWPdRin1BpLUYvRVBWa98YmSLfifFYf4il6ARlo0n2QBai9vGVMDzswljy4RhkCQVrxiNIRDYI8AXgaXNmfWX3tlsG\\\\\\\/bkZhiCq4atQsr0XHF+h8bmD8lwg02muDeer9J1p8xMGCiKxiY8QnHF7iSUu7NkEWaGYGr9f0P+lzhapkqzEbyMucysqoWu7NgI4XooKVu7n8hinQoCe2qNEI+bI\\\\\\\/OwushIeZfpczltUVjepsZ\\\\\\\/ZBbZp+Wu0a4wT365H1gPQV1P1q5xC59pLoLm5\\\\\\\/n35aWWmXl\\\\\\\/mMuhFt+KCF3YEaZ60FC1iq3bZXOGCTvztFgVWXAVwgg\\\\\\\/fwcR\\\\\\\/jdlvM5w8khUJ7\\\\\\\/1Tk9LENiI4F5WUEKSntB1C8eNg61TmL2lYHQZ0pImscooYmvbeyLZFnOEvIfhUWtepLfYr6pBTFRPvcv3YH80PI54nh4HHK6+wNXbkRhySxweTMp0OAaI1CjCydz\\\\\\\/kryQ1kgN\\\\\\\/0rXY646VGgx5vxl\\\\\\\/lfd0FJY5JFUxZlSvhKn8g20aawBvfqAv2ZXhAsH84b9m1Uro47eG7FT23CqJ2ERuXvmh9asHPU5d4hzWUe6fAkNcEI9D5KY3yI3JDOmsUAKL2toBFe5wrfNISMdSOZy05MBMvFS6JjS55MX9y0tjIQVEUPLh1znbznpt0STqE2TxWHcdAymNmd+\\\\\\\/fVO9LTpJGf5iXZgobb+uU2o42MQXrZq7hW8DQ9mKrL6qsl\\\\\\\/wGH2w8caJeK1J+hL7N7XCgTgaDuuYgQ3y8hPdryLwwSx4gdtDycE\\\\\\\/51kJxc\\\\\\\/JU6tsjmzBpk02lQtTvqMhOdKGA91C2QSOrZgODAg2lf78yK8jPTPDy6\\\\\\\/m0WkmbxqjSQVymRjZW4PidQsGIOYkNrAt41lDVhSs5U5FHR1o\\\\\\\/Xw6WbSPJxpCU+bNg5SuK9ZXCcQC\\\\\\\/9vBna8fufZ9KOBnVZBo0Xr70pyZwla2eiZLHwsvHfQkZ1GGnSq3bTFdUYciJ5i5BTC8Bp6PcasJ5OaJc+++6Gt6ewgma3EPI\\\\\\\/4Uasv8JdTaXo4sdopY\\\\\\\/wOj4MdFsrjuDIhTZfSkhsoaAK3vBROLrAyRaPySuZkEfkWBwT+WwfelS5vMabglswei3TxfDB2f5pYPp6cXhJKdrECUXTCj7zHjldhmouclu07aEQcLI2SwHIrFj6PzhhIP3qAPRX+iU0FMxTaFG5h6HHpyhtqga7+WicvtEvOjIcsfb0KYmBsPWWqOX+GAr9A8bNsz3uqHvTBcX5RRBP+gnG8OOHdX25gXgmExIGJPuaXqQkd9boM3SrIthGhn4ZRjq+CKq3sf\\\\\\\/MTwvvLLEPukvAP6p1Zec3dl9nbCqYL+OwdOo+QDatlOh5ec+82aWj6muByD1qgiJQbdjk5MlAbKJ8jPziqadlOR0MANri86nsg2o8HuAKu9aKPp5GGCvTClULWVkLhn\\\\\\\/vQWdvJqhcALYKCLLi8Ti9jPqubNL5sYPCuhEneyjnjV2lHW3c67QkKCFQbjdYki9t1JBXdcsT1TwRArTLqIHJwlciqD17EwY6dMq5rqdP42A24tfE+7YzhbMFQnRseKvX0Tf5+w\\\\\\\/jyh0q7OnKDd4m5vaxOI3+KECLpuDHv6R+vpSQonEYPKvlMtZcfjFAqv6EO20KorulFB8GdGmWnuZi7Pc3zwYPpZY6DK9ryn2RPZbdVXepZhFF+3QwsMfzq6lFKCvIHsqVOIYMCkkh6xbclaNWvCxNGJzG+4YpIZlg9bKPjgOJ8XFnY5ULwiSMjTZODfbOIBZJ3Jgttm6RYClna5\\\\\\\/ILKqTRFd4HisDgQD1ukU68Klu4NadX9rilkhN9Fj+QymW\\\\\\\/A7gz7HI5k9akFL9oYEoFZz9WtByEULmhXtAEhoUBsXT42KED\\\\\\\/QHYGMhLetih6Npnu5Rf8JMsSWZMP0R1OtLuf31vokwEB4lrgAKXI082ecVm5mLozpCUAiZv8vf+HBi6h1KZOIjS6sPnEYNijm++TPuA11OM8SLWmr8LuNchDDYwb4NCTo2NVtl8L7oBxqdBLTj\\\\\\\/J7Q2dJIgxW07HcabHVAmBMafClo9zBLsew6DUUdDk3sfsRCRAFoJInLBLtZGeAFxPAc68yDSma4qLa1ZxAHk\\\\\\\/ElDLaOAkkpCSmaCLiKhDNzT1AzVF37MLVpIsQNlzrLfGOjRoPG0FGyUaaDLVfLZw88FqHWIiDHUhIO8opjRW5sdkHYoO0eAbRJZjvv1A7hP2GN8mEi7pQMsJA\\\\\\\/Y1WwJM1O34UiAK+3hZ8NMIzvJKWiHJtLIvy8HXi8ywI3bKqC1GyZO4rCReD\\\\\\\/pV151SQAEj8uhlGl7P2MipX6kzg77x9v+n+gOyVVjCOC3\\\\\\\/l0jTlVeSnagd8EJgAX4TihrdlyAvnyQvrlcQfu+Su9yJcTh5Bg+rhEJsZW\\\\\\\/72iLjR4P9eQ3XFYIb1Yeg\\\\\\\/bjzOlJysKRIULDqV5Y\\\\\\\/X+9Br3vLG2\\\\\\\/loJfXrwK4\\\\\\\/q9iwFO2NZ8WE0MMbAwCxKJaPUr5kljP2PEebyo2G5yOJHU5G0KtSwUOPUHvaNrxKZyzXfSSYEYMABs0fqA0Li0bWMbBQrbNTNu3PC7NRs8V8x0RdVvcfEB33UfQ+gSNGQyVQV8THC6yvT9YWuV1v+KcJfWm2YMxpcP+F+nX8EjSR4UrIQ+474YutL+0E2kiVo+i9Iv\\\\\\\/iwtJin5Gd+bJSzXjL1rF4m8vBfOSInyx6MKv4YGHj8iGszPKrHAUe6ZHW9Df5hx4TAkwJNuY51HfIr\\\\\\\/4FNYeM0AmoxQunj7XNADzkRZKmJGp+sYhAE7jWWZ+h2WKEo\\\\\\\/62kZh7sGy8YECBsMxb7RUTr9S6oYb+npolrWLXyeV81DkxLKGhUGKPruNB+eTFpjSmS4CXDpstuW0p6Kd6iDdpMRo310Rw1nCaQa441lr2ZJwK1hDtU4\\\\\\\/D0rNcp73JVmQjEm+xJSmU9GqKuKiP5Eur72NyEDYYilbVFTQJfzHkW6ZJstEmv3wPhnUzVJKfHA55\\\\\\\/sWUQZKyrBqIuy5B4UdiXr9VMGwv3vXPcXXPM29iyrahX5oHP\\\\\\\/YAiOESK4s5+9vzCnt2p3VfeeSsxrrplq1hFE7dOKq2yHIfjhWIzBK\\\\\\\/Stg+uHiSTMPXIsLtKZST9GDOIWbhxdjE0fbCBnvT3x7AZYHmpOtVFksGoR98wZ5Hz00Q1ML3zSfrz4b5JC3CizuhybY6FQyE7xYqJWu9iHaMFE4vsFQxA6pN3eb77ZMpiC5tfQk4AylraFbWpH3kVThboBCEmM3HN8+VOFGFMn1MTv9QrczsTOjor6lP47epsDt8SdzNJR2bwbzVw8e7QLfbrE2UiTdLwn+c4L20Wbl5\\\\\\\/Um0aCy528tRfEGrvu+zYzYqUIZG+FhzjOTDSvuDdwBoMwWAUvsbpEgiRSrA1FFZm7SrQOn5\\\\\\\/ZdISAqbTPJxlUrdEhqUh3nzJZeX7pdZ+flLu934yJImd5oWsvkoZAXhPuH4ngqh3f+XPhjBoZ7kIS5rQZCSlQtAWheB3OB7Cdj6IkwZ7H1HjnRWUF+fO3CXYjbxPvQooQuW0+RfF\\\\\\\/vZ6bpMNfUCzpDmZLDHVLg6Ze\\\\\\\/MzzSnw17dH3gA7B9ODSQr04Ts3lGvub8sDxfpd+kdaFNs3XI5ZWYjBKMTVIKXpMAWaSNaXdhi9FkZm6cxMBIaEBcxK0KKOfjoVZDqs1i4KmIPKBmkaV5BtHkAnSs3oTnq1FIMwntQkj93CJ10QepTPjR3PAXZMX61IMhS3MQ2fu\\\\\\\/Xyx43IleElJumX0WKQqnBW4CxK4XUj\\\\\\\/945pmrq5SYly31LCOBxgU4yJMyjWl0BGab6LaBE44GYEKRXrgD8kTskJ5u2uvw5yhVC0XmQwyQoO3+NeJcuQB4B46X2kyyzQbyc5waKwVfRntaeEWTUc5TYcHLm8vvMBPZDxjL7UtJ+28bhKXoZusrwutTmfCQTTo1fWI5srcNqcxobI7a6V8bZVsWXZ0Ipv4o5hsUVz0ulVa1a\\\\\\\/\\\\\\\/J9H51pKmXuEWYma1h6YFOBNhMcAVhAE5Q8p2rkiNODQynVG6Pe4386wz1Ss79iFrMS9UEv2wtgbBSLlOfOOJEFdqF3m9H\\\\\\\/MgHOXWoRwtmrdocmWB9+8lEwN2Nvyv2PwtCsCPjGTR9ZyFBZRHNCKKQ\\\\\\\/5VPWAgoBgxkrhe9DsZDmtImw+F0k9+NByW1y1J7K1YFYdnUVWZlGaRf0ysp7tDOPNFAnODPmJlmasRGSGiwmuz2y5SPOdmUx8QLO\\\\\\\/\\\\\\\/ogMPdCKv+8uJvSYnJTgTRBmtzZ2offP8QF3WBp+FvgaRlgTV3Sla35luIUiUNl+Id9E3H6ljvzB7jHSFkq4TAPf9TQMLZOcw50icX8VoE2GsWiax6hkUO1VKjhLRMLNRRhycWg\\\\\\\/UR0pIydTlvYpSbpv2+jc6pUhxcAxN2+kqYQbAcVNNyD4MVMcgFnPKqW0O9ZPWHrl\\\\\\\/LdfO5PTmO6h\\\\\\\/CpIv4Vp+0n\\\\\\\/bY7wqFs+hrmeXroKm4W\\\\\\\/kLwn0w2PHC0K6cbmuXID79cOkxVdvVELTXAoegOj4\\\\\\\/IF+dzuAuTZCrEO6g0vNPWsVeyarFdJyG4nj7cQxqzo+L4yYgnkduRvlSkxPi1IheJ9sAeGqCm0UaMQ7kDMymx5G3eW88eDlvwj12SQAkuG7qCD7eR\\\\\\\/RSK4SbVtM5i+j08oHFQK4ugFejEcKKOmud5AyFHlLTce0DVmUj6sqJXSb8ipGjsTqOBKzZ4c+Yhn9u5uF\\\\\\\/mcZ\\\\\\\/cIT8iaHIGX7HzH4VS1IUsL\\\\\\\/6pLHb\\\\\\\/9JeeK3rqpKbh0EUbZRQg8b8JtRQjdrQeeUqmyvs1W6BUXDPn2vqHcbqnYm3CkSV7CmfegOWrjN6RqWwSyMZkeRtkR\\\\\\\/hszGabRlVYmmo3JYv\\\\\\\/BlA9qnae93WGl6UtIiYydecQY4M0QEQWKqlDNzVQnBnDEI5o8D+SsPJOYQJ5X4VQMNoR1svvGp4l5Ga4yGh4mT9impzx\\\\\\\/4KoVPnMP8\\\\\\\/gyvzK052cg0EeaM5HSKzRSGXbp84w36OFRKddB4CBqGJ\\\\\\\/DNGPGFEubhyYbf6SkYQHzD25S8xEmMJnw1yo54oRljL4TXL834tBiLpLSJKxH8hXSFkNMvHq1hApslmaU3qwJJsDBl8XK+Mldh2hrTp0eAZPlHHKjXMnMw83jeDA7qwpcraCDjY3La6rEpTUj6J6+5Hh5v9wKY3ufzYl7y9MX7vPFRcXZfc92Rh\\\\\\\/STVqkKpAcG\\\\\\\/3BWlueFsPZnuFZqDK\\\\\\\/dUzSTJdEKE3aAxfDJ6FptsteM2cvIvPHMPKUn3AR1rAExvk50HVN2iRL5zjM93Fub5Kxt+1qt7eCQucQTO7ZJrEFfHnieBQ8KE8bcIcMZpZXakM1etUQH0Hx8lUYkQSx+dbcWg614zq3Gyt2PVcwIxceXBjKV6om87YGCFjOcDL7qeCxlxWAYyohmdooLjp2G+Neqhcv79S6LPAEhiaIXyXUE8p7U09Hq3uPAJ3rGN6flKT5QHNqXTs7cXmuaOFtF1dGSVXmo\\\\\\\/OarF0wxEAa9T6UyqPt3oDzEVLpVWPW0Qcqbzo+ihjRngPOiIISSMsoztHO\\\\\\\/xY8p4IVacORHTgX26N8BjZSpwETE3lAtJWSpcROPbOWhFZ6ySN9swFDktI\\\\\\\/hbwh2aUjO2ScfGX6oBj0Pa0FAcAxTwcYKBWCYpM\\\\\\\/DuF65nPGoP2T5R9pOdgqK4OnYh3++6EQXw1FZYomlHeyWcI+njpGs0VPaDuXv0BtGhXB0aitOVEtaPTYpTV56pydZewfQsQ9AsOzugL0kggpEXVQLSolywduhDi+VTXfPDIrrS5hckJSI5uhUlXVufMEFLboH+VLi5\\\\\\\/HZUz+AfkNZP1f1xEPvvEwD+ph+bFaDTSmXJlkkp3IPhsHwXLHwI9BZHAz51+aDbZ0o4xy6uaZuQdwt5\\\\\\\/lsgGpm7KOigYjTslQBSc+AoqUfxD6XxsKdzobHGay\\\\\\\/nFsm71AFbG8hopHbSqs88zunp5fUkkaQqp76oZwFCaSrVpB3XxBFicB\\\\\\\/OmL92ikuRkYDR+tc+AjLSOun2GoHiccAP2hYKxJhaf64LJX\\\\\\\/WjwLDPhctsPQzQoaataNPQoC3\\\\\\\/PeyOMJAGblQ6lEarGItDEwxgTtA8iEEi\\\\\\\/0+gJheMj8beD1sf77vGDFRVIomqTLZEjuIk2z3BFbofMIXchBAujWBmWu64oUkdK7e82S7\\\\\\\/Z8Bd3GuxkALnGM5wbFo72NK9gRGxkXr1RZJFEoF3TaT11tLpDhrnsaIIf99q7fPcf954s6w7U3atc9p\\\\\\\/BdqHfeQERq2TePrDRaHTmeZfp4I\\\\\\\/j4TfVAtCyRnLJ7oVUuyKAH3LSVws9F1+919Vz6NLwgui\\\\\\\/vIvOCXWKES\\\\\\\/OB0f84d5hXkRIO14jDydDP+3WV4qJ4UE5NNfjtfkwXVa+gAKeH5UubW5VAqxvRpUJHVPDI9B8Pp9jQPOSEh+90\\\\\\\/xwzWBo9uZdep1KGa7\\\\\\\/nO+rCH763FgnE\\\\\\\/0z3oA9ZMOxsF3AB4vUn5727hIqkKExpxrYeAik12ljAVColb0spzy6oO6lWgfvjVNOCosTuX6ysBrPtg1+Vhe+yDtXYhpspvrui3yOQdRMAwTcmPYGEZi9uRo7eFKCIL4hYkGIeDXGOrntDUgG7Z7zK9W7iaGTEu1Y9Whw4Sg8dCGvCd6Mi8gratMpaR25ouEX\\\\\\\/7BJyg6HLzxKq4UVg7esJ5TQQbaOrELTsBP7afBh4xxjtC0KDunJh1mDN0YWtRcBviQc8ZFtI0nu0XIWL7cuwMn3C+4NOCWTAHJmaYSy2ZZLmkAw7\\\\\\\/o\\\\\\\/bNDIDmswfbmP9Pm2Vwwy7LQXsKIZ6P0M7MAEIKaRL7uOwSrP03NwwaeHwG028FpSi7m8IY5dzc5Q8LVaqmXOjGw2KExeBTmASmM\\\\\\\/q6XvtzN91V3AFsMKhCmHF5PC7blNRHFNvtpI8DD+kv5XKYjhehATSC4r9JVuVy\\\\\\\/OgsZbdSj6wq7j9Ft\\\\\\\/kIz5726dLHZT+XJA\\\\\\\/iiKpma3cZYV73v5mTj3sBs18HIqnIHYqjeH1vUJNuYTU6Pi1gQ54uNPvk02tZOUeRlgKw95mwJJP0ZGjc41yse65TZ\\\\\\\/N98JtVCaIs0XL8ISA1LHhu2aEu9TxL9jhS1TF4UQQS\\\\\\\/KuNLnct5eqoRo8F4LKM14yea7k6OUFvsw+yNPCMZ10wMqN9yvg58AAj4sScusAeVrWcCSsxZC+8yIvJYsSimFWUlsDrewF1LO1JCe7ioNC0zYsbfFhGGN3O+GEq93zqfTPzdUSqXP5A5D3XAZXavZgZYBPV8FJjc2x0qEMmE+UnLbtt25H+cGYxn8STq1V1Y2sOBTB7N3WENK5doUP+56EAUC+39yP\\\\\\\/8rX4MXBrm\\\\\\\/8azq8xdU9aIQucWCJbsv+VAgknQeDD\\\\\\\/\\\\\\\/DrSfcuUBObLOr2VilaLAUqnCZECz9rxgHIO8Y9VfQysopwq2OYDzw3QXcDFhzCzroFyhI4chN8qt6pnwiogUCpPI2oQB\\\\\\\/czoloFP3Wb64RY45QOHhv8Em3wW2x1GEYk3PNKOP2SZ1dxcuaNGkqOz1MFElLmPj24d+WNq7NBKTWJa7eAJL9Kt4mC1cWVT3BJ0tY1TbLRusiHkI4dw58GMYe6BH4OHpxNrMkY0HWJdXuzU16vB3nx2pAP5RH4ZXjsosnUxJ6cA4RCCjC05cNvAKCVHxTVKpaUAQ03q6N3ZCkEiZ4nT\\\\\\\/uI0D\\\\\\\/T1X8yo8Z5ijOAS4pQHs2Am0KOkpjRnS5s5Ki9wxN5yVGpShaxV\\\\\\\/NmXB\\\\\\\/2ZTazZD6lHyU9AluOc\\\\\\\/bIWf2Pit0h4DwWcMDDOtkoefeBTxEYeC09\\\\\\\/lsDLp+bbrAOqz9KY4OnL3l4vwP24jpZRmKU9f4usUuRYpTahZWoisBGLOZtZ9JVAl8VjxlHmg8H+k6LOKpMu4JxXNax4Ia3cIafZo7FfVM0598TBO5A4i6ugW0asIS0z0LnUD9R2mPofLlHHN8T2HHH4MzgmiQQfX3IvWdjJ6\\\\\\\/dI3T8OQz6ujtHKyMIqFxm5beUuR7VSzYV3qIcBd5Urm5JotIsU9c95iqyBOBRiUVREzOLLLuUCQWe9kO3xanfMD3tOREHW9zfLtcV3+Cq+L8x5R+dXCFmRKN936uDPL61dRLh41iEbw8+kF+Yj8J1lcMumoNUw15skrYfTUKI9VBuZqmL1Hm1AIbUtOEWtX64+Kv3MZAd5\\\\\\\/TCc2qfGZ+pe1ym+uVVnq1jrIJ+Uv26hz8YfLBmvlPYFyiBMrqB2ZQ7G85\\\\\\\/HWVeTTUjFmEch7rc0a0xViNhwCW4BqgqeM7nZsZoDZJ5KdU95XUTM+yO6JFWpxx0\\\\\\\/iR8o5J6RV1UsRp86urVeOVp9ojmFOKYsv8mwyWJCDAz7\\\\\\\/08v0hWi51GqvgmNXs\\\\\\\/KsqkXH1gNr9CJosfEgJ7Ww2ySotsjI9KVnQh+wQXgpuS6FDheRMofLnT2qawv990fnSTKqHO3bExFgPSQ1RFzRYmK5DAloPuXnhOolC6c9HPdowtJ1U2eaC4nLA\\\\\\\/vUnztFWhYpKgd\\\\\\\/WpBCFTiRvkJqCWu1sANV3S4dL1h4+fnuWo1Q9REqL4nx4ZCgHA42As7yawSl\\\\\\\/MyRxrX7MfkwMJDpdXJALiN9SAfp8M89nOx9s8rXmXboqyHZssGNVhckoKRXIOJ7uosWJYHpWHgjnJOyI9Oc+Q610yQ2WUMFt3D0T+\\\\\\\/yanhH5eSYvkVbLLpLKDGr7HacWTHhR00K4Fo0hZNfEKb3iQ\\\\\\\/pLSFdwOr7YaEZWjyhKabpeUgEuF29GVXTGqh25wNG7xGS6AYiIklrNe0tZVvxGjEbMwSWsPHmOdGo7atvyaajq4QD3mRWHs7zAYoOCxNief9EMN4QRyaGVmHhEI\\\\\\\/YmyU15bcmainmaT6SpiSyHHsLw5OO86QMCf3vRBuQqK2A1sOPcVLaBxse3pIudObV7Xtr4+z2+zY5uoEphyHzI72tCOijdwD3OrJgGutG9QhZVuly7zs6\\\\\\\/LroIuqWwAYBgBardNvYzl2cryi4YD6mnfwY4qy2cI+kYqf38ogf6g\\\\\\\/2vJCMW\\\\\\\/55v8CGtebHfNcnVqggPBCX3D\\\\\\\/vn24pHVty94WZHn51PxqTsXQLgGhh4MGfNFTmvwYO9VsIFsUE+1uRPBJlurWpaTtMcA0buQnq1pOf6iUgyrrZzYgXclWMCSkG1JS05SuFtZzR5eJCj2aSxiXrssf7SMewlLT2CJ+nR3mU0AM5QmkX4WjvuJOFTN1xoGu8AfWafsIsG3eWMy9e5rqNHI1vHLzKsOQ6yGbJ6uLmWwMO93B\\\\\\\/iMyQCY6j0R4Dmq5KUHBxArlUBNNFkwb4azg7sDF8IvARYlMYqpdH5kbYFknUZgVxkBNXPHqM9iviOFGaUJrHt\\\\\\\/7XnBK5MuZgcfc5RLR2srewujblwhFJuKIbcw\\\\\\\/VNoeRX4ftSTwgrS6mk00mrIujhf90jLAWrcTmFHB290pQ\\\\\\\/RvF26LtOROknk2PE+E9MurkpzU6KpupgLEGCXa1wwXkrpja6yVLUufKSH+wu12urzaMhDKjXRZdsElvaHh\\\\\\\/qjbtKbv\\\\\\\/gO+8t4LDZOkz2QPuhqA08E1m9enN6cK6VuQAeMF5Y7hYHj\\\\\\\/uW7XdPvWQPu5ggxA\\\\\\\/x10mNJiCynihm44qvWZBW7iFXZzQ8Ha9hAORez7x9JgN\\\\\\\/TXIVobHDlKugUBp\\\\\\\/alqLykY\\\\\\\/0umntuggMyRGtU4sB7ATt61yUdp\\\\\\\/oj3OBJHjJgzAF1abKV7xnbv0sCqJjMG\\\\\\\/L9rSNTeAiSWcGSQroAOZoEqlxXJOU3oEED6qFHWBpou+aiQz225fU+jblTW8vv4IzaefP49TV07TFZCfKR9qE5kdzqAMIuHeEJZ1XsK81WMFIpQwu8qrJGUa8zJIUEtKKaYgC5WRnED9n8acbhFG5g\\\\\\\/GO6wizL0SFJwTTuz45vBt+LLDLbaGr+UrcxJohTGMfR2+mSHoeGZ6qnZYkKjxErbtcND9ogHsUWeqbempfNG+hhmJ7w2\\\\\\\/E91gDNo9ihnMaIvSSbMcsUxdgKV1FAR9cZcZZHn8xk7mI6KB8RjeUz1vhnR\\\\\\\/mgpF2kFSOIxKEr\\\\\\\/tg8jYSqRv0MAPM3cryunjGAethfTW0YPLe+fb1xl\\\\\\\/k6SAGs83sDVSENVAXEIkPZuAdGCyxb+dd4gj5Oxsk3tWm0NMIeedH\\\\\\\/hrMBpoxywV\\\\\\\/BdsAGezZK+\\\\\\\/Gaw74JcGVhloK6bDi35IonXyfa\\\\\\\/Mznnh7JFZAF46DTXUGuiRJHLmKlqhBjQhCFtT6Byv3B9Qi0EiIl0p87G3p8ceCngyPKHzH0hkz5EQRLi+yUPKEI4b5\\\\\\\/O3Iwn6n5US8ZhFnfk+ZIh5+8CW3GQlTOAijy2djreJx9Ja9WaOTWppM6xyGrbefFZzWFmqyiGV5XJwOWgi\\\\\\\/aD4lkL1fjxDQpsS4xg3zKeOzIpqPwRTMAE1Xs+Sj2fi+jduFxDD\\\\\\\/263uOKZF+1JH3oP9pTP5m0S9t1JE3HHnPoGJRrtgspsm9dy23NTyQychcQWQBCKH\\\\\\\/nOniPsLLL4m6DOC+rnl6e28LgsnsqcoLdI3DPNUV6x8m4urMti73AFzsGyYrRGhlHxWiHSkn5fmMIR+2oyh+ZGY0zfhPJ\\\\\\\/YTUHPnUMlV+VM47OMIsEy5OhvJ7PZC8JNjZ42rvnuqPDaX\\\\\\\/9ka\\\\\\\/6b44NBwMEEi4048fb+iBLMkNgEXo+iRBwD21RaMOqRyU4a4l9vHXEqvcxQHdHoaVAdUM3Tyy65lAmkuX09ppvsXFK3XRoSSpN\\\\\\\/kUPKQH96OoU5U3GRBKRA5vC\\\\\\\/nDaKCvpY5ZvO35E6Ay5+8MrXxqVCXq683pc204CHtN6K1FB+DgANmFWZaHnwXFy8cV3symCgkf9OAz9TlLpTtHXj7L2Y3c649jw49e+xc\\\\\\\/jXcxhF7kQ1jj5xuOTXE8BCjdn7N1beJSIr93+\\\\\\\/v+gu97q5zROxC98hrwIFSc6j3Yum6wesHgIS16oHlZfNouWQLviaCXtwosu6PftGNb9C4YhtPuT6DItJEztXXVJ2eZjQsNtsHxDQ2TYWpaWlhwuH4nYhQWeiD3EwOyWlCLEmaDvfJowiejw0OEEzYZoYppHJ9X6QASYjoSCPlTK5AVADysOrC6lOtJbs1zfOYBdQzqTZ7\\\\\\\/xvkzVRz7WES+O\\\\\\\/TDS0m6HJo\\\\\\\/jAwhx0rLEmzKZHDTD63lVRErgmTy7LXuUDXV5WdxVDBIakAmZBgtK3wg4\\\\\\\/kzcv8Nhz3MbMyNeTl04U0dX7JwwfyJitN743YqFwYZ0iMX15dl1hlLUxP+mG\\\\\\\/xxOsrHbFS\\\\\\\/o9Rsnqsv+rgdm2xAof7vDriU1xxnjEWXi3biZywWpVm4ucQL8ZhwS4j4J8HHlYEV9sxQVQRzVyogC3uzyAEEbyqNZTmWwU9kSlMIHgs\\\\\\\/EgV7JLEifL+YdecNMfI8+rNEqwvOQOqh9R9qZanbFNC8u28E9A6TQU\\\\\\\/z18d0HSUJf+maDotqRAFEGr6690oW1HsqtewMEUpwgxyE1JWzfwZs2a5RL\\\\\\\/Kh08En+3\\\\\\\/D+OvQfNgYs7tJTZ1tL150MAYLNdSlTscajeLyURq68dEcD1y5Xkhc2xG6C1g84dRghuLoJbU+3QnkKwaRUg8L8j9oTMZ1Gk6TOWBgTZtKwqJGpdWwNEox0b0+p8SEIIp\\\\\\\/UNLhYDGVv0\\\\\\\/v5B7D3IDCK8b0FTCJyp6d34F4bjxGefA3wNYTksa0aGWzHajdz+IbjHU78fPEyeKuBdHCxroZGOuB0ZI8KOzlVrhpCR32qNdiPhFAmDJ8ieGKnWGMKUtdenhcDuF6H\\\\\\\/Zj0J0BiCzCGu+v2BKumWF3T6sX28cT6TGU9ykueYPbmSqFKRsuVenSTgnJCexu59d\\\\\\\/D8s1czmJsDucLa1Oxtbxu8rJYhUz43XF4jhJVbLM9dGSv3RNZ\\\\\\\/LUeyNmFntXtKIrk4wgLHOZESNtFkAu6Rqf\\\\\\\/ISwD2J5VOHtYwOg2UIECBragWLdLIisaVwfBNJTFuK9\\\\\\\/3IIvpT4E+nzCP2fxyR5mnkb3jJ\\\\\\\/Ux8f2B9QonYHou6XUXLwPF1ZaDKPBrcFKKsTJXqsLRnfZ1EqAPOFRg3hVdIwJHJgDxOWTkHrAJZIG65ZjfWnvPmm7Fe\\\\\\\/EuY6GmJ1s1mV5uiLKfRTIV8h+yQeVMtUVBOusqiOGxOzxxZnxhwIKweyMWkbVFiT07Dcj3PFeRH3wiZVx9jF1xhAgo226c6SjZfeBmWqrWNfnWfxASYsUXv21AQrTDg68xxlV74+nSMLOZbokmtT8JPmf0239kvOwDV\\\\\\\/Xibv+jCcC6TFAyoaBw0E5wcoUNPrJKu9FbyfBPmOUjbjRKtLWwGrEcsr\\\\\\\/\\\\\\\/9GyqA\\\\\\\/MyIaLdnWR6via6lUGYULfo7XtsYECxWGjjY+8qtsjzpTW0V\\\\\\\/irjKPwapPPpruQgTtF6c8TVbwJi\\\\\\\/kWBlQhVOY\\\\\\\/798Lyj4SHyqugHV+tFS5w697jaCsPOM4trygPqIFfe1s2VCsCViP9NybRiZUI9KmmyaioGDyZ8UaTm7bSE9k4NL65gnxnD0zpNQzIviZvcHYXUrHP6LXl+nZ0K94SKY8tqEecCtf\\\\\\\/VEwWfzdqZDffXQ77BIj9j1YfJ6u+9veII0gj4qYLQpe655GFH4c+Ja3NzUdQg3EuKJEjPniWoROhxMgBaUOaisdyVkOh8vdew6qNlDKG3NOBRRs2ROGPrIT0G06eS8LfEIf9w\\\\\\\/tvNUIyLW\\\\\\\/p3WP5Zr3EICWLastUGwArf76sGQQGDg\\\\\\\/Ed4NodALxgAeIDD1Dqyn6IH5UslCMIasF3mMBeLNlK56Ib8qmcQKMfD6DC1LVU2JJI2Et0Iucy96trJ+CTHQaK\\\\\\\/66fb88mawYvLE6sr+scLHaMHsPEywx+BhvnBZklD8ER0iJ9GT6hbDPYB9FUmL0+lhXN2ePhneT42ryEhvqBY50h\\\\\\\/Xf6bhKflEj87UyT3veKZsE9CU\\\\\\\/Dr7BSg2Pz89jDgB74bcfRPdhnb3aayvr53Mli9jHlH3op3ASDcDedm+5n7XIWYGA9MZMwJcS+05hqpWsu7s9HNZwuI+pbT7V+Qi2SMCQxAlt9\\\\\\\/Sys+JvZdGbWno9XFCwM9\\\\\\\/ySXgQskJZvcX+96ZJWbKLyth6Xl4Hrs1RSXe2kKSnb2CwJlkrsVNtqwyoSGP\\\\\\\/ADr7ZYS843udUBzWqZ0D2TPMQNYPxgwlg\\\\\\\/kMqO3naN4JtIMr6toSaseAOtALCa4WzQFFYog\\\\\\\/mDj1\\\\\\\/Fg9CvDK\\\\\\\/6VKq4LpbZLThFMryvxhdgMiDYWAByY+iR0h50fTlMgRoszNFUfJTUAb3OQN+LYrVDfy1mo532TYo7\\\\\\\/oCsxoJl566nItv3ft\\\\\\\/yylUtlRrQGQ3wRRI94xbAKSl20aL2L7jsdkKgnolgGx7ZubtLG9sdsKGMgG8PUm5PvfD4rgxr7kY0EfLyhXCF2SWQB1jxsa8Fyt3PZtQareslhN3KIL9YAUof2u706VGtWNAO82HrZ0Z8cZ6BvT3TEtl8536KEVQyhPlzVh6g66MFXMyM3zmthQi1hPOaqrwFvLXtz5ZB7fzpQk0Y++sIpHQHcdYJBurzLj77xlA9seWTsJQoJPUbPLg2Wv0UVV7\\\\\\\/Q7ckrO3GfE6yirB5agV4CXjEKe4gLXUnOBm9VgReQZmgLCZZ8RdaxmIIP8XRdGy+foiInxTaerhXzXi9idrni\\\\\\\/MLAQLtNDs8gddbfXjkzaCiuElecK7bIb+fuXVwvjz0pBuzTfA0pM9pswFyjVyNqgccDOdHj99h8kSoZPwCyfXwMxxJkGpKo4Fk\\\\\\\/gDkJ6G3wIXr59QhSivX\\\\\\\/z6LJvGgZ+JUtN74xqZdypOkPMxc034ve4Lj2pBy3QSAkztbgqBWsdi4kcxmWo3uvaaMC+Eh9TkARE0YcWUl98l3kn34qX0saKcg1aOsoHzVE68B1C9OGlsy2B70C7GyIgtmjbDJfWzaGHsdh4CWRoi9uNrCHsQ7+yzkrHAVbN8gNxQvXu5jms9O1csTaJRNZ6L7\\\\\\\/cY8xJOW8RCI9Sg4tvnBdjOMeT4IMfGi++n3CYSEgBLaPxrNGdaoBrhbQnB9WCWn6LMwCUj721UguUE8W+ENT2qitk+DmLE7JVazEQUREzwFbQWIK9w0Z2XkHTbHgz4plkvkNOFYS7PZRdeqTYhuan92EFIBJpjWOJlYXATsQPMY915wJAz86m1SzjKcECRw63i0mZIWsfjQ9OGTnWDw9imWxputB4RnYcfskkHiaxP8kLpg1hBG1Gv0nUxjL29wwmtaWkTrBx9DbZiR4XYoi9zPm3or4jQH7P8HqrgceVY8r189jgg0RqNmWY3atrYor958Q7RxzlZO0CHJ7S6ZCR8pVIPF0UdfakoYiX\\\\\\\/bhE0m\\\\\\\/Ihcua23hg16iaM3fX65nekSZQSyH9pStWOqVgvH8xrSNaZhS7KUD5XpqRqagqAoY95PyQ3K2GQn1w34\\\\\\\/NqNDphEQYv4COkRjYHiLC\\\\\\\/UD4YYmfN28uNJfQbP7K8nAFN5IlASjE6++NiEGZQtwWAn\\\\\\\/tvPekw6c1hXxxXmZXbbsnClV2Zgw1oNKX\\\\\\\/3bqrVvUNufussQabFzmF9Fp7K1CgTsWQr4An2egM72\\\\\\\/4yLLHiUYqyAXSmwJQRR3aIY+6c29wIqKISqWumChP1aw8dVLYv92\\\\\\\/xd7XpHFsL7b2TcmGJZEdKcD2ByaYghcRh9Oi\\\\\\\/rZlGB+4aBHr7gtZmVdYzJ9oL7k5kcH2YB8CCEHySNXUSN0hHTESvJknRPpS1eaeFrutbRWMklE0FcyTnWRHV3xlGkgruEciuxG3bFgYSzK3\\\\\\\/TF3uKgLe22Bs\\\\\\\/j1SZcOhpn0ASuW1gbr0zQa5L6IjPflZFXFfuhnDtDoTbYNHZUn7g5IDCNb5SbCea4AaAVUuxc+CJAzBvsoqyhOyANsJxqBcbL\\\\\\\/BoJ91W08yr68WSapc66ZYq7dNBWq2VcrNFpfms1AaXIr8woYZRBE9b\\\\\\\/pcpMINziNoEecIiGCVtdDtzZRV6ooFR8hwoesEZS8Mb6nMCVIHe8JZSu+Yae0\\\\\\\/+4ziyX6WTTk+KrE1t6317mKPnU2yRduOe9GeVcAp4peM5OKRxqRqO5l+txL9M3qxqMI4z9a+IuL8fbnBk8h+a7cCnR7fW8kAWSYYFcpWm7B6wwT3GDzzTfI\\\\\\\/IIgDHJ3hqt\\\\\\\/znfky8Up\\\\\\\/lnrWZ02sidcsw9J76Ure0XjCNd7UexNyhe6DkgsHFj5MNZnsllZukDtScYYzfG5yIaZDtBKc6NwP\\\\\\\/U4RFoi6dByEfmNh6bFhZg\\\\\\\/s8equJzGotoA5kYLWDIYt7WBzzfD43I3RhYU8syWOEw1yWHBPozEqGW\\\\\\\/SCPewBUWKjtqnvg8nvSKIpu1Ac+OItdaO3X0HHomhL5E\\\\\\\/3ZgBhN5Y1duOm7BvD0CCAUwscw2bjgTHVdj2ae5LOii2YRSPLIL5ot1V60VxppOLUzXvSl+x9lRdnCwfn+osaioupDoLdOyZ8n\\\\\\\/XaPvVgWMI8MEUDWBEj+iE\\\\\\\/GDNnkSzFZg2yGXhbC25CH2Is1NFbPIV8MdwFGD4HnCEpAnI7fGjsE2wuarXFF8HRsKfuS37aokjvnJVNVu0SE+0p7TGVss2ZifL0TjM1OKUNVgZqnf1cxL5\\\\\\\/KcsoJPg2PfwbsX3GAx8yKuHXcKeZPB6xK7EYWND5YmhGwFCrlYx4GBUaeamZ666sGGf2FOdp6MbzkYx1JPwIDsmMiOKC3v3+SRc7C5SgaXQOPZSHt65hqkXAjSn3UJITfDn+X2EOFUMvlhgQkuIunjZa6\\\\\\\/y7qpVeOS8Y51umUxhNnV5eOZ8kMSTha0mzculAh1uLe9wjnXnwHBtkk5OfGn\\\\\\\/+nBlO0iqQYVpry0G5eh5OUWIg8TMZTZSXrJXg9iLec6olCOPJc3vZdz10PFxLpfdvZKNmCGkxyx5UPaJ3543OUfLQWBU45wrka8m910E7ZvnJ+cOlv6WImSX8nVZ\\\\\\\/nu6btcd4yrMGdjlOeB2INWLeu9Cq4\\\\\\\/NE\\\\\\\/246qICrLdNYmXNbS+mQHnqUrh2BYdP\\\\\\\/DfCHlBESxoxG35gMxdD56F3p81B6bHhceoRZZ4GGGKq2tbahMgxLr\\\\\\\/G+lQP1o6vQ65cE3Ze9EvV+8XrNqtU01oOfD3dYhsYHlc8riE6AgiuiVmawftaUYonXnvVqX6a0\\\\\\\/OZO+NWqdjzC6jzvDuHSXTHfDaGnXBpDUi0ETJ51YDsYCAWxbLqbHhSUCTVAGSud+NSwbySKmskfSuN2EOrlifeWas++csEUyqGaGCGcwISPTBaATgQDZlFnuHDWof6f7YLEwg+ThzZQv+VWk1joicXo3HrDA\\\\\\\/t9CL0mphHrBruQoiIyiiytYFG9MQkvLZhRI\\\\\\\/ePj3g19QD\\\\\\\/9gkb17d4v+3iQHmqKMmAeXrJbe2XGMKYKUjS8fBJiOObA\\\\\\\/x7FZ2STvt3\\\\\\\/3jfiLMheeKQpoUhli3aXL4cOCX7a0qc5pDEyN5HzdHNpHdRSaZ1xjaxoqoSf5t5VSUeaGBdAwAywcxFzpai0dmUnJz\\\\\\\/7NXxzV1lVAI\\\\\\\/4rc+2tPULwnIWq+PYHR1RjXJc04ZTNv7e8YTV\\\\\\\/mMDVPSsFVE5v9RNl6PgMyOXx5wXMzUWXPEMsgdjs+v9AxQVktQKEH7m8CndTl8esjoWO5HO\\\\\\\/vjaPzSDHiKkF4wU2thU19ivLaL+2FwX\\\\\\\/fsIYh+PxX9\\\\\\\/PaECwIoTPMmjqqwBOx8AHTGndK15qLWCZvD1iQnJSZPUGtzWTcFlSMLXi+njaWDSnraD2HL3d0lsR42mab8+ElPznzzwQ8LO+2Y0tuvHGvFAwaI7baiW+kJMUNT\\\\\\\/alOnRSGUNg8nk5g3L5aWibS59\\\\\\\/5Pz0rKDYLs9Ezpi5gYh4LzoZ4yq4NZA2JOAVz2lvK6YNGpf7NdLnY78mpHoQmmVeL1q61BLYeJ3WfKGGDVl0bL0arvtEFMwayJIkijDy3qXXb1oFPkjWnmb\\\\\\\/mdJg9Yto\\\\\\\/QR5hFmtQJYtfs9+hL3yIkUITZia\\\\\\\/s4K9GWMMgwNOsyrbxe3O+LIhy\\\\\\\/sTilU4pyK8+CxK3sUwekOZXMJipG4+gdtra0g1P6kERNq0eedomezrM7khrrBxLc13x26s7nZYqbXO5+YvqoN\\\\\\\/NxXG+ucZBIBakqR7VEyvx6xSQ+A4EtK6bwDvMMcdXOKLZiRA+Am3Y6BwvtWonu8tbmRFhoVczX179pPYt9S2VyIn0IbsnSf7e3sAzNbt5zjykmjyEiBAaGr4MkZgXX8IF+fXDRw8z2\\\\\\\/F+QV4QdUqRkKu\\\\\\\/Gxxw89YYqeZ1GcfHhX645m513VVQ+v91noeSmMSlPo\\\\\\\/rzNoSWE+1PNVQKggyTfHzLn4WV\\\\\\\/7awCV1XsVxNDU7B00JmZyGFWU7mYMckypFAOe3haoid3lic+mrA9ruddr3\\\\\\\/uXeD0dtvQs5USyO7\\\\\\\/3bnKzPduKv5BpCpANeOjJfs7t0LOOF4rkK8d\\\\\\\/7PPtE0tMYEaJKy68a7QgoU8y9I2vHzLJ6NnZbZiaJZg3AWHI2OedE6vhL9oOtuRJg\\\\\\\/cb0AzytdqLOSXXB7OY8WK9fB4Szj3vm85IV3OhTF+LIuygbS1ycI6bQMWkaXxEENZG5Yay300lg+FSJxKzS1DVU97ZLTu3RJvPaHhbVprT5we47xrFUvoJnOieGV4SiK6RU7ghfx552YQu\\\\\\\/3HVeBihg0FHhs4cuCKWssbd3EySjnL\\\\\\\/bSjH2+dZBCsnr7bNgaDJv8DJC3FpYMheIOER2M5vHl4wIB61rY+WgwKN9xOptNDOl9fJLB5\\\\\\\/W\\\\\\\/uZ20mmhFIICYeXD9OsQmGoNfyIJZojlz5sQtIPZfnhe91w+nbTAhnJG6vdvHuNiC6kobVuxtwXkLZB\\\\\\\/m6OjmFBKqtQmU9UUe5A4jRhIv4uIpyytAguqYWhS5D+zmCNWl4qCkNGQzUGmGKjWBu0ulUwDM7QHtUz5VVQNKeAP81opLaBSaKapO\\\\\\\/lcBn8JnOAdvr8MucCgNCaxPM7hJCfDp1ayd06S15u+UaH3eDYN0F0vStjWc4F65Sztlf0v2oCaG\\\\\\\/zNEGN\\\\\\\/WvSTfSo+BcjyZXaOVN3CC9scMAmlyqFs4zXavuLq4QcZzzxQJ8EKU9hlSHbs28eVBsRGgfOa4TC7JSH0qMiF31LtSPLQo2WeyclGDDA6BOrtRA4smc1xi7rsm+Lgkv6fhgbQ\\\\\\\/bXQxGLGvTeHSIbZs2GcO+MaeLv9OxKtH+mDpfb8RoSAgc79eL07l6UTpKDHUGIHygN6GqNLf4L1YH+QUSNRABrO21jO4AAjDZ8IcqCRRKh2wpUrg4acLOt\\\\\\\/eDnKZ7zy5zYkEIz0rLLkApIrUexexDh4rkOJeotpRyzTXHxoDMU7J0YPkmPof4LVRxszyzw3y97anku5WsnfC0wvB0httcUR+TYH5lRhD3Dj3FK8GtFJu9nynUgC9ylPc5KnrXbzmf7chPjdKy1zeEQ+soPvjmDp0RDTEKSC+0TbZmpGU5fRlN9PUWEL+Pss3B5C3MFPbg0pYII2LJrQgecqZQqlWQSVPS2C1y+BDyodx7EVaPIzmmk8yZMJ6GxB2RJAf\\\\\\\/4C5Jupz6iEZ+pQir9vjmhw5j3Yugd95A\\\\\\\/GGYyTZ2+TJ6IDY\\\\\\\/HRvuhmGCBTstktRJkYCyEwKb7YPJrCUAwIAwClXpea7dING2+GPm+d2XcXgx0weuRxIgpxdyATNgz8XVycUWT5JTWpsGj+eXIoPVHJoGkMUEP2iz5YhGPKAadmMai92tVpMi8bXzt+O9PmtzD+kM57B0niTx9QpHVuqRc5p8Kwm1kqrwOoEfClAYc1RtNaKVEz1rbPik4u3SneCS\\\\\\\/NFzaWmHWhNbt1mk15q8vnrIT90\\\\\\\/TeVLge8a5p1PBoQevSRfwNEnS8qzcn+97wtao4vRnk0qpFly15BjAvWsW7GdMhW9l+qKF2oYSXNdHHZZqmuCKYOZi7IwGXsvg7evTjQ4QVtgIHxwwPogCzDA6vn9mwA6T7NEYnqQ0X\\\\\\\/C+B5vtZJQd5K758astxz3CX5wNSxOBkGDP35ZgCHKt+eGptxBEaZ7zlnukZFOAJSIju5otU6s0vGSmNDezS2QCxI3YFyyVPd6dTSnLrbt+QkN6XJbO+kdL+PrUBkmwh4rLyMajk6+Qfyr5ni6uA6nSrhJoZ0s7cvAuser3g\\\\\\\/W6Gv0CiYeXZ05qDfKdqUAUk\\\\\\\/wIR+HwHtPUw8USGM1Z4d86vjlHgebiqWnpsyrSLAEZz1gA1gNDipddcfygw2RMcwbTxL+lFjka2ls+j6adx\\\\\\\/nR58a8w7a\\\\\\\/Ejn\\\\\\\/6xVd9r53FLvyJeSgtoSUKUnpDF+sZBkj5tDg54rI04ZypKmKvLgHxYYEXLfjpmPJqMLF5t9Fb\\\\\\\/mXjOIgTGw3Z6raGzos+xtpInhwA\\\\\\\/Op7FN05l7jjfpTXGnT8pMg8xkxSpp4qXFqVpT9abETjl\\\\\\\/fvjjJx+EZo11gUvTgi6qsbzvVTsjn\\\\\\\/iJedcl+rFqBrNuq4GAwSOSqCakhtdVvge6sRbMxWy3DzDdjAc2gzgiJzjcrd\\\\\\\/Hp\\\\\\\/6s7UDSTNSxAFYH4iATTqABsKbjEc+9UjgOOVBCvgbFoOGJcIpZI4KAGocIO0hzfCXWXiXrJolrUWc1R5D95j4rZE3QHa7f6s14cCUzEEYgR2+oVG8QrogSIdC\\\\\\\/py9uW679qYGAD+hfp4GEnvxsOTNFTK0G5qL1sH4eCpj\\\\\\\/juTnU1JewcQCLnGwU8q0rYrlUQ215dldnk0QajlOg4vAHFvC1lAxTFvgOAKpr64eQ\\\\\\\/mfKt+BYSKOHxPA9812iPhbCM+FXxnzn52fHt2+GTfwjCB8Zth65g8Im9UHuvVHOYpYSIEsxqOjMYQzdFsn0unt5lDnKcScTP\\\\\\\/zRsEaCufypNRXPng1l\\\\\\\/L92YDGZNH+nxfcuiWz\\\\\\\/4veoD+5hhedYn4qOwUKMDn4uriWLsto83kiDzxW0IptZbp3qW2Pv5C1jJpJBX6lqbj\\\\\\\/Alof+WkMVwuPm2k\\\\\\\/2WEuT9E9LUXIG6R8l0zD9wmyddNkrySCvWglNHe19krqFQRLyWhBJRBBEj+16H3nQ9rKkhH+BMc5Dybj1Kutk+m\\\\\\\/K\\\\\\\/NWWg9TUShZvVzR\\\\\\\/eYHKohEPXgqZSaORUtCxHa7AXeBush8pR5qMRYBkrgviWBQYYF\\\\\\\/1gqlxSzAhbo0AgBuZzvPnWK8iHg3pieYksU6b\\\\\\\/HdxYr5CwIz36Wq9so5o\\\\\\\/FH4cK\\\\\\\/IcH32mASqMZlSgsnSLRc0rkTSghJHDO3Ib7DYLK+fbhHnXch9JX8EVT5AlEIzO1mh+HzcR1uOnYk1UQKxqhhTd1CJVZxFmQajgWNiXEMwt2bsynx1Xo5jDbRwGgG4Md0L7NVetVPl5NwVtfiISuonr17p8QWLNhd5QiUOLXFoKZLJRz0H9MYLwF0lHk4mDTCp4bZBda3Oam9QtIhfeukdflp7pxgrzqL+kMUxr2Yx+27MznvqjOK+NEqK0BtPRfGVZx69qmPFg7KeBzLU2nRrxPtMk3j7SkcqtEqF\\\\\\\/+o4oifiQJ7tFvdwR\\\\\\\/axZ9O6HN6TBqp+ZN+mr5v8W5xnlL8ly+re3IRBwH2biRBTXd3l3PU8alpr8Choae9hErjh9yHAp\\\\\\\/uBsuhgAje1d7xvHodQe5zbM9WiqZnUKxjC+0qNCEYptkKPhCPmxMCyShc4UtC2Qsh0BXii93c1S3KvXbE4qtef5GM6YOflPTdUt++SBovztLMVIBjxuxdgVH2HxburdlT1FPxcwlmUWWqZf+VHGRrimpPMDkzrphfMWT9zx0DA6z3bCp2knb+aDR2oyTrK9saXznqQ5foU42+TNs9Flx\\\\\\\/wLos9tAPvmv0FxNd\\\\\\\/C9Hz3gYTIIhaTrID7vCzpN7306nOA7\\\\\\\/sYqbNtIGSKlIf4ofjN36Kfh7VS23nJk9LIQ7pDq5QvuBYDz2FfMIp\\\\\\\/0nYwG4dmJBX4gm1rfTsf+9VaQFGsCVD4xYEjrkHoKgA79M1b55tUeDsgrDtrwL5LL99VWOJsNuLOkvXJkyQ1B4lpNLgcHSSV8tBzWXkDONBLHUuxfBa9p1loVvvpjMuMQXk0dZ9YH0aS\\\\\\\/k+Lu8GRSYrrkCvEVoLkboLFd7AjenJbzBFilrmnRVSW8YlZ2BawpcBEVN4C0o\\\\\\\/2iHGMCEJVdQvg+j9gAXIni1vLXeGGbx1kL+yOp4PazzwvdwO3arLV8Epu5Pd2Ap64Ba73XFktHjKMBkXtqBA\\\\\\\/md6Sw7sv3\\\\\\\/oA0kcjSEh71\\\\\\\/7h2osN\\\\\\\/yeoFA0kCaW5k2P3dcIyXuxU+y2rHlRZcnganSVU6NbcF60acF0cA3JqFexK6Kyx6DeC+LmTfowVstH\\\\\\\/P\\\\\\\/TKoIj+9WnXH2awmcDWPn6z63DJHGkgy9Ufav2o+Yeum5LldnmnJWJO6F\\\\\\\/Dz1Y52LiMMlz1jST1SVM+iVuIgHTNMtkVxjKw1JebWBou\\\\\\\/Blws9uS3xzNN3AoDfNBBxiejYKQZ8GuUeraDl6fK4gaBO1Uz82+jmsDFx9lSrLXScydf4\\\\\\\/EoRbM0oPDBPbVQ9GD7V14wyYxGTMXPil69VBJzT+Uum9ButMpN68fa6ETcuKe+nxzVI4lMLvzHCT9XDhXdl1\\\\\\\/61o6lqL6DV+3j6XfBlPODPL\\\\\\\/qhcZMU7HsoKh605qXSW9EJhn8VHEDxk\\\\\\\/6JuSJLGnHkliNPcm9J6g5kjxrn5x4WvldQua1S9I5NaM7tpHbqnY0JhYXeFzKlnjrD2\\\\\\\/94xOcogM9gIbEoK01siHFDcWybW6bBIoVUG4j3U00H+t1xO0zTxIP9YzC7cuVgg\\\\\\\/z7bDDLSSDHJqHvrnjkzmhjK0KMZOfR6P4EDCDXqnk6FSjn8EHRId+aOAtb+zDbEaxQXunYlA4iTqfpXvEkzv8jieHNxydcja+k9MCWIFu7TGSCoa04ruvCBgXryl3j\\\\\\\/N5IsQMwbtr6WqzbLxvEHKJSxpq6ZQb4GU3osIxyv+TYCDJAg0Dp+2D2EkKTLhiZy3M\\\\\\\/xmVFICfxDYeyYHzzMua0qqGz5sp9SXTkdKNXnKYPvz0h202Pn8bnYmjAdUzo3prIe23C+pFGZyQ7ZZUe6ePmXb3yx1S8XiH+2TWMCXnHDmweERraV5AaNsqL\\\\\\\/DzJGnWwfYWS7pEsvS6nVmBPxDA9S7W4ggcHLmaEK1aGvWSBOHapQy7dMeYcZrSugRwD0jNgt3DWyYXr4cC7UKYHJpY\\\\\\\/C9SttzfNQ8Ra\\\\\\\/ghZswPXQNUpKlkySatDpoNGZX97bmi9cvubZe7YC\\\\\\\/JQ6xin6NAMJGa5JevY5EIIGcwmrDlUeY4jNIR0l62QDe+5WCuIO+gW5cuKd+CDlNZHCd2iCdd+yL34xh8ndXVRO9Z1HRfRS+YCF3HzrBW59N6GNzoRUTeY\\\\\\\/QNNUDeWHo7c3U9Ro1hz7NGBFnLujSUPtmw+tfsfI6+I7EPKx9l2OT1hOFoHdvqzM3WwF42anMxQYwwSsNF9RWR\\\\\\\/PAUjjfp7zhM4FovvMzHMK9t7ZVkyBrmTn\\\\\\\/WgLcdw0j45UWGO8Dw5jdXb+F47i7WSnFXitCek5fnDikCDA9KoEpBfpDNiPdheBO7u8P0z8zeU8qQ7jZh\\\\\\\/\\\\\\\/v6yjtjBSrdjBUGBtwlf+qij1UR2YmU5uQhVjnWRe42ezD1YJXoW3hmVkYHFUzjGwo97qml2Iyvc+sHqFi6E7huWdiOSUen5m+dDBMHmF8MR1ciI0PaGNOKVkPKlPLLmkKRqTAsNhD5eGYUFPogyD8MEgvQPLp0JTAmVsgHqJPewUQXmGU5f\\\\\\\/okKdDvp2TlFRe4q51Xg0ood6i8GXoLnavy5xqpGmxrotZ0u6AT2XneHM7e3\\\\\\\/z4yESGxUJzGyCWdFWdrSl8YXDt8K6OsJvHOgZxsp\\\\\\\/bmEUP5gO0uCx7vpsIJHfu\\\\\\\/Lr6GhFBxNpNlkPwQO6MGmiXGeO1GLbMFrZsyhVBd1xr\\\\\\\/Ckjps8D9eyI79zF6icZeSysb0Nzq7y+NcjXvFrYJeuZitsYlILPNanGZ2sf\\\\\\\/n0bUrDbsl1+9HLSDI976QOv0ETQc3uANY8H\\\\\\\/4fXBmPFwCZi3tY5pySjUGtDYLKZMxjRIykwVdqqnNhtulCD+Wtk1PsKHyvqCOvsTiaUTVgEeCXzgZjaZ51+kFBgb38p3wr18I1Id7ilak1TmmXdopox+lV4BaV4Pgcb8gohbMZcsHQ44amH1hC3Ih1x7r+yPgMcs+6H0TSLpOobZzjGWgI6iidxUk17p5ulR4VVbwURe5ylvJact2lgbmrFc+DKOJLVIh5xWv3ZVevkP5iDaht84r2G7ewUUzYW2dHZ+su4APpwFBfS+hyjqY4XhrytgrSHsle8p6iLGEcM9ti3hXUxhRr0a+Re9jdlzOnN50+1o\\\\\\\/etrR7G1pfwPbs2V6w4+\\\\\\\/xB6qfVtjlxFgCBENLDp5VJQoAlRZCN0edW1wnFqFjr2wvC8TDNUM\\\\\\\/X1cuiiOsVViFWN6oAxwahwVIYBu8Zkq4GpmBlnZ3SBkHh8gkrLo4e5fArmY\\\\\\\/zyi\\\\\\\/1RDcaVDlV2Tdl4jJZUh7kGhUWvW33kxck6VSCWBIVksSehSGo2J4AnebKanvzQk4JGP0Y8\\\\\\\/AJKpDV5mhQPOlYGVBGSsa+TkZQVUPrIS3s+7\\\\\\\/uBYS6PWA5xVz5RLy4hXvw99pHBZOJXwkS1igdeCOUyEg1Q2yaxywufNdT983R8L5EXTVCU+s\\\\\\\/1oN4n2MuMj1QBbAaUY2Y17L0mTyxVw2s47vWjno1QZK1LzcAqhAz7oh8LWc4yoTMn3wD55mhgRgDeZSHEb8Uary2yx8mPYpd\\\\\\\/xap1nGcnTkRs8MDmVJGQXBQWENbCgG4OwIdxciStlIeosrU+tGuyoWmj94PUl\\\\\\\/nePdr1Mdt02\\\\\\\/y\\\\\\\/g4smNvDBNwNuAYJY9HEe8gSmYdFNk0aDZnqzGXBc24kwshJzWVlhw5a3PNJM9JrwsNy5Tge759IscTVvcSlqKb8tRaiyMjDrfXYnyejWGAISUk7LKPWMXCIMxr7wn9k7yD2y8XbTEbi5br82\\\\\\\/8v24ykM+p36fhn7oZUTsyf+I4cmvo7BD8MNOM9Fck70Kz+V1ERBeMxIAPj8ARQ0SqOsZCp0Y1KFod2ZWTmFEF9kjMmo6pETVebPtIdk6njPnj0+IG6UmZS76o4uTGJBKV+WoHzODGIXEX2\\\\\\\/sQlU6xV3ZgWhtgOneHbTZdUDIlUeh+Qza5LeWstuzR8pt\\\\\\\/qBP4mMeXABMYjrUQ00G4riUCWGNnWLsuexlKzsLI4yw17TOwvhTrcj4DkwLvribQJaYel7nQPZAI6EpU5AgCNCkfShb10zyU0x1Pr77DWw0nCBq03kn38yaAXs+mu+r\\\\\\\/t5Gc2ds8QYGhlV3K8xxa2sftG6VISrVktHdsu59V6lHgJqwCvwveg1mhsn\\\\\\\/3OuMP5cqBY2Rluu3pQXFET5PCdNK0N8TqSapZNzAeOgCqI1AGhKubFJilDXNElDJ7d4b4ZzLIvN2v7C0lntoEmaVRnnfQKHKw6ZTgXIuWvunSHxqYbra+7OG43AIYz+MvHj+\\\\\\\/UKZPygZPC5qHbTthkd7l7KocMRsIiOVg9zvLt69ZTaIPfBqFyKhsmAllJekgsL\\\\\\\/ZvXwpTly9MUmE68XX7fdVcg2rQYf5imKev7ydqJMp2hmp55zKLuGizJxysvR0RRfZFpbzHsVUv8JiikI87KyyBAIaqLvzXLOLT+Vai3\\\\\\\/tHjFM5zwkI4s\\\\\\\/Sfrwzhpxj\\\\\\\/dYojwOFXcE7GbjIlM6q\\\\\\\/3Ub9qSwy9oMDiZjqT3SDm99fw63YVgDjowVp45l8FtbISuiCR7C0oDCUP99OqlcASET1ATruCHN2BxwoZRnFZHfp1GKOsyj\\\\\\\/TznKAChNXGcWAFaO1IydQEUN\\\\\\\/80+0qHdcdhD9YY4FJp71NXFZ25rOOuKGColA7kqIhOqB74dmzO6gEKsQxOvj5y6lyAq+SH4zXGm1ZAUi30QvM2MoSU8qOhEkToowTl9yZajRMfPJtv5BIzPWiIH1UQbhTuy36VsGkTAQOlEEq1\\\\\\\/gGrKx84zR2j3XY2pR+qJUC4bU22RelbxGX6VUAkXdiAk34p0p6CE4Jd3FdXq1X0GWvcAPw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022V86cJX\\\\\\\/gUzFhCLRQ6Uxe6Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dc98b6aa11ad79c83700564aff8a6ccccb236ef1a68defd419e2b9dccc8cf381\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wEld5hzsfu6YMY4OZ5Ko+p6G+1zO+ovgWKIBgh\\\\\\\/KPDCKKmFHsErdehtkMFSMbpk57+pnNpk415Kgu9QSvvr+k3Lj7wm\\\\\\\/AUWdzGZSP2N4fE8WfDn3sS17Xcf2F+rYbPVVtouCH9UMpY23r94dETKFRkXAGJPQIG3vsiEE2GLwVW6sprkZuhk2N0QFg7kV6IHdixsint9qL33w5HQ2vlTP22pTnQkwjieO+Bab2zlrv\\\\\\\/TzfSiPSvoNraBBF2XbEwQz\\\\\\\/SX7aezgrJDgRZdzEi3jPk7NOoto0je\\\\\\\/HUJQNvAHdOpyuUMIcMJ8NK99PCZEDtFuL1UA1TNQjtE5NVx8+paaAAde0M8e\\\\\\\/u1FK70grvuRVvdGDqZolGg\\\\\\\/9Wc8xj5sFYm5AO\\\\\\\/4\\\\\\\/WElGlWaeSNBE39NBscu3vd7kEs7pWBwLR3VQUq1D1vsu40SOoaRTlz3s7tUojY8v9uXD8OX\\\\\\\/Zf6YQ3\\\\\\\/BuDEWh4n1CF6UCAanV0NTwA8NGBC9+F7RQo94EYgspckCB4rWfqRyOZClCt636YaF8U0Mn3\\\\\\\/B9jqdUpVtgU15CxcGKNeedNXLDGjIe0YfmDEGt6O3RF4hEfzPgjf387AtBfsyffV7r+tPN0n1X3yPd7XzHJbi848jjXrAyI20zKM8UlYTE06s64GeJlg+Ed+ySLtzhLtlH7LRmvzxKkth4kvRl5aEnRfXP0ilOU\\\\\\\/bsBFslwyRlNIRly+9P7jYJW9vYS8qZNFkUjk2MSO+HZH9VZ1M4KAW0aA1SLH\\\\\\\/avrruV1O2jRJkJ+XcaltRw37s3Qc9q3gj\\\\\\\/jh3IjeJWNl7eQ4ipFeEnaSbUDsDYLBvpng\\\\\\\/pXbYv4xfrxK3pAvw9Fo9Cjlk26VhNVid8jEoqU8HmyBAAPltd9aB+2GBcQbiEHSTVcSDgu4XcMn6oB8gF3XCS8jL2gyt2Ek47KxZe3pmUxblYAmuL3iDeHgFaI\\\\\\\/ZrizwkkgtmXA8NNj2erNQOH6fLfWXDnuA76bHK3BPdBAwIwBlQCqSjdd3HiuloX0\\\\\\\/x5XfBcRWGgdVlZOyD\\\\\\\/dkpfwsZtmLsyOUwMACD0k3yumwvYsYr\\\\\\\/v1X5B+FLIYiMAQ5ZyDKaN9lWJSqckrzx6Ve6HlcL3l7183VtDorNx1QPUKiNdJJAWG19Q037HjMdIoSdpwu+I1k84DvDflRaCu3i\\\\\\\/7Ww2x23sxeJ\\\\\\\/Zn8vkVVgVwl5+1ecpdTLv9vZgf9\\\\\\\/9Y0wNHl3a6HrkJajIl8j\\\\\\\/OCgY\\\\\\\/RviXIR87xw3T56Ys99rJ3PHKXkKTUe5JRSkjBVdTQkZ7DxP\\\\\\\/2+52Cst03XEjAqWMIzsshdQQmkD2vBgIa0VLheyFxPPIdCwUNvCfuzQHOdUVWJ8u4rEDfAkCl14qpHNaqwAiX7D0NQeCCX8DKzyBA2UvNprgJV3meSW1m7VvI0WC6dWXQfTmgmwwXz9xRMuRzeuZhvAsgjICFCh7zqPag5AoqNe2UHmarJBU+rliD8zpqj1M3d94Q+DkOFAaEmiwyAIg5ELzQR\\\\\\\/S4V3fa6EZgV5WSwZKjPGfatHMWn96RZbV12wvLptqTq3PRMx9M0AxQCi2iQzS17VHwxOwxeYDdmK0SV\\\\\\\/JD0c2UY30AnV1QTL\\\\\\\/IMPZfTxvYa2xv8\\\\\\\/ycRkdrePjpnTxsYPS\\\\\\\/lrA61pOdAwfJIM5SATFRq7kFAz\\\\\\\/U5CCIC6PWPNBHaaqQhLVMnXLAoM2BhYyD8Q38R+GIaMZ05mLv9XUdv96lGFsYy0YEmRUvNLUmuQ4dHIpjwVKC1I4M2wtvZA2cJc2R+yF47G3tjCOVGb2nMpTS4AaOPF\\\\\\\/W4+e+6EArVncbzymb37YKveYU\\\\\\\/Q9xmQ7vKVrqj+a7ZbXTqoOh\\\\\\\/nGBZySp1C0V6beUxz+Sz5wN28kA01OBAcFwDUpgxIDuflicyeEX1dPX2jke4nMw2JxHQnZbHG4Wu49nfN7f0t0IG2Rlc6AxUMSD0NI+scKnsF\\\\\\\/FmCL4ceqhHj2eYvtoAqQc95C0KQrx2bBQ\\\\\\\/bKeGDr5WKasx8UMH7yWLkzz\\\\\\\/w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022IR69spVjcshigw5\\\\\\\/iSC6Ww==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002203d2accd89938200066b434c043ffcae668d1d8658e5b56565bf5d780589d96d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489473 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1455" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-emtnGScsrp2JVsVy4vJ3x6QSplurWJAovH31EuAlaPg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1455" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-307hS65Rp8P3dXXREK1l135dM-5vJGFW5X4i7D9j-cQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1439" title="Cold Shock as a Screen for Genes Involved in Cold Acclimatization in Neurospora crassa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1461" title="Comparative Transcriptomics Among Four White Pine Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489473" id="node489473" data-pisa="ggg;8/5/1455" data-pisa-master="ggg;g3.117.300465" data-apath="/ggg/8/5/1455.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast Ordered Sampling of DNA Sequence Variants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3296-5811" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anthony J. Greenberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1455-1460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300465 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489473" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anthony J. Greenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Bayesic Research, Ithaca, NY 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BJ.%2BGreenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenberg%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnthony%2BJ.%2BGreenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3296-5811" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anthony J. Greenberg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tonyg@bayesicresearch.org" class="" data-icon-position="" data-hide-link-title="0">tonyg@bayesicresearch.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/5/1455" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489473" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1455.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489473" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1455.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489473" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1455.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489473" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/5/1455.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489473" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489473/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Explosive growth in the amount of genomic data is matched by increasing power of consumer-grade computers. Even applications that require powerful servers can be quickly tested on desktop or laptop machines if we can generate representative samples from large data sets. I describe a fast and memory-efficient implementation of an on-line sampling method developed for tape drives 30 years ago. Focusing on genotype files, I test the performance of this technique on modern solid-state and spinning hard drives, and show that it performs well compared to a simple sampling scheme. I illustrate its utility by developing a method to quickly estimate genome-wide patterns of linkage disequilibrium (LD) decay with distance. I provide open-source software that samples loci from several variant format files, a separate program that performs LD decay estimates, and a C++ library that lets developers incorporate these methods into their own projects.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/nucleotide-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-nucleotide-polymorphism" rel="nofollow">nucleotide polymorphism</a></li><li class="kwd"><a href="/keyword/random-sampling" class="hw-term hw-article-keyword hw-article-keyword-random-sampling" rel="nofollow">random sampling</a></li><li class="kwd"><a href="/keyword/statistical-genetics" class="hw-term hw-article-keyword hw-article-keyword-statistical-genetics" rel="nofollow">statistical genetics</a></li><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">genomics</a></li><li class="kwd"><a href="/keyword/c" class="hw-term hw-article-keyword hw-article-keyword-c" rel="nofollow">C++</a></li></ul><p id="p-3">Growth in the amount of genomic data available is matched by increasing power and storage space of consumer-grade computers. Using such low-cost systems to perform genomic analyses can speed development cycles and empower users operating under economic constraints. The range of such analyses can be extended with light-weight software tools that carefully manage system resources.</p><p id="p-4">Collections of single-nucleotide polymorphisms (SNPs) and copy number variants (CNVs) genotyped in groups of individuals are fundamental to numerous applications. These data sets are stored in a variety of formats (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">The International HapMap Consortium 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Purcell <em>et al.</em> 2007</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Danecek <em>et al.</em> 2011</a>) and often contain millions of variants genotyped in thousands of individuals. It is often desirable to create random subsets of such large polymorphism tables. For example, relatively small samples can be used to quickly test software pipelines. In addition, when using genome-wide SNPs to predict phenotypes of individuals using genome selection methods, it is often important to learn the minimal marker set that achieves good accuracy (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Spindel <em>et al.</em> 2015</a>). Finally, repeated creation of data subsets is a variant of the jack-knife procedure (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Efron 1979</a>) and can be used to construct empirical distributions of genome-wide statistics.</p><p id="p-5">To be useful for a wide range of applications, any sampling scheme must meet several criteria. The subset generated must be in the same order as in the original data set. Variants must be sampled without replacement, each locus has to be picked with the same probability, and the size of the resulting data set must always reflect the value required by the user. The time required to sample variants must grow at most linearly with the number of polymorphisms in the sub-sample. Furthermore, because the original data set may be very large, the time required to pick loci must be as insensitive as possible to the overall size of the data set. In addition, since my aim is to empower researchers with limited access to powerful hardware, the implementation should minimize the use of system resources, particularly avoiding reading large files into memory. The general problem of ordered on-line sampling of records from files was solved 30 years ago (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Vitter 1984</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">1987</a>). Unfortunately, this work is relatively unknown with limited application in computer system management and sampling of business data streams. No genetics papers appear to reference Vitterâs articles and I was unable to find any software that uses this approach for biological data.</p><p id="p-6">I implemented a version of Vitterâs algorithm (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Vitter 1987</a>) that samples loci from a variety of variant file formats while minimizing system resource use. I examine the methodâs performance compared to a simple sampling scheme and provide an example application to estimate genome-wide patterns of linkage disequilibrium. I also provide a library that allows developers to incorporate these methods into their software. All source code, data, and analysis methods are openly available.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Sampling scheme</h3><p id="p-7">The original motivation for the ordered on-line sampling I am employing here was for random selection of records from tape drives. The fundamental constraint was that reading was slow compared to directional searching, and rewinding was undesirable. Thus, sampling of records as they were arranged on the tape was the most efficient solution. More modern incarnations of this problem include sampling of data streams, such as stock tickers or social media posts. Additionally, modern solid state drives support direct access to specific addresses within a file. This makes them analogous to tape drives in that reading of records dominates seeking within a file.</p><p id="p-8">Vitterâs main insight was to derive a scheme that samples the number of records to skip given how many remain to be picked and the number left in the file (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Vitter 1984</a>, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">1987</a>). There are additional speed-ups available if one is willing to store the index values in an array, but I opted to conserve memory instead and save the sampled records to the output file right away. Preliminary tests suggested that file I/O time dominates random number generation even on a machine with a solid state drive (SSD, results not shown), so the increase in sampling speed would not be noticeable.</p><p id="p-9">Other than the above deviation, I implemented Vitterâs method D as described in the Appendix A, algorithm A2 in <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Vitter (1987)</a>. The implementation uses a hardware random number generator (RNG) (<a href="https://software.intel.com/en-us/articles/the-drng-library-and-manual">https://software.intel.com/en-us/articles/the-drng-library-and-manual</a>). If not supported, the software substitutes the 64-bit Mersenne Twister (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Matsumoto and Nishimura 1998</a>) seeded with the processorâs time stamp counter. The decision is made automatically at run time and does not involve user input.</p></div><div id="sec-3" class="subsection"><h3>Included software</h3><p id="p-10">This report describes three pieces of software: a C++ library libsampFiles and two stand-alone programs: sampleSNPs produces ordered samples from variant files and sampleLD uses locus samples to calculate distributions of linkage disequilibrium statistics. All software is released under the BSD three-part license. The whole set of programs can be trivially compiled using the included Makefile, with no external dependencies required. Compilation was tested on Mac OS High Sierra and FreeBSD (version 11.1-RELEASE) with llvm/clang v9.0, on RedHat Linux using the GNU compiler collection (gcc version 4), and on Ubuntu Linux with gcc version 4 and 5. The easiest way to obtain and install the latest version of the package is to clone the GitHub repository (<a href="https://github.com/tonymugen/sampleSNPs/">https://github.com/tonymugen/sampleSNPs/</a>).</p><div id="sec-4" class="subsection"><h4>C++ class library:</h4><p id="p-11">The libsampFiles library allows users to easily include in their own software support for sampling loci from most commonly used file formats (.tped and .bed from plink (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Purcell <em>et al.</em> 2007</a>), VCF (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Danecek <em>et al.</em> 2011</a>), and HapMap (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">The International HapMap Consortium 2003</a>)), as well as a generic text and binary file. Reading and writing in these formats is supported, as well as limited manipulation (see the reference manual for details). Format conversion is not supported at present. Random number generators and population indexing facilities are also available. The library is constructed using hierarchical classes and is built with extensibility in mind. File manipulations are implemented to reduce random-access memory (RAM) use, without unduly reducing execution speed. The trade-offs were tested on a laptop with a solid state drive (SSD) and 16 gigabytes of RAM. Performance may differ on other system types.</p><p id="p-12">In addition to the software, a directory with example SNP files is provided in the distribution for testing purposes. The project GitHub page (<a href="https://github.com/tonymugen/sampleSNPs/">https://github.com/tonymugen/sampleSNPs/</a>) provides a mechanism for users to report problems. Detailed library interface documentation is available at <a href="https://tonymugen.github.io/sampleSNPs/">https://tonymugen.github.io/sampleSNPs/</a>.</p></div><div id="sec-5" class="subsection"><h4>Sampling variants:</h4><p id="p-13">I used the libsampFiles library to write a stand-alone program, sampleSNPs, that subsamples variant files. All formats mentioned above are supported. The program runs via command line using standard Unix-style flags to pass execution parameters. The README file included with the project and available on the GitHub documentation page has detailed instructions. Sampled SNPs are saved into a file in the same format as the original. Auxiliary files, if present (<em>e.g.</em>, .fam and .bim for .bed), are modified or copied as appropriate.</p></div><div id="sec-6" class="subsection"><h4>Linkage disequilibrium among sampled loci:</h4><p id="p-14">As an example of an application of locus sampling, I implemented a stand-alone program that estimates genome-wide LD decay with between-locus distance. A full accounting of this relationship would require the calculation of linkage disequilibrium statistics for all <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-1.gif"/></noscript></span></span> pairs of loci, where <em>n</em> is the number of genotyped variants. This task quickly becomes unmanageable as the number of genotypes in the data set grows. One solution, implemented in plink (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Purcell <em>et al.</em> 2007</a>), is to calculate LD only among loci falling within a neighborhood window on a chromosome. A complementary approach, implemented here, is to sample <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-2.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-3.gif"/></noscript></span></span> is the desired number of sampled pairs) loci using Vitterâs method and calculate LD between consecutive pairs. Justification for this approach is provided in Appendix A. Once a pair of loci is picked, sampleLD calculates two linkage disequilibrium statistics: <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-5.gif"/></noscript></span></span> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Lewontin 1964</a>). Missing data are removed (only individuals successfully genotyped at both loci are considered). If there are not enough genotypes to produce a meaningful result, â-9â is reported. If a file with a population index is provided, the program will calculate LD statistics within each population and report them separately.</p><p id="p-15">Unlike sampleSNPs, sampleLD currently only supports plink .bed files as input. The auxiliary .bim and .fam files are also required. A detailed description of input file requirements, command line flags, and output format are in the README file included with the project and on the documentation page.</p></div></div><div id="sec-7" class="subsection"><h3>Test data</h3><p id="p-16">Execution timing was performed with SNP files extracted from the <em>Drosophila</em> genome nexus (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Lack <em>et al.</em> 2016</a>). I used the Zambia and France populations from that data set. LD measurements were performed on cultivated rice (<em>Oryza sativa</em>) genotypes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">McCouch <em>et al.</em> 2016</a>). I extracted a random sample of 100 <em>indica</em> (IND) and 100 tropical <em>japonica</em> accessions, and filtered out loci with minor allele counts less than two. I estimated the smoothed relationships between LD and distance, with their confidence intervals, using the ggplot2 R package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Wickham 2009</a>).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-17">File S1, Timing of Sampling Schemes, is an archive of the directory that contains the R and C++ code, as well as data files, necessary to reproduce the algorithm timing analyses presented in this article. Compilation and running instructions are included. File S2, LD Analyses, is an archive of the directory that contains the R code and data files necessary to reproduce the linkage disequilibrium results presented in this article. File S3, Software Source Code, is an archive of the directory that contains the source code of software described in this paper. Compilation and testing instructions are included. An up-to-date version can be found on GitHub (<a href="https://github.com/tonymugen/sampleSNPs/">https://github.com/tonymugen/sampleSNPs/</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g35968987">https://doi.org/10.25387/g35968987</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results And Discussion</h2><p id="p-18">I required ordered random samples from SNP files for my own work, mainly to quickly test software during development. For the samples to be representative of the whole data set, I needed records to be sampled uniformly without replacement and to be in the same order as in the original file. After an extensive search I was unable to find existing software that performs this task. The widely-used command-line tool plink (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Purcell <em>et al.</em> 2007</a>) does have a function (accessible via theâthin âflagâ) that extracts a random sample of SNPs while preserving order. However, the program simply examines each locus and includes it with the specified probability. Thus, the resulting sample varies in size (Supplemental Figure S1) from run to run.</p><div id="sec-10" class="subsection"><h3>Ordered sampling of loci</h3><p id="p-19">Given that no other software appears to be available, I set out to implement a light-weight solution that quickly generates ordered samples without replacement even from very large data sets. The simplest idea is to examine every variant record in turn and decide, given the current number of loci picked, the number remaining in the input file, and the total to be sampled, whether to pick the current one (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Fan <em>et al.</em> 1962</a>). The selected records are read into memory and saved to the output file. While this solution is obvious and easy to implement, it requires an examination and a (pseudo)random number sampling step for each line in the input file.</p><p id="p-20">An alternative approach has been proposed by Vitter (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Vitter 1984</a>, <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">1987</a>). The idea is to decide how many loci to skip, given the current number already picked and remaining to be examined. <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Vitter (1987)</a> demostrated that this approach (Vitterâs Method D) is faster than the simple line-wise decision-making outlined above (referred to as Method S). However, the tests were performed 30 years ago. The files were stored on tape, and random number generation was computationally expensive. Therefore, I implemented both Method S and Method D in C++, using comparable language facilities (see Methods and Supplemental files for details), and tested them in a number of scenarios to determine which scheme is preferable on modern-day computers.</p><p id="p-21">Several variables can influence algorithm execution speed. Random (or pseudorandom) number generation is used extensively to generate samples from required distributions. However, code profiling (not shown) revealed that at least the hardware RNG I chose for this implementation (see Methods for details) is never the rate-limiting step, even when files are stored on a fast solid state drive. Rather, it is file input/output that takes most time during a given execution cycle. There are two parameters to consider when we investigate file read/write timing. One, storage can be either on a solid-state (SSD) or a spinning drive (HDD). The former is generally faster and allows for random file access. Second, the files can be either in a binary format (I use the popular and highly-compressed plink .bed), or in plain text. The important difference is that lines in files with text records are in general variable in length. Thus, if we want to skip several records we have no choice but to read each row in turn and discard unwanted loci. In contrast, binary formats use fixed-size fields, leading to uniform row sizes. It is then trivial to compute the number of bytes to skip without reading before arriving at the desired record.</p><p id="p-22">Given that I am interested in creating a tool that can be used on personal workstations and laptops, and since solid state drives have become the standard choice, I focus on execution timing on a laptop (mid-2015 15-inch MacBook Pro) with an SSD. However, I also replicated the results on a 2014 Mac Mini with an HDD with essentially the same results (see Supplemental Figures 2 and 3). I first held the input file size constant and varied the number of loci sampled. As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, time taken by both Method D and Method S grows approximately linearly with the number of loci sampled. This is the case for both binary and text files. Method D (Vitterâs skip-over algorithm) outperforms the simpler Method S several-fold when the number of records is much smaller than the total in a binary file. This is not surprising, given that in this case Method S examines and discards many more loci for each one picked. As expected, the difference largely disappears when we sample from a text file. This is because both methods have to at least read and discard file lines one at a time. Interestingly, I obtained similar results on an HDD (Figures S3 and S2), even though a spinning drive should not allow the same level of random access as an SSD. It is notable that in every case working with a binary file is about an order of magnitude faster, even though I am sampling more loci from a bigger file (500,000 loci in the binary <em>vs.</em> 100,000 in the text file). Finally, although the performance benefit of Method D is not always dramatic, it never underperforms Method S. The relatively small amount of extra time taken by Method S likely reflects the additional operations that are necessary to decide whether to include a given record.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Execution timing with samples of varying size. (A) Execution time (y-axis) as a function of the number of samples picked from the file (x-axis). The total number of SNPs is held constant. The input file is in a binary (.bed) format. (B) The ratio of the Method S to Method D execution time as a function of the number of samples taken from a binary file (the timing data taken from panel A). (C) Arranged the same as panel A, but the input file is in a text (.tped) format. (D) The same as B, but using the .tped format. Distributions are derived from 15 replicate runs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-333980319" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Execution timing with samples of varying size. (A) Execution time (&lt;em&gt;y&lt;/em&gt;-axis) as a function of the number of samples picked from the file (&lt;em&gt;x&lt;/em&gt;-axis). The total number of SNPs is held constant. The input file is in a binary (.bed) format. (B) The ratio of the Method S to Method D execution time as a function of the number of samples taken from a binary file (the timing data taken from panel A). (C) Arranged the same as panel A, but the input file is in a text (.tped) format. (D) The same as B, but using the .tped format. Distributions are derived from 15 replicate runs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1455/F1.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/5/1455/F1.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489695" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Execution timing with samples of varying size. (A) Execution time (<em>y</em>-axis) as a function of the number of samples picked from the file (<em>x</em>-axis). The total number of SNPs is held constant. The input file is in a binary (.bed) format. (B) The ratio of the Method S to Method D execution time as a function of the number of samples taken from a binary file (the timing data taken from panel A). (C) Arranged the same as panel A, but the input file is in a text (.tped) format. (D) The same as B, but using the .tped format. Distributions are derived from 15 replicate runs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Given that Vitterâs method decides ahead of time how many loci to skip before sampling, I would expect that it should be relatively insensitive to the total size of the input data set. Indeed, this is the case for binary file sampling (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, panels A and B). Increasing input file size fivefold results in no measurable rise in execution time. Method S execution time, and that of both methods on a text file (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, C and D</a>), grows approximately linearly with input size. Again, Method D is always at least as fast as Method S.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Execution timing and total record number. (A) Execution time (y-axis) of sampling 50,000 SNPs as a function of the total number of records in the (binary .bed) file. (B) The ratio of Method S to Method D timing, derived from the data in panel A. (C) The same as (A), but for sampling 10,000 loci from a text .tped file. (D) Method S to Method D execution time ratio for the data from panel C. Distributions reflect 15 replicate runs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-333980319" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Execution timing and total record number. (A) Execution time (&lt;em&gt;y&lt;/em&gt;-axis) of sampling 50,000 SNPs as a function of the total number of records in the (binary .bed) file. (B) The ratio of Method S to Method D timing, derived from the data in panel A. (C) The same as (A), but for sampling 10,000 loci from a text .tped file. (D) Method S to Method D execution time ratio for the data from panel C. Distributions reflect 15 replicate runs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1455/F2.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/5/1455/F2.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489720" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Execution timing and total record number. (A) Execution time (<em>y</em>-axis) of sampling 50,000 SNPs as a function of the total number of records in the (binary .bed) file. (B) The ratio of Method S to Method D timing, derived from the data in panel A. (C) The same as (A), but for sampling 10,000 loci from a text .tped file. (D) Method S to Method D execution time ratio for the data from panel C. Distributions reflect 15 replicate runs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Given that Method S never consistently outperforms Vitterâs Method D, I included only the latter in my implementation. While I do include the facility to read various text variant file formats, it is clear that using the .bed binary files, ideally on an SSD, results in optimal performance.</p></div><div id="sec-11" class="subsection"><h3>Linkage disequilibrium distributions</h3><p id="p-27">Estimating rates of LD decay with distance on a chromosome are necessary, for example, in genome-wide association studies where such rates determine peak resolution. Because calculating LD between all pairs of loci is infeasible and unnecessary, a typical approach is to estimate linkage statistics in sliding windows. This technique is employed in plink. I implemented an alternative approach, picking loci according to Vitterâs algorithm (see Methods for details) and then calculating LD statistics between consecutive variants in the sample. To test my implementation, I used a data set of 638,699 SNPs from the rice high-density array (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">McCouch <em>et al.</em> 2016</a>, see Methods for details). I first ran plink to calculate <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-7.gif"/></noscript></span></span> between loci no more than 500 kb or 20 SNPs apart. This yielded more than 12 million locus pairs, stored in a 1.2 gigabyte file. The relationships between linkage disequilibrium and distance are depicted in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3 (A, B)</a>. As expected, precision of LD estimates between distant loci diminishes due to undersampling. I then analyzed the same data set using my approach, sampling 30,000 SNP pairs (the resulting file occupies a mere 1.4 megabyte). While the confidence intervals from these estimates are wider (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A, B</a>), the pattern of LD decay is the same as that captured by the considerably larger sample set produced by plink. Thus, my light-weight approach may be the best option when great precision is not required and computational resources are limited.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium among sampled loci. The  (A, C) and the  (B, D) statistic as a function of distance between SNPs. Thick lines reflect the means and colored areas depict confidence intervals of a generalized additive smoothing function. Panels (A) and (B) compare the results from sampleLD and plink. Panels (C) and (D) depict estimates from O. sativa indica (IND) and japonica (JAP)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-333980319" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Linkage disequilibrium among sampled loci. The &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-8&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (A, C) and the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B, D) statistic as a function of distance between SNPs. Thick lines reflect the means and colored areas depict confidence intervals of a generalized additive smoothing function. Panels (A) and (B) compare the results from sampleLD and plink. Panels (C) and (D) depict estimates from &lt;em&gt;O. sativa indica&lt;/em&gt; (IND) and &lt;em&gt;japonica&lt;/em&gt; (JAP).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1455/F3.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/5/1455/F3.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1455/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489721" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">Linkage disequilibrium among sampled loci. The <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/F3/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/F3/embed/mml-math-8.gif"/></noscript></span></span> (A, C) and the <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/F3/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/F3/embed/mml-math-9.gif"/></noscript></span></span> (B, D) statistic as a function of distance between SNPs. Thick lines reflect the means and colored areas depict confidence intervals of a generalized additive smoothing function. Panels (A) and (B) compare the results from sampleLD and plink. Panels (C) and (D) depict estimates from <em>O. sativa indica</em> (IND) and <em>japonica</em> (JAP).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">As the number of locus pairs picked for the LD analysis increases, the average distance between them gets smaller. This leads to an increased precision of LD estimates between SNPs close to each other, at the expense of undersamling and hence diminished accuracy of disequilibrium calculations between distant loci. The user should keep this trade-off in mind when selecting the sample size. A further increase in LD estimate precision between distant variants can be achieved if one collects several sparse samples. However, care must be taken to eliminate redundant locus pairs because sampling without replacement is not guaranteed in this case.</p><p id="p-30">An extra feature of my sampleLD program, unavailable in plink, is the ability to make separate LD estimates for each population present in a set of individuals. I illustrate this possibility by estimating linkage disequilibrium in <em>indica</em> and tropical <em>japonica</em> rice varietal groups (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C, D</a>). It is well established (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">McCouch <em>et al.</em> 2016</a>) that LD levels are lower in <em>indica</em>. My analyses recapitulate this pattern.</p><p id="p-31">The software described in this report enables users to quickly generate subsets of large SNP or CNV data sets, opening the door to numerous applications that constrain resources available for genetic data manipulation. For example, it enables quick testing of bioinformatics pipelines on representative data subsets before full-scale deployment. This work exemplifies the kinds of approaches needed to speed discovery cycles and empower researchers lacking access to expensive hardware.</p></div></div><div class="section other1" id="sec-12"><h2 class="">Web Resources</h2><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-32"><strong>Project name:</strong> sampleSNPs</p></li><li id="list-item-2"><p id="p-33"><strong>Project homepage:</strong> <a href="https://github.com/tonymugen/sampleSNPs/">https://github.com/tonymugen/sampleSNPs/</a></p></li><li id="list-item-3"><p id="p-34"><strong>Project documentation:</strong> <a href="https://tonymugen.github.io/sampleSNPs/">https://tonymugen.github.io/sampleSNPs/</a></p></li><li id="list-item-4"><p id="p-35"><strong>Operating systems:</strong> Unix-like (Linux, BSD, Mac OS)</p></li><li id="list-item-5"><p id="p-36"><strong>Programming language:</strong> C++</p></li><li id="list-item-6"><p id="p-37"><strong>Other requirements:</strong> No dependencies other than the C++11 standard library</p></li><li id="list-item-7"><p id="p-38"><strong>License:</strong> BSD three-part</p></li></ul></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">The idea for this project was spurred by Kevin Lawlerâs blog post on Vitterâs work (<a href="https://getkerf.wordpress.com/2016/03/30/the-best-algorithm-no-one-knows-about/">https://getkerf.wordpress.com/2016/03/30/the-best-algorithm-no-one-knows-about/</a>). The data for implementation testing were obtained from the <em>Drosophila</em> Genome Nexus (<a href="http://www.johnpool.net/genomes.html">http://www.johnpool.net/genomes.html</a>) and the Rice Diversity website (<a href="https://ricediversity.org/data/index.cfm">https://ricediversity.org/data/index.cfm</a>). Janet Huie provided valuable comments on the text. Comments from two anonymous reviewers improved the manuscript and the software.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX A</h2><div class="section" id="sec-13"><h3>LD SAMPLE SCHEME DERIVATION</h3><p id="p-39">For the sample to accurately reflect whole-genome values, the probability of picking each pair must be equal. Furthermore, pairs must be sampled without replacement. To derive such a scheme, I order the list of all possible locus pairs as they would appear in a pair-wise relationship matrix. Since LD measures are symmetric, we need only concern ourselves with the upper (or, equivalently, lower) triangle of this matrix. The first row of the upper triangle lists the pairings of the first locus on a chromosome with all <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-10.gif"/></noscript></span></span> subsequent loci other than itself. Next row lists <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-11.gif"/></noscript></span></span> relationships between the second variant and the rest, excluding itself and the first locus. The process continues until we reach the last locus, which has no additional pairs. Sampling from this list will yield the desired uniform representation of locus pairs. Each variant on the list of pairs is represented <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-12.gif"/></noscript></span></span> times in a row, where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-13.gif"/></noscript></span></span> is the index of the locus position. Thus, instead of going through the pairs list (which contains <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-14.gif"/></noscript></span></span> elements) we can use a two-step scheme. We start by picking the first locus in a pair by sampling variants with weights reflecting the length of their run in the pairs list. We would then randomly pick another variant from the remaining loci. Finally, we go back to the first SNP or CNV in the pair and use it to sample the jump length to the next locus according to a weighted algorithm and repeat the process until we have the desired number of pairs. The initial sampling weight for locus <em>i</em> under this scheme is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-15.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-16.gif"/></noscript></span></span> is the probability of sampling locus <em>i</em>. Since, as mentioned above, each variant is represented <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-17.gif"/></noscript></span></span> times on the list,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-18.gif"/></noscript></span></span>Since <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-19.gif"/></noscript></span></span> are probabilities, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-20.gif"/></noscript></span></span> This leads to the expression for <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-21.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-22.gif"/></noscript></span></span>Thus, the deviation from an equal-weight random sampling (with all <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-23.gif"/></noscript></span></span>) depends solely on the value approximately <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-24.gif"/></noscript></span></span> which is tiny for large data sets (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-25.gif"/></noscript></span></span>) we typically encounter.</p><p id="p-40">According to the scheme presented above, once we have the first locus in a pair, we would then sample randomly from the loci further down on the chromosome to obtain the second variant for LD calculations. The next round would then require us to go back in the file to the first locus in the pair and continue with our scheme. This step is potentially computationally expensive, especially for text files with variable-width lines. To eliminate this complication, I further simplify the algorithm by instead using Vitterâs method to sample <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-26.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1455/embed/mml-math-27.gif"/></noscript></span></span> is the desired number of sampled pairs) loci. I then calculate LD between consecutive pairs of variants. A slight correction is needed only when more than one chromosome is present in the data set. In such cases, locus pairs that are located on different chromosomes are discarded and the additional pairs are sampled to restore the total to the required value. The resulting scheme approximates true uniform sampling very well when data sets are large and sample sizes are relatively small (preliminary tests suggested that sample sizes as large as 1/3 the total number of SNPs still yield reasonable results).</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42"><em>Communicating editor: J. Comeron</em></p></li><li class="fn" id="fn-2"><p id="p-43">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/genetics.5968987">https://doi.org/10.25387/genetics.5968987</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-11-21"><span class="received-label">Received </span>November 21, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-03-06"><span class="accepted-label">Accepted </span>March 6, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2018 Greenberg</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.1" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">2156</span>â<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.2" data-doi="10.1214/aos/1176344552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efron</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span>â<span class="cit-article-title">Bootstrap methods: Another look at the jacknife.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>â<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344552</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979GL00300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.3" data-doi="10.1080/01621459.1962.10480667"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rezucha</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span>â<span class="cit-article-title">Development of sampling plans by using sequential (item by item) selection techniques and digital computers.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">298</span>): <span class="cit-fpage">387</span>â<span class="cit-lpage">402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1962.10480667</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D57%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1962.10480667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1962.10480667&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.4" data-doi="10.1093/molbev/msw195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lack</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">A thousand fly genomes: An expanded <em>Drosophila</em> genome nexus.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">3308</span>â<span class="cit-lpage">3313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw195</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw195%26rft_id%253Dinfo%253Apmid%252F27687565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27687565&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span>â<span class="cit-article-title">The interaction of selection and linkage. I. General considerations; heterotic models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">49</span>â<span class="cit-lpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D67%26rft.atitle%253DTHE%2BINTERACTION%2BOF%2BSELECTION%2BAND%2BLINKAGE.%2BI.%2BGENERAL%2BCONSIDERATIONS%253B%2BHETEROTIC%2BMODELS%26rft_id%253Dinfo%253Apmid%252F17248194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI0OS8xLzQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE0NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.6" data-doi="10.1145/272991.272995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Mersenne twister: a 623-dimensionally equidistributed uniform pseudo-random number generator.</span> <abbr class="cit-jnl-abbrev">ACM TOMACS</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">3</span>â<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/272991.272995</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACM%2BTOMACS%26rft.volume%253D8%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1145%252F272991.272995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/272991.272995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.7" data-doi="10.1038/ncomms10532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maron</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Open access resources for genome-wide association mapping in rice.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">10532</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms10532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D10532%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms10532%26rft_id%253Dinfo%253Apmid%252F26842267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms10532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26842267&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.8" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">PLINK: a toolset for whole-genome association and population-based linkage analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">559</span>â<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.9" data-doi="10.1371/journal.pgen.1004982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genomic selection and association mapping in rice (<em>Oryza sativa</em>): Effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1004982</span> <span class="cit-comment">(PLOS Genet. 11(6): e1005350)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004982%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004982%26rft_id%253Dinfo%253Apmid%252F25689273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25689273&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.10" data-doi="10.1038/nature02168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">The international HapMap project.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>(<span class="cit-issue">6968</span>): <span class="cit-fpage">789</span>â<span class="cit-lpage">796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02168</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D426%26rft.issue%253D6968%26rft.spage%253D789%26rft.epage%253D796%26rft.atitle%253DThe%2BInternational%2BHapMap%2BProject.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02168%26rft_id%253Dinfo%253Apmid%252F14685227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14685227&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1455.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187342000046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.11" data-doi="10.1145/358105.893"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitter</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Faster methods for random sampling.</span> <abbr class="cit-jnl-abbrev">Commun. ACM</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">703</span>â<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/358105.893</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BACM%26rft.volume%253D27%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.1145%252F358105.893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/358105.893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.5.1455.12" data-doi="10.1145/23002.23003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitter</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">An efficient algorithm for sequential random sampling.</span> <abbr class="cit-jnl-abbrev">ACM Trans. Math. Softw.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">58</span>â<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/23002.23003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACM%2BTrans.%2BMath.%2BSoftw.%26rft.volume%253D13%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.1145%252F23002.23003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/23002.23003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-book" id="cit-8.5.1455.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/5/1455.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1439" title="Cold Shock as a Screen for Genes Involved in Cold Acclimatization in Neurospora crassa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1461" title="Comparative Transcriptomics Among Four White Pine Species" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/5"><span>Volume 8 Issue 5, May 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/5.cover-source.jpg?itok=Bl7MXdvV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1455" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_dwnHgan530E7BJgoC7VsSQ741VUB2-8VdZr9_xW_EU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/5/1455" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/5/1455" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/5/1455" class="active" data-icon-position="" data-hide-link-title="0">Fast Ordered Sampling of DNA Sequence Variants</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489473" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Oc1KWLpIY4RKwIoyDoZtw68mBMI0vmlvPk_0De-9Ycc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1455.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/5/1455" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-YdTwnRhVIaLYsDDxJoM5ZHyZBF-6z4ego1IL_mGcq4g" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/5/1455" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1455.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489473" id="node489473--2" data-pisa="ggg;8/5/1455" data-pisa-master="ggg;g3.117.300465" data-apath="/ggg/8/5/1455.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast Ordered Sampling of DNA Sequence Variants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3296-5811" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anthony J. Greenberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1455-1460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300465 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489473--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anthony J. Greenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Bayesic Research, Ithaca, NY 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BJ.%2BGreenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenberg%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnthony%2BJ.%2BGreenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3296-5811" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anthony J. Greenberg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tonyg@bayesicresearch.org" class="" data-icon-position="" data-hide-link-title="0">tonyg@bayesicresearch.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/5/1455" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/5/1455&amp;title=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/5/1455&amp;title=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/5/1455&amp;title=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/5/1455&amp;text=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/5/1455&amp;title=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/5/1455&amp;t=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/5/1455&amp;title=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/5/1455&amp;title=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489473/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489473" id="node489473--3" data-pisa="ggg;8/5/1455" data-pisa-master="ggg;g3.117.300465" data-apath="/ggg/8/5/1455.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fast Ordered Sampling of DNA Sequence Variants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3296-5811" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anthony J. Greenberg</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1455-1460;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300465 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489473--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anthony J. Greenberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Bayesic Research, Ithaca, NY 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BJ.%2BGreenberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenberg%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnthony%2BJ.%2BGreenberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3296-5811" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anthony J. Greenberg</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tonyg@bayesicresearch.org" class="" data-icon-position="" data-hide-link-title="0">tonyg@bayesicresearch.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1455&amp;rft.epage=1460&amp;rft.atitle=Fast%20Ordered%20Sampling%20of%20DNA%20Sequence%20Variants&amp;rft.volume=8&amp;rft.issue=5&amp;rft.date=2018-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Greenberg%2C+Anthony+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489473/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489473/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489473/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489473/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489473/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489473/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489473/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489473/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489473/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489473/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489473/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489473/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/5/1455" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results And Discussion</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Web Resources</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX A</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/5/1455.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/5/1455.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/5/1455.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/5/1455.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
