<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/5/1687" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/5/1687.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/5/1687.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/5/1687.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/5/1687/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/5/1687.atom" />
<meta name="HW.pisa" content="ggg;8/5/1687" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers" />
<meta name="DC.Identifier" content="10.1534/g3.117.300548" />
<meta name="DC.Date" content="2018-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Jiang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.5986933" />
<meta name="DC.Description" content="Genome-wide prediction approaches represent versatile tools for the analysis and prediction of complex traits. Mostly they rely on marker-based information, but scenarios have been reported in which models capitalizing on closely-linked markers that were combined into haplotypes outperformed marker-based models. Detailed comparisons were undertaken to reveal under which circumstances haplotype-based genome-wide prediction models are superior to marker-based models. Specifically, it was of interest to analyze whether and how haplotype-based models may take local epistatic effects between markers into account. Assuming that populations consisted of fully homozygous individuals, a marker-based model in which local epistatic effects inside haplotype blocks were exploited (LEGBLUP) was linearly transformable into a haplotype-based model (HGBLUP). This theoretical derivation formally revealed that haplotype-based genome-wide prediction models capitalize on local epistatic effects among markers. Simulation studies corroborated this finding. Due to its computational efficiency the HGBLUP model promises to be an interesting tool for studies in which ultra-high-density SNP data sets are studied. Applying the HGBLUP model to empirical data sets revealed higher prediction accuracies than for marker-based models for both traits studied using a mouse panel. In contrast, only a small subset of the traits analyzed in crop populations showed such a benefit. Cases in which higher prediction accuracies are observed for HGBLUP than for marker-based models are expected to be of immediate relevance for breeders, due to the tight linkage a beneficial haplotype will be preserved for many generations. In this respect the inheritance of local epistatic effects very much resembles the one of additive effects." />
<meta name="DC.Contributor" content="Yong Jiang" />
<meta name="DC.Contributor" content="Renate H. Schmidt" />
<meta name="DC.Contributor" content="Jochen C. Reif" />
<meta name="article:published_time" content="2018-05-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide prediction approaches represent versatile tools for the analysis and prediction of complex traits. Mostly they rely on marker-based information, but scenarios have been reported in which models capitalizing on closely-linked markers that were combined into haplotypes outperformed marker-based models. Detailed comparisons were undertaken to reveal under which circumstances haplotype-based genome-wide prediction models are superior to marker-based models. Specifically, it was of interest to analyze whether and how haplotype-based models may take local epistatic effects between markers into account. Assuming that populations consisted of fully homozygous individuals, a marker-based model in which local epistatic effects inside haplotype blocks were exploited (LEGBLUP) was linearly transformable into a haplotype-based model (HGBLUP). This theoretical derivation formally revealed that haplotype-based genome-wide prediction models capitalize on local epistatic effects among markers. Simulation studies corroborated this finding. Due to its computational efficiency the HGBLUP model promises to be an interesting tool for studies in which ultra-high-density SNP data sets are studied. Applying the HGBLUP model to empirical data sets revealed higher prediction accuracies than for marker-based models for both traits studied using a mouse panel. In contrast, only a small subset of the traits analyzed in crop populations showed such a benefit. Cases in which higher prediction accuracies are observed for HGBLUP than for marker-based models are expected to be of immediate relevance for breeders, due to the tight linkage a beneficial haplotype will be preserved for many generations. In this respect the inheritance of local epistatic effects very much resembles the one of additive effects.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/05/01" />
<meta name="citation_mjid" content="ggg;8/5/1687" />
<meta name="citation_id" content="8/5/1687" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/5/1687" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/5/1687.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/5/1687.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/5/1687.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300548" />
<meta name="citation_pmid" content="29549092" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1687" />
<meta name="citation_lastpage" content="1699" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yong Jiang" />
<meta name="citation_author_institution" content="Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2824-677X" />
<meta name="citation_author" content="Renate H. Schmidt" />
<meta name="citation_author_institution" content="Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8037-3581" />
<meta name="citation_author" content="Jochen C. Reif" />
<meta name="citation_author_institution" content="Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany" />
<meta name="citation_author_email" content="reif@ipk-gatersleben.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6742-265X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Akdemir;citation_author=J. L. Jannink;citation_title=Locally epistatic genomic relationship matrices for genomic association and prediction.;citation_volume=199;citation_year=2015;citation_issue=3;citation_pmid=25614606;citation_doi=10.1534/genetics.114.173658" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J. L. Jannink;citation_author=J. Isidro-Sánchez;citation_title=Locally epistatic models for genome-wide prediction and association by importance sampling.;citation_pages=74;citation_volume=49;citation_year=2017;citation_issue=1;citation_doi=10.1186/s12711-017-0348-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=E. Bauer;citation_author=M. Falque;citation_author=H. Walter;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Intraspecific variation of recombination rate in maize.;citation_pages=R103;citation_volume=14;citation_year=2013;citation_issue=9;citation_pmid=24050704;citation_doi=10.1186/gb-2013-14-9-r103" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=D. Boichard;citation_author=F. Guillaume;citation_author=A. Baur;citation_author=P. Croiseau;citation_author=M. N. Rossignol;citation_title=Genomic selection in French dairy cattle.;citation_pages=115-120;citation_volume=52;citation_year=2012;citation_issue=3;citation_doi=10.1071/AN11119" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. P. L. Calus;citation_author=T. H. E. Meuwissen;citation_author=A. P. W. de Roos;citation_author=R. F. Veerkamp;citation_title=Accuracy of Genomic Selection Using Different Methods to Define Haplotypes;citation_pages=553-561;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202394;citation_doi=10.1534/genetics.107.080838" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. L. Calus;citation_author=T. H. E. Meuwissen;citation_author=J. J. Windig;citation_author=E. F. Knol;citation_author=C. Schrooten;citation_title=Effects of the number of markers per haplotype and clustering of haplotypes on the accuracy of QTL mapping and prediction of genomic breeding values.;citation_pages=11-11;citation_volume=41;citation_year=2009;citation_issue=1;citation_pmid=19284677;citation_doi=10.1186/1297-9686-41-11" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Ö. Carlborg;citation_author=C. S. Haley;citation_title=Epistasis: too often neglected in complex trait studies?;citation_pages=618-625;citation_volume=5;citation_year=2004;citation_issue=8;citation_pmid=15266344;citation_doi=10.1038/nrg1407" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=A. G. Clark;citation_title=The role of haplotypes in candidate gene studies.;citation_pages=321-333;citation_volume=27;citation_year=2004;citation_issue=4;citation_pmid=15368617;citation_doi=10.1002/gepi.20025" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_issue=1;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. C. Cuyabano;citation_author=G. Su;citation_author=M. S. Lund;citation_title=Genomic prediction of genetic merit using LD-based haplotypes in the Nordic Holstein population.;citation_pages=1171;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=25539631;citation_doi=10.1186/1471-2164-15-1171" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=B. C. Cuyabano;citation_author=G. Su;citation_author=M. S. Lund;citation_title=Selection of haplotype variables from a high-density marker map for genomic prediction.;citation_pages=61;citation_volume=47;citation_year=2015;citation_issue=1;citation_doi=10.1186/s12711-015-0143-3" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=B. C. Cuyabano;citation_author=G. Su;citation_author=G. J. M. Rosa;citation_author=M. S. Lund;citation_author=D. Gianola;citation_title=Bootstrap study of genome-enabled prediction reliabilities using haplotype blocks across Nordic Red cattle breeds.;citation_pages=7351-7363;citation_volume=98;citation_year=2015;citation_issue=10;citation_doi=10.3168/jds.2015-9360" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=D. B. Allison;citation_title=Predicting genetic predisposition in humans: the promise of whole-genome markers.;citation_pages=880-886;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21045869;citation_doi=10.1038/nrg2898" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_issue=2;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_issue=7;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. A. F. Garcia;citation_author=S. Wang;citation_author=A. E. Melchinger;citation_author=Z. B. Zeng;citation_title=Quantitative trait loci mapping and the genetic basis of heterosis in maize and rice.;citation_volume=180;citation_year=2008;citation_issue=3;citation_pmid=18791260;citation_doi=10.1534/genetics.107.082867" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=M. Perez-Enciso;citation_author=M. A. Toro;citation_title=On Marker-Assisted Prediction of Genetic Value: Beyond the Ridge;citation_pages=347-365;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586721" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=B. J. Hayes;citation_author=H. A. Lewin;citation_author=M. E. Goddard;citation_title=The future of livestock breeding: genomic selection for efficiency, reduced emissions intensity, and adaptation.;citation_pages=206-214;citation_volume=29;citation_year=2013;citation_issue=4;citation_pmid=23261029;citation_doi=10.1016/j.tig.2012.11.009" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. He;citation_author=A. W. Schulthess;citation_author=V. Mirdita;citation_author=Y. Zhao;citation_author=V. Korzun;citation_title=Genomic selection in a commercial winter wheat population.;citation_pages=641-651;citation_volume=129;citation_year=2016;citation_issue=3;citation_doi=10.1007/s00122-015-2655-1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. He;citation_author=J. C. Reif;citation_author=V. Korzun;citation_author=R. Bothe;citation_author=E. Ebmeyer;citation_title=Genome-wide mapping and prediction suggests presence of local epistasis in a vast elite winter wheat populations adapted to Central Europe.;citation_pages=635-647;citation_volume=130;citation_year=2017;citation_issue=4;citation_doi=10.1007/s00122-016-2840-x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_issue=1;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. M. Hickey;citation_author=T. Chiurugwi;citation_author=I. Mackay;citation_author=W. Powell;citation_title=Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.;citation_pages=1297-1303;citation_volume=49;citation_year=2017;citation_issue=9;citation_doi=10.1038/ng.3920" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=J. C. Reif;citation_title=Modeling epistasis in genomic selection.;citation_volume=201;citation_year=2015;citation_issue=2;citation_pmid=26219298;citation_doi=10.1534/genetics.115.177907" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Y. Jiang;citation_author=R. H. Schmidt;citation_author=Y. Zhao;citation_author=J. C. Reif;citation_title=A quantitative genetic framework highlights the role of epistatic effects for grain-yield heterosis in bread wheat.;citation_pages=1741-1746;citation_volume=49;citation_year=2017;citation_issue=12;citation_doi=10.1038/ng.3974" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=D. Jónás;citation_author=V. Ducrocq;citation_author=M. N. Fouilloux;citation_author=P. Croiseau;citation_title=Alternative haplotype construction methods for genomic evaluation.;citation_pages=4537-4546;citation_volume=99;citation_year=2016;citation_issue=6;citation_doi=10.3168/jds.2015-10433" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Lehermeier;citation_author=N. Krämer;citation_author=E. Bauer;citation_author=C. Bauland;citation_author=C. Camisan;citation_title=Usefulness of Multiparental Populations of Maize (Zea mays L.) for Genome-Based Prediction;citation_pages=3-16;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236445;citation_doi=10.1534/genetics.114.161943" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. F. Mackay;citation_title=Epistasis and quantitative traits: using model organisms to study gene-gene interactions.;citation_pages=22-33;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=24296533;citation_doi=10.1038/nrg3627" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. R. Muñoz;citation_author=M. F. Resende;citation_author=S. A. Gezan;citation_author=M. D. V. Resende;citation_author=G. de los Campos;citation_title=Unraveling additive from non-additive effects using genomic relationship matrices.;citation_volume=198;citation_year=2014;citation_issue=4;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Peréz;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_issue=2;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Pettersson;citation_author=F. Besnier;citation_author=P. B. Siegel;citation_author=Ö. Carlborg;citation_title=Replication and explorations of high-order epistasis using a large advanced intercross line pedigree.;citation_pages=e1002180-e1002180;citation_volume=7;citation_year=2011;citation_issue=7;citation_pmid=21814519;citation_doi=10.1371/journal.pgen.1002180" />
<meta name="citation_reference" content="R Core Team, 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. http://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. R. Sailer;citation_author=M. J. Harms;citation_title=Detecting high-order epistasis in nonlinear genotype-phenotype maps.;citation_volume=205;citation_year=2017;citation_issue=3;citation_pmid=28100592;citation_doi=10.1534/genetics.116.195214" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=A. Scheben;citation_author=J. Batley;citation_author=D. Edwards;citation_title=Genotyping by sequencing approaches to characterize crop genomes: choosing the right tool for the right application.;citation_pages=149-161;citation_volume=15;citation_year=2017;citation_issue=2;citation_doi=10.1111/pbi.12645" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. B. Taylor;citation_author=I. M. Ehrenreich;citation_title=Genetic interactions involving four or more genes contribute to a complex trait in yeast.;citation_pages=e1004324;citation_volume=10;citation_year=2014;citation_issue=5;citation_pmid=24784154;citation_doi=10.1371/journal.pgen.1004324" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. B. Taylor;citation_author=I. M. Ehrenreich;citation_title=Higher-order genetic interactions and their contribution to complex traits.;citation_pages=34-40;citation_volume=31;citation_year=2015;citation_issue=1;citation_pmid=25284288;citation_doi=10.1016/j.tig.2014.09.001" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=A. Legarra;citation_author=M. A. Toro;citation_author=L. Varona;citation_title=Orthogonal estimates of variances for additive, dominance and epistatic effects in populations.;citation_volume=206;citation_year=2017;citation_issue=3;citation_pmid=28522540;citation_doi=10.1534/genetics.116.199406" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=D. Wang;citation_author=I. S. El-Basyoni;citation_author=P. S. Baenziger;citation_author=J. Crossa;citation_author=K. M. Eskridge;citation_title=Prediction of genetic values of quantitative traits with epistatic effects in plant breeding populations.;citation_pages=313-319;citation_volume=109;citation_year=2012;citation_issue=5;citation_pmid=22892636;citation_doi=10.1038/hdy.2012.44" />
<meta name="citation_reference" content="Wimmer, V., T. Albrecht, H. J. Auinger, and C. C. Schön, 2015 R Package “synbreedData”. CRAN. http://CRAN.R-project.org/package=synbreedData." />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=D. Wittenburg;citation_author=N. Melzer;citation_author=N. Reinsch;citation_title=Including non-additive genetic effects in Bayesian methods for the prediction of genetic values based on genome-wide markers.;citation_pages=74-74;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21867519;citation_doi=10.1186/1471-2156-12-74" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. R. Wood;citation_author=M. A. Tuke;citation_author=M. A. Nalls;citation_author=D. G. Hernandez;citation_author=S. Bandinelli;citation_title=Another explanation for apparent epistasis.;citation_pages=E3-E5;citation_volume=514;citation_year=2014;citation_issue=7520;citation_pmid=25279928;citation_doi=10.1038/nature13691" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=S. Xu;citation_title=An empirical Bayes method for estimating epistatic effects of quantitative trait loci.;citation_pages=513-521;citation_volume=63;citation_year=2007;citation_issue=2;citation_pmid=17688503;citation_doi=10.1111/j.1541-0420.2006.00711.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Yang;citation_author=R. J. Tempelman;citation_title=A Bayesian antedependence model for whole genome prediction.;citation_volume=190;citation_year=2012;citation_issue=4;citation_pmid=22135352;citation_doi=10.1534/genetics.111.131540" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhang;citation_author=W. Wang;citation_author=W. Valdar;citation_title=Bayesian modeling of haplotype effects in multiparent populations.;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236455;citation_doi=10.1534/genetics.114.166249" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C. W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. T. Zondervan;citation_author=L. R. Cardon;citation_title=The complex interplay among factors that influence allelic association.;citation_pages=89-100;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14735120;citation_doi=10.1038/nrg1270" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide prediction approaches represent versatile tools for the analysis and prediction of complex traits. Mostly they rely on marker-based information, but scenarios have been reported in which models capitalizing on closely-linked markers that were combined into haplotypes outperformed marker-based models. Detailed comparisons were undertaken to reveal under which circumstances haplotype-based genome-wide prediction models are superior to marker-based models. Specifically, it was of interest to analyze whether and how haplotype-based models may take local epistatic effects between markers into account. Assuming that populations consisted of fully homozygous individuals, a marker-based model in which local epistatic effects inside haplotype blocks were exploited (LEGBLUP) was linearly transformable into a haplotype-based model (HGBLUP). This theoretical derivation formally revealed that haplotype-based genome-wide prediction models capitalize on local epistatic effects among markers. Simulation studies corroborated this finding. Due to its computational efficiency the HGBLUP model promises to be an interesting tool for studies in which ultra-high-density SNP data sets are studied. Applying the HGBLUP model to empirical data sets revealed higher prediction accuracies than for marker-based models for both traits studied using a mouse panel. In contrast, only a small subset of the traits analyzed in crop populations showed such a benefit. Cases in which higher prediction accuracies are observed for HGBLUP than for marker-based models are expected to be of immediate relevance for breeders, due to the tight linkage a beneficial haplotype will be preserved for many generations. In this respect the inheritance of local epistatic effects very much resembles the one of additive effects." />
<meta name="og-title" property="og:title" content="Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/5/1687" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide prediction approaches represent versatile tools for the analysis and prediction of complex traits. Mostly they rely on marker-based information, but scenarios have been reported in which models capitalizing on closely-linked markers that were combined into haplotypes outperformed marker-based models. Detailed comparisons were undertaken to reveal under which circumstances haplotype-based genome-wide prediction models are superior to marker-based models. Specifically, it was of interest to analyze whether and how haplotype-based models may take local epistatic effects between markers into account. Assuming that populations consisted of fully homozygous individuals, a marker-based model in which local epistatic effects inside haplotype blocks were exploited (LEGBLUP) was linearly transformable into a haplotype-based model (HGBLUP). This theoretical derivation formally revealed that haplotype-based genome-wide prediction models capitalize on local epistatic effects among markers. Simulation studies corroborated this finding. Due to its computational efficiency the HGBLUP model promises to be an interesting tool for studies in which ultra-high-density SNP data sets are studied. Applying the HGBLUP model to empirical data sets revealed higher prediction accuracies than for marker-based models for both traits studied using a mouse panel. In contrast, only a small subset of the traits analyzed in crop populations showed such a benefit. Cases in which higher prediction accuracies are observed for HGBLUP than for marker-based models are expected to be of immediate relevance for breeders, due to the tight linkage a beneficial haplotype will be preserved for many generations. In this respect the inheritance of local epistatic effects very much resembles the one of additive effects." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
    <title>Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"h2atZjBOaG9sels5JsCN3c6PBbmB5SSGKhIQsjo7AVY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489447","apath":"\/ggg\/8\/5\/1687.atom","pisa":"ggg;8\/5\/1687","ac":{"\/ggg\/8\/5\/1687.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/5\/1687.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1687"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1687"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/5\/1687":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wZqi85GA45IJYlkqAXmMk7c72xlhaRinXV4ZRu5qs4hUjeRYK5gNUJiNvNmKzGayMIUXtxaO1iM1QdwOvU1qIYUL9Te6xRgIzRzdVKsMrRdvf0oxciZE1W3zUm93e46yOws2YItdSnm4vEDtXtmD+jRQfHpcCH1wyVecxV76OyvAnJNA3soN1i\\\\\\\/TdVjy\\\\\\\/C7WOXSq0ug3wU2f7fHGf347kyp7J5\\\\\\\/9jWmVftDSOlu1\\\\\\\/IPnAbldmnOKsr3\\\\\\\/OYldswTPtCnrpB5eQ1tk94roNSTcwlmEmVGou6X6N+1MtYqiLIAMKsQNDO0LKIBKcMx6BYK43fsTw+VZU4lm6IHKnI\\\\\\\/HDJtaHShWGH7U6ZejLNTGDirQ2NeUGKOZUjS7o5Diy8FrpMaEaBN7TZHbPWoHC1lAzJge\\\\\\\/A6kN5OJPJ1bwTlAE3fAc+V2JsJGIFioshKyB3oMqoEi\\\\\\\/bBye7VYc2KVkNsFP95NmFHhQEs8RSV\\\\\\\/va24qgWSUVA6cPy4QQLbvTySOPSDjDYAHmpJQV9kybtdUuuDD3n0XHppZDdx6tbgo+8QJI4nPr8Silce9qJAdB+B5l9ljVFa37uArXUTadDFml9GDlomMGBvR9xbecssdR5AOoOCsc5aahSSqhMooTSUA\\\\\\\/yNoyMsOhaA41XfvWx7H7YK2H9imqC\\\\\\\/sDM6h+JRAqBD73Vz9SeRO\\\\\\\/\\\\\\\/JGK0\\\\\\\/Oxc875O4ihnAT1QGZRWEkVvaOYvFL0QgWWG9puBQbOQgTrJ1SQQGE+iL3sNgv7b2bETuRycf5zUDSgzhfinUfqTxJmupF6FGSyHUfuI0InI2nf1n6dfGYn7\\\\\\\/ZbZG5cjSEpG2lsymfdc0viyDKQ04X+mkO0VsOwDpA4wNmKBcPELMqeC4Mk3o430b2OGWlYFJwXkyqkjtjiC6oWJTFKrb7tCqSEItDA0jqJZ8VJexWZXjKBGEQFd+TPn7+0KPqhZTjjMhvsgzeVMcRedznM9JisqDglhhJI3i1mZbWX7bd4WHYmjbF+HiZq50Vv0o5xCWtRFvxpEevOB8eG2NeUPgwmEIPfUDcED3190ZIvPV1zCCjm4CjeW6RGMHziHLKu6xgDFqIR8p4bBlo\\\\\\\/ESRVGBGgcjisTRbN\\\\\\\/RiKJkMOdRxEflae\\\\\\\/IMi7asD9+Vu78JBs8n5dIniLfgdisv5XcMwEFXwy9VRGw5gMAF9oDE8wd3yTibuJ\\\\\\\/tIqIQAIWw8vpLUHJcMRy7saXZ+dEpvZShMJmjWoq6RMt1GkbDJfG7Nr8O6RCaFJTJFXkcZvhfv5qO1Q\\\\\\\/LATZ4dbv\\\u0022,\\\u0022iv\\\u0022:\\\u0022o94bewdNFr\\\\\\\/Aa5NwwTUfHg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228a22f58f7bf80733a23059b66bd9bdf053d4488f21f017bbea8aa5cf2dc94c58\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226Y9W\\\\\\\/+WHVRVAm+sVJTHzYUkd2kAm7j1IkMB11R29bPWYYiThG3P43sGyqLcuIB5zAT47RtKudCfDBmYm0XivLB+C0Nm0oRLpM4Z7HlGAgBijwIv8nr4PyVhS\\\\\\\/zPIOoY3rFw1lcKofs0\\\\\\\/fMaDMF4QLxkBpwTE1DcPP8YGLD6noHjUxj+4EAzRQh3I4r3yvMH2l\\\\\\\/wc+p9g\\\\\\\/B6IshH83j25YzjbaPdnudUd94wH7WkukLfPHQLaTaxRc3iJqahH70gv4RqGuPIHAiPmxLvwJ2zo6IK6ijkYbtAcoBJISH+zH0xp\\\\\\\/PLpHxpybvA2TmjKY\\\\\\\/EZ4gneHdo+\\\\\\\/sreNeT0K61oJQiEtZE5Pt0HRY0v6Md83Zq\\\\\\\/S02G7CCWInDBhkPjmJeNP\\\\\\\/cXfPGLiHm59NFF+aMjQIJGgoqcFa9ebDNdrAMWO+q4zoFemOcPB1VT1Tj39R+R8sMhuTQp1caCqOoLS09sPnSkydDli5YJzC8Ot8GM0wNMMLP4Lo9832WiX+vH6ttnAUk6Z056q1FH0dOAOfVxMu\\\\\\\/pybKcOlraby+kfYNv3HaMiu\\\\\\\/aw9zDQ5UAisBvKnMqBMicuTF9Fa76t3mqeMZwhZxvPkFM34Mw1N5QPz7HWFHy+VqVqEFH5zJulPbh5HrlXb0A29bf7QqM63pQnaP7efmEEvy9rMVojKW0R1HENmLTiVcfNo\\\\\\\/txFrSMpWXCnuTN3u+3\\\\\\\/qhnPOJjzZeBntJK3ZvMDISk+uFwPFeEylG\\\\\\\/LW92SBAJ2qCnWyLd345qp\\\\\\\/lWdExpZSIzau\\\\\\\/ftV0NYsdSbs58ikPfEVaqiYNf7r+VtqS1u5yowg9U0Uq5A+wzPbBNv8Va7M59X5onRsxAZEyh9zCnBwwJ5G5gdpJO5\\\\\\\/LRQO4CGzn\\\\\\\/XLt1onPNf1AFBGY3Q0wi4X5mHSpD7mJyqQgQkVRDErIPYKtlodZ5u5+HaewcKCKFhtOKQgU8EOemmkTTgvIdUfO8pgwCa19emQ2yXEIJH4UZl4u3pGItEz4UudiLdvz855FhBlTgM35HUNfe9Eidnt3zVVIVemRbdISwVd763I0kn58uSsdo0kUYhYI4ZVENP9q8DfHhm8CYbqdcpJd8gAX57aOUup1gH3n\\\\\\\/U9b8IQfO3vdbz\\\\\\\/Mel0f+obNfleuVcXRo\\\\\\\/Gue\\\\\\\/tp5kIB+IE+xmyXxPoVHvEfWH6XTIOqVxIu6WWUhYWv8GBDbs\\\\\\\/WDE6B1z+tm2U1CNCQOlirmAQi8D++ZpaI6rzi0BbXKA1X\\\\\\\/4T4Kf8kOjOpVIu7lmk\\\\\\\/LXphzXMBpT2eX3gUFakO7XwTrgGSGbZNdGmnG17Mvp4p7ynlrhFU+hy2y3Bf3IpeFepcdbwOf0Pn6zzW06NaYZf1CKnvz9F1hR+2I9qNPD+7j20LOhAEWYGkZ4rsd7LjCO7xaqjI6by37fDfJB23HRS2wrbC4hVnu6PUw2EVbf7D01UZDFMRxCZEnWk\\\\\\\/r5NDrFtK9VYNHhBsApP5LaTsxr40nQJhEH2uwW7cFpUdoXrXc0Pp8N6z8a0fViFb1mLmhXs1HD3LHwBrsbfpNtsnWmnxyyoU4XCjscBFHUHB7b41rIGAVOjgrC8VzsG9jR8q01ayGRGLzp8Wd207JVRwXMhgTkOi5OKEaxts\\\\\\\/W\\\\\\\/zrf3CK\\\\\\\/XZc+SvSRe0boGSaePYXTISFEcTD0OlThkcIeBxv9RfriF0Q1ccYQzE0N6VDGukEF5RU537Pf1BKEhg1KnYnqAvKdmg1wPpIRpZB34TfkGDBOAKHdxYqGENOqdahqdXFr74M7Cj8YIDuzzTl4uJXARa4fmrnEUNtZfolLYj020DvNxCqaUsC1ykPOUW4CVuHXhIiDIAZB3+ukhvGGxVrmMfIUBsWsoFsnCC46CkXWuTJSNOtfjwU\\\\\\\/B+WURjSpJCcjwn5Ox8G+HzjvD2IsZVE1AS8j3YbHepPDGChH\\\\\\\/QmHNaDFJScxAg59TPkN8mAGFFdN\\\\\\\/tvfcnLADSIwQsEBuL3jWbms4jl0AXQuf+GMXEh4sZcZD0fsMfsDiTysreFNl3Ba8emeZjaGhRfJBarV4NQvBphq1JF1xJvAaccD8xIEQYkdYm5Fvx9Jkg9Orc4hqwTnaCCA2kixDqgTbw2MrCrICKv5KEEzysdEhTQRp6zOJxsKyZe9OmhCA4kt+HX\\\\\\\/mnuS7M\\\\\\\/XwDJx8wlm24etzH\\\\\\\/iAYQ+pPW6DuWK3vV8nLwGUq2MvGUn2XDnVVeJPhpUg5+HOZ5nmhut\\\\\\\/Pd49lp7fLc9Q+Sn731I4XNF7bN8eRyll3F9WGDtsznnSTmwGc8Cx71uTM6f5gcNn7v5hv4dLothU4n53xvt3bgOcFrua5NcoCCV+AdmrPQ3NLWhanr74nliI3w\\\\\\\/2C3rHqyxiEVPgnGB21sfBgLhiKNCdtKgTQPNxdq6GS4ul7YoOOIZhOuX7DnqY05YYpXswJN1lkyAjH2wFcTdCMLAQcRjFIxvuoJ4JQtAQynsW2jPRHb5PmSmGrKerzktwzbUQzHaXMuzGRv\\\\\\\/QANxZBK6QGnkDKvmH+TAK38lnIUd2LAGCrs1+LFU8bsLoEvsIDYleFlrkMQMPR74fCVE+bvBLLijj2d8PfzwKDRqNuXtCoRMyS3VtK8nAuUOydSQv\\\\\\\/lvIeh74fcsdGeZD98NAwT\\\\\\\/Bho6AYJmC7Pi5letRaLBtxpES5MnSu9NO1ukB3Ul8Vpv3utU90PIL3eiSs5qgy\\\\\\\/4wLu4QAsRuK6Pc+P61JQO71xIhe1gVsWoNDSrEexxHIJls5PIcQt2eO9+2hWwyeOaDCFi5WldAVsuADkmcFNlMoRWJSLQnAT+RlZvFmPB1V8T8ZEDzl7JTHzg8kZnGQsxuQB1e1j8JUhxHdDXDsgBgi1obtxzzqT7xOZ7Aa8GYi8f4jDPjcemG+Iw9UxM682QkCAqgfvzH9UTwViQYZhuIi+cwg3GNkZjJ5in349JkfNPy6iRQgSngUQKWgWp1CY0WQcUIJzm8Y3zBm8wl1P72o6uPzSgLAhClFre6wLW7+CiXxXHSTXOWSVfK5U8fIsWOGVLyZxg5U\\\\\\\/l64IapynLrKTIvjjg3nP2cXi3+IlVtptNrOChwla7qqjIMDj+NhlX9Q\\\\\\\/+3TNkPOoL\\\\\\\/hNMkoz0hoCAGGN6oZ3mj5sdex6L+6SBh\\\\\\\/mko2hgjc7J\\\\\\\/CiBVMH8WrENF+ev4RmJqNH1GVN33xfOGRZHLmTUoV+5Xhs7bzDbpqtLBRFFJbxhf6l5osLv8txOxkzKfN024X6NovhOStwyWI\\\\\\\/VNLoY0m2usSW3bsxndL8a\\\\\\\/U5kafve915k4cTjS8GpbBa1semaZwVLcUq7wCCtZYWlBFU6Zu8S6bIYALERd8MLtyZ5fXBLGJXh+6YlvP3ZyPmj62LhMHcBPlrAWN5U1bBNQolKkIo2jkBadVsk6+haUAuzNNeIm2ddU6XMHNVMLWm5DGpq\\\\\\\/1S8xjkxqDyY1QFxH8hkTFZ+xw8Iaz97ri9I2m4ixpNYSF5Wl24wS2I321mhCu1udausfQaqUBumYdcBniE7qdXnGEHKWWvLmGmLhybeBJ7ED+qE7uA1u5vj2eoK6RLGuQqy2ktkWG7szRU7RUccBUK6a7+hKXhlM2evRoxqqpufi79TA5F7Qulr2ZBZZeG8b3sIJNrPWsNnF1uanAABbmRm9C0VA8sXnNcs4wnNKzMpDt2swzoj5L2ulZYd+U83SWsnYHB5MZ1pxmYIRezeWj9c44PCCkBIbqelaqO41+Rup4sciNLWfh+P6TX85Jvcj\\\\\\\/iWdZaYsk6iyyB\\\\\\\/iqFGOX6Twwp9n7fHm0C8qG5i5alEDis\\\\\\\/Ecx5lI3RYKUBleQz+WRAypxaAex8aSgmJD6MQDb\\\\\\\/yBmk4C2TbzvSAS92dHj9P8\\\\\\\/6xzR1FU\\\\\\\/Cy6YVzn1DlDIcxal1bNaw1ngdl+jIGn1z8oLJyGXn4KUW64qEyIA4+8m7zfwztR+9LIdl9JRcYcXvsPuegFyT8ZnuclUJIHPtWOgoPsfjAWd28syowmc15g4Ra+Z2omFbm6pkOCoZ5ZBkUCneZ5UF63\\\\\\\/7IVIo+HLFAOIOXp88lzAiYsJXb\\\\\\\/Fgu2Kjo6hvHv+geBGZErhAWR\\\\\\\/oGcCleyi2GI2cV+YPHGsbU8uH15dMLrprHZ4Za9VpSIAKX1hk1Y9pKEC\\\\\\\/7dgZqfvyJ+GDeDX13fk3rkwALvt3n\\\\\\\/JOmE6CBZxBgq2qgw2OuGNwNkQ5rsur7thBbtjakBH+hxFAigyS5k3gXGcXErejHRuW4jaJ47fKA\\\\\\\/agdk2UEdfzeKhRep\\\\\\\/p0RW2MM+UiYLcYEQx2797R6DtP2OXM1n6X\\\\\\\/8IkoAnTlPqFyNIA0ij6EunqC+lfRTQgWVro0SpR8e+Ecc1OKmCs7pIcsGeH4Kj6qaAacMiPw79NaDJSE76bb8tqaK65efboGtNKAZ0XoPma3HkgKn16rkGFjOdwyjxQmgCYvMby+sH4BlPdsCHiNheMJqAfb5k2CxF9iFmfAZgpLQN+ll+lLQvVylV8HOkPMkvw+UDN1NDtPwEK0yzdTNHX6HntOQRCdx6ffBL1by20amTg+NG8AhcIFaJpJ6PTubU4fl9VARxrq9fRlhwkpDmKpeQW4WNjC7GrmV0GXJHDZYH8U9Coo4aKj\\\\\\\/mBqMaj83f+xiGpJ0PnBeZz6ax51RX7kv+06LxiC169cnvMwD9d0M+01u\\\\\\\/jeHgVQdl2Y0FjWh3+t6p\\\\\\\/jCaS7w9iZlfHQ5fVb0VeBdSSFHn+N13KBskDjoC7Fc1S7+Yjn2I+kMVp59Lk1kgs4mOkcUmtNTFmiGjgEu9mHJ7R7diLwB3Ngw67EnXema5sfkWRX7OvJOFstDYpNaKai\\\\\\\/cQvt81ZtxxanZM8uxB1PiFaW\\\\\\\/vzjZIb4V1Ikeie\\\\\\\/\\\\\\\/Xxqdtrf1yvZXMa0ikGv8Orwxfl6HXVvn0GgHLG8ElhPVbiw99njEIHXyM6kzNi0Uv2MW337w+PzTqovHnLMlOpb32F+v9ZKVCv8PBBxzHMIap+WTuR+zr4G26sazyXDJ61WG7BJm5ZD9yR1M9i0ncswmsHrmYXhbz+nKm8v4Ff0XB\\\\\\\/pRTfL10JCYkVFvRsUzl9dtfO7g1vlrPPE8YTi\\\\\\\/OjPMnV68qRtSAo1QB1UApg5DKaQIDy\\\\\\\/hgeFVzccDOEisOkGZUWoEQVae3AfW+TDieCTJdobYQzo5c\\\\\\\/vjiOYPkklnfqMri7im2xZ8MifbPxdh2CFtBZsvm7zuMu9ncF+RYwgDLftENYxwXmzpP1SjBHVWOFcWhwR85dqzb0LqnTPMKL\\\\\\\/xM1OAIhhjV2gayE7SwpjcLco9usLnnAciAQgmMYMq0U2QPXHRR4nJqadIsRfxELTmCjs6xIDr9LdMcQmzTmhBM2DFycKKpkrfhqcQflTjMAGC2iN1oZKdCy5KjhHsJdjoP5GKzv7GKFknoel2m\\\\\\\/rB6TmiJnjuuvXsXnt2rP+1hhTmdzaNPwFqt1Qbbol7D\\\\\\\/FZmdaJD50kT6AaLmEJphVufcTOAfdArV19BnBYcVegMhuIfAdB7I9YGDk2vUsqaqZ4qthGLsZed\\\\\\\/NSfIX8bTJm6uQPU+BBxVjU\\\\\\\/8Am13B+iGVSgnTFmM8YYFSqBDOcqTVjbn4xclVdj1kXIEAFpS9s5wtoIrimb037RLYbqeePI1k46jrJYlFtOeglNlAkK3G6OehVdbvr74oTdZM8PRowZEiS3ay0jYXJFV3FLKtMfqa9TCPjRiKa0HPt0lFGVvhHvPGuZ7dBCyvjPMjb\\\\\\\/cYCMuzV67tMpjtXv3DlL7zV7PyTueTH7\\\\\\\/o\\\\\\\/0Pexo3oynxZHI7BNQeAmT2yk10of8y4UK2iWGs50HDU5FnxwMu7zBVNQFMWw6M3zgKmFcX7eQg5JsllCJzNnI0D+pMXC6CAzcDHnYWoXK6MAG1zpVBSw1EI6QMzvnLD7qOirKweO8BFsPcT1ZnyrCQ4TmU2lhcQgWFTEfqw5coDN1m51qu3CIxb\\\\\\\/XO152Q5hXmg7w05hTvOSszbG2Y9v5uZKEtcZBDmbk51BjQ9143KmBGun3fdpdAflHYj3bFPVZz3QpLoiJMCVeSN2hhJwoX5liXhZ5u1mkBF2koc+d9SCgAbAscOAHZo\\\\\\\/OqeiIDdzW4njwMGl82iMrMx\\\\\\\/jKbtnSP1RnualgFQPMlYgv+YLid3ECuH8Q23XGqJvYmLcRA28yEnUuC637XESz3+iHKXsK05U4xaKR0yKwqE1tXNn8shpWOTXl7Z8I\\\\\\\/61mLvMZzheSj2l871BcJLE9f1Moo5GaSOaVoTzxjF9lFyawON5lgkQmvlfmy4THyY\\\\\\\/wQhba58smmPnFyHlyX\\\\\\\/qXDCuxYul0xzdIwv7hEOLqW8X2u\\\\\\\/r4XL1CHdBCg9OYnhofB\\\\\\\/CF1oxHid5Zdij\\\\\\\/Fy5048tzlLhIRjy+IW3\\\\\\\/XqWlTd6Ed5Cs2BMq\\\\\\\/hV71HRheTEKb+u3dbV4yHjTp41ZHex+nDHoiK3g1n\\\\\\\/xR0G74xYzMy+BWXjc9EfNlAlcQnw79S+UKy3uZHCLYZ+6W\\\\\\\/QwvMESRANHB7pWsj41q+E037JV2vrWiSk8bZsNNqrL+79p9RBY091GOxSuTSv8cHmU9CIHbFSFGYB1snudpbxqEw3PipE32eVts69gwLPtx0\\\\\\\/8AcqFB85yT7jdp3NGCq1r3SrpQA3lXRGUmGmOiqE7qfEbs5fPgcL0w7tui8lI4cRxreRbNwmwe2ux0VXD95V4+4uTckPJcD7YA7gxESa+MaBnlkvFjba3cyAcSFFiooDLko01a882O7sXvy6nAep07s8HDSsed0BnrkAB1y9OWEsBGy06TY7gH5tNwlnAQjgwtT4j6bdIgE3JbpZVcij0D1Qu1yKW2ma142SB1hrSQfBs6xaNEL\\\\\\\/MFQiYVocycMkzqbLz0rfhGdD0Z8Rww9+BjckklfIIq6ZmWRWqzzhDb0Hj0CddOKoCDWsMoJlLStjzOAo60lne5f7UxQxGX6u3ohOkDzvOt6oPQZLDrvkdtizpf1HsLYtIQQ\\\\\\\/X78NrOvvjhdaNHLdu8dMRDTTyBPCQ62IEDsr2he\\\\\\\/Zlfwa4aqLvOS2x93O3a\\\\\\\/DZOlsdk\\\\\\\/DU5bcQ\\\\\\\/3GbTh3fWQnF1SwXNQu66IDHGEEC2W+mVUI4++klTz04ykP1tPzDuNKaXKf8oNZimzJIORBuCQ0REjhPkXwsvVZh+XQQHQn5zV4Ij8hc9F4jMlcz9nVJ0pxEnXzrEiPIK1uodpashmu8s\\\\\\\/LomxVtJtFogjkiFsyl3KH6Tp2NWcBtw7ck21cEm\\\\\\\/6qd7xPLK0NPR\\\\\\\/5I612ScLK+9OW++cu4y10v4kz\\\\\\\/01rkB7EbOp5ZtuPf7uLNMy0Nc4xcCMiGm1QkGxxSx4WOyBdNJ19fI\\\\\\\/oj4+78BrTLK0+7gl66EBP1O\\\\\\\/wX\\\\\\\/APenL2W+kP6AzTZ9RAUs3jxEnCv1uecE9G5e\\\\\\\/B3EjmRjk8Tb380FBmV9cQ8QsaVQC5PA+9vWJK54STdnvV8KIiLnezyG9mnFd338A91LZq3CEi148G1HuHRmYcxaAiJ+5kFrM6AdGhidTszeZfviW20P+ExfIaOIOUmjZH\\\\\\\/p+WsY+XazYD6JRdQ22SvfAfQMUaRuvDxQFraZLX2Y\\\\\\\/3ROT5OF0g8fzJtmOe0Ne3hr2J3ZKtUwHMp6ogfhjHUgoPe9iXjNZv5wwt4iLr1EoRRHeATljYilZGiaPdUSvdKuEqJWCXH+g7SVQxj4822H1ijBy\\\\\\\/JznhxA\\\\\\\/TuYLUDeIsssYXDeNYROoyOBWPACP2UhC\\\\\\\/WntHyH24vBgSAJJlVxpsNzQdsoPivxYVe+rW+v7walWMq\\\\\\\/ZlWtaaBnSXUSpGnm+EBqaDJK8y+zTkF1tNweUOvn6i4FOEcXkZ1B8oCES6zZfG13vFd9h3WxF0PiGs28ZKaSWJent5eCHjcKYDTnvv6Ij8OsvA7yNcdPzuaoIKMJoMTw6poZHCmgzF77xr3Xy4bFBhZ5\\\\\\\/Wqo47+eukS\\\\\\\/SN5aymREbc7ajsT6wwAg2LLX4VlSSonbbIDqV62mo0iFCFRaHagVXPUbA+oQECkCuhUvxoKtQU8pBseVDcWZZ4Z\\\\\\\/7NypHs4r9qXB+4swFZpRl2DWhNORY8hhwa\\\\\\\/08xdIuy9baKByzrHbi+OpEKivaFHaF9kS49tFI7eGCKVxH3o3Erw4S1oJZdLpi\\\\\\\/t6WEAVRlwH6k+o9b6lfURbE4O3hVpiW3\\\\\\\/rE0a12HP0DXk8tgWNA1fpSh9anaCKtLfU+c6O8P5wpGA8WGWdqQ\\\\\\\/j21\\\\\\\/OmjeYnG1WQjllbL\\\\\\\/aYNvTW5Fqav\\\\\\\/XwM2E7LImiVgs7lwiuU2+xOuQl3GJoUWr3dAB1FvroHYhwZ\\\\\\\/dVEvr2+a0\\\\\\\/+dKoTwKsPDG5jYfGj71vhJUhfIQ6zUgtAC8kMKT3jYZpfrS6DTS9Jqo3rE9hpCiIccnD0e\\\\\\\/AS4lIXj1HmQRs1LxptpEAGoKozzS3aFa\\\\\\\/5T9M83TGPKoflBwzyMAIKWQpK03HEl\\\\\\\/qtkogtQa7lDcZkkVmptHwhY8fTYpheM0QvcPNHqWCa+Kot3PHdlZuC+gdvoSyBYUDKgr3pkhIjcYSg00oHGUc4eTk0S9wFkNMq7H1iwc+552l0MQQ24Sz6kQ0BLFqbM22JZtf+H8YgclLlPN2WEf6302sC9xGDLSNK0cyvBS4Ej2R4Muz93w4zcuX2KTyBZAbLvpmxKVsrtxiik8rJGgUlJ+CczWty6LflHPWpU8pI7Bu0O0uKYpw7gXwJeP\\\\\\\/W1dvDGCqO0yOQOOSK2tAi\\\\\\\/StDapVqTKo1LW2H6C8UR6FnlSyDHDBGoUKVEsmcOdXoFgLcp7GD4RfiB46flouopWNVijwpp1scZGiJf3hP+Tqqwpo5LIylo6yxgRElC0THf6yCeAy4k2sZ97yWOVolWclGcwC6TSbCHHrQNr\\\\\\\/k13boBFoZG9OPmecz9wm7lDeBKTPeRyVS3qhQASjE3V1rivecl62jusolHrtZ632d9YkoaSZtCDrikvwX6\\\\\\\/kWS+y4CbdBvxZSC43235BS9Hct0Bm7qglO+LQgnFNrCJuJUjCOJIMnqh1ta7evNoUF88YtHrGUg3FgbD9xemsQ6oo1DOKjn8gmWuicA3FMUZ\\\\\\\/u\\\\\\\/XfNEQvd9qSnUtCgxA\\\\\\\/o6lz59FAsu3J5tncY7nynzi6Eo6FIsZyp7QMcTqOC+WTds31HVaDvrWHTP3I7zKvsRrWqJqeSiQCS63BUV3Pb\\\\\\\/1GPpCYtkOWs+OiiwiS6gzZsJ9Nzy5+dcWOqpBj\\\\\\\/1TiWv1c93aP90pwKTQHapdlWiLDvIbZyASZ85ToDduCeO1lafJOcHGxnno5cB2HfH4OdQlP2SyeONEWe\\\\\\\/rmxr1XEzFY7Fwz0cmG436d74oPVrPD16oXeuTQP\\\\\\\/E6KFhXWp15yp1KOjFQ4IlyNlrhJUQMixqRmahVpyyRzgJfT9QcfjxzMfMBmeokReYZSpDl7MNPndCAGTxpzwoEd3eYhD68QWr0XJtF7VSDLTSI2bNANgxUfUsJrIPJBtypTB0dRG+tZ2MKnaxGwzpzO2Fr\\\\\\\/za19+b49yqyJS9aREz07+j7YDZgCQEOnjCAXgYdHmif3sCT0iKb5O7fptFpbCptWqTuXYGPqAIqmoG9VZ7mDnsnwJNULkMsqdBPm+h\\\\\\\/51XKJTGim2wZ1XRFDsot0p0E5bj1gWJ788HzOYcvIlSLc80VL90n17HQH0gIbrPQfNqmLZDHm4XDskrVbuygUyZMeYSDJ5IE8+U8QB5Sy7xokJkP9HOhfrKWiatiGLGU5dikREnNuM1F6EnM6lAHlh5CkM0fBNCg6fLrhfVyHBFrpKmKtVdaLpPOxh8xKaW9e7Oy\\\\\\\/v\\\\\\\/LX9yX+tXiwVxhxWRcBS8PFw5HWwJ8touvfip7JiMnq8qAqoTVpXlh0qbaIaq0MdLMpLmW8Hfr3kAPlvBitIxgyg\\\\\\\/Fk25A6towpoz5v8d\\\\\\\/YL9SrKKJLJftp4V7Gudrwt1P5kC2enwbkKEgk6p7170PTITJ2oL4SJVKHQQih6NMneYdLjgvIonAfmYLC4CaVn2W1H\\\\\\\/ddbZpgFWXxritEvCW9PB8BtWi8QUuLQOlTVqgn18gOgtDcsHvnghAc4hEm08hbbdOSYC6KGK6s0FxFmtt\\\\\\\/7\\\\\\\/Cz9PKaLQd1Ui8hgxOKuGxa93JF5qsv9\\\\\\\/9q2dEDvcG3UutnnwFwrsy8FiMSQPsluJEfD64EGOxFpV4bqBLbmW6U4Z\\\\\\\/dTBd65a8LoV+9RRCEppj1Q5RyiDeXORxXG1bTsC1fViGeMjYREssSpuozbP+9RNs1Zkr7up1ANEVmewwPi0ICW6efgJtK4mQ5ZJes8yw17zv5wJaSUEryar1bkZjjLjxh4c8Cb+GehP8Iw8SZWyCDy51qa49lpg3dsaGgFgT6AI3Iof9mSYX+92bDrxIoVqI8ggaMFVe9pf7mNcvdJjNVhWdGiKITlUe6ixiFSVTmyvwmV5ikb4qnvN+6UtyT1QKsiRIzh3KTCQ+8LYOws6frAhXMbygLbGux2ngaVHgvHuh1Zmcgylakoimbb2R7kATsCNQ9QRkdfLF9vJCTvIS7I20+Py7GE0\\\\\\\/kF5EFP47fk+GhjAZUtQwgZ7W4uOqS7UJPtYZAZR8TU7tMjSUPflF385Uu6Su0XWQexINkUAefv9Bec5iLI+5sRCz+T7NVBWCx\\\\\\\/y+4Sgfap9NlR256y2DxY6eIAotGJCLIP\\\\\\\/22lkpmvtWxPm7GH7fGS3czCwrPE2poA9bMBSKt2ual2CDpkzY5BLqHweysLpIz+w+NN11uqI17fU+fdGO\\\\\\\/nv+7fXB4K2mzOBVinOP2b09rB5aAgwEJdlZ7QX+uxS7mAsTk+P8dWvXZWs1NG6f8Lcw50xd67ypzz6i0z+ZO0zpiBK83PbrnweJ2biwda07oqeI5BaE8O1DNUrGtY2YL7gD0v4uzsV3iPwo5Em4Vg3\\\\\\\/6O57zN3+sgwU\\\\\\\/6Fnw\\\\\\\/ry3H4gOeNF9mtkTJ8Xlqjqywwqi7PSPHb2TY3nXQ5axmPavLKzKbdBNI1N1RvcneZLOGiREJBXnsIFvH6mpYVARi6U8ON\\\\\\\/+LWwygCcm8FFxUXZEsNcnBSiCeSCbdi8waiBp9c0R682lQri0rbdWHhcKVvJYr8ADwBGTcn8YFvBpl4YbI3yMAufA2MUObeB9z4EksztAmh8Alh5KViKl8uC0vxqkpuWlbnzSaTI5wM4xEjXvX8tPw+EHtD7q6W3AxKi\\\\\\\/np8ZuCUt+5Lxl3beJt1v6GfzwDXMMm2p+ydVM+9pCf6\\\\\\\/vB8VdtGQo9boMHoncB4Onp47iyawn6K4hcJ5\\\\\\\/J4dUhKpJuvfv\\\\\\\/9GGSShts+d\\\\\\\/KEF\\\\\\\/392Jn9ge7hXSr8CSV2L9yTTVKp6ur8TMhYwSEt05rLSikN0LFJE4bJ\\\\\\\/qOA8uy4gyi3NIc+sbsWVrbANwFWFD0nUnje4GHoihdTIOFBu0AJz1RIBpRxFdXVGOZYkoHm87DEvxq3i5FESuD6dy+KklOwLJapiJMUhxGXoKOnTFz1kFSGTt4EbTdPXSA1twZ8lItTqqf\\\\\\\/Hd1BnjqpHtJB5io9RxgyL4yVXXC008YLKJyAACrZ70v6yibNG+weaMWrVQmW7IqwUlXRSXrY3LHMkrJ9Rex\\\\\\\/DzQIdMUGICySBf\\\\\\\/+rPgVh4CvkTnG+NWoTRccAd5Z7QpsF\\\\\\\/DWhmb2\\\\\\\/f6MC+l4t7SRWZ8DvNyISBvmjgTi1+u8Y0c7VYiMv1NSuaPSEsKdlk0swRfK5JX7X4FJ7bkXqXTlxaSAqITq2NVvlH3cNG3nDkhKGvo4AIqMt9qZ9+b40mr\\\\\\\/U2xwF3QRARfTNA5l7hOkPp8kZKEV1fXldlveXW6WmkR0SBdP9XaVe0bLiJSgsEz0iq3WHvwYgVC+vC1ZdczlpoAUdwKdL1kWcxKNkl6LUoP9KQ99ZbadlIrZcxi7tfDlYYFwbo3yYrLP2KY3XBeva531tLF+1xDuLcUthRlkFod+KK3Zk0kO0qZMLJw8YsuZ\\\\\\\/EbBGgRRu6PUO9g1OrdSlFzO+uERd0a\\\\\\\/HytSygpseNdUk8hwT2lc5zhmftSO58mbnUi6EQdHKBh+OgoIcIXuiDL2+GbeMF6jhjaBfMsL2yAuSA3WqUDr7EhjaKW21jIvA9R9tKlWC3nDReqSVyFPhpUd7\\\\\\\/+Xrf3ZyODPOcaMU34LGw7miOxnQ2klIAMdmL9g7qdwYAVwg7v3zBZpGFwf8nUupjFuzGJQyy5rXS344fQTJA4mpCnenyxKIIQoWmgLgsLH4xCHMRmGAS1khh\\\\\\\/YRCYaWXMYR3ZFAlyjJ4uKuh5e2NtrMb4AF4LefPldgkOpjweqFPDf81oUKzcAZw8E7wLEp76zn1m+LUNWOUSqbrwVmALizSAPOsWm+P1QM86pLRvumkwqQi1X7+PhCw9Zl81AXtUP1yzP0r++fCffhjq2GBDmGZq+lR2qEsd5MEovrmnf2dIYiEslqG42nA\\\\\\\/VZPSI7TVhcDRi2V2J+bubJdAGst8NwDAMLFVt66N3i7wBt6R7JgTdAEqHummec0uI5enEJkuXUgBEU30gdk3w04Ijfyhc\\\\\\\/wV8L5\\\\\\\/6hN1S3P7yW7JLYZWi2Pb8X3kVfNKS0+grLF49Uwe\\\\\\\/dBUBCIl5++zTeVeigTal1pnlSR9OXSi0hCmAdpiN+rtNWczwa7E7eOrqOAvOSEAjVkhcaq8grw88Sw2ruz2DiKNUR24mHyi+XLd6QeN5ogNJBzgbN0HYGA88Zt1dKOGzgZHd3HyWDQf2qSyO7\\\\\\\/Fs3L6L9rFEKdZwRVbgGzx13+E4lK\\\\\\\/XzUizBVpgZacm2la5UiTNwdUZRM7bIajSP45L70sdkqUrP2uZMu0pjgdzq0z7RjmAukfdTpaiF+6QjMxLL4bfC3jxJI6x0CmE6lbQt2MkT6oQXEfwqItjDB5FMPF1EQiRHug91VW78mp0lqg8Q7hm0wlP8P2kaa47HONEmcIXKGPRrlmr12l+wvZeVRC2OscaQGQNdxrHwSDKFXqAW9Kx7Wf6klW+UAHdI0IcHTizEpdswy2wPfGm2eAcW3wuxQa8YxBkP6+G5FYOx5TDDfOr\\\\\\\/PEfss33585mipv4SeAfo7i51eX3kP9s+PxkjinPL6Qd7Mc0QxzTHlHvSVkT7vMWf8LUK88CYwnnp5mTnsp9SOmgB8BgnOF5l\\\\\\\/Pa10QlqRBAvTwAqe517RgkMRup6dqI86\\\\\\\/km6o25rHTeWTAzJaAZLNpd+vSrfxKZfYwI5xOqI3lztKv3oLm08ZxXgMQJtf\\\\\\\/7l\\\\\\\/VtvbkA3VWCButiwVzeuTOguXRAP9t4+h+bYbbv9YpjjmJ+LWI2TlMyzCGDAAX+DkTqgJpuwWZyg40szJP1Ls9kcvSncfd2h\\\\\\\/VjkXz9ebTSgSDqYVfHT8CPQEczdcfNcJdxXIvekrrFjGSCx5Ct6LRb355MHa+nGd9fwCj\\\\\\\/C2Yd3annWSVahMALKDpNJ1t1KCbBAFkl0QCO\\\\\\\/Igx76lfWCxsoR6qZSLx+1b8FK3ogZebIOJ3hrJ6gdxt0OAEIlVnUrJP6K+jU4ZNFtqOArpc80SjxK632t5CKwenah4BNIdpfw6RRQXFBClxoB5XRc8FZpCB5j9DDHr26vYawqGBAnBZsye6IiLQazGOCxPHtE0269mqR5e7sWY8obWAkTdjfNZsjX0odYkDTVRmmCCjyebenI+7mgTJdZSVSGsM87ENfxvGEEecXppZdmvJCnmw5JQZiiRN16IR74zquA+vo8eT4UokMyac104AYez45Xddc2Xm10BTcEMKWuj9v0jr\\\\\\\/vXsyhMcJ3JXj4o1JrH5THZR14kwXR5rkzE4SeRB9Hp5nQHx2vobl9cwpF+5ccp9IY4YnLwqsqRzVnTbwQLvz4iMxRAsLxmTVTrbGp4R\\\\\\\/rC04qujMW+HHRJaBRWP23sZ13gSRV11AXTsG3WmwRpJtXQHVeGi6EIabzta7ZHZAyDuevNGGFk64o6wtm2DTFN3U\\\\\\\/ENCf34TkPxhNhA67Ni+HjTKcVZpI6stvGQwtOd9Zw\\\\\\\/UJskPfyO8FE+H8t4Zm7eIYWibxLHFlT50z+JIEE3iuHK+qx3mXyG5Vr4RteemxTgR5TVhNmFmHBdrhcJKuhA1nDhfPLARKyvHgOHgKjjX8s8tLQ7a\\\\\\\/cIAyRg9h5g1opquwXSECN5ZB+iufwJ2H5TbY7gqEdhcBeebxAR0ctsULB5quoQd6fuAPJDjU0VS1WigFeVm0AhTJwEE688t3jS2F0a635a\\\\\\\/\\\\\\\/iTzMfmRMofTFr\\\\\\\/33Pp5FfyimlZTdtyE9LcvNpVZqevkRwB5ddsIbkU6xmiqq6QXjAiXAp3ixQEcEPigwDqnJLBKY4H8BBGoM+8EYxgtcjSaroflSkqfw4ZUipmx5BmiwwgSR\\\\\\\/EzduoC77Et0WtyRTShxV1ck3tuzlQ2pKVLwkXpUuOprXK0OA1gYWns\\\\\\\/o+jcWsNbp\\\\\\\/RIJCR9TAkuYa4+o7GJu+MM\\\\\\\/6m0QnpMC1jiP4bLMNuFBQ+XDKjQCZDd0LZCTpmX8NLwgRQjNsE3rihxGLVgnANVuE5fTjBFm09hsHaTMiwyuW0+Z\\\\\\\/jvO3ROkb4kaVZSbQSCJi0ci8fJ80igMzo3WLwa\\\\\\\/2EukWtcInCQnomcOsrypENPLl3umrkLSmlYLZx8pvra8bM6koiSh6wLdMSu0ygW9K2AH8V6y8sFwapwee3CKthltot9VgrDdOpQ9z1GG4N7NhDaivaxw9U2lSJahaNlwVzU50aIOnX9v3kjwUsgJCw1P\\\\\\\/fJXw8ypFU+QywY62Kt2Ydtxrdd0XQAgfY8v5TlQA6MYhyxQu0FC4mpnVAu5Tec1uvxvCSmyXmQEEtD493mZbgrHHh3edUysC13rHMgDVIGJtWMyRqtTnBi5gmaHfGGSAxUweSnCX5AeRlW2fbRdc3Z6BZiqx2F8dJCBej2CSdK4IIEsqM+\\\\\\\/QQmeIiYYwK4NqsqydOh3WwKzUT6a+sEQ5b35\\\\\\\/NzoSIhCuY9G0VkqjTg39ccEoUdHi61q4kZo2Dd1RBRPqu5GWFEIGRCWsDolzKEmTDCBLCFenCvOH3hGAK4e5IfH8VQETfFTUP5YIRSbYF3GzhV9DbDoJ0Pd\\\\\\\/pUYHqMYoaBsao6XdfUO8\\\\\\\/zcX0qcNCC8DbB2nFZt4T0EB3kLePxYAn3v8v64K2Iwx9NnR3Nkxdva8Gnx5C4TGOpcWB4wiSwH6wS8o44o7821y9VKwsuLTaDarHqopyv2nuUGyr47P8k6GByfeKC+LkWVYSUxso53ZFubx\\\\\\\/Ff48en5piHGr7RRZLDxBkx2i3v2w6VKcnxDcptWhSC4xEHgo9O17aj3y\\\\\\\/2\\\\\\\/rt2e9EmR6wXqpK0uX35Tf\\\\\\\/Oq\\\\\\\/RgVOtOCtPifBYFXvSSb+h5t0Ivl2ytVMhfvJo9yA1Gi4SE7EseCp256afqkCEiU5KeKjGXbyT0qfghs4KbGQKI3RKoUlAxQmkqXSOzc91ridTLnqeF\\\\\\\/PBbqntQC0QtENz8+u3sumKnUd+T5f7UYupD\\\\\\\/dtWPIW3OY+uP89gKV\\\\\\\/VyRgnwYtG0cPgPwU8tMmIKsfq1q4W16bcl2LSNkMdW4VwzaTYjf\\\\\\\/AQtzCy3nfyt1sgI5SuvenVEDFAMzXnAucdwWljFQ7YgX3dorNU\\\\\\\/+C85l8a2c04tr3d79IqKtJxCE+yrMRs7Lu+dVQy1zAUasNpxwzQKNisdagz79aJjUhlHEDQ5wvlftxcKZykA4owzy+7dVO7adw0sVxgVvEcHPZ0TWmDauU12CbHbDiaWWmvnaj7fnvD14NoXvxlhMRUmVdFj6yFFSGj20h9DHiGtshQrPg5kQ7X76hcBa81IX76P26fr2kFOF+t6ZH4kSIXGNHDbKckWrjPJNgDiTX4dtBEf9JXe4HRnviGIKlOl2W\\\\\\\/iQLoRZ0UQEceR40LjLKhq+9c7WuLe2lIErOD63+Zn2HpB1jZL\\\\\\\/c79kD2iDJ6voA3LSnE\\\\\\\/HMYjZLe8eTKMjrPGmuM+Joh9NYHAF4LqVxCSrkXuj1CEHTx95JUxUgYOO+l\\\\\\\/iBZFIPJFu0HgVCLV50TDudvcyqEAJsHvamXahotaxf+pPOW6HEfgJWSXn1lnmxeuQRy0JhV8SCj2X\\\\\\\/X\\\\\\\/F0gO\\\\\\\/ijFYqSOFOhjoTq0qcREB4saY2r\\\\\\\/I63OfXymkgl\\\\\\\/GsxWurfyUjL3OaXKv0xS8LKQEN9AiPDc7o5lUyXFgg1Kj8S7JXGhUn1uyhd6FTqTKPBCd\\\\\\\/8G1YpcQmV8VpoekxeZc4e0tvs8JQ0\\\\\\\/CzufsVCOBsUbQBLgRsKKt6t6dDNg3krk6w3ghxkh0OItna5mh4YeFjttTEiXlz6W94M0Lmh\\\\\\\/o7hAEBHAVoxgB2CBuU0p30KAd595XEZpuMGsauf2flUQIQVMEoqU0s18A09uSG6vwpLlCCUdOnDxbMf0V6cWn4CB7d1qdiH9+VNZ\\\\\\\/hVh0L3xOgKD3aZ7+SIAmNzYRbMBc1XdiHkq1303zgmyb4kKPUp\\\\\\\/n\\\\\\\/jAT8i1cyOh4Aq8YyAwAK0IA2K8LlyjqxNhWXlOLrwL8lbLjfXQADZU2qcQ6rccWhrtymRc39OE7vzdaDIT3M2JncRKUADrxxxlpDJg0sw4lsPO5I3vMHTXTk8CzDIgDtjzTttQxqkrJz5eNE+lBL3cS7PYrYJEEBlTSEZiretJ3mpnDFFzrcoSlHXTwvhlq1Cqsnsa3RbWUHBwaAzZ8JI5pQaljh527ZVwnvTiYVGKKGRkeQSw9+bYKZgU5R02EdJ1yMv2s2QGdhfkFpuDBgxBHDUNIcAeMr15aJN4QTTSutV1QHNEtWKpG65lecumdMGgjIVCr1pI7IwH776daWoBDUqsQEVy7v3KGjtxrIRxZhR+hXdqlRzB5cIzitnEZf+KdXx1qIsZhdRP9NN8uBGmOtJDQxzuWZmH9FDnJ+9zRQnorIkNB6d7zQApAA5yycUdiprsofK1lhy8vs2ebP5EtcVWoofDCo7n9sk1d1Dtv14etilOA4FXVCtt2BlGezp35W9on0GD8uexmIjczWCNzvfzQJ6knrSWxbZcNg4+\\\\\\\/jhqOUB6mibiZCMZOsjmB44udf4wMLFWiNkhfdEHcINzj6e+BIBgNAb5G3qG9XLbhvHYEDv3I9uXTWp6NgjYWH40h5PrmvgqP\\\\\\\/Dn+ZVM4jttU0rVORw8n9zhOadrcOTL+APgv6BnaoC4o8Z6sAf0fDRXwC4Ec2wlhKPc2hemSJ7i2FP6CAUkjzHYV8eOTqH2jPjEkemEfXx6HfASOQsJh3fQTr86XZl7uk6k9xEILzYHkfRcOM4RKrHgVpeV9WOquQ9tv26ZX9XLvvjC59eckne7Bn3wqDcMB2\\\\\\\/d3t62IfITnani0r0IbCQzGJJEl9y+vB\\\\\\\/4LK\\\\\\\/Hc0WBmqdzB50168HYlsOeIcv9e+SQ354oPYqpAJ20ccPro8hnxQ7wkJi1\\\\\\\/98Uf9fA2p0j1alg59lfHpx2Egcfj+sZRlnzjmFe4hokXk+L+jQ1g5CRmI4h90LlOIOjeoc91ppUvbZx+ee2YD6ck3JwiMhKJSkR37oeNSpgPE8xOjOptHw9F0RLBrHT7tOGjrPanQhJPuMG4qcmLAIoEAjiFU\\\\\\\/NFYPKXRWnbd6KeuKWPQtCG0rONahvXA0+WAyb2ilCEjfbHJo3ENLvEJD0I5wCltl1iG7oyARwPwpbDDS6l5LgAoyP0jG+DBAzioBLG\\\\\\\/HYRCMqkKO7pIXe71vrYALTW3R\\\\\\\/vTAPDxjq0CL\\\\\\\/k4Wt8kBrwJ1kOke69j9XJF5G\\\\\\\/BlWrrR\\\\\\\/dB1alrLH5b2LONd5XYg+tCK4YmkeilankbipLJu3CDKrtAVcWov8yJ1zZ\\\\\\\/2p8GR00SZf0j39d12eDLqRD2wxfOLCP2rBpiuWO\\\\\\\/lZYYxYdpxKRWAG6PRVrdkaglpqbFcsvCGvv73yDLGYET4ePqunIqZb7zv5JLVXWpmskiKcxzwCaPWjSgDa6QoVco7jn4l7A8KCOsh2CWfgSF+Ajw9ERDJx7ZF5Yc73+CWISkB08q+1SQ1wIr5IOg1Ldhq1uyC\\\\\\\/ncn+1GZE4mAJC3HO3ThW2kkDfsQdSnELbqSeSqn90xdBUhTVGaYMwuSELIHtCtwih+t2n7cE+oKDAhXwSXjTNtEXTgI3LJWUimqiWf7J5CJ1K\\\\\\\/C0QWRH6DCfZ7bh5UAO64x9mQAEdgdHaldsZPTAsFPPmynAPqNAUqunZFBdlgcHDhvQB4YAWzXu9qYI+IevxGjGLD0cTth52NkNHJDN28k3XTnrhPO\\\\\\\/S6IoBzXIWHUNHKrGLA5PCxOyDIz\\\\\\\/tcRA1YeGwLwlBYe1DF1yj39tqzCkMA4PqgIFwYy0TV1+cj8TIu+vkYrgrBS6Yzo6rmV2NulWwM9VcONLpNMdylP+0ZCLQgUo8gd8OPESap4rwF1pnfdbplHddfp++Skc1IuwzKMxAiW2CwbA3JtLcc5pXZl7LyPftbfdjB5AxejiqMWYxUHWWTNOZjYgZsf3KrB4hhtBf3lNhHMZEiPG1CMmP+csktY03dnopbAOhhFg17jMUvFUB079U0oxbsJrGoATsDkFnN2GsFipmT7W\\\\\\\/4m2fDIDG6ulJ99A6+9bUPe41qIFYd\\\\\\\/asyAdmI4\\\\\\\/XReG3dbXt9ocKjrjeLRQLIQ2XN1sbKHQsslygs+jUPAYBNtR0V5636SO2qMsz3woEKerZGt1OFQSQONQn8awOlcQWMz0QqYqQQxHVYb12gLKBxaHAbgHZiAYOxUNYOsnXGGNKpHDvtZTC29lsK7enYDagIpOSU1ggczMH\\\\\\\/e8RwvwTDGexFCy4J0VgbVAvXREf6Y\\\\\\\/q80fEnWID8LffkWji9Wx5XMqXxFrsMSToAjmkkcd2\\\\\\\/Tn2DO14CLrrf8cBRLeQEAG84hzXgXCoj5V45O909rQDNq\\\\\\\/qMvlnou3rLXmhJpWTp+zXzBj93fjjowsncYOgkyc18sw6jOGKOW6cl4hG0Mnr8giUb6\\\\\\\/LvUJhZPv3vsB5MfOsnsTObRH8mxSpCtZHzVgDKwOjWoZYZb3Sp2\\\\\\\/3ZiJK1MfPHG6YHSoNUVqF7tdlEhUb2bXC8p0bRHr2Z1vjcgq2M0pj7KPZNOFOyT53gOYMHRW5SLolL2hwZ4GIZ1mXJReIRVcV83Xbxdf5CL8Ry0BK4nT\\\\\\\/0JR3YKFQsiSVVRAZvXlk\\\\\\\/AjHWL+\\\\\\\/aeqBE+Gh4wbwRnc263zdkVpqR1+2NNn\\\\\\\/QNFnPJ0ZCk7dGwRnGvy1pHy7zY9SABOJnxkflK2yZ1FUmnzF0a6JycnZtnInp5tWvrLYFeBHCr7L5OYAjqxmpx6HcMB3jQTIn1PMXweti6JqaJOgq2GtJUh2d+fy7fhIRcChMQsfaTorxoIrBUBQVE8Y44E0AhFgPkILKIUZQOxeu2xbYTJ9A3JYtMBHl4V3LeyN3g7+PJI5yloiC2iL54978Y26XfVtLo4N+05jjbIf\\\\\\\/Yok8BMhgW3NQRcTbcDOe04OY36cw1sk2l7DwGiO07fxi5TeKdK\\\\\\\/2r+uUHWJVj5FJL5sWZ9brBFLz+z2CYMJnX39upTCyiM\\\\\\\/Axh1z8tGFizw\\\\\\\/BhPGL8MAKWf5Beal4D3DmPsN0b817lUzHJUwyA\\\\\\\/JduJlSbVZNvfOemvUi5k2q+f1IGWksigYX\\\\\\\/aGaVQjITXa\\\\\\\/0sP5iDNjsYQ7xubwbS8+7g0HHDfnqCJZaWndANJNNrIdaEi6Wr0UYSE+QEFBY7a0AArV0mr2QUT2CTTIVQWjDWc\\\\\\\/wGQhnK5fiSyvznwzZyk5cqCu23PhEsiZKwoEl9Ri99gg89o0vZ9+VmZEvAll4rxIQaLStkQaDkoILjIdgJDtnjVgKGPjW\\\\\\\/av74au2obOOsm18p\\\\\\\/6gJ1nc9nRxpY2cLgZq8eHxHjK7P3fYB0Y7gZ4SLK+8l1g2BsDq6FSPJBaKuwlre8u5yL2G4z3jBHY04wyLH2Beq0dym945XAQeadn0bAgZ6F8mVPt5w5Ae1nwHbwPK0Q\\\\\\\/2D9dUktECqU7GDTCk2MOSqd\\\\\\\/Vt5yOypsJBsnck5hpdI5UtiUmeteKiFSr6DR64+K6lBTVFQ0s58OftKlT5epmiWaj2yor8U1OEoLYAIYABLNfe9vnwgW0RVWv4HJrKGFgtprb\\\\\\\/vYxAUUb41tHyCip6ToknhlfCBXXF5m+bwKEqyx4KbmrsukUxt9wzDzDyIJ8hesi2J3wRumK7AzZhYkRQq0ban56F1F679J1JdjcNeRT5ryTnIkAl0ais55XaO63KRfu5uPpXAKL0tkaRuQFJ1\\\\\\\/TR5aqArN7qcCYcn6rmPl4TLWUTa5WLq9CH43DFNGQeuk2IxgpnYLd81rQYG8gvwNHef0P7KgZXI+BYu+XrBcKU091aex8FBuYMVsHV32FAJgkxwBdPjQPmV\\\\\\\/JvnVe6G2LUR2c52HR49z6fFwoNIoV4UmbYf8JUSUQNEYabKgLYsDlaMiaHjezxhIZwZSy+Ao4Qt2GPELkRhE0K958jEYfUdpnJfbDEd+IR2p3GjlJMuEtlBDX72vWmOVPo790jMcTpZuY7qasmyhwABYkjDuVmINBIG6Way8TKbY\\\\\\\/ovsBT+\\\\\\\/P\\\\\\\/ntbK7dVRslQeg7l9YXrq5Q5mhtf4CyYHqr71nbBBTsLHVrQ1tgCMLTNlNrD3ejlaNuZsaalT9Mw\\\\\\\/ma09A4hjqpg2G2ldwEoJCj1MKQPrtJ9PLnwt9yDON7DEUd9aIzZ2oTEQVSy9fMcYrL0ANbrNrWPJqSHRr7RmP+Rk6BaKA\\\\\\\/yBQLG24RyZdBwm9bqLuCGrNg+nCgVTiBPiqXwMqAzdCScCW+3dm7Wrhoq0wIp6I\\\\\\\/0usHzSQ4X+9PnNtIJUcNBadmfvYiOKovDF18ZjhnUMT5Me41tFq+cAfOQovTFwe6MhowWDzjcb\\\\\\\/\\\\\\\/E\\\\\\\/6OtgUPGDwAewQIm+kQO0nP+Rjra2yKCCkHmf5\\\\\\\/64NkkSUK+DlmtB6xb91Eu05p8Rs8LZOY9G690ajj6wo6NfRBjhtX2D9DH3d0sueQf3856ZnvfKW3Hko1luBqtJoYsIT\\\\\\\/ZJRMROxETDzKvEPoUKB0UgfhuFMd+ynG3mN1yAfXnyG4kMG2lFMPOycRt32a9b0TgleDjWfx4hpnNU5kXKL1XwsHNf0d\\\\\\\/ojMX0UwBI\\\\\\\/v+Wd22k6Oo7QENQoBgXuJAty5eRA4+xe1AjKgPNzvhQrLk8svg\\\\\\\/\\\\\\\/DnRsPkMK6JQB5WmgtFnZ41iaBsG4XiGKTPU829d\\\\\\\/+oBSXOnMS3NvqAF0DjK6Y9H3ouhLvlsJAaHNcRWTM49YtUk9gTJwm1nr1\\\\\\\/Iz1i5wVuX91T0jGRQIzW3ZRMUmfGmy5Kw3gAk1OLN1ulLJIemTOIIEBqEAPAH8fJnlH+p5xcsi5jdiCAjvx4gKXO7KPWMAKzV\\\\\\\/2PTV2ZdniUjM9K4MXiDddkvfGIL01IGyfgOEeXIJ6oiaDJwGgqOlNPTn749LAUjSbBAKrppkGswpaXi4fBcRDiRX+pZwv1RF0MrdgEWsrUlXT8766vnP1Jy+hyzozuFdc\\\\\\\/En3Dhbta2OZUFskKcDGK2P0ftL+o\\\\\\\/ryRz0k6Putr9ZXAeMmWbz4OD1fyt3UYxLme9nJiyLDNQEjy72mOGpAIJOvUDcKXDydYVkSBDWBtlDk81MUnwG0sIxI5N1HkgH3cXAWdSTpaZ1zjUPdpbH+ltehB8ROVRhM3ozAhQF9anzm6vS\\\\\\\/iJSE5inE9T6fff4J\\\\\\\/hI4q8qQ41+kDJN1aBSElREpH+VnThtuSNaAnQibmRzzebFqxhw7mguQ4yDyH046mZiBlaLZkYtbAmUr7sQdYpJAhImFJPl4CKdITwY66jLXqtDUtzm5n1fMqeUGmZzfjJ08r7FHM48XHgxTIkukj\\\\\\\/jfk7S4NJcJI1Zegc+nP5VeegIICTPOCSqQ7vmigiHTBh72x7S+8SZIKZemb+87KGkBQ2AtDdtHEAsKYCrb4fO9M0Z6DXZdJK5\\\\\\\/0n4f\\\\\\\/vRuJSc8\\\\\\\/3nRF6E1JvwJ+4atZdXn9kleILFFf6kVEHXDF6aovSvWr3OJQeacXT106dXWiE90toUWYNz9nnPRV006uLd6vhRwazbFk2VAoSJLSLB\\\\\\\/7sOnSBdVJEA1MOboRJLPhjAI8TdQmSVtN0\\\\\\\/IA1yvLnKW3j5e8iUnqXJyGDYIpXvKJr9nR46x8r4MoYaGFQ7xEqlvmlsTET1E7uyKP4OawA0EfOJqZotIYvaGO1QX2ZLpj56EsWsRWZyEsnAQaQ8syynbEiZpSWb7kD74\\\\\\\/h\\\\\\\/hRzlebxgomiLaZnK5EbtaLvMUpRbffQ7+JboYbnhM9TTeXEoIaM\\\\\\\/ZVkbYIcIwbfubLd5cemmTEl4UvIrhFpGXotUFVjRRoSCkRyEWfcxmazm4Lpg6xFqV5DddxBB7PBjtEFDrotDD4f6r7WUcg0fnH4n3w8AJa467zU3fa6xkQXmJ1rneGlU1Fvo1T4GN9xemu5JMSYPD6wG8IScN23FKJaqlFR5JbW0m3QESg0CyOsQLl1s8MZNn0IhLppn\\\\\\\/+UIqf8zYfaTOJHjKwFrWjcreupDSPudIPGKQH1i1VrGG2qS3B82SKz2h6Q5s6FtvXiECAaHPep9zZWH1dQttrt3c5xOVk4BmJs4rmfo05jxCtza3ijroMk9ymGbAELe09HG6sLG2mpkiQ95viHtms1t5PV8nTeqog2vG7XWZfVsrai2qpgbjfZsrr\\\\\\\/h\\\\\\\/T5JPduCkDK6UKCO9uhu52x+krFOyCGu\\\\\\\/+1\\\\\\\/2SL5N5ToGAiBes24iBBfxUKVxpKi9C+i6R7wnN28LQlR5wSqVCm1jcvnJZ\\\\\\\/htyxKiBBoE78W+fgOAv+IUzN57kPPXnJdV\\\\\\\/FzRbILpOrl9kyB0uYoaQJgZhKaILwB2GEMOuME2q3XYRpqBogSBDud3dabx0CX6rkJWFxwsRV66j1g2z92G5\\\\\\\/oaG+3pv9a9ABo4aEUs5EGOnbBcGES+K\\\\\\\/dmD7OBEGocuzpGf6neIB\\\\\\\/jNvqTUBLu3QWlpeJGzJDwnzIzWAkWAGiEu0e\\\\\\\/GZDXgaD6eddMwYmeGZagyXYK0OqEdTnodnM533KBsguKfFxFlKaIFhUAqe6Kx9PH1ZYtlftLAtPYw\\\\\\\/hYkZWlfAXjChpd+VusRo+VWzKXonkFgzJgOI7y\\\\\\\/KbEhZXynPg0+WhxnNLnO4DEbhKc5vQMH9zbnqvbImf9q2HyzHYDFZjzjFy7vSTk932\\\\\\\/RNAxeVHcvExcgBiGqOstMDMF5HvPqWYCh\\\\\\\/ay0ng4IQ5NDeCYMIb2vb\\\\\\\/CkAHeS9x4rLh0kbYkWoiHUntZWX9AigLGBBlhwdGF9x9PhaJaDnm6xrRNiU\\\\\\\/zSpaiIjbKJbastOT6pGuOdK04n39wteDnZXUdslQyMgQUeFMXz\\\\\\\/7W+8zwpB1zuKL0lkwy00N1MmPcYntHOlpmIVTqAWkZoBSK5zEtfgJidcEvX2USVGbJM8VRGHE7keEDLtXM3WWwpl2NphpRsm+wF\\\\\\\/tBZqR\\\\\\\/7YdKbT50w8bDy8Wg1yaj3liUzhV9QWU0+FpvwnIJi5\\\\\\\/nvHNzFEVZtQURhD+ZwxmXC6AXHI2wpuyB8459CSBaNcZFubrrdYZQ2rA9mmz4VcFDkIamI0rgXyfd5RvXzeoBbB\\\\\\\/bRgJUpkL0m10QKdAuNmg\\\\\\\/WtZtYTjl+Dzg6tiyaE1CA7vPoBL5D8NDsYhH\\\\\\\/tpTg7ccFkhYJeHlPD7an8k6JmUOweBj7UClwv3hs55wQWQNaY2\\\\\\\/CuKmMIH2Iry4M2G7gQvbf+GaOXuLCoaEKyzuIhik6Y6+uDhPqChX2ZhAFDQUm+taxmRbsC5LbrOlyfIr70hiYvEsv9eeuxch5C1KzPy8mze3nb1LV\\\\\\\/gmDQfxqV8vd9BwUIJ5yLU7rF1FF0HAW3sq2wwGmemaYf1rXuPYiZdZxhoEI+qWmO31aQRw0MU6QgoOJNxKBQI+OiDGYhW29bd\\\\\\\/LQs9RkW2mZQvGag+9rXAsO4eYc\\\\\\\/Gmy0WTsDTj39w6u34jx6AImCQ3JbFjWPr3kLKvg5OpCxzv3hxbjVL0JgpKG96Zx674IacXp+EDvLAdRbZCaLZdCvfoGvc2i2Q0HIGorG\\\\\\\/QLH0FvIpwmwrWZXfTx9Vw8+EdJ8w0GDeEtrDEMAoLhhco2XNKBj0GKewIu1Si5DEegyy+tkBJHOeomlSgXYn78kAAtIyeLLrZ68DVXGRmclqmuPa2vG3EuB3nMbQQBsTKdudh7AEnFGLtmCgHNAsyCBGPrMFb3C1O1QHia9faf0ckJXlqZ\\\\\\\/YWPdC0Xq55cP+8cWmmRfe+MPmcHY7Nv9FjIE7zeRK3ztY9Pjkucuhw7t4hEmWr40Fb6+E3LmdZWbnUtSZuF43g3OXesd\\\\\\\/vbcG6zrX9M6udSdPSovVd2ZVYxugM51VjRwV+vYdEAqLO5CvA1TEqH\\\\\\\/mvq29LFii4hQVEh1rWeAiDqeyB15ix\\\\\\\/+X056SsxoKkE6ecbQi53\\\\\\\/kB9S1ZfehzBSU6ynmYCJ1wJNjIZbRkVqfwPDBrPshcOdjXUDcsEPq9Dze1SpgcGtVqkNgQ6Wpec\\\\\\\/UG8DcaR0zvgq\\\\\\\/JjZP7SORwwAImOMCo\\\\\\\/9MBjIsgv1MG\\\\\\\/mScaswlve5oShhp5pF+rH5QAwidoJpvg0G4K\\\\\\\/XsY3ccNy7PY7UQijLFPwyjjAvJ65n5ncY3ETSFoqE7LYXdE4xElPRYiYZza7OSkqbf\\\\\\\/\\\\\\\/SpoggookIeFyzsOpdQ3rlU2Z+Bg81NYiNVvkI+LuTSfFLzjIxRrQoLB0lzme6tmeFsqDY\\\\\\\/MqWgKrs332Si49X3PIpXYk9TpbT\\\\\\\/Stfbui9Vji1n0qUQQRafMDa+\\\\\\\/2t3IE9a3jhpxT+GSFAdihZaQ2al8ZPETIRuLEVKeRe\\\\\\\/7tnjohSy7D0mew1bbxFLOal82sPgj8yrwrqyJ8N36Kli4bzLyBmtJvPQWKIiqiNRDuA\\\\\\\/shj9vr3Wn3hrzXR9miS++BHv5pYdqvxNEjAf+6be7Dxjl+1DjFxZ8mC4rDziKcIzwLjozBr5mQmFq0Ztt7lJDUHbfzmauj9zUrEvclmUPrREZPT+WECc51wlFse7arF+7NrVQyNSv0ELpyhWQ1b6lSxDvqwmFxwI0qND7JCVMoQRgrXrkRYX6Tpmfa68+QT6dnOw2WZrSkkQJwjSTRsaGiw+\\\\\\\/cM5wRtRKINPI+TaVvG9Anvx8tMZbaZM2qBiQgkDxta5f3Nk8wpKpYbXgZbGV9130iL9x5ZMHbQG65gWn56Y9a506pGtWan09wBlVxES75DK5yNPYsMuE4ahm1ZOvkTfopkYXiWwpQyupemqovyDYHkh+xQfozY\\\\\\\/MvZAU06SDjhQbozPE0headOo2Uv05dIpaWpC+T9RmSHC5adt7UCqo3LjRrBbMpSMpenG0Et8VOHZzwE+Yix2Be+xgNm2CSui0UifxBLiUHywq66ky2xSRVXBnMsQZ8yrRyLw1SRT6+6PXedA4vECwA5voBMIH4i98tdH7YOkPulSy4g5zYOzZwZ0U5XCm1kVhH8O8NXQ9lBxt9ayQ2ssBewp87Zqw27il3XmAseo+jgHwq1ppRN2CdKFFF+mU39vxXBkwqlSkXC6RsK6x4pOzUL9n5gll2WkN+j1BStMs11oniVsFd+I1\\\\\\\/z1\\\\\\\/zZLmK8HQl7cJ+F\\\\\\\/dcchlcJ7St3iy332W6kRYVi7ExloaHPELvwxIGDdIqSDAkPX1BfDcHHvkv1nrgVluskOj7mPC9JOYT6yqTuXB3t\\\\\\\/pkL1xdI95euk6XNQXNNq6A3yER4nWn\\\\\\\/mS9j9D5yKKKK1q8gMUscLmO4VUOyA0AfPMCAnqIfqv6J2aS\\\\\\\/bamvpShj9RA7o9tebB\\\\\\\/naQsA99SrunwUyLa2XyV8O0j+5ZBMLI1RJrsbtG\\\\\\\/iba2uzcDdBbjmCLPzh7j4ptgRDe19OEZHz65d+vA5ONX1U+7FjqBtLS5\\\\\\\/dDVSHcFUfI4QEG25k6x5S3h+zlK5AXuzqb5O7MSM1Cw7Q24M6ekx9IUs7EQ5sHWVH5BJ3R4aWNHHwbWi8jVtu9gQaEJeAIqqALW4ICmM0MuxtYdrAIqMrYwURspi8I88H5oIXVIejwbX08ehZkpj1RnlFpMoKbDbQx4pnuh8c9edt9WLVZ2Lz97uvxeK9RwN6ZBSXJiLAoWiittw7bTAR7EWrY+0zUW3Es9qqV05WDAdnvPEDqB4hssrM71jfa5bXe\\\\\\\/5mksvjWVbh0m94GdzHqtQojBtnyEQcMf3Z5qPwE3E5j2tvPxyXvUHMWVED8OHAqBbQyXYVbSC+gXZWeAlS\\\\\\\/UsYFMCQNZZaCthDZUfJqVf3\\\\\\\/MOWFQcSiQgOq6gMbamteeCv+vbCEssDz\\\\\\\/OR3JOnCwSGKg9Umwc3mfH0z8Z6SbQ8RkDQoTdiMfpsAoidwW7t39AHUPxh9gRxygH+n5\\\\\\\/tlHV+nVy0zZwZxS4Va2Z+Di0RNf+HsjrDmGPxcYpHGTnLCSRD6ThfUd0QgVsC43gbSQeX9smvoMOYpMMSKwKedOjL\\\\\\\/lxQ+Vfd3up74lLFbXhh+Z8treN1pTrx\\\\\\\/+lrYHQvrb4f01+ZBTv+oH\\\\\\\/lVTpGm1UIYgd\\\\\\\/3RyYTmJO4SjicYve52KrBdawNyA7wWq6HUYzwPv+FdB2bxVK9bdvFforTGGIkRYWdDnMaElYKRxIWm8QWfWUecYnDl0bg4qJwLMO++w1OsVAAWuZOZbIHOrZAdsdI9Fah80XjGl\\\\\\\/8LP\\\\\\\/yA7\\\\\\\/UEYhhfuyooEubMORuiRp3y0TPfIc44U96GGNSxirK1fhfxjOXld0RoMErhQqL7JEF9rwwihmpHqjoGmuHFQ4Lk+f2V5kxloN9ChWMqGvVYs0LSELbjyaAXBTxmJr4cMskiYDCaFnNwmIUaTo\\\\\\\/zRjQuorYgbPulttWVmlzjzvFzl67QPBo0HjTBXbQKjUoHNZIRbDOBbGmaEzkGbZYuvQnieruuOyj5NE9zINP+MD+tgUndbI38imIsbpP6WzXItcPIVBlLlBtqVKlgp30kfulBm7G7bcZwrc7IBIzO4OjxkUILWT9vh5LNZZZyBBJUJu0kfBeGZ5izJNU6ijbsUviqqGIJrtABQTetnrztppKC2vflo3bEhrNyK1rrzylKiy2bnpl7pYkLd8HPL0Fk3lh9QVYYeA\\\\\\\/3vcbnLDoCcJ1V6Lrdzgfcei\\\\\\\/stMwcW1E\\\\\\\/KUrS4C7p9OFpRx4VN98LylDNcbxT0YoBZYWTiyuyKtKueyamGQdGMngc+0+V81frCOUV0HbU1c5zrh\\\\\\\/uF4cT+ztRZeIuKl3zC1Ylfcymn91dEJD3QnqKzk34d+HRx8sXRpQ1NKRTbGLppSELKVqALDeXooNvt0mTveGu1XNQnXeym2or911JeaiVhGhSEpoWFDjv2H6qnueqRrMLi4AajHTzOuD9rCmIHLJWcIJqb6zqcyCM6XK55jzJ\\\\\\\/8yPxtGfiqfglimenD3bltEslR4yToXcgSV01ZRr\\\\\\\/ebRM+J6iJkSTwAnqljjaG4TtYmsaY+oNBWt1468Wxx6\\\\\\\/9TROwdymFEy\\\\\\\/9cNKpl8mBAy1fW4djn8651iNbpKx7wADW+RJJmEs++xJ1IPA1SlfFhXdNL9cDczh9A+uBkoMexItthijIfLfvuHbt4ieLW7WceCsKdI\\\\\\\/UrOClvdGfQbaRQIuTaWG9hnkTLwRsj\\\\\\\/aoDDbCy8eNWgxgOFwNGUNNrG8F\\\\\\\/+oQvQ8geXEOKvZBc0HhzbL6tfNZ9okOGdQ+Re7w5tuZw+ABCVGTdK7ug7qSKK1ylazsP4U7SipljOexTCHx\\\\\\\/HTb7p0SkfHCE9a1DXedWmMnFJ6YLt8m7EuqaaXRvvpHe4fZOD0sqCU5r5FKAsB7ZM\\\\\\\/FbbhssNzwSIMDPbpyw0Xmpp5vtw6aKBgak7eoGyd9BUivjk8UiqvE+oZGafl0LBxumKEba3UxIXmQwZ6l\\\\\\\/V8xRGrpYMzMJD58qtrD7cbk6sD0JGHexmlQHRsA3DCUt4MdNaaBgr31ImLgAPK2NglDfPhKgYFwaYGBwkGJxSSGXzzcuLhlw6muVyzTTo8g6FOjGQE7o7TkCfXcFNODD6jrojKc2OcoZDBLM79j+EYpxwDueeKGLgebYFkDuhlu6kCiAU8SloCgH3rWNLqS1Qxpw556kv0clkvaFex2hzqJK3y9OaLuiCV2VfTkkT8NhFUcMEBbA+hu4IGsirsFua8z77qDmmf5ha5xRTFPbZfrBPem1sKS6vs+KH7KsdKKgLAnbHzytcomVhqubHjA5Al8UJeAHjP4LKhV4pBVlPU1BoJmCfMljckjZUUxBIBIIt5h5663j5GPdwGy42xI2PWM0soTxdonmAuQEYUmnfGg8D\\\\\\\/xeVvuik24XUNHmsd7uq5OQqXJSxw21w739bCXwSrgrqTPl0x4sXaeH7\\\u0022,\\\u0022iv\\\u0022:\\\u00221LY7OnQWs+uB+uIFV\\\\\\\/N0xA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002257d45a288667fbd61a4bd11f9a29743ab091e224f398e87058a62139fd1d3e51\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ONhQP4D3AawmzsjraKSEPpPI00XPizxU09Nsup\\\\\\\/zqNAgW1MdWYCmJ72ye4IymvJpQo6nH6sjT3\\\\\\\/5NBQLprEGIFFhOXz9U3jT8Vx5KKbYKmCHfrILwV6CEaAHaZczhZj05Gr8TfRpNC9LWpp0FDP65HF4Nrqe\\\\\\\/moW7tv2SNSEwHjFAqwHo3kb9AwxIW0O784GEEDbANxHt6gkXAIa93WFEjO\\\\\\\/4UrWRWmevS4OlIq8edcJfNdGcUyRtkVasYMJ3r7lQ98McxAzS81lvQfNPyoPiqiDk6pTEBGeVSBRQQ5GNnUM28kYafRLbJcrzSU6Ymt8BY+678Um7LXEO0o8qnzs25MztxoGuqYr0Kztztzp2PucdvtAbjgvGEdfH74KLkVxVe6iS3HMK2Gi\\\\\\\/HK2unywagvLencibWQen4wCurujdzb5efcI9fXMmo9KhPQ9AYbL6ZjkHDkNHdmS7hdDmZk9AL22PG4ii62JxivSew73C2bypklnbVlYiUpOHS13YaSaK9IMJ232u5Ki34uFXkGZ9LJmRf6v5OjYMl6+jpMq\\\\\\\/abHfs+zl59V9OPUQfCSLlf4CCeNp4oAKXTIinMAAWsM418fzB5LlNa5o78oTliLxzdphtO6wOFnrbhGSf+9ebxGMKj2HHMa92Gto4qP7CFJUxzzV4w9pBPi7ME9R4ZejfUnhdE2k7+ErhKO\\\\\\\/V30F4S0xdG6w8A6etKXL9lKuB9ULV2W9ivRGxK50gElSMWpUpeK\\\\\\\/YjZBkvRZhraTswvyU\\\\\\\/8mFnTjDHiKoyrs5EIl1XqueOyrnB48YfYD77sg\\\\\\\/swmTkgj+qDhFa22r0QikyLKchF4D+VcLODordxHnPXHZwCCNimMMXy384Y1b8hZX61QunC5IhNtU0WkCivGg+4mgLYuu8O+ukGWlAgRZ0mACvdu8w+uCxT3rZuCa9tW4ACU9UzetzYQVxcrDEkSmTaZVIkZCo8SUOoPWBCkv643F5zFftRBIvkagwnPWPqXCSvWVUnOFzyvbCdS\\\\\\\/JqV0Eq9cpa5fTlHGKADlx+EciXInVQb9m8YwZVGvN+L67QDcaTy8xTeQc5hor3zHQ8WL9MTi7nBHzcsYR4+zuAr6jGXag\\\\\\\/VwRVA\\\\\\\/HjStopAutskVDQs9CXLTlWLGhapmkdrEraM5LGEOa79bfCn4SStLv1fNl85rqB8TFhiuWXVbn\\\\\\\/DSbyLzuadunIF5p54bTkDJLMBMIhlVQTiGc+VRPR4MbSGBrHHrA98OJ0xDxz5pk8P5PFCuC6w\\\\\\\/EvgPOrZSsW5wCVav8+U0mlcIaV4kit639XhPXeCxIUaK2YaBCY+uqRqS3kDF0Ihm21eYrHujli+Ge2sgDfZrzKuBpg+Bn4eNnVNz6OM7zSkQzZzMLmLSp3egSSih2ic1WLM5VPNS9NBUN2nw1PDiHCAmwmGOn0OaUdzIpOpbpn5Q3QFznwWtHZsgRpEbYUl6im+USbDgAywfUsGYddOP8gEWGxugsukz6w5Fa+BSEV4OzmHw9u7NhO3Ihjrs3aYeuIQ1pSevPyvfwYIrGYNtyhWBKP5shJkpVG3+9zd0NymJza1F+5a9yeZo86BMET+JR3dStoM+XuDEqp3zVI\\\\\\\/1rhcI4GJOtWaYBTi\\\\\\\/DznBtR6WVQdKNIIQGH7COtgW44a0V+vy5P3ybKmiXWSAWeA5nmujKo558Y9HB7k1\\\\\\\/8LrWDEtMkRFmujl4NEChEsjSnjXF6QQFVAn7vmydd4uaiav1UnmWO6Uo3EsMNogC+0ObzINGXPn9y3Lqg0S\\\\\\\/qlhTpnM\\\\\\\/vsDwFncWa5+tyFvv6+5d\\\\\\\/ETUjXfVhhBe+xHmf805xWDu0FHYjaTqjloyiEeKvlJnC+D1Egt\\\\\\\/uIb884Oi+lF2ha91QYNDQ8wkKDvbCQsq6I\\\\\\\/XxCmf\\\\\\\/qSimns4hRZCe7nwejpHrhiLm8r4tPmMLdMmEMdJC3sRkumNor1GyD+34SdCfkZH1uDOCWdhrKrqjvi1GJv\\\\\\\/\\\\\\\/cavT1R3cN49ECz5i5cF\\\\\\\/0Hx54YmN\\\\\\\/m+y3lwEM\\\\\\\/lyeMa4\\\\\\\/QptW+\\\\\\\/YHK6R4JhbC1Zk1A\\\\\\\/BeDR0maQ0\\\u0022,\\\u0022iv\\\u0022:\\\u0022K4Ykb+iz6ao\\\\\\\/mwJgrCPusA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d0e6d6d2ebfdc132e00a417e9524e14478ba53d5e284d3f1007f8ffc2462b4a4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489447 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1687" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-njiyit8mRqXBHaewZbwop23og_jTa02TTCN31VVqVp0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1687" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ylnrqBDPUX3CtBWFALGgsjRIVlsyyzbU61mJyHLw_7g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1675" title="A Whole Genome Assembly of the Horn Fly, Haematobia irritans, and Prediction of Genes with Roles in Metabolism and Sex Determination" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1701" title="Programmable Single and Multiplex Base-Editing in Bombyx mori Using RNA-Guided Cytidine Deaminases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489447" id="node489447" data-pisa="ggg;8/5/1687" data-pisa-master="ggg;g3.117.300548" data-apath="/ggg/8/5/1687.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong Jiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renate H. Schmidt</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6742-265X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jochen C. Reif</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1687-1699;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300548 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489447" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong Jiang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Renate H. Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renate%2BH.%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenate%2BH.%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renate H. Schmidt</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6742-265X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jochen C. Reif</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/5/1687" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489447" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1687.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489447" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1687.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489447" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1687.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489447" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/5/1687.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489447" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489447/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genome-wide prediction approaches represent versatile tools for the analysis and prediction of complex traits. Mostly they rely on marker-based information, but scenarios have been reported in which models capitalizing on closely-linked markers that were combined into haplotypes outperformed marker-based models. Detailed comparisons were undertaken to reveal under which circumstances haplotype-based genome-wide prediction models are superior to marker-based models. Specifically, it was of interest to analyze whether and how haplotype-based models may take local epistatic effects between markers into account. Assuming that populations consisted of fully homozygous individuals, a marker-based model in which local epistatic effects inside haplotype blocks were exploited (LEGBLUP) was linearly transformable into a haplotype-based model (HGBLUP). This theoretical derivation formally revealed that haplotype-based genome-wide prediction models capitalize on local epistatic effects among markers. Simulation studies corroborated this finding. Due to its computational efficiency the HGBLUP model promises to be an interesting tool for studies in which ultra-high-density SNP data sets are studied. Applying the HGBLUP model to empirical data sets revealed higher prediction accuracies than for marker-based models for both traits studied using a mouse panel. In contrast, only a small subset of the traits analyzed in crop populations showed such a benefit. Cases in which higher prediction accuracies are observed for HGBLUP than for marker-based models are expected to be of immediate relevance for breeders, due to the tight linkage a beneficial haplotype will be preserved for many generations. In this respect the inheritance of local epistatic effects very much resembles the one of additive effects.</p></div><span class="kwd-group-title">Key words:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/local-epistatic-effect" class="hw-term hw-article-keyword hw-article-keyword-local-epistatic-effect" rel="nofollow">local epistatic effect</a></li><li class="kwd"><a href="/keyword/genome-wide-prediction" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-prediction" rel="nofollow">genome-wide prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">Genome-wide regression is a powerful tool to analyze and predict quantitative traits which are regulated by many genes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meuwissen <em>et al.</em> 2001</a>). Various genome-wide prediction approaches have been explored and applied for human (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Yang <em>et al.</em> 2010</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2010</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2013b</a>), animal (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hayes <em>et al.</em> 2013</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2013a</a>), and plant populations (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2014</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heslot <em>et al.</em> 2015</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hickey <em>et al.</em> 2017</a>). In most genome-wide prediction models, effects of molecular markers such as single nucleotide polymorphisms (SNPs) were used as explanatory variables (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cuyabano <em>et al.</em> 2015a</a>). Alternatively, molecular markers can be combined into haplotypes, which are then used to implement genome-wide prediction models (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Calus <em>et al.</em> 2008</a>). Haplotype-based prediction approaches are favored if alleles at quantitative trait loci (QTL) were more closely linked to haplotype alleles than individual SNPs (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zondervan and Cardon 2004</a>). Moreover, it is hypothesized that haplotypes can capture epistatic interactions between SNPs (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark 2004</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2014</a>). Therefore, haplotype-based approaches potentially boost prediction accuracies (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cuyabano <em>et al.</em> 2014</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">2015a</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">b</a>).</p><p id="p-4">The potential to exploit local epistatic effects among markers in haplotype-based prediction is interesting with respect to two points. First, epistasis has been recognized as a biologically influential component contributing to the genetic architecture of quantitative traits (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Carlborg and Haley 2004</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mackay 2014</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jiang <em>et al.</em> 2017</a>). The role of epistasis in genome-wide prediction has been extensively studied, but mostly in terms of marker-based approaches. Several marker-based models either implicitly or explicitly including epistatic effects in addition to main effects were developed (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xu 2007</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gianola and van Kaam 2008</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wittenburg <em>et al.</em> 2011</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jiang and Reif 2015</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Vitezica <em>et al.</em> 2017</a>). Taking epistasis into account can increase prediction accuracies (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2012</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Muñoz <em>et al.</em> 2014</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">He <em>et al.</em> 2016</a>). Second, decomposing epistasis into global and local effects is pivotal for evaluating the long-term impact of epistasis in plant and animal breeding, as there is a reduced chance that local epistatic effects will disappear after generations of recombination (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Jannink 2015</a>). First attempts in exploiting additive and local epistatic effects for genome-wide prediction were carried out with marker-based models resulting in good predictive performance and useful explanatory information (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akdemir and Jannink 2015</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akdemir <em>et al.</em> 2017</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">He <em>et al.</em> 2017</a>). Nevertheless, it has not been clarified why and how the haplotype-based approaches take local epistasis into account at the level of statistical models.</p><p id="p-5">The aims of this study were 1) to provide a formal theoretical explanation how haplotype-based genome-wide prediction models intrinsically exploit local epistatic effects among markers, 2) to investigate with simulation studies under which circumstances haplotype-based models perform better than marker-based models, and 3) to explore the potential of haplotype-based genome-wide prediction models using three published empirical data sets.</p><div class="section other1" id="sec-1"><h2 class="">THEORY</h2><p id="p-6">This section was organized as follows: First we introduced two genome-wide prediction models. Haplotype effects were used as explanatory variables in the haplotype-based genomic best linear unbiased prediction (HGBLUP) model, while additive and local epistatic effects among markers were utilized as predictors in the locally extended genomic best linear unbiased prediction (LEGBLUP) model. Then we proved that the haplotype-based model HGBLUP exploits local epistatic effects among markers by establishing a link between HGBLUP and LEGBLUP for the case in which all loci are homozygous. At the end of section, two examples were given to illustrate the theoretical results.</p><p id="p-7">Throughout the section, we made following conventions: Let <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-1.gif"/></noscript></span></span> be the number of genotypes, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-2.gif"/></noscript></span></span> be the number of markers. In this study we only considered bi-allelic markers. Suppose that the whole genome is divided into non-overlapping haplotype blocks; local epistasis is defined as interaction effects among two or more markers within a defined haplotype block. Let <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-3.gif"/></noscript></span></span> be the number of blocks. For <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-4.gif"/></noscript></span></span>, let <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-5.gif"/></noscript></span></span> be the number of markers in the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-6.gif"/></noscript></span></span>-th block. Let <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-7.gif"/></noscript></span></span> be the number of different haplotype alleles in the <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-8.gif"/></noscript></span></span>-th block. Linkage phases were assumed to be known. Vectors (matrices) are always denoted by lower (upper) case Latin or Greek letters in bold font.</p><div id="sec-2" class="subsection"><h3>The HGBLUP model</h3><p id="p-8">This model has been used in previous studies (<em>e.g.</em>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cuyabano <em>et al.</em> 2014</a>, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">2015a</a>) and here we called it HGBLUP. Independent from the definition of haplotype blocks, the HGBLUP model can be described as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">[1]</span></span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-10.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-11.gif"/></noscript></span></span>-dimensional vector of phenotypic records, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-12.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-13.gif"/></noscript></span></span>-dimensional vector of one’s, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-14.gif"/></noscript></span></span> is a common intercept term, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-15.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-16.gif"/></noscript></span></span>-dimensional vector of haplotype effects in the <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-17.gif"/></noscript></span></span>-th haplotype block, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-18.gif"/></noscript></span></span> is the corresponding <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-19.gif"/></noscript></span></span> design matrix of the <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-20.gif"/></noscript></span></span>-th block, the <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-21.gif"/></noscript></span></span>-entry of <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-22.gif"/></noscript></span></span> is the number of the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-23.gif"/></noscript></span></span>-th haplotype allele in the <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-24.gif"/></noscript></span></span>-th genotype (hence, it is 0, 1 or 2), and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-25.gif"/></noscript></span></span> is the residual term. In the model we assumed that <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-26.gif"/></noscript></span></span> is a fixed parameter, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-27.gif"/></noscript></span></span> for any <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-28.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-29.gif"/></noscript></span></span>. We assumed no covariance structure among these variables.</p><p id="p-9">The formulation of this model is similar to ridge regression best linear unbiased prediction (RR-BLUP, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Meuwissen <em>et al.</em> 2001</a>) except that the marker effects were replaced by haplotype effects. Note that there are in total <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-30.gif"/></noscript></span></span> unknown parameters in the model. This number can be even larger than the number of markers, which makes the computational load very high. However, the model can be implemented in an alternative way similar to the marker-based genomic best linear unbiased prediction model (GBLUP, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">VanRaden 2008</a>):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">[2]</span></span>where <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-32.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-33.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-34.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-35.gif"/></noscript></span></span> are the same as in Equation 1; <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-36.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-37.gif"/></noscript></span></span>-dimensional vector of genotypic values. We assumed that <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-38.gif"/></noscript></span></span> is a fixed parameter, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-39.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-40.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-41.gif"/></noscript></span></span>. Setting <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-42.gif"/></noscript></span></span>, it becomes obvious that the two models are statistically equivalent, as the equivalence between GBLUP and RR-BLUP (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>).</p></div><div id="sec-3" class="subsection"><h3>The LEGBLUP model</h3><p id="p-10">This model is a local version of the extended GBLUP (EGBLUP) (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jiang and Reif 2015</a>). EGBLUP exploits epistasis between any pair of markers while LEGBLUP only considers local epistasis inside each haplotype block. Assuming only digenic epistasis, the model can be described as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">[3]</span></span>where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-44.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-45.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-46.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-47.gif"/></noscript></span></span> are the same as in Equation 1, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-48.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-49.gif"/></noscript></span></span> matrix of marker profiles, the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-50.gif"/></noscript></span></span>-entry of <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-51.gif"/></noscript></span></span> is the number of a specific allele of the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-52.gif"/></noscript></span></span>-th marker carried by the <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-53.gif"/></noscript></span></span>-th genotype (hence, it is 0, 1 or 2), <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-54.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-55.gif"/></noscript></span></span>-dimensional vector of marker additive effects, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-56.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-57.gif"/></noscript></span></span> design matrix for additive-by-additive epistatic effects for markers in the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-58.gif"/></noscript></span></span>-th haplotype block, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-59.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-60.gif"/></noscript></span></span>-dimensional vector of epistatic effects in the <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-61.gif"/></noscript></span></span>-th block. In the model we assumed that <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-62.gif"/></noscript></span></span> is a fixed parameter, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-63.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-64.gif"/></noscript></span></span> for any <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-65.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-66.gif"/></noscript></span></span>. We assumed no covariance structure among these variables.</p><p id="p-11">Note that there are <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-67.gif"/></noscript></span></span> unknown variables in the model with <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-68.gif"/></noscript></span></span>, and this number can be very large. Hence, the model can be implemented in an alternative way as:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">[4]</span></span>where <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-70.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-71.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-72.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-73.gif"/></noscript></span></span> are the same as in Equation 1, <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-74.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-75.gif"/></noscript></span></span>-dimensional vector of additive genotypic values, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-76.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-77.gif"/></noscript></span></span>-dimensional vector of genetic values accounting for local epistasis. We assumed that <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-78.gif"/></noscript></span></span> is a fixed parameter, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-79.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-81.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-82.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-83.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-84.gif"/></noscript></span></span> is the Hadamard product, <em>i.e.</em>, entry-wise product, of matrices. Setting <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-85.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-86.gif"/></noscript></span></span>, it reveals that the two models are statistically equivalent. The reason is the same as the equivalence between EGBLUP and an extended RR-BLUP model including epistasis (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Jiang and Reif 2015</a>).</p><p id="p-12">Note that in the above descriptions the LEGBLUP model only includes digenic local epistasis, <em>i.e.</em>, local epistatic effects between two markers. In fact, the LEGBLUP model can be generalized to include all possible higher-order epistatic interaction effects within each haplotype block, as the EGBLUP model. Briefly, we only need to extend Equation 4 to:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-87.gif"/></noscript></span><span class="disp-formula-label">[5]</span></span>where <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-88.gif"/></noscript></span></span> is the vector of genetic values accounting for (r-1)-th order epistasis, <em>i.e.</em>, epistatic interactions among r markers. The kinship matrix for <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-89.gif"/></noscript></span></span> can be derived using t-fold Hadamard product of <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-90.gif"/></noscript></span></span> (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Jiang and Reif 2015</a>). This model is denoted as full LEGBLUP.</p></div><div id="sec-4" class="subsection"><h3>The link Between HGBLUP and LEGBLUP</h3><p id="p-13">We first concentrated on a single haplotype block, thus subscripts to differentiate blocks can be ignored. We assumed <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-91.gif"/></noscript></span></span> markers and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-92.gif"/></noscript></span></span> haplotype alleles and then the HGBLUP model (Equation 1) reduces to:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-93.gif"/></noscript></span><span class="disp-formula-label">[6]</span></span>where <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-94.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-95.gif"/></noscript></span></span>-dimensional vector of haplotype effects and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-96.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-97.gif"/></noscript></span></span> design matrix. The assumptions were that <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-98.gif"/></noscript></span></span> is a fixed parameter, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-99.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-100.gif"/></noscript></span></span>.</p><p id="p-14">For LEGBLUP, a unified expression of Equation 3 was needed to extend it to full LEGBLUP. We defined <strong><span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-101.gif"/></noscript></span></span></strong> to be a vector whose components are marker main effects together with epistatic effects up to the <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-102.gif"/></noscript></span></span>-th order, <em>i.e.</em>, all possible epistatic effects among any number of markers in the block, not only digenic epistatic effects. Thus, the dimension of <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-103.gif"/></noscript></span></span> is:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-104.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-105.gif"/></noscript></span></span> denote the Gaussian binomial coefficients. Let <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-106.gif"/></noscript></span></span> be the corresponding <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-107.gif"/></noscript></span></span> design matrix. With these notations, the full LEGBLUP model can be simply written as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-108.gif"/></noscript></span><span class="disp-formula-label">[7]</span></span>The assumptions were that <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-109.gif"/></noscript></span></span> is a fixed parameter, <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-110.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-111.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-112.gif"/></noscript></span></span> is a diagonal matrix containing different unknown variance parameters for additive effects and different orders of epistatic effects.</p><div id="sec-5" class="subsection"><h4>Claim:</h4><p id="p-15">If all loci under consideration are homozygous, then there exists a <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-113.gif"/></noscript></span></span> matrix <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-114.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-115.gif"/></noscript></span></span>.</p><p id="p-16">The above claim was the key to bridge HGBLUP and LEGBLP. As its proof requires more techniques in linear algebra, we presented it as a separate subsection below.</p><p id="p-17">Now we assumed all loci to be homozygous. Setting <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-116.gif"/></noscript></span></span>, HGBLUP (Equation 6) can be expressed as:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-117.gif"/></noscript></span><span class="disp-formula-label">[8]</span></span>The newly defined vector <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-118.gif"/></noscript></span></span> has the same design matrix as <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-119.gif"/></noscript></span></span> in the LEGBLUP model (Equation 7). Thus, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-120.gif"/></noscript></span></span> includes marker effects as well as epistatic effects among markers. Accordingly, Equation 8 is the same as Equation 7 and hence HGBLUP has the same base equation as LEGBLUP.</p><p id="p-18">Nevertheless, there is one important difference between the two models. In LEGBLUP, the covariance matrix for <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-121.gif"/></noscript></span></span> is assumed to be a diagonal matrix <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-122.gif"/></noscript></span></span>, hence, no covariance between different variables is assumed. But in HGBLUP, although the distribution of <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-123.gif"/></noscript></span></span> is still multivariate normal, its covariance structure is:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-124.gif"/></noscript></span></span>In general, the matrix <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-125.gif"/></noscript></span></span> is semi-positive definite but not diagonal. Thus, HGBLUP implicitly assumes a non-trivial covariance structure.</p><p id="p-19">Now it is straightforward to generalize the results to the case of a full model including all blocks, since no inter-block effects are modeled and the linear transformation <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-126.gif"/></noscript></span></span> can be independently applied to each block.</p></div></div><div id="sec-6" class="subsection"><h3>The proof of the claim</h3><p id="p-20">As the loci under consideration are homozygous, there are <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-127.gif"/></noscript></span></span> independent variables in the LEGBLUP model (Equation 7). For the HGBLUP model, there are at most <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-128.gif"/></noscript></span></span> different haplotype alleles, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-129.gif"/></noscript></span></span>. But note that if there are <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-130.gif"/></noscript></span></span> haplotype alleles, the number of independent variables in the model is <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-131.gif"/></noscript></span></span> because of collinearity, similar to the biallelic case (<em>e.g.</em>, SNP markers) in which there is only one independent variable. Hence, we can assume <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-132.gif"/></noscript></span></span>. We shall consider two cases.</p><div id="sec-7" class="subsection"><h4>Case 1: All possible haplotype alleles occur:</h4><p id="p-21">We assumed that all possible haplotype alleles occur in the data, then <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-133.gif"/></noscript></span></span>. We started from the HGBLUP model (Equation 6). Recall that for any <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-134.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-135.gif"/></noscript></span></span>, the <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-136.gif"/></noscript></span></span>-entry of <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-137.gif"/></noscript></span></span>, denoted by <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-138.gif"/></noscript></span></span>, is the number of the <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-139.gif"/></noscript></span></span>-th haplotype allele carried by the <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-140.gif"/></noscript></span></span>-th individual. Since all marker loci are homozygous, <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-141.gif"/></noscript></span></span> must be 0 or 2. As we assumed that all possible haplotype alleles occur in the data, for any <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-142.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-143.gif"/></noscript></span></span>) there exists <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-144.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-145.gif"/></noscript></span></span>) such that <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-146.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-147.gif"/></noscript></span></span> for all <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-148.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-149.gif"/></noscript></span></span>. Combining the <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-150.gif"/></noscript></span></span> rows <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-151.gif"/></noscript></span></span> of the design matrix <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-152.gif"/></noscript></span></span> results in an <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-153.gif"/></noscript></span></span> submatrix <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-154.gif"/></noscript></span></span>. It is clear that <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-155.gif"/></noscript></span></span> is invertible because it can be transformed to <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-156.gif"/></noscript></span></span> by row permutation. Correspondingly, we took the <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-157.gif"/></noscript></span></span> rows <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-158.gif"/></noscript></span></span> of the design matrix <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-159.gif"/></noscript></span></span> in the LEGBLUP model (Equation 7). This also yielded an <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-160.gif"/></noscript></span></span> submatrix <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-161.gif"/></noscript></span></span>. We observed that <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-162.gif"/></noscript></span></span> is also invertible. The proof of this fact was presented separately at the end of this subsection. As <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-163.gif"/></noscript></span></span> is invertible, we can define <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-164.gif"/></noscript></span></span> and hence <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-165.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-166.gif"/></noscript></span></span> being invertible.</p><p id="p-22">We then claimed that <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-167.gif"/></noscript></span></span>. In fact, for any <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-168.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-169.gif"/></noscript></span></span>, the <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-170.gif"/></noscript></span></span>-th row <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-171.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-172.gif"/></noscript></span></span> must coincide with <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-173.gif"/></noscript></span></span> for some <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-174.gif"/></noscript></span></span> because <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-175.gif"/></noscript></span></span> exhaust all the possibilities of row vectors for <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-176.gif"/></noscript></span></span>. Correspondingly, the <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-177.gif"/></noscript></span></span>-th row <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-178.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-179.gif"/></noscript></span></span> must coincide with <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-180.gif"/></noscript></span></span>. Since <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-181.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-182.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-183.gif"/></noscript></span></span> are corresponding rows in <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-184.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-185.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-186.gif"/></noscript></span></span>. As it holds for any <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-187.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-188.gif"/></noscript></span></span>.</p></div><div id="sec-8" class="subsection"><h4>Case 2: Not all possible haplotype alleles occur:</h4><p id="p-23">Now we assumed that not all possible haplotype alleles occur in the data (<span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-189.gif"/></noscript></span></span>). In contrast to the case that considers all haplotype alleles, the submatrix <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-190.gif"/></noscript></span></span> in the LEGBLUP is not <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-191.gif"/></noscript></span></span> but <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-192.gif"/></noscript></span></span>. So we need to adjust our arguments. In fact, <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-193.gif"/></noscript></span></span> has full row rank: using the results in the previous case, <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-194.gif"/></noscript></span></span> can be viewed as a submatrix of a full rank <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-195.gif"/></noscript></span></span> matrix. Hence, there exists a right inverse <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-196.gif"/></noscript></span></span> which is a <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-197.gif"/></noscript></span></span> matrix such that <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-198.gif"/></noscript></span></span>. Defining <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-199.gif"/></noscript></span></span>, we still obtain <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-200.gif"/></noscript></span></span>, and hence <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-201.gif"/></noscript></span></span>.</p></div><div id="sec-9" class="subsection"><h4>The proof of the fact that <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-202.gif"/></noscript></span></span> is invertible:</h4><p id="p-24">Recall that <em><span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-203.gif"/></noscript></span></span></em> is an <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-204.gif"/></noscript></span></span> matrix, where <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-205.gif"/></noscript></span></span>. The columns of <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-206.gif"/></noscript></span></span> can be naturally indexed by the set<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-207.gif"/></noscript></span></span>In fact we can denote the entries in <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-208.gif"/></noscript></span></span> by <em><span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-209.gif"/></noscript></span></span></em>. When <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-210.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-211.gif"/></noscript></span></span>is just the number of alleles of the <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-212.gif"/></noscript></span></span>-th marker carried by the <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-213.gif"/></noscript></span></span>-th genotype, which serves as the coefficient for the main additive effect of the <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-214.gif"/></noscript></span></span>-th marker. When <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-215.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-216.gif"/></noscript></span></span> is the coefficient of the epistatic effects among the markers <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-217.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-218.gif"/></noscript></span></span>,… and<span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-219.gif"/></noscript></span></span> for the <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-220.gif"/></noscript></span></span>-th genotype. With the above notations, the column vectors of <strong><em><span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-221.gif"/></noscript></span></span></em></strong> can be denoted by <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-222.gif"/></noscript></span></span>.</p><p id="p-25">The rows of <em><span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-223.gif"/></noscript></span></span></em> can also be labeled by the set <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-224.gif"/></noscript></span></span>, which is trivial because <em><span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-225.gif"/></noscript></span></span></em> has the same number of rows as columns. But we can introduce the following natural labeling: If a genotype is coded as 2 in the markers <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-226.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-227.gif"/></noscript></span></span>,…,<span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-228.gif"/></noscript></span></span> and 0 in the remaining ones, then we label the corresponding row as <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-229.gif"/></noscript></span></span>. With these notations, the entries in <em><span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-230.gif"/></noscript></span></span></em> can be written as <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-231.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-232.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-233.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-234.gif"/></noscript></span></span>. By definition we have:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-235.gif"/></noscript></span><span class="disp-formula-label">[9]</span></span>To show that <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-236.gif"/></noscript></span></span> is invertible, it is sufficient to show that the column vectors <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-237.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-238.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-239.gif"/></noscript></span></span>) span the space <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-240.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-241.gif"/></noscript></span></span> denotes the set of rational numbers. The space <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-242.gif"/></noscript></span></span> has a natural basis <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-243.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-244.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-245.gif"/></noscript></span></span> is the vector whose <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-246.gif"/></noscript></span></span>-entry is 1 and all other entries are zeros.</p><p id="p-26">We first considered <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-247.gif"/></noscript></span></span>. In this case we have only one vector <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-248.gif"/></noscript></span></span>, which is the coefficient of the epistatic effects among all <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-249.gif"/></noscript></span></span> markers. From Equation 9 we know that the only non-zero entry in <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-250.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-251.gif"/></noscript></span></span> and it equals <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-252.gif"/></noscript></span></span>. So <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-253.gif"/></noscript></span></span>.</p><p id="p-27">Next we considered the case <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-254.gif"/></noscript></span></span>. In this case we have <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-255.gif"/></noscript></span></span> vectors <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-256.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-257.gif"/></noscript></span></span> denotes that <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-258.gif"/></noscript></span></span> is absent in the sequence <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-259.gif"/></noscript></span></span>. Again using Equation 9, we know that there are only two non-zero entries in <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-260.gif"/></noscript></span></span>, namely <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-261.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-262.gif"/></noscript></span></span>, both values are <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-263.gif"/></noscript></span></span>. Hence, <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-264.gif"/></noscript></span></span>.</p><p id="p-28">Repeating the procedure for smaller<span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-265.gif"/></noscript></span></span>, we can see that all basis vectors <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-266.gif"/></noscript></span></span> can be written as linear combinations of the vectors <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-267.gif"/></noscript></span></span>, which completes the proof.</p></div></div><div id="sec-10" class="subsection"><h3>The case of heterozygous loci</h3><p id="p-29">Recall Equation 6 for HGBLUP and Equation 7 for LEGBLUP. Different from the case in which homozygous loci are considered, now the elements <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-268.gif"/></noscript></span></span> in the design matrix <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-269.gif"/></noscript></span></span> can take the value 1, in addition to 0 and 2. More precisely, when the paternal and maternal haplotypes are different, the corresponding row vector of <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-270.gif"/></noscript></span></span> will have two non-zero entries, both being 1. This essential difference makes it impossible to find a matrix <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-271.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-272.gif"/></noscript></span></span> holds in general. So there does not exist any linear transformation <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-273.gif"/></noscript></span></span> such that the base equations of HGBLUP and LEGBLUP become the same. This result was proved by giving a counterexample (see Example 2 in the next subsection).</p></div><div id="sec-11" class="subsection"><h3>Illustration of the theoretical results</h3><p id="p-30">In this section, two examples were provided illustrating the theoretical findings for homozygous (Example 1; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and heterozygous loci (Example 2; <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489633/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489633/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of SNP marker coding and haplotype alleles for the six individuals considered in Theory, Example 1</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489483/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489483/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of SNP coding and haplotype alleles for the 6 individuals considered in Theory, Example 2</span><div class="sb-div caption-clear"></div></div></div><div id="sec-12" class="subsection"><h4>Example 1:</h4><p id="p-31">We considered six individuals and one haplotype block with two SNP markers (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). As outlined above the two homozygous genotypes were coded as 0 and 2 resulting in four different haplotype alleles.</p><p id="p-32">The vector of haplotype effects is <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-274.gif"/></noscript></span></span> and the corresponding design matrix is <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-275.gif"/></noscript></span></span>. As the fourth column of <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-276.gif"/></noscript></span></span> can be obtained by subtracting the sum of the other three columns in the vector <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-277.gif"/></noscript></span></span>, the last variable can be dropped resulting in <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-278.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-279.gif"/></noscript></span></span>. Then the HGBLUP model has the following form:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-280.gif"/></noscript></span><span class="disp-formula-label">[10]</span></span>with the assumptions <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-281.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-282.gif"/></noscript></span></span>.</p><p id="p-33">The vector of marker effects is <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-283.gif"/></noscript></span></span> with the design matrix <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-284.gif"/></noscript></span></span> with the third column of <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-285.gif"/></noscript></span></span> being the element-wise product of the first two columns; so the LEGBLUP model has the form:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-286.gif"/></noscript></span><span class="disp-formula-label">[11]</span></span>with the assumptions <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-287.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-288.gif"/></noscript></span></span>.</p><p id="p-34">We took the first three rows in <span class="inline-formula" id="inline-formula-275"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-289.gif"/></noscript></span></span> and formed the submatrix <span class="inline-formula" id="inline-formula-276"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-290.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-290.gif"/></noscript></span></span>, as each of the first three individuals carries a different haplotype allele. So <span class="inline-formula" id="inline-formula-277"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-291.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-291.gif"/></noscript></span></span>. Then we accordingly took the first three rows in <span class="inline-formula" id="inline-formula-278"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-292.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-292.gif"/></noscript></span></span> to form the submatrix <strong><span class="inline-formula" id="inline-formula-279"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-293.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-293.gif"/></noscript></span></span></strong> and defined <span class="inline-formula" id="inline-formula-280"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-294.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-294.gif"/></noscript></span></span>, thus, <span class="inline-formula" id="inline-formula-281"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-295.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-295.gif"/></noscript></span></span>. Assuming <span class="inline-formula" id="inline-formula-282"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-296.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-296.gif"/></noscript></span></span> resulted in <span class="inline-formula" id="inline-formula-283"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-297.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-297.gif"/></noscript></span></span> with<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-298.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-298.gif"/></noscript></span></span>Hence, the HGBLUP model (Equation 10) is equivalent to<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-299.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-299.gif"/></noscript></span><span class="disp-formula-label">[12]</span></span>with the assumptions <span class="inline-formula" id="inline-formula-284"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-300.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-300.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-285"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-301.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-301.gif"/></noscript></span></span>.</p><p id="p-35">Since in Equation 12 the parameters <span class="inline-formula" id="inline-formula-286"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-302.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-302.gif"/></noscript></span></span> have exactly the same design matrix as <span class="inline-formula" id="inline-formula-287"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-303.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-303.gif"/></noscript></span></span> in Equation 11, the base equations of HGBLUP and LEGBLUP are indeed the same. Setting <span class="inline-formula" id="inline-formula-288"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-304.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-304.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-289"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-305.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-305.gif"/></noscript></span></span>, we can see that the only difference between the models is that in LEGBLUP (Equation 11) the covariance between additive and epistatic effects was zero while in HGBLUP (Equation 12) the covariance was <span class="inline-formula" id="inline-formula-290"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-306.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-306.gif"/></noscript></span></span>.</p></div><div id="sec-13" class="subsection"><h4>Example 2:</h4><p id="p-36">We considered six genotypes and a haplotype block with two SNP markers (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). In contrast to Example 1, we assumed presence of heterozygous loci. The vector of haplotype effects is <span class="inline-formula" id="inline-formula-291"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-307.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-307.gif"/></noscript></span></span> with the design matrix <span class="inline-formula" id="inline-formula-292"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-308.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-308.gif"/></noscript></span></span>. As outlined in the first example, we can simply set <span class="inline-formula" id="inline-formula-293"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-309.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-309.gif"/></noscript></span></span> owing to linear dependency and <span class="inline-formula" id="inline-formula-294"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-310.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-310.gif"/></noscript></span></span>. The vector of marker effects is <span class="inline-formula" id="inline-formula-295"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-311.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-311.gif"/></noscript></span></span> with the design matrix <span class="inline-formula" id="inline-formula-296"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-312.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-312.gif"/></noscript></span></span>.</p><p id="p-37">In the following, we showed that there does not exist any matrix <span class="inline-formula" id="inline-formula-297"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-313.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-313.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-298"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-314.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-314.gif"/></noscript></span></span>, <em>i.e.</em>, the HGBLUP and LEGBLUP model have the same base equations. For the proof, we assumed the contrary, that there exists a matrix <span class="inline-formula" id="inline-formula-299"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-315.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-315.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-300"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-316.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-316.gif"/></noscript></span></span>. Then for any submatrix <span class="inline-formula" id="inline-formula-301"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-317.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-317.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-302"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-318.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-318.gif"/></noscript></span></span> and the corresponding submatrix <strong><span class="inline-formula" id="inline-formula-303"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-319.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-319.gif"/></noscript></span></span></strong> of <span class="inline-formula" id="inline-formula-304"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-320.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-320.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-305"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-321.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-321.gif"/></noscript></span></span> must hold. Let <span class="inline-formula" id="inline-formula-306"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-322.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-322.gif"/></noscript></span></span> be the submatrix of <span class="inline-formula" id="inline-formula-307"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-323.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-323.gif"/></noscript></span></span> consisting of the first three rows, so <span class="inline-formula" id="inline-formula-308"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-324.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-324.gif"/></noscript></span></span>. Accordingly, <span class="inline-formula" id="inline-formula-309"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-325.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-325.gif"/></noscript></span></span>. For <strong><span class="inline-formula" id="inline-formula-310"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-326.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-326.gif"/></noscript></span></span></strong> to be true, the only choice for <span class="inline-formula" id="inline-formula-311"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-327.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-327.gif"/></noscript></span></span> is that <strong><span class="inline-formula" id="inline-formula-312"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-328.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-328.gif"/></noscript></span></span></strong>. Nevertheless,<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-329.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-329.gif"/></noscript></span></span>which is a contradiction. In fact, we can clearly see that only the last row of <span class="inline-formula" id="inline-formula-313"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-330.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-330.gif"/></noscript></span></span> differs from <span class="inline-formula" id="inline-formula-314"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-331.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-331.gif"/></noscript></span></span>. So the problem occurs when at least two loci are heterozygous for some genotypes.</p></div></div></div><div class="section materials-methods" id="sec-14"><h2 class="">Materials and Methods</h2><div id="sec-15" class="subsection"><h3>Simulation study</h3><p id="p-38">Based on the genomic data of a panel of maize lines belonging to the flint heterotic pool (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bauer <em>et al.</em> 2013</a>), simulated traits were generated. Six scenarios were considered with different types and patterns of epistatic QTL effects (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489501/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489501/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of the six simulation scenarios</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">In all scenarios, 100 markers were randomly sampled as QTL for each of the 10 chromosomes, resulting in 1,000 QTL per scenario. In scenario 1, only additive effects were simulated. Hence, the genetic values are <span class="inline-formula" id="inline-formula-315"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-332.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-332.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-316"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-333.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-333.gif"/></noscript></span></span> is the vector of additive QTL effects and <span class="inline-formula" id="inline-formula-317"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-334.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-334.gif"/></noscript></span></span> is the marker design matrix. The additive effects were independently sampled from a normal distribution of mean 0 and variance <span class="inline-formula" id="inline-formula-318"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-335.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-335.gif"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-319"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-336.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-336.gif"/></noscript></span></span>. In scenario 2, we simulated additive and global epistatic effects. 1,000 pairs of markers were randomly selected to present digenic epistatic effects. Hence, the genetic values are <span class="inline-formula" id="inline-formula-320"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-337.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-337.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-321"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-338.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-338.gif"/></noscript></span></span> and <strong><span class="inline-formula" id="inline-formula-322"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-339.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-339.gif"/></noscript></span></span></strong> are the same as in scenario 1, and <span class="inline-formula" id="inline-formula-323"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-340.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-340.gif"/></noscript></span></span> is the vector of epistatic effects with design matrix <strong><span class="inline-formula" id="inline-formula-324"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-341.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-341.gif"/></noscript></span></span></strong>. The epistatic effects were also independently sampled from a normal distribution, <em>i.e.</em>, <strong><span class="inline-formula" id="inline-formula-325"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-342.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-342.gif"/></noscript></span></span></strong>.</p><p id="p-40">In scenarios 3 to 6, we simulated additive effects and local epistatic effects. For local epistasis, we first randomly divided each chromosome into non-overlapping blocks, each consisting of 2 to 5 markers. Epistatic effects were simulated only inside individual blocks. Thus, the simulated genetic values <span class="inline-formula" id="inline-formula-326"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-343.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-343.gif"/></noscript></span></span>, where w is the number of blocks, <span class="inline-formula" id="inline-formula-327"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-344.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-344.gif"/></noscript></span></span> is the vector of additive and epistatic effects inside the k-th block and <span class="inline-formula" id="inline-formula-328"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-345.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-345.gif"/></noscript></span></span> is the corresponding design matrix. In scenarios 3 and 5, only digenic epistatic effects were simulated. In scenarios 4 and 6, all possible epistatic effects were considered, hence, including higher-order epistasis. In scenarios 3 and 4, all effects were assumed to be independent. In scenarios 5 and 6, epistatic effects inside individual blocks were assumed to be correlated.</p><p id="p-41">For each scenario, we considered one trait with two different simulated heritabilities (<span class="inline-formula" id="inline-formula-329"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-346.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-346.gif"/></noscript></span></span> = 0.7 or 0.5) and two ratios of variances (<span class="inline-formula" id="inline-formula-330"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-347.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-347.gif"/></noscript></span></span> = 4:3 or 3:1). In case of <span class="inline-formula" id="inline-formula-331"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-348.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-348.gif"/></noscript></span></span> = 4:3, the covariance matrices of genetic effects in the individual haplotype blocks were directly derived using the method described in the Theory section, <em>i.e.</em>, the covariance matrix equals the matrix <span class="inline-formula" id="inline-formula-332"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-349.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-349.gif"/></noscript></span></span>, which gave the ratio 4:3 and determined the variances for higher-order epistatic effects. To simulate a situation in which the variance of epistatic effects was less relevant, we considered also a 3:1 ratio. In this case, we modified the matrix <span class="inline-formula" id="inline-formula-333"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-350.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-350.gif"/></noscript></span></span> as follows; we changed <span class="inline-formula" id="inline-formula-334"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-351.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-351.gif"/></noscript></span></span> from <span class="inline-formula" id="inline-formula-335"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-352.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-352.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-336"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-353.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-353.gif"/></noscript></span></span> and accordingly modified all variance terms of higher-order epistasis by keeping the ratio of any two epistatic variance terms (<em>e.g.</em>, <span class="inline-formula" id="inline-formula-337"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-354.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-354.gif"/></noscript></span></span>). The variance of additive effects <span class="inline-formula" id="inline-formula-338"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-355.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-355.gif"/></noscript></span></span> and all correlations were not changed. When only digenic epistatic effects were simulated, the rows and columns corresponding to higher-order epistasis were deleted.</p><p id="p-42">As the final step, the phenotypic values were simulated as <span class="inline-formula" id="inline-formula-339"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-356.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-356.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-340"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-357.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-357.gif"/></noscript></span></span> is the simulated genetic value as described above and <span class="inline-formula" id="inline-formula-341"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-358.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-358.gif"/></noscript></span></span> is the environmental error term. The error terms were independently sampled from a normal distribution, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-342"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-359.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-359.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-343"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-360.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-360.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-344"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-361.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-361.gif"/></noscript></span></span> is the genetic variance calculated from the simulated genetic values. For each scenario, trait heritability and variance ratio, simulations were repeated 20 times.</p></div><div id="sec-16" class="subsection"><h3>Empirical data</h3><div id="sec-17" class="subsection"><h4>Mouse data:</h4><p id="p-43">The mouse data set used for this study comprised 1,940 heterogeneous stock mice genotyped with 12,545 SNP markers. The measured traits were body weight at age of six weeks and growth slope between six and ten weeks of age (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Valdar <em>et al.</em> 2006</a>).</p></div><div id="sec-18" class="subsection"><h4>Rice data:</h4><p id="p-44">The rice data set comprised a diversity panel of 413 varieties genotyped with an Affymetrix 44K SNP array (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> 2011</a>). Individuals were highly homozygous. After quality control, 39,601 SNP markers were used in this study. Phenotypic data of 26 traits with contrasting genetic architectures were available.</p></div><div id="sec-19" class="subsection"><h4>Maize data:</h4><p id="p-45">The maize data set comprised a large half-sib maize panel from the flint heterotic pool generated within the European PLANT-KBBE CornFed project (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bauer <em>et al.</em> 2013</a>). The panel consisted of 11 half-sib families with 833 doubled haploid (DH) lines. After quality control for missing rate and minor allele frequency, 29,466 SNP markers were used for subsequent analyses. Phenotypic traits under consideration were dry matter yield, dry matter content, plant height, days to tasseling and days to silking (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lehermeier <em>et al.</em> 2014</a>).</p></div></div><div id="sec-20" class="subsection"><h3>Genome-wide prediction</h3><p id="p-46">For the simulated and empirical data, we considered three marker-based models, GBLUP, EGBLUP and LEGBLUP, and one haplotype-based model, HGBLUP (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For LEGBLUP and HGBLUP, we defined haplotype blocks using fixed lengths, varying from 2 to 5 (10) SNPs for the simulated (empirical) data. For the mouse data set, in which the linkage phase of the marker data are unknown, we treated each allele of a heterozygous locus as having equal probability (<em>i.e.</em>, 50%) to be maternal or paternal. The prediction accuracy (ability) was defined as the Pearson correlation between the predicted and the simulated (observed) genetic values for simulated (empirical) data. For each model the mean prediction accuracy was estimated with fivefold cross validation. All models were implemented using the statistical software R (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Core Team 2016</a>) with the package BGLR (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Peréz and de los Campos 2014</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characteristics and relationships of genomic prediction models considered in this study. The genetic effects exploited by the model were indicated in brackets. GBLUP: genome-wide best linear unbiased prediction; RRBLUP: ridge regression best linear unbiased prediction; EGBLUP: extended genome-wide best linear unbiased prediction; LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. The gray arrows indicate that the models differ with regard to the type and number of effects that are exploited. The equivalence of the LEGBLUP and HGBLUP models that was shown for inbred populations is illustrated by the double arrow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845346279" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characteristics and relationships of genomic prediction models considered in this study. The genetic effects exploited by the model were indicated in brackets. GBLUP: genome-wide best linear unbiased prediction; RRBLUP: ridge regression best linear unbiased prediction; EGBLUP: extended genome-wide best linear unbiased prediction; LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. The gray arrows indicate that the models differ with regard to the type and number of effects that are exploited. The equivalence of the LEGBLUP and HGBLUP models that was shown for inbred populations is illustrated by the double arrow.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1687/F1.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/5/1687/F1.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489631" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-47" class="first-child">Characteristics and relationships of genomic prediction models considered in this study. The genetic effects exploited by the model were indicated in brackets. GBLUP: genome-wide best linear unbiased prediction; RRBLUP: ridge regression best linear unbiased prediction; EGBLUP: extended genome-wide best linear unbiased prediction; LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. The gray arrows indicate that the models differ with regard to the type and number of effects that are exploited. The equivalence of the LEGBLUP and HGBLUP models that was shown for inbred populations is illustrated by the double arrow.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Data Availability</h3><p id="p-48">All empirical data used in this study have been published. The mouse data set was included in the R package SynbreedData (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wimmer <em>et al.</em> 2015</a>, <a href="https://cran.r-project.org/web/packages/synbreedData/index.html">https://cran.r-project.org/web/packages/synbreedData/index.html</a>). The rice data set was published in <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> (2011)</a> and can be downloaded from <a href="https://ricediversity.org/data/sets/44kgwas/">https://ricediversity.org/data/sets/44kgwas/</a>. The genomic data of the maize data set was published in <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bauer <em>et al.</em> (2013)</a> and can be downloaded from <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50558">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50558</a>. The phenotypic data of the maize data set was published as File S1 in <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lehermeier <em>et al.</em> (2014)</a> and can be downloaded from <a href="http://www.genetics.org/content/198/1/3.supplemental">http://www.genetics.org/content/198/1/3.supplemental</a>. Figure S1 shows the prediction accuracies of GBLUP, EGBLUP, LEGBLUP and HGBLUP for simulated traits with heritability 0.5 and <span class="inline-formula" id="inline-formula-345"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-362.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-362.gif"/></noscript></span></span> = 4:3. Figure S2 shows the prediction accuracies of the four models for simulated traits with heritability 0.7 and <span class="inline-formula" id="inline-formula-346"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-363.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-363.gif"/></noscript></span></span> = 3:1. Figure S3 shows the prediction accuracies of the four models for simulated traits with heritability 0.5 and <span class="inline-formula" id="inline-formula-347"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-364.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-364.gif"/></noscript></span></span> = 3:1. Table S1 provides the prediction accuracies of the four models for the 26 agronomic traits in the rice data set. File S1 contains the R code used to generate the data for the simulation study. File S2 and File S3 contain sample genomic and physical map data sets for running the code.</p></div></div><div class="section results-discussion" id="sec-22"><h2 class="">Results and Discussion</h2><div id="sec-23" class="subsection"><h3>Modeling haplotype effects exploit local epistasis among markers</h3><p id="p-49">We compared two genome-wide prediction models to study whether local epistatic effects among markers are formally taken into account by modeling haplotype effects. The first model utilizes haplotype effects as predictors and has been used in previous studies (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cuyabano <em>et al.</em> 2014</a>, <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">2015a</a>), here we called it HGBLUP. The HGBLUP model is similar to the well-known GBLUP model (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">VanRaden 2008</a>) which exploits a marker-derived relationship matrix among genotypes. In HGBLUP the marker-derived relationship matrix is replaced by the haplotype-derived relationship matrix. Note that modeling a haplotype-derived relationship matrix is equivalent to explicitly modeling haplotype effects (Equation 1, 2), just like the equivalence between GBLUP and RRBLUP (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>). The second model we considered takes into account additive effects as well as additive-by-additive local epistatic effects among markers and was termed LEGBLUP. LEGBLUP is a modified version of EGBLUP (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Jiang and Reif 2015</a>). EGBLUP exploits epistasis between any pair of markers while LEGBLUP only considers local epistasis inside each haplotype block (Equation 3, 4). Note that local higher-order epistatic effects can either be included (Equation 5) or excluded (Equation 3, 4) in the LEGBLUP model. The relationship between the different models was illustrated in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>.</p><p id="p-50">A theoretical link between HGBLUP and the full LEGBLUP including local higher-order epistatic effects was established for the case in which all marker loci were assumed to be homozygous. Then the HGBLUP model was proven to be almost statistically equivalent to the LEGBLUP model (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, and see <strong>Theory</strong> for details). More precisely, the base equation of HGBLUP (Equation 6) is linearly transformable to the one of LEGBLUP (Equation 7). After transformation, only one difference remains; the HGBLUP model assumes non-trivial covariance structure for the additive and local epistatic effects (Equation 8), while in the LEGBLUP model all effects are assumed to be independent (Equation 7). This theoretical derivation provided a formal explanation why and how haplotype-based genome-wide prediction models exploit local epistatic effects among markers.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A brief outline of the theoretical relationship between HGBLUP and LEGBLUP. The essential case of a single haplotype block is outlined. LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. In the HGBLUP model,  denotes the vector of observed phenotypic values,  is the n-dimensional vector of ones where n is the number of genotypes,  is the common intercept term,  is the vector of haplotype allele effects inside the haplotype block,  is the corresponding design matrix, and  is the residual term. In the LEGBLUP model,  is the vector of main additive and local epistatic effects of all markers inside the haplotype block,  is the corresponding design matrix, other terms are the same as in HGBLUP. In both models,  is assumed to be a fixed unknown parameter,  and  are random vectors with distributions shown in the figure, and the residual term ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845346279" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A brief outline of the theoretical relationship between HGBLUP and LEGBLUP. The essential case of a single haplotype block is outlined. LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. In the HGBLUP model, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-365&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; denotes the vector of observed phenotypic values, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-366&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the n-dimensional vector of ones where n is the number of genotypes, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-367&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the common intercept term, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-368&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the vector of haplotype allele effects inside the haplotype block, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-369&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the corresponding design matrix, and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-370&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the residual term. In the LEGBLUP model, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-371&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the vector of main additive and local epistatic effects of all markers inside the haplotype block, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-372&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is the corresponding design matrix, other terms are the same as in HGBLUP. In both models, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-373&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is assumed to be a fixed unknown parameter, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-374&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-375&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are random vectors with distributions shown in the figure, and the residual term &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-376&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1687/F2.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/5/1687/F2.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489608" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-51" class="first-child">A brief outline of the theoretical relationship between HGBLUP and LEGBLUP. The essential case of a single haplotype block is outlined. LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. In the HGBLUP model, <span class="inline-formula" id="inline-formula-348"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-365.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-365.gif"/></noscript></span></span> denotes the vector of observed phenotypic values, <span class="inline-formula" id="inline-formula-349"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-366.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-366.gif"/></noscript></span></span> is the n-dimensional vector of ones where n is the number of genotypes, <span class="inline-formula" id="inline-formula-350"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-367.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-367.gif"/></noscript></span></span> is the common intercept term, <span class="inline-formula" id="inline-formula-351"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-368.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-368.gif"/></noscript></span></span> is the vector of haplotype allele effects inside the haplotype block, <span class="inline-formula" id="inline-formula-352"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-369.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-369.gif"/></noscript></span></span> is the corresponding design matrix, and <span class="inline-formula" id="inline-formula-353"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-370.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-370.gif"/></noscript></span></span> is the residual term. In the LEGBLUP model, <span class="inline-formula" id="inline-formula-354"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-371.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-371.gif"/></noscript></span></span> is the vector of main additive and local epistatic effects of all markers inside the haplotype block, <span class="inline-formula" id="inline-formula-355"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-372.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-372.gif"/></noscript></span></span> is the corresponding design matrix, other terms are the same as in HGBLUP. In both models, <span class="inline-formula" id="inline-formula-356"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-373.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-373.gif"/></noscript></span></span> is assumed to be a fixed unknown parameter, <span class="inline-formula" id="inline-formula-357"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-374.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-374.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-358"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-375.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-375.gif"/></noscript></span></span> are random vectors with distributions shown in the figure, and the residual term <span class="inline-formula" id="inline-formula-359"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-376.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F2/embed/mml-math-376.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Note that although almost all genome-wide prediction models assume independent marker effects, it was anticipated that some of the effects may be spatially correlated within chromosomes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> 2003</a>). Moreover, it was reported that the prediction accuracy can be increased by the Bayesian antedependence model considering correlated marker effects (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yang and Tempelman 2012</a>). Hence, the covariance structure among the additive and local epistatic effects suggested by the HGBLUP model can be beneficial and is interesting for further study.</p><p id="p-53">A counterexample showed that the base equation of HGBLUP cannot be linearly transformed into the one of LEGBLUP in case that heterozygous loci need to be considered (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, Example 2 in <strong>Theory</strong>). Hence, further empirical studies are needed to compare HGBLUP with marker-based models to provide more insight into the similarities between marker- and haplotype-based prediction approaches for non-inbred populations.</p><p id="p-54">Our theoretical derivations did not rely on a specific definition of the haplotype blocks in the HGBLUP model. This is important to note since the performance of haplotype-based models has been shown to depend on the method to define the haplotype blocks in experimental studies (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Calus <em>et al.</em> 2008</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">2009</a>, <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Cuyabano <em>et al.</em> 2014</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jónás <em>et al.</em> 2016</a>).</p></div><div id="sec-24" class="subsection"><h3>Simulation studies showed that haplotype-based models indeed capture local epistatic effects</h3><p id="p-55">Simulation studies were used to scrutinize that the HGBLUP model exploits local epistatic effects among markers. Six scenarios which differed with respect to the nature and pattern of epistatic effects were utilized (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) to compare the performance of HGBLUP with those of GBLUP and EGBLUP (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In scenarios in which no local epistatic effects were simulated, the highest prediction accuracies were achieved by GBLUP (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3a</a>) and EGBLUP (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3b</a>) and no benefit was observed for HGBLUP. In the four scenarios in which local epistasis was simulated, considering window sizes from 3 to 5 HGBLUP clearly outperformed GBLUP and EGBLUP in two cases (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3d, f</a>), but not in the scenarios in which only digenic local epistatic effects were simulated (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3c, e</a>). According to the theoretical derivation, the HGBLUP model assumes correlated local epistatic effects and considers not only local digenic but also higher-order epistatic effects among markers. This explains why the HGBLUP model did not perform well in scenarios in which the latter assumption was not fulfilled.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction accuracies of GBLUP, EGBLUP, LEGBLUP, and HGBLUP using simulated data. The data were simulated assuming a trait with the following features;  = 0.7,  = 4:3. (a). Scenario 1: only additive effects were simulated; (b) Scenario 2: additive and global epistatic effects were simulated; (c) Scenario 3: additive and digenic local epistatic effects were simulated, effects were assumed to be independent; (d) Scenario 4: additive, digenic and higher-order local epistatic effects were simulated, effects were assumed to be independent; (e) Scenario 5: additive and digenic local epistatic effects were simulated, effects were assumed to be correlated; (f) Scenario 6: additive, digenic and higher-order local epistatic effects were simulated, effects were assumed to be correlated; GBLUP: genome-wide best linear unbiased prediction; EGBLUP: extended genome-wide best linear unbiased prediction; LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. Standard errors of the estimated prediction accuracies are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (i.e., number of SNPs), varying from 2 to 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845346279" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Prediction accuracies of GBLUP, EGBLUP, LEGBLUP, and HGBLUP using simulated data. The data were simulated assuming a trait with the following features; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-377&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; = 0.7, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-378&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; = 4:3. (a). Scenario 1: only additive effects were simulated; (b) Scenario 2: additive and global epistatic effects were simulated; (c) Scenario 3: additive and digenic local epistatic effects were simulated, effects were assumed to be independent; (d) Scenario 4: additive, digenic and higher-order local epistatic effects were simulated, effects were assumed to be independent; (e) Scenario 5: additive and digenic local epistatic effects were simulated, effects were assumed to be correlated; (f) Scenario 6: additive, digenic and higher-order local epistatic effects were simulated, effects were assumed to be correlated; GBLUP: genome-wide best linear unbiased prediction; EGBLUP: extended genome-wide best linear unbiased prediction; LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. Standard errors of the estimated prediction accuracies are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (&lt;em&gt;i.e.&lt;/em&gt;, number of SNPs), varying from 2 to 5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1687/F3.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/5/1687/F3.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489565" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-56" class="first-child">Prediction accuracies of GBLUP, EGBLUP, LEGBLUP, and HGBLUP using simulated data. The data were simulated assuming a trait with the following features; <span class="inline-formula" id="inline-formula-360"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F3/embed/mml-math-377.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F3/embed/mml-math-377.gif"/></noscript></span></span> = 0.7, <span class="inline-formula" id="inline-formula-361"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F3/embed/mml-math-378.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/F3/embed/mml-math-378.gif"/></noscript></span></span> = 4:3. (a). Scenario 1: only additive effects were simulated; (b) Scenario 2: additive and global epistatic effects were simulated; (c) Scenario 3: additive and digenic local epistatic effects were simulated, effects were assumed to be independent; (d) Scenario 4: additive, digenic and higher-order local epistatic effects were simulated, effects were assumed to be independent; (e) Scenario 5: additive and digenic local epistatic effects were simulated, effects were assumed to be correlated; (f) Scenario 6: additive, digenic and higher-order local epistatic effects were simulated, effects were assumed to be correlated; GBLUP: genome-wide best linear unbiased prediction; EGBLUP: extended genome-wide best linear unbiased prediction; LEGBLUP: locally extended genome-wide best linear unbiased prediction; HGBLUP: haplotype-based genome-wide best linear unbiased prediction. Standard errors of the estimated prediction accuracies are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (<em>i.e.</em>, number of SNPs), varying from 2 to 5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">The results shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a> were obtained for a trait with a simulated heritability of 0.7 and a ratio of 4:3 for the simulated variance of additive effects to that of epistatic effects, <span class="inline-formula" id="inline-formula-362"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-379.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-379.gif"/></noscript></span></span>. The ratio 4:3 represents an optimized ratio for HGBLUP as it was derived in the linear transformation from HGBLUP to LEGBLUP (see <strong>Materials and Methods</strong> for details). We observed that the findings in case of a lower heritability of 0.5 in conjunction with <span class="inline-formula" id="inline-formula-363"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-380.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-380.gif"/></noscript></span></span> equaling 4:3 followed the same pattern (Figure S1), suggesting that the conclusions are valid for traits with a range of heritabilities. If <span class="inline-formula" id="inline-formula-364"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-381.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-381.gif"/></noscript></span></span> was set to 3:1, the advantage of HGBLUP was reduced in scenarios in which higher-order local epistatic effects were simulated (Figure S2d, f and Figure S3d, f). These results are expected as the relevance of epistasis was purposely weakened by the applied ratio of 3:1 for <span class="inline-formula" id="inline-formula-365"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-382.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-382.gif"/></noscript></span></span>. In summary, the results of the simulation studies confirm that local epistasis is indeed exploited by the HGBLUP model.</p></div><div id="sec-25" class="subsection"><h3>Haplotype-based models are especially useful when local higher-order epistasis is important</h3><p id="p-58">Our theoretical derivations showed that the haplotype-based model HGBLUP is able to exploit local epistatic effects among markers, since HGBLUP and the marker-based model LEGBLUP were shown to be almost statistically equivalent. As a next step, we asked under which circumstances the haplotype-based model outperforms the marker-based model. In order to minimize the demand on computational resources, discussed in detail in the next subsection, we implemented the LEGBLUP model such that only additive and digenic local epistatic effects were considered (Equation 3). Under these constraints, two differences exist between HGBLUP and LEGBLUP. First, higher-order local epistasis is considered in HGBLUP but not in LEGBLUP. Second, HGBLUP assumes correlated local epistatic effects, while LEGBLUP assumes independent effects. The relative impact of these factors was assessed by comparing the performances of HGBLUP and LEGBLUP in our simulation study. In scenarios in which higher-order local epistasis was simulated, HGBLUP outperformed LEGBLUP regardless whether correlated or independent local epistatic effects were simulated (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3d, f</a>). In contrast, in scenarios in which only digenic local epistasis was simulated, the prediction accuracies of LEGBLUP were higher than those of HGBLUP (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3c, e</a>). In scenario 4 (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3d</a>), the assumption that local epistatic effects were independent should have favored LEGBLUP, nonetheless HGBLUP outperformed LEGBLUP suggesting that the influence of the effect pattern was masked by the inclusion of higher-order local epistasis. In scenario 5 (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3e</a>), local epistatic effects were assumed to be correlated, this should have favored HGBLUP, yet LEGBLUP yielded higher prediction accuracies than HGBLUP, indicating that the exclusion of higher-order epistasis had a stronger effect than the effect pattern. Thus, among the assumptions favoring HGBLUP, the presence of higher-order local epistasis was found to be the most important. This conclusion holds for a range of simulated heritabilities (Figure S1). However, when the ratio of the simulated variance of additive effects to that of epistatic effects <span class="inline-formula" id="inline-formula-366"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-383.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1687/embed/mml-math-383.gif"/></noscript></span></span> increased the advantage of HGBLUP decreased (Figure S2d, f) and/or even disappeared at certain window sizes (Figure S3d, f).</p><p id="p-59">The contribution of higher-order epistasis to the phenotypic variation of complex traits is poorly understood because higher-order epistasis is difficult to detect in genetic mapping studies (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Taylor and Ehrenreich 2015</a>). Nevertheless, evidences for higher-order gene interactions from model organisms were reported (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pettersson <em>et al.</em> 2011</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Taylor and Ehrenreich 2014</a>) and new approaches were developed to detect them (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sailer and Harms 2017</a>). The comparisons of the prediction accuracies of HGBLUP <em>vs.</em> single marker-based approaches pave the way for a new approach to provide insights into the relevance of higher-order epistasis for complex traits.</p></div><div id="sec-26" class="subsection"><h3>Haplotype-based models are computationally efficient in exploiting local epistasis</h3><p id="p-60">In the analyses of the experimental data, the LEGBLUP model was implemented in a way that only additive and digenic epistatic effects were included (Equation 3). Thus, two kinship matrices were considered in the LEGBLUP model, the additive kinship matrix and the digenic local epistatic kinship matrix. In contrast, the HGBLUP model is based on a single kinship matrix. We compared the speed of HGBLUP and LEGBLUP with 100 cross validations using a maize data set with 833 individuals and 29,466 SNP markers (see <strong>Materials and Methods</strong>). The computer used for the test was equipped with Intel(R) Core(TM) i7-6700 CPU (3.40 GHz) and 32.0 GB RAM. The computational time was with 51 min for the LEGBLUP model nearly twice as long compared to the HGBLUP model which took only 28 min. Although the full LEGBLUP model potentially may yield comparable prediction accuracies as HGBLUP when higher-order epistasis is relevant, it would be far less efficient than HGBLUP, therefore we did not implement the full LEGBLUP model which includes local higher-order epistasis in our data analyses. In summary, the haplotype-based model HGBLUP is computationally much more efficient in exploiting local epistasis compared to marker-based models. This point may be of particular relevance for future studies since ultra-high density SNP data sets are emerging for plant and animal populations owing to the rapid progress with regard to genotyping-by-sequencing approaches (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Scheben <em>et al.</em> 2017</a>).</p></div><div id="sec-27" class="subsection"><h3>The performance of haplotype-based genome-wide prediction models in empirical data sets</h3><p id="p-61">Our theoretical and simulation results have shown that the HGBLUP model increases the prediction accuracy for inbred populations when local epistasis is abundant. To explore the potential of HGBLUP, we compared the performance of HGBLUP with the three marker-based models GBLUP, EGBLUP, and LEGBLUP using one animal data set and two crop data sets.</p><p id="p-62">The mouse data set comprised non-inbred genotypes. For both analyzed traits (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), the HGBLUP model clearly outperformed the other three models, suggesting that the HGBLUP model also exploits local epistasis in case heterozygous loci need to be considered. This result is of particular relevance since it was not possible to prove theoretically that the HGBLUP model is able to exploit local epistatic effects in case of non-inbred populations (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Given that haplotype-based genome-wide prediction models have been successfully applied in non-inbred cattle populations and outperformed alternative marker-based models (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boichard <em>et al.</em> 2012</a>, <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Cuyabano <em>et al.</em> 2014</a>, <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">2015a</a>, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">b</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Jónás <em>et al.</em> 2016</a>), the haplotype-based genome-wide prediction model is an attractive tool for non-inbred populations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the mouse data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Standard errors of the estimated prediction abilities are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (i.e., number of SNPs), varying from 2 to 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845346279" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the mouse data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Standard errors of the estimated prediction abilities are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (&lt;em&gt;i.e.&lt;/em&gt;, number of SNPs), varying from 2 to 10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1687/F4.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/5/1687/F4.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489653" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-63" class="first-child">Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the mouse data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Standard errors of the estimated prediction abilities are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (<em>i.e.</em>, number of SNPs), varying from 2 to 10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">For the rice data set, 26 agronomic traits (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> 2011</a>) with different genetic architectures were evaluated (Table S1). We observed that for three traits, such as protein content, HGBLUP outperformed all other models (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5a</a>). For two traits, including flowering time, HGBLUP gave slightly higher prediction accuracies than GBLUP and LEGBLUP, but lower ones than EGBLUP (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5b</a>). There were six traits for which only EGBLUP outperformed the other models, as shown for panicle fertility (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5c</a>). For the remaining fifteen traits, including plant height, GBLUP yielded the best prediction accuracies (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5d</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the rice data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Whiskers indicate standard errors of the estimated prediction abilities. The LEGBLUP and HGBLUP models were implemented with different window length (i.e., number of SNPs), varying from 2 to 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845346279" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the rice data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Whiskers indicate standard errors of the estimated prediction abilities. The LEGBLUP and HGBLUP models were implemented with different window length (&lt;em&gt;i.e.&lt;/em&gt;, number of SNPs), varying from 2 to 10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1687/F5.medium.gif" width="440" height="405"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/5/1687/F5.medium.gif" width="440" height="405"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489657" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-65" class="first-child">Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the rice data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Whiskers indicate standard errors of the estimated prediction abilities. The LEGBLUP and HGBLUP models were implemented with different window length (<em>i.e.</em>, number of SNPs), varying from 2 to 10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-66">For the maize data set, HGBLUP provided no benefit for the five traits under consideration. In fact, in all cases the best prediction accuracy was observed for the GBLUP model which only takes additive effects into account (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the maize data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Standard errors of the estimated prediction abilities are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (i.e., number of SNPs), varying from 2 to 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-845346279" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the maize data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Standard errors of the estimated prediction abilities are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (&lt;em&gt;i.e.&lt;/em&gt;, number of SNPs), varying from 2 to 10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1687/F6.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/5/1687/F6.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1687/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489577" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-67" class="first-child">Prediction abilities of GBLUP, EGBLUP, LEGBLUP and HGBLUP for the maize data set. GBLUP: genomic best linear unbiased prediction; EGBLUP: extended genomic best linear unbiased prediction; LEGBLUP: locally extended genomic best linear unbiased prediction; HGBLUP: haplotype-based genomic best linear unbiased prediction. Standard errors of the estimated prediction abilities are indicated by whiskers. The LEGBLUP and HGBLUP models were implemented with different window length (<em>i.e.</em>, number of SNPs), varying from 2 to 10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">The contrasting results we observed for different traits in the crop data sets indicated that the haplotype-based model will not generally boost prediction accuracies in crop populations. Instead, the effectiveness of HGBLUP may depend on the complexity of the trait. Analysis of the trait flowering time in rice and maize revealed that HGBLUP increased prediction accuracies in rice (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5b</a>), but not in maize (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6d, e</a>). As a matter of fact, HGBLUP failed to increase prediction accuracies regardless which trait was analyzed for the maize data set, in contrast to the results for the rice data set. These findings are in accordance with those obtained in a recent study (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Akdemir and Jannink 2015</a>) where a semiparametric mixed model with multiple marker-derived local epistatic genomic relationship matrices was applied to wheat, barley, and maize data. It was observed that the local epistatic model performed well in the wheat and barley data sets but not in the maize data set, possibly indicating the different relevance of epistasis in selfing and outcrossing species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garcia <em>et al.</em> 2008</a>).</p><p id="p-69">As the models GBLUP, EGBLUP and HGBLUP capitalize on different genetic effects in prediction, comparing the prediction accuracies of these models provides a first insight into the genetic architecture of a particular trait in a given organism. There is however a risk of misinterpreting local epistatic effects due to “apparent epistasis” (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wood <em>et al.</em> 2014</a>), a phenomenon which refers to the fact that multi-locus genotype tags may mimic tight linkage disequilibrium with an unobserved functional variant in the genome for a single marker. In such a case, the HGBLUP model would actually exploit the hidden additive effects of the unobserved variants, instead of the local epistatic effects. The fact that HGBLUP incorporates both additive and local epistatic effects for prediction is of particular relevance for breeders; in cases in which HGBLUP outperforms GBLUP, local epistatic effects or effects that are due to apparent epistasis are expected to be passed on for several generations, very much like additive effects.</p></div></div><div class="section conclusions" id="sec-28"><h2 class="">Conclusions</h2><p id="p-70">In this study, we investigated the relationship between haplotype-based and marker-based genome-wide prediction models. We provided a mathematical proof that modeling haplotype effects is equivalent to modeling main and local epistatic effects of markers, but with a different covariance matrix. Our simulation study confirmed the theoretical results and revealed that haplotype-based models are superior to marker-based models when there is abundant higher-order local epistasis. The fact that haplotype-based models exploit local epistasis among markers is especially relevant for applied breeding as the local additive-by-additive epistatic effects can last for generations like the additive effects. Thus, haplotype-based models have the potential to increase the accuracy of genomic selection. This hypothesis was partly supported by our empirical data analyses as we observed in certain cases that modeling local epistasis is indeed better than only modeling main effects. Further studies are needed to find out for which traits and in which species the haplotype-based models can be beneficial in genomic selection.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-71">We thank Dr. Yusheng Zhao for fruitful discussions about the mathematical proof of the relationship between LEGBLUP and HGBLUP. Y.J. was supported by the Federal Ministry of Education and Research of Germany (Grant FKZ031B0184A).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-72"><em>Communicating editor: J. Ross-Ibarra</em></p></li><li class="fn" id="fn-2"><p id="p-73">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.5986933">https://doi.org/10.25387/g3.5986933</a></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-12-21"><span class="received-label">Received </span>December 21, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-03-11"><span class="accepted-label">Accepted </span>March 11, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Jiang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.1" data-doi="10.1534/genetics.114.173658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Locally epistatic genomic relationship matrices for genomic association and prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">857</span>–<span class="cit-lpage">871</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.173658</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173658%26rft_id%253Dinfo%253Apmid%252F25614606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvODU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.2" data-doi="10.1186/s12711-017-0348-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isidro-Sánchez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Locally epistatic models for genome-wide prediction and association by importance sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0348-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0348-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0348-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.3" data-doi="10.1186/gb-2013-14-9-r103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Intraspecific variation of recombination rate in maize.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">R103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-9-r103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR103%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r103%26rft_id%253Dinfo%253Apmid%252F24050704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24050704&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.4" data-doi="10.1071/AN11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boichard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croiseau</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in French dairy cattle.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">115</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/AN11119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D52%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN11119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000301064800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.5" data-doi="10.1534/genetics.107.080838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of genomic selection using different methods to define haplotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.080838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D553%26rft.epage%253D561%26rft.atitle%253DAccuracy%2Bof%2BGenomic%2BSelection%2BUsing%2BDifferent%2BMethods%2Bto%2BDefine%2BHaplotypes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080838%26rft_id%253Dinfo%253Apmid%252F18202394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.6" data-doi="10.1186/1297-9686-41-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knol</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrooten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Effects of the number of markers per haplotype and clustering of haplotypes on the accuracy of QTL mapping and prediction of genomic breeding values.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D41%26rft.spage%253D11%26rft.epage%253D11%26rft.atitle%253DEffects%2Bof%2Bthe%2Bnumber%2Bof%2Bmarkers%2Bper%2Bhaplotype%2Band%2Bclustering%2Bof%2Bhaplotypes%2Bon%2Bthe%2Baccuracy%2Bof%2BQTL%2Bmapping%2Band%2Bprediction%2Bof%2Bgenomic%2Bbreeding%2Bvalues.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-11%26rft_id%253Dinfo%253Apmid%252F19284677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19284677&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.7" data-doi="10.1038/nrg1407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Epistasis: too often neglected in complex trait studies?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">618</span>–<span class="cit-lpage">625</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D618%26rft.epage%253D625%26rft.atitle%253DEpistasis%253A%2Btoo%2Boften%2Bneglected%2Bin%2Bcomplex%2Btrait%2Bstudies%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1407%26rft_id%253Dinfo%253Apmid%252F15266344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266344&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223039200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.8" data-doi="10.1002/gepi.20025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The role of haplotypes in candidate gene studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">321</span>–<span class="cit-lpage">333</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D321%26rft.epage%253D333%26rft.atitle%253DThe%2Brole%2Bof%2Bhaplotypes%2Bin%2Bcandidate%2Bgene%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20025%26rft_id%253Dinfo%253Apmid%252F15368617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15368617&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225524800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.9" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.10" data-doi="10.1186/1471-2164-15-1171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuyabano</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction of genetic merit using LD-based haplotypes in the Nordic Holstein population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-1171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1171%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1171%26rft_id%253Dinfo%253Apmid%252F25539631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25539631&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.11" data-doi="10.1186/s12711-015-0143-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuyabano</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Selection of haplotype variables from a high-density marker map for genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0143-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0143-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0143-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.12" data-doi="10.3168/jds.2015-9360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuyabano</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Bootstrap study of genome-enabled prediction reliabilities using haplotype blocks across Nordic Red cattle breeds.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">98</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">7351</span>–<span class="cit-lpage">7363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2015-9360</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D98%26rft.spage%253D7351%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2015-9360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2015-9360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.13" data-doi="10.1038/nrg2898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allison</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predicting genetic predisposition in humans: the promise of whole-genome markers.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">880</span>–<span class="cit-lpage">886</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2898</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253Dde%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D880%26rft.epage%253D886%26rft.atitle%253DPredicting%2Bgenetic%2Bpredisposition%2Bin%2Bhumans%253A%2Bthe%2Bpromise%2Bof%2Bwhole-genome%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2898%26rft_id%253Dinfo%253Apmid%252F21045869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2898&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21045869&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.14" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.15" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e1003608</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.16" data-doi="10.1534/genetics.107.082867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quantitative trait loci mapping and the genetic basis of heterosis in maize and rice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1707</span>–<span class="cit-lpage">1724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.082867</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.082867%26rft_id%253Dinfo%253Apmid%252F18791260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8zLzE3MDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">On marker-assisted prediction of genetic value: beyond the ridge.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D347%26rft.epage%253D365%26rft.atitle%253DOn%2BMarker-Assisted%2BPrediction%2Bof%2BGenetic%2BValue%253A%2BBeyond%2Bthe%2BRidge%26rft_id%253Dinfo%253Apmid%252F12586721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.18" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kaam</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.19" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.081190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.20" data-doi="10.1016/j.tig.2012.11.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewin</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The future of livestock breeding: genomic selection for efficiency, reduced emissions intensity, and adaptation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">206</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2012.11.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D206%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.11.009%26rft_id%253Dinfo%253Apmid%252F23261029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.11.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23261029&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317315000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.21" data-doi="10.1007/s00122-015-2655-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirdita</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection in a commercial winter wheat population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">641</span>–<span class="cit-lpage">651</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2655-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2655-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2655-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.22" data-doi="10.1007/s00122-016-2840-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bothe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide mapping and prediction suggests presence of local epistasis in a vast elite winter wheat populations adapted to Central Europe.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">635</span>–<span class="cit-lpage">647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2840-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D635%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2840-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2840-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.23" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.03.0249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.24" data-doi="10.1038/ng.3920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiurugwi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth cit-collab">Implementing Genomic Selection in CGIAR Breeding Programs Workshop Participants</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3920</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1297%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.25" data-doi="10.1534/genetics.115.177907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling epistasis in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">759</span>–<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177907%26rft_id%253Dinfo%253Apmid%252F26219298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.26" data-doi="10.1038/ng.3974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A quantitative genetic framework highlights the role of epistatic effects for grain-yield heterosis in bread wheat.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1741</span>–<span class="cit-lpage">1746</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3974</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1741%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.27" data-doi="10.3168/jds.2015-10433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jónás</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducrocq</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fouilloux</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croiseau</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Alternative haplotype construction methods for genomic evaluation.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">99</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">4537</span>–<span class="cit-lpage">4546</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2015-10433</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D99%26rft.spage%253D4537%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2015-10433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2015-10433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.28" data-doi="10.1534/genetics.114.161943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krämer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauland</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camisan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Usefulness of multiparental populations of maize (<em>Zea mays</em> L.) for genome-based prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">3</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.161943</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLehermeier%26rft.auinit1%253DC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D16%26rft.atitle%253DUsefulness%2Bof%2BMultiparental%2BPopulations%2Bof%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2Bfor%2BGenome-Based%2BPrediction%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161943%26rft_id%253Dinfo%253Apmid%252F25236445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk4LzEvMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNS8xNjg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.29" data-doi="10.1038/nrg3627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Epistasis and quantitative traits: using model organisms to study gene-gene interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">22</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3627</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3627%26rft_id%253Dinfo%253Apmid%252F24296533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24296533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.31" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unraveling additive from non-additive effects using genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.171322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.32" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peréz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.33" data-doi="10.1371/journal.pgen.1002180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettersson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besnier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ö.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Replication and Explorations of High-Order Epistasis Using a Large Advanced Intercross Line Pedigree.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e1002180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPettersson%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De1002180%26rft.epage%253De1002180%26rft.atitle%253DReplication%2Band%2Bexplorations%2Bof%2Bhigh-order%2Bepistasis%2Busing%2Ba%2Blarge%2Badvanced%2Bintercross%2Bline%2Bpedigree.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002180%26rft_id%253Dinfo%253Apmid%252F21814519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21814519&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-8.5.1687.34"><div class="cit-metadata unstructured">R Core Team, 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.35" data-doi="10.1534/genetics.116.195214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sailer</span>  <span class="cit-name-given-names">Z. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harms</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Detecting high-order epistasis in nonlinear genotype-phenotype maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1079</span>–<span class="cit-lpage">1088</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.195214</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.195214%26rft_id%253Dinfo%253Apmid%252F28100592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS8zLzEwNzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.36" data-doi="10.1111/pbi.12645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheben</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genotyping by sequencing approaches to characterize crop genomes: choosing the right tool for the right application.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">149</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12645</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.37" data-doi="10.1371/journal.pgen.1004324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic interactions involving four or more genes contribute to a complex trait in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e1004324</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004324%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004324%26rft_id%253Dinfo%253Apmid%252F24784154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24784154&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.38" data-doi="10.1016/j.tig.2014.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Higher-order genetic interactions and their contribution to complex traits.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">34</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2014.09.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D31%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.09.001%26rft_id%253Dinfo%253Apmid%252F25284288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25284288&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.39" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1840</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.40" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.41" data-doi="10.1534/genetics.116.199406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Orthogonal estimates of variances for additive, dominance and epistatic effects in populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1297</span>–<span class="cit-lpage">1307</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.199406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199406%26rft_id%253Dinfo%253Apmid%252F28522540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzEyOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.42" data-doi="10.1038/hdy.2012.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Basyoni</span>  <span class="cit-name-given-names">I. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baenziger</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskridge</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits with epistatic effects in plant breeding populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">109</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">313</span>–<span class="cit-lpage">319</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2012.44</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DWang%26rft.auinit1%253DD.%26rft.volume%253D109%26rft.issue%253D5%26rft.spage%253D313%26rft.epage%253D319%26rft.atitle%253DPrediction%2Bof%2Bgenetic%2Bvalues%2Bof%2Bquantitative%2Btraits%2Bwith%2Bepistatic%2Beffects%2Bin%2Bplant%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.44%26rft_id%253Dinfo%253Apmid%252F22892636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892636&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-8.5.1687.43"><div class="cit-metadata unstructured">Wimmer, V., T. Albrecht, H. J. Auinger, and C. C. Schön, 2015 R Package “synbreedData”. CRAN. <a href="http://CRAN.R-project.org/package=synbreedData">http://CRAN.R-project.org/package=synbreedData</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.44" data-doi="10.1186/1471-2156-12-74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittenburg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melzer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinsch</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Including non-additive genetic effects in Bayesian methods for the prediction of genetic values based on genome-wide markers.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-12-74</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DWittenburg%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D74%26rft.epage%253D74%26rft.atitle%253DIncluding%2Bnon-additive%2Bgenetic%2Beffects%2Bin%2BBayesian%2Bmethods%2Bfor%2Bthe%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Bbased%2Bon%2Bgenome-wide%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-74%26rft_id%253Dinfo%253Apmid%252F21867519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-74&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21867519&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.45" data-doi="10.1038/nature13691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuke</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nalls</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandinelli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Another explanation for apparent epistasis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">514</span>(<span class="cit-issue">7520</span>): <span class="cit-fpage">E3</span>–<span class="cit-lpage">E5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13691</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D514%26rft.spage%253DE3%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13691%26rft_id%253Dinfo%253Apmid%252F25279928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13691&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25279928&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.46" data-doi="10.1111/j.1541-0420.2006.00711.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">An empirical Bayes method for estimating epistatic effects of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">63</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">513</span>–<span class="cit-lpage">521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1541-0420.2006.00711.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D513%26rft.epage%253D521%26rft.atitle%253DAn%2Bempirical%2BBayes%2Bmethod%2Bfor%2Bestimating%2Bepistatic%2Beffects%2Bof%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2006.00711.x%26rft_id%253Dinfo%253Apmid%252F17688503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2006.00711.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17688503&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246843900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.47" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.48" data-doi="10.1534/genetics.111.131540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tempelman</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A Bayesian antedependence model for whole genome prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1491</span>–<span class="cit-lpage">1501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.131540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131540%26rft_id%253Dinfo%253Apmid%252F22135352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0OTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.49" data-doi="10.1534/genetics.114.166249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bayesian modeling of haplotype effects in multiparent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">139</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.166249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166249%26rft_id%253Dinfo%253Apmid%252F25236455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2ODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.50" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1467</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1687.51" data-doi="10.1038/nrg1270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zondervan</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The complex interplay among factors that influence allelic association.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">89</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DZondervan%26rft.auinit1%253DK.%2BT.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D89%26rft.epage%253D100%26rft.atitle%253DThe%2Bcomplex%2Binterplay%2Bamong%2Bfactors%2Bthat%2Binfluence%2Ballelic%2Bassociation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1270%26rft_id%253Dinfo%253Apmid%252F14735120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14735120&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1687.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188602400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/5/1687.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1675" title="A Whole Genome Assembly of the Horn Fly, Haematobia irritans, and Prediction of Genes with Roles in Metabolism and Sex Determination" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1701" title="Programmable Single and Multiplex Base-Editing in Bombyx mori Using RNA-Guided Cytidine Deaminases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/5"><span>Volume 8 Issue 5, May 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/5.cover-source.jpg?itok=Bl7MXdvV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1687" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UK9R6WT7OySOQl_8nxseu9hDTCGxv9JC-i-G6zuSW1w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/5/1687" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/5/1687" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/5/1687" class="active" data-icon-position="" data-hide-link-title="0">Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489447" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wFSBTUav2P3KHevhXgfssB8RLNd1h1a6ROaRfNwwSeQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1687.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/5/1687" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JBphexpn9MuJMlBHvnLqkR0dYUW4k4uOMe9b0G6Gl38" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/5/1687" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1687.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489447" id="node489447--2" data-pisa="ggg;8/5/1687" data-pisa-master="ggg;g3.117.300548" data-apath="/ggg/8/5/1687.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong Jiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renate H. Schmidt</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6742-265X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jochen C. Reif</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1687-1699;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300548 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489447--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong Jiang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Renate H. Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renate%2BH.%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenate%2BH.%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renate H. Schmidt</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6742-265X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jochen C. Reif</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/5/1687" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/5/1687&amp;title=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/5/1687&amp;title=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/5/1687&amp;title=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/5/1687&amp;text=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/5/1687&amp;title=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/5/1687&amp;t=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/5/1687&amp;title=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/5/1687&amp;title=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489447/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489447" id="node489447--3" data-pisa="ggg;8/5/1687" data-pisa-master="ggg;g3.117.300548" data-apath="/ggg/8/5/1687.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotype-Based Genome-Wide Prediction Models Exploit Local Epistatic Interactions Among Markers</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yong Jiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Renate H. Schmidt</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6742-265X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jochen C. Reif</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1687-1699;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300548 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489447--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yong Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yong%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYong%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2824-677X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yong Jiang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Renate H. Schmidt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renate%2BH.%2BSchmidt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmidt%20RH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARenate%2BH.%2BSchmidt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8037-3581" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Renate H. Schmidt</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jochen C. Reif </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Breeding Research, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, 06466 Stadt Seeland, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jochen%2BC.%2BReif%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reif%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJochen%2BC.%2BReif%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6742-265X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jochen C. Reif</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:reif@ipk-gatersleben.de" class="" data-icon-position="" data-hide-link-title="0">reif@ipk-gatersleben.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1687&amp;rft.epage=1699&amp;rft.atitle=Haplotype-Based%20Genome-Wide%20Prediction%20Models%20Exploit%20Local%20Epistatic%20Interactions%20Among%20Markers&amp;rft.volume=8&amp;rft.issue=5&amp;rft.date=2018-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jiang%2C+Yong&amp;rft.au=Schmidt%2C+Renate+H.&amp;rft.au=Reif%2C+Jochen+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489447/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489447/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489447/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489447/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489447/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489447/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489447/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489447/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489447/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489447/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489447/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489447/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/5/1687" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">THEORY</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="odd"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/5/1687.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/5/1687.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/5/1687.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/5/1687.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
