<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/5/1649" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/5/1649.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/5/1649.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/5/1649.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/5/1649.atom" />
<meta name="HW.pisa" content="ggg;8/5/1649" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform" />
<meta name="DC.Identifier" content="10.1534/g3.118.200087" />
<meta name="DC.Date" content="2018-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Wei et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Real-time sequencing of short DNA reads has a wide variety of clinical and research applications including screening for mutations, target sequences and aneuploidy. We recently demonstrated that MinION, a nanopore-based DNA sequencing device the size of a USB drive, could be used for short-read DNA sequencing. In this study, an ultra-rapid multiplex library preparation and sequencing method for the MinION is presented and applied to accurately test normal diploid and aneuploidy samples’ genomic DNA in under three hours, including library preparation and sequencing. This novel method shows great promise as a clinical diagnostic test for applications requiring rapid short-read DNA sequencing." />
<meta name="DC.Contributor" content="Shan Wei" />
<meta name="DC.Contributor" content="Zachary R. Weiss" />
<meta name="DC.Contributor" content="Zev Williams" />
<meta name="article:published_time" content="2018-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Real-time sequencing of short DNA reads has a wide variety of clinical and research applications including screening for mutations, target sequences and aneuploidy. We recently demonstrated that MinION, a nanopore-based DNA sequencing device the size of a USB drive, could be used for short-read DNA sequencing. In this study, an ultra-rapid multiplex library preparation and sequencing method for the MinION is presented and applied to accurately test normal diploid and aneuploidy samples’ genomic DNA in under three hours, including library preparation and sequencing. This novel method shows great promise as a clinical diagnostic test for applications requiring rapid short-read DNA sequencing.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/05/01" />
<meta name="citation_mjid" content="ggg;8/5/1649" />
<meta name="citation_id" content="8/5/1649" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/5/1649" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/5/1649.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/5/1649.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/5/1649.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200087" />
<meta name="citation_pmid" content="29540443" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1649" />
<meta name="citation_lastpage" content="1657" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Shan Wei" />
<meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032" />
<meta name="citation_author_email" content="zw2421@cumc.columbia.edu" />
<meta name="citation_author_email" content="sw3203@cumc.columbia.edu" />
<meta name="citation_author" content="Zachary R. Weiss" />
<meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032" />
<meta name="citation_author" content="Zev Williams" />
<meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032" />
<meta name="citation_author_email" content="zw2421@cumc.columbia.edu" />
<meta name="citation_author_email" content="sw3203@cumc.columbia.edu" />
<meta name="citation_reference" content="citation_journal_title=BMJ;citation_author=P. R. Brezina;citation_author=D. S. Brezina;citation_author=W. G. Kearns;citation_title=Preimplantation genetic testing.;citation_volume=345;citation_year=2012;citation_pmid=22990995;citation_doi=10.1136/bmj.e5908" />
<meta name="citation_reference" content="citation_journal_title=Regul. Toxicol. Pharmacol.;citation_author=K. S. Butler;citation_author=M. Y. L. Young;citation_author=Z. Li;citation_author=R. K. Elespuru;citation_author=S. C. Wood;citation_title=Performance characteristics of the AmpliSeq Cancer Hotspot panel v2 in combination with the Ion Torrent Next Generation Sequencing Personal Genome Machine.;citation_pages=178-186;citation_volume=74;citation_year=2016;citation_doi=10.1016/j.yrtph.2015.09.011" />
<meta name="citation_reference" content="citation_journal_title=Wellcome Open Res.;citation_author=J.-M. Carter;citation_author=S. Hussain;citation_title=Robust long-read native DNA sequencing using the ONT CsgG Nanopore system.;citation_pages=23;citation_volume=2;citation_year=2017;citation_doi=10.12688/wellcomeopenres.11246.1" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Chen;citation_author=S. Li;citation_author=W. Xie;citation_author=X. Li;citation_author=C. Zhang;citation_title=Performance comparison between rapid sequencing platforms for ultra-low coverage sequencing strategy.;citation_pages=e92192;citation_volume=9;citation_year=2014;citation_issue=3;citation_doi=10.1371/journal.pone.0092192" />
<meta name="citation_reference" content="citation_journal_title=Genet. Med.;citation_author=Z. Dong;citation_author=J. Zhang;citation_author=P. Hu;citation_author=H. Chen;citation_author=J. Xu;citation_title=Low-pass whole-genome sequencing in clinical cytogenetics: a validated approach.;citation_pages=940-948;citation_volume=18;citation_year=2016;citation_issue=9;citation_doi=10.1038/gim.2015.199" />
<meta name="citation_reference" content="citation_journal_title=Reproductive biomedicine online;citation_journal_abbrev=Reprod Biomed Online;citation_author=A. H. Handyside;citation_title=PGD and aneuploidy screening for 24 chromosomes by genome-wide SNP analysis: seeing the wood and the trees.;citation_pages=686-691;citation_volume=23;citation_year=2011;citation_issue=6;citation_pmid=22033395;citation_doi=10.1016/j.rbmo.2011.09.012" />
<meta name="citation_reference" content="citation_journal_title=Nano Today;citation_author=F. Haque;citation_author=J. Li;citation_author=H.-C. Wu;citation_author=X.-J. Liang;citation_author=P. Guo;citation_title=Solid-State and Biological Nanopore for Real-Time Sensing of Single Chemical and Sequencing of DNA.;citation_pages=56-74;citation_volume=8;citation_year=2013;citation_issue=1;citation_pmid=23504223;citation_doi=10.1016/j.nantod.2012.12.008" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=M. Jain;citation_author=I. T. Fiddes;citation_author=K. H. Miga;citation_author=H. E. Olsen;citation_author=B. Paten;citation_title=Improved data analysis for the MinION nanopore sequencer.;citation_pages=351-356;citation_volume=12;citation_year=2015;citation_issue=4;citation_pmid=25686389;citation_doi=10.1038/nmeth.3290" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Jain;citation_author=H. E. Olsen;citation_author=B. Paten;citation_author=M. Akeson;citation_title=The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community.;citation_pages=239;citation_volume=17;citation_year=2016;citation_issue=1;citation_doi=10.1186/s13059-016-1103-0" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT–the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_issue=4;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=Y. Kukita;citation_author=R. Matoba;citation_author=J. Uchida;citation_author=T. Hamakawa;citation_author=Y. Doki;citation_title=High-fidelity target sequencing of individual molecules identified using barcode sequences: De novo detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients.;citation_volume=22;citation_year=2015;citation_issue=4;citation_pmid=26126624;citation_doi=10.1093/dnares/dsv010" />
<meta name="citation_reference" content="Li, H., 2017 Minimap2: versatile pairwise alignment for nucleotide sequences." />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. J. Loman;citation_author=A. R. Quinlan;citation_title=Poretools: a toolkit for analyzing nanopore sequence data.;citation_volume=30;citation_year=2014;citation_issue=23;citation_pmid=25143291;citation_doi=10.1093/bioinformatics/btu555" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=N. J. Loman;citation_author=M. Watson;citation_title=Successful test launch for nanopore sequencing.;citation_pages=303-304;citation_volume=12;citation_year=2015;citation_issue=4;citation_pmid=25825834;citation_doi=10.1038/nmeth.3327" />
<meta name="citation_reference" content="citation_journal_title=EMBnet.journal;citation_author=M. Martin;citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads.;citation_pages=10-12;citation_volume=17;citation_year=2011;citation_doi=10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Quail;citation_author=M. Smith;citation_author=P. Coupland;citation_author=T. D. Otto;citation_author=S. R. Harris;citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.;citation_pages=341-341;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22827831;citation_doi=10.1186/1471-2164-13-341" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=U. M. Reddy;citation_author=G. P. Page;citation_author=G. R. Saade;citation_author=R. M. Silver;citation_author=V. R. Thorsten;citation_title=Karyotype vs. microarray testing for genetic abnormalities after stillbirth.;citation_pages=2185-2193;citation_volume=367;citation_year=2012;citation_issue=23;citation_pmid=23215556;citation_doi=10.1056/NEJMoa1201569" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria;citation_author=R. D. C. Team;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=Y. Wang;citation_author=Q. Yang;citation_author=Z. Wang;citation_title=The evolution of nanopore sequencing.;citation_pages=449;citation_volume=5;citation_year=2014;citation_pmid=25610451;citation_doi=10.3389/fgene.2014.00449" />
<meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.;citation_author=R. J. Wapner;citation_author=C. L. Martin;citation_author=B. Levy;citation_author=B. C. Ballif;citation_author=C. M. Eng;citation_title=Chromosomal microarray vs. karyotyping for prenatal diagnosis.;citation_pages=2175-2184;citation_volume=367;citation_year=2012;citation_issue=23;citation_pmid=23215555;citation_doi=10.1056/NEJMoa1203382" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Wei;citation_author=Z. Williams;citation_title=Rapid Short-Read Sequencing and Aneuploidy Detection Using MinION Nanopore Technology.;citation_volume=202;citation_year=2016;citation_issue=1;citation_pmid=26500254;citation_doi=10.1534/genetics.115.182311" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cytogenet.;citation_author=H. Zheng;citation_author=H. Jin;citation_author=L. Liu;citation_author=J. Liu;citation_author=W.-H. Wang;citation_title=Application of next-generation sequencing for 24-chromosome aneuploidy screening of human preimplantation embryos.;citation_pages=38;citation_volume=8;citation_year=2015;citation_issue=1;citation_doi=10.1186/s13039-015-0143-6" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="twitter:description" content="Real-time sequencing of short DNA reads has a wide variety of clinical and research applications including screening for mutations, target sequences and aneuploidy. We recently demonstrated that MinION, a nanopore-based DNA sequencing device the size of a USB drive, could be used for short-read DNA sequencing. In this study, an ultra-rapid multiplex library preparation and sequencing method for the MinION is presented and applied to accurately test normal diploid and aneuploidy samples’ genomic DNA in under three hours, including library preparation and sequencing. This novel method shows great promise as a clinical diagnostic test for applications requiring rapid short-read DNA sequencing." />
<meta name="og-title" property="og:title" content="Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/5/1649" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Real-time sequencing of short DNA reads has a wide variety of clinical and research applications including screening for mutations, target sequences and aneuploidy. We recently demonstrated that MinION, a nanopore-based DNA sequencing device the size of a USB drive, could be used for short-read DNA sequencing. In this study, an ultra-rapid multiplex library preparation and sequencing method for the MinION is presented and applied to accurately test normal diploid and aneuploidy samples’ genomic DNA in under three hours, including library preparation and sequencing. This novel method shows great promise as a clinical diagnostic test for applications requiring rapid short-read DNA sequencing." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
    <title>Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GQ1UHI0sWML0Z4agHQc8ccyjD50LPMDAenfgysAhHOo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489470","apath":"\/ggg\/8\/5\/1649.atom","pisa":"ggg;8\/5\/1649","ac":{"\/ggg\/8\/5\/1649.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/5\/1649.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1649"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1649"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/5\/1649":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WxHN5XwnUd\\\\\\\/93r4mCWm9XzRzfFxa3aVq0Pxun1pSHcY9NBWmrWuJmHnu8n6xiUJFp5QakYuLSzSgOkaRyjrJEpxqiXVLRRWfHWiEo+ntevFzKyJP4qGF7kmUkObHxSv9hWD+MPCcrPH\\\\\\\/oDpaJUTgcHqdr3nGpOgfblTDg2X+GmaKRpYkqPS42eiQQioKO0Ce23cYFMdoEFfE9jOEn9\\\\\\\/4Wcwvpo7I5AVxsoURrlOser5iB\\\\\\\/bH6bT54lAjfw235Nm\\\\\\\/AOtwMOilmaWRMiARYJ2PD\\\\\\\/jGEZ79nK\\\\\\\/dqVoxZmH1hbZa+raQnOrhxi9DcntYGdUmz2V2hA4NwNuxh82Hgsdr1Qj4aCt612xg1eJtKDtsBe6jUvZkFezML+PWFFklWuYiCrZG2TJP\\\\\\\/Rs5h9ltZRDF1vqN977sLPURQRMrKHzbsMd891BgQO5e87fuyUMX\\\\\\\/oJ\\\\\\\/3iCgfbYbPIUSpKLMl0iKrW1rI5fswphpymYs73dPGfW8StHJFfCkUA6AUkKSlBz0U+wcxgCGTcpQ8jQlATxcQTYMKC6cBAaUkdzbRSWuFkgf7LRYxGhvEV0udzHHfNaYhnYruosJCwD2N6DbvuprEqV++Mu6mKgHHArpUqQdSAav4I\\\\\\\/cDq5NY0C\\\\\\\/RuImQ2jlwhp68ewLty9Qy7p6UZWMTHDR0j7zoVPA3U3iZLxhVY5qO2NEiaDpmAlUvGeyFA1ffHru86All6F5dr8qnTAeJG5PmJ6xzRhffpvAebjKS8KCnR+zIxeKuz0on7J+SNKKHrIP6\\\\\\\/5XsthxUetZlOrSLcPkDaEeNjiX0svRkvJDvF8V5GWv7D1ab386I+2Fw5TfoeOe5ChIslZ5TbhSsFUy0d904gG1CChD1Evk6un8+KRi0opMwBA3msv8Pn6OO9RXCmrwML4CzL4awKcXnVSV4+hSOIthgo\\\\\\\/RMDM4PRVo7luLc1RjseDgdeGiWatIL6UztZYuF3sGrTr\\\\\\\/L7K+Gy1EvgIpD1cGYONQUjRj8eLThraR4TVJq3awIAhdnvFjiD7ktjbpLa\\\\\\\/XPRgMdYRcH1bgnFyHPz6zXwc797Cr9pUdtQ4h5hjmnhrBjFQFZ0Yo43HoWs6EBIQLwB7JLMK7B1cJEECA0Z2AiS0qa3m8sM1Z0EO4lbr+rqbmhaUpJW61TmoRVCxkITrsUpeXlZQQojbRDELSRWULemcbYBuFg2CRgFm3mG9BdGw8NQweX85ReAAhJPRplf3o8+9gX9OesjGcmbCnhNn5rn7Nxg\\\\\\\/ewG2ji9mF3bZz+ky2q\\\\\\\/xHaYq2+sq5\\\u0022,\\\u0022iv\\\u0022:\\\u0022dox9FiElZEP1xWwy85yooA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e175f3c976f8b2117965f86bc15ae056e44efdb0e145997d5e8d88644ec6c34\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w4SXnN+BX\\\\\\\/meonyTUIHuy2Exuo5up3WvyEP9spRtp319jRfUjRRL8uSr89FtJ4vqEFYDLCAKsVmrxftUfG6StxtMb8wy\\\\\\\/iJmS8WupFBJpFDC7HaFM\\\\\\\/3rqy3MpZyiXjHxRKVqnjAHPZdE95x0ZypWWypBr5HpOMczwns4wSG+YdFB+PLGD5AcNsyyBCFQw6x2DD6kVkYgxmM6oYEqLx7mCtmGzip1lfpe8SOgbJxY+u5RK\\\\\\\/iFOunlbcI9JdbLS2QBIutmNog95T1x0lIeYC1T6hwCUCuzksI5y4rkqr\\\\\\\/WutJybHRxxjow7K4A6UoNyHug\\\\\\\/+Lq+IOh2mntmcfSwDFFvJtPskDFOaaP7kYEpcIwadB8g8aNpNnAj8zDuwTsFxp0qPOkQo1gNvniLwJMOlpUWTTdBLCOz+B\\\\\\\/D3RPYUkWqp432SJVEubwJoIKSqXX42hpF184PWR24TFclBLEz0gd3kF4q3fLu79OO1PrE30vWL+bMSIJCFVHOZ\\\\\\\/ydwUTyzQvGXkq86vtF5PCPC2+m152gwt1FvkBdcBqikXVkMGQNG0nV1wzI\\\\\\\/dcWosN9YCS\\\\\\\/4hxfiHRpJcmNFlRIyoQ+rU3zeaLLWlPnaBQF1rTENQaXyohfxtw3YNIdn+fQ34tdHwf+U9uwb5aLsDspxx7NStJ75PpWLVWwXCHBi5vt0rzDNM00tLN20cWPjqdJYrnUE6\\\\\\\/v\\\\\\\/fpxJ5ArPOqADZdGHDbJZYvE0XAvIHsf4gB1ZD+DL65OGYHlscV6bN3LEmONXq5D\\\\\\\/jNUEi0JEjUNh3s\\\\\\\/jBdPtxYDkg2c28ue0Jma\\\\\\\/5u3M4y89+\\\\\\\/64K5EzNOQIBJ4JbLDaHob1bZeDUM4g3gm1BGRyQOfQI0G8iX6JgOX\\\\\\\/G7EwtrZWee9qMxzjdbtYp6f8\\\\\\\/+d243Yxq5q7WeOaeXF1byeH1rAofYho6Mc33OkAhIVzLFlXQyMckiDzokEt79B5bWfQLGyj6x3ArKgD4Hp7lGpXuzyL4xNvEinioeE+\\\\\\\/a9pWLuFCk0gl2dykOBjc1liuTulgdzzod8xDdGFlMdVXfGK5HXm6ldtz\\\\\\\/PwVUa3zNAfmEuBWr4xx1qntOSzJnVZR0IgDq+AFg8h+O8TZff8MMRCqif8QO0wzGEV61W4fYN7n0zLeZfm7tyCvpV9bFggq8rIqXXY0\\\\\\\/3tbBerXFBhTBLYi0peHCbhKD7ICn167FyrPxLEx1chGTx2vQGbhhqdJU\\\\\\\/YLT8vKbwJ6U\\\\\\\/bniJWlIvCt98ODrAXIMVVgep5QEsPMPYaPmUjcN9njWwA5LzybgbbyW1JSuGPKraSM\\\\\\\/CyP3pXU1rmUFd9ZTKz3KOlQVLXuBX\\\\\\\/ePQQRB6wDvrMPrKOBPqkPtKOSgKEaM8eDF05\\\\\\\/4euOF4fWmMZQuTXRGLoBLiE6uShyb7BQo+ihThJpcQh2ztFSZB25q+8iuOiURZd7MJPy0hWGY9GJuaBJtNubv\\\\\\\/C2AvT5swN9\\\\\\\/uRFo7fGWg5CW7vUPMMpT32UMC6vnAmk+3JDSbtAsXRW0iwf8UOhOxPmTpBipCjVhsKIOjI5T12kgNgTrQtuAZjpnorUbzXAOruxhvYFmlRvkz+gKprXD78Ct+7kRdIQku\\\\\\\/PyLffcQfI+Gd6iWKO5klDVcJrrtYQY0f0gEyO\\\\\\\/3XtgwRDMHGQscLaGgu9Y0hByuw2J180YwFoCP+1u9QdRVwL+6+eInKAh5H0BIRYzNi3QW6zxw+yF6EHWnrZ\\\\\\\/H5XeAES1NJQAiggkgIyT5rcATecGjX7nCmv7VTxN7aC+O8jFuUr+R9dA7mp0OWWf65Wp+akGPsXqjbfcDQ8UXSOGdSagjw3Pgvvom0LR5ilp+isfE1ZLvQd9jdtWpjNimjpJejcg0fTlsuiUSGHO1saEBAquLw7JQJQZkAAJjPjP5jdyxMvij5RfpPysxOnT+EXlqTF31WetDcAYhAvgCkY+CfgSnxm6ZOSNvNuVb4vzQ9dlsIuUYqDaWCHYhfp4ZipIi8XZ\\\\\\\/F0x\\\\\\\/BCY43ukwUsy\\\\\\\/3G24lp+YjGJ8xKCJReDfFPhhrOTSJ8Jnts6iOY24+VE9N01XEqN14sGD+jc\\\\\\\/zawS6ijhlE9eXU6QbMams0yyIakWHZq8vyfoRShw+jZI353iXt6+ElKnM\\\\\\\/mfCedBab5ZXFOtLL\\\\\\\/7dY0\\\\\\\/YFwWdXcR9exLQfTODodxowU22isGlGcDxVD2BEuy85TjCw50bvMvpf1uBOuS7Po6R3lPkK+Nima4K49PEdxvalBkdlPSFagHMXKsE34+1Gc00RcbevM5AjxpIk2D1cuPcZqbkmliHphU+mYdjwG2D0SeFoS5qzpgQN8WMm8Ykgmyuhzt6zS4NPl57G8s7WYHwaP14LSW7KXpV2MEg6vmgxI+K9tT+UJea+cPqndwF5dC6N6Cofl8v0TS9sQqNR4sp9DBJqNRU5eBQo3L7e7zxCKHhfQEPPNTpZ79mworLgDrbkJrNtU4rKREzV3rWhB07o823GrwQJvOFbqTAtCc0kWldd4Et\\\\\\\/3gQu8ED7JJ7RarSSS\\\\\\\/rKZ0s6lTuqAz4ADvxyExYJTiV1ks\\\\\\\/58uAM7q3KvTW9LcMYGlhlcuGQcIzBsMqibIlQrCP5TDpcr7T7PGlY1z1KOwc8MJ5xRhVl5XCZhv7Au+iIPAcK2PVou3nleJARJLSXxGYsNmE41chqFGBA9AA+Q23CyN5dkdX2vw\\\\\\\/l27qHg7Ut6dBcc\\\\\\\/mpxTyr2cmTVKo59QVXL+vlLZ9Huuq1vShliwusVFWFT70q23Ze4ogYCr559umD9gWn9eftJrYRAWSmppuoqAsnj\\\\\\\/VGrYiZg\\\\\\\/UFHlJHTOui1yDbV4X8DQXsQ54qNjQf8\\\\\\\/fi4LgE0suP6tTRPmaDQtlC7cTk3elE9oqorgby9Q3\\\\\\\/gOvm9VoBFwW5pDdr1Xevie+s+V+NV2j+t5C40M4MEDUO5kyq8BA\\\\\\\/3Y9ryZCjvL7fy2ED4W63jZrOBhg0DsB2AYWmqRpjBCNbIuzewHuA3MAg3CKs5Xl4xxRT+h3rjfR0Ja9RZYYHkKw0ntQHQFhLsaCJpd9HL4z2IQeo\\\\\\\/adHR\\\\\\\/H1f+Kexlk7nsLuVQyrVpHBqdq+HOCtoRiMFM7kPu12+s\\\\\\\/bh6YnzC5cEpscghjWZzPapLCPl8j5OlfuEMY64k+RjLDV9Du9ZjJgxPrO55hFk1gxPk2FNax9wQao3MCTQyYowx3swxBTw5MMSn9ZElKVENcVe7hcs3Mtz3KXO8QNp9FV9zhs0YxktpySLfyJdPfeOdaO2Ruw5YJ+28GLWx\\\\\\\/EF9gA0jri22yXygFhLNGyjikDnvaWDFLzy\\\\\\\/ekDYsR1Px6Di5A2fcUX5cb3kePXGcGizC6Krub1gNyTPQ2XbaBOYJlLDLtWvXvQZKIAijtJ+3wfM7Tl6srm2nvt\\\\\\\/uvQXY44CXGoX1D0isd2H2WJTcYGjOA1bdiR\\\\\\\/RhkPzN5DmXZ70tPRfqWMLzJSOaL2S0pmGUWHZOStXx084jxL2\\\\\\\/mJqefLhxJKvk+KXm2et63mPhqjy6Ket6D16i6NFV6+SI+TOV6fAN9dyqZwxPEjI+5uFVq8AIi\\\\\\\/spN1nG1U57aOQ3fSG7B0zlOclUM+K+RFx7AfPShp+wlNfSdfWbabHc4aGgk28leYlC06RIbM3yvSaTGtVASC1osuRMZauf\\\\\\\/kg3p2uod0KshnnqewJjuhFaBWUC65oxyHFMg\\\\\\\/fdVEnoozT1QSKUbYWV6quZZD30hPHxsTEY4dXGicilYjX66m1Q9jS2A8gQt\\\\\\\/EWVHSWt8ZME1XK+a2tFUmZ7TeoNWDfKRgBoEvuanotnRAg+42uSWVKaEWvlYo2PJS40kbBaDPL\\\\\\\/ldKSt825aDgd+Zt9BVPNfYPnoYSNm6cyBTgJf8QJrnBN2aNRXzMk41v49ioNrAorOITWrijRAKD0rcbB5xUSRAJIGNKyrDZyEEcacJTzovrFP3iEbClBsvz080MhfXUzdx\\\\\\\/qu4eVfKQLB7on8bC4ge4JPE3enl7vHhusaogQTDPRlyIT03zvNmf16fvjk6U\\\\\\\/3aACOm2u7aS94l1TI0pzqFCSKrxjP9wPgbo4wuGHHcRCLteQ09QwrHXX+A+IFahhQz4lN\\\\\\\/i+cSEB6aRdH+r1aQonwOQD0ctgStNkifyWiyNLeHf7Q\\\\\\\/veJz1\\\\\\\/JeQzkNiRlNWY15n6hTBti\\\\\\\/vLj6CODJCn6yoWGJJ1PZb+09lW44oDush3sBnuCwNi\\\\\\\/CRdEGRautm2XAlnVMFTdt+P9Qn+tx5vV89Zs+gjueCPftl6MD8F5M42vJT1v\\\\\\\/WKVCPVvyEXBghOxH3xHiMA8enidMCErT1yaX8Bx3cvL9sn0AsuUIHXeZx8UKqTAcNdr1nVhQVsJuGd\\\\\\\/9jCDwNnUt\\\\\\\/BpESR6l+tw5rcjU3jGm6q7Ofi6s3P1QWwxyoF1OlOJxGDZy+EGPL5KvIgZeNpVs1Qvmh8LG2L5bWuKkFMj+ODCx0XIvFfFz6uuG3HV3s102N4k8SYDxnpcb\\\\\\\/BnmdDAc7+B5RCL3aM9DXgWcAMKiU2dAS7AOhqF7FSMr8VEMbCifTMGv3vP2WQbNumHf+6NqSh5RbouTBbjg90cB0BqzJZpUVk6qcU4vKa4+R5AF3Py76+88R2mErGc0CInfn4S+\\\\\\\/g1b2BwL9YhRZmcW7S57B7jpTalmfGMjPfcAfQpjaAHm8RxtcAanrvZ66tY2XXPnHaXWUzs8TN2UU5lu52V\\\\\\\/I5z2jZVCFrfOECz\\\\\\\/eRj4yTgdb04zArvEUuMU512B\\\\\\\/utxSMz2rQdWidkFtUiO1WQHSNZgWjjKfzI7OZqZRMsbyG4gAuWjEDo1GjLMuE9U7zBl0M0ZlLg\\\\\\\/PAjMIgsuhZkdW+aDnIyoTOC2RNDbfoOgQvH1DAEKH9TW7Ash395Q3GETgqI9\\\\\\\/g3cvDDqT\\\\\\\/eZMQ0a6NZyQ0xjam9il9w+mFHlCmo5SNUGPi\\\\\\\/AW4SkkNVhUMCz2aRwRekAhRolkxyl8qYSNuScOWo3eYrsxzQGu7Pp9bRzcs+9UvgGz+grPZht\\\\\\\/fZfubtkBAmX+ELHdvgj1YtxwQl3JFi7vNWa84q0rbb+HBDdMeuRuvmg71iUixPf5vnIWPIhgu8wb\\\\\\\/XC\\\\\\\/4llOjyLiijhS\\\\\\\/EM4e8AoPxioPYbCtLmOgNy4PbxvSi5+RxeBTIqRLAhR7QXAuCJ8poeqP0m2Fj7+rqQN+0A0XQhQAD57yQqnomqAdrpWDFGFOHKOIrB3HmCNwpmuq7txbHvm5QbvM7WQg1c3ErEXFMZCVK8t\\\\\\\/h2aRexI4rXEFbwJdHkWnHw4GK\\\\\\\/q\\\\\\\/Xpnt0Kubnqa1CySOCpL6Cg2s6t27J3ULzb0QeF3k0BqnY4IcSbhxKXRQMdjemx\\\\\\\/tfhlcHQviH\\\\\\\/+WJhdvV2FUucW7AfeSUMuh1PbhvvqEeCDcCCX3qQaKUNMZzz0vp1TIWKw4YvDpui8HGXsvS\\\\\\\/hQV4URzM0Pu9pOzPGt1WDvaDTMyThcFnhhWMhYRsz9VyO132bNQjOTifHusqcsKTwBrzYsCOITaZLtpqO5fkBEablLT5NhZHiEeZD7MrkGqV5Dlqi9++YqIZBaSlf502PQApX3bXgpmlP5n7t0N3o5RlVi80Ni+NfR4QGPEZ3U1Aj+nzGTTtDtgGiXjQ0eHhVLTCN110FcEp6\\\\\\\/itNTnr6JzPuQzQXZWu2ACtEBLxc9\\\\\\\/W\\\\\\\/PWWxwL5YNkpgUHrAZRNdo8O2zJpA30Kcr3Fsp\\\\\\\/pv\\\\\\\/yEGDlUAUWDztlkFDaaljWZ6f1nZ\\\\\\\/V+x\\\\\\\/TXNTuTQVhXAOPpaimxGuJklGoTeEon3+JeISEY4j+uXKqWSI5k1X2vjcMN\\\\\\\/XvxDE66rdfwFJicEgokO67zVyHETBxO0wT\\\\\\\/hEARDjssDBTrySLHfPXEv6TknfAzl\\\\\\\/T6eQJHSaY4n8hPQ5ZIKzyuU8h5\\\\\\\/hozD5PAUwNXb+oaZVI\\\\\\\/ZTDFnsRTABuCNKceYDQfjItygufAyrfgmg5I+YMtSLvpiREVtcfm10y17Ts5BM+FcqoiM\\\\\\\/Kx5ygv1LFL8iKtIDbgQUGsrw66650CNMKH+AmjUMxUw\\\\\\\/mI\\\\\\\/+bn3R2kOw1ct4ml9uGJyfxhu\\\\\\\/Zvh27XgU0XZtpxMYjCHmKImzAC4txCx4V\\\\\\\/fSWd3hf\\\\\\\/RK4H5tfxSAK7XCFywrQt8bk4ojyNZwff3FLViJcbfb1C50BobJlIqjg1vp1rzk3DBC4oDMMcA3ZY8e98IuKrtGN+cVUNpuErVsvdEvJstPgiOSOfoHgMRTtCz5dHL3lJuNOvmL3nSVXdHcIv5XHP9ez5hObDKO3N26NKMKgzZD+9sD2IHofM2vcgWX\\\\\\\/4ID8z\\\\\\\/owWfkkmKcoHgkA8HPV+NMdv0ivecAcCPcTzUAJx3rMwbCJAycdV1B2Ev0X8KA8qKO250OrVbRdUF0oX2GEuW2C9kRKVnzqxHO0eWXDN3K4HLd6BYholBXqlgcukk02S\\\\\\\/EfoLHePtoadLFxVJq+t9QOip3rGNDYOKoCyhQKe4sDDBDRhYsBAa+bLogqJBgK6k3vsojoJ83Bx+3WyHR1Dhn8xojVdZ1JC\\\\\\\/qVqrpQzjC\\\\\\\/Oc1CyG5mn726gtFW7i1TVjtg+AvR168YU0+RjWUlaIv6ZBa+rDNTqhZ2Jg5opODlNvq5NEcNSvrD0B8gw9N7sncSsF0Hf2i9NGh3Ess2+AeiSFMBw7hyX8lh1ayMLY9yXYUOYR4ksnpTbb21FojCLqt0ZxWVtZu3u9keJDLF3P+58lKh5dDBO+sgVAh\\\\\\\/CldHFbOEiFO2w5EEXxa3OM7WeGJLHi\\\\\\\/O2VUXXodtXsvXNra2jC8C7zWN1B\\\\\\\/ZVFcTOy6Mz8bq\\\\\\\/7bUS0GO1C3phQl6zAQqEOWeRaxTTs4fD5AYOMBGRAeNzZMuyhBCJB9Wr9tBpU2d3iiwXdymxClvTez+7ZtrFJ1gPUOIehd2pLhF7+J\\\\\\\/UBjhy9LVK1ZTnE4p9vvbstY4Y8561wrQQnCzKQAR2F8OEoKnP4gmlcB82K6KaMLzk5Hc\\\\\\\/xqAW8LVo5G72TpAKb8KSF3HqHphCm34JAm1Kl8qVYtOBupx7Ywas\\\\\\\/GBxH538DJywbK22ei08vrZX3rgHutzet1y\\\\\\\/m4Y+0nDXZGNd8C4CQPpMPEWzREYzsQRb8e7Z4El2wDkL4wehMDENsEBL9trvd3wHUdrdJ4zvGgJ+szfUaU4xo\\\\\\\/SGFqD\\\\\\\/FRG5iVMMpQmsxyS1Ncyzr1YQDgfGGemAwVyxmmXooIQkSckvShBPirNsQfObAFvifNBoNAtfs9LwmsDN9ypGnkSMXP5idYGKR5ZI0QK\\\\\\\/T7nl98EklVwKTgfPWIL8sePxUJPg00BaLVD5by0obMo6pKCIdufR7MZHT0qwBy3OTGYEknDYi+LO85IzU2CRvhNlWCYg3XEiCpj9OM2ma4Ww+J0NKKs2aSo+ZEw031hOtK8bk26\\\\\\\/ileDgdefGNEs2OC\\\\\\\/fxvZ3UT+uoyIUXbP9MzqJp46NJKx6tVYnv3oJ9TqoChGSmULqrg2rDGMiDak3tVf7HK8SHe7DFg8E7+uNp8hGVt3BIKLhkw8RzcxoVYSiHqvprGFPNEyySPpEE5MJFWNsJMH71hq5qG0zvYZYqskqtJ2lfP8sw7VpsiwptRQ\\\\\\\/ifa7iYJG7057jQ7JB1eiB2FlpHPQhzHlHoZfj+mOxkceCrR7o0ylCeIYWzGkkGMtBx1aUPbPUn3jYZgh3Z0FIFc5hDONR9JI9wXrdm0EuqNeducoQQNYd1NgywFspqoGQree4Dc5PETy9zBhOwRRC7SgAtwKhsRiBi0giW+npc6IPY+kzOb0nCBqmdOlG1X3R8U8wKd6aRHwBoyIQ86+C1PuzPUfZjlOxy4Zrou25BNl+0KX3j2a97aBDW1zVWcpiUmA6x1UITre1ZgpMeU3m4qYNNR1CF71U35XjP7XzuzqWqR4F1zr\\\\\\\/5OEQEtKUnj4YiwfEoQ7uUHJmuLh4Oep7bEk1fnbnm8xT81IAKAVjpHB96d5rejDXHSjNIuqo0+8ngpu5RoR4P+qKNxpMgqE7NlAehSXQQzhAUWtFMBSCo5IOnvxOo6yoRUyLE8whI6vkdJvG0BC7XxCVsvnGMbVdPFopUH86gkCMI9dQK5gWS6Xu1cYjMv6FQApdIf9oHqC18iOEn4du3zDX16\\\\\\\/LtNcpxvXUyRqjUaWRMyWSCHAP4qqO2sKUrE8ccGkKgE6a815P8kUAfId5cIks0rByD6zi76W59gBAvAw+icFmuS1aXJb7TTIUWGhuyiwx6K5lcLVu0M5jgcBLPk5Rg6\\\\\\\/YxGww1tzkkK2CGWm\\\\\\\/zXmqaVKuaKlgN13TWiiw44htGk1tTJe0PzCeS7KsaCwfRA2xNANnTCVF+Jic27MXIW7bl7sUTOGYwkSeDPFnsclUv94vOJSYFqYDhGDDr22\\\\\\\/JNl2ksFyT1iPs\\\\\\\/6ysxOGfcjBPOybXm9CsgCrbvkl90M8aSvLfipPTDjIj+oUapDYNL4y8JEQhgJq7gBRPFFrDXaaE4vxPhraVFxsumrPXzORJXBs48YY8f3AmF25ML750Qp1nQrubYoMx6iAul56qmcTgWVH2PNGcfngliZZHpEK4Ty+EdZVm7LQMu+RYnsgUEjib+Cl4KnJ9jXgHPw7GzUdTF0W69WaEfWH3takcPdkinUNoMAz3w8ZRQnW1XMci2PA4xzx7gz4sWoYutNBNCsn9jADu4DxftldccoAtDHQSBMtGbVYJeH41b8Ia8OLeez2UuwcqPDGaZh0dNDeaWZodRSJ+HCD8hdGejiBMvWtbQQ87BE+rVU2RNWluIkx29IKM2rfSzBl\\\\\\\/\\\\\\\/KWAMhYic8bDtNWjUfTl9U8WAuQBIcSMs0yfO1zeTgLQ\\\\\\\/Phco30Z8lVMmU9I6YfV8uAdrtkUuJot0xCUG2LztBVBxS3nMzq8Rdwf1KLQOCKnjKYbknYWZ+shpFaLc\\\\\\\/3yV5C26wkTVoIdAlIYtWIaX0xgsHlKsu\\\\\\\/6\\\\\\\/47Spjvgedv5dImCdzPAXgz7I7zBtULVl+PFdnEqjFA0C9L4OFYyeNwAgUnzDJEw95AVMCD8vzaH0gd2EdAuXcWAX3wy+uvnsbY5NUjh18hs1RTlMCNGDWL\\\\\\\/wUWbxL9cN4wXNffZINiZDXH69njpBrfMmsNBVZZ0EHDqOEkj7JH1\\\\\\\/K24l1dNASQxgFFWepEzrFgH7muovZksPvAqnUM4b1jqoBRO5V1R1JuTFfNZkO6eR3ENcw8JhleFvL7Ehfh3acqmrG4laS5X+4QyjjuC8+7uSgcgo96MhoeKUSzV9fMHY1NXKSxVsNEVHV6VXmCi\\\\\\\/iPLPbRuOBySvICKylrzn\\\\\\\/XD9gzuPLIh3L8jNoGgakBKVZpuzYDYrhZoGZb2+rH9C\\\\\\\/sJhmstP8OnjZKiediPr311R\\\\\\\/Lsk1SwQpWAN0\\\\\\\/5AYxU2q2kVyDvd7PdPQ+CkLHtmKNbJf3IABcmpkQ81w3sOuYuKDs+QkyzIlvIRFnNhptGUEQXbnNNNQpe98WdGKgrm3FeTQI0L3FnVbDvZxvMbbZBJZhZH1krZju1rMz1fCFRPdtIWBBGXGGE3weoC9S79c06p1ZtDRzwkcR7qW2rKE\\\\\\\/9nVIXD25lwLshI3WjynRiUEbLzryOQ\\\\\\\/mI11pnx6EgHHJId7MaONZSC8MsSUp+13Pv+wMbUDIdEw702UJ7B92W\\\\\\\/g8btrDahst6Y7x1i0qWI85SRUrLQ7yzN\\\\\\\/RD0ecrn2TJZFRLEdjI0rkMPXz+w2rPh6BwNiyhEhEqOdT7ez89fjgtTKjLagJ4Ruz1wppMMV03Er7l4V28QEQ9\\\\\\\/Sk2nuvDbGgczAZH8F8lEdapnhkwSvUK+z0t99a33rAnboEJ4DeMRV0cZvyeoccHb32m2XcGej1oog4rzim1vsMuZH7DidkFAx+LEs6QpDv2PrOuNWUO96QxKPweOC18bxd22O3dDxjpChGF\\\\\\\/93k43hPxAaDdWy39E77fRX0mRggRz\\\\\\\/x2H7GnBVHiK66zxUV0bV1TUJUK+ziXgeqBYHhnVGatCdDlh8KqWe2WRSNEz8M3HqfvRNFDUqPW7Pa9UxlvdIeqXqq3mLutSPQ1HOFz75Ddbk\\\\\\\/w1xnpd42dvzoFKvK736K3LGgQ0zbGhkzrJZrrtdJAXCiIrN6dl8bfJyjcbYJm83U2ytenXR3saG3hRzxgzGNteaxvmTCz\\\\\\\/LRe7u+Fl4NMGdxbDmCZ3qBtOAQUg+5B93yUUF2HJV2T3RS8y19NfwnUPWTFQ7USQ8\\\\\\\/1V+8B89RizMOVZ0fJUEhaiLnyEcQ0FQyx5of06hWdbsRklXSWJEzqaGyuUJ8qmdrXIw0P1SgV\\\\\\\/DWiJLpSGp0xT5lu3jrrevZfeqbdN8U+BgE8qo0LKhr8ukp3APCyDz5XsoPR4uTnzZH9p6jF8eJ6svRqVfs5pOdLJWekG+Dp1auJtBh2a267+aoUnUHSpcC6pkAPNptEKBVQLEEBFmvTPsSLEz0ieTwEPaGXEwYQUxyqB6v4zd62MR2uYbK\\\\\\\/tPaq4Mf4IhnkCqNcRjTRD2jjMCkBwBK\\\\\\\/\\\\\\\/WvLLLlCB31XtejicMoLb+hTuK4yK6C\\\\\\\/Vd9LHUtk6LErXEuqnOD3i1M5FTRNg7v\\\\\\\/SxsGsBee0kbIsktE9tbZ7p7VJu0xc8OkWeZkY+\\\\\\\/+mspCqNyINZ902AICtKPFB0aVBHbKsxaORVB\\\\\\\/rXcHbGY41vwOLEvm+\\\\\\\/GoHPoXKcbwwW0cYJKWotM9PRjZu8jTXp2MutRp6COJjK3YZYaKYfOqmBo7+OevAWUxtloCFbC46YQhP2dQfuHmC6CEobdeehMl\\\\\\\/YXB0Pf5TpfEDj2j594EBa37AQ7yjPIb9xj7uh1l1z693YVDssz\\\\\\\/+RjbMaIpm6F8Z6X2gKe7ilOCABwyd0s6aaEUDm5jowTs4PnyZfYvymD823X+cn9aMyjAl6RpXSeBf48O3io6LxijGIXn5EsDEzBckbVXQsKHItqZmZD4gAH0H\\\\\\\/cIIHxLJlDZxTS7Pqsq15pzWNpuRVII20MjtdYt3MUG79Tu10Uo5VImHD\\\\\\\/f5zHVbFLihymndjb47d85VdLJEwZSQouAsI87hpivnN80pILWlytwusAu50jvU34Ad5kdcPE0WYHN0sn\\\\\\\/8KG3ekY\\\\\\\/6I5gQnudhVbEqjMx77kVSYUQNWe1ivXB7MseS2m3\\\\\\\/I20NvBJEB+YLRoXQaa19Ra6PA5a0ZdOLP2Nyf050RFO7fqLg9nhjJ7JJRXLgOWjWBCCXis8J5amHxlPZkyeBhnZXnUFBnq8qxUS9w9NEYkSxYKPjiTZy6sjBjtsOaF8WigLTTB6gJq8PnzY2yt8ZG332JwJfP3r6J2zn\\\\\\\/Lp\\\\\\\/S1x2NREJKbs5y+mCwN9cMVISfitwsqdfd7HccEbvSXT5k9GCg2TR3eN03ycBtHHVW2mV48gn64bFKPKznmTZF8pxkxuLKB0Zn8T9hJzHgXPCfBGnqGCIv6ZLmyciIX3K\\\\\\\/VvrJ5uitI0p4vJtEdpiOfdguPFr1f8cKci1hIo7eVQXdOppetFsO6oc++0zMprm5Vknu+AduDbdXTJlmJVX4A\\\\\\\/FCglr4kihkXdtgCKHITlNDed65hwkih1nc2\\\\\\\/096sl2+0pCVDIJee8Rnbz+o2JqbWN591V+VuduVCrc2yYk7GFv6b3uORG3D2XpmHObBZc6vmyKpoKTkKIfNaXIU\\\\\\\/9IsnY51CeGz9YVESrNT60xRMuaqDoZubCBIvVv47CwLzFF83UX5gXABAAM1ar3Cr6Gq52KfTPJMuqhKXbKnIFRCIqgNxJtG4VklmfMkFVF3WedOn4Z8yxrAZpvzZrB5dIfziLEQOGdfhNpn4lTPqIQOilwNy7O+oHZighPvoB3N9185virbxqUceKjaeCArVIUHf8ujKEoUdzrBE2CbqxATthN7vsJfMxERHXZNyNAhfMlNBoRgUhkdXw8Eqju0aSRV6q2UHjKRrHkaJKdPum7wovzJjH+K8uWCxGH+msHcwJm1\\\\\\\/1ChAS9SBTWi\\\\\\\/CNFFq+bDkZrkBoAL78tU5WQnFK+F4g2+Hj4fmCNM7VQJzYmv\\\\\\\/MIWPiJZdGmSiDtPz1ohAM0zVo660WNdqxKb2hZVQos8a9Np9SVdROY1M7m3d3BMwc8W3puRgcA\\\\\\\/AC92PUAqnEMAW7zSJ9CxeQrMx9a3rVaq61E0OlVs\\\\\\\/mPhzot7sQ6gmaJimavQCUsaTa7rObYVqKt0mxjq+rWMB5rCJAcKLEMxIsBjnHMmtNzdNnzeQaFnHh6\\\\\\\/aT+2J6xPp2IQT7WfnHD8KPAfqUINg5bQlXSN3334spp0pti+mTxY7nXqdmmGzY+AZMSUiMBJftRvz3ImooZjulsD\\\\\\\/NcW3JmpA\\\\\\\/AzZhxTfaY45d0BqkjPGfIZsLCkYwZknGhGWGzp4mhC8S7CA7THFjA9syKT7d4uofYNlKsPP4Jo9uA75pkih0FNBZ\\\\\\\/LgYRhPvvJLSDnStiE4vt0O+SMXKRdAhmWAsMjUjj+qx\\\\\\\/4xoXbIEsFMfe1pP+RymMrnTNa+qdv3rBiHjfBmU5c1jCQcwD2Sj466kPEhurLdHE++VpIanEwfuHOLOSWNn+al41RQY5DmFRmy0t7Su0c4\\\\\\\/aWXSKveaiBOEHGDqw6zNp0vowO8Ftj6PwIbDjBny4+Kx68VYsxyWLj38DGmUfhdul6\\\\\\\/nXTIJNi4FsK7+lTOTWTm4yaj0lUHSB9h3BD7Rk2ntkIb3aYkeww\\\\\\\/ltlX4ulLi15jKOL\\\\\\\/TE7CcKXjdz+g9JJisWGQj6f6KkmG6Kt8xpjNYXAdswPuiTjaiVNlvclv9AUIxR50VldjYcC\\\\\\\/qschO1pm3rCS7UA8ZUQ1ggSh8Vrr73Vd5ENEM8n5E4XSLBVw3cFEIWZlDvc\\\\\\\/WnHRyYoNDUPZfPgPdsKshuoQ2qA8jtvTXyNTFsWRufi2ZOnMTy9Id66hvsjL1B9Ka1D9GDl1Jd0FIb3iMIrIFicQ8reuMqrHdlGpcecv1bkuh6CuLzoXK1m32ww\\\\\\\/J5jxuRWwE6E+5MiYY1XKCaMpoa1onP06pllVcyAnwp1jJQRiQ+599TlyygvDNyzeXRsWpdFksv8C0VS0449Nhfmdwc7LmWo6I1P\\\\\\\/yqvGRmqCg4QcX9qEAi\\\\\\\/9HPADIYAAGNJcTyK3owRkPOigJsUzjiSQKQnN+q389bP6pYlYKoxBEk2gYs0hNREz\\\\\\\/F2NCpAiGp0P7cKbhKjNuNsCezXYBnxn5Ohlz62\\\\\\\/S5532SlH4W249tsj2IUZmsM6pGy5D79KoY3uFv4TBBo+641VcU\\\\\\\/avIT6RRDXNv3neQ138Mpb5t1vqIhWh477ndjMcCn+xJjqp9Y3b8eyuhUUvgxZx9NydPCDtuQpRBPJNHHNJss0yhG4tDWYCL6ia9E9csrRtVXYcs1jLeUpFRHidTiHQTimXj14rpg9y\\\\\\\/rHGicYycNE4YqUEAYfFQsfSbSv9PuEXSww2G6\\\\\\\/y33A5laW4enQmc9OJf7nQ\\\\\\\/i\\\\\\\/QAYKddUzMbMtZwfebO3hwy2b9jAX3Zo4VdOh0U\\\\\\\/yqdUfPYfcmWpuGn7vVGao\\\\\\\/ZEBt6nHEOwmOxofXy90x3HE3vgjnzVt6sKmy9b20PXDkpO8YfU2qT1n2bl8fmCtzkhC2XpbwOK74QtOsifyQ4r+2\\\\\\\/YQ8LGZjiCimP0v7q7UFhxHCjhekk+qz4riMQdLOsNskGnYqjFg0k1gddoRmbaOAUxBktvOMqa1pv2HDkXcQDyeKSjzJwXF8dD9f7CCbzwh63jMdsEgMKl8RIM9AEavSFWFljFJwu\\\\\\\/PLeVIoOJuO0HgHKq2pQypjZm4nB+x5uNPMI0uc\\\\\\\/w67fE\\\\\\\/86lVsmWagmLl\\\\\\\/6cdoUtcn51wbHy6MoSceg9r+3EfPC155YPK0\\\\\\\/TuAu\\\\\\\/XejtkPaqXeilymC91MAG9Uv4NjIapLhorCtvozmBRvBgfMTLTxAT45nVxlM0Jy6o42OSEToKivqVL\\\\\\\/Pj+Fkz0a3yrdnir+3pV7uFm8gJqy2HPEmaeldU\\\\\\\/agT\\\\\\\/Tr8FA3ew97hUDa6HjVberucMNrFBEo1KS3R00KsWkWVisgCnvIigV4\\\\\\\/S3AB5e6S+M3qeUQ8uf30s\\\\\\\/0vcO\\\\\\\/NLETf\\\\\\\/4VFLUM8e+LiwDha7WPF0HFHJhFq+pJ14iP2vVycvichds\\\\\\\/6rZTRuIVr4ATPap9s+UvaIjYiz2byhHm9CeU3lx8aa8+WphjX\\\\\\\/7X1Re\\\\\\\/\\\\\\\/G\\\\\\\/u26KDBxclbq3\\\\\\\/zMcYkxW7ZT+S9N22lcfWjf4SdBpj26M\\\\\\\/p50Traa3rsLvGe6vjAcRTA49v7S2E3fCGFuZAEtldM0hVUw4k\\\\\\\/dQJX35BFqzi18uifZYxOkZquJLAyY2ucH8oIELhoffZp2g+r1F6Au1Umn7esehkq+6tXEIg5qIpOwop7\\\\\\\/bHLJfwwRBiBZJRW4yalHaJqO1HDItdB6UtZd4wBqtTeGH31E5KZ4st7nNzOy6eH+hafmj94nPwcb13aLiFRxg+1MVTvqJimkgcrmdseOntkBCtdhxnL48k+IYiEsLiCuIzSKOhx3Z+0u1MesBDjWZtxAp6cjZTD1bP0uMfhF2+ex7zcb2jENx45QLejm3GFyyTHdWwQSgeNxPNeUoWkAvJxjJp52l\\\\\\\/L8jgiaRBiyJaBj4RbxWL0umvpni0NvGruM7ltQV8AVGm3IhErH39Ptik61JOhGC\\\\\\\/w6HA\\\\\\\/QPSLYQkxchkwhZr2n+CEbvY\\\\\\\/Ivi5U4x7xZFYj1IGs9EtEqx9ojReggG9HxEEAs7dI+4+Byxa+LO9v8KDpXXuYJM6EhklbCNBpBUG\\\\\\\/rSD8wIpv1nwJpTT8HnYCEDJoBemZue5wHeWH4PNX2nAvnTdVlfOXruZN30Bc9rEjtDVrbOf2KdXqYRs6ykDe69M0Daz9CEIurNfZVB2X8tf7\\\\\\\/Gz2AEENUhYU3B+eUHVWELJXTmCb1YG1N9NnNuK09\\\\\\\/7ID7SQ8i7MS0\\\\\\\/KRPYHbryqFdKv+fJJW3VGUMyYhMwv3GWWPMwdX+Y7O8m4AcfA\\\\\\\/5NoWqBXlj9UnkKAZRi\\\\\\\/QB0iP0h3ZaUkaw2Y\\\\\\\/5\\\\\\\/j17h1sikx7vpa7zSmsFSCXHSzuPb8Apnw0gfipawNfl7sqSvUGvLhLvD89iWK9S1ggbhvwtev+hsml14KTe\\\\\\\/AdUjVQOWBskp4k\\\\\\\/PxayXXPV\\\\\\\/PZMNdekW1UMbXw2BiXtfUCvKkly7pBCfHKsT5ms2YaNyP0dyx97hKvkCNrsEeYlR6RA64jj8DvODKV0Zn130JBRNkUBpzzq1ffTAUusxUDW0BKtrUaZ\\\\\\\/ubDmSe94kBRMrFyG4q0kWV5Z1shqsq1P6RZ3ICRLLvaqXaWGRSUwEnJymMKQasg7G9UJEdrK3EKhY+Y2uZOUa1xpSq1HgMQrrqEEflJ4zavRct9LNmh6weWjuqgVeSn+5dESmsyfDH460KAieI8kdr+p0L3F\\\\\\\/CNPMV4aS7B3t1oF8p3X3R9DhVBWHykX+kPv6Sky5xG7TaN\\\\\\\/mRU2wuPh+4IV+AKacLgg06dD1z6tK\\\\\\\/68UEN9lgKpUJkBOqp0LZfJM4edfw18cHDWDZ0FdIRhsjQT3irzbEBC3e9KBq0wrGbx6dDK4OMOD1eiOJSuYshVGNyMLoG5YHva58DWi0SxoWaWz1WI75bA3BPR8wRg7ddI7\\\\\\\/cbNzjHW\\\\\\\/BnXYMuq4nmb\\\\\\\/BduZXFxTozJU\\\\\\\/vW0put3KaEeQcJj4qEQBIa1jOXfj5sxVEC\\\\\\\/SPEx7uBv0sLULZT8w3HuhGpGV0rS2cW0kwnJ45sfJ3BgtX4s3WhATdweVm6zU5yI3SSTAhm4oRfFQY9WQjLVkbWdnbjpNR7I+wxt7Tbrir0IDkzm8cMvlHNVc7WerE99sINIs+9F5vW+t9qXe39BGR3C2QN2r9hf53ASG14cXXOy8xHlHJ47Cz\\\\\\\/KsYjedcGH5qGMYNJ6b93CAnvLOSejA7a54Sb\\\\\\\/m9DVXAge3sqQq6J6Ye4aDNUDzuMLwKebU+nXxbNSOTdWQ2EvelFXFjzZNt7QlFxlC\\\\\\\/eHWL+k5lkk5aqBJon3GWqN3RbOCPwpBjUaIWOj+d1IrIOoDPKGPYgcR7MINjQj4upLKXwCWqEM7rz7XWIg56rU6dblo+FtN99aitTq0qtGh2G6DYHIAIPDDkr35PL9V9FmfNBahx3VF5Sod0GztJF2MPnpqKl5t0Z0RR4Vew+Tv9h+IXK+ww6Dsy+6erksdSORsr4oXykuOVc25ddVUlkwSDPcM0mlWybGSsBrd3gNxS56jAN8uG8Uvrd4lwb4vwxxzAVrTviVqlBUhdEtVJcxL+oqG\\\\\\\/C1Aaac+o22yzs43c1+D+ve80CX8Z\\\\\\\/0njzHik0bZMGcxWU6qEPOTfleIJngDozODvUAV7QTLjUE0wahKcm31nS5uNm7mJpGpQMYv0PAj\\\\\\\/3tpXy8CxWYSxIaIRgbjSOXOJU76VX0w9RRQir1mVgJ6rKPh\\\\\\\/1nBsdoxONdP93dR5Unq9G7GbXh9IZj+O9GXrZC34YPaias0gCNCZntMyqqJF4rsuRoEYcXjCzj0Vsoxuhf5+OGB4LfIbdrwaxVtwBXDVG1e\\\\\\\/yknbV\\\\\\\/3HV0ynX5NaZlK7pAmnJZfIpczWVuVXQPo6Z4yw2O2KFFjx3btpkSnJ9VN9MWcMlnPR+WkrYRPmdo8sdG35gZWSRZKN2SFZPm2z9Dx+BZmJK+f+uOMhRY80iK6AjEjwxY76XjZW\\\\\\\/Fuh92MJyflr0kBjijOy0LJnKDjYjrkOxKVqAMC7TaIvG34edN1UYSm1rhELrZiGiCcG0i90rAiCmgKqF\\\\\\\/TJXSQGPEYDyVWWeGIHdPHxNNg9T41PfwVr6mjma1iqAnwR9cc8UvxvktQd0sDXOv90Aj5yTPlkWBCvW2MNYSrxncVqVRIQbLjZEWsN8PAYG3DFQ2GMKJTdvR84BwjvOWGHubcRvWy8nR9oweHp8BHcwehSjklKkNOMIYTF0c\\\\\\\/yiUgvwMq4ZnD4VQtOTNrojB6fxMjQfBFDlsZsnygXHmsd5KXjzyDlBWDNrcjNHLFxYLcNqvtoVT94Qr7ZIsOR4Rghi59ZfMAYOUHpSBI\\\\\\\/e7dwg9TncxVcHznFo2Ms1pjzKbFXqrA72hY\\\\\\\/FC2Pxyh4Aq1Gy8nweOQ+bxKEef8wAMglIGArLvODPrew8BPxte2xADiirpx26Sd5u4IJO8i5NqDZyEtZnWo3DAZxhBLH7H11yWvz1im0w+l5ok8Aa1g23bbcS4UGik8PtfOmsm3iTwGePejdcasYg0ncNPJamR\\\\\\\/fNyoPKcdtl+BEvz4JDBBvk2j+b481FnUb9LbzP7LR5ZeMgLzDa5KmZEHPTB45ht3Y4x7gdSwtBNdeY9JYKho739VPG\\\\\\\/2DhXmhtCZQiU5SLkbkuCzpgmGWTqFdCxey1oy+UtaaVNK4Lv68GlScZKnVw+3h2undIMvcdP05WGaQd6871pMa1xvOSw0QQ\\\\\\\/kkirD621xk0yEU7WWzDUQWKD4Or4hrP+4pTIai8DX52P\\\\\\\/CepzTaNXMHs4uQcFY2xsTuN8dbcsUIjkq\\\\\\\/2TwAVGss8l4HeEWOG5zIYx3ukTnVTzG5Xt13K1LKhMbtKGfH\\\\\\\/GPf5sMdsqJrQ\\\\\\\/UpHlitXQ0u77C0kz9bRBBzmK5lqOg4TWsbgfrBEMZ82zIJJEaTF\\\\\\\/AL7d0rD19yp6KoygQReVIwnhVpQZYi2J19kE6uTb3Li0wTHNMMVeXkdZVF4287Cyxhs0p\\\\\\\/4ckktdPyLze1Kcva71y6ka\\\\\\\/1Cls0x3FzVn2a6JrJRaPPT+shvjm2D3CRX8HhCW2L6lOvqTgterjxXAzcB85LfQheqbGOnqrF0vi5QSmRpjD8quETklsn9bZLEDJzxkaGkzAqBoTtLJuw0eT0dLoDCDr3Vvu3DjUA\\\\\\\/BAbz4+rpL7VHZdTEBxDyF9\\\\\\\/3iLoo8kbXhHB2tyy01\\\\\\\/3sjstR5z9urYMU5whAKzB4oKOo05Rjo2zWXAx76m1IYzmGhElWt7KqF\\\\\\\/JAZgvYRl+N45rnIGA05sO323RqfiyWVZfILu\\\\\\\/ty+zXM2x\\\\\\\/Z5f4wAXjn+blcfD8QIrx2JaEGO25ZOZfa4eudKRkQLkPKEzOROt3pZ+Av8bY6WfkKi9VGhclKRH20etXuGBImPsvKThf6TAtBOKkTZIy3zpCYyHjcv9aB7tVVNKgnGHtsC7tjCPfPmUMhUsPA6QCHo+pfeSUNTeT31\\\\\\\/4ikUbqOrniGeZB92kX9miMVn0bb+Nxd09b78NTiIp7BKkTRULut5qIju9iC4qZ\\\\\\\/aFvSmXGJNn+DmiRqDVOlZ+xyQFWsCqZGDVAqQddCQNwO+bmbYgA2yUTBkV90dQ+VLeAcAgRK8pmqcR8NEktndQkQMaDsOjJBnWjpY\\\\\\\/LDdqV5nZBUxSGRMZrnmW4KM94PEsxBpARF7ztx6Pl7lecgT60Qcmgus5ZUL6PbxR84Z9krJ1UfSWu+z44qBeHr1KAU1XNI5G\\\\\\\/3eF9UK\\\\\\\/mJ7V1O\\\\\\\/sClz\\\\\\\/4VlPc8QUQdIZO3DhfNa2PKrM+9KB0ScEm44fxNC5HnJTHdTTCA8J9vavKA5qZjP1tat0cORu3S6dlGQpLRIaQ\\\\\\\/FU\\\\\\\/B5rt9BOqk3X4KwD5qGAiyAg+K5oudU+udf4WQDoOQuEcDm1SOyOJH1WGe8gLc\\\\\\\/V02JLIeY4RcAYg4BIKUxiVggAR5NI\\\\\\\/+Wt+Ps9aeN7euK81BCLEIPLAq5yovG4vYOe8sXiaHWROz79rvHRwYHZgy8Gwp5hMPBEUmn5qVYapoK2Q21jozEmeq\\\\\\\/xmxxNZcsUfn7q5AzzHJcO+FNVfCuWuMmeZkQxLLBEvSR9de8Kf8KEHxorV2DHB3sx6f\\\\\\\/8O0pW\\\\\\\/Kboh1GsdA6v9IGh5Y\\\\\\\/+E9vveswJ61X1XohagJxpXrrYHG9fo37PSAL\\\\\\\/wK8CNpo3Z6lnp23pti\\\\\\\/5OIQRG6wJ4wiRNggxu6RR2rOE0VDzNdRdxCf2MzqTlot2eRtozam+TS+n4HjW2ljGk54Fl9ckFd8ZeUtCTN5af7c4qLJCHDyaSQBiBrGfyAVCINGpdfw5wCRSynM+68iHp290ZV4I29Lc1tiPOIO9YWQr7PcTU+3pmS5ssebJRX5K88ZDloso\\\\\\\/lly8Wo1zQ\\\\\\\/h\\\\\\\/beZQ32sLyVl0iipTe8kuPMSKsJKlID0xlv9zgvei1AsJzK9WTwA82iX7ODV1k2icw1EL9RxPfkEAyVVvk22nIx5xc7fOjuZd50CNzJ7XQKUMT9CXziHzPCiHgQWNzOGcCiy1xlXAA4xhdQ7OFKA5rZeBQwpaBnNVRdBNN4zE+VuUrmKtpStjOwQHvbNu4R2mmuEYyzr2l5lz+iJVInVe1kaWZ68ZMW9whs\\\\\\\/lcoeR7M0qTpaPtjWz\\\\\\\/YVE58BacLXh3wCC0Pgz7NEIRt4Z2CzDaXkiA6rmrrKyQPd17D7igVouoQpxErfzSzjgPEW3WQ7vdVyi6KxaSTHEvLB+MrDquLv2K8KonpB6bfRAb30bAGNyPz4LuDnSOaOV5jyZBNa8hFnD2WtkJ10c+jqKtZrH2c8ypkJXL2j4++724yFRgs53kErK+rzUGdo+SpCz6sGGBlMOhcmjomoW1nKEg14Eczaku30aE8JmNloAT7+ZZgzrs5xmN4XBPvUha2FEpxMUpUrYh7zrdUp4RHxUohvHx+Y2itwPOhxNrx4CvW00+d1NkaGWUKzyn11QYJq32JZ63qJTPl1StJMmT63P0i2aJIvcCGfQL8io6LdGsiyp205tl4NCc3fL\\\\\\\/SoXqJrd0wTS0ItiCP2lQJ8xKEP18rudQtT7jtJ9xJ68\\\\\\\/GbXRgHOM1Dg5+jfKSEH0icabJ27PrqDDjjL7wDnMW+\\\\\\\/JlFJ3v12XnNlps3ZenAlLS\\\\\\\/jywOXSi7oSelbA6cboxLSMLUUY3QI5HGgNXrI+0zRVFcnY647njvXdQTSq0dpiFZDnWynQRomTtXmOGhayVhMS2glD3hOz6mvbXUFhCBCuHNE92mJC3bX5GtN4QtNGUIn9vmg086Ew3bjMqoRGEg8ONfp5gz5BEwAQ6cx5PvRpolp3qZho0St6RiSKjn\\\\\\\/o0LOOkVEZxNmvdpClOiKt9RJjOWkMoBXll4Kr7oOSsUdzKkddxA\\\\\\\/5K7ImAklKxfe3WmekceVxP3banEYkURsU0MRI9hsxo+hTIC0E9FHdclbBuum79yu4n+t73xTxmxDrUBN6TRq3UwNIacd5YgHhdVTpiPhuih38BRT38qDWmWUXu9D7N1s9Dj2xPWCf7gY97gLEoEDkg11pF6WEszfZa+alammMn4t7rMymHGuH1+Fw\\\\\\\/sl3EZbEblx6lAP4LKfcDxFKEEPl6Dv1n1cAco9ip9bQb8e0OrWf1djB4yQiDQdChS6StsvRugRqyaLpN9TfXKTUVACeYPP04YZaKMDvwZuEIENluIt9SmEHlHw23S7dUJBi8R3wARRChndfn03eCUUIbWgfqo3fWJS0iN\\\\\\\/EemFR+v8fqE+6hp7WsfWVdk7NSdbPPxzkK3Ygh2FtDnsPhxVK+ZZt1C\\\\\\\/8dw\\\\\\\/Ppf8E2UtQlv979WROvvF+W0DV3Jgp5skDJIvvFC6UWl5XB83+21weoqcgWOrQ87wooYCCm+JDEgS7cPyUYyIpg6Nk7ueUQxQ6b16GUs8F5PVYyH21aky4epZ3NmEVnfw6E6rGQoMs3s6VRkkVec4Xn4IEVqGcghFcfZkCykGI9nrPPniKqxyfGEwx6l2zrlj2PnEDUEwhvohW1wI\\\\\\\/29CastdNQBYXLLd3Utofj3Ir5s+GHqi6GVMh\\\\\\\/c8cfnwb4CAJPxyycgCVmxFocB7hC8gedJbEPBWOV6d64HJCmNUq5hoozWmpaRRsFn\\\\\\\/bYwQ9RlMJe8iMEl9NSS5g1En2WcSVwK2Oj7E73DIYVRVrRdPY7xSeSTPLlHl2UKwrKSWYRkvhMpRQGzk4jcOc36bkkKgs9Qmr67wRq8qSI\\\\\\\/7QqUB\\\\\\\/B5ccfdfJ5J3C2cHe1IV4mmWYYnDeBGTsUMjdndhZjQwS7HkDX1LT9uuE7fjoMRd0RhnBsmso954VoSkGkvs6g6hYXnA0by+8zCZyivNi1v5xa6ldaYVzqOskFl0D7ENa6HjuJBtVF7jRxDf\\\\\\\/rhdQMaNeQQp+mLTAsATLnIIXubewD7C3t9dg4mBUSp7L5IoWkNOIL5oDByJbjujUT\\\\\\\/MGDo0MTHN+UeyGYOw\\\\\\\/Ky0fQFPts8DhwVgf9yg78NPRmlhPsd2emThq74ubgal1zrlT0XDkK87arq9D7D8q4+XxsfwIkX9iYtuxDVNuiZNFbxn\\\\\\\/Nwzd4tLj9gnxr982v08RDB5Sha+rbEYimPhfa93WrAHVLD1zrFj\\\\\\\/ueNqf+lq3GWYrDAakdFQ9vlu22srCtmG5PNlBO1sJNGu4NJqqvaosk\\\\\\\/khJxJpP33IjA+Sv2IP+AQ1yqiLbg9KaE1i+b0Q1Qp3dOGeZs7kguG9e3kruwMtgkYmPiDjDGrlV4+1ltv7DmiUOhYZNjrIJhx1o3CsidkjBrddIaMV2VxkjuYkL5OrKbAbxYn8RTXuPyp3WfRF6Hj6wfCeysVoJY2R26\\\\\\\/ajEeO5e8pUwVAYcVgAEvBKTdDrsTw2bsUlDF2lJ+ZZhXDC4IIYHB4ODgZPocR5O3IKY7enk6+LKwLEfRZBDIIo4Q+bf9inYktBd3awZEmxOkFg2Lcd69lIGARfxOqtUZ1v6sZTkcU6DrvBLnML70p1UWhOh5K5QMLKSVqYSKSXYk\\\\\\\/MimpNDkhNb\\\\\\\/Ftb7MPIDvie8zZB4RaBM+zy19opXq\\\\\\\/8NwIy7oa4BGfsxTZmQ4JvkLYct5bWVkYS4jwT\\\\\\\/vPHEf9MfDpY4HX4eN0DP3k8XZEFmR80eBAUkGBIF99MCV6DdyXz0cnAnGsNWW1Gw+UCi4O1PWqMtzWgOYD5ARwIIQpF6EVLzIV0ngX4bM+zwU5CPuLg+p681qss1cVngdvVDI3x+ZHwSPhocxbd07anRO49DbT9zKGWTPW5w0wUq\\\\\\\/iM27BY\\\\\\\/NhgWbtzCNLVZnnBop3VWwBcGL7HUcLDtckcbRzTp+4GCQBq\\\\\\\/DNvh2ycNYnEv3UGWPDLvhuDud5zKNm5yhSk97mWmopZn4BLlPCPYwm64VC7UfC8XfRmvLtl0TKTIKpQDm\\\\\\\/zehpE2JrfDk5ngJAefSW8i9C1ZFCrEGq4RTMd3e\\\\\\\/StKIEvgE63GaJjKch7jWDIYZtMbbYUY0GWfJi3DEyWv065k3Sh1Znp4Jm5UcOUFxbSFbE+Tx4rznqEz6fNuFL8XWyjvbWyVNrWRvD4NDOlV1qnsmlSixiP\\\\\\\/76\\\\\\\/cpLziyiarkd8T9FQVpcohJko+9pbkXjV9fuI4bKp\\\\\\\/gFGtUgXW8M8HfgEbShCAkk4B661DsKBmfvBXOhiJ7alF+fvVoQQ6p2fdp7UuslgpuCMy5ZQVUwRT\\\\\\\/X3UiAcHuj4Kxm\\\\\\\/ovuJRyWwgqt0PlVbM05zwLMobeFu28zkI8SrRU3qwXek7VmBBZKu4EdvMiq6pHKNZ4DaBhmYxf2mW6W\\\\\\\/TgeIWvmhf7IcJcEBiQ2Zb6YNHxWoeHIVsMe14PLtgn4AZQKZbVyXJi2b+7VCuaqM8S\\\\\\\/o9n5c9pvmDQFVEBj6OCGYHEFPMaMyrJHa3+jQEvbfJA78MxGV4641cVZV9DntQpUY8DDFDVGoSpOHEqtV3EqnHBOm6YkuyYBz8kHIYc3DAnoey+HhFFjyEIdgdLJC1aVK0w98UaQ1\\\\\\\/aWkeZkKgdEbMboxXkv1OYU4KdyHfPBncngqlfaQ+sZ2YczcBLsMQIfrWy5p2zY9A5IvNPZWzQjInn91aAFndrRSbI+rDPspaDrXanB\\\\\\\/twOyFhvkaAeiAvgX4QThtxCdZ\\\\\\\/\\\\\\\/o26TKNN35odV28efl\\\\\\\/SKEQe2tLJ4A57VnW0Hw4gPoHj36nef\\\\\\\/PK1GOZBqtrsQkSvTpFa3kClgiCcatT4Qtl0lF0i4fGT1jk4FrwkpQ6FZZiWR0kU6CY1j9+KAvImxhIf1JRe1YCyD03xMg3KuqXU1lp1UDWwqD8liYMdCMWYRLudkg4EBV8nS1aK\\\\\\\/rdxMBRlkLZPU6jI6ybHcc\\\\\\\/hWslFrSuaASkj7G1SUOCAesboPYJ6fsYU5+MsjzUUjVRQSskxHo1zyef1LiF8i4M8f+2h0r8uVF3bHehPBYk7IerF7CcvJpXq13oxnB4ah6j+zxO7dYr7PL9AA\\\\\\\/9P\\\\\\\/Xk374DTKrIcCqSXHmfgs+j1yiEc9vRuQM6MTNmZAunZR5TustE1jFrySiIXKik01Ka2bpNFH9sqUAdzUHAtP\\\\\\\/jK0+Ucf62KYzky8KBtEspqaAimBcZZXoROnmPPbDM2lI+bIrbBhctrJUOaU15lBUNzJ5rICiiPlPYVBfVK8qzuKqkIq0f5pJRoWTfVATJIhpspJJg2JcPx3tRFxyGIUlqFHQrB4tbTDiLFP8PZoLGTUOUSzxVGGNk9hejthpPOrOro+fHDbydyRN+C\\\\\\\/A7KZbQatrtsCyj1j\\\\\\\/NVO5WMsj\\\\\\\/KOtme9gSpg+ZfTxNNfIgzRnDyAivD0CoUiBVg10qE2UHDEElsov93FufuFNXbo\\\\\\\/92f2ZAWbFazOE0rODZa06cxdHGkOY\\\\\\\/uJ1ly0VFvzggJPBlB5YNnhAh08PXj2IbOA9sRdAx6wiMCPEYKJVCj4sj05vla3Kf5ovj9\\\\\\\/\\\\\\\/Nve0LNYqkAlpjsneFlfeoqIiZVauZ6vsxaEoncRL6jhKUqGw8mY+TBHB\\\\\\\/BReJDhSoAByhvqnMen1fE2cqLfwEcDgKO3YrLMEvri3\\\\\\\/yv45IIZTs44X7rYeSXX\\\\\\\/aHJy2I4Bz9HoFyzuAa4tVAgkPBtCpirWToTC1mYJMyK+Tp6mUNo6eCXwHzxiot4OKH839a0FDE4RGY2F+32BvCsblWVrOnO46nt2rHvBM+KUKAiG++2dn1kPCM2t+b0m6UX1Y5DvBfpxs+9ar55MkgmImSuM9ROAV4kj1dMp5KfgBoS2gapGax83\\\\\\\/zDXwByHUEJ+YNkIbSm73WyZbyLYnhijwKHbGGQclYty5nPifSIdKBWYzb8TlVdAjWs+wOLrO0qJyEOIK2KwUKzOraBg1CVmnl3USyo1KmXKObwCgy\\\\\\\/\\\\\\\/8FDz5eIq8PAEnmojOhhIBm9new8HsQ2UOAkIQpO6RqqX3zddAauzkXj\\\\\\\/PS5WTSJXHEq2U0plDBACQG00DzS4MfvMPSYeHrIucsKR4pSZQDHt\\\\\\\/eFHFOhk7l1EgB4+bww4norxd0Kw+3pM3zqYkEs95j5Btoh+g9ULV73YmJa0TrvTwdSWFqIw2jnFARO5G4MeSSu4AC3NqWADGvhNRv\\\\\\\/q4sUtwtj9Be1d9U1i1xi2oB9qsQilnlBDI0JvAV3oXocl7TDdFw+oSixEIfW0KHD3S3X+hiP7rIbLVJM+hCw95PqxYavVgGCcOl9e6axQlryjNOFHD6mrmf1WGqDeb3ueJNNupNpOsPS9fHGL\\\\\\\/GllENU+ThtNr9pDDifa1VkGx0Q7sKFEgjM1SL6+2XpLLD8jf0CFJ7gnpvBTwLn0ZBFgnbDFSLi69Shr8w6OtYhoCEaD7qrR67OGPkMNXIwTsuLtnS6XLcIAwiHIUsClEOWIjOiMNmB1LsEXfIuz58rkbL7UTyU14ENjHRtfl\\\\\\\/hy+J1TtGQX33SO9+RpFCvvjk5hceT1RxrXQbL5xGIU7oprpkd4u6dDs9YgqroN50dWQCUrlSxtT2XOrlw0tIg9zUwstzyYdOPbJTBtdQYXgrSwVAoa5p5J8ENoDfAES2fiELAzslUnjssQDipRTCJZ\\\\\\\/1XN2U6ceyy1otHJ1A\\\\\\\/S5207FBNWe9g4urr1NEuRXbCXxQUhMMKAIxhXS1n3fJ\\\\\\\/okXQSVqC3lhKXz0auXyedpNYzJygtxhgoEqcv0QpyNCWUMY4JPZeR0r7tqEAy+\\\\\\\/1ffODiLgyolpPkeNMQSoQya+PzV39S5mSkhxge+kaaeUCFxhYW3f8hePaLnK4n9K\\\\\\\/r+0eQuzfET85LAnYO7Vzhx4vr5qhiXd9vXGATsziGqOcJU87hAU0zQk4ukztlPEm16hU8hgqvRdLL1mgO\\\\\\\/ZKbbArn8mxzUZ87TL0O8a+FCBYFQyDTDfdvEy+U3CEUo0KVbP3NNn0jLRRWYx2R0ACaV7oNeRenWF7Dp5ppAYmcMw+rvLm1f0vB2DIchMhstTC6yvsY9h7VetzYm3tCXkrko3a0UUSyaHp3s1r6bkGbIgOdKrGwLAyJsTqTbQ3BYu6+4u6L1saktEt1MEHzlW1QEpsaQ+fWBtP5yqJLpT0g6Co1Byel4ZhTDkvu92k+GRpAg3FkW\\\\\\\/6aF7nNCmZ21\\\\\\\/Q4Nb76NKTYMDRn4xuiqv1GKl2t6z7QisFewVkn3qQE6aCNpazGcKYjqkixsGF2l29kKJD7OGt4aTxDWAbfQzDyRpkCGBjs\\\\\\\/GDR0tJOpN9ETTUY5GW3m3yicq773gSIz3cnTZdt7qrHFcI\\\\\\\/8vftEc9BrvjOi0VidwgKClQsZ6uWu8yuPmqMV0BA2QAGO3X5YBtekxK+TAJKCH+STfxYSX5WM4vxBLyPNn3qa\\\\\\\/i4bYI9IURKznOui2SSUj1+v1eSW4rNc0bmMdgWCiKbud\\\\\\\/R4ts2iiv1rFqZsVMc6ujz464GkUaejekNgZ5qHy6\\\\\\\/OnYTpaobNKrKeBcFHzd+c1GIwBbAJQXzO1+kvDs+9nb90EA3So1ytjCu\\\\\\\/OOAcEG8Dzws7pKdLuu0xW32DBiSb6urVYJB626loGmuCm6UPmgYcaGYtHZ6iiWjD+wUxBMUNEl8rv6ydo95W19CGYTdQF\\\\\\\/uO5cra88QNs9eoXgSoSicA+tkb9HucOf+vkzeL+12pWBrOnIR+E3gOhNMMhHYYxStGuYMzxWuNeXQ58VjBiq3V1rgwYCtqJTjNN+hTE3lhaXaDtQ8Px9y8r2Yz4o9UyNAvgqTAlaOSPor8gKH2s2Dk9fdb+dxPPk9AywfoKrYtq7ylppBCK10t4Zat5kXcV2Wz7LP6ukiLqj40lZU5bDEklE3luJgdZP07Iohb4Aql6ZP2ggTVXhsscpZrdLPXkl3m1WBUqFy8mdA5IlX\\\\\\\/+buzsIx3no+MDqTNX7EtD3h0y7PXzmocQBXUbfsc8TfvFlTitv06pfLa8ZxstBNF0mWa8TwZ3afbmubDcRcMPyGMvl7ax124m38PFzBbLdHNJrt2bng6pRERgIwH9Vqe9\\\\\\\/M6wuofqHkO2ZDvltKfLma9YZX5ptFVG8CdHt0QBdTWTlxkInBmf4QGdprR6Rk7E8mELn9akBVwJ2jC96CgKYZr6NcS+U4kOQWzXP16WJLoufEAO0wb1fY9G1VJEdiDPRJrsUa33k\\\\\\\/sxOugckPYUo5Z2NDmHD8Ej6GGuvHCYUbki4e9NPGyfeTbIadB1k0MIpn245PChvmeR5uYLtIGrrAkarwEdFjzglO5O1QCAal9VF1P7Z6mOL1LNuwwnZQAR3OxV2q7JTsB1YhReYyMw\\\\\\\/5IFNWSjEolwqUpUNW4uTrVYkckl5jnUEdtMttpr2ikvaMp9yVIovSqJZegiZyOHjUZjzSqA7cS21IJz1t8ZdKD\\\\\\\/M0NsoyT2dU1xo1FGjcA8vSzTMV8UakEqxtRRnwJowhhdFiXc3i6QCTLah\\\\\\\/iT2FNs5BW3ofVPrNVU+lwV1nNCNCVSPUlhHVfk1uwn0wRUdhhHdeJqIqU0unXlUG2CzYyCOKBUczNGRiwQHMELyY06IHO1NO0AZppT6ab5rBotl3EM6UgD6evlj\\\\\\\/BGOwt3dEHZlqZHdrIJrwkeoxEvbOONWFY6hCwsRC+gs+hSj3yhj1yjxPwaHj3ICCszsKH7XZM988\\\\\\\/VlpPPsQcRhRoTcY0DXtnhBxWC5LTpXHtIxRFU4H3DVdNn4AVZaDVeqObVnRoV3REZRN9HaIRdxIsqR3m49LWcPzllCmXUc5532TveX5RXIabPcBg3inHgYVifq73cMSqfaGRDtQv99aNeymFvpRX96O5VxE6VrqZkmqUdH5dMovulHwtJkQSzHm2TxY6le9H88g154GSZ0l6JTAs1\\\\\\\/O9hNEl0SXOXzsZuh\\\\\\\/YhHCGwPpGTSUndbQr0Q62tleyS9JxgFCih9kU5QFrK0JGWQJTgL04lbnM5lrWqYIxa\\\\\\\/JZtHK78XS6SBA\\\\\\\/W43p6+ZuyGwLHySMLMcltAO\\\\\\\/Ie7oPGiyDYln8dx5Ulp19cfJXfW6n18HCeeyri668V6S5InqQ96lId69UM29mrXX5y+g9kN4ok+itq5f38C864t5VDW7HnyK+R21yL8CGobRPxDDLRP4qT8WpiNgqNFqphV7u290WiT93lwUWdpt1\\\\\\\/s0jeU0Ue5k3TNEbg6yXbvB7QY7Xkd8PeyjBFC9DJLoweuGrZfdHt1+6Mhnzh9KSBkrN0Y+L+nBefDgHk9rtJwJmG\\\\\\\/ZZtNVhIrxrgDKx5dQnjDYKfxTrrsOWkLobjj1V+Jvp9qRq\\\\\\\/hRuwyV8mEpEbEzlJqBjWV0r0q20fJQ9Ckro6Tq+CzYfIEkuoX33JHnMuYSIDgEw4bFogqK7q9WTuNZ2uky0xRfgOvDTtJ9w+StLaW5Fu3mlzNV\\\\\\\/cTl7CmUunMpSlJyknehXqJVYkeua9k3JSr4CY48lG8D9Uycj98LZkpaSwPJ7uFHrQ+8TejhoQV0klt4mZKQx8Y11V47jQxHSnG16CKSYwqxqqV\\\\\\\/qp7GrKDWDQYNH7DTHMRd3CrGJBj\\\\\\\/ZJneoBAraF9n8Aa\\\\\\\/qMlG0WgzYbGimgEfhphVbPchO9jjN1dMee3IM\\\\\\\/bCP1dJwFAZhVeC\\\\\\\/d9LKPMDrlccy5kjRk386kB6LRZPEHIm3MU\\\\\\\/hIKgDomEGNb9XjjaU0Wd89QFJ1Pz\\\\\\\/0sEQyN6tTOqT9tPvQmx8vg3SgXVgwiPLnB2MRXXNmpwPUqC0Yp9vKmCTzrwJVLvXG28XvuZZ0PooSwNLOvT1bOb1U7Z\\\\\\\/MZAHFPHJ\\\\\\\/odycq45zSPoqH9jv5dTcR2T7Bt02jqShCyR3Ng+IOtM++D6XDJqai+NTwxb\\\\\\\/\\\\\\\/3+3\\\\\\\/GIgJNmWcTnf+mFjG6P8TyFzG90RRkoftecuhi9NoLqU6QdUkdl1bWpVuxRpNmWxTcY2GNhzUiPWeuAMxSFrcDe8BkWY2WW3zuKQxqMEpAaYTgwaT9GRfrhdyhOok2iekuALZ5tocM\\\\\\\/f+z7c7dKQj47lHJp\\\\\\\/RYLu3jdy3Z+TZvA2mSPfEQsrmQShD2DhhEe3bKNbGHb1+uSrAgKIIgoVI8qSVEnbWlSi13duaRUFUyNimwft2qNPVlW4Yv0EWLQS0JJljdnpqv+lGm05yx94IajG6E9lTnSx1TifbbEFe91WrKau3E3GICawgkrCXDbiVIAgkOJDCrEcYmionBQyUNQqw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qi7Xf63CZW0K\\\\\\\/u6PlAteEQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f9019a4afba7b22c2acc518c97f461b04bfcafb1dbe750e6ec09601ee8be433\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ak1MCUFDz4D0kcTt3l4ndhSzJMqXi4EMv1gRr0quTiJT9ZRBbliQZjhe4EbpHBCqzITCIhWNYIKpYxJwQcBecEajITteallWPh3et9g0+ZEzgxRkzKT1rXa+v9aQ9vrpuD\\\\\\\/SLmB1Dp\\\\\\\/\\\\\\\/3Mk5nbY55zqfdnadeEilnfpb3SrmYtI\\\\\\\/UCoQMohEL7dS4NAcBPXYV1n4U+\\\\\\\/rK6ACb81gujmXmHfUkk1UGRPKydpjdTriOagTL5gPiq9MOAeEI3ij3nBf9yePcbWwwD23P7\\\\\\\/Zz8x8AqOKtLlEN7thKT+WxNkdPZd\\\\\\\/vrJ2ObxK35zK4iL5v\\\\\\\/g8JVF5P0uUee7WXAFrj+WSPvlAJneNiq0x8p8Rtlj\\\\\\\/lHSmfrC5O5q1PDUjDfk4+f98FLOvdcTag\\\\\\\/Itkviu03WkNOT7tCMTLn9xi1ZffADp8wncuXJ3sQlFxRyFrDa\\\\\\\/n8EMhemvlewBuwFMoQ7d16DpjoPAaTAlpYKbZfQqNVIfR2qq4EPbc9XkY37haL9nhFsAsQE1hLF\\\\\\\/8ng4DZ054XPsAX6ymuDJG\\\\\\\/7zb0sfM1N+yi9kLP5VWx5l3dBz1SqXmAUbKIK8rZsFtmGGmXQT0r77KvI8V8RgyGZ3V+jh3NXqdyOmGW1CPyfHBSZ3jnaZWKPKTD3lQLQLEr5yymYLaxIBE920\\\\\\\/0ssxmHBAz69KU3Y+Vtd34VBpSARu3cfsE8b9fIgJ189EYE1LRhYuI4iiuVEYxmv7c9Oxo8TLnHRW7GnhAxSKQad2F76YWNtGnp0eLK697ItgoxJG9v0V25zprlkQbYn7\\\\\\\/ZaYqFWmd7AZ5Z+Se7rquILGeJupLtsFTvkMafo4qrRDJuFjBMrl\\\\\\\/gx8hTAoJnr0jDGBYCHOK4ZB3xRJ7cWZJEUyYari4qzbqYCkq6holyFPdp6xxZGXcr5ZBK9mYDf1h886D0qJQDnCAN3vmQY8ISlsWhUGUyAjcNDvEHB+Dzcv2CbvgLYXuGaX\\\\\\\/\\\\\\\/Xcdu3DM5u4UNssulvSa3RNtQUfho8S8gc2JLgjWC8kemQ64OHA4lg6e8ZUpo4z\\\\\\\/Nf90chegpeOoDQ7+xnwLAmu4MhCnL0gaoMzxQZeXtdFirnfPKyJfhQAnUIJlBCRnSE2IqcGAZH6mvG3nKU+4UETLQQDF0VZE5a8YUsZ6+cY701p0bld0kZW6t8\\\\\\\/uqJTvsY0vf3xaPVuwAsR\\\\\\\/GS3FQnlD5uAPgWkVwYtUvNtxai0rUHROZO\\\\\\\/y6XWxxIbgsKpcpqpK94KogBy8KyD0YzwoF8cyS9ZEHzlzasdFMIU9Ko\\\\\\\/sm4IcjOIKAJVI+p6Elmtfs0TuauDjNV5rhXUdAmfoPdUl0qPcNi7OnyFy3MhjFJFNnYRYYOIlpUC+LBFkzM+8Gnth9P2Z5DPq946GG8a7x1xNTINsiHZ44PJllt3TUwjqmJrLMwe1TVbeYSPsimnh+S07fFBARdHNVMwqonS9cwVrWQSRQyuZ4qToNc84dGr2hFJeHzFThsNKnHGclYrHO2HIDVpE4sDJkswdruq8HB8n4DLYYgCsmvx4NOrXmj3PPRc9\\\\\\\/wkg\\\\\\\/isPF4QjNosLZmQP0tJuzxguTM48olgACK3tUY1\\\\\\\/oZLuFQiOKsavv5ySBzSWIQX5lob0Cpy84ew01JcKBnqcM6i0QvHzAXIw0hRVBIE7V29oUOLCaZVJPsgwtesWoY\\\\\\\/tqAeBvXSPP22jW8OqHQ19fSmEsLD\\\\\\\/ts32LUbnQlS5PqBYmt5kIMnGQkaKKxoj4bIWAoxarnTjJtrh0Bdx8ixO8Aw0Wr\\\\\\\/Jl5p6IxFDc8fAspO7woEBcJLAitB3n1YKdwmvA65y91SMdCHSzHT+Kljc5TdW\\\\\\\/D3fxk4b8ACUpRqVvC4bO7zIooKf7TKghzzkLT\\\\\\\/XcTZ2N0BaQRwrBS+Ntbqg+G5sIGiHemZdSSGH29KAjMJO38UQgWJ0hO5nEreIVwEHXQe8AAHG2jC4ZhwcspAZ7XaJ5fnHvh5WgxUu8vme7Am6EeMQRjk+nu+JchRbrYhxIgme1HyMQcMAhZIA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022zkpvskaEcj34XWhTONqD+w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002254a664ea2b42499e4e2a69ac888f75a9615436ee6914a4ccf388de2cfd129134\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489470 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1649" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0LlhkFc5RovWTSLgBArX0HmGAbRMPbVqI3YYPY3X5JI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1649" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6MQVqnC18x5T8Wf0nhbO38ZPknJWR7pAHfvmvjkok18" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1637" title="A Host-Pathogen Interaction Screen Identifies ada2 as a Mediator of Candida glabrata Defenses Against Reactive Oxygen Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1659" title="Functional Validation of Candidate Genes Detected by Genomic Feature Models" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489470" id="node489470" data-pisa="ggg;8/5/1649" data-pisa-master="ggg;g3.118.200087" data-apath="/ggg/8/5/1649.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shan Wei</span>, <span class="highwire-citation-author" data-delta="1">Zachary R. Weiss</span> and <span class="highwire-citation-author" data-delta="2">Zev Williams</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1649-1657;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200087 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489470" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shan Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shan%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShan%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zw2421@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">zw2421@cumc.columbia.edu</a>
<a href="mailto:sw3203@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">sw3203@cumc.columbia.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zachary R. Weiss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BR.%2BWeiss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weiss%20ZR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZachary%2BR.%2BWeiss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zev Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zev%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZev%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zw2421@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">zw2421@cumc.columbia.edu</a>
<a href="mailto:sw3203@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">sw3203@cumc.columbia.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/5/1649" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489470" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489470/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1649.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489470" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489470/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1649.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489470" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489470/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1649.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489470" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489470/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/5/1649.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489470" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489470/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Real-time sequencing of short DNA reads has a wide variety of clinical and research applications including screening for mutations, target sequences and aneuploidy. We recently demonstrated that MinION, a nanopore-based DNA sequencing device the size of a USB drive, could be used for short-read DNA sequencing. In this study, an ultra-rapid multiplex library preparation and sequencing method for the MinION is presented and applied to accurately test normal diploid and aneuploidy samples’ genomic DNA in under three hours, including library preparation and sequencing. This novel method shows great promise as a clinical diagnostic test for applications requiring rapid short-read DNA sequencing.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/aneuploidy-detection" class="hw-term hw-article-keyword hw-article-keyword-aneuploidy-detection" rel="nofollow">aneuploidy detection</a></li><li class="kwd"><a href="/keyword/minion-nanopore-sequencing" class="hw-term hw-article-keyword hw-article-keyword-minion-nanopore-sequencing" rel="nofollow">MinION nanopore sequencing</a></li><li class="kwd"><a href="/keyword/multiplex" class="hw-term hw-article-keyword hw-article-keyword-multiplex" rel="nofollow">multiplex</a></li><li class="kwd"><a href="/keyword/small-dna" class="hw-term hw-article-keyword hw-article-keyword-small-dna" rel="nofollow">small DNA</a></li><li class="kwd"><a href="/keyword/ultra-fast" class="hw-term hw-article-keyword hw-article-keyword-ultra-fast" rel="nofollow">ultra-fast</a></li></ul><p id="p-3">Rapid sequencing of short DNA reads may be useful for a wide range of clinical and -research applications including targeted mutation analysis, cancer-panel testing, and aneuploidy screening (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11"><span class="sc">kukita</span><em>et al.</em> 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22"><span class="sc">zheng</span><em>et al.</em> 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2"><span class="sc">butler</span><em>et al.</em> 2016</a>). However, the time and skill required for library preparation and sequencing using existing DNA sequencing methods limits their widespread clinical use. Nanopore sequencing technology is one of the fastest growing 3<sup>rd</sup> generation next-generation sequencing (NGS) technologies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7"><span class="sc">haque</span><em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19"><span class="sc">wang</span><em>et al.</em> 2014</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9"><span class="sc">jain</span><em>et al.</em> 2016</a>). Different from the 2<sup>nd</sup> generation NGS sequencing platforms, such as illumina MiSeq and Ion Proton, the 3<sup>rd</sup> generation NGS platforms, including nanopore sequencing, sequenced nucleotides at single-molecule level (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16"><span class="sc">quail</span><em>et al.</em> 2012</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9"><span class="sc">jain</span><em>et al.</em> 2016</a>). MinION was the first portable nanopore sequencing platform to be commercially released (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14"><span class="sc">loman</span> and <span class="sc">watson</span> 2015</a>). It detects the electric current of single-stranded DNA (ssDNA) as it passes through a small protein channel, called a nanopore, and converts the electric current data into the corresponding sequence (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14"><span class="sc">loman</span> and <span class="sc">watson</span> 2015</a>). As this method relies on the physicochemical properties of ssDNA rather than an enzymatic reaction, sequencing occurs at speeds that are faster than 2<sup>nd</sup> generation NGS systems (15,000 nt/min <em>vs.</em> 1 nt/min using Ion Proton).</p><p id="p-4">While MinION was originally developed for sequencing long strands of DNA (&gt;8 kb and even &gt;1,000 kb), it was recently demonstrated that changes in the chemistry and library preparation could allow the device to be used for sequencing of short DNA reads (∼500 nt) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). However, library preparation for a single sample took 4 hr and multiple technically complex steps to complete, sequencing took an additional 1-4 hr, and only a single sample could be sequenced at a time (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). These factors limited the clinical utility of the MinION.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of MinION library preparation workflows. 2D library is a previously reported workflow (wei and williams 2016); 1D multiplex library is manufacturer&#x2019;s workflow using a native barcoding kit and a 1D genomic sequencing kit on the current MinION platform; Rapid 1D multiplex library is a new rapid barcoding MinION library preparation workflow reported in this study developed to sequence short reads (" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343495751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of MinION library preparation workflows. 2D library is a previously reported workflow (wei and williams 2016); 1D multiplex library is manufacturer&#x2019;s workflow using a native barcoding kit and a 1D genomic sequencing kit on the current MinION platform; Rapid 1D multiplex library is a new rapid barcoding MinION library preparation workflow reported in this study developed to sequence short reads (&lt;1000 bp) on the current platform. The length of each bar indicates the time needed. The steps are color-coded (Yellow: fragmentation; Red: end preparation including end-repair and dA-tail; blue: size selection and purification; dark blue: MyOne C1 bead purification; purple: sequencing).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1649/F1.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/5/1649/F1.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489678" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Comparison of MinION library preparation workflows. 2D library is a previously reported workflow (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>); 1D multiplex library is manufacturer’s workflow using a native barcoding kit and a 1D genomic sequencing kit on the current MinION platform; Rapid 1D multiplex library is a new rapid barcoding MinION library preparation workflow reported in this study developed to sequence short reads (&lt;1000 bp) on the current platform. The length of each bar indicates the time needed. The steps are color-coded (Yellow: fragmentation; Red: end preparation including end-repair and dA-tail; blue: size selection and purification; dark blue: MyOne C1 bead purification; purple: sequencing).</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">In this study, we report a new method that simplifies and accelerates nanopore-based short-length DNA library preparation and sequencing, and apply this method to test a panel of normal and aneuploid genomic DNA samples. We were able to accurately screen for aneuploidy using pure genomic DNA in 5 multiplexed samples in under 3 hr, including the time required for library preparation and sequencing. Using this method, the MinION nanopore sequencer can be used for a wide range of applications requiring rapid short-read DNA sequencing.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>DNA Samples</h3><p id="p-7">Normal male (NA12877), normal female (NA12878), Monosomy X (NG08006), trisomy 21 (NG05397) and trisomy 18, 15s+ (NG08016) genomic DNA (gDNA) from the Coriell Institute were used for development and testing of this protocol. The study was approved by the Institutional Review Board of Albert Einstein College of Medicine and the Institutional Review Board of Columbia University Medical Center and complied with Coriell Institute NIGMS Human Genetic Cell Repository and NIA Aging Cell Culture Repository policies.</p></div><div id="sec-3" class="subsection"><h3>Development of rapid ligation conditions</h3><p id="p-8">To develop conditions for rapid ligation that could combine ligation of both the TA end and the 6-bp sticky-end, thus reducing the time needed for library preparation, a range of ligation enhancers were tested. The ligation efficiency of 6-bp sticky-end ligation and TA ligation was estimated using different adapters for each ligation. For the 6-bp sticky-end ligation, 2 pmol of a 58 bp asymmetric adapter with a 3′-ATTGCT overhang (MP1-6bp) and 2 pmol of a 34 bp adapter with a 3′-AGCAAT and 5′ blunt-end (ME-6bp) were used (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>) (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). In addition to the 10 µL basic ligation substrates (2 µL of 1 µM MP1-6bp adapter, 2 µL of 1 µM ME-6bp adapter, 6 µL of 2 mM Tris-Cl, pH 8 (1/5 Buffer EB, Qiagen, Cat. 19086)), 50 or 10 µL Blunt/TA ligase master mix (NEB, Cat. M0367) and 0, 1.2, 1.8 or 2.4 µL enhancer mix (83.3 mM MgCl<sub>2</sub>, 16.7 mM ATP) were added to each ligation reaction and incubated at 25° for 10 min (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Each ligation reaction was purified using a DNA Clean &amp; Concentrator-5 column (Zymo, Cat. D4003 or D4013) following the manufacturer’s protocol. Each reaction mixture was mixed with DNA binding buffer at a 1:7 volume ratio (Zymo, Cat. D4003 or D4013); the ligation products were eluted in 20uL 2 mM Tris-Cl, pH 8, and products ≥30 bp were retrieved. The purified ligation products were then analyzed using 3% agarose gel electrophoresis and ImageJ (<a href="http://imagej.nih.gov/ij/">http://imagej.nih.gov/ij/</a>) densitometry analysis with 2 technical replicates (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Optimization of MinION library preparation. A). Optimization of ligation condition for TA ligation and 6-bp sticky-end ligation. Condition 1. The manufacturer&#x2019;s suggested condition; 2. the condition reported before (wei and williams 2016); 3-5. the conditions with addition of 6%, 9%, and 12% enhancer mix. Efficiencies of 6-bp ligation were estimated using a pair of adaptor MP1-6bp and ME-6bp carrying complementary 6-bp sticky ends. Efficiencies of TA ligation were estimated using a pair of adaptor MP1-T and ME-A carrying complementary 3&#x2032;T and 3&#x2032;A overhangs. B). Titration experiment of Native Barcode (NB) adapter. 6.5ng, 9.8ng, 13ng of NB adapters were added in to the 1-step ligation reaction which contains the same amount of dA-tailed DNA and MP1-6bp adapter. The expected final products with 2-end ligated to a barcode and MP1-6bp adapter were marked in bold. The products separated on gel were also illustrated in cartoons (MP1-6bp adapter: green; NB adapter: blue; dA-tailed DNA: purple). C). Optimization of end-repair/dA-tailling condition. Lane 1, the input 434bp control fragment; lane 2, manufacturer&#x2019;s recommended protocol; lane 3. the optimized condition; lane 4. The optimized condition with supplementation of Bst 2.0 WarmStart Polymerase. The expected products with 2-end ligated to an adapter were marked in bold and the products separated on gel were also illustrated in cartoons (434bp dA-tailed DNA: purple; MP1-T adapter: dark green). D). Optimization of AMPure XP bead purification by changing the volume of bead. 100 ng 50bp ladder and 2 pmole 204bp control fragment were used as input, and subjected to onefold, 0.65-fold, 0.sixfold, 0.55-fold AMPure XP bead purification. The expected products are bands &gt;500 bp, and it&#x2019;s marked in bold E). Optimization of AMPure XP bead purification by adjusting the concentration of PEG in wash buffer. 100 ng 50bp ladder and 2 pmole 204bp control fragment were used as input, and subjected to 0.62-fold AMPure XP bead purification using wash buffer containing 10%, 9%, 8.5% and 8% PEG. The expected products are bands &gt;500 bp, and it&#x2019;s marked in bold. F). Optimization of tethering condition. Lane 1-5: 1&#xB5;L BAM adapter with 0-4&#xB5;L ELB buffer after 3min incubation at 37&#xB0;C. The expected tethered BAM adapter was marked in bold. The products separated on gel were illustrated in cartoons (BAM adapter: gray; tether: pink-black)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343495751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Optimization of MinION library preparation. A). Optimization of ligation condition for TA ligation and 6-bp sticky-end ligation. Condition 1. The manufacturer&#x2019;s suggested condition; 2. the condition reported before (wei and williams 2016); 3-5. the conditions with addition of 6%, 9%, and 12% enhancer mix. Efficiencies of 6-bp ligation were estimated using a pair of adaptor MP1-6bp and ME-6bp carrying complementary 6-bp sticky ends. Efficiencies of TA ligation were estimated using a pair of adaptor MP1-T and ME-A carrying complementary 3&#x2032;T and 3&#x2032;A overhangs. B). Titration experiment of Native Barcode (NB) adapter. 6.5ng, 9.8ng, 13ng of NB adapters were added in to the 1-step ligation reaction which contains the same amount of dA-tailed DNA and MP1-6bp adapter. The expected final products with 2-end ligated to a barcode and MP1-6bp adapter were marked in bold. The products separated on gel were also illustrated in cartoons (MP1-6bp adapter: green; NB adapter: blue; dA-tailed DNA: purple). C). Optimization of end-repair/dA-tailling condition. Lane 1, the input 434bp control fragment; lane 2, manufacturer&#x2019;s recommended protocol; lane 3. the optimized condition; lane 4. The optimized condition with supplementation of Bst 2.0 WarmStart Polymerase. The expected products with 2-end ligated to an adapter were marked in bold and the products separated on gel were also illustrated in cartoons (434bp dA-tailed DNA: purple; MP1-T adapter: dark green). D). Optimization of AMPure XP bead purification by changing the volume of bead. 100 ng 50bp ladder and 2 pmole 204bp control fragment were used as input, and subjected to onefold, 0.65-fold, 0.sixfold, 0.55-fold AMPure XP bead purification. The expected products are bands &gt;500 bp, and it&#x2019;s marked in bold E). Optimization of AMPure XP bead purification by adjusting the concentration of PEG in wash buffer. 100 ng 50bp ladder and 2 pmole 204bp control fragment were used as input, and subjected to 0.62-fold AMPure XP bead purification using wash buffer containing 10%, 9%, 8.5% and 8% PEG. The expected products are bands &gt;500 bp, and it&#x2019;s marked in bold. F). Optimization of tethering condition. Lane 1-5: 1&#xB5;L BAM adapter with 0-4&#xB5;L ELB buffer after 3min incubation at 37&#xB0;C. The expected tethered BAM adapter was marked in bold. The products separated on gel were illustrated in cartoons (BAM adapter: gray; tether: pink-black).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1649/F2.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/5/1649/F2.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489744" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-9" class="first-child">Optimization of MinION library preparation. A). Optimization of ligation condition for TA ligation and 6-bp sticky-end ligation. Condition 1. The manufacturer’s suggested condition; 2. the condition reported before (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>); 3-5. the conditions with addition of 6%, 9%, and 12% enhancer mix. Efficiencies of 6-bp ligation were estimated using a pair of adaptor MP1-6bp and ME-6bp carrying complementary 6-bp sticky ends. Efficiencies of TA ligation were estimated using a pair of adaptor MP1-T and ME-A carrying complementary 3′T and 3′A overhangs. B). Titration experiment of Native Barcode (NB) adapter. 6.5ng, 9.8ng, 13ng of NB adapters were added in to the 1-step ligation reaction which contains the same amount of dA-tailed DNA and MP1-6bp adapter. The expected final products with 2-end ligated to a barcode and MP1-6bp adapter were marked in bold. The products separated on gel were also illustrated in cartoons (MP1-6bp adapter: green; NB adapter: blue; dA-tailed DNA: purple). C). Optimization of end-repair/dA-tailling condition. Lane 1, the input 434bp control fragment; lane 2, manufacturer’s recommended protocol; lane 3. the optimized condition; lane 4. The optimized condition with supplementation of Bst 2.0 WarmStart Polymerase. The expected products with 2-end ligated to an adapter were marked in bold and the products separated on gel were also illustrated in cartoons (434bp dA-tailed DNA: purple; MP1-T adapter: dark green). D). Optimization of AMPure XP bead purification by changing the volume of bead. 100 ng 50bp ladder and 2 pmole 204bp control fragment were used as input, and subjected to onefold, 0.65-fold, 0.sixfold, 0.55-fold AMPure XP bead purification. The expected products are bands &gt;500 bp, and it’s marked in bold E). Optimization of AMPure XP bead purification by adjusting the concentration of PEG in wash buffer. 100 ng 50bp ladder and 2 pmole 204bp control fragment were used as input, and subjected to 0.62-fold AMPure XP bead purification using wash buffer containing 10%, 9%, 8.5% and 8% PEG. The expected products are bands &gt;500 bp, and it’s marked in bold. F). Optimization of tethering condition. Lane 1-5: 1µL BAM adapter with 0-4µL ELB buffer after 3min incubation at 37°C. The expected tethered BAM adapter was marked in bold. The products separated on gel were illustrated in cartoons (BAM adapter: gray; tether: pink-black).</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">The efficiency of the TA ligation was estimated by the same method as 6-bp sticky-end ligation but using a different pair of adapters that had 3′ T/A overhangs (MP1-T and ME-A) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>). 4 uL 200 nM MP1, 4 µL 200 nM ME-A adapters, and 2 µL 2 mM Tris-Cl, pH 8 were subjected to each ligation condition as described above (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>).</p><p id="p-11">Although the 12 native barcoding (NB) adapters in a 1D native barcoding kit (Oxford Nanopore, EXP-NBD103) are designed to be supplied at 670 nM (communications with the manufacturer’s technical support), the actual concentration of each NB adapters in a kit can vary from 2-13 ng/µL. Thus, the DNA content of each NB adapter was quantified using the Qubit dsDNA HS assay (Invitrogen, Cat. Q32851). To determine the right amount of NB adapter needed in a 1-step ligation reaction, a titration ligation experiment was performed. 6.5 ng, 9.8 ng, or 13 ng NB adapters were added to a 21.2 µL ligation reaction mixture containing 0.2 pmole dA-tailed 434bp, 1.6 pmole MP1-6bp, 10 µL Blunt/TA ligase master mix, and 1.2 µL enhancer mix (83.3 mM MgCl2, 16.7 mM ATP) in 2 mM Tris-HCl. Each ligation reaction mixture was incubated at 25° for 10 min, and purified using a DNA Clean &amp; Concentrator-5 column as described above. The purified ligation products were analyzed using 3% agarose gel electrophoresis and ImageJ densitometry analysis (<a href="http://imagej.nih.gov/ij/">http://imagej.nih.gov/ij/</a>) (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>).</p></div><div id="sec-4" class="subsection"><h3>Development of rapid end-repairing/dA tailing condition</h3><p id="p-12">A short DNA control fragment was used to assess the efficiency of different conditions for end-repairing/dA-tailing. This fragment was generated by PCR using M13 forward and reverse primers to amplify a 434 bp fragment from a <em>pCR-Blunt</em> vector (Invitrogen, Cat. K2700-20) using Q5 High-Fidelity DNA Polymerase (NEB, Cat. M0491S) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>) as reported previously (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Wei and Williams 2016</a>).</p><p id="p-13">The resulting fragment was mixed with the NEBNext Ultra II End Repair/dA-Tailing Module (NEB, Cat. E7546L) in a clean 8-well 0.2 mL PCR strip and incubated for 5min at 20° for end-repair reaction, and then 5 min at 65° for dA-tailing reaction. 3 conditions were tested to optimize end-repair/dA-tailing: 1) ∼0.5 pmol DNA (∼162.5 ng) in 60uL total reaction mixture; 2) ∼0.5 pmol DNA in 30 µL total reaction mixture with addition of 0.9 µL 100 mM dATP before the 65° incubation; 3) ∼0.5 pmol DNA control fragment in 30 µL reaction mixture with addition of 0.9 µL 100 mM dATP and 1µL Bst 2.0 WarmStart DNA polymerase (NEB, Cat. M0538S) before the 65° incubation. Each reaction mixture was then purified using a DNA Clean &amp; Concentrator-5 column (Zymo, Cat. D4003 or D4013) following the manufacturer’s protocol as described above.</p><p id="p-14">The efficiency of end-repair/dA-tailing of each of the 3 conditions was assessed using a 57-bp asymmetric adapter with a 3′T overhang (MP1-T) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>). The adapter was diluted to 0.4 μM in MinION adapter buffer (50 mM NaCl, 10 mM Tris-HCl, pH 7.5), and ligated to the dA-tailed control fragment in a 10:1 ratio as previously reported (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). 21.2 µL ligation reaction included 4 µL 50 nM dA-tailed DNA, 5 µL 0.4 µM adapter, 1 µL nuclease-free water (Ambion, Cat. AM9937), 10 µL Blunt/TA ligase master mix (NEB, Cat. M0367S), and 1.2 µL enhancer mix (83.3 mM MgCl2, 16.7 mM ATP). The ligation reaction was incubated at 25° for 10 min, purified using a DNA Clean &amp; Concentrator-5 column as above, and analyzed with 3% agarose gel electrophoresis and ImageJ densitometry analysis (<a href="http://imagej.nih.gov/ij/">http://imagej.nih.gov/ij/</a>) (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>).</p></div><div id="sec-5" class="subsection"><h3>Development of library purification conditions</h3><p id="p-15">10% PEG 8000 (PEG) wash buffer (10% PEG, 750 mM NaCl, 50 mM Tris-HCl, pH = 8) was prepared following the protocol in the Oxford Nanopore MAP003 kit. 9%, 8.5%, and 8% PEG buffers were also prepared to determine which concentration would most efficiently remove excessive adapters through AMPure XP bead purification (Agencourt, Cat. A63881). PEG wash buffers were stored at 4°. 2 pmole (∼260 ng) of a 204 bp DNA fragment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>) and 100 ng of a 50 bp ladder (Invitrogen, Cat. 10416014) were mixed in Buffer EB (Qiagen, Cat. 19086) to make a 60 µL reaction mixture.</p><p id="p-16">To determine the optimal concentration of AMPure XP beads for the 1-step ligation reaction purification, beads were added to the reaction mixture at 1:1, 0.65:1, 0.60:1, or 0.55:1 bead: sample ratios in each well of a 0.2 mL PCR strip (USA Scientific, Cat. 1402-4700) and purified according to the manufacturer’s protocol on a 96-well magnet plate (ALPAQUA, Cat. A001322). However, in the wash steps, the 10% PEG wash buffer described above was substituted for the 70% ethanol used in the manufacturer’s protocol. After two washes, the beads were resuspended in 20 µL Buffer EB (Qiagen, Cat. 19086) and incubated at 37° for 5 min. The resuspended beads were allowed to pellet on the magnet plate, and the eluate was carefully transferred to a 1.5 mL low retention tube (USA Scientific, Cat. 1415-2600 or Phenix, Cat. MH-815S). The AMPure XP bead purified products were analyzed with 2% agarose gel electrophoresis (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>).</p><p id="p-17">To determine the optimal PEG concentration in PEG wash buffer to remove extra adapters, AMPure XP bead purification was performed using a 0.62:1 beads: sample ratio and washed in each PEG wash buffer described above (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2E</a>).</p><p id="p-18">To test the efficiency of adapter removal for multiple reactions, 8 reaction mixtures were purified by 0.62X AMPure XP bead purification and washed in 8% PEG buffer as described above. The efficiency was not altered (data not shown).</p></div><div id="sec-6" class="subsection"><h3>Development of library tethering conditions</h3><p id="p-19">At the end of library preparation, each 1D barcoding sequencing adapter (BAM) (Oxford nanopore, EXP-NBD103) must be annealed to tethering oligonucleotides (tethers), which carry a hydrophobic group on their 5′end, included in the elution buffer (ELB) (Oxford nanopore, SQK-LSK108). This process is called tethering in the manufacturer’s protocol. Tethering in ELB buffer can assist the barcode sequencing adapters (BAM) to reach the nanopores faster. Motor proteins are pre-attached to BAM adapters. When a BAM adapter reaches a nanopore, the motor protein can unzip dsDNA into ssDNA and drive the resulting DNA strand through the nanopore at a fixed speed. The ideal tethering condition was determined by mixing the BAM adapter with ELB buffer in 1:1, 1:2, 1:3, and 1:4 ratios and incubated at 37° for 3 min. The BAM adapter was completely tethered at a 1:2 BAM: ELB ratio (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2F</a>). Tethering was also tested at 25° and on ice at a 1:2 ratio for 10 min but these conditions were less efficient than tethering at 37° for 3 min (data not shown).</p></div><div id="sec-7" class="subsection"><h3>Rapid multiplex MinION Sequencing library preparation</h3><p id="p-20">Each optimized step was combined to form our rapid multiplex MinION sequencing library preparation. ∼500 ng gDNA was fragmented with a Covaris microTUBE using the manufacturer’s default 500-bp setting. ∼0.5 pmole of fragmented gDNA (165 ng ∼500-bp DNA) was subjected to a 30 µL end-repair/dA-tailing reaction using the NEBNext Ultra II End Repair/dA-Tailing Module (NEB, Cat. E7546L) as detailed above with addition of 3 mM dATP after 5min incubation at 20°, followed by 5 min incubation at 65°. A Select-a-Size DNA Clean &amp; Concentrator (Zymo, Cat. D4080) was used to purify each end-repair/dA-tail reaction following the manufacturer’s protocol. 156 µL size selection mix (500 µL Select-a-Size DNA binding buffer + 5 µL 100% ethanol, can be scaled up for more reactions) was added to each end-repair/dA-tailing reaction tube and gently mixed by pipetting. The size-selected DNA was then eluted in 12 µL 2 mM Tris-Cl, pH 8.</p><p id="p-21">A 1D native barcoding kit (Oxford Nanopore, EXP-NBD103) and a 1D Ligation Sequencing Kit (Oxford Nanopore, SQK-LSK108) were used for library preparation. As the native barcoding (NB) adapters in each kit (Oxford Nanopore, EXP-NBD103) can vary in concentration, the DNA content was quantified using the Qubit dsDNA HS assay (Invitrogen, Cat. Q32851). 6-10 ng NB adapter were used for each adapter ligation reaction.</p><p id="p-22">In each 1-step ligation reaction, 15 µL ligation components (65 ng size selected dA-tailed DNA, 6-10 ng NB adapter, 8 µL BAM adapter, topped up to 15 µL using 2 mM Tris-HCl), 15 µL Blunt/TA ligase master mix, and 3.6 µL enhancer mix were mixed thoroughly. The mixture was then incubated at 25° for 10 min. Two or more ligation reactions can be performed for 1 sample using 1 barcode to increase the final library concentration. In run 1, 1 NB adapter was used to barcode 1 sample and 4 ligation reactions were prepared for this sample; In run 2 and run 3, 5 NB adapters were used to barcode 5 sample and 2 ligation reactions were used for each sample (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). 26.4 µL Buffer EB and 21.6 µL AMPure XP beads were added to each ligation reaction to perform bead purification on a magnet plate (equivalent to ∼0.62 fold beads: sample ratio when counting the PEG in Blunt/TA ligase master mix). 8% PEG wash buffer was used as described above. The beads from all the ligation reactions were pooled in a clean 1.5 mL low retention tube after the first wash, pelleted, and washed again on a magnet stand (Agencourt, Cat. A29182). The resulting library was eluted in 16-20 µL ELB buffer by incubation at 37° for 3-5 min. The DNA concentration was quantified with Qubit dsDNA HS assay using 1µL library. The final concentration of the library was ∼15-20 ng/µL. In manufacturer’s protocol, for DNA shorter then 3kb, it’s suggested to use 0.2 pmoles gDNA for library preparation. The concentration would be ∼6ng/µL for a library for 500bp fragments. It’s not sufficient to generate sufficient yields. It’s suggested to use a pre-sequencing library with ∼20ng/µL or higher to produce comparable results.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489516/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489516/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">MinION assay results</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>MinION sequencing</h3><p id="p-24">12 µL of the library was loaded into a MinION MIN106 flow cell for sequencing following manufacturer’s protocol. MINKnow v1.3.30 software and run protocol NC_48Hr_sequencing_Run_FLO-MIN106_SQK-LSK108_plus_Basecaller were used to control and monitor the sequencing in real time. To maximize the data output for rapid MinION sequencing, the data acquisition was restarted after a 1 h run, and stopped when sufficient data were generated. A 30 min sequencing run generated enough reads for 1 sample; a 1-3 h sequencing run generated enough reads for 5 barcoded samples. The average number of real-time pores used for strand sequencing was 114-145 as monitored on the MINKnow software and the total number of pores generated sequences was &gt;500 (data not shown).</p></div><div id="sec-9" class="subsection"><h3>Data analysis</h3><p id="p-25">The MinKNOW run protocol NC_48Hr_sequencing_Run_FLO-MIN106_SQK-LSK108_plus_Basecaller generated sequencing results in FAST5 format. Sequences that passed the protocol’s quality filter were converted from FAST5 format to FASTA format using Poretools version 0.51 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13"><span class="sc">loman</span> and <span class="sc">quinlan</span> 2014</a>; <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). The FASTA files were processed with Cutadapt version 1.14 to demultiplex the data using parameter (-O 20 –e 0.20 –m 50) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15"><span class="sc">martin</span> 2011</a>). At least a 20 bp match to a barcode with a maximum error rate of 0.20 was required to be considered a match to the barcode. Sequences ≥50 bp after demultiplexing were kept for downstream analysis. Sequences were aligned to human reference genome GRCh37 using Pblat (BLAT in parallel setting) (<a href="http://icebert.github.io/pblat/">http://icebert.github.io/pblat/</a>) using the parameters stated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS2.docx">Table S2</a> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10"><span class="sc">kent</span> 2002</a>; <a id="xref-ref-21-8" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). The performance of Pblat was evaluated using 20,000 sequences sampled from run 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS2.docx">Table S2</a>). A new alignment software, minimap2, was also evaluated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS2.docx">Table S2</a>) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12"><span class="sc">li</span> 2017</a>). This software is designed for various platforms including nanopore sequencing. It requires less computational resources and less run time at the cost of 5–6% less reads that can be used for downstream analysis. In this study, we used the alignment results from Pblat software to achieve the maximum number of reads for downstream data analysis.</p><p id="p-26">The first 9,000 uniquely assigned (UA) from each sample were used for aneuploidy detection analysis. Given 9,000 UA reads, &gt;41 UA reads were assigned to chrY in a normal male sample. Data analysis and statistical analysis were performed in R version 3.4.0 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18"><span class="sc">team</span> 2012</a>). Under Poisson distribution, when 41 UA reads were assigned to one chromosome (λ = 41), the type I error for false positive detection of a 50% increase (<em>e.g.</em>, a full trisomy) p(x &gt; 1.5 λ) = 0.0008; the type II error for false negatively detecting a 50% changes p<sub>β</sub>(x’ &lt; 1.5 λ) = 0.0021 and the type II error for false negatively detecting a 30% changes p<sub>β</sub>(x’ &lt; 1.3 λ) = 0.04 as estimated by ppois function in R. UA reads aligned to each chromosome were summarized and analyzed using the modified Z-score method to identify normal diploid and aneuploid chromosomes as reported previously (<a id="xref-ref-21-9" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). A known normal male sample was used as a reference. The standard deviation of the relative chromosomal copy number of normal autosomes included in this study, sd_normal = 0.0897 (n = 219), was used to determine aneuploid chromosomes using the modified Z-score method as reported before (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS3.docx">Table S3</a>) (<a id="xref-ref-21-10" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-27">Sequence data for normal male (NA12877) and normal female (NA12878) from 3 sequencing runs in this study are available on European Genome-phenome Archive (EGA) with the accession number EGAS00001002650. The bed files of all each individual containing no personally identifying genetic information (PIGI) are available upon request.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><p id="p-28">In this study we developed a rapid and practical protocol for preparing and sequencing a 1D multiplex genomic DNA short-read sequencing library using a nanopore sequencer. This was achieved by systematically and quantitatively evaluating and optimizing each module in the library preparation workflow. This provided robust reaction conditions in each module, and reduced required library preparation time from 2-4 h to ∼45 min and sequencing time from 1-4 h to &lt;30 min when compared to the original protocol (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-21-11" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>).</p><p id="p-29">A 2D library generates sequences from both template and complementary strand of each DNA fragment; a 1D library generated sequence from the template strand (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9"><span class="sc">jain</span><em>et al.</em> 2016</a>). On previous MinION MAP006 platform, a 2D library was prepared to increase the sequencing accuracy (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). However, on current MinION MIN106 platform, the sequencing quality was improved due to improvements on nanopore protein and basecalling algorithm (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8"><span class="sc">jain</span><em>et al.</em> 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3"><span class="sc">carter</span> and <span class="sc">hussain</span> 2017</a>), a 1D library can be prepared to generate sequences of comparable sequence quality as a 2D library on old platform (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-30">A rapid 1D multiplex library includes 5 steps (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). 1) DNA fragmentation: DNA is sheared to ∼500bp by ultrasonication. 2). End preparation: sheared DNA fragments are repaired to blunt-ends with 5′ phosphorylation modification, and then 3′dA-tails are added. 3). Size selection to remove reads &lt;400bp: Zymo Select-a-Size columns can perform size selection during column purification to remove reads &lt;400 bp. 4) 1-step ligation for barcodes and sequencing adapters: a native barcode (NB) adapter carries a 3′T overhang on one end that can be ligated to a 3′dA-tailed DNA fragment; it carries a 3′ 6bp overhang on the other end that can be ligated to the 1D barcode sequencing adapter (BAM). The TA ligation between DNA fragments and NB adapters and the 6-bp sticky-end ligation between NB and BAM adapters are carried out in one optimized ligation reaction at the same time. 5) Purification: the ligation reaction needs to be purified to retrieve fragments that are attached to NB and BAM adapters and eliminate unnecessary materials such as enzyme, extra adapters, etc. There is a pre-attached motor protein on BAM adapters which will assemble on a nanopore to unzip dsDNA into ssDNA strands and drive it for nanopore sequencing (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9"><span class="sc">jain</span><em>et al.</em> 2016</a>). The motor protein is sensitive to protein denaturants, freeze-and-thaw cycles and heat. Hence a wash buffer containing PEG was used instead of 70% ethanol in washing steps during bead purification. When the library is eluted off the beads, there is a hidden tethering step to anneal tethering oligonucleotides in elution buffer (ELB) to BAM adapters on fragments. The tethering oligonucleotides can help the fragment to reach a nanopore faster. This is done simultaneously at the end of the library purification step and no extra action needs to be taken. The rapid 1D barcoding library preparation method we present is a simplified and optimized method comparing to the current manufacturer’s 1D native barcoding library preparation method.</p><div id="sec-12" class="subsection"><h3>Development of rapid ligation conditions</h3><p id="p-31">Purifications are the most time-consuming steps in a library preparation workflow. It can add 15-30 min to each step during library preparation (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). To bypass several ligation/purification steps in manufacturer’s 1D native barcoding library preparation workflow (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>), we sought to develop a 1-step ligation condition that could allow simultaneous ligation for TA and 6-bp sticky ends, thereby enabling dA-tailed fragments to be ligated to a native barcoding (NB) adapter and then to a barcoding sequencing adapter (BAM) in one step. The NB adapter is designed asymmetrically with a 3′ 6-bp on one end and a 3′ T overhang on the other end. To quantitatively evaluate the ligation efficiency of 6bp sticky end ligation, a 62-bp asymmetric adapter (MP1-6bp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>) with a 6bp overhang and a 34 bp adapter (ME-6bp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>) were mixed with the complementary 6-bp overhang in a 1:1 ratio and subjected to a range of ligation conditions (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2A</a>). The portions of ligated and unligated MP1-6bp adapters were analyzed by 3% agarose gel electrophoresis and measured by densitometric analysis. To evaluate the corresponding ligation efficiency of TA ligation, a 57-bp asymmetric adapter (MP1-T) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>) was mixed with a 3′T overhang and a 20 bp adapter (ME-A) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>) was mixed with the complementary 3′A overhang in a 1:1 ratio and both mixtures were subjected to the ligation conditions described above (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2A</a>). Addition of enhancer mix increased the concentration of MgCl<sub>2</sub> and ATP in a ligation system in a 5:1 ratio. Compared to the manufacturer’s protocol, using a 100 µL ligation mixture, 20 µL ligation conditions were more efficient on TA ligation, and robust on 6-bp sticky end ligation; unligated MP1-T adapter was reduced from 24.4 to 12.5%, and the unligated MP1-6bp adapter percentage from 5.1 to 4% (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a>, Condition 2 <em>vs.</em> 1). In a 20 µL TA ligation reaction, addition of 6–12% volume of enhancer mix further reduced the percentage of unligated MP1 from 12.5 to 6.8–9.3%. Addition of 6–12% volume (1.2 to 2.4 µL) of enhancer mix further reduced the percentage of unligated MP1-6bp from 4 to ≤ 3% (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2A</a>). This method provides an efficient one-step ligation condition to simultaneously attach NB adapters to DNA fragments and BAM adapters to NB adapters without denaturing the motor protein of the BAM adapter.</p><p id="p-32">During development, we noticed inconsistency in the concentration of NB adapters provided in the same kit. A titration experiment of NB adapter with a fixed amount of MP1-6bp adapter and dA-tailed DNA fragments was performed to determine the right range of NB adapter to be used in the 1-step ligation (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2B</a>). Adding 6.5 ng-9.8 ng NB adapter resulted in most DNA fragments having NB adapters ligated on both sides, and most NB adapters were ligated. Only 1 BAM adapter is needed to sequence one DNA fragment in a 1D nanopore sequencing library. With addition of 13 ng NB adapters, NB adapters exceeded the amount of MP1-6bp adapter which resulted in more DNA fragments that ligated to NB adapters only but not MP1-6bp. In actual library preparation, ∼10ng NB for each ligation experiment was used as guided by this experiment.</p></div><div id="sec-13" class="subsection"><h3>Development of rapid end-repairing/dA tailing condition</h3><p id="p-33">The NEBNext Ultra II End Repair/dA-Tailing Module (NEB, Cat. E7546L) combines end repair and dA-tailing in one reaction. We evaluated the efficiency of this product by mixing dA-tailed DNA samples with 10-fold diluted MP1 adapter (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS1.docx">Table S1</a>), and subjecting them to the optimized ligation method determined above. Using the same ligation conditions, TA ligation efficiency was affected by the proportion of fragments that had successfully been end-repaired and dA-tailed. The manufacturer’s Ultra II End-RepairR/dA-Tailing Module protocol did not provide comparable dA-tailing efficiency to that reported in our previous study (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2C</a>, lane 1) (<a id="xref-ref-21-12" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>), The majority of control fragments were 1-end ligated. However, reducing the volume of each preparation of the Ultra II End-Repair/dA-tailing module from 60 µL to 30 µL and supplying 0.9 µL 100 mM dATP after the 20° incubation significantly improved the reaction efficiency from ∼30% two end-ligated to ∼80% (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2C</a>). During the development phase, we noticed batch effects on the Ultra II End-repair/dA-tailing module. Addition of 0.9 µL 100 mM dATP before the second incubation did not affect the ligation efficiency if the module had already reached its maximum ∼80% efficiency; it did, however, increase the efficiency to ∼80% if the module had only reached ∼50% efficiency (data not shown). Addition of more Bst 2.0 WarmStart DNA Polymerase (NEB, Cat. M0538) did not improve the dA-tailing efficiency (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2C</a>). The end-repairing/dA tailing could be performed in 12min, providing ∼80% 2-end ligation products, which is faster and more efficient than our previously reported system, which required &gt;1h for the reaction and purification, and provided ∼63% two-end ligated products (<a id="xref-ref-21-13" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>).</p></div><div id="sec-14" class="subsection"><h3>Development of library purification conditions</h3><p id="p-34">There were two purification steps in the standard library preparation workflow. The first purification was performed after the end-repair/dA-tailing reaction. A Select-a-Size DNA Clean &amp; Concentrator was used to perform a quick column-based reaction purification and size selection for short fragments &lt;400 bp. Addition of 5 µL 100% ethanol to 500 A Select-a-Size DNA binding buffer discards fragments &lt;400 bp while retaining fragments ≥400 bp efficiently (data not shown). The size selection and purification can be performed in ∼7 min.</p><p id="p-35">The second purification was performed after the ligation reaction. This purification step is gentle to protein and efficient in removing excess adapters. AMPure XP beads with PEG-based wash buffer were used to protect the motor protein attached to the BAM sequencing adapter. We first used 0.62-fold equivalent AMPure XP beads to achieve size selection for fragments &lt;400 bp after the purification, but a noticeable amount of ∼200-bp adapter (BAM+NB) was retained after the bead purification. 1.2 to 2 pmol NB and BAM adapters were ligated to 0.2 pmol DNA fragments in a ligation reaction. To determine the optimal AMPure XP bead purification condition the purification efficiency was evaluated by mixing 100 ng 50 bp ladder and 2 pmol 204bp DNA fragment, and subjected to different purification conditions (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2D</a>, <a id="xref-fig-2-18" class="xref-fig" href="#F2">2E</a>). onefold AMPure XP bead did not eliminate most of the 200-bp DNA fragments. ≤0.65-fold AMPure XP bead reduced the 204-bp DNA fragment to &lt;5% of the original input, but it was still visible when analyzed (<a id="xref-fig-2-19" class="xref-fig" href="#F2">Figure 2D</a>). Reducing the AMPure XP volume to as low as 0.55-fold did not significantly reduce the 204-bp DNA fragment retention when compared to 0.65 and 0.60-fold AMPure XP bead. (<a id="xref-fig-2-20" class="xref-fig" href="#F2">Figure 2D</a>). The wash step of the purification protocol was then optimized. The PEG-based wash in the MAP003 kit (Oxford nanopore, MAP003) contains 10% PEG, equivalent to the PEG concentration in a ∼0.eightfold AMPure XP bead purification reaction. The size selection effect in reducing the PEG concentration in PEG-based wash buffer from 10 to 9%, 8.5%, and 8% coupled with 0.65-fold AMPure XP bead purification was tested (<a id="xref-fig-2-21" class="xref-fig" href="#F2">Figure 2E</a>). All conditions retained most of the fragments ≥450 bp. The wash buffers with 10%, 9% and 8.5% PEG retained visible 200 bp fragments, but no 200 bp fragments were visibly retained using 8% PEG wash buffer, the optimal condition.</p></div><div id="sec-15" class="subsection"><h3>Development of library tethering conditions</h3><p id="p-36">To concentrate the oligonucleotide to be sequenced at the bottom of the sequencing flow cell where the nanopores are located, the BAM adapter needs to be tethered to an oligonucleotide in the ELB buffer that attached to a hydrophobic group. Using Dynabeads MyOne Streptavidin C1 bead-based 2D library purification (Invitrogen, Cat. 65001), the library was eluted from the beads in ELB buffer at 37° for 10 min (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-21-14" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). Using manufacture’s 1D library preparation protocol, the library was eluted from the beads in ELB buffer at RT °C for 10 min (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>). This does not only elute the library from the beads, but also anneal the adapter to the tethering oligonucleotide. The tethering efficiency was tested by mixing 1 µL BAM adapter with 0-4 µL ELB buffer incubated at 37° for 3 min (<a id="xref-fig-2-22" class="xref-fig" href="#F2">Figure 2E</a>); it’s also tested by mixing 1 µL BAM adapter with 2 µL ELB buffer with incubation at RT for 10 min, and at 37° for 2 min, 5 min and 10 min (data not shown). 3-5 min incubation at 37° of 1 µL BAM and ≥2 µL ELB buffer was determined to be the optimal tethering condition.</p></div><div id="sec-16" class="subsection"><h3>MinION sequencing and data analysis</h3><p id="p-37">Each of these optimized conditions combined to form a robust rapid multiplexed MinION Sequencing library preparation workflow in ∼45 min (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>). The current MinION MIN106 platform provides higher sequencing quality for 1D reads than previous MinION platforms. Libraries were sequenced for 1-3 h and stopped when sufficient data were generated (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The read length of majority of reads felt between 500-1000bp, and the quality score (Q-score) of each base ranged from can range from 4 to 16 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/FigureS1.tif">Figure S1</a>). The mean Q-score is ∼9-10 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/FigureS1.tif">Figure S1</a>). The platform generated ∼70K raw reads per hour. 55–80% reads could be assigned to a unique barcode, and 92–95% reads with a barcode could be aligned to a unique genomic location (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). For a single sample, 12 min of sequencing generated 9K UA reads, 27 min generated 15K UA. This compares favorably to our prior method which required 57 min for the same number of reads (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) (<a id="xref-ref-21-15" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). For barcoded samples, 1-3 h was sufficient to generate data for aneuploidy detection for up to 5 samples (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Three batches of samples were tested in this study, and aneuploidy was determined using the adjusted Z-score method reported in our previous study (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS3.docx">Table S3</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B-E</a>) (<a id="xref-ref-21-16" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). The normal male and female, trisomy and monosomy cases were detected concordantly with their karyotyping (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MinION Run performance and assay results. A). The run performance of the rapid 1D barcoding MinION library preparation method. B). Illustation of MinION assay results for a normal male, normal female, trisomy 21 and a monosomy X. Normal and aneuploidy on each chromosome was indicated by color (Normal: black; Aneuploidy: red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1343495751" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;MinION Run performance and assay results. A). The run performance of the rapid 1D barcoding MinION library preparation method. B). Illustation of MinION assay results for a normal male, normal female, trisomy 21 and a monosomy X. Normal and aneuploidy on each chromosome was indicated by color (Normal: black; Aneuploidy: red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1649/F3.medium.gif" width="319" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/5/1649/F3.medium.gif" width="319" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1649/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489616" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">MinION Run performance and assay results. A). The run performance of the rapid 1D barcoding MinION library preparation method. B). Illustation of MinION assay results for a normal male, normal female, trisomy 21 and a monosomy X. Normal and aneuploidy on each chromosome was indicated by color (Normal: black; Aneuploidy: red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">This method does have limitations. During development, we evaluated the time needed using pure genomic DNA. In actual lab or clinical applications, time used to obtain sufficient DNA needs to be added. It takes ∼1 h to extract gDNA from body fluids and cell lines, and 2-3 h to extract gDNA from tissues, and longer for difficult samples such as fixed tissue and slides. Time and computational resources are also needed for data analysis. In the current pipeline, ∼30 min is needed to analyze each sample (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS2.docx">Table S2</a>). Analyses can be performed in parallel in 30min given sufficient computational resources. A new nanopore sequence aligner, minimap2, can be a potential substitute in to perform data analysis in 10min with less computational resources at the cost of 5–6% loss in UA reads for downstream analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1/TableS2.docx">Table S2</a>).</p><p id="p-40">In this study, the enlargement of satellite DNA on chromosome 15 could not be detected due to the detection limit of ultra-low-coverage-sequencing (ULCS)-based methods (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). ULCS-based method used sequence reads that can be uniquely aligned to human reference genome for data analysis; sequences aligned to highly repetitive regions such as satellite DNA and other genomic repetitive elements were eliminated from the analysis. The method is currently used for aneuploidy detection at the whole chromosome scale. At the current sequencing depth of 9K reads, we estimate that copy number variations (CNVs) can only be reliably detected if they are &gt;30MBs in length; to detect CNVs of &lt;10Mb, we estimate that a read depth of &gt;30K reads would be necessary. Reducing number of samples or increasing sequencing time might be needed for detection of large CNV events, though this would need to be confirmed experimentally.</p><p id="p-41">Last but not least, alignment-based copy number estimation as applied in this study has limitation in detecting some polyploid cases when the sex chromosomes are at the same ratio as a normal male or female (<em>e.g.</em>, 69, XXX). A Karyotype or SNP-based approaches is more reliable in polyploidy detection at this point (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6"><span class="sc">handyside</span> 2011</a>).</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-42">In this study, we developed a clinically viable protocol for rapid sequencing of short DNA fragments utilizing the MinION nanopore sequencer. As the nanopore sequencing platform was developed for ultra-long fragment sequencing, the manufacturer’s protocol is not ideal for preparing short-read sequencing libraries. Thus, we systematically optimized each module of the library preparation workflow, including the end repair/dA tailing reaction, column size selection, 1D barcoding ligation, and bead purification, to achieve a practical rapid 1D barcoded sequencing library preparation protocol. The adjustments in each step were critical to performing robust and efficient reactions and purifications. The resulting protocol is less dependent than previously described protocols on the version of the sequencing library kit used. It utilizes the NB and BAM adapters in the native barcoding kit and the ELB buffer in the 1D genomic sequencing kit that are readily available whenever a compatible version of flow cell is on the market.</p><p id="p-43">We then successfully used this method for aneuploidy testing in genomic DNA samples. Using this method, up to five samples can be multiplexed to produce sufficient sequencing data for aneuploidy detection in 1-3h on a single flow cell, including library preparation time. It also takes additional 1-3h for gDNA extraction from human tissues or body fluids, and 30min for computation analysis in the development phase. Faster computation analysis can be achieved by allocating more computational resources and performing analysis on a SSD drive, or using newly developed software. Fetal aneuploidy testing is routinely performed as an essential component of prenatal testing (<em>e.g.</em>, amniocentesis and chorionic villus sampling (CVS)), preimplantation genetic screening (PGS) of embryos after <em>in-vitro</em> fertilization (IVF), and evaluation of miscarriage tissue (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1"><span class="sc">brezina</span><em>et al.</em> 2012</a>; <a id="xref-ref-21-17" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). A rapid diagnosis is clinically important as it enables timely management. Current standard methods to diagnose aneuploidy, such as karyotyping and microarray assays, take 7-21 days to complete (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17"><span class="sc">reddy</span><em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20"><span class="sc">wapner</span><em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5"><span class="sc">dong</span><em>et al.</em> 2016</a>). It also costs &gt; $1,000 per assay. Ultra-low coverage sequencing (ULCS) for detection of aneuploidy is a new and powerful strategy for whole-genome aneuploidy detection with shorter turn-over time, but still requires 15-24 h to complete and requires technically advanced library preparation and costly sequencing platforms ($80,000-$128,000) that cannot be easily applied in a physician’s office or in low complexity settings (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4"><span class="sc">chen</span><em>et al.</em> 2014</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5"><span class="sc">dong</span><em>et al.</em> 2016</a>; <a id="xref-ref-21-18" class="xref-bibr" href="#ref-21"><span class="sc">wei</span> and <span class="sc">williams</span> 2016</a>). Thus, the method described here is the fastest sequencing-based method for aneuploidy detection reported to date and testing can be performed for ≤$150 per sample on an affordable sequencing platform ($1,000). It can be set up easily at the point-of-care or in low complexity settings.</p><div id="sec-18" class="subsection"><h3>Conclusion</h3><p id="p-44">We reported a robust rapid multiplex short-read MinION sequencing library protocol for ultra-fast aneuploidy detection for single and multiple samples. It shows promise for translation to clinical applications with a low assay cost and the fastest turnover time to date.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We sincerely thank Drs. Thomas Tuschl, Jan Vijg, Yousin Suh, Brynn Levy, Steven Josefowicz, Barry Coller and members of the Williams lab for their helpful input and advice with this project and manuscript. This research was supported by the National Institutes of Health Grant HD068546 and U19CA179564. IRB Protocol #: 2011-642, AAAR4452.</p><p id="p-46">Author contributions: Z.W. conceived of the project. S.W. performed the experiments and prepared figures. Z.W. and S.W developed the method and analyzed the data. Z.W., S.W., and Z.R.W. prepared the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-47"><em>Communicating editor: J. Kim</em></p></li><li class="fn" id="fn-3"><p id="p-48">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.118.200087/-/DC1</a>.</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-01-29"><span class="received-label">Received </span>January 29, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-03-06"><span class="accepted-label">Accepted </span>March 6, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Wei <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.1" data-doi="10.1136/bmj.e5908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brezina</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brezina</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kearns</span>  <span class="cit-name-given-names">W. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Preimplantation genetic testing.</span> <abbr class="cit-jnl-abbrev">BMJ</abbr> <span class="cit-vol">345</span>(<span class="cit-supplement">sep18 2</span>): <span class="cit-fpage">e5908</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1136/bmj.e5908</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMJ%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fbmj.e5908%26rft_id%253Dinfo%253Apmid%252F22990995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYm1qIjtzOjU6InJlc2lkIjtzOjE3OiIzNDUvc2VwMThfMi9lNTkwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNS8xNjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.2" data-doi="10.1016/j.yrtph.2015.09.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">M. Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elespuru</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Performance characteristics of the AmpliSeq Cancer Hotspot panel v2 in combination with the Ion Torrent Next Generation Sequencing Personal Genome Machine.</span> <abbr class="cit-jnl-abbrev">Regul. Toxicol. Pharmacol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.yrtph.2015.09.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRegul.%2BToxicol.%2BPharmacol.%26rft.volume%253D74%26rft.spage%253D178%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yrtph.2015.09.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.yrtph.2015.09.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.3" data-doi="10.12688/wellcomeopenres.11246.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hussain</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Robust long-read native DNA sequencing using the ONT CsgG Nanopore system.</span> <abbr class="cit-jnl-abbrev">Wellcome Open Res.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">23</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/wellcomeopenres.11246.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWellcome%2BOpen%2BRes.%26rft.volume%253D2%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.12688%252Fwellcomeopenres.11246.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/wellcomeopenres.11246.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.4" data-doi="10.1371/journal.pone.0092192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Performance comparison between rapid sequencing platforms for ultra-low coverage sequencing strategy.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e92192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0092192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De92192%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0092192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0092192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.5" data-doi="10.1038/gim.2015.199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Low-pass whole-genome sequencing in clinical cytogenetics: a validated approach.</span> <abbr class="cit-jnl-abbrev">Genet. Med.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">940</span>–<span class="cit-lpage">948</span> <span class="cit-comment">(erratum: Genet. Med. 19: 129)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/gim.2015.199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BMed.%26rft.volume%253D18%26rft.spage%253D940%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fgim.2015.199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/gim.2015.199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.6" data-doi="10.1016/j.rbmo.2011.09.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Handyside</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PGD and aneuploidy screening for 24 chromosomes by genome-wide SNP analysis: seeing the wood and the trees.</span> <abbr class="cit-jnl-abbrev">Reprod. Biomed. Online</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">686</span>–<span class="cit-lpage">691</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.rbmo.2011.09.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReproductive%2Bbiomedicine%2Bonline%26rft.stitle%253DReprod%2BBiomed%2BOnline%26rft.aulast%253DHandyside%26rft.auinit1%253DA.%2BH.%26rft.volume%253D23%26rft.issue%253D6%26rft.spage%253D686%26rft.epage%253D691%26rft.atitle%253DPGD%2Band%2Baneuploidy%2Bscreening%2Bfor%2B24%2Bchromosomes%2Bby%2Bgenome-wide%2BSNP%2Banalysis%253A%2Bseeing%2Bthe%2Bwood%2Band%2Bthe%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.rbmo.2011.09.012%26rft_id%253Dinfo%253Apmid%252F22033395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.rbmo.2011.09.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22033395&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303045500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.7" data-doi="10.1016/j.nantod.2012.12.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haque</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">X.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Solid-State and Biological Nanopore for Real-Time Sensing of Single Chemical and Sequencing of DNA.</span> <abbr class="cit-jnl-abbrev">Nano Today</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">56</span>–<span class="cit-lpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.nantod.2012.12.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNano%2BToday%26rft.volume%253D8%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.nantod.2012.12.008%26rft_id%253Dinfo%253Apmid%252F23504223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.nantod.2012.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23504223&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.8" data-doi="10.1038/nmeth.3290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiddes</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miga</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paten</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improved data analysis for the MinION nanopore sequencer.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">351</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3290%26rft_id%253Dinfo%253Apmid%252F25686389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25686389&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.9" data-doi="10.1186/s13059-016-1103-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paten</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akeson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">239</span> <span class="cit-comment">(erratum: Genome Biol. 13: 256)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-1103-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1103-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1103-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.10" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT–the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE2NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.11" data-doi="10.1093/dnares/dsv010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kukita</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matoba</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uchida</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamakawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">High-fidelity target sequencing of individual molecules identified using barcode sequences: De novo detection and absolute quantitation of mutations in plasma cell-free DNA from cancer patients.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">269</span>–<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsv010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsv010%26rft_id%253Dinfo%253Apmid%252F26126624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsv010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26126624&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-other" id="cit-8.5.1649.12"><div class="cit-metadata unstructured">Li, H., 2017 Minimap2: versatile pairwise alignment for nucleotide sequences.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.13" data-doi="10.1093/bioinformatics/btu555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Poretools: a toolkit for analyzing nanopore sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">23</span>): <span class="cit-fpage">3399</span>–<span class="cit-lpage">3401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu555</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu555%26rft_id%253Dinfo%253Apmid%252F25143291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25143291&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.14" data-doi="10.1038/nmeth.3327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Successful test launch for nanopore sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">303</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3327</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3327%26rft_id%253Dinfo%253Apmid%252F25825834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25825834&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.15" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads.</span> <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.16" data-doi="10.1186/1471-2164-13-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQuail%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DA%2Btale%2Bof%2Bthree%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%253A%2Bcomparison%2Bof%2BIon%2BTorrent%252C%2BPacific%2BBiosciences%2Band%2BIllumina%2BMiSeq%2Bsequencers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-341%26rft_id%253Dinfo%253Apmid%252F22827831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827831&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.17" data-doi="10.1056/NEJMoa1201569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saade</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorsten</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Karyotype <em>vs.</em> microarray testing for genetic abnormalities after stillbirth.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">367</span>(<span class="cit-issue">23</span>): <span class="cit-fpage">2185</span>–<span class="cit-lpage">2193</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1056/NEJMoa1201569</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D367%26rft.spage%253D2185%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1201569%26rft_id%253Dinfo%253Apmid%252F23215556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1201569&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23215556&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311890600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1649.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Team</span>  <span class="cit-name-given-names">R. D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>R: A language and environment for statistical computing. R Foundation for Statistical Computing</em>, <em>Vienna</em>, <em>Austria</em></span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>, <a href="http://www.R-project.org">http://www.R-project.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.19" data-doi="10.3389/fgene.2014.00449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The evolution of nanopore sequencing.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00449</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D449%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00449%26rft_id%253Dinfo%253Apmid%252F25610451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25610451&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.20" data-doi="10.1056/NEJMoa1203382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wapner</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballif</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosomal microarray <em>vs.</em> karyotyping for prenatal diagnosis.</span> <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">367</span>(<span class="cit-issue">23</span>): <span class="cit-fpage">2175</span>–<span class="cit-lpage">2184</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1056/NEJMoa1203382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D367%26rft.spage%253D2175%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1203382%26rft_id%253Dinfo%253Apmid%252F23215555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1203382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23215555&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1649.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311890600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.21" data-doi="10.1534/genetics.115.182311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Rapid Short-Read Sequencing and Aneuploidy Detection Using MinION Nanopore Technology.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">37</span>–<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182311%26rft_id%253Dinfo%253Apmid%252F26500254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAyLzEvMzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTY0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1649.22" data-doi="10.1186/s13039-015-0143-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Application of next-generation sequencing for 24-chromosome aneuploidy screening of human preimplantation embryos.</span> <abbr class="cit-jnl-abbrev">Mol. Cytogenet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13039-015-0143-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCytogenet.%26rft.volume%253D8%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13039-015-0143-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13039-015-0143-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/5/1649.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1637" title="A Host-Pathogen Interaction Screen Identifies ada2 as a Mediator of Candida glabrata Defenses Against Reactive Oxygen Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1659" title="Functional Validation of Candidate Genes Detected by Genomic Feature Models" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/5"><span>Volume 8 Issue 5, May 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/5.cover-source.jpg?itok=Bl7MXdvV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1649" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-267Kv62-9CvHRxQrmEuIlheGFW4YOJvx1sVf0k_G1Ds" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/5/1649" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/5/1649" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/5/1649" class="active" data-icon-position="" data-hide-link-title="0">Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489470" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QulCtNwg8OBDlWRr7dlsKCBbmxYaQfqYPH2_WCF6JXU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1649.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/5/1649" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Rn47-CiwGhtLYIX1KwqtCC1mlK_apS1tM2w1Ecnp_IA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/5/1649" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1649.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489470" id="node489470--2" data-pisa="ggg;8/5/1649" data-pisa-master="ggg;g3.118.200087" data-apath="/ggg/8/5/1649.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shan Wei</span>, <span class="highwire-citation-author" data-delta="1">Zachary R. Weiss</span> and <span class="highwire-citation-author" data-delta="2">Zev Williams</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1649-1657;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200087 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489470--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shan Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shan%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShan%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zw2421@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">zw2421@cumc.columbia.edu</a>
<a href="mailto:sw3203@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">sw3203@cumc.columbia.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zachary R. Weiss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BR.%2BWeiss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weiss%20ZR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZachary%2BR.%2BWeiss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zev Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zev%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZev%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zw2421@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">zw2421@cumc.columbia.edu</a>
<a href="mailto:sw3203@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">sw3203@cumc.columbia.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/5/1649" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/5/1649&amp;title=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/5/1649&amp;title=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/5/1649&amp;title=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/5/1649&amp;text=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/5/1649&amp;title=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/5/1649&amp;t=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/5/1649&amp;title=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/5/1649&amp;title=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489470/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489470" id="node489470--3" data-pisa="ggg;8/5/1649" data-pisa-master="ggg;g3.118.200087" data-apath="/ggg/8/5/1649.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Multiplex Small DNA Sequencing on the MinION Nanopore Sequencing Platform</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shan Wei</span>, <span class="highwire-citation-author" data-delta="1">Zachary R. Weiss</span> and <span class="highwire-citation-author" data-delta="2">Zev Williams</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1649-1657;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200087 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489470--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shan Wei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shan%2BWei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wei%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShan%2BWei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zw2421@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">zw2421@cumc.columbia.edu</a>
<a href="mailto:sw3203@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">sw3203@cumc.columbia.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zachary R. Weiss </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BR.%2BWeiss%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weiss%20ZR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZachary%2BR.%2BWeiss%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zev Williams </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Obstetrics and Gynecology, Columbia University Medical Center, New York, NY 10032</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zev%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williams%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZev%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zw2421@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">zw2421@cumc.columbia.edu</a>
<a href="mailto:sw3203@cumc.columbia.edu" class="" data-icon-position="" data-hide-link-title="0">sw3203@cumc.columbia.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1649&amp;rft.epage=1657&amp;rft.atitle=Rapid%20Multiplex%20Small%20DNA%20Sequencing%20on%20the%20MinION%20Nanopore%20Sequencing%20Platform&amp;rft.volume=8&amp;rft.issue=5&amp;rft.date=2018-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wei%2C+Shan&amp;rft.au=Weiss%2C+Zachary+R.&amp;rft.au=Williams%2C+Zev"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489470/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489470/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489470/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489470/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489470/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489470/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489470/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489470/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489470/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489470/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489470/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489470/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/5/1649" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/5/1649.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/5/1649.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/5/1649.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/5/1649.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
