<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/5/1755" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/5/1755.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/5/1755.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/5/1755.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/5/1755.atom" />
<meta name="HW.pisa" content="ggg;8/5/1755" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations" />
<meta name="DC.Identifier" content="10.1534/g3.117.300512" />
<meta name="DC.Date" content="2018-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Tomotaka Matsumoto, Hiroshi Akashi. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Inferred ancestral nucleotide states are increasingly employed in analyses of within- and between -species genome variation. Although numerous studies have focused on ancestral inference among distantly related lineages, approaches to infer ancestral states in polymorphism data have received less attention. Recently developed approaches that employ complex transition matrices allow us to infer ancestral nucleotide sequence in various evolutionary scenarios of base composition. However, the requirement of a single gene tree to calculate a likelihood is an important limitation for conducting ancestral inference using within-species variation in recombining genomes. To resolve this problem, and to extend the applicability of ancestral inference in studies of base composition evolution, we first evaluate three previously proposed methods to infer ancestral nucleotide sequences among within- and between-species sequence variation data. The methods employ a single allele, bifurcating tree, or a star tree for within-species variation data. Using simulated nucleotide sequences, we employ ancestral inference to infer fixations and polymorphisms. We find that all three methods show biased inference. We modify the bifurcating tree method to include weights to adjust for an expected site frequency spectrum, “bifurcating tree with weighting” (BTW). Our simulation analysis show that the BTW method can substantially improve the reliability and robustness of ancestral inference in a range of scenarios that include non-neutral and/or non-stationary base composition evolution." />
<meta name="DC.Contributor" content="Tomotaka Matsumoto" />
<meta name="DC.Contributor" content="Hiroshi Akashi" />
<meta name="article:published_time" content="2018-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Inferred ancestral nucleotide states are increasingly employed in analyses of within- and between -species genome variation. Although numerous studies have focused on ancestral inference among distantly related lineages, approaches to infer ancestral states in polymorphism data have received less attention. Recently developed approaches that employ complex transition matrices allow us to infer ancestral nucleotide sequence in various evolutionary scenarios of base composition. However, the requirement of a single gene tree to calculate a likelihood is an important limitation for conducting ancestral inference using within-species variation in recombining genomes. To resolve this problem, and to extend the applicability of ancestral inference in studies of base composition evolution, we first evaluate three previously proposed methods to infer ancestral nucleotide sequences among within- and between-species sequence variation data. The methods employ a single allele, bifurcating tree, or a star tree for within-species variation data. Using simulated nucleotide sequences, we employ ancestral inference to infer fixations and polymorphisms. We find that all three methods show biased inference. We modify the bifurcating tree method to include weights to adjust for an expected site frequency spectrum, “bifurcating tree with weighting” (BTW). Our simulation analysis show that the BTW method can substantially improve the reliability and robustness of ancestral inference in a range of scenarios that include non-neutral and/or non-stationary base composition evolution.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/05/01" />
<meta name="citation_mjid" content="ggg;8/5/1755" />
<meta name="citation_id" content="8/5/1755" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/5/1755" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/5/1755.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/5/1755.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/5/1755.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300512" />
<meta name="citation_pmid" content="29588382" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1755" />
<meta name="citation_lastpage" content="1769" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Tomotaka Matsumoto" />
<meta name="citation_author_institution" content="Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan" />
<meta name="citation_author_institution" content="Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3456-7553" />
<meta name="citation_author" content="Hiroshi Akashi" />
<meta name="citation_author_institution" content="Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan" />
<meta name="citation_author_institution" content="Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan" />
<meta name="citation_author_email" content="hiakashi@nig.ac.jp" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5325-6999" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Inferring weak selection from patterns of polymorphism and divergence at &quot;silent&quot; sites in Drosophila DNA.;citation_pages=1067-1076;citation_volume=139;citation_year=1995;citation_issue=2;citation_pmid=7713409" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Molecular Evolution Between Drosophila melanogaster and D. simulans Reduced Codon Bias, Faster Rates of Amino Acid Substitution, and Larger Proteins in D. melanogaster;citation_pages=1297-1307;citation_volume=144;citation_year=1996;citation_issue=3;citation_pmid=8913769" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=H. Akashi;citation_title=Within- and between-species DNA sequence variation and the 'footprint' of natural selection.;citation_pages=39-51;citation_volume=238;citation_year=1999;citation_issue=1;citation_pmid=10570982;citation_doi=10.1016/S0378-1119(99)00294-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_author=S. W. Schaeffer;citation_title=Natural Selection and the Frequency Distributions of &quot;Silent&quot; DNA Polymorphism in Drosophila;citation_pages=295-307;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9136019" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Akashi;citation_author=P. Goel;citation_author=A. John;citation_title=Ancestral inference and the study of codon bias evolution: implications for molecular evolutionary analyses of the Drosophila melanogaster subgroup.;citation_pages=e1065-e1065;citation_volume=2;citation_year=2007;citation_issue=10;citation_pmid=17957249;citation_doi=10.1371/journal.pone.0001065" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Akashi;citation_author=W.-Y. Ko;citation_author=S. Piao;citation_author=A. John;citation_author=P. Goel;citation_title=Molecular Evolution in the Drosophila melanogaster Species Subgroup: Frequent Parameter Fluctuations on the Timescale of Molecular Divergence.;citation_volume=172;citation_year=2006;citation_issue=3;citation_pmid=16387879;citation_doi=10.1534/genetics.105.049676" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=F. Alvarez-Valin;citation_author=O. Clay;citation_author=S. Cruveiller;citation_author=G. Bernardi;citation_title=Inaccurate reconstruction of ancestral GC levels creates a &amp;quot;vanishing isochores&amp;quot; effect.;citation_pages=788-793;citation_volume=31;citation_year=2004;citation_issue=2;citation_pmid=15062812;citation_doi=10.1016/j.ympev.2004.01.016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Aoki;citation_author=M. Ito;citation_author=W. Iwasaki;citation_title=From β- to α-Proteobacteria: The Origin and Evolution of Rhizobial Nodulation Genes nodIJ.;citation_volume=30;citation_year=2013;citation_issue=11;citation_pmid=24030554;citation_doi=10.1093/molbev/mst153" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=D. J. Begun;citation_title=The Frequency Distribution of Nucleotide Variation in Drosophila simulans;citation_pages=1343-1352;citation_volume=18;citation_year=2001;citation_issue=7;citation_pmid=11420372;citation_doi=10.1093/oxfordjournals.molbev.a003918" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Bolívar;citation_author=C. F. Mugal;citation_author=A. Nater;citation_author=H. Ellegren;citation_title=Recombination Rate Variation Modulates Gene Sequence Evolution Mainly via GC-Biased Gene Conversion, Not Hill–Robertson Interference, in an Avian System.;citation_volume=33;citation_year=2015;citation_issue=1;citation_pmid=26446902;citation_doi=10.1093/molbev/msv214" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=F. Clemente;citation_author=C. Vogl;citation_title=Evidence for complex selection on four-fold degenerate sites in Drosophila melanogaster.;citation_pages=2582-2595;citation_volume=25;citation_year=2012;citation_issue=12;citation_pmid=23020078;citation_doi=10.1111/jeb.12003" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Clément;citation_author=G. Sarah;citation_author=Y. Holtz;citation_author=F. Homa;citation_author=S. Pointet;citation_title=Evolutionary forces affecting synonymous variations in plant genomes.;citation_pages=e1006799;citation_volume=13;citation_year=2017;citation_issue=5;citation_doi=10.1371/journal.pgen.1006799" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=T. M. Collins;citation_author=P. H. Wimberger;citation_author=G. J. P. Naylor;citation_title=Compositional bias, character-state bias, and character-state reconstruction using parsimony.;citation_volume=43;citation_year=1994;citation_issue=4;citation_doi=10.1093/sysbio/43.4.482" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Duret;citation_author=M. Semon;citation_author=G. Piganeau;citation_author=D. Mouchiroud;citation_author=N. Galtier;citation_title=Vanishing GC-Rich Isochores in Mammalian Genomes;citation_pages=1837-1847;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524353" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=A. Eyre-Walker;citation_title=Problems with parsimony in sequences of biased base composition.;citation_pages=686-690;citation_volume=47;citation_year=1998;citation_issue=6;citation_pmid=9847410;citation_doi=10.1007/PL00006427" />
<meta name="citation_reference" content="citation_title=The Genetical Theory of Natural Selection;citation_author=R. A. Fisher;citation_year=1930;citation_doi=10.5962/bhl.title.27468" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Galtier;citation_author=E. Bazin;citation_author=N. Bierne;citation_title=GC-biased segregation of noncoding polymorphisms in Drosophila.;citation_volume=172;citation_year=2006;citation_issue=1;citation_pmid=16157668;citation_doi=10.1534/genetics.105.046524" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=N. Galtier;citation_author=P. Boursot;citation_title=A new method for locating changes in a tree reveals distinct nucleotide polymorphism vs. divergence patterns in mouse mitochondrial control region.;citation_pages=224-231;citation_volume=50;citation_year=2000;citation_issue=3;citation_pmid=10754064;citation_doi=10.1007/s002399910025" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. A. Gaucher;citation_author=S. Govindarajan;citation_author=O. K. Ganesh;citation_title=Palaeotemperature trend for Precambrian life inferred from resurrected proteins;citation_pages=704-707;citation_volume=451;citation_year=2008;citation_issue=7179;citation_pmid=18256669;citation_doi=10.1038/nature06510;citation_georef=2011015744" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. A. Gaucher;citation_author=J. M. Thomson;citation_author=M. F. Burgan;citation_author=S. A. Benner;citation_title=Inferring the palaeoenvironment of ancient bacteria on the basis of resurrected proteins.;citation_pages=285-288;citation_volume=425;citation_year=2003;citation_issue=6955;citation_pmid=13679914;citation_doi=10.1038/nature01977" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Glémin;citation_author=P. F. Arndt;citation_author=P. W. Messer;citation_author=D. Petrov;citation_author=N. Galtier;citation_title=Quantification of GC-biased gene conversion in the human genome.;citation_volume=25;citation_year=2015;citation_issue=8;citation_pmid=25995268;citation_doi=10.1101/gr.185488.114" />
<meta name="citation_reference" content="citation_journal_title=Stoch. Models;citation_author=R. C. Griffiths;citation_author=S. Tavare;citation_title=The age of a mutation in a general coalescent tree.;citation_pages=273-295;citation_volume=14;citation_year=1998;citation_issue=1-2;citation_doi=10.1080/15326349808807471" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Groussin;citation_author=M. Gouy;citation_title=Adaptation to Environmental Temperature Is a Major Determinant of Molecular Evolutionary Rates in Archaea.;citation_volume=28;citation_year=2011;citation_issue=9;citation_pmid=21498602;citation_doi=10.1093/molbev/msr098" />
<meta name="citation_reference" content="citation_journal_title=Biology Letters;citation_journal_abbrev=Biol Lett;citation_author=P. R. Haddrill;citation_author=B. Charlesworth;citation_title=Non-neutral processes drive the nucleotide composition of non-coding sequences in Drosophila;citation_pages=438-441;citation_volume=4;citation_year=2008;citation_issue=4;citation_pmid=18505714;citation_doi=10.1098/rsbl.2008.0174" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=M. Hasegawa;citation_author=H. Kishino;citation_author=T. Yano;citation_title=Dating of the human-ape splitting by a molecular clock of mitochondrial DNA.;citation_pages=160-174;citation_volume=22;citation_year=1985;citation_issue=2;citation_pmid=3934395;citation_doi=10.1007/BF02101694" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Context Dependence, Ancestral Misidentification, and Spurious Signatures of Natural Selection.;citation_volume=24;citation_year=2007;citation_issue=8;citation_pmid=17545186;citation_doi=10.1093/molbev/msm108" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=J. P. Huelsenbeck;citation_author=J. P. Bollback;citation_title=Empirical and hierarchical Bayesian estimation of ancestral states.;citation_volume=50;citation_year=2001;citation_issue=3;citation_pmid=12116580;citation_doi=10.1080/10635150119871" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=B. C. Jackson;citation_author=J. L. Campos;citation_author=P. R. Haddrill;citation_author=B. Charlesworth;citation_author=K. Zeng;citation_title=Variation in the intensity of selection on codon bias over time causes contrasting patterns of base composition evolution in Drosophila.;citation_volume=9;citation_year=2017;citation_pmid=28082609;citation_doi=10.1093/gbe/evw291" />
<meta name="citation_reference" content="citation_title=Mammalian Protein Metabolism;citation_title=Evolution of protein mole- cules;citation_author=T. H. Jukes;citation_author=C. R. Cantor;citation_pages=21-123;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. D. Keightley;citation_author=J. L. Campos;citation_author=T. R. Booker;citation_author=B. Charlesworth;citation_title=Inferring the Frequency Spectrum of Derived Variants to Quantify Adaptive Molecular Evolution in Protein-Coding Genes of Drosophila melanogaster.;citation_volume=203;citation_year=2016;citation_issue=2;citation_pmid=27098912;citation_doi=10.1534/genetics.116.188102" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. D. Kern;citation_author=D. J. Begun;citation_title=Patterns of Polymorphism and Divergence from Noncoding Sequences of Drosophila melanogaster and D. simulans: Evidence for Nonequilibrium Processes.;citation_volume=22;citation_year=2005;citation_issue=1;citation_pmid=15456897;citation_doi=10.1093/molbev/msh269" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=R. M. Kliman;citation_title=Recent selection on synonymous codon usage in Drosophila.;citation_pages=343-351;citation_volume=49;citation_year=1999;citation_issue=3;citation_pmid=10473775;citation_doi=10.1007/PL00006557" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=W. Y. Ko;citation_author=R. M. David;citation_author=H. Akashi;citation_title=Molecular phylogeny of the Drosophila melanogaster species subgroup.;citation_pages=562-573;citation_volume=57;citation_year=2003;citation_issue=5;citation_pmid=14738315;citation_doi=10.1007/s00239-003-2510-x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=J. M. Koshi;citation_author=R. A. Goldstein;citation_title=Probabilistic reconstruction of ancestral protein sequences.;citation_pages=313-320;citation_volume=42;citation_year=1996;citation_issue=2;citation_pmid=8919883;citation_doi=10.1007/BF02198858" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=J. Lachance;citation_author=S. A. Tishkoff;citation_title=Biased Gene Conversion Skews Allele Frequencies in Human Populations, Increasing the Disease Burden of Recessive Alleles.;citation_pages=408-420;citation_volume=95;citation_year=2014;citation_issue=4;citation_pmid=25279983;citation_doi=10.1016/j.ajhg.2014.09.008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Lartillot;citation_title=Interaction between selection and biased gene conversion in Mammalian Protein-Coding Sequence Evolution Revealed by a Phylogenetic Covariance Analysis.;citation_volume=30;citation_year=2012;citation_issue=2;citation_pmid=23024185;citation_doi=10.1093/molbev/mss231" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=X. Maside;citation_author=A. W. Lee;citation_author=B. Charlesworth;citation_title=Selection on codon usage in Drosophila americana.;citation_pages=150-154;citation_volume=14;citation_year=2004;citation_issue=2;citation_pmid=14738738;citation_doi=10.1016/j.cub.2003.12.055" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Matsumoto;citation_author=H. Akashi;citation_author=Z. Yang;citation_title=Evaluation of Ancestral Sequence Reconstruction Methods to Infer Nonstationary Patterns of Nucleotide Substitution.;citation_volume=200;citation_year=2015;citation_issue=3;citation_pmid=25948563;citation_doi=10.1534/genetics.115.177386" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Matsumoto;citation_author=A. John;citation_author=P. Baeza-Centurion;citation_author=B. Li;citation_author=H. Akashi;citation_title=Codon Usage Selection Can Bias Estimation of the Fraction of Adaptive Amino Acid Fixations.;citation_volume=33;citation_year=2016;citation_issue=6;citation_pmid=26873577;citation_doi=10.1093/molbev/msw027" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. H. McDonald;citation_author=M. Kreitman;citation_title=Adaptive protein evolution at the Adh locus in Drosophila.;citation_pages=652-654;citation_volume=351;citation_year=1991;citation_issue=6328;citation_pmid=1904993;citation_doi=10.1038/351652a0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Muyle;citation_author=L. Serres-Giardi;citation_author=A. Ressayre;citation_author=J. Escobar;citation_author=S. Glémin;citation_title=GC-Biased Gene Conversion and Selection Affect GC Content in the Oryza Genus (rice).;citation_volume=28;citation_year=2011;citation_issue=9;citation_pmid=21504892;citation_doi=10.1093/molbev/msr104" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. Nagylaki;citation_title=Evolution of a finite population under gene conversion.;citation_pages=6278-6281;citation_volume=80;citation_year=1983;citation_issue=20;citation_pmid=6578508;citation_doi=10.1073/pnas.80.20.6278" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Nielsen;citation_author=V. L. Bauer DuMont;citation_author=M. J. Hubisz;citation_author=C. F. Aquadro;citation_title=Maximum Likelihood Estimation of Ancestral Codon Usage Bias Parameters in Drosophila.;citation_volume=24;citation_year=2006;citation_issue=1;citation_pmid=17041152;citation_doi=10.1093/molbev/msl146" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=M. Pagel;citation_author=A. Meade;citation_author=D. Barker;citation_author=J. Thorne;citation_title=Bayesian Estimation of Ancestral Character States on Phylogenies.;citation_volume=53;citation_year=2004;citation_issue=5;citation_pmid=15545248;citation_doi=10.1080/10635150490522232" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. T. Perna;citation_author=T. D. Kocher;citation_title=Unequal Base Frequencies and the Estimation of Substitution Rates;citation_pages=359-361;citation_volume=12;citation_year=1995;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y.-P. Poh;citation_author=C.-T. Ting;citation_author=H.-W. Fu;citation_author=C. H. Langley;citation_author=D. J. Begun;citation_title=Population Genomic Analysis of Base Composition Evolution in Drosophila melanogaster.;citation_volume=4;citation_year=2012;citation_issue=12;citation_pmid=23160062;citation_doi=10.1093/gbe/evs097" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Evol.;citation_author=J. R. Powell;citation_author=E. Sezzi;citation_author=E. N. Moriyama;citation_author=J. M. Gleason;citation_author=A. Caccone;citation_title=Analysis of a Shift in Codon Usage in Drosophila.;citation_pages=S214-S225;citation_volume=57;citation_year=2003;citation_issue=0;citation_pmid=15008418;citation_doi=10.1007/s00239-003-0030-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. A. Sawyer;citation_author=D. E. Dykhuizen;citation_author=R. F. DuBose;citation_author=L. Green;citation_author=T. Mutangadura-Mhlanga;citation_title=Distribution and Abundance of Insertion Sequences Among Natural Isolates of Escherichia coli;citation_pages=51-63;citation_volume=115;citation_year=1987;citation_issue=1;citation_pmid=3030884" />
<meta name="citation_reference" content="citation_journal_title=Adv. Bioinforma.;citation_author=P. Sethupathy;citation_author=S. Hannenhalli;citation_title=A Tutorial of the Poisson Random Field Model in Population Genetics.;citation_pages=257864;citation_volume=2008;citation_year=2008;citation_doi=10.1155/2008/257864" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. D. Singh;citation_author=P. F. Arndt;citation_author=A. G. Clark;citation_author=C. F. Aquadro;citation_title=Strong Evidence for Lineage and Sequence Specificity of Substitution Rates and Patterns in Drosophila.;citation_volume=26;citation_year=2009;citation_issue=7;citation_pmid=19351792;citation_doi=10.1093/molbev/msp071" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Takano-Shimizu;citation_title=Local Changes in GC/AT Substitution Biases and in Crossover Frequencies on Drosophila Chromosomes;citation_pages=606-619;citation_volume=18;citation_year=2001;citation_issue=4;citation_pmid=11264413;citation_doi=10.1093/oxfordjournals.molbev.a003841" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. V. Terekhanova;citation_author=G. A. Bazykin;citation_author=A. Neverov;citation_author=A. S. Kondrashov;citation_author=V. B. Seplyarskiy;citation_title=Prevalence of Multinucleotide Replacements in Evolution of Primates and Drosophila.;citation_volume=30;citation_year=2013;citation_issue=6;citation_pmid=23447710;citation_doi=10.1093/molbev/mst036" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_author=H. A. Guess;citation_title=Is the most frequent allele the oldest?;citation_pages=141-160;citation_volume=11;citation_year=1977;citation_issue=2;citation_pmid=867285;citation_doi=10.1016/0040-5809(77)90023-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=C. C. Weber;citation_author=B. Boussau;citation_author=J. Romiguier;citation_author=E. D. Jarvis;citation_author=H. Ellegren;citation_title=Evidence for GC-biased gene conversion as a driver of between-lineage differences in avian base composition.;citation_pages=549;citation_volume=15;citation_year=2014;citation_issue=12;citation_pmid=25496599;citation_doi=10.1186/s13059-014-0549-1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. J. Wilson;citation_author=R. D. Hernandez;citation_author=P. Andolfatto;citation_author=M. Przeworski;citation_title=A population genetics-phylogenetics approach to inferring natural selection in coding sequences.;citation_pages=e1002395-e1002395;citation_volume=7;citation_year=2011;citation_issue=12;citation_pmid=22144911;citation_doi=10.1371/journal.pgen.1002395" />
<meta name="citation_reference" content="citation_title=Evolution and the Genetics of Populations. Vol. 2 The Theory of Gene Frequencies;citation_author=S. Wright;citation_year=1969" />
<meta name="citation_reference" content="citation_title=Computational Molecular Evolution;citation_author=Z. Yang;citation_year=2006;citation_doi=10.1093/acprof:oso/9780198567028.001.0001" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: Phylogenetic Analysis by Maximum Likelihood.;citation_volume=24;citation_year=2007;citation_issue=8;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Yang;citation_author=S. Kumar;citation_author=M. Nei;citation_title=A new method of inference of ancestral nucleotide and amino acid sequences.;citation_pages=1641-1650;citation_volume=141;citation_year=1995;citation_issue=4;citation_pmid=8601501" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="twitter:description" content="Inferred ancestral nucleotide states are increasingly employed in analyses of within- and between -species genome variation. Although numerous studies have focused on ancestral inference among distantly related lineages, approaches to infer ancestral states in polymorphism data have received less attention. Recently developed approaches that employ complex transition matrices allow us to infer ancestral nucleotide sequence in various evolutionary scenarios of base composition. However, the requirement of a single gene tree to calculate a likelihood is an important limitation for conducting ancestral inference using within-species variation in recombining genomes. To resolve this problem, and to extend the applicability of ancestral inference in studies of base composition evolution, we first evaluate three previously proposed methods to infer ancestral nucleotide sequences among within- and between-species sequence variation data. The methods employ a single allele, bifurcating tree, or a star tree for within-species variation data. Using simulated nucleotide sequences, we employ ancestral inference to infer fixations and polymorphisms. We find that all three methods show biased inference. We modify the bifurcating tree method to include weights to adjust for an expected site frequency spectrum, “bifurcating tree with weighting” (BTW). Our simulation analysis show that the BTW method can substantially improve the reliability and robustness of ancestral inference in a range of scenarios that include non-neutral and/or non-stationary base composition evolution." />
<meta name="og-title" property="og:title" content="Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/5/1755" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Inferred ancestral nucleotide states are increasingly employed in analyses of within- and between -species genome variation. Although numerous studies have focused on ancestral inference among distantly related lineages, approaches to infer ancestral states in polymorphism data have received less attention. Recently developed approaches that employ complex transition matrices allow us to infer ancestral nucleotide sequence in various evolutionary scenarios of base composition. However, the requirement of a single gene tree to calculate a likelihood is an important limitation for conducting ancestral inference using within-species variation in recombining genomes. To resolve this problem, and to extend the applicability of ancestral inference in studies of base composition evolution, we first evaluate three previously proposed methods to infer ancestral nucleotide sequences among within- and between-species sequence variation data. The methods employ a single allele, bifurcating tree, or a star tree for within-species variation data. Using simulated nucleotide sequences, we employ ancestral inference to infer fixations and polymorphisms. We find that all three methods show biased inference. We modify the bifurcating tree method to include weights to adjust for an expected site frequency spectrum, “bifurcating tree with weighting” (BTW). Our simulation analysis show that the BTW method can substantially improve the reliability and robustness of ancestral inference in a range of scenarios that include non-neutral and/or non-stationary base composition evolution." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
    <title>Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__s8QOcygh8E98DF_LbAuPN6EyCkdDabfVnTBMiJ7Uybw__NJGM_0BsNSHNDWobhqGJ98XG4UkC62DVr_Lbb2Mjwf4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"95iklO2bejldUjkyWxgcT1PrYZKujZm9z9xK_gQNkHE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489477","apath":"\/ggg\/8\/5\/1755.atom","pisa":"ggg;8\/5\/1755","ac":{"\/ggg\/8\/5\/1755.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/5\/1755.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1755"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1755"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/5\/1755.figures-only":true,"\/content\/8\/5\/1755":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vTrYxoRh4prIzNs2INWrruNdlK023Kve88O4twPZ3hezXvAZ6tZiOJKp0fngbBD5QIaGf+5\\\\\\\/e8dH7IZqwsoGnVBMSLOXpuM1fyYUf9n8fhLi6y9J\\\\\\\/2DlpVwfzmNAf3epZYsnEd9KhQC1YXBx2c3IgBzzcpG2ljFRyhLx6EoSpP7Xt9s2ZYSQhvfvSNV43zdbzPABV1aoawZRfP42VGxg26Wa1n2VKtkGnBxJw1CUvU4v64\\\\\\\/J+6DaUkzbI2H7bu+gN5h79cTmjsORitrt7\\\\\\\/VjBIofssZdpQNrZUFBCS\\\\\\\/QPv0DWZnlqwUFVcczQgGpIb\\\\\\\/u7CI65O8tw3wCEM8vO1oqTS6+0j0RBXjHmBC7dZ7ZZSy7BN5O0VMjoLjYJ\\\\\\\/WU8cs5RvF4VeCK1Dpp1KCchrH2w7f7Egr\\\\\\\/hsatkK4QGfbz\\\\\\\/c0YQylq5OPdmW6euew90G7nmQxM5fhlw740IKOub6h8N8Pc6kqiJTSu+HvGpHUpTepWxrQIdXrIRnFQBXokbqZDvBEJUmVOimFm76gxrnR5QNlSONQB9Kz2B1omh6Dg9+s3d13yzyM2JPXO0AvgW9eUlyK70fkfHqvL1rsRaQOzBSe6+5jgwtkVhrN8Ak55d8tik\\\\\\\/4D7PlqbzcE\\\\\\\/lEhl6fb96JYYuaheXn4d5SgMNTDkHPDGLkyW2C7+enzClt9Mny+RIA1ZFr8S0NuTr69B4RfqRP7Vs0QO5Qitc+3pM\\\\\\\/52Mg9BdmxNbnD5eYc2aMOzosm0ZpD7Zpjlvwy7\\\\\\\/Xva7oMUfM7nxg28Pu1OKu0q8cXNzs3LvYcW7YCY0MNBwPoHbmGSA+Q17vGtth2CK9I1TT1XIPfPFFJI6JX6uD3Soqn\\\\\\\/mTdtJvgyUaYDauGTS0BR\\\\\\\/IfbH9gQ7OZF4D5bI\\\\\\\/d76j3Lgw7MylLseg9nhvptA1AEjD85Is9UC+gSrPocLIkTVw0GcaGxLLM2Tk4Z5bnwTdmCKgsD3puNUmVQ+ai\\\\\\\/4hXGxJfxGEDbt+GbdtigRPnGyAXygYp2N8x44TOePZctk23m4868+waZyl9sOqkXA2JNZpKtIYMrDULmHGoqWtQA6+6EVPRH3XwsgH2lnfKn\\\\\\\/5Ox2jbF55UeL2TY0Pr0g+ml1USIDG6X2gt+NqIPIzVPTKw0OPfgi+xPvs9jb0\\\\\\\/AsDGFnSn4f0lsrfyvt5R2NsIuvHKrd01wkt5plEZQPPm1Y41ifXZk\\\\\\\/NkjJu\\\\\\\/RQdWwOyYCTw35djcNoHLqH68jgGjOxqbDMGQul89gVUUns9D0R+sEDPSeXw\\\\\\\/cM6iboCiDWKd\\\u0022,\\\u0022iv\\\u0022:\\\u0022S6MAiz2Eie2+dIw4jmEhhQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002237a0ecdda97831db6603efde0c56153406b292c8fc1645bef6e7a4210d077fa2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227AfPUHqwVt6RN2KayR8IfubXR7GMRXVI0HuNxZgGPBbY4eTPo7KTNHgRRSpieuIjUu56eALgeSIHhcTeK0gLwKWNmLHztTYXM8Xjir2LTM1iXy7d14N+EiG7\\\\\\\/1XDmCqt4UDz8ihTNL79bl5mN8bLNTvIk89z6oSO7Iwn9dkWuHAHBu2I12byCvmkLQOmQiREMJo\\\\\\\/mP1TdT2fRhPiw7rxVaOVNcevCihBrzeVWF8VtSYx31m+cab1MoXY2B0CpD6cU8cKfXxUxhr3\\\\\\\/f\\\\\\\/nDlll8UNhbqliXZ5UrtTPbjTLAcv5jiocCSc9Q5tornpFdju2BeeIbNrSSu2gQmh2MYkr9njd75eaxMtCRHsdtv\\\\\\\/dX86ZOEelrZJcEWvlYtYsW3GtYZjgRfOR9xCTnWnE+h4WhYB3tpJG4bkEXR68sZK3mFTfFrObCVnddWVHWrbmqBQz6pDUw9SxwFylb14hyM2s2aiaf5j\\\\\\\/bmYHC7aERxPLl\\\\\\\/lFehljhrSGiBXIDwYL90\\\\\\\/FCn+EP+M3JkbENESbew2Oadfg2AMiz1jhpKEUHXp\\\\\\\/uYLHEJXKDozT1dAzBtTVv3WFfDZk1BCS5+7pkkqthU\\\\\\\/VreI5eHZoXe5rn+Kfmez\\\\\\\/eOLUBaJhSjGJSEKfTiqlHpkwiciB2LvlZyyU4CbDdFn0Zee60Mdx4qnMwkAVtjXkX+2VRrnFeWiqHhj14Iqwu\\\\\\\/yLcYWHw\\\\\\\/bfFalw0GGMfDCEeYgfxqsYyqaDdG8\\\\\\\/hC+FPoeTEmj4RF3B3ZOU+NwUQiUu1lpDCEqacpFYyuG4wR3101YANHsO\\\\\\\/ryzQLH7nbhtiy0Rty13WE\\\\\\\/ygIfv0qLbHq67wMfZQj5fKZeVhtUdK8qEezLfkdR6y1xa4c+AHWjmtcDPn8VMJzLrmCx47X\\\\\\\/O\\\\\\\/210R026geaxmIMJHE98gqEPAdK88AA9zIhjlUHLBY+72xUsEVS8onzYQM5QkeKLFVnX1gAX9uVuev7rKHMZje7gC\\\\\\\/4ormG9KAVvC5Hqp9RzXPFe9lifsr4+944PckLmxv+F+1ARIOn4J4Swo3U6jixFCj0P60UWMD01oWABEwabNf0Cjm+gUyzTslRaAAJ2OL2pYoLReAHZLh+zjnpD+5QIRXd9u3y80lougteQwWucklwtw0D298aEd2jagyJ2PedSAeNQY5eTCdP1ZJDUmZx+mpZsWV8G987uLRbODTNkY\\\\\\\/wepOj4KzDIk8alaTsIhKeiT4YbU3K3IZev1BKjYY8HqX+iY9h2zF1Fp7Escx63olwfXakiPyKk6Ru0OJ2mCUNmyiOmGjwDcvov2qRp9odIICzixp2RZnHNcKIan+ZpSpS5DYDI6HgHPhpeoxdY4SDwl9jPvSS\\\\\\\/jP73k8sp7QxOUY7GQQA+5wYlXCnezmJkfHQ7WyUXoHr5nLHEfp1okXY6jrnXJAoxmaSAsEh\\\\\\\/jxPJ3yDKEEO4JwlnzVKaWjWSEQ65xFBNJ7OSJ2kcFDJCpg1dMhUHQ\\\\\\\/\\\\\\\/L3k8SJaKvWd\\\\\\\/\\\\\\\/n\\\\\\\/8RhWX6Z4WvH0ATpLWENkJUGtOPGoosPhJWrv0GL3zsYIuhjIeZW470ENol+7xW1Xd58ifPYXN5wiXII9lvqUJ3w03UUapJmtgU3j67eC+Wh+ix03JsRZI\\\\\\\/xAqe9CgW4YQ+YTBIqW6HtrUq+Zy3qSGUs\\\\\\\/eZpb9Rsi8j0zkBeuyFBT944uwZc6Llh2lvA5UwsJ0zL2sd7ElmQ++wjmxVgzYRFzn+SwoKDPpdsXzBPJpfl5lwTVJaKso0xrUUaT4KKo2iL6SCeqnlh3Js0uU9+ag8y2k0olSLIo0wUiMO8D+5ZwVZ3nUeuVvtd6+olIQHiQvXfesVbnBXlmuyQFOHtMxqNkMsgddI9TvN+UaWqcbPafmN31kRUEMSxE\\\\\\\/RU8zsdMME0nOIspDtkqF\\\\\\\/H11wH\\\\\\\/674tpMaqD7E0PPE1v7FWaRwwoI6hviLmsihHCSZ6ae7nLDtVOd\\\\\\\/2qnyRYRRYAGiqzwwOg9K0a4RFDX1vZlAgE+ToK\\\\\\\/kuQEanPyxnDIr78o4jTRyH5U79Js4MW5rHkQEr8YfTWckKDCg52Mc05WS+SY7R8I9KNbeOIDdTwmaxHxl+aGtPUkJGVF1UGHtkN+twYSHJoLKMcl20wnKuNNuoB7R8MeqvmzcEgSw6KCSYOC0Xu4B0EZcfc0yui+OBnMwTZHf2h24LwiYzHSiPmRhTTQC5bxya9FZW8eLNDMNb5qIlKVf8TLmzhb9NvAI4hOcGmWNmw+qT2m1T7j09FDyOZz9V9deeWRrspnLnEVfmz2kCxA+88A2JkBXshikxVEyhNzJeSgYrIgs6fZpWjWmKlukN0rXQelghC5UMusEN\\\\\\\/GAtj1y\\\\\\\/acLxRvVqv8SOYoNs7j1GA8V7a\\\\\\\/v8kbfOFyFc9bedm2pGYZPv6bdhbQCnPcoHE0ECTetrRL5wA72dg8oUV92Rehiz3A5c316JUYrObGAOF3r8IJndjl86AX+SigBIM75HQWw56RCA77xtLLl5\\\\\\\/EZmkQq4txQVlzebR6Prkx\\\\\\\/5NFoghvDruJe72CAO9oySFGavT\\\\\\\/dw1ryKS4THIk5xLBHWH5UwAUd4J7dM3m3Y8lGCw7q3kK8WRGHdLua\\\\\\\/acbeL1B32BYkZTQYdDD9y0PxO2lFi+j0zeUvjWV0wQTaw9gZORP\\\\\\\/CWPpDEeU2yyW0evpD4znj9R6nlVZFlLLaXR5cKAOQfaZps1TtBPs1TYPclcF30MJCYpcErcbOr2ExC4lbZI17lWM8GADzvX4+Fqz254Fok7KTkwJ+EOSxoBQDKqChFA6BrFIfmfO4eu7FuORQk6YR9bh+vK6WQpQcyRMfTEWbrF9jNPLujLKK5daoxIF0P9Jb+d1+PbOC+RkNx1YqynoDr\\\\\\\/1mby0LeRL\\\\\\\/PE\\\\\\\/HJm+QvUXbK6PrankLLbNOOeB4KzpMstGnvxacNrht4EbkpQdTiK0L64GAEz3dRAFZLBPu2V+WouEwXUm6JvYvf4kZu667hDoPmmKOV+f2C4KNimvVXaIhx9YvEyX1zGXvhPQbbDxZQQ4PuXOZUO+GXFfsZB\\\\\\\/6d1j+uCoXNttMhZKl6G7ykAoO9gE9VpPXAzL2lnC9kWgaJmKoJM8GWYXXNroqal+3gJHZoTZKdd+UPJturVWNPFQYNBuOOIyTibl\\\\\\\/JQ1OakRGQ2pLVt2FNoxYliWqM2nlTypmADR0IAlItYTmtzAqOmIGhuZyIGcf99U6om3xW577nXOZBK+f8ssc6aJDYtPZiPRs6cHmVUYb1vS2fp1+tDMtjygEt+TyIyd7Q1Qu6eeH+txU5wqubW8TDmvs1rNQp00XGIu7iPpytm5fOb\\\\\\\/1wqmzcUBUEgqYjex6BioLIyCSYAXGdLG2Pp3o0tpH+z0hfLwrXLFYcpUFEWxpzLnijFDXVGt9W4v+liWTrpUaU8twwpUK8VHdA1yteA6FYnPueYK4DsvO32a\\\\\\\/EcXzyE9ca42kHDLy4QddK0ukgp\\\\\\\/Av5cFcfkWbG0tSUYN7SJA78ySlD\\\\\\\/TfQYbJ4T4+JzlbhRPjYUZA8yZlVWFGZZiE+t5AjdmRV3ww0hlNGKvZNytLcy3Gjr+WO\\\\\\\/rs8lkHgpuh9pZrBY5lVeCINrRvBVsDPTyHmhpU2rZl2ZAc17QPGcH+O4UazcaWDP8lYmLzQGl8LW+lu7QbI1zjkIEUmThwUQSOAakYxGY1zt\\\\\\\/yUUMqlO7hvXyLf4cvSXTVo\\\\\\\/oXgEttAlLxdsBIH4w0DIFNwLSx6vOLKEZtalLwRi0qfywC7QSyAfP6fQSclXiVlGHr0Ls5AIy634lTavFXNXL6TH+neIa5wnKn7jtXTtin8liHiNZsH7zVP7UZ0TlV2EaHwBKtnJ3Vsd7wC6om2VbPy+WIpnDj10VpU3dw1p6\\\\\\\/039QAnQck2qBw8NYDSw4k1leHCY46S9tLbg7SZcQWb1M0Kpp\\\\\\\/vjbaGbjhWk5vm5FewbDcjl86ccBGbILIJJFZrMeUvGrUxhGWw8CPLuoxHlLrZfSzPheXosSr+JNi97tFA4eNpQF3DQF6OMRPq3CiDb49efyb\\\\\\\/e1x74IkNt6+sK3Wa7sb5cT9zvj8B6BgMc99cA4HHiddnDXoORbruVRahdhAaEPoyOdr0qEBmeshenVeaQ2dG+CsYzzTQnrcH8HZGYaQodHriO4iTc5HETrqFw4wInXFBEzNwpCEvDywmt1liKp+q7CbyR3UZapC2ebo9sCSgGPZj5CUFIXhklHR9numOvz7uv5\\\\\\\/A+HaDyddxVlWPab9bssGTdutCX9hyHi17NemK74le9VkKCZ8iW6QS9g0U2n\\\\\\\/IKug0386ECbnxsJb6PNpLKaoiyrhUlgGk2EJH6pHrekGvgYIR+FvXde0kA1ADaWeGwOQr4uW3DV9XpUf394SWmWF8aIz+GKbldGqBdUZZzjHIjA5l+8jVb1eRLPNsv909MOx\\\\\\\/YXAKGO5R+K5UPb52MqEDPT485AVS3VB\\\\\\\/fwEgV0upYQzkJ0YIJlsdFxvTTtrwRR54ha98CHQLBS7dlpAOZSOoDpm1mdROGQzhmG\\\\\\\/3a2Dh9DrVLPsTNEi+6sYSepnBqBp0By8sZ\\\\\\\/4WErIdLQ3QJ+HbdPt97RT+5MhmvKf+SqYOZ8mpoWnoCuCvFd8Z0uX9OY5qMNY4ilUkbY1+Yp0\\\\\\\/dPYb0+7GkISLiLNVC\\\\\\\/DPRO4vvmTUc4HTkjYOBD4TiSTwgHuj+oNyjVeKwjE4o4OBPNLIsnb1vqBRpL4Cb2Jk0r4rnMbZFrXlaB9xC9RcjoXQrpuWQ\\\\\\\/2rreFqKHuyuo4F+MaLIIOc2dkNbtpp+bKzF\\\\\\\/hOINZDn3m\\\\\\\/A0P6CT6OdkFgn2f1dk0Lni4Txc81LZR5b4VBMbdhihA9oBx30Q9qB34rTYyJmVdE8HAXlDWmaRVRPh1F3Ctkx7o2m4CnKgcs2J7npBYvWVuwG92oLhcIHbqVOnWl74dRChz7SYYmH1eKx9KMw\\\\\\\/ALwB8cq1XALj8umRDZRclNZNoY2knn\\\\\\\/0hGECVM4Ad98750Xep\\\\\\\/jtI6\\\\\\\/655+sDhNFqsu8C8BHNiV7R6Ht4mPNGD9yIs22gO\\\\\\\/1m9wv8jOzPuZ7agnSxXfI\\\\\\\/Mvp6Fx3GGYdtVfxAIFoUfOlhyuh33yOLhZd4kRO21EYGbbSiBmVKyiuX6VhWGLRMx2pUmpVFZALflcUaqz4qjOIAh3aOGAxD0rVIRu39ZQc8bn0CwgBtqvcPnurlL6gckpzUYu67EwJdlCmLt2QpXIYu15Dey6k+hHSF3TbgmqB7TSq+mh0nsWWRa12+x7ScBOFGFAQQ4jRkz8ZMBX9HKbs+Rg9gznR6bXknm55D4t5Q\\\\\\\/zCi80oV\\\\\\\/87aT4CItwwGECwg04yFMKw30KfXxVEKsO81h2n6rWLgW9CdxB8jgA9mUIWbLk9fetMWp79Kp00DFEwzx3jfvFBkXKZwxk20+IiStyTYpdl3u3huQU9o05dRMz\\\\\\\/VsDS6jR8CQkhS7RIT1Dr2eOeVFspOZ92zsHsepuyK8iaBLe0yH+WBbSSAhBn6b1CqT9QP278kh05+oj6R343Y6kz9BokOe4tqhRARGw43RetqSThC8fmqJZ2P2Hot4kJc\\\\\\\/yywQNUnX7MzpaLC2X51XToef7I3mVVzWBBnpOhh0cNHHXsOIxRjoA3xnk3mAW43eCGhX36hAJmA9PbamnqgOf9hzOqtdx59pDpcGzERV+H+vkFrbJ5UAQXlxS3e1PY0wOkd+Mx5l1jYNz1MYC93RWApTXBIZHlvLLHcqj60wxgVyVgOfzZZbBELZHMrN5bEs7B0HXvsEf1DbK0KiqaXdQseJDcSPDbemMR3w9eP54AbPkkWO2IDbzUqv1v1xGOWisjmhcSK8ZnCWRvRuG572fbIFVZ\\\\\\\/YCiVxGnTeqCa4jL5\\\\\\\/MBMpnQRv07zrYe1if+A5\\\\\\\/EKcaqsRt6xJY2imnuOnQwRCCRuFvXWV2VO+TN3GpTP7EHm8PFWgerst60EnVOYgmDYaKt7\\\\\\\/2kQgtJv8qRLHs86w0F6RE6kt7UHlwmewMqETYYkgBoP3LWtJpICbWGb+mTvqZVDtSd1QY1\\\\\\\/sq7NJTmW3Auzc8Pa0LbOAZzUqS+sN2cLEZi70l6fIVGRjFBvac0WprUpd6Ly7QI5CSPybFqciELhWxb03N14T13\\\\\\\/PgVtwlaKrEgrNWn\\\\\\\/7hH4DxOY9FMtqruqjH+nepeZU+YzHKkUzr6KwilC0AkooGJmHk80xAMdrfswhrDSTaEvwefxy8zINQI6BET6Z2LWXGWJxDLoCd1av7ajOJV5CyvVkPu730f7woRJJKOD0FVyCNl8AJ5hARieQxzKjh2QAXzT3xI73W0dPA3uMOKg9GOlwnAWZlaJK4qVB7Am6g9ahBR6gu+wEQ+Its6ryvmiVNBrSTfNpH\\\\\\\/ETeTBkhGfdCeXVy1LnX+xbOS9LurmKIlRqP3Z9LsmeF1T7SfHBxYG6AO5QlamQtjq1NZS6i2Gc7MYNswAn5w9\\\\\\\/3B8rdk3jSV6Wz\\\\\\\/rAcwxz3z2RpEFJImWtujJZedaYZSjs6EjFDygNgLBcqqkx1MAR5wYlMQYRRBqI\\\\\\\/HzxBSypf9WNmTZel2tov4ZPgnXnAugcuCqaLfkTp++lo4NnNlCmCGnegDZEbFDwG9IyF5NV9aJeHQaslwmO7lnVAzOL6SvDo2z1Voy75H8rJktwQZUdxu4ZG7CoUOgkPyKvn+TKayzLf2krkODfh693c6+DiunX8rhaNozROcivinwA7HDstOxEPjbmEC1xdvx4pMA0VYk74Ti4JwEsi3gJ2di0lqBIvw4T+TFVEKp\\\\\\\/S7BBqv2MAwlNzH\\\\\\\/zyIjKwZOYaRuHE8Awv\\\\\\\/rDkwLqnrPO9l52mEywZArMIBEWYU8JEYyNuX8n\\\\\\\/98DfXnoavHgDiEFGri5zWNDDWobmWhnY7wsRuKav9vyfGW8hr7+18FHywZYkuZAD9ZhIm+tcEilZPV+RUfpunuEFgRdlgssElhXhDbuiOuIzRm\\\\\\\/4VIEK8dM1T6JVmpdHzo95lVLg04Cx8UmoqWCnq38rJQuuN7x2XsgPPLnH7NCbuACLQ5+LARWxRb4fuZ7dpFloD+X1XsJgrF7QJZ1Sij+esss8DcefdXBvJP3g50A09q5Gr5wuzhRPRIvN5VgcZzGVL2UANr\\\\\\\/pQ6WBfzOo\\\\\\\/2HISx3fLn0ykLeRP4sLo\\\\\\\/pEDqRNn6BbTvP0LJpgjW1kx7xVi6CWDpldqbkKCGgqrYyaHHSjp5YZfxA5XLNWErPaQLkUfdE+kWpOfFZYGwhWVrFHInZxPecX4Oy8i05eJMkuEy8VNowqanIHK77bifyo6F4QA73UHu8HupGsRbuFzJEu0hjbe5hgG34aLXZsdQ74QIBGx8sAnTopBHNZhnPVOkqRAW+4gna8sioD3ELxtFudBtPt9VVyDP+vTZyOn\\\\\\\/Z0hyvn0sCvAz64Ha0Z\\\\\\\/9Q9phnHNKQb9ZqNiV8gkGoX1xaimd\\\\\\\/T9RJBngkZZnn5T6l9JjUfHwmiyC9dH0XuLQinSFTR1DlBUxrahU6iJOmr1RIgSS6USMKg1Y4oakNGamobrTuU0YKJhPvRPG2iZPbK1eZST9f\\\\\\\/JheiSMG5WaLJ472n74F90KECnI0cbx9uvTK+O57Q68biLX8oK0LTPGbs5lpRgpvFOBSRYd2zC0aPAWPbEqIJnY3e\\\\\\\/xvM0qlsxC6uX16Y5XpJA2yQy1Z0b4TY51LBg2o4LdA0ES1K8uYsX7hbr05xBRGbGYAAK0Dfc5RU0mr0j6W2PCf6JeqQ41BadZ\\\\\\\/VB\\\\\\\/CgkOYuxbO24YtUoGStVHsSIBCQDPRe7YEhrQ+cnjxsN5394RmKjhiVKCO7ZEe+0S5QW3RZGJGPl4pgx18h5mW7u6e5ZgsHZj8IDS5By6TDx17D2UMv0EQ2Gya0bi+Ma82lIsSJ1eQR8\\\\\\\/iO\\\\\\\/0Z+2XpoI1LO9n4u4C4TKJwqlTmgxAOmCwJdNcJvSV1x0T8bKXiQsjme7UZDASxrFxkB8l\\\\\\\/k4PTap95e0WSgMlHUyEsV\\\\\\\/Acl7xVsxNQK8E40NgrLD76GDfqYhoigMh1wB668jpdph9U8qDBRklCPGKFBbh27LjaF4QpEzUrHT7GQP1dznp2sgv+IpkfUG1V76dUBU++1\\\\\\\/1bphigI1y\\\\\\\/MPZ\\\\\\\/nQ258Bwvr0CLYIu6IaqteNp2gREMr7\\\\\\\/\\\\\\\/1Ve9w5sfGwI4Makz227ZjjS9d8b99va6V3G1okOGRzAEaCjfXrpg1ME\\\\\\\/odLg5sldtC1bgBJwVOzxiPhFnf6xuQfU1XenAgzHUEa9kiA6bc0IqqwyqvNCmyijMs\\\\\\\/ooPd8UELF3qCYuQqAi6ixiU8+1Ij9b8M9hW6eH8ZNCSSsCzBY5FI5tvlu6LPMS3N4dlxnCLo4hSK7vRLeKAmnsXOU6tMkxd5GKwbrjE1rNH4Os+5w3YRoBiy5bsO5ztqyorXxbt7MB8ADVGlIj5ofywcfiwUGU6MPxO2ZtoixGtBXHE3TEcByh46OWgNRDt+DuxfKJ+M8jdKAU155nMksyPcoc4ZWCZd4QkUTVSc7GhsEifZkwNXEUZoHHPqRwaoTehOS2LJPMqs5jJJh9pJ3XLLYsUrrmka213ckiy3+L9gW9k9KA8wNWPqbkJhRIRu7+i1xpGFHaWYbLPfHsOxEu0q9Hsqw5KAqMrVR\\\\\\\/NiSEvQOdEUzMU7Bon\\\\\\\/zHvbb9jVhRwNebtnpHbqm1Iqvx0xrnGeG5\\\\\\\/MgBL8lpBMbBTmw8rnnLzXZ0hLRtSV9bYEEIrorV\\\\\\\/UieKhP1jPTMRDeehUdqImrfVyUHFPnXH5FUPC8XyXCHkJ20+dJ3wdH2nq4Qi+R6KloSjhvKs+xKtII36vJshbQgPU1IchKQCxtC6x6ycxV+W6DRJDry9wjoU4L2SQSnSYDiidtUufxid8ocePa+xYYAnOakkz5xSm6uAgVAPlkGU+syn2oLhWP3aR5SxsxGyEQTCXzalmZRCaVqtmuAopRLSGgTtqdWZW2gF1EFuUgiRj7BsUPMWmJx4nCicqSPdczHa7QO2ExOcTd4FNvbS9osr0yVTAnCsB5bgxHnnX8z1\\\\\\\/6lHD1NqhEgVywxhsK8Ya40+jxSLU3I4nO2949\\\\\\\/7Pp6ttkrj2SqCII9lylCDt4Vk89AHHk\\\\\\\/3bHCQ2OZh\\\\\\\/MpzzGeIet8Afo7EjAAmRg0cXgvlNjelpyrqbSED3IFHBdJwyQP5jL5wRns+JQONcSJ0dVnt6DKIOqJjsrEuJE2N+bB9k+9nNK7hNhrh1ywfzPiQCFaP\\\\\\\/k9ZkFp2SepPHz2MslvEct4d\\\\\\\/D4XLCqOLbXsaDlVAJDmAnXH905g7gYnUjsp3MTapWjg7t\\\\\\\/IVzWW0GcS2an66LbbftuIXaShSKdYSmMXjWywTEcxgd9a8gHvRiqhmoEQkAOpDK9reAFDWOxV0IfAkfda1VIdw\\\\\\\/CDG3Ppr78NfVN102TL9ujhZg63JFGACtY9t3WeqvMEIsWprGHBrxTx2fdm8pm+hircDsz4A\\\\\\\/b5YKt+rVf1S4MeZhh\\\\\\\/yfADmCokeSQ5S2+Fxn\\\\\\\/uLeT8pUc0PEc0t3e3om61507A8PW0aV5EPsx7zEAl33SfuFGcXU\\\\\\\/bEcTUsNmgjfle1MS5TrH140K9SoDwM9kS+9ycaA\\\\\\\/ZkxJ2R9oH1GYg\\\\\\\/xzmMq2qvFA1MviGAN0mrWwel89ScS3J0tJPO\\\\\\\/l8AObGto\\\\\\\/fxQh8LohocPzBpAc94FlwZQoQrE0W88ZpEofnQNDyi\\\\\\\/Zrh\\\\\\\/eyp2EQrqAgyRZkfPXC8Vik7FoCYiky1PipHM6wa5bQSRqr7ShVubdrEPRr7h1fXbQMpc6P87\\\\\\\/dfrIf\\\\\\\/IkSEp1QPPKvNm6Fz7aoZFXFFcLuqU9bfXRKJ1E7lsz8BMzN0jh7923Aoa7eLI4O6XEh9p4m9d+XEoGjhmWsgoVrJ6xdXypUC0B2Fn23fHMZ9\\\\\\\/qM8nYwLlK2OubRmE4ZqymeYjRC8D0s+8TECP6iUIFyYqQihebXWmBR5EbpBdCBSS2iBUw3s8+ineOZliW7nwSUjUvM1widwAr9hXFOzhFvUxbnX9bj7G3\\\\\\\/35HC6TkiL7yhSyUjTsEUtffttd6qcIy3dNqSr4JBdA7C85w6+OpnQltlJChwbLdI40bbuMgBo+RaeQ3P5DeKdTZqLHfy2Cvar8ASJ94Xs1LGRaBsjiWeHQ0wrHQopd4pkjFwT5R1EqMd2D3Rg5PCrfxIDbHvHVzqpU5NNiaIJ\\\\\\\/c55oSWXIXQgIYgdg7uTnN6MLx5nDnACu+6iGAQUy515euSE0GP6A0dFQnu0hzUyk7VDtuYz9JdSrfPhLxuxe4ATfroYz3BElqkXdvynItrj8hloM9Pa5+33uIDCif48ycizXg5\\\\\\\/GIVRMGuNWHXDWbo6yhh4ISgjz7GUcJO0OAPdmaoc\\\\\\\/SzLeAZxFfKXpc+g24VEEBes+5oTkqQGDIZvP5luWiwRrckyu6HEuoo2tiNq\\\\\\\/fiSHhj42Ka8XgiN3KofYbYQw+uUjdUxetu+dOkZSoGI4lVMAc9LjLaS\\\\\\\/lxtrdgoSIC\\\\\\\/WFeFlgjTB2+rGK5rdayxX7Z\\\\\\\/UIXZYQaIMedRpDba2ROIlwgsNWMcYjh8aM8RluvLMn3howHrkEIZGy3vIcN55I+e5Fi0gzzNCCw3AanNXgs1EJYKJhiGSh4NnwZ1N7ic2+BJFo7FgQtR1FJKL76WZgl5Tb580QkQv2xByxGRZQAsXtZ04nQO\\\\\\\/IoqN5inWa0YfRnZ9M098XeROu0vE7nzaiDYivXMcVLavEm4sal+rRkeBFjEOOBD9DvWidsYcEG0FaHlc3geBq+Q8Onag3jkEWZcNVrqb6zT+Sdzax9vf7Lzi7RKKOlF1ybeK1NYGFw+ahHksRuqIgVrq2P69z03F3Fn0MDON3lmEC9MOs2rrsOBGXFpJj+8vRQpjm0y78MEElekL36uovykknt5vSONEvp0P2b4fLxEfwy8DwoxbqU66a9izVtv0iX5809BwAMOTKM8OZbuNN\\\\\\\/l3Q5lmBUgKrZhbl0+xskmjCPwi0NOHhwCjA3N9ZXPW81m+rZ7fSLFDLZXsVSMx22Ja48f8ETEKk\\\\\\\/zbCC\\\\\\\/wxayjwmCtA14rHhoov1HX63uW3af9jl2ZLX6S4bW+HUF5WIfvBF6qWewDCtOwxH\\\\\\\/yTIeQj220yPOYLWeEKpKopKtMBTEi27HrLTfTRzu5j0OG\\\\\\\/gQlAKyFyodYF1Vy167RDvmDgqg7TubHknltbs2GhYctJ9wvb3qKiDJpUKFqguVyhCnF6GTca44hd3w7DnZY+68cQoAplGM8Wo7EjF9rPZH48C0ILyoA6+Vx9TfrCgtdkdK1tNw0wnRy+UOqwT66tG9RBpWfOST0xIMslPAJ2NsHwBYNETjxeoCx8Pb+BBdEx8hE6AjIJ9iM\\\\\\\/81wjy4ZjDs5P4shbfZ4q6szLIHEdkwl2\\\\\\\/YvF7Lui1Su7DwJvbEOcsJOWr3php0eEqe7vYJMNjMyLwPbXoYZ7AYxWXJ61ZbiRhxDXE5r26z6j4rblN7D9vvPVcSdGtKAEsm21Zpzf501+ey6ILbnovH6AWudwsW18cHgXTlNLVw0UzKKaQ2r4JLjXo3H5KNgnGN+NgEquYcx7NpjoAIDyNnZ\\\\\\\/7YVvwMLy+mJZNdf8rbM9+qokztQ1wpfmGZ2l2jtiOL17DlXZVRp\\\\\\\/RltPVsY2nOPFJWQ9PkrxmgamgpNJnzLs2MWuhfrZpAmnZ1h+hxOhunRDNKykFXT7EjiCj3DBzv++EpWnlE34znd6M9SBFxcv\\\\\\\/ZvKUaW6Ypu\\\\\\\/ojGHiXWSLAG1Rl3I8jXAAZd\\\\\\\/NHaqYAjNmZmCScF8ltttq7avA6Iqy6d0M8JpO3tkEszg+P6ofQ78w3r4ZU\\\\\\\/1aDFxIOf\\\\\\\/Jqkly\\\\\\\/Ad1UdUAOrCNtnGeHwQbO153TixRY3fcp649d\\\\\\\/1F+DrvhHdzR9iyI61aNARUrU7IezPB1YfcoCOTML9xxQrZ0D46FT2EAnoGMW5gVLH2PeOROjNPugmJ6Cm9mv6IdC2LBo\\\\\\\/9iAYtpk5QV5i4LYa2uM9L\\\\\\\/gjfqkwCfFtJVTGAzRZqui0ndvuXqrsAA4ajuPPJTnDwBrEPNSiLVb+X5Q7s3sR3b4g89Vvf8MWNHlrNuBcbAwljtw9G+73C9rbHBZhv3C43WZVakQMAbKXkfXQun2qGS8wlYGzN1aJq0j7JvRxGsMN9FQcPIi8cg\\\\\\\/3km9BK8B1pyFALivj9ZsWAmzHWZdzr215iDYZQIcc\\\\\\\/e6ZgRTIn8cETVyQZ3xpLeXo6k4gi07d5RaXVPdjEeQEGg+hkFSK9CiRiatFf7WRIGqV+FTKA68O71+o7Gyx\\\\\\\/Y39o8MBD\\\\\\\/b4ZOwX6rVkfk3L9o03GvoeDScnCR+D9SDV2Xogn7Fl1s3LGMtiXcp3Qd73G2WAoSHNO7C0STrZqhY\\\\\\\/AP893WLfc5BQMqe1Gz8+eIwMyGGPWuVHtu3gE9RtZT1slyu0whql\\\\\\\/H40jCVtv7RMq3Ix3Tib8clortHdHo6HbxgU25W0oFj6n+vJPlZee7dxq\\\\\\\/\\\\\\\/IcXte+oY0f8rl6k3Yh5UyU3RfhIyA4aLSCLb+dyAw7MdZqGVCVm8N6Sp4eYmg\\\\\\\/k5e1GnhRvVjdL5BjpjLJjuENjdU2mCJWj81X2Y8BzlOIZI1p1ArCtdlgb79BcEnfagRxm1pqvDj2T1hRfn99R4qUOrAlzlxyZkAfFOv0BMaFAoBlFWoXZExNoWX0C25BcLypm7pJHfMjCoqGHPg1PGTRSeEcol3sAK8PRpxWCSaZw6zuv3zn1Ya5ul47p9wZfE\\\\\\\/nNn0swQuAPmv4r1ZMNk\\\\\\\/6N04XszfijRGzZmYGZ9hvwIPS\\\\\\\/t5jMTF7xkuq3603G7OvxE5Wkd2G\\\\\\\/OZD2+wvgRSgqnHRf3qQW5wtU5mOenWffU9mtkXaGYwd5wbYLtauuHBfQoFgYMSP\\\\\\\/zn++3MXjpS4F0Mf9FS33IGw3N0vv81ULtMhshIBmmyHZ3\\\\\\\/h5iCSfnb1KYPhqZhiqIw8DiP6h\\\\\\\/F8rFVxML51ImzOntrEqISExpcmZbpYDfSOT93\\\\\\\/RBCiCyPQyUk+TNx0x1eStYlmbVWcdYY8S2v1A7bZucCSEL0GUG+EnsaVMKgXIhzV8lcrTkzyAkKbZfo8nbpxTxipbHwyElz+zJHanC1bfNwN1LKtMEdE5RBtxDj+GSDexcvbEcd93fy1EU3j3twrQkwfwq9o+HYd4zYtOgwLkKQO9a7RBZVExDy5IulcEwL8eBDMexTzbklDSt1S2akIlVUrKXTJa7OlONNSm5y1qXGK8YCpBtiI5XoodR0oS59y8F3G58oMz+V+KHqucunP5hnzm0DrIm1FwHBvmae+NVJNnvaxYp15v9Dk6FVntQ8ju5XCvUfmaNXmKKUxVGHOZASxj7cdLz8ea9314FTF+ocq0I9y2lGM4Z35odx1WvKB1HQHv\\\\\\\/S6RKpr1M9rho2YgDQkkSJwuktpoktVxyD3OY9Xf709yn8dJA8id0Xil0ygK1ivK8O4\\\\\\\/lV+4aXZ5nS31BwTu0YVKzzRcq4YzX95TwX84+TvspBG03uT9OuCRaMwrp1UiLykZfeZPjj0H9LPoz0t5L0VEqEHtLWQWrnWNZV00tyKWdZiWNi1G2ocLyuHWHZ132gK95Fy+KFmV+6YACeSVHDPVbuwy52AaKO9iosDIWALzaHxMIdRrzjadVrf\\\\\\\/BnIlAUCAATA81MDdnxZXt9LHCL9R2W2BaydHMC0uyWdPlNCOsv4tNd\\\\\\\/lxu+JOp8SOWl\\\\\\\/D64UN72PCn\\\\\\\/wY7zzm4SLlnflWFQFENGeI1w94i9pl8zXNckse0MoGwh7JPVyGXyvsZ5OMHCGRDVrQ4WPTOJk6GloSXCtHX9kPxu9pyvFXaOz18H0M+cYHG839zSBiLM5yn2lNe4x1qDLfg+U3VU\\\\\\\/C+u8IYM74b3PfBZMm+EjZryCt1kv43j1PU6VLN+ACoI+N6sb6gC8ynwGeggHE+AU6XUNHtLfvtxFiA6KTXRis5SMtXhQBrJaitT6IvJNSZMhpid1vJMLStWoUQ2CHkXzOenaLVw7ChFDiOR+ntmMXTnBuoW9DXrPBPzFE3qNzLXk+fLANMM9I5QjffVZ8b7\\\\\\\/M3lVKzeoplGWmYP3QhKvKznDj1Im8eSHMiDNLdsYOpGen7UVoxyCOfAdYW6j\\\\\\\/0xqXZAZ+S6zPrRX5QDhAQHnHqakXc3pzn0TXqBnrEiJVQJA81ZlU1Pc5O9SlPIG5SZMQv4wJXClc9W6CRSYUF10lCAqmwJB82lDP+N2rbYwwqd5PQ1V2mcdJXF77PBbDozqcjKJhdnJGr0qkd2He4zmDnzbeucaRbBxP0ds4K38XOFLk4CzLbPECC3iD9Dp3mrCQUlY6CGGjJVpsFQFp04BLlE\\\\\\\/dvRdN9sr5XLgxapumZfjGDLMDp7F8NKukk1Hs6jEXUbc+gXrklyyQEEKRWOWY8jb\\\\\\\/X98Henxo0oOdft2H2bcOM4Gn48hE5S6gmg46Y0WD086rIEUM7PrIB\\\\\\\/ME51NcYKq6Wmlfuw4prRAgpkXI7dSjOPW0QFchSTm8CNTCq5rv4WuqwUltZuapRVvb3N5RnwoC5PAQ6GiqK\\\\\\\/8aYkBN7Qvji6F1AVzVXW06KJ1izNqKu4SRWAwEPZ1XvkBIg9wkRQUcFKeKbVGrQzqLcal9FE2p9Jb5GS52Xf46gmbIOiU93WI5+jAPdEXmGRp9Ql3LVCfSfXKhn177m3zB1VNgX6IMbmnYy5ZfMh30Uy6+zXPaqoV4nDjLLFRlzNRphKhTvmg03bp\\\\\\\/ZZ2iG2Y\\\\\\\/AS0Xwt4eJuQJ1Ng2IrJsUjwaSwD7\\\\\\\/lIdisqTM024Zx\\\\\\\/Tgx5ge1lXRIboNEdXTF2CpUdZIwm\\\\\\\/vnceDXyUjl+G+nDXHT\\\\\\\/82OzTddCG2zz\\\\\\\/f4DMlLd3\\\\\\\/NQH4IJoHd2D\\\\\\\/Vl62hFUz9Mr98ajNXRebWHuWXNg5\\\\\\\/2k8lhjGtKg7QgfXgPjcksvjyfMeREGL0b54RFIL4uDFvFg9QOJcIme2bJIvvoJolqETDWtnn7mtmIIcOWHwiS9TOo9SG3TrBSbbdKfRHWlXrO4H1Z6leTSdkzBi0X1RWrqcqdZwof6gMAO4G1zU8uMwQ\\\\\\\/apE1xC8rjU79emKGm814FedqFyaxOq\\\\\\\/rtqJxbOEkO694axIN1Zoh5sf\\\\\\\/1sL4jewkTs1GE67k31vI6Elk38AzCRYXMpPmTFwdjnclANMeXnuw2rhLoIotzsejOwr11KGDzEkiEW5p1H\\\\\\\/snCWWUJhjnzTUJs0WVUuQtXFrFVaqwbtESqweKjvxT9Guqg5U+lqNyIfjvN9WMAg9pp86BjIlF0wJiiPzDqV5ckTwIWeotT3RGZcAh8wre85jlykomUjGatEntKDk2gAoQ3YWnWLcMPPmv\\\\\\\/tmQcRtnq2E5h2uhstMCCpH6BPobYP1l6BJJNl3OyGcqZBtCQ+jWWihklmVcfCs8j3otS01cfg2Y9yEuHwPo5XZ3YVQIvLiVktsabMd2fUyuNlAoQKr6j68+c\\\\\\\/keR8BFwM5GR\\\\\\\/SQSugL+44An8S7lco6wqRz4FEqBqBKBLiTaR6C3inyFTBkfedlCoqUCfBv8C\\\\\\\/AvOGa84xnd8W1OryIVafLSj6hlyFm3xD4yLxEWkorURsdBbfmlohCxW7JA5yJl13ixah40AhO9l8r9HgbVy01PrHaWJzfvPE+n1yXEFmpodWlLNFsdiBUY5KhPe+wmBkA9JBmsB5jkLCpZP72vvcxVE7xt0cvvEPsGSKdAHrfGDzV3B7+b1p1pLIBVFL67UvrMMjxtsiQ3xxIGTRQrRMTRcNQlGmCbbxdG9xPvw3hjDtxFcpAWl4Vsh5ocGA1MLHwkUWlhx7x0mgaQb\\\\\\\/6PgxEyGfGOp\\\\\\\/mgA2i9fzKatGpfFZ5foMUT9\\\\\\\/RWhs9ea4WG2bK8DTYPVuMxItlr5xCGUKTc3X896zY9VvM9YXgSI0V3xO6cWCe7jv7jrm1LBrPT\\\\\\\/vd9OkW\\\\\\\/2oQotx+t1JThoz9gFQxZ1+ShpRliH7ULhv5B5SiCa05qFX5Dv0FNrBX8OoVAPAHsYfp2y3RVPI9wKRTbVvhRTRTu2Omi3PB1+j2QpTxy13Fo1OGtwXF2HXtbVMAxBfBnJCo07tI0vAw50AkhbEJl+bDdai7hJVYBvJATwQ8mI3yZJ634pFpvVgiPNa7PVnPZ14uVp0oYeAmYFKYuC4rrZ24GDt4jeVXSydwOQ3n\\\\\\\/\\\\\\\/d\\\\\\\/5MN46\\\\\\\/o8SNoQtwgo2SGoeA2zx\\\\\\\/RAAaHzGoKKw41be4OimZAqHG+Psc2\\\\\\\/yBRyod\\\\\\\/bO27vUtzQRPikpOnaZD2O+9tupMO03Tx\\\\\\\/Kc1BnXTvEjixXe8GSg2NXbxmCD2kaCEaucvK6p5PUxR8X6mLCA+6V3Vq6qHOz2V0z\\\\\\\/AfdM\\\\\\\/cws9HqFQXWLiIrjbTBKdzlQjbqUCR6qQlWf\\\\\\\/aep\\\\\\\/Q8nvfIabDB12UQKy9KGdral5x4B3tPfoTV4MrY9CRccclxMmdmENxPN1y48w\\\\\\\/kctEHyPOu2GtpOikhPVmmMBHLkNSd2ylUqFBCqieFDPHomyVSwwBTaWCtCElvy\\\\\\\/4in6cKorVab48eG9L6u7adYPTnQnQq86Man2CXG40fBrT61IWG8zJJcwic4pRynM8q1Ieja+utIShnX0M2fL5B40N8W6x\\\\\\\/h6+QJ5KYvVdxkQuPsP9hGy3rgIpnj5a9fNISbABW4HYkEYnNx0uy4HSjjZqRqHwuwfT+YosscnGKToyC4JZ8MGnnVJg3pTSsQ0OS4yj5pOk65nKE0PM7FLYA30gNI6TUCVKUWGCTj22+9Vy2keIf9NdN+zJFYsMADUH9\\\\\\\/sPBOyRLkycys7Jp7vTNlTIl0AC\\\\\\\/4BQEaLsT4j27NXull0qbz8QZpWdbkPOt27YaOdTCzYueVyplonkgKkHLOGxywM1uXWtQXEAbwdZPFYW6EG+AM0rptYaceyTCgdTGfM+6veBiywadEDmpHP2Oe8dlcrueptvqVzs4HoaZN1ZeBo2cfkqIyFD2U32XJ4\\\\\\\/QCISljxyNq6+BmRMhKE20T6UpGopspWPYn5kfRJJkvzLA72DrCE1Dx3Yj66kj6UwxgkabmZuykC561onzfvRnVta7eOew6uYArr9WJR0tFCysfX9ZlJCXvO2sAt5IBGyB3+oxlge71MjDMRy7dJskAxu6Wic2LcM7oDuTV2FpRnGlOaPmBTcoFihBo2AQ49pF54HEmymwkDU+\\\\\\\/MN\\\\\\\/BI+QdvEE7WslZCHYc1Dt\\\\\\\/4CUb\\\\\\\/1pgxX2WB9EKXxnYmHly4pLSOyy3IslfVssJW0pcwev3wik5CqYQvWkrpWj0w1Rc7HJc6efWFVEkXVjNdM2r0fdtnXnDwP+L9gryrupTiwx7UZefESIzo4WuS3AosHahiJ6VZAHX4G6aLzObnmHReyP6MG9cFWgSnjlNuwjs8u\\\\\\\/3Al1XVK3EqK17AAy1sAOzxm\\\\\\\/hmoYjxJg2VFko5OHQSLsYvjJ4icIjudkjYyTY8S9kgceu1ykWVRf391jjgfPixJ5\\\\\\\/a6Tkt7xxYL43cdM+6Czu+IrN\\\\\\\/M0+fTvfniWNIMuhaJrBQlPCC5QnXKHXDTEO7KXe8sc8JrGxfMNc\\\\\\\/Y7\\\\\\\/G9M7YrOXsgxvuGvoe40PTgW9jXSgl7jLf0n\\\\\\\/R1pJpg0SULexfsyUidldWyWp9fUve\\\\\\\/mXG\\\\\\\/c8xqqBsFrBIsc\\\\\\\/HSbtd6rKZ\\\\\\\/58XOTbCHZ3xRcC9ibyuhvDdPLxILZx+we3YWnn49JHpAPnfzp0Sb73crkJ03FjR8bwJINOvUOCPPOSTO\\\\\\\/CrXmVgfn+Me9y8TOVRtxBCf6k1S9azo9vQWmnJ4ppDtPcE4BqfLES0dKTNQsHPaYZLS9LlgXwniOveL15CVtXpI4OrDzAXyVF\\\\\\\/zR\\\\\\\/eAFXSu91hYByS3f7X82Uo+aW6RuySd738KmwBhCqqt0S+pQ6nO8IpjhEUHyKlfvHNiueyvHlmTiSkfCZRv3IdEZl\\\\\\\/K1hbYhYJvkXVDp049xncfwzKcgQB3HS1418YkZbX+iKSSS1FM+zxHdgVfqpwC7gFStJ+JcyqRtxD6rkC6XyOWPDTye+LUumHS4dv7keLz9AGp+VacQvdN\\\\\\\/jUNLHTgcFwQwRsnFlqZvBlk1PO3XPeQyztqx6+JSS1r4bldbMpcno\\\\\\\/MZOgYcZYLVvLXTwBvZo9z2ZJmRg3bmk2dwXY3r1YEsFwnv+Y+koQr7yY40uE\\\\\\\/X8hjLNeEx5oBLqnqSB58hDb3mHOHNG7vyZqQQ43GdzQ7oWDFrT5h5eNZos2ZUoHmS\\\\\\\/vW8VtH0smtyQ0I2fw2ODh\\\\\\\/PELrVaUpcwhzMgX9g8jYuZPnBz82Hi0eLA1Sattsghrbrf0R2sGRA6jmGtaazXCA8Ba\\\\\\\/H6AregNa8o4cCRzHUQ0H\\\\\\\/p35AL2AWgJaUXtVEzVqobDZMwkRoIv\\\\\\\/SEi3Xa3IohO9lpsV9y6BK7WT1Vw4zHDv05PshUwZk6Itw5VFIj6JIHdqFZVPBsGB\\\\\\\/t2PvMRQAEvEppEiLT98zcD5qNt6HmZ3cw9jSDpj1hz1FQ+oWKXTkOOjhC5OYlT49MGdSCL5dNs5C+Vm+FcPymtkiqyh7RfLN+fvI\\\\\\\/UNDSeVHM2ucbDw07OgJrWVnuM0RAprxiQlXzSFT2Gk0qT89Jbek3Cb+R1iyWiYXbsc39o27ln0QkfMOgZx1jbUn5jzFjsR22g3jzdOmUx\\\\\\\/FVN3ONTgm+yj+RRz38C6q3amSZlw3tu9wXunE5ZLvBgL2VF2A0yeUa\\\\\\\/gm6nxRXwEQ7gjQxsKt8OHgjw7wAE50pM\\\\\\\/zhS7eVrhxPoanbtVyFV+DXGtfdjvgpT6PAmKy4xKKE+f+Tp6eyrMLxYZ1qukE9cC2TncAibIxb3k1kE2sJStl43c+MCoc7cKOf3bGM7DA1OG7xoEk1tlqXJ6pwLOtONb1aFzs3E9qyUvt8WqnakpCs9VRuOVmbP4EOimdGMAC2aOaO6hvHUCHFQoA6PqMF\\\\\\\/rMO716Vrke5Sfu+tgR0j1TpIb5+c4plhMZSG1CXaa4+OGOwK6X4Lr9\\\\\\\/Ffn1mtvKJuo5OKemr2wrlYxduZeVtie+tKMe0dtkKlsWZSVo7UYUsTOI58dZWQzCWicVrO6\\\\\\\/sxoGzBv7smaKBs\\\\\\\/ci\\\\\\\/c4ufo8S3rY52sl7N1vGf4JAWTlFIhXPgXksP\\\\\\\/8kzSBln5c0IFu6g7M9hl7oJyDLTf1UVnEpF25j1keEfWDulOB7\\\\\\\/LpSYGm8I9dp0nzVJ7xFphdLllSUtroYYWipaQeShCYtvCMU47s55qDyrzAwZYaAGo5ircF+xzsF8u0P93w7rzBIjH6kOtAVMRKG67NNi0kH94i+Q61yG2a\\\\\\\/m+rGk9otdDNepFgHhAngnB8ORIGD2mpUVlA2IM+ztZyOKxfBsszfePxSZ+l92LJcASssTR7DCcYPe8stcDIyopforyEx+p+iVSquMix6dZbu7N5NhxJVBosIVPoGDShIComulGZtCpea9IEQgUmhDeIqsvxI8pdGuzViIDadNB3Jw0+i7HS3ZvThX4NJJ6X38nQIo+Au2Fu8VHWoLOhrCHCjUSSq5QNyxRr4PWs9Bo3osHeQEQBtrvuM2L\\\\\\\/QY39keF2+5yxCz62YHYVtfGjrvbuVHeqv840oP8+dK3SRx9YOZx9z9XvgAbIgiIOiiNMGFAeDyckx3IyZHtvstFETJOHuMGIRTyxK28o3Gla4eS0PMerbTBrTnu0l+y5i3Rmsxh2W1ms778UALr\\\\\\\/hasl5g2FoshPA06w176Kr9zzgbA\\\\\\\/oF\\\\\\\/CyyM5Fvjr0NyOZR\\\\\\\/kmIjAILqTHSmK8DBtg1h3Y6RDDg65Rv0+nnz4XG7LiwCAJxbzWnavRgDNEfcGKttZOZgZx9SppUa46+oo11PTZ6qXVxiyoMK8MKdxnR7u6ZvvHLELyjtJcwlVlsX8UyMYTENyy8z5\\\\\\\/TFxDelE7X2Z6RwqQcb82Yw5JmA1jBeOVDgrd4ITfgyJpr1OQegE11xa0deBKdHk+YygtLiTb5MdsO3py5mScxoJKQ32nRIWaoW58pE2P+250UbD2yQinHqi26WD5GYp11pOwo1qKmzwMgvD9lyVx9jNRkul6W7W6PuWJgoRM9y8zhB1pS59\\\\\\\/PygUYW6PVzQ6yoOSIMLbcjUL\\\\\\\/h2Z1Z3HqCfWytadkKMf8VbgyqRItzD5XuMjfdt03lYtFuM9IbVr\\\\\\\/OTN4TJM5s2tMiBSKbzOO6ZFpP5BfcU6ahtszrql\\\\\\\/FhNdHRzSJeufSMXkW5nVx3Xfqjh3AWdbPh0pbAUTUnXCbckuf2dFb6IU3g9\\\\\\\/aAZ841UrzGjPfnhFmRUoYqWp3OniFu8JLI15pRlNrTnKvr5b536xYV2gpjm75LD5qraNpOfiS6kmyxSbDMsaQ\\\\\\\/rXVQernofZ3vMYo7TJlnHtq\\\\\\\/4aYacJ7kE8g3RTOGytdtqrAo\\\\\\\/zn4A\\\\\\\/TTAsvMOYBB+czB06I+LtoT9YWp6pjxTOTTt6C2hWC9CYt64+S7XbWIVIwj1gCJ\\\\\\\/a0hJNLO64pRQw9XZAHMUOmnqy0o1dPgJTglyeK6JZ0M+NCthhL7IvGKbEVS6R8xIzueimXPxMda1jLYsSHpoJgxnJlZeyrSNZWj1EGXhdU5MLu3o6wyDFEuLyCdZxIyjmX9AJTZZn4rBKufAo0GxYPXfD6SmsW+z1oDXQD3vED\\\\\\\/+r9sh2eLrmUbdB9m8+dAY6jTpnFRNuyyAviGBohK84YivuDDtVyfIoeRwbGk+Cy0dCLfh\\\\\\\/N11ktWsJhvPtMlZQkJszY3AAhcjHoQQFFqZME2KdwvknU823QSk\\\\\\\/l6tQ8axNd3IEFqBAKoXPqdTspxE2D5wB0UepxqS\\\\\\\/FV5vHQpGFvo5OwJG2sce3F6AXEyFAzgW1gC9Sa4xw8Fo03ENX25T7sgDb9FmGxXq8PLyxpTWmW2t8kSQzGTvM3LvZAmtApUI+jgspa0RPTn5EXIUFMZheiMZ1sPwuMf1Y8\\\\\\\/BpNVTJn1qRHwjCjXNgHyc+5TdOqPV3vl7IGaAsNO37PzlM7V0Kw1iPJZmfBDfcDx66c6NYtyriF48oCZQ0AzT9f8b\\\\\\\/sGjG2iGaW0cOH0mtxwt2fwRRdyXL2TyJD+2kHcJhtveiTvcpVUYRu25Ih4j9QWT8jSatLmGfnrjDwFZ+uJaIrb7WezK\\\\\\\/8mLsrpLif1YQZtQZFcUfBWe3oS00rgrqIvw\\\\\\\/dULoEPp9EhMnOEOf+3\\\\\\\/3+IBZhdw2r5H778gbQSoB0NTfXBJDgHQiiff5lwIECatrTBj7IEFv5nREFTO8MBnxgsxU91IEgBDtouqo\\\\\\\/7aRYtQWmEl4tO\\\\\\\/pC\\\\\\\/VCjpeKc4weWOCC17H9f93\\\\\\\/QiGI3rrDN6ZJBhi0L8lVlGGHeEMtO2XzfZXiXL3umFBX9s4Gy5hWaUumnRXRb+O7hADk26Lp54LgszBnZqEMMc2RMhUp1LYZAU02iNL3R5MmrcWPAUcitlFTm1B4utJxQWACZcd7Xrz3\\\\\\\/mZ7OUBoJ8i2s6mCRxXR3lXTDncc2jqtR8k8v7Krtah8Wcwv3IpmNedLWSMKotiWp94pbntT8zw9h2c4O8YzVDfRHXey3Ffv7lJC+UjUNZiTN+oVbbZoFDjQvt21fel8stkvMI4KPWtioTVi7GWbAODrKjqip+p09FTSlrnJndKz2adlX4mY0JKtc86iBMprrCvrfc2VQlH17EFiC5U5NFVQtww\\\\\\\/nQg5q0ps8AbmhNUEVRQIbB6lmNdJFipWwvAdFZex+V6uPrAuHEkIPU592whZ4QygjNSST3keu5X8bntV9y0Uef8RtRh8OfSmjSoZOcMQyYeM368ex2i6Jb7HXW9C9uuY\\\\\\\/IOUmQu6A1S1P\\\\\\\/LPyP+yCu7td4eni+t8YS1WNO7JWLj1Wj62UUXRq0t4x+UMWYpUin6zoqsHtzbzWamSHloEU0XgqiSmVRcDB4avkg6m4VGSVASgD3AUHEUtfgBq3ObPMuMxhTOZJn2DlmkQwK8gj2XTHRj5PXLDm0NvcWMAW50cm8bMK33jdzYV7aJXRYL5+0P5K0FRojSzoOb1JUf79KoYwdoA+Ssp0dHsNBmXqFT9U5OZ\\\\\\\/yDmIFKUp0+JoH1mAlkXyLRwddmcnNwN7e+f\\\\\\\/i1gVxE7TS7r38lJRvaYw4LQNkCnrgqK6q2EVRe0VsWurEQLWc+BRIfSLTF5opfCon01BPlLd+Ky7wqfAJ5v73ol9GqW4RHLyD+8b0mp3sm5r8iingZzmgF+Rr231zsNRlKB5nHzYJw8P8RIEiimjLLVbuKVb+mvK191ggNka8IT\\\\\\\/zOsdSwmVaqrVKfjsYPN9a952l7fG4jFIn60BustGVbbNFGg+PVMaVSKuNC1U+UHo5tYw4l3LqC5xFz+oYfPINStOnxwgoUQCODu7ks7w+5gvqNr4Sgpvs3l8zexy16g4S5D1QNWCwswAxukf8\\\\\\\/e0BPz5LY117822D8\\\\\\\/hsaE7nx6Hz4Kj1DsUfUt2nne40VqpDFXGTMFNRiIP0m7oT5DR\\\\\\\/QfJBdFVnBxd7HMYdLh+Jp\\\\\\\/E\\\\\\\/ST5dVWjVASEmacS+JNmd2bq2opJssPifkldJ2m3Q2mRXMkfFChvx1hN5Bts0GT2q0XQEpm9mI0XJbAcDsyaY9tIAUL+9wH0zhkdMQi2BPWeohPjyeIaO3YL0wBeY8BuELaDBJaoDQesi9vozfYGHvWht6SVfu0wDTF62dq5ZubzFk1vRxKJgK1nDnfA+lg763MFhyKkbD4xYI1ISXzY\\\\\\\/\\\\\\\/nerIS6URRpezhZm2Y0HPQRcNm3vx\\\\\\\/wgP0NTgNs3764\\\\\\\/gkXF+9f56Km\\\\\\\/sS5HKH1OzgsCZj4FXGHazkpMLWhm9mmxt7ZcpuKB0n0Y0tW\\\\\\\/MnJfEw+7g8rWipysnngKmopyaJ6518kl9lzyVONFRb9y9sEVc8olg55Z2krabupgFG7Bqd3FyRgOWI8VoJlAHVPBRu5q+o6pzpko22ughDGSX4n1D5fypsOT1AyYqQMrjobc5xfb5KMxfOnH1C9zHtK3fWZKJBEdg7g9+EDLfCULjuxHzdVE9FLDG2WaSWrRjN2YftP\\\\\\\/Oqjayaq2ObWRlwcutehm9XN8KQZaUhU5x9e848wHPWYrL5oLPpbLb8sdq2V10lPuddvDaG1+OL4WjLuvI34ZkCvZIaEw6SL1aWw2Q4LpspspMJGOgtzM\\\\\\\/DMlyZNKWFs2lwaIyRlWDf76ZCa0bICf5HJLDZdiFV5XEC+Dwaz3s7Q7L3uNsYG2JjYBaMBJ2FDf7xMtYzO34Xjed2G7V2WHcX20\\\\\\\/qhTIEwvHkW4X6O1AY55dTsGGq6LKNqkoKdK1KMla+SVHQVVfM7hHlzqZPHNAAIVehm66BRVe0ZLndJkQrYWstELwKHYJeWB02t70UEHeA8j4vu2i5t4TlSblNTc0RSr4lLHNdkTTp+inylLOC71fMPM9CPqaEIwPNKgxY9b\\\\\\\/CaiDnfE\\\\\\\/fwlqTlGonGf5vGYXn2lAb3ImHd5Iz2t2H\\\\\\\/SGqWzOQ7RAN1O9AUDoTOuT06kbmGdmCHige2gQzKwCqN+zKMSD3q0NJGPbJ8y+33vDOUwu67PMEHNbp2OxyQCETnasRFh\\\\\\\/OqWrabuU2wp38Qw7KLrVEfBGBNcLpudrWDLBJoqqt0nqGdzkS0i4cokU\\\\\\\/fcYXKZh7kLoQlA4ifBpUirLfIaFDkI9NexmnvHO4bfLxeX1297R3yE4Juc6ZXNXAL7oEqtr2cFI3ItBiCf3nNbTiDrTSZx\\\\\\\/4M194bBUngRXFewRmo1JifWWkIaC5e4r2alGa\\\\\\\/QKUTDZrSwsJ4Q\\\\\\\/SdCN\\\\\\\/NvjVQCG8l\\\\\\\/qB7aBHU71OKdfP6w2XAM3tjseZdg60HGlkKq0t28xzWIUPIWd\\\\\\\/7H7KJCW1wgHozieS87t2Cr7RBN4r+ohEOOH0qKCl7Mx2yZWtaINjiQvCdSYeiQL1JBpL9Vh6J\\\\\\\/noRF5sqqcOy8Uwam\\\\\\\/JIWieUV8O3c9NvlxD2Ld6RKIZBBnVVSE6XZx31nGlrxXgH9nrA75lTR2mNAcDBkg4IdutCo9lLM69hJgk3d31Cg+ZRTaZbDmuj4YECStvtyVzg6YdhMkVVc+cnI5RVv39G5wxhbDmWdJvdqOlXnZoFW2QZnOU3eL7+6Y6s4Otj+Nf\\\\\\\/WQzfnDUZNcceTYI467NoYFTj9bq\\\\\\\/DNFkPNBGkIkwBVg6MpGSsarYxJxqPVt6mRsZt4Z53y8lMENJoYokm\\\\\\\/xCkXvNH56nyW+F3sl+yynhcI3tBgksBzZlXuAo6\\\\\\\/eNCBvWD1PiG8h8CkaNK1OHi8oyFgI\\\\\\\/pg0MiMYp\\\\\\\/hTULJughADC0F4CXsLWcxYaCMnj42sO9vy7j5PX\\\\\\\/E4JvJu8NB5TRkDXsWV5pyp0bEHyGKOhtuWGI21WuNYqTknZNiaLObe7\\\\\\\/HoOM7xlgzZDJ2LH9y2nWO2IRLev4I53EHNI2nTOsuP\\\\\\\/9h82TKGH75FLpi6YorzOMmAnRbzSpwHf74ma1OjqC9SPhSXe8byFgW6n+xijcFDqvtirc8uXhF\\\\\\\/6B6odnlNroG9gTrTW+EwIX0CHuY8satYjrWU0hU+7ISvdqx3QvHB+jPE\\\\\\\/7B+MGu7FGCjeXJvLe2GPZbiQ8emexIPQIUICXLwzx8DoKG5I4V7uYyfk9J+GqvXq7szYUkrua0vOmQmWNBvXWCNq9P\\\\\\\/WL7ElmcAvILM3jycArTYAcbtPWLWBmmzkIm1ByZVBxPxpBfotyZk9o7wWtupNN0W6T0\\\\\\\/32LNe\\\\\\\/dcO+Vnx3rgqurFaUGJvPHO62CLaujT7wvU03NCHLojEwKLWOMNa4S8p\\\\\\\/PDtfbMLvSeaQ8gOhRQfeKg3kh23rNx6qQISdT9s3hjhHG7M1VqX+B+gYD7WdkeD5WfdG9+AUVWnCShMdjUEnjRfqaZEG0vNKpWoF3ucGxyGr1XlbjkpejZgqrT0veAe9B2xW1NwuQdJDv0m7YeMxLDEZzqorPbiDQZamWjCW0bzqDrFRxooyseI694rwjAtYqdYeEWcaxpYFIFfrRFys2hCe2r4sx2ZPBc0s6kiUctxlB1J\\\\\\\/+CuX0BoFwsi6OqQRFAIFrdUwQJVzlj5w1rirmE3Y+C\\\\\\\/LHjvWdtRJniw+aINp1uFFu0sGqJ3rgT7luP0crIU+d\\\\\\\/PF8V4G2kLcJp8PS00m81GTtTw4MI54fmlXLr0JLREgAcyzxE16mYZ6wzV+oppznvpWtDPTGwrL7fifXbrnuJklihtI344RWajPnO3wv4wrNarXGpjigFFw3R7C575VNW\\\\\\\/5\\\\\\\/FiuyWJg5F3fpC+fLZIKmGzQ5FK4pXC3uJCfKByT4DCMhvnHqtTA0t7Ekv5eUTQGd\\\\\\\/dQ62V4TPomdAsvo+f\\\\\\\/e7xDbbMMcL1kpUt4AiQqm58O2JsWs8O+PPvxOpl1dN4VsXLjILSdccFp282W6Y4HCsYDGUf28fkPB6oUOTEh3gx88Stt51NejXMjLqndNk0X6XVlG4dFL7K\\\\\\\/+LzP5ejEK7B8zaUYSMIexRcl0dc+7LoH2W2NZlRhTabQ+UmgC5y\\\\\\\/mNOAOoe7nkmCJZC0piXeClGBeFzhjJ4tHNMhdaUgryBIFKQ9gI\\\\\\\/EYIfDfBbXOUxnEVF25v6BhtZTfYu6+cOrRqRHKTY+jsCd26i4UhUxrlATYTaNm1xIZdzIyRNUJBgXXRk5PmU2mEfEzRmuJTmHWrGOznPAljnkEcV0LAz\\\\\\\/+ZqmeOLbzbfOOnPGxR1vM0igfUTpL6\\\\\\\/k\\\\\\\/SOaXz4VgPMzdcMv3ySGuOJBOztZxVaYZAO4feCGKr8K\\\\\\\/foF5oe33kBv8+fQ+Haj4Rq1bVMGom8RZpBdRQ8+ZuB6bXbnW28e14aYnOzzl3u3ExgxrrqheWZ8ToVdetfIQU2linB4wRU65GT3u14yLemq8JsV8Swake6GtaIU2HO1gw5\\\\\\\/WILrjgaEPcS0K\\\\\\\/AiEe0N+q9LfTzRX4ewfaRDa2oe5sgO1tNeQtiHEftMGoTmZkKMOcFPiVbwbVYhguPAbvxTFxwZswZcgM0w8bsxFaoHG10vVfHrMruU66h9qYdC3ifLJeGmBeDE7FLSKQalHx4HzyJwpKO+IeXypMtnpA91paEmzPL3yThP9o0nKe4TLMosKZS+Vyc8ESGrnn4g4Gl9IEuc61Cvg5f180Ztr0A13pkQMJ\\\\\\\/SfHoXKLiW6LEK+GVf1Amj9t7M6Md6c+TRUlphsb8+1QOyeLW7D+XC3IrcYnIU2UAxHA6M0FHqUrpCfxGjElVdgsQ5aI\\\\\\\/ldRswIsbhu7aCMr\\\\\\\/aqPHlVTiG98J+BndC\\\\\\\/hwxiqpQAE57iJpBu40Er0CEhu\\\\\\\/sC8zpfMfYBTXk52acJqI4stuaJp7NomwvyF4itcUKM2a0FSt7FkFv7OKJbjocjSqJ0eDuU\\\\\\\/jkUFKvMsv0M5uZJWRpfhsIZSfIgEr\\\\\\\/bEZnEl7uLdbGRlJABZwqYejt\\\\\\\/etN7cCqLQW80qmSbf19U+HIxQ6yuECFBdgBHcbOYAC1gnokUB+V1XYkoMXyxPcET66vEHMXkfx49xrCd8GlSfmFlDrq\\\\\\\/g8XGcL6Q8R14ZOG8kGTokhoe4J3fMFPavvkG5XVuKoJsSFdof\\\\\\\/4AX0ymltAOayZZWH483KVn+wXG64Ec\\\\\\\/btBYB8VGf8oqpYQZ+ZamOON3d6ior6t5Hpq+Ks7nYgqUn5pVKGuaeAeVlHzAdz3tgrFesLf1Rpbv3wndyXx79RRtrCiROe59sIOnJNWZnpmY5iQXElWgX31DcKb\\\\\\\/TGMa5\\\\\\\/JB+\\\\\\\/Cq2MyruXhnmKwLDeUgROztjNXgUAlDBrWOuHN8h5VxeRjYXhcpTH2aQoMlhZv+DBKjbAQ4acK8mVBgaPGJMl9MuSuVUzYmMjxgJ1tzPf9s1DGF9xT3q3qdtE6QA1iiZ9UoOUOYGnljZLwqjzy34wJH54eMkF2HofjKvI3IK7Oo+q0ojcbhxuLjFQaKmB1RiBFCYChYurVyhSI08eM306\\\\\\\/0JBIqAIlScxr3adHnbWI859HuRxUBxyYI1OXooR1xQEmfmU1c3u4ghSrDD87okn0T0jUJ7XreSCSobQ2nSIVGgwYABEp\\\\\\\/cobq8uFEyABAc6Fzj+q5wki1\\\\\\\/lmHCt8HsSUR2pYhpEKG2U2ADyk6LBlCkox\\\\\\\/nifM7WppgFT3kJEjRPzxvkME1USQPoPNri2\\\\\\\/7jFicT3Bwzk9UWa76z223luTctxAuvx+Sf\\\\\\\/z5fWtGISPK9fnz06nFqeRMjsGpruu\\\\\\\/0wX2pJ42\\\\\\\/rqqLkGW\\\\\\\/GAofyjc\\\\\\\/+eiCzwlJlpbVPaJEjWaAo57FrgaWcxfXZ+6\\\\\\\/3tIFKBB0PPhTqJMM0PY+whxzFZDtwP5tkBV8iMoo+NONrPmX1TCO2a7ESYC7oX+xLcxoATMCwMRKRCgArFIpmF3uz0ECcKuqt3iE2w0BXsNO8iGywMzxbVsIqZViKqU\\\\\\\/jjOi9QNAkJLaXBXo9uGi3xNp5t2JhtTqYCjLAR4Ktm\\\\\\\/br\\\\\\\/56jOlATVjMJBUkO8Hierqztla0qgVPl6rTVXJ7uZwsBFwxhIYTIhMow642jIkdPbmfnTWatKW82Is\\\\\\\/ARB4sfgSXspDvgzPB0IQ1JIimW0JDBXe5iUbtk1IJjblXQ1fYqKJwpUQyq7+xoMF5cbKGN2+IocVH4\\\\\\\/mSLZ9ZZmcTbiO+bMzVQmVNa1Q5q3Qcv\\\\\\\/tkKorlJM\\\\\\\/7inWtQUBF\\\\\\\/AvnbFByGyyCaIAuP6hxZC5w7jGBN\\\\\\\/oDEtC5R0ARMsLSkfHRcYhBJZseEGBsW2npHCW1QYw+8Ygc3G5jL2CrLVOA44TY\\\\\\\/VibZuH6SkKfma7kALCM0G2PXyUvSiFNlni7hPNd9nNZIZQzxiQ+W+WykfGmP6M06TqiWNZtqwfigHXtUfwTHBuNWNIcsKAVc\\\\\\\/+4jNxNT524LyY5usW0FV2riC4Itdfe4f3ktgnR7VnvjBZ7uKqiWOHqD434+FvoRjqAfVrZjmIf2UFQMRlqNzh9Ihq33M7cWDtneaJJMMrHni18Y3NMH\\\\\\\/cg53m\\\\\\\/EbUy6XSEf6ES9xhY7k91ypQKorPAyTLP4Y5XOp5X0EefuYPwyicz\\\\\\\/W6Rbe6hJ3629JWf2otkO82\\\\\\\/\\\\\\\/G7a0NGgZOwU2xQCMFmU6etX92rGp\\\\\\\/DpNRre3xNVgqqxTuki28aks9KXuTpbMyvZ3giLNaJlKZCKEUnwFqR96qaTYCpPUMxS5SP5Nu0nEnv7AQphEmA12VASQkoAi3KA+7RVLQq6grndn6oH2y7Q4vEC6Ky9gLRm\\\\\\\/6UCOI2Zw8fa5ZZKfulignIQwUr7QJxFwmRVy0kcxcsyFbyx77Yi5Q7xDz\\\\\\\/hvirAtJUSncM13HRO2RJc8dMFr+3L8znDHpWbdRrA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022U+GUnpqqU2WPMCj5SZLYjw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f52cbcca4f0cafde6f1d376bea8ab14718e42e62ec5222a2b004d86cf50e9848\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Cmn+H+2Sfp0NHNlvwFwJli6P+zva2A2pm465wKHQqWnsHOOCg51z9LDfEL1CEXexd7giquwf4NrjEPZ1+QYVpK49ejFMJIbVyXUXfHHzh\\\\\\\/5wtrr9IQ9KasrHIzHkALD6u9oVZgfMcbIXcoD2WM1IbvXDc4SG7Ce9vAnUqUR4AxFm+l4X3Fenm9huiSeCGHXf7Frq13HDuReJqYLXNpP0u8rMFZvDpJ5+ImHfzo731d6HprNTKybAZxiwJ2IF74dvUctV4oQgeoCFkpjgNT2APdJlcWEQxdDFbVMjBzbBp7vwguTiYsRfRLJTOsTODONa8XlEWMxXq7Mw9iSmrw3dTIHdRbvMpbsIdsvIqsVOZCbaoISJjRmFxyt1tWDkvxDo3RPVGcUPoopcfwUQwNygfMO63Qij3xbj8GPKLP6LnKsCttvJQoBvxMibD462UqSf2EdyBlAxJobIN0q3nAql\\\\\\\/Ej4XiQCZmURCl5OdMEByj7w3PHb73vPRpjzGbmgR8J8eAHvbCUy+R1SY85TXh5Tky+qEbxGiQdjwVvjlMmht4cA\\\\\\\/SF9AvjoaObFJ24vbzJrcFvlsQ+AmjESUnCHkzj2FvUuiYLg4M6EBcFFWeCugdn7KRnNw6XKDe0hdwDiFsnpZTAtoLm4JZ8kpbBj31wAa8OY8MlghEvOoBWzNuAeR053Zl\\\\\\\/rQ\\\\\\\/GrkKBYHGsZUQFLhvav9CKiXPqSyFZI4nuedazM+80h27BojePzVeIFoLLh\\\\\\\/YLXzfjalMsWCCfb0MNDV+HiUF4wY8qN7zhAFo9Kl1fU3+TVk0cZV7knogue393qvCuN6\\\\\\\/P7x87l5Afzg8oBaM+3D16G7x\\\\\\\/EppvFAF6HlvoQGjs+vt5CMVjy4Kg5AkO7a5LyHACDc86rrH2MwpWuz2ldJMDezpsqsgYzrjQH\\\\\\\/jWdHWhCIRixynPXzthaIoBxsrnjbH8KuiDvFDD4g8Ym6ONN4FjAl40o3hfyodxeYNZ1J7IOdZCn+C6nU3EVpuNoToLpkeBJ2wyh2eRiFn2gNgI1cdnivB1Zt7VjVwstNLY8HfTUMvEB1BK5Ieb49eDBQm5hYD1dPZ7Cqfy7\\\\\\\/bg4jl8t9zIfN\\\\\\\/YTkhMAW9w7LC0dZkdkri9sDqUUvJozyVrHxukZ9HmhizOcUrOyWPdFeBNli5TcnFY0A\\\\\\\/py67AJ6jErIMjpgGvQV93cw3C7zOGhRmX4GW9axWKa796U4jRtoTJOmNt8QDHJxnjmBKpQJ4UNROqKOLUFfmff49JhTnxFEiD85Tohze1a\\\\\\\/kf+IcETbAyODHOa2huy4d\\\\\\\/iaoiairRMBmh6hvDGet0reiA6GMeO\\\\\\\/tLkl\\\\\\\/SO59CJdT7Sg6thqXUbuCK+jzAJMLolxfvnqhPE2n9kTnki0sB4Kk4OXLwuNXdOoNBe18wcNUy2l1ZjPlJuuWFVBSlDZBY84V2TRyQ\\\\\\\/7nqoqggaMW+dpfsYn1gnXTuXr918KcvsJmmV1TDBNY4DjKfccjeTR9FM6oYjIqb89GNU6YERT\\\\\\\/bbwT9N3VutpugtSnUGTa6yBZDZ8T9uFZMGc00IoETM8L3Fk5AQ\\\\\\\/y1N2Hlh07Pp0hjz5HEvIcn+15+sTzjpYag+z7WNGRsmuhiP81jzFTLaGz\\\\\\\/CfBYUWN2EhjXZTZ4gGD0JCfdIe7kYx94XZVSnQ6TUXME9cR\\\\\\\/6lt+Myrtcg0YXK\\\\\\\/DPAFMJixKMgvE15K07HWIjwZpj6RdZaCanGcBf+xtugUitJBUqSZ2Bp\\\\\\\/VAkTAM6MfCzGxFqtu861Rmq5ZFXPbAC3GtM4fmQe+BSXYP5k0lOGdv+EDXUHh\\\\\\\/1gra0j8JGY977ICC\\\\\\\/IqIaNYGS\\\\\\\/xdpB4heUmt7gZ4uFBUyKW6JbIhdfo4AE9tFjYXzKZ51C00BIq3EBEjanGfyndJ67XC63v758fePHXJMzd\\\\\\\/Pu2Ks6MrhDBq\\\\\\\/YheOdoANvdxLVxagCoqBQpqhLJA\\\\\\\/S6KJ6OfK8pEebEGy0zDLadMZ1hLdO0vUdhph2rXQMgPDgeK3pg2p43FB2C8NQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00224+EGNzldNzysDdNa4dJBzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002219d4404dbe37a8c4c77fce1f1a24e194e523126220f54d70c161a00a5bc4b0ed\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489477 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1755" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4BpWBjCAT0UTA3u3mecxi49Zzgbbm4Y__-TvKZW-fvk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1755" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-V3QjANbJq1T9AzWrDjGgLUITAZe3HOE4_hlVnVBGm8M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1747" title="Site-Specific Fat-1 Knock-In Enables Significant Decrease of n-6PUFAs/n-3PUFAs Ratio in Pigs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1771" title="A Bayesian Genomic Regression Model with Skew Normal Random Errors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489477" id="node489477" data-pisa="ggg;8/5/1755" data-pisa-master="ggg;g3.117.300512" data-apath="/ggg/8/5/1755.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3456-7553" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tomotaka Matsumoto</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5325-6999" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hiroshi Akashi</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1755-1769;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300512 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489477" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tomotaka Matsumoto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomotaka%2BMatsumoto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matsumoto%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATomotaka%2BMatsumoto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3456-7553" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tomotaka Matsumoto</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hiroshi Akashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BAkashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akashi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHiroshi%2BAkashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5325-6999" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hiroshi Akashi</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hiakashi@nig.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiakashi@nig.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/5/1755" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489477" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489477/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1755.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489477" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489477/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1755.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489477" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489477/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1755.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489477" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489477/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/5/1755.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489477" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489477/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Inferred ancestral nucleotide states are increasingly employed in analyses of within- and between -species genome variation. Although numerous studies have focused on ancestral inference among distantly related lineages, approaches to infer ancestral states in polymorphism data have received less attention. Recently developed approaches that employ complex transition matrices allow us to infer ancestral nucleotide sequence in various evolutionary scenarios of base composition. However, the requirement of a single gene tree to calculate a likelihood is an important limitation for conducting ancestral inference using within-species variation in recombining genomes. To resolve this problem, and to extend the applicability of ancestral inference in studies of base composition evolution, we first evaluate three previously proposed methods to infer ancestral nucleotide sequences among within- and between-species sequence variation data. The methods employ a single allele, bifurcating tree, or a star tree for within-species variation data. Using simulated nucleotide sequences, we employ ancestral inference to infer fixations and polymorphisms. We find that all three methods show biased inference. We modify the bifurcating tree method to include weights to adjust for an expected site frequency spectrum, “bifurcating tree with weighting” (BTW). Our simulation analysis show that the BTW method can substantially improve the reliability and robustness of ancestral inference in a range of scenarios that include non-neutral and/or non-stationary base composition evolution.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ancestral-reconstruction" class="hw-term hw-article-keyword hw-article-keyword-ancestral-reconstruction" rel="nofollow">ancestral reconstruction</a></li><li class="kwd"><a href="/keyword/codon-usage" class="hw-term hw-article-keyword hw-article-keyword-codon-usage" rel="nofollow">codon usage</a></li><li class="kwd"><a href="/keyword/gc-content" class="hw-term hw-article-keyword hw-article-keyword-gc-content" rel="nofollow">GC content</a></li><li class="kwd"><a href="/keyword/unfolded-site-frequency-spectrum" class="hw-term hw-article-keyword hw-article-keyword-unfolded-site-frequency-spectrum" rel="nofollow">unfolded site frequency spectrum</a></li><li class="kwd"><a href="/keyword/nucleotide-substitution" class="hw-term hw-article-keyword hw-article-keyword-nucleotide-substitution" rel="nofollow">nucleotide substitution</a></li></ul><p id="p-3">Knowledge of ancestral sequences allows inference of changes on each branch of a phylogenetic tree and can be highly useful for evolutionary analyses. In some cases, ancestral DNA samples can be sequenced directly, but for the vast majority of studies, ancestral states must be inferred statistically. Several methods have been developed to infer states at ancestral nodes based on likelihood calculation assuming a phylogenetic tree (<em>e.g.</em>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Yang <em>et al.</em> 1995</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Koshi and Goldstein 1996</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Pagel <em>et al.</em> 2004</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yang 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>). These methods can incorporate complex transition rate matrices and allow accurate inference of ancestral states for nucleotide and amino acid sequences in a range of evolutionary scenarios. However, transition rate matrix/fixed tree (TRMFT) approaches require knowledge of phylogenetic relationships among genome sequences and assume that the same relationships hold across sites (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Yang 2006</a>). This restriction is problematic for ancestral inference for sequences taken from a single population because recombination can cause different genomic regions to have different genealogies.</p><p id="p-4">Determining patterns and testing the causes of base composition evolution often relies on ancestral inference (<em>e.g.</em>, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akashi 1995</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kliman 1999</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Begun 2001</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Takano-Shimizu 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Duret <em>et al.</em> 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gaucher <em>et al.</em> 2003</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Groussin and Gouy 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Aoki <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Terekhanova <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Glémin <em>et al.</em> 2015</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bolívar <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Clément <em>et al.</em> 2017</a>). Previous studies have revealed fixation biases (weak selection and/or biased gene conversion) elevating GC content (<em>e.g.</em>, <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akashi 1995</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">1999</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kliman 1999</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Maside <em>et al.</em> 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Galtier <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Haddrill and Charlesworth 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Muyle <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lachance and Tishkoff 2014</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Glémin <em>et al.</em> 2015</a>) and have shown strong lineage-dependence in base composition evolution (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akashi 1996</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Duret <em>et al.</em> 2002</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Powell <em>et al.</em> 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Akashi <em>et al.</em> 2006</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nielsen <em>et al.</em> 2006</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Singh <em>et al.</em> 2009</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Groussin and Gouy 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lartillot 2012</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Weber <em>et al.</em> 2014</a>). Although long-term evolutionary analyses of base composition have incorporated sophisticated models using TRMFT ancestral inference methods, population genetic analyses of base composition evolution have employed a limited number of approximate methods for ancestral inference. <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Akashi (1995)</a> and <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Akashi and Schaeffer (1997)</a> inferred ancestral states of nucleotide sequences in <em>Drosophila</em> species using maximum parsimony (<em>i.e.</em>, assigning ancestral states to minimize the numbers of evolutionary changes required to explain the observed data). <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clemente and Vogl (2012)</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Poh <em>et al.</em> (2012)</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Bolívar <em>et al.</em> (2015)</a> and <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Clément <em>et al.</em> (2017)</a> employed a similar approach but used only sites at which all outgroup species have the same state. In such cases, they assumed that the outgroup state is the ancestral condition in order to estimate the number of fixations and site frequency spectrum (SFS) of GC content-altering mutations.</p><p id="p-5">Other studies have employed likelihood methods for ancestral inference assuming a common genealogy for all sites. <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kern and Begun (2005)</a> assigned a star tree to samples from a single species. In contrast, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Akashi <em>et al.</em> (2006)</a> and <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Matsumoto <em>et al.</em> (2016)</a> assigned nucleotide states segregating within species to bifurcating lineages within each species. Finally, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jackson <em>et al.</em> (2017)</a> employed single sequences from each sampled species to infer ancestral states and assumed identity of ancestral states at within and between species nodes. These approaches allow TRMFT ancestral inference with complex models (<em>e.g.</em>, different transition probabilities among 12 nucleotide mutation classes).</p><p id="p-6">Assuming a single gene tree for polymorphism data is unrealistic for recombining regions and the reliability of the ancestral inference approaches described above for within- and among-species data are largely unknown (but see <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hernandez <em>et al.</em> (2007)</a> for maximum parsimony approach). Some studies have proposed Bayesian methods to consider phylogenetic uncertainty giving a prior distribution of phylogeny (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Huelsenbeck and Bollback 2001</a>, <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Pagel <em>et al.</em> 2004</a>). However, these methods were developed to address uncertainty in deep phylogenies and their applicability to within species variation is unclear (the number of possible genealogies for a given region can be very large and trees can differ among regions). Previously, we implemented the GTR-NH<sub>b</sub> model which allows lineage-specific transition rates and appears to be a robust approach to inferring ancestral states under several evolutionary scenarios of base composition (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>). This method is currently only available in the BASEML software package which employs a TRMFT ancestral inference method (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Yang 2007</a>).</p><p id="p-7">In this study, we focus on TRMFT likelihood-based ancestral inference methods to study base composition evolution within and among closely related species. We use BASEML for ancestral inference and examine the accuracy of estimations of the numbers of fixations, polymorphic mutations and SFS using the methods that assign a single tree to within species polymorphism. We evaluated the accuracy of the three methods explained above. For simplicity, we refer to the methods in <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jackson <em>et al.</em> (2017)</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Matsumoto <em>et al.</em> (2016)</a> and <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kern and Begun (2005)</a> as “single allele” (SA), “bifurcating tree (BT)” and “star tree” (ST), respectively. We also introduce a new approach that incorporates allele frequency information in the inference of ancestral states at polymorphic sites (“bifurcating tree with weighting (BTW)”)</p><p id="p-8">In order to access the reliability of the methods, we generated nucleotide sequences by computer simulation assuming neutral or nearly-neutral base composition evolution. Using nucleotide sequences generated for six species, we employed the tree assignment methods described above and conducted ancestral inference under a likelihood based method assuming a complex nucleotide substitution model, GTR-NH<sub>b</sub>. For these data, SA, BT, and ST approaches suffer from considerable estimation bias. However, the BTW method using the expected SFS under neutral equilibrium (BTW<sub>ne</sub>) allows accurate estimation of the SFS in several scenarios of base composition evolution and this accuracy is relatively robust to the fit of the expected SFS. In addition, even when the actual SFS shows strong departure from the SFS under neutral equilibrium, an iterative approach to BTW substantially increases the accuracy of the estimation. Our analyses support that ancestral inference using the approximate BTW method may be sufficiently reliable for within-population analyses of the base composition of non-coding regions, introns or codon usage bias in a wide range of evolutionary scenarios. However, we point out some limitations of models that do not incorporate population genetic processes, especially ancestral polymorphism.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><p id="p-9">We describe below (1) the computer simulation to generate nucleotide sequences, (2) the process of inferring polymorphisms and fixations from ancestral inference results, and (3) the four ancestral inference methods evaluated in this study. Acronyms used in the text are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/489722/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/489722/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Key acronyms used in the text</span><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Computer simulation</h3><p id="p-10">Computer simulations were designed to emulate base composition evolution among six species using a tree and approximate distances for the <em>Drosophila melanogaster</em> subgroup (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In the discussion below, “<em>a-b</em> lineage” will refer to the lineage connecting nodes <em>a</em> and <em>b</em> and “<em>a</em> node (population)” will refer to node (or population) <em>a</em> using the node names in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Forward-running simulations were conducted similarly to <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Matsumoto <em>et al.</em> (2016)</a> except we consider a four- (rather than two-) state model of base composition. Each simulation was initiated with a “burn-in” process to allow populations to reach equilibrium base composition and a stationary frequency distribution for polymorphic sites. Burn-in parameters were set to: population size <em>N</em> = 1,000, mutation rates between any two nucleotides <em>u</em> = 2.5×10<sup>−6</sup>, and recombination rate between neighboring sites in each mating pair <em>r</em> = 0.01. Recombination rates of <em>Nr</em> = 10 give results close to free recombination expectations (Figure S1 in <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Matsumoto <em>et al.</em> 2016</a>). We modeled haploid genomes with 10<sup>5</sup> nucleotide sites. Evolution proceeds in discrete generations and in some scenarios, we assigned higher viability to individuals with elevated GC content with fitness of individual <em>j</em> having <em>x</em> A or T nucleotides calculated as <em>f<sub>j</sub></em> = (1 - <em>s</em>)<em><sup>x</sup></em>. After viability selection, sequences undergo mutation and random pairing/recombination and the process is repeated with the resulting sequences. We will refer to selection coefficients and GC contents at the <em>mstyeo</em> node (end of burn-in) as <em>s</em><sub>0</sub> and GC<sub>0</sub>, respectively. Three values of GC<sub>0</sub> were employed: 0.5 (<em>s</em><sub>0</sub> = 0), 0.7 (<em>s</em><sub>0</sub> = 0.00043) and 0.9 (<em>s</em><sub>0</sub> = 0.0011). Note that evolutionary dynamics under this scenario of weak selection favoring GC are indistinguishable from expectations under <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nagylaki’s (1983)</a> model of GC-biased gene conversion without selection. We will employ the term “fixation bias” to include selection and/or biased gene conversion favoring GC.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships and evolutionary scenarios used to generate simulated sequences. Tree is a simplified depiction of relationships among six D. melanogaster subgroup species (Ko et al. 2003). Two selection schemes, stationary and fixation bias change scenarios were considered in the simulations. Lineage-specific selection intensities are expressed by different line-formatting in the phylogeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1098483939" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic relationships and evolutionary scenarios used to generate simulated sequences. Tree is a simplified depiction of relationships among six &lt;em&gt;D. melanogaster&lt;/em&gt; subgroup species (Ko &lt;em&gt;et al.&lt;/em&gt; 2003). Two selection schemes, stationary and fixation bias change scenarios were considered in the simulations. Lineage-specific selection intensities are expressed by different line-formatting in the phylogeny.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1755/F1.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/5/1755/F1.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489651" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Phylogenetic relationships and evolutionary scenarios used to generate simulated sequences. Tree is a simplified depiction of relationships among six <em>D. melanogaster</em> subgroup species (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ko <em>et al.</em> 2003</a>). Two selection schemes, stationary and fixation bias change scenarios were considered in the simulations. Lineage-specific selection intensities are expressed by different line-formatting in the phylogeny.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">The initial base composition of all sequences was set to expected equilibrium values at the start of each burn-in and lineages were evolved for 20,000 generations. The resulting populations were used in downstream simulations. When a simulation reached time points for lineage splitting (internal <em>ms</em>, <em>ty</em>, <em>eo</em>, <em>tyeo</em>, <em>mstyeo</em>, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>), the population of the ancestral lineage was duplicated to create initial populations for the two descendent populations (ancestral polymorphism can segregate in the derived lineages but there is no gene flow after the split). We recorded sequence changes throughout the simulation to track actual numbers of fixations and polymorphic mutations, and SFS. 10 sequences were randomly chosen from each population shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> to test inference methods. Some of the analyses employed subsets or alterations of the samples as explained below.</p><p id="p-13">The two main evolution scenarios considered in this study are shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>. The first is a stationary process in which all lineages share burn-in parameter values. Lineage lengths were set to give rough correspondence to synonymous divergence within the <em>D. melanogaster</em> subgroup (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Akashi <em>et al.</em> 2007</a>): 3,610 generations for <em>mstyeo-tyeo</em> lineage, 10,840 generations for <em>mstyeo-ms</em> lineage, and 7,230 generations for all other lineages give expected sequence divergences of approximately 2.5%, 7.5% and 5%, respectively, in the stationary scenario with GC<sub>0</sub> = 0.7. In our simulations, the expected time to the within-population most recent common ancestor (MRCA) is 2<em>N</em> = 2000 generations (for stationary, neutral evolution). Lineage lengths in these simulations are much greater than the expected time to MRCA and shared ancestral polymorphism should be rare among terminal node populations. The second scenario models non-stationary evolution, in which the <em>mstyeo-ms</em>, <em>tyeo-ty</em> and <em>ty-t</em> lineages are affected by stronger selection (2<em>s</em><sub>0</sub>) and <em>ms-m</em>, <em>ty-y</em>, <em>eo-o</em> and <em>mstyeo-tyeo</em> lineages are affected by weaker selection (<em>s</em><sub>0</sub>/3) compared with the burn-in values (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). Other parameter values are identical to those in the stationary scenario. We refer to these non-stationary simulations as “fixation bias change” scenarios.</p><p id="p-14">We also simulated non-stationary evolution with demographic changes. Two scenarios consider population size reductions in the <em>ms-m</em> lineage (demA: population size reduced to 100 at 7220<sup>th</sup> generation, demB: population size reduced to 100 at 7130<sup>th</sup> generation). The other two scenarios experience population size expansions in the <em>ms-m</em> lineage (demE: population size increased to 10000 at 7130<sup>th</sup> generation and demF: population size increased to 10000 at 6230<sup>th</sup> generation). The SFSs in the <em>m</em> population of these scenarios showed large deviation from the SFS under neutral equilibrium. Parameters other than population size were identical to those used in the stationary scenario for GC<sub>0</sub> = 0.7. The dem scenarios are similar to the depiction in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a> except for changes in <em>N</em> on the <em>ms-m</em> lineage. In this study, we focused on whether the deviations from the SFS under neutral equilibrium caused by the demographic change scenarios affect the accuracy of ancestral inference. We considered other demographic scenarios that have smaller impacts on the SFS (see Figure S7 and Table S2).</p><p id="p-15">We conducted a computer simulation for each parameter set to produce data sets to test ancestral inference methods. Ancestral inference employed the species phylogeny shown in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a> and we inferred probabilities of ancestral states using BASEML under the GTR-NH<sub>b</sub> model (except where noted otherwise). For a given simulation, we replicated sampling and ancestral inference 100 times (all steps in the ancestral inference process were conducted independently for each replicate) to estimate confidence intervals on counts of mutations in three categories, <em>pu</em> (preferred to unpreferred, GC→AT, G or C to A or T), <em>up</em> (AT → GC) and <em>pp+uu</em> (GC → GC and AT → AT). “Polymorphic” or “segregating” sites will refer to sites at which multiple states are found within a population sample. “Fixations” will refer to derived states that are shared among the sampled alleles (including mutations common to the entire population as well as those shared within the sample but not the entire population). Fixations include mutations that arose after the split with a sister species as well as mutations that pre-date species splitting (ancestral polymorphism). For example, among fixations on the <em>ms-m</em> lineage, the majority originated after the <em>ms</em> node but a substantial fraction originated prior to this ancestral node (in the stationary scenario with GC<sub>0</sub> = 0.5, this fraction was about 28%).</p></div><div id="sec-3" class="subsection"><h3>Reconstructing polymorphisms and fixations</h3><p id="p-16">We conducted ancestral inference at internal nodes using the sequence samples and trees as input. Unless noted otherwise, all analyses employed the GTR-NH<sub>b</sub> transition rate model implemented in BASEML software (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Yang 2007</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>) and ancestral reconstructions were used to infer polymorphisms and fixations. Following <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> (2015)</a>, we ran BASEML 10 times and employed only the result with the highest likelihood. For inference of within-species polymorphisms, we compared estimated joint probabilities of ancestral states (probability for a set of ancestral states at all internal nodes) by BASEML and states of the 10 sequences sampled from the population. For example, consider a case where nine individuals have nucleotide A and one individual has nucleotide C at a given site among the sampled sequences. If the estimated ancestral states at this polymorphic site are A and C with probabilities 0.6 and 0.4, respectively, we counted 0.6 A → C polymorphic mutations in frequency class 1 (singletons) and 0.4 C → A polymorphic mutations in frequency class 9. Repeating this process across all sites, we estimated the number of polymorphic mutations and their SFS. Sites at which more than two nucleotides are segregating were filtered in all analyses; the proportion of such sites was &lt; 1% among all sites and 5–7% among polymorphic sites.</p><p id="p-17">Ancestral inference under GTR-NH<sub>b</sub> allows estimation of the number of fixations considering lineage and mutation-category specific parameters. We examined the accuracy of the estimation of the number of fixations as follows. If the estimated states at ancestral node <em>a</em> and descendant node <em>b</em> are T and G with probabilities 0.6 and A and C with probability 0.4, respectively, we “counted” 0.6 T → G and 0.4 A → C fixations on the <em>a-b</em> lineage. This method, referred to as AWP (Averaging Weighted by Probability), slightly underestimates the numbers of fixations when multiple hits occur at a same site within a lineage (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>). In this study, we used AWP only for the BTW method and for other methods, we used EMC (Expected Markov Counting) implemented in BASEML in which the number of substitutions is counted over the Markov chain process of substitution considering the change of base composition along a branch (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>).</p></div><div id="sec-4" class="subsection"><h3>Ancestral inference method 1: Single allele (SA)</h3><p id="p-18">We performed SA analyses using a single sequence sampled from each population as described in <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Jackson <em>et al.</em> (2017)</a>. First, we randomly sampled 10 sequences from each population in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a> and then randomly selected one sequence for the ancestral inference step. We assumed that the states at the <em>ms</em>, <em>ty</em> and <em>eo</em> nodes are the ancestral states at polymorphic sites in the (<em>m</em> and <em>s</em>), (<em>t</em> and <em>y</em>), and (<em>e</em> and <em>o</em>) populations respectively (<em>i.e.</em>, in the <em>ms</em> clade, we assumed no fixations after the split between the <em>ms-m</em> and <em>ms-s</em> lineages). We estimated the numbers of polymorphic mutations and their SFS in the <em>a</em> population by comparing the inferred ancestral states at the <em>ab</em> node and the observed states in the sample of 10 sequences from the <em>a</em> population as explained above. We estimated the numbers of fixations on the <em>ab-a</em> lineage using the EMC approach.</p></div><div id="sec-5" class="subsection"><h3>Ancestral inference method 2: Bifurcating tree (BT)</h3><p id="p-19">In the BT method, we constructed a pair of representative sequences (referred to as “collapse-pair” sequences) from the population samples for each species. The detailed process is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. For sites that are monomorphic within the species sample, the same nucleotide was assigned to both collapse-pair sequences. At polymorphic sites, the two segregating nucleotides were assigned randomly to the collapse-pair sequences. Note that site frequency information is lost and associations among sites are randomized in this approach. The phylogeny shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> was assigned to 2×6 collapse-pair sequences and we inferred probabilities of ancestral states at the nodes representing the common ancestors of the collapse pair sequences (<em>e.g.</em>, <em>m</em>’, <em>s</em>’, <em>t</em>’, <em>y</em>’, <em>e</em>’ and <em>o</em>’). In the BT and other methods below, we compared the inferred ancestral states at the <em>X</em>’ node with observed states among the 10 sampled sequences to estimate the numbers of polymorphic mutations and their SFS in the <em>a</em> population. We estimated the numbers of fixations on the <em>ab-a</em>’ lineage using the EMC approach.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Process for creating input data for bifurcating tree (BT) ancestral inference. (A) The process to make two &#x201C;collapse-pair&#x201D; sequences in the BT method. (B) phylogeny used in the BT method. Node names are from Figure 1. x&#x2019; shows the MRCA of population (species) x. x1 and x2 are collapse-pair sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1098483939" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Process for creating input data for bifurcating tree (BT) ancestral inference. (A) The process to make two &#x201C;collapse-pair&#x201D; sequences in the BT method. (B) phylogeny used in the BT method. Node names are from Figure 1. &lt;em&gt;x&lt;/em&gt;&#x2019; shows the MRCA of population (species) &lt;em&gt;x. x&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; and &lt;em&gt;x&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; are collapse-pair sequences.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1755/F2.medium.gif" width="440" height="425"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/5/1755/F2.medium.gif" width="440" height="425"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Process for creating input data for bifurcating tree (BT) ancestral inference. (A) The process to make two “collapse-pair” sequences in the BT method. (B) phylogeny used in the BT method. Node names are from <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>. <em>x</em>’ shows the MRCA of population (species) <em>x. x</em><sub>1</sub> and <em>x</em><sub>2</sub> are collapse-pair sequences.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Ancestral inference method 3: Star tree (ST)</h3><p id="p-21">For ST inference, we assigned a star phylogeny (a single root node for within-species data) to each sample of 10 sequences for ancestral inference. To reduce computational time, we applied this method only to the <em>m</em> population. For each of the other five terminal nodes (species), we employed a single sequence consisting of the ancestral nucleotide states of each site (<em>s</em>’, <em>t</em>’, <em>y</em>’, <em>e</em>’ and <em>o</em>’ nodes in Figure S1). Note that this effectively reduces terminal lineage branch lengths compared to input data used for the other inference methods. Using the phylogeny shown in Figure S1, we inferred ancestral states at node <em>m</em>’ and estimated the numbers and SFS of polymorphic mutations. We also estimated the numbers of fixations on the <em>ab-a</em>’ lineage using the EMC approach.</p></div><div id="sec-7" class="subsection"><h3>Ancestral inference method 4: Bifurcating tree with weighting (BTW)</h3><p id="p-22">The BT method does not incorporate allele frequency information within the sampled nucleotide sequences and we expected inference error and/or biases to arise from effectively assuming that all site frequencies are 0.5. We employed a straightforward approach to weight probabilities of ancestral configurations using an expected SFS. We refer to this approach as the BTW method. In most cases, we used an expected SFS for neutral mutations sampled randomly from a population at equilibrium (SFS<sub>ne</sub>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fisher 1930</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wright 1969</a>) for weighting. Consider a site configuration with <em>i</em> nucleotides in state <em>A</em><sub>1</sub> and <em>n</em> - <em>i</em> nucleotides in state <em>A</em><sub>2</sub>. The BT approach gives ancestral probabilities <em>P</em><sub>BT</sub>(<em>A</em><sub>1</sub>) and <em>P</em><sub>BT</sub>(<em>A</em><sub>2</sub>) for ancestral states <em>A</em><sub>1</sub> and <em>A</em><sub>2</sub>, respectively, at this site. We weight these probabilities using the SFS<sub>ne</sub>. We calculated <em>F</em>(<em>i</em>) and <em>F</em>(<em>n</em> - <em>i</em>), the expected proportions of polymorphic sites with derived states with counts of <em>i</em> and <em>n</em> - <em>i</em> in a sample of <em>n</em> sequences respectively using equation [30] in <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sethupathy and Hannenhalli (2008)</a>. We calculated the expected proportion of sites where nucleotide <em>A</em><sub>1</sub> is the ancestral state, <em>P<sub>ne</sub></em>(<em>A</em><sub>1</sub>) = <em>F</em>(<em>i</em>) / [<em>F</em>(<em>i</em>) + <em>F</em>(<em>n</em> - <em>i</em>)] and the expected proportion of sites where nucleotide <em>A</em><sub>2</sub> is the ancestral state <em>P<sub>ne</sub></em> (<em>A</em><sub>2</sub>) = <em>F</em>(<em>n</em> - <em>i</em>) / [<em>F</em>(<em>i</em>) + <em>F</em>(<em>n</em> - <em>i</em>)]. The weighted joint probabilities of ancestral configurations are <em>P<sub>BTW</sub></em>(<em>A</em><sub>1</sub>) = <em>P</em><sub>BT</sub>(<em>A</em><sub>1</sub>) <em>P<sub>ne</sub></em>(<em>A</em><sub>1</sub>) and <em>P<sub>BTW</sub></em>(<em>A</em><sub>2</sub>) = <em>P</em><sub>BT</sub>(<em>A</em><sub>2</sub>) <em>P<sub>ne</sub></em>(<em>A</em><sub>2</sub>). If the estimated ancestral state differs from both segregating nucleotides, we set the probability (<em>P<sub>BTW</sub></em>) to zero (note that such sites were very rare and showed low probabilities in the bifurcated tree method, in most case ≈ 0.01). We then adjusted joint probabilities so that, at each site, the probabilities of possible ancestral configurations sum to one. We refer to the BTW method using SFS<sub>ne</sub> as BTW<sub>ne</sub>.</p><p id="p-23">We also examined an iterative approach to SFS estimation. An initial round of ancestral state/SFS inference is conducted under BTW with an arbitrary SFS for weighting (<em>e.g.</em>, BTW<sub>ne</sub> using SFS<sub>ne</sub>). The resulting SFS is employed for weighting in the next round of BTW analyses and this process is repeated. We refer to this method as iterative BTW<sub>est</sub>. Note that we used the AWP approach to estimate the number of fixations because the BTW methods adjusts joint probabilities at ancestral states by SFS weighting and the EMC approach is not applicable.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-24">Computer programs and input files used to simulate sequence evolution, generate collapse-pair sequences and conduct BTW analysis are available at <a href="https://github.com/tomotakamatsumoto/Matsumoto_Akashi_AIpoly_codes">https://github.com/tomotakamatsumoto/Matsumoto_Akashi_AIpoly_codes</a>. Open source license information is available at <a href="https://opensource.org/licenses/MIT">https://opensource.org/licenses/MIT</a>. Figure S1 shows the phylogeny used in the ST method. Figures S4 and S7 show the actual SFS’s for the simulation scenarios. Table S14 shows the comparison of the SFSs estimated under BTW and the method proposed in <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Keightley <em>et al.</em> (2016)</a>. Other supplementary figures and tables show the accuracy of fixation and polymorphism inference under several evolutionary scenarios and methods. All supplementary figures and tables are provided as a single file. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.5996342">https://doi.org/10.25387/g3.5996342</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Stationary scenarios</h3><p id="p-25">We first examined the performance of three previously suggested ancestral inference methods that have been employed to estimate the numbers polymorphic mutations and their SFS and the numbers of fixations among closely related species. We discuss results for stationary base composition scenarios without selection (GC<sub>0</sub> = 0.5) and with selection (GC<sub>0</sub> = 0.7) for the bifurcating tree (BT) and star tree (ST) methods because these approaches showed contrasting biases in the estimation of SFS. We then compare results for the single allele (SA) method.</p><p id="p-26">The BT method biases the estimated SFS toward a “flat” (uniform) distribution. For the <em>m</em> population in GC<sub>0</sub> = 0.5, this bias causes underestimation and overestimation of the proportions of polymorphic mutations in low and high frequency classes, respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). Such errors bias SFS toward patterns consistent with positive selection or deviation from equilibrium after population size reduction. We measured goodness of fit between estimated and actual SFS to compare the estimation accuracy among methods (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, Table S1). Accuracy of the estimation of the total numbers of polymorphic mutations was examined separately (Figure S2). BT inference results in substantial differences between estimated and actual SFS for all mutation categories both at GC<sub>0</sub> = 0.5 and GC<sub>0</sub> = 0.7.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Actual vs. estimated numbers of polymorphic pu mutations for each frequency class. Results for four inference methods: BT, ST, SA and BTWne are shown. The legends applies across graphs. The simulation assumed stationary evolution with GC0 = 0.5 and results are shown for the m population. Population sampling and following ancestral inference were replicated 100 times. The figures show the average and the 95% confidence interval among the replicates. The scales of unlabeled axes are shared across graphs in the same columns and in the same rows. This standard applies to all following figures." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1098483939" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Actual &lt;em&gt;vs.&lt;/em&gt; estimated numbers of polymorphic &lt;em&gt;pu&lt;/em&gt; mutations for each frequency class. Results for four inference methods: BT, ST, SA and BTW&lt;sub&gt;ne&lt;/sub&gt; are shown. The legends applies across graphs. The simulation assumed stationary evolution with GC&lt;sub&gt;0&lt;/sub&gt; = 0.5 and results are shown for the &lt;em&gt;m&lt;/em&gt; population. Population sampling and following ancestral inference were replicated 100 times. The figures show the average and the 95% confidence interval among the replicates. The scales of unlabeled axes are shared across graphs in the same columns and in the same rows. This standard applies to all following figures.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1755/F3.medium.gif" width="417" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/5/1755/F3.medium.gif" width="417" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489530" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Actual <em>vs.</em> estimated numbers of polymorphic <em>pu</em> mutations for each frequency class. Results for four inference methods: BT, ST, SA and BTW<sub>ne</sub> are shown. The legends applies across graphs. The simulation assumed stationary evolution with GC<sub>0</sub> = 0.5 and results are shown for the <em>m</em> population. Population sampling and following ancestral inference were replicated 100 times. The figures show the average and the 95% confidence interval among the replicates. The scales of unlabeled axes are shared across graphs in the same columns and in the same rows. This standard applies to all following figures.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of ancestral inference methods for estimating the SFS of polymorphic mutations in stationary GC content scenarios. goodness of fit statistics were calculated using actual vs. estimated numbers of polymorphic mutations for each mutation category. In each frequency class, the proportions of actual polymorphic mutations were used to calculate &#x201C;expected&#x201D; values to compare to the inferred numbers of polymorphisms (&#x201C;observed&#x201D; values).  statistics were calculated for each replicate with these expected and observed values. The gray scales and cell values give the numbers of replicates showing &#x201C;poor&#x201D; fits between observed and expected values ( &#x2265; 13.0) and &#x201C;good&#x201D; fits ( &#x2264; 3.5). Low and high  cutoffs correspond to P &#x2265; 0.9 and &#x2264; 0.1 for  goodness of fit tests with the degree of freedom = 8. Note that  values strongly depend on the number of polymorphic mutations; results are comparable among different methods for the same simulation scenario and mutation category, but are only comparable among different scenarios or mutation categories if their sample sizes are similar. The numbers of polymorphic mutations in each scenario and mutation category are shown in Table S1. The SFS of the m population was estimated under four inference methods: BT, ST, SA and BTWne. The simulation assumed stationary evolution with GC0 = 0.5 and 0.7. Population sampling and ancestral inference were replicated 100 times." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1098483939" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Performance of ancestral inference methods for estimating the SFS of polymorphic mutations in stationary GC content scenarios. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-1&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;goodness of fit statistics were calculated using actual &lt;em&gt;vs.&lt;/em&gt; estimated numbers of polymorphic mutations for each mutation category. In each frequency class, the proportions of actual polymorphic mutations were used to calculate &#x201C;expected&#x201D; values to compare to the inferred numbers of polymorphisms (&#x201C;observed&#x201D; values). &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-2&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; statistics were calculated for each replicate with these expected and observed values. The gray scales and cell values give the numbers of replicates showing &#x201C;poor&#x201D; fits between observed and expected values (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-3&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &#x2265; 13.0) and &#x201C;good&#x201D; fits (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-4&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &#x2264; 3.5). Low and high &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-5&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; cutoffs correspond to &lt;em&gt;P&lt;/em&gt; &#x2265; 0.9 and &#x2264; 0.1 for &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-6&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; goodness of fit tests with the degree of freedom = 8. Note that &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-7&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; values strongly depend on the number of polymorphic mutations; results are comparable among different methods for the same simulation scenario and mutation category, but are only comparable among different scenarios or mutation categories if their sample sizes are similar. The numbers of polymorphic mutations in each scenario and mutation category are shown in Table S1. The SFS of the &lt;em&gt;m&lt;/em&gt; population was estimated under four inference methods: BT, ST, SA and BTW&lt;sub&gt;ne&lt;/sub&gt;. The simulation assumed stationary evolution with GC&lt;sub&gt;0&lt;/sub&gt; = 0.5 and 0.7. Population sampling and ancestral inference were replicated 100 times.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1755/F4.medium.gif" width="440" height="370"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/5/1755/F4.medium.gif" width="440" height="370"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489556" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Performance of ancestral inference methods for estimating the SFS of polymorphic mutations in stationary GC content scenarios. <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-1.gif"/></noscript></span></span>goodness of fit statistics were calculated using actual <em>vs.</em> estimated numbers of polymorphic mutations for each mutation category. In each frequency class, the proportions of actual polymorphic mutations were used to calculate “expected” values to compare to the inferred numbers of polymorphisms (“observed” values). <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-2.gif"/></noscript></span></span> statistics were calculated for each replicate with these expected and observed values. The gray scales and cell values give the numbers of replicates showing “poor” fits between observed and expected values (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-3.gif"/></noscript></span></span> ≥ 13.0) and “good” fits (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-4.gif"/></noscript></span></span> ≤ 3.5). Low and high <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-5.gif"/></noscript></span></span> cutoffs correspond to <em>P</em> ≥ 0.9 and ≤ 0.1 for <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-6.gif"/></noscript></span></span> goodness of fit tests with the degree of freedom = 8. Note that <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F4/embed/mml-math-7.gif"/></noscript></span></span> values strongly depend on the number of polymorphic mutations; results are comparable among different methods for the same simulation scenario and mutation category, but are only comparable among different scenarios or mutation categories if their sample sizes are similar. The numbers of polymorphic mutations in each scenario and mutation category are shown in Table S1. The SFS of the <em>m</em> population was estimated under four inference methods: BT, ST, SA and BTW<sub>ne</sub>. The simulation assumed stationary evolution with GC<sub>0</sub> = 0.5 and 0.7. Population sampling and ancestral inference were replicated 100 times.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">BT estimation employs collapse-pair sequences that are constructed without regard to the prevalence of the states in the within-species sample. This procedure biases ancestral inference toward equal probabilities that each polymorphic state is ancestral. Importantly, in GC<sub>0</sub> = 0.7, estimation error is greater for <em>up</em> than for <em>pu</em> mutations and results in a stronger bias toward a flat SFS for <em>up</em> changes (Figure S3A and B). Such patterns arise because the numbers of polymorphisms differ among mutation categories. In GC<sub>0</sub> = 0.7, <em>pu</em> mutations outnumber <em>up</em> because of high GC content and therefore, misidentifying <em>pu</em> as <em>up</em> has a larger proportionate impact on estimated <em>up</em> counts.</p><p id="p-30">Ancestral inference errors can bias tests of natural selection. Differences in the SFS between different mutation categories interspersed within DNA can reveal differences in their fitness effects (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sawyer <em>et al.</em> 1987</a>). In GC<sub>0</sub> = 0.5, all mutations evolve neutrally and their SFS are similarly biased under the BT method. However, in GC<sub>0</sub> = 0.7, the stronger bias toward a flat SFS for <em>up</em> mutations than for <em>pu</em> mutations can exaggerate differences in their SFS. In the stronger selection case, this bias increases because the ratio of <em>pu</em> to <em>up</em> polymorphisms increases (in GC<sub>0</sub> = 0.9, the estimated number of <em>up</em> mutations in frequency class nine is about twice the actual value). Note that BT biases should not produce false signals of selection if high GC is maintained by mutation biases at equilibrium because the expected numbers of <em>up</em> and <em>pu</em> polymorphisms do not differ in such a scenario. <em>Changes</em> in mutation rates, however, can lead to unequal numbers of polymorphic mutations among mutation categories when base composition has not reached equilibrium. For example, if GC → AT mutation rates increase, the numbers of <em>pu</em> mutations will become larger than that of <em>up</em>. In such a case, the stronger estimation bias toward a flat SFS for <em>up</em> mutations could cause a false positive signature of fixation bias.</p><p id="p-31">Although SFS estimation is unreliable under the BT method, the total numbers of polymorphic mutations appear to be estimated accurately (Figure S2). This similarity between estimated and actual values appears to reflect a counter-balance between overestimation of the number of polymorphic mutations in high frequency classes and underestimation of those in low frequency classes and may not apply for other evolutionary scenarios.</p><p id="p-32">The BT method underestimates the numbers of fixations in all mutation categories (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). This bias appears to reflect a difference between our simulation of population genetic processes and BASEML parameterization caused by parallel fixations of ancestral polymorphism (PFAP). BASEML employs a transition-state model that assumes parameter homogeneity (across sites and over time) within lineages. “Fixations” in our simulations are heterogeneous; for a given transition (ancestral and derived state), the probability of parallel fixations of ancestral polymorphism in sister species is considerably higher than the probability of independent originations followed by fixations in two lineages. We confirm that the estimated number of fixations are similar to actual values filtered for PFAP occurrences (actual<sub>fPFAP</sub>) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). In our simulations, PFAP events are common because terminal lineages are relatively short, levels of ancestral variation are high, and population sizes are maintained through species splitting events.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Actual vs. estimated numbers of fixations under four ancestral inference methods. Results are for fixations in the ms-m&#x2019; lineage (ms-m lineage in SA). Among actual fixations, parallel fixations of ancestral polymorphism (PFAP) in the ms-m&#x2019; and ms-s&#x2019; lineages are shown separately from non-PFAP fixations (actualfPFAP). The legend applies to both graphs. The simulation assumed stationary scenario with (A) GC0 = 0.5 and (B) 0.7. Population sampling and ancestral inference were replicated 100 times. Averages and 95% confidence interval of counts among the replicates are shown. Note that y-axis values do not start at zero." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1098483939" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Actual &lt;em&gt;vs.&lt;/em&gt; estimated numbers of fixations under four ancestral inference methods. Results are for fixations in the &lt;em&gt;ms-m&lt;/em&gt;&#x2019; lineage (&lt;em&gt;ms-m&lt;/em&gt; lineage in SA). Among actual fixations, parallel fixations of ancestral polymorphism (PFAP) in the &lt;em&gt;ms-m&lt;/em&gt;&#x2019; and &lt;em&gt;ms-s&lt;/em&gt;&#x2019; lineages are shown separately from non-PFAP fixations (actual&lt;sub&gt;fPFAP&lt;/sub&gt;). The legend applies to both graphs. The simulation assumed stationary scenario with (A) GC&lt;sub&gt;0&lt;/sub&gt; = 0.5 and (B) 0.7. Population sampling and ancestral inference were replicated 100 times. Averages and 95% confidence interval of counts among the replicates are shown. Note that y-axis values do not start at zero.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1755/F5.medium.gif" width="392" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/5/1755/F5.medium.gif" width="392" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489718" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Actual <em>vs.</em> estimated numbers of fixations under four ancestral inference methods. Results are for fixations in the <em>ms-m</em>’ lineage (<em>ms-m</em> lineage in SA). Among actual fixations, parallel fixations of ancestral polymorphism (PFAP) in the <em>ms-m</em>’ and <em>ms-s</em>’ lineages are shown separately from non-PFAP fixations (actual<sub>fPFAP</sub>). The legend applies to both graphs. The simulation assumed stationary scenario with (A) GC<sub>0</sub> = 0.5 and (B) 0.7. Population sampling and ancestral inference were replicated 100 times. Averages and 95% confidence interval of counts among the replicates are shown. Note that y-axis values do not start at zero.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">In contrast to the BT method, the star tree approach (ST) leads to overestimation of the proportions of mutations in low frequency classes and underestimation of those in high frequency classes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Under this method, inference is strongly biased toward attributing the prevalent state in the sample as ancestral because such configurations reduce the required numbers of changes in the tree (Figure S1). For example, consider a site at which the frequencies of two polymorphic nucleotides, A and T, are 0.6 and 0.4, respectively in a sample of 10 sequences. The scenario in which T is ancestral requires six independent changes compared to four changes for the scenario in which A is ancestral. If the probability of nucleotide change within a lineage is low, two additional changes would greatly decrease the probability that the rare state, T, is ancestral. Overestimation of the probabilities that common states are ancestral causes over- and under-estimation of the numbers of polymorphisms in low and high frequency classes, respectively (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Differences between actual and estimated SFSs are considerable; the estimated numbers polymorphic mutations in high frequency classes are close to zero (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, Table S1).</p><p id="p-35">ST estimation errors may impact data analyses because mutation categories are not equally affected by the biases. Almost all <em>pu</em> (<em>up</em>) mutations in high frequency classes are mis-inferred to be <em>up</em> (<em>pu</em>) mutations in low frequency classes. If the numbers of <em>pu</em> polymorphisms are larger than the numbers of <em>up</em> polymorphisms, the skew toward rare variants in the SFS will be larger for <em>up</em> mutations. Such biases are problematic when the SFSs or average frequencies of <em>pu</em> and <em>up</em> mutations are compared to test for fixation biases caused by weak selection or biased gene conversion in base composition evolution. In scenarios of non-neutral base composition bias (<em>e.g.</em>, GC<sub>0</sub> = 0.7; Figure S3C and D), the estimation error explained above would underestimate the advantageous effect of <em>up</em> mutations (or deleterious effect of <em>pu</em> mutations), or the effect of biased gene conversion toward GC. In addition, changes in mutation rate can cause a false signature of selection and biased gene conversion (similar to the BT method). For example, SFS of <em>pu</em> mutations can show stronger skews toward rare variants if base composition has not reached equilibrium after a <em>up</em> mutation rate increase. This can cause a false signature of a deleterious effect of <em>pu</em> mutations (or advantageous effect of <em>up</em> mutations), or biased gene conversion toward GC.</p><p id="p-36">ST inference also yields inaccurate estimates of the total numbers of polymorphic mutations when base composition is under selection (Figure S2B). In this method, almost all <em>pu</em> (<em>up</em>) mutations in high frequency classes are inferred as <em>up</em> (<em>pu</em>) mutations in low frequency classes. Therefore, the difference in the total numbers of polymorphic mutations in GC<sub>0</sub> = 0.7 causes large over- and under-estimation of the total numbers of <em>pu</em> and <em>up</em> polymorphic mutations, respectively (Figure S2B, S3C and D). In the neutral case (GC<sub>0</sub> = 0.5), the total numbers of <em>pu</em> and <em>up</em> polymorphic mutations are almost equal and therefore, the counter-balancing of the errors results in similarity of estimated and actual numbers of polymorphic mutations (Figure S2A). This error-balancing also occurs when high GC is maintained by mutation rate bias at equilibrium without selection.</p><p id="p-37">The ST method shows relatively accurate estimation of the numbers of fixations (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). This accuracy also appears to reflect counter-balancing effects among different errors. The ST method overestimates the numbers of fixations by attributing polymorphic states at higher frequency to the MRCA node. This overestimation can be partly compensated by the underestimation because of PFAP. However, the compensation is not exact and the ST method shows overestimation of the number of fixations when fixation biases affect base composition (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-38">Differences in polymorphism/divergence ratios (<em>r</em><sub>pd</sub>) among mutation categories interspersed within DNA can reveal differences in their fitness effects (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">McDonald and Kreitman 1991</a>). In the ST method, estimation errors for the total numbers of polymorphic mutations can be considerable and can differ in direction among mutation categories (Figure S2B). For example, for GC<sub>0</sub> = 0.7, <em>r</em><sub>pd</sub> for <em>pu</em> and <em>up</em> mutations are 15% underestimated and 7% overestimated, respectively [this reflects 6% underestimation and 10% overestimation of the numbers of polymorphisms (Figure S2B) and 9% and 3% overestimation of the numbers of fixations for <em>pu</em> and <em>up</em> mutations, respectively (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>)]. Because mutation-selection-drift predicts higher <em>r</em><sub>pd</sub> for deleterious <em>pu</em> mutations than for advantageous <em>up</em> changes (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Akashi 1995</a>), these errors can diminish evidence for weak selection/fixation biases in <em>r</em><sub>pd</sub> comparisons between these mutations categories (<em>i.e.</em>, reduce the absolute values of estimated fitness effects). At mutational equilibrium in the absence of fixation biases, the estimation of the numbers of polymorphic mutations is accurate and <em>r</em><sub>pd</sub> does not differ between <em>pu</em> and <em>up</em> mutations. However, an increase in <em>up</em> mutation rate can cause an elevated ratio of <em>up</em> to <em>pu</em> polymorphism and generate a signal similar to fixation biases favoring <em>up</em> mutations if base composition is on the approach to equilibrium under a new mutation rate.</p><p id="p-39">The SA and BT methods show similar estimation biases toward flat SFS (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A and C</a>). The SA method is more accurate but deviations from actual values remain appreciable (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, Table S1). An important source of error in SA is the assumption of identity of states at the within-species MRCA and sibling species nodes (<em>e.g.</em>, <em>m</em>’ and <em>ms</em>). In our neutral simulation, approximately 5.2% of sites underwent fixations in <em>ms-m</em>’ lineage. Polymorphic sites in the <em>m</em> population that experienced fixations in the <em>ms-m</em>’ lineage violate SA inference assumptions. For example, consider a site at which, in a sample of 10 sequences from <em>m</em> population, two nucleotides G and T segregate with frequencies 0.9 and 0.1, respectively. If the actual ancestral state at <em>m</em>’ is G as a consequence of a T→ G fixation in the <em>ms-m</em>’ lineage, SA estimates a T → G polymorphic mutation with frequency 0.9, rather than a G → T mutation with frequency 0.1. In this example, SA would overestimate high frequency <em>up</em> polymorphisms and underestimate low frequency <em>pu</em> polymorphisms. Estimation error also occurs in the opposite direction (underestimation of low frequency classes and overestimation of high frequency classes). However, if the actual numbers of polymorphic mutations in high frequency classes is much smaller than that in low frequency classes, the overall bias will be toward flatter SFS as shown in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>. Similar to the BT and ST methods, when the numbers of <em>pu</em> and <em>up</em> polymorphic mutations differ, the mutation category showing the smaller number experiences stronger inference biases. From these reasons, potential effects of the SA method on SFS comparisons are similar to BT.</p><p id="p-40">The SA method shows slight overestimation of the total number of polymorphic mutations (Figure S2, about 2∼4% of the estimated number of polymorphic mutations are overestimated). If the inferred ancestral state differs from both segregating states, this method “counts” two different polymorphic mutations at such a site. Thus, fixations in the <em>ms-m</em>’ lineage can cause overestimation of the total number of polymorphic mutations under SA. Consider a case similar to the example above with G and T segregating at frequencies 0.9 and 0.1, respectively but in which C is assigned as ancestral at the <em>ms</em> node. If the actual ancestral state at <em>m</em>’ is G as a consequence of a C→ G fixation in the <em>ms-m</em>’ lineage, the SA method estimates a C → G polymorphic mutation with frequency 0.9 and a C → T mutation with frequency 0.1, rather than a single G → T mutation with frequency 0.1. The magnitude of overestimation becomes larger for <em>up</em> than for <em>pu</em> mutations with increasing GC bias because of the reduced ratio of <em>up</em>:<em>pu</em> polymorphism. In addition, overestimation increases with the numbers of fixations in the lineage prior to the within-species MRCA.</p><p id="p-41">The use of a single allele in the SA approach may lead to error in estimating numbers of fixations. Mutations on terminal lineages fall into two classes: those that occurred prior to and those that occurred after the MRCA of the population sample. The latter class are polymorphic in the sample but are counted as fixations under SA. Thus, the numbers of fixations are overestimated as a function of the depth of the MRCA within the terminal lineage. In GC<sub>0</sub> = 0.5, roughly 22% of the estimated <em>pu</em> and <em>up</em> “fixations” under SA are actually polymorphic among the 10 sequences sampled from a population. Filtering all polymorphic sites yields estimates of the numbers of fixed difference under SA similar to actual<sub>fPFAP</sub> (data not shown). However, such an approach complicates the interpretation of polymorphism/divergence analysis because different sets of nucleotide sites are examined at short <em>vs.</em> long time scales.</p><p id="p-42">Under neutral evolution (including GC<sub>0</sub> = 0.5), <em>r</em><sub>pd</sub> estimation biases are similar for <em>pu</em> and <em>up</em> mutations and we do not expect false positives (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>, Figure S2). However, in GC<sub>0</sub> = 0.7, the average proportions of post-MRCA changes are roughly 27% and 18% for <em>pu</em> and <em>up</em>, respectively. Differences in the rate of accumulation of mutations on the pre- and post-MRCA parts of the lineage among fitness classes can impact <em>r</em><sub>pd</sub> comparisons. As shown in <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5B</a>, the magnitude of the overestimation of the number of fixations is much higher for <em>pu</em> mutations (roughly 23%) than for <em>up</em> mutations (roughly 10%). Estimation error is much smaller for the numbers of polymorphic mutations than for fixations (Figure S2). In total, <em>r</em><sub>pd</sub> values for <em>pu</em> and <em>up</em> mutations are about 18% and 5% underestimated, respectively, in GC<sub>0</sub> = 0.7, which diminishes <em>r</em><sub>pd</sub> differences (and the signal of selection) when fitness effects differ between mutation categories.</p><p id="p-43">SA inference can also bias tests of equilibrium base composition among lineages. Comparisons of the numbers of <em>pu</em> and <em>up</em> fixations can be employed to infer the direction of base composition evolution among lineages and the causes of non-stationary evolution (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Akashi <em>et al.</em> 2007</a>). Greater overestimation of the numbers of <em>pu</em> fixations than that of <em>up</em> causes a false signal of decline of the favored state (excess <em>pu</em> fixations). The magnitude of inference error depends on the strength of selection/fixation biases and can produce a pattern similar to expectations under a uniform reduction in fixation biases within a lineage (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Akashi <em>et al.</em> 2007</a>).</p><p id="p-44">In summary, the three methods examined above can show considerable error in estimating the numbers and SFS of polymorphisms as well as the numbers of fixations among closely related species. The SA method shows the most accurate estimation of SFS among the three methods but can suffer from error in the estimation of the total numbers of polymorphic mutations and fixations. For estimation of the total numbers of polymorphic mutations and fixations, the BT method is the most accurate among these approaches.</p><p id="p-45">The frequencies of polymorphic mutations themselves contain information about the identity of ancestral and derived states (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Watterson and Guess 1977</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Griffiths and Tavare 1998</a>). Under neutral and weak selection cases, substantially larger proportions of derived mutations are expected to be found at low frequencies within a population (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Fisher 1930</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wright 1969</a>). We attempted to incorporate such information to improve SFS estimation under BTW. We began by employing the expected SFS under neutral equilibrium, SFS<sub>ne</sub>, for weighting. <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figures 3D</a>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> and Table S1 show that this method, BTW<sub>ne</sub>, more accurately estimates SFS than BT, ST, and SA. Estimation remains reliable for some scenarios of weak selection on base composition where the SFS<sub>ne</sub> assumption is violated (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, Table S1, Figure S4B). The numbers of polymorphic mutations are also estimated reliably under BTW<sub>ne</sub> (Figure S2) but the numbers of fixations are underestimated because of PFAP (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). Fixation numbers are close to actual<sub>fPFAP</sub> values but are underestimated because the AWP approach does not adjust for multiple fixations (<a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>).</p></div><div id="sec-11" class="subsection"><h3>Non-stationary scenarios</h3><p id="p-46">We examined the robustness of the BTW<sub>ne</sub> method in non-neutral, non-stationary scenarios of base composition evolution. In the fixation bias change scenario with GC<sub>0</sub> = 0.7, the BTW<sub>ne</sub> method accurately estimates the SFS (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, Table S2) and numbers of polymorphic mutations (Figure S5). These results suggest robustness of weighting by SFS<sub>ne</sub> in at least some scenarios in which <em>pu</em> and <em>up</em> mutations are not neutral and base composition is not stationary. Similar to the results for stationary scenarios, BT, ST and SA methods did not show the accurate estimation of SFS (Table S3). We also found that the numbers of fixations show PFAP and AWP underestimation under BTW<sub>ne</sub> (Figure S6), which is also consistent with the results for stationary scenarios.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of the BTWne methods for estimating the SFS of polymorphic mutations in non-stationary GC content scenarios.  goodness of fit statistics were calculated using actual vs. estimated numbers of polymorphic mutations for each mutation category. The gray scale and cell values give the numbers of replicates showing &#x201C;poor&#x201D; and &#x201C;good&#x201D; fits between observed and expected values. The procedure of  calculation and the meaning of the gray scale and the number inside each cell are described in the Figure 4 legend. The numbers of polymorphic mutations in each scenario and mutation category is shown in Table S2. The SFS of the focused population was estimated under the BTWne method. The simulation assumed fixation bias change scenario with GC0 = 0.7 and four demographic change scenarios, demA, demB, demE and demF. Population sampling and ancestral inference were replicated 100 times." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1098483939" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Performance of the BTW&lt;sub&gt;ne&lt;/sub&gt; methods for estimating the SFS of polymorphic mutations in non-stationary GC content scenarios. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-8&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; goodness of fit statistics were calculated using actual &lt;em&gt;vs.&lt;/em&gt; estimated numbers of polymorphic mutations for each mutation category. The gray scale and cell values give the numbers of replicates showing &#x201C;poor&#x201D; and &#x201C;good&#x201D; fits between observed and expected values. The procedure of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; calculation and the meaning of the gray scale and the number inside each cell are described in the Figure 4 legend. The numbers of polymorphic mutations in each scenario and mutation category is shown in Table S2. The SFS of the focused population was estimated under the BTW&lt;sub&gt;ne&lt;/sub&gt; method. The simulation assumed fixation bias change scenario with GC&lt;sub&gt;0&lt;/sub&gt; = 0.7 and four demographic change scenarios, demA, demB, demE and demF. Population sampling and ancestral inference were replicated 100 times.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1755/F6.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/5/1755/F6.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489644" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-47" class="first-child">Performance of the BTW<sub>ne</sub> methods for estimating the SFS of polymorphic mutations in non-stationary GC content scenarios. <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F6/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F6/embed/mml-math-8.gif"/></noscript></span></span> goodness of fit statistics were calculated using actual <em>vs.</em> estimated numbers of polymorphic mutations for each mutation category. The gray scale and cell values give the numbers of replicates showing “poor” and “good” fits between observed and expected values. The procedure of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F6/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F6/embed/mml-math-9.gif"/></noscript></span></span> calculation and the meaning of the gray scale and the number inside each cell are described in the <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a> legend. The numbers of polymorphic mutations in each scenario and mutation category is shown in Table S2. The SFS of the focused population was estimated under the BTW<sub>ne</sub> method. The simulation assumed fixation bias change scenario with GC<sub>0</sub> = 0.7 and four demographic change scenarios, demA, demB, demE and demF. Population sampling and ancestral inference were replicated 100 times.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">To test the robustness of the BTW method to demographic violations of the standard neutral model, we analyzed sequences simulated under four demographic change scenarios. These scenarios result in substantial departures from SFS<sub>ne</sub> (Figure S7); demA and B show a flatter SFS whereas demE and F show excesses of rare mutations. Results for two other scenarios (demC and D) that showed smaller departures from SFS<sub>ne</sub> are given in the supplementary materials (Figure S7; Table S2).</p><p id="p-49">BTW<sub>ne</sub> reliability varied considerably among the tested demographic scenarios: SFS estimation was accurate for demA and demE but was unreliable for demB and demF (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, Table S2). However, estimation of the total numbers of polymorphic mutations is accurate for all four scenarios (Figure S8). The estimated numbers of fixations in the <em>ms-m</em>’ lineage under BTW<sub>ne</sub> is also very close to actual<sub>fPFAP</sub> (Figure S9).</p><p id="p-50">We tested whether ancestral inference error for demB and demF can be attributed mainly to incorrect SFS weighting. If this is the case, then employing an accurate SFS for weighting (without changing any aspects of the upstream ancestral state inference) should improve estimation accuracy. Table S4 shows marked increases in the accuracy of SFS estimation for demB and demF when the average actual SFS’s among replicates is employed as the “empirical” SFS (BTW<sub>emp</sub>).</p></div><div id="sec-12" class="subsection"><h3>Strongly biased base composition scenarios</h3><p id="p-51">In stronger selection (or fixation bias) scenarios, actual SFS show strong departures from SFS<sub>ne</sub> (Figure S4C, E and F). BTW<sub>ne</sub> can accurately estimate the SFS in some of these scenarios (Table S5). The total numbers of polymorphic mutations are accurately estimated (Figure S10), and fixations are similar to actual<sub>fPFAP</sub> (Figure S11) in all of the considered scenarios. However, similar to the demographic change scenarios, strong departures from SFS<sub>ne</sub> can result in considerable reductions in the accuracy of SFS estimation under BTW<sub>ne</sub> (<em>e.g.</em>, <em>ty-t</em> lineage in the fixation bias change scenario in Table S5). Again, incorrect SFS weighting appears to be the main issue; BTW<sub>emp</sub> gives accurate SFS estimation for the strong and/or non-equilibrium base composition bias scenarios (Table S4).</p></div><div id="sec-13" class="subsection"><h3>Iterative BTW<sub>est</sub></h3><p id="p-52">The BTW<sub>ne</sub> method shows robustness to moderate differences between the assumed and actual SFS but stronger departures can degrade the accuracy of ancestral inference. Because weighting by realistic SFS (BTW<sub>emp</sub>) can yield accurate inference in such cases, iterative SFS estimation may be a promising approach for ancestral inference when SFS<sub>ne</sub> may be strongly violated. To test this notion, we performed ancestral inference followed by multiple rounds of SFS weighting. The first round employed SFS<sub>ne</sub> for weighting and the resulting SFS will be referred to as SFS<sub>est1</sub>. The latter was employed to weight a second round of ancestral inference and this process was repeated for five rounds (<em>i.e.</em>, the last round employed SFS<sub>est4</sub> for weighting). Iterative weighting was tested with data from three scenarios, demB, demF and fixation bias change scenario with GC<sub>0</sub> = 0.9. From each scenario, we choose a single replicate that shows a large departure between actual and estimated SFS under BTW<sub>ne</sub>. Iterative BTW<sub>est</sub> allows accurate estimation of the numbers of polymorphic mutations and SFS across mutation categories in all of the examined scenarios and appears to provide a feasible method to considerably enhance the accuracy of ancestral inference (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). In the cases examined, the fit between actual and estimated SFS improved considerably in the second round of estimation (using BTW<sub>est1</sub>) and largely converged within a few rounds (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). For dem B and demF, we also applied the iterative BTW<sub>est</sub> method for replicates and confirmed a large improvement in the accuracy of SFS estimation compared with BTW<sub>ne</sub> inference (Table S2 and S6).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of the iterative BTWest method for estimating the SFS of polymorphic mutations in non-stationary GC content scenarios. SFS of the t population in fixation bias change scenario with GC0 = 0.9 and m population in demB and demF were estimated under the iterative BTWest method, and  goodness of fit statistics were calculated using actual vs. estimated numbers of polymorphic mutations for each mutation category. The procedure for  calculation is described in the Figure 4 legend. This figure shows results for a single replicate that showed relatively large  value in the BTWne analysis. The estimation under iterative BTWest was repeated for six rounds (the first round was BTWne and the following five were BTWest using the estimated SFS of the previous round). The cell values show the calculated  value and the shaded cell means that the  &#x2264; 3.5 which is the criteria of low  value. The results of all 100 replicates for demB and demF are shown in Table S5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1098483939" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Performance of the iterative BTW&lt;sub&gt;est&lt;/sub&gt; method for estimating the SFS of polymorphic mutations in non-stationary GC content scenarios. SFS of the &lt;em&gt;t&lt;/em&gt; population in fixation bias change scenario with GC&lt;sub&gt;0&lt;/sub&gt; = 0.9 and &lt;em&gt;m&lt;/em&gt; population in demB and demF were estimated under the iterative BTW&lt;sub&gt;est&lt;/sub&gt; method, and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; goodness of fit statistics were calculated using actual &lt;em&gt;vs.&lt;/em&gt; estimated numbers of polymorphic mutations for each mutation category. The procedure for &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-11&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; calculation is described in the Figure 4 legend. This figure shows results for a single replicate that showed relatively large &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-12&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; value in the BTW&lt;sub&gt;ne&lt;/sub&gt; analysis. The estimation under iterative BTW&lt;sub&gt;est&lt;/sub&gt; was repeated for six rounds (the first round was BTW&lt;sub&gt;ne&lt;/sub&gt; and the following five were BTW&lt;sub&gt;est&lt;/sub&gt; using the estimated SFS of the previous round). The cell values show the calculated &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-13&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; value and the shaded cell means that the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-14&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &#x2264; 3.5 which is the criteria of low &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-15&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; value. The results of all 100 replicates for demB and demF are shown in Table S5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1755/F7.medium.gif" width="320" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/5/1755/F7.medium.gif" width="320" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1755/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489593" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-53" class="first-child">Performance of the iterative BTW<sub>est</sub> method for estimating the SFS of polymorphic mutations in non-stationary GC content scenarios. SFS of the <em>t</em> population in fixation bias change scenario with GC<sub>0</sub> = 0.9 and <em>m</em> population in demB and demF were estimated under the iterative BTW<sub>est</sub> method, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-10.gif"/></noscript></span></span> goodness of fit statistics were calculated using actual <em>vs.</em> estimated numbers of polymorphic mutations for each mutation category. The procedure for <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-11.gif"/></noscript></span></span> calculation is described in the <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a> legend. This figure shows results for a single replicate that showed relatively large <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-12.gif"/></noscript></span></span> value in the BTW<sub>ne</sub> analysis. The estimation under iterative BTW<sub>est</sub> was repeated for six rounds (the first round was BTW<sub>ne</sub> and the following five were BTW<sub>est</sub> using the estimated SFS of the previous round). The cell values show the calculated <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-13.gif"/></noscript></span></span> value and the shaded cell means that the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-14.gif"/></noscript></span></span> ≤ 3.5 which is the criteria of low <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5/1755/F7/embed/mml-math-15.gif"/></noscript></span></span> value. The results of all 100 replicates for demB and demF are shown in Table S5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Although the frequencies of polymorphism are informative for ancestral state inference, assuming a particular SFS may bias reconstructions. We tested how the shape of the starting SFS (assumed SFS in the first round) affects the reliability of iterative BTW<sub>est</sub>. We employed SFS<sub>ne</sub>, uniform frequencies and excess common SFS<sub>ec</sub> (SFS<sub>ne</sub> was flipped on the vertical axis) as starting SFS for three independent iterative BTW<sub>est</sub> processes for data from the stationary scenario with GC<sub>0</sub> = 0.5. SFS<sub>ne</sub> as the starting SFS is a realistic assumption in this case, but the other starting SFS differ considerably from the actual SFS. The iterative BTW<sub>est</sub> analysis accurately estimated the SFS with all three starting SFS (Table S7). Estimated SFS under iterative BTW<sub>est</sub> can converge to the accurate values despite inaccuracies in the assumed SFS in the first round (see also Table S6).</p><p id="p-55">Our main simulation scenarios did not included biased mutation or genetic linkage but we show reliable SFS estimation for <em>pu</em>, <em>up</em> and <em>pp</em>+<em>uu</em> mutations in such cases (Table S8 and S9).</p></div><div id="sec-14" class="subsection"><h3>Ancestral polymorphism</h3><p id="p-56">As we noted above in the discussion of PFAP, shared ancestral polymorphism violates the assumptions of the TRMFT approach and can lead to bias/error in ancestral inference. At least three cases can be problematic: 1) parallel fixations of non-ancestral states (PFAP), 2) maintenance of ancestral polymorphism among sister lineages, and 3) fixation of the non-ancestral state in one lineage and maintenance of both states as polymorphism in the other. Cases 2 and 3 affect polymorphism analysis but in our simulations, polymorphic mutations that originated in an ancestral population and remain shared in sister species are rare (<em>e.g.</em>, about 1% of all polymorphic mutations are shared between the <em>m</em> and <em>s</em> populations in the stationary scenario with GC<sub>0</sub> = 0.5). To test the effect of shared ancestral polymorphism on the accuracy of ancestral inference (the input tree is incorrect for such sites), we examined the performance of the BTW<sub>ne</sub> method for data simulated under a tree with shorter branch lengths (2,000, 1,000 and 100 generations) for the <em>ms-m</em> and <em>ms-s</em> lineages (all other branch lengths are unchanged). These branch lengths result in about 5%, 10% and 25% shared polymorphism, respectively. Among these scenarios, the reliability of BTW<sub>ne</sub> inference is high at 5 and 10% shared polymorphism but decreased at 25% (Table S10). Importantly, filtering sites showing the same polymorphic states in the <em>m</em> and <em>s</em> populations (<em>i.e.</em>, candidates for shared ancestral polymorphism) does not increase the estimation reliability (Table S11) perhaps because the 3<sup>rd</sup> class of errors remain unfiltered.</p></div><div id="sec-15" class="subsection"><h3>Ancestral inference under maximum parsimony</h3><p id="p-57">We showed that previously employed ancestral inference approaches for polymorphism data, BT, ST and SA, can yield significant estimation error for the numbers of polymorphic mutations and fixations as well as the SFS of segregating sites (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figures 3</a>-<a id="xref-fig-5-9" class="xref-fig" href="#F5">5</a>, Figure S2, Table S1, S3). Maximum parsimony (MP) was not examined in detail here because the approach is known to suffer from large estimation error in scenarios with unequal nucleotide transition rates (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Collins <em>et al.</em> 1994</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Perna and Kocher 1995</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eyre-Walker 1998</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Galtier and Boursot 2000</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Alvarez-Valin <em>et al.</em> 2004</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Akashi <em>et al.</em> 2007</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hernandez <em>et al.</em> 2007</a>; <a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>). We emulated maximum parsimony by combining a Jukes-Cantor transition rate matrix (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jukes and Cantor 1969</a>) and single best reconstruction (<a id="xref-ref-38-9" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>) and inferred polymorphisms and fixations in the stationary scenario with GC<sub>0</sub> = 0.7 under MP. As expected, MP shows considerable error in estimating the numbers of fixations at moderate base composition bias even among closely related lineages (Figure S12A) and we believe the approach should not be employed as a sole or primary ancestral inference method (<a id="xref-ref-38-10" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>). Because the underestimation is much larger for advantageous (<em>up</em>) than deleterious (<em>pu</em>) mutations, this bias causes a false signal of non-stationary base composition evolution (<a id="xref-ref-38-11" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>). For SFS estimation, maximum parsimony results in underestimation and overestimation of low and high frequency classes, respectively. The biases are similar in direction, but somewhat lower in magnitude, than for BT inference (Table S12). The numbers of polymorphic mutations are estimated accurately (Figure S12B). <em>r</em><sub>pd</sub> is 23% overestimated for <em>pu</em> mutations and 41% overestimated for <em>up</em> mutations. Because <em>r</em><sub>pd</sub> decreases monotonically with <em>N</em><sub>e</sub><em>s</em>, <em>r</em><sub>pd</sub> tests will underestimate the advantageous effect of <em>up</em> mutations and overestimate the deleterious effects of <em>pu</em> changes. In summary, for comparisons of <em>pu</em> and <em>up</em> mutations, maximum parsimony exhibits moderate bias in the SFS estimation but considerable biases in estimating the number of fixations. The latter biases are expected to strongly impact data analyses.</p></div><div id="sec-16" class="subsection"><h3>Sample size and SFS estimation</h3><p id="p-58">Finally, we show how sequence length (and the number of polymorphic sites) affects the reliability of the inference of polymorphisms. In the above analysis, we used 100,000 nucleotide sites. Among these nucleotide sites, about 4,000 are polymorphic among 10 sequences sampled from <em>m</em> population in GC<sub>0</sub> = 0.5. We also conducted analyses using subsets of this data (10,000 and 1,000 sites) for the stationary scenario with GC<sub>0</sub> = 0.5. We analyzed 10,000 and 1,000 sites from the same sequences used in the above analysis and estimated the SFS under the BTW<sub>ne</sub> with 100 replications. Our result shows that even for the smaller data set, the estimated SFS is quite accurate. With 10,000 sites, all 100 replicates showed close matching between estimated and actual SFS for <em>pu</em>, <em>up</em> and <em>pp</em>+<em>uu</em> mutations. With 1,000 sites, the very small numbers of polymorphic mutations precluded goodness of fit evaluation but the estimated SFS’s are close to actual (Table S13). However, we have to note that the accurate estimation of SFS with 1,000 sites may not be a general result. Complexity of the evolutionary scenario or the number of analyzed species can affect the number of sites required for the accurate estimation. In addition, if bootstrap analysis is conducted to calculate the confidence interval of SFS, ancestral inference with small number of sites can show highly variable result across replicates (<a id="xref-ref-38-12" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>) which makes the confidence interval large.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-59">The rapid expansion of genome-scale polymorphism data has generated a need for methods to infer ancestral and derived states for within-species variation. Numerous studies have demonstrated the impact of nucleotide transition model on the accuracy of TRMFT-based ancestral inference (<em>e.g.</em>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Collins <em>et al.</em> 1994</a>, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Perna and Kocher 1995</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Eyre-Walker 1998</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Galtier and Boursot 2000</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Alvarez-Valin <em>et al.</em> 2004</a>, <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Akashi <em>et al.</em> 2007</a>, <a id="xref-ref-38-13" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>). However, for genomes undergoing recombination, small genetic regions, even individual sites, can have different genealogies in within-species samples. In such cases, the requirement for a single phylogeny is an important limitation of likelihood-based approaches to ancestral inference. Because region-specific gene trees may be difficult to reconstruct accurately (especially when the regions are small), we have explored approximate approaches to infer fixations and polymorphisms among closely related lineages.</p><p id="p-60">Several previous studies have employed likelihood-based estimation methods using ancestral inference that incorporate population genetic and phylogenetic models to estimate the number of fixations and polymorphic mutations and their SFS. However, most such studies have not employed models (<em>i.e.</em>, transition rate matrices) that can incorporate mutation rate biases and differences in fixation probabilities of mutations caused by selection or biased gene conversion. For example, the method proposed in <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wilson <em>et al.</em> (2011)</a> employs both population genetics theory (expected stationary SFS) and phylogenetic ancestral inference based on a transition rate matrix to calculate likelihoods of states at ancestral nodes. Their main focus was detection of adaptive protein evolution and fixation biases are considered only for non-synonymous changes; synonymous changes are assumed to evolve under the HKY85 model (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hasegawa <em>et al.</em> 1985</a>) which has limited applicability for base composition evolution scenarios including those explored here (<a id="xref-ref-38-14" class="xref-bibr" href="#ref-38">Matsumoto <em>et al.</em> 2015</a>).</p><p id="p-61">More recently, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Keightley <em>et al.</em> (2016)</a> developed a method to estimate the unfolded SFS. Their method estimates the expected number of nucleotide fixations per site, <em>K</em>, that can explain the observed polymorphism in a focal species. The estimated <em>K</em> is used to estimate the likelihood of ancestral states at each site. The method can assign different <em>K</em> for transition and transversion mutations. However, fitness effects/fixation biases for different mutation categories (<em>e.g.</em>, <em>up</em>, <em>pu</em> and <em>pp</em>+<em>uu</em>) are not considered and the number of outgroups for the analysis is limited to two. Therefore, this method may also be limited for studying base composition evolution. Variation in fixation probabilities across different mutation categories and non-stationary evolution among multiple lineages are known aspects of synonymous site evolution, and we have focused on ancestral reconstruction approaches that can account for such phenomena. The <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Keightley <em>et al.</em> (2016)</a> method estimates SFS for pooled mutations (pooled SFS) and we compared it with the estimates under BTW. Table S14 shows that both methods estimate the pooled SFS with high accuracy in several evolutionary scenarios. However, in four of five scenarios we considered, BTW showed slightly, but significantly, more accurate estimation than the <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Keightley <em>et al.</em> (2016)</a> method.</p><p id="p-62">The three approaches used in previous studies (ST, SA, and BT) can employ nucleotide transition models that incorporate fixation biases and non-homogeneous evolution (<em>e.g.</em>, the GTR-NH<sub>b</sub>). However, our results show that these implementations do not allow reliable inference of fixations and polymorphisms among closely related species. In the BT method, the estimation of the total numbers of polymorphic mutations is accurate but that of the SFS is biased toward flat distributions. The ST method showed the largest error among the three methods. The estimated SFS showed strong under- and over-estimation of the number of mutations in high and low frequency class, respectively. In addition, the estimated total number of polymorphic mutations was also inaccurate. The SA method showed the smallest error in the estimation of the SFS among the three methods. However, this method overestimates the number of mutations in high frequency class and showed large estimation error for the total numbers of polymorphic mutations and fixations. The BTW method proposed here allows accurate inference of polymorphisms under a range of scenarios of base composition evolution. The iterative BTW<sub>est</sub> approach is more broadly applicable and gave accurate estimation in all scenarios examined in this study including cases where actual SFS showed large departures from SFS<sub>ne</sub>.</p><p id="p-63">In contrast to the inference of polymorphisms, fixation estimation shows error even under BTW. This is because the ancestral inference methods using nucleotide transition rate matrices do not consider parallel fixations of ancestral polymorphism (PFAP). The expected extent of PFAP under neutral evolution can be calculated given information about mutation rate and ancestral population size and the effect on particular data sets must be examined on a case by case basis. Inference errors for fixations and polymorphisms in the previous three methods and in BTW may have critical impacts on inferences of selection and biased gene conversion. We have found that the magnitude and direction of biases depend on the method as well as actual evolutionary scenarios. The possibility that inference biases can produce patterns that mimic biological scenarios of interest (such as relationships between apparent strength of fixation biases and GC content) should be a carefully considered in data analyses.</p><div id="sec-18" class="subsection"><h3>Recommendation for data analysis</h3><p id="p-64">Our simulation results suggest that BTW and iterative BTW<sub>est</sub> methods can provide reliable inference of polymorphisms across a range of evolutionary scenarios. For data analysis, data pooling is an important aspect that requires balancing sample size and fit to model assumptions. Larger sample sizes are necessary when employing complex substitution models but the methods assume homogeneity (<em>i.e.</em>, shared parameter values) across sites. In many cases, data partitioning by GC content, genomic location (<em>e.g.</em>, autosome <em>vs.</em> sex chromosomes, euchromatin <em>vs.</em> heterochromatin), expression level, etc may be warranted and robustness of results to the binning/pooling strategy should be tested.</p><p id="p-65">BT approaches assume no shared polymorphisms among species but the approach shows some robustness to this assumption. High proportions of shared polymorphisms decrease the accuracy of the estimation of SFS under the BTW method, however (Table S7) and samples should be screened for shared polymorphism before applying these approaches.</p><p id="p-66">Overall, the iterative BTW<sub>est</sub> method should have broad applicability and is our recommended approach. Although the demographic change scenarios considered above are relatively simple, iterative BTW<sub>est</sub> converged to accurate estimations within a small number of rounds in all the scenarios we considered. Because the computational cost is small, we recommend 5∼10 rounds of iteration for data analysis. In general, employing information from allele frequencies in ancestral state inference should allow more reliable inference if the assumed SFS is correct. However, if the assumed SFS differs from the actual pattern, then this approach could, in principle, lead to estimation biases. Iterative BTW<sub>est</sub> allows a test of such biases; for a given data set, iterative SFS estimation can be conducted with a range of starting SFS (uniform, equilibrium neutral, excess rare, etc). The approaches should converge to a similar SFS if the inference is accurate. In this study, we found this to be the case for a range of actual SFS (Table S5, S6). These results support that iterative BTW<sub>est</sub> is a reliable and robust method that allows accurate estimation of a wide range of SFS.</p></div><div id="sec-19" class="subsection"><h3>Future improvement of ancestral inference for polymorphism data</h3><p id="p-67">Our findings emphasize that ancestral reconstruction methods must be considered carefully when analyzing within and between species genetic variation even among closely related species. Prevalent biases in some ancestral reconstruction approaches that can give results that mimic biological scenarios of interest are concerning but the approximate approach, BTW, appears to reliably infer polymorphism and SFS data under a broad range of conditions. However, we note that all of the scenarios that we simulated were at least roughly consistent with the input tree and nucleotide transition model employed for ancestral inference (a GTR model with lineage-specific base composition and rate parameters). Considerable error in inferring polymorphism and fixation patterns can occur when assumptions of the TRMFT approach are violated (especially effects of ancestral polymorphism). Further development of population genetic approaches for ancestral inference may be necessary to overcome such issues.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">We are grateful to K. Kawashima, H. Yamashita, and Z. Yang for discussion and two anonymous reviewers for comments that helped to improve this work. This study was supported by Grant-in-Aid for Young Scientists B (No. 17K15169 to TM).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-69">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.5996342">https://doi.org/10.25387/g3.5996342</a>.</p></li><li class="fn" id="fn-2"><p id="p-70"><em>Communicating editor: Y. Kim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-12-10"><span class="received-label">Received </span>December 10, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-03-23"><span class="accepted-label">Accepted </span>March 23, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Tomotaka Matsumoto, Hiroshi Akashi</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Inferring weak selection from patterns of polymorphism and divergence at “silent” sites in <em>Drosophila</em> DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1067</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D139%26rft.issue%253D2%26rft.spage%253D1067%26rft.epage%253D1076%26rft.atitle%253DInferring%2Bweak%2Bselection%2Bfrom%2Bpatterns%2Bof%2Bpolymorphism%2Band%2Bdivergence%2Bat%2B%2522silent%2522%2Bsites%2Bin%2BDrosophila%2BDNA.%26rft_id%253Dinfo%253Apmid%252F7713409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8yLzEwNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Molecular evolution between <em>Drosophila melanogaster</em> and <em>D. simulans</em>: reduced codon bias, faster rates of amino acid substitution, and larger proteins in <em>D. melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D1297%26rft.epage%253D1307%26rft.atitle%253DMolecular%2BEvolution%2BBetween%2BDrosophila%2Bmelanogaster%2Band%2BD.%2Bsimulans%2BReduced%2BCodon%2BBias%252C%2BFaster%2BRates%2Bof%2BAmino%2BAcid%2BSubstitution%252C%2Band%2BLarger%2BProteins%2Bin%2BD.%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F8913769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC8zLzEyOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.3" data-doi="10.1016/S0378-1119(99)00294-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Within- and between-species DNA sequence variation and the “footprint” of natural selection.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">238</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">39</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0378-1119(99)00294-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D51%26rft.atitle%253DWithin-%2Band%2Bbetween-species%2BDNA%2Bsequence%2Bvariation%2Band%2Bthe%2B%2527footprint%2527%2Bof%2Bnatural%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900294-2%26rft_id%253Dinfo%253Apmid%252F10570982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00294-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10570982&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083005400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Natural selection and the frequency distributions of “silent” DNA polymorphism in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D295%26rft.epage%253D307%26rft.atitle%253DNatural%2BSelection%2Band%2Bthe%2BFrequency%2BDistributions%2Bof%2B%2522Silent%2522%2BDNA%2BPolymorphism%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F9136019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzEvMjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.5" data-doi="10.1371/journal.pone.0001065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Ancestral inference and the study of codon bias evolution: implications for molecular evolutionary analyses of the <em>Drosophila melanogaster</em> subgroup.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e1065</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0001065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De1065%26rft.epage%253De1065%26rft.atitle%253DAncestral%2Binference%2Band%2Bthe%2Bstudy%2Bof%2Bcodon%2Bbias%2Bevolution%253A%2Bimplications%2Bfor%2Bmolecular%2Bevolutionary%2Banalyses%2Bof%2Bthe%2BDrosophila%2Bmelanogaster%2Bsubgroup.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001065%26rft_id%253Dinfo%253Apmid%252F17957249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17957249&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.6" data-doi="10.1534/genetics.105.049676"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">W.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Molecular Evolution in the <em>Drosophila melanogaster</em> Species Subgroup: Frequent Parameter Fluctuations on the Timescale of Molecular Divergence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1711</span>–<span class="cit-lpage">1726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.049676</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049676%26rft_id%253Dinfo%253Apmid%252F16387879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE3MTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.7" data-doi="10.1016/j.ympev.2004.01.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvarez-Valin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clay</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruveiller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Inaccurate reconstruction of ancestral GC levels creates a “vanishing isochores” effect.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">788</span>–<span class="cit-lpage">793</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2004.01.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DAlvarez-Valin%26rft.auinit1%253DF.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D788%26rft.epage%253D793%26rft.atitle%253DInaccurate%2Breconstruction%2Bof%2Bancestral%2BGC%2Blevels%2Bcreates%2Ba%2B%2526quot%253Bvanishing%2Bisochores%2526quot%253B%2Beffect.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2004.01.016%26rft_id%253Dinfo%253Apmid%252F15062812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2004.01.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15062812&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220948000030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.8" data-doi="10.1093/molbev/mst153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aoki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwasaki</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">From β- to α-Proteobacteria: The Origin and Evolution of Rhizobial Nodulation Genes nodIJ.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2494</span>–<span class="cit-lpage">2508</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst153%26rft_id%253Dinfo%253Apmid%252F24030554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24030554&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.9" data-doi="10.1093/oxfordjournals.molbev.a003918"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The frequency distribution of nucleotide variation in <em>Drosophila simulans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1343</span>–<span class="cit-lpage">1352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a003918</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D18%26rft.issue%253D7%26rft.spage%253D1343%26rft.epage%253D1352%26rft.atitle%253DThe%2BFrequency%2BDistribution%2Bof%2BNucleotide%2BVariation%2Bin%2BDrosophila%2Bsimulans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003918%26rft_id%253Dinfo%253Apmid%252F11420372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003918&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11420372&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169846400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.10" data-doi="10.1093/molbev/msv214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolívar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugal</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nater</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination Rate Variation Modulates Gene Sequence Evolution Mainly via GC-Biased Gene Conversion, Not Hill–Robertson Interference, in an Avian System.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">216</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv214</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv214%26rft_id%253Dinfo%253Apmid%252F26446902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26446902&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.11" data-doi="10.1111/jeb.12003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clemente</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogl</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evidence for complex selection on four-fold degenerate sites in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">2582</span>–<span class="cit-lpage">2595</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.12003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DClemente%26rft.auinit1%253DF.%26rft.volume%253D25%26rft.issue%253D12%26rft.spage%253D2582%26rft.epage%253D2595%26rft.atitle%253DEvidence%2Bfor%2Bcomplex%2Bselection%2Bon%2Bfour-fold%2Bdegenerate%2Bsites%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12003%26rft_id%253Dinfo%253Apmid%252F23020078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23020078&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.12" data-doi="10.1371/journal.pgen.1006799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clément</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarah</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtz</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Homa</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pointet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolutionary forces affecting synonymous variations in plant genomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e1006799</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006799</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006799%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006799&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.13" data-doi="10.1093/sysbio/43.4.482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimberger</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naylor</span>  <span class="cit-name-given-names">G. J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Compositional bias, character-state bias, and character-state reconstruction using parsimony.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">482</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/sysbio/43.4.482</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F43.4.482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/43.4.482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994QE10000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piganeau</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Vanishing GC-rich isochores in mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1837</span>–<span class="cit-lpage">1847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D1837%26rft.epage%253D1847%26rft.atitle%253DVanishing%2BGC-Rich%2BIsochores%2Bin%2BMammalian%2BGenomes%26rft_id%253Dinfo%253Apmid%252F12524353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzE4MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.15" data-doi="10.1007/PL00006427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Problems with parsimony in sequences of biased base composition.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">686</span>–<span class="cit-lpage">690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/PL00006427</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D686%26rft.epage%253D690%26rft.atitle%253DProblems%2Bwith%2Bparsimony%2Bin%2Bsequences%2Bof%2Bbiased%2Bbase%2Bcomposition.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006427%26rft_id%253Dinfo%253Apmid%252F9847410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9847410&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077280900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1755.16" data-doi="10.5962/bhl.title.27468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source">The Genetical Theory of Natural Selection</span>, <span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Oxford</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5962/bhl.title.27468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BGenetical%2BTheory%2Bof%2BNatural%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.5962%252Fbhl.title.27468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5962/bhl.title.27468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.17" data-doi="10.1534/genetics.105.046524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">GC-biased segregation of noncoding polymorphisms in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">221</span>–<span class="cit-lpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.046524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.046524%26rft_id%253Dinfo%253Apmid%252F16157668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvMjIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.18" data-doi="10.1007/s002399910025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boursot</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A New Method for Locating Changes in a Tree Reveals Distinct Nucleotide Polymorphism <em>vs.</em> Divergence Patterns in Mouse Mitochondrial Control Region.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">224</span>–<span class="cit-lpage">231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s002399910025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DGaltier%26rft.auinit1%253DN.%26rft.volume%253D50%26rft.issue%253D3%26rft.spage%253D224%26rft.epage%253D231%26rft.atitle%253DA%2Bnew%2Bmethod%2Bfor%2Blocating%2Bchanges%2Bin%2Ba%2Btree%2Breveals%2Bdistinct%2Bnucleotide%2Bpolymorphism%2Bvs.%2Bdivergence%2Bpatterns%2Bin%2Bmouse%2Bmitochondrial%2Bcontrol%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002399910025%26rft_id%253Dinfo%253Apmid%252F10754064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002399910025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10754064&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086183400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.19" data-doi="10.1038/nature06510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaucher</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindarajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganesh</span>  <span class="cit-name-given-names">O. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Palaeotemperature trend for Precambrian life inferred from resurrected proteins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>(<span class="cit-issue">7179</span>): <span class="cit-fpage">704</span>–<span class="cit-lpage">707</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature06510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGaucher%26rft.auinit1%253DE.%2BA.%26rft.volume%253D451%26rft.issue%253D7179%26rft.spage%253D704%26rft.epage%253D707%26rft.atitle%253DPalaeotemperature%2Btrend%2Bfor%2BPrecambrian%2Blife%2Binferred%2Bfrom%2Bresurrected%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06510%26rft_id%253Dinfo%253Apmid%252F18256669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2011015744&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=18256669&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252929500038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.20" data-doi="10.1038/nature01977"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaucher</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benner</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inferring the palaeoenvironment of ancient bacteria on the basis of resurrected proteins.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>(<span class="cit-issue">6955</span>): <span class="cit-fpage">285</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature01977</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGaucher%26rft.auinit1%253DE.%2BA.%26rft.volume%253D425%26rft.issue%253D6955%26rft.spage%253D285%26rft.epage%253D288%26rft.atitle%253DInferring%2Bthe%2Bpalaeoenvironment%2Bof%2Bancient%2Bbacteria%2Bon%2Bthe%2Bbasis%2Bof%2Bresurrected%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01977%26rft_id%253Dinfo%253Apmid%252F13679914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01977&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13679914&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185370900042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.21" data-doi="10.1101/gr.185488.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantification of GC-biased gene conversion in the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1215</span>–<span class="cit-lpage">1228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.185488.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185488.114%26rft_id%253Dinfo%253Apmid%252F25995268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTIxNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNS8xNzU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.22" data-doi="10.1080/15326349808807471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavare</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The age of a mutation in a general coalescent tree.</span> <abbr class="cit-jnl-abbrev">Stoch. Models</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">1-2</span>): <span class="cit-fpage">273</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15326349808807471</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStoch.%2BModels%26rft.volume%253D14%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15326349808807471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15326349808807471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.23" data-doi="10.1093/molbev/msr098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groussin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Adaptation to Environmental Temperature Is a Major Determinant of Molecular Evolutionary Rates in Archaea.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">2661</span>–<span class="cit-lpage">2674</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr098%26rft_id%253Dinfo%253Apmid%252F21498602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21498602&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294552700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.24" data-doi="10.1098/rsbl.2008.0174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Non-neutral processes drive the nucleotide composition of non-coding sequences in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">438</span>–<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsbl.2008.0174</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2BLetters%26rft.stitle%253DBiol%2BLett%26rft.aulast%253DHaddrill%26rft.auinit1%253DP.%2BR%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D438%26rft.epage%253D441%26rft.atitle%253DNon-neutral%2Bprocesses%2Bdrive%2Bthe%2Bnucleotide%2Bcomposition%2Bof%2Bnon-coding%2Bsequences%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2008.0174%26rft_id%253Dinfo%253Apmid%252F18505714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InJveWJpb2xldHQiO3M6NToicmVzaWQiO3M6NzoiNC80LzQzOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNS8xNzU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.25" data-doi="10.1007/BF02101694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishino</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">160</span>–<span class="cit-lpage">174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02101694</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DHasegawa%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D174%26rft.atitle%253DDating%2Bof%2Bthe%2Bhuman-ape%2Bsplitting%2Bby%2Ba%2Bmolecular%2Bclock%2Bof%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02101694%26rft_id%253Dinfo%253Apmid%252F3934395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02101694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3934395&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AWB1600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.26" data-doi="10.1093/molbev/msm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Context Dependence, Ancestral Misidentification, and Spurious Signatures of Natural Selection.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1792</span>–<span class="cit-lpage">1800</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm108%26rft_id%253Dinfo%253Apmid%252F17545186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17545186&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.27" data-doi="10.1080/10635150119871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bollback</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Empirical and hierarchical Bayesian estimation of ancestral states.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">351</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10635150119871</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150119871%26rft_id%253Dinfo%253Apmid%252F12116580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150119871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12116580&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169823200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.28" data-doi="10.1093/gbe/evw291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in the intensity of selection on codon bias over time causes contrasting patterns of base composition evolution in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw291</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw291%26rft_id%253Dinfo%253Apmid%252F28082609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28082609&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1755.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Evolution of protein mole- cules</span>, pp. <span class="cit-fpage">21</span>–<span class="cit-lpage">123</span> in <span class="cit-source">Mammalian Protein Metabolism</span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.30" data-doi="10.1534/genetics.116.188102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Booker</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inferring the Frequency Spectrum of Derived Variants to Quantify Adaptive Molecular Evolution in Protein-Coding Genes of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">975</span>–<span class="cit-lpage">984</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.188102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.188102%26rft_id%253Dinfo%253Apmid%252F27098912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzIvOTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.31" data-doi="10.1093/molbev/msh269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Patterns of Polymorphism and Divergence from Noncoding Sequences of <em>Drosophila melanogaster</em> and <em>D. simulans</em>: Evidence for Nonequilibrium Processes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">51</span>–<span class="cit-lpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msh269</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh269%26rft_id%253Dinfo%253Apmid%252F15456897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msh269&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15456897&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225730100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.32" data-doi="10.1007/PL00006557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Recent selection on synonymous codon usage in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">49</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">343</span>–<span class="cit-lpage">351</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/PL00006557</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKliman%26rft.auinit1%253DR.%2BM.%26rft.volume%253D49%26rft.issue%253D3%26rft.spage%253D343%26rft.epage%253D351%26rft.atitle%253DRecent%2Bselection%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006557%26rft_id%253Dinfo%253Apmid%252F10473775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10473775&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083473400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.33" data-doi="10.1007/s00239-003-2510-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Molecular phylogeny of the Drosophila melanogaster species subgroup.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKo%26rft.auinit1%253DW.%2BY.%26rft.volume%253D57%26rft.issue%253D5%26rft.spage%253D562%26rft.epage%253D573%26rft.atitle%253DMolecular%2Bphylogeny%2Bof%2Bthe%2BDrosophila%2Bmelanogaster%2Bspecies%2Bsubgroup.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-003-2510-x%26rft_id%253Dinfo%253Apmid%252F14738315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-003-2510-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14738315&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186257600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.34" data-doi="10.1007/BF02198858"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koshi</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Probabilistic reconstruction of ancestral protein sequences.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">313</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02198858</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKoshi%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D320%26rft.atitle%253DProbabilistic%2Breconstruction%2Bof%2Bancestral%2Bprotein%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02198858%26rft_id%253Dinfo%253Apmid%252F8919883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02198858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8919883&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UC69800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.35" data-doi="10.1016/j.ajhg.2014.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lachance</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Biased Gene Conversion Skews Allele Frequencies in Human Populations, Increasing the Disease Burden of Recessive Alleles.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">408</span>–<span class="cit-lpage">420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2014.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D408%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.09.008%26rft_id%253Dinfo%253Apmid%252F25279983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25279983&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.36" data-doi="10.1093/molbev/mss231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lartillot</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Interaction between selection and biased gene conversion in Mammalian Protein-Coding Sequence Evolution Revealed by a Phylogenetic Covariance Analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">356</span>–<span class="cit-lpage">368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mss231</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss231%26rft_id%253Dinfo%253Apmid%252F23024185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23024185&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314122000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.37" data-doi="10.1016/j.cub.2003.12.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maside</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selection on Codon Usage in <em>Drosophila americana</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">150</span>–<span class="cit-lpage">154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2003.12.055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DMaside%26rft.auinit1%253DX.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D154%26rft.atitle%253DSelection%2Bon%2Bcodon%2Busage%2Bin%2BDrosophila%2Bamericana.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.12.055%26rft_id%253Dinfo%253Apmid%252F14738738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.12.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14738738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188504400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.38" data-doi="10.1534/genetics.115.177386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evaluation of Ancestral Sequence Reconstruction Methods to Infer Nonstationary Patterns of Nucleotide Substitution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">873</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177386</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177386%26rft_id%253Dinfo%253Apmid%252F25948563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzMvODczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE3NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.39" data-doi="10.1093/molbev/msw027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baeza-Centurion</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Codon Usage Selection Can Bias Estimation of the Fraction of Adaptive Amino Acid Fixations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1580</span>–<span class="cit-lpage">1589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw027%26rft_id%253Dinfo%253Apmid%252F26873577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26873577&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.40" data-doi="10.1038/351652a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Adaptive protein evolution at the Adh locus in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">351</span>(<span class="cit-issue">6328</span>): <span class="cit-fpage">652</span>–<span class="cit-lpage">654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/351652a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcDonald%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D351%26rft.issue%253D6328%26rft.spage%253D652%26rft.epage%253D654%26rft.atitle%253DAdaptive%2Bprotein%2Bevolution%2Bat%2Bthe%2BAdh%2Blocus%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F351652a0%26rft_id%253Dinfo%253Apmid%252F1904993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/351652a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1904993&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT11200066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.41" data-doi="10.1093/molbev/msr104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muyle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serres-Giardi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ressayre</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Escobar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GC-Biased Gene Conversion and Selection Affect GC Content in the <em>Oryza</em> Genus (rice).</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">2695</span>–<span class="cit-lpage">2706</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr104%26rft_id%253Dinfo%253Apmid%252F21504892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21504892&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294552700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.42" data-doi="10.1073/pnas.80.20.6278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagylaki</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Evolution of a finite population under gene conversion.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">6278</span>–<span class="cit-lpage">6281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.80.20.6278</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNagylaki%26rft.auinit1%253DT.%26rft.volume%253D80%26rft.issue%253D20%26rft.spage%253D6278%26rft.epage%253D6281%26rft.atitle%253DEvolution%2Bof%2Ba%2Bfinite%2Bpopulation%2Bunder%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.20.6278%26rft_id%253Dinfo%253Apmid%252F6578508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMjAvNjI3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNS8xNzU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.43" data-doi="10.1093/molbev/msl146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer DuMont</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Maximum Likelihood Estimation of Ancestral Codon Usage Bias Parameters in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">228</span>–<span class="cit-lpage">235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msl146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl146%26rft_id%253Dinfo%253Apmid%252F17041152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17041152&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243069700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.44" data-doi="10.1080/10635150490522232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pagel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meade</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorne</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Bayesian Estimation of Ancestral Character States on Phylogenies.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">673</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10635150490522232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150490522232%26rft_id%253Dinfo%253Apmid%252F15545248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150490522232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15545248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224960100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perna</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kocher</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Unequal base frequencies and the estimation of substitution rates.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DPerna%26rft.auinit1%253DN.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D359%26rft.epage%253D361%26rft.atitle%253DUnequal%2BBase%2BFrequencies%2Band%2Bthe%2BEstimation%2Bof%2BSubstitution%2BRates%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1995QK08500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.46" data-doi="10.1093/gbe/evs097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ting</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">H.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langley</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population Genomic Analysis of Base Composition Evolution in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1245</span>–<span class="cit-lpage">1255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evs097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs097%26rft_id%253Dinfo%253Apmid%252F23160062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evs097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160062&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.47" data-doi="10.1007/s00239-003-0030-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sezzi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moriyama</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleason</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caccone</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Analysis of a Shift in Codon Usage in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">0</span>): <span class="cit-fpage">S214</span>–<span class="cit-lpage">S225</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-003-0030-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BEvol.%26rft.volume%253D57%26rft.spage%253DS214%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-003-0030-3%26rft_id%253Dinfo%253Apmid%252F15008418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-003-0030-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15008418&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186396000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dykhuizen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DuBose</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mutangadura-Mhlanga</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Distribution and abundance of insertion sequences among natural isolates of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSawyer%26rft.auinit1%253DS.%2BA.%26rft.volume%253D115%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D63%26rft.atitle%253DDistribution%2Band%2BAbundance%2Bof%2BInsertion%2BSequences%2BAmong%2BNatural%2BIsolates%2Bof%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Apmid%252F3030884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTE1LzEvNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.49" data-doi="10.1155/2008/257864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sethupathy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannenhalli</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A Tutorial of the Poisson Random Field Model in Population Genetics.</span> <abbr class="cit-jnl-abbrev">Adv. Bioinforma.</abbr> <span class="cit-vol">2008</span>: <span class="cit-fpage">257864</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2008/257864</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BBioinforma.%26rft.volume%253D2008%26rft.spage%253D257864%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2008%252F257864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2008/257864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.50" data-doi="10.1093/molbev/msp071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Strong Evidence for Lineage and Sequence Specificity of Substitution Rates and Patterns in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1591</span>–<span class="cit-lpage">1605</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msp071</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp071%26rft_id%253Dinfo%253Apmid%252F19351792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19351792&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266966200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.51" data-doi="10.1093/oxfordjournals.molbev.a003841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takano-Shimizu</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Local changes in GC/AT substitution biases and in crossover frequencies on <em>Drosophila</em> chromosomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">606</span>–<span class="cit-lpage">619</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a003841</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DTakano-Shimizu%26rft.auinit1%253DT.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D606%26rft.epage%253D619%26rft.atitle%253DLocal%2BChanges%2Bin%2BGC%252FAT%2BSubstitution%2BBiases%2Band%2Bin%2BCrossover%2BFrequencies%2Bon%2BDrosophila%2BChromosomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003841%26rft_id%253Dinfo%253Apmid%252F11264413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a003841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11264413&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167898300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.52" data-doi="10.1093/molbev/mst036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terekhanova</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazykin</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neverov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seplyarskiy</span>  <span class="cit-name-given-names">V. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prevalence of Multinucleotide Replacements in Evolution of Primates and <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1315</span>–<span class="cit-lpage">1325</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst036%26rft_id%253Dinfo%253Apmid%252F23447710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23447710&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.53" data-doi="10.1016/0040-5809(77)90023-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guess</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Is the most frequent allele the oldest?</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">141</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(77)90023-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D141%26rft.epage%253D160%26rft.atitle%253DIs%2Bthe%2Bmost%2Bfrequent%2Ballele%2Bthe%2Boldest%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252877%252990023-5%26rft_id%253Dinfo%253Apmid%252F867285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(77)90023-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=867285&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977DG64500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.54" data-doi="10.1186/s13059-014-0549-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boussau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romiguier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evidence for GC-biased gene conversion as a driver of between-lineage differences in avian base composition.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">549</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0549-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D549%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0549-1%26rft_id%253Dinfo%253Apmid%252F25496599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0549-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25496599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.55" data-doi="10.1371/journal.pgen.1002395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Population Genetics-Phylogenetics Approach to Inferring Natural Selection in Coding Sequences.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1002395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWilson%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De1002395%26rft.epage%253De1002395%26rft.atitle%253DA%2Bpopulation%2Bgenetics-phylogenetics%2Bapproach%2Bto%2Binferring%2Bnatural%2Bselection%2Bin%2Bcoding%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002395%26rft_id%253Dinfo%253Apmid%252F22144911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22144911&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1755.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-source">Evolution and the Genetics of Populations. Vol. 2 The Theory of Gene Frequencies</span>. <span class="cit-publ-name">University of Chicago Press</span>, <span class="cit-publ-loc">Chicago</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1755.57" data-doi="10.1093/acprof:oso/9780198567028.001.0001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Computational Molecular Evolution</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/acprof:oso/9780198567028.001.0001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2BMolecular%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Facprof%253Aoso%252F9780198567028.001.0001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/acprof:oso/9780198567028.001.0001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.58" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: Phylogenetic Analysis by Maximum Likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1755.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1755.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A new method of inference of ancestral nucleotide and amino acid sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1641</span>–<span class="cit-lpage">1650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D141%26rft.issue%253D4%26rft.spage%253D1641%26rft.epage%253D1650%26rft.atitle%253DA%2Bnew%2Bmethod%2Bof%2Binference%2Bof%2Bancestral%2Bnucleotide%2Band%2Bamino%2Bacid%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F8601501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS80LzE2NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTc1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/5/1755.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1747" title="Site-Specific Fat-1 Knock-In Enables Significant Decrease of n-6PUFAs/n-3PUFAs Ratio in Pigs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1771" title="A Bayesian Genomic Regression Model with Skew Normal Random Errors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/5"><span>Volume 8 Issue 5, May 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/5.cover-source.jpg?itok=Bl7MXdvV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1755.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vBOXDgREV4cAkMQHvkhPM2O3Z9n8aGbhPC3hxIn2I3s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/5/1755" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/5/1755" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/5/1755" class="active" data-icon-position="" data-hide-link-title="0">Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489477" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5d1ZmiNZ3lioeFyQz13fKJa0PzcYHmM3cXsF_-eE1Ok" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1755.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/5/1755" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6HZ4MSOfKt1ORj22Nfo_8T5fYC-T0qkFVrTZH2P4TAA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/5/1755" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1755.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489477" id="node489477--2" data-pisa="ggg;8/5/1755" data-pisa-master="ggg;g3.117.300512" data-apath="/ggg/8/5/1755.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3456-7553" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tomotaka Matsumoto</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5325-6999" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hiroshi Akashi</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1755-1769;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300512 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489477--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tomotaka Matsumoto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomotaka%2BMatsumoto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matsumoto%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATomotaka%2BMatsumoto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3456-7553" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tomotaka Matsumoto</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hiroshi Akashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BAkashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akashi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHiroshi%2BAkashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5325-6999" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hiroshi Akashi</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hiakashi@nig.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiakashi@nig.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/5/1755" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/5/1755&amp;title=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/5/1755&amp;title=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/5/1755&amp;title=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/5/1755&amp;text=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/5/1755&amp;title=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/5/1755&amp;t=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/5/1755&amp;title=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/5/1755&amp;title=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489477/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489477" id="node489477--3" data-pisa="ggg;8/5/1755" data-pisa-master="ggg;g3.117.300512" data-apath="/ggg/8/5/1755.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Distinguishing Among Evolutionary Forces Acting on Genome-Wide Base Composition: Computer Simulation Analysis of Approximate Methods for Inferring Site Frequency Spectra of Derived Mutations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3456-7553" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Tomotaka Matsumoto</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5325-6999" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hiroshi Akashi</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1755-1769;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300512 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489477--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tomotaka Matsumoto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tomotaka%2BMatsumoto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matsumoto%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATomotaka%2BMatsumoto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3456-7553" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tomotaka Matsumoto</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Hiroshi Akashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Evolutionary Genetics, National Institute of Genetics, Mishima, Shizuoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hiroshi%2BAkashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akashi%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHiroshi%2BAkashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5325-6999" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hiroshi Akashi</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hiakashi@nig.ac.jp" class="" data-icon-position="" data-hide-link-title="0">hiakashi@nig.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1755&amp;rft.epage=1769&amp;rft.atitle=Distinguishing%20Among%20Evolutionary%20Forces%20Acting%20on%20Genome-Wide%20Base%20Composition%3A%20Computer%20Simulation%20Analysis%20of%20Approximate%20Methods%20for%20Inferring%20Site%20Frequency%20Spectra%20of%20Derived%20Mutations&amp;rft.volume=8&amp;rft.issue=5&amp;rft.date=2018-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Matsumoto%2C+Tomotaka&amp;rft.au=Akashi%2C+Hiroshi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489477/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489477/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489477/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489477/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489477/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489477/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489477/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489477/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489477/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489477/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489477/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489477/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/5/1755" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/5/1755.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/5/1755.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/5/1755.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/5/1755.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
