<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/5/1513" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/5/1513.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/5/1513.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/5/1513.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/5/1513.atom" />
<meta name="HW.pisa" content="ggg;8/5/1513" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of NCED4 in Lettuce (Lactuca sativa)" />
<meta name="DC.Identifier" content="10.1534/g3.117.300396" />
<meta name="DC.Date" content="2018-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Bertier et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="CRISPR/Cas9 is a transformative tool for making targeted genetic alterations. In plants, high mutation efficiencies have been reported in primary transformants. However, many of the mutations analyzed were somatic and therefore not heritable. To provide more insights into the efficiency of creating stable homozygous mutants using CRISPR/Cas9, we targeted LsNCED4 ( 9-cis-EPOXYCAROTENOID DIOXYGENASE4) , a gene conditioning thermoinhibition of seed germination in lettuce. Three constructs, each capable of expressing Cas9 and a single gRNA targeting different sites in LsNCED4 , were stably transformed into lettuce (Lactuca sativa) cvs. Salinas and Cobham Green. Analysis of 47 primary transformants (T1) and 368 T2 plants by deep amplicon sequencing revealed that 57% of T1 plants contained events at the target site: 28% of plants had germline mutations in one allele indicative of an early editing event (mono-allelic), 8% of plants had germline mutations in both alleles indicative of two early editing events (bi-allelic), and the remaining 21% of plants had multiple low frequency mutations indicative of late events (chimeric plants). Editing efficiency was similar in both genotypes, while the different gRNAs varied in efficiency. Amplicon sequencing of 20 T1 and more than 100 T2 plants for each of the three gRNAs showed that repair outcomes were not random, but reproducible and characteristic for each gRNA. Knockouts of NCED4 resulted in large increases in the maximum temperature for seed germination, with seeds of both cultivars capable of germinating &gt;70% at 37°. Knockouts of NCED4 provide a whole-plant selectable phenotype that has minimal pleiotropic consequences. Targeting NCED4 in a co-editing strategy could therefore be used to enrich for germline-edited events simply by germinating seeds at high temperature." />
<meta name="DC.Contributor" content="Lien D. Bertier" />
<meta name="DC.Contributor" content="Mily Ron" />
<meta name="DC.Contributor" content="Heqiang Huo" />
<meta name="DC.Contributor" content="Kent J. Bradford" />
<meta name="DC.Contributor" content="Anne B. Britt" />
<meta name="DC.Contributor" content="Richard W. Michelmore" />
<meta name="article:published_time" content="2018-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of NCED4 in Lettuce (Lactuca sativa)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;CRISPR/Cas9 is a transformative tool for making targeted genetic alterations. In plants, high mutation efficiencies have been reported in primary transformants. However, many of the mutations analyzed were somatic and therefore not heritable. To provide more insights into the efficiency of creating stable homozygous mutants using CRISPR/Cas9, we targeted &lt;i&gt;LsNCED4&lt;/i&gt; (&lt;i&gt;9-cis-EPOXYCAROTENOID DIOXYGENASE4)&lt;/i&gt;, a gene conditioning thermoinhibition of seed germination in lettuce. Three constructs, each capable of expressing Cas9 and a single gRNA targeting different sites in &lt;i&gt;LsNCED4&lt;/i&gt;, were stably transformed into lettuce (Lactuca sativa) cvs. Salinas and Cobham Green. Analysis of 47 primary transformants (T&lt;sub&gt;1&lt;/sub&gt;) and 368 T&lt;sub&gt;2&lt;/sub&gt; plants by deep amplicon sequencing revealed that 57% of T&lt;sub&gt;1&lt;/sub&gt; plants contained events at the target site: 28% of plants had germline mutations in one allele indicative of an early editing event (mono-allelic), 8% of plants had germline mutations in both alleles indicative of two early editing events (bi-allelic), and the remaining 21% of plants had multiple low frequency mutations indicative of late events (chimeric plants). Editing efficiency was similar in both genotypes, while the different gRNAs varied in efficiency. Amplicon sequencing of 20 T&lt;sub&gt;1&lt;/sub&gt; and more than 100 T&lt;sub&gt;2&lt;/sub&gt; plants for each of the three gRNAs showed that repair outcomes were not random, but reproducible and characteristic for each gRNA. Knockouts of &lt;i&gt;NCED4&lt;/i&gt; resulted in large increases in the maximum temperature for seed germination, with seeds of both cultivars capable of germinating &amp;gt;70% at 37°. Knockouts of &lt;i&gt;NCED4&lt;/i&gt; provide a whole-plant selectable phenotype that has minimal pleiotropic consequences. Targeting &lt;i&gt;NCED4&lt;/i&gt; in a co-editing strategy could therefore be used to enrich for germline-edited events simply by germinating seeds at high temperature.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/05/01" />
<meta name="citation_mjid" content="ggg;8/5/1513" />
<meta name="citation_id" content="8/5/1513" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/5/1513" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/5/1513.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/5/1513.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/5/1513.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300396" />
<meta name="citation_pmid" content="29511025" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1513" />
<meta name="citation_lastpage" content="1521" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lien D. Bertier" />
<meta name="citation_author_institution" content="Genome Center" />
<meta name="citation_author" content="Mily Ron" />
<meta name="citation_author_institution" content="Department of Plant Biology" />
<meta name="citation_author" content="Heqiang Huo" />
<meta name="citation_author_institution" content="Department of Plant Sciences, Seed Biotechnology Center" />
<meta name="citation_author" content="Kent J. Bradford" />
<meta name="citation_author_institution" content="Department of Plant Sciences, Seed Biotechnology Center" />
<meta name="citation_author" content="Anne B. Britt" />
<meta name="citation_author_institution" content="Department of Plant Biology" />
<meta name="citation_author" content="Richard W. Michelmore" />
<meta name="citation_author_institution" content="Departments of Plant Sciences, Molecular &amp; Cellular Biology, Medical Microbiology &amp; Immunology, University of California, Davis, CA 95616" />
<meta name="citation_author_email" content="rwmichelmore@ucdavis.edu" />
<meta name="citation_reference" content="citation_title=Principles of plant genetics and breeding;citation_author=G. Acquaah;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Altpeter;citation_author=N. M. Springer;citation_author=L. E. Bartley;citation_author=A. Blechl;citation_author=T. P. Brutnell;citation_title=Advancing Crop Transformation in the Era of Genome Editing.;citation_volume=28;citation_year=2016;citation_pmid=27335450;citation_doi=10.1105/tpc.16.00196" />
<meta name="citation_reference" content="Anonymous, Crop Values: 2014 Summary. USDA, http:// usda.mannlib.cornell.edu/MannUsda/homepage.do (2015). 11." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Arribere;citation_author=R. T. Bell;citation_author=B. X. H. Fu;citation_author=K. L. Artiles;citation_author=P. S. Hartman;citation_title=Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25161212;citation_doi=10.1534/genetics.114.169730" />
<meta name="citation_reference" content="citation_title=Biotech. Adv;citation_author=L. Bortesi;citation_author=R. Fischer;citation_title=The CRISPR/Cas9 system for plant genome editing and beyond;citation_pages=41-52;citation_volume=33;citation_year=2015;citation_issue=1;citation_pmid=25536441;citation_doi=10.1016/j.biotechadv.2014.12.006" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. G. Doench;citation_author=N. Fusi;citation_author=M. Sullender;citation_author=M. Hegde;citation_author=E. W. Vaimberg;citation_title=Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9.;citation_pages=1-12;citation_volume=34;citation_year=2016;citation_doi=10.1038/nbt.3437" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=D. E. Engler;citation_author=R. G. Grogan;citation_title=Variation in lettuce plants regenerated from protoplasts;citation_pages=426-430;citation_volume=75;citation_year=1984;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=F. Fauser;citation_author=S. Schiml;citation_author=H. Puchta;citation_title=Both CRISPR/Cas-based nucleases and nickases can be used efficiently for genome engineering in Arabidopsis thaliana.;citation_pages=348-359;citation_volume=79;citation_year=2014;citation_pmid=24836556;citation_doi=10.1111/tpj.12554" />
<meta name="citation_reference" content="citation_author=V. Funk;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol.;citation_author=J. Gao;citation_author=G. Wang;citation_author=S. Ma;citation_author=X. Xie;citation_author=X. Wu;citation_title=CRISPR/Cas9-mediated targeted mutagenesis in Nicotiana tabacum.;citation_pages=99-110;citation_volume=87;citation_year=2015;citation_pmid=25344637;citation_doi=10.1007/s11103-014-0263-0" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=S. B. Gelvin;citation_title=Plant proteins involved in Agrobacterium-mediated genetic transformation.;citation_pages=45-68;citation_volume=48;citation_year=2010;citation_pmid=20337518;citation_doi=10.1146/annurev-phyto-080508-081852" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Plant Sci;citation_author=S. B. Gelvin;citation_title=Traversing the Cell: Agrobacterium T-DNA's Journey to the Host Genome.;citation_pages=52-52;citation_volume=3;citation_year=2012;citation_pmid=22645590;citation_doi=10.3389/fpls.2012.00052" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Haeussler;citation_author=K. Schönig;citation_author=H. Eckert;citation_author=A. Eschstruth;citation_author=J. Mianné;citation_title=Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR.;citation_pages=148;citation_volume=17;citation_year=2016;citation_pmid=27380939;citation_doi=10.1186/s13059-016-1012-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=H. Huo;citation_author=P. Dahal;citation_author=K. Kunusoth;citation_author=C. M. McCallum;citation_author=K. J. Bradford;citation_title=Expression of 9-cis-EPOXYCAROTENOID DIOXYGENASE4 is essential for thermoinhibition of lettuce seed germination but not for seed development or stress tolerance.;citation_volume=25;citation_year=2013;citation_pmid=23503626;citation_doi=10.1105/tpc.112.108902" />
<meta name="citation_reference" content="citation_author=T. B. Jacobs;citation_author=P. R. LaFayette;citation_author=R. J. Schmitz;citation_author=W. A. Parrott;citation_pages=16;citation_volume=15;citation_year=2015;citation_pmid=25879861;citation_doi=10.1186/s12896-015-0131-2" />
<meta name="citation_reference" content="citation_author=R. Kesseli;citation_author=R. Michelmore;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Kim;citation_title=Highly Efficient RNA-guide genome editing in human cells via delivery of purified Cas9 ribonucleoproteins.;citation_pages=1-32;citation_volume=128;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Kim;citation_author=T. Ishidate;citation_author=K. S. Ghanta;citation_author=M. Seth;citation_author=D. Conte;citation_title=A Co-CRISPR strategy for efficient genome editing in Caenorhabditis elegans.;citation_volume=197;citation_year=2014;citation_pmid=24879462;citation_doi=10.1534/genetics.114.166389" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=H. Kim;citation_author=S.-T. Kim;citation_author=J. Ryu;citation_author=B.-C. Kang;citation_author=J.-S. Kim;citation_title=CRISPR/Cpf1-mediated DNA-free plant genome editing.;citation_pages=14406;citation_volume=8;citation_year=2017;citation_doi=10.1038/ncomms14406" />
<meta name="citation_reference" content="citation_author=I. Kozarewa;citation_author=D. J. Cantliffe;citation_author=R. T. Nagata;citation_author=P. J. Stoffella;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=S. Kumar;citation_author=P. Barone;citation_author=M. Smith;citation_title=Gene targeting and transgene stacking using intra genomic homologous recombination in plants.;citation_pages=11;citation_volume=12;citation_year=2016;citation_doi=10.1186/s13007-016-0111-0" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=C. Le Blanc;citation_author=F. Zhang;citation_author=J. Mendez;citation_author=Y. Lozano;citation_author=K. Chatpar;citation_title=Increased efficiency of targeted mutagenesis by CRISPR/Cas9 in plants using heat stress.;citation_pages=377-386;citation_volume=9;citation_year=2017;citation_doi=10.1111/tpj.13782" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=H. Lindsay;citation_author=A. Burger;citation_author=B. Biyong;citation_author=A. Felker;citation_author=C. Hess;citation_title=CrispRVariants charts the mutation spectrum of genome engineering experiments.;citation_pages=701-703;citation_volume=34;citation_year=2016;citation_doi=10.1038/nbt.3628" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=X. Ma;citation_author=Q. Zhu;citation_author=Y. Chen;citation_author=Y. G. Liu;citation_title=CRISPR/Cas9 platforms for genome editing in plants: developments and applications.;citation_pages=961-974;citation_volume=9;citation_year=2016;citation_pmid=27108381;citation_doi=10.1016/j.molp.2016.04.009" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=M. Malnoy;citation_author=R. Viola;citation_author=M. H. Jung;citation_author=O. J. Koo;citation_author=S. Kim;citation_title=DNA-free genetically edited grapevine and apple protoplast using CRISPR/Cas9 ribonucleoproteins.;citation_pages=1904;citation_volume=7;citation_year=2016;citation_doi=10.3389/fpls.2016.01904" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=R. Michelmore;citation_author=E. Marsh;citation_author=S. Seely;citation_author=B. Landry;citation_title=Transformation of lettuce (Lactuca sativa) mediated by Agrobacterium tumefaciens.;citation_pages=439-442;citation_volume=6;citation_year=1987;citation_pmid=24248927;citation_doi=10.1007/BF00272777" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Rep.;citation_author=M. Mikami;citation_author=S. Toki;citation_author=M. Endo;citation_title=Parameters affecting frequency of CRISPR / Cas9 mediated targeted mutagenesis in rice.;citation_pages=1807-1815;citation_volume=34;citation_year=2015;citation_pmid=26134856;citation_doi=10.1007/s00299-015-1826-5" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Genomics;citation_author=B. Mou;citation_title=Mutations in lettuce improvement.;citation_pages=1-7;citation_volume=2011;citation_year=2011;citation_doi=10.1155/2011/723518" />
<meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.;citation_author=N. Podevin;citation_author=H. V. Davies;citation_author=F. Hartung;citation_author=F. Nogué;citation_author=J. M. Casacuberta;citation_title=Site-directed nucleases: A paradigm shift in predictable, knowledge-based plant breeding.;citation_pages=375-383;citation_volume=31;citation_year=2013;citation_pmid=23601269;citation_doi=10.1016/j.tibtech.2013.03.004" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=H. Puchta;citation_title=Applying CRISPR/Cas for genome engineering in plants: the best is yet to come.;citation_pages=1-8;citation_volume=36;citation_year=2017;citation_doi=10.1016/j.pbi.2016.11.011" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Reyes-Chin-Wo;citation_author=Z. Wang;citation_author=X. Yang;citation_author=A. Kozik;citation_author=S. Arikit;citation_title=Genome assembly with in vitro proximity ligation data and whole-genome triplication in lettuce.;citation_pages=14953;citation_volume=8;citation_year=2017;citation_doi=10.1038/ncomms14953" />
<meta name="citation_reference" content="citation_journal_title=J. Cell. Physiol.;citation_author=K. Rodgers;citation_author=M. Mcvey;citation_title=Error-prone repair of DNA double-strand breaks.;citation_pages=15-24;citation_volume=231;citation_year=2016;citation_doi=10.1002/jcp.25053" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=S. Schiml;citation_author=H. Puchta;citation_title=Revolutionizing plant biology: multiple ways of genome engineering by CRISPR/Cas.;citation_pages=8;citation_volume=12;citation_year=2016;citation_pmid=26823677;citation_doi=10.1186/s13007-016-0103-0" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Index Switching Causes “Spreading-Of-Signal” Among Multiplexed Samples In Illumina HiSeq 4000 DNA Sequencing.;citation_author=R. Sinha;citation_author=G. Stanley;citation_author=G. S. Gulati;citation_author=C. Ezran;citation_author=K. J. Travaglini;citation_pages=125724;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=Y. Sung;citation_author=D. J. Cantliffe;citation_author=R. T. Nagata;citation_title=Seed developmental temperature regulation of thermotolerance in lettuce.;citation_pages=700-705;citation_volume=123;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. J. Truco;citation_author=R. Antonise;citation_author=D. Lavelle;citation_author=O. Ochoa;citation_author=A. Kozik;citation_title=A high-density, integrated genetic linkage map of lettuce (Lactuca spp.).;citation_pages=735-746;citation_volume=115;citation_year=2007;citation_issue=6;citation_pmid=17828385;citation_doi=10.1007/s00122-007-0599-9" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. J. Truco;citation_author=H. Ashrafi;citation_author=A. Kozik;citation_author=H. van Leeuwen;citation_author=J. Bowers;citation_title=An Ultra High-Density, Transcript-Based, Genetic Map of Lettuce.;citation_volume=3;citation_year=2013;citation_pmid=23550116;citation_doi=10.1534/g3.112.004929" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=H. Tsutsui;citation_author=T. Higashiyama;citation_title=PKAMA-ITACHI vectors for highly efficient CRISPR/Cas9-mediated gene knockout in Arabidopsis thaliana.;citation_volume=58;citation_year=2017;citation_pmid=27856772;citation_doi=10.1093/pcp/pcw191" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=M. Van Overbeek;citation_author=D. Capurso;citation_author=M. M. Carter;citation_author=G. R. Hoffman;citation_author=C. K. Fuller;citation_title= Article DNA Repair Profiling Reveals Nonrandom Outcomes at Cas9-Mediated Breaks.;citation_pages=633-646;citation_volume=63;citation_year=2016;citation_doi=10.1016/j.molcel.2016.06.037" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=D. F. Voytas;citation_title=Plant genome engineering with sequence-specific nucleases.;citation_pages=327-350;citation_volume=64;citation_year=2013;citation_pmid=23451779;citation_doi=10.1146/annurev-arplant-042811-105552" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. W. Woo;citation_author=J. Kim;citation_author=S. Il Kwon;citation_author=C. Corvalán;citation_author=S. W. Cho;citation_title=DNA-free genome editing in plants with preassembled CRISPR-Cas9 ribonucleoproteins.;citation_pages=1162-1164;citation_volume=33;citation_year=2015;citation_pmid=26479191;citation_doi=10.1038/nbt.3389" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Y. Zhao;citation_author=C. Zhang;citation_author=W. Liu;citation_author=W. Gao;citation_author=C. Liu;citation_title=An alternative strategy for targeted gene replacement in plants using a dual-sgRNA/Cas9 design.;citation_pages=23890;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep23890" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=C. Zhu;citation_author=L. Bortesi;citation_author=C. Baysal;citation_author=R. M. Twyman;citation_author=R. Fischer;citation_title=Characteristics of Genome Editing Mutations in Cereal Crops.;citation_pages=38-52;citation_volume=22;citation_year=2016;citation_doi=10.1016/j.tplants.2016.08.009" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of NCED4 in Lettuce (Lactuca sativa)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
<meta name="twitter:description" content="CRISPR/Cas9 is a transformative tool for making targeted genetic alterations. In plants, high mutation efficiencies have been reported in primary transformants. However, many of the mutations analyzed were somatic and therefore not heritable. To provide more insights into the efficiency of creating stable homozygous mutants using CRISPR/Cas9, we targeted LsNCED4 ( 9-cis-EPOXYCAROTENOID DIOXYGENASE4) , a gene conditioning thermoinhibition of seed germination in lettuce. Three constructs, each capable of expressing Cas9 and a single gRNA targeting different sites in LsNCED4 , were stably transformed into lettuce (Lactuca sativa) cvs. Salinas and Cobham Green. Analysis of 47 primary transformants (T1) and 368 T2 plants by deep amplicon sequencing revealed that 57% of T1 plants contained events at the target site: 28% of plants had germline mutations in one allele indicative of an early editing event (mono-allelic), 8% of plants had germline mutations in both alleles indicative of two early editing events (bi-allelic), and the remaining 21% of plants had multiple low frequency mutations indicative of late events (chimeric plants). Editing efficiency was similar in both genotypes, while the different gRNAs varied in efficiency. Amplicon sequencing of 20 T1 and more than 100 T2 plants for each of the three gRNAs showed that repair outcomes were not random, but reproducible and characteristic for each gRNA. Knockouts of NCED4 resulted in large increases in the maximum temperature for seed germination, with seeds of both cultivars capable of germinating &gt;70% at 37°. Knockouts of NCED4 provide a whole-plant selectable phenotype that has minimal pleiotropic consequences. Targeting NCED4 in a co-editing strategy could therefore be used to enrich for germline-edited events simply by germinating seeds at high temperature." />
<meta name="og-title" property="og:title" content="High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of NCED4 in Lettuce (Lactuca sativa)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/5/1513" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="CRISPR/Cas9 is a transformative tool for making targeted genetic alterations. In plants, high mutation efficiencies have been reported in primary transformants. However, many of the mutations analyzed were somatic and therefore not heritable. To provide more insights into the efficiency of creating stable homozygous mutants using CRISPR/Cas9, we targeted LsNCED4 ( 9-cis-EPOXYCAROTENOID DIOXYGENASE4) , a gene conditioning thermoinhibition of seed germination in lettuce. Three constructs, each capable of expressing Cas9 and a single gRNA targeting different sites in LsNCED4 , were stably transformed into lettuce (Lactuca sativa) cvs. Salinas and Cobham Green. Analysis of 47 primary transformants (T1) and 368 T2 plants by deep amplicon sequencing revealed that 57% of T1 plants contained events at the target site: 28% of plants had germline mutations in one allele indicative of an early editing event (mono-allelic), 8% of plants had germline mutations in both alleles indicative of two early editing events (bi-allelic), and the remaining 21% of plants had multiple low frequency mutations indicative of late events (chimeric plants). Editing efficiency was similar in both genotypes, while the different gRNAs varied in efficiency. Amplicon sequencing of 20 T1 and more than 100 T2 plants for each of the three gRNAs showed that repair outcomes were not random, but reproducible and characteristic for each gRNA. Knockouts of NCED4 resulted in large increases in the maximum temperature for seed germination, with seeds of both cultivars capable of germinating &gt;70% at 37°. Knockouts of NCED4 provide a whole-plant selectable phenotype that has minimal pleiotropic consequences. Targeting NCED4 in a co-editing strategy could therefore be used to enrich for germline-edited events simply by germinating seeds at high temperature." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/5.cover-source.jpg" />
    <title>High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of NCED4 in Lettuce (Lactuca sativa) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6-HXSeQGNBeJSf9_60Pxmt-1cjrTVMj8IujI0o5Vnq8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"489462","apath":"\/ggg\/8\/5\/1513.atom","pisa":"ggg;8\/5\/1513","ac":{"\/ggg\/8\/5\/1513.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/5\/1513.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1513"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/5\/1513"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/5\/1513":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DkrKP7mTlueISKI1\\\\\\\/Pd9\\\\\\\/\\\\\\\/d6g6G8h114VYVxeN0hlxu9s07G\\\\\\\/su2x0NbmYj1ypcqihLh4PJ7p1qIisxRGZMtQsc4QtBtkQiqS5L3CqS\\\\\\\/\\\\\\\/bmrNKHkivZsGLmQpa+LaaNTdij\\\\\\\/f+iNoqAJj1rrfH5jUZbCMOYGZ7Fv+YwLNS1mnohINAfufB8atQYcAeC6XJhdFMSory6fqn6uDW557I6PMkI+5jZSe9TjP5gEI03zGhhQD9\\\\\\\/Xh0yw6qeA+sBuarrI7TAptJEL5YthOQaEsHiGkfd62rXIBdQAqb2iymbWpP5R+jSSx0+at58hnNz2u4x0A7yCQLIhoiZRyj0KsWMiJFWxpkA75IS0+fRYaObFChSUdTJj4JJdgLohkmCYOizwxPNw1bKjIly4DL\\\\\\\/xI0dRg9ciKSpZMc9Ee5VOJLX6oFB0f8+kJEvXJ5wFBEOO4bVAOCm7a4ns+guQaCMKft9PCc2YWl5xEWUxsRRIMcsYbtnmaFm4KXoP2iBM\\\\\\\/rSxMAzajEvqZX9XzGerUGN18eH7fVfUHDz\\\\\\\/L6qvq\\\\\\\/0\\\\\\\/azQg6gGlnQYqeBsa9kFIlEMjsFKfin2dfwcuKPRELxS7uis2LfWUA4kgd\\\\\\\/5pCg3iZ3tu6M\\\\\\\/Gf0HYzbrmVWDSj7AS+nxCBxSKkvCmUY7oXyuLuQ9uwGBSpiU7bUE5f6PJZ1im1O8ZVA2bQtxH1H1oqvddLEQDffIZnXR0Lr3yj3j+eUNUY8AR0pjsb8vQodf+XBNy6SeJHOHo3vkn2TpsaORfAOFLkAz2qG9CHoJJKXcyIbjsNB+sKaxgPHypn4LajRGDzvBcSpyK4MKjyhv2HloDNamK1k3izw2z1wM3bqCIym9QiIk7QHaC+d7yTI3O+2sVPC+IZuImG6eStBOW6SzVvj4o+RPiRLi1VbFpsXcunY99kDs\\\\\\\/tYxGP\\\\\\\/DYVbUooNzPIyRllXYjfcZYwL0Yz6ldT6Sc1Hi7jpvHHCe1NrzWKzqdN\\\\\\\/S3vt5oKfHv6t2rRrrYkrZeqEdj5QCspk1AUDQ3YZqW1tW7ryUTTKrSss3CXDoNuxQCaisfLaNVs7B1DwdQNRCozqDn1X\\\\\\\/5sntNd01ecAPMtb6p+FAhaKpj7WsF1JiMwUdut1k1QBZhT07pR\\\\\\\/K\\\\\\\/dMHM9JlaFGPH03oUg9ohRIoein+NXjFoQtaDY9fpLVmGaCEqPkJL6h2bCNRRnd2ncgyONv4bjVnkHUJX9w7L1HgJ4+pLf2pdB9XPXUnQU6v8aWieRVCDF6wKvMlggO8xgwRi\\\\\\\/hzytjc+AGvpONrtV3A3\\\u0022,\\\u0022iv\\\u0022:\\\u0022vryjJLj8yoYbwZqyYeBFtw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022678e5402ab53e67de8482b24f120a6f68edaf9730e10cbcef356b1ccbb3e112a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PizMdYIMc5KycIkaSC0BrvUbgUXVffQ0w6dypOyXmYaXRPa\\\\\\\/Tf6er6Je4r7n6K9D4r2eSykF3CdunB8tOm5HGkHqmJzNc5gUVSEROQ2LPhSt6jF1\\\\\\\/0WCyIoEU9imYl30bnkZXxbFX5R4e5oNPyAHuyUClQeuv02TxM6TE2NrWzbE4YQ9+mrfD2g5Afppw\\\\\\\/rv\\\\\\\/wykxOG9kmTQmj6wv4ZQV+VKHs7Vq89U62M0vZo0DyBFBZU4LAToGRMVtBl3DeFWF0QT0cuBNHAodh2tiewXkMHxrjZFM23gHMD4leRJ4BiCFW4IcaR33GY8Y+YmptL1oY+BmiZJ\\\\\\\/JJdEwQW6zuOkNoC3PJwxGk49Q1+b6lJnAByCxO5D9C6C+2hPJ9Huk\\\\\\\/dJAWOK5EYvN0SkIiWTas3OVhnh\\\\\\\/zRts8GOJuN2uVkrIxfSUu+9PRcjgY4Fx9CEcpCNqsY17H3qc5zwMLx5tM1yTTDSvGX5cyalpFLZHsRZPcMdpozbUr889JoPNCJiJJ+9oFQGeM+ew1oFNhYwZpnEyZ7FA2ofuhxu4mWz1Fs+DzccgpgSIClu+McbgWJyB8yQkPlsKY2F3bMPp+DemEDW1XAENqmNbHnJ1E9Po7UhzmH7oksACAcEOMygvs3Y31SwXJm1Vj5J4983NtJwdOuOzo1D64hTxF6aXxzCvbPyM1pGrUdMiGRa3bQ9H4rGXT\\\\\\\/Z\\\\\\\/QLt4otQwD0c+\\\\\\\/sST8KaABDH+iljBdXfP2hF84vhTOKZiUMIhzn8jZlDf1oifrdGd959VMcIN3vIX\\\\\\\/iX5cgjl\\\\\\\/OhryDtILKI\\\\\\\/PHUPZ6v0UnibfNCOfwVTXDTi5RpRMQPM17ady50OmsPPWeN5v2qQgswtJSoovhprCN9DAGU\\\\\\\/MQXfzi5TiV4wdeCQRM8630+QdIctr4bZPRC2B4QiBqkO0Lh+oeffY2vVRW1xS5PtGXBriut8fiRWMaZDrYxib84Qio2I9XNvGUv3X+wbxD\\\\\\\/Uwv8xN1ueAJKou+Euri\\\\\\\/k5+pqidmnLC79cDa2CWD+\\\\\\\/s233rnkDQF8EjXZCuV40t1uukAMOJzJZmKMNJR77bqIwrWBTJuT5kgtYNxnFQ9iMORFZE51OadDSymnffV1SqBHTn0fOFzZJl1FEdaXld+3+fjOQkgWxU1GsejCsGRGfHQI0+W6IPAZvSJlwdMWliCMZHFY7CbCg6W5mEY99mobUuED6vT2SKwu3GSgIDZtBtpBxOQ5WcOiUICOe\\\\\\\/qoqBQYj6IycPi1AfeyN7vePQKKhXTDb7VoAptu766YMYQsKVV9PJzXnjIvod+HTi34JWcy60kTAEf+s8b\\\\\\\/\\\\\\\/9frP7xaU+eBHhGQfARgE4vdCdj31RbwZ8OsHx9Z12hPeNjb+pIlINz\\\\\\\/l\\\\\\\/5AfkopQ+FMu1Lk9JHAdKsk7nT4N8I9ola60ZkIzYtK7W+vlpy82Ps6K4ySihL8nWo\\\\\\\/bRLPERTv4ukIbG3cNC3zstRRmkktXen9YiAm8BBykxqZJQUB7VRKUOddAUpK26kn4P7\\\\\\\/YUqj9YJyTg94ExT7rMMBASL2ojlWtpcM11ks2fu6QLE7lqO7cj1P8ZMRyFGreLufT2\\\\\\\/wQmfU8tDPD2wL8xKwB1WsdVUg5JHhVbSvGEUSsqeLjgz+8+WTEMtJEZAQVuNo6zVhu0LLi3+2+clt9s5sW51T\\\\\\\/P8BjNbapDuCn9w6u0tzELogSaMHsn8JBTgBrsKxfQAIwusx\\\\\\\/MJpU75gp2DbyMfkW0k2iPQLsoTGKvZkP8HZP5sjaSjNV49Io\\\\\\\/Lw6vibhr0L08lI7ji7yvr2dg6FOOd8wLvUCPASp5Nv1D++2j0ZsuAMWr\\\\\\\/X+STwlJ7QAsKW1VcTxlYURuzSLglMd4WVFc3P6wMkb+oFlsTYLYpBq3o\\\\\\\/2z+v9YB\\\\\\\/s\\\\\\\/\\\\\\\/dZ3Zx0qOfy+6yBdhoNXtb5BejvX7pRt4p5Ait\\\\\\\/ZxWqCzpQivUXRZwk5bC0ndh2AJyQb5N0iVsO95pkUoNsqweIIdI8WDa0f2EMV+mEZg1D\\\\\\\/SAzAyGIPmuH6cVgludkb2qeazEcgaHRKi4\\\\\\\/I0csBO0kxTzQbbCvdB+TZ9B6UZN8soFQ3MVZXGUnpiIAW98NuPNi79vc\\\\\\\/2StTDrR7VdpY\\\\\\\/cDD0szs+DVyCDFAgQKXwHMWm7+LyBsffdQ0lseFkJapqDRgH8u\\\\\\\/msEhPTevn1MoMSV3Z6wjb6ThpXchLgpi2hq34UM2Yz1P6l9eq6F8+EgDbnJaVjC3uG2AqQaSfZhRefNy+EqEQv\\\\\\\/1bxXuPyMIjgM8tEPKB6\\\\\\\/ZzrwcoTuWlHOe+ney5Lrg9fzu9sq1uf7hLw9FanyOARlojgbBPBHZwdtWJGetucQUhSFzb7CBv9+bQqjd\\\\\\\/UxB\\\\\\\/Zx326DXVnpPW\\\\\\\/uW5JWnRRmVUCj0ZyAwQeN0vVAG\\\\\\\/Pap1isLLC\\\\\\\/gY0HuxzdT0hppfC74d2wRDp3zashEXQz7GwFKJnAemuIMnOKjqoDmD5CM2\\\\\\\/bbkyVP33dT4yxof\\\\\\\/L+DwGY8iJiGm3fsj\\\\\\\/wJNyMHW5iBi28prdETmjlgddNBgbIrzy6m7i5aFwlAdTS7gnPZwWciUkTq2+UfSQ\\\\\\\/ThUt8HUBTR1bM637bdnzLoId3t7W0nYvZUwy2npCodBdcleRWYDaekOYrgku9FN64E6pMyi7R0\\\\\\\/ym6a5S1pAfAJClGIoPN4FlLzDBx\\\\\\\/6QdckhHibGhqd\\\\\\\/4APxNDo8pAiIwZ4Xt8+RL0384iaGJZsk+ZCbnajNj0QPtZY2SeF0U65B8JSuBmSoYHwbohEp8VSFgsxovFdlJMCjuQJ2Bmj5KSg87FGkXYpBvjUARIhyUtbCzY\\\\\\\/h7rdMmdlBDbJ9y0KQCTQpVqDzKvFyHLl34C5jgRDAwEhNhkBxyQkM8yMSeNnMBbdtcvdGGoi3OMsJ+oFVHRhrvfyPQclzuRIiHoY8mMG3yjl23sPV6izpjPPDdh19+QrbUTNhhgDexzO\\\\\\\/dDhMBED8WivWe8B0OYk\\\\\\\/wzOxbsEJ+5Jv+tWKOC1K8EDsjnlVh2ifuo1uRP6y91piZe7WqF8uHj6+Ob269kXbcD\\\\\\\/eCEHqvSNOUqUu2bBUvUyntK1I91Bz2\\\\\\\/hEZ4xuXjLMhEZITKE7AbkauJBw6QFv6rQ8cltHA8j9k\\\\\\\/FdFgTl+AHT50pJmGQWqXxZcUVFsuFup3NqFcSgz7TtmpJfqeeco8bwh0Rt417wXxudFn9j+8I9gCbm2T92FuspbckdA3r0Ef24sPbyzbSoWIUUGICCY3fXq22kLNN6fFjxJYrQQvWQgWt\\\\\\\/qKKuAEnWtUkBOQ5LDsdSAA0gS83V\\\\\\\/tO0M8PWeanLZVhdgqwPZdopPuIbnMWU3psuGS0PdUhVjGwie2p6yqnI+77fAda1B7rLOLBxr+dkEJhxgaa+ewkL6ivtMeWbsNqYR3n63NQmrX5cKzvEPlTiLZme2RN18ewr2yA\\\\\\\/NIWajSEJvJnFldy5duz1LDYODqezroMdbppi\\\\\\\/15ELFY02AWDsK7g+uVP7WAtv3iJP2k\\\\\\\/lY+T\\\\\\\/09y9+70eMP6lyK8R3K9+mKr5+UmyN8Aa\\\\\\\/ql1fgtoD\\\\\\\/Z7aaes8+AS8GY1gnLWECD96vPkbOCLSlFmn6bQTg46HySbo0MFEWFjPNI+CDm\\\\\\\/mQXRTyXXRcL11A6qkvcl\\\\\\\/Q1dnfbOGN8FZtxcNDL9dzRMeWkitSa0ZE4wWGOM6mZPMQqqqPZHPYyHy0esnhhqQqcAQ7DTLCiVM50hVJXAS1qk5Kc+08TuvNDbWUOH4IdNsQcGhRUji8RbuQeupvSpWMpgGKIofVmPimS3bqXAzi5yvbuqB6s6WFrlhjMEolWAkTzIXXJRxxVIuxzECYme1cQ46LBq2Rllmnxqkq3Ll42EY5SHMbqMRqaRMJ7+xCXKpNmgzK389+PdJzZc\\\\\\\/Libi1WKvf5m0lYWmCI3P+8XiMad4UvXVa05Q3Ou\\\\\\\/mkx9wt72YkhpTpkcdbJjN1uPXkzG+\\\\\\\/imQfKxjEUBCwr6RdcucU8bInSUEb\\\\\\\/23QJXFx3\\\\\\\/J4jmmDR3uzDBiFe4JDVh6boxVUxeSW2bYBNjElFfH2iUMkzMV5ZfqXsG\\\\\\\/SnUqsQZkrdUIP\\\\\\\/bXlIt6PcnhTuX78q5OmPtbsLeNYJXqO9CcegIVVUT713h38FCWzg5J0bNpI82wcAYEHiaSqrKRnXdqt33BicvdXvPglKqcLsSc2ZxW8v7Oh0Naho7\\\\\\\/OpnzPy\\\\\\\/KaLWPjsGPLnm5wgCS3LhQg+I+YL496J+QDWcO96wl56jIZw6r0EcTbloB4DtImupFS5lOX4uPD1nbQzc9KooD7MAIOvGICd6pBNwXqJ0fM8OxkwOnVL3e\\\\\\\/LHt+Gdc6syYsbxP+xwVlLZVoHz9DPO4D0XrSCEOr5SthhQafKHs7cTHg30Ig9rY6TE5Wn+EkoSPrwXd4ULFYIHWfRZ\\\\\\\/65048aljl0Cr3pJELXBhFSyCfvKPbAF6On3pZGsx0fUBtBmVhYr7MSWhqZgbUQEpgJ2av8GAjHCMykfLCKMIhT0ubdZhyMd3bBCU42rb3rD4DI\\\\\\\/UmBXGptQr4PTqg+QpVDOM1O+3LGrH2vAFmuYwL1WLbb5Ld0vXRjCDTLFOIJ5AbsInu1GhDfiSUibZqbdyQBL9SxTmh+L8MQPA94PM6BjPXwLSgt8bbKlAoIVK1\\\\\\\/CW3q\\\\\\\/YjHOkADCvhinTswTW6Yi1u7Pwd0LCJlXV5RGhPYx0Eo1Xfi98W278sCwxx5EOU6FFvjb2AYHrTq42qh7egVjtDFgumDHAaLzm6e\\\\\\\/zTpG9Hrco+tLR3SSobAnmDagkGHsNQxEIa9sAlnabvgI8nwQoetJHQakNjhSe2xAOK3RJ0Klk29HlgnVKg1tpKB1sCSPmP64qKoZodawmTRGEQq9OfNebHGxzMFeAT92jN7+Ybs\\\\\\\/pkOnzMDqaPdb0AYWYYE\\\\\\\/RgKG8KzNtRN4hsAIzWi16TiGjwS7W7qlpqQLoD1lzElV\\\\\\\/RPDj9T5wYm0EZPePqxQQFPtr2goPyAvckd+gN9ZwLttPzPLkBpvWY5\\\\\\\/jfee9B7LnRaDYZ3lWk\\\\\\\/n9g0UCup0xS4TJAzTGTjT\\\\\\\/Go9I8S2Mz\\\\\\\/P2UTvGumbuiz97T1BaVq3QzT1yqhe\\\\\\\/iUT4oBFqVFIZPu8IGtBYWTzCYWUCAoQtyxXqfucX6VIhJ9oDLvX3FogJF+biILhWA1oBna0Ttoo5TVGlxOCuk2iwXX+r5akrovYgIT5OIGKgTNFLP9y0bZqFco3GuIas3i00wku34s7Kt7jGcIMQQZuQ2zN8lhChT\\\\\\\/u4eNBtNYA0V34DkpkIh4Swn4uR7YB1OO17Q0rxzHY3gnRpL1513LbuJrVwF8aELK\\\\\\\/vjRd9EXcWLkxahWH18XSl5qHvk+5a3y7\\\\\\\/i9v1U\\\\\\\/YbMPnsFn3PyrQ0pxy8VI2io5aCSlwAHvjk4QEKfh5zpRLrsWkoaI\\\\\\\/tBScuEr0qn3da0yG2T5UwhlisRO+IryjYd\\\\\\\/mfsLDT4P\\\\\\\/FASefPnBiHxZOlYYX\\\\\\\/yvVBtO\\\\\\\/MCawVTV6Y2uTX0nnSxSHKX4F1IdqkXaSYt3VU\\\\\\\/\\\\\\\/bQr1sLC4tP3+2QF1DOGUQLu9tjs+INVFoqQPz5QviQJKt7G+d+IT0stQTUFDf\\\\\\\/FP3ah4kYmfuztwXSee6c5Y061IUMMZJJf7Wim9NZ8rP2f0hi0xBypfcPog8VXIAKKR+AtVvBwISV22dSedZLv5ImCP+QE+dgGwWD43Q389egeZfM9H\\\\\\\/LlhxmQCrUjkiDNc\\\\\\\/8bPA0dAUJii5Rv2hp6\\\\\\\/A6UGOTPDGcKhiKhkCtFPHjHkLzP+v0WB16ER6AHqv6VKXTuSaoIO3AN9ldlKYOHGDn2WFNOiMpCBpPDyZ1ItIpxwrpNTF5YlP8gR+caJXJ96pcTECZqswsOl+UxtxAHAxUvA7nUm0qM+y\\\\\\\/BtusT\\\\\\\/0l00pZVbPhsLIAK3oYr8iRamcbMoSAdg1BryXp6bDzOkWCTVjjNR92ENfdCGmIfXOE6ND\\\\\\\/DKT2j7326RlwwfSVKGJWlG3dJwdcmJCsOAIdtyrjtzWaCGpyUNdIOdidQDBzSksyvlUnWi9dcrRTuz+z2QSflHg\\\\\\\/nPI17FWYFwOVdIUnJ1PIZ5DiXd2Rh9fo0pmfEVoMmrqXMOabyof62RdzYgAG7EqINQWrHZV4pfzxLHqx3rsyu8JW+8m3f8ESsoDxEX2AF65PDOWZhtfLt21tU\\\\\\\/IZ7oZpFYrtnjZespJ\\\\\\\/Kypn\\\\\\\/DRWZxFwGIRTtRFq\\\\\\\/5Q\\\\\\\/iq83OpcGJrem3JgmPZHMpykmT9oDeowwPmZMJxvOcuVFbv8++vAkmGB7eq7yrH1fPA57lEBkeuSMiTy0HqC691JWst1jofEHq9txaTD9\\\\\\\/39G8DaKdyUNuA6dbmg9cyExbF2VLD6ywso5yo9s2Clr8BE38xk66c\\\\\\\/++JOM\\\\\\\/hkwwJA6qcWlVibnUOlFW381zs0Q0id1mk7AYBuRAiqvoQF3\\\\\\\/5rPAJwS2RgOCusJd\\\\\\\/VUwMwqKziMQP60nxpRJRXswE2ql47wk2V1FlqmtzgNPauKwEGD4X9mc5pLXiA1rFT4DIJl8E2PWB76sA3OI12ehjyR+zFGfvFUwOTLs811utYLG3eCLh+gbGQ4NPXl+ddLWl\\\\\\\/zDxtcqZinBvT10xlTgXnvZ75Bl9uzuYI9r9lPwimKUnU4F+7HMFseaurqo5voE6yTvK1\\\\\\\/Ur3MWWC5DF8tOqBWH+391XU1gBXcNKhQ4vM9wqNlB6+9cpcxPdfjQQhGv+F4q5pU08xtyM5GYZl21FvPhfVbMyUFjtR7y+e5dAoHrDy5QF6WhJWX91wuUSBHUP7OiNG1AMhxKOrucOKkMETe6XbzuPEV2ybiIRvt7SzbewMUnDOMpr9sgZVyPT5QkDRcZoPZqU9LcT1RbldXBCR2U7RrjsOnZAFCb269E1j\\\\\\\/atPFL0adk7fEVfaqZhUB\\\\\\\/827q3p2iyHaLu1enkFjQwuOq\\\\\\\/cz4\\\\\\\/auSFcZ3+KhaM7pX5jH\\\\\\\/+QSNFf9rHpIQv\\\\\\\/KJEZ+Z6+bki31NzynVgOEUeFSndrSoiw2CHeWbwteIJfYSgHsBGDFQPOTTA4ZUHEoPlVWikMr36nbm83f69PWE9MRUt10lXhssaSDygSgK6SyiZ50Q\\\\\\\/E9anzfGt2fefsG5TbzJVqnCOanlL2ZuWgELDzs62kvbm2jecpoBBav5ka8K6JtWS\\\\\\\/fHyG7als8u3gt8sedk9ZHfCUwowTuEmQp4aPW5Rp3QYT+MhktAuxuW54kEfF5s42wZ1JWmO+\\\\\\\/p3yNWn+wiJk1PIfhgEGBb1BM\\\\\\\/RjMJ9OvrTS678MT7IDXqwJx2vBRaCrC3SH3mXqROfglPTB38CG1Otjyjl1AUwzVScmrDh\\\\\\\/TDPQ25KNEG\\\\\\\/eazvAIUmX289fvGIosKz6RRW1a+wXBayb7QvjpwRHYL+eziEWzfOVh4kf6heVTwo4v0YKLTuqjyjdzDvDLLEfTLGPbet9KuR0n\\\\\\\/xsHZtL3Px+\\\\\\\/xUdN1TLTdv\\\\\\\/fGH1RjQV+2YBHlSKUx39UusdQo5zQhz6gV\\\\\\\/3LFgHxZQ+9tdlkgOAEhatSIJIDoolR17U2rQBQIUhYuJIIixmlwTAcCV7Qy2qc1ZdiTUKTDacTE3woB8mGays\\\\\\\/Kh3q36vT6gIyseB80r3p11XUyRrjU3hnHa3uY9a5tW6+AxljJiZlhYedUWKDEFURlJL9Y8o38UJ2hmFPCoCBbi0ZkgyaMklSc5x7PMuS4X1lHmYP5NrZ3MP3guTHKiZbI6Ce4QVPEH59tqnsOdgQ1kw3cZFQqaPeQ1K3Y\\\\\\\/Xpi1\\\\\\\/AETiRCMQW+EAHPR8TjE8EwTf2ifq\\\\\\\/Mr\\\\\\\/T5RVoNAp\\\\\\\/\\\\\\\/YmjPmVjQO3ih+NEm+azsuPuLL3uqFH3jgbf4AqsiUUm3WFiBOX+g1aH\\\\\\\/n5HYPGUY0eea4WOcO+ipH7Vp9mnBk3B8cbFfoyuBR6qUYRrVA7DTTF+G1sUTVX7sj4KQbkHDp9iSp1nHG2c64OZrZbqtHkkG91dPMXyBxrYjLN6d90Hs2VjsY39reRaqDSxsmFXnGP2YBctx4kIPNk8ufeTu7KRgFzJlEXrf1jlJhaDjqQuFcE9\\\\\\\/Gm6HAjnnMHw2SmhFmG+08dGVK00mzcSmkYr3NkzYetE24qviGId0OCeCvRgoe6oYSnLx7lELsXt0NkWXdZOgMenKxi\\\\\\\/wqwbqWD3I5lzyapukLWV\\\\\\\/QUHtLBfvyil8bjh+5QhUFs1AzQFy8jzjp7zZU4SglG9hXU4Xs644R26huHYnOt2mdeqr46EjC4AqJOL8B5KtCoWncTVr9XWzPjlxiBwuMsfhsTP4Sma0nubrhJIwEkJD3DoLhIAxMep3csF4qc3hInGV+FbA7Tei87HHJg43Vt3lx9anffEFWVzWRv2CcuMKyE7J6Gxkrkvcr4mmHXIc6oGcXkNkXxrVE1xv\\\\\\\/Wj\\\\\\\/TX7o6CZVboyS71JNmG6aAE4g8kTS7WXFSu9I825oqDkLqPo79\\\\\\\/hKy6CTU7BXKAaIXi27bdrq+4Pz3jsLyWUl\\\\\\\/kpJBdFj+uFtAKaayIaEdwkPDbpo5x6q7mWHSpVZf5aE6FsKjLymMvLoSFfYtuCy4AVlrqrp1rdFTIK0Aen0bL8Io3pGc1EVSfViHdjmr88PQpOoSdzpeEpA1nkK8R\\\\\\\/rYscB5mGv0jM0JxVBNSXsFqP7DNcC3yx4vUUWBJ+ecRFGzELqs8RnxKDD8JyzjW++pDsSpi9vA8UrTIlJsGcWk6HYSWY6GC+HY61mASBl2L9yqkio8CTx\\\\\\\/+PQ3mhTSLpH\\\\\\\/uHD0gSDqf0tlxRyfkSZLBsjY1tN5XEwnbQPc29AFhi\\\\\\\/kP0piAonOMe5ljfV3IEKXvhKMI32\\\\\\\/3csmFgggxu7LmL6TLLnO5ntFySTv4kYJgghYzZmEeKxRr\\\\\\\/Oif\\\\\\\/aedIkDFHOyMXrZkmnlIW13oozBsDj4pSAB3EiXpSX6zT+ILiU0Q4AXlzpQlTAZuTtWXOfVThwK2agx4ECa9MxyVuS+nmH2GYrE1Q+xIfywHLMiP1OK4747iTu8TMFnfhSXligqNywiTsoXxMRR8gUlTW4j4AtK5PxmhZ5+Sd3VX2xN\\\\\\\/gQewuIzxXDF6sj3159bgQRxXsxO3fpwJXH+xn433glDOSMR2fEpPkZqOyCP0gR9+5s5LwgMfuepq6HUmefTBPN8LFv4JVqf52v6r+jFXVodZ0q0JK5ZYsJGl5irbXAMCiLcyRzxXLaRMIBL\\\\\\\/lsiZ5EDhA86BIcxEHA7IQs1U08CMgFgTj\\\\\\\/O\\\\\\\/uQi0\\\\\\\/kt3+usoc9nHDdfbWnhrakt3xU1ix0Y\\\\\\\/NPkCIIb5yq\\\\\\\/BP51r3NLo3+apoXRJuvNqAOAhF8vEv4csGKVC7lEvpNEWXkmqTG0LsYQsZXP5pR527oyMmVSvhmOzlB+JzER1FOqvFllxlLbBSzZLLWRcI+QiOHHR\\\\\\\/JdS2DiGg6zDdi2SfragoORcHZd0PJItP++v0+GCgaa7AtmOXHcwc\\\\\\\/i5rbri6oeQd\\\\\\\/jb9RBJA+yo4io\\\\\\\/R29rIr579coHxLeGE7Uynzn4bJ46dgGpq7VzNuJQcVz12s\\\\\\\/eHtrMBtNqfHnOfvCSNdKndOEjXGwenTokpCl5Ak5tmvOcLRsXRuD2WjfJLXO+IB3RYMNH1dr2kNVIpJKoz5aXLRkrpMRDRu4vL8HhUZfRP3HIeKbaTcy3Ch8xUWyasTXOwlmCZGDJaWSITgLB3rqfqMJXWyZOwNkJFfT4Da4L0BDXoI6jw4lKJoqlsAgDD0fOZ0RSBnXombb8KqJvrTkldk6yzGcVYHdHi00SMB6lCIgCz8a2KLGE3aOxkI+\\\\\\\/yz4hbhufiQxJP5pvdHApDavLyoAfIC819CRLJyVLsPuivCVX\\\\\\\/4kab9jmrR5R8NOuKXpci8y8idzmDz5cQW4QABRXfKgfqlLVvv3nlJEYNICTT3Lw3vbXCZfviuuQcPvhsCV9nZJG4f\\\\\\\/vNKdC3QhP6VPxm\\\\\\\/vhSaHtvEDSoqhEBoxZpvy\\\\\\\/SbhEm5NEtmfJNhI+SgqchohuG\\\\\\\/Tn9ZOVObkI3sWmxQtLX+OqmB+YNiY9vbs5dNOsNIh4SBJ9vV9ANBEVJwUl8KvzN98gZfdHXYXoTVCrUNGvQv2TsTF+0HvVGGTud462mtQrHraviZyVHkC9uMU0A\\\\\\\/p4qFhGrDeLSyA3CGEuXg+mBJ4mNvAscMq101LTBwKfta4Nb23kTTTvfOcRutgNfqzQ+QpjZTnmRAik0CwMK+rR4uP4JrDr4Wkbi7AVyTkXV3Qx8\\\\\\\/e53xA178vy1GkFTt35qjz5O+\\\\\\\/xc1ycj\\\\\\\/LmTnLFpVNneePH\\\\\\\/U0t8kojK\\\\\\\/SeifzyLhwTlYpmyq4\\\\\\\/E9fRndf5EHohMk0F9TyUG2WOBzR3RFqdPnC1GP2DmWT2dPI\\\\\\\/F2m4lODYAQ0wT1KBIBg0EynedK7oONBRpnL4x1luJzcvoxLMuB+Kfa4ebYhvugz8GFO7OI6Q5yOow1TqhXLYIIXsnSnEUzLGQtnlXeU8C7zShmwBDJBuoBZgEwKJTn34HY0iwaqHn5Vt9eeH\\\\\\\/W2A4kzE9nnAu915+524Szg2ze3\\\\\\\/Azxctt17IRVWO3RKIGlqA8n+rs3OrCqU\\\\\\\/cT7ZHmrELuh5xfOGisW5U\\\\\\\/qDXBuVz7Kz6sIZM1enu8e1UdVOY6ukuuSr9VqoFNiY\\\\\\\/Tygr3lfmJijaVU3c8J4bc4gQku85S4fWx0RkCN8lMLtk0Gtsr8YllbF9+pdviW\\\\\\\/yAVDtuk2qgoenNvolinGDwv24LSTCPQWA8Fv9\\\\\\\/QXuSLOiYDGRP7QOa5Y2vveF9zUCWmOm+4mEXCMBJ\\\\\\\/iWlTewbAIwPEOJnbnNYWoA8MgDfP4nA54Q01QON60nwASRRzhKewbY0EhyHy8TSKE73PTSJjXep0dFCyyYOTFbhGDUNxYNOrOLIYt+HgJptM6RQRbLdm3RzTQYlkKb5TmPC1YtRKeFaedemqoTo8JfzP2OcGM9KsrjY2Oyx+dMNqAldsTDq0nZmN52qNJeurJPe+7OmPusWUDvbRIUBrpnCZpuMzBV4Vy3XJLFgf70XK4P332eNIp9sLbr3YCgO685HHPsQNyGkN7fIPaIwun92FrXGoLp1YJbClEfTkOYN0oLWvX\\\\\\\/cL40wa0TygU5W9X7R95yRrlYHTRLplUahSSJfin8iSzf8poBjN7dkZJWibEMRf9Gyc76reyu7u\\\\\\\/pwxzkPcddv8okjsp1ufKK6c5fiTSR\\\\\\\/b3arXXos2mgrwiDUN9Map2MeMvf9kFtUezYAI08zRg+uNzrLxgUaKzjHtFFhvgEAVFttjoHBhvQRLjJ1ZEZHLW8GkyQsJvL7DunaCQpXxxQ+pSw+VXQJJM7LeiNLRjDnyNV7wuA0EWIrAlgCWTzbb3tkmso8AGIN41cq7192yGkbO4rNWgyJM5wmHE5HDONxTAQG7\\\\\\\/ti6BtujMOHWbPqD9SNTT7LQLBqjkU6ic7hO\\\\\\\/Hjdd\\\\\\\/1mgag78zmTysr6fCbqzex5Tr6db1etcl+wKzSGYDnarJurs37U5obRm4LUrOC7rPWDDWb9td1Zbm7EDFlakqp9QzB6W3Zed39TU3\\\\\\\/prRJkj7jrMm4fEexfoz4EGqtpDGKsRBlgVFa\\\\\\\/bgBOBcf0iGNXQfU47eAuuXVei3tIjdJSOseYpUB2j4vkXj7aNewfuDdve\\\\\\\/UZW99ECZaEY0pjpfMqYvmExxiaEEFvO3H6nwqzu9UDdlNyjAnsmlCMhCcUuvmLbDHgcuqfoW1UMEjXwYM5f+fSM17xuaemY6v1TifYKSBR7HJlGT9ptOsJLO+n7hhAH1BSvx4nbNZSVpqxUKqstIIT6CiH3k2gXVAJfIIJyumCGaw1oLQQfPVONO+hBB2lbEmkpsGrTeyyxUEX+LekIIPlr7Tnerc1UiUnWC3JAaDvzqkau02mZUd4H85sUVKEd9Mzmc6lQ8917j3xHChFmzyu9YtNeluvfEhtuMvDBjTZwx3WLjJp+5xo3Z+4GGyn4x2XwmU8xsi2XKK8+q7AEA1YKGFxNtLKWVCxLA07\\\\\\\/u9cRHa8ExNeZglZvahl6UJQj5wm\\\\\\\/NmtBsCwuxPw7gZrLH0OmaFnVDM6ic2lQIKDGVvyFqn\\\\\\\/c1xC7eay5upF1ny2j3D+44I9XTjnGJea8x9K1gy9M+g2sgShpacJiJ0vvuBlOaIxmanXTLMjyDbe0osuGL+nDHSSTJ1GQPw03x4u8JheQGuj7LZ0qdg7EJLb60pwHc1le9pQYfRSjmXTuDnbA\\\\\\\/YM8Ib9eXyqfmNganObsJeD73da5+TF\\\\\\\/4STg7neYx\\\\\\\/3rixRDAr82ZI4yTvE0XtT8wiwG12BBZSz7j5z7J6IshrAqN92bQKz2wphTPTYLTG7GoYD+9GKX3HD681Z9fVENv4Ydhoc7vMTqN3iZMyRwJ7N9AzMsnOuEp3ysuy1VUFzmGpJxqGQiN63imNktbhef0fXMmbFnZcZmf5fIbgZVmY8P5fOJ\\\\\\\/BiJt8sfyWVD6DqoPbh6zpgL\\\\\\\/JXyMrwqFfk\\\\\\\/M6yGXKNENRcUqqblYXyvp403z0g38NSz+Zn42Rfk9KVc3GAidHffybI1WPxyvNnghkEG8Ko4bdubDXAj0Rto0LC5PpQGcvQVQ13YQ\\\\\\\/T2+fO2k0gJcH\\\\\\\/9RHAyO5hVRL4O\\\\\\\/0Ro7NP+PeeikdfCmiqswRGqssY+3Iu+jaRrc2xbotwDUJnANneYZY7mANljJfuOdbEpKl6UXLipLI8uZEXMXDMdmv+y\\\\\\\/fFURM72i\\\\\\\/8D5pLYTzRz1jPxh3kcogGWmbK829F\\\\\\\/cbJG6VgE2Ubw3nBYJTu+iBXxhju5ZE2bcO2H0CF2ygSih0XyDjRIxqPu9BfP0eieqtW0GdIxdwyJOrVGLYpEwNRaRwIP4R17fVZAwziVT\\\\\\\/zJNdRXEotIsiCZpQLa6xiaFaOAZhwNTcb5Xta\\\\\\\/GriIcErx3HXnpX48sHan9J2m6ig1wxHUqODrNCLesxyCbr2RLvNGHE\\\\\\\/Cq2cw55fDB0pAKq2nck85JXI3I3qLcWIjh4dRyZ28K2r2aDlrYlRaTva8YXMXNFJUIeoTay6vhvI1MT3fSuT1DzLl4ele3dGwo9YQuQHuO1mJx6Xig50ecU\\\\\\\/FrOGkEhZNDEXH\\\\\\\/8nlvdR4VVnyiiIe8Ge6l+Sr1zdEdyggJwnc5WvGp1hwaq8Z7Lcoh5xK\\\\\\\/jKK\\\\\\\/RpOgx0M\\\\\\\/0W\\\\\\\/NcU9XSA9LTpp\\\\\\\/xKKq0a5Hrl9qdN6GNj8qoat1YubH3mpczPugrlJDrjwfqLGMQ33HO85\\\\\\\/Nk0IcDn49dUxgtqcPat6xC792dVLIRXSgtn8HxWdtHBkdaja0On3lywQjKYJ++dSh\\\\\\\/3t4qtoSbNxGLAyFUTlo0r\\\\\\\/G4TufTmGkeydaeoPbfk\\\\\\\/E9yNggGBO9uEQ1SEvSNaL8g+vWTf4q\\\\\\\/aYGo11o9N3sBGHJxuKgWgadG9AA3r65jEEYrdhYPuli0DcAY8DaacONFjh2zWlDXTBsfXTRh43Xccl+PhmE\\\\\\\/oeqYOKleFAv4vM41ne7HEA8GnGMHIdJ8t\\\\\\\/\\\\\\\/RVMyvwGbOMafT4WJ0ufPVeqZIPc5Lmj7Oc8oGscbVZN3A18UToLg4iZBsWUGtGLIHdvh+XkXnfl2UuTQNVFUunvgbxkmGt7OHqSHhjkli8c\\\\\\\/qqUlnjqgPAX4\\\\\\\/eI2Su9yDWxV1HZ2XhgSzuwhKIxKl7iRLpETg3gB4r\\\\\\\/X3\\\\\\\/lzwKQfC47t0tL0aBVXZJWnS4jFV0wYtZMI2PJHXudZ6RAnNY0dxcUSEZ5IEkFrQWJw9wjsoNWoNRSuFIgNxQ+NTiwsDN+T9J2bl8z5gKV64kGhzFlVWIQXc2\\\\\\\/iUaZLMfw9p+v2lXlF1QVpo\\\\\\\/7TCPEp1Y4SRRpFz8VVngmvGuYvYoc5dhKkKDSKj54LNWWqbPin+PPW8WNUgJYYf3Oapb4ZGVhqleCT4tOMOxUE6jlr382aaRHpha3dHI+fTKI\\\\\\\/tYvR8WtWDFTRFf6M6quO51Czy4NSigoK7TfbQZfUSA6rAJH9U+Mpx2AXYW17FPYX8dh7oFL6xC6zeTOz07nX\\\\\\\/ZtRL0U4EIWMYBy4gSI2TUKIhbYCok5R3ocrqaUy5A9q6bNfRovVahe+X6Dc6ADkTMEtgEq0dZ2JBWCuwH5QohVaOj6UmAilB7ugny9ql5qayuuxB4ZPSo9eVxIsCfFZKjOOynpF8mfIicAszANFTxfP9eqnjVhgRqFkJqNADOLhP4+Hasj\\\\\\\/FSKAs\\\\\\\/H4NOZUUnLG7whlaSjATF4dDc2MFC5aUm2q8E1UNjMD4CquzZY4KNuDtsSXjN6DxkY665gCS1mbUwWjQ9sfFuX7wMYFF1xhoZxckEh0xADGjD4mdNV4+r4QK52gFRASyAwlPNgcpivBzsQnEpYGqbxqDxGz0sx3aUWj2kSOoZbh5cMvMLiAzFpImwIjzw6TmlyL9N5eAUqGC5hGv8p7J\\\\\\\/9FR4iN1vJvPa+JC\\\\\\\/JiHPjq1LPfeE8t1c2sdFLPE\\\\\\\/jy3PE0\\\\\\\/De3RJ2NWWIYtUD3ci7vrEQyrM9HhHCYwNfsfANI84lCb54zyHQ7kpRhYm9YGrzGF1SMgWAn1skpZVaJFK3UcyqKPS\\\\\\\/QmNiYJpLrlb9JoNMxW3OnheQ\\\\\\\/2CWAlme2CFZvIG++LaShkKEwYEnV0ClHkr+F2CSkAuxUjNcGSyxIbYkPv5THsVem8JCZnlDIMS7VyXkPyRCLJ0pPGaYxJx15J5sggHK2mW2mLLaPLe8fk\\\\\\\/uwq5ruNrDyfUMzisGC21u7Nd+zdy4Dgl5DNsfJBf0juf8o4nrH+yTTvQM1Kf8tWFFWtrlGOJkuq4j0G5N2BT85epVIbUHmCnGZQSWQqwQokYbk8bf5sLKYaChWOEPhcmT4fkizweYvsquCfREH0f2UXIw3b4c8fmlFoSKgg80pgC\\\\\\\/LDLUxsKyjZubOqaAGMA2+UD2wzb2TCcn8xw6NvLHxEznpR76wQWQOSdncNxOp9ds9BsFTfla37ehFjw+YbgO+uAAltIgVICJprYaRbIVh08srXOeNn3sVNU7pob2sDwwP8KAt64Xpeh3m+gmY83bbv5bIfHV8XDGfD6fV6R+yUdAeB2VNSHAueWPg+uHkeQOf7q\\\\\\\/4nnTogibY+zgvHl1BhS88Y+uL8BnQgIQ9Giz818+VZWfszRu4LhV6ZgozLly+1h1SEgq8XjkWPu7R481lu9rOh2UB896QMyrRoye\\\\\\\/QZoXwTsxEi8Ojijlh0PmWjD\\\\\\\/k1TZoCqLdi+N3d7YwWFWF3URvE\\\\\\\/bBzVAAU9S0eBj0uVuu40l2A4noCDxgbdlB7936ZEwHNZfPeNXXMJwqYx7D7XJZOWetFaVr6aA+O360jcyxQwht2OT5LXDsxHfQWXCNtPh3AyF+CJO6IJ9SruTqlNwFIxscr\\\\\\\/5FZcoUqhN3smyvKfl9nkly1sZNJysO7b6me+h0ycVzymFdQbYyXk8A3iLx70cx8p+6ScZGs6aK8Asl8Xm9RxOgGx1kAJYhRGxAPRUXvH+MFp1bYiQgxCjORhrEeTNIx05ZbclJG3P\\\\\\\/Vl9UmujOhmqh8jcQohmtaMJe\\\\\\\/ugiss9P06OQzJMQLm317zFJy7pq20Kw8bzn0UzYJVX4QzExovQOELtUiWIHahzEojP7iejOHFyopKKRBD3IvNi+Nw8pYRjI8AhC+VDDqO9h36G3JEEIhlIoSYwgM\\\\\\\/GTiOtQOPsvliI1gI\\\\\\\/isdQZJY0PqMrGPa5qXRl8F8kvRIg6PWdpSnOjmoWqJKGSb0AQrYbJqIDsMzY8l\\\\\\\/dYkgIJM0+9xdAATtqV221zB+nO\\\\\\\/ThE2HnGJwaMaDtwMd92\\\\\\\/kw0HuVNFRRcILpgzd2tZiL6WLq9Ap6+ey2ux1uzfhNh83O5SeGjNVmWm1UJsEKM+DtssanomxJcBXTpXezkY4znRdVHCZnWccxuloVorsm9VTqJM4VXfXDIw50wXyhSGG8zi\\\\\\\/Wbjh6Fa1evuae1NeaCAwlVBZlo1EaSTH4vJlJheKPNi\\\\\\\/cBJWQk1zJ5xVTAQGm8d3iHTUR\\\\\\\/hYqKLk4Uo+0kd2T3Tjre2P4f3uoQT0xRLuemBho7yrFuvi2PqCn9Fsh4nuU2on0ktn6iC47062QW05AFOnVFWz8lOCh7kyxkUPLF3SKOJvpOmKHC\\\\\\\/OjAC5RYQLh0Os87s2qZTQocPUmwf9XtOR\\\\\\\/5Mb55Dn8eno7GTote+Cp\\\\\\\/BAjbtym+VKcDq5AL+nyuPxh2pHYClxrp22o2\\\\\\\/\\\\\\\/yHaia\\\\\\\/7T2bmjd1QUf5R3mRt5A\\\\\\\/gBpXgPP4xo3pkRfL6c1\\\\\\\/zycVCNFtRubpmtwGe71jEeKXk66SWfgCzW8Hjk\\\\\\\/9t2gW0GC4xbLnk9spSnFng5dMqlAK5HoCSDL\\\\\\\/82V5kKlgqm3WkgHDr4GchVV4YE7a9ZArtcN2pC90\\\\\\\/8DN8E8chqCKlpR+fsbObZsoXv2rgAZMdhRSb5d\\\\\\\/DDvc4w+0w5+f1SsT4pGqi+gZYos7JkCA9E0oaPHqOO19KLwZbNDE4foqxEPdSa4+YDO1CB\\\\\\\/sMAjEEfadcUDJYri5myghpj5bOQpVA5ThqZxw9ENs6WwyX+3gnmlMXl8uzNbUIC0EK6YZu9PrbJm4M+Cl5s\\\\\\\/m6nySWFSZv2pLse2f0auOqQbzJ\\\\\\\/XXEf9XM2be7ROd+4as2E4PBh4f2uTEHTWIs41lz7MFAynKpHUqvABWPj7lDnRk6Q8LkcNYhRI1r9FmVpGxfL3r3A8dNE7SuzbM\\\\\\\/BRUDu\\\\\\\/9pz1VcG6bV5lKq\\\\\\\/6pxYzvpytgs1pkHLYZ5oWYKVI1SUyIEWItKZjFCGLQ7wN8UPdSPaujEUnWjJFd8RCiIzSnIcZCiHblUyo\\\\\\\/u0oBN3XbV0uYp6qXN0tLpeAOCOgBAX78VedbniLSkOSWTJWPF+FdZ6xbslmu+Nc40ksFW\\\\\\\/yAyAAG99o+gDKCutxkCWXXKEqhdSuf1V9otcELqXOnJVleXFdLbtbDoeq5jEHl+7aYeyJPrMuxOxHpVFV5Wrn7H2xf1KFHo52ITJ9m3lyPKzltqH66j04LUdB0MNwjXxUwmNhPwmlazVSvTk8RvC+w3BomxPaxqxTP2dtTRBgI5n2KVjCubEpRPZd\\\\\\\/AxgQ0bas+q5IbBd8lER7ID7T0xMcJGL45pZB4VzENCZYe9OSdtr1h5AJ4x9DeqRTaWD6Up8CKofSvJrKsDmTYHa0oBUhOJJ3Dx867KmLHF2Nhg9a7MFhvor0VL\\\\\\\/iUBp3A7B9p4MpWnV0IwWb1bQzsKrWfHPSJKBUXgyL36gfQGJgUfBSGCqEkpKP1is\\\\\\\/4nWReH9PBRgKYKd0lVt2KqCUQWC2EtzYjRuBrvcQY1KNnAt\\\\\\\/yUSP3ZzoXF340mRf8Rk0yAJllqdLE78TR6zTTaR97X9Qz4Uqu6XtsTsRQCfhugmdNm0V3PLFlCpCWe8ahKzGNwPB5WujMYY9prD+4bpLKYxRybu6MKefc51qDhrUXdYVBAwY+xufcktNSm2zI+LbD2ddBuuoPGv+QGm1ps0Ig\\\\\\\/XlKbmCULTSno+l4HrXzI+w3pxTLIyLk8eA7fLpmVK0TJm7DY1k7+tRy42m4KvslzQ8GGumjzmaTX71XNjghkXEyjHCtwc\\\\\\\/AXlCTv8tjpFD9CkCf1\\\\\\\/0j8dkNnn8f6iR8DQPVxWPjRKm7lrjFluIIGQeLfwMDK2rJAJNEbNPtWdIRG69FWx42EI\\\\\\\/Z0CwPc\\\\\\\/HaTPPKU4fZGf4EK4V1M4YH\\\\\\\/fsnrn2+Ed3fa\\\\\\\/j\\\\\\\/UA7Zi4b0pDGxVfD9qI1hR0QZikFoI5gxeNp340i64fYJWcvxg7ZmYgol2Pk4K3TApnU8H1\\\\\\\/VUbGbnjigpJuMQh87I6KO\\\\\\\/lvDDnrgOWZ66T\\\\\\\/1Mf9ZTxHWZ96TRTjLKdNobUsFOw6WfKWrLzLGL9LQSiWcOeY4GH92kzOmJtGGr5UMToFzRtAPlN3+jqRqV+1hkqNeT+PRc1aZM3MLQgTEJo86ZcxE1R4+JVEyfj0G\\\\\\\/K3cCVS6LfmELg6H2LRqgoFlIiExO\\\\\\\/rbIFkqIxxPi1Kn1a90R\\\\\\\/IDLW4VCotzd2DJm3mb9BbAoPi3o6JGxeXGkR+0Sex27WmRvWzIr9xoXmKKvTNnXGfn21uxKnEqOUxZlLbfzSwGfsTPmOAuKOuZ8Mlt\\\\\\\/mChL82ThMcq6KS9oL4+VRlRXoKLYF7dDBTqLm85Ne7+lQiLgaLEhR\\\\\\\/BV90uJiAnhZKSlZtEaN7Thnh9g1FtEjlLDd1HJk9m0hNOY+ql9ho5LV3Ecid72pU1T6ZYbXmheGzLYlkolD4WqmR9yG1CBWC3TPyGHWfv60qlRTc7NmTEImUT5rTgHaTypOxfxrngDobVIHgI39oeQ3xocBifZmc\\\\\\\/cglVv+ZRvm0T2V\\\\\\\/sCEUsXtXzJLC46mht+EPeY7hkqR1VxHHGYF9xVoC4\\\\\\\/a4yUHQMCafuOZJ0SHKtsHyftlUqGh0THHlR2iyKS0IEYlpuU7eNOC2wD1cA8bql8qBrewWbzhGv2eRmdySxYzDPqQl6oK8yj5Rru5CGCsUUQwUwEF7kvNTnLdOkZ+kyFiNGYi6fjz8m5mNkWscd8XqHWlnTyp2DWxClIIKbrDVAliRawIaAyMiAXfk0+Fhqe2Eg8cVn\\\\\\\/r7euJ5SvVxqejvUrUN4DrP9lS7NbBgQ8wUarVxp\\\\\\\/fsp0OhlYuK4j1OEjGF+WJZ2Dpo7J7ORX3pQAExcH0Zh4R3bQ3No+3DZvlwUSHf1\\\\\\\/Ztr1PqqFTqQO8Rh1pqFH7rwmbNxuifB7Oo8\\\\\\\/cOVWN4VFcBX\\\\\\\/O57wRAM1B3G\\\\\\\/+an69enzMjVUEkoiipFYORADsWNWyyq8iSvHlduFLZebpeSz4EGDJyhEod+vHV14gXVHwiNIYyCXJkKbc6hqXMzJ60xzNAwKfpjkF5hOguwdJ80X6Jvd3bUOnIaUBKbVqFFJeQe3ZTBkaFXk0bJJLs4p3vcPsWhDPuuCPc+WKWUu8vlNV0bEkBrDG+PENfTvg6JpNAlhl8qXieq\\\\\\\/adNyOIi9FSZoczHztXKsnR9CtH\\\\\\\/99mZdFxwKgzrr\\\\\\\/Rdxq7gMkragv8\\\\\\\/lZPSM3H1laGPhNnq6GalSlxzf8+LBy\\\\\\\/htnqF0d87G0BTi8Imln2zTYJNZqw0ufCYfRXAE\\\\\\\/iZ3Qqc7qVlhgJfG9jPI7mPtOBLlHplIEjxBF4YHLEwtQPUG0+ofYCYU3JZOKgyEkpWLiT2U2eyjjvyfq2XaxnDgioFJ1BvksLkLFpu+AyZHFN6k3DW1d1NcAgG2UOh0gZ+A2at\\\\\\\/vKDmSb\\\\\\\/BjMRV9UFmt6lF+LBVMarEom4BdHfheQu8HoQrqJK9E\\\\\\\/hpFaH1zBzfguDXNqzej7l5269zV738MlolQHsQIybd31aw9CFqS8t8CuhFQUxElUdYtbWq0PaoDsISQoJNIwPnaEhovnuQOkmZr79WLBMvcTuzUgXYQ58S+ADNoETaqBitHWqVoIl+WTSezAzuN3KSO21xPy9goPpGxUZZlfUg4medoM1oMH\\\\\\\/3RRiR+oKT0dkNM1kH5ONySbB6yG+Vk+J7z\\\\\\\/JDZiwsLhcTxoWavARCl4r8PsrBRZGvTx9k2eu9\\\\\\\/jNqRwrEB8ggmLM0CJ0TWsDAuQ7lHJo5EgBRxwR4tAYe5qvPNABFBSUqT\\\\\\\/cFgpM9K8yXWmcJuOxPRv4IheHTLyjupMIt1w3iNWPPXTy7ciI6Jh7Nqm8zq8EKaxiU3O5EjTW5XTHrymVb4oHY1CBf4IBx22hWujN+YZzJldNIRZg65++UcDbxAo1diS2F3omSCMSIeR+NSbJ+9M8uikZCzpnYVHgNHNv9X8jEAAetkO39WxniYm2Y4KisZDFg6ZsAdGKWXGmfT3K5m2OdN9gLmKPpYtpIiq6cujLgLRmrqWgZxV9+kQWfo92daFnyp0+ylvaZXN7+quom4\\\\\\\/IJ\\\\\\\/ATG5tvCCDNUsCcDiGwREu82\\\\\\\/8yZMcHsa2Q+xKTP5PWmmQC5Uy0XO9qB1KapOlAlO84oX26JG0co\\\\\\\/lEbJQsDH6Derr1QS7kom2\\\\\\\/qUaNxcrafQq\\\\\\\/3+tW7BufIrXGwPe4gvZiyeknKYSFLj7Fk1tui35SK00fxI5CRCLPNN4AiNltvw9beTnpZb8bsGNg5+dSTFgfnQs1RL6fdDERGwWcCF8ob3pQBnV9iBg0tF8SlA1LiPkve7mt\\\\\\\/aqnlM\\\\\\\/3KxGJ\\\\\\\/BPHWC6d2mWdgAZpnSslxzHDfYXQV4Gtlg6M7+eKXOzfODlWEHk8rmcd1ljQUH8XAJDMw\\\\\\\/OMiWglU7YWa9Dn8yIpiQTSacSCGoRZAErJIx1Hcl\\\\\\\/CqwnGISRySVaPuRabV5PiftFW7CfymbIquTV+PQjaSDNQ\\\\\\\/MegnSnSlCfNMoN6VhHWnOPbvUUxbhMTEV6sQlBn+WGJB+Vqri95KlgarPLL3Wy2LSsiFvZ9ffQCO8IFkf5OhzvIU9oz5xZd2yzm5xs1cia8CVF\\\\\\\/A6xl6gFk0IFhKNDTxr7TiZcIrUTdcDq02nQhIg44hpqNFtYL7lb7G\\\\\\\/WYOU3vY77n3ahmp6rq15Q6\\\\\\\/amjgGV9Pj6cNTygogMEmZmtAA22ZTnFwof1F\\\\\\\/P2ydBBBNiP92iXvV\\\\\\\/AD6CEdG+cksNXKsilhYOMsHhV4KnSZzgr5doUUvMuQbukwb+lR5EFoOk4cg5znNmoRJDh5A3zcOvDEOVqSzVioD7dA\\\\\\\/\\\\\\\/VQBkf8vkOpckUGFkN0WoDFuL2jXD4PGaot0To8H\\\\\\\/86dIZ\\\\\\\/s7mfiFc2j48JQ0QKll2Mmp3UaB5aktV6Q9Moya0zJhnIRAkmWymrK3u2IMWKctiZftK\\\\\\\/uzB+Llq62RMdlecpiPZR1oEb3z3zfcrNbNKP5YFg9Qmi++9rKfObTePAWD\\\\\\\/YvCuv5fmhHJpVG0AdI+AEASyKEYwczDbWgwr6L5wDVuPWNAGPe7m1TaTd\\\\\\\/auwGOkSSP6lVCnnj+nBUIwwO61+W2a+lbHJzcCqvroAbfFTrjUskD1sXgmaHghx1BnKIxlcBERXkKiVr6Xg4rzRxLLmkP1CvWYlG79WPr4t4nzdsOFXKc4g6KmhDxH\\\\\\\/93IiKRB5ngv+YSN8lzKKtAdvVPUxigZsZt1avm7ayYSoUyH+hN3nmnz2shUVbkSn+8xZkgyHquBdGgJe+vjTzuPjxI\\\\\\\/U7HlFiJQ1LR8mxVe3DZQFZcRwg26+bgstP81ZDWjXVrL4BWya\\\\\\\/vmlNln+GwGNem8cJVFRiIXlDxv2tqpMWekV20XbJczyNX85QMzXDrjek4vVz17BwhZb\\\\\\\/P48n8rtLLt98ATqLCdcxc1gpt52txM7QR\\\\\\\/Situy5yTVUlEtLVZwcDj+hIZwnsYGuuNUJDoF\\\\\\\/umdf7FqOZ5aQalZ29PCrvWpxSAsIo2rmK4PJJ64ef5gwQ4UU2P5T\\\\\\\/UwnehURLUsyCQsPiUjyEIZYBhMOVq+NG1bIW5deG73iJt1NNzMNhvdf\\\\\\\/04tiQNfobxrTrFRd3bzjl0uI7Pzdy9arpHZ9vSXXZ1zOJp8oWwPRDkvxcAjks8PW5ORGcKvDLMrSlAD1gWULqY+rA\\\\\\\/EB5+mtK4PeYF3f9nCGYcbLSQ7oAsbYVt2NlTEBmvk\\\\\\\/gGqXA+DDxic3pYy3vjdMRggjr\\\\\\\/TbodJYUDUhiZcn1oXR9NSf\\\\\\\/OZdO3SBeSHXrUON7zFxEz8MBmFbfOlUBMdAs\\\\\\\/OjvWs8Hhicrqqgb+0Fqd9PSoIlX6MOx54hskULyP9ruf63WX8h5xEX8t\\\\\\\/5TzEPrZTRNywoOdG2G9bECIs2FM5plLDTbyi44gMeAhiEFVHstVgzCRnRu3gCrptu+wTADgj4ALRtLNU0Tm3a3YD0ztSaeCngxuSitmnkDplRRo\\\\\\\/3bOB2c4ILJF3CoDg+oeM4AJ5Ksfwb3pUNcylpr8SGb81qR8sEFG19XJUL1pz8Xu9aF4moUgS7gjEA5sFWVxRq67eNGrc2wj58wxEL3upzfD5GWDCaR\\\\\\\/swlAQBy+yMq6GjbI8+F2za9jsr0ocxJjg\\\\\\\/PTnb3JxvXg+GtCFQCp8HHwFhBUuRDs2EcsMrE2FfNEME\\\\\\\/quWHxNdAJtE\\\\\\\/Yijh7QMf03r9DG8rHIcbK+neep7mDR90lkkAdTVmvT1JBDfKXvriZOdv2vTI+FUnaKTX6ui0lhmBwyVeQUOnN4LnQ5t3eb8tcy7Wd+K9zTtLl9GQBBRYhyFa4Xy\\\\\\\/mCYgvP4b5cTBHsxzYmddFSiXRoK4NV9JpjyFOOPhm6XTkO775i8oPyqpEF9SGsK\\\\\\\/jrQ\\\\\\\/r0yA43wTxy4sbEPbo7tomXta6x0QWnFkHn11xrGPlwbWapCvjMLwwoCVIQZwbM0L7CE\\\\\\\/MgLDTaJSjMALc6\\\\\\\/0g3cKpT8pkvh95gT0wPVcwNh+1vtnb7nOUauXQpft+ZjLKidsSOPYkLtIwXBosLyIa7D5oSpjNzZhzgD6mbwOMnP2ex46nP1dPuoEkfwgk04vM9r+3XslVxCVZuqisjaenrFwtebWD3VUgcUY9ckynNcU0NDXxHbMl+lwAWekupvlUpKSft7BD+yiv3FQXtt2Bi8riNc7NiVghYxF23BbmiS9Tp+8WAYPUR\\\\\\\/M5pH8k7llbBFaEOHO3d6TstJKYB5BSO2NOawAUk5Lc8kWjcxerc3jQBpmz4eUbJUY3Ov6qeLC1OvHxrVhVHkthdj90mq3e7Qzhi4z0eITSztBkgFxxMYNFdcnfft4QOFmXcpVmR8KYjugdwZIzwaIov5M0NVUiTBIfP9eVpWY6x0BapjOYMMu6qidvQtez+s2BcSY8x\\\\\\\/9dN\\\\\\\/mjqUUMlO1zFHA5+OP34a8MCQwKd\\\\\\\/zxMJja7G2gHZUtY+VMoisrKRtoKtcrXRdfUkK7OLYC0dsS7okRLLfB0bMiAUzO9XyGvgNG0L3gXvA5MNByLlXaIn1J21rZ63rz1dV1Ol0PqiHQcujkok9H55AOvNSjyZ72kXrKkgP8i+blB3pgu3ohOXo7cwK\\\\\\\/z95R9LD5svnP9F3XYpUXfZ7m\\\\\\\/JkBgSVjmsM4isbQU+\\\\\\\/CtHjzOFbcdjj1DqvYstT\\\\\\\/JfAgNGkTJd6FDV93\\\\\\\/hRLuUjj33K3HKhjHT4d7T0q\\\\\\\/cp2icCUe8Ce7Px39AJYgat9CZHMYunWjkELWbvNYyxg5Im14ZPKpVTjNJ\\\\\\\/l4HabxHj7FwtGa0xV9qkKHznVNmqwnwvWfm+z\\\\\\\/D+EjCjKSeZ1anDhrC6U2N36\\\\\\\/ULPzorrVVcRGpy+j2zH21xCR9ukJfdhy8wAmsLBOjwN1hn8BEIO9ufJn7ioyh2Emb5vq17vSDGCzEdaLH8RE1WHz3QrgKGxq\\\\\\\/1cOAXTjmzAfcyBm3Wb\\\\\\\/0Db2PagFHtdusd9+QdKr8O6qBJMdl02MpnkNtmnETyReGxTLgewW48Dx5oI8NmVsLQMaUCYvWEpDeAK040Mgg8mAPV2rVN5aF8GwEzDW\\\\\\\/TI88HKWYu7V0+bbA3kx+TufYrGZA9HXXDOVv0Kon40qtQvTScTy8XWtZokBGHUkdNdnLXjFH9bUaVeph16gQCzsFBlTyjiJWVApmTfJlQM7vt1hEsuvN0OaKfA+OqoM34v9xUL17D0ZvNm0PvmMtxuMgr5M5xO13+yms2FK\\\\\\\/Ts\\\\\\\/kpkDKrIG03yt3Jqfp0vg0qDmpqgFYWs9uvv7q5r8au+cb4HkSYWD5Wnrc4Y83TkUV\\\\\\\/lo315wof\\\\\\\/W0E0vgjGLMuncJrUGIFBQVNxKhKAX5MCYo8oH7lq5VFubI\\\\\\\/UHitAih8QsebMKytJLrurSG23w978ojRPw1iDfWHzQTy3u1SoUJhaCSJJHPlXpaHIIPyZFDumAbHcUCaC5zCUapJquW2hOlpe6R56l2Qm9Vx83FkKkj4uLmKrgvYIQv3FqemccMcSRCTfk4ntBZPIO1S4ix\\\\\\\/PqVnsFt0xwgSWXmsNk6ZXSs99gRz9o3X15Z22sW5jor1vzHtz983rieJ46tTNeRbFSTJK202eG\\\\\\\/6vF8qrD+HpysJlMVUkENPQrDquYzavIq0fvm11sYsYjDn4ze\\\\\\\/Yjbl6RbIhiO0pG8lkApdKNPdc90V38TAntjIjo21FhoBz0NPIwpBSquyWJoPENgio+GIJ6H6vAR3AMnfejUZhkzmsA5eV9jlsb6Z9bWKPI1vHVtaG4CTBy19892cDGZquGfoi2bgYxWOEmSVXhYrpLHPiwVUCbwSGMl1\\\\\\\/FXrLmpIuvOZSsUyoEc5\\\\\\\/d0KtBK57o\\\\\\\/IBbh4H4sdEkSLiQSQOrZJKmetixoid1pnbTWXYi4XOcfbS60PjecXVFMX3tBT9uVWJy6O5YgGoF0F9vZeF6dMHU4cYrPTOeP0qjbuym+HSXNNXRPJev6gLClJ932lWJrMVnzepmZjE1G6PNYx3g3UQ27YIThmjZGMHSxKX+\\\\\\\/Fu1dC6iJKoAdWk+8nFKI8+xBiLVb9fLN8ZZNHkrfxb5\\\\\\\/zncKPPwJ+yVDynp1yaIB8hJUAQWYey\\\\\\\/behekbH2PWErNG9qKP0En7U8h6ZtXPTmaX61trmQ7WQAyaw\\\\\\\/oZ7hFEOB6NgDItZfkf6B+EU4cTMycF2jC+bINAG0t90Esvdku3fUbWMI6lov1HOwVK+64zXsDtj\\\\\\\/g8QxLvPgmM693pfbS029FIgKuSldED84fBTZWrPmJFoVa\\\\\\\/k\\\\\\\/LuPu6CcCSRZ26pTUYBZ5cDxdWDMcmrVT09GvgMt6yoMM\\\\\\\/cVIsY+3RWX2g2JexXwODg0+aGpxN4cH6KQmYE6UPTD+mhWv1FGn4RcoCrcY9qJiuMstX31SxIM3MK9elbnRnQ7iN3a2PBQ1G5gE2fsnA71Qu801y9uqvmUiuDWqcRQ18qz79zd3Ro\\\\\\\/o6yKILz9qsrd65SAcaoZQBi53LaOh6dKLklbRKN2PlcokaoNiCsmzCQYJtv\\\\\\\/apH+G7Wwh0aPHDPFqBpILpE1shV7vhCpWkoMhrqffH1mwumkngDUy0sUQdMDP7HmADg1dpU7i3iUQf9R6aUQVABsYukV812rdYqbLlbH4pP0kjMMdahLt\\\\\\\/arMuRxKLEy4E5lpDptpw2TgUSVveK1WZkyvjp2U1yQ3V8C19GY9IH7oiquSunvnGS85opT0NZrVgmtBUx\\\\\\\/IRqkZFHLiZnE61tXvv9n+kFFr6H6sFoZ0wIennpva4ZFoqqvwwtqI6jxFkh4Taad\\\\\\\/B0gy1M\\\\\\\/P1jjDmrb4vqpXQ1xYiKdqZPhaghXRNb5dar79\\\\\\\/WJRXsllhgnz1y63Q5w9rx8SfYLiy6sO0Rt1bMJp5CZqnWcnNkOIEMUhDp\\\\\\\/GCXw5+\\\\\\\/fg\\\\\\\/Qi7xXgJXQphSBVXRCQCbL1pDkA9Y19FKUI+5XSZ8bM8nTBpiI2K\\\\\\\/MYMQvWbpVy0y\\\\\\\/MgBrywX9RCS+oq0qUbB5Vir5YRQxiM+r2PamD4THPCxREHy5USEJORpZXHpQ42Twvdwo7aTPr+q59dlMcuQV\\\\\\\/iNdIcDJlh1f251NYj7rfElahtQq99VOCm+td7MMU1YjzRSWLXnPSIs8gOQGcyBDC7nNvFQJlkACdIEOtQHfoZ4uzTNGG7SiCICcWdOILRSeAliAgoqE4LSfrmGuqjpp4MCZ1pUxORY1V\\\\\\\/NKXCb8kaRuYG+zKAHTcMq0yoQyxpesQhlJXGsMTdg3fNVeEhrznOcUC3JJKMC3QC9kIT\\\\\\\/BOebMQIsByIQGXuO\\\\\\\/Zp98ivvbXD2YAy4zSDEjRzpk8GcbK9\\\\\\\/vJNNhNBqxU7d\\\\\\\/QA3ZMpBP3eesYg4EjOzP00lAbfdNKi2M21l0dG2\\\\\\\/pcytHlSadYEeBZTMtJ9LcTcuLrUFNRkj+nhv0a+TDvzhxhEqRLWt1bSOKsglLaD2nrMyo1Guji6ppIDIZB\\\\\\\/9lRZtzR3Wy\\\\\\\/ZzkK6kdxXLo9dqN3INcVhS4XTfvBwcffRSgj1LMt2ujCq6WV5eEpErloCJxGd5BoMgX0FcCCrBKUOmeLA7mQRdagw2MgPUSh7ZngG+4\\\\\\\/9IFPZEctBdTi1pAmmQ5sF84p2VENDESMkfnHBJstcUOmGQzR9lzt9QWF5Gxd\\\\\\\/fKU1uxzgHWaQ8\\\\\\\/l4bSqSMXKuvucE421H+BGMD8VvBfVSgU4kOl\\\\\\\/r+lzCQZ0vOzmMCJiAbMjENdRx\\\\\\\/F7hnu8RIbiTWhQvlqB2g3d\\\\\\\/z3asPhnDwV7RBvkY93P6ZKFZbNqN26CueEiywp6crCVvYqusEL\\\\\\\/Yilq3\\\\\\\/A0ybjcs+UM57CTui2bwGiYjzeNmRQ\\\\\\\/nB+InF1wnIVfMVk4\\\\\\\/+B8q3TaFsCda5oG4gBA5LMf\\\\\\\/TVwUwxcYCv6M2xhYuar9gc43u3Pc2I0OqR8eLx5jvMid6Eldm8DSl3+qJg9+6rve5XgqiuarnpRGcYsnsQzXo2oB5FZtaht\\\\\\\/pFMH7jxoS9aAIiFGtvKNdpqMWLeSVqnfD1yO16v5gfXqdXL1\\\\\\\/nLOHxhusvJxNDbh6MAis8MiQaxlGFcG\\\\\\\/jgrHAiDsJzYTplXEMejZ3ICSawFvM40fK\\\\\\\/\\\\\\\/+UXBKgR4ZGVFdXI6V2NmTEXxSPWENeVtKAI9TjGvJUOJ0eVQjV3birnNTBoFiun9z1pDQZ9QURhBCLjO0raXCPJjNV2FM++NQAzSNorWKmeobOzn2IiftmQLGb63vzoQKZTm9YY4BaLKearePX6t+W9q7vP\\\\\\\/tVmLvwGNKS7PGY47wHZE8E\\\\\\\/wu1pTNAu6DVZZblxPwsbi5nvPi+ZYjtYcL584Uz0iu\\\\\\\/v44QcO9CUfUx7W4WP4ezRWJKcSRcoHRxaQPu\\\\\\\/Oa5E\\\\\\\/6IneDxJHzIlMAIqynsztBFcQ2wSZsues9+5oZC9EqID43Z+tM1AGxennbEMya29yFA6aDoZDINZkj1WgYHeDw4EzkmGKUo7DaSImADrYATTc27+iibw3RqXCVo6+yHXNRbMwCsSrLl4FaWhsHsurVjmS7xEZMkYW\\\\\\\/LtkH0WcVSi6SSiUo2UE543\\\\\\\/\\\\\\\/aWXQAU1NuwyYfUREdbAM79rGxPG31I0fAAac4codo3POy3mxhmyPL\\\\\\\/5iLy5cqCuZHhy2gAEJc6I1vnW\\\\\\\/QczsMknb8B0JsBo1wjqCVMx0f855KOzeMEHch7vUdWAL5cfd4GMKXQ6SbCqK1lg6jdn5t+wJsOQ8rzEGV2wty7vVx5RpZA0sOZ\\\\\\\/by0I4lHOLnX8y4x32UJsIigJDUxgLSHAxVvBzthaHTSLx2TG2y1f8F7hPHfDh4x46+nO4ozxbgQlogh12Jpkfj+mKzPuVTqiI1618fZtPGASy9kyA2HXGI6Cg1YhMMfSyK5hiw+U+vExXn7wrxnIbnnXpWWHuvpN7oe\\\\\\\/ZRCMYowkti74WZWQBHE\\\\\\\/uGBu3VX7daDk9rKfkI3eSh9fxwlVDxEOk9BChcYRO70foN1HVRY3C3oiNt7y8SggvhyW\\\\\\\/UyhLvMw4VNAGRFt5UMSsxm4VYKOHIuScZGQX9ggGuCnOhiYrg2VcIQNUP9mO5C47BdUT+aTgLzMJ4mL\\\\\\\/ncKxuPqOb2V5cRmmo2WES\\\\\\\/LamHmm6RxiakZl3jC\\\\\\\/ODWICkbirHk04Eta7FoCpjRjKMcc6K48ywCj3IUA8afZVxt9ZT7bUjEDPiaJzPTg1obx4yfbUGEi+M7JCdqX9w+u8BaqR\\\\\\\/1miYP0X9FrqlQVTnhLJw\\\\\\\/VThXm3Mj9VNK+gZ+sw+1iwpbhM7DOhLqlTIyFn1r2ra90lIHXFQp8oCHv5QDJO7lNOsHiq0ic\\\\\\\/FoCyyyEg4Xs4JdoBlhmZp0VjU83BBAEwUNHz0zB68BBsav4Tn+BMC7LbAwSzhM9hb1NhGaIt6P2JTwTqWzRIOOHB3msNqa5qSplXkdt\\\\\\\/s6HxN\\\\\\\/MNfHk7aLfnlRreD2hbyA63IFWkAz1\\\\\\\/I5rYpOEniDJ+mMosTNl9\\\\\\\/GT1AwrpDKKjzDR4rxBHB\\\\\\\/fL4rQtsjdBvq4YSg7YLgQT5eqSVHjQKpRUny5jEXBRkfnB0+js4JT4b2wSMvFw5S6c+\\\\\\\/2ciQB2\\\\\\\/2S+l80BiwEqJaHH7g7UbAAN0MJFXnnT9iNzshlaYaeja8jQ766H\\\\\\\/9iDJc17aY3LooiRPiqB33Zrb1RDlgw0pEWpo+DoiEEfYswNTIL+XtMEbYUJDhazF8B8yMM4Bm3BaP+MeIcjziK+8UiGwXB6KBvlfmk3f180ha3kawvg4sbnsWuBVN07IAAvZ7iEkBtjLM8y6yJfvCqlyF6RBUUagHCDKhYdeMqRm0u\\\\\\\/N\\\\\\\/16KWiBYRBv59xyb4BxlBklk9ueRQ7AjaOwcXC8OxkiMg5vJUi0hQByn\\\\\\\/+nfmUglarEzdaCnqu3WLh\\\\\\\/zv4G8\\\\\\\/smNNZ4VQnQxGEXqdP7n4cCMI5WBFfjyANCkpt+UDbMPHiqy4VqN2gFY+KC+WamKALm+0ZYNkOyTL2RXxP9Cw6CWPH4j698bVVkef8QDKWn6B9gBfYPr60R+DQi3jwSuoAh03rKcMHx0PtOWCfaIXEShfUYSQyqqtfhnp62cEpbOTQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022JDGtu3rKZ17vVPLkneTXQw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d7d8cca56358fce326dc4d1a963e85970db6d9c16d5505d6d2e459fbe8d510e5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PTzt46krC\\\\\\\/6RS5yi\\\\\\\/4DRRGd8NJVRxcq53lnOExhHHuVhJdJg4N5f\\\\\\\/D66UCBo9\\\\\\\/fvM0j00BKPQARoypZLWkYLRX2rndOlcDHp1wL7YBkJb3\\\\\\\/yNEzRBp2m5cZiwVj1luARz5r907Hl5gFaQLwQquekPEIKU37hlblzdo2hkxA\\\\\\\/IEPIE5Tx8nVt4r7FkMLbtHFv0ai4voO9KXA+yOaH+Y1Mskohm91s0ST55mQKzHGHPA29DbVDx\\\\\\\/pnUBOvQce5roBbxzRj2krKV+N92zxErQw9sFmUgDTBVqsAywGOh1lLEowQcpsMtVj8FgcBzIGy\\\\\\\/HB1a8dHmVFTuNLOSDea9Nt4O8BWzuF1PpFOWgcJq80bQXYBJJqIrInTZK\\\\\\\/QfqTiLt5mZZYKc9DNSD4KCGSDekN+GSqrW1mp8aEv\\\\\\\/a2An\\\\\\\/T+cAD9k8LEgMOygONPi1VH+febSyRs\\\\\\\/TCYGV9u4GGgbV9wLOrWajFI2fN+KguJlf+rdII1IYeeTyP65uMe5x0U7n5pZDzqxu\\\\\\\/vW3g\\\\\\\/jb6cKgtMF3lldHfdclZutZoZ1I8AVqNnETls6y3x0LPjS8RtXu\\\\\\\/ahUd9vUqfYsEZ46cT3Qmx\\\\\\\/eR2dxFWYBTB6PcRGEmpwUHDfMwor5tS02hCmmMtVFUTcPtpKbdEGEEkiCyLhOC0g7fsMUrYM1ATrjQvoGP\\\\\\\/89vlmIL5kGeA1DCPz7xG7K3KIKOSZIAww0b\\\\\\\/ZjCHUzwKH+yQPkBy\\\\\\\/Fe3766bcJ91FhUOEd2MjmHQ7bvDSJRe9s\\\\\\\/OsDXPHDGzXh8PEojueycDXitF3k4izTK8v+f4sP+2jrwUVIF501iQcCbUG2Wcu7DwkRISWhEPX\\\\\\\/43eCNkar9xGdUfwK5WE3ijax11kiXTSfegf4LWw4W38p4arxQYZ5yIFTHKCijLwMMVjn49eGwAd8MMNq7Z1jalC+O\\\\\\\/5fq4qaWmfBB3XVUssn\\\\\\\/Ig7xnPHOLYSfQRoCNjn+T8Gj6sSO0j6HHon1ufHUuQ+BLKTpCK3OQfghs58dY5kdZLrSVuZ6InCBszABCkvU9Sa\\\\\\\/zMkjyDR\\\\\\\/yuCRl3pQDNJ\\\\\\\/DthUeK2Bewpoukqzq2\\\\\\\/8uI8fowae3AtDbZ8CraiiWiX8OXJacqb8M\\\\\\\/PO6mu45bppm9nZs9rovStIG4YuEotPmu+N0xbA4wVuAwX9zFdmGBr2+e+BWRH4TmU9Iwfe+Hkk1Bi8s8e93TPxFYNcYNWBLMwyO3Uuh6oVt8tPiXv6A+R0XmK+L1CVxoofo3sGD8YUepjYEzpvcsUC7xf0LKRyrEQPtS6l+JrquouFpMzY\\\\\\\/eC6kz5salZmCt+cFSfEpgcx817OLnbQSOwkl7hOisLHBJKuz\\\\\\\/5QUQiV7NsRTZSRiHIFDfbIa1bmQbWcJBnk0CK+FcTZlZhecpO5MHLruVAu8bS18dtAx4Tgby\\\\\\\/13DjCKxKSMa0GSynEnRkSEIiNVHxZsNEbfne3SBe3ey5aOf8\\\\\\\/xvKOgSoYuHLN8ZnWwL04Cx0qYJZDOQX7l3oTIsL4XfZLsTdxW+n9pksBn9iSAhsyzBKRuJoFhwjLz9\\\\\\\/RIQHKMw0+7sESE2na4m1ZW+\\\\\\\/70mW2mSXsPSKAbQv7eSzvSjhAqP68UVtc4JOWqhMG8zYDWS\\\\\\\/FhH\\\\\\\/3jYzllS8CogppfQwohfei0Ue51xJkmPaKk996hXm+UCWP\\\\\\\/iW2nhk+JSP0wzQy9MfpE0O6eZXq7fHZnAB2J\\\\\\\/DkhKN185BKupVbClvL14O4PaKGuOQNL5MDZRRCyH8bm7i2yb1yHlumiEBg+hw0rhRq9VRLIXFqU+cLkkT+T++4+eNc5rrggX3F6ihV3XEe64K69aQ56lirsvCu+vvLupMbA3goAxv4fWM2MeEMJu1mFEHdP70CZzmrr5+pH71v59mxsK84ZLidQ+vN4C+ecrkNPtFxVSZ3X1h3W5igEUn29tE6DVRhb+9ShEouVEjJEmMJf8t8muohgge7ajw\\\\\\\/SAwZLmo\\\\\\\/KoduMLT1Umf8wgjG\\\\\\\/ds8KTI2BcXtTJgKGX2G\\\\\\\/k23mDA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022p7Vwo87XmdVPHbfBlYeueA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022584856a69551a8dadda315ae7c6ad0f5995c8a29d1865ec0c8cf51c5bc8c064f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-489462 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1513" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Wo_wa0gsqBIsI16jwPWwmX7TW8n9706fGZcsqZ8GN7Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1513" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qoRnsuwwkkYklRTQ3ozrCnImUszSyCUPibgrYTcgf4s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1497" title="A Flow Cytometry-Based Phenotypic Screen To Identify Novel Endocytic Factors in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1523" title="Identification of General Patterns of Sex-Biased Expression in Daphnia, a Genus with Environmental Sex Determination" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489462" id="node489462" data-pisa="ggg;8/5/1513" data-pisa-master="ggg;g3.117.300396" data-apath="/ggg/8/5/1513.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of <em>NCED4</em> in Lettuce (<em>Lactuca sativa</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lien D. Bertier</span>, <span class="highwire-citation-author" data-delta="1">Mily Ron</span>, <span class="highwire-citation-author" data-delta="2">Heqiang Huo</span>, <span class="highwire-citation-author" data-delta="3">Kent J. Bradford</span>, <span class="highwire-citation-author" data-delta="4">Anne B. Britt</span> and <span class="highwire-citation-author" data-delta="5">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1513-1521;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300396 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489462" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lien D. Bertier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lien%2BD.%2BBertier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertier%20LD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALien%2BD.%2BBertier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mily Ron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mily%2BRon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMily%2BRon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Heqiang Huo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, Seed Biotechnology Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heqiang%2BHuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeqiang%2BHuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kent J. Bradford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, Seed Biotechnology Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BJ.%2BBradford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bradford%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BJ.%2BBradford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Anne B. Britt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BB.%2BBritt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Britt%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BB.%2BBritt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Plant Sciences, Molecular & Cellular Biology, Medical Microbiology & Immunology, University of California, Davis, CA 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/5/1513" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:489462" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:489462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1513.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:489462" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:489462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1513.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:489462" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:489462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/5/1513.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:489462" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:489462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/5/1513.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:489462" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:489462/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">CRISPR/Cas9 is a transformative tool for making targeted genetic alterations. In plants, high mutation efficiencies have been reported in primary transformants. However, many of the mutations analyzed were somatic and therefore not heritable. To provide more insights into the efficiency of creating stable homozygous mutants using CRISPR/Cas9, we targeted <em>LsNCED4</em> (<em>9-cis-EPOXYCAROTENOID DIOXYGENASE4)</em>, a gene conditioning thermoinhibition of seed germination in lettuce. Three constructs, each capable of expressing Cas9 and a single gRNA targeting different sites in <em>LsNCED4</em>, were stably transformed into lettuce (Lactuca sativa) cvs. Salinas and Cobham Green. Analysis of 47 primary transformants (T<sub>1</sub>) and 368 T<sub>2</sub> plants by deep amplicon sequencing revealed that 57% of T<sub>1</sub> plants contained events at the target site: 28% of plants had germline mutations in one allele indicative of an early editing event (mono-allelic), 8% of plants had germline mutations in both alleles indicative of two early editing events (bi-allelic), and the remaining 21% of plants had multiple low frequency mutations indicative of late events (chimeric plants). Editing efficiency was similar in both genotypes, while the different gRNAs varied in efficiency. Amplicon sequencing of 20 T<sub>1</sub> and more than 100 T<sub>2</sub> plants for each of the three gRNAs showed that repair outcomes were not random, but reproducible and characteristic for each gRNA. Knockouts of <em>NCED4</em> resulted in large increases in the maximum temperature for seed germination, with seeds of both cultivars capable of germinating &gt;70% at 37°. Knockouts of <em>NCED4</em> provide a whole-plant selectable phenotype that has minimal pleiotropic consequences. Targeting <em>NCED4</em> in a co-editing strategy could therefore be used to enrich for germline-edited events simply by germinating seeds at high temperature.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crisprcas9" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9" rel="nofollow">CRISPR/Cas9</a></li><li class="kwd"><a href="/keyword/nhej" class="hw-term hw-article-keyword hw-article-keyword-nhej" rel="nofollow">NHEJ</a></li><li class="kwd"><a href="/keyword/targeted-mutagenesis" class="hw-term hw-article-keyword hw-article-keyword-targeted-mutagenesis" rel="nofollow">targeted mutagenesis</a></li><li class="kwd"><a href="/keyword/lettuce" class="hw-term hw-article-keyword hw-article-keyword-lettuce" rel="nofollow">lettuce</a></li><li class="kwd"><a href="/keyword/germination" class="hw-term hw-article-keyword hw-article-keyword-germination" rel="nofollow">germination</a></li><li class="kwd"><a href="/keyword/thermotolerance" class="hw-term hw-article-keyword hw-article-keyword-thermotolerance" rel="nofollow">thermotolerance</a></li><li class="kwd"><a href="/keyword/amplicon-sequencing" class="hw-term hw-article-keyword hw-article-keyword-amplicon-sequencing" rel="nofollow">amplicon sequencing</a></li></ul><p id="p-4">Site-specific or targeted mutagenesis is a powerful method for making specific, intentional changes to the DNA sequence in order to study and alter gene function. Targeted modification of DNA sequences can be achieved by exploiting DNA double-strand break (DSB) repair pathways. DNA repair follows two major pathways: (i) error-prone non-homologous end joining (alt-NHEJ or microhomology-mediated end joining (MMEJ) can introduce insertions or deletions, often leading to frameshift mutations; and (ii) homology-directed repair (HDR) that involves a DNA repair template with complementarity to the target locus to effect a specific modification (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Voytas 2013</a>).</p><p id="p-5">CRISPR/CAS-mediated genome editing is revolutionizing foundational and translational research fields, including plant breeding (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Podevin <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bortesi and Fischer 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Puchta 2017</a>). Breeding pipelines include an initial pre-breeding phase to identify genes determining traits of interest and a second breeding phase to introgress these genes into commercial cultivars. Breeding programs currently rely on classical genetic approaches that are laborious and time-consuming for both phases. The time from the phenotypic identification of a gene of interest to release in an improved commercial cultivar is often more than 10 years (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acquaah 2012</a>). Genome editing has multiple applications as an advanced breeding tool. It can be used: (i) to validate the functions of candidate genes using single gRNAs or pooled CRISPR libraries (knockouts); (ii) to modify alleles into more desirable ones (allele editing); (iii) to add or replace genes into cultivars without linkage drag (gene replacement); and (iv) to stack genes at single chromosomal locations so that they will be inherited as a single Mendelian unit (gene stacking) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kumar <em>et al.</em> 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhao <em>et al.</em> 2016</a>). Knockouts are increasingly straightforward in plant species, while allele editing, gene replacements, and stacking need further development to become routine.</p><p id="p-6">The efficiency of obtaining mutations with CRISPR/Cas9 delivered to plants by <em>Agrobacterium tumefaciens</em> can be high (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gao <em>et al.</em> 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jacobs <em>et al.</em> 2015</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhu <em>et al.</em> 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Tsutsui and Higashiyama 2017</a>), but many of these mutations are somatic, which makes selection of the desired mutants inefficient (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ma <em>et al.</em> 2016</a>). Editing of several plants, including <em>Arabidopsis</em>, rice, soybean, and lettuce, has been achieved using Cas9 and Cpf1 ribonucleoproteins (RNPs) transfected into protoplasts (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Woo <em>et al.</em> 2015</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Malnoy <em>et al.</em> 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kim <em>et al.</em> 2017</a>). Mature plants have been regenerated from Cas9 RNP-edited protoplasts of lettuce (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Woo <em>et al.</em> 2015</a>). The use of RNPs for editing has advantages, such as reduced off-target activity (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kim 2014</a>) and lack of integration of foreign DNA, which could be beneficial for the commercialization of edited crops. However, there are also disadvantages, as regeneration of mature plants from protoplasts is difficult for some genotypes and species. Also, tissue culture, especially protoplast culture, often results in a high frequency of somaclonal variation (Engler DE 1984; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mou 2011</a>).</p><p id="p-7">Lettuce (<em>Lactuca sativa</em>) is a representative diploid member of the Compositae, one of the most successful families of flowering plants (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kesseli and Michelmore 1997</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Funk 2005</a>). It is a major crop with a worldwide production of twenty million tons and is a $2.4 billion industry in the United States, mainly in California and Arizona (“Anonymous. Crop Values: 2014 Summary. USDA, <a href="http://usda.mannlib.cornell.edu/MannUsda/homepage.do">http://usda.mannlib.cornell.edu/MannUsda/homepage.do</a> (2015). 11.”). The reference genome sequence of lettuce cv. Salinas and detailed genetic maps are available (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Truco <em>et al.</em> 2007</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Reyes-Chin-Wo <em>et al.</em> 2017</a>). Production of lettuce could benefit from genome editing of numerous traits (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Michelmore <em>et al.</em> 2017</a>). The <em>9-cis-EPOXYCAROTENOID DIOXYGENASE4</em> (<em>NCED4</em>) gene is a key regulatory enzyme in biosynthesis of abscisic acid (ABA) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Huo <em>et al.</em> 2013</a>). Huo <em>et al.</em> (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Huo <em>et al.</em> 2013</a>) showed that silencing of <em>LsNCED4</em> in <em>L. sativa</em> cv. Salinas using RNAi prevented inhibition of germination at high temperature but did not have negative pleiotropic effects on normal growth or stress tolerance. Therefore mutants of <em>LsNCED4</em> could be commercially valuable in production areas with high temperatures during crop establishment such as in the Imperial and Yuma valleys of southern California and Arizona. In addition, <em>LsNCED4</em> could provide a simple, whole-plant selectable marker for use in a CRISPR co-editing strategy; simultaneous targeting of <em>LsNCED4</em> and an unrelated gene of interest would enable the selection of plants enriched for editing events in the gene of interest simply by germinating seeds at high temperature.</p><p id="p-8">In order to better understand the timing and nature of editing events in lettuce, we conducted a quantitative study of the efficiency, outcomes, and inheritance of CRISPR/Cas9-induced mutations in the NCED4 gene. We evaluated the timing of modifications induced by CRISPR/Cas9 over multiple generations of stable lettuce transgenics. This revealed frequent mono- and bi-allelic somatic and germline mutations that occurred both in cell culture and later in regenerated plants as well as variation in efficiencies and mutation events induced by different guide RNAs (gRNAs).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Construct design and transformation</h3><p id="p-9">Three gRNAs (gRNA_1, gRNA_2 and gRNA_3; 20 nt-NGG, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/TableS2.tif">Table S2</a> and Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>) were manually selected in the first half of the intronless <em>LsNCED4</em> gene and checked for lack of off-target specificity by BLAST to the lettuce cv. Salinas reference genome (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Reyes-Chin-Wo <em>et al.</em> 2017</a>). The Gateway-compatible vectors pEn_Chimera and pDe_Cas9 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fauser <em>et al.</em> 2014</a>) were used to generate single gRNA expression vectors (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). The <em>Cas9</em> gene is codon optimized for expression in <em>A. thaliana</em> and expressed from the parsley ubiquitin (PcUbi) promoter and terminated by the pea 3A terminator (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Fauser <em>et al.</em> 2014</a>). The gRNA is transcribed by the Pol III AtU6-26 promoter.</p><p id="p-10">pDe_Cas9 was adapted for selection of kanamycin resistance in plants by replacing the <em>bar</em> gene cassette with the pNOS-nptII-tNOS cassette. Synthetic gRNA oligos with <em>Bbs</em>I overhangs were cloned into <em>Bbs</em>I digested pEn_Chimera and the customized gRNA pEn_Chimeras were then transferred into the binary vector pDe_Cas9 by single Gateway LR reactions (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>).</p><p id="p-11">The resulting binary vectors were electroporated into <em>A. tumefaciens</em> strain LBA4404. Transformations of <em>L. sativa</em> cv. Salinas and cv. Cobham Green were done by cocultivating the constructs in <em>A. tumefaciens</em> with excised lettuce cotyledons, after which plants were regenerated via somatic organogenesis (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Michelmore <em>et al.</em> 1987</a>) by the UC Davis Plant Transformation Facility (<a href="http://ucdptf.ucdavis.edu/">http://ucdptf.ucdavis.edu/</a>).</p><p id="p-12">Multiple false-positive transgenic T<sub>1</sub> plants that were resistant to kanamycin but did not contain the complete Cas9 gene occurred in this study (12 out of 59 T<sub>1</sub> plants). This is likely due to the position of the <em>NptII</em> cassette in pDe_Cas9 immediately adjacent to the right T-DNA border. Since T-DNA integration occurs from right to left border, plants with partial T-DNA integrations (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gelvin 2010</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2012</a>) would contain a functional <em>NptII</em> cassette, but an incomplete and thus non-functional <em>Cas9</em> cassette. The original study with pDe_Cas9 analyzed T-DNA segregation of T<sub>2</sub> <em>Arabidopsis</em> plants by germination in the presence of antibiotic in media (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Fauser <em>et al.</em> 2014</a>) and so partial T-DNA integrations would have gone unnoticed. The higher number of false positives detected for cv. Cobham Green (11/33) compared to cv. Salinas (1/27), indicates that the plant genotype could possibly influence partial integration of T-DNAs. We have since modified pDe_Cas9 so that the <em>NptII</em> cassette is positioned immediately adjacent to the left T-DNA border; no false positives have been detected in subsequent experiments (L. Bertier, M. Ron, A. Britt, R. Michelmore, unpublished data).</p></div><div id="sec-3" class="subsection"><h3>Analysis of calli</h3><p id="p-13">Calli were analyzed for editing events three weeks after cocultivation. The frequencies of CRISPR-induced mutations were analyzed in five calli for each combination of gRNA and cultivar (30 total) using three approaches. Genomic DNA was extracted using the GenElute Plant Genomic DNA Miniprep kit (Sigma-Aldrich, St. Louis, MO) and the complete <em>NCED4</em> gene (1,737 bp) was amplified using primers NCED4_Salinas_1F and NCED4_Salinas_1R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/TableS2.tif">Table S2</a>) and digested using unique restriction enzymes (gRNA 1: AgeI; gRNA 2: MfeI; gRNA 3: FseI) that had recognition sites at the predicted Cas9 cut sites (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). An undigested band was indicative of Cas9-induced indels that had mutated or deleted the RE recognition site. The undigested 1737 bp <em>NCED4</em> amplicon was also Sanger sequenced for 10 calli using primers NCED4_Salinas_1F and NCED4_Salinas_2R. For the same 10 calli, we also generated short ∼150 bp amplicons specific to each gRNA for amplicon sequencing.</p></div><div id="sec-4" class="subsection"><h3>Amplicon sequencing</h3><p id="p-14">Calli, T<sub>1</sub>, and T<sub>2</sub> plants were genotyped by deep amplicon sequencing of the <em>NCED4</em> gene. For T<sub>1</sub> and T<sub>2</sub> plants, DNA was extracted from leaf tissue of young plants using a custom DNA extraction protocol using a 5 M guanidine isothiocyanate buffer. For T<sub>1</sub> plants, two sets of primers were designed to generate amplicons of ∼150 bp. Primer pair NCED4_gRNA1_F/NCED4_gRNA1_R spans gRNA1 and primer pair NCED4_gRNA2+3_F/ NCED4_ gRNA2+3_R spans gRNA2 and gRNA3 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/TableS2.tif">Table S2</a> and Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). For T<sub>2</sub> plants, amplicon libraries were made with the same primer sets as for T1 plants as well as with a third primer set (LDB_226/LDB_227) that amplified a larger ∼380 bp amplicon spanning all three gRNAs. Results shown for T<sub>2</sub> plants are obtained with primer pair LDB_226/LDB_227. Amplicon libraries were made following the 16S Metagenomic Sequencing Library preparation protocol as provided by Illumina (<a href="https://www.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/16s/16s-metagenomic-library-prep-guide-15044223-b.pdf">https://www.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/16s/16s-metagenomic-library-prep-guide-15044223-b.pdf</a>) with some modifications. For T<sub>1</sub> genotyping, amplicon and indexing PCRs were done using Phusion Hi-Fidelity polymerase (New England Biolabs, Ipswich, MA). For T<sub>2</sub> plants, GoTaq polymerase (Fermentas, Waltham, MA) was used for the amplicon and index PCRs. Dual indexing was done using the Nextera XT system (Illumina, San Diego, CA) using 16 i5 indexes (S502-S522) and 24 i7 indexes (N701-N729) enabling multiplexing of 384 individual libraries. SR 100 bp or PE 150 sequencing was done on an Illumina HS3000 or HS4000 platform. Sequence analysis was done using CrispRVariants (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lindsay <em>et al.</em> 2016</a>).</p><p id="p-15">Given the recent concerns of index switching during sequencing of dual index multiplexed samples using Illumina platforms (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sinha <em>et al.</em> 2017</a>), the pooled libraries were treated with Exonuclease VII (NEB, Ipswich, MA) to remove all residual single-stranded index primers, after which we performed a BluePippin (Sage Science Inc., Beverly, MA) size selection treatment for removal of all fragments smaller than 100 bp. After sequencing, we analyzed the occurrence of index switching in our datasets by demultiplexing the unused index combinations and mapping the reads to the reference. For the two 384-plex libraries analyzed, we calculated an average rate of index switching (the average read number of the used index combinations mapping to the reference divided by the average read number of the unused index combination mapping to the reference) of 0.25 and 0.47%. While these levels of index switching are low and do not impact our results, index switching should be considered and evaluated by demultiplexing the unused index combinations and mapping to the reference sequence(s). Appropriate thresholds for low frequency variants should be set to take possible index switching into account.</p></div><div id="sec-5" class="subsection"><h3>High temperature germination assay</h3><p id="p-16">Germination of wildtype seeds from five different years (2006, 2007, 2011, 2014, 2016) was assayed at 25°, 28°, 32°, 35° and 37° by imbibing 75 seeds (3 replications of 25 seeds) per temperature in Petri dishes containing germination paper (Ahlstrom grade 628, Stults Scientific, IL) and 3 mL milliQ water. Germination frequency was recorded after 72 h. Phenotyping was done on T<sub>2</sub> families from 44 T<sub>1</sub> primary transformations. As above, 75 seeds (3 replications of 25 seeds) were imbibed at an inhibitory temperature (Cobham Green: 32° and Salinas: 35°) and germination percentages were recorded at 72 h. After 72 h, non-germinated seeds were separated from germinated seedlings and were transferred to room temperature to allow germination. For lines without germination at high temperature, DNA was extracted from leaf tissue of 8 (Cobham Green) or 16 (Salinas) seedlings and a bulk DNA sample was genotyped by amplicon sequencing. For lines having germination at high temperature, DNA was extracted from a young leaf of 16 individual seedlings that either showed germination at high temperature or not, in ratios representative of the total germination frequency. For example if the germination frequency at high temperature for a T<sub>2</sub> family was 25%, we extracted DNA from 4 seedlings that germinated at high temperature and from 12 seedlings that did not germinate at high temperature. Individual genotypes for all plants were generated using amplicon sequencing.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-17">Amplicon sequencing data for T<sub>1</sub> and T<sub>2</sub> plants is available at the SRA (BioProject number PRJNA429829). Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a> contains an alignment of the four <em>Ls</em>NCED paralogs. Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a> shows the analysis of editing in calli. Figures S3–S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a> show Sanger sequencing profiles from the calli that were also sequenced by Illumina. Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a> shows the T-DNA construct. Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a> shows the temperature/germination relationship of seed lots of different ages for wildtype Cobham Green and wildtype Salinas. Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a> shows violin plots of the distribution of the percentage of mutated reads for individual T<sub>2</sub> plants that had different germination phenotypes. Figures S9–S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS2.tif">File S2</a> show mutation profiles derived from amplicon sequencing of individual T<sub>2</sub> plants from families with germination at high temperature.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Selection of gRNAs and transformation</h3><p id="p-18">Protospacer sequences for three different gRNAs in the first half of the intronless <em>LsNCED4</em> gene were selected that had restriction enzyme recognition sites at or close to the predicted Cas9 cut site, 3 bp upstream of the PAM. The gRNAs were also checked for homology against the other <em>NCED</em> paralogs. The three other <em>NCED</em> paralogs have a different PAM at the paralogous gRNA sites so that off-target events in these genes will be precluded. (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). This region of the gene was identical in lettuce cvs. Salinas and Cobham Green. No off-target sites having significant sequence homology to the gRNAs were detected by BLAST searches of the lettuce genome (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Reyes-Chin-Wo <em>et al.</em> 2017</a>). Each gRNA sequence was cloned into pDe_Cas9 (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Fauser <em>et al.</em> 2014</a>) that had been modified for selection of kanamycin resistance in transgenics. The three resulting constructs were used in independent cocultivations of <em>A. tumefaciens</em> with cotyledon explants of cvs. Salinas and Cobham Green. Kanamycin-resistant calli formed within 3 weeks on all explants and plantlets regenerated via somatic organogenesis from 4 weeks after cocultivation. The plants analyzed in this study were derived from independent explants.</p></div><div id="sec-9" class="subsection"><h3>Editing was only detected at low frequency in young calli</h3><p id="p-19">Calli were analyzed for editing events three weeks after cocultivation. The frequencies of CRISPR-induced mutations were analyzed in five calli for each combination of gRNA and cultivar (30 total) using three approaches. Genomic DNA was extracted and the complete <em>NCED4</em> gene (1,737 bp) was amplified and digested using unique restriction enzymes that had recognition sites at the predicted Cas9 cut sites (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). An undigested band was indicative of Cas9-induced indels that had mutated or deleted the recognition site. Some calli had a faint undigested band that represented &lt;10% of the total amplicon (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). One callus (Sal_3_c5) showed amplification of an additional band at ∼850 bp, indicating a ∼900 bp deletion (Figure S2Ca-b in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). The undigested amplicons from 10 calli with possible indels were Sanger sequenced. The majority of chromatograms showed no clear evidence of multiple sequences, indicating that there had been little or no editing (Figures S3–S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). The ∼900 bp deletion in callus Sal_3_c5 was evident starting around 100 bp before the Cas9 cut site for gRNA 3 (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). The restriction enzymes were suboptimal for detecting indels induced by gRNA 1 and 2 because the recognition site did not span both sides of the predicted Cas9 cut site. Therefore, libraries of amplicons were Illumina sequenced for the same 10 calli (Figure S2D-F in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). Multiple mutations were detected at low frequency for all 10 calli; the highest frequency was ∼15% of total reads. For gRNA 1, 13 different indels were detected at a threshold of 1% of reads for at least one callus sample, the majority of which were deletions of 1 to 12 bp. For gRNAs 2 and 3, only three alleles were detected, two of which were 1 bp insertions. The higher frequency of deletions for gRNA 1 is likely due to 6 bp of microhomology (CCAACCN<sub>11</sub>CCAACC) at the protospacer region. All three methods of analysis indicated that the initial frequency of editing in calli was low. This could have been due to low Cas9 activity or DNA repair processes in rapidly dividing cells, or due to technical reasons such as differences in promoter activities driving either the <em>Cas9</em> gene or the gRNAs relative to the <em>Nos</em> promoter expressing the <em>NPTII</em> selectable marker gene in calli. Also, at least some of the kanamycin-resistant cells in the calli would have lacked a functional <em>Cas9</em> gene due to the location of the <em>NPTII</em> gene next to the right T-DNA border (see below and Methods for details).</p></div><div id="sec-10" class="subsection"><h3>High frequency of mono- and bi-allelic T<sub>1</sub> plants indicates that Cas9 was active in the founder cells</h3><p id="p-20">A total of 59 primary transgenic plants (T<sub>1</sub>) were derived for the six gRNA/cultivar combinations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/TableS1.tif">Table S1</a>). Although all plants were resistant to kanamycin, PCR amplification of the <em>Cas9</em> gene was negative for 12/59 plants (11/32 cv. Cobham Green; 1/27 cv. Salinas), indicating that these plants contained only partial T-DNA integrations and lacked a full <em>Cas9</em> cassette. This was likely due to the position of the <em>NptII</em> cassette in pDe_Cas9, immediately adjacent to the right T-DNA border (see Methods for details). A variety of editing frequencies were detected in the 47 <em>Cas9</em> positive transformants. Short ∼150 bp fragments containing each of the gRNA target sites were amplified from DNA extracted from an early leaf for all 59 T<sub>1</sub> plants (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/TableS1.tif">Table S1</a>). Libraries from each individual leaf were Illumina sequenced to an average coverage of 5,000 reads. None of the <em>Cas9</em>-negative plants contained editing events in amplicons from T<sub>1</sub> leaves, indicating that T-DNA integration rather than transient T-DNA expression was necessary for detectable mutations. Out of 47 <em>Cas9</em>-positive T<sub>1</sub> plants, 27/47 (57%) contained at least 5% edited reads in the <em>NCED4</em> amplicon from leaf tissue; 20/47 (43%) leaves had &lt;5% edited <em>NCED4</em> reads. No difference in editing efficiency was detected between the two genotypes (Cobham Green: 11/21; Salinas: 16/26). Mutation efficiency was highest for gRNA 1 with 15/17 <em>Cas9</em>-positive plants containing editing events at the target site. Only 5/15 plants were edited for gRNA 2 and 7/15 plants were edited for gRNA 3. All alleles except one (-6:6D for gRNA 1; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) were frame-shift mutations. The most common mutations were single base-pair insertions (A/T or C/G) and deletions (4 to 22 bp). Seven different mutations above a threshold of 5% reads for at least one T<sub>1</sub> leaf were detected for gRNA 1, while four different mutations were detected for both gRNA 2 and gRNA 3.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Amplicon sequencing analysis of 47 CAS9 positive T1 plants. Each row represents an allele, with the reference allele on top. The 23bp gRNA sequence is circled with a black box, with the PAM circled by the small black box. The predicted CAS9 cut site is indicated with a black line. Each column represents the sequence read distribution in a young leaf of a single T1 plant. A wildtype control is shown on the left of each panel. Plants were divided into four genotypic classes according to their mutation profiles. 43% of plants showed no editing (20% reads for a single allele indicating one early event (indicated in blue); 8% were bi-allelic (2 early events with &gt;20% reads, indicated in green), and 21% were chimeras (containing multiple low frequency alleles adding up to &gt;5% of the total reads, indicated in red). Bar on bottom of plots represents distribution of NCED4 genotypes for all CAS9 positive Cobham Green (left) and Salinas (right) T1 plants. * For plant Sal_1_7, a mono-allelic 44bp deletion was present but only detected in T2 plants where a larger amplicon was sequenced (see Figure S11 in File S2). Plant abbreviation is Genotype_gRNA_plant. Genotypes: CG = Cobham Green; Sal = Salinas." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-219073739" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Amplicon sequencing analysis of 47 CAS9 positive T&lt;sub&gt;1&lt;/sub&gt; plants. Each row represents an allele, with the reference allele on top. The 23bp gRNA sequence is circled with a black box, with the PAM circled by the small black box. The predicted CAS9 cut site is indicated with a black line. Each column represents the sequence read distribution in a young leaf of a single T&lt;sub&gt;1&lt;/sub&gt; plant. A wildtype control is shown on the left of each panel. Plants were divided into four genotypic classes according to their mutation profiles. 43% of plants showed no editing (&lt;5% mutated reads in the amplicon, indicated in purple); 28% were mono-allelic with &gt;20% reads for a single allele indicating one early event (indicated in blue); 8% were bi-allelic (2 early events with &gt;20% reads, indicated in green), and 21% were chimeras (containing multiple low frequency alleles adding up to &gt;5% of the total reads, indicated in red). Bar on bottom of plots represents distribution of NCED4 genotypes for all CAS9 positive Cobham Green (left) and Salinas (right) T1 plants. * For plant Sal_1_7, a mono-allelic 44bp deletion was present but only detected in T&lt;sub&gt;2&lt;/sub&gt; plants where a larger amplicon was sequenced (see Figure S11 in File S2). Plant abbreviation is Genotype_gRNA_plant. Genotypes: CG = Cobham Green; Sal = Salinas.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1513/F1.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/5/1513/F1.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489561" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Amplicon sequencing analysis of 47 CAS9 positive T<sub>1</sub> plants. Each row represents an allele, with the reference allele on top. The 23bp gRNA sequence is circled with a black box, with the PAM circled by the small black box. The predicted CAS9 cut site is indicated with a black line. Each column represents the sequence read distribution in a young leaf of a single T<sub>1</sub> plant. A wildtype control is shown on the left of each panel. Plants were divided into four genotypic classes according to their mutation profiles. 43% of plants showed no editing (&lt;5% mutated reads in the amplicon, indicated in purple); 28% were mono-allelic with &gt;20% reads for a single allele indicating one early event (indicated in blue); 8% were bi-allelic (2 early events with &gt;20% reads, indicated in green), and 21% were chimeras (containing multiple low frequency alleles adding up to &gt;5% of the total reads, indicated in red). Bar on bottom of plots represents distribution of NCED4 genotypes for all CAS9 positive Cobham Green (left) and Salinas (right) T1 plants. * For plant Sal_1_7, a mono-allelic 44bp deletion was present but only detected in T<sub>2</sub> plants where a larger amplicon was sequenced (see Figure S11 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS2.tif">File S2</a>). Plant abbreviation is Genotype_gRNA_plant. Genotypes: CG = Cobham Green; Sal = Salinas.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Four categories of mutants could be identified from the mutation patterns of the T<sub>1</sub> plants (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) and segregation patterns in T<sub>2</sub> families (Figures S9–S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS2.tif">File S2</a>). Out of 47 T<sub>1</sub> plants, 4 plants (8%) were bi-allelic mutants, 13 (28%) were mono-allelic mutants, 10 (21%) were chimeras that contained multiple alleles at low frequency in the T<sub>1</sub> leaf and 20 (43%) had no editing. None of the chimeric T<sub>1</sub> plants contained more than three alleles (at a threshold of 5% reads per allele), suggesting that somatic Cas9 activity was low. However, in contrast to the T<sub>1</sub> plants, leaves of T<sub>2</sub> progeny often had more than three alleles, indicating high somatic Cas9 activity in the T<sub>2</sub> generation (Figures S9–S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS2.tif">File S2</a>).</p></div><div id="sec-11" class="subsection"><h3>Knockout of NCED4 resulted in high temperature germination and can serve as a whole-plant selectable marker for germline editing events</h3><p id="p-23">The germination performance of wild type seed was established for each genotype over a range of temperatures between 25 and 37° (Figure S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). Germination of younger seed lots (≤5 years old) of cv. Salinas was unaffected until 32°; at 35° there was a sharp decline in germination percentage and almost no germination was observed at 37°. Older seed lots were slightly more sensitive to warm temperatures. Germination of cv. Cobham Green seeds was more sensitive to temperature; germination of more recent seed lots was unaffected at 25°, less than 50% at 28°, and less than 10% at 30° and above for all seed lots.</p><p id="p-24">To assay the germination thermosensitivity of potentially edited lines, all 59 T<sub>1</sub> plants were selfed and the T<sub>2</sub> seeds harvested. Almost a third of T<sub>1</sub> plants (16/59) were partially or completely sterile (≤150 seeds/plant); sterility is common in lettuce plants derived from tissue culture and was not correlated with the <em>NCED4</em> genotype in the leaves of <em>Cas9</em>-positive T<sub>1</sub> mother plants and three of these plants did not have a functional <em>Cas9</em> gene (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/TableS1.tif">Table S1</a>). Twenty T<sub>2</sub> seeds of the 43 non-sterile T<sub>1</sub> plants were pre-screened for viability by germination at room temperature. After 96 h, viability was &gt;99%. Seed thermoinhibition at high temperature was further evaluated by germinating T<sub>2</sub> families of cv. Cobham Green at 32° and of cv. Salinas at 35°. T<sub>2</sub> families of the same 43 T<sub>1</sub> plants (23 of cv. Cobham Green and 20 of cv. Salinas) were evaluated for germination at high temperature by imbibing 75 seeds per T<sub>2</sub> family at 32° for cv. Cobham Green and at 35° for cv. Salinas. Germination data were collected at 72 h (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/TableS1.tif">Table S1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) and germinated seedlings were separated from non-germinated seed. The latter were shifted to room temperature and allowed to complete germination. Genotypes were then determined for individual T<sub>2</sub> seedlings in families where germination was greater than zero at high temperature. For T<sub>2</sub> families with no germination at high temperature, DNA was extracted from young leaves, pooled, and amplified; a library representing equal amounts of DNA from each plant was then sequenced.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Germination at high temperature for T2 and T3 families. A, Germination of 23 Cobham Green T2 families germinated at 32&#xB0;C; B, germination of 20 Salinas T2 families germinated at 35&#xB0;C. Germination is plotted against the percentage of edited reads in the bulk T2 family. Color coding refers to the T1 leaf genotype. C, D: Germination of T3 families at different temperatures for Cobham Green (C, mean of 6 T3 lines) and Salinas (D, mean of 6 T3 lines) compared to wildtype (mean of 3 lines from different years for each cultivar)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-219073739" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Germination at high temperature for T&lt;sub&gt;2&lt;/sub&gt; and T&lt;sub&gt;3&lt;/sub&gt; families. A, Germination of 23 Cobham Green T&lt;sub&gt;2&lt;/sub&gt; families germinated at 32&#xB0;C; B, germination of 20 Salinas T&lt;sub&gt;2&lt;/sub&gt; families germinated at 35&#xB0;C. Germination is plotted against the percentage of edited reads in the bulk T&lt;sub&gt;2&lt;/sub&gt; family. Color coding refers to the T&lt;sub&gt;1&lt;/sub&gt; leaf genotype. C, D: Germination of T&lt;sub&gt;3&lt;/sub&gt; families at different temperatures for Cobham Green (C, mean of 6 T&lt;sub&gt;3&lt;/sub&gt; lines) and Salinas (D, mean of 6 T&lt;sub&gt;3&lt;/sub&gt; lines) compared to wildtype (mean of 3 lines from different years for each cultivar).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1513/F2.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/5/1513/F2.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489760" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Germination at high temperature for T<sub>2</sub> and T<sub>3</sub> families. A, Germination of 23 Cobham Green T<sub>2</sub> families germinated at 32°C; B, germination of 20 Salinas T<sub>2</sub> families germinated at 35°C. Germination is plotted against the percentage of edited reads in the bulk T<sub>2</sub> family. Color coding refers to the T<sub>1</sub> leaf genotype. C, D: Germination of T<sub>3</sub> families at different temperatures for Cobham Green (C, mean of 6 T<sub>3</sub> lines) and Salinas (D, mean of 6 T<sub>3</sub> lines) compared to wildtype (mean of 3 lines from different years for each cultivar).</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">T<sub>2</sub> progeny derived from T<sub>1</sub> plants with different mutant profiles in their leaves exhibited a range of high temperature germination frequencies as illustrated in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="High temperature germination phenotype. Germination of 4 Salinas T2 families after 72 h at 35&#xB0;C. A: Sal_2_2, non-edited T1; B: Sal_2_7, bi-allelic T1; C: Sal_1_9, mono-allelic T1; D: Sal_3_6, mono-allelic T1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-219073739" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;High temperature germination phenotype. Germination of 4 Salinas T2 families after 72 h at 35&#xB0;C. A: Sal_2_2, non-edited T1; B: Sal_2_7, bi-allelic T1; C: Sal_1_9, mono-allelic T1; D: Sal_3_6, mono-allelic T1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1513/F3.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/5/1513/F3.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489642" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">High temperature germination phenotype. Germination of 4 Salinas T2 families after 72 h at 35°C. A: Sal_2_2, non-edited T1; B: Sal_2_7, bi-allelic T1; C: Sal_1_9, mono-allelic T1; D: Sal_3_6, mono-allelic T1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The consequence of editing <em>NCED4</em> was evaluated by plotting the germination frequency against total editing in each T<sub>2</sub> family (percentage of edited reads in amplicons bulked prior to sequencing or <em>in silico</em>; <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A-B</a>). For both cultivars, the highest germination frequency was detected for T<sub>2</sub> families with mono- and bi-allelic T<sub>1</sub> leaf genotypes, consistent with an inverse correlation between <em>NCED4</em> expression and germination at high temperature. Only one bi-allelic line germinated close to 100%. Three out of four T<sub>2</sub> progeny from plants with bi-allelic T<sub>1</sub> leaf genotypes had germination percentages of around 50%; genotypes of these individual T<sub>2</sub> plants revealed a 1:2:1 segregation (homozygous for allele 1: bi-allelic: homozygous for allele 2) indicating that the high temperature germination phenotype was not absolute and not all homozygous mutants germinated at high temperature (Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS1.tif">File S1</a>). Germination of the T<sub>2</sub> progeny with mono-allelic T<sub>1</sub> leaves varied between 0% and 73% (average 21%, n = 8). Germination of T<sub>2</sub> families with chimeric T<sub>1</sub> leaves ranged between 0% and 12% (average 4%, n = 7). None of the T<sub>2</sub> lines derived from a chimeric T<sub>1</sub> had a high germination percentage, indicating that T<sub>1</sub> plants with chimeric profiles contained mostly somatic mutations that were not inherited through the germline. Germination of T<sub>2</sub> progeny with non-edited T<sub>1</sub> leaves was low (average 2%, n = 14). For these T<sub>2</sub> families derived from non-edited T<sub>1</sub> plants (but with <em>Cas9</em>), substantial amounts of editing was observed in T<sub>2</sub> leaves (average 23%, n = 14), indicating that <em>Cas9</em> remains active and new mutations can arise in the <em>NCED4</em> gene in T<sub>2</sub> plants.</p><p id="p-29">T<sub>3</sub> progeny from T<sub>2</sub> plants that were homozygous for <em>NCED4</em> knockouts consistently showed a complete loss of thermoinhibition of germination regardless of whether the T<sub>2</sub> progenitor had germinated at high temperature in the previous generation, indicating that non-genetic factors influenced germination of T<sub>2</sub> individuals. T<sub>3</sub> seed was collected from 11 individual T<sub>2</sub> plants that were homozygous for the <em>NCED4</em> gene and either positive or negative for the <em>Cas9</em> gene. Germination tests were performed at 28, 35, and 37°. All homozygous <em>NCED4</em> mutants in both genotypes showed significantly higher germination percentages at 35 and at 37° Compared to wild type (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B-C</a>). At 37°, mutant lines in both genotypes had average germination percentages above 70%, while germination was completely inhibited in wild type plants.</p></div><div id="sec-12" class="subsection"><h3>Mutational events are not random but distinctive for each gRNA</h3><p id="p-30">Genotyping of individual T<sub>2</sub> plants revealed that the types of mutations were not random, but specific to the gRNA (Figures S9–S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS2.tif">File S2</a>). Since a mutant allele inherited through the germline should be represented by either 50% (heterozygous) or 100% (homozygous) of the reads in the amplicon, any low frequency alleles (&lt;50%) in leaves of T<sub>2</sub> plants represent new, somatic events due to continued <em>Cas9</em> expression. This is illustrated by the mutation profiles of 38 T<sub>2</sub> progeny of line Sal_1_9 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The T<sub>1</sub> plant containing gRNA1 had a single 11 bp deletion in the young leaf that represented 46% of the reads in the amplicon. Segregation in the T<sub>2</sub> progeny indicated that this was a mono-allelic germline mutation (9/38 were homozygous wild types, 9/38 were homozygous mutants and 20/38 were heterozygotes). The <em>Cas9</em> gene was still present in all but two of the 38 T<sub>2</sub> plants, one of which was a homozygous mutant. Therefore new alleles could have arisen in 28/38 plants. In all of these 28 T<sub>2</sub> plants, the same events (a 1 bp C and a 1 bp A insertion) that were detected most frequently in the 17 other T<sub>1</sub> plants for gRNA 1, occurred independently. Similarly, for 16 T<sub>2</sub> progeny of T<sub>1</sub> plant Sal_3_8 containing gRNA 3, which had no detectable mutations in the T<sub>1</sub> leaf (Figure S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS2.tif">File S2</a>), the same mutations (a 1 bp C, A or T insertion) arose independently in multiple T<sub>2</sub> individuals. Moreover, for gRNA 3, mutation patterns seem to be especially reproducible in T<sub>2</sub> sibs (Figures S12 and S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1/FileS2.tif">File S2</a>). This repeatability indicates that the outcomes of NHEJ repair are not random but influenced by the gRNA sequence.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutation profiles of 38 T2 progeny of Sal_1_9. Each row represents an allele, with the reference allele on top. The 23bp gRNA sequence is circled with a black box, with the PAM circled by the small black box. The predicted Cas9 cut site is indicated with a black line. Each column represents the sequence read distribution in a young leaf of a single T2 plant. A wildtype control is shown on the left, followed by the mutation profile of Sal_1_9, the T1 mother plant. Plant abbreviation is Genotype_gRNA_plant. Genotypes: CG = Cobham Green; Sal = Salinas." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-219073739" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutation profiles of 38 T&lt;sub&gt;2&lt;/sub&gt; progeny of Sal_1_9. Each row represents an allele, with the reference allele on top. The 23bp gRNA sequence is circled with a black box, with the PAM circled by the small black box. The predicted &lt;em&gt;Cas9&lt;/em&gt; cut site is indicated with a black line. Each column represents the sequence read distribution in a young leaf of a single T&lt;sub&gt;2&lt;/sub&gt; plant. A wildtype control is shown on the left, followed by the mutation profile of Sal_1_9, the T&lt;sub&gt;1&lt;/sub&gt; mother plant. Plant abbreviation is Genotype_gRNA_plant. Genotypes: CG = Cobham Green; Sal = Salinas.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/5/1513/F4.medium.gif" width="440" height="82"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/5/1513/F4.medium.gif" width="440" height="82"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/5/1513/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/489737" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Mutation profiles of 38 T<sub>2</sub> progeny of Sal_1_9. Each row represents an allele, with the reference allele on top. The 23bp gRNA sequence is circled with a black box, with the PAM circled by the small black box. The predicted <em>Cas9</em> cut site is indicated with a black line. Each column represents the sequence read distribution in a young leaf of a single T<sub>2</sub> plant. A wildtype control is shown on the left, followed by the mutation profile of Sal_1_9, the T<sub>1</sub> mother plant. Plant abbreviation is Genotype_gRNA_plant. Genotypes: CG = Cobham Green; Sal = Salinas.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-32">Multiple strategies for inducing CRISPR/Cas9-mediated mutations in plants have been reported (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schiml and Puchta 2016</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Puchta 2017</a>). Most commonly, Cas9 and one or more gRNAs have been combined in a single expression construct, which was delivered by <em>A. tumefaciens</em> into embryogenic callus or somatic explants. Different parameters can affect the mutation efficiency, including Cas9 codon optimization (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Ma <em>et al.</em> 2016</a>), promoters and terminators controlling expression of Cas9 and gRNA (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bortesi and Fischer 2014</a>), transformation procedure (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altpeter <em>et al.</em> 2016</a>), callus culture time (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mikami <em>et al.</em> 2015</a>), gRNA protospacer sequence (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Doench <em>et al.</em> 2016</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Haeussler <em>et al.</em> 2016</a>), and possibly other factors.</p><p id="p-33">We obtained efficient germline editing in lettuce using Cas9 optimized for expression in <em>A. thaliana</em> expressed from the parsley ubiquitin (PcUbi) promoter and a single gRNA driven by the pol III AtU6-26 promoter that was mediated by cocultivation of <em>A. tumefaciens</em> with cotyledon explants. Although low frequencies of editing were detected in callus samples analyzed three weeks post <em>Agrobacterium</em> cocultivation, amplicon sequencing of young T<sub>1</sub> leaves revealed 13 independent mono-allelic and four bi-allelic plants out of 47 Cas9-positive regenerants. At the time of sampling, callus had been under selection for antibiotic resistance for 18 days. The early sampling time could have been a factor in the low frequency of editing because there can still be some non-transgenic callus this early in selection (D. Tricoli, personal communication). The plants are likely exhibiting more editing than the callus because of the continual selection for the presence of the T-DNA over the 4 to 5 month period. Amplicon sequencing of 368 T<sub>2</sub> plants derived from 23 T<sub>1</sub> lines provided a high-resolution analysis of mutation profiles and inheritance patterns. Individual genotypes of T<sub>2</sub> plants derived from the mono- and bi-allelic T<sub>1</sub> plants showed clear segregation of the mutated alleles, indicating that the mutations detected in the T<sub>1</sub> leaves had been inherited through the germline. Average germination at high temperature was highest for the T<sub>2</sub> families derived from bi-allelic and mono-allelic T<sub>1</sub> plants. None of the 11 T<sub>2</sub> families derived from chimeric T<sub>1</sub> plants showed high germination at high temperature. Although average germination in seeds of T<sub>2</sub> families derived from non-edited T<sub>1</sub> plants was low, substantial amounts of editing were detected in the young T<sub>2</sub> leaves, indicating that there had been a burst of Cas9 activity in the early growth stages of the T<sub>2</sub> seedlings. This could be due to increased promoter activity in young tissue or to the high temperature germination treatment itself because Cas9 was recently reported to work more efficiently in plants during heat stress (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Le Blanc <em>et al.</em> 2017</a>). In combination, these results indicate that Cas9 activity and NHEJ was strong during early regeneration of explants or early development of seedlings but weaker in callus and in established leaf tissue.</p><p id="p-34">Our data also provide insights into mutational outcomes of NHEJ in lettuce. The most common mutations were 1 bp insertions that could be any base pair. In addition to insertions, deletions were detected for all three gRNAs, but were more prevalent with gRNA 1, which is likely due to 6 bp of microhomology (CCAACCN<sub>11</sub>CCAACC) in the protospacer region. When microhomologies are present in the vicinity of the DSB, MMEJ is often the dominant mechanism for DSB repair (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rodgers and Mcvey 2016</a>). Repair outcomes were not random, but rather were consistent for each gRNA. Mutation signatures from sustained somatic Cas9 activity seemed to be particularly reproducible in T<sub>2</sub> sibs. In mammalian cell lines, repair outcomes after CRISPR/Cas9-induced double strand breaks were also not random, but dependent on the gRNA, and independent of genomic location and cell line (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Van Overbeek <em>et al.</em> 2016</a>). Similarly, independent hairy roots and somatic embryos of soybean had consistent mutation signatures with the same gRNA. These data are indicative of an unknown, controlled mechanism governing the types of mutations that are favored at a given target (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Jacobs <em>et al.</em> 2015</a>).</p><p id="p-35">Off-target activity was not assessed in this study; however, care was taken to avoid off-target activity in the three other <em>NCED</em> paralogs by selecting gRNAs that differed in the PAM regions in the parologous genes. Off-target activity elsewhere in the genome is a possibility, although no off-target matches were found that had complete homology in the seed region (9bp upstream of the PAM). Somaclonal variation in any plant that has gone through tissue culture will likely be higher than off-target effects. Backcrossing would remove both somaclonal variation and any off-target effects.</p><p id="p-36">Our phenotypic and genotypic data indicate that the <em>NCED4</em> gene can be used as a selectable marker for germline editing events. Although efficiency of mutagenesis has been shown to be high in multiple species, many mutations in the T<sub>1</sub> plant are somatic, making selection of the desired inheritable mutants a burdensome process. The most desirable outcome is a bi-allelic germline knockout mutation in the primary transformant. Selfing will then give rise to homozygous or bi-allelic T<sub>2</sub> mutated progeny with a 3:1 ratio of <em>Cas9</em>-positive <em>vs.</em> negative segregants. Mono-allelic T<sub>1</sub> germline mutants will give rise to Cas9-free, homozygous mutants at a ratio of 15:1, assuming no linkage between the T-DNA and the target gene. High frequencies of co-editing have been reported in <em>C. elegans</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Arribere <em>et al.</em> 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mouridi <em>et al.</em> 2017</a>). Therefore, the occurrence of germline editing of the <em>NCED4</em> gene is expected to be correlated with germline edits of an unrelated target gene. Consequently, selecting for edits of the <em>NCED4</em> gene by germinating seeds at high temperature may enrich for desired edits in other genes that are more difficult to screen for phenotypically.</p><p id="p-37">Germination thermotolerance due to inactivation of <em>NCED4</em> provides a useful whole-plant selectable phenotype that has little if any pleotropic effects on growth or stress tolerance (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Huo <em>et al.</em> 2013</a>). However, the level of thermotolerance can vary between genotypes and among seed lots of the same genotype. Germination of seeds of wildtype cv. Cobham Green, a butterhead cultivar, was much more thermosensitive than seeds of cv. Salinas, a crisphead cultivar. Therefore, it will be necessary to determine the optimal temperature for selection of editing events in <em>NCED4</em> for each genotype. In addition, lines of the same genotype that are grown under different conditions can have different sensitivities to temperature (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sung <em>et al.</em> 1998</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kozarewa <em>et al.</em> 2006</a>). Screening efficiency may be enhanced by maturing seeds at lower temperatures to maximize high temperature sensitivity in non-edited seeds. Furthermore, fresh seed (germinated immediately after seed maturation) was more sensitive to high temperature than seed that had been stored for a few months (data not shown).</p><p id="p-38">Until recently, cloning of causal genes for agriculturally important traits has been a lower priority for crop improvement than obtaining genetically linked markers. Once a gene had been mapped to a sufficiently small genomic region, closely linked markers have been adequate for marker-assisted selection to introgress the gene into elite genotypes in breeding programs. However, precise introgressions or alterations in existing alleles using genome editing depend upon the availability of cloned causal genes. Our data show that CRISPR/Cas9 is an efficient tool to aid in gene identification in lettuce by creating single-gene heritable knockouts in one generation. <em>LsNCED4</em> could also be used in a co-editing strategy aimed to simplify selection of mutations in candidate genes with more complex phenotypes.</p><p id="p-39">In conclusion, we have demonstrated the utility of the CRISPR/Cas9 system for generating gene knock-outs in lettuce. Experiments to identify candidate genes for resistance to downy mildew caused by <em>Bremia lactucae</em> and a bacterial root rot caused by <em>Rhizomonas suberifaciens</em> using the <em>NCED4</em> co-editing strategy are underway. In addition, we are extending the system to achieve allele editing, gene replacements and gene stacking using the insights gained in this study.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We thank Daniel E. Ellison and Paola E. Perez for technical assistance, Pauline Y. Sanders for greenhouse support, the UC Davis DNA Technology Core for sequencing, and the UC Davis Parsons Plant Transformation Facility for generating the lettuce transgenics. This work was supported by the National Research Initiative (NRI) Specialty Crops Research Initiative (SCRI) of the USDA Cooperative State Research, Education and Extension Service (CSREES) award # 2015-51181-24283.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-41"><em>Communicating editor: N. Taylor</em></p></li><li class="fn" id="fn-3"><p id="p-42">Supplemental Material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300396/-/DC1</a>.</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-10-29"><span class="received-label">Received </span>October 29, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-02-25"><span class="accepted-label">Accepted </span>February 25, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Bertier <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1513.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Acquaah</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Principles of plant genetics and breeding</span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.2" data-doi="10.1105/tpc.16.00196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altpeter</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartley</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blechl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brutnell</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Advancing Crop Transformation in the Era of Genome Editing.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1510</span>–<span class="cit-lpage">1520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.16.00196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.16.00196%26rft_id%253Dinfo%253Apmid%252F27335450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI4LzcvMTUxMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzgvNS8xNTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-8.5.1513.3"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Anonymous, Crop Values</span>: <span class="cit-pub-date">2014</span> <span class="cit-comment">Summary. USDA</span>, <a href="http://usda.mannlib.cornell.edu/MannUsda/homepage.do">http:// usda.mannlib.cornell.edu/MannUsda/homepage.do</a> (2015). 11.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.4" data-doi="10.1534/genetics.114.169730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arribere</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.169730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169730%26rft_id%253Dinfo%253Apmid%252F25161212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvODM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE1MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-8.5.1513.5" data-doi="10.1016/j.biotechadv.2014.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bortesi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The CRISPR/Cas9 system for plant genome editing and beyond</span>. <span class="cit-source">Biotech. Adv</span>. <span class="cit-vol">33</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">41</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.biotechadv.2014.12.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotech.%2BAdv%26rft.volume%253D33%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biotechadv.2014.12.006%26rft_id%253Dinfo%253Apmid%252F25536441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biotechadv.2014.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25536441&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.6" data-doi="10.1038/nbt.3437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doench</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fusi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullender</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaimberg</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3437</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.5.1513.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engler</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grogan</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Variation in lettuce plants regenerated from protoplasts.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">426</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DEngler%26rft.auinit1%253DD.%2BE.%26rft.volume%253D75%26rft.issue%253D6%26rft.spage%253D426%26rft.epage%253D430%26rft.atitle%253DVariation%2Bin%2Blettuce%2Bplants%2Bregenerated%2Bfrom%2Bprotoplasts%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.8" data-doi="10.1111/tpj.12554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fauser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiml</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Both CRISPR/Cas-based nucleases and nickases can be used efficiently for genome engineering in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.12554</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D79%26rft.spage%253D348%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12554%26rft_id%253Dinfo%253Apmid%252F24836556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24836556&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339439200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.5.1513.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Funk</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-comment">Everywhere but Antarctica: Using a supertree to understand the diversity and distribution of the Compositae, in Plant Diversity and Complexity Patterns: Local, Regional and Global Dimensions: Proceedings of an International Symposium Held at the Royal Danish Academy of Sciences and Letters in Copenhagen, Denmark, 25–28 May, 2003. Vol. 55. Kgl. Danske Videnskabernes</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.10" data-doi="10.1007/s11103-014-0263-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPR/Cas9-mediated targeted mutagenesis in Nicotiana tabacum.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11103-014-0263-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%26rft.volume%253D87%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-014-0263-0%26rft_id%253Dinfo%253Apmid%252F25344637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-014-0263-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25344637&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.11" data-doi="10.1146/annurev-phyto-080508-081852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelvin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant proteins involved in Agrobacterium-mediated genetic transformation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">68</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-phyto-080508-081852</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DGelvin%26rft.auinit1%253DS.%2BB.%26rft.volume%253D48%26rft.spage%253D45%26rft.epage%253D68%26rft.atitle%253DPlant%2Bproteins%2Binvolved%2Bin%2BAgrobacterium-mediated%2Bgenetic%2Btransformation.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-phyto-080508-081852%26rft_id%253Dinfo%253Apmid%252F20337518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-phyto-080508-081852&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20337518&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282062400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.12" data-doi="10.3389/fpls.2012.00052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gelvin</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Traversing the cell: Agrobacterium T-DNA’s journey to the host genome.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2012.00052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BPlant%2BSci%26rft.aulast%253DGelvin%26rft.auinit1%253DS.%2BB.%26rft.volume%253D3%26rft.spage%253D52%26rft.epage%253D52%26rft.atitle%253DTraversing%2Bthe%2BCell%253A%2BAgrobacterium%2BT-DNA%2527s%2BJourney%2Bto%2Bthe%2BHost%2BGenome.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2012.00052%26rft_id%253Dinfo%253Apmid%252F22645590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2012.00052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22645590&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.13" data-doi="10.1186/s13059-016-1012-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haeussler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönig</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eschstruth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mianné</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-1012-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1012-2%26rft_id%253Dinfo%253Apmid%252F27380939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1012-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27380939&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.14" data-doi="10.1105/tpc.112.108902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunusoth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Expression of 9-cis-EPOXYCAROTENOID DIOXYGENASE4 is essential for thermoinhibition of lettuce seed germination but not for seed development or stress tolerance.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">884</span>–<span class="cit-lpage">900</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.112.108902</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.108902%26rft_id%253Dinfo%253Apmid%252F23503626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvODg0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOC81LzE1MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.15" data-doi="10.1186/s12896-015-0131-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LaFayette</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrott</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-comment">Targeted genome modifications in soybean with CRISPR/Cas9. BMC Biotechnol</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12896-015-0131-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D15%26rft.spage%253D16%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12896-015-0131-2%26rft_id%253Dinfo%253Apmid%252F25879861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12896-015-0131-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25879861&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-8.5.1513.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-comment">The Compositae: systematically fascinating but specifically neglected, pp. 179–191 in <em>Genome Mapping of Plants</em>, edited by A. H. Paterson and R. G. Landes, Georgetown, TX.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Highly Efficient RNA-guide genome editing in human cells via delivery of purified Cas9 ribonucleoproteins.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft.volume%253D128%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.18" data-doi="10.1534/genetics.114.166389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishidate</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghanta</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Co-CRISPR strategy for efficient genome editing in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1080</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.166389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166389%26rft_id%253Dinfo%253Apmid%252F24879462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzEwNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTUxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.19" data-doi="10.1038/ncomms14406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">B.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.-S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR/Cpf1-mediated DNA-free plant genome editing.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">14406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms14406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D14406%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms14406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozarewa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantliffe</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffella</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-comment">High Maturation Temperature of Lettuce Seeds during Development Increased Ethylene Production and Germination at Elevated Temperatures. Science 131: 564–570</span>. <a href="http://agris.fao.org/agris-search/search.do?recordID=US201301097705">http://agris.fao.org/agris-search/search.do?recordID=US201301097705</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.21" data-doi="10.1186/s13007-016-0111-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barone</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Gene targeting and transgene stacking using intra genomic homologous recombination in plants.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13007-016-0111-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D12%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13007-016-0111-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13007-016-0111-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.22" data-doi="10.1111/tpj.13782"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Blanc</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lozano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatpar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Increased efficiency of targeted mutagenesis by CRISPR/Cas9 in plants using heat stress.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13782</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D9%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.23" data-doi="10.1038/nbt.3628"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biyong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hess</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CrispRVariants charts the mutation spectrum of genome engineering experiments.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">703</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3628</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D701%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3628&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.24" data-doi="10.1016/j.molp.2016.04.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CRISPR/Cas9 platforms for genome editing in plants: developments and applications.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molp.2016.04.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D9%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molp.2016.04.009%26rft_id%253Dinfo%253Apmid%252F27108381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molp.2016.04.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27108381&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.25" data-doi="10.3389/fpls.2016.01904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malnoy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viola</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koo</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">DNA-free genetically edited grapevine and apple protoplast using CRISPR/Cas9 ribonucleoproteins.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2016.01904</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D7%26rft.spage%253D1904%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2016.01904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2016.01904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coaker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bart</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beattie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bent</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-comment">Foundational and translational research opportunities to improve plant health. Mol. Plant-Microbe Interact. MPMI-01–17–0010-CR</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/MPMI-01-17-0010-CR</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.27" data-doi="10.1007/BF00272777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seely</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Transformation of lettuce (Lactuca sativa) mediated by Agrobacterium tumefaciens.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00272777</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D6%26rft.spage%253D439%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00272777%26rft_id%253Dinfo%253Apmid%252F24248927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00272777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24248927&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987L335900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.28" data-doi="10.1007/s00299-015-1826-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mikami</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endo</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Parameters affecting frequency of CRISPR / Cas9 mediated targeted mutagenesis in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell Rep.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1807</span>–<span class="cit-lpage">1815</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00299-015-1826-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BRep.%26rft.volume%253D34%26rft.spage%253D1807%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00299-015-1826-5%26rft_id%253Dinfo%253Apmid%252F26134856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00299-015-1826-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26134856&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.29" data-doi="10.1155/2011/723518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mou</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutations in lettuce improvement.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2011</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1155/2011/723518</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BGenomics%26rft.volume%253D2011%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2011%252F723518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2011/723518&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mouridi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lecroisey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tardy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leclercq-blondel</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-comment">Reliable CRISPR/Cas9 genome engineering in C.Elegans using a single efficient sgRNA and an easily selectable phenotype. G3 (Bethesda) 7: 1429–1437</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.040824</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.31" data-doi="10.1016/j.tibtech.2013.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Podevin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nogué</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casacuberta</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Site-directed nucleases: A paradigm shift in predictable, knowledge-based plant breeding.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">383</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibtech.2013.03.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D375%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2013.03.004%26rft_id%253Dinfo%253Apmid%252F23601269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2013.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23601269&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.32" data-doi="10.1016/j.pbi.2016.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Applying CRISPR/Cas for genome engineering in plants: the best is yet to come.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2016.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D36%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2016.11.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2016.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.33" data-doi="10.1038/ncomms14953"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Chin-Wo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arikit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome assembly with in vitro proximity ligation data and whole-genome triplication in lettuce.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">14953</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms14953</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D14953%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms14953&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.34" data-doi="10.1002/jcp.25053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodgers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcvey</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Error-prone repair of DNA double-strand breaks.</span> <abbr class="cit-jnl-abbrev">J. Cell. Physiol.</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jcp.25053</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell.%2BPhysiol.%26rft.volume%253D231%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcp.25053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcp.25053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.35" data-doi="10.1186/s13007-016-0103-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiml</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Revolutionizing plant biology: multiple ways of genome engineering by CRISPR/Cas.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13007-016-0103-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D12%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13007-016-0103-0%26rft_id%253Dinfo%253Apmid%252F26823677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13007-016-0103-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26823677&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gulati</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ezran</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Travaglini</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Index Switching Causes “Spreading-Of-Signal” Among Multiplexed Samples In Illumina HiSeq 4000 DNA Sequencing.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">125724</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantliffe</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagata</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Seed developmental temperature regulation of thermotolerance in lettuce.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D123%26rft.spage%253D700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.38" data-doi="10.1007/s00122-007-0599-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antonise</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavelle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochoa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A high-density, integrated genetic linkage map of lettuce (Lactuca spp.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTruco%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D115%26rft.issue%253D6%26rft.spage%253D735%26rft.epage%253D746%26rft.atitle%253DA%2Bhigh-density%252C%2Bintegrated%2Bgenetic%2Blinkage%2Bmap%2Bof%2Blettuce%2B%2528Lactuca%2Bspp.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0599-9%26rft_id%253Dinfo%253Apmid%252F17828385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0599-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828385&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249635000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.39" data-doi="10.1534/g3.112.004929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An Ultra High-Density, Transcript-Based, Genetic Map of Lettuce.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004929</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004929%26rft_id%253Dinfo%253Apmid%252F23550116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvNC82MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy84LzUvMTUxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.40" data-doi="10.1093/pcp/pcw191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsutsui</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higashiyama</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">PKAMA-ITACHI vectors for highly efficient CRISPR/Cas9-mediated gene knockout in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/pcp/pcw191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcw191%26rft_id%253Dinfo%253Apmid%252F27856772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcw191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27856772&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.41" data-doi="10.1016/j.molcel.2016.06.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Overbeek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capurso</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"> Article DNA Repair Profiling Reveals Nonrandom Outcomes at Cas9-Mediated Breaks.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">633</span>–<span class="cit-lpage">646</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2016.06.037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D63%26rft.spage%253D633%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2016.06.037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2016.06.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.42" data-doi="10.1146/annurev-arplant-042811-105552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voytas</span>  <span class="cit-name-given-names">D. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Plant genome engineering with sequence-specific nucleases.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-arplant-042811-105552</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft.volume%253D64%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-042811-105552%26rft_id%253Dinfo%253Apmid%252F23451779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-042811-105552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23451779&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321699500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.43" data-doi="10.1038/nbt.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Il Kwon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corvalán</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA-free genome editing in plants with preassembled CRISPR-Cas9 ribonucleoproteins.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D1162%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3389%26rft_id%253Dinfo%253Apmid%252F26479191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26479191&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F5%2F1513.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.44" data-doi="10.1038/srep23890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An alternative strategy for targeted gene replacement in plants using a dual-sgRNA/Cas9 design.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">23890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep23890</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D23890%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep23890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep23890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.5.1513.45" data-doi="10.1016/j.tplants.2016.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bortesi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baysal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Twyman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Characteristics of Genome Editing Mutations in Cereal Crops.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2016.08.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2016.08.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2016.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/5/1513.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/5/1497" title="A Flow Cytometry-Based Phenotypic Screen To Identify Novel Endocytic Factors in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/5/1523" title="Identification of General Patterns of Sex-Biased Expression in Daphnia, a Genus with Environmental Sex Determination" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/5"><span>Volume 8 Issue 5, May 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/5.cover-source.jpg?itok=Bl7MXdvV" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/5/1513" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ApYfY-9qpHc1ccQeruZVXmPIEfbnM1g30VOn_JCgq5U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/5/1513" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/5/1513" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/5/1513" class="active" data-icon-position="" data-hide-link-title="0">High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of NCED4 in Lettuce (Lactuca sativa)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/489462" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-fOQjIrzYiP68i5jo0aus6H7JjVVewCmrXSJfAVsJmcQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1513.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/5/1513" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0nJPJ61kkPRe3M-YVUoHuUshuobSIDI8kXOJGAL-mC8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/5/1513" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/5/1513.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489462" id="node489462--2" data-pisa="ggg;8/5/1513" data-pisa-master="ggg;g3.117.300396" data-apath="/ggg/8/5/1513.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of <em>NCED4</em> in Lettuce (<em>Lactuca sativa</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lien D. Bertier</span>, <span class="highwire-citation-author" data-delta="1">Mily Ron</span>, <span class="highwire-citation-author" data-delta="2">Heqiang Huo</span>, <span class="highwire-citation-author" data-delta="3">Kent J. Bradford</span>, <span class="highwire-citation-author" data-delta="4">Anne B. Britt</span> and <span class="highwire-citation-author" data-delta="5">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1513-1521;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300396 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489462--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lien D. Bertier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lien%2BD.%2BBertier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertier%20LD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALien%2BD.%2BBertier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mily Ron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mily%2BRon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMily%2BRon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Heqiang Huo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, Seed Biotechnology Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heqiang%2BHuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeqiang%2BHuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kent J. Bradford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, Seed Biotechnology Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BJ.%2BBradford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bradford%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BJ.%2BBradford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Anne B. Britt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BB.%2BBritt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Britt%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BB.%2BBritt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Plant Sciences, Molecular & Cellular Biology, Medical Microbiology & Immunology, University of California, Davis, CA 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/5/1513" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/5/1513&amp;title=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/5/1513&amp;title=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/5/1513&amp;title=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/5/1513&amp;text=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/5/1513&amp;title=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/5/1513&amp;t=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/5/1513&amp;title=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/5/1513&amp;title=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/489462/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="489462" id="node489462--3" data-pisa="ggg;8/5/1513" data-pisa-master="ggg;g3.117.300396" data-apath="/ggg/8/5/1513.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Resolution Analysis of the Efficiency, Heritability, and Editing Outcomes of CRISPR/Cas9-Induced Modifications of <em>NCED4</em> in Lettuce (<em>Lactuca sativa</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lien D. Bertier</span>, <span class="highwire-citation-author" data-delta="1">Mily Ron</span>, <span class="highwire-citation-author" data-delta="2">Heqiang Huo</span>, <span class="highwire-citation-author" data-delta="3">Kent J. Bradford</span>, <span class="highwire-citation-author" data-delta="4">Anne B. Britt</span> and <span class="highwire-citation-author" data-delta="5">Richard W. Michelmore</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1513-1521;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300396 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node489462--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lien D. Bertier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lien%2BD.%2BBertier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bertier%20LD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALien%2BD.%2BBertier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mily Ron </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mily%2BRon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ron%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMily%2BRon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Heqiang Huo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, Seed Biotechnology Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Heqiang%2BHuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huo%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHeqiang%2BHuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kent J. Bradford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Sciences, Seed Biotechnology Center</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BJ.%2BBradford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bradford%20KJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BJ.%2BBradford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Anne B. Britt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BB.%2BBritt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Britt%20AB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BB.%2BBritt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard W. Michelmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departments of Plant Sciences, Molecular & Cellular Biology, Medical Microbiology & Immunology, University of California, Davis, CA 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BW.%2BMichelmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelmore%20RW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARichard%2BW.%2BMichelmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rwmichelmore@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">rwmichelmore@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1513&amp;rft.epage=1521&amp;rft.atitle=High-Resolution%20Analysis%20of%20the%20Efficiency%2C%20Heritability%2C%20and%20Editing%20Outcomes%20of%20CRISPR/Cas9-Induced%20Modifications%20of%20NCED4%20in%20Lettuce%20%28Lactuca%20sativa%29&amp;rft.volume=8&amp;rft.issue=5&amp;rft.date=2018-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bertier%2C+Lien+D.&amp;rft.au=Ron%2C+Mily&amp;rft.au=Huo%2C+Heqiang&amp;rft.au=Bradford%2C+Kent+J.&amp;rft.au=Britt%2C+Anne+B.&amp;rft.au=Michelmore%2C+Richard+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/489462/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/489462/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/489462/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/489462/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/489462/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/489462/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/489462/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/489462/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/489462/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/489462/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/489462/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/489462/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/5/1513" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/5/1513.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/5/1513.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/5/1513.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/5/1513.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
