<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/6/1921" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/6/1921.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/6/1921.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/6/1921.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/6/1921/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/6/1921.atom" />
<meta name="HW.pisa" content="ggg;9/6/1921" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA" />
<meta name="DC.Identifier" content="10.1534/g3.119.400094" />
<meta name="DC.Date" content="2019-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Poudel et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The lowland ecotype of switchgrass has generated considerable interest because of its higher biomass yield and late flowering characteristics compared to the upland ecotype. However, lowland ecotypes planted in northern latitudes exhibit very low winter survival. Implementation of genomic selection could potentially enhance switchgrass breeding for winter survival by reducing generation time while eliminating the dependence on weather. The objectives of this study were to assess the potential of genomic selection for winter survival in lowland switchgrass by combining multiple populations in the training set and applying the selected model in two independent testing datasets for validation. Marker data were generated using exome capture sequencing. Validation was conducted using (1) indirect indicators of winter adaptation based on geographic and climatic variables of accessions from different source locations and (2) winter survival estimates of the phenotype. The prediction accuracies were significantly higher when the training dataset comprising all populations was used in fivefold cross validation but its application was not useful in the independent validation dataset. Nevertheless, modeling for population heterogeneity improved the prediction accuracy to some extent but the genetic relationship between the training and validation populations was found to be more influential. The predicted winter survival of lowland switchgrass indicated latitudinal and longitudinal variability, with the northeast USA the region for most cold tolerant lowland populations. Our results suggested that GS could provide valuable opportunities for improving winter survival and accelerate the lowland switchgrass breeding programs toward the development of cold tolerant cultivars suitable for northern latitudes." />
<meta name="DC.Contributor" content="Hari P. Poudel" />
<meta name="DC.Contributor" content="Millicent D. Sanciangco" />
<meta name="DC.Contributor" content="Shawn M. Kaeppler" />
<meta name="DC.Contributor" content="C. Robin Buell" />
<meta name="DC.Contributor" content="Michael D. Casler" />
<meta name="article:published_time" content="2019-06-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The lowland ecotype of switchgrass has generated considerable interest because of its higher biomass yield and late flowering characteristics compared to the upland ecotype. However, lowland ecotypes planted in northern latitudes exhibit very low winter survival. Implementation of genomic selection could potentially enhance switchgrass breeding for winter survival by reducing generation time while eliminating the dependence on weather. The objectives of this study were to assess the potential of genomic selection for winter survival in lowland switchgrass by combining multiple populations in the training set and applying the selected model in two independent testing datasets for validation. Marker data were generated using exome capture sequencing. Validation was conducted using (1) indirect indicators of winter adaptation based on geographic and climatic variables of accessions from different source locations and (2) winter survival estimates of the phenotype. The prediction accuracies were significantly higher when the training dataset comprising all populations was used in fivefold cross validation but its application was not useful in the independent validation dataset. Nevertheless, modeling for population heterogeneity improved the prediction accuracy to some extent but the genetic relationship between the training and validation populations was found to be more influential. The predicted winter survival of lowland switchgrass indicated latitudinal and longitudinal variability, with the northeast USA the region for most cold tolerant lowland populations. Our results suggested that GS could provide valuable opportunities for improving winter survival and accelerate the lowland switchgrass breeding programs toward the development of cold tolerant cultivars suitable for northern latitudes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/06/01" />
<meta name="citation_mjid" content="ggg;9/6/1921" />
<meta name="citation_id" content="9/6/1921" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/6/1921" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/6/1921.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/6/1921.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/6/1921.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400094" />
<meta name="citation_pmid" content="30971392" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1921" />
<meta name="citation_lastpage" content="1931" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hari P. Poudel" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706" />
<meta name="citation_author_email" content="hpoudel@wisc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9427-0153" />
<meta name="citation_author" content="Millicent D. Sanciangco" />
<meta name="citation_author_institution" content="Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author_institution" content="Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9472-3167" />
<meta name="citation_author" content="Shawn M. Kaeppler" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706" />
<meta name="citation_author" content="C. Robin Buell" />
<meta name="citation_author_institution" content="Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824" />
<meta name="citation_author" content="Michael D. Casler" />
<meta name="citation_author_institution" content="United States Dairy Forage Research Center, USDA-ARS, Madison, WI 53706" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Abyzov;citation_author=A. E. Urban;citation_author=M. Snyder;citation_author=M. Gerstein;citation_title=CNVnator: an approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing.;citation_volume=21;citation_year=2011;citation_pmid=21324876;citation_doi=10.1101/gr.114876.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=R. Austin;citation_author=C. Morgan;citation_author=M. Ford;citation_author=S. Bhagwat;citation_title=Flag Leaf Photosynthesis of Triticum aestivum and Related Diploid and Tetraploid Species;citation_pages=177-189;citation_volume=49;citation_year=1982;citation_issue=2;citation_doi=10.1093/aob/" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype× environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Calus;citation_author=A. De Roos;citation_author=R. Veerkamp;citation_title=Accuracy of Genomic Selection Using Different Methods to Define Haplotypes;citation_pages=553-561;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202394;citation_doi=10.1534/genetics.107.080838" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. Casler;citation_author=S. Sosa;citation_author=A. Boe;citation_author=S. Bonos;citation_title=Soil Quality and Region Influence Performance and Ranking of Switchgrass Genotypes.;citation_pages=221-232;citation_volume=59;citation_year=2019;citation_doi=10.2135/cropsci2018.06.0409" />
<meta name="citation_reference" content="citation_title=Switchgrass, Green Energy and Technology;citation_title=Switchgrass breeding, genetics, and genomics;citation_author=M. D. Casler;citation_pages=29-53;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=BioEnergy Res.;citation_author=M. D. Casler;citation_author=G. P. Ramstein;citation_title=Breeding for biomass yield in switchgrass using surrogate measures of yield.;citation_pages=1-12;citation_volume=11;citation_year=2018;citation_doi=10.1007/s12155-017-9867-y" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. D. Casler;citation_author=C. M. Tobias;citation_author=S. M. Kaeppler;citation_author=C. R. Buell;citation_author=Z.-Y. Wang;citation_title=The switchgrass genome: tools and strategies.;citation_pages=273-282;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.10.0026" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. D. Casler;citation_author=K. Vogel;citation_author=C. Taliaferro;citation_author=N. Ehlke;citation_author=J. Berdahl;citation_title=Latitudinal and Longitudinal Adaptation of Switchgrass Populations;citation_pages=2249-2260;citation_volume=47;citation_year=2007;citation_issue=6;citation_doi=10.2135/cropsci2006.12.0780" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. D. Casler;citation_author=K. P. Vogel;citation_author=D. Lee;citation_author=R. Mitchell;citation_author=P. Adler;citation_title=30 years of progress toward increased biomass yield of switchgrass and big bluestem.;citation_pages=1242-1254;citation_volume=58;citation_year=2018;citation_doi=10.2135/cropsci2017.12.0729" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. D. Casler;citation_author=K. P. Vogel;citation_author=C. M. Taliaferro;citation_author=R. L. Wynia;citation_title=Latitudinal Adaptation of Switchgrass Populations;citation_pages=293-303;citation_volume=44;citation_year=2004;citation_issue=1;citation_doi=10.2135/cropsci2004.2930" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=V. Soberanis;citation_author=S. Pérez-Elizalde;citation_author=P. Pérez-Rodríguez;citation_title=Genomic prediction of genotype× environment interaction kernel regression models.;citation_pages=1;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.03.0024" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. de Roos;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Reliability of genomic predictions across multiple populations.;citation_volume=183;citation_year=2009;citation_pmid=19822733;citation_doi=10.1534/genetics.109.104935" />
<meta name="citation_reference" content="citation_author=I. DeLacy;citation_author=K. Basford;citation_author=M. Cooper;citation_author=J. Bull;citation_author=C. McLaren;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_journal_abbrev=Planta;citation_author=A. L. Dhanaraj;citation_author=N. W. Alkharouf;citation_author=H. S. Beard;citation_author=I. B. Chouikha;citation_author=B. F. Matthews;citation_title=Major differences observed in transcript profiles of blueberry during cold acclimation under field and cold room conditions.;citation_pages=735-751;citation_volume=225;citation_year=2007;citation_issue=3;citation_pmid=16953429;citation_doi=10.1007/s00425-006-0382-1" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. Díaz-Uriarte;citation_author=S. A. De Andres;citation_title=Gene selection and classification of microarray data using random forest.;citation_pages=3-3;citation_volume=7;citation_year=2006;citation_pmid=16398926;citation_doi=10.1186/1471-2105-7-3" />
<meta name="citation_reference" content="citation_title=Genotype-by-environment interaction and plant breeding;citation_title=Beyond the analytical methodology-better interpretation and exploitation of genotype-by-environment interaction in breeding;citation_author=R. Eisemann;citation_author=M. Cooper;citation_author=D. Woodruff;citation_pages=108-117;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Evans;citation_author=M. D. Sanciangco;citation_author=K. H. Lau;citation_author=E. Crisovan;citation_author=K. Barry;citation_title=Extensive Genetic Diversity is Present within North American Switchgrass Germplasm.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.06.0055" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=M. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_issue=2;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=P. P. Grabowski;citation_author=J. Evans;citation_author=C. Daum;citation_author=S. Deshpande;citation_author=K. W. Barry;citation_title=Genome‐wide associations with flowering time in switchgrass using exome‐capture sequencing data.;citation_pages=154-169;citation_volume=213;citation_year=2017;citation_doi=10.1111/nph.14101" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Plant.;citation_author=L. V. Gusta;citation_author=M. Wisniewski;citation_title=Understanding plant cold hardiness: an opinion.;citation_pages=4-14;citation_volume=147;citation_year=2013;citation_doi=10.1111/j.1399-3054.2012.01611.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. A. Hopkins;citation_author=K. Vogel;citation_author=K. Moore;citation_author=K. Johnson;citation_author=I. Carlson;citation_title=Genotypic Variability and Genotype x Environment Interactions among Switchgrass Accessions from the Midwestern USA;citation_pages=565-571;citation_volume=35;citation_year=1995;citation_issue=2;citation_doi=10.2135/cropsci1995.0011183X003500020047x" />
<meta name="citation_reference" content="citation_author=T. Hothorn;citation_author=F. Bretz;citation_author=P. Westfall;citation_author=R. M. Heiberger;citation_author=A. Schuetzenmeister;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=M. C. Joseph;citation_author=D. D. Randall;citation_author=C. J. Nelson;citation_title=Photosynthesis in Polyploid Tall Fescue : II. PHOTOSYNTHESIS AND RIBULOSE-1, 5-BISPHOSPHATE CARBOXYLASE OF POLYPLOID TALL FESCUE;citation_pages=894-898;citation_volume=68;citation_year=1981;citation_issue=4;citation_pmid=16662021;citation_doi=10.1104/pp.68.4.894" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. Lado;citation_author=P. G. Barrios;citation_author=M. Quincke;citation_author=P. Silva;citation_author=L. Gutiérrez;citation_title=Modeling genotype× environment interaction for genomic selection with unbalanced data from a wheat breeding program.;citation_pages=2165-2179;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2015.04.0207" />
<meta name="citation_reference" content="citation_journal_title=Ind. Crops Prod.;citation_author=R. Lal;citation_title=Adaptability patterns and stable cultivar selection in menthol mint (Mentha arvensis L.).;citation_pages=176-181;citation_volume=50;citation_year=2013;citation_doi=10.1016/j.indcrop.2013.07.008" />
<meta name="citation_reference" content="citation_title=Engineering and Science of Biomass Feedstock Production and Provision;citation_title=Switchgrass and giant Miscanthus agronomyin;citation_author=D. K. Lee;citation_author=A. S. Parrish;citation_author=T. B. Voigt;citation_pages=37-59;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Legarra;citation_author=C. Robert-Granié;citation_author=E. Manfredi;citation_author=J.-M. Elsen;citation_title=Performance of genomic selection in mice.;citation_volume=180;citation_year=2008;citation_pmid=18757934;citation_doi=10.1534/genetics.108.088575" />
<meta name="citation_reference" content="Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint arXiv:1303.3997." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. E. Lipka;citation_author=F. Lu;citation_author=J. H. Cherney;citation_author=E. S. Buckler;citation_author=M. D. Casler;citation_title=Accelerating the switchgrass (Panicum virgatum L.) breeding cycle using genomic selection approaches.;citation_pages=e112227;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0112227" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. E. Lorenzana;citation_author=R. Bernardo;citation_title=Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.;citation_pages=151-161;citation_volume=120;citation_year=2009;citation_issue=1;citation_pmid=19841887;citation_doi=10.1007/s00122-009-1166-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. R. Martin;citation_author=D. Kinnamon;citation_author=M. A. Schmidt;citation_author=E. Powell;citation_author=S. Zuchner;citation_title=SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.;citation_volume=26;citation_year=2010;citation_pmid=20861027;citation_doi=10.1093/bioinformatics/btq526" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. Moser;citation_author=B. Tier;citation_author=R. E. Crump;citation_author=M. S. Khatkar;citation_author=H. W. Raadsma;citation_title=A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers.;citation_pages=56-56;citation_volume=41;citation_year=2009;citation_pmid=20043835;citation_doi=10.1186/1297-9686-41-56" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The bayesian lasso.;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=D. J. Parrish;citation_author=J. H. Fike;citation_title=The biology and agronomy of switchgrass for biofuels.;citation_pages=423-459;citation_volume=24;citation_year=2005;citation_doi=10.1080/07352680500316433" />
<meta name="citation_reference" content="citation_author=P. Pérez;citation_author=G. de los Campos;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=H. P. Poudel;citation_author=M. Sanciangco;citation_author=S. Kaeppler;citation_author=C. R. Buell;citation_author=M. Casler;citation_title=Quantitative Trait Loci for Freezing Tolerance in a Lowland x Upland Switchgrass Population.;citation_pages=372;citation_volume=10;citation_year=2019;citation_doi=10.3389/fpls.2019.00372" />
<meta name="citation_reference" content="citation_title=bioRxiv;citation_title=Extensions of blup models for genomic prediction in heterogeneous populations: Application in a diverse switchgrass sample.;citation_author=G. P. Ramstein;citation_author=M. Casler;citation_pages=124081;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. P. Ramstein;citation_author=J. Evans;citation_author=S. M. Kaeppler;citation_author=R. B. Mitchell;citation_author=K. P. Vogel;citation_title=Accuracy of genomic prediction in switchgrass (Panicum virgatum L.) improved by accounting for linkage disequilibrium. G3: Genes, Genomes;citation_pages=1049-1062;citation_volume=6;citation_year=2016;citation_doi=10.1534/g3.115.024950" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=T. Schulz-Streeck;citation_author=J. Ogutu;citation_author=Z. Karaman;citation_author=C. Knaak;citation_author=H. Piepho;citation_title=Genomic selection using multiple populations.;citation_pages=2453-2461;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2012.03.0160" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. M. Simeão Resende;citation_author=M. D. Casler;citation_author=M. D. Vilela de Resende;citation_title=Genomic selection in forage breeding: accuracy and methods.;citation_pages=143-156;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.05.0353" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Technow;citation_author=A. Bürger;citation_author=A. E. Melchinger;citation_title=Genomic prediction of northern corn leaf blight resistance in maize with combined or separated training sets for heterotic groups. G3: Genes, Genomes;citation_pages=197-203;citation_volume=3;citation_year=2013;citation_doi=10.1534/g3.112.004630" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=K. Vogel;citation_author=A. Hopkins;citation_author=K. Moore;citation_author=K. Johnson;citation_author=I. Carlson;citation_title=Winter Survival in Switchgrass Populations Bred for High IVDMD;citation_pages=1857-1862;citation_volume=42;citation_year=2002;citation_issue=6;citation_doi=10.2135/cropsci2002.1857" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Regist.;citation_author=K. P. Vogel;citation_author=R. Mitchell;citation_author=M. Casler;citation_author=G. J. J. P. R. Sarath;citation_title=Registration of ‘Liberty’switchgrass.;citation_pages=242-247;citation_volume=8;citation_year=2014;citation_doi=10.3198/jpr2013.12.0076crc" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=D. A. Warner;citation_author=M. S. Ku;citation_author=G. E. Edwards;citation_title=Photosynthesis, Leaf Anatomy, and Cellular Constituents in the Polyploid C4 Grass Panicum virgatum;citation_pages=461-466;citation_volume=84;citation_year=1987;citation_issue=2;citation_pmid=16665462;citation_doi=10.1104/pp.84.2.461" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. C. Wientjes;citation_author=R. F. Veerkamp;citation_author=M. P. Calus;citation_title=The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.;citation_volume=193;citation_year=2013;citation_pmid=23267052;citation_doi=10.1534/genetics.112.146290" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. G3: Genes, Genomes;citation_pages=1427-1436;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.003699" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Wong;citation_author=R. Bernardo;citation_title=Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.;citation_pages=815-824;citation_volume=116;citation_year=2008;citation_issue=6;citation_pmid=18219476;citation_doi=10.1007/s00122-008-0715-5" />
<meta name="citation_reference" content="citation_journal_title=Biomass Bioenergy;citation_author=L. Wright;citation_author=A. Turhollow;citation_title=Switchgrass selection as a “model” bioenergy crop: a history of the process.;citation_pages=851-868;citation_volume=34;citation_year=2010;citation_doi=10.1016/j.biombioe.2010.01.030" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=S. Wullschleger;citation_author=E. Davis;citation_author=M. Borsuk;citation_author=C. Gunderson;citation_author=L. Lynd;citation_title=Biomass Production in Switchgrass across the United States: Database Description and Determinants of Yield;citation_pages=1158-1168;citation_volume=102;citation_year=2010;citation_issue=4;citation_doi=10.2134/agronj2010.0087" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. Wullschleger;citation_author=M. Sanderson;citation_author=S. McLaughlin;citation_author=D. Biradar;citation_author=A. Rayburn;citation_title=Photosynthetic Rates and Ploidy Levels among Populations of Switchgrass;citation_pages=306-312;citation_volume=36;citation_year=1996;citation_issue=2;citation_doi=10.2135/cropsci1996.0011183X003600020016x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=T. Würschum;citation_author=J. C. Reif;citation_author=T. Kraft;citation_author=G. Janssen;citation_author=Y. Zhao;citation_title=Genomic selection in sugar beet breeding populations.;citation_pages=85;citation_volume=14;citation_year=2013;citation_pmid=24047500;citation_doi=10.1186/1471-2156-14-85" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. E. Zalapa;citation_author=D. L. Price;citation_author=S. M. Kaeppler;citation_author=C. M. Tobias;citation_author=M. Okada;citation_title=Hierarchical classification of switchgrass genotypes using SSR and chloroplast sequences: ecotypes, ploidies, gene pools, and cultivars.;citation_pages=805-817;citation_volume=122;citation_year=2011;citation_issue=4;citation_pmid=21104398;citation_doi=10.1007/s00122-010-1488-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Zhang;citation_author=J. Zalapa;citation_author=A. R. Jakubowski;citation_author=D. L. Price;citation_author=A. Acharya;citation_title=Natural hybrids and gene flow between upland and lowland switchgrass.;citation_pages=2626-2641;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.02.0104" />
<meta name="citation_reference" content="citation_journal_abbrev=Genetica;citation_author=Y. Zhang;citation_author=J. E. Zalapa;citation_author=A. R. Jakubowski;citation_author=D. L. Price;citation_author=A. Acharya;citation_title=Post-glacial evolution of Panicum virgatum: centers of diversity and gene pools revealed by SSR markers and cpDNA sequences.;citation_pages=933-948;citation_volume=139;citation_year=2011;citation_issue=7;citation_pmid=21786028;citation_doi=10.1007/s10709-011-9597-6" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=W. Liu;citation_author=T. Würschum;citation_author=H. P. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations.;citation_pages=769-776;citation_volume=124;citation_year=2012;citation_issue=4;citation_pmid=22075809;citation_doi=10.1007/s00122-011-1745-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. C. Dekkers;citation_author=R. L. Fernando;citation_author=J.-L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: A barley case study.;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="twitter:description" content="The lowland ecotype of switchgrass has generated considerable interest because of its higher biomass yield and late flowering characteristics compared to the upland ecotype. However, lowland ecotypes planted in northern latitudes exhibit very low winter survival. Implementation of genomic selection could potentially enhance switchgrass breeding for winter survival by reducing generation time while eliminating the dependence on weather. The objectives of this study were to assess the potential of genomic selection for winter survival in lowland switchgrass by combining multiple populations in the training set and applying the selected model in two independent testing datasets for validation. Marker data were generated using exome capture sequencing. Validation was conducted using (1) indirect indicators of winter adaptation based on geographic and climatic variables of accessions from different source locations and (2) winter survival estimates of the phenotype. The prediction accuracies were significantly higher when the training dataset comprising all populations was used in fivefold cross validation but its application was not useful in the independent validation dataset. Nevertheless, modeling for population heterogeneity improved the prediction accuracy to some extent but the genetic relationship between the training and validation populations was found to be more influential. The predicted winter survival of lowland switchgrass indicated latitudinal and longitudinal variability, with the northeast USA the region for most cold tolerant lowland populations. Our results suggested that GS could provide valuable opportunities for improving winter survival and accelerate the lowland switchgrass breeding programs toward the development of cold tolerant cultivars suitable for northern latitudes." />
<meta name="og-title" property="og:title" content="Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/6/1921" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The lowland ecotype of switchgrass has generated considerable interest because of its higher biomass yield and late flowering characteristics compared to the upland ecotype. However, lowland ecotypes planted in northern latitudes exhibit very low winter survival. Implementation of genomic selection could potentially enhance switchgrass breeding for winter survival by reducing generation time while eliminating the dependence on weather. The objectives of this study were to assess the potential of genomic selection for winter survival in lowland switchgrass by combining multiple populations in the training set and applying the selected model in two independent testing datasets for validation. Marker data were generated using exome capture sequencing. Validation was conducted using (1) indirect indicators of winter adaptation based on geographic and climatic variables of accessions from different source locations and (2) winter survival estimates of the phenotype. The prediction accuracies were significantly higher when the training dataset comprising all populations was used in fivefold cross validation but its application was not useful in the independent validation dataset. Nevertheless, modeling for population heterogeneity improved the prediction accuracy to some extent but the genetic relationship between the training and validation populations was found to be more influential. The predicted winter survival of lowland switchgrass indicated latitudinal and longitudinal variability, with the northeast USA the region for most cold tolerant lowland populations. Our results suggested that GS could provide valuable opportunities for improving winter survival and accelerate the lowland switchgrass breeding programs toward the development of cold tolerant cultivars suitable for northern latitudes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
    <title>Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pWWWQ7hFFH2I8gyLs1Qb16GeJ0mByOCGYzuxTwrdKLI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493240","apath":"\/ggg\/9\/6\/1921.atom","pisa":"ggg;9\/6\/1921","ac":{"\/ggg\/9\/6\/1921.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/6\/1921.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1921"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1921"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/6\/1921":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mqfByxzUoIXJq6aRJRUpJ6HYLDZ6d2hrxSPG8hNQUBAUhFBm9nMCbYwXX95DIeD1fGonGOdoDIfsaLxqmD8m2Lzo5XEXqaMSAKxvp4NeY8NgGhSsEOaqlZxsHxlfrgHOZS54GaGfDRvuaTAXNhVbDk+feKoKbCoRe9Rbtm0FXc23cV8kmdL4xAIza4COcAgaynX2aDJJ\\\\\\\/2mIzWXl8xWrUWQVKThoeb1d8Hc2HazalmxUkCpXqm4x+W+Rg1xeQBvJu6nHkUwP7HU+GTX5f\\\\\\\/VYPH6ofgnfdV0i2zTJd94hvS4v5uzOp9J\\\\\\\/7zDDGROgHO4cR43abDvyAheP7jTn5U8if2YOwSVr1XkOa0u0zR0NeFf2cYMgC9H5Uv4mOYPGON9if0U444\\\\\\\/b+XcqQ5jTabdlPEGUwtcYbwOpohlWjCiTKd8RF1H4lLYxEVatjKBrj8KZF3Ppk49dpVVW6+9POHOhYZSg08PlL23QdvZkt3w0lsfEcBLxLf+z30G+1IByOMf4AXDu+PH2G+iXSeoPPW+3a8EyH7tu+MMUUuPrfWmW9DvRN4HeFHxK2JsXadDIxycGr4PyOUQlMSrZDCfn5IUSOfZ+Ec372VovnARhqtUob0mSUzkQVBIcGlyQ9Z3Pp\\\\\\\/6xLCW\\\\\\\/zyozdYbE46IMopfMd311PqM1umze2S+zy+YLAxjdn01MFG0\\\\\\\/Fr\\\\\\\/6io11N6PNopGzvNavicNju6YlTVyukn5wbMWdrufrMHCCMd1OE5+en2gemBDjHkzlbaye5wnccH8+TtTHyLUZDMFD\\\\\\\/Ram\\\\\\\/rr7ldQK4bvQP82KxKJZ0NeBdAcUyVE29l9lDMkC2XaJQlwab4FvQCYz\\\\\\\/nYrmVW3y0XgDvU3Jnv6CwcDApvN4Yi9jZLTu\\\\\\\/f5qQmVvAKa+\\\\\\\/0YuAE\\\\\\\/DKWFal1uUUNL2RdJUZg\\\\\\\/YI4sbs3qaTOM2bwHHJ9SvaaX9XmmqvWAJcmIqPVRSoj6ozOuBrnYzaXNYEArcitpNvskzbTv5LCqJeeHvYnZf0QRCQBwT5V3fhwQr961Kt2WTMsX6b3Pju3IgGpDzMCl\\\\\\\/9CRZJ5ZnJl8QMaEZQiyO3WDROhR4pRBLbulgfBoamzS46pJ7yGpTbLAYCNuGOumiuy\\\\\\\/imQPFtj0lN5kjnjKYJbMijeFVvhJwDwdlxIbU\\\\\\\/Ub9DRg\\\\\\\/GpwMgN6A33k060TL36F4bDVPD45IIR6FSMi12y1sHsx1fjIfTjlJ6sfhpI\\\\\\\/eKO3\\\\\\\/+vvNg\\\\\\\/4AnOEB37wJW99OBO+\\\\\\\/lnnRw1yCx8NZ+tPY681LYbtfkleDl25\\\u0022,\\\u0022iv\\\u0022:\\\u0022KZC\\\\\\\/gGhZgUTjBFDxMUKOzg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002231b3386c3299923328d221adec65a8b8c18bd688f6946d5b413600a97ba3341b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bUu4APAfTN6VMr8RhgIE+5vUXTP2WubX5I+RF9+9n4dqkxbuUtiKVmq373lwwkQR6PKfaSSfxjsOJc\\\\\\\/xmnBLFqXc+zjnbMEnK+HhsvVX\\\\\\\/5wrQCDDI9Jd7RbQiftQcabm5em6uAz6PBKHtD2DTTmWQ+T90AkLfDdIP\\\\\\\/bqwt9CONajN56oj6oC4nKQ0xfcA3ympc3Sfzh3TAkuczvzWySkTR2r0WtEOY\\\\\\\/OarYxXjqTDpwWF+krfVRYTTIJGAbUSQmh9Ch7qr9K6Eiwpz0fdfB6p3Oc172Q8EaBN7IXh+vCZIAXyeOZzAHYfnwLom8nXAA\\\\\\\/RlBN\\\\\\\/+Kc3H8belFmh6VRwQPlDqUCiM+6erPNrAznfwiSbQN6CoqeePRpBs\\\\\\\/KiCLs67rOQX9pL2hKaz6pBp0bpt7k6dc1WvAZQHGRhaybW+7eRoZ2XgKQUmhqoQmXOdgdLklNasfuhNFi5a\\\\\\\/8efIk9glW2GQ0mZSlNVNYJJ1VycI0k\\\\\\\/os1HX0EreW5Qgp\\\\\\\/bKGLuCNdKvhpOycrh\\\\\\\/\\\\\\\/kxQkGRgaQ22bjfyk1pXOxghjoBv0mLKpFUxAeJi209ibqH3JTb0lRKnUadWdcIHu7rZ0p0rf\\\\\\\/lGY9kky\\\\\\\/sVWmdGnKPjm743E8qcahe9qqStzD7YXwEi+F185E9GNvFQt2iG+vYz5tnaz2vnqc9YlTBP6Mk5cgxmqVsO8GG+eFNId4KQi8bVjkQIreZD3ZGHpp3VQtQJLqXoAEEy8k86dpMgQ2H0bdx3+PsfaGN2KcrgtN0yv+xxhAfXDpTXcnU7I7GNxEK8wnKP1YlGnebEk7J7v4PwwNn7P+CDjs2s0UccgetRscFal2m3kg0h2yGGj5CCquT0do4FhLrQw8m0jvDULvlBwzaTWKnHn1smG70i7+oJJ4D1k38odRSb2rzIdAIHZ0cBPnlXM20L5hpmpQX3Hb54vLNtLbtDCiYIYAtNoHPjLqQgIJ8zjv6debdHLbWk9g2+rua4Jvhehyf+uYwjGRX4XcaYoqi7XUWRZsr+T\\\\\\\/PX581rONrSIIagEuUANQef7Q3GU6aq53xeF9W0cyL7tTsbpdShE2k0mUX4cfuETdL77RM2VYHM\\\\\\\/2v4Ba8+QkJ7jXp2BnqiAJLxDKe\\\\\\\/UlJ3ZocBm\\\\\\\/qcSRtDoU+4a6Ix+QXRtLsAnQuABPdJ\\\\\\\/dN5RQBa\\\\\\\/2vhiAOxUbfLPyVv2tUbPdWY5l6zRvTnaSyPMmY55AuU4I6qOmqb4DFDnoOIs3m5Ha6jp\\\\\\\/vwkm3uFcuExvMw3YMrRpboqSLBDNWY7JoRm94fmMF+1Uet5YeBLEFc80wRuWuK5grpnU8Vun66W3SEFYE59yqOu3NheyPKBf1de2E0Uj0HIchWanRe3RdxkK0ajAVQf8bwhXZKDXV11kp1rz5tH78uajZXMGaYApEJt2m2giOgdPTj93DVcpOg4hby71X6UsB1PSfM\\\\\\\/pugNsETUl\\\\\\\/p0JxZfKnL77GwJFUcwkQlPW3k84aigKLknB9u2r4cpd7aQT5nRbQg1czao1EklAyymkACtrD+mr+U75HyqVVv7k2xVltxdHF9fjN2PvXCQ7vrgSVu6aVEOFShk1vb+323Eb15PqInSfoWJDfOnWMcWm63VLuTYFPG18BQ4o8XnE4eI5YYPc7vo2YEQgmLauTjGqTWCk3+Zdu95P6nJRYOxNvnDgV6J\\\\\\\/ekTCGuXLs+w2IXeSzzTw8WAm\\\\\\\/YA9+7XFCGuS0pNS4yN70Efpnh6uGDGMRXFsADwCs\\\\\\\/1BHPNRsiPfbnpOfwU+7BeCkO5y0hOUjdrJv0zTkikKgjV\\\\\\\/0C+0Mqbj9xRsFL7vpvpMIUmg49\\\\\\\/meH8WaN3dWM2b8VzUBb0sQbyjVQYiS7n\\\\\\\/mCb4duRGoLc3zXEJcfaJJKXujLjbpI8F3kbc9IoKMqp65tPcXH5V\\\\\\\/eqJOSVMeen+xQLn\\\\\\\/QfaAFyxeI2jl3YzvT\\\\\\\/9N\\\\\\\/TyeJcSt4vGht4jSVaJdLfzlqOMKazEVrgh8ZSPsvMrJXYjO+xUbKbHMXmx7DM3+zHyOsRlUvBqPLrQIwRg4nr\\\\\\\/x75gUIKGRvxN40FHQiIbcnE7xqd6eWuWCKes2k5U0156iEIUKy+n4kpYnYY66F8u4\\\\\\\/qTDRX9wNXeojPkkOymoEhwQ+3fmN6uVkge+YuDtilCVurs1OGNeTs7u6x712oG\\\\\\\/0zTvT\\\\\\\/5NB9ZxwuNQX+kLOkseZc3IL3MD\\\\\\\/29vgdibdBf94c1NxaK7+wnmw4BuaJGBq2bCYHQDCEaQ6GXrcUilS4aZqpqUm9nzDyOhOgQnhy6Z3OhMmCzeCcUwDyl2w4T+3Nbm4zeaBXJHuF1dC9EpbuioD+y2DDds5ClLxk9gvofSQMc8OcgVB9dzPmUB1OpPnB2JSn9s+H8lQnee3frss5glW1MF\\\\\\\/t66fhYV9etGV018H7PzxtWdX+zc694VY2dwk1f8+ToYsi9VNF+VLr7yDokEj8uO9noJEWpiC8vWygU11ZwaG9oQYXCnmHoZK05fDmHe6Ka9gFbfIgQjFoh0e3wTWuhVeIXg2H6tHWIiJ1QRioN3AOx8I+3Sj8AJM\\\\\\\/oth4LTuBbrg2e9i5Ew\\\\\\\/ykFlNWtlyxkZorg6r+C9klzWAW+qkuCwOstWxDkZjh8VAmRdDKZrEtJ6FCxbGtLlewdASEUrfhJ7Ap+bPtbuFns4RVqChV8IRuoxEkaFbirUVVOQ0R6fWajU3p\\\\\\\/bHTP8OdREH5PnfYzB0y80cV71B00ie+pTx8oEMS35HACHdjz4MhoiWgK+YTRsJe4wO9R6osZ0OJD9upYGBIRos6pYHZg64M9tVwOlhh5\\\\\\\/inp6ei+N\\\\\\\/iE75BtoSNTP2Ys4ri9nYmfBblWXrAFdOHuHI3lJzUQK5l3FEiFe2zucQImbs5uiuHbuK7Ua0Ca2E8cpu5Ka9WWLv5KICixpdjkW6W1hN8arXT5QUyq4eZaU5\\\\\\\/0wUDlsURbmAfYc0Q7UeSQd81Ujl7SBZQsUjiAoBfZ7ETKqpBipCJq4003qhPZmiUrQmCKtERHX5OwloTsJCM8wonUA0AGQgNKBIn8+NwFIKozKyhcqn9XVAyY8nsttzBB2Q1kBKNqDs3VmJhnYga8RMc63YJxOt7+mYlbK5xuKxqgdEmMAJ7w+1ql4EccdGkWxFFQg6g0ethUknx8UfcK+JO6u1+JOfugac8Dbtu82KM8w\\\\\\\/jzf5T2UmUyCJB0iD9P2KEsilrDYyUsIUgY83XtVEhA0nCOf6hv1CxsRCJTy7XGrlILxCrU9tF6h4lhkS7UD4HxhHH5iyk30sPpNkQIBQP4nA8l7Js98O9S2b2aTjjncDW+uLj4tg7Y\\\\\\\/4bX0CRNAVsXpYVzoHoYw4K8LkVkXrMHWBtsDSeMYPlFOM3pRiKeDDNQHTWLh6tliR43Z+MsXonfsVUNfJsOu\\\\\\\/TUbkDmxtmeYd6NPI1wybnPD7d2Iz5+icur28zTjtYHk5GGdhb4T4CRHKYPJKuRNbZxW7JVj\\\\\\\/PM\\\\\\\/H5WmI25jw5YDGYVslqVdjNwld4Kv8BOf5EqSxviUu7jAK0bAioHslR7brvHMiCBKd9\\\\\\\/YlxnRTocV20POihZAysPyFmKO+1VZ5ie8wHfJno2i3iIwaDwb+7HPbcFQ4Mi8GGPZqc4tKsGdgvflWC8nledI52uw9oCB0\\\\\\\/pq\\\\\\\/wR8rLtCegLG7McjGe+JgG\\\\\\\/TY8YTM4YPkILfw4HH8W3tW2dJ2qR64oH09K\\\\\\\/n8C\\\\\\\/VFfFcSN3o4xn5jxuFByfov59aPxruTb95YYEw+CzXKIwNFKnGMaaH165CZdUYG7BeuNdkoO\\\\\\\/VMOQXGJk+Wdh+JvlqC0m9KKTt7O7ROh7TCGbksZLOKWOLV9Mcx\\\\\\\/wcMa359SEPiAcdI5t4xtWLFqIFQLu4Jax+Hwj7+en\\\\\\\/wQXKIR+kWYYs8N\\\\\\\/n5oBELdMXclsEH4V6xCjwf6dW5TANpLpOT+MynWa2WBYPWcpHULh\\\\\\\/ycyTUlJTLj1aiDgPyJfYcwcaT14YePdYfxyEg6OH9kPWmmBP2CmI9eCJlKkXrshRTao8ipxt\\\\\\\/QlYp2KUUJStOrSPBGBT7GuvWmOLJ2RnmLQVBcpwW5bL6PBF9GkTIjhIIhhloOfpDOTcpdfZlcz\\\\\\\/MlqmlMIMCWzkKFHPS285toemJdOZBgrxFEm4Azxb046\\\\\\\/sKKneag5mn7aVQMDmq4JBf9CrPqQNLWZRYM26Q7mTQLBOVr6KIbvJBrjSW8A1gdKA0Jvp1erLM3Yuk0HJpDgkcqOHJiFE5ROfC3RVhHqeE10BvKEReOFNOX9YIKdwYqCDKr1q2AKL01LBxj+CfNu0T6JqVj1flfSXt0zer6cOnLTKswrh+TrPocdX2auWFUwhUvRnHqUOJHLkTYmrWS7xAvABRQdOdiHu8aAuVW7wVWa7ObJnFXvADdwSByCTzceVWITS6OnKfnFPB0a8OOVo6pfUyna+y+TZkFJFHKm7TqY4IIS9o\\\\\\\/XAq5EAc\\\\\\\/o8bPCE4kJIsLy2mYj2RZvj9j0Px\\\\\\\/cBQ0fpjDnMYUHm75o4\\\\\\\/OReDX+ZKovVOza4MnwDiAdjP4riCMwft3Jok+2cG3vgT9W3lvdWhCQBY8Fhi7BMK3zWpVJOpiaHuHVZV\\\\\\\/HlToW1uTz78QoHZt\\\\\\\/XfembP3CVxFXDNsFUkKK7RwKWnM0wU2G9zYN6MhElMxP1rIB68yeytueVqNvnBFCyv7NKR8D19yOLgYmMZFnuelofC\\\\\\\/UjQ8jr4tnB0\\\\\\\/c1zWoVVKYATKpjnTiAylAIe4C6nXnZQuu4x6ytJ0xaYC6EmXdqLDW6Q5e77gHbZWP218WPUmC9y2MvkLh+dyB\\\\\\\/DRAbQRtKgJrClbGFu0hr546WBwHoxfopm2RTm9OyIOVSianlBNXIhyOu1zreDaGN3XbXstePSTc11TOmlpi5UoFyg8oUP901KpCQiRyDrRV8RYTZvdyvydeiwnbNsMGlcpZn38t94Ge4AAJ7bjfKOtBP7HSFMF5UVrhVOyZU1w8BKZoMRf6W6W2pdHsijLzh8JnhQ\\\\\\\/MI6FmRcHtDZjUgpcLrcnKgcZY8LAQbYl\\\\\\\/BgkXICO7RHj83KOQ+UNOmCMJUCThFJTZR8P0HhyA5X2k0zFb469I2pNRV0aJxy\\\\\\\/i0kFKrD7hcKv0hxbDE0vzwzymC+8EBXa+STNlVMAD19UFh0P4z6ziKBHvnPJ9qqM3k21tBx6g1fg7ZnW4bevZorQfJSPwd2ajEHOmK\\\\\\\/nnEXKvXxQfZxb4Ii0WMMG4FLVJ2NzdG\\\\\\\/aMQkraNQs83Ze9q6aT1\\\\\\\/O\\\\\\\/pROZyf05x17zEktiwk9vIb6rAmBqiNe65Lmrm+Fr2zDtkg+Ce+Q1ulXgix2LBCmaBLduk5ciW8+eQbyzFL0EgnrTgrnpsy64Quz3XpMDkyWH58TZlA1xuv7iqe2lasV6Kg54jPEDtsR5cDQ\\\\\\\/f0ZIF4pE13AOAgg0wsQH6OT1Zb7XIyea3sXAHzDd8PvhMPChOCOOiveO8VWs4f+M8trd0wZ+LM7FYU4Z8\\\\\\\/DoBnIv5ZQLe\\\\\\\/yiqy8j5jKGiyOW1DTnoEdKD6ztOsPwb2dYQ6Xa+RtlKGc5hBs7McsQiLyLJkJxnUP2hm2FVuGB7vD21Dbx3fxl8RvHppBtNpX\\\\\\\/pdFQxv0BzopkS19Ot4pqWhXGZ6Gwmh+kg4a\\\\\\\/jYMOw2lq6wXbhD321RZs8CL9wtr8elkHR5eEEemIUkbFS3d2EEeoLrVXdJ2P2pPbmFwV4Ku2N5gE3+2N6nXSPGhf7dMA+qIwE6qU8RGsKhT5EKMLX2HJ4jphwZokToVwnsmgdvEUtLrXWLzko4Yo8cyki8VBPlhvUg253o05UFuaSVMF15y\\\\\\\/0PdhEmE1KfRib\\\\\\\/IJ2yZdYSdEDPQ\\\\\\\/2RoZyQErOBq4YxPzbyOwQiqeBQMwwF2X7kwaTE+PoK9JfseLvo4a9Zn05C93FqcCj2DSo4sZYSK\\\\\\\/ucpoqaEdCz9QT3HhIl0LN05GsYLH1CXXe3XPZmd2PiOON5kA2Be2dnWVNlveY7R6xQKxJPSQGEHJYizZamDzLt29x7iCKajwJJXdYAXVAPGQ7fyR7Rrqbjq\\\\\\\/GMo9YZHZ0qIEpMQKo6q68Bu82tTkyBWXwvFDBWTRHT6gQ0GwCQbCyi+8U1YRK4MK5\\\\\\\/MlJ1aCu3ds6ON\\\\\\\/l5BrtN1pjYHQts26TYFOAIQrI2fHkCTYEoplL5wxtLwUC\\\\\\\/MT87mc9IZqcIykRL8BzLxVHeedN70k8UmJ8Y8BjozGwqu8RHhPP5I1odwCcZav81XG92IZThNdBNnvgAIfnVdhYU36oST9s9eVHMjDS9O6TIgq4LOzG9PcRf\\\\\\\/hH+YwBMj6hpUV35wXO7ueS4rHXE+Qvm+mK+vTw5HVPV7digBN3gC5xUVhqdOEksYlTJmGZpg3i1v2xCitidJp2LoeLONQyW2c+P0ixYOujPKkgCyAwpBR9VMORALhEIZg91hodzXaOwGLMz0E8l3\\\\\\\/qR5d9+PFrnvS1PAgZRfwTTl4gTQHxoinrNX0hzYs7e1Bh+Grj1T0tyJ586UmsZjXSfBDZ8yJ9YuWTEf4PgfONpry0TISwKBSKhxS4kZK+U7\\\\\\\/jG9\\\\\\\/84rnC\\\\\\\/eGy3O5lZvIqiOGcV3\\\\\\\/xq5FNaijsVep1XXUh+ZOd7MKN7fciDCbhZKAKUxh7hcvRetR+JrbiVQge2UD9gYVCnDs1wGvm13OXcwZxbAb8FLv71hI9VICP5Slf9UWIF616h7QIMttiDawmsIbU+ut8Qaa5qFjAH1UetqyxsfTru4M\\\\\\\/owAOzg3MKt5KhnE8MHBBLvBZ8bgaZQYrbGhRruoijkJULnL+NYbgDUs9UiLwj0P8ao9gbCNRbVqUk7lOd6YlPPGdg4+b6qlePk\\\\\\\/6O\\\\\\\/jF2u8LHf9PXgp8MD3sYT64fLNr5B52FlmaT2a7zkyjAH9vJ\\\\\\\/dd0lUnPPrsxVfhPyysOqeKc0SkL8G4OKsZuJmhziJYrZqc6tJO1\\\\\\\/EBwc5YVwHEKX\\\\\\\/l5ZfEOOjmRpj5lM0lkVcM4x0DdaYoZTAyIG2q9UR1\\\\\\\/ThS9NBvUAKIJLBo5av7AQ3A8fYCelEVXBc6OVse2pHuhwJ4JA8KzedSoca31jWImaP\\\\\\\/IctdeqhfZv7VT9k3VjgEOfbRWIANUjkId3xUwf87nmKlac4S0\\\\\\\/9fVq9y3U0aBq0JMqLFnuYBDg4BTwUuRsTk8RAW2Ei7kxik7xqedSBeT7et0N3d+m6NPUMFTzrs\\\\\\\/Cam\\\\\\\/4DUAXsDfVbFd9PP9nRQugDSVQqB8f\\\\\\\/k0ySlVR9d\\\\\\\/ly2yr0Ty7UHC4lQmJ47SNl7+sei\\\\\\\/gPOkIbNQXdtKllnfEEa9mf+X\\\\\\\/aEnQuh6LjX\\\\\\\/72HR+ysAPTGdZ23b514HlWJULxMQd\\\\\\\/tBTJ7F1HfAiDxR7teCypioENEKdNVSE17TgZN\\\\\\\/5n9iO6Yy1zvPqHCebSjBWFUJRrXUw+cUvNWIaQaYyWs5OVUg0JEKZdQjmwPDRBN7wPVmITnYCaYcb6v1U2Oo9Rr5p3HuJRzaq630py8\\\\\\\/R7i08tBibqufXeYWZEM0w7DR8P3q7TrsCJPvg3Hn7A7NadcQjIW5hnzzZ26WZRTKwEsp6d8DMXEg5fZMQwTQfoXSjkpzr7QOjbo5m9jvFuFM9fguq8hcboOLkqAXQ3EcWyRQmi0X4wk95REbcxA4jd3pkJIK3YThE87JfIWpuEmNci36lhB83RsTDcTtzTefEVYIcP4F3Gl9eg3wjLK5zTmD7MQmVEIqOxotxKuuMnRF1JPPzetgM8LD7cQDAiEl4mNy8Q\\\\\\\/GMM1MgWbMV7BCNpisjHqvgmPdtcw0akDimwEactZ5hRm7AMk4zgnOrqA9AjIJZupc2BA6dRhEZGILBdRsSkgUh7sPukoyb4Aq2AzO+ue\\\\\\\/WRUztzU8QvdhTHSv2ihRURwON0IwwZIbiGCruM8ZiT1bwHGito\\\\\\\/PFqTN3rTDJa7Z5oj1za0imu6iotKbjtYzDdiB5eZbXDIPeb6ZJAXKsjGFo1jNSFShyAXhsxYILTujRZKilLC1PXBXW3AaPmL6BxsS97x\\\\\\\/WTy9jn3E04bvg5IuSn8Xxls5sbx\\\\\\\/aXsXGxOCgU60HFWB9T+kB8T3OpvJvomdHmokipGEpEmocTSsiyhGpV19u\\\\\\\/0gdMBflYnU4BG3Ra5kxq2KhV71tf+S9pGFnD8x9vU7fIe1PPVNwxAzGc7voSu466Qwr5IG9YZikJhHjwsdI+\\\\\\\/DZXqPlpYCgkKQwFVcC0v+UvjSqYXCwhdS2Uj4ehVzH0ka8dsr3Cyaq2beNxO\\\\\\\/aFnE4eZR4XddA5f6wKzsym2ckWRNzF+jgiQkkVUKyRNzShlqgIzrQGxTHyzlDuzaKE0TGimar2zUq\\\\\\\/nnshpU+ivBSx\\\\\\\/5k7i+v4W+pHpxZcyHlUsZTZE0XvN8sa\\\\\\\/AFKPACcLOJfe9kygh+kkcYZ2K5+GJft5YVsLw0wVQjd+UmT0Y1u9g3eKYFUlSqgu3lwIcqra75DhvOKMxNIiUaZsjWb5c7hCzm\\\\\\\/dYyfnbQHwOVhcmkWW5j7wXy4GDUdcyC9SigzWWpnv7KX1IXMkYlVyScslsDLOnjr9xC9sktC\\\\\\\/9bgx9Ey3dHLiSsqmHqUcfySuWpaCHTtdVG\\\\\\\/q0ps0J97kbzYl96Knc6GCFuPAShjWp\\\\\\\/\\\\\\\/6zme\\\\\\\/mGhefZG84jOB449PVefeCLxaYg3ekd5bv\\\\\\\/ZZtKzvEdnsm2eeZDnggrlZWnb8W3v8fdsCXb\\\\\\\/nZ5hxhj+3rUQ8IGfejijZPkKEg2eQGNobDJ93L2\\\\\\\/P8WGivUgi+AKDihB0+clwvhc74I\\\\\\\/V68jgHibbuHrqutKusZbVEucxyh6r7T7FF2PLl+GOkwbXJaod8Kw90euG2h6LEWdQL47VoFnIjGgxsQ38HndPODnH3GMT0C+Mzn0WliBwMUM2dxuhHURQ41Dz8HYJWzDpHuqMzTTCT1OnXn415dYgQMlNGy\\\\\\\/z+RLwlyH0cwidpqlJe465xd6LkcHVvaVAL2ijpO2PfkmClRjytAoy0BYwURI\\\\\\\/YGu\\\\\\\/KMyN5O7MkidYx3Uq2FKqfXTcNlQp31kIY7skvoZBJAxOJSr2+A3kYCHl5o5pbA4HYeIvJvtUeM7aohF1jXjKlq+OJNWDPMNnN+FdZ50OSMlGK73unKcpyxM2dNTzgui4A+kMTlHWkzxqbOrEeTf9V7ionv+zXqhsseFAyi2vFuL9nQVH03HSxKHpuUCdDtr81o\\\\\\\/vWMP5jjXlT\\\\\\\/p\\\\\\\/MI+ijLh87v0TKgBD6zE+8ISrWvnh2OwabK3ytMdIuRqfcV4i6rsWoTKhz\\\\\\\/zlexlANoJSjDvRXSD1htneskvB36xz\\\\\\\/hDuHMAF5AszGtm\\\\\\\/DhLA3QkJUjlylTo5JZL7klgc5lQO0XT3NYoVOpQv+26+n0cYMkcFke\\\\\\\/IrVEL3tN4xcuB0Q9IFHwrNp\\\\\\\/tbXighZCwFhrXY8kAUe8iTDwhl6smWYNAz54KwpKfMxaxrMrHuxdL3T5R+NX08LeWjl77K6rthanudBvtBZx6uESoLEigLoOud03feWq2r4eRw1H8jGBKYVjtvoyR6e+OVG0CyrC39m9JgGgzXkSY\\\\\\\/5PF0XRwaxfKk3mS510D45Nn3vde20T3WLSjX39rywoPJLe42ZAj93T9U62Nrpnq+sVIQNAtTI6FTOQQ2Tz+G\\\\\\\/S22HuH7uFKqk5cNXyw3LtFadrciTrVz\\\\\\\/mdbhk33uEwtndagFEi382eoB5wVaYqdF2ok6h5pWLpKm0MRkQfwyMAseGMjWt9r\\\\\\\/KyrQE4h3amCndWweGp9G42rsMIUDEJtrpWi3i3LYvwG8JREx4Pah4QhJXQZ3JGGJ4ZeafaLdJIAhJ\\\\\\\/7Cky3+PdYqC6wDUZZAOxhLsbdM68TM6fHIDhXOMOHq0jGaPCc0ecWfumBEM4CgrdcBXwBI5sGJql00\\\\\\\/vG1f3a2l7DOAmsUvaV8p+LA4pYmykrM4eCUu8NNhhSJ+r20Y3KLd+39BOXzdCpPoHunnZIXcJ699\\\\\\\/eaRZxVRpX23H9L0uUPpUiNYACVRa7afFnN1M1Ff2Q10xK63Q0SFddZ3aMORNuOlz2TL8NOg2E9gF+LlZVq\\\\\\\/9VuVSDylEG4UxSAB0ZYX6MtTISe2i4\\\\\\\/AtBqIGf9svpLBLe7FzUV0dLmOBVByqYLnjXLh\\\\\\\/FdPUQK2wqypZ24NE6pZXDzdYLJEZ\\\\\\\/koKfSFgbLRxQqpbuu2qnLR7vQKPGlMxozrJWvDq\\\\\\\/LS2QueeMRJTco832hqsskj1pGACwlevBZsfCKtppxuNZ1o7iVPzDcN03DPRFfKzMh9X9i92mkJJKhlhixJk209691BgEm4Cz3X1Te73h+Xo2pIcdoM82ptdFYQt32+TJzXkSgDKnp12OsSA5pNPpl\\\\\\\/n4FcO1mvomM3mV3Ox6uCjF9u2kuKeIus3P5kOFeSN6yxkDdn7m6A9af2WX2e0SytXa2U28pWX4VwPm4Eg6KK6UYL7HH+2zth092i55+rfj0zphreirQuBdSzNS94r6euS0clqHOG24VccLkD8pH9tweH0hgwfLOgJwNTPyvcASzjCBu+NjalHqbLwjtE4fvgOiceuqlg0+YkpBcLUg7EuXNN7Js4YFBoAkQ2Qqkp9U8yLIULUgiICOqTQuxcpTsZ5KG5iyGhM9E7t+MbxSVdwEK\\\\\\\/1x8r5KUkSpHTnl+Fg\\\\\\\/Dyw8k3w+D0mQ4dF5TIt2Ft6b5S1SNb7AkbFa8jc1ERTmlMrqX9faH+8D3KK7950ryagmxE5OjBofrSTq3azL+auzi1ACMkFc+X8u+l5CfOCbsjb8Vq2jxUuTUcimH9CHiZCJTZjws3u1EtKnupicxkE7bFsW+cbgSKSupvdKp88xAMTRCiy2hCWpYwhkdUniZDMPW7knu2T4R\\\\\\\/yVskGoiVHU4o\\\\\\\/JUf9vMVFgSv7NpAJFMUXDSKq4J71mM9cIGme8y+vEKcU5R7\\\\\\\/1ZWevi53Q5iC8U4Hk8o029P+Bkocc8kX1Go1ZlrqLGSDkyB+UlilFQEinikWUQSRY\\\\\\\/ILQqCPyf+qRmhw9Ygf1eNHk0FfQ7MJXAyms4+aojOpOR\\\\\\\/C\\\\\\\/FMNUAh529+UEXuO\\\\\\\/tp7okW3AtWPWvZE6f7Rf8eSVLiSCRuTzq6xdaKteMbZVc3EmecKIe48Y99cb+qmi6pOTOtewG3vPv4XVFH9NH6clH5Qn6GyjjX\\\\\\\/kaEU4yo24eqmHZ1M3QN0YiXoOnGifHRYlarmqp6iaIj\\\\\\\/jjQ+cXkw0klON3l8RyEHZah30bEse1cweWCbG+KG3YZ\\\\\\\/6TarMEQRBP5Sdxxx2Jfbqgrwiks4eJbKet\\\\\\\/77lBrwxnIc6LA3Z3lW15GJ8UUdAbvyk+bHsF6G0hecLFjYkcCLi0ZgxQv8Ns6N7zhlfJaqlgM69hkNhIfT+WXdh\\\\\\\/by0v3qj\\\\\\\/xSKpTWw0M+zs1CY6dFkZ6djsQEH\\\\\\\/faQ9zzWfyzMfpLZ\\\\\\\/gxFtTNc2KK64CdytTU8SXBlA9\\\\\\\/eDASa5jk6ox86nwyBCfmTFt\\\\\\\/dhcf1YcouCOOAmYxODAJEwn2tcmZLftkPhQdaL5PzTCA\\\\\\\/krP8KZRK6BA6AA8IkxR4S+6iTDlC25k2\\\\\\\/coYAm5qoAJEsgtBCRzwMVHUh\\\\\\\/uksz0y66nc9OxiiFgg\\\\\\\/5CMAZLy4Uewc3Q\\\\\\\/N8yoE9rvaPKlSnQwmZEugUZ4hRo7eIgCVnuGceywAdDPHJclshu76PKiOV\\\\\\\/6GtVJNlN6gH1JXa\\\\\\\/fcjSHKBvR0gOTd1FJ+vrgvlx4bcweF1ZL1JUh1IPWM\\\\\\\/VqlL05VaHvHkrpx\\\\\\\/aKrBbTkIH53zpg0xN\\\\\\\/HGJd0w1\\\\\\\/JFzfoPTP70zq7bgMqbfX75U3zudL5ECUhLsY7NccmLQd0QqRxTBQkFzim9GU5ydSncVT1BVHuhlel+Ja7sR5UQ9FHfZL7+Aw4KkfR2fTAsk2lYpqe6LcI6Wg6NPv2yXMbM6vFUP2sVUkRP+\\\\\\\/IM8CrGtyndICj535o22m\\\\\\\/DEd1mj\\\\\\\/+IzafUXOWBJ0wQDHjvEi8z2IxE7AdTbIXwGHYduBYjVIpF7+ywSyzdnOwRiX15Q1gv4yiUJ6bCEDjBBwfrWiaIiAH4BuqQvQFBcn3ukTy4jWmS5S5QszBq9+eSbCeKO\\\\\\\/DH1Hjenc+W4mgEkpaVeNNXwWC4QNAQJ5m1sKPY+DQE2pz4+BxEIuHwYGSrpd5aLSG2QSAks\\\\\\\/4o+2myne3izTdC\\\\\\\/IsEHSv5kWPP4SymldMDUUSX0ryso9DSD8HiBzoLZXjBq9gaa1E\\\\\\\/efiEo9TdThp3i23JV9HmrpCoyxxg6E9Td2mkzQMVEmGI8mMX2LUf8HGIyJ6hGwxdcyRPUvlPybGiAJ3b6lTIFJovYB7l1AZlu+Ev1QZkzs11Hlnd2uP1SdEqjTFEbQ\\\\\\\/YO3Lj1tt0Hrh3Vn+82SjO8ipMQZgCgN6RfqEB8S2hJrzzNmjl2Va7nUh7a6U\\\\\\\/bcR04VU+0jhEl7H79zlup+SP4rzTEPKQWIUB4Fo6vMMdOBnoz88o4\\\\\\\/Gdno7tFpkIToLhS3lv0a9bVYU3Lnk\\\\\\\/tQMqSncp\\\\\\\/cCkiNNFei2+GqYgxX1kT0YO9p1o\\\\\\\/na\\\\\\\/8t5eA14Hu94vSS2qYIKXlLMkwUgn8irnAzbqpuZEuDEQcSbLG6rF+Fx2Riov\\\\\\\/eFUdSDUTrRBiMz7dQtx+pmhvuBq7i+OcN2\\\\\\\/cXcXtztKyzPy4QtoCzMfm37k06qQ1QmtPE+FzofdJSzlxjremAFcxmfi8cWkoqQM1sFWi8pZZ9yhYtq8kH5iFyJPbBEZSwiS0uokfBUprMNoSqvKN1uJAKXZqnYCgUhGpGYl2xymQrlZj72zp\\\\\\\/zO+lO3GwpWxmdCx22jn\\\\\\\/ai8ixwlZmdwX8cC4qJbImlY+7VQlCe9WwPL7E\\\\\\\/0LliJoYk3FY9uMiJBlZw+WLfr9OEpG6on4HK2OL\\\\\\\/NmgzdQOy7e7B3nqVR+xTsZy2az4SDfMe2qJpDazRFQyjncb++YNPgggaRP3Q7Rqeu0ne1yA4ubWpGx9JfxFbJFoNemfEU2unZ9jJMs4Een\\\\\\\/6Vo33jZ+yYKefsZ66ePXoyPNYmTAWsfXLCsi5JW8Hmk\\\\\\\/dcXQS2AIa3cv3wjTGt7yqo+SPJR15Ng2y7ed5x0LD4qSAQG1U1lF1i+PtUzhJcJfqEKZsmj\\\\\\\/2duVMwMtti\\\\\\\/l8\\\\\\\/28eqHOmw7q6S4xQUqGnZIa016GNnoFChP5Nnm9ysOSr9N86fKSx7vvwgMHlSiRY3IPBZrD\\\\\\\/sllaEJCkYNamK2pjVYNBA52mIZmGhlldBEyKPKDk\\\\\\\/9FpKuj+wT7X77t2zApg2utXxtHF8gCQQjtnmJ6oTLFzSEokS2+d\\\\\\\/ZocjnWI1B5jtFRnL+rz+Iryk39lDJMhcQwi2blFvB0ZAFcyxojGSuuwHp6eQFOV\\\\\\\/hYawQ3IURL0VLKQ1caMHH3q2shCrSZhHtJ2nfsZflUXM5\\\\\\\/RZ3NE31URT5Tt9WRCsfOHQzCTq1RhcWa7hPvwmGOHR32nHt59DTXUTxK6SzSJzfRX9r58OmRWtF1PK6ACKdTQguOmKYOFqBUO2dgJA5GhTqDX5Ph5i+pfCi8ph7lgZsnp5lpXymvibtkaVi+YOd26B13Dls005NjSbbxdr8GHtAQSsCEjdXQhTcy\\\\\\\/K632WKlUWIaisJvrbAE6ZWhdFeapZ+4oBNDIo3FzkUBcWapHwO2sxnMOn40Ge9QZ9EHtCC9BTVW+A7PjaCo7KS4R+stVWPp2T0BZIPXHw9c3kRjmGvvRe\\\\\\\/kCoQg2rjErZKRwfS2urZN1VQ8h0xMgUNzDj+j1XZDCtBfMr3VKTF33YMcQ39agRhjz6dKpodAu5uf\\\\\\\/oimt+\\\\\\\/1lQd\\\\\\\/fUadkqCGINIQrDKDsSRqp8Oe8Yfeq2UhajcriLos7dweUMIGJoeKc6W9n1rnsmm4FKbLGZ3UnR2QifScXoMbmiElqwM3aAj+lDVNBi6GvzYLLAFLSPbhPpnWtv86w3ILFQV37FgsFndvV4uPeUe4ZXmku5KvkdHNULKwz7J9erEEv7nl7nPuwT7R34f9Y68cfueDFsQenC7eTC5wqvuNeoNy9T2+2d\\\\\\\/J1clqq3vudfpaoMSE7Je+SJZk3Yi27neZZti4AUb+jXuPHbjispYQKTIhhZCimISsw+0GhZrKYniCaEiDQIc3bRLuyKQYbf5IKJmxdQ4pGRjo988prPODHVdDyIZZfRewEdsgud6kvBqd3UGHL1vGIHel30+Qcmf8SWo+koAO95n3XpO7fwwJ31g1UYr6ecBWzSFnd7YTW0iBg1qMMkyngpS4ejDLGHlzH3JyoqP2yuqRaVhO8Ggkbk\\\\\\\/Gl10JpoekHU7aX68qXe9sjEBe9m7JIC3zsmiS7bhg+riwl7o3VySOj4Rxeq0LKvkb06FJmjatjzOYP7vQFjp1y9AG6HV0ofG66WTToSaaBYWmAuEkgB4YoLOgLhWQq0nKCp3EBaeaLxDlCW9tNcS+Okm8LL8xNiZOjkvTdYZfeRBKI7kEYZASr6yWFFW0zt7FLZhRILEOC1WMtI58uQBfuTkfgn70+WMVNSeTRBaHNYIVgICCQGK0BFsXlqSrJmPYqnqZncWwI8ueL0\\\\\\\/2q66GT8onXP3ldQpLqxrfLo8h3oXpuhVUoHTMuklvHWMqXR5O+0UA4TvEmpuluxqCWJAAkLq2hwbykaQATGiYzlg+eqjaCHI7XQ0FioGR+bpmMUbyrskqoY41\\\\\\\/gyFTiLLc\\\\\\\/f0pUy1hv8MBPqS\\\\\\\/BEXzXil1UKkyWWuiH9o2Vnyp0we6J1Kzwz6S4Ju9+7CLnLfhDpJoBPJuJb\\\\\\\/y2QfW8EpbChKJxNm+xXeGOB8S4fCXuRfBFaxxTJe7ysMcErFgoxIU\\\\\\\/11XtBD7k3mYf9DnnuQeuY1HboCvqWQh0gQyfyKxmKiUAOXrYcmyMPEBrlzZciPM95zt4REtpc3erGGgbotc3GhjtcK1TOzDSlnrvg+4w2n\\\\\\\/ek\\\\\\\/cWDJ088LdloTSYR5\\\\\\\/DC2wqef6f8SphG3M4fs\\\\\\\/jxUn0Pt+6qvIQwV\\\\\\\/n1hf85XSbf0jWcWJQhr6YAioRamFRXkIH+87CL6b7cfCMaIDNw6vS\\\\\\\/PERB0tT9z1Sf3zqj1ZBFBQNIEYMOmgrwTzEycUnpxryfd5GkBBGiS544X8qVJe7WgCyEtSBVEkHWEukow\\\\\\\/QW60qEQKjR+CQyjjgr7uOqTHGZ21wchdxZPDV1VxN82e2JJzyfL+N0NAW7KAiL\\\\\\\/uwGJtcd9\\\\\\\/VvhjngIf+vyCfIDkGwQsvnlETzKYcxJS2CfUve7+KELT21yD9FqClcbGdcb9uId2hblArGcttKBsLSXbORNBtmaojiDYdOccavu7DxXzu5354gGU38KoiYY09FiT23SXVNcGPrpTLSnh7zPme+IBqDO7Ygsfd4wO3w0\\\\\\\/hlGaYP6jDqqLpbGn3U\\\\\\\/gWsnoOLp\\\\\\\/4fdenQ0rzwtxBSUGlvIDz9SuK6vSpso0aY3GVu6tNq2ZU6av31qpCTc0\\\\\\\/4BMHQydU6+MIqj9Xg+khKg7GbX+jjSEboKT3HumcrfeA13O4O56I7wIgQznLSiRpkmlb074RbCYbN6ptNBwMVqKxp7e6ZJWupTA+Mr8aQ5RD2e6OTtLl90LUVuRsY3ANwMUrRTouj08lxkzxo12DZXuawgWHDGqHW\\\\\\\/sPGbnWI5o9KsHP518lUFrGpEde9+PflONA9wc0FpwAAPiWmZnPIPpTF4oI3veaVjrqcTCtgYTNXGrTtRWOuPHFcKOWhyx+kCZnAarQ6Yms9OYh+BonXub08zX86lUOj2oCQssxlu4FuuHqg9nbUMFkLGE8K9exfjikxZUvbYfSXOgcA82CH9C2HgTfkF6oMsQQG4SFsiRGd5eYg\\\\\\\/f2lH6qmUzAi8r3MQvWrLuXcZKPyMkqhf2EnTVR19N1QeRBvFrRd8WLKXg7vaadaoE5Zd3r+5NNJ8cUUgeYvRiGEcE+KAgxEujgFBzT3QZS2AMnhc3\\\\\\\/7VktBDKJCbjxo0KVjQfNENmF4tEDqwrOtWcTwGHFrAnrJKvtMfzsHFr5aRSCG5jzX2+bmgkk7WkHOaCBjePdG+YutJjVnW1uWg3HEYQDZSdaDWuKUJuXUawWHflDs8L1t6ZTyW+btpgjvnvh2nCggt0TPhISEpXZQ6\\\\\\\/GQcdDP995dDLIin7jcw+o4s9wu1fIRk+dsqZohs1WEawqQG426MvfOonrevSgnecRJa+cLSjaUhIFNT8UQJqIow8C+jKg5yRpFO1z7sKiFElWbmVi39UOCQkvNwB7MWHgtJhUHVOPVBWvmf9cgHdeXZBtPKMhxeh9LEYk0dU+AsxsoYkChbqtsvvaKN3flWs9irCMMDI0RaA\\\\\\\/VOMas0W5JVEJzV10gHdBgthhZnkAfIsqlEEZA1UKQpIHVkec8ayFmt99FGTcuZVdcB4upUo+otn2yNH+UTE3h9CMIcz9UD9l+Xus+5QSMoCb1M8yBeLPFBIcdmFy4aKJUU\\\\\\\/pbKGDhb\\\\\\\/UDQbzb7NPYA5h1jah+ZHbfX+uux4D5s\\\\\\\/ESSyymUHf8nA9hlvmktxkxzvK1FluN8+OafV8N6uumJ9j+5thy\\\\\\\/eBoez38X\\\\\\\/MYCteanwga8G0heyt4WJRPDQjlWiG1BJFMcaHpIkIuxqggxBixfSNWmk6+4rR8sMleab\\\\\\\/U8x+dgMusymJte2fB90jCHilJYHrwzTGXrYw7HKSo2ZrWQ\\\\\\\/hEkw211DmH4CdZ45n0KEQmfGqYI2ienjGEbqc09HLIAanjc9w2LAeSCaVd9kSVakgIwX6chak8EsdnxzIaclrc\\\\\\\/W1MvmA4lCZp5UScwL3651pZpBDCRjdl81GY3653C2xFuuM1FzxJkHeQHsKWiWMQaxd7hCQ\\\\\\\/TquISJU8JasJNM1f3s0QIrAlH3NHUL4ai1SAjgpIWlJ0uNMvpSwVj1q7BIjowCws1cR7Kye4iNnflSEczc3USq7IHgk\\\\\\\/mZ9P9oqsnrZRzB0ESz3xkCydLd9sccFH\\\\\\\/F5g0hVZGbKMJefCY8PqVGZmlZh5HhcRkzT7GYGtvjpkdNcTLIVwAtBkUDuowQLx++HxqbWsjhXwG+peLIth9uNvhg68q+SbRqHZxcyRw9qx+Vf5vs22DxkOzXpIVDR2eFl2BuiyXllZEa9R\\\\\\\/oEgFcrO+9ZYQdH1dxlUJmMAU\\\\\\\/EzwhetFpuXhNcRAsx8qXLexbqzQmEpMG3TwwvCwj7cqvXiV89WhknbqCGKnSFR8A6E+xkwq59T3jfn2hmN2LGE7cEw2fHrOVgLVjmJiLWXQOzu8DkcQajCqKG\\\\\\\/9rkczsJA8oQHBnzpdqBsC\\\\\\\/+WMR23B8m7\\\\\\\/w4Vf8Uq12DC95CkZ3tpXJGLPyWnmc3+bPc\\\\\\\/AIehIodzhH2xrPmvEM0U\\\\\\\/kRh7wSzvXSG3e1FBuY\\\\\\\/4HpkJFP41BuNpkTYyJDz841ojVSBs\\\\\\\/kgnMBAAlz1ZRnQTchus4ejx+80vJjUzId0a+3kCl3yxsnYN4gghXcKmlC8nhPWZNLZQrJfkzZkR86o7308wQerKpOIjpK9R1FINFmZezZV1hz1qaH14Zduk05Ap\\\\\\\/SS55nT1EAGIfltPXw45MIMovnqsYWD3G8yNcZBncgOZsVA7YvUZTnJ+319Yy2o6rPuuFMd+pkPObhs54ohcZTf+eGEukxSXZUz5exllaxSL8l9p386Crjs+F+THlh9oDoR5yLjhOVLAAnAzDgxFck2FAsNgbtSmHsSpzMASXrCKCDQisGBjeTum3YiTCoh9KiD6CFYaGOy+5aiBRTpf8GD35M\\\\\\\/rCaqtZFP47f+2BNFE2ZGpVxniKjBUArAWnQgmvAr5L2HOx53CT1c5b97CV+CWSme2fGpqMrJI8GBvFyxn64JO4YLWyfmYlnMvWXNPIEYlfMz\\\\\\\/eC2bk3JKe62FWvxAPZBf+MCqYPWavUlT3gty04gWWfEqGHJKKN08FZvHDG4091G4ZjYHqb5RMt0NjC1jy2kWhOt7iALH8Km0UZR7SBXKTs7q4M1QgUVpkiXc0aBJerCtkdSHdeozT\\\\\\\/teZeMpPyeDIhhQIdw+5zcQdIDe\\\\\\\/uwnlYJX7tj8AAZmAR88traGQfLkbenXGXZlSxhuPODAhnYRPjkMYOSGLC9ELCbxHHyqRP0Ix2tJAyXTLp4G6eljQMjsow2wpqmkneFuR52IOjj8VolOEeE25dKit4ZruApdcBmL29tiF988lRzuEZYwFVyQqn0tUXjV8G\\\\\\\/aDKpcrAlpp9gBxVUSlk7jJxUhcykrL2fa2omQDfwiMKKO3Ul98goqwUj\\\\\\\/9EIGS4wz++5Sv6YHhRmWJ\\\\\\\/Fr11y3LsY32KfHaXpAorA0ecDhrTmmreiUyLjqn1fC3fr9QLz\\\\\\\/U8uU0k0THYDdgkUv4y3ejppeUbuL3ciM2hP6LUcYZjSaPZeZvjOzSz3nYuAz2UxcEECOX0tiEWLpWJAj3BYWDTC1PveLG\\\\\\\/rPZ+7YTzpAgxeHKaRPgYrVYfZGQOzHcLE13vTwWkO72xpUdFGMFWpEs4ez8ztiTPDmPFlq\\\\\\\/p3acnThEFgmdX\\\\\\\/oouWSU7GEKtJtORMOp6K062HVJ5QrieCM1SaHto5fhVWVyS7SZQLBtckJyChGGoBHyTZJet6+zT+K98FleostjSj4zokwd2a0CLsEG6FYI9txDnOG1NXblOGibCpVzjc9Pki0vboxqiqMq6bux2pppuitJdz44pXiT1zLLsi\\\\\\\/OueCA\\\\\\\/+NW4FeIWL97rl0rbpkZ3DC\\\\\\\/t3eR+syRnkqWDNpxRwBDqS1B3VIgx4+zOM\\\\\\\/YXtS1iN3FwnoXseKxBo3k30bn0D58pv8VuY7goNALBUgQ7qQ+EpU9yrmdkwzVS50XefBMKVqDkW1aAFpj7MCUBOfWzbRJ1VbNZ5U16+caxMHr3Nss2xgNXwbkVqHvjVALnR7JQgpMyXvHjorVJuKCIMHfYdXqbAlQJatBCQjTpUEZMLIBFNrfoetzK4ixuYOfI693aTBMrUzAGJ9TscNyiambicYy\\\\\\\/9WnJ6bN9uC5Yz4rpe+OfDSbQtNRba9QPCzQB0LuoCU2gUF\\\\\\\/pFBAVVCvrpb4r942Fb6M37so\\\\\\\/zzOTEYY4MAiRpBHpKTt2602WuK7M28JnVQ\\\\\\\/PBVD6hR8tUd7UHzg96PCTZLmwJOYEXyCpYWdov54bvwMOm71JstdYNCRW9au+vIIoRBz4\\\\\\\/IDCH4wjXd+tv7CPXqV+uktl1+IT6AjGbTYe1oJ00nrjZ8Y1wWCWLfzSjtNFCtsJSM9AqRThA+5sZz6RqlxEy0hOy2skUbpmy8DvG\\\\\\\/bUvSn0hOJ4SV9Hmn9D\\\\\\\/5UArXfDufX1Nu1Fk1mpRnPrCjXuuMfQXWqQRelPK6PFfBLvN1WUxr3ROTFlKIDwHMRG0qnbr4l8bsK\\\\\\\/t2d1oSz0+BkJoALO62\\\\\\\/QPjU6HR5F3WfpHovBUdYDHVQ+mnyy9gXUknC4edzU8RB0sDZsPqilX0n0fnkVWfIFqhYIzG8ThO3cgs3029dgZ4qLXjoyPZBwpl3klwlb4xsbnexzzZ85Pi2h58DdhnxwHAI2IJDKTOc1dEHEYLTkI18Bkzmiy6a5Ea963LbrOmw0RJaICC2Www7nnO8RYwuNeYVUHgy9md2ewR+9\\\\\\\/oH2Qh67EF3kTNTLsg04wMbB+Kh50vnaCEjpVtJYxnMmSKoTMHGOWKgGvKF90EgU0d9JAK4B3wtjTHT0BEEo9m+g302g77Xlwnc3aVQV4iIWmOS2JaP2KgYOCbpMoaEwi8IAa4aPg5HhgNuKGpiyphBai+9zOjn1L0IA72pX51ty9MkGUKqVJZSIebV84a\\\\\\\/OMIYXQ+fGQ4w3LeW8vCV1ir96DNmZhc+G9NzAP1I3owVOBFEodGSJExm4RsFxq4F\\\\\\\/kCGMGSv3PbN9QX6wSuzfKQYuHeLO2NwpplN8DoWwcUvQQDW4679i5VXPNwoYAA+VpUX274cFGwsDinQ3kW4t7H41d8yp8EiSXXPKef40zWpPlTdeFtUBgIjQ1qyoeOv539NybDkpkTjh0yrAq+YFrmfMcthlxatCOaOKZxGX+9R5ZsM4guDHffCLWz9cw71ICyYLoJzHDZ\\\\\\\/xf+qXghw5hhGueYNNpH5fEORcBU5IlabF6Lvd9CgCevG\\\\\\\/b+oipWjX7JeU6cEnY19fs+pIUfmkVlOvUFZ\\\\\\\/U9kh13uKo4T\\\\\\\/UGBKfE54Wej8E7jx2K6jEemLlQ4Hda0k9Wir2RL4byXvcIx6JSCsr9hPNPKXStk1syj70auohTRPLaU+wrs+dJjdPxgNugmGlDUhaCOgXoZ1MFojVhqc0qkQlM8MNy7xONgQFLbdPKMhntAzi8TlUIZEwQnYXoXWNSYKW3t6fvl+jEDPILlp7Q2BHcn3LPVdS3uuXwanPUhLheYz6UJj4x5fVIhTKt+0ODYGRVyI6tXfHkKjtkueaOVJ45b7UyLbmsUbNCBeHIGdZajfdQwdcBCe7WP9UB2NLORjprhETfcdjrddp6cKm9BcnnaB5VsEvucAQcDWBgCnNzQd0E997QKiClilhqBaWpcnus0Ynwny9ceXAWAiEzzd+8WNW1Vp5kTS7kV70IVxXqCH157BGV9DMdL+sdrZO8zhRebj+5Ma9n7xK6bZ3PD4ff0t9rdviFVb5O\\\\\\\/LaLWz9hMxM0MT1HG36J3TbEQxFu9ocnHEEl3TuNXrtsDuX2GqzRh05aFW1maRCM73eteLUy1MfYmNky8IGRfFbjMgW5xaiUdyUSUvepN494J\\\\\\\/HRrtgeLO3RqxRrZnkGK+eQo0sJkRCFJUFDDkJojVHTIcULSuR3fxMriRIBR6w4tpASdYV6et4cb4\\\\\\\/0breaFm9K\\\\\\\/xvFC3bLnTeiGAqVRgnl9TVZ1S4ROYW\\\\\\\/\\\\\\\/7FVDFlBC4vdEcWyvGnaAFQpTdOHIHg6E9TwcYAt\\\\\\\/lMEdZYI8E\\\\\\\/gUAYU7BVNDd3hnUeYaNLFjT38PEx\\\\\\\/CFBaqMEPz1CjLgK5\\\\\\\/HwyoU9Cv\\\\\\\/asYIPsVkzV+Cz\\\\\\\/bG0XfOtRm8UqvD6JEGn4SNgom6d0FEe5vicZwq5fAToZ8gYdvHrNffIZkOgQCL1o1vXavL\\\\\\\/KotNwO2jrzv+skxZRtbh9mMGrywJ7SIjVFdAp4mc4+prwWe2+bYWtTAyEFODPsM2ZR8wn9ZcFZEsNyzXuza5BmpcIFZS7xOZLnuJqv1mYzhoMOi1Yqwu1\\\\\\\/om\\\\\\\/RQl8FD2nlT1Xsy6D6dFSZGDXZfSwHldmHevkoeR7gJiytVoTTwZ6mHrDkQMS+CUfXVB0cQNniVmbY5cbGm0WQtD\\\\\\\/ppGpDp2NwSdfJQmX7Bs914+vsinkl37rJzcluwAMC28mVh\\\\\\\/+Ek+pu89329x3ibNe\\\\\\\/2g8hyGz7AI5+gXAqOw29L\\\\\\\/Ihedt+iPx9UhXnxt1fvuBxpDP05c2LsI\\\\\\\/Vzf4Q9b2b\\\\\\\/guhvNWrFowsao2rW5u2SEi4+6dUqrPs7DT4JQjUglh6\\\\\\\/tpnF+R6gtUfQFcwnwztj1xvqef2Ss23WL49ecB4f1FRzTeA0Ciuk\\\\\\\/+euko2fPOzOndhZzqqm2ID75EiwlA6XgYCW2hFJ3BilL7OTGL0OlRxw3de6bOn8qkaawrmZZpfMZIqZ932l5nWi0XEb9RHQF6S\\\\\\\/BcodQl31hEHU3vJj3kzAi3XtPAz5ZStOcoQV4CNxHckawJGhzWmcvQLsoepBOmqcz0gcpXV5WoF3yyzUF14ohaZWWl3bQ7B8r6B5esrRvJpHKgg0X2njr74fD293x7vjqEv6gizEJOvbpA2aujRF9KUqHtpQ5k8zXNYVPW1tyFJs2RyoZXE8ii+qr7O1QBYJal5fmuaTIRFIrZZt8XLAHq435Dp6GsIVXsSSpQB3O+jonk5RevHQe8zFVPX5I+zfkSNoLQtAHTsFeuXZxTHRZmiaJDuunJDJZRc3bm31DnibQ4TNVVwXNXth3BveRSaKDkKIzMD3SV7l7XMdjnqf2C9o2TJGx\\\\\\\/ypQaSNRXuPFRWIyQJRZ+8lkl3VgKPmHxRr791iQ14\\\\\\\/2RG5NsQv76S4BiqnwH43nYQGCRxBilLGboLBhUU41v43w9HBk41L+s\\\\\\\/TVIyQcm0HpCRvKhe2zDrQEvb5\\\\\\\/CVpFNGYG3oZtWoZyn6R\\\\\\\/UVTyiIYBf6ixXSrMYEz6oa1UtfiPM1ssg3ZASSAzhyljDBCWw2BhgMPWklfzVBuIwgIsxCPgDLlOSSkHPFwvbwNWk6NJHj5Of9Jmuu\\\\\\\/fT5gkTaBm7qDvevkjOTuxCiVTfNFF5\\\\\\\/8NnSsP\\\\\\\/5I94bChg3z0ygrWLkelG7k+\\\\\\\/8UFvYCQOXt2VER2B1pvp1aD1LcUiE79Od3PuuStFxFFeD7aEd+kJyjpOz7X3yeMj42x6mRNHhDatjg5Di8vQHFH6UpTCYxlIs5t4GfDiWaB39FDTCv2EW4rss5b9wXz5iSNy6jWyVDg7RuqWeBKAKxHFP0IyuPJgUVmiY5dKBVPrjoh1lD1ETCTGYpPDYEIK88RanBOXjM7nxW4Z+4OT2tsku\\\\\\\/SnKLaEgt0jJiiYAt5IPqTlnWD94mwqSeT9zKRV8nHhw\\\\\\\/q0LCRoptFWIT\\\\\\\/mvG2uqL9SwAIwtM44Mkb6GCVBhI8eHn4AEAHpaRNgbMJGvLRTGwmi3111Sw60+bm4GvyHmqwgE3uwPFcl0DAD4cAPuDwlHzZTiBfM+KSZMKdXZFhadlC9hcMSH8MmfKlh1NZu5LRBsGxI5mFL17B0GlhXZaF8NJKW8vamQiVhTO6BdW7wAUrDvBwfRZGHIJZ2UMxNJABQYQsTbj3TwHIQKBolj0oEWFpBSu1X88F0Ea0AdyOwfiBbf25GSkGS4d4TB236os1UYXm7AzQUm397EL33YMnMHIiKihr5IYjCahVaj3iKFGI4iqsBegtZITmr4aNAtOcC+WQHH5BrrifY4ullJNxqX\\\\\\\/tA6JwwZZCpcI32pOPBhecQB+eA60CzXXxAS1S12ZkSRA65FiytQRjIwIvdwlaf1XrYOBWVVBN4ntxIVeKdXfAQdZMbXfLlRqCL1BiMcuEp9IE0HVPulzfnwPznuy+aYuK9v96pasz1vNucR+RhiCJIJNNSjhE+7XKWYUSO3Nl9P0ZnHw\\\\\\\/KDIaaTORikTjRpPsiuWpd+NVsRj3RPL\\\\\\\/Tg8VIeGbtXzZ\\\\\\\/MlCfxDk+SEeOLRWRatDgbNoRG3\\\\\\\/UfhBh+dV4RkpmLh7Fh6GhX7M5Rqyod9bZY7ygiCpP\\\\\\\/9qsoJL2IC5NsLFlI2sXHfGyv944ThAhSBrLKclEGaaK+OxSdPz9OruTWqP\\\\\\\/g4WxUIE6SiKA94CQD25OsnjM4K01CyabXfIMy1LCMaTr\\\\\\\/w+hd9wGtBEr+DDi2c9ySQ29WYfP7YScvxUN2+YqGpm8m2fQvTywov39gyzECV1LqCwWK2PosZnhLE1c8DK5JdFhLcGZrQzkgxZ2hFCxOM+zzZANN78zsYNfPBTxjCY3I0pufC17hzZrBkLyVRSMLFLHR74LRFaT+CRl7Sl651iqtUraoWB6nELfQqkMQ0TITVCLTK76ToQsBeB3XdL5FQYQWf3YD0wzgDKFYz0wW8XzQj8UqHtXa4NxzaKKm1e4WKBMtRgMpRatUIHydVagwXSbSWCZ9KohJz0jrNEc3bPVynyTyqEbgAFpuXVcuJPwR13suHHo32NcGg2MNwBlveUJ10v29Hx5TUnxb\\\\\\\/\\\\\\\/74v8T+Q\\\\\\\/2VEtw\\\\\\\/wI+Pk1HJU4\\\\\\\/FM56I55gqUtgawNDMX6SRah5E6nbr37NFmPvji6pdWqtvzmM06ZRd8Go\\\\\\\/85kaXV7PYe1FEqla8IWdcL84aHDn7Y4+NiFxTjm2dNuT8IIlkhpffmcXU+fOyMMVjnm\\\\\\\/8LkVNAY7PnnaghFTruP96tleVcycosXE3zG2vT8pgdlIR72Ed2E4z491NoKY7d+IpLEMBiMvwT1FO9onUxWG70UtLRDTsSF\\\\\\\/vlHu8ZMAWLKbt2clruUKQHtWhOnIZye4beOb\\\\\\\/W8p9b1a\\\\\\\/AiEEnY9MvAwL+B81S8dndI9Sp8SiYlDjU\\\\\\\/LpslAOHlRU9C58qAIaMIhlCS0CPTUQPxYlIw67zGXQMjJyjmee2d9iHR\\\\\\\/\\\\\\\/P9DtMh6Q8r6LLcVgYlmCxBp+Sq3LiL9Utv4eIsdh0djT8mH885cUT7eObMJobIuRZhhOixPyvdHh6Dpd1akBiDhV9KcEvsZPlp92M7+iQP+mD1hj8lP+MNhCOug0oW2Uces6yiLG\\\\\\\/k0+0ONKoYNhL+gei49UqsEnCSToyhH3h+kxx6ua\\\\\\\/+Qosm2sIkqyccnt8Gnlxn80ikdPDTcJfmWdwxrDXgx9hfTz7lME4cn9A5GoAq3NsFCFdBATLfG+JRzW4oRbY9FisGcBsYJddFLFvqlsLxUR+ow7TRPgCBSdz9DRIU\\\\\\\/QhlsunnUvEfNFrW0ynXNf41o+Blj\\\\\\\/40M\\\\\\\/CoTy6lhd8p09WUwvvcukSwM5il9De56foTZV9ZiUsFn9SMybnLChWw0ZmgxV+B26w90i47BLpQxIH58XHgUe7X9L5SveHADS9vZ4ehvvfnxF1h9hOC\\\\\\\/96ecy2aqBAFgAOCG9UNpm\\\\\\\/Q8Rdv02X4wkGvBSIAHGmYKL9ChAsYPk+vRSOzzRjYvRPfCVcDZvcLD40bajXd9oLpA3GFP7DxqODDYNnjWYnsVJFmnHHT\\\\\\\/vJqNae36DCHhG3QSm8x6E5Gwpl5YGC0Q9UyIDMfh0VHNFPFmiXPXxJxILR0dQ+cOgbg8wk+gsLxzRWcnyqMFBGwnN4+2y6Pa\\\\\\\/WzCArVLRT0n5KXeGTXu1276ZJHWuf7\\\\\\\/25dtihOqFjMtNOR9\\\\\\\/qcEv4r3rxjhCMzoFCmj5NIIlNStcJywkkzsFpr4KYwATrcAcroGzlDmBc+AICN1fp1dpnUHX0dTBGU7JwP73sjz4GwD0\\\\\\\/\\\\\\\/7GBGyOpTuy13eNVbtBUpxSXP3ClzFcxXNxy4iqsXtCXmAML1IhRAlSJD\\\\\\\/mNM9+HJrOKH506szVAcPeN84726dMAOA42b\\\\\\\/MGQ95LmG3VW6mRTQRpRFrW5OZ75FB\\\\\\\/J8z8FeIqQzPgAZ4O\\\\\\\/qi9kL\\\\\\\/oXbZdGQGv0GiM3wQ+UuRBQBJkz7XiexQuR9QKFCWDHfTCzQ1l1GTlLSrha3VfdCDQLgQV1hMJx7qDqRnzSBlXbFZbIG8ARTyoUsqAorMpZlfIuBARsQFWK41cPc8G8m5ii7FiveEfL1ya6rg\\\\\\\/BU\\\\\\\/Rr71v5GB7piRLE7tZwCPqYmQjvqrbBPHjin4NbzPoXpxLdoc\\\\\\\/cpdzTvG7Hz0cD7DpGQAYbjSDxu6DgKERz8tWRl4le38\\\\\\\/94VbE5EcOCYx\\\\\\\/U61FVnHeszh8uOHraF8Z0wZJHqh3PVfyhCqqy6rGiKh2ww2YtUP2yci4MWgv3yMIb1sXf2qxKTGf0t9MbvknzrFaPC7PQ\\\\\\\/pCYgylqctnens9ykX0onAuatKCH9a004pAcTYLOOzT2k0CxzRKi\\\\\\\/9M8\\\\\\\/KrzzPhPo\\\\\\\/Ndzs0+cDBa58mtP5QE9dLSCki4VIYhiF3qrXO7MximjTUBKnt3ScJBTbMEC3ibN5uTq9WL\\\\\\\/\\\\\\\/ZW9grhCi2h2ToLyIIsRZgcpi8HGeDDj\\\\\\\/k4UIrjrhNtG+\\\\\\\/FhqBV3F9g52ySbetuyI+hUxvfKACqz\\\\\\\/jGAwyHqRNjR1v8ckhDUF7eRe78M24cKOGNyIpDOizx1gR3LDG1qyi\\\\\\\/xZFggsRIgrJ45onKmFNmYtxeylyLaP3cYyxqVnf9wgH0dmtO55PgfaMA0ol2zNv+WlhAs3w9r+i3zt8GaaurQ2u24+JeauxwNnxDh6lGZtlhYdXJJRtMBLvSSRq9M6FtiVyJo3A2kWkgq\\\\\\\/JmRFFuL7nXlkAtw6AkTqTK7OLLeKqCQEX5wN3eOgvlph1KDAeQnpZJ5HFycvPieuegddHNDOLhCAYOs9P5mzKGcZw93EZo2FFrE4A2AViTZGoVVq8IcRH4yJ9qjPm62E87fheC9nXEHWC4\\\\\\\/lb9u1CG\\\\\\\/S5n4lh4WCUEMsNHxm8AyrkurA69wEemTocvFThEDIyc1j3s45pRqHAYhbBL43em3uQzSdZjhKNOjDYijbXY0WiBZiYY980FnZNzpRluT5i4\\\\\\\/tdfzCpJycu\\\\\\\/JAB\\\\\\\/79Yc6ZXrhI5XACnsZCAeekR+WUus5mtztgUZpS5F4xHLhQ7n3QNzh99YdKG130qK9RYpOc0zpQTi96AnJL4Q0ShdewpGqjeotrEZNSrzOpVbrCXyYZTIy2aBd30IA2HzcPIJdyuU9SE2p4AFNE98oKIuQ7Mvr6G+A4vOrEkT\\\\\\\/nYxSJ\\\\\\\/SJDr6VGsFUCDLGi2qKLEbccUB9e+C6SyuiPp4DD7OdZEquOUuLI+ZgqYMs7xZg1+o6rDkP0BUeLSHRTEOukYr3PtW8MrxzmwJPVfS6+Gxy02Q8tESmodl0nYbh1DHmM+w7GvC5dUjnPFujhec+E9v45i3Kh0RBpvTibNcQjnBQg30BOORYFDVIv0pBt4K6RrnKOvVrUqSXojZP09RlFJrLcQIugnZcBIW6LYhhThjYY7JsvyReC1ZdebIXn5mzB81QoII1zvXpkG7JWwDXVfRPMjHYqEWfjSKmaC9f5dTsHxDouqoXWGv1uZvUwxE6XmdLIClNcM7b5ACkkiUdDLwGcF6ZACXw3mPSGu93n+QUpIqDk5XQid6+c5aXMvmIKI19hjFQTuWiFEb2VYxfqkXWoz8c+DMnEJsB\\\\\\\/z0rsL6nWzTdaMCDzjp1h3yJUazr7jjwEzglgJ6KQRv51M7Z+jbNDEgKMYURlau29D2LqYi6KJvxcmY5G9zbBpjLZ6hB++23O9F9PDyUJbtsXwcyNC58uK0BqaVFMaEKZMS2p4m1nxQ4jq+b4hGeI+2+bbl87qcQCfuCilqnC2OcuYphnCuLBPt1CROud1Moy26UHGNQUZppFezIUyCYR007SiIMUq+3eZM6HbzkBRdL+IGv3qJwVPqgKwk87S5+mLDe+AIoNmqtmIhIbzRkGKCI+HEvtoUe3EuI4WLNkb4wbeOP2Klf4WUHow\\\\\\\/58+7xMfXBIS7291qnwSkPZbbLpegtaFOlXy\\\\\\\/1xYy3uF5N7DjQP3e5d5aGZSIEFKt5yJXWqwpgZE8cjvJUjPDYSGHYhsC38Bhx4HPyoNx7njJqKBoINgXiP0+eWYO+ypFIItvqp+HfPe2S+F4s7OFCj975wDCz\\\\\\\/KqtB3DCO7wCrkFSjjRmHsKPj1mgEQHuX3SfM+0VeVeAc1AgXZBwBnoZz1toJiLsXVetMJCkpNaF9quSwzUklCEEAHljbOLhcUgzFT\\\\\\\/HlPmX24g33dJzOQQlZntiFrhNgpkTuxUzJjWoDiuAl4Wlt+h9bL4JNorLC0YD4qPxN9MykrDFjjtQnTWPg\\\\\\\/+Lfman\\\\\\\/KBKihzI6\\\\\\\/EtFgaY+w0bZZxAgvza6RvOhVW4wwCG8gXpOo2O\\\\\\\/cvVtpyDxtMzfFxQebPhdf07FgB28N2WLoqjUT\\\\\\\/7UGlf9mxik5v3EC51\\\\\\\/oLN1alMHOkqU9Zt\\\\\\\/pMxPmWiPQiXJchD\\\\\\\/Mad4+DquvwUpUtPU\\\\\\\/yXjZoXXIfkZNZ8pVrl66JJ80X0ucisK2MnYYb4i8w1BSoZbi5uCZMwWQ3FrIYvH\\\\\\\/wImYH96nNMb4C10e+ry+RVdC9My02WoS7qkKB\\\\\\\/O9EwcfepGvVv4lWfM\\\\\\\/2v6yiM7fYJJbAS7lGvBh\\\\\\\/BMpuunYY4QC5oxZQXAySqi0UbBU4tgcydCOxt27BwNfCN5JQNtY2hgbcWOzmzZ8uotXJ0P\\\\\\\/03ZdABW2PGJWlIG7sEMJwVD0d+WOBWEE3OgIqAqBV643drqE842+70CclAG\\\\\\\/pHSdOlZ2CSNXEq1iMnFBfVF4TFu+Pb6fIxQb4VyDDKPx2QQHMVIKTwMZ2N56ZST1Gmprm9lOfQocv3D2wrNJZtgkgAJVMdgWXAd3WVtWujounVdPX3xewKsUWc0+FKnzSPEhGA6yx\\\\\\\/te8oLQHyUhU25TaotxJ2DZODS80ZQ\\\\\\\/aqIq16LmIytJRq+17f8C+FODuyp1PmKDAPmz3+mDAmjZrlTGt54E+vHI\\\\\\\/i0vMEq4932OO\\\\\\\/vqles8RiDufbJVujvtGr+gJiF5RMgw+QrBOx9rarLG7TqKKQAgg3EoxG3zD\\\\\\\/8AIDjzxRRfdyugzz6z0p5ASxzUelj60T0kt0BKhH5uR5H+Fo8MMcgUxBwE8T2e4ec90Ss+iS9k8yhcy420N1JZxuS35T7Tk34FC7P9foTSKWY5YgQDDIOZCEoeTTqThY3jd9gMfn+Rz+QYzbK+0bWV46O2v89dXcKs\\\\\\\/mAAMgA7tyAMTkp0nuOWM2kubX7TiGhGMgulH+zwNNNZ4wvHYqyyDm\\\\\\\/IO1Q7iIEUoEnVdcNVhwlpo7w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vgkzJWYUBAMsd1KzLotKnQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223a85c85acd12cddceb5aa290b4dd6e77ef91e25a6bde300062437e01ae27a542\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022V5+SLgDn4K0n5u+mX78IRR34NzeocSs55kwFKa9xC4Q9hVdaBIlk2oFyvyX4H\\\\\\\/CtDD\\\\\\\/nUmYTPk\\\\\\\/KCZO1y2KMqHEwTL1DaiWWGuVVQTUfE9c4Y6m3ogt7QDzRi1LRvjtQhWRv61\\\\\\\/FA8Y94TtMOEbW+lCjPS2\\\\\\\/0m6mmUeh3KymMxoPPaZlEgKzZ8NfMLa+0dWhPpZzBd5VSX4N4r9t+7C+dA9J6\\\\\\\/G0yZA+O0Y\\\\\\\/qWU1mbj3wWhPccuR5\\\\\\\/3BOxfDgWL2AWJVV9cEtYVSKe4e37Til3v9cH1hhSIaf2+ajcvt9vEHuBCamnjrEZ\\\\\\\/Ag8NWziT8C701z6tp6KpdE9W+K+8H0avAbxAZfpPsSh9JiHp9jfhc1c0TY2ZJKps\\\\\\\/AfU8LL7\\\\\\\/24WMreCObu0ixqmbgi6+prFSWEQ8+LXXfoTuGc7oRvDhYcJBTN1ZTF2phURM503TL3v8262plIvdQwO1zYqHm3aNyQS3f18xkGvecUGmAAbV+VfCzm3Ti4GJE+0sLhOMJyRkhbhXMeCwCQX+wzQi\\\\\\\/cUzyU6OsgYMzITQxuKRe0mqyJ3VbsM8evSAjWLym56CHhgbm16NMveFFvd\\\\\\\/i75Wb\\\\\\\/lLIyUsNmnV1IAVjJ8USZ+MHMhXyBToLbR02QPpBhOCzPIKSBFf20Hvrj2ftuLGyKefSwZ9cP5cxnGmArECE4N9b5TtkT+DaEnUP\\\\\\\/RCHkXvjYUTH5cPA1FijzNm9n94G9iC7GUapeg+QpHYicSXbCmAswK4OT3VKWZi1Se8RBqui5gd9zlxqoRP3ILkf4X6JnYLmIzJxptBjNB40jIQ8pzqPzrOmzXMlknt70xaTALdtHcYa5xJ4\\\\\\\/bW1WR\\\\\\\/g85oxZsCR2cxNeeyU8KFP6j+1EFVyMhXUFAdG8C9MyaRrZhUAjzPLd\\\\\\\/Fx+PZsa1HenBPQSdqVjEV0csL\\\\\\\/hnBMlz784DDxeTdKuGXDOnOjgWb810jm60SpXY2sCd+slXwQ8IZMyJrORgwbEJVJZoQOf5xN84I5Q8HdZPWiuoThb5Z+xavs22kyihsiYXxOSeMKSd6dVx\\\\\\\/5UW0g764C2Y5zpBMVxLqpWPgo2Z+MHhXuJP9+hq6cQEUYDXQiQe6gxB01+96ZGsMXZaEiCkuDUKXmdChE3CyoJ1hqujIlHdn2biXTKAMs3I8oe9mwasTqAuQxs62X5ZaklOoEtrGv8dUnq4cB\\\\\\\/S+en6msqxWXMFYl3fhbZ+4ZC8j4qJwRPulKz3sb8TdQolD+jkKQeskvvOPPxGnwyhfChm+0vhL7De3GOTHSuavtnnoaRiOq7DjR0pSdhs0b41p7G5PSc0tPh31pwbhu9zdez+4oV79PF+9DQ9S60hUyxlOd92YpoNrr6ZFQBwfzaedIpn0b6njDu8jakgHb+wpNDabE+BxhPVzvt0YHdyYt8zdhq9v6tqdUpaniCJb6uUDUX4W2VtQzNqGJvyF69eAvBIJGslKp6+kdqTYkzxvNIknplrn6te5t9bJmbSXxu4WldyCJSZsAKW7CUCfomH9OJJu8bJRf1gxn2fVJWdYiCIgv4arliUrUHGDO9\\\\\\\/Nw0rDkGNQ5+CH60WIe2HkjCe8P8KaUscYqz4SRlEMCD1wRNSGshGjoOgwmvWx6Y7+jnotpSHRM7rh9nJ9OJgnzJRLBXjv1PYdxqRi5mB0Ob3+lXz7oOQMr9CfrE2qJt4ItCQISlzZeUEtPlK90eWspSHLm3iQNekQjorJhR6QTiqntp7PlsqaA6Rr9VAfb9YkvdPFVxiLjPZhcZHVNyBdC1L3jk4aDAq8u+Rz1XJtgy9d4+Dg+e1xVev80Z4GSZt6QDga\\\\\\\/4OVpRq7qG2WQopyQ950tC1oBC5M0THjLbYV55D2NBVUmRy\\\\\\\/gpMoZDD7b0wZaNSNwtQabtVyCTLl4rHEFvQQ9kp3tkfr00iuB381oMqKaylDUlh1AaVxPeA2N621o\\\\\\\/T2DyRif1sBaiXp2xU\\\\\\\/rnw99YSEOW\\\\\\\/Ma9tQ3KYywT0a9FxRgU+mCbeALI02yHD67w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Dyslke8Wn\\\\\\\/6Geh9dfWgBRw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221d493cf4236c2f7aad38f014c4118f2728e5bd8eb1997a83420fb2d8f7fdfa9d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493240 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1921" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-F-4CtEdl7yzdYmCEwGOKJDhYNivVCAeammh-LnOxevI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rXXaZUyw-p4GfDWgyO1RmUEaeJbOLd3aelyQ5DQ5Rh4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1909" title="Toward Spider Glue: Long Read Scaffolding for Extreme Length and Repetitious Silk Family Genes AgSp1 and AgSp2 with Insights into Functional Adaptation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1933" title="Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493240" id="node493240" data-pisa="ggg;9/6/1921" data-pisa-master="ggg;g3.119.400094" data-apath="/ggg/9/6/1921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9427-0153" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hari P. Poudel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9472-3167" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Millicent D. Sanciangco</span>, <span class="highwire-citation-author" data-delta="2">Shawn M. Kaeppler</span>, <span class="highwire-citation-author" data-delta="3">C. Robin Buell</span> and <span class="highwire-citation-author" data-delta="4">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1921-1931;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400094 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493240" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hari P. Poudel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hari%2BP.%2BPoudel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poudel%20HP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHari%2BP.%2BPoudel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9427-0153" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hari P. Poudel</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hpoudel@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">hpoudel@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Millicent D. Sanciangco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Millicent%2BD.%2BSanciangco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanciangco%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMillicent%2BD.%2BSanciangco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9472-3167" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Millicent D. Sanciangco</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shawn M. Kaeppler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BKaeppler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaeppler%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BM.%2BKaeppler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>United States Dairy Forage Research Center, USDA-ARS, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/6/1921" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493240" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1921.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493240" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1921.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493240" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/6/1921.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493240" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The lowland ecotype of switchgrass has generated considerable interest because of its higher biomass yield and late flowering characteristics compared to the upland ecotype. However, lowland ecotypes planted in northern latitudes exhibit very low winter survival. Implementation of genomic selection could potentially enhance switchgrass breeding for winter survival by reducing generation time while eliminating the dependence on weather. The objectives of this study were to assess the potential of genomic selection for winter survival in lowland switchgrass by combining multiple populations in the training set and applying the selected model in two independent testing datasets for validation. Marker data were generated using exome capture sequencing. Validation was conducted using (1) indirect indicators of winter adaptation based on geographic and climatic variables of accessions from different source locations and (2) winter survival estimates of the phenotype. The prediction accuracies were significantly higher when the training dataset comprising all populations was used in fivefold cross validation but its application was not useful in the independent validation dataset. Nevertheless, modeling for population heterogeneity improved the prediction accuracy to some extent but the genetic relationship between the training and validation populations was found to be more influential. The predicted winter survival of lowland switchgrass indicated latitudinal and longitudinal variability, with the northeast USA the region for most cold tolerant lowland populations. Our results suggested that GS could provide valuable opportunities for improving winter survival and accelerate the lowland switchgrass breeding programs toward the development of cold tolerant cultivars suitable for northern latitudes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/exome-capture" class="hw-term hw-article-keyword hw-article-keyword-exome-capture" rel="nofollow">exome capture</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/winter-survival" class="hw-term hw-article-keyword hw-article-keyword-winter-survival" rel="nofollow">winter survival</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Switchgrass (<em>Panicum virgatum</em> L.) is a North American native perennial warm season grass and a promising cellulosic bioenergy feedstock. The United States Department of Energy (US DOE) recognized switchgrass as a model bioenergy feedstock owing to its higher biomass production potential, perennial growth habit and stress tolerance (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Parrish and Fike 2005</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wright and Turhollow 2010</a>). There are two ecotypes of switchgrass, lowland (tetraploid) and upland (tetraploid and octoploid) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casler 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lee <em>et al.</em> 2014</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Parrish and Fike 2005</a>). The lowland ecotype has generated considerable interest due to its higher biomass yield and late flowering characteristics relative to the upland ecotype (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Casler <em>et al.</em> 2011</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wullschleger <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Poudel <em>et al.</em> 2019</a>).</p><p id="p-4">Lowland cultivars may have double the biomass yield of upland cultivars when planted in southern locations (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Casler 2012</a>). As such, breeding programs in the northern USA are focused on adapting southern lowland germplasm for use in the northern USA, where winter conditions are considerably colder than in the southern USA. Two traits form the focus of these breeding programs: late flowering to extend the growing season and winter survivorship to ensure stand longevity. Three strategies are being employed to accomplish this goal: (1) the use of upland x lowland hybrids (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Vogel <em>et al.</em> 2014</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Poudel <em>et al.</em> 2019</a>), (2) selection for late flowering within northern-adapted upland germplasm, and (3) selection for winter survivorship within late-flowering but unadapted southern lowland germplasm. The latter strategy is the focus of this research.</p><p id="p-5">Lowland ecotypes planted in the northern latitudes exhibit very low winter survival. Lowland cultivars, when planted one hardiness zone north of their origin, resulted in 9 to 17% reduction in both biomass yield and survival (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Casler <em>et al.</em> 2004</a>). While winter survival (WS) can be improved by selectively saving surviving plants exposed to low-temperature stress, this approach requires several years for the completion of a single selection cycle and its success is highly dependent on the presence of winter conditions that generate the appropriate selection pressure. Moreover, selection based on controlled environments does not always reflect selection under realistic field conditions, due to differential gene expression (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dhanaraj <em>et al.</em> 2007</a>). As such, the efficiency and effectiveness of controlled-environment selection may not represent the optimal solution. Genomic prediction and selection could potentially enhance switchgrass breeding for winter survival by reducing generation time (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lipka <em>et al.</em> 2014</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Simeão Resende <em>et al.</em> 2014</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wong and Bernardo 2008</a>) while eliminating the dependence on weather.</p><p id="p-6">Genomic selection (GS) is emerging as a powerful tool in animal and plant breeding (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Heffner <em>et al.</em> 2009</a>). Through dense genome-wide marker coverage to predict the breeding value of complex traits (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Meuwissen <em>et al.</em> 2001</a>), it facilitates rapid selection of superior genotypes and accelerates breeding cycles. Several GS models such as genomic-best linear unbiased predictor (G-BLUP)(<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hayes <em>et al.</em> 2009</a>), ridge regression-BLUP (RR-BLUP) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Endelman 2011</a>), reproducing kernel Hilbert space (RKHS) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gianola and van Kaam 2008</a>), Bayesian models (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Park and Casella 2008</a>) and random forest (RF) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Díaz-Uriarte and De Andres 2006</a>) along with their regularization are being used to optimize gains in accuracy. Despite the role of prediction procedures, the efficiency of GS models is primarily affected by heritability of the trait, effective population size of the training set, the relationship between individuals in training and testing sets, and marker density (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Calus <em>et al.</em> 2008</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Habier <em>et al.</em> 2013</a>).</p><p id="p-7">Genomic studies relating predictive ability by combining multiple populations into a single training set are limited. Increasing the size of the training set by combining populations, generally improves prediction accuracies because the markers effects are estimated from a larger number of phenotypes, assuming some degree of homogenous population structure (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lorenzana and Bernardo 2009</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">VanRaden 2008</a>). However, increasing size with unrelated individuals or with a heterogeneous population structure might lead to inconsistent linkage disequilibrium (LD) patterns between the marker and causal variants (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wientjes <em>et al.</em> 2013</a>) and thus might reduce prediction accuracy. The use of high-density markers in GS predominantly captures small-effect quantitative trait loci (QTL) and outperforms the breeding efficiency based on phenotype or marker-assisted selection. Marker density is a critical component of GS predictive ability because a higher number of markers increases the probability that the causal variant will be in LD with at least some markers across the population (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Calus <em>et al.</em> 2008</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Goddard 2009</a>). In a simulation study in cattle, combining populations had been beneficial in improving predictive ability at high marker density and was more accurate for traits with low heritability (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de Roos <em>et al.</em> 2009</a>).</p><p id="p-8">The assessment of GS in switchgrass had previously been performed by <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Lipka <em>et al.</em> (2014)</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ramstein <em>et al.</em> (2016)</a>, and <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Casler and Ramstein (2018)</a> for morphological and biomass quality traits, which all demonstrated the possibility of increased gain per unit time. However, GS for winter survival has not been reported. In this study, we examined the potential of GS for winter survival in lowland switchgrass by combining multiple populations in the training set. Specifically, we assessed the performance of genomic prediction procedures and applied the selected model to two independent datasets for validation purpose in two contexts: (1) using indirect indicators of winter adaptation based on geographic and climatic variables of accessions from different source locations and (2) using winter survival estimates of the phenotype.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Training population</h3><p id="p-9">This study used metadata on 405 half-sib (HS) families from five separate lowland switchgrass experimental nurseries (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) namely: Liberty (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ramstein <em>et al.</em> 2016</a>), WS11L, WS08L (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2018a</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Grabowski <em>et al.</em> 2017</a>), WS09L, and Expresso. These five experiments represented multiple breeding populations of lowland or hybrid switchgrass. All nurseries were planted using a row-plot design with three or four replicates per location either in 2011 or 2012. Plots consist of one row of five plants on a 30 cm spacing with 90 cm between rows. The experiments were evaluated in a randomized complete block design (RCBD) at two locations: Arlington, WI (Plano silt loam; fine-silty, mixed, superactive mesic Typic Argiudoll) and Hancock, WI (Plainfield loamy sand; mixed, mesic Typic Udipsamment) or Mead, NE (Sharpsburg silt loam fine, smectitic, mesic Typic Argiudoll). Liberty was evaluated in Arlington, WI and Mead, NE; whereas WS08L, WS09L, and Expresso were evaluated in Arlington and Hancock, WI. The WS11L experimental nursery was the only exception, planted in a completely randomized design only at Arlington, WI. Plots were harvested every year after the first killing frost and biomass was removed at the time of harvest. Plots were fertilized with 100 kg N ha<sup>-1</sup> in spring of each production year.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493301/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493301/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of switchgrass source populations and half-sib families used for genomic prediction of winter survival (training data sets)</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">Individual plants were scored for survival approximately 7-10 days after the initial spring growth for two evaluation years: Spring 2014 and Spring 2015. Scoring was done by visual assessment of the percentage of living shoots on the scale of 0-20, where 0 dead, 1 = one green shoot, ..., and 20 = no winter damage (Supplemental Table 1). The best linear unbiased predictor (BLUP) of each half-sib family was estimated for each experiment considering all effects as random and using the following linear model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-1.gif"/></noscript></span></span>where: <em>y<sub>ijkl</sub></em> is the response, <em>l<sub>i</sub></em>, <em>f<sub>j</sub></em> and <em>r<sub>k(i)</sub></em> are the effects of location <em>i</em>, family <em>j</em> and block <em>k</em> nested within location <em>i</em> respectively, and <em>ϵ<sub>ijkl</sub></em> are residuals. For WS11L, the following model was used:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-2.gif"/></noscript></span></span>where: <em>y<sub>jkl</sub></em> is the predicted response, <em>f<sub>j</sub></em> and <em>r<sub>k</sub></em> are the effect of family <em>j</em> and replication <em>k</em> respectively, and <em>ϵ<sub>jkl</sub></em> are residuals. In order to account for experiment-wise phenotypic variation, the phenotypic value for genomic analysis was set to be non-centered means such that<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-3.gif"/></noscript></span></span>.</p></div><div id="sec-3" class="subsection"><h3>Genotype and Quality Control</h3><p id="p-13">The maternal parent of each HS family was genotyped using the exome capture platform. Exome capture sequence reads were cleaned using Cutadapt (<a href="https://cutadapt.readthedocs.org/en/stable/">https://cutadapt.readthedocs.org/en/stable/</a>) and the FASTX toolkit (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>) and were aligned to the AP13 switchgrass reference genome (Version 1.1) using BWA-MEM (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li 2013</a>). Variants were called using the GATK package (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McKenna <em>et al.</em> 2010</a>) and copy number variants were called using CNVnator (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abyzov <em>et al.</em> 2011</a>). Genotype calling was implemented using the EM algorithm of <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Martin <em>et al.</em> (2010)</a>. Markers were filtered for the proportion of missing values lower than 20%; minor allele frequency greater than 1/2N, with N being the total number of individuals; and Hardy-Weinberg equilibrium (HWE) based on a chi-squared test p-value &gt; 10<sup>−4</sup>. Missing observations after filtering, less than 5% of the total, were imputed by the population mean.</p></div><div id="sec-4" class="subsection"><h3>Training (Calibration) Dataset</h3><p id="p-14">Of the 405 HS families, 368 HS families were used to develop GS models. Genotype data for 20 HS families were not available and 17 families were excluded from the analysis because genetically they were classified as the upland ecotype, as determined by structure analysis, or were classified as octoploid as demonstrated by the deviation in distribution of heterozygote biallelic SNP’s alternate allele frequencies from the mean value 0.5 to either 0.25 and/or 0.75.</p><p id="p-15">The resulting matrix contained markers at 435,919 polymorphic sites for 368 HS families. The HS families in this dataset were collected from different breeding programs and are subject to strong population structure (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, Supplemental Figure 1). A population structure based on Admixture (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2009</a>) was performed to identify potential population heterogeneity. Based on the results from the admixture analysis the training population was divided into three distinct population groups: Southern Lowland, Eastern Lowland, and Lowland x Upland, closely corresponding to the subdivisions shown earlier for the northern and southern association panels (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2018a</a>).</p><p id="p-16">Genomic heritability was calculated as the variance explained by the GBLUP model <em>i.e.</em>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-5.gif"/></noscript></span></span> is the variance of random effects of markers and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-6.gif"/></noscript></span></span>is the residual variance.</p><p id="p-17">Each panel was evaluated for prediction accuracy using five GS models. The standard statistical model for genomic prediction was genomic BLUP (GBLUP). The GBLUP model is defined as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-7.gif"/></noscript></span></span>where y is the HS family BLUP, µ is the grand mean, Z is the marker identity matrix; u ∼ N(0, Kσ<sup>2</sup><sub>u</sub>), K being the n x n genome estimated relationship matrix and e is the residual. The GBLUP model assumed additive, linear and homoscedastic effects of the markers and was fitted using rrBLUP (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Endelman 2011</a>) R package. In addition, Bayesian models BayesA (BA), BayesB (BB) and Bayesian lasso (BL) were considered (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Park and Casella 2008</a>). The Bayesian models were fitted using BGLR R package (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pérez and de los Campos 2014</a>) with 20000 iterations and 5000 burn-in. To account for heteroscedastic and nonlinearity, a machine learning method, Random Forest (RF), was implemented. The RF model was fitted with 200 decision trees and 1/6 of total markers as the number of variables to be randomly sampled as candidates at each split using R package randomForest (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Díaz-Uriarte and De Andres 2006</a>).</p><p id="p-18">The prediction accuracy in the training set was determined as the Pearson coefficient of correlation between the phenotypic BLUP and predicted GEBV as determined using a fivefold CV scheme (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Legarra <em>et al.</em> 2008</a>). In this scheme, the observations were randomly divided into five non-overlapping groups, four of which were used as a pseudo-training set and the remaining group was assigned as a testing set. The CV was replicated 20 times and the average was used as the predicted GEBV for GBLUP model and un-replicated for all other models. The GEBV generated from heteroscedastic and/or nonlinear models were compared to those from the GBLUP model using Dunnett’s paired <em>t</em>-test using R package multcomp (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hothorn <em>et al.</em> 2017</a>).</p></div><div id="sec-5" class="subsection"><h3>Validation Datasets</h3><p id="p-19">Several training models were evaluated for validation performance using two independent datasets: validation datasets #1 (VDS #1) and #2 (VDS #2) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">3</a>). All validations were conducted using three different training sets: All Populations (Southern Lowland, Eastern Lowland, and Lowland x Upland), All Lowland (Southern Lowland and Eastern Lowland), and Southern Lowland populations.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493344/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493344/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean winter survival percentage and family-mean broad sense heritability (H<sup>2</sup>) for five switchgrass populations and five population groups, based on field data from Spring 2014 or 2015 at Arlington and Hancock, WI</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493302/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493302/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Mean GBLUP prediction accuracies for five combinations of switchgrass populations based on winter survival in either spring 2014 or 2015</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Validation dataset #1 consisted of 253 individuals comprising 43 populations or accessions, which were defined as lowland based on structure analysis and were independent of the training set (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2018a</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Grabowski <em>et al.</em> 2017</a>). Accuracy of prediction was calculated as a Pearson’s coefficient of correlation between adjusted genome estimated breeding values (GEBV) and climatic or geographic variables: 30-yr mean minimum temperature of the coldest month (<a href="http://www.worldclim.org/">http://www.worldclim.org/</a>), latitude, and longitude of the site of origin of the source population (Supplemental Table 2). Adjusted GEBV was calculated by fitting a linear model to account for the effect of individuals within the population.</p><p id="p-23">This dataset is subject to strong population structure. Therefore, additionally, a mixed population matérn (MPM) model (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ramstein and Casler 2017</a>) was performed to account for population heterogeneity using the following linear model<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-8.gif"/></noscript></span></span>where: <em>y</em> is the n-vector of phenotype, <em>u = Zµ</em>, <em>Z</em> is marker matrix (n x m), ○ is the element-wise product, Ω<em>n</em> is n x n covariance matrix depicting population differentiation among individuals, <em>K</em> is the matérn kernel function with <em>v</em> and <em>h</em> as scale and shape parameters and (<em>P<sub>i</sub></em>, <em>P<sub>j</sub>)</em> marker estimated principal components (pc) for any pair of individuals (<em>i,j</em>). This model is denoted by GBLUP-MPM throughout this paper.</p><p id="p-24">GBLUP-MPM model is the extension of the GBLUP model which accounts for population heterogeneity using principal components calculated from marker data on the combined (calibration and validation) dataset. The number of principal components (d) to be used was determined based on the variance explained by each component and number of clusters revealed through the principal component plot. The optimization of <em>v</em> and <em>h</em> parameters in <em>Ωn</em> were estimated by maximizing the restricted likelihood of the above model using R function constrOptim, given the starting point for (<em>v<sub>o</sub></em>, <em>h<sub>0</sub></em>) as (0.5, 0.5). The Euclidean distance matrix (n x n covariance matrix) depicting population differentiation among individuals was estimated from <em>d</em>, <em>v</em>, and <em>h</em> using the matérn function in R.</p><p id="p-25">Validation dataset #2 was a subset of data, mostly consisting of natural populations originated from Southern USA (Supplemental Table 3) from a winter survival study in Wisconsin (unpublished data). Seedlings that were 16 weeks of age were transplanted in June 2015 at two locations: Arlington, WI and Madison, WI (Kegonsa silt loam; fine-silty over sandy or sandy-skeletal, mixed, superactive, mesic Mollic Hapludalf), in a randomized complete block design with eight blocks and 25 plants per plot. Plants were spaced 0.3 m apart within rows and rows were 0.9 m apart. The winter survival of individual plants was scored in Spring 2017, as described for the training set. A random set of 178 individuals representing 23 populations were genotyped as described for the training set (Supplemental Table 3).</p><p id="p-26">The winter survival phenotypic values for populations were estimated using the following random effect model.<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1921/embed/mml-math-9.gif"/></noscript></span></span>where: <em>y<sub>ijk</sub></em> is the predicted response, <em>p<sub>i</sub></em> and <em>b<sub>j</sub></em> are the effect of population <em>i</em> and block <em>j</em> respectively, and <em>ϵ<sub>ijk</sub></em> are residuals. The effect of location and location x population were significant (α &lt; 0.05) for winter survival, so locations were analyzed separately for prediction purposes.</p><p id="p-27">Two prediction procedures were used for this validation set. First, phenotypic winter survival scores for the exact same individuals genotyped in the VDS #2 (N = 178) were used to calculate prediction accuracy. Second, adjusted GEBV for populations were computed on a population basis independently for the Arlington and Madison locations. Adjusted GEBV and phenotypic BLUP values were then used to calculate prediction accuracies on a population basis for each location.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-28">Genotype and phenotype data of training set and validation dataset #2 and supplemental materials necessary for confirming the conclusions presented in the article are available at Figshare. Geno_TS.rds is the training dataset consisting of 368 individuals at 434,254 markers, geno_vds2.rds is the validation dataset #2 with 178 individuals at 460,822 markers position. Pheno_TS.csv and pheno_VDS2.csv are the corresponding phenotype data files for the training set and the validation data set #2.</p><p id="p-29">The genotype data used for validation dataset #1 can be accessed through Dryad Digital Repository under the name: snipe_slap_sapper_filtered_biallelic_snps_final_reheader.txt.bz2 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Evans <em>et al.</em> 2018b</a>) (<a href="https://doi.org/10.5061/dryad.mp6cp">https://doi.org/10.5061/dryad.mp6cp</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7336187">https://doi.org/10.25387/g3.7336187</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Phenotypic variability for winter survival</h3><p id="p-30">There was considerable variation in winter survival among populations, ranging from &lt;1% in Expresso to almost 50% in Liberty, a population derived from lowland x upland hybrids (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). As expected, HS families from Liberty had the highest mean winter survival in both years, likely due to the upland genome in their ancestry. Liberty had the lowest broad-sense heritability compared to the other populations, all of which had broad-sense heritability &gt;0.62. The lowland populations derived from southern USA germplasm possessed considerable phenotypic variability for winter survival, both among and within populations, the latter evidenced by the relatively high heritability values.</p><p id="p-31">The structure analysis of these genotypes, using the DNA marker matrix, resulted in a strong split of lowland individuals between an Eastern Lowland population and a Southern Lowland population (Supplemental Figure 1), which was similar to results of <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> (2018a)</a>. The Eastern Lowland population derives largely from New York and New Jersey (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2018a</a>), while the Southern Lowland population derives from the Carolinas, the Gulf Coast, and the southern Great Plains. Again, largely as expected, the Eastern Lowland population had a higher mean winter survival compared with the Southern Lowland population (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-9" class="subsection"><h3>Genomic structure and relatedness</h3><p id="p-32">The distribution of minor allele frequency (MAF) of the selected markers in the Eastern Lowland and Southern Lowland populations differed from that of the Lowland x Upland population. The 95<sup>th</sup> percentile of the markers in Lowland x Upland had MAF &lt; 0.39 while it was only 0.18 and 0.29 in Eastern Lowland and Southern Lowland, respectively (Supplemental Figure 2) indicating the proportion of alleles at or near fixation was higher in natural populations than the bi-parental hybrid population. Similarly, the pattern of LD decay in the Lowland x Upland population was slower than the other populations (Supplemental Figure 3) in accordance with the smaller effective population size of the Lowland x Upland population. The within-population identity-by-state (IBS) was highest in Eastern Lowland (0.35) followed by Lowland x Upland (0.17) and lowest in Southern Lowland (0.07). The among-population coefficients of IBS were nearly zero, which can be inferred from <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1921/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker-derived genomic relationships (identify-by-state, IBS) for 368 switchgrass half-sib families in the genomic prediction training set (bold bordered area) and Validation data set #2 (VDS #2). The label &#x201C;Population number&#x201D; on the y-axis refers to an arbitrary number assigned to 368 half-sib families of the training set plus 178 individuals in VDS #2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385320696" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Marker-derived genomic relationships (identify-by-state, IBS) for 368 switchgrass half-sib families in the genomic prediction training set (bold bordered area) and Validation data set #2 (VDS #2). The label &#x201C;Population number&#x201D; on the y-axis refers to an arbitrary number assigned to 368 half-sib families of the training set plus 178 individuals in VDS #2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1921/F1.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/6/1921/F1.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1921/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1921/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493382" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Marker-derived genomic relationships (identify-by-state, IBS) for 368 switchgrass half-sib families in the genomic prediction training set (bold bordered area) and Validation data set #2 (VDS #2). The label “Population number” on the y-axis refers to an arbitrary number assigned to 368 half-sib families of the training set plus 178 individuals in VDS #2.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Assessment of genomic prediction procedures</h3><p id="p-34">The assessment of genomic models was conducted separately for five population groups: All Populations, All Lowland, Southern Lowland, Eastern Lowland, and Lowland x Upland. Prediction accuracies for the GBLUP model were highly variable across population groups, ranging from 0.02 to 0.91 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Prediction accuracies were highest for the most diverse population groups with the greatest population size. Prediction accuracies were lowest for the Lowland x Upland population, probably due to the low heritability of winter survival in this population. None of the other genomic prediction models resulted in any improvement over the GBLUP model (data not shown).</p></div><div id="sec-11" class="subsection"><h3>Validation of genomic prediction models (VDS #1)</h3><p id="p-35">Validation data set #1 was composed of exclusively lowland populations, most of which were of southern origin (Supplemental Table 2). Therefore, it was of interest to determine which of three potential training sets provided the best predictions of GEBV from minimum temperature, latitude, or longitude of the site of origin for the populations in Supplemental Table 2. Correlation coefficients (r) of GEBV with minimum temperature and longitude were always negative and those with latitude were always positive, with the only exceptions occurring for the Southern Lowland population (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Predictions using GBLUP-MPM model were more accurate than GBLUP model as evidenced with increased |r| value by up to 0.19 while using models based on All Populations and the All Lowland population. The underrepresentation of Eastern-origin individuals in the Southern Lowland training set probably led to the inconsistency in prediction accuracies for that population.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493420/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493420/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Correlation coefficients between genomic estimated breeding values (GEBV) for winter survival (Spring 2014 or 2015) and climatic or geographic variables of switchgrass accessions comprising Validation data set #1</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">The highest and most consistent correlation coefficients across all variables, years, and methods were observed for the All Lowland population as the training set (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Multiple linear regressions of the GEBV obtained using this model with geographic variables latitude and longitude were performed to observe the relative contribution of each variable on winter survival. The minimum temperature was not included in the model because of collinearity (<em>r</em> = 0.88) with latitude. Longitude was statistically significant in all combination of years and methods (α &lt; 0.01) however latitude was not significant for the GBLUP method in 2015 (α &gt; 0.05, <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). However, the prediction method GBLUP-MPM improved the prediction such that both geographic variables were statistically significant (α &lt; 0.01) in both years. By accounting for population heterogeneity, the fitness of these models, as explained by adjusted R squared, increased by 0.11 and 0.14% for the 2014 and 2015 datasets, respectively. The map representing the origin of the populations with their adjusted GEBV using GBLUP-MPM method is presented in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. This result indicated that winter survival of lowland switchgrass populations had a linear relationship to the origin of the population <em>i.e.</em>, gradually increasing from south to north and from west to east.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493375/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493375/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Coefficients of multiple linear regression for regressions of adjusted genomic estimated breeding values (GEBV) predicted from latitude and longitude of the origin for all lowland switchgrass populations within Validation data set #1</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1921/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Adjusted genomic estimated breeding values (GEBV) of the populations in Validation data set #1 (VDS #1) plotted over the map of USDA hardiness zones. A. Predicted winter survival 2014 and B. Predicted winter survival 2015." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-385320696" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Adjusted genomic estimated breeding values (GEBV) of the populations in Validation data set #1 (VDS #1) plotted over the map of USDA hardiness zones. A. Predicted winter survival 2014 and B. Predicted winter survival 2015.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1921/F2.medium.gif" width="359" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/6/1921/F2.medium.gif" width="359" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1921/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1921/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493259" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Adjusted genomic estimated breeding values (GEBV) of the populations in Validation data set #1 (VDS #1) plotted over the map of USDA hardiness zones. A. Predicted winter survival 2014 and B. Predicted winter survival 2015.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Validation of genomic prediction models (VDS #2)</h3><p id="p-41">The predictive ability of models developed using training sets All Populations or All Lowland, when applied to all individuals of VDS #2 using the GBLUP individual prediction method, was less than 0.20 (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). The low accuracies for these validations were most likely affected by two factors: inclusion of unrelated individuals in the training set (the VDS #2 mostly had a southern origin) and possible inclusion of octoploids in the validation set (see discussion). The use of exclusive southern-origin populations in the training set (Southern Lowland) significantly increased the predictive ability of the genomic prediction model for all prediction methods. This increase in the prediction accuracy could be supported by the fact that the populations used in VDS #2 were mostly natural populations collected from the southern USA and had a closer resemblance to the individuals in the Southern Lowland population (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493378/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493378/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Prediction accuracy for winter survival of switchgrass populations in Validation data set #2. Predictions were based on GBLUP methods applied to training data sets from Spring 2014 or 2015 and applied to independent populations evaluated in 2017 at Arlington or Madison, WI</span><div class="sb-div caption-clear"></div></div></div><p id="p-43">For all combinations of the training set and GBLUP prediction methods, the prediction accuracies for the dataset consisting only putative tetraploids were higher than for the dataset consisting of all individuals for 2014 data, but not for 2015 data. Prediction accuracies based on Arlington data were always higher than for Madison data which is probably associated with stronger winter pressure at Arlington due to colder temperatures. Arlington was also one of the sites used for calibrating the training model.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-44">The winter of 2013/14 had consistently lower minimum temperatures throughout the winter than the 30-year normal minimum temperature (data not shown, <a href="https://www.ncdc.noaa.gov/cdo-web/">https://www.ncdc.noaa.gov/cdo-web/</a>), suggesting that relatively high selection pressures were likely exerted on these nurseries in the training set. This was evidenced by relatively high phenotypic variability between and within populations (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). The winter survivorship scores were highly correlated between evaluation years 2014 and 2015 (85%), as the effects from the second winter were largely additive to the evaluation from the first year. However, the data were analyzed separately for each year as a test of consistency in prediction and due to a significant genotype x year interaction.</p><p id="p-45">This study used meta-data from five experimental nurseries comprising multiple populations tested in multiple environments. Several studies have proposed that multiple-environment GS models can capture the variability due to the environment and increase the performance of genomic prediction (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cuevas <em>et al.</em> 2016</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lado <em>et al.</em> 2016</a>). The common strategies to deal with genotype x environment interaction (GxE) are to ignore, reduce or exploit it (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">DeLacy <em>et al.</em> 1996</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eisemann <em>et al.</em> 1990</a>). For each experiment in this study (except WS11L, which was evaluated in only one location), there was a significant GxE interaction. The strategy we employed here was to reduce the GxE interaction by modeling it during the estimation of phenotype (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lado <em>et al.</em> 2016</a>). We did not model to exploit the GxE in this study because the goal of this study was to determine the possibility of GS for winter survival and to use genomic prediction of population performance to develop more broadly adapted switchgrass populations across multiple locations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lal 2013</a>). Candidate switchgrass cultivars to be developed from this research will require fairly broad adaptation across hardiness zones 4-6 before they can be released for commercialization.</p><div id="sec-14" class="subsection"><h3>GS prediction procedures and validation</h3><p id="p-46">Most of the GS studies consider optimization of the genomic prediction procedures to improve the performance of prediction (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Moser <em>et al.</em> 2009</a>). There are several reported studies which used marker data transformation for GS. While marker data transformation by accounting for LD and weighing marker effects may be useful in reducing the effective number of markers (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Ramstein <em>et al.</em> 2016</a>), it did not significantly or consistently improve the prediction accuracy for dry matter yield in switchgrass. Similarly, higher marker density had an important role on the reliability of predictive ability while predicting across multiple populations because higher numbers of markers will increase the probability that the causal variant will be in LD with at least some markers across the populations (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Calus <em>et al.</em> 2008</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Goddard 2009</a>). In many studies, combining different groups of related populations as part of the training set has shown to increase prediction accuracy (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schulz-Streeck <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Technow <em>et al.</em> 2013</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Zhao <em>et al.</em> 2012</a>). Therefore, we incorporated all markers in our model regardless of marker effects and LD (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Ramstein <em>et al.</em> 2016</a>). The higher predictive ability of GBLUP over other models in this study could be because of the low LD (Supplemental Figure 3) among markers, supporting the results by <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhong <em>et al.</em> (2009)</a> who reported GBLUP models to perform better than Bayesian models when markers were in weak LD and the individuals were highly related.</p><p id="p-47">The model based on a training set that includes the Lowland x Upland population had lower predictive ability compared to the model based on the exclusively Lowland-ecotype training set when evaluated against the two independent-validation datasets (VDS #1 and VDS #2). This is likely due to the strong genetic dissimilarity of the Lowland x Upland population compared to the other populations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), owing to the differences in divergence between natural lowland populations and the selectively bred population that includes a significant dose of upland germplasm in its parentage (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhang <em>et al.</em> 2011a</a>). These results are similar to those of <a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Ramstein <em>et al.</em> (2016)</a>, who demonstrated low prediction accuracies between a diverse upland population and the Lowland x Upland population used in our study. Such low prediction accuracies might be due to the low degree of IBD between ecotypes (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Würschum <em>et al.</em> 2013</a>) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>) and differences in mean performance for the trait between training and validation sets (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Windhausen <em>et al.</em> 2012</a>).</p><p id="p-48">Switchgrass ecotypes, cultivars, and breeding populations are highly sensitive to GxE interactions. Latitude and photoperiod combine to form the most important factor in determining adaptation. In general, ecotypes or cultivars planted more than one hardiness zone north or south of their origin are frequently subject to significant reductions in both biomass yield and survival (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Casler <em>et al.</em> 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casler <em>et al.</em> 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hopkins <em>et al.</em> 1995</a>). Breeding for broad adaptation, including selection for winter survivorship under optimal conditions (<em>i.e.</em>, with favorable selection pressure) can alter this relationship by broadening the adaptation of some populations (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Casler <em>et al.</em> 2018</a>). Longitude is also a significant factor influencing GxE interactions of switchgrass, largely in the form of a moisture gradient from the Great Plains to the Atlantic Seaboard (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Casler <em>et al.</em> 2004</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Casler <em>et al.</em> 2019</a>). The prediction model based on all lowland populations was highly effective at predicting GEBV for winter survival in Wisconsin, based solely on latitude and longitude of the site-of-origin (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Tables 4</a> and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">5</a>). These results indicate that genomic prediction methods can be utilized to target germplasm exploration efforts within regions that possess the greatest potential for contributing favorable alleles to the breeding program. Previous germplasm exploration efforts have been focused vaguely and broadly on collecting lowland ecotypes from across the entire southern USA to capture the late flowering trait (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Casler 2012</a>).</p><p id="p-49">Using the validation results for VDS #1, future efforts can be directed to more specific sites and regions with a higher probability of containing alleles useful for improving freezing tolerance of northern-adapted lowland populations, as illustrated in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. Indeed, inspection of <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> suggests that genomic prediction may not be a necessary component for targeting switchgrass exploration and collection to broaden the allelic base for winter hardy lowland germplasm. Within the Great Plains, hardiness zone is the key component, with the most hardy germplasm coming from the most northern hardiness zones. Likewise, there is a clear trend within hardiness zones 8 and 9, that the most winter hardy germplasm tends to be more eastern in origin. Alleles for winter survivorship and genotypes capable of surviving Wisconsin winters appear to reside within nearly all investigated populations, at varying frequencies, but the more northern and eastern the origin, the higher the frequency of these alleles and genotypes.</p><p id="p-50">An additional application of these results will be in the utilization of genomic prediction procedures within the breeding program, <em>i.e.</em>, accelerating the breeding of freezing-tolerant lowland populations through genomic selection. For this application, the Southern Lowland prediction models appear to have the most applicability, as evidenced by the highest prediction accuracies likely due to the high mortality rates associated with populations of southern origin (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). The Eastern Lowland populations were not useful in this regard, due to their relatively low mortality in the field studies, owing to their northern and eastern origin (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Furthermore, these Eastern Lowland populations tend to be early to moderate in flowering date, as determined by their evolution at 40 to 42°N latitude.</p><p id="p-51">Previous studies of GxE interactions in switchgrass, demonstrating strong adaptational responses associated with both latitude and longitude, have been largely based on upland cultivars and ecotypes (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Casler <em>et al.</em> 2007</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hopkins <em>et al.</em> 1995</a>). This study expands upon those results, demonstrating that these adaptational responses also apply equally importantly to genotypes within the lowland ecotype of switchgrass. Furthermore, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> also demonstrates that most lowland accessions of switchgrass possess some alleles for winter survivorship in hardiness zones 4 and 5, as well as a small frequency of genotypes capable of surviving winters in these hardiness zones. Remnant genetic variability for winter survivorship within Southern Lowland populations is likely due to the relatively short period of time since the last glacial maximum when northern and southern populations, as well as upland and lowland ecotypes, were compressed together into a narrow adaptational zone (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhang <em>et al.</em> 2011b</a>). Hybridization between extreme genotypes was frequent during glacial maxima, essentially creating wide hybrids and new genotypes that were useful in repopulating prairie and savanna ecosystems during the past 11,000 years. The polyploid nature of switchgrass is likely an important factor, at least partly responsible for the remnant variability for extreme winter survivorship within these Southern Lowland populations.</p></div><div id="sec-15" class="subsection"><h3>Effect of the octoploid and tetraploid admixture in prediction accuracies</h3><p id="p-52">Flow cytometry of 11 populations within VDS #2 revealed an admixture of tetraploids and octoploids, with two out of eleven populations classified as octoploids. Genomic analysis was performed to determine the ploidy of all populations similar to that by <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lu <em>et al.</em> (2013)</a>, using allelic read depth counts to generate apparent segregation ratios. For each individual, the reference allele frequency at each marker position was calculated as a ratio of reference allele read depth to the total read depth. A distribution of the reference allele frequency of heterozygous markers (segregating in a 1:2:1 Mendelian ratio) was plotted. All tetraploids had a peak at allele frequency 0.5. Conversely, the individuals which were octoploid had a peak at allele frequency 0.75 (Supplemental Figure 4) (<a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2018a</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Grabowski <em>et al.</em> 2017</a>). The determination of ploidy on the remaining individuals was based on similarity of segregation ratios to known tetraploid types or octoploid types. Based on these observations, 40 individuals from five populations were predicted to be octoploid.</p><p id="p-53">Because there are very few reported lowland switchgrass that are octoploids (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zalapa <em>et al.</em> 2011</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Zhang <em>et al.</em> 2011a</a>), population structure analysis was required to identify the origin of these five populations. A combined population structure analysis of VDS #2 with the northern association panel (<a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">Evans <em>et al.</em> 2018a</a>) revealed a hybrid origin of these five populations (Supplemental figure 1B), similar to observations made by <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Zhang <em>et al.</em> (2011a)</a>. The five VDS #2 octoploids populations in this study originated from Alabama, Georgia, Mississippi and Tennessee, where most of the intermediate octoploids accessions originated, and where both upland and lowland genotypes are known to be adapted (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Zhang <em>et al.</em> 2011a</a>).</p><p id="p-54">The octoploid populations in this study had higher average predicted winter survivorship than tetraploids when the training set containing all lowland populations was used. This is likely due to their hybrid origin, containing significant upland genomic content, and unlikely due to ploidy <em>per se</em>. This result was in contrast to the phenotypic value recorded from the field evaluation and likely had a small adverse impact on predictive accuracy. Higher ploidy levels are generally associated with increased photosynthetic activity and greater stress tolerance in several crops (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Joseph <em>et al.</em> 1981</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Warner <em>et al.</em> 1987</a>), while some reports indicated a decrease in photosynthesis activity (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Austin <em>et al.</em> 1982</a>). However, in switchgrass, higher ploidy (hexaploid) was reported to have lower photosynthetic activity than tetraploids (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wullschleger <em>et al.</em> 1996</a>). Irrespective of the photosynthetic activity, the field evaluation of populations in VDS #2 demonstrated no differences in winter survivorship between octoploid and tetraploid populations that originated from the Southern USA. Due to the small differences in prediction accuracy between the full validation set and the tetraploid-only validation set (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>), it appears that these genomic prediction equations are relatively insensitive to ploidy. Rather, the geographic source of populations, reflecting their underlying genetic makeup, diversity, and linkage relationships, are the more important determinants of prediction accuracies within independent validation sets.</p><p id="p-55">The overall aim of this study was to assess the GS models for predicting winter survivorship in lowland switchgrass populations. The measurement of winter survival under field conditions is very challenging in switchgrass because it is highly dependent on the presence of winter conditions that generate appropriate selection pressure. This was clearly evident in the evaluation of the VDS #2 population, with experienced &lt;5% mortality following the winter of 2015/2016, compared to 50–75% following the winter of 2016/2017, even though the average minimum temperature during these two winters was similar. This limitation could be addressed to some extent by conducting selection in a controlled environment, but controlled environments do not often mimic real field conditions. Field grown plants are exposed to a varying light spectrum, light intensities, wind pressure, variable snow cover, and biotic stresses that are not present in controlled environments, resulting in differential gene expression (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dhanaraj <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gusta and Wisniewski 2013</a>). Furthermore, the performance of perennial grass species cannot be effectively evaluated in a single growing season, as evidenced by multiple short-term breeding cycles leading to relaxed selection pressure for winter survivorship in high-digestibility switchgrass populations (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Vogel <em>et al.</em> 2002</a>). In general switchgrass breeding requires 3 to 7 years for the completion of a single phenotypic selection cycle (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Casler 2012</a>) and field evaluations should occur over a minimum of three winters (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Casler <em>et al.</em> 2019</a>).</p><p id="p-56">Prediction accuracies, based on validations with independent populations in this study showed a high degree of predictive ability, within the range for which GS is thought to be beneficial for improving the efficiency of breeding programs (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Simeão Resende <em>et al.</em> 2014</a>). Results from the VDS #1 population indicate that it is possible to target sites for additional lowland switchgrass germplasm collection in the northern Great Plains, certain regions along the Gulf Coast, and in the northeastern USA. Similarly, the validation results from the VDS #2 population indicated that GS could be confidently used for predicting winter survivorship of individual genotypes within breeding populations, serving to both (1) accelerate the breeding program and (2) provide a basis for consistent selection pressure that does not necessarily depend on waiting for optimal selection conditions in the field.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-57">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7336187">https://doi.org/10.25387/g3.7336187</a>.</p></li><li class="fn" id="fn-10"><p id="p-58"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-14"><span class="received-label">Received </span>February 14, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-04-10"><span class="accepted-label">Accepted </span>April 10, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Poudel <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.1" data-doi="10.1101/gr.114876.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abyzov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">CNVnator: an approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">974</span>–<span class="cit-lpage">984</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.114876.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114876.110%26rft_id%253Dinfo%253Apmid%252F21324876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTc0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.2" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.3" data-doi="10.1093/aob/"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhagwat</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Flag leaf photosynthesis of Triticum aestivum and related diploid and tetraploid species.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">189</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.aob.a086238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DAUSTIN%26rft.auinit1%253DR.%2BB.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D177%26rft.epage%253D189%26rft.atitle%253DFlag%2BLeaf%2BPhotosynthesis%2Bof%2BTriticum%2Baestivum%2Band%2BRelated%2BDiploid%2Band%2BTetraploid%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252F%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.4" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype× environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.5" data-doi="10.1534/genetics.107.080838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Roos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of genomic selection using different methods to define haplotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.080838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D553%26rft.epage%253D561%26rft.atitle%253DAccuracy%2Bof%2BGenomic%2BSelection%2BUsing%2BDifferent%2BMethods%2Bto%2BDefine%2BHaplotypes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080838%26rft_id%253Dinfo%253Apmid%252F18202394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.6" data-doi="10.2135/cropsci2018.06.0409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sosa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonos</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Soil Quality and Region Influence Performance and Ranking of Switchgrass Genotypes.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">232</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2018.06.0409</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D59%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2018.06.0409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2018.06.0409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1921.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Switchgrass breeding, genetics, and genomics</span>, pp. <span class="cit-fpage">29</span>–<span class="cit-lpage">53</span> in <span class="cit-source"><em>Switchgrass</em>, <em>Green Energy and Technology</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Monti</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.8" data-doi="10.1007/s12155-017-9867-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramstein</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Breeding for biomass yield in switchgrass using surrogate measures of yield.</span> <abbr class="cit-jnl-abbrev">BioEnergy Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12155-017-9867-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEnergy%2BRes.%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-017-9867-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-017-9867-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.9" data-doi="10.3835/plantgenome2011.10.0026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The switchgrass genome: tools and strategies.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.10.0026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.10.0026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.10.0026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312661700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.10" data-doi="10.2135/cropsci2006.12.0780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taliaferro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehlke</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berdahl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Latitudinal and longitudinal adaptation of switchgrass populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">2249</span>–<span class="cit-lpage">2260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2006.12.0780</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.%2BD.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D2249%26rft.epage%253D2260%26rft.atitle%253DLatitudinal%2Band%2BLongitudinal%2BAdaptation%2Bof%2BSwitchgrass%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.12.0780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.12.0780&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000251409000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.11" data-doi="10.2135/cropsci2017.12.0729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adler</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">30 years of progress toward increased biomass yield of switchgrass and big bluestem.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1242</span>–<span class="cit-lpage">1254</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.12.0729</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D58%26rft.spage%253D1242%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.12.0729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.12.0729&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.12" data-doi="10.2135/cropsci2004.2930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taliaferro</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wynia</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Latitudinal adaptation of switchgrass populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2004.2930</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasler%26rft.auinit1%253DM.%2BD.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D293%26rft.epage%253D303%26rft.atitle%253DLatitudinal%2BAdaptation%2Bof%2BSwitchgrass%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.2930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.2930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187633900041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.13" data-doi="10.3835/plantgenome2016.03.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soberanis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of genotype× environment interaction kernel regression models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.03.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.03.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.03.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.14" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003608</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.15" data-doi="10.1534/genetics.109.104935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reliability of genomic predictions across multiple populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1545</span>–<span class="cit-lpage">1553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.104935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104935%26rft_id%253Dinfo%253Apmid%252F19822733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE1NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeLacy</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basford</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> Analysis of multi-environment trials–an historical perspective. <em>Plant Adaptation and Crop Improvement</em> 39124.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.17" data-doi="10.1007/s00425-006-0382-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dhanaraj</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkharouf</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beard</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chouikha</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Major differences observed in transcript profiles of blueberry during cold acclimation under field and cold room conditions.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">225</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00425-006-0382-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.stitle%253DPlanta%26rft.aulast%253DDhanaraj%26rft.auinit1%253DA.%2BL.%26rft.volume%253D225%26rft.issue%253D3%26rft.spage%253D735%26rft.epage%253D751%26rft.atitle%253DMajor%2Bdifferences%2Bobserved%2Bin%2Btranscript%2Bprofiles%2Bof%2Bblueberry%2Bduring%2Bcold%2Bacclimation%2Bunder%2Bfield%2Band%2Bcold%2Broom%2Bconditions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-006-0382-1%26rft_id%253Dinfo%253Apmid%252F16953429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-006-0382-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16953429&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243964000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.18" data-doi="10.1186/1471-2105-7-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Díaz-Uriarte</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Andres</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Gene selection and classification of microarray data using random forest.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-7-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDiaz-Uriarte%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DGene%2Bselection%2Band%2Bclassification%2Bof%2Bmicroarray%2Bdata%2Busing%2Brandom%2Bforest.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-3%26rft_id%253Dinfo%253Apmid%252F16398926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16398926&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1921.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisemann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Beyond the analytical methodology-better interpretation and exploitation of genotype-by-environment interaction in breeding</span>, pp. <span class="cit-fpage">108</span>–<span class="cit-lpage">117</span> in <span class="cit-source">Genotype-by-environment interaction and plant breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-publ-name">Louisiana State Univeristy</span>, <span class="cit-publ-loc">Baton Rouge</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.20" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.21" data-doi="10.3835/plantgenome2017.06.0055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanciangco</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisovan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Extensive Genetic Diversity is Present within North American Switchgrass Germplasm.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.06.0055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.06.0055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.06.0055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-web" id="cit-9.6.1921.22"><div class="cit-metadata"><cite>Evans, J., M. D. Sanciangco, K. H. Lau, E. Crisovan, K. Barry <em>et al.</em>, 2018b Data from: <em>Extensive genetic diversity is present within North American switchgrass germplasm</em>, Dryad Digital Repository<span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5061/dryad.mp6cp</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.23" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kaam</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.24" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-008-9308-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%26rft.volume%253D136%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DGenomic%2Bselection%253A%2Bprediction%2Bof%2Baccuracy%2Band%2Bmaximisation%2Bof%2Blong%2Bterm%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.25" data-doi="10.1111/nph.14101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabowski</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome‐wide associations with flowering time in switchgrass using exome‐capture sequencing data.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">169</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D213%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.26" data-doi="10.1111/j.1399-3054.2012.01611.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusta</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisniewski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Understanding plant cold hardiness: an opinion.</span> <abbr class="cit-jnl-abbrev">Physiol. Plant.</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1399-3054.2012.01611.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BPlant.%26rft.volume%253D147%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-3054.2012.01611.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1399-3054.2012.01611.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.27" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.28" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672308009981</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.29" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.30" data-doi="10.2135/cropsci1995.0011183X003500020047x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genotypic variability and genotype× environment interactions among switchgrass accessions from the Midwestern USA.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">571</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1995.0011183X003500020047x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHopkins%26rft.auinit1%253DA.%2BA.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D565%26rft.epage%253D571%26rft.atitle%253DGenotypic%2BVariability%2Band%2BGenotype%2Bx%2BEnvironment%2BInteractions%2Bamong%2BSwitchgrass%2BAccessions%2Bfrom%2Bthe%2BMidwestern%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1995.0011183X003500020047x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1995.0011183X003500020047x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QP34600047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-web" id="cit-9.6.1921.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hothorn</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bretz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westfall</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiberger</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuetzenmeister</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Package ‘multcomp’. Obtenido de <a href="http://cran.stat.sfu.ca/web/packages/multcomp/multcomp.pdf">http://cran.stat.sfu.ca/web/packages/multcomp/multcomp. pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.32" data-doi="10.1104/pp.68.4.894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Randall</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Photosynthesis in polyploid tall fescue: II. photosynthesis and ribulose-1, 5-bisphosphate carboxylase of polyploid tall fescue.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">894</span>–<span class="cit-lpage">898</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.68.4.894</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DJoseph%26rft.auinit1%253DM.%2BC.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D894%26rft.epage%253D898%26rft.atitle%253DPhotosynthesis%2Bin%2BPolyploid%2BTall%2BFescue%2B%253A%2BII.%2BPHOTOSYNTHESIS%2BAND%2BRIBULOSE-1%252C%2B5-BISPHOSPHATE%2BCARBOXYLASE%2BOF%2BPOLYPLOID%2BTALL%2BFESCUE%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.68.4.894%26rft_id%253Dinfo%253Apmid%252F16662021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo4OiI2OC80Lzg5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.33" data-doi="10.2135/cropsci2015.04.0207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrios</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quincke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Modeling genotype× environment interaction for genomic selection with unbalanced data from a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2165</span>–<span class="cit-lpage">2179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.04.0207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2165%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.04.0207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.04.0207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.34" data-doi="10.1016/j.indcrop.2013.07.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lal</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adaptability patterns and stable cultivar selection in menthol mint (Mentha arvensis L.).</span> <abbr class="cit-jnl-abbrev">Ind. Crops Prod.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">181</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.indcrop.2013.07.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInd.%2BCrops%2BProd.%26rft.volume%253D50%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.indcrop.2013.07.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.indcrop.2013.07.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1921.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voigt</span>  <span class="cit-name-given-names">T. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Switchgrass and giant Miscanthus agronomyin</span> <span class="cit-source">Engineering and Science of Biomass Feedstock Production and Provision</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>, <span class="cit-fpage">37</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.36" data-doi="10.1534/genetics.108.088575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert-Granié</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Performance of genomic selection in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">611</span>–<span class="cit-lpage">618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.088575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.088575%26rft_id%253Dinfo%253Apmid%252F18757934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-web" id="cit-9.6.1921.37"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint arXiv:1303.3997.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.38" data-doi="10.1371/journal.pone.0112227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accelerating the switchgrass (Panicum virgatum L.) breeding cycle using genomic selection approaches.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112227%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.39" data-doi="10.1007/s00122-009-1166-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenzana</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genotypic value predictions for marker-based selection in biparental plant populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-009-1166-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLorenzana%26rft.auinit1%253DR.%2BE.%26rft.volume%253D120%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D161%26rft.atitle%253DAccuracy%2Bof%2Bgenotypic%2Bvalue%2Bpredictions%2Bfor%2Bmarker-based%2Bselection%2Bin%2Bbiparental%2Bplant%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1166-3%26rft_id%253Dinfo%253Apmid%252F19841887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1166-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19841887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271939900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.40" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.41" data-doi="10.1093/bioinformatics/btq526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinnamon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuchner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeqEM: an adaptive genotype-calling approach for next-generation sequencing studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2803</span>–<span class="cit-lpage">2810</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq526%26rft_id%253Dinfo%253Apmid%252F20861027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20861027&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283919800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.42" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.43" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTkyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.44" data-doi="10.1186/1297-9686-41-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crump</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A comparison of five methods to predict genomic breeding values of dairy bulls from genome-wide SNP markers.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-41-56</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DMoser%26rft.auinit1%253DG.%26rft.volume%253D41%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DA%2Bcomparison%2Bof%2Bfive%2Bmethods%2Bto%2Bpredict%2Bgenomic%2Bbreeding%2Bvalues%2Bof%2Bdairy%2Bbulls%2Bfrom%2Bgenome-wide%2BSNP%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-56%26rft_id%253Dinfo%253Apmid%252F20043835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20043835&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.45" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The bayesian lasso.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214508000000337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.46" data-doi="10.1080/07352680500316433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parrish</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fike</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The biology and agronomy of switchgrass for biofuels.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">459</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07352680500316433</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D24%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07352680500316433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07352680500316433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234512400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Genome-wide regression &amp; prediction with the BGLR statistical package. <em>Genetics</em>:genetics. 114.164442. <a href="https://doi.org/10.1534/genetics.114.164442">https://doi.org/10.1534/genetics.114.164442</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.48" data-doi="10.3389/fpls.2019.00372"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poudel</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanciangco</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Quantitative Trait Loci for Freezing Tolerance in a Lowland x Upland Switchgrass Population.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">372</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2019.00372</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D10%26rft.spage%253D372%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2019.00372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2019.00372&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-web" id="cit-9.6.1921.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramstein</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Extensions of blup models for genomic prediction in heterogeneous populations: Application in a diverse switchgrass sample.</span> <span class="cit-source">bioRxiv</span> <span class="cit-fpage">124081</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.50" data-doi="10.1534/g3.115.024950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramstein</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Accuracy of genomic prediction in switchgrass (Panicum virgatum L.) improved by accounting for linkage disequilibrium. <em>G3: Genes</em>, <em>Genomes</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1049</span>–<span class="cit-lpage">1062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.024950</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D6%26rft.spage%253D1049%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.024950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.115.024950&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.51" data-doi="10.2135/cropsci2012.03.0160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karaman</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection using multiple populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">2453</span>–<span class="cit-lpage">2461</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.03.0160</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D2453%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.03.0160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.03.0160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.52" data-doi="10.2135/cropsci2013.05.0353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simeão Resende</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilela de Resende</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection in forage breeding: accuracy and methods.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.05.0353</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.05.0353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.05.0353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.53" data-doi="10.1534/g3.112.004630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bürger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of northern corn leaf blight resistance in maize with combined or separated training sets for heterotic groups. <em>G3: Genes</em>, <em>Genomes</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004630</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D3%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.54" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.55" data-doi="10.2135/cropsci2002.1857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopkins</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">survival in switchgrass populations bred for high IVDMD.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1857</span>–<span class="cit-lpage">1862</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2002.1857</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DVogel%26rft.auinit1%253DK.%2BP.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D1857%26rft.epage%253D1862%26rft.atitle%253DWinter%2BSurvival%2Bin%2BSwitchgrass%2BPopulations%2BBred%2Bfor%2BHigh%2BIVDMD%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2002.1857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2002.1857&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000181430200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.56" data-doi="10.3198/jpr2013.12.0076crc"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarath</span>  <span class="cit-name-given-names">G. J. J. P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Registration of ‘Liberty’switchgrass.</span> <abbr class="cit-jnl-abbrev">J. Plant Regist.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">242</span>–<span class="cit-lpage">247</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3198/jpr2013.12.0076crc</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegist.%26rft.volume%253D8%26rft.spage%253D242%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2013.12.0076crc%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2013.12.0076crc&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.57" data-doi="10.1104/pp.84.2.461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Photosynthesis, leaf anatomy, and cellular constituents in the polyploid C4 grass Panicum virgatum.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.84.2.461</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DWarner%26rft.auinit1%253DD.%2BA.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D461%26rft.epage%253D466%26rft.atitle%253DPhotosynthesis%252C%2BLeaf%2BAnatomy%252C%2Band%2BCellular%2BConstituents%2Bin%2Bthe%2BPolyploid%2BC4%2BGrass%2BPanicum%2Bvirgatum%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.84.2.461%26rft_id%253Dinfo%253Apmid%252F16665462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo4OiI4NC8yLzQ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xOTIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.58" data-doi="10.1534/genetics.112.146290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wientjes</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of linkage disequilibrium and family relationships on the reliability of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.146290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146290%26rft_id%253Dinfo%253Apmid%252F23267052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNjIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.59" data-doi="10.1534/g3.112.003699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. <em>G3: Genes</em>, <em>Genomes</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.003699</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D1427%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.003699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.60" data-doi="10.1007/s00122-008-0715-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-008-0715-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWong%26rft.auinit1%253DC.%2BK.%26rft.volume%253D116%26rft.issue%253D6%26rft.spage%253D815%26rft.epage%253D824%26rft.atitle%253DGenomewide%2Bselection%2Bin%2Boil%2Bpalm%253A%2Bincreasing%2Bselection%2Bgain%2Bper%2Bunit%2Btime%2Band%2Bcost%2Bwith%2Bsmall%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0715-5%26rft_id%253Dinfo%253Apmid%252F18219476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0715-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18219476&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254262600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.61" data-doi="10.1016/j.biombioe.2010.01.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turhollow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Switchgrass selection as a “model” bioenergy crop: a history of the process.</span> <abbr class="cit-jnl-abbrev">Biomass Bioenergy</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">868</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.biombioe.2010.01.030</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiomass%2BBioenergy%26rft.volume%253D34%26rft.spage%253D851%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biombioe.2010.01.030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biombioe.2010.01.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000278356500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.62" data-doi="10.2134/agronj2010.0087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borsuk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunderson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynd</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Biomass production in switchgrass across the United States: database description and determinants of yield.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1158</span>–<span class="cit-lpage">1168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2010.0087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DWullschleger%26rft.auinit1%253DS.%2BD.%26rft.volume%253D102%26rft.issue%253D4%26rft.spage%253D1158%26rft.epage%253D1168%26rft.atitle%253DBiomass%2BProduction%2Bin%2BSwitchgrass%2Bacross%2Bthe%2BUnited%2BStates%253A%2BDatabase%2BDescription%2Band%2BDeterminants%2Bof%2BYield%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2010.0087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2010.0087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.63" data-doi="10.2135/cropsci1996.0011183X003600020016x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLaughlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biradar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rayburn</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Photosynthetic rates and ploidy levels among populations of switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1996.0011183X003600020016x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWullschleger%26rft.auinit1%253DS.%2BD.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D306%26rft.epage%253D312%26rft.atitle%253DPhotosynthetic%2BRates%2Band%2BPloidy%2BLevels%2Bamong%2BPopulations%2Bof%2BSwitchgrass%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1996.0011183X003600020016x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1996.0011183X003600020016x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.64" data-doi="10.1186/1471-2156-14-85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraft</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection in sugar beet breeding populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-14-85</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-14-85%26rft_id%253Dinfo%253Apmid%252F24047500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-14-85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24047500&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.65" data-doi="10.1007/s00122-010-1488-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zalapa</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobias</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hierarchical classification of switchgrass genotypes using SSR and chloroplast sequences: ecotypes, ploidies, gene pools, and cultivars.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">805</span>–<span class="cit-lpage">817</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-010-1488-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZalapa%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D122%26rft.issue%253D4%26rft.spage%253D805%26rft.epage%253D817%26rft.atitle%253DHierarchical%2Bclassification%2Bof%2Bswitchgrass%2Bgenotypes%2Busing%2BSSR%2Band%2Bchloroplast%2Bsequences%253A%2Becotypes%252C%2Bploidies%252C%2Bgene%2Bpools%252C%2Band%2Bcultivars.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1488-1%26rft_id%253Dinfo%253Apmid%252F21104398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1488-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21104398&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.66" data-doi="10.2135/cropsci2011.02.0104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalapa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakubowski</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Natural hybrids and gene flow between upland and lowland switchgrass.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2626</span>–<span class="cit-lpage">2641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.02.0104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2626%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.02.0104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.02.0104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.67" data-doi="10.1007/s10709-011-9597-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalapa</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakubowski</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Post-glacial evolution of Panicum virgatum: centers of diversity and gene pools revealed by SSR markers and cpDNA sequences.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">933</span>–<span class="cit-lpage">948</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10709-011-9597-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenetica%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D139%26rft.issue%253D7%26rft.spage%253D933%26rft.epage%253D948%26rft.atitle%253DPost-glacial%2Bevolution%2Bof%2BPanicum%2Bvirgatum%253A%2Bcenters%2Bof%2Bdiversity%2Band%2Bgene%2Bpools%2Brevealed%2Bby%2BSSR%2Bmarkers%2Band%2BcpDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-011-9597-6%26rft_id%253Dinfo%253Apmid%252F21786028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-011-9597-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21786028&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.68" data-doi="10.1007/s00122-011-1745-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Würschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-011-1745-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D769%26rft.epage%253D776%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2BEuropean%2Bmaize%2Belite%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1745-y%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1745-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1921.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1921.69" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: A barley case study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.098277</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/6/1921.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1909" title="Toward Spider Glue: Long Read Scaffolding for Extreme Length and Repetitious Silk Family Genes AgSp1 and AgSp2 with Insights into Functional Adaptation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1933" title="Tissue-Specific Split sfGFP System for Streamlined Expression of GFP Tagged Proteins in the Caenorhabditis elegans Germline" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/6"><span>Volume 9 Issue 6, June 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/6.cover-source.jpg?itok=Ub13TpI0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1921" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rlnzNhnd5rwl_wkaFvH6Jh7lO9rmU5GyX11Mpb9rlwI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/6/1921" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/6/1921" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/6/1921" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493240" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-05sqja_SjJrR6zE9tNvJpKuuA1qI4WA9oZHtQRu9FpI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1921.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/6/1921" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-_LlhIcfXF6EvsKYYDr0ho4fTc0TjzBJ9rCnq1e0IWEg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/6/1921" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1921.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493240" id="node493240--2" data-pisa="ggg;9/6/1921" data-pisa-master="ggg;g3.119.400094" data-apath="/ggg/9/6/1921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9427-0153" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hari P. Poudel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9472-3167" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Millicent D. Sanciangco</span>, <span class="highwire-citation-author" data-delta="2">Shawn M. Kaeppler</span>, <span class="highwire-citation-author" data-delta="3">C. Robin Buell</span> and <span class="highwire-citation-author" data-delta="4">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1921-1931;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400094 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493240--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hari P. Poudel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hari%2BP.%2BPoudel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poudel%20HP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHari%2BP.%2BPoudel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9427-0153" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hari P. Poudel</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hpoudel@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">hpoudel@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Millicent D. Sanciangco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Millicent%2BD.%2BSanciangco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanciangco%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMillicent%2BD.%2BSanciangco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9472-3167" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Millicent D. Sanciangco</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shawn M. Kaeppler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BKaeppler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaeppler%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BM.%2BKaeppler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>United States Dairy Forage Research Center, USDA-ARS, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/6/1921" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/6/1921&amp;title=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/6/1921&amp;title=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/6/1921&amp;title=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/6/1921&amp;text=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/6/1921&amp;title=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/6/1921&amp;t=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/6/1921&amp;title=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/6/1921&amp;title=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493240/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493240" id="node493240--3" data-pisa="ggg;9/6/1921" data-pisa-master="ggg;g3.119.400094" data-apath="/ggg/9/6/1921.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction for Winter Survival of Lowland Switchgrass in the Northern USA</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9427-0153" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Hari P. Poudel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9472-3167" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Millicent D. Sanciangco</span>, <span class="highwire-citation-author" data-delta="2">Shawn M. Kaeppler</span>, <span class="highwire-citation-author" data-delta="3">C. Robin Buell</span> and <span class="highwire-citation-author" data-delta="4">Michael D. Casler</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1921-1931;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400094 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493240--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hari P. Poudel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hari%2BP.%2BPoudel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poudel%20HP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHari%2BP.%2BPoudel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9427-0153" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hari P. Poudel</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hpoudel@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">hpoudel@wisc.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Millicent D. Sanciangco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Great Lakes Bioenergy Research Center, Michigan State University, East Lansing, MI 48824</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Millicent%2BD.%2BSanciangco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sanciangco%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMillicent%2BD.%2BSanciangco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9472-3167" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Millicent D. Sanciangco</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Shawn M. Kaeppler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shawn%2BM.%2BKaeppler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kaeppler%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShawn%2BM.%2BKaeppler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">C. Robin Buell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, Plant Resilience Institute, and MSU AgBioResearch, Michigan State University, East Lansing, MI 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BRobin%2BBuell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buell%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BRobin%2BBuell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michael D. Casler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>United States Dairy Forage Research Center, USDA-ARS, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BD.%2BCasler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Casler%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BD.%2BCasler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1921&amp;rft.epage=1931&amp;rft.atitle=Genomic%20Prediction%20for%20Winter%20Survival%20of%20Lowland%20Switchgrass%20in%20the%20Northern%20USA&amp;rft.volume=9&amp;rft.issue=6&amp;rft.date=2019-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Poudel%2C+Hari+P.&amp;rft.au=Sanciangco%2C+Millicent+D.&amp;rft.au=Kaeppler%2C+Shawn+M.&amp;rft.au=Buell%2C+C.+Robin&amp;rft.au=Casler%2C+Michael+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493240/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493240/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493240/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493240/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493240/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493240/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493240/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493240/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493240/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493240/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493240/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493240/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/6/1921" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/6/1921.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/6/1921.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/6/1921.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
