<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/6/1825" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/6/1825.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/6/1825.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/6/1825.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/6/1825.atom" />
<meta name="HW.pisa" content="ggg;9/6/1825" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data" />
<meta name="DC.Identifier" content="10.1534/g3.119.400273" />
<meta name="DC.Date" content="2019-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Routh. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Poly(A)-tail targeted RNAseq approaches, such as 3′READS, PAS-Seq and Poly(A)-ClickSeq, are becoming popular alternatives to random-primed RNAseq to focus sequencing reads just to the 3′ ends of polyadenylated RNAs to identify poly(A)-sites and characterize changes in their usage. Additionally, we and others have demonstrated that these approaches perform similarly to other RNAseq strategies for differential gene expression analysis, while saving on the volume of sequencing data required and providing a simpler library synthesis strategy. Here, we present DPAC ( D ifferential P oly( A )- C lustering); a streamlined pipeline for the preprocessing of poly(A)-tail targeted RNAseq data, mapping of poly(A)-sites, poly(A)-site clustering and annotation, and determination of differential poly(A)-cluster usage using DESeq2. Changes in poly(A)-cluster usage is simultaneously used to report differential gene expression, differential terminal exon usage and alternative polyadenylation (APA)." />
<meta name="DC.Contributor" content="Andrew Routh" />
<meta name="article:published_time" content="2019-06-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Poly(A)-tail targeted RNAseq approaches, such as 3′READS, PAS-Seq and Poly(A)-ClickSeq, are becoming popular alternatives to random-primed RNAseq to focus sequencing reads just to the 3′ ends of polyadenylated RNAs to identify poly(A)-sites and characterize changes in their usage. Additionally, we and others have demonstrated that these approaches perform similarly to other RNAseq strategies for differential gene expression analysis, while saving on the volume of sequencing data required and providing a simpler library synthesis strategy. Here, we present DPAC (&lt;b&gt;&lt;i&gt;D&lt;/i&gt;&lt;/b&gt;ifferential &lt;b&gt;&lt;i&gt;P&lt;/i&gt;&lt;/b&gt;oly(&lt;b&gt;&lt;i&gt;A&lt;/i&gt;&lt;/b&gt;)-&lt;b&gt;&lt;i&gt;C&lt;/i&gt;&lt;/b&gt;lustering); a streamlined pipeline for the preprocessing of poly(A)-tail targeted RNAseq data, mapping of poly(A)-sites, poly(A)-site clustering and annotation, and determination of differential poly(A)-cluster usage using DESeq2. Changes in poly(A)-cluster usage is simultaneously used to report differential gene expression, differential terminal exon usage and alternative polyadenylation (APA).&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/06/01" />
<meta name="citation_mjid" content="ggg;9/6/1825" />
<meta name="citation_id" content="9/6/1825" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/6/1825" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/6/1825.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/6/1825.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/6/1825.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400273" />
<meta name="citation_pmid" content="31023725" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="1825" />
<meta name="citation_lastpage" content="1830" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrew Routh" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology" />
<meta name="citation_author_institution" content="Sealy Centre for Structural Biology and Molecular Biophysics, University of Texas Medical Branch, Galveston, Texas, 77555" />
<meta name="citation_author_email" content="alrouth@utmb.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2874-5990" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Arefeen;citation_author=J. Liu;citation_author=X. Xiao;citation_author=T. Jiang;citation_title=TAPAS: tool for alternative polyadenylation site analysis.;citation_pages=2521-2529;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. W. Chang;citation_author=W. Zhang;citation_author=H. S. Yeh;citation_author=M. Park;citation_author=C. Yao;citation_title=An integrative model for alternative polyadenylation, IntMAP, delineates mTOR-modulated endoplasmic reticulum stress response.;citation_pages=5996-6008;citation_volume=46;citation_year=2018;citation_doi=10.1093/nar/gky340" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Chen;citation_author=Y. Zhou;citation_author=Y. Chen;citation_author=J. Gu;citation_title=fastp: an ultra-fast all-in-one FASTQ preprocessor.;citation_pages=i884-i890;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty560" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_author=Y. Chu;citation_author=N. Elrod;citation_author=C. Wang;citation_author=L. Li;citation_author=T. Chen;citation_title=Nudt21 regulates the alternative polyadenylation of Pak1 and is predictive in the prognosis of glioblastoma patients.;citation_year=2019;citation_doi=10.1038/s41388-019-0714-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Derti;citation_author=P. Garrett-Engele;citation_author=K. D. Macisaac;citation_author=R. C. Stevens;citation_author=S. Sriram;citation_title=A quantitative atlas of polyadenylation in five mammals.;citation_volume=22;citation_year=2012;citation_pmid=22454233;citation_doi=10.1101/gr.132563.111" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=N. R. Elrod;citation_author=E. A. Jaworski;citation_author=P. Ji;citation_author=E. J. Wagner;citation_author=A. Routh;citation_title=Development of Poly(A)-ClickSeq as a Tool Enabling Simultaneous Genome-wide Poly(A)-site identification and Differential Expression Analysis.;citation_pages=20-29;citation_volume=155;citation_year=2019;citation_doi=10.1016/j.ymeth.2019.01.002" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=N. Ignatiadis;citation_author=B. Klaus;citation_author=J. B. Zaugg;citation_author=W. Huber;citation_title=Data-driven hypothesis weighting increases detection power in genome-scale multiple testing.;citation_pages=577-580;citation_volume=13;citation_year=2016;citation_doi=10.1038/nmeth.3885" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. Kim;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=HISAT: a fast spliced aligner with low memory requirements.;citation_pages=357-360;citation_volume=12;citation_year=2015;citation_pmid=25751142;citation_doi=10.1038/nmeth.3317" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. Y. Lee;citation_author=I. Yeh;citation_author=J. Y. Park;citation_author=B. Tian;citation_title=PolyA_DB 2: mRNA polyadenylation sites in vertebrate genes;citation_pages=D165-D168;citation_volume=35;citation_year=2007;citation_issue=suppl_1;citation_pmid=17202160;citation_doi=10.1093/nar/gkl870" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. I. Love;citation_author=W. Huber;citation_author=S. Anders;citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.;citation_pages=550;citation_volume=15;citation_year=2014;citation_pmid=25516281;citation_doi=10.1186/s13059-014-0550-8" />
<meta name="citation_reference" content="citation_author=M. Martin;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=P. Moll;citation_author=M. Ante;citation_author=A. Seitz;citation_author=T. Reda;citation_title=QuantSeq 3′ mRNA sequencing for RNA quantification.;citation_pages=i-iii;citation_volume=11;citation_year=2014;citation_doi=10.1038/nmeth.f.376" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Routh;citation_author=P. Ji;citation_author=E. Jaworski;citation_author=Z. Xia;citation_author=W. Li;citation_title=Poly(A)-ClickSeq: click-chemistry for next-generation 3-end sequencing without RNA enrichment or fragmentation.;citation_pages=e112;citation_volume=45;citation_year=2017;citation_doi=10.1093/nar/gkx286" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=P. J. Shepard;citation_author=E. A. Choi;citation_author=J. Lu;citation_author=L. A. Flanagan;citation_author=K. J. Hertel;citation_title=Complex and dynamic landscape of RNA polyadenylation revealed by PAS-Seq.;citation_volume=17;citation_year=2011;citation_pmid=21343387;citation_doi=10.1261/rna.2581711" />
<meta name="citation_reference" content="Smit, A., R. Hubley, and P. Green, 2013–2015 RepeatMasker Open-4.0. http://www.repeatmasker.org." />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=Z. Xia;citation_author=L. A. Donehower;citation_author=T. A. Cooper;citation_author=J. R. Neilson;citation_author=D. A. Wheeler;citation_title=Dynamic analyses of alternative polyadenylation from RNA-seq reveal a 3′-UTR landscape across seven tumour types.;citation_pages=5274;citation_volume=5;citation_year=2014;citation_pmid=25409906;citation_doi=10.1038/ncomms6274" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. Zhang;citation_author=J. Hu;citation_author=M. Recce;citation_author=B. Tian;citation_title=PolyA_DB: a database for mammalian mRNA polyadenylation;citation_pages=D116-D120;citation_volume=33;citation_year=2005;citation_issue=suppl_1;citation_pmid=15608159;citation_doi=10.1093/nar/gki055" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Biol. Sci.;citation_author=Y. Zhang;citation_author=S. A. Carrion;citation_author=Y. Zhang;citation_author=X. Zhang;citation_author=A. L. Zinski;citation_title=Alternative polyadenylation analysis in animals and plants: newly developed strategies for profiling, processing and validation.;citation_pages=1709-1714;citation_volume=14;citation_year=2018;citation_doi=10.7150/ijbs.27168" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=D. Zheng;citation_author=X. Liu;citation_author=B. Tian;citation_title=3′READS+, a sensitive and accurate method for 3′ end sequencing of polyadenylated RNA.;citation_volume=22;citation_year=2016;citation_pmid=27512124;citation_doi=10.1261/rna.057075.116" />
<meta name="citation_reference" content="Zhu, Y., X. Wang, E. Forouzmand, J. Jeong, F. Qiao et al., 2018 Molecular Mechanisms for CFIm-Mediated Regulation of mRNA Alternative Polyadenylation. Mol Cell 69 (1):62–74 e64. 10.1016/j.molcel.2017.11.031" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="twitter:description" content="Poly(A)-tail targeted RNAseq approaches, such as 3′READS, PAS-Seq and Poly(A)-ClickSeq, are becoming popular alternatives to random-primed RNAseq to focus sequencing reads just to the 3′ ends of polyadenylated RNAs to identify poly(A)-sites and characterize changes in their usage. Additionally, we and others have demonstrated that these approaches perform similarly to other RNAseq strategies for differential gene expression analysis, while saving on the volume of sequencing data required and providing a simpler library synthesis strategy. Here, we present DPAC ( D ifferential P oly( A )- C lustering); a streamlined pipeline for the preprocessing of poly(A)-tail targeted RNAseq data, mapping of poly(A)-sites, poly(A)-site clustering and annotation, and determination of differential poly(A)-cluster usage using DESeq2. Changes in poly(A)-cluster usage is simultaneously used to report differential gene expression, differential terminal exon usage and alternative polyadenylation (APA)." />
<meta name="og-title" property="og:title" content="DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/6/1825" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Poly(A)-tail targeted RNAseq approaches, such as 3′READS, PAS-Seq and Poly(A)-ClickSeq, are becoming popular alternatives to random-primed RNAseq to focus sequencing reads just to the 3′ ends of polyadenylated RNAs to identify poly(A)-sites and characterize changes in their usage. Additionally, we and others have demonstrated that these approaches perform similarly to other RNAseq strategies for differential gene expression analysis, while saving on the volume of sequencing data required and providing a simpler library synthesis strategy. Here, we present DPAC ( D ifferential P oly( A )- C lustering); a streamlined pipeline for the preprocessing of poly(A)-tail targeted RNAseq data, mapping of poly(A)-sites, poly(A)-site clustering and annotation, and determination of differential poly(A)-cluster usage using DESeq2. Changes in poly(A)-cluster usage is simultaneously used to report differential gene expression, differential terminal exon usage and alternative polyadenylation (APA)." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
    <title>DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zNczm7AVCJ_lwciXBcV_kn05r5YdiPNbP0M0QmPL6ak","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493254","apath":"\/ggg\/9\/6\/1825.atom","pisa":"ggg;9\/6\/1825","ac":{"\/ggg\/9\/6\/1825.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/6\/1825.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1825"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1825"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/6\/1825":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022mvJyehICkWlyflV3yS5f6ckGHjNzt79oCzxePGvckmTF2dthhJBhqxpIT2ZNxwtUvImgTk6JP8pVnGFaFJf2DAlTM1DxOKQBbozFp0zLCR5Iusuzt\\\\\\\/ws9z8Vd4g7DTBGUbQJzPKe2jAvGM3+beUH4PP4EswSP2cQjnEme2EC5vt6r+8G9fcst21EmJWAnel73KhXvjcS9OVFRvBYjhhinOtvORF0E\\\\\\\/\\\\\\\/sdeBX1t2w9HlXZa2JP16XriAW5iKzo1g+JreIS1pTfyGVe8zDjZNp7lt9zmAm0HYzAevZ\\\\\\\/Psl0i33yyD0w6uWKCLQ8r7g+tzQb+yyAOPZJ63ZlD1QXkfAGeEqn9h6Sr8EM4FbVTmxZ5rblZHNzQouu4BZ21NdadYHbXQenqIqUnio2DdUaiCtY6w8YbmqFqPFDL7bbdkwUnY+tnOhOy022lXIwi1u\\\\\\\/1m0OqAxs1KwgCLnQER77p6YWqcEo8TOREgIejJeKNpzY5rjWUAWcuGp5Ro7rplsnksiHBZDoh5rnYzIK\\\\\\\/T+IegOsxnCdhQ1w4EaxJofNs6FlUhC733ezqvLa9eDV05sk37rmFeTk2ByUAyDhNAPVR4VK4fqmo9Oh3PYd0QWnAX17Qfrd8MDXcCf9fk7tz7oRcwAINfSwh2HATRHXzB\\\\\\\/GEGz67PXjQmva20OwCDjLjEkcjzq6lr7HwJi9m3VinZQjkphQylCN+TtiRtpqJmcAebuIIa+\\\\\\\/WJQRdvVy7B0vfEYt4x4Q+8cTk2\\\\\\\/nXxUYZ2eDQDLtEfo7gahHIVwlUcmN6Vt8NpqmJ\\\\\\\/TlEkPyqB0YP78qtXKdJSLKAnz0x7WUHJ+P\\\\\\\/LduPhxnQ6+dpWXOn2nqy3idNq9UEU2cW\\\\\\\/Tl1Pz1Yt3+nPBmvQ8dVgmGCajJjSZz3Smmy65Y+KkQQD4jhgG1t58MqypURXBwp8GPuuGuyh+gE0DkZXisn\\\\\\\/5e0la\\\\\\\/STQtU5QQc\\\\\\\/w6VeI5EnBSGbOLl3VBhvy7C\\\\\\\/g0SIHmq2vTJYQf3TmdN8A8SHhq0yXhiDaQfSl14kl3E0+qg9HLEG2EQse3kcSvxe8J17DlYAxusX6Tzq+Mu2BxE\\\\\\\/LKhbYL8VAOSUf9PSX+ZX9wYWKXy9UlxFgCYYd1PCfllNub\\\\\\\/d2TRpMEjPVQ5BWhh0nqSQQ2RItime2B0CIVI634Hp9iK6YRG1IYLHDrItDBszNmwj6n8yxCMo1Jshmd5E0ZWbhq5x8aeyx8emQ9fHO167O4n3oUMeEn+bB8e6z8ygUuxhoTwusiQ4aD\\\\\\\/550Zl6c\\\\\\\/OksHFiGOsP\\\u0022,\\\u0022iv\\\u0022:\\\u0022nnksm3NXzSeMdUwZxu3w2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b705ae5da03cef77fa3ef7d8a5c199fbc08a50d7bcb15cf38ecd2a15aa15ddf4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Aw45PHb1Zf\\\\\\\/D89qnqY0VEG0\\\\\\\/1ua4wWd6kJICKEEb7HK5wHq3pjxnff0auMb8hh+G9GsyMUoS\\\\\\\/sPfG7Mhnou3TOsmWmEpbPdKRCh\\\\\\\/vXLmWSQjL4ucziMkx0rlEbD\\\\\\\/YnJzJHLkKmNka29D3xuXss1n3PXCeshmFypikhsgFk6t0z2gTjQvlmDzP+NWz9+fFs3W2H0gcm2CQRYnQtarocS4k3QDJp36SzeGNsDsUNEoF2U3g7bMFYsXnFostMiylMb\\\\\\\/bwGStZMr76pSkFeLqb2\\\\\\\/HYOOFU0XeJAifg+IUDDQcNK8BoO2IuH2BcLz3Stx+JwZ4c9Wtao3M5Pty1x7wv7V2Kd41bRbO9sag\\\\\\\/uYoOoUf3JirBaYYS8pVtvdrd4ZvcwC7VPiqlUiGBS2sOhbund0fEcSSsbBNq6GX6RxQN1CdbDnkOM\\\\\\\/JU\\\\\\\/a9oQmTLIWdK\\\\\\\/jEDIl2U\\\\\\\/\\\\\\\/xn2LMgpH1hQFmheUTHh3enuRcowPqQmDinwR7NQq8Lk+aWk7MXlzHgC0mPoagOmNMa64XPkhA\\\\\\\/LK0\\\\\\\/eb\\\\\\\/rkR9MvFBm4RHfdPLeQfRItZkstp9seYn8WPJ\\\\\\\/IecEDtcY06jLLmkfzvaS\\\\\\\/i2myAbb1YgcFUaz0PCwMxDGfbQtaMxOAg6Sbw3JtpUX44y9PKaM0unURjTx69CoLZNpKIS\\\\\\\/1AUu+TIO+9O50blrKOdVZaATUEdoyqH3JML5xLTf2QAdnbZ\\\\\\\/Ju2oHKIa2YLC0VwEV+9sTTfeZZdmAN7IbIGcx4lIFfG7oLhX3P4emuAvd+QLAI9hkXe5lqGKWmtYv58C11tFrBxJFj0RDd3IuTtMmvyW4m0xRT5Q8zbVk7L35ru8WqlnFrWZkG7YLRof42B5xEJgBMjgWw+volzft1IYblFi6w+cKIwX5+2ypYx+1KQvfVg26U7iHZ0bzcAqLahatEhex\\\\\\\/U5g27m+nnrNzsL2Hz3y940xoSF0ouq3sfJo52srHtrb\\\\\\\/YshJAfGaOFmcqo0dn5bDAg1DCSvVwer1FgK16VY9nbdnEePHRoTfxmA43DrYDciAlTMU8TsOyvxS\\\\\\\/TrqvsjLgDICoyiOraXMOsIeEbSykc3T5kZg0PB0P62kHS2fxGK\\\\\\\/Uft\\\\\\\/+s9JtlkyWQ\\\\\\\/I7BEB7aVau5mM8eUDzc5BPmZoe\\\\\\\/EdqwvQkxZlN7vrr1ougtFbgthRDztY9rH5Hh7eoyTULhQvq9fvk79bhim56fj1UR22eq\\\\\\\/jQDJLBJHhNHzQx2c5UilDJPbB11jryuSDJ3ExyZoUbSM5WjsM6oSBMEi3Xxxh4LIW9k7tt8Eglan\\\\\\\/yzWmtP\\\\\\\/wCovYs8oNenTtxsQ9bJTFtDl1mUTIC2dUJaHSd\\\\\\\/1dQRT4bc816p0B5ezpiWCRLccG82suOiGu9Mdi0Y+mVnbdGpsz85Gx3LUx+b\\\\\\\/7IUlPZQQm\\\\\\\/\\\\\\\/XkLwckTjjuVvjUmYt507P2Pna30CQ8fRkKZHByDUok6s2lnB4XYfSalWgHrhF8Xyj\\\\\\\/BGgRL7a54vnImqiwoLckXsUCydkic28sJTJqjKLdKcCp97xX0jME1FeZF08jDIyiWVyNnfUTZmkJxIUv5lhk04p12wqmTNBCeQbMdfMkUdqFfqgJ9a63A3yiniEr0Sqp2jQe+xM7zu7luUzrUFKJvjwso7tqqTl6hpBJ9bMKiUgP6+SYHN0fDbuH+wKs9PvqTzWkq56wJ1x6klds0\\\\\\\/\\\\\\\/jE\\\\\\\/08Jvni4P5goJMwIhUYTwAikmRkMuJtCGKqyv01WQxn43Sohx\\\\\\\/wg2ESBSAN180ri8iV7JKFGnpObWZPV7gfFIzxUyXMw5gA2c7oRhJQL+2gMj\\\\\\\/pkbft2yNoSukNSD5DeAaGyXUZiKf0e06gwZ0iAqvsjsKpKw9rBMGejLU73epR1vqcdtFZQZc9TD50VvHHtrYX\\\\\\\/yYzdI+pfc2YepJSFFL7qKZHYy2h2YXCLstJPSBApfsjPY9N3tO7nfVZckYlaPxiOvN7GIQfFxR5r0iSpAjxvFufA53opu\\\\\\\/LEEgqHB25UL7KpTQp\\\\\\\/caQT16JUOIBYimiEjW0NvAK2ePu\\\\\\\/XxjKNS8f\\\\\\\/R\\\\\\\/mhTYGW52e\\\\\\\/XT+oBRv\\\\\\\/02lh0ts4eccO1qfX3MBZXL54nCchyo0ai+h\\\\\\\/OjPTzBGyppWBrIKa3CChk2xMSReJvdWt7NKjdv6RKh8YpvopINZbL2vkl9VBQa7Y9e5EYCW9Cx\\\\\\\/7ayW75bq6VXIk7g0I8YHVqby6Gu3pDbrEkwwo5jB85CzfiLrJWQWifkgyC1NkLyRK+fuBWHLmT9K9blpoWGxKy4ApUjDkXX\\\\\\\/iW2kFA7nOC2wSmOFeB508pgzsxEUjpgioz1sJHx96Pid0YRTfSXzOcF8VIFuK+PmMZkRPQFB5ibEt0tPAiP2s2HWFq5em5tnixpkW++cEJ7cusl6KwEj37Da+n17wqXtsJbkjzpp3WjXcSe91CO77GbTVT2SrZkJhqlzU2zawU0Q5BXIamXrpmgA9QNy9cjs1KK0NXfiZ1A0kZrthOkfHLPVlp+T71vEXKRkuSMq0lVHjHe6YcrqHVqrFwFhwzhaOFTNq6J\\\\\\\/XmKRPjZbXAN4uZPBIpjJafgB7Aq5aSK56ob4WkJSpP0+Neoac+Mhiz\\\\\\\/cijHDu+gMwantWtjnhTf1j8w3IOqatEwzGmsWIYmpmur5yJOgBHkQeORlO8uYQ6rY9SmyUAyaxXglDKoUZFD5Hfk\\\\\\\/ASFGmOVX3bAYB+egrjpoRNBpZMMF108lgULs3mmTeGSUo6Hfy6i5g7sac+VZvSG3+pX2YHzPO52MzR97Jr33d4Q4qhxgABeVUX+5PgiS3kK02+Bk1dzX7OxWGZyMgJgxaeAZMAlzZdJyMsUZDw1noCtL+R6UX7UMNnDDRXhKw4IkUT1P3A+vZuVnkDzK3ZtQUfVcT8V2VpNcYLVYFZhT+Cacwtem1RnEzGDNV13bMmWSGzWNYBkpBY\\\\\\\/o44JLnzTKqHH8O+r2aAnlkBlW4Ie89YinhvMz7Mzm0iaEJiPFERpalanwqcpPK6pETQM3imoYm96htQEmriO1tEp6TTkZFB\\\\\\\/oU+k3eT+kN09xuONXOIX9XtgOfwugsLuNtycMO8OYf8a3nLjjn0bCBsNxt+GNQb5AieEgYnP232tO6VjgHQ830xpZNHkG0v1uOCT7RUkRXe4Bcvuh+PRSSprEl6oOYeVkQGDLq9z+sXibf8dEmmLhj8zPohyhVaSY7B1LwftOqglZZ9KifLgSck6tn+XxClTQfJPGqmTLPrq7uyKT3BxkUT\\\\\\\/b3EkXORbbnG\\\\\\\/M5G24Ta++B788EMQ+v+mNfomaBk8k39I+T0k3QrRXI0F4PNmmfBUl\\\\\\\/8CJead8V\\\\\\\/6ia3JQDAzRfRl+BYqJrr8babvKCVEwXSDfoARVWdzkuTaxdvCsttGsVuam3HwSrnOh4lfjbIqWZisJW541oQszU4d0PdGMTeFnKNrkK0eGwOitlVxxiUeDPJ7LOWEF+sZ7VNedeam1yV07B6Q1URS\\\\\\\/eXKZ9R89GUXXxgEahal61UJB4+j3uQKKG8rqEu5X05lMrRkJHjj5L8P6PKdDeQT43Lks46xF8Gxq1YhszhCD1P9OIgvOTl+yqk\\\\\\\/msGDHftajZs2M6siE5QJPHWieo5Txzmbqqc9Nk4motdkgE9t8+KVbzVTY4xfVk3Ypqk0iif4WBpjE3LH589+ij7rkBCkbmIo5ekdSL4N1eIEDw7QCAVu44TEUbMPKvnqlb0sqwIwdnSvM+jznh+wwXud6BtKLdzV+bv+DDuaFRAa0QsWuWWBOKd8HOCSXcI7ecTEuD6ryiafs8B3IA\\\\\\\/af1prx7paX\\\\\\\/DLV2IQRYrMvmENz6TxmIkPTXWP5XZWmVDQrYBpfMcKplt0xRUTakGnwpEtoQene32kc6vOuAP61kIR12R3MKRaY8gP2jQrh3Ebm1p2+CofEe+ZL\\\\\\\/LbYAIX5lgEFZ42xYAXfBDcPbuHF4\\\\\\\/Ia5E6Gh3bnsZNo5HM\\\\\\\/ubX+2AIUHbXQHZqNKa2f7CrvtqbDEjOw7nZNReau98Ti+YgKPVbLS72IxDzQPb5LZ1nh7jtNvaEypBnP4v+rLQ89iMrmPkjy\\\\\\\/ffuiOlRx4sd9mQ\\\\\\\/lLAg37mh27Pa0JhRBFL14Ax515DQgmPuxkrOUEqT9QwdVXUDWRK0f+x8GLChVlYPvNBvDfyimFK+e+Fkq1Mi1UXV7DJBy+Bx8by5RzqNXOdIfSSpwUNncUjuDeIhCDtvOF6llVyuHzFHy1eDxmk88rMHtug0LLUJosPqT90XQMgpovWOyNuBXgB9ybq1ZRn4kq9Jcb1dZjD2zphYCQqsOlWKjvs8RNY3v5uSR679rcHon+8QW+DHM252q4JrRpZR\\\\\\\/6dLBtBzIA5pBmbi8dJyaZ+CfSOeI6QyJJDNGtK5KHUD+lik0j6JTUs2NIBPbBn27xPY8xRDiXiBL6phnzHkDEy\\\\\\\/sDLcHo3cdWcO3vQMC9pQLkdn6gGmR3mgx8f1qxUiwjoqg4wxSO4rIrNct9saxsqEejNcn\\\\\\\/WS+2RZNamrgb1yBxYOOmYMCXPmNFOBRjhJM4KXL1ht1iWsa0XJ89aRawgqMrjOFSQlVNLh6PF5RamZGW9Rya5UeS4HlJ6wOWSlMKwzCGcCLmFHyNcDzbR57sQGiiRNQt9yx\\\\\\\/Qm29rdRj5uPcG+v2NoqJIQo27oa6eKb6y382o4l\\\\\\\/DYns4o9f9zwWCBgZD3cHPg0e43vwDDX44LA6p7cVAwUnEpq7+FVchtBRnM9zowBieWPCmebsoGlXurge8Szk9QGUMGyOxAaKX6eoqwj5h6bohYlqWBOOlpSKAiuUF+aaM+BDRWK0ubFVzO4Ykm4gaRtp6yoxxYN8WjkZbn2TGF1DH5xYgxrrZJLiqGAbmeShNlo2YXh+7\\\\\\\/Wz+pt0fWN8Bztq\\\\\\\/MLvNOjGx9a5apgycOHr\\\\\\\/X+xi+AoMxPy5dcq9epTM67M0lNWx52e\\\\\\\/k4ZvjKfPVla2\\\\\\\/7CXrKFoz4Xd+QB9M4It4Lioyv3MrAYiQ121PEBN22mZjnKyTvzRpYXCBtk9TUMHraklJfO15KbutFhZlVqgEfrd7NT50ZKiig0qJImpu0OFofe1+IEnbZoAh4EdXXzx8HGnE5MWsNUEPEzZcFo+VK24pPGG\\\\\\\/uYig6gANeO2aVrxgbdKUF8ZoiYpvrrWS80hf1AvqxanaITS5\\\\\\\/wWNRI2HroF2GhFvvzdkogMXLs9NZ+1qkXZvpk\\\\\\\/hsysqO6kzNFzFAttS74PbOumPYT0L5uLZzyoXZPAlbPOvTr6izurBPmtpLPaENkHgHxDzYR6y\\\\\\\/ZD+HSSQNDs+nQiXxEC0dfRFg1zvV2nAafgu049dM+s28qirMK0zivjz69T8X1nv5Oe3O4aSoo101Vt1d6CX2NuXrWtyReNXeJ8+lY6WOAS5j3c5xsSATbNZVhAo7GToCHXT7ui9UKTB\\\\\\\/ckws4xnsncXd5ov0Twhy6s5Y1mMZED8qnpc2ZFX6QAMvWpVx0Y5RNJpiMGp34kkUAI+PYYWOLGx5x3CKh0RWLrO1jkRPLthY3o0SggulODpqepXE0y8a4HxA523qCGhGf33A2qGKxcSOuaEIPCuOOQD3QdVd+uuUB+tPowCJhvdOXS3L1CUMA+V5g5j6fAKuBeoWXOPFWULlmBhhZqMhnkj8SCl16X+W7fTSta6EQZ35xx490tXsYUM5uW49iDAr3J7DwH1nm1egKF+svbdu2OdeusbqY352SID53lC+Zo2HeyewcqYqlKjOx6\\\\\\\/05m2KaNJHeXPaZY5oKeGkaWVH8CIbDUwXtaPo+H2kFl0gXO5FoTiWChFWJubzjSf8LzGXtEVCQLFU+BSxexTMCrLP6+s1KEbD4j9ptkpCVMF81179GOeWzGq1fwLPrruvVKEZWQC8LWDJIgxJNOeussHygE5duA8VbmuBGcVSyaBcaovU7878nbaD9snOPC7NaDwbJY9PS9g8yTXXwarHmq9U6RtA2kBPFl\\\\\\\/314nu9J2BpBtKeTRjbqQ2KcuQpQQjeBninXn0cK1Pm0ZVO0rtvQVlamrHa1lu8rqNMONO58EzU0J2+Vh0QoKowLOYsQgCq0MElx0sBNBwkXgVpb1yI43MXu1qGZqXrIBqboIyGfrH0ORfMO1GeeoKfqgNRzw8IB6zVmqP9cMEGZLJ6\\\\\\\/sH5RcdlOnIPtHiKLySStPa+0OWTGdcjTBqvLtPgxUmZGIVOM7XjDEJoLAWIc1PUe5xSWoHLNMa\\\\\\\/YlfO1\\\\\\\/kAUSeeYLd65MD6TN0za8Go2eEWB6DWCZ09d+qQWBBaENE2\\\\\\\/GX\\\\\\\/oz4Mw9Q9lMJJt+8a+zLtN4M48A2fNdl10u3eMPPuLhR9a6ILk6DQswKfKs2lKgs\\\\\\\/fq+I4o8Tu9naSm23gxEwAe4CmX4MrBj39cQyOgGmKWGKJ41tyqYMOnYOw5clAU0VW0HjtzfFuyhtJD7vC9VYIv1AbZ+DKl5NHhUzcBLZv2doDvYKIlog\\\\\\\/dQI1iO32PC6ZP6AFHg1eA+WxhHjNOPQrBz3VNzYZHzqGy2KASnivD7d+sUf1+dsqrjxmNsSzzgQ+Na01iorslQyODtyih5EzdepTy1Vxz0SqKH2wLC57yoG9J87WeM9Jx\\\\\\\/AtNIVDgu4RwmsVZSUN1lJl6Mr8AexxePgEVoGWFQ\\\\\\\/tWPUNXdAAjFadmPAoOpBMTwkUYSXztpKEtk4X+IWWrrVc8GxmUfBhtt8DaAHtaf+b0\\\\\\\/x6UkwCgeaOWahmywfh9hhkdHx5eublZaec535d7DUgj++JfWNU5ZPt2ELw14bqn0zLrXIJBBTb\\\\\\\/JBnR9k6wOKS2Miky9WwPtQJq9GZD\\\\\\\/6mcQ2TzMa7iYZHjvih2IrtDBWBcuC7fj5fiIp2\\\\\\\/T9t6V\\\\\\\/dWbasP\\\\\\\/UOsG42yInrCRz3sDBjPK\\\\\\\/3dyJT\\\\\\\/iJtwl9iFMK0ugYJCKksLp\\\\\\\/9UrQDj7sM\\\\\\\/MEXk4ogb3\\\\\\\/o464UnHq9M5gkkPxMfIBrdPosGDiw6wPkZ\\\\\\\/xJ+Gm\\\\\\\/9AWGoNXLS4BqOc0+Fn1Oqy+4xGH4eRkEjwATCOvTYMn41+UzS8OHLi66dPN8r4sT0q4CbGmsb4BepizkwVzBIeHMshw+KgfGpeEXETu7x3ZzTRHIRrIcg8K0cO0SmhHzzflRffQ64eSaGY0P+OLFKnFRtF\\\\\\\/wAOmfYlnQnZ0+fSrg\\\\\\\/y3PpoQ9j3djFBeykE1vFk3qxLEqgXex7PLjC8Go+RH2qxjqdu7WzBe\\\\\\\/vjtdDjxs03uWMjasQMx0g\\\\\\\/lqZiGNYFz7ASiI2\\\\\\\/kh7878xQQw\\\\\\\/OS6wBrpqioKv+t5JGz245YPveQGAi9W8AHZ52AmTEqvuEwGlP\\\\\\\/BfaXKrvk5LNiF3wc98Z+xTK+Mm7zhWJCr3NsIr7ESCcek7sWq3ANh+4tMxOP2G3UjxsxxXDpGT3uFcL2S\\\\\\\/\\\\\\\/0+rYhq34jwF3CFBb4CrrjSGTebh97NsYOP8D4F2Jom75YYGjzxxdRZxYPot6YxsZeumd40s6DiJ8cYGWCgjBj5aJ8MwWDiWR2N3aH031YbANs9FDEXOIor905+k\\\\\\\/7FYG85G\\\\\\\/DZpO+q9iiYcHmgbXLM6MUU68x+Ao9EdyYmPOtvPYNbph8RYojtbiiLzWO6voZ3eOeqUYCAnp908B3e98WKb7dR1TfmEWOdxTKD6HRKYqyVERt9KR\\\\\\\/ASN\\\\\\\/WoBREpILp9HK8i5bzqliBwqmh91IU9pR4IVITUeLg2DrHl\\\\\\\/4XDnL4KsuWGr1b4a2MxJC2AJdntpwNr\\\\\\\/fagsk08ZHa+7Qy6Vf\\\\\\\/BBpUuVE71xFQ9GpmkF4gQ5CUickozDwdxnBIRIOFK+VGE0abEHQY1Id9ji8Ez8uotkdEmQTGFq56y9rYbm4Hr5ExGlGnZbINM9tv\\\\\\\/KAoMrqcWkZryfqRapLlxgYUtNMhzBvNnkjOLTx0y6my\\\\\\\/KDq3PD937tvQdlRBugQNwfpGcsQjtcGimwLpSnG0W7+d6Icwh3pQy1zE1duiRqANe+iM9WWfZrf0F5ygZL80jQZdOVxhD9qaslae6vmQ0diRkLHilNVeR29LlPnRw+XdgiG5Qm56tZBMiSnZoz0pqOTC6\\\\\\\/1hnFLe7wid1lcKknJnax6pSJ7M6OmHbMmwt87oHt6BHsCORnSDYQGe\\\\\\\/2BFnJCX8a0CJFjRQN2sRot46Dv\\\\\\\/vNKiSbfvwSa1xxNSKPSQ+TdqBlE+0Vt3MxCn8FBl3zA2NAD+KGttOQrtSJKFsM+4+9xRIyOimQBugJQNbJs9lUAnqQnwLY3El+rUKGjTusooNeVWCGux3E8\\\\\\\/ZJKOkEi\\\\\\\/0dpbUasiQS+SkbnCy8gWQSbyGhv+kZPdCbYO12ribo8WC8vfLSWNRY4fUX7gj+z9X72d7YjZEdTC5lj0Dg6M+df1Yvu7XopC0vcBBFK0fGWPDzxJvOtq2Q6TcWxlB1uZNVc4r6m1F49U668U\\\\\\\/3WTYzTv1idCF7qeldxBG6cujKKaa2uv1NC57Kc+isV6J6jmk3sSOAcwlpaf7HgcsBowwNPrmAJPALpoUxOyC8mLPfoMXFF2AoHMQqlQ8LKt6a3zhR971\\\\\\\/5WvjACLEcXZV2in6RkXMXtWbBnHlKAXZb+Q+UCzY0hV58SjzksPydzk\\\\\\\/fVDKgDpDUgK\\\\\\\/2hDvD4kw5KNcqpDzSEXIXC9QrEtTWMAiT\\\\\\\/UXNtVtjRT+LI5kM4s1yhs2Nbbo8P9G6vlPYFWLG4lspqAJBBNoa\\\\\\\/AzU2j3M6mPvBt\\\\\\\/rivA84+h\\\\\\\/lh4sPae5KgXKtamEYaFD\\\\\\\/IB6DxkOJqs2Yh+7aE3YHcB1t8CvwtK4MgktHIfUtQpo1OsiFUiceegF\\\\\\\/hrWqD1\\\\\\\/ONQarnoVQa2q2Yfrwe+Fao2HjmwxAZPTG4tXd9flnd2cgGSsVWNqWqMRopf+lQE3XDEI+AgpPKmDLrIABcfxJqGDdigOuK0VVbFwG+YvtxxMVQcviueVvivXg8ozWOFm73uNAbFVS67pOFQHcIA6V9uWYJnw2lTkhXocSNVM9nbecxJVHK9uWZ9louueljehJQhPojIk1ATC9IyyQhPJ5CHuyu1oZ52Jhv4w\\\\\\\/rQXmWtk+yRs1GSFsYqGj5ut6xDrx5XACPCoHFOd2zTJKx\\\\\\\/\\\\\\\/SZ09K\\\\\\\/TUodU6P4NmplZ21VlH\\\\\\\/EpgLfuXIqRBwNxOqm7yOGk5+OlXspU0y3fDIAURNxQ4B+YwWsdtky32LAYvtItImZ04FZxbjCny\\\\\\\/BzkK4wT3xDfltImTJClrsoZyLLaiIBwtdTGaqg0hr4rhp7gbNB5maSDYugeHWvSdr2iPgir50qNlho2fHVdtxx6ZHtuOV9LCdO8HLlCCpNSZnfgH4ofpsm3lr1wt0TU9yGE404mchU1yRux0XNWg\\\\\\\/uIMf4esHHRsCFaLwqf9+xesP7N4ZvppoeAzW8hMT9RkPbiCmLekk5JTQTBvPcnC+Wu9mrKqljQE9D5y9JKZvokZnTDlPO91MttH\\\\\\\/tABHE5Ix650Ur2ADxuR5l4IGuqad2e+XhigWKfffmqADVo\\\\\\\/LZ9Drb5ACeIBlfpc\\\\\\\/Ru9xtu4Bid6eE8LS9k79vaM0kfcttySpIzmP4nLcB57+mdaeUNc\\\\\\\/v552FBHc6Cpuoyz2Nk2UvS5tPWoWLZgaV4lWryDDamv4M\\\\\\\/AMv9kFTdXTTCa1IdsjdA9rZJApAmsgJyQBWY7aq3suytP9An8\\\\\\\/oGsaurVZz0o4nimqL443asesiRsIMzZz0drhlnAnKPn4SJ5GhJBoeLaTGqiqu7agWvyDhJKJwW5KO9lMRnOU4xaUHasa0I1ZnJy9FG3FagaFHy4W7KkU\\\\\\\/zx7gipxOx1B4s773nxr7i\\\\\\\/7FMdHBbEm7IItTwcer+mD1PBjOUbwxS7YH0DlIYn9oK0Uq2cZoBRRxCO4ANdFzR2w4xoHRF8Tkp1gH0C4SieM4StwRu+B3XyEKrPlQdfNuZmpL6UyB8zFOIMxtaGABc3sniDon3BXxDaHqHNU3eIJgeIYrZam4L7SYKV0J8lUa54lsPykFsg5AI5LxIWKx\\\\\\\/8rALqOcg+aFxrT6LF7hCmtRURSHEH85hKjKhobqJfFqLO8djZRlbzFbmoCwM+GQIxlO+01nof6TriMbL3v4OH\\\\\\\/pmw6PEBHJG29+vtvZE1t0BkB1xOExuq0a01a8A2WUL3YtmB2N2KGP1pCP4hpn091\\\\\\\/Y1HudvJk9LAg3YNkmvd1Et62lPEgG+HhC52LXmbXck\\\\\\\/gu4FUB59NtzJDZ8e2s53H8mV5av472GkNJj9ZY6fRgCwMIpD7cJoKGH8\\\\\\\/3LClQfcXhQYWZTm6QhvceTjPfmFprYyilHKGxnYJF9v9IgxXIoaD9oTauBcub3cqey7z+5CSSxKpkiU\\\\\\\/k7fOjoyku3hlVtvO2AlemlLXBSdR73\\\\\\\/o03B+26NO9tgo3bgwa+fpr9TYZBEHyVOjmFmtm91nKc5gi4r2Plm6Uosk6wxdIC6XkXHCzXsxrb3iNzFfKrizK6sDfJMTEZN0YNb3fJ5YwgEYUrvR9wOtPpDZS5fDmNxXsUKijaCUi1YquwV7sm3cfMie2upGVvi8uWyaqhi0PXbZp157wDC4ZGdDOchr5rrOhKM11j+DsZTbcUzdJ3zC+wI0soMtCFT1kXRod0rkJzbFiixEKURKxRW3Ys3VGv648eb+flUoeiSsTol9q132qioxRo\\\\\\\/RSYGKei30c37S+nKyvn0C5yMg+3Xuc5usxlBBLtw7BnQadQWpVGogZLfpQZrRFGaaTJ8vPCzznFdtZnmjpd4\\\\\\\/kODWSYtzP\\\\\\\/i\\\\\\\/5coz8PUFkBd2AN9W7L\\\\\\\/glzQOfb\\\\\\\/yOCTvLRmdH\\\\\\\/8jWi7cALM3p4qCveJ8L3C1pCG+lgW495sgOYrHVSqVN8Kh3G6P9ZEr6E\\\\\\\/u9rWblJja3q92yf0mFFqFXgljQRUg0pXZzGlExxM7wJ8aXiostpraAmUSw2lJpcr07\\\\\\\/8r6SBaocnjjcphN2Osz+LnLQT1\\\\\\\/0OoqU2hJPRBlfjie8q5GumLmGDslX2klmxKzqcEZsYBvwIj99zuWuAfJTpAyah9bFAbhNE5WMJ6W+yT2MaeyU7LJu9kIZxy+PsNanyY9ER2DSD1WCa63GcRduziYsjwy50SAde5jggd45y7og1twBWOneFC54EHmzBFu01pFdufxWn0HaAvCVXIqRnMYdVLnHbDRvHCqS6waEBkPCpkT6uaMXjDfgEexMbO6bc7CyvXtk7QgWdwxgnWvEdjcEeu5xZsHS8sfIGW8jQEPefQANBkEAm55eeKVNOluW8o8BsOqMFmLyq9Z6\\\\\\\/uaoVRd0m3rM5cgvy8fYu0p5UOKHNc0QsxQz6LpOo\\\\\\\/eAPT1RBMszhkYOqBBv0psm0\\\\\\\/G+zd1CyVvwJOZKNb9LDcI0xK0ek4iTWfCKBFAUVu2eMmsauSarXZwjqxDzARBcdzDY79bVfxL5vebSVGc3e9DJsguuIPAs8BnVZvef+Ca1EH3ioqAKNKEIwET6nn+wQadxNdxOAtPocxszot\\\\\\\/09JrbwNKPcFxgweLB2B1ADnRwTpzsd5ykNIHn0GcIIrUwCh+aEmO5rHw\\\\\\\/148q5WlL5RX\\\\\\\/vEYnXqLa8mfAeAqpeM+WVbqD5As6pNQsK1zRZWr0RY1yEBqZejskrZpXFy+86dIUuEbDVFeOADnpas2OEl3hlG5sIPSsV0mMlWAggZwbluokvsIcN8egrOE20Kk8F5gzrh1qFfIJ0Ifp76buSKaADye1JaFf8Y7tUdCH3\\\\\\\/EQiX27MDIg9u+DxTrWa\\\\\\\/m+g7K27gu0cvz+oA2k4to2cQTCRU8mpLFi+fz2QX8lnWAglsoOG4pm4q94XQBclDRfOfXHWa2VBzxhP9N5t4mLli+jfbcEgDvHEH1NNv2hb7U8dgWaq7QMS2ZvUicOqVYEvXhUS+nQ\\\\\\\/LI62Y\\\\\\\/9gIhVsP6kwxF65DsnuVvV8hzu3YYa\\\\\\\/tYJx2MlloS5j3pP8sXef4LXl9WG5L3HUzovL2wDx5sJna+DS8LW45evBKppuP+lzam0tSc+qmKCEJ4Uijh\\\\\\\/8tuYuaYPUJlk1GdDhCpU8aBGJwzelT5AtawvEM34ELBIN8bOi9Z7UbVdbQZNXQtb7R08rOSomtd7RfpulFDyQ+ycvgJrKEba6KqQEtU3oIxC2QF\\\\\\\/RiyTdr8cKQXTeTzacMKp\\\\\\\/Jj8u2JzrO6G3ZPgVkiZBTzh63VcvHWB3nGDBckJwnHZP3U5aNHQLJFPqq77PzBsiiaB+f\\\\\\\/YkDADRksyPdK+z038mkgG7heTOg2wKJQ1YdtD\\\\\\\/XnFfdhiyEUJfz6+88vJA6DObdghPcgVvTUdMNFaqVMjh0tFxDl\\\\\\\/z1OUt91rfMy6xCGRAe9\\\\\\\/mal9cQZme5o1YagBSGCOlNWBNhZA90E9k\\\\\\\/taPvXrSJOpkoQFtJwjf1NdUPGi3JzQ6+QmSKa\\\\\\\/WsSsjoOm+I2\\\\\\\/+5c+McvUho+iuQ3wPJdkz8BYA+euVBGODHMU34hUY3251dZrpOdx0BES43fTdZ5wiVqnONsHZklkFiuDGFJihmARfdTECM2EzQ8TsJe\\\\\\\/84YEVt18dR\\\\\\\/\\\\\\\/zh0i130AlnHc9lEDb4++1fllYS4Z8HEQsj20rhei+hEwx\\\\\\\/\\\\\\\/P2V+oBuwvZLU0m2JVoO1lJ6ntKQBP5LBhQ71ZdT911uCJdB0wyND8c9PbAiynFvRPkkpovRxTs9Ztib5QG551j1BbTK\\\\\\\/IoKND2iqNE+CxWwhd7sMdz+rDsFPo1qbFNUbHXWA6UheUJBPCqiodr0u8aA4Tk\\\\\\\/PfVhOlIAPSK+8r9TMIAIdDhemTSUrggCm8sc557dMwKOdzcxfyD7gmV5lpJ9\\\\\\\/yWA6TQzH+f2VEmAzgRHBgGljDYux3hrCEcCa0Bj+\\\\\\\/X4ytCR8GiErcHOqTDMhIsnZ\\\\\\\/gbnwbmI0opdn0CkJxrgjsPIUiTsxdRMcNRPEtHsrr+f7P91sqre6PPuzGUoaaiR5KCLn6Gst8Kocw3XW9g+pmj7Qs6t9EfizHKXAiPDRzzkY4M7hIuaudDxehT8yYGac3Qz6EmRD7+\\\\\\\/dhXPtRpGIUvHIABmeEv7rbqj44GMy6g5p7cPjYDtRLgPFavoJkwpDhaUCb7EgyXLl9rXATzLE2uVHrysJcQSEKDVDcpN4bZyGXUUQLUUUuYaeajmB\\\\\\\/l4w9ZI63TcgKkQE+Icu+4Ggv8Pu4nuvJzi7bskWz1MMXwzfpGKdrxccW0u4dyd7Hi2VxLE5ZrAnG\\\\\\\/eCS\\\\\\\/Gow4oT19rtIk6Jry0YfjashcEA08R16nI5O6O50BtnmNdnz+wuJR+MYCr3H15lTh5ADBPFdumcVoInfcYIPEGxDbT25T86eJaC7\\\\\\\/fWeAuvwOz4hiJMrlHZBgpwTH9Sjoinl3Nj9+n7B3UboyQnR31tmWOLRdYePF\\\\\\\/Dx\\\\\\\/aVd\\\\\\\/sAUKGhbXZmcVQ4\\\\\\\/JecHYKGkUSydIEm\\\\\\\/cjRX+25y09wPV7faDsnSlj3rEB+YlUkKjdYZAGWn+9+Obdb+gSeAsoFA57+ZcnVE3zZ\\\\\\\/mbcXL1t48B9FWxVOFkvtHIxfxW6YWoFSd0QVqE9e3GEk+UZwSgeJ5kRtF1WdSr62qi+jVXIwUYpsTLsJk6513MS5YlUVygjCPzMfJiq4ahUOQ1VV\\\\\\\/Qzmdf6MTm6OF2Yw9XAWaUB1gt\\\\\\\/8CrgTepQyLVEFUrSruX9BPg0j4LVSXtP8PNP6jFmNWVNBLH+ibGl+JUwEeFZXqYXIzdEkPZtw8N54AN+OsDTtbk\\\\\\\/kdVRx+hwIwLqjrAoLRFZf5a0EM6U5ZXeKMzvAR\\\\\\\/2pVY4RyFVpmjwriKGvjCdQzVcjIExH2GFJWxNA+4kVr7s9ZfTu5Nma5BKPYvRxJstBPukWaVxLy3nVVai6M1ZXpja8pWEdl26KSBwIs42iC6e5TNefrh+Pf8y74KxanIe\\\\\\\/EecoLLGyHIrWZQIYPiluqFBFzhyD8dwGmg6XORFvax8mn8SRXvf+kts1MnygrozMnOtqP7z9IhxJTJ5hNg7aoYqAhbz5LL6HVKHOaeADUCyqFtBPpIDiOS15ITsNeo7dRzFPp4n6IPF1ZD81WHtD65JEDr9FobBOgguEeCHkgpvwEMHX5yFySdg11tsoGjpLLSj6DBaYdxiYYvU3v5\\\\\\\/0UbGhcZxFwAPfX98BO+62TSQR94NTJaablMLuxs545eHkBM\\\\\\\/VYZibRr728MPJ7EH0MCa0mi3k3okzk+P6vtrfu0BDJHiUOAKvRwaZc2e\\\\\\\/DPaJiQ46Vlji\\\\\\\/X7ynE1YrDLkwSPNFfS4DkQeJoIonhW7tndm1M2fNsdFob5r7gSLdvLA5Qvd3uHgzEuuWPHrcm1HZ6yKTMEP5dsZ1f0NSczxLxIYhSDTwS0R7wi7p7qVsoHF2cjcvIH8mmBqVqmhtcLyu6+ZNJ\\\\\\\/9A5BBmt7AU5ISrIYgRCTr\\\\\\\/Kz3dDjhTs+r7bTz3DtIvSoGphECl71Go\\\\\\\/qRvhBenrRKD4uByIqOozXR8Qnd48dppq0f0qPnO+WwAyjsfzqNMfCxFMslCtCogiErF506bIZwZnty3UdEWaNtrV9HVKWERmzclH0ZfYoy+hC3P\\\\\\\/UK1vbBiDjs2oRA7vX8biIX01x03vQ2ntfs+4I7s16K+aBH+466FWo+9B+IOBgsZe63jTDrEjDbbjJqlMjCTu2on4Nrf\\\\\\\/ef5Z80Ek\\\\\\\/sW+7AeeceldyzDNYWTpEF7BpbI\\\\\\\/jGovvNBvIThet2vE09HshtNRKFeXJ+sck22EyTYwYUvxSSzghyRN9qcvaADJIBnCAytNIXGzW70MBOaFRvy5db9OFAYU1L1ie4ISDEFtsqdvnf9Vx9SexneIZdjYP3cTbTM4WRhHJ5u+j5m0uBg0PYce4KMQGA3IUb7kWdz7Fx2IwhGTErCRwmqcTQDtp3cN305qeKhyeSJ+t7qb7KKfv7v9gq7FF\\\\\\\/8wKqSwGJWw3JPZVW\\\\\\\/1urtloYRzfRpAWNCaImh0FwuxB6T8FboN1K9WADCa743H+Mq5vTQ2J2ikwPMYj7rU0EINqJlYQFGx9T7Qn1YalhNmBTi6Qp6GQ+xJ1cyM\\\\\\\/rUt6XOH7csZzLDx3CZZ1wN7iBScQ+TvSv822+RHATKHVuY9vt7GLsRZIPQz3ra4hyEw0whSMx4JZDeoSXZVHaGw8bmMBi18CQ1CVDkR5vyTx\\\\\\\/SM72l87cd\\\\\\\/IS+iY0D2lJ6CNiGA4hd6zLJgOLaxsRqiJdGgALfhlhDinsJLWYEgw\\\\\\\/p8QPxSAWBPcegvzf9kC3yrkbNJkkrJNjOXN3kaZ5MqWsXHTMqBe6gY+4Gc8z8bNkiJkB5vJxubSRcM+JVVu76\\\\\\\/PPcwQTaQ+7w5JdrDQ8H\\\\\\\/c+2pGkdUJ3bxUxyMgvxbplZ6iQM+TRSIsaMpJgou\\\\\\\/47NWmEFLk86WFy49FwcPuMMDjE1okAHYMLMQwO2iDeQLuy8notuC2pTw3u9ruMzODFiOaDe2mOtN5AlbPDuHOXQaytvy2eu3VrIM7eMLtgLZSXSyNLc8Z1vr8s1r7PDgcykHwwNoOIRAIJCe4qw6mWGYnLRqFMsHw8WiJuXO2taFMP0my\\\\\\\/V5YGbTgazwXOreza\\\\\\\/+S1+HxR2pvoY6PkZnjMcy5KFZrkNOh2oIWM7YHGBD13OfaJ1b5H80PID8qywvP454WCNEgfbLSOAJOTYHNW2MzrDOXSWxBWMnOsyXRi5KxozfQ\\\\\\\/STr2q94i1ZW\\\\\\\/OKKKTSy1QonBqgXsYTElc7hH37paqQkvUght4kgJTqtuv9wvH49WnB\\\\\\\/72M1lGD2irEWwxherqkrYAEn0jSca43480pfidTONKTCVed7pmhbIWhAtZuZBJuJ2jqVAdmkAklSMjPpst17mVXSb0JDSjrzRVMXcIz8VyxlUf6R6JqLd+INThLTSWoAB2RXzy4p4AMMFfMffNJnN4yYW97YpGCzKp0wQhGt77kV2ZNJEIsuEMktMUdETwt22udz4Aj7NSmXovaWboavYYs0\\\\\\\/JzsPmOFfPyzObD7MTaWAHL3Lid45agou2cwtOCKLruFs\\\\\\\/p5gA8r+sI\\\\\\\/d\\\\\\\/GGETC\\\\\\\/DMadny7rqRdTAORicrLutAFPvyEOmI9Crgw+LbqXwwRtGa3XatIIXudal7y2\\\\\\\/dxLKUm6hRvrBm6ZA5SOZLLfbcPFNHBsetSa8tHz\\\\\\\/s6eaj5g3YUZ3Rn9S58oOTsF3MJ4NQKm7MDL3vwuAP6AHp4qyx3iPRPmCS1ETp0M6PWaFleY494FJ6OQ3uqmeJSJC5N2pv0Q6ToexZnMajaEOCch2EYRhtIoVkBLuTsdMEP7ZN6Ih+iueLjtJtOupCE99J+NNZbWEryUM\\\\\\\/\\\\\\\/sO\\\\\\\/a0nCcf50ntGChsCKRBO18S3M+vdKwx2mBAGZF1ZjhwrK7SA2R3lNDvCl4+UMGHYyG5mzw9zELkPYPIKSNrjXzepwL2ANhwyFzTEo\\\\\\\/PJi8tEpzSwxo28sU7YPrQTGPnJyU19b7H3GgwUSF3MkUc1D+gAEQEGCJMj7EmvL5ZMh4bQDUCcPltaCO11BLUb2zh7FH3YGlwrlkyLmi+r51mUjPTnP1Avi38ytXhevH10NuRfQyNY2wmXhpN9xoKM9AxSe+ssvz3gUhVPP3YjyXab1VyexEP8VESYNbquhvQ4RSCazSoZjzZ0sRG\\\\\\\/9PT0+mpYxynvzl1lXxn6XRHU63ksTiX9EMDbJ6NDSefAgjPFenDgk27lwD+Ugkg4ZxwXyARK+GmbAG3FtPOSH+4GRbgtOt0tNOSBts3ikVeoI5JN9LY8M4iHo5BkmDIysV7wTBnqvcILsOljF\\\\\\\/77WDM42\\\\\\\/sQI3T\\\\\\\/TB4VwdebwoRR4ystshQ7E2NziRk7t\\\\\\\/\\\\\\\/MqZqkJHqLN\\\\\\\/dzFD064X6JagcukTyDSTwFzhesdKk+lVdrdG1OZTX5jmMRKE0UYY+PisWUSYh9NFr1cdT8mGWVx6\\\\\\\/+CA7cWpXarOlK6gbf\\\\\\\/5Kvk1aKbvRE81hDJ87UFdQ95yUi7oSHLiZdD97E6ZveFiJ2WLBTLKxomBfUhyEioEQzIsq3+qMT6\\\\\\\/kETzWaoEYEk1cG68sMgYuv9aqyVxEtr7Ot1qokJwHC4kcTRcWTlFMFDlk+pWkNdIx8HiHN3NlXxMERx15H6QYJmjqO060w\\\\\\\/IAL2o1hzgXc+ALEi5XWte2m9X73oP1JQLxv2NW\\\\\\\/Ecil7B78q5iioidZEJazBaG22uielT0VaM92ifLVACzILNHXHglE9Fo0u2gqt+soPc2HfF04nU8ZNQa82xKmqk4EWngbTwDSRaY6yGbzBu5DS1ohuiNWLKqURiZ4v72knBtqHTnxXAL4xXXvBiAShyWPI9W1nWHoFrvc2UsPDDWmhImcUIiqkP65HsMSM4B1+fED1okqq3DHQczr\\\\\\\/yWJ71WS3oj50hAk0Nr+N035uTZMZqzlVsPZFwNz4265VeOh960rLxgF9P5ouPCUnPQj8Z4XDjJGaX0tb1drL\\\\\\\/gvHxEdjInfKg6IF3QVlRgQ8Z4GWpH5RmkH\\\\\\\/szfy3r9nW05\\\\\\\/ZsGUUxlLjPP0CtAP6xzNwL2erFvxTh+nQSuixG5i\\\\\\\/KusnDIYT6A3hNss\\\\\\\/fKbVYv8\\\\\\\/s7oOenpKDzfSaooF7pceVXh0NGsARl5S73bxCwNJtIpuO349D0p6vfLWdhj5fHMk9dFNTmuSFCZ4FWfxIAF1CRVZk++kGlSpmLTSQ7y3V7Xzr8K9TpuDK7ilCuUeNU92KaR+aTXsBojdhu45FjgMdPE5GV3ukPssF+7tQ1v9WKorQxw97mM0Xx3\\\\\\\/LtDxOXFxXLDJbNdRGY0oK3LGYC04vx3VVacTRF0dcYHT6yhu82uuhOdRWWklRr1lQlVhZygOQB3vSdjh1r9RJhGX9xtNBGBVOO8kVQavivdNeozLpVowLbsGE4D5A8VCiTmx\\\\\\\/avDszE6E\\\\\\\/6JrOcD5HuiMmOlOXG0bvT4KPxZsSHBqAObWh0KsuWhqmnA0\\\\\\\/gmj+9MMHdMKoPN9\\\\\\\/ELSFqt9ugtzul08L9KFex1gQPzr4sWvhD\\\\\\\/BZYBF+ECscxFI6jgxalqngyR4i+NjL0eq7B6645b5nqzdDxobLxFOkXkN+yaGWPApSbl2FhajK6UMPuLXlz87bJcMjkmhSjD4zGl+OaBV6QT\\\\\\\/KXa79zoVmJQWCIUNe501JSZXaw5EPxYQZxs1XwKwZQuO2XEv\\\\\\\/219x\\\\\\\/xGqgtlUOHztVsVoRwo5BZFyc7\\\\\\\/WAmzXUcOh63rYBPAeXJ4\\\\\\\/XjYkpZ5Eg5Zzsp51Bf8FCn33GtJhWiIVL3aKdIBQpr+TjOC1OcapaZU\\\\\\\/IdfSjWx6Ew3TrmnVn15d2cAWYB1gTfR+dvcCHaXQN0v7Q\\\\\\\/hmanoFX8ZKRFSP0U2\\\\\\\/nbpS0Fb1zOklBrDjjn9FqDqPb9fDQc2jrrwuG06ft4gc+LpSzIebV+Xfs0gqjK4NR9+3OAYhYiAZ9bnwPPAnQqwjwSAWFI32Duzo6CmfNLZk1oz6t8odsfbX4IAYO\\\\\\\/FBtrq+ZDu7kHUaLlEI4M7572sYLGi9QCINB8KJFeCE0dDweORToHfXKdXIsJGTFuPRX3RxvyJCF4XLrbOv03xv+yNgAwnsHd3A4O+8I3ZS4SWZcRCxuehxs3pQI6DuOQyLORsvzOardgkYEUv\\\\\\\/RSTqrLPMfBj46xvjbFSK\\\\\\\/u2AQjYoqxcbh2UwAtvt22X1JUM3gRNL4mie2Hkf7YL9lIAtWOzMAUcy\\\\\\\/Hfo4oKrzEp0BNQItBeACOmRK6qpvXW1Y5fRliSzmO9XPCQ8DZsFvKRMNXgtggLzfwZf5biO9Y\\\\\\\/3gkjl6RYKV08FiU8mLQ69K2cC5WhtWOjKyWycanRW4bZEjnbT3IG6bJFHP2KEY+hP14oRBeB1l8mNKdS88+7OGvLJHwPUHKRxmodzdfRhwXAGeXtlV+hKrys88OzlC0EoyeXbymBqdKS2Z\\\\\\\/CpnQipd2WbU1tOBw5HPaYJqHaE2gwwJ9AUuY5xaiQSUsyq+DJX0Se67vcNVY31RKSJnDH2cM9O20Za25+uam7kejAv2RQrfKUXpHv\\\\\\\/5GrgBYvxsti5cQkQ4PjAmHE7pdjdvPAEmw37L1gTTl6L3OcCVGHH7NJLlMr4jcuELuJCHwT7DxpL6GeD3fPGFdkLqGbxAUU0jGj1UhcnHtm7evxdyI54J0YRnDV9pTMtX\\\\\\\/K9W9J5IPyal1Rl1sShlbQ2f6QZNUIHDbtiRPP8EQU+jxOvzMmQv0hhmeCPV78xwVH\\\\\\\/VFPubW4YPm9EMXY0qjqWy8XeKohk\\\\\\\/POg2\\\\\\\/taYVEp3QN6R6BmqnjCe\\\\\\\/yHVQurspsw9qMVolzZUSth7s3Amcrgk\\\\\\\/vx9NFHYXni4DFqkXktwxYrhj4eYngT\\\\\\\/l09B28mTCoO8geXRzPl+GEr\\\\\\\/zgs5LNBUo0Z\\\\\\\/fXBTIdfp8pIo557rJzl6nZLg7lQAeqKmjUoglvyT227UrFTZsX12zopcp+u1iNaWMUk3yaRiateQ+plLynd\\\\\\\/xazMk3nv+NGITDzc98m6XTTnK6cwiNbl3kDmbyIiet21zOFqZbW\\\\\\\/o6dYJaACin6q3wcGF8sUAYwGjeN2sa8lNu6yPRbQUBGh0p0RpYTjf93lcwre\\\\\\\/6hewEUBktfcMaAzZVou7hXClOkMcG8rQL+i5HHiQkLgr98UvQp3wCPA7q+c3JJay\\\\\\\/Gn76Q4M5Iuy9oVoFCFlbcImpNVJViZYKGCYV1lODmSN\\\\\\\/t1WOTCRIVVXZ1YTQCytlW3qKJ2\\\\\\\/paN81xQqRcArtOkAmdCh4xmBVIpm+PoNrzKe6sRwYyO7Bw9zKmym3JU+7a0APuftsnl+6HL6yvUAaIAAvltX5+Uukpr1NsDda6vTNxrjeI+jWJRvwYduoJ\\\\\\\/ad3jExQjtc+IXIXBPe6s1AtCNRD7c720C0DyRWgo4lgxbkByZqLLqYT7MZ5hepQgDEktNVIF1cHjXx5S5ZYf0Xpy9vEoo\\\\\\\/XkNO57LbkNw5dpP45x87f3JMiGZzLbDw1eo+Yvykg+GM7SSs6sinJDjLs0tKy8yA8vB2MS0zgruH7Vk7g+QDXZt8x5SHbJRc3aUMVin+xMK6XQSvLjAxmeMfdsW6WU3H0sZqWz\\\\\\\/bfwMVqIWg+jB0F6kBDGsgFIEke6f4hmxjOEPRzMGPwXyPphbPa02IfpCreEddUQixdQARcRo4f0kfc\\\\\\\/3LZ9o7f9q2gJ9QnD2rfBsUwGu18ps2PeYzA6Mx1sgkRF\\\\\\\/afySTPedGjA\\\\\\\/7TAwXQZC+HE\\\\\\\/Pu3ntC99lzTdQIqhBMRcoVSX0kwaFQX3bM\\\\\\\/ZMNdaLBNz3kQgid0msw2GcNlkMtS\\\\\\\/SQYbSUSywTjKEY5EpdJ3EBFWC+TJ9pniSQJETnSsRrO7mLZXpy7tIn0YXw4Po0j3WUax9mwhBTXssIGO2gN+hIzR2UlVqClw6Ub7e3bgjnSj8ttAYqsiMtZerp0zttWi30BNt9RONfcBs6Tr\\\\\\\/mYq8VKDZXHmWJvV6sDFCogDyFrAu5FC5cRq1wqD7x6iXCaxmTcEe9yqF\\\\\\\/FwZNYJnWTQBW99QlUW7xjy3pAzcte82ANUgT8vHdbuZtYNCNT4V2AG3EwcZViflKPNDQPM0+Xn5Qj4InvZSZGE7ZI+sa3CqJlFXcN9i5sFXXDPficf1nASKT+\\\\\\\/wbnOP07gUNkAypnTEseXbeVbqoSwPsfMZUH09x4gbAL3a7ck92f5DILyKzbTjm\\\\\\\/jys+yw3zG5QWW4vaASuCERdM8IR7piXap+0qzn8yXa7qhYOeu0dlK6z2jtiUcsEEFn9Pn3w+Pp9DnZxfRFIRPdwz3fD0mqM\\\\\\\/GJkKLWX0zxHjpgvJXQf0ysKsfMNwcHokwsaNNZvw46PeokrTURzPu9JIkaFlXbNFU7YSEj3Q53plcFeSAfi4Y2ci5mS2OW9WG1eqzgM1IIwEeZs1kgeaLA88vvkdS648cTvhg4mA1S5VEw+Jgr0wHm8b0LTqOxHXx6wa\\\\\\\/G4IKEn98Z6XwvlKFXlI49RLBarq6DgfQFKvJWbqtZ1Q3Jw+YVIlbOwH+P+ThiIJeqKbk6GF5L\\\\\\\/+PKloP2sjTF4nsjBiq5kv3cv9smYwjn2wlJKj6O2+799cmSbKXWD1V8aQbdYS6isqu9sqB97DReNEvBBIhyIzHOYyQkTXVt\\\\\\\/wIPXEHP3MMfSUzvjolrWPpjdPiFy3QQOHbpqMPkbkjiK+cf1x77z1l9U7aA7bYtxxAxM7XwTEYFAM\\\\\\\/doCwUUVawnGonsQ9KtCyr0Dtkpm1hYbxIqGI\\\\\\\/UkLWEyRt3t78C5tA8GyIXvVsDSVlkvkT0\\\\\\\/tkATUQsoHouAiI6Jj0hOwsRJ2AMemdfqwbercCE5jXH+u3jxz7m2lGnaXEDThGEBhXb1IgKxB4WChLLYyfxeAwrbDys32IUJk8mzrC3Afz8XY0ta\\\\\\\/Bq7lb44kABsx5rPqBvH5sFBoSyM6m9ojezAKBn4ay8bRXrkT5uf5BaRSHDdpC3vpl84897770qxiT\\\\\\\/BQkmtpwNZREHIWWh9z7MYzXcF0RCYF5EoE+Yu09ah8Jnl\\\\\\\/TNWOsy5RBlpAF\\\\\\\/ZX7Ynb37MaHPIvTkdIQnLmPN8BqClhyyVVkFrc7eTQxCiA\\\\\\\/481vzy9PA6G5olBl7Ul6o0wBB4bl3TBjqITQAbJjYF6aLAsbbCoGv1DRwFeBUqdShfNu1uyFZ0tLYECxMRt3Et8RlQHraDoNtiHYaLUXQntzWfb19qQwqhGDAaiz9MnlGZUwGEw\\\\\\\/qQheU+EqjpsDn\\\\\\\/9bMxvggEXXwpWt+IjyW\\\\\\\/jzdUdjsgILsQSaMc0yH7wF1VqWGQVoyqE\\\\\\\/rSTLofmDylz5ZqtrDekm74V55PE0qNKH3X0U3SfNy2ThUfkEaxM7aXCe\\\\\\\/C93bF6pEt8DUJmDIxOEoiBDka7y+UDSN2qdmB4DOLwqKLNx3Py9gayGEmALwlhcLA0kG2YAqj5RaorS1XNyseBUUVfX6RtWg\\\\\\\/W3CwPMlf1vSvTrjdH9FMf+4AA0ovGnnp1i3VDLTtQIytlf\\\\\\\/Yy29gjj7mQfIfGYLA5gtJ7\\\\\\\/+72vZa59qRUvOsbFzPr1ahFLb9LW8bE4iiRorg2ZhswmylNgPc9O\\\\\\\/d62EjZqNyDsyRkzu\\\\\\\/BcVSWs78ZnJrPoo8\\\\\\\/JFQ1QeLDW2mRGU+egHTwVGN4YFLxXAmAKRapRxtsFUlhjof9BTPVcvXdvrQkrAlFtQcyG5q1L26F5GH65kWWwxnY+2Ss2ulu+NQQmVHEXLW677CVgGRRbvDBP60HDghdSxZC75DvVmj5AuaYn1zFHzn8lx28KSm4KMvN3QMX\\\\\\\/ebdKyHKpbnNMzzMIgZ72Zg7biQAEBmfgTplZxIlH+PNh2fLBraQGhvU3v0LIC9iP3N3F8cSXbAMnhJZxAtwxaqhwU9qp1vLM2orWCThI08iJFKEfF8DCey0iAqWhOi9+yKOcyzn06dNKJjsiahAQfiRVnLLX8QGBDNBiYofQ4E6r9dxBnBzAkHd54zHZ+tvw8uK5gOX3Mpp2e8J1xitfBIxE6Fh2dPTeYPQmh8id5a6PHOyv9WkD3DxzTDL0USiCoWF9oMGPzgpF+wYJefXmIPiTSWooQXABAxxsJzkxDAPSJ4o6fp913V15Pj4Cg7ZmVDhoI92Z8fThxAyTCRmUoKGYutG+fND2Vt4EJOzzlBPPYT6zYQRDpKwAUEnq1plaTY6VHlpJNt5UqZcDTizstCJJ\\\\\\\/IWn4rgZbee8mF5xiN9M6cRBU2MzohrQOkljrvUKO7t10si0S6wHdMnyujVJQ\\\\\\\/StFou8aXgIxLluoDW3q+55UDiB+TDWTdMNJpokESW5Y+o9ZqRWOiRcoSw+hpCtDNVEd7z6vG0HrnuyrUHc0+Tte1z5D0ue8hl88Y5lJ030mVlTE3shA46wITuAYHZ3JUgKS2PZ67ZFuBesV6J+pZipk8js8k\\\\\\\/0uH5MA7N84XAeD5KoDx2OSo6omd7krUK72smQRNYnx2NwMcxSxQ4wHc5H2K7Shdyh\\\\\\\/y5jAo6yLMs4yw+PM8EPvfg0nI0MUHfGDHa2w2AQov7oq9NqZeMXGLTbi2\\\\\\\/00tFlSxDNiQf08vwnaBkYkRrSMAL0tu\\\\\\\/liqBFnoP0QgRqaUe3zor0kRVOyna8RUzT1547ti4\\\\\\\/+CG8gcDpC3wmaRYwQuQ4rj3FMDiHh0pLFjCSjFs7W8VZk4dR4w3aajG5WO3bPa4iajEwMixCl\\\\\\\/FCttPL2JNDMANTW9XhTAqsHsXTSX2udujC\\\\\\\/6GEtVxdqDJSdj7hyfskIqNcn3017IF9Cy8QLp7187bSRpV9Xt5j3Jbkdco2vrWuSNpUZUM\\\\\\\/4kegjCM1K9ugeED7pRsYSomrC9zzzezDvvu8iAyYhJ9HNr1ee+6uBlff0RR6xHvMlD\\\\\\\/kwz\\\\\\\/BB5S0RS6XxVncOAV+gKwIWsr4o\\\\\\\/xFIfbEE0b83QIlrWl77SVuA\\\\\\\/ro7s+a+iLxfbfw8XWFaPYPODqo1zNKpH3xx0tqB\\\\\\\/YV4BokLTTU2vnjIPckOf0fKsSU5kl\\\\\\\/9xzIibfvZmzyIDKO0O0ghpbUGMlgZ1nOWgsZU6kkzqDq2V77pYplDDLm7E7K4QfaphSy3ByC9SLE5XSoytua+CwwMAyAXRgoCr2\\\\\\\/rmN+vUQQecq2dD5\\\\\\\/0DSAANWteOhQAcZ2SJNY1mNKWsA65jw3whhIg+vdrLUL+EcHuk78wq++pDrZjcdcYx+NeMqVMOP5LtOjjKkic+74VcndZCPZOpV1tQY7\\\\\\\/kXFOR1ELiT5Fm1k+ACa5eKB9Qw7QVStZAH5q6a2wArCggOJuThqmxYU2n60NK7ZP8sJjcT\\\\\\\/3YLVIl1UpUv6w\\\\\\\/EpHiQXl\\\\\\\/81qGcxsxkXXBwTTl8cTdC62ViLpfa2t93lWiX\\\\\\\/XhReHG6JuvQ2tzAb8BlW9nFXTGZx4kWwkyJ9S6i\\\\\\\/nkjC2usv9J\\\\\\\/+1BPSG1PNSSkyNcD5lezVRztRlj2VFBUHESeKqfA03oWIZ0KaP4pQEgmTTgG1PY1amIFSedfhspL6k0xbcdVwD9STFsk076\\\\\\\/9ylv1v7Og3PjxTu47c76XINWVo893Qz5JcMscRP0kvjwbEYGLBTIGakAQHzfHDST0zw691P34zT3YkBKMC4uY6j61Gj5XEesVygqLwhWdyl77mu03JbtWbgRYD4jZQYcpW+NDUkUvUoZeeFYz6FTxJJ\\\\\\\/zhFeUPKJsqCFSNzA2GrbC8D1FI2qoK25l5bg2LhhXsAMcbDWNQRaYGoNv2l2X4bX4a1Qe9T3\\\\\\\/ExNm3VPU+5tjYzVB28SXuysf7GvHxXeEZo6aigWuj+zhaF7fKjZkMDR7XBkU5IfIFqOoJIdEcUGIX0V3ZDodgpuJuh6eHMpVhvIt4Aay5okckgKQhsbsnbYqMCuWDebmBAlw\\\\\\\/Hle9UPEs2BV61WH0foc4A5GJ6posc0mWUhLvE7KFjHyt8Rj9zvXCqiypzl5\\\\\\\/veaE1WnCEjLzX33Ue5HqU0THFyjubAxTGJYK4G5W3t6PKMQUXKpKowgeGpYd38a7TF6\\\\\\\/xdi1hzcNQSkMZ7NLfdvZ63G+ACbKB56M+c8ZhIGODuu239iAyAlotyRXQ0CfOQ41J6XjMEyisK4o66dOqCPW5ODrLRDPqVhF0bVYyf5lUXbwzhxvmvG6ALvD+0pMv6KPRv4NOZiqu4d1AmL4Z2hOPUYmaLKDCPQ3N4YC3fefUTnPVy+b9iBT29WKx\\\\\\\/WI9gO488zo6JPSvbcWkR3d2iikuSBzoqEEn5b7JU7HYwIKZhgGhwfSpm1y\\\\\\\/A1ANA+ZLqDSCHzCgOcYwW1s4js8JOohoflwK+VF2pU0yePZ3q3nOhP+x5mF9jOErBxDPaLghWSp7DjZ0MGynWzPX\\\\\\\/6SZGMgPDJTiHht4mQRs6hFZc8pQYFYPLyADBhiNMU1qbPTqdtKZKp5aVU\\\\\\\/sxfRbkBwBlMnGN0Rl9x3bY0tU835NDaroXYAoqvHocHGXaOo03jr6SKWWjc19lca4UdP+LjcbG4ZNHXr3l+LPAXp9LZayNen7W2qZLrXZ9nMYcgxp6dKqJZxPCXrWJC2jDC0WzKrMMgOcKcTk6Eu63J8H2UuNI6nRfJtI\\\\\\\/m9wIIw5I60gmTDLhf5NkxL2gGtyYDHGqTc1NTo97YQr7vLuNSavn\\\\\\\/xRSKB1HRUzVMhirlxbD45hw3Jvw9zWnvTGNcr1du0YlGCoJWFUzIbkHWMcnZH0Lf2iHcfgqSh49OjhFw5DqxomdyQiVeEYwvFw3Zf8isRdlCafapPtjK1S1Q2Xaz0aLJ+RJWH0QKV+pmrWghNygnZkExxQQsbGUvtE5mVQYJsG+h7THGSXStc8TngLsN4VvmPcsRz8LyJuhA0stWOegpXgMvDNzQ6xlKL5lU7WNG\\\\\\\/7kdiAS2RBsXFpeswU5BVbO\\\\\\\/pcTsJWT9MkTUkruC8BKXvhE0rtTBzcVbPdR4TSeq\\\\\\\/oti0zGT1IbaiCWnVYzq9kwM0wUWrFDF3dU4PmD\\\\\\\/FbYlvXb\\\\\\\/NddbSCqR6I76YyiPJ0vUa6IhcO8QtNJ4OUyXhMmjf5thczF6klFRXgVwdlZrOFp3aOfI8dazHsymfqdaqjk8fvWir\\\\\\\/kqhti5rkridnIMGPBzL7PtvngtA02InOQqqhlUB64oghFW9uKUtXdjGucZIyAwm08V6x5WSHikGMr3Ql+iMHcXjAmGnjY6frf99XM6v\\\\\\\/CiIw9Skml1gf+S02KxUvQT+4ZjSsNidKlQgdPupthizKCebEGcZWjUv8F2ifVNI1loOgtkVewJqLWFUE9cD9prq1kcJSB6tihp0hwgSHoHLOaorZi4\\\\\\\/GIHyrnVxCDkZYlSXFsBeeCPNxOgtFgmC1sdSDO67jPf0elkyGATqAXyzXns43d\\\\\\\/byg7WkAsoloDQ5hd9XG5+YRKaVE6WPq7g\\\\\\\/cvXaon2UzHBTxfMkMLDfQv4Ah62g+NONgSHKtIG6mYvbwxKWLSSJoQF5nrkM115Hum2jy8XSQx2BiaxKsx99CkgjTXcnc2s\\\\\\\/5r4IZ55tUFrteS\\\\\\\/SeRLRuST9Rv4tQ3hvqFOKI5Ks+NooW5rgmxFpiKXBiKXaqPqhLdv9kIKs9EqLWiZMBPlB3sO9byObGVLYECNqTVUmDfatkmj\\\\\\\/nDHtjHMMii\\\\\\\/Ps69imLI5jhjKLUOVUTIRVCDlBrZOOFimg6LamuszkQKnO22\\\\\\\/eVNWweq0IxsZ9d7DNREIZ5PcPz13CvCvdEfQrqROY+aVDuFPwnO0PHo7BFSlYWq3AhO+P1gnXb8\\\\\\\/vQ37MHZcP0wHRnpRqb+Vv3GBU7tU0LMfFjgNpMQbvpfbkUpaMPsAzs1eZ64TVVIEerN75y2s6mvq48LEX25tcKMbpOVp04JEGKwLh05lGZbCjvo+2BkEkfY7Crw2jKYCi894EXyH\\\\\\\/ddrAk9JdXJ96mdNaXCnlDb2RdAeryDK1jJo0FF1sp8ig8G4fgghQrcJXDay11W5vKcmU2I7tatQ5ODyBMTunGilGKLbIJHXgp\\\\\\\/qpe9xTtPCukab4zHKXnigt2WsWZXibxeBf1LlOoWFQt3F8YxP1vd57sZdWoR8c1v85jMXPZ4iQaXqZK84xf4g3uAvREITX4ekyAdPmxBQ\\\\\\\/Mer9p04oHG8+NiCXNFFT6+9a85z7\\\\\\\/Dz2nYiUTvVAarjGRcSJYjcWUSKdoAuEOS\\\\\\\/fkOqot2Xn7SWHgMdH\\\\\\\/\\\\\\\/k\\\\\\\/dvntY3tOZ5WevZHF93zxRiHle43nYzHDe\\\\\\\/UltFMwH6OtkuNPFLPoEYV9F2dtAcxTk9qd2LWG8WSoV7l2MJmVrNJdcZ1\\\\\\\/GTdmH0W2qSGjG2lfEiSIUCkQO3sAlH8TbXhzbCPcaADINoGgBENk1hfUJJykzBlDLSVUiMJdiaIet1QLgnFVowvokQKS61PnC7uk7d35NI+6VogVslBJCRSgHIvGj9nI51E03aLp4U2moV49wIcKhZ7NqGaMkYq9l\\\\\\\/El8Ecsc\\\\\\\/DOBxkPpBNaOlCct8hEhFVcvear40DVoYwIae3nrrZUOwGtRka90twaKr86vN5vylVS3IVVd65WiW6dFr6WJQT3LGgsbpRZGb4J13G2sF7B\\\\\\\/GtAIRHTGWR5a6pQQusdTSLNXIz12r8w\\\\\\\/wvO4WBQpYetRBhWLYU4jyVVuHDUdH8j5Bp4XHONk8k2KM1LW3xuxXM8jR\\\\\\\/Cd8iyGoq3s5KembEia7wZqUHbsf2EVJW3+ePjVZe3kphaf7ognOc+NF5OKqbfZ5TNnDVnfr89XZCZxuGoTblDiXh7Ncwi2VE7kgbXU\\\\\\\/DEDQmNcmilIkXMZ7DiYXRNB5sbepbGW7I+dPERepjFwCS8uxe0FTSXim9CzFgSQGaRB8efws3VGIGe5+H2X\\\\\\\/ZG8X6VgO51oSibd+9q0968m\\\\\\\/AJkgLvZ0uQO\\\\\\\/z8WNWHXcxca17jCmL4ORfOypeVR2pnVC4G4eVsLKUdsTkPvpBKc0xRQrR\\\\\\\/mZ6AwvgEcejgynwsiTNf+LdjQ59Kjpaqw6DB1laUxvmMLOdlhDymQmZDqEnltfTp+YAVymQ27sP\\\\\\\/SkXC3fCTO1Rm6PO\\\\\\\/uTf\\\\\\\/mGPT\\\\\\\/0DVt\\\\\\\/QzOg+xgDF2TSZ9X1pXgSY10SVktTFrlZ4PvKA7Wvuxozg08rnbPuGjrfWBPBcB4exC8eshoQMlEtf8FYlCzEEoGB2NkQFdqYvw6URbQwgpvmCcy+e2ZAUQtzFcng4p4HN1Byz16j2kZIxJvSskYuRU04WGK\\\\\\\/VHWxyIN37yBka5o8H6U\\\\\\\/LH4n1ifPeRWGnrwtBft2x6qVOTNKoPfW4yfuhbS\\\\\\\/YrhfpKl7aqA189cPlYH8lElRkhyhh5q8GlMFeT0EjM89IsTSQhEB3ipFAyY\\\\\\\/ZvxgLLijfwAco2urw0stRE9EASajWAa8dfriJCEhUTn7S\\\\\\\/RIoxReoiWLWH3np4smYuYoU+dLu4u85HlPcAsUNs0J3N6Ckw3KKa\\\\\\\/Mnxe\\\\\\\/8oD8e1hhRUwfD9eG733M3ssNDhiQS5Q4R6pygTV5fDaztNkxlr0\\\\\\\/EsaMY4gGbnLm6f6VIurb2YzJdvmP05GAIXpCuHFMeUXAvN1RSZm42lsr3vtsyCiSy+4uyDnnrgm4eQ70NhWubMouAl9MqX\\\\\\\/b52tX8EpKq4FCMksgmKr2Dg9pBoD9qMZqH4ot4yZheqTLqk6\\\\\\\/1XeqD9+V7YUckB09lI9f1LoUckg+CI\\\\\\\/SWVHbQ8hOjZzpjBmdGliJR1eS7EbVLKQPqyb5AWijcCJrEXmVR58CdYxTmQQPHhcL8OdEA82Mz6dQtLkkVB\\\\\\\/WFlw+ueAvvcO4DHw\\\\\\\/ClaYkVGra91P5gc09aHAoV2baWAyb8yt6TWNSN\\\\\\\/nHfO5NZlrvNxzQN80xYGoK+ugC+lGNUMB6LGAucGIHnBO0RWVDwZTuJdvHczeDwuaavcI4KDrC6tSnW8cbPHze2otgOQwP6M9cidityQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022e3mANSAk2xXut3JhGYAkFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228fb26c6f5ea94576c9480971214021d3cdcd436c919f688af66a62ac4846a687\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/susieOhtWXm1HeQ+RFl7vMP58yneiS30bjjUZrj3BXAopOrtjl5LdKj5AGFdyV8SwaERX44vCpWcuuDdUdonfh4yQ1qsTe3Rl74cpoTvAaAWHswa8PGVmAR2U1ffid2u2R7bnQPD7X47hn4tAYjNmpvQyHh8nlGZM9GeEvxQmFQEqvEdIfMX5kdY+izvToIi2Fy\\\\\\\/nr\\\\\\\/3S2Bx4Hw1zTcTFsR7Q\\\\\\\/xiMaRxYnrhbWrkp7yIGn2+WMYwRwnb6p2\\\\\\\/+dXuun4t+LiqpydWjZQBllX529vyQc41jMhsAmi5OpNSSVS46Cn+PXjkuKSbyLBT+y\\\\\\\/iFxJra3mjYbQVMSuNNUq7JvcEEK3VdBLCGNq4sgG1wMndSWo2yWiAhZ3Q22nsbTp+G97RVvHCE3MzjQ+mnbvt0KQfhqsgUkq+QEGANjnucMAx+dWiXbRTPKe+Z1xotMb\\\\\\\/YpFUUQyeVpGMarknjqNi4aCoh5VOyRE3eOaVJay\\\\\\\/SLr4Na0OmMbAiw3IpS6dvsHPpoT6u\\\\\\\/WwKkLDr0qpb4RTP17DbK6pPK85vWNzWzpDMVw4Avw1kMuAdvSqjpmk99\\\\\\\/+rahvyLUWFDZMpXd6xb9noSwv12849n4a7IMM0QutPvzOWOMXEo315YOoqaL36DNX1Dn\\\\\\\/cEB3hU0HtnfCKkMe4VlXhHpGStV4GTt+lN69oD9MmaZfwc57fR29IqczOz6e39tfNxPFrzhYiWRShP9PCgLRNQZOrF0Qqp12A8j8y4W3C8WrBsNO8OQdM27A258U\\\\\\\/3uaPBeyYWBFZ2H17gh3s6612t9FSoEFDtCgs1Or0BoxvcCjfJ5TjObiyKc1tXSgPDZd62NOKtY+p0z\\\\\\\/4tUGExqFDvvHP6gCLjqlLDiVkUXQpiO45bpSrTn9LEOGBUCO7Pawhk5Lp6DUJe4P2QwvRWJOjhVjilxRV+EC7odtwJhoxON2sp74YIIax47jb9KyxvP7c8HTKqxEsrfejK1DvMaqXw2BahvD83eZw7IV3aSEI\\\\\\\/f0M0X1Rvi1dG\\\\\\\/pKb3R\\\\\\\/z\\\\\\\/QL5tjnV8xjDRPggKEVLZTnrxAGAspBTr79EzW3ijxXbt6XJ+Zyu4WrMIS2XT4elbXg4SZKbaUivprstsCvzc0f5kp8W0uipFajfMSiqt+Qa9Fxpo8n0NGI7Xx+kD1fFjmFgwZ01UBvgdLWLBafeYej9IvTAa\\\\\\\/WiQmE7BUUjpcmePccbxezZrCuTbKy2OHccUlQ+621NiY+LABGv3oV3Ez7yaiV+jX5vvzPFLq3c8HkAfcuw5g14dcAMVS9Cn3Mp+\\\\\\\/xRmlAgukq5oi2JX8Hijz2Tc70Hzcj2d2arC3kHvgMRsc+LxdbOOfpvvhthMQm7iQBMZcAvsSe11LXoyZP2yW7wC5BSkvgmGKV6WqpXxjedNi92b5LCHC\\\\\\\/CBeMwPv8S8bLrsaOcCP9ECrNVReflK8m5YnWWxsDw1b8BJboOtH6CrvzFHbqwCymgQ5ek5493197ZeM3RdxtRdtDdpNu5FdK2IMyq59soyqyNkH+001u3ntVPsWMn6UopUAyCee0U+fX\\\\\\\/i2e2QxHsJVAzdZh9LalJESfLAyFKrOU2WYOMK+xwTm4czA5H+9\\\\\\\/BBOdYokfxEUrscwuZWVhuVt4VCdJBnf55kWlYHupxiYUyz3H2K13oMZYEcGqjqk+R1jyXrwi9dTasXSy04bXX4\\\\\\\/4lOdjd8BQBT6zypJt8E8ayrlKRg6xides5EaiAMF1FYy31\\\\\\\/jbCv6KImOItTW8qqK5uPJYh3x4EgWtJnMMWs5zqv\\\\\\\/KqdeOZ3XyB7EMBFKwGiJqbZPeD4NlmpnFx3PfS2FEu1leP3C6uCwW4eEABV2\\\\\\\/16K92NWZybqnXSlTov3s62ZCnRb+KtEi29e6eSpER+a4xG6Rq6h49YDuiteN1HxZ0D7YDMHngBAJXw3wEzE3eiPgx6QiTCTEmqg1zgEU094mTJiX7TUoVaGFc8kQ+wx6YO7C8OKY7vMUXLQVjDWJcik5FtzE7MEqOLEhA8TjXgKOnZJw==\\\u0022,\\\u0022iv\\\u0022:\\\u00228j5uQWf6pU5l6IoUO1iW0g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002235a8758027a41ffb23a38d70785d71d878b9b8e5c3b8ca3590fc6a4b57d8aa8a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493254 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1825" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GEc015Q1HCULUIkeFTQsSBlrs7KtBcGbqArydQiUSZ4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1825" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rL9eEA0RrKQs1AZWc6h0ofGbyNbnKHF4MOP0Bw_kQaI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1815" title="Negative Regulation of the Mis17-Mis6 Centromere Complex by mRNA Decay Pathway and EKC/KEOPS Complex in Schizosaccharomyces pombe" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1831" title="Full-Length Transcriptome Sequencing and the Discovery of New Transcripts in the Unfertilized Eggs of Zebrafish (Danio rerio)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493254" id="node493254" data-pisa="ggg;9/6/1825" data-pisa-master="ggg;g3.119.400273" data-apath="/ggg/9/6/1825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2874-5990" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew Routh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1825-1830;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400273 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493254" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew Routh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Sealy Centre for Structural Biology and Molecular Biophysics, University of Texas Medical Branch, Galveston, Texas, 77555</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BRouth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Routh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BRouth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2874-5990" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew Routh</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alrouth@utmb.edu" class="" data-icon-position="" data-hide-link-title="0">alrouth@utmb.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/6/1825" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493254" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1825.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493254" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1825.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493254" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/6/1825.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493254" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Poly(A)-tail targeted RNAseq approaches, such as 3′READS, PAS-Seq and Poly(A)-ClickSeq, are becoming popular alternatives to random-primed RNAseq to focus sequencing reads just to the 3′ ends of polyadenylated RNAs to identify poly(A)-sites and characterize changes in their usage. Additionally, we and others have demonstrated that these approaches perform similarly to other RNAseq strategies for differential gene expression analysis, while saving on the volume of sequencing data required and providing a simpler library synthesis strategy. Here, we present DPAC (<strong><em><span class="underline">D</span></em></strong>ifferential <strong><em><span class="underline">P</span></em></strong>oly(<strong><em><span class="underline">A</span></em></strong>)-<strong><em><span class="underline">C</span></em></strong>lustering); a streamlined pipeline for the preprocessing of poly(A)-tail targeted RNAseq data, mapping of poly(A)-sites, poly(A)-site clustering and annotation, and determination of differential poly(A)-cluster usage using DESeq2. Changes in poly(A)-cluster usage is simultaneously used to report differential gene expression, differential terminal exon usage and alternative polyadenylation (APA).</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polya-sites-0" class="hw-term hw-article-keyword hw-article-keyword-polya-sites" rel="nofollow">Poly(A)-sites</a></li><li class="kwd"><a href="/keyword/alternative-polyadenylation" class="hw-term hw-article-keyword hw-article-keyword-alternative-polyadenylation" rel="nofollow">Alternative polyadenylation</a></li><li class="kwd"><a href="/keyword/differential-gene-expression" class="hw-term hw-article-keyword hw-article-keyword-differential-gene-expression" rel="nofollow">Differential gene expression</a></li><li class="kwd"><a href="/keyword/clickseq" class="hw-term hw-article-keyword hw-article-keyword-clickseq" rel="nofollow">ClickSeq</a></li></ul><p id="p-3">The abundance of RNA transcripts as well as poly(A)-site positions can be determined directly from RNAseq techniques that target the junction of 3′UTRs and poly(A) tails. Numerous approaches, including 3′READS (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Zheng <em>et al.</em> 2016</a>), PAS-Seq (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Shepard <em>et al.</em> 2011</a>) and Poly(A)-ClickSeq (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>), are commonly and commercially available and can be used to estimate transcript abundance, differential gene expression, alternative terminal exon usage (TE) and alternative polyadenylation (APA). In addition to providing information on the location of poly(A)-sites (PASs) in mRNA transcripts, these methods provide a simple and powerful alternative for gene abundance quantitation to randomly-primed RNAseq in both bulk and single-cell RNAseq experiments. We and others have recently demonstrated that poly(A)-targeted RNAseq approaches perform differential expression analyses similarly to other RNAseq strategies, while saving on the volume of sequencing data required and providing a simpler library synthesis strategy (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Elrod <em>et al.</em> 2019</a>).</p><p id="p-4">We present DPAC (<strong><em><span class="underline">D</span></em></strong>ifferential <strong><em><span class="underline">P</span></em></strong>oly(<strong><em><span class="underline">A</span></em></strong>)-<strong><em><span class="underline">C</span></em></strong>lustering) as a pipeline to preprocess raw poly(A)-tail targeted RNAseq data, map to a reference genome, identify and annotate the location of PASs, generate poly(A)-clusters (PACs) and determine the differential abundance of PACs between two conditions. DPAC comprises four major stages; 1) Pre-processing of raw poly(A)-tailed RNAseq including estimation of length of poly(A)-tail tracts; 2) mapping to a reference genome; 3) an optional step that locates all PASs in the provided data and generates annotated poly(A)-clusters (PACs); and 4) a differential expression analysis of PACs using DESeq2. By determining changes in individual PAC abundance, DPAC will calculate changes in terminal exon usage and gene expression by collapsing read counts from individual PACs if they are present on the same exon/intron and whole-gene respectively. DPAC compiles these results and generates a final output table simultaneously describing changes in gene expression, terminal exon (or intron) usage and alternative polyadenylation.</p><p id="p-5">We demonstrate the utility of this pipeline by re-analyzing published 3′READS+ (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Zheng <em>et al.</em> 2016</a>) and PAS-Seq (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Shepard <em>et al.</em> 2011</a>) datasets as well as our previously published data using Poly(A)-ClickSeq to measure changes in PAC usage in HeLa cells knocked-down for mammalian Cleavage Factor I 25kDa subunit (CFIm25) (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>). As expected, DPAC reports that CFIm25 depletion results in substantial shortening in 3′UTRs, while only minimally affecting overall gene expression levels. DPAC, along with annotated poly(A)-cluster databases generated in this manuscript, is maintained and available at <a href="https://sourceforge.net/projects/DPAC-Seq/">https://sourceforge.net/projects/DPAC-Seq/</a></p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-6">DPAC is a simple bash batch script with associated python3 scripts, run with a single command line entry. Details of the locations and identities of raw data are provided by a user-generated tab-delimited metadata file. The pipeline can be broken down into 4 main stages, each of which can be invoked independently to allow re-analysis with new parameters. A number of software dependencies are listed, though these are common in RNAseq pipelines and on bioinformatic servers. A flow chart of each of the main stages of DPAC is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A flow-chart summarizing each of the stages, required input files and returned output files for the DPAC pipeline. Command-line options used to invoke each stage are illustrated: -P for raw data preprocessing, -M for mapping, -C for poly(A) cluster generation, -A for poly(A) cluster database renaming, -B for bedgraphs, -D for the final differential PAC usage analysis. Examples of the output of the DPAC pipeline are shown for three genes: SCL35E2A, MEGF11, and CD9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-852203061" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A flow-chart summarizing each of the stages, required input files and returned output files for the DPAC pipeline. Command-line options used to invoke each stage are illustrated: -P for raw data preprocessing, -M for mapping, -C for poly(A) cluster generation, -A for poly(A) cluster database renaming, -B for bedgraphs, -D for the final differential PAC usage analysis. Examples of the output of the DPAC pipeline are shown for three genes: SCL35E2A, MEGF11, and CD9.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1825/F1.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/6/1825/F1.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493349" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">A flow-chart summarizing each of the stages, required input files and returned output files for the DPAC pipeline. Command-line options used to invoke each stage are illustrated: -P for raw data preprocessing, -M for mapping, -C for poly(A) cluster generation, -A for poly(A) cluster database renaming, -B for bedgraphs, -D for the final differential PAC usage analysis. Examples of the output of the DPAC pipeline are shown for three genes: SCL35E2A, MEGF11, and CD9.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">In this manuscript, we differentiate between poly(A)-sites (PASs) and poly(A)-clusters (PACs) as follows: a PAS corresponds to the exact nucleotide of the junction between a 3′UTR and a poly(A) tract to which either one or more sequence reads has mapped; PACs refer specifically to the annotated regions within the genome (whether exonic, intronic or intergenic) in which either a single or multiple clustered PASs are found. The size of the PAC is determined as a function of the DPAC clustering algorithm, which in turn is determined by the chosen clustering window size (described below) and the distance between adjacent PASs.</p><div id="sec-2" class="subsection"><h3>Initial data prep and poly(A)-site (PAS) mapping</h3><p id="p-9">3′ end sequencing methods including Poly(A)-ClickSeq (PAC-Seq) (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>) generate raw sequence reads overlapping the junction of the 3′ UTR and the poly(A) tail of mRNA transcripts. The preparation of raw read data in terms of adapter trimming, poly(A) tail trimming, poly(A) tail length and quality filtering are essentially the same as previously described (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>). Mapping to a reference genome as well as extraction of poly(A) sites is also performed as previously described. Briefly, reads are trimmed and quality filtered using <em>fastp</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chen <em>et al.</em> 2018</a>) (parameters: -a AGATCGGAAGAGC -f 6 -g -l 40 –Q). If using approaches such as 3′READS (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Zheng <em>et al.</em> 2016</a>) where the poly(A)-tail is present in the reverse orientation (<em>i.e.</em> a poly(T) tract is present at the beginning of a read), an addition reverse complementation step is performed using the <em>fastx toolkit</em> (specified using -c). Trimmed reads are trimmed a second time using <em>cutadapt</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Martin 2011</a>) to remove and measure the poly(A)-tail returning reads that are longer than a user-defined length (default of 40nts) (parameters: -b A{15} -n 2 -O 10 –m 40). Reads containing poly(A) tracts shorter than 10 A’s are discarded. Next, reads output from this step are compared to the raw data to determine how many A’s (if any) were removed from the 3′ end of the read. This number is appended to the name of each read for future quality filtering. The preprocessing steps of DPAC are invoked by default or by using the ‘–p P’ command-line argument.</p><p id="p-10">After data preparation, reads are mapped using default settings to the reference genome using <em>HISAT2</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Kim <em>et al.</em> 2015</a>). The mapping step of DPAC is invoked by default or by using the ‘–p M’ command-line argument. If required, DPAC will also output the individual bedgraphs annotating all poly(A)-sites and mapping coverage for each sample by using the command-line argument ‘–p B’. These files can be loaded into canonical genome browsers and may be useful when generating figures. However, they are not required for the downstream analysis. An example of these output data are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, illustrating the mapping of PAC-Seq reads and identified PASs for two samples.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read coverage and the detected poly(A) sites (PASs) over the CD9 gene for two samples of Poly(A)-ClickSeq analysis of mocked treated HeLa cells (blue) and CFIm25 siRNA treated HeLa cells (orange) are depicted. Poly(A)-Clusters (PACs) are illustrated as a track (red) in the UCSC genome browser. The most frequently detected poly(A)-site within the poly(A)-cluster is highlighted as the thicker portion of the whole poly(A)-cluster in the track." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-852203061" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Read coverage and the detected poly(A) sites (PASs) over the CD9 gene for two samples of Poly(A)-ClickSeq analysis of mocked treated HeLa cells (blue) and CFIm25 siRNA treated HeLa cells (orange) are depicted. Poly(A)-Clusters (PACs) are illustrated as a track (red) in the UCSC genome browser. The most frequently detected poly(A)-site within the poly(A)-cluster is highlighted as the thicker portion of the whole poly(A)-cluster in the track.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1825/F2.medium.gif" width="440" height="196"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/6/1825/F2.medium.gif" width="440" height="196"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493260" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">Read coverage and the detected poly(A) sites (PASs) over the CD9 gene for two samples of Poly(A)-ClickSeq analysis of mocked treated HeLa cells (blue) and CFIm25 siRNA treated HeLa cells (orange) are depicted. Poly(A)-Clusters (PACs) are illustrated as a track (red) in the UCSC genome browser. The most frequently detected poly(A)-site within the poly(A)-cluster is highlighted as the thicker portion of the whole poly(A)-cluster in the track.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Generating Poly(A)-Clusters (PACs)</h3><p id="p-12">The poly(A)-clustering stage of DPAC requires specific BED files of annotated genes, exons and introns which can be obtained from the UCSC genome browser table browser. This is not invoked by default, but by using the command-line entry ‘-p C’. To maximize the power of PAC annotation using PAC-Seq, the data from all samples provided by the user in the metadata file are first combined and the 3′end of all reads are used to identify the location of PASs across the reference genome. The number of A’s found in each poly(A)-tract (as determined in the first pre-processing stage of DPAC) is utilized to score the confidence of each PAS. By default, PASs are output to a raw bedgraph file if a PAS is identified by at least 5 reads each with a poly(A)-tract of at least 25 A’s. These parameters can be adjusted in the command-line and must be chosen carefully depending upon the strategy used for poly(A)-seq library synthesis. In Poly(A)-ClickSeq libraries (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>), the oligo-dT primer used is 21nts but it is not anchored and therefore can prime anywhere within the poly(A) tail. Therefore, by selecting a number greater than 21 (<em>e.g.</em>, the default is 25), this ensures that PASs are only reported if a greater number of A’s were in the poly(A)-tail of the sequencing read than can be derived solely from the oligo-dT primer. In the case of PAC-seq, this provides a valuable tool to filter out reads resulting from internal or mis-priming events at the RT stage. For other techniques such as PAS-seq (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Shepard <em>et al.</em> 2011</a>) and 3′READS+ (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Zheng <em>et al.</em> 2016</a>), the number of A’s found in the poly(A)-tract at the ends of the read is usually only 10-15nts. In these cases, the number of required A’s in each sequencing read must be reduced accordingly to allow PAS annotation. Finally, PASs are filtered for internal priming by counting the number of A’s in the reference genome immediately downstream of the identified PASs. If 12 or more A’s are found within 20 nts downstream, these events are ‘masked’ and not further utilized.</p><p id="p-13">PASs are predominantly found at a ‘GA’, ‘UA’ or ‘CA’ dinucleotides, although the exact site is variable (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Derti <em>et al.</em> 2012</a>). By default, single PASs occurring within 25 nts of one another are merged into poly(A)-clusters (PACs), which are subsequently treated as singular features in downstream analyses. To annotate PACs, exon and intron annotations are first obtained from the UCSC database. The overlap of each PAC to annotated exons and introns is then determined using <em>bedtools</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Quinlan and Hall 2010</a>). PACs are annotated according to: the gene name; whether the PAC is exonic, intronic or found just downstream of a terminal exon; the genomic coordinate; and finally assigned a number depending upon the number of other PACs found within the same exon or intron (for example see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). PACs found in intergenic or otherwise unannotated sequences are numbered sequentially depending upon the total number of unannotated PACs found. This naming scheme is used in the final stage of DPAC to differentiate between alternative polyadenylation events and alternative terminal exon usage.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493288/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493288/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Example of count table used or DESeq2 for CD9, CD9 exon, and CD9 poly(A)-clusters</span><div class="sb-div caption-clear"></div></div></div><p id="p-14">Rather than performing <em>de novo</em> PAC annotation, pre-existing databases of poly(A)-clusters generated by DPAC, such as the ones used in this report, can be found online at <a href="https://sourceforge.net/projects/dpac-seq/files/Poly(A)_Clusters_BED">https://sourceforge.net/projects/dpac-seq/files/Poly(A)_Clusters_BED</a><span class="underline">/</span>. Alternatively, other established poly(A) databases such as from the PolyA_DB (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Lee <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Zhang <em>et al.</em> 2005</a>) can be provided to DPAC. As the naming conventions for PACs described above is essential for the downstream stages of DPAC, the ‘-p A’ argument must be selected in these instances to evoke a short script that will rename and sort the PACs.</p><p id="p-15">Examples of two specific poly(A)-clusters within the CD9 gene are illustrated in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> in the hg19 Poly(A)-Cluster track in red. Information regarding the most frequent PAS nucleotide or each PAC is retained as extra columns in the output BED file, illustrated as the thicker BED line Poly(A)-Cluster track.</p></div><div id="sec-4" class="subsection"><h3>Determination of differential Poly(A)-Cluster usage using DESeq2</h3><p id="p-16">In the final stage of DPAC, the mapped reads from each individual samples are used to determine the frequency of PACs in each dataset by determining the overlaps of the 3′ ends of the mapped reads with the provided poly(A) cluster database using <em>bedtools</em> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Quinlan and Hall 2010</a>). A PAC is counted if the 3′ end of a mapped read overlaps within a user-defined distance (10nts by default) of an annotated poly(A)-cluster and count tables of PACs are returned. Next, if multiple PACs are found within an exon or intron, then these are collapsed into a single entry, generating a new count table for exons and introns. Similarly, if multiple PACs are found within a single gene, these are also collapsed to create a count table just for whole genes.</p><p id="p-17">By default, only PACs found in exonic regions are collapsed into gene counts as introns can often contain repetitive and/or transposable elements whose inclusion can artificially inflate count numbers. However, intronic PACs can play important roles in the regulation of gene expression, particularly for long transcripts (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Wang <em>et al.</em> 2019</a>). Therefore, this parameter can be overturned to force inclusion of intronic PASs by selecting ‘<strong>-i</strong>’ in the command-line. In this case, to help prevent gene-count inflation from reads mapping to repetitive elements that may potentially be mis-mapped, an additional option (‘-m’) is provided to filter out PACs that overlap with annotated repetitive and mobile elements (provided by the user with an additional BED or GTF file), such as from the RepeatMasker database (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Smit <em>et al.</em> 2013–2015</a>).</p><p id="p-18">Three sets of count tables are thus generated (see example counts for CD9 and its exons/PACs in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) and passed individually into DESeq2 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Love <em>et al.</em> 2014</a>). Data normalization and statistical tests are applied using the canonical DESeq2 pipeline using <em>local</em> dispersion estimation and Independent Hypothesis Weighting (IHW) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ignatiadis <em>et al.</em> 2016</a>) to estimate false discovery rates and for power maximization. Thus, differential usage of PACs, exons/introns and whole-genes are calculated and the results are output as csv files. As illustrated in the flow-chart in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, these files are returned for inspection, figure generation and other downstream analyses.</p></div><div id="sec-5" class="subsection"><h3>Output</h3><p id="p-19">After DESeq2 analysis, a final compiled table is generated containing information about PAC usage for each gene (including only exons unless the <strong>–i</strong> option is selected). If a gene only has one PAC and thus one terminal exon, only the gene information is returned. Genes with differential expression (fold-change &gt; 1.5; <em>padj</em> &lt; 0.1) are annotated as ‘DOWN’ or ‘UP’. If there is no significant change, genes are labels as ‘NC’ (No Change). Alternative polyadenylation (APA) or differential terminal exon usage (TE) is reported when a gene has two or more PACs (minimum occupancy of 5% per PAC), with at least one PAC undergoing differential usage with an IHW <em>padj</em> &lt;0.1 and resulting in a fractional change of the PAC usage by at least 10%. If two PACs are found in different exons, this is annotated as a differential terminal exon usage event (denoted as ‘TE’). If the two PACs are found in the same exon, then this annotated as an APA event. The relative locations of the PACs is then used to determine whether the APA results in 3′UTR shortening or lengthening. If three or more PACs (again with minimum occupancy of 5% per PAC) are found within a single exon and one of the middle PACs changes in abundance, then this can simultaneously result in changes of abundance of both upstream and downstream PACs, resulting in both a shortening and a lengthening phenotype. These are annotated as ‘APA_both’.</p></div><div id="sec-6" class="subsection"><h3>Data Availability Statement</h3><p id="p-20">All data used in the manuscript is available at the NCBI SRA database as deposited by their respective authors: HeLa cell PAC-Seq data (PRJNA374982); HeLa cell 3′READs+ data (PRJNA328218); and MEF cell PAS-seq data (PRJNA436720). Annotated PAC datasets generated in this manuscript are available in supplementary material and at <a href="https://sourceforge.net/projects/dpac-seq/files/Poly%28A%29_Clusters_BED/">https://sourceforge.net/projects/dpac-seq/files/Poly%28A%29_Clusters_BED/</a> DPAC is freely available (MIT license), is maintained and available at <a href="https://sourceforge.net/projects/DPAC-Seq">https://sourceforge.net/projects/DPAC-Seq</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7635971">https://doi.org/10.25387/g3.7635971</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Re-analysis of Poly(A)-ClickSeq data of CFIm25 knock-down in HeLa cells</h3><p id="p-21">To evaluate this pipeline, we re-analyzed the PAC-seq data deposited at NCBI SRA (<em>PRJNA374982</em>) from the original PAC-Seq publication (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>). The six datasets were derived from total cellular RNA extracted from three technical replicates each of mock-treated and CFIm25 KD HeLa cells. We applied our pipeline to locate and annotate <em>de novo</em> poly(A)-clusters (PACs) and then to determine the differential usage of PACs between each condition. Annotation data and example command-line entries are provided in the DPAC manual to repeat these analyses.</p><p id="p-22">During <em>de novo</em> PAS clustering, a total of 44,422 poly(A) clusters (PACs) with &gt;25 reads were identified in the datasets. Of these, 27,958 were exonic, 7,441 were intronic, 928 were found within 250nts downstream of annotated 3′ terminal exons; and 8,094 were intergenic or otherwise unannotated (<strong>Supplementary Data 1</strong>).</p><p id="p-23">To detect differential poly(A)-cluster usage, we performed the final stage of DPAC using the three following conditions: 1) only considering exonic PACs, 2) considering all PACs (exonic or otherwise) but filtering out PACs overlapping with the hg19 RepeatMasker database; and 3) considering all PACs (exonic or otherwise) but using the poly(A) database (PolyA_DB) instead of <em>de novo</em> PAC annotation. Summaries of the output are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Reports of both differential gene expression and differential PACs usage (IHW-padj &lt; 0.1, fold-change &gt;1.5) are provided in <strong>Supplemental Datasets 2, 3</strong> and <strong>4</strong>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493308/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493308/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summaries of findings of DPAC analysis of CFIm25 KD HeLa cells using three different sets of parameters</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">When considering all PACs (including intronic) our pipeline found PACs mapped over a total of 12,886 genes, of which 5,880 (47%) exhibited multiple PACs (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), similar to rates previously reported (<a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Lee <em>et al.</em> 2007</a>). By virtue of measuring differential usage of each individual PAC independently, DPAC revealed differential usage of PACs outside of annotated regions. Indeed, of the total 1,233 identified differentially expressed PACs, 94 (7.6%) were intronic and 99 (8.0%) were found in unannotated regions. Volcano plots illustrating changes in gene expression and PAC abundance are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Due to changes in PAC abundance, DPAC reported that 647 genes exhibited APA with the shortening of 457 3′UTRs and lengthening of 78 3′UTRs. 109 exons exhibited both lengthening and shortening, due to the presence of multiple PACs within 3′UTRs. The predominant shortening of 3′UTRs upon knock-down of CFIm25 is the expected phenotype and is consistent with our and others' previous analyses (<a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">Routh <em>et al.</em> 2017</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Zhu <em>et al.</em> 2018</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chu <em>et al.</em> 2019</a>). Differential PAC usage resulted in alternative terminal exon usage in 236 genes and only 25 genes exhibited APA and TE simultaneously.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Volcano plots of the differential expression of Genes (left) and Poly(A)-Clusters (right) in HeLa cells upon siRNA KD of CFIm25 using default settings of DPAC (data from Table 2, column 1). Red dots indicate genes or PACs with a fold changes greater than 1.5 and a p-adjusted value less than 0.1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-852203061" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Volcano plots of the differential expression of Genes (left) and Poly(A)-Clusters (right) in HeLa cells upon siRNA KD of CFIm25 using default settings of DPAC (data from Table 2, column 1). Red dots indicate genes or PACs with a fold changes greater than 1.5 and a p-adjusted value less than 0.1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1825/F3.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/6/1825/F3.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1825/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493410" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Volcano plots of the differential expression of Genes (left) and Poly(A)-Clusters (right) in HeLa cells upon siRNA KD of CFIm25 using default settings of DPAC (data from <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, column 1). Red dots indicate genes or PACs with a fold changes greater than 1.5 and a p-adjusted value less than 0.1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">DPAC outputs a report detailing changes in gene expression, exon usage and PAC usage (<strong>Supplemental Datasets 2, 3</strong> and <strong>4).</strong> Specific examples of the final output are shown in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>. SLC35E21 has two PACs found in two different terminal exons, but there is no significant change in their usage and therefore no alternative terminal exon usage. MEGF11 has only one PAC, and this gene is significantly up-regulated upon CFIm25 KD. CD9 is also up-regulated upon CFIm25 KD due to the up-regulation of one of two PACs found within the same terminal exon with a net effect of 3′UTR shortening. The mapping of the raw data over CD9 and the detected PASs and PACs are shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>.</p></div><div id="sec-9" class="subsection"><h3>Re-analysis of 3′READs+ and PAS-Seq data</h3><p id="p-27">The DPAC pipeline was conceived during the development of analysis of Poly(A)-ClickSeq datatypes. Nonetheless, the DPAC pipeline is applicable to any data type provided that there are poly(A) tracts (or poly(T) tracts in the negative sense) retained within the read data that are of at least 10nts in length and the read length after poly(A) trimming is greater than 25nts. There are many current poly(A)-tail focused methods for RNAseq (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Zhang <em>et al.</em> 2018</a>), that yield similar read data focused on the 3′UTR and poly(A)-tail junction. So to demonstrate this functionality, we ran the DPAC pipeline using previously deposited and published datasets to generate <em>de novo</em> PAC datasets: 1) 3′READs+ data derived from HeLa cells (human) (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Zheng <em>et al.</em> 2016</a>) and 2) PAS-Seq datasets derived from MEF cells (murine) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chang <em>et al.</em> 2018</a>). Summaries of the output are shown in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and the final PAC datasets (BED format) are available in <strong>Supplementary Datafiles 5</strong> and <strong>6</strong> respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493412/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493412/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summaries of Poly(A)-Clusters annotated using 3′READs+ and PAS-Seq datasets</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><p id="p-28">In summary, DPAC performs each of the necessary steps required for preprocessing, poly(A)-site identification, poly(A)-clustering and differential PAC usage required for poly(A)-targeted RNAseq experiments. Our pipeline is suitable for analysis of multiple different strategies for poly(A)-tail sequencing, provided that stretches of the poly(A)-tail greater than 10nts are retained within the sequencing data. We further recommend that read-lengths are sufficiently long, once accounting for removal of the poly(A)-tract, so that at least 40nts of 3′UTR sequence remain. While shorter reads can be tolerated (<em>e.g.</em>, as demonstrated in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), this ensures that reads are mapped unambiguously, which may be particularly important for large genomes.</p><p id="p-29">In principle, DPAC may also be used to analyze canonical random-primed RNAseq data or RNAseq data enriched coarsely for 3′UTRs, such as in QuantSeq (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Moll <em>et al.</em> 2014</a>), as many of these reads will map over the junction of the 3′UTR and poly(A) by chance, although the frequency of these reads may be low. However, there exist other sophisticated tools for poly(A)-site annotation and measurement of alternative polyadenylation such as DaPars (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Xia <em>et al.</em> 2014</a>) and TAPAS (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arefeen <em>et al.</em> 2018</a>) that are designed specifically for these data types.</p><p id="p-30">DPAC reports the expected findings upon reanalysis of Poly(A)-ClickSeq datasets comparing mock and CFIm25-knockdown HeLa cells. By virtue of assessing changes in all PACs regardless of whether they are found in annotated genomic regions, DPAC may also allow discovery of novel mRNA transcripts and/or changes in the expression of ncRNAs and/or non-coding transposable elements. DPAC therefore provides a singular pipeline to simultaneously report differential gene expression, terminal exon usage and alternative polyadenylation.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">A.R. is supported by start-up funds from the University of Texas Medical Branch. I thank Dr. Eric Wagner and Dr. Mugé Martinez-Kuyumcu at UTMB for providing critical support and advice.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-32">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7635971">https://doi.org/10.25387/g3.7635971</a>.</p></li><li class="fn" id="fn-2"><p id="p-33"><em>Communicating editor: T. Hughes</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-26"><span class="received-label">Received </span>January 26, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-04-25"><span class="accepted-label">Accepted </span>April 25, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Routh</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.1" data-doi="10.1093/bioinformatics/bty110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arefeen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">TAPAS: tool for alternative polyadenylation site analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2521</span>–<span class="cit-lpage">2529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D2521%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.2" data-doi="10.1093/nar/gky340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">An integrative model for alternative polyadenylation, IntMAP, delineates mTOR-modulated endoplasmic reticulum stress response.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">5996</span>–<span class="cit-lpage">6008</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253D5996%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.3" data-doi="10.1093/bioinformatics/bty560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">fastp: an ultra-fast all-in-one FASTQ preprocessor.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">i884</span>–<span class="cit-lpage">i890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty560</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253Di884%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.4" data-doi="10.1038/s41388-019-0714-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elrod</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Nudt21 regulates the alternative polyadenylation of Pak1 and is predictive in the prognosis of glioblastoma patients.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41388-019-0714-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41388-019-0714-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41388-019-0714-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.5" data-doi="10.1101/gr.132563.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrett-Engele</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macisaac</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sriram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A quantitative atlas of polyadenylation in five mammals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.132563.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132563.111%26rft_id%253Dinfo%253Apmid%252F22454233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTE3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xODI1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.6" data-doi="10.1016/j.ymeth.2019.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elrod</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaworski</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Routh</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Development of Poly(A)-ClickSeq as a Tool Enabling Simultaneous Genome-wide Poly(A)-site identification and Differential Expression Analysis.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ymeth.2019.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D155%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2019.01.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2019.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.7" data-doi="10.1038/nmeth.3885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ignatiadis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klaus</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaugg</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Data-driven hypothesis weighting increases detection power in genome-scale multiple testing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3885</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D577%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3885&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.8" data-doi="10.1038/nmeth.3317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HISAT: a fast spliced aligner with low memory requirements.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3317</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3317%26rft_id%253Dinfo%253Apmid%252F25751142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751142&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.9" data-doi="10.1093/nar/gkl870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PolyA_DB 2: mRNA polyadenylation sites in vertebrate genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D165</span>–<span class="cit-lpage">D168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl870</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLee%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D35%26rft.issue%253Dsuppl_1%26rft.spage%253DD165%26rft.epage%253DD168%26rft.atitle%253DPolyA_DB%2B2%253A%2BmRNA%2Bpolyadenylation%2Bsites%2Bin%2Bvertebrate%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl870%26rft_id%253Dinfo%253Apmid%252F17202160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl870&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17202160&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243494600034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.10" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">550</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0550-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Cutadapt removes adapter sequences from high-throughput sequencing reads. <em>EMBnet.journal</em> 17 (1):10–12.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.12" data-doi="10.1038/nmeth.f.376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seitz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reda</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QuantSeq 3′ mRNA sequencing for RNA quantification.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">i</span>–<span class="cit-lpage">iii</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.f.376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253Di%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.f.376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.f.376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.13" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.14" data-doi="10.1093/nar/gkx286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Routh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaworski</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Poly(A)-ClickSeq: click-chemistry for next-generation 3-end sequencing without RNA enrichment or fragmentation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-elocation-id">e112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx286</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253De112%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.15" data-doi="10.1261/rna.2581711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shepard</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flanagan</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hertel</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Complex and dynamic landscape of RNA polyadenylation revealed by PAS-Seq.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">772</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.2581711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2581711%26rft_id%253Dinfo%253Apmid%252F21343387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNzYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-web" id="cit-9.6.1825.16"><div class="cit-metadata unstructured">Smit, A., R. Hubley, and P. Green, 2013–2015 RepeatMasker Open-4.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> Regulation of Intronic Polyadenylation by PCF11 Impacts mRNA Expression of Long Genes. <em>Cell Rep</em> 26 (10):2766–2778 e2766.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2019.02.049</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.18" data-doi="10.1038/ncomms6274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donehower</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neilson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dynamic analyses of alternative polyadenylation from RNA-seq reveal a 3′-UTR landscape across seven tumour types.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5274%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6274%26rft_id%253Dinfo%253Apmid%252F25409906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409906&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.19" data-doi="10.1093/nar/gki055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Recce</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">PolyA_DB: a database for mammalian mRNA polyadenylation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">D116</span>–<span class="cit-lpage">D120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gki055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DH.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD116%26rft.epage%253DD120%26rft.atitle%253DPolyA_DB%253A%2Ba%2Bdatabase%2Bfor%2Bmammalian%2BmRNA%2Bpolyadenylation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki055%26rft_id%253Dinfo%253Apmid%252F15608159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15608159&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1825.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226524300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.20" data-doi="10.7150/ijbs.27168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrion</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinski</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Alternative polyadenylation analysis in animals and plants: newly developed strategies for profiling, processing and validation.</span> <abbr class="cit-jnl-abbrev">Int. J. Biol. Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1714</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7150/ijbs.27168</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BBiol.%2BSci.%26rft.volume%253D14%26rft.spage%253D1709%26rft_id%253Dinfo%253Adoi%252F10.7150%252Fijbs.27168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7150/ijbs.27168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1825.21" data-doi="10.1261/rna.057075.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">3′READS+, a sensitive and accurate method for 3′ end sequencing of polyadenylated RNA.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.057075.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.057075.116%26rft_id%253Dinfo%253Apmid%252F27512124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8xNjMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4MjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-web" id="cit-9.6.1825.22" data-doi="10.1016/j.molcel.2017.11.031"><div class="cit-metadata"><cite>Zhu, Y., X. Wang, E. Forouzmand, J. Jeong, F. Qiao <em>et al.</em>, 2018 Molecular Mechanisms for CFIm-Mediated Regulation of mRNA Alternative Polyadenylation. <em>Mol Cell</em> 69 (1):62–74 e64.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2017.11.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D69%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2017.11.031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2017.11.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/6/1825.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1815" title="Negative Regulation of the Mis17-Mis6 Centromere Complex by mRNA Decay Pathway and EKC/KEOPS Complex in Schizosaccharomyces pombe" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1831" title="Full-Length Transcriptome Sequencing and the Discovery of New Transcripts in the Unfertilized Eggs of Zebrafish (Danio rerio)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/6"><span>Volume 9 Issue 6, June 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/6.cover-source.jpg?itok=Ub13TpI0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1825" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Wi7oye2BiWk_oJyOJu57A2ULLmBKmhC1x-u16v1blGo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/6/1825" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/6/1825" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/6/1825" class="active" data-icon-position="" data-hide-link-title="0">DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493254" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JJx2mTj5d4N57_wDn-OE5cbFPGWW41zBv24CpG0RGFg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1825.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/6/1825" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3jOrIs1D62AzNZ3Djw27_5bZx-boun6VeUdw1JMYgXM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/6/1825" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1825.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493254" id="node493254--2" data-pisa="ggg;9/6/1825" data-pisa-master="ggg;g3.119.400273" data-apath="/ggg/9/6/1825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2874-5990" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew Routh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1825-1830;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400273 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493254--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew Routh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Sealy Centre for Structural Biology and Molecular Biophysics, University of Texas Medical Branch, Galveston, Texas, 77555</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BRouth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Routh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BRouth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2874-5990" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew Routh</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alrouth@utmb.edu" class="" data-icon-position="" data-hide-link-title="0">alrouth@utmb.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/6/1825" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/6/1825&amp;title=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/6/1825&amp;title=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/6/1825&amp;title=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/6/1825&amp;text=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/6/1825&amp;title=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/6/1825&amp;t=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/6/1825&amp;title=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/6/1825&amp;title=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493254/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493254" id="node493254--3" data-pisa="ggg;9/6/1825" data-pisa-master="ggg;g3.119.400273" data-apath="/ggg/9/6/1825.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">DPAC: A Tool for Differential Poly(A)–Cluster Usage from Poly(A)–Targeted RNAseq Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2874-5990" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrew Routh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1825-1830;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400273 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493254--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrew Routh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Sealy Centre for Structural Biology and Molecular Biophysics, University of Texas Medical Branch, Galveston, Texas, 77555</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BRouth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Routh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BRouth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2874-5990" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew Routh</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alrouth@utmb.edu" class="" data-icon-position="" data-hide-link-title="0">alrouth@utmb.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1825&amp;rft.epage=1830&amp;rft.atitle=DPAC%3A%20A%20Tool%20for%20Differential%20Poly%28A%29%E2%80%93Cluster%20Usage%20from%20Poly%28A%29%E2%80%93Targeted%20RNAseq%20Data&amp;rft.volume=9&amp;rft.issue=6&amp;rft.date=2019-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Routh%2C+Andrew"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493254/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493254/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493254/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493254/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493254/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493254/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493254/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493254/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493254/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493254/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493254/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493254/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/6/1825" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/6/1825.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/6/1825.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/6/1825.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
