<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/6/1851" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/6/1851.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/6/1851.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/6/1851.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/6/1851/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/6/1851.atom" />
<meta name="HW.pisa" content="ggg;9/6/1851" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Effect of Neutral Recombination Variation on Genome Scans for Selection" />
<meta name="DC.Identifier" content="10.1534/g3.119.400088" />
<meta name="DC.Date" content="2019-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Lotterhos. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.119.400779" />
<meta name="DC.Description" content="Recently, there has been an increasing interest in identifying the role that regions of low recombination or inversion play in adaptation of species to local environments. Many examples of groups of adapted genes located within inversions are arising in the literature, in part inspired by theory that predicts the evolution of these so-called “supergenes.” We still, however, have a poor understanding of how genomic heterogeneity, such as varying rates of recombination, may confound signals of selection. Here, I evaluate the effect of neutral inversions and recombination variation on genome scans for selection, including tests for selective sweeps, differentiation outlier tests, and association tests. There is considerable variation among methods in their performance, with some methods being unaffected and some showing elevated false positive signals within a neutral inversion or region of low recombination. In some cases the false positive signal can be dampened or removed, if it is possible to use a quasi-independent set of SNPs to parameterize the model before performing the test. These results will be helpful to those seeking to understand the importance of regions of low recombination in adaptation." />
<meta name="DC.Contributor" content="Katie E. Lotterhos" />
<meta name="article:published_time" content="2019-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Effect of Neutral Recombination Variation on Genome Scans for Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recently, there has been an increasing interest in identifying the role that regions of low recombination or inversion play in adaptation of species to local environments. Many examples of groups of adapted genes located within inversions are arising in the literature, in part inspired by theory that predicts the evolution of these so-called “supergenes.” We still, however, have a poor understanding of how genomic heterogeneity, such as varying rates of recombination, may confound signals of selection. Here, I evaluate the effect of neutral inversions and recombination variation on genome scans for selection, including tests for selective sweeps, differentiation outlier tests, and association tests. There is considerable variation among methods in their performance, with some methods being unaffected and some showing elevated false positive signals within a neutral inversion or region of low recombination. In some cases the false positive signal can be dampened or removed, if it is possible to use a quasi-independent set of SNPs to parameterize the model before performing the test. These results will be helpful to those seeking to understand the importance of regions of low recombination in adaptation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/06/01" />
<meta name="citation_mjid" content="ggg;9/6/1851" />
<meta name="citation_id" content="9/6/1851" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/6/1851" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/6/1851.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/6/1851.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/6/1851.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400088" />
<meta name="citation_pmid" content="30971391" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1851" />
<meta name="citation_lastpage" content="1867" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Katie E. Lotterhos" />
<meta name="citation_author_institution" content="Department of Marine and Environmental Sciences, Northeastern University, 430 Nahant Road, Nahant, MA 01908" />
<meta name="citation_author_email" content="k.lotterhos@northeastern.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7529-2771" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=A. Abdellaoui;citation_author=J.-J. Hottenga;citation_author=P. de Knijff;citation_author=M. G. Nivard;citation_author=X. Xiao;citation_title=Population structure, migration, and diversifying selection in the Netherlands.;citation_pages=1277-1285;citation_volume=21;citation_year=2013;citation_pmid=23531865;citation_doi=10.1038/ejhg.2013.48" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=G. Abraham;citation_author=M. Inouye;citation_title=Fast principal component analysis of large-scale genome-wide data.;citation_pages=e93766;citation_volume=9;citation_year=2014;citation_pmid=24718290;citation_doi=10.1371/journal.pone.0093766" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=G. Abraham;citation_author=Y. Qiu;citation_author=M. Inouye;citation_title=FlashPCA2: principal component analysis of biobank-scale genotype datasets.;citation_pages=2014-2017;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=K. J. Galinsky;citation_author=G. Bhatia;citation_author=P. R. Loh;citation_author=S. Georgiev;citation_author=S. Mukherjee;citation_title=Fast principal-component analysis reveals convergent evolution of ADH1B in Europe and East Asia.;citation_pages=456-472;citation_volume=98;citation_year=2016;citation_doi=10.1016/j.ajhg.2015.12.022" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. T. Barney;citation_author=C. Munkholm;citation_author=D. R. Walt;citation_author=S. R. Palumbi;citation_title=Highly localized divergence within supergenes in Atlantic cod (Gadus morhua) within the Gulf of Maine.;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3660-3" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=M. A. Beaumont;citation_author=R. A. Nichols;citation_title=Evaluating loci for use in the genetic analysis of population structure.;citation_volume=263;citation_year=1996;citation_doi=10.1098/rspb.1996.0237" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=P. R. Berg;citation_author=B. Star;citation_author=C. Pampoulie;citation_author=I. R. Bradbury;citation_author=P. Bentzen;citation_title=Trans-oceanic genomic divergence of Atlantic cod ecotypes is associated with large inversions.;citation_volume=119;citation_year=2017;citation_doi=10.1038/hdy.2017.54" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. Berner;citation_author=M. Roesti;citation_title=Genomics of adaptive divergence with chromosome-scale heterogeneity in crossover rate.;citation_pages=6351-6369;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14373" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Bonhomme;citation_author=C. Chevalet;citation_author=B. Servin;citation_author=S. Boitard;citation_author=J. Abdallah;citation_title=Detecting selection in population trees: the Lewontin and Krakauer test extended.;citation_volume=186;citation_year=2010;citation_pmid=20855576;citation_doi=10.1534/genetics.110.117275" />
<meta name="citation_reference" content="citation_title=Machine Learning and Knowledge Discovery in Databases. ECML PKDD 2013;citation_title=Area under the Precision-Recall Curve: Point Estimates and Confidence Intervals;citation_author=K. Boyd;citation_author=K. H. Eng;citation_author=C. D. Page;citation_volume=Vol. 8190;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Bradburd;citation_author=G. Coop;citation_author=P. Ralph;citation_title=Inferring continuous and discrete population genetic structure across space.;citation_volume=210;citation_year=2018;citation_pmid=30026187;citation_doi=10.1534/genetics.118.301333" />
<meta name="citation_reference" content="citation_title=The Mathematical Theory of Selection, Recombination, and Mutation;citation_author=R. Bürger;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=W. S. Bush;citation_author=J. H. Moore;citation_title=Chapter 11: Genome-wide association studies.;citation_pages=e1002822;citation_volume=8;citation_year=2012;citation_pmid=23300413;citation_doi=10.1371/journal.pcbi.1002822" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Caye;citation_author=B. Jumentier;citation_author=J. Lepeule;citation_author=O. François;citation_title=LFMM 2: fast and accurate inference of gene-environment associations in genome-wide studies;citation_pages=852-860;citation_volume=36;citation_year=2019;citation_doi=10.1093/molbev/msz008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Charlesworth;citation_author=M. T. Morgan;citation_author=D. Charlesworth;citation_title=The effect of deleterious mutations on neutral molecular variation.;citation_pages=1289-1303;citation_volume=134;citation_year=1993;citation_issue=4;citation_pmid=8375663" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. Charlesworth;citation_author=M. Nordborg;citation_author=D. Charlesworth;citation_title=The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.;citation_pages=155-174;citation_volume=70;citation_year=1997;citation_issue=2;citation_pmid=9449192;citation_doi=10.1017/S0016672397002954" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Charlesworth;citation_title=The effects of deleterious mutations on evolution at linked sites.;citation_volume=190;citation_year=2012;citation_pmid=22219506;citation_doi=10.1534/genetics.111.134288" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Cheng;citation_author=B. J. White;citation_author=C. Kamdem;citation_author=K. Mockaitis;citation_author=C. Costantini;citation_title=Ecological genomics of Anopheles gambiae along a latitudinal cline: a population-resequencing approach.;citation_volume=190;citation_year=2011;citation_pmid=22209907;citation_doi=10.1534/genetics.111.137794" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. B. Corbett-Detig;citation_author=D. L. Hartl;citation_title=Population genomics of inversion polymorphisms in Drosophila melanogaster.;citation_pages=e1003056;citation_volume=8;citation_year=2012;citation_pmid=23284285;citation_doi=10.1371/journal.pgen.1003056" />
<meta name="citation_reference" content="citation_title=An Introduction to Population Genetics Theory;citation_author=J. F. Crow;citation_author=M. Kimura;citation_year=1970" />
<meta name="citation_reference" content="citation_author=J. Davis;citation_author=M. Goadrich;citation_year=2006;citation_doi=10.1145/1143844.1143874" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. De Mita;citation_author=A.-C. Thuillet;citation_author=L. Gay;citation_author=N. Ahmadi;citation_author=S. Manel;citation_title=Detecting selection along environmental gradients: analysis of eight methods and their effectiveness for outbreeding and selfing populations.;citation_pages=1383-1399;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12182" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Duforet-Frebourg;citation_author=E. Bazin;citation_author=M. G. B. Blum;citation_title=Genome scans for detecting footprints of local adaptation using a Bayesian factor model.;citation_volume=31;citation_year=2014;citation_pmid=24899666;citation_doi=10.1093/molbev/msu182" />
<meta name="citation_reference" content="citation_author=T. Fawcett;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. L. Feder;citation_author=S. P. Egan;citation_author=P. Nosil;citation_title=The genomics of speciation-with-gene-flow.;citation_pages=342-350;citation_volume=28;citation_year=2012;citation_issue=7;citation_pmid=22520730;citation_doi=10.1016/j.tig.2012.03.009" />
<meta name="citation_reference" content="citation_title=The Genetical Theory of Natural Selection;citation_author=R. A. Fisher;citation_year=1930;citation_doi=10.5962/bhl.title.27468" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. R. Forester;citation_author=M. R. Jones;citation_author=S. Joost;citation_author=E. L. Landguth;citation_author=J. R. Lasky;citation_title=Detecting spatial genetic signatures of local adaptation in heterogeneous landscapes.;citation_pages=104-120;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13476" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. R. Forester;citation_author=J. R. Lasky;citation_author=H. H. Wagner;citation_author=D. L. Urban;citation_title=Comparing methods for detecting multilocus adaptation with multivariate genotype-environment associations.;citation_pages=2215-2233;citation_volume=27;citation_year=2018;citation_doi=10.1111 /mec. 14584" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=O. François;citation_author=H. Martins;citation_author=K. Caye;citation_author=S. D. Schoville;citation_title=Controlling false discoveries in genome scans for selection.;citation_pages=454-469;citation_volume=25;citation_year=2016;citation_pmid=26671840;citation_doi=10.1111/mec.13513" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=O. François;citation_author=K. Caye;citation_title=Naturalgwas: An R package for evaluating genomewide association methods with empirical data.;citation_pages=789-797;citation_volume=18;citation_year=2018;citation_doi=10.1111/1755-0998.12892" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. Frichot;citation_author=F. Mathieu;citation_author=T. Trouillon;citation_author=G. Bouchard;citation_author=O. François;citation_title=Fast and efficient estimation of individual ancestry coefficients.;citation_volume=196;citation_year=2014;citation_pmid=24496008;citation_doi=10.1534/genetics.113.160572" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=E. Frichot;citation_author=O. François;citation_title=LEA: An R package for landscape and ecological association studies.;citation_pages=925-929;citation_volume=6;citation_year=2015;citation_doi=10.1111/2041-210X.12382" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=Z. L. Fuller;citation_author=S. A. Koury;citation_author=N. Phadnis;citation_author=S. W. Schaeffer;citation_title=How chromosomal rearrangements shape adaptation and speciation: case studies in Drosophila pseudoobscura and its sibling species Drosophila persimilis.;citation_year=2018;citation_doi=10.1111/mec.14923" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. R. Garud;citation_author=P. W. Messer;citation_author=E. O. Buzbas;citation_author=D. A. Petrov;citation_title=Recent selective sweeps in North American Drosophila melanogaster show signatures of soft sweeps.;citation_pages=e1005004;citation_volume=11;citation_year=2015;citation_pmid=25706129;citation_doi=10.1371/journal.pgen.1005004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Gautier;citation_title=Genome-wide scan for adaptive divergence and association with population-specific covariates.;citation_volume=201;citation_year=2015;citation_pmid=26482796;citation_doi=10.1534/genetics.115.181453" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Gautier;citation_author=K. Gharbi;citation_author=T. Cezard;citation_author=J. Foucaud;citation_author=C. Kerdelhué;citation_title=The effect of RAD allele dropout on the estimation of genetic variation within and between populations.;citation_pages=3165-3178;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12089" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Grau;citation_author=I. Grosse;citation_author=J. Keilwagen;citation_title=PRROC: computing and visualizing precision-recall and receiver operating characteristic curves in R.;citation_volume=31;citation_year=2015;citation_pmid=25810428;citation_doi=10.1093/bioinformatics/btv153" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=R. F. Guerrero;citation_author=F. Rousset;citation_author=M. Kirkpatrick;citation_title=Coalescent patterns for chromosomal inversions in divergent populations;citation_pages=430-438;citation_volume=367;citation_year=2012;citation_issue=1587;citation_pmid=22201172;citation_doi=10.1098/rstb.2011.0246" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Günther;citation_author=G. Coop;citation_title=Robust identification of local adaptation from allele frequencies.;citation_volume=195;citation_year=2013;citation_pmid=23821598;citation_doi=10.1534/genetics.113.152462" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 2: Flexible, interactive forward genetic simulations.;citation_volume=34;citation_year=2017;citation_pmid=27702775;citation_doi=10.1093/molbev/msw211" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=B. C. Haller;citation_author=J. Galloway;citation_author=J. Kelleher;citation_author=P. W. Messer;citation_author=P. L. Ralph;citation_title=Tree-sequence recording in SLiM opens new horizons for forward-time simulation of whole genomes.;citation_year=2018;citation_doi=10.1111/1755-0998.12968" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=S. Hoban;citation_author=J. L. Kelley;citation_author=K. E. Lotterhos;citation_author=M. F. Antolin;citation_author=G. Bradburd;citation_title=Finding the genomic basis of local adaptation: pitfalls, practical solutions, and future directions.;citation_pages=379-397;citation_volume=188;citation_year=2016;citation_pmid=27622873;citation_doi=10.1086/688018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=K. Bailey;citation_author=D. Skarecky;citation_author=J. Kwiatowski;citation_author=F. J. Ayala;citation_title=Evidence for positive selection in the superoxide dismutase (Sod) region of Drosophila melanogaster.;citation_pages=1329-1340;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013910" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Kapun;citation_author=D. K. Fabian;citation_author=J. Goudet;citation_author=T. Flatt;citation_title=Genomic evidence for adaptive inversion clines in Drosophila melanogaster.;citation_volume=33;citation_year=2016;citation_pmid=26796550;citation_doi=10.1093/molbev/msw016" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=R. E. Kass;citation_author=A. E. Raftery;citation_title=Bayes Factors.;citation_pages=773-795;citation_volume=90;citation_year=1995;citation_doi=10.1080/01621459.1995.10476572" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Keilwagen;citation_author=I. Grosse;citation_author=J. Grau;citation_title=Area under precision-recall curves for weighted and unweighted data.;citation_pages=e92209;citation_volume=9;citation_year=2014;citation_pmid=24651729;citation_doi=10.1371/journal.pone.0092209" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=J. Kelleher;citation_author=K. R. Thornton;citation_author=J. Ashander;citation_author=P. L. Ralph;citation_title=Efficient pedigree recording for fast population genetics simulation.;citation_pages=e1006581;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pcbi.1006581" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Kirkpatrick;citation_author=N. Barton;citation_title=Chromosome inversions, local adaptation and speciation.;citation_volume=173;citation_year=2006;citation_pmid=16204214;citation_doi=10.1534/genetics.105.047985" />
<meta name="citation_reference" content="citation_author=B. J. Knaus;citation_author=N. J. Grunwald;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=A. Korte;citation_author=A. Farlow;citation_title=The advantages and limitations of trait analysis with GWAS: a review.;citation_pages=29;citation_volume=9;citation_year=2013;citation_pmid=23876160;citation_doi=10.1186/1746-4811-9-29" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=J. Lachance;citation_author=S. A. Tishkoff;citation_title=SNP ascertainment bias in population genetic analyses: why it is important, and how to correct it.;citation_pages=780-786;citation_volume=35;citation_year=2013;citation_pmid=23836388;citation_doi=10.1002/bies.201300014" />
<meta name="citation_reference" content="citation_title=Numerical Ecology;citation_author=P. Legendre;citation_author=L. F. J. Legendre;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_author=J. Krakauer;citation_title=DISTRIBUTION OF GENE FREQUENCY AS A TEST OF THE THEORY OF THE SELECTIVE NEUTRALITY OF POLYMORPHISMS;citation_pages=175-195;citation_volume=74;citation_year=1973;citation_issue=1;citation_pmid=4711903" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Li;citation_author=P. Ralph;citation_title=Local PCA shows how the effect of population structure differs along the genome.;citation_volume=211;citation_year=2019;citation_pmid=30459280;citation_doi=10.1534/genetics.118.301747" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. E. Lotterhos;citation_author=M. C. Whitlock;citation_title=Evaluation of demographic history and neutral parameterization on the performance of FST outlier tests.;citation_pages=2178-2192;citation_volume=23;citation_year=2014;citation_pmid=24655127;citation_doi=10.1111/mec.12725" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. E. Lotterhos;citation_author=M. C. Whitlock;citation_title=The relative power of genome scans to detect local adaptation depends on sampling design and statistical method.;citation_pages=1031-1046;citation_volume=24;citation_year=2015;citation_pmid=25648189;citation_doi=10.1111/mec.13100" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=K. E. Lotterhos;citation_author=J. H. Moore;citation_author=A. E. Stapleton;citation_title=Analysis validation has been neglected in the Age of Reproducibility.;citation_pages=e3000070;citation_volume=16;citation_year=2018;citation_doi=10.1371/journal.pbio.3000070" />
<meta name="citation_reference" content="citation_title=Data from: The effect of neutral recombination variation on genome scans for selection;citation_author=K. E. Lotterhos;citation_year=2019;citation_doi=10.5061/dryad.rj0kj10" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=K. Luu;citation_author=E. Bazin;citation_author=M. G. B. Blum;citation_title=pcadapt: an R package to perform genome scans for selection based on principal component analysis.;citation_pages=67-77;citation_volume=17;citation_year=2017;citation_doi=10.1111/1755-0998.12592" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lynch;citation_author=K. Ritland;citation_title=Estimation of Pairwise Relatedness With Molecular Markers;citation_pages=1753-1766;citation_volume=152;citation_year=1999;citation_issue=4;citation_pmid=10430599" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=H. Martins;citation_author=K. Caye;citation_author=K. Luu;citation_author=M. G. B. Blum;citation_author=O. François;citation_title=Identifying outlier loci in admixed and in continuous populations using ancestral population differentiation statistics.;citation_pages=5029-5042;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13822" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. M. Maynard-Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. McVean;citation_title=A genealogical interpretation of principal components analysis.;citation_pages=e1000686-e1000686;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19834557;citation_doi=10.1371/journal.pgen.1000686" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. G. Meirmans;citation_title=The trouble with isolation by distance.;citation_pages=2839-2846;citation_volume=21;citation_year=2012;citation_issue=12;citation_pmid=22574758;citation_doi=10.1111/j.1365-294X.2012.05578.x" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Population genomics of rapid adaptation by soft selective sweeps.;citation_pages=659-669;citation_volume=28;citation_year=2013;citation_pmid=24075201;citation_doi=10.1016/j.tree.2013.08.003" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. W. Nachman;citation_title=Single nucleotide polymorphisms and recombination rate in humans.;citation_pages=481-485;citation_volume=17;citation_year=2001;citation_issue=9;citation_pmid=11525814;citation_doi=10.1016/S0168-9525(01)02409-X" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. A. F. Noor;citation_author=S. M. Bennett;citation_title=Islands of speciation or mirages in the desert? Examining the role of restricted recombination in maintaining species.;citation_pages=418;citation_volume=104;citation_year=2010;citation_doi=10.1038/hdy.2010.13" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Novembre;citation_author=M. Stephens;citation_title=Interpreting principal component analyses of spatial population genetic variation.;citation_pages=646-649;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18425127;citation_doi=10.1038/ng.139" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Pavlidis;citation_author=J. D. Jensen;citation_author=W. Stephan;citation_author=A. Stamatakis;citation_title=A critical assessment of storytelling: gene ontology categories and the importance of validating genomic scans.;citation_volume=29;citation_year=2012;citation_pmid=22617950;citation_doi=10.1093/molbev/mss136" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=A. L. Price;citation_author=M. E. Weale;citation_author=N. Patterson;citation_author=S. R. Myers;citation_author=A. C. Need;citation_title=Long-range LD can confound genome scans in admixed populations.;citation_pages=132-135;citation_volume=83;citation_year=2008;citation_issue=1;citation_pmid=18606306;citation_doi=10.1016/j.ajhg.2008.06.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. K. Pritchard;citation_author=A. Di Rienzo;citation_title=Adaptation - not by sweeps alone.;citation_pages=665-667;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20838407;citation_doi=10.1038/nrg2880" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=J. K. Pritchard;citation_author=J. K. Pickrell;citation_author=G. Coop;citation_title=The genetics of human adaptation: hard sweeps, soft sweeps, and polygenic adaptation.;citation_pages=R208-R215;citation_volume=20;citation_year=2010;citation_issue=4;citation_pmid=20178769;citation_doi=10.1016/j.cub.2009.11.055" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. Privé;citation_author=H. Aschard;citation_author=A. Ziyatdinov;citation_author=M. G. B. Blum;citation_title=Efficient analysis of large-scale genome-wide data with two R packages: bigstatsr and bigsnpr.;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty185" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Rellstab;citation_author=F. Gugerli;citation_author=A. J. Eckert;citation_author=A. M. Hancock;citation_author=R. Holderegger;citation_title=A practical guide to environmental association analysis in landscape genomics.;citation_pages=4348-4370;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13322" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. B. Robbins;citation_title=SOME APPLICATIONS OF MATHEMATICS TO BREEDING PROBLEMS III;citation_pages=375-389;citation_volume=3;citation_year=1918;citation_issue=4;citation_pmid=17245911" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. Saito;citation_author=M. Rehmsmeier;citation_title=The Precision-Recall plot is more informative than the ROC plot when evaluating binary classifiers on imbalanced datasets.;citation_pages=e0118432;citation_volume=10;citation_year=2015;citation_pmid=25738806;citation_doi=10.1371/journal.pone.0118432" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. W. Schaeffer;citation_author=W. W. Anderson;citation_title=Mechanisms of genetic exchange within the chromosomal inversions of Drosophila pseudoobscura.;citation_volume=171;citation_year=2005;citation_pmid=16143625;citation_doi=10.1534/genetics.105.041947" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=F. Schlamp;citation_author=J. van der Made;citation_author=R. Stambler;citation_author=L. Chesebrough;citation_author=A. R. Boyko;citation_title=Evaluating the performance of selection scans to detect selective sweeps in domestic dogs.;citation_pages=342-356;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13485" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=F. K. Mendes;citation_author=M. W. Hahn;citation_author=A. D. Kern;citation_title=Soft shoulders ahead: spurious signatures of soft and partial selective sweeps result from linked hard sweeps.;citation_volume=200;citation_year=2015;citation_pmid=25716978;citation_doi=10.1534/genetics.115.174912" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=S/HIC: Robust Identification of Soft and Hard Sweeps Using Machine Learning.;citation_pages=e1005928;citation_volume=12;citation_year=2016;citation_pmid=26977894;citation_doi=10.1371/journal.pgen.1005928" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Supervised machine learning for population genetics: a new paradigm.;citation_pages=301-312;citation_volume=34;citation_year=2018;citation_doi=10.1016/j.tig.2017.12.005" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=T. Schwander;citation_author=R. Libbrecht;citation_author=L. Keller;citation_title=Supergenes and complex phenotypes.;citation_pages=R288-R294;citation_volume=24;citation_year=2014;citation_pmid=24698381;citation_doi=10.1016/j.cub.2014.01.056" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=K. Tang;citation_author=K. R. Thornton;citation_author=M. Stoneking;citation_title=A new approach for using genome scans to detect recent positive selection in the human genome.;citation_pages=e171-e171;citation_volume=5;citation_year=2007;citation_issue=7;citation_pmid=17579516;citation_doi=10.1371/journal.pbio.0050171" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. J. Thompson;citation_author=C. D. Jiggins;citation_title=Supergenes and their role in evolution.;citation_pages=1-8;citation_volume=113;citation_year=2014;citation_pmid=24642887;citation_doi=10.1038/hdy.2014.20" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=B. F. Voight;citation_author=S. Kudaravalli;citation_author=X. Wen;citation_author=J. K. Pritchard;citation_title=A map of recent positive selection in the human genome.;citation_pages=e72-e72;citation_volume=4;citation_year=2006;citation_issue=3;citation_pmid=16494531;citation_doi=10.1371/journal.pbio.0040072" />
<meta name="citation_reference" content="citation_journal_title=PLOS Biology;citation_author=B. F. Voight;citation_author=S. Kudaravalli;citation_author=X. Wen;citation_author=J. K. Pritchard;citation_title=Correction: A Map of Recent Positive Selection in the Human Genome.;citation_pages=e154;citation_volume=4;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=I. J. Wang;citation_author=G. S. Bradburd;citation_title=Isolation by environment.;citation_pages=5649-5662;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12938" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.;citation_pages=661-678;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554300;citation_doi=10.1038/nature05911" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. C. Whitlock;citation_author=K. E. Lotterhos;citation_title=Reliable detection of loci responsible for local adaptation: inference of a null model through trimming the distribution of FST.;citation_pages=S24-S36;citation_volume=186;citation_year=2015;citation_pmid=26656214;citation_doi=10.1086/682949" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_title=EVOLUTION IN MENDELIAN POPULATIONS;citation_pages=97-159;citation_volume=16;citation_year=1931;citation_issue=2;citation_pmid=17246615;citation_doi=10.1007/BF02459575" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Wright;citation_title=The Distribution of Gene Frequencies Under Irreversible Mutation;citation_pages=253-259;citation_volume=24;citation_year=1938;citation_issue=7;citation_pmid=16577841;citation_doi=10.1073/pnas.24.7.253" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Yeaman;citation_title=Genomic rearrangements and the evolution of clusters of locally adaptive loci.;citation_volume=110;citation_year=2013;citation_pmid=23610436;citation_doi=10.1073/pnas.1219381110" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=S. Yeaman;citation_author=S. Aeschbacher;citation_author=R. Bürger;citation_title=The evolution of genomic islands by increased establishment probability of linked alleles.;citation_pages=2542-2558;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13611" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Effect of Neutral Recombination Variation on Genome Scans for Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="twitter:description" content="Recently, there has been an increasing interest in identifying the role that regions of low recombination or inversion play in adaptation of species to local environments. Many examples of groups of adapted genes located within inversions are arising in the literature, in part inspired by theory that predicts the evolution of these so-called “supergenes.” We still, however, have a poor understanding of how genomic heterogeneity, such as varying rates of recombination, may confound signals of selection. Here, I evaluate the effect of neutral inversions and recombination variation on genome scans for selection, including tests for selective sweeps, differentiation outlier tests, and association tests. There is considerable variation among methods in their performance, with some methods being unaffected and some showing elevated false positive signals within a neutral inversion or region of low recombination. In some cases the false positive signal can be dampened or removed, if it is possible to use a quasi-independent set of SNPs to parameterize the model before performing the test. These results will be helpful to those seeking to understand the importance of regions of low recombination in adaptation." />
<meta name="og-title" property="og:title" content="The Effect of Neutral Recombination Variation on Genome Scans for Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/6/1851" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recently, there has been an increasing interest in identifying the role that regions of low recombination or inversion play in adaptation of species to local environments. Many examples of groups of adapted genes located within inversions are arising in the literature, in part inspired by theory that predicts the evolution of these so-called “supergenes.” We still, however, have a poor understanding of how genomic heterogeneity, such as varying rates of recombination, may confound signals of selection. Here, I evaluate the effect of neutral inversions and recombination variation on genome scans for selection, including tests for selective sweeps, differentiation outlier tests, and association tests. There is considerable variation among methods in their performance, with some methods being unaffected and some showing elevated false positive signals within a neutral inversion or region of low recombination. In some cases the false positive signal can be dampened or removed, if it is possible to use a quasi-independent set of SNPs to parameterize the model before performing the test. These results will be helpful to those seeking to understand the importance of regions of low recombination in adaptation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
    <title>The Effect of Neutral Recombination Variation on Genome Scans for Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6n0Fzw0Z46opMF07JOKMpqIh1gvWdpvTfycjXO8N5uQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493256","apath":"\/ggg\/9\/6\/1851.atom","pisa":"ggg;9\/6\/1851","ac":{"\/ggg\/9\/6\/1851.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/6\/1851.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1851"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1851"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/6\/1851":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022AewAg50kMslm9\\\\\\\/eHddTuHyNa9K+2KdbKW8RmV8r8helFAaN3lO64J6nh78CCOTs7nwnCUCuBAiXCPfiCKZZ608exigj1vHpNC7+5+WbGyfnDSUkIMWDbaqh+k0wXEJZzQTYgNixNRbeKi3TlAyW+DphHjDnseU4k\\\\\\\/PrG1rco13vaTVYkKRH8yjZw+vsVFXW2\\\\\\\/cU189xMXt96KwejXgap41T8X5z9vFJkdzLYkZ\\\\\\\/KSgpbGpDzibPUTU4mJjsqfKIY3umrFbVonu101S0hGnzhJe26SOSVXDoLNObec6Nel8sUQ55v2f0xI9\\\\\\\/ndbVrm8aSnAzv6rt2IS7RYX++9Jj1F1JcdejOIanI55zEam9iQaSvPUUY8a7AxS8Sus65lTsvXAq8DGNdxRFGVTQVqNvu2k\\\\\\\/l8DVB3X+Ex7A3nN7rjvQke\\\\\\\/ePrpMhGwtPaw+eBfIdFNpiEsd4S+JWL7vsqZ8x8\\\\\\\/fZkyJrZ6dbPD0CI51Hi6WdEcDsWzugnhMWxB6iefdmkD+nMc2ROyO2QdCXXsZhJ5JgnrIyq4HZ+m2P3SiYHWoTA0A+ls4TL2QwbOKeyBhg757\\\\\\\/94QlzPmGxYeOKOIgizDCjdSIg4tnmUwSEnss6RBsH8Qq8vQKl0rdC+F5daP2cqv+D5K027R\\\\\\\/MOXLzTOCayhFzznJoh0iEA+IpEcJJdT6PhJNJc1l9\\\\\\\/d6OdVEfNFABKsMYNM5+1XWGzUS8DiY+mb+Vvz21kMK5EDLql1GLELpf6DBkIB+kQHFHrFu\\\\\\\/tzc7u98UTed7s7ozvX+TcyXqBJnGU+G6mrhSMtsN9Kech347L7DDUWQPT1GZsk9vWq69Lkdui8UHcYKi4djr5XeBCPBgDx76EwSkTD6Yn8ngmzfGhztYVhFgPXpNBB5FkJJJ4uNpJ+whsnwKCyKIRiXS8ekiMbjEw18plzqdcBnkbIZriYXURQ2MJzrRXm566kykwZvvRWDKhF2o2Acz75ribzSUqfnkpRngmwT4EIPZDcOoyJN0SXIDtfExiUpqe8bJok0Sn2\\\\\\\/A8NZEcCJP5K2+eUyqtlg9ZbonI01vSoH4rGRGKmpuP3gB1jQUYy8ILjTFqw1LTVZBhu6z+nk5si\\\\\\\/8fuZrrYr27iW61rJFVCGCqyV3RIgmrwMHufDrWFjPUPR3idwZul8vUWmhprSg6vkaDQLq6k8cN7G82nnp1kuIYWrULLm2Grd62b1ZpqXw1wXyTj5xx86r998OrrgUnAz\\\\\\\/IQPPmkYO+CRL0UB87L6cLctEj1YA1B6aWJ7E5TbN7\\\\\\\/l\\\u0022,\\\u0022iv\\\u0022:\\\u0022TimCW6m5Av+X6Ektre1aIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002223968bc5f370fa44efde0ec2e64d234eb25b9826440a5e1350754bd2a9b99a28\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OO0akPbafVUh3duX\\\\\\\/sRNDAcp1YnaNSJwsVTqYwl8lARy2VoyE3ndWTqxxb1Bv8mgDL+YfJ8i+sPlLquerinRZDSx\\\\\\\/+ef97EDPV4O5c4qjybHfxoXdPFN7uJ19trmwEEhW9+OG4XyT1zzssqjz131Hx90GdL6UnnpRh6+zWu0lCPWwmY4PxMhML3zHPs7bpkSj2dnEc0yW1GziqtHSCrkgfuf0ECrQEkNPyEvsk4YBANkqCv1N25Tkk7rOPrXqrlJsiLVsNN59Pv+ipgTGjC1xTHdiTWbH78IPopz5H6B\\\\\\\/0GSydwT15Hrkiub\\\\\\\/AhN185p9zpSi2Xa7xSppNtviedrsUTPwr3tr1KI2Z75+30iRi62TWyLQbqgvbpH4Yu5Ycp+LEvJtCk9+\\\\\\\/UmF7nDjFhmsf4V5IO9mQMhXAH5UioYTD5dpmJoobMLYKoBALVZ3cN6L0isLosIs5JXX9XOYnpF5MzRoX2YidbK3qOQH5llr99dHIiJrpQ25J+N9foVkYq4psH842wjzJVoOeX\\\\\\\/bs+pE+HIN6iriT+l\\\\\\\/SM1lHszIezQcfGMgZIb+PckdRJN\\\\\\\/\\\\\\\/HIRlmkFto+pvF2mwsywOYBjxpu\\\\\\\/4BZ+g7PY8EJcS2YnCNdTfPBCDJ+37DJcEOla\\\\\\\/OgchisHT0EEgW32StvrQIsXI\\\\\\\/Q1f+PGAfXeIOpuiq9q\\\\\\\/Q0RashxsEHWRCuYSBbiSFoSibjZTh\\\\\\\/wUn6wawu09GsjE1c9ntuXY2fPSQRAsi1PA737Hgkw2OKt\\\\\\\/YW2BXPJ5cAQgceLQB7uqP5asTZNdByHhbIgwq9LhUDGyUzKOp+IYhoQW\\\\\\\/glh7he3WK3H5iYx2IKz2d2I044Pu3uHbjuOgVUeSWCdIem3TkLyO7I2GcmGTwQmwtul+4dfgbZYxUvyM97uG0jDf8T3g5AoyLfDygZUJ4fHPai31Yb3obu\\\\\\\/cJ1tFUVRKOnokr\\\\\\\/LVaLSXK2aY1kkbdnZjwkrK\\\\\\\/2IHDUTLkMPvLEge6lCYwQheELPsffiND7YOqDyVUjozr045p3ZvGWMqYwMGKtdXgis75TofEVxXs6Ew63vdxMbdIJfV6HvzM8CjS2BSUswDwuDf2XQ7hNBEOq9rfkRAC38ozBxQaN5Dlw00ya6i0Pgl0RjzlVxIXyYAd36rUgRoYskHwSJaXMD8xpk6PuQPDQFxG+eeVsHvVMMkBCSuC1rOswctOPd27imQ\\\\\\\/OrX1TK26KZcR4Et7lynFM+7a+UBRJoxy1MYPv8vH1hyv75Gglx6S+hINtAut5EmRK5+Aj3Ql51+uENZpB5ZirbE\\\\\\\/VWO3YbevVEN015rNQtIGRkUGuJS+gzw945EwIGOy838BGP1YSATn0z0bFA+Yju0Fr7elzpR7bpVHC11usddbliTJPJng0vnLOUkAu55i6Yt8ujIA7WAY8\\\\\\\/6zYWvixJ\\\\\\\/VCvjKW8DzQL6xavzx7xG3hrqvqSbue0DAAnefSwmp2vJCB+x9Pps4\\\\\\\/0QnL7sOWqz1CPDiOImPWnERey79s6OwrlIq13Tb8q8W3j+qRqAhoy3aCHW\\\\\\\/hsD2R7XOlDI+sD4r8ylK3dVwHqmZeQeLr5RToWLlTl4H+0KJcQ8mq3AdX0RG63rfeNId0m\\\\\\\/E5WY6XKHrrenvxP+1qDAv5H\\\\\\\/kiHxVBqmGwyvLVhMRYBK8dfWZg8q1d2cjCoqkb4XsW+VeaAL8uO4EXhhgImBx8WXNNq6AJfywvD+qPeyqf5GZS\\\\\\\/SZQXxeKMn+nw9qjIh3dg\\\\\\\/gwdUNQymKjMvlxD1xMisGLeJJ5LwiNEhijUlwhuH30BdlLmbu5JzOi7N6nNQzrvejWpbJiE1umXrRDemF855H1hJYXXMmo041UFILQgqipN4Bn6cMMuhFZK4BVEO0POW11i55AMG9oS1V7HJYXMRjhyYacn\\\\\\\/SGFOWZmcKlQTknLFHkqpid42XQwsQyHbqJ\\\\\\\/vTJjB1Q\\\\\\\/94RpLGPwzePTrbLFyfyfhY2y6kgYFssKxBCmnvlqCf+Q8yiphhf3+joWJensxLVdqmdHOIjN1ZSet449gDyar7sguZdxbZa5IxPAD+53mCE+cEXbU0rnFmeWtlOl27kKIWl2CkIIiJs2Tq9s0jOYTBtiPXvTMQDtEy1YVZ2++eHNyR2oKtKjRVSk0Q\\\\\\\/XF2Yrb0G95ObCd5FG+OTmY27TdoRzhWCvFhAmIbvhz692102G0gYsVmvDy8vhocLK8M5SkqIZUi0HUL3iIRl1eWyCJfo7gEhWBtGXEJbODl4l6ejCK7EYy1AWJWRzta1DcE9S4bQtcj6kq+siwf1KEEZ7blZtwmB\\\\\\\/ibnvmeyF9+Ztbuiv+b4x9YCPDKyPoaTYu0kyfVghpy5p3+dQAY+m2AquUJWIVdNgfcwDkSQ24YPFQUvt0640SEZVl3gYH9EkldU1SfsYcmrpgZcRq6Ey1Y+kQamPoFlbk38DB\\\\\\\/ys3SRzDAumXthEMEI4e13H279tXNfNauUmr9erQr+M\\\\\\\/QoQ9fJwEHcCzJOw6onTJ+5TRItUJJC2Sdg2LORsdxf6WxTLmOWm9Dpa02ZOGUGc6aXLOEwzonOFRPFtdOa\\\\\\\/QVVmX3xvsA+3qCRHX9Muhl7nL+LNjqqdjYmsElMX6qVIlq6NcGzm0IcbKRELBucX9ENOE+mReoSouGzzBsYyibsnbSGA3Q3gct9a\\\\\\\/a0q2+jqLtSO90BOUvd3mGpzQM3GZkw5zqKcfH83MqxN3txLrA0bCHAc0mtuX9Z9FdJcw5jugdmzbza1iatKbVgGlviZUDqELG1icPLHkr8qcIElji1ZoNxrTK0DS8W2UDiJp0kL\\\\\\\/8bjzPeIqs0RQjBJiLBq66ZnDtAWu5pE38+lNAy4bT+YQzJRsRF74ua1dljQG5Fr15ICHdaz9voOw8Sv5mI5kzBHoLilGPJ6sx4WYVn0jhXSquUfxa56N7HUgCW4aWzcJcA4fpyxR1KE61xPt9hr5h12gwnv90hqaEzTIan+mhPPpk7wyYhMU8kCC\\\\\\\/+hTEo8e1oHcgzvOp6C69L1mfqRwvjk\\\\\\\/BJGPQuUn3m2Pv3+gG3mpVWGwEkdX3xFoHVvWG0mhKl+g688vDoF6UuCBUdvNlddmCc9fCM+vQU6peQ7CNMyBNG46C\\\\\\\/MHDWIIGCP03EbcoRrPuwhdTBHPi8jIJg2am7BFFEGSADT+FjUiJbejYcx+2px7RQotBsq6WZurovZfkf5rEChC6boOXwrDZCjj5NZz2u1UfPBiFyI9otgmTBjvraYJ86apj98exx6vLQz2kYoKOtYqjogG0LeHmvnW1OpCtxdKQcANoIsQljSal87rOUKTYRuJEY5VU\\\\\\\/iEDGq3WXvTdZgNQVarr4mXwcuvOnKfYgYjX5AIOe+RUyM3Bq179DGmqlAoqnqLUZRj4mjE1\\\\\\\/22Xp42wXXvKuS\\\\\\\/Nerne\\\\\\\/DK0XH\\\\\\\/4ehiDeIKp4vGoJevGuhJg2LKcoPF9InAipCCarxfVlp6vLTKhMoYg\\\\\\\/caNE3v5GOKEUCWHZTlXBVShazRupOXIrscu8BoSjEKlytclgqpgAhfl5Fp9UGH27omcjJiIZ1ynHeZWnuEYCiOx2I27miBV5IQetymFGnUrTgGp\\\\\\\/oCOkHIQuxLAndzye9Z\\\\\\\/KSByfRRHQ7xmMJq\\\\\\\/uXaAfbzW5AmYVnOAl9EniIZOdNz1VsUy1e2btKU2tBQxSF4LpUxFY\\\\\\\/WY4bzjiMh1VDelz7kfS8OQHrVwDLnRJNA\\\\\\\/EKIYzi+Wk\\\\\\\/g20P3knrvoEe3NSCQy+KLPa72+pLTfGDARSZxFLODyTKtefHnPKo6X7AaWhDYYiAldHSsbeR3+O4vkztTVszOlbWJSoglGHNgWWfqAjvNB\\\\\\\/E5VnScSq3+AxCYxlqRxLtA1TM9rYDQjvS3saa+CO3WZOGG6pDyxXT7CGcXza8V7cl0XtjiSI3pxEOA4Ub0IfFJW8glNmS0EhbYKMkkcdAWXuxCOnnGVbC8oj3aU5TX0ndCKq+evBXGQp0TogukouWVSlTGl7vUvgtz2VGoN5Q9J\\\\\\\/8DhXGKzlBji+P3xUtK6aC379dG9hpmU4OLrllySUu+oI5p4Aowlg44Roh3H+B3WObcQmYbIYUa6WlSV46xPXIyVTLu1838WOtzR7hlmdJ9MctjoQLL734dHDfTWTkuz3Si3u3dx4nZW+vzNuK2j74vYRa3sChqAJh4FnG\\\\\\\/ICjL0Qqa288NIzOgE4Pp1TsBfqzuQNfHvheg0ex9CR++b65HGYZ\\\\\\\/mRkUptxbZ3U9SUaA2XUn85J+2d27CWW9HiOb\\\\\\\/IDyMJpb\\\\\\\/Gy00NHy6JCMKLXnsLqjVhnqfAswUFEnpNjuuFN5fym016qvP87jazcIPvHEGEfYRomGpXq7wk\\\\\\\/wrv\\\\\\\/oJLQBzss7AyINhIq2CCxdXxsL7+NNCYoIoWPjbwm9n7Bu0UqPqpdBnixeaQRYhzK2Gh\\\\\\\/bLNYr7+vPrIMDPkHvYa4TxRoYoiAzQ5nQ2EAdEl8VMkZm7BTk5k4PfDy277f7cFMlJg8rJPSLf7HuyNL9i3\\\\\\\/EAG+iOJAuTjVIwWHYJK850\\\\\\\/mpZUjIJqobAv249x45C5o0COnzWaCjkeeHQyOaW7BDyB3Jy8Fgpc9KcR6nBpfKNXXcwW51BQ2uxQbHpr+z21s9fxCUyoNaUplcQxn9J9xESUwZVJqMUuB\\\\\\\/r6B1kQvJTg4OtFh1fuTyCuhnUMlzxVuOpw7Dec4hh\\\\\\\/Vr6ymNH67wFc7Pbvoxsm36afb9vdDj6xhUci6rRpD2h+pKoDm4vXg+ni3ooM9hjzudMoOwm5kWtKdVdhu8dh06rQGJRN9vbmlM3RFF4nt6tDfhFMGjXMAxJZVD6YltVSLp3yNj6Xrzon2SZC6CjNkc9kU6oQXiWJr+VJohAPKAkvQnT+Hd2m3sPpbtyVv9c2nztiTeCRzplUz0pcyrX1gzL0g\\\\\\\/7zbfn+K7LPMlhT4mYg4Nz4utHDIqUVtGF2YhBbPRiewSgwv3lR+FzeaedwBXOCMVjhVS\\\\\\\/kOr10iNEKz5av2WqNLrl+pI45oH8zcMzy2vVCAqyGAa8tncoEXTIamI0CqFUXtnsA+85F26HfTVBNWNQIHTuOHsXwJeUh4kOVJbfQr+fZAR9DUuoYaXuFJ0eD8YaT\\\\\\\/U1Dl2B3SdNjYx22i5DB1XeRxngyXgJjMfATVv3aJKLWj\\\\\\\/95Ib91T5rjd0bmM7+Vxe9pPXIPNR76bFoGptkqemOpxDYUMIyf2afW6a869A2yBITG2fVC+MbxhUfkaJExMKH7MTPc0H1u9C90xwet03aiVv3sKRqog2BdTxNhGsOt25+YPfUexZ6XG+VBnJ2QiAP7e116vGlShuOG0xCCf42XHaOCJ+05Ce27nezClJ0EZliHDMHAnsGbqwJCmGOQ7fHOkLk+B2Gy7hvSnZG55UtedZ0FxSHMJE+Ol\\\\\\\/m7cWCmI0FNnmHKM+Fxy0bnpajsxITJmKwi17AR6jZGYzEVx9X2SydsaERSY\\\\\\\/yomZeCCxdBc9sKrnvHieMuge41OL1\\\\\\\/vsVDz\\\\\\\/pH7LVRfQkHn3jjo4G9wWzS2PnOE01lBp3J2\\\\\\\/YxKmxjKmOPnqhT\\\\\\\/CRSPhSMpz+adQXZKC+1EcwGX20VKCasMbPcTPfSB+Vio9tDqgn\\\\\\\/rGRRA\\\\\\\/5uo3gXS+9\\\\\\\/cfEJ0TPY5Pc6BVwgJdDvEgWAF6PUkf39VwTLRmSLKucoTFTxiQIFu1et4ykq4T83S74svCq6yElV3xoD4mABC3f94+\\\\\\\/jKjelkPubM8obn1QZ9TMBLZeFYh7zZOSuZpRlyKj5zRcUo4UPQbb7hQEfpuERJmQwcWMH+Jd6eXBKoJSyHgi+Iuw9+QvQ4Xl1HhXkhc+ShkPr+CEb4HGRoJKS6rNSy9zaOsfm2nPmhR4iwFhOTCYSuxcDXrEPCZXfp+WJJTM8bFIOx4WLMD2mlk3BL21I7cj0xkBGZXCR3Ny+GClOvx59Tz08dT1TbXS9ATBdU5EeL6Qxxri9rCaLlb6okbtewaTOXPqfjYc82\\\\\\\/iqGLYqzqykQ5aKgTAh34uHrhVcDH47M3X4kd4BTPMShD6QP0Q6xH731e6MDNDIUZaXolVpIVF5c5QwDFwVSdvb0DY7\\\\\\\/WCnHZIud14aW7RXFgUW5ZWloIOVUvRiM2cDmaBElpDDIfXeAa1fo6blAPF9BZsbPUWlamOb\\\\\\\/WbFkOK8ErnKo1wHginuNX3wMfZDYpFHn4+u20C\\\\\\\/2e1pHBD8eMsri+Lnt1t6ySELPzeMeLf6vEue9ECfSwfHGkfYdyO9oCqAYa01vh7rsDrFgSv4e9f2Az9j9N0ex2mTWsXTGyWTJW96e8mGDspXYOYDmteA+6mGjOtoHdc2MyLZxrZ9doR8+OF8LI14C1sEmux0axKUZVmhELvxCd\\\\\\\/wQNVSYxFJNcBT25RjCLXtQdegLuuhTbugGdWg\\\\\\\/tVgqYfybIn3oVG1Xjnaf7trJcoIY\\\\\\\/+gb\\\\\\\/Sv4Sj5iyqbmj6HpGFGR0XqbJjMIDnVBiBB3l0Z3Vuhdytbkw\\\\\\\/M5DYAaR397lmeVWYSY5P5pCC9APeIkTf2tk\\\\\\\/LcvcoNUgZis+RHGvEoRlu52r3pxBUIgzc0ZmxT0KhFevNVXaYml+Dn2OBeflEZlC04m7ecKVIk+TUaRW+SnvDKcdgs\\\\\\\/aAExQtK+odj6OmJBhDtTO6cl3ty8H+CHzvq7Rm+RjkBsEdxHYtm5NpMgTzaaoiUoULswMvoo3hWd+nQObB\\\\\\\/NGN1q8VkOT8h6Wv3BTLTnUI\\\\\\\/qX6KUjR3tHO2bRnuMVBmJ0D\\\\\\\/F3lxtBIBeWr1LgUxDN2oBVUe\\\\\\\/UoCQG0+9CaWHVXdrhEotL\\\\\\\/nsQ07x83X+AuFHTYeDzWq1jjVhGYw1\\\\\\\/DP0+vCXEZm\\\\\\\/38TSvhXIVWGqk1lK4+EPRPWu\\\\\\\/qSLpy4CkCiwM1ch3Npgr8GuPr\\\\\\\/tBoBvpLCzIa1DbJndQ4GNULleqZsK85mgHPwhyhHKMS7HeAaC0xZBlmegXm8ri9\\\\\\\/tRRRDS8Bwd5KMomHR5y+Y0lfyB2RU8Ukx8HQGm5bZgJ5BLDxizO0HD7nfA65AXSuIO0puyJAItJLEkrT9Tx9vxQDS3eRfY1cVlhLOTTRIClYeYUjAUJHDW+SVMIpT77llZZKcYxgkDwIHtEfBA7cyDAC2lZEqtKsm7EGaXdwaEN1AK1YQDeCQkGJKcKwoFhyx03Xygq2eysdjYGsimZ7akSHGbb0QMiJMx5jiwidQySnf5L88W996GLICKSh6efO6TDNQPd26Xz3p\\\\\\\/RPhiMnfckiLHXY0QfeVxhs61eaY71UOjfMsCnt6s0GF9GuCMEhKfFcbwTA7s7qIwJFXzRh6IZu31SuIzJCBpUJdrnwcVDLeL7IDteEBWDPQGi2qGEYdwdBDAo4FxkuA6+XQUDdpK\\\\\\\/t3iXDHto741dIan8pURWeRP14nOzo\\\\\\\/EWeAyyZRj+5xBPrQHoZ7XIp8fii4AWT9nXiy+R2NxjeI1YI8Mzl9md1B9DqMr2HCQ60CpyhzRaStrsvFC5abJNOYsrYRch\\\\\\\/0p3yjNTMIbZoqCwU7jXSWGB8LZhO9ztBI4y60n7GL5Rq7k8dwdy1JUuE3uHB9RmP6Ho5dPY8slS0cvdiqATTaZgP2qMJSjLcw0\\\\\\\/0freSvzWTKZWVDZX8DcVO4rWpjGCTdDW46Yp+Rnq9jkUbOZjzKMyW+wRz2nwluyJrFq2fsBAM+TZWQb7BCRPC5DDeMFlSRqdEMIKdq7RQIaOE08LxpK9e23jQrUbSEpEJ2XqriXyF7cjPOTrivNb6+6v3RqJuum9RlFPqLORx+Y+M+ez55qJ15XS7TA2S5PIeu2LmOnKLcV+LwnQtbCGzawv77JFRPS947zm+cE2zOuggMbY1BvPCNwbvlN3cqXy+QFrPXyQNhMrCKcG6CHBgDGZxlnILPtCJMoBom4k0HSVcmgB4uwZTIW2+m6Sqn\\\\\\\/Pm4j9IZGvZ\\\\\\\/QqetNwJ\\\\\\\/vJQYPPbEylkYL4kxDNW1wyOzNynUBAaoEFTz9OPlo9FZvvnfIPOwVdsIau30T8\\\\\\\/2Lw0Ca5WN2sVo8nHpHDJOBOtrOlT8JTpM1VOb4pvRYTpz1klk8iIgTd9uQ\\\\\\\/4SrDgaPM41REiaKKSQ914BQ\\\\\\\/FlwRMzMUHjCcy\\\\\\\/vfanOsI4ozBmRbKWa\\\\\\\/2m623NJuD83rSszPcFcsw13I\\\\\\\/MkMGtyBTFwmUttVD9YHwmeTx0AvyNpBZlF4AkNU+tvohOcLkUqB6KoDUcsMydAAZS\\\\\\\/hgsQyOKm4guM2woG1yrAUZkP9B\\\\\\\/gX8Xa+aJ5gdXanWuRI8ytqwgP8Qz81dZnfcmPj\\\\\\\/kAv89uVj7elhbWsBOYpzRAIK2\\\\\\\/1V\\\\\\\/cYwfCCI3ZOnO5IZp2BmuGzxWdG29XLtc9jdoj\\\\\\\/rpn7WVIKdBOUNHyD22hkRKCRqHPfwbIwgh6G+tSy4I8iFuf0JHPE73cYR54LRhmYQROJ+oArObv5\\\\\\\/UhddIGfmLyGP7lOWfG4tsWQLzq06QusTHwoPpZgk\\\\\\\/RyaKj29Oqc47kYKqz+v2pe4cQLe1yV0F7BjvMwGo8AUa25WVnmCvZEybcpxz3BLzkQerUEsIBZn5WoaDeEJxMWDLl3mASVdbKdvev0Vu7GII8oavdKSXDzZi43dzoI1gDA35r8jAIRjBR8UiNdliSK\\\\\\\/o6gyXg7BKRmKvTdpVk1JiBizeKCGL1CGYDtWs1nA9mbFybVCM0R8z8E93fALR7XbMOYMitusUXruDUAw7R9z53AiFZAiOVvrx\\\\\\\/0QBpEnWFNqwrOCI7soc1t9qoSZOcGhndDnudV\\\\\\\/Nx0JpOT5EFMj6ysNW7hWGWJ\\\\\\\/ViDLUVMAWw5FcTlUi7PwH62WI0uYJbJ2MdbqJ9co3L57jrf5aQztKqofaCJZ+LSVmR38K2tEA6nHVLKmLhziiSSDFeh\\\\\\\/DsezLl7wpTdlfyM9vZ13J\\\\\\\/EU1HqwDCxyoln7MqhEAP9ibUTpOUasR7ERLgX0o7xSL6IppLxRwjGM3aJy\\\\\\\/LYk2ryR3fMQsxvmq7TwiempmQvsebL44VQ1tM4YBdLzqVgCUw2rpWz442KnSVksE\\\\\\\/b1fRVQEYF63F5x5P4XNM9RTolVVeQPONAJIFpzMRGU7ZHszMyYx2dEjH76JKk0U6ZcR9H2EizVR3PfG075weFa\\\\\\\/\\\\\\\/18YLAPWavBFKXddebmsHYmxkWq1YYy+WA3taEO4vHCXs\\\\\\\/\\\\\\\/EWfVG8ptYYjxEQN3sbYOQ4+gH9\\\\\\\/e6puKOYGwyAaiLazvIdghhZbFgMBD0KdjArB1oC5GBgNz0M3\\\\\\\/hfTDnjEyhkBhJORrtNC3TqEJNC3JF6X4HpB1XBLmZPaPzf351iAMhHT9Dp9AmTRy5uZ\\\\\\\/WgA8DOXDlfyEMfi9wf0TZqS\\\\\\\/7E48cMqK36hYKnuz8oebQk9Moq7i9hboHi7V3Qe0s76WlyhAErEyV912o5Hy5wdPnSaEOVuTeSzWrkCKJpNRISJc8BzlrfiziQ8aK4gDuyQ1aqRuuNQcK3TQc28whVYFouEWdK3WRFPmEK\\\\\\\/ac2pCUjAXvLGA3MBoTXUMaWWjpw0XJAE9r97JRc7Xne+Ci0IVMIUbehbabBmCCXkUKweJutzVPX78F6mqkOdDxmYewuDyvsZSoRDkxRcW2MgKIoaMYrjdafS6UeUyK0\\\\\\\/AIe8OuW9ffzPAtXN8qVDyUDRUP8jOKTfCu8dR0vNLz8EryTHbfO7a3ylj0czPUBCR64XjobrgTBN0nqI8FrRZMkng0+i6DZ8Ukm+P5pAC2NSerCgMxiKKZp679MTUg9IiblMVPDCuo4JDGi+i\\\\\\\/oJb5ApD5dbSidiq81rciKFl7l\\\\\\\/rbIKXaAL2e+PmTtVLDmg\\\\\\\/rl\\\\\\\/gsTI23jmHQmg6zHu4O0Tgp5OiI1RWno0nCTpV4N1NLUu+bTNj24OpngYMYdCiGyQuwZ\\\\\\\/XoUul\\\\\\\/3wFvaxlMCMbqWpuin41TB1HF0+2W2VYOVjFWzgtuP9BhmQSMyCdK3yQeBtwiRt9n5gY\\\\\\\/\\\\\\\/uYI3gn3RVL2S7WYUuZu4nbWTkirxk7JbMbOjQOOOHiaHeioJwp7iRKh04A3rCqoJQDKGLEDxT9DoaMj6LRcMvBBbZV0IzFqt6Fly0TbO0b\\\\\\\/E39UHS3crp2qny0EnNrbPkxuxCDya5zt6q4Qtb6F8R7YKVY8aXn5mGBSef67PHzSX5HeCRyCRJYHu2WdQLIsWU83BCTMesYHVXtjknD\\\\\\\/8YhdsA7VdAFT36GSJh0EWKgV\\\\\\\/ARwbcldUb1EXjOLBul+qr2KJ+Gb\\\\\\\/Ta0y1qES6pOxQ+SssAqnaVXQwSeFfZXBhgVlI0jaCng9W2EL3NnF8XgoF8Fu8F1e8UC7wn23qvpk2pmXSi\\\\\\\/BkszK3gFOaREluz9xMIuCuE7uvKsCM5N\\\\\\\/dHc8l2p0GykpByFUxuYhjJ+ecRMEqxP4FqDEOJS+JN8Gtg9r4GUroaaQxeW9aaPAY2LV+Bc\\\\\\\/oOBSG73gunYNDu+amBw5vzx6U0S\\\\\\\/cAxgWEDO1wFsrj1X3yF87BVHYl00tN7DgbC6esHhEMyZg\\\\\\\/BbaxDZCp4whBq7PV90NPd2k42tQDEL3tXh9e8+WLGG4pcYokFWbqbnm5w5RL63O+UrUlVSNSXVmPbagBbJSCpJWLrFDEkyywrp7\\\\\\\/Z+yJW+uYL5eYi01lXWwBduRQjcK\\\\\\\/LCy7YiiTy1HW0ikZdXpZhGB0136mu+iW\\\\\\\/I0EQF4erKlyBId4fqBc1QgAUQ\\\\\\\/SRYh\\\\\\\/KZAQYfuDlvbPRcSYtxhVjCKx2J\\\\\\\/9kpgbDBEAZnTv7cHdLr3lwIyOtasH3boOTbRL3TrV9EcpTYtO71JedL9A\\\\\\\/ssD8HvdLUxRBrsuTIoAUOG4EDE9Ag9o7F8KjikoARl+xMFz8NnpmxF3\\\\\\\/i6oXbJwFub4lA9kFh8McxHKV6BR7TtZ7CoWGKUaStr5NFz8qIbPq+CiHTK5CGW4GhQyakXzED0CDkfRxmnwk\\\\\\\/\\\\\\\/arLJSuEZbSFyqg+WHxlcWe1x7sgwCGfZKu9WFUJJZnE1G+lQqwRtxDbPyxcQCI0G82tgYZM\\\\\\\/SFlCqghrjYxiGD+1figEB5rRHQ1861jOYtmjkQRSDZNew9UruO03+xYw0+EPPEFaxR640ONnSNX8fqNr3JptZd+B1X8uJJkeDdYKGYp06QPNMI3UxVTFJoWgxrRQ8AdvP0+gPNeMxf4s\\\\\\\/Z3Y9Jx2VcTT1TDh4xidrNlujlIaqwRrqrhflXify0SV5KDkagO5YgxzkC\\\\\\\/2lR7p1QyTmjqSPE7qnigfhOt87PLmA63xdSeLjAcESOvn3hIZ4\\\\\\\/K7Yv\\\\\\\/23fEwmuDRY+US30AHjPFgQbw8YTF\\\\\\\/O0tgB2HkBp+gP8YUiCC\\\\\\\/iQcgfAz\\\\\\\/wMnbEBuiaspLdlcyMscGeyV2jFYnjWds\\\\\\\/Rl\\\\\\\/Yf+RF0XhNbHg4ra1DP6ZB4sMCHjuyE7jnWM8zVRNIAUpTSx6qqEeg2XASFZzrLlmiC4ZSgeoVaG6lnFuXizuftY8zAWgKiPiNubABtEz2La3KuPjc1apRNsmZs3zLiwunGC4acRKTKfl+KNThbDteE8IJoVdl3PY+Bo1upvuLtBfGptdIfikxk5hyWsRmbMvURn10Xtvv9ig6utBMeQCqOTV6e5aehvPB6W2Jj0J37DfFylzub2\\\\\\\/A9ECGUuAi7Z028cYozN1mPnNr3TAoFZMghwx6wIZvolHnZTz\\\\\\\/WZLBpUKHdk3z99UnRjztHT\\\\\\\/w\\\\\\\/6331L9rA7rP4ByXyS7nDTchdC0LEsYCddgt7ZJZgOj\\\\\\\/10MsYKJJxfY5Yx8tEurf83RNolVZ4NRzG1WmQnP+Fcx9rIDvwcuwsnxOoE910xfSpfz\\\\\\\/HGObXMkmzc2wCszXgudzWxxGI2FYJYdt5TJjOkK1dtq8hqZla7CG3PaN6n+p58O2arK1WSam38Hsdt\\\\\\\/o7FTqJIqa3J+Izy4\\\\\\\/BwyeyCBas\\\\\\\/8k50N8p37qYLQivVNlKK9199PIpg9UZWdj7V\\\\\\\/QMiERO+OHcZ74K60k\\\\\\\/41vQRkTGowJUWaGtixKxYD5rrHxwvCsIhWsMEij+doir0HJ5cPCsjgIBEJDkMrjkqi+D9Ww5mcvbTlX2IhiqF\\\\\\\/qXCaxRfil+a1YD4Qv9vGRU+Mw9qse3gSmzq2W12+nyg+b9biWQAqZ1gdv\\\\\\\/HMKpUgicdawn1hFXS1yLYAmtjj3yPtS8NnGj8TckOUoWvkNxKGQJNQmghra7Bg4pQ3ybxTaiK2n5uyzmGTGjfVw5KgNy6NjlV4Kga22kAdbEdVZHY8ESkd2cxShe+yQS4m76bpbV8UzZfAT194EwDxjXLRK5okRIM5QJOC3vBNRosI35pLagHsl95hlgd6x9pHZIE7\\\\\\\/mxXNqkjtj5LI6khxiFeQDnGFJIgZJ4GWoxRBw63liVgBpZFy4k6AlMFH41EGM4eIeLW4iU7BQZMtSR2S1EUO1hBr0JGgLc5\\\\\\\/Gu70\\\\\\\/96RQgWhDeceObDjYn6DfuZKbMkJLQilGl1hDWo8W5gnjSc7HzkZFtR2V4Dz0XNbtB5JMgGi8MbvxvItJGZIU+5I63LBUySZvYgDHx+nJ7rYTPQEJVNH6uC4\\\\\\\/+czk\\\\\\\/J+2DzuYXqgDkcW4JPV2LuxjVCjBjg0ftOx72tXET6hJT2Qu010RlQS5ji+fyH10IJoQVvAk8AFwLpSC3RbJrwAgL0jKMzfbq8rA2L5AueoFEWpzEIdAJUvrY9n18tOvgk26B2owNEXrpb4ve\\\\\\\/1yuTPh0uvSgqTSlTEWDeJhnN47QPosv5HxxKb8MpterErqk3tmWxqP\\\\\\\/CACwvc1CkCoyyzUi3GDGr5Y\\\\\\\/7h6de8W2idgq2ySw67adRNTEw+DCTUAKSnkZjiKFHHtQbZ0sO7W5Pg3ZLLOakizCVWNvJN8qk7ilagP5K\\\\\\\/cUubd10ciSaeVYdfSNjDxjk7PlikIEsGJQCn5kewFvTroAJkooXFNh\\\\\\\/uSKcR98AvF7tHqwNL8TeHiW2imGKS8sXqC3PEIDdEG0\\\\\\\/UZkIw4wuaTzXANI8ueq32SGFXoa2XqdkvcE\\\\\\\/QSVDbyPMwZWbWC1LJ6ZnT+8\\\\\\\/YezPAdvUbtBoRid9NWv6xUAC091n9bIAexC\\\\\\\/Pj03lJ6XVRHr2gbucxnJ0odDDSs4MhKBOZnBXLmxx75a+T9xfo0dywX+tqqkbo\\\\\\\/PAsXxxxE1Rym2Yo4sWW1vyaueMeLbbkDDyeAa5bhZ59UftjAZhNr+z1xWR2NM0IvLi6WchkW14qNiTrouxxVJBDKCz87w1hAG17iPCmBcbuN8JLSqYnNdXi\\\\\\\/T6o85wU9EZmR5pIZ8ITggwwtOdwmHl\\\\\\\/NZxn4mL3V7rQrX7MXBvhkLLWsUmlvaSWMVv2+ErEnUTnT5XZ\\\\\\\/10TICgPBV5mFaIv7nGr+7zDXDycwcPwXtaz5hTbzVzo4ykFiiWJoss+R0InR\\\\\\\/a1oLib\\\\\\\/8N7d1Rslc1blaJDayxEpjXKOASdhnVQEaLLat2KsMzqDKHnkJaNawMiHWTdQbHMlfSaG6uDO7GBi5aYAWG\\\\\\\/Gk\\\\\\\/g3xD5a+Uibzydh2jfQuF6aaJHo793u9yarZh1hByQY6VV7BJFySvuOxm71u4NxQH0+yQo+e9Hrrm7\\\\\\\/KQ1LU\\\\\\\/XVLVxqCpoDq3ovrF5WUojGKcCtCZMRGBAXZomq5VreC7O6m3\\\\\\\/HGTG8Tj0+ixmIWlL9s15FfH4swPtF02kGD8nE0DYbSVI92a7Iw5xiQuRUv0Pbbex5aZAmyhowXO3JCbkSBY3t8jS1OjIj3MhafmeChhT3M36Ah3FSelm2yQ5clKiumBP3UdYUBT2jmgigUVCFbDC1hvx23WVlbpjMlHsFgHpsAZ0WBBjqshO73oFiI0h2OAeHTXuUUUsESgkRqVNqTM4o1s\\\\\\\/RQJnZBXld1qEJJYEZoN80FNaqeevVUNZXWBMg5md0jeooXtbShXikXyosT3\\\\\\\/WklpgZKRM+eknI3mnidcD1laR6H5YQL9zHppdKsp\\\\\\\/D4C7QArEXG3rVj8jyeumg3\\\\\\\/Fe2tDc2IVup2ZU60Od3q9kL1WFaGB837cV5oa0VkbMIa8NNI9+BAmNrId+RugKHk2qN\\\\\\\/avSYJQeB4uBZ1HiN3iJ3Vads\\\\\\\/FUPLhSjCb2IzSsZgaSafUquOF62lgs0Q1TR\\\\\\\/DlCSl3Y3v3Cbe7DreA76LCO3subT\\\\\\\/gNuz6p3tYi8DQlKH8pcA8jkbJoahAJFqPsQjDiPqjwwFfrvCHKS3b95xBwdU7XWclj0JntApybWNSZ5l\\\\\\\/mIohb+4yInApTewHzsc2juBgcJuOtTQT7jScQW3NyiOyqi79425hFtt\\\\\\\/xFwCqgs6g6OzypdTz3ZdOirGcbIB51czp6a41Qf1XY4rAei0\\\\\\\/y60dzOyY69BsZkiIilV6H7nSLEOcd9Led\\\\\\\/6drMw0YiSxDWITsLdXkF9EjQWfZMNtpFc1e2ClvOVS9e+ja1S\\\\\\\/Hw27kszeNPNvvMr6f3zEYt87+KZFXDjuUbXAugsFM4VYH6nRJh5jxFqg2U2jVo9SGN09RtziI6bJVbVtJq5dAuhueZKV1aD3EgHy5lP+9Zgjjs10ayhHgsN1DjEsGAxM96oKd8tyfHB4cFFPNo03i+C41iuug0KFq2iS9585mC2H0+vb\\\\\\\/F7woHbih4qQEGvG0nrR4POXiWRNqXnCV9\\\\\\\/Fj8qwaGH8WUZ\\\\\\\/91kFfwsQvf\\\\\\\/tpIvvVmTjopz9GqH6p6mbZhD20VAYXItQ2N3EKlRxNgruXR6l7hVlCLyPYffUX\\\\\\\/XkmOFT9Jc8ocehV1WlG3DO1c+IthhJrBcmOBE40Z7X1w3ClRd3YEZxG6ak2j3LYR43JusIUzyRqKaWMoqY6X7TX2aJHkrIONQE0KLSM\\\\\\\/glOQEw3wa0rYnpF3ialqSA2EPxR52zVuuTCClaOAXilkhwmnkosdhIyR9XpFW2SXSVSTsL+XqRWvV1BaWLjt3GwFTW9QjcMi1RcLN0B7eHug2CapJAfPAR8slauhI\\\\\\\/SMoEBR2ijKBaZf77IZlytsEeq0892EKbhRRGNE8ih\\\\\\\/1PWlDiAoteZPmcv9oUCaywuhNWx8NByJifDpmn3V+i2IGdtQr+HYXdp4ED3uj5U6uSN3QyeZYlsvGs0fXodncutwljbrEkDp4oKjk3VV\\\\\\\/cPbPZ5VpnJJQCptl3F7lpyzNpfiu6x5jRa3WMmtR85dVWWU0+hAn6ROmW58S2JegpOoBw8eDo\\\\\\\/lrQBjUNwdpG08bCwsbX4xBLCaAT0EfXjiaDhV0TvPvF\\\\\\\/WPwZ9K0C+2R6iyaJATaNdk6tMrvbs\\\\\\\/StzpYsNtKneb5e6VmyCJifF\\\\\\\/PNHIIGa6AnOmBurD6+EWlVP4Pb3TzK45Uj0kdZlGr6kRVE0BMeE8M0VV6I8zw8oHJEvtaXbLmNPMNu8Q4vchuI3r9jBQfYA6v+tgg8\\\\\\\/5dsHPTmuzaaLX3bxZ5vb17WSxae1J5uycwgOzercPse\\\\\\\/Xe9TGQcbxZBbVpsRBHMhpOSQqFvvLXdIT0neiiOms5qG3AjLfsFNpHRfjkNzImY6oOlGTjZpZ6utCKgPaayGvJdU1i8wYHq63bRGi+PPQHVk+TTlHV9ALqzM1CBpyEeIEEZEtJIJIu\\\\\\\/2LsFnFgAZ71p7QU7sp8KeaBuoBigzPWh9NDVykNgakOEcuImMp3WbcKfi2fsm5HQnPgfxrMYh9sk6pBb50DJbPLPJ0W9mlCF\\\\\\\/zh2wFKz1Em7jrNH0tpMLd6Fia38NZVZi5Itv2yireMaf0q+AbRUM9F8lgv9DCDeUovXjmfnmlJiW1yzwrvNmczB+Y3leu3oKZBa4USKMusxdxoRY7eqIVvbwBnvsKwXLgDGBbKNYsGHXlf+IvejhGJVHJcGKfPQUjjjidVbZcGTfISt\\\\\\\/ScRzUSiEqnvOh89v+63c7g+FNYF\\\\\\\/+sdEaEWuEYMgOdsKQP+3\\\\\\\/DPmkgOcE47f+m4HaP0F\\\\\\\/rs\\\\\\\/DRC7dZp1VWpnKKWp5Hgey79L9AdGqV2wYM9opCAe1ink8nshwGPB2rCi90dZTQKCGHbhIiTFh6zguv6HFCnFkUW1I1JLOUCamCxCIjv0Bb7UEU13xS3Zr1+lPH88hW4d1f1n5xdJgtenoUTnRp1xrqyRZvSQ4I20cwYZR2wSVv90Yn9Jij5jGHYpvRPlmuwo90dFuos1NeFC7\\\\\\\/h5qkfpt\\\\\\\/mtvBU0zcnhs4EsqMRVnGYLRe9r9+z3TBvJQNV0pQ77ZbiAgNowoJWH0Y2SWgF9ozNtiMH0ewhKeZ4MLBWfbDN5J1USwAShZd1dkbQK1CWsIdbKSlT4MB4zWUAJFpM2\\\\\\\/YKx84hzJ47qrlL5jHyLeJ5ZETjMyikGCHJ4FsalQL5gYzVldKJuHvwxK32353yXmyZtC7Exnmz2B6jW3Gw4T9nISvQc\\\\\\\/pAIblTa5wDx5yfqG8LDKPm2EK2qkSN1i9yokdfvKCTFOl8dzyd8xecrRJ1Sm2skv1NPtpPLN0ieWE0UN9yEGN0rIihLbYiBv8Doi0u5JQiRsIqz6R5uTpcma5nJAZBP8Pab6RGFjI6DvKVwsbQPoRfkoH01NrYcnecgO0I8XU9m28\\\\\\\/02bZjYUuEtCrJ9GPy5NJzGudS4tOsCJC\\\\\\\/HKLbl9FCWM\\\\\\\/UP4Fn4JVRuRlovb\\\\\\\/wug5QWXKJtV6BNIadJ4M0u1+GQGHYia\\\\\\\/QfUu771206NY23VElhCbSpvHxXzdCg4BL1fAXqVCwyMe6wRsFN\\\\\\\/kxH0SJwwUKHm5RLI311j9me8PVq5o\\\\\\\/dmrqQnNRHZq50GiaK6S4L7o37CAXMY9MC5azhebj0Vk+mGppGjq82\\\\\\\/8Sut82KBLwIx0s9G2ruzZ911hMvrD68EQNk9SpNheMSGP6OqKJBjuh2NwuVWDBpDQD2IrANZuI81oZ6wyL+l7JWnVK1cjh\\\\\\\/SzUUrIQwuk3CtPJLbdj9zx4+trxyJphL0i6ftE6u4XnmTay2Mwgd5\\\\\\\/z3GLpNsbB+aw5rPRx91ZujqysMVjx3Q7CQ25Lnq4rRyZOu6UNn3W7cl9F69hKaULDfv+f7SHV+oomIJbCJTYBxmsItDgEYe5f9IqGHzMNImFQhhR4M8XgNvuFhIHoAn\\\\\\\/u62bm9rQ9pe2QWmzK6bhzprF0pStF+XPxwga7weInIfs1IrhqJtMKjs4OWiS7gseSrcIwaMB9n2VuuPcnkELKUjxQAf6ZvSM8r0whV6zeb3SSjGtbHFpXFNk+xmwgZZvA8PmLRQoJRmAawFA8alTcnZkIe32s3ghM1OQyqKcfsgConPIRK+bMlx3VYb9Ir9vfulArvslJg5BRUX6\\\\\\\/ZtUVNmyBhT4aaKvz19curYc9HCVoSdUt4zQiZ7Yb2ulcSDpBtLW1y+eXIyaTBjT6EkUtz2kJ\\\\\\\/F7s\\\\\\\/v8BA14UZlMv9blHHIGbRGLuXPfdigWkQ3VpynhBZQq37xtbLm5s8rvPi8ruqnzqAVE4RpHDzcsFf2jv7vtgi0FkrqaocuWL\\\\\\\/HMsNeXRCMFkmuptWBWjy6koEj94YiLKfiitcLpGMiKP0WOSamJ4rtfE926\\\\\\\/oNDS2qvwWpPgLJVM8d27UnuwmNrEWbUFkEI2Crh6T+thNnuDJmF\\\\\\\/VLdxW\\\\\\\/HRYET7G82FIb7HHwEVX1YP50TofPPxmydFaqU\\\\\\\/OfPxxN0exby65z8QehFhamWGK5hOJQTm9Kf\\\\\\\/cNqCfD8NE1idiVtVLtuiRy74ycBMGvpuQ9K9rul0DjcEoRo0XR0yNBSEfIZbGOULleRxWC3BhkyRVUT76SHTYDRt\\\\\\\/l1AnqVB\\\\\\\/EUsPoccfYYEzrD+4Fgb7UWZe2WIJ3naQjTA2JWU3IMd4qJfYHsun0qo81i5tF+ZvJ2QxDRDgx\\\\\\\/vOzebFXI4T8UpuwHyszlAs7OKyf35tr7KiVINxt6Jntfyzw1khCHvNunaC\\\\\\\/bLIdvnl4Qu6aFsl55kof79BeypC8ykskym2ntHJ4mSa31vxYECgVEmBNGkJWQ7eQnUGu2oDJQD204lKUOPtJx70T1TlJPxTO2qNQ2i+a\\\\\\\/0VmmeNe4h0xfkaH+EFZ7lQJWj+MbydNIoPzqrtQ7dDPXIiibJ1hNQkIjhqRELD\\\\\\\/6ljMVwKW+1Y32aqnTq3wMETUYhLUiBuAj4BNyWwPoySiwgBrWMqJGQwcfcfHalkf+o\\\\\\\/m2RNXJfBDdDHkQWtWl5vDjG4PPayujGMNNNQCDkK8q\\\\\\\/p\\\\\\\/S5DibjTXu90VYsUAc6NZQ1H2n6vafu3C3Un9nkF2in6e+rVCW0ZmJG6ItOL7B\\\\\\\/1ZGL+mzxQFjEYGZxyYGC+sULYc9uovZeGd5NYuJEGqD22zPIrx4QOEqy8PGDFwSF2g4qxUP+alBcMvXi24bUkqTftpvzEX0319rna2jLzw1gl4p\\\\\\\/kpKqEOURHXoPR2z5BY2MBwlnVKwu9j6d9A6Jwwpk6tjJsxCWurIHXyh6y5yiWbVm7RN7iK0CGR8ZCfAJgrLNbhZ7YUFA74eJEYKz0fEG5L8vl0dmPZnpnqE8I3Bc40Pq41vaPZwMpMhUxgbr2BOHAXYH4GVhMMsxA8eimYQSVMv8J9OdAyYx1WbLDuU7jdmJ23klOX2gNUKWblKREg1L5kfys\\\\\\\/TZvtL\\\\\\\/980Ia8mqsLxb9CTLqYPHWSHtPshyqhBKaTz8DLJzP56v9xNQ8qqQi+HhtjhYZVC5orPc3yLF9c6EvlI+MxRTvVoLwEOXLQbIAOahwgqP60iZNxbWx3tiIJ7XlwyXdSgIZH9CP7dAXLFogBmgjR4dpaTXEtYyUkVnIzJ57VSrPW\\\\\\\/fPh8mxp9dZUVdspL1S0I+JavhsjDAoZyoPVogR+EhOBS2DhGvz6PryDXJANhhJGKSvaRfuVmjx8ym2TpzB0rIAZU\\\\\\\/8tjYoPi7oVE50NwhA0kQTV6SHAR2D9qpLAj\\\\\\\/bohXWu4yKeosP7SvPxYz0o2NTcVVJekPCtuiMpkr32hKjG6M8XqaKnxxpoU6Pt2URFxy3cJWWYawJkmVyOB6VSB+ufTXA0jtLlHuTiSWtghbS2NLyw2dDImUxwvaarKvoAC+7NmciAZPbuU2cjL5RyLUSXh8UKlBrg7pxf\\\\\\\/iGQP22PKRFDFoLKN6Y81p6nrSf4WxPABQ5JA9laeIJ5uktFK4jyIap5ndAqZ7KgePxIgAIFCTrNVWPx+eNdPwXm6qhUx\\\\\\\/AD0Yds5UF6Sdvm8Pg+pIk\\\\\\\/+7jVfqovWEg+mFRgoTRAfsDV4cLkfFM0ymSq94Ht1\\\\\\\/g47B8LB39crPHhavM9NTMPZFcq2iZhSm0KBG8L8HV8zvMaUl3ZBKy+R7xBEdXW4tMcHDj53SJm4JiDwW6BY6aegsS2z35Bn3iJb6vH6p+72YN2zuIBWk+O1fHuKXgbgL9Hu\\\\\\\/EGmz8frqseUGsvSlPHpxF+FYaYz7+S91JT5SKGAX8Nqh+eSdoo2RRtcZlcCBa9PFhqEvMh2ckbyCvYt7k9LoC2IjkNUQoLkYlGTBsZoCXAyymAj9Lrk5mZRsfOV9kas86rJ1NnaRPQoFeLSxl9kWOr259svAeqn9UtIx+rTc4kLT6WuO4kJIE+TySrfy9dQHLU1yUO0G9GZ6XwsvlUF\\\\\\\/q\\\\\\\/b6O\\\\\\\/KWVNT7bT\\\\\\\/tnqDmgyYWwTiWAEPGGllrgzZ1LMnQaYWrwch+94f2CFZsqSjCXWYVJ38C\\\\\\\/HMG+5t49eTBNDcw3E+Ba5Ap9a6ColUYTsFGboSY6DBBgVUKNKhgg1Pbpa8UO3VEEylTOMzBHpEi3o7EXODX8VOEF9Za\\\\\\\/vDj\\\\\\\/bTLVJyESK+VK4522imi99MZmFaiV85X5u42fKlZ7Z\\\\\\\/e7QSh6RQJGGUre9iCDvR50aGxzskW1jQlEEXYy6KN61PWGeYHAQxpQof4b0PDj4MrejzSsWSRZvL9dsX88D1wdH0oKc6cRF0+RkSq2hBDoMmE5Aa678JDs8bPWBCrLHBjTCF0Boq36r6Dto6sAMETeYgp+XtfdtSraOGZIUXBOgDp36oP8dfh9ZZn2HykRKCtca6Wu7+j6TAEl4U+Xid+g2503Kqg9FaoXcpy7B2KQFc3kGE8wbp2EJQ3ss6oLv72zQMogUUEjC13dMhcCmyfapqBSIPm29zcpPbJ4rI05PqCwNw1wcw5eipzR8Jz5FtbDdhxoRfVeIu+YpuaNbNQQRwoPl5Cx0mTAOCKILSlnjGp9yFiv+p+RyHJRptY2kjbd7i8Gg2LcTbprVYkWUBn2cZ+q6\\\\\\\/xetYDBKpIVLFrb+\\\\\\\/gpOFwyh+Tf0i70bsTedg9idUUgzweNPk0G9aupx6oYnR+ZNgYHmYUY93zccAz\\\\\\\/V0x31mAfoEeYMn571EMC352\\\\\\\/AdfCdzlxdPBL2QDafiQFHC6Hpk4xDlpALhfrtCbhPlWvfp7SL6yWmgOwH9glGUYzyaIpgJkembF2rW7pCWzzb\\\\\\\/EuSVJJ+18uphJqf6hzvumm2jPrppFv01DHKBlMsIvApbp++tt5cqk9Hc4KFUmLDUbdarHY1HkwUoiP\\\\\\\/kUA4Rs3kmVJUjmeBwghm6XfAGxWKMomYmje38piN9FPOvpPYvmQnDAFlqcHrzcgnXsuz+CrtHlCQoZnz14LTXvLzSe6lvzCxfb9JJtwdruZ+YB4mJXE8XNgDs\\\\\\\/FU87nxqiywkIgCGkqspS9\\\\\\\/1ks0gneIAHZP22RV5svV02VAj1SLivYMcIeoBqVewHvnAdAo3AK63nIQORAsua32f2Mu8\\\\\\\/4AOcM2CcC9cIR06gPMelftsD9pEas1STT9WqlryHbsmdqQ6oQDdUxJIT0wxuond9MIiNzuNW52+uhMjgtNeXKGYnotzRQWa7KgtMcdZVqTTqFNWNViavD\\\\\\\/M7GsaOwldfuHrZrDHe4Tuf2EYZhVXBqFKTS9qNSBOn+bpOKhEVOGpDzLAYD5Co0uC62cJfSjWLzBWY8LAqqWH0RaDZQ5Pb8WjqNYyIQSZcyBlv\\\\\\\/m+0kq+ZE+VyJnkoRGHkYFoO\\\\\\\/INSODogiXXtQW2y2DB6uzZPY8zOTRbY97y0A00UZTFgMvrwficKcL6IprBKQMqyQuZbaoNGmJEq1jJSlC9\\\\\\\/hd8dDRhTw5fGql4rusVMXtg\\\\\\\/r7CryEkUU2A65esQBpYnOmZBbjFl4aoAMCRKax25YKsTigp15rqYgJPMuigiIcRsR5\\\\\\\/pG32XjyUkYWCrakNR1Pdpv\\\\\\\/Gt03R7UBPL6IDvPeyB6db6Vo29W1TXQZIhE0fa1Ld6KshQMx8pbibKmWawW3yQoc0G8RkMhKfWK+X2REr6nT4QiyYRttIm2ZEdjhVvicDsOOm5ajmDPqXcpMwYxmDyACTfGJK6RN9KURjYf6V2Mt3KAw2U7\\\\\\\/jRmI4DPZ9jFzVXTe6ibhzTSVz3gisFaXTc8JIWfVP0VOXW4lkHjffLASnRGU2iAdi20IV0x3yMIERkf\\\\\\\/sd7cuX6VipqXKY8QSoxgp179eXbtfmAkUZ5W8EAzq4+beGF\\\\\\\/fsynApzHTOQHt\\\\\\\/pkJ6c4z+3G\\\\\\\/y5pbJEtzsoti9U1+WRtnL0hRiWZAgbJpNyXAAgXBx2B6oHZlw6uLyBalweDhiv0wrp5Y4GIIODr9w7BYxSDt\\\\\\\/XSRK8a\\\\\\\/+Ifo5\\\\\\\/1c\\\\\\\/wVGeQcjQ7KL\\\\\\\/LffZPwoD+Lm4J21l4PkLJH8BQWFoMINg8YuhyLTq0umE8pvc9MNRfIdi9xE9eC+QNggioDLfGHOLR30kXvglQn34LMupaY0COOQyN49oa369BVc4oxa7gxHvmI0LL7ycYn1TAJL+PVuIXvZo8ttU8DcFADkwW4xVd1g\\\\\\\/DP9w+r9v6orNwKnyuGuLQZv5pkpCfpcI8DqX+OG1UB6oAesypLN1mrVrUCiCQ8xO7Nm76E1oZGcad80a5EQrZFwiw582k\\\\\\\/Bpzb6Wrm94kC1LU\\\\\\\/+xzMaehwZRK8zi8loSAnwl7mxcpTa3e0nScCy5eRdIOj+dEfaLOBWsXzIjtNVZtnoJfOnn42HmvIQIdu0ERW6ZyFYhMcqnN9dA3WiNCse\\\\\\\/XFWVID6y4bRjn7tElLXhgXq8k7a8n3nSYTcQDv3PhXmBut5QDtC+uQvAPMP8caWauVgwiPojqiDPenASsMnOZXyRvAzXSDo+v6gBBXfE8Ur1yYejYPMAb2dO6s\\\\\\\/b3HFOEpx0dvmxXM5x6OsiW4uJisfPOitmQWVvjiHIYk\\\\\\\/2nQYYkbwcGBLwHoa4\\\\\\\/HjZ3LHUBWoSnlKGiVqMj\\\\\\\/J\\\\\\\/AVsuj9nurbUsHfGRCMRDHfDfc2DB9SnO4pMKgDWHn\\\\\\\/wZNG2xeVWJin\\\\\\\/dgOQGUvk1b88YLTJoBsbnAAuhJPeSe32xpoue7yHcGNL\\\\\\\/tLi2sujm7T79OKyv0tWQdPdGAmq2NrbmtnVKW5wwgQsnMKQiWvFNlho2f+Cw7aki3WYcWbzQMnUsvX\\\\\\\/\\\\\\\/S14SHOMoPK4arsco5POyVjMc7MPovwmvBPlPfWazp3z21Zg+pFlJ\\\\\\\/GR2XaNz3j21+Ex+z00HhRMQyk3pPspCYypWwWtg5ix30qIu1EPxkWYDvBuQqLt0lmvp2Qq\\\\\\\/QROme0sHNk0H+6KXt0CmrJIsnV6jtVQYKe6lKwaYDDTVYp9VMkSuycvuL8y4AvIKA1jtTgDDM3294p9XysztSO5UMAaloeDDztwladNlBwWfn7m018tXPF0I8Ltvp4vlC5zxDB7MafQBTiWDLfBVo64ZK7QV\\\\\\\/t1pwSSXVmJgw7Vxb42vmTMOwugEWSRYhZfH55xBSi4ARkuXdvSprb6BybgzfQXOJB36fkNj1s5ksgH4fCrR01tPOnVyhk8b0zuRflx8kRrKKbBBMTB8YmOUemOlcRDwwIea5EMbZCr3MgNSoB07LSrzt1ul5+ho5c\\\\\\\/c4SobOV0PxhAiazbIo4\\\\\\\/FgMQiC6U1noMQVKTh1QgOvFzPnxjkjZ5s3dGuetaFUfCKcmGSRZ8TkOYlrdAWuWKmms6jo3xZ06KNMuVjrD0QngF+mnA+TeP3Iz3EZyjz0PyNVBLt3NRGOTJvNkIN7o9d2yjdCHuZtlVhRNxZkYym1lgTZlMFx2vFrEBymBYTqX9QdxJZU3QiIf7X1EIXK+sm4FHfQAifeAwfYh1X5o8ApqULwjW0gbKcmaBYA0o1y0VdoDMV9eklSxWyRopCTu+HmWKt\\\\\\\/6cMFcuO8DX6k7SY1tejyY8ik9jMV\\\\\\\/qgUwyw8NFO\\\\\\\/1EG2Z3D3p8+XPl+pu6kJebvWo\\\\\\\/AzMJsEljH436h623LwM1Dz2Z\\\\\\\/MlR01TSzswpjONy7Pa\\\\\\\/hZS6U0RrEaYn6WsWcmpwQoOOx81UdZgdzcSAwh+mqaF5uC5INgT\\\\\\\/aSfjjQL7dYOdXyoxVPd0aY83EKCBYb8fg\\\\\\\/Y2M70U2gMRQUGIuRrDL1Pef1jFLaPZtq1cSDkM9jqZSlofY6kxwEHK\\\\\\\/5IGqtRON9I3eOkFFRhvqU5bcf4jZ70sgwHKnamXmWQ32qw9hwPsDmbFOgQwBSR\\\\\\\/dMVC8dClExfP1+pVYwJVhxXR8y9foV4f3JCZUGQCYhR0YsqmWsZwxc\\\\\\\/kJlNEUGk2YJcrkRfhsdG\\\\\\\/WycOPfGvlu9CIEPEVgvqzoOffOGFhlTj\\\\\\\/aCVxuCjUL3LzwuTN32o+kJaEPKSj5+phoZuJqJIY63DKS39kxRnXJ08BzSYfki8PxFq5yFAJjCOXLtSPdssrg568153Id9Bszpztat3spm5CBb4ibpJ9Llp5hTjmIm1uvXI8NuX1GQe+WdAH9hF2U20+ZwsJIQyS6XkqfO+toCfcwfvaKtELc6B1uUQDsylcQBuiEuJA6hC02e+Uq8GLJSu6O0OfyI1JeR4AXn1\\\\\\\/ZKrLLaoKafm51aqx7ucNxzYqlHDC4HD8UUMCVknC+joAVb2PHENoo8\\\\\\\/0Cov3ZhU83hyMtsKkQFOvZDCOo9akVlK83k+YuRxdNNo+b6C5J2LLOw6YqiyL9p0fDGGi\\\\\\\/U1JGgCZaZC6mLumz9AXVkAwXeO5WB9G6OaHe6J7CEWl3vWOy6jTXMKlVUHZR08jlzvlZtrd+P7FpHYYzSrwLnbWi9ZR8J96PJdh07cQoIUiSJcG7yswVcF\\\\\\\/AsB5suZFSowzkKA3g\\\\\\\/OS3vNFeGkKStCMslPcwc7EW1mY38LaOfQJAAfZtPqp\\\\\\\/zcDwh3p4l8CKfqB4bhYvMoRmTx5l72yf13ymkuezaFQKkIflxkCt8tun5ekGSzTBSvjpHn3cbez5KHW6hovmu1zJemXqspTuimFXXyBKOAtQBiD4Yn8JGEGKLExRTqjrjpq4TKUxFLBSGze7+gNgQtd9\\\\\\\/2rFx9dS1Co\\\\\\\/xUBkWJunMpP0wWNlb8JhijHuxE3iR5se8A+nDFvV\\\\\\\/uzSM5WQEpYHDFAi2ZaO85yclvpTa40tM5bNEMcwVFhEhDBI3Llypq1s6IuVj85PmpFw5zwRLqn\\\\\\\/aycdvNIzRmMP\\\\\\\/DvTlwjxCyF\\\\\\\/FmLEYJGXJRdvc2DrKomts2vyCe\\\\\\\/teL4ZjaQR3h\\\\\\\/TX68ZA+Lm\\\\\\\/NnzIF+Fl6uS3\\\\\\\/9\\\\\\\/xbsxOaEv30cnoHz+bD6HXs\\\\\\\/Lp9H89NsIuRq66vq3B3r4UJfK1TEXQVs0va26J8k1uz\\\\\\\/AZ2EsbtUPZB7XipOyRcXEDenQTdmULU3x9K20uQLGF54\\\\\\\/9tyOJmP3ntvbbIx6r+iAB8mgNHsgxRJwd\\\\\\\/2ja5Zc9OA97M6kx+STYNq2kC\\\\\\\/3KXKcxbBvOXlvg+EWBd0b4vJp0JL3XYlueL9ONExupm42n6xDmA+YxeBq1U4tel87m19isBQIK11z4tYhFJzF6Nv43QXrfky2Oqyc3tOAAVZ\\\\\\\/N+qUy01egPszlN0Ifc6SisaM7ZlCmCrinCcpOXW4aCxXn81swBNnYTS7sLB+9J0rzxKsMeLERAOHCxLNPaI9by4zIx6g3iCjhZVP9IXJoMw8E6lOejpTVN38rM1qT97kkMlJ0W3RpnqCNPa6oGVvpmN9IKfuKt\\\\\\\/5W4uTQM2KWD9GYso9jGsBlJA\\\\\\\/a6eu43qg0VkOyekU7oF\\\\\\\/9943GMH2Wq76hllyJJY1UyWqWgRztIhv3NSJqKrnAddiTJVyMrU6crn8ibxr8y2Kf5\\\\\\\/PC\\\\\\\/hwgar2PCDF4UgJfl+BxQIZUZvOtEPilcqM5ADf7wgfmiGX987IWeTW3Ux7Ymkeh0gXrqWTnP200lVeP0xwpMIgS175uh9zWOa08mPOwsa0U70h52\\\\\\\/IG4SkEyfc4bgzn2gyKZ47MucfJ\\\\\\\/Asbggl1Sb8fWJZUdWxd3w1FmSb5JocN3zewt8F6Zln1teVOliQqCna1\\\\\\\/mImMa24elDFoR\\\\\\\/W23Y3JLxlOXcSVg0pGFIquSXfvbA+Zulc0qoblvt2pcum+MTiypimZ+bjvauxQFxne2q5DCsnwQZzAS3Yw2jbYWNUN66y\\\\\\\/tIOSiUHPRIPJquEu3GZ9H5ULszbIYCSInq7wp8d58\\\\\\\/7Nbpwf53cygXdttOdeq9AJYI27w15qTIFd1YG1rEBi8pe4uPyP9pQ\\\\\\\/mp+uDMAX7SNuOJyWc8FVXNKDuTh7Sm11I7LhCJCTxZQCPWllrrqixQmt0LO2cYPxAaP3eJeCglesHxaSDgh9o4iSm6Ou9MKleCt7aOf0sudTjrLAH41o7GK4tVIOqjgw2q1EybA4HPnjPwewcM8mZEfUOQ9HSxU6H2crs7kjgNLmhErkEC2Ze2LgDMSE\\\\\\\/DzcESULIfU1gC\\\\\\\/xCRg5b9+C5127qtVn8y2tSgxWFhDUn9+Th8QvuHx4k0qFcTtNTh\\\\\\\/YaJ9BG8fU5ufumdDvUjyuPR0bAnpICilA1uy392ZksYNr05ZyTkn1ItFNfheDl7PsmEWr+4c8qPetanQznK+MbAcmDyLwk4MKPLho5dbUy5z7b+tAPJavSMsNZOT7UIm7eAgZvan7Hn0oxkB7HfoYE7EappHlvaqVM0HLdZNsrtdwkKBKuLdeAAdbJgvDzqZqy0ZBOI+Y9yur912bgEc0HJPvxbfvRzBX0ZOwQhtboBDkkficQia0P7qnxowolx4nixR2FGIqWmqc4HrT+FTpSI5eLiCNOvrYn3+oGc3Shkm3p1id298s0LB+IESKZpkpB6Xa4tGd8EvRNc2OdzZFMWZRB7bDXcJnLWrIHzcR4teWrHFQ1lLRqMa1m0ONUSWHxfKeHGoyj9dKnkXOWwXRIoGKAHk\\\\\\\/sTkq0E1CxVP1ZvlbcWSv5HGV9N6FqsYb5aDjUSfBRnd27i3fcxXEBZrG0MqVywWxP8NXtkxoOJTPFmaAy5XXA\\\\\\\/Qg+cEV5dDJJsmaq0k6dSU9De7hZqfML3FSdCYjOTQTDC5g\\\\\\\/gHIVK82+sLyjMBkFBO7JPmdkt1G7vS0HnLWZtAbhYKa+hHuVqS2bTaR6VFy2wd5Qa6ktRDawuK4fiAG5OwT\\\\\\\/WwVKaFGuJcq1EA87On4TMRWGqkYX+nyEn42hxSN43E2nnyLnAh3bJBS9BIOABmAYRMj1m22nDTw4gR2AUPapQiwubP133dtPZff7f2f8vibt2+sHTIp4eUjGKl3t7ZGBTsMiWpNceDod1JQ2t7v2JpwOhB0FjTpb+II0Khx4P7ERVzWuP1pEUkiyt6zS01MFVEwHvAFUh2ARr0ljDwbmXhh1cei1D320d0jhljWJdzAqH7QWb9INz3XoFLurJDJ\\\\\\\/rj6a6Ms+YF8ukIhCZK6KExFfB9LafvEJnkoJfEO7Fwl0QIS8xcSAyHIOIWXxFsDZhR0vOc00RRSRIFZEpcs4Y5JTcDCn1IMwvdFNh0w8VInGvJLeFfbwfdK09+0v3Zw8nlir3R0tE6lm9BP1gNiAHYa0giBmgPx1EiCqP0vyoQQ3\\\\\\\/P+E55GqBMgs\\\\\\\/5u3Qg\\\\\\\/lZUZGas8Nrm1auobiqGRk0KhnG5CrH2K2hsokCaiTHHI+yXaIw\\\\\\\/PZ3AEQ+fpninXMPEDiRfT42y2nvei6NS5bArhSLQRHdwH5JUkQcYeJ0qNyeY+aErZPh7dYSEd0B5XjAfsBgJQ8A8f0GcCUvd9ljvXStDvl7ZZZZ9B3XJBLECKyjJgrDuQ5NDV2EklFAMesS9bvShba53xG+eAozR2EiJA4cg2wLlPN3MMCPhxhRvJzG15Nyu8ild88OiwNuEC5IZhI2L0ew4ELCjjkANtyzM3kflwK7zbpJhO5leeFAas6b9B+LIrkrki4sg56lAsX9uJRoAko8sw9o+0fND62c7x4mokdPMQXXRBSgw2Z5tOGxE69S3YDJUmStQb3Vy5eUUEvIyaFuUYtMqqLey2evxF2EbfxDjU9h6zdFA6pTzj9DTjJ0fKfqN4ysL0YXGiCeZiyvjYSKQEo+2glsWQZdH78HOXGed22FqAaqjJZPsWgSIyl08xft\\\\\\\/Lx+VRrtB89q5GE3ywXTIm7SSk4Cws7VhnrU61RG5YMFfPm\\\\\\\/ZGCXVFvBhsF1GItvkmtgKolMuEomEFNXRdR\\\\\\\/9T8DGo9q+gpAA2oC2aShHRbbFnk0TaLuwS\\\\\\\/KCGw2Fn7tKA3CzVQs38\\\\\\\/r3XIHCjdMCGOGpk+rtbq7pD21fzNuo0GOROH5++b59eu16tuEjyJh7vbSuLGaV4jb+onNc4RUmsGXmihtdqCTV37cMImBAfFzY6MVqlb3RF2vIpqGsfqfQw4LHZ0jN5M+H1UhcQHxHNIME8rizp5gIE4XGOYl4u27VP515lBfhRbXdvHpkwg9sZOTqgPYksRkYKqlY0zLHU8WF2zzK0b71WEm5bvuJMS\\\\\\\/o3AeCjfOecczEplVpsxu2reKI2i3a1A0zdp9xFejsUphQV4WxgSYf20C6ik9TSFyBF+338s\\\\\\\/4lBExw5tttp18Wckv9BF4sB0jKOp8J\\\\\\\/vpGxeFnZaTOwq1F9b+pmU+RFlfa+FOGQDYHm4jRu5fm+qCtiDA3h206FlqZuxQLGcSrHB\\\\\\\/t\\\\\\\/NJO6WwDlPg\\\\\\\/jTgE3poJQ0scD6+BvaD7jQ6UxeVxYuj2dvxyM933QK0l3\\\\\\\/PASTVbihKuGXtaObXPXi8lwDmZvW1RJCvYud+W6U4Z9ycQMxPgybLNNDPGnPjmfCb2NBzbMzvfnSrhOxQkZrdtjSW1eBslYi3lawTPh3Bk6Xaf0nJy9T69ttXoFmtdeLQO7Jt\\\\\\\/LQ+NwDq78XvRDnVH8TW3SmCMsKcXHBl\\\\\\\/eECov6St8w648AOhniwPeQdh8hnODJPyjKEt6y44JabUXD8IsCJpht\\\\\\\/HHWq+SrsE2GVTW7+ZcuAKNWIIWiYSkJ1YKZcqDDKJPSpfFxYdCS+qs6s8aqQEkZnVbP6YkOFOpOC1zPZNp1m2hM6GmWbkU1gG\\\\\\\/JpLA22pYAPZiG+tBM4lbeSZU7lCoMWcUWtje3KPM6xFTilvGEhWz6llALlOCE4KDXNuHoC66jxAyG+JDH7\\\\\\\/UWu8t8++ReinAJRFpy3tkmlLlSm6fVKaJQ1NfbY+ORp+Jfaithcg9bgGiR+zcF3RsgVvMKMQvBU5M9ncfqL5LQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022pn4Sm\\\\\\\/i6dYrvwOq0nNIMJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002225542a3f8d35b10035fc3d50808fcc2a0539cdfed46d149e3ba7b9d55d99231e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KmXFDmat9yPrNBSBXCrZvPXVwzP3KNXnnYAChuXLWkU2qWz7rmvHgtG7\\\\\\\/LYiSHk8qKIHxuVPNBnTllfbUghT+Hb0Vz0cZxRxfT+1sYLCvgh3A2d98QOmzfjdaRJaTZzUIj84bCzMkX4HF3GKNkesWt\\\\\\\/CBDXA4NNAUgaKIwVPCKO+tFfZjS88A58Az6NMD9GNWw3fQyaka1S1dh6SbnbUxJ3Qs9oo8SgeTrAcwcVAekTrs0X3+9taj04M2r++HQJ8izt2SNmKwpOt\\\\\\\/A03apzsevIaLIQHdCsg0AuUIfMr\\\\\\\/f\\\\\\\/OKZ0YT5YYKIKZFEPO8ilxcfQq5UsJuajbBlPyQso7+wljPRbQBL1N7t7aP0YhFE94yQCCBbaake92TWn0Zq32mPShFZTznD1z67Zfd6vs6h+r4n0M48ihs\\\\\\\/1lrP7OTWsz\\\\\\\/73TBD44lj3L1yI1v7dOiCiubFAPw3G78cdl4Wu\\\\\\\/oS0OT8EBovGXg+zeW0cb9jKaJpIh4M6YZkMdNKPQ6RFQX4FVaLvQdV3BCqxD71DdwJwPuSHHGiYc+y3Swe2XgIrki3nOQwXMJaaLlBJ3j9yfL1ArF2HKx1ayruEjGV30IwdXa9o7SwubKKZbEzPbJetafRNkhoMGEMGR9r2GJEhGF15S8HvQdo8u68r27dmLZneKlY6XRFatuusJaQ\\\\\\\/VufTwf0LJ38\\\\\\\/KDyEfgD1SdGaiAaJwIuvJivIK\\\\\\\/UUKOfcUuniRastmGMifqomu5ButHaP4hHjSkA6ZG82FPoIrBqgJRKRzyLWdQVpL4\\\\\\\/jwNVldjziRS7BRuVYT7tMBKXApMJtCMe+XO07HhRfCKBEE+RSX9aC8zh6lpKq96KRsC454WciQH49JGYccCoiDIdHSzRIZRtiAePNEPUyJtzpR2nlJUybmk2exOrV0dksZ4Pq0QxDBi7ji290tOdFqts\\\\\\\/uCmMJYuoPhbam13cnLR2oaTiKDgAS5WshQMlktQ5F6O1kBvupD3lHNyqNKGYwZb3vZqG5bHAw5C1lBHZTIVkQKr2OxhKeQlPqHqJKRHW4gYX1plDArs30FaG7G\\\\\\\/h9frlNYPhWI4HMThCFdn2b3D7552QRfG+0c3BTGe+Zeq2QPhKvlVHrwQicziLMSWNRTjsKVNRUATg8PMtJuE3DvKHBUspzRkXL\\\\\\\/6pJIZ9O\\\\\\\/JC7QUhYon5VrwEr6X8Fb2jDpuUP9sZJNwaR1yb9VRTIcV62pJ+CBOqzolkLw0\\\\\\\/Jh7V2\\\\\\\/lFez9lPWutGlq3Di\\\\\\\/e9XpuSIPN7XyzI7fE1IoGd5QQ5dPwHzFiEWPdgEjJFwv8k5N3i0kq0DmIycgdbipqR0YZ0uWQrpkKjrLgzOma5yA+TXjmWdvtKo0wGKqBDM0hEy6wbgIzQE0UpgSjrzWIbqUoy\\\\\\\/IIPE5ELnGAbGVdpZBjT+hLMzJh6QeUcuKrZY77CABa1v8D1QDMzljCFwbaN9RREkfbV9bmbGvFdWAvKF3HuuoEIJkzhphK7rCYKx3v9Es1EW7Dx+UoR3AtQV102ncJGCfmkZRXusaVO9r9HuiZ6uWP+T4MOqWaNF8F\\\\\\\/LC0Q1CSj3J2RMGoJCM2qalnd76jsUIpzeZW5GpIZWGVSvS+a1Po8JSlc7AbCRqmDTwD2qxWz1Qrdl6+y9H3xGGknZ\\\\\\\/xteZidErNdORuM0\\\\\\\/t1tPrfdG7erb9BmTF3UAiOr0uCLRjrSaEdu54S2kvzW8qJUM2puQoZWrbvuBMHiYUbNQJ0G9GxiW7QGPlM3HOjpAWB8j6BsccdtXmkNvOvRnYlLejqNZGMdky3DOzh3tl6mlC\\\\\\\/eFyWOPGw0np6gZn4K0fYyS8RwvLq4ZZYpJDI4YvsR5KPkVG\\\\\\\/QtdtvDV1odpPIPyJXDsBNA9BSt8v1imdWW4jP0pipZ2EhEcTM\\\\\\\/OuhW3wBwFLkD2+bo1bXV6hMdONT5B\\\\\\\/XafUekDzPsSnjXxNukp7ZrqKcvoDsTianMNQf84+1a9xJ+gIv9tKCcVlcug94E5vRPbp+UhD3cZx99tFi9tYPMKv2w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022oqIT\\\\\\\/c65O3iucx5VWXeDEQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225a5b345fc9a2bbe7669532579bbe49cf91ced7275960de7549b4edb32bc859a3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493256 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1851" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wSQfDcu8ON-1xiT8BhQNOtjFwcL_y55irDa8G5iCAPY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1851" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GG7d9pHd9KATj0jJIhxaqT_ABQu8-FG2ht7Z3MCXDZw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1839" title="Mutation of the ATPase Domain of MutS Homolog-5 (MSH5) Reveals a Requirement for a Functional MutSγ Complex for All Crossovers in Mammalian Meiosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1869" title="An Interaction Network of the Human SEPT9 Established by Quantitative Mass Spectrometry" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493256" id="node493256" data-pisa="ggg;9/6/1851" data-pisa-master="ggg;g3.119.400088" data-apath="/ggg/9/6/1851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effect of Neutral Recombination Variation on Genome Scans for Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7529-2771" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katie E. Lotterhos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1851-1867;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400088 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493256" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katie E. Lotterhos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Marine and Environmental Sciences, Northeastern University, 430 Nahant Road, Nahant, MA 01908</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katie%2BE.%2BLotterhos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lotterhos%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatie%2BE.%2BLotterhos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7529-2771" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katie E. Lotterhos</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:k.lotterhos@northeastern.edu" class="" data-icon-position="" data-hide-link-title="0">k.lotterhos@northeastern.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/6/1851" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493256" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1851.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493256" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1851.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:493256" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:493256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1851.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493256" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/6/1851.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493256" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493256/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-75022 first last"><a href="/content/9/12/4245" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - December 01, 2019</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Recently, there has been an increasing interest in identifying the role that regions of low recombination or inversion play in adaptation of species to local environments. Many examples of groups of adapted genes located within inversions are arising in the literature, in part inspired by theory that predicts the evolution of these so-called “supergenes.” We still, however, have a poor understanding of how genomic heterogeneity, such as varying rates of recombination, may confound signals of selection. Here, I evaluate the effect of neutral inversions and recombination variation on genome scans for selection, including tests for selective sweeps, differentiation outlier tests, and association tests. There is considerable variation among methods in their performance, with some methods being unaffected and some showing elevated false positive signals within a neutral inversion or region of low recombination. In some cases the false positive signal can be dampened or removed, if it is possible to use a quasi-independent set of SNPs to parameterize the model before performing the test. These results will be helpful to those seeking to understand the importance of regions of low recombination in adaptation.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/principal-components" class="hw-term hw-article-keyword hw-article-keyword-principal-components" rel="nofollow">principal components</a></li><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">adaptation</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li></ul><p id="p-3">A recent focus of evolutionary biology has been to understand the genetic basis of adaptation. Recent studies have uncovered a range of complex genetic architectures underlying traits, and the idea that adapted loci may be co-located and physically linked in the genome has received a large amount of attention. These observations range from extended signals of selection deemed “genomic islands” (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Feder <em>et al.</em> 2012</a>), to clusters of tightly linked loci located within inversions deemed “supergenes” (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Schwander <em>et al.</em> 2014</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Thompson and Jiggins 2014</a>), to an important role of restricted recombination in local adaptation within species and maintaining boundaries among species (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Noor and Bennett 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Berner and Roesti 2017</a>). These empirical observations are expected based on theory. Theory predicts that rearrangements can bring locally adapted alleles together (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Yeaman 2013</a>) and that linked alleles have increased establishment probability (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Yeaman <em>et al.</em> 2016</a>). Theory also predicts that rearrangements that capture locally adapted alleles will prevent shuffling of alleles in heterozygotes, and that regions of suppressed recombination results in reduced gene flow that enables locally adapted alleles to evolve (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Guerrero <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Fuller <em>et al.</em> 2018</a>).</p><p id="p-4">Many of these observations of extended signals in the genome are based on the statistical analysis of population-genetic datasets using genome scans. <strong><em>Genome scans</em></strong> are statistical tests used to infer the genetic loci either affected by selection or that affect a trait (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Genome scans can be divided into three groups: those that seek to identify loci more differentiated among populations that expected from neutrality (differentiation outlier methods, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hoban <em>et al.</em> 2016</a>), those that seek to identify loci that are associated with a variable such as a phenotype or the environment where the individual was sampled (association methods, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bush and Moore 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Korte and Farlow 2013</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Rellstab <em>et al.</em> 2015</a>), and those that seek to characterize allele frequency or haplotype shifts due to positive selection acting within the sample (sweep methods, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Pritchard and Di Rienzo 2010</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Pritchard <em>et al.</em> 2010</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Schlamp <em>et al.</em> 2016</a>). Despite the theoretical expectations that we should find signals of selection within inversions or regions of low recombination, we still have a poor understanding of how genomic heterogeneity, such as variation in recombination rate, may produce false positive signals in the very regions we expect to find adaptive signals. Every genome scan method outputs a <strong><em>test statistic</em></strong> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) for each locus, or a numerical summary of the signal at that locus, and the statistic itself or the significance of that statistic can be biased if the data violates assumptions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493352/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493352/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Explanation of terms used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-5">Unfortunately, the rate at which new genome scan tests are being developed is rapidly eclipsing our understanding of their performance, and the effects of heterogeneity in the data (such as recombination variation) have not been well evaluated with simulations. Linkage disequilibrium (LD) is the non-random association of alleles at different loci (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Robbins 1918</a>). Regions of extensive LD may be found following hard selective sweeps as a new beneficial mutation arises in frequency (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Maynard-Smith and Haigh 1974</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Charlesworth <em>et al.</em> 1997</a>) as well as around regions of low recombination such as centromeres or inversions (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Kirkpatrick and Barton 2006</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Berner and Roesti 2017</a>). In this case, the potential issue is that the extensive LD that is expected to arise around a nucleotide due to strong selection is the same extensive LD that may be observed in neutral regions of low recombination, and that test statistics based on the allele frequency spectrum or haplotype structure (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Messer and Petrov 2013</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Schlamp <em>et al.</em> 2016</a>) may be similar between these two types of regions. Often measures of nucleotide diversity and measures of the allele frequency spectrum are correlated with recombination variation in genomes, but this is confounded with the action of positive selection and background selection (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Charlesworth <em>et al.</em> 1993</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Nachman 2001</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Charlesworth 2012</a>). Whether tests for selective sweeps have false positive signals in neutral regions of low recombination has not been systematically evaluated with simulations.</p><p id="p-6">A second potential issue is raised by a handful of empirical studies that have found that even when data across the genome is used, long-range LD at a particular locus can bias estimates of population structure and neutral demography (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Price <em>et al.</em> 2006</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Corbett-Detig and Hartl 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdellaoui <em>et al.</em> 2013</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Privé <em>et al.</em> 2018</a>). Accurately correcting for neutral demography in genome scans is important because non-independence among populations can create false positive signals. For example, early differentiation outlier methods basically assumed that migration was equal among all populations (<em>e.g.</em>, all populations were equally related to each other, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lewontin and Krakauer 1973</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beaumont and Nichols 1996</a>). These methods had high false positive rates because they did not account for complex demographic histories that create varying degrees of relatedness among populations in the data (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bonhomme <em>et al.</em> 2010</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">De Mita <em>et al.</em> 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Lotterhos and Whitlock 2014</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">2015</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Hoban <em>et al.</em> 2016</a>). It was also realized that association methods that did not correct for demographic history would exhibit false positive signals because of spatial autocorrelation in allele frequencies (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Meirmans 2012</a>). For this reason, most of the recently developed differentiation outlier and association methods use <strong><em>neutral parameterization</em></strong> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <em>sensu</em> <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Lotterhos and Whitlock 2014</a>) to calibrate or calculate the distribution of the test statistic expected under the neutral demographic process. Because genome scans must correct for population structure in the statistical test, long-range LD that distorts estimates of population structure will also distort the significance of the test statistic (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Price <em>et al.</em> 2008</a>). This suggests that distortion in population structure due to LD may also bias the results of differentiation outlier and association tests that require neutral parameterization, but this also has not been systematically evaluated with simulations.</p><p id="p-7">Overall, the effects of genomic heterogeneity in recombination and long-range LD on neutral parameterization, as well as on estimates of population structure, have not been well characterized. This is in part because most of the evaluative studies have simulated unlinked, independent loci and are not useful for evaluating this problem (e.g., <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Price <em>et al.</em> 2006</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Novembre and Stephens 2008</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">McVean 2009</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">De Mita <em>et al.</em> 2013</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Lotterhos and Whitlock 2014</a>, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Forester <em>et al.</em> 2016</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2018</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Martins <em>et al.</em> 2016</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Luu <em>et al.</em> 2017</a>). Here, I develop a novel set of simulations to evaluate how genomic heterogeneity in recombination affects the ability of genome scans to identify partial sweeps from a new mutation (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hudson <em>et al.</em> 1994</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Voight <em>et al.</em> 2006</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Pritchard <em>et al.</em> 2010</a>), complete sweeps from a new mutation (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Smith and Haigh 1974</a>), or causal quantitative trait nucleotides (<strong><em>QTNs</em></strong>, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) that affect a trait under spatially heterogeneous selection. One novel aspect of the simulations is that each of the 9 linkage groups (LGs) simulated were subject to a kind of realism expected in actual genomic data (<em>e.g.</em>, neutrality, QTNs, selective sweeps, an inversion, a centromeric region, and recombination variation). Having genomic heterogeneity in the same simulation is essential not only for simulating realistic genealogies (as determined by the contributions to fitness from different regions of the genome), but it is also essential for the evaluation of genome scans because false positive signals or biased neutral parameterization will affect the false discovery rate. As shown below, the genomic heterogeneity does confuse some analyses, but other analyses are unaffected. It is precisely this case when genomic heterogeneity confuses analysis that demonstrates how caution must be applied when analyzing genomic data.</p><p id="p-8">I use a basic demography that results in an isolation-by-distance pattern in the data to explore the effects of genomic heterogeneity, rather than the effects of different population demographies. While the genomic heterogeneity in LD among SNPs in the simulations is caused by variation in recombination rates among proximate SNPs, similar heterogeneity might be present in a dataset which contained a mixture of random sets of SNPs from the entire genome and long sequences of proximate SNPs obtained by sequence capture.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Simulations</h3><p id="p-9">I conducted 200 replicate forward-time simulations of a metapopulation adapting to a heterogeneous spatial environment (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) with SLiM v. 3.2 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Haller and Messer 2017</a>) to create SNP data for each individual. The simulations resulted in a population that had isolation-by-distance structure along an environmental gradient (<em>e.g.</em>, isolation by environment, <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Wang and Bradburd 2014</a>). For simplicity in interpreting the results, only one type of genomic heterogeneity was simulated on each LG, such that each LG evolved approximately independently. Each of the 9 LGs were 50,000 bases and 50 cM in length. The base recombination rate <em>N<sub>e</sub>r</em> = 0.01 (unless manipulated as described below) gave a resolution of 0.001 cM between proximate bases. The recombination rate was scaled to mimic the case where SNPs were collected across a larger genetic map than what was simulated (similar to a SNP chip), but still low enough to allow signatures of selection to arise in neutral loci linked to selected loci (in the simulations 50,000 bases / (<em>r</em> = 1e-05) * 100 = 50 cM; in humans 50,000 bp would correspond to 0.05 cM). Thus, SNPs at the opposite ends of linkage groups were likely to have a recombination rate between them of 0.5 (unlinked), but there would otherwise be some degree of linkage among SNPs within linkage groups. For all LGs, the population-scaled mutation rate <em>N<sub>e</sub></em>μ equaled 0.001. For computational efficiency, 1000 individuals were simulated with scaling of mutation rate and recombination rate as described above (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fisher 1930</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Wright 1931</a>, <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">1938</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Crow and Kimura 1970</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bürger 2000</a>). In the first generation, individuals were placed randomly on a spatial map between the coordinates 0 and 1. Individuals dispersed a distance given by a bivariate normal distribution with zero mean and variance <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-1.gif"/></noscript></span></span> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example landscape simulation. Each box is an individual, colored by their phenotypic value. The background is the selective environment. This output was generated after 1900 generations of selection by the environment, resulting in a correlation of 0.52 between the phenotype and the environment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example landscape simulation. Each box is an individual, colored by their phenotypic value. The background is the selective environment. This output was generated after 1900 generations of selection by the environment, resulting in a correlation of 0.52 between the phenotype and the environment.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F1.medium.gif" width="440" height="439"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/6/1851/F1.medium.gif" width="440" height="439"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493292" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Example landscape simulation. Each box is an individual, colored by their phenotypic value. The background is the selective environment. This output was generated after 1900 generations of selection by the environment, resulting in a correlation of 0.52 between the phenotype and the environment.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493348/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493348/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Parameters used in the simulations and their values</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">A unique aspect of these simulations is that individuals experienced six components of fitness to reflect a core set of biologically realistic pressures acting on the genome: (i) local adaptation of QTNs with additive effects on a phenotype subject to selection by a heterogeneous environment, (ii) competition, (iii) mating success, (iv) a beneficial new mutation (hard sweep) at a single site introduced 300 generations before sampling, which was enough time for the sweep to be near or at fixation for varying lengths of time (“full sweep”), (v) a beneficial new mutation (hard sweep) at another site introduced 60 generations before sampling, which was enough time for the sweep to typically reach a frequency from 0.5-0.85 (“partial sweep”), and (vi) weak negative frequency dependent selection on the inversion, which was included to maintain the inversion as polymorphic in the population (but was unrelated to local adaptation to the environment). Total fitness was calculated as the multiple of the fitness components after they were scaled to relative fitness for that component. The parameters chosen for competition and mating (described in Appendix) resulted in local neighborhoods or clusters of individuals (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), which could later be used to assign individuals to populations for methods that required it. The remaining fitness components are described below in the “Genetic Map” section.</p><p id="p-12">I tested for coalescence with preliminary simulations and found for some replicates that even 50N generations of burn-in was not enough time for the metapopulation to coalesce. Therefore, I only simulated non-neutral mutations in SLiM with tree sequencing implemented to record the geneology, and then used recaptitation with pyslim (v0.1) and msprime (v0.6.1) to reconstruct the ancestry of the initial genomes using the coalescent and to add neutral mutations (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Haller <em>et al.</em> 2018</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kelleher <em>et al.</em> 2018</a>). Note that at this time recapitation of a SLiM simulation can only be performed on a uniform recombination map. To evolve more realistic patterns of haplotype diversity under recombination variation, the SLiM simulation was run without spatially heterogeneous selection for 10<em>N</em> generations, followed by a period of 2<em>N</em> generations during which the metapopulation experienced spatially heterogeneous selection by the environment. After recaptitation and the addition of neutral mutations, a vcf file was produced for subsequent analyses with genome scans. See <em>Data Availability</em> for scripts and code.</p></div><div id="sec-3" class="subsection"><h3>Data filtering</h3><p id="p-13">VCF files produced from the simulations were first filtered for individuals that were related and then for loci that had MAF &lt; 0.01 with the R package vcfR (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Knaus and Grunwald 2016</a>). Relatedness was calculated using the statistic of Lynch and Ritland (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Lynch and Ritland 1999</a>) with the R package related for a subset of 800 SNPs with MAF &gt; 0.05, and for pairs of individuals with a pairwise relationship coefficient greater than 0.5, one individual from the pair was removed. The test statistics described below were then calculated from this filtered set of data.</p></div><div id="sec-4" class="subsection"><h3>Genetic map and fitness components</h3><div id="sec-5" class="subsection"><h4>LG-1 and LG-2: Neutral:</h4><p id="p-14">The first two linkage groups were simulated under neutrality as described above (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). No loci in these regions affected fitness components.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Example genetic map with 9 linkage groups (LGs), each 50 cM long. LG-1 and LG-2 were neutral; on LG-3 and LG-4 quantitative trait loci (QTL) were allowed to evolve (but exact locations and effect sizes depended on the simulation); on LG-5 and LG-6 a hard selective sweep from a new mutation occurred at the purple arrow; on LG-7 a neutral inversion was simulated in the center; on LG-8 a region of low recombination (3 orders of magnitude lower than the base rate) was simulated in the center; and on LG-9 recombination variation was simulated randomly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Example genetic map with 9 linkage groups (LGs), each 50 cM long. LG-1 and LG-2 were neutral; on LG-3 and LG-4 quantitative trait loci (QTL) were allowed to evolve (but exact locations and effect sizes depended on the simulation); on LG-5 and LG-6 a hard selective sweep from a new mutation occurred at the purple arrow; on LG-7 a neutral inversion was simulated in the center; on LG-8 a region of low recombination (3 orders of magnitude lower than the base rate) was simulated in the center; and on LG-9 recombination variation was simulated randomly.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F2.medium.gif" width="440" height="147"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/6/1851/F2.medium.gif" width="440" height="147"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493363" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-15" class="first-child">Example genetic map with 9 linkage groups (LGs), each 50 cM long. LG-1 and LG-2 were neutral; on LG-3 and LG-4 quantitative trait loci (QTL) were allowed to evolve (but exact locations and effect sizes depended on the simulation); on LG-5 and LG-6 a hard selective sweep from a new mutation occurred at the purple arrow; on LG-7 a neutral inversion was simulated in the center; on LG-8 a region of low recombination (3 orders of magnitude lower than the base rate) was simulated in the center; and on LG-9 recombination variation was simulated randomly.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h4>LG-3 and LG-4: Quantitative trait nucleotides:</h4><p id="p-16">On these linkage groups QTNs that had additive effects on the trait could evolve with the probability of a new mutation being a QTN equal to 0.02 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). If a new mutation was a QTN, the effect size was drawn from a distribution with mean 0 and variance σ<em><sub>QTN</sub></em>. In this manner genetic architecture (the distribution of effect sizes and linkage relationships among causal mutations) was allowed to evolve. Because they were subject to spatially heterogeneous selection, QTNs that evolved to explain a major proportion of the genetic variance created signatures that could be detectable by differentiation outlier methods and genetic-environment associations (GEAs) (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Hoban <em>et al.</em> 2016</a>). Because they had additive effects on phenotypes, they could also be detectable by genome-wide association studies with phenotypes (GWAS) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Bush and Moore 2012</a>).</p><p id="p-17">To simulate local adaptation, the trait was subject to spatially heterogeneous stabilizing selection with the optimum for each location in space dependent on the environment (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). To allow some standing genetic variation at the QTNs to evolve prior to the onset of heterogeneous selection, the phenotype was under weak stabilizing selection with a mean trait optimum <em>θ =</em> 0 and variance <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-2.gif"/></noscript></span></span> for the first 10<em>N</em> generations. At 10<em>N</em> generations selection by the environment was initiated. For each location, the environmental optimum was randomly generated at 25 locations on the grid by adding random variation (σ = 0.5) to a longitudinal cline (from -1 in the west, to 1 in the center, and -1 in the east) to mimic a mountain range, and then interpolated to determine the environmental optimum <em>θ<sub>xy</sub></em> for the trait at any <em>{x,y}</em> location (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). When selection to the environment was initiated, the environmental optimum at each location changed from the historical optimum to the new optimum linearly for 100 generations, and then remained at the new optimum for the remainder of the simulation.</p><p id="p-18">For each individual in each generation, the local-adaptation fitness-component was determined by a Gaussian function given the difference between the individual’s phenotype and the optimum at that location, with the strength of stabilizing selection as the variance of the Gaussian function <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-3.gif"/></noscript></span></span>. The fitness component for individual <em>i</em> at location <em>{x,y}</em> was:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>z<sub>i</sub></em> is the phenotype (calculated from the sum of QTN effect sizes) of individual <em>i</em>, and <em>θ<sub>xy</sub></em> is the optimum at the location where the individual was located.</p></div><div id="sec-7" class="subsection"><h4>LG-5 and LG-6: Selective sweeps:</h4><p id="p-19">In the center of LG-5 and LG-6, positive selection on a single new mutation was simulated to create the signature of a hard selective sweep (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Because these sites arose in frequency across the metapopulation, it was not going to be detectable by differentiation outlier, GEA, or GWAS methods. Instead, these sites would be detectable by methods based on the allele frequency spectrum or haplotype-based methods (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Messer and Petrov 2013</a>; <a id="xref-ref-80-3" class="xref-bibr" href="#ref-80">Schlamp <em>et al.</em> 2016</a>). I simulated two strongly selected mutations with the objective of creating obvious signatures that should be detectable by methods based on allele frequency spectrum or linkage disequilibrium. Each positively selected mutation was dominant and had a selection coefficient <em>s<sub>sweep</sub></em> = 0.5. Since some genome scans are sensitive to whether or not the sweep fixed, I simulated an older sweep on LG-5 that was at or near fixation by the end of the simulation (“full sweep”, introduced 300 generations before the end of the simulation) and a more recent sweep on LG-6 that typically rose to a frequency of 0.5-0.85 (“partial sweep”, introduced 60 generations before the end of the simulation). To ensure that the sweep was present at the end of the simulation, it was reintroduced if it was lost to drift within 20 generations of first introduction.</p></div><div id="sec-8" class="subsection"><h4>LG-7 Neutral with a large inversion:</h4><p id="p-20">A 10000 base pair region in the center LG-7 was simulated as a large inversion. While in nature recombination may occur between inverted and ancestral haplotypes in a heterozygote, a single crossover would result in inviable gametes that lack critical genes. Since this process effectively results in essentially no recombination between inversion and ancestral haplotypes, for computational simplicity recombination was allowed between homozygotes but completely suppressed between heterozygotes.</p><p id="p-21">To ensure that the inversion was segregating in the population at the end of the simulation, the inversion was maintained under weak negative frequency dependent selection. Individuals without the inversion had a relative fitness equal to 1.0, while individuals with the inversion had a fitness component that depended on the frequency of the inversion in the population <em>f</em>: <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-5.gif"/></noscript></span></span> = 1.0 - (<em>f</em> - 0.5) * 0.1. Thus, at any point in time the fitness component of the rarer haplotype was slightly higher than the common haplotype. Note however that the fitness of the inversion was not related to local adaptation to the environment, and all mutations within the inversion were neutral. These parameters resulted in a range of minor allele frequencies of the inversion haplotypes across replicate simulations (see Results).</p></div><div id="sec-9" class="subsection"><h4>LG-8 Neutral with a large region of low recombination:</h4><p id="p-22">On, LG-8 a stretch of 10,000 bases at the center of the LG was simulated with a recombination rate of 10<sup>−8</sup>, which was three orders of magnitude lower than the base recombination rate, but would have been higher than recombination between the two inversion haplotypes where recombination was completely suppressed.</p></div><div id="sec-10" class="subsection"><h4>LG-9 Neutral with recombination variation:</h4><p id="p-23">LG-9 was simulated with random recombination rate variation, where 9 breakpoints were randomly chosen from a uniform distribution and the recombination rates for the 10 resulting regions were generated by 10<sup>-</sup><em><sup>b</sup></em>, where <em>b</em> was a random normal variable with mean 5 and standard deviation of 2. This generated recombination rates between ∼0.01 and 10<sup>−9</sup> at random breakpoints, with a mean of the base recombination rate at 10<sup>−5</sup>.</p></div></div><div id="sec-11" class="subsection"><h3>Population structure and neutral parameterization</h3><p id="p-24">Because of the potentially confounding effects of long-range LD, many population-structure methods recommend that an “independent” set of SNPs be used to estimate population structure (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bradburd <em>et al.</em> 2018</a>). Here I will use the term “quasi-independent,” because all SNPs are located within the same genome and can never be truly independent. A common way of obtaining a quasi-independent set of SNPs is to thin for linkage disequilibrium (<strong><em>LD or SNP thinning</em></strong>, <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), which typically moves along a genome in a sliding window and thins SNPs based on linkage disequilibrium with each other. This may be based on a combination of (i) “pruning,” which sequentially scans the genome and performs pairwise thinning based on a given threshold of correlation, (ii) “clumping,” which may incorporate some information about the importance of SNPs based on summary statistics, and (iii) removing SNPs in long-range LD regions (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Privé <em>et al.</em> 2018</a>). While SNP thinning is a recommended practice for obtaining a set of SNPs to estimate population structure, it is unclear what the best practices are for genome scans because SNP thinning will (i) remove the characteristic signature that selection will leave in the genome and (ii) potentially remove causal or adaptive SNPs (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>: “<strong><em>Neutral parameterization on thinned SNPs and genome scan on thinned SNPs</em></strong>”). Methods vary widely in how they control for structure, their flexibility, and their guidelines for users. For example, some methods allow the user to use one set of SNPs for neutral parameterization and perform the statistical test on another (more complete) set of SNPs, and clearly recommend that a random and independent set be used for the first step (<em>e.g.</em>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Günther and Coop 2013</a>) (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>, “<strong><em>Neutral parameterization with thinned SNPs and genome scan on all SNPs</em></strong>”, “best practice”). Other methods could be implemented in these two steps - but that is not the default and best practices are not well articulated in the resources for users - and so naïve users tend to use the entire set of SNPs for neutral parameterization and to perform the test (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>, “<strong><em>Neutral parameterization on all SNPs and genome scan on all SNPs</em></strong>”, “naïve approach”). Note that for some genome scan approaches like haplotype-based statistics, this latter approach is the only option. In many cases, the best practices for implementing genome scans in the presence of long-range LD are unclear.</p><p id="p-25">When applicable, the “naïve approach” for the method was run on all SNPs with MAF &gt; 0.01. When applicable (as noted for each method below), the “best practice” involved implementing the statistical tests in two steps (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). In step 1, the whole genome data were thinned for LD to obtain a quasi-independent set of SNPs, and this set of <em>thinned</em> SNPs were used for neutral parameterization. In step 2, these null parameters were used to calculate the test statistic for <em>all</em> SNPs. SNP thinning was implemented in R packages bigsnpr and bigstatsr with the function snp_autoSVD, which uses sliding windows to remove SNPs correlated greater than 0.2 with the SNP with the highest MAF in that window and removes regions with putative long-range LD (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Privé <em>et al.</em> 2018</a>).</p><p id="p-26">Because the issues with some genome scans arise through how they capture population structure via the process of neutral parameterization, I also evaluate the “naïve approach” (all SNPs with MAF &gt; 0.01) and the “best practice” (thinned SNPs, <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>) on population structure estimates from principal components. Principal components analysis was conducted on the SNP matrix (coded as 0, 1, or 2) using the function pcadapt() in the R package pcadapt v3.0.4 (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Luu <em>et al.</em> 2017</a>). This function output the score for each diploid individual (`$scores`) and the loading of individual SNP (`$loadings`) onto each PC axis. I evaluated the degree to which scores reflected the isolation-by-distance population structure that evolved on the landscape, as well as the degree to which loadings were sensitive to genomic variation in recombination rate.</p></div><div id="sec-12" class="subsection"><h3>Genome scan approaches</h3><p id="p-27">I compared different methods for detecting selection from genomic data, including differentiation outlier tests (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Hoban <em>et al.</em> 2016</a>), genome-wide association studies with phenotypes (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Bush and Moore 2012</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Korte and Farlow 2013</a>), genetic-environment associations (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Rellstab <em>et al.</em> 2015</a>), and statistics for detecting selective sweeps based on shifts in haplotype frequencies (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Pritchard and Di Rienzo 2010</a>; <a id="xref-ref-80-4" class="xref-bibr" href="#ref-80">Schlamp <em>et al.</em> 2016</a>). When applicable, a method was run with a “naïve approach” and the “best practice.” The “best practice” was determined by any of the following: (i) following recommendations in the citation and the user guides, (ii) communication with the developer, and (iii) preliminary data analysis on a subset of simulations.</p><div id="sec-13" class="subsection"><h4>Selective sweep methods:</h4><p>The statistics evaluated are designed to detect the shifts in haplotype frequencies around the new mutation under positive selection (LG-5 and LG-6 in the simulations). I evaluated three statistics:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-29">iHS: integrated haplotype score; a measure of the amount of extended haplotype homozygosity (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Voight <em>et al.</em> 2006</a>);</p></li><li id="list-item-2"><p id="p-30"><em>H<sub>12</sub></em>: estimates haplotype homozygosity by combining the frequencies of two most frequent haplotypes into a single frequency and adding it to the total haplotype homozygosity (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Messer and Petrov 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Garud <em>et al.</em> 2015</a>); and</p></li><li id="list-item-3"><p id="p-31"><em>H<sub>2</sub>/H<sub>1</sub></em>: compares the haplotype homozygosity using all but the most frequent haplotype to the total haplotype homozygosity (<a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Messer and Petrov 2013</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Garud <em>et al.</em> 2015</a>). This value is expected to be smaller for hard sweeps, so was transformed to -log<sub>10</sub>(<em>H<sub>2</sub>/H<sub>1</sub></em>) for performance evaluation (see Supplementary Materials).</p></li></ul><p><em>iHS</em> is known to have lower power than <em>H<sub>12</sub></em> when the sweep is near fixation (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Tang <em>et al.</em> 2007</a>; <a id="xref-ref-80-5" class="xref-bibr" href="#ref-80">Schlamp <em>et al.</em> 2016</a>). Also, <em>iHS</em> is standardized to correct for variation in recombination rate over a range of SNPs with similar derived allele frequencies (<a id="xref-ref-87-3" class="xref-bibr" href="#ref-87">Voight <em>et al.</em> 2006</a>), but <em>H<sub>12</sub></em> and <em>H<sub>2</sub>/H<sub>1</sub></em> are not. Statistics were trimmed within 1000 bases of the ends of the linkage groups to reduce potential biases there. Because these methods are based on haplotypes, they could only be implemented on the complete set of SNPs. For details of all calculations see the Supplementary Materials.</p></div><div id="sec-14" class="subsection"><h4>Differentiation outlier methods:</h4><p>The differentiation outlier methods are designed to detect loci that are subject to spatially heterogeneous selection and evolve to be differentiated above and beyond that expected by neutral demographic history (QTNs on LG-3 and LG-4). I compared three methods that differ in the way that they correct for population structure:</p><ul class="list-unord " id="list-2"><li id="list-item-4"><p id="p-33">OutFLANK: identifies <em>F<sub>ST</sub></em> outliers after estimating the neutral parameters on the <em>F<sub>ST</sub></em> distribution (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Whitlock and Lotterhos 2015</a>);</p></li><li id="list-item-5"><p id="p-34">PCAdapt: identifies outliers along the principal components that describe structure (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Duforet-Frebourg <em>et al.</em> 2014</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Luu <em>et al.</em> 2017</a>); and</p></li><li id="list-item-6"><p id="p-35"><em>X<sup>T</sup>X</em>: identifies outliers along the covariance in allele frequencies among populations based on the model of Bayenv2 (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Günther and Coop 2013</a>) as implemented by Gautier (2015).</p></li></ul><p>For each statistic, the “naïve approach” was evaluated using the results from running the algorithm on all SNPs. The “best practice” was evaluated using the results from running the algorithm in two steps: first, using a quasi-independent set of thinned SNPs for neutral parameterization, and then second, using that parameterization to run the model and obtain <em>P</em>-values or test statistics for all SNPs. See Supplemental Materials for details on all calculations.</p></div><div id="sec-15" class="subsection"><h4>Genome-wide association (GWAS) methods:</h4><p>The GWAS methods are designed to detect loci that have effects on phenotypes and evolve to be associated with the phenotypes (QTNs on LG-3 and LG-4). I compared two methods that adjust for coefficient inflation in a latent factor mixed model (LFMM) association between genotypes and phenotypes: ridge regression and lasso (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Caye <em>et al.</em> 2019</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">François and Caye 2018</a>). LFMMs model unexplained variation with latent factors, which are estimated jointly with the main effects in the model. Thus, neutral parameterization with these methods happens internally (<em>e.g.</em>, the latent factors and model coefficients are estimated jointly), and so the algorithms can only be run in one step on the set of all SNPs. To accurately estimate latent factors that capture the genetic population structure in a GWAS, genotype is modeled as a function of phenotype (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">François and Caye 2018</a>). I tested two different methods for adjusting for coefficient inflation in large data sets with collinear predictor variables (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Caye <em>et al.</em> 2019</a>):</p><ul class="list-unord " id="list-3"><li id="list-item-7"><p id="p-37">ridge regression, which adjusts all model coefficients by a shrinkage term; and</p></li><li id="list-item-8"><p id="p-38">lasso, which adjusts for coefficient inflation by minimizing the residual sum of squares with a penalty.</p></li></ul><p>More details can be found in the Supplementary Materials.</p></div><div id="sec-16" class="subsection"><h4>Genetic-environment association (GEA) methods:</h4><p>The GEA methods are designed to detect loci that are subject to spatially heterogeneous selection by a specific environment and evolve to be associated with that environment (QTNs that evolve on LG-3 and LG-4). I compare four methods that measure the association between the allele frequency and an environmental variable:</p><ul class="list-unord " id="list-4"><li id="list-item-9"><p id="p-40">latent factor mixed Bayesian hierarchical model: measures environment-allele association corrected for structure with latent factors (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Frichot and François 2015</a>);</p></li><li id="list-item-10"><p id="p-41">Bayes Factor from BayPass: measures strength of evidence of an association, corrected for structure with population covariance matrix (Gautier 2015). The “naïve approach” and “best practice” were implemented as described for <em>X<sup>T</sup>X</em> above;</p></li><li id="list-item-11"><p id="p-42">Spearman’s ρ: uncorrected association between allele frequency and the environment; and</p></li><li id="list-item-12"><p id="p-43">redundancy analysis: a method to extract and summarize the variation in a set of response variables (SNPs) that can be explained by a set of explanatory variables (environmental variables) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Legendre and Legendre 2012</a>). Performance was evaluated using the loading of SNP on the constrained axis (<em>e.g.</em>, the environmental predictor) following <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Forester <em>et al.</em> (2018)</a>; note this approach does not correct for structure.</p></li></ul><p>For details on these calculations see Supplementary Materials.</p></div></div><div id="sec-17" class="subsection"><h3>Comparison of performance</h3><p id="p-44">Causal SNPs were counted as true positives if they contributed more than 1% to the additive genetic variance of the trait. Because SNPs were filtered for final data analysis using standard cutoffs (MAF &gt; 0.01), some causal SNPs were below this threshold (rare alleles). Some of these rare alleles were of small effect or just recently introduced in the population, thus contributing very little to the overall genetic variance of the trait. The proportion of additive genetic variance for causal SNPs was approximated as the additive genetic variance for SNP <em>i</em> standardized by the total additive genetic variance:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-6.gif"/></noscript></span></span>Where <em>α<sub>i</sub></em> is the effect size and <em>p<sub>i</sub></em> is the allele frequency of the derived allele at SNP <em>i</em>. Causal SNPs whose contribution to the additive genetic variance was greater than a proportion of 0.01 were then counted as true positives (hereafter: “causal SNPs”). Note that some of these causal SNPs were rare alleles (<em>e.g.</em>, MAF &lt; 0.01) of large effect, and so they were not present in the set of analyzed SNPs but they were still counted in the calculation of error rates because they explained more than 1% of the additive genetic variance (see Results).</p><p id="p-45">The performance of each metric was then summarized as the area under the curve of the precision-recall graph (AUC-PR) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Davis and Goadrich 2006</a>). This is preferred to the area under the curve of the Receiver Operating Characteristics graph (AUC-ROC) graph (Fawcett 2004), because in biological applications when the number of samples in each classifier is imbalanced (for example in these simulations there are many neutral loci and only a few selected loci), a method can achieve a high AUC-ROC even when the majority of positive hits are false positives (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Saito and Rehmsmeier 2015</a>). In these cases AUC-PR more accurately captures performance. An AUC-PR = 1 represents the case when all causal SNPs have higher scores than neutral SNPs. The AUC-PR that is expected by random is equal to the proportion of causal loci in each dataset, which varied slightly from simulation to simulation but was generally less than 1%. The AUC-PR was calculated with the R package PRROC (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Grau <em>et al.</em> 2015</a>) using the continuous interpolation method (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boyd <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Keilwagen <em>et al.</em> 2014</a>).</p><p id="p-46">I summarized performance of each method for the ability to detect (i) all causal QTNs and (ii) a 2000 bp region on either side of a selective sweep (note in this latter case the sweep nucleotide was not always in the dataset because it was at or near fixation). The distribution of AUC-PR over replicate simulations was visually compared among methods with boxplots. I inspected how signals in different regions of the genome affected the performance of each metric by calculating the empirical cumulative distribution function for neutral loci (ecdf() function in R) and evaluating the average quantile of different regions of the genome (<em>e.g.</em>, sweeps, QTNs, inversion, etc.) relative to this empirical neutral distribution (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Lotterhos and Whitlock 2014</a>).</p></div><div id="sec-18" class="subsection"><h3>Data Availability</h3><p id="p-47">To facilitate validation of new methods and their sensitivity to genomic realism, I have created a github group called “Test the Tests.” Within this group is a repository for this study, <a href="https://github.com/TestTheTests/TTT_RecombinationGenomeScans">https://github.com/TestTheTests/TTT_RecombinationGenomeScans</a>, that includes all the scripts needed to recreate the simulations and figures presented in this manuscript. The repository is structured so that the simulations may be easily analyzed by new methods, and that the results may be easily added and visually compared using the same metrics presented here. In addition the repository and “.trees” files from the simulations are archived on Dryad at <a href="https://doi.org/10.5061/dryad.rj0kj10">https://doi.org/10.5061/dryad.rj0kj10</a> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Lotterhos 2019</a>). Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7973438">https://doi.org/10.25387/g3.7973438</a>.</p></div></div><div class="section results" id="sec-19"><h2 class="">Results</h2><p id="p-48">After filtering for minor alleles, on average datasets had 6647 SNPs. On average, 5% of SNPs were located within the non-adaptive inversion on LG-7 and 2% in the region of low recombination on LG-8. The simulation parameters resulted in a range of minor haplotype frequencies in the inversion and a range of ages for the origin of the inversion (Supplemental Figure S1).</p><p id="p-49">On average 9.6 causal SNPs evolved that contributed more than 1% to additive genetic variance (V<sub>A</sub>) in the trait. The distribution of causal allele frequencies and their effect sizes are shown in Supplemental Figure S2. Typically, 4-13 SNPs explained 80–98% of V<sub>A</sub> (based on 0.05 and 0.95 quantiles), and the locus in each simulation that explained the most genetic variance explained on average 45%. Rare alleles of large effect (that were filtered for analysis because they were rare) typically left 3% of V<sub>A</sub> unexplained, but this could be as high as 15–20% in a few simulations (Supplemental Figure S3).</p><p id="p-50">The full hard selective sweep generally was at or near fixation (frequency &gt; 0.9) by the end of the simulation, and the mutations that fixed were fixed for 1-200 generations (up to 0.2N generations, Supplementary Figure S4 A, B). The partial hard sweep generally reached an allele frequency of 0.5-0.85 by the end of the simulation but never fixed (Supplementary Figure S4 C).</p><div id="sec-20" class="subsection"><h3>Effect of inversions and thinning on population structure</h3><div id="sec-21" class="subsection"><h4>PC scores of individuals on PC axes:</h4><p id="p-51">When all the SNPs were used to conduct a PCA, scores of individuals along PC1 depended on their inversion haplotype on LG7 in the majority of replicate simulations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). This occurred despite the SNPs in this region representing on average 5% of the total number of SNPs in the data. Their scores along PC2 depended on the simulation, but generally were determined by their haplotype in the region of low recombination on LG8 or in the region of recombination variation on LG9. This means that when all SNPs in the data were used, the population structure was reflecting these regions in the genome rather than the isolation-by-environment pattern in the data.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PC scores of individuals on PC axes for one replicate simulation. A) PC scores calculated from all loci. Structure along PC1 was determined by the number of copies of the inversion (&#x201C;0|0&#x201D; lacked inversion, &#x201C;0|1&#x201D; inversion heterozygote, and &#x201C;1|1&#x201D; inversion homozygote). In this simulation, structure along PC2 was determined by haplotype in the region of low recombination on LG-8. B) PC scores calculated from a quasi-independent set of thinned SNPs, which resulted in PC1 reflecting the isolation-by-environment structure in the data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;PC scores of individuals on PC axes for one replicate simulation. A) PC scores calculated from all loci. Structure along PC1 was determined by the number of copies of the inversion (&#x201C;0|0&#x201D; lacked inversion, &#x201C;0|1&#x201D; inversion heterozygote, and &#x201C;1|1&#x201D; inversion homozygote). In this simulation, structure along PC2 was determined by haplotype in the region of low recombination on LG-8. B) PC scores calculated from a quasi-independent set of thinned SNPs, which resulted in PC1 reflecting the isolation-by-environment structure in the data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F3.medium.gif" width="288" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/6/1851/F3.medium.gif" width="288" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493407" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-52" class="first-child">PC scores of individuals on PC axes for one replicate simulation. A) PC scores calculated from all loci. Structure along PC1 was determined by the number of copies of the inversion (“0|0” lacked inversion, “0|1” inversion heterozygote, and “1|1” inversion homozygote). In this simulation, structure along PC2 was determined by haplotype in the region of low recombination on LG-8. B) PC scores calculated from a quasi-independent set of thinned SNPs, which resulted in PC1 reflecting the isolation-by-environment structure in the data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">In contrast, when a set of quasi-independent thinned SNPs were used, the scores of individuals along PC1 reflected the isolation-by-environment pattern in the data (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p></div><div id="sec-22" class="subsection"><h4>PC loadings of loci onto PC axes:</h4><p id="p-54">The above results can be explained by understanding how loci load onto PC axes. When all SNPs in the data were used in the PCA to estimate population structure, in the majority of simulations the neutral inversion on LG7 loaded most strongly onto the first PC axis (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). This explains why in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, PC scores of individuals along PC1 could be explained by their inversion haplotype. The genomic location that loaded most strongly onto the second PC axis varied from simulation to simulation, but was typically the region of low recombination on LG8, or less frequently the region of recombination variation on LG9 or the neutral inversion (if it didn’t load onto the first PC axis) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Thus, in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, PC scores of individuals along PC2 were segregated according to their haplotype in the region that had the highest loading along this axis.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Loadings of loci onto PC axes across all replicate simulations. Each panel shows the frequency distribution of genomic locations that had outlier PC loadings for that scenario. A) When all SNPs were used, typically SNPs in the inversion would have outlier loadings along PC1 (corresponding to scores along x-axis in Figure 3A). B) When all SNPs were used, typically SNPs in one of the low recombination regions would have outlier loadings along PC2 (corresponding to scores along y-axis in Figure 3A). C) When a set of quasi-independent thinned SNPs were used, the QTNs that adapted to the environment (or neutral loci linked to them) had the highest loadings along PC1 (corresponding to scores along x-axis in Figure 3B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Loadings of loci onto PC axes across all replicate simulations. Each panel shows the frequency distribution of genomic locations that had outlier PC loadings for that scenario. A) When all SNPs were used, typically SNPs in the inversion would have outlier loadings along PC1 (corresponding to scores along x-axis in Figure 3A). B) When all SNPs were used, typically SNPs in one of the low recombination regions would have outlier loadings along PC2 (corresponding to scores along y-axis in Figure 3A). C) When a set of quasi-independent thinned SNPs were used, the QTNs that adapted to the environment (or neutral loci linked to them) had the highest loadings along PC1 (corresponding to scores along x-axis in Figure 3B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F4.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/6/1851/F4.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493357" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-55" class="first-child">Loadings of loci onto PC axes across all replicate simulations. Each panel shows the frequency distribution of genomic locations that had outlier PC loadings for that scenario. A) When all SNPs were used, typically SNPs in the inversion would have outlier loadings along PC1 (corresponding to scores along x-axis in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>). B) When all SNPs were used, typically SNPs in one of the low recombination regions would have outlier loadings along PC2 (corresponding to scores along y-axis in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3A</a>). C) When a set of quasi-independent thinned SNPs were used, the QTNs that adapted to the environment (or neutral loci linked to them) had the highest loadings along PC1 (corresponding to scores along x-axis in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">In contrast, when a set of thinned SNPs were used, the genomic regions that contained QTN that adapted to the environment (LG2 and LG3) had the highest loadings onto PC1 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). More specifically, the QTNs selected for local adaptation to the environment (or neutral loci linked to them - whichever were present in the trimmed set of SNPs) in each replicate simulation had the highest loadings. This explains why in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3B</a>, individual scores along PC1 reflected the isolation-by-environment structure along the environmental gradient.</p><p id="p-57">In preliminary analyses with other methods based on using allele frequencies to assign individuals to ancestral populations (<em>e.g.</em>, STRUCTURE and similar methods, <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Frichot <em>et al.</em> 2014</a>), I also found that only the estimates of population structure based on a set of thinned SNPs were accurate in describing the isolation-by-environment pattern present in the neutral data (results not shown).</p></div></div><div id="sec-23" class="subsection"><h3>Performance of genome scans</h3><div id="sec-24" class="subsection"><h4>Selective sweep methods:</h4><p id="p-58">Performance was assessed as the Area Under the Precision-Recall Curves (AUC-PR). Sweep statistics generally had low performance (AUC-PR &lt; 0.3) for detecting the full and partial sweeps (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5 A and B</a>, first three columns). The low performance based on AUC-PR could be because (1) low power to detect the sweep due to lack of signal around the sweep area, and/or (2) large false positive signals in other areas of the genome. These effects can be visualized by the empirical quantile of the signal in different regions of the genome relative to the neutral loci simulated on LG-1 and LG-2. For instance, <em>H<sub>12</sub></em> and <em>H<sub>2</sub>/H<sub>1</sub></em> had strong signals at the partial sweep (0.8-1.0 quantile) while iHS had a relatively weaker signal (0.6-0.9 quantile) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). However, <em>H<sub>12</sub></em> also had elevated signals in the region of the inversion and region of low recombination (&gt;0.75 quantile), while <em>iHS</em> in these regions overlapped with the median signal at neutral loci (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, see also Manhattan plot in Supplementary Figure S5). Overall, <em>H<sub>12</sub></em> and <em>H<sub>2</sub>/H<sub>1</sub></em> had similarly large signals at the sweep mutations but also elevated signals in regions of low recombination rate, which led to lower overall performance. <em>H<sub>2</sub>/H<sub>1</sub></em> outperformed <em>H<sub>12</sub></em> in some cases because it was less sensitive to recombination variation (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>) and neither statistic was affected by the minor haplotype frequency within the inversion region (Supplementary Figure S6). On the other hand, the low performance of <em>iHS</em> was caused by less power to detect the sweep mutations, even though it was not affected by recombination variation (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Area Under the Curve for Precision-Recall (AUC-PR) measures the overall performance of each statistic: AUC = 1 means that all causal loci have larger signals than neutral loci, and an AUC &#x223C;0.01 is expected under random chance (horizontal gray line). A) AUC-PR evaluated in the region of the full sweep (2000 bp on either side). B) AUC-PR evaluated in the region of the partial sweep (2000 bp on either side). C) AUC-PR evaluated at all causal QTNs that explained greater than 1% in the additive genetic variance. Each method is shown in a different color, and the naive practice is shown with hatched bars when applicable." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Area Under the Curve for Precision-Recall (AUC-PR) measures the overall performance of each statistic: AUC = 1 means that all causal loci have larger signals than neutral loci, and an AUC &#x223C;0.01 is expected under random chance (horizontal gray line). A) AUC-PR evaluated in the region of the full sweep (2000 bp on either side). B) AUC-PR evaluated in the region of the partial sweep (2000 bp on either side). C) AUC-PR evaluated at all causal QTNs that explained greater than 1% in the additive genetic variance. Each method is shown in a different color, and the naive practice is shown with hatched bars when applicable.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F5.medium.gif" width="257" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/6/1851/F5.medium.gif" width="257" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493351" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-59" class="first-child">Area Under the Curve for Precision-Recall (AUC-PR) measures the overall performance of each statistic: AUC = 1 means that all causal loci have larger signals than neutral loci, and an AUC ∼0.01 is expected under random chance (horizontal gray line). A) AUC-PR evaluated in the region of the full sweep (2000 bp on either side). B) AUC-PR evaluated in the region of the partial sweep (2000 bp on either side). C) AUC-PR evaluated at all causal QTNs that explained greater than 1% in the additive genetic variance. Each method is shown in a different color, and the naive practice is shown with hatched bars when applicable.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots of empirical quantiles of each test statistic in different regions of the genome. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplots of empirical quantiles of each test statistic in different regions of the genome. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F6.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/6/1851/F6.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493267" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-60" class="first-child">Boxplots of empirical quantiles of each test statistic in different regions of the genome. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">The performance of all statistics depended on the allele frequency of the sweep mutation at sampling, with higher power of all statistics to detect sweeps that had not yet fixed. <em>H<sub>12</sub></em> always had higher power than <em>iHS</em> across allele frequencies and after fixation (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The power of <em>H<sub>12</sub></em> was maximized at an allele frequency of 0.7-0.9, while the power of <em>iHS</em> was maximized at an allele frequency of 0.6-0.7 (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7 A</a>). Both statistics retained similar power between fixation and up to 200 generations after fixation (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7 B</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical quantiles of H12 and iHS across replicate simulations as a function of (A) allele frequency and (B) for fixed sweeps, the number of generations since fixation. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Empirical quantiles of &lt;em&gt;H&lt;sub&gt;12&lt;/sub&gt;&lt;/em&gt; and &lt;em&gt;iHS&lt;/em&gt; across replicate simulations as a function of (A) allele frequency and (B) for fixed sweeps, the number of generations since fixation. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F7.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/6/1851/F7.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493365" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-62" class="first-child">Empirical quantiles of <em>H<sub>12</sub></em> and <em>iHS</em> across replicate simulations as a function of (A) allele frequency and (B) for fixed sweeps, the number of generations since fixation. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h4>Differentiation outlier methods:</h4><p id="p-63">Both the naïve and best practice approach was evaluated for PCAdapt, OutFLANK (<em>F<sub>ST</sub></em>), and BayPass (<em>X<sup>T</sup>X</em>). In general, OutFLANK and BayPass did not have a large difference in performance between the naïve approach and best practice (compare “naïve” to “best practice” for the two methods in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5C</a> and <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). However, the AUC-PR of PCAdapt increased greatly when the neutral population structure was computed on the set of thinned SNPs (best practice) rather than the entire set of SNPs (naïve approach) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a> for AUC-PR, compare PCAdapt “naïve” to PCAdapt “best practice”). This difference in performance occurred because using the naïve approach resulted in large false positive outlier signals in the inversion and regions of low recombination (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>, compare PCAdapt “naïve” to PCAdapt “best practice”). These outlier signals occurred in the naïve approach because of the way that regions of low recombination loaded onto the principal components (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots of empirical quantiles of each test statistic in different regions of the genome. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line. (A) Differentiation outlier statistics, comparing the &#x201C;naive&#x201D; to the &#x201C;best practice&#x201D; (BP). (B) Association tests." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplots of empirical quantiles of each test statistic in different regions of the genome. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line. (A) Differentiation outlier statistics, comparing the &#x201C;naive&#x201D; to the &#x201C;best practice&#x201D; (BP). (B) Association tests.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F8.medium.gif" width="297" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/9/6/1851/F8.medium.gif" width="297" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493337" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-64" class="first-child">Boxplots of empirical quantiles of each test statistic in different regions of the genome. The genome-wide median at neutral loci (0.5 quantile) is shown as the horizontal line. (A) Differentiation outlier statistics, comparing the “naive” to the “best practice” (BP). (B) Association tests.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">The “best practice” for all methods had similarly moderate performance (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5 C</a>, AUC-PR ∼0.1-0.8), similarly large signals at the largest effect QTL in the data (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8 A</a>, quantile ∼1.0), and median quantiles in the inversion and low recombination regions similar to that expected based on neutrality (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8A</a>, median quantile ∼0.5 across replicate simulations). In some replicate simulations, a haplotype in a region of low recombination drifted to different frequencies among subpopulations and resulted in an elevated signal within that region (<em>e.g.</em>, Supplementary Figure S7, Manhattan plot). When comparing the methods for the “best practice” scenario using the AUC-PR, PCAdapt and OutFLANK (median AUC-PR ∼0.35) had slightly higher performance than BayPass (median AUC-PR ∼0.3, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>).</p><p id="p-66">Overall, there was considerable variation from simulation to simulation in which differentiation outlier statistic performed the best and had the most power to detect smaller effect QTNs. For example, in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9A</a> OutFLANK had the best performance: although PCAdapt had higher precision for the largest-effect QTNs, OutFLANK had higher precision for moderate-to-small effect QTNs and this resulted in an overall higher AUC. However, in <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9B</a> PCAdapt had the best performance: although <em>X<sup>T</sup>X</em> had higher precision for the largest-effect QTNs, PCAdapt had higher precision for moderate-to-small effect QTNs and this resulted in an overall higher AUC. In <a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9C</a>, OutFLANK had the highest precision to detect QTNs of all effect sizes. These results illustrate that no single statistic is ideal in all scenarios.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Precision-recall curves for three replicate simulations. For each method, the Area Under the Curve (AUC-PR) is indicated in the legend. Because large-effect loci tend to have the largest signal and be discovered first, the power of methods to detect them are captured whichever method has a larger area under the curve on the left side of the plot where recall is low (e.g., only the loci with the largest signals have been discovered). The power of methods to detect moderate-to-small effect loci are captured by whichever method has a higher AUC-PR in the center and right side of the plot where recall is higher (more of the causal loci have been discovered). Precision decreases with recall because many neutral loci have signals that are larger than small-effect causal loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1436916253" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Precision-recall curves for three replicate simulations. For each method, the Area Under the Curve (AUC-PR) is indicated in the legend. Because large-effect loci tend to have the largest signal and be discovered first, the power of methods to detect them are captured whichever method has a larger area under the curve on the left side of the plot where recall is low (&lt;em&gt;e.g.&lt;/em&gt;, only the loci with the largest signals have been discovered). The power of methods to detect moderate-to-small effect loci are captured by whichever method has a higher AUC-PR in the center and right side of the plot where recall is higher (more of the causal loci have been discovered). Precision decreases with recall because many neutral loci have signals that are larger than small-effect causal loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1851/F9.medium.gif" width="243" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/9/6/1851/F9.medium.gif" width="243" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1851/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493342" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-67" class="first-child">Precision-recall curves for three replicate simulations. For each method, the Area Under the Curve (AUC-PR) is indicated in the legend. Because large-effect loci tend to have the largest signal and be discovered first, the power of methods to detect them are captured whichever method has a larger area under the curve on the left side of the plot where recall is low (<em>e.g.</em>, only the loci with the largest signals have been discovered). The power of methods to detect moderate-to-small effect loci are captured by whichever method has a higher AUC-PR in the center and right side of the plot where recall is higher (more of the causal loci have been discovered). Precision decreases with recall because many neutral loci have signals that are larger than small-effect causal loci.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h4>Association methods:</h4><p id="p-68">For associations with phenotypes (GWAS), ridge regression outperformed lasso regression for latent factor mixed models (LFMM, <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>). The lower performance for LFMM-lasso occurred because although both methods were able to detect the largest effect QTN (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8B</a>), the lasso sometimes had elevated signals in the inversion and the regions of low recombination (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Figure 8B</a>, see also Manhattan plot in Supplementary Figure 8).</p><p id="p-69">For associations with environments (GEA), Spearman’s <em>ρ</em> and RDA had similarly high performance, followed by LEA, and then BayPass (<em>BF</em>) (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5C</a>). All methods had extreme signals at the largest effect QTN and were not affected by recombination variation (<a id="xref-fig-8-7" class="xref-fig" href="#F8">Figure 8B</a>). Although BayPass (<em>BF</em>) showed a slightly elevated signal in the inversion compared to the genome-wide median (<a id="xref-fig-8-8" class="xref-fig" href="#F8">Figure 8B</a>), these signals were far below anything that would be considered significant (average log<sub>10</sub>(<em>BF</em>) in the inversion was -11.2; for a log<sub>10</sub>(<em>BF</em>) to be considered “decisive evidence” it should be larger than 2 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kass and Raftery 1995</a>)). Finally, it should be noted that in some replicate simulations a haplotype in a region of low recombination could drift to different frequencies among populations and show elevated signals for some association tests but not others (Manhattan plot in Supplementary Figure S8).</p><p id="p-70">Finally, the SNP loadings on the unconstrained axes of the RDA (<em>e.g.</em>, the principal components) reflected genomic variation in recombination rate, with the inversion typically loading onto the first PC and the other regions of low recombination loading onto the subsequent PCs (Supplementary Figure S9).</p></div></div></div><div class="section discussion" id="sec-27"><h2 class="">Discussion</h2><p id="p-71">Genome scan methods have different strengths and weaknesses, and it is important to understand what these are when detecting signals of selection in genomes. This study used the process of analysis validation (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Lotterhos <em>et al.</em> 2018</a>) to illustrate how inversions and regions of low recombination can confound some genome scans. Methods designed to detect selective sweeps based on haplotype frequency may show elevated signals in regions of low recombination. Non-independence among SNPs due to LD can disproportionally affect estimations of population structure, especially those based on principal components. For some differentiation outlier and association methods, even a small percent of non-independent SNPs in the data can bias estimates population structure and create false positive signals in these regions. This negative consequence can be mitigated if a set of SNPs that have been thinned for LD are used to first estimate population structure or calibrate the null distribution, which better captures the neutral demography of the study species. Given the importance of inversions (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Schwander <em>et al.</em> 2014</a>; <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Thompson and Jiggins 2014</a>) and centromeric regions (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Berner and Roesti 2017</a>) in local adaptation, these results illustrate the importance of validating genomic pipelines on simulations before making conclusions about the genomic architecture of adaptive traits.</p><div id="sec-28" class="subsection"><h3>Estimating population structure with principal components</h3><p id="p-72">Many seminal studies that have been used to evaluate genome scans have used independently simulated SNPs to evaluate the methods. In the seminal paper by <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Price <em>et al.</em> (2006)</a> where principal components were first introduced as a way to control for population structure in GWAS, they simulated an independent set of SNPs to evaluate the method. Following this seminal paper, a few studies have illustrated some of the caveats of using principal components to estimate genetic structure using independently simulated SNPs. For instance, <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Novembre and Stephens (2008)</a> showed that PCA can depend on the details of a particular dataset, including distribution of sampling locations and amounts of data. Similarly, <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">McVean (2009)</a> showed that PCA can be strongly influenced by uneven sampling among populations and biases in SNP ascertainment, and that the accuracy of PC axes to capture structure increases with the number of independent SNPs.</p><p id="p-73">By illustrating the effects of genomic heterogeneity in recombination on principal component estimates of genetic structure, this study builds on these previous studies as well as ground-truths empirical studies that have documented the effects of linkage disequilibrium on PCA. For instance, the <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Wellcome Trust Case Control Consortium (2007)</a> found that data from a 500K human SNP chip was dense enough to create high loadings of SNPs onto regions of linkage disequilibrium, and as a result four of the first six PCs on population structure reflected local genomic LD. They also showed how related individuals can distort projections of samples onto PC axes and admixture proportions. Similarly, <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Price <em>et al.</em> (2008)</a> articulated concerns about how long-range LD and inversion polymorphisms can produce signals of unusual ancestry and should be accounted for in genome scans for selection. <a id="xref-ref-75-4" class="xref-bibr" href="#ref-75">Privé <em>et al.</em> (2018)</a> showed how long-range LD regions from a celiac disease dataset (∼300K SNPs) load onto principal components and how these effects can be removed with SNP thinning. Most recently, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Li and Ralph (2019)</a> showed that inversions and regions of low recombination can dominate patterns of mean relatedness along the genome as determined by local principal components.</p><p id="p-74">Three main conclusions can be drawn from the simulations presented in this study and empirical studies. First, for each PC axis, individual scores will be dominated by the SNPs that have the highest loadings along that axis. Second, using a SNP dataset that has <em>not</em> been trimmed for LD may result in local genomic LD loading onto some of the principal components. On one hand, this may be a good way to identify local regions of LD, and to quickly visualize and group individuals according to their haplotype within that LD region (for an empirical example see <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barney <em>et al.</em> 2017</a>). Even low density SNP datasets (consisting of tens of thousands of SNPs) may be susceptible depending on the range of LD; for example the resolution in these simulations was a SNP only every 0.07 cM (∼50 cM per LG divided by ∼700 SNPs per LG), which corresponds to every ∼70,000 bp in humans (corresponding to a 50K SNP chip). Third, thinning SNPs for LD can more accurately capture neutral genetic structure (although scores will still be dominated by SNPs with the highest loadings). These caveats and best practices will be useful to those using various kinds of software that incorporate principal components (PCs) of genotypes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Abraham and Inouye 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Abraham <em>et al.</em> 2016</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Galinsky <em>et al.</em> 2016</a>; <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Price <em>et al.</em> 2006</a>).</p></div><div id="sec-29" class="subsection"><h3>Variation in performance of genome scans</h3><p id="p-75">This study compared the ability of three statistics to detect hard selective sweeps from new mutation. In these simulations, <em>iHS</em> statistic overall had low power than <em>H<sub>12</sub></em> and <em>H<sub>2</sub>/H<sub>1</sub></em> to detect the sweep across allele frequencies. This contrasted with previous studies that found <em>iHS</em> had high power to detect partial sweeps that had not reached fixation (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Tang <em>et al.</em> 2007</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Schrider <em>et al.</em> 2015</a>; <a id="xref-ref-80-6" class="xref-bibr" href="#ref-80">Schlamp <em>et al.</em> 2016</a>). <em>H<sub>12</sub></em> and <em>H<sub>2</sub>/H<sub>1</sub></em> had similarly large signals at the hard sweep across a range of allele frequencies, which is in agreement with previous studies that found these statistics had high power to detect the a mutation under strong selection simulated here (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Schrider and Kern 2016</a>). However, both statistics also showed elevated signals in the inversion and regions of low recombination and this led to overall lower performance. In the simulations these neutral regions were dominated by only a couple haplotypes segregating in the population, and so statistics based on combining the frequencies of the two most common haplotypes were inflated. In <em>Drosophila melanogaster H<sub>12</sub></em> peaks were not associated with inversions (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Garud <em>et al.</em> 2015</a>), which suggests that the results from this study may not apply to all cases.</p><p id="p-76">Of the three differentiation outlier methods tested with these simulations, the naïve PCAdapt was shown to be the most susceptible to genomic variation in LD, but when best practices were employed this method had similar performance to other methods. This sensitivity for the naïve approach occurred when all SNPs were used to estimate the principal components, because regions of high LD had high loadings on the PC axes. OutFLANK and BayPass were not sensitive to the set of SNPs used. However for OutFLANK, it should be noted that bias in the estimation of the neutral mean <em>F<sub>ST</sub></em> and <em>df</em> has been observed with high-density genomic data compared to a subset of trimmed SNPs (KEL, pers. obs.), and that best practices should still be employed. Previously, the model that BayPass is based on was shown to perform poorly under isolation by distance compared to other methods (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Lotterhos and Whitlock 2015</a>), and this should be considered when interpreting the results presented here.</p><p id="p-77">Almost all the association methods had similar performance. There was some variation in the behavior of three latent factor mixed models (the ridge regression with phenotype, lasso regression with phenotype, and LEA with environment), with the lasso regression sometimes showing outlier signals in the inversion or regions of low recombination. In the implementation of these models in the R package, the ridge regression is solved analytically while the lasso regression is solved numerically, which may indicate that the ridge estimates are more accurate (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Caye <em>et al.</em> 2019</a>). However, in some of the evaluations performed by <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Caye <em>et al.</em> (2019)</a>, the lasso regression appeared to outperform the ridge regression (<em>e.g.</em>, in the Celiac disease dataset), and so additional validation may be required for these implementations to understand their different strengths and weaknesses. Although both Spearman’s <em>ρ</em> and redundancy analysis had high performance to separate signals of selection at QTNs from signals at neutral loci, it is important to note that these methods do not correct for population structure: even though causal loci have large signals than neutral loci, many neutral loci can have significant <em>P</em>-values when structure is not accounted for (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Meirmans 2012</a>).</p></div><div id="sec-30" class="subsection"><h3>Limitations of simulations</h3><p id="p-78">While the simulations presented have certain types of realism in terms of including local adaptation, selective sweeps, and recombination variation, each type of realism was simulated on discrete linkage groups. Such compartmentalization of genomic heterogeneity is unlikely to be found in nature. Additionally, natural inversions may show evidence of exchange between arrangements, which may occur during rare double crossovers or gene conversion events (<em>e.g.</em>, <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Schaeffer and Anderson 2005</a>). This type of exchange was not captured by the simulations. The simulated data were also idealistic in that every individual in the population was sampled, their genotypes were known without error, and there was no missing data. There was also no environmental noise that affected the phenotypes (which were also known exactly without error), which is part of the reason for the high performance of GWAS methods. While these properties of the data are idealistic, the effects of sampling and genotyping errors could easily be explored with the archived files. Since these simulations used scaling of mutation and recombination rate to population size, they may fail to capture some important dynamics that are unique to large samples. The ability of genome scans to produce accurate results can also be affected by a number of qualities of the data, including sampling strategy (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Lotterhos and Whitlock 2015</a>), ascertainment bias (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lachance and Tishkoff 2013</a>), allele dropout (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gautier <em>et al.</em> 2013</a>), and missing data.</p></div><div id="sec-31" class="subsection"><h3>Selection in regions of low recombination</h3><p id="p-79">Many empirical studies suggest that inversions may capture sets of adaptive QTNs, deeming these regions “supergenes” (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cheng <em>et al.</em> 2011</a>; <a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">Schwander <em>et al.</em> 2014</a>; <a id="xref-ref-86-3" class="xref-bibr" href="#ref-86">Thompson and Jiggins 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kapun <em>et al.</em> 2016</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barney <em>et al.</em> 2017</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berg <em>et al.</em> 2017</a>). Others have similarly inferred that centromeric regions of low crossover rate facilitate adaptive divergence (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Berner and Roesti 2017</a>). Do the simulations results suggest that the importance of “supergenes” or regions of low recombination have been overstated in the literature? Robustly answering this question would require a thorough meta-analysis of how these regions were identified, which methods were used, and whether the authors employed best practices when scanning the genome. Many of the classic examples of inversions underlying adaptive differences among populations or phenotypes within a species are from traditional QTL mapping, which was not evaluated in this study. However, the association tests in this study tended to perform well and not be susceptible to false positive signals from the inversion or regions of low recombination. In general, caution should be applied in drawing conclusions about the importance of a specific genomic regions based solely on the functions and ontologies of genes located within that region (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Pavlidis <em>et al.</em> 2012</a>) or based solely on chromosomal structure.</p><p id="p-80">When applicable, employing the “best practice” suggested here should help discern false positive signals in neutral inversions from true positive signals in adaptive inversions. Models show that locally adapted inversions have higher genetic differentiation than the genome-wide background or drifting inversions (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Guerrero <em>et al.</em> 2012</a>, but note neutral inversions under genetic drift can produce coalescent patterns similar to locally adapted inversions of intermediate age). The ability of methods to discern different genetic architectures (<em>e.g.</em>, monogenic <em>vs.</em> polygenic) within adaptive inversions, however, is an important area for future research, and one that could give important insights to the “supergene” hypothesis. Discerning whether an adaptive inversion has a monogenic or polygenic basis may be a difficult task with population genomic data because of the extended signals resulting from low recombination rates within the inversions. However, new approaches based on machine learning are being used to discern among different types of genomic heterogeneity, including different types of selection acting on a single sequence, and may hold promise for this problem (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Schrider and Kern 2018</a>).</p></div><div id="sec-32" class="subsection"><h3>Conclusions and best practices</h3><p id="p-81">The single best practice that can be employed when analyzing genome scans for selection is to validate your genomics pipeline (including filtering, imputation of missing data, genomic realism, and analysis methods) on simulated data (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Lotterhos <em>et al.</em> 2018</a>). In most cases this will not feasible, as it could be as much work as the empirical study itself. The following steps may help to validate analyses with genome scans when simulations are not available. First, related individuals should be removed from the analysis (<em>e.g.</em>, <a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">Wellcome Trust Case Control Consortium 2007</a>) or the relatedness should be accurately controlled for in the statistical test. Next and when applicable, estimate the neutral population structure (either by principal components, a covariance matrix, or by estimating parameters on some model) on a quasi-independent set of SNPs that has been thinned for LD. When applicable, use this same subset of SNPs across all methods to control for population structure in testing for selection across all sites in the data. When applicable, also inspect <em>P</em>-value histograms and <em>Q-Q</em> plots to check assumptions, and calculate the genomic inflation factor (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">François <em>et al.</em> 2016</a>). Next, evaluate the sensitivity of your results to the decisions made in the pipeline, such as filtering SNPs, imputing genotypes, phasing haplotypes, etc. The effects of unequal sample size should also be evaluated to ensure that they do not affect the results (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">McVean 2009</a>).</p><p id="p-82">Finally, datasets and pipelines that reproduce the results should be required for publication, including widely used file formats (<em>e.g.</em>, VCF) that are produced after the process of filtering reads, mapping reads, and calling variants. Only when easy-to-reproduce pipelines are provided will it be possible to re-analyze data if a weakness is identified in one of the methods used in the pipeline.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-85">Thanks to the editor, anonymous reviewers, Kieran Samuk, Sam Yeaman, Sara Schaal, Kevin Freeman, Brett Ford, Áki Láruson, Alan Downey-Wall, and members of the Noor lab for helpful feedback on earlier versions of this manuscript. This research was supported by funding by a grant from the National Science Foundation (#1655701) to KEL.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-33"><h3>Fitness component: competition.</h3><p id="p-83">Individuals that experienced more competition had a lower fitness component for competition. Competition for a single individual was experienced as the sum of interactions with all other individuals, which declined with distance and increased with phenotypic similarity. Total interaction strength (<em>P<sub>Ci</sub></em>) for any individual <em>i</em> at spatial location <em>x<sub>i</sub></em>, <em>y<sub>i</sub></em> equaled<em>:</em><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-8.gif"/></noscript></span></span> = [ <em>x<sub>i</sub></em>, <em>y<sub>i</sub></em>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-9.gif"/></noscript></span></span>] of individual <em>i</em>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-10.gif"/></noscript></span></span>is the phenotype of individual <em>i</em>, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-11.gif"/></noscript></span></span> = [ <em>x<sub>j</sub></em>, <em>y<sub>j</sub></em>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-12.gif"/></noscript></span></span>] for individual <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-13.gif"/></noscript></span></span>, and Σ was a 3 x 3 diagonal matrix with equal variance <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-14.gif"/></noscript></span></span> and no covariances. Thus, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-15.gif"/></noscript></span></span> described the spatial and phenotypic distance over which individuals interacted. The competition fitness-component for individual <em>i</em> was estimated from <em>P<sub>Ci</sub></em> as an exponential decay (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-16.gif"/></noscript></span></span> = exp(0.0009*100*<em>P<sub>C</sub></em>*(1-<em>P<sub>C</sub></em>))), such that individuals that had 0 total interaction strength had a relative fitness of 1 and individuals that had large interaction strengths (&gt; ∼7) had a total fitness of about 0. In this manner, local density dependence was implemented in the model and the maintenance of some genetic variation was maintained within genetic neighborhoods (<em>e.g.</em>, weak disruptive selection).</p></div><div class="section" id="sec-34"><h3>Fitness component: mate choice.</h3><p id="p-84">Individuals also experienced mate choice. The total interaction strength for mating for individual <em>i</em> (<em>P<sub>Mi</sub></em>) was also given by Equation 2, such that individuals that were closer in space and phenotype had larger interaction strengths, with the exception that Σ was modeled with a different variance (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-17.gif"/></noscript></span></span>). The mating fitness-component (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1851/embed/mml-math-18.gif"/></noscript></span></span>) was equal to <em>P<sub>Mi</sub></em> scaled to relative fitness for this component. Thus, assortative mating favored the erosion of genetic variation (<em>e.g.</em>, weak stabilizing selection) within local neighborhoods.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-86">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7973438">https://doi.org/10.25387/g3.7973438</a>.</p></li><li class="fn" id="fn-2"><p id="p-87"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-12"><span class="received-label">Received </span>February 12, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-04-05"><span class="accepted-label">Accepted </span>April 5, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Lotterhos</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.1" data-doi="10.1038/ejhg.2013.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdellaoui</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hottenga</span>  <span class="cit-name-given-names">J.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Knijff</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nivard</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population structure, migration, and diversifying selection in the Netherlands.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1277</span>–<span class="cit-lpage">1285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ejhg.2013.48</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D21%26rft.spage%253D1277%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2013.48%26rft_id%253Dinfo%253Apmid%252F23531865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2013.48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23531865&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.2" data-doi="10.1371/journal.pone.0093766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inouye</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast principal component analysis of large-scale genome-wide data.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e93766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De93766%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0093766%26rft_id%253Dinfo%253Apmid%252F24718290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0093766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24718290&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inouye</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">FlashPCA2: principal component analysis of biobank-scale genotype datasets.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">2014</span>–<span class="cit-lpage">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft.volume%253D12%26rft.spage%253D2014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.4" data-doi="10.1016/j.ajhg.2015.12.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galinsky</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhatia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georgiev</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Fast principal-component analysis reveals convergent evolution of ADH1B in Europe and East Asia.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D456%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.12.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.12.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.5" data-doi="10.1186/s12864-017-3660-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barney</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munkholm</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walt</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palumbi</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Highly localized divergence within supergenes in Atlantic cod (<em>Gadus morhua</em>) within the Gulf of Maine.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3660-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3660-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3660-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.6" data-doi="10.1098/rspb.1996.0237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Evaluating loci for use in the genetic analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">1619</span>–<span class="cit-lpage">1626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.1996.0237</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1996.0237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1996.0237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.7" data-doi="10.1038/hdy.2017.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Star</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pampoulie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentzen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Trans-oceanic genomic divergence of Atlantic cod ecotypes is associated with large inversions.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">418</span>–<span class="cit-lpage">428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2017.54</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2017.54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2017.54&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.8" data-doi="10.1111/mec.14373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomics of adaptive divergence with chromosome-scale heterogeneity in crossover rate.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">6351</span>–<span class="cit-lpage">6369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14373</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D6351%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.9" data-doi="10.1534/genetics.110.117275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chevalet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdallah</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detecting selection in population trees: the Lewontin and Krakauer test extended.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.117275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.117275%26rft_id%253Dinfo%253Apmid%252F20855576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvMjQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1851.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Area under the Precision-Recall Curve: Point Estimates and Confidence Intervals</span>, <span class="cit-source">Machine Learning and Knowledge Discovery in Databases. ECML PKDD 2013</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Blockeel</span>  <span class="cit-name-given-names">H.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kersting</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Nijssen</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Železný</span>  <span class="cit-name-given-names">F.</span></span>, (Lecture Notes in Computer Science), <span class="cit-vol">Vol. 8190</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.11" data-doi="10.1534/genetics.118.301333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradburd</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Inferring continuous and discrete population genetic structure across space.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301333</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301333%26rft_id%253Dinfo%253Apmid%252F30026187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjEwLzEvMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1851.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bürger</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source"><em>The Mathematical Theory of Selection</em>, <em>Recombination</em>, <em>and Mutation</em></span>, <span class="cit-publ-name">Wiley</span>, <span class="cit-publ-loc">Chichester, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.13" data-doi="10.1371/journal.pcbi.1002822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chapter 11: Genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002822</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1002822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D8%26rft.spage%253De1002822%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002822%26rft_id%253Dinfo%253Apmid%252F23300413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23300413&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.14" data-doi="10.1093/molbev/msz008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jumentier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lepeule</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">LFMM 2: fast and accurate inference of gene-environment associations in genome-wide studies</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">852</span>–<span class="cit-lpage">860</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msz008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D36%26rft.spage%253D852%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsz008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msz008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The effect of deleterious mutations on neutral molecular variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D1289%26rft.epage%253D1303%26rft.atitle%253DThe%2Beffect%2Bof%2Bdeleterious%2Bmutations%2Bon%2Bneutral%2Bmolecular%2Bvariation.%26rft_id%253Dinfo%253Apmid%252F8375663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNC80LzEyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.16" data-doi="10.1017/S0016672397002954"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The effects of local selection, balanced polymorphism and background selection on equilibrium patterns of genetic diversity in subdivided populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672397002954</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D70%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D174%26rft.atitle%253DThe%2Beffects%2Bof%2Blocal%2Bselection%252C%2Bbalanced%2Bpolymorphism%2Band%2Bbackground%2Bselection%2Bon%2Bequilibrium%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bsubdivided%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397002954%26rft_id%253Dinfo%253Apmid%252F9449192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397002954&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9449192&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071240500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.17" data-doi="10.1534/genetics.111.134288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The effects of deleterious mutations on evolution at linked sites.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.134288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134288%26rft_id%253Dinfo%253Apmid%252F22219506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTkwLzEvNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xODUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.18" data-doi="10.1534/genetics.111.137794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamdem</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mockaitis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costantini</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ecological genomics of <em>Anopheles gambiae</em> along a latitudinal cline: a population-resequencing approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1417</span>–<span class="cit-lpage">1432</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.137794</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137794%26rft_id%253Dinfo%253Apmid%252F22209907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.19" data-doi="10.1371/journal.pgen.1003056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of inversion polymorphisms in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003056</span>. Erratum: 9<span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/annotation/b1cace11-17ed-456e-b8a9-006c09125bd0</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003056%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003056%26rft_id%253Dinfo%253Apmid%252F23284285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284285&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1851.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">An Introduction to Population Genetics Theory</span>, <span class="cit-publ-name">Harper and Row, Publishers, Inc.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.6.1851.21" data-doi="10.1145/1143844.1143874"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goadrich</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> The relationship between Precision-Recall and ROC curves, in <em>Proceedings of the 23rd International Conference on Machine learning</em><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1145/1143844.1143874</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1145%252F1143844.1143874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1145/1143844.1143874&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.22" data-doi="10.1111/mec.12182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Mita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuillet</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmadi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detecting selection along environmental gradients: analysis of eight methods and their effectiveness for outbreeding and selfing populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1383</span>–<span class="cit-lpage">1399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D1383%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000315414700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.23" data-doi="10.1093/molbev/msu182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duforet-Frebourg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome scans for detecting footprints of local adaptation using a Bayesian factor model.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2483</span>–<span class="cit-lpage">2495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu182%26rft_id%253Dinfo%253Apmid%252F24899666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24899666&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343401100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-9.6.1851.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-comment">ROC Graphs: Notes and Practical Considerations for Researchers. Kluwer Academic Publishers, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.25" data-doi="10.1016/j.tig.2012.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egan</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosil</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genomics of speciation-with-gene-flow.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2012.03.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DFeder%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D342%26rft.epage%253D350%26rft.atitle%253DThe%2Bgenomics%2Bof%2Bspeciation-with-gene-flow.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.03.009%26rft_id%253Dinfo%253Apmid%252F22520730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22520730&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306252300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1851.26" data-doi="10.5962/bhl.title.27468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source">The Genetical Theory of Natural Selection</span>, <span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Oxford</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5962/bhl.title.27468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BGenetical%2BTheory%2Bof%2BNatural%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.5962%252Fbhl.title.27468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5962/bhl.title.27468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.27" data-doi="10.1111/mec.13476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landguth</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Detecting spatial genetic signatures of local adaptation in heterogeneous landscapes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13476</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13476&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.28" data-doi="10.1111 /mec. 14584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Comparing methods for detecting multilocus adaptation with multivariate genotype-environment associations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2215</span>–<span class="cit-lpage">2233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D27%26rft.spage%253D2215%26rft_id%253Dinfo%253Adoi%252F10.1111%2B%252Fmec.%2B14584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111 /mec. 14584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.29" data-doi="10.1111/mec.13513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martins</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoville</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Controlling false discoveries in genome scans for selection.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">454</span>–<span class="cit-lpage">469</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13513</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D454%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13513%26rft_id%253Dinfo%253Apmid%252F26671840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26671840&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.30" data-doi="10.1111/1755-0998.12892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caye</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Naturalgwas: An R package for evaluating genomewide association methods with empirical data.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D18%26rft.spage%253D789%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.31" data-doi="10.1534/genetics.113.160572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathieu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trouillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchard</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fast and efficient estimation of individual ancestry coefficients.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.160572</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160572%26rft_id%253Dinfo%253Apmid%252F24496008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzQvOTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.32" data-doi="10.1111/2041-210X.12382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">LEA: An R package for landscape and ecological association studies.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">929</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/2041-210X.12382</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.33" data-doi="10.1111/mec.14923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">Z. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koury</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phadnis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">How chromosomal rearrangements shape adaptation and speciation: case studies in <em>Drosophila pseudoobscura</em> and its sibling species <em>Drosophila persimilis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-fpage">mec.14923</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.34" data-doi="10.1371/journal.pgen.1005004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buzbas</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recent selective sweeps in North American <em>Drosophila melanogaster</em> show signatures of soft sweeps.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005004%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005004%26rft_id%253Dinfo%253Apmid%252F25706129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706129&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.6.1851.35" data-doi="10.1534/genetics.115.181453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide scan for adaptive divergence and association with population-specific covariates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1555</span>–<span class="cit-lpage">1579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.181453%26rft_id%253Dinfo%253Apmid%252F26482796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS80LzE1NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.36" data-doi="10.1111/mec.12089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cezard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foucaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerdelhué</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of RAD allele dropout on the estimation of genetic variation within and between populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3165</span>–<span class="cit-lpage">3178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3165%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.37" data-doi="10.1093/bioinformatics/btv153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grau</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grosse</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keilwagen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PRROC: computing and visualizing precision-recall and receiver operating characteristic curves in R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2595</span>–<span class="cit-lpage">2597</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv153%26rft_id%253Dinfo%253Apmid%252F25810428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25810428&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.38" data-doi="10.1098/rstb.2011.0246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guerrero</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Coalescent patterns for chromosomal inversions in divergent populations.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">367</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2011.0246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DGuerrero%26rft.auinit1%253DR.%2BF.%26rft.volume%253D367%26rft.issue%253D1587%26rft.spage%253D430%26rft.epage%253D438%26rft.atitle%253DCoalescent%2Bpatterns%2Bfor%2Bchromosomal%2Binversions%2Bin%2Bdivergent%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2011.0246%26rft_id%253Dinfo%253Apmid%252F22201172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2011.0246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22201172&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.39" data-doi="10.1534/genetics.113.152462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Robust identification of local adaptation from allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152462</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152462%26rft_id%253Dinfo%253Apmid%252F23821598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzEvMjA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.40" data-doi="10.1093/molbev/msw211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">SLiM 2: Flexible, interactive forward genetic simulations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw211%26rft_id%253Dinfo%253Apmid%252F27702775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27702775&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.41" data-doi="10.1111/1755-0998.12968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galloway</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Tree-sequence recording in SLiM opens new horizons for forward-time simulation of whole genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755‐0998.12968</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.42" data-doi="10.1086/688018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoban</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradburd</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Finding the genomic basis of local adaptation: pitfalls, practical solutions, and future directions.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">397</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/688018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D188%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1086%252F688018%26rft_id%253Dinfo%253Apmid%252F27622873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/688018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27622873&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skarecky</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Evidence for positive selection in the superoxide dismutase (Sod) region of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1329</span>–<span class="cit-lpage">1340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1329%26rft.epage%253D1340%26rft.atitle%253DEvidence%2Bfor%2Bpositive%2Bselection%2Bin%2Bthe%2Bsuperoxide%2Bdismutase%2B%2528Sod%2529%2Bregion%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F8013910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzEzMjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.44" data-doi="10.1093/molbev/msw016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabian</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flatt</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic evidence for adaptive inversion clines in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1317</span>–<span class="cit-lpage">1336</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw016%26rft_id%253Dinfo%253Apmid%252F26796550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26796550&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.45" data-doi="10.1080/01621459.1995.10476572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raftery</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayes Factors.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">795</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1995.10476572</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D90%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01621459.1995.10476572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01621459.1995.10476572&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.46" data-doi="10.1371/journal.pone.0092209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keilwagen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grosse</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grau</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Area under precision-recall curves for weighted and unweighted data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e92209</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0092209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De92209%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0092209%26rft_id%253Dinfo%253Apmid%252F24651729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0092209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24651729&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.47" data-doi="10.1371/journal.pcbi.1006581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashander</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Efficient pedigree recording for fast population genetics simulation.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">e1006581</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1006581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D14%26rft.spage%253De1006581%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1006581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1006581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.48" data-doi="10.1534/genetics.105.047985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Chromosome inversions, local adaptation and speciation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">434</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.047985</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047985%26rft_id%253Dinfo%253Apmid%252F16204214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-other" id="cit-9.6.1851.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knaus</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunwald</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> VcfR: an R package to manipulate and visualize VCF format data</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.50" data-doi="10.1186/1746-4811-9-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The advantages and limitations of trait analysis with GWAS: a review.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1746-4811-9-29</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D9%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-9-29%26rft_id%253Dinfo%253Apmid%252F23876160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-9-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23876160&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.51" data-doi="10.1002/bies.201300014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lachance</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP ascertainment bias in population genetic analyses: why it is important, and how to correct it.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">780</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.201300014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D35%26rft.spage%253D780%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201300014%26rft_id%253Dinfo%253Apmid%252F23836388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201300014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23836388&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327793500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1851.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">L. F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Numerical Ecology</span>. <span class="cit-publ-name">Elsevier</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DDISTRIBUTION%2BOF%2BGENE%2BFREQUENCY%2BAS%2BA%2BTEST%2BOF%2BTHE%2BTHEORY%2BOF%2BTHE%2BSELECTIVE%2BNEUTRALITY%2BOF%2BPOLYMORPHISMS%26rft_id%253Dinfo%253Apmid%252F4711903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMS8xNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.54" data-doi="10.1534/genetics.118.301747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Local PCA shows how the effect of population structure differs along the genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301747</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301747%26rft_id%253Dinfo%253Apmid%252F30459280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjExLzEvMjg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.55" data-doi="10.1111/mec.12725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of demographic history and neutral parameterization on the performance of <em>F<sub>ST</sub></em> outlier tests.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2178</span>–<span class="cit-lpage">2192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12725</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D2178%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12725%26rft_id%253Dinfo%253Apmid%252F24655127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24655127&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.56" data-doi="10.1111/mec.13100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The relative power of genome scans to detect local adaptation depends on sampling design and statistical method.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1031</span>–<span class="cit-lpage">1046</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D1031%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13100%26rft_id%253Dinfo%253Apmid%252F25648189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25648189&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.57" data-doi="10.1371/journal.pbio.3000070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stapleton</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Analysis validation has been neglected in the Age of Reproducibility.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">e3000070</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.3000070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D16%26rft.spage%253De3000070%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.3000070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.3000070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1851.58" data-doi="10.5061/dryad.rj0kj10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-source">Data from: The effect of neutral recombination variation on genome scans for selection</span>, <span class="cit-publ-name">Dryad Digital Repository</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.5061/dryad.rj0kj10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DData%2Bfrom%253A%2BThe%2Beffect%2Bof%2Bneutral%2Brecombination%2Bvariation%2Bon%2Bgenome%2Bscans%2Bfor%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.5061%252Fdryad.rj0kj10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5061/dryad.rj0kj10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.59" data-doi="10.1111/1755-0998.12592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">pcadapt: an R package to perform genome scans for selection based on principal component analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">77</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12592</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D17%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Estimation of pairwise relatedness with molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D1753%26rft.epage%253D1766%26rft.atitle%253DEstimation%2Bof%2BPairwise%2BRelatedness%2BWith%2BMolecular%2BMarkers%26rft_id%253Dinfo%253Apmid%252F10430599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi80LzE3NTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.61" data-doi="10.1111/mec.13822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martins</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identifying outlier loci in admixed and in continuous populations using ancestral population differentiation statistics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">5029</span>–<span class="cit-lpage">5042</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13822</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D5029%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.62" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard-Smith</span>  <span class="cit-name-given-names">J. M..</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haigh</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300014634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.63" data-doi="10.1371/journal.pgen.1000686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genealogical interpretation of principal components analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000686</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000686</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000686%26rft.epage%253De1000686%26rft.atitle%253DA%2Bgenealogical%2Binterpretation%2Bof%2Bprincipal%2Bcomponents%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000686%26rft_id%253Dinfo%253Apmid%252F19834557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834557&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.64" data-doi="10.1111/j.1365-294X.2012.05578.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meirmans</span>  <span class="cit-name-given-names">P. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The trouble with isolation by distance.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2839</span>–<span class="cit-lpage">2846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2012.05578.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMeirmans%26rft.auinit1%253DP.%2BG.%26rft.volume%253D21%26rft.issue%253D12%26rft.spage%253D2839%26rft.epage%253D2846%26rft.atitle%253DThe%2Btrouble%2Bwith%2Bisolation%2Bby%2Bdistance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05578.x%26rft_id%253Dinfo%253Apmid%252F22574758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05578.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22574758&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304903100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.65" data-doi="10.1016/j.tree.2013.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics of rapid adaptation by soft selective sweeps.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">669</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2013.08.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D28%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.08.003%26rft_id%253Dinfo%253Apmid%252F24075201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24075201&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326666200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.66" data-doi="10.1016/S0168-9525(01)02409-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Single nucleotide polymorphisms and recombination rate in humans.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(01)02409-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D17%26rft.issue%253D9%26rft.spage%253D481%26rft.epage%253D485%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphisms%2Band%2Brecombination%2Brate%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252801%252902409-X%26rft_id%253Dinfo%253Apmid%252F11525814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(01)02409-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11525814&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170662500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.67" data-doi="10.1038/hdy.2010.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noor</span>  <span class="cit-name-given-names">M. A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Islands of speciation or mirages in the desert? Examining the role of restricted recombination in maintaining species.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">418</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2010.13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D104%26rft.spage%253D418%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000275898200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.68" data-doi="10.1038/ng.139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Interpreting principal component analyses of spatial population genetic variation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">646</span>–<span class="cit-lpage">649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.139</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNovembre%26rft.auinit1%253DJ.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D646%26rft.epage%253D649%26rft.atitle%253DInterpreting%2Bprincipal%2Bcomponent%2Banalyses%2Bof%2Bspatial%2Bpopulation%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.139%26rft_id%253Dinfo%253Apmid%252F18425127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18425127&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.69" data-doi="10.1093/molbev/mss136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A critical assessment of storytelling: gene ontology categories and the importance of validating genomic scans.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3237</span>–<span class="cit-lpage">3248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mss136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss136%26rft_id%253Dinfo%253Apmid%252F22617950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22617950&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309927900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.70" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.71" data-doi="10.1016/j.ajhg.2008.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weale</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Need</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Long-range LD can confound genome scans in admixed populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">135, author reply 135–139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2008.06.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D83%26rft.issue%253D1%26rft.spage%253D132%26rft.epage%253D135%26rft.atitle%253DLong-range%2BLD%2Bcan%2Bconfound%2Bgenome%2Bscans%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.06.005%26rft_id%253Dinfo%253Apmid%252F18606306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18606306&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257784000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.72" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.73" data-doi="10.1038/nrg2880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptation – not by sweeps alone.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">667</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D665%26rft.epage%253D667%26rft.atitle%253DAdaptation%2B-%2Bnot%2Bby%2Bsweeps%2Balone.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2880%26rft_id%253Dinfo%253Apmid%252F20838407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838407&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281911300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.74" data-doi="10.1016/j.cub.2009.11.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetics of human adaptation: hard sweeps, soft sweeps, and polygenic adaptation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">R208</span>–<span class="cit-lpage">R215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2009.11.055</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR208%26rft.epage%253DR215%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bhuman%2Badaptation%253A%2Bhard%2Bsweeps%252C%2Bsoft%2Bsweeps%252C%2Band%2Bpolygenic%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.055%26rft_id%253Dinfo%253Apmid%252F20178769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178769&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.75" data-doi="10.1093/bioinformatics/bty185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Privé</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aschard</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziyatdinov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Efficient analysis of large-scale genome-wide data with two R packages: bigstatsr and bigsnpr.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2781</span>–<span class="cit-lpage">2787</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.76" data-doi="10.1111/mec.13322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rellstab</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gugerli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holderegger</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A practical guide to environmental association analysis in landscape genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">4348</span>–<span class="cit-lpage">4370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D4348%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robbins</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1918</span> <span class="cit-article-title">Some applications of mathematics to breeding problems. III.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRobbins%26rft.auinit1%253DR.%2BB.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D375%26rft.epage%253D389%26rft.atitle%253DSOME%2BAPPLICATIONS%2BOF%2BMATHEMATICS%2BTO%2BBREEDING%2BPROBLEMS%2BIII%26rft_id%253Dinfo%253Apmid%252F17245911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIzLzQvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.78" data-doi="10.1371/journal.pone.0118432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rehmsmeier</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Precision-Recall plot is more informative than the ROC plot when evaluating binary classifiers on imbalanced datasets.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0118432</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0118432</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0118432%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0118432%26rft_id%253Dinfo%253Apmid%252F25738806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0118432&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25738806&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.79" data-doi="10.1534/genetics.105.041947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mechanisms of genetic exchange within the chromosomal inversions of <em>Drosophila pseudoobscura</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1729</span>–<span class="cit-lpage">1739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.041947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.041947%26rft_id%253Dinfo%253Apmid%252F16143625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzE3MjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTg1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.80" data-doi="10.1111/mec.13485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlamp</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Made</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stambler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesebrough</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evaluating the performance of selection scans to detect selective sweeps in domestic dogs.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13485</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D342%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.81" data-doi="10.1534/genetics.115.174912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendes</span>  <span class="cit-name-given-names">F. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Soft shoulders ahead: spurious signatures of soft and partial selective sweeps result from linked hard sweeps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.174912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174912%26rft_id%253Dinfo%253Apmid%252F25716978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzEvMjY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.82" data-doi="10.1371/journal.pgen.1005928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">S/HIC: Robust Identification of Soft and Hard Sweeps Using Machine Learning.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005928</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005928%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005928%26rft_id%253Dinfo%253Apmid%252F26977894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26977894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.83" data-doi="10.1016/j.tig.2017.12.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Supervised machine learning for population genetics: a new paradigm.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2017.12.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D34%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.12.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.12.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.84" data-doi="10.1016/j.cub.2014.01.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwander</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libbrecht</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Supergenes and complex phenotypes.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">R288</span>–<span class="cit-lpage">R294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2014.01.056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D24%26rft.spage%253DR288%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2014.01.056%26rft_id%253Dinfo%253Apmid%252F24698381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2014.01.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24698381&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.85" data-doi="10.1371/journal.pbio.0050171"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoneking</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new approach for using genome scans to detect recent positive selection in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0050171</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DTang%26rft.auinit1%253DK.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De171%26rft.epage%253De171%26rft.atitle%253DA%2Bnew%2Bapproach%2Bfor%2Busing%2Bgenome%2Bscans%2Bto%2Bdetect%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050171%26rft_id%253Dinfo%253Apmid%252F17579516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050171&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17579516&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.86" data-doi="10.1038/hdy.2014.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiggins</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Supergenes and their role in evolution.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2014.20</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D113%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.20%26rft_id%253Dinfo%253Apmid%252F24642887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24642887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.87" data-doi="10.1371/journal.pbio.0040072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A map of recent positive selection in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e72</span>. Erratum: 5: e147.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0040072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DVoight%26rft.auinit1%253DB.%2BF.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De72%26rft.epage%253De72%26rft.atitle%253DA%2Bmap%2Bof%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040072%26rft_id%253Dinfo%253Apmid%252F16494531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16494531&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.6.1851.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Correction: A Map of Recent Positive Selection in the Human Genome.</span> <abbr class="cit-jnl-abbrev">PLOS Biology</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BBiology%26rft.volume%253D4%26rft.spage%253De154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.89" data-doi="10.1111/mec.12938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradburd</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Isolation by environment.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5649</span>–<span class="cit-lpage">5662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12938</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D5649%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.90" data-doi="10.1038/nature05911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Wellcome Trust Case Control Consortium</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">678</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature05911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D661%26rft.epage%253D678%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B14%252C000%2Bcases%2Bof%2Bseven%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05911%26rft_id%253Dinfo%253Apmid%252F17554300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.91" data-doi="10.1086/682949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reliable detection of loci responsible for local adaptation: inference of a null model through trimming the distribution of <em>F<sub>ST</sub></em>.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">S24</span>–<span class="cit-lpage">S36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/682949</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D186%26rft.spage%253DS24%26rft_id%253Dinfo%253Adoi%252F10.1086%252F682949%26rft_id%253Dinfo%253Apmid%252F26656214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/682949&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26656214&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1851.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.92" data-doi="10.1007/BF02459575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Evolution in Mendelian populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D159%26rft.atitle%253DEVOLUTION%2BIN%2BMENDELIAN%2BPOPULATIONS%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02459575%26rft_id%253Dinfo%253Apmid%252F17246615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIxNi8yLzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.93" data-doi="10.1073/pnas.24.7.253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">The distribution of gene frequencies under irreversible mutation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.24.7.253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D253%26rft.epage%253D259%26rft.atitle%253DThe%2BDistribution%2Bof%2BGene%2BFrequencies%2BUnder%2BIrreversible%2BMutation%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.24.7.253%26rft_id%253Dinfo%253Apmid%252F16577841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiIyNC83LzI1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xODUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.94" data-doi="10.1073/pnas.1219381110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic rearrangements and the evolution of clusters of locally adaptive loci.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E1743</span>–<span class="cit-lpage">E1751</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1219381110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1219381110%26rft_id%253Dinfo%253Apmid%252F23610436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzE5L0UxNzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE4NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1851.95" data-doi="10.1111/mec.13611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aeschbacher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bürger</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The evolution of genomic islands by increased establishment probability of linked alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2542</span>–<span class="cit-lpage">2558</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D2542%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/6/1851.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1839" title="Mutation of the ATPase Domain of MutS Homolog-5 (MSH5) Reveals a Requirement for a Functional MutSγ Complex for All Crossovers in Mammalian Meiosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1869" title="An Interaction Network of the Human SEPT9 Established by Quantitative Mass Spectrometry" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/6"><span>Volume 9 Issue 6, June 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/6.cover-source.jpg?itok=Ub13TpI0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1851" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MdN5BT5kUV-UMIHntpanAwR1S5bfKgzqi_qkdA0OLQI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/6/1851" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/6/1851" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/6/1851" class="active" data-icon-position="" data-hide-link-title="0">The Effect of Neutral Recombination Variation on Genome Scans for Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493256" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2-Otqz5UsfW9j32ECHGl9tjdALWQlOZ3AIF71O1K4Ng" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1851.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/6/1851" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wcXfCqXpyEb1pOPUDIFTMN4yAXGwETne5DeiB7VPr2M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/6/1851" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1851.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493256" id="node493256--2" data-pisa="ggg;9/6/1851" data-pisa-master="ggg;g3.119.400088" data-apath="/ggg/9/6/1851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effect of Neutral Recombination Variation on Genome Scans for Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7529-2771" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katie E. Lotterhos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1851-1867;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400088 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493256--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katie E. Lotterhos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Marine and Environmental Sciences, Northeastern University, 430 Nahant Road, Nahant, MA 01908</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katie%2BE.%2BLotterhos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lotterhos%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatie%2BE.%2BLotterhos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7529-2771" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katie E. Lotterhos</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:k.lotterhos@northeastern.edu" class="" data-icon-position="" data-hide-link-title="0">k.lotterhos@northeastern.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/6/1851" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/6/1851&amp;title=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/6/1851&amp;title=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/6/1851&amp;title=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/6/1851&amp;text=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/6/1851&amp;title=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/6/1851&amp;t=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/6/1851&amp;title=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/6/1851&amp;title=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493256/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493256" id="node493256--3" data-pisa="ggg;9/6/1851" data-pisa-master="ggg;g3.119.400088" data-apath="/ggg/9/6/1851.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effect of Neutral Recombination Variation on Genome Scans for Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7529-2771" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katie E. Lotterhos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1851-1867;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400088 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493256--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katie E. Lotterhos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Marine and Environmental Sciences, Northeastern University, 430 Nahant Road, Nahant, MA 01908</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katie%2BE.%2BLotterhos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lotterhos%20KE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatie%2BE.%2BLotterhos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7529-2771" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katie E. Lotterhos</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:k.lotterhos@northeastern.edu" class="" data-icon-position="" data-hide-link-title="0">k.lotterhos@northeastern.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1851&amp;rft.epage=1867&amp;rft.atitle=The%20Effect%20of%20Neutral%20Recombination%20Variation%20on%20Genome%20Scans%20for%20Selection&amp;rft.volume=9&amp;rft.issue=6&amp;rft.date=2019-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lotterhos%2C+Katie+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493256/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493256/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493256/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493256/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493256/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493256/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493256/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493256/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493256/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493256/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493256/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493256/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/6/1851" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/6/1851.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/6/1851.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/6/1851.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/6/1851.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
