<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/6/1975" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/6/1975.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/6/1975.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/6/1975.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/6/1975/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/6/1975.atom" />
<meta name="HW.pisa" content="ggg;9/6/1975" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes" />
<meta name="DC.Identifier" content="10.1534/g3.119.400154" />
<meta name="DC.Date" content="2019-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Yu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="With the advent of high-throughput phenotyping platforms, plant breeders have a means to assess many traits for large breeding populations. However, understanding the genetic interdependencies among high-dimensional traits in a statistically robust manner remains a major challenge. Since multiple phenotypes likely share mutual relationships, elucidating the interdependencies among economically important traits can better inform breeding decisions and accelerate the genetic improvement of plants. The objective of this study was to leverage confirmatory factor analysis and graphical modeling to elucidate the genetic interdependencies among a diverse agronomic traits in rice. We used a Bayesian network to depict conditional dependencies among phenotypes, which can not be obtained by standard multi-trait analysis. We utilized Bayesian confirmatory factor analysis which hypothesized that 48 observed phenotypes resulted from six latent variables including grain morphology, morphology, flowering time, physiology, yield, and morphological salt response. This was followed by studying the genetics of each latent variable, which is also known as factor, using single nucleotide polymorphisms. Bayesian network structures involving the genomic component of six latent variables were established by fitting four algorithms ( i.e. , Hill Climbing, Tabu, Max-Min Hill Climbing, and General 2-Phase Restricted Maximization algorithms). Physiological components influenced the flowering time and grain morphology, and morphology and grain morphology influenced yield. In summary, we show the Bayesian network coupled with factor analysis can provide an effective approach to understand the interdependence patterns among phenotypes and to predict the potential influence of external interventions or selection related to target traits in the interrelated complex traits systems." />
<meta name="DC.Contributor" content="Haipeng Yu" />
<meta name="DC.Contributor" content="Malachy T. Campbell" />
<meta name="DC.Contributor" content="Qi Zhang" />
<meta name="DC.Contributor" content="Harkamal Walia" />
<meta name="DC.Contributor" content="Gota Morota" />
<meta name="article:published_time" content="2019-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;With the advent of high-throughput phenotyping platforms, plant breeders have a means to assess many traits for large breeding populations. However, understanding the genetic interdependencies among high-dimensional traits in a statistically robust manner remains a major challenge. Since multiple phenotypes likely share mutual relationships, elucidating the interdependencies among economically important traits can better inform breeding decisions and accelerate the genetic improvement of plants. The objective of this study was to leverage confirmatory factor analysis and graphical modeling to elucidate the genetic interdependencies among a diverse agronomic traits in rice. We used a Bayesian network to depict conditional dependencies among phenotypes, which can not be obtained by standard multi-trait analysis. We utilized Bayesian confirmatory factor analysis which hypothesized that 48 observed phenotypes resulted from six latent variables including grain morphology, morphology, flowering time, physiology, yield, and morphological salt response. This was followed by studying the genetics of each latent variable, which is also known as factor, using single nucleotide polymorphisms. Bayesian network structures involving the genomic component of six latent variables were established by fitting four algorithms (&lt;i&gt;i.e.&lt;/i&gt;, Hill Climbing, Tabu, Max-Min Hill Climbing, and General 2-Phase Restricted Maximization algorithms). Physiological components influenced the flowering time and grain morphology, and morphology and grain morphology influenced yield. In summary, we show the Bayesian network coupled with factor analysis can provide an effective approach to understand the interdependence patterns among phenotypes and to predict the potential influence of external interventions or selection related to target traits in the interrelated complex traits systems.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/06/01" />
<meta name="citation_mjid" content="ggg;9/6/1975" />
<meta name="citation_id" content="9/6/1975" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/6/1975" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/6/1975.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/6/1975.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/6/1975.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400154" />
<meta name="citation_pmid" content="30992319" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1975" />
<meta name="citation_lastpage" content="1986" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Haipeng Yu" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8923-9733" />
<meta name="citation_author" content="Malachy T. Campbell" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8257-3595" />
<meta name="citation_author" content="Qi Zhang" />
<meta name="citation_author_institution" content="Department of Statistics, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author" content="Harkamal Walia" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9712-5824" />
<meta name="citation_author" content="Gota Morota" />
<meta name="citation_author_institution" content="Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061" />
<meta name="citation_author_email" content="morota@vt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3567-6911" />
<meta name="citation_reference" content="citation_title=Principles of plant genetics and breeding;citation_author=G. Acquaah;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. L. Araus;citation_author=J. E. Cairns;citation_title=Field high-throughput phenotyping: the new crop breeding frontier.;citation_pages=52-61;citation_volume=19;citation_year=2014;citation_pmid=24139902;citation_doi=10.1016/j.tplants.2013.09.008" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. L. Araus;citation_author=S. C. Kefauver;citation_author=M. Zaman-Allah;citation_author=M. S. Olsen;citation_author=J. E. Cairns;citation_title=Translating high-throughput phenotyping into genetic gain.;citation_pages=451-466;citation_volume=23;citation_year=2018;citation_doi=10.1016/j.tplants.2018.02.001" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Growth Regul.;citation_author=A. Bhattacharya;citation_author=S. Kourmpetli;citation_author=M. R. Davey;citation_title=Practical applications of manipulating plant architecture by regulating gibberellin metabolism.;citation_pages=249-256;citation_volume=29;citation_year=2010;citation_doi=10.1007/s00344-009-9126-3" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=P. B. Brewer;citation_author=H. Koltai;citation_author=C. A. Beveridge;citation_title=Diverse roles of strigolactones in plant development.;citation_volume=6;citation_year=2013;citation_pmid=23155045;citation_doi=10.1093/mp/sss130" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Graph. Stat.;citation_author=S. P. Brooks;citation_author=A. Gelman;citation_title=General methods for monitoring convergence of iterative simulations.;citation_pages=434-455;citation_volume=7;citation_year=1998;citation_doi=10.2307/1390675" />
<meta name="citation_reference" content="citation_title=Confirmatory factor analysis for applied research;citation_author=T. A. Brown;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=C. S. Byrt;citation_author=J. D. Platten;citation_author=W. Spielmeyer;citation_author=R. A. James;citation_author=E. S. Lagudah;citation_title=Hkt1; 5-like cation transporters linked to na+ exclusion loci in wheat, nax2 and kna1.;citation_volume=143;citation_year=2007;citation_pmid=17322337;citation_doi=10.1104/pp.106.093476" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=L. Cabrera-Bosquet;citation_author=C. Fournier;citation_author=N. Brichet;citation_author=C. Welcker;citation_author=B. Suard;citation_title=High-throughput estimation of incident light, light interception and radiation-use efficiency of thousands of plants in a phenotyping platform.;citation_volume=212;citation_year=2016;citation_doi=10.1111/nph.14027" />
<meta name="citation_reference" content="citation_title=Some new algorithms for computing maximum likelihood estimates of variance components;citation_author=T. P. Callanan;citation_author=D. A. Harville;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. T. Campbell;citation_author=N. Bandillo;citation_author=F. R. A. Al Shiblawi;citation_author=S. Sharma;citation_author=K. Liu;citation_title=Allelic variants of oshkt1; 1 underlie the divergence between indica and japonica subspecies of rice (oryza sativa) for root sodium content.;citation_pages=e1006823;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1006823" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Crain;citation_author=S. Mondal;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Poland;citation_title=Combining high-throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.05.0043" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. de los Campos;citation_author=D. Gianola;citation_title=Factor analysis models for structuring covariance matrices of additive genetic effects: a Bayesian implementation.;citation_pages=481-494;citation_volume=39;citation_year=2007;citation_issue=5;citation_pmid=17897592;citation_doi=10.1051/gse:20070016" />
<meta name="citation_reference" content="citation_journal_title=Articles;citation_author=M. Denwood;citation_title=runjags: An r package providing interface utilities, model templates, parallel computing methods and additional distributions for mcmc models in jags. Journal of Statistical Software;citation_pages=1-25;citation_volume=71;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Regist.;citation_author=G. C. Eizenga;citation_author=M. Ali;citation_author=R. J. Bryant;citation_author=K. M. Yeater;citation_author=A. M. McClung;citation_title=Registration of the rice diversity panel 1 for genomewide association studies.;citation_pages=109-116;citation_volume=8;citation_year=2014;citation_doi=10.3198/jpr2013.03.0013crmp" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=R. T. Furbank;citation_author=M. Tester;citation_title=Phenomics--technologies to relieve the phenotyping bottleneck.;citation_pages=635-644;citation_volume=16;citation_year=2011;citation_issue=12;citation_pmid=22074787;citation_doi=10.1016/j.tplants.2011.09.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=M. A. Toro;citation_author=H. Naya;citation_author=C.-C. Schön;citation_title=Do molecular markers inform about pleiotropy?;citation_volume=201;citation_year=2015;citation_pmid=26205989;citation_doi=10.1534/genetics.115.179978" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Biol.;citation_author=R. Hay;citation_title=Harvest index: a review of its use in plant breeding and crop physiology.;citation_pages=197-216;citation_volume=126;citation_year=1995;citation_doi=10.1111/j.1744-7348.1995.tb05015.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. Henderson;citation_author=R. Quaas;citation_title=Multiple Trait Evaluation Using Relatives' Records;citation_pages=1188-1197;citation_volume=43;citation_year=1976;citation_issue=6;citation_doi=10.2527/jas1976.4361188x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. M. Hickey;citation_author=T. Chiurugwi;citation_author=I. Mackay;citation_author=W. Powell;citation_author=A. Eggen;citation_title=Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.;citation_pages=1297-1303;citation_volume=49;citation_year=2017;citation_pmid=28854179;citation_doi=10.1038/ng.3920" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=T. Horie;citation_author=F. Hauser;citation_author=J. I. Schroeder;citation_title=HKT transporter-mediated salinity resistance mechanisms in Arabidopsis and monocot crop plants.;citation_pages=660-668;citation_volume=14;citation_year=2009;citation_issue=12;citation_pmid=19783197;citation_doi=10.1016/j.tplants.2009.08.009" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=R. Huang;citation_author=L. Jiang;citation_author=J. Zheng;citation_author=T. Wang;citation_author=H. Wang;citation_title=Genetic bases of rice grain shape: so many genes, so little known.;citation_pages=218-226;citation_volume=18;citation_year=2013;citation_pmid=23218902;citation_doi=10.1016/j.tplants.2012.11.001" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=T. Sang;citation_author=Q. Zhao;citation_author=Q. Feng;citation_author=Y. Zhao;citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces.;citation_pages=961-967;citation_volume=42;citation_year=2010;citation_issue=11;citation_pmid=20972439;citation_doi=10.1038/ng.695" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_title=Multiple-trait genomic selection methods increase genetic value prediction accuracy.;citation_volume=192;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Jiao;citation_author=Y. Wang;citation_author=D. Xue;citation_author=J. Wang;citation_author=M. Yan;citation_title=Regulation of OsSPL14 by OsmiR156 defines ideal plant architecture in rice.;citation_pages=541-544;citation_volume=42;citation_year=2010;citation_issue=6;citation_pmid=20495565;citation_doi=10.1038/ng.591" />
<meta name="citation_reference" content="citation_journal_title=Psychometrika;citation_author=K. G. Jöreskog;citation_title=A general approach to confirmatory maximum likelihood factor analysis.;citation_pages=183-202;citation_volume=34;citation_year=1969;citation_doi=10.1007/BF02289343" />
<meta name="citation_reference" content="citation_title=Basic and advanced Bayesian structural equation modeling: With applications in the medical and behavioral sciences;citation_author=S.-Y. Lee;citation_author=X.-Y. Song;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=W. Liang;citation_author=F. Shang;citation_author=Q. Lin;citation_author=C. Lou;citation_author=J. Zhang;citation_title=Tillering and panicle branching genes in rice.;citation_pages=1-5;citation_volume=537;citation_year=2014;citation_pmid=24345551;citation_doi=10.1016/j.gene.2013.11.058" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S.-F. Lo;citation_author=S.-Y. Yang;citation_author=K.-T. Chen;citation_author=Y.-I. Hsing;citation_author=J. A. Zeevaart;citation_title=A novel class of gibberellin 2-oxidases control semidwarfism, tillering, and root development in rice.;citation_volume=20;citation_year=2008;citation_pmid=18952778;citation_doi=10.1105/tpc.108.060913" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. R. McCouch;citation_author=M. H. Wright;citation_author=C.-W. Tung;citation_author=L. G. Maron;citation_author=K. L. McNally;citation_title=Open access resources for genome-wide association mapping in rice.;citation_pages=10532;citation_volume=7;citation_year=2016;citation_pmid=26842267;citation_doi=10.1038/ncomms10532" />
<meta name="citation_reference" content="citation_journal_title=Articles;citation_author=E. Merkle;citation_author=Y. Rosseel;citation_title=blavaan: Bayesian structural equation models via parameter expansion. Journal of Statistical Software;citation_pages=1-30;citation_volume=85;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. Miura;citation_author=M. Ikeda;citation_author=A. Matsubara;citation_author=X.-J. Song;citation_author=M. Ito;citation_title=OsSPL14 promotes panicle branching and higher grain productivity in rice.;citation_pages=545-549;citation_volume=42;citation_year=2010;citation_issue=6;citation_pmid=20495564;citation_doi=10.1038/ng.592" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=G. Morota;citation_author=D. Gianola;citation_title=Evaluation of linkage disequilibrium in wheat with an l1-regularized sparse markov network.;citation_pages=1991-2002;citation_volume=126;citation_year=2013;citation_pmid=23661079;citation_doi=10.1007/s00122-013-2112-y" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=G. Morota;citation_author=B. Valente;citation_author=G. Rosa;citation_author=K. Weigel;citation_author=D. Gianola;citation_title=An assessment of linkage disequilibrium in Holstein cattle using a Bayesian network.;citation_pages=474-487;citation_volume=129;citation_year=2012;citation_issue=6;citation_pmid=23148973;citation_doi=10.1111/jbg.12002" />
<meta name="citation_reference" content="citation_title=Linear models for the prediction of animal breeding values;citation_author=R. A. Mrode;citation_year=2014;citation_doi=10.1079/9781780643915.0000" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=R. Munns;citation_author=R. A. James;citation_author=B. Xu;citation_author=A. Athman;citation_author=S. J. Conn;citation_title=Wheat grain yield on saline soils is improved by an ancestral Na+ transporter gene.;citation_pages=360-364;citation_volume=30;citation_year=2012;citation_issue=4;citation_pmid=22407351;citation_doi=10.1038/nbt.2120" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=R. Munns;citation_author=M. Tester;citation_title=Mechanisms of salinity tolerance.;citation_pages=651-681;citation_volume=59;citation_year=2008;citation_pmid=18444910;citation_doi=10.1146/annurev.arplant.59.032607.092911" />
<meta name="citation_reference" content="citation_title=Learning bayesian networks;citation_author=R. E. Neapolitan;citation_volume=Vol. 38;citation_year=2004" />
<meta name="citation_reference" content="citation_title=Causality: Models, Reasoning and Inference;citation_author=J. Pearl;citation_year=2009;citation_doi=10.1017/CBO9780511803161" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=F. Peñagaricano;citation_author=B. Valente;citation_author=J. Steibel;citation_author=R. Bates;citation_author=C. Ernst;citation_title=Searching for causal networks involving latent variables in complex traits: application to growth, carcass, and meat quality traits in pigs.;citation_pages=4617-4623;citation_volume=93;citation_year=2015;citation_doi=10.2527/jas.2015-9213" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=S. Peng;citation_author=G. S. Khush;citation_author=P. Virk;citation_author=Q. Tang;citation_author=Y. Zou;citation_title=Progress in ideotype breeding to increase rice yield potential.;citation_pages=32-38;citation_volume=108;citation_year=2008;citation_doi=10.1016/j.fcr.2008.04.001" />
<meta name="citation_reference" content="citation_author=M. Plummer;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Redoña;citation_author=D. Mackill;citation_title=Quantitative trait locus analysis for rice panicle and grain characteristics.;citation_pages=957-963;citation_volume=96;citation_year=1998;citation_doi=10.1007/s001220050826" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z.-H. Ren;citation_author=J.-P. Gao;citation_author=L.-G. Li;citation_author=X.-L. Cai;citation_author=W. Huang;citation_title=A rice quantitative trait locus for salt tolerance encodes a sodium transporter.;citation_pages=1141-1146;citation_volume=37;citation_year=2005;citation_issue=10;citation_pmid=16155566;citation_doi=10.1038/ng1643" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. J. Rosa;citation_author=B. D. Valente;citation_author=G. de los Campos;citation_author=X.-L. Wu;citation_author=D. Gianola;citation_title=Inferring causal phenotype networks using structural equation models.;citation_pages=6-6;citation_volume=43;citation_year=2011;citation_pmid=21310061;citation_doi=10.1186/1297-9686-43-6" />
<meta name="citation_reference" content="citation_journal_title=Articles;citation_author=M. Scutari;citation_title=Learning bayesian networks with the bnlearn r package. Journal of Statistical Software;citation_pages=1-22;citation_volume=35;citation_year=2010" />
<meta name="citation_reference" content="citation_title=Bayesian networks: with examples in R;citation_author=M. Scutari;citation_author=J.-B. Denis;citation_year=2014;citation_doi=10.1201/b17065" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=N. Shakoor;citation_author=S. Lee;citation_author=T. C. Mockler;citation_title=High throughput phenotyping to accelerate crop breeding and monitoring of diseases in the field.;citation_pages=184-192;citation_volume=38;citation_year=2017;citation_doi=10.1016/j.pbi.2017.05.006" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Sun;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_title=Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.;citation_pages=1;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.11.0111" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=M. A. Tanner;citation_author=W. H. Wong;citation_title=The calculation of posterior distributions by data augmentation.;citation_pages=528-540;citation_volume=82;citation_year=1987;citation_doi=10.2307/2289457" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=M. J. Thomson;citation_author=M. de Ocampo;citation_author=J. Egdane;citation_author=M. A. Rahman;citation_author=A. G. Sajise;citation_title=Characterizing the saltol quantitative trait locus for salinity tolerance in rice.;citation_pages=148-160;citation_volume=3;citation_year=2010;citation_doi=10.1007/s12284-010-9053-8" />
<meta name="citation_reference" content="citation_title=Abiotic Stress Adaptation in Plants;citation_author=M. J. Thomson;citation_author=A. M. Ismail;citation_author=S. R. McCouch;citation_author=D. J. Mackill;citation_title=Marker assisted breeding;citation_year=2009;citation_doi=10.1007/978-90-481-3112-9_20" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Töpner;citation_author=G. J. Rosa;citation_author=D. Gianola;citation_author=C.-C. Schön;citation_title=Bayesian networks illustrate genomic and residual trait connections in maize (Zea mays L.). G3: Genes, Genomes;citation_pages=2779-2789;citation_volume=7;citation_year=2017;citation_doi=10.1534/g3.117.044263" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Umehara;citation_author=A. Hanada;citation_author=S. Yoshida;citation_author=K. Akiyama;citation_author=T. Arite;citation_title=Inhibition of shoot branching by new terpenoid plant hormones.;citation_pages=195-200;citation_volume=455;citation_year=2008;citation_issue=7210;citation_pmid=18690207;citation_doi=10.1038/nature07272" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. D. Valente;citation_author=G. Morota;citation_author=F. Peñagaricano;citation_author=D. Gianola;citation_author=K. Weigel;citation_title=The causal meaning of genomic predictors and how it affects construction and comparison of genome-enabled selection models.;citation_volume=200;citation_year=2015;citation_pmid=25908318;citation_doi=10.1534/genetics.114.169490" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=A. Vazquez;citation_author=D. Bates;citation_author=G. Rosa;citation_author=D. Gianola;citation_author=K. Weigel;citation_title=An r package for fitting generalized linear mixed models in animal breeding 1.;citation_volume=88;citation_year=2010;citation_pmid=19820058;citation_doi=10.2527/jas.2009-1952" />
<meta name="citation_reference" content="citation_author=T. Verma;citation_author=J. Pearl;citation_pages=255-270;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=Y. Wang;citation_author=J. Li;citation_title=Genes controlling plant architecture.;citation_pages=123-129;citation_volume=17;citation_year=2006;citation_issue=2;citation_pmid=16504498;citation_doi=10.1016/j.copbio.2006.02.004" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Xavier;citation_author=B. Hall;citation_author=S. Casteel;citation_author=W. Muir;citation_author=K. M. Rainey;citation_title=Using unsupervised learning techniques to assess interactions among complex traits in soybeans.;citation_pages=200;citation_volume=213;citation_year=2017;citation_doi=10.1007/s10681-017-1975-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=W.-H. Yan;citation_author=P. Wang;citation_author=H.-X. Chen;citation_author=H.-J. Zhou;citation_author=Q.-P. Li;citation_title=A major qtl, ghd8, plays pleiotropic roles in regulating grain productivity, plant height, and heading date in rice.;citation_volume=4;citation_year=2011;citation_pmid=21148627;citation_doi=10.1093/mp/ssq070" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C.-W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=F. Zhou;citation_author=Q. Lin;citation_author=L. Zhu;citation_author=Y. Ren;citation_author=K. Zhou;citation_title=D14–scf d3-dependent degradation of d53 regulates strigolactone signalling.;citation_pages=406-410;citation_volume=504;citation_year=2013;citation_pmid=24336215;citation_doi=10.1038/nature12878" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="twitter:description" content="With the advent of high-throughput phenotyping platforms, plant breeders have a means to assess many traits for large breeding populations. However, understanding the genetic interdependencies among high-dimensional traits in a statistically robust manner remains a major challenge. Since multiple phenotypes likely share mutual relationships, elucidating the interdependencies among economically important traits can better inform breeding decisions and accelerate the genetic improvement of plants. The objective of this study was to leverage confirmatory factor analysis and graphical modeling to elucidate the genetic interdependencies among a diverse agronomic traits in rice. We used a Bayesian network to depict conditional dependencies among phenotypes, which can not be obtained by standard multi-trait analysis. We utilized Bayesian confirmatory factor analysis which hypothesized that 48 observed phenotypes resulted from six latent variables including grain morphology, morphology, flowering time, physiology, yield, and morphological salt response. This was followed by studying the genetics of each latent variable, which is also known as factor, using single nucleotide polymorphisms. Bayesian network structures involving the genomic component of six latent variables were established by fitting four algorithms ( i.e. , Hill Climbing, Tabu, Max-Min Hill Climbing, and General 2-Phase Restricted Maximization algorithms). Physiological components influenced the flowering time and grain morphology, and morphology and grain morphology influenced yield. In summary, we show the Bayesian network coupled with factor analysis can provide an effective approach to understand the interdependence patterns among phenotypes and to predict the potential influence of external interventions or selection related to target traits in the interrelated complex traits systems." />
<meta name="og-title" property="og:title" content="Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/6/1975" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="With the advent of high-throughput phenotyping platforms, plant breeders have a means to assess many traits for large breeding populations. However, understanding the genetic interdependencies among high-dimensional traits in a statistically robust manner remains a major challenge. Since multiple phenotypes likely share mutual relationships, elucidating the interdependencies among economically important traits can better inform breeding decisions and accelerate the genetic improvement of plants. The objective of this study was to leverage confirmatory factor analysis and graphical modeling to elucidate the genetic interdependencies among a diverse agronomic traits in rice. We used a Bayesian network to depict conditional dependencies among phenotypes, which can not be obtained by standard multi-trait analysis. We utilized Bayesian confirmatory factor analysis which hypothesized that 48 observed phenotypes resulted from six latent variables including grain morphology, morphology, flowering time, physiology, yield, and morphological salt response. This was followed by studying the genetics of each latent variable, which is also known as factor, using single nucleotide polymorphisms. Bayesian network structures involving the genomic component of six latent variables were established by fitting four algorithms ( i.e. , Hill Climbing, Tabu, Max-Min Hill Climbing, and General 2-Phase Restricted Maximization algorithms). Physiological components influenced the flowering time and grain morphology, and morphology and grain morphology influenced yield. In summary, we show the Bayesian network coupled with factor analysis can provide an effective approach to understand the interdependence patterns among phenotypes and to predict the potential influence of external interventions or selection related to target traits in the interrelated complex traits systems." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
    <title>Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"HScvW_NGddoDQG99-XvVDDttK4lCRSkByvy3KohgexA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493251","apath":"\/ggg\/9\/6\/1975.atom","pisa":"ggg;9\/6\/1975","ac":{"\/ggg\/9\/6\/1975.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/6\/1975.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1975"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1975"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/6\/1975":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Dpt0TxdLYdx\\\\\\\/W+WXIj+hYR2VaV2C7hYp5FHMmmlQXu6ULTejXJJ4iq2Tw62Li6DyiujSxJmuevn0L7HHUO2vYOGW8WIhYXUTFBmScnecwm9zhmXGq2spx9g0AnZiLNfgXx9yeaQmB+u9DZwbHDjYe9+XJOFiCXAXDEL6Ge32zHoN7nrTEucINSVAafI2uY6FZPkvm\\\\\\\/hr1BnUG89is50Ka6ZOYeoZLctjvlGUKk5Uq7mefrQsjQ2JYSlVmkIyeWlgVmCk47MICIjrwwooDXs0OCa9PVCxEYXGbi3pvae9kMdHUebBhLQS334o5XgrwGI3vFkzLPozlyGvgxS8SV8IhOTWDGCNN+P6t5jy8s7Gi4arnV085LuM+fwd\\\\\\\/mboUJrFWu5HmFCSdYicKwtZhiIa\\\\\\\/6WqzVYzcWEUpkIcblHK+mT91tTnDS+A6r8dKPZsQ3r70g+OJozZIQZglWv9SAkMBd\\\\\\\/4ykS1uH2Jo\\\\\\\/m6mhfprxIv7Zs\\\\\\\/8I7mvO9ib\\\\\\\/ARA4J0M2gWorJs0EpFkTouRPs7HV28vBKv4vm0gQA9cAbYTsyMRW268\\\\\\\/erwl5sRBz1nP37l0uUVfG\\\\\\\/6l8tb5nJTURH4Mv2DsGOTR2EDgBaalzxGYT+M\\\\\\\/naSrqGH6pTXsbno\\\\\\\/PpwHTYH98XewEm7MTRBLoUrZaeYCdQbfxTdVYiMrgW7olj32wMBYNBV8fUAxnAVlub3DiRPoumqsAxoCd3H9L1meQGkdAixxZBYpAdk\\\\\\\/e3Ly420c8+JeqioXziQXmk\\\\\\\/SANh1KpklmejSRUxKJmF5dKffbwXXQA+KmLYvroUqaTiMbmx4Yq5ZmFJsxkUn++L\\\\\\\/1CFxebCyVXjbRKma7+uvAtCRHVFOvpdympnT\\\\\\\/o4g+HADOs5ODzmm\\\\\\\/+70x36BfLa74qcUFx+vXHizJAC4RvJf6kTGMUm+Rw9AYyLBc\\\\\\\/74LrQGj25klP3DeHSQgxgInlL5TASbpy3+pZkoYmY5WFxRNLX8ilwwcIIG3MXqxgYII53gQZktCYMZXc1H3RjSmENPNNGlnmRVqDwBTWYWOBtNGkKtmQFzcOVwp62u\\\\\\\/2\\\\\\\/pl3undY7BioUUXWIUc\\\\\\\/UD98DIyNGcTjFQbYxytha4twVDzQiAG520dMvqnfEY+2oxNWMBbLoMEnJIoipfShjo2MZHH8FzYyh5vbA0\\\\\\\/6BZWKcWGlNwkRxBePC42YknDi22WX3B1Q8byQVhz05pmpowMep4mC0sbYRM5dezQy8cw6qIXdqFLGgA0ZWgbUAOABJEjZWom5WWcVsCyFxElLB2HF3i6p\\\u0022,\\\u0022iv\\\u0022:\\\u0022fI8jAKKSJSiekjvm16gbcg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a32576dc5dd0233ad7aadd289558629656f9d7f06ba3d08df4178b3b856635ab\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RE\\\\\\\/M+sduMDB53XnH8qxJe8Pwzn61s787E5MraDCebhdnREWsYQjq422ndpAa67mKh5TTdMYvxA084kKpvyASeiXgafZlDbT9eWrSXCLcK7pjjNi\\\\\\\/Hwq7ODSUsGF4HMhlBt\\\\\\\/ijls8OeWhGA69mm2dPegFkQe90JphG448EGO9w9\\\\\\\/j0mfeV9ZSc\\\\\\\/o0pfAZPUw44bH\\\\\\\/3UWo5NqTGm0vYAh5DOv1pDsuB2eKFJtGCGT7FI1CDfnOYz7J7Nm8dSaSTP2V7s7cy\\\\\\\/Q7hDF0\\\\\\\/a2VQ7VSfOkDZGW7w1WoK5jZaw9oKldhYxff2VBY1vwfcqWGOvbm\\\\\\\/08J7qVQsP+kvzzschlC+Sw\\\\\\\/ee\\\\\\\/xC+QijKmZYfC\\\\\\\/JIliLzPYfn9TYn4BhHkWY67thpHB0Zf1IrF\\\\\\\/Yn\\\\\\\/miarizuXBlQyPj6LqSYWhh7Pvr835ZCSsODAXttYixBmI+gyJghniV92qb6MLz49VamXiJYlq8XMJEja5IUJnFu5HNSjwVg\\\\\\\/ES9uCTSHcBIG7IRsrlV+KcAvPMv\\\\\\\/LqyqpGT7NIu0ERj3ncXhkt0Gpzeu08yBdQy7sX3VnMM5SC1FaRyhy7XLTdv6uwBEOC7F7OEAq+K18rYSdpgU0uyruQTeuUB8LXZCZZyymXQTOwfECj3FDtwIdcsVsMf8R3ywRY3KMBXW4\\\\\\\/B9rVbtdp\\\\\\\/l+A0TB\\\\\\\/h4V3QeGj14Elo5mJcJboMu0V81Vrsbhecjwiz77zAzjp99T9JFZFnZICjrVBkaRrEvRjR9YTqhR22ohOB12qQXoK5ANakeeGJF\\\\\\\/hM0k6uZL+srTrEpYQFlaCwldto5yXy07MkYMVXhdBpPuIv5rvXVy+pZKdUOQ3qNt1ZyFQn4XtceKrm1sGvsJBtEi\\\\\\\/l\\\\\\\/ixQcLt3Nyga8dEWweoo5Z8wN4rBC8pfzXzCISEYzG2KNpY315xbaedQ9SChLCUGOKFRegUJqOUlPY5v1tuE+6o+j\\\\\\\/v7XCaeseruYyMUAZ\\\\\\\/G21RIAvpYfNBoj7wicNOkXYmV3ECKkOo2NBsEgrCH\\\\\\\/yrrvFRT5yzxQbsYsTB7Fq61zs6qazQhG9x5yVOJJpH5dTuWdsXF0wzEJBjXbrtCFjMN22iCOxOhVDM7ky14V7X9w5SeKrIc639J9IGN+dclxZ2Xdoby5ukAhzBQseffmuB10Wm3ah7araQuHv9wyFk69VtT0daPiCxLmwy9sc8NvB+DzqrQQnUpJj0JOt8NwlYYjWjgAN4gayk8stWC+XZEtJRT+IJ6HQHAacI9OIdnEj7qcJNQstxAtlHuEa1aprC4gtZsiXH34fiY6XSx4TY4UE5Ds0KwJYNsE43+VAxSkywpxYDeK5yJDErgVVx5Epx8nk9tdV\\\\\\\/x7DgsUdwfIKdGr5DDMKeUm2GrBRqBh92RE0h93ZZ1rBZBVgva1U8OVl35yfRZ1KMaRD0NQkT8Gx1oX3\\\\\\\/ftijjZdAFBe3B55q5OXgiH4nevR2VqXbAg5GAqv\\\\\\\/21HtHALwiBRCdZY2HI5McwzC4dO1fxiQTlYAoJu4s6JxxAjBfomHFVuy1hOPq8ADeAfxx8sQdAJA1qMVM\\\\\\\/fkaTsHyXcUjrfvewktfFGnkHTGplQNFawNHJa8cOd4K7w52VMvgvBNrw+NV8X5+xkCFr5Ez5MC5hgBl9lHCYp93dvWYFZD4FVcqdUi1UQhyNoE882S0OaP+iNepJhpvK5Z5s7HhVlFzaZYtFPKCLFmy4w97ZWVAWWlyfJBbtMu51rnUWHSi9Pp5rRP+jahck6Hchr\\\\\\\/5nU+sFvzH3a\\\\\\\/JU\\\\\\\/e\\\\\\\/puOYyYb79GZ6kZ18BT0NY9Lj+XtiEAPh8Ifo51W4pevMEIOlbCntPQMaNJ7+CF\\\\\\\/dWkmEcY6cVZ+yP2Vw9txS29ICQMrjDx0JHmTE4mWVy8RnN3h2TGjs0MLVX\\\\\\\/Z6SZ0uoNN+xhqT5ilmbaYoauxmHVh0dbyKX+EcUus60FRArdUTKRfs6nXGUsxd2S2OtQe29soQrmugwdPeYJzPPWOxG9MajDVZc+kJR1vrcYpk4Tzd5K9DWigDUKyFJLElHo7AQnDu2WJcIArTAkmvWpq0tdA6qcCR65XKFC2HjU+cSkrkglG\\\\\\\/JrpfFOdngsEL9P\\\\\\\/gX20qdfLFYzrokPjffThmgSe5l5W5VGFzJKohbKEaWf+DGlVlVWtHxw3SDzlZRlRMYNX0qaen8m2Y0QZWO90I2kybpOZWFN5y00M0nVbS1qM6Jb6UsT5FN6x7Ijc2YoxqLV7hs7ynfrrpoR9OV+MQeqDYLD6BBMLa86RSIEyDhfRHIZOTjUFtP5W7dFn0ZI\\\\\\\/BtE1NZ\\\\\\\/rqX3yR5Yyq6NgtmFEo1qJWNEpKHmHkgWj+wpdweC4s92QiMR8JdVXWSk8fHg2q7AVzANpD12B5Oork1j8yIeSDm\\\\\\\/98rCIMGjKa+txqncu6NLyfcMBN0E3mc4LqVnLSNoXPxda9muNuwhPPBaD4jjtaklTWkv8lNLv9gKVCsMwfM7Mgck1yLIC8K0E\\\\\\\/6CQ4Xwh\\\\\\\/f7aiL75Dzg1zCOaPsRcsPT0OA9xr7fT2D0qW+VOicz+epZVK34jaF1hRgqE10z1abcuJAj9MTDOF5lmGWKpKg4mjt8eCaC5U\\\\\\\/lOBwflYkEtUNT08n4S39zrW7xTtY3kmGcIeQPrM2YIgncNNmGSMYaaGdxyAgHqQVyVo4DG\\\\\\\/SrVu9bDsPHBuu0efh1X1fFeAjjxC3l\\\\\\\/RGm7wEGP\\\\\\\/otoxcgCyr5Fpzt6SRC14rzhFQ6JMSHqrhF4yMG1\\\\\\\/wG11T+1VWPLdY53PRPksgqT+ejAz1OKFsGZIyIgHXSUJ\\\\\\\/POo3OI\\\\\\\/oISxOlgbIPeAgiHiLMqrXZyyytH5uC1EelfTqCmVtCFhavkQkKaTtUDzipq4fPPkgdaCs4qhApmpnb52reRIjR2JTLCv2mM5z8x3NQ+9XSFISgPmq4DPecjkMBfB4sdEyIqIhETt9FmVKhhCXz+goOzkKZUDlqIdD8pt\\\\\\\/UUZmbDEZnhb+n\\\\\\\/VL+p7hxaFOqYMftLPU7CjYtS2dVJoC\\\\\\\/NX3B62XmNS7vdNZKbjD2GR0+7Bu8RkENaV4S+OcTcxEwhlqwpbv5JvEo3ylvf1iwZJ8F7dY9QhfQCAQw8t1Nh1+jdlv38N8IdJzqtnB9mFyIDeXkzj+GfUfF3SxsyTHBuWfSRSZtpyT3nK9oqh0sNGsO8ZeG+PM4+TwGlWAh6nyGf+Q3yxyqm0e33VcvCAe58\\\\\\\/2NNfckge4Xp1b3LAnfvK1i+wqkgGKlEwuQRsbzl\\\\\\\/ArripmIRMDd2iYXkRNYvJ9wv6Jf6XcvcsfTx\\\\\\\/Wue15KByRpNr3x7dv67TyoKhPXcH4ylvrjKmqWJ++TpeJtyM9vbW\\\\\\\/Kge8NnVZikDhF+b+8MQZVIVNx6ydeut9bgAnV8Ts4\\\\\\\/Nw8xRaeYYCPZDqRHZl5x6meqkR169kUX\\\\\\\/h6NkkAkN\\\\\\\/leSu4sQEP9zm7LDzctpeXoaQLZT+lWexXoKwYfCCwc9TlI3iGZ7vyzeVQ+KIsMt0yvCHaeimio1fAMhBZS\\\\\\\/Bq32NSjsJ9sSwW4ak9wKdwJ\\\\\\\/eQKOvKSwyOOcvstwM3QAVJioZZGeAWzuJh7aL3t1qDA2as+IJ0WICDgqONREPhs5GiI4URlicHbvE6EmCCvnNedJU7JeCGJ9pqHIlnYUGPV7WNBdDvTKk3CbsivBMgvyr1Anh8W5c3AbHkgAAdkNK14Fge+Lallsp2QPHS0Ko97mDPEM0yK0TJwus4Mx32V1EHyKUDQaKxNauKAE5Mba4gJpyZ2a5s+gNuw+cUKS5jOlS0gYuCLF3x3O1XdtFOxumcOzR2mNmgBh9n8RYPNnUD1lPCQnKa49TZPctJrKmtX7vmHsPUA04zD2x8kqpS5SkCUHgy6FkakJxNYxTcu3XlAiW4Lx9eiuMyanbrRvBvu0cGqWBbVdYtCUGMKAoZgUpgWrvV4t8AYKH673zSW9IDDjJQ829VwGIFY1iRWp2goxZ90CffO9N2W510DxmtCL4tpXGK1mlibgRi2TGRYQScht9X2XtJIjt93dHwdIwwq7iJ9jAQ8i\\\\\\\/ruT2eXFaOVVztaUX\\\\\\\/8L5WrGEvwQwbTsgiyyKbNLpEKV4ek+7gkM5+wkj5viZ5beV8O6WAQL63TKXEYbXxPb8ObtC2c7st+jnbrhFwmDY5zVCmRfA0ckSS0MikajHeY3Ofeq9+H0YuAtoKYCPOzazek2pRLb84AXw\\\\\\\/ANtQ+ncUQmp9Z1qRj+LKyU1\\\\\\\/r2uFZvPjctE393L5sY8aMTsYNWZn+hX\\\\\\\/InnVw2rWiIEYwvbGxmCGmMoE9GXpRHIK9cEiKrdjKapFTb1WFVznYHtVnWMt45qxTYImoVvuUSfMqkSRx\\\\\\\/4r49WGKTSP+R+4yeaqhPaiPBtkRuYCBuFjM1d9VuQw\\\\\\\/h6IS8vFs3n0Lws5P25Ld37YKx+U\\\\\\\/2R19+84xfQWu3g\\\\\\\/ZLmj\\\\\\\/zi+svW0EypdZXcq8zIfJmLsnMLgujhItHuKCKiiWV2yfx2JTvjcSlEwzyitg7zjaTansSVR1fZefUI0Y+liGHrqtWycAS0L4grwTLL\\\\\\\/1wPPChDugmFn93m4\\\\\\\/DzE1obS3nBKbC01lWYyQ1HQvIOl+5IwUzJ86AWnWG8fZ4z3\\\\\\\/1x7ZIwvA\\\\\\\/bMFnew5ny6U4ePn8fUwTYDTOwZMcDb8V9C7cbSB3APobU9k2l5hwdvvQgUm7GutqbNdBfSdkxClxrYZLSjWBXfGFuDATkG\\\\\\\/PDH\\\\\\\/iE\\\\\\\/mvzvbKfniQyTAWw8hoekN\\\\\\\/\\\\\\\/GZwM8z5G3brswFCK7Butg45iu4mb2FpGCZnEX5X0mQOFAUQnSfowD7fDN7wHaTSBQ3Wot2m8lLbXQWt2xuuVx+kercsFp0TjwJmZjEE6eFC9exAK2ktgDTlMIiljI3\\\\\\\/bMiKmbbuorZIMQwCu5pwV2zxmGakxWdS0cV9T3OZp6vQ1jQA+KvMl6ozDu6se9C4VjIf2Khvr7FnM02DTqV+Nknega2AtFAG8jDE9yqP1O+zAjJwhkoGk7pM8WN6PVOZmfWdh9WDeIJknhrIE1ObQDumI\\\\\\\/J5mg0gqITLsyW4Fvq8ER64WF1b\\\\\\\/bU6t+uQQkebhqkZeLPaEs\\\\\\\/9XD+HzDjoZz4y\\\\\\\/Sj2aZ8iVjplY8URjRywabo33u+GmodFpH1h0wPbIEJMRAYqY09UlElGNY+omPxBYg0jQjVXt+8i8bZCRDuRBkxbc9Ph7ukCiehvziI4E2zhDoaXbx8hjiAqutP\\\\\\\/A7zT3e2+dK9U5wmY8FTLs7LsANIF0jg39Me57Re0gIse3NzN2KCnRKnabjNVPO+2JVd7A8DZFGnCs0QBQtBV6hieU87ComcUVNi2P2jztKVH60Cw69X\\\\\\\/ya3hySe+XvNn9Afxl7KfrfAySC5SoPKNrr7bxOdWnSe47cqSl7KmaXLaPcQw\\\\\\\/WANIZrH84h8G6d4ZaQo9k9Y\\\\\\\/5Zkobd2pJr+eBNsXYIGB1RToOEjV4rfmTceshDlpi6ltVmB\\\\\\\/Fe6G89djV2WXbWr2OR07Qr9pUBAN5irnmzPZ1vXPqIPq01fyp6IKBDwehEz6PHb1IsXPugrDQgYhaJ7dula8bOkGhohuXw\\\\\\\/b4X5NiY9uyzFwwZspNsH7fF4w8SdnxKUkNyeDjgeaT9J7TqaYu2kTbJAKhlO0EiS3QC5zdScbfxplNYLVp2+EAhHMusLo0L82Z\\\\\\\/64mPlOIboeeuq59ZmdzpOyxNRsd2zBAlx+dj9F6GkyTx71rtzB1wR0dOy3aEd+NsO7mN55caQxVLA9nG6cQ1ToYi3N\\\\\\\/mPYlZKcRMHUFcTKNNRhSgpe99pDGHe9xbM9Pn0T7RToLCdZbGWGBC3emiEoS3uPqpI0oE0M06h6vd5w8o5PiaZ7XVJX4UWQhwY7igDyo7uVWTZte4K\\\\\\\/orjeOJ3a+Mgbs2ifYBYzeZ6Fj6z2zPCOsYmIyNT2q4br\\\\\\\/hmQyoss9jwsltW\\\\\\\/Dj\\\\\\\/f48bYFQP93uM65lmteTmitp7LMMmD17bs+hI8Aon4B0n02xkQeuYckwbq5dvSODZv43P1Ygp8Y69w5rIPnbuYBgWGIAIoTlKVJmg4+MboucwYT20kSmBRNLx8Gy5ctyWllXKt6i9O9kKPTUViwlGKeNilnIIj4MtXiUwbiU+Uxeo74B2BfyarETog7NQcsWRoXFL8O6bgjJ6HW9o8AELOcI4ohhfQbGlE2tJQ3dkE5uZE1JMZw70\\\\\\\/tb+pxUIya4XK11RSa4K9LpP9t4vlYEnNC0dPXwrjZRdYGLdUMUcsziXcfs1zZbY0QRh7nYkgLGfr\\\\\\\/uMlflr1rrExEXclueFarmM+67CzVYQfY2FY4qs66zK8VLvZSsjlVuNch1olaRPTx6g2jcpMFv+LmvnFT6lj+3EChCIilAaKccToyG8wnGROYPom+ogpVTa5awFEHb2ZLRqGdzrH1y0o1\\\\\\\/L4Fs5nBoHlo0entvoGEBQvmNTgmCOUTxBkKesKfWw4dV57u8CMlOhiW\\\\\\\/UZZu+hp\\\\\\\/MOTLPadqxpdTfGq2KIvHdP857gARjqNDAQuosYHSK7\\\\\\\/\\\\\\\/jR4K5dZfc+x9z4wvQtnbbxHGeIzt\\\\\\\/WhyOMeGpgb1emRuM\\\\\\\/thK2KNm7e5\\\\\\\/8G2q+bp4BoOj+dmI60abySAr8aWzjEIjp2kt1+w+i37Gm0thpBD2JT9uc9QuqrFMp6eacpssm6HvMBNQrsvIyU3gpjf9\\\\\\\/w8gqRwNHWrO1OqwJ93Q9EdByhSdILItGn1iRaENRRFCdUDCGvuu19o7Xm0dFfIKpnc2+VVi6XUzx0Z+fwDoDIWm6u0L5ZKXrpWB8Zz9T8X2dSGsvJWYBllsSEKuDcfO2e8G2IbWwoBOSFsg\\\\\\\/sZGnLKV7q0Qyy4yGkkvmSXdtejYowI1Dbg4UBM0DQZ50BRkRh+oFYKNrs6LlZhnnaz50rURd88g0WTD9wjoUK8gez9u2KV5oqV6h9WVINKxxJ+GbTg\\\\\\\/zPZVILdPqKFE2y8Vl3aNaLEx7NZnxzYxA7jFKnzU\\\\\\\/7IqqUCaOWLr+HSb4isnJzPNeu2e0fCQiv2XCoHikFXOr6bkB41EM1mCYoqd5OPotydc+4irS8gDrUpEVRB+kdazJvoJQ8BSjn3mOwIIDu2\\\\\\\/zQIaI7uAHcWCKYF8Qa52tqdiARyhEd\\\\\\\/1uC7aAJ3DAu7qohQj0T4iEYOAjHwF1\\\\\\\/ynmaCKUFq\\\\\\\/ND+ULCoWIiWEUQ4IAe3nNAh8QEfh0vulgWkIrfNhzcH6I3EGbYaMXuvrGto2HI2Nld9g4s77eJILI3\\\\\\\/qMczeRv+nYEMX8YX4xuFRVBmPPYnXMRwnUnZrumdfXxR28Ehv6Qg0vtwsHirNubwCsaDzQnydZB2hBv4V0kjTUntlTkeLvSNl8JY37JKYIQRVyjhfFcIApBN6FxFLpzdBoM1EOkuHPwNBvhFNdGBRJa6un9MPfrfXGF1LfB7e7z1rYNsy8IfwS85rPKIy5deVAxv+cMNqeE6EnYV4TeGInBfkiMSgcAE3X7EuFWEUwJUqFMxP9VRMh59QYaMRZTbfDr9nt+cuR7RpXBABctxedMKhg03OA0g348Z2DXMuO9H9H6vB1d9SYwoEIMqgfbU4r9Gwt0\\\\\\\/i4S6Q8aC\\\\\\\/lKCNLAvH\\\\\\\/iuPiuQSd+2EMtMAMdCA0FTnvMK68xrre+SOXRHbOweAs3HFnMLfxOki74TDSQWNib0+gcf7bKbt7qma5fPRg+sWDxuTAI3p36hkiTlgtQalM52s8CirMlvWJSDNwiTuEhtj6tS6UTtrn4YiVFExf5PvocIS3YGN\\\\\\\/I2cDO+W+ci\\\\\\\/KnEyZnVLCkloXkLplSwqKYnaA+d+bNol0F87SMpVOGugRY1Mzt\\\\\\\/5\\\\\\\/vjldE7zLKcOUEEE6tjPBvQd4dy90G4Ox3N9NmaAseYau2ercd4Hdb3wT7XFoTDNgBSgbIDUCZLMBvNDb6zb\\\\\\\/h4fvBO6xbDAAawWX+N9cuenQeprga1d5LjpXAOuoL+EJsOTBjTCQZyAyzeJ4rjOcom3MHnMCkNPY+9W62ryopgXgG4Z6pX9qYFlN2+49vUbIl4HSJEHQ8n72EhpOT0hugkALljL2wB9\\\\\\\/Yb7f9BjYe93MLX4soAe1y07\\\\\\\/GlKsCXbrte7CBAGMzqe\\\\\\\/rUSGwbpvVmj\\\\\\\/W4u3UGyEMQt72cuhgy1QHnHZusw\\\\\\\/4LXLc9n0hsbghRUsMgXDscuJEunFcG7jVGT83ok3NLDq7oQBmRu1Mw\\\\\\\/NzPMB8QBAwiTFMc7ef87f952v\\\\\\\/HKw3YNORtoM\\\\\\\/oHDUQB+2Efuwuj\\\\\\\/2ez+85lhc5SrSmRDrC9Cg6ZMnyTJYY9sQTdSsbN\\\\\\\/\\\\\\\/rgpj9ppuH83rXRnciyGdeH\\\\\\\/0MRFyBTrAScGfHZYSrsBgk+AYCIg7EVyqWNTdiWJk+zucPZCBw9i5r+UHeSOsSVPsrL5eft61SaZYz+SSVcVSFI7mxyN8HVA8uTGyuwzHwJAV1E9CpmeE\\\\\\\/ywJBotdzi9pPuK8jJwMQbfpcR4plgcb2mELrEoDHU1QQIZRD0cc6Yvtx5DlKBTZRSywL5397nd1IMLxrOOneU6EAKi7HpOH6NmofVW2yvhtbcU+q0FcVz+ll54L+FWZQlIFcJVO5QgAi4ybj1015ph9LECz9Xh9buutRkF9jy+mJcO60iLWBPB\\\\\\\/bhdXWsyf+n406flzYA4uPjzFyxREOIAVXWTtuQZIjO4LD9MNW\\\\\\\/wXVNSZVxFcHayLZwZZbnE1ZuFMp9Na\\\\\\\/hEJwxhWj2lJWd6FN5J5drwA\\\\\\\/IesyM2a9YeUFbBLUp4V\\\\\\\/nzpa+UdJ4S6stPRvh8BpdqGFe2l+PSI3QZEL29qdrtj9FnNWaFhD2ViEQ8OpgKzbngVYKw8OhtCfLCHYhcPFlKo3iTCXksuTVtBzswkwHc26c7jJVHktZsrR67dWnjDb4n\\\\\\\/BcV09LFF7D+waSA2pPlZRnRL6MqCxgzHtJphBX3B7E7ypxm04MUiLpB5gf\\\\\\\/a+CFIpWjdiuoIsvF0h8wiUWgbGiu5736SmAtcnhh4oN4toWIqlpa2sZhHn4UDePRIkmkBP9\\\\\\\/26H9i89t6DOGaRYN8wP8tQYfiNz7Ggd9ejKSvlXd3jUYgfE4psGzJ0Z2zfPzJJqZOFqtdH9NzhEthH51QeRLDW8RxxJUEsxv73pzcVycPRMY5e2o3xXRbsDiVMrvvhymUg2dIlwyYT\\\\\\\/yIRK0hw5LaTIKJMCpfx9iCq4GDE3fXpK7dV316dps4tPjAFAkitXKA5rL7JTgPwvjdnVDlt50Cpj7j40rng\\\\\\\/U5FVR7ry5w4sBGk\\\\\\\/46K43LTtI7Tw4Jklq8RExhicF50EE7b4MhziAbdG5vY6rG0zBBxycRRZhfARLa7K+2OX3rme0FqxULZjxPTnC\\\\\\\/WduTDB\\\\\\\/mPA9uMqdNvsXZl06BGhb+hOh8cvKNpE8+ELb+J\\\\\\\/mPkNR534OaU2w2GbX6xIDSTvdYcpTM\\\\\\\/91EUH\\\\\\\/zxKMUeatgxkn56AyOTidYFNQ4kBb3pqHFYlrLUbRTHZ4B9kSNU4\\\\\\\/PJT5S72+uDGRNJCYWNgTliQqG4yiQSo55M98L8pJ1hECTBQhe2UAbKMA1fkRdcHCQpfCQUbGP5PIpoYpr53C6uri+M6H1bh9igqVkRKieNdhLRuhW\\\\\\\/l2aStnQ7RJHwRQaINaW+bfclcL3iczRcchBWwOWzwZIenTlmpgJjvnq9BF8u8CGv8UPsSU3vH34Xd7tHbV2Oegx+K03hhdegLg3wGF9EvEfXb5MZtpKdl5yYMjDNzViINWaKfgCHbhNkUs\\\\\\\/SMYZXOTfdN74Ze1eGbcb0DDywoP+C1zl2BUxNRLEkU6\\\\\\\/L623vII\\\\\\\/gYvCqaCHRlGFRLkQXXfKlLjwAM1HKYqRieTSUwaws9KzNIkV8Psr99QqPWYoPAI\\\\\\\/AksbwuTpeVQCsAwWKRL4dXrQjkRyIY6ai+wyONSQC0aRT7FEFwpfGM7URwmwwel+jTa+X05lceBwMyraM6kUzH0ZOZzrdwgejcu6cQQYhfq60A2Bfi92VV1WmVJDZHRbbN7MSBH8OB5rS5PfCc0esvasfVStqV0H06EdLU+gSwGr6OFpS6CzE\\\\\\\/HfivUVPV7wRyuUo9J+J07gRV8euYRdKmL0f0zJbrti0vBfJEPDVM8ST7GC57X30BSP43NDXyPGDauoeNjsdAmEUh77FGuy4OMu0RVIUg4dI85a2MNWnVGyN1YKWcBO1IvhKQI8HrrgRVqmyJOIVZBXhGOcv+vXdI+HMvSUMvEUgOkqpn1sCa9Y3oMdNpT\\\\\\\/9YnsmFNew8XtJuWw8tAIGtmly59pDmE5AcuyymRC+oMzNxZtNY7YOgvfMxYrUn\\\\\\\/xJueVX0XSO+7NrwjXXivqAkLnwEYA66hTpDbbpKV0snMMz+M88cqpHAPmYYqAR4ejwr9TroTXTzixJX3YaSoWZViKtSuGPwnOdfvs1VYEw9EWMX+QkrJ2Tmps+rL53zkW3grcYHS0F4QO8zsfNxWlaZdpfqTX9pqxh+m2OP+q24UPIf52XYrc2jR80SN2upCOdMQci7VrCIVR8r1cHzm68xz5XEfUFNP31q6kClGMF5KQ4JFptr1pOV8UIn2Hl+8\\\\\\\/JPrHw7eCZKd\\\\\\\/STk9eP244bhWQuvAC7reP0ZqCKcVoc+DufqGlqBhiqxDH1uTiTeY7mBL0iUThnv7Zg1RhQqrt4GBV5xcUgbvmCh2qzqq8tS3qn7ngNZsJryVoR61X\\\\\\\/QFg\\\\\\\/jb8u2wLH4xVSmZyz8gX5R\\\\\\\/Ji7qfhIew\\\\\\\/o4S3p2lcBtMAOBuQLoWIjKNRgDt5poCbwbYBrxFb2TebE5+LgBbfgT3IqOy4yOsP9yT2K6Mo+15xlj7Py\\\\\\\/0pZixEVmlhJu4u6FNC9q6J02qu5BmGuEV5fLpP1RZvqcxlQoQrkfWiKWNkLKw9i8EW\\\\\\\/iQPOU5rbThzInnIwBKixS9bKOITFY1cAjwLl5\\\\\\\/3ac7SYUXlWJGf2iVi4GM93GZJiuWvHHiKnRa24zeTQYV8W\\\\\\\/4ZCIN0LsXL+l04URSvz8kdsd0yLXSpjy\\\\\\\/NDInIZ7wJwon3oNh+NY+Tc5BrrrqY9uNMLj5tkbQL5nx+avhSZF3OCZO1U3AJ71oHQzBwXjLjABdaTG+Bw3aU81UJuclcgerq0V7qsO7haKOq67FVtyEctaLowoFPz8g+8E\\\\\\\/wg91jTyORVVYSbq8CjmNLx7Dr4HZ9TAaYi\\\\\\\/CNysiZ1FRg+hvyJMKyextEBDd\\\\\\\/i8KHWi7rclSKNV9dJJPzEGVYQJYF6hO8LSOkIjLJlKI5cZfdeNvT3\\\\\\\/+GPs5bTcgcrRggD8I2xYty7pSPchFpfDSBeRQfzsnTDcdxbVbUbRAkoT2AzGMtvfyTz6U5xmrIX70zkXYvIXvYr7oDAm35UR0BH\\\\\\\/q5owap1y4OajqfzXjzENe3uXdkeyiUOqYzDLGxSWl6WUHzqdfipKA5R+yTH6\\\\\\\/6qawHbXutFm7k7qg+oOn8hqo8673zSZ+idNfImzAHn\\\\\\\/sSyceyNzO+ahY0SzJ21+ixIsD1sKemngreCJML7XYxIcjqp1cDsJOkjZON6RZ2G04Vo5xALdbt3wJ4LyEgMwgDTZaiG4fBJMI30+eySPnQCyTrNcz3HIhtLtMhmC9Emtm6fYwKA0zzDo1Onlnrx8yDP8+UJ2y2+F2LeE5r2nbYKSYptQrZPE1aCpexph1tjA0V+HtixRSK0DsNYdy0qDpsXusXv9tQlwhrt5pbYf5LWNFPMkUlbKRfg\\\\\\\/vgYysAN+m52gF+IDpo2gCr4HS6gTfj5fUXJyJmgpKfBxnG47iQt\\\\\\\/MW36LvyJ\\\\\\\/Cz8FKN6XxgdKLB7CfxfF8c4bP9z\\\\\\\/u69rJLXiRoywlRhWYSeaTOYaJbKUk8omcbzzPIfNaSmGSiAwuOQmD+JLnqcrd0GVuulvOkVFhxyzy3ggZPMw\\\\\\\/ttrlhwmVMuNRLJN0QXosyADfsx2fA9f5vY2vWB5SU1zqyZ+jgopW+obbTxMqfnQv0dFVvRFTbkJ8oDqU\\\\\\\/Kk9OlZW871e5WLJsUX1SbW3tc5ylXW0bYp4MQuusTmiEww8mC8GR6TTjREMNVlcX\\\\\\\/tn7Xdst96SSQH8tPL0v1osRJOe2dqZmjhJdj6ehVWF8acWy3Cg0Vqhwfl9y2M7yHaAxq4B\\\\\\\/f0vHtpJbrS5MGIhG0TyY2Qh1AzGJxHhDlDrHMx0vpE2aVR\\\\\\\/WFBqVvMbrwUcAzmLAZFez\\\\\\\/vT0PDCndd2IUfQmSMgW4Vk\\\\\\\/NST+rZL9ZOtLY\\\\\\\/RnAOQEd0\\\\\\\/vSF1Gb1Kg+yDuO0PFmjgHIQAvi7\\\\\\\/cTloRxUPIJpnD+pKbsblcKNvxuTvnA3x2badcETQqqDDj6RucwiHAiVwlHYFwafen8n+aX0TnVOWiRj63ERtHz1g\\\\\\\/uVCALzQlGHujfFc8lgwD\\\\\\\/Jx70RlxLTI3M0bqEJ5mkO5O8JiKtDUu\\\\\\\/TueoREH3G0XIL4k+7Xu66GD2jvHjmSGaaHl3SstFRFfNgIOGBUREM7ogD1q\\\\\\\/FlkIerGDHcJU\\\\\\\/g+LOZZTO4pi0Pt\\\\\\\/hjAAMX4s\\\\\\\/RQSWa6nsmK0pIiMuKvMynZomTQDkoZWk\\\\\\\/iqmx0AF5AEhSqITfO79hrFu0G6vLMLxkiWKrjpw5WN9SMqhXyrZFUYRF6+7ayrJLSQFUHoKXy\\\\\\\/KuJlO9KhiphyvbrDiYXGEs5V7iMO\\\\\\\/nrhvTj30hsEO3OQotIlHFrZN1biLLuvMdFzHjJ6goflBiiQU5etkISgFaoJgVx6XVtpICANNJils+oOUgH39UGMwwalFhjdiJDf2D4bieSeIH8X9Qkj894xETZ5p7pu5Y\\\\\\\/5u8hwC\\\\\\\/pd5JiCqlSjlGeCrgAyXRzcBqrnKjv\\\\\\\/EWv\\\\\\\/dLJQQIX3wvt01C3Bo\\\\\\\/xke4CRPVqSeug3\\\\\\\/gjUUmj8Bs+H1ELtA5uAwvQHCDnIdnPJhL1BmqTcVZGgWG\\\\\\\/s80BTrhSFLiEUHb0OzMR4rFro4WNWLL79WBShIuCKa+harkislMUxzIsjSkzhdpx3TEVTRflUGPFLMwd1yeQxuR1w3cTTiH5I4xkzmgZfSZRh8Kwkh56n5KvKnjlTWm9cB5kdUMQVH\\\\\\\/e82BN9xSEF2+PrpnocLUlV67kQbmnEIASVHr1IbNmCpITD6m92rYVw1bKlFo1M6Nv9y2NAzDYShnUHfT\\\\\\\/Fgxy+McPsyi0CvlK5aqWSlaeF0sXT+rI87x65hUiUhCT+58cyhAk7LaLlS4seB4TlwWfavYGMRlRKbQpDHPl+c63BYpS0+wZj3kSM+18obhZDRUgsa1khocsLYCeXRwpraO0oKoyA2YUA0I8U705bAjA+Q9HkKKqEVyzlAm2nZ89q0c3oW\\\\\\\/9\\\\\\\/3Orx261R9pbe+PIQL+TmEI+g4jx+3GeQiYZ7BP8Uuq7kxcAt+NwJ+2SGiNd2xkr36nhma\\\\\\\/Wnu9MasiaGKgk+1233M05M0OPc+X5y\\\\\\\/7m0iaZ9lSWCH6lqQTeTB1GjxoiS5yWb6Q5KTck7RyrhgPqyWIH9JA24pLBn4OpxGJHypipwDa9FU660ur6OZjYC79a4erVJJTh9e\\\\\\\/CP\\\\\\\/JVSMMr2xWrrTe+UXOsDqdttiAKfVxfg2AZBL2vVdti4KQ25cuDhUsJBiDe4DztDLbqn2PlPaF1dPoQH3j+KiBj8wpy8Coad++8TbhhMy3Dyj+CIB9g8K9e7\\\\\\\/X8+rgV+TXqbjg31OeRNuzGwEscFfxTV2CMxu9CO6+sOpzmg89rbfvyoCGWnfuo9OMS0CGkuzJ57mbxVdY9ywJDeECr6eMeQozrvJJ+iTggC1NeeSfEQQcfZosWHVJ6VAKn8jpH83TI6TZ1Jv+ICDVUeGPjFAiLQI7f+sHlsWAFeYAirqMOGyhHd0PdraFO7VzJ5JPT5mRrr5Lo0Ph2yndSKdXit942fTobMstFSuStbVZn+Z0+6j10LLc6zLK7ZCYz5GfTFGd\\\\\\\/Hg4TaE\\\\\\\/ve1O2eX9IU9JmG5QO3u9TtRWfDmeUZBou1lpeSS+cqAntpRWy5Z6YRSPdwIi8ych\\\\\\\/5orkpphV12aM7IxMrjtk5AbWEma43GKq6w7NsVMjQ\\\\\\\/1B1WSIC9KMRlkqQnc+Qfz69HhIkryy0OH9kWIyU0y8K3TlHwNJhm0DRy2C3GlPVDt\\\\\\\/cSoQbbPXCIVt58uixTNMLwVcrZK1cW94vwcO4upEU+odqZoJV+eoqjwar3zfabw5kOdNo52UtsDWHr3ziEdXJnmZIyeMvcVEIpZ4SW+IUfxDC\\\\\\\/CohJdk6hCFfBc68V+QtP+7euOmO4xsfgL00Bs+kwpvTrqcDRtgg1V54t1stYy9Vd6K2W3a+JQISZEnv8KGChZpgrZI1tLWQAYVtrwxa5RG6jTOdMZM46elrcoYo3hc9n9CakdxOyG8ze4mmcugD2ZGOIcB0zFw30HFGPGZ5dK+9YsGREYJoitlaRFlpJ7YsdAFynSseUO7GIRDimurJwnFMcX\\\\\\\/x7HKHfW\\\\\\\/hrsrOSMvRWQ\\\\\\\/eXZQ\\\\\\\/qso+F3eA6NRPK5EgsDypVhNOPMENoN1bmiE3gHMt0mDdUGbz4JvCkfEw\\\\\\\/TXqeP6KFlgkI8TYBMYYucBNGVky3M1I4lKowWJFBzxpf+C\\\\\\\/rvrEkhYLV4ydRivhwz\\\\\\\/zKy3bucoDh6aqcH850DHp7iYpT9zwPdoV+BsL1vo4Fbc3h4Xj7pCwRk\\\\\\\/ur3DHFYXOtNaeViIYz9SY5pNHqjOD3ZaxgZgirWxY5Cllcs4N4DaZ5HoSoSNpziLRtffVudB2ySua4XdI0uf1+RIvYQETiuK9GTuXoD+K3J+4jZgNiHbVNbCM1VQgup0Rxg7JdFna+fVSyPEOZpkZPc7IxaOWsvawIeaVHWqQxBsomz4r0JAmZjY961RCxFbzTJSeaLj3NFu4uLtRKaU85EeFFyxgiIt7GetWvLpdjm1G4x0o5ut\\\\\\\/d5flnlrxNdfj5xkErzEVYD\\\\\\\/76vBeYfFKEpH34Pdu2F6OZ5LHh7o5xbWPN+l7sIuI4CS3CdmPW+r7sXvx1nhX5tIb9fwfrVWQF9SwpCxeylBfKE2qeJxE2AFotrcu+LCxmNeHeF5wcCXcUeMG+iIRAyQCd1lKXnpvkk9j4uAzhIsXouNb\\\\\\\/pMr7WdfWteiopRRn9zV1TcdvpCCrX\\\\\\\/NK\\\\\\\/exf5Oh1DcvmiVhv3MzUug5jOW\\\\\\\/uzfZiX0QTvZMnSGkoQcpIaCi98ew52MjmJ6d0UFSAXuoPULyCt0XA0lzYNhHFXpKbKqP0gH5EXnEgIWxSy7aN7lTYZpn\\\\\\\/p8hbHhd9TX7p6+bpH3vAV+lq8T+hQ6mWfm+m8eeL4NKUqEEeGIC+1+pcrOQQNZ7gnFCUUvelCky8CHMobir9srGfoFRiErZ+aFkA20+k7u+ymtd8Wa7MJusBwD5Q3LRXIZAr8IUHow6vDLOS+Ols1S+KaOpl24RNnYIjWbbQ\\\\\\\/adHJTmmW2IGDzeWP\\\\\\\/MrwVUYB81WwskyseS7GsnP85jDTOhxIfrR3iFyQsCKDEg25jI7l6+fbG2Twwj53erPskP4gPgv1uaCcOUv5vcQIMAq0\\\\\\\/+CFSwKPNpbKRnBHJZWI1MnhuLtKhSkaBEvv0IVL3kE0F3U2m\\\\\\\/uQwbTTeRNAtWV\\\\\\\/wnJJBA22o5NvTw+wgyUHC7g9Uk0NG68N6dbyC76lZsGw7hDtMSWl6UaMz1ledjHZ3bkQt1boxNJi9O21yFzF9Ir9cj8u1QnWIsVk0643C5isChUbhTlAKXqejfmmyPAL7dY7Py\\\\\\\/32ka+X93Bodj50pVXLCVn9IcC5+EQptQQ8A\\\\\\\/teBx1f9NZK27vCAtnWdPtEEF0\\\\\\\/BTOxeIfJxrSwqs\\\\\\\/HSIvW\\\\\\\/QqmFRwC7FlG5TKESL6jcRXojScus6m\\\\\\\/WHulrt\\\\\\\/mra+xfvdlH9xaGiSUo\\\\\\\/0Ds0+3q5\\\\\\\/RE2rPV678Sv7y+AKe51DNFbChczGdA7w4b4W3tdh3uxv2PK7tMCxZl+mgv0kk0CmxQ9fQ2z3jL9\\\\\\\/pArq6n6AGP9fYwzU3DBRTOAHsFVkMdwG7JFw1uxkNua0aYXl0NJEjH3ltAFfKUvmR7evaR4qbxgQCzSigq7QEGb4YHTuwFLvStTEqtTz0tp5ZjT0lsSgoK5nEbI5nnhWye8eywhMj5YhFyqq167siA8OjnssvOGNppAsRJa+BEFQtNDoXcMUcenDXpZygKVeFj3HQG6ghUwsndKdJXp9fnEeZlnN4ylvrTR16LoLlFP8ms156NsPv4Fxxw2wUOY84OsWEiZFFUI2eot5ZH3os\\\\\\\/4FVSNxqQFpl9fia\\\\\\\/r1cm7ulDFr3ZfEDm6unc1Wws3sDFe+cDCmmNqDGQsEC22m\\\\\\\/eJCTnQ9duBCTf6zU3\\\\\\\/9LAvAGahTQiIKEobm2Pbv8s2hAaJnr0UKzIu26m0FT1WAGtyJak1KueU8ZmgfipEwmscMV58tlIg+VE+6\\\\\\\/45PSGeDo385SY24PSVT5pzxMJt0uKGLYk6opjwVWJDj+YoSMxdwSkicoURg8Lyhi\\\\\\\/KnZgOPOuTypDPgjb\\\\\\\/SucR9TVsur0jVNBHzWG09lBHrtn\\\\\\\/JF3Kq\\\\\\\/k7TTewjrpq7Q8PsLWe+1LCuZEvnTSs6ZSYBNKxLs7O9J515i59zV+XkCRsVRxGLVrTPlZFoGbI4KWTCr2OUWiO9bSkj1+QFIFrrfSt\\\\\\\/vdSxiimRQXOWCqQji6M\\\\\\\/iaL\\\\\\\/p9WuCCykE98LjNCPU9BefhVtGadJNSfDaiDzgmuvOHB52324WAJab51kYE6Kgl3YULs8qdpFEyGjl50GbZc4IvR9bjtbvFpJFiIMyJUMSGWrYb\\\\\\\/7zqa7DWMeLfYJUpMeesbgNgkXZhaX57qeIBMkRJTEQ1U1rbgVwI0tYwl1yfFG\\\\\\\/kmjVTU\\\\\\\/8Tw+2WVE\\\\\\\/4zg4eQ5AlP1sFCbkGs2GRubuAoOqtHhYsQtjn98x5ra1RPvS3\\\\\\\/4TAGAlfyf7Vzg9cvQ+IBUF3viiNzD+qUCXEIbAebNTGen1aUpO5FYcNCIsc+vMsC3nEidF4stFwIaCZKKp4M0Iip49O678Ql\\\\\\\/VlPWC5UffTdepuTSyV8JjPo6qH7oPKGZeNuyog58hIhlgjIkJvZUg05L8Kt0uBNohchntzfP13PX3RdQz2c\\\\\\\/xF\\\\\\\/RRJcY5gSkUlGGMOewraiO5QqEkpnRBOQdR1o0WAUtxtJmmRdjVNxACJ26ZI3mjPv8060Aczs7mv1gWXUPL8bsQeTK1Mj2mxwjwwRiJP7pQXyRRKuN1cFB4fKpFi1t3yS+DZny3kkAFQ\\\\\\\/Vt9c9qbyJh2HI4mhPm6wJhBQG9cv41084aNVq+V9G42+PuKHYm0mDQTmZ\\\\\\\/3OLqCfg5TIp2Gr0zwSuuDMXpUthEGDZo\\\\\\\/n2QHCeWu6OD6MC+0eA1JOXfNIu41IOIUflYD738vjNNJQdY9U2LXFxjAFHUH4VlA4y5SfNZoMAGtQp1NVy3hzQDJMPmlBihXRThjQqTf8thmdrh06P7YJBzenLHwjQHiEwuWMURYesBdPJm4EDuAkkatJKR\\\\\\\/2pXqIeaBgL\\\\\\\/BCunBShZWqoRvOTpwN+LJSNUFb0emCYEn0UlXGMXBMn12KHGFr2cOOsMbYPFxFI5xFN2Q0\\\\\\\/YdjNuF7FCYgiQTRNVz7qhq3GRbxf5i5QVJiggLZz87CLA1oKgNfAIUiRADY66yOMTnjSgDrq9essFXYifByoO7kdD+RG3kFohLiw1vDiFgRqYUyBb4g9e8a5OYwm49doH5q9I4cae1h\\\\\\\/SpajaR6MnQwRwNPk6kfipssZKXTcQU8hdwkEffj4Ggb36dRp3uRRdkn0Zdp6ebd4sB4W1a1bTchimO+uxVS\\\\\\\/FPWQGJYNK0gzEF22+Z6loURVGanBhD2VqpQjaaqGUKlqJYPVS1JDKmH0R2+3b1zhjvjtZdt2jATtkLbZewlq64SCay3oDwSvtCT33A7l99LOyuUNIUeiC+dg04XmWW9cPqSOb8Pl3EeGTgbsYMRITTsexs79bBpkxSwd79uDX1v+yjuOPH0MGoRY7O9tP+vPmqH5E9vv2jiRifC9uYMbwvjrXmhVS4zC3Tk8veM\\\\\\\/HVi1l4vIaWmHRy2uaEmpdlwZRaDYkyTaX5qbcrh7U4DjbagH0StMBSTRPcKmqcanYWuItmkZAFa7BqJQSuK09Lm8MM4ZEwOqEwWte6EtIVJA0knUfSDlxHkZMVoA4HO\\\\\\\/IMZdBMxSIhsczcfJ245i+XpZwaf8BCxWRZ1fliJol4ZAAHsuH1lGZMJCHv+k+JQGG2UJ+79aeVlMrAK8oeU1xeq+Khf0MUvURdkvRnyGeRxVEcR\\\\\\\/0649lN6aaMM3TzFE0tAwG4jJiXNmPPTfk+t9mZfoFI0idq3vCUpZoXltDKIGGEQvTAEB5saVIhN96mtDRb9nDMiTxKP4UoSkui3XKhFNX+GQi1i7P\\\\\\\/PxZTAlU5tWoLafd6q+H6V34KD2HAgLUqEwBBgKerMIlrj3dhEQvVDl6uH8pypgGYjFKin8UtitAyiCI92L3BcpswrWpP7hJRgAl3\\\\\\\/b77FtI4j3e06MS4\\\\\\\/jAA3C63lStN4+mY9ARhhBC0ltoN0Ld++cV3WUbazH6nLX2MVY9XjbZIqzSKjDfBmn1SzZifF2\\\\\\\/JvzBDA+Xb6qfhWIn6QK2+KEFBbI0cc086Jd7ce26Aj0Ksq8Hz04Ot+1a1QPJZ4rU1w3MUkmdwfF2IVEMupaisWL1xrypiJyllsZuZFAe+WhBV8AZDLWeOO8PfSixhnEujwS+VjjlYzvyqbxjR0kyUSRTlOA9aYdEBGyvh7ewEmfi5zqNYf9pcrI71xjBdQyWS1BKVnF8WLA6DyKH4QBnP3atoSeYn9CKC37agrpJma4HyZJORfitUUhC\\\\\\\/gV1+BLO67atrZ0PABQUksByrBSJCE51chH7yTQRBMUawlUFpuVFycQu+j8vE3Sqiw4BNnUPSjw2fBghvFXNZwvX8duHmmp8bbXHLvF8JUCvCSVkpRjuFiiB36rn2xMGzNuFULnX3Lj4Pa6zstz4NdPNKwzwSlEp0\\\\\\\/O3ut+f49yJNffhliSIFgDpEwHSX0AYy3TeC34GkNjjHXkdm6ZNUOzsmncnSv5g3mbAw6RM4dVX4UhXJOyq+yL6oh2C9aIGl7b5a\\\\\\\/seUC4g+3n5n\\\\\\\/Tjr+WSpsHQDskXXKhQJG7LbuYE1cXUtxQB\\\\\\\/tMtbw+bqb3QiGLWU\\\\\\\/CjD\\\\\\\/\\\\\\\/d73qqG6YaG+tayBMr\\\\\\\/C6d84kurGkNb4wZmC2CRZOa0D4m0gkovIoMENz0i4bbPOpbDuF6+YQJwOm\\\\\\\/gj2I85n24jfoheFqhFuOs4qC6aLALwnsokLZvrAaf6k8j46Tr6J16fWzP12gnOQXLndVsc9E6zr\\\\\\\/\\\\\\\/F5DKQ\\\\\\\/6BUZhIb49SQFQle2Km19m8YPiRxNqf85oVvW5R6yk9KsV4\\\\\\\/iNyvZLj6s\\\\\\\/pLthd8A9RDw8LTthjm2aTIIAe\\\\\\\/W5HgOyJDGVdUhlSAW88lIIbI2Fyj5WMtlLMIl7DgQ7jKs2BPOocgZ8p+r\\\\\\\/TIoyanhz8W8gR4MLWggDLmnaU\\\\\\\/WGRUCffwQrjdPTwRu+7mG8h0Tu+P4Y1m6oDpqNor5AjcE\\\\\\\/QoqrhrXcm6qVdHvarKw29oTiTP25SSgbSGpANE+iafRGqLuCgOMD1+ZrL\\\\\\\/GVn1\\\\\\\/udYZD6v8VU5mAQu105FzoPsuEHqWpVsl+56DfB\\\\\\\/xsiDT4QClIgLvL0igFTSrQhkd0j6oP0i\\\\\\\/Xp8YlD9GTXrvYVzKlhtOZhxwtIsyI36BtxrxZmr+JwFSCLFV3B2FJK1fWSXd3l\\\\\\\/VECkIfL2vcYUN7tOBmWLtH83BiPe133lXK3sDU2mecs\\\\\\\/vx1XlKOrCPBtukz3WIDEJPBaTD4PVEoKTSDQP5r6v5DTMsTlgoXG23oR3bzfsvoNO2+utcYOKsezC+THWnYiPa1Ij3gMZup0W+GKenQLXxoM6SgxQTc7sn+cTQv1sAqmAYxMh+mbibWVW4+gyuxNjWDMRs2eX9rk+goy2KbXuoharW\\\\\\\/\\\\\\\/IDTcHGKbaWiJ3dedWAHKH+3WCmljK0usVT+L1A20MjRYcJ1S7W690lAcBmETArl5r\\\\\\\/amFGJ\\\\\\\/Ha8ho2cqSAlpGfIs+8VCaXDvtlO03GkffJpdrkIX5UcqctR79HJxU2KlYGQjx+zpMzVCvodSBHQsEtyYKXcIcMOgdLbJwBeJD5VedHmPq2LS5NqRsYaTxYc96a1EuKw4duY3DA0TwT8AaZ1QRLzpd7jQpsZ9b7bN7bVpEiVBCNmwiPOvjG\\\\\\\/D\\\\\\\/Q0v\\\\\\\/gziG5yt6yC1nbxtQBZkcRwe5ALM\\\\\\\/yieaPLJgY2ophAiBbb2xsgvZbzdXB4NWyE7m7gqf81nZrwbZxu059EdbQYrXPy7YwKHyqY79DfGX2yi64lyMxYljsdPSnmWugWRrAhkVv9kMkmi2gBzdCG9XGazpS13mw1g8aRnu6PkXzMDpZQ+tofOJnuHaGw+gzJduOHSqMSwmsgueDGpQ7t75rseudpPCdzcO2Svu\\\\\\\/6HY5kiaukg9a5\\\\\\\/81JIPrt8Aanq8HDopzZhLFzHI+3200X83rNHDGHHFWsaUqw38f24n89\\\\\\\/LEmSlGDgsHheERodGjo3huKmHwSleeIoNlo4B\\\\\\\/Bn7QDeRSOpJk5D5yVK53wzhyyYNJEaig6uTICkx5Q3koAq9UUJd6ZzYzwY55TfBv+kzF6naCk9tpa+gorYrM3WQE7Sy\\\\\\\/VphwyYYhx0seDTbczqLG+T7TW0Qo24iJCflnUa2IHHepg0cOT\\\\\\\/B5GZ0rxED88haFWhszsszimzrWqld9U73CtH2Lz1zGMcrfSMVAjWqoe7IpuKt3bWGDYDxUMaA5X+v9rEZVKo3UK531gxFFECRjVDRovZBMqcd6bQ86NWuoMlbveX2hxoOXG3\\\\\\\/8WGVcCwFfUMYlDBxL7CjwxWXtXlBzRtQ5napHBM\\\\\\\/acdfeht+q8HmCGHi0QNdfifQCZup6zgcGNFtk01oYk6\\\\\\\/d9HLl9JVWVXxHtB88NL18IQbtgamPiP9liiSvdi7weIeVIqN76jSGjzESIUO0JWZm\\\\\\\/2SSCNVrlhZ4aL7K3Jr0DJ2oEpmsqu+At6i+qh0pj+lpfwG0zKTWiB8o0D+uufgWjZbNbzL9fgL94XNDITyFBxnR7SYzALDJXbjsdOGlmD5BKhWMeLRrsZY6l+SGkebwFmWZZWx9zvMkB8Eal\\\\\\\/YUcvb3sdNEmU+4yMm8Sa66HqtSRnBd5gXkNBrNV9iM3Q0F6W10OjGsRvTxczYLOr4x\\\\\\\/CL16lre8iS0UGWZzHhS\\\\\\\/+ot8DMJho6VjhGEeWQCjtGgTTZvV2etJX6ZVs88Q2QiWBZp+YpK\\\\\\\/E4ZM7ehmIaZpp4GM6k\\\\\\\/T8R5QhBk9n45L7cm5o+BW1oxMpEWDfhGfYw8rH5WvrgJ3UHLVQ52FOgrhvfcSlKfKGq4Hs0WMPhmc3gISjku1m0VOdJw3hjyMEro7XA3pN5R6Cri8WPmJ+D0dJir\\\\\\\/hSI5+l5DgfcX7Z41O53Xb5sn9g7CVYf9t8BZhcQLk90aaQ75Y2Ff0FeMLavOjj9DkK+YZsMgvfrJR1qHzdO6ctg8DQ1ib3gbtZyEs8D6lVy7B6XZUK6Ewl6aqVYDlgAff6C9m6FFtW\\\\\\\/\\\\\\\/EFjvd65o7FYljd5LeABU1\\\\\\\/+UU0MIDRuSVJvxmDuTFCav8xeqWbZn3CwhPXoXPqwOfUz2VyaoZqxyavV11JyrW19raP+7iVJuRSyaUMAy\\\\\\\/wG9GKrQefHuUK+ciH7+4ELV365nPwuh3qkG3gLk\\\\\\\/VUrCkX5K5QUkS3dKSn+wXsMMVt6aqRAnlpdFuJLFjSgmM9zHo84507+PYYseeaNBOASTZbmdGASJzc8Zy\\\\\\\/zKMAY7ObiDZtTobZ2pyd1c8MP20x7fJjLUT4FVFAK4t0UD\\\\\\\/kc0dT4b9zg\\\\\\\/lacS9CwP6pDLkzfjhoDUjBvbeKdew+adtjFn3S5lIy2AVH+eB0qPu0eqXV6qHOIz5xtojlw+EXq3RiR\\\\\\\/UR8xdqXd36DjAjDJSFhEizBI\\\\\\\/QJwfzwJnHrvaMvlTv0qLe19irwna1tWzHpXxHdve4G0vlFj\\\\\\\/+6yfmrQDIkVA1ZequPg1zrzuAfZihB3r4Pc7uYxFEywUwGILyudrY884jB3iy5a7HeIjfhHQ8ZTh952yhmicypoDbbTl6ykNiRNYtdqoGnW1NCH6UIypVCYDj7EArSRAz1dXbFrbO1cO57ei2BAtvxCUZ9aFkbXDH8j04QmG7TEgFUXPThdwclMExBeTw948qPXgwJocHHXXCu6mHyM4jEz+XDU6oPtL2e\\\\\\\/ubQHmGcuky1eTvJwOsOF8L\\\\\\\/jkRxWoucFXrI01Qd8T2kcgXw9AWQVUQWK0qPQZ\\\\\\\/gUcgbWvnBYEzzPKj6LBuCg4gck6SKxzWNpMKLbyUnbL9eH1uqI7NvZ51xdiTSNuovOzFZe3KvizBY5D190IRk3mLBFL5gHkzrhT1rv6Z0l2Wpfjzz2AYfadJlyLYjLoSc2ka+jTny\\\\\\\/JDp3eO3+yGy2XhaJuYAl1\\\\\\\/1CTIPLi0g0K901EkTFHBmMLagvyHj8JtaKI4c1ptoYuievm0xif5ZxJIT+24ZvUGGI9jgfjKbCsx27+Rt2Gg2z6xyo2kq9GQ7fVlYdm6cDsfpMOh9h8mJSYwyKb+jR3GMjWoHMSXuq9zfwssI0WH2kMm6v6RTE9Uok5+VgLv9VbMhX6wzk2zFjKUot7KrYf7c\\\\\\\/BPNYw516k0qhSq9ao\\\\\\\/mt+835kL9vGFlpFm67Jqn8ugX7DYuAcgGAkDZUVKKU2CqFE60EjWhogsXX9DwZ8b0i2yZGqcXzatoYS4W\\\\\\\/9UEE0oXyruCut+tuPbscYJ1lQ6PFn+9TZ4GLq+\\\\\\\/kMK6taDyvmUshCUatO+8BD4AS0tkprIQZErB7Wo3jmujd320hepfEePP92qpy9gEGbrnjoO7FeUnx0A+T2NcUl4SqcvM7\\\\\\\/xD96aXrQVokm3w7N7XTM5NLsM8KnW\\\\\\\/I9v49GxVhwQrKq+WvCgyYcfqFj0L4WX8AjBPhyYCcbIPbsK3iUk0tgbpo7xbdigANM5MDHeJOMLcvRUDZoRK9G+csYudV5LHmJ0ixouTicclR57lYrY1Vv\\\\\\\/W7\\\\\\\/JZykeiFNGrgNkEJV8GmGQz+L3T0hKTwtS9yMS1dnYp+gMSyp7QYYM3W7\\\\\\\/j+geefnLBwuEOxQRHvtbQmqH3FCfBeDuAotNCzdWztIsfHilsHMeiKF4EL7zsAvioSIOAUUQWIg1MBHJL56A1uiEIBAYcJBVuFEf\\\\\\\/PGTvQjXOc\\\\\\\/qloHVi1t3b5LjQZcwlgTYvxwdMvZy4MBFxBQTQNdUR7Oie+7jB45QzretpY95rEbMuGumlf+LiDeQMzmXijxVtuI3bMC0kj+AiQG09mixs4+NnlYTV7vXqtrM3UanV0aFG5BRNlKFMvK8VGGofPC6ckM6TWyyEYlD8ASCYNRNdzgzy2dXl\\\\\\\/ZElkXy\\\\\\\/fTnHD\\\\\\\/MnE8vSwZyk35f716P7PMj+TmfMF4m53a6SB8k9ZUTmQbbm8PuWD27YenyHm+Wi8KxMkuHCQpNtV\\\\\\\/iQlTt7ADvPPHKRYF5ymJNcH9DUO0vBDizuF\\\\\\\/UDgXxd\\\\\\\/PI5AX\\\\\\\/7KKZP2nRzHqI3zSFIRpwx7bPidh6\\\\\\\/4U+bLCIRH4tEIJhUBv9c8W6yWVnuFAhWKnfX\\\\\\\/aGJ25oEPNjywbfgcVpuDWyDwmvlrAiXP4tD0lpWQ8FzclngpJz2nMWJE\\\\\\\/GtmkuS7cEddBbQFXxQuD9RdLhwjohLtEZsbnfXJ4Wf5BiICoRZXBdXPYy+90Qbpmy5rcOmvY6eGEUadDyNXWTtlvOxuBPrAHMLNL\\\\\\\/q+uV5DnacVbh6TGm1p\\\\\\\/j8rF0jUlhA0MsObYubX4YBkC7IjNYeb5Gde9+g9hvhrq0GeSV7bSwFyWfjNjiizge389UpEm5oKOAgGj2\\\\\\\/VQXHrnWFt2nZ+yFhSzU5DlBLz+lbGRHVpVZjEj9DrGRY547a91Kd6m0M7mFIjkb0ncdIvHr3g2hHDSQw8M9PIUznw0amCeZ8\\\\\\\/ix18s0oxNQnoBMxeaeR3StErFtgAp+\\\\\\\/Kfc66IGvi9ts+MwoajJ4iCg2pYqJOAaKgm+z6Et20GhG8TonhhyOTCEJKfBBxX5wlYh+azGyjJB4e\\\\\\\/eSIJxTBFZLM3VSnHcy0JsnZN0YpHlzws1h9DccHWSYcuXjeZLgs921HRzRm9lB715EMh+FUvFueLIqpdat5aNtZalDT2lOv5NcmfWU7ksyrwZHUEZDWgycC1RH0JfO9XIQazs25OWV7LFQaOZo9m7jG34gHXO4Akvo5UTX9NO6ZFk4calckKTR0KYwy50R3qlbzeor5NjKQty8L+SGRUXg9kUNt5TdxuTRA\\\\\\\/dEZ11\\\\\\\/sZl9ZxMv\\\\\\\/c2H4dnh52GEVWNIX+hhKDSkxw5knEGRc6eqsTwyselKRnLif+dbxyWGu9vfx6RYR1uwsey7yirEMJtPSr+b8Ay1nQ+Zqkcv+vOOypB67d3K80xfVS67PdMOJfvyLjTzxQ9HpUnuwVurEneKuQjrxyW8dZdQDuxHYtjjeO4826DsQ6KfSMmzfgsQ4ZT2Hx4M2KjNj6gttXmHQ+9ToPDXgOcAlsxoagrlloliRIuIJpgSY\\\\\\\/4y24KzMeM9hD750MOBLjCj5UBjCs2uvAT1OTk+gZl7LRASTMCuxzMg4hbjbRjcvdyHhjA4Bu3TBkhbSRm63U46Y9r8VRxwUNrAC\\\\\\\/iFQXb2uQ1pCwWdjWd55v0Midr\\\\\\\/93\\\\\\\/BHnOoBn3z46Y8zLFbTOPWS67pyjra24gkPJNnn8cKvG4GDTbJQcytqs4THZuE8yV7K3NenGCdpgBidYMUltOl4pBw47i\\\\\\\/mHTuu6KszfCZvm3d9P32yylCmdw6c9tQRamSCwOQDUSCdRcPPN13R5Qll7uJmNaOp2cKoTjTZKp+vrBwh+J1ZWX\\\\\\\/lonfG5u6q2epQZMl5nz3M8AvUfd7HZMFAbYPyCylDJL+R9psNwvvhEfpUyTFFhT9\\\\\\\/1V26enO1ceTWofZ7HQxdxJQCZMv8kpTvWjL7bOUo1q96pobu7t8O0uLj8cn8WzZ42XcQvFCt9TZDfc2TUwFIbVCf5ZDHFPTRe\\\\\\\/qeSJgvZ6yftAkf46noDnTy+UgQCFXAfsLNH9qslM3xEkB7EyKUucapEBWU2ejAyEhsvMG6NSyMyAmFEWOJ0CRTl0nSh2z5gx2nkqgoXIl3yiLrQ8AkLxCF8XgsI20wa+K1CJxxa5z3BSQNVmok7N6bt7E+vCajCqTWkZLfttf+Nq0gKVuiLvGt+VAV\\\\\\\/s+9\\\\\\\/iULPxAYV96xiYMYeJol5FbZP5Ex9ktZMHE\\\\\\\/0H0ty8UtfO0UIHvJYMpBrnCvmX01IXEO9Hcd6fG8YxwYR+0zA6gPgBMTNFWG2FBiN6cxTIbqyAFagHa\\\\\\\/J\\\\\\\/\\\\\\\/pIBohSH+WppOCKV+DTE2F84anjmNp6dWEPw2NHh+Iv\\\\\\\/E+UMe2PyZQjiZOeHsgHC7snD2+8wqg\\\\\\\/VI1gYA3BVUqB6TRjKpMlga51cNeLwtov7yjAit+xxlNo8LP1PhD4Yt0E6bnvrTDcDZfbw\\\\\\\/v6tV4vvqFvSlLOj\\\\\\\/dH5StGab3mRpTb0P3N72OiWK9tVEosi83kQKPOF\\\\\\\/XdXOqbzBSeM+jKhncgn85HWioOxeRhecuru0y06qct3Kz8qfKtrNJ\\\\\\\/SAc6Ci+4LGsaHC7IfJf3w\\\\\\\/YAAt1vg1NaTjfnRp9GhcD4\\\\\\\/2vN3WXPofizHi4DuwCOaamyLsQY69BWvWdhiQBLgKBNB\\\\\\\/7lmcBBLLPSB3bs0BJD3FWdNlx6CRaQ2gUoJWtFCXVEnBtx5TQ9v9I0R9zkHM4Tcp2Z+PbbgqsgdlbJrUpLe5r1Uf4vKjPG2cSRZCkuAZrjiw8JcbjLWqY4+wpL+W\\\\\\\/E5Sl+686o+JVFFihUOv24zcyOLSccOLKsQbvTpbmokWAUVQKkWjewOtDDqpW+QlNYdKuim1cQ5Ik+Im91BfwPqZHXPkistFJucZNbXA7fZu+rFRP++aomEocSQRSFD5+KdtiwTnfX\\\\\\\/bQXA338ZAF1Yyl58Mr4o0SHDKojPVqmI21HjGDA5KfRC\\\\\\\/nkd2VHBXMUXB+g5W9eR\\\\\\\/Bj8NByiDToFs5AVNc9sgJKsfp1u6eDXgHc2uQYEn1rSpomHG06+vSGAzhSduRvHwWzIvAEGmwLKf0LkpBCQm+lOWQkdUwrGymz5xnF1cLvVfbOZ6LBuDYNNYteSnliXcr\\\\\\\/lkoYh0yLELS+ulh8yh\\\\\\\/\\\\\\\/nJHDnAi+krbgIsNUIG3cDK1hMavg0JYBMvmRMttg+WkZiUVxLpcGgkIGm7kRDJJPmuIklNj3QAMOVL59t\\\\\\\/ERDfQEnuEGZlkPgovxuHHWJtQgek4ajxZHR5ySm+\\\\\\\/THikS\\\\\\\/yu82+460QxYLOb8R+m17FqdcZ0LiiSEtrjidbHJxyNfADrUjBZv\\\\\\\/a\\\\\\\/C7ZumYff2TGhNuR6k8+UtMnBYGjOJef5VDFz7pEedCNyd2GpQCAfGreAqtJjQZI9MdgYPyYx7pHWw7V9RTZ0R83BFWHzAhWY93+jliyP4lFeInMFRaeNLEAiQt28\\\\\\\/WyjZ87t0+gWDm2tXYxbsy6mnMrlse4bgkX\\\\\\\/AHiwmkQyP6tvwLyisb84uugRY0O9nWCY\\\\\\\/jJxU9lqwzaivFVYb825I0IiNNXPcK8gdF16xOPqzL2P8Mzvd\\\\\\\/xs+ZGsUesB\\\\\\\/PEpqqohsVOqnP0Dc7DjA+8cFSCh9bOkEvL7M7I1xQFeHqW\\\\\\\/62MDZ0MBmJ4aH9I2ZpYNA1zaudFBM1fyvmTcTbpJJf47O9LdbtCvBZKJ3oAIzFYfjk1lgqJ9ANGVP51bTSgTBzt7GpgkrWsFyFLWE2djb5eIRcYAXExGblRp24PvsuUQCvwtI0K4uxzZRkwig28dLfIHxORDMvS2MbgzDmFHKF2+qh4rAlUUNWx3O7ICq2hGUdDGbvTuc95IqEgGas3vf10cJgHIteCFAyaRUaPHpF3rvso8tFB+TmSItWdGRK5XARd6KGlunHRW0rc+xGGFGhikDlmtKazXILzDEfd0OkxSN+BLo\\\\\\\/xgtyrAad6118ZCTDsl51fSSJDhBJOWL2Bzvtc84M0tMJEpJ2P\\\\\\\/og9Xn4pbp7vQ\\\\\\\/KKkVnUUTjpBfKUIg8Hhh6TIJNw1wYYFdT0ZD7InVgh99yUmRRyqRrSpOJOCgofB6QYWIxsza3lMqGxiHw80c\\\\\\\/CKJVlwTljmakyIPKyPpGBPFR9JRQeVKVwGM79UUrFJZIn\\\\\\\/Vr3VVKwQkUPLQAHYTXvpK8AsofQF7X08jHH0BjPbkp0g8A7k9RXyxmI6KEgF+FalLY8VV2GOAkSiQ1dt4pGbn70tLytnUFM9Fk7drrixmg7EZoA2YIwduUne1Ge6OXZazfYyR5rozE3lDhnGRB+Ht3dJMlToJ2WKKt4yReWTh8qyTEJLB7Il8XVUm4Xne+kTyvVYab2HausvvVGe8X5ll\\\\\\\/t9OtvVl+Phkt1gMjxSWTs+aNv2\\\\\\\/f+LaZDynVmcgjUUuIreS++8aB20sr++lhffAXN8H0JIO+HGk8GDDMpn9YooV2mAnDBKXQEwEG2rdsxIsgpXLLlL1AzlbNXskovxnuFw+iLS9rPGEVteZrwwtvyZEae2r8ugtcP7+bJFHrg3zb8lR60ghuJ4WqTQYqs3SjV2XTfcSSio0nUN3uoDC8Rnla2jIedNqneSBQBrhP58FXZ4rvi95VbNY020Pb03rPyxbpIcnO1NTCo4giV56fvXO4c1evYNrJSrnZkEApYMK07ubLYJXFTWrxHgB7H7zr2Fw0NVvJ9PAGDsFd7CMxdnDjI7CwlGo6KMLb65s8p\\\\\\\/VPyydzUjVEix38V7DNsaYaHUW8H8gRZWTTm8Y0npE8Hdsq9ABNAdt8e5SbyH65VT2r5hDEVA3N1S11iBTKhlL+DWp31Qw6CZZLONxcy64P8sY2PxKlmE2Ba8w1dY7r9nZ5lcdZvupOgC3d4UoiYXphINOudGmSFj4vFviksExac0teYmsv7i2vF2RGAS\\\\\\\/C53WqpdOYj2E13s6FS4mC8tmkn+6OuP2fyMn0vUelgXt5lF4zF7D35tv6R6vaZ8ME70Oxi36R0fY7p36r8B6XYtuJ3oA\\\\\\\/1EfJoixDHO34aiYLkLZ6sVQtddZDqL6N8yb8\\\\\\\/PLzry81AJAG2O3sOCDJqgU\\\\\\\/hBhiQLSYH+ChXw4y32u\\\\\\\/G0XLgxP2cVxCMHP5S8mbQUTlMol16mLRz3kTrd+PGKsm+oU\\\\\\\/n36lWRPSW8p+IvlRgboEKoSHcvvZazDjiHRwkxbIizuBhWYzGY6tDpzcB5buRHVhlPlbyMSu7Wb2uDqlofdxVaeflre39wjDIw+SoUDKHvYF3784NVOTEap0a1Ly0Yec1KGEdpf4+BRrznhcT2mtrO1G\\\u0022,\\\u0022iv\\\u0022:\\\u0022n9lMwbYS8TmIyUFi8J16uw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022792fa00f2b4143d2254c6fd63728e2bf096dc8ed384b7064e30b1d510fdcdf9c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uRsd3yfYrYW7rpYrJ2YB9ZhSCtPpkH9eFOtlkFRnb66QhJuPz4sxqSOwVpWgR99\\\\\\\/TR\\\\\\\/IVEOcFhngfF8lai1CsMocnGGUpryHg\\\\\\\/sNZkUiA8TgSDaYnWQBnFZ+UL1mqKYkQqAETk7C5RTDjLZkpruIJdGq\\\\\\\/X2pe3aOpS0BOLZLnr9H1\\\\\\\/HUJjk65C9R8Lk+E6u3o\\\\\\\/41liznRN88lA225\\\\\\\/BD46XvliMunTVt8gZwBpdu8GcsC+a3UJ8IfgyVUqaMxVugCWROifjILuMXxG28+Jsz+mEVT+OfGIogUU9XiKL4Km\\\\\\\/Kakp7yF3OOF3GELgdPyOJg4IzTFKQVAokHM8yuGVoAYrYcUxOLK4ANGAlqQ4n3N1o0bk2m2jqcMmxWuR\\\\\\\/AMtNiFtH4jpBYN63gD6nSM3s\\\\\\\/g9HfL4pTVQG6gu1lTctz8FKJ5yGnMbg+Fcy2U+jZePzwZuE1Mra8dnvOwnzUAhZnr9e7n3AIYqV+Nm56fyXZz6LhMBpsHOZz7eaWCUQOQggmeyPT\\\\\\\/eDAgqafQYYRfOrWlCVBSAeZOUByHzP+03D3clSLSF1O3GR1JMn6HvWgdAm42OeVMgldPhwJAxGfqZUo6ZlVcGykLUfiXLIMM\\\\\\\/udDJsAnLt8WQ+IdOXPF\\\\\\\/HzREsJm7nFwo6BBo3OFpDK3uw5l51PpXuPpx1Y+r7h\\\\\\\/BFKMP59jEJEjXR7OlOEoItMpRu9F\\\\\\\/HUV+yD7VNz00iDAI9CmTdSFJPN01EO8V2s3GE34fTwBBbOXEskjsPy0n6WmFS4\\\\\\\/FDEhXKPmHaRyUCu+NQ5DrncHCfM166ukBR3jLH6Y0wXgdu91YQrdi1scN0Q6aW37NdUvT3MZG0uQTlNfK2hqI7NqiQgsqU8aUPw6wZMXzVZCIHUO4tmpRxsY24eBQBFLZfRYkO9OvSEW2fwFL7NHRJhIFXdr9kCRb3JT+U4PAJUysAnIW0RXZ0SCQIZtsO4P3sXmtDDNRSzt70kef9\\\\\\\/EUVfRQN8giRb\\\\\\\/+jvX5ZaL6V5ydPtO6vlnjRBx8UTbnZT6iitbc2kaYDKi3s9HYBzahMFsZxDp9\\\\\\\/5hMLoT1sMOua76Kmc8Bvn+ovIG85W7fvoqPr9wq2Mwf0HmSCtIEQ\\\\\\\/7lEojtzCCAAstdDLrh4He9zw9lDRGefBbWUl7TYNOJnh0ldn+VOZkU0J3w5uzfd4nIiwafoaB9dsO0rG8gHIrst\\\\\\\/NNUiCHnwD8l1hWvCc7\\\\\\\/wQGMoL1LFiBbED34WwPmS7L0T49SN+SaS5S4EEYDZNPAyE6mLNlQk4D40i3o\\\\\\\/AHZcrlYeHbcJ50QPEBfGeiSJ3U1CW9fMx4234g2UMiqVspkf8p0pyFWQvA\\\\\\\/QwZd4aHy8mZO5zca645H3yAVgZEKa0lesQP0rELiFxb46ARzesITHCbFP\\\\\\\/bZzKIVtEblI4Qh2\\\\\\\/QPTEHvqVZyTttWnSwcK4+G47TlU08vHR8NqHKeR5Hpk0JRqYb7eBxVmLw0osnovFJEhT2adWW8M\\\\\\\/fF8YatxzcrYMVf1tDIU7kgiyPAwUFdY\\\\\\\/CtFwysN7inUGBRm9kvLH19+YI9uPHrjWCh3SEMoBCMtLOV7bzX3UDbnPcVzwnWg71CkbMBO0FDysg1OGi1yNJ7kBvC4zdUzfYC+0v\\\\\\\/VD\\\\\\\/e2y25NE9iIIotQaQ+lJPtq5wo4Z631EJorjiEtpXh2MBa9wxbK1ijklG8HoK+tIVeiSMaIMmldZDytoL7RzcHHUoJXubiNr8Cm7V4mywMDdIk\\\\\\\/MRvp1lTj4Ny91ODHoauWtS7NVrLzj5+Unr\\\\\\\/nn8t82hbiGLOSyY65LEwNvpwvK4kdrnRcNwZV8qY5jKHAWEGUqjuZk9EVBT1S6cEOno9oZRt7UUYASie\\\\\\\/TdGgNr4v5FtFZl68lVRqkdj9ME5lCw\\\\\\\/cCVBradKOYUXdkK2ak0Ug42BxiRysPs1yyiLXliR+6YuKuB7W\\\\\\\/M1FCbJ+ehahBiOEle40ymCuu0BeGXL3y8FYEDB1Rbdjnix54gA1AS\\\\\\\/VEB3LYEqb8+EOl1DA7c9QuNbznZU5h58zws5DoF8EWjrbiKl4ygmxSCoQLPC\\\\\\\/F+j\\\u0022,\\\u0022iv\\\u0022:\\\u0022tNCHjcEu0Vb0ugWu+XRgyA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002289cd20b01ef2cc1ab80342b3f3732f90234308b291ce5a9cb620b164f144561c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493251 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1975" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-E2CL-OOTHoHdBe5An_9xWVuKUheFWnXErwPhN7pA1nQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1975" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ztLNUMjZrR_06YraF0nlQxwB06bmlZGAKck1XzWKNCQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1957" title="Genome-Wide Analyses Reveal Footprints of Divergent Selection and Drought Adaptive Traits in Synthetic-Derived Wheats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1987" title="Little Evidence of Antagonistic Selection in the Evolutionary Strata of Fungal Mating-Type Chromosomes (Microbotryum lychnidis-dioicae)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493251" id="node493251" data-pisa="ggg;9/6/1975" data-pisa-master="ggg;g3.119.400154" data-apath="/ggg/9/6/1975.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8923-9733" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Haipeng Yu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Malachy T. Campbell</span>, <span class="highwire-citation-author" data-delta="2">Qi Zhang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harkamal Walia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1975-1986;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400154 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493251" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haipeng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haipeng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaipeng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8923-9733" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Haipeng Yu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malachy T. Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malachy%2BT.%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalachy%2BT.%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Malachy T. Campbell</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Harkamal Walia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harkamal%2BWalia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walia%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarkamal%2BWalia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harkamal Walia</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@vt.edu" class="" data-icon-position="" data-hide-link-title="0">morota@vt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/6/1975" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493251" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1975.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493251" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1975.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493251" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/6/1975.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493251" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">With the advent of high-throughput phenotyping platforms, plant breeders have a means to assess many traits for large breeding populations. However, understanding the genetic interdependencies among high-dimensional traits in a statistically robust manner remains a major challenge. Since multiple phenotypes likely share mutual relationships, elucidating the interdependencies among economically important traits can better inform breeding decisions and accelerate the genetic improvement of plants. The objective of this study was to leverage confirmatory factor analysis and graphical modeling to elucidate the genetic interdependencies among a diverse agronomic traits in rice. We used a Bayesian network to depict conditional dependencies among phenotypes, which can not be obtained by standard multi-trait analysis. We utilized Bayesian confirmatory factor analysis which hypothesized that 48 observed phenotypes resulted from six latent variables including grain morphology, morphology, flowering time, physiology, yield, and morphological salt response. This was followed by studying the genetics of each latent variable, which is also known as factor, using single nucleotide polymorphisms. Bayesian network structures involving the genomic component of six latent variables were established by fitting four algorithms (<em>i.e.</em>, Hill Climbing, Tabu, Max-Min Hill Climbing, and General 2-Phase Restricted Maximization algorithms). Physiological components influenced the flowering time and grain morphology, and morphology and grain morphology influenced yield. In summary, we show the Bayesian network coupled with factor analysis can provide an effective approach to understand the interdependence patterns among phenotypes and to predict the potential influence of external interventions or selection related to target traits in the interrelated complex traits systems.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bayesian-network" class="hw-term hw-article-keyword hw-article-keyword-bayesian-network" rel="nofollow">Bayesian network</a></li><li class="kwd"><a href="/keyword/factor-analysis" class="hw-term hw-article-keyword hw-article-keyword-factor-analysis" rel="nofollow">factor analysis</a></li><li class="kwd"><a href="/keyword/multi-phenotypes" class="hw-term hw-article-keyword hw-article-keyword-multi-phenotypes" rel="nofollow">multi-phenotypes</a></li><li class="kwd"><a href="/keyword/rice" class="hw-term hw-article-keyword hw-article-keyword-rice" rel="nofollow">rice</a></li></ul><p id="p-3">A primary objective in plant breeding is the develop high yielding varieties with specific grain qualities, resilience to pests and abiotic stresses, and superior adaption to the target environment. As a result, plant breeders devote considerable resources to extensive phenotypic evaluation of germplasm and select on multiple traits. These traits are often correlated at a genetic level through common genetic effects (<em>e.g.</em>, pleiotropy) or linkage disequilibrium between quantitative trait locus (QTL). Since multiple phenotypes may exhibit mutual relationships, knowledge of the interdependence among agronomically important traits can improve the efficacy of selection and rate of genetic improvement in systems with complex traits.</p><p id="p-4">In a standard quantitative genetic analysis, multivariate phenotypes can be modeled through multi-trait models (MTM) of <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Henderson and Quaas (1976)</a> or some genomic counterparts (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Calus and Veerkamp 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jia and Jannink 2012</a>) by leveraging genetic or environmental correlations among traits. In particular, MTM has been useful in deriving genetic correlations and enhancing the prediction accuracy of breeding values for traits with low heritability or scarce records via joint modeling with one or more genetically correlated, highly heritable traits (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mrode 2014</a>). Conventional MTM strategies may provide important insight into the genetic relations between agronomically important traits, but they fail to explain how these traits are related. For instance, consider a case where we have three genetically correlated traits: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-1.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-2.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-3.gif"/></noscript></span></span>. With MTM, we cannot address whether the relationship between <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-5.gif"/></noscript></span></span> is due to direct effects, or if the relationship is driven by indirect effects mediated by <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-6.gif"/></noscript></span></span>. Bayesian Networks (BN) offer an effective approach to elucidate the underlying network structure in multivariate data and infer network relationships between correlated variables. A BN is a probabilistic graphical model that represents conditional dependencies among a set of variables via a directed acyclic graph (DAG) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Neapolitan 2004</a>). In the DAG, the variables are represented by nodes, while their conditional dependencies between nodes are indicated with directed edges. In the context of plant breeding, BN can used to elucidate the interdependencies among traits and inform selection decisions for simultaneously improving multiple traits. For instance in the latter case above (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-7.gif"/></noscript></span></span>), selection directly on <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-8.gif"/></noscript></span></span> will affect the quantity of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-9.gif"/></noscript></span></span> without an effect on <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-10.gif"/></noscript></span></span>.</p><p id="p-5">With the advent high-throughput phenotyping (HTP) platforms, plant breeders have been provided with a suite of tools for phenotypic evaluation of large populations (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Shakoor <em>et al.</em> 2017</a>). These platforms leverage robotics, precise environmental control, and remote sensing techniques to provide accurate, repeatable and high resolution phenotypes for large breeding populations throughout the growing season (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Araus and Cairns 2014</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Shakoor <em>et al.</em> 2017</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Araus <em>et al.</em> 2018</a>). These data can be used to redefine characteristics underlying superior agronomic performance by quantifying secondary traits associated with seedling vigor, plant architecture, photosynthesis, transpiration, disease resistance, and stress tolerance (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cabrera-Bosquet <em>et al.</em> 2016</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sun <em>et al.</em> 2017</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Crain <em>et al.</em> 2018</a>). However given these new approaches, breeders are faced with the new challenge of efficiently utilizing these large multidimesional data sets to improve selection efficiency. The primary challenges associated with multivariate analysis and BN approaches using HTP data are that robust parameter estimates can be untenable because the number of estimated parameters within the model increases with the increasing number of phenotypes. Moreover, even in cases where MTM or BN can be applied, interpreting of interrelationships among a large number of phenotypes can be difficult.</p><p id="p-6">One approach to characterize high-dimensional phenotypes is by using factor analysis (FA). The central idea of FA approaches is to reduce the dimensions of multivariate data sets by constructing unobserved, latent factors, or modules, from correlated phenotypes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos and Gianola 2007</a>). The biological importance of these latent factors can be interpreted by inspecting the phenotypes that contribute to each factor. Thus, the advantage of FA for large, multivariate data sets is two fold. First, FA provides a means to reduce the dimensions of multivariate data sets thereby providing statistically sound parameter estimates, and easing visualization and interpretation. Second, the latent variables/factors themselves may be representative of underlying biological processes that cannot be observed or measured in the population. For instance, several studies have highlighted the effects of plant hormones such as GA on multiple morphological attributes (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wang and Li 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lo <em>et al.</em> 2008</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Umehara <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bhattacharya <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brewer <em>et al.</em> 2013</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhou <em>et al.</em> 2013</a>). Thus, a latent factor constructed from these morphological traits may provide information on the biosynthesis or sensitivity of these hormones for individuals within the population. If a certain amount of knowledge regarding the biological role of the variables is already known, a varaint of FA, confirmatory factor analysis (CFA), can be used to estimate latent variables based on predetermined biological classes of observed traits (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jöreskog 1969</a>). These latent variables underlie observed phenotypes and can be evaluated for how well the data support the hypothesis. For instance, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Peñagaricano <em>et al.</em> (2015)</a> performed CFA in swine to derive five latent variables from 19 phenotypic traits and inferred BN structures among those latent variables, thereby demonstrating the potential of this approach.</p><p id="p-7">This study aimed to leverage CFA and graphical modeling to elucidate the genetic interdependencies among traits typically recorded in breeding programs (<em>e.g.</em>, yield, plant morphology, phenology, and stress resilience). First, we constructed latent variables, using prior biological knowledge obtained from the literature. Then we connected the observed high-dimensional phenotypes with these to establish latent variables via Bayesian confirmatory factor analysis (BCFA) to reduce the dimensions of the dataset. Further, factor scores computed from BCFA were considered new phenotypes for a Bayesian multivariate analysis to separate breeding values from noise. This was followed by adjustment of breeding values via Cholesky decomposition to eliminate the dependencies introduced by genomic relationships. Finally, the adjusted breeding values were considered inputs to assess the network structure between latent variables by conducting a Gaussian BN analysis. This study is the first, to our knowledge, in rice to characterize various phenotypes with graphical modeling such as BCFA and BN.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Phenotypic and genotypic data</h3><p id="p-8">The rice dataset comprised <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-11.gif"/></noscript></span></span> 374 accessions sampled from six subpopulations: temperate japonica (92), tropical japonica (85), indica (77), aus (52), aromatic (12), and admixture of japonica and indica (56) (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> 2011</a>). The improvement status of each accession was obtained from the USDA-ARS Germplasm Resources Information Network. We used <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-12.gif"/></noscript></span></span> 48 phenotypes and data regarding 44,000 single-nucleotide polymorphisms (SNP). After removing SNP markers with minor allele frequency less than 0.05, 374 accessions and 33,584 markers were used for further analysis. Of those, 27 phenotypes were reported in <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> (2011)</a> and <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McCouch <em>et al.</em> (2016)</a>. These phenotypes can be classified into four categories: flowering time (flowering time at three locations, photoperiod sensitivity), grain morphology (seed length, seed width, seed surface area, seed length to width ratio, seed volume), plant morphology (culm habit/angle, flag leaf length and width, plant height at maturity), and yield traits (panicle fertility, seed number per panicle, number of primary branches on the main panicle, panicle length, and the number of panicles on each plant). <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> (2011)</a> evaluated flowering time-related traits using data from three locations, while the remaining traits were evaluated at one location (Arkansas). The remaining phenotypes were assessed from the salinity stress experiments conducted in <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> (2017)</a>. These traits were classified into three categories: morphological salt response, ionic components of salt stress, and plant morphology. The class morphological salt response represents how plant growth is affected by salinity stress and is composed of the ratio of shoot biomass of salt stressed plants to control, the ratio of root biomass of salt stressed plants to control, the ratio of the number of tillers for salt stressed plants to control, and two metrics that represent the ratio of shoot height of salt stressed plants to control. Ionic components of salt stress are composed of traits that quantify ions important for salinity tolerance (Na+ and K+) in both root and shoot tissues. Morphology traits are those that describe the growth of the plant in both control and saline conditions (<em>e.g.</em>, shoot biomass, root biomass, shoot height, and tiller number). The data used from <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> (2017)</a> were derived from three to six independent greenhouse experiments performed between July and October 2013. Information for all experiments were combined and best linear unbiased estimators were calculated for each line as described in <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> (2017)</a>. The detailed descriptions of the phenotypes are summarized in Supplementary Table S1.</p></div><div id="sec-3" class="subsection"><h3>Bayesian confirmatory factor analysis</h3><p id="p-9">A CFA under the Bayesian framework was performed to model 48 phenotypes. The number of factors and the pattern of phenotype-factor relationships need to be specified in BCFA prior to model fitting. We constructed six latent variables (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-13.gif"/></noscript></span></span>) from previous reports (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acquaah 2009</a>; <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> 2017</a>). The six latent variables derived from our analysis represent the grain morphology, morphology, flowering time, ionic components of salt stress, yield, and morphological salt response (Table S1). Each latent variable captures common signals spanning genetic and environmental effects across all its phenotypes. The latent variables, which determine the observed phenotypes can be modeled as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-14.gif"/></noscript></span></span>where T is the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-15.gif"/></noscript></span></span> matrix of observed phenotypes, Λ is the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-16.gif"/></noscript></span></span> factor loading matrix, F is the <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-17.gif"/></noscript></span></span> latent variables matrix, and s is the <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-18.gif"/></noscript></span></span> matrix of specific effects. Here, Λ maps latent variables to the observed variables and can be interpreted as the extent of contribution each latent variable to phenotype. This can be derived by solving the following variance-covariance model.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-19.gif"/></noscript></span></span>where Φ is the variance of latent variables, and Ψ is the variance of specific effects (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brown 2014</a>). Six latent variables were assumed to account for the covariance in the observed phenotypes. Moreover, latent variables were assumed to be correlated with each other. Prior distributions were assigned to all unknown parameters. The non-zero coefficients within factor loading matrix Λ were assumed to follow a Gaussian distribution with mean of 0 and variance of 0.01. The variance-covariance matrix Φ was assigned an inverse Wishart distribution with a 6 × 6 identity scale matrix <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-20.gif"/></noscript></span></span> and a degree of freedom 7, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-21.gif"/></noscript></span></span> and an inverse Gamma distribution with scale parameter 1 and shape parameter 0.5 was assigned to <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-22.gif"/></noscript></span></span>.</p><p id="p-10">We employed the blavaan R package (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Merkle and Rosseel 2018</a>) jointly with JAGS (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Plummer <em>et al.</em> 2003</a>) to fit the above BCFA. The blavaan runs the runjags R package (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Denwood 2016</a>) to summarize the Markov chain Monte Carlo (MCMC) and samples unknown parameters from the posterior distributions. Three MCMC chains, each of 5,000 samples with 2,000 burn-in, were used to infer the unknown model parameters. The convergence of the parameters was investigated with trace plots and potential scale reduction factor (PSRF) less than 1.2 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brooks and Gelman 1998</a>). The PSRF computes the difference between estimated variances among multiple Markov chains and estimated variances within the chain. A large difference indicates non-convergence and may require additional Gibbs sampling.</p><p id="p-11">Subsequently, the posterior means of factor scores (F), which reflect the contribution of latent variables to each accession were estimated. Within each draw of Gibbs sampling, F was sampled from the conditional distribution of <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-23.gif"/></noscript></span></span>, where θ refers to the unknown parameters in Λ, Φ, and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-24.gif"/></noscript></span></span> This conditional distribution was derived with data augmentation (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tanner and Wong 1987</a>) assuming F as missing data (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lee and Song 2012</a>).</p></div><div id="sec-4" class="subsection"><h3>Multivariate genomic best linear unbiased prediction</h3><p id="p-12">We fitted a Bayesian multivariate genomic best linear unbiased prediction to separate breeding values from population structure and noise in the six factor scores computed previously.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-25.gif"/></noscript></span></span>where μ is the vector of intercept, X is the incidence matrix of covariates, b is the vector of covariate effects, Z is the incidence matrix relating accessions with additive genetic effects, u is the vector of additive genetic effects, and ϵ is the vector of residuals. The incident matrix X included subpopulation information (temperate japonica, tropical japonica, indica, aus, aromatic, and admixture), as the rice diversity panel used herein shows a clear substructure (<a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> 2011</a>).</p><p id="p-13">A flat prior was assigned to μ and b, and the joint distribution of u and ϵ follows multivariate normal<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-26.gif"/></noscript></span></span>where G represents the second genomic relationship matrix of <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">VanRaden (2008)</a>, I is the identity matrix, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-28.gif"/></noscript></span></span> refer to <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-29.gif"/></noscript></span></span> dimensional genetic and residual variance-covariance matrices, respectively. An inverse Wishart distribution with a <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-30.gif"/></noscript></span></span> identity scale matrix of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-31.gif"/></noscript></span></span> and a degree of freedom 6 was assigned as prior for <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-32.gif"/></noscript></span></span>. These parameters were selected so that relatively uninformative priors were used. The Bayesian multivariate genomic best linear unbiased prediction model was implemented using the MTM R package (<a href="https://github.com/QuantGen/MTM">https://github.com/QuantGen/MTM</a>). Posterior mean estimates of genomic correlation between latent variables and predicted breeding values (<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-33.gif"/></noscript></span></span>) were then obtained. The convergence of the estimated parameters was verified by trace plots.</p></div><div id="sec-5" class="subsection"><h3>Sample independence in the Bayesian network</h3><p id="p-14">Theoretically, BN learning algorithms assume sample independence. In the multivariate genomic best linear unbiased prediction, the residuals between phenotypes were assumed independent through <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-34.gif"/></noscript></span></span>. However, phenotypic dependencies were introduced by the G matrix for the additive genetic effects, thereby potentially serving as a confounder. Thus, a transformation of <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-35.gif"/></noscript></span></span> was carried out to derive an adjusted <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-36.gif"/></noscript></span></span> by eliminating the dependencies in G. For a single trait model, the adjusted <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-37.gif"/></noscript></span></span> can be computed by premultiplying <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-38.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-39.gif"/></noscript></span></span>, where L is a lower triangular matrix derived from the Choleskey decompostion of G matrix (<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-40.gif"/></noscript></span></span>). Since <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-41.gif"/></noscript></span></span>, the distribution of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-42.gif"/></noscript></span></span> follows <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-43.gif"/></noscript></span></span> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Callanan and Harville 1989</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Vazquez <em>et al.</em> 2010</a>)<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-44.gif"/></noscript></span></span>This transformation can be extended to a multi-traits model by defining <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-45.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-46.gif"/></noscript></span></span> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Töpner <em>et al.</em> 2017</a>). Under the multivariate framework, u follows <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-47.gif"/></noscript></span></span> and the variance of <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-48.gif"/></noscript></span></span> is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-49.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-50.gif"/></noscript></span></span>. This adjusted <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-51.gif"/></noscript></span></span> was used to learn BN structures between predicted breeding values.</p></div><div id="sec-6" class="subsection"><h3>Bayesian network</h3><p id="p-15">A BN depicts the joint probabilistic distribution of random variables through their conditional independencies (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Scutari and Denis 2014</a>)<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-52.gif"/></noscript></span></span>where G represents a DAG = (<em>V</em>, <em>E</em>) with nodes (<em>V</em>) connected by one or more edges (<em>E</em>) conveying the probabilistic relationships and the random vector <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-53.gif"/></noscript></span></span> is <em>K</em> random variables. The joint probability distribution can be factorized as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-54.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-55.gif"/></noscript></span></span> denotes a set of parent nodes of child node <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-56.gif"/></noscript></span></span>. The DAG and joint probability distribution are governed by the Markov condition, which states that every random variable is independent of its non-descendants conditioned on its parents. A BN is known as a Gaussian BN, when all variables or phenotypes are defined as marginal or conditional Gaussian distribution as in the present study.</p><p id="p-16">The adjusted breeding values <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-57.gif"/></noscript></span></span> were used to infer a genomic network structure among the aforementioned six latent variables. There are three types of structure-learning algorithms for BN: constraint-based algorithms, score-based algorithms, and a hybrid of these two (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Scutari and Denis 2014</a>). The constraint-based algorithms can be originally traced to the inductive causation algorithm (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Verma and Pearl 1991</a>), which uses conditional independence tests for network inference. Briefly, the first step is to identify a d-separation set for each pair of nodes and confer an undirected edge between the two if they are not d-separated. The second step is to identify a v-structure for each pair of non-adjacent nodes, where a common neighbor is the outcome of two non-adjacent nodes. In the last step, compelled edges were identified and oriented, where neither cyclic graph nor new v-structures are permitted. The score-based algorithms are based on heuristic approaches, which first assign a goodness-of-fit score for an initial graph structure and then maximize this score by updating the structure (<em>i.e.</em>, add, delete, or reverse the edges of initial graph). The hybrid algorithm includes two steps, restrict and maximize, which harness both constraint-based and score-based algorithms to construct a reliable network. In this study, the two score-based (Hill Climbing and Tabu) and two hybrid algorithms (Max-Min Hill Climbing and General 2-Phase Restricted Maximization) were used to perform structure learning. A flow diagram to illustrate the concept of constraint-based Bayesian netwrok structure learning algorithm is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flow diagram to illustrate the concept of constraint-based structure learning algorithm for a Bayesian network. The A, B, C, D, and E represent five nodes or latent variables. S refers to a set of d-separation. The directed acyclic graph shown in Step 3 is one possible completed partially directed acyclic graph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-152330997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flow diagram to illustrate the concept of constraint-based structure learning algorithm for a Bayesian network. The A, B, C, D, and E represent five nodes or latent variables. S refers to a set of d-separation. The directed acyclic graph shown in Step 3 is one possible completed partially directed acyclic graph.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1975/F1.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/6/1975/F1.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493321" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Flow diagram to illustrate the concept of constraint-based structure learning algorithm for a Bayesian network. The A, B, C, D, and E represent five nodes or latent variables. S refers to a set of d-separation. The directed acyclic graph shown in Step 3 is one possible completed partially directed acyclic graph.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We quantified the strength of edges and uncertainty regarding the direction of networks, using 500 bootstrapping replicates with a size equal to the number of accessions and performed structure learning for each replicate in accordance with <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Scutari and Denis (2014)</a>. Non-parametric bootstrap resampling aimed at reducing the impact of the local optimal structures by computing the probability of the arcs and directions. Subsequently, 500 learned structures were averaged with a strength threshold of 85% or higher to produce a more robust network structure. This process, known as model averaging, returns the final network with arcs present in at least 85% among all 500 networks. Candidate networks were compared on the basis of the Bayesian information criterion (BIC) and Bayesian Gaussian equivalent score (BGe). The BIC accounts for the goodness-of-fit and model complexity, and BGe aims at maximizing the posterior probability of networks per the data. All BN were learned via the bnlearn R package (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Scutari 2010</a>). In bnlearn, the BIC score is rescaled by -2, which indicates that the larger BIC refers to a preferred model.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-19">Genotypic data regarding the rice accessions can be downloaded from the rice diversity panel website (<a href="http://www.ricediversity.org/">http://www.ricediversity.org/</a>). Phenotypic data used herein are available in <a id="xref-ref-63-6" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> (2011)</a>, <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> (2017)</a>, and Supplementary File S3. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7970642">https://doi.org/10.25387/g3.7970642</a>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-20">To elucidate the genetic interdependencies among traits typically recorded in breeding programs, we utilized a collection of 48 publicly available phenotypes recorded on a panel of diverse rice accessions (<a id="xref-ref-63-7" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> 2017</a>). The phenotypic data were derived from two independent studies. The first set of phenotypes was recorded from materials grown in two field environments in Arkansas and Faridpur Bangladesh, and in a greenhouse in Aberdeen, UK (<a id="xref-ref-63-8" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> 2011</a>). The 34 phenotypes were recorded at maturity and were largely associated with yield (panicle characteristics flowering time, plant morphology (<em>e.g.</em>, height and growth habits), and seed morphological traits. The second study consisted of 14 phenotypes were recorded in a greenhouse environment on plants in the active tillering stage (<em>e.g.</em>, 30 day-old plants) under control and saline (14 days of 9.5 dS m<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6/1975/embed/mml-math-58.gif"/></noscript></span></span> NaCl stress). The phenotypes from this study can be classified into three categories: morphological traits (<em>e.g.</em>, shoot and root biomass, and plant height), morphological responses to salinity (<em>e.g.</em>, the ratio of morphological traits in saline conditions to control), and the ionic components of salinity stress (<em>e.g.</em>, Na+, K+, and Na+:K+ in both root and shoot tissues) (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> 2017</a>). The complete data set provides an in-depth characterization of phenotypic performance at vegetative and reproductive stages in rice using several classes of traits.</p><div id="sec-9" class="subsection"><h3>Latent variable modeling</h3><p id="p-21">The BCFA model grouped the observed phenotypes into the underlying latent variables on the basis of prior biological knowledge, assuming these latent variables determine the observed phenotypes. This allowed us to study the genetics of each latent variable. A measurement model derived from BCFA evaluating the six latent variables is shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. Forty-eight observed phenotypes were hypothesized to result from the six latent variables: 7 for flowering time, 14 for morphology, 5 for yield, 11 for grain morphology, 6 for physiology, and 5 for salt response. The convergence of the parameters was confirmed graphically with the trace plots and a PSRF value less than 1.2 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brooks and Gelman 1998</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Merkle and Rosseel 2018</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between six latent variables and observed phenotypes. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time. Abbreviations of observed phenotypes are shown in Table S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-152330997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between six latent variables and observed phenotypes. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time. Abbreviations of observed phenotypes are shown in Table S1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1975/F2.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/6/1975/F2.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493273" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Relationship between six latent variables and observed phenotypes. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time. Abbreviations of observed phenotypes are shown in Table S1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The six latent factors showed strong contributions to the 48 observed phenotypes, with standardized regression coefficients ranging from -0.549 to 0.990 for flowering time, -0.349 to 0.925 for morphology, -0.085 to 0.790 for yield, -0.476 to 0.990 for grain morphology, -0.265 to 0.983 for ionic components of salt stress, and -0.022 to 0.939 for salt response. The latent factor flowering time showed a strong positive contribution to flowering time in Arkansas (Fla) and Flowering time in Arkansas in 2007 (Fla7) (0.990 and 0.926, respectively; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), indicating that larger values for the latent factor can be interpreted as a greater number of days from sowing to emergence of the inflorescence. The latent factor morphology showed the largest positive contributions to traits describing height during the vegetative stage (<em>e.g.</em>, height to newest ligule in salt (Hls), 0.920; height to newest ligule in control (Hlc), 0.899; height to the tip of first fully expanded leaf in salt (Hfs), 0.907; and height to tip of first fully expanded leaf in control (Hfc), 0.925;) suggesting that this latent factor is an overall representation of plant size. Yield showed large positive contributions to the observed phenotypes primary panicle branch number (Ppn) and seed number per panicle (Snpp) (0.790 and 0.780, respectively), suggesting that larger values for yield indicate a higher degree of branching and seed number. Observed phenotypes describing seed size (<em>e.g.</em>, seed volume (Sv) and brown rice volume (Bvl) (0.990 and 0.986, respectively)) were most strongly associated with grain morphology. The latent factor ionic components of salt stress showed strong positive contributions to two observed phenotypes that quantify the ionic components of salt stress (shoot Na+:K+ (Ks) and shoot Na+ (Nas) (0.983 and 0.975, respectively), indicating that higher values for the latent factor result in greater shoot Na+ and Na+:K+. Finally, the latent factor describing morphological salt response showed strong positive contributions to the observed phenotype describing the effect of salt treatment on plant height (ratio of height to tip of newest fully expanded leaf in salt to that of control plants (Hfr) (0.939)), thus larger values for the latent factor may indicate a more tolerant growth response to salinity.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493416/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493416/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Standardized factor loadings obtained from the Bayesian confirmatory factor analysis. PSD refers to the posterior standard deviation of standardized factor loadings</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Genomic correlation among latent variables</h3><p id="p-24">To understand the genetic relationships between latent variables, genomic correlation analysis was performed. Genomic correlation is due to pleiotropy or linkage disequilibrium between QTL. The genomic correlations among latent variables are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Negative correlations were observed between morphological salt response (Msr) and all other five latent variables. In particular, flowering time (-0.5), yield (-0.54), and grain morphology (-0.74) were negatively correlated with morphological salt response. These results suggest that accessions that harbor alleles for more tolerant morphological salt responses may also have alleles associated with longer flowering times, smaller seeds, and low yield. Similarly, a negative correlation was observed between morphology and yield (-0.56) and between morphology and grain morphology (-0.31). Thus, accessions with alleles associated with large plant size may also have alleles that result in low yield, small grain volume, and lower shoot Na+ and Na+:K+. In contrast, a positive correlation was observed between grain morphology and yield (0.49) and between grain morphology and ionic components of salt stress (0.4). Thus, selection for large grain may result in improved yield, and higher shoot Na+ and Na+:K+.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic correlation of six latent variables. The size of each circle, degree of shading, and value reported correspond to the correlation between each pair of latent variables. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-152330997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic correlation of six latent variables. The size of each circle, degree of shading, and value reported correspond to the correlation between each pair of latent variables. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1975/F3.medium.gif" width="440" height="416"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/6/1975/F3.medium.gif" width="440" height="416"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493360" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Genomic correlation of six latent variables. The size of each circle, degree of shading, and value reported correspond to the correlation between each pair of latent variables. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Bayesian network</h3><p id="p-26">To infer the possible network structure between latent variables, BN was performed. Prior to BN, the normality of latent variables was assessed using histogram plots combined with density curves as shown in Supplementary Figure S1. Overall, all the six latent variables approximately followed a Gaussian distribution.</p><p id="p-27">The Bayesian networks learned with the score-based and hybrid algorithms are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The structures of BN were refined by model averaging with 500 networks from bootstrap resampling to reduce the impact of local optimal structures. The labels of the arcs measure the uncertainty of the arcs, corresponding to strength and direction (in parenthesis). The former measures the frequency of the arc presented among all 500 networks from the bootstrapping replicates and the latter is the frequency of the direction shown conditional on the presence of the arc. We observed minor differences in the structures presented within and across the two types of algorithms used. In general, small differences were observed within algorithm types compared to those across algorithms. The two score-based algorithms produced a greater number of edges than two hybrid algorithms. The Hill Climbing algorithm produced seven directed connections among the six latent variables. Three connections were indicated toward flowering time from morphological salt response, ionic components of salt stress, and morphology, and two edges to yield from morphology and from grain morphology. Other two edges were observed from ionic components of salt stress to grain morphology and from grain morphology to morphological salt response. A similar structure was generated by the Tabu algorithm, except that the connection between salt response and grain morphology presented an opposite direction. The Max-Min Hill Climbing hybrid algorithm yielded six directed edges from morphological salt response to grain morphology, from ionic components of salt stress to grain morphology, from ionic components of salt stress to flowering time, from flowering time to morphology, from morphology to yield, and from grain morphology to yield. An analogous structure with the only difference observed in the directed edge from morphology to flowering time was inferred with the General 2-Phase Restricted Maximization algorithm. Across all four algorithms, there were four common directed edges: from ionic components of salt stress to flowering time and to grain morphology, and from morphology and grain morphology to yield. The most favorable network was considered the one from the Tabu algorithm, which returned the largest network score in terms of BIC (1086.61) and BGe (1080.88). Collectively, these results suggest that there may be a direct genetic influence of morphology and grain morphology on yield, and physiological components of salt tolerance on grain morphology and flowering time.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bayesian networks between six latent variables based on two score-based (4a: Hill Climbing and 4b: Tabu) and two hybrid (4c: Max-Min Hill Climbing and 4d: General 2-Phase Restricted Maximization) algorithms. The quality of the structure was evaluated by bootstrap resampling and model averaging across 500 replications. Labels of the edges refer to the strength and direction (parenthesis) which measure the confidence of the directed edge. The strength indicates the frequency of the edge is present and the direction measures the frequency of the direction conditioned on the presence of edge. BIC: Bayesian information criterion score. BGe: Bayesian Gaussian equivalent score. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-152330997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bayesian networks between six latent variables based on two score-based (4a: Hill Climbing and 4b: Tabu) and two hybrid (4c: Max-Min Hill Climbing and 4d: General 2-Phase Restricted Maximization) algorithms. The quality of the structure was evaluated by bootstrap resampling and model averaging across 500 replications. Labels of the edges refer to the strength and direction (parenthesis) which measure the confidence of the directed edge. The strength indicates the frequency of the edge is present and the direction measures the frequency of the direction conditioned on the presence of edge. BIC: Bayesian information criterion score. BGe: Bayesian Gaussian equivalent score. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1975/F4.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/6/1975/F4.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1975/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493370" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Bayesian networks between six latent variables based on two score-based (4a: Hill Climbing and 4b: Tabu) and two hybrid (4c: Max-Min Hill Climbing and 4d: General 2-Phase Restricted Maximization) algorithms. The quality of the structure was evaluated by bootstrap resampling and model averaging across 500 replications. Labels of the edges refer to the strength and direction (parenthesis) which measure the confidence of the directed edge. The strength indicates the frequency of the edge is present and the direction measures the frequency of the direction conditioned on the presence of edge. BIC: Bayesian information criterion score. BGe: Bayesian Gaussian equivalent score. Msr: morphological salt response; Iss: ionic components of salt stress; Grm: grain morphology; Yid: yield; Mrp: morphology; Flt: flowering time.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-29">This study is based on the premise that most phenotypes interact to greater or lesser degrees with each other through underlying physiological and molecular pathways. While these physiological pathways are important for the development of agronomically important characteristics, they are often unknown or difficult to assess in large populations. The approach utilized here leverages phenotypes that can be readily assessed in large populations to quantify these underlying unobserved phenotypes, and elucidates the relationships between these variables.</p><p id="p-30">Understanding the behaviors among phenotypes in the complex traits is critical for genetic improvement of agricultural species (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hickey <em>et al.</em> 2017</a>). Graphical modeling offers an avenue to decipher bi-directional associations or probabilistic dependencies among variables of interest in plant and animal breeding. For instance, BN and L1-regularized undirected network can be used to model interrelationships of linkage disequilibrium (LD) (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Morota <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Morota and Gianola 2013</a>) or phenotypic, genetic, and environmental interactions (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Xavier <em>et al.</em> 2017</a>) in a systematic manner. Importantly, MTM elucidates both direct and indirect relationships among phenotypes. Inaccurate interpretation of these relationships may substantially bias selection decisions (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Valente <em>et al.</em> 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> 2015</a>). Thus, we applied BCFA to reduce the dimension of the responses by hypothesizing 48 manifest phenotypes originated from the underlying six constructed latent variables as shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> assuming that these latent traits are most important, followed by application of BN to infer the structures among the six biologically relevant latent variables (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Note that there are two differences between the approach employed here and a path analysis. A path analysis 1) uses observed variables rather than latent variables and 2) assumes a network structure is known priori. Thus, one advantage of our approach is that it can model a network structure at the level of latent variables and infer a network structure directly from data when prior information is not available from the literature or previous experiments. The BN represents the conditional dependencies between variables. Care must be taken in interpreting these relationships as a causal effect. Although a good BN is expected to describe the underlying causal structure per the data, when the structure is learned solely on the basis of the observed data, it may return multiple equivalent networks that describe the data well. In practice, searching such a causal structure with observed data needs three additional assumptions (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Scutari and Denis 2014</a>): 1) each variable is independent of its non-effects (<em>i.e.</em>, direct and indirect) conditioned on its direct causes, 2) the probability distribution of variables is supported by a DAG, where the d-separation in DAG provides all dependencies in the probability distribution, and 3) no additional variables influence the variables within the network. Although it may be difficult to meet these assumptions in the observed data, a BN is equipped with suggesting potential causal relationships among latent variables, which can assist in exploring data, making breeding decisions, and improving management strategies in breeding programs (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rosa <em>et al.</em> 2011</a>).</p><div id="sec-13" class="subsection"><h3>Biological meaning of latent variables and their relationships</h3><p id="p-31">We performed BCFA to summarize the original 48 phenotypes with the six latent variables. The number of latent variables and which latent variables load onto phenotypes were determined from the literature. The latent variable morphological salt response (Msr) contributed strongly to salt indices for shoot biomass, root biomass, and two indices for plant height (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Thus, morphological salt response can be interpreted as the morphological responses to salinity stress, with higher values indicating a more tolerant growth response. The latent variable yield is a representation of overall grain productivity, and contributed strongly to the observed phenotypes primary panicle branch number, seed number per panicle, and panicle length. The positive loading scores on these observable phenotypes indicates that more highly branched, productive panicles will have higher values for yield (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Seed width, seed volume, and seed surface area contributed significantly to the latent variable grain morphology (Grm) (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Therefore, these results indicate that the grain morphology is a summary of the overall shape of the grain, where high values represent large, round grains, while low values represent small, slender grains. Considering the grain characteristics of rice subpopulations, temperate japonica accessions are expected to have high values for grain morphology, while indica accessions have lower values for grain morphology. Latent variable morphology (Mrp) is a representation of plant biomass during the vegetative stage (28-day-old plants) (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Shoot biomass, root biomass, and two metrics for plant height contributed largely to morphology, suggesting that accessions with high values for morphology are tall plants with a large biomass.</p><p id="p-32">Genomic correlation analysis among the six latent variables showed meaningful correlations among several pairs. These genetic correlations can either be caused by linkage or pleiotropy. The former is likely to prevail in species with high LD, which is the case in rice where LD ranges from 100 to 200kb (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2010</a>). A negative relationship was observed between morphological salt response and three other latent variables (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). For instance, a negative correlation between morphological salt response and yield indicates that accessions of samples harboring alleles for superior morphological salt responses (<em>e.g.</em>, those that are more tolerant) tend to also harbor alleles for poor yield (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). The rice diversity panel we used is a representative sample of the total genetic diversity within cultivated rice and contains many unimproved traditional varieties (∼12% of lines in the study are landraces and ∼33% classified as cultivars; Supplementary File S2) and modern breeding lines (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Eizenga <em>et al.</em> 2014</a>). While traditional varieties exhibit superior adaptation to abiotic stresses, they often have very poor agronomic characteristics including low yield, late flowering, and high photoperiod sensitivity (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Thomson <em>et al.</em> 2009</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">2010</a>). Moreover, the indica and japonica subspecies have contrasting salt responses and very different grain morphology. Japonica accessions tend to have short, round seeds and are more sensitive to salt stress, while indica accessions have long, slender grains and often are more salt tolerant (<a id="xref-ref-63-9" class="xref-bibr" href="#ref-63">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> 2017</a>). The negative relationship observed between morphological salt response and grain morphology suggests that lines that harbor alleles for high grain morphology (<em>e.g.</em>, large, round grains) tend to also harbor alleles for a tolerant growth response to salt stress. However, no studies have yet reported an association between alleles for grain morphology and morphological salt response. Therefore, it remains to be addressed whether this relationship is due to LD or pleitropy.</p><p id="p-33">Genetic correlations observed between other latent variables may suggest a pleiotropic effect among loci. For instance, a negative relationship was observed between morphological salt response and ionic components of salt stress, indicating that accessions harboring alleles associated with superior morphological salt response also tend to harbor alleles for reduced ion content under salt stress (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). The relationship between salt tolerance, measured in terms of growth or yield, and Na+ and Na+:K+ has been a documented for decades (reviewed by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Munns and Tester (2008)</a>). Moreover, natural variation for Na+ transporters has been utilized to improve growth and yield under saline conditions in rice and other cereals (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ren <em>et al.</em> 2005</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Byrt <em>et al.</em> 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Horie <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Munns <em>et al.</em> 2012</a>; <a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Campbell <em>et al.</em> 2017</a>). Therefore, the negative genetic relationships observed between morphological salt response and ion content may be due to the pleiotropic effects of some loci.</p><p id="p-34">The genomic relationships among latent variables including morphology, yield, and grain morphology may have resulted from the selection of alleles associated with good agronomic characteristics. A positive relationship was observed between yield and grain morphology, suggesting that alleles that positively contribute to productive panicles also may contribute to large, round grains. Furthermore, the negative genomic correlation observed between morphology and yield indicates that alleles negatively influencing total plant biomass also have a positive contribution to traits for productive panicles. This genomic relationship may reflect the genetics of harvest index, which is defined as the ratio of grain yield to total biomass. Over the past 50 years, rice breeders have selected high harvest index, resulting in plants with short compact morphology and many highly productive panicles (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hay 1995</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Peng <em>et al.</em> 2008</a>).</p><p id="p-35">Although BCFA may yield biologically meaningful results, a potential limitation of BCFA is that we assumed each phenotype does not measure more than one latent variable. This assumption may not always strictly concur with the observational data. Therefore, further studies are required to allow each phenotype to potentially load onto multiple factors in the BCFA framework. An alternative approach is to derive the number of latent variables and determine which latent variables load onto phenotypes directly from observed data, using exploratory FA. This approach was not pursued here because accurate estimation of unknown parameters in the exploratory FA requires a large sample size, which was not the case herein (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Brown 2014</a>).</p></div><div id="sec-14" class="subsection"><h3>Bayesian network of latent variables</h3><p id="p-36">The BN is a probabilistic DAG, which represents the conditional dependencies among phenotypes. The genomic correlation among latent variables described in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a> does not inform the flow of genetic signals nor distinguish direct and indirect associations, whereas BN displays directions between latent variables and separate direct and indirect associations. Therefore, the BN describes the possibility that other phenotypes will change if one phenotype is intervened (<em>i.e.</em>, selection). However, caution is required to interpret this network as a causal effect, as the causal BN requires more assumptions, which are usually difficult to meet in observational data (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pearl 2009</a>).</p><p id="p-37">Four common edges or consensus subnetworks across the four BN may be the most reliable substructure of latent variables and may describe the dependence between agronomic traits (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). For example, edges from grain morphology to yield and morphology to yield can be interpreted as final grain productivity is dependant on specific vegetative characteristics as well grain traits. This is because yield, which represents the overall grain productivity of a plant, depends on morphological characteristics such as the degree of tillering, an architecture that allows the plant to efficiently capture light and carbon, and a stature that is resistant to lodging, the degree of panicle branching, as well as specific grain characteristics such as seed volume and shape. Moreover, there is a direct biological linkage between specific vegetative architectural traits such as tillering and plant height, and yield related traits such as panicle branching and number of seeds per panicle. The degree of branching during both vegetative and reproductive development is dependant on the development and initiation of auxiliary meristems. Several genes have been identified in this pathway and have shown to have pleiotropic effects on tillering and panicle branching (reviewed by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liang <em>et al.</em> (2014)</a>). For instance, <em>OsSPL14</em> has been shown to be an important regulator of auxiliary branching in both vegetative and reproductive stages in rice (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jiao <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Miura <em>et al.</em> 2010</a>). Moreover, other genes such as <em>OsGhd8</em> have been reported to regulate other morphological traits such as plant height and yield through increase panicle branching (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yan <em>et al.</em> 2011</a>). The biological importance of these dependencies can also be illustrated by viewing them in the context of genetic improvement, as selection for specific architectural traits (represented by the latent variable morphology) and grain characteristics have traditionally been used as traits to improve rice productivity in many conventional breeding programs (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Redoña and Mackill 1998</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2013</a>).</p><p id="p-38">While the above example provides a plausible network structure between latent variables, edges from ionic components of salt stress to flowering time and to grain morphology are an example of instances where caution should be used to infer causation. As mentioned above, there is an inherent difference in salt tolerance and grain morphological traits between the indica and japonica subspecies. The edges observed for these two latent variables (ionic components of salt stress and grain morphology) in BN may be driven by LD between alleles associated with grain morphology and alleles for salt tolerance rather than pleitropy. Thus, given the current data set, genetic effects for grain morphology may still be conditionally dependant on ionic components of salt stress and the BN may be true, even if there is no direct overlap in the genetic mechanisms for the two traits.</p><p id="p-39">We found that there are some uncertain edges among BN in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>. For instance, direction from morphological salt response to grain morphology is supported by 65% (Tabu), 58% (Max-Min Hill Climbing), and 58% (General 2-Phase Restricted Maximization) bootstrap sampling, whereas the opposite direction is supported by 56% bootstrap sampling (Hill Climbing). An analogous uncertainty was also observed between morphology and flowering time, <em>i.e.</em>, the path from morphology to flowering time was supported 60% (Hill Climbing), 51% (Tabu), and 52% (General 2-Phase Restricted Maximization), while the reverse direction was supported 51% (Max-Min Hill Climbing) upon bootstrapping. In addition, the two score-based algorithms captured edges between morphological salt response and flowering time with 70% and 76% bootstrapping evidence. However, this connection was not detected in the two hybrid algorithms. In general, inferring the direction of edges was harder than inferring the presence or absence of undirected edges. Finally, the whole structures of BN were evaluated in terms of the BIC score and BGe. Ranking of the networks was consistent across BIC and BGe and the two score-based algorithms produced networks with greater goodness-of-fit than the two hybrid algorithms. The optimal network was produced by the Tabu algorithm. This is consistent with the previous study reporting that the score-based algorithm produced a better fit of networks in data on maize (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Töpner <em>et al.</em> 2017</a>).</p><p id="p-40">In conclusion, the present results show the utility of CFA and network analysis to characterize various phenotypes in rice. We showed that the joint use of BCFA and BN can be applied to predict the potential influence of external interventions or selection associated with target traits such as yield in the high-dimensional interrelated complex traits system. We contend that the approaches used herein provide greater insights than pairwise-association measures of multiple phenotypes and can be used to analyze the massive amount of diverse image-based phenomics dataset being generated by the automated plant phenomics platforms (<em>e.g.</em>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Furbank and Tester 2011</a>). With a large volume of complex traits being collected through phenomics, numerous opportunities to forge new research directions are generated by using network analysis for the growing number of phenotypes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">This work was supported by the National Science Foundation under Grant Number 1736192 to QZ, HW, and GM and Virginia Polytechnic Institute and State University startup funds to GM.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7970642">https://doi.org/10.25387/g3.7970642</a>.</p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: P. Brown</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-05"><span class="received-label">Received </span>March 5, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-04-15"><span class="accepted-label">Accepted </span>April 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Yu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Acquaah</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Principles of plant genetics and breeding</span>, <span class="cit-publ-name">John Wiley &amp; Sons</span>, ‎<span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.2" data-doi="10.1016/j.tplants.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Field high-throughput phenotyping: the new crop breeding frontier.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2013.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139902&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330013700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.3" data-doi="10.1016/j.tplants.2018.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kefauver</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaman-Allah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Translating high-throughput phenotyping into genetic gain.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">466</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2018.02.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D23%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2018.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2018.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.4" data-doi="10.1007/s00344-009-9126-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kourmpetli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Practical applications of manipulating plant architecture by regulating gibberellin metabolism.</span> <abbr class="cit-jnl-abbrev">J. Plant Growth Regul.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00344-009-9126-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BGrowth%2BRegul.%26rft.volume%253D29%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00344-009-9126-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00344-009-9126-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000277939500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.5" data-doi="10.1093/mp/sss130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koltai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beveridge</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Diverse roles of strigolactones in plant development.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mp/sss130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fsss130%26rft_id%253Dinfo%253Apmid%252F23155045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/sss130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23155045&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314117100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.6" data-doi="10.2307/1390675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">General methods for monitoring convergence of iterative simulations.</span> <abbr class="cit-jnl-abbrev">J. Comput. Graph. Stat.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BGraph.%2BStat.%26rft.volume%253D7%26rft.spage%253D434%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1390675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1390675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077362100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Confirmatory factor analysis for applied research</span>, <span class="cit-publ-name">Guilford Publications</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.8" data-doi="10.1104/pp.106.093476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrt</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platten</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spielmeyer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagudah</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Hkt1; 5-like cation transporters linked to na+ exclusion loci in wheat, nax2 and kna1.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1918</span>–<span class="cit-lpage">1928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.106.093476</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.106.093476%26rft_id%253Dinfo%253Apmid%252F17322337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQzLzQvMTkxOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xOTc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.9" data-doi="10.1111/nph.14027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera-Bosquet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fournier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brichet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welcker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-throughput estimation of incident light, light interception and radiation-use efficiency of thousands of plants in a phenotyping platform.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.14027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callanan</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harville</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Some new algorithms for computing maximum likelihood estimates of variance components</span>, <span class="cit-publ-name">Iowa State University. Department of Statistics. Statistical Laboratory</span>, <span class="cit-publ-loc">Ames, IA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.11" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.12" data-doi="10.1371/journal.pgen.1006823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al Shiblawi</span>  <span class="cit-name-given-names">F. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Allelic variants of oshkt1; 1 underlie the divergence between indica and japonica subspecies of rice (oryza sativa) for root sodium content.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e1006823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006823</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006823%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.13" data-doi="10.3835/plantgenome2017.05.0043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Combining high-throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.05.0043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.05.0043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.05.0043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.14" data-doi="10.1051/gse:20070016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Factor analysis models for structuring covariance matrices of additive genetic effects: a bayesian implementation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">481</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-39-5-481</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253Dde%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D481%26rft.epage%253D494%26rft.atitle%253DFactor%2Banalysis%2Bmodels%2Bfor%2Bstructuring%2Bcovariance%2Bmatrices%2Bof%2Badditive%2Bgenetic%2Beffects%253A%2Ba%2BBayesian%2Bimplementation.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A20070016%26rft_id%253Dinfo%253Apmid%252F17897592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:20070016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17897592&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denwood</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">runjags: An r package providing interface utilities, model templates, parallel computing methods and additional distributions for mcmc models in jags. Journal of Statistical Software</span>. <abbr class="cit-jnl-abbrev">Articles</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArticles%26rft.volume%253D71%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.16" data-doi="10.3198/jpr2013.03.0013crmp"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeater</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClung</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Registration of the rice diversity panel 1 for genomewide association studies.</span> <abbr class="cit-jnl-abbrev">J. Plant Regist.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3198/jpr2013.03.0013crmp</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BRegist.%26rft.volume%253D8%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.3198%252Fjpr2013.03.0013crmp%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3198/jpr2013.03.0013crmp&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.17" data-doi="10.1016/j.tplants.2011.09.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furbank</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tester</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phenomics-technologies to relieve the phenotyping bottleneck.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2011.09.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DFurbank%26rft.auinit1%253DR.%2BT.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D635%26rft.epage%253D644%26rft.atitle%253DPhenomics--technologies%2Bto%2Brelieve%2Bthe%2Bphenotyping%2Bbottleneck.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.09.005%26rft_id%253Dinfo%253Apmid%252F22074787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22074787&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298724900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.18" data-doi="10.1534/genetics.115.179978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Do molecular markers inform about pleiotropy?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.179978</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179978%26rft_id%253Dinfo%253Apmid%252F26205989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.19" data-doi="10.1111/j.1744-7348.1995.tb05015.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hay</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Harvest index: a review of its use in plant breeding and crop physiology.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Biol.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1744-7348.1995.tb05015.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BBiol.%26rft.volume%253D126%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7348.1995.tb05015.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7348.1995.tb05015.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QZ99600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.20" data-doi="10.2527/jas1976.4361188x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quaas</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Multiple trait evaluation using relatives’ records.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1188</span>–<span class="cit-lpage">1197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1976.4361188x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1197%26rft.atitle%253DMultiple%2BTrait%2BEvaluation%2BUsing%2BRelatives%2527%2BRecords%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1976.4361188x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1976.4361188x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976CP41700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.21" data-doi="10.1038/ng.3920"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiurugwi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eggen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3920</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1297%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3920%26rft_id%253Dinfo%253Apmid%252F28854179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3920&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28854179&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.22" data-doi="10.1016/j.tplants.2009.08.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Hkt transporter-mediated salinity resistance mechanisms in arabidopsis and monocot crop plants.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">660</span>–<span class="cit-lpage">668</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2009.08.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DHorie%26rft.auinit1%253DT.%26rft.volume%253D14%26rft.issue%253D12%26rft.spage%253D660%26rft.epage%253D668%26rft.atitle%253DHKT%2Btransporter-mediated%2Bsalinity%2Bresistance%2Bmechanisms%2Bin%2BArabidopsis%2Band%2Bmonocot%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2009.08.009%26rft_id%253Dinfo%253Apmid%252F19783197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2009.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19783197&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272855700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.23" data-doi="10.1016/j.tplants.2012.11.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic bases of rice grain shape: so many genes, so little known.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">226</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2012.11.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D18%26rft.spage%253D218%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2012.11.001%26rft_id%253Dinfo%253Apmid%252F23218902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2012.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23218902&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318333700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.24" data-doi="10.1038/ng.695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies of 14 agronomic traits in rice landraces.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">967</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2B14%2Bagronomic%2Btraits%2Bin%2Brice%2Blandraces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.695%26rft_id%253Dinfo%253Apmid%252F20972439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972439&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.25" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiple-trait genomic selection methods increase genetic value prediction accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTk3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.26" data-doi="10.1038/ng.591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regulation of osspl14 by osmir156 defines ideal plant architecture in rice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">541</span>–<span class="cit-lpage">544</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.591</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJiao%26rft.auinit1%253DY.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D541%26rft.epage%253D544%26rft.atitle%253DRegulation%2Bof%2BOsSPL14%2Bby%2BOsmiR156%2Bdefines%2Bideal%2Bplant%2Barchitecture%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.591%26rft_id%253Dinfo%253Apmid%252F20495565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20495565&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278081500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.27" data-doi="10.1007/BF02289343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jöreskog</span>  <span class="cit-name-given-names">K. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">A general approach to confirmatory maximum likelihood factor analysis.</span> <abbr class="cit-jnl-abbrev">Psychometrika</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02289343</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D34%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02289343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02289343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1969D643000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Basic and advanced Bayesian structural equation modeling: With applications in the medical and behavioral sciences</span>, <span class="cit-publ-name">John Wiley &amp; Sons‎</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.29" data-doi="10.1016/j.gene.2013.11.058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Tillering and panicle branching genes in rice.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">537</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2013.11.058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D537%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2013.11.058%26rft_id%253Dinfo%253Apmid%252F24345551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2013.11.058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24345551&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331428300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.30" data-doi="10.1105/tpc.108.060913"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">S.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsing</span>  <span class="cit-name-given-names">Y.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeevaart</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A novel class of gibberellin 2-oxidases control semidwarfism, tillering, and root development in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2603</span>–<span class="cit-lpage">2618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.108.060913</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.108.060913%26rft_id%253Dinfo%253Apmid%252F18952778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMC8yNjAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.31" data-doi="10.1038/ncomms10532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maron</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Open access resources for genome-wide association mapping in rice.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">10532</span> (erratum: Nat. Commun. 7: 11346).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms10532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D10532%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms10532%26rft_id%253Dinfo%253Apmid%252F26842267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms10532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26842267&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merkle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosseel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">blavaan: Bayesian structural equation models via parameter expansion. Journal of Statistical Software</span>. <abbr class="cit-jnl-abbrev">Articles</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArticles%26rft.volume%253D85%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.33" data-doi="10.1038/ng.592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsubara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">X.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Osspl14 promotes panicle branching and higher grain productivity in rice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">549</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.592</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMiura%26rft.auinit1%253DK.%26rft.volume%253D42%26rft.issue%253D6%26rft.spage%253D545%26rft.epage%253D549%26rft.atitle%253DOsSPL14%2Bpromotes%2Bpanicle%2Bbranching%2Band%2Bhigher%2Bgrain%2Bproductivity%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.592%26rft_id%253Dinfo%253Apmid%252F20495564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20495564&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278081500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.34" data-doi="10.1007/s00122-013-2112-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evaluation of linkage disequilibrium in wheat with an l1-regularized sparse markov network.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1991</span>–<span class="cit-lpage">2002</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2112-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D1991%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2112-y%26rft_id%253Dinfo%253Apmid%252F23661079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2112-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23661079&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.35" data-doi="10.1111/jbg.12002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An assessment of linkage disequilibrium in holstein cattle using a bayesian network.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">474</span>–<span class="cit-lpage">487</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMorota%26rft.auinit1%253DG.%26rft.volume%253D129%26rft.issue%253D6%26rft.spage%253D474%26rft.epage%253D487%26rft.atitle%253DAn%2Bassessment%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2BHolstein%2Bcattle%2Busing%2Ba%2BBayesian%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12002%26rft_id%253Dinfo%253Apmid%252F23148973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23148973&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.36" data-doi="10.1079/9781780643915.0000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Linear models for the prediction of animal breeding values</span>, <span class="cit-publ-name">Cabi</span>, <span class="cit-publ-loc">Wallingford, UK</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1079/9781780643915.0000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLinear%2Bmodels%2Bfor%2Bthe%2Bprediction%2Bof%2Banimal%2Bbreeding%2Bvalues%26rft_id%253Dinfo%253Adoi%252F10.1079%252F9781780643915.0000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/9781780643915.0000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.37" data-doi="10.1038/nbt.2120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munns</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Athman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conn</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Wheat grain yield on saline soils is improved by an ancestral na+ transporter gene.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">360</span>–<span class="cit-lpage">364</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMunns%26rft.auinit1%253DR.%26rft.volume%253D30%26rft.issue%253D4%26rft.spage%253D360%26rft.epage%253D364%26rft.atitle%253DWheat%2Bgrain%2Byield%2Bon%2Bsaline%2Bsoils%2Bis%2Bimproved%2Bby%2Ban%2Bancestral%2BNa%252B%2Btransporter%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2120%26rft_id%253Dinfo%253Apmid%252F22407351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22407351&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.38" data-doi="10.1146/annurev.arplant.59.032607.092911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munns</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tester</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mechanisms of salinity tolerance.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">681</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.arplant.59.032607.092911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DMunns%26rft.auinit1%253DR.%26rft.volume%253D59%26rft.spage%253D651%26rft.epage%253D681%26rft.atitle%253DMechanisms%2Bof%2Bsalinity%2Btolerance.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.59.032607.092911%26rft_id%253Dinfo%253Apmid%252F18444910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.59.032607.092911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18444910&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256593200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neapolitan</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Learning bayesian networks</span>, <span class="cit-vol">Vol. 38</span>. <span class="cit-publ-name">Pearson Prentice Hall Upper Saddle River</span>, <span class="cit-publ-loc">NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.40" data-doi="10.1017/CBO9780511803161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearl</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>Causality: Models</em>, <em>Reasoning and Inference</em></span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">New York, NY</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/CBO9780511803161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCausality%253A%2BModels%252C%2BReasoning%2Band%2BInference%26rft_id%253Dinfo%253Adoi%252F10.1017%252FCBO9780511803161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/CBO9780511803161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.41" data-doi="10.2527/jas.2015-9213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peñagaricano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steibel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ernst</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Searching for causal networks involving latent variables in complex traits: application to growth, carcass, and meat quality traits in pigs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">4617</span>–<span class="cit-lpage">4623</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2015-9213</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D93%26rft.spage%253D4617%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2015-9213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2015-9213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.42" data-doi="10.1016/j.fcr.2008.04.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khush</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Progress in ideotype breeding to increase rice yield potential.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2008.04.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D108%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2008.04.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2008.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257479400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.6.1975.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> Jags: <span class="cit-comment">A program for analysis of Bayesian graphical models using gibbs sampling. In Proceedings of the 3rd international workshop on distributed statistical computing, volume 124, p. 125.10, Vienna, Austria.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.44" data-doi="10.1007/s001220050826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Redoña</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackill</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Quantitative trait locus analysis for rice panicle and grain characteristics.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">957</span>–<span class="cit-lpage">963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s001220050826</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D96%26rft.spage%253D957%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050826&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.45" data-doi="10.1038/ng1643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Z.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A rice quantitative trait locus for salt tolerance encodes a sodium transporter.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1643</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRen%26rft.auinit1%253DZ.%2BH.%26rft.volume%253D37%26rft.issue%253D10%26rft.spage%253D1141%26rft.epage%253D1146%26rft.atitle%253DA%2Brice%2Bquantitative%2Btrait%2Blocus%2Bfor%2Bsalt%2Btolerance%2Bencodes%2Ba%2Bsodium%2Btransporter.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1643%26rft_id%253Dinfo%253Apmid%252F16155566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1643&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16155566&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232185600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.46" data-doi="10.1186/1297-9686-43-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Inferring causal phenotype networks using structural equation models.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DRosa%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D43%26rft.spage%253D6%26rft.epage%253D6%26rft.atitle%253DInferring%2Bcausal%2Bphenotype%2Bnetworks%2Busing%2Bstructural%2Bequation%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-6%26rft_id%253Dinfo%253Apmid%252F21310061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21310061&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Learning bayesian networks with the bnlearn r package. Journal of Statistical Software</span>. <abbr class="cit-jnl-abbrev">Articles</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArticles%26rft.volume%253D35%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.48" data-doi="10.1201/b17065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scutari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">J.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Bayesian networks: with examples in R</span>, <span class="cit-publ-name">Chapman and Hall/CRC</span>, ‎<span class="cit-publ-loc">Boca Raton, FL</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1201/b17065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2Bnetworks%253A%2Bwith%2Bexamples%2Bin%2BR%26rft_id%253Dinfo%253Adoi%252F10.1201%252Fb17065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1201/b17065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.49" data-doi="10.1016/j.pbi.2017.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shakoor</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mockler</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High throughput phenotyping to accelerate crop breeding and monitoring of diseases in the field.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">184</span>–<span class="cit-lpage">192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2017.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D38%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2017.05.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2017.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.50" data-doi="10.3835/plantgenome2016.11.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.11.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.11.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.11.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.51" data-doi="10.2307/2289457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanner</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The calculation of posterior distributions by data augmentation.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">540</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01621459.1987.10478458</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D82%26rft.spage%253D528%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2289457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2289457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987J105700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.52" data-doi="10.1007/s12284-010-9053-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Ocampo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egdane</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sajise</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Characterizing the saltol quantitative trait locus for salinity tolerance in rice.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12284-010-9053-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D3%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12284-010-9053-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12284-010-9053-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000282788200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1975.53" data-doi="10.1007/978-90-481-3112-9_20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ismail</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackill</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Marker assisted breeding</span>, pp. <span class="cit-fpage">451</span>–<span class="cit-lpage">469</span> in <span class="cit-source">Abiotic Stress Adaptation in Plants</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-90-481-3112-9_20</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAbiotic%2BStress%2BAdaptation%2Bin%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-90-481-3112-9_20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-90-481-3112-9_20&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.54" data-doi="10.1534/g3.117.044263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Töpner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bayesian networks illustrate genomic and residual trait connections in maize (Zea mays L.). G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2779</span>–<span class="cit-lpage">2789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.044263</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D7%26rft.spage%253D2779%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.044263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.117.044263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.55" data-doi="10.1038/nature07272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Umehara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akiyama</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arite</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Inhibition of shoot branching by new terpenoid plant hormones.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">455</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07272</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DUmehara%26rft.auinit1%253DM.%26rft.volume%253D455%26rft.issue%253D7210%26rft.spage%253D195%26rft.epage%253D200%26rft.atitle%253DInhibition%2Bof%2Bshoot%2Bbranching%2Bby%2Bnew%2Bterpenoid%2Bplant%2Bhormones.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07272%26rft_id%253Dinfo%253Apmid%252F18690207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18690207&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259090800040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.56" data-doi="10.1534/genetics.114.169490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valente</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peñagaricano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The causal meaning of genomic predictors and how it affects construction and comparison of genome-enabled selection models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">494</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.169490</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169490%26rft_id%253Dinfo%253Apmid%252F25908318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE5NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.57" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.58" data-doi="10.2527/jas.2009-1952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An r package for fitting generalized linear mixed models in animal breeding 1.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2009-1952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2009-1952%26rft_id%253Dinfo%253Apmid%252F19820058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2009-1952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19820058&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274040700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.6.1975.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearl</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> Equivalence and synthesis of causal models. In <em>Proceedings of the Sixth Annual Conference on Uncertainty in Artificial Intelligence</em>, UAI ’90, pp. <span class="cit-fpage">255</span>–<span class="cit-lpage">270</span>, <span class="cit-publ-loc">New York, NY, USA</span>, <span class="cit-publ-name">Elsevier Science Inc.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.60" data-doi="10.1016/j.copbio.2006.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genes controlling plant architecture.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.copbio.2006.02.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DWang%26rft.auinit1%253DY.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D123%26rft.epage%253D129%26rft.atitle%253DGenes%2Bcontrolling%2Bplant%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2006.02.004%26rft_id%253Dinfo%253Apmid%252F16504498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2006.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16504498&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237135800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.61" data-doi="10.1007/s10681-017-1975-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casteel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rainey</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Using unsupervised learning techniques to assess interactions among complex traits in soybeans.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">200</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-017-1975-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D213%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-017-1975-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-017-1975-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.62" data-doi="10.1093/mp/ssq070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">W.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.-P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A major qtl, ghd8, plays pleiotropic roles in regulating grain productivity, plant height, and heading date in rice.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mp/ssq070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssq070%26rft_id%253Dinfo%253Apmid%252F21148627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssq070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21148627&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288802100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.63" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in oryza sativa.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1467</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1975.64" data-doi="10.1038/nature12878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">D14–scf d3-dependent degradation of d53 regulates strigolactone signalling.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">504</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">410</span> (erratum: Nature 532: 402).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12878</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D504%26rft.spage%253D406%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12878%26rft_id%253Dinfo%253Apmid%252F24336215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24336215&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1975.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328575300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/6/1975.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1957" title="Genome-Wide Analyses Reveal Footprints of Divergent Selection and Drought Adaptive Traits in Synthetic-Derived Wheats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1987" title="Little Evidence of Antagonistic Selection in the Evolutionary Strata of Fungal Mating-Type Chromosomes (Microbotryum lychnidis-dioicae)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/6"><span>Volume 9 Issue 6, June 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/6.cover-source.jpg?itok=Ub13TpI0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1975" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zShELFhJFUO2MjAphqj94lTXeJG8jnRFL8TtRb-SjNw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/6/1975" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/6/1975" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/6/1975" class="active" data-icon-position="" data-hide-link-title="0">Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493251" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HL3V42WINZ75c6twOYB4g2LCkXJ4SSDEEhvXsDYvyWI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1975.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/6/1975" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1TRlMFjXiuie9eyzpVMnm5XnUVclGsOA0vHRjVqP2W8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/6/1975" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1975.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493251" id="node493251--2" data-pisa="ggg;9/6/1975" data-pisa-master="ggg;g3.119.400154" data-apath="/ggg/9/6/1975.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8923-9733" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Haipeng Yu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Malachy T. Campbell</span>, <span class="highwire-citation-author" data-delta="2">Qi Zhang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harkamal Walia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1975-1986;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400154 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493251--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haipeng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haipeng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaipeng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8923-9733" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Haipeng Yu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malachy T. Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malachy%2BT.%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalachy%2BT.%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Malachy T. Campbell</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Harkamal Walia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harkamal%2BWalia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walia%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarkamal%2BWalia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harkamal Walia</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@vt.edu" class="" data-icon-position="" data-hide-link-title="0">morota@vt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/6/1975" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/6/1975&amp;title=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/6/1975&amp;title=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/6/1975&amp;title=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/6/1975&amp;text=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/6/1975&amp;title=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/6/1975&amp;t=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/6/1975&amp;title=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/6/1975&amp;title=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493251/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493251" id="node493251--3" data-pisa="ggg;9/6/1975" data-pisa-master="ggg;g3.119.400154" data-apath="/ggg/9/6/1975.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Bayesian Confirmatory Factor Analysis and Bayesian Network To Characterize a Wide Spectrum of Rice Phenotypes</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8923-9733" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Haipeng Yu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Malachy T. Campbell</span>, <span class="highwire-citation-author" data-delta="2">Qi Zhang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harkamal Walia</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gota Morota</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1975-1986;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400154 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493251--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haipeng Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haipeng%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaipeng%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8923-9733" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Haipeng Yu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Malachy T. Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malachy%2BT.%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalachy%2BT.%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8257-3595" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Malachy T. Campbell</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Statistics, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Harkamal Walia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Lincoln, NE 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harkamal%2BWalia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walia%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarkamal%2BWalia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9712-5824" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harkamal Walia</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gota Morota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gota%2BMorota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morota%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGota%2BMorota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3567-6911" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gota Morota</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:morota@vt.edu" class="" data-icon-position="" data-hide-link-title="0">morota@vt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1975&amp;rft.epage=1986&amp;rft.atitle=Genomic%20Bayesian%20Confirmatory%20Factor%20Analysis%20and%20Bayesian%20Network%20To%20Characterize%20a%20Wide%20Spectrum%20of%20Rice%20Phenotypes&amp;rft.volume=9&amp;rft.issue=6&amp;rft.date=2019-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Yu%2C+Haipeng&amp;rft.au=Campbell%2C+Malachy+T.&amp;rft.au=Zhang%2C+Qi&amp;rft.au=Walia%2C+Harkamal&amp;rft.au=Morota%2C+Gota"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493251/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493251/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493251/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493251/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493251/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493251/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493251/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493251/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493251/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493251/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493251/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493251/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/6/1975" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/6/1975.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/6/1975.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/6/1975.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
