<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/6/1795" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/6/1795.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/6/1795.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/6/1795.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/6/1795.atom" />
<meta name="HW.pisa" content="ggg;9/6/1795" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain" />
<meta name="DC.Identifier" content="10.1534/g3.119.400071" />
<meta name="DC.Date" content="2019-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Sarsani et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Isogenic laboratory mouse strains enhance reproducibility because individual animals are genetically identical. For the most widely used isogenic strain, C57BL/6, there exists a wealth of genetic, phenotypic, and genomic data, including a high-quality reference genome (GRCm38.p6). Now 20 years after the first release of the mouse reference genome, C57BL/6J mice are at least 26 inbreeding generations removed from GRCm38 and the strain is now maintained with periodic reintroduction of cryorecovered mice derived from a single breeder pair, aptly named Adam and Eve. To provide an update to the mouse reference genome that more accurately represents the genome of today’s C57BL/6J mice, we took advantage of long read, short read, and optical mapping technologies to generate a de novo assembly of the C57BL/6J Eve genome (B6Eve). Using these data, we have addressed recurring variants observed in previous mouse genomic studies. We have also identified structural variations, closed gaps in the mouse reference assembly, and revealed previously unannotated coding sequences. This B6Eve assembly explains discrepant observations that have been associated with GRCm38-based analyses, and will inform a reference genome that is more representative of the C57BL/6J mice that are in use today." />
<meta name="DC.Contributor" content="Vishal Kumar Sarsani" />
<meta name="DC.Contributor" content="Narayanan Raghupathy" />
<meta name="DC.Contributor" content="Ian T. Fiddes" />
<meta name="DC.Contributor" content="Joel Armstrong" />
<meta name="DC.Contributor" content="Francoise Thibaud-Nissen" />
<meta name="DC.Contributor" content="Oraya Zinder" />
<meta name="DC.Contributor" content="Mohan Bolisetty" />
<meta name="DC.Contributor" content="Kerstin Howe" />
<meta name="DC.Contributor" content="Doug Hinerfeld" />
<meta name="DC.Contributor" content="Xiaoan Ruan" />
<meta name="DC.Contributor" content="Lucy Rowe" />
<meta name="DC.Contributor" content="Mary Barter" />
<meta name="DC.Contributor" content="Guruprasad Ananda" />
<meta name="DC.Contributor" content="Benedict Paten" />
<meta name="DC.Contributor" content="George M. Weinstock" />
<meta name="DC.Contributor" content="Gary A. Churchill" />
<meta name="DC.Contributor" content="Michael V. Wiles" />
<meta name="DC.Contributor" content="Valerie A. Schneider" />
<meta name="DC.Contributor" content="Anuj Srivastava" />
<meta name="DC.Contributor" content="Laura G. Reinholdt" />
<meta name="article:published_time" content="2019-06-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Isogenic laboratory mouse strains enhance reproducibility because individual animals are genetically identical. For the most widely used isogenic strain, C57BL/6, there exists a wealth of genetic, phenotypic, and genomic data, including a high-quality reference genome (GRCm38.p6). Now 20 years after the first release of the mouse reference genome, C57BL/6J mice are at least 26 inbreeding generations removed from GRCm38 and the strain is now maintained with periodic reintroduction of cryorecovered mice derived from a single breeder pair, aptly named Adam and Eve. To provide an update to the mouse reference genome that more accurately represents the genome of today’s C57BL/6J mice, we took advantage of long read, short read, and optical mapping technologies to generate a &lt;i&gt;de novo&lt;/i&gt; assembly of the C57BL/6J Eve genome (B6Eve). Using these data, we have addressed recurring variants observed in previous mouse genomic studies. We have also identified structural variations, closed gaps in the mouse reference assembly, and revealed previously unannotated coding sequences. This B6Eve assembly explains discrepant observations that have been associated with GRCm38-based analyses, and will inform a reference genome that is more representative of the C57BL/6J mice that are in use today.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/06/01" />
<meta name="citation_mjid" content="ggg;9/6/1795" />
<meta name="citation_id" content="9/6/1795" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/6/1795" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/6/1795.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/6/1795.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/6/1795.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400071" />
<meta name="citation_pmid" content="30996023" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="1795" />
<meta name="citation_lastpage" content="1805" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vishal Kumar Sarsani" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Narayanan Raghupathy" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Ian T. Fiddes" />
<meta name="citation_author_institution" content="UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California" />
<meta name="citation_author" content="Joel Armstrong" />
<meta name="citation_author_institution" content="UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California" />
<meta name="citation_author" content="Francoise Thibaud-Nissen" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894" />
<meta name="citation_author" content="Oraya Zinder" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Mohan Bolisetty" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Kerstin Howe" />
<meta name="citation_author_institution" content="Wellcome Sanger Institute, Hinxton, Cambridge CB10 1SA, UK" />
<meta name="citation_author" content="Doug Hinerfeld" />
<meta name="citation_author_institution" content="NanoString Technologies, Inc, and" />
<meta name="citation_author" content="Xiaoan Ruan" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Genomic Medicine, Farmington, CT" />
<meta name="citation_author" content="Lucy Rowe" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Mary Barter" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Guruprasad Ananda" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Genomic Medicine, Farmington, CT" />
<meta name="citation_author" content="Benedict Paten" />
<meta name="citation_author_institution" content="UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California" />
<meta name="citation_author" content="George M. Weinstock" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Genomic Medicine, Farmington, CT" />
<meta name="citation_author" content="Gary A. Churchill" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Michael V. Wiles" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author" content="Valerie A. Schneider" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894" />
<meta name="citation_author" content="Anuj Srivastava" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Genomic Medicine, Farmington, CT" />
<meta name="citation_author_email" content="laura.reinholdt@jax.org" />
<meta name="citation_author_email" content="anuj.srivastava@jax.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8455-3993" />
<meta name="citation_author" content="Laura G. Reinholdt" />
<meta name="citation_author_institution" content="The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME" />
<meta name="citation_author_email" content="laura.reinholdt@jax.org" />
<meta name="citation_author_email" content="anuj.srivastava@jax.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4054-4048" />
<meta name="citation_reference" content="citation_journal_title=Environmental and molecular mutagenesis;citation_journal_abbrev=Environ Mol Mutagen;citation_author=M. A. Beal;citation_author=T. C. Glenn;citation_author=S. L. Lance;citation_author=C. M. Somers;citation_title=Characterization of unstable microsatellites in mice: No evidence for germline mutation induction following gamma-radiation exposure.;citation_pages=599-607;citation_volume=53;citation_year=2012;citation_issue=8;citation_pmid=22930577;citation_doi=10.1002/em.21726" />
<meta name="citation_reference" content="citation_journal_title=Mutation Research;citation_journal_abbrev=Mutation Research;citation_author=M. A. Beal;citation_author=T. C. Glenn;citation_author=C. M. Somers;citation_title=Whole genome sequencing for quantifying germline mutation frequency in humans and model species: cautious optimism.;citation_pages=96-106;citation_volume=750;citation_year=2012;citation_issue=2;citation_pmid=22178956;citation_doi=10.1016/j.mrrev.2011.11.002" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. A. Beck;citation_author=S. Lloyd;citation_author=M. Hafezparast;citation_author=M. Lennon-Pierce;citation_author=J. T. Eppig;citation_title=Genealogies of mouse inbred strains.;citation_pages=23-25;citation_volume=24;citation_year=2000;citation_issue=1;citation_pmid=10615122;citation_doi=10.1038/71641" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. Berlin;citation_author=S. Koren;citation_author=C. S. Chin;citation_author=J. P. Drake;citation_author=J. M. Landolin;citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.;citation_pages=623-630;citation_volume=33;citation_year=2015;citation_pmid=26006009;citation_doi=10.1038/nbt.3238" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=K. Buac;citation_author=D. E. Watkins-Chow;citation_author=S. K. Loftus;citation_author=D. M. Larson;citation_author=A. Incao;citation_title=A Sox10 expression screen identifies an amino acid essential for Erbb3 function.;citation_pages=e1000177-e1000177;citation_volume=4;citation_year=2008;citation_issue=9;citation_pmid=18773073;citation_doi=10.1371/journal.pgen.1000177" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. S. Chin;citation_author=D. H. Alexander;citation_author=P. Marks;citation_author=A. A. Klammer;citation_author=J. Drake;citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.;citation_pages=563-569;citation_volume=10;citation_year=2013;citation_pmid=23644548;citation_doi=10.1038/nmeth.2474" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. M. Church;citation_author=L. Goodstadt;citation_author=L. W. Hillier;citation_author=M. C. Zody;citation_author=S. Goldstein;citation_title=Lineage-specific biology revealed by a finished genome assembly of the mouse.;citation_pages=e1000112-e1000112;citation_volume=7;citation_year=2009;citation_issue=5;citation_pmid=19468303;citation_doi=10.1371/journal.pbio.1000112" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. M. Church;citation_author=V. A. Schneider;citation_author=T. Graves;citation_author=K. Auger;citation_author=F. Cunningham;citation_title=Modernizing reference genome assemblies.;citation_pages=e1001091-e1001091;citation_volume=9;citation_year=2011;citation_issue=7;citation_pmid=21750661;citation_doi=10.1371/journal.pbio.1001091" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Cutler;citation_author=L. A. Marshall;citation_author=N. Chin;citation_author=H. Baribault;citation_author=P. D. Kassner;citation_title=Significant gene content variation characterizes the genomes of inbred mouse strains.;citation_volume=17;citation_year=2007;citation_pmid=17989247;citation_doi=10.1101/gr.6754607" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. K. Das;citation_author=M. D. Austin;citation_author=M. C. Akana;citation_author=P. Deshpande;citation_author=H. Cao;citation_title=Single molecule linear analysis of DNA in nano-channel labeled with sequence specific fluorescent probes.;citation_volume=38;citation_year=2010;citation_pmid=20699272;citation_doi=10.1093/nar/gkq673" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. E. Dickinson;citation_author=A. M. Flenniken;citation_author=X. Ji;citation_author=L. Teboul;citation_author=M. D. Wong;citation_title=High-throughput discovery of novel developmental phenotypes.;citation_pages=508-514;citation_volume=537;citation_year=2016;citation_pmid=27626380;citation_doi=10.1038/nature19356" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Fairfield;citation_author=A. Srivastava;citation_author=G. Ananda;citation_author=R. Liu;citation_author=M. Kircher;citation_title=Exome sequencing reveals pathogenic mutations in 91 strains of mice with Mendelian disorders.;citation_volume=25;citation_year=2015;citation_pmid=25917818;citation_doi=10.1101/gr.186882.114" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. T. Fiddes;citation_author=J. Armstrong;citation_author=M. Diekhans;citation_author=S. Nachtweide;citation_author=Z. N. Kronenberg;citation_title=Comparative Annotation Toolkit (CAT)-simultaneous clade and personal genome annotation.;citation_volume=28;citation_year=2018;citation_pmid=29884752;citation_doi=10.1101/gr.233460.117" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Florea;citation_author=A. Souvorov;citation_author=T. S. Kalbfleisch;citation_author=S. L. Salzberg;citation_title=Genome assembly has a major impact on gene content: a comparison of annotation in two Bos taurus assemblies.;citation_pages=e21400-e21400;citation_volume=6;citation_year=2011;citation_issue=6;citation_pmid=21731731;citation_doi=10.1371/journal.pone.0021400" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=E. Garrison;citation_author=J. Siren;citation_author=A. M. Novak;citation_author=G. Hickey;citation_author=J. M. Eizenga;citation_title=Variation graph toolkit improves read mapping by representing genetic variation in the reference.;citation_pages=875-879;citation_volume=36;citation_year=2018;citation_pmid=30125266;citation_doi=10.1038/nbt.4227" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. A. Graubert;citation_author=P. Cahan;citation_author=D. Edwin;citation_author=R. R. Selzer;citation_author=T. A. Richmond;citation_title=A high-resolution map of segmental DNA copy number variation in the mouse genome.;citation_pages=e3-e3;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17206864;citation_doi=10.1371/journal.pgen.0030003" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=QUAST: quality assessment tool for genome assemblies.;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=E. Hodges;citation_author=M. Rooks;citation_author=Z. Xuan;citation_author=A. Bhattacharjee;citation_author=D. Benjamin Gordon;citation_title=Hybrid selection of discrete genomic intervals on custom-designed microarrays for massively parallel sequencing.;citation_pages=960-974;citation_volume=4;citation_year=2009;citation_issue=6;citation_pmid=19478811;citation_doi=10.1038/nprot.2009.68" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=X. S. Hu;citation_author=F. C. Yeh;citation_author=Y. Hu;citation_author=L. T. Deng;citation_author=R. A. Ennos;citation_title=High mutation rates explain low population genetic divergence at copy-number-variable loci in Homo sapiens.;citation_pages=43178;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep43178" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Konig;citation_author=L. W. Romoth;citation_author=L. Gerischer;citation_author=M. Stanke;citation_title=Simultaneous gene finding in multiple genomes.;citation_volume=32;citation_year=2016;citation_pmid=27466621;citation_doi=10.1093/bioinformatics/btw494" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet;citation_author=J. Lilue;citation_author=A. G. Doran;citation_author=I. T. Fiddes;citation_author=M. Abrudan;citation_author=J. Armstrong;citation_title=Sixteen diverse laboratory mouse reference genomes define strain-specific haplotypes and novel functional loci.;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-018-0223-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. E. Locke;citation_author=M. Milojevic;citation_author=S. T. Eitutis;citation_author=N. Patel;citation_author=A. E. Wishart;citation_title=Genomic copy number variation in Mus musculus.;citation_pages=497;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1713-z" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Marshall;citation_title=Genome sequencing. Public group completes draft of the mouse.;citation_pages=1005-1005;citation_volume=296;citation_year=2002;citation_issue=5570;citation_pmid=12004098;citation_doi=10.1126/science.296.5570.1005b" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=I. Matera;citation_author=D. E. Watkins-Chow;citation_author=S. K. Loftus;citation_author=L. Hou;citation_author=A. Incao;citation_title=A sensitized mutagenesis screen identifies Gli3 as a modifier of Sox10 neurocristopathy.;citation_volume=17;citation_year=2008;citation_pmid=18397875;citation_doi=10.1093/hmg/ddn110" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=B. Milholland;citation_author=X. Dong;citation_author=L. Zhang;citation_author=X. Hao;citation_author=Y. Suh;citation_title=Differences between germline and somatic mutation rates in humans and mice.;citation_pages=15183;citation_volume=8;citation_year=2017;citation_pmid=28485371;citation_doi=10.1038/ncomms15183" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. H. Waterston;citation_author=K. Lindblad-Toh;citation_author=E. Birney;citation_author=J. Rogers;citation_title=Initial sequencing and comparative analysis of the mouse genome.;citation_pages=520-562;citation_volume=420;citation_year=2002;citation_issue=6915;citation_pmid=12466850;citation_doi=10.1038/nature01262" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. W. Myers;citation_author=G. G. Sutton;citation_author=A. L. Delcher;citation_author=I. M. Dew;citation_author=D. P. Fasulo;citation_title=A Whole-Genome Assembly of Drosophila;citation_pages=2196-2204;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731133;citation_doi=10.1126/science.287.5461.2196" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. R. Quinlan;citation_author=R. A. Clark;citation_author=S. Sokolova;citation_author=M. L. Leibowitz;citation_author=Y. Zhang;citation_title=Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.;citation_volume=20;citation_year=2010;citation_pmid=20308636;citation_doi=10.1101/gr.102970.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Rosen;citation_author=J. Eizenga;citation_author=B. Paten;citation_title=Modelling haplotypes with respect to reference cohort variation graphs.;citation_pages=i118-i123;citation_volume=33;citation_year=2017;citation_doi=10.1093/bioinformatics/btx236" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=J. L. Salisbury;citation_title=Centrosomes: Sfi1p and centrin unravel a structural riddle.;citation_pages=R27-R29;citation_volume=14;citation_year=2004;citation_issue=1;citation_pmid=14711432;citation_doi=10.1016/j.cub.2003.12.019" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. J. Schimenti;citation_author=S. K. Feuer;citation_author=L. B. Griffin;citation_author=N. R. Graham;citation_author=C. A. Bovet;citation_title=AKAP9 is essential for spermatogenesis and sertoli cell maturation in mice.;citation_volume=194;citation_year=2013;citation_pmid=23608191;citation_doi=10.1534/genetics.113.150789" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. A. Schneider;citation_author=T. Graves-Lindsay;citation_author=K. Howe;citation_author=N. Bouk;citation_author=H. C. Chen;citation_title=Evaluation of GRCh38 and de novo haploid genome assemblies demonstrates the enduring quality of the reference assembly.;citation_volume=27;citation_year=2017;citation_pmid=28396521;citation_doi=10.1101/gr.213611.116" />
<meta name="citation_reference" content="citation_title=Mouse Genetics;citation_author=L. M. Silver;citation_year=1995" />
<meta name="citation_reference" content="Smit, A., R. Hubley and P. Green, 2013–2015 RepeatMasker, pp." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Srivastava;citation_author=A. P. Morgan;citation_author=M. L. Najarian;citation_author=V. K. Sarsani;citation_author=J. S. Sigmon;citation_title=Genomes of the Mouse Collaborative Cross.;citation_volume=206;citation_year=2017;citation_pmid=28592495;citation_doi=10.1534/genetics.116.198838" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=B. A. Taylor;citation_author=L. Rowe;citation_title=Genes for serum amyloid A proteins map to Chromosome 7 in the mouse.;citation_pages=491-499;citation_volume=195;citation_year=1984;citation_issue=3;citation_pmid=6088946;citation_doi=10.1007/BF00341452" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Pathol.;citation_author=D. Venkatesh;citation_author=D. Mruk;citation_author=J. M. Herter;citation_author=X. Cullere;citation_author=K. Chojnacka;citation_title=AKAP9, a Regulator of Microtubule Dynamics, Contributes to Blood-Testis Barrier Function.;citation_pages=270-284;citation_volume=186;citation_year=2016;citation_doi=10.1016/j.ajpath.2015.10.007" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. E. Watkins-Chow;citation_author=W. J. Pavan;citation_title=Genomic copy number and expression variation within the C57BL/6J inbred mouse strain.;citation_volume=18;citation_year=2008;citation_pmid=18032724;citation_doi=10.1101/gr.6927808" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=M. V. Wiles;citation_author=R. A. Taft;citation_title=The sophisticated mouse: protecting a precious reagent.;citation_pages=23-36;citation_volume=602;citation_year=2010;citation_pmid=20012390;citation_doi=10.1007/978-1-60761-058-8_2" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Zuo;citation_author=X. Du;citation_author=J. Zhao;citation_author=H. Yang;citation_author=C. Wang;citation_title=Analysis of microsatellite polymorphism in inbred knockout mice.;citation_pages=e34555-e34555;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22509320;citation_doi=10.1371/journal.pone.0034555" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
<meta name="twitter:description" content="Isogenic laboratory mouse strains enhance reproducibility because individual animals are genetically identical. For the most widely used isogenic strain, C57BL/6, there exists a wealth of genetic, phenotypic, and genomic data, including a high-quality reference genome (GRCm38.p6). Now 20 years after the first release of the mouse reference genome, C57BL/6J mice are at least 26 inbreeding generations removed from GRCm38 and the strain is now maintained with periodic reintroduction of cryorecovered mice derived from a single breeder pair, aptly named Adam and Eve. To provide an update to the mouse reference genome that more accurately represents the genome of today’s C57BL/6J mice, we took advantage of long read, short read, and optical mapping technologies to generate a de novo assembly of the C57BL/6J Eve genome (B6Eve). Using these data, we have addressed recurring variants observed in previous mouse genomic studies. We have also identified structural variations, closed gaps in the mouse reference assembly, and revealed previously unannotated coding sequences. This B6Eve assembly explains discrepant observations that have been associated with GRCm38-based analyses, and will inform a reference genome that is more representative of the C57BL/6J mice that are in use today." />
<meta name="og-title" property="og:title" content="The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/6/1795" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Isogenic laboratory mouse strains enhance reproducibility because individual animals are genetically identical. For the most widely used isogenic strain, C57BL/6, there exists a wealth of genetic, phenotypic, and genomic data, including a high-quality reference genome (GRCm38.p6). Now 20 years after the first release of the mouse reference genome, C57BL/6J mice are at least 26 inbreeding generations removed from GRCm38 and the strain is now maintained with periodic reintroduction of cryorecovered mice derived from a single breeder pair, aptly named Adam and Eve. To provide an update to the mouse reference genome that more accurately represents the genome of today’s C57BL/6J mice, we took advantage of long read, short read, and optical mapping technologies to generate a de novo assembly of the C57BL/6J Eve genome (B6Eve). Using these data, we have addressed recurring variants observed in previous mouse genomic studies. We have also identified structural variations, closed gaps in the mouse reference assembly, and revealed previously unannotated coding sequences. This B6Eve assembly explains discrepant observations that have been associated with GRCm38-based analyses, and will inform a reference genome that is more representative of the C57BL/6J mice that are in use today." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/6.cover-source.jpg" />
    <title>The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"i9mh2FtlrVQ8kO3eOoEVYWjtqR1gqQQ5VwbFLt2Cm_E","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493239","apath":"\/ggg\/9\/6\/1795.atom","pisa":"ggg;9\/6\/1795","ac":{"\/ggg\/9\/6\/1795.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/6\/1795.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1795"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/6\/1795"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/6\/1795":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228RcSbFF5zH9fjcW2mzBfs7dwZPuEOlYGHXbN4ryM00tyw\\\\\\\/b\\\\\\\/OSNODopcZ7cPO8kLcfmBhBzxU1\\\\\\\/2PNcrhX\\\\\\\/mQixjlJmWZ3pvTGFVHiUs1R5PUGZ8jT9ZP2c+M9vBbrVwdhzkKhC0ft79pTJKo\\\\\\\/Jqz9F\\\\\\\/zL9hY8m9KAsBeXPmV9EWsxc4AUzk4reqCRogvzHyJtSK2W1UluyHVO67iuRRVtSsCvnj\\\\\\\/ibNHUZFVtUVVsnfPt7nX0J99KZ5V3YYytHckoluqpVNmxgOtuaHHpcTelTK9ts6q2EdyE1x9p7nlAwXL9lLWPv2jsWNG1wCahG5aqa5nxDxclQCUpAssbRQnZ2sUMFill\\\\\\\/KuSLDgio9U8IWFRp3qV81KC84kTgktOzbAl8Ij2kH2PBDx8UerJi5zIBlNFr\\\\\\\/pl5yzEOjqSshvk76XsrncPOPxUh478lapvkg\\\\\\\/O50W3PsAYlrFEIzKkBQJVbeRTV1UxoaUPUuzC1moO\\\\\\\/3CxyIjlQFNx0KsHZsjxEurtNW0wCnTNJRuCH8By53c4oTHtPIlW42U1Izv6c7\\\\\\\/AfeACJhsYoOxC0B0Jj7NXWnTobvkQSTt7BXcwYGnI7zKAvZNL++760SMcDcxnBhtm1vHvQA2TE\\\\\\\/oay0XKLA7GtMmFXfDNEmOIJAE6fMb+dfCbLqv7GSI5al5TswfKHyOLret0V89GMcIqdYPPuzqWuZG8Ggj7s94UPWO9eopoMwcTI8UDxqXlxOw4j05siCl0dQww0jdfmbxX0XAYU2djShtZQRS7olijBSu0iqXvBS\\\\\\\/BB5Ot3lo\\\\\\\/zknXttG6KmUuMz+K9AFIcEgFYy7ZEZMCJS5Z88Tn3Pg8dBVV+rvj4auH9NbH4SsrNGE9mUjPHTytto2vxod8kPzyeyyrBn24qyhGFCjNLVaUeZxMj\\\\\\\/3nz6QCKnoMUfXriMgJxaaBmAcATvFiXfvuFEQyhcBo\\\\\\\/N575aZQxVtmFt6Qjs7YcqiHInxtNJ+2XU9TAijVKI4BpabNySQsYTQpoBMdfuqgnCQ7QrUIshbP7Fd4ZpOIcpHL8NzpAAzWaXODw+bhrfI2CiHKGc31xVy9sy8X2zGKAfuUexLaBqhJe0S7ggQhrfo8tNHel6C4s13yZ8rEstg9a+A21SjvKAOwTzjNhfhcPXA3\\\\\\\/FbWeyjRWCJyydHEgB7P8tPjAyfy0MdgzU5hK7Wl+0tzTGnHrC9HNWFotj4bROTMtUP5tfdPcJwfXzsqBUfWOHZ0YrjSrUwKgXaKxMNbxUy\\\\\\\/XRiZuGwkX9bDScMe+ev0Gf\\\u0022,\\\u0022iv\\\u0022:\\\u0022dVbWOG03YYis8Qs+V4SUmg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c4b62b558cb771b316d30aa77eba633a5bee1199df8bbab43e1843357fd53ae9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l0JfaRyV2xUQuMNsx6A8pU6MX+LEuaOx9qGhm6pjqh6iOKyIv3Rt+FKGHzfVizF4Cqu1jni4PdwAeXm2uebWvmWhEFkIZFco7f+YcrLt2rjDjsPoEvcz2FMruz1nbejWAkGzB3x9Q+FVJKGjI+VLhtew\\\\\\\/Tl4y5RkdHbAjIzehOvnDw\\\\\\\/F9lhQESpLx1Iu9ez\\\\\\\/GUxihkAPk4yd+eWk7ZG7dLVFe6gLNwSDv7H6zN6eIYsxnW3dboEO5Iyq6aTyHsO7Mkkf0yOxmIGJZzeqzPgzKILXBTXuKBWVQvJkBM0tSjOaTcRpnFDegTqu6HbXS\\\\\\\/tq+66R0LpJtTlZWNtAVbMnx67QlaFZWdZOirXDGvmRVwi1K2TyL9m0gSnzTxboO264Byrdol8UKclgFn7eXmNCedbYP8250tN4y1h2VUaXsNmhANnmwAvjkRov9ZLRp3cLVTIiBOG80C46hg2SQ+sX7cTfCiB3xpS8faf\\\\\\\/e2eqqcik\\\\\\\/K6yIOyYnEe69mp0LCknHsTOHI1dqL6UAIb2s0f3CBQyR9ezToNhB1GO5ax0VGwHgEEHE2Ji10QC2VJxazZqXFvpo0sj6yGe7M5MA7hHqzXYL9UdgPNxnXpfQqQOCONkhcciVaa1x8zkKJ7FLAOA+C4iMFta5NgOwpWr0woD7AL9KchJY8Jf5byLRL4L\\\\\\\/a3lNpEk9wMtivg8OL+vAh\\\\\\\/noT8LTKOKn9i+iYDk+56OfKwQVLQrEbnHu2BkMafDds8DNw7Bz3LzTR9YnaV0QyGtJBReZu4iWSxEw6145Okv2Pe+nSWWKi8U5ieNvSHCL24QRv3UaWcfuny3OKJDw0oe661dhEdKwpZHq6IDQMJW9N0KsWj2bv7IMkXvmwrhnFcdSdALCBvkBxPcirHMi2ExvqLBTvBQWBWpq2pYEzFSy8hFblRSlU2uv+jshZvnjR+maqs+sDS6niOBYoBsxtm2V+deCNQ7maZB68epamAymEjus7tg\\\\\\\/RtLCuNn8Yy9iJPdq9ff7EPdU2Nzu\\\\\\\/aCL9p4DxNrP49LuuzbgV74gOZYlibnSThEZenic2btnEG+uGw5wNczC2B4ONAimLLbcZfNFj5u8+om1ubXd7qte0tuHWiZn3ZWg3LkvaoWheem7G7\\\\\\\/P4I6XjR2lIRTMQFEbPsubS6mqqNRjG6zORo+9Bz0pNQcm+JEohyT2q2oFW3P1EYvw\\\\\\\/O+smkE9LCf1TCSdArTW2lFCN3+M73ER3PhBSeQuoEp3K62g\\\\\\\/nGcu9iX\\\\\\\/QiwxwcF5cAGrhevcahr\\\\\\\/+\\\\\\\/N8L2cflULy4bExNCGESK4dlK1JMwFLn6vy9vyZxHzkOG8UdntZQbvbqaA2NWM9gSdWAuKwhhOuy92HT8hb0wizhN1DbOB0m1wX\\\\\\\/SvTsmHTWiIUrU5OLPz51KkiqH0ZV9is3lzixDqo4kP05q89wNw9A5lzXo2OJTaQzczENp14+6GrdMgzQyaYXv5JgBNaGgEdITFuKiumrfBnJP\\\\\\\/1eSnmynz4QnMj\\\\\\\/KCDceFRBO8A958oT71lB5w3W2qGaqPT306lN4UyQ+Ej25JOue9vauSDvumIxsecMYQjQdtTv\\\\\\\/yTCkr9mwjObzvNhhz7gQ5EvLoSt5d9CTLmVUr2orghjk37EbQa6x77mEMnlWAkIGNeQ7IcpLZiPv3zyau89HF2ncr4t3Y\\\\\\\/FP8mfdUqTgVZoqM3BD7oizGFDKXBkEiJsP41qz6lb+wyEoszQpjxh04ZGbCJeyKJIgf6MNyxN8YIF+X19mkakPmWhPY5IOUNcy\\\\\\\/ByZOcdZZl\\\\\\\/HtIwSqxryFPq6Utho0lNUzO+wuB+YjZ3migJ+8R0Y8D7prkQYJETPVhObJLx+E\\\\\\\/RHISj2dBosf+L65CfLegcIRa4ycYoYDZs0f1bCc9qwqVyRfVAR+2sIsiK3Baiz5Bw6zs1GsmNVJ4uuWs0TUqXIB39qQN6ooBmbrhfxpzpeu8IQ2fFR5FHSrM3yGZD\\\\\\\/ruCT+Qt\\\\\\\/EBDW2w2Sg9ZSPhn0ecYLTaWfQcfCGazyiozlsDLE69w+hKSFW8RThlN8cbUPtZ0\\\\\\\/\\\\\\\/RTHOW\\\\\\\/twIKT4MuO9MIZ8fhj+Ha5pU9hPysBuHmpR\\\\\\\/d0ZqvKfqf38UH2LQbISIourgW6L96vFGUbdEfSSRz\\\\\\\/CRJSIG\\\\\\\/\\\\\\\/vqAX3ANi7KZO7WiH3qb0hUdNWj+o5YXSbQxO93nvBJOfRCi7zhKccmkVrWdv\\\\\\\/+Fznk2Ts1FjkizfFIxhbcKw7eT++QDbvAinIEEMsn+CC5XaFgZPuieTwacaHygsLPM6Ju1RPWFiJzAfbonPhesJdYh6ihe2I0PuLHmb0p7rSSzUIq8yWZBqZx6HOY6HHMoVdtpplrMOvYDwyT418GiriyH605NJmJaaKRx+lTZGPHmMN1NQMel6pyc5FpqVwqluk\\\\\\\/0BKlRgE2TSQBj9tLbkuhCQYab5mdO2Vhm4Nw+8rgujPKYfomP1uUCix6ULVF5xsVb25wC6Pnm2i7Q9Pz43krE50a25EO4hKI2otX9qpBi34W03+OYQS8F0wjRaZUlt+dl7J2NLyiEtEhLxqsf6hYYVTIlm1rynRawt9ClttF81pqaOsC3DDRElr3fqykmC3KiTz1e8YGt8b2JyElOZ4SjCkgSdQ9Ni9IDcij02w68XwT6Ya+K+B6w8ONJ517eWvs1q+B9DFA1E81VEy5L65CZWJyXGhnUpdDZdJlItlHzZDg4V6+434xZE8enr017OSb3VEt64Tj2VOAVJhXmBMI4t7s8sjDl7h1DTypCBXSeSh+x\\\\\\\/dx4VCz\\\\\\\/2X0faAIsq56CKFrjJ2ggWV+tOHIV2SAomsHPW3lKAJg6BC4GzXdwyvGnfUlkS87OdqN\\\\\\\/GYZFtcn0uzrdy18QH3Mz5Gv9qKUhGsqEXSNxGYgujs+y4cnyfT3D\\\\\\\/p5+Z+wwR54uZ\\\\\\\/3+3WpS6zSE\\\\\\\/5s2ctb45awBYa6TY24PjZ\\\\\\\/gTsdBvd\\\\\\\/N81rL0pnmGM4DWe\\\\\\\/FkOGgOEZtJMassFuR\\\\\\\/tMer\\\\\\\/3WhyVV07WiasLn9WD63H5c\\\\\\\/UuCmpcyTZ44rdgeUSOXwWLXNdBL9OyVo5k68pULmOw9mAlCTyCb\\\\\\\/C1Mhoz7XEuAGyyQQy14lW82Yhv1SoFOqW+fSqzY59P1HEAqN3TlMhzODADQlis+Aubn46q\\\\\\\/GZ0\\\\\\\/J+jahW7gzM\\\\\\\/Tw7bopxtMICxmS7+XcjQpMt+Em4O1QiOpkvynA87xFZUWjjHEvZe4bVwRhdxZe7ICfT1zyRPvylZb\\\\\\\/cXEo\\\\\\\/JXzyAoxvOQ7JTvZRDi8c9cO4w2Ke5K5j6K1H3gIFzsPNofmip77Fsajl2+WvhPDs3dx0R\\\\\\\/nu2plv2mbnZq0EoHGqixI9XaRZUPAtq0gpfBCOGlhPWPUHNbvRCNsthQ4sfqMzNbcNWtwfe1iT8mXwCfHazg0Krkd4bQH3EhxO+ieZjNAyHGInlQaF8hanNZst+R6QzvhPbBoYuHaQvuB0O0ccr9QJ0CIA9EUGcIR665RA4n8\\\\\\\/5ne3rQ8i4pScBtg9hx5kEH4MzlUGw5cfG1N4znhJrHp63+nqUuy5NFCDdvWmrHf6jjmtvJp9ll35I2b5mjms2R3F44BrOXBl8zM9QxsaspLMpASsP5yuBkLltqGF+NkpptkIPva9n4PnowCAkgXLnFF+ixU9mUtzBFQAJihflv3uy8BkXcrKagcVlM\\\\\\\/lzKXXZhmwTb5kzkq6qP8rCsvNT4bvYDo3wdugSYbXq6yrsv2Lh0fTOpX7WAm0piAdf4yjZ4sr1lDG94JkfQ+jbI5NcRMX3M9GB3vW5WNKyGw6V6Awtnivtjk5bo6ukofjiPeajiUXUCoO\\\\\\\/Yk+wj0ualiYFwb+XLLCxQDIv7WUBOTJw9x0QarZmX22N+ZYpsWvEI7Tf01O7ry9prgU2NFIHTkKsDTCySPxFiLwYBl2gXUibQpAAPnsam1tFkzrpi3AePmxnj5wyidIAVwCvtQdJS9qELmsN77YtIn2vyaJU280PUxgsbUkYTxfDxhhvmuWeYEeHwQxXniOpPZMSSND6ohiFb27+PG+GaGHPD76QUUZr2ZMzs6cOglfiIUXVk3MmorhybAQVsxJUw6P76cmoKdNj5FoRzWgMgUAW\\\\\\\/G6kf\\\\\\\/6\\\\\\\/B2uOvQtfPBWowdbSJK97pXRoG6asKMQkIl61WQB\\\\\\\/vhLjR2GjI+1YFiPzUMAR0g102OmcfLwqMM4+cc7DgO3KPgmV1KUFK5QE\\\\\\\/atQvz2yLbBpJj+LyFfq8M4yVi5DMkT3gD1lqXN6yYBx0kgwxO\\\\\\\/DFUW2Mscmc9BkX03U0MdxygrueO6FlPhYggUGJwI+wOCfOLqi7NdcfwdAT6kOlsUMQhV9ExIWZoYPz4qUG00VdSlGSMJpGkwX55UYf3H5L1x9d6d7x\\\\\\\/jxkCfay70s0YcHrssk0TSCOgJBxS81U1X5jJQPzOf7X6Gyoeu\\\\\\\/CaAvPiTWPlosQVksm66lrtanknDtECC1EeagNhJtqFsTI7gXJR+W7pQriwKzBx5lfnPn4fhu2lFQFuF3i3SJ+q+11BNc4h1PuxiIsk5BwmhEMSZGUtyFAM013HGKwAmPVTxLs4jcYgG4DvkXjwa\\\\\\\/8bH3A5Pr665QOI7GaXhv6E5lGdWgwa8o0uD6IOEi1cS\\\\\\\/WKCvNpziD5gxV7CA0yW9Kzm2sb0iVUXYr+E1AdCH2oFpRQ84ySai2kfNu1o872AtxOUhM8CsFaPJjIlepR9gSPJvVw8eDe7SWkhkgPmRGeV8B8bYVkyjGW6Camm3WJO4OaQYPSwQdaL9C4GBD+GQ6Fj4vsAqGk7XuTWSPrHGoxPM4rrmuZBviM3aE33kdDyanQ6\\\\\\\/P6bMy5vvBUjj4HEYAcLWKg6AHj3r0klGAImuHW5wgSc4s26hOny8ReVAQGvmkLmNMLm+0sbp7toSeS41klMuvgTf4RQzIrsaFl3mddnquDfCtIhoyGBU0oHTznhRu30xCJOyN9u\\\\\\\/XC5PjY2lHXMPEfzAdDeeegpEbP8Es1Xbc1MlfbsRz0AZ2Vmtht7JCdkTru5dXh4kErwBGq2o67GhZyyA\\\\\\\/jd2v5xKcToem6EgXR6xB2ztkVJtCXIIun+MDDaLZZGw6IN9rslrw4vgmMIvX8Fu94bk4i6Z15tRbp7sALeFcgJmXoRRvtj76HoaEzQXeXIxuEY6iMWCRE4APw+9XsbICxIyFqKi0snA9Mf9behOfvWaK6VEC7lGfUDFG4IZvPpXKq9J7WQb9HBrqQnqDd5qh+mEqzTIaCUoDFVYreOFag0RoJSMd8QHbPmrh1f6yOisGFTRexQTVuNZ3KFu5TXJc9Lot5nRYEPPleekHMSeUQUnq58MeWy8M8p1Zl7KxiehsGcyIMOe2n053p9AYSj9leWNAioydcI1n7aUDMvLSCRexYSahikRkYCMuRlybyZyfK\\\\\\\/Fbq5H5n5vIkTBR\\\\\\\/nsVyI9EzAFDejHIS8+9BmZHWNn0tLRdO8CL6Z0VR8Zp64PuhBZcAYtzGpyWhXQiIJUX\\\\\\\/8J+7IlIO+8uhbhkMBIVPiqR0GcoKlm8FK\\\\\\\/RUfpx\\\\\\\/8B5tA2ljIIgliSf2hwwIRRdHWYY3rhQ\\\\\\\/jmNBqwNTVMHoTQPjpNog6Wa6CzT5Ygvh44l66JeKX4PWHLkY+N55u2IEwrBC3J+6MZrJ\\\\\\\/2VkqTC3C2zvG9QyUNGMZaAdgB0FMc+zSOSVLLceWlFIzieoiS5uMNtz4v3T944dLlOgzjvJnQXc9XuDIxZ3ErLHt2AwF0sNkeEf1H7UeIqg7zVjAE2fVdIFd6RTZv65q3+ZLmtctNdkU1xdx2gUakmiF\\\\\\\/wdSsoypb44qEcOF+\\\\\\\/TIUtg5qRqQi\\\\\\\/auRLHM7NWc3p4Ny7lbs9lOcvEzLmbSPXnnGhyl5rR7xyH6MRaFrbVZF7FWr8ky7wPIYsKJc7+tGYFG\\\\\\\/H6R0vOTHaahBgOZovQ6mngL+2Qb9+P0LByi0IJTFoDIwfJEfx5D\\\\\\\/j8DPAF3TPoAPdUrbC5WbRhWB9T\\\\\\\/ZjI5RQA79L1TZOs6UJ2QpKCg6hQVyp3gHmWxxrpHjaS9g\\\\\\\/3vCj7McZA\\\\\\\/Z8CDv4t7em5hf6+lC+oXBf+Hwn+RxBs\\\\\\\/OoF66N\\\\\\\/uRvuCstgFOlga1hd3XAYC11pmxB0YUjflOc96arFA\\\\\\\/Lj+dqJWuNce07PB4zHxufimULMNTETf0CNCQrG\\\\\\\/F1RX4Pf0+s2TqSSSPkoeFo7ELcw+2ljPVNVhSyK+NkmXRqsz01aHBwvuVTbWC\\\\\\\/UcSd2rdVT1mO+tottxYsZy67amUHQ2M15Q4Nml7ZHMT20JB8qG88ozaT9RQ6MQqxc2+JkRI9VIoo5j\\\\\\\/20f2MLEoxFIHjh8Zu9kYtniewjsZtpJyZyJRcvQKwIDzPPzjnkzlK6zrMjttGDewpWGdC9+iF4fw5gxZ\\\\\\\/3sJamZB0W0QdT3v+l5uMd5zWdpYtbzgUKR2hN6eEM\\\\\\\/Nvczgf5I1F6MmSQQiCS2s6xZEH0+pxwbCq0mSEWMJI\\\\\\\/tC5+tUBQzSRUBRbLlRryd4TLmT529jx+hKLFQ7IBpy+7xG8NawPRdiQBLkNj1UW8pgVAbG3qL3nT3nYFXjSRkPEo8PJxAUttAiCAIAwu88GMOXYfxFdw7eUJZxri3o96NrANzxwIm\\\\\\\/NvjSH2tKx+AN3+JtHcD3CLWCU5aG9k96iPCRay74BQLkpJv7fnTtlvBQTsXJHk3GyLRgiQKDb\\\\\\\/t40Xf3uxCoAyWKGrrXGBE7o4fg8qLQyiIkNCwY09BC3686hmpNtJ1\\\\\\\/qko3UqJPYWZMJek\\\\\\\/NeLkEaoBZ\\\\\\\/8iJ9Bh3NpRYmcH0PrJ5cjUi3IIdrfA4V1IZLDZZyvhVfmcH\\\\\\\/qHuuPJo3NMkPMf5a\\\\\\\/rzdPui6XgRkF2F8knqEk++yJQYAuGVPCpEvMFlHjB5SVf9dti3uNPhltHrG5kGdr+CJmPRkTynPnn+gO6GQGbnbOaBUzqVXpEhuhdQ7BabkXY+\\\\\\\/GxR705zMu95B\\\\\\\/Jqj6a4Gxx5GJpV9d4T4fte2xEdZeGmsDwX0n\\\\\\\/+xvCTCF3i7thwRtmGm8T0C5Krn\\\\\\\/C1UA4tDEsWKIwRfs2tvXMkRRXsNHiyuhES9O7N\\\\\\\/Be9BHYv4gmTftmtZrd92PKcZ7XERPrKQpz2Hzh6i0plHbRGURd1d1XimqlhEYZdXFr36o7HmcqT5+zrPAcBVIkAb3aTvTngUbPxtYqS3m9VPp3IJiLmjT+\\\\\\\/MxZGnkKWGiwOMJZlvo0F6KhKO4J3BMdmESV5qyehdXRrAh5tKPz2UWXZa\\\\\\\/IgDFyErLuCCebqs5va0VmbqOZGqjzCBPtod795tK2RMACJED5qeFPypV6NJpTS9O2VrPq6WSqRsfwE4Gjem\\\\\\\/kaG+\\\\\\\/BXBHpNTR1g0bSdDXrs2m3MhZzMgQj5jfHwh3fTV1kbepxxvi9ZP91muRXMFvJkvMWqe8QkS9CYPQWi+6ZaSfRTQbOAyMaX4P7l\\\\\\\/c52KdL280HeKNTlnIHXpVp2rcXWy+rI2qTafgea0lVjSVK8m91s6HJQ5P1NWCs+fn67Vq8gDOl5AhUvbsbvP+TqfkgLWmUwfCK575cIGzzAylmuZ1GSWUPcawZqTrTUrfGwxJc5XRh6faR57xSrEspgKeiEYBtYmz8fEWju9Qjxl3FUolgvA9upnVDzgjzUGS9LVvjUJKE0B+28dwf6MQkpnZNcZr9nQlZ7QCq68jPnv5P0pN+tjhp5hzWOINwz6VzX\\\\\\\/F26t+aXP7o0hcGTdDKmgvZXTep+p1vxW7fFAnhexpAPqfZCEYsOn\\\\\\\/RAyaseGkD9IMDAxjl1zQ4xRXeeZikmQ4jYJmOVOJ7kC6iS3Wq77WzY9f0s5FhI3p4PyKTaygxdtC28eOZOoqAl\\\\\\\/TC6K+zj3qEcY10wV7ma\\\\\\\/h8RDtC4OMJN1QQL0sjy16zH8agBQw3blrUUGF1JVEmzBUN\\\\\\\/2jkwg3yh44fm5F0AhiFW3UZjf2uT73IgIEvyWDEMg\\\\\\\/sGQxnvdoKzWH1vy8wxmGZft6Q2VDL456zeTnz6qP4WFkVCKEGyKDnMtkpB1Re2FgnhoAZJ\\\\\\\/Wh584md0x6wv2UYDq339aEDLdI1OugnJIvCTblIGF77EgIEPN5JKf7W+jzLyf8yG40\\\\\\\/26kKRRWusfGCeeJC5JVNGp7dWbYIT51xhocL3S9fYU+DOEwcLHad63e8fVWYLzkuV\\\\\\\/fsgtS0hzXttgl94uvs3VM7JmFzLIp4Gxo14ggzhKJFXjrY8eBpMaAru\\\\\\\/2FhPdALGXbH8muhGsMhrxhLPt99KqgHtm4kgUh4MvA0QUE4htxKoJ1hWp0HPRnuniCmnK+67vmPXN+RJii\\\\\\\/i8emKPwCFDWzvCabTxiwduwZmtmkuQg4dmXg2NS6X38Cp6wj20cEjCqU6A3ZL8xdqIz8g38rKT7WaALK9dObHINeL8q5Sz0ZtF5JeeeP7jIV3nwr7Vwgbm+JVDz5lVkBesfsdfg3vvbZ2Zh3eEw1CYTxDyVTCmnCNfNkpm3XcqOrUKHeICOmWJvO6zEEBprU5PGZ9V7+cUVZs\\\\\\\/wDRjXmXma4+kI+l6tZUwevPtUW+FOiYt3xmJVAgA1+JjNr0WIcCHlUwDjquLgSvNIgdJSQUaZT6JDYeW\\\\\\\/Pf38+K30uo5YuoPadkGJwQr2fSAgdZiVmyPwvp8msG\\\\\\\/33LuwzttK4EwhpZs0L4T+9RGh42MbF7I2evs4z+fCg2Nbt15Ae8zX7V1qAZHSupBXAPgu26Y8aWX\\\\\\\/OpuokWEsJHVS+TJFUl41iOT\\\\\\\/CTaOCMdis3BDBQzPNBTzW1M+3XZohzgvbSGZMBIn0oA1Y+nQK9i+ziO3wxcECxMm\\\\\\\/w5R6EJEcr8FDJvATDQW4IFmW7Pujd4753GSSOPFPwbJVPzb\\\\\\\/cYuYNlhLGYuFDxMFABJVuEH3Zdj7yGQ8S5FC+ivASKFsOzrlQ6MiCGWMdQVRm1V3AScxbN\\\\\\\/KHeTJsoUdJGTsiq1TWx5v1vEpfy2rIqpZmxM1yrdH4UYKq98BkBDmWqKnBfCOON2hH\\\\\\\/IZ9Tj3Rp5MhSIQHJc2KFyonX8Du6cwzIEHp3Ifmr0uYjcHQlfxY8sEqyJEtIpfwIPOY89+rZYKGABTpA+jjnRHgYsW91IT9Z4Kh9Y8PQ4tTCNcaUbe1sNgNh3RYycyhXIKGxJo022eM1r2TWoXpPRewEdxGr32wTGXLHLn\\\\\\\/liitcQGaKMbGnVxelV8MGlY94oIaiZ+GfYQj+Z1KMt5u2XtPXPsPgkj87ggvLEmDq2CMORTJcf2m5IrGL1qqcDUaRrOvD5YnIn19rNJz9H38YidQea3BLBhlNjOZJ\\\\\\\/Su+R4vNk1Q0YnjmgrHwRReAdgdtp4C8cMAeSXlhkD7tBH+bFIOBkYFFY3a6KH9DMtNy92RZZg5Uu7zQv7igGmaxeZsvJlLAcBwC9HWGr81fnSrC9p44rFAhu0X70b4h1ml5mXcqlaMn7QkEr9jj7C\\\\\\\/NuVytBz\\\\\\\/NAfM5CFZi3Ehh3q5V6m2C+2T019ozP764\\\\\\\/MsW4RVDqunnpfTXQUmHZWy4CSTIALFAW3sEhpH0wbt+OP72FL3H19tLg6vTiWsNEbjlomHVAvmd0Um7xRge37jUAd3atcMmMjCzvyEjzA3iphSedmyuairrsqPsKlCXtSwsiVriQ57+YAlRU4f7gbPu5EiIK64INFdXw9+KsAKGIHz0J3O7WyKPCPrKPwEuwjOfVICqfbdaoBwaRG0BLkWRbeRhmVMYnuG6ttQca94VhKM4z4NYfZ8w0wsSDWcITiTldxN+VwItXN9KfGmnilTwCTwi1ZvljLLMkAWe\\\\\\\/5VtO+0iHatsRK40bw1\\\\\\\/n1KcMOlgonPGuj83AE4+iwZENgBb0PDgGyrYewn6fO2i\\\\\\\/Zs6bYe33jSmRtXF64UFRNbIWKTCnWcKcKkCRNdEX8prVAdJwT6I555loOFie2KAa0whwZcOBPEW4Tn9iM9tCsi2BBDkPD+Lr70cubj+TXdK0vZyYKfraNS+YZzsvQm6yRAovocGjWfuC5uib7nq3ao7aaM7M3NbzKgDevWiQNaQK\\\\\\\/OEsnG+O2YvmrFU8bT9TVxwhD7QMX1FhwU0gBSOqRzAk1RriLTAs\\\\\\\/N3ETuzQhuL5vud8aPWd9Z04ogXhBg9ureAQm8L\\\\\\\/bQ\\\\\\\/eD0gSzv3YUHSumM+dShJxAQSHhrzywGkipNhLwcNnxvlTj56tsYtVqQ2uBZCBngdS9Mg2a6mW50W9GIFiUHFr+Bq5awg0RFNFC9tYFDLN1CUlvTNh0rAJaTKodAcVnSNWAMjSKfl8X1FP01BwRwq2KC\\\\\\\/gYLsP1TA3Jrnkq4o49fVZE4\\\\\\\/bVk8\\\\\\\/EeORMA3uGHJbrMkgQA4grlJPatNKt39VkvuM3GYmymy\\\\\\\/qBeRXUhRyTd3ejlv3XcKxDMjpATshEDtViHPkhFmIHEwDvYziaLvLJPfHX8LvYg4EA05gxPDC37KrBLXO2uPUBuqkM4cH6az1Ky14suhiM5KX5HiEv\\\\\\\/Wlf96gZUG6cDETYgIwbTlEzXDTij7Ej5htBasORkszYeulRp3YI6KgsAyO7MibvAksOldKyH\\\\\\\/2m7PNN6Mr\\\\\\\/SmKXPN24hSkTCE8CHXDjm3KdQOY\\\\\\\/3iYMRUGZZI\\\\\\\/h5eNx70YHhnFxddCbaZp7oa+2UJGcsLx5g3LE1eOzMA8ir2JcfWy58s2GW4o0kKotNwxaAY1AIQOImJwyLUbdG1EV\\\\\\\/CIrHVfNJAcbayVI36MmX4JAqGojPruURDChjVf4yp4qnDRSFmddQa45XF88vm5CjrfRlWWw4GVlOvS25GkkH3F2b36byA3\\\\\\\/hHqtDrwzpLiEpjpHJ1\\\\\\\/4px06SO0SmJNB6Z+db35O7wST\\\\\\\/e4v1mgILaaLsrX2zdhvp55TvB05q\\\\\\\/rr29nMRO4MTKJs7ZTvWBkKZSo7l0za4GYzr2iO3jBKhR2NK9ZFLPzx5gMejeTYoe0v+olAaAb6NqFckclEsMlLqZqKA7EagBnenADVjgXy56welMcvE8eVVZwHzlDVUUYvXFQ62i7f7Vz9USIDck0HC2X50sqbVSjI6GUpZ6v1qxbIFKLaqvGjmkI9BC0pTbn43RwIxArsktoE8uJCalI29mTFWv6Ukn7I8nGXpKdAbudFrlqGTPvoITMNCMD7oMxam8ypcMuoSqzSQzRhEE3D0N53YAWhmgcteGOrhmUGLUCrsFmIPuipiIwnqk+fIFCGdPCahm2DiWh5qBuRDxa6Uo79eePpXLM4LmuuyuOehGkpEtNtNNtCIHraG6C+b0v14j1Z7QEJ2CFGBGMHbO9cbl42r571Krr7kd64LcctBLcsnP8\\\\\\\/B8F79xjanZu7kGUfLbglyiK\\\\\\\/9iKW8w1UrwgXSbxJOJcHXo0JTw7C4sUgkMlfEb9zlTKrm6eFeSYbgDwzxVzaahxrxD3sxAYqq4OB7f1\\\\\\\/wdUR05gCzXY03mwQIjWxBIjrBbCN6QSmwTkE6Y32Q4g5dc5X5W2EpKwfhgCbUqGfxXFC20obvO+XYyZqMf0pvx\\\\\\\/vM\\\\\\\/gDfsI7nJ1NxXnbO4kHzo6vAuuw5+H3rs9xrQ0OVhG1qr1jwDA5+aDVzN33PCvBqq2ioEt5GfVl3SOLUWzAAONGbhKTkcPd2H9McLFnxAjeN04xwI4RhL8jHVLbANd0juMDSc2M9HbHHTx9G7CMIAn9Me9khp3OB68nODaEUaZ5Kw4JfTpUoUjV0yoiU+GQh7FR\\\\\\\/7tvE75JxQEB5XkyKUdHi451laSJs2xJMVODbSFtpwwXJX6i4Cn1szrgJ8NXsMK47N\\\\\\\/wUSxKBaMd6x5tFjj\\\\\\\/0d1D0SlOqG5uyp0TnFcgLFNg5xn6\\\\\\\/2Aif0cLcA10RcOYpeMPPVW6bKEaY\\\\\\\/V1qGZlDKblqpFDb2HcPHhniE\\\\\\\/QI9Rszu2+9HCgyliWna4RYVcC4jo6QKphQpFUTiJ8NfcM19AXP8qEyNZQeA97QUpeh1l0iRs37v3bfCZrWsXK\\\\\\\/fwHDuhBL9GrjPKoOcfkmWjIDxLvOAdObsMSaZMZRzHP6OkhlOWIUbX5CdVJOozSmCDVHHo\\\\\\\/wdJBOSJgJJajhdqQGYXrjGqOoDOsRE+Ge4GKfxafVu7pLyPbyAx3eHg0U8p6O8cM6kGXl0QUel9thlNgE7KnWAfD9CsBVi9R2yqiiz6L2WxiuZG6oMPADe8gl9BFPz+736T\\\\\\\/0PggYSn6pmWNYPXZ2PFg+vgxmlZMXW1\\\\\\\/we33nyto\\\\\\\/xNPup3kDuT7wW\\\\\\\/zc2ij3sHEM8H0peTazuOILi3DOit6BX8JyjOVxh1w7rUDjI1GhjVr3HemAxXsd\\\\\\\/QFKTYgeI9ArpLTjRFpTUbzshhJiw\\\\\\\/o14hP7nYuWTfGTC0kj73GKGrg+IQ3mZZX0jL\\\\\\\/SmJaKwxIgyuKmyNdFfLPH+FaOWWDPgXgm62YVPg8SSB4MgJUyuaIJYS4u9IcLmEgPbVPZQcar2A7oDfWHCDte4mNUWMhN+2rG1BLndVvQp8HrkTtoYdFRW+hzsY2zqLG6au5CakyJFLEGS\\\\\\\/Fib8LC0mQfeMhf5ImJ5QZvWZSHRF5P0KRgCXPvpQpZFXtR+v7Wc9xzUDfYN69MQbpACMI\\\\\\\/1IOxNJZt7dD6sfoetu8i6yqfZKoH8q5EJldpo97Ls1nadJoUb6gWNU9q81Zm3dnQtxdTBNg7BGO0pXBmaKcJ7DmPuJo8ZcL01IJkFMy7dNr8mWWITl727uqhgjQZR7nW6pHBC0cGSwDX7j1og9fGrmQ6aumngF+wcCFIuYn1QlDwiRW19LaU+J4GbYS2jTFlUz9BHcpBGDgPzYhBZfmyTC5T8b9ChjAZXqKOfX3cqJ7IC6ZBbuJ0XE8VWspa+E88TRNi+Cm1\\\\\\\/l5jAnfDDc6VxD2LOyEMSyFMecp\\\\\\\/MPjUP4zfPWcUjiqUoAdCwABiwwN1akjPuhFFgSgms\\\\\\\/\\\\\\\/eha2xw70b6HjZqr8BQFEQNuO+QihzIxHpWCRHMz7WIHEwXizCFOQPOz+HWf4zW0DBcT1oOYSIVbSjOrJ5Fq7cQhjGvNvFgfxgpEvKvsBaxIlO2WaYMgLpoUn5vw23w41hifAzsh\\\\\\\/No4TX5vS+gyGJ7MVcFLuhwzS+m3h+JFGEMshDH0lErzX7vJuK+h1VPQ8RwnkCVlX0OUo9NzTUWV196O\\\\\\\/t0OWODtQEeh+GUYIy0UxUBc+ApWQD49nCFXRJEwK+muxjAjaw+Ukc0ACrwllcKwAtyZNX0pK+3cwY1kw\\\\\\\/IV7Ila7TfJ7aYHxVLLmO19PGfKtcfmoiyVyPsRQAa7wcmvy7qx9Y4\\\\\\\/gwmNHEWnDkGzfu+bOb4mUUcSZsP0b2thXW3UrUc0yoXasxkej9oAk2xREvHGxamFGE3\\\\\\\/PVsO9x\\\\\\\/FOUpCUqHB7kSNeU087ALcxI8yBDyr5WpUZFNQqKZAsxvuUpMKtYe85+CaA9OGSXjeIULh4Q9+H7P391BuhhxSCxTQcd4gvniuIwIuZsfcWeKKBx8XCi2yHurZQ2dSuVh5y3+\\\\\\\/jRFL1puzyo0fECgqk8uMHP71hUwE0qA4+ABR0x72WW5VC5lx1KtrJItBw3DoKDe9e+YkeOL+iPUnTnf6hxZc0+GxC+lQpuIhN9RCLJQ+NfhBmAAj1bu2HbtQc\\\\\\\/DNKoemo4hKEZ8bto0Jyb0T3p6gE\\\\\\\/Pr\\\\\\\/aKxPydE9SzoFF0koOOBEVOs1nvPEezLTa9lbdWNU25Qm6NiJY9C+RMxUdk4e8bGH4DSQfaekYjYNxYSPIt144pUSIxSQfWK1Bz+z5t4PQ1ey4YOHeoZG4H9srtAAI9GFU9ba52buOElmmBE28HOegSbzkv3luFJgVkxskw0\\\\\\\/35IHCbKdzlcFqFDZQWvcOIDka972cknqug7r7OlloZryGKzqH2mrFAUgNX2xJ\\\\\\\/QTT0+FcS3M51+DPzPfXUAvA\\\\\\\/RRjv2peNxqoBrsrqDo+ybwGoaMkmxnZ9uN4FCLJLOIGBfjXrty1BIiu0Nfs2cn3\\\\\\\/fN5K3SJYqDNT+ztS1zVB9Lib6aYh\\\\\\\/P2hHKLrv57nHHjXkY6gW+XpsemVbj3L6fPJYNgzNJwS\\\\\\\/826nxLFE7vS3EqMTKLB9atmDjuUyLu40iITakQHYH7eHU8UEiYTSpcN4roz0hTPEbXYC4IAaUbiJctXELraSYaJDfNcW77jcqxuT2sAlSfpLEdbv91\\\\\\\/0YYlBOCBJQLbi26rENYQvZGTUCpok6L0SAYSUhO1a0pUpTm\\\\\\\/LylGYHFv6iUnPTjjSigfDyvKJUyURyIcONpMOAA7dLkgbL18WuXifhYTVdFVxYTmGIkjxWlM7zvPiU2uCWq12NFoZo2LMaK7zC7hzCjvBvuEZlPuM+rimEy5e3irBqWxeDA+lrrvpkIUEWOmUmrKrhl2COuMwbJjagIAXAoUOCzFQKGO0BoPnlxtV\\\\\\\/wXhOu4EbXGqeUmUDSk\\\\\\\/4XqBgYD1GJ8pt77exBhCpEVlW9X9MfWpuAUcTqwX972CykWQEJcpYMjMTl9oqNlOLYVmXQxD6+wf5q6AJy7ir+GhLM1BD+jAMTrly0rBXb+iNudHh96TgWSnKKQ1\\\\\\\/3Cgf\\\\\\\/7ARmKiFX+DlGxyEqFGuNa2o+4urmiiNHXWvKy+r6TBlPl4Nz45sY8wMvA1vsVxSj8SKcv01VJY8FAaZ6ZFo+0lFylcadatkg2eDkEO1n5XQoYB0hoDilWaxlPr3wP5Gj8j51DWZXrLWY4liN3wG6ykZk0CSopRk4pJRQWN1U5W\\\\\\\/b7JdUzQXM2d0oNDotUTKA0qhs0PY3Mp3Qo8hXfSoGuwz732mTp6t92\\\\\\\/WmDqhopL6STn8YU+Shy7YLpUrE++cWKl1n76qcZPnaKjEqDAUyN+rJWn1dLN8UMy5ELwWeiDMYncrVAs9\\\\\\\/UQfwP6QR+f6aLhT2AKR5VkHrlJm9TfMEz7yqE09rY+UUdTzg7SafJtkbSZQ1GuafxGZoTPI5h0wENIIPtVz5l3h7kfxihMXghnkU3r9Rl5P+ansca8\\\\\\\/l\\\\\\\/KfN8TVgCkvqmUrY83aqj8TMJyODhnNPy0mzhg3m5Vdt00Ozn4OphHo6auAgBCPNZFEU6r34WkdJWPUpJ18VaIftYmYbKGJ8XyQZ9Xm7tP8vXNYld4E5xA5nqIpIokaSJFcMmbxKQNvu9OaOfudKzWu1ZeEIP6ptHQ94HkN6lFhwvuILW2DjWTHPy7PxPRjMb3FDuGYV1pTPSlsXox5GAkN1O0lCkCAPtGWaCsQJIyxLO\\\\\\\/5aRja\\\\\\\/SKiNU7Wj45hT9+4D+7spJI0yn2N2QK63p1r+Sg6iALEKV3XS0+aFnnQj+mM4gOIg2rwDazpMhEj9qlioWtWCvgIuaBYR7T+hjguOz0gJe7trCYSPIlfrlnDsMBxLIoNJSbb\\\\\\\/bPxfGjmeGyl5mRUpZaBpiZPW\\\\\\\/dD11upCaPFURhcfz6Wa+cowvXwqlqA7YtuPLnpNMpibcAJoDIE+RLM\\\\\\\/KCHjSiMgFgOcdHn6reAUghuzDQt2ocMbFw6kznI83w0Avvv2xpPH\\\\\\\/lsRW1SWFf4Y5dFv8G5MRwuJO8S0pDwdUW81P8mfLjUTm9dVo5yy7q1mV7xx06mgZke9o552eunsd3RMEmea32QpNWRbEY9zj7l7krKz43zTp\\\\\\\/wq9G2TH70lv5suEJCSOq9iUV\\\\\\\/Y\\\\\\\/P8+JQNDMsrO\\\\\\\/RviKGQqMo\\\\\\\/lR1bJrOTeOUrIlNHOgSlZlKzTOcopnPvsr0wt0GTQTSGW6wreb5sxFgMXy5yuwgtSD3q3PkhFbt8sEV0UmYqD2Ysf\\\\\\\/E60bBT\\\\\\\/JZSgJ1GMCbb9msz+AqxEjKA0zPJN\\\\\\\/wMPBttcCWr5VeuSltkN8mXjn2ygjpObyODK2F2lIoH1PIMlPCUmEpCYraGbNOJMQO9Pt7CneLe9oEHrcwEtX\\\\\\\/1VKDLY+rqAbe6FzRPoMvTM8vz\\\\\\\/AsfshyhD7Y1PIZWdlZeJejnvaPBXIt4cLcRnjA2XnPGtaz\\\\\\\/1GOOYwdOQkb6fHWlq82b\\\\\\\/AVctHKkaLbpC3tQs3X\\\\\\\/yz51JBofpKGUsGS4E0Y2O+HFKKQRBX2S9NRQ6W6982d4POqSgivdq4LDDS4qpirbZFwwMZ86e1rtDAJb8tLx603UGtvAD7rWR04UPmqtnW5Snf9HdM83mqO6QEM\\\\\\\/bBnA\\\\\\\/zCU4cxD\\\\\\\/PgRrgUis3QGOh+jM0tzIz93lTaLkOh2Cr42El1VMYnkch2Va3rMfX7PXKzNPl1WPrw2RlED3+JsMlEU9su\\\\\\\/vMs0VUU\\\\\\\/LjYWJc+lXj2q3lCIjG0Ccf\\\\\\\/ob71k85pWxyQuKrMPYVsuJlfAQ+ErtRHMfTaX4rm1mhB0VRhrMUAN36o0imHWyw3vRHKOrDh52\\\\\\\/sldj2BtmMUXTRJp9831ZmYIvNnySIlyypLn7rpVqo17pG1qaGzeWr3iok8mGSh1bw\\\\\\\/qr+3VLxOjWIa82i8LjwGF2uiJacuWiBBNQYPVazzzGUxR6kwH2xJyxzb7uBJMjEiLJu9qnIBktC\\\\\\\/IW4z0dhIhUvUT6nMVeLQn7mVdhUGIrVJS4MCaLnWmtz0hmGIFh0Y6WmmFBL+j4aSxitMuey9RlhADFIwsCnfHdY9Ihtt31OUOwR4fdL\\\\\\\/hlpUNHoB5xD4FzYKELM6xt0hnTxTskKVKNRJ5P\\\\\\\/XPOWih7YanWLRQfHIdY6I7TkcNaSxjtXe97pgMrZK\\\\\\\/Zd5j2jpbTgVY+z4LqPhGOo3Wd2QjvfG57u4f\\\\\\\/DqK5CF3+hlKdEtocF+9l8yFK\\\\\\\/75C7n1o912aRAPbMxPdqcst0Y0kWMFydtjXeLk9y3vFgyWx5C9HT8RXC0V749VjCqW\\\\\\\/6ak\\\\\\\/dVyL05dcXOtzhTXt+4k83Nul6eMQfvJ1leamHO0cXEy\\\\\\\/8LcKLtpODcRzR8HEAJ5ORx3DDrT2v7FnEIuTNcvGNB0i7A9Kv7lny7JSIvwUNz3kxXDvmGkcRponWuwDW9Uf3eVgJopxqFFD3c29kx\\\\\\\/Mvv62DPSpBL9i6erDC650drHAoVUAV7A95lbTIdpgN3AD\\\\\\\/wcjdHxoVMqVCl6snMvKeeeDaznQ1k6cdXXiZkk3OS0MGLsxEWFCpGQ56QSohySUGk248HXW5g1OSWI2fbWoe8\\\\\\\/hGVtTDSmzXIxe2lJrMqTEqp6dvC35Buo1MmiISaim6rCOOLGT6zx8XtU9pe6lu+2FedD2nFYprJRDzsycoOLLIheUerfzTZ21nNfHjeNFYcPzjEbaN2rR1zvGyc\\\\\\\/P9hCDPNzKpECanyfwY+brcP4jww8XOe6xS9svXKu4C5HXhohMwvq8azffJpCffKnHEE9ZiAXjd\\\\\\\/YatngBomUeRnVDdZ6pHy\\\\\\\/k03vZBvxfWtUdphbrB0DfqExOQsrWzw\\\\\\\/MRReWAl6IF6aQzr1BbhJA+ewpk9pB5gq8SVtXc85euQuyBBRc0Fl75NXCV2MONsBHg7b3uz\\\\\\\/Uq7DkkEg2Br0qTtIK+LoNayZqj1C4iIZSrP4ntHyGNSoyIm6Ze9MMYnelWUqphCoyhguJIkep6l3byuRzC6A9IbVcewpp\\\\\\\/cipYV\\\\\\\/eWwhuQYg4frc1YeG8bb5OG1ncjXNmD7s4WzukG1mZSgRVFZequkHSjUIw\\\\\\\/qzlg2gYohKRJ1r4z5bgzV085CdGAVmLeUfkm9jccSCHgogV6ZhNwOr3y5cuHavSkZjQYE9ngG\\\\\\\/27Kqeny\\\\\\\/w+qCc6WmUwn4GenIpBwaOTlzP3MdNvTArGi2YQAGjrqgu61aUlL2Sbf\\\\\\\/\\\\\\\/plml1zz2b9MzB0n2tPpmBfmKC9jGr1HzHwD1d5OqaH4ukkTaqhQWXK\\\\\\\/kpbNQqS15PNuvkqTnaRxqLgVrNYNpMrinuSVcbbIPNnVOtbIOsCjYJt1mdQUaLxAHpHsfxH9oevRuvaCMErLiqdnmyh\\\\\\\/S3SlRy9vNrerU69mS08xESSNayFUgo10SKSk1JaimAvHpQfNAdHwDDCbVn7DZd1oMoIe6WO95tQaDg7HjcERRlFX7Za1yy\\\\\\\/oDpkRs78MkLR99OiDqaVVNjnyIPzvCvFHBqcmd4Opha0J5cfNHj7vdi9e3C4bj\\\\\\\/42qxabrMTVmM\\\\\\\/O2kyz3vLMj33XVaXvUut3PWZnY2jEk6Eb2s0eWMu0P4OGoeHCdLbvqQeGyIXxJHFNr6D0LN\\\\\\\/uclk+LLpMwTJAI2Eri\\\\\\\/5keeseIE2wIRK0X9WZMlWthJybeHyXh7SEXf66fdu8NY56ttTg7AvVgX95M9YBhW32inL1FEnwng5y+2oA1jVv1tAkUUk37folmBByr8zcBOR5uAnuOFc6pxun7Ia+e7QdCWf20jJ0MeUAKjap5UU9uhxiDxtdblCeBXerWKtNVojomKXkTbcMJgGrDxjim\\\\\\\/m3hqCJVY48lzqG5p0QAEAtDl+JnTMpZpWy\\\\\\\/2T73RC3PbnEMN+kU+Ff7VIX6W6rVHOOGeYFa2hIk6H9R+q+ZNE+H2lq2CA0fPgCm4lWmO8fAexn0L9rTf5ymZmVPpplgzE6sjtzWFLht7xi2hKJwWujwg7HesY1PaJTuYLTR9fT2rm8sa4fL2h45OLE2bQGm\\\\\\\/ND4QfxosCGFFr8l7gBB7+nh0i7Cad8QjTgQb5KG8WVmORD7fHX5m5JZ8NKVQpzEbQJgmdhl2n7N814+AFDNu+PuRT4cPlifpMY7B6cFmLIWSXpFN4xmWrRSJZ8CADcy5AWAdyCn4fCYoWoeaTGE4cz79ICen8PgZspzvXcs\\\\\\\/wxDwRWu7br9+Iwb5FvhHulhT6z2JkFYLfxSP4XQRDt6hbb24ws7iiCJzmkAJObWCzxYv4v2f6Wf92RruoSnDkNDfo5PM1TTAjNEd1Ondl8ORuB9ZYwvXHnZ2fIvCY4Nu4+W4SLUeBS3JAIKDj1p+9JevHnyaoRY3zKyzwF4xej70Jtk+02MOFOi26C2o0MsRmDHRGnZNL3mpXKRhDnwXuWu9ja7LzqrYur1jmm4IebYvap2fsmL6Jb1GkgAohRqhlaZ2HVKcpj+W7DrKq8rI6aO7UZMYZ+reZHiQTGby4ETCw1BTQwa+NptZHHjymJf8meHOn30bIffGaWRzXnFhXmb9uTYP6RhO2rO5meDOwgzTJh4hYOmhuMiixFjJQ2ezo7I9FEFPcURtMHqEsLa+QQiPXFDYULWgZ\\\\\\\/5KOBzpxjBLfkT\\\\\\\/MHy8glB\\\\\\\/ovRM+suv7JfsjxWSa0a8DqzV7wRyMNFq9Rs8vOeS\\\\\\\/eIDClclEPyl2lrSwYb31vh5G\\\\\\\/QSnXiX0a4bccs2Vbudjb3jHQ7VF8o6HXq8\\\\\\\/uhrjZmPaEm2EAFL4Y0uywZj+uVEt8p5LwqQ1DXIekA55v2OS6POcia0uEPAxzpm\\\\\\\/7lyWD2gtDXxctWjvDDaviuyLTFS6dJwAjQiBOAad99JXAdWtKMtets4AnqO8FbBo+5muuR\\\\\\\/zFFea08jUCSd9Ark+dLIulFUrF7JzTG9I8MFrcDGnhePtU1f8j00Dhml3Qa0stOc2Bg+7y96EPFBaXStK9IBISZyOxXM70zYZE4toBjNztAOrz7HV5EnwZ\\\\\\\/fTWN6ieIEwT9pewWxnn7c\\\\\\\/KlHbpQZeVjTjbMnK93HLWeZ\\\\\\\/Q7bZz73Fg3VRsYvFK3AHMqvLUnxHyJXFB57QiNY74G++v2AX8b\\\\\\\/Rw78+G9Z+zhW3ted0pZOrlWRMDCzTnK0eaX\\\\\\\/e4HM9FrtRKlCHVpHQ\\\\\\\/Vlohezuu5OIbqg0eQ9OLPUD+1vSSXBFNYoyaofQhOuXG9pYbyVT1TfDvrH3gMRu6SJWtoOLIKzc0SK793GzaJgqf8MJr2L2XNp+8DfSbF+F6ESt0JiwLIMluJQnhR1PHAU8\\\\\\\/eLqKy86K\\\\\\\/NXQLhCNU9LXqiARhPCm0DRFqYNkEi5evnbPO9372HXYvAKQhlMbABAZkB4H9ylz\\\\\\\/0Hm+ozcs5Wbn52GdUECZ0v6VYjno8dXqY50WjhCk9PMjnT3xgaYFv7leaYMh6sbiAVTzD\\\\\\\/MDGfeZi8W963F7wSFY9QRHFQLqjbksE5xsYFTXbN\\\\\\\/6sgelCmbbQ8\\\\\\\/7NGILzTfP4+xmNFbU7DAGR\\\\\\\/6XJYc+0CTR+g30XQ9PTrzcuBZE0taSem9eUG\\\\\\\/DtZBoy8R6CIAsC2Ed5rc+ClzjkyhE0\\\\\\\/b5vWeJvco26jbR1Nqcw2uJ5ctn+t1Cvm0JsGWEnL+JStDDapIsVUCPpC\\\\\\\/UTnTxSuqDREJtq\\\\\\\/U+3y4k2lXC0c7lqgsvda80th8d4XnDQ3I5sUbHsuO\\\\\\\/Wu\\\\\\\/4ssDTbF6HBguDCFKS9m9Dgh5lqpaBH2idyEphURHMBwSX1TOGtay2GM3sWn+yZwCXJi\\\\\\\/ixjxnBf5IUypc0lhoAXCqsk35wTCIWG4hWhMauIPgV+IAh9M+l+\\\\\\\/GAta7B40q+GHVmmUCpzMWmFG0AUf4HFGKMGCpNK\\\\\\\/\\\\\\\/ESlQoFg3AvZ\\\\\\\/beC8+AaUCQw9KqdlYO+phHutCqtj+0XrDic+ngI8\\\\\\\/s0pVO58RxoeKVHOW76wMcFogA2ExxGa8CZC4YfXDhXyuGaKIc7uhsqAldg0bAD6\\\\\\\/I6Vz7Xbw4GB9igdg2payjch1yYQvZ37e9lMKy7jTDRdS6PPQRj0Y54bkWZi+\\\\\\\/S40J1HPkWNgnoNNNGF+V+PRZdmH8e0jTZ1TAnuT27fp10V65Desqk4II7fzAcmJSiJpGBN0Oll9M7UpULYi\\\\\\\/WLWh9if8auKnxd3HFRjkYxfdgzQvwAEAFm2UkUG4yz+RSRMuKGkkZzQrT4THtbMhnoscpd5VpJBIoPnx9\\\\\\\/46Bp7KUvz8Dr\\\\\\\/hUK8aS2z5MTLPNsEd5TJIn9\\\\\\\/7pz8TE4FnXYeiaZ5SzNFE4R+wbSVPQ8MNKX9XhiCEi6Q8lM0W\\\\\\\/TN1IU4p0qoA8Ovz48z4SYB1HXx54oxLTQ+vHDkLPVhz9P+oQSE4DMAHelXCYu9C0L3hxHfPHsn4seNhZXQD5YftjHE18n5Im7FLNHZa7BWDv0ulOZ1dVEBTzKHtWkzia8p7DnbzW3kQgA6qripaGTXqDsJqQ08mQnLZWE58fs5gLA4DsDXlxgGbp4UM+ydRC6UaitS06aymGlk2E0O6x\\\\\\\/nFyly0T\\\\\\\/HzH\\\\\\\/dabype3RsWJJIxiXDK5rtRL\\\\\\\/PLw0x+i4sWFbf8hPfEGH3CqSD\\\\\\\/ZR7rg5C7PmCnTmMctw1Vhi08s25NJU++WUwWZMew4LMyQyVQiDbjB5vBoxmqrB1ipQAfAoHUxq67StTWMYOcZqg6qNtfmmib532nE+tWkpGSo18h3DHDZhQP0mkplYmvKreglH31SnbbcFP1pDFoNeKODbXrs1MWwoau2RZHAta1AQPN1wRYLo7ECl4TSuXYpvW9psrzMInHXM03DZgOgWcalH7p59cHGZ7YqOsns62rg2xAt4AzlLRViwfZcs+dLSxeWu\\\\\\\/PaMsC\\\\\\\/8+jUNoBj\\\\\\\/GH\\\\\\\/rJfcsj1uYQC9xwLGGi+UIAQY8QzbRb4d18\\\\\\\/DMLN8cNR4SkwUx46JeZQsrYhrzHLvuOAknBF0xWt2gCjTxqCDOLR336b2DpYZweADku8ohUP\\\\\\\/+kuolH57Ep3Z0KQf3TgUckpOQ95TPNLK4H\\\\\\\/qQvrBAjuVF0ugGSX3v6vPSfd+L\\\\\\\/MF6B9rr2tNBZKRzJ0QTJc+XxZuzrGPL8G+Hd1ZS59Dpz\\\\\\\/c4DI+ncW+dGOGed454vur4a4sD81INzfUF4sFVagWeQN5msDjyAgmCiWfdiHyg2\\\\\\\/9AZJ+ofsfAmypFCksVfW4b0GuRf64ZvWQAihfPuuaetNMSeKiPPEz+lKTRVsjz8blrNyi7tjhdAB8AaJGN+b2GH8YUfq3MRu1sTZOglzKJG7mATj4GYuhnhDHhBC2RxjG7OBvgPpT38sFs7wxtM5lmU67CFeHKw9LmytjG5SW\\\\\\\/WQF7YKBFSyJeqSd4wYFK7DLkBzVKO\\\\\\\/5+9K0dXTBg43HarDSPZqr8\\\\\\\/P2YjPHkp05MXv8mgnnbJC78yOeM4UTnYKBXmd2pSX3yFfFml4bVtnbNM3AnRSRyV8M3rsfpnX83mruFV9uYhUoLtkGe7yB6OiAV9mEhwsV87Gf7ctrr2dc+qdImpnxpuVSV8YB7IEzd3zrfW3EWXotJgS48dJIJ68lLk+gpfuMA2f1LXqh+p+P6zPhCMXjkUDALO64E79ZWkaGZ\\\\\\\/xIURFggROqeOcncl6gC1z\\\\\\\/UHKL07CCCviZmEoML1I5lxfVPR3uCtdrA9iWKhj34jS7+G8tmV5gtbsEhKEYaxFwMpu0v4cBt2Q9MJ6a6UwvQIUYm4skfyXpg1sd6FDgBOTNLaOIp\\\\\\\/h7IB+uTRsb7hc1WvsFlmgtjO1flH8yOTN36Sz\\\\\\\/8fU4sUfl+VptLcdlUe0UN\\\\\\\/Lraz2faxuMlS3LQBOx7O3oAe8zGMQfiG4aXn0M0iyieZ48crdg8ddtjb2r5k9bc8rvYAv6e+ttaa9SolVV8nVw3bzs+i64yufXo4yccOaozMBABCeHlaKaRnclgvTRKPUIB2qK92rp7MnawtC76b4P5J0T3RPv3SI\\\\\\\/rtAq4dKwfQQYzIbo6pnGcSqY90788wPE1ZlD0bN5YNHxnJeqDgRahm1ov6tSXCGeAaef5k2aIWGz0lAc94tSb7xYiLT2tKPkbdQP955+Sgm1OPUoFdCcZwmC0Qm8Krl60oa1lNYvX+\\\\\\\/OcEbW+Q+QGSg0TkfBMlbHytQqWS2UNQRT8EKoW\\\\\\\/9V1Q6Lr2tVujD8zQDiizMDCT96zYcDAmR2H7yIuH6iKChfuGHliyFo0Bqh0cQaXdQ88aEFfH+JYyZRy6NK5cowOI1xkX9df3Koc9FB1Ud6Nkw8UJdndn2aqZNVxgahbc29eUUfopMhQQD6CCb+LA5ZrB4cUtiD1EsJ2gnSuWiYmvEJwZ\\\\\\\/yMAU+h1lW96Eta9CrLMdmmoFQYzE1Ekgr+fE6eqOikvTsT0Pq2KbdnT1IScBXPiLl\\\\\\\/b33QJCnfRqJw+CrHNJHmRmsRnX8RVC97iH6XD13DHFybZsfvsnpWrMejU9PeyzZWPIgIunHRZXa3whzDaVKNdUT1uJmeY1FXofH\\\\\\\/kKVQyazCai5NV+d4Q0rSFRwJS\\\\\\\/Yg0saGAuck3huytXM2Gp6+7HhF\\\\\\\/c\\\\\\\/4mUVt9iLTeBCu81KN9jfJKzgf\\\\\\\/HZFViaJHYiY62y32UPqq4SVKRvW6OH9PtUmBwm76KxilCYXi119IizFX9tVzNZVafFqmshZd+NhoM+sLEcCbHUEgxSWQD\\\\\\\/jGtTwZW9DLH4Mdn+d2OHIYkTF8qkFfuB2wfSmXiiMacVw52YZ+kFwxKh+ynOM\\\\\\\/W0OAXhgt1Vhm3xYb7GT7hUTPXfHRU+k3jMH8CvPxpQOQZGqzF2\\\\\\\/9i3Ew2bujVtHwVHrpXS2q2cyLOgpYKaIx2ql4R4Xut8aOc7dYvtefozoS\\\\\\\/NiwZs9G1CDFOjwPviAkBQTFLiqgRb9bZ103+KTzE3brsNz7R+tTon9BKvh\\\\\\\/qjeQOEPhnk8iUeSPRKFjoRnSigv8qs5dE9BrxAgMVE6758Djcuy4X1aD++eTeZ2fmUpa5zk8ZArykemzhMdIc0+nq6SXORDRyerfoWu8GRF8HJdJ3en\\\\\\\/n9TFFPpsafQsiaN3Wim2EiDUyxSEDgoq80UlgzOW0FWa4IDzsFw5Bbch93wUKje1z6F7Q4ZSfV2uZDMiQ4Kns43QX8jyQhV4PDBH5A6WfIgvGYfEzQMwmJy65cVjRNPt8gtgg+\\\\\\\/hKzYsyuyftrAy471KWfWHFPbgwkWuiLQ7YwviiJ07K2tRSbPPyKxgFTEBXfv0Rxw0ex9ESioxp7Q\\\\\\\/IXUgH5m2UqdElV2GeJosgvGKrQVXiE7K26519YupVKoMouFGSX2UBMv+eXwB6Y0fqWsGzl5VDlrN4e0gU36NlunEJ5lgSlaTEXsf87YqH0B8gLiRnciCttj9XYLdnsKzUiGn9Y72srmJ4yWHCLD4i+ATltXKsvixJzMzDiwAls80mypU+bWvYtpahuOKeTC1sXXQ1+nj5FQcVq1kIw6jvEc\\\\\\\/mCshDWhNum+M26cESw+n5B+ZbPanEGcQ67ss3qYwXwu7DWIdDMUeVhoV12q6GDrDKI8xzCheBWf7ABkdz+wckE87xqMhXZcCjr7A\\\\\\\/X59iImZAHXq3SVWzbpzOPZd0l\\\\\\\/Aug\\\\\\\/X3FlmTSowbmt3f1kifKguyIi1CheUGVxFtnLLcrh+ElKoJNhQAzeNddYq7VNdGfeBMBOGRcHmHycP6UzpO1pN8lP40GVD18KstNvXEHerFVYnwvl848l5CG4oiH4D2qh1Ge3u2ympiG778g2b95Fs9VT77dNj3n4sJR178+rP6EFG4pANAOUL7iga1\\\\\\\/ceTd5+rxap1I04TLi1t7UvrcSIXQq3UQyV\\\\\\\/LQntfkr8u+zXiMQl+2qYUELU+YSOarS2EpRk0oy8DZowqHuW625mQAX6123NT60o2Ku0umZHqith3bci70ozuUfqs7qtHmD6Lu8RtFhrx9ibRyMkaCS+dAvLsKBrELywnIM5JCvd9xy6mJXzfN+1BiQsmfoqsRO7Tzm63IsHMtAzhrAipS9vy6jXLRjh67P6BxXAB\\\\\\\/+9Omc\\\\\\\/\\\\\\\/yFF9vWzCdmR\\\\\\\/DRxUJWJWjXroP2pO0V3xTlzcr3JWeqzAYnQJNDz5RG3nEE5eN1t6VgEoo+LBbMMY2ImEEH9tpOpgXcIdkfAkFga5uFnM8GPOq6GfkN1ug\\\\\\\/9\\\\\\\/je8KyjBDwp6KmDRJtDqTUzkSEqMAszAJ7ALfCn1CTDj020PAOjVriZAq7je3N2i3yjSxoKOZixwvHIMdEf9XryIluEOazTI\\\\\\\/0eiNAGM4NeGobXdg1S3VOyUGbkMDyDU3HHdtyZ7\\\\\\\/8KRZqtlyngLTUMfl7lqNb+VhrKYlviItNCn3L6DNAbFOVsNYuk2MGXYVt37b2XrnKviUNuldVWPd9\\\\\\\/3D1hieWOKyVjAtG0onkI7w5tXtiU3vTb5Kt6nZ\\\\\\\/zQC9CSVy1fcn3C3r\\\\\\\/LzzC0zB+VEBFtMtBGf85JGysTgXVnuE3dVDcrIzlWUiCBGuM4AlwjZVc9gQf+ptvkE\\\\\\\/bRKRU04alB8R2XSCqyvyF4vII2yMcIhjTX9iP9vm5R\\\\\\\/PnRR1ziyDC8FGnydtgqioz1ybg5mP\\\\\\\/TJwpNtSgl1OozBJBnKxKeF\\\\\\\/v5+q8Z44VUubN096fQOqf7GW97qXtx2m9+kmicvFo26vywkyGoNkzDCrmHlH\\\\\\\/sJjvFHdORxYqPPw+feYGU3uH4GxOOUoHX9BL0GzKMQcY8asl6OpfjfKTrJ2tC85f1ssSX092m+lv9tj8+LM+WnF3zWsz6GHyfG9JZsD3ra+CHdU3xgxuSTLvKqkg6CABKh\\\\\\\/PhwQRdO4VLF2s2VmQRhK4c5\\\\\\\/cpE6C7\\\\\\\/sovwSXh+DNd7Teo6vOpgBWGerNcd+H9vnRlC+iFG5Oy0J93QvCLWZ5X4Rcet9DjktC+2QofyGJPJGIMcL5rFy1VuDvseuSEtMUkgGSmcM0+XpFUJL18SNhrGZgst+v18Yq08JFCF6A0e+F948V09+QgzR+aIu7c\\\\\\\/EHwq23kVc7o6Fk9pzgKMXJtmv6BruS5G0SuqETbX1A\\\\\\\/mJMSvUiqLvXBiEVa2zsN2LoHbRw2VQn2ryrY3q2Z\\\\\\\/XptFeqjZcT\\\\\\\/N66\\\\\\\/F93FWJ5N1tLQSLDvAuWTgoOfivXqMpXQRW9fAd4NEvTJoFsLU+\\\\\\\/xB53trzhha76jo+\\\\\\\/R7lpZz09zD7JCaYLExqahuiXXe5qJRKuD1Ap2MGDTud4HEnoknTgUe9Cp056qWf2VKZ1Te\\\\\\\/nDfJRPY\\\\\\\/e0FqNLFU\\\\\\\/GOnU8GboFwd9MskqjvzHQKnCEuK9DSbzcigfW1WKF4OjkkyqymX1mzioPnsvHu89DARzW9KIFwOdRu+tgFeslopAbS2XnDBK1eK4kZIzrRtGmUaKOJI2vrJL6wWw2N1L4BzZL+wzDgzPpBte\\\\\\\/0fh\\\\\\\/aS5cBdcJudX6B9bwo\\\\\\\/hlpGfYstEYLJ+\\\\\\\/mY0CJ6web9bGylPm84vPqcjN8n+vJ06IL5ZDWvUz9pgCfAj4cZfXazHJpbfrAbuIPI96E\\\\\\\/ElhFTBWFU0N3ntv4nBn4yUTIC8ASKC0eMniFV9Asx7hQcZcGYdi0zcpXIetsrU5Fn1RbBPRmPtL8AJ05GtmQMryRP0MCBphYd88qFbJJp56N2u97X6f0J\\\\\\\/iQWhBihziDXkvxb0izNR2CwWuln3BySu58bFpJmqN+mIK7XDfcZ1HDzTrUb6dUeismJyguLYn6+AUqcLxRvx5Dyn74qMiAzck5lAbHBwnvN2CY3QDiGySafUcU2tMAhko+7QpzmNnjOt7oahHLj4kimqCwr2JR4RoBmfoC4XTMySdtRhKFHW9AfbJnJYACt2WDerHwpHGY6N8LgxB7Qg2BTeVf5xWzvLHeE4qIbIk3i6SrddOQu9IUwaDNPZ6loj0uvfcZ0HGhsNEyvb9EtCz0EsPEBTE6JUHJR5K6o+2b4mypOuO1gWiWJDjqmd+lpe5lZwqfnqR4q70nHt\\\\\\\/tHsZv5I9eQJQQFrbMwNdc0KTbOmki4qHJp39nd2Skf8WuJIRQjYfK1hxCvEODZ1Dy7AeEudUBpKqeOkiQuQJHyvb6GnY5By2YdQDjCMvDnic5+K\\\\\\\/RAjkB\\\\\\\/tE0rxhgK3rpUu\\\\\\\/rKOkaniQEXU8dz9Ug5NtV\\\\\\\/UsQQNa6GwSk8r8DsLNRVkQwGJAsqw3rjtVRC5wLKEAAZSrvIg4cw6hY\\\\\\\/gLihrgpmdzFdYtP8EnDDdgTc1HsRYqcaVDyIBEiiPFEsD76kdEDa3WEwWcLAT1ooz+nycPDW4kkIFBj1tXGF1LXCS7cp+9\\\\\\\/CrPqaOndVo3hOpNta+qug3tApHNEOoAug9jU8gAMRBnQDwBAnK+oWzFe67EqzJuqvh74NVLxVlyvRtwr6+BbbcZpwuE54yxsPKYBAoyTJJ90v0iymMOXg2zgK1kxTCEWDk4sNQrVrmgunDknPZfIYteGOjq+5To5b9CTYM5TdDW9LoOMg653FB33RcUQxgkrbPbzZkiqF2lK2Z1TMLjjUd4I3FP8X70+mOg2ngKBjLL3daf7xbyk3aY5\\\\\\\/0oDkoYLyZ9sl2EJIgIUb2Kl0tlWeFDAn5jG6BqrMBCCAx6aVbP0gv2DMxBAjqiwMcHI3RZRm32IYkpqojSSiNpGqnbkt0qQya\\\\\\\/lj4JM5+bMGIshYx3azrrMqjc2aAKuCTPwZo\\\\\\\/9CbxnP9P\\\\\\\/y8CcoXyHm\\\\\\\/kQpD9x3MUpexBBI4d\\\\\\\/CTTDsJUyTzyV+aQxMPQlrgULWogGiv4YX+98EbQ+46+bojkc6Y9NEBvmYRE0EOpFIGRmS1MBSxyDdkNwC280aGY7u7p\\\\\\\/1VgxUqyo7H+vXiqSviWdKx2jlEZ0fZSM6PeeWpPglFgDZK1AeQrry9+Dzapwz6duRxEVMnAT6Q1ZAOgJTPct56VdKjDlTrNbmdCkmMx6b5\\\\\\\/ljkLlMDnkpij6zZojKqzBvzX3NeS7428Q6ajdRB9zyZl3dPnIhEYrzZQi14E4p6HCl+Hk8uyHF5XTKW6fQ2cNrDVRzID0uoMdUl4YTU\\\\\\\/e5VkzNSFP1IPNkx9+ChcfSTbk9SlYW\\\\\\\/9QUS+pxnKiLf6Jok8IxnUdHoaxbDh\\\\\\\/IfQz7Z8P5jNtP8UM\\\\\\\/6ZWUwqW2Z13MAFtWKNWpBMhk3fshRkXsFw5DiwU6abFRObbXJ8p2NWQOLwih+6g5\\\\\\\/iPoP0crtB54\\\\\\\/G92RvJCI1mAm5gPaDJaAWCNz8Tq21T15buj8X2O+p4Y\\\\\\\/KB2oAMgRo0dLw1Ybfq4dReZVS5GqtLhUZzGPC5DIF2GpxlsHCvT3rHv+tzLrftv0wu90g9Fm8sfvo4xNKgFa2WKvhaYYnpMM3uHEswWidi4HOyZoRLiZqUTO5NFLeUjlbYms3mFCb8LpqcNMwcamkE0HFDy5dEZau1Be\\\\\\\/CvQRqVWqpLmEqYND9hW4JS8x05Qa7r4bUFB5dDPWYvqwDico2ceidPo1OHhtZAt8ljfyxArz4SbEvr8r7Q4DKvb4LcrrIcNMRQydHmLyfsgB9aLcF94O4suoQhn0F+5TrJLSW0896bmLIItSvF8y+QQK3R7CRv5W88T2YisHEk4ohcUhDvlOnjJDxJbe1QxZ86h9THHH5vJ\\\\\\\/sKFVfMrnApbEVBb8ooLhX1LqZoBe0CV\\\\\\\/ESlXHpSLAeaBQzhYC1vhjpoIscurgbv8mnVikcKcl+HUOCllfM3\\\\\\\/cZOXy2bT6vpqqGjQtUOh6aD3wFUsSVRo+5GaorjHUWMYxkCWUNeIggPkQVkAH\\\\\\\/wUyRSdCjS+tLv6pd8jpBbEb9yjJK13l+qBH7FOvs6VSboSXTndMl6NZwl6Akf2ZU5hafGAhTilAoyMbKkxpTlTfpXHAfxdYZyOd7Dy\\\\\\\/FA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022sB0OXVO6QXwNzeaRrP5KnQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002244928aef8f582fafc4b2f67e077a0c6d79bc27c8328dcee96f1317dc30aeee74\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ga1ZwSMYX9rIC7r91pm8\\\\\\\/\\\\\\\/lJLHJ0vCYA4WF96tuUzu2W7\\\\\\\/veB5zIzFMgCoFkDgQTMNflrDZiiVKaDNbtCxZLaQLoKM+0WxNrVlVJyD2tA8QTrwnQNu+o8IdNeK+5vkkI7poqYP8BMGazJRoFzMRqALyAy3kH\\\\\\\/LTWRPNdgtr2AszpH+xO7eFRcptMfq0+Ppgwv\\\\\\\/rrpeYWB41XwGNKYaX0Q3jrmvjyxHkwNrOJ56r\\\\\\\/2fiSMk8cR7tdsu7\\\\\\\/zBRWSx1x3RnPFuPaF446rfdRQnvW7doa+u0B\\\\\\\/MtdUDiu5txmEttgslEgQmRizab+4ixm1c2YATRkJZ32JvoCdVCnmfuisDlFZ2Brs9m5I1EuSoJ28dtELgdZQxc++USiLqlpdfU8D525XrQjoGjVTIlNLfgKoKLhwBk5yuT5bHaUYLG52esLzsaw+7LOGBtTjUZl7HUdOMrPaikMXcF\\\\\\\/x011qKfkRendzuJsKzcyQ9oETluH0qfLzgpR\\\\\\\/urZwzR9LSIG8j0LWpzHbIS3eGv4qnDekdQIIgN3fqXxb4Ni8g0+klBLAfspmkWE98c9HdGueE8onLAnP6MDfV3OtAZBNSyXrs+kAdA1TcURcWPx+iCm0CFsfOhk1Y+LsMlT2Sl0EyNfLisbhRkEtK5BCYTR2fUYH0SUnyZloysvZVzSOVhPCO1XESBJDsKql7LNM9EXrkRjfulGjYY6bYS\\\\\\\/Hl7c115Y+2Yf6vAkLG+fPx8uvyR1f5EZXyWAWKddCc2XOF7xFS8pPdkwFv\\\\\\\/p0lx35zJmo2UqB6AotGzsejSMzG\\\\\\\/ZOcB6EJFcTlJaQEp+x48VdgdDDnqzXPT3NJ9l\\\\\\\/MVAVgHIPmmXT22z9+nD5VghgbY17NzvtJBaTcT9aYqA7xdi\\\\\\\/B6v6oqiGRhi2Do8uXRAJGGOuPUgfIAwgklkwSDWDK47c8bdNeUVLPCxKgyXOACKCjqk2TuvXWNCPCvEG5wT\\\\\\\/WvoV\\\\\\\/6SsdZG+TEB+7ZttrzFJRxteBB8rWPfZrVFpIYsMmnRlYeqfTEcIUiU547g7v5qJjkWIk0QW8leteSVReSlUnx9MmZttSA4fIO\\\\\\\/ygdjRUkeW\\\\\\\/f9Dm5YPi2W2GTihXgIAHVHHIRHAyQPj1Xu22zw5jcpoH\\\\\\\/BimiK+5zUw0KtZIOQri\\\\\\\/tBicgD5NQg4QUNoIPz7yCbsSGQEmPO3vyNcSknGv9AErxqO\\\\\\\/+1unbaFmlDKtwRpGeH7zEqNw7H4F\\\\\\\/9HuT5MqgPmLXrFibk15JIb4DVzIFSCUu8yYd3qmEi43VhTc8w3QQ60FqTv7p84UP7zPjvOzO1FlFy+U5jf\\\\\\\/rtGMzIW\\\\\\\/c+gJGJzYRjflUHixuSLR7Y+YOuWo\\\\\\\/ltv3gYRhOYy+7V\\\\\\\/9tI3937fp1IRYlC2wbIv0DtbnTiIykVhqkAgIpLfBqnyAcHhGDAqRqmT102vsEDRlg\\\\\\\/DQCDl0GX7cqRTmvB2izerQAAr4GaZ4Hk\\\\\\\/FgqjyqRaUWUu+aFH0sH\\\\\\\/Y9dA3sCVpXC7JJFNlkr3tlb26xL9jwPFekgmRl8\\\\\\\/OIqUznaMAk3DHP0qtgrdSijYzSxgSw6SD\\\\\\\/veXsAcVSwGN4mbbBKBowQ6aRLtchf2GNeJafXMggD926lXxohkIHCYB6oZmiqqVNi9fk7MB9hpKa3a3QITbeOuKM0lt4uZapLFpnfkUR9m5UGOvoB9hI2j9emhW+tOxGUgLrV0sPBA8UFtBDLu0wfcbU9cqGiFQeIPUr6eHhqCvoLirKBKGdttXDlLQ68I3pRvQ+T8zu0R3KWAEGsyZAHymsU8QhulEE1S0q5IUBXHQNezThSBAcoV\\\\\\\/kiuN9Yj\\\\\\\/7I0+z8\\\\\\\/b\\\\\\\/mRiEDBUXsQONAGFmiJWhtKvegk6WlfMDLb6sj\\\\\\\/ZOExaK6cCDBMveJdlFcIUgllMgmmqnbGDI3\\\\\\\/lGw+U\\\\\\\/kbknOgjnuT4I2+MrjnPnCF1jmBqZrFPHKTQhEfZm9kAUBAV2W7iMs51lwGoN5BwMyGV2FTxMgjsoogKVVwgmKI3ePMVJVKYS1\\\\\\\/q0bmf7alOiA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lbQRCTfgFMHEdL7vH6cPJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224ac56a73bef59f3723d6f86beece429bb7ff335ca21a3035388fc44691679240\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493239 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1795" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yRVxcdI2FcSIKkhvE4ycUG0W9rLY5vi0rCZeWvvTOjI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1795" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-67b2z7VuvYDAvRUq_nssNrFiS0lYVLN-DmxivQuGAHo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1785" title="Whole Genome Sequencing and Re-sequencing of the Sable Antelope (Hippotragus niger): A Resource for Monitoring Diversity in ex Situ and in Situ Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1807" title="Evolution of Antibiotic Synthesis Gene Clusters in the Streptomyces globisporus TFH56, Isolated from Tomato Flower" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493239" id="node493239" data-pisa="ggg;9/6/1795" data-pisa-master="ggg;g3.119.400071" data-apath="/ggg/9/6/1795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vishal Kumar Sarsani</span>, <span class="highwire-citation-author" data-delta="1">Narayanan Raghupathy</span>, <span class="highwire-citation-author" data-delta="2">Ian T. Fiddes</span>, <span class="highwire-citation-author" data-delta="3">Joel Armstrong</span>, <span class="highwire-citation-author" data-delta="4">Francoise Thibaud-Nissen</span>, <span class="highwire-citation-author" data-delta="5">Oraya Zinder</span>, <span class="highwire-citation-author" data-delta="6">Mohan Bolisetty</span>, <span class="highwire-citation-author" data-delta="7">Kerstin Howe</span>, <span class="highwire-citation-author" data-delta="8">Doug Hinerfeld</span>, <span class="highwire-citation-author" data-delta="9">Xiaoan Ruan</span>, <span class="highwire-citation-author" data-delta="10">Lucy Rowe</span>, <span class="highwire-citation-author" data-delta="11">Mary Barter</span>, <span class="highwire-citation-author" data-delta="12">Guruprasad Ananda</span>, <span class="highwire-citation-author" data-delta="13">Benedict Paten</span>, <span class="highwire-citation-author" data-delta="14">George M. Weinstock</span>, <span class="highwire-citation-author" data-delta="15">Gary A. Churchill</span>, <span class="highwire-citation-author" data-delta="16">Michael V. Wiles</span>, <span class="highwire-citation-author" data-delta="17">Valerie A. Schneider</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="18"><a href="http://orcid.org/0000-0002-8455-3993" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anuj Srivastava</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="19"><a href="http://orcid.org/0000-0003-4054-4048" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laura G. Reinholdt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1795-1805;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400071 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493239" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vishal Kumar Sarsani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vishal%2BKumar%2BSarsani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sarsani%20VK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVishal%2BKumar%2BSarsani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Narayanan Raghupathy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Narayanan%2BRaghupathy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghupathy%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANarayanan%2BRaghupathy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ian T. Fiddes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BT.%2BFiddes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fiddes%20IT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BT.%2BFiddes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel Armstrong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BArmstrong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Armstrong%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BArmstrong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Francoise Thibaud-Nissen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francoise%2BThibaud-Nissen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thibaud-Nissen%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancoise%2BThibaud-Nissen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Oraya Zinder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oraya%2BZinder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zinder%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOraya%2BZinder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mohan Bolisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohan%2BBolisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolisetty%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohan%2BBolisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kerstin Howe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Wellcome Sanger Institute, Hinxton, Cambridge CB10 1SA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerstin%2BHowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howe%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerstin%2BHowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Doug Hinerfeld </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>NanoString Technologies, Inc, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Doug%2BHinerfeld%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hinerfeld%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADoug%2BHinerfeld%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Xiaoan Ruan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoan%2BRuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoan%2BRuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Lucy Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucy%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mary Barter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BBarter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barter%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BBarter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Guruprasad Ananda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guruprasad%2BAnanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ananda%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuruprasad%2BAnanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Benedict Paten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benedict%2BPaten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paten%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenedict%2BPaten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">George M. Weinstock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BM.%2BWeinstock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinstock%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BM.%2BWeinstock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Michael V. Wiles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BV.%2BWiles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiles%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BV.%2BWiles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Valerie A. Schneider </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valerie%2BA.%2BSchneider%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneider%20VA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValerie%2BA.%2BSchneider%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Anuj Srivastava </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srivastava%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnuj%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8455-3993" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anuj Srivastava</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.reinholdt@jax.org" class="" data-icon-position="" data-hide-link-title="0">laura.reinholdt@jax.org</a>
<a href="mailto:anuj.srivastava@jax.org" class="" data-icon-position="" data-hide-link-title="0">anuj.srivastava@jax.org</a></span></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Laura G. Reinholdt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BG.%2BReinholdt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinholdt%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BG.%2BReinholdt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4054-4048" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laura G. Reinholdt</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.reinholdt@jax.org" class="" data-icon-position="" data-hide-link-title="0">laura.reinholdt@jax.org</a>
<a href="mailto:anuj.srivastava@jax.org" class="" data-icon-position="" data-hide-link-title="0">anuj.srivastava@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/6/1795" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493239" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1795.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493239" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/6/1795.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493239" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/6/1795.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493239" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493239/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Isogenic laboratory mouse strains enhance reproducibility because individual animals are genetically identical. For the most widely used isogenic strain, C57BL/6, there exists a wealth of genetic, phenotypic, and genomic data, including a high-quality reference genome (GRCm38.p6). Now 20 years after the first release of the mouse reference genome, C57BL/6J mice are at least 26 inbreeding generations removed from GRCm38 and the strain is now maintained with periodic reintroduction of cryorecovered mice derived from a single breeder pair, aptly named Adam and Eve. To provide an update to the mouse reference genome that more accurately represents the genome of today’s C57BL/6J mice, we took advantage of long read, short read, and optical mapping technologies to generate a <em>de novo</em> assembly of the C57BL/6J Eve genome (B6Eve). Using these data, we have addressed recurring variants observed in previous mouse genomic studies. We have also identified structural variations, closed gaps in the mouse reference assembly, and revealed previously unannotated coding sequences. This B6Eve assembly explains discrepant observations that have been associated with GRCm38-based analyses, and will inform a reference genome that is more representative of the C57BL/6J mice that are in use today.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/reproducibility" class="hw-term hw-article-keyword hw-article-keyword-reproducibility" rel="nofollow">reproducibility</a></li><li class="kwd"><a href="/keyword/reference-genomes" class="hw-term hw-article-keyword hw-article-keyword-reference-genomes" rel="nofollow">reference genomes</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/de-novo-genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-de-novo-genome-assembly" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">de novo</em> genome assembly</a></li><li class="kwd"><a href="/keyword/long-read-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-long-read-sequencing" rel="nofollow">long read sequencing</a></li><li class="kwd"><a href="/keyword/laboratory-mouse" class="hw-term hw-article-keyword hw-article-keyword-laboratory-mouse" rel="nofollow">laboratory mouse</a></li><li class="kwd"><a href="/keyword/c57bl6j" class="hw-term hw-article-keyword hw-article-keyword-c57bl-6j" rel="nofollow">C57BL/6J</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mus-musculus-domesticus" class="hw-term hw-article-keyword hw-article-keyword-mus-musculus-domesticus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mus musculus domesticus</em></a></li></ul><p id="p-3">The inbred mouse strain C57BL/6 (B6) is the most commonly cited and well-characterized laboratory strain used in biomedical research. For that reason, this strain was selected by the Mouse Genome Sequencing Consortium (MGSC) to represent the laboratory mouse reference genome (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Marshall 2002</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mouse Genome Sequencing Consortium <em>et al.</em> 2002</a>) in 1999 and it is the background strain on which the Knockout Mouse Project (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dickinson <em>et al.</em> 2016</a>) is creating and phenotyping null alleles for all protein-coding genes. The original whole genome shotgun (WGS) draft assembly of the C57BL/6 genome (MGSCv3) was later updated to a finished, clone-based assembly (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Church <em>et al.</em> 2009</a>). The finished assembly is comprised predominantly of Sanger sequencing data from two bacterial artificial clone (BAC) libraries, RPCI-23 and RPCI-24, derived from the DNA from pooled tissues of 3 females (kidney and brain) and one male (spleen and brain) mouse, respectively, representing inbreeding generation F204-F207 from production colonies at The Jackson Laboratory, hence the sub-strain designation C57BL/6J (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Church <em>et al.</em> 2009</a>). Since 2010, the Genome Reference Consortium GRC has actively maintained the mouse reference genome and produced updated assemblies, beginning with GRCm38 (GCA_000001635.2) in 2012 and its six subsequent patch releases.</p><p id="p-4">Despite being one of the best-assembled and curated mammalian reference genomes, GRCm38 still contains 523 gaps within chromosome sequences, and there are nearly 300 unresolved issues that have been reported to the GRC (<a href="https://genomereference.org">https://genomereference.org</a>). In addition to gaps, these issues include reports of localized sequence mis-assembly, missing genic and non-genic sequences, sequencing errors and suspect variation. These types of assembly issues inflate false positive rates in reference-based variant calling. For example, we reported an analysis of systemic exome variants called across a wide variety of mouse strains (including C57BL/6J) and showed that a significant fraction of these overlap with regions with annotated reference assembly issues and/or gaps (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fairfield <em>et al.</em> 2015</a>). While a small fraction of these are expected due to private variation in the reference genome, the vast majority are likely technical artifacts resulting from unreported issues in the reference genome assembly, or regions where paralogous gene copies are not fully represented.</p><p id="p-5">In an effort to close gaps and resolve other issues in the current mouse reference genome, to minimize variant calls associated with GRCm38-private variation (<em>i.e.</em>, to bring the mouse reference genome sequence closer to the C57BL/6J mice that are currently in use), to provide a <em>de novo</em> assembly representing a single individual, and to identify additional data to support unannotated genes, we used high coverage, long-read sequencing, optical mapping and short-read data to generate a <em>de novo</em> genome assembly from C57BL/6J Eve.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material &amp; Methods</h2><div id="sec-2" class="subsection"><h3>Sample preparation and sequencing - PacBio</h3><p id="p-6">Genomic DNA samples were extracted using both kidney and brain samples from the C57BL/6J Eve female (MouseID 03-03685 (The Jackson Laboratory), Strain ID 00664, born 8/27/2003, generation F223) by phenol-chloroform extraction of a nuclei-enriched pellet. DNA samples were resuspended in TE buffer to a final concentration of 300-400 ng/ul, 260/280 1.8-1.9(<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Taylor and Rowe 1984</a>). The PacBio data were generated from three libraries prepared using the Pacific Biosciences SMRTbell Template Prep Kit 1.0 (Pacific Biosciences, Menlo Park, CA, USA) using the “20-kb Template Preparation Using BluePippin Size-Selection System (15-kb Size cutoff)” protocol obtained from PacBio SampleNet. The BluePippin (Sage Science, Inc, Beverly, MA, USA) was set to collect from 7-50 kb. After sequence length QC, the resulting sized libraries were repaired using the “Procedure &amp; Checklist- 10 kb Template Preparation and Sequencing” protocol. All libraries were sequenced using 294 SMRT cells on Pacific Biosciences RS II platform (P6C4 chemistry). One of these libraries was generated and sequenced (10X) by Pacific Biosciences using the same protocols and chemistries. The other two libraries and the remaining coverage were generated and sequenced at The Jackson Laboratory.</p><p id="p-7">For IsoSeq cDNA sequencing, RNA was extracted from archived whole brain samples from C57BL/6J Eve. 1 microgram of input RNA was used to generate cDNA (Clontech SMARTER cDNA synthesis kit), cDNA was size selected (3-6 kb) by BluePippin, and 400 ng of SMRT-Bell library was prepared as above. The library was sequenced on the Pacific Biosciences RSII platform (P6v2 chemistry), 532,941 reads were generated with a mean insert length of 3,004 bp. Quiver (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2013</a>) was used to predict consensus isoforms and for polishing. There were 31,076 high-quality isoforms, and 11,661 low-quality isoforms with average consensus read length of 3,042 bp.</p></div><div id="sec-3" class="subsection"><h3>Sample preparation and sequencing – Illumina short read</h3><p id="p-8">Genomic DNA was fragmented and Illumina whole genome libraries were constructed using the methods described in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hodges <em>et al.</em> (2009)</a>. Steps 1-28 were followed to produce a whole genome library that was then sequenced rather than used in the enrichment portion of the protocol. The library was quantified by QPCR and sequenced on six lanes of an Illumina HiSeq GAIIX (Illumina, San Diego, CA, USA) using a 100 base paired end sequencing protocol.</p></div><div id="sec-4" class="subsection"><h3>Sample preparation and sequencing – Bionano optical mapping</h3><div id="sec-5" class="subsection"><h4>DNA Isolation:</h4><p id="p-9">High-molecular weight DNA was extracted from mouse spleen. 70 mg of mouse frozen spleen tissue was place on a Petri dish over ice and chopped with a razor blade into approximately 2 mm chunks. The tissue was then transferred into a 15 mL conical. 1 mL of fixing solution (2% (v/v) formaldehyde, 10 mM Tris, 10 mM EDTA, 100 mM NaCl, pH 7.5) and left on ice for 30 min. Fixing solution was pipetted out and discarded. Tissue was washed 3 times by adding 2 MB Buffer (10 mM Tris, 10 mM EDTA, 100 mM NaCl, pH 9.4), swirling tube, and pipetting off MB Buffer. 2 mL of MB Buffer was added after the third wash. The tissue was blended using a fixed rotor-stator homogenizer (TissueRuptor, Qiagen #9001271) on high speed for 10 sec. The homogenate was transferred to a 2 mL microfuge tube and spun down at 2000 rcf for 5 min. at 4°. Supernatant was removed, pellet was resuspended in 1.5 mL of MB Buffer, and spin was repeated. Supernatant was removed and final pellet was resuspended in MB Buffer.</p><p id="p-10">Resuspended cells were embedded into low-melting point agarose gel plugs, using the CHEF Mammalian Genomic DNA Plug Kit (BioRad #170-3591), following the manufacturer’s instructions. Plugs were made with 10 mg, 15, mg, and 20 mg equivalents of the original starting material (mass equivalents calculated based on volume of final resuspended pellet). The plugs were incubated with Lysis Buffer (Bionano #20270) and Puregene Proteinase K (Qiagen #1588920) overnight at 50°, then again the following morning for 2 hr. (using new buffer and Proteinase K). The plug was washed, melted, and solubilized with GELase (Epicentre #G09200). The purified DNA was subjected to 4 hr. of drop dialysis (Millipore, #VCWP04700) and quantified using the Quant-iT PicoGreen dsDNA Assay Kit (Invitrogen/Molecular Probes #P11496). The plug made with 10 mg equivalents of starting material had a concentration of 286 ng/µL and was clear and viscous, so it was selected for further processing.</p></div><div id="sec-6" class="subsection"><h4>DNA Fluorescent Labeling:</h4><p id="p-11">DNA was labeled according to commercial protocols using the NLRS kit (Bionano Genomics, #80001). Briefly, 300 ng of purified genomic DNA was nicked with 7 U nicking endonuclease Nt.BspQI (New England BioLabs (NEB), #R0644) at 37° for two hrs. in NEBuffer3. The nicked DNA was labeled with a fluorescent-dUTP nucleotide analog using Taq DNA Polymerase (NEB, #M0267) for one hr. at 72°. After labeling, the nicks were ligated with Taq DNA Ligase (NEB, #M0208) in the presence of dNTPs. The backbone of fluorescently labeled DNA was counterstained using the DNA Stain from the NLRS DNA Labeling Kit.</p></div><div id="sec-7" class="subsection"><h4>Data Collection:</h4><p id="p-12">The labeled DNA was loaded onto Irys chips (Bionano, #20247) and inserted into the Irys instrument. The instrument automated the electrophoresis of the DNA into nanochannels, thereby linearizing them with uniform stretch throughout the molecule. The stationary molecules were then imaged, and the automated process of electrophoresis followed by imaging was repeated for multiple cycles until the desired amount of data were collected. The stained DNA molecule backbones and locations of fluorescent labels along each molecule were automatically detected using the in-house software package, IrysView. A total of 244 Gbp (∼80X coverage depth) of data were generated, using 3 Irys chips.</p></div></div><div id="sec-8" class="subsection"><h3>Processing raw data - PacBio</h3><div id="sec-9" class="subsection"><h4>Quality trimming of sequenced reads:</h4><p id="p-13">Raw data from 294 SMRT cells were imported into the SMRT portal (<a href="https://bit.ly/2S63Tav">https://bit.ly/2S63Tav</a>) and subreads were extracted from the raw h5 files within PacBio SMRT portal. Subreads with polymerase read were further filtered according to the following criterion: quality &lt; 75, read length &lt; 50, and polymerase read length &lt; 50. Finally, after filtering, 32,210,376 subreads (mean subread length of 5,753) were extracted from 20,081,751 raw reads, providing theoretical coverage of 66X for <em>de novo</em> assembly.</p></div><div id="sec-10" class="subsection"><h4>Error correction:</h4><p id="p-14">Error correction of reads was accomplished with the MinHash Alignment Process (MHAP)(<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Berlin <em>et al.</em> 2015</a>) within PBcR. Continuous benchmarking of correction parameters (k-mer size, hash size, min-mer size, error rate) was done to obtain the best possible set of corrected subreads. Our analysis indicated that usage of more sensitive parameters (MhapSensitivity = highOvlErrorRate = 0.05) significantly increased run time but overall improved the quality of corrected reads.</p></div></div><div id="sec-11" class="subsection"><h3>Sequence assembly - PacBio</h3><p id="p-15">Corrected reads from MHAP were assembled using the Celera assembler (CA 8.3) (default parameters)(<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Myers <em>et al.</em> 2000</a>), which requires ∼60-70 gigabases of corrected sequence and consists of overlapper, unitigger, scaffolder and consensus steps to reconstruct genomes from corrected long reads.</p></div><div id="sec-12" class="subsection"><h3>Hybrid scaffolding</h3><p id="p-16">Single molecule high-resolution maps of the B6Eve genome were obtained using the Bionano Irys System (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Das <em>et al.</em> 2010</a>). Label positions captured in images and molecule map lengths were stored in CMAP format files (consensus map). The hybrid scaffold tool from Bionano genomics was used to further extend the scaffold size by combining the PacBio <em>de novo</em> assembly and genome map data of B6Eve. The hybrid scaffold pipeline created an alignment between the datasets and constructed super scaffolds excluding the conflicting alignments<sup>33</sup>.</p></div><div id="sec-13" class="subsection"><h3>Polishing and assembly evaluation</h3><p id="p-17">The hybrid scaffolded assembly was polished using Quiver (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2013</a>) to improve consensus accuracies in the range of Q60 and to reduce the high indel errors that are expected in the PacBio sequencing data (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zuo <em>et al.</em> 2012</a>) . pbalign (<a href="https://github.com/PacificBiosciences/pbalign">https://github.com/PacificBiosciences/pbalign</a>) was used to create a bam file from all h5 files of SMRT cells and Quiver trained on P6-C4 chemistries were used to obtain the consensus corrected assembly. This assembly was further improved by using Pilon (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Walker <em>et al.</em> 2014</a>) with default parameters, that corrects bases, fixes mis-assemblies and fills gaps provided a draft assembly and paired-end Illumina sequencing data. Nearly 32X of Illumina data from B6Eve was used as an input to Pilon to fix the bases in B6Eve assembly. The GATK variant calling pipeline (following best practices) was used to call variant using Illumina data on the B6Eve assembly to judge the improvement in overall quality at each step of polishing. Finally, the QUAST tool (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gurevich <em>et al.</em> 2013</a>) was used (–split-scaffolds) to compare the final assembly with GRCm38. The split-scaffolds option breaks the assembly and performs reconstruction of “contigs” which were used to build the scaffolds to compare the effectiveness of scaffolding.</p></div><div id="sec-14" class="subsection"><h3>RefSeq transcript alignments</h3><p id="p-18">Murine “known” RefSeq transcripts (those with NM and NR prefixes) were queried from NCBI Entrez on September 11, 2017 and aligned to the Pilon-corrected B6Eve assembly and GRCm38 full assembly (GCF_000001635.20). From these analyses, the counts of transcripts with low quality alignments, split alignments or no alignment to the GRCm38 primary assembly unit and B6Eve were determined, as were the counts of transcripts dropped for co-location, (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Taylor and Rowe 1984</a>). From the same set of RefSeq transcripts, we additionally identified alignments to GRCm38 and to the Pilon-corrected B6Eve containing frameshifting and non-frameshifting indels in CDS (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schneider <em>et al.</em> 2017</a>). The frameshift analysis of the pre-polished/corrected assembly used a set of known RefSeq transcripts queried on February 28, 2016.</p></div><div id="sec-15" class="subsection"><h3>LiftOver construction</h3><p id="p-19">LiftOver was performed between the Eve assembly and GRCm38 reference genome using the same species lift over construction procedure<sup>9</sup> outlined by University of California Santa Cruz Genome Bioinformatics Group. Same species lift over construction contained two steps a) BLAT alignments and b) chaining and netting to obtain the lift over file. Genome loci of the Eve assembly were further converted into GRCm38 coordinates using the LiftOver<sup>8</sup> tool from UCSC utilities.</p></div><div id="sec-16" class="subsection"><h3>Repeat content assessment</h3><p id="p-20">The repeat elements in the GRCm38.p6 (excluding ChrY and alternate loci scaffolds) and B6Eve assembly were determined by RepeatMasker (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Smit <em>et al.</em> 2013–2015</a>) trained on the mouse model by excluding RNA elements (-norna). A chi-square test was performed to identify the repeat classes that are enriched in one genome over the other.</p></div><div id="sec-17" class="subsection"><h3>Repeat analysis of unaligned B6Eve sequences</h3><p id="p-21">There were unaligned B6Eve sequences from three different alignment methods a) Cactus based alignment using UCSC Comparative Annotation Toolkit which was used for the B6Eve annotation, b) NCBI BLAST-based alignment of B6Eve to GRCm38, and c) QUAST (minimap2 aligner). A consensus set (common among all three set), was constructed using BEDTools (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Matera <em>et al.</em> 2008</a>). RepeatMasker was used to identify the repeat content in the common unaligned region. A chi-square test was used to test the differences in repeat content for each of the repeat classes of common unaligned sequence against GRCm38.</p></div><div id="sec-18" class="subsection"><h3>Resolving recurring variants</h3><p id="p-22">Recurring variants present in ≥ 75% of strains, detected in previous whole exome sequencing efforts <sup>10</sup> were extracted. Fixed (homozygous) variants from the mouse Collaborative Cross genome<sup>11</sup> project, as well as fixed variants from 24 C57BL/6J pedigrees descendent from Eve<sup>12</sup> were also obtained (G. Ananda and G. Churchill, unpublished data). We used LiftOver to remap the genomic coordinates, and a recurring variant was said to be resolved if the ALT allele of a recurring variant in exome data matches the REF allele in the B6Eve assembly. The analysis was restricted to only homozygous variant calls.</p></div><div id="sec-19" class="subsection"><h3>B6Eve annotation</h3><p id="p-23">B6Eve was annotated using the Comparative Annotation Toolkit (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fiddes <em>et al.</em> 2018</a>) (<a href="https://github.com/ComparativeGenomicsToolkit/Comparative-Annotation-Toolkit">https://github.com/ComparativeGenomicsToolkit/Comparative-Annotation-Toolkit</a> commit c7852b4). As input, CAT was given a progressive Cactus alignment generated with rat rn6 (GCA_000001895.4) and human GRCh38 as outgroups as well as the GENCODE VM11 annotation on mouse GRCm38. CAT was provided with extrinsic transcript information from RNA-seq as well as IsoSeq. For mouse GRCm38, the same RNA-seq used in the CAT publication was used. RNA-seq data were generated from whole brain of a female C57BL/6J Eve descendent and aligned to the B6Eve assembly. To guide AugustusPB in detecting novel isoforms, a total of 26,188 IsoSeq full length cDNA reads were aligned to the B6Eve assembly.</p></div><div id="sec-20" class="subsection"><h3>Novel isoform detection</h3><p id="p-24">To detect novel isoforms, homGeneMapping (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Konig <em>et al.</em> 2016</a>) was used to map GENCODE VM11 annotation coordinates onto B6Eve, and these splice junction coordinates compared to AugustusPB and AugustusCGP transcript predictions filtered for IsoSeq support. Transcripts with annotation support were filtered out. The remaining candidate novel isoforms then were checked to see if they overlapped a comparatively annotated locus and if they contained either a fully novel exon or a splice site shift based on bedtools (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Quinlan <em>et al.</em> 2010</a>) intersections.</p></div><div id="sec-21" class="subsection"><h3>SV detection &amp; gap filling</h3><div id="sec-22" class="subsection"><h4>PacBio read alignment:</h4><p id="p-25">Raw PacBio reads were aligned to GRCm38 using the long-read aligner NGMLR version 0.2.6. CoNvex Gap-cost alignments for Long Reads (NGMLR) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Beal <em>et al.</em> 2012a</a>) is a long-read aligner designed to align PacBio reads with the focus on identifying structural variations. Stringent alignment requirements were used for identifying SVs: -i 0.85 argument to disregard alignments with identity with less than 85% and -R 0.5 option to ignore alignments containing less than 50% of the read length.</p></div><div id="sec-23" class="subsection"><h4>Structural variant calling:</h4><p id="p-26">Sniffles (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Beal <em>et al.</em> 2012a</a>) (default parameters) was used to call SVs from the alignments produced by NGMLR. GATK was used to process Illumina WGS data from B6Eve. Best practices were used to generate a BAM file and SVS were called using Delly v0.7.7. SURVIVOR-1.0.3 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buac <em>et al.</em> 2008</a>) package was used to perform the integration of PacBio and Illumina calls.</p></div><div id="sec-24" class="subsection"><h4>Gap filling:</h4><p id="p-27">We extracted the coordinates of the gaps from the GRCm38 chromosomes and further extended this to include the 50 Kb flanking both sides of the gap. We aligned B6Eve scaffolds to these padded regions using minimap2. We filtered the candidate alignments according to the following criterion: a) Must be the reciprocal best hit, b) Total alignment length &gt;= 80KB, and c) Align to one unique location to the reference (extracted this information from assembly-assembly alignments). The retained alignments were further visualized in Integrated Genomic Viewer (IGV) to inspect insertion/deletion patterns around the gap region. To confirm and extract the gap spanning a B6Eve scaffold, we performed the reciprocal alignments, aligning the padded gap regions to B6Eve scaffolds, using minimap2. We filtered out candidate alignments not satisfying criteria mentioned above and visualized the retained alignments in IGV to inspect whether we observe the opposite of previously found insertion/deletion pattern. The sequence and locus of confirmed gap spanning B6Eve scaffolds were extracted and subjected to GRC internal curation.</p></div></div><div id="sec-25" class="subsection"><h3>Data availability</h3><p id="p-28">C57BL/6J mice and breeding generation information are available through the Jackson Laboratory. This Whole Genome Shotgun project has been deposited at DDBJ/ENA/GenBank under the accession LXEJ00000000. The version described in this paper is version LXEJ02000000. The raw PacBio, Illumina and Bionano data used were deposited at NCBI BioProject under accession PRJNA318985. The B6Eve assembly along with annotation and an assembly hub are available at ftp link (<a href="ftp://ftp.jax.org/b6eve">ftp://ftp.jax.org/b6eve</a>). Visualization of the assembly can be found at <a href="https://genome.ucsc.edu">https://genome.ucsc.edu</a> → MyData →Track Hubs → My Hubs with the following URL: <a href="ftp://ftp.jax.org/b6eve/assemblyhub/hub.txt">ftp://ftp.jax.org/b6eve/assemblyhub/hub.txt</a>. All supplementary figure, table, and file names and descriptions are listed in FileS1. A list of the B6Eve scaffold names mapped to GenBank accessions is available in FileS2. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7977044">https://doi.org/10.25387/g3.7977044</a>.</p></div></div><div class="section results-discussion" id="sec-26"><h2 class="">Results and Discussion</h2><p id="p-29">The Jackson Laboratory manages the rate of genetic drift through periodic replenishment of foundation breeding colonies from pedigreed, cryopreserved embryo stock (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wiles and Taft 2010</a>) that are three generations removed from a single brother-sister breeder pair, “Adam” and “Eve” (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This process introduces a controlled bottleneck that minimizes the accumulation of genetic change. These two individual mice capture an evolutionary snapshot in time at inbreeding generation F223 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Therefore, any C57BL/6J individual obtained from the production colonies at The Jackson Laboratory today is limited to a maximum of 24 inbreeding generations removed from the mice whose DNA was used to generate the C57BL/6J reference assembly, GRCm38 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Under the highly selective breeding paradigms employed for inbred laboratory strains, this genetic distance is sufficient for rapid fixation of 98.7% of variants, such that today’s C57BL/6J mice are by definition a sub-strain of the animals from which GRCm38 is derived (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Green 1981</a>). Therefore, we chose DNA isolated from one of these individuals as the material for our <em>de novo</em> assembly with the goal of providing a genome sequence from a single individual that is not more than eight generations removed from C57BL/6J mice sourced from The Jackson Laboratory today, and that might also be used to improve the current C57BL/6J reference assembly (GRCm38). We chose C57BL/6J Eve (B6Eve) to get balanced representation of the X chromosome and the autosomes. Future efforts are focused on a <em>de novo</em> assembly of Adam, where <em>de novo</em> assembly the Y chromosome will require more specialize approaches.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Origin of the inbred strain C57BL/6J. Inbred laboratory mouse strains are maintained by brother x sister mating. Filial (F) generations from which mice contributing to the reference assembly clone libraries and from which the B6Eve mouse were derived are shown. Cryopreserved embryo stock is represented by blue snowflakes at F226, 3 generations from Adam and Eve at F223. Generations subsequent to the cryopreservation event are F226p###, e.g., F226p230, which means embryos cryopreserved at F226 were recovered and there were an additional 4 generations of subsequent inbreeding." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1445155088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Origin of the inbred strain C57BL/6J. Inbred laboratory mouse strains are maintained by brother x sister mating. Filial (F) generations from which mice contributing to the reference assembly clone libraries and from which the B6Eve mouse were derived are shown. Cryopreserved embryo stock is represented by blue snowflakes at F226, 3 generations from Adam and Eve at F223. Generations subsequent to the cryopreservation event are F226p###, &lt;em&gt;e.g.&lt;/em&gt;, F226p230, which means embryos cryopreserved at F226 were recovered and there were an additional 4 generations of subsequent inbreeding.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1795/F1.medium.gif" width="392" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/6/1795/F1.medium.gif" width="392" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493328" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Origin of the inbred strain C57BL/6J. Inbred laboratory mouse strains are maintained by brother x sister mating. Filial (F) generations from which mice contributing to the reference assembly clone libraries and from which the B6Eve mouse were derived are shown. Cryopreserved embryo stock is represented by blue snowflakes at F226, 3 generations from Adam and Eve at F223. Generations subsequent to the cryopreservation event are F226p###, <em>e.g.</em>, F226p230, which means embryos cryopreserved at F226 were recovered and there were an additional 4 generations of subsequent inbreeding.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-27" class="subsection"><h3>Sequence assembly and evaluation</h3><p id="p-31">To generate data for our <em>de novo</em> assembly of the B6Eve genome, we used a range of technologies, including Pacific BioSciences (PacBio) long read technology at 66X whole genome coverage (Table S1), Illumina short read at 32X whole genome coverage, and Bionano Genomics (BNG) optical maps. The overall assembly procedure involved 1) correction of PacBio reads, 2) creation of contigs from PacBio, 3) extension of contigs to scaffolds using optical maps, 4) polishing of the assembly, and 5) further correction of the assembly using Illumina data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic overview of the de novo assembly procedure for B6Eve. Details are described in Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1445155088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic overview of the &lt;em&gt;de novo&lt;/em&gt; assembly procedure for B6Eve. Details are described in Methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1795/F2.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/6/1795/F2.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493387" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Schematic overview of the <em>de novo</em> assembly procedure for B6Eve. Details are described in Methods.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">To assess base-pair level improvements in assembly quality afforded by each of these steps, we mapped the Illumina reads of B6Eve and called variants at each step using the GATK HaplotypeCaller (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Depristo <em>et al.</em> 2011</a>) (Table S2). The scaffolded assembly yielded 1,664,599 variants, the majority of which were insertions (∼70%), followed by SNPs and small deletions. This pattern is reminiscent of the error profile generated from PacBio technology (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Zuo <em>et al.</em> 2012</a>). To improve base pair accuracy, we used Quiver software (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chin <em>et al.</em> 2013</a>) to polish the assembly and reduced the total number of variants to 505,782. We found that compared to the unpolished assembly, Quiver reduced the number of insertions to just 22%. Finally, we used B6Eve Illumina data itself to correct the polished assembly with Pilon (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Walker <em>et al.</em> 2014</a>), a tool that improves the quality of the draft assemblies using read alignment analysis. After Pilon correction, the number of variants was narrowed down to 310,205; of which 227,523 were considered high quality (PASS) by GATK HaplotypeCaller (File S3).</p><p id="p-34">Since these sequencing data sets were generated from the same individual female, we surmised that most of these high-quality variant calls were due to technical artifacts arising from errors in the B6Eve genome assembly, alignment errors, or errors in the Illumina data. Although we couldn’t exclude the possibility of true variant calls resulting from somatic variation since different tissues were used to source DNA. To explore these variants in more detail we plotted the distributions of alternate allele frequency, genotype called, and total depth relationship for these (File S4). We found that the vast majority of these variant calls were low frequency calls, which are more commonly due to technical artifacts or somatic mosaicism. To place these variants onto the mouse reference genome, we lifted them over to GRCm38 where roughly 95% (210,571 [SNP:152,486 and Indels:58,080]) could be placed onto a chromosome (File S5). We found that 62% percent of these variants mapped repeat elements making it likely that the variants are technical artifacts due to alignment issues. Many of these variants also mapped to a prominent cluster on Chr12 ∼17.5 MB containing 10,464 variants within a 1.5 Mb region including exonic, intronic, and intergenic sequence (File S6). The non-random distribution of these variant calls also supports the conclusion that they were technical artifacts due to misalignment of reads in a poorly assembled region in B6Eve. A closer look at the GRCm38 assembly revealed the presence of segmental duplications in this region, which may have caused a collapse in the B6 Eve assembly.</p><p id="p-35">We also called variants using Eve Illumina data aligned to GRCm38 and found significantly fewer variants (69,089 [SNP: 50,143 and Indel: 18,946]), about 10% of which were shared with the variants resulted from the mapping of Eve Illumina data to the Eve assembly (File S7). Except for a few minor clusters, including one on Chr1, these shared variants were uniformly distributed. The shared Chr1 cluster had 560 variants which were present in both variants call sets (Eve Illumina-Eve PacBio Assembly and Eve Illumina to GRCm38). Of these variants, there were two prominent clusters on Chr1: 85062206-85279614, Chr1: 88212297-88310615 with 100 and 110 variants, respectively. The first region includes the <em>Csprs</em> gene and second includes a known GRCm38 assembly issue which was also unresolved in the B6Eve assembly. Similar to the Chr12 region above, these regions on Chr1 are flanked by segmental duplications in GRCm38.</p><p id="p-36">We used an automatic assembly quality evaluation tool, QUAST (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gurevich <em>et al.</em> 2013</a>), to assess the overall quality of the Pilon corrected assembly. We found that during assembly-assembly alignment, 96.8% of the B6Eve assembly aligned to 97% of the GRCm38 (excluding ChrY and alternate sequences) reference genome chromosomal sequences and 54% of unplaced &amp; unlocalized sequences (Figure S1). Only 166 and 2,456 B6Eve components comprising of 3.7 Mb and 7.2 Mb of sequences remained wholly or partially unaligned to the reference genome, respectively. We also found that the K-mer based completeness of B6Eve was very high at 97.4%, suggesting high coverage and per-base quality. The detailed QUAST report for complete and broken-down (breaks the assembly by continuous fragments of N’s of length ≥ 10) versions of the assembly is found in File S8. Taken together, our resulting PacBio-only <em>de novo</em> B6Eve genome assembly was 2.53 Gb consisting of 14,551 contigs (longest contig = 4,574,471 and 2.3% of total contigs exceeding 1Mb) with an N50 size of 401,294 bp. Our complete PacBio-Bionano hybrid assembly yielded an N50 of 1,290,032 bp with a total assembly size of 2.79 Gb, which was a 3X improvement (in N50) over the PacBio-only assembly (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493367/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493367/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Number of sequences, N50 size and assembly length for Bionano optical map, PacBio <em>de novo</em> assembly and scaffolded assemblies</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-28" class="subsection"><h3>Gene Content Analysis</h3><p id="p-37">We also evaluated the gene content of the B6Eve assembly as another measure of assembly quality. Akin to previous analyses (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Taylor and Rowe 1984</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Schneider <em>et al.</em> 2017</a>) we aligned 36,009 RefSeq transcripts to the GRCm38 primary assembly (also excluding unplaced sequences and alternate loci scaffolds from other mouse strains that are a part of the full assembly) and to the “Piloned” B6Eve assembly versions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and Table S3). We observed that B6Eve provides a comparable representation for total gene content, as compared to GRCm38, with only 19 non-chromosome and ChrY associated sequences having no alignment. Consistent with the more fragmented nature of the B6Eve assembly, however, a greater number of aligned RefSeq transcripts exhibited partial alignments or alignments split over multiple scaffolds than in GRCm38. We examined the co-placement of transcripts representing different genes as a proxy for measuring the collapse of segmental duplications. Although B6Eve showed a greater number of co-placed transcripts than GRCm38, these numbers were consistent with those seen in other high-quality long-read derived WGS assemblies, demonstrating the utility of this mouse assembly. To gauge the impact of the Illumina-read correction step on the quality of protein representation in the B6Eve assembly, we looked at the incidence of frameshifting indels in aligned RefSeq transcripts prior to and after this step(<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Florea <em>et al.</em> 2011</a>) (Table S3). Although the Pilon corrected assembly still exhibited more frameshifts than GRCm38, we found that this step resulted in a substantial improvement in functional representation (protein coding sequence).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493290/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493290/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><strong>RefSeq Transcripts Alignment Table From NCBI</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h3>Reference Assembly Gap Filling</h3><p id="p-40">The GRCm38 chromosome assemblies contain 440 gaps (excluding centromere, short arm, and telomere gaps). We assessed whether sequences in the B6Eve assembly could resolve these gaps. Based on our gap-filling methodology (see Methods), the B6Eve assembly spanned 23 gaps in the GRCm38 chromosomes (Table S4 and Figure S2). In several instances, we observed discrepancies between the gap length reported in GRCm38 and the amount of sequence provided by the B6Eve assembly. For example, the B6Eve assembly spanned a 1,760 bp intra-scaffold gap located at Chr2:172,624,657-172,626,416 bp in GRCm38, with 1,620 bp (a 140 bp relative deletion) (Figure S2a). In other cases, B6Eve spanning sequences were longer than assembly gaps. For example, a B6Eve assembly scaffold spanned the 100 bp intra-scaffold gap at Chr1:183,334,907-183,335,006 bp in GRCm38, with 595 bp sequence (Figure S2b). These discrepancies were not unexpected, however, as the methods used to estimate reference assembly gap sizes do not always offer base-pair level resolution, and also because the GRC assigns default gap lengths when no sizing estimates are available (<a href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/TPF_Specification_v1.8_20131106.docx">https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/TPF_Specification_v1.8_20131106.docx</a>). Consistent with prior reports that remaining reference assembly gaps are in complex genomic regions (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Church <em>et al.</em> 2011</a>), we observed that our gap spanning sequences had a repeat content of 45.9% (<em>vs.</em> 42.5% of GRCm38 total sequence) repeats, with simple repeats accounting for 18.1% (vs 2.6% in GRCm38 total sequence).</p></div><div id="sec-30" class="subsection"><h3>Variant analysis</h3><p id="p-41">Previously, we reported whole-exome sequences from of a collection of nearly 200 unique strains of spontaneous mutant mice maintained at The Jackson Laboratory (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fairfield <em>et al.</em> 2015</a>). In our analysis of these exomes, we found that there were 855 coding variants (SNPs) common across 75% or more of the samples, which we attributed to errors with the reference genome itself due to their significant inclusion within the component-mapped boundaries of GRC incident features. We investigated the subset of these exome variants (n = 126) that were homozygous across all strains (100% allele frequency). We found that 10 (7.9%) matched the B6Eve assembly allele rather than the GRCm38 allele, supporting the assertion that high frequency alleles are putative indicators of reference assembly error (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Table S5).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ideogram of GRCm38 assembly annotated to highlight resolved gaps (vs. current reference), structural variants, and fixed variation using B6Eve data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1445155088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ideogram of GRCm38 assembly annotated to highlight resolved gaps (&lt;em&gt;vs.&lt;/em&gt; current reference), structural variants, and fixed variation using B6Eve data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1795/F3.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/6/1795/F3.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493270" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">Ideogram of GRCm38 assembly annotated to highlight resolved gaps (<em>vs.</em> current reference), structural variants, and fixed variation using B6Eve data.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">To extend this analysis to the whole genome, we performed a similar analysis using variant calls from sixty-nine multi-parent, recombinant inbred strains (Collaborative Cross (CC) strains, a panel derived from eight founder laboratory strains) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>). We found 14,757 variants (SNPs) shared across all strains, using C57BL6/J as a reference genome. Out of 14,757 variants, 2,407 are homozygous across all strains. Consistent with the results of the exome variant analysis, 307 of these variants (12.8%) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Table S5) matched the B6Eve assembly allele rather than GRCm38.</p><p id="p-44">Finally, we analyzed variant calls from whole genome short read sequencing of 24 recent descendants of B6Eve, representing multiple inbred lineages. We found 3,203 homozygous variants (SNPs) common across these samples; of these, 2,194 (68.5%) met minimum alignment criteria for remapping to B6Eve. Of these 2,194, we found 393 cases (12.6% of the total variation and 17.9% of net variation) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, Table S5) where the reported alternate alleles matched the B6Eve assembly.</p><p id="p-45">Taken together, our analyses identified 503 single nucleotide positions in GRCm38 (excluding variants common in three datasets) that are not representative of today’s C57BL/6J mice. Two of these are nonsynonymous SNPs in <em>Akap9 and Sfi1</em>. <em>Akap9</em> (A kinase anchoring protein 9) is a protein that is responsible for cytoskeletal organization and is required for formation and maintenance of the blood-testis barrier, and male fertility (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schimenti <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Venkatesh <em>et al.</em> 2016</a>). Using allele specific PCR, we confirmed the presence of the alternate <em>Akap9</em> allele in B6Eve and in randomly selected descendants of Eve, and not in an ancestor of B6Eve. Therefore, <em>Akap9</em> represents a variant that arose in and is now stably maintained in C57BL/6J mice in the years since the sequencing GRCm38. <em>Sfi1</em> is predicted to be a spindle assembly associated protein on the basis of homology with a yeast cytoskeletal protein of known function, however no phenotypic alleles have been reported in mice (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Salisbury 2004</a>). When we attempted to validate this variant, however, we discovered that the alternate allele is indeed represented in the mouse reference genome, though it aligned to an unplaced scaffold (JH584304.1: 14,038-14,339). Flanking sequence variation between this scaffold and Chr11 allowed us to design allele specific primers, with which we confirmed that both alleles are present in DNA samples from B6Eve and from randomly selected descendants of B6Eve, supporting the idea that the unplaced scaffold indeed represents C57BL/6J sequence. Previously published SV data for C57BL/6J showed that the mouse genome potentially harbors 20-30 copies of this gene (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Quinlan <em>et al.</em> 2010</a>). Therefore, the recurrent “variation” observed in this gene is likely not allelic, but due to mis-mapping of reads from paralogous gene copies to the <em>Sfi</em> locus that is currently represented on GRCm38 Chromosome 11. Paralogous gene variation may be a previously underappreciated source of variation, since we observed a relative enrichment of variants within certain genes (<em>e.g.</em>, <em>Tulp4</em>, Table S5). Previous studies have shown that GRCm38 is missing paralogous copies of many genes (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Church <em>et al.</em> 2009</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Church <em>et al.</em> 2011</a>), some of which may be represented on unplaced scaffolds as we found for <em>Sfi1</em>.</p></div><div id="sec-31" class="subsection"><h3>Structural Variation</h3><p id="p-46">We aligned raw B6Eve PacBio reads to GRCm38 using NGMLR (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Beal <em>et al.</em> 2012a</a>) and called structural variants (SVs) with Sniffles (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Beal <em>et al.</em> 2012a</a>). We also aligned Illumina WGS data from B6Eve and called SVs with Delly (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beal <em>et al.</em> 2012b</a>) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The median size of detected duplication, deletion and inversion events from Delly were 901, 2,610 and 12,362 bp, respectively. Similarly, from PacBio data, the median size of duplication, deletion, inversion and insertion events were 432, 77, 1352, and 92 bp, respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493272/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493272/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><strong>Counts of various structural variation classes detected in the comparison of B6Eve Sequences to GRCm38 using PacBio and Illumina data</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-47">We found 12 deletion, 43 duplication and 4 inversion calls that were common in both Illumina and PacBio data (Table S6). Of these common SVs, 8 deletions, 30 duplications, and 4 inversions overlapped genes (Table S6), though mostly within noncoding intronic regions. We used DGVa to further investigate the SVs overlapping genes. Each of these were associated with multiple (21-124) DGVa entries representing germ line SV across genetically diverse inbred strains from multiple strain surveys of SV (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cutler <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Graubert <em>et al.</em> 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Keane <em>et al.</em> 2011</a>). Some of these regions contain genes that have been previously been shown to be subject to positive selection of copy number variants in inbred laboratory mouse strains. Our data showed that even within a strain, we detected SV in these regions, which suggests that these regions are by their very nature susceptible to rearrangements, <em>i.e.</em>, through suppressed recombination. Alternatively, recurrent SV calls could reflect either private SVs in the reference assembly, or mis-assembly of these regions.</p></div><div id="sec-32" class="subsection"><h3>Repeat analysis</h3><p id="p-48">Repetitive sequences present challenges to assembly, as highly identical repeat sequences from different genomic regions are often incorrectly assembled together. This is a particular concern for data generated from short-read technologies, which are too short to span longer repeats. One advantage of long read sequencing reads are their ability to span a greater range of genomic repeats into unique sequence, enabling resolution of repetitive regions that cannot be resolved in unlinked short read assemblies. We used RepeatMasker (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Smit <em>et al.</em> 2013–2015</a>) to compare repetitive sequence representation between B6Eve, GRCm38, and two Illumina WGS assemblies (GCA_000185125.1, GCA_000185105.2) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Locke <em>et al.</em> 2015</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Milholland <em>et al.</em> 2017</a>). This analysis revealed that the B6Eve assembly consists of 42.0% repetitive sequence (1,065,403,997 out of 2,537,631,632 bp, excluding N’s). This fraction is very similar to GRCm38 (excluding the Y chromosome and alternate loci sequences), 42.5% of which is repetitive (1,088,395,156 out of 2,559,396,830 bp, excluding N’s and X’s). Consistent with the challenges of assembling repetitive sequence with shorter reads, the Illumina based assembly GCA_000185125.1 had 32.5% (833,318,654 out of 2,257,461,872 bp excluding N/X-runs) and GCA_000185105.2 had 33.9% (849,891,926 out of 2,279,058,378 bp excluding N/X-runs) annotated as repetitive sequence (Table S7).</p><p id="p-49">We also used RepeatMasker analysis to assess repeat content in scaffolds from our B6Eve assembly that failed to align to GRCm38, as we surmised these scaffolds might contain repetitive sequences that could not be resolved with genomic clones. To do this we focused on sequences which are unaligned by all of three of the following methods a) Cactus based alignment using UCSC Comparative Annotation Toolkit b) NCBI assembly-assembly alignments and c) QUAST evaluation. The common unaligned sequences (total 6.12 MB) (File S9) between CACTUS, NCBI, and QUAST had significant enrichment for repeats relative to aligned sequences. The repeat content accounted to 77.6% (4,754,330 out of 6,128,602 bp) with the microsatellite repeat class showed significantly enrichment when compared with GRCm38 (59.9% <em>vs.</em> 0.1%, chi-square test: X<sup>2</sup> = 80,332,000, p-value &lt; 2.2e-1 (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). While more work is needed to determine the underlying cause of failed alignment, the enrichment of microsatellite repeats in these scaffolds is compelling. Microsatellite repeats are prone to slippage during DNA replication, and as a result their copy number is highly polymorphic in eukaryotic genomes; a phenomenon known as microsatellite instability (MSI). In inbred laboratory mouse strains and in the human population, mutations that change copy number occur at rates that are up to 10,000 times higher than single nucleotide mutation rates (1-3 × 10<sup>−4</sup> (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Beal <em>et al.</em> 2012a</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Beal <em>et al.</em> 2012b</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Zuo <em>et al.</em> 2012</a>) per repeat per generation for microsatellite sequences <em>vs.</em> 2-4 × 10<sup>−9</sup> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Milholland <em>et al.</em> 2017</a>) per nucleotide per generation for SNV in C57BL/6J). Similarly, in the human population, CNV are estimated to occur at rates that are 100-10,000 times higher than the point mutation rate (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hu <em>et al.</em> 2017</a>). Taken together, CNV are a major source of intrastrain variation and divergence from isogenicity (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Watkins-Chow and Pavan 2008</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Locke <em>et al.</em> 2015</a>). Therefore, failed alignment of these microsatellite containing scaffolds could be due to repeat polymorphisms that have arisen over the intervening years in C57BL/6J. Alternatively, failed alignment could be due to assembly issues in either genome.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493409/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493409/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title"><strong>Comparison of various repeat class in common unaligned sequences with GRCm38</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-33" class="subsection"><h3>Gene prediction</h3><p id="p-50">Long read sequencing of cDNAs (IsoSeq) provides full-length transcript sequences and highly accurate representations of splice junctions and isoforms. To determine if long read sequencing data of B6Eve cDNAs could support more accurate gene prediction for the mouse reference genome, we generated IsoSeq data from RNA extracted from archived B6Eve brain. We used the Comparative Annotation Toolkit (CAT)(<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Fiddes <em>et al.</em> 2018</a>) to identify 107,192 transcripts (82,187 protein-coding) representing 41,669 gene loci (20,182 protein-coding). 2,426 transcript predictions had splice junctions that were novel relative to GENCODE VM11, and we found additional support for these junctions in RNA-Seq data generated from the brain of a female C57BL/6J descendent of Eve. Analysis of the transcript predictions produced by AugustusPB and AugustusCGP revealed 206 exons with splice site shifts relative to GRCm38, nine putatively novel exons and ten putatively novel loci (Table S8). Three of the novel exons detected in the IsoSeq data reveal deletions in GRCm38: (1) 640 bp in <em>Mia3</em> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), (2) <em>Traf5</em>, and <em>Slc26a6</em> (Figure S3 and Figure S4). In support of these data, there are GRC incident reports describing deletions at each of these loci in GRCm38. Contiguity analysis in the B6Eve assembly showed that 616 genes mapped across two or more scaffolds and four genes had projections split on the same scaffold. A total of 258 protein-coding genes exhibited signs of gene family collapse, with 156 pairs of genes being resolved to the same locus.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The Mia3 locus from the perspective of both the B6Eve assembly (top) and the GRCm38 mouse reference (bottom). CAT annotation of B6Eve identified three isoforms with an IsoSeq supported exon not found in the reference. The cactus alignments (blue bars) show that there are 43 bp of reference sequence that does not align to B6Eve, and that there are 638 bp of B6Eve not seen in the reference. These 638 bp contain the extra exon. This result is confirmed in the B6Eve IsoSeq GRCm38 alignment, which shows an insertion (white blocks between gray exon alignments)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1445155088" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;Mia3&lt;/em&gt; locus from the perspective of both the B6Eve assembly (top) and the GRCm38 mouse reference (bottom). CAT annotation of B6Eve identified three isoforms with an IsoSeq supported exon not found in the reference. The cactus alignments (blue bars) show that there are 43 bp of reference sequence that does not align to B6Eve, and that there are 638 bp of B6Eve not seen in the reference. These 638 bp contain the extra exon. This result is confirmed in the B6Eve IsoSeq GRCm38 alignment, which shows an insertion (white blocks between gray exon alignments).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/6/1795/F4.medium.gif" width="440" height="285"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/6/1795/F4.medium.gif" width="440" height="285"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/6/1795/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493408" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-51" class="first-child">The <em>Mia3</em> locus from the perspective of both the B6Eve assembly (top) and the GRCm38 mouse reference (bottom). CAT annotation of B6Eve identified three isoforms with an IsoSeq supported exon not found in the reference. The cactus alignments (blue bars) show that there are 43 bp of reference sequence that does not align to B6Eve, and that there are 638 bp of B6Eve not seen in the reference. These 638 bp contain the extra exon. This result is confirmed in the B6Eve IsoSeq GRCm38 alignment, which shows an insertion (white blocks between gray exon alignments).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-34" class="subsection"><h3>Conclusions</h3><p id="p-52">The value of isogenic mouse strain backgrounds in biomedical research was recognized by geneticists in the early 20<sup>th</sup> century leading to the creation and description of the over 450 unique inbred mouse strains to date (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Silver 1995</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beck <em>et al.</em> 2000</a>). Twenty generations of sibling intercrosses are required for the generation of a new inbred strain; a breeding method that creates genomes in which more than 98% of loci are homozygous. Therefore, individuals within a generation, within the same vivarium are essentially, genetically identical. The remarkable genetic architecture of inbred laboratory mouse strains is shaped by the frequent bottlenecks required for the on-going maintenance of these strains. This accelerates genetic drift and is a major source of the often unexpected, genetic variation that can be observed across generations and/or between vivaria. For example, a reference-based alignment of the inbred laboratory strain C57BL/6J yields approximately 900 raw variant calls (SNPs/Indels), despite it being the same inbred strain as the mouse reference genome(<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Fairfield <em>et al.</em> 2015</a>). While a subset of these variant calls are the expected result of genetic drift, we previously found that a significant percentage of these variants are located in regions of the reference genome where there are reported assembly issues(<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Fairfield <em>et al.</em> 2015</a>) and regions that contain missing paralogs, which are a known source of false positive variation due to mis-mapping of reads (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Church <em>et al.</em> 2011</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hu <em>et al.</em> 2017</a>). A major goal of this <em>de novo</em> assembly was to generate long read sequencing data that could potentially be used resolve these regions and to provide an updated representation of the variation that is present in the most recent inbreeding generations of C57BL/6J.</p><p id="p-53">The generation of the B6Eve assembly also provides important insights into the relative merits and limitations of different sequencing and assembly approaches. As we demonstrate, long-read WGS assemblies can resolve regions in which there is no coverage in clone-based assemblies. As some genomic regions are recalcitrant to cloning in vectors, alternate technologies like long-read WGS are critical to completing a gapless assembly. However, even long-read WGS assemblies of mammalian genomes are prone to collapse of highly repetitive or segmentally duplicated regions where these lengths are greater than the average read length. The impact of these collapses manifest as false positive variant calls in regions where paralogous variants were miscalled. As sequencing technologies improve and read lengths get even longer, we may reach a point at which the need for genomic clones in assembly is obviated. However, to generate very high-quality reference assemblies, it will also be important to further reduce the error rates associated with long reads, even beyond the corrections achieved with short reads. Currently, generation of the highest quality assemblies requires a mix of techniques.</p><p id="p-54">Using variant data from our B6Eve assembly, as well as data from several other large sequencing efforts (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>), we provide a “truth” call for over 500 high quality, recurring variants (SNP/Indels) that can be used to update the mouse reference genome (GRCm38.p6). We also found evidence for over 40 structural variants (inversions, deletions, and duplications) involving protein-coding genes in our B6Eve assembly compared to the reference genome. The majority of these SV calls were found in DGVa across a variety of strains, suggesting that they are likely recurrent SV calls that, similar to recurrent variants, are due to mis-assembly of paralogous sequences or reference specific SVs. Further, our data fill 23 gaps of varying length in the mouse reference genome which will be used to inform the upcoming release of GRCm39.</p><p id="p-55">Our IsoSeq data provided improved/more accurate gene models with previously unrecognized splice junctions for over 2,000 genes. This is likely an underrepresentation since our analysis is limited to only those genes expressed in brain. We also found evidence for novel exons, as well as evidence for novel loci (expressed regions that lack gene annotation). This demonstrates that even in a well-curated reference genome assembly, gene annotation remains subject to change as new technologies provide improved representation of transcribed sequences and access to more highly specialized cell types.</p><p id="p-56">Overall, our <em>de novo</em> assembly of Eve is not as polished as GRCm38, a clone-based assembly that benefits from more than 20 years of on-going curation and annotation, but it does provide key enhancements and a full picture of the types and sources of technical error in re-sequencing. Whole genome sequencing data are now available for hundreds of standardized laboratory inbred mouse strains (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Keane <em>et al.</em> 2011</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lilue <em>et al.</em> 2018</a>). These data reveal the remarkable architecture of inbred genomes, and provide a stark reminder that isogenic mouse strains are subject to genetic drift; a feature that directly conflicts with the idea of a ‘reagent-grade’ laboratory mouse. Careful breeding practices, cryoarchiving, and routine sequencing are key steps toward maximizing reproducibility of studies that rely on these living reagents. Ultimately, <em>de novo</em> assembly captures the full spectrum of genetic variation resident in inbred strains, some of which harbor significantly more variation than distantly related human populations. Recently, genome graphs have been used to represent “population reference genomes” as a means to improve read mapping and to minimize false positive variant calls (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Rosen <em>et al.</em> 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garrison <em>et al.</em> 2018</a>). As applied to mouse genomes, this approach would ideally provide a framework for future representation of the laboratory mouse reference genome as a graph of many inbred strains upon which emergent variation can be more accurately discovered and used to guide experimental research involving laboratory mouse strains.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">LGR, VKS, AS, NR, and OZ were supported in part by NIH R24 OD02135 awarded to LGR and The Jackson Laboratory. The work of VAS and F. T-N was supported by the intramural research program of the National Library of Medicine, National Institutes of Health. We are grateful to the services provided by The Jackson Laboratory Genome Technologies and Computational Sciences Core, which are supported by a grant to The Jackson Laboratory Cancer Center, NCI P30 CA034196.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-58">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7977044">https://doi.org/10.25387/g3.7977044</a>.</p></li><li class="fn" id="fn-4"><p id="p-59"><em>Communicating editor: F. Pardo-Manuel de Villena</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-02-08"><span class="received-label">Received </span>February 8, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-04-16"><span class="accepted-label">Accepted </span>April 16, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Sarsani <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.1" data-doi="10.1002/em.21726"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glenn</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lance</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Characterization of unstable microsatellites in mice: no evidence for germline mutation induction following gamma-radiation exposure.</span> <abbr class="cit-jnl-abbrev">Environ. Mol. Mutagen.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/em.21726</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Band%2Bmolecular%2Bmutagenesis%26rft.stitle%253DEnviron%2BMol%2BMutagen%26rft.aulast%253DBeal%26rft.auinit1%253DM.%2BA.%26rft.volume%253D53%26rft.issue%253D8%26rft.spage%253D599%26rft.epage%253D607%26rft.atitle%253DCharacterization%2Bof%2Bunstable%2Bmicrosatellites%2Bin%2Bmice%253A%2BNo%2Bevidence%2Bfor%2Bgermline%2Bmutation%2Binduction%2Bfollowing%2Bgamma-radiation%2Bexposure.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fem.21726%26rft_id%253Dinfo%253Apmid%252F22930577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/em.21726&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22930577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.2" data-doi="10.1016/j.mrrev.2011.11.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glenn</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Whole genome sequencing for quantifying germline mutation frequency in humans and model species: cautious optimism.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">750</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mrrev.2011.11.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DBeal%26rft.auinit1%253DM.%2BA.%26rft.volume%253D750%26rft.issue%253D2%26rft.spage%253D96%26rft.epage%253D106%26rft.atitle%253DWhole%2Bgenome%2Bsequencing%2Bfor%2Bquantifying%2Bgermline%2Bmutation%2Bfrequency%2Bin%2Bhumans%2Band%2Bmodel%2Bspecies%253A%2Bcautious%2Boptimism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mrrev.2011.11.002%26rft_id%253Dinfo%253Apmid%252F22178956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mrrev.2011.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22178956&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304184300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.3" data-doi="10.1038/71641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafezparast</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lennon-Pierce</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genealogies of mouse inbred strains.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/71641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBeck%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D25%26rft.atitle%253DGenealogies%2Bof%2Bmouse%2Binbred%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F71641%26rft_id%253Dinfo%253Apmid%252F10615122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/71641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10615122&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084609200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.4" data-doi="10.1038/nbt.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span>. Erratum: 33: 1109.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3238%26rft_id%253Dinfo%253Apmid%252F26006009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26006009&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.5" data-doi="10.1371/journal.pgen.1000177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buac</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watkins-Chow</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loftus</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Incao</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A Sox10 expression screen identifies an amino acid essential for Erbb3 function.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1000177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000177</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBuac%26rft.auinit1%253DK.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De1000177%26rft.epage%253De1000177%26rft.atitle%253DA%2BSox10%2Bexpression%2Bscreen%2Bidentifies%2Ban%2Bamino%2Bacid%2Bessential%2Bfor%2BErbb3%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000177%26rft_id%253Dinfo%253Apmid%252F18773073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18773073&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.6" data-doi="10.1038/nmeth.2474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klammer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2474</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2474%26rft_id%253Dinfo%253Apmid%252F23644548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644548&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319668700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.7" data-doi="10.1371/journal.pbio.1000112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Lineage-specific biology revealed by a finished genome assembly of the mouse.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1000112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DChurch%26rft.auinit1%253DD.%2BM.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1000112%26rft.epage%253De1000112%26rft.atitle%253DLineage-specific%2Bbiology%2Brevealed%2Bby%2Ba%2Bfinished%2Bgenome%2Bassembly%2Bof%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000112%26rft_id%253Dinfo%253Apmid%252F19468303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19468303&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.8" data-doi="10.1371/journal.pbio.1001091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Modernizing reference genome assemblies.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1001091</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001091</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DChurch%26rft.auinit1%253DD.%2BM.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253De1001091%26rft.epage%253De1001091%26rft.atitle%253DModernizing%2Breference%2Bgenome%2Bassemblies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001091%26rft_id%253Dinfo%253Apmid%252F21750661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21750661&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.9" data-doi="10.1101/gr.6754607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baribault</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassner</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Significant gene content variation characterizes the genomes of inbred mouse strains.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1743</span>–<span class="cit-lpage">1754</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.6754607</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6754607%26rft_id%253Dinfo%253Apmid%252F17989247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xNzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.10" data-doi="10.1093/nar/gkq673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Austin</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akana</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Single molecule linear analysis of DNA in nano-channel labeled with sequence specific fluorescent probes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-elocation-id">e177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq673%26rft_id%253Dinfo%253Apmid%252F20699272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20699272&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.11" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.12" data-doi="10.1038/nature19356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flenniken</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teboul</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">High-throughput discovery of novel developmental phenotypes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">537</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">514</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature19356</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D537%26rft.spage%253D508%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19356%26rft_id%253Dinfo%253Apmid%252F27626380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27626380&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.13" data-doi="10.1101/gr.186882.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fairfield</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ananda</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exome sequencing reveals pathogenic mutations in 91 strains of mice with Mendelian disorders.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">948</span>–<span class="cit-lpage">957</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.186882.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.186882.114%26rft_id%253Dinfo%253Apmid%252F25917818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzcvOTQ4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.14" data-doi="10.1101/gr.233460.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fiddes</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diekhans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nachtweide</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronenberg</span>  <span class="cit-name-given-names">Z. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Comparative Annotation Toolkit (CAT)-simultaneous clade and personal genome annotation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1029</span>–<span class="cit-lpage">1038</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.233460.117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.233460.117%26rft_id%253Dinfo%253Apmid%252F29884752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI4LzcvMTAyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNi8xNzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.15" data-doi="10.1371/journal.pone.0021400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Florea</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souvorov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalbfleisch</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome assembly has a major impact on gene content: a comparison of annotation in two Bos taurus assemblies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e21400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0021400</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFlorea%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De21400%26rft.epage%253De21400%26rft.atitle%253DGenome%2Bassembly%2Bhas%2Ba%2Bmajor%2Bimpact%2Bon%2Bgene%2Bcontent%253A%2Ba%2Bcomparison%2Bof%2Bannotation%2Bin%2Btwo%2BBos%2Btaurus%2Bassemblies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0021400%26rft_id%253Dinfo%253Apmid%252F21731731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0021400&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21731731&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.16" data-doi="10.1038/nbt.4227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siren</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Variation graph toolkit improves read mapping by representing genetic variation in the reference.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">879</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.4227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D36%26rft.spage%253D875%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4227%26rft_id%253Dinfo%253Apmid%252F30125266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30125266&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.17" data-doi="10.1371/journal.pgen.0030003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graubert</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cahan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selzer</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A high-resolution map of segmental DNA copy number variation in the mouse genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0030003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGraubert%26rft.auinit1%253DT.%2BA.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253De3%26rft.epage%253De3%26rft.atitle%253DA%2Bhigh-resolution%2Bmap%2Bof%2Bsegmental%2BDNA%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030003%26rft_id%253Dinfo%253Apmid%252F17206864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17206864&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-other" id="cit-9.6.1795.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> Genetics and Probability in Animal Breeding Experiments.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-349-04904-2</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.19" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.20" data-doi="10.1038/nprot.2009.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharjee</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamin Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Hybrid selection of discrete genomic intervals on custom-designed microarrays for massively parallel sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2009.68</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHodges%26rft.auinit1%253DE.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D960%26rft.epage%253D974%26rft.atitle%253DHybrid%2Bselection%2Bof%2Bdiscrete%2Bgenomic%2Bintervals%2Bon%2Bcustom-designed%2Bmicroarrays%2Bfor%2Bmassively%2Bparallel%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.68%26rft_id%253Dinfo%253Apmid%252F19478811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19478811&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266546200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.21" data-doi="10.1038/srep43178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ennos</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High mutation rates explain low population genetic divergence at copy-number-variable loci in Homo sapiens.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">43178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep43178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D43178%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep43178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep43178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.22" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10413</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.23" data-doi="10.1093/bioinformatics/btw494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konig</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romoth</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerischer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Simultaneous gene finding in multiple genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">3388</span>–<span class="cit-lpage">3395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw494%26rft_id%253Dinfo%253Apmid%252F27466621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27466621&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.24" data-doi="10.1038/s41588-018-0223-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lilue</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doran</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiddes</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrudan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Sixteen diverse laboratory mouse reference genomes define strain-specific haplotypes and novel functional loci.</span> <abbr class="cit-jnl-abbrev">Nat. Genet</abbr>. <span class="cit-vol">50</span>: <span class="cit-fpage">1574</span>–<span class="cit-lpage">1583</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-018-0223-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0223-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0223-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.25" data-doi="10.1186/s12864-015-1713-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Locke</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milojevic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eitutis</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wishart</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic copy number variation in Mus musculus.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">497</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1713-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1713-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1713-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.26" data-doi="10.1126/science.296.5570.1005b"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genome sequencing. Public group completes draft of the mouse.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">1005</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.296.5570.1005b</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMarshall%26rft.auinit1%253DE.%26rft.volume%253D296%26rft.issue%253D5570%26rft.spage%253D1005%26rft.epage%253D1005%26rft.atitle%253DGenome%2Bsequencing.%2BPublic%2Bgroup%2Bcompletes%2Bdraft%2Bof%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.296.5570.1005b%26rft_id%253Dinfo%253Apmid%252F12004098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.296.5570.1005b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12004098&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.27" data-doi="10.1093/hmg/ddn110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matera</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watkins-Chow</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loftus</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Incao</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A sensitized mutagenesis screen identifies Gli3 as a modifier of Sox10 neurocristopathy.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2118</span>–<span class="cit-lpage">2131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/hmg/ddn110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddn110%26rft_id%253Dinfo%253Apmid%252F18397875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddn110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397875&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.28" data-doi="10.1038/ncomms15183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milholland</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Differences between germline and somatic mutation rates in humans and mice.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">15183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms15183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D8%26rft.spage%253D15183%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15183%26rft_id%253Dinfo%253Apmid%252F28485371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28485371&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.29" data-doi="10.1038/nature01262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Mouse Genome Sequencing Consortium, </span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindblad-Toh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Initial sequencing and comparative analysis of the mouse genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">562</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature01262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaterston%26rft.auinit1%253DR.%2BH.%26rft.volume%253D420%26rft.issue%253D6915%26rft.spage%253D520%26rft.epage%253D562%26rft.atitle%253DInitial%2Bsequencing%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01262%26rft_id%253Dinfo%253Apmid%252F12466850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12466850&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179611600053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.30" data-doi="10.1126/science.287.5461.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dew</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasulo</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A whole-genome assembly of Drosophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2196</span>–<span class="cit-lpage">2204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.287.5461.2196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DE.%2BW.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2196%26rft.epage%253D2204%26rft.atitle%253DA%2BWhole-Genome%2BAssembly%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2196%26rft_id%253Dinfo%253Apmid%252F10731133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.31" data-doi="10.1101/gr.102970.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide mapping and assembly of structural variant breakpoints in the mouse genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.102970.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102970.109%26rft_id%253Dinfo%253Apmid%252F20308636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzUvNjIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.32" data-doi="10.1093/bioinformatics/btx236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paten</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Modelling haplotypes with respect to reference cohort variation graphs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">i118</span>–<span class="cit-lpage">i123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253Di118%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.33" data-doi="10.1016/j.cub.2003.12.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salisbury</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Centrosomes: Sfi1p and centrin unravel a structural riddle.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R27</span>–<span class="cit-lpage">R29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2003.12.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSalisbury%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253DR27%26rft.epage%253DR29%26rft.atitle%253DCentrosomes%253A%2BSfi1p%2Band%2Bcentrin%2Bunravel%2Ba%2Bstructural%2Briddle.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.12.019%26rft_id%253Dinfo%253Apmid%252F14711432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.12.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14711432&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188118500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.34" data-doi="10.1534/genetics.113.150789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schimenti</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuer</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffin</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovet</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">AKAP9 is essential for spermatogenesis and sertoli cell maturation in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">457</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.150789</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150789%26rft_id%253Dinfo%253Apmid%252F23608191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.35" data-doi="10.1101/gr.213611.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graves-Lindsay</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouk</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evaluation of GRCh38 and de novo haploid genome assemblies demonstrates the enduring quality of the reference assembly.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">864</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.213611.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.213611.116%26rft_id%253Dinfo%253Apmid%252F28396521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvODQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-9.6.1795.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source">Mouse Genetics</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-9.6.1795.37"><div class="cit-metadata unstructured">Smit, A., R. Hubley and P. Green, 2013–2015 RepeatMasker, pp.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.38" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarsani</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigmon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomes of the Mouse Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.198838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS82LzE3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.39" data-doi="10.1007/BF00341452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Genes for serum amyloid A proteins map to Chromosome 7 in the mouse.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">499</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00341452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DTaylor%26rft.auinit1%253DB.%2BA.%26rft.volume%253D195%26rft.issue%253D3%26rft.spage%253D491%26rft.epage%253D499%26rft.atitle%253DGenes%2Bfor%2Bserum%2Bamyloid%2BA%2Bproteins%2Bmap%2Bto%2BChromosome%2B7%2Bin%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00341452%26rft_id%253Dinfo%253Apmid%252F6088946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00341452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6088946&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TC14900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.40" data-doi="10.1016/j.ajpath.2015.10.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mruk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herter</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullere</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chojnacka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">AKAP9, a Regulator of Microtubule Dynamics, Contributes to Blood-Testis Barrier Function.</span> <abbr class="cit-jnl-abbrev">Am. J. Pathol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">270</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajpath.2015.10.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPathol.%26rft.volume%253D186%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajpath.2015.10.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajpath.2015.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.41" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouelliel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.42" data-doi="10.1101/gr.6927808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watkins-Chow</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavan</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomic copy number and expression variation within the C57BL/6J inbred mouse strain.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.6927808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6927808%26rft_id%253Dinfo%253Apmid%252F18032724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjE4LzEvNjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzYvMTc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.43" data-doi="10.1007/978-1-60761-058-8_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiles</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taft</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The sophisticated mouse: protecting a precious reagent.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">602</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-60761-058-8_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DWiles%26rft.auinit1%253DM.%2BV.%26rft.volume%253D602%26rft.spage%253D23%26rft.epage%253D36%26rft.atitle%253DThe%2Bsophisticated%2Bmouse%253A%2Bprotecting%2Ba%2Bprecious%2Breagent.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60761-058-8_2%26rft_id%253Dinfo%253Apmid%252F20012390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60761-058-8_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20012390&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.6.1795.44" data-doi="10.1371/journal.pone.0034555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Analysis of microsatellite polymorphism in inbred knockout mice.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e34555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0034555</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZuo%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De34555%26rft.epage%253De34555%26rft.atitle%253DAnalysis%2Bof%2Bmicrosatellite%2Bpolymorphism%2Bin%2Binbred%2Bknockout%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0034555%26rft_id%253Dinfo%253Apmid%252F22509320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0034555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22509320&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F6%2F1795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/6/1795.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/6/1785" title="Whole Genome Sequencing and Re-sequencing of the Sable Antelope (Hippotragus niger): A Resource for Monitoring Diversity in ex Situ and in Situ Populations" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/6/1807" title="Evolution of Antibiotic Synthesis Gene Clusters in the Streptomyces globisporus TFH56, Isolated from Tomato Flower" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/6"><span>Volume 9 Issue 6, June 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/6.cover-source.jpg?itok=Ub13TpI0" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/6/1795" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Z0ZSUDcXX0bE6SWZe8D-5BLfoDI5-q0UVk6o9hqnbpU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/6/1795" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/6/1795" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/6/1795" class="active" data-icon-position="" data-hide-link-title="0">The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493239" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-xKyfpQ6Xk7JSujSutoOxaf1aLmIAKe3V5BHIF_xk0P0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1795.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/6/1795" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-uWHRn_Ug8iPPUZnHVQsvRHUQYQaPs_FsPSLtBvgs8xg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/6/1795" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/6/1795.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493239" id="node493239--2" data-pisa="ggg;9/6/1795" data-pisa-master="ggg;g3.119.400071" data-apath="/ggg/9/6/1795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vishal Kumar Sarsani</span>, <span class="highwire-citation-author" data-delta="1">Narayanan Raghupathy</span>, <span class="highwire-citation-author" data-delta="2">Ian T. Fiddes</span>, <span class="highwire-citation-author" data-delta="3">Joel Armstrong</span>, <span class="highwire-citation-author" data-delta="4">Francoise Thibaud-Nissen</span>, <span class="highwire-citation-author" data-delta="5">Oraya Zinder</span>, <span class="highwire-citation-author" data-delta="6">Mohan Bolisetty</span>, <span class="highwire-citation-author" data-delta="7">Kerstin Howe</span>, <span class="highwire-citation-author" data-delta="8">Doug Hinerfeld</span>, <span class="highwire-citation-author" data-delta="9">Xiaoan Ruan</span>, <span class="highwire-citation-author" data-delta="10">Lucy Rowe</span>, <span class="highwire-citation-author" data-delta="11">Mary Barter</span>, <span class="highwire-citation-author" data-delta="12">Guruprasad Ananda</span>, <span class="highwire-citation-author" data-delta="13">Benedict Paten</span>, <span class="highwire-citation-author" data-delta="14">George M. Weinstock</span>, <span class="highwire-citation-author" data-delta="15">Gary A. Churchill</span>, <span class="highwire-citation-author" data-delta="16">Michael V. Wiles</span>, <span class="highwire-citation-author" data-delta="17">Valerie A. Schneider</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="18"><a href="http://orcid.org/0000-0002-8455-3993" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anuj Srivastava</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="19"><a href="http://orcid.org/0000-0003-4054-4048" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laura G. Reinholdt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1795-1805;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400071 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493239--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vishal Kumar Sarsani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vishal%2BKumar%2BSarsani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sarsani%20VK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVishal%2BKumar%2BSarsani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Narayanan Raghupathy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Narayanan%2BRaghupathy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghupathy%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANarayanan%2BRaghupathy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ian T. Fiddes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BT.%2BFiddes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fiddes%20IT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BT.%2BFiddes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel Armstrong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BArmstrong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Armstrong%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BArmstrong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Francoise Thibaud-Nissen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francoise%2BThibaud-Nissen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thibaud-Nissen%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancoise%2BThibaud-Nissen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Oraya Zinder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oraya%2BZinder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zinder%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOraya%2BZinder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mohan Bolisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohan%2BBolisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolisetty%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohan%2BBolisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kerstin Howe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Wellcome Sanger Institute, Hinxton, Cambridge CB10 1SA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerstin%2BHowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howe%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerstin%2BHowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Doug Hinerfeld </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>NanoString Technologies, Inc, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Doug%2BHinerfeld%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hinerfeld%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADoug%2BHinerfeld%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Xiaoan Ruan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoan%2BRuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoan%2BRuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Lucy Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucy%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mary Barter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BBarter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barter%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BBarter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Guruprasad Ananda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guruprasad%2BAnanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ananda%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuruprasad%2BAnanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Benedict Paten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benedict%2BPaten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paten%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenedict%2BPaten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">George M. Weinstock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BM.%2BWeinstock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinstock%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BM.%2BWeinstock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Michael V. Wiles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BV.%2BWiles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiles%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BV.%2BWiles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Valerie A. Schneider </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valerie%2BA.%2BSchneider%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneider%20VA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValerie%2BA.%2BSchneider%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Anuj Srivastava </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srivastava%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnuj%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8455-3993" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anuj Srivastava</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.reinholdt@jax.org" class="" data-icon-position="" data-hide-link-title="0">laura.reinholdt@jax.org</a>
<a href="mailto:anuj.srivastava@jax.org" class="" data-icon-position="" data-hide-link-title="0">anuj.srivastava@jax.org</a></span></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Laura G. Reinholdt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BG.%2BReinholdt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinholdt%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BG.%2BReinholdt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4054-4048" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laura G. Reinholdt</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.reinholdt@jax.org" class="" data-icon-position="" data-hide-link-title="0">laura.reinholdt@jax.org</a>
<a href="mailto:anuj.srivastava@jax.org" class="" data-icon-position="" data-hide-link-title="0">anuj.srivastava@jax.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/6/1795" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/6/1795&amp;title=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/6/1795&amp;title=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/6/1795&amp;title=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/6/1795&amp;text=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/6/1795&amp;title=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/6/1795&amp;t=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/6/1795&amp;title=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/6/1795&amp;title=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493239/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493239" id="node493239--3" data-pisa="ggg;9/6/1795" data-pisa-master="ggg;g3.119.400071" data-apath="/ggg/9/6/1795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genome of C57BL/6J “Eve”, the Mother of the Laboratory Mouse Genome Reference Strain</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Vishal Kumar Sarsani</span>, <span class="highwire-citation-author" data-delta="1">Narayanan Raghupathy</span>, <span class="highwire-citation-author" data-delta="2">Ian T. Fiddes</span>, <span class="highwire-citation-author" data-delta="3">Joel Armstrong</span>, <span class="highwire-citation-author" data-delta="4">Francoise Thibaud-Nissen</span>, <span class="highwire-citation-author" data-delta="5">Oraya Zinder</span>, <span class="highwire-citation-author" data-delta="6">Mohan Bolisetty</span>, <span class="highwire-citation-author" data-delta="7">Kerstin Howe</span>, <span class="highwire-citation-author" data-delta="8">Doug Hinerfeld</span>, <span class="highwire-citation-author" data-delta="9">Xiaoan Ruan</span>, <span class="highwire-citation-author" data-delta="10">Lucy Rowe</span>, <span class="highwire-citation-author" data-delta="11">Mary Barter</span>, <span class="highwire-citation-author" data-delta="12">Guruprasad Ananda</span>, <span class="highwire-citation-author" data-delta="13">Benedict Paten</span>, <span class="highwire-citation-author" data-delta="14">George M. Weinstock</span>, <span class="highwire-citation-author" data-delta="15">Gary A. Churchill</span>, <span class="highwire-citation-author" data-delta="16">Michael V. Wiles</span>, <span class="highwire-citation-author" data-delta="17">Valerie A. Schneider</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="18"><a href="http://orcid.org/0000-0002-8455-3993" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anuj Srivastava</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="19"><a href="http://orcid.org/0000-0003-4054-4048" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Laura G. Reinholdt</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1795-1805;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400071 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493239--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vishal Kumar Sarsani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vishal%2BKumar%2BSarsani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sarsani%20VK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVishal%2BKumar%2BSarsani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Narayanan Raghupathy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Narayanan%2BRaghupathy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghupathy%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANarayanan%2BRaghupathy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ian T. Fiddes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ian%2BT.%2BFiddes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fiddes%20IT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIan%2BT.%2BFiddes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joel Armstrong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joel%2BArmstrong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Armstrong%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoel%2BArmstrong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Francoise Thibaud-Nissen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francoise%2BThibaud-Nissen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thibaud-Nissen%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancoise%2BThibaud-Nissen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Oraya Zinder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oraya%2BZinder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zinder%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOraya%2BZinder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mohan Bolisetty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohan%2BBolisetty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bolisetty%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohan%2BBolisetty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kerstin Howe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Wellcome Sanger Institute, Hinxton, Cambridge CB10 1SA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerstin%2BHowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howe%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerstin%2BHowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Doug Hinerfeld </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>NanoString Technologies, Inc, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Doug%2BHinerfeld%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hinerfeld%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADoug%2BHinerfeld%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Xiaoan Ruan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoan%2BRuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ruan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoan%2BRuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Lucy Rowe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BRowe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowe%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucy%2BRowe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mary Barter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mary%2BBarter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barter%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMary%2BBarter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Guruprasad Ananda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guruprasad%2BAnanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ananda%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuruprasad%2BAnanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Benedict Paten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>UC Santa Cruz Genomics Institute, Santa Cruz, California, Department of Biomolecular Engineering, University of California, Santa Cruz, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benedict%2BPaten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paten%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenedict%2BPaten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">George M. Weinstock </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BM.%2BWeinstock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinstock%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGeorge%2BM.%2BWeinstock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Gary A. Churchill </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BA.%2BChurchill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Churchill%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BA.%2BChurchill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Michael V. Wiles </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BV.%2BWiles%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wiles%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BV.%2BWiles%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Valerie A. Schneider </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD 20894</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valerie%2BA.%2BSchneider%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneider%20VA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValerie%2BA.%2BSchneider%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Anuj Srivastava </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>The Jackson Laboratory for Genomic Medicine, Farmington, CT</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BSrivastava%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Srivastava%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnuj%2BSrivastava%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8455-3993" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anuj Srivastava</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.reinholdt@jax.org" class="" data-icon-position="" data-hide-link-title="0">laura.reinholdt@jax.org</a>
<a href="mailto:anuj.srivastava@jax.org" class="" data-icon-position="" data-hide-link-title="0">anuj.srivastava@jax.org</a></span></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Laura G. Reinholdt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The Jackson Laboratory for Mammalian Genetics, Bar Harbor ME</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laura%2BG.%2BReinholdt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reinholdt%20LG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALaura%2BG.%2BReinholdt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4054-4048" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Laura G. Reinholdt</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:laura.reinholdt@jax.org" class="" data-icon-position="" data-hide-link-title="0">laura.reinholdt@jax.org</a>
<a href="mailto:anuj.srivastava@jax.org" class="" data-icon-position="" data-hide-link-title="0">anuj.srivastava@jax.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1795&amp;rft.epage=1805&amp;rft.atitle=The%20Genome%20of%20C57BL/6J%20%E2%80%9CEve%E2%80%9D%2C%20the%20Mother%20of%20the%20Laboratory%20Mouse%20Genome%20Reference%20Strain&amp;rft.volume=9&amp;rft.issue=6&amp;rft.date=2019-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sarsani%2C+Vishal+Kumar&amp;rft.au=Raghupathy%2C+Narayanan&amp;rft.au=Fiddes%2C+Ian+T.&amp;rft.au=Armstrong%2C+Joel&amp;rft.au=Thibaud-Nissen%2C+Francoise&amp;rft.au=Zinder%2C+Oraya&amp;rft.au=Bolisetty%2C+Mohan&amp;rft.au=Howe%2C+Kerstin&amp;rft.au=Hinerfeld%2C+Doug&amp;rft.au=Ruan%2C+Xiaoan&amp;rft.au=Rowe%2C+Lucy&amp;rft.au=Barter%2C+Mary&amp;rft.au=Ananda%2C+Guruprasad&amp;rft.au=Paten%2C+Benedict&amp;rft.au=Weinstock%2C+George+M.&amp;rft.au=Churchill%2C+Gary+A.&amp;rft.au=Wiles%2C+Michael+V.&amp;rft.au=Schneider%2C+Valerie+A.&amp;rft.au=Srivastava%2C+Anuj&amp;rft.au=Reinholdt%2C+Laura+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493239/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493239/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493239/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493239/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493239/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493239/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493239/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493239/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493239/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493239/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493239/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493239/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/6/1795" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material &amp; Methods</a></li><li class="odd"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/6/1795.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/6/1795.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/6/1795.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
