<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3203" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3203.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3203.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3203.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/10/3203/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3203.atom" />
<meta name="HW.pisa" content="ggg;8/10/3203" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Estimating Genetic Relatedness in Admixed Populations" />
<meta name="DC.Identifier" content="10.1534/g3.118.200485" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Sethuraman. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Estimating genetic relatedness, and inbreeding coefficients is important to the fields of quantitative genetics, conservation, genome-wide association studies (GWAS), and population genetics. Traditional estimators of genetic relatedness assume an underlying model of population structure. Each individual is assigned to a population, depending on a priori assumptions about geographical location of sampling, proximity, or genetic similarity. But often, this population assignment is unknown and assumptions about assignment can lead to erroneous estimates of genetic relatedness. I develop a generalized method of estimating relatedness in admixed populations, to account for (1) multi-allelic genomic data, (2) including all nine Identity By Descent (IBD) states, and implement a maximum likelihood based estimator of pairwise genetic relatedness in structured populations, part of the software, InRelate. Replicated estimations of genetic relatedness between admixed full sib (FS), half sib (HS), first cousin (FC), parent-offspring (PO) and unrelated (UR) dyads in simulated and empirical data from the HGDP-CEPH panel show considerably low bias and error while using InRelate, compared to several previously developed methods. I also propose a bootstrap scheme, and a series of Wald Tests to assign relatedness categories to pairs of individuals." />
<meta name="DC.Contributor" content="Arun Sethuraman" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Estimating Genetic Relatedness in Admixed Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Estimating genetic relatedness, and inbreeding coefficients is important to the fields of quantitative genetics, conservation, genome-wide association studies (GWAS), and population genetics. Traditional estimators of genetic relatedness assume an underlying model of population structure. Each individual is assigned to a population, depending on &lt;i&gt;a priori&lt;/i&gt; assumptions about geographical location of sampling, proximity, or genetic similarity. But often, this population assignment is unknown and assumptions about assignment can lead to erroneous estimates of genetic relatedness. I develop a generalized method of estimating relatedness in admixed populations, to account for (1) multi-allelic genomic data, (2) including all nine Identity By Descent (IBD) states, and implement a maximum likelihood based estimator of pairwise genetic relatedness in structured populations, part of the software, InRelate. Replicated estimations of genetic relatedness between admixed full sib (FS), half sib (HS), first cousin (FC), parent-offspring (PO) and unrelated (UR) dyads in simulated and empirical data from the HGDP-CEPH panel show considerably low bias and error while using InRelate, compared to several previously developed methods. I also propose a bootstrap scheme, and a series of Wald Tests to assign relatedness categories to pairs of individuals.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3203" />
<meta name="citation_id" content="8/10/3203" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3203" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3203.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3203.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3203.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200485" />
<meta name="citation_pmid" content="30104261" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3203" />
<meta name="citation_lastpage" content="3220" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Arun Sethuraman" />
<meta name="citation_author_institution" content="Department of Biological Sciences, California State University San Marcos, CA 92096" />
<meta name="citation_author_email" content="asethuraman@csusm.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8201-8292" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=F. W. Allendorf;citation_author=P. A. Hohenlohe;citation_author=G. Luikart;citation_title=Genomics and the future of conservation genetics.;citation_pages=697-709;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20847747;citation_doi=10.1038/nrg2844" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. D. Anderson;citation_author=B. S. Weir;citation_title=A maximum-likelihood method for the estimation of pairwise relatedness in structured populations.;citation_volume=176;citation_year=2007;citation_pmid=17339212;citation_doi=10.1534/genetics.106.063149" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=J. C. Avise;citation_title=Introduction;citation_pages=99-99;citation_volume=92;citation_year=2001;citation_issue=2;citation_pmid=11700602;citation_doi=10.1093/jhered/92.2.99" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=F. Balloux;citation_title=EASYPOP (Version 1.7): A Computer Program for Population Genetics Simulations;citation_pages=301-302;citation_volume=92;citation_year=2001;citation_issue=3;citation_pmid=11447253;citation_doi=10.1093/jhered/92.3.301" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=M. Blouin;citation_title=Dna-based methods for pedigree reconstruction and kinship analysis in natural populations.;citation_pages=503-511;citation_volume=18;citation_year=2003;citation_doi=10.1016/S0169-5347(03)00225-8" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Boehnke;citation_author=N. J. Cox;citation_title=Accurate inference of relationships in sib-pair linkage studies.;citation_pages=423-429;citation_volume=61;citation_year=1997;citation_issue=2;citation_pmid=9311748;citation_doi=10.1086/514862" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. M. Cann;citation_author=C. de Toma;citation_author=L. Cazes;citation_author=M. F. Legrand;citation_author=V. Morel;citation_title=A human genome diversity cell line panel.;citation_pages=261-262;citation_volume=296;citation_year=2002;citation_issue=5566;citation_pmid=11954565;citation_doi=10.1126/science.296.5566.261b" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=S. W. Coleman;citation_author=A. G. Jones;citation_title=Patterns of multiple paternity and maternity in fishes.;citation_pages=735-760;citation_volume=103;citation_year=2011;citation_doi=10.1111/j.1095-8312.2011.01673.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=H. E. Collins-Schramm;citation_author=C. M. Phillips;citation_author=D. J. Operario;citation_author=J. S. Lee;citation_author=J. L. Weber;citation_title=Ethnic-difference markers for use in mapping by admixture linkage disequilibrium.;citation_pages=737-750;citation_volume=70;citation_year=2002;citation_issue=3;citation_pmid=11845411;citation_doi=10.1086/339368" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=M. P. Conomos;citation_author=A. P. Reiner;citation_author=B. S. Weir;citation_author=T. A. Thornton;citation_title=Model-free estimation of recent genetic relatedness.;citation_pages=127-148;citation_volume=98;citation_year=2016;citation_doi=10.1016/j.ajhg.2015.11.022" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=A. P. Dempster;citation_author=N. M. Laird;citation_author=D. B. Rubin;citation_title=Maximum likelihood from incomplete data via em algorithm.;citation_pages=1-38;citation_volume=39;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. P. Epstein;citation_author=W. L. Duren;citation_author=M. Boehnke;citation_title=Improved inference of relationship for pairs of individuals.;citation_pages=1219-1231;citation_volume=67;citation_year=2000;citation_issue=5;citation_pmid=11032786;citation_doi=10.1086/321195" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Eriksson;citation_author=A. Manica;citation_title=Effect of ancient population structure on the degree of polymorphism shared between modern human populations and ancient hominins.;citation_volume=109;citation_year=2012;citation_pmid=22893688;citation_doi=10.1073/pnas.1200567109" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="Falconer, D. S., and T. F. C. Mackay, 1996 Introduction to Quantitative Genetics, Ed 4. Longmans Green, Harlow, Essex, UK." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Notes;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of population structure using multilocus genotype data: dominant markers and null alleles.;citation_pages=574-578;citation_volume=7;citation_year=2007;citation_issue=4;citation_pmid=18784791;citation_doi=10.1111/j.1471-8286.2007.01758.x" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=A tutorial on how (not) to over-interpret structure/admixture bar plots.;citation_author=D. Falush;citation_author=L. van Dorp;citation_author=D. Lawson;citation_pages=066431;citation_year=2016" />
<meta name="citation_reference" content="citation_author=A. Ghalanos;citation_author=S. Theussl;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=M. J. Hubisz;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inferring weak population structure with the assistance of sample group information.;citation_pages=1322-1332;citation_volume=9;citation_year=2009;citation_issue=5;citation_pmid=21564903;citation_doi=10.1111/j.1755-0998.2009.02591.x" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=A. Jacquard;citation_title=Genetic information given by a relative.;citation_pages=1101-1114;citation_volume=28;citation_year=1972;citation_issue=4;citation_pmid=4648793;citation_doi=10.2307/2528643" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Jakobsson;citation_author=N. A. Rosenberg;citation_title=Clumpp: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.;citation_volume=23;citation_year=2007;citation_pmid=17485429;citation_doi=10.1093/bioinformatics/btm233" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=D. A. Konovalov;citation_author=D. Heg;citation_title=TECHNICAL ADVANCES: A maximum-likelihood relatedness estimator allowing for negative relatedness values.;citation_pages=256-263;citation_volume=8;citation_year=2008;citation_issue=2;citation_pmid=21585767;citation_doi=10.1111/j.1471-8286.2007.01940.x" />
<meta name="citation_reference" content="citation_journal_title=Human heredity;citation_journal_abbrev=Hum Hered;citation_author=C. C. Li;citation_author=D. E. Weeks;citation_author=A. Chakravarti;citation_title=Similarity of DNA fingerprints due to chance and relatedness.;citation_pages=45-52;citation_volume=43;citation_year=1993;citation_issue=1;citation_pmid=8514326;citation_doi=10.1159/000154113" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genomics;citation_author=Y. Liu;citation_author=T. Nyunoya;citation_author=S. Leng;citation_author=S. A. Belinsky;citation_author=Y. Tesfaigzi;citation_title=Softwares and methods for estimating genetic ancestry in human populations.;citation_pages=1;citation_volume=7;citation_year=2013;citation_pmid=23289408;citation_doi=10.1186/1479-7364-7-1" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=M. Lynch;citation_title=Estimation of relatedness by DNA fingerprinting;citation_pages=584-599;citation_volume=5;citation_year=1988;citation_issue=5;citation_pmid=3193879" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Lynch;citation_author=K. Ritland;citation_title=Estimation of Pairwise Relatedness With Molecular Markers;citation_pages=1753-1766;citation_volume=152;citation_year=1999;citation_issue=4;citation_pmid=10430599" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Manichaikul;citation_author=J. C. Mychaleckyj;citation_author=S. S. Rich;citation_author=K. Daly;citation_author=M. Sale;citation_title=Robust relationship inference in genome-wide association studies.;citation_volume=26;citation_year=2010;citation_pmid=20926424;citation_doi=10.1093/bioinformatics/btq559" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. G. Milligan;citation_title=Maximum-Likelihood Estimation of Relatedness;citation_pages=1153-1167;citation_volume=163;citation_year=2003;citation_issue=3;citation_pmid=12663552" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Moltke;citation_author=A. Albrechtsen;citation_title=Relateadmix: a software tool for estimating relatedness between admixed individuals.;citation_volume=30;citation_year=2013;citation_pmid=24215025;citation_doi=10.1093/bioinformatics/btt652" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=M. Nei;citation_author=R. K. Chesser;citation_title=Estimation of fixation indices and gene diversities.;citation_pages=253-259;citation_volume=47;citation_year=1983;citation_issue=Pt 3;citation_pmid=6614868;citation_doi=10.1111/j.1469-1809.1983.tb00993.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. A. Oliehoek;citation_author=J. J. Windig;citation_author=J. A. M. van Arendonk;citation_author=P. Bijma;citation_title=Estimating relatedness between individuals in general populations with a focus on their use in conservation programs.;citation_volume=173;citation_year=2006;citation_pmid=16510792;citation_doi=10.1534/genetics.105.049940" />
<meta name="citation_reference" content="citation_journal_title=Behav. Ecol. Sociobiol.;citation_author=D. E. Pearse;citation_author=F. J. Janzen;citation_author=J. C. Avise;citation_title=Multiple paternity, sperm storage, and reproductive success of female and male painted turtles (chrysemys picta) in nature.;citation_pages=164-171;citation_volume=51;citation_year=2002;citation_doi=10.1007/s00265-001-0421-7" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=T. J. Pemberton;citation_author=M. DeGiorgio;citation_author=N. A. Rosenberg;citation_title=Population structure in a comprehensive genomic data set on human microsatellite variation.;citation_volume=3;citation_year=2013;citation_pmid=23550135;citation_doi=10.1534/g3.113.005728" />
<meta name="citation_reference" content="citation_title=Numerical recipes 3rd edition: The art of scientific computing;citation_author=W. H. Press;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Pritchard;citation_author=M. Stephens;citation_author=N. Rosenberg;citation_author=P. Donnelly;citation_title=Association mapping in structured populations.;citation_pages=170-181;citation_volume=67;citation_year=2000;citation_issue=1;citation_pmid=10827107;citation_doi=10.1086/302959" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. C. Queller;citation_author=K. F. Goodnight;citation_title=Estimating relatedness using genetic-markers.;citation_pages=258-275;citation_volume=43;citation_year=1989;citation_doi=10.2307/2409206" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Ramachandran;citation_author=O. Deshpande;citation_author=C. C. Roseman;citation_author=N. A. Rosenberg;citation_author=M. W. Feldman;citation_title=Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in africa.;citation_volume=102;citation_year=2005;citation_pmid=16243969;citation_doi=10.1073/pnas.0507611102" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=M. D. Ramstetter;citation_author=T. Dyer;citation_author=D. M. Lehman;citation_author=J. E. Curran;citation_author=R. Duggirala;citation_title=Benchmarking relatedness inference methods with genome-wide data from thousands of relatives.;citation_volume=207;citation_year=2017;citation_pmid=28739658;citation_doi=10.1534/genetics.117.1122" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=K. Ritland;citation_title=Estimators for pairwise relatedness and individual inbreeding coefficients.;citation_pages=175-186;citation_volume=67;citation_year=1996;citation_doi=10.1017/S0016672300033620" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=K. Ritland;citation_title=Multilocus estimation of pairwise relatedness with dominant markers.;citation_pages=3157-3165;citation_volume=14;citation_year=2005;citation_issue=10;citation_pmid=16101781;citation_doi=10.1111/j.1365-294X.2005.02667.x" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=N. A. Rosenberg;citation_title=Standardized subsets of the HGDP-CEPH Human Genome Diversity Cell Line Panel, accounting for atypical and duplicated samples and pairs of close relatives.;citation_pages=841-847;citation_volume=70;citation_year=2006;citation_issue=Pt 6;citation_pmid=17044859;citation_doi=10.1111/j.1469-1809.2006.00285.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. A. Rosenberg;citation_author=S. Mahajan;citation_author=C. Gonzalez-Quevedo;citation_author=M. G. Blum;citation_author=L. Nino-Rosales;citation_title=Low levels of genetic divergence across geographically and linguistically diverse populations from India.;citation_pages=e215-e215;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194221;citation_doi=10.1371/journal.pgen.0020215" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. A. Rosenberg;citation_author=S. Mahajan;citation_author=S. Ramachandran;citation_author=C. F. Zhao;citation_author=J. K. Pritchard;citation_title=Clines, clusters, and the effect of study design on the inference of human population structure.;citation_pages=e70-e70;citation_volume=1;citation_year=2005;citation_issue=6;citation_pmid=16355252;citation_doi=10.1371/journal.pgen.0010070" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. A. Rosenberg;citation_author=J. K. Pritchard;citation_author=J. L. Weber;citation_author=H. M. Cann;citation_author=K. K. Kidd;citation_title=Genetic Structure of Human Populations;citation_pages=2381-2385;citation_volume=298;citation_year=2002;citation_issue=5602;citation_pmid=12493913;citation_doi=10.1126/science.1078311" />
<meta name="citation_reference" content="citation_author=A. Sethuraman;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=E. A. Thompson;citation_title=The estimation of pairwise relationships.;citation_pages=173-188;citation_volume=39;citation_year=1975;citation_issue=2;citation_pmid=1052764;citation_doi=10.1111/j.1469-1809.1975.tb00120.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=T. Thornton;citation_author=H. Tang;citation_author=T. J. Hoffmann;citation_author=H. M. Ochs-Balcom;citation_author=B. J. Caan;citation_title=Estimating kinship in admixed populations.;citation_pages=122-138;citation_volume=91;citation_year=2012;citation_issue=1;citation_pmid=22748210;citation_doi=10.1016/j.ajhg.2012.05.024" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. A. Tishkoff;citation_author=F. A. Reed;citation_author=F. R. Friedlaender;citation_author=C. Ehret;citation_author=A. Ranciaro;citation_title=The genetic structure and history of Africans and African Americans.;citation_volume=324;citation_year=2009;citation_pmid=19407144;citation_doi=10.1126/science.1172257" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. M. Visscher;citation_author=W. G. Hill;citation_author=N. R. Wray;citation_title=Heritability in the genomics era--concepts and misconceptions.;citation_pages=255-266;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18319743;citation_doi=10.1038/nrg2322" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Wang;citation_title=Triadic IBD coefficients and applications to estimating pairwise relatedness.;citation_pages=135-153;citation_volume=89;citation_year=2007;citation_issue=3;citation_pmid=17894908;citation_doi=10.1017/S0016672307008798" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Ecol Resour;citation_author=J. Wang;citation_title=COANCESTRY: a program for simulating, estimating and analysing relatedness and inbreeding coefficients.;citation_pages=141-145;citation_volume=11;citation_year=2011;citation_issue=1;citation_pmid=21429111;citation_doi=10.1111/j.1755-0998.2010.02885.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Wang;citation_title=Unbiased relatedness estimation in structured populations.;citation_volume=187;citation_year=2011;citation_pmid=21212234;citation_doi=10.1534/genetics.110.124438" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Wang;citation_title=Effects of sampling close relatives on some elementary population genetics analyses.;citation_pages=41-54;citation_volume=18;citation_year=2018;citation_doi=10.1111/1755-0998.12708" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. L. Wang;citation_title=An Estimator for Pairwise Relatedness Using Molecular Markers;citation_pages=1203-1215;citation_volume=160;citation_year=2002;citation_issue=3;citation_pmid=11901134" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=B. Weir;citation_title=The Effects of Inbreeding on Forensic Calculations;citation_pages=597-621;citation_volume=28;citation_year=1994;citation_issue=1;citation_pmid=7893141;citation_doi=10.1146/annurev.ge.28.120194.003121" />
<meta name="citation_reference" content="citation_journal_title=Journal of Forensic Sciences;citation_journal_abbrev=Journal of Forensic Sciences;citation_author=B. Weir;citation_title=Matching and partially-matching DNA profiles.;citation_pages=1009-1014;citation_volume=49;citation_year=2004;citation_issue=5;citation_pmid=15461102;citation_doi=10.1520/JFS2003039" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. S. Weir;citation_author=A. D. Anderson;citation_author=A. B. Hepler;citation_title=Genetic relatedness analysis: modern data and new challenges.;citation_pages=771-780;citation_volume=7;citation_year=2006;citation_issue=10;citation_pmid=16983373;citation_doi=10.1038/nrg1960" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating f-statistics for the analysis of population-structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_pmid=28563791;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Wright;citation_title=Genetical structure of populations.;citation_pages=247-249;citation_volume=166;citation_year=1950;citation_issue=4215;citation_pmid=15439261;citation_doi=10.1038/166247a0" />
<meta name="citation_reference" content="citation_author=Y. Ye;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. H. Yue;citation_author=A. Chang;citation_title=Molecular evidence for high frequency of multiple paternity in a freshwater shrimp species Caridina ensifera.;citation_pages=e12721-e12721;citation_volume=5;citation_year=2010;citation_issue=9;citation_pmid=20856862;citation_doi=10.1371/journal.pone.0012721" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Estimating Genetic Relatedness in Admixed Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="Estimating genetic relatedness, and inbreeding coefficients is important to the fields of quantitative genetics, conservation, genome-wide association studies (GWAS), and population genetics. Traditional estimators of genetic relatedness assume an underlying model of population structure. Each individual is assigned to a population, depending on a priori assumptions about geographical location of sampling, proximity, or genetic similarity. But often, this population assignment is unknown and assumptions about assignment can lead to erroneous estimates of genetic relatedness. I develop a generalized method of estimating relatedness in admixed populations, to account for (1) multi-allelic genomic data, (2) including all nine Identity By Descent (IBD) states, and implement a maximum likelihood based estimator of pairwise genetic relatedness in structured populations, part of the software, InRelate. Replicated estimations of genetic relatedness between admixed full sib (FS), half sib (HS), first cousin (FC), parent-offspring (PO) and unrelated (UR) dyads in simulated and empirical data from the HGDP-CEPH panel show considerably low bias and error while using InRelate, compared to several previously developed methods. I also propose a bootstrap scheme, and a series of Wald Tests to assign relatedness categories to pairs of individuals." />
<meta name="og-title" property="og:title" content="Estimating Genetic Relatedness in Admixed Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3203" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Estimating genetic relatedness, and inbreeding coefficients is important to the fields of quantitative genetics, conservation, genome-wide association studies (GWAS), and population genetics. Traditional estimators of genetic relatedness assume an underlying model of population structure. Each individual is assigned to a population, depending on a priori assumptions about geographical location of sampling, proximity, or genetic similarity. But often, this population assignment is unknown and assumptions about assignment can lead to erroneous estimates of genetic relatedness. I develop a generalized method of estimating relatedness in admixed populations, to account for (1) multi-allelic genomic data, (2) including all nine Identity By Descent (IBD) states, and implement a maximum likelihood based estimator of pairwise genetic relatedness in structured populations, part of the software, InRelate. Replicated estimations of genetic relatedness between admixed full sib (FS), half sib (HS), first cousin (FC), parent-offspring (PO) and unrelated (UR) dyads in simulated and empirical data from the HGDP-CEPH panel show considerably low bias and error while using InRelate, compared to several previously developed methods. I also propose a bootstrap scheme, and a series of Wald Tests to assign relatedness categories to pairs of individuals." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Estimating Genetic Relatedness in Admixed Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2WCNm1_Z8l5PuVXkXf1V5rHNouGljspfP1-N9HyysoU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490975","apath":"\/ggg\/8\/10\/3203.atom","pisa":"ggg;8\/10\/3203","ac":{"\/ggg\/8\/10\/3203.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3203.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3203"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3203"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3203":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225nUwdA2zNPtfNgACDG1WM4Ev9IbDPkkQpxDgtW9\\\\\\\/wF76xpIoNhDERJfbJKgCeBbphy3I7mQOl0T6PrmKyd5o3sO83inffw+zZyXIn02Ik9NrubIsawrUMrp1z5CUC7BuJDKT8derXBpSUsSHN7G\\\\\\\/0o6BUjce+ng98bn4fFf881DUJO95SBiLGgCFvtQ9kYHrScSfLXN7fpsU86PaDS808xhaQZlF9qjniPe+g4harmEwIb5lKKnbHoux8+iwlz3u7lOfZeLwoPAxgbLIFAYMajel3p+hmiuXmgSktRvWiXfAdWJQyO1t0heuBRnND1+HPg9jbtsG4ekSI\\\\\\\/GCSXdvXurbvJt+qPT5JlhM5JEnvKlk7\\\\\\\/KWmky21IZc9PrVSmyM+0UbPZ7s5Ew\\\\\\\/aQhmiDE0wXXUvNoxCADEKuAnPk8rhUI7VimE7iPBgXY5Qt\\\\\\\/rj20CWosqX+Qso+K6ogQwbXQHwpcP+3zQufwOAVqjw\\\\\\\/JWjBeYHFGPYsG2YdN6a1le8xUmBmEPgKTyc3wNuibODaY1Rq5EWXsrOtbMPjKrH\\\\\\\/QHnmy0L2admF5uMC4GhIRJn8R7QO4ixds7FHwTveEArh0854CcZ3DFKIfaxiqxqzo7+ep5xQwTnTM+XuC37y0KZyO46ASbUD\\\\\\\/Ys8urLuHfXOz50F9Pvvfd0WvXdRe0eL0Lf5tAW5szfNDUFvVAXGK3zRYh4ba87S2XwKwDwxiKWxaT9lwQ9CoBDkh2PsLSXyNCyn7W5TJ4bg+c8vKvdQZGJstBCZ0H9O4tBzsxTa00B+H\\\\\\\/Jz8gyVWl88U3T0oZ2xY+oYeZbHMgWTZo1OKvoLjECVVcFFaiam+qegayfznIWynKcbm6WHSZ5zGBNKMt7Bd+JZPnMltvknUjBX8u67MTriVffxygWYoPDM3WQEyKmhbxgF\\\\\\\/IJO8xz\\\\\\\/eAgEDYowhc0G6tVaJjrGF9HgRFl3eOiUzvZKbghVdkyI1AQovmsvcL2lBKv+mMqGCYH9MRWLbG\\\\\\\/65tNG5ja7vICHoHs\\\\\\\/Dml+8hbyWr3QzkyQX8gSw0JyCGKT0zm6pGbIG8qakYLbqOJr7kJQsLmumAS8htBrszN1aQ+eh\\\\\\\/vqWQADETAs4T1ehYwnuc39doLlsljS7Mk0SRXSyA\\\\\\\/2iZsCxsLqHx7COQZFs085TPZ4KfM+\\\\\\\/TYgw1DY9eIKDFOyimVvxfob\\\\\\\/Ee3TW1nwJ6SRgDvKPwDxC5URTTs8sAaBvDFAQShplt7c16Qdpx8ZX9uAPi2RgEigmwONOV5\\\\\\\/7Ar73G3dyBT2Drc\\\\\\\/yeZi0\\\u0022,\\\u0022iv\\\u0022:\\\u0022tZOl3AneNix2YqDFS3dspg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002203c5d2a2fce59f0d971cb3790c5453dd30000bef3fe9b7e3ceaf57c87ee47229\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ROX5s0Kh6+cjKOvE6XpjsvHD6qAAT4ls787cxjbSIM0DYD1NDowMYmROyNHhA7Hrs9XJqKML366nSZbnXhjLU4H\\\\\\\/WXQX9tPmpuPw\\\\\\\/imlnEn2ScLaD4qPkE2aU49uD09A9JsfP6StNAmKIb7sMqMLQqAs7gBUfgEY0mk8bTolzr+2a7JXYOemdAXK+ByrPOm1\\\\\\\/ca2t8bNmJS\\\\\\\/k9A+sFC5iEU4fiZL7f8w7iQZrZvcl3+hndxAfpxBax3oBLThbE4nInuS4laf2SApqFPTlG1a5Lzb1ZHJK9J5\\\\\\\/TLcHWrsl+630TkyrIeJPN28Na+F0febov2gUL3EbNbaYf5GQ6cFWTUXfPnK+spWayBvTvjPiRm3fz2Y0gIrq3owKLhCdm9qJ+BlPUdomB07lJetdOhzfBBugn8yH9QUGQIA\\\\\\\/odK4Kr2bOpNgR16tWg7ruKX\\\\\\\/ZyryYoNtM72TjZ\\\\\\\/reTCuuwT9uqlW+OYcsVATRvzl29835\\\\\\\/t9I0alljUbHboqyBhj0ziSyT8fP2sIuwv43C\\\\\\\/UI9\\\\\\\/ZVdACXwcmcZOoQEQcneFdfufOooSXrcqGrlp+RtpP3lFOMwa968odz5BKZPgXr0l\\\\\\\/XvygA28gv9JoKaxxGO0XMQqc58SXip11M3BTLDk1IY7LatefFD4\\\\\\\/TRVYitsIetfQkovQA6tAbnmuiLV+jTD6WMuM7ljXSHLz8tkwk4QEr0CqESp54\\\\\\\/zFukGdKykKyPD9U0Wwqme+eEGXpzpE7nV8BaNyhm9AvHYQMpciRPRNqfzHiqTC1MKHYXSgb\\\\\\\/GRa5k8\\\\\\\/+0Q6\\\\\\\/apaIudRdViH+8Y15cnd8+d03bLRfyhtX0Np+fRIXCL6GfTfaL\\\\\\\/w2ZusvTbl4YS3SjkOh0kRIFpdiEE7JskxyG7PNBCO8n8kA3bpBNAeioFPKKA3vfKWCq8KNtxBf0U+FPLP7msVi9YBi33CzpLPMGXWXjaCADU0FdzBa7LTO5KOYfQuIQhkGVoe0jzlomfogro4gIDhcfPyRNokcI+dMOK3WCWHKFCrsJut4gICHbLOWng6tPdf6+zr\\\\\\\/LKOmcnZN9WIhuHw01bmEhbGvsOzB5qG03bHN5TKECtxhgw\\\\\\\/X1+EwLHm37ZVmlTcMtcb6RHw0UgENV+P9SEhdGCA4UyUV\\\\\\\/Qz58tSLyIpeAgPy0lFKkbA\\\\\\\/pSEbklEyrgwIG+WX1aPPq6sEBIBkMv4mmZJoxXCv7LFlBUPBcoVYUDpMrhCuHneW3qr1stJvyfso1dEaQZXci6nvtOgKuOiqzrSjZpBPOlWfiDbLZv+yB44y1hO68B0M4LnupI8FvkhxIcziOLAmaGbY4XGmZ+PGihy0JMwWYXjOulW6BUkt6nhtHYT8fS64Bq2Q7GT1elklQYaK90MKfC4tZPAnyJ\\\\\\\/z+50QDIpXqOXMrmd3eVIxbeZQeOljivL7pfA31KRtM6RcVYgscKyaikNqGIhJwlI39SzTPbtat72je5Y8gq3tEqdVMcA784UyvWJIfnP\\\\\\\/fRPEQurhNc6GTIZBcTDFUqMAPJSutOw5USIsiifERAa5pTz+fW49bNR413gvNBmW+uRc6M96PCCIS0MkEFCLfD12L2NstyW1IFnwT+5m7vk1XcYXPLHqP1\\\\\\\/nO1JxILkqOyE\\\\\\\/VmFGBjn+kxu3IN1\\\\\\\/+JymMQEydYFDBB0ROD7g5R2hSbcA31NyZEwiX8b7ZYDfLXxpf\\\\\\\/U\\\\\\\/hjAyS02UcERL9VDYBoLyfe\\\\\\\/bEFz2oyTD7HMBNCwsNI0kpug2g6I4fK9ExV+U1rF87nOQJb8wWJOOQ75EJdcNSiYen8vXQjY\\\\\\\/yLoqqM1+zv6JNt19gPzrvJE6nRXTaHjbzpPqwXkiRAKuW8zXRBXsEWqyMswzkNp0G1zvIvYTb8a5qW+S\\\\\\\/pJRToX5\\\\\\\/lFIPdCaQSr42ZnggTERpqxzw\\\\\\\/6i\\\\\\\/tLUmrJ\\\\\\\/l2uNnFNdZPiAqfbVh0iw87FE9c+RcSLQLbGuzp6jYGfO3zhHZj76jRohGdo1TMorgoPcPB3N3L9JTyHeao6aL93DHPwWHQCNZ5+ZK\\\\\\\/S6pEZ8+jUnX2xNc6I5dNbid1396dZsuAjc+KPGEqV4LrrEi\\\\\\\/jeYI7pnA\\\\\\\/oo+kZ+T87IQrUzg5JR4DA39Y0Jzx\\\\\\\/XcdYVauai7sr0Jb\\\\\\\/g99nV6llG5gEjPop7GIxkHXTI6KRTYiYsgHZwMvOHPm\\\\\\\/eag3zev2gTMTFESq5EQf\\\\\\\/jNgWyOBaY8y\\\\\\\/BpDewXPdVD9652RUmnlCvOFbUr0hWlxO6QCIN2pvdRvAnngs3\\\\\\\/ro4QY+\\\\\\\/Q4riqs8CDVdIo4Qms+IdQRCTYRcCsrrUA0LNKScJamQBmntmFz\\\\\\\/7GAQf0o70AcpEMLrBDKpLTeR2OMy81oYdGCc88OrnDYurevY2RiK8sd19q981i7zNkB6qpjwr78Ae+dynZEQsfN47t5QPHwyprGsamIvYyumQ\\\\\\\/6ObAGY74MwLhWF\\\\\\\/ncGKlnAd9Jp7QnyIUI9En6ru6PitAc3TXK1Sy0hAB\\\\\\\/LLLNSoflG4JCg9Mt1JLSCfX6MMYPZac998J8sDB70k\\\\\\\/GOvVYtgVDBA10Q1SjmfL8VwTiLps3jK8\\\\\\\/OfFgf3LRXGpAQgwkrpTrf4qMSIihcVPUS9giNryJPoCsoHqHszt2NN9GsEscOMLca6EhGSWWO\\\\\\\/8sPIGFZX7asrTl0izhs+tqJZyTgsLicIEVdQ3\\\\\\\/wcgV6h9\\\\\\\/jf8LttlQQh0Y7wx2FUZsUTjBlDpPi4+89dcgtINMenZYCXtgxfEO3lOs\\\\\\\/miwaEheW94LEBgbot6yAqJrFrUkoH6yBLr3mLVIUr4tWwXPNgyozGKN1bN\\\\\\\/A\\\\\\\/nTKSU4WepL+kVv8JEVuQ6KOQYNV3uhOmxViXFB2OKGmLUmZvXgran9P+bGRyNsLEne\\\\\\\/idEFRu5Dko\\\\\\\/jNSqrlRecJXtvn\\\\\\\/+os0lnxdr+vbIqQgI7LHPISoKQ5Ha1b7zg3cdId0GcujelK5fjLQS5OWjYbCk3NPFoJg7odHw0V7sbFvEqudnegJSeHStE7LA+6NszJpQd0CKiNvH7XVvVI0BqAggqK5X0f8\\\\\\\/FnpbVAXVM3QfTcQFSPKkyVMM83SZ7tNv256HMclEh5i0QWwdPiQo1ZjtM3dZ76GL+sIpQ0Lhnx\\\\\\\/k5+sfVgwzYh+oJmGiW8wYbQDxoMf9c+pFCjWGCbslxigU9K74jgnwGoSRBbRtpAhJzn7tQffas7cOSFjUdO0pZEUZiBojNLHSBjh9uEodUhaxn8oV4zG5K9\\\\\\\/rWYCk5ALT7OYbpc4d9ZAuyPMNm70QZeVch\\\\\\\/\\\\\\\/QV+x4UPq2+yJJbUDkT0oIz2QqWLHNUAlCX7EhHvHi8PGwNpvBQKMoNKFl+PlKYCqStvKKeb8c8YQMK859LOpbOm3HfxnAJgLWGTDfrQbjGPW1AxcuuAm0r\\\\\\\/f1NRXQMwNC589ViMlOaF4\\\\\\\/FnXe9nSTISorn6cUOjC\\\\\\\/pcXpGvmgg3r4Z4PXN3Ekzfzb2X4fOMcB6RbBXqcmHTJCPuP\\\\\\\/k3GESM\\\\\\\/1+plRtzkLCgJhqfFRZy39O2bQgH3ZJuAiluV3WwQYu\\\\\\\/8DrrZsEaDunylhOpzvtmCcq00ptIyIs83uw29jbRYh1Pq0DsCZ5LDTBd2\\\\\\\/O47rJVWjclzaRaIjwv3KFVT4jFkZSF8ArNoM77jhTfFM45I6q32Dw5mLY6HC07y3h4cojdSNwdfnF8DNFgzKlNzjNygh+qNmykJG6zecwepIFo1Qlvmn1nQMBMQXxJld4DAUF2BS69qYt\\\\\\\/9Yai51LFsF8xrpHni+WysVeI5QXS+rAtey7Bv6DyOwep2i8KdO05lf1mUCohjUoGzOK9gsOSFI7ebnlOugfa+VNGf0T8LZ\\\\\\\/LnipWG7oX4aN1ADH8dJiqbIOXFl++8FjDWdN5jxnrgXCNy6+l2vOeIgtU7PlS+KbeKGYpz8\\\\\\\/uaoV8zDD9348wIj+Bzk0Wja1CWM9dLRihJ9zUqpEPrz590rEf+iqmZwUYQIye8MBEv\\\\\\\/WiKw\\\\\\\/sWQ3b\\\\\\\/e1mTA5iMRcmfqiVihK39lsidudYE13nKzMy95WZ4eHMDkE98+uWNYmKu9uVyRd4tlKdxVOrSkpPmUQi6BrxnY2CuwktHrR8T2T\\\\\\\/ErXeXo2cmQui6v2GPKFdH83KvSdkXF5ortuGl8qofPoqCOX\\\\\\\/nxcEwwvlvp4ccnAttSTYw78uDWauyaxaGy0ueGYOvleO6T8158DNKiCv+B+\\\\\\\/Rk2++jyTgFsk4M8UGuI8IL7foCwlTuv6ISxWr25sucNs+bKWWglUPWdPGSqNAPfkfUIrWXjYgPO0AJgEeXY8aSx22PNiViIDZfggBc9Rc8Bh3osgOQnt5Smi9\\\\\\\/K5fH5lsfTDgBY1NJWKu0VbJb5EgjME7cbE83Da5OcvtO4lIqguc6kttwpw9zq6xexfR2mNMPqt1q\\\\\\\/qMTOiIgnCSvns5zfR6K+v6O0ic7IgvDleDV3gBKAfYIVYWsrRI3f+XIb+CBHskhvviWyxh9bWJPzNoMO+zHbFmxj\\\\\\\/q2WGRXYkgohTYrWW53KKfKcxupcCJFkro5cGEHT4QffMHpeC4cE+EArrGZ+3bdwNVQ\\\\\\\/Quu8AL9hPCrBYoAUySBfkuPdyUUFGFRN37GA9QykcTxEqUSR++w0E8uDcDRKACttEGxK3h36BXfLCO9ISMV01QRxVAraLdyvR\\\\\\\/nkCxSRq9xWFiAqbYIK2KepbdocWggfTeyOSLz\\\\\\\/PO21BXeNjPiaGC7X1CsMMgQch+NRsIavrP8f2vVylbW4Gc3vlhMVZAMBs3YqiANl1wWkSHQI2Z177R21ruVkezcMn2d7C3b2FVjs37faHHV1UJje6P+jeEQatqNysRCv2nO6wCKuCuOFW0Ad2MG+wSocuUNYEC2+2MoCfHHj7RK9blFcgcBsmHCJLgNxneo\\\\\\\/IghDRXqcWf465IxyZGIJsHUFQif5LFeqn6kwvzgQA3y6PgEWU7D+BJXIUTmYFKIY1u+4WlrKfzeUXp17c04SOFtsJQxySKtaNcve5wWYPnpjMPXqp\\\\\\\/tva1fAvsaDUEF1CxKyrD7ct\\\\\\\/MRwX3YDbov7WrMynuf17UBCosoqRtEpZZf2L+XniGGAPWQAmJyqJdZDC74iROX3Rgu9ImSzNXbpZEQ2eiCfAphDKEVPotCbiw1+3lPFiw7i4BaA4vnxcEDi6dT68L5b7KYwUQ2YfyEXlANvhvFnDZqEs7XrL3\\\\\\\/T1HtUWq20a4uL8GwgKZqPyJ+2Ui\\\\\\\/zy\\\\\\\/KVZPTj2A2CREIwcaCioVmD2Y1hWugR9vDAZSXkME\\\\\\\/GelZ3h0v1is9LoavNO412klrX6H\\\\\\\/T7XgbtY579Ezuq6fQa3Jicjdjr4DlCkTSYBAi1xb0BImI0S9UqPnAI1522co2Dfudp7JgoZG\\\\\\\/m6jXf5\\\\\\\/RE7Ux5R7GOu5VAk8b46dDw7e3UffA0\\\\\\\/x+haLYf2cR\\\\\\\/NqGehCHCRBlBACH6FD6rWjt+remXwE6b+B9j6kQHOy05IUfYrfkSZXf8BCeAGFlCGtnJd2f4YDheDaVaPgnnAOIWBAo1iIW9I5v\\\\\\\/Fg+Q\\\\\\\/nVc0GVqncMH+52kVyx44VXllJ1+PtHv62SqTe90TYUcKr32JKE+iCTVW5f3jcyB\\\\\\\/dlT+1cEib17Ug\\\\\\\/3V8ByeNwcUQiaoavbEmOmAMbKpltCDtWqDGaxR4tUaMUU3yRCYHWdUqsXiQvWrwoQsyPXxSiw\\\\\\\/490r0ctVs6ag0bK3P+Q1powY\\\\\\\/CtckUDTkSy6SesNHN8\\\\\\\/ssL\\\\\\\/MtDBsGZkQx1GiouovDUqIQv3SaOf1mzlgvo1ZCaygQZ4vUYxgbKJvQXFgLlQDGTyEAO1j9EoXLfShaVhEAKjytZ5LMC\\\\\\\/6pFxJeTKodjpxU09pL64vdw9LM77y7fSgg6kjCIBDdg+B2DYxCkoh17eOaC04JxtQrBk\\\\\\\/RwcRodHiv5M\\\\\\\/24C37BhXICSNL0KU5kX9ktzxNMun9Ypqct1oplYdGMOGMJQaXNeTdPeMxtexpA5MRUlmC6+fb12BWaiGMbo9mPAWyJyuake639KFEmG2XYJYXIgJMN4N\\\\\\\/GeZ\\\\\\\/rsjP9Ttki2keK5AyDpWb6lWJvfVSbHArArJ3RmJAFaIKtrwL8edEwXL\\\\\\\/DBGopYsBM73uMxCoHiMOdQD83Rexv5BOQMEde9YI6s8JNeH1tK0YQMx\\\\\\\/DszPVK8xo3IFSoImUJitCoJzElXa5vXI6UcT70ual5wkfDKOYG8rqsWBw05x4cVR7J4ln6RtgsWXiovrzVhe7\\\\\\\/9C+tC6RkkNxD\\\\\\\/PUyY39S8V0kaqqaQVCpFLoT+WXttIzB0y++C8ral\\\\\\\/Y4BgvfAXgPf+jk7qNQwHblT0dUPOZyNbEJV4iGjyqG+mogf4iLwVaGFVK0OajqgAgk6YlRc82zItU0aK9JSPwXIo+8KIUyYL89Ifh9kipXlssMR+leQK7pLL0JsOIv8LszmXETdM9uF1S6JV0Wce8LKqGmovo4+Ia4AG7snAYQ2Skb9VYW3nEBKBNfNT+WpiSrx1CNGf3iCs2PSg4DegwQi+8arWYCBAnodaIQAqQiKRwCnQGxD2TbllgtrDxbCAI+m3Mm1RsD6\\\\\\\/jxlLEhcv3A+dPWuHjaObSRsh82GXu+pjtMXMSPfQjzQUifPI1z3Srdlv2U9bMu0JqCvsRJW6J1B99tYJF+7sOqNqddR9CavPdC7Dx6850vhvjcr9eC6nhPqpjIxINxlA6FWdT7MIAsRuKlynnQ57h+fXrfOHVxcwlRkhrGEKJJq9Do6t+9RJJ0CYaocrwgupT5SqvUig2Y2SsSv+bk0Io7haaHL\\\\\\\/+43VquKWW1xNmE3C169SzXtUf6s2nbo3U4O19orFMfLdYpQq\\\\\\\/s+OxpbPO4rvYTLWvt9bsalK+ZSvrjtN57LlH7\\\\\\\/haDdSMsy26JShvykdIbKeMN+xc1J3fkLtb9ikKwR4SDIZ7mHr5ZwLe\\\\\\\/g5bn4IYToc6FwxHYyjesM5pLtl94NwrzDxyDTofnnJcgSjiL1mpvBo9Nxrd\\\\\\\/iqkDIKey0J1MV28rUFoAAOVJ\\\\\\\/\\\\\\\/6PPuE+rLSK9VcHjmymY8ioRxFcoV4eZAPxj32jUwZFD1Jv5iHPvRVzIuDbZHDnyTgaXMJ9A1Dg\\\\\\\/ztY4OnainFYSqocmIXtWOW1ztHQdKYSbBl2EUXXB4YbouVDn6dT1PVYiYAP9IKZez8O9wthRI8pA4UXj7tO4WC6xGl2BxZ3vpZ8yLiadB+r1TjUVyOPoDKwCsbKcqWtNuS\\\\\\\/4E5nROEldqgQSfJGjMACMpjxz+VuV2QBteXNexeyN7dNeSjuUkkvDlcpb3US9cHbUlvrMIuAsn6PrqcNbb0+7ZYjV6l3APd3QXoBAONdCEUuHRKnNK73dGYMAQ9atx9OmYBqRMIit0N3f2P+z+J7DqFGIbbDJsGMV2BONeLtY\\\\\\\/okhdjNFAQNhmBF+xKhPSVGBuz7Zmm3HNSX3Uip1bW4vhpqLl5uNtG7P789BDsHCvFP9fBs\\\\\\\/BvA8m9ZrPb5K4kjb+ZNHYRpv+dS+0pUL5UCvEAejA1bnqywiOBM8H\\\\\\\/i0\\\\\\\/uLzOvYXHlmMi2Q\\\\\\\/dfy2bM\\\\\\\/Lb\\\\\\\/h2UHFHPU0bffZNC+QNHG6vYp1XyHMYpjVl\\\\\\\/+9T4I8h0PF5l013JrU6z2t5qWRaxopmZsP92K12u2Z2uDDkPYc8lxYmDcuWkUIRR3EAYkTfBxD47MNZsFYhNgxkfbaxSec5Ycd5I5hVpXZg6MTf+1EmqsFSOHJA+h3AE3I\\\\\\\/QmFnuhESFRj+pNwHBJJitaMEsLJgWcWMgN+1u1aP1FWPGWb92DvMGgVmBVwyptFYITHGY2sG3FYB+IqpthZ7MQ6YfbR+rG\\\\\\\/\\\\\\\/ZyM\\\\\\\/I7toJ\\\\\\\/iepJ3C0nnRFdikHtIDHkzzWris8cxShttp30RsKUz5JBZI+nT5Oj7NwuB3RIlVj19Gymu5MUZvnA1oU6bJ3N5BtKUj2WmVu9G8EZIwgx3\\\\\\\/203y79\\\\\\\/2aK6MrpQmfY704di\\\\\\\/uZHIjHsWox71sw3GAs9zqZ3WT0BLCWBtMKLbqzw6oGTLbKwjjVwmN8zT5HBLQD4ev\\\\\\\/19rX6m\\\\\\\/omezRrPMJyNYEV8q+HRCOTqw6Bf9WcApjThJJvlZaPxNwkkYZ8RHiu9thClCILQL3M6rdhYFl3iJw4jXrt2bb+ejpL9uwpD4kPmGTUkoPlZkxdBJdiIFrMtGdHxszcPTJcJ3loBJB+5Y+Rpy+KknLPCq6NIBa3yB8Q\\\\\\\/SbQJ71diZojLHgAy7\\\\\\\/KjdG\\\\\\\/8g+jMqv+94ENPC7ddcmYN+0GpfhHqOoLX0PSVJbDJwM9Nvr+k9op9TiL+Yz56Efx09xbXNygLTp9\\\\\\\/CONnPkn35xSHybIEx1dTz2KbMBmwBSDyZQg0myw1vSvyQmMOtoFhVbu6+7oEkAeOUJYCEVsk\\\\\\\/OpPf\\\\\\\/\\\\\\\/es\\\\\\\/kwrMaA7N2EET3xoB5U274V8sEn6iOj0edAAqOuQrkvi2mvQNv43UmCGEbHwNquH3d\\\\\\\/AYZK1Kpz4d\\\\\\\/3+lQScJBYCo0AbUfAKbyTo+4QnrVYLXe46ssS6jsW45UVH6tNoVXKFk8TcU1PiHuMQo2aCxWoYhj6OfkN8qFowgPN+CiYz8Mf\\\\\\\/KbqvEeKQxrpzDHwg4Z4c817lcZiR61XLd0JJPitOYwtBkmEd0d4wacoXqeEDE65KYgQOoXkwNJKccL1qoKLOqt+o2VwV01iLyMdC2VXszsq8EXyH1+fY\\\\\\\/X\\\\\\\/5Kf+0yWUOwKnpkvFrxMv8gUPRkaqGPfmE574Nij2Ex69I2HmaQEyNPhAA+HJzYX3n72e7IW\\\\\\\/TC94pTm4hWCogXHRL5Qj2UpR126MCsWI\\\\\\\/ZW+pH\\\\\\\/rif1ZQndgLgQtLJtAguLFaswe45dr85T9p\\\\\\\/wFvAD8wtZQcK\\\\\\\/0\\\\\\\/TuSc8srkEbEh0wzjFpnbEcco77PbrMDQjvuMRassYl37pXcEvimpOqpoSLCulrKc5OW\\\\\\\/UohsFlg4VuyT9xMZjIiMOwdT9UmyHq+bVp1Pvaem2cXENAZq\\\\\\\/JaRXtgGmx\\\\\\\/ftslNmbzEUELnee7\\\\\\\/gtCysBh+kG97OIEggkb5O9mKtjlFr58b4d7N2ii1zfS5NMKQv9pCcUmkf63BtDqa\\\\\\\/HVnt2SCw+E9\\\\\\\/YRDleSXOY\\\\\\\/CxbsIju8ektaSkdG8P\\\\\\\/2jUojDXk5LH1WiRMaJxP4FELhfDrhYxFdF1Y0\\\\\\\/AtpKN+TUZQQOtXMJtPAp0xR5\\\\\\\/JDMbtBQrEoPxC16+5zgVfvtK6nequwGNfcZgsHhhPqucGz6kbrZq1EH8Ia4sjTWZgP188HX9inBVItsyLDCuziDKhC+T0j9CEEBzrK6v2I5+mSI9qc7\\\\\\\/3YMvQqS94o8pqLIphxee7txDuixbVacu0KnjdHsWmozNvi7djRCS7w3cRzAt3oVt2CwcGtdB0UnmMopdol31x4hNP5nrkpLmJsOWaJ6t3eMEM0jbBHVwTkCCLj5LCksuxpDxV9bOsvKYvfVIj9IlbC+IajET2dg1lTK9gZr2XaGkZUpve4hE2y8i2X3ABWRb2BLtrkAa59xrd2035cW8hibJL+fizkgxp+\\\\\\\/UETF8y275Pma1rNGaLq5504C8Xy3+ZSIlXFXOLvTzpg9AoYec9gbReUdCW0nx00QhBvWIe9Xj66LW7TOuxY3I9idAdYEPWeuxfeIaxmcbJSpUK23cpZ9PZkR0Rt6WZzjsFSuMYlvmA8tjdfHn0Wz9SB4Cc3bcBRK0N+qGiPNk4MxP08Zr9oPqEvpRvbEx1QQ9an+k6OvbD1vU6abIcknj4J6fIzoevWc2jfM1E68YnYkgH8q2UMvL4WRahiUaaUtd\\\\\\\/VT6\\\\\\\/M6ayMdmopinjudUlpHZLpICIUnu\\\\\\\/bDh3lV0u6umYJ7szhiJmEt\\\\\\\/WRWHMvgr6n3mat4xIPsIDvAEVwg9oW3gursxC1hr\\\\\\\/HoezbkOe0eTWMq\\\\\\\/rjdZnhQSCLF38X29+9Epa0nGgwkBWdHVq6DVGs6kqa4zXNdTGK8KPoaRtCOijplrzZN1eSIcersbl2DkV6HmvEA8sp11wTCX5u6FHBIKdTlY1mpXEdFUiP1JIlge9TX5ZpU9nnq8RJMilXpRhE0zyeOBjQp6JY5PIK11QQbDKne86Y9AqIhbu+2741OczxnuY+tFTmqeBDI+DSPVxVlduNOFNgcfeP6HPPQoazkTSFtfP2Vg485WHHmt8FKa+l\\\\\\\/c0\\\\\\\/\\\\\\\/FdJqQKyvnJaE9qPhOjpFecpOOyxN9WyN3s5DMaLu7evclAV1L+0M5uNiogExyTgEszCMMBQM\\\\\\\/CMYzKw1b\\\\\\\/PrOs76liI0toZqDZgqdL7fA4FwSfv7jsVYcEEUhESWVPACWBWuq6nDaJfggtFxLoqJCuxodxNAGtFWLYlIFccrMrRQWtljoG7y5E470+ve0lghEVr8vFCQDLu36xvpKuCPvvTyMdPE5qz0krcPjoESpn\\\\\\\/+e03yEeG19EYoTz6xCEXGdI3nuUCAeRlcNDE3evYXRLc6T8nMrXE2ugDVJhwA1CQWxYhH5NIeSysxFFBTwdqOBQ1r6ztDCw\\\\\\\/wzbvmnlqhEeA05UqLXvD58Y82YssvEyFE4QXk9pKrOC4RXjCO\\\\\\\/Iq40YyTPJ3wT6\\\\\\\/aXBCt3JRNmE3tU89zPxL+5PzjGnr\\\\\\\/igZG52uNx431kQ2B1iZJvZSxrwZzGhza7ICW+jnlJ3AF\\\\\\\/hStHRCPZ\\\\\\\/O13cVRtHXdXsWwi730A+6wwjkjKNYvdyUv4t7l4nY4SjhoZYwiiDxpA0STBxHvioZo8f3q6XYVtc+xrbW9s7WWvOeNkdVWS16ucGn0qhhAV46XpHezHzJlD3rKaTC34Cm8WJkusSW+0fN+dlzithePCsmLiOyFSQPPaHSfspLQnTKqnCOGR9ksOusvpWg7CmZIiNoyVxD+X5F5Xqmvl79YvcaXNnttiCEChC4QvRpnL4801IboGeI8HujgDzjMrM6oxlu1r4yyrZiGBigGma+BSHdwdRdrvuwiyMRhu12HowM8+ib+fRX+BfDj9\\\\\\\/tv8zoMXST\\\\\\\/FFRXhfK4N2pJy6paW6E+vljkr4sPDo\\\\\\\/TULi9\\\\\\\/ZJjZ0YCD20edsiE9nxUxxRswGnkd46SJoETuCwVkfQodSw4R6xjBOdvq5Vru5a3KOff2\\\\\\\/4UYQejqW48FPKaXCJYbOTjv9OB+LyI1nzSamtYGPiJZrxsby4KJIkBa01mspj+qwrmNZQe6Q3qDc5Wg1dEp7zIK2X5abZGsZObLo3iYw5bYIJpPUyE+5ItIT\\\\\\\/ezLQWMC+4DnGQQjJX07b2QkbQMrpeD57fVLOU6PSktVZDW5NXZ1MbobKjlIkMtEi1cUXG4KgBK63HrqZkI6OnBiUD616560P99EcNyAL5UF9E4XtKKHMtTyjnlROef9ZihPXkuBu4mqct6Jw0juvWmmNuG+FOTA8siW2wQHnTThjAXbmEr8xPlHgo4L8GfSLDtgJnt6hcSztUVxdncCdlyJL8ZyIkwmYwiYZMK9b8D9d+BgrNlXOJD3j6\\\\\\\/lVms1wri3HTX\\\\\\\/cShKk\\\\\\\/gsJyqiRLaQZvr8BvejfEI\\\\\\\/LbBQM6TmQmApylLXl2BT8\\\\\\\/srvZ96RRMNMEF2VnPY81ylIaQc2W3E3Y3ahEhpThSwqK6CZiRxLq+c2L7bcAj65Nx4uY+mKnUCb6IKM6ihuzEsHoNqC5QAyoQsnMbWyEUloDlskofUizKGUCNZt4jFomAuvU5efFJ3FAEy1+BgOAJFO686o9R5gdX5yEVzYWOA3XUJ7NHiRsHoYa3IqLLn93fLhayk3cnUe0qHOwRwvvdsurh\\\\\\\/kqmd20xieuOolDmU2aOxIXcRj5cTF2zm3moEbzMBZIfD2t538CL72VSXaNPd87vpigSQpv0H0z7aA\\\\\\\/DJCLt\\\\\\\/hMXsOP81Ia5B0DgC7skQCFkOlr+n8XcUWrDHi\\\\\\\/yvXw\\\\\\\/55XROcgFZrLLTb9SE7yRBlSeRr+b688xSeUqC5JI\\\\\\\/J6JHJjOVWsAiFQi09ZzxGy2jLObgGOtEwqypJmW8z5OjQ+rF6k49LcDClwFz3evjM8pGes0zS5dlFoSiVou1V8H4ECPfuV4CSfonsNv7dxgGF+uZ7DQzeXU03Z+xLgqiC9X9XWxi37yp0PnvT06y\\\\\\\/yktNQHxA9Ykau9ze+2O5dhpUn+TsgwKURXhq5MHAAUUPQuHak7VDnaQGzdV73cErysmaWVdRIxB7HqdBunRQextLHmGNWlp8y6N+KhT3d9oU5sxBv4QA7A1roI7p9prHvuKSGYuL80n9f\\\\\\\/G\\\\\\\/KcXgxkWBt+NwD5KUyXbFkyk\\\\\\\/8cslgER3duPFwazomcAJ5YS735EGNuYsnarWRENFPcNny0wAs4XaCgmAh83qnG8jA7K7mpsjFiMSmGAmHh2nmOKnIZTLmngWsmngBozA4LpI9wTXiJd89XWxQ2S7CGHvrHIUfxTODQJ4KqReV4U3MWe9jw6+4eaoBdL77bAnX5zzqxxfx9nSXoG589mI+S1Tmmzg+6FGwdMaGcgjIcX\\\\\\\/2RM4t2Jy8h4zuMWunTThlKb9xfpBkdwcpu\\\\\\\/nIzJYtsExhpsgbWnCRj8BGyn6jS9qk9o7snwOXD1HLTIibjYH5Kjrw6kHwlCd\\\\\\\/CJnk8kf7rlSCM8SP9JM0dlSqIre6y+08XW+A0x\\\\\\\/QggpdVFZ4lIuvtLG9N6TKxodD3oweEzNZ530twHTFY20+qIn5xWWqy4Fd3aJOy5dozDnvsNeSBgtRba4QI7jZr49PwkKmoBV9Yz2OPZ8K74\\\\\\\/WSByCGUhMifI7YdZNjhF0F\\\\\\\/oZUBkyr3sYTXbV6gyGwDJck\\\\\\\/nmIti2W2phVRMu5v3cLbAi+dHlY6ojurn6gsmGZeI7OYZuCf5AvoVGfSWmHFT08J\\\\\\\/2xq34SkGYM5hYJ8i93HBurEvrJaWziUQ6s+rKssyJVv7BYwOqJT4kOKr6hUxu6HOxZgMND26rmQIfpmsZebqFBSSPxOgvR4fRVm7dGwT7KhJVIy71klNlH+683jFI+4YhnF1GiP118bGmIFdcq9Mkn66TqHa051rlSjWtaxy58b4zkHpufCQghIspjlASNEnqX3cnNU1ukn3BjRpR\\\\\\\/EYZnD0pTFPJuFPGarjmZGtm8GCYZTnQRHjh70ALI+nfjhIb+lyiso1qCc\\\\\\\/zow5jJz\\\\\\\/sZEhgljKr4RvPsyvMSQy2PctZyTH8yNtzwaEr1RHjGiDAFy2zfXkBmu9Bxvq69Lk9wKW38EDcNEz3O8C3XyaGYXsSE8Eil47Vx5GbbgcV0WEKc7SC0OPaN9FpkSf4twWW4Uawdjx1CcUL0Z4XdbNQmplHzkDKZ7RIhSk09D0k8d8ipHuHO4nyCi0KyN3a94KUhPtZfzRV3p5m9IA+pTRLWBx7QqqvoQgmqbtxv7nTmApVu\\\\\\\/Yekq9JBuw1dT68loUPSaE5G1Z0zGsrwoz4j5lhe+WkfqwQf2HmdI0OWnXnT9xoYQfkcPdwZEgnyAOBvNss8mlDmRioreSfJJIRhv8fth2ZXoNe0LXRK5RQs6yGbLRB64ONIWPJfWG3lyM8DMlCDVBESNHYRJUUHxRJsmF5HmyCCzk3Ohgo81gihJhxSAxpJ1PbOXEBPQOzyZqWBjU74RxBCM4zYjIrm4UMAZ4BgwWlIqr0FQY3ceO9rwOgpix0RwwwdBsGYhpN2yNe4BWfwBB81ARAvvTGKtQcYekSyExAOxK2IHmdEqqMYOFDUHwHCINEkBFtB0ZYYwfqcdXaFC4Eq6uwGjhXQwhtNUQCR6Un0hliY1puYt2UgAnQBgFrvsTqMc2suFcoTKrs8VjaJ2\\\\\\\/eN6B36WHS+l1k90UOs\\\\\\\/vQZ316O\\\\\\\/yAjrf6T9Uv0qyeEtNpsTEDqbBo8lCvwrCNwLOHY+wWHGBba62kFnAeAoaoGwHBWt9Gkf0FNn95FSnTVv3WGHR3kk2F2O+Ye2iqbw6r+TPEfxxEqJBrcUiAWyKljmOGH8sMWd7uUCJ0CkuV38POwkB\\\\\\\/zldHEFlJSTsbJqUH3KPyeKn6fmno7YW7mwc4N026zGhzZWZCKgQXQlJzCKBNZYoaO6BzTX8eQHcXzCRmE1TVscqNK3kIMst+JzdLAPrcJXdQKmVnMOBAHwS2z2UGNOYlkN17tRzSmbcNuIqI+4fNU\\\\\\\/A\\\\\\\/UajbFJQw0uNBbfxCTumZsFO3SjGFnIPkOWH8lm1yq3g4Dd1UEAQt5A7hdcjG38NoJ7hONm4xGGEOR305mrKlXYLKxdWzQn2WtM3ZRIk+S63f9J\\\\\\\/ULosCLQWd3SjOWZWxJCn+tqTO7CisfQJ1ugQWnHjdbAGDcKWW64OTrXx5AddZUX2m+yyDGWbPZTWYsyHW9dhprnzC\\\\\\\/tT8FtGxpvhF5UyyCIksbcXRsAHQgkUIYV6OGv51d6Dk0IDfLzGeYhULBPj3fJk6BJaF+ttS0aWu5v1WXTbnxLZGwEfkLxrmFXwsNbSSmru0L40ytNZpmnJQX5MqovO9prwFTKHlTYOa4hJDgI4gWZgV0gt90nGwfoYHQcVLMSRLrhoiu2Zq0joLqLGRMz3D0l8xa6OvjBrTCwL6pEmX1SLFWO9b67NGjHSgrUvw4Unj+OZqzFbnVCLLDLxta2j8IZVYY20nxvKSZx4L2AyRAU56yFNHHkRYjI7+0L1Vp3s95CTywH6kLHtLYaD8Kg0djvjhsaYRmixnbQxyBOT42Bz8gB7c5hguxPEoWlKDNzmpCp7+ORnG+GgY7QFMsnLWbADvZlMfoUMRmsv+Ck1ZHibkHE9122s7cK8HXUg5fWjQOZikVR5utccoP1lbPB912dnoKFHPhVAE3bVRTtB6y+TK8qQsWWWRmnjtZDcUy6NHPkYoE\\\\\\\/YrTvOEQAACh3HGjwyqDokNKPd\\\\\\\/vcGttCrw5CTsOhiuP4Jms2EPYkGmB2fKpCUo8K+HFa3IPvbkJ\\\\\\\/EPy93DmXB+81TVncnuRucGdM91MDI+FhKfLX5Ay+SVWAP9EfnqptbT4uaty7Ph85scaHB2BUcFy+wMe5eHzgPxbK+Hfy3OPm0Ef3s8URNNroIiSa+CUGTSE79sm92iRw9v+tiKuULZTgJ7OvPMqq10O4OIDWbPy2vk8Km+SNHRUsuvrHTxRW5Vy9c+Aw\\\\\\\/nVGy7u6txCzjILSJKCotn\\\\\\\/Q4F6+o+PDH2M9\\\\\\\/0djd6uWmk\\\\\\\/9vfLXT8zTQmp1JTvrteuruiqhV7AvalzjDckSdBE9NedsQAV+fVB943clFCiPxvXu7bJ68CVcQgl55LGeO20XDiinw+1TS4F8ep6tp84xG28e1CsGcL\\\\\\\/0Ce07JMbTb0b7czghar1euvaW\\\\\\\/SuNfmEi5bBYNjo3urdhttSdf51Cs7IVq8DybnY\\\\\\\/2f654uy\\\\\\\/nDOoKda\\\\\\\/ds8WBXT5NJrojtO1KadbBPoR6WePWs9NBBT\\\\\\\/fJb6ccuJrMk0eNWqBuSY0YjTNgQQhD9auXNJbAkfRAS7LfS7t6WBb1Bfpe5\\\\\\\/8mVVEL7gMYsKyk+yL+MQIVzOt5ZC\\\\\\\/X\\\\\\\/PkS1r5F2j79Ssomr6oIUhZ84ji3QbS3cHuwB8XlVQvgy2TXA3KUZ8j5QDKZYUSAUSnoSZ7SmHApTHNx4LSO4K6lF4XV1S815R1XBL80+R4iqDblfsMO7EpFuEG13JWZZ0XaPflmgPJxke2T43isu3Yiy\\\\\\\/WtG0z4F39JwZCLx+7tBYwy41dZGH1P1QcpxCM38fni6VMAMdD1YnqaifOh53oByFEv19GHuVt4p2\\\\\\\/+flU4pS8J\\\\\\\/J3v4nFLbrkaoysV3gf6HZgixvlXcIaceEei4biEcB05a5wNa0bozg54vYS3qshNSR3TsA4MvD6cV8E4iO2Zkf8Zi78k\\\\\\\/6BfSDwK5zPoJqGb2n15XyRvFJmVm1hGEx9IZMrn7jXyQaHwQiB2rsmM1uQeThraWH2EJaVhdp5dpLT0yFwT7OGaQnF9vxPD38vpG9Ga20tae4HXIvZpdjXatFWG7pV3dItv96y5SgtM0NyAoWw3JxYtNN8uoMeLScJOPtQSXPVJBZg0BPDrE7+PqTS0jNNY3GXp3wJdN1ztFmygQ4lTe1ms6GHksz0QAv2wVc5z3Q\\\\\\\/ndUH\\\\\\\/yWUntDQp0ZN3Fv9HAhmt1uwBh+LSl5WpUevilxKa1lKc8XflLgCC5PR1n170o3Hm\\\\\\\/6d\\\\\\\/1aphi3jyfw2RaECEZqyMzvbQeFMZ3\\\\\\\/uo9jbYUWHGO\\\\\\\/8+53MVuiYWLL8orbGsqRdQD8dSeLLl\\\\\\\/7nQ+2lLCThnnyqoUYeQJ0w1i3fxZESQZtZZbVLCQHnxWOsDmqJywS+1nsWQGUEfVbBD7LQFd\\\\\\\/6ZGz6QmjnlxiyIzBYE9ge70vfb7+M20flcF9jw6LYsiJDFrRH5haCcVUIAW5VjK0YNNjWs6vt+mSx1Gu7k5RkBgEl\\\\\\\/ZmHBkhfRFWVxz5RNdODKTRzHKuO9w9c6VxufgiUmNcD57bG63zR3Enk7fSBmTBVTdr+ONnbcIUjh8WiVLZMZWMx++ZKlZNG9kNMkobsd+J4Sc4rFpmhQ6pyqwh7slFwHkCe6fNL1yYs5z\\\\\\\/LPaKCoVJHaVOX2gLo3ArbiZmOntv6obpV0i2RVO8Bja9+LMKnX+LqpGGPwEWOV\\\\\\\/JBuu60inZLtDo0W4clsZ8YnZC4pglT8rLDexOYV8FbABlsvreyak9uKaPCuMMEVsA0Qg7XOkxGMNanfRxYUq6K0V4gXkNPCZfOgSLWZ7eYMy3S9kze\\\\\\\/zoftqBvmgoWVS5WF34xbgCzbz3dFROgfqACOQV1ld+epBcnZ7AQ0ghtChwQUaaPgbu02nyiNdGz0fYkh4xV1yw1iWm6pTaxjzmBKNDLaewe\\\\\\\/mOoaMgR\\\\\\\/DiCiaf\\\\\\\/c6EZFexY6bJumHWtGi66fC50JrNW5XSdpvE7UVMNq1l6Cc4slvRsS9umcCFeIWOno27x7HOy5A\\\\\\\/3mERcLqKUF2YUymtpPgmSA5AeAm8+nLsbG+BIhFF8SXOMOcJ50w+Jvru1isnH\\\\\\\/eyWQ+HtaeW2D7A1ckfDBiXfNNxp2bbxK+DdwxfgNw3pUuqSZVWOFRlXslqtMlWhzRLDPZW9PzW25cYOHe48lY6XTs04TsQpAZN9PxJP8ihZwr6G4BBsGP4a37PZtoDtsrh5F3rnEVV\\\\\\\/dFStqrehr2KbpGFdSq3tcEJtqLMWboGvyVDhcfGHKS3qVuKbTIX2JHvZclETUsHB12RyVfKLiGuG8\\\\\\\/ktcKVZEC7VanWm6+tnRU+ZitMAVHNvH90mWjb4Y0feI+Zu4nE6XipHngO738Pnw9UdjEBDq58E8hmWCNWWzTzse0244lAMt+lTfnzTp+pxWnnydHbqSesWD0QaAy12BcUcyaESAeDxSylpqkmBD1zh3uKYeCbt9RWlScS2wrky7zgH5eDOUoooTMnFKmtsgKkb0TNIUuUyQgO6oqQVaPBcG43xIR8HbzBHx0CTWvW6BPmdq3NCYsaMHncsTxbgxWuIq87Gzacsxt+ovIwtOaDIke3jd3NluktK7p0dkoFUHxk8O7mgX+3nu7YkfAN4bEeXjLXInsPFFNO9opHBwQCm6PRa8Qq9qxjibEl1AI6pxADLrS4NrRuOB3Beeyo9ZNhxDz7MpEgDxdVF3JV6pAr3V8HVyd8DdVk42IK8iNPG1rX8y+O0eLuTwLiFyfkeN80svNG71ZHhMJ0DOZCkQH3LyDQIeztixLAHr2ESEjYT9XK+XdETmvDEW2fp+YqHkLZOPrRD+8fLmNbqgS4IKDc2lAqH+6YSa6JpT9g\\\\\\\/q3aTkkJHmBDy1Dzp8ftZl3X8uFxs2Z+fDl0C3Hvg910sqIaMgPEGIGoppNC+c\\\\\\\/bkU5plf1trIcXBNiENv\\\\\\\/RNvkzxGOaPJfdsdr0cfWqfz\\\\\\\/u\\\\\\\/xy9baftHM3lo6oLwO7r+Itgryl80xg4cOJ1AmdqHxxepHg2X39mK6fmdKoU4LRg10TBqachirB3WEPz2p1X46SIBWDFyhckRtGgSwDUGjHnoq0JsyxnE\\\\\\\/Oj18+CFtOEBd4gzra1dzbNE4fpBnHw2dvKRoPTS6eqAq\\\\\\\/Cl5qmrpYag3agIS7CROPMnJGdIo2wlezw8csHfBwvT1jWYBu+JbRPhvYk6szRw5\\\\\\\/2SBG7pbZHTzJQQEXu3Q5VOyTNuAoURD9I7ITW6IN5eWtA0S7+5UVh40n\\\\\\\/AwOrlhURAKQIMtSMVJPOymla\\\\\\\/z27AsydHZlKyeVq+gAEmovOEoMgSyp9qkvDDREnby4f\\\\\\\/P7pLSqvy9t1bCwfUF2Ji99YsCJlz+aIV5VZVrYLy3enIbVEntUIdfs5dK6S+9pItMUCCdGjdvX2slFNPMmVpEHfW+cF1P0KgR8u+1AHoaxQVVyDg0p4JfQh74XNKuiTMjziUZAL7eUg7bYdr1VrrLU24SgIZ3MLulZS5VTcwtkXEx2qxGUsV5Sf5xWSUOjq\\\\\\\/fEf\\\\\\\/1q2XpYXAlfHjabpRHl4SPgi2R6T56bxH8Ky47kes\\\\\\\/TpsfUvGfqD0YAEfqKXQJJA8n4lWYfBpQkBk2Y8wt+EgqYSlli7+OigHiFTT6qMS4XwTy88a+sodX4HX7IP5YeoVisoMuG+\\\\\\\/DqMszxesdmemupz28CEdgF4GihrEAVKr4zAbnl0mPq\\\\\\\/meiTcZLg9tYSwj2eb9OPGxoFhLiajRX9vEDlS97baKGbVjhzm4vIcM\\\\\\\/kiarSfcYu4NqtleR6XlPIP56yfSWBJQhg3K\\\\\\\/\\\\\\\/xygV+ksC0kKykCKlRADgPDnAiU9rCN8goru82PPIByZk2sBwV6b22Sltd9WPx930YteA37ACC7TCbASrn0XS2ATOqkw52o0l5Dz0M42QMmuyn9SHdyEcPhAt7mTkFWg8ebqs2XSLjxc6I6UfPVtmcw1sGgWaV2PyABK0ME9GFSC9FjIwirwlKgyKmTmdBayMxVvaYUUd0gLTLzHP5b2iIUvUzsKz2P4zn1R2pPz+SnKAenmE8+vQFsLeGGD3P9YFLWxInpiGLDRXTizXK6WBhexfdupewf5QwajC9kCNDNCvuABn4usB7ms0HuFID7m563EnMpFn\\\\\\\/6ah0cAYbi18M8SnGDfGcFLtbY\\\\\\\/3L\\\\\\\/UChTyaTJzlZJOiDy4lWXyIiZPpX8Ma0bxrW36qIkfHy5oWapW1BQxaILoOSkwIRiejeD+JMyb72z8CbLk8ZGn4npf3THz650GVC1uUjYZpoJTukJaiGWL\\\\\\\/7J9XZkKTkz+JNy3jaiudQjssssYS\\\\\\\/Amd92Aum2iMOyx6Cu5I9n0GOlAeXMG6jPEH\\\\\\\/ZGpm6iLJK0Mk12FvEgDRnlpR5OAMfqyHJzksx1vt5F9kImDCr3Wtn7PeLrrnvOBTBNxzIgIatMwABLUneEysm7Op6fccT9t6jx1NQR0EMeztr6dTyybqDZdhr3IcDuVrjuG0Ac\\\\\\\/8ykxhFTM1Hz+pR6YdQT0v\\\\\\\/ybj7paE9mMyR1KDC+In9LsNq\\\\\\\/Ifv9fHSA3CtpDUULV6GIEeUuYbrJtvNY69EeS59Nn+0j0MQKYB5fyi4sbW8\\\\\\\/XxG+xaqYNINpaioQhBAU8LxnCwReZiDmXe+G1x\\\\\\\/UFEW7nVkrtyZ307daNnX\\\\\\\/Ywzn5CPIrNF9YIMav2QhtlmzkZzwzE5Q7nOjavuEc6+Yc5Z7DzFweukVrcWt8+\\\\\\\/YYWJohb66myXuw\\\\\\\/xogA7GSnsx8LTAu4H0TNhnyW1VdK38Y2iiOzZewrsl\\\\\\\/dXHH4nkt3JdWyp2N6uXPKCYWG\\\\\\\/b0Aq4\\\\\\\/eBJA3xoS5RgjwAteF\\\\\\\/DNWy2ol6Bv+vAiGigIYxjb7eccaYtfHALlQSRmQPW4LGYcVbEPuqVsPdL17X03eRL6u5gUiMaE2Si6tPXxnXfpuoSI6OtZj7XbZ\\\\\\\/K\\\\\\\/6Aqwc0U7Kyw5Zifg\\\\\\\/JNjAdzwfk5sXpTeTv0idYMYakfsjqv0MFK3cK\\\\\\\/cJ+WNFbuAe\\\\\\\/NU6frQxKgV121T3J0M5+2VW46dmMzBvERxp6KX+PGySQ06k\\\\\\\/thePWPTOWtEKzWl+Pxw3JgeN61+JbnEXwJm+LuPJIae07RnT2DkPJBCF1VpmWnQxy5RG8yPP0qH+X0Q8j75zu4POVY99Wls8BQyGy+4zPyxJ+LeTWgys7vvLNqbWDg\\\\\\\/Nftxr9HM\\\\\\\/Qjd61uMpf9orWt05B+bVnezIk1qG8eHXcX3hhN0AAkCsxtZAd68u7DRa1PWn7KYilV3ouFM3sREVAQjs5IirvHi1W8+9fJ2iMLQcZABucZjMdQKU5H6fXMGp4rpLfk23SiHYgdYSIoSXnQiCfagw5\\\\\\\/cYqFqvD7mJGg+xirm7olm1jFSvJne7nAnii+jQ4HPzBi+9gfV7YUFzvlnpnM1B7vJI+d5YWp8iLDvMG+OVYf9cQRugC+0hAFx+nEpr4kSIMfc7WSCjg6F8ORmI\\\\\\\/pq7CCxe7y0KO3SDPlXeY1UWqPXfSij8bjuTqtGIbuqqmcek4yzblQPB2hYFDaPqpgPGfU9uuDz6jzmIbGBPqJqsQFHQ5uep2+6H3UV9b5y\\\\\\\/O44stig6tFdceNyrQjyYwsPRqhElmlVTA4fFJwvmRyX2tL+\\\\\\\/2crsMAYANUCtsg6C3867Cb0l67EHYLM8OIvV+UfXuN2GQiVgoNaTZ9VmW9tSfbVuqoN3OGP1mqsZuFoeCj+HM44GQL+xc8u4p+bYV02dWUhap10om\\\\\\\/CoM3lcftr4f0szAmz1kzuYvbS3tLeD99Ibex2mkSLQHarfIZcTPvTGCnjGj1q2nLw487DluDS0NwseIiv93MuJTYzbIAaq5957ETfFwgPZfKyI3UpAP5lXMX0TdbN0UqL724SmhO56lBRZ01pVwag0jgueJ1aBlOuNWzQveKMeoXftCa4V\\\\\\\/DKguIMmebVeiNteqnzFFBSmIcB5dlvVPolCnVqDwnDSsn4tyx9ScyuLq4shBKMhycQKgCm0WN91m74vuI9n050XCsZvK049nqiJiatFpfKmVVE7YpDTtSGAwytmPEGTl6jqSsTaieBoeK5KU\\\\\\\/ySDT9ZEVolC7DaDNtviKadvTtj+LIYk1qMMV7iJkusAGFDRaOXdx5\\\\\\\/OTg1fTYO\\\\\\\/H2NSeMwFBR2vblxQchVDjT432U8mQ0Np0cy\\\\\\\/iz0sejOW8RnKSinXKsTfSJWzfVct6R6qlO4WR4iLogkp+z\\\\\\\/NyovFCrC0Svyug3ByeE6pJeFW5ocTBEf4QcvUoF5tZBr8LDtmvEr+a9puAcxa58TEOUkzJu6xuZA3QQ6aofrWKyJW2Aw3ZuBEQhZFnjW5CmmvaFxXwKYhh7v0iZzrir9WkD\\\\\\\/7NkDB9bpv0ZLym79+13Dp+w2SZW1G\\\\\\\/kKDkWUrRku\\\\\\\/YyLxsKPXBDFqH9mzUTDRj8zPyZCo3LuIAKsdKqJRCdfWdfsARdEGA+IHTLJ9gS6478YxrOjFmYAHbBSwIxIyppuBVijwLlWNbWogI59b5ZrEo1CJx6XG\\\\\\\/5pkkDK3Eu4E7fjosvDzsxBcRC8fACIWKAJYiA46W4Su87AlNNXTIChn3trmsKU6sR1fUiAldJiTcjUOFSNNM8NFFYdVZ4BbYkymaw2HQGdrQkVaoB2u50MyX8GNUBmwOxwSmppj1MAlXLWqtdGa9lehyba1tCOKHcpR\\\\\\\/\\\\\\\/6z2fYCBO6DuWCV0Fp1krzpxM277gPF+XuhjnqBAS6vbzLjc2sqs8zYbV2i8pjuQaELjlQNs\\\\\\\/ll5d4eWmj5wPFY8REqpegKOHnpTcWLda1QeDvnpaLrkIeH4\\\\\\\/g7brkOjUloN\\\\\\\/UWSNkcIKkSQVuAuEc6tDIBqI8lz2oz3ftqwLqH96JjoUcpJTpoGPRA0YtyVVAoK\\\\\\\/9S\\\\\\\/viAk3B7QBH9g\\\\\\\/P2rIJX5pexNYJ\\\\\\\/NyJeA6O0q26LeCXLO3tQoC2Qtn5Me3rPHz+pNcJqHKebibBO0nLJVylXVNlguRWvrXxXHKCw+jqjsRXHamKivH29qfVhTBMHnZ70ivkzIo6tyNQnungVLI6ES0gYro0eNywKcmG2wnw111WERkNm3TF2+r2zROLim6v3Aky+rrhtqul7ezPeg634JoDIdq2vmA9In75uhfnzSB\\\\\\\/53eh7DG1275hzs44+BnyAgX4lHy\\\\\\\/r3WjcYwVDMpDsc2GUCFTA1kQ0MtxANgl1TEDRCw++hxvuu\\\\\\\/Toh2L8J0Ln3SqKcAiy1u+FT3fWH3j0YfakmDamMg9PlD58dlGdHt9rify8jC6pU39bf2OvE0omgsoO\\\\\\\/iHCcXKfePlnGhRBpy0EDgQa\\\\\\\/BlXK6gycGq3INxh12quGCvqVylGvHFoE6TvInzeQ+o49wiBulwVEAKOaQ0EMshBF5gmxFmBw420sSZA2DZiAiSyhkD66SbguQrF\\\\\\\/umDAAKPIpiHHhSidkQHGxnesNVsY08jsrmykagg6sbIt2duO5MmiQuoHFXx9H0eEyFpoe2OgNy2qQkAu7Rvrz7udqNspfAFLROk5ssqeSTWQDLhF\\\\\\\/5P3\\\\\\\/ZoRRJk7OMPJJxM3yVtZuPPdDHv28RuxDd33+qlJp\\\\\\\/lT+edADLPxiKYeUPcWSafyhpjbTAfryPpz2V2041rmVO3WHDTLCIhQa3FTr8Jmrj3YQA2LkUgzwRV3chVEyKpxwkaPgBLho2qGRRwGLxTL\\\\\\\/sy+8odcxjtu3+cMLcJQVwwa3fUtQsT3wwtq0MOaVWzknEP594AeBOuOBUTqFOVeQzZs6WHRdC9Puo2ZGKPuoTrqHNuk0jqCgSGSpm9rVIRXBxBfCXRhYs9zO7+mWQV1BrMBl8HPNvCoK8x8oN74MTNm4Zk6sUhtE\\\\\\\/gfXJT2fqDFbCdj8BS5KwZ6jTV5AcSIM0\\\\\\\/TtmHnOtXjmsYwofB9nT0R3ggHfTrheTu+9PPUwXf608g3FvlenfS2DglL7uysXCB3ynO4FwMAES9q0CXwSrGmM4+xaEK04CJ9PEzEdEpp2zCA\\\\\\\/HAGXgdcxPFw712Gs4UCuG4GyhLnWAW2+wePqJ6Odr75e2g1r6u1WZigu\\\\\\\/CQ7HBEL8mrRbC\\\\\\\/UDp5oECktD3XWBWxwJoQEUcAW52rhD3\\\\\\\/V377gEqPrJX5j8pCMXQ2xRQL+3jkm5Cr+xN5zpfoAZxtQ2Q4VxXKXNNMAKDwHnFk0LpDIUfhEN5iQl97qY4ZVukxjsnFIqGeW0Feoa0Z9i\\\\\\\/TbdD8y9C6YWoeetzkOKZyh8JLQxaRkoDwJevRGrlzhZUF4yokDr53oO4a5vmEBCUsYMQWXHrAdQ0sZ+13849STt2U7oZGUjl0G3XUKsPekZg4LVJDkpx10q05OAnz0Uhm2tj4CA5gUUmPZl+RONl44ZAsYEiv6uHSVEmS9BPq+lbWs99hQMIRQPi7H30VygnJBkdL+fVToYH1wcs23OtDBHS8C\\\\\\\/lXUiT+6Ywrcr6MObOg37oVUwjjlj1bcrle40uGChSp2CgGpXaDBEBRFXKGMwnBBX8oLQR4FcvV4CZpuaO0DUNaXoqzmGLonsnuHUQkqczi96tV+EYWJRerMp3R+bPKoPP8F5\\\\\\\/WKjoLMuZValey14GT2qJlLVYhCZUkW8iQKWcGnmTqfWdbXU0Yr7kPWUixcC0\\\\\\\/vVw+7Pb1V\\\\\\\/leFL48t8vFjZRbiBjLRhgX2rOuExhew0KNlL5wgY87+SFiXnJyxJQOHc0emrRyNRT3EFSs\\\\\\\/JFGhmXNij\\\\\\\/AnDgr2Wzy3\\\\\\\/4xC5RThBQU\\\\\\\/xIgAYyUk3AcyVEOfzWxi+PQ0qV3I1QQhdlCFa40Q\\\\\\\/7roDwfQ5\\\\\\\/NCHz1zroBHyq4303mi6PkYanGBnsHB0NlKjx401MbPu\\\\\\\/i5u4VCtHkCXQJVme1MxdrN2U7cOD1YCNjkgUeDrI0LdJQKiVqB88WJh9zAFg1WNVxE8vD2Xs9WW5OV+XZxTuwKZUwzsGo6OPvqgrwr13Y2Qk1cIhv4fqawBGwQOzifikNJOww1hhkp\\\\\\\/tz5sh6n\\\\\\\/yc9YrbrTAgRDkfSvXbKJufQTLMhoHd++METeNsi2DZ6nc4Aad6MSQpPoPoUNniKZRLSVA+UrKSze6RNUgTiiuMtqBP2mH\\\\\\\/WONrddsnm+7SC3hiY+UjVVl+aehE\\\\\\\/\\\\\\\/a62J9dibB7\\\\\\\/EJTxzB7xJVGpAJkGrPWy8LtUrDi6i8pEkElGH\\\\\\\/yC9MtbLkkUZMtRqSEKxdu3DAxPz9\\\\\\\/vd8F7pa5Bc9N8QL0SOYaFCb1dlBvng9VWiaE+EVD8ZAzgGN0k12tTB2RIPwx61kMzrAAgKuPRx1TuCgrJpVNEtUV3JQDui1OtBHjGk2dzwE4hLy+g9xx+26wjtZvnCo5Ze5YVA6NKthQd6eT4VHoJuSQhw7ICjFqJh7tspfF+1Hhgt0wpaW04PSUcx7T5y+z6MRcP4DDzno\\\\\\\/J2ctQkbFsKl1SFQkuDyC2NYFrZslxrcY9fjCilSicRdOqFuTHxwxUTrQx8l1pnp95wGPw123Ow6HihUqmNtXsuJ2feWpnGxs8Dw05WPT+NMfSg2rrbGEt+tc26p48jitfMomdjuk6eZRFYGspnYeKUC1vtCgazbpTp3qeyxwB7w99I0U3xc7fpYtgnduc0IUyqCaT5QGVFlWGeEpha9Meefgqy7slircz1hKrqFjUpqQUxaUbTduc5M2uegnoRGlGHXWcr9\\\\\\\/e9EZuVnWcWTE428AxUmTUq3Y2Y0I1najh\\\\\\\/Hz7t6U6aXWN+Ec4B2O4QV6pmox1eTZC0GNR7PWuRVVAiRZWrR+2yn1l1yVTNP6kinbT2z7nqvb22bt1zZGU6sEy\\\\\\\/F0I2wHff30wxmG5jNDIMmRzP8Mho5HVz9M8ROKypO1+wJexTxB9jlBk+FHGekkppuScDRyAxSsrku2vGaWfXgwbPRpyYzcQ+tkvcXzjMH+IRC1IloQD2tHJhC+CF8cCua+ZLq453moMzCKAz2jmCHh4oxPJIOqQcXJA1HuAlUedJZ2B2JCOYWMbNUTOt6pN1E3Vv\\\\\\\/3EDZCI+2AdJavMOMY+Wh584wJcjqxCxexRWbLHhWKm9OdCBih006Xsmt2x+XTsT5GVWGfG6W8urDHqVeAX04VfcX9zPt+DWxWOBoe95tomGsQnjKT25CpXOI+1bLvSLtwvj+8acvFV17jweeIoSVFB0lBIT0IqxL0cFZDmM\\\\\\\/2nE\\\\\\\/UHgPA3twh3eB40+aQ0JPqZo9TuG\\\\\\\/o3uO1rpZyvz94Yj4va83cYJHS8dOp4cm2kG3+RV5gVEP3QdYVFmhucJtFBAuSh6uu\\\\\\\/l62YXnntvvKwCtFKFvIdgW1hKNYkQyXjUpD5loHLKu9P3mR73QZvFG4IgAOXDGCzGGglhK5llweIjDt3viYBrZArM61S0WbPVYm\\\\\\\/liK9HYSwqmvfjR5ZcooDcQ2sytBG9btgLKqh8knlBmZQYjQ5AyOiSMcVpeQfdc3H1tnfNXk1xmb2r2MKTQsj9T3PmFPAx0cmAJIebro6nGwqbYvgy7SJtCGRgatV6uIo5V67HPgKB9PdkAHY8k4yLnb+HFaZ1hkUj87ZAXCiVAwUWCtrJiMqwoNFZRpMFqC3TfZgsoijGrwDqH+sOZLkqr7ZYvMpQ\\\\\\\/0oaIZzTq\\\\\\\/0ld1FsudlcpYL\\\\\\\/MIuuV0fEmFO8iLEvyhc\\\\\\\/1PSDRRc4yRvFhpTD2dOhnGVg6vEI7HvsrOp11rhBsseBwsSHjeOncOwjj+HXFZGIFl92nXIYE6HrTL+mTcf6GMahivT83FiuxauVFluvAoDnN96BGbvzlKYHEE2CwzfO\\\\\\\/Z82qlrR7LIYCulIz2ANChkQ0UueI4yQG1cDUwWlg+wGOzFdSCud\\\\\\\/VWWT\\\\\\\/j8wXt72GJejwOfp4gVEkHiINfu2wRXORC4U\\\\\\\/KvPusdQHKqui7B9f1KR8OiI8puScQllGoGNJ6bcmZFI1USgyIF6U1tE71sM0UdFlVs5EyenCW1YmCmJMQCFKC7vHGVGm\\\\\\\/LXLYZGCG2yNkpQ2ghYHjY6rREM250vGpKbpoiBXE5s3y6PL\\\\\\\/4GpKbgHAo+zM3\\\\\\\/GMBtFj+pZrI\\\\\\\/wTDZm1sHT41WQlXptUZGSgJUm\\\\\\\/YzQt3UiF9QETl8LtrBXECOc3jXdwWIhkry7EjCnkwk8tInipuVNcC1lLG6jMl3mDqSsnynbpuTm98AgVaYJRi6En4ivbaMPmfIWrRllTOdafv2fRAZChFoFA6fQgTJjQlF6CUeMBLddL0ODAir+oKTD+Je+KCOVGVuD1O\\\\\\\/ZocOD8CLXeXoUisuinFvyqS4d+RQsl6Vvq3WnMC1RFd0OaqXE+892Yf4V9gGZjb6AyHXD4uXIEbImJvO6vOMmmMNuiAAKFVdks7\\\\\\\/VePAudHOuFQaqwm4fg2UbvgVpdGfO+wpEMb8FBmpGiP7akOAlEdfnigDvbwqEcntP4g8BBF+T8tgE44NC8aN4f0UN5SZ1QXpm7Lh8wZ1fytRvgUkZB6F0w7fCQBOMaRM5THRdhqrLTUc7Z5D33QP7zhB8XCz4B4Eax3VZMxQDq7c2X7qDfO\\\\\\\/0H9MmkfSS1B38pb+nOXpy39yX3\\\\\\\/qH3rlgjKX1+F1NQG9BS0SCHQwNjF7olA0TAX1RBVMFqBqVTBU3blihl0ylWhtDQULQEfLB5Y86vDtQPMxt7BaDcf18C0aOqoee0Cg7g7LnT9ChjAnxVpoMTXOP\\\\\\\/mReSNPsqz1uewDZXxKh9FuvxHNBouh3f1kUmzlxj6+acvlCxT1RY6mIwjitISyN9wD2sjnPHZFVnfHfEFKMMzqnuU7Ui1aLei\\\\\\\/Ccght7Q2Vp5mUmqfYX\\\\\\\/PIQGb4NPyMY6ATtbRX42d0wk\\\\\\\/H9VaJfW2+SfzXhoACMm2FD3Z6Brsn275YOTmJp3MNdUvBBmm\\\\\\\/3iVPhNy2WYkob2qwF1uFKxj\\\\\\\/UPB21GNBfAf2mVfj5sB7GoJPEAjqfaHaVGUoDMMsmnygfWbNRI7fWXrXiR\\\\\\\/12kO6vli1JXWikNda5GA2InSSy2vOFmxqeHl0ZgcFkKhMEMQabjRYp6RnTW17pzkO94JLtQ5zfEmm6GGlJdmzauVtCmEVet5No+lOx0dhgGj0sFJMe9838uwFw7Rbm4yOspTAmR4J64PfmECguZrhrTayakIZ5ttt4ij37Jlifz8KR\\\\\\\/xyTKxwVaSR0vwV3jM2LQuhe6ayOzoynIfMYmfs6F37AG0\\\\\\\/nqotogAh9q5fCJUH+ztjL5TAr8pQY50h5D+nojUmgJfEzNn6XNTpuDdU+8KkNND1ppthDxXPJcVB+9j4FwLiRLDr6LgoJoGacIWz2SdDRGX7GTTuFD3ceVRa4yQ1vk8Nrv7K2SC7MaTG0JskZne5h+GGO0c3cGME7edFMrMnCHuwA2e\\\\\\\/bO1cbO3HZnrsGcmBh2rwk1N8e4VbZPNqU09foRjr0MrfzGpAtcnLA7mEaWtMJsW9sVmbq1AdA\\\\\\\/O25Ig+SmvBD+TAQ20TwY9vR3gYpTYengP5TZ3YLoogSNkP0b3\\\\\\\/I7cgtwQw8b9rSheWh\\\\\\\/MU+7S1XGI9uNEUFX1aCV0nQye5poFMiPw018w8i1XzL8SaeNECHH+IOQixI3LeVn6u3ZfGVatD0ffveAoi1cOpln\\\\\\\/uU\\\\\\\/iXlTQUcoXwiFXvhYrC9j0D1d675ulFOFIz8\\\\\\\/2wz8+wkj1Mf0M5vfPFYulJDlDPdG2qL16nU7pp6C0eUhazKjPmvo77cIfaf4spaTnzbW+5MUl3\\\\\\\/+UNxDuietPLtC3lZ5nBTdFaV2gW+diRwHEucT9B9iBLZgAgkRoFHyv5j+PZZNZ7QWlg7MXoo+rS0r0Mb0oGNHC4fdptDA4spr\\\\\\\/hngpqeRql2qN0OzerC7yiMuG2Q8m10wFdrE6hIB5OYmt6oEv7rWthsJeGhAKZ4bCji7kiSf8zti4cYFJodgj0N584tyeNYqPgoAHo3ZMM10F5LuWyf6NJ4\\\\\\\/PLyMhre6Wf0SIMkow+lkHjAH60\\\\\\\/WIwIraOLqjxF3SsEms5fnWCzSo717mj1rIcN42WotZVcIcKSXGN5Y2K4Tn1gq5kJmqPkvphrht+sZAVPf8OrGev2UeYllTRgrPfpQPN5SigReEqBPDtLFCVxHq0kx5vViplsU3LwPS46BQBkUVS9QzuG5IelcVNTEX2JglzFq\\\\\\\/+jiCGQsGMW+hYiy2uGtMCy5z4CSpLJb1k8kNoVXwj3Osuv9YwsQnMx1ecUEqGzldBT+Qu\\\\\\\/GpPvmXBgN6SNQMtEzj7xST8x9eIi2FNB5zwTyOlWAd+qtgfZflTCRngDQr797Nhs\\\\\\\/\\\\\\\/+VjbUCzk5RfSY+nvzHi+PO9GG0vqIpQu2TR5ctF\\\\\\\/w0kfFLdm\\\\\\\/gTdnsAEV2SJhFpE9YJrMZBThWs2m2NZwE9UNOG\\\\\\\/ClaYH+6ailWNAjx5yy60m+BIrUpfBNk9Jn3wanss5gcbpjBs70kGYAh06RUwe6kMKQRhlbDsh+z5Mo5R+Cy\\\\\\\/EE5afkC2md6MTZ1Vq2O0wWPg+WqX3SBZSyVxJXxYaHd6p3T8JxwC+dpAWGhuRRbS2t8xZehJLAeVNHe9TrpsEl41YMZJGInozR\\\u0022,\\\u0022iv\\\u0022:\\\u0022PHnrZ76H8AsnEwJ6Wvpz0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f0c4115e4497bb19ebfbb1ad7fcac2ff97a158d87f0dede5d361209c34f172df\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gzQ5\\\\\\\/OkYxOs0wUjBgPA9R8XRSIPADAEfkc5s7DJ2d900zBUaiqbNCJr11vq5NfdUDzJtKqrN4\\\\\\\/wYfEFpSMb+SaIDR6BZoM8EQ1sQeOlFQZZIWxXBtv27yuSQ0kSoHds0nZrHJLVuNx7Qh28yXpFPK30j5gsvv0IMK+yT0PBn+chsS6X0U10SOSDjSKZ2zwvsH8ei\\\\\\\/CA8ULinp\\\\\\\/c+t9U9AOCwERuAK+Zq1ftOxVH4zLk8GPlvcOZ5i0ZSU5bvkFdBfGdfXJCEkTyqJURzpNEwIvsFrsdiclK3ctfZMiO9NQpV7sQCkwXwQU1LFHPh1v+\\\\\\\/qmAromZcCBntLitIRfqA31iRbozd5njq7J0pIeYcpiaE0\\\\\\\/6ARXk6aa8ar6p3IGU1Av8RWyK88AZ7a40+xZBl\\\\\\\/m1E1uIDdnDeY5+S4tYOtG+mjnV277KkY02gN2XKS6nMv\\\\\\\/evjIBpMN2oBJZFMbqH3vPryxEArfsEhX3RTTut8x2XkbATAUokUf68PHbB7WKQSapdcNFULkYuMQIj\\\\\\\/PHFiFskCFalHq8VhiMURXqWUpZ2iQm2IuOmTNLHb0JOEW0gdtBDv7uS9x+Ckrp9MiQbzV4lyrdA2\\\\\\\/7yk\\\\\\\/rs1d4hdWyplAX0sPKyAugTpx9XFfOJGc0b1PBIOylrF\\\\\\\/hE4\\\\\\\/2eJlV7cXig9J8qEXruHhS4km0oG0tVqUtYl+VXy5q62VfiVfNPX93LWN4zBz1G2j6B4vT5LpjLH5OnJW3UwWvuj8DY3EU3UP11kbT716Xae5sFTna8E1GKITe0DuyqE00KUCuklK2Uz97PNmahj03\\\\\\\/4stuNv65AY827JXub9VX00oi3pcxmrsrIiUtTakdL8hVSXIGqK0UKu8wFECRXwPfYWrmOAM7W1oglESyssFqwGh4hNeJmUbI01k+Z\\\\\\\/45Dj9ubSmujndXCzc7PQaV2uVQaMwVeNf4gxK6zpATMfNS7CNMI9LlC0JIQL6MvyeQYLI8w7aXEwG+rp0PZ\\\\\\\/D3xZajRoNmemxgEk76cwIvKqmEPkI\\\\\\\/eDqwW5lcjS+ySHl\\\\\\\/0qX9RQ3OOfP8NfwwIyDQDSBUr+hN0Y7dwZGz8xQlnJDtP7VvS7TIHFMfp\\\\\\\/UBQHs8AByHQHe7PFic6ykonuq1ZtOSx9FVi5pzwNHEISa\\\\\\\/4hqBm+8d5Mw7dk5pN\\\\\\\/ieR6d0knvgaa75s85hgbBjKzCA0YvJcFeVjijzmAFzZqkItu4nGpq4g8JUuv55\\\\\\\/ClXyAynG6aaJT7VViMc4dzUMK\\\\\\\/I4Xxby7aKd0DRF45ysxsGeb1VK8PpC8fRjT7oPO2F1eX0noe9tuTS1unelRnYJuiPQajOzDWUc09t\\\\\\\/++5EiSzQ37uUxW\\\\\\\/4cb0Ni0rZrYeU4RPq7Q5auErDvdbNHaNlb5upXryNadsplqD520lMFjZBagC54tOg41N1wQKub6Dzf52i\\\\\\\/M122D6lP3kXaPtzov+fK4l5nCOVaar+TOK7LdiXUCzOJ40+DlMDBh9x520v1hMJeVXzdukgBA3\\\\\\\/QhQKTJ1dCJ24\\\\\\\/2Pa6fQ5TYoTi5X0eZRUhJzr01D5pWe\\\\\\\/LB4D3Zj+BgW6KxtOBRzG2jzrSaVr1q1lTjbh41S5QWo1H8BdSCIlB6osiJbEF4it9Bu74GTPCGByj50IS0zL3AM\\\\\\\/h4d6\\\\\\\/PDy7rOOBKCJG+vxknvavjLOaCdKChCRE\\\\\\\/Lak4eZPKzLJ0DtZVtDA9YxkIdf++5wJUisE5pFU6abLd65QB0GhxE7Z5ZRzog6iO8gXEb3b2MGu6whJT3S7frws0dwGUd6H2tG1Fz1+grmORSn+6Sd6PbIagpdxNfYQTuzV7OhvYfOQrsrnkeNrayJOxsK6Rj6n+GKdaY0wDKPndPjMBdorrD6gdJv3WtY3OcTm7LNwmX8+813diS9DjamVUbBt7ymMLyXW+7kRlPV0vsPSZf\\\\\\\/8zTqvgz+Rd91ifRNfhLyqwcsGDL9d\\\\\\\/Q8sIcDe6ktjeh6vsSC0gY9A6mBi9ycEKNbXsMJ1jXDCw1+q4FdaE8puQOW2RKdhBa1PrHqiqe6QYZyUIy2bE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022kKysggHc4RISh4dCo2jZcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223b982ec5f4b4e9177ac9b97c50da8dd6b4c6416f20ea14338b5826e430f3fa1c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490975 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3203" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Bh4CzG3PTWveC2V3a3HwqadveWwRID0D1OCdTbVDgSo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3203" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5dSYeg_i2V5KuYu7lokerT9sDk3xPXX1wYuH3q5fE5o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3185" title="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3221" title="Null Mutation of the Fascin2 Gene by TALEN Leading to Progressive Hearing Loss and Retinal Degeneration in C57BL/6J Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490975" id="node490975" data-pisa="ggg;8/10/3203" data-pisa-master="ggg;g3.118.200485" data-apath="/ggg/8/10/3203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Estimating Genetic Relatedness in Admixed Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8201-8292" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arun Sethuraman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3203-3220;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200485 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490975" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arun Sethuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, California State University San Marcos, CA 92096</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arun%2BSethuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sethuraman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArun%2BSethuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8201-8292" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arun Sethuraman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asethuraman@csusm.edu" class="" data-icon-position="" data-hide-link-title="0">asethuraman@csusm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3203" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490975" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490975/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3203.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490975" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490975/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3203.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490975" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490975/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3203.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490975" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490975/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Estimating genetic relatedness, and inbreeding coefficients is important to the fields of quantitative genetics, conservation, genome-wide association studies (GWAS), and population genetics. Traditional estimators of genetic relatedness assume an underlying model of population structure. Each individual is assigned to a population, depending on <em>a priori</em> assumptions about geographical location of sampling, proximity, or genetic similarity. But often, this population assignment is unknown and assumptions about assignment can lead to erroneous estimates of genetic relatedness. I develop a generalized method of estimating relatedness in admixed populations, to account for (1) multi-allelic genomic data, (2) including all nine Identity By Descent (IBD) states, and implement a maximum likelihood based estimator of pairwise genetic relatedness in structured populations, part of the software, InRelate. Replicated estimations of genetic relatedness between admixed full sib (FS), half sib (HS), first cousin (FC), parent-offspring (PO) and unrelated (UR) dyads in simulated and empirical data from the HGDP-CEPH panel show considerably low bias and error while using InRelate, compared to several previously developed methods. I also propose a bootstrap scheme, and a series of Wald Tests to assign relatedness categories to pairs of individuals.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/genetic-relatedness" class="hw-term hw-article-keyword hw-article-keyword-genetic-relatedness" rel="nofollow">Genetic Relatedness</a></li><li class="kwd"><a href="/keyword/coancestry" class="hw-term hw-article-keyword hw-article-keyword-coancestry" rel="nofollow">Coancestry</a></li><li class="kwd"><a href="/keyword/admixture" class="hw-term hw-article-keyword hw-article-keyword-admixture" rel="nofollow">Admixture</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">Population Structure</a></li></ul><p id="p-3">Estimating genetic relatedness is an important problem in biological statistics and population genetics. For instance, paternity or maternity assignment (see <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Avise 2001</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pearse <em>et al.</em> 2002</a>, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yue and Chang 2010</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Coleman and Jones 2011</a>), and forensic studies (reviewed in <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Weir 2004</a>) require a robust statistical framework to infer relatedness between genotyped individuals. Genetic relatedness also plays an important role in the study of quantitative traits where the proportion of trait variability explained by shared alleles indicates the strength of the genetic component of the trait (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Visscher <em>et al.</em> 2008</a>). In several allied fields, accurate estimation of genetic relatedness is critical. For instance, association studies and linkage analyses without accounting for the increased relatedness due to population genetic structure could lead to spurious associations (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pritchard <em>et al.</em> 2000a</a>). Genetic relatedness is also important in fields such as conservation genetics (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Oliehoek <em>et al.</em> 2006</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wang 2018</a>).</p><p id="p-4">The genetic relatedness, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-1.gif"/></noscript></span></span> between two individuals <em>X</em> and <em>Y</em> can be defined in terms of the probability that their alleles are Identical By Descent (IBD). <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-2.gif"/></noscript></span></span> is thus also twice the coefficient of coancestry, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-3.gif"/></noscript></span></span> and can be thought of as the inbreeding coefficient of any offspring that <em>X</em> and <em>Y</em> may sire (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Weir <em>et al.</em> 2006</a>).</p><p id="p-5">Conventional relatedness estimators work in either of three ways: (1) estimating a coefficient of relatedness between two individuals using multilocus genotype data, and linkage data to inform the length of IBD tract sharing; or (2) assigning sib-ship partitions, reconstructing pedigrees, and using the pedigrees to estimate relatedness; or (3) directly estimating relatedness from known pedigrees (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Weir <em>et al.</em> 2006</a>). All relatedness estimators, however, have high variances, primarily owing to difficulty in parsing out true IBD states from observed Identity By State (IBS) states (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blouin 2003</a>). This delineation of IBS <em>vs.</em> IBD is achieved by estimating the conditional probabilities of observing a genotype at a locus in one individual <em>X</em>, given the observed genotype at the same locus in individual <em>Y</em>.</p><p id="p-6">In the presence of population genetic structure though, localized inbreeding makes individuals within the same subpopulation ‘more related’, than as suggested by their pedigree. Pervasive or specific inbreeding in recent generations past between two related individuals can be quantified though, if sufficient information is available on the existing genetic subpopulation structure. The estimated inbreeding coefficients (<em>e.g.</em>, <em>θ</em>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Weir 1994</a>) affect the aforementioned conditional probabilities (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Weir 1994</a>). Alternately, maintenance of advantageous alleles in subpopulations by selection within a total population could also yield ‘artificial’ patterns of relatedness between individuals that share alleles, but not necessarily by direct descent.</p><p id="p-7">Not accounting for such ‘shared’ allelic ancestry by utilizing true, or estimated subpopulation allele frequencies leads to incorrect estimates of genetic relatedness. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> subvert this issue of estimating subpopulation allele frequencies by directly quantifying the amount of inbreeding due to subpopulation structure, conditioned on <em>a priori</em> knowledge of existing subpopulations within a total population. Thus estimates of relatedness using the inbreeding coefficient <em>θ</em> in its formulation could be potentially biased.</p><p id="p-8">Several other methods also utilize current population allele frequencies as proxies for ‘ancestral’ (this could mean subpopulation allele frequencies of the current generation, as in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Anderson and Weir 2007</a>, or allele frequencies of subpopulations from generations past, equated to current allele frequencies, as in <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wang 2002</a>) subpopulation allele frequencies, under Hardy-Weinberg Equilibrium (HWE), in their estimates of the inbreeding coefficient, <em>θ</em>. This assumption can be problematic because we do not know the precise number of ancestral subpopulations. However, the number of ancestral subpopulations can be approximated by the current subpopulation structure in a reference population.</p><p id="p-9">Most methods for estimating pairwise genetic relatedness also assume that individuals whose pairwise relatedness is being estimated are derived from the same single, panmictic subpopulation. The methods of <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, and <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, that attempt to relax this assumption by handling samples from multiple subpopulations, assume that individuals derived from different subpopulations are genetically unrelated. However, in the presence of genetic admixture and migration, alleles are shared between subpopulations.</p><p id="p-10">To account for unobserved population structure in bi-allelic genetic data, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Moltke and Albrechtsen (2013)</a> develop a two-step method (RelateADMIX), which estimates population genetic structure as admixture or ancestry proportions, and subpopulation allele frequencies. This method, when compared with other popular tools for estimating relatedness, including REAP (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Thornton <em>et al.</em> 2012</a>) and PLink (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Purcell <em>et al.</em> 2007</a>) shows considerable reduction in bias in estimating IBD probabilities. This method uses the following information: (a) admixture proportions of alleles at multiple bi-allelic loci in individuals, in “most likely” genetic subpopulations, as determined by likelihood or Bayesian methods such as those implemented in STRUCTURE (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Falush <em>et al.</em> 2007</a>), ADMIXTURE (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>), and MULTICLUST (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sethuraman 2013</a>); and (b) subpopulation allele frequencies that are estimated as parameters in the model. Specifically, the model uses the probability distribution that an allele at a locus in an individual, or a multilocus genotype of an individual, was derived from a subpopulation in the recent past. IBS probabilities for two individuals, conditioned on the three IBD states <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-4.gif"/></noscript></span></span> <em>sensu</em> <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jacquard 1972</a>, <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Anderson and Weir 2007</a>) are then calculated. This calculation contributes to a likelihood function (<em>sensu</em> <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Thompson 1975</a>), which is then maximized using an Expectation Maximization (EM) algorithm (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dempster <em>et al.</em> 1977</a> to obtain maximum likelihood estimates for relatedness coefficients. These IBD coefficients are then used in calculating pairwise genetic relatedness, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-5.gif"/></noscript></span></span> and coancestry coefficients, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-6.gif"/></noscript></span></span> This method however assumes that alleles derived from different ancestral subpopulations are not IBD, and hence accounts for recent population structure. Here I develop an alternate formulation that utilizes estimated subpopulation allele frequencies, and ancestry proportions to estimate genetic relatedness in structured populations to include all nine IBD states <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-7.gif"/></noscript></span></span> and to be applicable to multi-allelic data, which accounts for ancestral subpopulation structure, where alleles derived from different subpopulations can also be IBD in an ancestral population. I develop a new package, InRelate based on a non-linear programming solution to this problem. I then address several questions based on the new framework 1) how does this estimator of pairwise genetic relatedness compare with other estimators of relatedness for structured and unstructured populations in simulated and empirical datasets?, 2) how does this estimator compare to other estimators with increase in available information (measured in terms of the number of genotyped loci)?, 3) how do bias and mean squared errors (MSE’s) in estimation using InRelate change with demographic model of evolutionary history?, 4) how does erroneous estimation of subpopulation structure due to label switching affect estimates of relatedness under the InRelate model? I also describe a method of bootstrapping and a series of statistical tests in order to obtain confidence intervals around estimates of relatedness.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Relatedness Under the Admixture Model</h3><div id="sec-3" class="subsection"><h4>Theory:</h4><p id="p-11">I use the admixture model introduced by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pritchard <em>et al.</em> (2000b)</a> to model population structure, since it makes few assumptions about the demography or history of the studied population.</p><p id="p-12">It is to be noted that this model assumes that all individuals in the sample are unrelated, which in our case, is not actually true. If there are however, proportionately few relatives in the sample, then estimation under the admixture model should be reliable. For samples with rampant relatedness, pedigree estimation, or using methods that rely on linkage information may be more appropriate.</p></div><div id="sec-4" class="subsection"><h4>Data:</h4><p id="p-13">Assume that a sample of <em>I</em> largely unrelated, diploid individuals has been collected from a population possibly consisting of <em>K</em> unknown subpopulations. Each individual has been genotyped at <em>L</em> unlinked, codominant, neutrally evolving loci. Assume that locus <em>l</em> exhibits <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-8.gif"/></noscript></span></span> possible allelic states in the sample. For example, at SNP or AFLP presence/absence markers, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-9.gif"/></noscript></span></span> Microsatellite markers evolving under the infinite alleles model theoretically have infinite states, but we observe some <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-10.gif"/></noscript></span></span> in the finite sample. Missing data due to failed genotyping are allowed, but assumed to be missing completely at random.</p><p id="p-14">The observed genotype data from diploids can then be combined into a three-dimensional matrix <em>X</em> of size <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-11.gif"/></noscript></span></span> Thus, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-12.gif"/></noscript></span></span> is the <em>m</em>th (first or second) allele at a locus <em>l</em> in individual <em>i</em>. The data can then be reduced to sufficient statistics. Specifically, let <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-13.gif"/></noscript></span></span> be a jagged array with entry <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-14.gif"/></noscript></span></span> the number of alleles of type <em>a</em> observed at locus <em>l</em> in individual <em>i</em>.</p></div></div><div id="sec-5" class="subsection"><h3>Relatedness Under the Admixture Model</h3><p id="p-15">The admixture model <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Pritchard <em>et al.</em> (2000b)</a> posits that all the alleles in an individual are independent draws from a mixture of <em>K</em> subpopulations. Each subpopulation is characterized by its allele frequencies: <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-15.gif"/></noscript></span></span> is the frequency of allele <em>a</em> <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-16.gif"/></noscript></span></span> at locus <em>l</em> <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-17.gif"/></noscript></span></span> in subpopulation <em>k</em> <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-18.gif"/></noscript></span></span> Each unrelated individual is characterized by a particular mixture of the <em>K</em> subpopulations: each allele of individual <em>i</em> <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-19.gif"/></noscript></span></span> is derived from subpopulation <em>k</em> with probability <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-20.gif"/></noscript></span></span> The parameters are constrained such that <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-21.gif"/></noscript></span></span> for each individual <em>i</em>, and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-22.gif"/></noscript></span></span> for each subpopulation <em>k</em> and locus <em>l</em>.</p><p id="p-16">The likelihood of the observed multilocus genotype data, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-23.gif"/></noscript></span></span> given the parameters <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-24.gif"/></noscript></span></span> under the admixture model is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Relatives are then characterized by their shared alleles, <em>i.e.</em>, shared alleles that are identical by descent (IBD). As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, the four alleles at a locus in two diploid individuals can be in one of nine possible, unobserved IBD states, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-26.gif"/></noscript></span></span> The marginal probability distribution over the IBD states for a pair of individuals at a locus is determined by their relationship. I use the notation <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-27.gif"/></noscript></span></span> for this distribution. For example, in non-inbred populations, unrelated pairs are in state <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-28.gif"/></noscript></span></span> with probability <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-29.gif"/></noscript></span></span> while full siblings will share no alleles at a locus with probability <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-30.gif"/></noscript></span></span> one allele with probability <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-31.gif"/></noscript></span></span> and both alleles with probability <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-32.gif"/></noscript></span></span> assuming their parents are unrelated.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nine possible Identity By Descent (IBD) states for the observed genotypes of two diploid individuals i and j at a genomic locus l. In each IBD state (), The alleles are connected by a line if they are IBD. Observed Identity By State (IBS) states are not shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Nine possible Identity By Descent (IBD) states for the observed genotypes of two diploid individuals &lt;em&gt;i&lt;/em&gt; and &lt;em&gt;j&lt;/em&gt; at a genomic locus &lt;em&gt;l&lt;/em&gt;. In each IBD state (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-33&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), The alleles are connected by a line if they are IBD. Observed Identity By State (IBS) states are not shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F1.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3203/F1.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491160" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Nine possible Identity By Descent (IBD) states for the observed genotypes of two diploid individuals <em>i</em> and <em>j</em> at a genomic locus <em>l</em>. In each IBD state (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F1/embed/mml-math-33.gif"/></noscript></span></span>), The alleles are connected by a line if they are IBD. Observed Identity By State (IBS) states are not shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We only know if alleles are identical in state (IBS), and each IBD state is consistent with one or more of the nine IBS states, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-34.gif"/></noscript></span></span> Methods of relatedness estimation, use the IBS states observed at multiple, independent loci of two individuals to estimate <em>δ</em>, and hence their relationship.</p><p id="p-19">Consider two individuals <em>i</em> and <em>j</em>. We observe their IBS state, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-35.gif"/></noscript></span></span> at each locus <em>l</em>, where <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-36.gif"/></noscript></span></span> Each <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-37.gif"/></noscript></span></span> follows an observed configuration in <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-38.gif"/></noscript></span></span> but the true IBD state, <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-39.gif"/></noscript></span></span> is unobserved. Given a known relationship, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-40.gif"/></noscript></span></span> between <em>i</em> and <em>j</em>, the likelihood of the observed data are<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-41.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>If two individuals were full siblings from parents from the same subpopulation, genetic relatedness estimated using ancestral subpopulation frequencies would be expected to account for deep descent, and potential inbreeding of the parents. The relatedness between these full siblings, estimated using the parameters of the admixture model, should be as close to the true estimate, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-42.gif"/></noscript></span></span> as possible. On the other hand, if two individuals are full siblings from parents derived from two different subpopulations, genetic relatedness estimated using current subpopulation allele frequencies would likely be an over- or under-estimate, because the recent admixture event between the two parents in the previous generation is not accommodated. This result permits defining conditional probabilities of IBS states, given their IBD state using this new parametrization, <em>sensu</em> <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Jacquard (1972)</a>.</p><p id="p-20">Following the leads of <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Jacquard (1972)</a>, <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, and <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, define the set of nine IBD states (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-43.gif"/></noscript></span></span> given a diploid locus between two individuals, 1 and 2. Each IBD state could have nine, or more possible IBS states, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-44.gif"/></noscript></span></span>. Under the above assumptions, the probability that an allele <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-45.gif"/></noscript></span></span> is observed at a locus <em>l</em>, in individual <em>i</em> is <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-46.gif"/></noscript></span></span> the probability that an allele <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-47.gif"/></noscript></span></span> observed at the same locus <em>l</em>, in individual <em>j</em> is <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-48.gif"/></noscript></span></span> and so on. All the conditional probabilities, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-49.gif"/></noscript></span></span> are shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The likelihood of the IBD states over a single locus, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-50.gif"/></noscript></span></span> can be written as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-110.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>, where Δ is the set of 9 IBD states observable, <em>X</em> is the observed data, and <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-111.gif"/></noscript></span></span> is the observed IBS state of <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-112.gif"/></noscript></span></span> Over <em>L</em> independent loci, this likelihood can be written as a product of individual locus likelihoods as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-113.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>This likelihood function can be maximized using the constraints that each IBD coefficient, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-114.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-115.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-116.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-117.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-118.gif"/></noscript></span></span> I used the <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-119.gif"/></noscript></span></span> function in the <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-120.gif"/></noscript></span></span> package in <em>R</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ghalanos and Theussl 2012</a>), which implements the augmented Lagrange method of <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ye (1988)</a> to solve this nine-dimensional problem with linear constraints. The coancestry coefficient, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-121.gif"/></noscript></span></span> between two individuals, <em>X</em> and <em>Y</em> then can be calculated as <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-122.gif"/></noscript></span></span> and, by definition, the relatedness as <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-123.gif"/></noscript></span></span> Note that <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-124.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-125.gif"/></noscript></span></span> only if the population is outbred (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-126.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-127.gif"/></noscript></span></span>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491140/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491140/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Conditional Probabilities <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/T1/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/T1/embed/mml-math-51.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Other Relatedness Estimators</h3><p id="p-21">I also implemented the methods of <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, and <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Wang (2011b)</a> under the same optimization framework, using <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-128.gif"/></noscript></span></span> The method of <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Wang (2011b)</a> is different from that of <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> in that it accounts for inbreeding. In both cases, subpopulation allele frequencies are modeled under the Dirichlet distribution, with the global parameter, <em>θ</em>, measured as the probability that two randomly sampled individuals from a subpopulation are IBD under an island model. <a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> do not state explicitly how they estimate <em>θ</em>, but <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Wang (2011b)</a> indicates using the Weir and Cockerham <em>θ</em> estimator <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Weir and Cockerham (1984)</a>, which I used as well in the framework of <a id="xref-ref-3-9" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> (and <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Wang 2011b</a>) to obtain comparable relatedness estimates. Regardless, under the equilibrium assumption that population subdivision is unchanging in time, the probability of drawing two of the same <em>a</em> alleles at a locus from the same subpopulation is <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-129.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-130.gif"/></noscript></span></span> is the frequency of allele <em>a</em> at that locus. This leads into the same likelihood framework described above (3,4), for the estimators of <a id="xref-ref-3-10" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, and <a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Wang (2011b)</a>. <a id="xref-ref-3-11" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> utilize a simplex method to obtain maximum likelihood estimates of the IBD coefficients, <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-131.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-132.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-133.gif"/></noscript></span></span> using the constraints that <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-134.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-135.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-136.gif"/></noscript></span></span> for large, non-inbred populations.</p><p id="p-22"><a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Wang (2011b)</a> offers another numerical solution by using Powell’s quadratically convergent method (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Press 2007</a>) to obtain likelihood estimates for all 9 variables above, as well as derives moment estimators under the same population structure framework, accounting for inbreeding using the inbreeding coefficient, <em>θ</em>, for other previously derived estimators (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Queller and Goodnight 1989</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lynch and Ritland 1999</a>, <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wang 2002</a>).</p><p id="p-23">In this manuscript, the same non-linear programming method in 9 variables (<span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-137.gif"/></noscript></span></span>) was used to obtain maximum likelihood estimates for both estimators of <a id="xref-ref-3-12" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-9" class="xref-bibr" href="#ref-56">Wang (2011b)</a>. Genetic relatedness, <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-138.gif"/></noscript></span></span> and the coancestry coefficient, <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-139.gif"/></noscript></span></span> were then calculated as before.</p><p id="p-24">Other estimators that were compared include those of <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Queller and Goodnight (1989)</a>, <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Wang (2002)</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lynch and Ritland (1999)</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lynch (1988)</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ritland (1996)</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang (2007)</a>, and <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Milligan (2003)</a>, as implemented in the program COANCESTRY (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang 2011a</a>). Note that all the methods implemented in <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Wang (2011a)</a> do not account for subpopulation structure (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). However, all these methods account for multi-allelic data, which allow for equitable comparison with InRelate. The methods of <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Thornton <em>et al.</em> (2012)</a> (REAP), <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Purcell <em>et al.</em> (2007)</a> (PLINK), and <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Moltke and Albrechtsen (2013)</a> (RelateAdmix), while more popular in recent years, are only applicable to di-allelic data (<em>e.g.</em>, SNP’s), and hence were not used for comparison in this manuscript.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490988/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490988/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">List of estimators tested and their references</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Bootstrapping and Pedigree Assignment</h3><p id="p-25">Under the assumption that sampled loci between two individuals <em>X</em> and <em>Y</em> are independent, we can obtain variance in estimation of relatedness by bootstrapping over loci. For every pair of individuals, loci are sampled with replacement to construct bootstrap replicates, and relatedness is estimated under the maximum likelihood framework. I then construct 95% confidence intervals of the estimated relatedness values. Simulated bootstrap standard errors are calculated as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-140.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <em>B</em> is the number of bootstrap replicates and:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-141.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>These variance estimates are then used in a series of Wald Tests, compared to a normal distribution, to assign relatedness categories to each pair of relatedness estimates. The Wald Test statistic is calculated as:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-142.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>After correcting for multiple testing by the Bonferroni method, pairs are assigned to a relatedness category at a p-value threshold of 0.05. Relatedness categories tested include: MonoZygotic twins - MZ, Full Siblings - FS, Half Siblings - HS, First Cousins - FC, Parent-Offspring - PO, Second Cousins - SC, AvunCular - AC, and UnRelated - UR.</p></div><div id="sec-8" class="subsection"><h3>Simulations</h3><p id="p-26">Five separate sets of multi-allelic genomic data were simulated to test the performance of relatedness estimates using InRelate (MC2013, hereon), against other estimators. In all scenarios, subpopulations from which individuals were sampled from were assumed to be the ‘true’ subpopulation, for comparison with other methods. Admixture proportions and subpopulation allele frequencies for all analyses were obtained by performing runs of MULTICLUST (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Sethuraman 2013</a>). MULTICLUST uses an EM algorithm to estimate parameters under the admixture model (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Pritchard <em>et al.</em> 2000b</a>), and extends the method of <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> (2009)</a> for multi-allelic data. It is much faster than STRUCTURE, and does not have MCMC convergence and mixing problems. Convergence of the EM algorithm was assumed if the log likelihood was not increasing by <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-143.gif"/></noscript></span></span> in all scenarios.</p><div id="sec-9" class="subsection"><h4>Scenario 1: Hierarchical Island Model:</h4><p id="p-27">Under Scenario 1, all initial allele frequencies were simulated at 50 diploid, codominant, multi-allelic (maximum of 50 allelic variants per locus), unlinked loci, using Easypop v.1.7 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Balloux 2001</a>). The Hierarchical Island Model was used, wherein each total population (out of 3) is comprised of subpopulations, which are in turn comprised of smaller subpopulations. I varied the number of subpopulations (<em>K</em>) to be one of 3, 5, 10, or 15. To allow for genetic admixture, I specified relatively greater levels of gene flow of 0.01 total proportion of migrant females and males per generation, between subpopulations inside each population, and relatively lower gene flow of 0.001 total proportion of migrant females and males per generation, between populations. Subpopulation sizes of 25 males and 25 females per subpopulation were held constant across generations. A forward-time simulation was performed for 3000 generations, and I utilized the last generation’s allele frequency distribution for all further simulations. All populations at generation 3000 were tested for Hardy-Weinberg Equilibrium (HWE).</p></div><div id="sec-10" class="subsection"><h4>Scenario 2: Island Model:</h4><p id="p-28">Under Scenario 2, I simulated multi-allelic genomic data using the same demographic parameters as in scenario 1 using a single Island Model (<span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-144.gif"/></noscript></span></span>), with no migration.</p></div><div id="sec-11" class="subsection"><h4>Simulating related dyads:</h4><p id="p-29">I then simulated <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-145.gif"/></noscript></span></span> replicate dyads each of Parent-Offspring (PO), Full Siblings (FS), Half Siblings (HS), First Cousins (FC), and UnRelated (UR) individuals under different levels of known population subdivision (<span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-146.gif"/></noscript></span></span>). For FS dyads, two parents were randomly picked from the same subpopulation, and two offspring were created from their multilocus genotypes by randomly sampling their allele distribution from either parent. Since these loci are unlinked, I did not explicitly account for IBD tract length distribution. For HS dyads, one shared parent, and two other parents were simulated, and offspring generated from each cross. For FC dyads, a pair of FS dyads were created first, then their mates were randomly picked from the same subpopulation, to create offspring from each cross. PO dyads were picked similar to the FS simulation, with two parents being sampled randomly from the same subpopulation to create an offspring, and one of the parents were sampled as part of the dyad. UR dyads were created by randomly sampling two individuals from the same subpopulation.</p><p id="p-30">Admixture proportions and subpopulation allele frequencies were estimated using MULTICLUST (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Sethuraman 2013</a>) at the ‘true’ assumed number of subpopulations (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-147.gif"/></noscript></span></span>). These estimates were then used in determining pairwise genetic relatedness with InRelate.</p><p id="p-31">I estimated <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-148.gif"/></noscript></span></span> using the <em>geneclust</em> package in R, and utilized those estimates in the same IBD-IBS framework in R to obtain pairwise relatedness by the methods of <a id="xref-ref-3-14" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-11" class="xref-bibr" href="#ref-56">Wang (2011b)</a>. The package <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-149.gif"/></noscript></span></span> implements the method of <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Weir and Cockerham (1984)</a> to obtain a normalized multi-locus global <em>θ</em> estimate. For comparison with methods that did not account for population structure, I used the program COANCESTRY (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Wang 2011a</a>). <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows a summary of methods tested in this manuscript.</p></div><div id="sec-12" class="subsection"><h4>Scenario 3: Effect of number of loci:</h4><p id="p-32">To quantify the effect of increasing the number of genotyped loci on estimates of genetic relatedness, I simulated datasets under the same models specified in scenario 1 above, and the number of observed loci were varied between 10 and 40, to simulate a realistic scenario wherein individuals are genotyped at <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-150.gif"/></noscript></span></span> variant STR loci.</p></div><div id="sec-13" class="subsection"><h4>Scenario 4: Effect of method of estimating <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-151.gif"/></noscript></span></span>:</h4><p id="p-33">Under scenario 4, I was interested in how the estimation of <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-152.gif"/></noscript></span></span> affected estimates of genetic relatedness using the methods of <a id="xref-ref-3-15" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-12" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, against MC2013 (InRelate using all 9 IBD states), and MC2013WI (InRelate using only the last 3 IBD states, assuming an outbred population, <em>sensu</em> <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Moltke and Albrechtsen 2013</a>). To study this, I simulated a total of 1000 individuals distributed among <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-153.gif"/></noscript></span></span> subpopulations, genotyped at 50 STR loci (<span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-154.gif"/></noscript></span></span> allelic states per locus), with a mutation rate of <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-155.gif"/></noscript></span></span> mutations per generation, and a constant bidirectional migration rate of 0.001 of total individuals per generation, for 5000 generations. This gives a theoretical <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-156.gif"/></noscript></span></span> of 0.2, while Weir and Cockerham’s normalized Θ estimated at generation 5000 by <em>geneclust</em> was 0.1038. Hundred FS pairs were simulated from the generation 5000 population as described above. Allele frequency distribution of the generation 5000 population was used in estimating relatedness by the methods of <a id="xref-ref-3-16" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-13" class="xref-bibr" href="#ref-56">Wang (2011b)</a>. Admixture proportions and subpopulation allele frequencies for use by MC2013 and MC2013WI estimators were obtained using MULTICLUST at <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-157.gif"/></noscript></span></span> as before. To compare the performance of the methods of <a id="xref-ref-3-17" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-14" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, I estimated relatedness under both methods using (a) theoretical <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-158.gif"/></noscript></span></span> of 0.2, and (b) using the estimated Weir and Cockerham <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-159.gif"/></noscript></span></span> of 0.1038.</p></div><div id="sec-14" class="subsection"><h4>Scenario 5: Effect of label-switching:</h4><p id="p-34">Under scenario 5, I was interested in understanding how ‘label-switching’ affected estimates of genetic relatedness in methods that accounted for population structure. ‘Label-switching’ in this context refers to misclassification of individuals to subpopulations. To study this, I used the same dataset simulated for scenario 3, switched the labels of either 0.1, 0.5 or 1.0 fraction of the total population, and re-estimated Weir and Cockerham’s <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-160.gif"/></noscript></span></span> <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Weir and Cockerham (1984)</a>, and genetic relatedness using the methods of <a id="xref-ref-3-18" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-15" class="xref-bibr" href="#ref-56">Wang (2011b)</a>. Since population assignment is not <em>a priori</em> for the MC2013 and MC2013WI methods, I used the same results obtained from scenario 3 for comparison with the methods of AW2007 and Wang2011.</p></div><div id="sec-15" class="subsection"><h4>Error and Bias:</h4><p id="p-35">Deviation from true relatedness was examined by calculating the Mean Square Error (MSE). MSE is measured as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-161.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>, where <em>R</em> is the total number of replicate dyads (here 1000), <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-162.gif"/></noscript></span></span> is the relatedness estimated using one of the above methods, and <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-163.gif"/></noscript></span></span> is the true relatedness value, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-164.gif"/></noscript></span></span> which is 0.5 for PO and FS dyads, 0.25 for HS dyads, 0.125 for FC dyads, and 0.0 for UR dyads. Bias was calculated as the deviation of the mean for all <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-165.gif"/></noscript></span></span> replicates under each scenario from the true mean.</p><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-166.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span></div><div id="sec-16" class="subsection"><h4>Scenario 6: Bootstrapping:</h4><p id="p-36">For bootstrap analyses, I simulated another dataset from the above data set of <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-167.gif"/></noscript></span></span> subpopulations, genotyped over 300 loci. I picked 5 dyads each of FS, HS, PO, FC, and UR individuals (total of 50 individuals). Boostrap datasets (200 replicates) were then simulated, with 50 individuals each by resampling loci with replacement. For each dataset, the true subpopulation structure was assumed to be comprised of <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-168.gif"/></noscript></span></span> subpopulations. Admxiture proportions and allele frequencies were computed using MULTICLUST (at <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-169.gif"/></noscript></span></span>), and relatedness was then estimated using InRelate. Relatedness category assignment was then performed using the procedure described above.</p></div><div id="sec-17" class="subsection"><h4>Scenario 7: HGDP-CEPH Data:</h4><p id="p-37"><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rosenberg (2006)</a> and several allied publications (also see <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ramachandran <em>et al.</em> 2005</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rosenberg <em>et al.</em> 2006</a>) describe the use of subsets of ‘unrelated’ individuals from the HGDP-CEPH Human Genome Diversity Cell Line Panel (<em>H1048</em> <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cann <em>et al.</em> 2002</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rosenberg <em>et al.</em> 2005</a>). In these studies, relatedness was estimated between all pairs of individuals from within each sampled locale using RELPAIR (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boehnke and Cox 1997</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Epstein <em>et al.</em> 2000</a>), and several putatively related individuals from both within and across sampled locations were identified. For the purpose of this manuscript, I mined the original <em>H1048</em> dataset for individuals reportedly related from within the African continent. The African continent was represented in this data set by 115 individuals, classified as Bantu (South Africa), Bantu (Kenya), Mandenka, Yoruba, San, Mbuti Pygmy, or Biaka Pygmy, and were genotyped at a total of 783 microsatellite loci. Average differentiation, measured as Nei’s <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-170.gif"/></noscript></span></span> between these sampled locations was estimated as 0.1169, using the method of <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nei and Chesser (1983)</a>, which indicates ‘moderate’ levels of differentiation (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wright 1950</a>). I estimated population structure within these 115 individuals using MULTICLUST, at an <em>a priori</em> <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-171.gif"/></noscript></span></span> Admixture proportions and subpopulation allele frequencies were then obtained for the 24 relatedness dyads reported in <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Rosenberg <em>et al.</em> (2005)</a>, and I used these in estimating pairwise relatedness using InRelate. Allele frequencies were calculated assuming sampled locations as subpopulations, and used in estimating relatedness by the methods of <a id="xref-ref-3-19" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-16" class="xref-bibr" href="#ref-56">Wang (2011b)</a> for comparison. Note that RELPAIR (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Boehnke and Cox 1997</a>) utilizes recombination information to obtain genetic relatedness, and is therefore very different from all the other methods compared in this manuscript. For the purpose of this comparison, I used RELPAIR estimates as the ‘truth’ to measure concordance with MC2013 and MC2013WI.</p></div></div><div id="sec-18" class="subsection"><h3>Data Availability</h3><p id="p-38">All simulated data, and R scripts can be accessed at <a href="https://github.com/arunsethuraman/inrelate">https://github.com/arunsethuraman/inrelate</a>.</p></div></div><div class="section results" id="sec-19"><h2 class="">Results</h2><div id="sec-20" class="subsection"><h3>Scenario 1: Hierarchical Island Model</h3><p id="p-39">In general, in all scenarios that measured genetic relatedness among FS, PO, and HS dyads, the InRelate estimator (MC2013) performed better, or comparably with the AW2007 (<a id="xref-ref-3-20" class="xref-bibr" href="#ref-3">Anderson and Weir 2007</a>) and Wang2011 (<a id="xref-ref-56-17" class="xref-bibr" href="#ref-56">Wang 2011b</a>) estimators (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>). FS and PO relatedness had the least bias, compared to all other estimators. Interestingly, MC2013 and MC2013WI underestimated relatedness in FC and UR dyads when compared to the AW2007 and Wang2011 estimators. Distributions of estimated relatedness using MC2013 and MC2013WI are shown in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figures 7</a>, <a id="xref-fig-8-1" class="xref-fig" href="#F8">8</a>, <a id="xref-fig-9-1" class="xref-fig" href="#F9">9</a>, <a id="xref-fig-10-1" class="xref-fig" href="#F10">10</a>, and <a id="xref-fig-11-1" class="xref-fig" href="#F11">11</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F2.medium.gif" width="440" height="190"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3203/F2.medium.gif" width="440" height="190"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491153" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Half Sib (HS) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Half Sib (HS) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F3.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3203/F3.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491020" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Half Sib (HS) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Parent Offspring (PO) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Parent Offspring (PO) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F4.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3203/F4.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491119" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Parent Offspring (PO) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 First Cousin (FC) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 First Cousin (FC) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F5.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/10/3203/F5.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491086" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 First Cousin (FC) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 UnRelated (UR) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 UnRelated (UR) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F6.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/10/3203/F6.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491139" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Comparing (a) MSE and (b) Bias in estimates of genetic relatedness between 1000 UnRelated (UR) dyads with increasing degree of subpopulation structure. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of estimates of genetic relatedness between 1000 Full Sib (FS) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between full sibs = 0.5 is indicated using the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of estimates of genetic relatedness between 1000 Full Sib (FS) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between full sibs = 0.5 is indicated using the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F7.medium.gif" width="440" height="179"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/10/3203/F7.medium.gif" width="440" height="179"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491154" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-45" class="first-child">Distribution of estimates of genetic relatedness between 1000 Full Sib (FS) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between full sibs = 0.5 is indicated using the dotted red line.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of estimates of genetic relatedness between 1000 Half Sib (HS) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between half sibs = 0.25 is indicated using the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of estimates of genetic relatedness between 1000 Half Sib (HS) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between half sibs = 0.25 is indicated using the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F8.medium.gif" width="440" height="180"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/10/3203/F8.medium.gif" width="440" height="180"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491013" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-46" class="first-child">Distribution of estimates of genetic relatedness between 1000 Half Sib (HS) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between half sibs = 0.25 is indicated using the dotted red line.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of estimates of genetic relatedness between 1000 Parent Offspring (PO) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between parent-offsprings = 0.5 is indicated using the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of estimates of genetic relatedness between 1000 Parent Offspring (PO) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between parent-offsprings = 0.5 is indicated using the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F9.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/10/3203/F9.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491037" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-47" class="first-child">Distribution of estimates of genetic relatedness between 1000 Parent Offspring (PO) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between parent-offsprings = 0.5 is indicated using the dotted red line.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of estimates of genetic relatedness between 1000 First Cousin (FC) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between first cousins = 0.125 is indicated using the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of estimates of genetic relatedness between 1000 First Cousin (FC) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between first cousins = 0.125 is indicated using the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F10.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 10" src="https://www.g3journal.org/content/ggg/8/10/3203/F10.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491052" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 10</span> <p id="p-48" class="first-child">Distribution of estimates of genetic relatedness between 1000 First Cousin (FC) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between first cousins = 0.125 is indicated using the dotted red line.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of estimates of genetic relatedness between 1000 UnRelated (UR) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between half sibs = 0.0 is indicated using the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of estimates of genetic relatedness between 1000 UnRelated (UR) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between half sibs = 0.0 is indicated using the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F11.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 11" src="https://www.g3journal.org/content/ggg/8/10/3203/F11.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 11" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491009" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 11</span> <p id="p-49" class="first-child">Distribution of estimates of genetic relatedness between 1000 UnRelated (UR) dyads with increasing degree of subpopulation structure using (a) MC2013, and (b) MC2013WI estimators implemented in InRelate. Number of subpopulations (K) here was varied between K = 3 to K = 15 under the hierarchical island model described in Scenario 1. True relatedness between half sibs = 0.0 is indicated using the dotted red line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">The other estimators that did not account for population structure consistently over-, or under-estimated genetic relatedness between dyads, with large mean squared errors (MSE). It was also noted (<a id="xref-ref-56-18" class="xref-bibr" href="#ref-56">Wang 2011b</a>) that all estimators that ignored population genetic structure had increasing bias, with an increase in the degree of population genetic structure, except in the inference of PO dyads, and UR dyads.</p><p id="p-51">Correspondingly, MC2013 had the lowest MSE in the estimation of relatedness in FS, PO, and HS dyads, while the methods of AW2007 and Wang2011 had the lowest MSE for FC and UR dyads. The Ritland (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ritland 2005</a>) estimator, and the methods of <a id="xref-ref-3-21" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-19" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, had the highest MSE for PO dyads, while the Ritland estimator (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Ritland 2005</a>) had the highest MSE in all the cases. The estimators of <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Queller and Goodnight (1989)</a> (QuellerG), <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Lynch and Ritland (1999)</a> (LynchRi), and <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Wang (2007)</a> (TrioML) performed similarly, with higher bias and MSE, than MC2013. Also, the estimators of <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Ritland (1996)</a>, <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Queller and Goodnight (1989)</a> may have values <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-172.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-173.gif"/></noscript></span></span> but these were not truncated to fall inside this range, as performed by <a id="xref-ref-56-20" class="xref-bibr" href="#ref-56">Wang (2011b)</a> in order to observe the true trend in estimation of relatedness.</p></div><div id="sec-21" class="subsection"><h3>Scenario 2: Island Model</h3><p id="p-52">In the absence of population structure, under a panmictic island model, all methods performed comparably, with low MSE and bias for all FS, PO, HS dyads. The method of <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Ritland (1996)</a> had considerably higher MSE compared to all the other methods in the estimation of FS, PO, HS and FC dyads. The MC2013 and MC2013WI estimators have higher MSE and bias in determining relatedness between FC and UR dyads (see <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figures 12</a>, <a id="xref-fig-13-1" class="xref-fig" href="#F13">13</a>).</p><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from a panmictic population (K = 1) under Scenario 2, as described in the methods. Methods compared in this figure are those of Anderson and Weir (2007), Wang (2011b), Wang (2007), Wang (2002), Lynch (1988), and Lynch and Ritland (1999)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from a panmictic population (K = 1) under Scenario 2, as described in the methods. Methods compared in this figure are those of Anderson and Weir (2007), Wang (2011b), Wang (2007), Wang (2002), Lynch (1988), and Lynch and Ritland (1999).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F12.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 12" src="https://www.g3journal.org/content/ggg/8/10/3203/F12.medium.gif" width="440" height="204"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 12" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491142" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 12</span> <p id="p-53" class="first-child">(a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from a panmictic population (K = 1) under Scenario 2, as described in the methods. Methods compared in this figure are those of <a id="xref-ref-3-22" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, <a id="xref-ref-56-21" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Wang (2007)</a>, <a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Wang (2002)</a>, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Lynch (1988)</a>, and <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Lynch and Ritland (1999)</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from a panmictic population (K = 1) under Scenario 2, as described in the methods. Methods compared in this figure are MC2013, MC2013WI, Ritland (1996), Queller and Goodnight (1989), and Milligan (2003)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from a panmictic population (K = 1) under Scenario 2, as described in the methods. Methods compared in this figure are MC2013, MC2013WI, Ritland (1996), Queller and Goodnight (1989), and Milligan (2003).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 13" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F13.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 13" src="https://www.g3journal.org/content/ggg/8/10/3203/F13.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 13" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491147" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 13</span> <p id="p-54" class="first-child">(a) MSE and (b) Bias in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from a panmictic population (K = 1) under Scenario 2, as described in the methods. Methods compared in this figure are MC2013, MC2013WI, <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Ritland (1996)</a>, <a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Queller and Goodnight (1989)</a>, and <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Milligan (2003)</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Scenario 3: Effect of Number of Loci</h3><p id="p-55">Bias and MSE estimates of pairwise genetic relatedness in FS dyads showed a trend of decrease with an increase in the number of loci (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Figures 14</a>, <a id="xref-fig-15-1" class="xref-fig" href="#F15">15</a>, <a id="xref-fig-16-1" class="xref-fig" href="#F16">16</a>, <a id="xref-fig-17-1" class="xref-fig" href="#F17">17</a>, <a id="xref-fig-18-1" class="xref-fig" href="#F18">18</a>, <a id="xref-fig-19-1" class="xref-fig" href="#F19">19</a>) across all estimators at <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-174.gif"/></noscript></span></span> 5, and 10, indicating the relative better estimation with increased genotypic information. Estimates of relatedness at K = 3, 5 and 15 are shown in <a id="xref-fig-15-2" class="xref-fig" href="#F15">Figures 15</a>, <a id="xref-fig-17-2" class="xref-fig" href="#F17">17</a>, and <a id="xref-fig-19-2" class="xref-fig" href="#F19">19</a> respectively. In general, InRelate had the least bias and least MSE in estimation of FS dyads across different levels of available information, measured as a function of the number of loci, with and without accounting for inbreeding (<a id="xref-fig-14-2" class="xref-fig" href="#F14">Figures 14</a>, <a id="xref-fig-16-2" class="xref-fig" href="#F16">16</a>, <a id="xref-fig-18-2" class="xref-fig" href="#F18">18</a>). The estimator that accounted for inbreeding (MC2013WI) outperformed all other estimators with the least bias and MSE in estimation of FS relatedness. All other estimators of relatedness which did, or did not did not account for subpopulation structure performed with consistent decrease in bias and MSE with increase in the number of analyzed loci, as expected. The Ritland estimator was the least accurate, at <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-175.gif"/></noscript></span></span> across <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-176.gif"/></noscript></span></span> followed by the estimators of <a id="xref-ref-3-23" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, and <a id="xref-ref-56-22" class="xref-bibr" href="#ref-56">Wang (2011b)</a>.</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from  subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-177&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 14" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F14.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 14" src="https://www.g3journal.org/content/ggg/8/10/3203/F14.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 14" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491014" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 14</span> <p id="p-56" class="first-child">Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F14/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F14/embed/mml-math-177.gif"/></noscript></span></span> subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40.</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 3), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 3), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 15" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F15.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 15" src="https://www.g3journal.org/content/ggg/8/10/3203/F15.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 15" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491019" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 15</span> <p id="p-57" class="first-child">(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 3), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line.</p><div class="sb-div caption-clear"></div></div></div><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from  subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-178&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 16" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F16.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 16" src="https://www.g3journal.org/content/ggg/8/10/3203/F16.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 16" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491048" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 16</span> <p id="p-58" class="first-child">Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F16/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F16/embed/mml-math-178.gif"/></noscript></span></span> subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40.</p><div class="sb-div caption-clear"></div></div></div><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 5), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 5), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 17" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F17.medium.gif" width="440" height="189"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 17" src="https://www.g3journal.org/content/ggg/8/10/3203/F17.medium.gif" width="440" height="189"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 17" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491003" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 17</span> <p id="p-59" class="first-child">(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 5), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line.</p><div class="sb-div caption-clear"></div></div></div><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from  subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-179&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 18" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F18.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 18" src="https://www.g3journal.org/content/ggg/8/10/3203/F18.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 18" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491001" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 18</span> <p id="p-60" class="first-child">Bias and Mean Squared Error in estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled from <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F18/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F18/embed/mml-math-179.gif"/></noscript></span></span> subpopulations, simulated under Scenario 5, with increasing number of genotyped loci between L = 10 and L = 40.</p><div class="sb-div caption-clear"></div></div></div><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 10), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 10), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 19" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F19.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 19" src="https://www.g3journal.org/content/ggg/8/10/3203/F19.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 19" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490991" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 19</span> <p id="p-61" class="first-child">(a) MC2013 and (b) MC2013WI estimates of genetic relatedness between 1000 Full Sib (FS) dyads sampled under Scenario 5 (K = 10), by varying the number of loci sampled between L = 10 to L = 40. True estimate of relatedness between Full Siblings = 0.5 is shown in the dotted red line.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h3>Scenario 4: Effect of method of estimating <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-180.gif"/></noscript></span></span></h3><p id="p-62">The methods of <a id="xref-ref-3-24" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-23" class="xref-bibr" href="#ref-56">Wang (2011b)</a> have larger confidence intervals in estimating relatedness in FS dyads, with the <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-181.gif"/></noscript></span></span> of <a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Weir and Cockerham (1984)</a> having lower deviation from the truth (<span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-182.gif"/></noscript></span></span>), compared to the theoretical <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-183.gif"/></noscript></span></span> The MC2013 and MC2013WI methods outperform both methods with smaller confidence intervals around the mean (as shown in <a id="xref-fig-20-1" class="xref-fig" href="#F20">Figure 20</a>).</p><div id="F20" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F20.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimates of relatedness for 1000 FS dyads simulated under the panmictic island model (K = 1). (a) Estimates of relatedness under Scenario 3 where method of estimating  was varied. MC1 denotes the method of MC2013, MC2 is MC2013 accounting for inbreeding, A1 is the method of Anderson and Weir (2007) using the estimated  of Weir and Cockerham (1984), W1 is the method of Wang (2011b) using estimated  A2 and W2 denote the above methods using expected  (b) Estimates of relatedness under Scenario 4, where the population ID&#x2019;s were shuffled to simulate &#x2018;label switching&#x2019;. MC1, MC2, A1 and W1 are the same as before. A2 and W2 are the methods of Anderson and Weir (2007) and Wang (2011b) respectively, with 0.1 proportion of labels shuffled, A3 and W3 have 0.5 proportion of labels shuffled, and A4 and W4 have 1.0 proportion of labels shuffled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimates of relatedness for 1000 FS dyads simulated under the panmictic island model (K = 1). (a) Estimates of relatedness under Scenario 3 where method of estimating &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-184&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; was varied. MC1 denotes the method of MC2013, MC2 is MC2013 accounting for inbreeding, A1 is the method of Anderson and Weir (2007) using the estimated &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-185&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; of Weir and Cockerham (1984), W1 is the method of Wang (2011b) using estimated &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-186&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; A2 and W2 denote the above methods using expected &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-187&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (b) Estimates of relatedness under Scenario 4, where the population ID&#x2019;s were shuffled to simulate &#x2018;label switching&#x2019;. MC1, MC2, A1 and W1 are the same as before. A2 and W2 are the methods of Anderson and Weir (2007) and Wang (2011b) respectively, with 0.1 proportion of labels shuffled, A3 and W3 have 0.5 proportion of labels shuffled, and A4 and W4 have 1.0 proportion of labels shuffled.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 20" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F20.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 20" src="https://www.g3journal.org/content/ggg/8/10/3203/F20.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 20" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491036" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 20</span> <p id="p-63" class="first-child">Estimates of relatedness for 1000 FS dyads simulated under the panmictic island model (K = 1). (a) Estimates of relatedness under Scenario 3 where method of estimating <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-184.gif"/></noscript></span></span> was varied. MC1 denotes the method of MC2013, MC2 is MC2013 accounting for inbreeding, A1 is the method of <a id="xref-ref-3-25" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> using the estimated <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-185.gif"/></noscript></span></span> of <a id="xref-ref-62-5" class="xref-bibr" href="#ref-62">Weir and Cockerham (1984)</a>, W1 is the method of <a id="xref-ref-56-24" class="xref-bibr" href="#ref-56">Wang (2011b)</a> using estimated <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-186.gif"/></noscript></span></span> A2 and W2 denote the above methods using expected <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/F20/embed/mml-math-187.gif"/></noscript></span></span> (b) Estimates of relatedness under Scenario 4, where the population ID’s were shuffled to simulate ‘label switching’. MC1, MC2, A1 and W1 are the same as before. A2 and W2 are the methods of <a id="xref-ref-3-26" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-25" class="xref-bibr" href="#ref-56">Wang (2011b)</a> respectively, with 0.1 proportion of labels shuffled, A3 and W3 have 0.5 proportion of labels shuffled, and A4 and W4 have 1.0 proportion of labels shuffled.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h3>Scenario 5: Effect of ‘label-switching’</h3><p id="p-64">InRelate estimators do not have problems with ‘label-switching’, since population assignment is determined by the clustering method, and hence the ancestry proportions and allele frequencies are recomputed every time. On the other hand, both the methods of <a id="xref-ref-3-27" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-26" class="xref-bibr" href="#ref-56">Wang (2011b)</a> show increased deviation from the mean (true <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-188.gif"/></noscript></span></span>) when labels are switched, due to the erroneous computation of differentiation (See <a id="xref-fig-20-2" class="xref-fig" href="#F20">Figure 20</a>).</p></div><div id="sec-25" class="subsection"><h3>Scenario 6: Bootstrapping</h3><p id="p-65">Out of the 5 dyads of FS, HS, PO, FC and UR categories were correctly assigned after 200 bootstrap iterations in 44% of pairs. All Parent-Offspring pairs were correctly assigned, while two each of FS, FC, and UR pairs were correctly assigned. None of the HS pairs were significantly assigned to any category. Plots of confidence intervals around estimates using both the MC2013 and MC2013WI estimators are shown in <a id="xref-fig-21-1" class="xref-fig" href="#F21">Figure 21</a>.</p><div id="F21" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F21.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) MC2013 and (b) MC2013WI relatedness estimates and confidence intervals for 5 different relatedness categories, constructed using 200 bootstrap replicates under Scenario 6. The simulation used K = 3 subpopulations, and a total of 5 dyads of FS, HS, PO, FC, and UR individuals were picked." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) MC2013 and (b) MC2013WI relatedness estimates and confidence intervals for 5 different relatedness categories, constructed using 200 bootstrap replicates under Scenario 6. The simulation used K = 3 subpopulations, and a total of 5 dyads of FS, HS, PO, FC, and UR individuals were picked.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 21" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F21.medium.gif" width="440" height="220"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 21" src="https://www.g3journal.org/content/ggg/8/10/3203/F21.medium.gif" width="440" height="220"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 21" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491006" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 21</span> <p id="p-66" class="first-child">(a) MC2013 and (b) MC2013WI relatedness estimates and confidence intervals for 5 different relatedness categories, constructed using 200 bootstrap replicates under Scenario 6. The simulation used K = 3 subpopulations, and a total of 5 dyads of FS, HS, PO, FC, and UR individuals were picked.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Scenario 7: HGDP-CEPH Panel</h3><p id="p-67">Across 24 dyads which were either identified as FS, HS (or Avuncular), or PO by <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Rosenberg <em>et al.</em> (2005)</a>, the MC2013 and MC2013WI estimators outperformed the methods of <a id="xref-ref-3-28" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-27" class="xref-bibr" href="#ref-56">Wang (2011b)</a> (see <a id="xref-fig-22-1" class="xref-fig" href="#F22">Figure 22</a>), with consistently lower bias (MC2013WI - mean bias = 0.0114 (sd = 0.0667), MC2013 - mean bias = 0.0114 (sd = 0.0667), AW2007 - mean bias = -0.2857 (sd = 0.0779), Wang2011 - mean bias = -0.3204 (sd = 0.0895)). The MSE was also considerably lower (MC2013WI - 0.0044, MC2013 - 0.0044, AW2007 - 0.0874, Wang2011 - 0.1103) when comparing the MC2013 estimators with AW2007 and Wang2011. As reported before, these populations are ‘moderately’ differentiated (with a <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-189.gif"/></noscript></span></span> = 0.1169), and have historically been reported to have significant levels of gene flow or admixture, as well as exhibiting serial founder effects (see <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tishkoff <em>et al.</em> 2009</a>, <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ramachandran <em>et al.</em> 2005</a>).</p><div id="F22" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F22.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relatedness estimates between 24 related dyads sampled from 6 locations in Africa, which were previously reported to be either FS, HS (or avuncular), or PO dyads by Rosenberg et al. (2005) using REAP (Thornton et al. 2012). The REAP estimates are plotted as the &#x2018;True&#x2019; estimate, while other estimators compared are those of MC2013, MC2013WI, AW2007, and Wang2011." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1147621584" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relatedness estimates between 24 related dyads sampled from 6 locations in Africa, which were previously reported to be either FS, HS (or avuncular), or PO dyads by Rosenberg &lt;em&gt;et al.&lt;/em&gt; (2005) using REAP (Thornton &lt;em&gt;et al.&lt;/em&gt; 2012). The REAP estimates are plotted as the &#x2018;True&#x2019; estimate, while other estimators compared are those of MC2013, MC2013WI, AW2007, and Wang2011.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 22" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3203/F22.medium.gif" width="428" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 22" src="https://www.g3journal.org/content/ggg/8/10/3203/F22.medium.gif" width="428" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 22" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3203/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491079" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 22</span> <p id="p-68" class="first-child">Relatedness estimates between 24 related dyads sampled from 6 locations in Africa, which were previously reported to be either FS, HS (or avuncular), or PO dyads by <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Rosenberg <em>et al.</em> (2005)</a> using REAP (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Thornton <em>et al.</em> 2012</a>). The REAP estimates are plotted as the ‘True’ estimate, while other estimators compared are those of MC2013, MC2013WI, AW2007, and Wang2011.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-27"><h2 class="">Discussion</h2><p id="p-69">The presence of ancestral subpopulation structure affects estimates of pairwise genetic relatedness between individuals from the same subpopulation, owing to pervasive inbreeding, and non-random mating in recent ancestral generations.</p><p id="p-70">The primary goal of this paper was to develop a maximum-likelihood framework using an alternate parametrization, to estimate pairwise genetic relatedness between two individuals <em>X</em> and <em>Y</em>, while accounting for the ‘true’ genetic subpopulation structure in the population. This ‘true’ genetic subpopulation structure is unobserved and can be inferred from the data. Since the proposal of an admixture model by <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Pritchard <em>et al.</em> (2000b)</a>, several tools have been developed to estimate subpopulation structure, primarily to infer the number of subpopulations, <em>K</em>, admixture proportions (here <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-190.gif"/></noscript></span></span>), and subpopulation allele frequencies, <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-191.gif"/></noscript></span></span> These estimates have been applied widely, including to infer ancestral migration patterns (<em>e.g.</em>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rosenberg <em>et al.</em> 2002</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eriksson and Manica 2012</a>), in association studies (<em>e.g.</em>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Collins-Schramm <em>et al.</em> 2002</a>), and to inform conservation decisions (see <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allendorf <em>et al.</em> 2010</a>). InRelate uses inferred information from population structure studies (using methods such as STRUCTURE (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Pritchard <em>et al.</em> 2000b</a>) or MULTICLUST (<a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Sethuraman 2013</a>) - see <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2013</a>) to inform the estimation of relatedness.</p><p id="p-71">Across my simulations and analyses of the HGDP-CEPH African datasets, InRelate estimators of relatedness (MC2013 and MC2013WI) outperform several previously developed methods for relatedness estimation in admixed populations with considerably less error and bias. This accuracy is more pronounced particularly in between pairs of full siblings, parent-offspring, or half-siblings. The previously developed methods of <a id="xref-ref-3-29" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-28" class="xref-bibr" href="#ref-56">Wang (2011b)</a> outperform InRelate in estimating first cousins or unrelated dyads in my simulations. As noted by <a id="xref-ref-3-30" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, estimates of relatedness in unrelated individuals are upwardly biased by all methods (see <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>)). I surmise this result is an artifact of ignoring subpopulation structure, in the presence of undetected ancient admixture, which results in an upward bias for all estimates. While MC2013 and MC2013WI account for this by using estimated subpopulation allele frequencies, the other estimators (AW2007(<a id="xref-ref-3-31" class="xref-bibr" href="#ref-3">Anderson and Weir 2007</a>, Wang2011(<a id="xref-ref-56-29" class="xref-bibr" href="#ref-56">Wang 2011b</a>) approximate it by using current allele frequencies, estimated from sampled populations.</p><p id="p-72">Of note though are general difficulties in estimation of relatedness between first cousins, second cousins, and other more distantly related or unrelated pairs. These are also seen and reported by other likelihood methods (see <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Thompson 1975</a>, <a id="xref-ref-3-32" class="xref-bibr" href="#ref-3">Anderson and Weir 2007</a>, <a id="xref-ref-56-30" class="xref-bibr" href="#ref-56">Wang 2011b</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Konovalov and Heg 2008</a>), other estimators that use summary statistics (see <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Lynch and Ritland 1999</a>, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Blouin 2003</a>, <a id="xref-ref-3-33" class="xref-bibr" href="#ref-3">Anderson and Weir 2007</a>, <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">Wang 2002</a>), and methods that utilize linkage or recombination information (see <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pemberton <em>et al.</em> 2013</a>, <a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Rosenberg <em>et al.</em> 2005</a>). This is primarily due to the fact that the most predominant relationship between two individuals is usually inferred, while the historical relatedness, due to evolutionary demographic processes, between them is ignored by most methods. Methods that account for this ‘deep’ relatedness are yet to be devised, and could help resolve issues with estimating deeper pedigrees, and relatedness between individuals. <a id="xref-ref-56-31" class="xref-bibr" href="#ref-56">Wang (2011b)</a> also notes this bias in estimating relatedness values close to the lower bound of 0 in the methods of <a id="xref-ref-3-34" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-32" class="xref-bibr" href="#ref-56">Wang (2011b)</a>.</p><p id="p-73">Varying the number of loci minimally affects all relatedness estimators. This outcome may derive from variation in allele frequencies being sufficiently explained by the parameters of the admixture model (admixture proportions and subpopulation allele frequencies), as against biasing all estimates using a single non-varying parameter, <em>θ</em> (or <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3203/embed/mml-math-192.gif"/></noscript></span></span>, <em>sensu</em> <a id="xref-ref-3-35" class="xref-bibr" href="#ref-3">Anderson and Weir 2007</a> and <a id="xref-ref-56-33" class="xref-bibr" href="#ref-56">Wang 2011b</a>). Several methods can estimate this coefficient <em>θ</em> and each method has its own biases and efficiencies. This approach could potentially cause increased bias and MSE in using the estimators of <a id="xref-ref-3-36" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a> and <a id="xref-ref-56-34" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, which could be addressed by utilizing a population structuring method to assign individuals to subpopulations, conditioning on that population structure in estimating <em>θ</em>. Regardless, increasing the number of sampled loci decreased bias of all estimators, as expected.</p><p id="p-74">InRelate estimators do not have problems with ‘label-switching’, since the subpopulation structure is inferred from the data and not assumed <em>a priori</em> as in all other methods. Correspondingly, all allele frequencies, and ancestry proportions are re-calculated with switched labels, which are then used in estimates of relatedness. While all my analyses have inferred admixture proportions at the assumed ‘true’ subpopulation structure (<em>i.e.</em>, K), perhaps the true utility of this method would be if this K was inferred from the data, and the corresponding inferred admixture proportions and allele frequencies used in the estimation of relatedness. However, this is a statistical problem (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Pritchard <em>et al.</em> 2000b</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Falush <em>et al.</em> 2003</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hubisz <em>et al.</em> 2009</a>, <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Sethuraman 2013</a>, <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>), with estimates of subpopulation allele frequencies and ancestry proportions confounded by (1) different demographic histories (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Falush <em>et al.</em> 2016</a>, (2) overparametrization, and a general improvement in the likelihood with increasing the parameter <em>K</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Evanno <em>et al.</em> 2005</a>), and (3) issues with label switching (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jakobsson and Rosenberg 2007</a>). InRelate and the method of <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Moltke and Albrechtsen (2013)</a> are hence both affected by the ‘accuracy’ of estimates of structure and admixture parameters.</p><p id="p-75">InRelate methods are of best utility when dealing with multi-allelic data, generated from individuals that are sampled from populations that are ancestrally structured, and generally outdo the methods of <a id="xref-ref-3-37" class="xref-bibr" href="#ref-3">Anderson and Weir (2007)</a>, and <a id="xref-ref-56-35" class="xref-bibr" href="#ref-56">Wang (2011b)</a>, which are both relatedness estimators under similar models. InRelate also does not require linkage maps, which makes it more utilitarian for estimating relatedness in non-model systems that don’t have detailed genomic information. I have also shown that InRelate outperforms all the methods implemented in the COANCESTRY (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Wang 2011a</a>) software, since all these methods do not account for ancestral population structure. However, the RelateAdmix method of <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Moltke and Albrechtsen (2013)</a>, which has been shown to outperform the methods of REAP (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Thornton <em>et al.</em> 2012</a>), PLink (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Purcell <em>et al.</em> 2007</a>), and KING (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Manichaikul <em>et al.</em> 2010</a>) is more applicable when analyzing SNP (di-allelic) data, generated from non-inbred populations that are recently admixed. When the underlying demographic history of the sampled individuals is unknown (or difficult to estimate), methods that are model-free, such as PC-Relate (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Conomos <em>et al.</em> 2016</a>) are bound to perform better (summarized in <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ramstetter <em>et al.</em> 2017</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-76">AS designed the method, wrote the code, performed all simulations, analyses, and wrote the paper. This work was part of AS’s doctoral thesis, and he would like to thank his doctoral co-advisors, Karin S Dorman and Fredric J Janzen for their guidance and help throughout the process. AS continues to work with KSD on improvements to InRelate. More recently, completion of this project was made possible by an NSF ABI Development Grant 1564659 to AS. All analyses reported were performed on HPC facilities at ISU, CSUSM, and Temple University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-77"><em>Communicating editor: J. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-06-06"><span class="received-label">Received </span>June 6, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-30"><span class="accepted-label">Accepted </span>July 30, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Sethuraman</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.2" data-doi="10.1038/nrg2844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomics and the future of conservation genetics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">709</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2844</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D697%26rft.epage%253D709%26rft.atitle%253DGenomics%2Band%2Bthe%2Bfuture%2Bof%2Bconservation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2844%26rft_id%253Dinfo%253Apmid%252F20847747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20847747&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.3" data-doi="10.1534/genetics.106.063149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A maximum-likelihood method for the estimation of pairwise relatedness in structured populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.063149</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.063149%26rft_id%253Dinfo%253Apmid%252F17339212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc2LzEvNDIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.4" data-doi="10.1093/jhered/92.2.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avise</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">DNA-based Profiling of Mating Systems and Reproductive Behaviors in Poikilothermic Vertebrates: AGA Symposium Issue, Yale University, New Haven, Connecticut, June 17–20, 2000.</span> <abbr class="cit-jnl-abbrev">Oxford University Press</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/92.2.99</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DAvise%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D99%26rft.epage%253D99%26rft.atitle%253DIntroduction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.2.99%26rft_id%253Dinfo%253Apmid%252F11700602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/92.2.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11700602&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169006700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.5" data-doi="10.1093/jhered/92.3.301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">EASYPOP (version 1.7): a computer program for population genetics simulations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/92.3.301</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBalloux%26rft.auinit1%253DF.%26rft.volume%253D92%26rft.issue%253D3%26rft.spage%253D301%26rft.epage%253D302%26rft.atitle%253DEASYPOP%2B%2528Version%2B1.7%2529%253A%2BA%2BComputer%2BProgram%2Bfor%2BPopulation%2BGenetics%2BSimulations%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.3.301%26rft_id%253Dinfo%253Apmid%252F11447253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/92.3.301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11447253&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169680200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.6" data-doi="10.1016/S0169-5347(03)00225-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blouin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dna-based methods for pedigree reconstruction and kinship analysis in natural populations.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0169-5347(03)00225-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D18%26rft.spage%253D503%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252803%252900225-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(03)00225-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185728100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.7" data-doi="10.1086/514862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Accurate inference of relationships in sib-pair linkage studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/514862</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBoehnke%26rft.auinit1%253DM.%26rft.volume%253D61%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D429%26rft.atitle%253DAccurate%2Binference%2Bof%2Brelationships%2Bin%2Bsib-pair%2Blinkage%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F514862%26rft_id%253Dinfo%253Apmid%252F9311748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/514862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9311748&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XX70100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.8" data-doi="10.1126/science.296.5566.261b"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Toma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cazes</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legrand</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A human genome diversity cell line panel.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">262</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.296.5566.261b</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCann%26rft.auinit1%253DH.%2BM.%26rft.volume%253D296%26rft.issue%253D5566%26rft.spage%253D261%26rft.epage%253D262%26rft.atitle%253DA%2Bhuman%2Bgenome%2Bdiversity%2Bcell%2Bline%2Bpanel.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.296.5566.261b%26rft_id%253Dinfo%253Apmid%252F11954565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.296.5566.261b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11954565&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175000300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.9" data-doi="10.1111/j.1095-8312.2011.01673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Patterns of multiple paternity and maternity in fishes.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">735</span>–<span class="cit-lpage">760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1095-8312.2011.01673.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D103%26rft.spage%253D735%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2011.01673.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2011.01673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000292865000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.10" data-doi="10.1086/339368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collins-Schramm</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Operario</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Ethnic-difference markers for use in mapping by admixture linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/339368</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DCollins-Schramm%26rft.auinit1%253DH.%2BE.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D737%26rft.epage%253D750%26rft.atitle%253DEthnic-difference%2Bmarkers%2Bfor%2Buse%2Bin%2Bmapping%2Bby%2Badmixture%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F339368%26rft_id%253Dinfo%253Apmid%252F11845411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/339368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11845411&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173880000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.11" data-doi="10.1016/j.ajhg.2015.11.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conomos</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reiner</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Model-free estimation of recent genetic relatedness.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dempster</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laird</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Maximum likelihood from incomplete data via em algorithm.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D39%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.13" data-doi="10.1086/321195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duren</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boehnke</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Improved inference of relationship for pairs of individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1231</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0002-9297(07)62952-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DEpstein%26rft.auinit1%253DM.%2BP.%26rft.volume%253D67%26rft.issue%253D5%26rft.spage%253D1219%26rft.epage%253D1231%26rft.atitle%253DImproved%2Binference%2Bof%2Brelationship%2Bfor%2Bpairs%2Bof%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321195%26rft_id%253Dinfo%253Apmid%252F11032786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11032786&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165091600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.14" data-doi="10.1073/pnas.1200567109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manica</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effect of ancient population structure on the degree of polymorphism shared between modern human populations and ancient hominins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">13956</span>–<span class="cit-lpage">13960</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1200567109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1200567109%26rft_id%253Dinfo%253Apmid%252F22893688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM1LzEzOTU2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.15" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software structure: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02553.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3203.16"><div class="cit-metadata unstructured">Falconer, D. S., and T. F. C. Mackay, 1996 Introduction to Quantitative Genetics, Ed 4. Longmans Green, Harlow, Essex, UK.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: Linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.18" data-doi="10.1111/j.1471-8286.2007.01758.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: dominant markers and null alleles.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">574</span>–<span class="cit-lpage">578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1471-8286.2007.01758.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BNotes%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D574%26rft.epage%253D578%26rft.atitle%253DInference%2Bof%2Bpopulation%2Bstructure%2Busing%2Bmultilocus%2Bgenotype%2Bdata%253A%2Bdominant%2Bmarkers%2Band%2Bnull%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01758.x%26rft_id%253Dinfo%253Apmid%252F18784791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01758.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18784791&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247758100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dorp</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A tutorial on how (not) to over-interpret structure/admixture bar plots.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">066431</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-web" id="cit-8.10.3203.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghalanos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theussl</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-comment">Rsolnp: general non-linear optimization using augmented lagrange multiplier method. R package version 1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.21" data-doi="10.1111/j.1755-0998.2009.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring weak population structure with the assistance of sample group information.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1322</span>–<span class="cit-lpage">1332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2009.02591.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DHubisz%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D1322%26rft.epage%253D1332%26rft.atitle%253DInferring%2Bweak%2Bpopulation%2Bstructure%2Bwith%2Bthe%2Bassistance%2Bof%2Bsample%2Bgroup%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2009.02591.x%26rft_id%253Dinfo%253Apmid%252F21564903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2009.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21564903&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.22" data-doi="10.2307/2528643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacquard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic information given by a relative.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1114</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2528643</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DJacquard%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D1101%26rft.epage%253D1114%26rft.atitle%253DGenetic%2Binformation%2Bgiven%2Bby%2Ba%2Brelative.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528643%26rft_id%253Dinfo%253Apmid%252F4648793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528643&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4648793&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1972O313600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.23" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clumpp: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.24" data-doi="10.1111/j.1471-8286.2007.01940.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konovalov</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heg</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">TECHNICAL ADVANCES: A maximum-likelihood relatedness estimator allowing for negative relatedness values.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">263</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1471-8286.2007.01940.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DKonovalov%26rft.auinit1%253DD.%2BA.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D263%26rft.atitle%253DTECHNICAL%2BADVANCES%253A%2BA%2Bmaximum-likelihood%2Brelatedness%2Bestimator%2Ballowing%2Bfor%2Bnegative%2Brelatedness%2Bvalues.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01940.x%26rft_id%253Dinfo%253Apmid%252F21585767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01940.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21585767&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.10.3203.25" data-doi="10.1159/000154113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Similarity of dna fingerprints due to chance and relatedness.</span> <abbr class="cit-jnl-abbrev">Hum. Hered.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000154113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bheredity%26rft.stitle%253DHum%2BHered%26rft.aulast%253DLi%26rft.auinit1%253DC.%2BC.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D52%26rft.atitle%253DSimilarity%2Bof%2BDNA%2Bfingerprints%2Bdue%2Bto%2Bchance%2Band%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000154113%26rft_id%253Dinfo%253Apmid%252F8514326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000154113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8514326&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KR38200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.26" data-doi="10.1186/1479-7364-7-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyunoya</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belinsky</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesfaigzi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Softwares and methods for estimating genetic ancestry in human populations.</span> <abbr class="cit-jnl-abbrev">Hum. Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1479-7364-7-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenomics%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1479-7364-7-1%26rft_id%253Dinfo%253Apmid%252F23289408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1479-7364-7-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289408&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Estimation of relatedness by dna fingerprinting.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">584</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DLynch%26rft.auinit1%253DM%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D584%26rft.epage%253D599%26rft.atitle%253DEstimation%2Bof%2Brelatedness%2Bby%2BDNA%2Bfingerprinting%26rft_id%253Dinfo%253Apmid%252F3193879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3193879&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988Q370900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Estimation of pairwise relatedness with molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D1753%26rft.epage%253D1766%26rft.atitle%253DEstimation%2Bof%2BPairwise%2BRelatedness%2BWith%2BMolecular%2BMarkers%26rft_id%253Dinfo%253Apmid%252F10430599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi80LzE3NTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.29" data-doi="10.1093/bioinformatics/btq559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mychaleckyj</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rich</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Robust relationship inference in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2867</span>–<span class="cit-lpage">2873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq559</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq559%26rft_id%253Dinfo%253Apmid%252F20926424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20926424&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283919800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milligan</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Maximum-likelihood estimation of relatedness.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMilligan%26rft.auinit1%253DB.%2BG.%26rft.volume%253D163%26rft.issue%253D3%26rft.spage%253D1153%26rft.epage%253D1167%26rft.atitle%253DMaximum-Likelihood%2BEstimation%2Bof%2BRelatedness%26rft_id%253Dinfo%253Apmid%252F12663552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2My8zLzExNTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.31" data-doi="10.1093/bioinformatics/btt652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moltke</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Relateadmix: a software tool for estimating relatedness between admixed individuals.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1028</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt652</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt652%26rft_id%253Dinfo%253Apmid%252F24215025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24215025&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334078300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.32" data-doi="10.1111/j.1469-1809.1983.tb00993.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesser</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Estimation of fixation indices and gene diversities.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1983.tb00993.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D47%26rft.issue%253DPt%2B3%26rft.spage%253D253%26rft.epage%253D259%26rft.atitle%253DEstimation%2Bof%2Bfixation%2Bindices%2Band%2Bgene%2Bdiversities.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1983.tb00993.x%26rft_id%253Dinfo%253Apmid%252F6614868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1983.tb00993.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6614868&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RB52000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.33" data-doi="10.1534/genetics.105.049940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliehoek</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Estimating relatedness between individuals in general populations with a focus on their use in conservation programs.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.049940</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049940%26rft_id%253Dinfo%253Apmid%252F16510792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.34" data-doi="10.1007/s00265-001-0421-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearse</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janzen</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avise</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Multiple paternity, sperm storage, and reproductive success of female and male painted turtles (chrysemys picta) in nature.</span> <abbr class="cit-jnl-abbrev">Behav. Ecol. Sociobiol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00265-001-0421-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BEcol.%2BSociobiol.%26rft.volume%253D51%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00265-001-0421-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00265-001-0421-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173559000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.35" data-doi="10.1534/g3.113.005728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeGiorgio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population structure in a comprehensive genomic data set on human microsatellite variation.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">907</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.005728</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.005728%26rft_id%253Dinfo%253Apmid%252F23550135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvNS84OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3203.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Press</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">Numerical recipes 3rd edition: The art of scientific computing</span>. <span class="cit-publ-name">Cambridge university press</span>, <span class="cit-publ-loc">Cambridge, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.37" data-doi="10.1086/302959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">a</span> <span class="cit-article-title">Association mapping in structured populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">181</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/302959</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D67%26rft.issue%253D1%26rft.spage%253D170%26rft.epage%253D181%26rft.atitle%253DAssociation%2Bmapping%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302959%26rft_id%253Dinfo%253Apmid%252F10827107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827107&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088926900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.38" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span><span class="cit-comment">b</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.39" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Plink: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.40" data-doi="10.2307/2409206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Queller</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodnight</span>  <span class="cit-name-given-names">K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Estimating relatedness using genetic-markers.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">275</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1989.tb04226.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D43%26rft.spage%253D258%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989T799700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.41" data-doi="10.1073/pnas.0507611102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roseman</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in africa.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">15942</span>–<span class="cit-lpage">15947</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0507611102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0507611102%26rft_id%253Dinfo%253Apmid%252F16243969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQ0LzE1OTQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3203.42" data-doi="10.1534/genetics.117.1122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramstetter</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehman</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curran</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duggirala</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Benchmarking relatedness inference methods with genome-wide data from thousands of relatives.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">207</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.1122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.1122%26rft_id%253Dinfo%253Apmid%252F28739658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA3LzEvNzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.43" data-doi="10.1017/S0016672300033620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Estimators for pairwise relatedness and individual inbreeding coefficients.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300033620</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D67%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033620&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UK49900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.44" data-doi="10.1111/j.1365-294X.2005.02667.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Multilocus estimation of pairwise relatedness with dominant markers.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">3157</span>–<span class="cit-lpage">3165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02667.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRitland%26rft.auinit1%253DK.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D3157%26rft.epage%253D3165%26rft.atitle%253DMultilocus%2Bestimation%2Bof%2Bpairwise%2Brelatedness%2Bwith%2Bdominant%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02667.x%26rft_id%253Dinfo%253Apmid%252F16101781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02667.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16101781&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.45" data-doi="10.1111/j.1469-1809.2006.00285.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Standardized subsets of the hgdp-ceph human genome diversity cell line panel, accounting for atypical and duplicated samples and pairs of close relatives.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.2006.00285.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D70%26rft.issue%253DPt%2B6%26rft.spage%253D841%26rft.epage%253D847%26rft.atitle%253DStandardized%2Bsubsets%2Bof%2Bthe%2BHGDP-CEPH%2BHuman%2BGenome%2BDiversity%2BCell%2BLine%2BPanel%252C%2Baccounting%2Bfor%2Batypical%2Band%2Bduplicated%2Bsamples%2Band%2Bpairs%2Bof%2Bclose%2Brelatives.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.2006.00285.x%26rft_id%253Dinfo%253Apmid%252F17044859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.2006.00285.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17044859&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241191400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.46" data-doi="10.1371/journal.pgen.0020215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Quevedo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nino-Rosales</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Low levels of genetic divergence across geographically and linguistically diverse populations from india.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020215</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De215%26rft.epage%253De215%26rft.atitle%253DLow%2Blevels%2Bof%2Bgenetic%2Bdivergence%2Bacross%2Bgeographically%2Band%2Blinguistically%2Bdiverse%2Bpopulations%2Bfrom%2BIndia.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020215%26rft_id%253Dinfo%253Apmid%252F17194221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194221&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.47" data-doi="10.1371/journal.pgen.0010070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Clines, clusters, and the effect of study design on the inference of human population structure.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0010070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De70%26rft.epage%253De70%26rft.atitle%253DClines%252C%2Bclusters%252C%2Band%2Bthe%2Beffect%2Bof%2Bstudy%2Bdesign%2Bon%2Bthe%2Binference%2Bof%2Bhuman%2Bpopulation%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010070%26rft_id%253Dinfo%253Apmid%252F16355252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16355252&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.48" data-doi="10.1126/science.1078311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cann</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic structure of human populations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">298</span>: <span class="cit-fpage">2381</span>–<span class="cit-lpage">2385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1078311</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D298%26rft.issue%253D5602%26rft.spage%253D2381%26rft.epage%253D2385%26rft.atitle%253DGenetic%2BStructure%2Bof%2BHuman%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1078311%26rft_id%253Dinfo%253Apmid%252F12493913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTgvNTYwMi8yMzgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3203.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sethuraman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> On inferring and interpreting genetic population structure-applications to conservation, and the estimation of pairwise genetic relatedness.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.50" data-doi="10.1111/j.1469-1809.1975.tb00120.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Estimation of pairwise relationships.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">188</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1975.tb00120.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DThompson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D188%26rft.atitle%253DThe%2Bestimation%2Bof%2Bpairwise%2Brelationships.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1975.tb00120.x%26rft_id%253Dinfo%253Apmid%252F1052764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1975.tb00120.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1052764&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AX72600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.51" data-doi="10.1016/j.ajhg.2012.05.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochs-Balcom</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caan</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating kinship in admixed populations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2012.05.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DThornton%26rft.auinit1%253DT.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D122%26rft.epage%253D138%26rft.atitle%253DEstimating%2Bkinship%2Bin%2Badmixed%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.05.024%26rft_id%253Dinfo%253Apmid%252F22748210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.05.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22748210&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.52" data-doi="10.1126/science.1172257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedlaender</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehret</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranciaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic structure and history of Africans and African Americans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1044</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1172257</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1172257%26rft_id%253Dinfo%253Apmid%252F19407144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzMC8xMDM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.53" data-doi="10.1038/nrg2322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heritability in the genomics era - concepts and misconceptions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.54" data-doi="10.1017/S0016672307008798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Triadic ibd coefficients and applications to estimating pairwise relatedness.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672307008798</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D89%26rft.issue%253D3%26rft.spage%253D135%26rft.epage%253D153%26rft.atitle%253DTriadic%2BIBD%2Bcoefficients%2Band%2Bapplications%2Bto%2Bestimating%2Bpairwise%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672307008798%26rft_id%253Dinfo%253Apmid%252F17894908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672307008798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17894908&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250908100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.55" data-doi="10.1111/j.1755-0998.2010.02885.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Coancestry: a program for simulating, estimating and analysing relatedness and inbreeding coefficients.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1755-0998.2010.02885.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D141%26rft.epage%253D145%26rft.atitle%253DCOANCESTRY%253A%2Ba%2Bprogram%2Bfor%2Bsimulating%252C%2Bestimating%2Band%2Banalysing%2Brelatedness%2Band%2Binbreeding%2Bcoefficients.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2010.02885.x%26rft_id%253Dinfo%253Apmid%252F21429111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2010.02885.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21429111&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.56" data-doi="10.1534/genetics.110.124438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Unbiased relatedness estimation in structured populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">901</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.124438</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.124438%26rft_id%253Dinfo%253Apmid%252F21212234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg3LzMvODg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.57" data-doi="10.1111/1755-0998.12708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Effects of sampling close relatives on some elementary population genetics analyses.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">54</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1755-0998.12708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D18%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An estimator for pairwise relatedness using molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1203%26rft.epage%253D1215%26rft.atitle%253DAn%2BEstimator%2Bfor%2BPairwise%2BRelatedness%2BUsing%2BMolecular%2BMarkers%26rft_id%253Dinfo%253Apmid%252F11901134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzEyMDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.59" data-doi="10.1146/annurev.ge.28.120194.003121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The effects of inbreeding on forensic calculations.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ge.28.120194.003121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DWeir%26rft.auinit1%253DB.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D597%26rft.epage%253D621%26rft.atitle%253DThe%2BEffects%2Bof%2BInbreeding%2Bon%2BForensic%2BCalculations%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ge.28.120194.003121%26rft_id%253Dinfo%253Apmid%252F7893141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ge.28.120194.003121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7893141&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PY64400025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.60" data-doi="10.1520/JFS2003039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Matching and partially-matching dna profiles.</span> <abbr class="cit-jnl-abbrev">J. Forensic Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1009</span>–<span class="cit-lpage">1014</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1520/JFS2003039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BForensic%2BSciences%26rft.stitle%253DJournal%2Bof%2BForensic%2BSciences%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D49%26rft.issue%253D5%26rft.spage%253D1009%26rft.epage%253D1014%26rft.atitle%253DMatching%2Band%2Bpartially-matching%2BDNA%2Bprofiles.%26rft_id%253Dinfo%253Adoi%252F10.1520%252FJFS2003039%26rft_id%253Dinfo%253Apmid%252F15461102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1520/JFS2003039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15461102&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.61" data-doi="10.1038/nrg1960"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hepler</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic relatedness analysis: modern data and new challenges.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1960</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253D771%26rft.epage%253D780%26rft.atitle%253DGenetic%2Brelatedness%2Banalysis%253A%2Bmodern%2Bdata%2Band%2Bnew%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1960%26rft_id%253Dinfo%253Apmid%252F16983373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1960&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16983373&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241158700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.62" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating f-statistics for the analysis of population-structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft_id%253Dinfo%253Apmid%252F28563791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28563791&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.63" data-doi="10.1038/166247a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Genetical structure of populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/166247a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D166%26rft.issue%253D4215%26rft.spage%253D247%26rft.epage%253D249%26rft.atitle%253DGenetical%2Bstructure%2Bof%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F166247a0%26rft_id%253Dinfo%253Apmid%252F15439261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/166247a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15439261&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1950UA21400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3203.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> Interior algorithms for linear, quadratic, and linearly constrained convex programming.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3203.65" data-doi="10.1371/journal.pone.0012721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular evidence for high frequency of multiple paternity in a freshwater shrimp species caridina ensifera.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12721</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0012721</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DYue%26rft.auinit1%253DG.%2BH.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De12721%26rft.epage%253De12721%26rft.atitle%253DMolecular%2Bevidence%2Bfor%2Bhigh%2Bfrequency%2Bof%2Bmultiple%2Bpaternity%2Bin%2Ba%2Bfreshwater%2Bshrimp%2Bspecies%2BCaridina%2Bensifera.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012721%26rft_id%253Dinfo%253Apmid%252F20856862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20856862&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3203.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3185" title="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3221" title="Null Mutation of the Fascin2 Gene by TALEN Leading to Progressive Hearing Loss and Retinal Degeneration in C57BL/6J Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3203" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fLpBvxYTLN2K2Aa1imq3bBdghLt6iMspbZrx9547prE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3203" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3203" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3203" class="active" data-icon-position="" data-hide-link-title="0">Estimating Genetic Relatedness in Admixed Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490975" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-M7CcV2W6O1dHGmyxq-xV1uQ8ehqzfXGLptMi6elKhe0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3203.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3203" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0XfJAbJVC7zPal8MSlYx3gRUviP59RaakfNSnZU5tzE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3203" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3203.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490975" id="node490975--2" data-pisa="ggg;8/10/3203" data-pisa-master="ggg;g3.118.200485" data-apath="/ggg/8/10/3203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Estimating Genetic Relatedness in Admixed Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8201-8292" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arun Sethuraman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3203-3220;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200485 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490975--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arun Sethuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, California State University San Marcos, CA 92096</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arun%2BSethuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sethuraman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArun%2BSethuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8201-8292" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arun Sethuraman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asethuraman@csusm.edu" class="" data-icon-position="" data-hide-link-title="0">asethuraman@csusm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3203" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3203&amp;title=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3203&amp;title=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3203&amp;title=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3203&amp;text=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3203&amp;title=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3203&amp;t=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3203&amp;title=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3203&amp;title=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490975/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490975" id="node490975--3" data-pisa="ggg;8/10/3203" data-pisa-master="ggg;g3.118.200485" data-apath="/ggg/8/10/3203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Estimating Genetic Relatedness in Admixed Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8201-8292" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arun Sethuraman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3203-3220;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200485 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490975--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arun Sethuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, California State University San Marcos, CA 92096</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arun%2BSethuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sethuraman%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArun%2BSethuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8201-8292" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arun Sethuraman</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:asethuraman@csusm.edu" class="" data-icon-position="" data-hide-link-title="0">asethuraman@csusm.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3203&amp;rft.epage=3220&amp;rft.atitle=Estimating%20Genetic%20Relatedness%20in%20Admixed%20Populations&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sethuraman%2C+Arun"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490975/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490975/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490975/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490975/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490975/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490975/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490975/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490975/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490975/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490975/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490975/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490975/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3203" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3203.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3203.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/10/3203.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
