<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3255" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3255.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3255.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3255.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3255.atom" />
<meta name="HW.pisa" content="ggg;8/10/3255" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies" />
<meta name="DC.Identifier" content="10.1534/g3.118.200502" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Wojcik et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The emergence of very large cohorts in genomic research has facilitated a focus on genotype-imputation strategies to power rare variant association. These strategies have benefited from improvements in imputation methods and association tests, however little attention has been paid to ways in which array design can increase rare variant association power. Therefore, we developed a novel framework to select tag SNPs using the reference panel of 26 populations from Phase 3 of the 1000 Genomes Project. We evaluate tag SNP performance via mean imputed r2 at untyped sites using leave-one-out internal validation and standard imputation methods, rather than pairwise linkage disequilibrium. Moving beyond pairwise metrics allows us to account for haplotype diversity across the genome for improve imputation accuracy and demonstrates population-specific biases from pairwise estimates. We also examine array design strategies that contrast multi-ethnic cohorts vs. single populations, and show a boost in performance for the former can be obtained by prioritizing tag SNPs that contribute information across multiple populations simultaneously. Using our framework, we demonstrate increased imputation accuracy for rare variants (frequency &lt; 1%) by 0.5–3.1% for an array of one million sites and 0.7–7.1% for an array of 500,000 sites, depending on the population. Finally, we show how recent explosive growth in non-African populations means tag SNPs capture on average 30% fewer other variants than in African populations. The unified framework presented here will enable investigators to make informed decisions for the design of new arrays, and help empower the next phase of rare variant association for global health." />
<meta name="DC.Contributor" content="Genevieve L. Wojcik" />
<meta name="DC.Contributor" content="Christian Fuchsberger" />
<meta name="DC.Contributor" content="Daniel Taliun" />
<meta name="DC.Contributor" content="Ryan Welch" />
<meta name="DC.Contributor" content="Alicia R Martin" />
<meta name="DC.Contributor" content="Suyash Shringarpure" />
<meta name="DC.Contributor" content="Christopher S. Carlson" />
<meta name="DC.Contributor" content="Goncalo Abecasis" />
<meta name="DC.Contributor" content="Hyun Min Kang" />
<meta name="DC.Contributor" content="Michael Boehnke" />
<meta name="DC.Contributor" content="Carlos D. Bustamante" />
<meta name="DC.Contributor" content="Christopher R. Gignoux" />
<meta name="DC.Contributor" content="Eimear E. Kenny" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The emergence of very large cohorts in genomic research has facilitated a focus on genotype-imputation strategies to power rare variant association. These strategies have benefited from improvements in imputation methods and association tests, however little attention has been paid to ways in which array design can increase rare variant association power. Therefore, we developed a novel framework to select tag SNPs using the reference panel of 26 populations from Phase 3 of the 1000 Genomes Project. We evaluate tag SNP performance &lt;i&gt;via&lt;/i&gt; mean imputed r&lt;sup&gt;2&lt;/sup&gt; at untyped sites using leave-one-out internal validation and standard imputation methods, rather than pairwise linkage disequilibrium. Moving beyond pairwise metrics allows us to account for haplotype diversity across the genome for improve imputation accuracy and demonstrates population-specific biases from pairwise estimates. We also examine array design strategies that contrast multi-ethnic cohorts &lt;i&gt;vs.&lt;/i&gt; single populations, and show a boost in performance for the former can be obtained by prioritizing tag SNPs that contribute information across multiple populations simultaneously. Using our framework, we demonstrate increased imputation accuracy for rare variants (frequency &amp;lt; 1%) by 0.5–3.1% for an array of one million sites and 0.7–7.1% for an array of 500,000 sites, depending on the population. Finally, we show how recent explosive growth in non-African populations means tag SNPs capture on average 30% fewer other variants than in African populations. The unified framework presented here will enable investigators to make informed decisions for the design of new arrays, and help empower the next phase of rare variant association for global health.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3255" />
<meta name="citation_id" content="8/10/3255" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3255" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3255.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3255.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3255.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200502" />
<meta name="citation_pmid" content="30131328" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3255" />
<meta name="citation_lastpage" content="3267" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Genevieve L. Wojcik" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7206-8088" />
<meta name="citation_author" content="Christian Fuchsberger" />
<meta name="citation_author_institution" content="Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109" />
<meta name="citation_author_institution" content="Center for Biomedicine, European Academy of Bolzano/Bozen (EURAC), affiliated with the University of Lübeck, Bolzano, Bozen, 39100, Italy" />
<meta name="citation_author" content="Daniel Taliun" />
<meta name="citation_author_institution" content="Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109" />
<meta name="citation_author" content="Ryan Welch" />
<meta name="citation_author_institution" content="Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109" />
<meta name="citation_author" content="Alicia R Martin" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0241-3522" />
<meta name="citation_author" content="Suyash Shringarpure" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305" />
<meta name="citation_author" content="Christopher S. Carlson" />
<meta name="citation_author_institution" content="Fred Hutchinson Cancer Center, University of Washington, 1100 Fairview Ave. N., Seattle, WA 98109" />
<meta name="citation_author" content="Goncalo Abecasis" />
<meta name="citation_author_institution" content="Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109" />
<meta name="citation_author" content="Hyun Min Kang" />
<meta name="citation_author_institution" content="Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109" />
<meta name="citation_author" content="Michael Boehnke" />
<meta name="citation_author_institution" content="Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109" />
<meta name="citation_author" content="Carlos D. Bustamante" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305" />
<meta name="citation_author_institution" content="Department of Biomedical Data Science, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305" />
<meta name="citation_author" content="Christopher R. Gignoux" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305" />
<meta name="citation_author_email" content="chris.gignoux@ucdenver.edu" />
<meta name="citation_author_email" content="eimear.kenny@mssm.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9728-6567" />
<meta name="citation_author" content="Eimear E. Kenny" />
<meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029" />
<meta name="citation_author_institution" content="The Charles Bronfman Institute for Personalized Medicine, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029" />
<meta name="citation_author_institution" content="The Icahn Institute of Multiscale Biology and Genomics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029" />
<meta name="citation_author_institution" content="The Center for Statistical Genetics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029" />
<meta name="citation_author_email" content="chris.gignoux@ucdenver.edu" />
<meta name="citation_author_email" content="eimear.kenny@mssm.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9198-759X" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Auton. A.;citation_author=L. D. Brooks;citation_author=R. M. Durbin;citation_author=E. P. Garrison;citation_title=A global reference for human genetic variation.;citation_pages=68-74;citation_volume=526;citation_year=2015;citation_pmid=26432245;citation_doi=10.1038/nature15393" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Banda;citation_author=M. N. Kvale;citation_author=T. J. Hoffmann;citation_author=S. E. Hesselson;citation_author=D. Ranatunga;citation_title=Characterizing Race/Ethnicity and Genetic Ancestry for 100,000 Subjects in the Genetic Epidemiology Research on Adult Health and Aging (GERA) Cohort.;citation_volume=200;citation_year=2015;citation_pmid=26092716;citation_doi=10.1534/genetics.115.178616" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. C. Barrett;citation_author=L. R. Cardon;citation_title=Evaluating coverage of genome-wide association studies.;citation_pages=659-662;citation_volume=38;citation_year=2006;citation_issue=6;citation_pmid=16715099;citation_doi=10.1038/ng1801" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=T. R. Bhangale;citation_author=M. J. Rieder;citation_author=D. A. Nickerson;citation_title=Estimating coverage and power for genetic association studies using near-complete variation data.;citation_pages=841-843;citation_volume=40;citation_year=2008;citation_issue=7;citation_pmid=18568023;citation_doi=10.1038/ng.180" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Genotype Imputation with Millions of Reference Samples.;citation_pages=116-126;citation_volume=98;citation_year=2016;citation_pmid=26748515;citation_doi=10.1016/j.ajhg.2015.11.020" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. D. Bustamante;citation_author=E. G. Burchard;citation_author=F. M. De La Vega;citation_title=Genomics for the world.;citation_pages=163-165;citation_volume=475;citation_year=2011;citation_issue=7355;citation_pmid=21753830;citation_doi=10.1038/475163a" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. S. Carlson;citation_author=M. A. Eberle;citation_author=M. J. Rieder;citation_author=Q. Yi;citation_author=L. Kruglyak;citation_title=Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.;citation_pages=106-120;citation_volume=74;citation_year=2004;citation_issue=1;citation_pmid=14681826;citation_doi=10.1086/381000" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=C. S. Carlson;citation_author=T. C. Matise;citation_author=K. E. North;citation_author=C. A. Haiman;citation_author=M. D. Fesinmeyer;citation_title=Generalization and dilution of association results from European GWAS in populations of non-European ancestry: the PAGE study.;citation_pages=e1001661;citation_volume=11;citation_year=2013;citation_pmid=24068893;citation_doi=10.1371/journal.pbio.1001661" />
<meta name="citation_reference" content="citation_journal_title=GigaSci;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7-16;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. I. W. de Bakker;citation_author=R. Yelensky;citation_author=I. Pe’er;citation_author=S. B. Gabriel;citation_author=M. J. Daly;citation_title=Efficiency and power in genetic association studies.;citation_pages=1217-1223;citation_volume=37;citation_year=2005;citation_issue=11;citation_pmid=16244653;citation_doi=10.1038/ng1669" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. J. Emond;citation_author=T. Louie;citation_author=J. Emerson;citation_author=W. Zhao;citation_author=R. A. Mathias;citation_title=Exome sequencing of extreme phenotypes identifies DCTN4 as a modifier of chronic Pseudomonas aeruginosa infection in cystic fibrosis.;citation_pages=886-889;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22772370;citation_doi=10.1038/ng.2344" />
<meta name="citation_reference" content="citation_author=C. Fuchsberger;citation_author=G. R. Abecasis;citation_author=D. A. Hinds;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. Fuchsberger;citation_author=J. Flannick;citation_author=T. M. Teslovich;citation_author=A. Mahajan;citation_author=V. Agarwala;citation_title=The genetic architecture of type 2 diabetes.;citation_pages=41-47;citation_volume=536;citation_year=2016;citation_pmid=27398621;citation_doi=10.1038/nature18642" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gravel;citation_author=B. M. Henn;citation_author=R. N. Gutenkunst;citation_author=A. R. Indap;citation_author=G. T. Marth;citation_title=Demographic history and rare allele sharing among human populations.;citation_volume=108;citation_year=2011;citation_pmid=21730125;citation_doi=10.1073/pnas.1019276108" />
<meta name="citation_reference" content="citation_title=PLoS Genet.;citation_author=S. Gravel;citation_author=F. Zakharia;citation_author=A. Moreno-Estrada;citation_author=J. K. Byrnes;citation_author=M. Muzzio;citation_title=Reconstructing Native American Migrations from Whole-Genome and Whole-Exome Data.;citation_pages=e1004023;citation_volume=9;citation_year=2013;citation_pmid=24385924;citation_doi=10.1371/journal.pgen.1004023" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. Gurdasani;citation_author=T. Carstensen;citation_author=F. Tekola-Ayele;citation_author=L. Pagani;citation_author=I. Tachmazidou;citation_title=The African Genome Variation Project shapes medical genetics in Africa.;citation_pages=327-332;citation_volume=517;citation_year=2015;citation_pmid=25470054;citation_doi=10.1038/nature13997" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=B. M. Henn;citation_author=L. R. Botigué;citation_author=C. D. Bustamante;citation_author=A. G. Clark;citation_author=S. Gravel;citation_title=Estimating the mutation load in human genomes.;citation_pages=333-343;citation_volume=16;citation_year=2015;citation_pmid=25963372;citation_doi=10.1038/nrg3931" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Henn;citation_author=C. R. Gignoux;citation_author=M. Jobin;citation_author=J. M. Granka;citation_author=J. M. Macpherson;citation_title=Hunter-gatherer genomic diversity suggests a southern African origin for modern humans.;citation_volume=108;citation_year=2011;citation_pmid=21383195;citation_doi=10.1073/pnas.1017511108" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=T. J. Hoffmann;citation_author=M. N. Kvale;citation_author=S. E. Hesselson;citation_author=Y. Zhan;citation_author=C. Aquino;citation_title=Next generation genome-wide association tool: design and coverage of a high-throughput European-optimized SNP array.;citation_pages=79-89;citation_volume=98;citation_year=2011;citation_issue=2;citation_pmid=21565264;citation_doi=10.1016/j.ygeno.2011.04.005" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=T. J. Hoffmann;citation_author=Y. Zhan;citation_author=M. N. Kvale;citation_author=S. E. Hesselson;citation_author=J. Gollub;citation_title=Design and coverage of high throughput genotyping arrays optimized for individuals of East Asian, African American, and Latino race/ethnicity using imputation and a novel hybrid SNP selection algorithm.;citation_pages=422-430;citation_volume=98;citation_year=2011;citation_issue=6;citation_pmid=21903159;citation_doi=10.1016/j.ygeno.2011.08.007" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Huang;citation_author=B. Howie;citation_author=S. McCarthy;citation_author=Y. Memari;citation_author=K. Walter;citation_title=Improved imputation of low-frequency and rare variants using the UK10K haplotype reference panel.;citation_pages=8111;citation_volume=6;citation_year=2015;citation_pmid=26368830;citation_doi=10.1038/ncomms9111" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=C. Igartua;citation_author=R. A. Myers;citation_author=R. A. Mathias;citation_author=M. Pino-Yanes;citation_author=C. Eng;citation_title=Ethnic-specific associations of rare and low-frequency DNA sequence variants with asthma.;citation_pages=5965;citation_volume=6;citation_year=2015;citation_pmid=25591454;citation_doi=10.1038/ncomms6965" />
<meta name="citation_reference" content="Illumina Infinium OmniExpress-24 v1.2 BeadChip Information Sheet." />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The International HapMap Project.;citation_pages=789-796;citation_volume=426;citation_year=2003;citation_issue=6968;citation_pmid=14685227;citation_doi=10.1038/nature02168" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Kircher;citation_author=D. M. Witten;citation_author=P. Jain;citation_author=B. J. O’Roak;citation_author=G. M. Cooper;citation_title=A general framework for estimating the relative pathogenicity of human genetic variants.;citation_pages=310-315;citation_volume=46;citation_year=2014;citation_pmid=24487276;citation_doi=10.1038/ng.2892" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genet.;citation_author=J. A. Kosmicki;citation_author=C. L. Churchhouse;citation_author=M. A. Rivas;citation_author=B. M. Neale;citation_title=Discovery of rare variants for complex phenotypes.;citation_pages=625-634;citation_volume=135;citation_year=2016;citation_doi=10.1007/s00439-016-1679-1" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=M. Li;citation_author=C. Li;citation_author=W. Guan;citation_title=Evaluation of coverage variation of SNP chips for genome-wide association studies.;citation_pages=635-643;citation_volume=16;citation_year=2008;citation_issue=5;citation_pmid=18253166;citation_doi=10.1038/sj.ejhg.5202007" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=K. J. Lindquist;citation_author=E. Jorgenson;citation_author=T. J. Hoffmann;citation_author=J. S. Witte;citation_title=The impact of improved microarray coverage and larger sample sizes on future genome-wide association studies.;citation_pages=383-392;citation_volume=37;citation_year=2013;citation_pmid=23529720;citation_doi=10.1002/gepi.21724" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=K. E. Lohmueller;citation_author=T. Sparsø;citation_author=Q. Li;citation_author=E. Andersson;citation_author=T. Korneliussen;citation_title=Whole-exome sequencing of 2,000 Danish individuals and the role of rare coding variants in type 2 diabetes.;citation_pages=1072-1086;citation_volume=93;citation_year=2013;citation_pmid=24290377;citation_doi=10.1016/j.ajhg.2013.11.005" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. K. Maples;citation_author=S. Gravel;citation_author=E. E. Kenny;citation_author=C. D. Bustamante;citation_title=RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference.;citation_pages=278-288;citation_volume=93;citation_year=2013;citation_pmid=23910464;citation_doi=10.1016/j.ajhg.2013.06.020" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_author=S. Myers;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A new multipoint method for genome-wide association studies by imputation of genotypes.;citation_pages=906-913;citation_volume=39;citation_year=2007;citation_issue=7;citation_pmid=17572673;citation_doi=10.1038/ng2088" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=E. Marouli;citation_author=M. Graff;citation_author=C. Medina-Gomez;citation_author=K. S. Lo;citation_author=A. R. Wood;citation_title=Rare and low-frequency coding variants alter human adult height.;citation_pages=186-190;citation_volume=542;citation_year=2017;citation_pmid=28146470;citation_doi=10.1038/nature21039" />
<meta name="citation_reference" content="citation_journal_title=Pac. Symp. Biocomput.;citation_author=A. R. Martin;citation_author=G. Tse;citation_author=C. D. Bustamante;citation_author=E. E. Kenny;citation_title=Imputation-based assessment of next generation rare exome variant arrays.;citation_pages=241-252;citation_volume=3;citation_year=2014" />
<meta name="citation_reference" content="citation_title=PLoS Genet.;citation_author=I. Mathieson;citation_author=G. McVean;citation_title=Demography and the Age of Rare Variants.;citation_pages=e1004528;citation_volume=10;citation_year=2014;citation_pmid=25101869;citation_doi=10.1371/journal.pgen.1004528" />
<meta name="citation_reference" content="citation_journal_title=Schizophr. Res.;citation_author=N. S. McCarthy;citation_author=P. E. Melton;citation_author=S. V. Ward;citation_author=S. M. Allan;citation_author=M. Dragovic;citation_title=Exome array analysis suggests an increased variant burden in families with schizophrenia.;citation_pages=9-16;citation_volume=185;citation_year=2017;citation_doi=10.1016/j.schres.2016.12.007" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. McCarthy;citation_author=S. Das;citation_author=W. Kretzschmar;citation_author=O. Delaneau;citation_author=A. R. Wood;citation_title=A reference panel of 64,976 haplotypes for genotype imputation.;citation_pages=1279-1283;citation_volume=48;citation_year=2016;citation_pmid=27548312;citation_doi=10.1038/ng.3643" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=K. Michailidou;citation_author=J. Beesley;citation_author=S. Lindstrom;citation_author=S. Canisius;citation_author=J. Dennis;citation_title=Genome-wide association analysis of more than 120,000 individuals identifies 15 new susceptibility loci for breast cancer.;citation_pages=373-380;citation_volume=47;citation_year=2015;citation_pmid=25751625;citation_doi=10.1038/ng.3242" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Moreno-Estrada;citation_author=C. R. Gignoux;citation_author=J. C. Fernández-López;citation_author=F. Zakharia;citation_author=M. Sikora;citation_title=The genetics of Mexico recapitulates Native American substructure and affects biomedical traits;citation_pages=1280-1285;citation_volume=344;citation_year=2014;citation_issue=6189;citation_pmid=24926019;citation_doi=10.1126/science.1251688" />
<meta name="citation_reference" content="citation_title=PLoS Genet.;citation_author=A. Moreno-Estrada;citation_author=S. Gravel;citation_author=F. Zakharia;citation_author=J. L. McCauley;citation_author=J. K. Byrnes;citation_title=Reconstructing the Population Genetic History of the Caribbean.;citation_pages=e1003925;citation_volume=9;citation_year=2013;citation_pmid=24244192;citation_doi=10.1371/journal.pgen.1003925" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. R. Nelson;citation_author=K. Bryc;citation_author=K. S. King;citation_author=A. Indap;citation_author=A. R. Boyko;citation_title=The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research.;citation_pages=347-358;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18760391;citation_doi=10.1016/j.ajhg.2008.08.005" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. R. Nelson;citation_author=D. Wegmann;citation_author=M. G. Ehm;citation_author=D. Kessner;citation_author=P. St Jean;citation_title=An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.;citation_volume=337;citation_year=2012;citation_pmid=22604722;citation_doi=10.1126/science.1217876" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. C. Nelson;citation_author=K. F. Doheny;citation_author=E. W. Pugh;citation_author=J. M. Romm;citation_author=H. Ling;citation_title=Imputation-based genomic coverage assessments of current human genotyping arrays.;citation_volume=3;citation_year=2013;citation_pmid=23979933;citation_doi=10.1534/g3.113.007161" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=I. Pe’er;citation_author=P. I. W. de Bakker;citation_author=J. Maller;citation_author=R. Yelensky;citation_author=D. Altshuler;citation_title=Evaluating and improving power in whole-genome association studies using fixed marker sets.;citation_pages=663-667;citation_volume=38;citation_year=2006;citation_issue=6;citation_pmid=16715096;citation_doi=10.1038/ng1816" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_author=M. Przeworski;citation_title=Linkage disequilibrium in humans: models and data.;citation_pages=1-14;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11410837;citation_doi=10.1086/321275" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Prüfer;citation_author=F. Racimo;citation_author=N. Patterson;citation_author=F. Jay;citation_author=S. Sankararaman;citation_title=The complete genome sequence of a Neanderthal from the Altai Mountains;citation_pages=43-49;citation_volume=505;citation_year=2014;citation_issue=7481;citation_pmid=24352235;citation_doi=10.1038/nature12886;citation_georef=2014009693" />
<meta name="citation_reference" content="citation_journal_title=JAMA;citation_author=K. Estrada;citation_author=I. Aukrust;citation_author=L. Bjørkhaug;citation_author=N. P. Burtt;citation_title=Association of a low-frequency variant in HNF1A with type 2 diabetes in a Latino population.;citation_pages=2305-2314;citation_volume=311;citation_year=2014;citation_pmid=24915262;citation_doi=10.1001/jama.2014.6511" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. C. A. Spencer;citation_author=Z. Su;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=Designing genome-wide association studies: sample size, power, imputation, and the choice of genotyping chip.;citation_pages=e1000477-e1000477;citation_volume=5;citation_year=2009;citation_issue=5;citation_pmid=19492015;citation_doi=10.1371/journal.pgen.1000477" />
<meta name="citation_reference" content="citation_journal_title=PLoS Med.;citation_author=C. Sudlow;citation_author=J. Gallacher;citation_author=N. Allen;citation_author=V. Beral;citation_author=P. Burton;citation_title=UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age.;citation_pages=e1001779;citation_volume=12;citation_year=2015;citation_pmid=25826379;citation_doi=10.1371/journal.pmed.1001779" />
<meta name="citation_reference" content="citation_title=Nature;citation_author=K. Walter;citation_author=J. L. Min;citation_author=J. Huang;citation_author=L. Crooks;citation_title=The UK10K project identifies rare variants in health and disease.;citation_pages=82-90;citation_volume=526;citation_year=2015;citation_pmid=26367797;citation_doi=10.1038/nature14962" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. E. Weale;citation_author=C. Depondt;citation_author=S. J. Macdonald;citation_author=A. Smith;citation_author=P. S. Lai;citation_title=Selection and evaluation of tagging SNPs in the neuronal-sodium-channel gene SCN1A: implications for linkage-disequilibrium gene mapping.;citation_pages=551-565;citation_volume=73;citation_year=2003;citation_issue=3;citation_pmid=12900796;citation_doi=10.1086/378098" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Wessel;citation_author=A. Y. Chu;citation_author=S. M. Willems;citation_author=S. Wang;citation_author=H. Yaghootkar;citation_title=Low-frequency and rare exome chip variants associate with fasting glucose and type 2 diabetes susceptibility.;citation_pages=5897;citation_volume=6;citation_year=2015;citation_pmid=25631608;citation_doi=10.1038/ncomms6897" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="The emergence of very large cohorts in genomic research has facilitated a focus on genotype-imputation strategies to power rare variant association. These strategies have benefited from improvements in imputation methods and association tests, however little attention has been paid to ways in which array design can increase rare variant association power. Therefore, we developed a novel framework to select tag SNPs using the reference panel of 26 populations from Phase 3 of the 1000 Genomes Project. We evaluate tag SNP performance via mean imputed r2 at untyped sites using leave-one-out internal validation and standard imputation methods, rather than pairwise linkage disequilibrium. Moving beyond pairwise metrics allows us to account for haplotype diversity across the genome for improve imputation accuracy and demonstrates population-specific biases from pairwise estimates. We also examine array design strategies that contrast multi-ethnic cohorts vs. single populations, and show a boost in performance for the former can be obtained by prioritizing tag SNPs that contribute information across multiple populations simultaneously. Using our framework, we demonstrate increased imputation accuracy for rare variants (frequency &lt; 1%) by 0.5–3.1% for an array of one million sites and 0.7–7.1% for an array of 500,000 sites, depending on the population. Finally, we show how recent explosive growth in non-African populations means tag SNPs capture on average 30% fewer other variants than in African populations. The unified framework presented here will enable investigators to make informed decisions for the design of new arrays, and help empower the next phase of rare variant association for global health." />
<meta name="og-title" property="og:title" content="Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3255" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The emergence of very large cohorts in genomic research has facilitated a focus on genotype-imputation strategies to power rare variant association. These strategies have benefited from improvements in imputation methods and association tests, however little attention has been paid to ways in which array design can increase rare variant association power. Therefore, we developed a novel framework to select tag SNPs using the reference panel of 26 populations from Phase 3 of the 1000 Genomes Project. We evaluate tag SNP performance via mean imputed r2 at untyped sites using leave-one-out internal validation and standard imputation methods, rather than pairwise linkage disequilibrium. Moving beyond pairwise metrics allows us to account for haplotype diversity across the genome for improve imputation accuracy and demonstrates population-specific biases from pairwise estimates. We also examine array design strategies that contrast multi-ethnic cohorts vs. single populations, and show a boost in performance for the former can be obtained by prioritizing tag SNPs that contribute information across multiple populations simultaneously. Using our framework, we demonstrate increased imputation accuracy for rare variants (frequency &lt; 1%) by 0.5–3.1% for an array of one million sites and 0.7–7.1% for an array of 500,000 sites, depending on the population. Finally, we show how recent explosive growth in non-African populations means tag SNPs capture on average 30% fewer other variants than in African populations. The unified framework presented here will enable investigators to make informed decisions for the design of new arrays, and help empower the next phase of rare variant association for global health." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uRqiP6TEoSZSV9BDZ_pCB16-BUcm1-jbeqBEGt-2aa8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490960","apath":"\/ggg\/8\/10\/3255.atom","pisa":"ggg;8\/10\/3255","ac":{"\/ggg\/8\/10\/3255.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3255.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3255"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3255"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3255.supplemental":true,"\/content\/8\/10\/3255":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hGoDMxPKJmTIH+WEQqYi5OzyLjW99THsQKd+2aSuD9ii7CkYgQtHgrXtYHq9V1045NH5fzvBXT4cwFRa91muQNiKaCUjBkKWuVO6mylnBcn+bnaYdyLsO+jOEkvdBCorEVO32Bkn7+STmoJA7LYtuQaW\\\\\\\/GtrGbonUw5vojjk8quneGPeKxsgRNvDjTOExRodVWfHggvJAffnr1KOds8YcfoU3jBSDjvqUn2g8kjUQ\\\\\\\/L8T3bvDCuZBu13Zp5XeZ6wlNvGKRMAIdVnuDGtLXjqcOAssbQPV52P\\\\\\\/xa4KrbZfoLSSru\\\\\\\/cal6hzRS7lvJF1te5pRVFEvXkj8GRyfn7kWnsXTX+fsP8YnLof2SCXgI1IJt1KDQ0JmE7fMeB7Kkq9cPw425Pxa8ce92y9DTyyx0AEiuPu6YJnjy0tTZ1Q3FTgtEs7wNp4\\\\\\\/Qc3g+DKQav1\\\\\\\/+Dgfs3GlV6TI8x8ZQ2mDhgrk1L37tbK64GVs\\\\\\\/FxZx\\\\\\\/Gls\\\\\\\/FN7m8dBxlq7qUOuLAy1bVdKPKS4UTDcNa1jUxy1tKCiY8xXz1zxOOWuyaeDQTFej3Tvbi3pBApLnYdugZT38ywr4T2EbDcRmWpx\\\\\\\/Ye+OQRZhxvqQE3DqbZbS8pAmY68NHYA7GJ3fgnz28MgmvjJBcoEwQj4vl\\\\\\\/7nKmBENKo3umJ7XtDSIE3Mzz8+gpFghKiQrc2JIO+F8Au2yxRw3GIjmzzpOCUWjSMVmvlHQ5YwgG7gIqUpJRBVU8hmX\\\\\\\/MDn+pPdusal2C528Xge\\\\\\\/2o0v+VC6HH6eeMTOllule+A3p2al+K5ZbW\\\\\\\/f6Kp8Ua+OqLovku2TLcDs04UVZhj4sWWrPlKyhvg4YTVpSfdo0MykmlDg3DBo+63Ebp6YYJBu2hF6JHBYMoaK9u3f+1w3q9brHN+HUko7dmrMd+ffnzQIHNikJrpisqJJwzaNhNO364zLFD6tOlIl5911K4nl1vvchnHwQg448iDa8kDDGcavCLBh5HrQMlYNJd1CBwfjsccmxnHJ0cpHbTAqUOCUU3tNUME8AtaZOcWda49E8G4cgrwoldcwjmIvR+pblphUON22SSZRIBDa57M3hUT9vRqKS\\\\\\\/fJ\\\\\\\/YJ4TAQpEaMVFjG7oeOapcTbPkJf57+3X9einb8j\\\\\\\/P2XqyfkWzAuO5TnNSV0jyqjH4iQBkv8hoN+mltCF5bs9c+hEK+0lKTFvS8jTnIKfUywBdx75cxvJqaCsAc007li4\\\\\\\/qTL3UkkGHeW\\\\\\\/v\\\\\\\/tMZYarvDhxiA4xZ5D4ubI1NLbDnt0vmTMwPkbXtt\\\\\\\/9Ux6\\\u0022,\\\u0022iv\\\u0022:\\\u00223ox1TTv3lGZKm0Vg6KlJeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f894f67f1d97aa9d305e861d3a9b0f585f620543ac4ff8bfc13fe37ea1debe64\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eoH3iryezGF1KTdfHnxseZ86TuTT47j4cjvjV3SGY5V92Du6EBYYYx1DyMh\\\\\\\/JpbyYLP5SFhuoeX2AUIguqmmUUOWIiFx+G9rPTJZOhlEeQJVx71Mlo6BprqfzqoPcLCDVVwmXjNioB22jWUL6wYfBO5idXQsXL6VOoDgx5E+mp2EMBDxnyZChavIDs1uaosr55dFnpWL3cMiDjr6U5rHqZmnnNVo1fJdIJ85OvnxzCdN3gIQ7QXLqEy8SNElcO84vQYeMgJXVS6sIHXO7BTRuhD4F5ItW+I3pCkjHTrXiISwNWVNHCOyx9qEgXoR7T5icY4I6QezBkhaSFeBXtZCW6qpbWXGQkJbWjs4WV6B6J5RuPUe60t0K5ToDJJhzLwLCBdDKrX17\\\\\\\/Ux1rUqVSG8HETpCxYePmAlwnYp4dDIHYqaqXIvN5zs1O6PYlfVbhSp\\\\\\\/WEes94gyX0v4Q9EXHtfgRbs9l16VRYvDlgvy6XIOnIDuUPRwaEI7EZ03C3Bte7IJkGhdTi5aZs7+I231VrSawJO3i6My3ILFd+LZl44P2kuVvBq7LS1tth406CiEHR7W9YtTpZmok+Hvw60zDu0Oz0uAKx7qgVpswYQSEh+vc9rGRmAR3Pr3IXL\\\\\\\/G0ds1KV9g7lJPV1TV5wZ95YinOidDykWYlHZ3Zw3S5KtxCNdLMU6E6tW2ImDfsjGnvsrdgVNWthv5BHgZanqAkFLg7dQsFykww+\\\\\\\/UiAYeY8iJiMGKMBqu6FIT+4Mesv+TnTVWiXriWnbBRS5kmnErVZrYFWWhyydNauLJqt2owlPi72wwdbuuomH5njz54L4uLVj2XfSb8AXxhSLHYCNqVDhM8PvuQGskAuBakOkVMPGtGx9953jDjTjMinuERNoJLEh8O0afhDdxv7xNW+LrPLhY938mkFBVxeHKbdfFS9e3WCtdoMyM22GJTDnVlwVRMu0r2rMcTrvWLgjyaqR6IjfX5ysHCN7ffLRVgTsTO8iSsGewrd4+MGIH8CorFdFJwIi3K1QSnHBgeO0H0t2xmrdc4NOKd2kxYQXPVNZ+Rpj66h2xHpQwA6b2VOOh+fCLx+p+x8pjbKmlX4qlBru38sqqa1Yesb79EiTv5NM+tsdkiW+OulUwqKl7IhZdAvF04qYAnUyuO+JXJpuwVc5XiST0Zxc45RaUIlLFAmi843gLhVsZqc9Eya3L0kdrxuPIqZBcIlpsUPBC4cn6tYgzgYrcmNmrk\\\\\\\/UuYdCmc+P7Cwc7D70Syornpa5LN5YszfWvwltFmtZH76xRBKDBtB2pHsowNAE35TdXtCT+cALj6+7AGa6CGiG53xsrAGzSRFzdZr83mVUOowEaZNrnzNpTDjhM6CvV2x954ntW+78hyLbDnawaghkKXVHodcokFmzj2EdHDf06T7ddorPppH4u44cLI3baqHnXvqfn\\\\\\\/ceEP5q59mC9+K0Y8LuN7M8gMIA1Kleqi5AjWv\\\\\\\/iKqVIEcSikXlzHfwhntp7wPmWkAeuiEwa7wahOW1aeEa8lygMx0NNVEhCa8pBvDdYUdDR4vO20qqvwYrIXTYs6LAalxBWITJ1K2dLNPMKrSIPxkFlLjkRWCKXEmqrvxBdS8FBpHHQK93I1PbUkxW\\\\\\\/+KkrYFhiziKXOv2S8d\\\\\\\/hrpcp4vwAeh7qVNWFC14\\\\\\\/DUlO8PJVOmOKo\\\\\\\/uxZOTi0wscTgyd4hjudL\\\\\\\/OAItUWGfSDUFhbwjykdJCnUcddDk9naMJe2nggJ3MjTpveGgWFdzTBoeXdWQZUZVhiUWARg\\\\\\\/AwbkyOCUzV7XYi4+F5aTZb6PGKfhNsE8bZLhF4X7KV7vDnAt8mPM4XmzOJC7I6\\\\\\\/\\\\\\\/G\\\\\\\/ia0irhQBV3bw0ISJU3nF11HkT8U9pjNzCRbhlRDCuJ0mCsXgqodGVPV574NZr25kbLO5AbvLQ1d6t\\\\\\\/SqEscCsy4zAWKD\\\\\\\/Q+EDl52T0Zo4X824YgmYQvTURhOhFZvTBVYcPQIS7veYNvrmGHdL+EUo7WPxLhEGObYvrFPKpJR+YSBoUZsGATVIwLy1cAdpMw\\\\\\\/ad7WrMH2XjJ3Ux\\\\\\\/StF\\\\\\\/43iHJV7y7JvfGzIczTFo+p0qC6bTBFaK2Amj1rdpMzrQmPipN5Zt8ThleM5gxoG6Zzw1hNymvltC7y19t6NOCmUZv05FSG2PsyQfQKr0pebS9R4yKS3\\\\\\\/3mVYVWASmIOcpRkxeBOWy4GDybk\\\\\\\/OZQf6cqIdXdiHtZkRIfXdB5ZM8hpQHwyH0JCns2H3mwEusK3ZIaWZxtHfIixBZj1bWJRwOBDDzoktuz8HAqw9n8VHZ+\\\\\\\/Ra4w0qdi6wf3fHbEC4\\\\\\\/LOsMghpKcjKbSbtY0EODSfRwFIi60QRT3N\\\\\\\/2B5AkflnGeymoaS1TRs8vvB1JE8MryeJQZRvYpdcRU8\\\\\\\/XDg6T8qL6BakLqBLhWOJNdU9osTduTSZIJ5\\\\\\\/cVeA1MsGfzjqJdT20RmsE9rW2+hgzoo1nl+XXiiE7eCjJ46O9eEpSf9K7c4D2OLwaiFJpQBy19L6tsVPFfak9nl9Hqo2TFrbrkQitl8ihIxp5GqF3TXAZh3ReL\\\\\\\/QuhoALf75JLVtuAixBgPEFfPWsPmocf\\\\\\\/3qDXi6VQchHm5v2Qum9rU5TYm02JSeowmcGPGWC5e7LBe3cbB08D2AYRTK6WRf1VOxsXMsL7gS3D0CyKOqNAzOfj10cRDIwKEOheFR0kUwxtJXEufKykVkzGQyFrj2b\\\\\\\/1FALWGAowrLqPOyb+ZGy1O0Whn2hyjByqEQJGOq6m9M2\\\\\\\/zBkiWAITgAIg3hSXiMUEMWUu2G0FmOXpKdD73quzKxNOf+gF19x47IxpqoixDf0dFaYpKsSi21lBg+UK3+2M81wK\\\\\\\/SFEIXYe09pqjWJiOOV1szw\\\\\\\/lpUnD82+QWk5zr\\\\\\\/bvebfYO6Yj496kEZWDuEPveoLgNH0Y2cXGtYTb+O3n7oJTct+FWCbcIGDxxT1Jj8HwnIQvkgb2BSoz3IUQBDEOJA\\\\\\\/8YN+QGGyV5oDgKJuymVb7HBfW7j+cd6tDmeW+PAssbFpNPRpUR\\\\\\\/ILBHZoaE8IDwDoKqOP+KtXyhoo+iy\\\\\\\/xgegwg+HFA4DUBHavUdKj\\\\\\\/UF2GaOc5jM52N7Yww9PHONxoviHZdR7Zj077fEPzJYopiXKBudK9azBOded5kHMKpbkJqKHrsWcNAObXgtUhnpSPzDGzA9CCKilQT6Q90V7Q2oW\\\\\\\/cQyJfcSNCHvlmV5yzCTv+wZByjcMJjm4zZzQo9Obh52jNHxYT8bDn+FkKIqFuAAImxA2J63jjue5LcLgbq9UZ3R5YXTiVbWazCROLQbgvZEcTDJ78KLqnAWwtSxAwZ9qa6jCVxx9exepRv11F8udCOkcSrcxfCxVXJ+9EovosMy4hrPLu+lINiXTy0\\\\\\\/RKSwCEsaF1edyqjB\\\\\\\/dEreqTv7nnRwSe3eFRZ2MOyPFriqZOSs0p0pPZeaPTxknCZXTaJOCjCL70qiJN8T9elwL2aG3fkK4stagANPCI6naB0cat7MkByt+eq5VV6B89s0K4glVb\\\\\\\/q9E28G7YHpNPRMbEoJEON3ahUM1NXgv9ZqY0LRXo\\\\\\\/ipYasdGYw+dYw0ApUR+iAw6vH8\\\\\\\/ye3isBfd2c54YpU3Ag6gK7uUzlWjqidHALT7QMXal7CrNnfH4vzqj9gsD6WoJHMoYbRqgNOIgvxtWvIKhNzP5dKjNsjUAtAzvsRpw0M9ZXHc3NmdftLtC88dhj4wV7sIopJyYHqeosXdd20q0kpLtAGm2NX8EsQbkyYoR2ZH8aE88BmWyM2gGe8YjM0pMaNfDyqyBuWYdaJp0fAFQBsJ1iRHWkZOP4cTS37NytEGgwcCCMyec\\\\\\\/4HNPsW0jjei6afZCsE2NuoN7rgci6FuFLXTaJSwypDSLjJCzVpEh5RL9eU46w+Wionl51bJFxTwLmEftZPm4bvqxWy9zVRWhksk6ENZ+jLJL\\\\\\\/NtBXRVoedT\\\\\\\/ZJSzSEyF1P4U7Qp04xiGCi\\\\\\\/Pa8XJhwLoR8Ej4TNw9PujpGX9\\\\\\\/Gvq3Jrc74XPm4yOzYKXTGgTb54xa70N7aaF3BqM\\\\\\\/IR9ozDLJAk4CCOnsINGb7bEO5dANzXmPZCB66CpxRW9cm7TkqF6kJXz66ZUhFHuFxns5JCR+3\\\\\\\/EhDtdVZWm2Xmil7uukl9UZ2WTrXjMoT+UD0QXA+4jDuw0s5sQ1RnxA19SbDv2uUkFTt\\\\\\\/AV4TKA6IxE1slrqcMYAQfWE\\\\\\\/EutBDTbB+WqkYV2sN9\\\\\\\/lkvSC5P51XEry2kqF2sz1WAXxDrJJUoXfZCDt5ZUxFHpOhGPV0OCv+CTtjabutIBUIHXQroq2pZ2lyc1xPJKRGTShZ1tASzZsyzpwvJ3tIi8ygDiuYVd\\\\\\\/XMz+T9Wws4q0jfy6wwKY5+2mG0gTg2i+D5eDqblvRtOymbsXFJm+JcEeQA8y0MkQcU3vewG\\\\\\\/4rIYxkz850esWHxuCklTKXgatk9w37fDZlRV\\\\\\\/hh3QUnknKbeqZBJFym74zUCf0QVc+9tGAy8HZOt5vqej3DzHSrTKPHYeaZcyGjOjqzDQFh5gd\\\\\\\/J7Ca61P0EjPF0wH0vTUnHCT\\\\\\\/Ek+RhqnGTEWJXX1BXtb001yD4kt3ANb\\\\\\\/S1j9ClrsANYMs1LAmB39dw+zmK\\\\\\\/gQqYkZXd74JYmiRaLPx23QHGPYLcnMDTyERmjPmy0+NF+2KYGGKQNkGtJ04rlzQKqWbnUkRJY8i7qLc28nJr2GjU02Mu42R8Ljbcsq\\\\\\\/4qAzQY6B6QjP5yAFd7QamvL6fXZALv728WACyndvAM7pq+qiSeKAA8kbbs1xPq+B+6tNIjx3AIESA0BosfTj60ff1ObIyFgB78Wts\\\\\\\/PBsab7NIotd6rW7HSukQNh6o89x3d5q4JsQHLl131ahru9cvVMy2wZqoNO55pe7OrM4l6ysKRZCRKFJMWYFtjFv5hZED+VRIEzQqF4AoMR6dFIFFImRyN6QfGw8zV1z3G6Kg\\\\\\\/OBHKFyn8USPqhwE7t3PmQ2XhMjKL0ZhU8+0xww24m8pRVP86Lk5COH8vmia+gxze6jTSRKn5lQo5y+0pDFFQD8\\\\\\\/TqiwP9Jl36ahwYCrNniWyJql\\\\\\\/tiwAlegYEtQdVPOxOJwZUoahbCuLW2eSkQW\\\\\\\/0mg3Wt6kD9px6ZSWe9dsCcY16P69MV9xCX9lXky3PlwixNjAu0mqDjPLB5GShGE\\\\\\\/xPM42hfOkFVM8lqINvH4OqfMxLc06tKfMY04IZMsc9iiVTBPLdbxz99qjvUsGF3RHJZ5vMES0mgnhtYdfFAqxfKfDCOavlBoj7NH56wIzNFUhWdqKRFkObMx5M62zRniJ+qciFoTIcTbHo1KQDAcBMlZOdk5FGakKRKOFO3lW0rg5f5KaxvyoLWDRgdJhCZDvLTsZRI2xFFinsN1+8oB9geJAIWAi2sL9lTZcqV6yOjH2aq7h8X9a1KiS3aczcvl4IM7jhVLte82WNpdy6Y\\\\\\\/LuG8sbxwSeIEEbvh6o5Wu6NueM2Y1WbbuoJUFjtEwIkelF+Ikyj4wNI2gHjmNGBBoHa3aYG3KKklnU7D4G9UdS8+htad0dNvReCQTGhLVQsKpqk+3jTw9sSkZ3wW1fru4BoefmGPRNo+0ruWwA5T7VXRcCV3ZH883Z9ECzZkBEQn+M4ny8Xqsvbep9ngpQfTYXZBFSfCobKS284WcGgW41SkXLA\\\\\\\/7X+7lOECBo9ZWAkkiJmkP5xsjH5PPgpoIVhPzZj4RQy8DAJmeI0q4tfYYa0U1fmU9ge9pN77HDE+ZSlQiuf9XTgSbMe3mPfquvZ1wG26+R5EaThj7WdjICVU5CJTqlKT3qKeqaaiEjguMX2uGsQccKYc2SEXvVTvTGNnvki8995ixe\\\\\\\/GjsLqjMdsnmISZ2ZcoHhg8khIfaaXX7D06wiALr+2JMhNCRpmAXTRfijqmEexAgpOmwAJnkXyJEuN6jIK1ifweFSnwtandpcv\\\\\\\/nyIBkHHP\\\\\\\/65962lao9HPxH3xFLm+y9I5qWSS7NN+CnGptB\\\\\\\/hlTzZc0S6gjLEknHpG51UZSknEe43b3e1MEebGtXF+skLEuaIABVH5JT7kjjnQbDvBPyucDXpG6Lt6sFHjSaT2VrZHqyORfJd0dWrLCA3043GU7\\\\\\\/uKHHbYPAPvM2EhorDTnROE+Gj\\\\\\\/H3cAyBAyCIdAOcibW9Db9yRooef7U\\\\\\\/0Dk9EJj6imQN7YSLqQYXkhmtcUFTwtEJpnc6kyieVXh45mycFWBsTCo2BFzgJldTQ5sHX4Yrc\\\\\\\/Te90aLCo9auSwR4dhb7tTEfNgqLbocdrJbpCAXKDBQLxoecVFAd6HjbQuJgf6HuQEh00euzhUHiL1B8lv0vqLj3107Il1sRrtP406S57TMT5ZYYuvAtpdJa0CbUdxED\\\\\\\/8LyMGFEsNWzDiYCqpQnlPEKphn4Y+83ZBDgKAxvxEugg++VyrJFkMxTMX\\\\\\\/mluR9ZpTt9ZbEbG17NK1j0H4FMr7Vl6EyzPPMSOevKiqJRDpUGOJLPA0Ys+qLaqv0wlA9enHy4KodnxpzrTTZoxLQTqkY2zQL1qw1s7QywDvD7xU+kR\\\\\\\/fHyIANaqOf\\\\\\\/9IQY\\\\\\\/Z9HXBTgcu6d3sPIWDu8uyuTp\\\\\\\/fJvnjD98Ztf0llmK1T5eG+wJE9M3gdlEdsAWHaY1jave5YQG1mWpZJI5MUjtOD8BnzTBKF1ntGmhLPd27D8nOpq3lp8AinYaiL3au6yNebSsokgqL7Jq4PooAGy022AvNSEgkDOIus21WtcueVqlaFdEWen92x3dOi8ogPeVOagV11FjH2h9KimsanESkb1XCFPfVv0dabLpegVXzChyFC8horXuRmgWeAOMOUUS3qAw917nbgvtAcBRchobF+8piKAIHom0Rp7b2KZT7TRbVps\\\\\\\/qAv1xTckE94JSHmob1oL\\\\\\\/KTQhmeY3RIP4KmuJ1TrSzrfmkftm6F5ecgGz4z7XavuMEk9mfRKg7IvyYc9Jf7p3hkS1TevLvbX5ThJTSzySXfziw1ZMmPtvlE1XN4Trh6wMfUOhfbzsyDF2KisNIVwO9O8\\\\\\\/e9F0PDHM45uuUFiIzBIGwUEPilkxcPOvDcsGWyG0WWKr4zC0Npt2D4pO\\\\\\\/wE8h8R0h5PJsgjlvuw\\\\\\\/W8glmS\\\\\\\/1NL2ZuBWnqUFd4sxXOAe2+3e8Y1pck3uplQ9e\\\\\\\/0qkmNzHg9FDc434\\\\\\\/cdQpa7aewgf93uJbsGWoaU43fhSyhJ1ae\\\\\\\/dMwTHBZrC9ecMDMUzu5Q69WP81dON0hijUH+3Af0vr8RAqdXH2mI2hUKtbgPzxDs+J+uM1jPpFHQX5TPCKYXEzAiYPSKpTcetfEJtVjD6K7u\\\\\\\/kD88wgltHw\\\\\\\/V0ZQwnsB7kIHBd6tR9ewWrJnkeNcNdIv5+iJhcABYMrTaahz2FEqAOAIwi2NenvGF3b9urfxd2yJku\\\\\\\/gROGDcTrzxBLTtj0oYZD7rjb7Lj6DtOrUM4tkKRbWI1hz3nMyO9XXcFhEp7zP5B8zR5BCkXMweNJ7mWB2ZYd4x7nFY1EuZW1zvxGNYvJYiPPkzZ9MYo32gXH6RvaiBVBejAOOIfC0jttzF+W6nyZN7+0I2bPCE+eGm+pgmaox4zsCcci59SjGScB+PkH0auVz1ArmtipEnV1R3ad0qp19DGvquRnSEHxJNOj15BvWQEXNHIzBDs6JNzviQLFFiZsQsCtHKpfc8H4BGlkV2aceQgkmtsnWwsAnrwahmkRpGm+aX+tQKCA8XU34BlSy85T50rKs+pLniLufg8a53xX1PxBzDyEXPTpwHHFQ6BKibqVyNlwZwigFGpwH5AGopGiCT22yRVBjvBro4DhxMyfjSRshVJ2Q1jWFBGDwLsUuXPbuabN7qzKiO3z5\\\\\\\/L9QnMIjJFbQGgOb997Sgzd\\\\\\\/HAK5xHui4Xk2jArQt8\\\\\\\/AUlPGxrujgl0TP2RYHS9A7+uBknH6UgYpVYdRdygafUg1dwP021jj5xhhy1yLcBlMQL6rA3DcKX60fk913mGopcfrhvFylDVSAiO7hNpLt3uk42igh2mjno9Cehl0XUlAne\\\\\\\/t0PJpPkD3FdHjeJIy11sf7dSu+T8LP5zI+LQ9SMpJ2HJ5oFCQ8Ft65mb2lUiWRjkObhIKA4tkJ+0JO+2c85PRftGEsfTcA9PpANjB\\\\\\\/FQ6b3HdCku8LkrUVOrukIgOBMhmie7gbgDdecSGA\\\\\\\/jW1LuwxVXOV4pSpYQYuRTxtCZ8TEOURx9E+AqvYZxzGh12iNGi1OWdHxGHZ0\\\\\\\/6zqthNbFS80bvCx0Qft\\\\\\\/np4uN+dCTQQ5XUFoMq1NtgO0TImwMyrKpFqxocnoZMrrdLIwOmLNlMtjm6Ec733U\\\\\\\/cgSatSuTUCiTbDHTe2JbJ8SJbldkNXnuBOd\\\\\\\/qPDC7e2\\\\\\\/OxbwTPFZrOgexAU3E\\\\\\\/BZ1Txy6sttzeb\\\\\\\/kRbMnElS7FdEkWJMu1WrARNm\\\\\\\/+EUhtf27eB0i9LyFWcDPYBelsiYaed\\\\\\\/J\\\\\\\/gAgnPQV\\\\\\\/DZwjp\\\\\\\/7Q4FgI+h8NLHhWfT0xPbNk7IodPtxq1ste2wxysh3VX2jDsPuOiaSzTv\\\\\\\/Wj1ya9Rc8CwPzgyJ5AfF\\\\\\\/\\\\\\\/q4uDtepoF4s1WlwcHf0RO7jy9J9TOZUiMLW9p13RIrkug5Y73eI8O\\\\\\\/GtDfjPDCjGb78uxuW6WtjgqzYH+xUTsk7sTavS9YGMVMV818ZnWxSBP2hNSZGBF75thYrEpB5da6sjd72tcJRbuA10WOD1hptplgp1LtizqD42dFE2rArZQFubfK2oEPBnCsw6LGRCMEBeQRUQAbEh\\\\\\\/hRKaF4WjIdkGp08ytFax8+Q6mowaVa0\\\\\\\/ADz9NmX2\\\\\\\/aiy5\\\\\\\/Qn+hgPaULsI6tCUBsKigNyp\\\\\\\/aVrVYJ4CYzawXVIXbtVgvgURfZLrmTS5bopPq0wBEKQvwb2C7M7J22j9H+yRhXuYlttLy3nTIBHShC9oEsPGdv50WkaDqpxFcDOTxQYrhYjT51FG88jX5P7Ax948+N+q+RqayzW9BCZXKRGXf+ACo8H0D9r5awgcO17o3lhoECYvxSpP0MTU1PJ\\\\\\\/bVpguRKAwn3QPJcOf\\\\\\\/s9t4sjRhfeVPkYJAQ+6Wb\\\\\\\/TdAzGzCOFxI+nkE9xa+Yv2UMdksnypvHDHfPJihL9\\\\\\\/UG1OCiw14nV2xh2QGo5XdDWElWMsnhUp7Bf33HlbQri46As6KJRXiULpUOM6bL3U\\\\\\\/rFIWDUSafMyvBFPi2O4\\\\\\\/pBP\\\\\\\/HB2Guhxs3eNOFbq7sxoqYhtfyMt1qmb+lOCVkmFesq\\\\\\\/bw71OG3nwsk8laeIBL1RK0vejHqng3b4AtQkC0VW4\\\\\\\/9DupFdFImxacYgUuw5r4Jw\\\\\\\/HA6mo4VrB6eiJTHvbIrVSfcz9370MuhYENi7lnaRg28N4vmlbpaSf6n2MgLUPnKePZ0NRv4aPahe97F3wCob8ZKW8AfyyDvO2YGTFknuQH4nXrNmGmUlI1kfihIJRS3GQwUoYobvWPDfLJy1hUWdcSbqjoWhEhC5uPj4tsg52kbxB5mNmY6uWxDmO89egJpKQko9rU1KKvls2qyGdwhc4CBXIImRRbE2JFhC0VK4zODfwACBjNU8URUH5Y615M0wMNEnW+Y37d6f7yg3ezKMu5R\\\\\\\/hfx3CgUDgVrHGSpSe\\\\\\\/kR+Aio75FJCXZbATo85iYqX5L2LCZ4o1Iw+9xSmLkrT9Ty4GbXxp4\\\\\\\/IABmREyvqe+B61MzkWWcv8wm2sWC58V3dAoy6X7VQA5ZpLadzxdOWA5KFN2SkC0wS5F0+JHT+5MCzTDBGgmZ5PZM25DTbvrLSHJ6boZHKMBRof27qYQnZZsM9aAgYmqr9aoYXh1oRmEMPlS546HcYtGtNKvwg5qP2JjxGMgmPTPbhnMuDcHmJPgeJFQxb5xd9J5cuMFdRJ5hlLL9QTRAluhYtZKnUEh5DAlLQhY5\\\\\\\/jcqO6EnmAJ\\\\\\\/P0NuH128Y7boPbFqrw3f2haCGKOMgwuaDZbaYxJMrJOM+0y1YS6wRo\\\\\\\/OGuEXyr29imUYBBZB+QI1uqHRlpTxEQ0bnvldBe+SMhYjSP3fK0lV5hDnOBQIjdJC87hNnzwaKgVXSaPvxsEqVW7Icm2AZ05kF1eMVw0bxi7yVj8BaEdsMXaU6JJaZclQwt0rZFug04p2CeRmrCx9XDgPVwKVbkEQiFaRGW+pYdxcPv2fJZVNkjLeeRxmEfiSe\\\\\\\/rjSSR79Ap2OoDYXLjf0JNsX6DU6awMnZ0zhjoo+VHqjLi4x8nHzCMBm99I3lrD4V0PU\\\\\\\/L5kmJ\\\\\\\/veV6gvaCFDAQzHjz6b6MiD4KXSu8GHVlTH9w4NajwQLPSHGNnW4l3G5vSF93w\\\\\\\/5XUU5zVbUTxuLqh5Xu8xT\\\\\\\/SeUjlz3g1KxbryJVeD3Bc9IptztMScbrJE+YK0qyIEn5xSvqNiYo6DNHbnKb9t82S4MJrqfL0fns4NRoSr3pDIHoNyZDBSMGoHXSjydIaZRt+aU59iCKmI06UwCuELElKuBXQW0aBFvHtTXenK6Gpin7BZEjfqQQ3w72eUhhSZrrGCqpU+Mn2ye\\\\\\\/T6p2AuceaJzrEsyR16R0wehmSCDRmoSzGwAijHKQK3WvoZcL8BaH0n555GfEtRzi\\\\\\\/YdADP3f+nlt\\\\\\\/cPXVne77L2mxTrI1zsa9fFReDl6FqqVQkS\\\\\\\/kygO5dArg\\\\\\\/Ul0k2U0sy0EzfGoJqxom5h4TioKqQnA\\\\\\\/FPHkfad4JKc6LrFSxjMK1dyCPNzvoechIlops3INtjW76wsJQJYFufGZuXjacZPAWsk+L31T2lmtW+S+uQQxKOkRXaZt6APaOPK3RlM2mzl2t6\\\\\\\/dVWvKOM4\\\\\\\/XmltjseZAOxNoHniUwK6K6LPyCOAYNWZKWSvSXCVjbxtm+ajwrKddgLVSGJz+QGtlc8mKDGDwJTu8tkVDhx9ASHG11zW7\\\\\\\/eB0F9vCdYUjxsRjGFWGT8vWnLM+qMzUCd54I74XVnOpStL3OWxX7r2MY99QjDi5AHpX0aK2A6miuVjVF+41rzXrjSQvQn\\\\\\\/2l+u79ZD9IVfaSzxvxoK+15G9dgbNcn2GkkVRE5f8YCN9niPR\\\\\\\/dAWi2Zo8ijRQAq\\\\\\\/eNnG5YkW8\\\\\\\/QcXWr5p0MWjkE2zW6sSbLTXr\\\\\\\/GsY3cWrJgoFD4hnaI8dDMCbOriEDu3REpbNIJqjIEV3VG7j1b5h8Ps+5kv2ytybd1FngVeAwZ2zLVXXdjfidT2x6Z4UH1U7z8vd4BCxaRfDSqGXrfn5ybGeGgay0428ZvRUMAMjdpPc0iK7HMlaqxMM9bK1B1se2UBYiv4VRefHv2a2CZ+J\\\\\\\/uKJW9EjD7HdetDGk+LUyG6RSzWXqqHgs702HTUaOWaOStCCAh74qES8EpxeQSaOrLRrSLxLiwe5c3kGbwf97WZ53JQ7ff6PukCMzFasYcQvM1wEZAEmdGHgIZ8l4zpbJQQpjOk5nYNq78Npk0JqiER4d0t2gP\\\\\\\/Olvd0ANdMSJaztNOYanqOBYWzpT6xUwOVhaf9fUyMe2Vj2uPRPpfWV0OAotWsPEUHFNNiodjz2UsgzCVv9HrcaJLlpf16c8NiaBRwKVmlm7u+qtueGsaRmRqsNeBhSi1fYHN04J5R9RtZUBKIP0st5LYLRSztZFY4W9OAbY6VjubYWteGQagJwBkcOKjQY9QTdplaqbYSsl+X0JkoUqsr90A6uuijBAncFHOPjOJ3yVeyl7TkA5ObUfpF4YxrZ40ZCzTCjX6nnZQJWws1b0hgfqrWyZx1jcgtOgPpG0NkNIlU8B2+yHFr5Oe5+ZwXb6xy7TJa4SQJTiKtztVvEYtHq9uZPUALM0SASpuajPy0UY1uG+n70YKT3z16XkzlEjIUcEhb+tEO5QSe4pCpI8r8IDMuO+kAh3+JuymxO14QE4I8+KRY4WxGq52LxREn73Vrmr6R\\\\\\\/52sR26\\\\\\\/0ojLmuGYIp\\\\\\\/xkXkbXkn6rocZf2dPV0dN4ievcMrgadqPW+WprIG544GY+tuiAJga5tMpzAAuq+w0voa48X2UcXdGGNViw99yGkDih36wx637tPo0w\\\\\\\/anTDwvpvWwXEIFXaUdVuXAilmFDutKqvarg9FN0UI4cXbkwxrCyzdZyfrEr1CRStm3GxXW2q9vMeAha\\\\\\\/pDBWTX0U3RHg+QTUYCZ697sipDMYwqI+IROTwuFHFK\\\\\\\/CXdv1HABAfx5rG8yiqSe3VkIP2YK39A0WYxfOuhCB2A\\\\\\\/hnnHlpaZH9tSUrgLUkg08E2MTuPbsb12RZRqLmgDJusDJrUYC3Gs3GiVMstOmYPueoJ5yp9wzOjl58q3y+gVFXIWBHOwlYOczSkY3Ki\\\\\\\/bsAMgzo7vR5n3OwScH0d9DCViPEmXvmiLJsc\\\\\\\/7vMpGQNIaO1lPl5dEeZM\\\\\\\/vETuD+dVfvaiqVY513aIHqrHzcPPDpjDEIgqR+vnqXVULYS\\\\\\\/K8x4IiEcJdIdUoduRRqqKMgFUT\\\\\\\/SB8TqrFkR+gMtheLh6pHFofvqAP6JejM9+LozwsmAbzW5zZK3gX2AnytrH2XNuQZftzph2phTX06XMZ8eURN31G7C\\\\\\\/Vd57b9HvXlUUIo\\\\\\\/388K51RZNE6txOCtObJ0aH84DMXpUuNMnbff8CoQUpTO5q6PpSdFi7rJvlH+K3W6AG8oMI6cagI53KPfz2H4hKQUG0ewSK06EkA3woHqdVB9k6VKtXMVT9CKxPdsqyxYem6c14IzA0BjZnRTU1Yj52Vp2X6ALTo4h3G7L\\\\\\\/BwGrrFAmIDMMFBneDGgsFiF1QNDNsfPvYM98tsdmkHLXdTHC9KXpCJkP8LuhBOn96vO4TfiYrqrK8LpoxhuMSD38pRSLKqXiat8BKBLH0CSkI+UFUcXnqxkwG8PDKLQVEO\\\\\\\/pe3Y4subt7ZgTIXsxd\\\\\\\/GbFYyRvARuFDt48AvTIiFzKBXBqRoZRzq8Gs1\\\\\\\/fn9rqq4Lgy\\\\\\\/kyO6+3PGHVNzPerpCegUup5Bsupy884FpJRIajVEeyDmtKmXF45YMIKCOh6Dr\\\\\\\/Ue4j9QIA0jJ3OErIHkzLn27KIpgnmqNrbgrvuiHWDVdlFuqaOyuMcj1EJZh8waj+kpLdJPRaCLS2KIY1jTuS9hChwva5GbNCEsSUuhLIVaCMjdzfEKuCpEIDRsFhPbBBAL8chWzLu24fODs10kPlhwANLopfpTXwY1UHZ5tVW46yNSAUG\\\\\\\/IhoOMPoRm7P9OGR7TfYcJMkduZB+UtnL8zGCzxnxisiQhKEKDDhqnZsms0lCzsXD1RvD4F+hAH\\\\\\\/YJYoN9dSc6e8crZcMssocojiLWI2nTleLeUIUHliOjgMEJOMXdjujhLjLw4UGe5wFESMkIEWMj9c4BpMBmTRZCXzf+wLC4goiNm6SztuYxjEdv2VF8TESmdE3Dhv5\\\\\\\/5SBcClRKgXW2N4hMVehIttVaoBWzT7cGols+MOilPdb3TklcYLcDZfLIevXhk37wBJVYB2WVMxvPTyRZ7YeibsbIkUyBmNrZQgw2K1G6DeNp5r5w+t9eBh5MbacNH7TexX765fGPb+6N7Kmf3tRY4xvPbNvDBLfiRMCan24XF1CtWuH56pNDaXDYwmj5xoOsF7NkoLwZLFokqSKY6Z1W4KjUCtU\\\\\\\/fWLegKl4LgpBZjmmL7j9P\\\\\\\/aSOxToVdTP4Vd4DtNnKdLBzuSnF+j+EQ1NTh4vd6yXd51sDyD\\\\\\\/D\\\\\\\/dRqFLJmJNy38p1KvR04CtWlFWOdSu3vzrljdpowhuvjAdCAj3rKl558Y81S5r7tx9QfW8IyifvA4bcFUdlkbkMF+9Dm5mkbap\\\\\\\/I5imXLv6IQr4Kwd\\\\\\\/nyqM\\\\\\\/uZ8u0trffCPK2TQirMdKhaUE7tcdCOH4UxEEybPH\\\\\\\/K7huGSuznnMT9h4wh7FN+k5f4LEzr4flCLjJHd1DEHnn5+Ecq1Z7yc+HqnE7ruupoz2+UrxmPM23kZQuM+1PpMylVU0o0Izrqibj5edaBW6wAok9PRd3zV4k2HK4wHzGqx8\\\\\\\/\\\\\\\/z\\\\\\\/d1MEZ1DiYioGrP+cAI9WpC06+efds9UuskoXa7iKD4PBxNZ8y6xaB7uloHl8hIHmFYJRFnllA4cnSnyyjxJLkBTPr+B\\\\\\\/EuWsx3KfeZNSogLx1n+8XkNuZhkRPy8jIekEVUTbcWmG7MG6VeD03IpCCKtMojKhrBkVn2I\\\\\\\/\\\\\\\/LhttUMms5lAQjafRs2QIHEHEWmnlPaCvK2T\\\\\\\/JA7o\\\\\\\/yjz18ykDChBDRn6PL2eDyZvorZDojQvDKjH+6J3QtK9a0tL4He7fx0f3OBzC8bZknwGZkXsaoC2h6Pfxi5JgyWeKViCOam1VFmxiOaj2qe8q4TdFbS9WZNEAwZsRw2qUY5i34mnABxTaPRiXKpzWhkozJ3cqnR13WYuTZExN23jKqULAMV9XSlVKxiosq5ItIOzdprW+Mqd7qHOJoIFSDpyjdSauqzIVnF\\\\\\\/vIp1os6wYVQB2Upy1nIn+WL98zuVBbfWw0Gcejn1UityIsXr1V1q\\\\\\\/cwgL0lQ8Xodc+ltSehCvzjpiTcdSQ5VKMwleaoJDspQrO5b92eTrJy0xzWzLmQHMC59FbroJ\\\\\\\/h6L+8ZJEd7N1nbgCe+v3A98l9NnmJCDmo47W\\\\\\\/tw7MlPxWXu9d2yaov5afJ9k\\\\\\\/DmbVIOabma43UibyWCKcQn09DC5547qvIMsBgdoG0gMhZzPvY+b0zkrB8us2VUOZMm0uSLinzrokMVl7v0wBT43FgOl1kIk9bcx5bP6RYe4G+Z6KCDlDKHjZ7J0vXuF1CFs7anPba92XvvPGhVHyecosVdOeCxRxXaZmzfnPLdVVxCNYnR2A2H7tlXPmEhMRSEdTquZCdd1OoSxIbRwCjld3qQinycbVrddme1F1o19ZnqrG+dFKd14i+pk4mVN1sIQuEeUi\\\\\\\/RR29DzTaGeGWOOTIojD8PsacO9E1pK3wLRN7GQ2cX+2hM5fGC0XtBD7EMHuAjZRyNmlgtDXiG2yTKSM4YBBC\\\\\\\/EoTf9IQWd93P9U+CQ79anlMjNRwYiI2AqLkeIBBrYvidrxCDBVeiTPERImyXRlnAk4rY+0J3Ci3nhHF52XrFJuKFGhb7fZ+lgPz4qQo4It4re1pIMVsBaCt+z3aNztljt+6F9xsvImsVXE82qr6gsEfJNYZA3o0C\\\\\\\/xP70sd\\\\\\\/V\\\\\\\/l77YRjv8dA4mwow+Jl\\\\\\\/nTrqNfx3nJ\\\\\\\/quWBoK1GdL3BZOlXLbXLRsBIpD6nKzrBZ2QAoCb906Haty4KgAR3fxV61cOc9jplrVp8qMxwGwya84oT293JqBhu7rinKuPzHHo8SXgbk8kE3ExkDcKD6aFTjeOXU1Goq8iTP5JOPGMU65Vw2jBJiE0JOMjGA2MwB9936NMEOwxawi1zHvCt\\\\\\\/K2OYR3GcjkbdTi1yRbQdgD5xpXXyTOeEaqbhM8mUUh6Y\\\\\\\/6XR4jz\\\\\\\/RXuKpLE5ukhEJzJH+7+y0hRbuypcJ01jQgEbDQ9ZEHEWm5G8830VGGrzDi+eN57mBe9RioLV79sRjV1GfgxieVkJrGuNKJe3vJ\\\\\\\/jYnTe7cIuTKGBHvXpUyLqsYO\\\\\\\/+RDJRv8q8vm8f0cH5u50BcejMqDMz5RN\\\\\\\/FtKWSIWWpt5okyQ\\\\\\\/bc76PkR53FoubNsmUtUzm5GMwt7dZV4ICQT35WSOrLEbtushtyrLqnHjobmJLJBjHM8Oajqi+FYmrvdokfSbgLSU8TNPitQGRLa3OQN3tgtWc5saI00Fzta46Ja8yFCE9SZgNoqHgP1n7evvjZeoN1sLrVAmhgBLAGPnFK8r52yGWj80LlSPOzajLgfQwvABWzVQ2ZybxWdSdPg5Zop4GXjicf6NJHQYS9ES7LAbWk++Ny+daVASdjvRwfMk+fmB7kZY2xZy7koyqFgFk0liahqEvx7dG6+jXyonIzxI8w\\\\\\\/uExoX1kAvXysgLM86iPJWB+D0DuW+8vJCteNxrSlt6QWn6sUuUeM9fk+GnMacNVO2wPWFbOBjrgxLmql\\\\\\\/BkaWW+YuCcUTrAWa2oC81+1jWIe6bSTjkhqu5+xDhsiLV66y5O7k4T4eCbWnDzyLGqYCqVNIgfhAPQryzpjhgvsGKAWivA4HiYHRq1ff1gWuicrFWKGaFRsIlnoahpr5c7NVslr3VxRujB+RdoETgFeikuOdeEylGOXUUJEag4V5iAMQM6A0P6Pl\\\\\\\/1q1PCdaoVZ4AtjPwJiyXveIRgeYRu9WpTzo4+P4GJ915eMDCzjmirJHQcWKgaeRomc43lM7UB2+IkCR8DuILAIOaQI0Yam0y0eraT6WOltWZZ1EgaVz29Tm1BEbdDl0h5S402DCB8YErYH33FS3WHNJwzKRXsPzVX0ssc33SmD6JuctQ018EUgcCknB8NSeQNXYT12VsnAMSDXSiP4UDmlKGSMfk3PJLl2nHyKBs0sCbr\\\\\\\/jTGDTJneNTOkMU86fmyRlv3AYs8nkYYtk\\\\\\\/yZapeQ+1xC5dAm9bdV9gd+pOdF5Ymi5gjwhunoXzseqfJY\\\\\\\/D7YFfu89Z+cRfKzKWT3NHMG2ZXHqMrdh84sKubhlpjAojy2IsMD9kV7T5gOfmll\\\\\\\/s2VTXwCJkXmAqoCVMmCZyHuFIBJHrBaVrJbBvKfV1YI6c3M7nYdl65vSPwdwHibKBs0JXCQnZyzGvxPtESqSb8+dXRA2QWRxmYcF7WxpNaYutnHioev2i\\\\\\\/pyNmOdzIKZhB\\\\\\\/afuyjNBv6v\\\\\\\/c2MlV9f52KMpnQ0\\\\\\\/LGEtMQFnvZ7iZpQn8oBLa\\\\\\\/TkNFRVGsRtNTiNkcNSJdtU3\\\\\\\/B+0MXr9S91KVAzz0xReNdQt31CP9r+jLGk8EwNGpOtV46dntSnHmqhJNObq3KJB4s7sKPn1j8p2SY1oVhiTuQkMnWJJ7k9xRxS2qkMS5TJyVBSzcuckHvpHQ4F2RBvxKFNTcJvsjapKlF08n1nKUH57QAtOY9EBQN4HlbKQQN9h7ieGdHHTOhdPiQOjQhzTshEQ6FG3HUWbRcdNUJsCrVV0hg9sbQBN7QZeN9C3sXXSSnIkVXY0qWMx3c7404kBO2e8AK3Wc64KMzIVyHM4IRXyOutmewDaKy5kpUeDPsL\\\\\\\/NGmFoXomqENdC79cPKPza9vt0ZLP8JBBfEAcVf5P6Yo88Y0izi+\\\\\\\/AnSL3+Au3jYnY\\\\\\\/xz2IINbHSz4\\\\\\\/c8CkWB+gonwB01m5RNEk5gbaYPDI57C+Yi5UICZ4nxngBN8oKs9x0mQmzZJwMYruhVPuz\\\\\\\/sBjk12m30CDAk7oK9CgzyAZZvsjO30MwlKLm\\\\\\\/TGXtRVoG52\\\\\\\/vCmVUqgGyVDULxSxHzh3ftyy2OLujk87ye0C01q+MYcIQuQQxUVeN0KgLVpV5Wy\\\\\\\/TzMpvosUCaaoM5VRa3O\\\\\\\/aloRAiJKN0XHvyJUTLrgzThwAqwYi8Zv5q9LWOJ39NHIK3b1Hwp3++dRRJVaKH1ZDffk9DxOhM9m6sYV5AyTO+BrkpGKZtpF0d3GKvUfaH677XnapKSGUuG5DNs39n0gZGonnSbwGQe2W2VDvn33qked1Gg0ysgQZNNKinUaCIWCQ\\\\\\\/HYEPbb6s1IzbKGC5Z0wrCGktrCimbW873bKNZp1AP0T8KokNtBOqDFq1fIuIt3Ua68tHBVRviD0DkYAm8bx60SHmrqHtSQmewiivvBM3h+GOa15wxgpxhPTneb2kCAugqQYwOSQgM\\\\\\\/ZFY8+OliFfHaQVIF90uQrfXf\\\\\\\/bTXLc\\\\\\\/E7c0jv5KjUp\\\\\\\/HRhKEEZHj0dXT0fn+wzSiWPoMZX6SZzaOzBJVWq0ay32zSvIzf99s0PFGpn7TJvO7K1Me+ilw\\\\\\\/zqI37uSMf3OWcFU0W+vhp0KmXt9FFl9bvRgrSxrRDBoOmT68oSwfVL0igL5xppjhn4MO2YNr+PyuSGmwkgf8j3ICfrw+JuKMb0YpyHGxumDqpmOPrNVGHPR77T77kVC92J9YQva\\\\\\\/vvux9FwyCx\\\\\\\/GwqQDoq2fmn1fNSieOJDl4nx8OzHBxAhr\\\\\\\/EPZP2c54Q7T67y1o+CjMqfDSNExrCQ8ROoi2gVpUwWheFoL8kBSdBgBBCwHVwSgWUXAlMpGMB+C7pJ0UUNxqmlAikUxcTfg6h9f+J2otOx6Ss733iN+XPXTEgDP5FGcWGzAhTkMrPUERuVMd6UJJ6jRhqaTQ5QX75MCguMh7Oc9Aiq0AyK+ULErB\\\\\\\/ZhN7dIOhefy5zHbkmnPbCUdGjHQpJ4FyjXDftcqCJYBwh9opHbJCeQr8ePuybDABb7V+s40lCe36SZYUheAdZCQp9FTZg0jgK\\\\\\\/aaM4+n8Sz8cdvjLRbv0O2ttXMQfKCvb0M89YvO2kUVQwJo2AlgTzXhb5sMqOeOOqjLp2SWe3Wi0I73GKI5n+NeZ74G+rAjkPMeaVHcsh+xWQnbb5MpPqt+YePqtBter2PWaYiNm17xUxxqE3N9MhGbRfkoxFTDt+tFoRdhDXLTbE8ISjIPbeMxuxMv2OAO2JV0X+tGuNA3mXgQzwESFVuLqObns0iIRELH4xHAEBNJKhXBMziBGea6pmZrcoZ0jUjxb6wzGPUvXxY1osE3rJCnwWnI2OinpFLyYGxRnQRtu2NfN1EiSiwj+MLaC7iplc0Mgsvh\\\\\\\/gP7SuDxxv9UvaemVqd38h54yw4BssfupvodGrytmWHhUacCF7wr3MOzUKSER0wBHxHUFu2i\\\\\\\/S8LrnzCAERmSDA8CQyg8VBFGFJKMavG6ufUbPNOu20IfMVDxklb3WXg05+n3\\\\\\\/N\\\\\\\/Az6GN2pv\\\\\\\/VT6xPK04KhcduJGlrISl08kDHflO\\\\\\\/\\\\\\\/K9Q7fWxspxejkm3GmQpQAZccMakVlasUPoOWVaASxqghuFKG3+yQFir\\\\\\\/UpT6wIYuzNOlUPFA\\\\\\\/MhcRMyzB+x722MiotDGbtmiDkJN5S0AbRR\\\\\\\/7WRlRI4f85CGzvtJ4\\\\\\\/b30VmrlRw4JQpEuUSBdAjzuExwofaHqiJQgfdBkUR4oilZm8ebBG\\\\\\\/ejCxf\\\\\\\/waLnSlnsdmPNKaNCK7qsbpINe2IGjkH8FYYIL8NWTTA8IpdArldtpRTTiSMSs+i2jp6OHwuRG90jh5kDvieDngRBuMRBqtZvx2cEUCAVul3Un9LtwP6l9oKNc+o6nD4Z3oFG3mZJyJXdfRi323H3ksWg+meuze3rAfJkgbMJrHlEaKUQrx1XCQr0jfqWbtJ8Slolob4yyWVMSGzokf\\\\\\\/6dilUJh3iwv7ZJbSTgbiP2L1wMmPEg4tV3D2jWeuYKmZ7mPVQKNzkbsJPHcfVXUgho3vszf3A91NumP4LBgiEi1x53k6CTaF284ofmoWTIwIWb5IvP6S2qH0W\\\\\\\/PvWgbxmpafX6u3XXoQOwIIjZ+NikQUjbg6jQnbp6gKubK7WZmap2PYUCn3a36551g+8IS2roHgI\\\\\\\/DjVOdFsdXL24IncJBBOHG7NB5\\\\\\\/PyAaw\\\\\\\/oowFEv6tLqQeSZiaR\\\\\\\/9nkb96FFBIS6KXabwbCQ3b0qJgOAJYOOPT9beUec3YxeE2x45TgJ1kWe6MpnREBrs36zPLsAWTVdk3pmm0pGVyV9UfDiWEr2iuvn3v5sYW+jiVR8Uezlknwicax9HrUmNV\\\\\\\/drGMTLIttSu4eNYqfse232htpCrGKcCdQVcRzn3wUfxhLD1qvPbft7Tz39A1320ZcNKBbJRuU8nydXbgAwDMfL7HmVLrHs7RKSSzRiz9Lyk9nO31mh+Rtkas+fzsu2oJqb4IN7JDoD8Ceahm\\\\\\\/u53Qy6\\\\\\\/qtOEq0PjbDRErSwcYwkm8NJ\\\\\\\/9ADIF5U+ytvAOCeLsnTcB+TVPsnx7IiM1C7IxH1ZTj2Pu3CQFskziUjCAI1I91oFPMp68KK0g2+D8H+91p25pcKDWdTt4lblq1nhl8TlZLRr6IbATWndiO6TWdCdAkMJQ0zd1pGxggirVj6nGmG7SvM+sQdw2sk+\\\\\\\/DozysxIG+xnLMHQNe92G2MrMbS5eUuZfEhVFATUdqc+tEpY5vUw8wS4muZyalIk7ScUPuL8gQ1eOULCDo5rYHH7xhkwsh0Upioc07oX91kXaaz0nzXUcVs\\\\\\\/Z5SYUebBPWKHj5paiwNs9roD257ss0fGsmRfh2PRkb9DNwBrzlJrCbTdStvWkRsrbjTRUr7SsLuMZ3dV\\\\\\\/ax1ScoWIStkVS0wivoTszixH9rkmXE703AlSkdPLxxbzyW12nJamoLmGdFS4tdZNaiGy\\\\\\\/UvZwFq4KYBe+L9ZQ4D7xR2iixzeba2MsJUhWAlAYGk8L9pCjOQ85SZj3g02TBgSMRAUBb\\\\\\\/p+0vgS+gQ5PnVImDp7KhBLSGUurWZlXlP+yjQ5TzLcCZSJg8iZ6KxYDWlg5whuARSUfcrdv7gziBB\\\\\\\/gmlVL6T4RSVsPlAoZcMe0YZJ495WkvwUcaZVD3VMp48q2jJDnz8OGXWye+Z9aNpJ\\\\\\\/q3x2JH8rrjFg71Eia1AwvwSnfdkOswAeS7It+9SIdCofILYfZyiQAiMhi2KDZ\\\\\\\/rV7VxWFVFe5ukVLJgp3Gwb0I+Zxm\\\\\\\/eUz9PdOTzXzTURkpwaEXeJEoBvscJCew54AzKLVLeWnlgP8j26udikuimc6CE1qO3ngf0WVqgLOUF0hRbroni3VNtpaWB7PiYi2ycPuPA0Ah4D\\\\\\\/EHrzPkWEjQBa5LMQ3RLa3HwXUbLQCZLRA55uq0hU5VdTyWU5wf1x\\\\\\\/ludh88pJIZThP51ygogBhvHYEa8oAs285jmSYNSk02bOQpQS8fcxhV0PssravZBYZx\\\\\\\/rEwdqCXyLZO7OKBR7AKnFIBzmUC9tZLv2MvNHR3egfvhidIQlVHFLIYwHlF3EjYyyh2Q8Nkg+Mjq5\\\\\\\/P7i2LnZYh0\\\\\\\/Oyuwvr1wJ3Qy+xwXkvlivAYioCIasBBTE5X+A0IyeTTLg0+cBDJKEHPaP8odogK23OV3DH9MvOPFoM4O6QCtFviQ5INH4RPp\\\\\\\/9w6KkEpaGUGudEKFrB1BZCj6avAogP5Q4WLrvF8a0b9eo\\\\\\\/NSJzhcsWLlmEI6yDIURQ0jyw2I9eb6z7XrQwsfOPvgwcsg6sw9jrtSqx6ylTxB+rF06RtXkqp4WKXR2aAXsUZDi09inf54I95c6f\\\\\\\/WTbrD03u5PcBBCc5oB+UgDN3jJBDqEfxRFAAQKYByhWErE\\\\\\\/eyMbVN96lTvGOHwEE+7+f6NOdx1+H9bJPZIlSDJLQKzcgusMLCY62V0iGefFXk9W7ooGdYqxzDI9hd4tIVtSlT9pkqDnlzGMW681jnJNX+NoGWqrnzaJ+OmR13mIwSnT7G7ue3\\\\\\\/MeixVy05bH5l4c0iX+0kB2hcVC\\\\\\\/FvvdwE+oWa03KoYIZ48hP3mdNy2d5r5f\\\\\\\/\\\\\\\/HRfQ6tEgDE+mRH+5dHRDqOu9Fu76I4J5HZ7MuauaQJoOBbctrDXaUJzYp6PhJe8mFPUWZfZGByyqUuQpmT7ZDA6DL+\\\\\\\/w\\\\\\\/ZNDcoBqKgOIrfs9ISWT+GxekZ\\\\\\\/ErMDxUo\\\\\\\/WtlUofDkSbUf7tmmlwjoggAf2IPuXcLSyaHER4T9K2\\\\\\\/9d40ZVj9eY4f5fw5lES5bITfQxEpzIDO42UxplnTT2PpMXp1hpUVestyOXd+SgRjNnylFI+qA28RUFluq5lKhVZORv6bjYnlRqYpLNA+heCociK2XZPpPjoC1dPcdZSE5\\\\\\\/gUIkNwB1ehV3H9dUaA5iFJa+aFZeaUSM09LAnwqf\\\\\\\/TZWUgMmOvA33D\\\\\\\/gJu\\\\\\\/x\\\\\\\/huImmr6YqBgSNVAtJUl29rwMNbvs+E6yYU6bftAeE\\\\\\\/il5qqo9A0VsPPTgrj+fu3XidXqnOJ8x3khWOt2JNJcAkRIQpKEw052rBCAB2QqzFl1jqWRL6r2ra2CI4lJRmABDkb5tRaWrVsHUKYNxhFOT0gKTJjFP5a\\\\\\\/hpK2xvfgUtovlUBnH0quNj5LMaR64bWG2FpyPImW7B4ezQZJnjzePDBck+nSvbx1vCCMgNng+h7vnSx6B0xVG0CNQS3iZNhh5VI9o7fEJVeWe\\\\\\\/JCJCBhyBlMWq6xn8h1xudVCo5BBHL9\\\\\\\/Xo+bjZtdImKPqiGs1IbzL\\\\\\\/o4cKsGZcGuT78q32QN\\\\\\\/T9WXoUp3tbqwYvV3bGuCIWGM8C9Zg+uHcSM3Ks6C9oexxoj1lKigiVv5TwqWO2c8WE5yX1FxGnks58Rm9az0gP0EbiQatzFYYgxTENiDNMv1M1ZVR6BYiz2x6Dd+Yby0tSa+ICsNDAIh4dEJ+24Nf68stya7Xz2e6ibIKnGXP4TeQgdGRybO1CbC5kh\\\\\\\/Z0WtHFyr5LF0TxJPo9JxgY51rOz62Os23kcluL0BpPob6kUq1Kk5kdU6BpnIoMn9sv+5\\\\\\\/Es7tOdrgCAzpJtKroB2RqCSUiQhkS0veapfKbmKUGrl6PTXBqa1zEnXdTn14OfuoHLwAScB28NQP1QHhQ51YCq5Z\\\\\\\/pfUPW6SDeoN+NKCtP9qNxnOSjHUJXVYOi43A8M2x5NAzPxUaqGw4bYLsg96t5IfKYsFeJycgPAsJ2M\\\\\\\/D02NxVAow7cuQ7x73t4Gyhunm76uYWSQRjYeAeiZnmrWdNO9ENFUYzYXZcdenNqh4fpJIl2rRaBY8gWjFjWnIISD6JwiBxsapC5ccxx2Ca1yHlbO5HaOL0Df8VOc2jU42ItkbIHcy45vQ4\\\\\\\/k\\\\\\\/meHUQ3QIn5l3vujzOaQuSgsty0QlNb4jXTQ6taoS+h8JtDI4WcErUWY0MMyoOM3lp00NMWBgnxHN64IAmDBtuA9APkXYHOMl8tXU+w\\\\\\\/FVxa+t0uS6qEsL5zc\\\\\\\/b3f3vVHfCYY6F0VvW1lRx7KF+z7p7ZKxMDLKF13jVwBqwwGMNOSourA\\\\\\\/8CyJnUSMXX\\\\\\\/D9ALcgsqTEGbGwZgToKm0OZL8u35Gc2bi8uqSTbtDTuuwhMzzkBW8Kb+TiFVUijRwDWAXm2JTI4b\\\\\\\/5TsKaa1nNnXAfb\\\\\\\/+cotBAduswMscC952GYkWbt9JL1VNkPwoNKviV7NSTo1R0TQQKuY\\\\\\\/RlUEiquoF8+R+T\\\\\\\/CjAljpHe+jfqW\\\\\\\/AuDPPAvb1\\\\\\\/mtQ42TPJVyvD7Tulgn7\\\\\\\/vVP+Yhp\\\\\\\/KDJpyw\\\\\\\/ONWjUFmGfS3zMSd3K9WlgrhvVf1R7hF9Bz8H13LiwF+Aes7CZM1shylxVbd8z5p926lWmh7lLC4bQe6yTe9anPcLRzhJ0nWk5D8NW+Q\\\\\\\/GzUa4\\\\\\\/5fwDv9ES55AWtruUH87czONkXCvGfTXISxEUM5Mv8IniBk6mBoCBoRjoeKxsqH17j3yo8zartT6d0DGum6LHH4W2gbI\\\\\\\/Lb3gOD40YEtiuK8Q3t3Ex2rA7C6mgKsH9L\\\\\\\/E0lEsmTvjr8232VOWr5R9SIeEPsV03Mf9MrBNkh9oOVb84MWe514r5PNUxWF0moWuJEynJP0Elf3lnYML2hVCiyUD1+WWMV56egrff9FxVrd1EDU+EUwmWPEbfRdy2cv\\\\\\\/yTuAfXCHvlVMnW7ZJe17LWAysMhrOdIc3PUV4GNixl5wnQMC5h4Yt5djzqGDNdawwiMhuDpHDfnyBbbHJHif3WG+WVHl3dyfjOXKVCVuFqaHbtLqyPDww5ENXmm6mBXVlfXoVg1WjPw0zq0eZquEUoWDIO28kqBrqpuMoT7R+e+8F6YCqyjzvD6QUK6RyUDAw8kFI\\\\\\\/GAeBJR37oSBp6MHQrgBzUZC\\\\\\\/Q\\\\\\\/ANS6kEWwizo2jSa+jeGYW8+PGfUimIW\\\\\\\/S1wPSvd7xuooKAALPaK3AmSCOvTFUNQuQBTbJiVxtW5TzYngLqgbIMm98wqiJOmhZFBEeQYiqZ\\\\\\\/+yR0Nmx+h1gWfUBblw+Bk7UENdoJO7Qi2R\\\\\\\/VHOCJOdIUgaUbARspWKg0+9rYQFhKsPmqfTFEGDM4uox+cRCN+A0YIVmasbX5A65Eqrs5oibKf+h9pZh5Dm9v0XDb1x7NCEcfbeqU+N5QvlNP55ZkpybmDEq9U9xzuvvsWrxfoNAmgZ1NtclwZ7o+Fswp1zJaTIH8SkbhmfMsk\\\\\\\/I2yrb2vpiR\\\\\\\/JcrbW84h3YvqkwmW84N8sa784plqg2e4YfMtr5gcE3WcDOgWRs2udUPSWJoaZ76ViuuuZzbyLYRl68tjO3+TpALwrwc6b\\\\\\\/XtD2jZUtj\\\\\\\/jhjZLHi9V6cq8vS3mxnrf1KQVQXdTvFERJj1hLtl45ftwQmp97e3NboMHoTrEmK+mB1CTqKx\\\\\\\/9UnyI86EZQxJ\\\\\\\/F8r5\\\\\\\/6Th1CX+H8oiajcsu7yBUYyGYyt+xyhvu8dF3CNRjXR4dxf\\\\\\\/D46zKRZQP8K8s8ZT4zvRVzTPnUgDVy0QXECILXYJkuGVfWIlPUfAxs20sxG\\\\\\\/jaqNYaEAJcA12pnWgU3\\\\\\\/npEQDHcpJTJ7KpH6vxBMisc0zxw4NAaly9Eh40\\\\\\\/ULxTi6lYEfCFLh0qBLQczGYx64utWYr+\\\\\\\/C8BAFRLXv5nEQ25ntzqVpRNRotCZ\\\\\\\/DsYl0ACPeeziN9Ohdj2ROZvDtUE+xZ0J5sKuMwA34KdWz2NrlUI92OtRmHUlcSKF\\\\\\\/isPXdC0RlA05naI7B8A2fKWYmHBlekDbMf+3dAqN16DbE1fB8q0prQlS56vA3G7HFet77slhAo\\\\\\\/gSE4TDG0EzpfJVggpFbQUGBvKcYKaM\\\\\\\/3T+jS99gS98W0wn4YgS7tHvg8MUQXzBDHO6QHZCSzr56eOfr1v9vvVIAAACbaxISmeCod5LMD9nVtxZXmGvTGCpC7f2+NV+Z109p+AkeM2Lfh+JnFiBo\\\\\\\/TfWAX9iw\\\\\\\/V85q9y+IYaH5xRPXsVOoUN0LElJgMGrUtHSn52Vj1fEbzYydeVAg7N6+8rUYKZOXIpJdR97AVg+xrJB0ahZtmKGyM+FG8gteT\\\\\\\/WnGElhvwzS7raTV7lalsx5bVkHPqK\\\\\\\/nwNCmnPBRyDqWAC034gRe3bfJnqMj3R2KkfGmkPMjBEikkMy9mAA1NF4JvyeMLR0fmKhNWj0ch0Rusnxj1WjhqLkYftg8iDmmBpNjkrIJVHdY0XNiNTpwVrmTBTGh9gxNCsjv5xEW2Wh5xFELJZ4pBgEd94YnhZ3HXNM7SbftRsLNoPX0Wozzq3hSZ\\\\\\\/xWGRg1LKw5SQkIiXu\\\\\\\/vxAxjtYIZusa9jfh4aiQ9owo2SQsExKGgmhb5ENoJaECQkgJ71L6TmH1Jl+BLsyv9DZ8CVISH2mV6tdwkkkPRFMfACNV9seK0jg05cehR0JNWAiJn+2SeDzJPYT6FHeWhOY7EKsI1+bHGVSXYe6OXf28nsRqJy7kG0Ga6nYyyEGXpRIOM7YWOKS+8ZraHSYbrjkyTIuy6K7rrK+540NCdQ2Q3gWdjrVLZptUcfmRMYwXFeJuLITCruVsob6UBhFpWcHV6D6VBgnR04KD5hUN9R4Frqtt+dkTjAS0wQzsVFvwTrCBTBvi6wi8FostdjQKvcIG6flyS6TSPEW\\\\\\\/Y0GjKUi\\\\\\\/W4bm5\\\\\\\/nl4wHzs1z7bvqgFVzhGnQ85ZUGtgQReHKInqsbZ8qgxGbk2EM8Hi3qZJMk0I6e89uMaYAYcGzNQbY0O4VEK68bWLXLSWDuu5eKZRiGPeIGaRI3FtkSvTXK8cWkDT2XXsfsqgd8937cr7Z2kNUJ9OLg0iYSPVw20\\\\\\\/5IKAwcwXlAh8aIkeXwZk0h9rSkYZhA5ReUv3pI+FPHadBporvceU2\\\\\\\/AgwKOeAg3\\\\\\\/GGErBOQHZRTOXtqb4LspPzFExfskfCwGvxFmc9\\\\\\\/WDFYCWs32UVEpQg\\\\\\\/Q4kPLOkjk42zQvq7WSkO99Yu1RH7AMHCrmHOOuLXvICVCqLEgvSNLdY+wDiT2\\\\\\\/vgBU6JZ7MK2Si6oHH9fKQyJVkoRPPo\\\\\\\/qf5pNUongAkf38VFXChxuNIxCHBANxwyp8qv2TUMrzZ9OZjlw4IfnQKQ+ISaJ9NWqsNyffLGnZFvGuIWEnTipEsLb1j6hhnoGIAxr9jj7K4+OFULc5BFFkwJiqsLKE05wdGWE+fNI8pKgfadm82qYDDSje89wp2O5WL\\\\\\\/sKsfboPo1SZQ5W3NEs1gHbS0FDBRCkYF1GqaTIaFfvZmfjSilXXOUjcpBq8hA3ovhxQd5QhQKt++XdLhiwCvYYC2uOpx0nFRpnauam7Txj5VKuDMEnvL7kvV69GfGjs7Y2uKTmZUroeJI7VG80\\\\\\\/UORfMwywBkXFEqtgHSFI5qowyRX2iG\\\\\\\/t0jySIczib9yAmKvhmtgjR0RqNUP9+vfAoVYnF39Cy4QaSKRERV+ww22wSXeuaFE4aw9fOdvDxfW9WJ9xSSXfFcyw77vCJp0rOfA9Qw20SMtOAN\\\\\\\/w6wrIjOCTHo+MHsYj7Hmsp+7LD1Om\\\\\\\/lBOLQb6ZYZKDqe\\\\\\\/xcfg8tekqgKhES0\\\\\\\/SSsRDwF99cJctKY4kI1uUo1OhwriaqgsPmdFfKYFqXSYXBrVUyqxI+K3fow4IlhIfpukOeWhTSHtaiUwB4fXKnoGJsqZd+W5asbWfQFq0QhQra9kp6GvUeTKIzXlXqxgRL7EaKtLX0DuR31mRRaYF0ypzYakNwkSwURFRFMoYa8mu6qwlHpWJpIp6vD5ENK4kSbagYhBbTPb8+a+e\\\\\\\/\\\\\\\/oTs85VyaC6vUlVEJcldiffjkYx3X8FsR2ZBIV+8t4iDKTRbmAQp6+LaWImvBMN541gVrMKTcV3bAVNVdQ9Vr1MQO45vKR40VrEXT6oALix1kxhNCy9zjbXpCSup6x7DLFbzeMhTXrsHN9RsKLUA3xTUeI4TcPXZO3EAjh8GwROptjRoKe8zzMFKW0zfajoIqeVDuPnrG79xbbUHHZj3+TNFGwWjMmEK2UGprP9pGIWfJuXjljb86hHLXb8iYIvS9yRE5vYHJekCTpsS41xmdQiHXMAk6Sw0mAvqPls0C\\\\\\\/v\\\\\\\/Q0f4wi3T44ni6j28XSngwZeoTK4jMfsXH7LfMdq68i+RdXKfPxKAwpsIsU11tgJU\\\\\\\/cwWqrIHzP2xFn0FsNuE9hfHnN1bPvmNlsQZ+ddqGyjnAAWXXfn8a8nqXYqwxvqTfxsGA4sAnQ8pSkW2t9asf7rJyMGDK52HbnHz5JNVWMOhsHDeHzWcp9eh+Cn+\\\\\\\/NZEkncGfk+QrV7wt8dpi\\\\\\\/0g\\\\\\\/2xERU0eLB3LMbVjQqk8Q2Lc0ukzpCKfA4hyCuab9MiSDyBnVgzPP04\\\\\\\/m8xBuazDAZnLwFmCav2mgNA4jd8qex0McG3Rp3qVCbCAlb3Sj0Le\\\\\\\/2VB4kwgYjBQjqBdCt5LSnYXfpFZlY7MTA2dPQmjvkXdJd9xdOW7G53LqKKLrhxIM1fAebDBuVI6yz2hT5JL0a+kFrfoiNa63ZLYzkWg1f4lav6XYCBr8eEN3ykHu36zFjeVpO0T\\\\\\\/otaMcVhZjetPvCYebrFuUeDj2at83AVtncgUF263s7jZ0qTW4lLlTec0VCsQ8KOuJEHIYZbx7GNKOcJY\\\\\\\/muwQR0f2PKJMCRuTzJFtbDyCm6OxldgHyzI0xVfUw9hgF+KO0RCTZlr42E1p4pTNvcS1Y2JZ+4lLSsdVd0tlSRtf0f0VGoY4DmwBbTrRArWtzqZhELQlzG+dkoXNvLYeyV1SzL1taA9dy0b1Ud+9GhwE1fZiL358nDislohbXi0X7elKnoF8ea999oUoa\\\\\\\/uGvLSIfyknvyYoWjcuWZmg8qfgKFMNPbC9Acgzo81RNc+gAZT5JWE0ElspmffGBQfeRnm1Y0YKBZzVP0ndmJR\\\\\\\/JW2Wb0L3FkS1WCAoXUy2kM9uO094j4OTZj57Tz4rfaVhGCsDLVz1vKfYUoof6JL9nipQP2sYHHpXpzPSxg1q37p\\\\\\\/m67QgCFQyTMuMhZGoglgxrKUndwX40ypL3vohmVLcAIAmqbQoRKAxPeyxHFnxD0grSrgJTx2u4nQnlYfafLzYdSkI+ByaQegWYEPFzOR\\\\\\\/DUjJrEWP2UFv9KnXYMLA0PPS+rIv5zm98pE97k4OM\\\\\\\/DanHQ3mM\\\\\\\/lprWirizYFvEjk7bmuH5cl2uurUDq1FjYuxjTP8JJ3tcTiMTL0Or4ECydXfq46fpDv5wnlvYk0vE0pGrZOKNqUsQJQF\\\\\\\/rsDDh7\\\\\\\/2kZWul1c71XDwzOuQwm7tr3VQjPAAw05fSYm17587YXncogwLCShU1HYp+rrakJsJJRrF6WA4VrD+yFnwJWbBtIvmRHMHoOZORx7AWAxlxaFyImPTdWsTZE20I6HVFE3YU\\\\\\\/5Jhng52OBvWTc4eIPPvfGOdwKa8FsRMnhauX89MTeTPfuJwR7Bto0wgSSBL\\\\\\\/kvKKUzAOyJg5FJXbLhBo9zr62fuX+ZDWeNdefH7qmVIi323Dmbc30eAJli\\\\\\\/XiHGbO7pYCTUd0oDw50f2lWCopjYU3bMP3M1Ku3S\\\\\\\/JpAiIOpV\\\\\\\/vWD\\\\\\\/L\\\\\\\/qb\\\\\\\/\\\\\\\/8NiGR57SA2zsTaSOamHtA7TeUzfOSCVJXYx9TXlpIPJ2QPsdhFRMu+\\\\\\\/\\\\\\\/6LmSaj9Zb74fjhYmlfZ\\\\\\\/cTx3\\\\\\\/9H5wQ0et3GX9r3p0Gx6\\\\\\\/2nWttQtx0FLb1KAjqm\\\\\\\/pyNqBY+QDuDUbFExxi6Kw4TCjneFTUcquLfmvZl65yOsYraVlc1Zl3pzY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022jVd81QsTZ7EvijWDB4t9Fg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f2100fdfefb0aa4829fe98968458c3471229912886cca8abf6303dacb473a5d1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KGWoB3xE5juAcTpptapzzIUU38\\\\\\\/V6ly8wz\\\\\\\/\\\\\\\/2Fx8T\\\\\\\/rE4g9iPjuma5r7IbgtskxMPyxLd32EBRpeo+gCXmttHEoQY3QFXAWhE3mhQIsMVXyZ8cBVaFb7eJ3FibtK7P43g5OXxs1zFfRHf5sj7VtIxwO1YVnZQL40Iutpu0v1cjgpkOJ5F8Jkux0Qw6qEHnTsQimmZ0ySlZZwyl7qENaEVaQwqo53MFHulrwLmVoyfBr2dYT6ATVCEak9k+XbnPzHe1OJyqtULWS4wi9xOSnOSS+ntb+mzho+1USNB2wM0WFQ08UgxCHJGz5eG8p6kflAonyk90OKULRc85KX3xF7suQuMGvCWXISIWxJ\\\\\\\/1o\\\\\\\/5LzztEpw6cxAoQ7cVrEMQM0HvyMrzCS\\\\\\\/+XE+jvz7XZff5Qcvm9tyU\\\\\\\/GvggVF3CoRI8ITOiruTqW7LXKFwA06VX4rx43DvmwMcu3BB9t6UzbMGmoLp5ru\\\\\\\/d+jWRJQw7i8E5EIKxarsKHLoPooJvw7rsbhb5ezfSPISWyqFiG7loHi0ENGqwozeArqKhYLylKVeQRtPrlKlGC81QieKdcpMRI5VCn7GQYQ9FuH\\\\\\\/IiiYyaASVOznDTXrMTv7xivpuBImUW0uVten2gahCeQUE4ij69\\\\\\\/8tA4\\\\\\\/kNa5cX8CW12HjXxgtvt1j5W+mzZIyaklyZrkGOhztKYjdF7gUXJTqiiiB+nIUZAq96IiLBGk\\\\\\\/mrIbcBVqJJoAen\\\\\\\/oDPIEF5UbZ8ADFG38+7iYkhg2wGb0wLWmHWH0SRwUMMWKkh1J14K9FDOT6AD\\\\\\\/V++ISPP5XllOP71lQpABeQ6SI8s1ExtQlU3Kp\\\\\\\/xpOq3sc20Bg1Vxim5gbdmwpl6MIyls7Ny7J7EvfB7SNWSQ3JfjqiezN+JB6v8KBj91TRCw7rPEVhoKFBL9eINwCWiaV7isX7p++nflyiK3rvKJ4d8Qn78+QfEJr6ZWRD6+IJ\\\\\\\/R90sJRC5N3UlYTEAM1DpHr+9yVvDxdpKJ06P4btDvkRltH9Oo1TehloJqb+fDg67wW4ihEtBgwSXz2HIj4EHv7hy\\\\\\\/S6qfLraZfBVZqsaAAy7lL40He7KoZusNAEfH7m2NhDcOJsMb8er+CDaMUogwSSGIi4rcIyr8D34oIBr6iti2+8o18G\\\\\\\/DccW2Ji4sbNlFSjU5RRTzk+nMNQsFPQBI6lTuX2wYqiKlIfb7l1ikAgtALIRlroyvKDvnDUZA\\\\\\\/iwcXttgDdzF4agW4wQ4fk\\\\\\\/\\\\\\\/i6YiNKU02ks\\\\\\\/ObWJbuZO0FnczzrsaumCgm3b91s0fqLD4jnE6U2M2NPNpaB2wV0HhEUEqOfF5a\\\\\\\/GkoukPvurbVrbJZnSorwh3nuLuMeBcr8KCuiC31bybvfFSQayfVZTjaKIhQRgB0EM4KNib13eTSWSfDgG52n+c+093meWsAcrIRuH90PRHDJ45TKJGLI3ADtUzok1vwTfMn27E\\\\\\\/0CRK20UAWRzsVEGgL\\\\\\\/OtxuxFkK8DP2fSDwOTNrSRNbcBluGulmKPNC67qBwDiwbqLfPkxFmgL25zCgxgM+2RlnuHe5iEh8SMCOdQKfJU3\\\\\\\/BYdV\\\\\\\/GGiROLYdwg0j+rAn5a\\\\\\\/ilNAD11LagaSehTwO6IN6Esl\\\\\\\/LnoZOfMjuuOnJcGEfVgQeIqErp0rKyqHueQGDiaYf\\\\\\\/7kizj0\\\\\\\/n6ZZZMD6VzYLk7KUnRxfENdRiszveEzi3\\\\\\\/GfybQ0WThFzEku4pMKbH5s9Wg1qdWF3dYJ\\\\\\\/NUGFO0f6zFXY3XFOOK94ILWKAsQfqKiyecFd3WesEzXJKhohe0eKm538DEfLzf0Il1l3Csnldb8Kg+0AkYoZnVPTDmIJV6F5tROIgcEdP+YWEQkWJzuwcA8PyZR6xgmKQkGSsj1\\\\\\\/LfSVkN9PQn8z\\\\\\\/qbNq6kuxVUrjh7qZyn0DIKCaVqaHfMBDmBqDZ6t41ZZdsVAJa\\\\\\\/1pEDJKhg1YSqda3e0fqb2DM5Hhq72fSTxGpYIwcjJkdy90JzV8HakLiAXVSAcGjaGqF3WhewAK94a8Fdi1KwKy8lzJBynO1vDhIvZ6NGPX++8uAY7KtJAOQgmKCQo4p9QAL9G7m9x8Q3E+q75DWEAFI1tPQQzIr1sF2HKLmgA2gjqiO2\\\\\\\/2VMfUGxgpVc3kHYsZntcKkmZXE8A3eATw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BdSH3CUbcZgxekrLWq+Ssg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022420ed83073b562308aea2bde7b554930050e2a57b8640034f355ebbe5c433b13\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490960 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3255" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8dY1Y93TPUM2RUgiYv6iuzATcmNDSb6AvyNjmwedVnc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3255" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6C4EOledDxT6d0-Lqbc9ice-RMzWpnZKP4FlpWmugxU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3247" title="Genomic Structural Variations Within Five Continental Populations of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3269" title="Genomic Signatures of Adaptation to a Precipitation Gradient in Nigerian Sorghum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490960" id="node490960" data-pisa="ggg;8/10/3255" data-pisa-master="ggg;g3.118.200502" data-apath="/ggg/8/10/3255.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7206-8088" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Genevieve L. Wojcik</span>, <span class="highwire-citation-author" data-delta="1">Christian Fuchsberger</span>, <span class="highwire-citation-author" data-delta="2">Daniel Taliun</span>, <span class="highwire-citation-author" data-delta="3">Ryan Welch</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0241-3522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alicia R Martin</span>, <span class="highwire-citation-author" data-delta="5">Suyash Shringarpure</span>, <span class="highwire-citation-author" data-delta="6">Christopher S. Carlson</span>, <span class="highwire-citation-author" data-delta="7">Goncalo Abecasis</span>, <span class="highwire-citation-author" data-delta="8">Hyun Min Kang</span>, <span class="highwire-citation-author" data-delta="9">Michael Boehnke</span>, <span class="highwire-citation-author" data-delta="10">Carlos D. Bustamante</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0001-9728-6567" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher R. Gignoux</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-9198-759X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eimear E. Kenny</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3255-3267;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200502 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490960" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Genevieve L. Wojcik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Genevieve%2BL.%2BWojcik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wojcik%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGenevieve%2BL.%2BWojcik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7206-8088" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Genevieve L. Wojcik</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian Fuchsberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biomedicine, European Academy of Bolzano/Bozen (EURAC), affiliated with the University of Lübeck, Bolzano, Bozen, 39100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BFuchsberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchsberger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BFuchsberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Taliun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BTaliun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taliun%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BTaliun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ryan Welch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BWelch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welch%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BWelch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alicia R Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BR%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlicia%2BR%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0241-3522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alicia R Martin</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Suyash Shringarpure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suyash%2BShringarpure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shringarpure%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuyash%2BShringarpure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Christopher S. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Fred Hutchinson Cancer Center, University of Washington, 1100 Fairview Ave. N., Seattle, WA 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BS.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BS.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Goncalo Abecasis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Goncalo%2BAbecasis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abecasis%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGoncalo%2BAbecasis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hyun Min Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyun%2BMin%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20HM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHyun%2BMin%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Michael Boehnke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BBoehnke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boehnke%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BBoehnke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Carlos D. Bustamante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biomedical Data Science, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BD.%2BBustamante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bustamante%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BD.%2BBustamante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Christopher R. Gignoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BR.%2BGignoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gignoux%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BR.%2BGignoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9728-6567" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher R. Gignoux</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.gignoux@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">chris.gignoux@ucdenver.edu</a>
<a href="mailto:eimear.kenny@mssm.edu" class="" data-icon-position="" data-hide-link-title="0">eimear.kenny@mssm.edu</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Eimear E. Kenny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Charles Bronfman Institute for Personalized Medicine, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>The Icahn Institute of Multiscale Biology and Genomics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>***</span>The Center for Statistical Genetics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eimear%2BE.%2BKenny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kenny%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEimear%2BE.%2BKenny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9198-759X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eimear E. Kenny</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.gignoux@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">chris.gignoux@ucdenver.edu</a>
<a href="mailto:eimear.kenny@mssm.edu" class="" data-icon-position="" data-hide-link-title="0">eimear.kenny@mssm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3255" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490960" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490960/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3255.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490960" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490960/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3255.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490960" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490960/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3255.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490960" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490960/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3255.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490960" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490960/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-6">The emergence of very large cohorts in genomic research has facilitated a focus on genotype-imputation strategies to power rare variant association. These strategies have benefited from improvements in imputation methods and association tests, however little attention has been paid to ways in which array design can increase rare variant association power. Therefore, we developed a novel framework to select tag SNPs using the reference panel of 26 populations from Phase 3 of the 1000 Genomes Project. We evaluate tag SNP performance <em>via</em> mean imputed r<sup>2</sup> at untyped sites using leave-one-out internal validation and standard imputation methods, rather than pairwise linkage disequilibrium. Moving beyond pairwise metrics allows us to account for haplotype diversity across the genome for improve imputation accuracy and demonstrates population-specific biases from pairwise estimates. We also examine array design strategies that contrast multi-ethnic cohorts <em>vs.</em> single populations, and show a boost in performance for the former can be obtained by prioritizing tag SNPs that contribute information across multiple populations simultaneously. Using our framework, we demonstrate increased imputation accuracy for rare variants (frequency &lt; 1%) by 0.5–3.1% for an array of one million sites and 0.7–7.1% for an array of 500,000 sites, depending on the population. Finally, we show how recent explosive growth in non-African populations means tag SNPs capture on average 30% fewer other variants than in African populations. The unified framework presented here will enable investigators to make informed decisions for the design of new arrays, and help empower the next phase of rare variant association for global health.</p></div><span class="kwd-group-title">Key Words</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomics" class="hw-term hw-article-keyword hw-article-keyword-genomics" rel="nofollow">Genomics</a></li><li class="kwd"><a href="/keyword/statistical-genetics" class="hw-term hw-article-keyword hw-article-keyword-statistical-genetics" rel="nofollow">Statistical Genetics</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">Imputation</a></li><li class="kwd"><a href="/keyword/tag-snps" class="hw-term hw-article-keyword hw-article-keyword-tag-snps" rel="nofollow">tag SNPs</a></li><li class="kwd"><a href="/keyword/array-design" class="hw-term hw-article-keyword hw-article-keyword-array-design" rel="nofollow">array design</a></li></ul><p id="p-7">There is a growing recognition in genomic research of the need for very large-scale associations studies and genome-wide arrays are often the cost-efficient technology of choice. In this study we explore ways to improve array design for rare variant imputation, an underused means to increase power in association studies. We describe a pipeline in which an array is empirically evaluated based on genome-wide imputation accuracy, rather than pairwise linkage disequilibrium, to improve tagging and give real-world estimates of array performance. We explore the impact of patterns of demography on array performance, and discuss the trade-off between accurate rare variant imputation and <em>trans</em>-ethnic utility. This work provides a framework and insights that can guide the next generation of array development.</p><p id="p-8">The vast majority of human genomic variation is rare (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nelson <em>et al.</em> 2012</a>), and an appreciable fraction of rare variants are likely to be functionally consequential. (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kircher <em>et al.</em> 2014</a>) The gold standard approach to assay rare variation (MAF &lt; 1%) is via deep sequencing. So far, large-scale sequencing studies have had some, but limited, success for discovery of rare variant associations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Emond <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lohmueller <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">SIGMA Type 2 Diabetes Consortium <em>et al.</em> 2014</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">UK10K Consortium <em>et al.</em> 2015</a>). There is a new appreciation that studies of hundreds of thousands or millions of individuals will be needed to drive well-powered discovery efforts. (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lindquist <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kosmicki <em>et al.</em> 2016</a>) Currently, genome sequencing on this scale is prohibitively expensive and computationally burdensome. In contrast, genome-wide genotyping arrays are inexpensive, with far less bioinformatic overhead compared to sequencing. The past decade of genomic research has seen the development of myriad commercial high-throughput genotyping arrays.(<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hoffmann <em>et al.</em> 2011a</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hoffmann <em>et al.</em> 2011b</a>) While initially designed to capture common variants (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">International HapMap Consortium 2003</a>), in recent years arrays have been leveraged to capture variation at the rare end of the frequency spectrum. One strategy is to ascertain rare variants directly on arrays, which is restricted to a very narrow subset of the rare variant spectrum due to array size limits. (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Igartua <em>et al.</em> 2015</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wessel <em>et al.</em> 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McCarthy <em>et al.</em> 2017</a>) Another strategy is to leverage the haplotype structure determined by common variants on the array, which form a ’scaffold’, for accurate inference of un-genotyped variation through multi-marker imputation into sequenced reference panels of whole genomes. The strategy of genotyping, followed by imputation, has the potential to recover rare untyped variants in very large cohorts of arrayed samples at no additional experimental cost. (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2015</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Michailidou <em>et al.</em> 2015</a>) Imputation increases the effective sample size, leading to increased statistical power. (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pritchard and Przeworski 2001</a>) This model bridging genotyping and imputation has prompted efforts to build deep reference sequence databases and a renewed interest in methods for improving genome-wide scaffold design. (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">UK10K Consortium <em>et al.</em> 2015</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">McCarthy <em>et al.</em> 2016</a>).</p><p id="p-9">Genotype array scaffolds have historically been designed using algorithms that select tagging single nucleotide polymorphisms (tag SNPs) that are in linkage disequilibrium (LD) with a maximal number of other SNPs. Tag SNP algorithms are optimized to maximize this score, typically described as pairwise coverage. However, imputation tools increasingly incorporate sophisticated haplotype information to impute unobserved variants. (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Howie <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fuchsberger <em>et al.</em> 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Browning and Browning 2016</a>) Consequently, it is not clear that tag SNPs that maximize pairwise coverage will be tag SNP’s that provide, in aggregate, the best GWAS scaffold for accurate imputation. (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de Bakker <em>et al.</em> 2005</a>) Further, most tag SNP selection algorithms use LD architecture in a single population (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Weale <em>et al.</em> 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carlson <em>et al.</em> 2004</a>), while we know LD patterns can vary extensively between populations. (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>) Historically, many commercial arrays were designed by selecting tag SNPs from European populations, although arrays targeting some other populations have recently entered the market. (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hoffmann <em>et al.</em> 2011a</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hoffmann <em>et al.</em> 2011b</a>) The number of SNPs tagged by a tag SNP can vary appreciably between populations due to demographic forces of migration, population expansion, and genetic drift. This may diminish GWAS scaffold performance in populations other than those in which the tag SNPs were selected, which in turn, can lead to reduced power for imputation-based association. This is a particularly pernicious problem in populations for which no targeted commercial array is available, in studies with multi-ethnic populations, and for accurate estimation of the transferability of genetic risk across populations.</p><p id="p-10">As association studies grow larger and increasingly diverse, there is a need to reassess design criteria for GWAS scaffolds and arrays. (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carlson <em>et al.</em> 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fuchsberger <em>et al.</em> 2016</a>) On the one hand, tag SNPs that tag lower frequency variants are likely to be on the lower end of the site frequency spectrum and, consequentially, more geospatially restricted. (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nelson <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bustamante <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gravel <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mathieson and McVean 2014</a>) On the other hand, as studies grow very large, cohort heterogeneity is likely to increase substantially. (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Banda <em>et al.</em> 2015</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Marouli <em>et al.</em> 2017</a>) Given finite GWAS scaffold density, examining the trade-off between lowering the frequency threshold for accurate imputation and extending utility to multiple populations will become important. (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nelson <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Martin <em>et al.</em> 2014</a>) In this manuscript, we describe a framework for developing well-powered tag SNP selection leveraging thousands of whole genomes from diverse populations for balanced cross-population coverage. In our study, genomic coverage is evaluated based on genome-wide imputation accuracy as measured by mean imputed r<sup>2</sup> at untyped sites, rather than pairwise linkage disequilibrium. Moving beyond pairwise metrics allows us to account for haplotype diversity across the genome and demonstrates population-specific biases from pairwise estimates. Assessing accuracy using leave-one-out cross-validation yields a real-world estimate of genomic coverage. We examine the effect of allele frequency, correlation thresholds, and population diversity on the selection of tag SNP and on the landscape of tag-able variation. This work demonstrates that, while there may be limits given current reference panels, improving GWAS scaffold design is an underused means to increase power in association studies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genetic Data</h3><p id="p-11">The genetic data are from the 1000 Genomes Project (1000 Genomes) Phase 3 data release, version 2 (7/8/2014) containing whole genome sequences for 2,535 individuals from 26 global populations. (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>) Sequence data were in VCFv4.1 format, mapped to the forward strand and variants annotated as reference or alternate alleles. Only biallelic SNPs were included in this analysis (77,224,748 SNPs total). A list of known cryptically related individuals was obtained from the 1000 Genomes FTP site, and one individual from each related pair were subsequently removed (n = 62). Individuals were assigned to their super populations according to the original 1000 Genomes assignments (EAS = East Asian, EUR = European, AFR = African, SAS = South Asian, AMR = Americas, comprising 503, 501, 495, 477, and 341 individuals, respectively). Two populations of admixed African ancestry (ASW and ACB) were removed from the African super population and formed a separate African American/Caribbean (AAC) super population (n = 156).</p></div><div id="sec-3" class="subsection"><h3>Tag SNP Selection</h3><p id="p-12">Allele frequency was estimated within super population for each SNP using Plink v1.9. (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2015</a>) Linkage Disequilibrium (LD) was also calculated within each super population using Plink v1.9 and settings for pairwise linkage with a minimum r<sup>2</sup> of 0.2 within a maximum distance of 1 megabase (mb). Tag SNP selection was performed per chromosome in the program TagIT (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Weale <em>et al.</em> 2003</a>) (<a href="https://github.com/statgen/TagIt">https://github.com/statgen/TagIt</a>), with frequency and LD files for each super population as input. The TagIT algorithm analyzed each super population separately. After filtering based on the minor allele frequency (set as either 0.5%, 1% or 5%), TagIT annotates the tag SNP that has the highest number of LD pairs with r2 above a minimum threshold (set as either 0.2, 0.5, or 0.8). The selected tag SNP and all of its linked SNPs are masked and TagIT finds the next tag SNP with the highest number of LD pairs. The output for each super population included for each index tag SNP the number of sites in LD, as well as the number of unique sites that weren’t already tagged by a previously chosen tag. The number of unique SNPs tagged across all populations per tag SNP was tallied in the final output.</p></div><div id="sec-4" class="subsection"><h3>Cross-population tag SNP ranking and scoring</h3><p id="p-13">The naive approach ranked potential tags by the absolute number of unique SNPs that are tagged across all super populations. From this list, the top SNPs were selected for the appropriate allocation. To ensure performance of the tags across multiple populations, the cross-population prioritization schema first ranks tags by the number of populations in which they are informative, meaning they tag at least one site (Supplementary Figure 1). This ensures that the top ranked SNPs are not biased to a super population with large LD blocks or high SNP density in which one tag can contribute information about many other SNPs. Within each one of these categories (all 6 super populations down to only 1 super population), the tags are ranked by the number of unique tags across all six super populations, as was done in the original approach. The appropriate allocation is selected from the top of this list, scaled to the size of the chromosome of interest.</p></div><div id="sec-5" class="subsection"><h3>Metric of Performance</h3><p id="p-14">Coverage and imputation accuracy were assessed using all polymorphic biallelic sites within the 1000 Genomes Phase 3 data release, version 2. Sites were categorized into ten discrete minor allele frequency bins: (0.005-0.01], (0.01-0.02], (0.03-0.04], (0.04-0.05], (0.05-0.1], (0.1-0.2], (0.2-0.3], (0.3-0.4], and (0.4-0.5]. The term “coverage” is used to denote the proportion of untyped sites that had at least one tag SNP with pairwise r<sup>2</sup> greater than a certain threshold (0.2, 0.5, or 0.8). Imputation accuracy was determined through a leave-one-out internal validation approach with the 1000 Genomes Project Phase 3 data using a modified version of Minimac. (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fuchsberger <em>et al.</em> 2014</a>) For this approach, each individual within the 1000Genomes data had the appropriate tag SNPs denoted as ‘genotyped’, with all other sites set as missing. These missing sites are then imputed using the rest of the 1000Genomes panel as a reference. Correlation was calculated comparing the estimated dosages from this imputation to the true genotypes from the original VCF files. While this internal validation approach may introduce overfitting of the data and an upwards bias of imputation accuracy, we sought the relative imputation accuracy for different methods and do not see any bias altering described trends.</p></div><div id="sec-6" class="subsection"><h3>Ascertainment Bias Analyses</h3><p id="p-15">Population-specific tags were selected separately through TagIT for each super population with a genome-wide allocation of 500,000 sites. All tags had a minimum MAF of 1% and a minimum r<sup>2</sup> threshold of 0.5. Each of the single population ascertained tag lists assessed for imputation accuracy in all six super populations, including their index population. Imputation accuracy was calculated as previously described and limited to chromosome 9.</p></div><div id="sec-7" class="subsection"><h3>Local Ancestry</h3><p id="p-16">Local ancestry was estimated using RFMix (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Maples <em>et al.</em> 2013</a>) assuming three ancestral backgrounds: African, European, and Native American, and is described in detail in (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Martin <em>et al.</em> 2016</a>) Tracts were dropped if smaller than 20 cM to improve accuracy in local ancestry estimation. Diploid ancestry with three ancestral backgrounds yielded six categories of variation. Imputation accuracy was then calculated separately per diploid tract category, with all other sections masked out. Results were aggregated across all chromosomes to calculate the genome-wide performance per diploid ancestry. Tracts were removed from analysis if the ancestral diplotype was found in fewer than 5 individuals. This included AFR-NAT and EUR-NAT within ACB which only occurred in 2 individuals each, NAT-NAT diplotypes in ASW which occurred in one individual, and AFR-AFR diplotypes in MXL which occurred in 3 individuals.</p></div><div id="sec-8" class="subsection"><h3>Cross-population patterns of linkage disequilibrium</h3><p id="p-17">To determine how many sites were in LD with tag SNPs across all 6 super populations, we selected one million SNPs for a GWAS scaffold using a minimum r<sup>2</sup> of 0.5 and a minimum MAF of 0.01 on chromosome 9. We calculated the number of polymorphic sites (MAF &gt; 0.5%) and the proportion of these sites that were in LD (r<sup>2</sup> &gt; 0.5 or r<sup>2</sup> &gt; 0.8) with at least one tag marker. To determine sharing of tags across multiple populations, we calculated the proportion of tag markers that were informative in other populations, conditional upon them being informative in the index population. The proportion of sites shared among multiple populations was calculated as the proportion of tag SNPs that performed in a certain number of populations (from 1 to 6 super populations) per super population.</p></div><div id="sec-9" class="subsection"><h3>Tagging Potential</h3><p id="p-18">Tag SNPs were selected with a minimum r<sup>2</sup> of 0.5 and a minimum MAF of 0.01 on chromosome 9. The potential for tagging was determined assuming an infinite site scaffold, using all possible tags until every pairwise relationship with r<sup>2</sup> above 0.5 was captured. The average number of sites captured per tag was calculated in each super population separately, using only the tags that were informative within that population. We also calculated these trends assuming a scaffold of one million sites, following the same procedures. The “dark sites” were calculated as sites in which there was no pairwise correlation with any other site with r<sup>2</sup> &gt; 0.2, determined separately for each super population.</p></div><div id="sec-10" class="subsection"><h3>Data Availability</h3><p id="p-19">The input data from 1000 Genomes Project, Phase 3 is publicly available at the following link: <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/</a>. The program TagIt is available on github (<a href="https://github.com/statgen/TagIt">https://github.com/statgen/TagIt</a>), as well as a tutorial for how to select tag SNPs as detailed in this manuscript (<a href="https://github.com/chrisgene/crosspoptagging">https://github.com/chrisgene/crosspoptagging</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6626762">https://doi.org/10.25387/g3.6626762</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Assessing population-specific imputation accuracy With standard GWAS scaffold design</h3><p id="p-20">First we designed an experiment to assess imputation accuracy performance comparing tag SNP selection from different populations. This experiment mimics the current design of many commercial arrays, in which tag SNPs were selected to capture the primarily variation in a single population or a closely related group of populations. We built a pipeline using the 26 population reference panel from Phase 3 of the 1000 Genomes Project and the Tagit algorithm(Taliun) for tag SNP selection. (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Weale <em>et al.</em> 2003</a>) (Supplementary Table 1) Individuals were split into mutually exclusive “super populations.” These included the Admixed American (AMR), East Asian (EAS), European (EUR), and South Asian (SAS) populations as described in <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> (2015)</a> In addition, we divided the African super population into two groups: four populations from Africa (AFR) and two populations of African descent in the Americas (AAC) (see <strong>Methods</strong>). Initially, to mimic the design of many arrays, tag SNPs were selected from a single super population. We assumed a genome-wide allocation of 500,000 tag SNPs, however analyses for a single population tagging strategy were only conducted on chromosome 9 with the allocation of 21,107 sites proportional to the physical distance of chromosome 9 compared to all chromosomes combined. Potential tags were required to have a minor allele frequency (MAF) ≥ 1% and be in pairwise LD with the tagged target site with a r<sup>2</sup> ≥ 0.5.</p><p id="p-21">The current generation of phase-based imputation algorithms (BEAGLE, IMPUTE2, Minimac3) leverage local haplotype information and sequenced reference panels to improve accuracy of variant inference compared to tag SNP approaches. (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Marchini <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Howie <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Marchini and Howie 2010</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Fuchsberger <em>et al.</em> 2014</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Browning and Browning 2016</a>) Therefore, optimal array design depends not only on tag SNP selection, but also on empirical evaluation of imputation performance. For each of the population-specific GWAS scaffolds, imputation accuracy was assessed in all six super populations by MAF bins (common, MAF = 0.05-0.5; low frequency, MAF = 0.01-0.05; and rare, MAF &lt; 0.01) by comparing the imputed dosages to the real genotypes through leave-one-out internal validation. (see Methods).</p><p id="p-22">Consistently across all super populations, the population from which the tags were ascertained had the highest imputation accuracy in the common bin. (S1 Fig) Trends in imputation accuracy follow known patterns of demography. For example, if the tags were ascertained in European populations, imputation accuracy was best in Europeans (EUR), followed by out-of-Africa populations (AMR, SAS, EAS), and worst in African ancestry populations (AFR, AAC). (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) If the tags were ascertained in African populations, the inverse was observed. (S1 Fig) As expected, the same trend of reduced imputation accuracy in non-ascertained populations was exacerbated in the low frequency bin. Imputation of low frequency variants in East Asian populations (EAS) was consistently most challenging; even when tag SNPs were selected from EAS, accuracy of low frequency imputation was the same or better in other populations. This can be explained by evidence of a recent tight bottleneck followed by rapid population grown in EAS, resulting in a large proportion of rare variants that are difficult to tag due to lower LD, especially with a limited scaffold of 500,000 sites. (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gravel <em>et al.</em> 2011</a>) In contrast, the imputation performance of tag SNPs ascertained in AFR, AMR, and AAC populations is the same or better compared to the performance in out-of-Africa populations. This is likely due to increased allelic heterogeneity in African ancestry populations, which results in greater haplotypic diversity and a higher chance that a rare variant is well tagged by a haplotype for imputation. (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>) The imputation accuracy of AMR higher in the rare frequency bin (MAF 0.5–1%), independent of the ascertainment population, is likely due to longer haplotypes resulting from recent admixture, allowing the rare variation to be captured accurately given the limited allocation. (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gravel <em>et al.</em> 2013</a>) Importantly, in each case we observe a notable drop-off in performance across most of the frequency spectrum when examining imputation coverage in populations diverging from the one used for tag SNP selection. (S1 Fig).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Imputation Accuracy by super population of tags selected in European populations for a scaffold assuming 500,000 genome-wide variants. Tags were required to have a MAF &#x2265; 1% and r2 &#x2265; 0.5 with target sites. This trend is observed across all super populations (S1 Fig)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1155323423" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Imputation Accuracy by super population of tags selected in European populations for a scaffold assuming 500,000 genome-wide variants. Tags were required to have a MAF &#x2265; 1% and r&lt;sup&gt;2&lt;/sup&gt; &#x2265; 0.5 with target sites. This trend is observed across all super populations &lt;em&gt;(S1 Fig)&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3255/F1.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3255/F1.medium.gif" width="440" height="263"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491113" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Imputation Accuracy by super population of tags selected in European populations for a scaffold assuming 500,000 genome-wide variants. Tags were required to have a MAF ≥ 1% and r<sup>2</sup> ≥ 0.5 with target sites. This trend is observed across all super populations <em>(S1 Fig)</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparing single <em>vs.</em> cross population tag SNP selection strategies</h3><p id="p-24">When developing a genotyping platform, it is useful to assess whether selected tag SNPs segregate in the population of interest and contribute to tagging by being in LD (high r<sup>2</sup>) with untagged sites. For example, using Illumina’s OmniExpress platform (Illumina) within the 1000 Genomes Project data, over 99.7% of the sites will be polymorphic (MAF &gt; 0.5%) in the overall dataset. However, when we stratify by super population, each group has a differential loss due to monomorphic sites. AFR loses only &lt;1% of sites with a MAF &lt; 0.5%, whereas EUR and EAS lose 4.4% and 9.2% of variants, respectively. Reduction in tagging can result in loss of statistical power for downstream analysis. We quantify this as “informativeness”, or the ability of a tag SNP to both segregate in the population and provide LD information (r<sup>2</sup> &gt; 0.5 with at least one untagged site). Balancing representation of variation across all groups becomes very important in multi-ethnic studies.</p><p id="p-25">To explore different approaches for GWAS scaffold design we compared three strategies for selecting tag SNPs; single population tag SNP ascertainment, in which all tags are selected from a single population; a ‘naïve’ approach, in which all populations are combined and tags are selected based on composite statistics derived from this multi-population pool; and a ‘cross-population prioritization’ approach, in which tags are prioritized if they are both informative in multiple populations and by the number of unique sites targeted across all groups (see Methods and S2 Fig). We generated lists of tags per method assuming a total genome-wide allocation of 500,000 sites and minimum thresholds of r<sup>2</sup> &gt; 0.5 and minor allele frequency (MAF) ≥ 1%. Using these parameters, an exhaustive set of tag SNPs were selected using the naïve approach with tags ranked by the absolute number of sites tagged across the 6 super populations, regardless of how many super populations had LD between tags and targets. We then re-ranked them using the cross-population prioritization approach (S2 Fig).</p><p id="p-26">To compare the three approaches, we tallied the number of informative tags per population for each method to investigate the added value of tags contributing information in multiple populations. (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) This was done for all 22 autosomes. As per the design, all the single-population tags were informative within the super population from which tag SNPs were selected. Comparing the naïve and cross-population approaches that selected tag SNPs across all populations, the cross-population prioritization approach increased the number of informative tag SNPs in all populations relative to the naïve approach. In the naïve approach, we observed that the majority of tag SNPs were selected from the AFR population, followed by AAC, due to African-descent populations having more polymorphic sites across the genome with lower linkage disequilibrium. (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Henn <em>et al.</em> 2015</a>; <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>) Whereas in the cross-population prioritization approach variation specific to a single population is down-weighted, leading to more balanced representation between all 6 super populations. By leveraging cross-population information the largest boost in the proportion of tag SNPs contributing linkage disequilibrium information compared to the naive approach was observed in non-African descent populations (10.5%, 28.6%, 25.9%, and 28.7% in AMR, EAS, EUR and SAS, respectively). Even the African descent populations (AFR and AAC), which dominate the naïve approach, have a higher proportion of tags in linkage disequilibrium with target sites with the cross-population prioritization approach (a 2.2% and 1.0% boost for AAC and AFR, respectively).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of tags that are informative by population with the three methods. (Left, lightest) tags selected from only a single population, (Center) tags selected by pooling all populations agnostically, and (Right) tags selected with the cross-population prioritization approach. Tag SNPs were informative if they were in linkage disequilibrium (r2 &gt; 0.5) with at least one untagged site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1155323423" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of tags that are informative by population with the three methods. (Left, lightest) tags selected from only a single population, (Center) tags selected by pooling all populations agnostically, and (Right) tags selected with the cross-population prioritization approach. Tag SNPs were informative if they were in linkage disequilibrium (r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5) with at least one untagged site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3255/F2.medium.gif" width="440" height="130"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3255/F2.medium.gif" width="440" height="130"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491067" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Proportion of tags that are informative by population with the three methods. (Left, lightest) tags selected from only a single population, (Center) tags selected by pooling all populations agnostically, and (Right) tags selected with the cross-population prioritization approach. Tag SNPs were informative if they were in linkage disequilibrium (r<sup>2</sup> &gt; 0.5) with at least one untagged site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">To assess performance across the frequency spectrum we also stratified our accuracy estimates by super population-specific MAF into common, low frequency, and rare bins, as previously described. We observed that the cross-prioritization approach results in a larger proportion of tags being informative compared to both the single-population and naïve for common tag SNPs (MAF &gt; 0.05) in all super populations. This is likely because the cross-prioritization approach prioritizes potential tag SNPs that provide LD information across multiple populations, therefore prioritizing common variants tagging common variation. However, by limiting tag SNP selection to these common variants only, the proportion of tags that provide LD information for low frequency variants is decreased compared to the single population approach, which had the highest proportion of informative tag SNPs in low and rare frequency in the target population. For example, when tags were ascertained using only AAC LD information, 19.5% of the 500,000 SNP scaffold were informative for rare variation (MAF &lt; 1%) and 62.8% for common variation MAF &gt; 5%) within AAC populations. When the cross-population approach was used, ensuring the prioritization of common variation, the proportion of tag SNPs informative for rare variation dropped to 6% while the proportion informative for common variation jumped up to 82.4%. This is consistent with low frequency and rare variants being population-specific, therefore not tagged by cosmopolitan common variation present in multiple populations. A notable exception is that the naïve approach contributes the most LD information for rare variants in the AMR super population. This is consistent with our previous findings showing highest imputation accuracy in the rare variation within AMR, even when the population from which tag SNPs were ascertained was different. The AMR on average exhibit longer haplotype lengths from the recently admixed populations in the Americas. (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gravel <em>et al.</em> 2013</a>; <a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>) Because of the long haplotype tract lengths, more limited haplotypic diversity, and the limited allocation of tag SNPs, a naïve approach emphasizing the absolute number of unique sites up-weights variation that is informative for at least one of the ancestral components present in these populations.</p></div><div id="sec-14" class="subsection"><h3>Cross population prioritization of tag SNPs increases imputation accuracy for all groups Across frequency spectrum compared to naïve approach</h3><p id="p-29">The goal of tag SNP selection is to inform the unmeasured haplotypes, and therefore their performance must be evaluated in aggregate. One way to assess this is through imputation accuracy. Following the observation that cross-population prioritization selects a higher proportion of informative common tag SNPs for each population, even compared to the single population approach, we next assessed what impact this would have on imputation accuracy. We deployed the same leave-one-out internal cross validation approach as before using the 1000 Genomes Project populations (see Methods). We again assumed a genome-wide scaffold of 500,000 sites and tags had to have a MAF &gt; 1% and r<sup>2</sup> &gt; 0.5 with tagged sites. Imputation accuracy was highest across all population-specific minor allele frequency bins when ascertaining in the target population in non-African non-admixed descent continental populations (EAS, EUR, and SAS). (S3 Fig) For the two African descent groups (AAC and AFR), the cross-population prioritization approach had the highest imputation accuracy across all sites. When stratified by MAF bins, the increase in informative tag SNPs for common variants with the cross population approach yielded higher imputation accuracy for common variation in all super populations. As previously seen, the population-specific nature of low frequency and rare variants led to decreased imputation accuracy in non-African descent populations for both the cross-population and naïve approach when compared to targeted single-population ascertainment. The cross-population prioritization approach had higher imputation accuracy than the naïve approach for all MAF bins.</p><p id="p-30">As scaffold size can dramatically affect imputation accuracy(<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Spencer <em>et al.</em> 2009</a>), we additionally examined allocations of 250,000, 500,000, 1,000,000, 1,500,000, and 2,000,000 genome-wide tags, which were all selected with r<sup>2</sup> &gt; 0.5 and MAF &gt; 0.01. These allocations approximate the size range of many commercially available arrays. The cross-population prioritization scheme performed better with higher imputation accuracy than the naïve method for all super populations across all minor allele frequency bins with tags selected. (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) The biggest improvement came with the smaller array sizes. The most marked improvement was found in EAS, which originally had the lowest imputation accuracy of the 6 super populations with the naive approach. Within EAS groups, the cross-population approach increased imputation accuracy overall by 9.8% (from 67.3 to 77.1%) for a tag scaffold of 250,000 sites. For a scaffold of 500,000 sites, an overall improve of 6.2% was observed (from 77.4 to 83.6%). Improvements were largely consistent with the increase of informative tag SNPs. (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) As with the naive prioritization approach SNPs were disproportionately informative within AFR and AAC, consistent with admixed ancestry reflected by reference panels. For the smaller sizes (250K), the greatest increase in performance incorporating cross-population information was found within common SNPs (MAF &gt; 5%). However, the larger sized scaffolds (1-2 million) showed the most improvement within the low frequency bins (MAF &lt; 5%).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Increased imputation accuracy with cross-population prioritization (solid line) vs. na&#xEF;ve approach (dashed line) for a minimum pairwise correlation threshold of r2 &gt; 0.5 and MAF &gt; 1% across different scaffold sizes. Imputation accuracy was calculated separately within minor allele frequency bins for each super population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1155323423" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Increased imputation accuracy with cross-population prioritization (solid line) &lt;em&gt;vs.&lt;/em&gt; na&#xEF;ve approach (dashed line) for a minimum pairwise correlation threshold of r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5 and MAF &gt; 1% across different scaffold sizes. Imputation accuracy was calculated separately within minor allele frequency bins for each super population.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3255/F3.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3255/F3.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491096" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Increased imputation accuracy with cross-population prioritization (solid line) <em>vs.</em> naïve approach (dashed line) for a minimum pairwise correlation threshold of r<sup>2</sup> &gt; 0.5 and MAF &gt; 1% across different scaffold sizes. Imputation accuracy was calculated separately within minor allele frequency bins for each super population.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Imputation accuracy varies by local ancestry background in admixed individuals</h3><p id="p-32">We also assessed imputation ancestry stratified by local ancestry diplotype in the two admixed populations, the AAC and AMR, for a genome-wide allocation of 500,000 tag SNPs. First, using phased data, we inferred haploid tracts of African, European, and Native American local ancestry along the genomes of all individuals in the AMR and AAC populations (see Methods, (<a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Martin <em>et al.</em> 2016</a>)). Then each variant was inferred to be on one of six ancestral diploid tracts; European-European (EUR-EUR), European-African (EUR-AFR), European-Native American (EUR-NAT), African-Native American (AFR-NAT), African-African (AFR-AFR) and Native American-Native American (NAT-NAT). In all local ancestry strata the cross-population prioritization yielded improved imputation accuracy when compared to the naïve approach. When looking at ASW population (Americans of African ancestry in South West US), performance was high overall with all diploid tracts having imputation accuracies of 92.8–96.8% for all sites with minor allele frequency above 1%. (S4 Figure) The lowest imputation accuracy was found in AFR-AFR tracts, especially at the lower end of the frequency spectrum. The highest imputation accuracy was found in EUR-EUR tracts (94% overall for ASW). In AMR populations, by contrast, the NAT-NAT tracts had the lowest performance of all. An example can be seen in the MXL population (Mexican Ancestry from Los Angeles), where the highest imputation accuracy was found in the AFR-EUR tracts (overall imputation accuracy of 90.1% for all SNPs with MAF &gt; 0.5%) and the lowest within NAT-NAT tracts (74.8% for all SNPS with MAF &gt; 0.5%). (S4B Fig) These performances could be reflective of the relative availability of reference data relevant to these specific ancestral components.</p></div><div id="sec-16" class="subsection"><h3>Evaluating impact of r<sup>2</sup> and MAF thresholds on tag SNP performance</h3><p id="p-33">Previous standards in scaffold design have considered minimum linkage disequilibrium (r<sup>2</sup>) and minor allele frequency (MAF) thresholds when prioritizing possible tag SNPs. However, the impact of these thresholds are often evaluated through pairwise coverage. We explored varying the minimum r<sup>2</sup> threshold (0.2, 0.5, 0.8) and MAF (0.5%, 1%, 5%) to assess their impacts on imputation accuracy, as well as pairwise coverage, assuming a genome-wide allocation of one million tags. For common variants, a higher minimum r<sup>2</sup> threshold (r<sup>2</sup> &gt; 0.8) resulted in slightly higher imputation accuracy. (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) However, the sites in the low and rare bin demonstrate population-specific accuracy only. (S5 Fig) For AFR, SAS, and EAS, a less stringent threshold of r<sup>2</sup> &gt; 0.2 had the worst imputation accuracy across all frequency bins. Low frequency and rare variation had higher imputation accuracy for an r<sup>2</sup> threshold of 0.5 compared to 0.8. Within AAC, AMR, and EUR, the low frequency variation had improved imputation accuracy with the lowest r<sup>2</sup> threshold of 0.2. However, the imputation accuracy within this low threshold was notably compromised for common variants. This indicates that low frequency variation is better captured by weak correlation structure, but at a cost to common variation in these populations. Analyses performed with r<sup>2</sup> &gt; 0.5 had the best balance of performance across all frequency bins with the highest overall imputation accuracy in all super populations except for EAS. (S2 Table) Overall, there were very small differences in imputation accuracy between the different r<sup>2</sup> thresholds. There were much larger differences in coverage, including both coverage evaluated with minimum r<sup>2</sup> (LD) of 0.5 and 0.8. (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>) Additionally, the best “performance” using pairwise coverage was highly dependent on the definition of coverage. Specifically, if pairwise coverage was calculated as the proportion of sites that are in LD with r<sup>2</sup> &gt; 0.5, then the best minimum r<sup>2</sup> threshold in tag SNP selection will be 0.5. This holds true for r<sup>2</sup> &gt; 0.8 as well.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Influence of (A) minimum r2 threshold and (B) lower MAF threshold on imputation accuracy and coverage (r2 &gt; 0.5 and r2 &gt; 0.8) within populations from the Americas with an allocation of 1M sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1155323423" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Influence of (A) minimum r&lt;sup&gt;2&lt;/sup&gt; threshold and (B) lower MAF threshold on imputation accuracy and coverage (r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5 and r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.8) within populations from the Americas with an allocation of 1M sites.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3255/F4.medium.gif" width="440" height="180"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3255/F4.medium.gif" width="440" height="180"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491039" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Influence of (A) minimum r<sup>2</sup> threshold and (B) lower MAF threshold on imputation accuracy and coverage (r<sup>2</sup> &gt; 0.5 and r<sup>2</sup> &gt; 0.8) within populations from the Americas with an allocation of 1M sites.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">The impact of minimum minor allele frequency threshold was negligible across variants with MAF &gt; 5% for all non-African populations (S6 Fig). Within populations of African descent, limiting tags to variants with MAF &gt; 5% resulted in increased imputation accuracy for all frequency bins, especially for common variants. Lowering the MAF to 0.5% reduced accuracy in African-descent populations across all frequency bins. For EUR, SAS, and AMR, tags with MAF &gt; 1% had decreased accuracy for variants with MAF 0.5–1% compared to when tags are limited to MAF &gt; 0.5%. (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>) The lowest limit of MAF (0.5%) showed increased accuracy for rare variation but at a slight cost to the accuracy for common sites (MAF &gt; 5%). We concluded that the best balance for tag SNP selection across all populations among these was MAF &gt; 1% within the population being tagged, as the imputation accuracy was best for MAF &gt; 5% for half of the groups (AAC, AFR, EAS) and best for MAF &gt; 0.5% for the other half (AMR, EUR, SAS). (S2 Table) However, the overall differences in imputation accuracy was minimal, with less than 1% between all lower MAF thresholds across all sites. Again, we observed large differences in pairwise coverage, despite negligible differences when performance is evaluated by imputation accuracy. (S6 Fig) This is particularly striking for African-descent populations (ASW and AFR), where there were large gains of pairwise coverage for MAF &gt; 1%, compared to MAF &gt; 0.5% and MAF &gt; 5%. As previously described, African populations have shorter LD blocks and a greater absolute number of polymorphic variants compared to other populations. (<a id="xref-ref-1-9" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>) Therefore, pairwise coverage underestimates performance compared to imputation accuracy, as addressed below.</p></div><div id="sec-17" class="subsection"><h3>Tagging potential differs between populations</h3><p id="p-36">Efficient tag SNP selection is an opportunity to boost power in downstream analyses. In our study, African and out-of-Africa populations exhibited distinct genetic architectures, which resulted in different performance trends. Even when cross-population performance was prioritized, it did not guarantee equal representation of all population groups within the tag SNP set. To determine the contribution of each population, we focused on chromosome 9 (42,215 tags), equivalent to one million sites genome-wide, selected with our novel cross-population prioritization scheme. This tag SNP allocation resulted in including all tags that were informative in at least 3 to all 6 populations in the scaffold. Out of all tags for chromosome 9, 17.96% were informative in all 6 populations. (S3 Table) No tags were included that were informative in only one or two populations. Of tags that were informative in 5 out of the 6 super-populations, only 54% were in LD with any target sites within EAS populations, while 93% were informative in AAC populations. (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>) This trend is consistent with cross-population tags tending to be less informative in EAS populations compared to the other populations. When tags are informative in 3 out of 6 groups, only 18% were informative in EAS, while 75% were informative in AAC. Tags informative in only 2 of the 6 groups were likely informative in AAC and AFR, the African descent populations, while very few of them were informative for non-African descent groups, consistent with capturing differential LD patterns in African populations.(<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Henn <em>et al.</em> 2011</a>) When tags are stratified by MAF (0.5–1%, 1–5%, and &gt;5%), these trends are exaggerated in the low frequency and rare MAF bins. (S7 Fig) As expected, the rare variation (0.5–1% MAF) was highly population-specific with no sites in this frequency bin being informative across all populations, or even 5 out of the 6 populations. (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Gravel <em>et al.</em> 2011</a>) For low frequency variation (1–5%), tags were the least informative within EAS, with only 36% of the tags informative in 5 out of 6 populations.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tag SNPs informativeness across population. (A) Proportion of sites informative (r2 &gt; 0.5, MAF &gt; 0.01, 1M site scaffold) across a number of populations, with lines corresponding to the index population. For example, for sites that are informative (r2 &gt; 0.5 with any untyped SNP in genome) in five out of the six populations, only slightly more than half are informative in East Asian populations while greater than 90% are informative in African populations. (B) Proportion of sites shared across populations, conditional on index population. For example, for sites informative in African populations, less than half are informative in East Asian, European, and South Asian populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1155323423" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Tag SNPs informativeness across population. (A) Proportion of sites informative (r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5, MAF &gt; 0.01, 1M site scaffold) across a number of populations, with lines corresponding to the index population. For example, for sites that are informative (r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.5 with any untyped SNP in genome) in five out of the six populations, only slightly more than half are informative in East Asian populations while greater than 90% are informative in African populations. (B) Proportion of sites shared across populations, conditional on index population. For example, for sites informative in African populations, less than half are informative in East Asian, European, and South Asian populations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3255/F5.medium.gif" width="440" height="141"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/10/3255/F5.medium.gif" width="440" height="141"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491156" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Tag SNPs informativeness across population. (A) Proportion of sites informative (r<sup>2</sup> &gt; 0.5, MAF &gt; 0.01, 1M site scaffold) across a number of populations, with lines corresponding to the index population. For example, for sites that are informative (r<sup>2</sup> &gt; 0.5 with any untyped SNP in genome) in five out of the six populations, only slightly more than half are informative in East Asian populations while greater than 90% are informative in African populations. (B) Proportion of sites shared across populations, conditional on index population. For example, for sites informative in African populations, less than half are informative in East Asian, European, and South Asian populations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Conditional performance, or the ability of a tag which is informative in the index population also being informative in an additional population, was also examined and found to be consistent with known population histories. Of tags that are informative within AFR, 94% were informative within AAC, while only 38% were informative within EAS. (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>) However, among tags that were informative within EAS, 81% were informative within African populations. Once again, the stratified analyses show exaggerated trends for the low frequency and rare MAF bins. (S8 Fig) For the rare variation (0.5–1%), only a very small percentage (&lt;10%) of tags are informative in other populations (AMR, EAS, EUR, SAS) if they were informative within African-descent populations (AFR and AAC). The high level of sharing between AFR and AAC is expected due to the high proportion of African ancestry within African-American and Afro-Caribbean populations. Of tags informative within EUR, 78% are also informative within AMR, largely due to the high proportion of European ancestry within some Hispanic/Latino populations.(<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Moreno-Estrada <em>et al.</em> 2013</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gravel <em>et al.</em> 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Moreno-Estrada <em>et al.</em> 2014</a>).</p><p id="p-39">The tags were also not equally informative in each population when it comes to the number of sites they tag with r<sup>2</sup> &gt; 0.5. For chromosome 9, it would take 81,416 tags to capture all possible tag-able variation with an r<sup>2</sup> &gt; 0.5 within AFR populations, while it would take only 28,473 tags within EAS populations to saturate coverage. However, each tag within the AFR populations captures on average 7.17 other sites, whereas for EAS populations, each tag captures on average 10.27 other SNPs. When restricting the design to a million tag SNP scaffold, each tag captures on average 16.16 other SNPs within EAS populations and 12.16 other SNPs in AFR populations. (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) This reflects the different underlying genetic architecture of these different groups.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491053/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491053/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Performance per tag SNP to capture all variation possible with r<sup>2</sup> &gt; 0.8 on chromosome 9, as well as within a one million site genome-wide scaffold allocation through cross-population prioritization</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Limits of tagging and imputation</h3><p id="p-40">Not all of the human genome can be captured through pairwise tagging given existing reference panels. For each super population, we filtered for sites that were polymorphic (MAF &gt; 0.5%) and had no pairwise correlation (r<sup>2</sup> &gt; 0.2) with any other site within one megabase. The number of these “lone sites” without any pairwise correlation was dependent upon population. AAC had the greatest number of lone sites, but that is likely due to the significantly decreased sample size compared to the other populations. (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) The lowest number of lone sites was found within AMR. Although these sites have no notable pairwise correlation with any other site in the human genome, haplotypes may be informative and allow the recovery of information for imputation. We again assumed a one million genome-wide tag SNP scaffold allocation with minimum MAF of 1% and minimum r<sup>2</sup> threshold of 0.5 and imputed to the entire 1000 Genomes reference panel. As expected, imputation accuracy and ability to recover information was population-specific. The imputation accuracy within AAC was an outlier when compared to other populations, with 80.72% of lone sites being imputed with at least the accuracy of r<sub>acc</sub><sup>2</sup> ≥ 0.5 and over 50% of sites being imputed with even higher accuracy (r<sub>acc</sub><sup>2</sup> ≥ 0.8). Many of these lone sites within AAC were captured with pairwise and haplotype LD within other populations, primarily AFR and to a lesser extent EUR. While there were likely insufficient allele counts for accurate correlation estimation within AAC due to the small sample size, this information could be recovered using a global reference panel. The number of unrecoverable “dark sites”, which had no pairwise correlation and were not recoverable with imputation using haplotype information, was the largest in EAS and is consistent with known demography and population history yielding an excess of highly rare variation compared to other populations.(<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Gravel <em>et al.</em> 2011</a>)</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491089/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491089/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Lone sites by super population and their imputation accuracy for a one million site scaffold</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Pairwise coverage <em>vs.</em> imputation accuracy</h3><p id="p-41">When evaluating the performance of a GWAS scaffold, there are numerous factors to take into consideration. These include the number of sites you have allocated to tag SNPs and what your priorities are for balanced representation. To a lesser extent, the benefits and pitfalls of prioritizing low-frequency variants must be weighed. However, we have demonstrated that the influence of these factors is highly dependent on how performance is measured. The notion of genomic “coverage” has historically been estimated using pairwise correlations, and therefore this term will be used to denote the proportion of polymorphic sites that are in pairwise LD (r<sup>2</sup> threshold) with at least one tag SNP. We calculated coverage separately per super population at an r<sup>2</sup> threshold of 0.5 and 0.8 within minor allele frequency bins identical to the imputation accuracy estimation analyses, assuming a genome-wide tag SNP set of 500,000 and 1,000,000. (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) For a tag SNP set of one million sites, coverage was lowest in AFR with an overall average of 59.15% for all sites with MAF &gt; 0.5% and r<sup>2</sup> &gt; 0.5. (S9 Fig) When the r<sup>2</sup> threshold is raised to 0.8, the proportion of sites in linkage disequilibrium with at least one tag SNP lowers to 28%. (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) The highest coverage was found in populations from the Americas (AMR) and East Asia (EAS). For a lower r<sup>2</sup> threshold of 0.5, 79.9% of AMR sites with MAF &gt; 0.5% were covered. When using the higher r<sup>2</sup> threshold of 0.8, East Asian populations had the highest coverage with 63.08% of sites in LD with at least one tag SNP. This difference is even more marked when looking at a smaller tag SNP set of 500,000 sites. (S10 Fig, S11 Fig) African populations now have an overall coverage of 33.17% with r<sup>2</sup> &gt; 0.5 and 14.10% with r<sup>2</sup> &gt; 0.8. East Asian populations have the highest coverage with 73.16% of sites covered with r<sup>2</sup> &gt; 0.5 and 55.09% with r<sup>2</sup> &gt; 0.8.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491162/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491162/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Coverage of 1 million and 500,000 tag SNP set by super population for all polymorphic sites on chromosome 9 with MAF &gt; 0.5%</span><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coverage (dashed lines) vs. Imputation Accuracy (solid lines), assuming a genome-wide scaffold size of one million tags. Coverage is shown with an r2 &gt; 0.8. While pairwise tagging values are low, particularly in African-descent populations, multi-marker imputation accuracy remains high across groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1155323423" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Coverage (dashed lines) &lt;em&gt;vs.&lt;/em&gt; Imputation Accuracy (solid lines), assuming a genome-wide scaffold size of one million tags. Coverage is shown with an r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.8. While pairwise tagging values are low, particularly in African-descent populations, multi-marker imputation accuracy remains high across groups.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3255/F6.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/10/3255/F6.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3255/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491094" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-42" class="first-child">Coverage (dashed lines) <em>vs.</em> Imputation Accuracy (solid lines), assuming a genome-wide scaffold size of one million tags. Coverage is shown with an r<sup>2</sup> &gt; 0.8. While pairwise tagging values are low, particularly in African-descent populations, multi-marker imputation accuracy remains high across groups.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">These trends are in striking contrast to those we observed in imputation accuracy. When comparing a tag SNP set of 1 million, pairwise LD coverage is the lowest in populations of African descent (59% with r<sup>2</sup> &gt; 0.5) yet imputation’s ability to recover un-typed sites is on average high and consistent with other populations (imputation accuracy of 89.62%) among SNPs with a minor allele frequency above 0.5%. This contrast is also found in East Asian populations, which had one of the highest proportion of polymorphic SNPs with r<sup>2</sup> &gt; 0.5 for coverage (76.95%), but the lowest imputation accuracy (86.28%). (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>) When sites are stratified by minor allele frequency bins, the differences in trends are even more striking. (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, S9 Fig) For example, within the lowest frequency bin (0.5–1%) for admixed populations of African-descent, the coverage of sites for a set of 500,000 tag SNPs with r<sup>2</sup> &gt; 0.8 falls below 10%, however the imputation accuracy remains relatively high at 77.82%. These trends are consistent and more dramatic when evaluated within a tag SNP set of 500,000 sites. (S10 Fig, S11 Fig) These observations reinforce the necessity of examining imputation accuracy, instead of pairwise coverage, when evaluating the performance of tag SNPs.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-44">As genomic researchers shift their focus to rare variant association in large and increasingly heterogeneous populations, it is important to design arrays with this ultimate goal in mind. There are currently two accepted methods of evaluating the performance of a tag SNPs: pairwise LD “coverage” and imputation accuracy. Coverage has historically been used as a term to denote the proportion of polymorphic sites that are in linkage disequilibrium with at least one tag marker above a certain r<sup>2</sup> threshold. (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett and Cardon 2006</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pe’er <em>et al.</em> 2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Li <em>et al.</em> 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bhangale <em>et al.</em> 2008</a>) Genotyping arrays are typically compared using this score averaged across the genome. However, as we and others have demonstrated, restricting performance assessment to this definition of pairwise coverage is limited by removing multi-marker information. (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Nelson <em>et al.</em> 2013</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Martin <em>et al.</em> 2014</a>) Evaluating imputation accuracy, particularly via leave-one-out cross validation, is highly computationally intensive, but provides a better assessment of how well untyped variation can be recaptured and a more realistic depiction of array performance than pairwise coverage. Imputation accuracy is also a more useful statistic in a practical sense, especially with the development of deeper and more diverse reference panels, (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Prüfer <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gurdasani <em>et al.</em> 2015</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sudlow <em>et al.</em> 2015</a>; <a id="xref-ref-1-10" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2015</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">McCarthy <em>et al.</em> 2016</a>) as performing GWAS with imputed variants is now the expectation. Emerging evidence suggests that rare variants (MAF &lt; 1%) that are poorly tagged by an individual tag SNP will be accessible via imputation, due to added haplotype information, particularly as sample sizes move beyond the thousands into the tens or hundreds of thousands. (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Nelson <em>et al.</em> 2013</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Fuchsberger <em>et al.</em> 2014</a>).</p><p id="p-45">Previous tagging strategies have predominantly focused on optimizing performance in a single population. In prioritizing potential tags by their ability to provide linkage disequilibrium information across multiple populations, we were able to demonstrate that cross population tag SNP selection outperforms single population selection. This boost in imputation accuracy exists across all populations and frequency bins. We simulated tag SNP sets for a range of sizes (250,000-2 million), as well as for several minimum minor allele frequencies (0.5%, 1%, 5%) and minimum r<sup>2</sup> thresholds (0.2, 0.5, 0.8). For investigators with limited real estate or budget for tag SNP selection, we found that the biggest improvement in imputation accuracy provided with our cross population approach was with the smaller array sizes (250,000) when compared to a naïve design or biased population ascertainment. As expected, the influence of MAF and r<sup>2</sup> threshold was population-specific. For African-descent populations, including tag SNPs with a low threshold of r<sup>2</sup> ≥ 0.2 resulted in lower imputation accuracy across all bins, while in other populations (EUR, AMR, SAS) tags at r<sup>2</sup> ≥ 0.2 led to increased imputation accuracy for low frequency variants to the detriment of common variation. This is due to the lower LD patterns overall in African haplotypes, requiring denser coverage. The best balance was found with a moderate r<sup>2</sup> threshold of ≥ 0.5 for those seeking to perform well across all populations. This compromise is also present in choosing the lower MAF threshold. Limiting tag SNP selection to common variants with MAF ≥ 5% produced the highest imputation accuracy across all frequency bins within African-descent populations. However, this threshold decreased imputation accuracy for low frequency and rare variants in all other populations. Therefore, the best balance is once again found in the moderate value of MAF ≥ 1%. Investigators will need to take their priorities into account when selecting the correct thresholds for their populations and if they have a specific target frequency bin. We chose to prioritize all populations equally to provide a design of broad global utility, which was adopted to construct the GWAS scaffold for Illumina Infinium Multi-Ethnic Global Arrays (Illumina) and Global Screening Arrays (Illumina). If a study is comprised of mostly one ancestral group, then the investigators should choose the appropriate thresholds tailored for their study.</p><p id="p-46">Consistent with demographic history, the potential to capture variation with a limited allocation is unequal between the different populations in the 1000 Genomes Project. The naïve tagging approach will bias tag SNP selection to be primarily informative within African-descent populations. The absolute number of polymorphic sites within African populations is much larger than other populations, and while LD tends to be lower than in other populations, the high number of potential tags and pairwise correlations overwhelms the other populations’ contributions without controlling for this unique pattern. By prioritizing potential tags that provide information across all populations, the population-level contributions are more balanced without detriment to the African-descent groups (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). The absolute number of rare variants (MAF &lt; 1%) is larger in African populations, but the frequency spectrum is more skewed toward rare variants in populations with recent bottlenecks and exponential population expansion, such as in East Asians. Contrasting these two populations (AFR and EAS), East Asian populations require fewer sites to saturate coverage, with each potential tag being in LD with more sites. However, far more polymorphic sites across the genome cannot be captured with either pairwise linkage disequilibrium or through haplotype information with imputation accuracy within these populations due to a dearth of LD information. This is amplified by the lack of comprehensive reference panels for many populations, such as East and South Asia. As reference panels are expanded, more variation will be captured to inform tag SNP selection and imputation accuracy, and we expect imputation accuracy to improve for all populations and across the frequency spectrum. (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Fuchsberger <em>et al.</em> 2014</a>).</p><p id="p-47">The power to identify relevant disease loci is inherently constrained by sample size and genome coverage. It is important to note that algorithmic development both on association testing and imputation methods have been a productive avenue of research since GWAS began, with new methods providing incremental improvements in statistical power. Here, we demonstrate a complementary strategy to improve statistical power by designing arrays optimized for imputation accuracy. Also, as cosmopolitan biobanks and large-scale multi-ethnic epidemiological studies become more commonplace, it will be important to have available platforms with built <em>in trans</em>-ethnic utility. As global reference panels become deeper and more diverse, more variation will be available for array design. The unified framework presented here will enable investigators to make informed decisions in the development and selection of GWAS scaffolds for future large-scale multi-ethnic studies. This increased representation of multi-ethnic genetic variation will promote the investigation of the genetics of complex disease and the improvement of global health in the next phase of GWAS.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">Research reported in this paper was supported by the Office of Research Infrastructure under award number S10OD018522 and the National Human Genome Research Institute under award numbers U01HG007376, U01HG007417, U01HG007419, U01HG009080 and R01HG000376 of the National Institutes of Health. CRG was supported partially by T32HG00044. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-49">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6626762">https://doi.org/10.25387/g3.6626762</a>.</p></li><li class="fn" id="fn-6"><p id="p-50"><em>Communicating editor: R. Hernandez</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-06-14"><span class="received-label">Received </span>June 14, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-08-03"><span class="accepted-label">Accepted </span>August 3, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Wojcik <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-5">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.1" data-doi="10.1038/nature15393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">A.</span>  <span class="cit-name-given-names">Auton</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A global reference for human genetic variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature15393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432245&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.2" data-doi="10.1534/genetics.115.178616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvale</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hesselson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranatunga</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characterizing Race/Ethnicity and Genetic Ancestry for 100,000 Subjects in the Genetic Epidemiology Research on Adult Health and Aging (GERA) Cohort.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1285</span>–<span class="cit-lpage">1295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178616%26rft_id%253Dinfo%253Apmid%252F26092716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEyODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.3" data-doi="10.1038/ng1801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evaluating coverage of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1801</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBarrett%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D659%26rft.epage%253D662%26rft.atitle%253DEvaluating%2Bcoverage%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1801%26rft_id%253Dinfo%253Apmid%252F16715099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16715099&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237954800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.4" data-doi="10.1038/ng.180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhangale</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickerson</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Estimating coverage and power for genetic association studies using near-complete variation data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">843</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBhangale%26rft.auinit1%253DT.%2BR.%26rft.volume%253D40%26rft.issue%253D7%26rft.spage%253D841%26rft.epage%253D843%26rft.atitle%253DEstimating%2Bcoverage%2Band%2Bpower%2Bfor%2Bgenetic%2Bassociation%2Bstudies%2Busing%2Bnear-complete%2Bvariation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.180%26rft_id%253Dinfo%253Apmid%252F18568023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18568023&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.5" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotype Imputation with Millions of Reference Samples.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.11.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.6" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/521987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.7" data-doi="10.1038/475163a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burchard</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De La Vega</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomics for the world.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/475163a</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBustamante%26rft.auinit1%253DC.%2BD.%26rft.volume%253D475%26rft.issue%253D7355%26rft.spage%253D163%26rft.epage%253D165%26rft.atitle%253DGenomics%2Bfor%2Bthe%2Bworld.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F475163a%26rft_id%253Dinfo%253Apmid%252F21753830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/475163a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21753830&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292690500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.8" data-doi="10.1086/381000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberle</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieder</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/381000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DCarlson%26rft.auinit1%253DC.%2BS.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D106%26rft.epage%253D120%26rft.atitle%253DSelecting%2Ba%2Bmaximally%2Binformative%2Bset%2Bof%2Bsingle-nucleotide%2Bpolymorphisms%2Bfor%2Bassociation%2Banalyses%2Busing%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F381000%26rft_id%253Dinfo%253Apmid%252F14681826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/381000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14681826&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187723500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.9" data-doi="10.1371/journal.pbio.1001661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matise</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">North</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haiman</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fesinmeyer</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Generalization and dilution of association results from European GWAS in populations of non-European ancestry: the PAGE study.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001661</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001661%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001661%26rft_id%253Dinfo%253Apmid%252F24068893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24068893&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.10" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vattikuti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">GigaSci</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13742-015-0047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaSci%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.11" data-doi="10.1038/ng1669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Bakker</span>  <span class="cit-name-given-names">P. I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yelensky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Efficiency and power in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1669</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253Dde%2BBakker%26rft.auinit1%253DP.%2BI.%26rft.volume%253D37%26rft.issue%253D11%26rft.spage%253D1217%26rft.epage%253D1223%26rft.atitle%253DEfficiency%2Band%2Bpower%2Bin%2Bgenetic%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1669%26rft_id%253Dinfo%253Apmid%252F16244653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16244653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233045200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.12" data-doi="10.1038/ng.2344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emond</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathias</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Exome sequencing of extreme phenotypes identifies DCTN4 as a modifier of chronic Pseudomonas aeruginosa infection in cystic fibrosis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">886</span>–<span class="cit-lpage">889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DEmond%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D886%26rft.epage%253D889%26rft.atitle%253DExome%2Bsequencing%2Bof%2Bextreme%2Bphenotypes%2Bidentifies%2BDCTN4%2Bas%2Ba%2Bmodifier%2Bof%2Bchronic%2BPseudomonas%2Baeruginosa%2Binfection%2Bin%2Bcystic%2Bfibrosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2344%26rft_id%253Dinfo%253Apmid%252F22772370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22772370&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> minimac2: faster genotype imputation. Bioinformatics.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.14" data-doi="10.1038/nature18642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flannick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teslovich</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic architecture of type 2 diabetes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">47</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature18642</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature18642%26rft_id%253Dinfo%253Apmid%252F27398621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature18642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27398621&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.15" data-doi="10.1073/pnas.1019276108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Demographic history and rare allele sharing among human populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">11983</span>–<span class="cit-lpage">11988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1019276108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019276108%26rft_id%253Dinfo%253Apmid%252F21730125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI5LzExOTgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3255.16" data-doi="10.1371/journal.pgen.1004023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrnes</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muzzio</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstructing Native American Migrations from Whole-Genome and Whole-Exome Data.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">9</span>: <span class="cit-fpage">e1004023</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1004023%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004023%26rft_id%253Dinfo%253Apmid%252F24385924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24385924&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.17" data-doi="10.1038/nature13997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurdasani</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carstensen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekola-Ayele</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pagani</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tachmazidou</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The African Genome Variation Project shapes medical genetics in Africa.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">517</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13997</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D517%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13997%26rft_id%253Dinfo%253Apmid%252F25470054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13997&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25470054&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.18" data-doi="10.1038/nrg3931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botigué</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Estimating the mutation load in human genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3931</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3931%26rft_id%253Dinfo%253Apmid%252F25963372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25963372&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.19" data-doi="10.1073/pnas.1017511108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jobin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granka</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macpherson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hunter-gatherer genomic diversity suggests a southern African origin for modern humans.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">5154</span>–<span class="cit-lpage">5162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1017511108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017511108%26rft_id%253Dinfo%253Apmid%252F21383195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzEzLzUxNTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.20" data-doi="10.1016/j.ygeno.2011.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvale</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hesselson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Next generation genome-wide association tool: Design and coverage of a high-throughput European-optimized SNP array.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ygeno.2011.04.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DHoffmann%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D79%26rft.epage%253D89%26rft.atitle%253DNext%2Bgeneration%2Bgenome-wide%2Bassociation%2Btool%253A%2Bdesign%2Band%2Bcoverage%2Bof%2Ba%2Bhigh-throughput%2BEuropean-optimized%2BSNP%2Barray.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2011.04.005%26rft_id%253Dinfo%253Apmid%252F21565264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2011.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565264&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293682700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.21" data-doi="10.1016/j.ygeno.2011.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvale</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hesselson</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gollub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Design and coverage of high throughput genotyping arrays optimized for individuals of East Asian, African American, and Latino race/ethnicity using imputation and a novel hybrid SNP selection algorithm.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">430</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ygeno.2011.08.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DHoffmann%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D98%26rft.issue%253D6%26rft.spage%253D422%26rft.epage%253D430%26rft.atitle%253DDesign%2Band%2Bcoverage%2Bof%2Bhigh%2Bthroughput%2Bgenotyping%2Barrays%2Boptimized%2Bfor%2Bindividuals%2Bof%2BEast%2BAsian%252C%2BAfrican%2BAmerican%252C%2Band%2BLatino%2Brace%252Fethnicity%2Busing%2Bimputation%2Band%2Ba%2Bnovel%2Bhybrid%2BSNP%2Bselection%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2011.08.007%26rft_id%253Dinfo%253Apmid%252F21903159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2011.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903159&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297664700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.22" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.23" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000529</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.24" data-doi="10.1038/ncomms9111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Memari</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improved imputation of low-frequency and rare variants using the UK10K haplotype reference panel.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms9111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D8111%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9111%26rft_id%253Dinfo%253Apmid%252F26368830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26368830&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.25" data-doi="10.1038/ncomms6965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Igartua</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathias</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pino-Yanes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ethnic-specific associations of rare and low-frequency DNA sequence variants with asthma.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">5965</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6965</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D5965%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6965%26rft_id%253Dinfo%253Apmid%252F25591454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6965&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25591454&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.10.3255.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Illumina Infinium Global Screening Array Information Sheet</span></li></ol><cite>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.10.3255.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Illumina Infinium Multi-Ethnic Global BeadChip Information Sheet</span></li></ol><cite>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.10.3255.28"><div class="cit-metadata unstructured">Illumina Infinium OmniExpress-24 v1.2 BeadChip Information Sheet.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.29" data-doi="10.1038/nature02168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The International HapMap Project.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">426</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02168</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D426%26rft.issue%253D6968%26rft.spage%253D789%26rft.epage%253D796%26rft.atitle%253DThe%2BInternational%2BHapMap%2BProject.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02168%26rft_id%253Dinfo%253Apmid%252F14685227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14685227&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187342000046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.30" data-doi="10.1038/ng.2892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A general framework for estimating the relative pathogenicity of human genetic variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2892%26rft_id%253Dinfo%253Apmid%252F24487276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487276&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.31" data-doi="10.1007/s00439-016-1679-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosmicki</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchhouse</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivas</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Discovery of rare variants for complex phenotypes.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">634</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00439-016-1679-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenet.%26rft.volume%253D135%26rft.spage%253D625%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-016-1679-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-016-1679-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.32" data-doi="10.1038/sj.ejhg.5202007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evaluation of coverage variation of SNP chips for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.ejhg.5202007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D635%26rft.epage%253D643%26rft.atitle%253DEvaluation%2Bof%2Bcoverage%2Bvariation%2Bof%2BSNP%2Bchips%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.ejhg.5202007%26rft_id%253Dinfo%253Apmid%252F18253166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.ejhg.5202007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18253166&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255260500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.33" data-doi="10.1002/gepi.21724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindquist</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgenson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The impact of improved microarray coverage and larger sample sizes on future genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21724</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D37%26rft.spage%253D383%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21724%26rft_id%253Dinfo%253Apmid%252F23529720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23529720&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.34" data-doi="10.1016/j.ajhg.2013.11.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sparsø</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-exome sequencing of 2,000 Danish individuals and the role of rare coding variants in type 2 diabetes.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1086</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.11.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D1072%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.11.005%26rft_id%253Dinfo%253Apmid%252F24290377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24290377&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.35" data-doi="10.1016/j.ajhg.2013.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maples</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">278</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2013.06.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D278%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.06.020%26rft_id%253Dinfo%253Apmid%252F23910464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23910464&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.36" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2796</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.37" data-doi="10.1038/ng2088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new multipoint method for genome-wide association studies by imputation of genotypes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">906</span>–<span class="cit-lpage">913</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng2088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D906%26rft.epage%253D913%26rft.atitle%253DA%2Bnew%2Bmultipoint%2Bmethod%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bby%2Bimputation%2Bof%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2088%26rft_id%253Dinfo%253Apmid%252F17572673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572673&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247619800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.38" data-doi="10.1038/nature21039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marouli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina-Gomez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Rare and low-frequency coding variants alter human adult height.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">542</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature21039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D542%26rft.spage%253D186%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature21039%26rft_id%253Dinfo%253Apmid%252F28146470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature21039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28146470&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3255.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wojcik</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-comment">Human demographic history impacts genetic risk prediction across diverse populations</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2017.03.004</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tse</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Imputation-based assessment of next generation rare exome variant arrays.</span> <abbr class="cit-jnl-abbrev">Pac. Symp. Biocomput.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPac.%2BSymp.%2BBiocomput.%26rft.volume%253D3%26rft.spage%253D241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3255.41" data-doi="10.1371/journal.pgen.1004528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathieson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Demography and the Age of Rare Variants.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004528</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004528%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004528%26rft_id%253Dinfo%253Apmid%252F25101869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004528&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25101869&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.42" data-doi="10.1016/j.schres.2016.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melton</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dragovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Exome array analysis suggests an increased variant burden in families with schizophrenia.</span> <abbr class="cit-jnl-abbrev">Schizophr. Res.</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.schres.2016.12.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSchizophr.%2BRes.%26rft.volume%253D185%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.schres.2016.12.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.schres.2016.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.43" data-doi="10.1038/ng.3643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kretzschmar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaneau</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A reference panel of 64,976 haplotypes for genotype imputation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1279</span>–<span class="cit-lpage">1283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3643</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D1279%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3643%26rft_id%253Dinfo%253Apmid%252F27548312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3643&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27548312&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.44" data-doi="10.1038/ng.3242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michailidou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beesley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canisius</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association analysis of more than 120,000 individuals identifies 15 new susceptibility loci for breast cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">380</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D373%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3242%26rft_id%253Dinfo%253Apmid%252F25751625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751625&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.45" data-doi="10.1126/science.1251688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gignoux</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-López</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sikora</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetics of Mexico recapitulates Native American substructure and affects biomedical traits.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>: <span class="cit-fpage">1280</span>–<span class="cit-lpage">1285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1251688</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMoreno-Estrada%26rft.auinit1%253DA.%26rft.volume%253D344%26rft.issue%253D6189%26rft.spage%253D1280%26rft.epage%253D1285%26rft.atitle%253DThe%2Bgenetics%2Bof%2BMexico%2Brecapitulates%2BNative%2BAmerican%2Bsubstructure%2Band%2Baffects%2Bbiomedical%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251688%26rft_id%253Dinfo%253Apmid%252F24926019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDQvNjE4OS8xMjgwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3255.46" data-doi="10.1371/journal.pgen.1003925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno-Estrada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakharia</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCauley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrnes</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reconstructing the Population Genetic History of the Caribbean.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003925</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003925%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003925%26rft_id%253Dinfo%253Apmid%252F24244192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244192&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.47" data-doi="10.1016/j.ajhg.2008.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Population Reference Sample, POPRES: A Resource for Population, Disease, and Pharmacological Genetics Research.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2008.08.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DNelson%26rft.auinit1%253DM.%2BR.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D347%26rft.epage%253D358%26rft.atitle%253DThe%2BPopulation%2BReference%2BSample%252C%2BPOPRES%253A%2Ba%2Bresource%2Bfor%2Bpopulation%252C%2Bdisease%252C%2Band%2Bpharmacological%2Bgenetics%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.08.005%26rft_id%253Dinfo%253Apmid%252F18760391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18760391&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.48" data-doi="10.1126/science.1217876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehm</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Jean</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1217876</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1217876%26rft_id%253Dinfo%253Apmid%252F22604722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5MC8xMDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyNTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.49" data-doi="10.1534/g3.113.007161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doheny</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pugh</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romm</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Imputation-based genomic coverage assessments of current human genotyping arrays.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1795</span>–<span class="cit-lpage">1807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.007161</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007161%26rft_id%253Dinfo%253Apmid%252F23979933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTAvMTc5NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzI1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.50" data-doi="10.1038/ng1816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bakker</span>  <span class="cit-name-given-names">P. I. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yelensky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evaluating and improving power in whole-genome association studies using fixed marker sets.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">663</span>–<span class="cit-lpage">667</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1816</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPe%2527er%26rft.auinit1%253DI.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D663%26rft.epage%253D667%26rft.atitle%253DEvaluating%2Band%2Bimproving%2Bpower%2Bin%2Bwhole-genome%2Bassociation%2Bstudies%2Busing%2Bfixed%2Bmarker%2Bsets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1816%26rft_id%253Dinfo%253Apmid%252F16715096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16715096&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237954800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.51" data-doi="10.1086/321275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Linkage disequilibrium in humans: models and data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/321275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D14%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2Bhumans%253A%2Bmodels%2Band%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321275%26rft_id%253Dinfo%253Apmid%252F11410837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11410837&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.52" data-doi="10.1038/nature12886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prüfer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Racimo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jay</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The complete genome sequence of a Neanderthal from the Altai Mountains.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12886</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPruefer%26rft.auinit1%253DK.%26rft.volume%253D505%26rft.issue%253D7481%26rft.spage%253D43%26rft.epage%253D49%26rft.atitle%253DThe%2Bcomplete%2Bgenome%2Bsequence%2Bof%2Ba%2BNeanderthal%2Bfrom%2Bthe%2BAltai%2BMountains%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12886%26rft_id%253Dinfo%253Apmid%252F24352235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2014009693&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=24352235&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329163300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.53" data-doi="10.1001/jama.2014.6511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">SIGMA Type 2 Diabetes Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estrada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aukrust</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjørkhaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burtt</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Association of a low-frequency variant in HNF1A with type 2 diabetes in a Latino population.</span> <abbr class="cit-jnl-abbrev">JAMA</abbr> <span class="cit-vol">311</span>: <span class="cit-fpage">2305</span>–<span class="cit-lpage">2314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1001/jama.2014.6511</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft.volume%253D311%26rft.spage%253D2305%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.2014.6511%26rft_id%253Dinfo%253Apmid%252F24915262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.2014.6511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24915262&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3255.54" data-doi="10.1371/journal.pgen.1000477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">C. C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Designing Genome-Wide Association Studies: Sample Size, Power, Imputation, and the Choice of Genotyping Chip.</span> <span class="cit-source">PLoS Genetics</span> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000477</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSpencer%26rft.auinit1%253DC.%2BC.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000477%26rft.epage%253De1000477%26rft.atitle%253DDesigning%2Bgenome-wide%2Bassociation%2Bstudies%253A%2Bsample%2Bsize%252C%2Bpower%252C%2Bimputation%252C%2Band%2Bthe%2Bchoice%2Bof%2Bgenotyping%2Bchip.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000477%26rft_id%253Dinfo%253Apmid%252F19492015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19492015&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.55" data-doi="10.1371/journal.pmed.1001779"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudlow</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallacher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beral</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age.</span> <abbr class="cit-jnl-abbrev">PLoS Med.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1001779</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pmed.1001779</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BMed.%26rft.volume%253D12%26rft.spage%253De1001779%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pmed.1001779%26rft_id%253Dinfo%253Apmid%252F25826379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pmed.1001779&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25826379&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3255.56" data-doi="10.1038/nature14962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">UK10K Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Min</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The UK10K project identifies rare variants in health and disease.</span> <span class="cit-source">Nature</span> <span class="cit-vol">526</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature14962</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14962%26rft_id%253Dinfo%253Apmid%252F26367797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14962&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26367797&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.57" data-doi="10.1086/378098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weale</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Depondt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Selection and evaluation of tagging SNPs in the neuronal-sodium-channel gene SCN1A: implications for linkage-disequilibrium gene mapping.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">565</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/378098</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWeale%26rft.auinit1%253DM.%2BE.%26rft.volume%253D73%26rft.issue%253D3%26rft.spage%253D551%26rft.epage%253D565%26rft.atitle%253DSelection%2Band%2Bevaluation%2Bof%2Btagging%2BSNPs%2Bin%2Bthe%2Bneuronal-sodium-channel%2Bgene%2BSCN1A%253A%2Bimplications%2Bfor%2Blinkage-disequilibrium%2Bgene%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F378098%26rft_id%253Dinfo%253Apmid%252F12900796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/378098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12900796&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185134000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3255.58" data-doi="10.1038/ncomms6897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wessel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaghootkar</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Low-frequency and rare exome chip variants associate with fasting glucose and type 2 diabetes susceptibility.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">5897</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6897</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D5897%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6897%26rft_id%253Dinfo%253Apmid%252F25631608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6897&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25631608&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3255.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3255.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3247" title="Genomic Structural Variations Within Five Continental Populations of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3269" title="Genomic Signatures of Adaptation to a Precipitation Gradient in Nigerian Sorghum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3255.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_5hcaov-Hy2AtKSr7Vg6KHXgNMnAs5_meJ9kM1IwgJw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3255" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3255" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3255" class="active" data-icon-position="" data-hide-link-title="0">Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490960" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-HZ_LUpBxtlkmOEIs9RQyVXydQkTnTLZOVUqDWNUSZRM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3255.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3255" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IER-6L8DAj50MmKG8gmqq1UNz1I9QMLiHoURhULezHg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3255" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3255.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490960" id="node490960--2" data-pisa="ggg;8/10/3255" data-pisa-master="ggg;g3.118.200502" data-apath="/ggg/8/10/3255.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7206-8088" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Genevieve L. Wojcik</span>, <span class="highwire-citation-author" data-delta="1">Christian Fuchsberger</span>, <span class="highwire-citation-author" data-delta="2">Daniel Taliun</span>, <span class="highwire-citation-author" data-delta="3">Ryan Welch</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0241-3522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alicia R Martin</span>, <span class="highwire-citation-author" data-delta="5">Suyash Shringarpure</span>, <span class="highwire-citation-author" data-delta="6">Christopher S. Carlson</span>, <span class="highwire-citation-author" data-delta="7">Goncalo Abecasis</span>, <span class="highwire-citation-author" data-delta="8">Hyun Min Kang</span>, <span class="highwire-citation-author" data-delta="9">Michael Boehnke</span>, <span class="highwire-citation-author" data-delta="10">Carlos D. Bustamante</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0001-9728-6567" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher R. Gignoux</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-9198-759X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eimear E. Kenny</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3255-3267;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200502 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490960--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Genevieve L. Wojcik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Genevieve%2BL.%2BWojcik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wojcik%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGenevieve%2BL.%2BWojcik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7206-8088" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Genevieve L. Wojcik</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian Fuchsberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biomedicine, European Academy of Bolzano/Bozen (EURAC), affiliated with the University of Lübeck, Bolzano, Bozen, 39100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BFuchsberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchsberger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BFuchsberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Taliun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BTaliun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taliun%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BTaliun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ryan Welch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BWelch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welch%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BWelch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alicia R Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BR%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlicia%2BR%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0241-3522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alicia R Martin</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Suyash Shringarpure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suyash%2BShringarpure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shringarpure%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuyash%2BShringarpure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Christopher S. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Fred Hutchinson Cancer Center, University of Washington, 1100 Fairview Ave. N., Seattle, WA 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BS.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BS.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Goncalo Abecasis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Goncalo%2BAbecasis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abecasis%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGoncalo%2BAbecasis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hyun Min Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyun%2BMin%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20HM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHyun%2BMin%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Michael Boehnke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BBoehnke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boehnke%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BBoehnke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Carlos D. Bustamante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biomedical Data Science, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BD.%2BBustamante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bustamante%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BD.%2BBustamante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Christopher R. Gignoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BR.%2BGignoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gignoux%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BR.%2BGignoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9728-6567" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher R. Gignoux</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.gignoux@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">chris.gignoux@ucdenver.edu</a>
<a href="mailto:eimear.kenny@mssm.edu" class="" data-icon-position="" data-hide-link-title="0">eimear.kenny@mssm.edu</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Eimear E. Kenny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Charles Bronfman Institute for Personalized Medicine, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>The Icahn Institute of Multiscale Biology and Genomics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>***</span>The Center for Statistical Genetics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eimear%2BE.%2BKenny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kenny%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEimear%2BE.%2BKenny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9198-759X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eimear E. Kenny</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.gignoux@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">chris.gignoux@ucdenver.edu</a>
<a href="mailto:eimear.kenny@mssm.edu" class="" data-icon-position="" data-hide-link-title="0">eimear.kenny@mssm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3255" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3255&amp;title=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3255&amp;title=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3255&amp;title=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3255&amp;text=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3255&amp;title=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3255&amp;t=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3255&amp;title=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3255&amp;title=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490960/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490960" id="node490960--3" data-pisa="ggg;8/10/3255" data-pisa-master="ggg;g3.118.200502" data-apath="/ggg/8/10/3255.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Imputation-Aware Tag SNP Selection To Improve Power for Large-Scale, Multi-ethnic Association Studies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7206-8088" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Genevieve L. Wojcik</span>, <span class="highwire-citation-author" data-delta="1">Christian Fuchsberger</span>, <span class="highwire-citation-author" data-delta="2">Daniel Taliun</span>, <span class="highwire-citation-author" data-delta="3">Ryan Welch</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0241-3522" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alicia R Martin</span>, <span class="highwire-citation-author" data-delta="5">Suyash Shringarpure</span>, <span class="highwire-citation-author" data-delta="6">Christopher S. Carlson</span>, <span class="highwire-citation-author" data-delta="7">Goncalo Abecasis</span>, <span class="highwire-citation-author" data-delta="8">Hyun Min Kang</span>, <span class="highwire-citation-author" data-delta="9">Michael Boehnke</span>, <span class="highwire-citation-author" data-delta="10">Carlos D. Bustamante</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0001-9728-6567" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christopher R. Gignoux</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-9198-759X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Eimear E. Kenny</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3255-3267;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200502 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490960--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Genevieve L. Wojcik </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Genevieve%2BL.%2BWojcik%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wojcik%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGenevieve%2BL.%2BWojcik%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7206-8088" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Genevieve L. Wojcik</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christian Fuchsberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biomedicine, European Academy of Bolzano/Bozen (EURAC), affiliated with the University of Lübeck, Bolzano, Bozen, 39100, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BFuchsberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuchsberger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BFuchsberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Taliun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BTaliun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taliun%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BTaliun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ryan Welch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BWelch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Welch%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BWelch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alicia R Martin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BR%2BMartin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martin%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlicia%2BR%2BMartin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0241-3522" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alicia R Martin</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Suyash Shringarpure </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suyash%2BShringarpure%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shringarpure%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuyash%2BShringarpure%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Christopher S. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Fred Hutchinson Cancer Center, University of Washington, 1100 Fairview Ave. N., Seattle, WA 98109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BS.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BS.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Goncalo Abecasis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Goncalo%2BAbecasis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abecasis%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGoncalo%2BAbecasis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hyun Min Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyun%2BMin%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20HM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHyun%2BMin%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Michael Boehnke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biostatistics and Center for Statistical Genetics, School of Public Health, University of Michigan, 1415 Washington Heights, Ann Arbor, MI 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BBoehnke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boehnke%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BBoehnke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Carlos D. Bustamante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Biomedical Data Science, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BD.%2BBustamante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bustamante%20CD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BD.%2BBustamante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Christopher R. Gignoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Stanford University School of Medicine, 365 Lasuen Street, Littlefield Center MC2069, Stanford, CA 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BR.%2BGignoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gignoux%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristopher%2BR.%2BGignoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9728-6567" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christopher R. Gignoux</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.gignoux@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">chris.gignoux@ucdenver.edu</a>
<a href="mailto:eimear.kenny@mssm.edu" class="" data-icon-position="" data-hide-link-title="0">eimear.kenny@mssm.edu</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Eimear E. Kenny </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>The Charles Bronfman Institute for Personalized Medicine, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>The Icahn Institute of Multiscale Biology and Genomics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div><div class='author-affiliation'><span class='nlm-sup'>***</span>The Center for Statistical Genetics, Icahn School of Medicine at Mount Sinai, 1 Gustave L. Levy Place, New York, NY 10029</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eimear%2BE.%2BKenny%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kenny%20EE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEimear%2BE.%2BKenny%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9198-759X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Eimear E. Kenny</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chris.gignoux@ucdenver.edu" class="" data-icon-position="" data-hide-link-title="0">chris.gignoux@ucdenver.edu</a>
<a href="mailto:eimear.kenny@mssm.edu" class="" data-icon-position="" data-hide-link-title="0">eimear.kenny@mssm.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3255&amp;rft.epage=3267&amp;rft.atitle=Imputation-Aware%20Tag%20SNP%20Selection%20To%20Improve%20Power%20for%20Large-Scale%2C%20Multi-ethnic%20Association%20Studies&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wojcik%2C+Genevieve+L.&amp;rft.au=Fuchsberger%2C+Christian&amp;rft.au=Taliun%2C+Daniel&amp;rft.au=Welch%2C+Ryan&amp;rft.au=Martin%2C+Alicia+R&amp;rft.au=Shringarpure%2C+Suyash&amp;rft.au=Carlson%2C+Christopher+S.&amp;rft.au=Abecasis%2C+Goncalo&amp;rft.au=Kang%2C+Hyun+Min&amp;rft.au=Boehnke%2C+Michael&amp;rft.au=Bustamante%2C+Carlos+D.&amp;rft.au=Gignoux%2C+Christopher+R.&amp;rft.au=Kenny%2C+Eimear+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490960/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490960/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490960/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490960/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490960/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490960/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490960/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490960/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490960/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490960/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490960/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490960/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3255" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3255.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3255.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/10/3255.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/10/3255.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
