<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3163" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3163.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3163.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3163.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/10/3163/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3163.atom" />
<meta name="HW.pisa" content="ggg;8/10/3163" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.118.200461" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Despres et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6936683" />
<meta name="DC.Description" content="CRISPR-Cas9 loss of function (LOF) and base editing screens are powerful tools in genetics and genomics. Yeast is one of the main models in these fields, but has only recently started to adopt this new toolkit for high throughput experiments. We developed a double selection strategy based on co-selection that increases LOF mutation rates using the Target-AID base editor. We constructed the pDYSCKO vector, which is amenable to high throughput double selection experiments, and show that the improvement in Target-AID efficiency generalizes across loci. Using modeling, we show that this improvement in efficiency provides the required increased in detection power to measure the fitness effects of thousands of mutations in typical yeast pooled screens. We show that double selection can also improve Cas9 mediated LOF rates, but that this multiplex genome editing causes programmable chromosomal translocations at high frequency. This suggests that multiplex LOF editing should be performed with caution and that base-editors could be preferable tools for some screens in yeast. Base editing using double selection is simple and straightforward and provides an alternative to homology directed repair based high throughput variant strain construction methods." />
<meta name="DC.Contributor" content="Philippe C Després" />
<meta name="DC.Contributor" content="Alexandre K Dubé" />
<meta name="DC.Contributor" content="Lou Nielly-Thibault" />
<meta name="DC.Contributor" content="Nozomu Yachie" />
<meta name="DC.Contributor" content="Christian R Landry" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;CRISPR-Cas9 loss of function (LOF) and base editing screens are powerful tools in genetics and genomics. Yeast is one of the main models in these fields, but has only recently started to adopt this new toolkit for high throughput experiments. We developed a double selection strategy based on co-selection that increases LOF mutation rates using the Target-AID base editor. We constructed the pDYSCKO vector, which is amenable to high throughput double selection experiments, and show that the improvement in Target-AID efficiency generalizes across loci. Using modeling, we show that this improvement in efficiency provides the required increased in detection power to measure the fitness effects of thousands of mutations in typical yeast pooled screens. We show that double selection can also improve Cas9 mediated LOF rates, but that this multiplex genome editing causes programmable chromosomal translocations at high frequency. This suggests that multiplex LOF editing should be performed with caution and that base-editors could be preferable tools for some screens in yeast. Base editing using double selection is simple and straightforward and provides an alternative to homology directed repair based high throughput variant strain construction methods.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3163" />
<meta name="citation_id" content="8/10/3163" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3163" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3163.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3163.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3163.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200461" />
<meta name="citation_pmid" content="30097473" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3163" />
<meta name="citation_lastpage" content="3171" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Philippe C Després" />
<meta name="citation_author_institution" content="Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1407-0643" />
<meta name="citation_author" content="Alexandre K Dubé" />
<meta name="citation_author_institution" content="Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author" content="Lou Nielly-Thibault" />
<meta name="citation_author_institution" content="PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3236-651X" />
<meta name="citation_author" content="Nozomu Yachie" />
<meta name="citation_author_institution" content="Research Center for Advanced Science and Technology, Synthetic Biology Division, University of Tokyo, Tokyo, 4-6-1 Komaba, Meguro-ku, 153-8904, Japan" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Tokyo, Japan" />
<meta name="citation_author_institution" content="Institute for Advanced Biosciences, Keio University, Tsuruoka, Japan" />
<meta name="citation_author_institution" content="PRESTO, Japan Science and Technology Agency, Tokyo, Japan" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1582-6027" />
<meta name="citation_author" content="Christian R Landry" />
<meta name="citation_author_institution" content="Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_institution" content="Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada" />
<meta name="citation_author_email" content="christian.landry@bio.ulaval.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3028-6866" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. Agudelo;citation_author=A. Duringer;citation_author=L. Bozoyan;citation_author=C. C. Huard;citation_author=S. Carter;citation_title=Marker-free coselection for CRISPR-driven genome editing in human cells.;citation_pages=615-620;citation_volume=14;citation_year=2017;citation_doi=10.1038/nmeth.4265" />
<meta name="citation_reference" content="Amberg D. C., D. J. Burke, and J. N. Strathern, 2005 Methods in Yeast Genetics: A Cold Spring Harbor Laboratory Course Manual, 2005 Edition." />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. Baker Brachmann;citation_author=A. Davies;citation_author=G. J. Cost;citation_author=E. Caputo;citation_author=J. Li;citation_title=Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.;citation_pages=115-132;citation_volume=14;citation_year=1998;citation_issue=2;citation_pmid=9483801;citation_doi=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Z. Bao;citation_author=M. HamediRad;citation_author=P. Xue;citation_author=H. Xiao;citation_author=I. Tasan;citation_title=Genome-scale engineering of Saccharomyces cerevisiae with single-nucleotide precision.;citation_pages=505-508;citation_volume=36;citation_year=2018;citation_doi=10.1038/nbt.4132" />
<meta name="citation_reference" content="citation_journal_title=J. Genet. Genomics;citation_author=A. R. Bassett;citation_author=L. Kong;citation_author=J.-L. Liu;citation_title=A genome-wide CRISPR library for high-throughput genetic screening in Drosophila cells.;citation_pages=301-309;citation_volume=42;citation_year=2015;citation_pmid=26165496;citation_doi=10.1016/j.jgg.2015.03.011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=P. Billon;citation_author=E. E. Bryant;citation_author=S. A. Joseph;citation_author=T. S. Nambiar;citation_author=S. B. Hayward;citation_title=CRISPR-Mediated Base Editing Enables Efficient Disruption of Eukaryotic Genes through Induction of STOP Codons.;citation_volume=67;citation_year=2017;citation_doi=10.1016/j.molcel.2017.08.008" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. E. DiCarlo;citation_author=J. E. Norville;citation_author=P. Mali;citation_author=X. Rios;citation_author=J. Aach;citation_title=Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems.;citation_volume=41;citation_year=2013;citation_pmid=23460208;citation_doi=10.1093/nar/gkt135" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. Engler;citation_author=R. Kandzia;citation_author=S. Marillonnet;citation_title=A one pot, one step, precision cloning method with high throughput capability.;citation_pages=e3647-e3647;citation_volume=3;citation_year=2008;citation_issue=11;citation_pmid=18985154;citation_doi=10.1371/journal.pone.0003647" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=N. M. Gaudelli;citation_author=A. C. Komor;citation_author=H. A. Rees;citation_author=M. S. Packer;citation_author=A. H. Badran;citation_title=Programmable base editing of A•T to G•C in genomic DNA without DNA cleavage.;citation_pages=464-471;citation_volume=551;citation_year=2017;citation_pmid=29160308;citation_doi=10.1038/nature24644" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="Hartl D. L., 2000 A Primer of Population Genetics. Sunderland, Mássachussets. U.S.A Sinauer Assoc. Inc. 26–31." />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=J. D. Hunter;citation_title=Matplotlib: A 2D Graphics Environment.;citation_pages=90-95;citation_volume=9;citation_year=2007;citation_doi=10.1109/MCSE.2007.55" />
<meta name="citation_reference" content="citation_author=M. Jinek;citation_author=K. Chylinski;citation_author=I. Fonfara;citation_author=M. Hauer;citation_author=J. A. Doudna;citation_year=2012;citation_pmid=22745249;citation_doi=10.1126/science.1225829" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. S. Kane;citation_author=M. Vora;citation_author=K. J. Varre;citation_author=R. W. Padgett;citation_title=Efficient Screening of CRISPR/Cas9-Induced Events in Drosophila Using a Co-CRISPR Strategy.;citation_volume=7;citation_year=2017;citation_pmid=27793971;citation_doi=10.1534/g3.116.036723" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. P. Kleinstiver;citation_author=M. S. Prew;citation_author=S. Q. Tsai;citation_author=V. V. Topkar;citation_author=N. T. Nguyen;citation_title=Engineered CRISPR-Cas9 nucleases with altered PAM specificities.;citation_pages=481-485;citation_volume=523;citation_year=2015;citation_pmid=26098369;citation_doi=10.1038/nature14592" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Kosicki;citation_author=K. Tomberg;citation_author=A. Bradley;citation_title=Repair of double-strand breaks induced by CRISPR–Cas9 leads to large deletions and complex rearrangements.;citation_pages=765-771;citation_volume=36;citation_year=2018;citation_pmid=30010673;citation_doi=10.1038/nbt0918-899c" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. Kuscu;citation_author=M. Parlak;citation_author=T. Tufan;citation_author=J. Yang;citation_author=K. Szlachta;citation_title=CRISPR-STOP: gene silencing through base-editing-induced nonsense mutations.;citation_pages=710-712;citation_volume=14;citation_year=2017;citation_pmid=28581493;citation_doi=10.1038/nmeth.4327" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=M. Lõoke;citation_author=K. Kristjuhan;citation_author=A. Kristjuhan;citation_title=Extraction of genomic DNA from yeasts for PCR-based applications.;citation_pages=v;citation_volume=62;citation_year=2017;citation_doi=10.2144/000114497" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. Maddalo;citation_author=E. Manchado;citation_author=C. P. Concepcion;citation_author=C. Bonetti;citation_author=J. A. Vidigal;citation_title=In vivo engineering of oncogenic chromosomal rearrangements with the CRISPR/Cas9 system.;citation_pages=423-427;citation_volume=516;citation_year=2014;citation_pmid=25337876;citation_doi=10.1038/nature13902" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=L. Maringele;citation_author=D. Lydall;citation_title=Pulsed-field gel electrophoresis of budding yeast chromosomes.;citation_pages=65-73;citation_volume=313;citation_year=2006;citation_pmid=16118425;citation_doi=10.1385/1&amp;ndash;59259&amp;ndash;958&amp;ndash;3:065" />
<meta name="citation_reference" content="McKinney W., 2010 Data Structures for Statistical Computing in Python. 51–56." />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=K. Nishida;citation_author=T. Arazoe;citation_author=N. Yachie;citation_author=S. Banno;citation_author=M. Kakimoto;citation_title=Targeted nucleotide editing using hybrid prokaryotic and vertebrate adaptive immune systems.;citation_volume=102;citation_year=2016;citation_pmid=27492474;citation_doi=10.1126/science.aaf8729" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=T. E. Oliphant;citation_title=Python for scientific computing.;citation_pages=10-20;citation_volume=9;citation_year=2007;citation_doi=10.1109/MCSE.2007.58" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=F. Perez;citation_author=B. E. Granger;citation_title=IPython: A System for Interactive Scientific Computing.;citation_pages=21-29;citation_volume=9;citation_year=2007;citation_doi=10.1109/MCSE.2007.53" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=L. S. Qi;citation_author=M. H. Larson;citation_author=L. A. Gilbert;citation_author=J. A. Doudna;citation_author=J. S. Weissman;citation_title=Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression.;citation_pages=1173-1183;citation_volume=152;citation_year=2013;citation_pmid=23452860;citation_doi=10.1016/j.cell.2013.02.022" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Rep;citation_author=W. Qian;citation_author=D. Ma;citation_author=C. Xiao;citation_author=Z. Wang;citation_author=J. Zhang;citation_title=The genomic landscape and evolutionary resolution of antagonistic pleiotropy in yeast.;citation_pages=1399-1410;citation_volume=2;citation_year=2012;citation_issue=5;citation_pmid=23103169;citation_doi=10.1016/j.celrep.2012.09.017" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. G. Robinson;citation_author=W. Chen;citation_author=J. D. Storey;citation_author=D. Gresham;citation_title=Design and analysis of Bar-seq experiments.;citation_pages=11-18;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.113.008565" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. R. Roy;citation_author=J. D. Smith;citation_author=S. C. Vonesch;citation_author=G. Lin;citation_author=C. S. Tu;citation_title=Multiplexed precision genome editing with trackable genomic barcodes in yeast.;citation_pages=512-520;citation_volume=36;citation_year=2018;citation_doi=10.1038/nbt.4137" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=J. D. Sander;citation_author=J. K. Joung;citation_title=CRISPR-Cas systems for editing, regulating and targeting genomes.;citation_pages=347-355;citation_volume=32;citation_year=2014;citation_pmid=24584096;citation_doi=10.1038/nbt.2842" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=N. E. Sanjana;citation_author=O. Shalem;citation_author=F. Zhang;citation_title=Improved vectors and genome-wide libraries for CRISPR screening.;citation_pages=783-784;citation_volume=11;citation_year=2014;citation_pmid=25075903;citation_doi=10.1038/nmeth.3047" />
<meta name="citation_reference" content="Shalem O., N. E. Sanjana, E. Hartenian, X. Shi, D. A. Scott, et al., 2014 Genome-scale CRISPR-Cas9 knockout screening in human cells. Science 343: 84–87. https://doi.org/10.1126/science.124700510.1126/science.1247005." />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=S. M. Sidik;citation_author=D. Huet;citation_author=S. M. Ganesan;citation_author=M.-H. Huynh;citation_author=T. Wang;citation_title=A Genome-wide CRISPR Screen in Toxoplasma Identifies Essential Apicomplexan Genes.;citation_pages=1423-1435.e12;citation_volume=166;citation_year=2016;citation_pmid=27594426;citation_doi=10.1016/j.cell.2016.08.019" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Cell Stem Cell;citation_author=Z. Tothova;citation_author=J. M. Krill-Burger;citation_author=K. D. Popova;citation_author=C. C. Landers;citation_author=Q. L. Sievers;citation_title=Multiplex CRISPR/Cas9-Based Genome Editing in Human Hematopoietic Stem Cells Models Clonal Hematopoiesis and Myeloid Neoplasia.;citation_pages=547-555.e8;citation_volume=21;citation_year=2017;citation_doi=10.1016/j.stem.2017.07.015" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=R. Verwaal;citation_author=N. Buiting-Wiessenhaan;citation_author=S. Dalhuijsen;citation_author=J. A. Roubos;citation_title=CRISPR/Cpf1 enables fast and simple genome editing of Saccharomyces cerevisiae.;citation_pages=201-211;citation_volume=35;citation_year=2018;citation_doi=10.1002/yea.3278" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. Wach;citation_author=A. Brachat;citation_author=R. Pöhlmann;citation_author=P. Philippsen;citation_title=New heterologous modules for classical or PCR-based gene disruptions in Saccharomyces cerevisiae.;citation_pages=1793-1808;citation_volume=10;citation_year=1994;citation_issue=13;citation_pmid=7747518;citation_doi=10.1002/yea.320101310" />
<meta name="citation_reference" content="citation_journal_title=Comput. Sci. Eng.;citation_author=S. van der Walt;citation_author=S. C. Colbert;citation_author=G. Varoquaux;citation_title=The NumPy Array: A Structure for Efficient Numerical Computation.;citation_pages=22-30;citation_volume=13;citation_year=2011;citation_doi=10.1109/MCSE.2011.37" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="CRISPR-Cas9 loss of function (LOF) and base editing screens are powerful tools in genetics and genomics. Yeast is one of the main models in these fields, but has only recently started to adopt this new toolkit for high throughput experiments. We developed a double selection strategy based on co-selection that increases LOF mutation rates using the Target-AID base editor. We constructed the pDYSCKO vector, which is amenable to high throughput double selection experiments, and show that the improvement in Target-AID efficiency generalizes across loci. Using modeling, we show that this improvement in efficiency provides the required increased in detection power to measure the fitness effects of thousands of mutations in typical yeast pooled screens. We show that double selection can also improve Cas9 mediated LOF rates, but that this multiplex genome editing causes programmable chromosomal translocations at high frequency. This suggests that multiplex LOF editing should be performed with caution and that base-editors could be preferable tools for some screens in yeast. Base editing using double selection is simple and straightforward and provides an alternative to homology directed repair based high throughput variant strain construction methods." />
<meta name="og-title" property="og:title" content="Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3163" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="CRISPR-Cas9 loss of function (LOF) and base editing screens are powerful tools in genetics and genomics. Yeast is one of the main models in these fields, but has only recently started to adopt this new toolkit for high throughput experiments. We developed a double selection strategy based on co-selection that increases LOF mutation rates using the Target-AID base editor. We constructed the pDYSCKO vector, which is amenable to high throughput double selection experiments, and show that the improvement in Target-AID efficiency generalizes across loci. Using modeling, we show that this improvement in efficiency provides the required increased in detection power to measure the fitness effects of thousands of mutations in typical yeast pooled screens. We show that double selection can also improve Cas9 mediated LOF rates, but that this multiplex genome editing causes programmable chromosomal translocations at high frequency. This suggests that multiplex LOF editing should be performed with caution and that base-editors could be preferable tools for some screens in yeast. Base editing using double selection is simple and straightforward and provides an alternative to homology directed repair based high throughput variant strain construction methods." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TsfZVxk-Z6TrnYCwBE-8-XTQRQkoKybP-u5y6P99wpM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490961","apath":"\/ggg\/8\/10\/3163.atom","pisa":"ggg;8\/10\/3163","ac":{"\/ggg\/8\/10\/3163.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3163.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3163"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3163"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3163":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MhhcfrTo5pBV0wfYgjstud8Z7dbJzQxjQkFh\\\\\\\/\\\\\\\/VRKKjGNP\\\\\\\/hgcH8viXJbmuObI8EacaLIGjWFotSlU2ugGirnsJwea4q0l8e+H68SPfwN7sx2kUQRU9hU+pzO1wO1yCfsfKLUMJwkAtKAK4Nxf7Mz9bqe3hKKq2ECr80fhp2NFKE3snTZFBRed3lckS\\\\\\\/HysgvnN3uxhOpLYX9nbCWYJIlEcOGdxohfC4aOt0FkwyyzACvC9ruSg16Hz9FkKSk6DqnMvzRKzfw9XfGBTze4DyydJeleUT3Saj\\\\\\\/hxhliVXVHoSI0VCvhcf3iOg4F0srfHI3I3rsuLDNkle629tkY2JrOUPb7UsqowhTboLhBFMHd9VoGvIz8R+9S2\\\\\\\/o4FoJAub86Brpl4WSOBv+v4xQVyodFpKjLUNHfetRwH+BLnmqwiorusIuTPLe80uHA+ziyDSXHMMTKSqzBrSxGk4H+FAI+6fnEtGz3aQtKBncQcpjay\\\\\\\/h2v\\\\\\\/JN0WSPCE15ps\\\\\\\/0sohiVV7qPRQbHhNm+D6Drt9ZxqgWGn0mzLShwgDtS64B8KbGn0hopVF3HCrmLRBNCNVrsq38JMw2MUimfS5ZSycDnn2YYChjKFer2NKbWG9CFJbWGt2cliB1idGr2uYo6q6DxyjaOxvaDs+rKJ8VqT2dlbZFq6p4PH5UayxuN2OP1aMEgup7pvSR6rRZncqe0r8znbaBYO0BEPeRWvjZWRLdeDbdvAlx0Plaw8DKMwigfyzFrNX2Q0+k3aSxm11QGmGTkn5VDQagqgrcpUS\\\\\\\/CT1nHvxO8SvjMR4y6qXc0od\\\\\\\/4CEs\\\\\\\/cmBms8XRYKoHKa4Q0rBD6cKsuEm+0+IZES1SPleNzEIoyaWjIQL4gdtIbdXzQAo35k5zcir61KAJO6\\\\\\\/cPYI\\\\\\\/Vl+GfTMuuWztqzKEezkDwNjOQlu9cxPlQ7nOJqgZk8ZfEldJhlritdJAy4c9DID+iQRG4e\\\\\\\/tZ+GD+UZTafNKQHMCyeQ80mbLpIAyhwmNStvU+rT3ZJWdUVN\\\\\\\/lcQ\\\\\\\/mQe7oNGJThfmLZiFdTaWHWd1He4+EnZ43x5OtDfP8yoRiPTaOLBwZ5Hnb7R\\\\\\\/afe0Ym1OJppFg9gcQRsHxZithvwGzYR9v3hWu2D6v8GkqdNxWoF9kfKIr4owBsJ5jIsbUS4JSypyUnMzkmlKr\\\\\\\/6kxaoppm9PMpDFInadUsCtb+ZCVPmIU6xd1wgRC5nI0\\\\\\\/KFgLBCN1OsWdghXS24VhV0TSKxSPy4Ttp0NMTLPTROCjkY1U8BOeMszox3IOgzFnqjn\\\u0022,\\\u0022iv\\\u0022:\\\u0022AReLbapmTlbT1Ywe06vFkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002275ff198bceb19c579875ef3e17536f4b6928f9f7bd2ab92d4baf311e62d98dfa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CTdYogypII6OubtnZUf6INa1Xt2NLjlvVYhDnmgSJ5x36Y\\\\\\\/nKyGT+PjuZPtMvnQoHPY7pACxFBHBr4J8fLn50VBG2WgmVam0ExgFC7wZ1T6diwWfgIr1Q6PEeZerSr6YHDGTrLOcqwGamjc2\\\\\\\/5e3JP94QXqR8uOrL2VZqEpIXVGwP4cCQ\\\\\\\/Lp+Ign9ysZIU1kz5wi5pCODi1f5H0lDse7EkZP67dGjYve7oRvy1w2iwbWVwIrQjHd\\\\\\\/fjHnCkarx09eEvaAuXpyps\\\\\\\/LHZipm2NZoxs8NfxhamgrAnNa2gPRaMysLV2GaI1wFzcS3loz97ZJ5L\\\\\\\/rXb8wbjOLUW+4t77oL8VZlsBK7uVGCSsG++Lufpkbd9BevzDs2tokYtBlov1twYUZqTCCrW4BtcC3TLXtcwifCSJ439omg4n1lybqb3FEb3bmqGckRsjk3jTNisUcSNhbvpceO0zxl8VylOPj6\\\\\\\/O5YlWl3UeNHesPuSH\\\\\\\/7AtCxOl73hxFJLlrcX4i\\\\\\\/H+gz1MMvOjfKN8mQS9ZlCI2v+QhCtSdVg8LL0+4ZIq\\\\\\\/v3UB5DTHrDYFiY3ocI5rrp1fVjGavWT1JTL23ZYwdsODILJ\\\\\\\/+HAE796RYHZ5kC3FoXedWXK4OLqRxeshJTnwFXdsxxufwC+Ur576iUUKOchR1tPwPyJFsdIP\\\\\\\/gRg3XhCBfW2IsNCiMOrqRjpCCEaFt1Zr6tpRglESGv2vmMQZF\\\\\\\/YgaOtlGSTkNl5y98bdrpWeXFZtU8xEFKK8GqYQecQkaU0ct3tEaer4T0AUmtkr+1P87noDiU9UTt4CrzfxUcsI0mmp46NycmpowsE5vyqmhzV5QB6fV+Y5WthWcWHSkC59d6ANb6gHSquIITPVHsT9L4HhYYSoIMAmYbSOMC1IYy4vZJFE+2Wy9w9db\\\\\\\/vEcCOGrtswqcn9e9xMPzXbwWcfaS7amzZVyERymw13NCRsW+Y1gxq2CPNcJTTUin\\\\\\\/7+\\\\\\\/Xle\\\\\\\/K\\\\\\\/b2IZW73VnlcvPgUtNkpEM6OBYoEdMszIlQOp0rSW5QAB6WqrjwE3Prr7Y11K7eKGB2mReh50eYIzOrfjwwHS5R\\\\\\\/98SiJ10PEgsmM3tU05fQzzX4Ay9Hm+zItLU+QO33efjEy5Z+P0v5GagjHqKCyxHcjgoOIK\\\\\\\/Vy+civbmZjugPSWhXTbXIdDq9rOBtJBCa9PvYZHf0+P9fU5YVPMZ1HiO3kKBGhlRPDki+OZbL9tcjDaHZlGcLuvGjip5U\\\\\\\/\\\\\\\/SrqSBPlF+PhL5vkDdzj5jhSao0lITiDa+d+UjhZNtsm\\\\\\\/UDM9nYBBHBBJazaVaBvVoK6zAt+mOr7FmY4KvARJcSGkgs\\\\\\\/5QwZNCUcdof1yTdX6ghTQiv4wr7kUIcyIB8yuh0+wm0KX1km2TNMWtq2wFG7NYW36DZqKvPVmIsVfK+jRrkTaxRL9DuXxAbM6X6aRuQJUhvcQqDSJ7tXizsrgSzKu0rbHFycJ072glK4Z2p1NFfdSpHqQrU4m\\\\\\\/97RjTnPGB7GAF4gR2SxyIsv8GWFtGwnUy4O+J3HD21y0BQXNG3W6qZ2uhdKlSAPoOwPeAF8k\\\\\\\/Rkh+B8RAR\\\\\\\/jUrVNNSy3hIewqxyjfJCuLk3R\\\\\\\/P9BQP4VtO3cC\\\\\\\/\\\\\\\/LgZFGkQigY5dkIJAF0rJUUm2bIqqIi8d2ki1AQWpsbNpwEbmonpXtwBpnL+YLqM\\\\\\\/QAlal6Qw9NQgHFJLc7uqpSmK7kIxx9rJtgZzZ8ZZ0fle\\\\\\\/zYwZ3UDa177K\\\\\\\/eq2f5PaT0+MG7dwKSouNyfVgDnpR5\\\\\\\/T7uIxtf2LuiRUUrWrPbWyqkCYH9qN2hksjrWAlT\\\\\\\/ZfjSMAI\\\\\\\/F8DNRFm1UZkK++NsflGQBkHCdBfcCHSydoUi1JKAkMJgamlBS\\\\\\\/rm6\\\\\\\/oY2fGsRx9mLqHhzsVDNxDB12IzszIcW028ddrr4QQJMM7uWNv1NqkbHeBN\\\\\\\/sLQD5kIS1y0jU3w2hqEy8qyhAFHkJBlboafBbFY9DWdlF+5qE+daifzrcDqPHZJV5OzymfUsTJa1N3YfyIA8kTWP5bfmZ8i0Q9hAb4JIKAMxnaSs+RGmOfzq8vjkxkG4umokEnLjccLQG+Mtoktsp2BqASliACVNg\\\\\\\/twgfNe7X6d7IsheNBgo3FCV7RWSjjEYM\\\\\\\/WD6W7FkYqNmOUTg7rGOGxGwgxl9AJLBz38KuWyzm6Xf6QPcUUjuI1cItNA0XRCc2WTAUps+tJlMyIWxTSGPoORnw4dt7Pv\\\\\\\/uJkO6j4gJmkeSNwJZzOPpaZ2XBxRZOA54vG+t8VOw0S\\\\\\\/I93baMvtGWNbdRPyfIMbUyNwSQiF5l9tjp9bDI1pD8MmgR8WRCs7QB0KC0VXA68snbykyqtl1ql1RTs9l8wJfxU\\\\\\\/AWFqidOXakT\\\\\\\/lBikZM5lpcDj\\\\\\\/rCIBcYGkRDS7ogv2JZ+avmSJbvc+\\\\\\\/v3+\\\\\\\/n89\\\\\\\/cqozol2Xfb5bRLtYQt\\\\\\\/LHe1Y47X31OKFXWub7R+3QvXdT3IPAxpnk\\\\\\\/tgwjN4EvHs5Ze1g4GxvKld9x4amCCORF8KflGMAm6fJD9k7BRAAriyuqE9YQIWtU4kLKHLlGDq4CHhJFRzqTMdQLjmJrJkG3K7kVrXgonkL6XhD25YIjdu3psKA\\\\\\\/0L1hOSWrm6SFrEdlb1DStdtRyPlc9w26u6xOKkNb+Etc0W1lKQnqJBqi+1b10s3lDD+rN8SaRzOt+F7TT5Lta4dR3nXd7M\\\\\\\/9kLRJmzZ7etr3m0ntxMxGQbWj3bYY19qPWU1f3Q1uKUPjrFRk5uwgM34l+45opPtGC2ToAShDJJ9wVX2Qb9MFhLWdiHPbjdR8u5FHCFiLmPcElBoUUAGwPOGKdNsnoJT0JLoihCDnKtt\\\\\\\/NsjaRienZGvBzF8xvsFgeI4XjXRL7haqpeBubS\\\\\\\/ToJAmdO1bCCoqeIT96gb3vw5etOe8A8lGKLXl131iY4uS1eAmKz9C8DIrUJOMSSy8Hg\\\\\\\/xBL06ZZTWkJ6Qi4xmV1aMdTCG7SeV87gAgASon9dCcy7jXNkIro+CczUNvnqJzDyeX\\\\\\\/uY\\\\\\\/0JiVm555mxiJNAE8uPaUTA1VMnumraTjdEejBHr38Ab2W3TsrItM6PzICFsrjTpoWWZ2YZ1AJvqpMRlYyZhIH3HTaULsQtjrNWtwkxAQce8EMAzSJZD+5xoi+Xs2b34yRBQoYP7xmTHmDLxQ0LvAm1BP7qjdDnQyKgTKUYgJT8XVqmGWGQEnx6Tik+iwch3GqAwqTbvubWaGsVc9a15Ktbp5p21TFKaMPOep6ZPk5zRH9vgMy0BDMfXBi9jJ7m6Y6WPbXMc4c2u6sm9glriys8MXQXW91xxy6bzN51gjVNlyndEruwdkiK8grYZpQEdjocIs1FeTlyzleXcYnU4RvYwBV4RW9mnHbZ7NSCxDMXZtuKJ8f92PLRvVp+zL0UVdWYR4HPCgxHo6Ap6W7vaGFYDLNlMmD3eNX1w08zWQsCfxTsxIonputqe4kBsTUmZImXmkqG0hh3FhaIiJmGVFF7IUdgBiySl0Js5LSHceOIoTuAl9bRx0zQSva2O57QrfQQfryxKV84pZmgnJTFgvGxqb1AkJRuQLVfi5z7n5H+1jBOwCKpGJfQcO0FdDvua4bFOw43Y1rzEvyzglc0HTrV9sAgu4QtS\\\\\\\/wW+unrSy8bhPo8Ay9VYFq\\\\\\\/lkd5SmjXrRMquE7aMtWxzcgpmiZpMdjMU3mKeayBhhZ49Ben5PzCXtlkKTEosZFwN0+B9XDUNnP5WJyDU2inVCFfgc4\\\\\\\/PGS8HgraqTY5E6Yf4nvB6ZzCFV0o3hQ5yOCjIT7GIRtWxla7rccMGEomeQLiwImeQR3TuBKfWZKDAaxflKlRGTj6WkL8f3VaRbBtDFO2XDUXZkq2ehCtdBappSNMeHfqp2yrpuEqUeTqd1XDL9LTGr6WH1U4s1lP+Kqbs3TwR8NmnJOi\\\\\\\/IPa2Z\\\\\\\/lr7bUbA2tqHDGLIvgurcjL+8GUJUyesHKDoDHMbDyANmlIVgq3siQTWmzC+1lvoIdF7aIQyxX1hedcF0WmjhSU5v4h61PtVSWTo2l32GVVEob1SmOv8hfVCBXloFL5wfrBSi1cnuy3E33nyHFtY8W9kMNb2RJ2ej4WH0VbQdgP\\\\\\\/TqCRLOpq0n3QfwLTYLOKsH5M4ZsyImQKeuhXnbCbjZBGC4atszUUUBgwgfHyNdPLoCgFEVFAOGRa21r4fPSdaFqayzgz+9xK3RRT319EoaX5aD4h8RGFeTqi\\\\\\\/vKJ9gvk0\\\\\\\/15m9Pj8E61w6e0zNv7w7aKXfkwcRZ+HZGFDt6RLoeoWo6BiX40LZ5Hx+2a9G2vnAxBGyMJi38ZHWwK8SXE67TbYmyGTCx0zNEuUPOu2LYiDyUuQGo6vLdDi+wYnusBJu\\\\\\\/OD7y0hF+yOC2PuCImw3IzCa6tUoLPbf9EuTpM8V50AK8V6v9k8tPVhzkKJvYJS8d7ZbOZVi+ZjjgBWZJJgywpOnIta4hljUQGSno4bbIzoi62DFvk4SwVV7xBr+vtYX0wTQEqWWbW4MqoPpK\\\\\\\/stJu3bLO084Ukdny4s12FC6Tzwmzhh26Jqf2lBpF2lvOU+KFwSLlb+3lqK90ib6QmS8jBcqL78kVNMUrRaZT24qphqex7lbjUH1wGEICNlCXrPNZk13BoqG\\\\\\\/Fa+3EInP2zWk2C4k0j7fOhGHE1hiZYUjYV8TW+1wpleSTB53mYCBwxY6H\\\\\\\/bnWMGDeLPmJ6Lsn26U3RsGGnNB4Q7hcZYDE29xqZOP7L7fqD3RQ1ESu9qpIVEvrKPefOBmJ\\\\\\\/hq9uBkIjEc5uj5UhX4eZT+pS9aIHPkKnQOajgkE6YlFj73T3CL+wzB56uFzT+0djcE6hWae6YGyzDVwsUn+axUNH5X\\\\\\\/hEm1kPwt1g5tXS1EKWDklAWJ7EuFlgdJSZiEvZ+kVrALZT3lsWkZJM8OsxWLmQ5jm3doFDK5oSwh2HdOCD7Vst1PezTL\\\\\\\/lvNFlXQVZaZt7xNLU2rl6uS7YC0hjgdNO40gE2t6eOD8ptNa0tpH017LPkvg0roD\\\\\\\/GuBK4jV7KGpvDa+dX99l16Y83PGKjFD7cCnOWDdNryGBCjJXUmyZbCXMPSpFVOLuHAu+qCt\\\\\\\/+zOgeYlPNr6iXhSJOGmgYSIbJ9msTxZ9VeqwGxZM4EcYtg2ocNFjqnJ3UJreEaz5Yjm+KOCgASGPkvgJVwYwtxGMrZ3OREqV6tqoJ0+dk6+UGRHakPEk6Z\\\\\\\/hxUkrItUoh0PbWp\\\\\\\/tkOLKW7P8IsqfHQpvPlfFrv+aZjm0+ROWUukgM0IeU3cHi2ymCxhU7reoeyscr48b5NNUsOCRV831twj7rr\\\\\\\/2YXbEjqM6A1U7z9yHC67ZqXgBu476HiuY9WSfn1+uWdqHm94cckaI8hsH9At9ePp5hQf4XfKXXFtwK+3VLzh8+wyNEjFesByyEtHeg169ubiPiHMdNBZj\\\\\\\/+Dc6Hqhx0wYOEJjNtdyRMbYjgwiP5fT2tCsUhy\\\\\\\/vrhxHIMXN3\\\\\\\/33rihZBsDCqmUMMFspGmumcN5C5AxijSyOdxP1wiaIjEYjRuer4Es\\\\\\\/I45FmIGUEgnQZqpcpZs4qnZfBivGnnu1vCJBXNhSyIssZx+rl1VvN3Di7GtEJnb95O8SuQodnnNJXdxu7nEBPufJq3Ytu9lBrJa1Cshek8ky6XbB3lUVZUHfZT68IhCTC4Hvmp10Hb47HgJM0Ej818eDzWbN50wn7WRxKecXgWHEH2ez36f9LyW9b4uDsVJUqHlPcQcMc5gT1P7Csl7gi\\\\\\\/D4METar+IeW8ymIU0tJRsUSZ0nF77TFR4GhFoMEEqOEr62Mg6Vu7l9cD4i2rX8ZdlmD32wQuy8kP+eaaJf4p3ie1YeRjOTLXfOlvJPWrjrAwJ\\\\\\\/nqNaw1oxrv55s1RRhYZ3yoNsUdMGb0l6QPHPXOwCoMWJFuLH7g43AGkpNwfbRPJF4lZflwjkDbzlJHHs3LDpsy715zD18vX8XhhYgu2l\\\\\\\/Bo2qPfgazBu\\\\\\\/FK1SOZdelUlCxYP7KhUWjkwS3vn3mGEB4anrEDPDAjhHjObdPhJkLPcUKNqpfqE9kt+qlxxr01\\\\\\\/bwt8TQBRbWA0fs6EDjhH9sQd7QokEWlaXHuPd+DMJO7fHW6fnou47yGPI6+NR6pFOqOGKPvxgQN9PgtgenZP3F7pL7mrElmz6CSshttnE2LPvKwckiy0wt3yr1Ui2iXbVVEAm6ByBmdNBmVfiJBKLVSVTENH+1ZgmRV4kTH80kbsiSkiG75jr1WLN2zFLWVBQgpdDE4FsJbcC4TKx0t7LjjEJBo15NrKTTJzL67HMAsg5dcaWLZYZynVt7rifbEP3zvgN8W5\\\\\\\/186IVpBjqgg8ZNAEFH2TNA7dMWU4TkQDgILyqUHW+H2DfQKP7AX\\\\\\\/BbxsXZ2p8fvnCHJ5zT73tja1uG9YtYGcviQ9QJDzuVYz9iA3fhfRIVucgdPkwKODgXFQa5+YOvk7hFnfxMSiXbNWjwwk06mcqRxcq+eNyjJG6qfleEM4dtCU9NbqoedagfMZ9umnPNmP9Qz5UNMeiNYL52YRaOi6x5qfC1mz7cPWpoImp1LezTBsPAqGpGk4FSaUZe01b6YDLcteGIwu1OPCWCS4wD+DgNk+Z3u92MXrh\\\\\\\/j9HUPvb3v7bz3EEvN56yV0fHgpGuRvLknuXMnFMl2kL10MwkGlJLfWuUkXsIGcpP28xYTO+ZvrjC9HIJTHngstqffiThjm5K9OgWIXBhyuHxU5v3hLjBAUXQnE8GWVYhj3lUk4YkHOkOjActifMNkTNDe7OHQkhmhHiYCcYGYute8QGyXwyZgaXOevq9GWsudVUgBJw\\\\\\\/yxPbVYdgIaRvjRcH6LnfZlAH8B3ZIZP44RTa61JB9pqbllr4tJp+Koc2o8DUChoMUhIkmLfm4xrjwog9k38OkT3FXfemuomzoNUDgR6vOv8h49KUasGf75r9DZ9s7+sX1hf1ekbwXYUPHzXRkfAb7BH+OqGwhjLt9mWk1bO8vWcKIq8xMCrGdLDKY5Au89Orjgi53LqORv0uwbkyGNthFDje5UY710rIpZvtPpSTII2cCZJua0yGXwPwETPYCBvmJgMyOvQgzm0T5bzpvYqUlHlTWp1Udcy3KdriuU+mtviOB83096yGSpZA13cljO1tDJEpnaQsFGsXvm5mW+ECHvw++xB4\\\\\\\/fSAo8PYYquODUZ3dswf1TqNaXIOmb8BxwQ3HzAZvv7DVx0VJpBMYmXLuiO5UI+cZ2CfGu4Uny2qu7MqO41l4jhQeDqlsU3+2B7980ihvAB0pvMlZLcV2xuSaV8DrmUB6A1P4aYorrZSQWnRS5sphX57OseRdsp76ToaqJP2DgeL8SrmAq8c\\\\\\\/npHVa6kQU0lzy0JedwoyrSZUbimSdd49f0b+WSTK+sZNUjyhyqcpB8DaJpLUN9+LbK8j\\\\\\\/a76pSxhM7WNa+ne9jypYZixYmRSxoR02nWdqbmCkwqI+mgXoCq9HO6KmIRpI9ygoOLYsfzkq6tDqbtstfXsk\\\\\\\/j\\\\\\\/ScadHoiV+cpol7m\\\\\\\/glBJNTOnyNLVnjqWqTD3hH4L4WvSlmTA7WMvw5UfYCScXp7rURw4+jWOs0he8VePn1HlhP4M428pxPn5i2fKmFzScd6KE9qxgB00WZh+aTN0eQtyuoS3mphNPyBaf62S+xrvUv\\\\\\\/OpouQ+Sb9Eg3cfT0j3la92TgJW\\\\\\\/D6mVEsbRhucNXqJ015\\\\\\\/\\\\\\\/lF3Enag7v+BOdzB6XG4NuvZ9fMNiwvdDkmAZEYYBhuNQRpIuEwC7foYCDOHJqo\\\\\\\/HjRQYrvVYD72WXiRXBJnpiGQUbPwCjXYUgd6mz+9y1Wac5LiRpUvJ15q0iN2R+pGHxjyQpMKl91k7tUEZUKmO08W8+SeNW3DYenk4IZV8X7WLt877ll9UKJVxQ3IFS9zhq2lcjf5hGmCSg3X6bdXTHeb8vEMVn+RLOhnCDvzso+eVSXrsmrS2i0DwX\\\\\\\/M2mTxHyn8pjayUt4HHuD7+sf9IYy4BP+Tk1a1w8Kd1qXRfdcpsJjd8wEKVW2XYDOg+0cBy+0cAiRLUtKJq+soQ69Tnh6QYp8npl8W10jZhgVOXkx62wi2Jec8XytkJdIiGs\\\\\\\/OlLzMgsRKMoDKQ+greQXIIF27d\\\\\\\/dPE7fd+p9TIgpBXHvyfXLvHGDYcF5l1rBqc8ZQcyHFKHRbEvGJiOnSPok48dIvwzK+CWlOdxArEhJwEBVxxmlPKOZ67hOKc97DmhX6uhb8u5NyDfPiVg76WWdFaKRIrwNSa4rqRnx+5ef\\\\\\\/mYnPhcQmdgbmP4DIec7tUuVMUfvV14RMRcrNdNkdDPw1SKPZzevwiwf8ijNuf2MPKwZrdR1QE+ZbYdgpMzyfeYWH4auSUxBkglZlV1Dd7B9fzGXJEmLSjtYbTlIb1QlD5L1x1gqXwwKU1sliGuy0\\\\\\\/dG+MYvi7RcHHL5NWCgKqMpZQy5BsyviZ6oXKfiJYhB+bL9YJ4hrIlj24Kd6CaYZqri529Ieq+BC8LiWPTe5FYMHzSwrXV\\\\\\\/qyOUD8qg2sVau2Q3VZwlclI22OOn5Alh0gLOk0JODTdPI7XpegxpYlFO0exMwcyKjtX+X8HzhF1eGwu6w1\\\\\\\/FbZ04FnZLh196saSgwp5P5cgV2shtK42YIU+KX14JVzLdoOKcHZgKAaer\\\\\\\/PgjopNfgCUjcTECwAKfCIBIArN7I7ml7dl1lD\\\\\\\/k8vs85Sq2erU4QdkiPLHNJwPwE\\\\\\\/0GXgXLKnnthWvPwtyYBY0HFLzotKBBvR6YkdIYqAicRkRDMHgtzwpoUbSb8MQffwJvtDa785F9wdhR4zvAEAWR7NH\\\\\\\/wZWzISEHGRvKQsLskFWtwF2KKXqWSJZEmkRhaTtUc\\\\\\\/j0Eu8b8xv5M7ERUam8mVfbI6O5guFftro9SPZN48MQ\\\\\\\/KVFwih9jqe7cukMNUbinKVbivGwlzJpY5FXw85BMUiWv7VPLY4MvDtJO2sz3LT+Ai0F2bWAxB2FqCeEGelAzicip7ZpqJ5e1T6XRaqxpfMAmH5+kcGRYOVIDpG+sHmoNOld7g8mBkAEoB+4iSKmv6jdYYzzAvbK2hi1pZYDE9l63F5bQI4BiT2OwSZyJ0jwHgk4WE\\\\\\\/ab7k2ezkQulDTQDeghUcf5vluZJwBmabU5i0gZIbPgA2NU4zG8VAakfA\\\\\\\/imiL2BaLZkpbZLEhc2zAxR664rqDBKNEPCOxNyiB9B5xqSVC6Lm\\\\\\\/CDoECCMNF3pZ4nOuVTlMVTASUgk+7Y7YJ6Z9k4MEQdYLlVW3Voe44aimPeGgcD19bRPbtDfkQFhefRYvUIC\\\\\\\/rDiIACuaEVqmMTkmoZHR8dGyrXRODwZCp6HlgkAPo9LkzA87bosVwK3R0CzgGatDA\\\\\\\/PRzli9cTqEm\\\\\\\/CoZxpBgl+ynaBZ9tMpWi8K3sYwH8OyuAk8\\\\\\\/u+HmZ5bAIsaOLnePcKXDNcpIgSEfy8UP5lZbsBeEEaQcIi6gceK+7axXpAoo4Vbr32dyWbTlfme7qCZ1q\\\\\\\/zmG\\\\\\\/49bRZi6P04foqBdlpQxTcPTRDFZsXS5548KTtMwGLhl5eQFCQ+YaSGhygQYhaQ2TEVUsiH84lV24PXnu8dxCz8At24SdMkTrm+IMmdx7pI6gOoeslLhKJ0L+L\\\\\\\/qOF2hrTdjbZ7yj6D8cX\\\\\\\/jFSwxPyIO6WqVmQub\\\\\\\/GnFpMdcnk38GB1kSBlBsVJQ9s1xkU4+EXWYd5IBG+7ABz4Y5F82Hb8wvSfBsCUyTvkKRSgu1+yQEtR8cLagpnmMiia0HBVCMZ8Z8kUoAQV+dRdjapQo9GDAUssd7MBdNoZx5Cn1VA53pPLUhsQp7g8ATs5dgAuvL3O1K+dOvuWNu18YUBriNo\\\\\\\/KerTDxAG5Jzpw7ZYPmweSHA7NWTtcD9OIC3VCASAgHjdQ+wPoRAnoNTrYuGAY10eREL6UKVOMOr9xN8dS+8tmr7iGGvnFRkS9eNkXFVsK+rnWOSzDuYEFcPMvWGC8KmCx0fQTY1pkvbtcvtE6u\\\\\\\/i9IZumRieVCHY0MDvljLtYKBcRdn6btF6aHrtB\\\\\\\/mF7Dnje\\\\\\\/IkWifnTthIweJfE5ZCeGvV54IZK5q4kqW2t3OQXXB7cmPaGBAmO23VUipWCz2G9QIH\\\\\\\/mUrazbekXUY4haYoYFjNeSBtV4zDuKzr2tQMkNqqkjz\\\\\\\/ZYdGjSAPzn4tiZ04fXLvPqWLMkgKBd1BpT6gJ9vpfnYxyKos4Vi9SLM8jFm3rGRcthYQdCoYJh32HXS2OmYCDjuft4lGoTsEGEaYPoCk38ofh8Tz4SzgUwYBnrJKYOhsZ\\\\\\\/7veFfuXsdvwb7qgGXJd88pdwiG8sI5EPcbpNTDXpwE0eTa+cI0M29B6063Opu6Iu4bOksWskiU83yT+xEwqbDD6LWf+bZ5UouemvlCsEYfdhbx8CipN2WURoLOD2I37rw1yn7b\\\\\\\/8+uyyCt+IwEzwOXR1HV81PVxgd+2ghCYZ+dLZHgV1fnDtLYcqj7LTJf2UIgQ1khIlcWWdHWhPq1Afxh7QpEdJQT6UCX3siL+iVCdFgN8M37h811FZTU\\\\\\\/+Yyzo6gZMQ2DiN0eVV0ZCUcYENhjM6fO1xbzsnRFW2\\\\\\\/qdtOOfUqCcsbfTsu1sdLR0j52Fq+YJQXPEP+dYPiQJQIpEx2QybbA+8CVUkE917eVcl3lwlHDIe1Cc+URaMt6qHQgtrmvAid2lzwpRCmN34nyUjbFv1AbgZXsJ08MZ8wUPDrGnw7J8g\\\\\\\/kaloYrVJHUKUGu8KMa0euUe9867TjVHrqq0zpuAIuqkLqfl\\\\\\\/fdMrdhQQmO4ra9ID\\\\\\\/73GyA+ppAO4QiOdwRkGud36e0nbvz\\\\\\\/lIxcmZ7zGqaKtK4tVxiSox12Z3sUfSyz6I23NsIxCVkKeQEXYS6oaA4sJrsuEtEEL1TV6b8h0FUKGAqRlTzXZZQob0pcNsI8lLa2I3n0QESdeCBHoNR9L58GKq9J6fq3ou\\\\\\\/3BPSEuJAp3RT6bSeLUrpRG9kac16xwqd2JT+godKLXlp4S1T0OcX2OFIIfDxECiVc3u2u1VPfA+H+6j4s4VZLAdu1wJUY\\\\\\\/laWB2Xyfr52YnABfG0p6tTJbPLPcAM0RYugf5pSmQwSsS8Pev\\\\\\\/DIMwIj+4yuTvvpljO3+m5+t3HgU5UO60GFdLe4f6noKvKyuOTnY9KHU49o275YICmH6ZL455xs3pQFQyPXaPvN6TmrfAA6o2HktBaAXieYq36mydTcgiAAjQCqMnr6iZAkE8kS2+fHTA+IB9q0Z\\\\\\\/U6OkGAwyliGK8Y8\\\\\\\/p+EnocYEmou3MDU5vrMVCq156S66v4wYNiZOfcZ2UlBCuhIPkh+KPjpBRThc7ujw30sfOUPCBnjwzy1ub+U3SUUJI3shxyJvBfdFoXYwH3G+JBwtaEAb0Z5P+entP9HDLQtwF7uX9HZAqKIbFLOj3JDRl3p2NWfOBtA3YsS0l3Y+TyoqI++OTPTNbec6yHmTIzdhQn8MACH\\\\\\\/qBECWkFDBrREvQ1Oc2\\\\\\\/pAjfe0OKt94a6ptdgs87ZjVOVSQ4m8XxL2L1QKH43xYmhJbANiLrwOuILdofTCr6bOA3Uv1AGx\\\\\\\/OTpxpWVcyjD44Vz4AYqfmCr+Z\\\\\\\/G97ntoNWBRh0TzJikvHvs\\\\\\\/Ao9cyeYgwqlDciXHjK3UVu9Wkhvi9B3ly1zcWO+3wS1gZcIvB96S6Rk0gbCy8rPahwZ1Zhw8bAnPhCHBGBs\\\\\\\/6k4HiS6DYtx\\\\\\\/K37Xd6aaB6eDEMgOvUrr\\\\\\\/CywlJ4aK9d3ydOIjbTl6GvVVEp2eWMBHNI3gFM3b84Z5MGdlvo\\\\\\\/T7ZI8D1fXo2tkRhGsSu1DZA1z50kd\\\\\\\/mX1PYlxpDs0XJXAd8wZBlK2mcScD\\\\\\\/PHldDTtRRR8o3RaK745OgDu6gWNDef4C8hxtGdotCF6HJLNDfIlEAPvPGTS1WyffKAJXJsvRLWHCM7mrO9Gf+hwBoEocQm682jowshxZPAojIkyf7zQFPFZxg6F18s0YRhIQ75ZyI4\\\\\\\/BhjDdgJKW1gjSQqMxhX48kSeWC3b3jAZMl6hiLzxyFnUsHTNfNFnnbwiMxeIStROmeOXC\\\\\\\/eoYbGfvf4mEAmP5IMQUPQ2FhxANusjPnI3ti8+u+uKFf5KOA5KCxhAjysPECQIk6Ttt7EVCqhfn1qvtJgaiv0vEt6PdTX4n4qvopX\\\\\\\/ZZKS1LPQtZ19GsM2EvqvJ\\\\\\\/q5bq+CXAiQ2WsaX\\\\\\\/+c7JVCvEfv4vGzSeqXEbqmxl+66ltKckAnZucyFlJGZ7fav\\\\\\\/JS6+Cin00pk4rAbx7tHQsGs6fylvi1\\\\\\\/O4zhJ291FziU4Wey\\\\\\\/oW6EfFCcRNJMHkvxQaRa790lHrHpXWInyUSLaDygukcMeQr+b1cuRrfPkvh5tgoPJ8R68qh8d0oLPwuM0t9ynz9Sj13lKDbP5aqY6ATvD+gku2Ot2ARHh0SEUWdhfDEiul4ZmEHDckJ2rVllY7dPpuUE04Rsj6FjZxMSwOO4EEss4t33\\\\\\\/8MnIoMrKnG\\\\\\\/MSp2UMbd5Fa17J\\\\\\\/D7T6sBH9CTt\\\\\\\/fSDCnU7cUyEUN4beSid9RiMznRbRsOCvig8AtSZ9WFTIxPcVhI6xHmotO9DLRHYRU5rg2i\\\\\\\/fegmpf0EAoplep53PR\\\\\\\/bavKvjtARdkPg0ekMv8pekrC3nrpuziiDSDq0CRAq1yg5NvoDT5OCGG8hnXfNBQVG9MkkFrnBsC\\\\\\\/9PmRv2JF95IQPzNOIMpTfVljWLLfsrYcfgnoTZfo3vdFgMJ2H\\\\\\\/7hjmex1GxpxWBCbLp66rjw8xjmihXdZJI6ATxgHAdRR+Y9zq2yACoC8F3sZrAeh+BTT5KvpSiGJwHH2vl53S1yDRugpmLAqA2f6PqbSzzGwkOJ5krzW82SChiVeKdDyxFT6cDXD9Jdm6b\\\\\\\/iCjlccP2sZCfkWUmqEH0BMbl9s8KYKz+AM4o229YrDCXnC6eXKLCPQ4jhux0d7Y1ROgGi+yosak+E7E0jnW9ItJ8c6QD1RD+81EKM+ZcMYeJtokkJMb6BZMkOpiJVAlXYBKulnnGo8L\\\\\\\/t+90dR71Dzo4o\\\\\\\/NH+NXf5NQPt87gbain1S3OuyusjgQMTpuVY19fPl5od67cuU1zBSXsqFiXO5k5H4RDoaKilubAWss9bxLzmUzUZNSR+Bjg9qC+066Qe4C7W8IufohuglHRcPQvg9hmBuoVv6HEj8vAozmv2HfjLmPmn0L+fqsqyB2\\\\\\\/b7A9p+zer9ds9Gxx4EReGy+clSIstA7iap6f0ZIVu6HPwkk6686QcwQwCr1Rh6WY+bDy5jUWjxIbL2E\\\\\\\/FwAPSy5byDFYRR75cRYTJT85gdwmpyPYS08eHf8VSsPhUnvnYqpxjwtRlJOlVIdOXdA80OdZNm5OoD6mwb9glYOS8HadPuYdGGTH1+cSR3iDnCQsNOzbdHxo3z1xhs9BzpHamZSsiDtTtPRKHnMo3y9hwZPNbklW66lHcN8Kl5SWjdXPN7d7ib5MTbdN4JVRPw2Qjhtki0b6p6\\\\\\\/odovEeW3mPATFzYrfrPsiHKU\\\\\\\/IuzG0Osvw0zcjHm+YBZQIKpEtxHXb0C0PBKCM6L7cOH19Q01Idz7tdFzPeh2LeecFnZumJGVLM4bT+JOtqqM5ec+ERFzcpX+mJCXsgqwmNXKWT5R5AcdFGLCP942CU4s2s8hKChWBwko1d6tP6bX+wHx44jR80mqpbb\\\\\\\/t6FgVgid7sZiYScx4y6qOBvO174vWRmNtb3h0ghRC5ZhhB5dM9lCl7\\\\\\\/uagaWUSZj3YayUZS0lQh72AS00uVwlFzD5j8QbzxtoewOdpEoVeAsnrCHL5ePfmmHR3xR0HJg+nsr\\\\\\\/GFWFee2s507j6A4xO3XjizKUtFzkKcIiJhca6dFh3oD+O4Gk7xzQ3WC5n\\\\\\\/RguM5QSIvCKsWkF\\\\\\\/n37vMRWDHGNRz4EyZd\\\\\\\/yUdpir51Imxpeg1cxfXQU4+hC1nxkfwVUwdCyiqs7mOtUuxibvF5LpvWnO76loXcqvqaLMMn3lsVQtWKJN2KAj1\\\\\\\/VVJAABNjy2m6Zf5+UcOtbHstytMEestIkl29QUDQjWrrhbgL8ZXx074l1snDqQey2srj2a9F\\\\\\\/sQegp1mxAgLKvdGF3swri+eQE\\\\\\\/V2tP9iSiS\\\\\\\/usnd+AkdCNQLEX6ehMl7\\\\\\\/JPFL3FMzJ\\\\\\\/KXfu4aPOSczW7SmTcua\\\\\\\/gRPMYO6vJag8ZxjXRE7bHLKOq8nEXt0B\\\\\\\/lh6\\\\\\\/h9oOG1MRPh1woz04jynkk\\\\\\\/PAftjFOGwkx4b+HkmfpYr3qGUOU4F5pK289c3F1ni2dcltQl7B\\\\\\\/TxCIs5WSr70NXgZroNHw2VjCZcxCZa+QloX1+Qsa2C6bjBqZbjpZjC8VE1+oJuHGz27s9d1GJeLLG3eKRxl4WT0pVFQC6uKWFqgEG7HQ3uAWdFVi5kIRRIfPK+7M+oEUcTXqOZyJroWE2K+JAafWz5bxJ9cIKFWaaBZSarcoVx2x9iY+5mjQR2amEfc\\\\\\\/CXUVprOV1czQLvWRRgIF\\\\\\\/+5kjxHitbmQqkhswxvXOVYvDBB+j9\\\\\\\/brc0YGSIeCt\\\\\\\/1v6p9eftCgjYztogzt9B9TAKTM0w+4E6sBm2hIzBk9xrC0vIHMDxbWGazVgxjx23pmTvhFZ3NUrMvAcjo0rJnvdDdbIWr4e\\\\\\\/S1k21kNf7y5aVKyWgsx3wB8652lYGX8PD9d4NBDpH9lxkCTdhPrnpfOLoO8D+G\\\\\\\/+Vh8YxCqZoJ3nifaGFXCRld9UaQadyGrUvOAYeH8p8VPSWItMoP2WJeM49f0a87ro2niKGVThAKYDfQxplJXaBKIF9Yy2eV\\\\\\\/\\\\\\\/NKrmZN9NW4zBqbCdtBDIFYsxZgVCSC5+i6LxCcED8v675FF7R6ZMJUeEYytisZ7IBjt2THlO60R17BxoxIf6uUtupKk2oATTCPXmhN0QIKPidB+zJRt7+2AODQ1u2fXk1bTsP2W\\\\\\\/LISLssMpV+y2QJIRkFeyshsPhXC+5cAAuqE5mb\\\\\\\/L8HoBWSdf4kqPNcxu7rtbiXXJT1xFnvE4H+0h+1vWuOthyx76vUbFhUdMqjqF4tjKFmVnH8iiFYkljHMOo8VoLO9T6jhe63jFNWnfMhz+Js9g7K6QtyptpDASBujhGrrmZV2NQXDSapZplkQ2QCBCjiu91H0jUrHqux9lx1LRyvCrgD6Odu4fT38oL7hWyfylhqxxjrPYyKWut5O68ILr3ptnsCBd56iu1Rs2kcCbdsHZJnWPZhgHQ56XBDUkQYF5br1u6mOzT0bjREXJUi2FVc1z1oFRH4YgHd8hqWvxFd\\\\\\\/rsH5NRjtuXLrbpZKcT\\\\\\\/Oyt1ut9COjP+omFyoDx8doz7hlLF+hnKgzmiWrNdS+zZiUKm6QZjYuRxPXFtzPXqyMjHMrN9zdA2s0\\\\\\\/xNggEBoeKHea\\\\\\\/4iuJmo+qRseUp+csyvQnoc9HjVnS+tL8WKu7R0bUJjpgpkpWg4IMDz7j3OTv+ybT4k7ylxzQUtkcGWMr+hezjtc3Fe0HVs0073nldjsd9f6IwCpC4CvaR8fqm2JmZrpocJEtAC+Y2\\\\\\\/p1+N65CouMWalOGj38qJYirYvOeGF8cdhbyMnL4yGCVz3MEFMsvoE2NFY5TCBc\\\\\\\/yZTj0vd2wGDjD9cMnMoK9SJ6wq45LaD8Av1V+UtC+bBaV3Re8Ixwj6POaXk2JBT370p5fHk3oLr8jIAfoLIbLMzgNVWmZhwHg8FdUVvNCm6ZTlw4aN97TMMvl871nRKGe\\\\\\\/QIvdt8MuzOY8MAK0GVC2DXt0A0d\\\\\\\/Lz6XCVWR\\\\\\\/HulJo3ONxwQaOPWRJwrWRHrfOvnLIHyLvvqP9Z53NoIQkWNBEDGN6JgUA45otdcduGCX9RV\\\\\\\/l12Wx6z860C6vCWUBTcy6ncIfBXIe8LL1IuS\\\\\\\/bqwRYKDA5WzrHXnH8wndTlE4KlhZynZ9so1\\\\\\\/BdxAr9v3xpNdoXaOr1SrTOhpnwu6M2u9lBn6JttsISLCPJ1iHgqL2hynFICD5n7bL+fMzTxBnujOUXn5bZXg409ht7uPGV7DaTm\\\\\\\/qIKn\\\\\\\/zlJkXcP1+oG08noiUSk1Elqy06Cm+5RV1nSrBAqwxKOIRSNF8iOFkKPZ1AFfXMMqQ8ROoBXYLU0xSpxsLHqSMz+wh5ctgT+41tCbVrBRymnRNf0CY33ecyoh3iz0MvfMdgvUQB+oeStwdkLr\\\\\\\/AEld6fgsRaG9d4uJeAFrNU41cxavl+CQHCA8KjBcnw20RpzrrAr+xkxpmCqZrJ3SSmM5quEwF4zeYbdd6dPUkc7fgKHBjTFdn6c4J2HtT7jiH4WJSlx6Rc02gN5SPN8tDgILvopKyQu2FGdUMyZYcvQnvYPeLQnCG3IfaonA\\\\\\\/HH7IMkb2eya7LTAolv44f85qHJmbsritp0G4vkfPMkFYik0kcSDTMx\\\\\\\/6zSFY7vocbpmWbjCUgR0U21Gj83a2nXWNSphABK9+iUhokoe9LMwV2KQBiQaiG6KhTmHGSUheeIK0WOMj0uR7\\\\\\\/j1czbVR3puZE2rVieD7FXtuzZiqPFZS4lm6vu\\\\\\\/6J0JoACFL+s2jqCxziGOKWPXSmfW1abPGtqBu5p7qyBBkW2OvyW\\\\\\\/fZ8S+fNYTW55sT5PwFWXnb7C2PZdKAGSGRYmPVCiwMcy7200U4iEvFdzzmMVtfh0dW1UY4zdxkp\\\\\\\/w9dsTEbiK0EiRwygzXgqX3YRbDgEu3SgD5JZwDsAVDp9K1qIxMCPumYUpWJHUfDxruxNkyrgJJC\\\\\\\/Uq1rR8xez2bsWd6PyrYW\\\\\\\/KWGt1iD0FS0oJhh8AYPoxBxT0XkDDqkDDQP1MaEXhhKMZPD\\\\\\\/xjNwL8aklHexD004uI66TW6UfpMLWWjjFU0uOdLy7ILvRZPJjA8SXO+Rv8HfrT5LSlZkq091077Yz1hFR62KNzqKbAgaVPkU2ovdEEoOjkaYp2LUIhLjv3+6DcmkI8OmFuo+08\\\\\\\/hEIA6Qxa1ipLy77RsV5bucrthQ6ZD9bj3YV5HXH36RFOSWdVXkTfS+O67QFXiSiQIkR28FsH26Ro1zgGpKxb5FVCl0pqbw7WwPZ9Vt1oJLZM5PWTI9VSIHAJIrb\\\\\\\/q22L5qKCSBnEM1TU4pJESamOIaiGjtRVYFOEFaQIq5eYdZjMAdd5rldz7+e6IiyXMr2geTiXQkQE5uzYQ+CO8DplkdGXbtBRaGrkEUQOWWr5D6EW12f3Ko7rogZ+LpEdJDVBxhEh07hxCghJCI5NY29WcIBv4C3dI84rzqEDnVL5b12YuoWmgTmfh\\\\\\\/YGcOxiWIrr\\\\\\\/JhMtSw2v3ZdKcvSZg++LtN6OVtWdbGgGgI9oPaVV8rQLwTcL0H3LJXFmkkTTeWovEyP7iWKkl\\\\\\\/5S8lHER22AbnUk0s7+rR7VD9cRFa8kPCaqwKF9dlgW5i68s29cyuObnrA9zqv00K9Y4JojGPLAYU3XV85sfOiYsVfQqYjxv5Y7X8b0XUBUyNf2V499Rqmw1udgbcPk\\\\\\\/bFGDKo1zb0d6Wc+gdfzkFv3HT1bjqvJF33M66b7vcHFZoRG7IB8Ej+IV1O6CB0KEwCR6C7piQ4sEgSOfdN8BKtuvFpO\\\\\\\/gZgRezSvVaeqcWJ0W\\\\\\\/RPJfOjo6eXiW3XONq6qs7AepDr1jrz4ADEx5ksEc6a33IJ\\\\\\\/UrmPD5ClGidttMSjsVmwAEK0Cobvphw94Hzyv8xpJN3rqM190Nrvseumr3qXxkJtwsGbwcQOxsgevJeuZeaPbumMcCvl0RSX8h2ZuOmXkgB0UBk1tiWEsHPO39GMmHH8NF+TnNLzXuWP3iGJdEWqwRoimTwRHgMWN6JUX9+bHaXT450jXmLRKfP3oHhiUChVtKbC+a0tmRBiyMiJhf+bDBOGAY22eMw\\\\\\\/+aJwhNxshNM8WLc2G9v19X1\\\\\\\/I4HcpN7w+1tKsx\\\\\\\/AoxecOilNsjd6mVBohF\\\\\\\/lHF93HegjElWlW3cuQmEQsIPx\\\\\\\/GHpQ5JS6\\\\\\\/390PXMGzZ3avWWU\\\\\\\/9dEGX0\\\\\\\/6C3abOpfM2nDrALnVBMRB3rgYBDw9VnjvNvy+ViuEgXoUcMuXqKCRWobb1\\\\\\\/hHPacBW6w4g+kW4B2\\\\\\\/tDPqsEWbzQqjnN92PNm3g2DohD9J75uXs4T\\\\\\\/xlsGWTWUTM5hjSvOiFZsyylCqfUHeEeeS1LZj7YdEc0qUq3L9YJIMt8o4h5F6\\\\\\\/2CG0DWAc\\\\\\\/kI38EePNYjgVfT5dJYcp14gZ89M9jPWGDcKcqzBHNz1pKApYQHk5dX3GpbRMeOdnaDAuS7FMc+LKrA5zftGwZyc1FKVtcjcVHPIIHaUf5iRLNIzktP4pnWMeVTPb4UW94JwsArckGHPPc14OWDi3KkxLPg6wi\\\\\\\/pw8NOb6Fqt+bJClu4rXeeTyD46SBAT5gm+gUpxOkTrCEDmHaR4Fk8GDabzNRB74lYlTPJRwr2gu43gZ0IOG\\\\\\\/O8EPB0BH\\\\\\\/rFImKN0sraeD7IliF078mvjoFN+bYHeuLza8VEuTfaGHqUD2pH458CTIXSoJd2udO7kqbN7MbRVNRiU5ChzWGEsOS084xop4qsf08H3zXWmFISu7XdzXLiVmG2jdKk3nYDvA+9AY948e90A\\\\\\\/2VlfELQJdWwmVBfLX+nCDm4M8X26gZZewgmYJIUrtWHUZfH2kyinHHD2BUa99WA0+2eh6LgrB+x80xipP4sWu\\\\\\\/nxFBE99kGI1zOgpdUfcVpXaD3neDD0GaH3rRnpFqqU\\\\\\\/fEdy4fu0eESuSucFnqF\\\\\\\/dEtL9Ljs\\\\\\\/Gay6bUJR6picxjB4Th8ryP+qGyTKzKTdR7vNPmRfJatu9GFT23S1hJ\\\\\\\/BBcuOIDYgopzhjnMwvDNGbRTkgIIAMOiT\\\\\\\/cMYf8stk4cBNOu+7nNPqb7yjpcZ17UvqCaTqazxPil26yWqzoHsOE8S920s1S7v0Iue3ninlu8dnb42yetvoq4iYVZAkzYIZXfzE\\\\\\\/\\\\\\\/bnd273jGMiHyxozZbtrPyJEaQnRAnRE7vLrRJpU1ubqXEvXTEgp8fQqRSv4DYrVrdpmiQt14a+LJhts7WGMtSkQx2krQGX6lU0AJqUkXwHp8\\\\\\\/4Ce4DYJGwJ6Oaao0OUgFYB4oGyHneJlXY6cIaFabVIG+framA9ZGkwbpg\\\\\\\/YDosruxKF0QlTI9wTh8hQ5QYegUrhJHzkyJuHwg4LZAA\\\\\\\/QkCW330V66zl8BLIagKz9HwiFNPNzTtShegUWrakMPO2a8nB3vQIynWH17jUGryRXPiegAGKBqMdYztS4db8NCMlMGQ2KRizPFrEi\\\\\\\/PnDcgmGR+EXAcZoM5ehgg1+92p69III1DnF0\\\\\\\/WYRIeKdvoSr1QiaO1K2llUpnBCLuqJNFqqsFZ04wh6TMut0yChf5vXg\\\\\\\/Pe4+VSanCPmTVtVPt2MACB8+RZa1jGamZEpbgPAu+W3CVpJsyXCLuqhtYGztweec9+yWN7lhWVMKlZVRFMDqPfSLQWljfjZNMZ5S5PEta+w+IoMclJB4lcXCs7ZALKoGMoZSc22WEhGKvElEhimbtSJWRQFgdqh7K6woOX3FyurMueCp1KBjglC8hxf7MlhKBUKWVEjhEfx8WW3GAJVFvrbFZ6ii6+RNC97QgwCgwAwxswcDVS4NXpJeBHunZG264oPv5SMqaahRGP\\\\\\\/F6jcCQnZNXI530w1moKhfh6v5h\\\\\\\/ojTvnaLGDNbWSK2OTARbgn5lzce5HAqFcSmCbW5O150uPVrupX7Ze\\\\\\\/eZykKhMOYpYb+DJZmiGZBn+L+XFL6bi7lz0PYfug7O07seLDEiZL4iGsvDJydcxkLoDNziaVSPyrGkjgCaG\\\\\\\/lAmHrm9viOAN2KyEQ7GncUj\\\\\\\/cXY7xljEVDP48LQPUIPW3H82HOSfDW5+cB6MxeZxhMVonBM52w90or+xt6yz+qITxwRfL5ibzb\\\\\\\/N5bY4FPQjxGqQhbFeE1KeSFY8\\\\\\\/X7NaCPFexMcfEDrb6hj+9F4CNW8F4tbuSHJSbI0vncieeXyVnqFA7ueIlvDue1+xTmxInjMkAc5yXj2NSjrDmtz9XFfPbYTs2D\\\\\\\/UN\\\\\\\/H916tCRXx95VAYoj1IabCOKmRi4L3+VRRDLn4vTgIG+STWkpeU2q0kk+Txe6eDY8EJDDtg03CuFT6QiSr1rxTOmf\\\\\\\/apvedv8zyjHKFLNm6zUHfChOaJXzOuecOsORy2371+zBoxC6qJrGprNwptuuufCYj48rJJ0TBx1Mn\\\\\\\/u1g1Gz9TUF9yerFURQAvdzy7Y15Gmmc0UY9+RdbUoUd8o0KEWwpDFiMK9jypGvhOmFdVTRDTL8\\\\\\\/On\\\\\\\/pb+YgdKVIjftxWmuX\\\\\\\/+z58gETBffZs3IfV8aH25Xee+L5CNHBFz1wjYuOE6FYrHATa9KW8935WRZdTtmvlH8FSVdD8Hc+ObOtMEaJhIfPf7yc7zuEhetlhQWt0TRapcbuYdlVi+myuVGZ9IqQZ90ZKAw0aF8R80lcoapvyyBGyaS9AiG0nb43tYjCzGrahBSc2NSssGynuKYEpURtTpOG1115J8cyw5+tsc+bn4jO2Q+CguRKa6pkOSXaZNez3jralU7pbCIjsFrFYigMC7dMYuIhhEmI2IVW7wapeiRO0Vfc5tNmySzOId2aj8mORcO0IAkEGuy9tbscejC5qT3oucNvmnLtsRKXLDLxec+Aht7MmNUxm49dlG2ztWPd4pTzQ+zW5zAC91oEd6cNX7yMnDQWqckuGLtNlHkjjLNqPd1upo1JTIwya5CO3n5wNnk5mHUvFblI7AEPDajO4ClnNPch++0ZvBvrV17qHl0Mj8b+odUJnkDjxIt7J34yd8MdWgfMCUBGmLrzaccGQ1y2qlEnFPe6vqf5RtCMapU4HxsjmVLBhlHvw2uxi3X1DdK0XMUT50JaKQZ6+Opm4JbcGYvDclxHFm\\\\\\\/YFbAoLSHXpJdUGIq9G5wVxxyfgWlg11Gn1MMksMaqwY1dZLD3q2VVEWGec4MUh2kgS5jUF+6vJq0DCJWYCrI5Gkr2A138smrd96LnMnofVbtzzJ1qAkX4pYZ5lMc4Ny5BELX9ThkKIJIUTP5qQ8UNO4NPabM40j7KBPuU5k9jXVQ6rXbrD+1vmGJmOuWewbPQreijkmqIqpPOhiGpCFnRm76WmnHyu2hPCqNFqVj+v24mDwH\\\\\\\/tbqlCeIbHsfwCAY4ZtZTRFqENTHIdwhTrWDX\\\\\\\/l0+Pd3f5EmK8twCh\\\\\\\/kHlMg2FX+Q0h698gA3Tcdf138TqF4hhcucsOI72UOxz57eqWpZik\\\\\\\/FdIR0Uiy7H967oOVcWYLB2Ukpua\\\\\\\/4lOW5M6Ztt6OwkzDbfnbIYOjO\\\\\\\/Xb\\\\\\\/XKK1GerzHKNPxwJztla4LzwHu\\\\\\\/bz8jvHZqCiNSzezSGlzbhiX8YoEZyYG9YxwQltB8zpEZd+5E4u8TzG\\\\\\\/BaE6T7gdQQIrUUlxY5xSxQsOyT9sEwkKnMAkqQYCpcmoJldHZqkfg1ZZoFmqjFeLGBcnuBGtFp480xxEMG+x0RzT0LNZMQSK5QJpg\\\\\\\/1jIkZdgRAYkdbT3ENmiHUxxWLt7s7tLcNoa11Fav3mPkZtCa5Q9D\\\\\\\/xRqpbpAxhWMA6TdjCWfXjKOgECgZNRYlCaNPYZDISjuWnQ7VCuwROHGwwon9wOcl6Nj19TgdcNfp6BKTXG2g25rJwlxKrTcQDxATApHxefL7biMCjceGb59AvBS3Qw2\\\\\\\/mUCKAqZBMc0lJNq9MCEACe02SP15waYS+7W4g+WwoyR80uajei11LK3m+SxVyQuTCHN48ppw1stGHUOYqwqcukR\\\\\\\/lDvmPsMON1r2aflzSMTBM0+FWSjZe4757egMx8jfFGEAHUjFLMhPTajZqeGHnEtH16aET5ny1Y3tlorECtPoXc1NOlb80pTD0mvyt+Axt9qQd3YYJ1N3WpRtlSVJOcaVehuNrt1tox3GmwDXa1lSAU+A+RA63iNUwIX1QdX+ypKExYz4eT2VjqwHKcObtbKINYOPlmHV2dzxi7lerBsAmst5+6j7eA83TIlxt5KcuvJAq7fTGknq0Oqre1Bk0i1xt7QV6PeopbOoQy0FiKrpp0q48pI02Cuh353O\\\\\\\/4sNx+1c88JELqqi9t3obSIGZesA5HqqA2vwnNs95rHrlHklLOc6Knkhq6\\\\\\\/uMCBngAxPz+byCiMm8jmZ2xn8mzgAHeFIQhX\\\\\\\/s0pw6ovcgcGo+sysTWu2DZjgtuT11EFWYL5pHgggkt1zIrHIyJXRlZKGL6woxV3rUec+bEHyXtC8SWL5n2zmsSa5ZDME7GaHPf8EYr5jWH7KFyfbfepBJsYu+tEZUEe1jqXir65zCc2o9E5ZYPJw5SgNigGfx\\\\\\\/xHFni4400WFIE6EyoAO0fJJ+bS2I1ewsObEsWq1dwvUOgHkcVSWnIuebQewDFYtk1ubt8GyyzkF5rOFwi2yOnQjGjGTAnUMmdYUMMQbQAtDzQkEZqnK6n482TtZ2MXaCSYNjV8rsBK\\\\\\\/Xn\\\\\\\/Y7tbifuY0FAXnqo3dZ+MIl8SrD4TtjHqychabQRJQtr\\\\\\\/eJm8S8QII0MzxQapTGvtzDKzOqePRWlmMMe5Lwp5PFl2yhvpsNB6hEukveAVSyhOTs02FYAyBvJxSFDIQcYgxvKrB5MdT4R19Qa58mrIWYEq8bJFWNPF1qUY13EJWCMBeUIa4NAcbdMzLm86mu2Q2EgivZs3bhw2gaU7mrDcHO4SybWRJjo4h4KcdrgeGbJTFxDQ0aTtSR2Of97E53fsC7Uik6oxsbdjLkAEm4hgtWeTSSY\\\\\\\/yJsTrkDcFnAjSTLYSHz\\\\\\\/nzgIw3Em2ems33QOkV18jOb9n0N\\\\\\\/LHfa6YI6wHDwKbCMpc6vzW3QFjTCgPJkSvt6jUdMNLUtQgKAxHb+0sOeCmrl\\\\\\\/ckxqgSOMvn+gcjTv9DgUyT+\\\\\\\/9EIPz9y6BREZRRyddsopxJo+UAGn\\\\\\\/57+T1QYCKyu95SctbAHgYJvUaBuFgT\\\\\\\/PKGWCFCTU5NNNB2Wq23B6XjOZ1ZkWgXyhN+gDLxMtJkwG0sjvGSFx+dt4RCUUgOcf4O8GyVks8N\\\\\\\/oo4x3VIzE2FPKhT\\\\\\\/\\\\\\\/dE3BFACqXt1hm8ZEktVVReYQ5NRfuQz26RnHDU5REtVSY1rJVB1p4KDTJsKlfrQaM8ITTVj\\\\\\\/cC62uFsATMfG3OxGGwvvUHqopTjoZ5LykNOApBoFzaV23LkHqI\\\\\\\/kVgbt3v9zFqA\\\\\\\/2P3ZXESKnV03GMG7chPpeHmMHXe5ZjkGSH8Wto7hn3SRBEiQuYsFhhxjxBwYUcV0HAVPi2NzrAKXgvofTHs8wTpjmzBboad3SVu9hMb7CjJbdbeoiwuXrl2TV3hKpvNuxAifI58cjwEyh2bHB+gpuhNbALiKYv52DdDJuOdVUEppoY4mUSsNMmlKomBcNxJsSt8yIseV0GnAw7zL\\\\\\\/8vIoAO0giRpeqN9hlghQqMaxSpjzUnBGyWZbEt\\\\\\\/eCkt8wZ\\\\\\\/oQQXQt7wYybZtDpWENZcF1pntBuQFN8x9ehebFH8FagItcdeZ1I4k+nQUmvxWOiHUXuUvHKIwgdEJVQZd4Gbi9o4qOCnqyiLq4b82MiTgA0nLGD7\\\\\\\/SJCEnAWAezOj7+XENlLVMj9w3tJg3MufeE379+zO52k8TgJfXXRn+tUIwvFBTAPs6dB\\\\\\\/pV5KoO9CMezqRkv9J0VWnlv1I6BAajeV9+sf8Ll6DAiz85G+FGaC2Q8uDoiNXk8XuyCSRE7P+WkCmvl69BSFbZXwy1h3tKa1JrGW1Cd55APWpnnJFR7aEfsV4PHzOdwgioXefBRpC4zNKoL7lE6WoD0pNXUPU1zXrGGchKfw2GROpOuMCtq7FgmmJJ7STAKKucMkDgDRZPBEoQL4OE0sKcV+lEc7ZlifgS3ITfDMB4tJjJYSSuRebhMXSLkTR6HBgeezk7JbANZGpUBWuPzXXIB1bZ458TsE1OFAyuWQTKlThmywBNVZCi7Rf9sjT2eHjLiWUQF1sZ4pd5lFe+DpKd0Jhn6GkabTjXeRq5o+\\\\\\\/PDc2Rx6wFEprmHCjkIP8KyzHdt6cPkNLpLQdhUBeSpb8um9JhAXiyF0ZRXDp+fxTRwXaXGXtsUQenFGBct9xP7+j7ojCHbtcrV4SdI1pyMzSrF6ch2j0KA2AqLhu4XT19sJLHP3feb0d1vYba9+BL9iqy2gACS69U4Ghv9dFkqG\\\\\\\/kmkrlSFtQFm9+rSgLsVVHEfdUF2+41Vnd4Z2y7ZTqh1nf5ywdFNUScPF9nGohJk3QvXank0JfwkbyF78XYKXWnRAvniziBfoNWXFZdB4igY7GEFJQOlnBu5eAVDgJnii5npnpDnqC5AURdltX5KuF2HyquZs0+MPdumQjIykPiPnGSv1c4DVJxbxMMTvQo9A97D2xiQeTjxzdDgGHQlWoRBatpjpiR5qAaykpaM\\\\\\\/K424BLjoDYvudTndG3CiLPD1uAljgIm0iFt7Tu0EDBJgNxI5R4MMJ7KFMXELe1Frt6B99OLOHTV02gGxZ+lXXRyiQZ6m8JsMmTVYP8IifQvaAS76Trxl3qbKzzzDNjcVFEdeXuYk0OdG2YCdXXY0cmxFhq7MPuSlvZeKPHWfbWbeLRGxwF7IKAzLX1kub+hen+DOrYIJh3etyxRTWBgg4EbO0Gk4wTs2zW+mxLDYb7q3b8IpFxik4b0aGBqAIE4Nv3HbCBm1ia6BmvOEnENlTFjx9bXl3Ydo6dC6BWKMJYPsVGjkejwBDn6f6\\\\\\\/v65A6cKPmtGRltmVQI+K5aB00KHEQ+x2BWHcFNJBGFYfhCKX43D7\\\\\\\/jRop+X0VZcflRM9YufZK2a8aS2oqEly8\\\\\\\/YbHOx8wbS\\\\\\\/kIidbgwfdzCp+mofxR0hSeVMi8or8YOPs\\\\\\\/F5ZFk+87t8AlqsMSOMuPuQlarDT06P22zAy+ho4ofYUkzo8TSTcsz3txhcdAntn1MfeRFao8qcVUwMNoY8UEmIQaAZuIgnQm6HOAOpY1GCNHsA8d9HGxT9KoUSWAOcWS73s148XtMZnDdhY9GSKA9mZO277HcZi3Mnwkii4LBXmRx+qDiIyN0mOUJ4Hx7FlnyJaa0P6Y5\\\\\\\/8yN9R6qmfqKhgW9a3Xi4e4oSLNODbZiM+9f2qR349pwyBQ8NdX70Iokdr8cwknIOoNLAu0m4SqnwIBudH6HjW7aoosrYrOL+ohFCf8WcjZlFgbcECa4G7mwws9CEdvhYDzsZBYcLvoCAbiocfYvGNBfWiFcil9kD44t7St0rdrmySm828EjppReyeEM7kKidpEa8fHT0Fq\\\\\\\/1RWK3dPkxuji0beopQHLBVZfBwIOErj+wOa7+7LeFLyXTBvaDBesoaUb+0TmqJDibPiNmWMLLSxyX+UbPIAQbVO3MVdSBAlbqDX3kHfcWq+xvzCzCRbR9YjQzajygG4ZLCjJvFxGSU3jrGoQ7wasMoajYbfeT6vHq5V7+IPRka8pyVhHuuNBX\\\\\\\/9152Zj03Uzc5u7AM2J3jNS1zyGFeIUYXwx69cc+a\\\\\\\/2uOTCY5TZkcm2EjJmZug791FnU+Un6ZaNKyvjtn1o8J2Qw0qkZ038HP1MH1BOVRVm6Yy9foam79OQBJegVEsVCafc1h0KoT7xeudRmg1bnGxw9lI16Y9KKRqk1pllauauH1Q+MmkeMFVoirDwLPoowDvDuSKDAsBwMO7G6hVswnPCUwlsTDgVsR9U0TN7ivmnCdfhTispqYo3UsuK5Pfffe8Os9O\\\\\\\/U4s3xguIQUwZ3SXZrV6eqBltXg72c6cUzhShVf1hkF\\\\\\\/rVM3o\\\\\\\/3g4R8ox7pxiLyOrB0lSiqGkJJvpAHi9yR8Dlv7ytvqm0JNO2279hHGKT05l9qA8Na43XTx5fBRtET2b46VIhv9wm\\\\\\\/nAGalBCAYRonTIssH0H0rmDYoECTzgE6+Dj9K\\\\\\\/K9GBRn57jfsdB8h3R2n2TNxfoNV+Kcxu2nf5INTUoVkgGZyO66gqNiN6dnDUkd1ekhNWmYDoy6NvcMpZMww3uwhD9HGVAg4srYiYlX7iRqmzEAJc82I7p1zjcUM89tLancDyEB0yHTKR7lju2pBGoLMUopN4qJLIFvy2jWnRp032mcZ+u4bL9DBog2lDmnqoV+iz02CGMaYm3JT4p3kJqqpkIALrfYJbBByj2ICOl\\\\\\\/CWXWMEzkKveJp16tMegNvqW3IWhI7jFM57TETJ\\\\\\\/mjjyKFO9kc3ZZBnt7pPgu7J4fLEqBfLEzEdne2o56Q548PqdoF51GYr0eXzwmbifaWTlTdn2mBAyjuxpQYXWs2KZ5phGaxr9b3rPc30xyTGQuFArKPWc+qTCx5us2Ky1G5Uk6Kf2s0ls4r1hCYxe1zeIasgHdXj845D4y22Mt17J\\\\\\\/v1y\\\\\\\/2XLbC\\\\\\\/UqgLfSqGjI4T546dtDxUp47GPIgzkMDtvZ5K0NOdGY2YUkVGSNccW4Ux1X2QYEsRg4tDp8v1dJx9h47daQmvGb4u7mn9HXrtEDwiTO6eJdEne8g69sJOpxlTlJ4e6i6QdaO61E8lCaDbSKoGO\\\\\\\/MkvX+ZTWlrvRRn6k8McsSYkL6svkmF99cCdhtMfPWaqiXw3ATVuxiNM6vu0n3MCLG5TPDEqZk+rd+RdKd9CTo4MKGCUzhDrVwo6UZN2idfB1xyfbKqxcwABUjI+NSYHA\\\\\\\/yaOF3ElQOIdG3vj0780fIaSFO9sf82sRL5FqlLtDeN8FpiOm9WuUiA1qTfNFBsXa7BTJiyoYoN6e4eW\\\\\\\/iZ7xj2Mpz9rvQ8tPZMhbOvGN6Iiv0zihhX\\\\\\\/JZzeZLXS0hBzs1TGsg96eSxc1u5RnFaELZ5IUVpKMUv\\\\\\\/nWu+NNn1aHnGqc0yL6DcYTxJ+qNhsNOPrY3aYFRLxHnvRE15Hmc1xUwYAt3owddAB\\\\\\\/mHgK2qUID2XdQ\\\\\\\/5I+hL9aLae\\\\\\\/kiLwQUANHl1ZIks6LPypp4sS4sI8VUFFA4GzXlJTOEcNB38VykRR9r2M16gos0KlSVspabb1PCOiPRnzUNu8FCUs\\\\\\\/wnJctGZc9rzGPBl5BcqDvVq9kpPxkq\\\\\\\/XFB02Vmlz4f6jDrSIUu6kNA0G2NF68y3einNA4OJtGxV80FRYI+kqomntKEKpRURbgeW6ZlhY4dy7zIqwtvfQCFSb6znB+dbGbcEiaWic9lck1vDc4NTXnAmis\\\\\\\/i3s97D3nFdtKhI6RNb1314R6xusIVyNUJwHCHi\\\\\\\/lHkaNuaIPJQBMsj\\\\\\\/JiqIjtKdnjVxcFmDTUoQuygVPOpEUVWugWWpyj4VLbcXIqW2EruaaPfdpd8FZazzlI5wu7\\\\\\\/3i7U1vjjLcgxqVfbGw5TC4qbVluSqQhGgsuaZd+BghlhfjcCKjMR7PqapknQA\\\\\\\/4tNkJmUpp1n+sdULUCweUah+TN7FCB7KM8W9Us9Ws\\\\\\\/pBfrCWjmjEV+TebXsP26wPS6M2u1v6K7\\\\\\\/7b28tcMYnzfcDnlIyISqhcnXEoepTiwlgxLpe+OOdfgXurdfP6CXIXsc8NUoaxAKh+v6AOEYZoBqdcfnH91rR9XtVQr8u4oJsRiXXhOSPQ0B01epVl9A6u0AMHtMtFUZFnsDKuuCGILupEZLcmf0PnBLN2j0YqAIOuBhm\\\\\\\/AbeIds8golsle+A+WrZHwExgSp9PB1qzFsx5uKhGCsYPPhwa4sbnWDOe7dS7BrTUt1E1sjaYrjPlTnEZbu2CZArKzXWZI2\\\\\\\/SloNYN0cdLhXnbJbGcIao6gVoGkBZVmRZbGQfGPmVGz+CSiOTh7fQmfuCZQZ2hOxwLThQqpsiwyp4\\\\\\\/sNX4f3bfggFr78Dyc1VovxDKOw5IcDfQG7imWf0bx8guWqnczjSYqsizbjkvXUoD\\\\\\\/oEx\\\\\\\/XxFr6kxgt1jn86bD1\\\\\\\/tuQZkHKEp1a9VBmobG5CFfkgFiGs2CGg3UhGGZzK3CLutWfeMFm75G0FXCErMe96QXI5mlj8CoEjJZRmm\\\\\\\/WF96GCZYldFX2LgRBur7kcgeLmYVAj65zFaCZnF4wpscnXNEH6SfPQYSfTEr297wMN\\\\\\\/DgUjLEt0kNZLRNdS+k6AdzeXAIUW1QRpN2EXgHN8H4OXfvfgKMP4KS50PyAMlGf0Wh\\\\\\\/XrmoLm9e2KS3B4l3DpChic8LG+Nvquia5Hq+p8tgrpcvdgWEyl3beTaDZ5edeS7RxAW0jULv9PWL+AnQl8yXApZiEVpVNWnxsr0QByizaxtpEnSNp8a\\\\\\\/kbHZ2XzN6sSBwViwRjLG+ENb6DqB\\\\\\\/U5wCLg57cE3li8PPMf0lm8GXeZWjTalWSIMojBzdgq504ZKjxU8wL\\\\\\\/1XL8OHA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MzUhYpGvbDAz7IYH95n52Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002257bec379a22fdc40cbd2ff4f3e98381fa4be389ff858e86d09b3bf093d12bf00\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eKbCXo17y4PBC2HUR\\\\\\\/0HU5Tks6nh6enpcVmKU2lThWVvDLQGghYr6CbgRkBAjtjYYMmW+x7Mg4qy5D1nNtslFAJawfIeT+e3asREKS4wJTsL3FmZgKc3SghGSdhfZKl44YjaF0QdD1BjmTdtzc+usiMsTG8m5y0WhrzfKJxQ4dffSoCylybZh2kWjATTQlKVJUYpMBxhpkYFN0dzOxf5yB9PIgu73G\\\\\\\/yzIwphF2cqIbpkmUTmM4sJSxHFmMuXWbgroJFnkdGUyVSR\\\\\\\/Sk18zVCOERzaOVreWBC5YmVDmJk7W1wnHSicdq5M420VXkQKqr1xe0KaRew\\\\\\\/ozl6OZpM6vEGK28jVOzp\\\\\\\/pRHck4c2VcJzo4\\\\\\\/dbFf00OkQT919Y7CQAMzBzDFu9P+bV9QTaJB5i2iRvmXlBj+EYQYR\\\\\\\/JqklnszpmN2jES0Cymd\\\\\\\/RnSLEMPrn3WUMV\\\\\\\/SRJrvjqCp7FsDTG3zyH+wxUuL73Hr7EN9TEQcP+ZleNXPESMK+Qb1GIHLfHTesNzh4VJpoBLE0HMoEm\\\\\\\/qtzh22j9LL9SpLbyEec++QmH2bb6DOqa4l4KmyXssUNRKOc9Xp+yxyNMYISU1sUv0xXyR1vvgvgeR3rGnDpmkuSRIvQhsQIE32WjAOokbV+Bro5gujDDwelwt1KsLGzH1RcltmFVv\\\\\\\/gZ8OcHRCPnNv30XMua\\\\\\\/8OHRzATfS5H6bQnmOV28+1zRpPlWEWa0A+GZD30Cg5tUnKhvF\\\\\\\/YccWXfOqD6\\\\\\\/UQVUpS9dKNKmvejK3+E9q54i7\\\\\\\/kiKlWbaqYuPxy3goDZC+8a06VjCGOXSKlKx\\\\\\\/\\\\\\\/+pG659slU5uxi\\\\\\\/lyF7GxZgkX76\\\\\\\/37baWFWd0dSrs6vvaaAM\\\\\\\/0uTRyN2FMQaI\\\\\\\/F5z6pPTzBfIXs63TkS3C19WnlWdF4E7F2p9v8G1GFOBRLmr1MLgZzE8yftleCOUGrHvBvsplrmDifC7JoJMzaT7C0ZVlb3QUOqC1YlXaolbJZzE3gTHqF0WBZSO3yTMsip6WxjW0jy6ZPMMU\\\\\\\/YCbIV+iQOGxtLJt8jvUuCHtrt6IrtiQpyjO\\\\\\\/OIDRp18vvP42oUZvpO6N+AkzCiSuXr+Zlj7xuT4RJpBzjTRgZRYRfR9tNy2JwXwD5mGVnWq\\\\\\\/kMdJjzOxkvJHmdgg65spgKycKI1GeBkKx6MU8H0yuL19zP2BuO1ecpNZSsp0XcXquYCVXmXrykAdmcFS9E0v6k1CeyC83tbtupJjxHV2Fb4qTAYrDLXfD3AhOlP6iCCjO13AABJX4q2ltjw95tGD0svSEhjhH1PGyKYrAb\\\\\\\/wfBj1mYSmi+nPoB\\\\\\\/dcN9ffR\\\\\\\/nykqay4Tpc4i57\\\\\\\/MMv5LmBNcSdqwDg7dFTfWmwtyp6b33MpjunkpxhimNxi\\\\\\\/S1YfM2YV6z9FRmq8SHQzLAMXUjUHoTJm02P8oRUAtoXKCjGnPem0AkiS8VzKs+4Jmua1wUT9abWb5m8+FFNgxjEGMFfrhnzaRwBYESFj2CB18meR3Jvw5u7XMwZ8kaeZ9ZCzb1Li0Fyu+Cb7aEId4DWoF6xCERDlOOyI+zkPIWbBjfxFMFQparT3DgnBM9tPUUImveW0fPHqBIshxMbcrrsLaHTBm4asajZF1kyn3AFUG2Rv9+5E4KBYQrw2LSc7FDdZ2kvVgHR+Q+ONnt\\\\\\\/R2jDNNppEPCtJaLd3BhOp3wRiuqEVWBgeCsO+c4KscceJrZejTg+6h165Eb1Four7PgDcqDj0X9dXUpJRzjEOdoXXfqMbHaJ0u1gOMA5UqysdNjUnrDX4B49IMa1QMD+0Pq9k0U9vSHsM9H9X5vGUrC7XKnycY0kpCVayLy3jL0Pj9f4q6yHwwGGISC54e4AaS05r\\\\\\\/HQygApSaeC3S9kCOywoYaZW358Vuovs0IhEtP77cLAZ+y0I8LVmBqIriRztnkblsVtrL657UUWQQ4vYsnCnKZENkbLL66W5lw\\\\\\\/K2mwntK48XTqfOf57gVYdTDihJp9bJmgEqs8twranAiBU0uy29Hqaw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BeWtUdmC9wiBQXQPhrd0iw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e95b0a94f33d2bdfa5f28565d2dc51f204ab1aec56f6ba78c3beb556cd9adf4f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490961 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3163" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3BOxki6GBAO-HaNpiBs9ZP7zfxvJD_RGhadDdClcEnI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3163" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jBCJaAcLUsJye1az7CJRADZjfRGAwqU8ASa1UYis0k4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3155" title="Mapping Quantitative Trait Loci for Tolerance to Pythium irregulare in Soybean (Glycine max L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3173" title="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490961" id="node490961" data-pisa="ggg;8/10/3163" data-pisa-master="ggg;g3.118.200461" data-apath="/ggg/8/10/3163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1407-0643" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philippe C Després</span>, <span class="highwire-citation-author" data-delta="1">Alexandre K Dubé</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3236-651X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lou Nielly-Thibault</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1582-6027" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nozomu Yachie</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3028-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian R Landry</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3163-3171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200461 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490961" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Philippe C Després </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BC%2BDespr%C3%A9s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Despr%C3%A9s%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilippe%2BC%2BDespr%25C3%25A9s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1407-0643" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philippe C Després</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandre K Dubé </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BK%2BDub%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dub%C3%A9%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BK%2BDub%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lou Nielly-Thibault </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lou%2BNielly-Thibault%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielly-Thibault%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALou%2BNielly-Thibault%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3236-651X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lou Nielly-Thibault</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nozomu Yachie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Research Center for Advanced Science and Technology, Synthetic Biology Division, University of Tokyo, Tokyo, 4-6-1 Komaba, Meguro-ku, 153-8904, Japan</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Tokyo, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Advanced Biosciences, Keio University, Tsuruoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>PRESTO, Japan Science and Technology Agency, Tokyo, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nozomu%2BYachie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yachie%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANozomu%2BYachie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1582-6027" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nozomu Yachie</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christian R Landry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BR%2BLandry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landry%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BR%2BLandry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3028-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian R Landry</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.landry@bio.ulaval.ca" class="" data-icon-position="" data-hide-link-title="0">christian.landry@bio.ulaval.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3163" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490961" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490961/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3163.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490961" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490961/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3163.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490961" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490961/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3163.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490961" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490961/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3163.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490961" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490961/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">CRISPR-Cas9 loss of function (LOF) and base editing screens are powerful tools in genetics and genomics. Yeast is one of the main models in these fields, but has only recently started to adopt this new toolkit for high throughput experiments. We developed a double selection strategy based on co-selection that increases LOF mutation rates using the Target-AID base editor. We constructed the pDYSCKO vector, which is amenable to high throughput double selection experiments, and show that the improvement in Target-AID efficiency generalizes across loci. Using modeling, we show that this improvement in efficiency provides the required increased in detection power to measure the fitness effects of thousands of mutations in typical yeast pooled screens. We show that double selection can also improve Cas9 mediated LOF rates, but that this multiplex genome editing causes programmable chromosomal translocations at high frequency. This suggests that multiplex LOF editing should be performed with caution and that base-editors could be preferable tools for some screens in yeast. Base editing using double selection is simple and straightforward and provides an alternative to homology directed repair based high throughput variant strain construction methods.</p></div><span class="kwd-group-title">Keywords:</span><ul class="kwd-group"><li class="kwd"><a href="/keyword/yeast-genome-editing" class="hw-term hw-article-keyword hw-article-keyword-yeast-genome-editing" rel="nofollow">Yeast Genome editing</a></li><li class="kwd"><a href="/keyword/crispr-cas9" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9" rel="nofollow">CRISPR-Cas9</a></li><li class="kwd"><a href="/keyword/target-aid-base-editing" class="hw-term hw-article-keyword hw-article-keyword-target-aid-base-editing" rel="nofollow">Target-AID Base editing</a></li><li class="kwd"><a href="/keyword/loss-function-screens" class="hw-term hw-article-keyword hw-article-keyword-loss-of-function-screens" rel="nofollow">Loss of function screens</a></li><li class="kwd"><a href="/keyword/chromosomal-fusions" class="hw-term hw-article-keyword hw-article-keyword-chromosomal-fusions" rel="nofollow">Chromosomal fusions</a></li></ul><p id="p-3">The CRISPR-Cas9 genome editing system has been engineered to create a comprehensive experimental toolkit (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jinek <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Qi <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sander and Joung 2014</a>). One of the principal applications of the system is for gene loss of function (LOF) via DNA repair errors within the coding sequence of a gene after creating targeted double stranded breaks (DSBs). The scalability of this technique allowed for the development of CRISPR-Cas9 LOF experiments at the genome scale (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sanjana <em>et al.</em> 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bassett <em>et al.</em> 2015</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sidik <em>et al.</em> 2016</a>), greatly facilitating systems biology and genomics experiments in models in which they were difficult so far.</p><p id="p-4">Despite yeast being a usual frontrunner in technological developments for systems biology, low CRISPR-Cas9 LOF efficiency in yeast (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">DiCarlo <em>et al.</em> 2013</a>) has so far made the application of this approach impractical. In the same experiments, mutagenesis using CRISPR-Cas9 and donor DNA via the Homology Directed repair pathway (HDR) was shown to be very efficient, but technical challenges prevented large-scale applications until recently. Both the MAGESTIC workflow (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Roy <em>et al.</em> 2018</a>) and the CHAnGE workflow (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bao <em>et al.</em> 2018</a>) rely on mass co-synthesis of the guide sequence and of donor DNA to perform pooled mutagenesis of thousands of variants at the same time in haploid cells. While these methods show remarkable promise, they may be limited for some applications. As synthesis error rates increase with fragment length, these methods require additional experimental resources (<em>e.g.</em>: supplementary NGS sequencing) to compensate for the high error rates both in guide sequences and donor DNA. The high toxicity of DSBs (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Roy <em>et al.</em> 2018</a>) also enriches the pooled populations for cells in which genome editing was not successful, further increasing the amount of sequencing resources required to accurately assess the impact on fitness of engineered mutants. Furthermore, these two workflows require a haploid background, which is rare in wild yeast: many wild and industrial strains are diploid and would therefore require genetic manipulation before they can be used in these high throughput experimental workflows. Therefore, a promising alternative would be to rely on direct base editing that does not depend on double stranded DNA cuts and repair.</p><p id="p-5">Newly developed base editors based on nCAs9 and dCas9 fusions, such as Target-AID (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nishida <em>et al.</em> 2016</a>), which induces C to G and C to T changes (with rare C to A), can also be used to create targeted gene LOF by using mutations to generate premature stop codons. While it has been shown that most yeast genes could be theoretically be inactivated using base editing (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Billon <em>et al.</em> 2017</a>), there has yet to be an experimental assessment of its capabilities as a tool for high throughput screening, even if it has been shown to be highly effective in mammalian cells (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kuscu <em>et al.</em> 2017</a>). Target-AID toxicity in yeast has been shown to be far lower than that of Cas9, but its mutagenesis rate remains lower than those of HDR/donor DNA mediated methods.</p><p id="p-6">One of the crucial determinant of large-scale pooled LOF screens is mutagenesis efficiency, which will influence the sequencing resources that have to be invested to detect fitness effects. Accordingly, different approaches have been used to optimize mutagenesis rate. One of them is co-selection, which relies on the selection of cells based on a marker that has mutated alongside the target loci that is positively or negatively selected. This was recently shown to enhance Cas9 edition rates in human cell lines (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agudelo <em>et al.</em> 2017</a>) and drosophila (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kane <em>et al.</em> 2017</a>), but has not been applied to yeast yet. Co-selection has also be shown to enhance base-editing efficiency in human cell lines (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Billon <em>et al.</em> 2017</a>). We developed a double selection system in yeast to test if it could increase Target-AID mutagenesis efficiency. We found that this double selection achieves a near 100% mutagenesis rate at some target sites using a vector amenable to genome-wide screening. In early studies, LOF mutagenesis based on DSB repair errors was shown to be highly inefficient in yeast (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">DiCarlo <em>et al.</em> 2013</a>). We revisited these experiments using our double selection strategy and found that this method can be used to increase over a thousand-fold Cas9 LOF editing efficiency, at the cost of inducing frequent chromosomal translocations between the two cut sites.</p><p id="p-7">While it is intuitive that a higher efficiency would allow for more efficient screening as well as the detection of more subtle effects on fitness, it is not clear by how much we need to improve editing efficiency to be able to perform experiments in yeast using CRISPR-Cas9 LOF or base editing so that they would compare in terms of power with those currently performed with the yeast deletion collection. We therefore also developed a statistical model to quantitatively assess the improvement brought by double selection on CRISPR-LOF screens and use it to show that it is highly significant, particularly for fitness effects in the range in which most of gene deletion growth phenotypes are observed across multiple conditions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and methods</h2><div id="sec-2" class="subsection"><h3>Experimental procedures</h3><p id="p-8">The pDYSCKO vector was built from p426-SNR52p-gRNA.CAN1.Y-SUP4t (Addgene # 43803; all plasmids used in this study are presented in table S1). Directed mutagenesis was used to create silent mutations in the AmpR gene and the <em>URA3</em> gene to remove BsaI restriction sites using the oligonucleotides mut_AmpR_For/mut_AmpR_Rev and mut_URA3_For/mut_URA3_Rev (all oligonucleotides used in this study are presented in Table S2). The DYSCKO cassette was synthetized as a gBLOCK (Integrated DNA Technologies, Coralville, USA) and Gibson cloned into the modified backbone amplified with Backbone_For and Backbone_Rev. The g.ADE1 ssDNA oligonucleotide containing the gRNA targeting the <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> gene (as well as the <em><a href="https://www.yeastgenome.org/locus/S000003690">VPS35</a></em> and KanMX4 targeting guides) was amplified by PCR using the gRNA_for and gRNA rev oligonucleotides. This target in <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> was previously used by Nishida <em>et al.</em> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Nishida <em>et al.</em> 2016</a>) to create LOFs using Target-AID and is also well suited to CRISPR-Cas9 gene disruption. Amplification reactions were pooled and purified using the EZ-10 Column PCR Products Purification Kit (Biobasic, Markham, Canada). Fragment concentration was estimated using a NanoDrop (Thermofisher, Waltham, USA). Guides targeting <em><a href="https://www.yeastgenome.org/locus/S000005046">POL1</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000005458">SDD3</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000005798">YTM1</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000004636">ARP9</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000002711">RSC3</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000003777">ILV3</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000000039">CDC24</a></em> were obtained as dsDNA from Arbor Biosciences (Ann Arbor, USA). The inserts were cloned into pDYSCKO using the Golden Gate Assembly Mix (New England Biolabs, Ipswich, USA) with the following parameters: 1 ul pDYSCKO vector (50 ng/ul), 1 ul insert (∼0.5 ng/ul) using the standard manufacturer assembly reaction conditions, which was incubated for 1 hr at 37° followed by 5 min at 55°. The ligated plasmids were transformed in bacteria and positive clones were confirmed by Sanger sequencing (CHUL sequencing platform, Québec, Canada) using the DYSCKO_rev or DYSCKO_For oligonucleotide. Plasmid p415-Cas9 was obtained from Addgene (#43804). The nCas9-Target-AID plasmid was a generous gift from Dr Keiji Nishida, Kobe University, Kobe, Japan.</p><p id="p-9">All experiments were performed in the haploid <em>S. cerevisiae</em> BY4741 strain (<em>MATa his3Δ1 leu2Δ0 met15Δ0 ura3Δ0</em>) or diploid BY4743 (<em>MATa/α his3Δ1/his3Δ1 leu2Δ0/leu2Δ0 LYS2/lys2Δ0 met15Δ0/MET15 ura3Δ0/ura3Δ0</em>) from <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baker Brachmann <em>et al.</em> (1998)</a>, except for the KanMX4 Cas9 perturbation experiments, which were performed in the ∆<em>HO</em> strain from the Yeast Deletion Collection, which was streaked on YPD+G418 (200 µg/ml) after being thawed. A single colony was then used to inoculate a culture for the competent cell protocol.</p><p id="p-10">Synthetic media recipes are detailed in Table S4. All yeast growth occurred at 30° and with shaking in the case of liquid cultures. Competent cells and transformations were performed using standard protocols (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amberg <em>et al.</em> 2005</a>), with a two-hour recovery period. Cells were plated on SC-UL and allowed to grow for 48 hr at 30° before the start of mutagenesis experiments.</p><p id="p-11">The same protocol was used for all mutagenesis experiments. Briefly, multiple transformation colonies were used to inoculate a 3 ml SC-UL + 2% glucose culture in which cells were grown for 24 hr. Enough cells to inoculate at 1 OD<sub>600</sub> a 3ml culture were harvested by centrifugation and placed in SC-UL+5% glycerol for 24 hr. Cas9 or Target-AID expression was then induced by switching the media to 3 ml SC-UL+ 5% galactose for a 12-hour period. Cells were diluted to OD of 0.1 in 3 ml SC-ULR +2% glucose + Canavanine (50 µg/ml) for a 16-hour double selection or 3 ml SC-UL +2% glucose at the same OD as the control condition.</p><p id="p-12">Mutation rates (through loss of function) were assessed by plating cells on SC-ULR + Can and SC-ULR after galactose induction and after recovery with or without double selection at appropriate dilutions. Mutation rate was assessed by calculating the ratio of red (or pink) colonies over total number of colonies. The <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> and <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> alleles of 8 white and 8 red colonies were sequenced to confirm mutations either through Cas9 or Target-AID with CAN1_for, CAN1_rev and ADE1_for, ADE1_rev. LOF rates for the KanMX4 gene measured by resuspending colonies plated on SC-ULR+Can after recovery sterile water, and then spotting 5 µl of the resuspension on YPD and YPD + G418 (200 µg/ml). Growth on YPD but not on YPD+G418 was used to detect LOF of the KanMX4 cassette.</p><p id="p-13">Mutations in essential genes and the non-essential control were also Sanger sequenced (CHUL sequencing platform, Québec, Canada), using the appropriate oligonucleotides (Table S2) and using DNA extracted directly from 100 µl of liquid culture using the Quick DNA method (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lõoke <em>et al.</em> 2017</a>). A custom Python script was used to calculate the intensity of mutant <em>vs.</em> wild-type peaks on the chromatogram. Briefly, peak locations were extracted from the phred file, and the area under the curve for each nucleotide was calculated in a small window around this position. Intensities were normalized by the total area under the curve at that window, and these values were then used either as is for a heat map or to calculate a mutant/wild-type peak ratio. The titration curve shown in figure S2 was generated by mixing different ratios of cells (based on OD measurements) with known genotypes and then performing a quick DNA extraction. The target site in <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> was then amplified twice and sequenced using the same primers previously described.</p><p id="p-14">The CAN1_For and ADE1_Rev oligonucleotides was used to amplify the <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a>-<a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> breakpoint and the resulting amplicon was Sanger sequenced (CHUL sequencing platform, Québec, Canada). Fusions with <em><a href="https://www.yeastgenome.org/locus/S000003690">VPS35</a></em> were engineered using the same approach but with pDYSCKO-<em><a href="https://www.yeastgenome.org/locus/S000003690">VPS35</a></em>. Fusions were detected by PCR using the VPS35-A and CAN1_rev oligonucleotides, and the resulting amplicons were Sanger sequenced (CHUL sequencing platform, Québec, Canada). Sequences were aligned using the MEGA 7 software (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kumar <em>et al.</em> 2016</a>).</p><p id="p-15">Chromosomal rearrangements were confirmed by PFGE (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Maringele and Lydall 2006</a>). Plugs were prepared using cells from overnight cultures in YPD for BY4741 and SC-ULR + canavanine for mutant strains. Migration time was 27 hr, with a switch time of 60 seconds and no ramping.</p></div><div id="sec-3" class="subsection"><h3>LOF screen model and simulations</h3><p id="p-16">In CRISPR-LOF screens, populations of cells are transformed with pools of vectors bearing different gRNAs that are amplified by PCR and used as barcode in a barcode-sequencing competition experiment (Bar-seq) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Shalem <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Robinson <em>et al.</em> 2014</a>), the gRNAs being associated to genes by sequence identity. We modeled this process by adapting haploid selection models, with the added challenge that not all cells with a specific guide will bear a LOF at the target locus because mutation rate is not 100%.</p><p id="p-17">Let <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-1.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-2.gif"/></noscript></span></span> be respectively the abundance of a control guide and the abundance of a guide targeting a gene which when inactivated has a selection coefficient <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-3.gif"/></noscript></span></span>. This mutation is present in a proportion μ, which represents the mutation rate after recovery and thus the relative frequency at the beginning of the competition. After a time t, the abundance ratio of these two barcodes is described by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-4.gif"/></noscript></span></span>Were <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-5.gif"/></noscript></span></span> is the absolute fitness of the control barcode population at time t and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-6.gif"/></noscript></span></span> the absolute fitness of the barcode associated with the mutation. Because the rate of mutation is not 100% for the strains harboring the barcode (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-7.gif"/></noscript></span></span>), a certain proportion of cells (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-8.gif"/></noscript></span></span>) will have the same fitness as cells bearing the control barcode so that:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-9.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-10.gif"/></noscript></span></span>By dividing both the numerator and the denominator of the rightmost factor by <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-11.gif"/></noscript></span></span>, we obtain<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-12.gif"/></noscript></span></span>And then to<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-13.gif"/></noscript></span></span>Because we assume that the growth rates of the mutants and wild-type are constant over time:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-14.gif"/></noscript></span></span>Given that the selection coefficient <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-15.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-16.gif"/></noscript></span></span> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hartl 2000</a>) we have<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-17.gif"/></noscript></span></span>And therefore<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-18.gif"/></noscript></span></span>Which is simplified to<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-19.gif"/></noscript></span></span>If we assume that the control barcode and the mutant are present in the same proportion at the start of the experiment, then <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-20.gif"/></noscript></span></span> and<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-21.gif"/></noscript></span></span>This gives us a theoretical population ratio at time t for the control barcode over the barcode for a mutant, as a function of the mutagenesis rate and the selection coefficient over a generation. Because we observe these ratios by sequencing the barcode pool at a given depth, the number of reads for each barcode will be influenced by stochastic sampling. If we have k guides in the experiment, and sequence at an estimated depth of d reads on average per guide, and we assume that all barcodes have equivalent abundance at the start, then the number of reads <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-22.gif"/></noscript></span></span>sequenced for a barcode at <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-23.gif"/></noscript></span></span> follows<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-24.gif"/></noscript></span></span>Because of our previous assumptions, this distribution is valid at <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-25.gif"/></noscript></span></span> for all barcodes regardless of μ or <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-26.gif"/></noscript></span></span>. For other values of t, if we assume that the distribution remains equivalent for the control barcodes, then R for a LOF barcoded strain will follow a binomial distribution influenced by the abundance change caused by the LOF of the target allele:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-27.gif"/></noscript></span></span>Where<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-28.gif"/></noscript></span></span>We can therefore obtain simulated sequencing read number distributions as a function of μ, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-29.gif"/></noscript></span></span>, t, k, and d. To assess whether a difference in read count for a barcode is significant, we can test for equality of proportion between a control barcode and the barcode of interest using the chi-square test, with the following table:</p><div id="T1" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-1"><col width="47.88%" span="1" id="column-1"/><col width="52.12%" span="1" id="column-2"/><thead id="thead-1"><tr id="tr-1"><th valign="middle" align="center" scope="col" rowspan="1" colspan="1" id="th-1" class="table-center table-vmiddle"><span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-30.gif"/></noscript></span></span></th><th valign="middle" align="center" scope="col" rowspan="1" colspan="1" id="th-2" class="table-center table-vmiddle"><span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-31.gif"/></noscript></span></span></th></tr></thead><tbody id="tbody-1"><tr id="tr-2"><td valign="middle" align="center" scope="row" rowspan="1" colspan="1" id="td-1" class="table-center table-vmiddle"><span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-32.gif"/></noscript></span></span></td><td valign="middle" align="center" rowspan="1" colspan="1" id="td-2" class="table-center table-vmiddle"><span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/T1/embed/mml-math-33.gif"/></noscript></span></span></td></tr></tbody></table><div class="table-foot"></div></div></div><p id="p-18">Such experiments include multiple control gRNA that can be pooled in the same category. Most of the time, there are also multiple guides targeting the gene. However, because Target-AID can be used to create specific point mutations that may not all be equivalent to LOF and may not have the same efficiency, it is useful to consider all guides individually. If we set the number of guides per target loci to g, the number of target loci to t, and the number of control c, then the total sequencing depth for a set d changes because:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-34.gif"/></noscript></span></span>Because we expect all controls to be equivalent, then we can assume that<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-35.gif"/></noscript></span></span>And for a single target<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-36.gif"/></noscript></span></span>We used this model to simulate a CRISPR-LOF screen with parameters analogous to a Bar-seq screen in yeast. We draw simultaneously a sample from each distribution and test the values for significance over 10 000 iterations. This was done for 100 increments of μ from 0 to 1 and 100 increments of <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-37.gif"/></noscript></span></span> from 0 to 1.</p><p id="p-19">The yeast genome contains ∼6,000 annotated ORFs, of which 1,156 are essential and not useful for a standard LOF screen. Control loci could include intron sequences, pseudogenes or predicted putative non-functional peptides, or often used controls like the HO nuclease. Therefore, to simulate a CRISPR-LOF screen in yeast, the following parameters were used <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-38.gif"/></noscript></span></span> (number of LOF target loci), <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-39.gif"/></noscript></span></span> (number of control guides), <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-40.gif"/></noscript></span></span> (number of guides per target loci), <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-41.gif"/></noscript></span></span> (expected average coverage per guide) and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-42.gif"/></noscript></span></span> (number of generations). The CRISPR-Cas9 LOF simulation was performed using a script written in Python 2.7 using the Ipython/Jupyter interactive environment (v4.1.1) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Perez and Granger 2007</a>), Numpy (v1.13.3) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">van der Walt <em>et al.</em> 2011</a>), Pandas (v0.20.3) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">McKinney 2010</a>), Matplotlib (v2.1.0) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hunter 2007</a>) and SciPy (v0.13.3) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Oliphant 2007</a>). Fitness effects of gene deletions in Bar-seq experiments were retrieved from Qian <em>et al.</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Qian <em>et al.</em> 2012</a>) and pooled across experiments. To generate <a id="xref-fig-3-1" class="xref-fig" href="#F3">figure 3b</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">figure 3c</a>, we used the same model, but used the fitness effects detected in the YP1_replicate of the Qian <em>et al.</em> dataset. We then calculated the detection success rate for μ values between 0 and 1 in 0.01 increments, using the same significance threshold and over a thousand iterations. For figure Sx3c, the expected coverage per guide parameter was changed to <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3163/embed/mml-math-43.gif"/></noscript></span></span>, but the rest of the parameters remained the same.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-20">The pDYSCKO vector will be deposited on Addgene (<a href="http://www.addgene.org/">http://www.addgene.org/</a>), as well as pDYSCKO-ADE1. The Python 2.7 Packages used are available through in the Python Package Index (<a href="https://pypi.python.org/pypi/pip">https://pypi.python.org/pypi/pip</a>) or in Conda (<a href="https://conda.io/docs/">https://conda.io/docs/</a>). Plasmids used in this study are listed in Table S1 and oligonucleotides in Table S2. A summary of the guides used in this study and their targets is presented in Table S3. Synthetic media recipes are presented in Table S4. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6936683">https://doi.org/10.25387/g3.6936683</a>.</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><p id="p-21">The double selection system uses the pDYSCKO (Double Yeast Selection CRISPR-KO) plasmid, which encodes for two guide RNAs (gRNA), one targeting the gene of interest (g.YFG) and the other the negative selection market <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Another vector is used for galactose induction of the effector enzyme, which can be different variants of Cas9 or Cas9-fusions. LOF of <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> confers resistance to the antibiotic canavanine, which means that canavanine media can be used to enrich the cell population for mutant cells.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The pDYSCKO-CAN1 plasmid for LOF double selection. a) Overview of co-selection using the pDYSCKO and the effector plasmid, expressing guide RNAs and the editing enzyme respectively. Only a fraction of cells are successfully edited during induction of the effector enzyme (galactose media), but cells with a LOF at one locus have a greater likelihood of bearing LOFs at both loci. Canavanine selection allows for enrichment for these cells. b) Overall structure of the DYSCKO cassette, which is on a plasmid with standard selection markers (AmpR and URA3). The two symmetrical gRNA expression units use the same promoters and terminators (from SNR52 and SUP4 respectively), insuring their co-expression. The stuffer is a short sequence containing two restriction sites for BsaI that does not match any sequence in the yeast genome. Custom gRNA insertion in the vector is performed through Golden Gate assembly (Engler et al. 2008) using a short dsDNA fragment containing the gRNA targeting the sequence of interest (g.YFG)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919119458" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The pDYSCKO-&lt;em&gt;CAN1&lt;/em&gt; plasmid for LOF double selection. a) Overview of co-selection using the pDYSCKO and the effector plasmid, expressing guide RNAs and the editing enzyme respectively. Only a fraction of cells are successfully edited during induction of the effector enzyme (galactose media), but cells with a LOF at one locus have a greater likelihood of bearing LOFs at both loci. Canavanine selection allows for enrichment for these cells. b) Overall structure of the DYSCKO cassette, which is on a plasmid with standard selection markers (AmpR and URA3). The two symmetrical gRNA expression units use the same promoters and terminators (from SNR52 and SUP4 respectively), insuring their co-expression. The stuffer is a short sequence containing two restriction sites for BsaI that does not match any sequence in the yeast genome. Custom gRNA insertion in the vector is performed through Golden Gate assembly (Engler &lt;em&gt;et al.&lt;/em&gt; 2008) using a short dsDNA fragment containing the gRNA targeting the sequence of interest (g.YFG).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3163/F1.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3163/F1.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491029" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">The pDYSCKO-<em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> plasmid for LOF double selection. a) Overview of co-selection using the pDYSCKO and the effector plasmid, expressing guide RNAs and the editing enzyme respectively. Only a fraction of cells are successfully edited during induction of the effector enzyme (galactose media), but cells with a LOF at one locus have a greater likelihood of bearing LOFs at both loci. Canavanine selection allows for enrichment for these cells. b) Overall structure of the DYSCKO cassette, which is on a plasmid with standard selection markers (AmpR and URA3). The two symmetrical gRNA expression units use the same promoters and terminators (from SNR52 and SUP4 respectively), insuring their co-expression. The stuffer is a short sequence containing two restriction sites for BsaI that does not match any sequence in the yeast genome. Custom gRNA insertion in the vector is performed through Golden Gate assembly (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Engler <em>et al.</em> 2008</a>) using a short dsDNA fragment containing the gRNA targeting the sequence of interest (g.YFG).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">We estimated mutation rates using a gRNA that inactivates the <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> gene by a C to G mutation that results in a premature stop codon near the start of the coding sequence. LOF of <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> leads to a red colony phenotype, allowing for direct LOF rate estimation after plating. We compared conditions with and without canavanine double selection after galactose induction. Target-AID LOF rates showed a median fold increase of 3 (median mutation rate: 61%) in haploid cells, and a median fold increase of 4.2 in diploid cells (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>), showing that our co-selection approach can indeed enrich a cell population for successful edition events. This LOF rate is equivalent to the mutation rate measured at the end of the MAGESTIC workflow (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Roy <em>et al.</em> 2018</a>), but lower than the estimated rate in the CHAnGE workflow (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bao <em>et al.</em> 2018</a>). As Target-AID performs C to T and C to G edition with rare C to A (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Nishida <em>et al.</em> 2016</a>), and the <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> targeting guide only inactivates <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> if the mutation is C to G (Figure S1), we suspected that the actual mutation rate at the target site might be higher than the LOF rates. Sanger sequencing of white and red colonies confirmed this hypothesis, with all white colonies sequenced bearing a S2L mutation that had no impact on colony phenotype, meaning that the perturbation rate might be higher. If the only goal is perturbation of the target nucleotide (eg.: catalytic site LOF) to change the encoded amino acid, then double selection can achieve near 100% mutagenesis at a target site and thus achieves the same efficiency as HDR/donor DNA methods. To demonstrate the usefulness of double selection, we used it to perform mutagenesis at multiple loci, with a focus on targeting essential genes. To avoid limiting sample size and thus biasing mutation rate assessment, we directly sequenced DNA from cell pools to semi-quantitatively measure allelic frequency in the population. We validated this approach using a titration curve based on variable mutant/wild-type ratios at the <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> locus (Figure S2), which showed that if anything the this method has a tendency to underestimating mutagenesis rates. Mutant/wild-type ratios showed variation in efficiency but systematically show in increased efficiency with the double selection, with on average a twofold improvement (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2c</a>). While none of these guides matched the efficiency of the <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> test, mutational outcomes of essential gene mutagenesis might cause fitness defects or be lethal, which would lower overall efficiency as only unedited cells could survive. For example, mutagenesis using the guide targeting the essential gene <em><a href="https://www.yeastgenome.org/locus/S000003777">ILV3</a></em> can either result in a premature stop codon (C to G, S142*) or a substitution (C to T, S142L), as was the case for the guide targeting <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em>. One would therefore expect the C to G mutation to be the most abundant, emulating the pattern observed for the other guide. However, as <em><a href="https://www.yeastgenome.org/locus/S000003777">ILV3</a></em> is essential, this mutation should prove to be lethal. As expected, the C to G mutant is entirely absent after recovery (Figure S3); the observed mutation rate is therefore both dependent on mutagenesis success rate and on the effect on fitness of the mutation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Double selection increases base editing efficiency. a) The same cell cultures were placed in different recovery conditions after induction of Target-AID expression. Recovery in glucose is neutral with respect with the CAN1 mutant genotype. Recovery in canavanine selects for cells that have LOF mutations at the CAN1 locus. Selection on the CAN1 marker increases the representation of ADE1 LOF colonies. Cells with non-functional ADE1 alleles accumulate a red metabolic intermediate that gives a characteristic color to colonies, allowing for easy identification of mutants, while mutants without LOF mutations in ADE1 have a wild-type phenotype. Plating was performed after a 16-hour recovery period. b) Double selection increase mutagenesis rates across multiple target loci. The same cell cultures were placed either in glucose or canavanine media for recovery after the same mutagenesis protocol, with guides targeting different loci (represented by different colors), with mutations at different possible sites (represented by different shapes). Mutation rates were estimated using Sanger sequencing chromatograph peak intensities (see Methods)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919119458" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Double selection increases base editing efficiency. a) The same cell cultures were placed in different recovery conditions after induction of Target-AID expression. Recovery in glucose is neutral with respect with the &lt;em&gt;CAN1&lt;/em&gt; mutant genotype. Recovery in canavanine selects for cells that have LOF mutations at the &lt;em&gt;CAN1&lt;/em&gt; locus. Selection on the &lt;em&gt;CAN1&lt;/em&gt; marker increases the representation of &lt;em&gt;ADE1&lt;/em&gt; LOF colonies. Cells with non-functional &lt;em&gt;ADE1&lt;/em&gt; alleles accumulate a red metabolic intermediate that gives a characteristic color to colonies, allowing for easy identification of mutants, while mutants without LOF mutations in &lt;em&gt;ADE1&lt;/em&gt; have a wild-type phenotype. Plating was performed after a 16-hour recovery period. b) Double selection increase mutagenesis rates across multiple target loci. The same cell cultures were placed either in glucose or canavanine media for recovery after the same mutagenesis protocol, with guides targeting different loci (represented by different colors), with mutations at different possible sites (represented by different shapes). Mutation rates were estimated using Sanger sequencing chromatograph peak intensities (see Methods).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3163/F2.medium.gif" width="440" height="180"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3163/F2.medium.gif" width="440" height="180"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491041" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Double selection increases base editing efficiency. a) The same cell cultures were placed in different recovery conditions after induction of Target-AID expression. Recovery in glucose is neutral with respect with the <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> mutant genotype. Recovery in canavanine selects for cells that have LOF mutations at the <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> locus. Selection on the <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> marker increases the representation of <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> LOF colonies. Cells with non-functional <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> alleles accumulate a red metabolic intermediate that gives a characteristic color to colonies, allowing for easy identification of mutants, while mutants without LOF mutations in <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> have a wild-type phenotype. Plating was performed after a 16-hour recovery period. b) Double selection increase mutagenesis rates across multiple target loci. The same cell cultures were placed either in glucose or canavanine media for recovery after the same mutagenesis protocol, with guides targeting different loci (represented by different colors), with mutations at different possible sites (represented by different shapes). Mutation rates were estimated using Sanger sequencing chromatograph peak intensities (see Methods).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">In a typical CRISPR-LOF screen, pools of gRNA are cloned and transformed in a population of cells. The fate of individual mutants is followed through competition assays in which gRNA sequences serve as barcode to estimate the abundance of each genotype individually by deep sequencing, which is analogous to yeast Bar-seq approaches with the yeast deletion (YKO) collection (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Robinson <em>et al.</em> 2014</a>). In this type of experiments, one main factor that limits the detection of fitness effects is the depth at which the barcode pool is sequenced. Low mutagenesis efficiency will increase the sequencing resources required to detect fitness effects, potentially making the experiment prohibitively expensive. In the yeast deletion collection, all cells bear a synthetic barcode linked to the gene that has been deleted. This is the same for CRISPR-LOF screens, because the population of cells bearing a specific gRNA are be a mixture of mutants and WTs, which cannot be distinguished from one another by sequencing barcodes alone. The signal observed for a mutant will therefore be a function of the effect of the mutation and the mutation rate within each population bearing the gRNA or associated barcode. A screen using base editors follows the same principles. Lower mutation rates will require increases in sequencing depth to detect the same fitness effects. Using existing data on Bar-seq-based fitness measurements from the YKO collection as a gold standard, we can examine how the improved efficiency of gene editing afforded by the double selection can increase the detection power of fitness effects.</p><p id="p-26">Our simulations show that the increased efficiency allows for more sensitive detection of LOF effects (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3a</a>), particularly in the critical zone encompassing fitness effects between 0% and 25%, which are the most frequent effects for the deletion of non-essential genes in yeast. The double selection strategy therefore makes genome-wide screening powerful enough to meet the high standards established by the yeast deletion collection (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3b</a>). For example, at a mutagenesis rate of 0.2, a LOF with a selection coefficient of 0.05 has a 25% chance of being detected, while a rate of 0.6 instead allows it to be detected in over 99% of cases. As visible on the heat map and on the recovery curves, a mutagenesis efficiency above 0.4 appears to be the minimal threshold at which large-scale applications can be considered at a 100x per target depth of sequencing. The model also predicts that the efficiency of Target-AID editing in diploid cells would be high enough to consider large-scale screening applications but would require increasing the coverage twofold (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3c</a>). Of course, such screens often rely on relatively error-prone pooled synthesis of the gRNA sequences, meaning that a significant fraction of cells within the population can bear a guide that cannot mutate the intended target. As such, additional sequencing resources are needed to compensate, something for which our model does not account. Because these experiments would be feasible in diploid cells and because our plasmid selection markers on our vectors can easily be changed to antibiotic resistance cassettes for G418 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wach <em>et al.</em> 1994</a>), Nourseothricin or Hygromycin B (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Goldstein and McCusker 1999</a>), double selection after Target-AID could allow for high throughput marker less gene disruption in non-standard laboratory and wild strains.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Improvement in mutagenesis rates enhance the power of CRISPR-LOF screens. a) The rate of detection of fitness defects is shown as a function of mutagenesis rate and the selection coefficient of the mutation. Improving the mutation rate above 0.3 dramatically increases the ability to detect growth defects, a requirement in many experiments including genome-wide CRISPR LOF barcode sequencing. The improvement is particularly noticeable for selection coefficient associated to the majority of gene deletion (YKO) mutants in various conditions (Qian et al. 2012), as shown in the panel above. b) Fraction of yeast genes for which a significant (P &lt; 0.05) fitness effect is detected in our model after LOF based on their YKO fitness coefficients in YPD as a function of mutagenesis rate in a haploid population. The different curves represent different minimal success rates (over a 1000 iterations) used as thresholds to decide whether or not a fitness effect was detected for a given gene at a given mutagenesis rate. c) Same as b), but in a diploid population. Because both alleles of the target must be mutated, higher coverage and higher mutagenesis rates are required to detect the same fitness effects." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919119458" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Improvement in mutagenesis rates enhance the power of CRISPR-LOF screens. a) The rate of detection of fitness defects is shown as a function of mutagenesis rate and the selection coefficient of the mutation. Improving the mutation rate above 0.3 dramatically increases the ability to detect growth defects, a requirement in many experiments including genome-wide CRISPR LOF barcode sequencing. The improvement is particularly noticeable for selection coefficient associated to the majority of gene deletion (YKO) mutants in various conditions (Qian &lt;em&gt;et al.&lt;/em&gt; 2012), as shown in the panel above. b) Fraction of yeast genes for which a significant (&lt;em&gt;P&lt;/em&gt; &lt; 0.05) fitness effect is detected in our model after LOF based on their YKO fitness coefficients in YPD as a function of mutagenesis rate in a haploid population. The different curves represent different minimal success rates (over a 1000 iterations) used as thresholds to decide whether or not a fitness effect was detected for a given gene at a given mutagenesis rate. c) Same as b), but in a diploid population. Because both alleles of the target must be mutated, higher coverage and higher mutagenesis rates are required to detect the same fitness effects.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3163/F3.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3163/F3.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491150" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Improvement in mutagenesis rates enhance the power of CRISPR-LOF screens. a) The rate of detection of fitness defects is shown as a function of mutagenesis rate and the selection coefficient of the mutation. Improving the mutation rate above 0.3 dramatically increases the ability to detect growth defects, a requirement in many experiments including genome-wide CRISPR LOF barcode sequencing. The improvement is particularly noticeable for selection coefficient associated to the majority of gene deletion (YKO) mutants in various conditions (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Qian <em>et al.</em> 2012</a>), as shown in the panel above. b) Fraction of yeast genes for which a significant (<em>P</em> &lt; 0.05) fitness effect is detected in our model after LOF based on their YKO fitness coefficients in YPD as a function of mutagenesis rate in a haploid population. The different curves represent different minimal success rates (over a 1000 iterations) used as thresholds to decide whether or not a fitness effect was detected for a given gene at a given mutagenesis rate. c) Same as b), but in a diploid population. Because both alleles of the target must be mutated, higher coverage and higher mutagenesis rates are required to detect the same fitness effects.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">After observing that double selection could enhance base editing efficiency, we hypothesized it could also help enhance LOF rates by Cas9 via errors in DSB repairs. We therefore performed the same assay as for Target-AID mutagenesis but using Cas9 as the effector enzyme instead. Double selection increased the Cas9 LOF rates over 1200-fold to a median mutation rate of 62% (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>). Experiments using multiple other guides targeting the KanMX4 resistance cassette showed LOF rates as high as 94% and as low as 34% (n = 3 for each guide, 32 colonies tested per replicate), consistent with the known variance in guide effectiveness depending on sequence context.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multiplex CRISPR-Cas9 causes chromosomal rearrangements in yeast. a) Double selection increases LOF rates by Cas9 mutagenesis in yeast. The same experiment as in figure 2a was performed. Red and pink colonies were considered to bear a LOF. b) Chromosomal migration patterns of the different canavanine resistant mutants generated using Cas9. Mutants without LOF mutations in ADE1 have the same phenotype as the wild-type strain. Colonies with a distinct pink color show chromosomal fusions. CAN1 only LOF (W1 and W2) and red CAN1, ADE1 LOF mutants show a chromosomal migration pattern that is comparable to the wild-type. Pink colonies from independent mutagenesis experiments show altered length for chromosomes I and chromosomes V, consistent with a reciprocal translocation centered on the predicted Cas9 cut sites. b) Breakpoint sequencing of 5 pink colonies from 3 independent mutagenesis experiments confirms PFGE patterns." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-919119458" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Multiplex CRISPR-Cas9 causes chromosomal rearrangements in yeast. a) Double selection increases LOF rates by Cas9 mutagenesis in yeast. The same experiment as in figure 2a was performed. Red and pink colonies were considered to bear a LOF. b) Chromosomal migration patterns of the different canavanine resistant mutants generated using Cas9. Mutants without LOF mutations in &lt;em&gt;ADE1&lt;/em&gt; have the same phenotype as the wild-type strain. Colonies with a distinct pink color show chromosomal fusions. &lt;em&gt;CAN1&lt;/em&gt; only LOF (W1 and W2) and red &lt;em&gt;CAN1&lt;/em&gt;, &lt;em&gt;ADE1&lt;/em&gt; LOF mutants show a chromosomal migration pattern that is comparable to the wild-type. Pink colonies from independent mutagenesis experiments show altered length for chromosomes I and chromosomes V, consistent with a reciprocal translocation centered on the predicted Cas9 cut sites. b) Breakpoint sequencing of 5 pink colonies from 3 independent mutagenesis experiments confirms PFGE patterns.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3163/F4.medium.gif" width="440" height="147"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3163/F4.medium.gif" width="440" height="147"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3163/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491129" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Multiplex CRISPR-Cas9 causes chromosomal rearrangements in yeast. a) Double selection increases LOF rates by Cas9 mutagenesis in yeast. The same experiment as in <a id="xref-fig-2-3" class="xref-fig" href="#F2">figure 2a</a> was performed. Red and pink colonies were considered to bear a LOF. b) Chromosomal migration patterns of the different canavanine resistant mutants generated using Cas9. Mutants without LOF mutations in <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> have the same phenotype as the wild-type strain. Colonies with a distinct pink color show chromosomal fusions. <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> only LOF (W1 and W2) and red <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em>, <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> LOF mutants show a chromosomal migration pattern that is comparable to the wild-type. Pink colonies from independent mutagenesis experiments show altered length for chromosomes I and chromosomes V, consistent with a reciprocal translocation centered on the predicted Cas9 cut sites. b) Breakpoint sequencing of 5 pink colonies from 3 independent mutagenesis experiments confirms PFGE patterns.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Most importantly, our experiment also revealed that multiplexed gene targeting for double-strand break with CRISPR-Cas9 in yeast has undesirable side effects. We recurrently observed colonies exhibiting an intermediate phenotype for <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> LOF (median rate: 18%), with a color pattern slightly different from the standard red <em>ade1</em> phenotype. We found that these cells harbored a chromosomal rearrangement that results into a <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a>-<a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> fusion. We first identified these fusions by PCR and Sanger sequencing and confirmed that they affect chromosome size by PFGE (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">4c</a>). Changing the DYSCKO target locus from <em><a href="https://www.yeastgenome.org/locus/S000000070">ADE1</a></em> to <em><a href="https://www.yeastgenome.org/locus/S000003690">VPS35</a></em> produced a different fusion but at a similar rate (Figure S4), showing that these fusions are likely programmable and generalizable. The observation that multiplex genome editing that does not rely on HDR integration of donor DNA causes chromosomal rearrangements at relatively high frequency is worrisome because many recent multiplex mutagenesis workflow rely on PCR based deep sequencing of the target allele to measure success rates (<em>i.e.</em>, CRISPR-Seq (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tothova <em>et al.</em> 2017</a>)) of genome editing at multiple loci. This method cannot detect chromosomal rearrangements, and as such the events may have gone unnoticed in many experiments so far. This ability of the CRISPR-Cas9 system has been known for some time (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Maddalo <em>et al.</em> 2014</a>). It should be noted that the high occurrence rate of chromosomal translocations might be specific to yeast, but efforts should be made to assess it in other models, even if this it made harder by the absence of an easily detectable phenotype. Recently, cleavage by Cas9 was shown to result in large deletions and chromosomal rearrangements at the target site in mouse and human stem cells (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kosicki <em>et al.</em> 2018</a>). Our findings may therefore be generalizable, with features specific to each cell type.</p><p id="p-31">As we have yet to observe any case of rearrangements when using Target-AID, base editors might be safer choices in most cases when attempting multiplex LOF mutagenesis. Base editor have already been shown to be viable alternatives to Cas9 based LOFs for high throughput screening based on mutations that creating stop codons (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Kuscu <em>et al.</em> 2017</a>). One possible tradeoff with these editors is that some of the induced changes may lead to silent substitutions, such as the ones we observed here, which effectively limit the effective LOF rate. Our approach relies on the <em><a href="https://www.yeastgenome.org/locus/S000000789">CAN1</a></em> gene that can be used for negative selection against non-mutated cells. This marker has been used in many large-scale and systematic experiments, for instance synthetic gene arrays (SGA) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tong <em>et al.</em> 2001</a>), which means that the edited strains are compatible in terms of genotype with a broad range of methods routinely used by the yeast community. If needed, other negatively selectable markers could be exploited, for instance <em>LYP1</em>, <em>FCY1</em> and <em>URA3</em>. The development of other base editors, for instance from A-T to G-C (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gaudelli <em>et al.</em> 2017</a>), will also offer new possibilities in terms of producing loss of function mutations without DNA cleavage. Base editing capabilities will also increase as new Type II CRISPR/Cas9 are adapted (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Verwaal <em>et al.</em> 2018</a>) or engineered for added versatility (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kleinstiver <em>et al.</em> 2015</a>), increasing their versatility. Base editing in yeast can become a powerful tool for yeast systems biology that can complement existing high throughput approaches while opening new experimental avenues.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-32">The authors thank G. Charron for help with the PFGE protocol, and M. Hénault and D. Yamamoto Evans for insightful discussions. P.C.D and A.K.D performed the experiments. P.C.D and A.K.D designed research with support from C.R.L and N.Y. P.C.D developed the model with L.N.T. P.C.D analyzed the data and wrote the manuscript with contributions from all authors. This work was supported by the Canadian Institutes of Health Research [299432, 324265 and 387697 to C.R.L. 364920, 384483 and Frederick Banting and Charles Best graduate scholarship to P.C.D.], the Natural Sciences and Engineering Research Council [Alexander Graham Bell graduate fellowship to L.N.T.], Université Laval [André Darveau Fellowship to P.C.D.] and the Japan Society for the Promotion of Science [ S15734 and S17161 to C.R.L. and N.Y.].</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-33">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6936683">https://doi.org/10.25387/g3.6936683</a>.</p></li><li class="fn" id="fn-2"><p id="p-34"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-29"><span class="received-label">Received </span>May 29, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-30"><span class="accepted-label">Accepted </span>July 30, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Despres <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.1" data-doi="10.1038/nmeth.4265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agudelo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duringer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bozoyan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huard</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Marker-free coselection for CRISPR-driven genome editing in human cells.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4265</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D615%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4265&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3163.2"><div class="cit-metadata unstructured">Amberg D. C., D. J. Burke, and J. N. Strathern, 2005 Methods in Yeast Genetics: A Cold Spring Harbor Laboratory Course Manual, 2005 Edition.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.3" data-doi="10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker Brachmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caputo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Designer deletion strains derived fromSaccharomyces cerevisiae S288C: A useful set of strains and plasmids for PCR-mediated gene disruption and other applications.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBrachmann%26rft.auinit1%253DC.%2BB.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D115%26rft.epage%253D132%26rft.atitle%253DDesigner%2Bdeletion%2Bstrains%2Bderived%2Bfrom%2BSaccharomyces%2Bcerevisiae%2BS288C%253A%2Ba%2Buseful%2Bset%2Bof%2Bstrains%2Band%2Bplasmids%2Bfor%2BPCR-mediated%2Bgene%2Bdisruption%2Band%2Bother%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%252819980130%252914%253A2%253C115%253A%253AAID-YEA204%253E3.0.CO%253B2-2%26rft_id%253Dinfo%253Apmid%252F9483801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(19980130)14:2&lt;115::AID-YEA204&gt;3.0.CO;2-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9483801&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071817800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.4" data-doi="10.1038/nbt.4132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">HamediRad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tasan</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome-scale engineering of Saccharomyces cerevisiae with single-nucleotide precision.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">508</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.4132</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D36%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.5" data-doi="10.1016/j.jgg.2015.03.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassett</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A genome-wide CRISPR library for high-throughput genetic screening in Drosophila cells.</span> <abbr class="cit-jnl-abbrev">J. Genet. Genomics</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jgg.2015.03.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%2BGenomics%26rft.volume%253D42%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jgg.2015.03.011%26rft_id%253Dinfo%253Apmid%252F26165496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jgg.2015.03.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26165496&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.6" data-doi="10.1016/j.molcel.2017.08.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Billon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryant</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nambiar</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR-Mediated Base Editing Enables Efficient Disruption of Eukaryotic Genes through Induction of STOP Codons.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1079.e4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2017.08.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2017.08.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2017.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.7" data-doi="10.1093/nar/gkt135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DiCarlo</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norville</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">4336</span>–<span class="cit-lpage">4343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt135%26rft_id%253Dinfo%253Apmid%252F23460208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23460208&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318167900046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.8" data-doi="10.1371/journal.pone.0003647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kandzia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marillonnet</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A one pot, one step, precision cloning method with high throughput capability.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3647</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003647</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DEngler%26rft.auinit1%253DC.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De3647%26rft.epage%253De3647%26rft.atitle%253DA%2Bone%2Bpot%252C%2Bone%2Bstep%252C%2Bprecision%2Bcloning%2Bmethod%2Bwith%2Bhigh%2Bthroughput%2Bcapability.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003647%26rft_id%253Dinfo%253Apmid%252F18985154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003647&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18985154&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.9" data-doi="10.1038/nature24644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaudelli</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komor</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rees</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Packer</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badran</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Programmable base editing of A•T to G•C in genomic DNA without DNA cleavage.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">551</span>: <span class="cit-fpage">464</span>–<span class="cit-lpage">471</span>. <span class="cit-comment">Erratum 2018 Jul doi: 10.1038/s41586-018-0070-x. 10.1038/nature24644</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D464%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24644%26rft_id%253Dinfo%253Apmid%252F29160308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29160308&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.10" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption inSaccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3163.11"><div class="cit-metadata unstructured">Hartl D. L., 2000 A Primer of Population Genetics. Sunderland, Mássachussets. U.S.A Sinauer Assoc. Inc. 26–31.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.12" data-doi="10.1109/MCSE.2007.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Matplotlib: A 2D Graphics Environment.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/MCSE.2007.55</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D9%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.55%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.13" data-doi="10.1126/science.1225829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jinek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chylinski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fonfara</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-comment">A Programmable Dual-RNA-Guided DNA Endonuclease in Adaptive Bacterial Immunity. Science 337: 816–821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1225829</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1225829%26rft_id%253Dinfo%253Apmid%252F22745249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5Ni84MTYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.14" data-doi="10.1534/g3.116.036723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vora</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varre</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padgett</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficient Screening of CRISPR/Cas9-Induced Events in Drosophila Using a Co-CRISPR Strategy.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.036723</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036723%26rft_id%253Dinfo%253Apmid%252F27793971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjcvMS84NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.15" data-doi="10.1038/nature14592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kleinstiver</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prew</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topkar</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Engineered CRISPR-Cas9 nucleases with altered PAM specificities.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">523</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature14592</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14592%26rft_id%253Dinfo%253Apmid%252F26098369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26098369&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.16" data-doi="10.1038/nbt0918-899c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosicki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomberg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Repair of double-strand breaks induced by CRISPR–Cas9 leads to large deletions and complex rearrangements.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.4192</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D36%26rft.spage%253D765%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0918-899c%26rft_id%253Dinfo%253Apmid%252F30010673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0918-899c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30010673&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.17" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.18" data-doi="10.1038/nmeth.4327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuscu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parlak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tufan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szlachta</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR-STOP: gene silencing through base-editing-induced nonsense mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">712</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4327</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D710%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4327%26rft_id%253Dinfo%253Apmid%252F28581493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28581493&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.19" data-doi="10.2144/000114497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lõoke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristjuhan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristjuhan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Extraction of genomic DNA from yeasts for PCR-based applications.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">v</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2144/000114497</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D62%26rft.spage%253Dv%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000114497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000114497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.20" data-doi="10.1038/nature13902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maddalo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manchado</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Concepcion</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonetti</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidigal</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">In vivo engineering of oncogenic chromosomal rearrangements with the CRISPR/Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">516</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">427</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13902</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D516%26rft.spage%253D423%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13902%26rft_id%253Dinfo%253Apmid%252F25337876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25337876&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.21" data-doi="10.1385/1&amp;ndash;59259&amp;ndash;958&amp;ndash;3:065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maringele</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydall</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Pulsed-field gel electrophoresis of budding yeast chromosomes.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1385/1–59259–958–3:065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DMaringele%26rft.auinit1%253DL.%26rft.volume%253D313%26rft.spage%253D65%26rft.epage%253D73%26rft.atitle%253DPulsed-field%2Bgel%2Belectrophoresis%2Bof%2Bbudding%2Byeast%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1385%252F1%2526ndash%253B59259%2526ndash%253B958%2526ndash%253B3%253A065%26rft_id%253Dinfo%253Apmid%252F16118425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/1&amp;ndash;59259&amp;ndash;958&amp;ndash;3:065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16118425&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3163.22"><div class="cit-metadata unstructured">McKinney W., 2010 Data Structures for Statistical Computing in Python. 51–56.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.23" data-doi="10.1126/science.aaf8729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishida</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arazoe</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yachie</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakimoto</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Targeted nucleotide editing using hybrid prokaryotic and vertebrate adaptive immune systems.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">563</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aaf8729</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf8729%26rft_id%253Dinfo%253Apmid%252F27492474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTMvNjMwNS9hYWY4NzI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.24" data-doi="10.1109/MCSE.2007.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliphant</span>  <span class="cit-name-given-names">T. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Python for scientific computing.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/MCSE.2007.58</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D9%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.25" data-doi="10.1109/MCSE.2007.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granger</span>  <span class="cit-name-given-names">B. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">IPython: A System for Interactive Scientific Computing.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/MCSE.2007.53</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D9%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2007.53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2007.53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.26" data-doi="10.1016/j.cell.2013.02.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2013.02.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D152%26rft.spage%253D1173%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.02.022%26rft_id%253Dinfo%253Apmid%252F23452860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.02.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23452860&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315710300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.27" data-doi="10.1016/j.celrep.2012.09.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Genomic Landscape and Evolutionary Resolution of Antagonistic Pleiotropy in Yeast.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1399</span>–<span class="cit-lpage">1410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2012.09.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BRep%26rft.aulast%253DQian%26rft.auinit1%253DW.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D1399%26rft.epage%253D1410%26rft.atitle%253DThe%2Bgenomic%2Blandscape%2Band%2Bevolutionary%2Bresolution%2Bof%2Bantagonistic%2Bpleiotropy%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.09.017%26rft_id%253Dinfo%253Apmid%252F23103169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.09.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23103169&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314457700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.28" data-doi="10.1534/g3.113.008565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Design and analysis of Bar-seq experiments.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008565</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.113.008565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.29" data-doi="10.1038/nbt.4137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vonesch</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multiplexed precision genome editing with trackable genomic barcodes in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">512</span>–<span class="cit-lpage">520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.4137</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D36%26rft.spage%253D512%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.30" data-doi="10.1038/nbt.2842"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sander</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joung</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CRISPR-Cas systems for editing, regulating and targeting genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2842</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2842%26rft_id%253Dinfo%253Apmid%252F24584096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2842&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24584096&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.31" data-doi="10.1038/nmeth.3047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanjana</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shalem</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improved vectors and genome-wide libraries for CRISPR screening.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">784</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3047</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D783%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3047%26rft_id%253Dinfo%253Apmid%252F25075903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25075903&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340075600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3163.32"><div class="cit-metadata"><cite><span class="cit-comment">Shalem O., N. E. Sanjana, E. Hartenian, X. Shi, D. A. Scott, <em>et al.</em>, 2014 Genome-scale CRISPR-Cas9 knockout screening in human cells. Science 343: 84–87. https://doi.org/10.1126/science.124700510.1126/science.1247005</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.33" data-doi="10.1016/j.cell.2016.08.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sidik</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huet</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganesan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">M.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Genome-wide CRISPR Screen in Toxoplasma Identifies Essential Apicomplexan Genes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1423</span>–<span class="cit-lpage">1435.e12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.08.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1423%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.019%26rft_id%253Dinfo%253Apmid%252F27594426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27594426&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.34" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1065810</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.35" data-doi="10.1016/j.stem.2017.07.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tothova</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krill-Burger</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popova</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landers</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">Q. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multiplex CRISPR/Cas9-Based Genome Editing in Human Hematopoietic Stem Cells Models Clonal Hematopoiesis and Myeloid Neoplasia.</span> <abbr class="cit-jnl-abbrev">Cell Stem Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">555.e8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.stem.2017.07.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BStem%2BCell%26rft.volume%253D21%26rft.spage%253D547%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2017.07.015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stem.2017.07.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.36" data-doi="10.1002/yea.3278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verwaal</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buiting-Wiessenhaan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalhuijsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roubos</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">CRISPR/Cpf1 enables fast and simple genome editing of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.3278</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D35%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.3278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.3278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.37" data-doi="10.1002/yea.320101310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pöhlmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippsen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">New heterologous modules for classical or PCR-based gene disruptions in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1793</span>–<span class="cit-lpage">1808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.320101310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWach%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D13%26rft.spage%253D1793%26rft.epage%253D1808%26rft.atitle%253DNew%2Bheterologous%2Bmodules%2Bfor%2Bclassical%2Bor%2BPCR-based%2Bgene%2Bdisruptions%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320101310%26rft_id%253Dinfo%253Apmid%252F7747518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320101310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7747518&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3163.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QJ42800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3163.38" data-doi="10.1109/MCSE.2011.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Walt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colbert</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varoquaux</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The NumPy Array: A Structure for Efficient Numerical Computation.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/MCSE.2011.37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BSci.%2BEng.%26rft.volume%253D13%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMCSE.2011.37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MCSE.2011.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3163.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3155" title="Mapping Quantitative Trait Loci for Tolerance to Pythium irregulare in Soybean (Glycine max L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3173" title="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3163" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PJ9f4ICnl0Qc51tiqqp_WrybQp06slnPEG8YA35qJxI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3163" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3163" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3163" class="active" data-icon-position="" data-hide-link-title="0">Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490961" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-jOsJp0CApac0XQI9nCTMr7GuCjjDkHR0gN8s5BVzxak" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3163.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3163" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-aDDLFU1YnJacU1ae6oEnXr1usJAEV7ec56xcXAbLbsA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3163" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3163.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490961" id="node490961--2" data-pisa="ggg;8/10/3163" data-pisa-master="ggg;g3.118.200461" data-apath="/ggg/8/10/3163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1407-0643" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philippe C Després</span>, <span class="highwire-citation-author" data-delta="1">Alexandre K Dubé</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3236-651X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lou Nielly-Thibault</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1582-6027" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nozomu Yachie</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3028-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian R Landry</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3163-3171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200461 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490961--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Philippe C Després </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BC%2BDespr%C3%A9s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Despr%C3%A9s%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilippe%2BC%2BDespr%25C3%25A9s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1407-0643" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philippe C Després</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandre K Dubé </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BK%2BDub%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dub%C3%A9%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BK%2BDub%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lou Nielly-Thibault </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lou%2BNielly-Thibault%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielly-Thibault%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALou%2BNielly-Thibault%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3236-651X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lou Nielly-Thibault</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nozomu Yachie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Research Center for Advanced Science and Technology, Synthetic Biology Division, University of Tokyo, Tokyo, 4-6-1 Komaba, Meguro-ku, 153-8904, Japan</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Tokyo, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Advanced Biosciences, Keio University, Tsuruoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>PRESTO, Japan Science and Technology Agency, Tokyo, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nozomu%2BYachie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yachie%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANozomu%2BYachie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1582-6027" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nozomu Yachie</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christian R Landry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BR%2BLandry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landry%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BR%2BLandry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3028-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian R Landry</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.landry@bio.ulaval.ca" class="" data-icon-position="" data-hide-link-title="0">christian.landry@bio.ulaval.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3163" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3163&amp;title=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3163&amp;title=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3163&amp;title=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3163&amp;text=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3163&amp;title=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3163&amp;t=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3163&amp;title=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3163&amp;title=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490961/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490961" id="node490961--3" data-pisa="ggg;8/10/3163" data-pisa-master="ggg;g3.118.200461" data-apath="/ggg/8/10/3163.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1407-0643" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Philippe C Després</span>, <span class="highwire-citation-author" data-delta="1">Alexandre K Dubé</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-3236-651X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lou Nielly-Thibault</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1582-6027" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nozomu Yachie</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3028-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christian R Landry</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3163-3171;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200461 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490961--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Philippe C Després </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philippe%2BC%2BDespr%C3%A9s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Despr%C3%A9s%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APhilippe%2BC%2BDespr%25C3%25A9s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1407-0643" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Philippe C Després</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandre K Dubé </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BK%2BDub%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dub%C3%A9%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BK%2BDub%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lou Nielly-Thibault </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lou%2BNielly-Thibault%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielly-Thibault%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALou%2BNielly-Thibault%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3236-651X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lou Nielly-Thibault</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nozomu Yachie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Research Center for Advanced Science and Technology, Synthetic Biology Division, University of Tokyo, Tokyo, 4-6-1 Komaba, Meguro-ku, 153-8904, Japan</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Biological Sciences, Graduate School of Science, the University of Tokyo, Tokyo, Japan</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Advanced Biosciences, Keio University, Tsuruoka, Japan</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>PRESTO, Japan Science and Technology Agency, Tokyo, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nozomu%2BYachie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yachie%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANozomu%2BYachie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1582-6027" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nozomu Yachie</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christian R Landry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Département de Biochimie, Microbiologie et Bio-informatique, Faculté de sciences et genie, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>†</span>PROTEO, The Québec Research Network on Protein Function, Structure and Engineering, Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre de Recherche en Données Massives (CRDM), Université Laval, Québec, Québec, G1V 0A6, Canada</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Département de Biologie, Faculté de sciences et Génie, Université Laval, Québec, Québec, G1V 0A6, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BR%2BLandry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landry%20CR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristian%2BR%2BLandry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3028-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christian R Landry</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christian.landry@bio.ulaval.ca" class="" data-icon-position="" data-hide-link-title="0">christian.landry@bio.ulaval.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3163&amp;rft.epage=3171&amp;rft.atitle=Double%20Selection%20Enhances%20the%20Efficiency%20of%20Target-AID%20and%20Cas9-Based%20Genome%20Editing%20in%20Yeast&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Despr%C3%A9s%2C+Philippe+C&amp;rft.au=Dub%C3%A9%2C+Alexandre+K&amp;rft.au=Nielly-Thibault%2C+Lou&amp;rft.au=Yachie%2C+Nozomu&amp;rft.au=Landry%2C+Christian+R"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490961/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490961/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490961/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490961/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490961/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490961/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490961/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490961/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490961/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490961/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490961/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490961/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3163" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3163.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3163.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/10/3163.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/10/3163.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
