<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3143" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3143.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3143.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3143.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3143.atom" />
<meta name="HW.pisa" content="ggg;8/10/3143" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.118.200162" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Solares et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6813398" />
<meta name="DC.Description" content="Accurate and comprehensive characterization of genetic variation is essential for deciphering the genetic basis of diseases and other phenotypes. A vast amount of genetic variation stems from large-scale sequence changes arising from the duplication, deletion, inversion, and translocation of sequences. In the past 10 years, high-throughput short reads have greatly expanded our ability to assay sequence variation due to single nucleotide polymorphisms. However, a recent de novo assembly of a second Drosophila melanogaster reference genome has revealed that short read genotyping methods miss hundreds of structural variants, including those affecting phenotypes. While genomes assembled using high-coverage long reads can achieve high levels of contiguity and completeness, concerns about cost, errors, and low yield have limited widespread adoption of such sequencing approaches. Here we resequenced the reference strain of D. melanogaster (ISO1) on a single Oxford Nanopore MinION flow cell run for 24 hr. Using only reads longer than 1 kb or with at least 30x coverage, we assembled a highly contiguous de novo genome. The addition of inexpensive paired reads and subsequent scaffolding using an optical map technology achieved an assembly with completeness and contiguity comparable to the D. melanogaster reference assembly. Comparison of our assembly to the reference assembly of ISO1 uncovered a number of structural variants (SVs), including novel LTR transposable element insertions and duplications affecting genes with developmental, behavioral, and metabolic functions. Collectively, these SVs provide a snapshot of the dynamics of genome evolution. Furthermore, our assembly and comparison to the D. melanogaster reference genome demonstrates that high-quality de novo assembly of reference genomes and comprehensive variant discovery using such assemblies are now possible by a single lab for under $1,000 (USD)." />
<meta name="DC.Contributor" content="Edwin A. Solares" />
<meta name="DC.Contributor" content="Mahul Chakraborty" />
<meta name="DC.Contributor" content="Danny E. Miller" />
<meta name="DC.Contributor" content="Shannon Kalsow" />
<meta name="DC.Contributor" content="Kate Hall" />
<meta name="DC.Contributor" content="Anoja G. Perera" />
<meta name="DC.Contributor" content="J. J. Emerson" />
<meta name="DC.Contributor" content="R. Scott Hawley" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Accurate and comprehensive characterization of genetic variation is essential for deciphering the genetic basis of diseases and other phenotypes. A vast amount of genetic variation stems from large-scale sequence changes arising from the duplication, deletion, inversion, and translocation of sequences. In the past 10 years, high-throughput short reads have greatly expanded our ability to assay sequence variation due to single nucleotide polymorphisms. However, a recent &lt;i&gt;de novo&lt;/i&gt; assembly of a second &lt;i&gt;Drosophila melanogaster&lt;/i&gt; reference genome has revealed that short read genotyping methods miss hundreds of structural variants, including those affecting phenotypes. While genomes assembled using high-coverage long reads can achieve high levels of contiguity and completeness, concerns about cost, errors, and low yield have limited widespread adoption of such sequencing approaches. Here we resequenced the reference strain of &lt;i&gt;D. melanogaster&lt;/i&gt; (ISO1) on a single Oxford Nanopore MinION flow cell run for 24 hr. Using only reads longer than 1 kb or with at least 30x coverage, we assembled a highly contiguous &lt;i&gt;de novo&lt;/i&gt; genome. The addition of inexpensive paired reads and subsequent scaffolding using an optical map technology achieved an assembly with completeness and contiguity comparable to the &lt;i&gt;D. melanogaster&lt;/i&gt; reference assembly. Comparison of our assembly to the reference assembly of ISO1 uncovered a number of structural variants (SVs), including novel LTR transposable element insertions and duplications affecting genes with developmental, behavioral, and metabolic functions. Collectively, these SVs provide a snapshot of the dynamics of genome evolution. Furthermore, our assembly and comparison to the &lt;i&gt;D. melanogaster&lt;/i&gt; reference genome demonstrates that high-quality &lt;i&gt;de novo&lt;/i&gt; assembly of reference genomes and comprehensive variant discovery using such assemblies are now possible by a single lab for under $1,000 (USD).&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3143" />
<meta name="citation_id" content="8/10/3143" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3143" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3143.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3143.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3143.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200162" />
<meta name="citation_pmid" content="30018084" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3143" />
<meta name="citation_lastpage" content="3154" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Edwin A. Solares" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Irvine, CA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3220-4927" />
<meta name="citation_author" content="Mahul Chakraborty" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Irvine, CA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2414-9187" />
<meta name="citation_author" content="Danny E. Miller" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author_institution" content="MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6096-8601" />
<meta name="citation_author" content="Shannon Kalsow" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Irvine, CA" />
<meta name="citation_author" content="Kate Hall" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author" content="Anoja G. Perera" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author" content="J. J. Emerson" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Irvine, CA" />
<meta name="citation_author_email" content="jje@uci.edu" />
<meta name="citation_author_email" content="rsh@stowers.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9474-0891" />
<meta name="citation_author" content="R. Scott Hawley" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author_institution" content="Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS" />
<meta name="citation_author_email" content="jje@uci.edu" />
<meta name="citation_author_email" content="rsh@stowers.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6478-0494" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Adams;citation_author=S. E. Celniker;citation_author=R. A. Holt;citation_author=C. A. Evans;citation_author=J. D. Gocayne;citation_title=The Genome Sequence of Drosophila melanogaster;citation_pages=2185-2195;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731132;citation_doi=10.1126/science.287.5461.2185" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. Alkan;citation_author=S. Sajjadian;citation_author=E. E. Eichler;citation_title=Limitations of next-generation genome sequence assembly.;citation_pages=61-65;citation_volume=8;citation_year=2011;citation_issue=1;citation_pmid=21102452;citation_doi=10.1038/nmeth.1527" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. Alkan;citation_author=B. P. Coe;citation_author=E. E. Eichler;citation_title=Genome structural variation discovery and genotyping.;citation_pages=363-376;citation_volume=12;citation_year=2011;citation_issue=5;citation_pmid=21358748;citation_doi=10.1038/nrg2958" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_title=DNA Sequencing Costs: Data. National Human Genome Research Institute;citation_author=. Anonymous;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Aparicio;citation_author=J. Chapman;citation_author=E. Stupka;citation_author=N. Putnam;citation_author=J.-M. Chia;citation_title=Whole-genome shotgun assembly and analysis of the genome of Fugu rubripes.;citation_volume=297;citation_year=2002;citation_pmid=12142439;citation_doi=10.1126/science.1072104" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Z. J. Assaf;citation_author=S. Tilk;citation_author=J. Park;citation_author=M. L. Siegal;citation_author=D. A. Petrov;citation_title=Deep sequencing of natural and experimental populations of Drosophila melanogaster reveals biases in the spectrum of new mutations.;citation_volume=27;citation_year=2017;citation_pmid=29079675;citation_doi=10.1101/gr.219956.116" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. Berlin;citation_author=S. Koren;citation_author=C.-S. Chin;citation_author=J. P. Drake;citation_author=J. M. Landolin;citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.;citation_pages=623-630;citation_volume=33;citation_year=2015;citation_pmid=26006009;citation_doi=10.1038/nbt.3238" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=M. Bernt;citation_author=A. Donath;citation_author=F. Jühling;citation_author=F. Externbrink;citation_author=C. Florentz;citation_title=MITOS: improved de novo metazoan mitochondrial genome annotation.;citation_pages=313-319;citation_volume=69;citation_year=2013;citation_pmid=22982435;citation_doi=10.1016/j.ympev.2012.08.023" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. J. Bowen;citation_author=J. F. McDonald;citation_title=Drosophila euchromatic LTR retrotransposons are much younger than the host species in which they reside.;citation_volume=11;citation_year=2001;citation_pmid=11544196;citation_doi=10.1101/gr.164201" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=K. R. Bradnam;citation_author=J. N. Fass;citation_author=A. Alexandrov;citation_author=P. Baranay;citation_author=M. Bechner;citation_title=Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species.;citation_pages=10;citation_volume=2;citation_year=2013;citation_pmid=23870653;citation_doi=10.1186/2047-217X-2-10" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=G. Bresler;citation_author=M. Bresler;citation_author=D. Tse;citation_title=Optimal assembly for high throughput shotgun sequencing.;citation_pages=S18;citation_volume=14;citation_year=2013;citation_doi=10.1186/1471-2105-14-S5-S18" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. J. Chaisson;citation_author=G. Tesler;citation_title=Mapping single molecule sequencing reads using Basic Local Alignment with Successive Refinement (BLASR): Theory and Application.;citation_pages=238-238;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22988817;citation_doi=10.1186/1471-2105-13-238" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=M. J. P. Chaisson;citation_author=A. D. Sanders;citation_author=X. Zhao;citation_author=A. Malhotra;citation_author=D. Porubsky;citation_title=Multi-platform discovery of haplotype-resolved structural variation in human genomes.;citation_year=2017;citation_doi=10.1101/193144" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Chakraborty;citation_author=J. G. Baldwin-Brown;citation_author=A. D. Long;citation_author=J. J. Emerson;citation_title=Contiguous and accurate de novo assembly of metazoan genomes with modest long read coverage.;citation_volume=44;citation_year=2016;citation_pmid=27458204;citation_doi=10.1093/nar/gkw654" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Chakraborty;citation_author=N. W. VanKuren;citation_author=R. Zhao;citation_author=X. Zhang;citation_author=S. Kalsow;citation_title=Hidden genetic variation shapes the structure of functional elements in Drosophila.;citation_pages=20-25;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-017-0010-y" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C.-S. Chin;citation_author=D. H. Alexander;citation_author=P. Marks;citation_author=A. A. Klammer;citation_author=J. Drake;citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.;citation_pages=563-569;citation_volume=10;citation_year=2013;citation_pmid=23644548;citation_doi=10.1038/nmeth.2474" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=M. De Freitas Ortiz;citation_author=E. L. Silva Loreto;citation_title=The hobo-related elements in the melanogaster species group.;citation_pages=243-252;citation_volume=90;citation_year=2008;citation_issue=3;citation_pmid=18593511;citation_doi=10.1017/S0016672308009312" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Feuk;citation_author=A. R. Carson;citation_author=S. W. Scherer;citation_title=Structural variation in the human genome.;citation_pages=85-97;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418744;citation_doi=10.1038/nrg1767" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. A. Gibbs;citation_author=G. M. Weinstock;citation_author=M. L. Metzker;citation_author=D. M. Muzny;citation_author=E. J. Sodergren;citation_title=Genome sequence of the Brown Norway rat yields insights into mammalian evolution.;citation_pages=493-521;citation_volume=428;citation_year=2004;citation_issue=6982;citation_pmid=15057822;citation_doi=10.1038/nature02426" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. A. Goff;citation_author=D. Ricke;citation_author=T.-H. Lan;citation_author=G. Presting;citation_author=R. Wang;citation_title=A Draft Sequence of the Rice Genome (Oryza sativa L. ssp. japonica);citation_pages=92-100;citation_volume=296;citation_year=2002;citation_issue=5565;citation_pmid=11935018;citation_doi=10.1126/science.1068275" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Gordon;citation_author=J. Huddleston;citation_author=M. J. P. Chaisson;citation_author=C. M. Hill;citation_author=Z. N. Kronenberg;citation_title=Long-read sequence assembly of the gorilla genome;citation_pages=aae0344-aae0344;citation_volume=352;citation_year=2016;citation_issue=6281;citation_pmid=27034376;citation_doi=10.1126/science.aae0344" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=QUAST: quality assessment tool for genome assemblies.;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. A. Hoskins;citation_author=J. W. Carlson;citation_author=K. H. Wan;citation_author=S. Park;citation_author=I. Mendez;citation_title=The Release 6 reference sequence of the Drosophila melanogaster genome.;citation_volume=25;citation_year=2015;citation_pmid=25589440;citation_doi=10.1101/gr.185579.114" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Huddleston;citation_author=E. E. Eichler;citation_title=An Incomplete Understanding of Human Genetic Variation;citation_pages=1251-1254;citation_volume=202;citation_year=2016;citation_issue=4;citation_pmid=27053122;citation_doi=10.1534/genetics.115.180539" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.;citation_pages=695-716;citation_volume=432;citation_year=2004;citation_issue=7018;citation_pmid=15592404;citation_doi=10.1038/nature03154" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Jain;citation_author=S. Koren;citation_author=K. H. Miga;citation_author=J. Quick;citation_author=A. C. Rand;citation_title=Nanopore sequencing and assembly of a human genome with ultra-long reads.;citation_pages=338-345;citation_volume=36;citation_year=2018;citation_doi=10.1038/nbt.4060" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Kajitani;citation_author=K. Toshimoto;citation_author=H. Noguchi;citation_author=A. Toyoda;citation_author=Y. Ogura;citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.;citation_volume=24;citation_year=2014;citation_pmid=24755901;citation_doi=10.1101/gr.170720.113" />
<meta name="citation_reference" content="citation_journal_title=Sci. Data;citation_author=K. E. Kim;citation_author=P. Peluso;citation_author=P. Babayan;citation_author=P. J. Yeadon;citation_author=C. Yu;citation_title=Long-read, whole-genome shotgun sequence data for five model organisms.;citation_pages=140045;citation_volume=1;citation_year=2014;citation_pmid=25977796;citation_doi=10.1038/sdata.2014.45" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. L. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=R12-R12;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14759262;citation_doi=10.1186/gb-2004-5-2-r12" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=K.-K. Lam;citation_author=A. Khalak;citation_author=D. Tse;citation_title=Near-optimal assembly for shotgun sequencing with noisy reads.;citation_pages=S4;citation_volume=15;citation_year=2014;citation_doi=10.1186/1471-2105-15-S9-S4" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. S. Lander;citation_author=L. M. Linton;citation_author=B. Birren;citation_author=C. Nusbaum;citation_author=M. C. Zody;citation_title=Initial sequencing and analysis of the human genome.;citation_pages=860-921;citation_volume=409;citation_year=2001;citation_issue=6822;citation_pmid=11237011;citation_doi=10.1038/35057062" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics.;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences.;citation_volume=32;citation_year=2016;citation_pmid=27153593;citation_doi=10.1093/bioinformatics/btw152" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=N. J. Loman;citation_author=J. Quick;citation_author=J. T. Simpson;citation_title=A complete bacterial genome assembled de novo using only nanopore sequencing data.;citation_pages=733-735;citation_volume=12;citation_year=2015;citation_pmid=26076426;citation_doi=10.1038/nmeth.3444" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Rajiv C. McCoy;citation_author=R. W. Taylor;citation_author=T. A. Blauwkamp;citation_author=J. L. Kelley;citation_author=M. Kertesz;citation_title=Illumina TruSeq Synthetic Long-Reads Empower de Novo Assembly and Resolve Complex, Highly-Repetitive Transposable Elements.;citation_pages=e106689;citation_volume=9;citation_year=2014;citation_pmid=25188499;citation_doi=10.1371/journal.pone.0106689" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=P. Medvedev;citation_author=M. Stanciu;citation_author=M. Brudno;citation_title=Computational methods for discovering structural variation with next-generation sequencing.;citation_pages=S13-S20;citation_volume=6;citation_year=2009;citation_issue=11 Suppl;citation_pmid=19844226;citation_doi=10.1038/nmeth.1374" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. P. Michael;citation_author=F. Jupe;citation_author=F. Bemm;citation_author=S. T. Motley;citation_author=J. P. Sandoval;citation_title=High contiguity Arabidopsis thaliana genome assembly with a single nanopore flow cell.;citation_pages=541;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-03016-2" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. Moschetti;citation_author=P. Dimitri;citation_author=R. Caizzi;citation_author=N. Junakovic;citation_title=Genomic instability of I elements of Drosophila melanogaster in absence of dysgenic crosses.;citation_pages=e13142;citation_volume=5;citation_year=2010;citation_pmid=20957225;citation_doi=10.1371/journal.pone.0013142" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Inf. Theory;citation_author=A. S. Motahari;citation_author=G. Bresler;citation_author=D. N. C. Tse;citation_title=Information Theory of DNA Shotgun Sequencing.;citation_pages=6273-6289;citation_volume=59;citation_year=2013;citation_doi=10.1109/TIT.2013.2270273" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. H. Waterston;citation_author=K. Lindblad-Toh;citation_author=E. Birney;citation_author=J. Rogers;citation_author=J. F. Abril;citation_title=Initial sequencing and comparative analysis of the mouse genome.;citation_pages=520-562;citation_volume=420;citation_year=2002;citation_issue=6915;citation_pmid=12466850;citation_doi=10.1038/nature01262" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. W. Myers;citation_author=G. G. Sutton;citation_author=A. L. Delcher;citation_author=I. M. Dew;citation_author=D. P. Fasulo;citation_title=A Whole-Genome Assembly of Drosophila;citation_pages=2196-2204;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731133;citation_doi=10.1126/science.287.5461.2196" />
<meta name="citation_reference" content="citation_journal_title=Front. Bioeng. Biotechnol.;citation_author=G. Narzisi;citation_author=M. C. Schatz;citation_title=The challenge of small-scale repeats for indel discovery.;citation_pages=8;citation_volume=3;citation_year=2015;citation_doi=10.3389/fbioe.2015.00008" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=L. Pascual;citation_author=G. Periquet;citation_title=Distribution of hobo transposable elements in natural populations of Drosophila melanogaster;citation_pages=282-296;citation_volume=8;citation_year=1991;citation_issue=3;citation_pmid=1712890;citation_doi=10.1093/oxfordjournals.molbev.a040649" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=K. Paszkiewicz;citation_author=D. J. Studholme;citation_title=De novo assembly of short sequence reads.;citation_volume=11;citation_year=2010;citation_pmid=20724458;citation_doi=10.1093/bib/bbq020" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=G. Periquet;citation_author=F. Lemeunier;citation_author=Y. Bigot;citation_author=M. H. Hamelin;citation_author=C. Bazin;citation_title=The evolutionary genetics of the hobo transposable element in the Drosophila melanogaster complex.;citation_pages=79-90;citation_volume=93;citation_year=1994;citation_issue=1-3;citation_pmid=7813919;citation_doi=10.1007/BF01435241" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Pool;citation_author=R. B. Corbett-Detig;citation_author=R. P. Sugino;citation_author=K. A. Stevens;citation_author=C. M. Cardeno;citation_title=Population Genomics of sub-saharan Drosophila melanogaster: African diversity and non-African admixture.;citation_pages=e1003080;citation_volume=8;citation_year=2012;citation_pmid=23284287;citation_doi=10.1371/journal.pgen.1003080" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Rahman;citation_author=G.-W. Chirn;citation_author=A. Kanodia;citation_author=Y. A. Sytnikova;citation_author=B. Brembs;citation_title=Unique transposon landscapes are pervasive across Drosophila melanogaster genomes.;citation_volume=43;citation_year=2015;citation_pmid=26578579;citation_doi=10.1093/nar/gkv1193" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. L. Salzberg;citation_author=A. M. Phillippy;citation_author=A. Zimin;citation_author=D. Puiu;citation_author=T. Magoc;citation_title=GAGE: a critical evaluation of genome assemblies and assembly algorithms.;citation_volume=22;citation_year=2012;citation_pmid=22147368;citation_doi=10.1101/gr.131383.111" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. dos Santos;citation_author=A. J. Schroeder;citation_author=J. L. Goodman;citation_author=V. B. Strelets;citation_author=M. A. Crosby;citation_title=FlyBase: introduction of the Drosophila melanogaster Release 6 reference genome assembly and large-scale migration of genome annotations.;citation_volume=43;citation_year=2015;citation_pmid=25398896;citation_doi=10.1093/nar/gku1099" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=I. Shomorony;citation_author=T. Courtade;citation_author=D. Tse;citation_title=Do Read Errors Matter for Genome Assembly?;citation_year=2016;citation_doi=10.1101/014399" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_author=A. F. A. Smit;citation_author=R. Hubley;citation_author=P. Green" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdóttir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. J. Treangen;citation_author=S. L. Salzberg;citation_title=Repetitive DNA and next-generation sequencing: computational challenges and solutions.;citation_pages=36-46;citation_volume=13;citation_year=2011;citation_pmid=22124482;citation_doi=10.1038/nrg3117" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: An Integrated Tool for Comprehensive Microbial Variant Detection and Genome Assembly Improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Ye;citation_author=C. M. Hill;citation_author=S. Wu;citation_author=J. Ruan;citation_author=Z. S. Ma;citation_title=DBG2OLC: Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation Sequencing Technologies.;citation_pages=31900;citation_volume=6;citation_year=2016;citation_pmid=27573208;citation_doi=10.1038/srep31900" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Yu;citation_author=S. Hu;citation_author=J. Wang;citation_author=G. K.-S. Wong;citation_author=S. Li;citation_title=A Draft Sequence of the Rice Genome (Oryza sativa L. ssp. indica);citation_pages=79-92;citation_volume=296;citation_year=2002;citation_issue=5565;citation_pmid=11935017;citation_doi=10.1126/science.1068037" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=L. P. Zakharenko;citation_author=L. V. Kovalenko;citation_author=S. Mai;citation_title=Fluorescence in situ hybridization analysis of hobo, mdg1 and Dm412 transposable elements reveals genomic instability following the Drosophila melanogaster genome sequencing.;citation_pages=525-530;citation_volume=99;citation_year=2007;citation_issue=5;citation_pmid=17622267;citation_doi=10.1038/sj.hdy.6801029" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="Accurate and comprehensive characterization of genetic variation is essential for deciphering the genetic basis of diseases and other phenotypes. A vast amount of genetic variation stems from large-scale sequence changes arising from the duplication, deletion, inversion, and translocation of sequences. In the past 10 years, high-throughput short reads have greatly expanded our ability to assay sequence variation due to single nucleotide polymorphisms. However, a recent de novo assembly of a second Drosophila melanogaster reference genome has revealed that short read genotyping methods miss hundreds of structural variants, including those affecting phenotypes. While genomes assembled using high-coverage long reads can achieve high levels of contiguity and completeness, concerns about cost, errors, and low yield have limited widespread adoption of such sequencing approaches. Here we resequenced the reference strain of D. melanogaster (ISO1) on a single Oxford Nanopore MinION flow cell run for 24 hr. Using only reads longer than 1 kb or with at least 30x coverage, we assembled a highly contiguous de novo genome. The addition of inexpensive paired reads and subsequent scaffolding using an optical map technology achieved an assembly with completeness and contiguity comparable to the D. melanogaster reference assembly. Comparison of our assembly to the reference assembly of ISO1 uncovered a number of structural variants (SVs), including novel LTR transposable element insertions and duplications affecting genes with developmental, behavioral, and metabolic functions. Collectively, these SVs provide a snapshot of the dynamics of genome evolution. Furthermore, our assembly and comparison to the D. melanogaster reference genome demonstrates that high-quality de novo assembly of reference genomes and comprehensive variant discovery using such assemblies are now possible by a single lab for under $1,000 (USD)." />
<meta name="og-title" property="og:title" content="Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3143" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Accurate and comprehensive characterization of genetic variation is essential for deciphering the genetic basis of diseases and other phenotypes. A vast amount of genetic variation stems from large-scale sequence changes arising from the duplication, deletion, inversion, and translocation of sequences. In the past 10 years, high-throughput short reads have greatly expanded our ability to assay sequence variation due to single nucleotide polymorphisms. However, a recent de novo assembly of a second Drosophila melanogaster reference genome has revealed that short read genotyping methods miss hundreds of structural variants, including those affecting phenotypes. While genomes assembled using high-coverage long reads can achieve high levels of contiguity and completeness, concerns about cost, errors, and low yield have limited widespread adoption of such sequencing approaches. Here we resequenced the reference strain of D. melanogaster (ISO1) on a single Oxford Nanopore MinION flow cell run for 24 hr. Using only reads longer than 1 kb or with at least 30x coverage, we assembled a highly contiguous de novo genome. The addition of inexpensive paired reads and subsequent scaffolding using an optical map technology achieved an assembly with completeness and contiguity comparable to the D. melanogaster reference assembly. Comparison of our assembly to the reference assembly of ISO1 uncovered a number of structural variants (SVs), including novel LTR transposable element insertions and duplications affecting genes with developmental, behavioral, and metabolic functions. Collectively, these SVs provide a snapshot of the dynamics of genome evolution. Furthermore, our assembly and comparison to the D. melanogaster reference genome demonstrates that high-quality de novo assembly of reference genomes and comprehensive variant discovery using such assemblies are now possible by a single lab for under $1,000 (USD)." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"f7L6MK_rTi55bCo8O65RTz1Pi4Fkm_MfyUWnjFI9PjI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490970","apath":"\/ggg\/8\/10\/3143.atom","pisa":"ggg;8\/10\/3143","ac":{"\/ggg\/8\/10\/3143.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3143.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3143"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3143"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3143":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jq3WcKqWsaLG0hWtJbP8N175UHrTrv5dtRayFId1A7lq9OxD4pO0Pc0agTRhfwYC9M7QIYUd\\\\\\\/akpVzfdlNvG+jV5JrwBfnXJFM8FI9l6XUrvmXPWdWnxTP6AnUNSG6d0twrOdNEfuyNyhqLN1Y0aHzA1wxJ4uAda3jmPDNmDZm141qNEK19pntf3U70j1l5CaKICWb9JYmwJLJGi0O8hONlNkipkpQMnegWn8IuE+ROma2uXfytnt4c\\\\\\\/zaBUd0KO2k94\\\\\\\/lMAR7dAqEW9wgU\\\\\\\/AQhVX9VMf0kJMC0UgzPTpbjZ5rEK96CHccBsvUMS76r0I\\\\\\\/5TnuDpp627VGnttKr+S6aQuP\\\\\\\/ot5botT9BzTUq\\\\\\\/w1oXIR6NICgLiMY1GY1Nb9eJqTi1aWTTEZEmkNMhWOEPaDEtB+eSqLGYOFbadr1TBQl5y8Tf3NL3ACNJBYTrxL833c3JohM0yBvkLe+lRQq6mYrC6X9pdDk1396Rui2U9TbjZoW7U8lePW+J2hlmIXx+RdxNWvHFASM9lBoK5Ln5vR3vXGommSaXz7rnyLDZTFij0s1aXtHCKdz4aDC0dCUP1t98WTysSyD5nNIktFuxCwfnoW4TJFYRL7RtcqUV7QlJqnJlLE\\\\\\\/4k4CZKWRv1TGHXlZ2F5EnXySra2+JjEYtEucHchP\\\\\\\/GhSr8lMEAUj8OXaJkHXwa9Ni26Rf7OBwp4zZWQ1OK0BnL6\\\\\\\/DgFRHZlb7WRguT1J5QfcKVxRQ1JouZFqqaXjEiQGKSq9ZQyI7SvX6oFaeBilhpRJ\\\\\\\/OU7cM+4t2G1WjOOjI\\\\\\\/OzsMuAqscku3dDUYZLOfizSzGB+mL3UrOXvsSgiyJdjMWnD7oyaYdRQ0S+ZgtoFEb0MymujJ3tRMgXkMYrsZ4ncdd3CSkFa8XreXhQglfI+zXY7CvAuTRyCnCQnL0zrbgeRhm7pAMYWMo3YiFlr\\\\\\\/sW8Q8XP+T2YBm1lyMxXidDyuVGMyIKOQGZcBOj5y4qwyqsyqA5PCem7Or8xaKfxt9WGKPtn7cOHalm58scaYQGPKXB7X6snIZYG5yQUJAjNNOFQA87Lhra3GmbetgOTncql6R5XRxF5JGWtsp0SqoILV098EFyUmegLL6UAdjQgcVEgGB26kw7h5VFxHJjyrkGSj4JC8ntr3UkrZAvfU8bt+5Zij6M+gjSaFDC3ojBxDYTyRgbgbRYEJgjpqG4tCU+rsQr2GYF+IdWlzpIWVLdfrrITzHHbYAc10p4\\\\\\\/bOMKRoeNWQjJ0NGk66ve6o5uwc7MFKo3ZWxpGr\\\u0022,\\\u0022iv\\\u0022:\\\u0022w1fkp08Ds0cbBZHONr\\\\\\\/LCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002226d55f2189fd76e117d741b9eead35be3f5594eba3a59307da4f05ee8ea85e5f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gAL0beT0IImXnUW2Av03qdhtYNCXCjs+Uspog1Y+V1CneglUUgQm+8jl+wr29tld5cj7S2qgcMf742Jtbd04dteTBX53eYTi7gVkdSOwmhmMtZjMuWdkcE8WFc3cBDeExRuufTK4yJL2hU26BqyiGlXGLHyy6K36py0Btj0nQv90zXBl3vWZiO2yGkPHNeBMKIJo\\\\\\\/vxy1tsNlFxi\\\\\\\/V9BmXLeV4qz9LcLrLbyQyqa\\\\\\\/2aIhq\\\\\\\/KknxyM3IE7SBKdt4DzWR\\\\\\\/KdmBuWOR+MVsHHgkFLH0mdEaIIQzoMgMbUgq\\\\\\\/v3CO\\\\\\\/64bY5gZbWckyDfrgTk3wDPNpVfGptS9ZC\\\\\\\/XDDAiJ1ol3bLme4cedE14CxPJBxSsxHfUn6oPXBPc2s+yBLN6qSBoddkGMToK8IyH+EXG6qpncfRVkkplSl\\\\\\\/J\\\\\\\/Y64S3wddIyQBwNSx1ZNwnvd2xbSDZFSKWDNDa1HTeYo+SVAqVuKeU1DGH\\\\\\\/UKTkzvXaCfOUuI\\\\\\\/GruGNW2oQnCAENtNbemjUexvaK1JEj2Qqc5tB622RaAPqjpS2D3icV9gup+BCVtE99I5PBz5pEveAvkdWxXV5MQcGTrdKY3LSFu3nN3uB6795PHcWTEHLBidZ3YkVhVj5ruAtHECJ7fxP9tlsmYTmrLdKa5Pe8ArrHhgj\\\\\\\/ZFv6rYZqsTGMIulOeMfQLBeFYE11fMmV5HxUyiDs4xy44k6NZhL5LQ6RSzfGMYGAmH+hwqxk1ybguH2sGHc+JcWFvWalYA4Jn2eAmMcn3NpKfqAwxRUFKtxrUuvXxlYBxLUKDxb7hq\\\\\\\/Vk0NFNd2ycLfCltAhgj0gYdcTl1kiW16J7yDCvczsYjIkZ5CXdgkAcUBJPCEnKdE\\\\\\\/C8zg0l96J3ojFMKdJ3nA\\\\\\\/7aLzi71zd0eYnXjNqSTH4ZcddI\\\\\\\/7ymOJ9J1UFx90jm0AhmMSNVWOIeCpMRaJiI8l19b0w4O4Dfi2r3w+AWMQQZJ1Dm\\\\\\\/e35NSzXse3qIkMGP1RL9SPWsdG2rhVsVf44phbxz2TEchmM+L6ffVMlSO+3H7qKZxaE9+02AAdF3cikZ5g+v03RcoTqDGr+vgzQtIX3sCvCIR8sMTUpw3wROmTuxCEsGUqAW2YQJFRg9XZmNhJYhbilN\\\\\\\/cAvn8hwyxQqIZvwKbCbq0fLdq4qsyg8CE7tbONNu61ZWP6thMoJw0Xm3OGw2OlOMKQttaUP3ISdMLS53oOAUI+XpyhkCsTt+ZCrG22ahqm1kEmaFbIoucQYkT1bq9EtE68Wg7Iz+l\\\\\\\/F42cF3s7RgSOH6sUF1plnRhn5dsFsjxVGd+gU02ObdTq3\\\\\\\/u\\\\\\\/Y9fH0Zx24nEEqMecuQg8mOTNiXDjer0h+qO1V3\\\\\\\/+3SulRqvpNxcrwRTDr4yveKQqlZqEjwPzWSti4Rk9s63JvBzUFbJYYW4N8ocSJfIF\\\\\\\/eL1sTY7pxfGKggGBsUffp95TspPghLRTH\\\\\\\/PJWqx7mthYGB4GxhALDvdRkD8Du2I4ZJFsjqO9lRwVJe31MSM9r\\\\\\\/xOpmG6Td9h0OSxdfDepsSsmQuLpm\\\\\\\/6tFx6prtgjgatfxF0LmZ1vX569E0v9s8af3SSCoVziDD62n6ckckAZYpK9TaQ4acHcifvBUiTDfAZQC7rC4bRde9C6L+lxXs6uKT7KapmyOPoD4GFn7gFCY5ZzmqmL\\\\\\\/wMvhiUo2KFd04aUnc89dboCUs6XxpMk2huYwwrpW8O+dpX3U3w7VqGJyLf65gpMje+u0pyigcNFOnesAyH6sjulaxB9y0hJCjbWJio97lK4sB1vKN4uNk7kgWDrVLpEML6KKfIFoX4K2E9dQbMEN33pZMt9Jhob\\\\\\\/FrYvQNPFj5GBB+U971q8JtxaA7wm+QWoNGpJwrKewwbicT1ar1nc51+oQh2O6RMiR17GrxQuowY2n91pBcDSFPku+ZfZF6xpfooxDS6QBHYR92GSotGId+qjVLmgVcIzMMLKTIQFKG5nUXHm1lOxudxR7SPOIF9Vj5IoljiEFS7fzRYOSnReI5HznFR52nv6snKBynGhso7wRa3n7Bvf9pf6gwDu0zqHiDLplLB8LphjXDEfMvN3+164hootHrs\\\\\\\/mqp6DROpl4XcjWy33yVxDTPFSJha0e03inHQi01Wj7J9gX9kHsvE21QDWknbd9RU9QKJusjF0BjCPtJu8FIzhsXyx\\\\\\\/GqwYkmXZNu\\\\\\\/w8yXHt9ZZYQvdGZY6MTPlT\\\\\\\/LeWqP0IZ2TyMZPiJG83GDcwGqHvE47hvAOGq5OEK7IosPq6aY8KzHe5sFQCA1Gr2gYFwtahP0jXaVhwilhFPZXJxsiQDxwnQFhdAUsbovgWeg+UzNSZUmWKR+f6vUd92OBzzvR+7wCvXYZMEsdps1z7IWfcokZ0Oyev3MqXORkKNpA4hOPCzyGLD7R\\\\\\\/nqxWandw\\\\\\\/M0f2RE3pXyQOfXGCZdo\\\\\\\/wJSTzf+XB3HturfU6thfKARsSQCBGkoLUk9sKTGcZJycYWT6qB2SD388HmhYeG9gnYMHhXYjQwIZt696cBcP\\\\\\\/K4n6iyBjjEC8A5ZrQWiPUVgzp0jdHJ3PJa0jzK1LIOGYHFeYSAIXMzBrTY\\\\\\\/p68Z\\\\\\\/itm8Jw3wrXMq7sN80yGDr5tx\\\\\\\/UmWaQJU5MeyNDQwqXRsDrp95kZiX2hi\\\\\\\/llBbP94KkOo6PUA3H\\\\\\\/Td+mMm35UPoc8oZ5rAVYsC87ZxTBbalAUgUkfcAsApR4Q30N2RFBs8IQBkhm3\\\\\\\/7L73y6H\\\\\\\/7OU9DAUt2h5h1g96sfXSgU8TcDcPgPWZG8mxAFO45XJvZd+4mvp8GKiOOff75YtfD3cnibp+eUx3VzgbmyCOrHe\\\\\\\/xM72QaTMZm\\\\\\\/0R42pmXWM1IkPHdExkPh5s09dSDpmOD0dTzpxM44F4Vb8s3nCcacqqvqyANbVmcnpETfsT4yFb3k8XV4tR+3YhvaayOKfbxrfR2lUFopHKQpPSN2ObuSdcCJOnHCSQtvkLTMexIS6bzcjB0ECVoZuL6HDZQINFTlYKMnR9Md667QgNYx2MkNBDyStxW4EtJAgug5bQL8kjZnlntblvvu5bl3\\\\\\\/UPkreHihuby+P3BzwC+430yB2lZ7MZz6ic8p1yKrNCSblIWPudXz6xhEZvu92jdWiV0oK5aKyprwwl0k7ICPf5qmSLs1f2N2UeKyDQLP4kuUXvtXVwCzohlt49k0V1Hju\\\\\\\/pKIKngbgSINerfWVirQmhYH+eH4BR\\\\\\\/nrctubjPKrICstRj5Op1AebyDRymz\\\\\\\/R3C2KxWrhdg5\\\\\\\/Qd1GBBZOUnRjmN6etpoTC7m9lJ89kR\\\\\\\/tlCfuzRQhn9UJq818FJhXrXxkt+DZ91Km9ji17D85F72hvSIoeO6026IB75V\\\\\\\/WimFoZWEMM+YPNLFJbonm3tdwtCAYIIL3qqh\\\\\\\/La8QK2qkp6H2tlb7+q0DIpjEa6J\\\\\\\/YcxCyPZaB9hIUjjQEe0HoHZhPOzPvGf+dgM\\\\\\\/oOW3xsD+zfAoc6ZJN8GTIckekG5npU4yyWclmlLOMDNTpSwEomxwXJG9rSjNqBIEHH5u57ySGjkhgDYiGxesYUAJ0TT3sNZk18nmHz++kZI3+RvWmtPmFgBzXuVrXGqfTIUclpkOy78jccfltBkc2KCGs12L2Wt68d2qgOss+fiVDECAP184VCIZ8Q4HLBGpJ+yEUZzeboKof\\\\\\\/2i6OkGyUOqrOsNCruzSV+kH4fDG6R6+ufKemoSZh3FeP4suQMH2soRC\\\\\\\/reTtzGUNOd\\\\\\\/6xhWgL3vGwCaZ7KOmnWkkucXCwG0tORDogZuxTdFZsQeHvxlVWd9sG2UR1ckzq5ll4c+cPwsPjADpD1TzmSJNGebWj45IoVbCrInxjvWR5uU4zI9rZJXHGd1Zpxr\\\\\\\/EZMWURd6UKvGvYdVFITtsSy6ASGGjwZtrP2tozj+RSlfq0CWXLICdOiCOzMqRazz1BalGr5lpDZCv0c9TqWweOpeVVss4GbTloDZKHS75UvJvdl8xZHNQJUgT1CT8jF01fcQ7T5wmiKP0iUdIih0SVq555PfuXWETGnSWru7qzdMCo9IPC2jAHtQUxf0GLsRG4tS7JR8D3oUVUsXZY\\\\\\\/R1oDmNzmedGYnVKW4V7oef9zNl0Hnnn6aHsi5HDW6NSoYlZrjfC0IF7fWRohvObruWYCrUV5u94g4B6yR4Ox+dqqxNr0vgfIfRA4\\\\\\\/xQO\\\\\\\/4DfWfi51mgAt9E+C8+F\\\\\\\/8RmO37lkJM4saHvKrTb8gydhMdHMB8h4WiUifcsjSFYXxAhb7cgg0WjEAWdSlrDVGlYa4Z\\\\\\\/jB0x\\\\\\\/9PF\\\\\\\/y3Bw62p8Au1j\\\\\\\/4r2D47MsmtEEC3JIu5NGUhC+A6YJ8JNIvjd4bwpQcjO45fcLpaFUMUXZPXkFQLDuDi\\\\\\\/lJzCgtjJtilh9QUOPJzYPh0dUOdkER0ft9UizlYDFlbJEUT7+3wD+ql9DwaL12\\\\\\\/qygL50YBYrTAclWaWmif7hvlYWUqLVNlo6m+\\\\\\\/UTKe8P2f4g2L2an2bgj8hJcG8\\\\\\\/VVDJ6SfJP\\\\\\\/mTqfxpFcq5xyzxuLyzeyCjrt8HIPFaJ0stkf7YmQg5HqcVgeBqKbxGBUin3cJraQA5jEZlaFniEV9GPTMjD\\\\\\\/v5vGgFRWt5ZLjwnYuyhrqN2tuKyv8qAyYfA+i1NQoqqHvhEzGGrFyPAa+SG41l0u4qPZchXbGDSrYsqXc1NIxD1deNVSpBzfBb59zY7SHOuhG8zS\\\\\\\/ioIkNV8iBW9nOPfbiXrMnu1iY902pnH+ISvgnJiCzT7ilbXErTN2P0geiiCnB3+sBcgPfX2aZZwl76pS\\\\\\\/4kM\\\\\\\/opd5gwvoWqoOsConoa5U4k5OFwncesGOlD2rz\\\\\\\/FvHZfj3iwBeNHD6BMyh8fc8HEyN3wWQZn7ygN3f42GWwB83Q0vBIJRH5EMXiPa0E2tyR17orf0Cvm3E9PvlrDAu+7TTmtpAdnmzivlqmA+Z3NT\\\\\\\/9WrJAf6XvmYgRw3WhBix7B+sBLrOQnd6aFQp9dCks0Kzr70qYOe8RLBlCgL4PWiLQ\\\\\\\/QorRPLfUD16s\\\\\\\/L1TQQJYC3En\\\\\\\/3G0IpOKVOpFiH0ZBYnPb\\\\\\\/FFEHZVZQ\\\\\\\/fqb42olLZaqb08wt3zJ6qiFhAcQ9hCi39AER3RWT9CYD2EGUixQALmLdzrO9JhWGH\\\\\\\/w83Cp\\\\\\\/KpICyIB0l1Xs5AXFzRzkbvLlBSqq+dWB0FRg9OoGFvzR2Sujxn8f3ZNWpcomXM0Y2uiKjj9zfsmUrmpbpTCxFgyRO\\\\\\\/Mxrxfc2\\\\\\\/7GZdYVCu4j3JYVjoqbxdYKpPtWnp\\\\\\\/ujCkdkMPlBmxO7QG6FDolkSyJ\\\\\\\/ToyRNmvy6sZKx4BgldgsDwlO1ulgWvzqKfPiszSXpMFdICYURGzKhG3ojsGLYLZT9vD1hMYSnQ5DE+BmxO9MEfb1z23tE35V6YLKQtOYdgkpwLxW8qZ5OzV3cdFwrMcWgB7tiwuyloAau1zE2ak8OrJHfWR9du+\\\\\\\/zkFiwmTu5akzsIV5HUp7lVei05iUKT\\\\\\\/eEn4escXP+2mPz6199cRdWUHbJYnpmf2aDTbU5BFz\\\\\\\/Ro8KoZe6hJilVLF5ZvR5yumrhPqJwgELzqMUVtuLcKF3XX+3ZFqRc4+yZUG51a4T30kUMAWCrYdvoVv7aIMbVz3p7TjgFUncF\\\\\\\/\\\\\\\/7TiUo\\\\\\\/muVSBumdNDaNEC42U4w97Fonl1z92HFFEZqWnLAmer8unnXF13F0pwof0aDmZYG2\\\\\\\/XbeWWnRvt5ReWcnpONMPX5DD5MuCw\\\\\\\/DP97W61VhbiZhwaTFTUTwi10CQ6a9qcX0j9gNjichTr\\\\\\\/HyNtaM5ZmVxsEp5WJKpuxY5BwN76dvkmdiczbDo7FHNvM7aEAklDI1uUcLAztbyG5KsmNNmRGF0d\\\\\\\/6ANzq33pffHiCFpRKg8u2kJ2BpGkl7Uqu1cpPuVeZqNcLPTPIgb87jZ2JDqV2pxcZlblfSod+\\\\\\\/uqjGWjQXsoVnCyHWKcRw6t7Zi4nJF\\\\\\\/gb7VP6ovXoL\\\\\\\/QMl4KMmn0ypI8lkfmgx97AodQx1DAONZTyaIeFrrq3+iCfw+jjbkAfP5hpMkDxVr7g65CBH+\\\\\\\/mwKXiEAeLXR3U1CU9Wlppq6\\\\\\\/WFVQMCr8+BNnarrjoytct1w+QUabQDjWeqKURBu9vHJPCljklxXRt1tH5vu6QPSoyDvGGTt9Ic80ykVQO+d6KX3uM0loikH4OiAdSk9Cv84IAA9Zx8Io2AmzplFSK4D11qUHx6k2uzP7oswtfCgYT4wC0xxbXUj5UGea9SgdSb+tm1omN4Tz\\\\\\\/BhgiP+oT6AV+lId4fi7NJUsPGiHXewqeYtdFGn+F44Q184RC20hbAK4dozA8YIwuZFlctaC2Sq6suVhFLm4E87NEP6JoL8Tjq2qfvKmESPTqBtNRpFKtLe3D+nYiLh2C54ejCu7jHNMPtANK2Qb\\\\\\\/7Kk0EOYF25KSrh69HyXMLRfiP6FLYewQrqG2bauyNDNd1rqivzjuboGgjeshpZdp2UIJO+vYu1SI5JF47gboQUl4TOAd7quKg1G1Q2OXqe3SA+8ZwuTa99RqMi9VDbxnimMX4mNTGVkHgWpeSWGvhWnAXarFSUfqJV\\\\\\\/7lmWzrZ\\\\\\\/aXwkX2QSKfuuad2VbpyAnb\\\\\\\/Dv++d9L9Zq\\\\\\\/\\\\\\\/GGchljB5UiCSmO3jItIwQiFwga7tAXCc39Ga+NkJzn8y5BKT3uWrz5ulVuDfjzmjeccopQWyXlA4PjnzGE3A26xYQN+\\\\\\\/bNa9CQor9cNhmoO5mImiox9xYan7ERMAks3nhqOhey2q2tz1JMPuomDaOwSok3\\\\\\\/zkKgQRwRUDcDcWoIf\\\\\\\/2OA+6DgJNFwaoDCPJQu3AltOXr\\\\\\\/Bvm\\\\\\\/Bw0MX7SQ\\\\\\\/NzfeGNMHWh+TeGEYuPqHQoWKOEeGY8GvwYkjoakYwXfOfYm\\\\\\\/3yQb0\\\\\\\/3xmkSgF8hrcoC2xarZFE2Khpy2WRBVLMNrgwT+w6zWKYhFsMaQFQn+j6dpNQ3K74PnhTAlM5z45vdsnQezlk+3KfLU37H6PFvMsWJIrvNcQaVcBP0mX4n7CYbIoX8ytuLHFpUBZdQXVx3wqpxJDjwEGKmpg6CrJijSynpWqOdgSXHbUT+GiAwZjzRJO4lEGh8gux6TZjAGEEXYLRv9Bgou\\\\\\\/5ezwEXd8CGGONvoajKJL\\\\\\\/0m9wY2j\\\\\\\/Im1gxmwk8PKTF4uyOJ3Inj3i9QyAlF7T2bnkByA8DppodsB1CP9csWnaEx5qPa9l3fgCoum9U3l7ytBzgUrZR0Yf2\\\\\\\/ZT5J1yxh0a9haYG8FOseK\\\\\\\/C\\\\\\\/Vy0i8oEKdxpzppdxm6+q39IY\\\\\\\/iQEant3nDRh0KxSfzr93zNbGEsuecWeAWqQYKTAcBzZn59r3ILkS8QTkCZyN6ttRPpmT7mloPdSmSNysTeFYi3dicr\\\\\\\/D7tYxyKqD0cGxq87iL+st+LS6pnzuJFxxuajaRs4Mo24FFhN89EDeZycq0aDhK2S0o7p1rFKd+kwiW15HmfsEhW62X6HyIxnCx3\\\\\\\/gn6xfLBMFX74t5q7M++20d4oIbUZ1ZbTgmkaqXdH46e++5DWG6ffKDAQY\\\\\\\/W5xpG5XRK7LMu\\\\\\\/w\\\\\\\/I7m8dSCdPtIFmgDgCaeAZlG7OG2jDA4LXtGFw1gCxZBYZrlIo9wm8hEAl\\\\\\\/YoDamxeOGKPKh3x7HjAe4LxCGuFXx+BsltdKKaAKxcrQT+vpCWLWYGqjzXP+nNsGOaTIvRVuS10YOjyz864obYvrhT1KkoRuBR4ddZcLFZALjq7NdoMuxGg968NYNAiFQyUl+yiAS+D6sICFWLKBBWLPbtE4nYCA3j8Py\\\\\\\/aVJcyDLR3sYLh21bRKVEPKcETPtrEjg9zmYVurlismTmHwv5DI1wa5lzmcD3vGiOR5zfIBzSylTGi5Wbk\\\\\\\/+jRfSpzgieB6emxUFJs0bdaEE1al+S0Bvk7crYqmld\\\\\\\/ig0nzPB5lGXAl+Z6FqMfUa6mVi7RIskt8vKL85nUmqbkRaULI3Ca1\\\\\\\/9XfXjGO\\\\\\\/AFXkTW9IwDLkM0MYAhwl9w8A8zjoXszfGIKtFcdSO0qshwwzxHa7NZS6oq1oOYNKH3fOGwuA8FP4QVgLxRYS7UH4y09ANSMl2xMu0s+PxyruQQ2XUy4GVpHn9JVFkHKLpjIicHZ6aas\\\\\\\/gF7RXarSDgbsc\\\\\\\/RqjAQiZFZjqIagH61MzrqsCevXbzLA\\\\\\\/S3ZuvOmhBRLxqweIuFKvU4XJYPkz10ASaPep8bAk6figsxKfz3wmON9UGKdnZuF+hKQNCwKo0UKGYv1cC08d7OMSxbqSZCPgYjmGFe1\\\\\\\/L7XjyjQyuHNzmNkSpwSiZ5qp74aw1WB2gOnq6XQ8k06zem7RU\\\\\\\/sNzEn7+U4FKYBVPaiqWsaStkaDj9Mhv9FMrnZQzaxtlNOD\\\\\\\/pBwbvFT+Nbg6xG240SdD85UUBQBqeVNDXO4BGuAhMrxnXgqA7BeqwxWqMbmseHWDh6v56kYlDdSi7aaOwA8OYdSf34c+REhobzrjIczhbySZoMrY5yYfBbKkek8czai+0EU7gnaJ0usi6bukSOkZth3mOgWv4yIdjq1gxamOy95ZKb8lWE+zdrCGr3Q8rSYrUna51n80gg8Svl2wwftpa6YdZ4hryI8wRh1fx7pKb1vmJzbU4595UJ3qVQyab4VBXch4jAx7yYQ2Vg9via4x6m+j2poSM2Fdfr2A\\\\\\\/tM+60Hqi0OfHS6xOgYNpcE7KEEcttuoHgfR+AFZtDrecxeBc\\\\\\\/zfVWgMJvV2hjpGjjwMYKbwhhQncA0eON4aZNoutYniZE4LjMXPrpQA03qOmgXtGdudmKITJGm1naF\\\\\\\/Os7zY2qzVDDnjdaoXEXBqj+i8rm5sFQWyEw9rWrkwNE6Bg+pIL7UKv0wegDU2XAuLrsveeneaQYlGCaEWkZc2+6FUh6mixD95D9eNxpYZJhRXd4pAJCiqydbrYlcihhfIVIMHLIVal0I7ImKHENYqH25RBx\\\\\\\/n2ZoCS9\\\\\\\/hNM7OvtqfHhh7FJRIk8gQ3hcj6Q6FuePplhltjXMKHtRUHxfmnQ5sIvRRdxWX5P9c6mDO+QlCCAnL1qGTNkmOc8k3uzbxNypoChP57BFt2Sqkb9ahm5e7iF6j7TNhUfLniDiBbmEuhMewKYq6JVDmYpyPQMZK8q+R8kKDV6QU5DPpm1bAnoh\\\\\\\/R58jGHC1OaIo58yzZFQ4l+rLhuWD2i0NJsA\\\\\\\/r2EpmOtJgGMN\\\\\\\/ox0\\\\\\\/\\\\\\\/z+JlypjJ5qTcGUx8UJxt7tDQ2cpJsF8K7VuMJvTis0aU1Ld6MyFfOvfiY+iUGIoRqBULQ+g\\\\\\\/Dhv+gobwQsx48RsKD\\\\\\\/66giSrHfSnrIb0KnEGD1D88LQk5kbRhcgFvpZqEs5\\\\\\\/7rer5CEvmTK3j5Cvk7QJMhyehCq+bXW\\\\\\\/Gq8VATx\\\\\\\/RuHoD80sEGUPuWHuaeLG4eM2wp8LH\\\\\\\/OX22rUdop5B\\\\\\\/u1r\\\\\\\/wit1wQNhoGMgk1Vvqqfx0tgOVSO6Nwo6wRbVMqUlHUiFW6ucXSCd9PUwAgprWDoEx45vBvpUyxNHMSPS3us2s0dOHrYVORK+CHatNnJL4G9fRn4ryBpI+cJ6uzMWRVviOYGS2rUyBMcjxzeIZzKedhzMbEGO5HWD6d6mrccfgtFO7zc5pCWxg5hOeP\\\\\\\/VpQQmYM8hVnbg14Eha2\\\\\\\/72CdeMcxD+1z3dO6GDczMpukdruoFL3sQV68Jc+vfltFJ+R5jKDg5OMuEnZPnSsVB4QPbLWXl2cJaiiQGSQBNUmG1\\\\\\\/ZeV8UHV5VAX+DEPEAK29SP0To0o18yKRm9zIu7yKv7JmgWgFv2fXnpNUAcHbNdi4ATt4GBWgTrhffWODx07+ukYHSMGJltx0OmD6vCfYqpoqQyYmEmak9UFeqNF98y1+CpKdvKhwPzlSvzwQDJ8vw3EUQCOoS0t8YFKmfM66Pb+Lp2ybNqxJsWwJyvFNwM6nNaaIcWZM2BM0fOwnYPb0FUIk\\\\\\\/YPq9I3r3q2sUeX\\\\\\\/ZPtiSklwO4z0V2qnKlmFXt\\\\\\\/veRdkDzFgoMGw3+Q++opKD1uLkxInauzL\\\\\\\/CwxGlUSCN0P0xEJ0F499ds38Zxdlp3CyUp+z50Iyt5Wwrjg9S5vqMDS7ZW0fUCTbnNu3B+2zgEcTjkQemfK5sDnQxTCOdbCxQSkHeMBWRBPv8jD8aupADZgjCZvHg134pyxI73FI\\\\\\\/4U\\\\\\\/WhjHIsdgaKE3LxzQqvxFKaw4MQKb2unNABN4KKebzt\\\\\\\/OcEqB8ukwV7nCp4DeCmWsYXUy2j4sfcJz69hdtk9vUHEhclBcAZmfx9vCzZx34s+IiAS9kMGd9BSMiq6GfZQge0931UvSgV7D+O4ZKcHqxBwF6GaInSpHsxesZ\\\\\\\/RkzatFlqFbUykGTyWNuQEj5KqL8cJfVlRo5uDiDLa9KzVwY2nF4ztlBNXtpsGsl5bdMMcVXaRvEUCA111KsMprown5Y0LpeYu7Qt6x18KGIqfyIv66ID8pa6IDNYoNLZ4sydOfHtTblfL0J6dyGsEnuC7e\\\\\\\/uitWC4tUd0rJYr9SoutE84VOkPozI9NSvCdbOX2zChLjjCXtJb68H5cBUGPcznjwYHvoR+KGHSf2E6m+MzpZlTogaSg0VD\\\\\\\/ercXrer9oHX+DVzyZedlQXG7Im1wXy6QojdqsODvsQfIOKzZWIboZSt3febv2vroX+50vfe7z1JZ3t0wf7at8oI5Z0Cs\\\\\\\/C+eXKOoAsvdOYhX2bqz2d7OcsOxmiQojjgwyyqrBn8hMlkJHLYFW+Cr8zl\\\\\\\/Vi19zatXj5lWR0pYAnoR48jyxhhpBVsQQz6veHE+fM9+rzz+ouEeMqjIGhVyL\\\\\\\/VHr7n4lfrYiQhwvIthG\\\\\\\/hAFjaxHi+k36F+P5WLI+WNmVYR6ejwHEyp7GU8+i+mPIAVkmFTasK5wn5BQWXIiBKWp9CBlsWWBScHy7WoSh34lEgExXI7Ehz\\\\\\\/\\\\\\\/oBcHF\\\\\\\/QfuzcQaxF1X4LtB4I+MKMAzi4\\\\\\\/9KqF6P\\\\\\\/EulMDZUtILt\\\\\\\/32hhiAASvOsJqsoMXf3zbZTXJqHOkaacnR2T5jddzc94Kxz1tPAe\\\\\\\/Czvc7H7Xr65G8YFPsUSdkKzk7FptPKAPJnLc+gKvBBzXE5tYq4ovIfDO04GPgp770Wi8f9hLUGxodYJj1fEfXWinHlYCnZSaWP1OikCp0bB0KriD7MVA6gsDlyV1k8wYvFYoVRgP9vNEIOSF8AXwrOKQn6btNdIuSwOSCfCC6OmYisM7TbOlzyKwEEedjOVZWKDFuViP4Oj+cugS\\\\\\\/hzXgMnsmls\\\\\\\/R8fS8XeK\\\\\\\/53VUhcUlj8k5UO3WMXsTLSC\\\\\\\/UziqfXAV39wTLpoK5439Q07dLCIAZVCe4PnEXbjI1TmRlDv5Nj3oombNftgFFMsct587wRgKaE\\\\\\\/x+3Jsz\\\\\\\/Gp\\\\\\\/ZNLZa1gd3hiR0dDLRVRUj87ip0B1HllnzCNEEnTxrJwxOlC5erdmluYnKlVUi59x+C\\\\\\\/uTjp4nWTR1i4E8kFUrYM1UPwtX84opsq731rkUSqjvNL+tI5O+EeqyDFhMQXzYoP9qbtZAlTont444dcRO99HON6IaTxI8DN\\\\\\\/cd8vk+DwOLEPLvgfG2Ecsy7Npy9YwqcNNLXlYUAqgxW6TJcphg4Kzdyh4BA6ZsXqozxmMbSRn7Cvxi8+2t8I2aWtxlzZRTHD7\\\\\\\/6dof7JPGYjh7HjAgnN1tE9oyBHMkfLZPsHXicmDKlFRVFiai0vx89jZlS4rfkgz0kfFLPuw9XnBstVdUmLJNYwa31b0NUF9+ZVv9dD2sjpRXUrBrbdMuC8Gb2B4TMynbrUin273aFEOOxKKlzk9aeBywKKs0hw6EJEmWsUzyMfseLJcx8SrL2Ja6goR0C9+7yeQC5yWPHOgrIJrSVJkPrFOBO+7T6cIKuaSVac2zSc6woIZaBtRlNEEZFO27\\\\\\\/U7TV5BMLPY\\\\\\\/wC091mDVdmrlpZTm7UnZW8n+7mPEzSq1XASSQoSwAb1I3v7K13B6pnlgveqC9NQs5yBCH+KG2v4iYDYOyI\\\\\\\/Cdi8padIs3MCPAs1Kv\\\\\\\/cXx7nSheJdnk7aPNApgdyYsbDco37jLzDhGK3NkFnu1+CIgUmW6c+WCaI\\\\\\\/CndPdjtpYNYaiGN\\\\\\\/Z\\\\\\\/dIkmZdedOvuQG4SrQMCtAKbGCr3j3iI6tELMQ4g5IQKvjjVBQQUG+vbRfLZeCA0FT44iQC4Mcae0UkxwDvkkyNhlnS\\\\\\\/jjU\\\\\\\/aLjESDwlkv4CUcK+AQXdGWmdUcHUR47yTGAreZamoVGuasYj599O223mUQbYW5Z1q7qo\\\\\\\/JtmbfcBgQte4dKPXkNnF59qcQ0howffLtfxo7W88o+iQ6MPDh9BzclBMb2FNgCa3rrNo8\\\\\\\/06uCvKC111VETPWt7pYKFB6ck9ellXsThgcAY26liPCO8NqXxi67KdMpKAvGg396MAhNsUzNZ9yK+AwBUv6WNCOOY1zR0H970icxW3FLt1KfdpCf4CsMkfDFwNDEXTq0mGAKj6Xj5wkGLTmpH4\\\\\\\/TSbkdNGLDjB0igfjtyJHZdvmJI22GyqwSPSza8BLu83SOlD9ujq\\\\\\\/8qbRJMI1yFER58ej\\\\\\\/Gf8lVlumaVIjj2YoOWdkeqpFJWdaBFUJ6\\\\\\\/n27ItFhnviT4zCEgvJc11CbI6ztJamwLksJ4McHccPL\\\\\\\/\\\\\\\/25eR0fSuAqnCQCjftA\\\\\\\/tD2sZtfTGGFvx53yiMp\\\\\\\/3xlf0urkcgPwAK6MT\\\\\\\/C0yTRxOIGmqBOT\\\\\\\/f1l3dhu7snu264xK0nCRuNqR+tk4VoCKAzkySyd4X9hP7jXxh\\\\\\\/OyJSXMOwLOrK71MBcGxkw0iiaTYmZt21ko47ifa+Q1puCzGj1qw5wsvJPsvslEJXGtr86+Vy89dUUkz6IUaQyzJci5r3rHwcl9pVPddH\\\\\\\/a4k9M3fXs1L2TUUjSCMbNRtd+khnR5aqgbT54fcL4M+rfdI36xVk\\\\\\\/x+cOEKt3T07X0LODaGIkntaUgUX3atou0rlC7ojkIorwy+Uic\\\\\\\/Y6ynokkEtNPSr79WjxCnzXN9dzqH1w2+2qMrhhKyQEsZllMXFRK0QoKIqQnwDw\\\\\\\/kJiXZkF\\\\\\\/nuy\\\\\\\/RB\\\\\\\/YvDzqDSw7J3K4USA5dNKnz3BO6rhW1YsDHl9Q6hUBAWWwEEn+CUwtMEZrSgIKOLNZyJL\\\\\\\/SZmwTglHLsMgmGc\\\\\\\/ymE1E886YupC0dN\\\\\\\/C3q9R+2\\\\\\\/8+aQ+kUpJijNFmnkfvWHaS9hw\\\\\\\/dgWQR5VOe5lUgudKBaoEn2ugmyATXjD15CsRkkmT1OCldca18ZlHYx1l441Fv8JIG3mk+VAoMtg22G1wpNV1U3hiqcvU1pZGYkEsqYE3EXYBPZM0fBIkiIKRSX2ajtLDoJjRqXfudhZGEC8n2rNQKu0rMz0NMSPJNDakwCbEj5RDSTeSm+def5cYVJ04R3jCMXlsiyDEa0XQez1+7NZ3d5d13HiAXv\\\\\\\/EtRC4HW3kRr4gyLDKhOLBsI2PxHu2Bosm3+WjUNpuUwtXXEwcFaq6vxPNRgnjFG125vDkSIzJgoivkd2kHu0vcRuOt4iKDB6XKPEd+gd0NCLAdUP4MsqEmJUYD5RMShK8vP9+smYNLcfxg0fkysmF5blscftAxKBDCWerxFiVwqmUts1N89Pk6flZJfcEbMfakKzGIID+k8\\\\\\\/9\\\\\\\/tM7XzGYcESGOWPNjJSmFTopADiYA\\\\\\\/2eU\\\\\\\/ddqrAk7cX+J+vkRumI0LZ7\\\\\\\/yjS6XETef1gp\\\\\\\/9bPb+LHZ+VzsO+gn5EzdzacUMPyfdSdH\\\\\\\/aJZXicfX7uyKkwmR5AOspgIWg5zXTg+ypmoU3QHbwIya7N2jDgXkKj+PdvKJVWeqAVcdtvX8d4BusPhA1GAwZ\\\\\\\/sC6rtFC8edXMaKCADClH2uHspMKn0QUOsa3Wsf8LQisPPfbfQLMQ9hqf5VNKN3HHYJfArmVetJQWZV5K\\\\\\\/TlshuaSSnmsh98sZrvdlEKgRIp4t+He9J8GS00c3sFn6ME8FOYo\\\\\\\/4qQYv7Q4Gnq2UHo+wpnkp10bvmG386f2pPvPK9HYZ9J3baGSVkXWwC3EQGLpyqDN7wKH3aHN5DOAHQeWm6aWsn0we3m4dJsNn4CzMi1vkCfUX5ONkPhwrw1EP6+E+jFMht04mRschLTxEUs+5YQGsdc2sUGmo5irJC04XEHlnT6y5tWnxf1vmmvVimEz2IvPbEpjcixZzTKGQoFiDD3J2ZKH\\\\\\\/Y6JA6rVv8oKn1DJvRfTY7e3JsNR3QTY+6hJ+cn75S7jIDHGTUak+Wtq90rY5\\\\\\\/IURn\\\\\\\/R6jMff9Q8rhoY03awaRgbG+X6JL65IERcylTmUKKJ27W8Ssk7k7sOSQnwVc6zHeO1Ul\\\\\\\/zUTzcIcLFdzwzHuC5jVUf1Doj36woHyMKY+SuCzvaXYB\\\\\\\/bttBKn7hX4iSE0mzrGAUWawSGdl6vaCATwI9r9EWsxvoV0Zs8hrocxH9PKJ3zWTtyK+brortjP82hsf64\\\\\\\/lt9QKFfYYtbO05Z9L70sncYTsNEPMJl7gRmu8RUTceIwITC1QwU1qG1U3iqe27jXQRBx65oXVj4MSz1ze1xAOAf2pRll7\\\\\\\/jbNhxR+Mm8KilLiWGffteOt5f6+EGxIubvjwA9cbVXwNJGCDXeOs3d78DqaRj148go+t5bRT+nVaEQmH1CNBo6WbZ9lOGChyWs\\\\\\\/mZB1vn47GcMo9Z+J5fwh9AabUrkYl3vY1D5F\\\\\\\/bQEHQYPCqCphvh48ExsP7ODweanb27bTVdD54hWAxfyZXztSXTLuvwhk0Hx64Q5wHhB7jYmnK8Q029ZteyCKJ1eSNRbLz+HFvUzTaV7sn1VL0ZvkJDhADvwwwXvk\\\\\\\/rpTGEGm+r4bYMH94oG6Nr0PkLkJWIC4khRbadd+TDrXxKLAEj5gnfrLWez+3taDAcmjwiPgD6Wr8krehtn3UBrP+lk2FUseNvWjfXMkLEkPfjG1ESAwULcyNDw4g5EXpBasYP\\\\\\\/cXAYAujjbgrQL\\\\\\\/5wHJQ4Xr6gqk4XvbiqFuvuPcK8tfu\\\\\\\/XjmUSd1XELTAAmVdxsD+m9qSKc9ca15S96cO\\\\\\\/qusm8cEhjWc\\\\\\\/E67Sw+LyrUVF37TRnanjZNnS2UXvc9G0bKkWou\\\\\\\/ZrB8DvSNWTFzKVpv1eo8g0XVgUMu2mBX2XZy3hJWiFkrFOWVwNhtsFtESWHbKrQDAOwmC64wzPJT87Znz60jm9SJPpPH6Ox1dVTMM3MlQ+C9i2uaw4MU3i3Sd6W3CbbQ7OznJiHHbfl23j6J0hHsjjs5wtanRkqp2PvhSfgZT1+DdCyvCyImOaveMAJkhteXyNqO+p99mV5MQXOsb9SQ0yfEb3ePu2o96tD4NgkXhJsljX8OJbiiS+KEAajPGdwzazoircSbTHw2VpdSCnl0PQ3J3\\\\\\\/2yuCuLBTUKyBeGTffyHDcG4Kzlbj68+4jfrvziX1mXe4Ki32vb0sb8ILLKSIlPe9ge0tgbhoOvrzZv6cEo0nnw7knW\\\\\\\/01kg87Q5794GcmCRD1R6NDxJCDGKkfFu+SPr3uhwRW4exqFJJjyL8na++Aogq9YySsYQlhpPYUBPjKpf76xk\\\\\\\/Jpwn69r5IGIQZuuABTu+QWNPxmhRFawg45i+1xZ7gHH85A97M+j1eCLutxy6T6RjzKOISN009GXAX6RoTBxHDkVBbtEZv1BhzbcqrmtQee4H3J61f8QsYPiwLbXl7eu8TLUZjEnyoG0URYuWgJSD6lVYDaMwP+Hw7tDcrF+KkhlDkzpILdcYUk02HftMr1oLcidROAohWdEvlhoKtKzbwPiCZ0WmHCKXr6a7iWM41k5CR3y3ERp34EZLt7WrnNFWNBsuGQQtpBSHR08TQq5DyJGmYa3cuIhxQ2TadHWNsL4+WclER51LqMmOpyzDtGG8TNjy5j7YvSmYVX+RLOsUZuv0MXDl+QDJ3nLxEa7ByB5lBBBkNm2MMJCPfZ182YHQgZTVKih0Vrai5niI+V30DsK7SP+jgN\\\\\\\/Fi1dmDPvqhKSI4x0OL1MP8MtbcirmlheR2546khrJv6BqQbDq2ITXuXLRwxm7OMOz2UvD72LBXJZwwIvwDNYkbwqQd7\\\\\\\/r481l+Qi5GxUPb1ZQ3p41hqbeDIRoQrAFuT8NpLF9WrQfR2VDM0lrcoeCEj9hNyFH5QVizRQAcTI34RK\\\\\\\/jiQcXop2loNyPBrW7h\\\\\\\/fwfHkC3bgbOOebJ6YgH4kReqyBuD49\\\\\\\/IGLC63mXSHp1HKAkjxX3nDUNocHVllop3D8tuI9MrrfYc3TdPUehDQBOLKdUP7gA2XkRxjIAGV0\\\\\\\/bykGHsMrzpUse0BxypFjcwtAN0kI1S4OM5AhM24yYV0z5WBWV4+NYLsyjNCWwwi0ePrV2rdL9YRcrXCBAOxe0KMrISlYUl2NVAcCcsTo4sYCoTe00gLAEBky7Wn1AF2dnrZmparbA0dcrXc+B7QQH8YU9AMxV+pZnKZUwRlJX73cFzs434GxrKeyQ712jFg8+lqPutAHMFIkJ8QEu9sxwY3mIiDHHwmHcFNX\\\\\\\/zwvrAtQtCaG7E34QujJcnkeis14UMSoJn4XWspVH935PeT0Ez+UWaifLiL\\\\\\\/onRhuPkm6u5XK2jog\\\\\\\/pfQgXjQJ0k\\\\\\\/HRLqt5fJjZZl+HDfLfByVK67pQqCK4lEQTQ4wtUJ5PxAYbEzvO2Ek4SkBoQ2X79EFAZGM4grAEF5p9wLdGeWBsxR9LBK5RPfaDtaEPyfETcNv3JN3GIRC3p8rZO6ldo9JTuLKPaE3JjIqCFk0jC5jhb5h9+be+oi224YyaWYu+32EzLxK74i2\\\\\\\/TmHhJQi2\\\\\\\/fq2qeh\\\\\\\/3wn+ZZaaBg1427Ht79khbQDv\\\\\\\/KK9uFYkAIdNd0vGnxvyDpZbsutl3FWmetD1hJFHmP4OXGM3bcfBioYeAacWV+nnxEFqjKGJpaARuHMa34O2jHadQyV0Z6jzgz0Sy4V2TRXbLe8hIl\\\\\\\/IC9zOcUuWL1xSmn+RI2j5Xmo+KRQERB2VOczwVclY4sTNvsXbQrpPUSqf+ghXDIdk91wOcbGSdNSSli\\\\\\\/h3WRmBjsK9PWHXBeHJJv3clWBMYceNoAbwflfyObTHu2kl9WTJD8K0Y5m94zSYKv+LqfLKy1gfGjDfEkXGlKUMs5mZ++Fb79NEGnjyp+kaB5xNteWdpLlcC4v6vPonHD2WEStK1xvkOTMeG83oziF4c1I+ep1xDdX8vz4d6xgJRNOkMmPcw8yiAeuz6FbGWE5DDbCTOXLHXo487rRAK0Gv9TOJHG9gKXNgH2Z\\\\\\\/3I0xxgWucu\\\\\\\/JpJuQlEPkM4Lkoxl5RV3VQJHeQKpPUTIN1JHkuF+zUW0OBOCoTGEV0X4qlbwvPVrFLi14uvaDsFmp0X8FYf5IWu0xLjAfeFjB+vlRKIWmPo\\\\\\\/D37sOR9MCSMk4A9plCwceeTnWDn1REfaaY+XOrScFN52ejRVUJfp488KDhbeG92j\\\\\\\/Vn6oJTUioHnowZOESuKwBYtNSqYOx7NYXbeKPjYY+38k5pvDeoUa29Z5t74JEBCGzY\\\\\\\/jje4ptPow0EnNDPyxj7q9wbku1sjQbKNakz7GSd4T6vU87hYpgV52fB2ZT1f4jos6+hROONvhbONy7XPzdlEbxxx1Y9kfZsFCoMKSY\\\\\\\/i+\\\\\\\/J3Rv+Qd7lQSA4CRLVyRzaFNi5jmVYt2\\\\\\\/Kqs2AYZjU1ujjs9zsh3wq4jLkUBYvZrvqz7O3mHSW4KG5+\\\\\\\/NkUYCkxiYhU3nGgUFTLw6PEmH947bSVZArnz4rcN15stjC7bIl5zoS0krm12Rgdr3yKD7bFi0JlCuDJeQG5qZrJzjtwGOZvXwW3eAffoGauOXDXC5hgvapbSKUq283YblkXWkHZlucOKxMyeFikFENTAmzB0Q0som3+OhF6r0wKN3k5RyhDeNxXIbzRmZ7b6a\\\\\\\/u5G6QlClSbd14XWhV32D\\\\\\\/uhXpCGfnhPZjvWYwgRCpHU\\\\\\\/zfgvDm6A8U1IJPQ\\\\\\\/YON+PpTMfJZZbI3L4qgcSbjr47\\\\\\\/jJC8gqpiCIQyEGNENetAViadN15jQhYgRolpnxmg16BQj9guIRT2a69jAhKeiFMrORunkFkhHOM5KWfIlfIn4SWqFjl8ZzkKuBH8lmWVCgV4RIOdK6jdrpYSlRseELHMsFNIyBzKtYqPbWKn0MqTioT\\\\\\\/gTvqrlZRuKP\\\\\\\/ZXUK+n1Z1sNIR+LZjQxuSDPLo2TOtBFhfkaMlbOguTSEIEOf0MbpR+YHuf\\\\\\\/FQO+FovZ4koKzcrvlXdF+kpaRf4wY+8rCXNK+Gw\\\\\\\/oIMZ6NJw99LBZ9vvm7npqws1CV3koGoYNtlzHwSXQlEkQbwIstkDRqHhkx3V4i6dt6zU5C5I0Ji3oY9p4xL2+Q+ZKLg3WL7kHxOHsxFzk\\\\\\\/hOYJKVMS9+q5DfdQCjmIl7YsXv4Ao4s0F8wdEL9BRkXyn9BQKSRiHXZWlwCUVQpjH6YIUZ7DZZj\\\\\\\/D+gIVIjmzvXMo4AND0xT8MNQ0i6MLP5mMHigwzv9UMy\\\\\\\/NEbn7uAU\\\\\\\/lpdNKvipExbjI\\\\\\\/S0JH+7fgzb0GcPxQlnVOzBLsKP51YFGuyUJqbZKv62bRTPOxfSmMyHurUxuzVluRmSOBX9oTpcO6K0DoEacCS3mSeqFelFuY9U1EkH3aaD3ifcob6SUJL1YedzensJ58Mr5FYSY7ghXN+eEnkBon+IDmUfRkRrJlYz8Ps5Ar76TC0yRm1uzmZ7j2nas1CRvhI7hB6xA6zFonknIR4f5E+SjI++QbKjG3hTxCfz0JRcJDp5r1YBpQ589niKsokk35PO0ZIwCPQ9R8Nw6QSiobMS0HYSVoJP9ntK0GegyQjBmANkNaWPFiZqgT6faZY02GmVnvkH4zrakDY3FwXPM4OxYjHvtHMymBu1dSnwQBhvqX6wuMxcayn85EgWPzre3qBRdXObP4oymZHFLFhBeIwwJRqvepjUmweJN0a2AeTa2tZf4Fl8Jdn+XAN8MHAOITcBaliZ3A56Q8jp35hFys+QL7rokSz7x1g6Uiq8AKc30nzYmckMby1GPbW\\\\\\\/xB5H2Cz2TbzyrpTMyo9t39X7Xw+3Hg8JnVDk8n4b5O4fF0KeMIYAarsnE7F2e\\\\\\\/7KySFhMOX9\\\\\\\/qm4Uuxv2eS\\\\\\\/lM4BH3bO6JwzuTCty65bvrSym6yIbn1T5SDTJRCVVb3I2p8qVNNBFxF0fc8YEkUXu6OQDu6FB4FiAYxe88jT+T9cay5IPJhcys4ttU0LWSolT\\\\\\\/7zGOOjAukWopgRAufMgHL3deeBaB09HsSzmG4HbwnL+zK\\\\\\\/kfun6oycFWI1b2KDX3SMuMqd8C8PNxw792JMhPq9IpFQMkpayfxwGhVS0zZyk52UeamOaemEC620G0RqM7NoqeZ\\\\\\\/VVocQRv21Zj4DKZItTj818QQ40tqEXQbHSLxqVDqQvul5XedyCBvnSFN5thwktOxBCRkILRUKl6ppbVNPeJUsOdzeHj9jB9ofp2q4BobXNYEuzNf84K0O8aoueDurZ9u+cQ9an+D8cT2YXdOmapx5QQAhNlOdtk+drT5DNgUK9pRc1ZOnkw67rb6Q\\\\\\\/zEV\\\\\\\/oFxp5RxIDYQK79OpLKAS7xiJN6hslTfRMJYwu2+BmlzAIANohyNdD7huQTdjiy+jFoJw06qpFywMnrytCfstn6KVgD+s1aVnZw77UFbtnANCG6kRzdD4XYn9M\\\\\\\/fRur0hAQok9Vk9ihrfwVVBeCBj1lXdiDdFC+HTOMTvcBgVXJzAJjc37veGpx6QE9X2LPelkQBmMtMFgnNKQ7lSnjCrc8MOWrdz7PXSEThCqAjsJURJgrvcDwvLG0NmxDjpEaJTTEcT+O51odZg2UccS0Jb58Fv6iyyg22adFhU6BWRf\\\\\\\/HA3AkCj\\\\\\\/FfuO2Nt2teVWh7uqiweOflA\\\\\\\/yilUqOUkLnoYmCXl1hAgiWq7\\\\\\\/xBxwclLYKVWke4qeQCMhz28+D8cFLd12Q1AVSoTw2AR6P+UxJGSmMaBgnMmL7DoKRaEeAhbWuaLl+4EtVpnIZqWdgCwIyQsN6WS7EVuAW+CpKLWmTaIMpr\\\\\\\/Oo\\\\\\\/AhDsOJSPa2WPEMA095HjCLGYxg3o33E66FyJsZuB1DIkoKo7RcSKruBco83BZ4OvW5qwlkF1qih4OWBR9UaDWz2itAnAblszstTOR4WSu4l+3mr0H+56\\\\\\\/p2JmDQCRYqBikFMDn6kO0vfFj5+g7\\\\\\\/Eq8cJIvlzkvJOvvNxDY\\\\\\\/Plr\\\\\\\/7m3uusZHGJQJpBuYNc9Fh0vuGzsnzkLtfBj+J\\\\\\\/KgNe4m10fqfvtCXG5cJzYM6AFCPgi6upAV9li5pm6sOAJbq4MQHJPcCMlLYOGX1GULuR\\\\\\\/f8jpyWKx+jL88X1GfmHmeYSStFLfsYAh3lS+K1KV+PakweYYubd\\\\\\\/hNYpQ2plFXBveAaF0YuNH8QlTPNTkqoDAQHRO93hqR+GSu\\\\\\\/hfyVnAid5bUJfqVwcfjJ3RF2QGOzZZ4JX5mNcM5mwdnw8dCVof4d2bQXTnNeJr4vGCuhwYUNRXzwi9CkIOSxEyUVeVdX+LEDNEGN9ZM+RRewj2ZX9Lb10nzeYBDeWjAz5WTZhEtU0fhW\\\\\\\/2E0E5Ul\\\\\\\/TioLF5YI6OhTOo+VYU7HIW+WY6HsCm+7c54DRKQCU6KVtonD8jMwigCJUOv0I3xC4UWkL0CpUx\\\\\\\/0SAdbbWHOKONsgce0YoLu\\\\\\\/HU+wgvyRRpOA4t+w2hw\\\\\\\/LrLU+wtGTDReoDk7DpilFxGRRgZotPk65jlXvZZj+XB8w2EYA+\\\\\\\/P24fvpdY8Cbmnn1KpIhYPCOntzV\\\\\\\/3kYVP+41W47aMboy49mT6j7ralSkiajy+RX\\\\\\\/Lpz5KUxzn37aQn1qAwvbf9FnGScDWU1tvrfgfTIBEbM2BXd+o1W8jk3IiS3OPHh365QK2FJ5TVVUK+eZeXFRjLE4IAMnttiD9iGW8roJhA4vrcDHWdxaaHacpuU6Z7JxjekiQAzdmNxj5GEUtXGnLeIXplBCBHlGAZYI+0SN7vKebX2F3v3FPV0zydBH8RgooUBRsy5tWHYZYRN5GwCYiGYvAU\\\\\\\/0e\\\\\\\/ixsPRl5x9H8ylSOH1+tY2exBpYdXEsJ9ZP1gMscCtOZ5ZT+VHIDT9eBpuvzncd0rqIHzovwg04QgC3eRnfAgBYibnbmy8JPwxry6um1LvDBhEFXdJ5yn\\\\\\\/A\\\\\\\/Sr5xL50A1qduf7D+GDyNddTkRMw8u57QE2TxV3VMTSiYoO8O4YZd5XyIGUF6SONERinSy7LX5QJruOBoz2akM4afQtBAQokAgAs6wt\\\\\\\/gUHXaHXdpCFMNpgF+69Jo2Se\\\\\\\/mPx7AypZpCuJi5pLl5ouLf+OquZfzo8rCy\\\\\\\/OaGKi6c81wQ6K7P4VxnGGJRfBC1AOQxeUXpv7A2a82qQA9wqrtwL2S9WPERB2qHOGci3shJrL9IkBBmnrMMAIzpJXBxjV5l8Rd5xQf3V2SIjPT0bJw6m4MUVuW29hAGrfQ20+jxUf4IcGAA0jkgLXpYo2H\\\\\\\/ht2l469VBkTaqeRqIekrEEVmmmu0qRISMBf5m+37gvOiOIzik4wqVNsAWZql0AF1so0zIZaUz4xnMEgkm6w2jGf43zxeIDdlrwEERasm10qSzexsnzy00xwmX7MqDRy18N3LTP+BkKsRXJN730kFE7GhOXr29XHiY\\\\\\\/YCWMmKt847J2Tuo6Gm06+AIDQ0Y9PMeYv2UPTR7hV\\\\\\\/Shg+Yp10a8YkmYcsCRzpR4zOVdeT3cLt6RoeU63XTqueEjiNOb+a0r+Ta6Gy2fFcBXDd0sykuV6y5qFREW91b6CSEESemouc1bnvpJSIsmTQMzV8AvFBxX5Q+xuO1TNxRPsb\\\\\\\/u8x\\\\\\\/bjubhIiVU0ofNqUlnVM00IdzVL4MN\\\\\\\/Ue\\\\\\\/vttfwSWjr1hMZB\\\\\\\/mbQzwWQTwaTdBicF81gkEGWNnDmrgXRD7JiUrobDmu6RR3eNOWo6n3ZPCuyyNGrYLkeBxJFdzof9px97Zlxkcswv0Wk2gXsnXaz\\\\\\\/0\\\\\\\/IKavW8lDBNePJjH8\\\\\\\/mRcGLFRI4zrMXP0IsXlwLnyAhAE0LfC9cDfzXdz6CmDzUk+RYyYT7LqlkaGJnTkbbxIKZUPUTDmVN2BsOJdrZ187JT1ZSeVfcmCSttObFUQ2whD22lGUnOr0gCXCwWRCNqfq02vNGlHxnlLvjyXyVrUrjtS3QL9sbciudMeCP8OHTjY0jBvuWVGUT8ynJwyr\\\\\\\/E7wJ8RvmhouPuuGwB02rCoe3Do8DJvLL+n3vh9RVnwO+BKhqA1IyhA\\\\\\\/e6w1haMZGS+8wyrVEbprv0x91TalUpRbSVhN9CSMm\\\\\\\/cXo\\\\\\\/f17kckV43FNdMZi4lxSvYRc3qUPdJf76crJXaqc4QHhTazT+K4SH0w3tFYD0JRGyNI7iPbcB2Z4equ5p93Fr0K6T6PEfMOQ2tDDKbV\\\\\\\/L0v6Uw\\\\\\\/a6O3eEYxjyWTOte6RqY0M9nbrIRYheAG87SWMAijMPO8eFRerDs\\\\\\\/SsEucKwo4m3C0pG6Y0cyiY4XdPC2slsjNvYKwBPwOYLMaDSnqV6MOuNpBDDiFgz\\\\\\\/5zNszKTSso6Ee929LVBpe2qzsbIyrUA2Mxdy1DvwkIvqSOupZaeCCTokI1Lz6mLLh+SkSuhPx5zeVfJ4uHtCAvv39UQ8zUVOVsbiNGnkVAlOlO27O6kC79q6eQODLGe13qsdhLzyUykgw3YouWDbIPqXMeWIuzXKkqXLaW3yMPCtQ3rK8MEyriLuY14dUaSZsfFt9ZGgFzLoZ0\\\\\\\/poeU0+fCUTdQQB+Gk2\\\\\\\/TQUt2ObjGX7UPJmqknVgPFHmHbL2PiGAImvf8UP7drZqfnJM3lp5sy1c8x0hr\\\\\\\/UP2BpDcu+VXukjhFZGecInbujgwQnctZi7lmDOio+fAI2FpwY4H22KoPEFvSr7ASSPBE80uS2mIidFOyPrIlrVYybMUUZXNm9fvcKYIi5l9TQKYLtatin73Tt99OM3grNQUOps3479VPgTh9t+STVSH2EKG8NyQILwb6THSQsdVZ92CmReE3+yBvJsoLSeOa3u0E+aTS4a3aPZw7lZi6gLs0mMxvMsE9L0ufBp56KLiTNRpn7pNOBtyevsvIZim0MTDCpofKMYxhRsmbSICxrSE19yp1YPDdgS7NS\\\\\\\/WH8M8W9EkTuLbjEgpDl95wFvJDCEdPJm9WNcOz93TFXZ7zyt+SsDSzFZY9JixWDHndN6rbOnmnBMz6AYlur1c442+UC3GfhIe49aDg4cgQIgrt4pWU+4nY8lwWfOacPrB\\\\\\\/zCQT5JkAECW9zefF2YBR93FjuGXIyat94H0sPVgm29cq280EgMqaR7QGAcPeCf2iW0YOXrog3PlfgVpL8kznX+jkfpG\\\\\\\/5l5sYIynq8Q43mXtl1m8YUn7cVWa9Zqo1ojbhbKpFWRWyR4B1WlYfMX0aWA7sm4XA1sGoVLgN9MvWxgpObNZR\\\\\\\/j77IOQoelkK51DN74dNEBAYcsQxWwoT4Is8hqFAd6LL6vLR5nJJPa8qxZOo6zkozsyxVjkhiyKc0OOMVtO8UbG8z+EKQNxU227YP2Oj3AHxJgT1QV3m1hlUV17WE93TxYOcFuAfyQg29OPvKJuPm4YnASoFchEo0n9oeZf9gksnjQkIlD\\\\\\\/ss10UTBVLlnaJWSqsQN0xmRTVSHddcp\\\\\\\/w3hzsuydMhFQnv4GbzqTs9lsynsiKL7QLzWntgdPDaW4jJgtGO2DuaIvsiuLWgcUa3I3bFXZuTttJh5rCnoSsIz92cusDeSIAbf\\\\\\\/b0sp7u6dMrmr37E9XNoR0ZH8sH8It5VYfhj82TgL\\\\\\\/IX5LyYEOmSzTCrJrJ4AFX5jODO+RM6o9U4EmdcrP7tuJifW1fsc\\\\\\\/hKu23K98hhVsp5WZU3naPQnN97KoBRx4jpgOkkOJTGDW5EOwWdBFvhTnBTE4Hj5xD21I9MkZHaS5p9Ka05u47M26jP7KE4A3ogPHUJGjBlJw9xw0cVOGKY+rMI1HqYfshOHkH+HwCXqNxENQZ4EvLSfLE+391ARMkovgLwaDNzGpvOZvHcb77\\\\\\\/EiqX7QodTbISmEftiFfir9JBr7iXdwacVl4+yjpgxKXA8GyxrHjW1JIyJzLF1YaWYOZbtfK7tyfHpTtAR0U9WBVQfWwm4cBRx\\\\\\\/PEGBmQnxP+Klk5V724eckn2TP4qm0HVllYASyd27lslz2gzkgkNwU7e28RIgC5tkWOUk6WPdooXFrNfbG2DIcVMsBgeY+Zfz8+LPpLxYnm5FS0aqchH+ycrtQvpL4LOihdV44+bamTwoysJsjjds1bk2VKLnNYcFMORGhbtJ1mR75jbDbUvZAqoE6h\\\\\\\/qami3x69Ef5SLe1jMlW4fGBqe+J524ZggUWJiOOz5dF+Ig4a6wt9uQ9kEShPVduFgUaov2UDWSzeHDiDuhnzlEtni+41CCI0Cx78CKGK88WuRdHnngDOHv8lkdRC2br0QxXRZpn2ZpVcVTmVt1hrhNcjFoZYy4kNUXNnU\\\\\\\/CWo7C1jCO644mTGjhBkF+LM5n4lfUGgMGZxGzgtCPRhsRsqmPxqSfy5IM6QAoFaIcuplVB2VOGKOa6MRDbz1m8X1Gf3yZzHSKEViHnDjbMPLM5+3dwgjVAbs1eKl44kmnAqdZDggAuo9yNlc3WqdUr2kAfe49DIXu6WJr\\\\\\\/kQkbJ70owhN8HnQ5+WcD5ZkUaTo8qzjZRKAxewXJz\\\\\\\/\\\\\\\/kh1vg3KAjV75\\\\\\\/wgrarf9WLTeIsDue0wC75ilVfNugnvFvrhQ+SERbKhPI+OwJ2J3iVgQ9MjO29rWTs7vWmaZgaNA\\\\\\\/iJ278xUnJFx4vjL96v934wZAmfrZZgE61VM0tVhrVC\\\\\\\/LaGxd7N2mjum9VfovFreJcikeUHPjG3kFdCVECoxYbXRbcG4Xn85Z0mMb3REz24Cep5FaiUMshlM2S1aanRc89Qgs0ZQ\\\\\\\/g4z\\\\\\\/O6KI6adIwUKtHk4gfHEmGNnNRaDGBwTxIe9TZflN4gdLuvatIWrAG2ASd+GigeFvG2p1lkcVWxGLUcUboKC\\\\\\\/eYTZGDc\\\\\\\/u92l1YS3XkzY3l8sXGVgYvPZ4ITEF8Ex+O4zszF1+rr7pgUzRh6ZlJTq3zk2uXeZOuApePaZDaSGRrOU9D29rTVSb5J2NxtVcxPIzgXEGeUzNWkfzCwqwwNZXYyJTnkesAN1liWIZO74skPlN1BtFfS6Uj0usI5klDpPE1vN8WeZj19\\\\\\\/MGLz2hcou+Za4R5891+aPKe97tGx0hIcnOAZ3jx9HYnZ9WAqc+nEuP5C14oou\\\\\\\/CaEWVHttxaRqxSpVvlYIEimGK\\\\\\\/qA5TWND4EPmGyqzIG9hJRIPxsgFWrBlCiL2BrwZH8cVz4hTi4ZvvqFB51Ki0sHU2XFiDBASt66dRYiBWvUbPrtvwG2RZl7rTPLZuvFcvpt\\\\\\\/4Ah39RNZroHFsqO0KlgfLpp0HdWoVGApvfL2NPLHbwz7HreJxhmpwOap6WXUW5h1t2j5VnkS\\\\\\\/ViapNFjFUTvzd8vYQB3gIEzK2GtaeBnY2X5phpu9fkFBYHCHyeR8iAbgBZRCpJNqmNKAfBdK1X3PAirGzxp1K\\\\\\\/pAq7DZXEqZIiAsdM68HoOwuIkRaQImId0SHSImeVCGJXmUW3otxT6D66tWtSjZnhmH0Y3irvcoPcvmOdl5SCD0dtoF\\\\\\\/TPf5yvnWxNepuPXS\\\\\\\/hcsmINBUy+02EeFgdul0PtqcCsnn+1XjtmUkFiw73j2evSmn65AaHJMYl5J0UY1TjpP5xUc5E6GvS0iBP2fn6qefr+THWjiVe0R21AEFLzmjWXnc8Ghsl7BHdM4FSFrEDgF8qMTpryrHeqr1iIpwd\\\\\\\/i+LVBrv4id5nd94k211Rg6at6EFeKSl+wOePPtQh8DbDS5N3x6JNWlrnc1Ti1qYLl6StIfMie+Ryb2zCskAiMP\\\\\\\/ODEhfhr+6YWUprXLWkwJC7ivbVs3Q3dukSTA56\\\\\\\/aS34RWgqn0gsCuHohmC4EtD4DMge1lRnAjG3PQaM5OrIW1EyMHfQPCgmDAw4AumfQsVCHAH88mCWp+5tOTrkH7iYRh7R4Qdt3OjnC9La5GOPTUJjr\\\\\\\/QjAZPZX7uGJLCN2ZwK3DsE4UllFK67oEP6xukqEQEBSZrCtBM\\\\\\\/AM3XIdy4mqHsbbG9mKfMjMgHA+YUX2QLy5BjzT527TIuu2K7OZ8F2JVNnzfqSQu4\\\\\\\/LWl6ud7DSxin87oLtsw6nDG7velkzWij1V11E2gl2JJGPtjglsp8giIYC2f8Y+EBciqEssaWSG8bhQ2\\\\\\\/Dy\\\\\\\/Rg8ahkbKfgAg747ZuNgcTo2yr412EemTtSi212VYgC8Q++iLdceo87\\\\\\\/OEwTPuVjMsh+eZvOwnLcbiestrD53F7EGztmMH\\\\\\\/3gYfcC+dRBEDzwPy9o+JWHfRocZJtk+t+qtJhp7S85gaiUvokzyb46mb+zlB5bzucEzyKI9mfNzoofk9EUp+NQ4BZ4AUoKsFmfqMTK2m6qTSw+Z4T+J8Zh9NP3DhVkC7L4oGQs2W7T9tGYYl0sSNcu8o4kZr4sZp6dp1Cv4P7cNm\\\\\\\/UqEOjhxIYRtPS4rgTOP0lh11zs8FnNmNi6aqIxKC\\\\\\\/ALFexnFjI83Zaqii3Ep9VjfqkIxxERM6FFUqtfm14AaWB+HfmMJFfqprJwafCy80WpYTZWdJSMJUz7efo4QDfhuxlSP8X42i+\\\\\\\/\\\\\\\/pzE6GrMfh9OsobKX5a9EvIiIrYi4E5B9geub02HcfK+LbN4tqilgxqWmmSDuilcBXgyGlcpz\\\\\\\/XLwnrN7pWvfwSqhVJIMQQFYmJcQ7qQ+3z5KWdhaDZUCIyusMwyTHB4ZOh5tWm92DD0JocO39o5NIhiKbAjMYHDd9aJJPU2afhOH7A9j4Vtv36TZqirKRuTJcAKTXlRcHZHA8svibR3\\\\\\\/6ZJL\\\\\\\/Z81ISX9Jj2Q+28bVH4PsQHX7k7c4jzCxyvcjVs3a6kYSx8mBrSpKt3CMt5oKl6TwCzC54i92aY1uWpfCMdKf5dusPNGUUsZdLZacdQNpIqCwbj81xPzAbazMPNfO+FIxbc0bZAZ3loOG2AcMCFPelhLJeVU3U5NqcPJzHU7tbVs4fbvEdsNTP4UQamDrsxvYiODRzV2Jh5od5dszNitVayuNTKtqq9Z1lULnPDAp800qYat6R5XJUPgx7dq8WpH\\\\\\\/fiboi5f\\\\\\\/SS5RICzHZUOmaFSc1YCvw\\\\\\\/ipJAW\\\\\\\/vbCaOh1TJLaeNoRIOn3LHZ6ltWSFFK3D8nobX19bXuOgNeY2TQfpaDpkgVAU8ZZN99EWUZLPpsHBL2VxeBT7kJ\\\\\\\/593CYcFyfeH1CyN2Sq5gK8XRGRMC8jWMuFBccDPPWYbM3YeYvCYtjPGMUeNIPXG9t7JgoYvTPckaFDxPUDFNoqvI9+jg4FwQ6keKqf91ZJKDI4xUi+33blrjfZ9qiXh7ZcmsZSvuZ7waxo8QJuw7uyGcwRdBbLRmul\\\\\\\/C84wodXQHD0OXjqBHR4Bf+vKN0t\\\\\\\/+Szf7z5aiXYbxBFh4A9YfYG6HLG12WC93jO2tGzEB0hiJ8ho0jXA5DeILsBVEzznL1+e72Y2sMroyfoZGK+xvHhKWxae\\\\\\\/mHfb5UKiDb\\\\\\\/uRo+xwQJwI80XRmpxlc9pIe\\\\\\\/if74IzN2daQI3JQeLIdGumg9rh1I4C79WC0e+YW0ANwaY0wWwgoBj0UylW4aHMGpMvt39kdyZiytYydI26yodnYwoF35uKN\\\\\\\/qyGgvzAhvmeguuUXvT4jUAqCn+ZaHzlKifot4HH0qeDyQ0tAjDwRwv9mJxFfWEgz3PBSURJVKisX4PLkk6o\\\\\\\/aK5cZVfo1h+zkeH9DnnNSv94g8Q\\\\\\\/L6RbHyXE+QecJztUYmaxw5VHsCEt+azIfFfDKqX08\\\\\\\/8BK8bfYoh7HVDwhjLK2cC+51Dbp8Cf0r5gVtujiVaGuQmEkkJauTMKf2oV4CgKp1mNEKKKBUUDnC2P23L6I5\\\\\\\/VCB0kDtAYpZ6SIaK1fslKFwvZIdmCO2jQIrIPNmM\\\\\\\/zCc2CtmH\\\\\\\/tk\\\\\\\/8ugCrpmCqUTRL\\\\\\\/1RIumgdHrGEDq6TBgZ2vcwdq9gdtijMlAF\\\\\\\/wEXMVM91vSVHoQjsE2BHYQojBsWqejgGyUXNCx3nhLf0jAqnpOX7JOVERUAjBmjaW4gjZl8wdBrRr98J2BEB8h2AXWrOafgUGi\\\\\\\/4jr9Hq94ITJ02jrlYfRt5zjzq8Qw7t\\\\\\\/boFmXFgOw5z+54jRwZV7EIw7FYNOX2zEpymc+MBYvWVkucLBn+8kBAyxw1+pRWGIAQ7IhHIy04PfiTxzuVjbMgzr8w\\\\\\\/UGPoybkySjk+SjEc31phjLAPIJkChTeSg5cQ2U1P6l6MHr9gDkLVmxnQEJDeVQaA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022c7+EPNr4evNe6ezxx\\\\\\\/3BKQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002266bb981fd9b726e159938eae95c16de1e81ba96b59c8d385bf17fbe615b3519b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ObluF0EyG7ndoA5gwR5KOYiXEk1vfS7+BWA5qXjAub4DzQUDeOZJIcwcrt\\\\\\\/bGY2d496ZUZxuCrJvpu7owRZKogcBK9Ra3ku1d\\\\\\\/r3P89uAUSDEb2YfRfoQqrmjndyCAlaX5KW6n6p+QPV2PjVfrTiPIgXD4HI+f3PGJWz+IdmP8H595AEJlZdqplb6Fl5TLtrd0O0WftQxv3\\\\\\\/Ef7DIsTAUpvGGOshYwDVsDWpvbGPMdFCUtXoaPvycLCz0E1AQ4LEfrNKYaxLUqSNbPF5xcgcTAXoOIj5IYcYUZS8E8EpXC7glAABHAIxxuhdF5dTk3sv1bbY0EfT+qugHM6XRnS60KQm6r\\\\\\\/+D1foxCNwjvoqEXVE9JTDCH2UO1YRrNqZt4gxQjAEm7piJmfVq8e1YH6LQ\\\\\\\/70QXUDxwuTbm+oRwMQbvsFRqa9r2USqMgSUlfRb9knOKKvVuNy0BO\\\\\\\/V7urx8WFdXxV0WB2s2X1VqMHMkeyVauZRcOUX44XPmn1NS0Yloi1j09l3enrLtHL5fuThu77F2WqGtlQ8Wjlv1UlY+LsZTrI4BECa1qHOrSNnxrt7N7X+TGpIGX8786WzDwCImkQnzybMsMz8fKIspaCvHEzCtdPZCZSCRS8Lzjl8MwLIUMJvDYl4EJj+snwjKR56ghqQW\\\\\\\/sKhCD7j8zHrPbVnnbyOpAWGBdeG1I7jeJ48nGdFdbpCWzJ164cpnt\\\\\\\/i3kk2kmeSZHgWDItL\\\\\\\/41QPMa8sY7T1YLUqVKm+4uzXVfz2mNe3noZIR\\\\\\\/h0th0p2vq4AV8WbH\\\\\\\/o5HJ3AUSrJE7rKP+Dd3LWDK9UGNFfuxznRFvCuCSH24cmlCCcZfvPVUg6A6hR5HIXdNlHz+hFqWhQOrfHmaotwPnsgMyktdeFXKxkMBUophdkEdDhH8spkpsgku4gfKFYkDZXiW\\\\\\\/UVHavvrImY2TCrlE3htUwKRwXemXezgrZWMbbbU8Y9yJHT+ltrjDZ+e8LjQ3gwG4v9vabrXvD1g7VlBhcA4EW8eWtgFOSNdIR9k\\\\\\\/OdrGJtk1LJwhbuiV27rZ8vD\\\\\\\/nj0uNCsfUawnciTetljmQwe72geKycDqyROWLxl1JMxmNQyUPN2xsb1nqqe5l7zRfYILjt2RfQAjM0N1fcVthkEAubdU6XhGXdrVeVpxxmAwS2KFsRLuQshigME6sn9EY2MaCHhwi5iG8++1sKOBY4iubvL4LZBsXHVnmH49v34BVMw6D9HdUtPu98nEggKZMz22Nzb+8yHUG7CZDnoZ3btiV2SbcfAPJfjqvqrpGkCeRUXFTiM3n7rPuPGYudQOs5TPN1v4RU6QEbuoVJLyhddwWUnUBwk4LU8xME0lhH\\\\\\\/OIrzvKjekM9HXQAy6xt2c4o7gdM5m7rWo15AujshUaQWbWoe59C8W74pCu8r3oqS6aDLRh3GRBGPB6RJjvJMpkzwvFTNiMMyEY6JwStdAiV0AJcnrfnWIOwIea0OUSfwACi1F55b4VVkYrvqlybyNBT3GUzmVLq62gdtxy7KnMPEHXEE2dwailnIGUJQjH6faRhNYLq5v\\\\\\\/uREoE0RIAxXs4dcVJqF2K+D9zvwTEVo5R\\\\\\\/lwkQ53syzn69XPgozqEkhqeE3p5lGhAY6IpWOlNhpNkTM\\\\\\\/hPyP66kk5LE8GK49\\\\\\\/fXAF2RZMbV7hkpsA6+dmuACDNlGmMMbbhgI5qNASf8CbtfJbs+30P\\\\\\\/1F8oxnkjnFA4bokCaSAo+\\\\\\\/3xkEHzbMjqwo48IVE8x48+C4d\\\\\\\/DerotrHCAZQn4Ahz\\\\\\\/tCBmyC5o7zAqk8hG9kV7Td\\\\\\\/HAI7OXwKOUQcbUTYenKau4SYJFjTtv3ZT9t5bfrrd\\\\\\\/KM2xtxaRnlRTbHwrvVBMCMG+F0DW9k9m8zK\\\\\\\/QtO+v04f0N+MwxtSQM7+cqNiD2+F+rjXOsTLsd374dPV53dyLe+kW1q90l0iQ32quS4wuGWSkXeMfl50ZMtKLCNBvY2XUCQRiPPP26Rp64x16wq\\\\\\\/DvwAzFvmfcaixML+Tr0nrUR\\\\\\\/3w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Y0VZik\\\\\\\/DFCiLfpqSm509ow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220cdbafd611d4b2cb7e8486bbde2a10e4aa67cac8eb32764c39bb33dda362b7c0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490970 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3143" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--Y5yHObKm8jjzZDjPXVIDHzd1H6lht-mUJhw25AI63k" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3143" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xb8o7T2yary-AResSLztwXIIL8mgczUiP9eK2pqLrHg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3131" title="Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3155" title="Mapping Quantitative Trait Loci for Tolerance to Pythium irregulare in Soybean (Glycine max L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490970" id="node490970" data-pisa="ggg;8/10/3143" data-pisa-master="ggg;g3.118.200162" data-apath="/ggg/8/10/3143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Low-Cost Assembly of the <em>Drosophila melanogaster</em> Reference Genome Using Low-Coverage, Long-Read Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3220-4927" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edwin A. Solares</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2414-9187" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mahul Chakraborty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="3">Shannon Kalsow</span>, <span class="highwire-citation-author" data-delta="4">Kate Hall</span>, <span class="highwire-citation-author" data-delta="5">Anoja G. Perera</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9474-0891" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. J. Emerson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Scott Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3143-3154;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200162 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490970" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Edwin A. Solares </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edwin%2BA.%2BSolares%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Solares%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdwin%2BA.%2BSolares%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3220-4927" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edwin A. Solares</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mahul Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahul%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMahul%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2414-9187" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mahul Chakraborty</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shannon Kalsow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BKalsow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalsow%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BKalsow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kate Hall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kate%2BHall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hall%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKate%2BHall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anoja G. Perera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anoja%2BG.%2BPerera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perera%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnoja%2BG.%2BPerera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">J. J. Emerson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJ.%2BEmerson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emerson%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BJ.%2BEmerson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9474-0891" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. J. Emerson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jje@uci.edu" class="" data-icon-position="" data-hide-link-title="0">jje@uci.edu</a>
<a href="mailto:rsh@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rsh@stowers.org</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">R. Scott Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BScott%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Scott Hawley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jje@uci.edu" class="" data-icon-position="" data-hide-link-title="0">jje@uci.edu</a>
<a href="mailto:rsh@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rsh@stowers.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3143" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490970" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3143.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490970" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3143.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490970" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3143.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490970" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3143.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490970" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490970/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Accurate and comprehensive characterization of genetic variation is essential for deciphering the genetic basis of diseases and other phenotypes. A vast amount of genetic variation stems from large-scale sequence changes arising from the duplication, deletion, inversion, and translocation of sequences. In the past 10 years, high-throughput short reads have greatly expanded our ability to assay sequence variation due to single nucleotide polymorphisms. However, a recent <em>de novo</em> assembly of a second <em>Drosophila melanogaster</em> reference genome has revealed that short read genotyping methods miss hundreds of structural variants, including those affecting phenotypes. While genomes assembled using high-coverage long reads can achieve high levels of contiguity and completeness, concerns about cost, errors, and low yield have limited widespread adoption of such sequencing approaches. Here we resequenced the reference strain of <em>D. melanogaster</em> (ISO1) on a single Oxford Nanopore MinION flow cell run for 24 hr. Using only reads longer than 1 kb or with at least 30x coverage, we assembled a highly contiguous <em>de novo</em> genome. The addition of inexpensive paired reads and subsequent scaffolding using an optical map technology achieved an assembly with completeness and contiguity comparable to the <em>D. melanogaster</em> reference assembly. Comparison of our assembly to the reference assembly of ISO1 uncovered a number of structural variants (SVs), including novel LTR transposable element insertions and duplications affecting genes with developmental, behavioral, and metabolic functions. Collectively, these SVs provide a snapshot of the dynamics of genome evolution. Furthermore, our assembly and comparison to the <em>D. melanogaster</em> reference genome demonstrates that high-quality <em>de novo</em> assembly of reference genomes and comprehensive variant discovery using such assemblies are now possible by a single lab for under $1,000 (USD).</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila</em></a></li><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">genome assembly</a></li><li class="kwd"><a href="/keyword/nanopore-sequencing" class="hw-term hw-article-keyword hw-article-keyword-nanopore-sequencing" rel="nofollow">nanopore sequencing</a></li></ul><p id="p-5">The characterization of comprehensive genetic variation is crucial for the discovery of mutations affecting phenotypes. In the last 10 years, the exponential decline in cost of high-throughput short-read sequencing has revolutionized our ability to assay genome-wide sequence variation. Short reads excel at identifying single nucleotide polymorphisms (SNPs) and short insertion-deletion (indel) polymorphisms in unique genomic regions. However, the majority of the sequence difference between individuals is caused by duplications, deletions, inversions, or translocation of sequences—collectively known as structural variants (SVs) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Feuk <em>et al.</em> 2006</a>). SVs are often longer than short sequencing reads (generally 50–150 bp), meaning genotyping of SVs is indirect and relies on features of alignments to a reference genome such as divergent read mappings, split reads, or elevated read coverage (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Medvedev <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alkan <em>et al.</em> 2011b</a>). However, comparisons of extremely contiguous <em>de novo</em> genome assemblies from humans (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huddleston and Eichler 2016</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chaisson <em>et al.</em> 2017</a>) and <em>Drosophila melanogaster</em> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chakraborty <em>et al.</em> 2018</a>) revealed that short reads miss 40–80% of SVs. Consequently, methods that are not susceptible to the shortcomings of short-read sequencing are essential to obtain a more complete view of genome variation (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Alkan <em>et al.</em> 2011b</a>). We propose one approach—comparison of contiguous and accurate <em>de novo</em> genome assemblies—that would overcome these limitations and drastically improve our understanding of genetic variation (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Alkan <em>et al.</em> 2011b</a>).</p><p id="p-6">Although short reads have been used extensively for <em>de novo</em> genome assembly, they fail to resolve repetitive regions in genomes, leaving errors and gaps while assembling such regions (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Paszkiewicz and Studholme 2010</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Treangen and Salzberg 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bradnam <em>et al.</em> 2013</a>). Such fragmented draft-quality assemblies are therefore poorly suited for identification of SVs (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Alkan <em>et al.</em> 2011b</a>) and lead to incomplete and/or missing gene models (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gordon <em>et al.</em> 2016</a>). Theoretical considerations of the genome assembly problem predict that, with sufficient read depth and length, genome assemblies can resolve even difficult regions (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Motahari <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lam <em>et al.</em> 2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bresler <em>et al.</em> 2013</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Shomorony <em>et al.</em> 2016</a>). Consistent with this, long reads produced by Single Molecule Real-Time sequencing from Pacific Biosciences (PacBio) and Nanopore sequencing from Oxford Nanopore Technologies (ONT) provide data capable of achieving remarkably contiguous <em>de novo</em> genome assemblies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Berlin <em>et al.</em> 2015</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Michael <em>et al.</em> 2018</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jain <em>et al.</em> 2018</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>). However, due to high error rates (∼10–15%), generation of reliable assemblies with these reads requires non-trivial coverage (generally 30x or greater) (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>). Nevertheless, until recently, long-read methods have required prohibitively expensive reagents, and technologies like PacBio also require substantial capital investment related to the housing and maintenance of equipment necessary to perform the sequencing. Combined with concerns about high error rates, widespread adoption of long-molecule sequencing for <em>de novo</em> assembly and variant detection has been tentative.</p><p id="p-7">Sequencing using ONT may produce reads that are hundreds of kilobases in length (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Jain <em>et al.</em> 2018</a>), though their application to <em>de novo</em> assembly of reference-grade multicellular eukaryotic genomes is not yet routine. To understand how effective assembly using ONT is when applied to <em>de novo</em> genome assembly of a metazoan like <em>Drosophila</em>, we measured the contiguity, completeness, and accuracy of a <em>de novo</em> assembly constructed with ONT reads. To accomplish this, we resequenced the <em>D. melanogaster</em> reference genome strain (ISO1) using the ONT MinION and compared the resulting assembly with the latest release of the <em>D. melanogaster</em> reference assembly (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hoskins <em>et al.</em> 2015</a>), which is arguably the best metazoan reference genome available. We followed an assembly merging approach (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>) to combine modest long-read coverage from a single ONT MinION flow cell (30x depth of coverage with an average read length of 7,122 bp) and Illumina short-read data. This assembly resulted in a highly contiguous and accurate genome assembly. Notably, with this approach, the majority of the euchromatin of each chromosome arm is represented by a single contiguous sequence (contig). Collectively, the assembly recovered 97.7% of Benchmarking Universal Single Copy Orthologs (BUSCOs). This is similar to the 98.3% BUSCOs recovered in the most recent release of the <em>D. melanogaster</em> genome (version 6.16). Scaffolding of the assembly with Bionano optical maps led to further improvements in contiguity. Finally, we examined the structural differences between our assembly and the published assembly of the same strain and observed several candidate SVs, the majority of which are transposable element (TE) insertions and copy-number variants (CNVs). These are mutations that must be either: 1) recent mutations that occurred in the genome strain since it was sequenced; 2) segregating in the genome strain due to incomplete isogeny; or 3) errors in one of the assemblies.</p><p id="p-8">Overall, we show that high-quality <em>de novo</em> genome assembly of <em>D. melanogaster</em> genomes is feasible using low-cost ONT technology, enabling an assembly strategy that can be applied broadly to metazoan genomes. This strategy will make high-quality reference assemblies obtainable for species lacking reference genomes. Moreover, <em>de novo</em> assemblies for population samples of metazoan species is now feasible, opening the door for studying evolutionary and functional consequences of structural genetic variation in large populations.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Stocks</h3><p id="p-9">The ISO1 <em>D. melanogaster</em> reference stock used for both Nanopore and Illumina sequencing was obtained from the BDGP in 2014 (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hoskins <em>et al.</em> 2015</a>). All flies were kept on standard cornmeal-molasses medium and were maintained at 25°.</p></div><div id="sec-3" class="subsection"><h3>DNA isolation and quantification</h3><p id="p-10">DNA for Nanopore sequencing was isolated from males and females using the Qiagen Blood &amp; Cell Culture DNA Mini Kit. Briefly, 60–80 flies were placed in two 1.5-mL Eppendorf Lo-Bind tubes and frozen in liquid nitrogen before being homogenized using a pestle in 250 µL of Buffer G2 with RNAse. 750 µL of Buffer G2 and 20 µL of 20 mg/mL proteinase K was then added to each tube and incubated at 50° for 2 hr. After 2 hr, each tube was spun at 5,000 RPM for 5 min, and the supernatant was removed and placed in a new 1.5-mL Lo-Bind tube and vortexed for 10 sec. The supernatant from both tubes was then transferred onto the column and allowed to flow through via gravity. The column was washed 3x with wash buffer and eluted twice with 1 mL of elution buffer into two 1.5-mL Lo-Bind tubes. 700 µl of isopropanol was added and mixed via inversion before being spun at 14,000 RPM for 15 min and 4°. The supernatant was removed and the pellet was washed with 70% ethanol, then spun at 14,000 RPM for 10 min at 4°. The supernatant was removed and 25 µL of ddH<sub>2</sub>O was added to each tube and allowed to sit at room temperature for 1 hr. Both tubes were then combined into one. DNA was quantified on a Nanodrop and Qubit. DNA for Illumina sequencing was isolated from males and females using the Qiagen DNeasy Blood and Tissue Kit according to the manufacturer’s instructions and quantified on a Qubit.</p></div><div id="sec-4" class="subsection"><h3>Library preparation, sequencing, and basecalling</h3><p id="p-11">For Nanopore sequencing, 1.5 µg of DNA (5.49 µL of 273 ng/uL DNA) was used to prepare a 1D sequencing library (SQK-LSK108) according to the manufacturer’s instructions, including the FFPE repair step. The 75-µl library was then immediately loaded onto a R9.5 flow cell prepared according to the manufacturer’s instructions and run for approximately 24 hr. Basecalling was completed using ONT Albacore Sequencing Pipeline Software version 2.0.2. Reads collected during the mux phase of sequencing were not included.</p><p id="p-12">For Illumina sequencing, ∼600-bp fragments were generated from 500 ng of DNA using a Covaris S220 sonicator. Fragments of 500–700 bp were selected using a Pippin and libraries were prepared using a KAPA High Throughput Library Preparation kit and Bioo Scientific NEXTflex DNA Barcodes. The library was pooled with others and run as a 150-bp paired-end run on a single flow cell of an Illumina NextSeq 500 in medium-output mode using RTA version 2.4.11. bcl2fastq2 v2.14 was then run in order to demultiplex reads and generate FASTQ files.</p></div><div id="sec-5" class="subsection"><h3>Genome assemblies</h3><p id="p-13">Canu (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>) release v1.5 was used to assemble the ONT reads. Canu was run with default parameters in grid mode (Sun Grid Engine) using ONT reads &gt;1 kb and a genome size of 130 Mb. To generate the De Bruijn graph contigs for the hybrid assembly, we used Platanus (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kajitani <em>et al.</em> 2014</a>) v1.2.4 with default settings to assemble 67x of Illumina paired-end reads obtained from the DPGP (<a href="http://www.dpgp.org/dpgp2/DPGP2.html">http://www.dpgp.org/dpgp2/DPGP2.html</a>) (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pool <em>et al.</em> 2012</a>). The hybrid assembly was generated with DBG2OLC (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ye <em>et al.</em> 2016</a>) using contigs from the Platanus assembly and the longest 30x ONT reads. DBG2OLC settings (options: k 25 AdaptiveTh 0.01 KmerCovTh 2 MinOverlap 35 RemoveChimera 1) were similar to those used for PacBio hybrid assembly of ISO1 (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>), except that the k-mer size was increased to 25 and the MinOverlap to 35 to minimize the number of misassemblies. The consensus stage of DBG2OLC was run with PBDAG-Con (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chin <em>et al.</em> 2013</a>) and BLASR (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">M. J. Chaisson and Tesler 2012</a>). Separately, minimap v0.2-r123 (using a minimizer size window of 5, FLOAT fraction of minimizers of 0, and min matching length of 100) and miniasm v0.2-r123 (using default settings) were also used to assemble only the ONT reads (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li 2016</a>).</p><p id="p-14">The Canu and DBG2OLC assemblies were merged using quickmerge (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>). First, the two assemblies were merged using the DBG2OLC assembly as the query and the Canu assembly as the reference. Thus, the first quickmerge run (options: hco 5.0 c 1.5 l 2900000 ml 20000) filled gaps in the DBG2OLC assembly using sequences from the Canu assembly, giving preference to the Canu assembly sequences at the homologous sequence junctions. The contigs that are unique to the Canu assembly were incorporated in the final assembly by a second round of quickmerge. In the second quickmerge run (options: hco 5.0 c 1.5 l 2900000 ml 20000), the merged assembly from the previous step was used as the reference assembly, and the Canu assembly was used as the query assembly (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assembly strategy used in this manuscript. A lower-contiguity assembly (Canu) is merged with a higher-contiguity assembly (DBG2OLC). The resulting assembly is again merged with the Canu assembly. The genome is then polished one or more times, here with nanopolish followed by Pilon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1370169722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Assembly strategy used in this manuscript. A lower-contiguity assembly (Canu) is merged with a higher-contiguity assembly (DBG2OLC). The resulting assembly is again merged with the Canu assembly. The genome is then polished one or more times, here with nanopolish followed by Pilon.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3143/F1.medium.gif" width="440" height="421"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3143/F1.medium.gif" width="440" height="421"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491104" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Assembly strategy used in this manuscript. A lower-contiguity assembly (Canu) is merged with a higher-contiguity assembly (DBG2OLC). The resulting assembly is again merged with the Canu assembly. The genome is then polished one or more times, here with nanopolish followed by Pilon.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Assembly polishing</h3><p id="p-16">Assembly polishing was performed two ways. First, <em>nanopolish</em> version 0.7.1 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Loman, Quick, and Simpson 2015</a>) was run using the recommended settings with reads longer than 1 kb. Prior to running nanopolish, the merged genome assembly was indexed using bwa, and ONT reads were aligned to the genome using bwa mem. The resulting bam file was then sorted, indexed, and filtered for alignments of size larger than 1 kb using samtools 1.3 (Li <em>et al.</em> 2017). Nanopolish was then run with a minimum candidate frequency of 0.1. Following nanopolish, we polished the assembly twice with Pilon (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Walker <em>et al.</em> 2014</a>) v1.16. For Pilon, we aligned 44x 150-bp and 67x 100-bp Illumina paired-end reads to the assembly using bowtie2 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Langmead and Salzberg 2012</a>) and then ran Pilon on the sorted bam files using default settings.</p></div><div id="sec-7" class="subsection"><h3>Bionano scaffolding</h3><p id="p-17">Bionano optical map data were collected following Chakraborty <em>et al.</em> (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Chakraborty <em>et al.</em> 2018</a>). ISO1 embryos less than 12 h old were collected on apple juice/agar Petri dishes, dechorionated using 50% bleach, rinsed with water, then stored at -80°. DNA was extracted from frozen embryos using the Animal Tissue DNA Isolation kit (Bionano Genomics, San Diego, CA). Bionano Irys optical data were generated and assembled with IrysSolve 2.1 at Bionano Genomics. We then merged the Bionano assembly with the final merged assembly contigs using IrysSolve, retaining Bionano assembly features when the two assemblies disagreed.</p></div><div id="sec-8" class="subsection"><h3>BUSCO analysis</h3><p id="p-18">We used BUSCO v1.22 to evaluate completeness and accuracy of all ISO1 assemblies (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Simão <em>et al.</em> 2015</a>). We used the Diptera database, which contains 2,799 highly conserved genes, to estimate assembly completeness.</p></div><div id="sec-9" class="subsection"><h3>Assembly comparison and quality metrics</h3><p id="p-19">Assemblies were compared using alignment dot plots. For dot plots, assembled genomes were aligned to the reference <em>D. melanogaster</em> genome (r6.16) using nucmer (using options: minmatch 100, mincluster 1000, diagfactor 10, banded, diagdiff 5) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kurtz <em>et al.</em> 2004</a>). The resulting delta alignment files were used to create the dot plots either with mummerplot (using options:–fat–filter–ps) or ggplot. QUAST v4.5.0 was used to compare each generated assembly to the contig reference genome assembly (<em>D. melanogaster</em> r6.16) for completeness and errors. QUAST was run in GAGE mode on contigs larger than 1 kb and with the reference assembly as fragmented, as it was originally the scaffolded assembly (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Salzberg <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gurevich <em>et al.</em> 2013</a>). GAGE was run on two types of reference assembly: the full reference assembly, and only the part of the reference assembly comprising ordered and oriented contigs on chromosome arms (<em>i.e.</em>, Muller elements) and the mitochondrial sequence. Quality score analysis was performed by aligning our Illumina short reads against each genome assembly using Bowtie. The number of variants were summed and divided by the total number of bases (following <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Berlin <em>et al.</em> 2015</a>) and also by the total number of bases aligned: P<sub>error</sub> = Variants/(Total Bases), where “Total bases” represents the total number of bases in the assembly or the total aligned bases (following <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>). These values are proxies for the probability of error and are used to calculate the QV score for each assembly according to the relation: QV=-10*log10P<sub>error</sub>.</p><p id="p-20">A modified pipeline originally published in McCoy <em>et al.</em> (2014) and modified following <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Berlin <em>et al.</em> (2015)</a> and <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a> was used (this pipeline relies on release 5.57 naming conventions) for calculating total genome reconstruction. Nucmer was used to align each assembly to the FlyBase 5.57 reference, and then separated and merged by euchromatic and heterochromatic regions for each chromosome arm using BEDtools (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Quinlan and Hall 2010</a>). Gene and TE reconstruction was completed similar to <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Berlin <em>et al.</em> (2015)</a>. For measuring the accuracy of gene models and TE reconstruction, both releases 5.57 and 6.16 of the reference were used. Fasta files containing all genes and separately, TEs for both releases were downloaded from FlyBase. In total, 17,730 gene models and 5,392 transposons from release 6.16, as well as 17,294 gene models and 5,409 transposons from release 5.57 were aligned to each genome assembly independently using nucmer. Nucmer was used to align each gene and transposon fasta file to each assembly independently. Alignments greater than 0%, 99% and equal to 100% similarity were reported.</p></div><div id="sec-10" class="subsection"><h3>Structural variant detection</h3><p id="p-21">Large (&gt;100 bp) SVs were detected by aligning the Bionano scaffolds to the FlyBase (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">dos Santos <em>et al.</em> 2015</a>) reference assembly using MUMmer v3.23 (nucmer -maxmatch) (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kurtz <em>et al.</em> 2004</a>) and then annotating the disagreements between the assemblies as indels and duplications using SVMU v0.2beta (commit 4e65e95) (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Chakraborty <em>et al.</em> 2018</a>). Insertions overlapping with Repeatmasker 4.0.7 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Smit <em>et al.</em> 2013</a>) annotated TEs were annotated as TE insertions. SVs were validated with at least two ONT reads spanning the entire genomic feature containing the SVs plus 200 bp on both sides of the SV. TEs were inferred to be segregating when corrected long reads supporting the TE insertions were contradicted by other reads showing absence of the TE. For validation with spanning long reads, we aligned Canu-corrected ONT reads to the FlyBase and Bionano assemblies using BLASR (v 1.3.1) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chaisson and Tesler 2012</a>) and the sorted alignment bam files were visually examined with IGV (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Thorvaldsdóttir <em>et al.</em> 2013</a>).</p></div><div id="sec-11" class="subsection"><h3>Mitochondrial genome identification</h3><p id="p-22">The mitochondrial genome was identified by using BLAST (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Altschul <em>et al.</em> 1990</a>) to compare our final assembly (the Bionano assembly) against the mitochondrial genome from r6.16 of the <em>D. melanogaster</em> genome. A single contig was identified (tig00000438_pilon_pilon_obj) with 99% identity to the reference mitochondrial genome. This contig contained two copies of the mitochondrial genome in tandem (assembly duplications of circular genomes are not uncommon when doing assembly with long-read data), therefore the first 16,806 and last 2,104 nucleotides were removed from the contig. We used MITOS (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bernt <em>et al.</em> 2013</a>) with default settings, metazoan reference, and invertebrate genetic code to annotate both the reference mitochondrial genome and our assembled mitochondrial genome.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-23">The Illumina and basecalled ONT data generated in this study has been uploaded to the National Center for Biotechnology Information (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>) under bioproject PRJNA433573. Genomes assembled in this study are available at <a href="https://github.com/danrdanny/Nanopore_ISO1">https://github.com/danrdanny/Nanopore_ISO1</a>. Releases 6.16 and 5.57 of the D. melanogaster genome used in this study are available on FlyBase (<a href="http://www.flybase.org">http://www.flybase.org</a>). Bioinformatic scripts used in this pipeline are available at <a href="https://github.com/esolares/DMI1">https://github.com/esolares/DMI1</a>. Original data underlying this manuscript can be accessed from the Stowers Original Data Repository at <a href="http://www.stowers.org/research/publications/libpb-1268">http://www.stowers.org/research/publications/libpb-1268</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6813398">https://doi.org/10.25387/g3.6813398</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Sequencing results</h3><p id="p-24">A 1D sequencing library was loaded onto a release 9.5 flow cell and run for approximately 24 hr (see Methods), generating a total of 663,784 reads. Basecalling was performed with Albacore 2.0.2, with 593,354 (89%) of all reads marked as “pass” (reads having a quality score ≥7) and an average fragment length of 7,122 bp (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The read N50 for those that passed filter was 11,840, with 41 reads longer than 50 kb and a maximum read length of 379,978 bp (Figure S1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491005/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491005/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Statistics of reads used for genome assembly<sup>*</sup></span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read length distribution for reads with quality scores greater than or equal to 7. Length is the sequence length after basecalling by Albacore, not the length that aligned to the genome. (A) Distribution of read lengths less than 50 kb. (B) Distribution of reads 50 kb or greater. The longest read that passed quality filtering was 380 kb." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1370169722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Read length distribution for reads with quality scores greater than or equal to 7. Length is the sequence length after basecalling by Albacore, not the length that aligned to the genome. (A) Distribution of read lengths less than 50 kb. (B) Distribution of reads 50 kb or greater. The longest read that passed quality filtering was 380 kb.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3143/F2.medium.gif" width="309" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3143/F2.medium.gif" width="309" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491022" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Read length distribution for reads with quality scores greater than or equal to 7. Length is the sequence length after basecalling by Albacore, not the length that aligned to the genome. (A) Distribution of read lengths less than 50 kb. (B) Distribution of reads 50 kb or greater. The longest read that passed quality filtering was 380 kb.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Genome assembly</h3><div id="sec-16" class="subsection"><h4>ONT-only assembly using minimap/miniasm:</h4><p id="p-27">To evaluate ONT data for <em>de novo</em> genome assembly, we performed an ONT-only assembly using minimap and miniasm (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Li 2016</a>). Together, these programs allow for rapid assembly and analysis of long, error-prone reads. We generated an assembly with a total size of 132 Mb, with 208 contigs and a contig N50 of 3.8 Mb (N50 is the length of the contig such that 50% of the genome is contained within contigs that are equal to or longer than this contig) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Evaluation of an alignment dot plot between this assembly and the <em>D. melanogaster</em> reference genome revealed high correspondence between our assembly and the reference genome (Figure S2). However, BUSCO analysis of the minimap assembly found only 0.5% of expected single-copy genes present—much lower than the BUSCO score of 98.3% obtained from the current release of the <em>D. melanogaster</em> genome (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). BUSCO analysis evaluates the presence of universal single-copy orthologs as a proxy of completeness. Such a low BUSCO score as found in the minimap assembly is unlikely to be measuring low completeness, but rather suggests a high rate of errors that disrupt the genes, making them difficult to properly assay.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491073/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491073/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genome assembly statistics<sup>*</sup></span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491145/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491145/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Busco scores demonstrating genome quality before or after polishing</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>ONT-only assemblies using Canu:</h4><p id="p-29">We also generated an ONT-only assembly with Canu using only reads longer than 1 kb. Alignment dot plots for the assembled genome and the <em>D. melanogaster</em> reference genome also revealed large colinear blocks between this assembly and the reference, indicating only one large misassembly on chromosome 2L (this misassembly was broken prior to merging and polishing) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The Canu assembly was marginally less contiguous (contig N50 = 3.0 Mb) than the minimap assembly, but resulted in a higher BUSCO score of 67.7% (Table S1). The errors in the Canu assembly and the low BUSCO score are consequences of inherently high error rate of ONT reads. However, due to the higher accuracy and completeness of the Canu assembly compared to the minimap assembly, we used the ONT-only Canu assembly for the remainder of our analysis.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dot plots showing colinearity of our assembled genomes with the current version of the D. melanogaster reference genome. Red dots represent regions where the assembly and the reference aligned in the same orientation; blue dots represent regions where the genomes are inverted with respect to one another. The vertical grid lines represent boundaries between chromosome scaffolds in the reference assembly. Horizontal grid lines represent boundaries between contig (A-C) or scaffolds (D) in the assemblies reported here. (A) Plot of the Canu-only assembly against the reference genome. (B) Plot of the hybrid DBG2OLC Nanopore and Illumina assembly against the reference. (C) Plot of merged DBG2OLC and Canu assemblies showing a more contiguous assembly than either of the component assemblies. (D) Bionano scaffolding of the merged assembly resolves additional gaps in the merged assembly." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1370169722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Dot plots showing colinearity of our assembled genomes with the current version of the &lt;em&gt;D. melanogaster&lt;/em&gt; reference genome. Red dots represent regions where the assembly and the reference aligned in the same orientation; blue dots represent regions where the genomes are inverted with respect to one another. The vertical grid lines represent boundaries between chromosome scaffolds in the reference assembly. Horizontal grid lines represent boundaries between contig (A-C) or scaffolds (D) in the assemblies reported here. (A) Plot of the Canu-only assembly against the reference genome. (B) Plot of the hybrid DBG2OLC Nanopore and Illumina assembly against the reference. (C) Plot of merged DBG2OLC and Canu assemblies showing a more contiguous assembly than either of the component assemblies. (D) Bionano scaffolding of the merged assembly resolves additional gaps in the merged assembly.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3143/F3.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3143/F3.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491148" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Dot plots showing colinearity of our assembled genomes with the current version of the <em>D. melanogaster</em> reference genome. Red dots represent regions where the assembly and the reference aligned in the same orientation; blue dots represent regions where the genomes are inverted with respect to one another. The vertical grid lines represent boundaries between chromosome scaffolds in the reference assembly. Horizontal grid lines represent boundaries between contig (A-C) or scaffolds (D) in the assemblies reported here. (A) Plot of the Canu-only assembly against the reference genome. (B) Plot of the hybrid DBG2OLC Nanopore and Illumina assembly against the reference. (C) Plot of merged DBG2OLC and Canu assemblies showing a more contiguous assembly than either of the component assemblies. (D) Bionano scaffolding of the merged assembly resolves additional gaps in the merged assembly.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Hybrid assembly using ONT and Illumina reads:</h4><p id="p-31">Modest coverage assemblies of PacBio long reads can exhibit high contiguity when a hybrid assembly method involving Illumina reads is used (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>). Therefore, we examined whether such assembly contiguity improvements also occur when ONT long reads are supplemented with Illumina paired-end reads. We performed a hybrid assembly using DBG2OLC with the longest 30x ONT reads and contigs from a De Bruijn graph assembly constructed with 67x of Illumina paired-end data. To optimize the parameters, we performed a gridsearch on four parameters (AdaptiveTh, KmerSize, KCovTh and MinOverlap), yielding 36 genome assemblies. We used a range of values recommended by the authors for low-coverage assemblies and verified our KmerSize by looking at meryl’s kmer histogram and found it coincided with a value that represented a 99% fraction of all k-mers (<a href="http://kmer.sourceforge.net/wiki/index.php/Getting_Started_with_Meryl">http://kmer.sourceforge.net/wiki/index.php/Getting_Started_with_Meryl</a>). We selected the best genome based on colinearity and largest N50. The best hybrid assembly was substantially more contiguous (contig N50 = 9.9 Mb) than the ONT-only Canu assembly (contig N50 = 3.0 Mb), had large blocks of contiguity with the reference (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), yet had lower BUSCO scores (47.7% compared to the 67.7% observed in the Canu assembly) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-19" class="subsection"><h4>Merging of Canu and DBG2OLC assemblies:</h4><p id="p-32">We have previously shown that merging assemblies constructed using only PacBio long reads with hybrid assemblies constructed with PacBio long reads and Illumina paired-end short reads results in a considerably more contiguous assembly than either of the two component assemblies alone (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>). To examine the effect of assembly merging on assemblies created with ONT long reads, we merged the ONT-only Canu assembly with the DBG2OLC assembly with two rounds of quickmerge (see Methods). The merged assembly (contig N50 = 18.6 Mb) was more contiguous than both the Canu and the DBG2OLC assemblies alone (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), exhibiting large-scale colinearity with the reference genome as seen in the dot plot (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). As expected, the BUSCO score of the merged assembly (58.2%) fell between the two component assemblies (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p></div></div><div id="sec-20" class="subsection"><h3>Assembly quality</h3><div id="sec-21" class="subsection"><h4>Polishing:</h4><p id="p-33">A high number of SNP and indel polymorphisms in all of our assemblies is consistent with other <em>de novo</em> assemblies created with noisy long reads with high error rates (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Loman <em>et al.</em> 2015</a>; <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>). Such errors typically lead to error-ridden genic sequence that can give the appearance that many important genes are missing (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>; Table S1). Many of these errors can be fixed via assembly polishing, and a number of assembly polishing tools exist. We chose to employ two: nanopolish, which performs consensus-based error correction using ONT reads (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Loman <em>et al.</em> 2015</a>), and Pilon, which performs error correction using Illumina reads (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Walker <em>et al.</em> 2014</a>). This approach of applying long-read consensus correction followed by short-read polishing has resulted in BUSCO scores comparable to that of the FlyBase reference genome (<em>e.g.</em>, <a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a> and <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Chakraborty <em>et al.</em> 2018</a>).</p><p id="p-34">To evaluate this approach for ONT data, we evaluated three different polishing approaches: one using nanopolish alone, one using only two rounds of Pilon, and one using one round of nanopolish followed by two rounds of Pilon (see Methods). Applying nanopolish alone recovered only 79%, 79.2%, and 78.5% complete BUSCOs for the hybrid, ONT-only, and the merged assemblies, respectively (Table S2), suggesting that polishing with ONT reads alone only partially improved assembly quality. On the other hand, polishing all three assemblies twice with Pilon alone fixed a large number of errors as evidenced by improved BUSCO scores of the resulting assemblies (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Simão <em>et al.</em> 2015</a>) (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). The merged assembly was polished once with nanopolish, then twice with Pilon, resulting in nearly all complete BUSCOs (97.9%) being present in our polished assembly, comparable to that of the reference assembly (98.3%) (Table S2). Variation in BUSCO scores generally tracked the method of polishing more than the type of assembly. However, the hybrid assembly did recover a slightly different subset of BUSCOs than the ONT-only assembly, leading to a slightly higher number of BUSCOs being recovered in the final merged assembly.</p></div><div id="sec-22" class="subsection"><h4>QUAST/GAGE metrics of quality:</h4><p id="p-35">The QUAST output comparing the Canu, DBG2OLC, merged, and Bionano genome assemblies against the <em>D. melanogaster</em> reference shows that the quickmerge assembly resulted in intermediate error rates and discordance as compared to the component assemblies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). All four assemblies exhibited approximately the same number of SNP and indel errors less than 5 bp, whereas the DBG2OLC assembly resulted in approximately 20% more indels greater than 5 bp in size than the Canu assembly (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4H-I</a>, Table S3, Table S4). Among ONT assemblies, the Canu assembly exhibited superior accuracy and fewer misassemblies, although our quickmerge assembly was nearly as good. When mismatches are measured on a phred scale, the ONT assemblies range in quality from 32 to 33, which is lower than the score of 37 from a PacBio assembly with threefold more data (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>) (Table S3, Table S4). Canu also exhibited the lowest raw contiguity for an ONT assembly as measured by N50/NG50 and L50/LG50. The N50 for our merged assembly (19 Mb) exceeded that of Canu (3 Mb) and was nearly double that of DBG2OLC (10 Mb), while maintaining large-scale colinearity with the reference (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). However, Canu outperforms the merging approach when considering the contiguity of error-free alignment blocks as demonstrated by the NA50 and NGA50 for Canu, which are greater than that of quickmerge (Table S3). We also observe that half of the assemblies accrue in fewer error-free alignments for Canu than for quickmerge, resulting in slightly lower values in LA50 and LGA50 for Canu than quickmerge (Table S3). As a consequence, when evaluating which approach to employ, we advise users to carefully weigh the tradeoffs between large-scale contiguity and local misassembly and sequence errors and to tailor their decision to the biological questions being addressed. For example, in creating a reference for QTL mapping, there might be a strong preference for high long-range contiguity even at the cost of local misassemblies. On the other hand, when avoiding misassemblies is paramount (as might be the case when characterizing the structure of individual loci), one could make the argument in favor of less long-range contiguity in exchange for fewer misassemblies.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QUAST was used to compare each assembly to the D. melanogaster reference genome with selected statistics presented here. (A) Greater than 90% of bases in the reference genome were aligned to each of our four assemblies. (B) The contiguity of assembly blocks aligned to the reference. (C) Total unaligned length includes contigs that did not align to the reference as well as unaligned sequence of partially aligned contigs. (D) The number of contigs that contain misassemblies in which flanking sequences are 1 kb apart, flanking sequences overlap by 1 kb or more, or flanking sequences align to different reference scaffolds. (E) Total count of misassemblies as described in (D). (F) Local misassemblies include those positions in which a gap or overlap between flanking sequence is less than 1 kb [(D) and (E) show those greater than 1 kb] and larger than the maximum indel length of 85 bp on the same reference genome scaffold. (G) Misassemblies can be subdivided into relocations (a single assembled contig aligns to the same reference scaffold but in pieces at least 1 kb apart), inversions (at least one part of a single assembled contig aligns to the reference in an inverted orientation), or translocations (at least one part of a single assembled contig aligns to two different reference scaffolds). Not all misassemblies are captured in these three categories. (H) Total SNPs per assembly are shown and were not significantly different among assemblies. (I) Indels per 100 kb can be divided into small indels (those 85 bp are considered misassemblies and are shown in panels D, E, or F." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1370169722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QUAST was used to compare each assembly to the &lt;em&gt;D. melanogaster&lt;/em&gt; reference genome with selected statistics presented here. (A) Greater than 90% of bases in the reference genome were aligned to each of our four assemblies. (B) The contiguity of assembly blocks aligned to the reference. (C) Total unaligned length includes contigs that did not align to the reference as well as unaligned sequence of partially aligned contigs. (D) The number of contigs that contain misassemblies in which flanking sequences are 1 kb apart, flanking sequences overlap by 1 kb or more, or flanking sequences align to different reference scaffolds. (E) Total count of misassemblies as described in (D). (F) Local misassemblies include those positions in which a gap or overlap between flanking sequence is less than 1 kb [(D) and (E) show those greater than 1 kb] and larger than the maximum indel length of 85 bp on the same reference genome scaffold. (G) Misassemblies can be subdivided into relocations (a single assembled contig aligns to the same reference scaffold but in pieces at least 1 kb apart), inversions (at least one part of a single assembled contig aligns to the reference in an inverted orientation), or translocations (at least one part of a single assembled contig aligns to two different reference scaffolds). Not all misassemblies are captured in these three categories. (H) Total SNPs per assembly are shown and were not significantly different among assemblies. (I) Indels per 100 kb can be divided into small indels (those &lt;5 bp) and large indels (&#x2265; 5 bp). Indels &gt;85 bp are considered misassemblies and are shown in panels D, E, or F.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3143/F4.medium.gif" width="248" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3143/F4.medium.gif" width="248" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491108" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">QUAST was used to compare each assembly to the <em>D. melanogaster</em> reference genome with selected statistics presented here. (A) Greater than 90% of bases in the reference genome were aligned to each of our four assemblies. (B) The contiguity of assembly blocks aligned to the reference. (C) Total unaligned length includes contigs that did not align to the reference as well as unaligned sequence of partially aligned contigs. (D) The number of contigs that contain misassemblies in which flanking sequences are 1 kb apart, flanking sequences overlap by 1 kb or more, or flanking sequences align to different reference scaffolds. (E) Total count of misassemblies as described in (D). (F) Local misassemblies include those positions in which a gap or overlap between flanking sequence is less than 1 kb [(D) and (E) show those greater than 1 kb] and larger than the maximum indel length of 85 bp on the same reference genome scaffold. (G) Misassemblies can be subdivided into relocations (a single assembled contig aligns to the same reference scaffold but in pieces at least 1 kb apart), inversions (at least one part of a single assembled contig aligns to the reference in an inverted orientation), or translocations (at least one part of a single assembled contig aligns to two different reference scaffolds). Not all misassemblies are captured in these three categories. (H) Total SNPs per assembly are shown and were not significantly different among assemblies. (I) Indels per 100 kb can be divided into small indels (those &lt;5 bp) and large indels (≥ 5 bp). Indels &gt;85 bp are considered misassemblies and are shown in panels D, E, or F.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Finally, the low-coverage ONT dataset presented here performed surprisingly well compared to a PacBio dataset three times larger (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>) in terms of contiguity (N50, NA50, L50, LA50), completeness (genome fraction), and accuracy (identity, SNPs, InDels, translocations, etc.) (Table S3). In all measures, the merged assembly performed almost as well as the much higher coverage PacBio assembly from <a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a>. Importantly, however, the GAGE corrected N50 of <a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a> was substantially larger than the ONT assemblies, likely due to its superior coverage (Table S3, Table S4).</p></div><div id="sec-23" class="subsection"><h4>Completeness: reconstructing genes, TEs, and chromosome arms:</h4><p id="p-38">We next evaluated the completeness of the assemblies by assessing the reconstruction of elements in the genome, including genes, TEs, and chromosome arms. For all following metrics, Canu and the final Quickmerge assembly performed similar to one another, with DBG2OLC performing somewhat worse. As a result, remaining comparisons will be made between the final Quickmerge ONT assembly and the PacBio assembly produced by <a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>, which we use as a standard of comparison.</p><p id="p-39">Our final assembly contained 96.37% of release 6.16 genes greater than 99% complete as compared to 98.4% for the <a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a> assembly (Table S5). However, our final assembly was able to reconstruct only 72.71% of gene models compared to 88.14% for the <a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a> assembly when only 100% complete gene models are considered (Table S5). Transposon reconstruction followed the same pattern with a 6.66% and a 15.51% difference in counts between the two assemblies for those with greater than 99% identity and complete reconstruction, respectively, with both favoring the higher-coverage PacBio assembly.</p><p id="p-40">Muller element (chromosome arm) reconstruction was more similar between the ONT and PacBio assemblies for the euchromatic regions, with an average difference of 0.47% and a maximum difference of 0.94% in the X chromosome (Table S5). The largest difference was in the number of alignments. Bionano and quickmerge assemblies covered the Muller elements in fewer segments than the <a id="xref-ref-30-13" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a> assembly, though they exhibited less total chromosome coverage. The differences were more apparent in the heterochromatic scaffolds, as our final assembly contained less coverage than the <a id="xref-ref-30-14" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a> assembly, with an average deficit of 4.44% coverage and a maximum deficit of 7.89% across all heterochromatic scaffolds linked to Muller elements (Table S6). The difference in the Y chromosome was the largest, as expected, for our relatively low coverage of mixed sex flies compared to the high coverage of all males for <a id="xref-ref-30-15" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a>.</p></div><div id="sec-24" class="subsection"><h4>Base quality:</h4><p id="p-41">We aligned Illumina short reads to measure the sequencing error rate following <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Berlin <em>et al.</em> (2015)</a>. The proportion of reads successfully mapped to assemblies varied from 93.5 to 94.6%. The aligned reads resulted in QV scores ranging from 40.2 to 41.1. A slightly different approach used by <a id="xref-ref-30-16" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> (2017)</a> yielded very similar QV scores in the range of 40.1–40.2 (Table S7), suggesting that our assembly was of relatively high quality in areas where reads align well.</p></div></div><div id="sec-25" class="subsection"><h3>Bionano scaffolding</h3><p id="p-42">Bionano fragments, which are substantially longer than ONT reads, can be used for scaffolding contigs across repetitive regions. We generated 81,046 raw Bionano fragments (20.5 Gb) with an average fragment length of 253 kb. To use these fragments for scaffolding, we first created a Bionano assembly (509 maps, haploid assembly size = 145 Mb) using 78,397 noise-rescaled fragments (19.9 Gb, mean fragment length 253 kb). At 145 Mb, the Bionano assembly is comparable to the reference assembly size (144 Mb). The Bionano maps were used to scaffold the merged assembly. The resulting scaffolded assembly was more contiguous (N50 = 21.3 Mb) than the unscaffolded contigs without substantially changing the number of SNP and indel errors (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>; Tables S1, S3-S4). Bionano scaffolding of our assembled genome did not change BUSCO scores (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>).</p></div><div id="sec-26" class="subsection"><h3>Structural variants</h3><p id="p-43">One advantage of high quality <em>de novo</em> assemblies is that they permit comprehensive detection of large (&gt;100 bp) SVs. Highly contiguous assemblies, such as the one generated here, allow comparisons between two or more assemblies, revealing novel SVs and facilitating the study of their functional and evolutionary significance. Although we sequenced the reference genome stock, structural differences between our stock and the published assembly are expected due to error, but also to new mutations—especially for TEs, which are the most dynamic structural components of the genome. Indeed, such mutations have been observed in substrains of ISO1 before (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zakharenko <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Moschetti <em>et al.</em> 2010</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rahman <em>et al.</em> 2015</a>). Similarly, our assembly revealed the presence of 34 new TE insertions and 12 TE losses compared to the reference genome assembly. Among the 34 TE insertions, 50% (17/34) are LTR TEs comprising chiefly of <em>copia</em> (5/17) and <em>roo</em> elements (6/17). Interestingly, 29% (10/34) of the TE insertions are defective <em>hobo</em> elements that lack an average of 1.7 kb of sequence (base pairs 905–2510 of full-length <em>hobo</em> are absent) from the middle segment of the element encoding the transposase. However, alignment of long reads to the assembly regions harboring the TE insertions revealed that 6/34 insertions are not fixed, but are segregating in the strain (Table S2). The high insertion rate of the LTR and <em>hobo</em> elements in this single strain mirrors the recent spread of LTR and <em>hobo</em> elements in <em>D. melanogaster</em> populations (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pascual and Periquet 1991</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Periquet <em>et al.</em> 1994</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bowen and McDonald 2001</a>). As expected, the majority (27/34) of the new TE insertions are located within introns, because insertions within exons generally result in gene disruption. Nonetheless, we found five genes (<em><a href="http://flybase.org/reports/FBgn0012037.html">Ance</a></em>, <em><a href="http://flybase.org/reports/FBgn0000273.html">Pka-C1</a></em>, <em><a href="http://flybase.org/reports/FBgn0051826.html">CG31826</a></em>, <em><a href="http://flybase.org/reports/FBgn0263400.html">CG43446</a></em>, and <em><a href="http://flybase.org/reports/FBgn0044047.html">Ilp6</a></em>) in which new TEs have inserted within exons (Table S2). We also found 12 TEs present in the reference assembly that are missing from our final scaffolded assembly, among which six are LTR TEs (five <em>297</em> elements and one <em>roo</em> element). The high rate of insertion and loss of LTR elements underscores their dynamic evolutionary history in the <em>D. melanogaster</em> genome. Because TEs can be locally unique, the presence or absence of such events does not pose a fundamental limitation to assembly when reads are long enough to span the TEs. Consequently, we predict most of these events to be new mutations rather than errors in assembly.</p><p id="p-44">Additionally, we identified several duplications present in our assembly. For example, a tandem duplication of a ∼9-kb segment has created partial copies of the genes <em>infertile crescent</em> (<em><a href="http://flybase.org/reports/FBgn0001941.html">ifc</a></em>) and <em>little imaginal discs</em> (<em><a href="http://flybase.org/reports/FBgn0031759.html">lid</a></em>). Another ∼2-kb tandem duplication has created partial copies of the genes <em><a href="http://flybase.org/reports/FBgn0032800.html">CG10137</a></em> and <em><a href="http://flybase.org/reports/FBgn0053116.html">CG33116</a></em> (Table S8). Apart from CNVs affecting single copy sequences, our assembly also uncovered copy number increases in tandem arrays with potential functional consequences. For example, we observed copy number increase in a tandem array of a 207-bp segment within the third exon of the chitin-binding protein gene <em><a href="http://flybase.org/reports/FBgn0040950.html">Muc26B</a></em>. While CNVs such as this have been challenging to identify and validate in the past, at least two Nanopore reads spanning this entire tandem array support the presence of a tandem duplication at this position (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Unlike TEs, classifying such tandem events as errors stemming from shorter Sanger reads or an actual mutation is difficult without the access to the original material from which the Sanger data were derived.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Copy number increase in a 207-bp tandem array located inside the third exon of Muc26B. (A) Three tracks showing Bionano assembly (top) with Nanopore long reads (blue) and reference (Flybase) assembly (red) aligned to it. The alignment gap in the reference assembly is due to the extra sequence copies in the Bionano assembly. (B) Alignment dot plot between the reference sequence possessing the tandem array to itself. (C) Alignment dot plot between the genomic region possessing the tandem array in the Bionano assembly to itself. As evidenced by the dot plot, the Bionano assembly has more repeats in this region than the reference assembly in panel B. (D) Alignment dot plot between the reference genomic region (x axis) shown in (B) and the corresponding Bionano genomic region (y axis) shown in (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1370169722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Copy number increase in a 207-bp tandem array located inside the third exon of &lt;em&gt;Muc26B&lt;/em&gt;. (A) Three tracks showing Bionano assembly (top) with Nanopore long reads (blue) and reference (Flybase) assembly (red) aligned to it. The alignment gap in the reference assembly is due to the extra sequence copies in the Bionano assembly. (B) Alignment dot plot between the reference sequence possessing the tandem array to itself. (C) Alignment dot plot between the genomic region possessing the tandem array in the Bionano assembly to itself. As evidenced by the dot plot, the Bionano assembly has more repeats in this region than the reference assembly in panel B. (D) Alignment dot plot between the reference genomic region (&lt;em&gt;x&lt;/em&gt; axis) shown in (B) and the corresponding Bionano genomic region (&lt;em&gt;y&lt;/em&gt; axis) shown in (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3143/F5.medium.gif" width="440" height="196"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/10/3143/F5.medium.gif" width="440" height="196"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491068" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">Copy number increase in a 207-bp tandem array located inside the third exon of <em>Muc26B</em>. (A) Three tracks showing Bionano assembly (top) with Nanopore long reads (blue) and reference (Flybase) assembly (red) aligned to it. The alignment gap in the reference assembly is due to the extra sequence copies in the Bionano assembly. (B) Alignment dot plot between the reference sequence possessing the tandem array to itself. (C) Alignment dot plot between the genomic region possessing the tandem array in the Bionano assembly to itself. As evidenced by the dot plot, the Bionano assembly has more repeats in this region than the reference assembly in panel B. (D) Alignment dot plot between the reference genomic region (<em>x</em> axis) shown in (B) and the corresponding Bionano genomic region (<em>y</em> axis) shown in (C).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-27" class="subsection"><h3>Mitochondrial genome identification and identity</h3><p id="p-46">After assembly, merging, polishing, and scaffolding, we used BLAST (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Altschul <em>et al.</em> 1990</a>) to identify the mitochondrial genome. Using the published <em>D. melanogaster</em> mitochondrial genome as the subject, we identified one 38,261-bp contig with nearly 100% identity to the reference mitochondrial genome. The first 16,806 and last 2,100 nucleotides of this contig were 99.6% identical to the reference mitochondrial genome, while the middle 19,228 nucleotides were 98% identical to the reference mitochondrial genome. This suggests that our assembled mitochondrial genome had been duplicated during assembly, which commonly occurs when assembling circular genomes using long sequencing reads. For the tandem assembled genomes, nearly all of the SNP and indel polymorphisms occurred in the last 4,000 nucleotides of the reference genome.</p><p id="p-47">To determine if all genes and features that are present in the reference mitochondrial genome were present in our assembled genome, we annotated both genomes using MITOS (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bernt <em>et al.</em> 2013</a>). Both assemblies were annotated nearly identically, with MITOS reporting that both assemblies were missing the origin of replication for the L region (OL) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The Nanopore assembly also resulted in two split genes not observed in the reference genome assembly: <em>nad4</em> and <em>nad6</em> were each annotated as two continuous genes rather than one single gene.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mitochondrial genome annotations generated by MITOS. (A) Annotation of the reference mitochondrial genome. (B) Annotation of the mitochondrial genome assembled in this project is identical to the reference except that nad4 and nad6 in the reference assembly were both annotated as two genes&#x2014;nad4 as nad4-a and nad4-b, and nad6 as nad6-a and nad6-b." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1370169722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mitochondrial genome annotations generated by MITOS. (A) Annotation of the reference mitochondrial genome. (B) Annotation of the mitochondrial genome assembled in this project is identical to the reference except that &lt;em&gt;nad4&lt;/em&gt; and &lt;em&gt;nad6&lt;/em&gt; in the reference assembly were both annotated as two genes&#x2014;&lt;em&gt;nad4&lt;/em&gt; as &lt;em&gt;nad4-a&lt;/em&gt; and &lt;em&gt;nad4-b&lt;/em&gt;, and &lt;em&gt;nad6&lt;/em&gt; as &lt;em&gt;nad6-a&lt;/em&gt; and &lt;em&gt;nad6-b&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3143/F6.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/10/3143/F6.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3143/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491115" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">Mitochondrial genome annotations generated by MITOS. (A) Annotation of the reference mitochondrial genome. (B) Annotation of the mitochondrial genome assembled in this project is identical to the reference except that <em>nad4</em> and <em>nad6</em> in the reference assembly were both annotated as two genes—<em>nad4</em> as <em>nad4-a</em> and <em>nad4-b</em>, and <em>nad6</em> as <em>nad6-a</em> and <em>nad6-b</em>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-28"><h2 class="">Discussion</h2><p id="p-49"><em>Drosophila melanogaster</em> was the first genome assembled using a whole-genome shotgun (WGS) strategy (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Myers <em>et al.</em> 2000</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2000</a>). This successful proof-of-principle led to the prevalence of the WGS sequencing approach as a tool in virtually all subsequent metazoan genome assembly projects (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lander <em>et al.</em> 2001</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mouse Genome Sequencing Consortium <em>et al.</em> 2002</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goff <em>et al.</em> 2002</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yu <em>et al.</em> 2002</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Aparicio <em>et al.</em> 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gibbs <em>et al.</em> 2004</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">International Chicken Genome Sequencing Consortium 2004</a>). While improvements in sequencing technology have led to a precipitous drop in the cost of sequencing (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Anonymous 2018</a>), stagnation and even regression of read lengths resulted in highly fragmented and incomplete assemblies (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alkan <em>et al.</em> 2011a</a>). Furthermore, complementary approaches (like hierarchical shotgun sequencing and other clone-based approaches) were required to obtain nearly complete and highly contiguous reference genomes, which adds complexity, cost, and time to assembly projects.</p><p id="p-50">Short reads provided by next generation sequencing technologies present limitations to what a pure WGS assembly approach can accomplish (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Alkan <em>et al.</em> 2011a</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Narzisi and Schatz 2015</a>). The advent and development of long-read sequencing technologies has led to dramatic increases in read length, permitting assemblies that span previously recalcitrant repetitive regions. Early implementations of these technologies produced reads longer than previous short-read technologies yet still shorter than relatively common repeats, while the cost and error rate remained high compared to the short-read approaches. However, continued improvements in read length overcame many of these difficulties, permitting nearly complete, highly contiguous metazoan genome assemblies with only a WGS strategy (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Kim <em>et al.</em> 2014</a>; <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Berlin <em>et al.</em> 2015</a>; <a id="xref-ref-30-17" class="xref-bibr" href="#ref-30">Koren <em>et al.</em> 2017</a>). Such approaches led to assemblies comparable in completeness and contiguity to release 6 of the <em>D. melanogaster</em> genome assembly (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hoskins <em>et al.</em> 2015</a>) for approximately $10,000 USD (<a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">Chakraborty <em>et al.</em> 2016</a>, <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">2018</a>). However, even with the rapid development of these approaches, substantial capital investment in the form of expensive instrumentation and dedicated genome facility staff was required.</p><p id="p-51">Here, we report an independent resequencing and assembly of the <em>D. melanogaster</em> reference strain, ISO1, for less than $1,000 USD in sequencing costs and without the need for extensive capital or personnel investment. The resulting assembly before scaffolding is comparable to release 6 of FlyBase in terms of contiguity and completeness (18.9 Mbp contig N50 and 97.1% complete single copy BUSCOs). We achieved this using 4.2 Gbp of sequence from a single Oxford Nanopore flow cell in conjunction with Illumina short-read data. Such reduction in complexity and cost permits a small team of scientists to shepherd a sequencing project from sample to near-reference-quality assembly in a relatively short amount of time. The addition of optical mapping data permitted ordering and orientation of the contigs, yielding an assembly nearly as contiguous as the published reference (scaffold N50 of 21.3 Mbp).</p><p id="p-52">Comparing this assembly to the FlyBase reference genome shows that it is both accurate (21 mismatches/100 kbp, 36 indels/100 kbp) and colinear (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, Table S3). Most of the small-scale differences are expected to be errors introduced by the noisy Oxford Nanopore reads that escaped correction via polishing. It is possible that some of these errors are SVs, which are expected to accumulate because the ISO1 stock has been maintained in the laboratory for approximately 350 generations (assuming 20 gen/year) since initial sequencing in 2000. This allows for the accumulation of new mutations by genetic drift, including ones reducing fitness (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Assaf <em>et al.</em> 2017</a>). Due to the high contiguity in the euchromatic region, our assembly facilitates detection of such euchromatic SVs. Several apparent “assembly errors” in our Bionano assembly are due to TE indels that are supported by spanning long reads. We found 28 homozygous euchromatic TE insertions, which are predominantly LTR and defective <em>hobo</em> elements, suggesting a high rate of euchromatic TE insertions (∼0.08 insertion/gen). That we observed a predominance of LTR and <em>hobo</em> elements among the new TE insertions mirrors their recent spread in <em>D. melanogaster</em> populations (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Pascual and Periquet 1991</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Periquet <em>et al.</em> 1994</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Bowen and McDonald 2001</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">De Freitas Ortiz and Silva Loreto 2008</a>). The abundance of defective <em>hobo</em> elements among the new insertions is particularly interesting given that these <em>hobo</em> elements lack the transposase enzyme necessary for mobilization.</p><p id="p-53">Although most novel TE insertions were found in introns, five were found within exons: the 5′ UTR of <em><a href="http://flybase.org/reports/FBgn0012037.html">Ance</a></em>, <em><a href="http://flybase.org/reports/FBgn0051826.html">CG31826</a></em>, and <em><a href="http://flybase.org/reports/FBgn0044047.html">Ilp6</a></em>; the 3′ UTR of <em><a href="http://flybase.org/reports/FBgn0000273.html">Pka-C1</a></em>; and the coding region of <em><a href="http://flybase.org/reports/FBgn0263400.html">CG43446</a></em>. Similarly, TE loss primarily involved LTR TEs, including loss of TEs from the 5′ UTR of the genes <em><a href="http://flybase.org/reports/FBgn0085450.html">Snoo</a></em> and <em><a href="http://flybase.org/reports/FBgn0033196.html">CG1358</a></em>. We also observed copy number increases both in unique sequences as well as in tandem arrays (Table S8), with one duplication creating a new copy of the entire coding sequence of the gene <em><a href="http://flybase.org/reports/FBgn0031759.html">lid</a></em>. Collectively, our assembly provides a snapshot of ongoing genome structure evolution in a metazoan genome, which is often assumed to be approximately invariant for experimental genetics.</p><p id="p-54">A crucial feature of this work is that it is performed in a strain used to generate one of the highest quality reference genomes available, ensuring that our inferences can be judged against a high-quality standard. This approach allowed us to demonstrate that assembly with modest amounts of long-molecule data paired with inexpensive short-read data can yield highly accurate and contiguous reference genomes with minimal expenditure of resources. This demonstration opens myriad opportunities for high-quality genomics in systems with limited resources for genome projects. Moreover, we can now conceive of studying entire populations with high-quality assemblies capable of resolving repetitive structural variants, something previously unattainable with short-read sequencing alone.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We would like to thank Christina Lin for assistance with the Bionano scaffolding, Melanie Oakes for help generating the Bionano data, and Angela Miller for assistance with editing and figure preparation. We thank Casey Bergman for providing examples of novel TEs in substrains of ISO1. DEM is part of the Oxford Nanopore Educational Initiative and has received free flow cells as part of that program. This material is based upon work supported by the National Science Foundation Bridge to the Doctorate Fellowship under Grant No. 1500284 and the National Science Foundation Graduate Research Fellowship under Grant No. DGE-1321846. MC is supported by US National Institutes of Health (NIH) grant R01 RR024862-01A6. JJE is supported by NIH grant R01GM123303-1 and University of California, Irvine setup funds. This work was made possible, in part, through access to the Genomics High-Throughput Facility Shared Resource of the Cancer Center Support Grant CA-62203 at the University of California, Irvine, and NIH shared-instrumentation grants 1S10RR025496-01, 1S10OD010794-01, and 1S10OD021718-01. RSH is an American Cancer Society Research Professor.</p><p id="p-56"><em>Note added in proof:</em> See Miller <em>et al</em>. 2018 (pp. <a href="https://doi.org/10.1534/g3.118.200160">3131–3141</a>) in this issue, for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-57">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6813398">https://doi.org/10.25387/g3.6813398</a>.</p></li><li class="fn" id="fn-6"><p id="p-58"><em>Communicating editor: S. Celniker</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-16"><span class="received-label">Received </span>February 16, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-03"><span class="accepted-label">Accepted </span>July 3, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Solares <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.1" data-doi="10.1126/science.287.5461.2185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gocayne</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The genome sequence of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2185</span>–<span class="cit-lpage">2195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.287.5461.2185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAdams%26rft.auinit1%253DM.%2BD.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2185%26rft.epage%253D2195%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2185%26rft_id%253Dinfo%253Apmid%252F10731132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.2" data-doi="10.1038/nmeth.1527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sajjadian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Limitations of next-generation genome sequence assembly.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1527</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DAlkan%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D61%26rft.epage%253D65%26rft.atitle%253DLimitations%2Bof%2Bnext-generation%2Bgenome%2Bsequence%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1527%26rft_id%253Dinfo%253Apmid%252F21102452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102452&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285712000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.3" data-doi="10.1038/nrg2958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alkan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coe</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome structural variation discovery and genotyping.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2958</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAlkan%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D363%26rft.epage%253D376%26rft.atitle%253DGenome%2Bstructural%2Bvariation%2Bdiscovery%2Band%2Bgenotyping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2958%26rft_id%253Dinfo%253Apmid%252F21358748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21358748&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289637600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.4" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3143.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anonymous</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-source">DNA Sequencing Costs: Data. National Human Genome Research Institute</span>, <span class="cit-publ-name">NHGRI</span>. <a href="https://www.genome.gov/27565109/the-cost-of-sequencing-a-human-genome//the-cost-of-sequencing-a-human-genome">https://www.genome.gov/27565109/the-cost-of-sequencing-a-human-genome//the-cost-of-sequencing-a-human-genome</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.6" data-doi="10.1126/science.1072104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stupka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Putnam</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Whole-genome shotgun assembly and analysis of the genome of Fugu rubripes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1072104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1072104%26rft_id%253Dinfo%253Apmid%252F12142439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTcvNTU4NS8xMzAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.7" data-doi="10.1101/gr.219956.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Assaf</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tilk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Deep sequencing of natural and experimental populations of Drosophila melanogaster reveals biases in the spectrum of new mutations.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1988</span>–<span class="cit-lpage">2000</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.219956.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.219956.116%26rft_id%253Dinfo%253Apmid%252F29079675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMi8xOTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.8" data-doi="10.1038/nbt.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span> <span class="cit-comment">(corrigendum: Nat. Biotechnol. 33: 1109)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3238%26rft_id%253Dinfo%253Apmid%252F26006009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26006009&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.9" data-doi="10.1016/j.ympev.2012.08.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donath</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jühling</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Externbrink</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Florentz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MITOS: improved de novo metazoan mitochondrial genome annotation.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">319</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2012.08.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D69%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.08.023%26rft_id%253Dinfo%253Apmid%252F22982435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.08.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22982435&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.10" data-doi="10.1101/gr.164201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Drosophila euchromatic LTR retrotransposons are much younger than the host species in which they reside.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1527</span>–<span class="cit-lpage">1540</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.164201</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.164201%26rft_id%253Dinfo%253Apmid%252F11544196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjExLzkvMTUyNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.11" data-doi="10.1186/2047-217X-2-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fass</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranay</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/2047-217X-2-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D2%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-2-10%26rft_id%253Dinfo%253Apmid%252F23870653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-2-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23870653&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.12" data-doi="10.1186/1471-2105-14-S5-S18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bresler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bresler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tse</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Optimal assembly for high throughput shotgun sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">S18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-14-S5-S18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253DS18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-S5-S18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-S5-S18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.13" data-doi="10.1186/1471-2105-13-238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DChaisson%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D238%26rft.epage%253D238%26rft.atitle%253DMapping%2Bsingle%2Bmolecule%2Bsequencing%2Breads%2Busing%2BBasic%2BLocal%2BAlignment%2Bwith%2BSuccessive%2BRefinement%2B%2528BLASR%2529%253A%2BTheory%2Band%2BApplication.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-238%26rft_id%253Dinfo%253Apmid%252F22988817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22988817&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.14" data-doi="10.1101/193144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M. J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malhotra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porubsky</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multi-platform discovery of haplotype-resolved structural variation in human genomes.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/193144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F193144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIxOTMxNDR2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.15" data-doi="10.1093/nar/gkw654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldwin-Brown</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emerson</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Contiguous and accurate de novo assembly of metazoan genomes with modest long read coverage.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">e147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw654</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw654%26rft_id%253Dinfo%253Apmid%252F27458204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27458204&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.16" data-doi="10.1038/s41588-017-0010-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">VanKuren</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalsow</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Hidden genetic variation shapes the structure of functional elements in Drosophila.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-017-0010-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-017-0010-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-017-0010-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.17" data-doi="10.1038/nmeth.2474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klammer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2474</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2474%26rft_id%253Dinfo%253Apmid%252F23644548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644548&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319668700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.18" data-doi="10.1017/S0016672308009312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Freitas Ortiz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva Loreto</span>  <span class="cit-name-given-names">E. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The hobo-related elements in the melanogaster species group.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672308009312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDe%2BFreitas%2BOrtiz%26rft.auinit1%253DM.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D243%26rft.epage%253D252%26rft.atitle%253DThe%2Bhobo-related%2Belements%2Bin%2Bthe%2Bmelanogaster%2Bspecies%2Bgroup.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009312%26rft_id%253Dinfo%253Apmid%252F18593511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18593511&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.19" data-doi="10.1038/nrg1767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structural variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">97</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1767</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFeuk%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D85%26rft.epage%253D97%26rft.atitle%253DStructural%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1767%26rft_id%253Dinfo%253Apmid%252F16418744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418744&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.20" data-doi="10.1038/nature02426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muzny</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sodergren</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome sequence of the Brown Norway rat yields insights into mammalian evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGibbs%26rft.auinit1%253DR.%2BA.%26rft.volume%253D428%26rft.issue%253D6982%26rft.spage%253D493%26rft.epage%253D521%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2BBrown%2BNorway%2Brat%2Byields%2Binsights%2Binto%2Bmammalian%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02426%26rft_id%253Dinfo%253Apmid%252F15057822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15057822&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220540100032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.21" data-doi="10.1126/science.1068275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">T.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presting</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A draft sequence of the rice genome (Oryza sativa L. ssp. japonica).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1068275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoff%26rft.auinit1%253DS.%2BA.%26rft.volume%253D296%26rft.issue%253D5565%26rft.spage%253D92%26rft.epage%253D100%26rft.atitle%253DA%2BDraft%2BSequence%2Bof%2Bthe%2BRice%2BGenome%2B%2528Oryza%2Bsativa%2BL.%2Bssp.%2Bjaponica%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1068275%26rft_id%253Dinfo%253Apmid%252F11935018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyOTYvNTU2NS85MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.22" data-doi="10.1126/science.aae0344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huddleston</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M. J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kronenberg</span>  <span class="cit-name-given-names">Z. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Long-read sequence assembly of the gorilla genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">352</span>: <span class="cit-fpage">aae0344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aae0344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGordon%26rft.auinit1%253DD.%26rft.volume%253D352%26rft.issue%253D6281%26rft.spage%253Daae0344%26rft.epage%253Daae0344%26rft.atitle%253DLong-read%2Bsequence%2Bassembly%2Bof%2Bthe%2Bgorilla%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aae0344%26rft_id%253Dinfo%253Apmid%252F27034376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNTIvNjI4MS9hYWUwMzQ0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.23" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.24" data-doi="10.1101/gr.185579.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoskins</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The Release 6 reference sequence of the Drosophila melanogaster genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">458</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.185579.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185579.114%26rft_id%253Dinfo%253Apmid%252F25589440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNDQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.25" data-doi="10.1534/genetics.115.180539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huddleston</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An Incomplete Understanding of Human Genetic Variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1254</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.180539</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHuddleston%26rft.auinit1%253DJ.%26rft.volume%253D202%26rft.issue%253D4%26rft.spage%253D1251%26rft.epage%253D1254%26rft.atitle%253DAn%2BIncomplete%2BUnderstanding%2Bof%2BHuman%2BGenetic%2BVariation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.180539%26rft_id%253Dinfo%253Apmid%252F27053122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi80LzEyNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxNDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.26" data-doi="10.1038/nature03154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Chicken Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">432</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature03154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHillier%26rft.auinit1%253DL.%2BW.%26rft.volume%253D432%26rft.issue%253D7018%26rft.spage%253D695%26rft.epage%253D716%26rft.atitle%253DSequence%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bchicken%2Bgenome%2Bprovide%2Bunique%2Bperspectives%2Bon%2Bvertebrate%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03154%26rft_id%253Dinfo%253Apmid%252F15592404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15592404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225597200038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.27" data-doi="10.1038/nbt.4060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miga</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Nanopore sequencing and assembly of a human genome with ultra-long reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.4060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D36%26rft.spage%253D338%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.28" data-doi="10.1101/gr.170720.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toshimoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.170720.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170720.113%26rft_id%253Dinfo%253Apmid%252F24755901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTM4NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.29" data-doi="10.1038/sdata.2014.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babayan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeadon</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Long-read, whole-genome shotgun sequence data for five model organisms.</span> <abbr class="cit-jnl-abbrev">Sci. Data</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">140045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sdata.2014.45</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BData%26rft.volume%253D1%26rft.spage%253D140045%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsdata.2014.45%26rft_id%253Dinfo%253Apmid%252F25977796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sdata.2014.45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25977796&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.30" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.31" data-doi="10.1186/gb-2004-5-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2004-5-2-r12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14759262&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.32" data-doi="10.1186/1471-2105-15-S9-S4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">K.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khalak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tse</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Near-optimal assembly for shotgun sequencing with noisy reads.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">S4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-15-S9-S4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253DS4%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-S9-S4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-S9-S4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.33" data-doi="10.1038/35057062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linton</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Initial sequencing and analysis of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">860</span>–<span class="cit-lpage">921</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35057062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D409%26rft.issue%253D6822%26rft.spage%253D860%26rft.epage%253D921%26rft.atitle%253DInitial%2Bsequencing%2Band%2Banalysis%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35057062%26rft_id%253Dinfo%253Apmid%252F11237011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35057062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11237011&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166938800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.34" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.10.3143.35" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics.</abbr> <span class="cit-vol">2009</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.36" data-doi="10.1093/bioinformatics/btw152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2103</span>–<span class="cit-lpage">2110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw152%26rft_id%253Dinfo%253Apmid%252F27153593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153593&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.37" data-doi="10.1038/nmeth.3444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loman</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A complete bacterial genome assembled de novo using only nanopore sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">733</span>–<span class="cit-lpage">735</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D733%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3444%26rft_id%253Dinfo%253Apmid%252F26076426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26076426&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.10.3143.38" data-doi="10.1371/journal.pone.0106689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCoy</span>  <span class="cit-name-given-names">Rajiv C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blauwkamp</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kertesz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Illumina TruSeq Synthetic Long-Reads Empower de Novo Assembly and Resolve Complex, Highly-Repetitive Transposable Elements.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e106689</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0106689</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De106689%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0106689%26rft_id%253Dinfo%253Apmid%252F25188499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0106689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25188499&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.39" data-doi="10.1038/nmeth.1374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Medvedev</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanciu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brudno</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Computational methods for discovering structural variation with next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">S13</span>–<span class="cit-lpage">S20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1374</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMedvedev%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D11%2BSuppl%26rft.spage%253DS13%26rft.epage%253DS20%26rft.atitle%253DComputational%2Bmethods%2Bfor%2Bdiscovering%2Bstructural%2Bvariation%2Bwith%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1374%26rft_id%253Dinfo%253Apmid%252F19844226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19844226&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277646700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.40" data-doi="10.1038/s41467-018-03016-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jupe</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bemm</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motley</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandoval</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">High contiguity Arabidopsis thaliana genome assembly with a single nanopore flow cell.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">541</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-03016-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-03016-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-03016-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.41" data-doi="10.1371/journal.pone.0013142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moschetti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimitri</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caizzi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junakovic</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic instability of I elements of Drosophila melanogaster in absence of dysgenic crosses.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e13142</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0013142</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D5%26rft.spage%253De13142%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0013142%26rft_id%253Dinfo%253Apmid%252F20957225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0013142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20957225&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.42" data-doi="10.1109/TIT.2013.2270273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Motahari</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bresler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tse</span>  <span class="cit-name-given-names">D. N. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Information Theory of DNA Shotgun Sequencing.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Inf. Theory</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">6273</span>–<span class="cit-lpage">6289</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/TIT.2013.2270273</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BInf.%2BTheory%26rft.volume%253D59%26rft.spage%253D6273%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTIT.2013.2270273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TIT.2013.2270273&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.43" data-doi="10.1038/nature01262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Mouse Genome Sequencing Consortium; </span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindblad-Toh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abril</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Initial sequencing and comparative analysis of the mouse genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">562</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature01262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaterston%26rft.auinit1%253DR.%2BH.%26rft.volume%253D420%26rft.issue%253D6915%26rft.spage%253D520%26rft.epage%253D562%26rft.atitle%253DInitial%2Bsequencing%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01262%26rft_id%253Dinfo%253Apmid%252F12466850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12466850&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179611600053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.44" data-doi="10.1126/science.287.5461.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dew</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasulo</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A whole-genome assembly of Drosophila.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2196</span>–<span class="cit-lpage">2204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.287.5461.2196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DE.%2BW.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2196%26rft.epage%253D2204%26rft.atitle%253DA%2BWhole-Genome%2BAssembly%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2196%26rft_id%253Dinfo%253Apmid%252F10731133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.45" data-doi="10.3389/fbioe.2015.00008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Narzisi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The challenge of small-scale repeats for indel discovery.</span> <abbr class="cit-jnl-abbrev">Front. Bioeng. Biotechnol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fbioe.2015.00008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BBioeng.%2BBiotechnol.%26rft.volume%253D3%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffbioe.2015.00008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fbioe.2015.00008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.46" data-doi="10.1093/oxfordjournals.molbev.a040649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Periquet</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Distribution of hobo transposable elements in natural populations of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a040649</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DPascual%26rft.auinit1%253DL%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D282%26rft.epage%253D296%26rft.atitle%253DDistribution%2Bof%2Bhobo%2Btransposable%2Belements%2Bin%2Bnatural%2Bpopulations%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a040649%26rft_id%253Dinfo%253Apmid%252F1712890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a040649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1712890&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FH39600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.47" data-doi="10.1093/bib/bbq020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paszkiewicz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studholme</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">De novo assembly of short sequence reads.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbq020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbq020%26rft_id%253Dinfo%253Apmid%252F20724458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbq020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20724458&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.48" data-doi="10.1007/BF01435241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Periquet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemeunier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigot</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamelin</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The evolutionary genetics of the hobo transposable element in the Drosophila melanogaster complex.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF01435241</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DPeriquet%26rft.auinit1%253DG.%26rft.volume%253D93%26rft.issue%253D1-3%26rft.spage%253D79%26rft.epage%253D90%26rft.atitle%253DThe%2Bevolutionary%2Bgenetics%2Bof%2Bthe%2Bhobo%2Btransposable%2Belement%2Bin%2Bthe%2BDrosophila%2Bmelanogaster%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01435241%26rft_id%253Dinfo%253Apmid%252F7813919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01435241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7813919&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PV62800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.49" data-doi="10.1371/journal.pgen.1003080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population Genomics of sub-saharan Drosophila melanogaster: African diversity and non-African admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003080</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003080</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003080%26rft_id%253Dinfo%253Apmid%252F23284287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284287&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.50" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.51" data-doi="10.1093/nar/gkv1193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chirn</span>  <span class="cit-name-given-names">G.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanodia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sytnikova</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brembs</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Unique transposon landscapes are pervasive across Drosophila melanogaster genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10655</span>–<span class="cit-lpage">10672</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1193%26rft_id%253Dinfo%253Apmid%252F26578579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578579&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.52" data-doi="10.1101/gr.131383.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magoc</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAGE: a critical evaluation of genome assemblies and assembly algorithms.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.131383.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131383.111%26rft_id%253Dinfo%253Apmid%252F22147368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTU3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.53" data-doi="10.1093/nar/gku1099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strelets</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crosby</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">FlyBase: introduction of the Drosophila melanogaster Release 6 reference genome assembly and large-scale migration of genome annotations.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D690</span>–<span class="cit-lpage">D697</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1099%26rft_id%253Dinfo%253Apmid%252F25398896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398896&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.54" data-doi="10.1101/014399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shomorony</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtade</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tse</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Do Read Errors Matter for Genome Assembly?</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/014399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F014399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwMTQzOTl2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.55" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>  <span class="cit-name-given-names">A. F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>2013-2015 RepeatMasker Open-4.0. <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.57" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbs017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.58" data-doi="10.1038/nrg3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Repetitive DNA and next-generation sequencing: computational challenges and solutions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span> <span class="cit-comment">(corrigendum: Nat. Rev. Genet. 13: 146)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3117%26rft_id%253Dinfo%253Apmid%252F22124482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22124482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.59" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouelliel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: An Integrated Tool for Comprehensive Microbial Variant Detection and Genome Assembly Improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.60" data-doi="10.1038/srep31900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Z. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">DBG2OLC: Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation Sequencing Technologies.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31900</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep31900</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D31900%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep31900%26rft_id%253Dinfo%253Apmid%252F27573208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep31900&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27573208&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.61" data-doi="10.1126/science.1068037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">G. K.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A draft sequence of the rice genome (Oryza sativa L. ssp. indica).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1068037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D296%26rft.issue%253D5565%26rft.spage%253D79%26rft.epage%253D92%26rft.atitle%253DA%2BDraft%2BSequence%2Bof%2Bthe%2BRice%2BGenome%2B%2528Oryza%2Bsativa%2BL.%2Bssp.%2Bindica%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1068037%26rft_id%253Dinfo%253Apmid%252F11935017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyOTYvNTU2NS83OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3143.62" data-doi="10.1038/sj.hdy.6801029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakharenko</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovalenko</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mai</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fluorescence in situ hybridization analysis of hobo, mdg1 and Dm412 transposable elements reveals genomic instability following the Drosophila melanogaster genome sequencing.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">530</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sj.hdy.6801029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DZakharenko%26rft.auinit1%253DL.%2BP.%26rft.volume%253D99%26rft.issue%253D5%26rft.spage%253D525%26rft.epage%253D530%26rft.atitle%253DFluorescence%2Bin%2Bsitu%2Bhybridization%2Banalysis%2Bof%2Bhobo%252C%2Bmdg1%2Band%2BDm412%2Btransposable%2Belements%2Breveals%2Bgenomic%2Binstability%2Bfollowing%2Bthe%2BDrosophila%2Bmelanogaster%2Bgenome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6801029%26rft_id%253Dinfo%253Apmid%252F17622267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6801029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17622267&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3143.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3143.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3131" title="Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3155" title="Mapping Quantitative Trait Loci for Tolerance to Pythium irregulare in Soybean (Glycine max L.)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3143" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-I0GvlDbWLUefBkjGDYbuEgNJynJZ-7GKJyRFQNnMYzM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3143" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3143" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3143" class="active" data-icon-position="" data-hide-link-title="0">Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490970" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1g0nB7XGR1aMg4RNZyVcb_tcK9qIks7V_dwIui3DheU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3143.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3143" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-TQtc_ErLSfF4KrkmowYFF9GkKTc55x_AVeYrAzXc--8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3143" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3143.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490970" id="node490970--2" data-pisa="ggg;8/10/3143" data-pisa-master="ggg;g3.118.200162" data-apath="/ggg/8/10/3143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Low-Cost Assembly of the <em>Drosophila melanogaster</em> Reference Genome Using Low-Coverage, Long-Read Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3220-4927" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edwin A. Solares</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2414-9187" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mahul Chakraborty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="3">Shannon Kalsow</span>, <span class="highwire-citation-author" data-delta="4">Kate Hall</span>, <span class="highwire-citation-author" data-delta="5">Anoja G. Perera</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9474-0891" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. J. Emerson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Scott Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3143-3154;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200162 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490970--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Edwin A. Solares </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edwin%2BA.%2BSolares%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Solares%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdwin%2BA.%2BSolares%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3220-4927" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edwin A. Solares</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mahul Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahul%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMahul%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2414-9187" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mahul Chakraborty</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shannon Kalsow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BKalsow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalsow%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BKalsow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kate Hall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kate%2BHall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hall%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKate%2BHall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anoja G. Perera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anoja%2BG.%2BPerera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perera%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnoja%2BG.%2BPerera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">J. J. Emerson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJ.%2BEmerson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emerson%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BJ.%2BEmerson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9474-0891" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. J. Emerson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jje@uci.edu" class="" data-icon-position="" data-hide-link-title="0">jje@uci.edu</a>
<a href="mailto:rsh@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rsh@stowers.org</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">R. Scott Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BScott%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Scott Hawley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jje@uci.edu" class="" data-icon-position="" data-hide-link-title="0">jje@uci.edu</a>
<a href="mailto:rsh@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rsh@stowers.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3143" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3143&amp;title=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3143&amp;title=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3143&amp;title=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3143&amp;text=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3143&amp;title=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3143&amp;t=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3143&amp;title=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3143&amp;title=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490970/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490970" id="node490970--3" data-pisa="ggg;8/10/3143" data-pisa-master="ggg;g3.118.200162" data-apath="/ggg/8/10/3143.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid Low-Cost Assembly of the <em>Drosophila melanogaster</em> Reference Genome Using Low-Coverage, Long-Read Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3220-4927" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Edwin A. Solares</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-2414-9187" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mahul Chakraborty</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="3">Shannon Kalsow</span>, <span class="highwire-citation-author" data-delta="4">Kate Hall</span>, <span class="highwire-citation-author" data-delta="5">Anoja G. Perera</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9474-0891" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>J. J. Emerson</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Scott Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3143-3154;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200162 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490970--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Edwin A. Solares </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edwin%2BA.%2BSolares%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Solares%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdwin%2BA.%2BSolares%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3220-4927" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Edwin A. Solares</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mahul Chakraborty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mahul%2BChakraborty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chakraborty%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMahul%2BChakraborty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2414-9187" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mahul Chakraborty</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Shannon Kalsow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shannon%2BKalsow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalsow%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShannon%2BKalsow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kate Hall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kate%2BHall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hall%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKate%2BHall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Anoja G. Perera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anoja%2BG.%2BPerera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Perera%20AG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnoja%2BG.%2BPerera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">J. J. Emerson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California Irvine, CA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BJ.%2BEmerson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emerson%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ.%2BJ.%2BEmerson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9474-0891" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for J. J. Emerson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jje@uci.edu" class="" data-icon-position="" data-hide-link-title="0">jje@uci.edu</a>
<a href="mailto:rsh@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rsh@stowers.org</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">R. Scott Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BScott%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Scott Hawley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jje@uci.edu" class="" data-icon-position="" data-hide-link-title="0">jje@uci.edu</a>
<a href="mailto:rsh@stowers.org" class="" data-icon-position="" data-hide-link-title="0">rsh@stowers.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3143&amp;rft.epage=3154&amp;rft.atitle=Rapid%20Low-Cost%20Assembly%20of%20the%20Drosophila%20melanogaster%20Reference%20Genome%20Using%20Low-Coverage%2C%20Long-Read%20Sequencing&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Solares%2C+Edwin+A.&amp;rft.au=Chakraborty%2C+Mahul&amp;rft.au=Miller%2C+Danny+E.&amp;rft.au=Kalsow%2C+Shannon&amp;rft.au=Hall%2C+Kate&amp;rft.au=Perera%2C+Anoja+G.&amp;rft.au=Emerson%2C+J.+J.&amp;rft.au=Hawley%2C+R.+Scott"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490970/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490970/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490970/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490970/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490970/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490970/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490970/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490970/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490970/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490970/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490970/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490970/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3143" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3143.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3143.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/10/3143.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/10/3143.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
