<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3173" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3173.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3173.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3173.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/10/3173/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3173.atom" />
<meta name="HW.pisa" content="ggg;8/10/3173" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" />
<meta name="DC.Identifier" content="10.1534/g3.118.200185" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Garcia et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.6084/m9.figshare.6970277" />
<meta name="DC.Description" content="Experimental evidence shows that synonymous mutations can have important consequences on genetic fitness. Many organisms display codon usage bias (CUB), where synonymous codons that are translated into the same amino acid appear with distinct frequency. Within genomes, CUB is thought to arise from selection for translational efficiency and accuracy, termed the translational efficiency hypothesis (TEH). Indeed, CUB indices correlate with protein expression levels, which is widely interpreted as evidence for translational selection. However, these tests neglect -1 programmed ribosomal frameshifting (-1 PRF), an important translational disruption effect found across all organisms of the tree of life. Genes that contain -1 PRF signals should cost more to express than genes without. Thus, CUB indices that do not consider -1 PRF may overestimate genes’ true adaptation to translational efficiency and accuracy constraints. Here, we first investigate whether -1 PRF signals do indeed carry such translational cost. We then propose two corrections for CUB indices for genes containing -1 PRF signals. We retest the TEH in Saccharomyces cerevisiae under these corrections. We find that the correlation between corrected CUB index and protein expression remains intact for most levels of uniform -1 PRF efficiencies, and tends to increase when these efficiencies decline with protein expression. We conclude that the TEH is strengthened and that -1 PRF events constitute a promising and useful tool to examine the relationships between CUB and selection for translation efficiency and accuracy." />
<meta name="DC.Contributor" content="Victor Garcia" />
<meta name="DC.Contributor" content="Stefan Zoller" />
<meta name="DC.Contributor" content="Maria Anisimova" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Experimental evidence shows that synonymous mutations can have important consequences on genetic fitness. Many organisms display &lt;i&gt;codon usage bias&lt;/i&gt; (CUB), where synonymous codons that are translated into the same amino acid appear with distinct frequency. Within genomes, CUB is thought to arise from selection for translational efficiency and accuracy, termed the &lt;i&gt;translational efficiency hypothesis&lt;/i&gt; (TEH). Indeed, CUB indices correlate with protein expression levels, which is widely interpreted as evidence for translational selection. However, these tests neglect -1 programmed ribosomal frameshifting (-1 PRF), an important translational disruption effect found across all organisms of the tree of life. Genes that contain -1 PRF signals should cost more to express than genes without. Thus, CUB indices that do not consider -1 PRF may overestimate genes’ true adaptation to translational efficiency and accuracy constraints. Here, we first investigate whether -1 PRF signals do indeed carry such translational cost. We then propose two corrections for CUB indices for genes containing -1 PRF signals. We retest the TEH in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; under these corrections. We find that the correlation between corrected CUB index and protein expression remains intact for most levels of uniform -1 PRF efficiencies, and tends to increase when these efficiencies decline with protein expression. We conclude that the TEH is strengthened and that -1 PRF events constitute a promising and useful tool to examine the relationships between CUB and selection for translation efficiency and accuracy.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3173" />
<meta name="citation_id" content="8/10/3173" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3173" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3173.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3173.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3173.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200185" />
<meta name="citation_pmid" content="30111621" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3173" />
<meta name="citation_lastpage" content="3183" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Victor Garcia" />
<meta name="citation_author_institution" content="Zurich University of Applied Sciences, Wädenswil, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland" />
<meta name="citation_author_email" content="victor.garcia_palencia@alumni.ethz.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6667-7098" />
<meta name="citation_author" content="Stefan Zoller" />
<meta name="citation_author_institution" content="ETH Zurich, Zurich, Switzerland" />
<meta name="citation_author" content="Maria Anisimova" />
<meta name="citation_author_institution" content="Zurich University of Applied Sciences, Wädenswil, Switzerland" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland" />
<meta name="citation_reference" content="citation_journal_title=Translation (Austin);citation_author=V. M. Advani;citation_author=A. T. Belew;citation_author=J. D. Dinman;citation_title=Yeast telomere maintenance is globally controlled by programmed ribosomal frameshifting and the nonsense-mediated mrna decay pathway.;citation_pages=e24418;citation_volume=1;citation_year=2013;citation_pmid=24563826;citation_doi=10.4161/trla.24418" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=V. M. Advani;citation_author=J. D. Dinman;citation_title=Reprogramming the genetic code: The emerging role of ribosomal frameshifting in regulating cellular gene expression.;citation_pages=21-26;citation_volume=38;citation_year=2016;citation_doi=10.1002/bies.201500131" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Akashi;citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.;citation_pages=927-935;citation_volume=136;citation_year=1994;citation_issue=3;citation_pmid=8005445" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Arava;citation_author=Y. Wang;citation_author=J. D. Storey;citation_author=C. L. Liu;citation_author=P. O. Brown;citation_title=Genome-wide analysis of mrna translation profiles in saccharomyces cerevisiae.;citation_volume=100;citation_year=2003;citation_pmid=12660367;citation_doi=10.1073/pnas.0635171100" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=P. V. Baranov;citation_author=J. F. Atkins;citation_author=M. M. Yordanova;citation_title=Augmented genetic decoding: global, local and temporal alterations of decoding processes and codon meaning.;citation_pages=517-529;citation_volume=16;citation_year=2015;citation_pmid=26260261;citation_doi=10.1038/nrg3963" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. T. Belew;citation_author=V. M. Advani;citation_author=J. D. Dinman;citation_title=Endogenous ribosomal frameshift signals operate as mrna destabilizing elements through at least two molecular pathways in yeast.;citation_volume=39;citation_year=2011;citation_pmid=21109528;citation_doi=10.1093/nar/gkq1220" />
<meta name="citation_reference" content="citation_journal_title=Cell Cycle;citation_author=A. T. Belew;citation_author=J. D. Dinman;citation_title=Cell cycle control (and more) by programmed -1 ribosomal frameshifting: implications for disease and therapeutics.;citation_pages=172-178;citation_volume=14;citation_year=2015;citation_pmid=25584829;citation_doi=10.4161/15384101.2014.989123" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. T. Belew;citation_author=N. L. Hepler;citation_author=J. L. Jacobs;citation_author=J. D. Dinman;citation_title=PRFdb: a database of computationally predicted eukaryotic programmed -1 ribosomal frameshift signals.;citation_pages=339-339;citation_volume=9;citation_year=2008;citation_pmid=18637175;citation_doi=10.1186/1471-2164-9-339" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. T. Belew;citation_author=A. Meskauskas;citation_author=S. Musalgaonkar;citation_author=V. M. Advani;citation_author=S. O. Sulima;citation_title=Ribosomal frameshifting in the ccr5 mrna is regulated by mirnas and the nmd pathway.;citation_pages=265-269;citation_volume=512;citation_year=2014;citation_pmid=25043019;citation_doi=10.1038/nature13429" />
<meta name="citation_reference" content="citation_journal_title=Virus research;citation_journal_abbrev=Virus Res;citation_author=I. Brierley;citation_author=F. J. Dos Ramos;citation_title=Programmed ribosomal frameshifting in HIV-1 and the SARS-CoV.;citation_pages=29-42;citation_volume=119;citation_year=2006;citation_issue=1;citation_pmid=16310880;citation_doi=10.1016/j.virusres.2005.10.008" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=N. Caliskan;citation_author=V. I. Katunin;citation_author=R. Belardinelli;citation_author=F. Peske;citation_author=M. V. Rodnina;citation_title=Programmed -1 frameshifting by kinetic partitioning during impeded translocation.;citation_pages=1619-1631;citation_volume=157;citation_year=2014;citation_pmid=24949973;citation_doi=10.1016/j.cell.2014.04.041" />
<meta name="citation_reference" content="citation_journal_title=Trends Biochem. Sci.;citation_author=N. Caliskan;citation_author=F. Peske;citation_author=M. V. Rodnina;citation_title=Changed in translation: mrna recoding by -1 programmed ribosomal frameshifting.;citation_pages=265-274;citation_volume=40;citation_year=2015;citation_pmid=25850333;citation_doi=10.1016/j.tibs.2015.03.006" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=A. Carbone;citation_author=A. Zinovyev;citation_author=F. Képès;citation_title=Codon adaptation index as a measure of dominating codon bias;citation_pages=2005-2015;citation_volume=19;citation_year=2003;citation_issue=16;citation_pmid=14594704;citation_doi=10.1093/bioinformatics/btg272" />
<meta name="citation_reference" content="citation_journal_title=Journal of evolutionary biology;citation_journal_abbrev=J Evol Biol;citation_author=D. B. Carlini;citation_title=Experimental reduction of codon bias in the Drosophila alcohol dehydrogenase gene results in decreased ethanol tolerance of adult flies.;citation_pages=779-785;citation_volume=17;citation_year=2004;citation_issue=4;citation_pmid=15271077;citation_doi=10.1111/j.1420-9101.2004.00725.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. B. Carlini;citation_author=J. E. Genut;citation_title=Synonymous SNPs provide evidence for selective constraint on human exonic splicing enhancers.;citation_pages=89-98;citation_volume=62;citation_year=2006;citation_issue=1;citation_pmid=16320116;citation_doi=10.1007/s00239-005-0055-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. B. Carlini;citation_author=W. Stephan;citation_title=In Vivo Introduction of Unpreferred Synonymous Codons Into the Drosophila Adh Gene Results in Reduced Levels of ADH Protein;citation_pages=239-243;citation_volume=163;citation_year=2003;citation_issue=1;citation_pmid=12586711" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. V. Chamary;citation_author=J. L. Parmley;citation_author=L. D. Hurst;citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals.;citation_pages=98-108;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418745;citation_doi=10.1038/nrg1770" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=Y.-F. Chang;citation_author=J. S. Imam;citation_author=M. F. Wilkinson;citation_title=The nonsense-mediated decay RNA surveillance pathway.;citation_pages=51-74;citation_volume=76;citation_year=2007;citation_pmid=17352659;citation_doi=10.1146/annurev.biochem.76.050106.093909" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. J. Clare;citation_author=M. Belcourt;citation_author=P. J. Farabaugh;citation_title=Efficient translational frameshifting occurs within a conserved sequence of the overlap between the two genes of a yeast Ty1 transposon.;citation_pages=6816-6820;citation_volume=85;citation_year=1988;citation_issue=18;citation_pmid=2842793;citation_doi=10.1073/pnas.85.18.6816" />
<meta name="citation_reference" content="citation_journal_title=Extremophiles (Print);citation_journal_abbrev=Extremophiles (Print);citation_author=B. Cobucci-Ponzano;citation_author=M. Rossi;citation_author=M. Moracci;citation_title=Translational recoding in archaea.;citation_pages=793-803;citation_volume=16;citation_year=2012;citation_issue=6;citation_pmid=23015064;citation_doi=10.1007/s00792-012-0482-8" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=W. J. Craigen;citation_author=R. G. Cook;citation_author=W. P. Tate;citation_author=C. T. Caskey;citation_title=Bacterial peptide chain release factors: conserved primary structure and possible frameshift regulation of release factor 2.;citation_pages=3616-3620;citation_volume=82;citation_year=1985;citation_issue=11;citation_pmid=3889910;citation_doi=10.1073/pnas.82.11.3616" />
<meta name="citation_reference" content="citation_journal_abbrev=Microbe Wash DC;citation_author=J. D. Dinman;citation_title=Programmed Ribosomal Frameshifting Goes Beyond Viruses: Organisms from all three kingdoms use frameshifting to regulate gene expression, perhaps signaling a paradigm shift.;citation_pages=521-527;citation_volume=1;citation_year=2006;citation_issue=11;citation_pmid=17541450" />
<meta name="citation_reference" content="citation_journal_abbrev=Adv Protein Chem Struct Biol;citation_author=J. D. Dinman;citation_title=Control of gene expression by translational recoding.;citation_pages=129-149;citation_volume=86;citation_year=2012;citation_pmid=22243583;citation_doi=10.1016/B978-0-12-386497-0.00004-9" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. RNA;citation_author=J. D. Dinman;citation_title=Mechanisms and implications of programmed translational frameshifting.;citation_pages=661-673;citation_volume=3;citation_year=2012;citation_pmid=22715123;citation_doi=10.1002/wrna.1126" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational chemistry;citation_journal_abbrev=J Comput Chem;citation_author=R. M. Dirks;citation_author=N. A. Pierce;citation_title=An algorithm for computing nucleic acid base-pairing probabilities including pseudoknots.;citation_pages=1295-1304;citation_volume=25;citation_year=2004;citation_issue=10;citation_pmid=15139042;citation_doi=10.1002/jcc.20057" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=D. Dulude;citation_author=Y. A. Berchiche;citation_author=K. Gendron;citation_author=L. Brakier-Gingras;citation_author=N. Heveker;citation_title=Decreasing the frameshift efficiency translates into an equivalent reduction of the replication of the human immunodeficiency virus type 1.;citation_pages=127-136;citation_volume=345;citation_year=2006;citation_issue=1;citation_pmid=16256163;citation_doi=10.1016/j.virol.2005.08.048" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=L. Duret;citation_title=Evolution of synonymous codon usage in metazoans.;citation_pages=640-649;citation_volume=12;citation_year=2002;citation_issue=6;citation_pmid=12433576;citation_doi=10.1016/S0959-437X(02)00353-2" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. Friberg;citation_author=P. von Rohr;citation_author=G. Gonnet;citation_title=Limitations of codon adaptation index and other coding DNA-based features for prediction of protein expression in Saccharomyces cerevisiae.;citation_pages=1083-1093;citation_volume=21;citation_year=2004;citation_issue=13;citation_pmid=15484285;citation_doi=10.1002/yea.1150" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W.-K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Trends in biotechnology;citation_journal_abbrev=Trends Biotechnol;citation_author=C. Gustafsson;citation_author=S. Govindarajan;citation_author=J. Minshull;citation_title=Codon bias and heterologous protein expression.;citation_pages=346-353;citation_volume=22;citation_year=2004;citation_issue=7;citation_pmid=15245907;citation_doi=10.1016/j.tibtech.2004.04.006" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=General rules for optimal codon choice.;citation_pages=e1000556-e1000556;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593368;citation_doi=10.1371/journal.pgen.1000556" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. J. Holland;citation_title=Transcript abundance in yeast varies over six orders of magnitude.;citation_volume=277;citation_year=2002;citation_pmid=11882647;citation_doi=10.1074/jbc.C200101200" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. C. Holstege;citation_author=E. G. Jennings;citation_author=J. J. Wyrick;citation_author=T. I. Lee;citation_author=C. J. Hengartner;citation_title=Dissecting the regulatory circuitry of a eukaryotic genome.;citation_pages=717-728;citation_volume=95;citation_year=1998;citation_issue=5;citation_pmid=9845373;citation_doi=10.1016/S0092-8674(00)81641-4" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=R. Hunt;citation_author=Z. E. Sauna;citation_author=S. V. Ambudkar;citation_author=M. M. Gottesman;citation_author=C. Kimchi-Sarfaty;citation_title=Silent (synonymous) SNPs: should we care about them?;citation_pages=23-39;citation_volume=578;citation_year=2009;citation_pmid=19768585;citation_doi=10.1007/978-1-60327-411-1_2" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system.;citation_pages=389-409;citation_volume=151;citation_year=1981;citation_issue=3;citation_pmid=6175758;citation_doi=10.1016/0022-2836(81)90003-6" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=T. Ikemura;citation_title=Correlation between the abundance of yeast transfer rnas and the occurrence of the respective codons in protein genes. differences in synonymous codon choice patterns of yeast and escherichia coli with reference to the abundance of isoaccepting transfer rnas.;citation_pages=573-597;citation_volume=158;citation_year=1982;citation_issue=4;citation_pmid=6750137;citation_doi=10.1016/0022-2836(82)90250-9" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=T. Ikemura;citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;citation_pages=13-34;citation_volume=2;citation_year=1985;citation_issue=1;citation_pmid=3916708" />
<meta name="citation_reference" content="citation_journal_title=The international journal of biochemistry &amp; cell biology;citation_journal_abbrev=Int J Biochem Cell Biol;citation_author=I. G. Ivanov;citation_author=A. A. Saraffova;citation_author=M. G. Abouhaidar;citation_title=Unusual effect of clusters of rare arginine (AGG) codons on the expression of human interferon alpha 1 gene in Escherichia coli.;citation_pages=659-666;citation_volume=29;citation_year=1997;citation_issue=4;citation_pmid=9363643;citation_doi=10.1016/S1357-2725(96)00161-6" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. Jacks;citation_author=H. E. Varmus;citation_title=Expression of the Rous sarcoma virus pol gene by ribosomal frameshifting;citation_pages=1237-1242;citation_volume=230;citation_year=1985;citation_issue=4731;citation_pmid=2416054;citation_doi=10.1126/science.2416054" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Jansen;citation_author=H. J. Bussemaker;citation_author=M. Gerstein;citation_title=Revisiting the codon adaptation index from a whole-genome perspective: analyzing the relationship between gene expression and codon occurrence in yeast using a variety of models;citation_pages=2242-2251;citation_volume=31;citation_year=2003;citation_issue=8;citation_pmid=12682375;citation_doi=10.1093/nar/gkg306" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. A. Jelinsky;citation_author=L. D. Samson;citation_title=Global response of Saccharomyces cerevisiae to an alkylating agent;citation_pages=1486-1491;citation_volume=96;citation_year=1999;citation_issue=4;citation_pmid=9990050;citation_doi=10.1073/pnas.96.4.1486" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=S. Kanaya;citation_author=Y. Yamada;citation_author=M. Kinouchi;citation_author=Y. Kudo;citation_author=T. Ikemura;citation_title=Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis.;citation_pages=290-298;citation_volume=53;citation_year=2001;citation_issue=4-5;citation_pmid=11675589;citation_doi=10.1007/s002390010219" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=S. Kanaya;citation_author=Y. Yamada;citation_author=Y. Kudo;citation_author=T. Ikemura;citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis.;citation_pages=143-155;citation_volume=238;citation_year=1999;citation_issue=1;citation_pmid=10570992;citation_doi=10.1016/S0378-1119(99)00225-5" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=R. Ketteler;citation_title=On programmed ribosomal frameshifting: the alternative proteomes.;citation_pages=242-242;citation_volume=3;citation_year=2012;citation_pmid=23181069;citation_doi=10.3389/fgene.2012.00242" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=C. Kimchi-Sarfaty;citation_author=J. M. Oh;citation_author=I.-W. Kim;citation_author=Z. E. Sauna;citation_author=A. M. Calcagno;citation_title=A ”silent” polymorphism in the mdr1 gene changes substrate specificity.;citation_volume=315;citation_year=2007;citation_pmid=17185560;citation_doi=10.1126/science.1135308" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=R. M. Kliman;citation_author=N. Irving;citation_author=M. Santiago;citation_title=Selection conflicts, gene expression, and codon usage trends in yeast.;citation_pages=98-109;citation_volume=57;citation_year=2003;citation_issue=1;citation_pmid=12962310;citation_doi=10.1007/s00239-003-2459-9" />
<meta name="citation_reference" content="citation_journal_title=Pharmacogenomics;citation_journal_abbrev=Pharmacogenomics;citation_author=A. A. Komar;citation_title=Silent SNPs: impact on gene function and phenotype.;citation_pages=1075-1080;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17716239;citation_doi=10.2217/14622416.8.8.1075" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=A. A. Komar;citation_title=A pause for thought along the co-translational folding pathway.;citation_pages=16-24;citation_volume=34;citation_year=2009;citation_issue=1;citation_pmid=18996013;citation_doi=10.1016/j.tibs.2008.10.002" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=P. Lu;citation_author=C. Vogel;citation_author=R. Wang;citation_author=X. Yao;citation_author=E. M. Marcotte;citation_title=Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation.;citation_pages=117-124;citation_volume=25;citation_year=2007;citation_issue=1;citation_pmid=17187058;citation_doi=10.1038/nbt1270" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=I. G. Maia;citation_author=K. Séron;citation_author=A. L. Haenni;citation_author=F. Bernardi;citation_title=Gene expression from viral RNA genomes.;citation_pages=367-391;citation_volume=32;citation_year=1996;citation_issue=1-2;citation_pmid=8980488;citation_doi=10.1007/BF00039391" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. Nakao;citation_author=M. Yoshihama;citation_author=N. Kenmochi;citation_title=RPG: the Ribosomal Protein Gene database;citation_pages=D168-170;citation_volume=32;citation_year=2004;citation_issue=90001;citation_pmid=14681386;citation_doi=10.1093/nar/gkh004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. S. Newman;citation_author=S. Ghaemmaghami;citation_author=J. Ihmels;citation_author=D. K. Breslow;citation_author=M. Noble;citation_title=Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise.;citation_pages=840-846;citation_volume=441;citation_year=2006;citation_issue=7095;citation_pmid=16699522;citation_doi=10.1038/nature04785" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. L. Parmley;citation_author=J. V. Chamary;citation_author=L. D. Hurst;citation_title=Evidence for purifying selection against synonymous mutations in mammalian exonic splicing enhancers.;citation_volume=23;citation_year=2006;citation_pmid=16221894;citation_doi=10.1093/molbev/msj035" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=J. L. Parmley;citation_author=L. D. Hurst;citation_title=How do synonymous mutations affect fitness?;citation_pages=515-519;citation_volume=29;citation_year=2007;citation_issue=6;citation_pmid=17508390;citation_doi=10.1002/bies.20592" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=E. P. Plant;citation_author=J. D. Dinman;citation_title=Comparative study of the effects of heptameric slippery site composition on -1 frameshifting among different eukaryotic systems.;citation_volume=12;citation_year=2006;citation_pmid=16497657;citation_doi=10.1261/rna.2225206" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. P. Plant;citation_author=P. Wang;citation_author=J. L. Jacobs;citation_author=J. D. Dinman;citation_title=A programmed -1 ribosomal frameshift signal can function as a cis-acting mRNA destabilizing element;citation_pages=784-790;citation_volume=32;citation_year=2004;citation_issue=2;citation_pmid=14762205;citation_doi=10.1093/nar/gkh256" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=T. E. F. Quax;citation_author=N. J. Claassens;citation_author=D. Söll;citation_author=J. van der Oost;citation_title=Codon bias as a means to fine-tune gene expression.;citation_pages=149-161;citation_volume=59;citation_year=2015;citation_pmid=26186290;citation_doi=10.1016/j.molcel.2015.05.035" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. M. Resch;citation_author=L. Carmel;citation_author=L. Mariño-Ramírez;citation_author=A. Y. Ogurtsov;citation_author=S. A. Shabalina;citation_title=Widespread positive selection in synonymous sites of mammalian genes.;citation_volume=24;citation_year=2007;citation_pmid=17522087;citation_doi=10.1093/molbev/msm100" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. M. Resch;citation_author=A. Y. Ogurtsov;citation_author=I. B. Rogozin;citation_author=S. A. Shabalina;citation_author=E. V. Koonin;citation_title=Evolution of alternative and constitutive regions of mammalian 5'UTRs.;citation_pages=162-162;citation_volume=10;citation_year=2009;citation_pmid=19371439;citation_doi=10.1186/1471-2164-10-162" />
<meta name="citation_reference" content="citation_title=Measuring codon usage bias. Codon evolution: mechanisms and models;citation_author=A. Roth;citation_author=M. Anisimova;citation_author=G. M. Cannarozzi;citation_year=2012;citation_doi=10.1093/acprof:osobl/9780199601165.003.0013" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=F. P. Roth;citation_author=J. D. Hughes;citation_author=P. W. Estep;citation_author=G. M. Church;citation_title=Finding DNA regulatory motifs within unaligned noncoding sequences clustered by whole-genome mRNA quantitation.;citation_pages=939-945;citation_volume=16;citation_year=1998;citation_issue=10;citation_pmid=9788350;citation_doi=10.1038/nbt1098-939" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms.;citation_pages=28-38;citation_volume=24;citation_year=1986;citation_issue=1-2;citation_pmid=3104616;citation_doi=10.1007/BF02099948" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. M. Sharp;citation_author=W. H. Li;citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;citation_pages=1281-1295;citation_volume=15;citation_year=1987;citation_issue=3;citation_pmid=3547335;citation_doi=10.1093/nar/15.3.1281" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=N. Stoletzki;citation_author=A. Eyre-Walker;citation_title=Synonymous codon usage in escherichia coli: selection for translational accuracy.;citation_volume=24;citation_year=2007;citation_pmid=17101719;citation_doi=10.1093/molbev/msl166" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Evol.;citation_author=F. Supek;citation_title=The code of silence: Widespread associations between synonymous codon biases and gene function.;citation_pages=65-73;citation_volume=82;citation_year=2016;citation_pmid=26538122;citation_doi=10.1007/s00239-015-9714-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C.-J. Tsai;citation_author=Z. E. Sauna;citation_author=C. Kimchi-Sarfaty;citation_author=S. V. Ambudkar;citation_author=M. M. Gottesman;citation_title=Synonymous mutations and ribosome stalling can lead to altered folding pathways and distinct minima.;citation_pages=281-291;citation_volume=383;citation_year=2008;citation_issue=2;citation_pmid=18722384;citation_doi=10.1016/j.jmb.2008.08.012" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Tuller;citation_author=Y. Y. Waldman;citation_author=M. Kupiec;citation_author=E. Ruppin;citation_title=Translation efficiency is determined by both codon bias and folding energy.;citation_volume=107;citation_year=2010;citation_pmid=20133581;citation_doi=10.1073/pnas.0909910107" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Tuller;citation_author=H. Zur;citation_title=Multiple roles of the coding sequence 5′ end in gene expression regulation.;citation_volume=43;citation_year=2015;citation_pmid=25505165;citation_doi=10.1093/nar/gku1313" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. Varenne;citation_author=J. Buc;citation_author=R. Lloubes;citation_author=C. Lazdunski;citation_title=Translation is a non-uniform process. Effect of tRNA availability on the rate of elongation of nascent polypeptide chains.;citation_pages=549-576;citation_volume=180;citation_year=1984;citation_issue=3;citation_pmid=6084718;citation_doi=10.1016/0022-2836(84)90027-5" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=T. von der Haar;citation_title=A quantitative estimation of the global translational activity in logarithmically growing yeast cells.;citation_pages=87-87;citation_volume=2;citation_year=2008;citation_pmid=18925958;citation_doi=10.1186/1752-0509-2-87" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=F. Yamao;citation_author=Y. Andachi;citation_author=A. Muto;citation_author=T. Ikemura;citation_author=S. Osawa;citation_title=Levels of tRNAs in bacterial cells as affected by amino acid usage in proteins;citation_pages=6119-6122;citation_volume=19;citation_year=1991;citation_issue=22;citation_pmid=1956771;citation_doi=10.1093/nar/19.22.6119" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=F. Zhang;citation_author=S. Saha;citation_author=S. A. Shabalina;citation_author=A. Kashina;citation_title=Differential Arginylation of Actin Isoforms Is Regulated by Coding Sequence-Dependent Degradation;citation_pages=1534-1537;citation_volume=329;citation_year=2010;citation_issue=5998;citation_pmid=20847274;citation_doi=10.1126/science.1191701" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="Experimental evidence shows that synonymous mutations can have important consequences on genetic fitness. Many organisms display codon usage bias (CUB), where synonymous codons that are translated into the same amino acid appear with distinct frequency. Within genomes, CUB is thought to arise from selection for translational efficiency and accuracy, termed the translational efficiency hypothesis (TEH). Indeed, CUB indices correlate with protein expression levels, which is widely interpreted as evidence for translational selection. However, these tests neglect -1 programmed ribosomal frameshifting (-1 PRF), an important translational disruption effect found across all organisms of the tree of life. Genes that contain -1 PRF signals should cost more to express than genes without. Thus, CUB indices that do not consider -1 PRF may overestimate genes’ true adaptation to translational efficiency and accuracy constraints. Here, we first investigate whether -1 PRF signals do indeed carry such translational cost. We then propose two corrections for CUB indices for genes containing -1 PRF signals. We retest the TEH in Saccharomyces cerevisiae under these corrections. We find that the correlation between corrected CUB index and protein expression remains intact for most levels of uniform -1 PRF efficiencies, and tends to increase when these efficiencies decline with protein expression. We conclude that the TEH is strengthened and that -1 PRF events constitute a promising and useful tool to examine the relationships between CUB and selection for translation efficiency and accuracy." />
<meta name="og-title" property="og:title" content="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3173" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Experimental evidence shows that synonymous mutations can have important consequences on genetic fitness. Many organisms display codon usage bias (CUB), where synonymous codons that are translated into the same amino acid appear with distinct frequency. Within genomes, CUB is thought to arise from selection for translational efficiency and accuracy, termed the translational efficiency hypothesis (TEH). Indeed, CUB indices correlate with protein expression levels, which is widely interpreted as evidence for translational selection. However, these tests neglect -1 programmed ribosomal frameshifting (-1 PRF), an important translational disruption effect found across all organisms of the tree of life. Genes that contain -1 PRF signals should cost more to express than genes without. Thus, CUB indices that do not consider -1 PRF may overestimate genes’ true adaptation to translational efficiency and accuracy constraints. Here, we first investigate whether -1 PRF signals do indeed carry such translational cost. We then propose two corrections for CUB indices for genes containing -1 PRF signals. We retest the TEH in Saccharomyces cerevisiae under these corrections. We find that the correlation between corrected CUB index and protein expression remains intact for most levels of uniform -1 PRF efficiencies, and tends to increase when these efficiencies decline with protein expression. We conclude that the TEH is strengthened and that -1 PRF events constitute a promising and useful tool to examine the relationships between CUB and selection for translation efficiency and accuracy." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"S62dc5duzlLr7j-JtdoB89VPyt1A3e9G8LBPh1lxhI4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490980","apath":"\/ggg\/8\/10\/3173.atom","pisa":"ggg;8\/10\/3173","ac":{"\/ggg\/8\/10\/3173.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3173.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3173"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3173"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3173":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Oel4H7fRehp1t1LGJxm\\\\\\\/hhrDJoX50\\\\\\\/5m303vl+uXoePvdKiFHdLSIwZGChEx3pi\\\\\\\/xgym1eYsUy\\\\\\\/AQ25T3hVwikX13DDhYNWb4VroCUUsmwD2DXUNfAUrFh8AwvgSWwDWBjcmYLVECl9DT8l46k2cTBQEIR6ejS1pTDUrdbc+ptefiDYLmYnjlPfO8YcyBfQLyGSfTH2d2m4QfdVYrqUlijcUogu\\\\\\\/iOkuEHW2C4xMd1wwJnGCFVka\\\\\\\/NmapKtAQXzf3YNykQZAFZPQDD4GIQs3k9VRmeCzHYtghCyAZQC1\\\\\\\/TeCbyyIe3pB\\\\\\\/dVyukCLc20rtEhU3h4uOJHrvwyoUQRrqz+Vcfe3uMF\\\\\\\/WrfXw0fcq\\\\\\\/gvr3TRgVpuhAq\\\\\\\/VkgGnKxHAmKQ6ieVQenhE\\\\\\\/dLqYqKRZrmC8Mz04T2FzOIUtHD5dJ01uARxMLA+AmXkT8gWo2K\\\\\\\/mIYhbUSFkKY6RY59ykcQdaOdpNCyH+2mjx0FT9cnTEFzmXIG6SCP8UuBtR70iDq0prMA\\\\\\\/S++sZWwBU53FMRmrXkRGC6YE+MZWPLtqEnfsEHJ508Xhws1NN8hjUpBOnjPUiKP576tFvMSIi6KrEQb\\\\\\\/wQ8yn6CuoHsANwfq+4ROCPqRMb5gxLt7Je\\\\\\\/ylngJQrojbgdNqkyM5zW7iRHbkHJXiO9IAlslS1M3ex7IA3ICNe+oS8W47bCUljFOehcjaALmctwXqCS4wDFGQi41AJla3LL7vDFDAlrGJ5zfQEz1VCKEKUVgmjWwtkAISu8uVi7BI9WQ+VMJL8tBv2UotP55gPOyw2B7lxpCAmA130XjuVeNnqx9++cF+d7Tm1ZUV2N23BwmAbEfUZugnuN5fiV9kTu46bW0NvAsn5Z1go\\\\\\\/pyf7dADF\\\\\\\/1jcV0myE62FChhWCQrCC+t+Rzr+dj+Yig7nt4S8pbAUo6IfSX4olM0jx0TW226xvIFYg57+Yiit6dNWY6BhPPbIxaJc1JUrh5N+TyVfozfJmd\\\\\\\/SeeRLP3b9dDR0yMn1waCB8WpfpZj3qmJVzQu59PxDLL8K0pRclcf8FNqiIQ0T4n2QuwE+xnJZQ0R9BUjShTIYAKdO6QHQobzRhAPT0Mzh3To\\\\\\\/or9pD+7PIwfSxMA7QIU3+5t\\\\\\\/J9NaZSyTSkPSnHenFzzWPmrDd3oS5bX+mc03E0mOCrH7uHQfMvkaMrLriF2nO4bZS28xP\\\\\\\/chDvsqXtTvU6U7hV8OsQh98ozEuZWYBS1zSWZ7GcCR3BBMAA3Btb60pFpfLiBnIRJ8Ac7k0Lj6hguG764Z\\\\\\\/hB\\\u0022,\\\u0022iv\\\u0022:\\\u0022KDd98XJreVinLyMEPZHASw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022350d43c825507330436118e514c60d40ea4cd3d29033a4b99d5b52075b786b68\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MJF4zIABNrO0VtOtsAs1WzzMsWXPqjwkuS1sofGidX8c7Yj\\\\\\\/TGGWef\\\\\\\/vqJ+fZ4oRFK625UnOdwpaBNXR47KZ0bLX5Cg\\\\\\\/8op58t0keT7GqxflR5D0b38vLZz6C52ta3TL+vSChTW6BrJf\\\\\\\/1TnD6+FIs7I24z3ALKXX0TIU97oSuF+uFiqn2LUNdYXMcXpRJOD0pLW60OiAZDh8WRXjEFdPZFEZF\\\\\\\/jjd0Nx\\\\\\\/SJtvoG\\\\\\\/n3SgTeWfIIwKxj6SgeufeAxnB8dBkzIZ9fjvipxJ67Wn6IBdiC+v42fjKkmbgVwJz9EqjvEvyOlkk9V\\\\\\\/tHGVWBM9yvk\\\\\\\/\\\\\\\/qPKZ+zKexjBRFjGwgj9j5dwdnlV246u6MYKyl942PjCKbHVS+nSr\\\\\\\/+3sj4CL60mCJvXtmbQCM78TuAQjqPioGPnTnrOZvkZlTo6i\\\\\\\/d5e5W59swrFOS\\\\\\\/Lb4XLS\\\\\\\/6H7\\\\\\\/SFZjxTjZj8LsWRlpC8LnX43eO7576IhdDC\\\\\\\/88sudGWqcynA\\\\\\\/jnm+7Csxuh\\\\\\\/DAT5xUsMRruSSX1VRRODJhDP0jCDRO+QlJqL51R+mpUCB7D9YrnxhwtRz6aKaEgB6Z6aVVf\\\\\\\/MX38a7bumZpFJEd7w9Z6VfHAQ0rua8sjzp8KC2jnttFEh6Zplq1bYn6r\\\\\\\/F1uSyHVngKm1O4rQHszv3WVcOiGYdI\\\\\\\/nW+4YkkEr6j2Fr2Yfa7Gvvwl4IavqC6gIm0mwof2JthVDEkAd+kQvfMl4I9Mw\\\\\\\/aHH3YQWXN9+gyNAPGfebqk8c\\\\\\\/wk49FxTjmrbFCesv8XGZMtCoi5laeiwOtI65CmTiCr6ims5E3HlP9pR5A7Poo6AFDQ\\\\\\\/ryjWgaX8GHUzwMRg2PdkHmWy5PbgE7nwOfv7n1GX7NTnI503nActe\\\\\\\/NyypPmhh2DAAMXtUWZZYsIP9hXEfT4oQ3V4BnxCJHHronBuBKYpxAON\\\\\\\/aumqZrES1b1vJJ2N2GNdA1GkGNxpgJrJ\\\\\\\/NnPlTWZDBNq5EN00+ftpALwAGY1VZsZ9A3BA+nL2QB\\\\\\\/zfkkyWlneUFT4i+vQc4ihgx8pU75qUVf+LsctDws5MvApSgQBRacIfQaNtxyG\\\\\\\/bTHkIxbW+LZAXxfoJOEyJ25GCq8pO5qXqL5kytirKCQ6N93sL1RNqdMne7zGqN5WcGiVOJ84dsJOr5Uk6d8ReptXINCHk4JbMhbaoW6rwL6wsjDqAttyb72xcxUFBS878P2uAqSnv7r3CWnxTwPgMsYjsws8p2QVPd85mTgd\\\\\\\/dFMprHKWKuFTvoVBWx9htQfTKPD2NZDOPWGah5Vwnk7o25xeS\\\\\\\/DhXkfk0bzJcdCmHzd0T1QvCu\\\\\\\/ehDMewLgy98Ew\\\\\\\/6nWkDlyvNGIf3jWLD3TNzMbmowEPJg9Rk\\\\\\\/BSSqrPAf3jE8d4SlvJYATn5W4Hez8nbXyJuQQGjHamlkSYsHkKms8sswrONYixBQalE36thL0L3lnWxQ6GSwztxUewjWY\\\\\\\/QGk7ZDqwIMgs\\\\\\\/hUM3imzMPYpuGn7U1GS3Vq7qlE6fN8BjFJTIJgDk5xSDfd9dPRXNB8OMqLQFjR5m6OYCbnQbq2dyaDs2TuE3HolAPYwXOH8gd8QCm4UyGQcXEVDgHdem4vRCgd3GuSUocCIzmTzAhR1zN9DrKQ3M8fXQXUTkyQ8UjWISlUI8pbg\\\\\\\/s58jEXfcfeHJb7ZTtHhi6eerWLn61TrQ\\\\\\\/S28wYfAKs8zpcpmLE55yGJslt1RG41wEeQwxgScoWU+BrjgC9vQ3+5AoBFqn2dSL0uUM4xK4r3CM0eRo282YzZ3KxgotLPC2zE4YSc4S9\\\\\\\/pfuX1eGS9ljtkHqyirlvZOtVOy1SzPRRulfJxdgCFOx9HL\\\\\\\/Z3BZf5nURr\\\\\\\/OFNW\\\\\\\/5sHigWVESlj9uUDPsaIzMEKCqYlIZGJq5oTuM8i2gcoWi5RESviMlnF84\\\\\\\/eVMoYdZ20Q8\\\\\\\/ntb3hXitJnjRgmK6Xxzw6De2vFbwb6+ILDighAydzeY\\\\\\\/DGU0OTIEQ9EopMRgHYMWi6GZqZ15KeqaJxc29BhZ9XDlt75ASDNF+UfN3Wdrs7JgQz6IqNEndOyAN\\\\\\\/J7++KHO8ZQUiJwPSwsfGVjHXhF+ySdFf5ozGPtpN\\\\\\\/WcdGDLLbfe+lfA9VldxLEhDlIP\\\\\\\/bsN8Vk\\\\\\\/Z+wymxTVvO6vAtR35ef3ED9P73QKZcIBmLSpxTwKeiftK8JoFuX5e5BQXaeJMFf54iSNZiJB1sZyfEqUoJRlF2Fx+tYVdUGLv4ej8X3yptHQLDeQuGFoNKNYmVoQ2NXoE1YQMS+VO8DwQnBge\\\\\\\/6QUdiyULKfMn2Pta7PlVdDlJVg8VjE43XKmRQ5a4XK8462t04wmF3zTTOmas3ohtc+wBBsIihBjNE6EZ4LOToT6tXpCUhGE6Ixf+RLmrUTsWOjJM0GfCGbS+7kt4FvMrnvfzGFcBhYmnqcDw7n0dkLew3oCQcANAz4T5YZEBGTPfu4qYp0iz\\\\\\\/RnuCxX\\\\\\\/5WgYoov6RVOAuYhZ8yguf7druP+MvK6WWtHqoOx5M+KEAOVPDKuKipxK7pGWpJ\\\\\\\/XQwVxTcqzgm9ueVV\\\\\\\/LUIaIrACHEdZWPFvfKdv62OW4Sm87Uq8wI5P0Ml1vKdUdAYhQq6SLXWPnIMm5JNcZBgeeLY6WIkMZNiGAMJTRb0WwxGoKdPvH6ts1CsPN\\\\\\\/vaBOaeb9poMD9Oh9muN2YSqHr\\\\\\\/zED2YHfLfFrKVgSCcWXO5u1NRxE9xSnDV2jJDv4m6YdAHUcx+nNkcVx97\\\\\\\/BZsaIWX2m3ij\\\\\\\/5oO81UoJ5bSqbl29KgKmK62hHX\\\\\\\/jL03+UZ7iRspmdBbmhhLjuDFzKdj0hezUxSj8wzsdSQpEYrL+2aA9ImFW30RXruOkwWy0Ak0NjXNUOFAEWdgo6XU9sASmGaVc0vqMe2gCYaxCK4ROC3wHSqhqfyGYnxONTjX6rz5qbK9lGfiyETiXluUo89xtwFrWN7jD02\\\\\\\/\\\\\\\/ZTE3az3V8D9ZGQ+z8SgFBqphP+hnNZR7hg1cN+WXWV4zDXb1vxiQkibrOw1AvtZOn3mhAlmXjDGKoxictpI4wD\\\\\\\/BnA7rgx8PImP8uIx5dYP\\\\\\\/i\\\\\\\/3Svdr1mEAHMrrAnwfpBaMfFCaoebtKTMnsYedIgkbed3wYSiPMmm\\\\\\\/\\\\\\\/mLb8J7cDcVjoKCBOmXihL6bQFh54rCoYWRN+acPI9z6YKpHhNwNNjs3JBcsBZ7EAqe+wP5O7doZqQd7g7T8Tptx2ZFcvdHR0twtJMCuvKopgw9qe29vzR6LRaksnE+4YxvA1gv+sRGOj4W205JtM70RQrjcQJNb12KVA+T2miVBnrEywoKxwLVCu4377M7slYDXFTQa2EKZ26P9oXQaE3+05\\\\\\\/x0RMF32dF\\\\\\\/ARBOEcJmi0+lidxpeQkbEjaTEQ97Xt+LuzcnQgKbS0t7LJR3EtVqjC8hWjapOUxoRZxYUromLC9aNR2fqHu0GOE2rvgVpE7GAYUfHxufqiOdQmqHWAYhtEphOVvjDEW6LRpcwnmrc0pZMUP0LlMNGEylxXrxl8llp0EipeSu1dkmnRZbT\\\\\\\/uNmqN\\\\\\\/5Cianel8eOkUhGH3TY1S4tfyOqkegVtfj8GJ1A5isTkLC4An+q1m5xgkbit2FXzkGKkM4rK5eJFeAhJF4FsKTcerZok8X9gU2UZCK3HWltxxGuPmJfiFAOB8qoENHF+FJOZsBxAe9Q7qmOM1\\\\\\\/3cnKHjAmcxKsaBbxwcnI1cXIn3jdpvH8SaSOrxCHb4R2i4cbwfLCW8fcuEFdhI0PjTUbUKwbaXWJzybPHWCMv73B4I++pnI1ZASgmhBLR+cXD20ERHUugrffK8QiG1cMyCV7oL\\\\\\\/0ZjcEbeedIGpYF64YETJdddZKIHSo9TGh70Kf86HrW27Fa10huNtx9pmfNCoQRpDlZBTReasP7xu8sgYz9kqmzs3O3Nplq886\\\\\\\/5Wy\\\\\\\/lBPa8vu+6XyTW35Q4GMdLIA\\\\\\\/su\\\\\\\/7EujHw0B\\\\\\\/Vw2VsFyp\\\\\\\/rZ5GD5mgzrLRXM3JvQ8B90JeIszWzBGh9zcPRyrPh9AWfI8QIB48j9vXOMhzLs9TuSnAYPA6tyv9aY\\\\\\\/VZi1lD06Nb2thp1o+i5vMQMefTrJZgoHIA5LAvpwfyPg\\\\\\\/w1F5AN7uMk3gQvo\\\\\\\/SeDaV6FX9u3Syeffx1DA0D+YOKYVl2lJs5bP9aOuw7ws9Um2BnmX0ZIoBRKxtLPAxWvZVk2s2Y7YSDMz97Tp9dHec8YnrqM3Fa5\\\\\\\/j\\\\\\\/6B1GqzfXk6AN83\\\\\\\/z2jsMuq0ao6rPSlQ2uQfMSfL\\\\\\\/Qti2DsbiTuYH8L0Qt8i6GNF5eWBZhsx1SvJftDzPivA+U4PtFUQE2yQCEpRwlqQj7sOsWUAz9F3pYqiEA6Gw3mmG4R7F464MMjB+EAi0Piogu2KtHvt5u3ah9hf7G4e5b5y66pjcOutr5YH2is4dgFXjp1cIzmDtl1ejMrl3IArMgM5+yOGEcTws2GYzkQhDxlyZx93i15zXWgJGvQpBxw+uNvF8owYHC7W2zodYHZ2FqqujZT\\\\\\\/LEZKMloC6Rw0h0qLdnKS3R5l4UT4AWCJzazvxyomocrVk2bIX+cxHdhJ8N\\\\\\\/ViRP3PIWeGtGB+Q7aSs\\\\\\\/Ujk95UrXQG3uYST3jqtfxHMT6medNbdA3hS3icG6Lkhv7EMLahRfdERbuGIsBEEWKHtbegMfDZbqbaiuUF0KTF5DO4x5856o3F\\\\\\\/TF7OzmG82bbkTgZZkubHrXRWBBoVDGuoAXg1LAn9kTHHZNuocjdUV896FV4pTOz3pS723HQ\\\\\\\/I+Ciyb06\\\\\\\/2Wtt2BA\\\\\\\/2zOamv39ccaXNNIz8BUjhgft8HAauEfYCchS+VZiKIcH+38dOLSxpHyeU0cauJBqF7WQAlrD3ouFMfTZoPKJmuUeyrgnNr9qkLGJldZ\\\\\\\/1EMDg53MUk3sn7HPug9N10vV69pDmnz51R07EJI\\\\\\\/\\\\\\\/V\\\\\\\/EYo8rH4CorpSMWBnNtbTXWx\\\\\\\/DZMBEPk6rhTjpiUDRDCTz7t9d+BIXKwpEFpuIOMVewcMt8ASRKNNq\\\\\\\/U+XXpAdcSeeFvmhov9ITcFKvW182RVvowbbSnS0Yvw9fMtxG5B\\\\\\\/1F++o7+7QtYI35KCJHQZimZiGrpMMBgXyi9ZlUrhldRS\\\\\\\/RFhIPceDsFsjc66h3Qvi5fuRSNI\\\\\\\/fcCSqGcXVj8pk\\\\\\\/OwXNvObZyOTtQI8FOMsymloPzD4tJR+XTel0NTVYF99un1BRMKldX3M+TKDVzUwAr5LoYB+HMjn3ltYksGo\\\\\\\/CIzD84n8kcJK5SWE1iN7G9mjwgP7i9uET5cXGZdxKU7A9e4VbVVvAe8n2qjKBf4WMTbZEtnU52FdddD0eMCWROWrUWBdPgOju2IDnq055suOeGSTB6OrA38W1CjW97dnJSmfbGSmak0Pm2eoXNeehm9pRuBTzt3dbTyqHyxjyK1CUQtkGj4F5c+0YARPkjEyDt0O3o1sxsTCwzLo6S\\\\\\\/P3cxo2Mfxi8UN4G+gFPqVj0hi33AyfTUtdiQz9fYoMY0pCSD7dRm98A03eUUvVeeKpNzR40SCHqMXzSsRhQzRYzAXcEjdywXqbNkvuhTEwO\\\\\\\/w\\\\\\\/7yvEaSo9Gz1fT3sg0bcoK9VmqBikvkFPWbhb2LcTVZ4EZ+zgG7FStcUzxJQsS\\\\\\\/AjWjA4h4QK\\\\\\\/565DQ8omgPD9We6IxDNNQ6MUK6bJZ58gUitJzJUfheeeh5jt0recu\\\\\\\/eoW+6+tTL10IXPYSdJty\\\\\\\/uJ0731nUJM\\\\\\\/4pAW9DVx\\\\\\\/KgG8KElRRaX5uNr58BGte3wA3LwLw881WfXLvUqhp97482OCORuj5SGBVC\\\\\\\/w2ALY8OQ\\\\\\\/2Wnk4Mnf0KAHf2p5uTBVv9B+IGqLIfZKoCv20aGl2kNWGFtgUgZgKyY\\\\\\\/t5ELPowDA5E48uN2XsSaO1a1DvfmC7J6QAP1OqKosZymMbI0nQklpFtBLMre7VcsRqVtMlb34JkIRTEiCpqVsbFao61i0LGR2vxg0mAyHadDBi1Dbs2morUIaL3ash5l8gvfeYIZACynuEApJ7ungi6TyTVLEI5E9FZUxniF1\\\\\\\/4E4dHDAYArfRLvuVB64vtM0WDkvCgqblBJvPX\\\\\\\/9S8AWuAp4G1IYvT7\\\\\\\/j5Qo81RRoVT3tFS5rgfXF9\\\\\\\/jMG4MZSeQQxpZB3Kqjb2NEzI0HIYQ3ksc7+ey1WC40ndi0yptnxcII\\\\\\\/211KZq3YVhI9pfRMDK1fRRyWF30paaNCLLyIqRizrtd26S3ELgnTmF2G+2mAk3oJAjBVXMHMfy5+INMNq588xWacfMhBQZojexLITy3bc\\\\\\\/uzUbcoWY8UMGGknYXPSsXDeTUZDE0p+RmHWqWeRY01kIcvmz\\\\\\\/mbXyIU\\\\\\\/voHtUh9S6DbFGQa5oOkLOm98jFLxUjcOnE\\\\\\\/VZfOml6kgOXAUN+HGx3k5ealprGqFq1WCpWhXly1nSiUXSYM4JsktLe0QhkDnC5c4YyN6cxcBwyvAp7RIWYKPGTkJZTa8meIH41d2hfW33rxWVGyoX4WTtgf11u\\\\\\\/kbH\\\\\\\/EX9HapMtTee6+A4swI9Jd6muYFmgvwJw0fd05s+GiJbPbHCQXUb+QKXolsV4pITfX06oQ9hUEzN25qUpSd18O7E1ndnWPBfmYdWlUnX5Gbau7qBvL91\\\\\\\/dM62Qwfe4xOsFhQrc7mfXxz5iWoblgBgBAb3fmJU7FQd8r+hkIN+GUaBVbx9i0VzRFi7SSLaE3esP7VNZvzNztuTwSkbghSEbPizRN5zjzCARaFFSDQjGQVZ6jXmYp7HQU\\\\\\\/qNFjoJfR1MGKLOMZLcqNQC8iLlxg7WKneUVfMpCgY03EiuKjPQirECRMfWTaXX+Y1lTqO3ZRFjBJVJV\\\\\\\/lWDDw1xl4LdcKe\\\\\\\/8oXiUTzkf4U2o\\\\\\\/L8SnZjLFXndzuztAOnOtPiPXzlFrS57+EeacUbmXC46uleb6OPUt\\\\\\\/jytw6U7AGR6GrBV0OMHEKvqClOAcodpyzWhCrm\\\\\\\/3v+0MoyjYd7Durjl7U1y1iNaSGEzpHMSftah5v46j5K2TobBZycAJm5udkWUq0LaO5\\\\\\\/JmLOEtm\\\\\\\/mCkle6TNC84U7QM\\\\\\\/HLwl+S24h2scG+L8Hk\\\\\\\/2TpzMpZuC4K1h7WOEnByf3FL028lJXWg27FOUt8WC4irxBof8l2JR4eUl\\\\\\\/B9dixd5k9lxK24NW8U1DOguQkEGNH70p1bQ+5ggW68wcSixI45RtlCvV8kW8m3kIJkhvS6Q7lTqC1n32gFWX9WWG9z\\\\\\\/4\\\\\\\/Cr6t2RN8HNcmobQUwhP4VFIOrtKSIiVgZ5FjfVww84XA69pMy\\\\\\\/z7UsD0AkYmRpJio6RpWDvkIOhO3OoLbuIKrcOozFG1mkbSQ073XffM69NbyXHT0yvaD66XeO6v0UKcHqqxI++Aa8M9kLeumMP3dtF3HCSYmwcWGF1rOYjMW2r5wLJ\\\\\\\/RS1qcVsWp4KLc8lUnBYUwlNO58WUG3isqAWIeAsLibyIXIApPCqsl78\\\\\\\/y5wbU3vbjmJx\\\\\\\/Q\\\\\\\/XsEaUN+GYJJaRiJj16orYyB5mkC91l1sNjuEwA9It6hBx3MbIW89Y9QmNkLxWE1cVdGlDzVVl+Vicbb0Xb5rA6bU6B6y70LKwJbxkGI03YL4R3V8penh2Pw6IRKN21I9z8EChEKLL7d7QlJv\\\\\\\/ShGR6IkEAO7Ka1tDnnLM3uFk35zRjlhCj\\\\\\\/hMUAxL4vV2ZpJXoRW7LfA7g8p\\\\\\\/BeVHFRte1y\\\\\\\/C158MgCjwTka7ltH3uD3cvi9VThBEfyrgOBeMkw82sjktrdGNxcvPbqoDEEOIFv5dTpRkW8BSTOeQpTdEMT3rTXH1FxwO8SBKOHRzf2A\\\\\\\/RkPzCpybWo\\\\\\\/ATfGA\\\\\\\/9INuU09wsr1eOslGe9hHpDv9FfOWOCN9aZp8Fb+cs1m\\\\\\\/Vho9d5XJIktPTih62Gl6IkebE4jQvjee3\\\\\\\/UJhirMK7RXDsTr3yu3+BhwTU3FKnnaMQRd2lO7dl2CDnVlOBG1mvU3ajKLA\\\\\\\/HprNGyJmUyNZazAy6fzBgTz6BRXNBtGAnZUUw6rzuVLsUEG+pJSvzO1T1sn00LklMfc48MqTGQGKQAv7X5zYAv7MSn5e5UKRrr+WOVhOWFP44c7x3MuvB9SQ+wXPE7qjZc4q97mtyzLHsQbFhsWoxp00inTKbXYVce2XWECpFhE4xl8n0+R4bpNOkgnnQn0FPnWVhkEDycu+i751ngL8jxE4OXDz7KCrLCZvnf1p2VaajqmsiU1WAGuz28MPUkOU0E1jBSnDIgIRelEgCbTdcWFccPJ7xt2cg5PUf2fkAQxbdGUVA5HSGVc4Sho3BR2lY4+iJmLemTIReVeoR+d2OPD+y849ppi8XvjY69HTjjpD1\\\\\\\/8UsEtoPguMKDsgUNzyyBQDBcVz6dquC7M+3WeKrZ9Pgygl0tvQf6\\\\\\\/+Ync6Gk2w9wtJjTAe2+u1ewhAmy\\\\\\\/oLkaXm7aWrvZYZNa6oinOlY6iMCyPwVDoMUh1Rup9DV1\\\\\\\/kJwqTl+47snilnCsrGSn1yGCF4klV5wm95Vt8\\\\\\\/5gA0s0bzDGuA7PXk8jzYzhXWle9ag0IhJzJCdDBjk2nz2VDxnfZInV3jzMd+5I7eVOhwMKdx3GS2CSjQE54tASaPfMTWyOf0qzUvD1+Msxf3tUD+RImsmhDAZ4eYuI4XEkwDS7g6tiMyVstODqyQ9FiyIHBknKFg5Mh26Yf+ec7botolp0gRaBA7Hr9wMidBkbou1pbAAIrdDkJB0pyzR1UsyaFBeJmyERIcJ09yCTlnda\\\\\\\/yhyOwyBrXeqbmGPsKdAde7LL77b0UiNHZbLcjUFHRk5kODNlgbkBMirbz5m22miYcdKNJ+uL9J5CPXdcfZOVb2Zh4tlAZqc8UuvB7mkusi5mHlwNxtLFlwIvGMReWFrfnnSr7GvMlgYX+IKJXVWlwb9EmXSqLcClbDWb9Ot9AzaH+WHHAEBDPETGNJi6o6dRmfS\\\\\\\/gGfrynJKNtkOrIiREZdQouKfZB5F1phwvWzj0ZVHdQaeWQdC4puVxy1+xfZ+tU+85p08bUG3NW4SDhRIqOpgvyFD\\\\\\\/cIR11zac3nXky8VGyax3JiNzKoSGm01wR4QtRQ9G9rpEKFgAHa24AWaLQxZA\\\\\\\/+T3m8wRmGDAuHQ2ohyOPpx08HJ2icpodptABLKNw0t\\\\\\\/PyxIVhf4kG2mltVu1dC2UOMKnPLZtmHfhz6zg6+OsyTMqmhjsT7Z8IpYjSo2bLzY7+ljNQAnR8UXqM8oV4C0Tl1b8K7fWncAi8\\\\\\\/a7Lj2XVuggRUco0YM\\\\\\\/Zcub2zRZfPAJg1mSkTkAu3z5q+AMZKqlj22Zo7f8OWsK5R4AhXi2mxrPATMYVCqDmq8pXa9zo3DlxeeVsCMyGzZy1MNgVQsJd6M8da5xNMq0BXBS+IDcdEgyxNSA19V7yIcdq6JYeAbz5iajo3IUmc2aEXtp6+wWJh2tZmcrVhk2uOUVEpZq25VmLMaDugX2cPCEbqucu2LcRdzdKQG12M6UlJ3txIu2uOxcqVO2eVn2YSr4thbaCnVQ\\\\\\\/tmgal3l5iwA+3whx4lDnmsfXy2BCnO6aNeHyYdvC6\\\\\\\/dHloI4LMn3Y76mi9PjT3B2Eom9LO6WlaV2I6jL2kpo63kYPt1lKvLPtBoNBEqEQDJciRT+imakSml6QUQ7uV26yV6\\\\\\\/+G2ghK3oTSR1fTYRgnauyqm68Yevu\\\\\\\/OpbPiLfFsq8kE8siNA5yPFlAez\\\\\\\/WWCf4oIMZobhfHdRxv1Q88smD7ljPnfOpu1JkyAbNHjFG0b7FSOzCpeCq07iOVJq1\\\\\\\/umJU+jcyYgiSPTo0hJmXDYfQx1O6MU0sqoB1NK+HMXeX3MH5ZzYYnkc080dCeoIIo8wX7qMSeoTGrmR1+I139UUozGNNW0wmY9AMd7dqordBls2klsNPX5VpuI+OV5ezb\\\\\\\/RAf3Q7VPP56Zdoafseo+zIVeMzBStH5xVm5kYB+fMCqG1irqdkp2RNNVm\\\\\\\/fp1DQoMBcXdYvEVfe2Lgt42h1vQgJum88EUXkeD3Ov\\\\\\\/FTgsoDTt23Y02Win2LSYQg3CHc\\\\\\\/9ScG0cPBRV9iaqe2Qqk11VzTamP6Sl\\\\\\\/aOEii0UCgARFUbzmWm7SuzhM2WagNeunCIxEr6UC7jnpjUag0nVAQLTg\\\\\\\/0sHgy9KgLwyrwCGxMr8UoglRV28yRDvFOKc8weEJ+9Vv6cbkZ2Ed3OaN3LdXeJ5JuIhgXWRyKQTvKzuH8P1mTXeHbfx3i6HO3o043gekA8XzcRoUwTfxLx1p++NM+gHy0nwhbGblH62+9bLUbUYSCkON6kdOtuvm3WgCGy38bt2bv490IoxrKnzR5GFLasJEIqKPoqQvQXzAr+Oly5nMtcxdT3JYqzWV\\\\\\\/6s0pttORQDiRcdtSoI6mulrgubgD6PS\\\\\\\/nAfD+WlRcPZtip054tHUHtPqz\\\\\\\/23LEEvj80y5WG0EQFVG5nZfyFc80hRCUfHgwnvAPuH1eml7o7iUNOIOSdvZGyc5MtmmoSe6fzcOWF21SJ5thWTtAknMPNmRx8aXFxhsyogFy9rz4KBX7Pp5gwVs3GacTq\\\\\\\/oetGc4GN8vj1\\\\\\\/aryeAIODH\\\\\\\/cdzO1tQkffI442zvVStHh24sLNJjhXIWQYFb9kywDr1FL1PWC89m3TRbKy7LZfyYnzLhw6bBic\\\\\\\/wO5Zhk+7FxSPa2NMnLYHYuOijs6tRg2pXadydbKmOs\\\\\\\/dsr578BgcijA9O\\\\\\\/W3b6luRfigEPwSzB3wvG4IfsrO6fKZksDpJYPsftehZOcGpnC75r0IEU2PKgvhkKfOHcYNb+AUlz5VGJeXbNyA5LZC7yPf5fULrU9cfacwHeg8ZRPIr6PaXm3QyaVGcRT\\\\\\\/LTt2VBl2VKVY\\\\\\\/r+9zWSYmvckP+BBRYxyvmRCXN3pAfmQcdm6sHg0dP5fayVM0jhOuv3jcrQCkqSI9bh6hV1f\\\\\\\/uZ+kjJXZ9WbzFOklbSUHiFzFrri8T\\\\\\\/y63MS4mmF6MJptN2wr4QErk9BXd3Np\\\\\\\/NmZgQOX71z0jSFEApHndcigaHNeezvEW4QE2ikLyesJ\\\\\\\/FWBJAqKfANKHiR0ihqqEtjY0vNI3IC09cHjVduVQFeOBVIrgHsx9RYmWOgabsBBqmES5Qx6Tx7VySumI\\\\\\\/+m4cFE0+YEkp04AA0FHY1mtRrJHt836QDoHZxQvWAgeLJQFlTECNva\\\\\\\/MuOKvrv02CrRwkCtVxupG1MxO5VbsuhR0tfMAApsTNMQKBbX31968pArj27qIdTsrAUtMkl6F1oEyQGGPm2jmX6fezwc3OE9ukWa14Jyq7tHjoBShVv57\\\\\\\/Mm9oAVEj015jaKsh9osBySov0iXtRVAR+UCEARyY5TbeZC0fIK6ummBa7teGxOlq\\\\\\\/dBYa5EDtqh+luib+FNfPO7aJvbhZk1A5eI+4RFkBlFKpLrV3FgfvLAeMsuVlLdXwehJerb0fd7VzXRHetDRC90QyVCIvM+K3fY2rM4VCdNFMObblUtafaZD3AxfCnjpGX+iNskSLMXijrxbVzCOC+u+rKzhGj\\\\\\\/sGuLP5cJLU8c4PukLJ6M+hhrMLhJgdTF9ZUZTrB900w1KP4yxVnmlsGizxfpC0Sx6KcEBna+YXEJQhrKb6w+HTodc4b6S768R7O+REXfHivtmb2HL1uCDkvW1RDZnM3hLRBTZktMrZFgGBPEQlMLNf5MV1LMAyTDZyVc79yof146yLGUnLrzcsWO\\\\\\\/1zAwzapHaA8vFXxyhk7LpkCrbRyfaPYQmK9GyZfI30ayAQzk4dRfz0XCbEFXSTIYB+5mgHYNmoORUYAZdrj6sc7myzF17IapWLWbdd\\\\\\\/ZM0lCs6Z9iIWXwMxE2uLDyiitzcHy3h1pEh1x3NmbLAbLUTSHq9DZ6PrCNi2\\\\\\\/a+ZaGzGkUvoKdv4QX4BNuJTYrTfOj5GJYKQZrBthnahvcryfbsW7+rk5ywRLGPGekHx7q0mUIXCYkQ5plh5WcGJgfsNvMrduBHXkyCW8geHUbGJN+lVtJdiPlpJyQDKpmen4iczs5HEHFs2aR\\\\\\\/I8J5YHIsMAMsb1CG5MOWebxV\\\\\\\/du4oBIurqf5XZkHrujL1CknaW15RIJW4KrwJTU6WZhcLlQH8XQAYAqgmWgqK8608qiF6Lfek32tK3q3Jz\\\\\\\/twA8gxme9LOaz5cA3SF\\\\\\\/uHJN0PXuKtx\\\\\\\/l6qs9feoKqhgD5pqYmHyIV9mUOuV+OCGZNWsLXg9cZcSyRJfJ0JIK+9VKstnD0VGPRVIlbT5g+x9Y4n8IaSbMYGg3pEI2JOtkVFO0hHgoJjsjY\\\\\\\/YJa+weIuqOZH3Lepm+Z84lIUZ1T6fY29xbdXerrJ1DITfoq96xB3uaHfPjWM1aTU6ezU4JKqr9p8DLGCQYZWNk+99FaxM\\\\\\\/LHALEp9uKWL19EV79hk\\\\\\\/9r6sVGydIUTEemGT4cdgKZHaQ3rmbc8JsMJimuoc8KM9rKYeC51uOSDbU7U4lnjYMdk852033W1YpBQYxeL+DbYQBnxC5eG7R9yxbTjbaibtJJOYIf7JRUv7sYnIzPMQVABfr9UN1qd+U5JmJoQoG0HlBm5oeqigRLZhoU\\\\\\\/O9TgZ1pWGgT6dmPDKelkT5JchyH3a1h4k0gSBaacNBLoRerVkCdSuusFAQj9CmTH2un+J4m8zZJJq1GRK3bU8yjAcG0D2L722VfvuyabKP2OjGbUnhjnsysZAiIIaXg4z98EqhQlSUCMT9Q\\\\\\\/U3beWR0d6I25arjJVqXXH8EcBuBFmJ0gDzqxIyuXDlEcTRgVoNfhtKExamQO8MQxE0KLfgAQtW\\\\\\\/cAw2yADwJQWkLXQ6hYge5uweJJrxLPhIA0Qbf+5pVeKh4bjh6h\\\\\\\/pF5pkk\\\\\\\/YJaEuH8F6T\\\\\\\/7bMe2\\\\\\\/jM8lb8DZokMOj+RnohUR4obDZso7XLOU23ALVByrFmUEqkB8tvAss6Gd1roQPR60fgGWp0PTSnPUFmAWmkGQhVCqh57GJ9do1J\\\\\\\/bkXqWkX8TqTOe2r5syxuEXxh9JgAE7AES1L35ZhsmAWyiK6xgx0OQ2jI7+dCBJar8pTKOToCdPTSRlZvXWy5p998NF1RjE8Vj7Oijy17k5vRPxUZktZnhHSkKcdl6qt4fEuf0ey69TIq2ubEZSTxhzGZ9q284KaQtQmgqQIT+kqYZn1YmzTvY3az5cCrtL5luIJRp1MNVSlqWUh03iEkW77QpI5M\\\\\\\/vIsKK+68lWOfFjTAAWVKm29s6cp4MdYX4D4A0RaMCXc7xD\\\\\\\/fl\\\\\\\/2soVRoTmydiFwJxuRMRwznVATXewvZCdw3Fcmi\\\\\\\/SQsoNRZJag37dXTmTr+T\\\\\\\/cuGtR4IQIGKiEBI7TJiFq+1D2Cuipc4\\\\\\\/owvJicEXlpORwa7b4G27VG\\\\\\\/U43aG70kpWyXdN5ifkW\\\\\\\/a+lOdTsT6LN+0iNttbOB7oipAZ+w2k2dPBoB9Qdhr61y3Qz5eWfadoMATYBlO6pe+LuHovelZRdXnZ\\\\\\\/khKbn7GGiXDDGe\\\\\\\/CYQnrexqIHGx5wNQq0UM9xRBm83HHcrg\\\\\\\/IjTiGqtpg5WSxaXfmsXxT4lGVGYeCpiSMOlQGNsO78GyHbU6F5R9nffACPMhVgoHsUkN3ygrodahCDHFVpWpfiClC0lX5cYT9Rb9AAFml1P+CRAzvAnbu0+IAqklonalhSwpWliL52Qe0laYkLC3qrKS5DZxzzNSCCbry8FNGfF50oq3h6qSRsxRkmK9P3+DJcu+5cIU8pZXVypJWDo1ImcPaGhGOtcroZSm+ZvhZh1s0PVoaQPES\\\\\\\/4\\\\\\\/TNhEOBd\\\\\\\/5czck1kCZOR+Fi+UuPkgdpk0BZIxgO7CkTgR84dG1tNz72YwYnFX7+WLkGqh2LFBV9GOftv+WiPI5i1lrkSL4WMVNiM6t29773HEQZJatEvVCp0MdDmL0KIkzuYaDBt2KAsZRex\\\\\\\/ZJtJ7ZuZne5l2cbfgtgNVw8U+wGWT97oESWTDI5TebTXq9zSg1qh59dD8DXmUpbnwRxFSdgupKpNODaeRcIm\\\\\\\/2stbfCg7daiDBCDRQv5fyf1WKDQ+8aywMGS\\\\\\\/xbTOtUNQeWd3OljK9XXvqDmXzsjKc9wTbw1RUh3EgznJH56tXIiAKt1bs\\\\\\\/dT0zM4DLyRHiY0+NO0u9ax4WiSmJn2YkWgqUoLAIXLqZoBBbMLlVOatnMUifvcJ4uL0aJ7w7N91gT9YsatJv3Qkqs6pgKliFwiy38mRVoKmtR3qVjkUw1OXmp9Va39D62e6W9mb+UKzOahICzCk6TDAevrsMMAETKCf6pJZQPV6UFpCo3rjM2LL+dPjGx5fBRozRraAiXpl2WT5r7INSynuE0IdmWdLJ0DDYQ9DSXRrVgMCrInCCgOWzECI\\\\\\\/i1NuSNf5DJNSxvpT+Cd5t5dMB6YsXCgD0LQGEwWPFSy8anrq5\\\\\\\/6HaXp5\\\\\\\/\\\\\\\/jhhSOCCP3Z0F4j+Cam3JKyo39IsXEwqgxa8lelYEzTKAcSawLF7YC7sY6XHo7IQ\\\\\\\/aq0CO+b12S8BphFXtJSyIphsQPFhE5PbV7rqZkCVDj3UpwGE1irWdJCzLR\\\\\\\/B0OQnDjuDcdBQ5RNnmwp9utBO5rr50HfBs1eWBePTp6L1B42zkYoorhdRbGU+h8G9Rf66IbptZ1C0GT9WLw63sPxjFVMu3AivUjX9EBPIie36eysWXhgjniyOQJ5jGYMW7MKDM4SoYHdxgiQJ6S+b+DsCjXELEgpQBJ2rreWP\\\\\\\/Ptf4tI\\\\\\\/\\\\\\\/VhEZaYKucdlbIvMmKisN6HH\\\\\\\/09V\\\\\\\/rbNn79Y8ifXvydhLPhrtKmmzWA0VnmQnBc2oBtQbBZabCgItDc0Yk8LGk+dzMiEfYBfWb+kxLDQY3vYsL1bFu8e7uq0vyQHUseqlFu1jd1XHdIPFNV0AQSRHn1jAD0liSmytwNCPcqy0dm7ObQ47mXLnZ9SwaBcgrv87jVEJFVPTsa2+dTIBQOTTkJacV5q7XE5rUbZejOjugi9eukLtO0yVrcy89cp2oiENqf66d1pmM00del90MJiwdGdYcZ8\\\\\\\/\\\\\\\/HyDyFtixiDuqHvBa5bqT1Ho8OUQr12OsU6x1lmsb\\\\\\\/BnTewBKlHCK5qYpgEOY4XvotpUwm+gY+mx+07yyrpj7Hxo\\\\\\\/lBfFaAe0pxyaRvxbsrhNMnA408ILuXiaH\\\\\\\/9SstOABR+s7jIJljFx9eLjZ8xRANv1I9NudWgcs7eGdL\\\\\\\/beb8xxv3qXInen7Edks3+OYTo15wBss5XiK402K70wJClHoS\\\\\\\/60IdcVJpOWsTDMNRT9U2qIFD8LgGhxWfxgIerWPnCcSz5QgKz+fUXPM8GwZRcBhSNWwSj+624Kso7yxtyXnUlWdCU\\\\\\\/6l0yXGLlpYYVzhirSFepW8Eyg+edsXFLQtqaESrmRxoAbUvU22KSBJfJbyPD\\\\\\\/kKFMqNSl9PyN2uB7n40NL8JH1KU46VMnBjDR3t1jLuH9POBrpGI\\\\\\\/6s3HFG0crzNug5YdHoGhC0bHU8qWEY4bklL8rTI\\\\\\\/POM+0QKwTxUx7lj+2M1XwZEXoN0NiXsV7fnLuNYSdJHb9C0omE5ou25dHk5aYVdC3gkS\\\\\\\/EgGuTsjyF2c60krGLj8G0a7gwYnnKk0WIQEmbAz62B81\\\\\\\/R0HUloxLRuZjZO0Cw+NhG7LpJi+h8AbxLLRmsyGr+PmyImmtjHwbwHZB7eYbzxKGySrZije2IjD44zAqymw5bMe\\\\\\\/j1Q9mZpIaFWcdOS3CdauCppghPEbWg\\\\\\\/UztxL6gNGEduDaq3k0F\\\\\\\/JChFdoFmrDIs\\\\\\\/b4Vuw+fLswioBP6y+DI857TegJcdxfCbSwbbq80DpMnfV3XI7eW7hHV+U2qTvY9t4YqiP6\\\\\\\/zFr6n0tNuLsZkz0Lage90iXPUMpcXsvvRbHc7J3brv9G5xJ\\\\\\\/RoBjZncnvJgnYewQZ4kHL0pxMWjSQjFis0hkcekl\\\\\\\/Nogd2xeG3kVfFptbwaeVxqTNLndG\\\\\\\/uT6LCDXGlT6yEoou+VgwQhi0pUxAUfekESky8EprD4rjfq\\\\\\\/qrcJnikERNuIBpgr2rfiljo0nzLKDUonEJ5SCr3uv\\\\\\\/y0y4jxiFbai9UsPkNtuyicQXMsWTcAqSjH9Wx9jP3faC\\\\\\\/LZSeBbRnd2hoTCYvPQXLBQtJonGWfkGVu\\\\\\\/3Q3mcuAcDEMrrRuGntccovItC4lfrQwTD5ToctgulcqH6G7YD2dZxYaGoPMw9rOVBRKdq8o4Qfdm1aL6sLOPv6LYkAzVcpFnJmbaNnvzkgJXzEm8\\\\\\\/RacVEBsLaKbHIeblJwWJBLvBR\\\\\\\/BUykWutDMmaa0ZVaWr3CNMHKEuqoN0oglO9SePLjb9nqnYi3CcgY0QIkLP3YQb7H467LIA4mCJ1rDB7tWAFLEdUcJvk7VtdgcTc41UH4mSs4+FObUvcm3k4N9Uncj5L3jXGonUOOHQHr\\\\\\\/2k6fgSwzzDM7eZyyPqkD72kvMJ7tCSXkNbnURV+VoxfizPclg1OodDHnnhqV2XD613p5N1djaQR1JOBStUKYO4T89kY29pmnzqJjUiUkjbXn\\\\\\\/qDg\\\\\\\/IHI8x5P5VWb4hzTcHe2ltaFL0UOJ9xskdvBDSN5eR511EhHLvvHJDPCEw8VwBt8S5l73SFO7UqU9CFXD+tCO9kqfGS34wHF5u6cbweN5\\\\\\\/2peCqOk4ZKpnib\\\\\\\/9GmP\\\\\\\/RpRYpuFBshXVkiEPOPmKyDM203pi4NFDd4hZdlU5G93zoEsft1IxK5QhBU8YJJk3gBW2YTdlt8daocJsDyTKC10dU1n8RXbQMRu9G8Yj3sZal905nPI8ZUw+LBrztOtpGbcDiKuZLW9WZtyu6XTwU70gDrevEUvhTQ8M4578OEBSAK+R6KtZtQXpRc\\\\\\\/GGXkhQ9ot\\\\\\\/bhdjiDZHFGLp6gx8dqVBrvl0oyLiapFeZAVIieiA8bwnr4WWn4pg8WuwQVPcFYNLSHrudnUJ2RlHFx56jB18XNqKJG7w5D7n4AvNU8VktVtuxN19MfSULWpVZsUXmpIX1UhaE0vkKNU19izjYZl2BfVYUtmUkacMoXHpfd04jh9fIFetBl+ci5vQb6v0RNkezdrZtJmpB\\\\\\\/xsiPXLTIroXoV5ZQ3FdGB3B+vML3NhAm7tUW6cTTyOLbMajLR6oAVIkGVcSZONGY5ETMrPq\\\\\\\/oVyVcGmig4CbaCFkwn0e8fOYruZ4XSaEAigrbMkcSRhzdyjhLWXhf4lhG9IZHxPYF6zRPbKE32ztlc7EcOWYLWruGRdj1UcAn99Z3989G8GO92PxuTeUucT7UC4YaYG74pZIVF2PsXHIKiS5jRAIcWl0Et2XL7+YNHkC3oRWImCVxA70qSIJavlnNIKzZDoklU5H8yStHTwBxV4NAcncdHDBGl3hX5IPOGv2T8VSLnciG9Jrp7LGNXxhmj1beWrUUHa6oYlJNVd+GYgyAB63ikm6ERYREj7c+U+OxzZUx75XpswANYbuPCSst7NSHpOZr2NRQPqGktRRb3z7hwfOiNSA38N9mO0MOyNtNVpZJPcu8MNSv\\\\\\\/mjAVzYuj8C9IMWGUaartJjSZ9ligXG\\\\\\\/iFbp3zk0OUsBjMxiq4CoefIgMn5n15XznK3mJX3tO9Jepsqt47YbNSxMbDJKgu9+PQ1S2cUMlSZ4EQFx52gz0CrKtMugBb73h2uxW8Oznds9+ak30aNzdjedF8HSXXCcZxjrfKqk0uUhqkY2ZoDqqmVi8nunGPYq9DAVq4hh3SVJyAqFi4fG\\\\\\\/w1lG4T9+1f52y6ZTKt48St1poRdup8v\\\\\\\/1j6ciyDVaKtcJjILS3HifjnwJODP+XBymetXNKLsUZiwg3zp8zVDsLNuWU6XzE8ZtaDxIPi0ZvjPHQuvHHIh8aUM+DHGWKpBesQt4L1uFutBUWkr\\\\\\\/vVaPb8Lc1sr2EHGk5wFGvjuXf\\\\\\\/8aaB4cXWou2r\\\\\\\/OrtatkEczVA2R4j\\\\\\\/GULRvIdAMfPnOzgxBQs7c9zSlIApxbToRGc5BgcTOlgTDSEhy5ek2FUlurjHvSwMERTr1UvCtb+MowM944mCEsmNCqI7DEatEFlc8b95lLHkyOr6mO2SRanr21daMnI+hXqZH3N0J29MHAszzD4RGAwKn4+01qa7k8Z\\\\\\\/sG\\\\\\\/MFBXvqocVTPUFispxlyarZh\\\\\\\/ddDnxwebPwp5iJj69wWYmIu84ROkpdi3dX1ZVnZ6Sqnoa3HWK2KDSqfy0vCYiaLuzhmxkWk5YYSskAW+C8xNN9zxld5SeyCXIR5Auk3v3G1wzwvn8m\\\\\\\/SQ1GtzU6aB97kIcet2XCGn0DG4LGf3e4hsGh7cRnSSoE2DLxXHqK08gezYt1MhKQrMpYf95KZolzC+BtRBmoERS45HMHMGbOz+ANdu72061+3\\\\\\\/X8h\\\\\\\/pbqeW60s6P2SFpkVZW1xE8KOYGp7H1ZqNL6dBKvKCAmxLad7Gf6Z46bmEdBOqtLGHOKtNbyJ\\\\\\\/uUriqgdxRWs28GxMQkS8SppcEbI33sDWs13YoLGt7mq6qLEys9\\\\\\\/6LXkGZWZN\\\\\\\/CEVu2SpS1EYWNyjz7fK7gyJemArH9AxX27ZiCGwSs5dWiao7BsZcHpTwyiwH+wnk3Po4xO3YLquMup6PnDWETtBE7v1\\\\\\\/nLHIwZE9sIqQT8pux7O8XtY9aes\\\\\\\/OlqoeLKy\\\\\\\/iFxsXjZ2yxMx+W1mhKiv3JeKfB2mN6RSjiMcxDSEiauMLUDiRoHkw0Fut+4KZsDd02Xg5QjeFn7ezBWqbJB+jNx50j89IJ5PV0cZhmJqh2raqmBRCpK2ycIxn5zMSQI4kJGwcVfeWagRR67mYEjFcHOPm0V+y+ApJG5qj5ixZVVKJFpybcr10odOPTlRGFwgpF3MUR0AFTktg6ZUiPFPJmvOzTXyzFy5EMzPoJSVXTTb4mll1kfM1i4XYfFZwZXHb2bmcfyLeqo7EgFfBIjy5UjUAhc3Ua9Xh\\\\\\\/\\\\\\\/lihn6GyicFb0j9H2RcpRQVlPxaD\\\\\\\/5shXGSLPeix4KOkQqmK7hlyfGqO7rgvSHGaLLmGjXw3HcFzb3QlS1llq2xds\\\\\\\/4WXJKGOE\\\\\\\/L1FFP1MEUe3BHwUIUqiWu40w04+GrHjUCBhwaywBIBpDJ9\\\\\\\/VwgRUdolTLlBW7r1Pk7NcxQaJML3D+hj2r5AjMGUCSfyc1UIYk6rPo8BbNyl\\\\\\\/keb3Keq7niLThS65IiKPqS2CrouSY325\\\\\\\/bsRkLqpd+EucCtNU3u9+2ZDmJKIZUSXOHCaNolJaNByjY94cWNfSp7mAmPxfp4PI526\\\\\\\/P7xa9UesXIAKZh3v4ijZf7mg7XDOie9YaIxNA9qZ3kUHHrIlKgcCa6YHFQEzGyA4OeviXFooDR4dyGcCqMKi4PZbNc8OCjQs4tOhhML8Vs8qcMGC2GeEFE3JLxRmiGxdklg5kOeFDwKCB2kpTJUvzAERj+pRTx5vRHPMJzBNY2XKKEqXVNwFvTOOtCP16\\\\\\\/Oh2c3yW5MzFY14qH3VJryGxb9Ki8xHFwOFZzzeQZ5YNzy1MilaVNM25FUr40EisDhQ10ZDBjVhInVZb0bwUn3X3J1KMvEFr2RhDc0RcOTBV25T7nR\\\\\\\/ORDdnmIHukxOT4fCmC7fLhFq+lc6bIh6GYhDcBRAPq4gz+2FTmQlnemLnCJ7252niUAIyKB3HblMJpqTLqoPme2vcwxuN+BErh9PnpTkUgYQrX5qCYBLOCyawyvL\\\\\\\/KPmSFsepDUVWlg6eky\\\\\\\/hSxDr2whiCSXMQkDO2MAoSvxPg+s7bqoSeYdZp9VyTJJU87UPnQv03WnAgVO9wsYJ6quiJ8wbNCM1eDO8hv9WedezA\\\\\\\/ArsMVKIxZnB3D8SmGMqjLf\\\\\\\/7Lgt+3JU6\\\\\\\/Vc+rabuG9LXBD\\\\\\\/XdIyJXz5Cu7qZqcUjCcUh14nUcBu7ifTuo23KcKQf3ANvXMmxtdkay4XdcG+NL9XsiKnNzwQGfvLB6sEOB\\\\\\\/z4ORDjSC6V3rOZA173hzLLpTTrZF0\\\\\\\/F4tSRpxcDbDZzJ9U++IB4Xr38Wp85sgEvr27p75pT9YUY\\\\\\\/nxSr2tn+vaXgk7cElyu\\\\\\\/dEOvia3qZPa0AyokvRzCjkHWYGBGDowuACwTdCarVCoBvyYJi5BIlvtiqdjBuFhuinOCsb5TuLaJPrSyguLoxSsfwRE3fDUToH3E3HfSgRtj8xa349cv9CimhsHPkaaDiWqTK+VN3p0qtGpmYeLecSJf93E2FiPhdUXHCy8sdH3UWq9muOhDSR9f5j03kDwlLLY1AsD2V2bjaKoMDWSKZCHQGpSs+2VhsB4w9SR0q8AZLDrhocqw+oWBa+2jOGmRzTJnTYwNx3rTPRUvl9tAKz3RIVLpDq\\\\\\\/rQV9GXbr4l\\\\\\\/NZit+R8rYSwgYHBC1ZwUArO+wvP3n9cR2J7kQWwPm7ewXPuYDyemfx9mkM4cybPq3sjfmDVA9sHCfTy3D+USbLJAhE283oidiv+tWIs5xHTQiqpV2vO\\\\\\\/\\\\\\\/Z4h3O8rRfGeyRzLkYq3uk6HjfkbwuIEi9Ud3nyq349tQtFj0DQso7Gn1T3KTcSnATL00U6ZQ3JOowpG\\\\\\\/UXWoW44cb\\\\\\\/d9b6GrS6fQxx3EAvnzNvWLb4S63oQdKO9\\\\\\\/4lrJKR4wgAJb6AQaYZwZ0A6l4kQiDcfecUskaVhf5D9BHcRe3HAXhkE1eyecMdmlaIl147QpoJNgEZdiFF\\\\\\\/Ut3sG29iW1eS6Lt0glXRRhV1EKUjv9+M4JRzBhTTK23OIZJECOJqFG6A7YXZIvzSwuw4+fZB4W\\\\\\\/naisKrGF\\\\\\\/qtVj+OgtCzQCcg3azOMLVnCl9fq2DPWaYrYSkID4YlbqSKvvoxQ9\\\\\\\/GWoOc88PQ0ARfAqsBiVjhPsBzy2g4oMr\\\\\\\/9FbAw\\\\\\\/56zGQ29mvSfebYUZmFwXbHb3eokUBFFOwzm2CudnmOkjAgTfL+A94DOlQubH65VEy8xy+np0ihZwzGzgHmRQyITqPVdhFFk18HNRmGWtdvmf\\\\\\\/uaqclcLpU62yBIf3sNXv9PCt+SozuFoWad1Vw\\\\\\\/Ii1UCy1o7k4iFI3nJl4LYyPJmw\\\\\\\/sMb7v\\\\\\\/hKLuh7T7mO1ln2P2xGSRAknT7Rql1JdjECLCVi\\\\\\\/urDC5Nwn53DUD0DIWiTqZFmuIZIymV1XJcdGUnTzKUQOKuTmIbg8988ceJZxsZx7akE5nhclduh5tjVTp33uPaJWcwwCFyVsAjJakwaBeF5Wp1pQ0uoArhYjASj9y\\\\\\\/HHoZYMcs7ezmyxy+z\\\\\\\/ZxZJyalxxDtB+jb6y1HZrmETJCxQPG\\\\\\\/hS4VHvELdIW6XWu+JpyefZQrtQVR0fNcJZpJgCt6wxeDqzZGLrFk0zricqnIKsHyMwk3WZQaZDF2u8iQ\\\\\\\/ywC7zbxzJ4\\\\\\\/U7yJSHYIc\\\\\\\/yWjmpm9tXZUzzwUlJMlYTpQMUUNeZKkhhoFFgU4S8eACZWdm74Dbv+Hzc+YRg28Q3fjAbjX3XWVhU8vF6qoHu9zrcrLjVg+WNya92TYK6oZn7XwNY96ZgvFzFyFSentTsec6q\\\\\\\/eFPiwvq5MNrfv6HqzLiD4f+qoBjjEYxq0SWF6oZWBYXTbt8EfDffNjogZEK5Sj13Z\\\\\\\/Zw8uj0Kvt6YhYdXTOIxjB5o\\\\\\\/KWh86C3Du10MbI05BE1vVL5QnZku5O\\\\\\\/834Rwru8f084akM\\\\\\\/RfuM+plDPZ5g7RrG+ycNd1Q1zwVWpn1BTy6g7avPG8ngW423hbULdCJ5K64QbvgTcwtYsknLD9IDqgr0EvPHt017CLc13ZxdKbqAkw9dGugd8QyUdzcOJLsxFfig6QpAseeHfM7h3D+0Eue9rJF2\\\\\\\/wpp81boiHWrucVWtu+cu+ThUT8sthvJ1MvKfjRU2F7KB7+aBRSkChvZmDQ9g8da72tyEm+UdNibEVzE5jrCqT\\\\\\\/m6qXVaM760cMQg08dEvVpk8ucKhZjxj37X+BNWZJZMKigWM94CMmZiFUk6ZG+r0oCcnfDzECXjG1p37EAYgIqboKdQ7gosSO\\\\\\\/sHKez3zeci\\\\\\\/MQej80jX\\\\\\\/+Hf0HXyqgbVWvo9cjqYlmSC7b\\\\\\\/ao+HB3Xa7NZLTMybxrn05sAfzngWbN9R61X7whaupvJbr+xnsX\\\\\\\/Q670KU\\\\\\\/JUvFfdn98U+UtDQe6QONnfcf2nB432GImF4E1MM\\\\\\\/cJeoptvb4bp05JO3ygLVaAJkwmUNeos7sQvdjubAuc9qB2v\\\\\\\/YSNp0jVEldZlIwVbpUvZNiA00ThfzbQGLm0r1TpITL3QPODjJT1MMRcz8Emigbw72xGWjz7Zb9FvT3rVmZJcNUXNBoYICfqaCYMWgm0JwY+rbKn\\\\\\\/3VRwfbkbcbaKD+rCcacYozJOkchggj4L0ez3G3puU0Uku70n+ncf+4+xeAjoLUfzFki\\\\\\\/1lkXKanxwSd2BgC1Q4bxKRkm048NiwghSGsKRKY5SHtmkucvEoXzMo2IFcaXE\\\\\\\/lOeCrHkGN38AVLwh0v\\\\\\\/Di\\\\\\\/pZxE\\\\\\\/E5KAewVYcFIQvOGwAWHcBn88GcEmSi\\\\\\\/w79G58fkbYLJsxT0dzQ6zTwypkmqSDYNIhJVsWTkfiXOkF7KjPwpgriPQNYL6tpL86fmfThjRGITjMC0ytuT0TTwJkX9cAggsaiI4r5+cruDM6hP9mkY3IppJ8SoIAzmVw7gHxJa6SxinpmlzKv+Eay4Z5EfHxsCzI+oG5CviIQZ6BXiY50+vCcdGi9ENhl7pU0MunnaGLwxL3U50w3GjnmYPdU9j8e0QswWegguZWU\\\\\\\/WZ\\\\\\\/agG5jqZUz6r8sua6yWDrpBFs1oJg3TlCnBxJMqWhU6sywSWiZLCZS4itZwwZOZDcZOPLC59R0dGZZt0GeMOYs\\\\\\\/6+0ZwGoKn3kOdZYxdl5kclCFyzftjgL2g4JLZRHMwelpBN5XH7Kb49\\\\\\\/z\\\\\\\/7I3QDIDgLvnudgldU0meP5MnX7LYddYdm\\\\\\\/C0rrBSWp9CzwIImzCwmj3RlZldJTfBLtIOm1mQdyvmKBNelmzKnnC0OkttxYkyzRolluwBpkepS12t+8xHqXm3u6IZIgvWQtigZUhqyC19o\\\\\\\/FzNaUidYjl7XoS7RT6roBevJ6BiKzZdOiDya9lh9ZoOIInbKME9Ow+uZu39zlhV76dC4vVJGlCTztUDj1YJayJPk7SEEcv\\\\\\\/zr0inkY0ul47OIpWVcbE+0xSRYmwQkUwo4eV99kY8\\\\\\\/KpKZpmLZod8u3nplWrV69ntRvSbT21JF1titkmyuKjSDUxd9Lw1rwlCQMxqP6OByRDvw66n2YZWS1RTrcX\\\\\\\/ynBGrzqxvKmp7x21+aWBbuI0VBFFYSfsKhEcZRRfm5dhHRHx9udejI6XPAE5S49mRDoJiRBhjU0k1Eelj7rpbuHfkNWwtTA05BAQ+57x1eAAKywScOIWUS4MXqebx4aN0fyj3PMJie3CLiPSstojnHzYX8k4w8qv5WlACGCWCq0VNnH7KjqfbTdrwOv1CgdhvA4sq37\\\\\\\/EzbGRs5QGrtPnxzhkrR5y3FVlwrdzLHSbliZbtM1MLsUY2US\\\\\\\/oL5H\\\\\\\/f+N+6nwO1oQCkbftyAVrx9DP5iJ1wOERsUKMI0uGZ+P++1gtpnGou0w3neT309f8fBjwNnGSEnsFJwkpSHrYsy4G\\\\\\\/HHcmbjReqScCjYdZWhRXNB\\\\\\\/xk0\\\\\\\/Ru0WLeoQkXg37md3vZ6AwuldBtPYMdJfVnZyBPK4U3nLLB5rowt9WzQvqrLsf3Ey5DQECz+6aPSOumZdE8bXcPVe6\\\\\\\/lDuPtXz\\\\\\\/5uU4kP5venr0Upr+ciPiCsqqX6wzw\\\\\\\/8mxYmKvsrVnGmweGxBb\\\\\\\/8tflERlnRYWxedakOnVTyE95DVRC6OfIxodrV9spbEGXwxdQzqnGus7ny5Lnhd03N2OCfEdKT\\\\\\\/YU0M+3DKDdANXNaryUhnmxG8DdIvVA\\\\\\\/0\\\\\\\/0KfKR9TeC7NIsEdTK0QBQi90T+PiB69eiynnO7\\\\\\\/T7MnpDl9Y9YD8CkNDGGQfehB6nfBZtAfV1ajxhhvjLCy2iHD0z3eh6fiToRmOzC+0vxfdMXq\\\\\\\/vq6hfpyVFA578rBMDZolJTj+Q4pdeJSgzqLhFPPM8sVeW\\\\\\\/Jrjo1s22+vw\\\\\\\/Xssz9qzwTsuBuADa\\\\\\\/+oVOjb1JGkgolWmcrgFkwiWd8wLsG2QHKljThvoLdNDKP2Xygn89K20wpkIVj\\\\\\\/tbpx7+JviOk4rS5uuyhk187vd49urJGn+KXNo7CWJljxfJL09M5t+z\\\\\\\/Dulzfk275HVOwYHTiEgb7wO\\\\\\\/pCbkgF2vTUHDnQlVx06uTNnzHfsMjsxRYwSiJOgAFa3SWfvF0Xlx\\\\\\\/Kj6uFGFpYvlfobq6MEvHTzVk3OENjz6tqBL2h81hocvALF43sN55bn3poBmw66v9BQlXGT68c23fjWfe3wUmQfSeEgU7mgxAbLKlke40p0kt4aMe+VoIJasfGl0Qv35bKcoZLYA8667mW3J6Mc3OoPoIDPtMZEWaBepnu20vrjzCzFuZSSt2iAg0oxUfODHyGt9qkmbxOHgzkJRkn\\\\\\\/Qf42b\\\\\\\/B2q3Wx5r0dC0apVEXToO8MZqennVqA47oakwAdsmbaxkTx4CM8XBzdeeRHvnswOmzl4z12Nq+Occb1I6Y4hA82wBN9ofk1\\\\\\\/K2OdGvTgHzyjvS5ANKji7IFElxCLjSRBRqB2kOzrFGBes1z86EJCl8NUgdSheCkrjEHeTdMM9a45CSyPMuYtsNdViCiUfjJM8ntIwoEGinrR2R92yyN2YV8Z3FXwxk3QZfN3oVnz+S6PguO7ZMFkf3fEFEu3i0\\\\\\\/P+hJyvicu1SA+jq43+noRSO9XCXHdtlR6gTWvsyXcQhscqi\\\\\\\/Oz80DlAqZmsvJluAbXqXpPb229gJQlMtNPUxKhbIV+HxrPYkVZzkYaH5YpP01HJt0\\\\\\\/zc9ZpGgckJS98XGoNkGDLzfxtqrVMQOdNA+G71VpHdZbFyx7k5dS++DlxjBChEufcmu7EYzDFzk97VlBrSWH4afG3DgFF\\\\\\\/heiBweouJI8a5Tk6+8GVOQMXGBluxFmAGRZvrAiHLljImtxp2OGIHAcsnYzEpNGDRPHx38ljGzvnbBFZIF78AGscTOcC+ModdFH3OOusJ2UbWa7uURW67iZNC3aBOLu6nYb2LBzz6dR6fRVJQDhrKcszBfDID4VNXz\\\\\\\/7Q0Yj\\\\\\\/87J8G99Ms4Wz8ZdGLGIofewsfGB5SGON\\\\\\\/q2vPuyOu80CyihKXt0P5eEJbd7Mu6WbhrFOWPijvZRDkMjfDG1UfwX59QeEwJsEpvEzVY8gyHSsjnsQVCugnmTxP+Rkh0Px37ZVBSTyx1l6mrxf9eb7+P\\\\\\\/\\\\\\\/NJLIvQRo3CWJwwq8roinzhRvo6NOWLEtoDJORjGf2Xew2arArRCr5rU63V\\\\\\\/O2NZU7lM2NEmId7Ah8ptbcE4Ij0cOa4qzRoKzuVA0t7ukEjYq+wEE5YJo+wb88WMQEZ4kuurbJBWwOfWenlOXv61K9U2LQLUG5YuS38Drhq3R2D0IP5oH6JNeNoY\\\\\\\/c8x+sDsH8TxMT4NWEjUY6HhXs+gila1P55EHGvxZcHo6gulOXkseYJMdcIHb0HadQjJATIrIb+UiWotT3\\\\\\\/vOCf+3ZEobaaQFr0JXViXmthHvWgKj7iEwtHjZYVfffDwynspv33WEWN8N4qUk6r13nGZjuGdZ8Pirm3wRBPWr0FgVjhMDaJ3nu7l\\\\\\\/KXYBRzt+4Zk2jJW7j7YnyolUe1iCnG+My2VvBowQhSNRg2qsm6a7EH\\\\\\\/nwP3FS9nwPalnthISnCuVGMoKh3TeTJTF7IRdL0ceaKielrZmZXZkYHHBVn8aJI1LXtnm0722akLMa1ghsu66oKOj0zHM91v8z8eYymultiI3xrqdz1n\\\\\\\/Xxx2kZZvvn3InLlvT2jBTcvyK8EYm8BCBcNcPjt\\\\\\\/AZC3UTjtigyFKkthp96ol0WrLA2fYWt2sxH3CBZvoaRQHK1GZ+FO8Jv\\\\\\\/jM5vYMMZLz1uj5mOL+DMtX7TgtAo9aatrHo+esrk1qEcF5xOihqCeZ9fkko2WbWGaPairSmh4Y\\\\\\\/KmZEYgXfZpthldqQhIQFpSRzHNjPxObiyapUsxj17ANkN8yi8H1DT0O71LhSdLc0ETEAEAexxFOjUfNR1MiKwL1gIxbnVKc40C5Ttz2ki5uW8EutFl0s43Y5DfW2bxo3cCo2fI3P8cV315jHqWr1xZp7uoMcXnFLO5BGrR7fd8THNLeJdsSij5pN5Y\\\\\\\/aEcNBfDE\\\\\\\/l4\\\\\\\/3sJSqKVbMgAOZdGncg9fobXVBck8BEzMPufiWi7pxlEJilBplYekoiQ80X8PHUyU6ZIzXVwOXmNFBG7oKV++pzX9X80JiireFGA9UwfZJxhQDHkrXrmWSrXmwm0Wdzg9zyNlDVSitC9IULRx2p4wVphh16pVjqUwRyr2zsrA+uWMEbynPgo4EO0fMDA7z2x2Po10+V6EE7h6mGg55HyRVm1Af9eGAuVCnNgPpmagbBbsJrNRYfZXm8jYW5CLW\\\\\\\/IaGR3fKrHGYUy+maIO8OFAuBzkRLYXIWLxmOZB4Fn5A4HOMZ6dlnWXOaaBR8f+znddUQpL2oHCnetWixitUqfhMo\\\\\\\/4xzcevJ4JaZeDw8w5DMrEkdt345aINC6l5YAiC\\\\\\\/asa1Mgqg+O2SEmYRg3uq35SUv89ITAKdqwSajDx5ujtgVqO4C7qUJWpEOekHaa7lTOwURGiqnGD4Ca\\\\\\\/4YgCr1XiUpyuRK2vYutoMTMP40vi8oKHCu4L+y+E13zDvpty3HxsEgF8b8L\\\\\\\/hkS4U2jFUE+gLMa6zwVoTge+mu\\\\\\\/4YffnUYFS1XK5JjZQREf6sperMIaZdfTs+UAMABg+cNWt3GeJWpzoE7yprj3E0JZMeJN\\\\\\\/U+UDng5FQKpbWnrzo43JN94GfDhi\\\\\\\/zdDNXO+uZqWua4m9ONf2NqSUYLGc7ldH4LKsH4gQkPrjNG+i7ZSOzCtnvbHx+7OVypmmc\\\\\\\/kLRY56hvtYQTsk2N7t2sKFck1vKnxECO\\\\\\\/tLgYgzc7RwSatqbtzQBj51ThxmQ1\\\\\\\/CKzQC68VRy6KqVlHDX9MvCeEDBjWyXXoT5ifj0i\\\\\\\/JYZFJccKNDsIwUxrdI+2zUZL1iXb2Ir26FLq0TuemvosR5e6VaW9IzrrCim6SW9YtlgaOPfF7wD5L49UtCTZT5wDRpnpya30BpZe43VujdAJrace3XNbjK0XA5FKW7ewZZlbQHU8\\\\\\\/k+X4\\\\\\\/hjtg7TD5Kbs6hHgB19d+y\\\\\\\/oNIpNrNL1SeMWKtauFB4ynyCDSFSevCyF1ZvCP\\\\\\\/P1trhINB0h9wLFae0XHAozqW9EaP4+g3EwswsXCUCJgb+n\\\\\\\/nYIp9sP+PRuJBWfzNGbquggolypyVED2yPsjPqZHPyDCZv2q7Riu7RiaBoJlnlvTE6OUpizQbSNLuN+TdK\\\\\\\/443DuegWo6mofKlhR91VUO0JWdgrsinVLjzHFUS2LN3NpLPZameInxDHc2uO339NFQBU42QXCfWanhJ7wkN2BHrSx\\\\\\\/E+Buvh8b9njTjqhOeoJRNQZ2NOoCM6R77sEv\\\\\\\/LKcTL2S63\\\\\\\/51EklvDWicXNkromSaiKV\\\\\\\/+wuzakrjIiFhaH7CLMzr3ky7aoBDpjlsSqEwSa18GRC2Q92eDhMocerAYwJMnJ6pf084jDQshbfdf7h\\\\\\\/hUXbMYSceRZNtTQuTmaHg654Rm3dG0QAGHzmmsp22ebQUaxzf90HGVx7yNW19UGPk+q4nXCwyFHelJKaEGOT\\\\\\\/uSrUUbtrM\\\\\\\/Xif7rC6SAKVW+742HiSnddzRABLKCEFpZHKrbW+WTKDymIavAyUNIbgmMn01aVrp2XIWTv5JZvlGIQzDy+EcuNTnh5QIo5Pu7js9yCDFTC3JhHc\\\\\\\/FN\\\\\\\/LgKqk\\\\\\\/eqrLymF7kkFZd7F764yjn5ddI3fdhg+H4eAThQhqHOhp1gMKhS87QQHZhAyfigMzk1EH1p0fwurGaSXNqfpFU0C7D0I\\\\\\\/UMCKPAJcUUEPtRFzI016U5QsjsObPUokxmGu0wWO8OPT84rfmSe9sE62yDT\\\\\\\/kcnzGyGD6PHDC\\\\\\\/jPVFTsQXroYLOb\\\\\\\/symHIc6y4uRUwCglWGF7Af2bADIX9aSABFVb3TufKBL7EFCeDxxA+MbQ567tNdRdU5bN3lijawUM\\\\\\\/\\\\\\\/TyFvqmpVsu8tqQ3jGAV2FO5v3jPIppwFuW6GEiWCvB0ZYvMKUAojZpFeMlAC1a3eEfsrcwughfTCQLX+NWzJ4MO9\\\\\\\/+wgleEXPKoeNkhLHQetdfcoym4NzhRoAabhs8DSaFxTmRQPZNigxF5\\\\\\\/VJ+ybZ08pCZoGxBl81gsGJCDPNDCb8bwpQpERQ234QmReeWwc0dvPGtdFZZruQkk\\\\\\\/BdwwdujML55ye3qIziS5iLasfCaEMtihGJr3FZnUbBsUvLkGgx0n0k3LJaloZVEJeNI25p0NeTfIH\\\\\\\/KeFMhSVjx2mqw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022MvBHDG2rktX+svoPh6FPNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220e2a0f1d04bea216ca30c2b939b4f598c7c79a02d766d05dd208b553c370e14c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dOp16c63+U5GeDVhjkj0ei8tW70EQ4EeHyShQgVEyQwOxiaxeAyWKzLQnBPng4gXV3TN5biAU7u2ROthuXfK7Jjzzes67YaLuCl7JloZBiBCn762tkTF5zTE2ImCgj2NprYAXq0nncXp7MFt3SeUpc8fxxVVmK0D2lqnck3N82KmPepJaFz3KTK0GK03F+t7++qnobAuZBwjU7vLRCrJjfvosYdqgELVkG5sbcyhLU6tFMobaz1J16MkjhWGC5f34WS2YFnQTMoK3GyYsJ\\\\\\\/AZYhiFQ2sYPFehPqb04bghhfC+fHc5N1hcT3Yu3GENwiSflahVSKKGLyg+ZYq84XvOyBaeBUlJmTWCRV2nCueO9JGwduqy2aQaa4d0STJA+eRiGnN8Cr+ySncZFIl0ypOYDR9nBDeTe3gNsHd\\\\\\\/LWk+dnfeZuj\\\\\\\/cr0fol0r3OuL51dmjQhnKJ713oYllS47YZBplkZNuUO+kFp37zx7ZetALtfqhfA8mPi3jrvmPixJeb7TIh83C9ehyTky9j9xsmCNsLXUy6vuEf6MKMdvl4O1ECmZ6bxxDpOBs5\\\\\\\/h40DS4sfvlLULizHkwn+2A9QHhalGiyfprgQGE\\\\\\\/4x0pfwCa5y\\\\\\\/rtvjXdWGNFPZmnQR6ZBcUyWMs1x3GQb4+Nbr1KygN22y9+DBtTfCj7+Hi2flI9YT2gktmqvT6EGhYf5m++SlKCJVB5lPu\\\\\\\/yPP6OR2aPH7C\\\\\\\/pS+GzOKjzdLBsoKwn86pxkEw6px\\\\\\\/g0E8if8rXTlbjPB6fyvifjTln9oXdCyzCrdYv4LPQ7aaBHqkXREA3H\\\\\\\/hQ3MOyuYOpgw9eRcwOhmZuLGj1cZByBLwcQ2ssGQc8alE+B8UbGKeMSlfbqqdwUr90bLmw2upeIJ4leH6yBvPgsKF6trBWgLEjNieGwlfnAOtH4FswBWUnwzYLuzOunph95LBJkTnq254dli1mBcIHz6J49c1GIfMwcDnswu4qHG2qphJVh\\\\\\\/c0yM3CTUmrqxQLSHFgonBKnYpkSAkfz1KVVTWJYI\\\\\\\/e9G36zgCQv4apBbBypAVBBQBNZNhyijWqKLpNeHu4i+pkzZEwwIIz3cBNNokBudmeZ1cW9oWKCo\\\\\\\/YhXHD6H1xpVk6Wd3hx8RTyFBHgu37Y5HzYURO62\\\\\\\/97DPlG8Op2RaegEj23OkPB7VlTR9FuSuDD5xDhxYHE7h3WFeBfaNGNlShXLelm03ZNs9MJ8d8v3rpMI33iT9kt78TWyOLYrGyHRU0\\\\\\\/8yBp2CsGbJn1iwNaKrowN0UvlWvgNcL4u4l9tic3YkIuiIMFuCCTHzUlBsmFi3\\\\\\\/21zuTBfsP6hOlXgnSsHfcQwBsRH1u6QYiqxxUNfq6oawzdbqY4V+Go47qRllJGuZ99aH3i2lj3yytJngzvaZcTnUcXxfDbNCiUwxbjHPFyL1dUy2fki5teujdAJ+dk2OLdC1L7JLazanTHqn\\\\\\\/BWTYvN6cc4f6\\\\\\\/hPRzDuSSQolsXjQIfqLUywb+90sNu6s6ouyYc1riki78EeEBfK7Y10kTdjlxivlM\\\\\\\/NYKDun0PrZofioclESvrOioirjwinjALnOYkt8PEeURLvzm2V6qPMWdCyjKZafs3cnmcetA+LYWUMBn83lSMd+pEeV0aqJ2jPpy13fIlWWSBAO\\\\\\\/RhYnM9Y2jlVBPYjz5hnD\\\\\\\/AtbEmDlWHUmBsSHzPrcWiEFY16nZYqBwk+NwvGYMmey8tCeSoRMLJL2PwpdyBZ0KAPM89o8dVpXqb6vgehPfvR5S5gqkWBCZ2\\\\\\\/EaZsiXnZJJh4XyqJCF5Pp\\\\\\\/X9RkNjN3lzEH8paftoLUixtQDdmHyL55ftmnDt2D95gZK4F1umuUr1Zb7ZvNiohw3AgWuK\\\\\\\/3K+Gt3oW2pjej87ioQzgX6kg8pClKaDq4WHN8nvSED6fMW1mZ205XxGopoR\\\\\\\/UmpXgGGDFewzP+zMj2B0KYB3jbwYhqDj8bLgSTi9J2HkvbQ8H7VL92ExWTZ5nHMh5+gwhnDyTjjkuSACH8ROZbl2Iw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vFMC\\\\\\\/rBHjKJCqSGR4sjScw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225e9e43b67b4a2acbfb0765d1339ac00f2454fd700d85f27af89dba0f0f46659a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490980 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3173" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IVk8O37r8v5_a_1STA2YIZoisG1e014pRp7qTeoDeJQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3173" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qz3W5SR6XS7rEABU3MRjdyPQdL91iIubxNGHgEg3wfY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3163" title="Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3185" title="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490980" id="node490980" data-pisa="ggg;8/10/3173" data-pisa-master="ggg;g3.118.200185" data-apath="/ggg/8/10/3173.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6667-7098" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Victor Garcia</span>, <span class="highwire-citation-author" data-delta="1">Stefan Zoller</span> and <span class="highwire-citation-author" data-delta="2">Maria Anisimova</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3173-3183;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200185 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490980" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victor Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Zurich University of Applied Sciences, Wädenswil, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictor%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6667-7098" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Victor Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:victor.garcia_palencia@alumni.ethz.ch" class="" data-icon-position="" data-hide-link-title="0">victor.garcia_palencia@alumni.ethz.ch</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stefan Zoller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>ETH Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BZoller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zoller%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BZoller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maria Anisimova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Zurich University of Applied Sciences, Wädenswil, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BAnisimova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anisimova%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BAnisimova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3173" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490980" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490980/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3173.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490980" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490980/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3173.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490980" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490980/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3173.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490980" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490980/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3173.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490980" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490980/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Experimental evidence shows that synonymous mutations can have important consequences on genetic fitness. Many organisms display <em>codon usage bias</em> (CUB), where synonymous codons that are translated into the same amino acid appear with distinct frequency. Within genomes, CUB is thought to arise from selection for translational efficiency and accuracy, termed the <em>translational efficiency hypothesis</em> (TEH). Indeed, CUB indices correlate with protein expression levels, which is widely interpreted as evidence for translational selection. However, these tests neglect -1 programmed ribosomal frameshifting (-1 PRF), an important translational disruption effect found across all organisms of the tree of life. Genes that contain -1 PRF signals should cost more to express than genes without. Thus, CUB indices that do not consider -1 PRF may overestimate genes’ true adaptation to translational efficiency and accuracy constraints. Here, we first investigate whether -1 PRF signals do indeed carry such translational cost. We then propose two corrections for CUB indices for genes containing -1 PRF signals. We retest the TEH in <em>Saccharomyces cerevisiae</em> under these corrections. We find that the correlation between corrected CUB index and protein expression remains intact for most levels of uniform -1 PRF efficiencies, and tends to increase when these efficiencies decline with protein expression. We conclude that the TEH is strengthened and that -1 PRF events constitute a promising and useful tool to examine the relationships between CUB and selection for translation efficiency and accuracy.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/codon-usage-bias-programmed-ribosomal-frameshifting-translation-efficiency-hypothesis" class="hw-term hw-article-keyword hw-article-keyword-codon-usage-bias-programmed-ribosomal-frameshifting-translation-efficiency-hypothesis-genetics-molecular-biology" rel="nofollow">codon usage bias programmed ribosomal frameshifting translation efficiency hypothesis genetics molecular biology</a></li></ul><p id="p-3">Across an organism’s genome, among those codons that decode for the same amino acids (codon families) some are used preferentially over others (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sharp and Li 1986</a>) (reviews in (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hershberg and Petrov 2008</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Plotkin and Kudla 2011</a>)). The variation of this <em>codon usage bias</em> (CUB) within organisms’ genomes is mainly explained by selection for translation efficiency by protein expression demands –the translation efficiency hypothesis (TEH) (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hershberg and Petrov 2008</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Plotkin and Kudla 2011</a>).</p><p id="p-4">The main support for the TEH stems from two separate lines of evidence that concern associations between CUB and tRNA abundances and CUB and protein expression, respectively (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Plotkin and Kudla 2011</a>). In the first line, studies revealed that the most frequently used codons –so called <em>preferred codons</em> (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Hershberg and Petrov 2008</a>)– typically match the most abundant iso-accepting tRNA in both <em>Escherichia coli</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ikemura 1985</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yamao <em>et al.</em> 1991</a>) and <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Ikemura 1985</a>). Similar conclusions were drawn for a number of additional species (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kanaya <em>et al.</em> 1999</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">2001</a>), but using tRNA gene copy numbers as proxies for tRNA abundances. The assumption here was that tRNA gene copy numbers and tRNA abundances correlate, as is the case in <em>E. coli</em> and <em>yeast</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ikemura, 1981</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">1982</a>). Further studies suggest that employing these more abundant tRNAs in translation provides efficiency (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Tuller <em>et al.</em> 2010</a>) as well as accuracy gains (at least for <em>E. coli</em> (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Stoletzki and Eyre-Walker 2007</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Varenne <em>et al.</em> 1984</a>) and <em>Drosophila melanogaster</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akashi 1994</a>)). Furthermore, exchanging preferred codons in highly expressed genes by non-preferred had strong effects on gene expression (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Carlini and Stephan 2003</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Carlini 2004</a>). If valid, these two mechanisms together imply that the disproportionately frequent usage of such preferred codons in an mRNA sequence is indicative of high translational efficiency.</p><p id="p-5">The second line of evidence concerns the existence of associations between CUB and protein expression (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Jansen <em>et al.</em> 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kliman <em>et al.</em> 2003</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Roth <em>et al.</em> 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Quax <em>et al.</em> 2015</a>). High protein expression demands are assumed to generate a selective pressure for translation efficiency (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Hershberg and Petrov 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chamary <em>et al.</em> 2006</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Duret 2002</a>). Given the mechanisms of the first line of evidence, these efficiency increases could be provided by synonymous mutations that alter codon usage frequencies toward preferred codons (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Hershberg and Petrov, 2008</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2009</a>). Taken together with the first line of evidence, these associations suggest that protein expression is also associated with translation efficiency, completing the support of TEH.</p><p id="p-6">The test of TEH by detection of CUB-protein expression association does therefore not rely on a direct comparison between translation efficiency and protein expression. Instead, CUB indices serve as a measure for adaptation due to translational selection and thus as a proxy for translational efficiency (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Roth <em>et al.</em> 2012</a>). CUB indices are computed from a given mRNA sequence. Thus, the test of the TEH by CUB-protein expression association hinges on the assumption that the CUB index of an analyzed mRNA appropriately reflects its translational efficiency.</p><p id="p-7">The phenomenon of <em>programmed ribosomal frameshifting</em> (PRF) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jacks and Varmus 1985</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Craigen <em>et al.</em> 1985</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Clare <em>et al.</em> 1988</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Caliskan <em>et al.</em> 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dinman, 2006</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2012b</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">2012a</a>) indicates that the validity of that assumption may not always be warranted. Programmed ribosomal frameshifting is a process by which ribosomal translation of an mRNA sequence is induced to stall at specific sites, termed <em>slippery sites</em>, which leads to a rearrangement of the ribosome on the mRNA sequence (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Plant and Dinman 2006</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Caliskan <em>et al.</em> 2015</a>). Translation then proceeds in a new frame that is shifted relative to the original <em>open reading frame</em> (ORF) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Belew and Dinman 2015</a>). This mechanism emerges in all domains of the tree of life (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cobucci-Ponzano <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baranov <em>et al.</em> 2015</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Caliskan <em>et al.</em> 2014</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Dinman, 2012b</a>). Here, we focus on events that shift ribosomes one nucleotide back in the translational direction, termed <em>-1 PRF</em> (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Caliskan <em>et al.</em> 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Advani and Dinman 2016</a>), further restricting ourselves to <em>yeast</em>.</p><p id="p-8">The function of -1 PRF differs across organisms. In viruses, -1 PRF is predominantly employed to package more information into available sequence material. By opening a new reading frame, the -1 PRF mechanism allows for dual coding. This function is exemplified in the <em>gag-pol</em> gene overlap appearing in many retroviruses (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Jacks and Varmus 1985</a>), in particular human immunodeficiency virus (HIV) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brierley and Dos Ramos 2006</a>). In these viruses, the appropriate proportion of <em>gag vs. pol</em> protein expression is regulated by a -1 PRF signal (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Maia <em>et al.</em> 1996</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dulude <em>et al.</em> 2006</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brierley and Dos Ramos 2006</a>). In eukaryotes, -1 PRF is predominantly <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-1.gif"/></noscript></span></span> used for gene expression regulation (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Plant <em>et al.</em> 2004</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Advani and Dinman 2016</a>). About 10% of genes in yeast are hypothesized to contain functional slippery sites (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Belew <em>et al.</em> 2008</a>). Ribosomes redirected to -1 shifted frames majorly encounter <em>premature termination codons</em> (PTC), that is, stop codons that appear well before the poly-A tails of mRNA sequences (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Advani and Dinman 2016</a>). This typically triggers the activation of the nonsense-mediated decay (NMD) pathway (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Chang <em>et al.</em> 2007</a>). The NMD pathway induces the degradation of both the mRNA, as well as of the partly assembled protein, while leaving the ribosome intact (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Advani and Dinman 2016</a>). Thus, -1 PRF acts as an NMD-mediated destabilizing element of mRNA (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Belew <em>et al.</em> 2011</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>). -1 PRF efficiency appears to be controlled by sequence specific elements, such as miRNA (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Belew <em>et al.</em> 2014</a>), but can be uniformly affected across all PRF-harboring genes by mutations as well as drugs (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>).</p><p id="p-9">The reason why CUB may not appropriately reflect translation efficiency for genes containing -1 PRF signals, is because -1 PRF is likely to conceal costs to translation. When a -1 PRF signal redirects a fraction of the mRNA-translating ribosomes to premature termination (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Dinman, 2012b</a>; <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Advani and Dinman 2016</a>), a translation efficiency cost is incurred that is not reflected in the sequence’s CUB index.</p><p id="p-10">In this study, we retest the CUB-protein expression association that lends major support for the TEH while accounting for the -1 PRF phenomenon. We draw from <em>PRFdb</em>, a database of predicted -1 PRF signals in the <em>Saccharomyce cerevisiae</em> genome (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Belew <em>et al.</em> 2008</a>). First, we devise a series of hypotheses for how -1 PRF signals are influenced by evolutionary pressures. In accordance with our hypotheses, we find evidence for the existence of a cost of -1 PRF mechanism maintenance. Second, we devise general corrections for any codon usage bias index of mRNA containing -1 PRF signals. We use sequences from PRFdb to compute the new, corrected codon bias indices, and compare them to uncorrected values. We find that the TEH is robust against these corrections and strengthened under biologically plausible assumptions of a -1 PRF dependency on protein expression. We conclude that the -1 PRF signals offer untapped potential to analyze translation efficiency in mRNA.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Slippery site data</h3><p id="p-11">We obtained a dataset of the <em>Saccharomyces cerevisiae S288C</em> strain genes with predicted -1 PRF events from the Programmed Ribosomal Frameshifting database (PRFdb) of the University of Maryland (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Belew <em>et al.</em> 2008</a>). These putatitve -1 PRF signals were identified using first a filter for slippery site identification, and subsequent detection of mRNA pseudoknots by means of the <em>Nupack</em> algorithm (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Dirks and Pierce 2004</a>). For a slippery site to be found, a confluence of signals in mRNA is required. First, a site must match the pattern (X XXY YYZ), where X is some base, Y is either A or U, and Z is A, C or U. Second, a spacer sequence of a minimum of 8 nucleotides in length needs to exist between the slippery site and the following pseudoknot. Third, a pseudoknot predicted by minimum free energy values of the mRNA secondary structure is expected downstream the slippery site. The dataset also included a full list of gene annotations, accession numbers, the relative position of the slippery sites within the gene in which they were found, as well as the gene mRNA. For each slippery site, we computed were premature stop codons first appear downstream. With this, we computed the frameshifted sequences.</p></div><div id="sec-3" class="subsection"><h3>Protein expression and mRNA level data</h3><p id="p-12">We obtained protein expression as well as mRNA level data from the Supporting Information of (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">von der Haar 2008</a>). Von der Haar has produced an extensive curated data set that merges data from various sources. For protein expression, the data include the seminal studies of Ghaemmaghami <em>et al.</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Ghaemmaghami <em>et al.</em> 2003</a>), Newman <em>et al.</em> (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Newman <em>et al.</em> 2006</a>), and Lu <em>et al.</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lu <em>et al.</em> 2007</a>). Additional data stems from 46 further studies, specified in (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">von der Haar 2008</a>). To ensure comparability, only studies were included in which yeast was grown in rich medium. Transcriptome data for mRNA levels were obtained from (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Holstege <em>et al.</em> 1998</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Arava <em>et al.</em> 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Holland 2002</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Jelinsky and Samson 1999</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Roth <em>et al.</em> 1998</a>). These data include about 6000 genes. The construction of the curated data set is described in detail in (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">von der Haar 2008</a>).</p></div><div id="sec-4" class="subsection"><h3>Indices for codon usage bias</h3><p id="p-13">Many indices have been proposed to assess the degree of codon usage bias present in a gien mRNA sequence. Here, we focus on the codon adaptation index (CAI) (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sharp and Li 1987</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Carbone <em>et al.</em> 2003</a>). In the following, we give the implemented definition of the CAI. Let <em>L</em> be the length of the mRNA sequence in codons, <em>c</em> is the index of the synonymous codons decoding the same amino acid <em>a</em>, and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-2.gif"/></noscript></span></span> is the observed count of the synonymous codon <em>c</em> of amino acid <em>a</em> in the sequence. <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-3.gif"/></noscript></span></span> is the index set of all codons within an amino acid <em>a</em>. <em>A</em> is the index set of all amino acids <em>a</em>. We define<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>as the relative adaptiveness of codon <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-5.gif"/></noscript></span></span> within amino acid <em>a</em> of a given mRNA sequence.</p><p id="p-14">Then, the codon adaptation index is defined as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-6.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-7.gif"/></noscript></span></span> is measured on the observed on the mRNA of interest, and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-8.gif"/></noscript></span></span> is taken from a reference sequence of highly expressed genes. For the reference sequence, we concatenated all of the mRNA of ribosomal genes (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Friberg <em>et al.</em> 2004</a>). The ribosomal genes were taken from the ribosomal gene database <a href="http://ribosome.med.miyazaki-u.ac.jp/">http://ribosome.med.miyazaki-u.ac.jp/</a> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Nakao <em>et al.</em> 2004</a>).</p></div><div id="sec-5" class="subsection"><h3>Data Availability</h3><p id="p-15">The code used for analysis of the data are available on Figshare (doi: 10.6084/m9.figshare.5955886; url: <a href="https://figshare.com/s/ff89f9e614b129469fd4">https://figshare.com/s/ff89f9e614b129469fd4</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.6084/m9.figshare.6970277">https://doi.org/10.6084/m9.figshare.6970277</a>.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>The cost of -1 PRF maintenance</h3><p id="p-16">Effective -1 PRF events classically comprise three constitutive elements. The first element is a heptameric sequence –the <em>slippery site</em>– (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Jacks and Varmus 1985</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Plant <em>et al.</em> 2004</a>). This site has the structure (X XXY YYZ), where X can be any base (A,C,G,U), Y is either A or U, and lastly, Z is either A, C or U (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Plant <em>et al.</em> 2004</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Plant and Dinman 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ketteler 2012</a>). The second element is a <em>spacer sequence</em>, separating the slippery site from the third element (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Dinman, 2012b</a>). The spacer sequence typically comprises around 1-12 nucleotides (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Ketteler 2012</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Dinman, 2012a</a>). The third element is a <em>pseudoknot</em>, an mRNA secondary structure that is thought to generate a mechanical tension on the spacer sequence when the slippery site is being translated (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Ketteler 2012</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Dinman, 2012b</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">a</a>). To release the tension, the ribosome, while stalling at the slippery site, is pushed one nucleotide back (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Dinman, 2012b</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Caliskan <em>et al.</em> 2015</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Ketteler 2012</a>). The frequency with which the ribosome is redirected to the -1 frame is called <em>-1 PRF efficiency</em> (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>; <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Advani and Dinman 2016</a>).</p><p id="p-17">These criteria were used to generate an extensive database of -1 PRF signals, <em>PRFdb</em>, of the yeast genome by algorithmic search (see <em>Materials and Methods</em>). If PRFdb comprises a sufficiently large set of functional slippery sites, the effects of putative -1 PRF induced costs to translation efficiency should be detectable.</p><p id="p-18">To this end, we devised three hypotheses. We derive these hypotheses from the assumption that the maintenance of -1 PRF signals carries an intrinsic cost to the organism. Furthermore, we assume that mutations that alter CUB are accumulated more slowly than mutations that alter -1 PRF efficiency, <em>p</em>. This is because the -1 PRF regulation mechanism is very likely to be affected by only a few mutations, whereas several mutations would be needed to significantly modify CUB. Hence, most short-term changes in protein expression demand in genes with slippery sites are more likely absorbed by -1 PRF.</p><p id="p-19">We first introduce a framework to formulate these hypotheses (see <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Let <em>L</em> be the length (in nucleotides) of an mRNA sequence <em>x</em>. A single slippery site is located at site <em>l</em> of <em>x</em>, downstream from the start. Ribosomes are redirected to the -1 frame with probability <em>p</em>, the <em>-1 PRF efficiency</em>. They continue with the mRNA’s translation until encountering a premature termination codon (PTC) at a distance <em>λ</em> post slippery site.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Notation for -1 PRF signals. A) The mRNA sequence in which a -1 PRF signal does not lead to ribosomal frameshifting is termed x (green). Arrows indicate the distances in nucleotides. B) The mRNA sequence with -1 PRF is termed . This is the concatenation of the mRNA x until the slippery site (green), and the -1 frame after the slippery site until a premature termination codon is encountered (violet)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1429157637" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Notation for -1 PRF signals. A) The mRNA sequence in which a -1 PRF signal does not lead to ribosomal frameshifting is termed &lt;em&gt;x&lt;/em&gt; (green). Arrows indicate the distances in nucleotides. B) The mRNA sequence with -1 PRF is termed &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-9&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. This is the concatenation of the mRNA &lt;em&gt;x&lt;/em&gt; until the slippery site (green), and the -1 frame after the slippery site until a premature termination codon is encountered (violet).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3173/F1.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3173/F1.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Notation for -1 PRF signals. A) The mRNA sequence in which a -1 PRF signal does not lead to ribosomal frameshifting is termed <em>x</em> (green). Arrows indicate the distances in nucleotides. B) The mRNA sequence with -1 PRF is termed <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F1/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F1/embed/mml-math-9.gif"/></noscript></span></span>. This is the concatenation of the mRNA <em>x</em> until the slippery site (green), and the -1 frame after the slippery site until a premature termination codon is encountered (violet).</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In a first hypothesis, we formed expectations about how the number of slippery sites per gene should vary with the ratio of protein expression to mRNA levels per cell. This ratio has been used as a measure for translation efficiency in past studies (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Tuller <em>et al.</em> 2010</a>). To formulate the first hypothesis, we conduct a thought experiment. Let us assume that an organism’s environmental conditions favor lower expression of a protein from a <em>-1 PRF</em>-gene. Then, increasing the average <em>p</em> in that gene will adapt the organism to the new conditions. Conversely, if such environmental changes demanded higher protein expression, genes containing -1 PRF signals may adapt by reducing <em>p</em>. If protein demands exceed the production capacity of a gene with very small <em>p</em>, the -1 PRF mechanism may offer only costs, but no benefits from protein regulation. Then, mutations that remove the heptameric slippery site signature will be selected for. Taken together, these effects imply an asymmetry in the probability of slippery site loss depending on the direction of protein demand changes. Losses of slippery sites are likelier when protein demands increase than when they decrease. -1 PRF signals should thus become rarer with increasing protein expression levels.</p><p id="p-22">The second hypothesis states that the costs of -1 PRF maintenance are reduced as slippery sites approach the 5′ end or start of the mRNA. Every time ribosomes slip through -1 PRF, the energy of translating a stretch of length <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-10.gif"/></noscript></span></span> of mRNA is wasted for protein production. Since <em>λ</em> is roughly constant across within-gene sites (see Fig. S1), the cost to translational efficiency incurred from -1 PRF should depend on <em>l</em>. Therefore, diminishing <em>l</em> will minimize cost. Thus, the distribution of slippery site positions relative to the gene length should be skewed toward mRNA starts, that is, small <em>l</em> values.</p><p id="p-23">This effect is expected to be weaker than the effect in the first hypothesis. In the first hypothesis, if protein demands exceed yields generated by a gene and <em>p</em> is small, a fitness cost is incurred from i) the reduction of protein expression, ii) the cost of NMD-mediated mRNA degradation and iii) the cost of unnecessary translation of frameshifted mRNA sequences. Displacing the slippery site would only reduce the latter cost. Thus, we expect the signal for this hypothesis to be weaker than for the first hypothesis.</p><p id="p-24">The third hypothesis states that a skew in the distribution of the location of the slippery site within a gene (relative to that gene’s length) should also become more pronounced with higher expression levels. This is because for fixed <em>p</em>, increasing expression levels should mirror an increase in -1 PRF induced translation costs. Thus, the benefits of slippery sites closer to translation-initiation positions should increase with expression levels. As in the second hypothesis, selection for such an effect is likely to be very weak compared to effects in the first hypothesis.</p><p id="p-25"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the results of testing all three hypotheses. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> shows that -1 PRF signals in genes become rarer with gene expression, consistent with our first hypothesis. <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> shows that slippery sites positions are more prevalent in the first half of a gene than in the second, consistent with our second hypothesis. The frequency of slippery sites diminishes toward both extremes of mRNA sequences, and more markedly so toward the stop codon of the mRNA. This skew persists when including all genes from the PRFdb (see Figure S2). <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a> shows that the average slippery site position is displaced toward the 5′ end of the mRNA as protein expression levels increase from 10<sup>2</sup> molecules per cell to 10<sup>5</sup> molecules per cell. Above 10<sup>5</sup> molecules per cell, the uncertainty around the averages becomes large due to low sample sizes, and unambiguous deductions become impossible. The decreasing trend in the average <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-11.gif"/></noscript></span></span> with protein expression is confirmed by a linear regression. Lastly, we retested all of the hypotheses with an alternative integrated data set from PaxDB, and obtained the same results (see Figure S3).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Testing the cost of -1 PRF mechanism maintenance. A) Number of slippery sites &#x2013;identified by relying on Nupack algorithm for pseudoknot detection (Dirks and Pierce 2004)&#x2013; per gene and per nucleotide across protein expression levels per mRNA level for genes from the von der Haar data set (von der Haar 2008). The red dotted line is the average number of slippery sites per gene. The blue line is a regression line through the data set. The text in the panels gives i) the Spearman correlation coefficient r and the p-value for the null-hypothesis that the correlation is zero and ii) the slope of the line and the p-value of the t-test for a non-zero slope value. B) The frequency distribution of the within-gene positions of the slippery sites, relative to the length of the gene,  To ensure comparability, only genes from the von der Haar data set are considered. The mean of the distribution, as well as its 95% confidence intervals, are smaller than 0.5 -the expectation in the absence of selective pressure. C) Slippery site positions relative to gene length across protein expression levels in the von der Haar data set. The red line is the average slippery site position (computed across 20 bins of equal width in logarithmic scale) and the widgets are the uncertainty ( standard error of the mean) around the average estimate. Averages with large uncertainties are in violet. Analogously to A), a regression line with the corresponding correlation coefficient r (with p-value) and slope (with p-value) are added." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1429157637" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Testing the cost of -1 PRF mechanism maintenance. A) Number of slippery sites &#x2013;identified by relying on &lt;em&gt;Nupack&lt;/em&gt; algorithm for pseudoknot detection (Dirks and Pierce 2004)&#x2013; per gene and per nucleotide across protein expression levels per mRNA level for genes from the von der Haar data set (von der Haar 2008). The red dotted line is the average number of slippery sites per gene. The blue line is a regression line through the data set. The text in the panels gives i) the Spearman correlation coefficient &lt;em&gt;r&lt;/em&gt; and the p-value for the null-hypothesis that the correlation is zero and ii) the slope of the line and the p-value of the &lt;em&gt;t&lt;/em&gt;-test for a non-zero slope value. B) The frequency distribution of the within-gene positions of the slippery sites, relative to the length of the gene, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-12&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; To ensure comparability, only genes from the von der Haar data set are considered. The mean of the distribution, as well as its 95% confidence intervals, are smaller than 0.5 -the expectation in the absence of selective pressure. C) Slippery site positions relative to gene length across protein expression levels in the von der Haar data set. The red line is the average slippery site position (computed across 20 bins of equal width in logarithmic scale) and the widgets are the uncertainty (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-13&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; standard error of the mean) around the average estimate. Averages with large uncertainties are in violet. Analogously to A), a regression line with the corresponding correlation coefficient &lt;em&gt;r&lt;/em&gt; (with p-value) and slope (with p-value) are added.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3173/F2.medium.gif" width="440" height="190"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3173/F2.medium.gif" width="440" height="190"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491057" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Testing the cost of -1 PRF mechanism maintenance. A) Number of slippery sites –identified by relying on <em>Nupack</em> algorithm for pseudoknot detection (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Dirks and Pierce 2004</a>)– per gene and per nucleotide across protein expression levels per mRNA level for genes from the von der Haar data set (<a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">von der Haar 2008</a>). The red dotted line is the average number of slippery sites per gene. The blue line is a regression line through the data set. The text in the panels gives i) the Spearman correlation coefficient <em>r</em> and the p-value for the null-hypothesis that the correlation is zero and ii) the slope of the line and the p-value of the <em>t</em>-test for a non-zero slope value. B) The frequency distribution of the within-gene positions of the slippery sites, relative to the length of the gene, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F2/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F2/embed/mml-math-12.gif"/></noscript></span></span> To ensure comparability, only genes from the von der Haar data set are considered. The mean of the distribution, as well as its 95% confidence intervals, are smaller than 0.5 -the expectation in the absence of selective pressure. C) Slippery site positions relative to gene length across protein expression levels in the von der Haar data set. The red line is the average slippery site position (computed across 20 bins of equal width in logarithmic scale) and the widgets are the uncertainty (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F2/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F2/embed/mml-math-13.gif"/></noscript></span></span> standard error of the mean) around the average estimate. Averages with large uncertainties are in violet. Analogously to A), a regression line with the corresponding correlation coefficient <em>r</em> (with p-value) and slope (with p-value) are added.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Thus, with increasing demand for protein production, the data suggest that production gains will primarily be attained by disposing of the -1 PRF mechanism present in a gene, rather than minimizing the cost of faulty translation. We speculate that, most likely, this occurs by altering the slippery site sequence. The test of the other two hypotheses involving slippery site displacement (or loss of large-<em>l</em> slippery sites) provide further evidence for an intrinsic -1 PRF cost to translation. The selective pressure on this latter adaptive process appears to be weak.</p></div><div id="sec-8" class="subsection"><h3>Retesting the translational efficiency hypothesis while accounting for -1 PRF genes</h3><p id="p-28">Since -1 PRF signals carry a translational cost, their presence in an mRNA sequence changes the meaning of the associated CUB index. In fact, a slippery site may indicate a mismatch between the CUB index of that sequence and the actual efficiency with which it is translated (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="-1 PRF signals bias codon usage bias indices&#x2019; predictiveness of translation efficiency. A) An mRNA sequence (green line) without a -1 PRF signal. Within a given period of time, the transcribed mRNA in the cell will produce a finite number of proteins. We define the efficiency of the protein synthetisation process as the energy content of the proteins (the nominator) divided by the energy expended by the mRNA transcription and translation processes (the denominator) (Tuller et al. 2010). The value of a codon usage bias index should reflect the efficiency of the protein synthesis process under the TEH. B) The same mRNA considering a slippery site (green point). The slippery site redirects ribosomes to a -1 shifted frame, where they encounter premature termination codons (PTC). The same number of proteins are produced as in A). An additional cost, unaccounted for in A), is incurred for protein production due to NMD-mediated protein decay. Thus, the effective translation efficiency when accounting for -1 PRF must be smaller than A). This diminished efficiency should be reflected by a correction in the codon usage bias index value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1429157637" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;-1 PRF signals bias codon usage bias indices&#x2019; predictiveness of translation efficiency. A) An mRNA sequence (green line) without a -1 PRF signal. Within a given period of time, the transcribed mRNA in the cell will produce a finite number of proteins. We define the efficiency of the protein synthetisation process as the energy content of the proteins (the nominator) divided by the energy expended by the mRNA transcription and translation processes (the denominator) (Tuller &lt;em&gt;et al.&lt;/em&gt; 2010). The value of a codon usage bias index should reflect the efficiency of the protein synthesis process under the TEH. B) The same mRNA considering a slippery site (green point). The slippery site redirects ribosomes to a -1 shifted frame, where they encounter premature termination codons (PTC). The same number of proteins are produced as in A). An additional cost, unaccounted for in A), is incurred for protein production due to NMD-mediated protein decay. Thus, the effective translation efficiency when accounting for -1 PRF must be smaller than A). This diminished efficiency should be reflected by a correction in the codon usage bias index value.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3173/F3.medium.gif" width="440" height="373"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3173/F3.medium.gif" width="440" height="373"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490992" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">-1 PRF signals bias codon usage bias indices’ predictiveness of translation efficiency. A) An mRNA sequence (green line) without a -1 PRF signal. Within a given period of time, the transcribed mRNA in the cell will produce a finite number of proteins. We define the efficiency of the protein synthetisation process as the energy content of the proteins (the nominator) divided by the energy expended by the mRNA transcription and translation processes (the denominator) (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Tuller <em>et al.</em> 2010</a>). The value of a codon usage bias index should reflect the efficiency of the protein synthesis process under the TEH. B) The same mRNA considering a slippery site (green point). The slippery site redirects ribosomes to a -1 shifted frame, where they encounter premature termination codons (PTC). The same number of proteins are produced as in A). An additional cost, unaccounted for in A), is incurred for protein production due to NMD-mediated protein decay. Thus, the effective translation efficiency when accounting for -1 PRF must be smaller than A). This diminished efficiency should be reflected by a correction in the codon usage bias index value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">To elucidate this point, consider an mRNA without -1 PRF signal. In the -1 PRF signal’s absence, the CUB index of a translated mRNA sequence appropriately reflects how efficiently ribosomes elongate it in a pre-specified time (see <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Tuller <em>et al.</em> 2010</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Stoletzki and Eyre-Walker 2007</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Akashi 1994</a>). However, given a functional slippery site in that same mRNA sequence, the interpretation changes (see <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Let us assume an average -1 PRF efficiency <em>p</em>. Then, a fraction <em>p</em> of the times in which a ribosome is in the process of translating a specific mRNA, it will be redirected to another frame by -1 PRF. This redirection leads to degradation of the mRNA. Thus, the production of the same number of proteins will require more energy and time in the presence of a -1 PRF signal than it would in its absence. However, traditional CUB indices will give no indication of that process. If accounted for, -1 PRF must therefore lead to a downward correction of the mRNA’s translational efficiency and with it, of the associated CUB index.</p><p id="p-31">Since a substantial fraction of genes may contain -1 PRF signals (hypothesized to be ≈ 10% of genes in yeast (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Belew <em>et al.</em> 2008</a>)), neglecting this effect might introduce considerable biases in CUB index values. These biases could affect the associations found between CUB and protein expression (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Jansen <em>et al.</em> 2003</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Kliman <em>et al.</em> 2003</a>; <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Roth <em>et al.</em> 2012</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Quax <em>et al.</em> 2015</a>). To address this issue, in the following we propose two general corrections for CUB indices in the presence of -1 PRF signals, and retest the basis for the TEH stemming from CUB and protein expression correlations.</p><div id="sec-9" class="subsection"><h4>Codon Usage Bias Index Correction from Translation Efficiency:</h4><p id="p-32">In a first approach, we derive an estimator for a CUB index correction from basic relationships between CUB indices and translation efficiencies of an mRNA. In the absence of a -1 PRF signal, the CUB on <em>x</em> will be measured by some index function <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-14.gif"/></noscript></span></span> As noted, if a functional -1 PRF signal is present, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-15.gif"/></noscript></span></span> does not account for the translation efficiency loss due to the unproductive translation of the hybrid sequence <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-16.gif"/></noscript></span></span> (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). We aim to derive a “corrected” index <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-17.gif"/></noscript></span></span> that more appropriately reflects the diminished translation efficiency in the presence of -1 PRF signals.</p><p id="p-33">To find an expression for <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-18.gif"/></noscript></span></span> we begin with the assumption that there exists a monotonous mapping <em>F</em> that maps the translation efficiency <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-19.gif"/></noscript></span></span> of a sequence <em>x</em> (in the absence of -1 PRF) to a codon usage bias index <em>I</em>: <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-20.gif"/></noscript></span></span> <em>F</em> is a monotonously increasing function, where increases in translation efficiency are reflected by increases in the codon usage bias index. Research by Tuller <em>et al.</em> further suggests that <em>F</em> is concave (<a id="xref-ref-69-5" class="xref-bibr" href="#ref-69">Tuller <em>et al.</em> 2010</a>), such that codon usage bias index values saturate with increasing translation efficiency values. We define translation efficiency <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-21.gif"/></noscript></span></span> classically as an input-output energy ratio. More specifically, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-22.gif"/></noscript></span></span> is the ratio of the total per-protein energy <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-23.gif"/></noscript></span></span> contained in the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-24.gif"/></noscript></span></span> synthesized proteins from <em>x</em> to the energy, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-25.gif"/></noscript></span></span> used for producing those proteins in a sufficiently long time frame <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-26.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Here, the energy going into the synthesization machinery, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-28.gif"/></noscript></span></span> does not contain the expenditure for -1 PRF’s regulatory use. Thus, for a sequence <em>x</em> that additionally carries a functional slippery site, we have<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Here, the same number of proteins were produced as when -1 PRF was not considered, that is, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-30.gif"/></noscript></span></span> Since the final proteins are structurally equivalent to those in the absence of -1 PRF, we also have <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-31.gif"/></noscript></span></span> However, more energy was expended to produce these proteins, and therefore <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-32.gif"/></noscript></span></span> Their synthesization requires at least <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-33.gif"/></noscript></span></span> A part of the expended energy on translation is not implemented in the proteins, but in the translation of <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-34.gif"/></noscript></span></span> and the NMD pathway activation, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-35.gif"/></noscript></span></span> Thus, it follows that <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-36.gif"/></noscript></span></span></p><p id="p-34">The corrected usage bias statistic <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-37.gif"/></noscript></span></span> is defined as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>Since the form of <em>F</em> as well as the values of <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-39.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-41.gif"/></noscript></span></span> are either difficult to measure or unknown, we aim to to compute <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-42.gif"/></noscript></span></span> indirectly from <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-43.gif"/></noscript></span></span> To this end, we separate the translation efficiency <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-44.gif"/></noscript></span></span> into two components:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Solving for <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-46.gif"/></noscript></span></span> gives<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-47.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-48.gif"/></noscript></span></span> With this, we have<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-49.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>Further analysis of the ratio <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-50.gif"/></noscript></span></span> is complicated by the inability to directly measure <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-51.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-52.gif"/></noscript></span></span> To address this issue, we pursue an approach where the ratio is approximated by information about the relative one-elongation energies spent translating <em>x</em> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-53.gif"/></noscript></span></span> More specifically, we assume that each time <em>x</em> is translated, an energy input of <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-54.gif"/></noscript></span></span> is expended per time unit <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-55.gif"/></noscript></span></span> The total number of times translation is initiated on the mRNA sequence, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-56.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-57.gif"/></noscript></span></span> This is because <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-58.gif"/></noscript></span></span> corresponds to the fraction <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-59.gif"/></noscript></span></span> of times that an mRNA-elongating ribosome remains in frame. Analogously, the number of times translation is interrupted by a -1 PRF event is <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-60.gif"/></noscript></span></span> Hence, each time the sequence <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-61.gif"/></noscript></span></span> is translated, an energy expenditure of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-62.gif"/></noscript></span></span> is ensued. Note that <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-63.gif"/></noscript></span></span> does not correspond to a protein number. The ratio between these two is approximated by:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-64.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>which is independent of the number of synthesized proteins <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-65.gif"/></noscript></span></span> Here, we have assumed that ratio of the per-run translation energies expended for the non-frameshifting to the frameshifting scenarios corresponds roughly to the ratio of the lengths of the translated sequences <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-66.gif"/></noscript></span></span> to <em>x</em>, respectively. However, there is an extra cost to each frameshifted mRNA, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-67.gif"/></noscript></span></span> stemming from NMD-mediated degradation. This cost is accounted for by <em>π</em>, which is an unknown fixed cost associated with each -1 PRF event. <em>π</em> is measured in units of equivalents of translation cost per nucleotide.</p><p id="p-35">With (9), we can approximate the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-68.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>With this, we are ready to address the last approximation required to find an analytical expression for <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-70.gif"/></noscript></span></span> Since <em>F</em> is concave, it follows that <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-71.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-72.gif"/></noscript></span></span> Thus, setting <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-73.gif"/></noscript></span></span> and using definition (4), we find a lower bound for <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-74.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>The approximation for <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-76.gif"/></noscript></span></span> results in reasonable and useful properties of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-77.gif"/></noscript></span></span> More specifically, as <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-78.gif"/></noscript></span></span> we have that <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-79.gif"/></noscript></span></span> This entails that no mRNA is translated into proteins, as expected. The same follows if <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-80.gif"/></noscript></span></span> The extraction of <em>a</em> from within the brackets leads to the undesirable behavior that as <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-81.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-82.gif"/></noscript></span></span> we have that <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-83.gif"/></noscript></span></span> Instead, <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-84.gif"/></noscript></span></span> should approximate some minimal value <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-85.gif"/></noscript></span></span></p><p id="p-36">A major drawback of <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-86.gif"/></noscript></span></span> is its reliance on <em>p</em> and <em>π</em>, which are both unknown. Typically, <em>p</em> is assumed to lie between <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-87.gif"/></noscript></span></span> but may reach up to <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-88.gif"/></noscript></span></span> (for example the <em><a href="http://www.yeastgenome.org/locus/S000004310/overview">EST2</a></em> gene in yeast, (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>)). In the following, and if not stated otherwise, we assume that <em>p</em> is constant across all genes. It is unclear how large <em>π</em> should be. We have found no evidence that the value of <em>π</em> is dependent on any characteristic of mRNA, such as its length, or codon composition (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Chang <em>et al.</em> 2007</a>; <a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Dinman, 2012b</a>). NMD is an evolutionarily conserved surveillance pathway (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Chang <em>et al.</em> 2007</a>). Its activation may thus differ in energy expenditure between organisms (indeed, in <em>yeast</em>, NMD does not require an exon-junction complex, unlike most other eukaryotes). However, we have found no indication that the energy required to fully complete the NMD process will substantially vary from mRNA to mRNA within an organism. Thus, we assume that <em>π</em> is uniform across genes, although we do not know how large it is relative to other energy inputs. Again, if not stated otherwise, we assume that <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-89.gif"/></noscript></span></span> in order to give a conservative estimate of the effects of -1 PRF on the TEH.</p></div><div id="sec-10" class="subsection"><h4>Codon Usage Bias Index Correction from Averaging:</h4><p id="p-37">In a second approach, we define a correction for a -1 PRF-aware CUB index using a balancing principle. To this end, we add the CUB index <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-90.gif"/></noscript></span></span> in successful, non-NMD mediated translations of <em>x</em> and the index <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-91.gif"/></noscript></span></span> of untranslated <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-92.gif"/></noscript></span></span> while weighing both with their respective probability of occurrence:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-93.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>The corrected index <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-94.gif"/></noscript></span></span> is thus the expected value of <em>I</em> when considering that <em>x</em> is translated a fraction <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-95.gif"/></noscript></span></span> of the time, and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-96.gif"/></noscript></span></span> is translated a fraction <em>p</em> of the time. <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-97.gif"/></noscript></span></span> therefore acts as a penalization function. <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-98.gif"/></noscript></span></span> may underestimate the extra energy costs to -1 PRF incurred from the activation of the NMD-mediated degradation processes, since these are not comprised in <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-99.gif"/></noscript></span></span> Similarly to <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-100.gif"/></noscript></span></span> the major drawback of the estimator <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-101.gif"/></noscript></span></span> is its dependence on <em>p</em>. As before, we thus evaluate <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-102.gif"/></noscript></span></span> for different, plausible values of <em>p</em> to assess -1 PRF’s effect on the TEH. However, <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-103.gif"/></noscript></span></span> has the advantage that it is also well defined for <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-104.gif"/></noscript></span></span></p></div><div id="sec-11" class="subsection"><h4>Reexamining the TEH with corrected CUB indices:</h4><p id="p-38">Both corrections <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-105.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-106.gif"/></noscript></span></span> represent a change in the value of a codon usage bias index given a -1 PRF efficacy. The models for correcting <em>I</em> presented here do not relate these measures to protein expression levels, <em>P</em>. They do therefore not predict protein expression. Instead, they allow us to reexamine the association between protein expression and a corrected codon usage bias index that in part underpins the TEH.</p><p id="p-39">To examine how these corrections affect associations between CUB and protein expression, we used the widely employed <em>codon adaptation index</em> (CAI) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Carbone <em>et al.</em> 2003</a>) as an example for a CUB index <em>I</em>. We then computed both corrected <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-107.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-108.gif"/></noscript></span></span> for different values of <em>p</em> and for all genes in the von der Haar data set (<a id="xref-ref-72-5" class="xref-bibr" href="#ref-72">von der Haar 2008</a>) that contain -1 PRF signals. For <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-109.gif"/></noscript></span></span> we also assumed different values of <em>π</em>.</p><p id="p-40">A precise computation of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-110.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-111.gif"/></noscript></span></span> would require information about the value of <em>p</em> across genes. Since no such information is available as of now, we follow two approaches. First, we treat all -1 PRF sites equally, assuming an equal <em>p</em> value for all, independent of the protein expression levels <em>P</em> of the genes they are located in. Second, we assume that <em>p</em> is associated with <em>P</em>, and allow <em>p</em> to slowly decrease with the <em>P</em> of the gene.</p><p id="p-41"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows that correcting for the presence of -1 PRF signals with a -1 PRF efficiency of up to <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-112.gif"/></noscript></span></span> does not substantially affect the CUB index to protein expression relationship for both CUB index corrections, if the extra cost of mRNA degradation by NMD is neglected (<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-113.gif"/></noscript></span></span>). <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>) shows that all CAI values for all genes are diminished when using the correction <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-114.gif"/></noscript></span></span> A substantial correlation between protein expression and corrected CAI index remains. Accounting for the frameshifted sequences <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-115.gif"/></noscript></span></span> also diminishes codon adaptation measures <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-116.gif"/></noscript></span></span> considerably compared to uncorrected CUB index values, as shown in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). However, unlike with <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-117.gif"/></noscript></span></span> this reduction does not correspond to a uniform negative offset. Instead, the effect of accounting for -1 PRF is to both shift and broaden the distribution of <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-118.gif"/></noscript></span></span> values relative to <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-119.gif"/></noscript></span></span> (see Figure S4).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Corrected and original codon adaptation indices vs. protein expression levels for genes containing algorithmically identified -1 PRF signals and with known expression levels (von der Haar 2008), assuming  and  A) The original codon adaptation index values of mRNA sequences x of genes from the (von der Haar 2008) data set are shown as orange, semi-transparent points. The shown genes all contain algorithmically identified -1 PRF signals. The shape of the relation between CAI and protein expression is similar to that shown in (Ghaemmaghami et al. 2003) for all genes with known expression levels. Blue, semi-transparent points denote the values of -1 PRF corrected CAI values,  vs. same protein expression levels. Regressions between the -protein expression levels and both corrected an original CAI values are shown as orange and blue lines, respectively. B) is analogous to A) using the corrected CUB index " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1429157637" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Corrected and original codon adaptation indices &lt;em&gt;vs.&lt;/em&gt; protein expression levels for genes containing algorithmically identified -1 PRF signals and with known expression levels (von der Haar 2008), assuming &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-120&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-121&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; A) The original codon adaptation index values of mRNA sequences &lt;em&gt;x&lt;/em&gt; of genes from the (von der Haar 2008) data set are shown as orange, semi-transparent points. The shown genes all contain algorithmically identified -1 PRF signals. The shape of the relation between CAI and protein expression is similar to that shown in (Ghaemmaghami &lt;em&gt;et al.&lt;/em&gt; 2003) for all genes with known expression levels. Blue, semi-transparent points denote the values of -1 PRF corrected CAI values, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-122&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;em&gt;vs.&lt;/em&gt; same protein expression levels. Regressions between the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-123&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;-protein expression levels and both corrected an original CAI values are shown as orange and blue lines, respectively. B) is analogous to A) using the corrected CUB index &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-124&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3173/F4.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3173/F4.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491127" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Corrected and original codon adaptation indices <em>vs.</em> protein expression levels for genes containing algorithmically identified -1 PRF signals and with known expression levels (<a id="xref-ref-72-6" class="xref-bibr" href="#ref-72">von der Haar 2008</a>), assuming <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-120.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-121.gif"/></noscript></span></span> A) The original codon adaptation index values of mRNA sequences <em>x</em> of genes from the (<a id="xref-ref-72-7" class="xref-bibr" href="#ref-72">von der Haar 2008</a>) data set are shown as orange, semi-transparent points. The shown genes all contain algorithmically identified -1 PRF signals. The shape of the relation between CAI and protein expression is similar to that shown in (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Ghaemmaghami <em>et al.</em> 2003</a>) for all genes with known expression levels. Blue, semi-transparent points denote the values of -1 PRF corrected CAI values, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-122.gif"/></noscript></span></span> <em>vs.</em> same protein expression levels. Regressions between the <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-123.gif"/></noscript></span></span>-protein expression levels and both corrected an original CAI values are shown as orange and blue lines, respectively. B) is analogous to A) using the corrected CUB index <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F4/embed/mml-math-124.gif"/></noscript></span></span></p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The two corrections <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-125.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-126.gif"/></noscript></span></span> show different sensitivities to the -1 PRF efficiency. In particular, the sensitivity of <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-127.gif"/></noscript></span></span> to -1 PRF efficiency is mediated by the value of <em>π</em>. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> shows that the correlations between both <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-128.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-129.gif"/></noscript></span></span> to protein expression levels decline for increasing values of <em>p</em> and different values of <em>π</em>. As <em>p</em> increases, and for low costs of NMD-mediated mRNA decay, the correlation of <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-130.gif"/></noscript></span></span> to protein expression levels declines to very low values (<span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-131.gif"/></noscript></span></span> at <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-132.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-133.gif"/></noscript></span></span>). If the energetic cost of -1 PRF induced mRNA degradation, <em>π</em>, becomes larger (<span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-134.gif"/></noscript></span></span>), the correlation declines very rapidly with <em>p</em>, and almost vanishes when <em>p</em> reaches <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-135.gif"/></noscript></span></span> For <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-136.gif"/></noscript></span></span> estimates, correlations’ dependency on <em>p</em> is moderate, never falling below 0.4 at biologically implausible values of <em>p</em> of unity.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation values between corrections of CAI to -1 PRF presence () to protein expression levels P (von der Haar 2008) for different -1 PRF efficiencies, p. A) For  we considered different values of &#x3C0;, the cost of degrading an mRNA after -1 PRF, namely  For  correlation values were computed at -1 PRF efficiencies   was computed at the same levels except for  where instead  was used. B) Analogous to A), but with p dependent on protein expression. For each gene, the -1 PRF efficiency is computed by  where for  the baseline -1 PRF efficiency, we chose the same values as for p in A)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1429157637" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Correlation values between corrections of CAI to -1 PRF presence (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) to protein expression levels &lt;em&gt;P&lt;/em&gt; (von der Haar 2008) for different -1 PRF efficiencies, &lt;em&gt;p&lt;/em&gt;. A) For &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; we considered different values of &lt;em&gt;&#x3C0;&lt;/em&gt;, the cost of degrading an mRNA after -1 PRF, namely &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-139&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; For &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-140&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; correlation values were computed at -1 PRF efficiencies &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-141&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-142&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; was computed at the same levels except for &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-143&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where instead &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-144&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; was used. B) Analogous to A), but with &lt;em&gt;p&lt;/em&gt; dependent on protein expression. For each gene, the -1 PRF efficiency is computed by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-145&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where for &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-146&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; the &lt;em&gt;baseline -1 PRF efficiency&lt;/em&gt;, we chose the same values as for &lt;em&gt;p&lt;/em&gt; in A).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3173/F5.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/10/3173/F5.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3173/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491097" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Correlation values between corrections of CAI to -1 PRF presence (<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-137.gif"/></noscript></span></span>) to protein expression levels <em>P</em> (<a id="xref-ref-72-8" class="xref-bibr" href="#ref-72">von der Haar 2008</a>) for different -1 PRF efficiencies, <em>p</em>. A) For <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-138.gif"/></noscript></span></span> we considered different values of <em>π</em>, the cost of degrading an mRNA after -1 PRF, namely <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-139.gif"/></noscript></span></span> For <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-140.gif"/></noscript></span></span> correlation values were computed at -1 PRF efficiencies <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-141.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-142.gif"/></noscript></span></span> was computed at the same levels except for <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-143.gif"/></noscript></span></span> where instead <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-144.gif"/></noscript></span></span> was used. B) Analogous to A), but with <em>p</em> dependent on protein expression. For each gene, the -1 PRF efficiency is computed by <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-145.gif"/></noscript></span></span> where for <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/F5/embed/mml-math-146.gif"/></noscript></span></span> the <em>baseline -1 PRF efficiency</em>, we chose the same values as for <em>p</em> in A).</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">These results corroborate the support for the translation efficiency hypothesis. For uniform -1 PRF efficiencies and for <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-147.gif"/></noscript></span></span>-based estimates, correlations of corrected CAI to protein expression levels decline with <em>p</em>. They only become sufficiently diminished to challenge the TEH when both the values of <em>π</em> and <em>p</em> are very high. Indeed, according to our mathematical framework, <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-148.gif"/></noscript></span></span> corresponds to the cost of either translating a large gene, or tenfold that cost. Except for <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-149.gif"/></noscript></span></span> all correlation values between corrected CUB indices and protein expression levels remain at around <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-150.gif"/></noscript></span></span> for biologically relevant -1 PRF efficiencies of <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-151.gif"/></noscript></span></span> For <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-152.gif"/></noscript></span></span> estimates, correlation seems generally robust to changes in <em>p</em>.</p></div><div id="sec-12" class="subsection"><h4>Varying -1 PRF efficiency with protein expression levels:</h4><p id="p-46">Since the data in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> strongly suggest a cost to -1 PRF maintenance, we also explored how a -1 PRF efficiency decline with protein expression levels consistent with such cost would affect the correlations in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>. Unlike a uniform <em>p</em> across protein levels, corrections to <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-153.gif"/></noscript></span></span> are expected to be larger at low expression levels than at high levels. The -1 PRF decline is assumed as follows: <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-154.gif"/></noscript></span></span> where we call <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-155.gif"/></noscript></span></span> the <em>baseline -1 PRF efficiency</em>.</p><p id="p-47">For <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-156.gif"/></noscript></span></span> this additional assumption leads to a measure:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-157.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>Analogously, <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-158.gif"/></noscript></span></span> becomes<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-159.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span><a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a> shows that under this assumption, correlations between protein expression and <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-160.gif"/></noscript></span></span> rise with larger baseline -1 PRF efficiencies for all except the largest values of <em>π</em>, while again <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-161.gif"/></noscript></span></span> remains unaffected. We observe analogous results using the Spearman rank correlation (see Figure S5).</p><p id="p-48">When -1 PRF efficiencies decline with protein expression –in accordance with a cost to -1 PRF– most correlations between <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-162.gif"/></noscript></span></span>-based estimates and protein expression rise with -1 PRF baselines. Except for <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-163.gif"/></noscript></span></span> all correlation values increase with -1 PRF baselines. This also adds support for the TEH, suggesting that -1 PRF conceals stronger associations than measured with uncorrected codon usage bias indices.</p></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-49">In this study, we have investigated the hypothesis that -1 PRF presupposes translational costs to an organism, while at the same time generating benefits associated with protein expression regulation. We explored whether such a cost might be identified more directly in data and used both PRFdb (<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Belew <em>et al.</em> 2008</a>) and the von der Haar data set (<a id="xref-ref-72-9" class="xref-bibr" href="#ref-72">von der Haar 2008</a>) to address this question. We devised three hypotheses for likely signals of such cost in -1 PRF carrying genes. We could not find contradictory evidence for any of those hypotheses in the data. In a second step, we explored whether these costs, if not accounted for, bias important measures of CUB. We devised two new general approaches to correct CUB indices for the presence of -1 PRF signals. We then tested whether the concealment of such costs may unduly influence, falsify or strengthen, one classical test of the translational efficiency hypothesis: the association of CUB indices with protein expression levels. Under the assumption of uniform -1 PRF efficiencies, the energetic costs related with NMD activation would need to be implausibly high to warrant this conclusion. We find that on the contrary, assuming that -1 PRF efficiency decreases with protein expression levels -as suggested by the existence of a cost to -1 PRF-, the TEH is strengthened. Thus, taken together, our results suggests that high organismal demands for specific proteins are reflected in CUB-mediated translation efficiency gains.</p><p id="p-50">Our study comes with a series of caveats. A first caveat of this study lies in that the slippery sites are inferred (utilizing the <em>Nupack</em> algorithm (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Dirks and Pierce 2004</a>) for mRNA pseudoknot detection), but not always experimentally confirmed (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>). Previous studies suggest that inferred slippery sites are likely to be functional. In (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>) it is reported that in previous work, nine out of nine high confidence -1 PRF sites detected by methods used in the PRFdb in <em>S. cerevisae</em> were confirmed to be functional <em>in vivo</em>. Crucially, whether a -1 PRF site is regarded to be <em>functional</em> depends on whether it exceeds a predetermined <em>p</em>-threshold. For example, in (<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>), a search of -1 PRF slippery sites identified 10 candidate genes in <em><a href="http://www.yeastgenome.org/locus/S000004310/overview">EST2</a></em>, three in <em><a href="http://www.yeastgenome.org/locus/S000004223/overview">EST1</a></em>, 2 in <em>STN</em>, and 1 in <em>CDC</em> in mRNA involved in telomerase. Out of these, and employing a cutoff of 1% for -1 PRF efficiency, seven carry functional slippery sites (<em><a href="http://www.yeastgenome.org/locus/S000004310/overview">EST2</a></em>: 3, <em><a href="http://www.yeastgenome.org/locus/S000004223/overview">EST1</a></em>: 2, <em>STN</em>: 1, <em>CDC</em>: 1) (see (<a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>), Table 1). Had a cutoff of 0% -1 PRF efficiency been employed, all slippery sites except one would be functional. This limited sample provides confidence that the methods described by (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Belew <em>et al.</em> 2008</a>) appropriately capture biological mechanisms.</p><p id="p-51">Even with such uncertainty, it is unlikely that the presence of non-functional -1 PRF sites in the analyzed data would affect the claim of an intrinsic -1 PRF maintenance cost (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). This claim would only be biased if the probability of a candidate site to be functional were affected by either protein expression levels of the gene within which it resides or alternatively, the relative position in the gene of the putative slippery site. Note that these effects should arise from mechanisms that are independent of the ones studied here. Again, we are unaware of any such mechanisms acting in yeast, except for the requirement of a spacer sequence. A spacer sequence of a minimum length of say, 8 nt, will prohibit slippery sites to be located within 8 nt of the mRNA end. This restriction will slightly bias the <em>a priori</em> position of candidate slippery sites. However, as average mRNA by far exceed this length, such a restriction cannot explain the effects documented here.</p><p id="p-52">Another caveat of our study was the lack of estimates for <em>p</em> for the whole data set. Due to this restriction, we could not explore how -1 PRF efficiency <em>p</em> relates to CAI or protein expression. As assumed in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>, we expect that -1 PRF maintenance costs should on average translate into an inverse relationship between <em>p</em> and protein expression levels, because higher protein expression demands by the organism should be countered with reductions of <em>p</em>, minimizing the loss of mRNA to NMD-induced degradation. However, it is also possible that there exist cases in which -1 PRF has important functions in highly expressed genes. Hence, although there are fewer slippery sites for highly expressed genes and they are costly in terms of translation efficiencies, these sites could have large -1 PRF efficiencies. We have not found evidence in the literature to support this notion, and it would concomitantly contradict the evidence here presented. Additional testing of our hypotheses on the costs -1 PRF maintenance would be greatly helped if information on how -1 PRF efficiencies vary with protein expression was available.</p><p id="p-53">While our analysis of the behavior of <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-164.gif"/></noscript></span></span> gives us an indication how -1 PRF costs to translation efficiency could bias CUB indices, these insights rely on the assumptions made during <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-165.gif"/></noscript></span></span>’s derivation. Importantly, <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-166.gif"/></noscript></span></span> is a lower bound to <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-167.gif"/></noscript></span></span> which means that downward corrections of <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-168.gif"/></noscript></span></span> are potentially exaggerated. Thus, the claim that corrected CUB indices will leave the basis for the TEH unaffected is resistant to such a bias. Another key assumption is that CUB indices of an mRNA sequence should increase monotonically with translation efficiency. In practice, this association is surely not perfect, due to multiple additional influences on CUB from unrelated biological processes. For these processes to systematically bias our results they should i) dominate over translational efficiency effects or ii) result in directional effects when combined. Despite these possible shortcomings, the assumption reflects key properties with which many CUB indices are designed, namely to mirror translational efficiency. For example, the CAI uses ribosomal mRNA as a reference to compute preferred codons because it is highly plausible that they are translationally efficient.</p><p id="p-54">Moreover, our results suggest that utilizing expected values for corrected CUB index definition, such as in the case of <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-169.gif"/></noscript></span></span> is suboptimal. Analysis of the distribution of <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-170.gif"/></noscript></span></span> levels with <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-171.gif"/></noscript></span></span> of genes in the von der Haar data set show that corrected codon adaptation can increase. This behavior contradicts the rationale behind introducing such corrections in the first place, as, surprisingly, <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3173/embed/mml-math-172.gif"/></noscript></span></span> can occur.</p><p id="p-55">To test the hypotheses about the TEH derived here, we have only utilized genes with identified slippery sites, and not all genes. Testing on all genes would be indicated if the TEH was challenged. However, if the correlation coefficient measured within a subset in which a correction has been applied is not substantially diminished relative to uncorrected values, the same correction will not affect the whole set either. Therefore testing it on the whole set is not necessary.</p><p id="p-56">The results indicating intrinsic -1 PRF maintenance cost and TEH support are in mutual agreement. -1 PRF maintenance induces a reduction of the size of <em>l</em> with increasing protein expression levels (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>). At fixed <em>p</em>, as assumed in our analysis, significant reductions in <em>l</em> would diminish the penalization to the original, un-corrected CUB. This could only be compensated by increases of <em>p</em> or <em>π</em> with protein expression levels, contrary to intuition and available evidence (<a id="xref-ref-1-9" class="xref-bibr" href="#ref-1">Advani <em>et al.</em> 2013</a>). In fact, our analysis shows that a more plausible <em>p</em> dependency to protein expression leads to a strengthening of the basic CUB index to expression level association.</p><p id="p-57">These results have to be interpreted in the context of current codon usage bias research. The mechanism of <em>translational selection</em> (TEH) remains the main explanation put forward for selection based origins of codon bias. This explanation presupposes that silent mutations affect fitness via translation processes. These fitness increases originate from translation efficiency and accuracy gains, that in turn are hypothesized to stem from translation initiation and elongation processes. In fact, mRNA elongation rates do indeed appear to correlate positively with preferred codon frequencies (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gustafsson <em>et al.</em> 2004</a>), although better evidence would be desirable. However, for initiation processes –which are assumed to contribute the bulk of these fitness increases–, the effects of CUB on initiation rate increase remain subject of debate (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Quax <em>et al.</em> 2015</a>). In fact, Tuller and Zur have analyzed the effect of the structure of the 5′ end of an mRNA on translation initiation and elongation rates and found various regulatory signals that affect these rates in different ways (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Tuller and Zur 2015</a>). Indeed, the induced folding at the 5′ end of the ORF appears to affect translation efficiency.</p><p id="p-58">Overall, the evidence from codon usage bias statistics and its associations to tRNA abundance and protein expression offers a compelling narrative for the TEH. However, current research efforts aiming to identify the exact mechanisms that give rise to these associations must account for conflating selective forces. Indeed, the mechanisms laid out in the TEH may not be the only way in which selection shapes codon usage frequencies. For example, besides the abundance of tRNAs, other factors have been discovered to crucially affect elongation rates and hence, to be possible targets selection (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Chamary <em>et al.</em> 2006</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Resch <em>et al.</em> 2007</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hunt <em>et al.</em> 2009</a>). More precisely, specific synonymous changes can influence mRNA splicing, mRNA secondary structure, protein stability as well as protein folding (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Komar 2009</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Chamary <em>et al.</em> 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Carlini and Genut 2006</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Parmley <em>et al.</em> 2006</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Tsai <em>et al.</em> 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Komar 2007</a>). Synonymous changes may also alter the <em>secondary structure of mRNA</em> and thus affect the rate of translation –as established <em>in vitro</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ivanov <em>et al.</em> 1997</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Parmley and Hurst 2007</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Tuller and Zur 2015</a>) and subsequently <em>in vivo</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kimchi-Sarfaty <em>et al.</em> 2007</a>). A comprehensive review on how protein expression is fine tuned by codon usage bias is given by Quax (<a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Quax <em>et al.</em> 2015</a>). Further, in a very recent study, a synonymous difference between mammalian cytoskeletal <em>β</em>- and <em>γ</em>-actin proteins was found to affect co-translational processing, ubiquitination, and co-translational degradation, leading to differential stability properties of the corresponding protein products (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zhang <em>et al.</em> 2010</a>).</p><p id="p-59">Within genomes, the biologial role of a gene influences that gene’s CUB in various additional ways. Some codons are more abundant in genes depending on their function, displaying distinct codon bias patterns. Supek has reviewed how gene function modifies codon preferences (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Supek 2016</a>). Selective pressure to maintain (or alter) gene function is superimposed to what is expected from translational efficiency and accuracy optimization. Genes where altered CUB patterns have been found are involved in diverse functions: amino acid starvation responses, cyclical protein expression, tissue specific expression, cellular differentiation, stress responses, and carcinogenesis. While genes can differ in function, there are also differences in function within a gene’s sequence that also affect local CUB. For example, Tuller and Zur have surveyed the multiple roles of the 5′ end of coding sequences in gene expression regulation (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Tuller and Zur 2015</a>). They hypothesize that due to multitude of regulatory signals found in that region, selection pressures regarding codon utilization are likely different than in other regions. Unlike cross-gene function, such effects are local, which should not affect our analysis. How effects from gene function may affect our results, crucially depends on their frequency and direction. The literature reviewed here does not appear to warrant the assumption that all the function-dependent selective pressures will align to influence CUB in the same way, leading to systematic bias. The phenomenon of -1 PRF is thus only one of many ways in which CUB may be influenced.</p><p id="p-60">The particular appeal of the -1 PRF phenomenon lies in its potential to elucidate many of these processes. Because slippery sites are precisely localized and -1 PRF efficiencies measurable, -1 PRF signals constitute natural experiments to translation efficiency and accuracy theories. This is because the separation of the mRNA by a slippery site should create differential translational costs across that mRNA. This translational cost gradient should be reflected in codon usage bias differences. -1 PRF based approaches to analyzing codon usage bias behavior, like the one presented in this study, may thus offer novel tools to better understand the means by which translation efficiency gains are realized in nature.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-61">The authors thank Spencer Bliven and Lorenzo Gatti for stimulating discussions. This work was supported by the SystemsX – the Swiss Initiative for Systems Biology (grant number 51FSP0_163566).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-62">Supplemental material available at Figshare: <a href="https://doi.org/10.6084/m9.figshare.6970277">https://doi.org/10.6084/m9.figshare.6970277</a>.</p></li><li class="fn" id="fn-2"><p id="p-63"><em>Communicating editor: R. Kulathinal</em></p></li><li class="fn" id="fn-3"><p id="p-64">Manuscript compiled: Monday 13th August 2018</p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-26"><span class="received-label">Received </span>February 26, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-30"><span class="accepted-label">Accepted </span>July 30, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Garcia <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.1" data-doi="10.4161/trla.24418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Advani</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belew</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Yeast telomere maintenance is globally controlled by programmed ribosomal frameshifting and the nonsense-mediated mrna decay pathway.</span> <abbr class="cit-jnl-abbrev">Translation (Austin)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e24418</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/trla.24418</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTranslation%2B%2528Austin%2529%26rft.volume%253D1%26rft.spage%253De24418%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ftrla.24418%26rft_id%253Dinfo%253Apmid%252F24563826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/trla.24418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24563826&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.2" data-doi="10.1002/bies.201500131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Advani</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Reprogramming the genetic code: The emerging role of ribosomal frameshifting in regulating cellular gene expression.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.201500131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D38%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201500131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201500131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Synonymous codon usage in drosophila melanogaster: natural selection and translational accuracy.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D136%26rft.issue%253D3%26rft.spage%253D927%26rft.epage%253D935%26rft.atitle%253DSynonymous%2Bcodon%2Busage%2Bin%2BDrosophila%2Bmelanogaster%253A%2Bnatural%2Bselection%2Band%2Btranslational%2Baccuracy.%26rft_id%253Dinfo%253Apmid%252F8005445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM2LzMvOTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.4" data-doi="10.1073/pnas.0635171100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arava</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide analysis of mrna translation profiles in saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">3889</span>–<span class="cit-lpage">3894</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0635171100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0635171100%26rft_id%253Dinfo%253Apmid%252F12660367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzcvMzg4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.5" data-doi="10.1038/nrg3963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baranov</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkins</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yordanova</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Augmented genetic decoding: global, local and temporal alterations of decoding processes and codon meaning.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D517%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3963%26rft_id%253Dinfo%253Apmid%252F26260261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26260261&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.6" data-doi="10.1093/nar/gkq1220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belew</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Advani</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Endogenous ribosomal frameshift signals operate as mrna destabilizing elements through at least two molecular pathways in yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">2799</span>–<span class="cit-lpage">2808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq1220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1220%26rft_id%253Dinfo%253Apmid%252F21109528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21109528&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289628400035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.7" data-doi="10.4161/15384101.2014.989123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belew</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cell cycle control (and more) by programmed -1 ribosomal frameshifting: implications for disease and therapeutics.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/15384101.2014.989123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D14%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.4161%252F15384101.2014.989123%26rft_id%253Dinfo%253Apmid%252F25584829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/15384101.2014.989123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25584829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.8" data-doi="10.1186/1471-2164-9-339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belew</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hepler</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Prfdb: a database of computationally predicted eukaryotic programmed -1 ribosomal frameshift signals.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-9-339</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBelew%26rft.auinit1%253DA.%2BT.%26rft.volume%253D9%26rft.spage%253D339%26rft.epage%253D339%26rft.atitle%253DPRFdb%253A%2Ba%2Bdatabase%2Bof%2Bcomputationally%2Bpredicted%2Beukaryotic%2Bprogrammed%2B-1%2Bribosomal%2Bframeshift%2Bsignals.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-339%26rft_id%253Dinfo%253Apmid%252F18637175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18637175&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.9" data-doi="10.1038/nature13429"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belew</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meskauskas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musalgaonkar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Advani</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulima</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ribosomal frameshifting in the ccr5 mrna is regulated by mirnas and the nmd pathway.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">512</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13429</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13429%26rft_id%253Dinfo%253Apmid%252F25043019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13429&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25043019&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340508200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.10" data-doi="10.1016/j.virusres.2005.10.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brierley</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dos Ramos</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Programmed ribosomal frameshifting in hiv-1 and the sars-cov.</span> <abbr class="cit-jnl-abbrev">Virus Res.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.virusres.2005.10.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DBrierley%26rft.auinit1%253DI.%26rft.volume%253D119%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D42%26rft.atitle%253DProgrammed%2Bribosomal%2Bframeshifting%2Bin%2BHIV-1%2Band%2Bthe%2BSARS-CoV.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2005.10.008%26rft_id%253Dinfo%253Apmid%252F16310880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2005.10.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16310880&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238302800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.11" data-doi="10.1016/j.cell.2014.04.041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caliskan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katunin</span>  <span class="cit-name-given-names">V. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belardinelli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peske</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodnina</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Programmed -1 frameshifting by kinetic partitioning during impeded translocation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1619</span>–<span class="cit-lpage">1631</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2014.04.041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1619%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.04.041%26rft_id%253Dinfo%253Apmid%252F24949973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.04.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24949973&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340941900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.12" data-doi="10.1016/j.tibs.2015.03.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caliskan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peske</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodnina</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Changed in translation: mrna recoding by -1 programmed ribosomal frameshifting.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibs.2015.03.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem.%2BSci.%26rft.volume%253D40%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2015.03.006%26rft_id%253Dinfo%253Apmid%252F25850333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2015.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25850333&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.13" data-doi="10.1093/bioinformatics/btg272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinovyev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Képès</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Codon adaptation index as a measure of dominating codon bias.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2005</span>–<span class="cit-lpage">2015</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg272</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DCarbone%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D16%26rft.spage%253D2005%26rft.epage%253D2015%26rft.atitle%253DCodon%2Badaptation%2Bindex%2Bas%2Ba%2Bmeasure%2Bof%2Bdominating%2Bcodon%2Bbias%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg272%26rft_id%253Dinfo%253Apmid%252F14594704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14594704&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186448900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.14" data-doi="10.1111/j.1420-9101.2004.00725.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlini</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Experimental reduction of codon bias in the drosophila alcohol dehydrogenase gene results in decreased ethanol tolerance of adult flies.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">785</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1420-9101.2004.00725.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DCarlini%26rft.auinit1%253DD.%2BB.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D779%26rft.epage%253D785%26rft.atitle%253DExperimental%2Breduction%2Bof%2Bcodon%2Bbias%2Bin%2Bthe%2BDrosophila%2Balcohol%2Bdehydrogenase%2Bgene%2Bresults%2Bin%2Bdecreased%2Bethanol%2Btolerance%2Bof%2Badult%2Bflies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2004.00725.x%26rft_id%253Dinfo%253Apmid%252F15271077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2004.00725.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15271077&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222052100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.15" data-doi="10.1007/s00239-005-0055-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlini</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genut</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Synonymous snps provide evidence for selective constraint on human exonic splicing enhancers.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">98</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-005-0055-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCarlini%26rft.auinit1%253DD.%2BB.%26rft.volume%253D62%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D98%26rft.atitle%253DSynonymous%2BSNPs%2Bprovide%2Bevidence%2Bfor%2Bselective%2Bconstraint%2Bon%2Bhuman%2Bexonic%2Bsplicing%2Benhancers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-005-0055-x%26rft_id%253Dinfo%253Apmid%252F16320116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-005-0055-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16320116&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235127000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlini</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">In vivo introduction of unpreferred synonymous codons into the drosophila adh gene results in reduced levels of adh protein.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCarlini%26rft.auinit1%253DD.%2BB.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D239%26rft.epage%253D243%26rft.atitle%253DIn%2BVivo%2BIntroduction%2Bof%2BUnpreferred%2BSynonymous%2BCodons%2BInto%2Bthe%2BDrosophila%2BAdh%2BGene%2BResults%2Bin%2BReduced%2BLevels%2Bof%2BADH%2BProtein%26rft_id%253Dinfo%253Apmid%252F12586711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMjM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.17" data-doi="10.1038/nrg1770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1770</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChamary%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D98%26rft.epage%253D108%26rft.atitle%253DHearing%2Bsilence%253A%2Bnon-neutral%2Bevolution%2Bat%2Bsynonymous%2Bsites%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1770%26rft_id%253Dinfo%253Apmid%252F16418745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418745&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.18" data-doi="10.1146/annurev.biochem.76.050106.093909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imam</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">M. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The nonsense-mediated decay rna surveillance pathway.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">74</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.biochem.76.050106.093909</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DChang%26rft.auinit1%253DY.%2BF.%26rft.volume%253D76%26rft.spage%253D51%26rft.epage%253D74%26rft.atitle%253DThe%2Bnonsense-mediated%2Bdecay%2BRNA%2Bsurveillance%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.76.050106.093909%26rft_id%253Dinfo%253Apmid%252F17352659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.76.050106.093909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17352659&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249336800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.19" data-doi="10.1073/pnas.85.18.6816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clare</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcourt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farabaugh</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Efficient translational frameshifting occurs within a conserved sequence of the overlap between the two genes of a yeast ty1 transposon.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">6816</span>–<span class="cit-lpage">6820</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.85.18.6816</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DClare%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D85%26rft.issue%253D18%26rft.spage%253D6816%26rft.epage%253D6820%26rft.atitle%253DEfficient%2Btranslational%2Bframeshifting%2Boccurs%2Bwithin%2Ba%2Bconserved%2Bsequence%2Bof%2Bthe%2Boverlap%2Bbetween%2Bthe%2Btwo%2Bgenes%2Bof%2Ba%2Byeast%2BTy1%2Btransposon.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.18.6816%26rft_id%253Dinfo%253Apmid%252F2842793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMTgvNjgxNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.20" data-doi="10.1007/s00792-012-0482-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cobucci-Ponzano</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moracci</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Translational recoding in archaea.</span> <abbr class="cit-jnl-abbrev">Extremophiles</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">803</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00792-012-0482-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExtremophiles%2B%2528Print%2529%26rft.stitle%253DExtremophiles%2B%2528Print%2529%26rft.aulast%253DCobucci-Ponzano%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D793%26rft.epage%253D803%26rft.atitle%253DTranslational%2Brecoding%2Bin%2Barchaea.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00792-012-0482-8%26rft_id%253Dinfo%253Apmid%252F23015064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00792-012-0482-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23015064&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.21" data-doi="10.1073/pnas.82.11.3616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Craigen</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caskey</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Bacterial peptide chain release factors: conserved primary structure and possible frameshift regulation of release factor 2.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">3616</span>–<span class="cit-lpage">3620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.82.11.3616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DCraigen%26rft.auinit1%253DW.%2BJ.%26rft.volume%253D82%26rft.issue%253D11%26rft.spage%253D3616%26rft.epage%253D3620%26rft.atitle%253DBacterial%2Bpeptide%2Bchain%2Brelease%2Bfactors%253A%2Bconserved%2Bprimary%2Bstructure%2Band%2Bpossible%2Bframeshift%2Bregulation%2Bof%2Brelease%2Bfactor%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.82.11.3616%26rft_id%253Dinfo%253Apmid%252F3889910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODIvMTEvMzYxNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Programmed ribosomal frameshifting goes beyond viruses: Organisms from all three kingdoms use frameshifting to regulate gene expression, perhaps signaling a paradigm shift.</span> <abbr class="cit-jnl-abbrev">Microbe Wash. D.C.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMicrobe%2BWash%2BDC%26rft.aulast%253DDinman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D1%26rft.issue%253D11%26rft.spage%253D521%26rft.epage%253D527%26rft.atitle%253DProgrammed%2BRibosomal%2BFrameshifting%2BGoes%2BBeyond%2BViruses%253A%2BOrganisms%2Bfrom%2Ball%2Bthree%2Bkingdoms%2Buse%2Bframeshifting%2Bto%2Bregulate%2Bgene%2Bexpression%252C%2Bperhaps%2Bsignaling%2Ba%2Bparadigm%2Bshift.%26rft_id%253Dinfo%253Apmid%252F17541450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17541450&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.23" data-doi="10.1016/B978-0-12-386497-0.00004-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Control of gene expression by translational recoding.</span> <abbr class="cit-jnl-abbrev">Adv. Protein Chem. Struct. Biol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-386497-0.00004-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAdv%2BProtein%2BChem%2BStruct%2BBiol%26rft.aulast%253DDinman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D86%26rft.spage%253D129%26rft.epage%253D149%26rft.atitle%253DControl%2Bof%2Bgene%2Bexpression%2Bby%2Btranslational%2Brecoding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-386497-0.00004-9%26rft_id%253Dinfo%253Apmid%252F22243583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-386497-0.00004-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22243583&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.24" data-doi="10.1002/wrna.1126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Mechanisms and implications of programmed translational frameshifting.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. RNA</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">673</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/wrna.1126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BRNA%26rft.volume%253D3%26rft.spage%253D661%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwrna.1126%26rft_id%253Dinfo%253Apmid%252F22715123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wrna.1126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22715123&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307731000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.25" data-doi="10.1002/jcc.20057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dirks</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An algorithm for computing nucleic acid base-pairing probabilities including pseudoknots.</span> <abbr class="cit-jnl-abbrev">J. Comput. Chem.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jcc.20057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bchemistry%26rft.stitle%253DJ%2BComput%2BChem%26rft.aulast%253DDirks%26rft.auinit1%253DR.%2BM.%26rft.volume%253D25%26rft.issue%253D10%26rft.spage%253D1295%26rft.epage%253D1304%26rft.atitle%253DAn%2Balgorithm%2Bfor%2Bcomputing%2Bnucleic%2Bacid%2Bbase-pairing%2Bprobabilities%2Bincluding%2Bpseudoknots.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjcc.20057%26rft_id%253Dinfo%253Apmid%252F15139042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jcc.20057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15139042&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222115500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.26" data-doi="10.1016/j.virol.2005.08.048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dulude</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berchiche</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendron</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brakier-Gingras</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heveker</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Decreasing the frameshift efficiency translates into an equivalent reduction of the replication of the human immunodeficiency virus type 1.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">136</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.virol.2005.08.048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DDulude%26rft.auinit1%253DD.%26rft.volume%253D345%26rft.issue%253D1%26rft.spage%253D127%26rft.epage%253D136%26rft.atitle%253DDecreasing%2Bthe%2Bframeshift%2Befficiency%2Btranslates%2Binto%2Ban%2Bequivalent%2Breduction%2Bof%2Bthe%2Breplication%2Bof%2Bthe%2Bhuman%2Bimmunodeficiency%2Bvirus%2Btype%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2005.08.048%26rft_id%253Dinfo%253Apmid%252F16256163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2005.08.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16256163&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235117000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.27" data-doi="10.1016/S0959-437X(02)00353-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evolution of synonymous codon usage in metazoans.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0959-437X(02)00353-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D640%26rft.epage%253D649%26rft.atitle%253DEvolution%2Bof%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bmetazoans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252802%252900353-2%26rft_id%253Dinfo%253Apmid%252F12433576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(02)00353-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12433576&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179003900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.28" data-doi="10.1002/yea.1150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Rohr</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonnet</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Limitations of codon adaptation index and other coding dna-based features for prediction of protein expression in saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1083</span>–<span class="cit-lpage">1093</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.1150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DFriberg%26rft.auinit1%253DM.%26rft.volume%253D21%26rft.issue%253D13%26rft.spage%253D1083%26rft.epage%253D1093%26rft.atitle%253DLimitations%2Bof%2Bcodon%2Badaptation%2Bindex%2Band%2Bother%2Bcoding%2BDNA-based%2Bfeatures%2Bfor%2Bprediction%2Bof%2Bprotein%2Bexpression%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1150%26rft_id%253Dinfo%253Apmid%252F15484285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15484285&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225001300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.29" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature02046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.30" data-doi="10.1016/j.tibtech.2004.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindarajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minshull</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Codon bias and heterologous protein expression.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">353</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibtech.2004.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiotechnology%26rft.stitle%253DTrends%2BBiotechnol%26rft.aulast%253DGustafsson%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D346%26rft.epage%253D353%26rft.atitle%253DCodon%2Bbias%2Band%2Bheterologous%2Bprotein%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2004.04.006%26rft_id%253Dinfo%253Apmid%252F15245907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2004.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15245907&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222712000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.31" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.42.110807.091442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.32" data-doi="10.1371/journal.pgen.1000556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">General rules for optimal codon choice.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000556</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000556</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000556%26rft.epage%253De1000556%26rft.atitle%253DGeneral%2Brules%2Bfor%2Boptimal%2Bcodon%2Bchoice.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000556%26rft_id%253Dinfo%253Apmid%252F19593368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593368&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.33" data-doi="10.1074/jbc.C200101200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Transcript abundance in yeast varies over six orders of magnitude.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">14363</span>–<span class="cit-lpage">14366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.C200101200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C200101200%26rft_id%253Dinfo%253Apmid%252F11882647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvMTcvMTQzNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.34" data-doi="10.1016/S0092-8674(00)81641-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holstege</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jennings</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyrick</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hengartner</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Dissecting the regulatory circuitry of a eukaryotic genome.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">728</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)81641-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHolstege%26rft.auinit1%253DF.%2BC.%26rft.volume%253D95%26rft.issue%253D5%26rft.spage%253D717%26rft.epage%253D728%26rft.atitle%253DDissecting%2Bthe%2Bregulatory%2Bcircuitry%2Bof%2Ba%2Beukaryotic%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981641-4%26rft_id%253Dinfo%253Apmid%252F9845373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81641-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9845373&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077253700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.35" data-doi="10.1007/978-1-60327-411-1_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">Z. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambudkar</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottesman</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Silent (synonymous) snps: should we care about them?</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">578</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-60327-411-1_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DHunt%26rft.auinit1%253DR.%26rft.volume%253D578%26rft.spage%253D23%26rft.epage%253D39%26rft.atitle%253DSilent%2B%2528synonymous%2529%2BSNPs%253A%2Bshould%2Bwe%2Bcare%2Babout%2Bthem%253F%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-411-1_2%26rft_id%253Dinfo%253Apmid%252F19768585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-411-1_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19768585&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274946200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.36" data-doi="10.1016/0022-2836(81)90003-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Correlation between the abundance of escherichia coli transfer rnas and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the e. coli translational system.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">409</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2836(81)90003-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D389%26rft.epage%253D409%26rft.atitle%253DCorrelation%2Bbetween%2Bthe%2Babundance%2Bof%2BEscherichia%2Bcoli%2Btransfer%2BRNAs%2Band%2Bthe%2Boccurrence%2Bof%2Bthe%2Brespective%2Bcodons%2Bin%2Bits%2Bprotein%2Bgenes%253A%2Ba%2Bproposal%2Bfor%2Ba%2Bsynonymous%2Bcodon%2Bchoice%2Bthat%2Bis%2Boptimal%2Bfor%2Bthe%2BE.%2Bcoli%2Btranslational%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990003-6%26rft_id%253Dinfo%253Apmid%252F6175758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90003-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6175758&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MJ92600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.37" data-doi="10.1016/0022-2836(82)90250-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Correlation between the abundance of yeast transfer rnas and the occurrence of the respective codons in protein genes. differences in synonymous codon choice patterns of yeast and escherichia coli with reference to the abundance of isoaccepting transfer rnas.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">597</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2836(82)90250-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DIkemura%26rft.auinit1%253DT.%26rft.volume%253D158%26rft.issue%253D4%26rft.spage%253D573%26rft.epage%253D597%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252882%252990250-9%26rft_id%253Dinfo%253Apmid%252F6750137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(82)90250-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6750137&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982NY20900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Codon usage and trna content in unicellular and multicellular organisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3916708&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AAA8800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.39" data-doi="10.1016/S1357-2725(96)00161-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivanov</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saraffova</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouhaidar</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Unusual effect of clusters of rare arginine (agg) codons on the expression of human interferon alpha 1 gene in escherichia coli.</span> <abbr class="cit-jnl-abbrev">Int. J. Biochem. Cell Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1357-2725(96)00161-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Binternational%2Bjournal%2Bof%2Bbiochemistry%2B%2526%2Bcell%2Bbiology%26rft.stitle%253DInt%2BJ%2BBiochem%2BCell%2BBiol%26rft.aulast%253DIvanov%26rft.auinit1%253DI.%2BG.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D659%26rft.epage%253D666%26rft.atitle%253DUnusual%2Beffect%2Bof%2Bclusters%2Bof%2Brare%2Barginine%2B%2528AGG%2529%2Bcodons%2Bon%2Bthe%2Bexpression%2Bof%2Bhuman%2Binterferon%2Balpha%2B1%2Bgene%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1357-2725%252896%252900161-6%26rft_id%253Dinfo%253Apmid%252F9363643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1357-2725(96)00161-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9363643&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XF92100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.40" data-doi="10.1126/science.2416054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacks</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varmus</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Expression of the rous sarcoma virus pol gene by ribosomal frameshifting.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">230</span>: <span class="cit-fpage">1237</span>–<span class="cit-lpage">1242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.2416054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJacks%26rft.auinit1%253DT%26rft.volume%253D230%26rft.issue%253D4731%26rft.spage%253D1237%26rft.epage%253D1242%26rft.atitle%253DExpression%2Bof%2Bthe%2BRous%2Bsarcoma%2Bvirus%2Bpol%2Bgene%2Bby%2Bribosomal%2Bframeshifting%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2416054%26rft_id%253Dinfo%253Apmid%252F2416054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMzAvNDczMS8xMjM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.41" data-doi="10.1093/nar/gkg306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussemaker</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Revisiting the codon adaptation index from a whole-genome perspective: analyzing the relationship between gene expression and codon occurrence in yeast using a variety of models.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2242</span>–<span class="cit-lpage">2251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg306</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DJansen%26rft.auinit1%253DR.%26rft.volume%253D31%26rft.issue%253D8%26rft.spage%253D2242%26rft.epage%253D2251%26rft.atitle%253DRevisiting%2Bthe%2Bcodon%2Badaptation%2Bindex%2Bfrom%2Ba%2Bwhole-genome%2Bperspective%253A%2Banalyzing%2Bthe%2Brelationship%2Bbetween%2Bgene%2Bexpression%2Band%2Bcodon%2Boccurrence%2Bin%2Byeast%2Busing%2Ba%2Bvariety%2Bof%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg306%26rft_id%253Dinfo%253Apmid%252F12682375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12682375&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182161400029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.42" data-doi="10.1073/pnas.96.4.1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jelinsky</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samson</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Global response of saccharomyces cerevisiae to an alkylating agent.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1486</span>–<span class="cit-lpage">1491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.96.4.1486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DJelinsky%26rft.auinit1%253DS.%2BA.%26rft.volume%253D96%26rft.issue%253D4%26rft.spage%253D1486%26rft.epage%253D1491%26rft.atitle%253DGlobal%2Bresponse%2Bof%2BSaccharomyces%2Bcerevisiae%2Bto%2Ban%2Balkylating%2Bagent%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.4.1486%26rft_id%253Dinfo%253Apmid%252F9990050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni80LzE0ODYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.43" data-doi="10.1007/s002390010219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanaya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinouchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Codon usage and trna genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with cg-dinucleotide usage as assessed by multivariate analysis.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">298</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s002390010219</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKanaya%26rft.auinit1%253DS.%26rft.volume%253D53%26rft.issue%253D4-5%26rft.spage%253D290%26rft.epage%253D298%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bgenes%2Bin%2Beukaryotes%253A%2Bcorrelation%2Bof%2Bcodon%2Busage%2Bdiversity%2Bwith%2Btranslation%2Befficiency%2Band%2Bwith%2BCG-dinucleotide%2Busage%2Bas%2Bassessed%2Bby%2Bmultivariate%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002390010219%26rft_id%253Dinfo%253Apmid%252F11675589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002390010219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11675589&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171169600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.44" data-doi="10.1016/S0378-1119(99)00225-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanaya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Studies of codon usage and trna genes of 18 unicellular organisms and quantification of bacillus subtilis trnas: gene expression level and species-specific diversity of codon usage based on multivariate analysis.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">238</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0378-1119(99)00225-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DKanaya%26rft.auinit1%253DS.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D143%26rft.epage%253D155%26rft.atitle%253DStudies%2Bof%2Bcodon%2Busage%2Band%2BtRNA%2Bgenes%2Bof%2B18%2Bunicellular%2Borganisms%2Band%2Bquantification%2Bof%2BBacillus%2Bsubtilis%2BtRNAs%253A%2Bgene%2Bexpression%2Blevel%2Band%2Bspecies-specific%2Bdiversity%2Bof%2Bcodon%2Busage%2Bbased%2Bon%2Bmultivariate%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900225-5%26rft_id%253Dinfo%253Apmid%252F10570992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00225-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10570992&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083005400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.45" data-doi="10.3389/fgene.2012.00242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ketteler</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">On programmed ribosomal frameshifting: the alternative proteomes.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2012.00242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DKetteler%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.spage%253D242%26rft.epage%253D242%26rft.atitle%253DOn%2Bprogrammed%2Bribosomal%2Bframeshifting%253A%2Bthe%2Balternative%2Bproteomes.%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2012.00242%26rft_id%253Dinfo%253Apmid%252F23181069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2012.00242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23181069&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.46" data-doi="10.1126/science.1135308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">I.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">Z. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calcagno</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A ”silent” polymorphism in the mdr1 gene changes substrate specificity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">315</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1135308</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1135308%26rft_id%253Dinfo%253Apmid%252F17185560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTUvNTgxMS81MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxNzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.47" data-doi="10.1007/s00239-003-2459-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kliman</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irving</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santiago</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Selection conflicts, gene expression, and codon usage trends in yeast.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-003-2459-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DKliman%26rft.auinit1%253DR.%2BM.%26rft.volume%253D57%26rft.issue%253D1%26rft.spage%253D98%26rft.epage%253D109%26rft.atitle%253DSelection%2Bconflicts%252C%2Bgene%2Bexpression%252C%2Band%2Bcodon%2Busage%2Btrends%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-003-2459-9%26rft_id%253Dinfo%253Apmid%252F12962310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-003-2459-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12962310&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183723700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.48" data-doi="10.2217/14622416.8.8.1075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komar</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Silent snps: impact on gene function and phenotype.</span> <abbr class="cit-jnl-abbrev">Pharmacogenomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1080</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2217/14622416.8.8.1075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacogenomics%26rft.stitle%253DPharmacogenomics%26rft.aulast%253DKomar%26rft.auinit1%253DA.%2BA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D1075%26rft.epage%253D1080%26rft.atitle%253DSilent%2BSNPs%253A%2Bimpact%2Bon%2Bgene%2Bfunction%2Band%2Bphenotype.%26rft_id%253Dinfo%253Adoi%252F10.2217%252F14622416.8.8.1075%26rft_id%253Dinfo%253Apmid%252F17716239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/14622416.8.8.1075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17716239&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249355700024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.49" data-doi="10.1016/j.tibs.2008.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komar</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A pause for thought along the co-translational folding pathway.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibs.2008.10.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DKomar%26rft.auinit1%253DA.%2BA.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DA%2Bpause%2Bfor%2Bthought%2Balong%2Bthe%2Bco-translational%2Bfolding%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2008.10.002%26rft_id%253Dinfo%253Apmid%252F18996013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2008.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18996013&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263006000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.50" data-doi="10.1038/nbt1270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt1270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLu%26rft.auinit1%253DP.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D117%26rft.epage%253D124%26rft.atitle%253DAbsolute%2Bprotein%2Bexpression%2Bprofiling%2Bestimates%2Bthe%2Brelative%2Bcontributions%2Bof%2Btranscriptional%2Band%2Btranslational%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1270%26rft_id%253Dinfo%253Apmid%252F17187058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17187058&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243491000037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.51" data-doi="10.1007/BF00039391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maia</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Séron</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haenni</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardi</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Gene expression from viral rna genomes.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">391</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00039391</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DMaia%26rft.auinit1%253DI.%2BG.%26rft.volume%253D32%26rft.issue%253D1-2%26rft.spage%253D367%26rft.epage%253D391%26rft.atitle%253DGene%2Bexpression%2Bfrom%2Bviral%2BRNA%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00039391%26rft_id%253Dinfo%253Apmid%252F8980488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00039391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8980488&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VY81900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.52" data-doi="10.1093/nar/gkh004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakao</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshihama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenmochi</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Rpg: the ribosomal protein gene database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">D168</span>–<span class="cit-lpage">D170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNakao%26rft.auinit1%253DA.%26rft.volume%253D32%26rft.issue%253D90001%26rft.spage%253DD168%26rft.epage%253D170%26rft.atitle%253DRPG%253A%2Bthe%2BRibosomal%2BProtein%2BGene%2Bdatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh004%26rft_id%253Dinfo%253Apmid%252F14681386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14681386&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188079000038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.53" data-doi="10.1038/nature04785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ihmels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breslow</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Single-cell proteomic analysis of s. cerevisiae reveals the architecture of biological noise.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">441</span>: <span class="cit-fpage">840</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature04785</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNewman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D441%26rft.issue%253D7095%26rft.spage%253D840%26rft.epage%253D846%26rft.atitle%253DSingle-cell%2Bproteomic%2Banalysis%2Bof%2BS.%2Bcerevisiae%2Breveals%2Bthe%2Barchitecture%2Bof%2Bbiological%2Bnoise.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04785%26rft_id%253Dinfo%253Apmid%252F16699522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16699522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238254100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.54" data-doi="10.1093/molbev/msj035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evidence for purifying selection against synonymous mutations in mammalian exonic splicing enhancers.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">309</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msj035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj035%26rft_id%253Dinfo%253Apmid%252F16221894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msj035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16221894&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234718800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.55" data-doi="10.1002/bies.20592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parmley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">How do synonymous mutations affect fitness?</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">519</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.20592</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DParmley%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D29%26rft.issue%253D6%26rft.spage%253D515%26rft.epage%253D519%26rft.atitle%253DHow%2Bdo%2Bsynonymous%2Bmutations%2Baffect%2Bfitness%253F%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20592%26rft_id%253Dinfo%253Apmid%252F17508390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17508390&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246868300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.56" data-doi="10.1261/rna.2225206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plant</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comparative study of the effects of heptameric slippery site composition on -1 frameshifting among different eukaryotic systems.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">666</span>–<span class="cit-lpage">673</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.2225206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2225206%26rft_id%253Dinfo%253Apmid%252F16497657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.57" data-doi="10.1093/nar/gkh256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plant</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinman</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A programmed -1 ribosomal frameshift signal can function as a cis-acting mrna destabilizing element.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">784</span>–<span class="cit-lpage">790</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPlant%26rft.auinit1%253DE.%2BP.%26rft.volume%253D32%26rft.issue%253D2%26rft.spage%253D784%26rft.epage%253D790%26rft.atitle%253DA%2Bprogrammed%2B-1%2Bribosomal%2Bframeshift%2Bsignal%2Bcan%2Bfunction%2Bas%2Ba%2Bcis-acting%2BmRNA%2Bdestabilizing%2Belement%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh256%26rft_id%253Dinfo%253Apmid%252F14762205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14762205&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188989000051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.58" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2899</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.59" data-doi="10.1016/j.molcel.2015.05.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quax</span>  <span class="cit-name-given-names">T. E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claassens</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Söll</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Oost</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Codon bias as a means to fine-tune gene expression.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2015.05.035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D59%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.05.035%26rft_id%253Dinfo%253Apmid%252F26186290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.05.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26186290&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.60" data-doi="10.1093/molbev/msm100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resch</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mariño-Ramírez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogurtsov</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalina</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Widespread positive selection in synonymous sites of mammalian genes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1821</span>–<span class="cit-lpage">1831</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm100%26rft_id%253Dinfo%253Apmid%252F17522087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17522087&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.61" data-doi="10.1186/1471-2164-10-162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resch</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogurtsov</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalina</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of alternative and constitutive regions of mammalian 5′utrs.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-10-162</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DResch%26rft.auinit1%253DA.%2BM.%26rft.volume%253D10%26rft.spage%253D162%26rft.epage%253D162%26rft.atitle%253DEvolution%2Bof%2Balternative%2Band%2Bconstitutive%2Bregions%2Bof%2Bmammalian%2B5%2527UTRs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-162%26rft_id%253Dinfo%253Apmid%252F19371439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19371439&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3173.62" data-doi="10.1093/acprof:osobl/9780199601165.003.0013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anisimova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cannarozzi</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>, pp. <span class="cit-fpage">189</span>–<span class="cit-lpage">217</span> in <span class="cit-source">Measuring codon usage bias. Codon evolution: mechanisms and models</span>, <span class="cit-publ-name">Oxford University Press Inc</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/acprof:osobl/9780199601165.003.0013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMeasuring%2Bcodon%2Busage%2Bbias.%2BCodon%2Bevolution%253A%2Bmechanisms%2Band%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Facprof%253Aosobl%252F9780199601165.003.0013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/acprof:osobl/9780199601165.003.0013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.63" data-doi="10.1038/nbt1098-939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estep</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Finding dna regulatory motifs within unaligned noncoding sequences clustered by whole-genome mrna quantitation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">939</span>–<span class="cit-lpage">945</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt1098-939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRoth%26rft.auinit1%253DF.%2BP.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D939%26rft.epage%253D945%26rft.atitle%253DFinding%2BDNA%2Bregulatory%2Bmotifs%2Bwithin%2Bunaligned%2Bnoncoding%2Bsequences%2Bclustered%2Bby%2Bwhole-genome%2BmRNA%2Bquantitation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1098-939%26rft_id%253Dinfo%253Apmid%252F9788350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1098-939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9788350&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076323300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.64" data-doi="10.1007/BF02099948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02099948</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D24%26rft.issue%253D1-2%26rft.spage%253D28%26rft.epage%253D38%26rft.atitle%253DAn%2Bevolutionary%2Bperspective%2Bon%2Bsynonymous%2Bcodon%2Busage%2Bin%2Bunicellular%2Borganisms.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02099948%26rft_id%253Dinfo%253Apmid%252F3104616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02099948&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3104616&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986F784400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.65" data-doi="10.1093/nar/15.3.1281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The codon adaptation index–a measure of directional synonymous codon usage bias, and its potential applications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/15.3.1281</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D1281%26rft.epage%253D1295%26rft.atitle%253DThe%2Bcodon%2Badaptation%2Bindex-a%2Bmeasure%2Bof%2Bdirectional%2Bsynonymous%2Bcodon%2Busage%2Bbias%252C%2Band%2Bits%2Bpotential%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F15.3.1281%26rft_id%253Dinfo%253Apmid%252F3547335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/15.3.1281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3547335&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987F976100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.66" data-doi="10.1093/molbev/msl166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoletzki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synonymous codon usage in escherichia coli: selection for translational accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">381</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msl166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl166%26rft_id%253Dinfo%253Apmid%252F17101719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17101719&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243993100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.67" data-doi="10.1007/s00239-015-9714-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Supek</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The code of silence: Widespread associations between synonymous codon biases and gene function.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-015-9714-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BEvol.%26rft.volume%253D82%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-015-9714-8%26rft_id%253Dinfo%253Apmid%252F26538122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-015-9714-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26538122&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.68" data-doi="10.1016/j.jmb.2008.08.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">C.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauna</span>  <span class="cit-name-given-names">Z. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimchi-Sarfaty</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambudkar</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottesman</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Synonymous mutations and ribosome stalling can lead to altered folding pathways and distinct minima.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">383</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2008.08.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DTsai%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D383%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D291%26rft.atitle%253DSynonymous%2Bmutations%2Band%2Bribosome%2Bstalling%2Bcan%2Blead%2Bto%2Baltered%2Bfolding%2Bpathways%2Band%2Bdistinct%2Bminima.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2008.08.012%26rft_id%253Dinfo%253Apmid%252F18722384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2008.08.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18722384&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260665100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.69" data-doi="10.1073/pnas.0909910107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldman</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruppin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translation efficiency is determined by both codon bias and folding energy.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">3645</span>–<span class="cit-lpage">3650</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0909910107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909910107%26rft_id%253Dinfo%253Apmid%252F20133581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzgvMzY0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.70" data-doi="10.1093/nar/gku1313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zur</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multiple roles of the coding sequence 5′ end in gene expression regulation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1313%26rft_id%253Dinfo%253Apmid%252F25505165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25505165&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.71" data-doi="10.1016/0022-2836(84)90027-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varenne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buc</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloubes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazdunski</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Translation is a non-uniform process. effect of trna availability on the rate of elongation of nascent polypeptide chains.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2836(84)90027-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DVarenne%26rft.auinit1%253DS.%26rft.volume%253D180%26rft.issue%253D3%26rft.spage%253D549%26rft.epage%253D576%26rft.atitle%253DTranslation%2Bis%2Ba%2Bnon-uniform%2Bprocess.%2BEffect%2Bof%2BtRNA%2Bavailability%2Bon%2Bthe%2Brate%2Bof%2Belongation%2Bof%2Bnascent%2Bpolypeptide%2Bchains.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252884%252990027-5%26rft_id%253Dinfo%253Apmid%252F6084718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(84)90027-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6084718&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984AAV3300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.72" data-doi="10.1186/1752-0509-2-87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von der Haar</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A quantitative estimation of the global translational activity in logarithmically growing yeast cells.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1752-0509-2-87</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253Dvon%2Bder%2BHaar%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.spage%253D87%26rft.epage%253D87%26rft.atitle%253DA%2Bquantitative%2Bestimation%2Bof%2Bthe%2Bglobal%2Btranslational%2Bactivity%2Bin%2Blogarithmically%2Bgrowing%2Byeast%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-2-87%26rft_id%253Dinfo%253Apmid%252F18925958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-2-87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18925958&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.73" data-doi="10.1093/nar/19.22.6119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andachi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muto</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikemura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osawa</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Levels of trnas in bacterial cells as affected by amino acid usage in proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">6119</span>–<span class="cit-lpage">6122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/19.22.6119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DYamao%26rft.auinit1%253DF.%26rft.volume%253D19%26rft.issue%253D22%26rft.spage%253D6119%26rft.epage%253D6122%26rft.atitle%253DLevels%2Bof%2BtRNAs%2Bin%2Bbacterial%2Bcells%2Bas%2Baffected%2Bby%2Bamino%2Bacid%2Busage%2Bin%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F19.22.6119%26rft_id%253Dinfo%253Apmid%252F1956771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/19.22.6119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1956771&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3173.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GR78500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3173.74" data-doi="10.1126/science.1191701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalina</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashina</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential arginylation of actin isoforms is regulated by coding sequence-dependent degradation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">1534</span>–<span class="cit-lpage">1537</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1191701</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZhang%26rft.auinit1%253DF.%26rft.volume%253D329%26rft.issue%253D5998%26rft.spage%253D1534%26rft.epage%253D1537%26rft.atitle%253DDifferential%2BArginylation%2Bof%2BActin%2BIsoforms%2BIs%2BRegulated%2Bby%2BCoding%2BSequence-Dependent%2BDegradation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1191701%26rft_id%253Dinfo%253Apmid%252F20847274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjkvNTk5OC8xNTM0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTczLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3173.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3163" title="Double Selection Enhances the Efficiency of Target-AID and Cas9-Based Genome Editing in Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3185" title="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3173" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DBxIXj7OQVmh0AK7RTXTkA-N42tqSb6cBoxwsizEwWc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3173" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3173" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3173" class="active" data-icon-position="" data-hide-link-title="0">Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490980" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Is2TVwJA1IdRFlwtCAkRle7AST28-8WpVQLUjN2sn9E" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3173.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3173" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-saAbM8Pxmn8aO-JXvVillXTB7hyCqQEL9G-uON-5WJE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3173" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3173.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490980" id="node490980--2" data-pisa="ggg;8/10/3173" data-pisa-master="ggg;g3.118.200185" data-apath="/ggg/8/10/3173.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6667-7098" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Victor Garcia</span>, <span class="highwire-citation-author" data-delta="1">Stefan Zoller</span> and <span class="highwire-citation-author" data-delta="2">Maria Anisimova</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3173-3183;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200185 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490980--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victor Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Zurich University of Applied Sciences, Wädenswil, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictor%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6667-7098" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Victor Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:victor.garcia_palencia@alumni.ethz.ch" class="" data-icon-position="" data-hide-link-title="0">victor.garcia_palencia@alumni.ethz.ch</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stefan Zoller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>ETH Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BZoller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zoller%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BZoller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maria Anisimova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Zurich University of Applied Sciences, Wädenswil, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BAnisimova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anisimova%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BAnisimova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3173" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3173&amp;title=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3173&amp;title=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3173&amp;title=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3173&amp;text=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3173&amp;title=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3173&amp;t=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3173&amp;title=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3173&amp;title=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490980/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490980" id="node490980--3" data-pisa="ggg;8/10/3173" data-pisa-master="ggg;g3.118.200185" data-apath="/ggg/8/10/3173.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6667-7098" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Victor Garcia</span>, <span class="highwire-citation-author" data-delta="1">Stefan Zoller</span> and <span class="highwire-citation-author" data-delta="2">Maria Anisimova</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3173-3183;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200185 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490980--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victor Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Zurich University of Applied Sciences, Wädenswil, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictor%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6667-7098" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Victor Garcia</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:victor.garcia_palencia@alumni.ethz.ch" class="" data-icon-position="" data-hide-link-title="0">victor.garcia_palencia@alumni.ethz.ch</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stefan Zoller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>ETH Zurich, Zurich, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stefan%2BZoller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zoller%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStefan%2BZoller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maria Anisimova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Zurich University of Applied Sciences, Wädenswil, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Swiss Institute of Bioinformatics, Lausanne, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BAnisimova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anisimova%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BAnisimova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3173&amp;rft.epage=3183&amp;rft.atitle=Accounting%20for%20Programmed%20Ribosomal%20Frameshifting%20in%20the%20Computation%20of%20Codon%20Usage%20Bias%20Indices&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Garcia%2C+Victor&amp;rft.au=Zoller%2C+Stefan&amp;rft.au=Anisimova%2C+Maria"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490980/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490980/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490980/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490980/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490980/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490980/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490980/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490980/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490980/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490980/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490980/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490980/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3173" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3173.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3173.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/10/3173.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/10/3173.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
