<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3311" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3311.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3311.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3311.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3311.atom" />
<meta name="HW.pisa" content="ggg;8/10/3311" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases" />
<meta name="DC.Identifier" content="10.1534/g3.118.200656" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Arnesen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6949037" />
<meta name="DC.Description" content="Insect-pathogenic fungi use subtilisin-like serine proteases (SLSPs) to degrade chitin-associated proteins in the insect procuticle. Most insect-pathogenic fungi in the order Hypocreales (Ascomycota) are generalist species with a broad host-range, and most species possess a high number of SLSPs. The other major clade of insect-pathogenic fungi is part of the subphylum Entomophthoromycotina (Zoopagomycota, formerly Zygomycota) which consists of high host-specificity insect-pathogenic fungi that naturally only infect a single or very few host species. The extent to which insect-pathogenic fungi in the order Entomophthorales rely on SLSPs is unknown. Here we take advantage of recently available transcriptomic and genomic datasets from four genera within Entomophthoromycotina: the saprobic or opportunistic pathogens Basidiobolus meristosporus , Conidiobolus coronatus , C. thromboides , C. incongruus , and the host-specific insect pathogens Entomophthora muscae and Pandora formicae , specific pathogens of house flies ( Muscae domestica ) and wood ants ( Formica polyctena ), respectively. In total 154 SLSP from six fungi in the subphylum Entomophthoromycotina were identified: E. muscae (n = 22), P. formicae (n = 6), B. meristosporus (n = 60), C. thromboides (n = 18), C. coronatus (n = 36), and C. incongruus (n = 12). A unique group of 11 SLSPs was discovered in the genomes of the obligate biotrophic fungi E. muscae , P. formicae and the saprobic human pathogen C. incongruus that loosely resembles bacillopeptidase F-like SLSPs. Phylogenetics and protein domain analysis show this class represents a unique group of SLSPs so far only observed among Bacteria, Oomycetes and early diverging fungi such as Cryptomycota, Microsporidia, and Entomophthoromycotina. This group of SLSPs is missing in the sister fungal lineages of Kickxellomycotina and the fungal phyla Mucoromyocta, Ascomycota and Basidiomycota fungi suggesting interesting gene loss patterns." />
<meta name="DC.Contributor" content="Jonathan A. Arnesen" />
<meta name="DC.Contributor" content="Joanna Małagocka" />
<meta name="DC.Contributor" content="Andrii Gryganskyi" />
<meta name="DC.Contributor" content="Igor V. Grigoriev" />
<meta name="DC.Contributor" content="Kerstin Voigt" />
<meta name="DC.Contributor" content="Jason E. Stajich" />
<meta name="DC.Contributor" content="Henrik H. De Fine Licht" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Insect-pathogenic fungi use subtilisin-like serine proteases (SLSPs) to degrade chitin-associated proteins in the insect procuticle. Most insect-pathogenic fungi in the order Hypocreales (Ascomycota) are generalist species with a broad host-range, and most species possess a high number of SLSPs. The other major clade of insect-pathogenic fungi is part of the subphylum Entomophthoromycotina (Zoopagomycota, formerly Zygomycota) which consists of high host-specificity insect-pathogenic fungi that naturally only infect a single or very few host species. The extent to which insect-pathogenic fungi in the order Entomophthorales rely on SLSPs is unknown. Here we take advantage of recently available transcriptomic and genomic datasets from four genera within Entomophthoromycotina: the saprobic or opportunistic pathogens &lt;i&gt;Basidiobolus meristosporus&lt;/i&gt;, &lt;i&gt;Conidiobolus coronatus&lt;/i&gt;, &lt;i&gt;C. thromboides&lt;/i&gt;, &lt;i&gt;C. incongruus&lt;/i&gt;, and the host-specific insect pathogens &lt;i&gt;Entomophthora muscae&lt;/i&gt; and &lt;i&gt;Pandora formicae&lt;/i&gt;, specific pathogens of house flies (&lt;i&gt;Muscae domestica&lt;/i&gt;) and wood ants (&lt;i&gt;Formica polyctena&lt;/i&gt;), respectively. In total 154 SLSP from six fungi in the subphylum Entomophthoromycotina were identified: &lt;i&gt;E. muscae&lt;/i&gt; (n = 22), &lt;i&gt;P. formicae&lt;/i&gt; (n = 6), &lt;i&gt;B. meristosporus&lt;/i&gt; (n = 60), &lt;i&gt;C. thromboides&lt;/i&gt; (n = 18), &lt;i&gt;C. coronatus&lt;/i&gt; (n = 36), and &lt;i&gt;C. incongruus&lt;/i&gt; (n = 12). A unique group of 11 SLSPs was discovered in the genomes of the obligate biotrophic fungi &lt;i&gt;E. muscae&lt;/i&gt;, &lt;i&gt;P. formicae&lt;/i&gt; and the saprobic human pathogen &lt;i&gt;C. incongruus&lt;/i&gt; that loosely resembles bacillopeptidase F-like SLSPs. Phylogenetics and protein domain analysis show this class represents a unique group of SLSPs so far only observed among Bacteria, Oomycetes and early diverging fungi such as Cryptomycota, Microsporidia, and Entomophthoromycotina. This group of SLSPs is missing in the sister fungal lineages of Kickxellomycotina and the fungal phyla Mucoromyocta, Ascomycota and Basidiomycota fungi suggesting interesting gene loss patterns.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3311" />
<meta name="citation_id" content="8/10/3311" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3311" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3311.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3311.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3311.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200656" />
<meta name="citation_pmid" content="30111619" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3311" />
<meta name="citation_lastpage" content="3319" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jonathan A. Arnesen" />
<meta name="citation_author_institution" content="Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark" />
<meta name="citation_author" content="Joanna Małagocka" />
<meta name="citation_author_institution" content="Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark" />
<meta name="citation_author" content="Andrii Gryganskyi" />
<meta name="citation_author_institution" content="Department of Biology, Duke University, Durham, North Carolina, USA" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6037-0092" />
<meta name="citation_author" content="Igor V. Grigoriev" />
<meta name="citation_author_institution" content="US Department of Energy Joint Genome Institute, Walnut Creek, California, USA" />
<meta name="citation_author" content="Kerstin Voigt" />
<meta name="citation_author_institution" content="Jena Microbial Resource Collection (JMRC), Leibniz Institute for Natural Product Research and Infection Biology - Hans Knoell Institute, Adolf-Reichwein-Str.23, 07745 Jena, Germany" />
<meta name="citation_author_institution" content="Institute of Microbiology, Friedrich Schiller University, Neugasse 25, 07743 Jena, Germany" />
<meta name="citation_author" content="Jason E. Stajich" />
<meta name="citation_author_institution" content="Department of Plant Pathology and Microbiology, University of California, Riverside, California" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7591-0020" />
<meta name="citation_author" content="Henrik H. De Fine Licht" />
<meta name="citation_author_institution" content="Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark" />
<meta name="citation_author_email" content="hhdefinelicht@plen.ku.dk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3326-5729" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=S. Bagga;citation_author=G. Hu;citation_author=S. E. Screen;citation_author=R. J. St. Leger;citation_title=Reconstructing the diversification of subtilisins in the pathogenic fungus Metarhizium anisopliae.;citation_pages=159-169;citation_volume=324;citation_year=2004;citation_pmid=14693381;citation_doi=10.1016/j.gene.2003.09.031" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Entomol.;citation_author=J. J. Boomsma;citation_author=A. B. Jensen;citation_author=N. V. Meyling;citation_author=J. Eilenberg;citation_title=Evolutionary interaction networks of insect pathogenic fungi.;citation_pages=467-485;citation_volume=59;citation_year=2014;citation_pmid=24160418;citation_doi=10.1146/annurev-ento-011613-162054" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Burmester;citation_author=E. Shelest;citation_author=G. Glöckner;citation_author=C. Heddergott;citation_author=S. Schindler;citation_title=Comparative and functional genomics provide insights into the pathogenicity of dermatophytic fungi.;citation_pages=R7-R7;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21247460;citation_doi=10.1186/gb-2011-12-1-r7" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=Y. Chang;citation_author=S. Wang;citation_author=S. Sekimoto;citation_author=A. Aerts;citation_author=C. Choi;citation_title=Phylogenomic analyses indicate that early fungi evolved digesting cell walls of algal ancestors of land plants.;citation_volume=7;citation_year=2015;citation_pmid=25977457;citation_doi=10.1093/gbe/evv090" />
<meta name="citation_reference" content="citation_journal_title=Adv. Bot. Res.;citation_author=A. K. Charnley;citation_title=Fungal pathogens of insects: Cuticle degrading enzymes and toxins.;citation_pages=241-321;citation_volume=40;citation_year=2003;citation_doi=10.1016/S0065-2296(05)40006-3" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=M. C. Chibucos;citation_author=S. Soliman;citation_author=T. Gebremariam;citation_author=H. Lee;citation_author=S. Daugherty;citation_title=An integrated genomic and transcriptomic survey of mucormycosis-causing fungi.;citation_pages=12218;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms12218" />
<meta name="citation_reference" content="citation_journal_title=Adv. Genet.;citation_author=H. H. De Fine Licht;citation_author=A. E. Hajek;citation_author=J. Eilenberg;citation_author=A. B. Jensen;citation_title=Utilizing genomics to study entomopathogenicity in the fungal phylum entomophthoromycota: A review of current genetic resources.;citation_pages=41-65;citation_volume=94;citation_year=2016;citation_doi=10.1016/bs.adgen.2016.01.003" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=H. H. De Fine Licht;citation_author=A. B. Jensen;citation_author=J. Eilenberg;citation_title=Comparative transcriptomics reveal host-specific nucleotide variation in entomophthoralean fungi.;citation_pages=2092-2110;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.13863" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. A. Desjardins;citation_author=M. D. Champion;citation_author=J. W. Holder;citation_author=A. Muszewska;citation_author=J. Goldberg;citation_title=Comparative genomic analysis of human fungal pathogens causing paracoccidioidomycosis.;citation_pages=e1002345-e1002345;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22046142;citation_doi=10.1371/journal.pgen.1002345" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=S. Eddy;citation_title=Profile hidden Markov models;citation_pages=755-763;citation_volume=14;citation_year=1998;citation_issue=9;citation_pmid=9918945;citation_doi=10.1093/bioinformatics/14.9.755" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=A. J. Enright;citation_author=S. Van Dongen;citation_author=C. A. Ouzounis;citation_title=An efficient algorithm for large-scale detection of protein families;citation_pages=1575-1584;citation_volume=30;citation_year=2002;citation_issue=7;citation_pmid=11917018;citation_doi=10.1093/nar/30.7.1575" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_author=S. R. Eddy;citation_author=J. Mistry;citation_title=The Pfam protein families database: towards a more sustainable future.;citation_volume=44;citation_year=2016;citation_pmid=26673716;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genom. Biol.;citation_author=A. P. Gryganskyi;citation_author=A. Muszewska;citation_title=Whole Genome Sequencing and the Zygomycota.;citation_pages=10-12;citation_volume=4;citation_year=2014;citation_doi=10.4172/2165&amp;ndash;8056.1000e116" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.;citation_volume=52;citation_year=2003;citation_pmid=14530136;citation_doi=10.1080/10635150390235520" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Hu;citation_author=G. Xiao;citation_author=P. Zheng;citation_author=Y. Shang;citation_author=Y. Su;citation_title=Trajectory and genomic determinants of fungal-pathogen speciation and host adaptation.;citation_volume=111;citation_year=2014;citation_pmid=25368161;citation_doi=10.1073/pnas.1412662111" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=R. Jalving;citation_author=P. J. I. Van De Vondervoort;citation_author=J. Visser;citation_author=P. J. Schaap;citation_title=Characterization of the Kexin-Like Maturase of Aspergillus niger;citation_pages=363-368;citation_volume=66;citation_year=2000;citation_issue=1;citation_pmid=10618249;citation_doi=10.1128/AEM.66.1.363-368.2000" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=S. Kalyaanamoorthy;citation_author=B. Q. Minh;citation_author=T. K. F. Wong;citation_author=A. von Haeseler;citation_author=L. S. Jermiin;citation_title=ModelFinder: fast model selection for accurate phylogenetic estimates.;citation_pages=587-589;citation_volume=14;citation_year=2017;citation_pmid=28481363;citation_doi=10.1038/nmeth.4285" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Larkin;citation_author=G. Blackshields;citation_author=N. P. Brown;citation_author=R. Chenna;citation_author=P. A. McGettigan;citation_title=Clustal W and Clustal X version 2.0.;citation_volume=23;citation_year=2007;citation_pmid=17846036;citation_doi=10.1093/bioinformatics/btm404" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Li;citation_author=F. Gu;citation_author=R. Wu;citation_author=J. K. Yang;citation_author=K. Q. Zhang;citation_title=Phylogenomic evolutionary surveys of subtilase superfamily genes in fungi.;citation_pages=1-15;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep41926" />
<meta name="citation_reference" content="citation_journal_title=J. Invertebr. Pathol.;citation_author=J. Małagocka;citation_author=M. N. Grell;citation_author=L. Lange;citation_author=J. Eilenberg;citation_author=A. B. Jensen;citation_title=Transcriptome of an entomophthoralean fungus (Pandora formicae) shows molecular machinery adjusted for successful host exploitation and transmission.;citation_pages=47-56;citation_volume=128;citation_year=2015;citation_doi=10.1016/j.jip.2015.05.001" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=D. A. Martinez;citation_author=B. G. Oliver;citation_author=Y. Gräser;citation_author=J. M. Goldberg;citation_author=W. Li;citation_title=Comparative genome analysis of Trichophyton rubrum and related dermatophytes reveals candidate genes involved in infection.;citation_pages=e0025912;citation_volume=3;citation_year=2012;citation_doi=10.1128/mBio.00259-12" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. Meerupati;citation_author=K.-M. Andersson;citation_author=E. Friman;citation_author=D. Kumar;citation_author=A. Tunlid;citation_title=Genomic mechanisms accounting for the adaptation to parasitism in nematode-trapping fungi.;citation_pages=e1003909;citation_volume=9;citation_year=2013;citation_pmid=24244185;citation_doi=10.1371/journal.pgen.1003909" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Milne;citation_author=D. Lindner;citation_author=M. Bayer;citation_author=D. Husmeier;citation_author=G. McGuire;citation_title=TOPALi v2: a rich graphical interface for evolutionary analyses of multiple alignments on HPC clusters and multi-core desktops.;citation_volume=25;citation_year=2009;citation_pmid=18984599;citation_doi=10.1093/bioinformatics/btn575" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. J. Mondo;citation_author=R. O. Dannebaum;citation_author=R. C. Kuo;citation_author=K. B. Louie;citation_author=A. J. Bewick;citation_title=Widespread adenine N6-methylation of active genes in fungi.;citation_pages=964-968;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3859" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. Muszewska;citation_author=M. M. Stepniewska-Dziubinska;citation_author=K. Steczkiewicz;citation_author=J. Pawlowska;citation_author=A. Dziedzic;citation_title=Fungal lifestyle reflected in serine protease repertoire.;citation_pages=9147;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-09644-w" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Muszewska;citation_author=J. W. Taylor;citation_author=P. Szczesny;citation_author=M. Grynberg;citation_title=Independent subtilases expansions in fungi associated with animals.;citation_volume=28;citation_year=2011;citation_pmid=21727238;citation_doi=10.1093/molbev/msr176" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=T. Nepusz;citation_author=R. Sasidharan;citation_author=A. Paccanaro;citation_title=SCPS: a fast implementation of a spectral method for detecting protein families on a genome-wide scale.;citation_pages=120-120;citation_volume=11;citation_year=2010;citation_pmid=20214776;citation_doi=10.1186/1471-2105-11-120" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=O. Penn;citation_author=E. Privman;citation_author=H. Ashkenazy;citation_author=G. Landan;citation_author=D. Graur;citation_title=GUIDANCE: A web server for assessing alignment confidence scores.;citation_volume=38;citation_year=2010;citation_pmid=20497997;citation_doi=10.1093/nar/gkq443" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. N. Petersen;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions.;citation_pages=785-786;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21959131;citation_doi=10.1038/nmeth.1701" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. D. Rawlings;citation_author=A. J. Barrett;citation_author=R. Finn;citation_title=Twenty years of the MEROPS database of proteolytic enzymes, their substrates and inhibitors.;citation_volume=44;citation_year=2016;citation_pmid=26527717;citation_doi=10.1093/nar/gkv1118" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=T. J. Sharpton;citation_author=J. E. Stajich;citation_author=S. D. Rounsley;citation_author=M. J. Gardner;citation_author=J. R. Wortman;citation_title=Comparative genomic analyses of the human fungal pathogens Coccidioides and their relatives.;citation_volume=19;citation_year=2009;citation_pmid=19717792;citation_doi=10.1101/gr.087551.108" />
<meta name="citation_reference" content="citation_journal_title=Mycologia;citation_author=J. W. Spatafora;citation_author=Y. Chang;citation_author=G. L. Benny;citation_author=K. Lazarus;citation_author=M. E. Smith;citation_title=A phylum-level phylogenetic classification of zygomycete fungi based on genome-scale data.;citation_volume=108;citation_year=2016;citation_pmid=27738200;citation_doi=10.3852/16-042" />
<meta name="citation_reference" content="citation_journal_title=J. Invertebr. Pathol.;citation_author=R. J. St. Leger;citation_author=A. K. Charnley;citation_author=R. M. Cooper;citation_title=Cuticle-degrading enzymes of entomopathogenic fungi: Synthesis in culture on cuticle.;citation_pages=85-95;citation_volume=48;citation_year=1986;citation_doi=10.1016/0022-2011(86)90146-1" />
<meta name="citation_reference" content="citation_journal_title=J. Invertebr. Pathol.;citation_author=R. J. St. Leger;citation_author=R. M. Cooper;citation_author=A. K. Charnley;citation_title=Cuticle-degrading enzymes of entomopathogenic fungi: Cuticle degradation in vitro by enzymes from entomopathogens.;citation_pages=167-177;citation_volume=47;citation_year=1986;citation_doi=10.1016/0022-2011(86)90043-1" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_title=Insect Pathology;citation_author=F. E. Vega;citation_author=N. V. Meyling;citation_author=J. J. Luangsa-Ard;citation_author=M. Blackwell;citation_title=Fungal entomopathogens;citation_year=2012;citation_doi=10.1016/B978-0-12-384984-7.00006-3" />
<meta name="citation_reference" content="citation_journal_title=G3 Genes|Genomes|Genetics;citation_author=E. Whiston;citation_author=J. W. Taylor;citation_title=Comparative phylogenomics of pathogenic and nonpathogenic species.;citation_pages=235-244;citation_volume=6;citation_year=2016;citation_doi=10.1534/g3.115.022806" />
<meta name="citation_reference" content="citation_journal_abbrev=Sci Rep;citation_author=G. Xiao;citation_author=S.-H. Ying;citation_author=P. Zheng;citation_author=Z.-L. Wang;citation_author=S. Zhang;citation_title=Genomic perspectives on the evolution of fungal entomopathogenicity in Beauveria bassiana.;citation_pages=483-483;citation_volume=2;citation_year=2012;citation_pmid=22761991;citation_doi=10.1038/srep00483" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: Phylogenetic analysis by maximum likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="Insect-pathogenic fungi use subtilisin-like serine proteases (SLSPs) to degrade chitin-associated proteins in the insect procuticle. Most insect-pathogenic fungi in the order Hypocreales (Ascomycota) are generalist species with a broad host-range, and most species possess a high number of SLSPs. The other major clade of insect-pathogenic fungi is part of the subphylum Entomophthoromycotina (Zoopagomycota, formerly Zygomycota) which consists of high host-specificity insect-pathogenic fungi that naturally only infect a single or very few host species. The extent to which insect-pathogenic fungi in the order Entomophthorales rely on SLSPs is unknown. Here we take advantage of recently available transcriptomic and genomic datasets from four genera within Entomophthoromycotina: the saprobic or opportunistic pathogens Basidiobolus meristosporus , Conidiobolus coronatus , C. thromboides , C. incongruus , and the host-specific insect pathogens Entomophthora muscae and Pandora formicae , specific pathogens of house flies ( Muscae domestica ) and wood ants ( Formica polyctena ), respectively. In total 154 SLSP from six fungi in the subphylum Entomophthoromycotina were identified: E. muscae (n = 22), P. formicae (n = 6), B. meristosporus (n = 60), C. thromboides (n = 18), C. coronatus (n = 36), and C. incongruus (n = 12). A unique group of 11 SLSPs was discovered in the genomes of the obligate biotrophic fungi E. muscae , P. formicae and the saprobic human pathogen C. incongruus that loosely resembles bacillopeptidase F-like SLSPs. Phylogenetics and protein domain analysis show this class represents a unique group of SLSPs so far only observed among Bacteria, Oomycetes and early diverging fungi such as Cryptomycota, Microsporidia, and Entomophthoromycotina. This group of SLSPs is missing in the sister fungal lineages of Kickxellomycotina and the fungal phyla Mucoromyocta, Ascomycota and Basidiomycota fungi suggesting interesting gene loss patterns." />
<meta name="og-title" property="og:title" content="Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3311" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Insect-pathogenic fungi use subtilisin-like serine proteases (SLSPs) to degrade chitin-associated proteins in the insect procuticle. Most insect-pathogenic fungi in the order Hypocreales (Ascomycota) are generalist species with a broad host-range, and most species possess a high number of SLSPs. The other major clade of insect-pathogenic fungi is part of the subphylum Entomophthoromycotina (Zoopagomycota, formerly Zygomycota) which consists of high host-specificity insect-pathogenic fungi that naturally only infect a single or very few host species. The extent to which insect-pathogenic fungi in the order Entomophthorales rely on SLSPs is unknown. Here we take advantage of recently available transcriptomic and genomic datasets from four genera within Entomophthoromycotina: the saprobic or opportunistic pathogens Basidiobolus meristosporus , Conidiobolus coronatus , C. thromboides , C. incongruus , and the host-specific insect pathogens Entomophthora muscae and Pandora formicae , specific pathogens of house flies ( Muscae domestica ) and wood ants ( Formica polyctena ), respectively. In total 154 SLSP from six fungi in the subphylum Entomophthoromycotina were identified: E. muscae (n = 22), P. formicae (n = 6), B. meristosporus (n = 60), C. thromboides (n = 18), C. coronatus (n = 36), and C. incongruus (n = 12). A unique group of 11 SLSPs was discovered in the genomes of the obligate biotrophic fungi E. muscae , P. formicae and the saprobic human pathogen C. incongruus that loosely resembles bacillopeptidase F-like SLSPs. Phylogenetics and protein domain analysis show this class represents a unique group of SLSPs so far only observed among Bacteria, Oomycetes and early diverging fungi such as Cryptomycota, Microsporidia, and Entomophthoromycotina. This group of SLSPs is missing in the sister fungal lineages of Kickxellomycotina and the fungal phyla Mucoromyocta, Ascomycota and Basidiomycota fungi suggesting interesting gene loss patterns." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ukoEjxscApO8FkxGjkVNCUDjDy2Tsh1V8BQeAYnwE3w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490978","apath":"\/ggg\/8\/10\/3311.atom","pisa":"ggg;8\/10\/3311","ac":{"\/ggg\/8\/10\/3311.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3311.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3311"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3311"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3311":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EyyL6NpiTx0\\\\\\\/q3kDvCmjfETEFzDi2C7KaN928fK8dN+nUiDxPuhQ281MbMVhUhUrQJCXwVBHvvVh0oG7ZXxnDVZJ5IWm1ie0SPM5qPLm1FOk5Cq+ZAB8Qx8ca6kaZT82quJDrAAlpk2TYhXVCa+w\\\\\\\/CsWQTaROG471gUQfB4q2xosDEOlieNdAyAepMhTsaxYjeYvh7e7WtOibdBFy5mW2SSdCl\\\\\\\/wDGnpUpcYgLqnFRzj\\\\\\\/u7hBMTpyIBHZRPENxPwmN6myOHjFMHMmrT+EvkNxFdDGpZ2ijrKKex46MTzSxn7XJKtpe5btZYgA9T\\\\\\\/Lar0S71kZUfKmJWe3ril15zVo\\\\\\\/4Lx2ZC6xD5aj01ZuIkT3wFINT72+7VRVjBq5ciy3dq0IODMxxN+kTIYjSO9TenjRCqNuj7zpxY\\\\\\\/GjpQ6l+ylQeL9YFDbUf1jF1oO1oAU3R\\\\\\\/y3bpYopl2XqpItKk9+GMtZR6bdfnRhp8zOnNNlOR1dTRsX1WUPBQy2BcR+93OOQ2Jw\\\\\\\/MmNvA3e2YkVIa5JghIjI4dVVjjTukJ\\\\\\\/XwMHevU7huDei\\\\\\\/NwhczSeDkKM4+9JYyBqRiDPIb8liTWFIFPpJLEb9XZfQriHsmFCjGYd7gyQm0EC69BkPtP\\\\\\\/cNzOcAYQwChBhxrnon0o7dLy4SmxJ\\\\\\\/XkZfNBT1Q11\\\\\\\/NcVTcY6fuF+sCLVbZ7bfZJltF+uIFD6WRkP1yimV9cr1\\\\\\\/Mt4oCW8+3CLkXD3kHd7x6D4QD1u69j9PNuPkvsXgezb1WPt4IZr6uBViYpPh2IjUDd\\\\\\\/TOFb\\\\\\\/Skp6UuNLYmmCalEiMBTA6CYv\\\\\\\/Z4O+Oh\\\\\\\/yKKnAHOrA9ebmbl2ih\\\\\\\/kI3\\\\\\\/m3U93g30fEZPCVues+URcEWLKIIWljS8rw7i4pydG6ur0vbkZ4\\\\\\\/E4AuCLdV0juLIQ3AvvjpXMcQe8C2oFLO7CVb3lZG54C5GyGp8zutXShatFFW4ryZIsaOWupT\\\\\\\/pejKmxSKSuFA7S1HAnMo1cZrjyZA43On1Wnw0WyFv0ehTVoAA4SHIxF4gHVT9W6ZRPUnMHM0ZBC1SzyewKvYz5kISHiaH+c03AAvgcUN9bI62DpwACgGRKiMzWz+1PK2vn5N8VqBEFoGp3XRkYuFd34gE4c3KoOv74Ynw5SJonj\\\\\\\/IP4bm9jLz5yaKE8bYWtnElA2\\\\\\\/+NxEDTpufCWg3Rn3eIZIGffW4pkhL8t0E1nuTUW3x5AOHqKtqAYs+j2s7VxCc1UTtUgMVAI+o97OW41K+cM27BxFdheJav02TnBFnMIfmxpfRcDE8\\\u0022,\\\u0022iv\\\u0022:\\\u0022cyRNZ4gsJNzLgetJnpuC5g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227c6a0c3d7800714f3c47e95eb60e195dce73708fd317df0e748327316ea25ec3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002296k95XdqqyE4geexq36hBCBUrHeITWh4GK96FWrT4tKKbTkHQv\\\\\\\/jhcVgZkxqt088s2YWyWO75LEilEP6unAIvse3WozYcH+fsKOvOemuLWdFntbT7hMiIpivx8PCieq3+gqEhvBX+yVwfRop8ZpTbFuG6aKlrtE6UdVEd1SAVpkxJ3feGriD2gqEbLFSYsT8MyofjUzwKMuFv2+HTndixSxm8SQd8a5fJxq1zKsTfboxxVViO32ifsSw6pHMI9\\\\\\\/3MB+jEx5lAslCkPcyles5O0yibIXbouINph6uWnjnNtMroBny8YkUtAn9RxWBUMLei5LUDb9nA9piY1sNA8R0ihtdIaJ7YjMrkH3pmwNlBaNYXO7cLeGrBvB0yHB7pV6Kf\\\\\\\/40PC6GTpfdUB7cm553Boh8YSYCkIIwn++V+OkXfbePSFNIhJlN+UniSV3Mo1ySWMrIDwYWpEm\\\\\\\/2HUiLsNMGJ11r+\\\\\\\/ddRBb83a5ggciCiPHEAdSNIQUiKyMPc39uCwmO5wycvNMZm1uHXg0U0r6wutDweqPFpAWkCnAIlyLxs\\\\\\\/xP2T34a\\\\\\\/Lz0\\\\\\\/w1ae8vATLvvONKRTabQ6c4OceMflnfl6eBMkzwS6+eFUM4opZZ2vsPsGinuTC61gbbVpXVMkam1v8oQ63AmFry9ffoCygNT0nd\\\\\\\/iD11naFT99OUso5mEWxh9uUK1FjMfzpcn7\\\\\\\/v6eGrIjaaNEWty91EPxFfh7AVR9ha0mj2LXws3Znr19UqGnOOA\\\\\\\/YOrF6vJ\\\\\\\/SmZLsGCH22BDyx3lPGX3j7MIKK1VQ1LgLbQ56gg9VC5GZCjQ+VWRSj4tsYHVmX78YEkt7ZFYfRxxeakDHqSQB6ghQz4coODs+uWchCOVGXAPUR\\\\\\\/jCMaar3zF9Sm9MIyp1K7uuyghONahvBRfBfAAbHAsTWudEUHL+Ek9Sr2BsynpnCmPp16npYawfY8zNrqAyFhGWhKjSlS+1nglE9GxFM9bBEXBt42c71zYg2MJwpUF+3XRHM6thNTh3sOfonXtXrxsqRQCK7QhqGzyodHSVz2jczbqWC3MD3mMHG9j\\\\\\\/wEyxChC51c1\\\\\\\/kuQ4oPCAAe93yFA56FddGpFy72YGaT4ZRGHpm18lA0DLE5xH88FZ+PBk6yFnJplbL2iH7vK47Pi3ziUWsmfFPkm6SD2S4f\\\\\\\/AmfqXNFY+KTk\\\\\\\/WOIX179+pmB4xGjsFM3qbElTxDfQ0SwDYuNLreE8pWYJztUCz8PcbIad+51EAzjPmfXDwimalGVXUlOWrKov+5ognkzDAv4rgQEnbrwIYtbq3McHL6xYJBnu7Zm5rqtNjZF46PxjX6GijyOA7QQN4tO5+ki\\\\\\\/GEg57nZA+JDNfMqqZxg4ePeVc8kKNnapUKa+HmBSwMfSmt+K9QvaTw1HMWLvvfJ3VbHca3fnEpMpmHrFrpk0sdQttNUeJeqV+b2jzSBtDA4h1ovtBrzbG5W0KA0Z20g+8MDU4HDhwhAyTC2PsSGa9M9cqvWEPPVR+M8cZ22og\\\\\\\/f2xey7XPsyNilZzoi\\\\\\\/3xGszJJVksaK8ngD0xO+EF40iT\\\\\\\/yyoH8nPUoecDqkI3mkRW2ou5vTAg0EtSbb\\\\\\\/PdqnoTjrTZaJMK4CyJCSFBaClXDud6xJMQLg3PRTCyTis0vPFushToc6LFCghwIO\\\\\\\/YMms5KZ3vwo\\\\\\\/DvTQr0gAw+CX7zq\\\\\\\/5ueoseiSqDIj0JNpLw\\\\\\\/6cB8MA5JoEMpykcsdxaD+4IEOxrGf\\\\\\\/IFeF26gz+SO9EFtoPFdeSwn7\\\\\\\/2usZ9z4I+TodDQSz5Y0LLugSubbYxNhE8jr+PUc79A1b0tFPDNQrYFpJfJrxKgkEgbzO4QwDlQpjgdkLObCriPmsAKmSG5UJvWVuv24E2TLMICZ8pnQQnKDeX2aJt4mEFCeu6+oxEsXXGMD3MBmbJe6BpQBWDv3gSdw6X91fxoiqEdpcOio6piTp8lYBEvFogCFKyIaGWJq\\\\\\\/nOaLmn4B51p6LxGJgn\\\\\\\/R8FqG9P7XIzuweSL6zNfU78mQS1FnD+USyG\\\\\\\/aLLj3WOC1s9RBSLWmI6bIC7IbTRtB4LUaD4OA\\\\\\\/sdotqgCxBixvJtEWPpbkprxVC6bEidSwLSsE0QCeO+yQ0QpUfvWgmNSKmgDRwxdA3+63EAQ+itEGPvYpFgcswUXTC9vcVU64T0fajGd4dHHmLxn3k5\\\\\\\/cIVPtlU2dcyGOMwx34yyo0Voy30c\\\\\\\/Iy36YLaaWoLOptCNukrLxbZObgplB3dDQZ3AwF9gYt7Di6yYQFAQx6zvzU1dAYVHD9GYrAIo356XavfZ59OyRDHj8L13f0FQiPPCSzfOfjYfSA5izWkv4pVimyRvb3cfwmTqyHBxm8YA9EwaWUfSEFXGQ5KrKuG3wuK8ozkO4nTZH9ejV4Q0F7qN8fZJ1aVKMHZ1vlb1inivFFSPNTRGMeM9cEUjDldWX2T43xyP8EtFvbT8vpTHcvu09daHtbCuJyK\\\\\\\/bcA8EClEW3f17v4QP5oKnKKoaF8czKssx7n46AiyU4Zj9dzcpjDqD\\\\\\\/mipz+gTwwEh1utST43qMukPlfyaEijK6+tq0x9A56cS7IEXD3X0nyT0bUt+MaQXHSJl75JYpYafJfaBkLcQgavr3bdDwjubZ6Zs7oHIgfnVELt8cbo7UHh1d3EO09SHh6xMLPvP8WC7j7sjvR7CFwL\\\\\\\/YZEsRxMT2JzdvTML1ejpjgCUVHYzbQugd8ZyRkgslGNP62XL031H7RHFigbR2OCcUh2D11FBjSA8prGcnJw5UzywQz7ashhuEaOakcW5bU\\\\\\\/uy45784\\\\\\\/CasMRGtNFjH\\\\\\\/cNrRBiD6H5pqKev8Q9AIuSyzo0vAmzS2ZltyTs8zsRDPpoC3WTF08tunyaDHLz7e0sdaMk5eI3Arlr33BQMpRTnHGKhiaTckUvSpubv5oNqLHKcW4BCcL3XUDcC4j3CTcXrnfjquo0naAJPeF7JP1bIsteBR6a+aExMkPE7jRxB7NylvaiuQNRwSXgx2QqCl1ZyPe+ul6CrKXY9GUkEM3mXj0KM2I\\\\\\\/gJGkwHvQtVlkfwSiL0hnYNqV56BzeB1kBrs48s0yX2CRwdMcPWOEufIaXm2uCF8KPmNKFOStpBwZkkoU54ckgRss3O1mIyHsM0fHl5pSunyqtgcqy3UVBMf7ghD2rUnDgUdj2ITJg08NQLzRkn+QHuwi8OPzorIiI0yGx\\\\\\\/JXlePWgpk5LYefmUpfwZSX\\\\\\\/vOZw3t+CPE9nen91sv9MW83e4rwoo3JMSAtVyoLMTcZ6n4yAHj5ELaCrxGwY31wouYU3an+WdvBBiHocb2oFYn65wDr8l\\\\\\\/TEX4ro0Q9RdmntHxjz5w8MWDNY9dqtCiwWdJvwc9smVBKriO8DLkzVPPQOkY2GESzeVe\\\\\\\/Gq4wMt9E4EuTCtu52LHGwv\\\\\\\/ZZHenmVI7WMfeT0NAQTpWvo2Zif8uVhs\\\\\\\/yAE\\\\\\\/xd0tLkEiKSfKcdrB7VzfQf0pzXv0EoRTp1oPSF2AmOo0DaGpq8lqNlAsc+GOfH5zsFc1dKXiT4iiswFNcRkXksmjuZ4wt6aVGtsoY5QSBM3+pueEHQsd34P8aXh83E5\\\\\\\/CBZHMwNIbZb6YTRBRsIJkcFbtYqISTGyeGdmM890jBi09HhSWdAXvr1GNGxcNDUr8VLtzMa2SedaVW5pce5If5KRLgCy9RPc2U+GRfGzCZEIL5GQCkpVnXe6MrTG1t9iMXUxeaXhgGB5cnplEzaTQL6AeMIQoepVFuLhKMZtRmPOr9WZh6jrX+R7bcAM1uAL8wT3EWizc\\\\\\\/w7TyDokSCeRBzEDQyIANo7p7Wr\\\\\\\/IrmhMQLvCbFGow3dcz7L86\\\\\\\/Sm+aSq3v7OZtQR1y2VJ5vKVN6T34s7q2ucuBVzlYjwRT0cvjBZ338RsrfqaIldzyQmJiPKh5LkKUJvlaTIwCXBJDrfT3NNFJlytr5XWnlq2fe89p1oDcnCsWrAiV1j2J1oSZmDCAQwBmhBClxYbgq0qPxC1O7BeCGPWPXHTRfsM1tg4jHiJcwwG1mSxtJT+BXmtCTRYdRL0wtkaKScHf3IOJFkGughHGoy0J0k0Vzpii3NcZNjnJ0fNUUYL6Ee12itB45o6hkh0sYhoBeex9Ns5CaJEwNAtGT5sJgO3TB0awYvVHvY4DqPcbHThgdBuvRwOLHuMyNnV8dG4cLt+Gaec6eOBiCl8AodMWW4RPs0SsBAN9QFNRXWTgQdTpz5633h0L\\\\\\\/3C8LglzgrDKeXaWOwQutpc3eTCwz1KCFZVUPIztnADlh+7eBdw9\\\\\\\/u\\\\\\\/0rGVdTogW47tuy85j9d4tnR1Dqaa0PtrK4ixC8ikZwT8LoFF4MWYcK3TbOzsOYVAqTuwddP2WTrfyroCwywmwuaFqqpey4+ZKlflLvXiM+ggO8CBlxidCaQp\\\\\\\/CqTkfQo5IHtIMg743NxISeG8ZKVVGEk0he+9Iwhis+o+r8mYlg\\\\\\\/EFb0nwiZkRAbT\\\\\\\/qG20uifOcEBlSK+pU9k4BpOQHOgH3kKTRBR8m2p8LMLe0IbaTSacs9INOlyV+cFwbNBX5kvPcstpEuqElwGHfTKi9ySvFJ9+1GWTN\\\\\\\/yzrrBEb9GaVDCZM61aXVAFYilBr92U3RSUzxy1RAjR3OanVFFjxehHdvYq2NszFc6Xeptqjixqcth6sQWpft8ux61ujbvYnHlovDBWLES1cIeBN7B\\\\\\\/8mvPY1vR6Qh7G5S919eCUUZVgAOi48xOq4y8pJN1jnN2gNhkPEiszNlZPyOvtlC+K+55V+Vox+W8EdDCChqmJDy7NHyuRHDNSLcAZXVDIj2FbQUDbNSL8OFV8x5vR15e2EqY0oajHkS+DIRcGy1VtcJKcbT9wLTfKvt6ab\\\\\\\/6Ij8dI8AOc32kg4pMmi6+iTdnW3nF6i9OwELfBGksvPWGUftgvK+oPo+1lqohTlORwIbEQv4yNrmFIV5c64QDaGUfc\\\\\\\/mgPAhTOBLwkiaAjVtICYRQ7k4jkI9zwSrO50VekZozuhU04j1DuqoxJZP5b0K6AWFKjbqkWlj+LSqwPoIxC+pkfs+8HtNbqvSef1SEn+QNGObhfDn5sW2CGfSFMxUsGOaTDkCvDbzCaN1fsmNfD6RuRYCBmkcY3rPn90EVyl34\\\\\\\/Oaba6dh+uvALmg1xyschi9NiE3Y3sGzVJmotNy6KAmzwFUC3dArS2VQUSTmMvFwbbQ8wp6yuABB0+Z\\\\\\\/Re39E6Ml\\\\\\\/hnhpaJwgqqGfVtkgPBB9mGydCba\\\\\\\/pWkd8SqDzlo\\\\\\\/QBCpqLM9CPxIQ22J+PWD\\\\\\\/D6zkFri\\\\\\\/vbJnvWL0Fy9f110i1cOS24MkVWbptLTuJiYEvuglEFqKcNVFhdlZtTGS6NbTi1JEu5VMddjGNzO5gxl+jFvk9nARfP8LK5T\\\\\\\/LcpWdfZGBzUNgsTJwKNZ9Qq2FLn1upnKekyJwIYNiGP+VzmPbYKPNl8y+qyQVqy\\\\\\\/hquRtpX3hfiAVR3Iy\\\\\\\/HqjH0PLi7Iw\\\\\\\/gmB294JSqgUjHrrFKMV1u3BkguHajg82oeJmwnXXNMq90vXkp40sxhXin2XreAqDTtYk2+BRuar5spUvJxT3vFkh77+VWsHiz7s4UZ47J8t79eDl3ekaMpnuTGDDDquM7Ldeb8ny\\\\\\\/u+iKcKVd6zksI9uBfYiekS13bnG\\\\\\\/svkBxL8soiBoHipg8AzJlrW4PPdHO4hRoUrE9pXZKm00JcOc6Xno8bUCnIih8jXMToDQYZ3svyG6a\\\\\\\/INbfpJ2PbLSjLbDvo4Wq2SAzqlQh90Rx8UjGqC3Jr6TvHdiwJKUIbOj62TF7BJzeS5lvYOLXGkw9isk+gjCO094xqcUL0VpMWkRiMR9KMw+jFO2uGG\\\\\\\/kI2lEU0M1oMjlUrAmeJ+6um+YQp\\\\\\\/xszMaUwweYf4fgEGlfYOq2uYOrYqo4Ntwyv4SlPuKTZIc9S4vOGaaNmYBQwItWfCnI5sh4dCudRQUIEIDLTeI3JczCmzFP72RdZgiS0d8+ys1fFvY\\\\\\\/ym5zqB\\\\\\\/DwESkdyOvppNE6UoepdRVWsAGEsFvxD7uaXmqTKtEzlueliRRo6iGc8H44woyugiOY4j5O8FNxgp6dJik0OdIeghTD7QvHJCAShZCzhnP33po5olHXuhN932quCK4yk5KS2Qtqt3COK5MII0mXfYg52oHfsyVmN7uCy9HrreBEiwKl\\\\\\\/X00z0EKpUEOsQ7rZvmgKfObnSfdFLpM12rU\\\\\\\/yshAKokPLWonBHdQ3MzC\\\\\\\/ASuPPnG\\\\\\\/CyZXUMZcX0EM\\\\\\\/c9lTD0FRVVAT8WfcyYxQMWYgvUlDm2a1CxE008XgJrrI0tGwrZvhmn3HGEtN7eNJD8or0jzAidieKWqYRvW+21qes34y9vQMnPTOCIHyqj7SewhPEVkrtUeVL6GZbvqnGaAcJio0DP3eAMoF985x5YGWm9hySILJdO9rMd4t+NJif0APQd7XJtkIzmBifuNMKCehGiA2knPayOJ3RXnM21iNOZYe9\\\\\\\/v619hsGhb9qfj6Q\\\\\\\/rXHgvEFv6QKnyEo4aKCJqBVF3MoZ4T5uGgQrnPLnSOqSEszwqNJNnih52b+UL10aNIaaot2X93V9rVzaarNVPOPvjrx+Yy8lb9McGHxfhu2vs0KlDH5xQwQIBjizVyUi\\\\\\\/2xqaKNBoBW3Uv1lYt4I3WiCXRvx8nHVJWRPYMUcT64mYSxtjne+6XzyQwSzsx4cCcy+KiMcaqLkPY\\\\\\\/m6j8TG1JuFBKK0i+wTmuDOlBtHVg9XOeyCTJw1tScpofbqvYFZlHgVIzeBVvTEsD4j2xAgAWcmijVIYanA7Ke6xlIuxVA25kk7pFJ2yaFUQJA3msLSRTFtF7GNDaHSlHf293G+EEkqV1pCjmk31ia2k6TB5At9dBaj\\\\\\\/kVTK3zJhopz9AN4Qb3l3Yr\\\\\\\/hx+EyV0OQFvvzJ2OuTq5OUojtCHK50wNgIVMP9QV4+icp7W1GW9GwX1fdNqQG6BIoUB6jc1f+KkJfRX53vJE6+OX6Gz0b3Tub1nhCfPplwScoGAXamm8uRd8eYswYFkymk8cNPWQ+QqJpJi8BcWYJkaRTQiacksLYgDyEVMtgP2oBEht0pKakbetS5QrCyQwpXzCBsfTikKoXxbkDltRPzCDHK\\\\\\\/szis2K2TYYV8TqLFXaJ8PQni3GkVPdW3KZR7QpxByxc+kadvYypkkxftBfTT1F5S4hUEfECleDoQwWZ5qnajZVHt0h8TvzH0XYPmA4RMxZ397skDpg6A7TXPZ7NpCrgLxpWiVDhIL4ns8OCpAdWjKKloILTbE0axv6fnsG6EW8qpIwumoqda66erCXbSkO0VPe3bytL3NYvmcvSEfQUlOHG0mckIKKfZQ\\\\\\\/ayLnmgPHGggTgNTo4ioAmAnI1EwBVLSuhCmmNci71HgxBOlYEMBGKv0ZX\\\\\\\/rrHGR3snUVGjWA2D\\\\\\\/RNg3zrwBulV0uIdyDXPVoLjjSMhJpnmG0bn\\\\\\\/LZmKfZepfdSAwY5JzCoLSMZ\\\\\\\/HmsY2UyWLCvPTjAE+\\\\\\\/ienC950ytBUS9qfl6p0l17gu\\\\\\\/AhFGQmd876rFYZwvxHvs6znoxnMCUJiGSx4AwGoMbM+f3vhfAeLzMQNqcQkac8SCEOy2oyBzNWg5aar8IYk6h+EXymHvu\\\\\\\/Tra5u+pS8bp+AzflXCnMJVau51v4Fa\\\\\\\/mfC2Bg0eyMzahvDNQ\\\\\\\/JkZ1D+Ygy7uanMIvJv8CZiCUqs5ePTLH1B8Iui1B0dtrPFY99+\\\\\\\/OQ7cQVjsijv4elepNLKfx0vBPJxvq8dWTrsIhdRDjq9g3o2fNNEsj2Bth1NXL6fiDgzDgAlJy62LoHjLbv\\\\\\\/sYPqPDHxk2Rpy5RrnOM+mYu4q2Gfn56NtMIaYviSYx11jWs9ba0VjzFWjJdllRizF94MTgSqPDcZHPHUbJdMpZCBHvgsMXWXPnNWI92FJ0PmpM0+Jtdvkug3B3Ca2mletkR+Z0deoVqqKyb4t2izM22hGHnemHoQY7LeBDszkcyPbHxtZ858xOMr80\\\\\\\/KSO3PlP900cTbXCOECMIJjfnFNg7tYfmrKrYCIRX0ENHuW7hAK2RZH0ztQeI9p1WCSQnpxhJS29r4WopNEyjEQz3hnYkBZzbokGLu2VijcQFepGB8XW0J20ISXCTH8\\\\\\\/AhvzpYbex3P1q4M4L56mIvdRPG8K4UiYec4ZZUx0D0\\\\\\\/\\\\\\\/I0hrUQuElJAmjpbnIIUlr8nU7VQSrVIP6u3WAhMBunZCn5sqUEquWfCHxCGeUg4FFV1tdj1EiG9KqCihuvOD8i+RP0j6qRhg79c8NMiaJE5y9klDq0XQB\\\\\\\/kU7SWEo2o0dn3u32LlV2AM0lGu9UvWhoSlcMrAds4dn\\\\\\\/MtHHMyw1HsJUv39FOJngWY7+LARMR2i9ISVI0UhzaQiIFvfP04xcOiu51kJEIWSolvW6ZXSNiD4qHJ7kMWdxnECPCx8Uf0A9AYF56tPoqWbWMkn2WDV2byBnLGSduRtCXGiUGFvTbrtKfJjs6FCRR9bV84IxOvR7hGXAUrHcImSfUj1xCXP1L2DA41c0FXjXzo5KYiozryu3Miwi0K6d5NSxHgQKxtdnCav5MbMFi7ysod6sFAZsoOMNU5erlhp1O+lfYXryEQvp1A0T5Y7fbCSE3PMW8+6H4HOwhbP5mKqC8VNzsEcB43PylV04+PcQxAW1I9nSM1wRyF6yG0e9ePnPPGjK6h+BI30QXjjUGOXsQLN2L\\\\\\\/Weg+3AlSr01Hms2yJQA3wZwdnG8yfd5BXsz94Vivoh8srks8\\\\\\\/EVPmVQETh9jtRfCt7gxxpWjejrLjugD370eKJeLODX7b7t5UbwwH2H2sWeP9AJroZISGhEUXceQXWxqStDFNjMjtTSf9n3nHAkwKH5GSfXR6YWkxup7uaAwT5zpAAKuzIsmgGnJS3QP61yd00lONFd+DnxCnYtaifswOTK0Dpg+7dFpUOoLfjG2Msp5M8gtkfHr8h7CL4jUrio94dk0QXc\\\\\\\/Zm6hYEfiRLaAloDnwwtjcPvXy6mDgpMsGagXOO4Nc5I3ikV2Vrlc\\\\\\\/E43Tl3aEZ7N4MrOHsrfXE8VgEOHhcR\\\\\\\/xoXZab7tuj2vPCUEIT15OnUJQfciWkZQ5FVIEScECUGiwPyjADSoNeXOOY+69kAhm592MiehXvs6WeKlQHYowQHMhfAsN4ut9VzvIkjPDB3FoAS7NR3SlfmvG3TZxsJp3+RNHijZXdSZqzm6ob1umduN3AL0mVmvhI7nd6y3iHfZ9A+jYgntE3jfLyvL5+qscpBmwpcoxUbfVMyNFgHXAITY\\\\\\\/SGirkr3qT4ctzCii1EmZaGKdIaFYw\\\\\\\/D3u+yrSh1zxvHNBjG6YFw\\\\\\\/Y4dZPVDG9UUK7AX72R7DnZWbioNzVB12xv47WWMddjIa156p5xvbUWnPT9lgYeLVa1ZYt0lXf6W\\\\\\\/8mmn\\\\\\\/qIEBUWMxYpxgH\\\\\\\/Yw\\\\\\\/WjgSwSJsNeo+9QRaMGDtIbjIn4HazF37RDVoz2EAleayf8UJAq6IioiAoYU3yy\\\\\\\/rZkUylZAJEOFivI3IihGRafrhPAQ+0hA4kft0tZkRqeOnJovQrVFMpo8t+c34QhqumGh\\\\\\\/J2fNq1uXiR218z7PAuaHXNo8ouCwmMW7OHtXWnhtN+DvYL+lEVkpXhGQXY+bKflwZLMdc6lJzf4FfuO\\\\\\\/N7fz2isFbTOrxmYr42vSwlteEURoijMSIxjZFuvqJGmIqPkRZ4hLyw9utTHGix559K4It7NtvaW0i+V71E+4p4mK4CX\\\\\\\/5iI7GtzXVynpNVnw2mas16uHZBOVtpn1QO5P6woVllfy9Tb7vX1sPWYjlj\\\\\\\/jypONQpMUzsOFgmUihNXs\\\\\\\/mD2laU5KL94kJ7am3rtSHNaMQlGMgnCWLfPaCuvhszlRowW0Cu1E1SISa7XsNZ3fxgOZY4QEfnbKUqgkIas30gEWP9oA8QlxZEJkEM44USqaM623Hqeu6Ijqo5N5XHYGvcE77fb3n1WsylS1Xws9bN2fH\\\\\\\/1BJhNjQ3hZx23bP+jHmZsaqGCA5hoM4lqmk+bcl6xayIO6WF9HDQd+4SdUvVdKL1B\\\\\\\/aGHEFJx+TKJ+13dI938IX7a4OQHnecLrGeRTc8vcGoqXMs8DFmBKa9JVzJey3OP2OS4\\\\\\\/nZ+oiQlqQTI6cJLuZMv781dJbEXSyyOOtBM2HD0\\\\\\\/tYsehDXyqsLZWmzwKBqSOwfd82FXnQpjb0n+etfahsI1W\\\\\\\/8KNfgPgrmOQUyJiD\\\\\\\/xE7RVUyFdQk2WG0PEuxK9+f6RpSphJWdn1wFwNQSYvzH0wvQn1MovcZ5buaoVo\\\\\\\/\\\\\\\/y6WIXKlzJrJeFZfSTV0YqqNDqrheowksizS6VwGRcBgdR4Gye42PmeSjCuMiuHLEU5mGMR+EtWDY0qGDBBelc\\\\\\\/vJ875G2qjxYct1\\\\\\\/9Ic8BXvgnmgF7Xcy\\\\\\\/vYw8wIjPTiStar2Lo6Kur3gQX0IjRjoed6gBJJU\\\\\\\/ELr7OpodSp+qqvQqsYej3+IMUNPh\\\\\\\/Dt0Gdxe9uIHXDv2radGNhI1FoKsyJWW\\\\\\\/aPTVsVaxlv\\\\\\\/bCRIHWOb8jH0NwVRswxm3+iPL\\\\\\\/96CRXLfpAUPAyw0VSDk6NtoOlQDELVGrra2hPb\\\\\\\/xPzUkd2uovOeQXBRs5bCqCClEeFuIvqDZP1gVgxiVa34tNqXmh+tF8mIVwOO3NKmlULz2K1BgZ9OUDDBhCAZPxT5c4+j0ZVkgLgOB84W4MXzkhlEFMxI5asDxJMFalvZoxeRVZMZmtIXMakaMsf23S+49cR5Eh4stl3yYGn+7K6G43jtcC3g7FN\\\\\\\/yGYc3ZSO4oBq8JIr\\\\\\\/oMcRqmd+vnb2Q\\\\\\\/d6+tcFoSA5pxjtBuJB0vTJNckDpe49Em\\\\\\\/DPy0y2+tr9tDnsWnxUWC\\\\\\\/KXEwMWtUKVaIB+\\\\\\\/OLYx3fU4lotCnxxJxZBa0DLXe6jo25hxbkBXT3YogCjXtvJqD4+a6cWdTYdRB4oyoP7q5qgMHAG\\\\\\\/ya+cIqWqKwkowxC5RTgYWkbd3iIYQqYEav4aGb\\\\\\\/iDn3gYXMRfkBMZv3u6nfZ+T+LcGkl7jgi3UZhd0hJQpDh61T9WQ7G4B+pRBazl3KXuGCV76hUofkM50wXDmf5hK3oYtDZViCZeRRe4SDtVbBQL27s\\\\\\\/6HeJv\\\\\\\/4mGMU+4wBUJqdrek1IkwdE+EN6c0XEfwdRK1kteE4DKCqdwE+271rB\\\\\\\/myLivPdomkyB0rqB8pvM6PR4ls11L+IxdVRIZ57XbOKrgHdLDEXyK9VoPs5a\\\\\\\/vfDtozDiANHCTtHCtBGyB8B5JLxheKSO1O5wPnzCiPrCqxtMAkJkY1mjgWfLrLcoFrDD8D4bYwXSOyzuWyNFfUJAIicLbcaPa18ZEqUF6MoxOEWcc7sglrxpTeWzF0bTEz48mo79fQ\\\\\\\/GWvHJhI6\\\\\\\/g\\\\\\\/+HrB9g6xmU5ytDJ7rmXmu20\\\\\\\/IEhUoBCRO20yXdVnN4J15Gc9y04KTYwOA+sBnHe6fSREkDDT4urfCPIiRT7V0IZifvPmXEBQNpmob1s3yA4KQmibbFIQu25SXYABWjeVn4zpKxasxEd6cgLEz\\\\\\\/FX325dcs3HXJsgSZ2LEez85q5xco3Lv\\\\\\\/Nou1AX7UQV\\\\\\\/gAaJp23iAfWSXe\\\\\\\/\\\\\\\/l3TbWkqtVATwD9log\\\\\\\/YI15l0HGt+3af97Iwj0HzNnmjiThx60VMkMRCbGJlLd\\\\\\\/Vq+nac1AgOl\\\\\\\/NQGsCH4UbA2nD3lsAeWOQEHcjZ6QiZQrBk8\\\\\\\/hqHLJAvcx0yJcEsKkOvPehf5NWkmAL2G9XCMDtJwB8ikfek8KAxrUBUrdbeV0yoAJKeCPnn10Andlpn1\\\\\\\/Bj4NdPoxc2n0OeF9I9BCgGrHlO8jhjibc8h0LFEQmuALf8NJbzO5aDwpFNuRD+YvEpFOXmuPi5hYcVoWPheFVwXVoYOTMnthVo23YL7BY++Sj3DFujm\\\\\\\/5JDOqV12d3W5pKY7O3K\\\\\\\/14BZ1mHEKMht+ebXbaZXBbi9DFwFkvc6kkQWXR\\\\\\\/KM3fN5x2sGTYdg7kNJ7PxxVFyCXFNtuSHmZHKqQGqdhnmUJRV0DH7vqBOuXmCumDaVtpuZykl3oaFrYz3tGyjGpmCRz7vfX5A56HGZmUB+6LiN98qjj1xtlUYHfL0zry2WQhVI5y2vJuexnZ4GKIY\\\\\\\/+wrbT+OCJCGisoDsF90IDu2CNEcwMINXDuCaeWK\\\\\\\/\\\\\\\/nUcUEYEnr76UiBDuL\\\\\\\/6yGTVoHskNXvQyKDklkXZwXNRpFQasnDXZgU4de\\\\\\\/TtuFO7zoIqavlkgXWu1FZ0Khq7JgIVyU2qlisYTxqbqz1n6CLMCbB68Abdxq8KZjD2UQwxcKKFmipFOdI4BtkcoAfLHrReU25YMJMLDTp9QdaN6vja7YR24kxKTeygMEpC4powW1HXjrmrmtZsYnvj0lPCbR8wAT4C0gQzCblK3ox66QB9L9aOZ\\\\\\\/0OK2cCVlNtV8OdiqHbhoNQ19AsdDudn2ntEGNNhfNu+TdwlmXEQ3elMoIoiIV2Fs9\\\\\\\/HzzbRoA2hPnAbP2dZKDTCuZc5K0LD2JrAIKq4nvfOiqJ1EnRxx6RNPj1tsrFIxOzvoBqjnrNJXDbnA+uBOqNdB5B5E0hD93covBKTZJMGv6qhsvtEFVnSXPuvJQqG00a1+JbL51RfhcdNmh\\\\\\\/JgUoR1slE6N5A\\\\\\\/wehHmQOOIciCgxz+m4yPYDnKL5kvN2W3bNHSv06KZyMNPW8IkFcX8sisQB7LtyFSPwIN5J5miZ9ddfdJA\\\\\\\/yJhi9l75LQhxkFVy+RLYdIb\\\\\\\/7SiYRrbnLea8np1FNa5gvXHAKjhprsgGzksJtcM\\\\\\\/aZMbSprjp+JagQVzw4FweTxgBBmnqTdfbrYBiBm9bRDpsK61qHpfqlr\\\\\\\/b+iRwS6wrtuWS4a1rnNpnLJ+WeAvsSKQSc4T4Jt9X+8bClPCd4b2PwVZqWGTlP4U8bdwanjH7goBMXM9rF2fCqfuvkNJGC0gvdR8r35We0H1Zt9lRoRga2ZT0XxA8kREee+VoOoV6verWngvD3xtKM299ilfV4J53FWsHXwV6DxRlXCp3XzKCudbIrYkqm+SjrWJOiuOb7cKk2pbeIT2SNSzc5XpT38ND\\\\\\\/TiC4eztw\\\\\\\/\\\\\\\/95hnHbfP5y+7tNmU2U8KYZwpDFAc5iL8yyKBdrejYDcLai+aAElrYtpVzdBheMQNzfe2Xk5etttG8Y+e5zY6GRmz\\\\\\\/9OF+V6ZZiz9YmzlLjapOAa2f1myhqtxAhcwjlF+PceACJuE5cithQGQIvfppgsLPH1syCseeOK4W8rc4GSah1Ym2qWil64u7ERwyKdyPIn8EzHOMujUtxfaQKWRapBNxftQm8q1LBuurhZuKWGv8c5gLokc2NT5aNopj6WPuupvfWtNkPPp3uU0OWN0XTVrzw8meyURLcHEqV5jAajc3uqRwL8SqXxKPlxvMiS4US1cRZeMk7bcx9idxQHVRVBHtb+5KLDBipw98+SJanxH9Urx0C16HI7geniO\\\\\\\/cmGijpNDZ8fASuRMdgYqISGFqJUG\\\\\\\/b17\\\\\\\/6bBZsv4I2G5tnm7uUGfVU8FW9dsuX2uHPA+AHQIpSfhPwTllj1jMJFG9N9fhXYbHWgXijgJteENzou+mNaPr+UeNUd7z4V0IMpesWWA4rse08KVJvUx5WthazFLSgK2oD6VYZWov0fWMUAAAjrvgtkQaQWa4YTBXEGrlQCmIsTU28XWOhtlz6Ax5dDM\\\\\\\/S7HNeSGUfb6iFpHYfoWETPdVxDuImHec7ws7\\\\\\\/rzZZOJfQxvQuD4ObAbEhLRuqt6PRkjzEYAlNIn1K6nQPWlKn01rEwtQ\\\\\\\/PA7edtTS5cI9yNO+22RJTbHvgbfnmrxqoOZv5NKEhOpJpOuWcHFWyDFy7zc0kqH\\\\\\\/yw3THKw0c9EdokaJ1dgVx+Huf4c37lcmKEsu+rEHp2C5UecfpAVgxfu1FRKAkvh5fPe+8JIgPedxm3h5ziTjHR2wsLxEFgfDlxge9RSLmD48q5kql3N4yYXH0Z1GTIQH85NRAnWlCKoS+pvyrJmCp0PYZ\\\\\\\/QhftnvXzz4dya47HODgnJuuljdpq99aUOThB\\\\\\\/MhGtRPhaaT6v8J+pC+n3eKDBoM645kPH28KrFxs7RClvx6EOOsUdj12tULekLqOXBDc+k4gJw31028Gi4RxAtBmpS2elcybKsloTn9H7NaQ4tl8z3\\\\\\\/7LT4uYGyxnoZtIblVi03IpUDmyZKBzcwBCsi8X9To\\\\\\\/pFAqRwqmp5SS6n\\\\\\\/Al6lCIaO6uu8HzlmO8G\\\\\\\/xuGN+5sV9SaCiAm1wQ9FWzuZHNvybEg+NP1oOOqpjVfnYkh\\\\\\\/RTOsrjxYZkdHr26PNq9piT4sXI0xf7KpYjW0DJJKKXH+ioDZU8nNnUl8UlNEWb5ZIhbHw0eSsHycicNVWqOGZ7myNkYpNv\\\\\\\/IxZQLWx5AATRaSz04pgZNeExe3Yw+7PWPabwaeQCgTb83iITELfLpOCpIYFn\\\\\\\/bqb0A3Jt0DoiZV\\\\\\\/1Bap5hHjfADxeiON+UEYCAOGG0kFou3iP8ryncpVvAuxOP2poqeH2nY9IDBQ17FyFxbIENQdbXZYLZSzw\\\\\\\/jttba3G4YgoAIL4N26OK2kDj\\\\\\\/ZDqxGUsDtsgnVgcOi4usrCyu4NiJGxFQAQPq3pKAZlA0BdoY9uUsyVt+1JM0y1GuMKxJhehMSfzsifOpF\\\\\\\/Z7o+VIT6Czf4GD8KMNw0jNwkNOoT0UkuIpM1KbATfB0H3IridI0VXKq2P2WgxKk8arz0P0Eu8CQdtW\\\\\\\/vtqbgUWnpPLyj2eIQUow59eVj\\\\\\\/0gNgUnWk1nEexQK+X0st4WibXt\\\\\\\/lu\\\\\\\/4bQQC0RaiUBFeqS0hFcLezkimfqSxgmRxDHoUDtx+r4mB4gNx68eEu9\\\\\\\/Po6g9px+RczNQd28Jt0DSW3Vjqa9qH8KArPq50dMJ8NHcWjDeuU9M4aL0VQz04JQZVZ1OT9e4h08r8lD\\\\\\\/oolZmYxILlRyLnqdWpqfQGCA7MnYTybRXPZr9H\\\\\\\/QQq8d6\\\\\\\/Qc9LSjGcFmnwPbsPKd2XIeRkYUdOAumZzb5K6hmRyh1xVhKk5YMeQqRfjZwWzQjm9C7ZYRJv4MrLuC8RS0sOHkYytXxgWfC0FbTdf6r2v4SsG7Crvy\\\\\\\/gGI0Vu5CgHywlzHWUWib16UL\\\\\\\/hLpREe6VGowgsZuJZjimhJLsWPC8s2+y5jCUMdIjRcPBMfl+7El9WmbBWbrmal6C7P7oMBy5WVZ88WgAnrT1FLs4NgwAI4y1b3Mwnp9EwsIv6dkkKCAL0otlvWhwmgoZvEDbw0t9ovnukd6c8+0iCTZtRonbi9xiXLqs\\\\\\\/y4m5Df6vEdq5wDbzme6J7zPgsvBNcbnEbXG2lYdz\\\\\\\/qJi0qBrtq8Fqg3El9mJvPpEmewd7TJud\\\\\\\/zzhNBOGNI6DhImqkXMlwX7CyIx7mVKoMc1cbg0l5jA0VZhG8ULPH5nxpCEgPX\\\\\\\/Y6pO8NnNyD6eqQOjyGd7P2eOE1QuMg9szWUm\\\\\\\/YUvUXLg7uA0ICBl8MJEPAPJOoLxiK1Rvn4Uk7LMLq3mKrUxoskLG73lerYtg28PPubYc6pOIfUO\\\\\\\/5de6D3dngFdlEP92FGHe7ruMuTMscGH\\\\\\\/q\\\\\\\/4Mm5qzbH5lFq+j+0+JUUEy5ridtUsdGF7UY9daDKB0OLB+YjiUMTZ+ETqimcfAsVixxlxlJMOw6ANqFO+7N81D6dO\\\\\\\/mY8ExnJJ8T8xDsVIRNjNlQvkJgz1P1o4G1Lmb7PT4+KTl29ZhMFMob94Xz\\\\\\\/b8aARoiasESmLXdOKodRzLNleiii7JiIKU9O3YsRejrWfUUQk0gP9EZd5986IE3EGbkzPk6trlknm0DxgRFbgdSCMBTpzckCQ5JZHGRXAf8+Gp1RKKXZ13GSMknCzL1FUfmvZUOHqTZhir7\\\\\\\/WHoQomuEogYzwcjpwIZ1jW+39vZM5R05jhsrnlhk6B\\\\\\\/l1M+glY1HRcRDJKiZGCxMWdGv0PYIM8XRhYWo3mzXkl0eFJTyUJgiK+eZvzOl1yP6i+gmNkrPLqpbPjU1Oh1VkpKmyOlaDw+ivwzbuBYLZk2N40DovbdfkVO\\\\\\\/rnDLcQHljbk5V0icUnwV3mQHhz9GesqviMmgVbXx287OJF2\\\\\\\/XUUthrQu12BXVmULmukxJ7ZUEB6HxXZim+OtrIksMRMx6acBz7AHLt8cVq75QmtkzngL7TNCfEGXaPeKbvbi7s1dqco2kpvFRccSNsGbZJ3AHElNBLYt+clYPqVRHZmybO1ALx7od0zOF5FQlyRgQyYC9pfpRuSswQW6oDYTfyqqNBEUFsPM5s3McTigw5eZcc6yslTvCihD0gw1KKq3B+mzr3a1ibtzYNRFPd1QrHLCSYpuP7zT5iqKAJePE2IAtiW3PWxgCZAy6PdvcJSDGlNj8zZ8huqT0pUATBS1iN1FfnllmymYXdVt1RRXEreufYGFfWBfF3tluh3uXItN\\\\\\\/id4\\\\\\\/o673YFA+zpIe4D2WS+5hc7Nn614xbFjKVHDGbk2U8scm+eMkJB7ReDVoyuMkphKmwUON3MrQgXmtHxgibFoF4pNZr\\\\\\\/NqXFpvdc5Atl9N4knlnoB6GEH3gb8UI7DtEV4+mIWYVHiQ1hnmSTIRoqNRR12rQY2A4WCTPjU\\\\\\\/Ic+fCzmF7kMXCWyV+MWfJQ26k2jH5HltjCGoqpXGReP8r9siX6Hb5HGhLKaSUNFFEg9j2rTXyLr38BI5KZbhmV4h7ZUR1N5rc\\\\\\\/ydBZLiEF7Uta0asAN6vscZECDHbRMnfYJYbAFGWD0uVlycuMT09F5nsUTEFQwft+EM42GZ7sgVg2ZAmAA837gGqz8IA9lsTSPyCd5J7gAEwYzDbvd\\\\\\\/noyDDbbJOij2NCJywSQxGdf0RRiNeo5Zr+rHzZy0HJO+iPHEH9y0ifDlQmLcnQgdqGIaHOF\\\\\\\/c3tJQ7Icv4qg3cxTYhsxdfYUEZTnecQsUpImSTb6Fj1vLVcrutE50ayBhLNX+4EeqEciIkAN2LtEWr968rrnvvBxgGEp61XEkFonx+CiFIjC+4zrLPo494nSHmE9Gd0Ujqy5UCzIO0mHskLfoPi+0G90h8t+XZbNnJhUFRmUh320sXHRHfd+BAphMzkQEIU6IevuIvdVNKF\\\\\\\/SWc3sJXyCETZ4KumTzRYy9P\\\\\\\/jxoqDI35TWuzgmtlseKRgHpM3QtfMWwzcqeWHWAOtddR1gOjvwhUwD3iSaiB7wcG2DoPiPYCUJcBwZ\\\\\\\/zJ6NH8cECrbyOOctz5dCGOFbsBoZdZm9ioGhi4xDzChZxWX64QzIKUzBbmJdcIMNUof5TYAU6890sgQDyezM9dnohfqwnJDjRF59f\\\\\\\/V5n08LUlkiweAXOaSdV3OeGXCl3D5ve5OlgYwOKCoQ3NYctjZyVhODAVRxXfII84DVCXH34Bs2PeP2NI66w2CeCVBTE7lEEDq1WAPI\\\\\\\/6UKzIQi4Dn51sz5mb7t8vyoYmeyIgjmzk1mORmtbtAuKRknn\\\\\\\/XegRhE+xylh9jQ0SXakonftmWw7Js6sYJEzd5seECnXpJIHmfDZVi+qx7ErDaSCuG60zpN585YTDPiJV3KXUos6AFQhCI37LsnRC7jzzMEwCrT\\\\\\\/AttuZKsJ4Du4YhRzbdk\\\\\\\/GqFivhY5z9UqhYvXhw8Urd3vkyIR1lXaAWTU7x+4d\\\\\\\/2FCYXKHa4\\\\\\\/Eu3FkjJ7p7Iv3hL8r1pxNzunyk9e3qT\\\\\\\/8BKxdZh\\\\\\\/xVjJoCaobuHsoONXLedy4SRzOk2v6VtcuD0DK7vJap++3UWEBdEn5iMSHVi0dwJrbS+jUg0PKeY7zCTzu5DYKkdqyHNy3OGKv9zjpVXojoYighDUIZX\\\\\\\/1vBGzoXwcIHyQ7Ex0pQ1EUyj0PlMmL0hZQnYJ7KNl0oYgTSX2cK4x2Rtuc3wa1nULhPt\\\\\\\/lqDJjv7BwoyaJVbLWRycS3yx4yLex4L8Wp1B+bc\\\\\\\/dSDvqpbW6QQ5qyq\\\\\\\/ODdoUXp7ddS1YR498smqdfD1VwFpnNgZPWZ4ddqrkpolmkcsUvfF\\\\\\\/72w4eyW5Swg7mXTThP2LmEqhg61XlM5Aaj1SPiIKkj9a1wfc0LjBGUYFRFEC7DpHgFXvOU5dOIciIIeiaZ0HmJ3KpcGyudKyae7OHKBwpX9+huz5SDvW22vxlxEc8U\\\\\\\/WIYQNRrPj8x8lDPlmBBiJy3k3iQHHV3hQXjFAHUkgGyCG1gsvSDmdWjxad0QGULeDq1LxGXCClOEbyZ5RhyqOPrY6yw7NFq7vVwNM5TIsiRlcPYw5EpyXqrhB31Ek0gC\\\\\\\/KRZr5xNz8i114LWuHh63z1JwVKrB376W3Tvg6m1ABc1MDSD0K9S3njAhGCOr1PSIB9LhHxKWnAFvRMXVUxqb8ePRe6eFg29ZF43Fsr3aPpAFIFgs7hLmuB1UpeLQKGztpzILCmH415zNpNmkqWQjMHFLGZ\\\\\\\/3usAd1i0LurrhzO\\\\\\\/bUY8oyon5TSBrMhbRWkmP36xhqPs5c2+Z89EmyHAf3z7FIkdleap3+3cQyK56\\\\\\\/KWF+GQY82Qwzd2Awqhf+PO13nESDAvI4\\\\\\\/vsV40f4M9HUs04UaW2mqE3WQ\\\\\\\/vyCBsz1IFnt0ZnTme80qil03fw6B2QbH6p7t9EMP++7\\\\\\\/kxt9Nw+WTCXZRupef0bYMB4axYA081lguNNTFFOGNIOUcdlZSO00ZtN4FG6BZa8HFDD3WFEPE1009GyRFwHM\\\\\\\/S8d4C1qt54KPA0K90D+dRGf4y8JIcHETS+7iAkr2A6aWm3GzQ6ylWiUZQJloju6TFjcTCyQ6uMdQYbSJqUnqWiPqmaDsUOj8T2R\\\\\\\/MzcKecGCxyk4V4jYNd7n8cCGfKpeH3bFtbUswiwS6aerW8Ig+isZSzGdsOaXL7EiFlcB1LG\\\\\\\/g3OB\\\\\\\/4clpbQ0vRGGBz9u1oa6u6+dH5aPg+1WeufKNA\\\\\\\/vWpnyEJ9U7ZCDC8YgfLWKRcx\\\\\\\/Lborxidti2rB2PHvmBJ4AuglHi529brIrAPGKZzvJXLEbvz3Ml7OD8219ra1A4ZBnCHpISdRhEpnXh42n9BqPOBeBtNl4J9UAHg487jp8ZoE4qXF6rwrOTGXk1LsVrgTzz+LNAEr7tX0DDQWRMrQ4quyEo1KC2rq9tGCjvCS2bt5NDkr8iOqmuCLub2aEF7A3Haf7RrQKC9P0UqDLxMj\\\\\\\/k1jglVoTzMGhaeiwaVV4tfORyNmivXn4zKESbeV7zst1wMPO+7YpIOC\\\\\\\/L9ovQTsf+xVQR4kPkTtv6OgaNT+LKB6L9glP5P4jQAuWglYcWwv03RZNDUzvVgPPRCKyasBD\\\\\\\/wHF+XvMKR6ESxWySTh\\\\\\\/DHRdsh3jztWtJd+p0sGQU2Asla\\\\\\\/eZ9CCmncKU7FObUz08TYxT\\\\\\\/z3lQHejuIY36n85jTcq2OZfa4UCq+SyNsV++EQjA3QYl8zGHPsh2YSeUtotCl7W4iCXyu9C\\\\\\\/LNPkExG1ppaqROKVV5PoAQ5bD6OtBekBg5au4V+dNZbd+HVhNNrkvD1tRIU9Zs9fVDUBCcdHa5qXzZUttfkhQ1lDZwl4NiSvpBMFVmqy6xlAjxHvhiYT5T8mmepvycmlyeazKIAQL7yRU2p4DX0LrA9JQzosgTk\\\\\\\/3EegcjTHDiAadA2oYeSSBn7lvuC\\\\\\\/3nztM4rW8nLvePBhFOqpy8rpXfjPcKoINywZuChcJO5uosQjt13YDpgO6T1gitGaz7TM6PYytJOldhfYIOaZD5IJOmpYjMzwbSJy2a6NTlo6UKApYhu3rIJ7tuRrBRMN6Gmv3FE\\\\\\\/\\\\\\\/bEIZMo\\\\\\\/ss1PQXCGZK7neVzp\\\\\\\/qrtiStJizbDU\\\\\\\/P6Afn\\\\\\\/Xf1bLZDfV1qocIiTEL0JizSnW0BMdwuxItL7RiTzT6nVMHVt4VNEo\\\\\\\/8gwNY0TPqhGUDu7IXYzYXOO77LnqGg+sOCDU2Eq4Y4kELTlp6dQs59FDQJa3FVYeQAv6UPWUYpNxMAgBA0kMmGiDhJDj7nOd7JnZdDjzNES99Kw\\\\\\\/RtYhSa71c1r4J9en6durekrgxqaXFvz\\\\\\\/YqCvwVXLw\\\\\\\/Eexaivs8S3l2VrNVnRxPWNhygIPAU2Vd3GCfNIsiPysxWR\\\\\\\/yN6EvJ+Kt5eVSCmbgNXgWGknN+orhxwFolqedhriGHL3DqNdysVWX5mB6Q7J8mXdu0AJXpfWh2dNxtMS\\\\\\\/4sEofn9CBipjy0JCO9rvMyxC7AwnL5MU1visOiPORcK7zue0lOG4agV2RvT1\\\\\\\/kZpHdhaivBp3MQw3\\\\\\\/UOfCruFAqRcIZ\\\\\\\/K39iqu3czrj3iAhJ7qWKIzejlnxZYgq7Cv83uJUJctQ9VN0\\\\\\\/ZY3roV3Cnik+BmtDo9br+1tellPYYlqBln3ojN7NNyVAlx5oKOzZNFSa7MVn3TorbE3zUBWCO6xUrdBIDJmmkGc4cY5ZHIywDnHeIGF5JQLfMhv0TrLDB1WsaUUH116pNvpWzSGx84amgnNW5mI9IvZ7gv6pCHFlDqhl05LYFxDX\\\\\\\/YUwAsCKMM6WQBzo+BoSyjPD9KlEEhshQtxyVXd3yX+iFh3lgg5Z0sRMACrMkOdQSRCCWaWlT5y4rki3zkqjbY3fXlmZK1KfI8zOVOXXV8zeuTZMo4ZFil5LCEDfLhJ9TkgVjhv9DDSro55zGCMcc+213vgjFK9h5L7NOXn+6ZRXpyiUSzCWP\\\\\\\/5db6URwe9EzswKQhsyBXvTgHgX1W1u9fW5G6jRfkVBZ5QWJMxascHj8EavcYmLC3cmlB7+ozuGHnSz\\\\\\\/wFeaNzz\\\\\\\/cnHbgS6M0sr5lROlNQ\\\\\\\/fPHPy6Wr1HPkNZBZE+FniFibqhMhlMs1UOMJKKz6C\\\\\\\/2jFxcWUpa3XGvZtj1iyY0vEwOUuvvnx4Oin4KjPfmjScAZfWoqxMC0R0h+QyEObIjBjERxpc0tw4Cq\\\\\\\/rRO7FVMAbycLTBsMZj2004sKYD+rY+AppzfzKECLygDMt2vmGO89J6iLj8OK758Y+E7ZQRj4bTKyK9QSL4x6WWQOn5AYEto3btAw+nNHcJZ3IrpHnC1T9QxZ29e+F+9AXRRFzsHXTYzjHM\\\\\\\/pM4H2o88HtUM5Nl97rIyWv2xhR7apLbwQ+tKSydPXsnGe43ne\\\\\\\/Yx0a1J7V5TzgELw8bTL\\\\\\\/P9McpK+p6di\\\\\\\/4ZjJtU5kokcbo5Trz9BwTFKi2gqkeWIk87HuRGWnSI5DakaQZqVDP6Pa1rmtBK6zdqYpkwvymktB0C6+yl+\\\\\\\/RLqoq7msL\\\\\\\/NAkM\\\\\\\/ZS2cUcshmz3ZtorEyDJxyFId746z8VE5Th36q\\\\\\\/DUX0hawXhdGtcWnJLuPuZ5oSc8dB\\\\\\\/Zi0pVJJQ0ltKMzK1343Z\\\\\\\/AhggMvkE4XZIOVLUMEfBWixcAbDqjltxbHWnrTU0uRVTEaSpDfkjxR9u685CM0Vn\\\\\\\/sOpwkYk7blNSfK0w1xk9jGoitU6SON3H0n8V6Ef9kUMuOUa5nglgR0Pe2wjU7+wf5oSCF3dvsvef2OUiAl1oFHd7lpFfDBi7d96HEJO9lUOYe3EuLFOtOjusdVvxWLi5YULVlHTr3RNjIkIb4QQP\\\\\\\/fzO8ZBLJfoD0RvTNW9zKkfMFRkFS5q1uBzOBHZp9I2IUf+XakAU10llK85ugE1DtBrORzWMDfPSvuS4FUHojNMQkEkO\\\\\\\/QyYlpDxziMrQzhhV9nRwbUxdvjps79T7SKqZSLKxNqAWoSwzhzXJi9uBGEOmfDr1TypQFMTdUhfmaaQXtFHZJva5kDu0iOgHsymufQM28jOY82YOdSmlrmNLOw7jNY5EPbYzi7iOkKmK6QJ1iwRagULn4mjYW26mq4eevSUATjPWfXI5DmLAs0vzRgGZI9ocmaQDGNl0P519JXHO4DE+7DDHxZEA30E+0uM7L8N8I82ht7O4\\\\\\\/sAM6+klwyzqXg8LY833ZEUYkbfuTKv\\\\\\\/aSJmzXfFc8w0kGi7D8Ei8npl\\\\\\\/sYJw5Oh22K8Ja0i3qYixKGpZg1EEXIS7CiOkTsqXoJ+\\\\\\\/xgc1FtVOaxSvTUA47d3cGhkXiOb2TE0tTJTCwTaQaTotash2S00K6MuNqyyAkFWA3vypubDn\\\\\\\/hSHWRp0xBJR1mAUG1G8XG\\\\\\\/N7asJ6droHf4Vs7pTzkSi5biPc8MKGiled5QUlq0NUsu65Q8fNJ7f19TOp677\\\\\\\/+9uwBiwpid2GQdJ8FADJv9CMRzWhnFj15EkAlz++JkAIJThpxQPdL\\\\\\\/WQueovAYfdRgHTc3wv4ghBI9mVkX6vbbgxhyAqBUGxhRyx0CduNezSCIozFVPwXFZf0fw1sWJa4PSQyjwAKGCd5ShWwGwXo2F5XTl1j8w3AZOvIncCLQHU+ZOPuzsXaEYkDWK8YqLZqotoSaamwbbQX5rkok6culboweq990YloSBerjzD1v4vrN0n1Uw05MttEPoBoyDOO3OB\\\\\\\/7mzU7KfhWxspFbUuaavZz1f5MVpPQ+ywKPBl0UcWng9FUD8tigtNj6bbA8kV7WXOOHoa1dOwUHaQAoEG8JXI7bJybKj0xpSm7e4hRgGQRRw\\\\\\\/LRBpLTRyaWrdShYLJaJtLY4W5anB0LqfhoDgGmpBJ9GMxCUBSp6Wt8zg+c\\\\\\\/M43qGTTnCOgiU3Wtec9gEWVOeG+zoT6swMwGXcsVNFtTZKpPLJmo4HnDXHKwEXGHqGe6QosDcTANduXFER0dL\\\\\\\/DsatGhIdwGlps\\\\\\\/dH0Wjx3voSdQWCgpcl\\\\\\\/k+j0r8dDdTdkWfozFQEXW58ufQqGw7OgDIR7uGjPXfopOq\\\\\\\/ZVKzmOyyJuSEZSfPkaLhLsGjxgpNwfGwertd1KZlyt9J6utiOCfemypHOq3c9TvrS3zojdQuSXyxP59ajJLJVAgwB9izw2C4Qtjyn7ly+JuTE3sx+VWe+Xb8qQVNM499Chf+8G4lzAvXYrUnvXUMDyBYKdqSNhS2+nmGxuF9Fc5HZvosOpdPfXBAVenAtPpyX9RyCV+37B3D8Vvm2AjkSswy0TQHleg5WIVMG\\\\\\\/2JV5050CIhJU1gDIu3OiN89GZvOKkGNqFowr0rr8mAO6zA2yGki8Lh3Ekx\\\\\\\/oDt0W2m8eKa\\\\\\\/n4ZcRw+rWRMaHw\\\\\\\/l3RM6iqdKMxZ0n+F43\\\\\\\/XbLOydHUglqw5VMMdXhslox1YHKQ7tazctWqJXvNwxW7NSHCNGBITIhUdfHTUXTnwJp8LKWAnlymCKGzfzkZs7nP9VmGkZ3L43bcKfc2NBKf1YlYZu4KOlWvyqPHMvI+RgS1\\\\\\\/gPMgiJl\\\\\\\/MQfoMQvbBye3pjrx3ICmN+HkijFdVdoiecMjwAxTM9BPAYkW4uv8szYla7ldMdrxEOzoB\\\\\\\/OBXtlj0hYQX2Gh2akhwlF8dWqTaNCk2+6kOR9OWSdl25RoRaRvpotyuUzYorfH1ZHtK1lhENRgllKY5apdSibe1kYVNYDHeBjF6ALjMRMquNBte5wmUKqu\\\\\\\/SqbwbUAM0Q2Qg7VUfJfaVQvGxD5yhNCRqTYJqvVdKn1QXYo9wWaa7KDX9sO+C1Mpt5ukq+3tOiBGnPoCxD28QxDQZQw85DPEL9SazV9KPdPfAxxX2Yyfn1QeZIE4VpXhRzFqrYBjdBTg0+tXEWyl7PoVlh5UXKslkLI4Y5+5dk0IDk6VTJm\\\\\\\/MzkpAPBa9xZV0mW9Hhat9qia+b0lI50d8sRPE2Z8\\\\\\\/M0l\\\\\\\/VYemQvDkHFmaeT5D1vSDSYaLe6NjIyQ7MPkZ2LHpTnfMrfiAL6mmWPKXq6xNNe8PZLaNQRgzSO1SSZWgY7nRBHgvHXLI+AuPBQjkdfwhruLxkhjOzsIVGiOCcR58WQpWStv4tYG0eU0IjbqRHVYyFKPoFPaPXlhIIdBStPscG+Ts6WSwQFBhOwWpGHCf1j4vB76A3c791WWda55Sh6a8RVfzOFOsZ5Ip6m0F0PVA9R2ue9txS6uGFNue7FuqV8oEOalZ7AMG\\\\\\\/NEZerXCu\\\\\\\/ZpNGHDu5nO6sAUH4W\\\\\\\/dZ1GJaDLKYrKxhZUoA6too3H\\\\\\\/SLAU9KZRmGoo5l2tTzhdQbhmfD+Di28fSvnZIDxUdXx\\\\\\\/Wbj1RIB86HIWJVeil6DrOrw2Lcpyhj5KIwd5stFpwZgBQXX+MpWWE6D7Os6yNzktn9cGK7Adn7e6ivMjWXH4sJO++0okOUFxjV7Yki7IN6+okGjh6BenEXDT1h7eX0mgl\\\\\\\/CrBQ281UW5pAghrhNk1ETWxOVbgiNEDSVgKITx2szGJdAzNfqNbvLSE\\\\\\\/FY0Wix7ql5ldA5+6BntyCbgIMnJ6rW4mP+Sg2yo0BBeEjswQgFA4OLhMglI6YXtGU+Go6g58IfdCJMXoLTyYWFx6DtwfirPCWEwFmq7lskltZ1HZNmELxIn2siNDNJrVVCrzmxk02GPteB5\\\\\\\/\\\\\\\/1n2KGtpJuesKI348YpQ01qtxJnmIoberO6j\\\\\\\/bH4T8qbERLK7ugECKsOrJV6Iy2DVjse+RFhXYKhKsNvnipe\\\\\\\/zJmi85\\\\\\\/YmBPF5O7rDMz+uLy9MHhKEe0zunZCPIBWo7zEgGIqMlZHa5H10vHxGNYRcEwzkH3yy9wH5vrdt4KM\\\\\\\/KCh15CGzgSBxMRAgu9UpV+7IvFQf+bttUynWp70RivBxhdI7ag+9fU4QnnfM0Sjl6ynFM6ZclE92s4ewdJ3AiV6tJJbnwJBbebqCRYIkyxBbgk4yfxHkRKEmopAN9IMONkuvRX+3t64i9KmhI1+zLTZADyzyQ65adnsTZsj7iFNefOeVH8f+rWgA4FIqEYPuEs4H720SuodpJKKcTImCzVD\\\\\\\/d6kYPEaLtURQNaNzB20Wjpibj6eymL7R8eM++lvhJiyQOQgJStTDNur\\\\\\\/+WSY+AmNtCt5WFVi+6QpBfR5UfHNZOIGFEnKzWQlKFLHd0RTnjzXUuDU45rX6SSdz5sTOMkkqKdIIhWw60KtSeE6\\\\\\\/Y3rOBrr\\\\\\\/ytrsrDQwZoudqLBcRS4E3VJJAaSH2PTKV\\\\\\\/qC2k2u+B11DOGWlmvzWf3\\\\\\\/eWHZBw+J8uJFa2eq6G8szVoOU89IgpvmqPEuYNuAQ87cOVU9JH56Qu1yMOse+Ob1shNm9tr7Lr\\\\\\\/eP1UvRnld+XaQuKJp6mKR14u5lPxZwZQfVVDnhZUTTQ4fSQtDrfGCpvQzbb5sXc6mfDTVad297aA7TusekltjaCmef2vtPYKXlI2v57toIIHHTRCqEv07G6RmMqUpkfEl8LLRVGHhjepVR+mNKdGCNVlGbsOSJ19gXw3QqwUANNq1K\\\\\\\/piVYs0hqOZ7CiFfzhTErIv7t+rIjuh\\\\\\\/ovRmDY5b7gQkNUhZf68g8+hFP\\\\\\\/Z992yzgCoCfRQFyq6EF3wCPWJ3qoHbUbq0ehwsZ7z4G+6\\\\\\\/J81n3ZFnebgRe7srzElq9n29E9Ppv\\\\\\\/W5f0mdTG3VhS\\\\\\\/90+yy3PfmqEXvyovzZuwnmWYUR4xn+cbwF70J9Y1ZG4vZOhCGMS01FS0Fcuu+BlH+GcYAMgvGU1ljsVUlwjuCG6q9vVQ4BETccJoKXjq9BPnmJmUcmyuSYqmZI9oDruIJ1koN8fSuoiNxVLIY\\\\\\\/R6jTAF7FVsrD5nj3RuCKuPncTvCW5N2zxAALU191wV8+04koUDyQ2qiiee0lS5tUJQyzM5rIABf1Oy8UzWho\\\\\\\/Atyrl3yg298diK0II3JDzZNYCnKH3S48KXo4\\\\\\\/bnVx01zCCOkGAqmnSjE+woFeDPGqcrSYe4\\\\\\\/dFC+Na7Bw\\\\\\\/fahgRImU16EaLgdZO+uzVXvJ31ArBBk0PmZ\\\\\\\/Oe4cxAOZwxVqqNXhuPbF1C3StpYIVCEdVEFlgdxOGgh8tQJjZjIX28G4WjNnSA3I4tD0T\\\\\\\/GrTrMfMEFjOzjonHsNGEGz8VX7lsMhbTChzPiQPSuEfmde8ZN3ZT2rygIFBB15j22e4ypAHp6CDDcGK5x157hWavHuUng2wTxXYtq9uuLQetNxt9fp1cRvv2HLVeaDSF7ffLSRp0rgtocQfMqk3Q+tym6V9gD0d0VaRHbs6KlfufprpeiJY8U6UeFprascIzDKLUakmmsO8FnNm0KCeuxt0cu0Eu9rw6ZRUIowFtk+\\\\\\\/xdpMsQmVRD8E6DufLGnGgqSpB1nY+GyRxRCdQgZE9ei02zq0vLG+FQM+4A65lcY7\\\\\\\/Q\\\\\\\/fYzyYI89nfGvwKIOo\\\\\\\/sqR6gGivDkRHdnnWteN\\\\\\\/Tv0oC5Zhkm8Y1HYJxWFIjO4GGFdmbfI\\\\\\\/lXiUutSCF+PNBuxIH8RplQ5obk38\\\\\\\/LjMPeJE+v9qJEXA9BCQ0OpwEv\\\\\\\/I9ccKBkVHcjpPW6PRPneuwDrn5NuD80QR5bWX5aIacEFN4IC\\\\\\\/Tl\\\\\\\/TBKal\\\\\\\/FFgtIMPwzd8bTv+8eQckUGUzR8xNJJpYNqvdkE9C3fSZ93i3FX4IYv25ON+Zq6CZ\\\\\\\/Uz36PoQMlXshObXlSnZe6S7nGXN7iBXQr6EIHra08HmTy5D0T8a+UGdUgEv9sahr9ij8nRuEbP2MxEYt+BAcEnRqvOW0l9CgkZS9uS8oAGU94707U8HA1GC\\\\\\\/Mtpxy0F74zminvukVVGivWzfIKYXX\\\\\\\/JXDjQafpHCOx\\\\\\\/CdfLLe5USatsPoTnKP5u3Kh75k+x2CZQ4EHLBP2MYYx4lKb9oS1aTHO4bE9lO43ybQCP7CCgxIIb5ivIVvYkGq2raAVj7eM4BQ7mjwUINMy2Uvigx13DJLUu3BZ\\\\\\\/VOHybHMPyDHKzHs\\\\\\\/cWfCJc99FKxQPhiQwQVYsBy\\\\\\\/99MjIbilir2yKrV\\\\\\\/nYCxA6j2QVGq9N9fYkFmB\\\\\\\/\\\\\\\/4v8o1v\\\\\\\/Z96tF0wtXvGExWd5ztOSqrnKsmPslkSnwsVZm+cWVouJT1i3yW5AMM8E8n+kNPAh8IqRIOo+tTruS2SXyr8yHfxoI+UZ\\\\\\\/DJlm\\\\\\\/+N66SxWMwB\\\\\\\/2nMJUmEeJasuFwrszSsT1XV24oMNA1UwF+wo3d6sQJcOlJqGq8+7mWSR4gCJ0+5+\\\\\\\/PSX\\\\\\\/W8CKoDMpKKImvuFAxJUTauaRdHY3y9Av5zIc+4SrApUWPjXkAk2e8WssaL9At+1qjWvwa7FFz9NVYdJbPqgOplwqnYD+M+Y0UedMPG1Y\\\\\\\/ee7t3u7PaIDyEW6WjogrxtMY\\\\\\\/E+bwGIncMLXr9WWw+QXz+rkrKurQE7\\\\\\\/f\\\\\\\/Dwh\\\\\\\/av5tYLk6\\\\\\\/n2iRKfY+4H\\\\\\\/Rl7q90jm1mWPKexS\\\\\\\/McHAPgIkQYALMTDF143h1TKDewwaKtwC10jUzC46Vg4rHguZIl+20UGMbPFICswPT2q5MyxF3ZSnXqtn1KbCnK+UQdheUsZyPjwC825R36t\\\\\\\/4zP7QlLYv\\\\\\\/amzjJufWwgG5MfnJGlFSOXeJ1y5dbc6pq2voxQ2KWOs6FMfk71Q5Kw\\\\\\\/qde63ejVM1gqLIgwF3bTjPRRXZ5NsJbs9o11qus8HkCuUz8vzE10grZ8ujhgzbli+nRnuSFZ+xqqBuMRR3Ozvx5o\\\\\\\/niSV960Wc2Cg4hBJY\\\\\\\/ZMMH9o2lMZIqaLWuBUL1i5Q65l+lekfjrJPN7tY7Wv42b8TUUkoQVsz3\\\\\\\/SNHOf44qEjd3T+AZjuw9Ur57Ad\\\\\\\/N0bz4LkA1MLnCYbPMybos9vOp4C9blIyet1wqU3+eK2iQ6Q1arUo+IlDf71R4IodY7zmD2ve3E5Wcl8BR4i\\\\\\\/G076+x93jdty1+mtn71MKROBhiR7C3LMF6upq4LiKa68SoX92WU3Z2ln6EyDvQjI0xfjWGvFpZwIKUaLLKhd5uT+8DntYiR+KncgRe\\\\\\\/49stzzHqR8dK290Bjufls+9dNex0kSf4bBYolaWgVIOJh+XyDJidOwuSbOP2M5AXw6qJTOub\\\\\\\/+BAbyINAt2eOA+cs3Q+IqffjZYxxskIG7Bl5GXxBBXrwGDyjUJrUP+1mjA9hm1M8dzdLBMUwYOXeXdPK0MZ6MPClP3Oxduo33QxKXXDD2z2fatLlt6B9Qt6hs\\\\\\\/slCoi5ckppRVVqB\\\\\\\/mV9jcMKJ\\\\\\\/UuMRax+B7BrXSBKGOK4AoYlTiGlPiQfgWRGNq4uMDAZyy3waVXc+nOEzv4J6yLXhuZAHS4b\\\\\\\/vShWHC8uON5hlF1oKFproRhL5jMu93PAn56wZJPQMORDDavzPaiqPkHPHNJytxKQe\\\\\\\/ep8Ay4PvdhxSrlLOP4hOh8iFmEjY5LCVlh5vOh9bEHkbiJoA9coxN81jUypQSRQHMNK\\\\\\\/ioBlgVF6zABNUpaFiIjZETVFix90YoBxwlq55k5A2hhKwBCsuiVgvHn8Z761699Rj6IkVKT2LG65awjn77OVBmYt0WcoJIcm7wvOVkmcXm7MwarizOZ82CrUcoIdCRoQPtevWGjmVz7JQJZaZHItOlbln4M4ukJjNR\\\\\\\/1GUECdyHsNhXcgqbT3AdjeqAqzr1TRQwwTfx0lQ55Ef7gno7FKdk4\\\\\\\/kZ8AY4w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PvKmu5Mu\\\\\\\/UWOb3Gq6M4rjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002224dc0fe5c5775d4cb1b9fdd165193d91e02732384d90cc55c159a1c4c43ccbba\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022C307sPbvRH5si7uTkfd3qklEjeAkHh+JxTBvK7mkAg6P8i77Im18S4+DEQ1rhOV3e2H0o3OMoRdp5z3qJfB\\\\\\\/Hw2t2wG8IHUAuSkyCq3zzYbIgOPMpAs+9GHVzl\\\\\\\/a\\\\\\\/CM7M56xvkA1vp8zWA3mnQY6nH0qFo5NuV5umTfr6m6mhdq59THjla6WUS4zirBVhNRlTbmwjJdC859IDk0amcS477MMWEiWrGKgvA\\\\\\\/8Mq01Uw8XsTzOzsJ0HoFUVAxIz0JuNd9u0N9H4hoAU\\\\\\\/u7lZH0VR9wOT5+WaRmcIZElV8hhW02GfWjGUGTLZcckACo8caUyeZBUDzbIehM7+XSFuD9ncd8fXR2nDIRLsx9mrdWjzrJlCZGofsRdpAeLFIHqYe8X3N4QOTN1fmcrkg8NOi8KxHQSb2O2oSVIIegzLyz\\\\\\\/LfJ+B6jDSB5lR1oK2BSQCAN04X5hVes1vYeYDgRaBIZzDyogLL6dZXppBEWEiTZ0w7KbOO36I9pw5zjcJWyFh9w+FIMXyyGfXNposJXXyzoVA86Tl9VU4BL2R9k855FO7uLnf9MS+zrCJEYFRnnsL+\\\\\\\/ltTUsKTQGWPQJaTgNpwqT\\\\\\\/5T2bYarNREFrXskoRnBy4L+40\\\\\\\/ciezFWUnoW+RMhTTAWtxar2V54SBCmfsTpI2AqNpECSNcrsaCfHVsVIX949xo0wrBU3mGs0wp\\\\\\\/pQG2Eivv2p3B1\\\\\\\/+Unv2WIPHWSWKlzy46mq2FRnfqwf4gK\\\\\\\/6mCwbfNB6rpvZKGCVQVbNhXFAblDqfXJ+Yv\\\\\\\/LOvJQRRENmxw2mQABMtmkHlcwaHLLGtKJHwcdfem8lkuzxps\\\\\\\/grWiKTFlOhr4d1+i3T0OqYhFwmoFx1mcZlmtA+CnZ1mtPdJJYYafQuNQTG\\\\\\\/8x\\\\\\\/vkoBDNrTbtgOeA7PU3yBZiF9DeuYhU8Lr4Tsiju56fMM5NXHwrsOFG008QvIL2c6rUc4gsZbAlPa0ZdDMDX19dF1lVMU0lkIQER20ON\\\\\\\/BfZFEBB9MpF+lnMuTKRX5OJJyvKICEPhEep2JanYto2m70q3BskQRxBzYytFyyumdFuM\\\\\\\/rUgKKOV6khjEmN+hJ6nd3d80CpOD\\\\\\\/gXK\\\\\\\/8AlWM0I3BMhK0ZnBvyGT7QxyMGPt8sohB85pCcj0zo0oi0YMgEsVfDEjoGqo9ICSrPWiMNWD3BjDQcwoDYL1auiu5DcNIqP7m\\\\\\\/FLcBnyfDfyqWJDWMiMe7ZszSZ92V2Lhiue19wwG7vAgcNNoHnEzC5\\\\\\\/OdS8izNBElUZcX\\\\\\\/FyqOrfNCiWvV97EF7mbobwSt5c+sBdE3MIOi\\\\\\\/VhhhTh4MlhAJ8V\\\\\\\/cpJBNQykCycgrD573xSnllvCuSYfbLyJyyYyiZBFwL14Em59l8tNoBVS605qpj0Vqh7MYEP5YkCK66\\\\\\\/VHzWWHGTwoUk9huWWHItegH57ffD8k1TyrEf1Z1ejUViExptgjK9bDPsTuOX4tZDMZ8AdVfWW1c09t4H2UyFe5i4oEpnoJVBY5EOt9hfbiW2nhokgtb+J05wF6vd9iGhh2CLl0VgkGq\\\\\\\/Iik1C77YmL5fEhvKd\\\\\\\/+fwoE5bXxluoaWZ2w+Y\\\\\\\/ClYRxo4x1LBICOWE4SMadULH6JvqhTTppf+WNHV6r5Qp0XICFMHB3H5oafYSo3sK3U5EHSiy6p0sOCcWbOsIZhlZQvaT9czo0F7irWdL1KVjfA\\\\\\\/rsHa+Y3Gs3vsMrOMaKer2HZGZGlMR37KZi+1x1+vujEKzGkRvuEePT8Artz0CWncygJ+C0OUTa\\\\\\\/genKZ7LU33hsTMqZk+wMcMwMiiYsz3jU97+fZAff+A7FN09wY46GZU1vMgKjixjBcN9d+n9x6l+trqBrBAsHBLDORB2tgop9Y01WOGOdsRuRo1fWSxXQiQike2+3KSDp4Kw9Ve7LyeVKjlugej9zRMNDKwlzmZwQTa4AkvTVqJXFfR1MaWxQxc0Nc+Dt6T\\\\\\\/nWMwFT21tbluLX6Hs1uwY5OeJhxei3qcXp26R281zm0YU5bhcNyPXMYw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022E1UyYb1makfi85OXLHKH6A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002266daadea1290199903d2589a0486312f14211c8dfb7167b452d69c5b34527a41\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490978 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3311" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aOfLkmSK0fWK-O8bSbh4sgaW70yX_3rvBE5HmF2Art4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3311" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NzJ3sQv0qhZpTO6za7hQbtZyfN-YaV_1Xv68LEm3Bwk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3293" title="Functional Interactions Between rsks-1/S6K, glp-1/Notch, and Regulators of Caenorhabditis elegans Fertility and Germline Stem Cell Maintenance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3321" title="Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490978" id="node490978" data-pisa="ggg;8/10/3311" data-pisa-master="ggg;g3.118.200656" data-apath="/ggg/8/10/3311.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jonathan A. Arnesen</span>, <span class="highwire-citation-author" data-delta="1">Joanna Małagocka</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6037-0092" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrii Gryganskyi</span>, <span class="highwire-citation-author" data-delta="3">Igor V. Grigoriev</span>, <span class="highwire-citation-author" data-delta="4">Kerstin Voigt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jason E. Stajich</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3326-5729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Henrik H. De Fine Licht</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3311-3319;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200656 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490978" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan A. Arnesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BA.%2BArnesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arnesen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BA.%2BArnesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joanna Małagocka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BMa%C5%82agocka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%C5%82agocka%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoanna%2BMa%25C5%2582agocka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrii Gryganskyi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Duke University, Durham, North Carolina, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrii%2BGryganskyi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gryganskyi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrii%2BGryganskyi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6037-0092" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrii Gryganskyi</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Igor V. Grigoriev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>US Department of Energy Joint Genome Institute, Walnut Creek, California, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BV.%2BGrigoriev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grigoriev%20IV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BV.%2BGrigoriev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kerstin Voigt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jena Microbial Resource Collection (JMRC), Leibniz Institute for Natural Product Research and Infection Biology - Hans Knoell Institute, Adolf-Reichwein-Str.23, 07745 Jena, Germany</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Microbiology, Friedrich Schiller University, Neugasse 25, 07743 Jena, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerstin%2BVoigt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voigt%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerstin%2BVoigt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jason E. Stajich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Pathology and Microbiology, University of California, Riverside, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BStajich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stajich%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BE.%2BStajich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jason E. Stajich</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henrik H. De Fine Licht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BH.%2BDe%2BFine%2BLicht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Fine%20Licht%20HH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenrik%2BH.%2BDe%2BFine%2BLicht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3326-5729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Henrik H. De Fine Licht</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hhdefinelicht@plen.ku.dk" class="" data-icon-position="" data-hide-link-title="0">hhdefinelicht@plen.ku.dk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3311" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490978" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3311.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490978" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3311.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490978" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3311.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490978" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3311.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490978" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Insect-pathogenic fungi use subtilisin-like serine proteases (SLSPs) to degrade chitin-associated proteins in the insect procuticle. Most insect-pathogenic fungi in the order Hypocreales (Ascomycota) are generalist species with a broad host-range, and most species possess a high number of SLSPs. The other major clade of insect-pathogenic fungi is part of the subphylum Entomophthoromycotina (Zoopagomycota, formerly Zygomycota) which consists of high host-specificity insect-pathogenic fungi that naturally only infect a single or very few host species. The extent to which insect-pathogenic fungi in the order Entomophthorales rely on SLSPs is unknown. Here we take advantage of recently available transcriptomic and genomic datasets from four genera within Entomophthoromycotina: the saprobic or opportunistic pathogens <em>Basidiobolus meristosporus</em>, <em>Conidiobolus coronatus</em>, <em>C. thromboides</em>, <em>C. incongruus</em>, and the host-specific insect pathogens <em>Entomophthora muscae</em> and <em>Pandora formicae</em>, specific pathogens of house flies (<em>Muscae domestica</em>) and wood ants (<em>Formica polyctena</em>), respectively. In total 154 SLSP from six fungi in the subphylum Entomophthoromycotina were identified: <em>E. muscae</em> (n = 22), <em>P. formicae</em> (n = 6), <em>B. meristosporus</em> (n = 60), <em>C. thromboides</em> (n = 18), <em>C. coronatus</em> (n = 36), and <em>C. incongruus</em> (n = 12). A unique group of 11 SLSPs was discovered in the genomes of the obligate biotrophic fungi <em>E. muscae</em>, <em>P. formicae</em> and the saprobic human pathogen <em>C. incongruus</em> that loosely resembles bacillopeptidase F-like SLSPs. Phylogenetics and protein domain analysis show this class represents a unique group of SLSPs so far only observed among Bacteria, Oomycetes and early diverging fungi such as Cryptomycota, Microsporidia, and Entomophthoromycotina. This group of SLSPs is missing in the sister fungal lineages of Kickxellomycotina and the fungal phyla Mucoromyocta, Ascomycota and Basidiomycota fungi suggesting interesting gene loss patterns.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/subtilase" class="hw-term hw-article-keyword hw-article-keyword-subtilase" rel="nofollow">Subtilase</a></li><li class="kwd"><a href="/keyword/insect-pathogen" class="hw-term hw-article-keyword hw-article-keyword-insect-pathogen" rel="nofollow">insect-pathogen</a></li><li class="kwd"><a href="/keyword/early-diverging-fungi" class="hw-term hw-article-keyword hw-article-keyword-early-diverging-fungi" rel="nofollow">early-diverging fungi</a></li><li class="kwd"><a href="/keyword/proteases" class="hw-term hw-article-keyword hw-article-keyword-proteases" rel="nofollow">proteases</a></li><li class="kwd"><a href="/keyword/phylogenomics" class="hw-term hw-article-keyword hw-article-keyword-phylogenomics" rel="nofollow">phylogenomics</a></li></ul><p id="p-3">Insect pathogenic fungi use a broad array of enzymes to penetrate the host cuticle and gain entry to the soft tissues inside (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Charnley 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">St. Leger <em>et al.</em>, 1986b</a>). In many cases, serine proteases are among the first enzymes to be secreted in the early stages of infection in order to cleave and open up chitin-associated proteins in the procuticle (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">St. Leger <em>et al.</em>, 1986a</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Vega <em>et al.</em>, 2012</a>), which later is followed by extensive lipase and chitinase enzymatic secretions (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Charnley 2003</a>). In particular, subtilisin-like serine proteases (SLSPs) have been considered important virulence factors in pathogenic fungi (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>). The first SLSPs from insect pathogenic fungi were identified in <em>Metarhizium anisopliae</em> (ARSEF2575), which secretes SLSPs as some of the key proteases during fungal growth on insect cuticle (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Charnley 2003</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">St. Leger <em>et al.</em>, 1986a</a>). Comparative genomic approaches have identified significant expansions of the SLSP gene family in the genus <em>Metarhizium</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bagga <em>et al.</em>, 2004</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hu <em>et al.</em>, 2014</a>), the insect pathogenic fungus <em>Beauveria bassiana</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Xiao <em>et al.</em>, 2012</a>), two nematode-trapping fungi <em>Monacrosporium haptotylum</em> and <em>Arthrobotrys oligospora</em> that are able to penetrate the chitinaceous cell wall of soil nematodes (Meerupvati <em>et al.</em> 2013), and dermatophytic fungi such as <em>Arthroderma benhamiae</em> and <em>Trichophyton verrucosum</em> that can cause nail and skin infections in humans and animals (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burmester <em>et al.</em>, 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Desjardins <em>et al.</em>, 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Martinez <em>et al.</em>, 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sharpton <em>et al.</em>, 2009</a>). Fungi that are able to utilize chitin-rich substrates, including many insect pathogenic fungi, thus appear to often be associated with a diversified and expanded set of SLSPs.</p><p id="p-4">Although SLSPs are expanded among insect pathogenic fungi, this group of proteases is ubiquitous among eukaryotic organisms. Most SLSPs are secreted externally or localized to vacuoles, and especially in saprobic and symbiotic fungi SLSPs constitute an important component of the secretome (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li <em>et al.</em>, 2017</a>). According to the MEROPS peptidase classification, the S8 family of SLSPs together with the S53 family of serine-carboxyl proteases make up the SB clan of subtilases (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rawlings <em>et al.</em>, 2016</a>). The S8 family of SLSPs is characterized by an Asp-His-Ser catalytic triad (DHS triad), which forms the active site and is further divided into two subfamilies S8A and S8B. Subfamily S8A contains most S8 representatives, including the well-known Proteinase K enzyme that is widely used in laboratories as a broad-spectrum protease. The S8B SLSPs are kexins and furins which cleave peptides and protohormones (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jalving <em>et al.</em>, 2000</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Muszewska <em>et al.</em>, 2017</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">2011</a>). Based on characteristic protein domain architectures and protein motifs surrounding the active site residues, the large S8A subfamily of SLSPs is further divided into several groups such as proteinase-K and pyrolysin. Besides these two major groups of proteinase K-like and pyrolisin subfamilies, six new groups of subtilase genes designated <em>new 1</em> to <em>new 6</em> have recently been found (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li <em>et al.</em>, 2017</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>). The analysis of fungal genome data from a wide taxonomic range has shown that the size of the proteinase K gene family has expanded independently in fungi pathogenic to invertebrates (Hypocreales) and vertebrates (Onygenales) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Muszewska <em>et al.</em>, 2017</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Sharpton <em>et al.</em>, 2009</a>). Closely related invasive human-pathogenic fungi, however, do not show the same expansions and related pathogens and non-pathogens can show the same expansions (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Whiston and Taylor 2016</a>). This suggests that the number of SLSPs that a fungus possess is not directly related to pathogenicity, but instead is associated with the use of dead or alive animal tissue as growth substrate (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Li <em>et al.</em>, 2017</a>; <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>).</p><p id="p-5">Most anamorphic insect-pathogenic fungi in the order Hypocreales (Ascomycota) are generalist species with a broad host-range capable of infecting most major orders of insects (<em>e.g.</em>, <em>M. robertsii</em> and <em>B. bassiana</em>) or specific to larger phylogenetic groups (<em>e.g.</em>, the locust-specific <em>M. acridum</em> or the coleopteran pathogen <em>B. brongniartii</em>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Boomsma <em>et al.</em>, 2014</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hu <em>et al.</em>, 2014</a>). The above-mentioned inferences of fungal SLSP evolution rely almost exclusively on insights from Ascomycota, and consequently have strong sampling bias toward generalist insect-pathogenic fungi. In contrast, the other major clade of insect-pathogenic fungi in the subphylum Entomophthoromycotina (Zoopagomycota, formerly Zygomycota) consists almost exclusively of insect-pathogens and many are extremely host-specific, naturally only infecting a single or very few host species (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Spatafora <em>et al.</em>, 2016</a>). The dearth of genomic data for Entomophthoromycotina has previously precluded their inclusion in comparative genomic analyses (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">De Fine Licht <em>et al.</em>, 2016</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gryganskyi and Muszewska 2014</a>). Here we take advantage of recently available transcriptomic and genomic datasets from four genera within Entomophthoromycotina: the saprobic <em>Basidiobolus meristosporus</em>, the saprobic and opportunistic pathogens, <em>Conidiobolus coronatus</em>, <em>C. thromboides</em>, <em>C. incongruus</em>, and the host-specific insect pathogens <em>Entomophthora muscae</em> and <em>Pandora formicae</em>, specific pathogens of house flies (<em>Muscae domestica</em>) and wood ants (<em>Formica polyctena</em>), respectively. We use phylogenetics and protein domain analysis to show that the obligate biotrophic fungi <em>E. muscae</em>, <em>P. formicae</em> and the saprobic human pathogen <em>C. incongruous</em>, in addition to more “classical” fungal SLSPs, harbor a unique group of SLSPs that loosely resembles bacillopeptidase F-like SLSPs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sequence database searches for subtilisin-Like serine proteases</h3><p id="p-6">We identified putative subtilisin-like serine proteases (SLSPs) from six fungi in the subphylum Entomophthoromycotina: <em>Entomophthora muscae</em>, <em>Pandora formicae</em>, <em>Basidiobolus meristosporus</em>, <em>Conidiobolus coronatus</em>, <em>C. incongruus</em> and <em>C. thromboides</em>. First, protein family (pfam) domains were identified in the <em>de-novo</em> assembled transcriptomes of <em>E. muscae</em> KVL-14-117 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Fine Licht <em>et al.</em>, 2017</a>) and <em>P. formicae</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Małagocka <em>et al.</em>, 2015</a>) using profile Hidden Markov Models with hmmscan searches (E-value &lt; 1e-10) against the PFAM-A database ver. 31.0 using HMMER ver. 3 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eddy 1998</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Finn <em>et al.</em>, 2016</a>). All sequences in the transcriptome datasets containing the S8 subtilisin-like protease domain (PF00082) were identified and included in further analyses. Second, all sequences that contain the PF00082 domain were obtained from the genomes of <em>B. meristosporus</em> CBS 931.73 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mondo <em>et al.</em>, 2017</a>), <em>C. coronatus</em> NRRL28638 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chang <em>et al.</em>, 2015</a>), and <em>C. thromboides</em> FSU 785 from the US Department of Energy Joint Genome Institute <em>MycoCosm</em> genome portal (<a href="http://jgi.doe.gov/-fungi">http://jgi.doe.gov/-fungi</a>). Third, predicted coding regions in the genome sequence of <em>C. incongruus</em> CDC-B7586 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chibucos <em>et al.</em>, 2016</a>), were searched for the presence of the S8 subtilisin-like protease domain (PF00082) as described above.</p><p id="p-7">Sequences encoding an incomplete Asp-His-Ser catalytic triad (DHS triad) characteristic of S8 family proteases were regarded as potential pseudogenes and excluded from further analysis. Although a subset may represent neofunctionalizations, the risk of including false-positive SLSPs in down-stream analyses were considered too high. A preliminary protein alignment made with ClustalW (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Larkin <em>et al.</em>, 2007</a>) using default parameters and construction of a Neighbor-Joining tree using Geneious 4.8.5 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kearse <em>et al.</em>, 2012</a>) revealed a highly divergent group of <em>P. formicae</em> SLSP-sequences that had significant homology with insect proteases (blastp, E-value &lt; 1e-6, ncbi-nr protein database, accessed June 2017). These putative insect-sequences likely originate from the ant host, <em>Formica polyctena</em>, and represents host contamination that were not filtered out from the dual-RNAseq reads used to construct the <em>P. formicae</em> transcriptome (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Małagocka <em>et al.</em>, 2015</a>). These divergent sequences were therefore removed and excluded from further analysis.</p></div><div id="sec-3" class="subsection"><h3>Protein domain architecture and sequence clustering</h3><p id="p-8">The domain architectures of all putative SLSPs identified within Entomophthoromycotina were predicted using Pfam domain annotation. The presence of putative signal peptides for extracellular secretion were predicted using SignalP (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Petersen <em>et al.</em>, 2011</a>). A Markov Clustering Algorithm (MCL) was used to identify clusters of similar proteins among putative SLSPs identified within Entomophthoromycotina. Clustering using MCL is based on a graph constructed by an all-vs-all-BLAST of SLSPs (BLASTP, E-value &lt; 1e<sup>-10</sup>). The Tribe-MCL protocol (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Enright <em>et al.</em>, 2002</a>) as implemented in the Spectral Clustering of Protein Sequences (SCPS) program (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nepusz <em>et al.</em>, 2010</a>) was used with <em>inflation</em> = 2.0. The <em>inflation</em> parameter is typically set between 1.2 – 5.0 (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Nepusz <em>et al.</em>, 2010</a>), and controls the “tightness” of the sequence matrix, with lower values leading to fewer clusters and higher values to more sequence clusters. To putatively assign protease function to the newly identified Entomophthoromycotina sequence clusters, the Tribe-MCL protocol was used to identify clusters of similar proteins between the putative SLSPs identified within Entomophthoromycotina and 20,806 protease sequences belonging to the peptidase subfamily S8A obtained from the MEROPS database, accessed November 2017 (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Rawlings <em>et al.</em>, 2016</a>). Investigation of the MEROPS protease sequences that clustered together with the identified Entomophthoromycotina sequence clusters allowed putative protease holotype information to be assigned to the identified clusters.</p></div><div id="sec-4" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-9">All identified putative Entomophthoromycotina SLSP coding nucleotide sequences were aligned in frame to preserve codon structure using MAFFT (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Larkin <em>et al.</em>, 2007</a>). Unreliable codon-columns with a Guidance2 score below 0.90 in the multiple sequence alignment were removed (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Penn <em>et al.</em>, 2010</a>). The best model for phylogenetic analysis was selected by running PhyML with <em>GTR</em> as substitution model and with-or-without Gamma parameter and a proportion of invariable sites (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guindon and Gascuel 2003</a>). The optimal substitution model based on the Bayesian Information Criterion (BIC) score (<em>GTR+G</em>) was determined using TOPALI 2.5 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Milne <em>et al.</em>, 2009</a>) and used in maximum likelihood analysis using RaxML with 10,000 bootstrap runs (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stamatakis 2014</a>).</p><p id="p-10">To identify branches that potentially contain signatures of positive selection among SLSP sequences we used maximum likelihood estimates of the dN/dS ratio (ω) for each site (codon) along protein sequences. A specific lineage (branch) was tested independently for positive selection (ω &gt; 1) on individual sites by applying a neutral model that allows ω to vary between 0 – 1 and a selection model that also incorporates sites with ω &gt; 1 using the software <em>codeml</em> implemented in PAML 4.4 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Yang 2007</a>). Statistical significance was determined with a likelihood ratio test of these two models for the tested lineage.</p><p id="p-11">To infer phylogenetic relationship of identified putative Entomophthoromycotina SLSPs with fungal SLSPs from other invertebrate-associated ascomycete fungi, two approaches were used. First, all S8A SLSPs in the MEROPS database from the insect-pathogenic genera <em>Metarhizium</em> (n = 240), <em>Cordyceps</em> (including <em>Beauveria</em>) (n = 44), <em>Ophiocordyceps</em> (n = 11), and the nematode-trapping genera <em>Arthrobotrys</em> (n = 33), and <em>Monachrosporium</em> (n = 8), were clustered by sequence identity with the identified SLSPs from Entomophthoromycotina (n = 152) using the MCL approach previously described. Second, from the three identified SLSP groups (A, B, and C) within Entomophthoromycotina, 10 representative group A SLSPs and all sequences from group B (n = 11) and group C (n = 11) were searched against the entire S8A MEROPS database (blastp, e-value = 1e-10). The non-redundant top-ten hits from this search (n = 149) were aligned with the Entomophthoromycotina query SLSPs (n = 32) and representative invertebrate-associated SLSPs from the clustering analysis (n = 350) using MAFFT. Unreliable columns with a Guidance2 score below 0.80 in the multiple sequence alignment were removed (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Penn <em>et al.</em>, 2010</a>). The optimal protein substitution model based on the Bayesian Information Criterion (BIC) score (<em>LG+i</em>) was determined using ModelFinder (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kalyaanamoorthy <em>et al.</em>, 2017</a>) and used in maximum likelihood analysis using RaxML with 100 bootstrap runs (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Stamatakis 2014</a>).</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-12">All data used are available through the US Department of Energy Joint Genome Institute <em>MycoCosm</em> genome portal (<a href="http://jgi.doe.gov/-fungi">http://jgi.doe.gov/-fungi</a>), or has been previously published (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Małagocka <em>et al.</em>, 2015</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chibucos <em>et al.</em>, 2016</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">De Fine Licht <em>et al.</em>, 2017</a>). SLSP sequence data and Tribe-MCL outputs analyzed in this manuscript are provided as a zip-compressed single file ArnesenEtalSupplementaryData.zip in the supplementary material. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6949037">https://doi.org/10.25387/g3.6949037</a>.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><p id="p-13">We identified 154 SLSP sequences from six fungi in the subphylum Entomophthoromycotina: <em>E. muscae</em> (n = 22), <em>P. formicae</em> (n = 6), <em>B. meristosporus</em> (n = 60), <em>C. thromboides</em> (n = 18), <em>C. coronatus</em> (n = 36), and <em>C. incongruus</em> (n = 12). Close inspection of the active site residues revealed two <em>C. incongruus</em> sequences (Ci7229 and Ci12055), which contained the active site DHS residues in the motifs <span class="underline">Asp</span>-Asp-Gly, <span class="underline">His</span>-Gly-Thr-Arg, and Gly-Thr-<span class="underline">Ser</span>-Ala/Val-Ala/Ser-Pro characteristic of the S8B subfamily of S8 proteases. These two sequences also contained the P domain (PF01483) indicating that they are S8B kexin proteases. All other 152 identified Entomophthoromycotina S8 protease sequences contained active site residues closely resembling the motifs Asp-Thr/Ser-Gly, His-Gly-Thr-His, and Gly-Thr-Ser-Met-Ala-Xaa-Pro characteristic of the S8A subfamily. Cluster analysis of these 152 S8A-protease sequences identified three groups of proteins that were designated as group A, B and C, with 130, 11 and 11 sequences in each cluster, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). These group memberships do not change when the Tribe-MCL <em>inflation</em> parameters varied within a range of 1.5 – 6.0 suggesting that these three groups were clearly distinct. Phylogenetic analysis of the identified 152 S8A SLSPs using maximum likelihood methods also recovered the same three distinct lineages (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Evidence of positive selection acting on specific enzyme residues on the branches leading to these clusters were not detected with branch-site tests (2ΔlnL &gt; 1.53, <em>P</em> &gt; 0.217).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum likelihood phylogeny calculated with RAxML and based on a 2,379 bp alignment of 152 subtilisin-like serine protease codon nucleotide sequences from Entomophthoromycotina that contain the peptidase S8/S53-subtilisin (PF00082) domain. Branches are colored for eachs species as Entomophthora muscae (Blue), Pandora formicae (Purple), Conidiobolus coronatus (Pink), C. thromboides (brown), C. incongruus (orange), and B. meristoporus (Green). For each SLSP, the accession number and protein domains additional to PF00082 are shown. The three identified clusters: Protease K cluster (A), Pyrolysin/osf protease cluster (B), and the new bacillopeptidase-like Entomophthorales cluster (C), are marked in the gray circle surrounding the tree and with gray background for cluster B and C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846824602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximum likelihood phylogeny calculated with RAxML and based on a 2,379 bp alignment of 152 subtilisin-like serine protease codon nucleotide sequences from Entomophthoromycotina that contain the peptidase S8/S53-subtilisin (PF00082) domain. Branches are colored for eachs species as &lt;em&gt;Entomophthora muscae&lt;/em&gt; (Blue), &lt;em&gt;Pandora formicae&lt;/em&gt; (Purple), &lt;em&gt;Conidiobolus coronatus&lt;/em&gt; (Pink), &lt;em&gt;C. thromboides&lt;/em&gt; (brown), &lt;em&gt;C. incongruus&lt;/em&gt; (orange), and &lt;em&gt;B. meristoporus&lt;/em&gt; (Green). For each SLSP, the accession number and protein domains additional to PF00082 are shown. The three identified clusters: Protease K cluster (A), Pyrolysin/osf protease cluster (B), and the new bacillopeptidase-like Entomophthorales cluster (C), are marked in the gray circle surrounding the tree and with gray background for cluster B and C.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3311/F1.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3311/F1.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491033" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-14" class="first-child">Maximum likelihood phylogeny calculated with RAxML and based on a 2,379 bp alignment of 152 subtilisin-like serine protease codon nucleotide sequences from Entomophthoromycotina that contain the peptidase S8/S53-subtilisin (PF00082) domain. Branches are colored for eachs species as <em>Entomophthora muscae</em> (Blue), <em>Pandora formicae</em> (Purple), <em>Conidiobolus coronatus</em> (Pink), <em>C. thromboides</em> (brown), <em>C. incongruus</em> (orange), and <em>B. meristoporus</em> (Green). For each SLSP, the accession number and protein domains additional to <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=PF00082&amp;atom=%2Fggg%2F8%2F10%2F3311.atom">PF00082</a> are shown. The three identified clusters: Protease K cluster (A), Pyrolysin/osf protease cluster (B), and the new bacillopeptidase-like Entomophthorales cluster (C), are marked in the gray circle surrounding the tree and with gray background for cluster B and C.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">To further characterize the three SLSP groups, the protein domain architecture of each of the 152 protease sequences were analyzed. The presence of a proteinase-associated (PA, pfam:PF02225) domain was only found in Group B that strongly suggested this cluster with 11 members is comprised of pyrolisin and osf proteases (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>). An additional Tribe-MCL cluster analysis (inflation = 1.2) of the 152 Entomophthoromycotina and all S8A proteases in the MEROPS database clustered these 11 Entomophthoromycotina sequences into a group of 779 MEROPS proteases. This group of proteases contained 42 members of the fungal S08.139 (PoSl-(<em>Pleurotus ostreatus</em>)-type peptidase) holotype (Supplementary data). The protein domain architecture of group A contained 130 Entomopthoromycotina SLSPs, and many of these 130 SLSPs contained a secretory signal and/or a peptidase inhibitor (Inhibitor_I9, pfam: PF05922) domain. The 130 Entomophthoromycotina SLSPs clustered with 3,046 MEROPS proteins of well-known fungal entomopathogenic protease holotypes (Supplementary data), including cuticle-degrading peptidase of nematode-trapping fungi (S08.120), cuticle-degrading peptidase of insect-pathogenic fungi in the genus <em>Metarhizium</em> (S08.056), and subtilisin-like peptidase 3 (<em>Microsporum</em>-type; S08.115). The 130 Entomophthoromycotina Group A SLSPs thus belong to the common Proteinase K group of S8A proteases (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>) based on conservation of the active site residues (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), and cluster membership of the MEROPS Tribe-MCL analysis.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Active site and domain co-occurrence variability of the three Tribe-MCL clusters identified among 152 Entomophthoromycotina subtilisin-like serine proteases. The columns DTG, GHGTH, and SGTS represents the closest amino acid sequence for each of the amino acids from the DHS catalytic triad. A. Amino acid alignment of the active site residues for the three identified groups (A-C) of SLSPs within Entomophthoromycotina. Accession codes are color coded as: Orange &#x2013; C. incongruus, Blue &#x2013; E. muscae, and Purple &#x2013; P. formicae. B. Sequence motifs of the active site residues for each group." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846824602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Active site and domain co-occurrence variability of the three Tribe-MCL clusters identified among 152 Entomophthoromycotina subtilisin-like serine proteases. The columns DTG, GHGTH, and SGTS represents the closest amino acid sequence for each of the amino acids from the DHS catalytic triad. A. Amino acid alignment of the active site residues for the three identified groups (A-C) of SLSPs within Entomophthoromycotina. Accession codes are color coded as: Orange &#x2013; &lt;em&gt;C. incongruus&lt;/em&gt;, Blue &#x2013; &lt;em&gt;E. muscae&lt;/em&gt;, and Purple &#x2013; &lt;em&gt;P. formicae&lt;/em&gt;. B. Sequence motifs of the active site residues for each group.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3311/F2.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3311/F2.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491157" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-16" class="first-child">Active site and domain co-occurrence variability of the three Tribe-MCL clusters identified among 152 Entomophthoromycotina subtilisin-like serine proteases. The columns DTG, GHGTH, and SGTS represents the closest amino acid sequence for each of the amino acids from the DHS catalytic triad. A. Amino acid alignment of the active site residues for the three identified groups (A-C) of SLSPs within Entomophthoromycotina. Accession codes are color coded as: Orange – <em>C. incongruus</em>, Blue – <em>E. muscae</em>, and Purple – <em>P. formicae</em>. B. Sequence motifs of the active site residues for each group.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The MEROPS Tribe-MCL analysis identified a third group (C) of 11 Entomophthoromycotina SLSPs, which only contained members from the order Entomophthorales and clustered with 402 MEROPS proteins (Supplementary data). Of these, 386 were classified as un-assigned subfamily S8A peptidases (S08.UPA) and the remaining 15 assigned to the bacillopeptidase F holotype (S08.017; Supplementary data). All 402 MEROPS proteases in this group originated from either Bacteria or Oomycetes, except for two proteases from the Fungi <em>Rozella allomycis</em> (Cryptomycota) and <em>Mitosporidium daphnia</em> (Microsporidia), respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a>, Supplementary data). The group C Entomophthorales SLSPs clustered as sister to these two Cryptomycota and Microsporidia proteins with strong support (ML bootstrap value = 99) in the phylogenetic analysis of the protein sequences (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>), in concordance with this group of SLSPs being an outlier from all other previously known fungal S8A SLSPs. The monophyly of group C Entomophthorales SLSPs together with Cryptomycota and Microsporidia proteins were further supported in the phylogenetic analysis including all three Entomophthoromycotina groups (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) and when compared to invertebrate-associated ascomycete SLSPs (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mid-point rooted maximum likelihood phylogeny calculated with RAxML and based on a (479 amino acid) alignment of 413 protein subtilisin-like serine protease sequences, which belonged to group C in the Tribe-MCL analysis (see text for details). Bootstrap values &gt;70 from 1000 iterations are shown for non-terminal deeper nodes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846824602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mid-point rooted maximum likelihood phylogeny calculated with RAxML and based on a (479 amino acid) alignment of 413 protein subtilisin-like serine protease sequences, which belonged to group C in the Tribe-MCL analysis (see text for details). Bootstrap values &gt;70 from 1000 iterations are shown for non-terminal deeper nodes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3311/F3.medium.gif" width="440" height="423"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3311/F3.medium.gif" width="440" height="423"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491092" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-18" class="first-child">Mid-point rooted maximum likelihood phylogeny calculated with RAxML and based on a (479 amino acid) alignment of 413 protein subtilisin-like serine protease sequences, which belonged to group C in the Tribe-MCL analysis (see text for details). Bootstrap values &gt;70 from 1000 iterations are shown for non-terminal deeper nodes.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic phylogeny and classification of the early-diverging fungi and related taxonomic groups principally based on Spatafora et al. (2016). Branch lengths are not proportional to genetic distances." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846824602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic phylogeny and classification of the early-diverging fungi and related taxonomic groups principally based on Spatafora &lt;em&gt;et al.&lt;/em&gt; (2016). Branch lengths are not proportional to genetic distances.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3311/F4.medium.gif" width="440" height="380"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3311/F4.medium.gif" width="440" height="380"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490994" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-19" class="first-child">Schematic phylogeny and classification of the early-diverging fungi and related taxonomic groups principally based on <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Spatafora <em>et al.</em> (2016)</a>. Branch lengths are not proportional to genetic distances.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagram showing taxonomic distribution of subtilisin-like serine protease clusters of major insect and nematode-pathogenic fungal genera. Black circles correspond to clusters with number of S8A proteases for each cluster, and the placement within the Venn diagram correspond to the taxonomic groups contributing sequences to a specific cluster. The asterisk (*) marks the 11 members of the new bacillo-peptidase like cluster C described within the order Entomophthorales (subphylum: Entomophthoromycotina). Entomophthoromycotina encompasses SLSP&#x2019;s found in the genera: Basidiobolus, Conidiobolus, Entomophthora, and Pandora. Hypocrealean entities consist of SLSP&#x2019;s in the MEROPS database from the genera: Cordyceps, Metarhizium, Ophiocordyceps, and Nematode trapping fungi are MEROPS SLSP&#x2019;s found in the genera: Arthrobotrys and Monacrosporium." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846824602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Venn diagram showing taxonomic distribution of subtilisin-like serine protease clusters of major insect and nematode-pathogenic fungal genera. Black circles correspond to clusters with number of S8A proteases for each cluster, and the placement within the Venn diagram correspond to the taxonomic groups contributing sequences to a specific cluster. The asterisk (*) marks the 11 members of the new bacillo-peptidase like cluster C described within the order Entomophthorales (subphylum: Entomophthoromycotina). Entomophthoromycotina encompasses SLSP&#x2019;s found in the genera: &lt;em&gt;Basidiobolus&lt;/em&gt;, &lt;em&gt;Conidiobolus&lt;/em&gt;, &lt;em&gt;Entomophthora&lt;/em&gt;, and &lt;em&gt;Pandora&lt;/em&gt;. Hypocrealean entities consist of SLSP&#x2019;s in the MEROPS database from the genera: &lt;em&gt;Cordyceps&lt;/em&gt;, &lt;em&gt;Metarhizium&lt;/em&gt;, &lt;em&gt;Ophiocordyceps&lt;/em&gt;, and Nematode trapping fungi are MEROPS SLSP&#x2019;s found in the genera: &lt;em&gt;Arthrobotrys&lt;/em&gt; and &lt;em&gt;Monacrosporium&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3311/F5.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/10/3311/F5.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491107" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-20" class="first-child">Venn diagram showing taxonomic distribution of subtilisin-like serine protease clusters of major insect and nematode-pathogenic fungal genera. Black circles correspond to clusters with number of S8A proteases for each cluster, and the placement within the Venn diagram correspond to the taxonomic groups contributing sequences to a specific cluster. The asterisk (*) marks the 11 members of the new bacillo-peptidase like cluster C described within the order Entomophthorales (subphylum: Entomophthoromycotina). Entomophthoromycotina encompasses SLSP’s found in the genera: <em>Basidiobolus</em>, <em>Conidiobolus</em>, <em>Entomophthora</em>, and <em>Pandora</em>. Hypocrealean entities consist of SLSP’s in the MEROPS database from the genera: <em>Cordyceps</em>, <em>Metarhizium</em>, <em>Ophiocordyceps</em>, and Nematode trapping fungi are MEROPS SLSP’s found in the genera: <em>Arthrobotrys</em> and <em>Monacrosporium</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-7"><h2 class="">Discussion</h2><p id="p-21">Subtilisin-like serine proteases (SLSPs) have many roles in fungal biology and are known to be involved in host–pathogen interactions. Independent expansion of copy number and diversification of SLSPs is widespread among animal pathogenic Dikarya (Ascomycota and Basidiomycota) (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Li <em>et al.</em>, 2017</a>). The repeated expansion of SLSPs among the generalist insect-pathogenic hypocrealean fungi has been interpreted as an adaptation to enable infection of insect hosts (<a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>), whereas comparatively little is known about the evolution and diversification of SLSPs among the early diverging fungal clades. To understand the evolution of SLSPs among the vertebrate and arthropod pathogenic fungi in the subphylum Entomophthoromycotina, we searched available genomic and transcriptomic sequence data to identify all Entomophthoromycotina genes with SLSP domains. We found 154 Entomophthoromycotina SLSPs, of which two copies were classified as S8B kexin SLSPs. The remaining 152 S8A SLSPs were clustered by sequence similarity and compared by phylogenetic analysis to show that the majority of the SLSPs (n = 130) are similar to and cluster together with “classical” proteinase-K-like fungal S8A SLSPs (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). A statistical test for a significant expansion of SLSP copy number among the insect-pathogenic Entomophthoromycotina was not explicitly performed in the present analysis due to uncertainty of total gene numbers from transcriptomic data sets of the specialist insect-pathogens <em>E. muscae</em> and <em>P. formicae</em>. In the sampled transcriptomes, the number of transcripts is likely larger than the genome gene count due to splice variants, post-transcriptional modifications, and allelic variants assembling into multiple transcripts per gene. In addition, the assembled transcripts only reflect actively transcribed genes expressed in the sampled conditions and time points, and may underestimate the actual number of genes. These confounding factors impact the estimated number of genes and make quantitative comparative analyses of gene family size between transcriptomes unreliable.</p><p id="p-22">We did identify 11 SLSPs that cluster together with 402 un-annotated or Bacillopeptidase F-like SLSPs primarily from bacteria and Oomycetes (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), but also including two fungal protease sequences from the early-diverging Cryptomycota <em>R. allomycis</em> and microsporidium <em>M. daphnia</em> lineages (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). These observations remained consistent even when exploring variation in the inflation parameter, which controls the “tightness” in the cluster analysis. The entomophthoralean and oomycetous S8A SLSPs form separate clades within this 402-sequence cluster of primarily bacterial proteases. Instead, the 11 Entomophthorales SLSPs form a distinct lineage together with the two proteases from <em>R. allomycis</em> and <em>M. daphnia</em> (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">6</a>). This indicates that within this group, the Entomophthorales and Oomycete SLSPs both share a most recent common ancestor with independent bacterial proteases and the 11 entomophthoralean proteases, together with the two SLSPs from Cryptomycota and Microsporidia, are a unique group of proteases exclusive to some of the early diverging fungal lineages.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum likelihood phylogeny calculated with RAxML and based on a (276 amino acid) alignment of 500 protein subtilisin-like serine protease sequences. SLPSs within Entomophthoromyctina from group B and C, and 10 representative SLSPs from group A are included together with the 10 most similar SLSPs in the MEROPS database for each group A, B, and C, respectively. To show the relationship between the three Entomophthoromycotina SLSP groups, representative SLSPs from insect-pathogenic and nematode-trapping fungi were included (see text for details). Bootstrap values &gt;70 from 100 iterations are shown for non-terminal nodes. The tree is rooted with the S8B subfamily type Kexin from S. cerevisiae (UniProt: P13134)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1846824602" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Maximum likelihood phylogeny calculated with RAxML and based on a (276 amino acid) alignment of 500 protein subtilisin-like serine protease sequences. SLPSs within Entomophthoromyctina from group B and C, and 10 representative SLSPs from group A are included together with the 10 most similar SLSPs in the MEROPS database for each group A, B, and C, respectively. To show the relationship between the three Entomophthoromycotina SLSP groups, representative SLSPs from insect-pathogenic and nematode-trapping fungi were included (see text for details). Bootstrap values &gt;70 from 100 iterations are shown for non-terminal nodes. The tree is rooted with the S8B subfamily type Kexin from &lt;em&gt;S. cerevisiae&lt;/em&gt; (UniProt: P13134)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3311/F6.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/10/3311/F6.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3311/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491069" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-23" class="first-child">Maximum likelihood phylogeny calculated with RAxML and based on a (276 amino acid) alignment of 500 protein subtilisin-like serine protease sequences. SLPSs within Entomophthoromyctina from group B and C, and 10 representative SLSPs from group A are included together with the 10 most similar SLSPs in the MEROPS database for each group A, B, and C, respectively. To show the relationship between the three Entomophthoromycotina SLSP groups, representative SLSPs from insect-pathogenic and nematode-trapping fungi were included (see text for details). Bootstrap values &gt;70 from 100 iterations are shown for non-terminal nodes. The tree is rooted with the S8B subfamily type Kexin from <em>S. cerevisiae</em> (UniProt: P13134)</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Functional annotation indicates apparent protease activity based on sequence similarity, but molecular function of the novel 11 SLSPs in group C is unknown. Eight of these SLSPs possess a signal peptide that suggest external secretion and thus indicative of a function on the immediate environment of the fungi, whereas the remaining three SLSPs might not be secreted or represent incomplete sequence models. Apart from the canonical protease S8 domain (PF00082), no other Pfam domains were found among this group C SLSPs. Searches against InterPro databases similarly did not reveal any other protein domains apart from the protease S8 SLSP domain (PRINTS: subtilisin serine protease family (S8) signature (PR00723), InterPro: peptidase S8, subtilisin-related (IPR015500), and ProSitePatterns: serine proteases, subtilase family (PS00138)). Notably, none of these SLSPs contain the proteinase inhibitor i9 domain (PF05922) often found among the classical protease K-like SLSPs (<a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">Muszewska <em>et al.</em>, 2011</a>). Extensive diversification of the amino acids immediately surrounding the active site residues in the DHS triad (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), further suggests that the new group C of SLSPs have evolved a different function than the “classical” fungal SLSPs in group A. Out of the six Entomophthoromycotina species analyzed here, only three: <em>C. incongruus</em>, <em>E. muscae</em> and <em>P. formicae</em> within the order Entomophthorales contained members in the new group C SLSPs (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). The unequal phylogenetic presence of the group C SLSPs could be indicative of specific functions related to niche adaptation. The two insect-pathogenic fungi specialized on house flies (<em>E. muscae</em>) and wood ants (<em>P. formicae</em>) contain five and two of the novel group C SLSPs, respectively (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). However, the soil saprobe and opportunistic human pathogen <em>C. incongruus</em> also contains four group C SLSPs, which provide evidence that group C SLSPs are unrelated to host-specific evolution of the specialist insect-pathogenic entomophthoralean fungi.</p><p id="p-25">The novel group C SLSPs were absent from the sequenced genome of <em>C. thromboides</em>, implying that the uneven taxonomic presence and absence of particular SLSPs within Entomophthoromycotina taxa are unlikely to be due to sequencing or sampling artifacts of data. This is further supported by the absence of group C SLSP S8A-proteases in Basidiomycota or Ascomycota (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). However, the genomes of ascomycete insect-pathogenic hypocrealean and nematode-trapping fungi within Ascomycota contain different SLSP’s missing in Entomphthoromycotina (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The present analysis shows that the two major groups of insect-pathogenic fungi within Ascomycota and Entomophthoromycotina contain a similar complement of SLSPs, but each clade also possesses unique sets of these proteases that most likely evolved independently (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>). Further studies including genomic comparisons of the host-specific insect-pathogenic fungi within Entomophthoromycotina will likely shed interesting new light on the gene content of these early diverging fungi (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">De Fine Licht <em>et al.</em>, 2016</a>). The presence of unusual genome organization, polyploidy and large genomes in many host-specific insect-pathogenic species within Entomophthoromycotina has previously been a hindrance to genome sequencing (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gryganskyi and Muszewska 2014</a>). However, this study highlights examples of many new proteins and enzymes that may be discovered through genome sequencing and data mining within Entomophthoromycotina.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-26">The work conducted by the U.S. Department of Energy Joint Genome Institute is supported by the Office of Science of the U.S. Department of Energy under Contract No. DE-AC02-05CH11231. Work by JES and AG was partially supported by funding from the National Science Foundation (DEB-1441715) to JES. KV expresses her gratitude to generous financial support from CRC/TR 124 Pathogenic fungi and their human host: Networks of Interaction - FungiNet. (Z1 and A6) granted by the Deutsche Forschungsgemeinschaft. HHDFL was supported by the Villum Foundation (grant no. 10122).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-27">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6949037">https://doi.org/10.25387/g3.6949037</a>.</p></li><li class="fn" id="fn-2"><p id="p-28"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-01"><span class="received-label">Received </span>May 1, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-08-09"><span class="accepted-label">Accepted </span>August 9, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Arnesen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.1" data-doi="10.1016/j.gene.2003.09.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bagga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Screen</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St. Leger</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Reconstructing the diversification of subtilisins in the pathogenic fungus <em>Metarhizium anisopliae</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">169</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2003.09.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DBagga%26rft.auinit1%253DS.%26rft.volume%253D324%26rft.spage%253D159%26rft.epage%253D169%26rft.atitle%253DReconstructing%2Bthe%2Bdiversification%2Bof%2Bsubtilisins%2Bin%2Bthe%2Bpathogenic%2Bfungus%2BMetarhizium%2Banisopliae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2003.09.031%26rft_id%253Dinfo%253Apmid%252F14693381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2003.09.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14693381&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188291100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.2" data-doi="10.1146/annurev-ento-011613-162054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boomsma</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyling</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eilenberg</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary interaction networks of insect pathogenic fungi.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Entomol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-ento-011613-162054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEntomol.%26rft.volume%253D59%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ento-011613-162054%26rft_id%253Dinfo%253Apmid%252F24160418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ento-011613-162054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24160418&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.3" data-doi="10.1186/gb-2011-12-1-r7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burmester</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelest</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glöckner</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heddergott</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schindler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative and functional genomics provide insights into the pathogenicity of dermatophytic fungi.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011-12-1-r7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBurmester%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253DR7%26rft.epage%253DR7%26rft.atitle%253DComparative%2Band%2Bfunctional%2Bgenomics%2Bprovide%2Binsights%2Binto%2Bthe%2Bpathogenicity%2Bof%2Bdermatophytic%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-1-r7%26rft_id%253Dinfo%253Apmid%252F21247460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-1-r7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21247460&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.4" data-doi="10.1093/gbe/evv090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekimoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aerts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Phylogenomic analyses indicate that early fungi evolved digesting cell walls of algal ancestors of land plants.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1590</span>–<span class="cit-lpage">1601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv090%26rft_id%253Dinfo%253Apmid%252F25977457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25977457&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.5" data-doi="10.1016/S0065-2296(05)40006-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charnley</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Fungal pathogens of insects: Cuticle degrading enzymes and toxins.</span> <abbr class="cit-jnl-abbrev">Adv. Bot. Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">321</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0065-2296(05)40006-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BBot.%2BRes.%26rft.volume%253D40%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2296%252805%252940006-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2296(05)40006-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.6" data-doi="10.1038/ncomms12218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chibucos</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soliman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebremariam</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daugherty</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An integrated genomic and transcriptomic survey of mucormycosis-causing fungi.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">12218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms12218</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D12218%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms12218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms12218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.7" data-doi="10.1016/bs.adgen.2016.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Fine Licht</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hajek</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eilenberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Utilizing genomics to study entomopathogenicity in the fungal phylum entomophthoromycota: A review of current genetic resources.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/bs.adgen.2016.01.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BGenet.%26rft.volume%253D94%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.adgen.2016.01.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.adgen.2016.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.8" data-doi="10.1111/mec.13863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Fine Licht</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eilenberg</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative transcriptomics reveal host-specific nucleotide variation in entomophthoralean fungi.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2092</span>–<span class="cit-lpage">2110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13863</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D2092%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13863&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.9" data-doi="10.1371/journal.pgen.1002345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desjardins</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Champion</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holder</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muszewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative genomic analysis of human fungal pathogens causing paracoccidioidomycosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002345</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDesjardins%26rft.auinit1%253DC.%2BA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002345%26rft.epage%253De1002345%26rft.atitle%253DComparative%2Bgenomic%2Banalysis%2Bof%2Bhuman%2Bfungal%2Bpathogens%2Bcausing%2Bparacoccidioidomycosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002345%26rft_id%253Dinfo%253Apmid%252F22046142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002345&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22046142&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.10" data-doi="10.1093/bioinformatics/14.9.755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Profile hidden Markov models.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">763</span> <span class="cit-comment">doi:btb114 [pii]</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/14.9.755</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D755%26rft.epage%253D763%26rft.atitle%253DProfile%2Bhidden%2BMarkov%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.9.755%26rft_id%253Dinfo%253Apmid%252F9918945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/14.9.755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9918945&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077489900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.11" data-doi="10.1093/nar/30.7.1575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dongen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouzounis</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An efficient algorithm for large-scale detection of protein families.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1575</span>–<span class="cit-lpage">1584</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/30.7.1575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEnright%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D1575%26rft.epage%253D1584%26rft.atitle%253DAn%2Befficient%2Balgorithm%2Bfor%2Blarge-scale%2Bdetection%2Bof%2Bprotein%2Bfamilies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.7.1575%26rft_id%253Dinfo%253Apmid%252F11917018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.7.1575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11917018&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174654200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.12" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggill</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberhardt</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eddy</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Pfam protein families database: towards a more sustainable future.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D279</span>–<span class="cit-lpage">D285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F26673716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26673716&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.13" data-doi="10.4172/2165&amp;ndash;8056.1000e116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gryganskyi</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muszewska</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole Genome Sequencing and the Zygomycota.</span> <abbr class="cit-jnl-abbrev">Fungal Genom. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4172/2165–8056.1000e116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenom.%2BBiol.%26rft.volume%253D4%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.4172%252F2165%2526ndash%253B8056.1000e116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4172/2165&amp;ndash;8056.1000e116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.14" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">704</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10635150390235520</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150390235520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14530136&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185732500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.15" data-doi="10.1073/pnas.1412662111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trajectory and genomic determinants of fungal-pathogen speciation and host adaptation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">16796</span>–<span class="cit-lpage">16801</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1412662111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1412662111%26rft_id%253Dinfo%253Apmid%252F25368161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQ3LzE2Nzk2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.16" data-doi="10.1128/AEM.66.1.363-368.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jalving</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van De Vondervoort</span>  <span class="cit-name-given-names">P. J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaap</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Characterization of the kexin-like maturase of <em>Aspergillus niger</em>.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.66.1.363-368.2000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DJalving%26rft.auinit1%253DR.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D363%26rft.epage%253D368%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BKexin-Like%2BMaturase%2Bof%2BAspergillus%2Bniger%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.66.1.363-368.2000%26rft_id%253Dinfo%253Apmid%252F10618249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjg6IjY2LzEvMzYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.17" data-doi="10.1038/nmeth.4285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalyaanamoorthy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minh</span>  <span class="cit-name-given-names">B. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">T. K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jermiin</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">ModelFinder: fast model selection for accurate phylogenetic estimates.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4285%26rft_id%253Dinfo%253Apmid%252F28481363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28481363&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.18" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.19" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackshields</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenna</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGettigan</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Clustal W and Clustal X version 2.0.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm404</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846036&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251197700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.20" data-doi="10.1038/srep41926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Phylogenomic evolutionary surveys of subtilase superfamily genes in fungi.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep45456</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep41926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep41926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.21" data-doi="10.1016/j.jip.2015.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Małagocka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grell</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eilenberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptome of an entomophthoralean fungus (<em>Pandora formicae</em>) shows molecular machinery adjusted for successful host exploitation and transmission.</span> <abbr class="cit-jnl-abbrev">J. Invertebr. Pathol.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jip.2015.05.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInvertebr.%2BPathol.%26rft.volume%253D128%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jip.2015.05.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jip.2015.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.22" data-doi="10.1128/mBio.00259-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gräser</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative genome analysis of <em>Trichophyton rubrum</em> and related dermatophytes reveals candidate genes involved in infection.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e00259–12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.00259-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D3%26rft.spage%253De0025912%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00259-12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00259-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.10.3311.23" data-doi="10.1371/journal.pgen.1003909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meerupati</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">K.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tunlid</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic mechanisms accounting for the adaptation to parasitism in nematode-trapping fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003909</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003909%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003909%26rft_id%253Dinfo%253Apmid%252F24244185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003909&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24244185&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.24" data-doi="10.1093/bioinformatics/btn575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husmeier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGuire</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TOPALi v2: a rich graphical interface for evolutionary analyses of multiple alignments on HPC clusters and multi-core desktops.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btn575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn575%26rft_id%253Dinfo%253Apmid%252F18984599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18984599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261996400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.25" data-doi="10.1038/ng.3859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mondo</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dannebaum</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louie</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bewick</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Widespread adenine N6-methylation of active genes in fungi.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">964</span>–<span class="cit-lpage">968</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3859</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D964%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.26" data-doi="10.1038/s41598-017-09644-w"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muszewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stepniewska-Dziubinska</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steczkiewicz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawlowska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dziedzic</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fungal lifestyle reflected in serine protease repertoire.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">9147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-09644-w</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D9147%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-09644-w%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-09644-w&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.27" data-doi="10.1093/molbev/msr176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muszewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szczesny</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grynberg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Independent subtilases expansions in fungi associated with animals.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3395</span>–<span class="cit-lpage">3404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr176%26rft_id%253Dinfo%253Apmid%252F21727238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21727238&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297227600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.28" data-doi="10.1186/1471-2105-11-120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nepusz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasidharan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paccanaro</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SCPS: a fast implementation of a spectral method for detecting protein families on a genome-wide scale.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-11-120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DNepusz%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D120%26rft.epage%253D120%26rft.atitle%253DSCPS%253A%2Ba%2Bfast%2Bimplementation%2Bof%2Ba%2Bspectral%2Bmethod%2Bfor%2Bdetecting%2Bprotein%2Bfamilies%2Bon%2Ba%2Bgenome-wide%2Bscale.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-120%26rft_id%253Dinfo%253Apmid%252F20214776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20214776&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.29" data-doi="10.1093/nar/gkq443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Penn</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Privman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashkenazy</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graur</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">GUIDANCE: A web server for assessing alignment confidence scores.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W23</span>–<span class="cit-lpage">W28</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq443%26rft_id%253Dinfo%253Apmid%252F20497997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20497997&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284148900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.30" data-doi="10.1038/nmeth.1701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1701</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPetersen%26rft.auinit1%253DT.%2BN.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D785%26rft.epage%253D786%26rft.atitle%253DSignalP%2B4.0%253A%2Bdiscriminating%2Bsignal%2Bpeptides%2Bfrom%2Btransmembrane%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1701%26rft_id%253Dinfo%253Apmid%252F21959131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959131&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.31" data-doi="10.1093/nar/gkv1118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rawlings</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Twenty years of the MEROPS database of proteolytic enzymes, their substrates and inhibitors.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D343</span>–<span class="cit-lpage">D350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1118%26rft_id%253Dinfo%253Apmid%252F26527717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26527717&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.32" data-doi="10.1101/gr.087551.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharpton</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rounsley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative genomic analyses of the human fungal pathogens <em>Coccidioides</em> and their relatives.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1722</span>–<span class="cit-lpage">1731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.087551.108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.087551.108%26rft_id%253Dinfo%253Apmid%252F19717792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xNzIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.33" data-doi="10.3852/16-042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spatafora</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benny</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazarus</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A phylum-level phylogenetic classification of zygomycete fungi based on genome-scale data.</span> <abbr class="cit-jnl-abbrev">Mycologia</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1028</span>–<span class="cit-lpage">1046</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3852/16-042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycologia%26rft_id%253Dinfo%253Adoi%252F10.3852%252F16-042%26rft_id%253Dinfo%253Apmid%252F27738200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibXljb2xvZ2lhIjtzOjU6InJlc2lkIjtzOjEwOiIxMDgvNS8xMDI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.34" data-doi="10.1016/0022-2011(86)90146-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St. Leger</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charnley</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span><span class="cit-comment">a</span> <span class="cit-article-title">Cuticle-degrading enzymes of entomopathogenic fungi: Synthesis in culture on cuticle.</span> <abbr class="cit-jnl-abbrev">J. Invertebr. Pathol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2011(86)90146-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInvertebr.%2BPathol.%26rft.volume%253D48%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2011%252886%252990146-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2011(86)90146-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.35" data-doi="10.1016/0022-2011(86)90043-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St. Leger</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charnley</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span><span class="cit-comment">b</span> <span class="cit-article-title">Cuticle-degrading enzymes of entomopathogenic fungi: Cuticle degradation in vitro by enzymes from entomopathogens.</span> <abbr class="cit-jnl-abbrev">J. Invertebr. Pathol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0022-2011(86)90043-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInvertebr.%2BPathol.%26rft.volume%253D47%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2011%252886%252990043-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2011(86)90043-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.36" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3311.37" data-doi="10.1016/B978-0-12-384984-7.00006-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">F. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyling</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luangsa-Ard</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackwell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fungal entomopathogens</span>, pp. <span class="cit-fpage">171</span>–<span class="cit-lpage">220</span> in <span class="cit-source">Insect Pathology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">F. E.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kaya</span>  <span class="cit-name-given-names">H.</span></span>. <span class="cit-publ-name">Elsevier Inc.</span>, <span class="cit-publ-loc">New York City, NY,</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/B978-0-12-384984-7.00006-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2BPathology%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-384984-7.00006-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-384984-7.00006-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.38" data-doi="10.1534/g3.115.022806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whiston</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative phylogenomics of pathogenic and nonpathogenic species.</span> <abbr class="cit-jnl-abbrev">G3 Genes|Genomes|Genetics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">244</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.022806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%257CGenomes%257CGenetics%26rft.volume%253D6%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.022806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.115.022806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.39" data-doi="10.1038/srep00483"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic perspectives on the evolution of fungal entomopathogenicity in <em>Beauveria bassiana</em>.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">483</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep00483</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DSci%2BRep%26rft.aulast%253DXiao%26rft.auinit1%253DG.%26rft.volume%253D2%26rft.spage%253D483%26rft.epage%253D483%26rft.atitle%253DGenomic%2Bperspectives%2Bon%2Bthe%2Bevolution%2Bof%2Bfungal%2Bentomopathogenicity%2Bin%2BBeauveria%2Bbassiana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep00483%26rft_id%253Dinfo%253Apmid%252F22761991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep00483&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22761991&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3311.40" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: Phylogenetic analysis by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3311.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3311.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3293" title="Functional Interactions Between rsks-1/S6K, glp-1/Notch, and Regulators of Caenorhabditis elegans Fertility and Germline Stem Cell Maintenance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3321" title="Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3311" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rElgLd5_BnhLpTnMOZtj2CWEWn7EYMnNB5sKkTxMpMw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3311" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3311" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3311" class="active" data-icon-position="" data-hide-link-title="0">Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490978" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CyKyCoOYBK11i9d99P1Znpw8SQRYkJ84gH6k_6dh3JE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3311.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3311" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-4s_nZj9-40kN_IETmLzely-kITeYzKa0n6AVSSGLEWE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3311" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3311.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490978" id="node490978--2" data-pisa="ggg;8/10/3311" data-pisa-master="ggg;g3.118.200656" data-apath="/ggg/8/10/3311.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jonathan A. Arnesen</span>, <span class="highwire-citation-author" data-delta="1">Joanna Małagocka</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6037-0092" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrii Gryganskyi</span>, <span class="highwire-citation-author" data-delta="3">Igor V. Grigoriev</span>, <span class="highwire-citation-author" data-delta="4">Kerstin Voigt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jason E. Stajich</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3326-5729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Henrik H. De Fine Licht</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3311-3319;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200656 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490978--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan A. Arnesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BA.%2BArnesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arnesen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BA.%2BArnesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joanna Małagocka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BMa%C5%82agocka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%C5%82agocka%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoanna%2BMa%25C5%2582agocka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrii Gryganskyi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Duke University, Durham, North Carolina, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrii%2BGryganskyi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gryganskyi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrii%2BGryganskyi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6037-0092" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrii Gryganskyi</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Igor V. Grigoriev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>US Department of Energy Joint Genome Institute, Walnut Creek, California, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BV.%2BGrigoriev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grigoriev%20IV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BV.%2BGrigoriev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kerstin Voigt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jena Microbial Resource Collection (JMRC), Leibniz Institute for Natural Product Research and Infection Biology - Hans Knoell Institute, Adolf-Reichwein-Str.23, 07745 Jena, Germany</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Microbiology, Friedrich Schiller University, Neugasse 25, 07743 Jena, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerstin%2BVoigt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voigt%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerstin%2BVoigt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jason E. Stajich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Pathology and Microbiology, University of California, Riverside, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BStajich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stajich%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BE.%2BStajich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jason E. Stajich</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henrik H. De Fine Licht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BH.%2BDe%2BFine%2BLicht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Fine%20Licht%20HH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenrik%2BH.%2BDe%2BFine%2BLicht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3326-5729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Henrik H. De Fine Licht</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hhdefinelicht@plen.ku.dk" class="" data-icon-position="" data-hide-link-title="0">hhdefinelicht@plen.ku.dk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3311" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3311&amp;title=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3311&amp;title=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3311&amp;title=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3311&amp;text=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3311&amp;title=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3311&amp;t=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3311&amp;title=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3311&amp;title=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490978/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490978" id="node490978--3" data-pisa="ggg;8/10/3311" data-pisa-master="ggg;g3.118.200656" data-apath="/ggg/8/10/3311.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jonathan A. Arnesen</span>, <span class="highwire-citation-author" data-delta="1">Joanna Małagocka</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6037-0092" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andrii Gryganskyi</span>, <span class="highwire-citation-author" data-delta="3">Igor V. Grigoriev</span>, <span class="highwire-citation-author" data-delta="4">Kerstin Voigt</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jason E. Stajich</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3326-5729" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Henrik H. De Fine Licht</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3311-3319;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200656 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490978--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jonathan A. Arnesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BA.%2BArnesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arnesen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BA.%2BArnesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joanna Małagocka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BMa%C5%82agocka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%C5%82agocka%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoanna%2BMa%25C5%2582agocka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrii Gryganskyi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Duke University, Durham, North Carolina, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrii%2BGryganskyi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gryganskyi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrii%2BGryganskyi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6037-0092" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrii Gryganskyi</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Igor V. Grigoriev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>US Department of Energy Joint Genome Institute, Walnut Creek, California, USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BV.%2BGrigoriev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grigoriev%20IV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BV.%2BGrigoriev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kerstin Voigt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jena Microbial Resource Collection (JMRC), Leibniz Institute for Natural Product Research and Infection Biology - Hans Knoell Institute, Adolf-Reichwein-Str.23, 07745 Jena, Germany</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Microbiology, Friedrich Schiller University, Neugasse 25, 07743 Jena, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kerstin%2BVoigt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Voigt%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKerstin%2BVoigt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jason E. Stajich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Pathology and Microbiology, University of California, Riverside, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BStajich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stajich%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BE.%2BStajich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7591-0020" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jason E. Stajich</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Henrik H. De Fine Licht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Section for Organismal Biology, Department of Plant and Environmental Sciences, University of Copenhagen, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Henrik%2BH.%2BDe%2BFine%2BLicht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Fine%20Licht%20HH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHenrik%2BH.%2BDe%2BFine%2BLicht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3326-5729" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Henrik H. De Fine Licht</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:hhdefinelicht@plen.ku.dk" class="" data-icon-position="" data-hide-link-title="0">hhdefinelicht@plen.ku.dk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3311&amp;rft.epage=3319&amp;rft.atitle=Early%20Diverging%20Insect-Pathogenic%20Fungi%20of%20the%20Order%20Entomophthorales%20Possess%20Diverse%20and%20Unique%20Subtilisin-Like%20Serine%20Proteases&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Arnesen%2C+Jonathan+A.&amp;rft.au=Ma%C5%82agocka%2C+Joanna&amp;rft.au=Gryganskyi%2C+Andrii&amp;rft.au=Grigoriev%2C+Igor+V.&amp;rft.au=Voigt%2C+Kerstin&amp;rft.au=Stajich%2C+Jason+E.&amp;rft.au=De+Fine+Licht%2C+Henrik+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490978/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490978/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490978/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490978/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490978/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490978/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490978/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490978/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490978/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490978/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490978/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490978/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3311" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3311.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3311.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/10/3311.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/10/3311.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
