<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3131" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3131.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3131.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3131.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3131.atom" />
<meta name="HW.pisa" content="ggg;8/10/3131" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.118.200160" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2018 Miller et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Drosophila genus is a unique group containing a wide range of species that occupy diverse ecosystems. In addition to the most widely studied species, Drosophila melanogaster , many other members in this genus also possess a well-developed set of genetic tools. Indeed, high-quality genomes exist for several species within the genus, facilitating studies of the function and evolution of cis -regulatory regions and proteins by allowing comparisons across at least 50 million years of evolution. Yet, the available genomes still fail to capture much of the substantial genetic diversity within the Drosophila genus. We have therefore tested protocols to rapidly and inexpensively sequence and assemble the genome from any Drosophila species using single-molecule sequencing technology from Oxford Nanopore. Here, we use this technology to present highly contiguous genome assemblies of 15 Drosophila species: 10 of the 12 originally sequenced Drosophila species ( ananassae , erecta , mojavensis , persimilis , pseudoobscura , sechellia , simulans , virilis , willistoni , and yakuba ), four additional species that had previously reported assemblies ( biarmipes , bipectinata , eugracilis , and mauritiana ), and one novel assembly ( triauraria ). Genomes were generated from an average of 29x depth-of-coverage data that after assembly resulted in an average contig N50 of 4.4 Mb. Subsequent alignment of contigs from the published reference genomes demonstrates that our assemblies could be used to close over 60% of the gaps present in the currently published reference genomes. Importantly, the materials and reagents cost for each genome was approximately $1,000 (USD). This study demonstrates the power and cost-effectiveness of long-read sequencing for genome assembly in Drosophila and provides a framework for the affordable sequencing and assembly of additional Drosophila genomes." />
<meta name="DC.Contributor" content="Danny E. Miller" />
<meta name="DC.Contributor" content="Cynthia Staber" />
<meta name="DC.Contributor" content="Julia Zeitlinger" />
<meta name="DC.Contributor" content="R. Scott Hawley" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Drosophila&lt;/i&gt; genus is a unique group containing a wide range of species that occupy diverse ecosystems. In addition to the most widely studied species, &lt;i&gt;Drosophila melanogaster&lt;/i&gt;, many other members in this genus also possess a well-developed set of genetic tools. Indeed, high-quality genomes exist for several species within the genus, facilitating studies of the function and evolution of &lt;i&gt;cis&lt;/i&gt;-regulatory regions and proteins by allowing comparisons across at least 50 million years of evolution. Yet, the available genomes still fail to capture much of the substantial genetic diversity within the &lt;i&gt;Drosophila&lt;/i&gt; genus. We have therefore tested protocols to rapidly and inexpensively sequence and assemble the genome from any Drosophila species using single-molecule sequencing technology from Oxford Nanopore. Here, we use this technology to present highly contiguous genome assemblies of 15 Drosophila species: 10 of the 12 originally sequenced Drosophila species (&lt;i&gt;ananassae&lt;/i&gt;, &lt;i&gt;erecta&lt;/i&gt;, &lt;i&gt;mojavensis&lt;/i&gt;, &lt;i&gt;persimilis&lt;/i&gt;, &lt;i&gt;pseudoobscura&lt;/i&gt;, &lt;i&gt;sechellia&lt;/i&gt;, &lt;i&gt;simulans&lt;/i&gt;, &lt;i&gt;virilis&lt;/i&gt;, &lt;i&gt;willistoni&lt;/i&gt;, and &lt;i&gt;yakuba&lt;/i&gt;), four additional species that had previously reported assemblies (&lt;i&gt;biarmipes&lt;/i&gt;, &lt;i&gt;bipectinata&lt;/i&gt;, &lt;i&gt;eugracilis&lt;/i&gt;, and &lt;i&gt;mauritiana&lt;/i&gt;), and one novel assembly (&lt;i&gt;triauraria&lt;/i&gt;). Genomes were generated from an average of 29x depth-of-coverage data that after assembly resulted in an average contig N50 of 4.4 Mb. Subsequent alignment of contigs from the published reference genomes demonstrates that our assemblies could be used to close over 60% of the gaps present in the currently published reference genomes. Importantly, the materials and reagents cost for each genome was approximately $1,000 (USD). This study demonstrates the power and cost-effectiveness of long-read sequencing for genome assembly in Drosophila and provides a framework for the affordable sequencing and assembly of additional Drosophila genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3131" />
<meta name="citation_id" content="8/10/3131" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3131" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3131.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3131.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3131.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200160" />
<meta name="citation_pmid" content="30087105" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="3131" />
<meta name="citation_lastpage" content="3141" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Danny E. Miller" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author_institution" content="MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS" />
<meta name="citation_author_email" content="danny.miller@seattlechildrens.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6096-8601" />
<meta name="citation_author" content="Cynthia Staber" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author" content="Julia Zeitlinger" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author_institution" content="Department of Pathology, University of Kansas Medical Center, Kansas City, KS 66160" />
<meta name="citation_author" content="R. Scott Hawley" />
<meta name="citation_author_institution" content="Stowers Institute for Medical Research, Kansas City, MO" />
<meta name="citation_author_institution" content="Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6478-0494" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Adams;citation_author=S. E. Celniker;citation_author=R. A. Holt;citation_author=C. A. Evans;citation_author=J. D. Gocayne;citation_title=The Genome Sequence of Drosophila melanogaster;citation_pages=2185-2195;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731132;citation_doi=10.1126/science.287.5461.2185" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. L. Allen;citation_author=E. K. Delaney;citation_author=A. Kopp;citation_author=S. F. Chenoweth;citation_title=Single-Molecule Sequencing of the Drosophila serrata Genome.;citation_volume=7;citation_year=2017;citation_pmid=28143951;citation_doi=10.1534/g3.116.037598" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. SchÃ¤ffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_title=Drosophila: A Laboratory Handbook;citation_author=M. Ashburner;citation_author=K. Golic;citation_author=R. S. Hawley;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. Berlin;citation_author=S. Koren;citation_author=C.-S. Chin;citation_author=J. P. Drake;citation_author=J. M. Landolin;citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.;citation_pages=623-630;citation_volume=33;citation_year=2015;citation_pmid=26006009;citation_doi=10.1038/nbt.3238" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. Bosco;citation_author=P. Campbell;citation_author=J. T. Leiva-Neto;citation_author=T. A. Markow;citation_title=Analysis of Drosophila Species Genome Size and Satellite DNA Content Reveals Significant Differences Among Strains as Well as Between Species;citation_pages=1277-1290;citation_volume=177;citation_year=2007;citation_issue=3;citation_pmid=18039867;citation_doi=10.1534/genetics.107.075069" />
<meta name="citation_reference" content="citation_journal_title=Nature Publishing Group;citation_author=M. J. P. Chaisson;citation_author=R. K. Wilson;citation_author=E. E. Eichler;citation_title=Genetic variation and the de novo assembly of human genomes.;citation_pages=627-640;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. C. Chiu;citation_author=X. Jiang;citation_author=L. Zhao;citation_author=C. A. Hamm;citation_author=J. M. Cridland;citation_title=Genome of Drosophila suzukii, the spotted wing drosophila.;citation_volume=3;citation_year=2013;citation_pmid=24142924;citation_doi=10.1534/g3.113.008185" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. L. Delcher;citation_author=S. Kasif;citation_author=R. D. Fleischmann;citation_author=J. Peterson;citation_author=O. White;citation_title=Alignment of whole genomes.;citation_volume=27;citation_year=1999;citation_pmid=10325427;citation_doi=10.1093/nar/27.11.2369" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. G. Clark;citation_author=M. B. Eisen;citation_author=D. R. Smith;citation_author=C. M. Bergman;citation_title=Evolution of genes and genomes on the Drosophila phylogeny.;citation_pages=203-218;citation_volume=450;citation_year=2007;citation_issue=7167;citation_pmid=17994087;citation_doi=10.1038/nature06341" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. J. Gratz;citation_author=A. M. Cummings;citation_author=J. N. Nguyen;citation_author=D. C. Hamm;citation_author=L. K. Donohue;citation_title=Genome Engineering of Drosophila with the CRISPR RNA-Guided Cas9 Nuclease.;citation_volume=194;citation_year=2013;citation_pmid=23709638;citation_doi=10.1534/genetics.113.152710" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=T. R. Gregory;citation_author=J. S. Johnston;citation_title=Genome size diversity in the family Drosophilidae.;citation_pages=228-238;citation_volume=101;citation_year=2008;citation_issue=3;citation_pmid=18523443;citation_doi=10.1038/hdy.2008.49" />
<meta name="citation_reference" content="citation_title=PLoS One;citation_author=C. E. Hjelmen;citation_author=J. S. Johnston;citation_title=The mode and tempo of genome size evolution in the subgenus Sophophora.;citation_pages=e0173505;citation_volume=12;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. A. Hoskins;citation_author=J. W. Carlson;citation_author=K. H. Wan;citation_author=S. Park;citation_author=I. Mendez;citation_title=The Release 6 reference sequence of the Drosophila melanogaster genome.;citation_volume=25;citation_year=2015;citation_pmid=25589440;citation_doi=10.1101/gr.185579.114" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Jain;citation_author=S. Koren;citation_author=K. H. Miga;citation_author=J. Quick;citation_author=A. C. Rand;citation_title=Nanopore sequencing and assembly of a human genome with ultra-long reads.;citation_pages=338-345;citation_volume=36;citation_year=2018;citation_doi=10.1038/nbt.4060" />
<meta name="citation_reference" content="citation_journal_title=Nature Publishing Group;citation_author=Y. Jiao;citation_author=P. Peluso;citation_author=J. Shi;citation_author=T. Liang;citation_author=M. C. Stitzer;citation_title=Improved maize reference genome with single-molecule technologies.;citation_pages=524-527;citation_volume=546;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Sci. Data;citation_author=K. E. Kim;citation_author=P. Peluso;citation_author=P. Babayan;citation_author=P. J. Yeadon;citation_author=C. Yu;citation_title=Long-read, whole-genome shotgun sequence data for five model organisms.;citation_pages=140045;citation_volume=1;citation_year=2014;citation_pmid=25977796;citation_doi=10.1038/sdata.2014.45" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences.;citation_volume=32;citation_year=2016;citation_pmid=27153593;citation_doi=10.1093/bioinformatics/btw152" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=Minimap2: pairwise alignment for nucleotide sequences.;citation_pages=321;citation_volume=3;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. Llopart;citation_author=S. Elwyn;citation_author=D. Lachaise;citation_author=J. A. Coyne;citation_title=Genetics of a difference in pigmentation between Drosophila yakuba and Drosophila santomea.;citation_pages=2262-2277;citation_volume=56;citation_year=2002;citation_issue=11;citation_pmid=12487356;citation_doi=10.1554/0014-3820(2002)056[2262:GOADIP]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=T. P. Michael;citation_author=F. Jupe;citation_author=F. Bemm;citation_author=S. T. Motley;citation_author=J. P. Sandoval;citation_title=High contiguity Arabidopsis thaliana genome assembly with a single nanopore flow cell.;citation_pages=541;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-03016-2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Mikheenko;citation_author=A. Prjibelski;citation_author=V. Saveliev;citation_author=D. Antipov;citation_author=A. Gurevich;citation_title=Versatile genome assembly evaluation with QUAST-LG.;citation_pages=i142-i150;citation_volume=34;citation_year=2018;citation_doi=10.1093/bioinformatics/bty266" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Nolte;citation_author=R. V. Pandey;citation_author=R. Kofler;citation_author=C. SchlÃ¶tterer;citation_title=Genome-wide patterns of natural variation reveal strong selective sweeps and ongoing genomic conflict in Drosophila mauritiana.;citation_volume=23;citation_year=2013;citation_pmid=23051690;citation_doi=10.1101/gr.139873.112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. J. Obbard;citation_author=J. Maclennan;citation_author=K.-W. Kim;citation_author=A. Rambaut;citation_author=P. M. OâGrady;citation_title=Estimating Divergence Dates and Substitution Rates in the Drosophila Phylogeny.;citation_volume=29;citation_year=2012;citation_pmid=22683811;citation_doi=10.1093/molbev/mss150" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=L. Ometto;citation_author=A. Cestaro;citation_author=S. Ramasamy;citation_author=A. Grassi;citation_author=S. Revadi;citation_title=Linking genomics and ecology to investigate the complex evolution of an invasive Drosophila pest.;citation_volume=5;citation_year=2013;citation_pmid=23501831;citation_doi=10.1093/gbe/evt034" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. M. OâGrady;citation_author=R. DeSalle;citation_title=Phylogeny of the Genus Drosophila.;citation_volume=209;citation_year=2018;citation_pmid=29716983;citation_doi=10.1534/genetics.117.300583" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. A. Perkins;citation_author=L. Holderbaum;citation_author=R. Tao;citation_author=Y. Hu;citation_author=R. Sopko;citation_title=The Transgenic RNAi Project at Harvard Medical School: Resources and Validation.;citation_volume=201;citation_year=2015;citation_pmid=26320097;citation_doi=10.1534/genetics.115.180208" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=A. N. Salazar;citation_author=A. R. Gorter de Vries;citation_author=M. van den Broek;citation_author=M. Wijsman;citation_author=P. de la Torre CortÃ©s;citation_title=Nanopore sequencing enables near-complete de novo assembly of Saccharomyces cerevisiae reference strain CEN.PK113â7D.;citation_volume=17;citation_year=2017;citation_doi=10.1093/femsyr/fox074" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. SimÃ£o;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=J. T. Simpson;citation_author=R. E. Workman;citation_author=P. C. Zuzarte;citation_author=M. David;citation_author=L. J. Dursi;citation_title=Detecting DNA cytosine methylation using nanopore sequencing.;citation_pages=407-410;citation_volume=14;citation_year=2017;citation_doi=10.1038/nmeth.4184" />
<meta name="citation_reference" content="citation_author=E. A. Solares;citation_author=M. Chakraborty;citation_author=D. E. Miller;citation_author=S. Kalsow;citation_author=K. Hall;citation_title=Rapid low-cost assembly of the Drosophila melanogaster reference genome using low-coverage, long-read sequencing.;citation_volume=8;citation_year=2018;citation_pmid=30018084;citation_doi=10.1534/g3.118.200162" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. L. Stern;citation_author=J. Crocker;citation_author=Y. Ding;citation_author=N. Frankel;citation_author=G. Kappes;citation_title=Genetic and Transgenic Reagents for Drosophila simulans, D. mauritiana, D. yakuba, D. santomea, and D. virilis.;citation_volume=7;citation_year=2017;citation_pmid=28280212;citation_doi=10.1534/g3.116.038885" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=S. Subramanian;citation_author=S. Kumar;citation_title=Temporal patterns of fruit fly (Drosophila) evolution revealed by mutation clocks.;citation_volume=21;citation_year=2004;citation_pmid=12949132;citation_doi=10.1093/molbev/msg236" />
<meta name="citation_reference" content="citation_author=G. Thomas;citation_author=M. Hahn;citation_year=2017" />
<meta name="citation_reference" content="citation_author=J. R. Tyson;citation_author=N. J. OâNeil;citation_author=M. Jain;citation_author=H. E. Olsen;citation_author=P. Hieter;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Vaser;citation_author=I. SoviÄ;citation_author=N. Nagarajan;citation_author=M. Å ikiÄ;citation_title=Fast and accurate de novo genome assembly from long uncorrected reads.;citation_volume=27;citation_year=2017;citation_pmid=28100585;citation_doi=10.1101/gr.214270.116" />
<meta name="citation_reference" content="citation_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Ye;citation_author=C. M. Hill;citation_author=S. Wu;citation_author=J. Ruan;citation_author=Z. S. Ma;citation_title=DBG2OLC: Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation Sequencing Technologies.;citation_pages=1-9;citation_volume=6;citation_year=2016;citation_pmid=27503664;citation_doi=10.1038/s41598-016-0001-8" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="The Drosophila genus is a unique group containing a wide range of species that occupy diverse ecosystems. In addition to the most widely studied species, Drosophila melanogaster , many other members in this genus also possess a well-developed set of genetic tools. Indeed, high-quality genomes exist for several species within the genus, facilitating studies of the function and evolution of cis -regulatory regions and proteins by allowing comparisons across at least 50 million years of evolution. Yet, the available genomes still fail to capture much of the substantial genetic diversity within the Drosophila genus. We have therefore tested protocols to rapidly and inexpensively sequence and assemble the genome from any Drosophila species using single-molecule sequencing technology from Oxford Nanopore. Here, we use this technology to present highly contiguous genome assemblies of 15 Drosophila species: 10 of the 12 originally sequenced Drosophila species ( ananassae , erecta , mojavensis , persimilis , pseudoobscura , sechellia , simulans , virilis , willistoni , and yakuba ), four additional species that had previously reported assemblies ( biarmipes , bipectinata , eugracilis , and mauritiana ), and one novel assembly ( triauraria ). Genomes were generated from an average of 29x depth-of-coverage data that after assembly resulted in an average contig N50 of 4.4 Mb. Subsequent alignment of contigs from the published reference genomes demonstrates that our assemblies could be used to close over 60% of the gaps present in the currently published reference genomes. Importantly, the materials and reagents cost for each genome was approximately $1,000 (USD). This study demonstrates the power and cost-effectiveness of long-read sequencing for genome assembly in Drosophila and provides a framework for the affordable sequencing and assembly of additional Drosophila genomes." />
<meta name="og-title" property="og:title" content="Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3131" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Drosophila genus is a unique group containing a wide range of species that occupy diverse ecosystems. In addition to the most widely studied species, Drosophila melanogaster , many other members in this genus also possess a well-developed set of genetic tools. Indeed, high-quality genomes exist for several species within the genus, facilitating studies of the function and evolution of cis -regulatory regions and proteins by allowing comparisons across at least 50 million years of evolution. Yet, the available genomes still fail to capture much of the substantial genetic diversity within the Drosophila genus. We have therefore tested protocols to rapidly and inexpensively sequence and assemble the genome from any Drosophila species using single-molecule sequencing technology from Oxford Nanopore. Here, we use this technology to present highly contiguous genome assemblies of 15 Drosophila species: 10 of the 12 originally sequenced Drosophila species ( ananassae , erecta , mojavensis , persimilis , pseudoobscura , sechellia , simulans , virilis , willistoni , and yakuba ), four additional species that had previously reported assemblies ( biarmipes , bipectinata , eugracilis , and mauritiana ), and one novel assembly ( triauraria ). Genomes were generated from an average of 29x depth-of-coverage data that after assembly resulted in an average contig N50 of 4.4 Mb. Subsequent alignment of contigs from the published reference genomes demonstrates that our assemblies could be used to close over 60% of the gaps present in the currently published reference genomes. Importantly, the materials and reagents cost for each genome was approximately $1,000 (USD). This study demonstrates the power and cost-effectiveness of long-read sequencing for genome assembly in Drosophila and provides a framework for the affordable sequencing and assembly of additional Drosophila genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"QwakUD79SCYVFmzhP2sF6AcfTocGX4YldGLdKZj8iPg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490958","apath":"\/ggg\/8\/10\/3131.atom","pisa":"ggg;8\/10\/3131","ac":{"\/ggg\/8\/10\/3131.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3131.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3131"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3131"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3131":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022My7I8ADKqi26caV0fPSVl6KKp7qhCpvm4VAxzCBmVA9pYxHmm2DOBoJYh3IyjeOzrGlUEEd6Yux2dlNV4FSFiwSQYgJtVSny4oBX941Y74dO7Rd7faW4lmrUq8JdyDTpCepeZl48+g8s7rTYbpXYMQSGtSRTugfcwvt4Ad3s4ygh0jnuvrRX+4iQ2pmcGwNa7Cn+mJknHO4D95MVzCfCAV9Ks7Z8UdmI+neesKOjC6Mwef7bQMtvBmq\\\\\\\/MaYpObFnIaC8I\\\\\\\/yDbeIspchLZqP9tsHD\\\\\\\/rTsHHMJZvHmhff3sjjjl9wrfIfNncIdgj4oh4mqeWgw5lDFVjLYS7qytHV2Y0IdjygbIiP6OAgQs5pRUFP2XnEHO68YbMdw3l5fd3EMhMDYNHfeRUX81t7ZJQCVUG\\\\\\\/qoyRSrt10pmr2LBhb1h2VWR9fggQyvSMnDerYNhn1R0Xv0KQcaq9Zclo+f0Rf2er7f974r0Y04HAvefRf5eU4OewortBD1yU9lGT4F\\\\\\\/J1wny+euxBKhab9taTJMbyrYjpychZLcfRr6MxptGvb2hDdMtsApGNux9tnTOIlbl9D4uKPLMNl1HJa\\\\\\\/8ifKq8sTGvp+o8v7gY9jPdFGGs7YxJ5pTrFuICDnfLwyrb1D6cZ1zFtLoA5HrNzXpbxInecnKzHugFfAzAx1iBk2PMltQzZFoXTZ8reFPvhr+KRgpmYaFETbi48MKnjrwAyCbKnea7IDCGxxxOkKRIBfczI5RFblc+r2NCWmHx\\\\\\\/6EfGGpZGdNiu2CtPAQA\\\\\\\/u8pCUwdbldqdzM7U\\\\\\\/H6Mti0rh2bvTnXI6s2MFRsAievBSrOTvZqxwhhDjFf3n\\\\\\\/XagtpZ\\\\\\\/3XITfmKC7yX+fdbWQ+WzZ0wsLO37uWm7ZIKt0jQ\\\\\\\/GqC\\\\\\\/HxiZM9Efuj+gl9O2ucSBVEi0a4IfyMCzZ8JR+dn0axjHetLQ+7MR2jkA\\\\\\\/\\\\\\\/CTQe5i5P\\\\\\\/0OmPXbQgeX3AsCnRFaSahbcVzMkQEiq7wua5RlX1vjl2IuPwTKuIUtD7sWERT1wdAWH4OUaj4orA\\\\\\\/qkohZdgyeXEQNcgC7UjeSq1qx\\\\\\\/3vJyt7ATDA2KCXU3Fl03Eqn9i0JtkKtcVthXb46gZjI2RtZd\\\\\\\/EOwsAvXOOf42yRrggGRukdCQnkxIOqj\\\\\\\/mcGlbcHYS6vS4bDUXppdv0ksTcahSQqz+aYD+Aqs0yWMUHv7qd9KqFvoroyfP42LF9a3lcCeR66Gvztvsa7WoxV4\\\\\\\/a2Qavx1euYsxappbeGe74NFg6XZHCMv+sXkFmV9u3CoamW\\\u0022,\\\u0022iv\\\u0022:\\\u0022bgjfU9ZR\\\\\\\/4D9D9EGmLI+iw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226c05c47b0276a12cc98eaf64cc57c3007a343d197e7bf5ad519be08230b84d99\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O+7DbUy9bAQgZ6og+LIO+FhnX6I7bYVCZOoTzefjL99AXI39Rz6RTG0GO6rh9yZdW\\\\\\\/HB6VEYIw4UHsZdsB75l6Or5\\\\\\\/cYDariCF3PVnZGEW4po86qgDgc3dD7RflEQ1HurVmil5MCpjiCxNJOic36jsFQmMIqTXkL0z3eTuBDrjLWSHF21XwkT6KCKU6fVr3cQofmCHwcIHSpFrC400i1vgMZnAst6VrkEDxm42HBMLxy5PeKov39nrDD+Gh6IEnEps+lRBN9BapjW\\\\\\\/V9MmMadaPdEmnM01cBvsoXyP3sDBfH3tGZJTE0NVMCmdgyV2EKiZAQqkBonILK5zjAvP3Vum11OBFm+NJ1BRh+OTzNos4HImd16HgXdvUZVhfvoQZLquy+tw97XpkSJVTXjCxuryB5CqxY5xRjGlFQQCU0jYvFU1NZfbZ3ruSqG6nU3R163EZ921qt4\\\\\\\/YhHDv4D+4EHUbJYis\\\\\\\/SZb4a5wzWi\\\\\\\/VTNpN9MbRINFefIs7WdyOnl4\\\\\\\/zuYdzvoAdOh39RPVVQUXi1rq9b6DW6I9k5ZJehiq3C247j8QnfSowTMGkWFHDpHYSwZJyB6jLjWSOPt9oeoeIDw9GiLJbis2NL17NruiiBPKW3bcwlznhxsBW3rlk61c9fCK4ecgx0lC1HkLZDalMgOK8XldkZq5JSKzRxuTSn8xLeVm6Tlzh\\\\\\\/QZxDCdXNaEMen0c9cU0WOOIJLQYHPHqTFRvqebiHa1pDYVCs0UcdAmmd9+62em01ouXU\\\\\\\/s0rqNQB+62Wj3fbEFKi25M\\\\\\\/2tQrcj7ubJz07SLSi1EHn2sCuXh5iVhHN8k4jVgZLjWnaMJYwVRJ8bHiK\\\\\\\/b5yP2sxbTVju0VPWL56WCXmbTa+JYZCHItuPwFQITmjYcv9Tfmvyp\\\\\\\/0urrEJaQr9iigbu5y48ClLZrELl8mKygOeIYZX8k6le+5R\\\\\\\/npdYgitFsRU24On\\\\\\\/pp+T0d66DBDUpZO57zlW0ni7HfCqwso6p8ErxNv6SSykkeHwU+9S2TOSoqp3pJgas9NT9x+lrtQeLYwDzi9dbtXopqCHl1UR+dof6FJz\\\\\\\/IObugxt6Q+WizPRWXVQsAr8JbFCAZlu4XdN+Qw1SKC2qDt9XczRR+RQxFYdTq5nSXGsCvaQ8ns6vA0f3LuSSn47QiFMDjPUxZWNhNbYGlYOSWup2YvV7YpAGEarArNpU5xcWzqqNIiGJmMhC1x3PX465ZrGNaVkOXmUjkyuFCiR3iREI4zMngv7BA0MWAeR\\\\\\\/PkYL9\\\\\\\/eloGV87hqG8beOM7JKbQD7W9xO0a3A2B6H3sQ4xoh6D7M6+JksQ5sqPsdI4MGHmQ++oXXT5Nw24+k+Q\\\\\\\/7\\\\\\\/t8SxpQy0ciOUtDpS3h\\\\\\\/39yoQ3WRqK3xg34vB1i9rUz35HA\\\\\\\/0D+UZ739TsdOaR3JTCDsMz20b4L8N0TMlYiPdEE+NIpE1WK41vLAXpZrD7\\\\\\\/eaI8TzDBXmvOKbFrlimaeLQjFIwonIUx+pjGElMbHOztYv1qjewx\\\\\\\/\\\\\\\/cnHA1LAbdtXuSpWzr9KMwPUDMy6edVmfZplohuLamPeo00eCwkqix98TFjDk9e5nhF\\\\\\\/sYo4ujfYObT26vG8tHehIeU1JNDXZGTDYgexC8PQzAo4pQQ2gE0AQ\\\\\\\/XfssmD8XFNkPxouqnOGGOYW0nbprbeDT5ivUGwQqjOP1PXJlVmrHs4f5DZJXX2LRIk8rnBxPvOufYZFpFFdx97ZLMbGhPZbGqsy13G3SXvjZkxvGHtNH6+ZfNrY8tlSD9TFLJh+aLSSHT1HzYYXeCXLDroTHsnW6WMWUh07O0YpbbWLGn9VAnZRG6Tax9DolxU6pZwHGu\\\\\\\/ETYLHTcakVFvCR9\\\\\\\/1BPMM4cFT3ckjqjZen2ICZw3udJUG8f06LvObUPxUVaBMGw7yVFTE9RFdyjmCWnFRYGGlQXEb9z9hlXWElbQu4WG1m6ZA6kPTpZrlGJSDGh0cLqoVf0CRwQBgsUdetg3FGweFoyRivAiCFUQCP2V5eaZz+gctLViuvZKOIc8JJugVGzQK79khp0k98fYO4uFg\\\\\\\/1x8WH35uew53qUSaIij5VUTeUdTLP8XMIX3smBueNVDJ+RkWkG0lYt0\\\\\\\/z9ii861X2FKWEatJR5N5wxuYDP9wP+XBZ8ZyRJgekrPO\\\\\\\/9xqWkWVkLPnHUrNObp2q0QCsDyt8zUgeqs8sH05BmXOlFGPGbkEkNpU2hfFrCkuNvY+2R+42XUEj\\\\\\\/qgALlTT05GUrUStlaQjyNRDaj1\\\\\\\/qM5BbwFS5Te4puUB\\\\\\\/L3y6Ld\\\\\\\/N8nEGX0DdXdWZtiiFW69HOLXalmmDWb9Suz6k\\\\\\\/gxRp6mQFnjkg6AoBSVU5JOvJ2jdkcafmuJV3WueAA5ww9eHaHC5EeSVeHu3QSnizuJ2P2aYJTBocN9a2Rr0mw3ikM01LfaiSc6dHM4ge+AAMvwerxhIo688z\\\\\\\/XI+mEiUVVTrrhIEBkq63rLIS++ZRWqCIOIO8dxdXh3ojUdibRrMqyXarjXE+Di6Lq6T7BuOg7UgTfiVLz3EkxM9ewCTrOmtNV3E+\\\\\\\/uy15qHWScczs+TmtKil0r+tr3ZnmQFG\\\\\\\/7Hw+rRDXKIj1iFJhf2uFbzAkKU6ON7yjS1eLIbeSqTCkXljSV25ejE5vbdAt4B4k1l0fb55XGgM+wjPF15LEYU3Tq+5hjhnstuHV7tngZUkRIqRFN3pVmKXmLuTFRRBIBdzVmDullEfOsLBJEZzeviSk7cRSzkdZ96YB2RFfveiOBvXF5R1ojL4ZWEwGKVL9GT+9ASPqsCfWusXaq6L1LJVVgW7+LEyoQoTz9cPLPHKslhycLLnUxHc3gX0P1woJOW6XpxZvOzgutR80l0beV2zJ77RXJLk\\\\\\\/z3cmvOelX07Ah9pFJmPzeyED2L\\\\\\\/8SbObVdwZyMc9cKh3B2DUv6OQNH3LG9OmK\\\\\\\/EtPVjCk3zZFl0J2QThwoCBCDmjWKDDl1RatmMfZGsVsjWlFi17I1tAYu0n0JlsHlgz\\\\\\\/8xHV6nOfGcWDl5RBm+isIBmlf0LqUNz2dbYmL0ADdaslpzLsScxaHSVM48S44oDumrTBNzu3Jk1qUYI1o+5w+pezwf3tV5tugfftnMGTJ\\\\\\\/iYCG9Nfr2SNOkxoyl+uMn3sft7ytQy9yPhDZCWEDO56vc0NxlFzejnFon6rAdbclcY76Ql2uphHC8afrWrRThbwZyBVP1xu\\\\\\\/HR+wPXy\\\\\\\/Y\\\\\\\/2NiP0bQWz3SYesUH3g5d\\\\\\\/CmhB75vptXBcgZUAnVEcgD765NByFhPcQTVsp+73K71wVoJepEcvCFGb6NOigQYM947+HOhWSq4VUb5obVXbElnpTH9SEbcS72HzMWt8WizRtgdMTI9CIdO+yMDd9hwU5bZLXHvpQkmhuxgygiXHYxHTTAi1Sbcu\\\\\\\/j5DLp3\\\\\\\/8X5iDjEqCczl8fkcH3ec6Eok1yJG1ZNlpn2G7xw6YMa5EY+OC2nfWGspRETn2\\\\\\\/A4dFsVfElh5zaYNeeci0tjH71u76f4EB7iRh5AiF2lkBSX2eY4Rvi5jVnhSsMusPm9zEXHzCSi3jyxvhu3rx1EV\\\\\\\/BgPU0cmOSiLrwZTVeypnQZcrlfNCvhGwQ6E\\\\\\\/G2DTZ4Z55PfI3Q7WEZp0kd0m9cXD4TN3KdVq8Gww5aEBr7rAcvbaAlYJaFAgd0kP5oo6\\\\\\\/ltFakB2Q+ww9pcN6nTrQr\\\\\\\/0HrfCs5mUHCPnK9ADgktqV4i+1NL8lSIYcrQ1MNJu1lM9DMJJidJgu0VS0iEtlvq\\\\\\\/ji1F+Vg2SZuh63VFZJLIj9HNpCFCesAyiO6ZjYGtuJAyX6labl9e8fYCYZGAVOCo\\\\\\\/aPtUqWMhAuCLBHNS99\\\\\\\/0kVmyLkra3aXQGxUcXFyKtm\\\\\\\/OYpWjRJ8L0eqpLd7pVF4IKkLvmS6\\\\\\\/oufGZ7lEYZqSgAzly6IUjJkpOAJuuCjF328TiOn4Uvo7G1R\\\\\\\/FRa7CgeA5gSqpUXuZlylz6pXjoIFhadzGbGf36AM\\\\\\\/Rn8w5sXusG6sqy5zQPvhszK4SnRvCrOmmDEt9swoUr9NxMJFMQq0KTWg4VoV\\\\\\\/Sa\\\\\\\/j88FywIk+Y5rHZrG7\\\\\\\/nsiYgwL6efabW0cTqWDVilgQWaDB7gnbX1Tk26JdyRHW\\\\\\\/BOnN09fGDhrJCFsb+d6Z9wglYgyruOURMINwbJgohqe4n7iG+PdZwZ19yWRziKjTmBqrvxu4MeShVgyFAUk3U2PJN0T2Oi5K8cfBnDPBP3fRStd9kHdZ3XylWkYIRfN9bkb\\\\\\\/v+Q49\\\\\\\/CfdhjwQPttK4kEUu600rXkPn3kfnyPOb\\\\\\\/1yqV4VVwNVA49j6QpQ1Yfi0LnimY+yn8NTX8AL\\\\\\\/v2QPDmwqsBl0ag3WSneQiZpJ1GwY9Sy5cvWU19Eigjhgxgs2joJ80VfqQ0Q4jtKmYLDxCPdQT4hFtgAJNRpk\\\\\\\/m5jMbNDClkYB\\\\\\\/hFQjz6Ibfk4D5Bone+9IpfcZ345GaGbAmJkS9W4SJQH4xpqoFF3QsOGM64iIvkgp1\\\\\\\/\\\\\\\/R\\\\\\\/KcORRaWmifGu\\\\\\\/uYY2qMNkdNSG1GCicfu4StjzNRqhFrF8hhX06aOXbaEcyarruKGIKyKuqwGy4ZC+6IlnLAvX4m1IdGESrWUzNXyZQ8Ajm9WD7\\\\\\\/HPYAtek\\\\\\\/ucLCjbbhqNQSM4a0aZ\\\\\\\/VCjNxphd\\\\\\\/RUZ84pU1zH8auOMEeMkCaedd3Y5w1fTKLfq+Qrlf\\\\\\\/lV7E2KXBowSxiAp3IRJUwjXvNgv+6OI\\\\\\\/IkG9yOVb44E1Az6OUrSGsV4gy1v8ZYxcTToPERjV5jcY4mudJlWjaUYbyS8VbJvYrR\\\\\\\/z4LuAUDkEbOoZZWh1Ai+\\\\\\\/bo13BZ3c4flaN0y6bcbSxXpGrlvTCA9wNzOg2\\\\\\\/MERPqodKlN0g79J4X2RQcjUCvY9IhHs\\\\\\\/E8yzqeajU16S0QqYsyekVMDjmWuMEbivTedNC9M\\\\\\\/ti4+D9F7QcPcpcSWPcaDUE7fEw\\\\\\\/GLAThL29+rI3ardPc0AIdsoaJBpc1Wum6DuHFI0CciFUlthA5Pxso\\\\\\\/72hmccYEFlmCWwkLtF8paH7y3aCzgejybzo\\\\\\\/9EyzW7mRTLOh\\\\\\\/ZaE8cGIyRO0KoslkUpr2Ag0ZhXcj2b1bqV+UAlY8BMCpoi8bUd5q+CQfyeKlxGyMlAWm1xLoGT\\\\\\\/vrhYM1f5mH9UScYOm0LPSktXJjQxjzhWgJIPK73pNYUCJ2p9xCYg38oUL8yft6tJFMLB1MPzcK5DPVTB1DEa87oS+lLCtoFpyi1q1nR2A8zUMsJ4sBtSjhzedjTh6uEhyqdNKfTuvX5aVSuan6VgrcTYhBOWdLTQeR1tDh6tIDuYISNmzZki4o46JeK4ta25Sd24tU2KKjAJVT\\\\\\\/+L1vKAZO3+ESKfBXrjbBQZ2vfBp9B67J0iXeRpZjxPCWQ1OysFv4I1rJ+LZxlgf6TR4dPuhr\\\\\\\/8MBzjk2s1zGGgYhG2hqPtkSGGsJgQDS4Q7L4Il6vdEeOMd8tR2aOlYzVR90PVwIwsxk0JaV+n3KFa7Yto9xrvXxD0745LaSw\\\\\\\/oGcX+lPjmjWzbf2GZjbRSPk6n157+BSMvD\\\\\\\/\\\\\\\/ie1qCuY4wYOOm465TGggtCsFn9BDrQ5fX\\\\\\\/\\\\\\\/dZk+D8gia3mPn07o+HY8JKL4ZfKi9cnQqYsVY0AI3NOJZR19QuNeJNRh8GiKnmt9SC8yxzJ1v\\\\\\\/c\\\\\\\/2LhtYDtqcqnvl9uod1qvF+P4Fh78XAAxPGSZijOvfFDNGQ9V3fv\\\\\\\/Vr0YGl44I0TjP+CikIq+pDE+\\\\\\\/7naxxkR0hIuZNQmOlCN9GyLje1EKxEBDVoe9F4a\\\\\\\/2r9QZuIRUjiiAGKQxT1BrIUG4t2E7jk3k5m7AoHTd8gT1p\\\\\\\/KzFKXpySC7Wp0bpu7PONpT1v8OlgpeME+jdORI2UxBnFa2KdURbl08rEodUmWwyjeSkz3ZMJ7lgXFQ25KkHQQOzTaNQmWqgpocK7j6Tk8ngNtGNIgLN6oTYlt2s+D4JS\\\\\\\/HT5Wqzvs+4lxwhiGGMF+UyyrHopbH6B33\\\\\\\/2Lotap8OyyftKZ+fJ24HiqqMaNxRiTqsyKIpQWF8IBN9nd\\\\\\\/gzPCK7VZ55Bp+na4KPGt26vKYYDHsPgFuHCZ98gFUHjgIXYKHqdFFfMCTxxtTNbMDlaGWUwkFcYht6f9AHE5vPMcRxdo+og0AvRequvk\\\\\\\/PV5\\\\\\\/81RmuayZu45qWzoEbMLvgoQ8s\\\\\\\/l9csKR7oyZJVhZ2SmvrcdmwWRZ4JWp06DVxbqwmOoXzgnpRsE30SxRznkGNI6zm0jMZf\\\\\\\/qag13BLRWeb1VgHOULWqn0xE8JnkEQfl6rF13+Qw2C8INKfIUcYGC8I9e1ynvXr2exZv6o\\\\\\\/5CdNsT151zdVOEiCbX4bVTHZyGkNQk4XAwEeDHe2MrrVq2ky9pOlENNYm8Sk\\\\\\\/s5l65U0avKHhRt1NyQp79KxtDaZDBiYJu3kLheSk+LOM40nxp3rGQ\\\\\\\/s65PzDDnvnX86ySwOsyDJNdrweV4SxWLGL5DtG0lhYvs8Qm67VWdSoE7mGr5yL9ruQ+t9fmK\\\\\\\/nEqLkeYzk3trh0WmPBjU5G8Q9Zc4TzEoRFOkAq7eQBbKeaPAXzVOMYHXlzNDjhd2xRYqJP\\\\\\\/I85Z2hXKLRrLOtIRmTMMJBbyuJeNbac8mC7bJU9If26wE\\\\\\\/8trjbJLj+aBWFHGmJnkyzqcLpwUCXmMFFEKGsGky1ak1IMkwb+WpVmOYs01c5z4QHmuJG50VmItjRGRRCywYz67Ulm\\\\\\\/4L4LSaUKlkSqMOowi0j7cA4uVKnAzeMm4Bvw\\\\\\\/MmgtEEBSCIi4bRwTx+X3J\\\\\\\/yeuX2lt69RXKdHf7NZIT2R6odRtq2Qrb3H4mmknheO9yxkyK2tQ8UU292ZKHLeNWve7qSDrK832KqD+z03MAgPNRf\\\\\\\/6V\\\\\\\/QVay02bmdFcwIklX8b5kpGIB9zEjLw33OW0HtiO20rnVxNpvfrW8iUGCI05u3ZmuKeD5x1x6WKdZfvm8ZMD6ZiX2HGs4LAx1q4ncttd1aF9OltrR9K+iWjUZmmgcqXC4uMGitku1XhWtNzcpPTpCfVnuTgwM8y0DqqaaBqKcn\\\\\\\/njubRBmuSaqZxasSw+sqN2RIr6qstlrTEdhHek8WxYS7UqkViYLkr5gT8GvkUoAIRZlU\\\\\\\/UK8KWgjgXpxNMKbD75qu\\\\\\\/xu6b+GKEJjB6DuWllJ+Ymydlz4L4hkvB\\\\\\\/Ijuu\\\\\\\/FydcmqTAP8JL5ZXrJOesKXFSx5eWZYSeoWhO70+HMTKiztFYWt2\\\\\\\/PP3azxVwpeo0NrjCDejANhyFHor3aRNJnLYv32BvX7VeqjamDr\\\\\\\/bFWd4WekTggtNznkAmU\\\\\\\/EUElC1w2PNFXaEfNfOnQ\\\\\\\/AXTbBSChBFVL3KewtbVj5M0mh2HhrDcKMvfEm9S1hufglf8A7XFHd9mcr6MZOq8peOUCfW8XXrmzrQEzKm5um+coCYoTo7O7+b9wMqt3Cia5P7wGfmjexgS\\\\\\\/onOwPX0L2vz2IuOK29364CBeMFQkTT529eSOg0pwAIQCl7G6f5PAzpKmP1IncsYDCBegw5f5Vluj9NlSukSXwKwYTYZTa0UF0myTx37ST0hrGYC+hQsWFP1hTgjLyBsrHzVBik2ngZewkMHPsznS9kku4q\\\\\\\/FYb9bUUsxMrnW9MUmovIz9euslNzUmdsbYJ21AQYB+hUFvnL+\\\\\\\/WpstPvQiiU00PH8kVuxxHXnvREEwjKlIvJbeFmj5BS476Hr+RRTBNLM2IA9oEZpbnPc+2GYxMN\\\\\\\/CG3Z2iQv13\\\\\\\/7Y5pDCFWJDDh6dny3FtlU9akU04dlRwqVrQTdpuRQ0OYRaCghWAyNaxz+LweGFHuln5vn5JSHCE4VKRqLczcy5+dZ3BYcz7xpJwacXwcOtZc42v\\\\\\\/xzZCqu7q1bFXDX3BP1fKXDcRzOiVZ86zbjRyFFhVA74w+HX11bHP2ZrKCXJrq5KBIHKWCSbrdLWsyDJt7y6\\\\\\\/guWVES3bHlKfh4gqoRCSoY+yG1rI5t9PeB8MtYC8jysbs2IaPfXKaynQ\\\\\\\/fNCc7p\\\\\\\/bqXd0JB3Ge0EUFvXhWFlK+03AIo39vARxIkQA3xVoroSjxuZ4kRxzNnTCiur6j1r3g7qqDEIHN39yYRtU48ChNZfVZwn4PymhoDYwiOk9afCS\\\\\\\/F45u4ir\\\\\\\/b9o5vvdLy\\\\\\\/5EPl+bnMXc62gJkKoAC+zQcEb5+Ba2aVdGYYzG5JQ6Hc4bedDCHmBtwkMiQkkVhOz2KxowzvYpsxY1Pi99A1Bcax\\\\\\\/Kr+8mVlNs9iHe8\\\\\\\/OtqubcOoidrQSOerrXqiQtMuA\\\\\\\/l0LxhWRpVbsxijTWINb1hBILw32jGkMcdNidPWsUTUC5saFBAIYlu407ew3V0Twsm6LIwRs+PIWxzVo9RK3c1KQzfMoUIM7awC5vLhbVHHRjZggcQqkVUo1K0s\\\\\\\/CKgJT4rDk42YFZWqTWvu7caKYUJcoyCouA4Diw+u\\\\\\\/3wBK5XRC11KSy7orgy2sLKfOtwSOD\\\\\\\/5v9WPF\\\\\\\/ijXDmVqSTDCbP4d433N2JpEEyJ70q45jTkIKpAs7XCe38hgUUXyNuaKk\\\\\\\/2TUNV\\\\\\\/US1iHCp84gVTJaDJ9w59PfmF+ode9DMpiraM+ag6B\\\\\\\/SiZhczM4c4DfW0yRWOix0wPImbNduWxTalTQoRpAG6RIfR5FXBq35BiJh2fWb\\\\\\\/nunu68Uzq0J+LUSExiZjThaXtYFDDGD7XBmXwUX01+xyJPrRafpOMH0BbiPH2Ud7yKt+uuj2IwiYconv7QRLfYCnhrAUJ8hZdGIpZE5cNMqMiXLoU+3ZRwUR8OQu+euvfKjCun6eEndufAbh36iSNkgq\\\\\\\/3mreWUAwdQQHyUb6teDpEpsSW5mZevDB\\\\\\\/1RzL0QOGPgVi9EMXeaIxIgCF7P4lywanBcZHsZqZPsj0sXXtsh1Mf2yPpYD2Q7aJ2gtsY6rCU2qtmjZrpt0dQ5BlEDCBqPSdjNrN71fESXuy7pMa8BtqPHUMT+s87QqBmo0TR15CU4doU2I7tm67H9\\\\\\\/8mbUDkrMD7dnOUQOT5l\\\\\\\/gfCsLHFYOovwgC3t8XLVmRPQ6OxRMfIv4uAuApwAzW+b+WF99D8moCm1XCmW3tcR7E0hKnMi1znIPxtUz4AI0dzTFUX4PnNytGn73gHLPlEJRdTzn3mVtCXTWNLPhJfm7IWXkopX2pmIqmGlcu\\\\\\\/DuWG4WEQcCArAsfgPVP9NGsFe2PqVZ4PIXMxQ9F+uqIAJv\\\\\\\/hgynhqX3y\\\\\\\/34nBD\\\\\\\/rk6ecOPoqnS+Mpm31NzoQBBrY1nj7o0X78G+5MmzAHQXiXXXEEGRm6h0r2+vVEVswJpb2UPot6QXyGxmy8\\\\\\\/EkfmaMgo7QLoDj2Qhu1gimBUl9aV3NUFH7iLQIeXrDEumfZLQLE6l1K8\\\\\\\/IAxhaw2SymtVEyQF9+iROk9mKViUz+IGv8UT32X2pQFdUzjOBuc8YFG80SThrICcsd9j98CoPdpogcYKh0nBNwu+pMA+uXHQkQ0J1lAi48ygXIdf36l2i50\\\\\\\/TI6dVN8KFkAQUUL28NJBnxZvD45lzpz2rmz753pqMujva4bQByialINJXZRg1N78dUmJDJjupcaiwLqd\\\\\\\/yFYOxFr4vCc9pIgl2zkQ9G7573gnJN1iJBMjjw9mRq0vNPH3ds81J0ahl0Vm\\\\\\\/FDqo\\\\\\\/V8V0LDv52WDAtlPn2HiD0qszI4ASHo4q85mRcZUjjI5LpFbyFNZrpdiaJajgCnEPahHXwAIYWiOF5xBunP0gZgIem60wUHIcBWa\\\\\\\/vonjCyLWvVwDFp3Q5\\\\\\\/kQR78Cdvd45LD+BjaSHl9aaghI0dH323JvQpMF48VqYUXCPXuQOrbjuGSmrB\\\\\\\/1xvVheN8MK8EM\\\\\\\/ipxbE9AhCra4cGlhRp3XbPIeCqHUSDYc6kytD+5NmsDkRdD+M8fg7vRD9rwye04r7N1l7ZvxWsBBySaQIKGxfehGYB\\\\\\\/tZ4YxGqzRbbkEzGpxyUbTV8mh+08gQ7JpiySsHHwAZw\\\\\\\/PUoKrYeUqdClM7ILwJy6N0azRHuD7zQHt732BDm5z2hOjqoh3AOKAYtT1iCFRE8dnw99DedAmOSkW3X6Vci504Rl7F1O\\\\\\\/4JfWSRihdmgqoatWMbpb245A3+vipp698AvAGpiV5eT2dV0jmnj2YLz2ONUydoLx2wORsc\\\\\\\/vjUODjthiF5nHHAPBw9FuOwgsm+DLKTsaEN+NyAUNwRFfziYDLPTvU3NWFb9yJ7BNvpo9yZvt7Qls+cJPb\\\\\\\/QSCBOs9YzbvD746qSlEB04FFGux9aa5YoKWc1ItIrO7QeZDvMr0\\\\\\\/GsHx0gOCg3MghlydgEWAVlZGdONSg7dTxTspFXyyka2aJOZ+Wu8K4jb\\\\\\\/Aq8veIYdrAELXeJhvhhpYppAXe7TprsPUBdP8vD7s9DARG6wGlrRcfdT2e6RhK\\\\\\\/l+aGkNpvcVddIs3yjgI9l0EnfDIS6rZxZd\\\\\\\/bPo7QI8vn7LgFH78li4wKjm6jnLRVyIZ7TFd11i58uD5hGwBm50oOMBu\\\\\\\/xjhGWSSUCVpz3pwcP5j15LtxRKUZgYNPwuvDL\\\\\\\/M8TwZSmKNwlHH0Rh8q3JSCmxxeiOC39ERz9HCx69QFez0222J1yT2hu9Yn4Qvveu7kKYiuGfe3zFF+xuo\\\\\\\/UwI2n0MGbIAoHdfMs5JmL5e\\\\\\\/kylFUZ+lmmwd+U4BzHcI6ahiNPE2yn3kegGh5VvxgaAyoe0QCUgayAYXELSgGUmdpbbce5mJMhZzs6GSH+EVdhuTONXKhaI7qrDOyYaaUWXJvb3T\\\\\\\/V+zZuEsnX2f53k4m5na4POdBQaqCtXPXhrs5ywpeLHKrclIghV3owNqhDTo2Zm7WvwolwFhjExqWtRWk7zdS1EnE27wYNpFfnEPoUgmhag0Kf1pZPM+yAj\\\\\\\/h\\\\\\\/TEQnmTwsfLYCaGpwn20K8Eo4Fzt3iaFOwR0L29bi6NpXHNJuh6mVT3oyKTSgP1U3rc9YD30YBs69EXZkbZKyvK0u9R2otJ0GG1FpQbKl6gDugDF5d4AJWllp2Ypt5yY5LkuOhlPEl70l9S4MJCv+PbeDtHk5G2hsV8ZrlbYx3GrpGULG9ilNXf4oCxrJmX0ZgsXDeT3WtlSXyLuoDymzcH9+h0g9eln31wiVxa6dRrnB48sjtJcwL+uui3V81BAKQKuYczDcFa0lpbsveSXnCAoXHjeTuJrQTxL42V1q9g55oNqADgZntS5TcdLX1T0nhG11i669x2CSkHgqLYfd3myl4sRSSM0LAO\\\\\\\/ZtD9OQLVbEvGupvfmKoCL9IeZlUTJSE8ntwfYXeMUK82te07o\\\\\\\/zX6P2ZGDKYgrL2UTe7BviE3x7M8xs793iPgUUEAsJ\\\\\\\/Wqhh3RNsZfcN4lJrBXEsa7VX8FkvuFYomIZv0re96kIuV675iCvkiDbVGMbz2jZ4G0xF+qhRMeLyLpS\\\\\\\/gakoh2DQMfmUshXq4d0yR1Nm9A9NA+TksM3IHOguGJ4UTPZ6nK4xvhzCzh771mnt\\\\\\\/gLemxR+30gJQmmJaB\\\\\\\/uR4qGzRzf2RgQ1csjUiwd5EsnrdHvAm3jxohKAggT7GGsSqDaHjJBZOIDckz7cVNgnNvseXkObCoWKZktjUtURF0uNuFsEEyyakgmX1vCfL4S35wREFnMydHUYrMjv1QIo5gnM7wBBEaCh0ydAzmKclR7e0atZ5k4tYMpeuzKrcXdUGnm0GidM8Iiut75hy+HxJ0FjCoAUn3GRuncQY0MMNrfFVSdWZfWCeV6SZnSoF4qUCe22Y0nsu2dRq5HggwkD85ecB6zbTBbzPd0wLzkCCZ9yoclf9RYOsbk5xYwezCoHcmvw5+jFt4RhhL9Dk7Q+uuApigJCN\\\\\\\/NnFuhREqGYokyEMtbyenFkyr271PTyKvLFdEkyv\\\\\\\/e1vENByIlwAuPYUSMjfUqsmaKU+ZHmqgaOQByTdKeWlxzZ5umrD8OyB85jp2RIV25IutaSqdje9gV4Y\\\\\\\/efl3179IG7JhcT6ZbtVTFNcEM6MKybUAWWpA49Gf5d1u3wU7+QsRxHaUpITqkJkH1JbdODLx4kLNz0cNwxlZzP5DV+xUke4umVNFZXJfhKbwzAAaozC61Nwj+\\\\\\\/dTgr\\\\\\\/ssSWKRRJAyx8ykBtjkFoWnT0maoh9rI6a9MfvR\\\\\\\/qBGATx6M27hoj5Egjg0Eloq5lbvNjBjHGQtsocNtVJswN5ZuMZl4b2hLJZtQrphIShHzrj+a37ZQdKPmEnqAov5G6HgZFN6lpoP0khlycSd+ciP7hDPE6+taiHtBhQLmb6UJmLsIIsM\\\\\\\/p+m1VKmHjpN7WtiaenYda08Xz4BgEBblP20lvp7Ci44QTQSC\\\\\\\/0V1K6hTmg6bBy6lvg2f+tjSsf0ZF+\\\\\\\/I2dAYbZGJs90pdS0CSAV26nVurdEUo3Rh+oROL0OxAu9Y1JlI6NdizegnGShg7ujJoWiJnMqrWcaGPxjT46xWs3qbjzv5ADUDxwVBq34QnbUfRiRJ+N+e22twLS\\\\\\\/4PbCVJ\\\\\\\/y32LS0VIBCtl7lEC05SWoWFIestopAwDjrOyr9eDlAvXuUinGP7cSpkNl4p5lJW0Y+kXJtIpIyNWAicNvX93LaHW6OwYJbNxo0WaJWd9fsNcY5zjLQs9Z6BHTLgYB\\\\\\\/sV3sDsBZpE\\\\\\\/7WrdKEk4Rz0uCqdBB68e1j\\\\\\\/w2dMKxgdUAD7vioLksVnyL8FFmj46pNWxFXe3fI5GUfDNhw+xpmIp5ZUWAiQyPM24Fj94OMhR38Rnde9dh2T2j5qUOTqNrypjkEe\\\\\\\/Tm3fTZwESofAAfIUJ9a63ah1r3l5n2w9Jx3\\\\\\\/xv0wpZji\\\\\\\/Q2YOCzdkazeQts+4PqZGZmWz3rLerE2M7E7+TxF1fRgB\\\\\\\/TTcUJeSWYQGBEZs6LI9OYKGkA8kEx8Bf10ywo5n\\\\\\\/qJKLLkB3oAy\\\\\\\/K1UsOb2OhozYa8zk8GqeMzynVqbEYxFK9cGMoIgAsxrhYL8hJfFvSFUGgtgXzXBWPBvBudxiA4DPa5vbs+4BYqamfLZYKGsSoyqwvMHmYW2nszNM2tVtXbALboSnJTyAVxavAWtIhLPrEiz5jL\\\\\\\/5yDgQ\\\\\\\/H2PPjoi2Zjuc5GyXB0FcepOYbNsVmFO6qB09Ef0lQK07iReAVqu\\\\\\\/BiuR8QHMO15vsecsE4FPxmmQmCqrDcoJXWA7aixhXIYe6mwU69qQIe4Agje0tBRAVJV4fi0qWklwKg5liiQbyNITYtaJQoZyi8kv1ubVvQvfYGjpf1Vrwl9bRkjXy2dpMDvka+U5vYkLNBNVjiJooLqgU+0ToW\\\\\\\/VdmT3zpl8t+GNqQhF6sep2iiUoOwRvaOaw1dTUpalS1ckJpozPxU+MJlfoKFqlhjC+zoySTbvhhzm\\\\\\\/Z5dZoPckNSXSxxKRLQhnaVlhhfp7Qr9mn2yaz57+pZWZnZwOL7d1mVWg079wkdvwD03oDFYE\\\\\\\/qdKpWc\\\\\\\/vwSNrbFxaDC9dE8Qecq6U51wh8y0oCDP80yiNCCQCDyjHjf4W1wld6YQoXJllRbzbxgHx3j0F5\\\\\\\/GqhXiQjBJqEZNhcYuv3pP99XHrJa6keqkQwAQ9OTXWY9\\\\\\\/GWeWPtYgmSa+dBP3eXBjvPC4oZx7axvu\\\\\\\/K+jf1NOAOFrqTjTH6POPC1ypcIIObQsV333GNYoNnSO2XYNpt57s6Po6CyLhXAZM\\\\\\\/ytaftZJVST0d2MDaAb3F6PaAhOBs47so19h7nuCDY76C1wKgXYMGf\\\\\\\/eLRdvoLNo96+agnfzjjONYdPWxp+G5xYdbRrQzF4ureykrotxueydxdYy0PFsF4D8PAOW6DJymswa87G7PUW5ydQffJlcmG2tkp73ktOU5VAI0m\\\\\\\/SK2XCiV\\\\\\\/SsPg9MkfHb5zHpeB0akPKh8t+m7bSr1oA6VPvJntu1f9lnRn60nhcoqAw2yNlSnGo67JCiVsHjB\\\\\\\/jTht3jjkygZYN9\\\\\\\/BpALlf3c7Gd08B6avVM9+qKujuTYBXQr+XjpZOhiifq0gWcL\\\\\\\/YppXtcxrj8+ssLuKuPoIJODtDE1izKR8XxxGEutTSWCUi01\\\\\\\/lQMyPYvckwPXMXExa9fxNH0uOKiGRRaFJvZbETViepC69NJyYTP+1+HwLE0J0odIxv8FwubhWWyXWAB6QTF\\\\\\\/R8ihnnSw5dlYUvICmJSycPrxcEGsRWZUN1urW6ZTV0OGF5Utrdv\\\\\\\/kmd1x36t+MgfrIfJCVg7pOVRs+aoPiLpfcxkUvXK\\\\\\\/tuwC5t0zVJiXoNesFG0Wdu3GVo7jIoWOCbmgaMsB\\\\\\\/MRjQO5W8hL\\\\\\\/alxERwSpTXwwWF+GJ8c15wDDO4GMS1BzPyFl\\\\\\\/hheQ0UE0WJND46qSotp5Jdh+Qzozv1Ubz+2i0HXL21RBiw28zCATgSlDllCqcxb\\\\\\\/P2Wa5UHz7wWlryzo8NhxH5\\\\\\\/kE6HzvSkVHzo1lsT2e\\\\\\\/10PaYkQV1LwATY9DU7TWhE7n8UGILPLIa3w4JL2wmPoRvT55iiZyKDQiS82EY5erCzbGgVhSTWf6yXUt+KfiSdyrDFPr1rqdaBwIMPtAPsXxLHcWpetRqRCgrJA7xSa4KCKWejVkBPmGEFzZVcD3rQTWIRoglWxTqZ2SiIsVOovRjgXw3aSG5rR\\\\\\\/8zS0H\\\\\\\/ZgEnCfqPu1jTZwusw5rKIA8OHvCdB+hgEvIRaepY86mIScFU+OtSJ2Am87QAi9s3WKhcSZiKK54mUpu007cB52UtxzbCGx7nNh\\\\\\\/XzcUkmu7EoXvsaBPk+nCUXSz4YcmaZ35erEwUcpGEUZWqoFFXtMMtdyzJCt59O4hAD88jv44A8ph7j\\\\\\\/cYRoURgiBLDXElwhH0OUq+hh8VkwXK3lYxDi5R8c2FW4kF3IBlvC2G\\\\\\\/G+bgjHWcqMHsc0\\\\\\\/ZLZrL4HDKjS7AkSScYcRHjRax\\\\\\\/zb78Veyt0Horh9bu3qamhr6CFvwhGfmK0b0ffiCowNxG38g2j9eztWbmWlocn3+hiAx0Kd8gctYLvFqYZsV\\\\\\\/s4T8pYVU87+N73HtjmxqBKYkajOAP3Cmc2jrJ0x7ZFN0CqFBwS7qu8GSLIQqWi9kh7LKF8aHaUuKZwbiOErPq9m2LePVMcNyYua8C4ec\\\\\\\/USis7RMc9IvZ0F\\\\\\\/isJ7h2zGl6zDjFFqkgWXg9qhjcgPlxs0ap1ILjOYLQmo1SQfZ5BjAqZyjGEBo3yLpl4h8PLzq5L8RMynKHp7iHU6ApwMlBlUkwuSeC1FRRTIQ0UaVnMPkFYXQUpHUAzg\\\\\\\/qfnKDsEkKCuwpYYzGd8svplcCkOtr9qOF2TlXTLf6TquE4F05uadCf7uVn9U9\\\\\\\/lzEOoMijj7d3CjXxbwiusBJZdsZA9SCJfyK17pMtKLX7pD9V4fvqiWT\\\\\\\/SWBu0taLwZ8Uo9O8\\\\\\\/pPq5vgIqVHt+jggumsvUGJ8xdgOLB23FMn9aa39rD4ldBXxxsX8Iymngpnx92A3FYD0uZHe+NaA42w2qRCrinsNRNqsXxQ5TuHRypXZYx037bsk+U2RmOidDe+LiPW\\\\\\\/ABPeTVVDkr4R+H\\\\\\\/S7nqp\\\\\\\/5kEBisAik2qibAf2yFtgJglrQcsArE1GN1ixs5uxSMJNHNtUkyoh90HgwgALURUrVHA6z1moGEFqdr4qYxW91exu+4UGfjzG\\\\\\\/VhysTBzO\\\\\\\/tc1NmWkmkcG6bttDOHh6Fce9vfsoWbHokYGF6c+94FvgsFtYsGClGtNOss5k\\\\\\\/u9ZUKhmMaZfPec2MTl6pJVLbd\\\\\\\/MGZp36VPkMxfNlf\\\\\\\/Gk4gylnjA63reEm27o+G+bm8eSFx4CrcmlDnbAhp4y9wBmjDX7qQAixzfnZg\\\\\\\/m0wWOz00CYYTgs2TfiPTDFK1qW54ATuDm8q5n4Y3+1n4lTywwpZOipZ7d0NTIoog6vEQVPVmS9XJdJj5TZ9didZ5ezCcP06e7sAn\\\\\\\/4EnfTe3uVIKHY7mOj9LfjmdH9e4I24pp0xQbZyvRIlTnttKhNsv+8Z9G1IC2SUB2tgthWwdBSqGLzCWbAlODoHCjicYA++EJDVCFj5ftDzoWJuccJyaIT6PSb8tL3SGTWSSqs0YwoPEdQXqY7vbuF4tOdjD87pPHfeNOLa2iRcxiaW3N3KixiObR\\\\\\\/77vnaCHQ9dCp8hXQs7g31zzEfrpZtwLzIrkwYbnm1EFxPxCVXew4G5ZSvMdoloXzSCT6Z7apY2\\\\\\\/FRNylTlVH0\\\\\\\/l5T+bGFWcf\\\\\\\/vLgu0tDBHtc5+dYfsbsNuGZ2JqhiWhnV3fPqLiMHglf+JRB66vne0pmCqr4KBEOp8dE4oJeiK8OD9qytIHgfLW33a94esJYiUSvED0Vc+2tQhpmNjTIzxD6vgnYCRzgBq7FmUGtWYQMn5EGTOj8IMphJwxSjAQz5\\\\\\\/iIgIj9UxEGakBYLXoF46ULIEd+JiovPdumK7U0vaLs8EN6y5b2CLOx4xS3U+WPmBUT32FgCIOBxTrjqOHCZCnc03k5esf1\\\\\\\/IXy2CVChNixLIciV6sHARIkVvwNR2Q3y79jE5yQJn3Xq4tmqIL7sLHxxUaDQrbA0zd+dy4zZwNpVxV95VNtGkJqszIFjB8byLxIe90douc+TMNeSZubyVC7Lk3uKczIrnFupVlh+Gcu2dwUpoe1\\\\\\\/TpMy0KMEcpDsAp15pH6DE8TMiVw4O14Z6nRfpOMK1eyrDzZr1njMG2MUM6JqHYruKjjN3\\\\\\\/aWiqUbDfqoJqvZQJJHzZ5\\\\\\\/zexhnPvaY8Uiz0JeAhKIjnuhWhOnCGHb+LTu\\\\\\\/e7Lv\\\\\\\/vR4DvuUgsA0luwRC7\\\\\\\/zhaS6vwVtBVHN56kDSyg3ugkKyWkbYnYWawmbyMFzUDfER7Z91MAkXDcpNDbfu8mVDniWE+kFxYISIWBFdK7qvAN470zGs9iHXXlNyicsyJQtqJvHUF6txVJwmkCtZjks6CPSl+HGAKUUmOhClnB3xKd0AyjSS9g8qYB\\\\\\\/ZDKvDaRZ\\\\\\\/91zYDHsfxz8e4cfRjb+t32agJ2HHSPVgGYxxDH3XGeYNtJ2toRTrzPkZaBBeUVvcFv1fVkfOKu\\\\\\\/8ezQoYLunpr\\\\\\\/KQSLF1Il2ebJjxozvhtf14+KjL9AcBP59jiSjZTLIy9AsRfXah3HKi67akpwX397wLcPHY1rypQGexseQVlIJu4eh0IwX07S52q5\\\\\\\/f\\\\\\\/obS+L\\\\\\\/BeTMjK+AzOoGkt9xkIqa0aT3hD2+OTNfcudieli3hpnDTWzuFK37uzNHiTZD5cMRUProYN9A0gGZezR7QxVHhPLKwXpjyqSi\\\\\\\/3zi1QS7uqiJhRZtxxsx2OGk81QEA25OiAgEmOSJn66lxVv9f0WcQk5ibHHu6nh1dU7b5RUg2TbG1ogP7+LAvNj5IteQvJgCK1Bq0meK9vYEbfn3VjTQQf+M5fbXkMLq8mO+s5gVlUNIHw2gh1jCiie0FYBoJjndcumcgE51+ux987Mfd9LrJTghmFVmF+TbtMOicUkyHFLA+Gyf\\\\\\\/auiRVRJbzvEQg6o88QSfdZWuXiWag2ViJQIAj1+1emQqQGlYeLnBmfXakahKtNOHHNcA4B2GtTXijcijlaOMcHDs34O2\\\\\\\/fHuGL2mxKG7FQA8g19AF1adjDJfSr\\\\\\\/ARmwBpfEnW7gxuJA5p6sd10IjiLco3KiFSTWXiTjMtHOUqtpCvGRDjG1hFK3IMEpx\\\\\\\/D\\\\\\\/dO1Y4BMMps7RgqUaTOnSCNfy1HGRVvnVTOpwMcChJ61SYcoYiX2J\\\\\\\/MMXd\\\\\\\/m+cUPvERxDVAtMIpTalfMj+ajriPWznDRDtIINWN8WQFsHm\\\\\\\/pjq2kYVYhtpsjReJsxncfuMpXm7PFA12ZQO+\\\\\\\/Ofs0sWomZIuYokIoapJ2vq7Ldf20nF+OuTeBsxKKHG7gfBp92Or8U95pg0zpdQDJcpmTpTZFXlpnx4xVKM9JSzcJ3kkc7HlvJPpQFObgRV4WmzhUTnTWNyYBYAf0CrBL7y8KMuwbPO6CJJqUSYm+8iN3U6w4XRCL4Vbr9fIbFO2ghKh0gtqQuC7tJI3K0XOcXNldF3tnE134u4CZ9rwdHfwWO4sZ+XHcgJwJ+x6LJY8+R5qIYDmyDseXMofElHq\\\\\\\/lBg0NyX9tDC6ifFtdEocW9eF+N8KZ6V3Yia0Gnu9ciaZFD9rtWjQEwXZUmEnj4V9cek8aiiM2c5h\\\\\\\/WVsv9SufrPJlIxIBlAtFMQJnJYUUns\\\\\\\/VlxVr1W64yGqEsXQ3mbJtynrkKj4\\\\\\\/dTyAxdtSjaCtp9uD60OY1AmKTUJ5e8C\\\\\\\/qGOabdkcnycL8gryqdq6r6\\\\\\\/cooRZEfCoX45M5wjrNPPHvdM+9w\\\\\\\/Tvl74zfNAcDdu2ExyCKy0WdukizvovZvpBoS+ER\\\\\\\/W0O8jqXNSvZISwb0nep1qUSVasSHV9DQD\\\\\\\/s4dSd+Z3gIa1hQFy1ko8MsLDVagac\\\\\\\/jO0kT\\\\\\\/7chK39jxZY6D5v5J5GehCDj+yzt4R6vsSKHidCAuxL66WJUF1eQCNHachXtOmfxENvWwZvL8dG5NlnbElQL\\\\\\\/epag8fDLJAG9bdtlSWUiVjvUcpab4EFX+gJ7wNUnofbRKyeTAUD3gCoNQwQ\\\\\\\/DsLzkwG7ExmkX7KO1flIKqwj5TmzesDw3CUwLRVXByX\\\\\\\/gDCOGxOSxQfxyNAMy7V3DkSXM0Z0klV29fn2PEObtOmBWwqKvXRnIu8fyrPEpgASup6aS6bQZgZDCz3XN65AuQ60Ui9n7ybjwIFLtCA+kP+PmziJVGaOwQZyf8LnRcpF1jQY4nVYuApTh0d\\\\\\\/xk5BuovXMwY6wQhkGTqWCmYvrwZDoGhqsMmy4Tste0YajPd6djP+kyIHCmpSfbW50UGv+RITdFo3PKs8u46I1uKUEpIGVHDjMU3yXLjLvCCtTFjPrno2wnSwW1ymyYOZLCUttc6hRmi5XH3vHnos4hSl+8TS09IxruGb7EZstE227\\\\\\\/0L4AT5uqGTP5GmTEFvHL4KKxwRc6fxo+W6XvX5BNeKjQMDFIbURiLKKexabgAxBbcyQnYMFfa3uPYieEgbtVLv9RN+XE9i0QqKk4p5VWmBhXS0YlpatQ\\\\\\\/F90qkxytxeOg5NDZRtQ8hEx995BN46RYhDK4poIX3iYRDF1zehZTIzE3WzZ9RGxiXfW7eqGoKyKTypUs\\\\\\\/mXgEjX7K4fI1Vf5Zc+uaW+YSwPr\\\\\\\/IrAnnlhVtu9x5AnAAsWUYyqrEqEWntTebt5hxcN5gqIcuHkfVesVo8pMHySKtiglz4wZnBlXxqToGgeFSg6Dd0zndZ5Oox4Qg8T1IL\\\\\\\/\\\\\\\/RDwZP18JiLikTAJd10bAM9EPV0Cx8\\\\\\\/tR4+WiYLX+cFXUF0JZIz3JkNBtcQfVagsfjOa7Isf4R67pMBnySobUUuxqN8juQuZw\\\\\\\/5sMBzkWwopr48N2Zluu1Z+T+yV5kUyl9R0HfeK1mM7WWYsr9ektRh+o12LJ0bZAQc3U85nH2Ljt76GRRHpaV5XHshTtTl3\\\\\\\/RwTSu9Gsviwr94\\\\\\\/OrGJhZcjUsXyDo3jeI7nKRqfz3SM7b8HuK4HR6KPfFpLzfGXNDWYD3gf7O1G89Li1hiyuzdZThh3Pgqb84p57BhJyDcNPDIEEgTKwrryvIrM1FaRIcqR2SqnUrhRDlQgwXV0iHleVCR4XZMejakWhHDaKeM5azNE2HAnrV\\\\\\\/K96DZCcPlTfQsJl8g6OhsIo\\\\\\\/Sq\\\\\\\/\\\\\\\/tqvLmnyBbpmnQhpDl118G9Ai5iV8V2DQ05bamIDj7EAFgR2Ez\\\\\\\/G2JketcPDdKC5cwZ5wR8SXCw3K6vritGTjaJJn8xvKf7Z0SIWwOZOqlMwoBSL1GwsFAb8rVzLqPg+DjsR3MQ5hq+y8Mn8NY5AaEgRsEydifvTJZ6buHxeKM\\\\\\\/a53X1IlIWPjKWs5IxtBhZWG\\\\\\\/0eX\\\\\\\/+bKE2t66kdeWDewJh\\\\\\\/cZDJPI0xWDd9QSypWi1TcM\\\\\\\/kzmHc4c13RvDHBQmHcKCwL7YyPDZD1mWLL\\\\\\\/mR0ZFaBCXIKNhzcJ3s8j1SzJ0t3HeMVFTtRiBb82pVeIr7c\\\\\\\/8fyGXjZ9EHy6c9mmTJnOz9uBQYVwayQ8flnXEDdzbtJy8MtuXMWQzpX7a1TgJTzM7j1EItgyTgP5MPcvyqOT5u5ttBUTEmHmfPmBl6yW5M0\\\\\\\/n0fH3d1uqTkEY4XWKzcZe8yoIHF97A9q39UdNvP4ZF3FHBTuHJU\\\\\\\/edvuDrfoAvE\\\\\\\/spRASysQYGqDue6rempRo6sTBpOzpXuFtpFVLhqxtStiXZ3Hnk\\\\\\\/8TZOMszbRvihlezvNcRQ1sRne6HbJVY2+dnIcLJ+K1YgjYTvsl9J1VqTMpQKVrz6xCU+Q\\\\\\\/LDetEnm\\\\\\\/o001BF1DcDAFQxhs7UQ30CBHBQzwVda0vSurXKQnNix\\\\\\\/uETemHMQRijtlmKinLQgFTxrQiMftMSKjdxWvbxvmOBxWZ7W+UOEaP3bWPX6VoHT4swR8rzzyS3f+i2\\\\\\\/NjjRtM5wNxYtx9nRBn9wiz+MSJotmyrq9mBQkZqGYDcg+hRyVuuHxKD6rc2c8CZZ+PFXTq3IR6INatwi36pMfkQW0TY0ULGuqQ2NjZS1oG8LTZymuNnlmoc2eI+QNLoVSAc4iv5lyNC57ZoZgi\\\\\\\/hOqrQPcxTU91d62LWx2jWMhRXE9h+lIw\\\\\\\/lQdCKSincx6SZungyoz1ZEm730bGdXRx2Na1zR+Ck03vOdeTpGa4+Sy0TDrem7GtGT+CtOJ9Cc4OzaKe1olZEGlSGkbW7cghn7Xa00zj3e6hcIIOV5wHv9US11DN2v8pzMAlKsvxUGTsm06lLZCNF45ZqkdAzvT1+wpGHn5LTha+57ii4GJtsBoq9oqJMVQPf4Of8wnNBDuXeiuIiTQVjRJUdrP5ncSp8x4rgA1FRZeClXiktj0Q0aSamcnQi5yTSDobGgFAutqskkUc7B5GNrSUoE9TrDxlFjucQ0\\\\\\\/J4WdPQUT6Ht5kXJyVo1u2SLQulpvasGXY8Z2CILtEpXmUH4Y+tcRMxEztw6bLNzkoqy2aosSpItGhoNaOM2fiNGtHBI7vYzwxEtolO\\\\\\\/ewhwRAb+tzKVnpphC3xtphrNsxRz0+Rwn7iOFP6hE9XZ\\\\\\\/CdGjewV\\\\\\\/88sTlMCm+svcB+25r48wcQhbrUshagQKIS50xwz8eTzTgQT\\\\\\\/aEa+bY+kZf9JkrhTNIWq0NDE18RUzFXpuP8PQtTshcUePxOXfE5qT6dz5rt0lGPCgF6hOIApX3KPclXFEljZI8eDMThnhs7E3vGTJtnx\\\\\\\/TbvVxtLkc7MkTCLYP5eyvqwDi3Eonh4A4lVHtJF4f4LAJGG7CI0ARRINmcJ9rfgyV7fl3xPCFhIlcgroWR2LkA9sK6bhFOlsM2uka4qNotY0aUX5iMQLf6LCv6hVlqXTf5uuN\\\\\\\/JuBDT59B5r6uGU1tdl45CoqgdZgVc+FdlCIIy+rLyj7C4DPu\\\\\\\/2kbb6+M+eQVyEoBVd2Q6vpuyjjEefuJl5MMZMMuwlR3c2mwN7Kjqu4IS0ZXEaO2fr0vGVjZd+0usd1ocPfIZsYnkBEneoHIsm2MK3S1luMVVljshgwxHxYjOw8CSBSlbYtVfwT+4Ns1TMMQdoE3CzcRZvb+AZQNVecx\\\\\\\/7qmzz\\\\\\\/\\\\\\\/78Rdkz97vqSjv2D3vWkFECJn7MAdpb4vxqntASrisyJz\\\\\\\/xlphv1deRCLbEI71OvlJX\\\\\\\/R\\\\\\\/I6vhyPPvIcgIZr1jX5TOMQPJCkjLCqql8D49+OgtznwbDIxVNTl5yWGy6IIM3Tfi5AE\\\\\\\/jRqin56YfKtP7epaHbxAMSXD0XkNjWTzPG8vKNoffiMc9NSSu9B3WPAVfM4vYm\\\\\\\/ljVfmyUmJvGNxdbBDEMEW1nRMVRFzSI3vKb2wOS4c2efRKfpGWtUrjRFciD8AlYtn36Lxw0I+tgJtiuh3yknK6YTA17o9S1u5gvxPqzrgdgTa6pnw8oDvkfCIdnSipl7TGTY2teRHbFaLLeKKY9BmLn\\\\\\\/4Kgh\\\\\\\/bKMZXfVruDHXVfUv7KOVcFzD6ruSLXXDLPWds2DOKwhbCXfAA0HymwKbBJXIYl3fNszOpmlCqAjWfC7Td+2dn4rRQP8ANMfkpA3h0lMmRRcu+cPqlrxZyjfA7s5dVZY7kUKxUI3vabsXwcrwkiZQr\\\\\\\/4P5Eipp66MIoT5+ViKa\\\\\\\/58Of5Qi71GRD9OjDm8P\\\\\\\/314P1e5c0dSeKyhJekbxW\\\\\\\/t1XmW7iPONBmZBzSbB2TwElRbA1jTagkkLkxVZOe6trWJMM4AArznVxwxCGG4cjeQEFQFvGUhInFjL2f7nwg2li53JZX1njWmm+sB79joC5Goj15ohAIJhmd2XGbf9Ap\\\\\\\/pDwPqV7+HDIa8t3Hq6R9GMarWFt6vJStSglo3UIIGm3Ok7vBDEPJGygkyHTCwgbca0pGGHLtefe+rLHdjfxOoMEq1oLNhdfEM49Go\\\\\\\/aI1RFHD1su+Eo\\\\\\\/L6NQu7WRDFzEkwswN2S\\\\\\\/bXn4i247lmiQxOtg2Qhky78E8ioHzG9RM5mI9yACShkbSIsspqOzykBzQRlUlcGoPzOGcL7Wc2Q7x1k945YCwUCrWOF6QpVJDO4ietWF\\\\\\\/tn7dB0fzebCUaS7GEcpTIgoLahAel0xZdRWnYKroewrTfuJTjIYR4pFYurGLTPPIhqhjo9MwPTQDn+FA+OzY4zCBZWAiz2MUJyeESmWCyiRzmhxfq+9WSCLFI1z1tGZTZJmenc5vrrV8vJqOTpYP5fF0IckU0Z+kNnmcgxIPi3q1t+w0aP7pHOjINQjDXGez7StV4scd0YzlVkKJ9vIRRm\\\\\\\/T1a\\\\\\\/k9d6W1k0\\\\\\\/Ag6nj+UFdzId70t1M0oOvMoCVPNo+0BAHnS4OGDuHztEVKz5FoV97vd03K7uuXj\\\\\\\/Fqtbsho8ZyUDCA8cUtymfPJ8dpVRJtXLog\\\\\\\/tgPhCj\\\\\\\/1jL3of8hYrkSWOYs+IvioIVYJXuwElh5bWrrBqCyolgLqx9obCrZ7tWL467z3g4ZR2W+Ga5cxYXT0I0XFZgfuNpBZsR\\\\\\\/X8qKCZZUUdUfxFsiruLOXCTKg1BZEacjejHbiQtCVJc25w26cEUoAevS1dfy+H3Ce7vGZCBCoOxakr4xmdPrBBZJ2+Z2ogTpg4qb0dzKzfUFxijNZwg7\\\\\\\/Y8B2n\\\\\\\/49GDs7gG0JZX09JEKmk1UG0m3n59srAS7zO9RAUdM6379zR6futqhFjzHywbsDMqfMzfWo8fJXynbWadBPcjSUTAwNb35lVRi3GGs9nLb563HjUxNwR3wkOCf0DJqaHeJV51kxxTtuBcziN4wzuEIHgilTaIrYNCXft30EwB25qLwZDBH9pD77U8CvH3XsEVs+B0E6N7Mx5DQPF\\\\\\\/OymypbymfEvqKGAmtwQGQ1rjh8rdj0ZQ0wio\\\\\\\/fPPpjuN5Nc6zPD5OmGo4JBqSZW+4Z9Hah3IHtmw+wXxkRaU\\\\\\\/YQAvqga0TxxWI2b\\\\\\\/WkZVi4ETr0x7NzrvPVuHbvOSIYQXFjJHezoyzFdVCd04L+1iDmdKdLcrrUh3wi+tWNMnHGsQ\\\\\\\/xVAQlUSIpdB1\\\\\\\/ieIrlnkt4sPX9Gx6NQ4WcnT4RN6YyrwhVhaW3EkFEnflxeH8FaD+0xQViTNbA5eAV4+F9+hUwZMK7INMzztrjuJ0gv0KQGSc1zdCZVxZ2qGvfckNjD\\\\\\\/NZZQbws1kDBGRCHP9o2f7uamD\\\\\\\/VAYzNmXjErxoku3rsEzUosoeB3\\\\\\\/2AheXwJQj8h3YJ9O53biW4E6ZREYymt80SshQOaEU7I5cJxG+KXK9TSw26Jmi+R4nNKE3MCzV2xAbQSx88\\\\\\\/eiUUd4vK3VcfHwXOy9Be9QGBGJj+jgqAvnLSCF19AI0cSwaq1Eiy1S7Ys6ZivsUjE7HhXHYHoCVSanvNIvqwg0b7J\\\\\\\/fE0rzqKjMIg5xyQ6e\\\\\\\/hzuWhfeBJ9+Tmn1Uns9BT\\\\\\\/WYJflKj8K9CRXuzJfLziKcZMKld6lFYNjgqTKQm7b7buBl2jldkjnaGpadtaO0DKVq+UPXLiUBvdCp+PQGEhQZkomIJaHyCruC41G0nOh2Ubd1su0iyclgWsYr35574cbghXMSU4\\\\\\\/umf+8i21WPaJoeNH8uPJq2B00cZu7m1R1QMFvSs9H0xKz8q+K76Tda7BrYaBUDajKfUwoe+YavPO7fEJ\\\\\\\/LGWz0b2JtAU4WsLcFJBPykx72jpozVJrbfuMF\\\\\\\/iJd5Jy7v1EDjynQIdJgsxuZPzFPo1bhdt\\\\\\\/7kJQqTx+TMZ+vSOeuKhLjwprkMS9qgef1rDiSHrdxUcN\\\\\\\/jMs0g3BsFSgKx3bXEGo8fmTHnaWRgumzH9nMjBsDtkT7NyxIiZono6FQq5NrEDWiWXRQEsOcHJzM0iwZ+psxtzKQYpv4Ezze7+dqVExWM5r++pXMeEzFTXSHu\\\\\\\/dG5r4IxJVAoAiVP5jB6PcDdmbUkwCb0wts7UtCBAAoXfLpfBjtXVo6z5cgEHBK3m2vYCNZjILUPQN\\\\\\\/+z841N5cC8SsqwCkYjYMNWfPALavD1qrIjU8L7GTsXlQrUdeQ\\\\\\\/Tiz524+c7xzToS3ehTxDPCOSBgHq1eu8qpYnlsmKeUedCg67DNW6rw\\\\\\\/Wyz\\\\\\\/JmTYuoX1ojRSeaL9H2BGBvf5HhM\\\\\\\/NQFPs2JtHPllWlLxczWI3b8\\\\\\\/vbdqFybce67LGw+tiALq41AHLPl7FzmT\\\\\\\/I38igP5qbwJbgdALc0Xw1b3ULLFQ5rB5bFPMtqU3RdFZHgF5z7r3RMX6\\\\\\\/GS3XhNjWBnIB0\\\\\\\/RQxUHVacT09OuAiM6sUrkFyINBeXvyff8tQ4h2f+11XkmmvUtuBOjxe\\\\\\\/HWOXxZ0vv+VDfhUzg3VSduuxA2JyMs3\\\\\\\/dofpK38q5vvw21pniXfR\\\\\\\/igmpi287jgL1JacmkKjUnNEKJuuU82Zt+gbjFE6ayHNQcET\\\\\\\/OP2NBVivzZcs7vt8IHI6W9q33vi0SzBy233jnCTgr7wsx+MtUu35QCNwB8LSl6B0KA+539LhtQwpXfDdaeZ4RRR+mZiSvZyJaQktGTWfFwjCFgmSWp7TMG5z654FfMvT833o5afpA8iwNxzMvD5UdOVgp9xRYH\\\\\\\/WNtLHxZ8YUXquMC50i2AVUBo4U1HYXO7YM776nvt59MWlW\\\\\\\/Q3JebW7\\\\\\\/MzJtRqtFJGCjQJs53AH8hxzXVvM35R3jCXZbkdYKJ1STmv9epyGTf5YJD3bm3INIcdvMsZ7Hi6Lc91fKb9xvENHUT23XPg\\\\\\\/lIZ7\\\\\\\/8\\\\\\\/+1TsoZN6z3qr4C8JotfumQX58n2bR3elh1Lf5HBj5UQeEIcT4SoYXyijatWI8xwW3WqyvCSLym1YZnxbc6Z0S0zPWiu0tHrpKwB1TC\\\\\\\/DB+eZ9ZHBSvosSAMuahZ4MnV0BCLING9YEI+2DANno\\\\\\\/N64c43Gc3+fXcQi00aPeE2Xv3EkOIjieVs693hqQR8jCVb+fKg8Ru+JdGHc8u75QJYXtoH1Nmc89sOIW4QEED30700C93bVuxc414J9Cl\\\\\\\/IsBoKz0zdfvbFV9L8lPUaNARgLkwuqmq7BcIuWLMafiVSsq1xjTGSjTRmZWrK\\\\\\\/nu0P+t1ZWsroWJiFa1aqY4Cc73JqCZAJOFiJn9ba6qKrBB7ZH+0g3W7ioiaWLm9eW2nyXe3naCqYDuwqvvbsEZ9F+PY5xhUkdw3oGZeoNKBqvs2pXgNSRNOBbSq2oAXosfaLvM26W7g2EA1mzM3ZHnW3BKhlwXweFDlqi2avSKfs0mA8g99XHmvbjQ7E3PVkIPponVfSrOrNSeznJ01cmgXRDrzLmG5RPkd2FBATWOZ1dbgkto9RZ2fn0i6eJ1CsK5aVO2nT1CXjRWT8PH9a3hGN\\\\\\\/efig7f0wropt2UUJg0Xl4kynAkqUKa\\\\\\\/D1DOy90fLQd41JPW0ijcpmQp54D4o+ScUn7QIeVXW3CPOTeepHOsD8WemaG5g69tzhshy9zdq3rmSyhS3a4PMc2x4Kbuj67cU9cZlDg\\\\\\\/VR2SGEofH9B+oEJTru58CcVI1b\\\\\\\/xKlrJbtCOmEsiDh+7agyTaChFB5pkndpvXbuC5ZJZLF8pp5ezfd2VEH88eMm5iPaGq\\\\\\\/+fzxOK2ZT96ldyPYlWM4TMsXAsWt6i65deW7x4UDnQdgsDZuJtp3zPJTb7\\\\\\\/I62zKPpL5G24XMocdxE48sBKdi2mMk+IxBEi0mjA5t5AivIh+U7HhA7m4pmT92TNlveceGMUj0hAD4+QFlmH4IDND6sTbpvHcwTdxz2ALrsZt4XZSjwDCJwbJMBpfWNZUJWDK2y6XcDO+sk15SNuE\\\\\\\/YQ3O4jcqDjYayRrppQef71b+zJ9C0jslR515q5Rki8NXfudCwgzN5l8XpsNYDRi6vKFSz53nlVUoob9QxXEw9mC+b+NUfg4ROG7vIhWGyraz12AF6w9vV41oyclvYCQvW2f3LUqenxiZo\\\\\\\/0RmEO0VQMkFVUfyVmEn+jblEQPP\\\\\\\/rV1QrynyOhs2ealiWEEOSDGpPzkVAoDT9JJFHY2S8jRsDvTUd3POU81SfghMf7pSpPphGZ0vlWYPQKhWIM\\\\\\\/k4PSx8Qp24hNhPCrdIxhktHdG8dQ8GYTKjzZMZwMgJk0htY8hJHF7M4Zig33Xf7+lz+hoiUJo6ESKu0EuzWG\\\\\\\/qmF4qjnrLRhaufejagGj0hACfYgS5hIi0UPq2tSLHDLW99GDkObTfnpiRn\\\\\\\/WN+14znLMD\\\\\\\/SCgY\\\\\\\/6QYhq+t1L53LvqBv3G3g1h2Q3cYIyrLi+KECXZDD3V7ayvUkqGPbwHeErD6AZzQY7umPpG2G8tjJdsC+kiEUJkeIu+qvqAjetnmIlBnL5HqdMSGslEAjrteAjJdunMPQfFiuPQYIhbe1HN5TBwg38oFkIb8MjyWJ68ileva9IhZdBawVIsTUBTgKO2+zNUNeru5VUKuzeNkPJRvL1Q4X6NHcJhtHSLsNlGqVJCY9gajVFJfgVvm62WE7EuIEAGbxS6WCYL1bYURSydrnfvli70wnq6e7SAdYw2PxnybnCAlTNv+rSQNWTq1mJ63YgFG9wTkBuA7\\\\\\\/DTHhLUfrHyc8PdKZMx226aNBDrjD7orjwNOjKeQ623r8LmyJtqqswmkS\\\\\\\/qlerWSLdHFhm8Mqud4pWCijIfrlc0Jmvjnhv5STB9uu+H+sp7zkV+GeizyoMmX2deB4nmajH+MxoimIRwm0ZO7ZY7zZPLpj7\\\\\\\/tbwG2jb37dj57v13k0QuTowSmdV2yce78VHUsDkQPJbCIxnzcrZnrdSMtHZpNjgx2typ\\\\\\\/yHDXhWvhboaj7vdEiLFba\\\\\\\/GomYlp56cxLB\\\\\\\/ka3igi+JzJRqpAxB5YIyQZxTwcGdXOqAo2qS+B4vRBkSOlW3TtEJCp1Tc862wqRpGEEZ40kPnU8dpf2e3eNIHh56pCMyxxmeMVcsJWcOLE5Qzp27LoW7fIoa\\\\\\\/j1IkVSgsz+VJxF9tsRvhPpOaSJgbgPNZVqFP+O1Qa2eDz1B4Dy0y7ydzhqioUV4VGfZUd+CO2OREnQhYI2t2\\\\\\\/D0BwwAdIXRfRtBEPi+OxZ83YCMdF9ebkbRbVSt3TR1YATWCFMZG6c5aaIUfoI3WV\\\\\\\/ll5EgYZDGKWEJj7CzV2Jsl+iurc8eKzxpIXLQ8S4krpLQGaGywLofOR7Dzh\\\\\\\/o0Ju07zLL5h2EEl66z5QlbGhsWBn\\\\\\\/ICYpfJZ7Lwh5kG94mM33l2eHhpEZOKWHTwzeyGuVlJr5kqe1lVrElbcyG6LqCjFeiyNtIBd6LUjklWV9wmqbHVyO+99ortXvLL9hzti8U97ktuQTdRoCDp1YNk7muDZeTLLZfqsj7da5w4HevHguaQWMfUdkYFsDAmUM4AgkkbaGsHE4SdICSktYC4VfXF4qyZ1c2t2HZ3j3q6yTW7VbKm9fubGD0KCzqpMIF482eVSCcrO+ap0DFunWCzRWsk3weREcfclomEpWZvTL2sLt5xXON3H2oH1zrF2xDcvNqrWXy9hr48b4FQk10SUKwrKHbQIgWsIDfyQy1yLQ7fxZTewulCUKtn20R40Bx6WQWtn\\\\\\\/FIjpIY2+Jtm30n2Z7d0NbBfj\\\\\\\/fOJOAff81SuPtPnglhxobttmzLt+6fxX5c36yVGC4SAp+G5E3c\\\\\\\/IfPhcyD8j5TMoZW+kD8dfFEOd296EE3BvUscykXzhj50975VbqsbmTyqmHFUUWbijg6KUUg7Hdj35V+hf+9zH1GE4RR1HU1XZ6GB2FuGhM0qclIjLgSszidlqIVhB+vpMVxwkluwlOjAKaZ1EU82D4GwtOSOX4hByDt9bsKY5fh9JBjjjMHKnns0kup8czdGv4rOIcE4V99jJv2jTxxKfX6oNqX7\\\\\\\/KyJWVDzd\\\\\\\/KkHffocHYCHnnmLs9fwpoNBm3M010uR1ArN7NRjrQGelCEr70oyHi6cnpQSUqsk38yPYd3t2u81crQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00224gWZ+txcfQT9jG3GJEjKsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002273d5edabf12a91ad2b2be22cea9ac9647e1d41586d6dc41d547dac38e16771e3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022r7NFY7ScmeZIjR9yEcMvCz7bwlB1GYfBdMTEL0vwH5oDK4NHjrl3ZFQd\\\\\\\/hfuEj8sbM5SVVvv2KfBGMRKJBapCJklzj3p\\\\\\\/81FRWL\\\\\\\/xbsYBMR6fOBKEOpY1a7XHzux0Wl1bauKkGvmRju8DgBJ+iuNWCdAyQK009h2sY1k7qnRiXXVA2o+\\\\\\\/FFBbJ5eIj6O1fuDJahavBYkuWsUoW8YSOPpg\\\\\\\/d8SjmHbtZwpsGf7r4H8WtWR1B4d0c7eWsGPUDKruS8FtGE9+k3T1Tc33xs\\\\\\\/b84PLs4EmEDNtMPtOV9vrG3z30jkj4y4gfWeMamc48Pod1rQBjq\\\\\\\/F31H7HtR\\\\\\\/XEPnnXdcvzfYUxbiqI5z3X\\\\\\\/sHDkvSHG3EP8PClLU2R3rqaghV8gasV0lIo0QMz2YxZBRiwqgROexkRCsLTlXBp74cSNrJe4e6NzS1zDirRKdz9NeaCIs+x+ccQQepGbyPmHeNXUD\\\\\\\/gmZ\\\\\\\/twilsp+bFUroP0qq9MsGe+EKL+HTLstk7zLds2Xm8+fxbu8GU8jfRdMneQK0RVk7Azd5oKyeVzayqWagnPACk4iAMw+BVORmAUNECZTDrTxD0GapfLumRXkkJwluWZH+xhMUdCsx\\\\\\\/qNecqQdbkDI+ppaqglCuYXyQCWDGGeFYDTSwrt2OwGhov6EXshGBmU7UgORAcM6\\\\\\\/U5Es6\\\\\\\/jbKPoKXaQq+Dn+7QF6C91wHz6EsT4pnf0ko3AgchjG+yxEQPNyLeUhauIrcDMogkm0KyGxu+4IpeMo01+OjmLue+Kt1RUlMrZXoSsQHJGO0zzV+DNAdACnMozMTPYRdYEMyhUetAVXbvhBEcBwJQKhQ\\\\\\\/LPBpGPPH8gwaRmzpFnABiHHG7xlMADLTzTaaDVGSWzAaAchISjsSguUFBmCtUyZUAGtZNpcuRrJPEkIHG4+wA61OZTFiV6wR5C2VI7bhgHuBYtdqneXCYCvMhLY4xOryFlDuZ6jAJRJUFkxlowoBmgGscrGGCYS8iMzdJ\\\\\\\/IPIMLLP\\\\\\\/ZUqdnQkWObQTSQKtIxrq5+ORQ4rlDHoV1m752PGpoT+SoalI1w\\\\\\\/4L37LP8L0c+T0ZeIzCdDSmD4xcSThW0Nda\\\\\\\/IXuuC9VZWPzlMwR5DC4gjQbRceMtiHY35Dr5RZ5lMFWD8h\\\\\\\/t6ilpmFaehNe0tz5x0lMgLh+P9uBUIcsmymHuj5lQbY+QVdlT94OC5ZRIr9SpEUDp4jR8UXoLkkiZjhkuQEN3d8EOoo4PWNWFwwcFLnOS5JOWMIBiEwjb87UbqPpvpp+tWKOX6wCBoSKNa2h53XvW8qVpTiHfsA1P0OBxJJrT+mGCVtqtFr2mrtxiTKWkaEcDrK0X\\\\\\\/qm5L2EW0jdpJgy\\\\\\\/jGgv+OuqoWzzIhCOl15JUcjdCKKHQuNlT6zOFQqDpL5aT6cFwiLwtT11+Gw+S\\\\\\\/H6HgJaY7fGy1I4hkv6UEoU0364rySKL80zdosEo8GXgUqquoP4J0oH7S2v1ubX5ekhbVTFbVIdUkN112E4zNiGNRQ8brXzRWO0XXClWQYStHz9zOe90A+tBuQ2gbjxweobHl3dKfymMPVhXdAG75vOGFpN5Bpr58gCuyRv92IP\\\\\\\/rWPIckP2G4e0wjypPUpOBcWvQaGBoKHm29kQP8p3Ac9+BSjLcI0wKN2fRcwSf8iyBlDh19cNU76qRUsh5kTExrQOvDUA1FPX0ogV2B1mq7dzhuzDoFdwjXsalucAy6hz34v76AS4IxBPzBofAEd5EQ18kTROd18b2U+Ity\\\\\\\/xm3gHPTT\\\\\\\/+a1NQo0dNQ+8QlZbO8FbhxxwAEUdFqkEMKyxSDJzTxk01lB\\\\\\\/ldh5HiVuZaw2iNSPdLD19T75pf4o77LiAKoB\\\\\\\/XnNJQMYmgmS5dU+PQsKu4JW4+8k9n3mBZ3D1wyeOrONT4TdHZO81Te9ur2W7xRLm20aHCzcmW5HcjB1yTSgp9eoVkR1czvMEfDM2kLdlMaSeFEWBvWwiyNQa1KIq+diNFED\\\\\\\/VF8Q0N6DPp1IaoONrBh+bwRdyA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LjfupbZ6M0F\\\\\\\/kFkBeswvtw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f0245046ece95ff86a5a92111c00601af04b9d2d7026594b4405f7fd2a0292f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490958 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3131" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2cWX712CenAZSs5frjI2LMRVZjupltzomP03n-5hd88" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3131" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DVaaAnpKcBQd3RzYAtQR5UjW6wq0F_SCLpdf2E9hXek" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3119" title="An Anopheles stephensi Promoter-Trap: Augmenting Genome Annotation and Functional Genomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3143" title="Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490958" id="node490958" data-pisa="ggg;8/10/3131" data-pisa-master="ggg;g3.118.200160" data-apath="/ggg/8/10/3131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Highly Contiguous Genome Assemblies of 15 <em>Drosophila</em> Species Generated Using Nanopore Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Cynthia Staber</span>, <span class="highwire-citation-author" data-delta="2">Julia Zeitlinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Scott Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3131-3141;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200160 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490958" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>â </span>MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danny.miller@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">danny.miller@seattlechildrens.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cynthia Staber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cynthia%2BStaber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Staber%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACynthia%2BStaber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Julia Zeitlinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Pathology, University of Kansas Medical Center, Kansas City, KS 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julia%2BZeitlinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeitlinger%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulia%2BZeitlinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">R. Scott Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BScott%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Scott Hawley</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3131" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490958" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490958/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3131.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490958" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490958/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3131.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490958" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490958/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3131.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490958" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490958/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The <em>Drosophila</em> genus is a unique group containing a wide range of species that occupy diverse ecosystems. In addition to the most widely studied species, <em>Drosophila melanogaster</em>, many other members in this genus also possess a well-developed set of genetic tools. Indeed, high-quality genomes exist for several species within the genus, facilitating studies of the function and evolution of <em>cis</em>-regulatory regions and proteins by allowing comparisons across at least 50 million years of evolution. Yet, the available genomes still fail to capture much of the substantial genetic diversity within the <em>Drosophila</em> genus. We have therefore tested protocols to rapidly and inexpensively sequence and assemble the genome from any Drosophila species using single-molecule sequencing technology from Oxford Nanopore. Here, we use this technology to present highly contiguous genome assemblies of 15 Drosophila species: 10 of the 12 originally sequenced Drosophila species (<em>ananassae</em>, <em>erecta</em>, <em>mojavensis</em>, <em>persimilis</em>, <em>pseudoobscura</em>, <em>sechellia</em>, <em>simulans</em>, <em>virilis</em>, <em>willistoni</em>, and <em>yakuba</em>), four additional species that had previously reported assemblies (<em>biarmipes</em>, <em>bipectinata</em>, <em>eugracilis</em>, and <em>mauritiana</em>), and one novel assembly (<em>triauraria</em>). Genomes were generated from an average of 29x depth-of-coverage data that after assembly resulted in an average contig N50 of 4.4 Mb. Subsequent alignment of contigs from the published reference genomes demonstrates that our assemblies could be used to close over 60% of the gaps present in the currently published reference genomes. Importantly, the materials and reagents cost for each genome was approximately $1,000 (USD). This study demonstrates the power and cost-effectiveness of long-read sequencing for genome assembly in Drosophila and provides a framework for the affordable sequencing and assembly of additional Drosophila genomes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-assembly" class="hw-term hw-article-keyword hw-article-keyword-genome-assembly" rel="nofollow">Genome assembly</a></li><li class="kwd"><a href="/keyword/drosophila" class="hw-term hw-article-keyword hw-article-keyword-drosophila" rel="nofollow">Drosophila</a></li><li class="kwd"><a href="/keyword/nanopore-sequencing" class="hw-term hw-article-keyword hw-article-keyword-nanopore-sequencing" rel="nofollow">Nanopore sequencing</a></li><li class="kwd"><a href="/keyword/third-generation-sequencing" class="hw-term hw-article-keyword hw-article-keyword-third-generation-sequencing" rel="nofollow">third-generation sequencing</a></li></ul><p id="p-4">The early availability of high-quality genome assemblies for 12 species of Drosophila fostered many studies in evolution and comparative genomics, reinforcing Drosophilaâs role as a primary model organism (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Drosophila 12 Genomes Consortium <em>et al.</em> 2007</a>). Since the publication of these sequences, improvements have been made to the original 12 genomes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hoskins <em>et al.</em> 2015</a>) and the genomes for several additional species have been reported (c. f. <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ometto <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nolte <em>et al.</em> 2013</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chiu <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allen <em>et al.</em> 2017</a>). Although the quality and value of these genomes is high, the cost and effort required to assemble new genomes remains prohibitive for many laboratories. These issues, as well as the difficulty of assembling repetitive or low-complexity regions using short-read technology alone, must be overcome before we can rapidly increase the number of sequenced species.</p><p id="p-5">Long-read, or third-generation, sequencing technology promises to simplify genome assembly by generating individual reads longer than many of the repetitive or low-complexity regions that have complicated genome assembly in the past (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chaisson <em>et al.</em> 2015</a>). Indeed, long-read data for the <em>Drosophila melanogaster</em> reference genome stock ISO-1 generated on a Pacific Biosciences RSII were released in 2014 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kim <em>et al.</em> 2014</a>). These data were used to assemble a high-quality <em>D. melanogaster</em> genome with a contig N50 of 21 Mb (contig N50 is a measure of genome continuity in which half of the genome is contained in overlapping DNA segments, or contigs, larger than the value given), demonstrating that long reads could be used to generate highly contiguous genome assemblies in Drosophila (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berlin <em>et al.</em> 2015</a>).</p><p id="p-6">Genome assemblies using Oxford Nanopore sequencing technology have been reported for several species, including <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Salazar <em>et al.</em> 2017</a>), <em>Arabidopsis thaliana</em> (Michael <em>et al.</em> 2017), <em>Caenorhabditis elegans</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tyson <em>et al.</em> 2017</a>), and <em>Homo sapiens</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jain <em>et al.</em> 2018</a>). This technology measures changes in current as a molecule (currently either DNA or RNA) passes through a small pore in a membrane. It has several advantages over other long-read technologies, including low DNA input requirements, ease of library preparation, and no theoretical limit to the length of a sequencing read. Although there are concerns about a high error rate with this technology, there are methods to mitigate this concern (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Walker <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Simpson <em>et al.</em> 2017</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Vaser <em>et al.</em> 2017</a>). For example, polishing using the original long-read data or data from another technology such as Illumina can correct SNP and indel errors by aligning reads to the assembled genome and identifying sites where modifications to the assembly result in resolution of a SNP or indel (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Walker <em>et al.</em> 2014</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Simpson <em>et al.</em> 2017</a>). The cost of Nanopore sequencing is also attractive for small to medium-sized genome assemblyâas of early 2018, up to 15 Gb of data could be generated on a single flow cell under ideal conditions for approximately $1,000 USD. Because of the advantages and the relatively low cost, we wondered if we could create a highly contigious, non-scaffolded genome assembly from reads generated using a single flow cell.</p><p id="p-7">Here, we report the sequencing and assembly of 15 non-melanogaster Drosophila species: ananassae, biarmipes, bipectinata, erecta, eugracilis, mauritiana, mojavensis, persimilis, pseudoobscura, sechellia, simulans, triauraria, virilis, willistoni, and yakuba (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Nanopore sequencing and assembly of D. melanogaster, including genome scaffolding using additional technologies, is presented in a co-submitted manuscript (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Solares <em>et al.</em> 2018</a>). These 15 species were sequenced to an average depth of coverage of 29x and an average read length of 5.9 kb. We rapidly assembled each genome using miniasm (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li 2016</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2018</a>), resulting in an average contig N50 of 4.4 Mb. Assemblies were polished using either Nanopore data alone, Illumina short-read data alone, or a combination of both. Polishing with both Nanopore and Illumina data resulted in genomes containing on average 97.7% of all single-copy genes expected to be present in metazoans, consistent with current Drosophila reference assemblies. In addition, for the 10 species included here as part of our analysis of the original 12 genomes project, 97.8% of transcripts either fully or partially aligned to our assembled contigs, only 1.7% fewer transcripts than aligned to the current published reference genomes.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic tree of flies sequenced in this report including two species (D. melanogaster and D. grimshawi) not sequenced here but that were part of the original 12 genomes project (Drosophila 12 Genomes Consortium et al. 2007). Adapted from Thomas and Hahn (2017)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1188536280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic tree of flies sequenced in this report including two species (&lt;em&gt;D. melanogaster&lt;/em&gt; and &lt;em&gt;D. grimshawi&lt;/em&gt;) not sequenced here but that were part of the original 12 genomes project (Drosophila 12 Genomes Consortium &lt;em&gt;et al.&lt;/em&gt; 2007). Adapted from Thomas and Hahn (2017).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3131/F1.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3131/F1.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491047" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Phylogenetic tree of flies sequenced in this report including two species (<em>D. melanogaster</em> and <em>D. grimshawi</em>) not sequenced here but that were part of the original 12 genomes project (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Drosophila 12 Genomes Consortium <em>et al.</em> 2007</a>). Adapted from <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Thomas and Hahn (2017)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Finally, after mapping contigs from currently published reference genomes to determine how many gaps could be closed using our highly contiguous assemblies, we estimate that an average of 61% of gaps in the currently published reference genomes could be resolved using our data. While each genome was highly contiguous, the overall materials and reagents cost was relatively low at approximately $1,000 USD per genome. This study demonstrates that it is feasible to generate a highly contiguous, low-cost genome for any member of the Drosophila genus and provides a framework by which additional Drosophila genomes may be assembled.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Stocks and reference genomes</h3><p id="p-10">All stocks used in this study (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) are available at the National Drosophila Species Stock Center (<a href="http://blogs.cornell.edu/drosophila/">http://blogs.cornell.edu/drosophila/</a>). Selected stocks were used either in the original 12 genomes project or for subsequent published genome assemblies. Originally, eight stocks were obtained from an in-house collection at the Stowers Institute (<em>ananassae</em>, <em>erecta</em>, <em>mojavensis</em>, <em>sechellia</em>, <em>simulans</em>, <em>virilis</em>, <em>pseudoobscura</em>, and <em>yakuba</em>) and eight stocks (<em>biarmipes</em>, <em>bipectinata</em>, <em>eugracilis</em>, <em>kikkawai</em>, <em>mauritiana</em>, <em>miranda</em>, <em>persimilis</em>, and <em>willistoni)</em> were obtained from the stock center when it was located at UCSD. Unfortunately, two of the eight stocks from the stock center (<em>kikkawai</em> and <em>miranda</em>) were found to be mislabeled. We were subsequently able to determine that the stock labeled <em>kikkawai</em> was in fact <em>triauraria</em>, but we were unable to determine what species the incorrectly labeled <em>miranda</em> stock was and thus removed it from our analysis. All flies were kept in standard cornmeal-agar bottles at 25Â°. The <em>D. mauritiana</em> reference genome was downloaded from <a href="http://www.popoolation.at/mauritiana_genome/index.html">http://www.popoolation.at/mauritiana_genome/index.html</a>, and other reference genomes were downloaded from either FlyBase (<a href="ftp://ftp.flybase.net/genomes/">ftp://ftp.flybase.net/genomes/</a>) or GenBank (<a href="https://www.ncbi.nlm.nih.gov/assembly">https://www.ncbi.nlm.nih.gov/assembly</a>) (Table S1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491134/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491134/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Stocks sequenced in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>DNA isolation and quantification</h3><p id="p-12">Either virgin or non-virgin females were collected under CO<sub>2</sub> and immediately frozen at â70Â° for at least 1 hr before DNA isolation. DNA was isolated using either the Qiagen Blood &amp; Cell Culture DNA Mini Kit or by a phenol protocol (Table S1). For the Blood &amp; Cell Culture kit (column-based isolation) 60â100 frozen females were placed in two 1.5-mL Eppendorf tubes and frozen in liquid nitrogen before being homogenized using a pestle in 250 Âµl of Buffer G2 with 200 Âµg/ml RNase A. 700 Âµl of Buffer G2 with RNase A, and 50 Âµl of 20 mg/mL proteinase K was then added to each tube, followed by incubation at 50Â° for 2 hr. Each tube was spun at 5k RPM for 5 min, then the supernatant was removed and placed in a new 1.5-mL Lo-Bind Eppendorf tube and vortexed for 10 sec. The supernatant from both tubes was then transferred onto the column and allowed to flow through via gravity. The column was washed 3x with wash buffer and eluted twice with 1 mL of elution buffer into 2 separate 1.5-mL Lo-Bind tubes. 700 Âµl of isopropanol was added and mixed via inversion before being spun at 14,000 RPM for 15 min at 4Â°. The supernatant was removed and the pellet was washed with freshly prepared 70% ethanol, then centrifuged at 14,000 RPM for 10 min at 4Â°. The supernatant was removed and 25 Âµl of nuclease-free water was added to each tube and allowed to sit at room temperature for 2 hr. Both tubes were then combined and stored at 4Â°.</p><p id="p-13">For phenol-based DNA extractions, 60â100 frozen flies were transferred to a 2-mL Kontes Dounce homogenizer (VWR #KT885300-0002) on ice in 1 mL homogenization buffer (0.1 M NaCL, 30 mM Tris-HCl pH 8.0, 10 mM EDTA, 0.5% Triton X-100). Flies were dounced 4â5x with each pestle (first the looser pestle A, then the tighter pestle B). Homogenate was transferred to a 1.5-mL Eppendorf tube on ice using a wide-bore pipet tip. Homogenizer was rinsed with 500 Î¼L homogenization buffer and combined with homogenate. Debris was pelleted by centrifugation for 1 min at 500x g at 4Â°.</p><p id="p-14">A wide-bore pipet tip was used to transfer supernatant containing nuclei in suspension to a clean tube. Nuclei were pelleted by centrifugation for 5 min at 2000x g at 4Â°. Supernatant was carefully decanted and the pelleted nuclei were resuspended in 200 Î¼L homogenization buffer by pipetting with a large-bore tip. Nuclei were transferred to a clean tube and lysed by adding 1.268 mL extraction buffer (0.1 M Tris-HCl pH 8.0, 0.1 M NaCl, 20 mM EDTA), 1.5 Î¼L proteinase K (20 mg/mL, Life Technologies AM2548), and 30 Î¼L 10% SDS. Lysis was aided by gently swirling and rocking, followed by incubation at 37Â° for 2â4 hr without agitation.</p><p id="p-15">Lysed nuclei were extracted twice with an equal volume of phenol/chloroform/isoamyl alcohol pH 8.0 (Amresco K169-100ML). Extraction was mixed gently on a rotator for 5 min then centrifuged 5 min at 5000x g at room temperature. The upper aqueous phase was transferred to a clean tube and the extraction repeated as above with chloroform (Sigma C2432). The final upper aqueous phase was transferred to a clean tube and precipitated by adding 0.1 volume 3M NaOAc (mixed gently) and 2 volumes absolute ethanol (mixed by gentle inversion). A wide-bore pipette tip was used to transfer the DNA (a white, stringy clump) to a clean tube. (Note: If high-quality low-retention tips are not available, it is recommended to Sigmacote the tips to prevent the gDNA from adhering to the plastic.) Excess ethanol was removed by pipetting. DNA was washed with 500 Î¼L 70% ethanol and centrifuged briefly at low speed with a tabletop centrifuge. Supernatant was removed by pipetting and DNA was allowed to air dry â¼10 min before resuspending in 100 Î¼L TE pH 8.0 at 4Â° overnight.</p></div><div id="sec-4" class="subsection"><h3>Nanopore library preparation, sequencing, and base calling</h3><p id="p-16">Libraries were prepared using the Ligation Sequencing Kit 1D (Oxford Nanopore) either according to or with slight modifications to the manufacturerâs protocol. To begin the prep an average of 2.3 Âµg of DNA was used, which is higher than the 400 ng recommended by the manufacturer. Water or TE was added to DNA for a total volume of 46 ÂµL. For 4 of 21 library preps (Table S1), the FFPE repair and dA-Tailing steps were combined in the following reaction mix: 46.5 ÂµL of genomic DNA in TE, 3.5 ÂµL of UltraII EP Buffer (NEB), 3.5 ÂµL of FFPE DNA Repair Buffer (NEB), 3 ÂµL of UltraII EP Enzyme (NEB), 3 ÂµL of FFPE Repair Mix (NEB), and 0.5 ÂµL of 100x NAD<sup>+</sup> (NEB). The combined reaction was prepared in a 200-ÂµL PCR tube and run at 20Â° for 1 hr followed by 65Â° for 30 min in a thermocycler. After cleanup and adapter ligation, 75 ÂµL of library (note that all 15 Âµl of adapter-ligated DNA, not the 12 ÂµL recommended by the manufacturer, was included in the final library), including Library Loading Beads, were loaded onto an R9.4 flow cell containing at least 800 active pores and run for 48â72 hr, or until no pores were available, on a MinION sequencer (Oxford Nanopore). Flow cells were restarted 3â6 times during a run in order to increase the number of pores in strand at any given time. Separate flow cells were used for each species. Nine species each utilized a single flow cell while two flow cells were used for the following six species: <em>D. virilis</em>, because of its large genome size; <em>D. simulans</em>, because of low read output on the first run; <em>D. bipectinata</em>, <em>D. erecta</em>, <em>D. eugracilis</em>, and <em>D. mojavensis</em>, because of a substandard library kit on the first run that produced fewer and shorter reads than expected (Table S1). Base calling was completed using Albacore Sequencing Pipeline Software version 2.1.0 (Oxford Nanopore) with default settings, and fastq files for either all reads or only those that passed the default quality filter (quality score â¥7) were combined for assembly and polishing (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, Table S2).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491103/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491103/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Base-called reads used for genome assembly. <sup><em><sup>1</sup></em></sup></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Illumina library preparation and sequencing</h3><p id="p-18">DNA for Illumina sequencing was prepared from 10 males using a Qiagen Blood and Tissue Kit (Table S3). Briefly, flies were frozen at â70Â° for at least 1 hr before DNA extraction following the manufacturerâs instructions. Genomic DNA was sonicated using the Covaris S220 and libraries were constructed on a Perkin Elmer Sciclone G3 NGS Workstation using the KAPA HTP Library Prep kit (KAPA Biosystems, Cat. No. KK8234) and NEXTflex DNA barcodes (Bioo Scientific, Cat No. NOVA-514104). Post-amplification size selection was performed on all libraries using a Pippin Prep (Sage Science). Resulting libraries were quantified using an Agilent 2100 Bioanalyzer plus an Invitrogen Qubit 2.0 Fluorometer and then pooled. Sequencing was performed on an Illumina NextSeq 500 instrument as 150 bp on a high-output, paired-end flow cell. Illumina NextSeq Real Time Analysis version 2.4.11 and bcl2fastq2 v2.18 were run to demultiplex sequencing reads and to generate FASTQ files.</p></div><div id="sec-6" class="subsection"><h3>Genome assembly, polishing, and quality evaluation</h3><p id="p-19">Minimap2 (Version 2.1-r311) (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li 2018</a>) and miniasm (version 0.2-r168-dirty) (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Li 2016</a>) were used to assemble either all reads or only those reads with quality scores â¥7 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Polishing was completed using either Racon (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Vaser <em>et al.</em> 2017</a>) or Pilon (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Walker <em>et al.</em> 2014</a>) alone or in combination (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Dot plots were generated with nucmer and mummerplot (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Delcher <em>et al.</em> 1999</a>); for clarity, only contigs or scaffolds &gt;100 kb were plotted. Assemblies using only reads that passed quality filter were subjected to either four iterations of polishing with Racon, six iterations of polishing with Pilon, or three iterations of polishing with Racon followed by three iterations of polishing with Pilon. QUAST-LG (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mikheenko <em>et al.</em> 2018</a>) was used to compare the miniasm assemblies with the published reference genomes and run on contigs &gt;10 kb with the fragmented and large options (Table S4). BUSCO version 2.0.1 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">SimÃ£o <em>et al.</em> 2015</a>) was used to evaluate assembly quality for all genomes using the metazoan_odb9 database, which contains 978 single-copy genes likely to be present in any metazoan genome (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, Tables S5âS8). For the 10 genomes assembled in this report that were part of the original Drosophila 12 genomes, transcripts were downloaded from FlyBase and aligned to each assembly and each reference genome using BLAST (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1997</a>). For each species and both genome types (assembly and reference), those transcripts for which at least 90% of the transcripts aligned to the genome with at least 95% identity were counted as proper alignments. To perform SNP calling, Illumina reads were aligned using bwa version 0.7.17-r1188 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li and Durbin 2009</a>) to assembled genomes that had been polished with Racon and Pilon. Samtools was then used to identify SNPs and indels. Only those SNPs and indels with quality scores &gt;220 were counted (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491054/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491054/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Assembly statistics for published assemblies using scaffold values and contig statistics of miniasm assemblies after polishing with Racon three times followed by Pilon three times</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491130/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491130/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">BUSCO scores<sup><em><sup>1</sup></em></sup> reveal assembly quality</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491123/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491123/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Number of single nucleotide and indel polymorphisms after polishing</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Alignment of reference contigs to Nanopore assemblies</h3><p id="p-21">Each reference genome was broken into contigs by separating scaffolds at every position that contained an N, and BLAST (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1997</a>) was used to align these to contigs from our Nanopore assembly using a custom script. We first aligned only reference contigs larger than 10 kb from reference scaffolds that contained no Nâs (that had no gaps) and counted how many of these reference contigs mapped with an identity &gt;99% to at least one assembled contig. We then aligned only reference contigs that came from reference scaffolds with two or more contigs (meaning the reference scaffold had at least one gap) one reference scaffold at a time. A gap was assumed to be filled if at least two reference contigs mapped to the same assembled contig.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-22">Illumina data generated for this project are available at the National Center for Biotechnology Information (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>) under project PRJNA427774, Nanopore reads that passed the default Albacore filter are available under project PRJNA471302. Illumina and Nanopore data for <em>D. triauraria</em> are available under project PRJNA473618. Scripts used in this project and genomes assembled in this project can be found on GitHub at <a href="https://github.com/danrdanny/Drosophila15GenomesProject/">https://github.com/danrdanny/Drosophila15GenomesProject/</a>. Original data underlying this manuscript can be accessed from the Stowers Original Data Repository at <a href="http://www.stowers.org/research/publications/libpb-1269">http://www.stowers.org/research/publications/libpb-1269</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6939758">https://doi.org/10.25387/g3.6939758</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Sequencing</h3><p id="p-23">Nanopore sequencing runs on a flow cell containing a maximum of 2,048 pores through which single-stranded DNA passes. These 2,048 pores are contained within 512 channels, with four pores per channel. Currently, only one pore in a channelâ512 total poresâare available for sequencing at a time. The sequencing control software, called MinKNOW, consigns each pore on a flow cell into one of four groups (often called mux groups) based on its performance level. Each mux group is run for a predetermined amount of time (8 hr by default), at which point sequencing switches to the next group of pores because pore quality deteriorates over time.</p><p id="p-24">For this study, we generally ran a sequencing reaction for 24 hr, allowing each of the pores in groups 1â3 to run for 8 hr (pores in group 4 are not used during a run). We then stopped and restarted the flow cell, allowing pores to be reorganized into new groups, which also allows pores originally assigned to the unused group 4 to be moved back into groups 1â3. We then monitored the flow cell and restarted again as the number of active pores decreased during a run, once again reorganizing pores into new groups. Because the number of active pores deteriorates over time and the amount of data output by a flow cell drops dramatically after 24 hr, there is an advantage to keeping as many pores as possible actively sequencing, or âin strand,â at a time.</p><p id="p-25">Preliminary testing in our lab revealed that using the manufacturer-recommended 400 ng of input DNA for a 1D library prep resulted in fewer than 50% of active pores in strand. This low number of active pores translated into low total data output by the flow cell. We found that starting a 1D library prep with 1â10 Âµg of DNA resulted in substantially more pores in strand during a sequencing run and gave higher data yields. We therefore performed all library preps in this study using 1â10 Âµg of starting material. Sequencing 15 Drosophila species in this manner yielded a total of 23 million sequence reads (Table S2).</p><p id="p-26">After base calling, these 23 million reads, with an average read length of 4,302 bp, yielded â¼99 billion bases sequenced at an average depth of coverage of 35x (Table S2). The base calling software, Albacore, separates reads into âpassâ and âfailâ bins by default, where any read with a quality score â¥7 is identified as a read that passed filter. The 76% of reads and 85% of total bases that passed filter had an average read length of 5,894 bp and an average depth of coverage of 29x, while those that did not pass filter had an average read length of 2,680 bp (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, Table S2).</p><p id="p-27">To determine whether variations in certain steps of the protocol might provide increased data output, we tested three different methods of DNA extraction and preparation. First, we compared column-based and phenol extraction (see Materials and Methods). While column-based methods are more convenient and safe, phenol extractions may reduce DNA shearing and loss. Second, in an attempt to reduce DNA shearing even further through reduced pipetting, we performed phenol extractions followed by a shortened library preparation protocol in which the FFPE repair and dA-tailing steps were combined. For reads passing filter, the average read length of samples prepared by a column-based method was 4,013 bp. Average read length sharply increased to 8,931 bp in phenol extractions that followed library prep instructions, while phenol-extracted samples that combined the FFPE repair and dA-tailing steps showed an average read length of 10,389 bp. These results suggest that combining these two library preparation steps is potentially useful, but the greatest increase in read length resulted from careful phenol-based DNA isolation.</p></div><div id="sec-11" class="subsection"><h3>Genome assembly and statistics</h3><p id="p-28"><em>De novo</em> assembly of 15 genomes can be computationally expensive since the software typically used to assemble long error-prone sequencing reads [for example, Canu (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Koren <em>et al.</em> 2017</a>) or hybrid assemblers such as DBG2OLC (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ye <em>et al.</em> 2016</a>), which uses both short high-quality reads and long error-prone reads] requires a large number of CPU hours to complete with default settings. As an example, for assembly of the <em>D. melanogaster</em> ISO-1 genome using only reads larger than 1 kb reported in <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Solares <em>et al.</em> (2018)</a>, Canu took longer than 1 week on a 32-core server with 1 terabyte of memoryâor at least 5,376 CPU hours. This may be cost-prohibitive for some labs. Alternatively, mapping and assembly using miniasm (a <em>de novo</em> assembler for long error-prone reads) has been shown to be highly efficient and less time-intensive (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Li 2016</a>; 2017).</p><p id="p-29">A study comparing assembly quality of the <em>D. melanogaster</em> ISO-1 reference genome using different assembly approaches found that assemblies using miniasm alone gave larger contigs (N50 3.9 Mb) than Canu alone (N50 3.0 Mb) but smaller contigs than DBG2OLC alone (N50 9.9 Mb) or a merged Canu and DBG2OLC assembly (N50 18.6 Mb) (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Solares <em>et al.</em> 2018</a>). This suggests that statistics from miniasm assemblies (as assayed by N50 value aloneâa measure of genome contiguity) may be comparable to some more computationally intensive long-read assembly software. We therefore chose to perform assembly using only miniasm. Each of the 15 assemblies was completed in under 1 hr using 32 CPUs on a computer with 1 Tb of RAM available (less than 32 CPU hours). While assembly statistics such as N50 are similar between miniasm assemblies and Canu assemblies, it is important to note that the quality of miniasm assemblies are much lower because Canu contains an error-correcting step (discussed below). The addition of an equivalent error-correction step to our miniasm assemblies did increase overall assembly and analysis time. Specifically, we find that one iteration of polishing with Racon takes approximately 32 CPU hours to complete, while one round of polishing with Pilon takes approximately 64 CPU hours to complete. In total, this adds approximately 288 CPU hours to a miniasm assembly, or approximately 10-fold less than the 5,000-plus CPU hours required for Canu assemblies of similar datasets.</p><p id="p-30">For each species, sequenced reads were combined into two fastq files, one containing all reads and one containing only those reads that passed filter. Separate genome assemblies were then completed for each category of reads and compared. Assemblies using only reads that passed filter had larger contig N50 values in 9 of 15 cases and fewer overall contigs in 10 of 15 cases than assemblies completed using all sequencing reads (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, Table S5). Because of the moderately improved N50 values and lower contig numbers, we decided to proceed with our analysis using only assemblies generated by higher quality reads. Assembly quality varied greatly among the 15 genomes, with an average contig N50 of 4.4 Mb, a maximum of 16.6 Mb (<em>D. erecta</em>), and a minimum of 0.6 Mb (<em>D. bipectinata</em>). For each species, assembly resulted in genome sizes that were smaller than the expected genome size (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bosco <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gregory and Johnston 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hjelmen and Johnston 2017</a>), with repetitive sequence likely accounting for the lower values than expected. We also compared assembly statistics for each of our <em>de novo</em> assemblies to the published assembly of each species and found that in all cases our contig N50 values were higher than those of the published assemblies but lower than the scaffold N50 values (scaffold N50 is a measure similar to contig N50, in which half of the genome is contained in linked DNA segments, or scaffolds, larger than the value given) in all but three cases (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Published assemblies contain both contigs (contiguous DNA segments) and scaffolds (one or more contigs generally separated by gaps where the DNA sequence is unclear), while the assemblies presented here contain only contigs.</p><p id="p-31">We next generated dot plots to better understand how our assemblies compare to the reference genome for the 10 species sequenced as part of the original Drosophila 12 genomes project (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Drosophila 12 Genomes Consortium <em>et al.</em> 2007</a>). Although overall our assemblies correlated well with the published reference assemblies, we did observe a few inversions, rearrangements, or duplications in each plot (Figure S1). For example, a small inversion is seen between the <em>D. yakuba</em> reference chromosome <em>2L</em> and our assembly contig utg000002l, similar to other large inversions that have been reported to be segregating in this population (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Llopart <em>et al.</em> 2002</a>). Indeed, large rearrangements and inversions are observed in several of the assemblies. It is possible that these represent changes that occurred in the stock after being initially isolated for genome sequencing, they may be due to assembly errors in the initial assemblies as they were generated using relatively low-coverage data, or the stocks we sequenced, while labeled as such, may not have been the exact stock used for sequencing by the 12 genomes consortium. We also observe differences in repetitive, poorly assembled regions of the genome. For example, the X-axis of the <em>D. sechellia</em> and <em>D. persimilis</em> plots are longer than the Y-axis, likely because our assemblies have collapsed repetitive regions that the published reference genomes have not. To further evaluate these differences in dot plots, we used QUAST-LG to compare our miniasm assemblies with their published assemblies using (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Mikheenko <em>et al.</em> 2018</a>). We find that on average, our assemblies capture 83% of the sequence present in the currently published reference genomes (defined as the total number of aligned bases divided by the genome size) (Table S4).</p></div><div id="sec-12" class="subsection"><h3>Assembly polishing and evaluation</h3><p id="p-32">While the N50 statistic has utility as a measure of genome contiguity, it is not necessarily a good indicator of genome quality. A highly contiguous genome may in fact contain many errors that make it difficult to use for downstream analysis such as gene finding or SNP calling. We did not expect our Nanopore-only assemblies to be of high quality because raw Nanopore reads have an observed error rate of 10â15% (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Li 2016</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Simpson <em>et al.</em> 2017</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Salazar <em>et al.</em> 2017</a>). It is, however, possible to improve assembly quality through one or more rounds of polishing, which is a process that improves assembly quality using the original sequencing reads, reads from another technology with a relatively low error rate (<em>e.g.</em>, Illumina data), or a combination of both. We therefore sought to evaluate both the quality of our initial miniasm assemblies as well as our assemblies using various combinations of polishing techniques.</p><p id="p-33">Assembly quality was evaluated using BUSCO, which searches assemblies for highly conserved genes generally present in a single copy in any genome (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">SimÃ£o <em>et al.</em> 2015</a>). For example, BUSCO analysis of published reference genomes for the 15 species sequenced in this study revealed that at least 96.6% of 978 highly conserved genes were present in at least one copy in each genome (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, Table S5). We ran BUSCO on all 15 of our miniasm assemblies with no polishing and found an average of 1.6% (min: 0.4%, max: 3.8%) of genes were present in at least one copy, suggesting that our initial assemblies, although highly contiguous, were indeed of poor quality. This is likely due to the high error rate of the reads used to generate the assemblies, resulting in frame shifts leading to premature stop codons within open reading frames.</p><p id="p-34">We then performed multiple iterations of polishing using either Racon or Pilon (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Walker <em>et al.</em> 2014</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Vaser <em>et al.</em> 2017</a>). Racon, which uses only base-called Nanopore reads for polishing, improved average BUSCO scores from 1.6 to 85.2% after a single iteration (min: 74.9%, max: 91.3%). Repeat iterations of Racon alone improved scores only slightly, reaching an average of 88.9% after four iterations (min: 80.1%, max: 95.6%) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>, Table S6). Pilon alone, which uses only Illumina data for polishing, improved average BUSCO scores from 1.6 to 72.6% after a single iteration (min: 62.1%, max: 79.5%), and to 93.0% after six iterations (min: 87.9%, max: 95.5%) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>, <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>, Table S7). While BUSCO scores tended to increase with more iterations, it is important to note that BUSCO scores fell for 7 of 15 assemblies between the 3<sup>rd</sup> and 4<sup>th</sup> iterations of Racon, suggesting that repeat iterations of polishing may have negative impacts on assembly quality. Alignment of Illumina data to Racon-polished genomes demonstrated that polishing using Racon alone failed to completely correct poly-N indels. Furthermore, while Pilon was run for 6 consecutive iterations, there was not significant improvement in scores after 3 iterations, demonstrating a limit to polishing using a single software application.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polishing improves assembly quality. Average BUSCO score for these 15 assembled genomes was 1.6% before polishing. The dotted line in all panels represents the average BUSCO score for all 14 published reference genomes (Table 4). Full polishing results can be found in Tables S5&#x2013;S8. (A) Complete BUSCO scores for four iterations of Racon alone. (B) Complete BUSCO scores for six iterations of Pilon alone. (C) Complete BUSCO scores shown for three iterations of Racon followed by three iterations of Pilon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1188536280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Polishing improves assembly quality. Average BUSCO score for these 15 assembled genomes was 1.6% before polishing. The dotted line in all panels represents the average BUSCO score for all 14 published reference genomes (Table 4). Full polishing results can be found in Tables S5&#x2013;S8. (A) Complete BUSCO scores for four iterations of Racon alone. (B) Complete BUSCO scores for six iterations of Pilon alone. (C) Complete BUSCO scores shown for three iterations of Racon followed by three iterations of Pilon.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3131/F2.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3131/F2.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490982" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Polishing improves assembly quality. Average BUSCO score for these 15 assembled genomes was 1.6% before polishing. The dotted line in all panels represents the average BUSCO score for all 14 published reference genomes (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>). Full polishing results can be found in Tables S5âS8. (A) Complete BUSCO scores for four iterations of Racon alone. (B) Complete BUSCO scores for six iterations of Pilon alone. (C) Complete BUSCO scores shown for three iterations of Racon followed by three iterations of Pilon.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Because both polishing techniques alone failed to achieve BUSCO scores equal to or better than the published reference genomes, we then polished using a combination of both Racon and Pilon. We first attempted to run Pilon and Racon in combination, one after the other (<em>e.g.</em>, Racon, Pilon, Racon, Pilon, etc.), but found that while BUSCO scores improved with each iteration of Pilon, they then fell with each iteration of Racon (Table S8). Because BUSCO scores plateau around 3 iterations of Racon or Pilon (see above), we wondered if combining the two approaches in tandem would result in improved assembly quality. We therefore ran three iterations of Racon followed by three iterations of Pilon and found a sustained improvement in BUSCO scores, reaching an average of 97.7% (min: 93.8%, max: 98.7%) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>, <a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>, Table S9). Importantly, this combination of polishing resulted in BUSCO scores consistent with the currently published reference genomes for 13 of the 14 previously sequenced species. Finally, to determine if currently available gene models properly mapped to our assemblies, we mapped transcripts for the 10 assemblies that are available on FlyBase to both our assembled genomes and the currently published reference genomes. We found that on average 97.8% of all transcripts mapped to our assembled genomes, while 99.6% mapped to the published reference genome (Table S10). This suggests that while the BUSCO scores of our genomes suggest similarity between our assemblies and the reference, a small number of genes that are not considered highly conserved by BUSCO may have been improperly assembled using our approach.</p><p id="p-37">To determine if high levels of heterozygosity of the stocks used for sequencing explained the lower BUSCO scores observed in the <em>D. bipectinata</em> and <em>D. triauraria</em> stocks (average of 93.9%), we aligned Illumina data from each stock to the assembly generated from three iterations of Racon and Pilon and called SNPs and indels using SAMtools (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li <em>et al.</em> 2009</a>). On average, we observed 63,702 indels, and 769,893 heterozygous SNPs in these two stocks, compared to an average of 13,365 indels and 116,445 heterozygous SNPs in the remaining 13 stocks. This high number of SNP and indel polymorphisms reveals a high level of heterozygosity in the stocks with low BUSCO scores. However, we also observed a high number of SNP and indels in both the <em>D. eugracilis</em> and <em>D. willistoni</em> stocks, two assemblies with BUSCO scores near 98%, suggesting that heterozygosity for SNPs and indels alone does not fully explain the lower quality scores observed in <em>D. bipectinata</em> and <em>D. triauraria</em>.</p><p id="p-38">These observations led us to ask why a large number of heterozygous SNPs were observed in four of our lines. Because no effort was made to isogenize these stocks before sequencing, it is possible that the high number of SNPs and indels in these lines is simply a consequence of selection for heterozygosity in these lines. However, this hypothesis would predict high levels of heterozygosity in the other sequenced stocks, which is not observed. Another possibility is that that these four lines were crossed intentionally with another stock in order to increase the fitness of the stock, or they were contaminated. Either scenario would have introduced the large amount of heterozygosity we observe.</p><p id="p-39">Finally, we wondered if polishing would merge or separate contigs and thus affect assembly statistics. We found that polishing with each program independently did, on average, increase the total assembly size and contig N50 of our 15 assemblies (Table S11). Furthermore, the total number of contigs decreased for 6 of 15 species and increased for none after polishing with Racon, while total contig number remained unchanged after polishing with Pilon. Overall, this suggests that polishing using Racon or Pilon had minimal impact on assembly statistics, but clearly improved assembly quality as determined by BUSCO scores.</p></div><div id="sec-13" class="subsection"><h3>Long-read data can close gaps in reference genomes</h3><p id="p-40">Other long-read sequencing technologies, such as PacBio, have previously been used to improve existing reference genomes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jiao <em>et al.</em> 2017</a>), and visual inspection of dot plots used to compare our assemblies with the reference genome suggest that our contigs could be used to link large reference scaffolds together (Figure S1). To test if our highly contiguous genomes could be used to fill gaps in the reference assemblies, we broke each reference genome into individual contigs and aligned those contigs to our assembled genomes. We performed two types of alignment. First, we identified reference contigs larger than 10 kb that contained no Nâs, or unknown sequence (these contigs are generally labeled as scaffolds in many reference genomes, which may cause confusion) and identified how many of those could be placed on our assemblies with &gt;99% identity. In doing so, we found that each reference genome contained an average of 205 singleton contigs larger than 10 kb, and that on average 95% of these could be placed on a larger contig in our assemblies (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). (<em>D. mauritiana</em> was excluded from this analysis because its genome assembly is based on release 5 of the <em>D. melanogaster</em> genome and thus contains only one contig larger than 10 kb with no gaps.)</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491026/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491026/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Number of singleton reference contigs that could be placed on the contigs assembled in this study and the number of gaps closed between reference contigs on scaffolds with one or more gaps</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">We then identified scaffolds from each reference genome that contained two or more contigs separated by Nâs and found that the 14 reference genomes contain an average of 1,284 scaffolds with two or more contigs (min: 15, max: 2,305). For each reference scaffold, individual reference contigs were mapped to the assembled genome and the gap between mapped reference contigs was determined. A single gap was considered closed when reference contigs from the same reference scaffold were mapped to a single assembled contig. This mapping revealed that, on average, our assembled contigs closed 3,854 of 5,770, or 61%, of gaps in each of the reference genomes (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>).</p><p id="p-42">A specific example of how long, contiguous assemblies may assist with reference genome correction is scaffold_4845 in the <em>D. erecta</em> reference assembly. This 22.6-Mb reference scaffold contains 75 contigs and 112,933 Nâs. We broke this scaffold into its 75 contigs (numbered 0â74) and were able to place 39 of these reference contigs (numbered 23â61) onto contig utg000010l, a single 17.4-Mb contig from our assembly. These 39 contigs were placed in the same order as they appear in the reference assembly (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), but with important exceptions. Specifically, three reference contigs were placed within larger reference contigs. For example, contig 53 is a 79,102-bp contig that was placed in the middle of contig 52, a 339,371-bp contig. Similarly, contig 39 was placed within 40, and contig 60 was placed within 61. All six contigs have &gt;99% identity to the positions that they were aligned to, suggesting either that the reference assembly incorrectly identified these contigs as unique segments of the genome or that our assembly collapsed nearby regions of the genome with high identity into single segments.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gaps in the reference genome assembly can be closed using long-read data. This example shows that a 17.4-Mb contig (utg000010l) from our assembly (bottom) closes the gaps (top, gray lines) among 38 contigs (top, shaded boxes) from the D. erecta reference scaffold (scaffold_4845), potentially resolving 3.7 Mb of sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1188536280" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gaps in the reference genome assembly can be closed using long-read data. This example shows that a 17.4-Mb contig (utg000010l) from our assembly (bottom) closes the gaps (top, gray lines) among 38 contigs (top, shaded boxes) from the &lt;em&gt;D. erecta&lt;/em&gt; reference scaffold (scaffold_4845), potentially resolving 3.7 Mb of sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3131/F3.medium.gif" width="440" height="54"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3131/F3.medium.gif" width="440" height="54"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3131/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491074" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-43" class="first-child">Gaps in the reference genome assembly can be closed using long-read data. This example shows that a 17.4-Mb contig (utg000010l) from our assembly (bottom) closes the gaps (top, gray lines) among 38 contigs (top, shaded boxes) from the <em>D. erecta</em> reference scaffold (scaffold_4845), potentially resolving 3.7 Mb of sequence.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Sequencing costs and conclusions</h3><p id="p-44">A major goal of this project was to determine if it was possible to create low-cost, highly contiguous genome assemblies of non-<em>melanogaster</em> Drosophila species using Nanopore sequencing. We are able to estimate an expected cost for each of our Drosophila genomes using publicly available materials and reagent prices from early 2018. Here, we used approximately one flow cell per species; when purchased individually, a single flow cell is $900 (USD) and when purchased as a pack of 48, a single flow cell is $500 (USD). A 1D library kit is $599 (USD) and will make six libraries, at a cost of $100 per genome. Reagents include FFPE enzyme, dA-tailing enzyme, ligase, and magnetic beads for cleanup. To simplify, we assume $50 in reagent and other costs per genome. Therefore, for Nanopore sequencing alone, the costs for materials and reagents range from $650â$1,050 depending on the flow cell cost. We also generated Illumina 150-bp paired-end data for polishing at an average of 64x depth of coverage (Table S3). While short read sequencing costs vary widely, we assume a cost of $250 per Drosophila genome is a reasonable estimate. Therefore, we estimate that overall sequencing and assembly costs for a highly contiguous Drosophila reference genome to range from $900â$1,300 (USD).</p><p id="p-45">In summary, we have generated highly contiguous genome assemblies for 15 species of Drosophila at a materials and reagents cost of approximately $1000 (USD). Relatively low-coverage Nanopore data resulted in genome assemblies with an average contig N50 value of 4.4 Mb, and polishing with Racon and Pilon resulted in average BUSCO scores of 97.7%, a score comparable to currently published reference genomes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Table 4</a>). That we were able to generate 15 different genome assemblies at relatively low cost and in a short period of time suggests a new approach researchers may take when studying other species within the <em>Drosophila</em> genus and suggests that genome assembly using a long-read technology should now be considered the standard for studies of new Drosophila species.</p><p id="p-46">It is now also feasible to consider what effort, if any, should be put forth in attempts to assemble and analyze a substantially larger number of genomes from across the entire <em>Drosophila</em> genus. The <em>Drosophila</em> genus represents at least 50 million years of evolution (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tamura <em>et al.</em> 2004</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Obbard <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">OâGrady and DeSalle 2018</a>) and spans a broad range of ecosystems, with the unique advantage of having a highly developed genetic toolkit available for many of its members (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ashburner <em>et al.</em> 2005</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gratz <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Perkins <em>et al.</em> 2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stern <em>et al.</em> 2017</a>). Low-cost, highly contiguous genome assemblies of hundreds of well-described species would foster studies of genome conservation and evolution orders of magnitude more detailed than any before and on a scale not possible in other species groups. Such work would provide a solid foundation to ensure the next 100 years of Drosophila research are as fruitful as the first 100 years have been.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank A. Bernardo Carvalho for identifying that the <em>D. kikkawai</em> stock originally included in this report was incorrectly labeled, Amir Yassin for identifying the likely source of the mix-up and for providing information on species identification in the <em>montium</em> species group, and Patrick OâGrady for confirming that the stock is indeed <em>D. triauraria</em>. We also thank Justin Blumenstiel, Rob Unckless and members of the Bachtrog laboratory and Hawley laboratory for helpful comments, discussions, and suggestions, two anonymous reviewers for helpful comments and suggestions, as well as Angela Miller for assistance with editing and figure preparation. DEM has received support in the form of free flow cells from Oxford Nanopore as part of the Oxford Nanopore Educational Initiative. RSH is an American Cancer Society Research Professor.</p><p id="p-48"><em>Note added in proof:</em> See <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Solares <em>et al</em>. 2018</a> (pp. <a href="https://doi.org/10.1534/g3.118.200162">3143â3154</a>) in this issue, for a related work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-49">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6939758">https://doi.org/10.25387/g3.6939758</a>.</p></li><li class="fn" id="fn-6"><p id="p-50"><em>Communicating editor: S. Celniker</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-02-16"><span class="received-label">Received </span>February 16, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-30"><span class="accepted-label">Accepted </span>July 30, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2018 Miller <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.1" data-doi="10.1126/science.287.5461.2185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gocayne</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">The genome sequence of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2185</span>â<span class="cit-lpage">2195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.287.5461.2185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAdams%26rft.auinit1%253DM.%2BD.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2185%26rft.epage%253D2195%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2185%26rft_id%253Dinfo%253Apmid%252F10731132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.2" data-doi="10.1534/g3.116.037598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kopp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenoweth</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Single-Molecule Sequencing of the <em>Drosophila serrata</em> Genome.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">781</span>â<span class="cit-lpage">788</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.037598</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.037598%26rft_id%253Dinfo%253Apmid%252F28143951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMy83ODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.3" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">SchÃ¤ffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>â<span class="cit-lpage">3402</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.17.3389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-book" id="cit-8.10.3131.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golic</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-source">Drosophila: A Laboratory Handbook</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.5" data-doi="10.1038/nbt.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">C.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landolin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">623</span>â<span class="cit-lpage">630</span>. <span class="cit-comment">[Corrigendum. 2015. Nat. Biotechnol. 33: 1109.]</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3238%26rft_id%253Dinfo%253Apmid%252F26006009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26006009&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.6" data-doi="10.1534/genetics.107.075069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bosco</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leiva-Neto</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markow</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Analysis of Drosophila Species Genome Size and Satellite DNA Content Reveals Significant Differences Among Strains as Well as Between Species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1277</span>â<span class="cit-lpage">1290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.075069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBosco%26rft.auinit1%253DG.%26rft.volume%253D177%26rft.issue%253D3%26rft.spage%253D1277%26rft.epage%253D1290%26rft.atitle%253DAnalysis%2Bof%2BDrosophila%2BSpecies%2BGenome%2BSize%2Band%2BSatellite%2BDNA%2BContent%2BReveals%2BSignificant%2BDifferences%2BAmong%2BStrains%2Bas%2BWell%2Bas%2BBetween%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075069%26rft_id%253Dinfo%253Apmid%252F18039867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8zLzEyNzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M. J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Genetic variation and the de novo assembly of human genomes.</span> <abbr class="cit-jnl-abbrev">Nature Publishing Group</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">627</span>â<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BPublishing%2BGroup%26rft.volume%253D16%26rft.spage%253D627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.8" data-doi="10.1534/g3.113.008185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamm</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cridland</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genome of <em>Drosophila suzukii</em>, the spotted wing drosophila.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2257</span>â<span class="cit-lpage">2271</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008185</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008185%26rft_id%253Dinfo%253Apmid%252F24142924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTIvMjI1NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzEzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.9" data-doi="10.1093/nar/27.11.2369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasif</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleischmann</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Alignment of whole genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2369</span>â<span class="cit-lpage">2376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/27.11.2369</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.11.2369%26rft_id%253Dinfo%253Apmid%252F10325427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.11.2369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10325427&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080678100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.10" data-doi="10.1038/nature06341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Drosophila 12 Genomes Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Evolution of genes and genomes on the Drosophila phylogeny.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>: <span class="cit-fpage">203</span>â<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DClark%26rft.auinit1%253DA.%2BG.%26rft.volume%253D450%26rft.issue%253D7167%26rft.spage%253D203%26rft.epage%253D218%26rft.atitle%253DEvolution%2Bof%2Bgenes%2Band%2Bgenomes%2Bon%2Bthe%2BDrosophila%2Bphylogeny.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06341%26rft_id%253Dinfo%253Apmid%252F17994087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17994087&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250746200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.11" data-doi="10.1534/genetics.113.152710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gratz</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cummings</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamm</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donohue</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genome Engineering of Drosophila with the CRISPR RNA-Guided Cas9 Nuclease.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">1029</span>â<span class="cit-lpage">1035</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152710</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152710%26rft_id%253Dinfo%253Apmid%252F23709638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NC80LzEwMjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.12" data-doi="10.1038/hdy.2008.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genome size diversity in the family Drosophilidae.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">228</span>â<span class="cit-lpage">238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2008.49</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DGregory%26rft.auinit1%253DT.%2BR.%26rft.volume%253D101%26rft.issue%253D3%26rft.spage%253D228%26rft.epage%253D238%26rft.atitle%253DGenome%2Bsize%2Bdiversity%2Bin%2Bthe%2Bfamily%2BDrosophilidae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2008.49%26rft_id%253Dinfo%253Apmid%252F18523443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2008.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18523443&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258598400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-book" id="cit-8.10.3131.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hjelmen</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">The mode and tempo of genome size evolution in the subgenus Sophophora.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">12</span>: <span class="cit-fpage">e0173505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0173505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.14" data-doi="10.1101/gr.185579.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoskins</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mendez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">The Release 6 reference sequence of the <em>Drosophila melanogaster</em> genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">445</span>â<span class="cit-lpage">458</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.185579.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185579.114%26rft_id%253Dinfo%253Apmid%252F25589440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzMvNDQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.15" data-doi="10.1038/nbt.4060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miga</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rand</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Nanopore sequencing and assembly of a human genome with ultra-long reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">338</span>â<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.4060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D36%26rft.spage%253D338%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.4060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.4060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitzer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Improved maize reference genome with single-molecule technologies.</span> <abbr class="cit-jnl-abbrev">Nature Publishing Group</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">524</span>â<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BPublishing%2BGroup%26rft.volume%253D546%26rft.spage%253D524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.17" data-doi="10.1038/sdata.2014.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peluso</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babayan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeadon</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Long-read, whole-genome shotgun sequence data for five model organisms.</span> <abbr class="cit-jnl-abbrev">Sci. Data</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">140045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sdata.2014.45</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BData%26rft.volume%253D1%26rft.spage%253D140045%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsdata.2014.45%26rft_id%253Dinfo%253Apmid%252F25977796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sdata.2014.45&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25977796&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.18" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>â<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.19" data-doi="10.1093/bioinformatics/btw152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2103</span>â<span class="cit-lpage">2110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw152%26rft_id%253Dinfo%253Apmid%252F27153593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153593&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D3%26rft.spage%253D321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-8.10.3131.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-comment">Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. May 10 [Epub ahead of print]</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty191</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.22" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>â<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.23" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>â<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.24" data-doi="10.1554/0014-3820(2002)056[2262:GOADIP]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Llopart</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elwyn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lachaise</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Genetics of a difference in pigmentation between <em>Drosophila yakuba</em> and <em>Drosophila santomea</em>.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2262</span>â<span class="cit-lpage">2277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2002.tb00150.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DLlopart%26rft.auinit1%253DA.%26rft.volume%253D56%26rft.issue%253D11%26rft.spage%253D2262%26rft.epage%253D2277%26rft.atitle%253DGenetics%2Bof%2Ba%2Bdifference%2Bin%2Bpigmentation%2Bbetween%2BDrosophila%2Byakuba%2Band%2BDrosophila%2Bsantomea.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282002%2529056%255B2262%253AGOADIP%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12487356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2002)056[2262:GOADIP]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12487356&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179683100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.10.3131.25" data-doi="10.1038/s41467-018-03016-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jupe</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bemm</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motley</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandoval</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">High contiguity <em>Arabidopsis thaliana</em> genome assembly with a single nanopore flow cell.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">541</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-03016-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-03016-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-03016-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.26" data-doi="10.1093/bioinformatics/bty266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mikheenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prjibelski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Versatile genome assembly evaluation with QUAST-LG.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">i142</span>â<span class="cit-lpage">i150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty266</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253Di142%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.27" data-doi="10.1101/gr.139873.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">SchlÃ¶tterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genome-wide patterns of natural variation reveal strong selective sweeps and ongoing genomic conflict in <em>Drosophila mauritiana</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">99</span>â<span class="cit-lpage">110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.139873.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.139873.112%26rft_id%253Dinfo%253Apmid%252F23051690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjIzLzEvOTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.28" data-doi="10.1093/molbev/mss150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Obbard</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maclennan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OâGrady</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Estimating Divergence Dates and Substitution Rates in the Drosophila Phylogeny.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3459</span>â<span class="cit-lpage">3473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mss150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss150%26rft_id%253Dinfo%253Apmid%252F22683811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22683811&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310167700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.29" data-doi="10.1093/gbe/evt034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ometto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cestaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramasamy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grassi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Revadi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Linking genomics and ecology to investigate the complex evolution of an invasive Drosophila pest.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">745</span>â<span class="cit-lpage">757</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evt034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt034%26rft_id%253Dinfo%253Apmid%252F23501831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23501831&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.30" data-doi="10.1534/genetics.117.300583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">OâGrady</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeSalle</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Phylogeny of the Genus Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300583</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300583%26rft_id%253Dinfo%253Apmid%252F29716983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMjA5LzEvMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzEzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.31" data-doi="10.1534/genetics.115.180208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perkins</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holderbaum</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tao</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sopko</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">The Transgenic RNAi Project at Harvard Medical School: Resources and Validation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">843</span>â<span class="cit-lpage">852</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.180208</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.180208%26rft_id%253Dinfo%253Apmid%252F26320097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzMvODQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.32" data-doi="10.1093/femsyr/fox074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salazar</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorter de Vries</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Broek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijsman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de la Torre CortÃ©s</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Nanopore sequencing enables near-complete <em>de novo</em> assembly of <em>Saccharomyces cerevisiae</em> reference strain CEN.PK113â7D.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/femsyr/fox074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffox074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/fox074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.33" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">SimÃ£o</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>â<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.34" data-doi="10.1038/nmeth.4184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuzarte</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dursi</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Detecting DNA cytosine methylation using nanopore sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">407</span>â<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4184</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D407%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.35" data-doi="10.1534/g3.118.200162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solares</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalsow</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Rapid low-cost assembly of the <em>Drosophila melanogaster</em> reference genome using low-coverage, long-read sequencing.</span> <span class="cit-comment">G3 (Bethesda)</span> <span class="cit-vol">8</span>: <span class="cit-fpage">3143</span>â<span class="cit-lpage">3154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200162</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200162%26rft_id%253Dinfo%253Apmid%252F30018084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjgvMTAvMzE0MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzEzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.36" data-doi="10.1534/g3.116.038885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crocker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frankel</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kappes</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Genetic and Transgenic Reagents for <em>Drosophila simulans</em>, <em>D. mauritiana</em>, <em>D. yakuba</em>, <em>D. santomea</em>, and <em>D. virilis</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1339</span>â<span class="cit-lpage">1347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.038885</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.038885%26rft_id%253Dinfo%253Apmid%252F28280212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNC8xMzM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.37" data-doi="10.1093/molbev/msg236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Temporal patterns of fruit fly (Drosophila) evolution revealed by mutation clocks.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">36</span>â<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msg236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg236%26rft_id%253Dinfo%253Apmid%252F12949132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msg236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12949132&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189149300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-book" id="cit-8.10.3131.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-comment">Drosophila 25 species phylogeny</span>. <span class="cit-publ-name">Figshare</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.6084/m9.figshare.5450602.v1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-other" id="cit-8.10.3131.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OâNeil</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-comment">MinION-based long-read sequencing and assembly extends the <em>Caenorhabditis elegans</em> reference genome. Genome Research gr.221184.117</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.40" data-doi="10.1101/gr.214270.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">SoviÄ</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Å ikiÄ</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Fast and accurate de novo genome assembly from long uncorrected reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">737</span>â<span class="cit-lpage">746</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.214270.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.214270.116%26rft_id%253Dinfo%253Apmid%252F28100585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-book" id="cit-8.10.3131.41" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouelliel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">9</span>: <span class="cit-fpage">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-8.10.3131.42" data-doi="10.1038/s41598-016-0001-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Z. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">DBG2OLC: Efficient Assembly of Large Genomes Using Long Erroneous Reads of the Third Generation Sequencing Technologies.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-016-0001-8%26rft_id%253Dinfo%253Apmid%252F27503664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-016-0001-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27503664&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3131.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3119" title="An Anopheles stephensi Promoter-Trap: Augmenting Genome Annotation and Functional Genomics" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3143" title="Rapid Low-Cost Assembly of the Drosophila melanogaster Reference Genome Using Low-Coverage, Long-Read Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3131" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VTZBl4Qq5DCo5Msb_gY1ZexYCV9e1byzkn4vFFgmTPQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3131" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3131" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3131" class="active" data-icon-position="" data-hide-link-title="0">Highly Contiguous Genome Assemblies of 15 Drosophila Species Generated Using Nanopore Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490958" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SzG-RDOmCSC-P8Dc_KjWstQ9yp0mskyGh1JRZGRejOM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3131.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3131" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kKU8PHaRrbQvfKp5InDa-uCQ4iWFVg5DGh4uggmBcEU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3131" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3131.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490958" id="node490958--2" data-pisa="ggg;8/10/3131" data-pisa-master="ggg;g3.118.200160" data-apath="/ggg/8/10/3131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Highly Contiguous Genome Assemblies of 15 <em>Drosophila</em> Species Generated Using Nanopore Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Cynthia Staber</span>, <span class="highwire-citation-author" data-delta="2">Julia Zeitlinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Scott Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3131-3141;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200160 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490958--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>â </span>MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danny.miller@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">danny.miller@seattlechildrens.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cynthia Staber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cynthia%2BStaber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Staber%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACynthia%2BStaber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Julia Zeitlinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Pathology, University of Kansas Medical Center, Kansas City, KS 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julia%2BZeitlinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeitlinger%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulia%2BZeitlinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">R. Scott Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BScott%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Scott Hawley</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3131" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3131&amp;title=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3131&amp;title=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3131&amp;title=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3131&amp;text=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3131&amp;title=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3131&amp;t=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3131&amp;title=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3131&amp;title=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490958/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490958" id="node490958--3" data-pisa="ggg;8/10/3131" data-pisa-master="ggg;g3.118.200160" data-apath="/ggg/8/10/3131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Highly Contiguous Genome Assemblies of 15 <em>Drosophila</em> Species Generated Using Nanopore Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Danny E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Cynthia Staber</span>, <span class="highwire-citation-author" data-delta="2">Julia Zeitlinger</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>R. Scott Hawley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3131-3141;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200160 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490958--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danny E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>â </span>MD-PhD Physician Scientist Training Program, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danny%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADanny%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6096-8601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Danny E. Miller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danny.miller@seattlechildrens.org" class="" data-icon-position="" data-hide-link-title="0">danny.miller@seattlechildrens.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Cynthia Staber </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cynthia%2BStaber%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Staber%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACynthia%2BStaber%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Julia Zeitlinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Pathology, University of Kansas Medical Center, Kansas City, KS 66160</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julia%2BZeitlinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeitlinger%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulia%2BZeitlinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">R. Scott Hawley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Stowers Institute for Medical Research, Kansas City, MO</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BScott%2BHawley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hawley%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR.%2BScott%2BHawley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6478-0494" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for R. Scott Hawley</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3131&amp;rft.epage=3141&amp;rft.atitle=Highly%20Contiguous%20Genome%20Assemblies%20of%2015%20Drosophila%20Species%20Generated%20Using%20Nanopore%20Sequencing&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Miller%2C+Danny+E.&amp;rft.au=Staber%2C+Cynthia&amp;rft.au=Zeitlinger%2C+Julia&amp;rft.au=Hawley%2C+R.+Scott"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490958/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490958/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490958/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490958/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490958/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490958/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490958/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490958/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490958/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490958/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490958/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490958/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3131" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3131.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3131.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/10/3131.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
